<!DOCTYPE html>

<html lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width, initial-scale=1.0" name="viewport"/>
<title>paper-trackr newsletter</title>
<link href="../docs/photo_51.ico" rel="shortcut icon"/>
<link href="../style.css" rel="stylesheet"/>
<link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&amp;display=swap" rel="stylesheet"/>
</head>
<body>
<aside class="sidebar">
<div class="sidebar-inner">
<a class="logo" href="../index.html">felipevzps</a>
<nav class="nav">
<a href="../index.html#about">About</a>
<a href="../index.html#research">Research</a>
<a href="../index.html#hackathon">Hackathon</a>
<a href="../index.html#contact">Contact</a>
</nav>
</div>
</aside>
<main class="newsletter-container">
<section class="newsletter-intro">
<h1>Latest in Science</h1>
<p>
        Hello, welcome to your daily dose of research!<br/> 
        Here you'll find some papers published yesterday, filtered for the topics I follow.
      </p>
<p class="note">
<small>
          * The curation is done automatically by my open-source project, <a href="https://github.com/felipevzps/paper-trackr" target="_blank">paper-trackr</a>.<br/>
          Want your own results? Install it via PyPI and get personalized updates!
        </small>
</p>
</section>
<!-- Filtro de keywords -->
<div class="filter-bar" id="filterBar"></div>
<section class="papers-list" id="papersList"><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Jan 13, 2026</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">DIVAS: an R package for identifying shared and individual variations of multiomics data</h2>
<p class="paper-authors">Sun, Y., Marron, J. S., +1 author, Mao, J.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Motivation: Multiomics data integration aims to identify biological patterns shared across different molecular modalities. Traditional methods focus on detecting either jointly shared variation (across all modalities) or individual variation (unique to single modalities), but overlook partially shared variation, shared by subsets of modalities. This limitation is critical because many biological mechanisms manifest across only some, not all, molecular modalities. Systematically identifying these partially shared patterns is essential for comprehensive understanding of complex biological systems. Results: We developed an open-source R package that implements DIVAS, a computational framework for systematically identifying jointly shared, partially shared and individual variations across multiple data types. DIVAS employs angle-based subspace analysis with rigorous statistical inference through rotational bootstrap. Unlike existing methods, DIVAS hierarchically searches through all possible combinations of data modalities, providing complete decomposition of multiomics data into interpretable components (modes of variation) with both scores and loadings. Application to multimodal COVID-19 data demonstrates DIVAS's ability to reveal biologically meaningful partially shared immune and metabolic dysregulation patterns that underpins disease severity and would be missed by conventional integration approaches. Availability and implementation: DIVAS is available as an R package on GitHub (https://github.com/ByronSyun/DIVAS_Develop). A step-by-step vignette is available on GitHub (https://byronsyun.github.io/DIVAS_COVID19_CaseStudy/).</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2026.01.12.698985v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Jan 13, 2026</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">A Realistic Simulation Framework for EvaluatingMicrobiome Normalization in Sample Stratification and Differential Abundance</h2>
<p class="paper-authors">Al Khafaji, A. K., Llorente, C. G., Camacho, J.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> </p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2026.01.13.699216v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Jan 13, 2026</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Short linear motifs - Unexplored players driving Toxoplasma gondii infection</h2>
<p class="paper-authors">Alvarado Valverde, J., Lapouge, K., +3 authors, Gibson, T.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Pathogens infect hosts by interacting with host proteins and exploiting their functions to their advantage. Short linear motifs, small functional regions within intrinsically disordered protein regions, are common mediators of host-pathogen protein interactions. While motifs have been more extensively studied in viruses and bacteria, the extent to which eukaryotic unicellular parasites use motifs during infection remains unexplored. Toxoplasma gondii is a widespread intracellular Apicomplexan parasite capable of infecting all warm-blooded animals and invading any of their nucleated cells. Toxoplasma's secreted proteins are key in interacting with host proteins during infection, making them potential sources for motifs. To highlight the role of motifs in Toxoplasma gondii infection, we curated 20 known motif instances in Toxoplasma proteins from the scientific literature. To identify more motifs in Toxoplasma secreted proteins, we developed a computational pipeline that annotates putative motif matches with structural and functional features. Through this approach, we identified a set of 27,832 motif matches in 296 secreted proteins. We highlight strategies for further prioritisation of likely functional motif matches by focusing on integrin motifs, degrons and TRAF6-binding motifs. We subjected four predicted TRAF6-binding motifs to experimental validation, supporting the predicted motifs in the Toxoplasma proteins RON10 and GRA15. Our motif predictions provide a valuable resource for generating hypotheses and designing experiments to study infection mechanisms. The characterisation of motifs in Toxoplasma will be key to understanding the molecular principles underlying its broad host range and more comprehensive Apicomplexan infection strategies.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2026.01.12.699020v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Jan 13, 2026</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">UniWave: A Waveform-Based Encoding Framework for Nucleic Acid Feature Extraction</h2>
<p class="paper-authors">Zhang, H., Qi, Y., Wang, L.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Nucleic acid sequence analysis constitutes a core research area in biomedical and health informatics, playing a critical role in infectious disease surveillance, epigenetic regulation, and genomic biomarker discovery. However, most existing sequence encoding methods rely on discrete representations, which are inadequate for capturing the intrinsic continuous structural properties of biological sequences. Inspired by waveform representations in modern physics, we propose UniWave, a novel encoding framework that converts discrete nucleotide sequences into biologically informative one-dimensional continuous waveforms through base mapping, windowed sinc interpolation, and wavelet-based downsampling. To enable more efficient spatiotemporal modeling, we design a learnable positional encoding module, WavePosition, which incorporates positional information to project the one-dimensional waveform into a two-dimensional continuous representation. In addition, we develop a lightweight dual-attention network, InceptionTime-ATT, to facilitate efficient multi-scale extraction of biological features. Experiments conducted on multiple representative genomic benchmark tasks demonstrate that UniWave consistently and significantly outperforms conventional encoding methods in viral genotype classification, cross-species enhancer identification, epigenetic modification site detection, and bacterial genome classification. Moreover, UniWave maintains excellent robustness under perturbation and noise stress tests, while its low-dimensional continuous representation further contributes to reduced model complexity. In summary, UniWave establishes a novel paradigm for biomedical nucleic acid sequence encoding that is interpretable, noise-resilient, and computationally efficient, offering new avenues for uncovering disease-associated genomic patterns and advancing data-driven biomedical research.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2026.01.12.698567v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Jan 13, 2026</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Hierarchical Encoding of Regulatory Mechanisms and Expression Syntax by a foundational genomic sequence-to-function model</h2>
<p class="paper-authors">Li, J.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Deciphering the comprehensive regulatory code encoded in the genomic sequence remains a central challenge in functional genomics, requiring a paradigm shift from descriptive annotations to a mechanistic understanding of transcriptional regulation. Here, we introduce HERMES (Hierarchical Encoding of Regulatory Mechanisms and Expression Syntax), a framework that progressively defines a fundamental sequence vocabulary and parses gene expression syntax into transparent biological insights for the complex regulatory genome. Specifically, we trained a foundational sequence-to-function model on a massive compendium of 137,127 functional genomics profiles spanning diverse biochemical marks and cellular conditions, including DNA methylation, TF binding, polymerase binding, histone marks, chromatin accessibility and RNA expression across various tissues, cell lines and cell types. By harnessing the high-fidelity sequence representations, we established a context-specific regulatory vocabulary comprising 40 distinct sequence classes and fine-grained subclusters spanning the entire genome. Interrogating this vocabulary revealed the sequence determinants of diverse promoters and tissue-specific enhancers, and identified a robust promoter class specific to housekeeping genes, driven by ETS, YY1 and CCAAT motifs. To further decode transcriptional regulation, HERMES was leveraged to predict cell-type-specific gene expression and the enhancer perturbation effects. Extensive evaluations confirmed that the cis-regulatory elements and their interactions were captured to predict gene expression underlying different cellular conditions, and simultaneously revealed divergent enhancer dependencies between housekeeping genes (HKGs) and highly variable genes (HVGs). Ultimately, we distilled the complex sequence-to-function model into a biologically interpretable rulebook of Enhancer-Promoter interaction grammar. Synthesizing all the insights, we propose a unified compatibility model, where HKGs utilize a strong-promoter architecture for high-output expression, while HVGs depend on a context-dependent syntax driven by compatible promoters and enhancers. In summary, HERMES bridges the gap between predictive modeling and biological mechanism, transforming sequence representations into a comprehensive functional encyclopedia and a quantitative grammar of the complex regulatory genome.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2026.01.10.698352v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Jan 13, 2026</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Unlocking Bacterial Cellulose Functionalisation: Comparative Genetic and Co-Culture Strategies in Komagataeibacter</h2>
<p class="paper-authors">Kishkevich, A., Gilmour, K., +4 authors, Ellis, T.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Bacterial cellulose (BC) is produced by diverse bacterial species, including members of the genus Komagataeibacter, and has emerged as a potential sustainable alternative to conventional materials such as plastics and leather. Its production by bacteria is rapid and easily scalable, and via genetic modifications or culturing the bacteria with other microbes, it is possible to rationally enhance the BC that is produced, for example by adding functional components. Here, we systematically evaluated BC functionalization strategies, including genetic engineering and co-culturing with other engineered microbes across the most widely used Komagataeibacter species: K. rhaeticus, K. xylinus, K. medellinensis and K. sucrofermentans. We established that all tested strains are amenable to DNA transformation and capable of expressing heterologous genes from identical genetic constructs. However, we identified species-specific differences in heterologous gene expression, cellulose production in varying environmental conditions and in the abilities of the strains to co-culture with Escherichia coli. Additionally, we demonstrated that all these cellulose-producing bacteria can establish functional symbiotic co-cultures with yeast. While inter-species variations in heterologous gene expression and co-culture dynamics are evident, BC modification through genetic engineering and co-culturing strategies remains achievable across all tested Komagataeibacter species. Our work provides a comparative framework to guide researchers in selecting optimal species based on their specific application requirements.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2026.01.12.699149v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Jan 13, 2026</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Protein-peptide Interaction Representation Learning with Pretrained Language Models</h2>
<p class="paper-authors">Zhan, X., Zhai, S., +4 authors, Siu, S. W. I.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Protein-peptide Interactions (PpIs) paly essential roles in diverse cellular processes, yet their systematic identification remains challenging due to the limited availability of experimentally annotated protein-peptide interaction data. To address this challenge, we present PepInter, a sequence-based Deep Learning (DL) framework that leverages large-scale pretraining on structurally derived pseudo protein-peptide pairs to learn interaction-relevant representations. Specifically, energy-dominant peptide fragments are extracted from protein complexes curated from non-redundant Protein Data Bank (PDB) structures, enabling the construction of pseudo protein-peptide interaction pairs that capture interface interaction patterns shared with canonical protein-protein interactions. This strategy allows the model to acquire interaction-aware priors in the absence of large-scale annotated protein-peptide complex datasets. Built upon the ESM-Cambrian (ESMC) architecture, PepInter adopts a two-stage pretraining strategy. In the first stage, masked language modeling is used to learn general protein sequence representations. In the second stage, the model is further trained to predict Rosetta-derived energetic scores, explicitly incorporating structural interaction signals into the learned embeddings. Following pretraining, PepInter is fine-tuned for both protein-peptide interaction classification and peptide bioactivity regression tasks. Across multiple benchmark datasets, including protein-peptide binding affinity prediction, PepInter consistently outperforms existing baseline methods and demonstrates strong generalization in identifying biologically meaningful PpIs. Case studies further highlight its ability to recover known interaction patterns and predict novel protein-peptide interactions. Together, these results establish PepInter as a scalable and effective framework for protein-peptide interaction prediction, with strong potential to accelerate peptide-based drug discovery.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2026.01.12.699173v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Jan 13, 2026</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">caRBP-Pred: Deep Learning-based Prediction of Chromatin-Associated RNA-Binding Proteins Using Short Peptide Sequences</h2>
<p class="paper-authors">SUN, Q., Wang, H., +2 authors, Yang, F.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> RNA-binding proteins (RBPs) are pivotal in cellular processes ranging from RNA metabolism to 3D genome organization. A distinct subset, chromatin-associated RBPs (caRBPs), binds directly to chromatin to function as transcriptional regulators. However, identifying caRBPs via traditional methods like Chromatin Immunoprecipitation Sequencing (ChIP-seq) and Mass Spectrometry (MS) is labor-intensive and costly. While computational tools for DNA- and RNA-binding protein (DRBP) prediction exist, they often rely on outdated Gene Ontology annotations and fail to capture the unique characteristics of chromatin association. Here, we introduce caRBP-Pred, a novel deep learning approach combining Convolutional Neural Networks (CNN) and Bidirectional Long Short-Term Memory networks (BiLSTM). Unlike previous methods utilizing full-length sequences, our model is trained on chromatin-contact peptides derived from mouse embryonic stem cells (mESCs). caRBP-Pred achieves a superior Area Under the Curve (AUC) of 0.81 using peptide sequence information alone, significantly outperforming existing DRBP predictors which exhibit low recall for caRBPs. We further predicted 52 potential caRBPs in mice. Notably, validation against human homologs confirmed that our model accurately predicts candidates with experimentally verified chromatin-binding capabilities. Collectively, caRBP-Pred is the first tool specifically designed to predict caRBPs based on chromatin-contact peptides, offering a valuable resource for investigating regulatory roles of caRBPs on transcription.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2026.01.12.698323v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Jan 13, 2026</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Generative single-cell transcriptomics via large language models</h2>
<p class="paper-authors">Choi, H., Shin, H., +1 author, Lee, D.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Single-cell and spatial transcriptomics have generated vast atlases of cellular states, yet these data are almost exclusively used for analysis rather than generation. Here we introduce the LLM-based model PGL, Portraying Gene Language, a framework that reframes single-cell transcriptomes as a language generation modeling problem. PGL represents each cell as a long sequence of gene expression tokens and uses a large language model to synthesize complete single-cell RNA-seq profiles from metadata alone, such as tissue and disease context. PGL-generated cells recapitulate dataset-specific transcriptomic structure, align with known cancer subtype biology, and mix coherently with real single-cell datasets. Notably, generated cells can be used as effective references for spatial transcriptomics, enabling accurate cell-type mapping without matched single-cell atlases. By shifting single-cell modeling from representation learning to cell generation, PGL enables virtual cohort construction, hypothesis generation, and reference-on-demand analysis, positioning generative language models as foundational tools for in silico single-cell biology.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2026.01.12.699186v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Jan 13, 2026</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">DPI-score: A deep learning-based metric for assessing protein-protein interfaces in cryo-EM derived assemblies</h2>
<p class="paper-authors">Bhujel, N., Whyatt, N., +4 authors, Malhotra, S.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Advances in sample preparation, data acquisition, and data processing have led to a surge of high-resolution cryoEM (cryogenic electron microscopy) derived structures. Although the global resolution continues to improve, local resolution within a map generally varies, and many structures are still resolved at intermediate resolution. Model building and refinement are usually challenging at lower resolutions ([≥]3 Å), and therefore, atomic model validation is crucial. CryoEM-derived assemblies often contain extensive protein-protein interfaces, yet no established metrics specifically assess the quality of these interfaces. Existing metrics typically focus on fit to density or overall model quality, such as geometry, but do not directly evaluate the protein-protein interfaces in these assemblies. To address this, a Deep learning-based Protein Interface score (DPI-Score) is presented. The proposed approach only requires the raw structural coordinates of the interface atoms without any derived features for training and inference, and achieves 87.53% validation accuracy. The developed interface focused, density independent metric is systematically compared to existing protein interface scoring functions and density dependent model validation scores. It is extensively tested for its performance to distinguish 'target-like' interfaces from a pool of predicted models (cryoEM oligomeric targets from CASP15 and CASP16) and score fitted structures generated using rigid body fitting. We further applied it to assess 29,120 interfaces from the 6,011 fitted entries (worse than 3 Å) associated with depositions in the Electron Microscopy Data Bank. DPI-score provides complementary information to other scores and is able to detect errors at the interfaces in modelled assemblies that were not identified using density-based scores alone. The source code for model validation is freely available for academic use on GitLab (https://gitlab.com/ccpem/dpi) or via the CCP-EM software suite, Doppio, at https://www.ccpem.ac.uk/software.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2026.01.12.699015v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, cancer biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, cancer biology</span>
<span class="date">Jan 13, 2026</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Synthesis and evaluation of novel N,N-dialkylcinnamic acid-based mitochondrial pyruvate carrier inhibitors: Biosynthetic and energetic lethality of targeting metabolic plasticity in cancer</h2>
<p class="paper-authors">Jonnalagadda, S., Ronayne, C. T., +2 authors, Mereddy, V. R.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Novel functionalized cyanocinnamic acid based MPC inhibitors based on pharmacologically privileged N-piperazinyl and N-piperidinyl drug templates have been synthesized for potential cancer treatment. In vitro cell proliferation inhibition studies with these derivatives 2-4 show activity in the low micromolar range. Seahorse XFe96 based mitochondrial stress tests also illustrate the ability of 2-4 to potently and acutely inhibit numerous parameters of mitochondrial respiration in MDA-MB-231, WiDr, and 4T1 cells. Further analyses of the lead compound 3 in permeabilized 4T1 cells provide evidence of specific inhibition of pyruvate driven respiration without affecting glutamate or succinate fueled respiratory processes. Combination studies with GLUT1 inhibitor BAY-876 illustrate the capacity of compound 3 to inhibit metabolic plasticity in triple negative breast cancer MDA-MB-231 cells and is synergistic in inhibiting cell proliferation in aggressive stage IV breast cancer 4T1.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2026.01.12.699093v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, cancer biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, cancer biology</span>
<span class="date">Jan 13, 2026</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Genome wide analysis of YBX1-mediated redistribution of JMJD6 in ER+ breast cancer cells</h2>
<p class="paper-authors">Gupta, A., Bhardwaj, S., Desai, K. V.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> The molecular mechanisms by which cancer cells proliferate under diminished ER expression leading to intrinsic and acquired resistance to endocrine therapy remain incompletely understood. Here, we identify a previously unrecognized transcriptional axis centred on JMJD6 and YBX1 that cooperatively reprograms chromatin, redistributes enhancer engagement and sustains tumor proliferation when JMJD6 levels are elevated (JOE cells). Through proximity ligation, ChIPreChIP assays, KO-based ablation experiments, and genome-wide profiling, we demonstrate that YBX1 is required for JMJD6 enrichment at regulatory sites. ~20% of JMJD6 and YBX1 peaks physically superimpose, and remaining peaks are contained within a neighbourhood of 50kb. A shared extended binding site (EBS) containing (i) flanking polyA/polyT tracts; (ii) a canonical YBX1 recognition sequence; and (iii) a second complementary motif enriched for zinc-finger proteins defines JMJD6/YBX1 regulated enhancers. Histone modification landscapes in JMJD6 overexpressing cells supported the presence of a transcription permissive chromatin. Further, JMJD6/YBX1 siRNA and overexpression analysis in MCF7 cells identified estrogen and interferon gamma signaling as prominent pathways. The most compelling observation was that ER loss was transcriptional in JOE cells and was accompanied by the emergence of JMJD6/YBX1 enhancer sites at alternate genomic positions relative to canonical ER/JMJD6 sites described in estrogen-stimulated cells. These findings offer a mechanistic insight into earlier reports from our group showing that high JMJD6 with YBX1 suppresses ER expression and this could promote endocrine resistance.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2026.01.12.698335v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, cancer biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, cancer biology</span>
<span class="date">Jan 13, 2026</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">PROTAC-Driven Protective Therapy increases the therapeutic window of anticancer drugs</h2>
<p class="paper-authors">Simon-Carrasco, L., Raya, S., +3 authors, Lopez-Contreras, A. J.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Targeted protein degradation is emerging as a powerful anticancer therapy, mostly focused on eliminating oncogenic drivers. In contrast, we propose using PROTACs that exploit E3 ligase defects in cancer cells to selectively protect healthy tissues from the dose-limiting toxicity of anticancer drugs. We term this approach PROTAC-Driven Protective Therapy (PDPT). PDPT consists of a combinatorial treatment of a given anticancer compound with a PROTAC that promotes the degradation of proteins required for the drug-induced toxicity. Potential targets of protective PROTACs include drug uptake transporters, enzymes activating pro-drugs, and the actual drug target in cases that mediates the drug-induced toxicity. Notably, these protective PROTACs must be designed to recruit E3 ligases that are mutated or defective in the cancer cells while remain active in healthy tissues. As a proof of concept of our strategy, we used CRBN-recruiting and VHL-recruiting PROTACs to demonstrate that PARP1 degradation alleviates the cytotoxicity of PARP inhibitors (PARPi) in E3 ligase-proficient cells, while E3 ligase-deficient cancer cells remain fully sensitive. Remarkably, PDPT also protects primary human bone marrow progenitors from PARPi-induced toxicity, which are the most clinically relevant cells affected by PARPi-associated side effects in cancer patients, supporting the clinical relevance of this strategy. We further uncover that TP53-mutant cancers display critically low expression of the E3 ligase MDM2 and show inefficient MDM2-recruiting PROTAC activity. This tumor-intrinsic feature enables PDPT using MDM2-recruiting PROTACs in TP53-mutant cancers. PDPT opens a new direction for targeted protein degradation by improving tolerability and expanding the therapeutic window of both established and future cancer therapies.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2026.01.12.698947v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, cancer biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, cancer biology</span>
<span class="date">Jan 13, 2026</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Mapping lineage and functional diversity in the high-risk human mammary epithelium</h2>
<p class="paper-authors">Waas, M., Zhang, B., +8 authors, Kislinger, T.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> </p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2026.01.12.699075v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, cancer biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, cancer biology</span>
<span class="date">Jan 13, 2026</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">A strategically designed homogeneous antibody-drug conjugate improves safety and therapeutic index in renal cell carcinoma</h2>
<p class="paper-authors">Thurakkal, L., Velayutham, K., +3 authors, Saha, T.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Renal cell carcinoma (RCC) is one of the leading causes of kidney cancer-related mortalities worldwide. Though the treatment method of RCC has advanced from nephrectomy to targeted therapy and immunotherapy, most patients suffer from therapeutic resistance, metastasis, and relapse. In recent days, antibody drug conjugates (ADCs) have emerged as a promising treatment modality by delivering cytotoxic payloads specifically to the cancer cells. However, previous efforts related to ADC have failed in the clinic because of severe toxicity. Here, we introduced a homogenous ADC (LT-025), which can target the kidney injury molecule (KIM) -1 receptor on RCC cells. We adopted the enzymatic conjugation involving microbial transglutaminase (MTGase) to attach the cytotoxic drug DM1 to the Fc region of the KIM-1 antibody. We demonstrated a site-specific conjugation of the drug-linker with a specific drug-to-antibody ratio (DAR) of 2. The ADC exhibited prolonged stability and superior antigen-binding capability, characterized using various bioanalytical methods. We proved that cellular internalization and cytotoxicity are dependent on the KIM-1 expression on RCC. The LT-025 displayed superior cytotoxicity to the metastatic RCC cells, then standard RCC treatment by sunitinib. The combination therapy of LT-025 and sunitinib showed increased therapeutic efficacy in vitro in RCC as well as in sunitinib-resistant RCC models. The LT-025 exhibited a safe and effective treatment in the syngeneic RCC mouse model. The combination therapy also showed an increased tumor growth inhibition in preclinical mouse model studies. A carefully designed KIM-1 targeting ADC can emerge as an effective treatment method for metastatic RCC.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2026.01.12.699160v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="machine learning, cancer biology">
<div class="paper-meta">
<span class="keyword">machine learning, cancer biology</span>
<span class="date">Jan 13, 2026</span>
<span class="journal">PubMed</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Improving rectal tumor segmentation with anomaly fusion derived from anatomical inpainting: a multicenter study.</h2>
<p class="paper-authors">Lishan Cai, Mohamed A Abdelatty, +7 authors, Jonas Teuwen</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Accurate rectal tumor segmentation using magnetic resonance imaging (MRI) is paramount for effective treatment planning. It allows for volumetric and other quantitative tumor assessments, potentially aiding in prognostication and treatment response evaluation. Manual delineation of rectal tumors and surrounding structures is time-consuming and labor-intensive. Over the past few years, deep learning has shown strong results in automated tumor segmentation in MRI. Current studies on automated rectal tumor segmentation, however, focus solely on tumoral regions without considering the rectal anatomical entities and often lack a solid multicenter external validation. In this study, we improved rectal tumor segmentation by incorporating anomaly maps derived from anatomical inpainting. This inpainting was trained using a U-Net-based model and trained to reconstruct a healthy rectum and mesorectum from prostate T2-weighted images (T2WI). The rectal anomaly maps were generated from the difference between the original rectal and reconstructed pseudo-healthy slices. The derived anomaly maps were used in the downstream tumor segmentation tasks by fusing them as an additional input channel (AAnnUNet). Alternative methods for integrating rectal anatomical knowledge were evaluated as baselines, including Multi-Target nnUNet (MTnnUNet), which added rectum and mesorectum segmentation as auxiliary tasks, and Multi-Channel nnUNet (MCnnUNet), which utilized rectum and mesorectum masks as additional input channels. As part of this study, we benchmarked nine models for rectal tumor segmentation on a large multicenter (num = 705) dataset of preoperative T2WI and nnUNet outperformed the other eight models on the external test. The MTnnUNet demonstrated improvements in both fully-supervised and mixed-supervised settings where human-annoated tumor masks and AI-generated rectum and mesoretum masks were used compared to nnUNet, while the MCnnUNet showed benefits only in the setting where mixed-supervision were used. Importantly, anomaly maps were strongly associated with tumoral regions, and their integration within AAnnUNet led to the best tumor segmentation results across both settings. The effectiveness of AAnnUNet demonstrated the value of the anomaly maps, indicating a promising direction for improving rectal tumor segmentation and model robustness for multicenter data.</p>
<a class="btn dark" href="https://pubmed.ncbi.nlm.nih.gov/41530192/" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div></section>
</main>
<script>
    // === EXPAND / COLLAPSE PAPERS ===
    document.querySelectorAll('.paper').forEach(paper => {
      paper.addEventListener('click', e => {
        if (e.target.tagName.toLowerCase() === 'a') return;
        paper.classList.toggle('expanded');
      });
    });

    // === FILTER SYSTEM ===
    const papers = document.querySelectorAll('.paper');
    const filterBar = document.getElementById('filterBar');

    // Extrai keywords
    const allKeywords = new Set();
    papers.forEach(paper => {
      paper.dataset.keywords.split(',').forEach(k => allKeywords.add(k.trim()));
    });

    // Cria botões dinamicamente
    const filters = ['all', ...Array.from(allKeywords)];
    filters.forEach(keyword => {
      const btn = document.createElement('button');
      btn.className = 'filter-btn';
      btn.textContent = keyword;
      if (keyword === 'all') btn.classList.add('active');
      filterBar.appendChild(btn);
    });

    // Filtro de papers
    filterBar.addEventListener('click', e => {
      if (!e.target.classList.contains('filter-btn')) return;

      document.querySelectorAll('.filter-btn').forEach(b => b.classList.remove('active'));
      e.target.classList.add('active');

      const keyword = e.target.textContent;
      papers.forEach(paper => {
        if (keyword === 'all') {
          paper.style.display = 'grid';
        } else {
          const hasKeyword = paper.dataset.keywords.toLowerCase().includes(keyword.toLowerCase());
          paper.style.display = hasKeyword ? 'grid' : 'none';
        }
      });
    });
  </script>
</body>
</html>
