<!DOCTYPE html>

<html lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width, initial-scale=1.0" name="viewport"/>
<title>paper-trackr newsletter</title>
<link href="../docs/photo_51.ico" rel="shortcut icon"/>
<link href="../style.css" rel="stylesheet"/>
<link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&amp;display=swap" rel="stylesheet"/>
</head>
<body>
<aside class="sidebar">
<div class="sidebar-inner">
<a class="logo" href="../index.html">felipevzps</a>
<nav class="nav">
<a href="../index.html#about">About</a>
<a href="../index.html#research">Research</a>
<a href="../index.html#hackathon">Hackathon</a>
<a href="../index.html#contact">Contact</a>
</nav>
</div>
</aside>
<main class="newsletter-container">
<section class="newsletter-intro">
<h1>Latest in Science</h1>
<p>
        Hello, welcome to your daily dose of research!<br/> 
        Here you'll find some papers published yesterday, filtered for the topics I follow.
      </p>
<p class="note">
<small>
          * The curation is done automatically by my open-source project, <a href="https://github.com/felipevzps/paper-trackr" target="_blank">paper-trackr</a>.<br/>
          Want your own results? Install it via PyPI and get personalized updates!
        </small>
</p>
</section>
<!-- Filtro de keywords -->
<div class="filter-bar" id="filterBar"></div>
<section class="papers-list" id="papersList"><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 05, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">geneSCOPE: gene Spatial Co-Occurrence of Pairwise Expression</h2>
<p class="paper-authors">Zhang, S., Saeki, K., Haeno, H.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Spatial transcriptomics captures neighborhood-dependent gene expression, but existing workflows do not always fully account for measurement scale and often treat space implicitly. We present geneSCOPE, a framework that integrates ecology-inspired statistics with network analysis. Molecules are binned on a grid whose width is chosen near the mode of the per-gene unit-invariant knee (UIK) distribution derived from Morisita's I{delta}--width curves. Pairwise adjacency-weighted spatial association is quantified with Lee's L. We then assemble a spatial gene network and identify gene modules by consensus clustering. To identify cell--cell interactions between different cell types, high Lee's L and low Pearson's r is examined. Applied to human colorectal cancer (three Xenium sections) and a lymph node, geneSCOPE recovered spatial gene modules that map to microanatomy such as invasive margins, luminal epithelium, fibroblast-rich territories and germinal-center subdomains, and highlights intercellular neighborhood patterns at tumor-stroma interfaces between LGR5-marked stem-like tumor programs and C3-centered fibroblast/complement-associated niches. geneSCOPE thus provides a scalable, interpretable analytical foundation that generalizes across spatial omics and supports a wide range of applications.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.03.691993v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 05, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Repurposing the diatom periplastidial compartment for heterologous terpenoid production</h2>
<p class="paper-authors">Patwari, P., Pruckner, F., +1 author, Fabris, M.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Diatoms are promising microorganisms to provide sustainable routes for photosynthetic terpenoid production from CO2, yet their potential for compartmentalized engineering remains largely unexplored. Here, we systematically profiled the biosynthetic capacity of Phaeodactylum tricornutum by targeting representative synthases for hemi, mono, sesqui, and tetraterpenoids to the cytosol, chloroplast, and periplastidial compartment (PPC). This comprehensive analysis revealed that all major prenyl phosphate precursors, DMAPP, GPP, FPP, and GGPP, are accessible in all compartments, including in the PPC and can sustain heterologous flux without major physiological penalties, although production efficiency varies across compartments and product classes. By determining precursor availability, we propose the diatom PPC as a minimal engineerable organelle directly interfaced with a eukaryotic chloroplast. Moreover, we highlight its utility as a unique interface to investigate metabolic exchange between the MVA and MEP pathways. These findings provide a systematic framework for compartment-specific terpenoid engineering in diatoms and open new opportunities for modular pathway assembly and synthetic biology in photosynthetic eukaryotes.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.05.692597v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 05, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">gffutilsAI: An AI-Agent for Interactive Genomic Feature Exploration in GFF files</h2>
<p class="paper-authors">Bassi, S., Gonzalez, C., Yang, T.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> The General Feature Format (GFF) is widely used to represent genomic annotations, but its hierarchical, multi-attribute structure makes manual querying and analysis challenging. Existing libraries such as gffutils provide programmatic interfaces, yet they require coding proficiency. gffutilsAI is a novel AI-powered command-line agent that enables researchers to perform interactive, natural-language-driven exploration of GFF files. Built on top the gffutils library and the Strands AI agent framework, gffutilsAI integrates local and cloud-based large language models (LLMs) such as Llama 3.1, GPT-5, and Claude 3.5 to translate human queries into executable actions. The tool supports coordinate-based queries, attribute and GO searches, hierarchical traversal, statistical summaries, and CSV export, offering a new paradigm for accessible conversational genomics.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.02.690645v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 05, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">SigSpace: an LLM-based agent for drug responsesignature interpretation</h2>
<p class="paper-authors">Khurana, R., Mangla, I., +2 authors, Merico, D.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Agent systems powered by large language models (LLMs) are increasingly applied in computational biology to automate analysis, integrate data, and accelerate discovery. Here, we investigate the capacity of LLM-driven agents to interpret transcriptional response signatures of drug perturbations in cancer cell lines, a task central to understanding drug mechanisms of action (MoAs) and supporting cancer drug discovery. Leveraging the Tahoe-100M dataset of 100 million transcriptomic profiles across 1,100 small-molecule perturbations and 50 cancer cell lines, we developed an LLM-based agent system, SigSpace, that processes differential gene expression signatures and generates concise, human-readable summaries of drug responses. We then tested whether blinded response signature summaries could be correctly matched to their corresponding drug identity or MoA. Our results show that LLM-generated summaries consistently outperform random baselines and that the choice of LLM model and signature score significantly influences performance. These findings highlight the potential of LLMs to enhance interpretation of complex transcriptional data to enable drug discovery. Future directions for improvement include exploring alternative response signature formats, improving summarization fidelity, benchmarking performance across different summary formats and lengths, and broadening applications to additional datasets and predictive tasks in drug discovery.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.02.691945v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 05, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Comparative evaluation of reference genomes and cell-type annotation frameworks for single-nucleus transcriptomic analysis in apple</h2>
<p class="paper-authors">Hu, S., Liu, Y., +10 authors, Jiao, J.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Single-nucleus RNA sequencing (snRNA-seq) provides a powerful tool to profile cell-type-specific transcriptional programs in woody fruit trees. Yet its application in Malus domestica has been limited by heterogeneous reference genomes and the lack of validated marker genes. Here, we first established a comprehensive set of marker genes corresponding to 29 plant cell types and identified their orthologs in apple. Using these resources, we systematically evaluated six publicly available apple genome assemblies as references for snRNA-seq analysis. Our results demonstrate that reference genome selection has a major impact on mapping performance and downstream cell type annotation, with the GDDH13 assembly consistently producing the highest transcript counts per nucleus and the greatest number of identifiable transcriptional clusters (15 clusters representing 11 distinct cell types across 35,557 high-quality nuclei). To achieve robust cell type annotation, we integrated classical marker gene-based annotation with two cross-species computational frameworks--Orthologous Marker Gene Groups (OMGs) and XSpeciesSpanner. These three approaches enabled confident identification of 11 major cell types in apple seedlings and provided cross-validation for ambiguous or previously unannotated clusters. Pseudotime analysis reconstructed the developmental trajectories of both procambial and stomatal lineages, revealing their progressive transcriptional transitions and enabling the delineation of key molecular programs underlying vascular differentiation and early guard-cell formation. Together, this work provides a reference-quality single-nucleus transcriptomic atlas for apple and underlined the critical impact of genome choice and multi-method cell-type annotation on single-cell studies in non-model crops.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.02.691976v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 05, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">MetaPointFinder: Detection of Antimicrobial Resistance-Conferring Point Mutations (ARMs) from Metagenomic Reads</h2>
<p class="paper-authors">Zomer, A. L., Nooij, S., +1 author, Mather, A. E.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> </p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.04.691125v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 05, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">High-Quality Synthetic Annotated Tissue Data Using Conditional Generative Adversarial Networks</h2>
<p class="paper-authors">Srivastava, S., Weijer, C. J., Bretschneider, T.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> We present a deep-learning based pipeline for generating high-quality synthetic instance-segmentation datasets of tissues undergoing early gastrulation in chick embryo. We create point-clouds using the Lennard-Jones potential and learn an image-to-image translation from these point-clouds to create synthetic tissue images and auxiliary flow tensors using a Generative Adversarial Network, from which the segmentation masks are derived. We evaluate the downstream utility of our synthetic data by training Cellpose and Stardist models from scratch under data replacement and data augmentation scenarios, show that the synthetic datasets effectively capture the statistical properties of the real dataset, and show that our synthetic data improves segmentation performance on a held-out test set. This approach substantially reduces expert annotation time, as late-stage gastrulation data are challenging to acquire and manually label, while similar synthetic examples can be flexibly generated from easily obtainable and annotated early-stage data. Code: https://gitlab.com/siddharthsrivastava/synthetic-tissue-data.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.04.690437v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 05, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">VUSVista: Enhancing the Curation of Variants of Uncertain Significance</h2>
<p class="paper-authors">Spiteri, E., Scerri, J., Ebejer, J.-P.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> </p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.02.691669v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 05, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">MS4MS: LLMs-driven Multi-agent System for Small-molecule Identification via LC-MS/MS</h2>
<p class="paper-authors">Guo, N., Guo, J., +8 authors, Zeng, D.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Small molecule identification is central to research fields such as drug discovery, but in complex systems like Traditional Chinese Medicine (TCM), traditional mass spectrometry analysis methods remain constrained by bottlenecks including insufficient database coverage, fragmented analysis workflows, and poor result interpretability. To address these limitations, we developed MS4MS, a large language model-driven multi-agent system that enables an end-to-end automated pipeline from raw data to small molecule identification. Validation on a public benchmark demonstrates that MS4MS achieves state-of-the-art performance in molecular formula prediction. Furthermore, its innovative small molecule identification agent enables efficient and interpretable compound elucidation. Verification using herbal extracts indicates MS4MS's outstanding performance regarding analytical coverage and the discrimination of isomers. Consequently, MS4MS offers a novel, accurate, interpretable, and high-throughput end-to-end automated strategy for small molecule identification, overcoming the analytical bottlenecks of traditional mass spectrometry in natural products and complex TCM systems.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.02.691830v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 05, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Comparative Analysis of Feature Selection Methods for Single-Cell RNA Sequencing Data</h2>
<p class="paper-authors">Alkhadrawi, A., Mahmoud, M. A. B., +1 author, Jaber, A. A.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Feature selection is a critical preprocessing step in single-cell RNA sequencing (scRNA-seq) analysis, directly impacting downstream clustering and biological interpretation. We systematically compared 16 feature selection methods across three diverse datasets: PBMC3K (immune cells), Visium Heart, and Visium Brain (spatial transcriptomics). Methods included established approaches (Seurat HVG variants, CellRanger HVG), statistical methods (Pearson residuals, variance-based, coefficient of variation), supervised methods (ANOVA F-test, mutual information, random forest), and deep learning techniques (IntegratedGradients, DeepLIFT, GradientShap). We evaluated methods based on execution time, feature overlap, marker recovery, and pathway enrichment. Our analysis revealed substantial variability across methods, with mean pairwise overlap of only 23.7%. However, a core set of 1,150 genes was consistently selected by 50% of methods. Supervised methods demonstrated superior recovery of known cell type markers, while unsupervised approaches captured broader biological processes. Deep learning methods identified unique gene sets with strong immune pathway enrichment but higher computational cost. Pathway analysis showed all methods successfully identified relevant biological processes, though with varying emphasis. These findings provide practical guidance for method selection based on analytical goals and highlight the value of ensemble approaches in scRNA-seq feature selection.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.02.691907v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, cancer biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, cancer biology</span>
<span class="date">Dec 05, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Active Liposomal Targeting for Head and Neck Squamous Cell Carcinoma Treatment</h2>
<p class="paper-authors">del Campo-Balguerias, A., Martin-Hernandez, C., +6 authors, Lorz, C.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Head and neck squamous cell carcinoma (HNSCC) is a major oncological challenge due to its high recurrence rates, biological heterogeneity, and poor long-term survival outcomes. Despite advances in surgery and radiotherapy, chemotherapeutic options are limited. Cisplatin (CDDP) remains the mainstay, while small molecule inhibitors, such as the selective PI3K3CA; inhibitor Alpelisib, are still under clinical evaluation. However, systemic side effects and suboptimal tumor specificity hamper the clinical utility of both standard and novel specific therapies. In this study, we developed passive- and active-targeting liposomes encapsulating Cisplatin (CDDP) and Alpelisib. To achieve active targeting, the anti-epidermal growth factor receptor (EGFR) antibody Cetuximab was conjugated to the liposome surface. All formulations were characterized based on size, morphology, encapsulation efficiency, and drug release profiles. In vitro assays demonstrated that liposomal encapsulation preserved the cytotoxic effects of both CDDP and Alpelisib in HNSCC cell lines, with CDDP showing adequate encapsulation efficiency for in vivo evaluation. In vivo near-infrared fluorescence imaging demonstrated that active-targeting liposomes achieved greater tumor accumulation and a better tumor-to-spleen ratio compared to passive formulations. Tumor growth by CDDP-loaded active-targeting liposomes was comparable to that of the free drug, likely due to the hydrophilic nature and rapid release of CDDP from liposomes in physiological conditions. Importantly, blood chemistry analysis indicated a favorable toxicity profile for the targeted liposomal formulation. These findings highlight the potential of liposome-based targeted strategies to improve therapeutic outcomes in HNSCC and underscore the value of nanotechnology in the development of next-generation oncological therapies.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.03.691158v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, cancer biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, cancer biology</span>
<span class="date">Dec 05, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Multiomic characterization, early detection, and therapeutic targeting of myeloid sarcoma</h2>
<p class="paper-authors">Nadorp, B., Lasry, A., +34 authors, Eisfeld, A.-K.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Myeloid sarcoma, an aggressive extramedullary subtype of acute myeloid leukemia (AML), occurs in ~10% of patients, and has not yet been included in large-scale genomic studies. The critical biological changes that drive tumor evolution are unknown, its detection in asymptomatic patients remains a clinical challenge, and treatment options are limited as patients are often excluded from clinical trials, rendering it a neglected disease entity. Based on comprehensive multi-omic profiling, we demonstrate that myeloid sarcoma evolves from medullary AML with distinct site-specific clonal evolution patterns. Additionally, we show that circulating tumor DNA sequencing can serve as a non-invasive method for molecular profiling of myeloid sarcoma, offering a novel avenue in molecular diagnostics. We characterize unique transcriptional profiles of myeloid sarcoma, reflecting immune evasion and adaptation to an extramedullary microenvironment. We provide evidence for a key role of RAS pathway activation and demonstrate in murine models of myeloid sarcoma that RAS inhibition effectively reduces tumor burden. Overall, our data highlight key differences between medullary AML and myeloid sarcoma including universal molecular evolution and RAS pathway activation as hallmarks of the disease and nominate RAS inhibition as a promising therapeutic strategy for patients with myeloid sarcoma.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.04.689069v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, cancer biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, cancer biology</span>
<span class="date">Dec 05, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">LARP1 integrates MYC and mTOR signaling to enable anabolic growth during tumor initiation</h2>
<p class="paper-authors">Fuentes, P., Iannizzotto, F., +11 authors, Gentilella, A.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Tumor initiation requires the integration of oncogenic signals with environmental cues to enable anabolic growth. Among oncogenic drivers, MYC is central to tumorigenesis, with its deregulation observed in more than 60% of human cancers. Oncogenic MYC profoundly rewires transcription, enabling cells to bypass cell cycle checkpoints and resetting metabolism. A cornerstone of this rewiring is the up regulation of biomass-producing pathways, particularly ribosome biogenesis. How and when MYC oncogenic program is translationally executed, either immediately or until a favorable metabolic context emerges, remains a central unanswered question in tumor initiation, limiting our understanding of tumor latency and early intervention windows. Here, we identify the RNA-binding protein LARP1 as a critical effector of MYC-driven transformation, connecting MYC oncogenic activity with mTOR signaling. Mechanistically MYC represses miR-26a/b, relieving post-transcriptional repression of LARP1 and leading to its up regulation. LARP1 associates with the translational machinery loading it with the anabolic translatome induced by MYC in a translationally poised state. Upon permissive mTOR signaling, and dependent on the phosphorylation of LARP1 at serines 689 and 697, this program is rapidly translated, fueling the biosynthetic processes essential for tumor development. Importantly, genetic deletion of LARP1 or pharmacological mTOR inhibition completely abrogates tumor initiation in a genetically engineered colorectal organoid model of MYC-driven tumorigenesis, underscoring the physiological relevance of this two-step mechanism. These findings reveal a fundamental mechanism underlying MYC oncogenesis, whereby LARP1 bridges the anabolic translatome primed by MYC with its metabolic execution controlled by mTOR. By temporally uncoupling transformation from metabolic permissiveness, this mechanism defines a critical checkpoint in early tumorigenesis and reveals a potential vulnerability for intercepting MYC-driven cancer before biomass expansion.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.05.689906v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, cancer biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, cancer biology</span>
<span class="date">Dec 05, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Androgen receptor contributes to radioresistance through DNA repair and autophagy in AR-positive prostate cancer cells</h2>
<p class="paper-authors">Kudo, K.-i., Feng, Q., +10 authors, Li, Y. R.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Androgen receptor (AR) is a critical therapeutic target in prostate cancer (PCa), and androgen blockade is known to act synergistically with radiation therapy. However, the mechanisms through which AR modulates radiation response are not yet fully understood. In this study, we aimed to investigate the role of AR in mediating radioresistance in PCa. AR-positive LNCaP and castration-resistant C4-2 cells exhibited significantly higher radioresistance than AR-negative cells, as determined by apoptosis and cell viability assays. Following irradiation, most LNCaP cells were arrested in the G1 phase, accompanied by rapid p53 activation and p21 induction. Consistently, AR silencing significantly increased radiosensitivity and reduced DNA-PKcs expression and phosphorylation, suggesting that AR enhances DNA repair, likely through non-homologous end joining (NHEJ). At the cellular level, irradiation markedly induced macroautophagy in LNCaP and C4-2 cells, as evidenced by increased LC3B-II accumulation and autophagic vacuole formation, and the upregulation of 11 autophagy-related genes was identified by whole-transcriptomic analysis. To assess their functional relevance, we performed siRNA-mediated knockdown of selected autophagy-related genes and assessed cell viability and Annexin V/PI staining. Notably, BECN1 and LC3 knockdown significantly enhanced radiosensitivity, with BECN1 knockdown showing an effect comparable to that observed with AR silencing. These results suggest that radiation-induced autophagy promotes the survival of AR-positive prostate cancer cells. Moreover, immunohistochemical analysis of ex vivo-irradiated, patient-derived PCa tissues from patients with newly diagnosed high-Gleason score prostate cancer undergoing prostatectomy further demonstrated that radiation-induced autophagy supports the survival of high-grade AR-positive tumor cells. Collectively, our findings reveal that AR promotes radioresistance in PCa by enhancing both DNA repair and autophagy.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.03.690226v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, cancer biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, cancer biology</span>
<span class="date">Dec 05, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Microenvironment-specific modulation of macrophage function and tumour progression by ADAMTS1 through Syndecan-4 shedding</h2>
<p class="paper-authors">Redondo-Garcia, S., Caracuel-Peramos, R., +7 authors, Rodriguez-Manzaneque, J. C.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Recent studies have emphasized the role of ADAMTS proteases in inflammation and immunity, particularly in the context of tumour progression. Since inflammatory cells can either constrain or promote tumour growth, understanding how ADAMTS proteases influence immune cell behaviour is crucial. Using syngeneic tumour models -B16F1 melanoma and Lewis Lung Carcinoma (LLC)- in Adamts1 knockout (Ats1-KO) mice, we observed model-specific outcomes underscoring the complexity of ADAMTS1 function in cancer. Transcriptomic and functional analyses revealed broad alterations in the matrisome and immune-related pathways across both tumour types. Strikingly, while tumour progression was impaired in B16F1-derived tumours, the LLC model -characterized by a stronger myeloid component- showed no dependency on Adamts1. To investigate this apparent resistance, we experimentally depleted macrophages and uncovered a profound functional defect in this population in Ats1-KO mice. In vitro assays confirmed reduced macrophage phagocytic activity. Mechanistically, we identified the transmembrane heparan sulfate proteoglycan syndecan-4 (SDC4), a known ADAMTS1 substrate, as a key mediator of this activity. Together, these findings reveal a previously unrecognized ADAMTS1-SDC4 axis that links extracellular matrix remodelling to macrophage phagocytosis, ultimately shaping tumour cell clearance and tumour progression.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.03.690963v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="machine learning, cancer biology">
<div class="paper-meta">
<span class="keyword">machine learning, cancer biology</span>
<span class="date">Dec 05, 2025</span>
<span class="journal">PubMed</span>
</div>
<div class="paper-content">
<h2 class="paper-title">QSAR machine learning-guided discovery of renieramycin right-half derivatives with cytotoxic activity against NSCLC cells.</h2>
<p class="paper-authors">Ismail Dwi Putra, Phornphimon Maitarad, +5 authors, Pithi Chanvorachote</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> This study explores the structure-activity relationships of renieramycin right-half and full-skeleton compounds using quantitative structure-activity relationship (QSAR) modeling. Linear (Genetic Algorithm-Multiple Linear Regression, GA-MLR) and non-linear machine learning approaches (Random Forest, Support Vector Regression, and XGBoost) were employed to develop predictive models with quantum chemical and molecular descriptors. The best-performing model, using Support Vector Regressor (SVR), achieved a coefficient of determination (R²) of 0.946 for the training set and a root mean square error (RMSE) of 0.246 for the test set. Key descriptors influencing cytotoxicity included charges at C2 and C4, HOMO energy, and polarizability. External validation with newly synthesized renieramycin right-half derivatives yielded an RMSE of 0.236. The machine learning-based QSAR models demonstrated exceptional accuracy and reliability in cytotoxicity prediction, underscoring their utility in guiding the design of novel renieramycin derivatives. The cytotoxicity test of the newly synthesized renieramycin shows an anomaly than the previous experimental findings, i.e., the O-benzyl containing derivative was more cytotoxic than the hydroxyl or quinone containing renieramycin derivatives. These findings highlight the potential of fine-tuned QSAR methodologies to accelerate the development of highly effective anticancer agents based on renieramycin right-half structures.</p>
<a class="btn dark" href="https://pubmed.ncbi.nlm.nih.gov/41350546/" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="machine learning, cancer biology">
<div class="paper-meta">
<span class="keyword">machine learning, cancer biology</span>
<span class="date">Dec 05, 2025</span>
<span class="journal">MED</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Emerging biomarkers in prostate cancer diagnosis and treatment: Insights into genetic, RNA and metabolic markers (Review).</h2>
<p class="paper-authors">Huang Y, Mao J, Li X.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Prostate cancer remains one of the most prevalent malignancies and a major cause of cancer‑related mortality among men worldwide. Despite widespread use of prostate‑specific antigen testing, current diagnostic approaches suffer from low specificity and limited ability to distinguish between indolent and aggressive disease, resulting in overdiagnosis and overtreatment. Advances in molecular biology, genomics and metabolomics have led to the identification of novel biomarkers that have potential for improving the precision of prostate cancer diagnosis, prognosis and therapy. The present review provides a comprehensive overview of emerging prostate cancer biomarkers, including genetic (such as BRCA1/2, HOXB13 and PTEN), RNA‑based (such as PCA3 and miRNAs), metabolic (such as citric acid and polyamines) and methylation markers (such as GSTP1, APC and RASSF1A). These biomarkers not only enhance diagnostic accuracy but also facilitate risk stratification, prediction of therapeutic response and real‑time disease monitoring through liquid biopsy technologies. Moreover, integrating multi‑omics data with artificial intelligence and machine learning may further improve early detection and personalized treatment strategies. Overall, the development and clinical implementation of these biomarkers represent a transformative step toward precision medicine in prostate cancer, enabling earlier diagnosis, optimized therapy selection and improved patient outcomes.</p>
<a class="btn dark" href="https://pubmed.ncbi.nlm.nih.gov/41347816/" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="machine learning, cancer biology">
<div class="paper-meta">
<span class="keyword">machine learning, cancer biology</span>
<span class="date">Oct 30, 2025</span>
<span class="journal">PubMed</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Integrated Proteomics and Metabolomics Profiling Unveils Biomarkers and Immune Characteristics for Pelvic Lymph Node Metastasis in Cervical Cancer.</h2>
<p class="paper-authors">Guanting Pang, Zhao Wang, +10 authors, Yue Feng</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Pelvic lymph node metastasis (PLNM) significantly affects the prognosis of cervical cancer (CC). However, current imaging examinations and serum squamous cell carcinoma antigen (SCCA) testing are inadequate for assessing the pelvic lymph node status in CC. To identify accurate noninvasive biomarkers for diagnosing PLNM and minimizing unnecessary postoperative lymphadenectomy and its associated complications, we performed a comprehensive proteomic and metabolomic analysis of plasma from 124 patients with CC, along with a proteomic analysis of 60 paired tissue samples. Through machine learning methods, we identified potential plasma biomarkers (TTR, MASP2, APOD, and 7α-hydroxy-cholestene-3-one) and constructed a diagnostic model. In the validation cohort, the diagnostic model combined with SCCA exhibited a higher sensitivity (72.4%) than SCCA (64.3%) and imaging examination (14.3%). The plasma protein biomarkers were consistently validated in paired tissue samples. Additionally, immune infiltration analysis demonstrated that CD4 and CD8 T cells were highly infiltrated in the PLNM group, suggesting a potentially enhanced response to immunotherapy. Here, we established a biomarker panel for PLNM and highlighted the altered immune characteristics associated with PLNM, offering valuable insights for the development of immunotherapy strategies for patients with PLNM.</p>
<a class="btn dark" href="https://pubmed.ncbi.nlm.nih.gov/41165339/" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div></section>
</main>
<script>
    // === EXPAND / COLLAPSE PAPERS ===
    document.querySelectorAll('.paper').forEach(paper => {
      paper.addEventListener('click', e => {
        if (e.target.tagName.toLowerCase() === 'a') return;
        paper.classList.toggle('expanded');
      });
    });

    // === FILTER SYSTEM ===
    const papers = document.querySelectorAll('.paper');
    const filterBar = document.getElementById('filterBar');

    // Extrai keywords
    const allKeywords = new Set();
    papers.forEach(paper => {
      paper.dataset.keywords.split(',').forEach(k => allKeywords.add(k.trim()));
    });

    // Cria botões dinamicamente
    const filters = ['all', ...Array.from(allKeywords)];
    filters.forEach(keyword => {
      const btn = document.createElement('button');
      btn.className = 'filter-btn';
      btn.textContent = keyword;
      if (keyword === 'all') btn.classList.add('active');
      filterBar.appendChild(btn);
    });

    // Filtro de papers
    filterBar.addEventListener('click', e => {
      if (!e.target.classList.contains('filter-btn')) return;

      document.querySelectorAll('.filter-btn').forEach(b => b.classList.remove('active'));
      e.target.classList.add('active');

      const keyword = e.target.textContent;
      papers.forEach(paper => {
        if (keyword === 'all') {
          paper.style.display = 'grid';
        } else {
          const hasKeyword = paper.dataset.keywords.toLowerCase().includes(keyword.toLowerCase());
          paper.style.display = hasKeyword ? 'grid' : 'none';
        }
      });
    });
  </script>
</body>
</html>
