<!DOCTYPE html>

<html lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width, initial-scale=1.0" name="viewport"/>
<title>paper-trackr newsletter</title>
<link href="../docs/photo_51.ico" rel="shortcut icon"/>
<link href="../style.css" rel="stylesheet"/>
<link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&amp;display=swap" rel="stylesheet"/>
</head>
<body>
<aside class="sidebar">
<div class="sidebar-inner">
<a class="logo" href="../index.html">felipevzps</a>
<nav class="nav">
<a href="../index.html#about">About</a>
<a href="../index.html#research">Research</a>
<a href="../index.html#hackathon">Hackathon</a>
<a href="../index.html#contact">Contact</a>
</nav>
</div>
</aside>
<main class="newsletter-container">
<section class="newsletter-intro">
<h1>Latest in Science</h1>
<p>
        Hello, welcome to your daily dose of research!<br/> 
        Here you'll find the most recent papers published yesterday, filtered for the topics I follow.
      </p>
<p class="note">
<small>
          * The curation is done automatically by my open-source project, <a href="https://github.com/felipevzps/paper-trackr" target="_blank">paper-trackr</a>.<br/>
          Want your own results? Install it via PyPI and get personalized updates!
        </small>
</p>
</section>
<!-- Filtro de keywords -->
<div class="filter-bar" id="filterBar"></div>
<section class="papers-list" id="papersList"><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 01, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Development of a universal imaging "phenome" using Shape, Appearance and Motion (SAM) features and the SAM Observation Tool (SPOT)</h2>
<p class="paper-authors">Zhou, F. Y., Norton-Steele, A., +3 authors, Lu, X.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Cells are plastic, highly heterogeneous and change over time. High-content timelapse imaging promises to reveal dynamic cell behaviors, enabling more accurate identification of cell state and cell fate prediction for biological hypothesis generation and perturbation screens. To empower live-cell imaging based screen, we report the development of 1) a Shape, Appearance, Motion (SAM) "phenome"; a universal set of 2185 image-derived features that act as an image-"transcriptome" to comprehensively quantify an object's instantaneous phenotype; 2) the SAM-Phenotype-Observation-Tool (SPOT), for image-"sequencing" analysis of phenomes. We validated the effectiveness of unbiased SAM-SPOT workflow on publicly available computer vision and 2D single cell imaging datasets. Importantly, we demonstrated that SAM-phenome outperformed features generated by deep learning AI models trained on &gt;1 million fixed single cell and &gt;5000 single cell video frames, respectively. SAM-phenome and SPOT delivers high-throughput, object-treatment-agnostic, comprehensive screening readouts of dynamics, promising to advance novel molecular target discovery and new medicine development.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.11.28.691105v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 01, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Mut-BPE: A Modified BPE Strategy Improves Variant Effect Prediction</h2>
<p class="paper-authors">Xu, Y., Long, W., +2 authors, Zhang, Y.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Byte Pair Encoding (BPE) is widely used in genome foundation models for its ability to compress long DNA sequences into fewer tokens. However, its variable-length tokens often span multiple nucleotides, limiting the model's sensitivity to single-nucleotide variations: an essential requirement for accurate Variant Effect Prediction (VEP). We introduce Mut-BPE, a training-free, plug-and-play tokenization strategy that augments BPE with explicit single-nucleotide resolution at variant sites. Mut-BPE preserves the efficiency of BPE while enhancing its ability to represent subtle genomic alterations. To evaluate its effectiveness, we applied Mut-BPE to DNABERT-2 and conducted extensive experiments across six diverse datasets spanning gene expression, pathogenicity, and trait-associated variants, under zero-shot , and fine-tuning settings. Mut-BPE consistently outperformed conventional BPE tokenization, yielding significant improvements in both AUROC and AUPRC, particularly in imbalanced datasets. These results highlight Mut-BPE as a practical enhancement for genomic foundation models in VEP tasks.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.12.01.691503v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 01, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Protein Compositional Ratio Representation (PCRR)Systematically Improves Human Disease Prediction</h2>
<p class="paper-authors">Madduri, A. V., Ellis, R. J., Patel, C. J.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Plasma proteomics captures a functional snapshot of human physiology; yet, most machine learning models treat protein abundances as independent variables, ignoring the fact that biological systems and proteomic measurements are inherently compositional. Many molecular processes depend not on absolute concentrations but on relative balances: receptor-ligand stoichiometry, enzyme-substrate ratios, and homeostatic feedbacks that govern signaling and metabolism. We propose that these relationships are best captured through pairwise protein ratios, which more faithfully reflect underlying biochemical constraints than raw expression values. We evaluate a machine learning framework that models pairwise log-ratios of proteins (log(A)-log(B)) as features, thereby encoding compositional structure directly into the learning space. Applied to the ROSMAP plasma proteomics cohort (n = 871), this approach substantially improved the classification of Alzheimer's subtypes (NCI, MCI, AD, AD+) with an average AUROC gain of +0.0995 over a strong baseline that incorporated raw proteomics and demographics. The top-ranked ratios (e.g., APOC1:ARID1A, FGF7:LBP) captured converging pathogenic pillars of Alzheimer's disease, including microglial activation, proteostasis dysregulation, and lipid-clearance imbalance, highlighting that ratio-based features recover biologically coherent axes of disease. To assess generality, we scaled the method to the UK Biobank proteomic dataset (n &gt; 53,000; 587 phenotypes). The ratio-based model outperformed raw-level models in 95.% of diseases, with statistically significant (FDR &lt; 0.05) gains in 56.7%. Together, these results suggest that proteomic data should be viewed and modeled as compositional systems, where relative protein abundances carry the accurate functional signal. This insight reframes how future proteomic studies, and potentially all omics, should represent molecular features for disease prediction and biomarker discovery.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.11.29.691286v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 01, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Full-length structure of CPR-containing self-sufficient cytochrome P450</h2>
<p class="paper-authors">Xie, Z., Liu, Z., +13 authors, Guo, R.-T.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Cytochrome P450 (P450s) are heme-thiolate monooxygenases that exploit electrons sourced from pyridine nucleotide to reduce the oxygen and heme iron to catalyze hydroxylation of inactivated C-H bond1-4. Self-sufficient P450s that contain the substrate-binding heme-domain and the electron-donating NADPH:cytochrome P450 reductase (CPR) domain in the same polypeptide chain are highly effective and have been engineered to catalyze various challenging reactions. The high efficacy attributes to the effective electron transfer rate, but how the electrons travel among the redox centers remains elusive owing to the lack of structural information of the full-length protein. Here, we report the structure of a homologue of the most extensively studied P450BM3 from Shimazuella soli (SoP450) resolved by single particle cryo-electron microscopy (cryo-EM) and X-ray crystallography. SoP450 primarily exists as a homodimer formed via the intertwined CPR-domains. The spatial alignment of the heme-domain that is linked via an extensive loop was also determined. Notably, a class of structure that lacks one heme-domain was identified from the cryo-EM analyses, indicating that the heme-domain is mobile. We suspected that the heme-domain could move to reach the CPR-domain for electron acquisition and built a model of putative catalytic state to reveal how the electrons are relayed from NADPH to heme. These results are of fundamental importance to understand the catalytic reaction of CPR-containing self-sufficient P450s, which shall provide critical information to the engineering and applications of these enzymes.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.11.29.689641v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 01, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">multiVIB: A unified probabilistic contrastive learning framework for atlas-scale integration of single-cell multi-omics data</h2>
<p class="paper-authors">Xu, Y., Fleming, S. J., +3 authors, Huo, B.-X.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Comprehensive brain cell atlases are essential for understanding neural functions and enabling translational insights. As single-cell technologies proliferate across experimental platforms, species, and modalities, these atlases must scale accordingly, calling for data integration framework that aligns heterogeneous datasets without erasing biologically meaningful variations. Existing tools typically address narrow integration settings, forcing researchers to assemble textit{ad hoc} workflows that may generate artifacts. Here, we introduce multiVIB, a unified probabilistic contrastive learning framework that handles diverse integration scenarios. We show that multiVIB achieves state-of-the-art performance while mitigating spurious alignments. Applied to atlas-scale datasets from the BRAIN Initiative, multiVIB demonstrates robust and scalable integration, including integration of diverse data modalities and reliable preservation of species-specific variations in cross-species integration. These capabilities position multiVIB as a scalable, biologically faithful foundation for constructing next-generation brain cell atlases with the growing landscape of single-cell data.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.11.29.691308v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 01, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Adapting Epigenetic Clocks for Cell-Free DNA High-Throughput Sequencing Data</h2>
<p class="paper-authors">Li, G., Huang, W., +15 authors, Zhou, G.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Cell-free DNA (cfDNA) methylation sequencing holds promise for developing epigenetic aging clocks. However, current clocks - primarily trained on array-based data - do not readily generalize to high-throughput sequencing (HTS) cfDNA profiles. Using datasets with technical replicates encompassing HTS data from both cfDNA and gDNA, alongside gDNA methylation array data, we systematically assessed factors influencing clock accuracy and reproducibility. We identified key strategies to overcome HTS-specific challenges: maintaining [≥]10x mean target depth, applying elastic net regression with strong L2 regularization, and imputing unreliable beta-values. Transfer learning further enhanced accuracy robustly across multiple independent cohorts. Our findings demonstrate that array-derived epigenetic clocks can be effectively adapted to cfDNA sequencing data. This work offers critical methodological insights and practical guidelines, advancing the feasibility of minimally invasive aging assessment using cfDNA.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.11.27.690895v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 01, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Not every gene is special: one simple rule to control the false discovery rate when analysing high-throughput sequencing data</h2>
<p class="paper-authors">Dos Santos, S. J., Murariu, A. C., +1 author, Gloor, G. B.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Differential expression and differential abundance analyses are commonplace in studies employing high-throughput sequencing approaches; however different tools often fail to return comparable results when applied to the same dataset. Most tools employ various normalisations to attempt to correct for technical variation in the count data. Previously, we demonstrated that these normalisations are often inappropriate due to incorrect assumptions regarding the overall scale (i.e. size) of the biological system in question. In this study, we conducted 100 permutation analyses of 10 RNA-seq datasets to show that scale misspecification results in poor control of the false discovery rate by several commonly-used analysis tools. Moreover, we demonstrate that this can be ameliorated by using a scale model in ALDEx2 or ALDEx3 to account for uncertainty around the size of the system scale. We show that there is an inherent trade-off between satisfactory control of false-discovery rates and sensitivity and that no tool offers both. We also quantify how increasing scale uncertainty affects the difference between groups required for a feature to be reported as differentially expressed. Finally, we provide universal guidance on choosing an appropriate amount of scale uncertainty for any type of analysis. Overall, our work highlights the strengths and pitfalls of commonly used tools for differential expression analyses and highlights the choice between sensitivity and false-discovery rate control that all researchers are making when analysing sequencing data.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.11.28.690516v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 01, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Best practices to cluster large molecular libraries</h2>
<p class="paper-authors">Lope Perez, K., Miranda Quintana, R. A.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> BitBIRCH is a novel clustering algorithm that enables the analysis of extremely large molecular libraries; however, its performance can be hindered by an excessive number of singletons or the formation of disproportionately large clusters. Here, we present a data-driven strategy to identify optimal BitBIRCH parameters that mitigate these limitations. Using the ChEMBL34 library as a case study (with additional datasets reported in the Supporting Information), we show that similarity thresholds between three and four standard deviations above the global mean provide a balanced trade-off between cluster count and medoid similarity. These values are efficiently approximate with the iSIM and iSIM-sigma frameworks. For the branching factor, values as high as computationally feasible are recommended, as increasing it to 1024 substantially reduced the number of singletons. We further introduce an iterative re-clustering procedure wherein the similarity threshold can be adjusted to merge related subclusters and singletons from the initial clustering, providing user-defined control over the extent of cluster fusion. This work provides practical guidelines to enhance the robustness and usability of BitBIRCH for large-scale molecular clustering.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.11.28.691214v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 01, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">scE2TM improves single-cell embedding interpretability and reveals cellular perturbation signatures</h2>
<p class="paper-authors">Chen, H., Lu, Y., +5 authors, Li, Y.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Single-cell RNA sequencing technologies have revolutionized our understanding of cellular heterogeneity, yet computational methods often struggle to balance performance with biological interpretability. Embedded topic models have been widely used for interpretable single-cell embedding learning. However, these models suffer from the potential problem of interpretation collapse, where topics semantically collapse towards each other, resulting in redundant topics and incomplete capture of biological variation. Furthermore, the rise of single-cell foundation models creates opportunities to harness external biological knowledge for guiding model embeddings. Here, we present scE2TM, an external knowledge-guided embedded topic model that provides a high-quality cell embedding and interpretation for scRNA-seq analysis. Through embedding clustering regularization method, each topic is constrained to be the center of a separately aggregated gene cluster, enabling it to capture unique biological information. Across 20 scRNA-seq datasets, scE2TM achieves superior clustering performance compared with seven state-of-the-art methods. A comprehensive interpretability benchmark further shows that scE2TM-learned topics exhibit higher diversity and stronger consistency with underlying biological pathways. Modeling interferon-stimulated PBMCs, scE2TM simulates topic perturbations that drive control cells toward stimulated-like transcriptional states, faithfully mirroring experimental interferon responses. In melanoma, scE2TM identifies malignant-specific topics and extrapolates them to unseen patient data, revealing gene programs associated with patient survival.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.11.27.691023v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 01, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Error Correction Algorithms for Efficient Gene ExpressionQuantification in Single Cell Transcriptomics</h2>
<p class="paper-authors">Zentgraf, J., Schmitz, J. E., +1 author, Rahmann, S.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Technological advances in single-cell RNA sequencing (scRNA-seq) allow us to sequence the transcriptomes of thousands of single cells in parallel, resulting in massive amounts of raw sequence data that must be processed efficiently to obtain a genes x cells expression matrix. In droplet-based scRNA-seq protocols, the sequenced mRNA molecules are tagged with a cell-specific barcode and a unique molecular identifier (UMI) within each cell. Both barcodes and UMIs may contain errors from production, amplification or sequencing. Correcting and resolving such errors before further processing yields more reliable data and more accurate expression measurements. We propose algorithmic advancements for barcode correction, read-to-gene mapping and UMI resolution, which we combine into a new method called arcane for efficient gene expression quantification from scRNA-seq data. We additionally provide an implementation as a workflow-friendly command-line tool, also called arcane. This work builds on the recently published Fourway method to efficiently discover DNA k-mers with a Hamming distance of 1, speeding up barcode correction and UMI resolution, and allowing for distinguishing k-mers into weakly and strongly unique ones during read-to-gene mapping. As a side result of separate interest, we show that for the mapping step, it suffices to store three genes per k-mer in order to cover almost all of the genes almost completely, thus avoiding arbitrarily large colors sets in the colored De Bruijn graph index. As a result, arcane is faster than existing methods while producing very similar results, as demonstrated in a comparison with CellRanger, Kallisto|bustools and Alevin-fry.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.11.27.690682v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, cancer biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, cancer biology</span>
<span class="date">Dec 01, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Multiplexed CRISPR/Cas9 Editing of Tumor Suppressor Genes Recapitulates Molecular and Morphological Features of High-Risk Endometrial Cancer</h2>
<p class="paper-authors">Vidal-Sabanes, M., Navaridas, R., +7 authors, Dolcet, X.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> High-risk endometrial cancers (EC), such as uterine carcinosarcomas (UCS) and serous endometrial intraepithelial carcinoma (SEIC), are characterized by frequent mutations in tumor suppressor genes (TSGs) and poor clinical outcomes. Traditional genetically engineered mouse models are limited in flexibility and scalability to study the cooperative effects of multiple TSG alterations. Here, we use a multiplexed CRISPR/Cas9-based approach to simultaneously edit the top ten TSGs commonly mutated in high-risk EC directly in the mouse endometrium via intrauterine electroporation. Using rolling circle amplification (RCA) and next-generation sequencing, we demonstrate that this method induces targeted gene editing in a mosaic manner, mimicking tumor heterogeneity. We demonstrate that this approach generates histologically and molecularly faithful models of SEIC and UCS. Importantly, some edited tissues remained histologically normal, emphasizing the complex multistep nature of endometrial tumorigenesis. These CRISPR/Cas9-generated murine models serve as robust platforms to dissect the molecular underpinnings of high-risk endometrial cancer and to accelerate preclinical evaluation of novel therapeutic strategies.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.11.28.691175v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, cancer biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, cancer biology</span>
<span class="date">Dec 01, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">ALKBH1 drives codon-biased, pro-oncogenic translation and tumor microenvironment remodeling in glioma via tRNA wobble oxidation</h2>
<p class="paper-authors">Nakayashiki, A., Nagai, A., +13 authors, Niizuma, K.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Gliomas rely on translational plasticity to sustain heterogeneity, stemness, and immune evasion. Here we identify the tRNA dioxygenase ALKBH1 as a central regulator of codon-biased translation and tumor microenvironment remodeling. LC-MS/MS profiling of patient tumors revealed enrichment of ALKBH1-mediated wobble cytidine oxidation modifications in high-grade gliomas. Genetic perturbation demonstrated that ALKBH1 overexpression slows proliferation in vitro but worsens survival in vivo by promoting glioma stem-like and neuronal states and suppressing anti-tumor immunity. Ribosome profiling showed that ALKBH1 establishes an A/T-ending codon-biased translational program, enhancing decoding of rare leucine codons (TTA/TTG) and driving synthesis of pro-stemness transcripts. Single-cell RNA-seq analyses further revealed that ALKBH1 promotes glioma heterogeneity and induces neuronal cell clusters and rewires intercellular communication ECM-related signaling while dampening immune pathways and reducing immune infiltration. These findings establish a mechanistic link between tRNA oxidation, codon bias, and glioma aggressiveness, positioning ALKBH1 as a potential therapeutic target for glioma.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.11.28.691056v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, cancer biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, cancer biology</span>
<span class="date">Dec 01, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">The subclonal footprint of pervasive early dissemination in pancreatic cancer</h2>
<p class="paper-authors">Fang, Y., Chan-Seng-Yue, M., +26 authors, Notta, F.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> 'Early is too late' describes a central problem in pancreatic cancer referring to its relentless drive to disseminate and highlights the need to understand how this disease spreads so rapidly. In this study, we profiled 1,013 samples from 277 donors, including tissue whole genome and RNA sequencing combined with plasma whole genome sequencing at ~25x. Strikingly, local primary tumours, including some reaching 7cm, were found to shed little to no circulating tumour DNA (ctDNA). Instead, metastatic burden in the liver but not extrahepatic sites, was a main physiologic determinant of ctDNA levels. Whole-genome duplication (WGD), high cell cycle activity, and non-glandular differentiation emerged as tumour-intrinsic features related to increased ctDNA shedding. By contrast, decreased shedding was related to extrinsic features including a reactive microenvironment and unexpectedly, humoral immunity. Analysis of tumour clonal architecture showed that in patients with low ctDNA levels, the signal disproportionately originated from subclones and this signal persisted even when the primary tumour was removed. Disseminated subclones were a significant source of ctDNA in early-stage patients. Longitudinal analysis of patients revealed that subclones seeded metastases and shed ctDNA in the blood years before detection. This first report of paired tissue and plasma whole genomes in pancreatic cancer is a unique resource and has broad implications for disease surveillance, treatment monitoring, and early detection in this disease.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.11.27.690823v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, cancer biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, cancer biology</span>
<span class="date">Dec 01, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Proteostasis Stress Drives Stem Cell Aging, Clonal Hematopoiesis and Leukemia</h2>
<p class="paper-authors">Zhou, F. J., Le, M., +7 authors, Signer, R. A.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Aging is the primary risk factor for clonal hematopoiesis and the development of hematologic malignancies (1-5), yet the selective pressures that shape stem cell behavior and clonal expansion during aging remain poorly defined. Here, we identify proteostasis stress as a central driver of hematopoietic stem cell (HSC) aging and clonal evolution. We show that Heat shock factor 1 (Hsf1) is activated in aging HSCs to preserve proteostasis and sustain self-renewal. However, this physiological, age-associated adaptive mechanism is co-opted by pre-leukemic Dnmt3a-mutant HSCs to resist proteostasis and inflammatory stress required to fuel clonal expansion during aging. In the context of co-occurring Dnmt3a and Nras mutations, which are frequently observed in human acute myeloid leukemia (AML) (6-13), mutant HSCs and progenitors exhibit heightened dependence on Hsf1 for expansion, malignant transformation and disease progression. Loss of Hsf1, or disruption of proteostasis, impairs expansion of mutant progenitors, delays leukemia onset, and prolongs survival. Together, these findings reveal proteostasis as a key constraint in the aging hematopoietic system that imposes a selective bottleneck. Hsf1 activation enables both physiological adaptation in aging stem cells and pathological clonal outgrowth in pre-leukemic and leukemic states, establishing proteostasis control as a pivotal mechanism linking stem cell aging to clonal hematopoiesis and malignancy.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.11.27.690982v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, cancer biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, cancer biology</span>
<span class="date">Dec 01, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Tissue architecture and immune niches govern ctDNA release in colorectal cancer</h2>
<p class="paper-authors">Kuehberger, S., Sallinger, K., +24 authors, Heitzer, E.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Circulating tumor DNA (ctDNA) is central to liquid biopsy-based cancer detection, yet its release into the bloodstream varies widely and remains poorly understood. To define the tissue-level determinants of ctDNA shedding in colorectal cancer (CRC), we integrated tumor-informed plasma sequencing with detailed histopathology, immunophenotyping, spatial transcriptomics, and in situ mutation detection in resectable stages (I-III). ctDNA detectability increased with tumor burden, and high ctDNA shedders exhibited a distinct architectural and microenvironmental phenotype characterized by expanded necrotic pseudolumina, frequent epithelial barrier disruption, and dense myeloid infiltration. Spatial profiling revealed stress-associated malignant programs and a myeloid-rich immune-luminal niche. In situ sequencing confirmed plasma-detected mutations within pseudoluminal debris, identifying these structures as focal reservoirs of shed DNA. These findings provide a mechanistic framework linking tissue architecture, immune remodelling, and spatially organized cell death to ctDNA release with implications for refining liquid biopsy applications.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.11.28.691139v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="machine learning, cancer biology">
<div class="paper-meta">
<span class="keyword">machine learning, cancer biology</span>
<span class="date">Dec 01, 2025</span>
<span class="journal">PubMed</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Advances in artificial intelligence-based radiogenomics for lung cancer precision medicine.</h2>
<p class="paper-authors">Yanqi Sun, Xiayao Guo, +2 authors, Xuemei Wang</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Artificial intelligence-based radiogenomics has emerged as a promising approach for precision medicine in lung cancer. By integrating medical imaging, genomics, and clinical data, radiogenomics enables non-invasive prediction of key oncogenic driver mutations, exploration of associations between imaging features and gene expression, and development of prognostic models in lung cancer management. Machine learning and deep learning techniques have been applied to predict the mutation status of genes such as epidermal growth factor receptor and Kirsten rat sarcoma viral oncogene homolog, which are crucial for personalized treatment strategies. Radiogenomic studies have identified significant correlations between radiomic features and gene clusters, providing insights into tumor heterogeneity and biological pathways. Moreover, radiogenomics has shown potential in predicting treatment responses, recurrence, and overall survival in lung cancer patients. However, challenges remain in standardization, comprehensive validation, model interpretability, ethnic diversity, and the construction of multi-omics databases. With the advancement of artificial intelligence and the expansion of multimodal databases, future research should focus on solving these challenges to improve the clinical value and generalizability of radiogenomic models, thus playing a greater role in personalized medicine for cancer.</p>
<a class="btn dark" href="https://pubmed.ncbi.nlm.nih.gov/41265027/" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="machine learning, cancer biology">
<div class="paper-meta">
<span class="keyword">machine learning, cancer biology</span>
<span class="date">Dec 01, 2025</span>
<span class="journal">MED</span>
</div>
<div class="paper-content">
<h2 class="paper-title">[Comprehensive bioinformatics analysis and machine learning to investigate the regulatory mechanism and clinical value of secreted phosphoprotein 1 gene in lung squamous cell carcinoma].</h2>
<p class="paper-authors">Shi L, Xu YD, +4 authors, Hong QY.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> This study aimed to comprehensively investigate the regulatory mechanism and clinical value of secreted phosphoprotein 1 (SPP1) in the three-stage progression of "smoking-chronic obstructive pulmonary disease (COPD)-carcinogenesis" in lung squamous cell carcinoma (LUSC) through integrated bioinformatics analysis and machine learning. The datasets for the three stages of LUSC were downloaded from the Gene Expression Omnibus (GEO) database, including GSE18385 (containing lung tissue samples from 31 healthy smokers and 21 healthy non-smokers), GSE38974 (containing lung tissue samples from 23 smoking COPD patients and 9 healthy smokers), and GSE12472 (containing lung tissue samples from 18 LUSC patients with COPD and 17 smoking COPD patients). The Cancer Genome Atlas (TCGA)-LUSC dataset (comprising 504 samples, including lung tissue samples from LUSC patients and their matched normal lung tissue samples) was downloaded from TCGA database for further analysis. Samples and follow-up information from 208 non-small cell lung cancer patients who underwent radical resection and mediastinal lymph node dissection at Zhongshan Hospital, Fudan University in 2005 were selected for prognostic analysis and validation. Weighted Gene Co-expression Network Analysis (WGCNA) was applied to screen stage-specific module genes. Differentially expressed genes (DEGs) were identified through differential expression analysis. The CIBERSORT algorithm and Gene Set Enrichment Analysis (GSEA) were used to characterize the immune microenvironment. Eight machine learning algorithms and protein-protein interaction (PPI) network analysis were combined to screen for core regulatory targets. Results from WGCNA and differential analysis of the GEO datasets indicated that SPP1 is consistently highly expressed across the three stages of LUSC. Analysis of the TCGA-LUSC dataset further verified that the relative expression level of SPP1 in lung tissues of LUSC patients was significantly higher than in normal lung tissues (9.13±2.01 vs 4.68±1.64, PP=0.045). Male LUSC patients with high smoking exposure exhibited higher SPP1 expression levels than those with low smoking exposure (9.56±2.23 vs 8.60±2.04, P=0.032). Survival prognosis analysis revealed that among male patients, the difference in median overall survival (OS) between the high SPP1 expression group and the low expression group was statistically significant [2.90 (95%CI: 2.11-4.64) years vs 4.69 (95%CI: 2.95-7.34) years, P=0.032). Data validation from Zhongshan Hospital, Fudan University, also showed that the 5-year survival rate of lung cancer patients with high SPP1 expression was lower than that of patients with low SPP1 expression (49.3% vs 62.6%, P=0.042). Results from the CIBERSORT algorithm indicated that high SPP1 expression drives increased infiltration of M2 macrophages (P&lt;0.001). Machine learning combined with PPI network analysis identified NTN1 and CX3CL1 as key regulatory targets of SPP1, which may be associated with the occurrence and development of lung cancer. SPP1 may promote LUSC progression by mediating M2 macrophage polarization through suppressing NTN1 or activating CX3CL1, suggesting its potential as a prognostic biomarker and therapeutic target for high-risk male smokers.</p>
<a class="btn dark" href="https://pubmed.ncbi.nlm.nih.gov/41320661/" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="machine learning, cancer biology">
<div class="paper-meta">
<span class="keyword">machine learning, cancer biology</span>
<span class="date">Dec 01, 2025</span>
<span class="journal">MED</span>
</div>
<div class="paper-content">
<h2 class="paper-title">SMAGS-LASSO: A Novel Feature Selection Method for Sensitivity Maximization in Early Cancer Detection.</h2>
<p class="paper-authors">Khoshfekr Rudsari H, Khorami-Sarvestani S, +4 authors, Irajizad E.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Sensitivity and specificity are foundational metrics for cancer detection tools. However, most machine learning algorithms prioritize overall accuracy during optimization, which fails to align with clinical priorities of early detection. We aim to develop a feature selection machine learning algorithm while maximizing sensitivity at a given specificity.We developed SMAGS-LASSO, a machine learning algorithm that combines our developed Sensitivity Maximization at a Given Specificity (SMAGS) framework with L1 regularization for feature selection. This approach simultaneously optimizes sensitivity at user-defined specificity thresholds while performing feature selection. SMAGS-LASSO utilizes a custom loss function with L1 regularization and multiple parallel optimization techniques. We used train-test splits and cross-validation, comparing against LASSO and random forest using sensitivity and AUC metrics. We evaluated our method on synthetic datasets and real-world protein colorectal cancer biomarker data.In synthetic datasets designed to contain strong signals for both sensitivity and specificity, SMAGS-LASSO significantly outperformed standard LASSO, achieving sensitivity of 1.00 (95% confidence interval, 0.98-1.00) compared with 0.19 (95% confidence interval, 0.13-0.23) for LASSO at 99.9% specificity. In colorectal cancer data, SMAGS-LASSO demonstrated 21.8% improvement over LASSO (P value = 2.24E-04) and 38.5% over random forest (P value = 4.62E-08) at 98.5% specificity while selecting the same number of biomarkers.SMAGS-LASSO enables the development of minimal biomarker panels that maintain high sensitivity at predefined specificity thresholds, offering superior performance for early cancer detection.This method provides a promising approach for early cancer detection and other medical diagnostics requiring sensitivity-specificity optimization.</p>
<a class="btn dark" href="https://pubmed.ncbi.nlm.nih.gov/40996315/" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div></section>
</main>
<script>
    // === EXPAND / COLLAPSE PAPERS ===
    document.querySelectorAll('.paper').forEach(paper => {
      paper.addEventListener('click', e => {
        if (e.target.tagName.toLowerCase() === 'a') return;
        paper.classList.toggle('expanded');
      });
    });

    // === FILTER SYSTEM ===
    const papers = document.querySelectorAll('.paper');
    const filterBar = document.getElementById('filterBar');

    // Extrai keywords
    const allKeywords = new Set();
    papers.forEach(paper => {
      paper.dataset.keywords.split(',').forEach(k => allKeywords.add(k.trim()));
    });

    // Cria botões dinamicamente
    const filters = ['all', ...Array.from(allKeywords)];
    filters.forEach(keyword => {
      const btn = document.createElement('button');
      btn.className = 'filter-btn';
      btn.textContent = keyword;
      if (keyword === 'all') btn.classList.add('active');
      filterBar.appendChild(btn);
    });

    // Filtro de papers
    filterBar.addEventListener('click', e => {
      if (!e.target.classList.contains('filter-btn')) return;

      document.querySelectorAll('.filter-btn').forEach(b => b.classList.remove('active'));
      e.target.classList.add('active');

      const keyword = e.target.textContent;
      papers.forEach(paper => {
        if (keyword === 'all') {
          paper.style.display = 'grid';
        } else {
          const hasKeyword = paper.dataset.keywords.toLowerCase().includes(keyword.toLowerCase());
          paper.style.display = hasKeyword ? 'grid' : 'none';
        }
      });
    });
  </script>
</body>
</html>
