<!DOCTYPE html>

<html lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width, initial-scale=1.0" name="viewport"/>
<title>paper-trackr newsletter</title>
<link href="../docs/photo_51.ico" rel="shortcut icon"/>
<link href="../style.css" rel="stylesheet"/>
<link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&amp;display=swap" rel="stylesheet"/>
</head>
<body>
<aside class="sidebar">
<div class="sidebar-inner">
<a class="logo" href="../index.html">felipevzps</a>
<nav class="nav">
<a href="../index.html#about">About</a>
<a href="../index.html#research">Research</a>
<a href="../index.html#hackathon">Hackathon</a>
<a href="../index.html#contact">Contact</a>
</nav>
</div>
</aside>
<main class="newsletter-container">
<section class="newsletter-intro">
<h1>Latest in Science</h1>
<p>
        Hello, welcome to your daily dose of research!<br/> 
        Here you'll find the most recent papers published yesterday, filtered for the topics I follow.
      </p>
<p class="note">
<small>
          * The curation is done automatically by my open-source project, <a href="https://github.com/felipevzps/paper-trackr" target="_blank">paper-trackr</a>.<br/>
          Want your own results? Install it via PyPI and get personalized updates!
        </small>
</p>
</section>
<!-- Filtro de keywords -->
<div class="filter-bar" id="filterBar"></div>
<section class="papers-list" id="papersList"><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Nov 02, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Singletrack: An Algorithm for Improving Memory Consumption and Performance of Gap-Affine Sequence Alignment</h2>
<p class="paper-authors">Lopez-Villellas, L., Iniguez, C., +5 authors, Marco-Sola, S.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Motivation: Advances in DNA sequencing have outpaced advances in computation, making sequence alignment a major bottleneck in genome data analyses. Classical dynamic programming (DP) algorithms are particularly memory-intensive, especially when computing gap-affine and dual gap-affine alignments. Existing strategies to reduce memory consumption often sacrifice either speed or alignment accuracy. Results: We present Singletrack, an efficient algorithm for backtrace gap-affine and dual gap-affine alignments that requires only storing a single DP matrix. Compared to classical DP algorithms, Singletrack removes the need to store additional matrices (i.e., 2 for gap-affine and 4 for dual gap-affine), significantly reducing memory consumption and, in turn, reducing pressure on the memory hierarchy and improving overall performance. Most importantly, Singletrack is a general backtrace method compatible with state-of-the-art DP-based algorithms and heuristics, such as the Suzuki-Kasahara (SK) and the Wavefront Alignment (WFA) algorithms. Our results demonstrate that Singletrack accelerates the SK implementation of KSW2, used within Minimap2, by up to 1.4. Similarly, Singletrack enhances the performance of the WFA implementation in WFA2-lib by 1.2-2.1x while reducing memory usage by 3x for gap-affine and 5x for dual gap-affine. Compared to the efficient linear-memory BiWFA algorithm, the Singletrack-accelerated version of WFA trades a practical increase in memory usage for up to 5.2x higher performance. Availability: All the implementations of the Singletrack algorithm presented in this work are available at https://github.com/LorienLV/singletrack.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.10.31.685625v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Nov 02, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">An Application for Automated Drosophila Locomotor Assay with Integrated Device Design and Computer Vision Tracking</h2>
<p class="paper-authors">Melkani, D., Harnwal, N., +2 authors, Melkani, G. C.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Drosophila has long served as a powerful model for investigating locomotor behavior, and geotaxis assays have generated valuable insights into genetics, aging, and neurobiology. Nonetheless, their use can be constrained by subjective scoring, modest throughput, and challenges in reproducibility. To complement and extend these classical approaches, we developed and validated an integrated hardware-software platform that enables automated, high-resolution locomotor analysis across 12 vials in parallel. The system integrates 3D-printed mechanical components, Raspberry Pi-based video acquisition, and programmable environmental controls to ensure standardized conditions. A deep learning pipeline segments vials with near-perfect accuracy (IoU &gt; 0.95), while computer vision algorithms quantify climbing trajectories, velocity, and positional zone occupancy at 60 frames per second. The end-to-end workflow converts raw video into time-resolved metrics, supports sex-specific aggregation, and incorporates advanced statistical analyses, including Linear Mixed Effects regression, harmonic mean p-values, and Mann-Whitney U tests. Relative to manual scoring, this automated pipeline yields 2.8-fold faster processing and nearly 800-fold higher data density. Application of the platform uncovered reproducible phenotypes of multiple genotypes. For example, a circadian mutant known as ClockOut, males displayed progressive climbing deficits with age, whereas females-maintained age-resilient trajectories. Moreover, male ClockOut exhibited a reduced performance compared to age-matched control (w1118), however, female ClockOut showed subtle reduction in performance. Additionally, glial-specific knockdown of PolG, encoding the DNA polymerase gamma catalytic subunit, revealed striking sex-dimorphic aging patterns: females outperformed controls at older age, while males exhibited marked decline. To promote broad adoption, a user-friendly Python interface (Tkinter GUI) enables accessibility independent of computational expertise. Collectively, this standardized, high-throughput framework advances the resolution of genotype-, age-, and sex-dependent locomotor dynamics, offering new opportunities in aging, circadian biology, and neurodegeneration research.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.10.31.685789v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Nov 02, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Clair-Mosaic: A deep-learning method for long-read mosaic small variant calling</h2>
<p class="paper-authors">Chen, L., Zheng, Z., +6 authors, Luo, R.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Mosaic variants, defined as postzygotic mutations occurring during an organism's development from zygote to adult, play critical roles in developmental biology, aging, and diseases such as cancer and neurological disorders. However, their accurate detection remains challenging due to low abundance in the genome and low variant allelic fractions (VAF). While current mosaic variant callers are primarily designed for short-read sequencing, no method is available for long-read sequencing, which can generate tens of kilobase-long reads to cover complex genomic regions inaccessible to short reads. To fill the gap, we present Clair-Mosaic, a deep-learning-based method for detecting mosaic small variants from long-read data. Clair-Mosaic was trained on hundreds of millions synthetic variants encompassing diverse read coverages and allelic fractions, enabling it to detect low-VAF mosaic variants with high sensitivity in paired-sample and single-sample modes. In addition to neural network prediction, Clair-Mosaic distinguishes genuine mosaic variants from sequencing artifacts by leveraging their inherent haplotype relationship in phased long reads. Furthermore, a Bayesian mosaic-germline discriminator is introduced to distinguish mosaic variants from germline variants. It also employs multiple post-calling filters, including a mosaic variant database and multiple germline population resources, to tag common germline and mosaic variants. Comprehensive benchmarking on synthetic datasets and real samples demonstrated Clair-Mosaic's outstanding performance in ONT and PacBio. Clair-Mosaic is also applicable to short-read data and outperforms methods like MosaicHunter, MosaicForecast, DeepMosaic, and DeepSomatic. Clair-Mosaic is open-source and available at https://github.com/HKU-BAL/Clair-Mosaic.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.10.31.685831v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Nov 02, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">InCytokine, an open-source software, reveals a TREM2 variant specific cytokine signature</h2>
<p class="paper-authors">Jha, D., Ancona, M., +8 authors, Bitton, D. A.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Cytokine and chemokine profiling is central to understanding inflammatory processes and the mechanisms driving diverse diseases. We introduce InCytokine, an open source tool for semiquantitative analysis of cytokine and chemokine data generated by protein array technologies. InCytokine features robust and modular image processing workflows, including automated spot detection, template alignment, normalization, quality control measures and quantitative intensity summarization to deliver consistent and reliable readouts from profiling assays. We evaluated InCytokine by profiling wild-type microglia, TREM2 knockout, and Alzheimer's-associated TREM2 R47H variant cells in response of lipopolysaccharide and sulfatide exposure. Differential expression analysis revealed unique sulfatide-specific and genotype-specific cytokine signatures in TREM2 variants. We also report an intriguing modulation of DPP4 and a divergent expression pattern of ENA 78 in TREM2 variants in response to lipopolysaccharide and sulfatide treatment. Such distinct expression signatures raise the possibility that TREM2 variants may play a role in modulating inflammatory signaling relevant to cardio-metabolic and Alzheimer's disease. These signatures were corroborated using transcriptional profiling of the same microglia cells, revealing also a good concordance between protein array and RNA sequencing technologies. Taken together, InCytokine is an interactive, user friendly web application for rapid, reproducible, and scalable analysis of protein array data, proven to generate meaningful insights for drug and biomarker discovery campaigns in pharmaceutical settings.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.10.31.685741v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Nov 02, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">iDeepLC: chemical structure information yields improved retention time prediction of peptides with unseen modifications</h2>
<p class="paper-authors">Nameni, A., Declercq, A., +4 authors, Bouwmeester, R.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Deep learning has notably advanced the field of liquid chromatography-mass spectrometry-based proteomics. Accurate prediction of peptide retention times significantly enhances our ability to match LC-MS data with the correct peptides and proteins , especially for DIA data. While numerous models predict peptide LC retention times with high accuracy, few can accurately predict the retention times of chemically modified peptides, particularly those with modifications not encountered during model training. In our previously developed DeepLC model, accurate predictions could be made for unseen modifications by leveraging the chemical composition of (modified) residues. Here, however, we present a further enhancement of this model based on chemical structural information. The resulting model, called iDeepLC, shows overall more accurate predictions, and better generalization performance for predicting the retention time of unseen modifications than DeepLC. iDeepLC is freely available as open-source software under the Apache2 license and can be found at https://github.com/CompOmics/iDeepLC.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.10.31.685771v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Nov 02, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Establishing benchmarks for quantitative mapping of m6A by Nanopore Direct RNA Sequencing</h2>
<p class="paper-authors">Ohnezeit, D., Loliashvili, E., +7 authors, Wilson, A. C.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Nanopore direct RNA sequencing (DRS) coupled with Dorado modification-aware basecalling enables mapping of epitranscriptomic modifications including N6-methyladenosine (m6A) at the level of individual RNAs. However, a lack of systematic benchmarking continues to raise questions regarding the sensitivity, specificity, and reproducibility of this method. To address this and to establish a best-practice workflow, we evaluated multiple Dorado versions using in vitro transcribed RNA and an m6A methyltransferase inhibitor as specificity controls. We established that stringent filtering is necessary to reduce false-positive calls and found strong concordance at high-stoichiometry sites when compared to an orthogonal m6A mapping method (GLORI). Further, by applying DRS to primary human fibroblasts and HD10.6 neurons, we uncovered cell type-specific differences in m6A stoichiometry, indicating a finely tuned epitranscriptomic regulation. Our study thus presents the first systematic comparison of Dorado and GLORI from the same input RNA and expands characterization of the m6A epitranscriptome to fibroblasts and neurons.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.11.02.686099v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Nov 02, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Large mRNA language foundation modeling with NUWA for unified sequence perception and generation</h2>
<p class="paper-authors">Zhong, Y., Yan, W., +2 authors, Bian, B.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> The mRNA serves as a crucial bridge between DNA and proteins. Compared to DNA, mRNA sequences are much more concise and information-dense, which makes mRNA an ideal language through which to explore various biological principles. In this study, we present NUWA, a large mRNA language foundation model leveraging a BERT-like architecture, trained with curriculum masked language modeling and supervised contrastive loss for unified mRNA sequence perception, understanding, and generation. For pretraining, we collected and utilized large-scale mRNA coding sequences comprising approximately 80 million sequences from 19,676 bacterial species, 83 million from 4,688 eukaryotic species, and 2.1 million from 702 archaeal species, and pre-trained three domain-specific models respectively. This enables NUWA to learn the coding sequence patterns across the tree of life. The fine-tuned NUWA model demonstrates superior performance across a range of downstream tasks. It excels not only in RNA-related perception tasks but also shows strong capability in cross-modal protein-related tasks. On the generation front, NUWA can produce natural-like mRNA sequences and, when provided with a protein sequence, optimize mRNA designs for enhanced codon adaptation index in host organisms. A key advantage of our approach is its adaptability: NUWA can be effectively fine-tuned on smaller, task-specific datasets to generate functional mRNAs with desired properties, without relying on pre-defined biological constraints. To our knowledge, this represents the first mRNA language model for unified sequence perception and generation, offering a versatile platform for programmable mRNA design.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.11.01.686058v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, cancer biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, cancer biology</span>
<span class="date">Nov 02, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Non-cell-autonomous mechanisms of tumor initiation and relapse by chromosomal instability</h2>
<p class="paper-authors">Lafirenze, S. J. A., Kops, G. J., van Gerwen, B., +8 authors, Jelluma, N.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Chromosomal instability (CIN) is a hallmark of cancer, and a primary cause of genetic heterogeneity in tumors. Depending on the degree of CIN and the affected tissue, CIN can promote or suppress tumor formation, and high CIN induction has been proposed as a therapeutic strategy. How CIN achieves these effects is unclear. Here we use a conditional mouse model of graded CIN in combination with longitudinal monitoring of DMBA/TPA-initiated skin tumors to show that low CIN increases the frequency of skin tumor initiation, while higher CIN accelerates onset and growth rates. Strikingly, gene recombination analysis of the tumors reveals that upon high CIN induction the fast-growing tumors originate from rare low CIN cells, suggesting a strong non-cellautonomous effect of high CIN. Gene expression analysis and immunohistochemistry show that high CIN causes epidermal hyperplasia, immune evasion and a regenerative response that stimulates low CIN tumor growth beyond what is achieved by induction of low CIN alone. Such cell-extrinsic effects may be a common mechanism of tumor formation by CIN, as we observe it also in CIN-induced tumors of the intestine, breast and mesentery. When high CIN is induced in established skin tumors, mimicking CINbased therapy, tumors regress but relapse quickly. Relapsed tumors, too, arose from rare low CIN cells. Our findings have implications for our understanding of the contributions of CIN to cancer initiation and progression and give caution to the rationale for CIN therapies.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.10.31.685838v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div></section>
</main>
<script>
    // === EXPAND / COLLAPSE PAPERS ===
    document.querySelectorAll('.paper').forEach(paper => {
      paper.addEventListener('click', e => {
        if (e.target.tagName.toLowerCase() === 'a') return;
        paper.classList.toggle('expanded');
      });
    });

    // === FILTER SYSTEM ===
    const papers = document.querySelectorAll('.paper');
    const filterBar = document.getElementById('filterBar');

    // Extrai keywords
    const allKeywords = new Set();
    papers.forEach(paper => {
      paper.dataset.keywords.split(',').forEach(k => allKeywords.add(k.trim()));
    });

    // Cria botÃµes dinamicamente
    const filters = ['all', ...Array.from(allKeywords)];
    filters.forEach(keyword => {
      const btn = document.createElement('button');
      btn.className = 'filter-btn';
      btn.textContent = keyword;
      if (keyword === 'all') btn.classList.add('active');
      filterBar.appendChild(btn);
    });

    // Filtro de papers
    filterBar.addEventListener('click', e => {
      if (!e.target.classList.contains('filter-btn')) return;

      document.querySelectorAll('.filter-btn').forEach(b => b.classList.remove('active'));
      e.target.classList.add('active');

      const keyword = e.target.textContent;
      papers.forEach(paper => {
        if (keyword === 'all') {
          paper.style.display = 'grid';
        } else {
          const hasKeyword = paper.dataset.keywords.toLowerCase().includes(keyword.toLowerCase());
          paper.style.display = hasKeyword ? 'grid' : 'none';
        }
      });
    });
  </script>
</body>
</html>
