<!DOCTYPE html>

<html lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width, initial-scale=1.0" name="viewport"/>
<title>paper-trackr newsletter</title>
<link href="../docs/photo_51.ico" rel="shortcut icon"/>
<link href="../style.css" rel="stylesheet"/>
<link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&amp;display=swap" rel="stylesheet"/>
</head>
<body>
<aside class="sidebar">
<div class="sidebar-inner">
<a class="logo" href="../index.html">felipevzps</a>
<nav class="nav">
<a href="../index.html#about">About</a>
<a href="../index.html#research">Research</a>
<a href="../index.html#software">Software</a>
<a href="../index.html#hackathon">Hackathon</a>
<a href="../index.html#contact">Contact</a>
</nav>
</div>
</aside>
<main class="newsletter-container">
<section class="newsletter-intro">
<h1>Latest in Science</h1>
<p>
        Hello, welcome to your daily dose of research!<br/> 
        Here you'll find the most recent papers published yesterday, filtered for the topics I follow.
      </p>
<p class="note">
<small>
          * The curation is done automatically by my open-source project, <a href="https://github.com/felipevzps/paper-trackr" target="_blank">paper-trackr</a>.<br/>
          Want your own results? Install it via PyPI and get personalized updates!
        </small>
</p>
</section>
<!-- Filtro de keywords -->
<div class="filter-bar" id="filterBar"></div>
<section class="papers-list" id="papersList"><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Oct 28, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">PlasticEnz: An integrated database and screening tool combining homology and machine learning to identify plastic-degrading enzymes in meta-omics datasets</h2>
<p class="paper-authors">Krzynowek, A., Faust, K.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> PlasticEnz is a new open-source tool for detecting plastic-degrading enzymes (plastizymes) in metagenomic data by combining sequence homology-based search with machine learning. It integrates custom Hidden Markov Models, DIAMOND alignments, and polymer-specific classifiers trained on ProtBERT embeddings to identify candidate depolymerases from contigs, genomes, or protein sequences. PlasticEnz supports 11 plastic polymers with ML classifiers for PET and PHB, achieving F1 &gt; 0.7 on independent test sets. Applied to plastic-exposed microcosms and field metagenomes, the tool recovered known PETases and PHBases, distinguished plastic-contaminated from pristine environments, and clustered predictions with validated reference enzymes. PlasticEnz is fast, scalable, and user-friendly, providing a robust framework for exploring microbial plastic degradation potential in complex communities.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.10.28.685028v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Oct 28, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">plantiSMASH 2.0: improvements to detection, annotation, and prioritization of plant biosynthetic gene clusters</h2>
<p class="paper-authors">Del Pup, E., Owen, C., +6 authors, Medema, M. H.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Plants produce bioactive compounds as part of their specialized metabolism, with applications in medicine, agriculture, and nutrition. The biosynthesis of a growing number of these specialized metabolites has been found to be encoded in biosynthetic gene clusters (BGCs), creating increasing demand for genome mining tools to automate their detection. plantiSMASH enables the identification of putative plant BGCs through a rule-based approach, available via both command-line and web interfaces. Here, we present plantiSMASH 2.0 (https://plantismash.bioinformatics.nl/), a major update that expands and improves the original framework with revised and additional BGC detection rules (now supporting 12 BGC types), substrate prediction for selected enzyme families, and regulatory analysis through transcription factor binding site detection. The updated plantiSMASH 2.0 database includes 30,423 putative BGCs across 430 genomes. Together, these improvements make plantiSMASH 2.0 a powerful and comprehensive platform for the detection and characterization of plant biosynthetic pathways, supporting and accelerating research in plant specialized metabolism and plant natural product discovery.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.10.28.683968v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Oct 28, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Orthogonal Inducible Transcriptional Control Systems forInvestigation of Gene-Gene Interactions</h2>
<p class="paper-authors">Azizoglu, A.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Genes do not act in isolation but through complex networks of interactions. While many double and triple gene interactions have been uncovered in Saccharomyces cerevisiae, most have been identified through overexpression or loss-of-function studies, leaving dosage-dependent interactions largely unexplored. Multi-gene control systems suffer from various drawbacks, including limited range and high cell-to-cell variation of expression across the population. To address this gap, we extend a recently developed inducible expression system that enables precise, tunable control of gene expression to additional repressor-inducer pairs: LacI/IPTG and LexA-hER/{beta}-estradiol. We demonstrate that both new systems can control endogenous genes on their own by placing various low, medium and high expressed endogenous genes under their control. Using all three WTC systems, we show simultaneous and orthogonal double and triple gene control that confirms known gene-gene interactions within the anaphase signaling network and reveals new dosage-dependent phenotypes. Given their precision, orthogonality and compatibility with all commonly used growth media, this collection of WTC systems represents a new tool suitable for precise investigation of dosage-dependent gene-gene interactions in yeast.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.10.28.685021v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Oct 28, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">CREation of an expanded plant memory gene circuit toolkit</h2>
<p class="paper-authors">Gong, P., Khan, A., +6 authors, Lister, R.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Genetic circuits rely on modular, well-characterized genetic parts to achieve predictable cellular behaviour. Despite their widespread adoption, biological parts are complex and when used in a new molecular context can result in unexpected outcomes. Site-specific recombinases have been widely used due to their unique ability to induce precise, irreversible changes to a DNA sequence, making them ideal tools for memory logic operations. However, unexpected context-specific failures of even the most widely used recombinase, Cre, has limited the expansion and complexity of synthetic gene circuits in diverse species. Here, through a systematic analysis of Cre failure in plant gene circuits, we uncovered multiple unexpected post-recombination interactions within the transcriptional unit between the promoter region, recombinase, and their cognate recognition sites. These significantly inhibit transcriptional activity, preventing circuit functionality. Specifically, post-recombination Cre recombinase exhibits an inhibitory property by binding to lox sites, with the remaining lox site repressing transcription. By thoroughly characterizing these dynamics, we restored Cre functionality and expanded the plant-based recombinase toolkit for Cre-related recombinases, developing combinations of recombinases and cognate sites with different activation levels for constructing logic gates with stable memory functions. We further developed multiple functional split-recombinase systems for simultaneous logic using a range of dimerization domains and constructed complex logic gates, including 3-input and 6-input AND gates. Together, this exploration of component context dependency in genetic circuits, even with commonly used parts like the Cre/lox system, has implications extending beyond plant systems, and significantly expands plant memory circuit capabilities.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.10.28.684748v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Oct 28, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">CellDiffusion: a generative model to annotate single-cell and spatial RNA-seq using bulk references</h2>
<p class="paper-authors">Zhang, X., Mao, J., Le Cao, K.-A.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Annotating single-cell and spatial RNA-seq data can be greatly enhanced by leveraging bulk RNA-seq, which remains a cost-effective and well-established benchmark for characterising transcriptional activity in immune cell populations. However, a major technical hurdle lies in the contrasting properties of these data types: single-cell and spatial data are inherently sparse due to its cell-level sampling scheme, leading to much lower sequencing depth compared to bulk RNA-seq. We developed CellDiffusion, a generative machine learning (ML) tool that bridges this gap. CellDiffusion generates realistic virtual cells to augment the sparse single-cell and spatial data, improving signals and the representation of rare cell types. The augmented data are more comparable to bulk references, increasing the accuracy of cell type annotation using bulk references and automated ML classifiers. We benchmarked CellDiffusion on single-cell and spatial datasets from human peripheral blood samples, white adipose tissues, and breast tumours. Our method significantly outperforms state-of-the-art methods such as SingleR, Seurat, and scVI. In addition, CellDiffusion provides critical biological insights, including the identification of novel cell subtypes and their function during cell state transition; the discovery of new marker genes for tissue-resident immune cells, revealing their functional shifts in myeloid populations; and the accurate characterisation of cell subtypes in spatial transcriptomics to decipher tumour microenvironment.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.10.27.684671v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Oct 28, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Predicting single-stranded DNA oligonucleotides 3D structures: an open issue</h2>
<p class="paper-authors">Bengaouer, S., Binet, T., +3 authors, MAFFUCCI, I.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Single-stranded Nucleic Acids (ssNAs) play major biological functions and represent an interesting biotechnological tool. Their function depends strictly on the specific foldings they can adopt. Therefore, information about ssNAs' tridimensional structures is fundamental to investigate their functions. In this context, in silico 3D structure prediction can facilitate ssNAs design. Many algorithms have been developed with this aim, mainly focused on RNA. However, the growing interest in single-stranded DNA (ssDNA), due to their greater stability as compared to RNA, has highlighted the need to adapt these methods for ssDNA. This study assessed three RNA 3D structure prediction methods (RNAComposer, SimRNA, and Vfold3D), based on their performance in the Critical Assessment of protein Structure Prediction 15 and/or 16, to evaluate their applicability to ssDNA. At this scope, a dataset of 93 experimentally determined ssDNA structures, including challenging motifs such as G-quadruplexes, was built. Various metrics, such as RMSD, GDT_TS, and INF were employed to benchmark the accuracy of the predictions. The three tools showed similar and moderate performances. In addition, they show strong difficulties in modeling G-quadruplexes, and structures containing motifs strongly increasing the intrinsic flexibility of ssDNA. Despite the recent efforts in the prediction of the 3D folding of ssNAs, it is clear that a significant improvement of the methods is needed. This should involve taking into account the conformational variability of this kind of molecules and paying attention to their specific 3D motifs.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.10.28.684999v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Oct 28, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">BioDH: A reversible data hiding framework for secure hiding sensitive data in biological data</h2>
<p class="paper-authors">Zhang, Y., Lin, Y., +1 author, Fu, T.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> With the rapid advancement of single-cell RNA sequencing (scRNA-seq) technologies, the open sharing of massive high-dimensional biological data has become a key driver in advancing precision medicine and fundamental research. However, while promoting data accessibility, ensuring effective protection of sensitive data has emerged as a critical challenge in bioinformatics. To address the issue, we propose BioDH (Biological Data Hiding) framework, a reversible information hiding framework tailored for biological data, aiming to simultaneously ensure data security and preserve scientific utility. The framework employs optimal lossless compression and strictly controls numerical perturbations to maintain precision. Upon extraction, both the secret information and the original data are perfectly recovered, achieving true reversibility. Validation on real-world datasets shows exceptional fidelity: when embedding capacity is 1.25 bit per byte, maximum and mean absolute errors are 8.54E-04 and 8.90E-11, respectively; PCA reveals a correlation of 1.0; UMAP testing exhibits no structural distortion; Mantel tests and clustering analyses (ARI = 1.0) confirm preservation of high-dimensional topology and cell subpopulations. All metrics surpass biological compatibility thresholds, indicating no detectable impact on downstream analyses.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.10.28.685003v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Oct 28, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">DNACSE: Enhancing Genomic LLMs with Contrastive Learning for DNA Barcode Identification</h2>
<p class="paper-authors">Wang, J., Wang, B., +3 authors, Zheng, P.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> DNA barcoding is a powerful tool for exploring biodiversity, and DNA language models have significantly facilitated its construction and identification. However, since DNA barcodes come from a specific region of mitochondrial DNA and there are structural differences between DNA barcodes and reference genomes used to train existing DNA language models, it is difficult to directly apply the existing DNA language models to the DNA barcoding task. To address this, this paper introduce DNACSE (DNA Contrastive Learning for Sequence Embeddings), an unsupervised noise contrastive learning framework designed to fine-tune the DNA language foundation model while enhancing the distribution of the embedding space. The results demonstrate that DNACSE outperforms direct usage of DNA language models in DNA barcoding-related tasks. Specifically, in fine-tuning and linear probe tasks, it achieves accuracy rates of 99.17% and 98.31%, respectively, surpassing the current state-of-the-art BarcodeBERT by 6.44% and 6.44%. In zero-shot clustering tasks, it raises the Adjusted Mutual Information (AMI) score to 92.25%, an improvement of 8.36%. In addition, zero-shot benchmarking and genomic benchmarking tests are evaluated, indicating that DNACSE enhances the performance of DNA language models in generalized genomic tasks. In summary, DNACSE has demonstrated excellent performance in DNA barcode species classification by making full use of multi-species information and DNA barcode information, providing a feasible way to further explore and protect biodiversity.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.10.27.684901v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Oct 28, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Natural variation in regulatory code revealed through Bayesian analysis of plant pan-genomes and pan-transcriptomes</h2>
<p class="paper-authors">Wei, W., Wu, X., +4 authors, Krasileva, K.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Understanding the genetic code of cis-regulatory elements (CREs) is essential for engineering gene expression and modulating agronomic traits in crops. In plants, CREs underlying rapid evolution of gene expression often overlap with structural variation in promoters, making them undetectable using single-reference genomes. Here, we develop K-PROB (K-mer-based in silico PROmoter Bashing), a computational tool that learns from intraspecies promoter sequence and gene expression variation in pan-genomes and pan-transcriptomes to identify CREs controlling gene expression. K-PROB deploys a k-mer-based Bayesian variable selection framework to prioritize causal variable identification. We demonstrate the effectiveness of our approach in maize and soybean, two staple crops species. Applying K-PROB to genes with the most highly variable promoter sequences and the most diverse patterns of expression, such as nucleotide-binding leucine-rich repeat receptors, we identified k-mers enriched for bona fide transcription factor binding sequences, and overlapping with open chromatin regions and DAP-seq binding sites. Notably, multiple significant k-mers are located within presence/absence structural variants, highlighting structural variation in promoters as key drivers of transcriptional diversity of highly variable genes. We further validated the regulatory effects of identified k-mers on gene expression using luciferase reporter assays. Our results showcase a high-throughput and pangenomic approach for probing natural intraspecies cis-regulatory diversity, discovering new causative cis-elements, and facilitating future expression engineering across plant species.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.10.27.684862v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Oct 28, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">CFMF: A Clustering-Free Cell Marker Finder for Single-Cell Transcriptomic Data</h2>
<p class="paper-authors">Yin, S., Liu, K.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> </p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.10.26.683836v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, cancer biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, cancer biology</span>
<span class="date">Oct 28, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Epigenetic Reprogramming of Host Chromatin by the Transforming Parasite Theileria annulata</h2>
<p class="paper-authors">Sharma, P., Thakur, S., +5 authors, Bhandari, V.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Theileria annulata a transforming apicomplexan parasite, extensively reprograms the chromatin architecture of bovine leukocytes to facilitate infection and cellular transformation. To elucidate the underlying epigenetic mechanisms, we characterized the chromatin landscape of infected lymphocytes using integrated proteomic, imaging, and functional assays. Our findings reveal that T. annulata lacks the DNA damage marker gH2A.X and its associated SQ/TQ motif, indicating an evolutionary divergence from canonical DNA repair signalling pathways. High-resolution profiling of histone post-translational modifications (PTMs) demonstrated distinct nuclear compartmentalization, with host (H3K27me3, H3K9me1/2), parasite-predominant modifications (H3K4me3, H3K18me1, H3K27ac, H3K9ac, and H3K36me3), and shared modifications (e.g H3K4me1/2, H3K36me2, H4K5/8/12/16ac, H3K9me3 and H4K91Ac) suggesting a coordinated epigenetic strategy driving host cell programs to sustain proliferation and survival. Our study delineates a novel host-parasite epigenetic interface, positioning T. annulata as a unique model of epigenetic parasitism. By bridging parasitology and cancer epigenetics, these findings unveil new therapeutic opportunities targeting chromatin vulnerabilities in parasite-induced transformation.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.10.27.684985v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, cancer biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, cancer biology</span>
<span class="date">Oct 28, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Adaptive Kinase Signaling Enables Escape from Small-Molecule Inhibition in Glioblastoma</h2>
<p class="paper-authors">Hoosemans, L., van Heumen, M. J. A., Piepers, J., +3 authors, Vooijs, M. A.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Patients with glioblastoma (GBM) have a median survival of 15 months. Despite an intensive treatment schedule with resection, radio- and chemotherapy, recurrence is inevitable. A significant challenge to overcome GBM treatment resistance involves intratumoral heterogeneity, characterized by molecular, phenotypic, and clinical distinctive GBM subtypes. Different small molecule inhibitors (SMI) have been designed to inhibit signalling proteins in oncogenic driver pathways in GBM. However, SMIs have been unsuccessful in improving patient outcomes. Here, we investigate whether crosstalk between signalling pathways and signalling pathway redundancy are responsible for single-agent resistance using a primary patient-derived glioblastoma organoid (PGO) platform. This study used nine FDA-approved small-molecule inhibitors, based on their ability to cross the blood-brain barrier, targeting key GBM driver genes. Although inhibition of downstream effector proteins reduced cell viability more effectively (IC50 70nM-1M) than inhibiting upstream membrane-bound tyrosine kinase receptors (IC50 1-15M), remaining cell proliferation was seen in all six PGOs. To uncover resistance mechanisms, we analysed phosphokinase activity following monotherapy in various PGOs and identified compensatory pathway activation, leading to the discovery of effective small-molecule inhibitor combinations, most notably CHIR99021 (GSK-3 inhibitor) with trametinib (MEK inhibitor). In conclusion, our findings highlight the potential of combination therapy targeting compensatory pathways to overcome single-agent resistance in GBM, emphasizing the utility of patient-derived glioblastoma organoids as a platform for personalized therapeutic development.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.10.28.684838v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, cancer biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, cancer biology</span>
<span class="date">Oct 28, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Leveraging death of drug-sensitive cancer cells to promote immune-mediated bystander killing of subclones of drug-resistant tumor cells</h2>
<p class="paper-authors">Tomaschko, M., Ng, K., +10 authors, Downward, J.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> The response of lung cancer patients to drugs targeting the G12C mutant form of KRAS is limited by the development of resistance through multiple mechanisms. In order to achieve lasting benefit with these therapies, effective strategies for tackling the evolution of drug resistance are required. We have developed a preclinical model system to mimic the development of resistance to KRAS G12Ci inhibitors (G12Ci) such as adagrasib and RMC-4998. Treatment of tumors containing a minor subpopulation of resistant cancer cells with G12Ci leads to their rapid outgrowth to replace the drug-sensitive cells within a few weeks. However, when combined with therapies that, at least in part, target the immune response, such as SHP2 inhibitors or PD-1 blockade, drug-resistant cells can be eliminated, even by drug combinations that do not impact their growth in the absence of drug-sensitive cells. This bystander killing of drug-resistant cells when drug-sensitive cells are targeted is dependent on an intact adaptive immune system. Mechanistically, these combination therapies lead to profound remodelling of the tumor immune microenvironment, with influx of T cells recognizing a tumor associated antigen shared between drug-resistant and drug-sensitive cancer cells. Promotion of immune-mediated bystander killing of drug-resistant cells may provide a paradigm for tackling the problem of drug resistance in cancer more broadly.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.10.27.684825v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="machine learning, cancer biology">
<div class="paper-meta">
<span class="keyword">machine learning, cancer biology</span>
<span class="date">Oct 28, 2025</span>
<span class="journal">PubMed</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Stabilization of VDAC1 by USP10 facilitates radioresistance and metastasis in nasopharyngeal carcinoma via MTOR-dependent autophagy pathway: an experimental study.</h2>
<p class="paper-authors">Baisheng Lin, Guofeng Xie, +5 authors, Jian Zhang</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Radioresistance represents a major obstacle in the treatment of nasopharyngeal carcinoma (NPC), often leading to tumor recurrence and metastasis. Although autophagy is a critical regulatory process in cancer biology, its specific role in NPC radioresistance remains poorly understood.</p>
<a class="btn dark" href="https://pubmed.ncbi.nlm.nih.gov/41147769/" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="machine learning, cancer biology">
<div class="paper-meta">
<span class="keyword">machine learning, cancer biology</span>
<span class="date">Oct 28, 2025</span>
<span class="journal">MED</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Discovery of a Potent Lysine-Specific Histone Demethylase-1 (LSD-1) Inhibitor for Cancer Treatment via Machine Learning, Molecular Modeling, and In Vitro Validation.</h2>
<p class="paper-authors">Wang S, Duo L, +10 authors, Tang B.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> The discovery of potent lysine-specific histone demethylase 1 (LSD-1) inhibitors is of significant importance in cancer therapy, as LSD-1 overexpression contributes to tumour progression and malignancy through the regulation of gene expression. We employed an integrated strategy that combined machine learning-based virtual screening, bioassay validation, and ligand-target interaction analysis. Virtual screening identified 29 candidate compounds, of which 17 demonstrated micromolar-level enzymatic inhibition in experimental assays. Among them, compound L01 showed strong potential as an LSD-1 inhibitor, exhibiting significant antileukemic activity (IC50 = 24 µM) and confirmed binding to LSD-1 in Kasumi-1 cells. MST and SPR analysis confirmed the direct binding between L01 and LSD-1 inferred from our in vitro studies. Molecular docking and molecular dynamics simulations further supported the interaction, revealing a binding affinity of L01 comparable to that of the well-characterised reversible inhibitor SP-2577. ADMET analysis revealed that L01 possesses excellent Caco-2 permeability and high target specificity; however, its moderate drug-likeness and potential toxicity issues underscore the need for further optimisation. Through computational modelling and experimental validation, our study identifies L01 as a promising LSD-1 inhibitor with strong potential for optimisation to improve bioavailability and safety in cancer therapy.</p>
<a class="btn dark" href="https://pubmed.ncbi.nlm.nih.gov/41147772/" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="machine learning, cancer biology">
<div class="paper-meta">
<span class="keyword">machine learning, cancer biology</span>
<span class="date">Oct 28, 2025</span>
<span class="journal">MED</span>
</div>
<div class="paper-content">
<h2 class="paper-title">An Interpretable Complex Knowledge Multi-hop Reasoning Model for Predicting Synthetic Lethality in Human Cancers.</h2>
<p class="paper-authors">Cheng J, Zhang Y, +2 authors, Lu M.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Synthetic lethality (SL) has emerged as a promising strategy in cancer medicine. However, complex biomolecular interactions make wet lab methods time-consuming and expensive. Machine learning methods have gained widespread adoption for SL prediction in recent years. Although these methods have demonstrated particular effectiveness, they suffer from weak interpretability, making it difficult for users to understand the specific reasoning processes of the models. Also, they typically focus on a simple gene pair, thus struggling with more meaningful reasoning tasks involving other medical factors as in real life. To address these gaps, we propose an explainable multi-hop reasoning model EFOL-SL based on first-order logic queries. We first construct query graphs with triplet transformations for different tasks. Node embeddings are then fed into a sparse Transformer encoder and a visualized graph attention decoder to generate comprehensive multi-hop logical reasoning chains. By masking nodes in intermediate reasoning steps, our model can explicitly predict each node, allowing observation of its exact reasoning process. Additionally, we conduct extensive experiments on two widely used benchmarks with complex SL prediction tasks involving diverse medical entities. Evaluations demonstrate superior performance of our model over state-of-the-art methods on various tasks. Notably, EFOL-SL provides specific multi-hop logical reasoning chains behind its predictions, offering meaningful insights into the model's reasoning process.</p>
<a class="btn dark" href="https://pubmed.ncbi.nlm.nih.gov/41150233/" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="machine learning, cancer biology">
<div class="paper-meta">
<span class="keyword">machine learning, cancer biology</span>
<span class="date">Oct 16, 2025</span>
<span class="journal">PubMed</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Serum Fingerprinting-Based Integrative Dual-Omics Machine Learning for Endometriosis-Associated Ovarian Cancer.</h2>
<p class="paper-authors">Man Zhang, Yunqiang Zhang, +5 authors, Chunhui Deng</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Dual-omics, by integrating molecular information from two distinct dimensions, can offer more comprehensive perspective for complex disease. Herein, we developed an efficient functionalized mesoporous nanoparticle-coupled laser desorption/ionization mass spectrometry (fMNPLDI-MS) platform capable of extracting high-quality serum metabolic fingerprints (SMFs) and serum peptide fingerprints (SPFs) from trace serum samples within 50 s. Integrating these SMFs and SPFs markedly improved early screening and subtyping of endometriosis-associated ovarian cancer (EAOC) more than single omics. Specifically, leveraging machine learning on the identified 6 metabolites and 6 peptides, the integration strategy attained an area under the receiver operating characteristic curve (AUC) value of 0.989 and accuracy of 93.1%, compared with 0.964/89.7% for metabolomics alone and 0.960/87.9% for peptidomics alone in distinguishing EAOC from benign controls. Similarly, the dual-omics integration strategy achieved an AUC value of 0.875 and accuracy of 86.7%, surpassing individual metabolomics (AUC: 0.869, accuracy: 83.3%) and peptidomics (AUC: 0.733, accuracy: 76.7%), in subtype classification. This high-throughput fMNPLDI-MS dual-omics platform provides a powerful tool for EAOC screening and subtyping, paving the way toward early detection and precision management of this malignancy.</p>
<a class="btn dark" href="https://pubmed.ncbi.nlm.nih.gov/41099367/" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div></section>
</main>
<script>
    // === EXPAND / COLLAPSE PAPERS ===
    document.querySelectorAll('.paper').forEach(paper => {
      paper.addEventListener('click', e => {
        if (e.target.tagName.toLowerCase() === 'a') return;
        paper.classList.toggle('expanded');
      });
    });

    // === FILTER SYSTEM ===
    const papers = document.querySelectorAll('.paper');
    const filterBar = document.getElementById('filterBar');

    // Extrai keywords
    const allKeywords = new Set();
    papers.forEach(paper => {
      paper.dataset.keywords.split(',').forEach(k => allKeywords.add(k.trim()));
    });

    // Cria botões dinamicamente
    const filters = ['all', ...Array.from(allKeywords)];
    filters.forEach(keyword => {
      const btn = document.createElement('button');
      btn.className = 'filter-btn';
      btn.textContent = keyword;
      if (keyword === 'all') btn.classList.add('active');
      filterBar.appendChild(btn);
    });

    // Filtro de papers
    filterBar.addEventListener('click', e => {
      if (!e.target.classList.contains('filter-btn')) return;

      document.querySelectorAll('.filter-btn').forEach(b => b.classList.remove('active'));
      e.target.classList.add('active');

      const keyword = e.target.textContent;
      papers.forEach(paper => {
        if (keyword === 'all') {
          paper.style.display = 'grid';
        } else {
          const hasKeyword = paper.dataset.keywords.toLowerCase().includes(keyword.toLowerCase());
          paper.style.display = hasKeyword ? 'grid' : 'none';
        }
      });
    });
  </script>
</body>
</html>
