<!DOCTYPE html>

<html lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width, initial-scale=1.0" name="viewport"/>
<title>paper-trackr newsletter</title>
<link href="../docs/photo_51.ico" rel="shortcut icon"/>
<link href="../style.css" rel="stylesheet"/>
<link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&amp;display=swap" rel="stylesheet"/>
</head>
<body>
<aside class="sidebar">
<div class="sidebar-inner">
<a class="logo" href="../index.html">felipevzps</a>
<nav class="nav">
<a href="../index.html#about">About</a>
<a href="../index.html#research">Research</a>
<a href="../index.html#hackathon">Hackathon</a>
<a href="../index.html#contact">Contact</a>
</nav>
</div>
</aside>
<main class="newsletter-container">
<section class="newsletter-intro">
<h1>Latest in Science</h1>
<p>
        Hello, welcome to your daily dose of research!<br/> 
        Here you'll find some papers published yesterday, filtered for the topics I follow.
      </p>
<p class="note">
<small>
          * The curation is done automatically by my open-source project, <a href="https://github.com/felipevzps/paper-trackr" target="_blank">paper-trackr</a>.<br/>
          Want your own results? Install it via PyPI and get personalized updates!
        </small>
</p>
</section>
<!-- Filtro de keywords -->
<div class="filter-bar" id="filterBar"></div>
<section class="papers-list" id="papersList"><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 18, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Polus: a Transformer-based Soft-decision Codec Enhancement Platform for DNA Storage</h2>
<p class="paper-authors">Ding, L., Wang, K., +6 authors, Zhu, Z.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> DNA storage offers exceptional information density and archival longevity, but is constrained by the complex, heterogeneous errors inherent to synthesis, storage, and sequencing. Conventional error-correction schemes often rely on excessive logical redundancy to mitigate these biochemical imperfections, thereby compromising storage efficiency. Here, we introduce Polus, a deep-learning- enabled platform that bridges the gap between biochemical constraints and digital reliability through soft-decision decoding. At its core is SeqFormer, a Transformer-based channel model that synergizes sequence context with quality signals to characterize platform-specific error profiles, generating calibrated per-base confidence scores. This mechanism transforms uncertain biochemical noise into informative "soft" erasures. In in silico benchmarks, Polus significantly enhances mainstream codecs: it reduces the sequencing coverage required for DNA Fountain by 38.9%, increasing effective physical density by ~80%, and eliminates persistent indel-induced errors in the Yin Yang codec. Furthermore, it enables a targeted resequencing strategy that achieves full recovery with 99.9% less overhead than brute-force deepening. To formalize these gains and address the lack of systematic benchmarking in the field, Polus establishes a standardized nine-metric evaluation framework that rigorously quantifies the trade-offs between reliability, density, and cost. This work provides a reproducible, quantitative foundation for next-generation, context-aware DNA storage systems.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.16.694663v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 18, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">FragmentScope - exploring the fragment space with learned surface representations</h2>
<p class="paper-authors">Elizarova, E., Morozova, I., +9 authors, Correia, B.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Exploring fragment chemical space for ligand design remains a major challenge in early stage drug discovery. This task is particularly challenging due to the small size, low specificity, and weak binding affinities of low molecular weight (MW) fragments. We present FragmentScope, a computational pipeline that uses learned protein surface fingerprints to guide fragment placement and small molecule generation. By using a contrastive learning model trained on protein-ligand interactions, we built a database of surface-fragment pairs, which enables fast and accurate fragment placement in the target protein pocket. We demonstrate its effectiveness on benchmark datasets, achieving robust placement accuracy. FragmentScope also enables the design of small molecules based on the predicted fragments ensuring synthetic accessibility. We experimentally validated Fragmentscope across 5 different targets with binding assays and structural characterization. Our approach shows high success rates in fragment discovery and yielded promising leads for designed ligands. FragmentScope offers a scalable, structure-guided approach for narrowing chemical space and identifying prospective scaffolds, accelerating the early stages of small molecule design.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.16.694391v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 18, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Spatially-smoothed quantification improves cell typing in imaging mass cytometry datasets</h2>
<p class="paper-authors">Gerber, R., Griner, J., +3 authors, Robinson, M. D.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Accurate cell type annotation in imaging mass cytometry (IMC) and related technologies critically depends on preprocessing steps such as normalization, segmentation, and marker aggregation. More distinct separation between negative and positive signals enables more precise cell boundary inference and more robust marker assignment to single cells. However, inherent limitations in spatial resolution and uncertain cell boundaries can lead to spillover, where signal leaks from one cell to a neighboring cell, distorting marker intensities and leading to incorrect cell type annotation. To address these challenges, we first systematically investigated the impact of spatial resolution and segmentation variability on per-cell marker aggregation using simulated IMC datasets, establishing upper limits for reliable marker separation in cell type annotation. We further analyzed technical biases in large scale studies, demonstrating that appropriate normalization approaches can significantly reduce batch effects without compromising biological variability. Finally, we benchmarked multiple spillover correction strategies across both (semi-)simulated and real IMC datasets. Our results revealed two simple methods: spatial smoothing of intensity images followed by mean marker aggregation, or resampling of cell masks followed by mean marker aggregation and median calculation, both of which improved annotation performance. Other methods often performed worse than the baseline of simple mean aggregation. Together, these findings underscore the central importance of spatial resolution, normalization, and marker aggregation in IMC data preprocessing for accurate single-cell annotation.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.16.693927v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 18, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">IDR searcher: a search engine solution for public image resource</h2>
<p class="paper-authors">Mohamed, K., Moore, W., +4 authors, Burel, J.-M.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> We introduce the IDR searcher, an open-source search engine designed to ease the exploration of datasets hosted in public bioimaging resources. The application offers a fast, efficient, cost-effective solution for datasets discovery and has the potential to address current disparities in finding quality datasets for exploratory research and to provide the foundation for cross-resource search.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.16.694566v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 18, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">A Multi-Modal Transfer Learning Framework to Reduce Health Disparities in Prostate Adenocarcinoma</h2>
<p class="paper-authors">Li, L., Wang, J., Wan, S.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Prostate cancer is the second most common cancer in men across the United States, of which prostate adenocarcinoma (PRAD) is the most common subtype. Despite remarkable progress has been made in PRAD diagnosis and prognosis, Black American men have been found to have disproportionately high incidence and mortality rate in PRAD compared with non-Hispanic White men in the United States. While machine learning (ML) methods like transfer learning (TL) have shown promise in reducing racial disparities in PRAD, its effectiveness is often compromised by the requirement for large-scale training datasets, which are challenging to obtain in clinical settings. In addition, existing ML approaches only leverage single-omics data without integrating multi-omics information. To address these concerns, we propose a novel approach called MOTLPRAD that develops a Multi-Omics integration Transfer Learning framework to reduce health disparities in PRAD. Specifically, we first investigated two multi-modal ensemble methods, Pearson correlation coefficient (PCC) based patient-pairwise similarity and variational autoencoder (VAE), to integrate different types of omics data. Then, we adopted a transfer learning model based on domain adaptation to pre-train the model on the majority group (e.g., non-Hispanic White Americans) and fine-tune the model using the minority group (e.g., Black Americans). To mitigate data imbalance across different ethnic groups, we leveraged the Synthetic Minority Oversampling Technique (SMOTE) to augment the sample size of minority groups, which could further improve the performance of reducing health disparities in PRAD. Results based on a series of multi-omics data (mRNA, miRNA, and methylation) from The Cancer Genome Atlas (TCGA) database suggested that our proposed model significantly outperformed conventional transfer learning models and other computational models for reducing racial disparities when predicting progression-free interval (PFI) prognosis for PRAD patients. In addition, our model also demonstrated that integrating multi-omics data could remarkably boost the performance of mitigating health disparities compared to using single-omics data. We believe our proposed framework could provide a new way to systematically mitigate health disparities towards underrepresented groups for PRAD diagnosis, prognosis, and treatment.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.15.694538v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 18, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">DOMINO: diffusion-optimised graph learning identifies domain structures with enhanced accuracy and scalability</h2>
<p class="paper-authors">Jia, P., Liu, N. W., +13 authors, Li, F.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Spatial transcriptomics enables molecular profiling in situ, and identifying spatial domains facilitates our understanding of functional compartments of tissues. However, most existing domain calling methods do not scale to rapidly increasing data sizes and focus only on local structure while missing the global view of the tissue. Here, we present DOMINO, a diffusion-optimised contrastive learning framework for spatial domain detection. DOMINO utilises graph diffusion convolution to propagate information beyond immediate neighbours, and jointly optimises local and global graph structure via contrastive learning. This novel framework yields biologically interpretable domains with clearer boundaries and scales to large datasets, outperforming state-of-the-art methods across healthy and malignant benchmark datasets. Applying DOMINO to a newly generated spatial dataset from endometriosis-associated ovarian cancers, which failed to be processed by existing domain detection methods due to its large size, we found that DOMINO reveals a shared epithelial continuum along a proliferation associated axis spanning FOXJ1 high proliferative to CD55 high non-proliferative domains, consistently coupled to coordinated stromal remodelling. DOMINO also identified subtype specific spatial domains, including spatially restricted WNT beta catenin activation in endometriod and discrete NDRG1 high metabolic domains in clear cell ovarian cancers, features that are not recovered by standard expression based clustering workflows.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.15.694536v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 18, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">LoMuS: Low-Rank Adaptation with Multimodal Representations Improves Protein Stability Prediction</h2>
<p class="paper-authors">Trillos, S. I., Singh, A., Kabir, A.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Protein folding stability is a key determinant for understanding protein dynamics, including molecular function, pathogenicity, and/or protein engineering. Yet, accurate prediction of protein stability changes remains a challenging problem due to the high-variability in the available data, especially from sequence-only information when structural knowledge is of low-resolution or unavailable. In this work, we introduce LoMuS, a Multimodal deep learning model that combines two complimentary aspects of the molecule and predicts unnormalized protein Stability effect from the primary sequence as input. In the core of the model architecture, a fusion network integrates explicit physicochemical descriptors with Low-rank adapted protein language model derived embeddings from the sequence that shows powerful and accurate generalization ability across various benchmark settings for predicting protein folding stability changes. We compared and rigorously evaluated our model capacity spanning from fold-induced stability changes to mutation caused stability effect prediction. This includes benchmarking against various held-out protein domains, out-of-distribution label settings and per-protein evaluation. LoMuS consistently outperforms other sequence-only protein stability baselines. It achieves an absolute performance gain by an at least 10% in the spearman rank correlation metric for predicting protein stability across many held-out domains and out-of-distribution stability label predictions. Per-protein validation additionally demonstrates promising performance gain of our model. Ablation analysis on the model architectural choices confirms that complementary signals from derived features are critical for this multimodal approach. We believe LoMuS advances protein engineering research and can aid in rational protein design by elucidating precise protein stability changes.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.15.694540v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 18, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Fourier Helicity Spectra as Quantifiers of Multiscale Chirality</h2>
<p class="paper-authors">Im, S. W., Ma, J., +1 author, Kotov, N. A.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Quantifying the chirality of three-dimensional (3D) objects is crucial for understanding interactions of nanoscale structures, biological molecules, and hierarchical materials. Although chirality is simultaneously present at multiple scales for each case, existing chirality measures rely only on singular scale-specific quantifiers, and some of them yield incorrect chirality assignments for biomolecules. Here we introduce the Fourier Helicity Spectra, a mathematical framework to decompose geometry of chemical and biological structures encompassing the scale dependence of mirror asymmetry. Fourier decomposition of 3D objects into helical harmonics enables one to compute a handedness-resolved characteristics of complex shapes, while reducing computational cost. We show its applicability for a diverse range of chemical systems, from biomacromolecular electron densities to tomographic reconstructions of inorganic nanostructures. Furthermore, inverse transform of isolated spectral components into real space enables visualization of helicity-matched interfaces, identifying how chirality governs biomolecular interactions. We anticipate that this methodology can be generalized with alternative basis sets to encompass a broad range of chiral symmetries governing biological signaling, polarization optics, drug design, and advanced synthesis.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.15.694500v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, cancer biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, cancer biology</span>
<span class="date">Dec 18, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Human bone marrow adipocytes drive prostate cancer bone metastasis progression via lipid-mediated induction of Angiopoietin-like 4</h2>
<p class="paper-authors">Hernandez, M., Shin, S., +9 authors, Attane, C.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Bone is the main metastatic site in advanced prostate cancer (PCa) and contains bone marrow adipocytes (BMAds), which account for more than 70% of adult bone marrow. However, their role in the progression of PCa metastases remains poorly understood. Herein, we developed a physiologically relevant 3D culture model using primary human BMAds from red hematopoietic rich-areas (rBMAds) and we showed that rBMAds engage in metabolic crosstalk with PCa cells. Specifically, rBMAds release free fatty acids (FFAs) through a non-canonical lipolytic pathway and these FFAs are taken up by PCa cells inducing a transcriptional reprogramming that promotes motility. Among the responsive genes, Angiopoietin like 4 (ANGPTL4) is the most upregulated via a peroxisome proliferator-activated receptor {gamma}-dependent mechanism, and its silencing abolishes the rBMAd-driven migratory and invasive phenotype. In bone metastases of patients with PCa, high ANGPTL4 levels correlate with poor survival. Together, our findings uncover a novel adipocyte-tumor interaction and identify ANGPTL4 as a key mediator and potential therapeutic target in bone-metastatic PCa.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.16.694397v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, cancer biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, cancer biology</span>
<span class="date">Dec 18, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Targeting glucocorticoid-induced CD20 activation in preclinical models of B-ALL</h2>
<p class="paper-authors">Bigot, J., Bouttier, M., +18 authors, Gerby, B.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Pediatric B-cell acute lymphoblastic leukemia (B-ALL) is effectively controlled with contemporary multi-agent chemotherapy, resulting to 5-year survival rates above 90%. However, relapse occurs in 15-20% of patients due to minimal residual disease (MRD), characterized by the presence of persisting and resistant leukemic cells, and associated with a poor clinical outcome. Despite its prognostic relevance, the molecular features driving MRD are poorly characterized. In this study, we developed patient-derived xenograft (PDX) models from matched diagnosis and relapse B-ALL samples combined to chemotherapy to mimic MRD in vivo. Drug-tolerant leukemic cells were profiled using single-cell RNA sequencing and we identified a transcriptionally distinct MRD-like population enriched for cell-quiescence, inflammatory stress, and B-cell receptor pathway signatures. Strikingly, the B-lymphocyte surface antigen CD20, encoding by MS4A1 gene, emerged as a consistent upregulated marker in MRD cells from PDXs and patients with diverse oncogenic subtypes. We further demonstrated that CD20 expression is induced by glucocorticoid exposure, creating a therapeutic opportunity where anti-CD20 monoclonal antibodies selectively eradicated MRD cells in vivo. Our data highlight CD20 not only as a biomarker but as an actionable vulnerability in B-ALL MRD, supporting clinical evaluation of anti-CD20 immunotherapy during induction treatment to kill drug-resistant cells and reduce relapse risk.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.16.694550v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, cancer biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, cancer biology</span>
<span class="date">Dec 18, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Romaciclib, a CDK8/CDK19 inhibitor, can overcome venetoclax resistance through a combinatorial strategy</h2>
<p class="paper-authors">Pakulska, U., Obacz, M., +19 authors, Mazan, M.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> The combination of venetoclax (VEN) and hypomethylating agents (HMA) is the standard of care in acute myeloid leukemia (AML) for elderly patients unfit for intensive chemotherapy. Despite its clinical success, most patients eventually relapse, creating an urgent need for effective therapeutic alternatives. In this study, we aimed to evaluate the potential of romaciclib, a first-in-class CDK8/CDK19 inhibitor, in combination with VEN to overcome stroma-mediated and primary/acquired VEN-resistance. We assessed the efficacy of RVU120+VEN combination in both sensitive and resistant AML cell lines and primary patient-derived models. Our finding demonstrated that romaciclib synergizes with VEN in AML cell lines and in 8 out of 11 patient-derived cell samples. The proteomic and functional studies demonstrated that combination induced apoptosis through caspase-dependent cleavage of MCL-1. In vivo studies confirmed the efficacy of RVU120+VEN, showing eradication of leukemic cells and bone marrow recovery. Importantly, the combination effectively overcame both stroma-mediated and transcriptionally dependent VEN-resistance. Mechanistic studies, focusing on transcriptomic analyses, identified key resistance-associated pathways, including IL6/JAK/STAT3, TGF-{beta}, PI3K/AKT/MTOR, and inflammatory signaling, being suppressed by combination treatment. Furthermore, an in vivo study using a VEN-resistant patient-derived xenograft (PDX) model confirmed the efficacy of the combination, demonstrating a significant reduction in leukemia burden and a decreased proportion of leukemia initiating cells (LIC) following treatment. These findings prove the highly synergistic mechanism of action of RVU120+VEN combination and the potential to overcome primary/acquired VEN resistance in relapse/refractory AML disease. Altogether, the presented results support ongoing clinical studies evaluating romaciclib and VEN in VEN/HMA-refractory patients (NCT06191263) and provide a basis for future exploration as a frontline therapy in VEN-naive patients.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.16.693978v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="machine learning, cancer biology">
<div class="paper-meta">
<span class="keyword">machine learning, cancer biology</span>
<span class="date">Dec 18, 2025</span>
<span class="journal">PubMed</span>
</div>
<div class="paper-content">
<h2 class="paper-title">CRISPRi screening in cultured human astrocytes uncovers distal enhancers controlling genes dysregulated in Alzheimer's disease.</h2>
<p class="paper-authors">Nicole F O Green, Gavin J Sutton, +16 authors, Irina Voineagu</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Genetic variants associated with complex traits often lie in distal enhancers. While candidate enhancers have been mapped genome wide, their functional state and gene targets in specific cell types remain unclear. Here we present AstroREG, a resource of enhancer-gene interactions in human primary astrocytes, generated by combining CRISPR inhibition (CRISPRi), single-cell RNA-seq and machine learning. By functionally testing nearly 1,000 PsychENCODE enhancers, we identified more than 150 regulatory interactions, revealing enhancers that control key astrocyte functions and genes implicated in Alzheimer's disease. The CRISPRi screen also provided valuable ground-truth data from a primary cell type for training and benchmarking prediction models of enhancer activity. We thus developed EGrf, a random forest (RF) model trained on these data, and applied it genome wide to predict regulatory interactions with high specificity. Together, our data provide a comprehensive functional map of enhancer-mediated regulation in a key glial cell type, shedding light on brain function and disease.</p>
<a class="btn dark" href="https://pubmed.ncbi.nlm.nih.gov/41413662/" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="machine learning, cancer biology">
<div class="paper-meta">
<span class="keyword">machine learning, cancer biology</span>
<span class="date">Dec 18, 2025</span>
<span class="journal">PubMed</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Mechanistically informed machine learning links non-canonical TCA cycle activity to Warburg metabolism and hallmarks of malignancy.</h2>
<p class="paper-authors">Lihao Lin, Francesco Lapi, +3 authors, Chiara Damiani</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Cancer cells undergo extensive metabolic rewiring to support growth, survival, and phenotypic plasticity. A non-canonical variant of the tricarboxylic acid (TCA) cycle, characterized by mitochondrial-to-cytosolic citrate export, has emerged as critical for embryonic stem cell differentiation. However, its role in cancer remains poorly understood. Here, we present a two-step computational framework to systematically analyze the activity of this non-canonical TCA cycle across over 500 cancer cell lines and investigate its role in shaping hallmarks of malignancy. First, we applied constraint-based modeling to infer cycle activity, defining two complementary metrics: Cycle Propensity, measuring the likelihood of its engagement in each cell line, and Cycle Flux Intensity, quantifying average flux through the reaction identified as rate-limiting. We identified distinct tumor-specific patterns of pathway utilization. Notably, cells with high Cycle Propensity preferentially reroute cytosolic citrate via aconitase 1 (ACO1) and isocitrate dehydrogenase 1 (IDH1), promoting [Formula: see text]-ketoglutarate ([Formula: see text]KG) and NADPH production. Elevated engagement of this cycle strongly correlated with Warburg-like metabolic shifts, including decreased oxygen consumption and increased lactate secretion. In the second step, to uncover non-metabolic transcriptional signatures associated with non-canonical TCA cycle activity, we performed machine learning-based feature selection using ElasticNet and XGBoost, identifying robust gene signatures predictive of cycle activity. Over-representation analysis revealed enrichment in genes involved in metastatic behavior, angiogenesis, stemness, and key oncogenic signaling. SHapley Additive exPlanations (SHAP) further prioritized genes with the strongest predictive contributions, highlighting candidates for experimental validation. Correlation analysis of DepMap gene-dependency profiles revealed distinct vulnerability patterns associated with non-canonical TCA cycle activity, outlining a characteristic landscape of genetic dependencies. Together, our integrative framework uniting constraint-based metabolic modeling and machine learning systematically reveals how non-canonical TCA cycle dynamics underpin metabolic plasticity and promote malignant traits.</p>
<a class="btn dark" href="https://pubmed.ncbi.nlm.nih.gov/41411367/" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div></section>
</main>
<script>
    // === EXPAND / COLLAPSE PAPERS ===
    document.querySelectorAll('.paper').forEach(paper => {
      paper.addEventListener('click', e => {
        if (e.target.tagName.toLowerCase() === 'a') return;
        paper.classList.toggle('expanded');
      });
    });

    // === FILTER SYSTEM ===
    const papers = document.querySelectorAll('.paper');
    const filterBar = document.getElementById('filterBar');

    // Extrai keywords
    const allKeywords = new Set();
    papers.forEach(paper => {
      paper.dataset.keywords.split(',').forEach(k => allKeywords.add(k.trim()));
    });

    // Cria botÃµes dinamicamente
    const filters = ['all', ...Array.from(allKeywords)];
    filters.forEach(keyword => {
      const btn = document.createElement('button');
      btn.className = 'filter-btn';
      btn.textContent = keyword;
      if (keyword === 'all') btn.classList.add('active');
      filterBar.appendChild(btn);
    });

    // Filtro de papers
    filterBar.addEventListener('click', e => {
      if (!e.target.classList.contains('filter-btn')) return;

      document.querySelectorAll('.filter-btn').forEach(b => b.classList.remove('active'));
      e.target.classList.add('active');

      const keyword = e.target.textContent;
      papers.forEach(paper => {
        if (keyword === 'all') {
          paper.style.display = 'grid';
        } else {
          const hasKeyword = paper.dataset.keywords.toLowerCase().includes(keyword.toLowerCase());
          paper.style.display = hasKeyword ? 'grid' : 'none';
        }
      });
    });
  </script>
</body>
</html>
