<!DOCTYPE html>

<html lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width, initial-scale=1.0" name="viewport"/>
<title>paper-trackr newsletter</title>
<link href="../docs/photo_51.ico" rel="shortcut icon"/>
<link href="../style.css" rel="stylesheet"/>
<link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&amp;display=swap" rel="stylesheet"/>
</head>
<body>
<aside class="sidebar">
<div class="sidebar-inner">
<a class="logo" href="../index.html">felipevzps</a>
<nav class="nav">
<a href="../index.html#about">About</a>
<a href="../index.html#research">Research</a>
<a href="../index.html#hackathon">Hackathon</a>
<a href="../index.html#contact">Contact</a>
</nav>
</div>
</aside>
<main class="newsletter-container">
<section class="newsletter-intro">
<h1>Latest in Science</h1>
<p>
        Hello, welcome to your daily dose of research!<br/> 
        Here you'll find some papers published yesterday, filtered for the topics I follow.
      </p>
<p class="note">
<small>
          * The curation is done automatically by my open-source project, <a href="https://github.com/felipevzps/paper-trackr" target="_blank">paper-trackr</a>.<br/>
          Want your own results? Install it via PyPI and get personalized updates!
        </small>
</p>
</section>
<!-- Filtro de keywords -->
<div class="filter-bar" id="filterBar"></div>
<section class="papers-list" id="papersList"><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Jan 27, 2026</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">A study of k-mer patterns within DNA molecules sequenced from a chondrite</h2>
<p class="paper-authors">Farage, C., Bachelet, I.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> The known tree of life occupies an infinitesimal region of the space of all mathematically possible evolutionary histories, yet our methods implicitly assume that any genetic material we encounter must have originated from somewhere within it, potentially rendering truly foreign genetic systems invisible to detection. Here we studied k-mer patterns in DNA molecules sequenced from internal samples within the Zag chondrite in order to associate these sequences to a known taxonomic neighborhood within the tree of life. In an attempt to capture every conceivable pattern, we used the distributions of k-mers across the BLAST NT database, in novel genomes, in various control groups including random sequences, and in long sequences of DNA composed entirely of unrepresented k-mers (primes) emulating foreign material. Our analysis showed that all &gt;1,000,000 k-mers (k = 21) comprising the Zag sequences are primes, with 0 represented in the NT database. Taken together, these results highlight a potential methodological limitation in current frameworks for interpreting newly found genetic sequences, which are implicitly calibrated to the known tree of life.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2026.01.26.701670v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Jan 27, 2026</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">UdonPred: Untangling Protein Intrinsic Disorder Prediction</h2>
<p class="paper-authors">Schlensok, J., Wagemann, D., +2 authors, Rost, B.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Motivation: Regions in intrinsic disordered proteins (IDPs) constitute important continuous aspects of protein function. While their existence on a structural continuum is widely accepted, most computational predictions have, nevertheless, focused on binary classifications. Existing datasets are severely limited in size and experimental evidence for continuous disorder. Results: Building on recently released datasets of continuous protein disorder and flexibility, we introduce UdonPred, a lightweight neural network exclusively inputting embeddings from the protein Language Model (pLM) ProstT5 to predict per-residue protein disorder from sequence alone. Training and evaluating UdonPred on seven datasets with divergent definitions of disorder and flexibility suggests that not model capacity, but agreement and nuance of disorder annotations, remains the main driver of performance. Binary disorder annotations can be reliably predicted from a multitude of different disorder and flexibility datasets, but there is still room for improvement in predicting continuous disorder.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2026.01.26.701679v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Jan 27, 2026</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Ratatoskr: A tool for automated retrieval of taxonomic type strain sequences and metadata</h2>
<p class="paper-authors">Turkington, C., Bastiaanssen, F., +2 authors, Hill, C.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Bacterial taxonomic type strains anchor species names to physical and genomic reference material, making them essential for reproducible and comparable prokaryotic research. While reference strains are often well-characterised through curated metadata, nomenclature histories, and sequence records, no single database holds up-to-date information on all these aspects, resulting in fragmented information. Gathering the complete set of information for a type strain is further complicated by inconsistencies in nomenclature between sources due to the often-numerous synonyms that can describe a single strain. As a result, collecting type strain data for taxonomic proposals and emendations can be an onerous task requiring extensive manual curation. To address this issue, we introduce Ratatoskr, a Python-based tool that automates the retrieval of sequences and metadata for bacterial taxonomic type strains. Ratatoskr facilitates this by collecting the latest type strain information of the List of Prokaryotic names with Standing in Nomenclature (LPSN) and using this information to query the BacDive and NCBI databases. By applying known taxonomic synonym information Ratatoskr is able to resolve cross-database inconsistencies and streamline the retrieval process. We show that through its use, Ratatoskr can obtain metadata and sequence data for type strains of bacteria within minutes to seconds, depending on the number of members within the requested taxon. By automating this retrieval, Ratatoskr provides fast, accurate, and readily shareable starting points for studies involving the use of taxonomic type strains and data, such as new taxonomic proposals or emendations. Ratatoskr was developed using Python 3 and is freely available at https://github.com/Fabian-Bastiaanssen/Ratatoskr under a GPL-3.0 licence.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2026.01.26.700362v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Jan 27, 2026</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Enhancing interpretability of cryo-EM maps with hybrid attention Transformers</h2>
<p class="paper-authors">Lin, J., Zhang, Z., +3 authors, Zhou, X.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Cryo-electron microscopy (cryo-EM) has become a central technique for determining the structures of macromolecular complexes. However, experimental cryo-EM density maps often exhibit limited interpretability due to noise and heterogeneous, anisotropic resolution, complicating downstream atomic model construction. Here we present DEMO-EMReF, a Swin Transformer-based framework specifically designed for cryo-EM density map refinement that integrates channel attention with hybrid spatial attention. By combining local window-based attention with grid-based sparse global attention, DEMO-EMReF captures fine-grained local density features while modeling long-range, cross-region spatial dependencies, enabling more robust density refinement in heterogeneous and anisotropic map regions. DEMO-EMReF was systematically evaluated on large benchmark sets of both primary maps and half-maps spanning resolutions of 3.0-8.0 [A] and was compared against multiple state-of-the-art map enhancement methods. DEMO-EMReF consistently improves resolution-related metrics, map-model correlations, and local atomic resolvability, with particularly robust gains in challenging heterogeneous or anisotropic cases. Importantly, the refined maps facilitate more accurate and efficient automated atomic model building. Together, DEMO-EMReF provides a robust approach for enhancing cryo-EM density maps and enabling more reliable downstream studies.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2026.01.26.701621v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Jan 27, 2026</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Collective Posterior Inference from Highly Variable Empirical Replicates</h2>
<p class="paper-authors">Ben Nun, N., Rosset, S., +1 author, Ram, Y.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> High-throughput experimental platforms now routinely generate data from dozens or hundreds of independent observations. Simulation-based inference (SBI) offers a powerful framework for estimating model parameters from such complex datasets, but standard methods struggle to scale to the noisy multiple-replicates regime without incurring prohibitive computational costs or careful hyperparameter tuning. Here, we introduce a new method for fast and robust collective posterior inference from multiple independent replicates using a robust product-of-experts aggregation scheme that automatically mitigates the influence of outliers. Evaluating it on synthetic and empirical evolutionary datasets, we find it achieves state-of-the-art estimation accuracy and computational efficiency, including inference from noisy observations. Our method is compatible with any SBI framework, providing a scalable, plug-and-play solution for inference from noisy multiple-replicate datasets.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2026.01.26.701673v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Jan 27, 2026</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Thyra: Bridging Mass Spectrometry Imaging and SpatialData for Unified Multi-Modal Analysis</h2>
<p class="paper-authors">Visvikis, T., Vierdag, W.-M. A. M., +2 authors, Cuypers, E.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Mass Spectrometry Imaging (MSI) is a powerful technique for mapping molecular distributions, and its integration with other imaging modalities is crucial for comprehensive understanding of molecular systems. Fragmented data formats and the limitations of existing standards like imzML, challenge spatial biology centric multi-modal data analysis and adherence to FAIR data principles. This paper introduces Thyra, a modern Python library designed to convert MSI data into the SpatialData framework, a unified and extensible multi-platform file format that crucially integrates MSI into the broader spatial omics ecosystem. Thyra's modular architecture, intelligent mass axis resampling, and sparse matrix backend address performance bottlenecks and facilitate seamless integration with tools for advanced spatial statistics and visualization. By adopting SpatialData, Thyra not only improves data interoperability, accessibility, and reusability, but also unlocks new avenues for multi-modal research, empowering scientists to integrate rich chemical information into diverse biological workflows.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2026.01.26.700792v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Jan 27, 2026</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Hidden causal inference delineates dynamic lncRNA regulation in autism spectrum disorder</h2>
<p class="paper-authors">Zhang, J., Wei, X., +2 authors, Fan, S.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Long non-coding RNAs (lncRNAs) are increasingly implicated in autism spectrum disorder (ASD), yet their causal regulatory mechanisms remain poorly characterized. Conventional static causal inference methods fail to capture the dynamic interplay of lncRNAs across diverse brain biological contexts. Here, we develop a hidden causality-based method Clean to infer dynamic lncRNA causal regulation in ASD. Our analysis reveals that dark causality, representing a hybrid form of positive and negative interdependencies, dominates lncRNA-mediated regulation in ASD. Moreover, dynamic lncRNA causal regulation emerges in diverse brain biological contexts, and 14 ASD risk lncRNAs significantly contribute to disease pathogenesis through disrupted gene regulatory programs. We also illustrate that lncRNA expression signatures enable robust classification of sex and cell type, offering potential diagnostic lncRNA biomarkers. Notably, 20 immune-related lncRNAs are prominently involved in the immune, suggesting a potential role for neuroimmune regulation. Additionally, cell-cell interaction networks diverge substantially between ASD and Normal cohorts. This study establishes a paradigm for delineating the dynamic causal regulation of lncRNAs, underscoring their diagnostic and therapeutic potential in ASD.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2026.01.25.701433v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Jan 27, 2026</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Dual-channel graph learning reveals similarity and complementarity in protein-protein interaction networks</h2>
<p class="paper-authors">Tang, T., Shen, T., +5 authors, Luo, X.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Protein-protein interactions (PPIs) are governed by two fundamental interfacial mechanisms: similarity-driven, often involving symmetric structural motifs, and complementarity-driven, arising from geometric and physicochemical matching between binding surfaces. Despite their biological significance, computational models have largely overlooked the coexistence and interplay of these twofold interaction modes. Here, we introduce DMG-PPI, a dual-channel graph neural network framework that jointly models similarity and complementarity in PPI networks, extending prior heterophilous GNN concepts to explicitly disentangle these dual interaction modes. The model consists of two parallel message-passing pathways: Alignment Message Passing (AMP), which aggregates signals from similar neighbors to capture symmetric interfaces, and Divergence Message Passing (DMP), which contrasts node features to extract complementary binding patterns. The signals captured by AMP and DMP are integrated via an adaptive fusion strategy within each block, and the outputs of blocks are aggregated using the MixHop framework to encode higher-order interaction patterns. DMG-PPI substantially outperforms state-of-the-art methods on classical benchmark datasets, achieving a 7.19% improvement in Micro-F1 over the second-best method. Additionally, the dual-channel framework provides interpretable insights into key binding residues by identifying interfacial mechanisms. Overall, DMG-PPI serves as a powerful tool that reveals the mechanisms behind accurate PPI predictions and facilitates downstream biological analysis.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2026.01.26.701665v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Jan 27, 2026</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">AstraKit: Customizable, reproducible workflows for biomedical research and precision medicine</h2>
<p class="paper-authors">Kurz, N. S., Kornrumpf, K., +1 author, Doenitz, J.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Fragmented bioinformatics tools compels researchers and clinicians to resort to error-prone manual pipelines. The success of precision medicine depends largely on the efficient interpretation of genetic variants and the selection of highly effective targeted therapies. However, the success of precision medicine and biomedical research depends on the availability of efficient software solutions for processing and interpreting genetic variants, interpreting multi-omics data, and integrating drug screen analyses. We present AstraKit, a unified KNIME workflow suite enabling end-to-end precision medicine analytics. AstraKit introduces three transformative innovations: 1) Dynamic variant interpretation with customizable annotation and filtering for disease-specific genomic contexts; 2) Multi-layered omics analyses integrating genomic, transcriptomic, and epigenetic data; and 3) Translational drug matching that correlates in vitro drug screens with clinical outcomes. Validated across oncology cohorts, AstraKit demonstrates concordance between experimental drug sensitivity and clinical trial responses, resolving discordances to uncover resistance mechanisms. By unifying variant analysis, multi-omics, and drug response modeling on a single customizable platform, AstraKit eliminates siloed workflows, accelerating biomarker validation and enabling clinicians to directly link molecular profiles to therapeutic decisions. As all AstraKit workflows are open-source and platform-independent, we provide a versatile comprehensive software suite for a multitude of tasks in bioinformatics and precision medicine. The KNIME workflows are available at KNIME Hub https://hub.knime.com/bioinf_goe/spaces/Public/AstraKit~lfVsGBY2HnPYc1h1/. The source code is available at https://gitlab.gwdg.de/MedBioinf/mtb/astrakit.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2026.01.25.701622v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Jan 27, 2026</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Getting over ANOVA: Estimation graphics for multi-group comparisons</h2>
<p class="paper-authors">Lu, Z., Anns, J., +10 authors, Claridge-Chang, A.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Data analysis in experimental science mainly relies on null-hypothesis significance testing, despite its well-known limitations. A powerful alternative is estimation statistics, which focuses on effect-size quantification. However, current estimation tools struggle with the complex, multi-group comparisons common in biological research. Here we introduce DABEST 2.0, an estimation framework for complex experimental designs, including shared-control, repeated-measures, two-way factorial experiments, and meta-analysis of replicates.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2026.01.26.701654v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, cancer biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, cancer biology</span>
<span class="date">Jan 27, 2026</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">SS18-SSX co-opts P300 to sustain oncogenic transcription independent of SWI/SNF activity</h2>
<p class="paper-authors">Doherty, A., Lagan, E., +10 authors, Brien, G.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Synovial sarcoma is driven by the SS18-SSX fusion oncoprotein, which has been assumed to promote tumorigenesis through its incorporation into the SWI/SNF chromatin remodeling complexes. Accordingly, therapeutic efforts have focused on targeting SS18-SSX containing SWI/SNF assemblies, yet these approaches have produced limited clinical benefit. Here, we demonstrate that SS18-SSX sustains oncogenic transcription independent of SWI/SNF activity. Despite efficient degradation and dismantling of SWI/SNF complexes, fusion occupancy at target loci and associated gene expression programs remain largely intact. Instead, we identify the acetyltransferase P300 as an essential co-factor supporting SS18-SSX chromatin binding and transcriptional activation. Targeting P300 displaces the fusion from chromatin, suppresses its transcriptional output, compromising synovial sarcoma viability. Notably, dual PROTAC mediated degradation of P300 and SWI/SNF produces strong synergistic effects, broadly disrupting SS18-SSX localization and function. These findings redefine the mechanistic basis of synovial sarcoma and reveal a mechanistically anchored therapeutic strategy for targeting its core oncogenic driver.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2026.01.26.701706v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, cancer biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, cancer biology</span>
<span class="date">Jan 27, 2026</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Sanguinarine activates ATM/ATR-mediated CHK-1 signaling to drive p53-dependent apoptosis in the C. elegans germline</h2>
<p class="paper-authors">El Ghali, R., Izadi, M., +3 authors, Pourkarimi, E.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Sanguinarine (SNG) is a natural component belonging to the benzophenanthridine alkaloids. Various studies have reported its therapeutic potential in treating chronic human diseases such as cancer. SNG is reported to cause programmed cell death in various cancer cell-lines. The mechanism by which SNG triggers apoptosis remains poorly elucidated, especially in vivo. Previous studies reported that sanguinarine induces apoptosis by increasing reactive oxygen species (ROS). In this study, we aimed to characterize the effects of SNG using an in vivo Caenorhabditis elegans (C. elegans) model. Treating C.elegans with various SNG concentrations resulted in apoptotic cell death in the proliferative germline. Interestingly, SNG-induced apoptosis depends on the core apoptotic machinery initiated by the DNA-damage-induced activity of the p53/CEP-1 protein. We have also demonstrated that the increase in germ cell apoptosis is caused by elevated levels of ROS following SNG treatment. Notably, the apoptotic phenotype induced by SNG was resolved upon treatment with the ROS scavenger. Altogether, our study demonstrates that SNG increases ROS, leading to activation of DNA damage-induced apoptosis in the proliferative germline of C. elegans.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2026.01.26.701699v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="machine learning, cancer biology">
<div class="paper-meta">
<span class="keyword">machine learning, cancer biology</span>
<span class="date">Nov 20, 2025</span>
<span class="journal">PubMed</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Integrated multi-omics analyses identified the H3K18la-based spatiotemporal characteristics and risk-stratified treatment strategy in lung adenocarcinoma.</h2>
<p class="paper-authors">Ao Shen, Sufei Zheng, +4 authors, Jie He</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Lung adenocarcinoma (LUAD) is the most common histological type of lung cancer, characterized by high mortality, recurrence, and metastasis. Despite advancements in therapies such as surgery, targeted treatment and immunotherapy, therapeutic resistance and immune evasion remain significant challenges. In our study, we integrated multi-omics data, including spatial transcriptomics, single-cell RNA sequencing, H3K18la ChIP-seq, CRISPR data and bulk transcriptomics, to explore the metabolic heterogeneity of LUAD, particularly focusing on glycolysis and histone H3K18 lactylation (H3K18la). Our findings revealed significant intra- and inter-tumoral metabolic heterogeneities, with glycolysis and H3K18la-related genes being more active in tumor regions. We also identified H3K18la-related gene activities as a marker of LUAD progression, demonstrating its strong correlation with glycolysis and tumor cell phenotypes. Based on these insights, we developed a machine learning-based prognostic model (termed as "Kla.Sig") that predicts patient survival and immunotherapy response, with validation across multiple cohorts. The model highlighted the immunosuppressive tumor microenvironment in high-risk score patients, with lower immune cell infiltration and higher immune evasion ability. In addition, we developed an online R shiny application "LUAD-Kla.Sig" to facilitate users' estimation of survival based on Kla.Sig model. In-silico drug screening suggests that targeting Polo-like kinase 1 (PLK1) with BI-2536 could be an effective strategy for high-risk LUAD patients. This study offers a deeper understanding of LUAD metabolism and immune evasion at single-cell and spatial resolution, proposing potential therapeutic targets and a risk-stratified treatment strategy for precision medicine.</p>
<a class="btn dark" href="https://pubmed.ncbi.nlm.nih.gov/41274397/" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div></section>
</main>
<script>
    // === EXPAND / COLLAPSE PAPERS ===
    document.querySelectorAll('.paper').forEach(paper => {
      paper.addEventListener('click', e => {
        // Ignora cliques em links
        if (e.target.closest('a')) return;

        // Ignora clique se houver seleção de texto
        const selection = window.getSelection();
        if (selection && selection.toString().length > 0) return;

        paper.classList.toggle('expanded');
      });
    });

    // === FILTER SYSTEM ===
    const papers = document.querySelectorAll('.paper');
    const filterBar = document.getElementById('filterBar');

    // Extrai keywords
    const allKeywords = new Set();
    papers.forEach(paper => {
      paper.dataset.keywords.split(',').forEach(k => allKeywords.add(k.trim()));
    });

    // Cria botões dinamicamente
    const filters = ['all', ...Array.from(allKeywords)];
    filters.forEach(keyword => {
      const btn = document.createElement('button');
      btn.className = 'filter-btn';
      btn.textContent = keyword;
      if (keyword === 'all') btn.classList.add('active');
      filterBar.appendChild(btn);
    });

    // Filtro de papers
    filterBar.addEventListener('click', e => {
      if (!e.target.classList.contains('filter-btn')) return;

      document.querySelectorAll('.filter-btn').forEach(b => b.classList.remove('active'));
      e.target.classList.add('active');

      const keyword = e.target.textContent;
      papers.forEach(paper => {
        if (keyword === 'all') {
          paper.style.display = 'grid';
        } else {
          const hasKeyword = paper.dataset.keywords.toLowerCase().includes(keyword.toLowerCase());
          paper.style.display = hasKeyword ? 'grid' : 'none';
        }
      });
    });
  </script>
</body>
</html>
