<!DOCTYPE html>

<html lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width, initial-scale=1.0" name="viewport"/>
<title>paper-trackr newsletter</title>
<link href="../docs/photo_51.ico" rel="shortcut icon"/>
<link href="../style.css" rel="stylesheet"/>
<link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&amp;display=swap" rel="stylesheet"/>
</head>
<body>
<aside class="sidebar">
<div class="sidebar-inner">
<a class="logo" href="../index.html">felipevzps</a>
<nav class="nav">
<a href="../index.html#about">About</a>
<a href="../index.html#research">Research</a>
<a href="../index.html#hackathon">Hackathon</a>
<a href="../index.html#contact">Contact</a>
</nav>
</div>
</aside>
<main class="newsletter-container">
<section class="newsletter-intro">
<h1>Latest in Science</h1>
<p>
        Hello, welcome to your daily dose of research!<br/> 
        Here you'll find some papers published yesterday, filtered for the topics I follow.
      </p>
<p class="note">
<small>
          * The curation is done automatically by my open-source project, <a href="https://github.com/felipevzps/paper-trackr" target="_blank">paper-trackr</a>.<br/>
          Want your own results? Install it via PyPI and get personalized updates!
        </small>
</p>
</section>
<!-- Filtro de keywords -->
<div class="filter-bar" id="filterBar"></div>
<section class="papers-list" id="papersList"><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 07, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Rethinking Molecular Beauty in the Deep Learning Era</h2>
<p class="paper-authors">Daina, A., Zoete, V.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> As (deep) generative chemistry rapidly enters the landscape of drug discovery, evaluating the models and their structural output relevance, tractability, and innovation potential becomes both a scientific and practical challenge. Existing metrics such as QED and SA Score, though widely adopted, are rooted in biased historical datasets and often fail to capture what medicinal chemists truly seek: compounds with meaningful pharmacological potential and realistic tractability. In this work, we introduce the OiiSTER-map - a simple, intuitive, and interpretable two-dimensional heatmap that classifies molecules based on bioactivity-informed usuality and structural elaboration derived from molecular fingerprints. Unlike traditional filters, the OiiSTER-map helps identify not only the drug discovery chemical 'sweet spot'. (Regular/Balanced), but also underappreciated territories such as Minimal/Unusual or Over-elaborated/Trivial regions, offering actionable insights into compound quality, relevance, and diversity. We hope that a bioactivity-informed, structurally aware, and easily interpretable tool like the OiiSTER-map, employed in combination with other well implemented metrics, can be decisive to go beyond current limitations of assessing (deep) generative models and to ensure a more mechanistically relevant, nuanced and useful evaluation of the thus designed virtual molecules.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.03.692079v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 07, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">GatorSC: Multi-Scale Cell and Gene Graphs with Mixture-of-Experts Fusion for Single-Cell Transcriptomics</h2>
<p class="paper-authors">Liu, Y., Zhang, Z., +6 authors, Bian, J.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Single-cell RNA sequencing (scRNA-seq) enables high-resolution characterization of cellular heterogeneity, but its rich, complementary structure across cells and genes remains underexploited, especially in the presence of technical noise and sparsity. Effectively leveraging this multi-scale structure is essentially an information fusion problem that requires integrating heterogeneous graph-based views of cells and genes into robust low-dimensional representations. In this paper, we introduce GatorSC, a unified representation learning framework that models scRNA-seq data through multi-scale cell and gene graphs and fuses them with a Mixture-of-Experts architecture. GatorSC constructs a global cell-cell graph, a global gene-gene graph, and a local gene-gene graph derived from neighborhood-specific subgraphs, and learns graph neural network embeddings that are adaptively fused by a gating network. To learn noise-robust and structure-preserving embeddings without labels, we couple graph reconstruction and graph contrastive learning in a unified self-supervised objective applied to both cell- and gene-level graphs. We evaluate GatorSC on 19 publicly available scRNA-seq datasets covering diverse tissues, species, and sequencing platforms. Across 14 benchmark datasets, GatorSC consistently outperforms state-of-the-art deep generative, graph-based, and contrastive methods for cell clustering, gene expression imputation, and cell-type annotation. The learned embeddings are used for accurate trajectory inference, recovery of canonical marker gene programs, and cell-type-specific pathway signatures in an Alzheimer's disease single-nucleus dataset. GatorSC provides a flexible foundation for comprehensive single-cell transcriptomic analysis and can be readily extended to multi-omic and spatial modalities.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.03.691688v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 07, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Unified Probabilistic Analysis of CyTOF: A Deep Generative Approach using CytoOne</h2>
<p class="paper-authors">Yang, Y., Wang, K., +3 authors, Wang, X.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Extracting meaningful biological signals from Cytometry by time-of-flight (CyTOF) data remains challenging due to heterogeneity, data characteristics, and the presence of various technical artifacts. Current analysis workflows typically rely on task-specific tools assembled into pipelines, which often make inconsistent distributional assumptions and fail to fully leverage the structure of the data. We present CytoOne, a unified probabilistic framework tailored for CyTOF data that integrates batch correction, differential analysis, and visualization within a single model. CytoOne is built upon a Bayesian hierarchical architecture inspired by Nouveau Variational Autoencoders (NVAE) and employs a novel quasi zero-inflated softplus-normal (QZIPN) likelihood to flexibly model the sparse and noisy nature of CyTOF measurements. We demonstrate via qualitative and quantitative evaluations that CytoOne effectively approximates both marginal and joint distributions of CyTOF data, removes batch-specific artifacts, enables fine-grained differential expression analysis, and facilitates interpretable embeddings for exploratory analysis.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.03.692122v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Dec 07, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Generative design and construction of functional plasmids with a DNA language model</h2>
<p class="paper-authors">Cunningham, A. G., Dekker, L., +1 author, Barnes, C. P.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> DNA language models offer a new paradigm for sequence design, yet their ability to generate functional genomic sequences remains underexplored. Plasmids act as a good testbed for evaluating DNA language model generation potential due to their simplicity and ease of construction. Here, we develop an end-to-end pipeline for generative design of Escherichia coli plasmid backbones, from large-scale data curation through fine-tuning, sampling, bioinformatic assessment, and candidate selection. A curated plasmid library was assembled from PlasmidScope and Addgene, and PlasmidGPT, a GPT-2-style DNA model, was fine-tuned on these corpora using circular-aware batching and random crops. Generations (1,000 per model) were produced under two prompting strategies: a minimal ATG seed to expose default tendencies, and a GFP cassette to enforce functional context. From 1000 generated synthetic plasmids, 16 candidates survived strict filtering and these were prioritised for wet-lab validation. Three shortlisted plasmids were synthesised and found to be functional, supporting growth, antibiotic resistance, and GFP expression in E. coli. These represent, to our knowledge, the first full AI-generated plasmids to be synthesised and validated in vivo. This work demonstrates that curated fine-tuning and prompt-aware generation enable DNA language models to progress from raw sequence sampling to experimentally testable plasmid designs. The approach offers a foundation for extending DNA design optimisation beyond E. coli, toward broader applications across engineering biology.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.06.692736v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, cancer biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, cancer biology</span>
<span class="date">Dec 07, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">PDLIM2 in Lung Adenocarcinoma Metastasis</h2>
<p class="paper-authors">Gao, F., Xiao, Y., +3 authors, Xiao, G.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Human and mouse studies have established the unique PDZ-LIM domain-containing protein PDLIM2 as a common tumor suppressor that is especially vital for suppressing the tumorigenesis and therapeutic resistance of lung cancer, the leading cause of cancer-related deaths among both men and women. However, the role of PDLIM2 in tumor metastasis, the predominant cause of cancer morbidity and mortality, is yet to be determined. Here, we report that PDLIM2 repression was positively associated with the metastasis of human lung adenocarcinoma, the major type of non-small cell lung cancer that accounts for more than 40% of all cases of human lung cancer. Interestingly, PDLIM2 repression was also correlated with oncogenic KRAS and/or TP53 mutations, two common drivers of human lung adenocarcinoma that often co-occur. In mice, in comparison to concurrently inducing mutant KRAS expression and TP53 deletion, additional co-ablation of PDLIM2 significantly increased the number and size of lung adenocarcinomas in the lung, and more importantly, the distant metastasis of lung tumor cells. The increased metastasis was accompanied by decreased anti-tumor immunity and increased pro-tumor inflammation. These data demonstrate the role of PDLIM2 in suppressing lung adenocarcinoma metastasis, thereby improving our understanding of this crucial tumor suppressor and lung cancer. They also provide a useful model for studying metastasis and testing new lung cancer treatments in vivo.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.03.692242v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, cancer biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, cancer biology</span>
<span class="date">Dec 07, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">IGFBPs define distinct pro-tumorigenic CAF subtypes in lung cancer tumour-microenvironment</h2>
<p class="paper-authors">Yengkhom, G. D., Nandgaonkar, K. K., +7 authors, Arandkar, S.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Cancer-associated fibroblasts (CAFs) play a significant role in influencing tumour outcomes; however, their heterogeneity and context-sensitive nature present a therapeutically challenging target. Here, we identify a group of Insulin-like growth factor binding proteins (IGFBPs), specifically IGFBP5, 6 and 7, as key regulators of CAF heterogeneity in lung cancer. Notably, these binding proteins are highly expressed in lung CAFs and independently modulate CAF phenotype. We demonstrate that tumour-derived cues, particularly TGF{beta}1 and IL6, affect IGFBP expression in CAFs. Transcriptome studies indicate that selectively knocking down these proteins directs CAFs towards different CAF subtypes. Functionally, CAFs with high levels of IGFBP5, 6, or 7 promote tumour growth in both ex vivo and in vivo settings and, critically, confer cisplatin resistance to tumour cells. Furthermore, the gene signatures associated with these proteins correlate with unfavourable prognosis. In conclusion, our study highlights the phenotypic plasticity of CAFs driven by these three distinct IGFBPs from the same family, consequently influencing the tumour microenvironment landscape.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.03.692056v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, cancer biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, cancer biology</span>
<span class="date">Dec 07, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">RBM15 promotes lung adenocarcinoma progression and palbociclib sensitivity via m6A-mediated STIL activation and downstream cyclin D1/CDK4 signaling</h2>
<p class="paper-authors">Chen, Y., Zhang, J., +8 authors, Wang, J.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> RNA-binding motif protein 15 (RBM15), an N6-methyladenosine (m6A) methyltransferase, plays a crucial role in the progression of lung adenocarcinoma (LUAD); however, the underlying mechanisms remain insufficiently defined. In this study, we investigated the contribution of RBM15-mediated m6A modification to LUAD pathogenesis. Our data showed that RBM15 was highly expressed in LUAD tissues and cell lines. Notably, its expression levels were positively correlated with tumor TNM staging and negatively correlated with differentiation status. The overexpression of RBM15 enhanced cell proliferation, migration, invasion, and cell cycle progression, while simultaneously inhibiting apoptosis. Conversely, RBM15 knockdown reversed these phenotypes and induced G2/M-phase arrest. In vivo experiments further confirmed that RBM15 knockdown led to the suppression of tumor growth. RNA sequencing, MeRIP sequencing, and bioinformatics analyses identified STIL as a gene directly regulated by RBM15 through m6A modification, a finding subsequently validated by luciferase reporter assays. Western blot and rescue experiments demonstrated that RBM15 and STIL cooperatively activate the cyclin D1/CDK4 signaling pathway, thereby promoting LUAD progression. However, this effect was markedly attenuated by CDK4/6 inhibitor application. Collectively, these findings indicated that the RBM15-STIL-cyclin D1/CDK4 axis represents a promising therapeutic target for LUAD.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.64898/2025.12.04.692261v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="machine learning, cancer biology">
<div class="paper-meta">
<span class="keyword">machine learning, cancer biology</span>
<span class="date">Dec 07, 2025</span>
<span class="journal">PubMed</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Integrative machine learning identifies lactylation-related gene signature prognostic for chemotherapeutic efficacy in colorectal carcinoma.</h2>
<p class="paper-authors">Hui Hu, Fang Peng, +2 authors, Yanjun Lu</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Lactylation, a recently discovered post-translational modification, has emerged as a critical regulator in cancer biology. Although chemotherapy remains the first-line treatment for metastatic colorectal cancer (CRC), only a subset of patients responds to it. This study aimed to identify key lactylation-related genes in CRC and evaluate their potential as predictive biomarkers for chemotherapy response.</p>
<a class="btn dark" href="https://pubmed.ncbi.nlm.nih.gov/41353668/" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div></section>
</main>
<script>
    // === EXPAND / COLLAPSE PAPERS ===
    document.querySelectorAll('.paper').forEach(paper => {
      paper.addEventListener('click', e => {
        if (e.target.tagName.toLowerCase() === 'a') return;
        paper.classList.toggle('expanded');
      });
    });

    // === FILTER SYSTEM ===
    const papers = document.querySelectorAll('.paper');
    const filterBar = document.getElementById('filterBar');

    // Extrai keywords
    const allKeywords = new Set();
    papers.forEach(paper => {
      paper.dataset.keywords.split(',').forEach(k => allKeywords.add(k.trim()));
    });

    // Cria botÃµes dinamicamente
    const filters = ['all', ...Array.from(allKeywords)];
    filters.forEach(keyword => {
      const btn = document.createElement('button');
      btn.className = 'filter-btn';
      btn.textContent = keyword;
      if (keyword === 'all') btn.classList.add('active');
      filterBar.appendChild(btn);
    });

    // Filtro de papers
    filterBar.addEventListener('click', e => {
      if (!e.target.classList.contains('filter-btn')) return;

      document.querySelectorAll('.filter-btn').forEach(b => b.classList.remove('active'));
      e.target.classList.add('active');

      const keyword = e.target.textContent;
      papers.forEach(paper => {
        if (keyword === 'all') {
          paper.style.display = 'grid';
        } else {
          const hasKeyword = paper.dataset.keywords.toLowerCase().includes(keyword.toLowerCase());
          paper.style.display = hasKeyword ? 'grid' : 'none';
        }
      });
    });
  </script>
</body>
</html>
