<!DOCTYPE html>

<html lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width, initial-scale=1.0" name="viewport"/>
<title>paper-trackr newsletter</title>
<link href="../docs/photo_51.ico" rel="shortcut icon"/>
<link href="../style.css" rel="stylesheet"/>
<link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&amp;display=swap" rel="stylesheet"/>
</head>
<body>
<aside class="sidebar">
<div class="sidebar-inner">
<a class="logo" href="../index.html">felipevzps</a>
<nav class="nav">
<a href="../index.html#about">About</a>
<a href="../index.html#research">Research</a>
<a href="../index.html#hackathon">Hackathon</a>
<a href="../index.html#contact">Contact</a>
</nav>
</div>
</aside>
<main class="newsletter-container">
<section class="newsletter-intro">
<h1>Latest in Science</h1>
<p>
        Hello, welcome to your daily dose of research!<br/> 
        Here you'll find the most recent papers published yesterday, filtered for the topics I follow.
      </p>
<p class="note">
<small>
          * The curation is done automatically by my open-source project, <a href="https://github.com/felipevzps/paper-trackr" target="_blank">paper-trackr</a>.<br/>
          Want your own results? Install it via PyPI and get personalized updates!
        </small>
</p>
</section>
<!-- Filtro de keywords -->
<div class="filter-bar" id="filterBar"></div>
<section class="papers-list" id="papersList"><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Nov 11, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Harnessing Plasmid Crosstalk in Transcription Factor Mediated Cell-Free Biosensors</h2>
<p class="paper-authors">Patterson, A. T., Piorino, F., +2 authors, Styczynski, M. P.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Plasmid crosstalk--unexpected changes to protein expression levels due to interactions between genetic cassettes in cell-free systems--complicates the creation of multi-plasmid cell-free tools. While the potential underlying mechanisms for crosstalk have been previously investigated, the practical impact of crosstalk on the implementation of cell-free genetic circuits has not been thoroughly examined. Here, we contextualize plasmid crosstalk in genetic circuits by examining its impact on the design and performance of multiple, diverse transcription factor biosensors. Guided by a deeper understanding, we harness crosstalk to enhance cell-free biosensor performance. Together, these findings demonstrate that plasmid crosstalk can serve as a tunable property in the development of cell-free genetic circuits.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.11.11.687754v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Nov 11, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Discovery and Functional Characterization of Salinity-Responsive Promoters in Pseudomonas putida KT2440</h2>
<p class="paper-authors">de Simone, C. M., de Siqueira, G. M. V., Guazzaroni, M.-E.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Microbial bioprocesses are associated with environmental advantages, but their freshwater demand remains a challenge. Seawater is abundant and a sustainable alternative; however, high salinity often limits microbial growth and survival. This study aimed to identify biological parts involved in microbial salt stress responses for applications in microbial engineering. Regulatory regions adjacent to genes related to salinity and other industrially relevant conditions in Pseudomonas putida KT2440 were cloned into a reporter plasmid for functional evaluation. We identified and characterized the promoters of PP_1833 and PP_4707 as salt-inducible in KT2440, showing fluorescence levels 1.9- and 2.8-fold greater than a standard strong constitutive promoter (Pj100) under 3.5% NaCl (w/v), respectively, while no activity was detected in Escherichia coli DH10B. These results provide a basis for developing salt-responsive regulatory circuits in P. putida KT2440.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.11.11.687801v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Nov 11, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">An Ehrlich-inspired retrobiosynthesis of pharmaceutical scaffolds</h2>
<p class="paper-authors">Rumpl, A. E. C., Kowalski, D., +5 authors, Pyne, M. E.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Brewer's yeast (Saccharomyces cerevisiae) acquires nitrogen from branched-chain and aromatic amino acids via the Ehrlich pathway, generating flavor (fusel) byproducts. Recently, diverting 4-hydroxyphenylacetaldehyde from Ehrlich catabolism of L-tyrosine has enabled microbial production of opioids and other plant benzylisoquinolines. Yet, fusel metabolism is versatile in substrate scope, offering an untapped entry point for synthesizing structurally diverse aldehydes. Here, we repurpose the yeast Ehrlich pathway into a modular biocatalytic conduit for manufacturing privileged pharmaceutical alkaloids. We utilize retrobiosynthetic analysis and enzyme screening to derive scaffolds representative of solifenacin, colchicine, and ephedrine pharmaceuticals from simple amino acids. We survey wild yeasts for catabolism of L-phenylglycine and demonstrate Ehrlich conversion to benzyl alcohol or (R)-phenylacetylcarbinol by 29 strains across nine genera. Implementing an {omega}-transaminase enables production of norephedrine from a simple amino acid input. This work unveils a generalizable biocatalytic route to clinically important alkaloids by exploiting metabolic logic from a yeast flavor pathway.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.11.10.687684v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Nov 11, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Automated Annotation of Plant Gene Regions Using Supervised Machine Learning</h2>
<p class="paper-authors">Simoes, B. M., Pinho, A. J., Pratas, D.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Rapid sequencing now enables analysis across many species, including plants, but repeat-rich, redundant plant genomes still hinder precise annotation. Precise gene calls are vital for understanding biology, comparative genomics, and discovering traits such as improved nutrition and disease resistance. We present GeAnno, a supervised machine learning method for plant gene detection. GeAnno uses an XGBoost classifier trained on curated plant annotations and a sliding-window scheme while capturing redundancy, base composition, and start/stop-codon spacing, among others. At inference, windows are scored as genic or intergenic and lightly smoothed. The output is a standard GFF3 with strand-specific genic regions. Benchmarking against ab initio predictors under matched training on 11 cassava (Manihot esculenta) genomes and cross-species evaluation, GeAnno achieved higher nucleotide-level precision and F1-score. For example, on cassava it reached 77.13% precision and 72.90% F1-score. Moreover, performance transfers to divergent species and parameters (window, step, smoothing, thresholds) are tunable. By improving accuracy and portability on complex plant genomes, GeAnno supports downstream functional studies and breeding, advancing food security and ecological sustainability.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.11.10.687603v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Nov 11, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Large Language Models for Accessible Reporting of Bioinformatics Analyses in Interdisciplinary Contexts</h2>
<p class="paper-authors">Yu, L., Kim, D., +11 authors, Yang, J. Y. H.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Health and life scientists routinely collaborate with quantitative scientists for data analysis and interpretation, yet miscommunication often obscures the interpretation of complex results. Large Language Models (LLMs) offer a promising way to bridge this gap, but their cross-discipline interpretative skill remains limited on real-word bioinformatics analyses. We therefore benchmarked four state-of-the-art LLMs: GPT-4o, o1, Claude 3.7 Sonnet, and Gemini 2.0 Flash, using automated and human evaluation frameworks to ensure holistic evaluation. Automated assessment employed multiple choice questions designed using Bloom's taxonomy to assess multiple levels of understanding, while human evaluation tasked scientists to score summaries for factual consistency, lack of harmfulness, comprehensiveness, and coherence. All generally produced readable and largely safe summaries, confirming their value for first-pass translation of technical analyses, however frequently misinterpreted visualisations, produced verbose summaries and rarely offered novel insights beyond what was already contained in the analytics. Our findings suggest that LLMs are best suited for easing interdisciplinary communication rather than replacing domain expertise and human oversight remains essential to guarantee accuracy, interpretative depth, and the generation of genuinely novel scientific insights.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.11.09.687479v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Nov 11, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Simulating Tandem Mass Spectra for Small Molecules using a General-Purpose Large-Language Model</h2>
<p class="paper-authors">Nguyen, T., Barupal, D.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> We show a practical application of the Google Gemini large-language-model for simulating tandem mass spectra for compounds from the Blood Exposome Database. This approach bypasses the need for domain-specific model training, suggesting that the chemical fragmentation knowledge could be latently encoded within the Gemini model. General-purpose LLMs represent a useful and accessible tool for expanding in-silico spectral libraries and may accelerate the compound annotation in mass spectrometry-based metabolomics and exposomics.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.11.10.687298v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Nov 11, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">LSTM-Attention-Guided Graph Neural Networks for Integrated Genotype-Environment Modeling in Maize Yield Prediction</h2>
<p class="paper-authors">Morshedian, A., Domaratzki, M.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> We present a deep-learning framework that combines an LSTM, a graph neural network (GNN), and transformer-style attention to model genotype-environment (GxE) effects for maize yield prediction. Weather data for a growing season is summarized using LSTM and encoded into a 21-dimensional embedding that is used as the environment node feature; 437,214 SNPs are summarized into 548 principal components that instantiate genotype nodes. Multi-head attention dynamically weights the edges during message passing. We compare three architectures: A (fully bipartite graph), B (A with intra-set top-k similarity within genotype and within environment), and C (B with a single learnable supernode readout that attends over all nodes after message passing). The joint representations feed a compact MLP for yield prediction. Using a forward-time split (2014-2021 train; 2022 test with unseen genotypes and unseen environments), performance improves monotonically from A to C: A (RMSE 2.7749, PCC 0.4115, R2 0.1693), B (2.3683, 0.6622, 0.4385), C (2.2120, 0.6945, 0.4823). Compared to A, C has a reduction in RMSE by 0.5629 ({approx} 20.3%) and an increase in PCC by 0.283 ({approx} 68.8%), indicating that global, content-adaptive aggregation promotes local GxE propagation. Performance of our approach remains consistent regardless of the number of genotypes per environment and has strong performance under variable or unbalanced genotype sampling expression across environments.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.11.09.687529v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Nov 11, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Cellohood: multi-granular discovery of cellular neighborhoods with a permutation-invariant set transformer auto-encoder</h2>
<p class="paper-authors">Mozejko, M., Schulz, D., +11 authors, Szczurek, E.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Discovering cellular neighborhoods and their roles in disease requires computational methods that consider the full breadth of data and offer multi-level granularity and interpretability. Here, we introduce Cellohood, a permutation-invariant set transformer auto-encoder equipped with a clinical association pipeline. Cellohood encodes full readouts for bags of spatially co-localized cells and supports multi-level analyses, providing interpretability by mapping latent dimensions to spatial tissue features. Our model surpasses current methods in accuracy of cellular neighborhood detection for spatial transcriptomics of the human cortex and CODEX spleen data measuring lupus progression. Applied to cancer data across three granularity levels, our method recovers neighborhoods along the expected immune-cold to immune-hot spectrum and further refines them into biologically and clinically meaningful subclasses, revealing spatial patterns linked to prognosis, histology, stage, and tumor mutational burden, and uncovering subgroups that transcend standard classifications. Overall, Cellohood enables in-depth analysis of complex tissues, revealing clinically informative spatial neighborhoods.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.11.10.687184v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Nov 11, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Prediction of lncRNA-protein interacting pairs using LLM embeddings based on evolutionary information</h2>
<p class="paper-authors">Choudhury, S., Bajiya, N., Raghava, G. P. S.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Interactions of long non-coding RNAs (lncRNAs) with proteins is responsible for numerous cellular processes, including transcriptional regulation, chromatin remodeling, cell differentiation, and intracellular signaling. In the past, numerous computational methods have been developed for predicting lncRNA-protein interacting (LPI) pairs. This study describes a highly accurate and reliable method for predicting LPI pairs built on largest possible non-redundant dataset having 262,244 interacting and equal number of non-interacting pairs. Initially, similarity-based approach BLAST has been tried which have poor discriminative power, due to low sequence similarity. Subsequently, we developed CNN based models and machine learning based models using traditional features and embedding. Our CatBoost model developed using embedding generated by DNABERT-2 and ESM-2-t30 achieved AUC of 0.989 with MCC 0.915 on an independent dataset. Our method performs better than existing methods on an independent dataset. We developed standalone software and web server LncrnaPI for predicting LPI pairs, scanning lncRNA interacting proteins in proteome and protein interacting lncRNA in genomes (https://webs.iiitd.edu.in/raghava/lncrnapi/).</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.11.10.687101v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, synthetic biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, synthetic biology</span>
<span class="date">Nov 11, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">AstraBIND: Graph Attention Network for Predicting Ligand Binding Sites</h2>
<p class="paper-authors">Goteti, A., Bozkurt, C., Vasilyeva, A.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Predicting ligand binding sites is central to computational biology and drug discovery. Existing machine learning approaches either use protein sequence, structure, or both. While structure-based deep learning models typically outperform sequence-based methods, they often require high computational cost or ligand-specific data, forcing a trade-off between accuracy and scalability. We present AstraBIND, a lightweight graph neural network that bridges this gap by integrating protein sequence, structure (experimental or predicted), and homology information to predict ligand classes and binding residues within minutes. The model employs a GATv2 architecture with 0.9M parameters, trained on ~250,000 curated protein-ligand complexes across 16 ligand categories. By encoding residue-level features and spatial geometry through graph attention, AstraBIND identifies binding residues and ligand types while maintaining structural consistency. In benchmarking, AstraBIND achieved a weighted macro-F1 of 0.47 across all ligand classes, with top performance for nucleotides (F1 = 0.79), porphyrins (0.74), and cofactors (0.73). Case studies, including p53 and CRFR1, demonstrate robust pocket localization for diverse proteins. Combined with its minimal inference time and broad ligand coverage, AstraBIND enables rapid in-silico screening and integration into laboratory workflows. Together with other Astra ML models, it represents a step toward real-time protein design and validation pipelines. Astra models are available at https://www.orbion.life.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.11.10.687555v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, cancer biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, cancer biology</span>
<span class="date">Nov 11, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Oncogenic mutations convert MET from a pro-apoptotic tumor suppressor to an oncogenic driver</h2>
<p class="paper-authors">Tellier, R., Fernandes, M., +17 authors, Tulasne, D.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Dependence receptors can exert both oncogenic and tumor-suppressive activities. In cancers, downregulation of dependence receptors or overexpression of their ligands are well-established mechanisms that drive tumor progression. However, direct genetic alterations abolishing the pro-apoptotic function of dependence receptors have not been documented so far. MET, a receptor tyrosine kinase classically viewed as an oncogene, has also been proposed to act as a dependence receptor through its caspase-mediated cleavage, but whether this property impacts tumorigenesis remained unknown. In ~3% of lung adenocarcinomas, MET mutations leading to exon 14 skipping (METex14Del) remove both the caspase site and the adjacent CBL-binding motif, thereby preventing generation of the pro-apoptotic p40MET fragment. METex14Del promotes sustained signaling, enhanced invasion, apoptosis resistance, and tumor growth in HGF-humanized mice. Genome editing revealed that combined, but not individual, mutations of the caspase and CBL sites phenocopy METex14Del. Moreover, inducible re-expression of p40MET in METex14Del-expressing cells restored apoptosis and suppressed tumor formation. Altogether, our findings identify MET exon 14 skipping as the first oncogenic mutation that drives tumorigenesis by abolishing the tumor-suppressive pro-apoptotic function of a dependence receptor, thereby redefining the oncogenic potential of MET.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.11.10.687614v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, cancer biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, cancer biology</span>
<span class="date">Nov 11, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Colibactin-producing E. coli promote carcinogenesis of gastroesophageal adenocarcinoma and simultaneously induce autophagy and differentiation</h2>
<p class="paper-authors">Proano Vasco, A. I., Obwegs, D., +20 authors, Quante, M.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> </p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.11.10.687579v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="bioinformatics, cancer biology">
<div class="paper-meta">
<span class="keyword">bioinformatics, cancer biology</span>
<span class="date">Nov 11, 2025</span>
<span class="journal">bioRxiv</span>
</div>
<div class="paper-content">
<h2 class="paper-title">TEIP: A Compact, Open-Source Framework for Predicting Tumor Epitope Immunogenicity in Glioblastoma Cancer Using Deep Learning and Multi-Modal Biological Features</h2>
<p class="paper-authors">Shekhar, A.</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Identifying immunogenic tumor epitopes (neoantigens) is critical for cancer vaccine and T-cell therapy design. We propose **TEIP** (Tumor Epitope Immunogenicity Predictor), an integrative computational pipeline that combines sequence-based deep learning, HLA-binding affinity prediction, and immunogenomic filters to predict and prioritize T-cell epitopes. TEIP uses a multi-branch neural network to model peptide HLA presentation and T-cell recognition, augmented with features like predicted binding affinity and gene expression. On benchmark datasets, TEIP achieves significantly improved prediction performance (e.g., area-under-ROC &gt; 0.89) com- pared to baseline tools. We demonstrate that TEIPs prioritized neoantigens show broad patient coverage, high tumor-specific expression, and include challenging targets (e.g., cancer-testis and viral antigens) often missed by prior methods. These results highlight TEIPs potential to streamline neoantigen discovery for personalized immunotherapy.</p>
<a class="btn dark" href="https://www.biorxiv.org/content/10.1101/2025.11.09.687510v1?rss=1" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div><div class="paper" data-keywords="machine learning, cancer biology">
<div class="paper-meta">
<span class="keyword">machine learning, cancer biology</span>
<span class="date">Nov 11, 2025</span>
<span class="journal">PubMed</span>
</div>
<div class="paper-content">
<h2 class="paper-title">Koina: Democratizing machine learning for proteomics research.</h2>
<p class="paper-authors">Ludwig Lautenbacher, Kevin L Yang, +22 authors, Mathias Wilhelm</p>
<div class="paper-abstract">
<p><strong>Abstract:</strong> Recent developments in machine learning (ML) and deep learning have immense potential for applications in proteomics, such as generating spectral libraries, improving peptide identification, and optimizing targeted acquisition modes. Although new ML models are regularly published, the rate at which the community adopts these models is slow. This is in part due to a lack of findability and accessibility of these models as well as the technical challenges involved in incorporating these models into data analysis pipelines and demonstrating their reusability for end-users. Here we show Koina, an open-source decentralized and online-accessible model repository to facilitate publication of ML models. Koina enables ML model usage via an easy-to-use online interface, facilitating the integration of ML models in data analysis pipelines. Using the widely used FragPipe computational platform as an example, we demonstrate how Koina can be integrated with existing proteomics software tools and how these integrations improve data analysis.</p>
<a class="btn dark" href="https://pubmed.ncbi.nlm.nih.gov/41219230/" target="_blank">
                Read full paper
                <svg fill="none" style="width: 0.85rem; height: 0.85rem;" viewbox="0 0 11 12" width="11" xmlns="http://www.w3.org/2000/svg">
<path d="M1.70985 4.5H7.7804M7.7804 4.5V10.5705M7.7804 4.5L0.780396 11.5" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.3"></path>
</svg>
</a>
</div>
</div>
</div></section>
</main>
<script>
    // === EXPAND / COLLAPSE PAPERS ===
    document.querySelectorAll('.paper').forEach(paper => {
      paper.addEventListener('click', e => {
        if (e.target.tagName.toLowerCase() === 'a') return;
        paper.classList.toggle('expanded');
      });
    });

    // === FILTER SYSTEM ===
    const papers = document.querySelectorAll('.paper');
    const filterBar = document.getElementById('filterBar');

    // Extrai keywords
    const allKeywords = new Set();
    papers.forEach(paper => {
      paper.dataset.keywords.split(',').forEach(k => allKeywords.add(k.trim()));
    });

    // Cria botÃµes dinamicamente
    const filters = ['all', ...Array.from(allKeywords)];
    filters.forEach(keyword => {
      const btn = document.createElement('button');
      btn.className = 'filter-btn';
      btn.textContent = keyword;
      if (keyword === 'all') btn.classList.add('active');
      filterBar.appendChild(btn);
    });

    // Filtro de papers
    filterBar.addEventListener('click', e => {
      if (!e.target.classList.contains('filter-btn')) return;

      document.querySelectorAll('.filter-btn').forEach(b => b.classList.remove('active'));
      e.target.classList.add('active');

      const keyword = e.target.textContent;
      papers.forEach(paper => {
        if (keyword === 'all') {
          paper.style.display = 'grid';
        } else {
          const hasKeyword = paper.dataset.keywords.toLowerCase().includes(keyword.toLowerCase());
          paper.style.display = hasKeyword ? 'grid' : 'none';
        }
      });
    });
  </script>
</body>
</html>
