<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Wednesday, 02 July 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Machine learning reveals genes impacting oxidative stress resistance across yeasts.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Katarina Aranguiz, Linda C Horianopoulos, Logan Elkin, Kenia Segura Abá, Drew Jordahl, Katherine A Overmyer, Russell L Wrobel, Joshua J Coon, Shin-Han Shiu, Antonis Rokas, Chris Todd Hittinger</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify gene families whose sizes are predictive of ROS resistance and show that overexpression of the old yellow enzyme (OYE) reductase increases ROS resistance in Kluyveromyces lactis, while Saccharomyces cerevisiae mutants lacking multiple mannosyltransferase-enc</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Reactive oxygen species (ROS) are highly reactive molecules encountered by yeasts during routine metabolism and during interactions with other organisms, including host infection. Here, we characterize the variation in resistance to the ROS-inducing compound tert-butyl hydroperoxide across the ancient yeast subphylum Saccharomycotina and use machine learning (ML) to identify gene families whose sizes are predictive of ROS resistance. The most predictive features are enriched in gene families related to cell wall organization and include two reductase gene families. We estimate the quantitative contributions of features to each species' classification to guide experimental validation and show that overexpression of the old yellow enzyme (OYE) reductase increases ROS resistance in Kluyveromyces lactis, while Saccharomyces cerevisiae mutants lacking multiple mannosyltransferase-encoding genes are hypersensitive to ROS. Altogether, this work provides a framework for how ML can uncover genetic mechanisms underlying trait variation across diverse species and inform trait manipulation for clinical and biotechnological applications.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40592811/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identification of plasma proteins associated with seizures in epilepsy: A consensus machine learning approach.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Saman Hosseini Ashtiani, Sarah Akel, Rakesh Kumar Banote, Fredrik Asztely, Johan Zelano</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified ten consensus candidate protein biomarkers (CDH15, PAEP, LTBP3, PHOSPHO1, NEFL, SFRP1, TDGF1, DUSP3, WWP2 and DSG3) that contributed to the classification of patients as</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Blood-based biomarkers in epilepsy could constitute important research tools advancing neurobiological understanding and valuable clinical tools for better diagnosis and follow-up. An interesting question is whether biomarker patterns could contribute additional understanding compared to individual marker values. We analyzed OLINK proteomics data from a large epilepsy cohort in which we have previously found four differentially expressed proteins (CDH15, PAEP, LTBP3, PHOSPHO1). Using two machine-learning techniques, we identified ten consensus candidate protein biomarkers (CDH15, PAEP, LTBP3, PHOSPHO1, NEFL, SFRP1, TDGF1, DUSP3, WWP2 and DSG3) that contributed to the classification of patients as being seizure-free or not. Six out of the ten consensus proteins were identified as differentially expressed in our previous study (although NEFL and TDGF1 not significantly so after multiple testing correction). The remaining four consensus proteins were newly identified by machine learning and were chosen for detailed analysis. In comparison to the four significantly differentially expressed proteins (CDH15, PAEP, LTBP3, PHOSPHO1), the newly identified consensus proteins (SFRP1, DSG3, DUSP3, and WWP2) and in particular a combination of all eight proteins, outperformed individual proteins in identifying individuals with recent seizures, highlighting the potential of multi-protein profiles. These findings emphasize the need for integrative bioinformatic approaches in epilepsy research and underscore the role of neuroinflammation and immune pathways in epileptogenesis. Our results support the applicability of plasma protein profiling for developing future blood-based tests for epilepsy seizure prediction, diagnosis, and treatment. Further validations in independent cohorts are required to establish these candidate biomarkers in clinical practice.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40591616/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Rhaponticin inhibits the proliferation, migration, and invasion of head and neck squamous cell carcinoma (HNSCC) cells through modulation of the IL6/STAT3 signaling pathway.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hongcheng Wei, Shanshan Wang, Jiayue Wan, Sicheng Li, Wei Wang, Jiajun Zhu, Lin Jiang, Yisen Shao, Yuan Wu</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> This study aims to investigate the specific molecular mechanisms by which Rhaponticin inhibits the invasion and metastasis of HNSCC cells.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Rhaponticin, a bioactive compound derived from rhubarb, has been demonstrated anti-tumor effects in various types of cancer. However, its impact on HNSCC remains unexplored. This study aims to investigate the specific molecular mechanisms by which Rhaponticin inhibits the invasion and metastasis of HNSCC cells.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40591033/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Spatial domain detection using contrastive self-supervised learning for spatial multi-omics technologies.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jianing Yao, Jinglun Yu, Brian Caffo, Stephanie C Page, Keri Martinowich, Stephanie C Hicks</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Proust, a scalable tool to predict discrete domains using spatial multi-omics data by combining the low-dimensional representation of biological profiles based on graph-based contrastive self-supervised learning with the ability to integrate multiple data modalities.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent advances in spatially resolved single-omic and multi-omics technologies have led to the emergence of computational tools to detect and predict spatial domains. Additionally, histological images and immunofluorescence (IF) staining of proteins and cell types provide multiple perspectives and a more complete understanding of tissue architecture. Here, we introduce Proust, a scalable tool to predict discrete domains using spatial multi-omics data by combining the low-dimensional representation of biological profiles based on graph-based contrastive self-supervised learning. Our scalable method integrates multiple data modalities, such as RNA, protein, and H&E images, and predicts spatial domains within tissue samples. Through the integration of multiple modalities, Proust consistently demonstrates enhanced accuracy in detecting spatial domains, as evidenced across various benchmark data sets and technological platforms.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40393810/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">StruCloze: A Unified Framework for Backmapping and Inpainting of Biomolecules</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhu, J., Fan, Z., Li, Z., Zheng, Z., Lindorff-Larsen, K., Chen, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present StruCloze, a deep learning framework for reconstructing atomistic structures from coarse-grained CG models and inpainting missing regions in structures, offering a practical tool for structural refinement and integrative modeling.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Atomistic resolution is essential for understanding biomolecular structure and function, yet coarse-grained (CG) models remain indispensable for simulating large and dynamic systems. Reconstructing accurate all-atom structures from CG representations, particularly across varied CG schemes and biomolecular types, remains a fundamental challenge. Moreover, flexible or disordered regions may well suffer from failure in structure modeling, making inpainting missing regions another challenging task. Here, we present StruCloze, a deep learning framework for reconstructing atomistic structures from CG models and inpainting missing regions for both proteins and nucleic acids. StruCloze generalizes across various CG levels and biomolecule types on single pretraining, with minimal fine-tuning required for optimal performance on specific representations. It achieves state-of-the-art accuracy in reconstructing both protein and nucleic acid structures, demonstrates superior transferability and speed compared to existing methods. Leveraging masked learning strategy, StruCloze also excels at inpainting structurally missing regions in structures, offering a practical tool for structural refinement and integrative modeling. Our framework provides a general solution for bridging reduced or incomplete representations with full atomistic detail of biomolecular structures, enabling rapid local structure prediction and further analysis on system dynamics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.26.661889v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AdSI-MIMO: Adaptive stain imputation with multi-input and multi-output learning for multiplex immunofluorescence imaging</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Li, X., Rana, P., Gide, T. N., Adegoke, N. A., Mao, Y., Berkovsky, S., Coiera, E., Wilmott, J. S., Liu, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose AdSI-MIMO, a novel deep learning-based stain imputation framework for multiplex immunofluorescence imaging, which achieves substantial improvements over state-of-the-art methods, particularly in the imputation of key T-cell and activation biomarkers, such as</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Multiplex immunofluorescence (mIF) imaging plays a crucial role in studying multiple biomarkers and their interactions within the tumour microenvironment. However, acquiring mIF images that include all desired biomarkers presents significant challenges due to the need for specialised equipment and costly reagents, which increase technical complexity, time and expense. Stain imputation offers a promising solution by synthesising target biomarker images using generative models, thereby eliminating the need for additional staining procedures. Existing deep learning-based stain imputation methods lack flexibility in generating multiple biomarker images from various input combinations. To overcome this limitation, we propose AdSI-MIMO, a novel stain imputation framework for mIF images. Our method features a multi-branch deep learning architecture capable of generating multiple outputs and incorporates an adaptive progressive masking strategy to accommodate the varying combinations of input biomarkers. This approach not only improves the quality of the generated biomarker images, but also eliminates the need to train separate models for each target biomarker. We evaluated AdSI-MIMO using two datasets, including a local dataset comprising mIF images from 257 melanoma patients and a public dataset of 55 urothelial carcinoma samples. Our method achieved substantial improvements over state-of-the-art methods, particularly in the imputation of key T-cell and activation biomarkers, such as CD8 and PD-L1. Specifically, across both public and proprietary datasets, our model achieved a 18.4% improvement in Pearson Correlation Coefficient (Pearson-r) for CD8 imputation and a 48.1% improvement for PD-L1 imputation under various input biomarker configurations on the external test set.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.27.661891v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Novel binning-based methods for model fitting and data splitting improved machine learning imbalanced data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Abdulnabi, H., Westwood, J. T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Akin split is a novel method proposed that objectively yields the most appropriate data split(s) and Epoch sampling is used to improve model fitting with imbalanced data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Machine Learning (ML) models may perform inconsistently on individual classes on nominal outputs or ranges on continuous outputs, collectively referred to here as bins. Models should be assessed through metrics that consider each bin individually, called bin metrics. Inconsistent model performance is often due to model fitting with imbalanced data. Towards improving modelling of imbalanced data, novel model fitting methods are proposed including using bin metrics as loss functions and the use of Epoch sampling. Imbalanced data also poses a challenge for appropriate data splitting. Akin split is a novel method proposed that objectively yields the most appropriate data split(s). Existing and novel model fitting and data splitting methods were assessed in two case studies. The first case study used synthetically generated datasets with different levels of noise and imbalance. On datasets with noise and greater levels of imbalance, Epoch sampling significantly improved the model performance by up to 23.6% while significantly using less resources (computation and time) by up to 57.7% compared to a standard model fitting method. The second case study used protein-genome interactions data that are often severely right-skewed. Akin split was used to split the data more appropriately than traditional methods. Model fitting methods were tried on two model configurations. The effects of the model fitting methods varied by the model configuration, but all models were significantly improved by up to 35.3% compared to the standard model fitting.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.26.661884v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Machine learning differentiates between bulk and pseudo-bulk RNA-seq datasets</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Low, B. H., Rashid, M. M., Selvarajoo, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Paired bulk and scRNA-seq datasets for bioinformatics and synthetic data generation .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Modern synthetic data generators and deconvolution methods rely heavily on single-cell (sc) RNA-seq data. Aggregated scRNA-seq (pseudo-bulk) is commonly assumed to closely match true bulk RNA-seq, making it a dependable benchmark for developing and evaluating new bioinformatics methods. Here, we investigated paired bulk and scRNA-seq datasets using machine learning techniques to assess the fidelity of pseudo-bulk profiles. Our results demonstrate that pseudo-bulks differ substantially from bulk RNA-seq in both analytic metrics and biological processes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.27.661895v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Whole-genome sequencing in Galicia reveals male-biased pre-Islamic North African ancestry, subtle population structure, and micro-geographic patterns of disease risk</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pardo-Seco, J., Camino-Mera, A., Bello, X., Gomez-Carballa, A., Castelo-Martinez, L., Martinez-Cadenas, C., Martinon-Torres, F., Salas, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify a significant North African/Middle Eastern autosomal ancestry component in the Galician population, suggesting a southern entry route to Iberia from North Africa and a male-biased influx of North African ancestry.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Galicia, located at the westernmost edge of Europe, has been reported to exhibit distinctive genetic traits compared to other Iberian populations. We present the first whole-genome sequencing (WGS) study of a Galician population (GALOMICS [GAL]; n = 91, 17.2 million variants; https://galomica.genpob.eu/), assessing its genetic variability in comparison with WGS data from other Spanish regions and continental populations (n = 1,078). Contrary to recent claims of extreme genetic stratification, the population structure of Galicia aligns with broader Iberian patterns, with one dominant cluster homogeneously distributed and four minor, more localized clusters. Genome-wide analyses of the Spanish National DNA Bank dataset (NDNAB; n = 453) support these findings, identifying only three Galician clusters, again with one overwhelmingly predominant. Phylogenetic analyses challenge earlier interpretations that placed Galicians at the deepest Iberian genetic node; instead, Galician clusters form terminal branches, suggesting recent diversification. Analysis of runs of homozygosity indicates slightly higher inbreeding compared to other European populations, primarily driven by the "Porto do Son" cluster, which raises the regional average. We identified a significant North African/Middle Eastern autosomal ancestry component (13.5%-16.5%), despite its distance from historically Arab-influenced regions. Genomic analyses point to an admixture event ca. 620-670 CE that introduced North African/Middle Eastern ancestry into a largely European gene pool. The signal, likely stemming from trans-Mediterranean contacts predating the 711 CE Islamic incursion and well before the Reconquista, shows a subtle South-to-North decline, suggesting a southern entry route. Y-chromosome (21.2%) and mitochondrial DNA (1.1%) analyses indicate a male-biased influx, pointing to a predominantly paternal contribution. This observation calls for a reevaluation of the commonly held assumption that Islamic rule alone accounts for North African ancestry in Iberia. Analysis of polygenic risk scores for common diseases (including breast and ovarian cancer, Alzheimer's, schizophrenia, and type 2 diabetes) reveals distinct micro-geographical patterns of disease risk in the region, stratified by genetic clusters. These insights highlight the importance of further research into implications for public health policy.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.27.662083v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Pollinate to Suppress: Sterile Pollen and Certation Effects in Palmer amaranth (Amaranthus palmeri)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wu, W., Mesgaran, M. B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A sterile pollen technique for reducing seed set in Palmer amaranth .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The sterile pollen technique, which involves applying X-ray irradiated pollen to female plants, has shown promising results in reducing seed production in dioecious Palmer amaranth (Amaranthus palmeri S. Wats.). However, field-scale implementation of this method requires a carrier for pollen delivery, as applying pure pollen on a large scale is not practical. Additionally, variability in flowering time within and among individual plants may impact the technique's effectiveness. Mass pollination may also lead to a female-biased sex ratio in the progeny, a phenomenon known as certation. Therefore, in this study, we aimed to: (1) identify a suitable dry (inert) diluent and the most effective pollen-to-diluent ratio for large-scale application, and (2) determine the most effective combination of initiation time, frequency, and number of sterile pollen applications, and 3) test for evidence of certation in A. palmeri. Plants were grown in the greenhouse in summer 2021. Sterilized pollen irradiated at 300 Gy was mixed with talc or wheat powder flour, respectively, at six v/v ratios (pollen%/powder%) of 0/100 (powder alone), 5/95, 10/90, 25/75, 50/50, or 100/0 (pure irradiated pollen). An equal amount of the pollen-diluent powder mixture was then brushed onto the standardized lengths of inflorescence of receptive female plants. Flower and seed number on each inflorescence was counted and used to calculate seed set in each treatment. The findings showed that a minimum of 25% irradiated pollen in the mixture, either with talc powder or wheat flour, can effectively reduce seed set in A. palmeri. Building on this, a second round of greenhouse experiments was conducted in fall 2021, where female plants were pollinated with a 25% irradiated pollen and talc powder mixture using a powder duster. Pollination treatments were initiated at different times after anthesis (7, 14, and 21 days), and the number of applications varied (once, twice, or three times) with intervals of one, two, or three weeks between applications. The greatest reduction in seed output was achieved when sterile pollen application began seven days after anthesis and was repeated three times at 7-day intervals. Evidence of certation was also observed in A. palmeri, with the progeny of mass-pollinated females showing a higher female ratio (58%) compared to the open-pollinated population (47%). Mass pollination also led to a slight, though not statistically significant, reduction in plant size traits such as height and number of branches, providing an additional potential benefit of the sterile pollen technique.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.30.662479v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Unlocking Genetic Diversity in Colombian Cassava Landraces for Accelerated Breeding</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhao, K., Long, E., Sanchez, F., Chavarriaga, P., Monroe, J. G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Cassava landraces are a reservoir of genetic diversity that could be used to improve modern varieties. We found evidence that deleterious loss-of-function alleles were purged by inbreeding, and found genes retaining LoF alleles despite inbreeding.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cassava (Manihot esculenta Crantz) is a staple food for hundreds of millions across the global south. In this study, we investigated genomic diversity among over 1000 cassava genotypes, with a particular focus on the addition of 387 newly sequenced landrace varieties originating from diverse climates across Colombia. As cassava was domesticated in or near the Amazon basin, these landraces represent untapped genetic diversity that could be used to help improve modern varieties. As theory predicts, we found that landraces retain high genetic diversity, observing variation lacking in breeding lines from Asia and Africa, where introductions likely caused population bottlenecks. Genetic differentiation in landraces reflects both space and climate of origin, suggesting the combined effects of demography and selection. To identify alleles with the potential to inform targets for gene editing, we assessed the diversity of loss-of-function (LoF) mutations across these landraces. We found evidence that deleterious LoF alleles were purged by inbreeding. Notably, genes retaining LoF alleles despite inbreeding were significantly enriched for functions related to the biosynthesis of coumarins and the regulation of plant immunity, suggesting selection on postharvest quality and disease resistance. We further identified specific loci associated with climates of origin, motivating future experiments using targeted knockouts to test hypotheses about the adaptive value of specific LoF alleles. This work supports longstanding hypotheses about landraces as a reservoir of genetic diversity and establishes the foundation to leverage this variation in cassava to discover alleles for accelerated breeding via gene editing.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.30.662420v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Transcriptional response to Phytophthora root rot in raspberry identifies RiABP19, a Germin-like protein (GLP) gene with a putative role in resistance</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Osama, R., Ogilvie, C. E., Fisher, S. R., Welsh, L., Fuller, J., Smith, K., Bezanger, A., Milne, L., McLellan, H., Simpson, C. G., Gilroy, E. M., Grant, M. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We study the transcriptomic response of raspberry root gene expression to phytophthora root rot and propose a new raspberry cultivar-specific induction signature for resistance.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Most phytophthora root rot (PRR) outbreaks in symptomatic commercially cultivated raspberry varieties are associated with the prevalence of Phytophthora rubi. Reduced availability of chemical actives and the persistent presence of Phytophthora oospores in the soil contribute to its devastating impact on raspberry-growing regions. In this study, we examined the variation in root morphology in two contrasting raspberry cultivars, Latham (PRR resistant) and Glen Moy (PRR susceptible). We performed RNA-sequencing on Latham roots challenged with P. rubi, to study the transcriptomic response and uncover mechanisms underpinning resistance. We established a new raspberry reference transcript dataset that allowed quantification of raspberry root gene expression. Transcripts significantly upregulated in Latham challenged with P. rubi, included many with characterised roles in resistance, such as Pathogenesis-related proteins and a Germin-like protein, designated RiABP19. The homologous Glen Moy RiABP19 gene showed no differential transcriptional response to PRR infection, indicating a resistance cultivar-specific induction signature following PRR challenge. Three-dimensional structural modelling predicts that RiABP19 contains conserved active sites implicated in auxin-binding and superoxide dismutase activity and can form a homo-hexamer like true germins. Co-immunoprecipitation assays confirmed that RiABP19 can form both homo- and heterodimers in planta. Virus-induced gene silencing of RiABP19 orthologs of in the model plant Nicotiana benthamiana strongly impacts immune signalling, enhancing Phytophthora infestans colonization and attenuating resistance and cell death triggered by the tomato Cf4/Avr4 interaction. These findings suggest that RiABP19 functions as a positive regulator of immunity and may represent a target for future crop improvement in raspberries.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.01.662284v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Variability in cadmium accumulation in quinoa grain: potential role of crop domestication and other factors mediating differences in resilience and uptake</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rodriguez-Sanchez, A., Jimenez-Castaneda, M., Filley, T., Brenner, D., Macedo, D., Casa, V. H., Hoagland, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of Cd-uptake in quinoa and its effects on phenological development .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Quinoa (Chenopodium quinoa) is a nutritious crop expanding in importance worldwide, though its capacity to accumulate cadmium (Cd) can put consumers at risk. The objective of this study was to identify genotypes that vary in Cd uptake and explore how domestication, saponins, and root-associated microbes could influence these differences. Twelve genotypes representing wild ancestors and modern landraces or cultivars from four distinct ecoregions were grown in a greenhouse trial using potting media amended with Cd. Differences in phenological developmental traits were recorded during growth, and plant biomass, microbial derived amino sugars in roots, and concentrations of Cd and other essential elements in grains were quantified at harvest. Phenological development of Ancestors were more adversely affected by Cd than modern genotypes, and Cd concentrations were higher in those developed later near coastal regions in South America and the United States, suggesting that quinoa become more tolerant over domestication and selection. Concentrations of several elements were also lower in the Ancestors vs. modern genotypes in response to Cd, indicating that modern genotypes may not downregulate root transporters as an adaptation to Cd stress. Fungal derived amino sugars were negatively correlated with Cd concentrations in the Coastal ecotypes which had the highest uptake, suggesting that the presence of root-associated fungi could help restrict translocation aboveground. Results of this study will provide breeders with new insights into factors that could aid in selecting for quinoa with low Cd-uptake. Early genotypes selected in the Altiplano region may be most useful in these breeding programs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.30.662325v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Mycotoxin-driven proteome remodeling reveals limited activation of Triticum aestivum responses to emerging chemotypes integrated with fungal modulation of ergosterols</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ramezanpour, S., Alijanimamaghani, N., McAlister, J. A., Hooker, D., Geddes-McAlister, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of the mycotoxin chemotype 15ADON/3ANX in Fusarium head blight shows a significant reduction in mycotoxin protective mechanisms in wheat and a novel connection between elevated ergosterol biosynthesis and fungal virulence and host defense responses.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Fusarium head blight (FHB), mainly caused by Fusarium graminearum, is a globally important wheat disease reducing yield and grain quality. The pathogen produces mycotoxins deoxynivalenol (DON), 3-acetyl DON (3ADON), 15-acetyl DON (15ADON), and nivalenol (NIV), which threaten food and feed safety. During the past 15 years, surveillance has identified nove trichothecenes 3ANX and NX, which show increased virulence compared to DON. In this study, we investigated the effects of 15ADON/3ANX chemotype on both wheat and F. graminearum proteomes to identify proteins and pathways responsive to the emerging mycotoxin chemotype. We defined a core wheat proteome across all strains (15ADON- and 15ADON/3ANX-producing, and untreated controls) to explore changes in protein abundance associated with defense response, grain development, and reduced photosynthesis upon infection. Conversely, we identified 32 wheat proteins exclusively produced in the presence of 15ADON/3ANX strains, providing further insight into chemotype-specific responses of wheat. Additionally, assessment from the fungal perspective, reported 119 proteins exclusive to the 15ADON/3ANX strains, including those associated with virulence and mycotoxin production. Lastly, investigation of strain-specific proteome changes showed a significant reduction in mycotoxin protective mechanisms in wheat upon exposure to two 15ADON/3ANX strains, as well as a novel connection between elevated ergosterol biosynthesis and 15ADON/3ANX producing strains. Together, our study characterizes distinct protein production profiles in wheat and F. graminearum in response to 3ANX and provides evidence that these molecular changes influence fungal virulence and host defense responses.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.30.662327v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Cell-type-specific execution of effector-triggered immunity</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chhillar, H., Jo, L., Redkar, A., Kajala, K., Jones, J. D., Ding, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that a core set of defense genes is broadly induced, while distinct cell types activate specialized immune modules. We infer that downstream immune execution is shaped not only by immune receptor activation, but also by cell identity and its associated transcriptional regulatory context.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Effector-triggered immunity (ETI) is a central component of host defense, but whether all cell types execute ETI similarly remains unknown. We combined chemically imposed immune activation with single-cell transcriptomics to profile ETI responses across all leaf cell types in Arabidopsis. Despite uniform ETI perception, we find striking divergence between transcriptional outputs: a core set of defense genes is broadly induced, while distinct cell types activate specialized immune modules. We infer that downstream immune execution is shaped not only by immune receptor activation, but also by cell identity and its associated transcriptional regulatory context, including local transcription factor availability and chromatin accessibility. We further demonstrate that transcriptional regulators preferentially induced in epidermal cells are required to restrict invasion by non-adapted pathogens. Their absence permits pathogen entry into deeper tissues despite intact recognition, revealing a spatial division of immune functions. Our findings uncover a layered immune architecture in plants, challenges the assumption of uniform immune execution, and provides a framework for exploring cell-type-specific resistance logic in multicellular hosts.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.28.662111v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Continuous Directed Evolution of a Plant Histidinol Dehydrogenase to Extend Lifespan</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Oliveira-Filho, E. R., Gayen, A. K., Leong, B. J., Belt, K., Millar, H., Hanson, A. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We used the yeast OrthoRep continuous directed evolution system to extend enzyme lifespan in vivo by selecting for growth rate while tapering histidinol concentration and escalating that of the inhibitor histamine.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Enzyme protein turnover accounts for about half the maintenance energy budget in plants. Slowing turnover - i.e., extending lifespan - of short-lived enzymes is thus a rational strategy to conserve energy and carbon, and raise crop productivity. Arabidopsis histidinol dehydrogenase (HDH) is a short-lived enzyme that can sustain life-shortening damage from its aminoaldehyde reaction intermediate. We used the yeast OrthoRep continuous directed evolution system in a his4{Delta}; strain to raise HDH protein abundance (a proxy for lifespan) by selecting for growth rate while tapering histidinol concentration and escalating that of the inhibitor histamine. Improved HDHs carried diverse nonsynonymous mutations and ranged 20-fold in level. Improved HDH performance was associated with higher HDH abundance in some cases and with greater catalytic efficiency or histamine resistance in others. These findings indicate that OrthoRep-based directed evolution can extend enzyme lifespan in vivo in addition to, as expected, altering kinetic properties.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.01.662441v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

