<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Wednesday, 16 July 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Decision level scheme for fusing multiomics and histology slide images using deep neural network for tumor prognosis prediction.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tingting Zhao, Yongyong Ren, Hui Lu, Yan Kong</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a multimodal fusion framework for integrating multiomics and pathological tissue slide images for prognosis prediction.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Molecular biostatistical workflows in oncology often rely on predictive models that use multimodal data. Advances in deep learning and artificial intelligence technologies have enabled the multimodal fusion of large volumes of multimodal data. Here, we presented a decision level multimodal data fusion framework for integrating multiomics and pathological tissue slide images for prognosis prediction. Our approach established the spatial map of instances by connecting the neighboring nuclei in space and calculated the characteristic tensor via graph convolution layers for the input pathological tissue slide images. Global Average Pooling was applied to align and normalize the feature tensors from pathological images and the multiomics data, enabling seamless integration. We tested our proposed approach using Breast Invasive Carcinoma data and Non-Small Cell Lung Cancer data from the Cancer Genome Atlas, which contains paired whole-slide images, transcriptome data, genotype, epienetic, and survival information. In a 10-fold cross-validation, the comparison results demonstrated that the multimodal fusion paradigm improves outcome predictions from single modal data alone with the average C-index increasing from 0.61 to 0.52 to 0.75 and 0.67 for breast cancer and non-small cell lung cancer cohort, respectively. The proposed decision level multimodal data fusion framework is expected to provide insights and technical methodologies for the follow-up studies.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40664732/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">scGPT: end-to-end protocol for fine-tuned retinal cell type annotation.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shanli Ding, Jin Li, Rui Luo, Haotian Cui, Bo Wang, Rui Chen</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We provide an off-the-shell solution of high-precision cell-type annotation using single-cell generative pretrained transformer (scGPT) and show its efficiency in handling complex data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell research faces challenges in accurately annotating cell types at high resolution, especially when dealing with large-scale datasets and rare cell populations. To address this, foundation models such as single-cell generative pretrained transformer (scGPT) offer flexible, scalable solutions by leveraging transformer-based architectures. Here we provide a comprehensive guide to fine-tuning scGPT for cell-type classification in single-cell RNA sequencing data. We demonstrate how to fine-tune scGPT on a custom retina dataset, highlighting the model's efficiency in handling complex data and improving annotation accuracy achieving 99.5% F1-score. This protocol automates key steps, including data preprocessing, model fine-tuning and evaluation. This protocol enables researchers to efficiently deploy scGPT for their own datasets. The provided tools, including a command-line script and Jupyter Notebook, simplify the customization and exploration of the model, proposing an accessible workflow for users with minimal Python and Linux knowledge. The protocol offers an off-the-shell solution of high-precision cell-type annotation using scGPT for researchers with intermediate bioinformatics. The source code and example datasets are publicly available on GitHub and Zenodo.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40665101/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Benchmarking methods for genome annotation using Nanopore direct RNA in a non-model crop plant</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Davis, J. M., Gagalova, K. K., Vincis Pereira Sanglard, L. M., Cuellar, S., Gibberd, M. R., Naim, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We benchmarked five genome annotation tools for plant-specific analysis using dRNA data from barley infected with Net Form Net Blotch disease and found substantial variation across tools.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> High-quality genome annotations are essential for transcriptomic analyses investigating plant responses to environmental stress. While Nanopore long-read direct RNA (dRNA) sequencing offers a powerful approach for improving genome annotations, studies benchmarking optimal tools for this process have primarily focused on animal models. In this study, we benchmarked five annotation tools: StringTie2, IsoQuant, Bambu, FLAIR, and FLAMES, using dRNA data from barley infected with Net Form Net Blotch disease. We observed substantial variation across tools in isoform detection, structural completeness, splicing classification, and handling of 5' read truncation. Several tools successfully identified novel transcripts, with the top-performing reference-guided approach detecting 994 previously unannotated transcripts, including candidates with predicted roles in disease response. Our results highlight the importance of plant-specific benchmarking of bioinformatic tools and demonstrate the utility of dRNA sequencing for improving genome annotations, supporting ongoing efforts to enhance reference resources for non-model plant species.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.14.664836v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Design of a Multi-Epitope Vaccine using β-barrel Outer Membrane Proteins Identified in Chlamydia trachomatis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Panda, A., Kapoor, J., Hareramadas, B., Naqvi, I., Ganta, S., Chhabra, R., Kumar, S., Bandyopadhyay, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a multi-epitope vaccine for chlamydia trachomatis infection using outer membrane {beta}-barrel proteins and a 3D structural model of MEV, which has strong and stable binding interactions with Toll-like receptor 4 and T lymphocyte receptors.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Chlamydia trachomatis is an obligate intracellular Gram-negative pathogen responsible for causing sexually transmitted infections (STIs) and trachoma. Current interventions, including screening and antibiotics, are limited due to the widespread nature of asymptomatic infections, and the absence of licensed vaccine exacerbates the challenge. In this study, we predicted outer membrane {beta}-barrel (OMBB) proteins and designed a multi-epitope vaccine (MEV) construct using identified proteins. We employed a consensus-based computational framework on the C. trachomatis D/UW-3/CX proteome and identified 17 OMBB proteins, including well-known Pmp family members and MOMP. Eight OMBB proteins were computationally characterized, which showed significant structural homology with known outer membrane proteins from other bacteria. Sequence-based annotation tools were used to determine their putative functions. B-cell and T-cell epitopes were predicted from the selected proteins. The MEV construct was designed using four cytotoxic T lymphocyte (CTL) epitopes and 29 helper T lymphocyte (HTL) epitopes predicted from six OMBB proteins, which were conserved across 106 C. trachomatis serovars. The vaccine was supplemented at the N-terminus with Cholera enterotoxin subunit B and PADRE sequence to enhance its immunogenicity. The MEV construct of 780 amino acids was antigenic, non-allergenic, non-toxic, and soluble. Secondary structure analysis revealed 95% random coils. The 3D structural model of MEV was generated and validated, confirming its structural reliability. Molecular docking between MEV and Toll-like receptor 4 (TLR4) revealed strong and stable binding interactions, supporting its potential to elicit a strong immune response. This study highlights OMBB proteins as promising immunogenic targets and presents a computationally designed MEV candidate for C. trachomatis infection.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.10.664186v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Predicting Future SARS-CoV-2 Mutations using Deep Learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ayaz, H., Ibrahimzada, A. R., Adebali, O., Fejzullahu, A., Azgari, C., Baysan, M., Buyukbingol, M. E., Demir, G., Karlik, E., Ozpolat, B., CAKMAK, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose several deep-learning-based methods to estimate the possible future mutations in SARS-CoV-2 genome and demonstrate that the best-performing methods achieve an F1-Macro score of 0.78.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> SARS-CoV-2 continues to spread over the world steadily as opposed to many earlier estimations that it would disappear in less than two years. Even though SARS-CoV-2 vaccines have reduced the speed of the infection significantly, they could not fully stop it. On the contrary, the World Health Organization has recently published cautionary statements that infection counts are on the rise, and a huge wave is expected in winter. Vaccines mostly target specific regions of the virus. The high mutation rate of SARS-CoV-2 is one essential tool that the virus exploits to escape from the available vaccines. Therefore, researchers have been working on designing next-generation vaccines against the new variants of the virus. Nevertheless, SARS-CoV-2 acquires new mutations faster than we can adapt our vaccines due to long clinical trial periods. Hence, there is a need for computational tools that can predict future SARS-CoV-2 mutations before they even emerge. In this paper, we propose several deep-learning-based methods to estimate the possible future mutations in SARS-CoV-2 genome. We design and evaluate various ensemble and bagging architectures enriched with a large set of genomic, biochemical, and phylogenetic features. We evaluate our models on the GISAID data and demonstrate that the best-performing methods achieve an F1-Macro score of 0.78.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.12.664533v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">IS1216 drives the evolution of pRUM-like multidrug resistance plasmids in Enterococcus faecium</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Allen, F., McInnes, R. S., van Schaik, W., Moran, R. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose an IS1216 driven model for the evolution of pRUM-like plasmids, which appear to have arisen in E. faecium and contributed to the international success of CC17 as an opportunistic pathogen.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> pRUM-like plasmids are commonly found in multidrug-resistant Enterococcus faecium, but the evolution of these plasmids has not been characterised in detail. When we analysed the genome sequences of two clinical E. faecium strains isolated in Birmingham, UK, we found two pRUM-like plasmids, pHHEf1 and pHHEf2. They were approximately 25 kb in size and shared the same 10 kb backbone, but contained starkly different accessory regions that were bounded by and interspersed with the IS26 family insertion sequence IS1216. pHHEf1 contained a complete set of vancomycin resistance genes, while pHHEf2 contained aminoglycoside and erythromycin resistance genes along with an integrated small plasmid, pCOLA. It appeared that IS1216 had driven the diversification of these accessory regions. We sought to characterise the role of IS1216 in the broader evolution of pRUM-like plasmids by performing comparative analyses on 152 complete plasmid sequences from five continents. Extensive IS1216-mediated variation included backbone deletions, acquisition and loss of 10 different antibiotic resistance genes, and the formation of cointegrates with plasmids of at least 10 different replicon types. Cointegration events have introduced accessory segments with diverse functions, including horizontal transfer determinants and genes for bacteriocin T8. The derivations of these acquired segments highlight the impact of IS1216 in driving gene exchange between Enterococcus and Staphylococcus species. We traced the emergence of the pRUM-like lineage to a putative ancestor found in a vancomycin-sensitive ST17 E. faecium isolated in 1997. The ancestral plasmid, pCANE, includes the entire pRUM backbone with an additional 44.9 kb in place of the pRUM accessory region. The 44.9 kb segment includes putative conjugation determinants, suggesting that the emergence of the pRUM-like lineage coincided with a loss of transfer functions. We propose an IS1216 driven model for the evolution of pRUM-like plasmids, which appear to have arisen in E. faecium ST17 and contributed to the international success of CC17 as an opportunistic pathogen.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.10.664085v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Inferring Progressive Disconnection in Alzheimer's Disease with Probabilistic Boolean Networks</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liu, Z., Zhang, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We apply probabilistic Boolean networks to model effective brain connectivity for the first time, introducing a novel framework for analyzing functional magnetic resonance imaging data from a cohort comprising normal controls, individuals with mild cognitive impairment (MCI), and Alzheimer's patients.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The modern understanding of Alzheimer's disease as a disconnection syndrome presents the challenge of quantifying the directed influence between brain regions. To address this, we apply probabilistic Boolean networks to model effective brain connectivity for the first time, introducing a novel framework for analyzing functional magnetic resonance imaging data from a cohort comprising normal controls, individuals with mild cognitive impairment (MCI), and Alzheimer's patients. Our robust statistical analysis identified five significant connections, each exhibiting a linear decline in influence throughout the disease spectrum. We observed a progressive disruption of pathways from the Default Mode Network to the Medial Temporal Lobe, capturing a key psychophysiological mechanism underlying Alzheimer's disease. These findings demonstrate the potential of our framework as a powerful tool for modeling network-level dynamics in neurodegeneration.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.10.664143v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comparative clinical transcriptome of pir genes in severe Plasmodium vivax malaria</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Boopathi, P. A., Singh, S., Roy, P., Tahbildar, S., Kochar, S. K., Kochar, D. K., Kumar Das, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a global transcriptomic study using custom designed microarrays, that compare the transcriptome of the Plasmodium vivax parasite responsible for severe Plas modal malaria manifestations, specifically hepatic dysfunction and cerebral malaria from India, with an emphasis on the pir genes, some of</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Vir genes, a multigene family in Plasmodium vivax that are a part of a larger superfamily of genes called the pir (Plasmodium interspersed repeat) genes have been reported earlier to be potentially involved in cyto-adherence and evasion of splenic clearance. Plasmodium vivax, historically characterized as a "benign" malaria parasite, has been associated with clinical outcomes including hepatic dysfunction, renal failure, and cerebral malaria in India and several global regions. It constitutes an economic burden and presents a public health challenge alongside other Plasmodium species. Here, we present a part of global transcriptomic studies using custom designed microarrays, that compare the transcriptome of the parasite responsible for severe Plasmodium vivax manifestations, specifically hepatic dysfunction and cerebral malaria from India, with an emphasis on the pir genes, some of which are reported to play a role in cyto-adherence. 23 patients with Plasmodium vivax malaria (Uncomplicated=6, Hepatic dysfunction=12 and Cerebral malaria=5) were subjected to microarray hybridization and the data so obtained showed a wide range of pir subfamilies have been differentially expressed. Upregulation has been seen in 24 pir genes in cerebral malaria samples (n=5) and 28 genes in hepatic dysfunction samples (n=12) belonging to different subfamily in at least 50% of the patient samples. Out of the upregulated pir genes in cerebral malaria manifestation, members of vir subfamily E and pvpir H are maximum in number whereas in hepatic dysfunction manifestation, members of vir subfamily E and C comprise a significant proportion.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.10.664071v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comparing the ability of embedding methods on metabolic hypergraphs for capturing taxonomy-based features</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cervellini, M., Sinaimeri, B., Matias, C., Martino, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the choice of hypergraph embedding has a significant effect on clustering outcomes, with the goal of capturing taxonomy-based classes.</p><p style="font-size: 16px; text-align: justify;">Background: Metabolic networks are complex systems that describe the biochemical reactions within an organism through pairwise interactions between chemical compounds. While this representation is widely used to study biological function, it fails to capture the full structure of metabolic reactions, many of which involve more than two compounds. Hypergraphs offer a more natural representation, where nodes represent metabolites and hyperedges represent reactions involving multiple participants. Clustering such metabolic hypergraphs can reveal systematic differences among evolutionarily distinct organisms, providing insight into ecological constraints and evolutionary pressures. Methods: In this study, we investigate how different graphs and hypergraphs embedding methods influence their unsupervised clustering, with the goal of capturing taxonomy-based classes. We apply 14 distinct embedding strategies to a large-scale dataset of 8,467 metabolic hypergraphs. Each embedding was followed by hierarchical clustering using a fixed linkage method. To assess performance, we compared the resulting clusters against known taxonomic groupings. Results: Our findings show that the choice of hypergraph embedding has a significant effect on clustering outcomes. Among the tested methods, Bag of Hyperedges with Jaccard distance, Histogram Cosine Kernel, and a Hypergraph Auto-Encoder consistently performed best. We also advocate that the embedding method should be chosen based on the goal of the downstream task.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.10.663860v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Dynamic of the transcriptomic landscape of OsHV-1 replication in haemocytes of Pacific oyster</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dotto-Maurel, A., Le Luyer, J., Faury, N., Degremont, L., Tragin, M., Renault, T., MORGA, B., Chevignon, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comprehensive transcriptomic study of virus-host interactions across multiple stages of infection in haemocytes from Pacific oysters showing contrasted survival when exposed to OsHV-1.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Since the 1990s, the Pacific oyster (Magallana gigas) has experienced repeated mortality events associated with Ostreid herpesvirus 1 (OsHV-1). Although the virus has been genomically characterised, its replication cycle and its interactions with the oyster immune system are still not well understood. In particular, little is known about the dynamics of OsHV-1 gene expression and the immune responses of haemocytes from oysters with varying susceptibility to the virus. While some studies have focused on the expression of specific viral and host genes on whole oysters, none have provided a comprehensive analysis of genomes-wide expression across multiple post-infection time points in haemocytes. The lack of oyster cell lines makes studying virus-host interactions in vitro challenging. However, haemocytes, the key immune cells circulating in hemolymph, can be maintained in vitro in the short term and represent a relevant model for analyzing infection dynamics. In this study, haemocytes from two M. gigas families, one highly susceptible and one less susceptible to OsHV-1, were infected in vitro. We tracked the viral and host transcriptomes over a 24-hour period post-infection using high-throughput dual transcriptomics. Our results provide a detailed overview of the OsHV-1 transcriptomic landscape in haemocytes from high and low susceptible M. gigas over time. In addition, WGCNA analysis of host genes expression provided insights into the haemocytes response to infection, and highlighted family-specific immune responses. This comprehensive transcriptomic study is the first to describe virus-host interactions across multiple stages of infection in haemocytes from Pacific oysters showing contrasted survival when exposed to OsHV-1.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.11.664376v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Exploring Hypoxia-Related Genes as Prognostic Indicators in Lung Adenocarcinoma</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Alghamdi, B., Rocha, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study aimed to identify hypoxia-related differentially expressed genes (HRDEGs) in lung adenocarcinoma (LUAD) through differential expression analysis and identified 17 key genes associated with patient survival.</p><p style="font-size: 16px; text-align: justify;">Background Lung adenocarcinoma (LUAD) is a leading cause of cancer-related mortality, with hypoxia contributing to tumor progression and treatment resistance. Identifying hypoxia-related biomarkers could enhance prognosis and therapeutic strategies for LUAD. Methods This study aimed to identify hypoxia-related differentially expressed genes (HRDEGs) in lung adenocarcinoma (LUAD) through differential expression analysis. Functional analysis and protein-protein interaction (PPI) network construction were performed to explore the biological roles and interactions of these genes. Kaplan-Meier survival analysis, univariate Cox regression, and Lasso regression were used to identify key genes associated with survival. Multivariate Cox regression was then conducted to assess independent prognostic factors. Results This analysis revealed 283 upregulated HRDEGs and 322 downregulated HRDEGs in LUAD. Functional enrichment analysis indicated that the upregulated genes were primarily involved in cancer-related and cellular signaling pathways, while downregulated genes were associated with immunity-related pathways. We further identified 201 common upregulated hub genes (including MMP9, CDH1, HSP90AB1, SOX2, CDKN2A, SPP1, EZH2) and 224 common downregulated hub genes (such as IL6, TNF, IL1B, JUN, CCL2, TLR4, FOS, PTGS2). Kaplan-Meier survival analysis, univariate Cox regression, and Lasso regression led to the identification of 17 key genes (ADRB2, ALDH2, CAT, CCNE1, MAP3K8, DSG2, EIF6, ABCB1, PIK3R1, RAD51, SFTPD, SOD3, CLEC3B, ADAM12, EXO1, FBLN5, and IGF2BP3) associated with patient survival. Finally, multivariate Cox regression analysis identified DSG2, EIF6, and EXO1 as independent prognostic factors for LUAD, highlighting their potential as biomarkers for prognosis and therapeutic targets in lung cancer. Conclusion In conclusion, DSG2, EIF6, and EXO1 were identified as key hypoxia-related genes in lung adenocarcinoma. These genes were found to be independent prognostic factors, highlighting their potential as biomarkers for predicting patient survival and guiding future therapeutic approaches.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.10.664061v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Auxin antagonism of ABA-induced stomatal closure in Arabidopsis requires the TIR1/AFB-AUX/IAA module and ROS suppression</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rios, C. R. A., Toum, L., Padula, L., Pieckenstain, F. L., Gudesblat, G. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Auxin inhibits ABA-induced stomatal closure, yet the underlying mechanism remains unclear. We quantified 3-Indoleacetic acid (IAA) inhibition of ABA and found that IAA also caused partial degradation of the DII-VENUS auxin reporter, and</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Auxin inhibits ABA-induced stomatal closure, yet the underlying mechanism remains unclear. In this work we quantified 3-Indoleacetic acid (IAA) inhibition of ABA-triggered closure in Arabidopsis thaliana mutants and transgenic plants and observed that IAA inhibition of ABA-induced stomatal closure was lost or weakened in tir1-1 and axr3-1 mutants, affected in auxin perception. The same was found in aux1-7, yuc5/8/9 and pils5-2/2-5 mutants, and in YUCCA8 and PILS5 overexpressors, all altered in auxin synthesis or transport. Stomatal sensitivity to IAA was also reduced in phyB-9 and rbohd, toc1 and cca1/lhy mutants, in certain ecotypes, and in response to mild drought. H+-ATPase mutant aha2-5 showed reduced stomatal sensitivity to IAA at 22C but not at 35C, while high-temperature-induced stomatal opening was not affected in tir1-1. IAA also caused partial degradation of the DII-VENUS auxin reporter, and inhibited ABA-induced ROS production in guard cells. Our results show that IAA counteracts ABA signalling via the TIR1/AFB-AUX/IAA module and ROS suppression, with genetic factors and mild drought modulating the response.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.15.664744v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Loss of Nucleotide Sugar Transporter (AtNST) gene function in the Golgi membranes impairs pollen development and embryo sac progression in Arabidopsis thaliana</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Diman, R., Malathi, P., Srinivasan, R., Bhat, S. R., Yelam, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A mutation in the Arabidopsis thaliana gene At3g11320 coding for NST protein causes defects in male gametophyte development, including collapsed, nonviable pollen, and impairment in female gamete development.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Nucleotide Sugar Transporters (NSTs) are transmembrane proteins that are localised in the Golgi membranes and transport nucleotide sugars from the cytosol to the Golgi lumen. Transported nucleotide-sugars serve as donors in post-translational modification of proteins/ lipids, and donate sugar moiety to the growing carbohydrate chain on the nascent protein/lipid molecule, a reaction catalysed by the enzyme glycosyltransferase. Here we reported that a mutation in the Arabidopsis thaliana gene At3g11320 coding for NST protein causes defects in male gametophyte development, including collapsed, nonviable pollen. This mutation also caused impairment in the female gametophyte progression by arresting it at the functional megaspore (FM) stage. Further, the mutant phenotype, including silique size and seed set, was reverted when the cDNA of the AtNST gene was over-expressed in the mutant background. No abortive ovules were found in the siliques from the complemented plants. The results suggest that AtNST (At3g11320) gene might be responsible for maintaining and regulating transport of nucleotide-sugars from cytoplasm to the Golgi lumen for the glycosylation of essential proteins/carbohydrates/lipids etc., which are necessary for both male and female gametophyte development stages. This study shed further light on the role of such nucleotide sugar transporters in plant reproductive development processes, at least in Arabidopsis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.10.664101v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Making a clean break: contrasting leaf abscission dynamics across temperate leaf habits</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kane, C. N., Rimer, I. M., McAdam, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel method to quantify abscission zone competency (AZC) in deciduous trees and show that senescence and abscissions are distinct and variably timed processes across temperate tree species.</p><p style="font-size: 16px; text-align: justify;">Background and Aims: Leaf senescence allows plants to reallocate nutrients from aging leaves, often storing them for future use. As chlorophyll breaks down, other pigments emerge, creating the vibrant colors of autumn. In many deciduous trees, senescence is followed by abscission the physical detachment of leaves at the abscission zone (AZ). Though closely linked, senescence and abscission are distinct; some plants senesce without abscising, while others abscise partially senesced leaves. Most research on these processes has focused on herbaceous and crop species, which do not naturally shed leaves. Methods: To better understand how leaf senescence and abscission occur in temperate deciduous trees, we developed a novel method to quantify AZ competency (AZC) we also measured gas exchange, chlorophyll content, water potential, and abscisic acid in four tree species representing deciduous, brevi-deciduous, and marcescent leaf habits. Key results: The two deciduous species showed contrasting patterns: one degraded chlorophyll and ceased photosynthesis before AZC developed; the other retained chlorophyll and continued photosynthesis until nearly all AZs became competent. The brevi-deciduous species lost most chlorophyll but developed AZs gradually over a longer period. The marcescent species fully senesced but did not develop AZC. Conclusions: These findings show that senescence and abscission are distinct and variably timed processes across temperate tree species.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.10.664177v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Fragaria ananassa DAM4 expression correlates with vegetative growth during endodormancy breaking</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> David, S., Kersten, P., Cao, X., Marcelis, L. F. M., Verdonk, J. C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Dormancy-Associated MADS-BOX (DAM)3 and DAM4 expression patterns are consistent with that of regulators of endodormancy.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Dormancy-Associated MADS-BOX (DAM)3 and DAM4 have been described as potential regulators of winter dormancy in cultivated strawberry (Fragaria x ananassa Duch.). These genes are upregulated under short day conditions and downregulated under chilling conditions. The aim of the current work is to test the hypothesis that the expression of DAM3 and DAM4 correlates negatively with vegetative growth during dormancy induction and breaking. The expression of DAM3 and DAM4 as well as plant morphology and physiology were analyzed during a period of endodormancy induction and breaking. Furthermore, to analyze whether there exists a correlation between DAM expression and chill requirement, DAM3 and DAM4 expression was compared between cultivars with different chill requirements. Lastly, the DAM1, DAM2, DAM3, and DAM4 expression of concurrently growing summer and winter leaves was compared at multiple stages of dormancy-breaking. DAM3 and DAM4 expression negatively correlated with leaf area and petiole length. We conclude that DAM3 and DAM4 expression patterns are consistent with that of regulators of endodormancy. DAM4 expression in particular correlates strongly with vegetative growth during endodormancy breaking. DAM3 or DAM4 expression were not found to correlate with the cultivar-specific chill requirements. We theorize that the chill requirement of strawberry cultivars is based on vegetative growth and does not affect DAM expression or endodormancy status. Lastly, no relevant differences in DAM expression were found between summer and winter leaves.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.10.664089v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Design-Build-Test-Learn guided engineering of a whole-cell pyruvate biosensor based on transcription factor</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gao, Z., Suarez-Diez, M., Candry, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a whole-cell biosensor that responds to intracellular pyruvate levels within 0.05-10mM range and showed that RBSs and transcription factor influence dynamic range and leaky expression.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Whole-cell biosensors are powerful tools for metabolite monitoring, yet challenges such as narrow dynamic range and high leaky expression limit their broader applications. Here, we present a systematic workflow based on two Design-Build-Test-Learn (DBTL) cycles to develop and optimize a transcription factor-based pyruvate biosensor in Escherichia coli. In the first iteration of the cycle, we constructed a biosensor that responded to intracellular pyruvate levels within 0.05 - 10mM range. In the second cycle, we implemented design of experiment (DoE) to systematically explore combinatorial effects of promoters and ribosome binding sites (RBSs). A first set of experiments were designed to identify factors with a significant effect on biosensor performance. The results showed RBS of report gene significantly influenced dynamic range by modulating basal and maximum expression, while RBS of transcription factor affected signal span. The Akaike Information Criterion was used to select a model incorporating two main effects and one interaction effects. The best-performing strain exhibited an 18.5-fold increase in dynamic range and a 37.2-fold reduction in leaky expression. Quantification of intracellular pyruvate confirmed an operational range of 1.23 - 6.81 mol/g DCW. Our work demonstrates the power of DBTL cycles with statistical modelling for biosensor engineering, enabling more precise metabolic regulation and screening applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.14.664704v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Engineering Microbial R- and S- β-Hydroxybutyrate Production</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kizil, B. C., Ulusan, S., Torun, A., Gumustas, M., Yuksel, S., Banerjee, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that E. coli strains with a prebiotic-activated circuit can synthesize enantiomeric ketone body BHOB and highlight its therapeutic potential for cancer and gut-brain axis involving disorders.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Beta-hydroxybutyrate (BHOB) is a therapeutically valuable enantiomeric ketone body that is synthesized by both prokaryotes and eukaryotes. Bacterial synthesis of BHOB has so far mostly been exploited for the synthesis of biofuels and to a lesser extent for the pharmaceutical industry. In this study, we carefully evaluated the expression, induction, selection, and detection elements and identified multigene pathways to direct the synthesis of physiologically relevant amounts of BHOB. Utilizing the reversal of the beta-oxidation pathway that has previously been used for biofuel generation, we engineered E. coli strains with a prebiotic-activated circuit to secrete either (S)-BHOB or (R)-BHOB. Our findings demonstrate high-yield microbial BHOB production with a defined enantiomeric composition and highlight its therapeutic potential for cancer and gut-brain axis involving disorders.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.15.664879v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

