<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Tuesday, 08 July 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multi-task genomic prediction using gated residual variable selection neural networks.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yuhua Fan, Patrik Waldmann</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A genome-wide prediction method that incorporates polygenic pedigree information into the genome- wide prediction process .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The recent development of high-throughput sequencing techniques provide massive data that can be used in genome-wide prediction (GWP). Although GWP is effective on its own, the incorporation of traditional polygenic pedigree information into GWP has been shown to further improve prediction accuracy. However, most of the methods developed in this field require that individuals with genomic information can be connected to the polygenic pedigree within a standard linear mixed model framework that involves calculation of computationally demanding matrix inverses of the combined pedigrees. The extension of this integrated approach to more flexible machine learning methods has been slow.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40624470/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multi-omics analysis of the effects of pla2g4a on the prognosis of various cancers and its experimental validation in breast cancer cell lines.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yao Qian, Quan Yuan, Hao Yu, Rongjie Ye, Ming Niu, Feng Liu</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify critical genes involved in Breast Cancer-associated platelet-related exosomes and evaluate their role in cancer prognosis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Platelet-related exosomes (PREs) are microparticles secreted by platelets into the bloodstream and are implicated in various cancer processes. This study aims to identify critical genes involved in Breast Cancer (BC)-associated PREs and to evaluate their role in cancer prognosis. PLA2G4A was identified as a key gene through the use of machine learning techniques and various genomic analyses, providing a foundation for precision medicine in BC treatment.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40624348/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multiple omics-based machine learning reveals peripheral blood immune cell landscape during acute rejection of kidney transplantation and constructs a precise non-invasive diagnostic strategy.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jiyue Wu, Lijian Gan, Xihao Shen, Feilong Zhang, Zhen Li, Huawei Cao, Hao Wang, Zejia Sun, Le Qi, Wei Wang</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A non-invasive diagnostic model for acute rejection in kidney transplantation and a predictive model for long-term graft survival .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Kidney transplantation is the optimal treatment for end-stage renal disease (ESRD), but acute rejection (AR) remains a major factor affecting graft survival and patient prognosis. Currently, renal biopsy is the gold standard for diagnosing AR, but its invasiveness limits the application of dynamic monitoring. This study aims to analyze changes of immune cell and gene expression in the peripheral blood of AR recipients and construct a non-invasive AR diagnosis strategy. All datasets were downloaded from the GEO database. Single cells were annotated based on the expression profiles of surface proteins and changes of immune cell in the peripheral blood of AR and stable transplant (STA) recipients were compared. The high-dimensional weighted gene co-expression network analysis (hdWGCNA) algorithm was used to analyze gene modules related to AR and to screen out hub genes by integrating bulk RNA-Seq. Based on hub genes, consensus clustering stratified recipients into two sub-clusters and a non-invasive AR diagnostic model was constructed using Convolutional Neural Networks (CNNs). Additionally, we also constructed a predictive model for long-term graft survival through combinations of 111 machine learning algorithms and validated the expression of hub genes in the rat AR model. AR recipients had higher abundance of memory B cells, effector memory T cells, terminally differentiated effector memory T cells (TEMRA), and NK T cells but lower Tregs in the peripheral blood compared to STA recipients. Through hdWGCNA analysis, we identified gene modules associated with these immune cells and screened out four hub immune-related genes (TBX21, CX3CR1, STAT1, and NKG7) after integrating bulk RNA-Seq. Based on these hub genes, recipients can be stratified into two sub-clusters with distinct clinical outcomes and biological characteristics. We also innovatively constructed a non-invasive AR diagnostic model using CNNs, which can effectively address the issues caused by batch effects and demonstrate a high diagnostic accuracy. Besides, the predictive model for long-term graft survival constructed using the RSF algorithm can divided recipients into high- and low-risk groups, with significantly higher rates of AR and long-term graft failed in the high-risk group. This study successfully identified immune cell subsets and hub genes related to AR. Based on hub genes, we successfully identified two distinct molecular sub-clusters of kidney transplant recipients, and constructed a non-invasive diagnostic model for AR and a predictive model for long-term graft survival. These models offer new tools for precise diagnosis and prognosis in kidney transplantation and may advance precision medicine.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40624201/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A multi-omics machine learning classifier for outgrowth of cow's milk allergy in children.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Diana M Hendrickx, Mariyana V Savova, Pingping Zhu, Ran An, Sjef Boeren, Kelly Klomp, Sumanth K Mutte, Presto Study Team, Harm Wopereis, Renate G van der Molen, Amy C Harms, Clara Belzer</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We integrate clinical, microbial, (meta)proteomics, immune and metabolomics data into machine learning (ML) classification, using multi-view learning by late integration, to improve classification of infants on outgrowth of CMA.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cow's milk protein allergy (CMA) is one of the most common food allergies in children worldwide. However, it is still not well understood why certain children outgrow their CMA and others do not. While there is increasing evidence for a link of CMA with the gut microbiome, it is still unclear how the gut microbiome and metabolome interact with the immune system. Integrating data from different omics platforms and clinical data can help to unravel these interactions. In this study, we integrate clinical, microbial, (meta)proteomics, immune and metabolomics data into machine learning (ML) classification, using multi-view learning by late integration. The aim is to group infants into those that outgrew their CMA and those that did not. The results show that integration of microbiome data with clinical, immune, (meta)proteomics and metabolomics data could considerably improve classification of infants on outgrowth of CMA, compared to only considering one type of data. Moreover, pathways previously linked to development of CMA could also be related to outgrowth of this allergy.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40407702/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A genome-wide approach uncovers the suite of genes important for swarming motility in the biocontrol bacteria Pseudomonas protegens Pf-5</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fabian, B. K., Foster, C., Asher, A., Elbourne, L. D., Hassan, K. A., Tetu, S. G., Paulsen, I. T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified 136 genes involved in swarming motility in Pseudomonas protegens Pf-5, including chemotaxis, c-di-GMP turnover, cell division, peptidoglycan turnover, lipopolysaccharide biosynthesis, metabolism of thiamine (v</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motility is a crucial bacterial trait, important for competitive success in many contexts. Plant growth promoting rhizobacteria, such as Pseudomonas protegens Pf-5, can protect agricultural crops against disease, relying on motility to access and colonize the plant rhizosphere. Swarming motility is the multicellular movement of bacteria powered by rotating flagella across a semi-solid surface which allows bacteria to move rapidly, search for and acquire food, avoid toxic substances, and colonize new niches. Here we performed genome-wide investigations, using transposon-directed insertion site sequencing (TraDIS) to identify 136 genes involved in P. protegens Pf-5 swarming motility. This method identified flagella biosynthesis, assembly, structure and regulatory genes, as expected. We also identified an additional suite of genes important for swarming, including chemotaxis, c-di-GMP turnover, cell division, peptidoglycan turnover, lipopolysaccharide biosynthesis, metabolism of thiamine (vitamin B1) and transport of pyoverdine. Overall, this study increases our understanding of pseudomonad motility and expands the breadth of lifestyles and habitats examined in relation to swarming motility. Understanding the genetic basis of swarming motility in biocontrol bacteria is important for successful application of these important plant growth promoting microbes in agricultural settings.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.07.663420v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Mechanism-Aware Protein-Protein Interaction Prediction via Contact-Guided Dual Attention on Protein Language Models</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Deng, S., Wan, X., Mu, Z., HUANG, S.-Y., Yan, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a mechanism-aware deep learning framework for biomolecular interaction prediction by incorporating residue-level interaction modeling.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurate prediction of protein-protein interactions (PPIs) is essential for understanding cellular mechanisms and disease. However, existing deep learning models often fail to generalize to novel proteins with low sequence similarity to training data. Here, we present a mechanism-aware deep learning framework to enhance generalization by incorporating residue-level interaction modeling. Our model integrates a dual-attention-based PPI prediction module with an inter-protein contact predictor based on protein language model-embedded geometric graphs. Joint training on structure-informed PPIs from the Protein Data Bank (PDB) enables the model to predict both interactions and the key residue pairs that mediate them. The model generalizes well to unseen proteins across species in the high-quality interactome dataset (HINT). Fine-tuning on Homo sapiens PPI data from HINT further improves the performance. Compared to D-SCRIPT, Topsy-Turvy, and TT3D, our model, referred to as PLMDA-PPI, achieves superior accuracy and robustness under strict sequence dissimilarity settings. This work provides a generalizable framework for biomolecular interaction prediction through the integration of structural and non-structural interaction data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.04.663157v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AIDO.Tissue: Spatial Cell-Guided Pretraining for Scalable Spatial Transcriptomics Foundation Model</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gong, J., Wang, Y., Ho, N., Cheng, X., Song, L., Xing, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A spatially-informed pretraining framework for single-cell spatial transcriptomics that scales with neighboring size and achieves state-of-the-art performance across diverse downstream tasks.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell spatial transcriptomics enables high-resolution insights into tissue organization and cell-cell interactions, yet poses significant computational and modeling challenges due to its scale and complexity. Here we introduce AIDO.Tissue, a spatially-informed pretraining framework. The design employs multiple cells as input and an asymmetric encoder-decoder architecture, making it effectively encodes cross-cell dependencies while scaling to large data. Systematic evaluation shows that our method scales with neighboring size and achieves state-of-the-art performance across diverse downstream tasks, including spatial cell type classification, cell niche type prediction and cell density estimation. These results highlight the importance of spatial context in building general-purpose foundation models for tissue-level understanding.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.04.663102v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genetic basis of traits and local adaptation in UK silver birch</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Carleial, R., Charters, M., Finzgar, D., Swift, P., Anthoney, E., Sahlstedt, E., Rinne-Garmston, K., Clark, J., Cottrell, J., Whittet, R., Buggs, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We sequenced the genomes of 2054 silver birch trees from 20-year-old field trials at three locations in Britain, each site containing trees from the same 29 source populations, and identified loci associated with adaptation to source local environments.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Knowledge of the genomic basis of phenotypes in forest trees lags that of other economically important organisms. We sequenced the genomes of 2054 silver birch trees from 20-year-old field trials at three locations in Britain, each site containing trees from the same 29 source populations. Our results lay foundations for genomic source-selection and breeding. We discovered single nucleotide polymorphisms (SNP) associated with tree height, wood density, carbon isotope composition and stem form. Using genome-wide SNP we predicted variation in several phenotypic traits of trees with a similar genetic background and estimated the breeding values of 148 selected plus trees. We identified loci associated with adaptation to source local environments. We used these loci to forecast adaptation to each trial site and future predicted conditions at source. Comparison of these forecasts to phenotypic performance in the trials suggests that genomic forecasts of adaptation to future climates should be interpreted with caution.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.04.662749v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Statistical knockoffs improve biomarker discovery fromtranscriptomic data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> CARTIER, J., LAGOAS, J., FERMANIAN, A., Azencott, C.-A., MASSIP, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We study the applicability of the knockoff (KO) procedure on transcriptomic data in the context of high-dimensional classification.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Advances in sequencing technologies have enabled the generation of large amounts of data, offering new possibilities to identify relationships between biological units (e.g. genes) and phenotypic traits (e.g. disease outcomes). Yet, identifying these associations using variable selection methods remains challenging due to the high dimension and the correlation structure of the data. To address these challenges, we study the applicability of the knockoff (KO) procedure. Introduced by Barber and Cand&egraves in 2015, the KO variable selection procedure has shown promising results on real biological data, such as Genome Wide Association Studies. This method seeks to identify the truly important predictors by overcoming the correlation structure between variables while controlling the false discovery rate. Here, we study the applicability of the KO procedure on transcriptomic data in a classification setting. We conduct an extensive simulation study using real transcriptomic data to evaluate the performance of the KO framework in the context of high-dimensional classification. We find that the KO framework outperforms widely used variable selection models, and that using KO aggregation to mitigate the effect of KO stochasticity improves stability while maintaining the same power. Finally applied to three real transcriptomic datasets, the KO framework made very few discoveries, highlighting its conservative nature and suggesting that other methods may substantially overestimate the number of relevant features.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.04.663147v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The evolution of function in the DNA binding domain of the CRP/FNR family</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Krishnaswamy, M., Seshasayee, A. S. N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A phylogeny of the CRP/FNR family of transcription factors and the co-emergence of base-specific DNA binding residues in Eco-CRP.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Transcriptional regulation, facilitated by transcription factors (TFs), contributes to bacterial response to environmental and cellular perturbations. How have TFs diversified and gained their functions? We use the CRP/FNR family to study this question across ~6000 prokaryotic genomes. Characterized by homology to TFs CRP (cyclic AMP Receptor Protein) and FNR (Regulator of Fumarate and Nitrate Reductase) in Escherichia coli, the two functions of this family - sequence-specific DNA binding and transcriptional activation by direct contacts to RNA polymerase - are conferred by the helix-turn-helix containing DNA binding domain. We constructed a rooted phylogeny of this domain and performed residue conservation analysis on extant and in-silico reconstructed ancestral sequences of CRP/FNR family members. Residues that confer DNA binding specificity likely emerged in the overall ancestor of the sequence family. This ancestor does not fall into classes containing either E. coli CRP (Eco-CRP) or E. coli FNR (Eco-FNR) clades, but instead a large and diverse class that we call CRP-like. Residues key for base-specific DNA binding remain largely conserved in the family. Residues (together called AR1) which make direct contacts with CTD of RNA Polymerase are conserved only in the Eco-CRP class and restricted to Gammaproteobacteria. The corresponding 287 determinant interface on CTD is again fully conserved only in Gammaproteobacteria, which contain Eco-CRP members, following step-wise evolution initiated at the Proteobacterial common ancestor. These suggest suggesting co-emergence of the interface residues in Eco-CRP. Our analysis hence shows that Class I transcriptional activation through AR1 on CRP and the determinant of CTD, as described in E. coli CRP, is phylogenetically restricted, while base-specific DNA binding has been present in the CRP/FNR family throughout its evolutionary history.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.04.663149v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A framework for integrating genomic profiles into captive breeding and reinforcement programmes: A case study on captive saker falcons</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hoareau, T. B., Barbosa, A., Velkeneers, X., Leveque, G., Lesobre, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Genetic analysis of Saker Falcons for ex-situ conservation and genetic relatedness of founders .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Ex-situ conservation is crucial for preserving endangered species by breeding to preserve genetic diversity and provide surplus for translocation, thereby supporting in-situ conservation and enhancing wild populations. Genomic tools can assist breeding strategies and ensure long-term success of ex-situ conservation efforts by assessing genetic introgression, determining genetic origin and status, and inferring genetic relatedness of potential founders. This study aims to develop a comprehensive genomic approach for assessing the genetic profiles of candidate founders for ex-situ breeding, with the goal of releasing surplus individuals while using the endangered Saker Falcons (Falco cherrug) as a study model. Genetic clustering of 31 captive sakers revealed both diverse origins, some matching wild Asian individuals (Mongolia), and a lineage (Group III) divergent from wild populations. Comparative analyses detected hybridisation signals in 61.3% of individuals, including three with gyrfalcon (F. rusticolus) introgression and Group III's pronounced divergence indicating past interbreeding with an unknown falcon species. All captive birds exhibited severe inbreeding (FROH = 0.352), far exceeding wild population levels (FROH = 0.131). Using the partial pedigree data of the captive sakers, we established a genetic relatedness threshold of 0.154 (95% CI: 0.096-0.211) to identify cases of related dyads (both full and half-siblings). At this threshold, 18.3% of captive dyads showed relatedness, with asymmetric genetic contributions between pairs, reflecting a functionally small breeding flock. To avoid risks from releasing admixed or inbred individuals, we recommend excluding introgressed birds, strategically pairing purebreds, and sourcing new founders from genetically validated wild sources, especially underrepresented Central Asian lineages. Applying this genomic framework, we demonstrate its role in safeguarding genetic integrity and preventing genetic erosion in conservation breeding programmes, thereby establishing a standard for genomic-led ex-situ conservation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.04.663124v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Rapid Identification of Metal Resistance Genes Using an Enhanced ResNet Deep Learning Model Trained on a largely Expanded BacMet-Based Database</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> CHEN, J., Gao, X., Zhang, C., Ge, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We expanded the existing BacMet database by incorporating 1,219,137 unique amino acid sequences through BLASTp analysis, thereby increasing the number of metal resistance-related acid sequences by more than 1,600-fold compared to the previous version.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Heavy metal pollution poses significant risks to both the environment and public health. Effective management requires not only reducing contaminants but also understanding microbial adaptation, which could be achieved through the comprehensive identification and classification of metal resistance genes. This study expanded the existing BacMet database by incorporating 1,219,137 unique amino acid sequences through BLASTp analysis, thereby increasing the number of metal resistance-related acid sequences by more than 1,600-fold compared to the 753 sequences included in the previous version. We employed various deep learning models for our proposed multi-label framework which could effectively overcome the well-recognized challenging issue of strict classification among metal translocating proteins such as CopA, ZntA, SilA, CadA, and CzcA due to their high sequence similarity and overlapping metal specificities, including AlexNet, VGG, GoogleNet, ResNet, BERT, ViT, and Mamba2, with ResNet demonstrating superior performance in terms of accuracy, robustness, and computational efficiency. The ResNet model achieved a Jaccard score of 98.91%, significantly higher than BLASTp (98.19%) and DIAMOND (98.16%), and was approximately 6,500 times faster in inference speed than these traditional alignment-based methods. Furthermore, the predictive performance of the model and the reliability of the expanded gene library were experimentally validated through overexpression and metal resistance assays of selected genes. Additionally, we developed the Predicting Metal Resistance Amino Acid Sequences (PMRAAS, https://s3.v100.vip:16165) website, facilitating online predictions of gene metal resistance. Our findings provided deeper insights into microbial adaptation mechanisms in metal-polluted environments and offered a robust tool for advancing heavy metal pollution management through enhanced bioremediation strategies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.04.663169v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Undersampling techniques for non-linear chemical space visualization</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Surendran, A., Zsigmond, K., Quintana, R. A. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> The visualization of high-dimensional chemical space is a critical tool for understanding molecular diversity, structure-property relationships, and for guiding compound selection, but the performance of non-linear dimensionality reduction techniques like t-SNE, Uniform Manifold Approximation and Projection (UMAP),</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The visualization of high-dimensional chemical space is a critical tool for understanding molecular diversity, structure-property relationships, and for guiding compound selection. However, the performance of non-linear dimensionality reduction (DR) techniques like t-Stochastic Neighborhood Embedding (t-SNE), Uniform Manifold Approximation and Projection (UMAP), and Generative Topographic Mapping (GTM) are often susceptible to the choice of hyperparameters, along with the high cost of their training for large datasets. In this study, we investigated the effect of undersampling methods on the choice of hyperparameter selection for these non-linear dimensionality reduction methods. Our results demonstrate that selecting small representative subsets of chemical data not only reduces computational costs associated with hyperparameter training but also serves as an innovative means to train non-linear DR methods, leading to projections that better preserve the local structure within the chemical space.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.03.663077v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ANABAG: Annotated Antibody Antigen dataset with unique features for Antibody Engineering Applications</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Grandguillaume, I., Etchebest, c., Barroso da Silva, F. L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present ANABAG, a new curated dataset of Ab-Ag complexes annotated at the residue level with UniProt sequence information and enriched with a wide range of structural and physicochemical features.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The analysis and prediction of antibody-antigen (Ab-Ag) interactions often overlook critical structural features such as glycosylation, physical chemical conditions like pH and salt concentration, as well as the lack of standardized criteria for selecting complexes based on structural properties and sequence identity. Common practices in dataset construction rely on removing redundancy using sequence identity thresholds, which can inadvertently exclude complexes with alternative binding modes that share identical sequences. To enable more precise Ab-Ag modeling and antibody engineering, it is essential to incorporate richer structural and physical information into both physics based and machine learning models. To address these limitations, we present ANABAG, a new curated dataset of Ab-Ag complexes annotated at the residue level with UniProt sequence information and enriched with a wide range of structural and physicochemical features. The dataset allows flexible filtering of complexes using a variety of descriptors available at both the complex and residue levels. Selected features are ready to use in machine learning workflows, while the structural files are compatible with antibody design and docking pipelines like Rosetta or Haddock. The complete dataset is available on Zenodo, and all accompanying scripts and usage documentation can be accessed via GitHub at https://github.com/DSIMB/anabag-handler.git.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.03.663065v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Competitions for tyrosine breakdown: In synthetic microbial communities and between a gut microbial pathway and a human pathway</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mitchem, M., Pauling, C. W., Shah, D. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present various metabolic outcomes of tyrosine within synthetic microbial communities which are prominently driven by specific enzyme activities of the human tyrosinesine breakdown pathway.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Tyrosine, a versatile amino acid that undergoes diverse transformations to produce both beneficial and detrimental metabolites. The modulation of these metabolites results from direct competition among different metabolic pathways responsible for the breakdown of tyrosine whether it be the competition between distinct microbes or the rivalry between a microbe and its host. The fight between microbes for the available tyrosine might drive potential changes to the communities present in various environments. In contrast, if the similar contest for tyrosine is presented between a gut microbial pathway and a human pathway, it can hold potential to affect the human health. In this work, we present various metabolic outcomes of tyrosine within synthetic microbial communities which are prominently driven by specific enzyme activities of tyrosine breakdown pathways. Additionally, we developed a metabolic diversion of the human tyrosine breakdown pathway facilitated by a gut microbial enzyme. This approach holds promise as a novel strategy to develop potential therapeutic interventions in future for addressing metabolic disorders like tyrosinemias (I, II, III), hawkinsinuria, and alkaptonuria, associated with the human tyrosine breakdown pathway.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.07.663581v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Validation of Morphology-Guided Computational Enhancement for Single-Cell Resolution Spatial Transcriptomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Elbialy, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Spatial transcriptomics technologies face a fundamental trade-off between transcriptomic breadth and spatial resolution, with widely-used platforms like 10x Visium capturing multiple cells per spot, limiting single-cell insights.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Spatial transcriptomics technologies face a fundamental trade-off between transcriptomic breadth and spatial resolution, with widely-used platforms like 10x Visium capturing multiple cells per spot, limiting single-cell insights. Current computational deconvolution methods attempt to address this limitation but uniformly suffer from reference dependency, platform effects, and complete neglect of tissue morphology. Here we present SpatialCell AI, a computational framework that achieves true single-cell resolution from spot-based spatial transcriptomics through morphology-guided computational enhancement. Unlike existing methods that rely solely on expression similarity, SpatialCell AI integrates AI-powered cell segmentation from histological images with spatial gene expression, eliminating reference requirements while leveraging tissue architecture. We rigorously validated our approach using publicly available matched colorectal cancer samples analyzed across Visium (55m), Visium HD (8m, 16m), and Xenium (single-cell ground truth). SpatialCell AI achieved strong accuracy with expression correlation of r=0.791, 7.82-fold improvement in expression accuracy, and 5.30-fold enhancement in gene detection compared to spot-based measurements. Comprehensive benchmarking against 28+ existing methods revealed that all computational approaches share identical limitations that SpatialCell AI uniquely overcomes through its morphology-first design. The framework converts standard Visium outputs from spot-level to true single-cell resolution (Cell_1, Cell_2, Cell_3...), enabling precise cellular interaction mapping and rare cell type identification previously impossible with spot-based technologies. By bridging the resolution gap between affordable spot-based platforms and expensive single-cell technologies, this approach enables broader access to single-cell resolution analysis for research and clinical applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.03.663047v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Evaluating Molecular Docking Programs for RNA-Targeted Ligand Screening: Influence of Binding Modes and Ligand Types</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, K., Jaladanki, C. K., DeFalco, L., Huber, R. G., Chai, C. L. L., Fan, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We evaluated the efficacy of three widely used docking programs for RNA-ligand docking screens and found that the average scores across docking programs and RNA structures yielded stable performance when compared to individual methods.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Structured RNAs are critical regulators of gene expression and represent promising targets for drug discovery. Despite this potential, structure-based virtual screening approaches, in particular, molecular docking, have been underutilized for RNA targets, largely due to the intrinsic conformational flexibility of RNA and the limited docking methods specifically optimized for RNA. In our study, we evaluated the efficacy of three widely used docking programs for RNA-ligand docking screens. The performance of these programs, Glide, Gold, and rDock, was evaluated using a dataset of 25 diverse RNA targets with 432 small molecule RNA binders. We found that the performance was similar across docking methods, but consistent differences emerged among RNA classes. We further explored consensus scoring across both receptor conformations and docking methods, finding that selecting the average scores across docking programs and RNA structures yielded stable performance when compared to individual methods. Finally, clustering analysis of RNA binders revealed distinct scoring function preferences for different chemotypes of RNA binders, underscoring the potential for tailored scoring strategies in RNA-targeted drug discovery. Together, this work provides a comprehensive evaluation of the performance of available docking methods in different cases of RNA-ligand docking and offers practical guidance for improving virtual screening against structured RNAs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.03.661502v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DrugDomain2.0: comprehensive database of protein domains-ligands/drugs interactions across the whole Protein Data Bank</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Medvedev, K. E., Schaeffer, R. D., Grishin, N. V.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present DrugDomain v2.0, the first comprehensive database linking evolutionary domain classifications (ECOD) to ligand binding events across the Protein Data Bank, and extend domain-ligand annotations to human drug targets.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Proteins carry out essential cellular functions - signaling, metabolism, transport - through the specific interaction of small molecules and drugs within their three-dimensional structural domains. Protein domains are conserved folding units that, when combined, drive evolutionary progress. The Evolutionary Classification Of protein Domains (ECOD) places domains into a hierarchy explicitly built around distant evolutionary relationships, enabling the detection of remote homologs across the proteomes. Yet no single resource has systematically mapped domain-ligand interactions at the structural level. To fill this gap, we introduce DrugDomain v2.0, the first comprehensive database linking evolutionary domain classifications (ECOD) to ligand binding events across the entire Protein Data Bank. We also leverage AI-driven predictions from AlphaFold to extend domain-ligand annotations to human drug targets lacking experimental structures. DrugDomain v2.0 catalogs interactions with over 37,000 PDB ligands and 7,560 DrugBank molecules, integrates 6,000+ small-molecule-associated post-translational modifications, and provides context for 14,000+ PTM-modified human protein models featuring docked ligands. The database encompasses 43,023 unique UniProt accessions and 174,545 PDB structures. The DrugDomain data is available online: https://drugdomain.cs.ucf.edu/ and https://github.com/kirmedvedev/DrugDomain.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.03.663025v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

