<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Wednesday, 01 October 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Computational Analysis of Exosome-Derived Signature in TNBC: Integrating Single-Cell and Bulk Transcriptomics for Prognosis Prediction.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang Y, Zhao M, Hou L, Jin L, Bai J, Dang Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We derive an exosome-Derived Prognostic Signature (EDPS) for triple-negative breast cancer and show that low EDPS scores are associated with poorer clinical outcomes, higher immune infiltrates, and immune checkpoint expression, suggesting better immunotherapy outcomes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Triple-negative breast cancer (TNBC) is a particularly aggressive subtype of breast cancer with limited targeted therapeutic options. Exosomes, small membrane vesicles secreted by cells, play a crucial role in intercellular communication and material exchange. However, the role of exosome-related genes (ERGs) in TNBC remains unclear. In here, we analyzed single-cell RNA sequencing (scRNA-seq) from 10 TNBC samples and bulk RNA-seq from TCGA and METABRIC cohorts. Starting with 121 EDPS curated from the breast cancer-specific ExoBCD database, we identified exosome-active cell populations and derived an Exosome-Derived Prognostic Signature (EDPS) through integrative machine learning. Our analysis identified 31,140 cells from TNBC samples, categorized into nine cell types, with epithelial cells exhibiting the highest exosome-related scores. A total of 232 differentially expressed genes (DEGs) related to exosome-related scores were identified, with 19 prognostic genes selected through univariate Cox regression, leading to the construction of an EDPS. Low EDPS scores correlated with poorer clinical outcomes, higher immune infiltrates, and immune-related pathways. Furthermore, we identified notable differences in biological functions and mutation profiles between the two EDPS groups. Additionally, the low EDPS score group exhibited lower tumor immune dysfunction and exclusion (TIDE) scores, immunophenoscore (IPS), and higher immune checkpoint expression, suggesting better immunotherapy outcomes. In conclusion, while derived from exosome-related genes, the EDPS primarily reflects immune-active tumor microenvironments. This signature may help identify TNBC patients likely to benefit from immunotherapy, though further validation of its relationship to exosome biology is needed.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40994257/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Radiation therapy response prediction for head and neck cancer using multimodal imaging and multiview dynamic graph autoencoder feature selection.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Moslemi A, Osapoetra LO, Safakish A, Sannachi L, Alberico D, Czarnota GJ.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We proposed an MVFS technique named Adaptive Graph Autoencoder Multi-View Feature Selection (AGAMVFS), based on dynamic graph learning and autoencoding, to train predictive models to predict outcomes of radiation therapy for head and neck (H&N) cancer.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> External beam radiation therapy is a common treatment for head and neck (H&N) cancers. Radiomic features derived from biomedical images have shown promise as effective biomarkers used to assess tumor heterogeneity and predict response to treatment. However, most studies employ only a single biomedical imaging modality to determine radiomic features or naively concatenate radiomic features from different imaging modalities.The objective of this study is to assess the effectiveness of multiview feature selection (MVFS) in identifying the most discriminative radiomic features determined from pretreatment quantitative ultrasound spectroscopic (QUS) parametric maps, as well as computed tomography (CT), and magnetic resonance imaging (MRI) modalities. These features were used to train predictive models to predict outcomes of radiation therapy for head and neck (H&N) cancer.70, 70, and 350 radiomics features were extracted from pre-treatment CT and MRI images, as well as seven QUS parametric maps, respectively. We proposed an MVFS technique named Adaptive Graph Autoencoder Multi-View Feature Selection (AGAMVFS), based on dynamic graph learning and autoencoder. In AGAMVFS, adaptive and collaborative graphs are learned at multiple levels to discriminate among view-specific features. An autoencoder is then applied to concatenated features to select the most discriminative ones. This approach fosters collaboration across different views and achieves a consensus projection for feature selection. Leave-one-patient-out cross-validation was applied to split the data into train and test sets and selected features were used to train two classifiers (support vector machine (SVM) and k-nearest neighbor (KNN)) to build a predictive model, tasked with predicting response to treatment for patients with H&N cancers. Fivefold cross-validation was applied on training set to tune the hyperparameters of SVM and KNN classifiers. Consequently, the performance of classifiers was evaluated using accuracy, F1-score, balanced accuracy, sensitivity, and specificity metrics. Additionally, a two-sided t-test was applied to the selected features. We compared the proposed method with a single imaging modality and state-of-the-art feature selection techniques.We recruited 63 (59 male (94%) and 4 female (6%)) H&N cancer patients with bulky metastatic neck lymph node (LN) involvement. The mean age was 58.9 ± 10.2 years. The AGAMVFS with the SVM classifier obtained the best performance and achieved 76% sensitivity, 91% specificity, 85% accuracy, and 83% balanced accuracy. Results showed the effectiveness of proposed method with superiority over other feature selection techniques. The most top-10 frequent features were six QUS radiomics, three MRI radiomics, and one CT radiomics features.The results demonstrated that the proposed predictive model is able to predict H&N cancer treatment response. MVFS provided better interpretabilityfor analysing features and preserved the inter-correlation among features from different imaging modalities.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40983829/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integration of Genetic Information to Improve Brain Age Gap Estimation Models in the UK Biobank.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mohite A, Ardila K, Charatpangoon P, Munro E, Zhang Q, Long Q, Curtis C, MacDonald ME.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new method for neurodegeneration age gap estimation based on genetic information, which can significantly reduce unexplained variance in the BrainAGE model.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Neurodegeneration occurs when the body's central nervous system becomes impaired as a person ages, which can happen at an accelerated pace. Neurodegeneration impairs quality of life, affecting essential functions, including memory and the ability to self-care. Genetics play an important role in neurodegeneration and longevity. Brain age gap estimation (BrainAGE) is a biomarker that quantifies the difference between a machine learning model-predicted biological age of the brain and the true chronological age for healthy subjects; however, a large portion of the variance remains unaccounted for in these models, attributed to individual differences. This study focuses on predicting the BrainAGE more accurately, aided by genetic information associated with neurodegeneration. To achieve this, a BrainAGE model was developed based on MRI measures, and then the associated genes were determined with a Genome-Wide Association Study. Subsequently, genetic information was incorporated into the models. The incorporation of genetic information yielded improvements in the model performances by 7% to 12%, showing that the incorporation of genetic information can notably reduce unexplained variance. This work helps to define new ways of determining persons susceptible to neurological aging decline and reveals genes for targeted precision medicine therapies.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40974298/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Chemosensory tobacco product toxicology part 2: toxicological testing, assays, and state of the science.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lin W, Hill T, Stroup AM, Sarles SE, Ogura T, Augustine F, O'Sullivan S, Rahman I, Robinson R, Jabba SV, Nuss C, Hensel E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> The toxicologic impacts on the normative function of the chemosensory system and the loss of its contribution to organism protection and homeostasis remain an underrepresented area of interest in the published literature.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The toxicologic impacts on the normative function of the chemosensory system and the loss of its contribution to organism protection and homeostasis remain an underrepresented area of interest in the published literature. The impact of chemical constituents in electronic nicotine delivery system e-liquids or aerosols on the chemosensory system is even less known, as are the effects on product selection and use behavior-and this may be an overlooked impact on the public health. This review is a snapshot of the current state of the science and opportunities for improving and increasing the volume of publications in chemosensory toxicology on the potential impacts of tobacco products. The proposed solutions rely on the determination of the scientific community to take advantage of an unexplored field of opportunity. Active research engagement and use of an integrative, risk-driven planning framework to address harmonization and data gaps in neurosensory research programs would support harmonization, improve scientific visibility in the published literature, and recruit additional investigators to this research community.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40705669/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CRISPR-driven diagnostics: Molecular mechanisms, clinical efficacy and translational challenges.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang Z, Wang Q, Zhang J, Li B, Li Y, Chen Z, Guo D, Feng S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> This article reviews the recent research progress in the CRISPR/Cas system for detecting nucleic acids, with an emphasis on CRISpl/Cas9, CRISpr/Cas12, and CRISP/Cas13.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In the realm of public health, among the primary perils menacing human well-being, the issue of pathogen infection persists as a significant concern. Precise and timely diagnosis of diseases constitutes the bedrock for effective therapeutic interventions and epidemiological monitoring. Hence, it is crucial to develop quick, sensitive, and highly effective methods for identifying pathogen and their variants.This article reviews the recent research progress in the CRISPR/Cas system for detecting nucleic acids, with an emphasis on CRISPR/Cas9, CRISPR/Cas12, and CRISPR/Cas13. Initially, we provided a concise overview of the nucleic acid detection mechanism utilizing the CRISPR/Cas system. Subsequently, we dissect the molecular mechanisms of CRISPR tools, compare their clinical efficacy against traditional methods, and explore frontier innovations such as amplification-free detection and AI integration.Ultimately, we argue that CRISPR diagnostics must evolve beyond technical optimization to embrace ecological adaptability, ensuring that precision medicine serves as a bridge-rather than a barrier-to global health equity.Core Mechanism: Explains the molecular basis of CRISPR-Cas (Cas9, Cas12, Cas13) for nucleic acid detection, leveraging crRNA-guided targeting and trans-cleavage activity for ultra-sensitive (aM level) and specific pathogen identification. Superior Performance: Outperforms traditional methods in speed, sensitivity, and cost, making it ideal for point-of-care use in resource-limited settings. Cutting-Edge Innovations: Covers key advances like amplification-free detection, portable device integration, and multiplex platforms. Translation Challenges: Discusses hurdles in clinical adoption, including inhibitor interference in complex samples, scalability limitations, the need for multi-center clinical data, and varying regional regulations. Future Outlook: Highlights emerging directions such as integrated "sample-to-result" systems and AI integration, while also addressing associated biosafety and ethical concerns, calling for robust regulatory frameworks.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41000010/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Tertiary Lymphoid Structures Across Organs: Context, Composition, and Clinical Levers.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Guillaume SM, Beccaria CG, Iannacone M, Linterman MA.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We examine the formation, architecture, and function of TLSs in non-lymphoid tissues and explore their clinical relevance in infections, autoimmunity, cancer, and allergy.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Tertiary lymphoid structures (TLSs) arise in non-lymphoid tissues in response to persistent antigen stimulation and chronic inflammation. Spanning organs from lung and liver to meninges, skin, and beyond, TLSs range from loose aggregates of immune cells to fully mature structures containing functional germinal centers (GC). In this review, we provide a comprehensive overview of TLS formation, architecture, and function across diverse tissues, highlighting both shared features and tissue-specific adaptations. We then explore the clinical relevance of TLS in infections, autoimmunity, cancer, and allergy, emphasizing their dual roles in mediating protective immunity and driving pathology. Finally, we discuss emerging technologies that are transforming our ability to dissect TLSs at high resolution (including spatial multi-omics, advanced imaging, and digital pathology), enabling mechanism-guided strategies to modulate TLSs therapeutically. Framing TLSs through the lens of maturation and tissue context provides a foundation for interpreting their clinical associations and for enhancing or dismantling these niches according to need.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40996881/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The peptide LyeTx I mnΔK induces transcriptomic reprogramming in a novel Multidrug-resistant  Acinetobacter baumannii</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Oliveira, F. G. d. C., Barros, K. d. O., Vianei, D. d. O., Martins, J. R., Duarte, J. C., de Laet Souza, D., Mamede, I., Moreira, R. G., De Aguiar, R. S., da Silva, F. A., dos Santos, V. L., Varani, A. M., Batista, T. M., Machado, C. R., Rosa, C. A., de Lima, M. E., Franco, G. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new multidrug-resistant strain of Acinetobacter baumannii with antibiotic resistance genes and a novel synthetic peptide for antimicrobial peptides .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Acinetobacter baumannii is a critical pathogen in healthcare-associated infections, and treatment is challenging due to the emergence of multidrug-resistant strains. Antimicrobial peptides, such as LyeTx I mn{Delta}K, a synthetic peptide derived of a toxin from the spider Lycosa erythrognatha , represent a promising alternative due to their broad-spectrum activity and synergistic potential with antibiotics like meropenem. This study aimed to compare the genomes of several A. baumannii strains, including a novel multidrug-resistant A. baumannii isolate (AC37), and to evaluate the antimicrobial effects of LyeTx I mn{Delta}K- alone and in combination with meropenem- through transcriptomic analysis. Genome assembly and annotation of AC37 revealed 31 antibiotic resistance genes, and phylogenetic analysis comprising 123 A. baumannii genomes, including the reference strain, identified three unique resistant genes in the AC37 strain. Mobilome analysis showed 13 genes associated with mobile genetic elements, including two of the unique genes, highlighting horizontal gene transfer events. Transcriptomic profiling revealed that treatment with LyeTx I mn{Delta}K peptide alone induced several differentially expressed genes, including two efflux pump operons. Additionally, pathways related to protein synthesis, export, and secretion were activated, indicating a broader cellular response to the peptide. The treatment with LyeTx I mn{Delta}K in combination with meropenem disrupted oxidative phosphorylation, further revealing the metabolic plasticity of the bacterial response to external stresses. This study characterizes a new A. baumannii isolate and provides new insights into the bacterial response to a potential novel therapeutic molecule.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.29.679197v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Synthetic collinearization of a chromosome segment enables genetic dissection of incompatibility between distant genera</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yang, S. H., Coradini, A. L. V., Hull, C. B., Lusk, D. T., Ehrenreich, I. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that collinearization can facilitate the creation of chimeric genomes to dissect the genetic basis of incompatibility and traits between organisms that cannot naturally hybridize.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Reproductive barriers and genomic structural differences hinder genetic analysis between highly divergent organisms. Here, we examined whether these challenges can be overcome by collinearization, the synthesis of chromosomes that retain the native gene content and organization of a host organism while incorporating the DNA sequences of another organism. We applied collinearization to Kluyveromyces marxianus, a yeast species that is both distantly related to and has a distinct genome structure from the model yeast Saccharomyces cerevisiae. We generated a ~35-kb K. marxianus DNA segment that was collinear with one-sixth of a S. cerevisiae chromosome and contained 17 protein-coding genes. Although this synthetic chromosome segment successfully substituted for its native counterpart in an S. cerevisiae cell, it imposed a significant growth cost due to the incompatibility of two K. marxianus proteins with the host proteome. We predicted the disrupted protein-protein interactions using AlphaFold and alleviated their cost by supplementing orthologous protein partners from K. marxianus. Furthermore, we completely eliminated the growth cost by replacing the two incompatible K. marxianus genes with their S. cerevisiae orthologs. These findings demonstrate that collinearization can facilitate the creation of chimeric genomes to dissect the genetic basis of incompatibility and traits between organisms that cannot naturally hybridize. They also suggest that hundreds of genetic incompatibilities exist between S. cerevisiae and K. marxianus, reflecting disrupted protein-protein interactions that may be predictable in silico.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.29.679105v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Programming human cell type-specific gene expression via an atlas of AI-designed enhancers</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Castillo-Hair, S. M., Yin, C. H., VandenBosch, L., Cherry, T. J., Meuleman, W., Seelig, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We trained deep neural networks on a large corpus of chromatin accessibility data from hundreds of human biosamples to generate tens of thousands of synthetic enhancers, targeting hundreds of cell lines, tissues, and differentiation states, aiming to maximize accessibility in target samples and minimize it in off-target ones.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Differentially active enhancers are key drivers of cell type specific gene expression. Active enhancers are found in open chromatin, which can be mapped at genome scale across tissue and cell types. Though incompletely understood, the relationship between chromatin accessibility and enhancer activity has been exploited to identify, model, and even design functional enhancers for selected cell types, but to what extent this design strategy can generalize across human cell and tissue types remains unclear. Here, we trained deep neural networks on a large corpus of chromatin accessibility data from hundreds of human biosamples. We used these models to generate an atlas of tens of thousands of synthetic enhancers, targeting hundreds of cell lines, tissues, and differentiation states, aiming to maximize accessibility in target samples and minimize it in all off-target ones. Experimental testing of thousands of designs in a representative subset of ten human cell types and in mouse retina demonstrated their function as specific enhancers, not only in the case of one-versus-all objectives but also when targeting two or three cell types. An explainable AI analysis, enabled by our large-scale enhancer measurements, allowed us to identify similarities and differences between the sequence grammar underlying accessibility and enhancer activity. Our results show that model-guided design of enhancers can help us decipher the cis-regulatory code governing cell type specificity and generate novel tools for selective targeting of human cell states.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.30.679565v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Advanced pipeline for CRISPR/Cas9 offtargets detection in Guide-seq and related integration-based assays</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Corre, G., Rouillon, M., Mombled, M., Amendola, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a rapid and versatile single-command pipeline designed for the comprehensive analysis of GuideSeq and similar techniques of sequencing, and a built-in tool for off-target site prediction.</p><p style="font-size: 16px; text-align: justify;">Background: The advent of CRISPR-Cas9 genome editing has brought about a paradigm shift in molecular biology and gene therapy. However, the persistent challenge of off-target effects continues to hinder its therapeutic applications. Unintended genomic alterations can lead to significant genomic damage, thereby compromising the safety and efficacy of CRISPR-based therapies. Although in-silico prediction tools have made substantial progress, they are not sufficient for capturing the complexity of genomic alterations and experimental validation remains crucial for accurate identification and quantification of off-target effects. In this context, Genome-wide Unbiased Identification of Double-strand breaks Enabled by Sequencing (GUIDE-Seq) has emerged as a gold standard method for the experimental detection of off-target sites and assessment of their prevalence by introducing short double-stranded oligonucleotides (dsODNs) at the break sites created by the nuclease. The bioinformatic analysis of GUIDE-Seq data plays a pivotal yet challenging role in accurately mapping and interpreting editing sites and current pipelines suffer limitations we aim to address in this work. Results: In this study, we present a rapid and versatile single-command pipeline designed for the comprehensive analysis of GuideSeq and similar techniques of sequencing. Our pipeline is capable of simultaneously processing multiplexed libraries from different organisms, PCR orientations, and Cas with different PAM specificities in a single run, all based on user-specified sample information. To ensure reproducibility, the pipeline operates within a closed environment and incorporates a suite of well-established bioinformatics tools. Key novel features include the ability to manage bulges in gDNA/gRNA interaction and multi-hit reads, and a built-in tool for off-target site prediction. The pipeline generates a detailed report that consolidates quality control metrics and provides a curated list of off-target candidates along with their corresponding gRNA alignments. Conclusions: Our pipeline has been tested and successfully applied to analyze samples under a variety of experimental conditions, including different source organisms, PAM motifs, dsODN sequences and PCR orientations. The robustness and flexibility of our pipeline make it a valuable tool for researchers in the field of genome editing. The source code and comprehensive documentation are freely accessible on our GitHub repository: https://github.com/gcorre/GNT_GuideSeq.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.30.679427v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Distilling Direct Effects via Conditional Differential Gene Expression Analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gu, J., Skelton, A., Staley, J., Popson, P. O., Peng, L., Song, X., Knowles, J. K., He, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce conditional differential gene expression (CDGE) analysis, a framework designed to identify direct effect genes, those whose changes in expression causally and directly impact downstream biological processes of interest.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding gene expression levels is crucial for comprehending gene functions, gene-gene interactions and disease mechanisms. Differential gene expression (DGE) analysis is a widely used statistical approach that offers insights by comparing gene expression across various conditions. However, traditional DGE methods focus on what are known as marginal associations, which refer to correlations observed between gene expression and a trait of interest, even if that association is indirect or not causal. To address this limitation, we introduce conditional differential gene expression (CDGE) analysis, a framework designed to identify direct effect genes. Direct effect genes are those whose changes in expression causally and directly impact downstream biological processes of interest. In applications to three RNA sequencing datasets (including one genome-scale perturb-seq dataset), CDGE analysis identifies that only a small fraction of differentially expressed genes has direct effects and mediate most other gene actions. These direct effect genes offer greater biological insight in enrichment analyses involving protein interactions and pathways. This suggests that CDGE yields more informative conclusions on causal gene effects and could become a key tool for studying biological pathways.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.29.678272v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Systematic comparative benchmarking of computational methods for the detection of transposable elements in long-read sequencing data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Seymen, N., Santos, R., Lakshmanan, R., Topp, S., Al-Chalabi, A., Al Khleifat, A., Breen, G., Dobson, R. J., Quinn, J. P., Karimi, M. M., Iacoangeli, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A benchmarking study of state-of-the-art long-read TE detection methods for mobile element insertions .</p><p style="font-size: 16px; text-align: justify;">Background Mobile element insertions, particularly transposable elements (TEs) such as Alu, LINE-1 (L1), SVA, and endogenous retroviruses (ERVs), represent a major source of human genetic variation and have been implicated in evolution, genomic instability, and disease. Although long-read sequencing generally outperforms short-read sequencing for the characterisation of such elements, their accurate detection with long-reads remains challenging, with different computational tools adopting varying approaches and producing divergent call sets. As gold standards currently do not exist for TE detection, benchmarking these methods is essential to understand their strengths, limitations, and biases. Here, we systematically evaluate the performance of available state-of-the-art TE detection tools on both simulated and real human genome data using highly characterised samples from the Genome in a Bottle consortium, population level reference databases and an in-house collection for which matching short-read sequencing data are available. Results Our results show significant differences in calling strategies, leading to substantial variation in precision, recall, and the spectrum of TE families detected across tools. Our benchmark also displays the differences between short-read and long-read calls, highlighting the importance of appropriate method selection. Conclusions The benchmarking results presented here will aid TE researchers make better informed decisions on which tool to use in their long-read TE analyses. Strengths and limitations of different tools have been highlighted in depth as well as their computational requirements, which will result in less time spent finding the best tool for the job and promote faster TE research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.29.679192v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Alice: fast and haplotype-aware assembly of high-fidelity reads based on MSR sketching</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Faure, R., Hilaire, B., Flot, J.-F., Lavenier, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Mapping-friendly Sequence Reduction (MSR) sketches, a sketching method for high-fidelity (HiFi) long reads, and Alice, an assembler that operates directly on these sketches.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We introduce Mapping-friendly Sequence Reduction (MSR) sketches, a sketching method for high-fidelity (HiFi) long reads, and Alice, an assembler that operates directly on these sketches. MSR produces compact representations that (i) are alignable sequences - two sequences align if and only if their MSR sketches align - and (ii) are collision-resistant, so distinct sequences yield distinct sketches with high probability, retaining small differences between closely related strains. Alice reduces long reads to short MSR sketches, uses a classic short-read assembly method to assemble those sketches and decompresses the result to obtain the final assembly. This strategy addresses the longstanding challenge of producing a strain-resolved assembly for a low computational cost. On an Adineta vaga genome, a mock gut community comprising five conspecific strains, and two real metagenomes (human stool and soil), Alice is an order of magnitude faster than state-of-the-art HiFi assemblers while delivering assemblies of comparable quality and improving recovery of highly similar strains.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.29.679204v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Label-free biochemical imaging and timepoint analysis of neural organoids via deep learning-enhanced Raman microspectroscopy</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Georgiev, D., Xie, R., Reumann, D., Zhao, X., Fernandez-Galiana, A., Barahona, M., Stevens, M. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a label-free, label- free imaging platform that integrates Raman microspectroscopy with deep learning-based hyperspectral unmixing for unsupervised, spatially resolved biochemical analysis of neural organoids.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Three-dimensional organoids have emerged as powerful models for studying human development, disease and drug response in vitro. Yet, their analysis remains constrained by standard imaging and characterisation techniques, which are invasive, require exogenous labelling and offer limited multiplexing. Here, we present a non-invasive, label-free imaging platform that integrates Raman microspectroscopy with deep learning-based hyperspectral unmixing for unsupervised, spatially resolved biochemical analysis of neural organoids. Our approach enables high-resolution mapping of cellular and subcellular structures in both cryosectioned and intact organoids, achieving improved imaging accuracy and robustness compared to conventional methods for hyperspectral analysis. Using our platform, we demonstrate volumetric imaging of a neural rosette within a neural organoid, and interrogate changes in biochemical composition during early developmental stages in intact neural organoids, revealing spatiotemporal variations in lipids, proteins and nucleic acids. This work establishes a versatile framework for high-content, label-free (bio)chemical phenotyping with broad applications in organoid research and beyond.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.29.679057v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A statistical framework for defining synergistic anticancer drug interactions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dias, D., Zobolas, J., Ianevski, A., Aittokallio, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We provide a fast and statistically rigorous approach to detecting synergistic drug interactions in combinatorial screens.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Synergistic drug combinations have the potential to delay drug resistance and improve clinical outcomes. However, current cell-based screens lack robust statistical assessment to identify significant synergistic interactions for downstream experimental or clinical validation. Leveraging a large-scale dataset that systematically evaluated more than 2,000 drug combinations across 125 pan-cancer cell lines, we established reference null distributions separately for various synergy metrics and cancer types. These data-driven reference distributions enable estimation of empirical p-values to assess the significance of observed drug combination effects, thereby standardizing synergy detection in future studies. The statistical evaluation confirmed key synergistic combinations and uncovered novel combination effects that met stringent statistical criteria, yet were overlooked in the original analyses. We revealed cell context-specific drug combination effects across the tissue types and differences in statistical behavior of the synergy metrics. To demonstrate the general applicability of our approach to smaller-scale studies, we applied the reference distributions to evaluate the significance of combination effects in an independent dataset. We provide a fast and statistically rigorous approach to detecting synergistic drug interactions in combinatorial screens.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.29.679166v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MitoNGS: an online platform to analyze fish metabarcoding data in high-resolution</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhu, T., Sato, Y., Fukunaga, T., Miya, M., Iwasaki, W., Yoshizawa, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present MitoNGS, a next-generation platform that succeeds the widely used MiFish pipeline, designed for high-resolution analysis of fish metabarcoding data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Environmental DNA (eDNA) metabarcoding has become a powerful tool for assessing fish biodiversity in aquatic ecosystems. However, accurate species-level identification remains challenging due to incomplete and contaminated reference databases, as well as ambiguous taxa sharing identical barcode sequences. Here, we present MitoNGS, a next-generation platform that succeeds the widely used MiFish pipeline, designed for high-resolution analysis of fish metabarcoding data. MitoNGS addresses these challenges by incorporating more comprehensive references including non-fish species and detailed annotations of heterospecific regions. Additionally, it introduces the "species group" strategy in conjunction with environmental habitat and geographic occurrence data to resolve ambiguous taxa. Furthermore, MitoNGS expands the functionalities of the legacy MiFish pipeline. It can analyze data from any mitochondrial markers and from Nanopore sequencing platforms. MitoNGS demonstrated excellent performance on our testing datasets from diverse locations, markers and sequencing platforms. MitoNGS offers a user-friendly, web-based solution for fish detection, biodiversity monitoring, conservation research, and bioresource management. MitoNGS is freely available via https://mitofish.aori.u-tokyo.ac.jp/mito-ngs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.29.679108v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Targeting SOS1 synergistically enhances efficacy of BCR/ABL tyrosine kinase inhibitors and overcomes resistance in chronic myeloid leukemia</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Garcia-Navas, R., Gomez, C., Zamora-Valdivieso, B., Calvo-Jimenez, S., Calzada, N., Fernandez-Medarde, A., Sierra, M., Sanchez-Guijo, F., Schenk, R. L., Hofmann, M. H., Kostyrko, K., Santos, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We demonstrate the therapeutic impact of SOS1 inhibition by its specific pharmacological inhibitor BI-3406 as single-agent or in combination with TKIs like imatinib in chronic myelogenous leukemia (CML).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Disease persistence and therapeutic resistance remain a significant challenge in chronic myelogenous leukemia (CML). Here, we evaluated the therapeutic impact of SOS1 inhibition by its specific pharmacological inhibitor BI-3406 as single-agent or in combination with BCR/ABL tyrosine kinase inhibitors (TKI) like imatinib in preclinical models of CML including p210BCR/ABL mice, human CML cell lines, and patient-derived bone marrow cells. In p210BCR/ABL mice, treatment with BI-3406 or imatinib was well-tolerated in vivo after single or combined use of the drugs. Treatment with imatinib alone significantly improved survival and corrected various hematological parameters of disease burden, while the combination with BI-3406 therapy yielded even more pronounced benefits, including a substantial increase in median survival, marked reductions in peripheral white blood cell and neutrophil counts, and a notable decrease in leukemia stem cells within the bone marrow. Additionally, the combination led to further spleen size reduction and restoration of normal splenic architecture. Human CML cell lines and primary cells from CML patients subjected to combined treatment with BI-3406 and imatinib or later-generation TKI drugs showed significantly reduced proliferation and enhanced apoptosis as compared to single-agent-treated cultures, revealing a strong synergistic therapeutic behavior of the BI-3406 +TKI combinations. Remarkably, the combined treatments including BI-3406 significantly restored imatinib sensitivity in CML patient cells harboring imatinib-resistant mutations. Cellular signaling and transcriptomics profiling suggested coordinated attenuation of RAS and RAC downstream signals as a mechanistic basis for the observed therapeutic responses. Our findings highlight the synergistic therapeutic behavior of BI-3406 and underscore the benefit of SOS1 pharmacological targeting as a novel strategy enhancing efficacy and overcoming resistance to TKIs in CML.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.29.679122v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Disrupting Notch signalling by a small molecule inhibiting dihydroorotate dehydrogenase activity</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Braune, E.-B., Wienke, D., Seshire, A., Heinrich, T., Haraldsson, M., Lain, S., Lendahl, U.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify potential Notch inhibitors, using a novel cell-based Notch reporter system for unbiased screening of compounds reducing Notch signalling.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The Notch signalling pathway is highly evolutionarily conserved and regulates differentiation and homeostasis in most organs. Given the critical role of Notch signalling for normal development, dysregulated Notch signalling is frequently linked to pathogenesis of disease and cancer. Hence, developing Notch-targeting therapeutics is warranted but has been challenging and Notch inhibitors have not yet reached broad clinical use. In this report, we identify potential Notch inhibitors, using a novel cell-based Notch reporter system for unbiased screening of compounds reducing Notch signalling. A library of 37.966 small organic compounds was screened for inhibitor candidates, followed by a counter screen to eliminate y-secretase inhibitor-like compounds and an orthogonal screen based on the role of Notch signalling in myogenic differentiation. This triage led to the identification of five Notch inhibitor candidate hits with different chemical backbones and unrelated to previous Notch antagonists. One candidate hit showed structural similarities to dihydroorotate dehydrogenase (DHODH) inhibitors, and we provide evidence that inhibition of DHODH activity reduces Notch signalling. In conclusion, our data support the notion that DHODH inhibition may be an interesting avenue to explore for the development of novel Notch inhibitors.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.29.679158v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Artificial intelligence in cancer care: revolutionizing diagnosis, treatment, and precision medicine amid emerging challenges and future opportunities.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chandrabose Selvaraj, William C Cho, Kulanthaivel Langeswaran, Abdulaziz S Alothaim, Rajendran Vijayakumar, Mani Jayaprakashvel, Deepali Desai</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Artificial intelligence applications in cancer care . . .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Artificial intelligence (AI) is increasingly being used in oncology to assist early detection, diagnosis, prognosis, treatment planning, and drug discovery. A systematic review is required to integrate evidence across various AI applications in cancer treatment. Systematically assess the use of AI applications in oncology, integrate study findings, highlight methodological issues, and set directions for future research. According to PRISMA guidelines, we searched systematically PubMed, Scopus, Web of Science, and IEEE Xplore between January 2013 and December 2024. Search terms integrated AI-related terms with oncology-related terms. Peer-reviewed original research studies with the use of AI on cancer care in human or human-derived datasets were the eligible studies. Two reviewers independently screened the studies, extracted data, and evaluated the risk of bias with suitable tools. 120 out of 4852 records were included according to inclusion criteria. Applications fell into five clusters: imaging/radiomics, genomics/biomarker discovery, drug discovery/repurposing, clinical decision support, and patient monitoring. Convolutional neural networks were predominant in imaging tasks, whereas ML classifiers were prevalent in genomics. Most of the studies showed improved performance with respect to conventional methods although most of the studies failed to conduct multi-center validation. Heterogeneity of data, interpretability limitations, and integration problems were common issues. AI holds great potential along the cancer care continuum but is at risk of being threatened by issues with data quality, validation, interpretability, and translation to practice. Addressing these issues will require collaboration among disciplines, reporting to standardized guidelines, and large-scale validation studies.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40978321/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">From histology to diagnosis: Leveraging pathology foundation models for glioma classification.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Camillo Saueressig, Claire Delbridge, Daniel Scholz, Azar Kazemi, Mohammad Zaid Khan, Marie Metz, Bernhard Meyer, Meike Mitsdoerffer, Peter J Schüffler, Benedikt Wiestler</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We demonstrate that computational pathology foundation models for glioma classification can be used to generate effective embeddings for downstream classification using three datasets (TCGA, n=839 samples; EBRAINS, n =786 samples; TUM, 250 samples).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The fifth edition of the WHO classification of brain tumors increasingly emphasizes the role of extensive genetic testing in the diagnosis of gliomas. In this context, computational pathology foundation models (FMs) present a promising approach for inferring molecular entities directly from conventional, H&E-stained histological images, potentially reducing the need for genetic analysis. We conducted a robust investigation into the ability of five established FMs to generate effective embeddings for downstream glioma classification using three datasets (TCGA, n=839 samples; EBRAINS, n=786 samples; TUM, n=250 samples) and state-of-the-art augmentation techniques. Our results demonstrate that FM embeddings enable competitive glioma classification performance, even with limited training data, achieving one-vs-rest AUC>0.93 on all three datasets. However, we observed substantial differences between FMs in their downstream performance, susceptibility to perturbations, and consistency across multiple datasets. Dataset diversity and content of central nervous tissue were associated with improved generalization, while model and dataset size were not. Common to all FMs was a propensity to capture dataset-specific features in their embeddings. We examined Macenko normalization and random convolutions as potential solutions to combat dataset-dominated embeddings and show that ensembling FM embeddings over multiple augmented views improves downstream classifier performance. In summary, our findings highlight both the promise and current limitations of computational pathology foundation models for glioma classification, emphasizing the critical roles of training data composition and downstream augmentation to achieve strong task performance.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40907260/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Radiomics-based prediction of HCC response to atezolizumab/bevacizumab.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Isaac Rodriguez, Abhinay Vellala, Timo Itzel, Jimmy Daza, Michael Vácha, De-Hua Chang, Manuel Debic, Michael T Dill, Max Seidensticker, Julia Mayerle, Stefan Munker, Stefan O Schoenberg, Lukas Müller, Peter R Galle, Arndt Weinmann, Dietmar Tamandl, Matthias Pinter, Bernhard Scheiner, Christel Weiss, Maciej Pech, Friedrich Sinner, Verena Keitel, Marino Venerito, Matthias Philip Ebert, Andreas Teufel, Matthias F Froelich</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A multimodal model combining clinical and radiomics data to predict the treatment response to atezolizumab/bevacizumaf in advanced HCC .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Advanced hepatocellular carcinoma (HCC) treatment has evolved with the introduction of atezolizumab/bevacizumab, showing improved outcomes over sorafenib. However, the response varies among patients, particularly between viral and non-viral etiologies. The present study aimed to develop and evaluate multimodal prediction models combining quantitative imaging and clinical markers to predict the treatment response in patients with HCC. Between March 2020 and May 2023, patients with advanced HCC treated with atezolizumab/bevacizumab were retrospectively identified from six centers in Germany and Austria. Patients underwent baseline contrast-enhanced liver MRI and follow-up imaging to assess the therapy response. Machine learning models, including RandomForestClassifier, were developed for radiomics, clinical and combined datasets. Hyperparameter tuning was performed using RandomizedSearchCV, followed by cross-validation to evaluate model performance. The study included 103 patients, with 70 achieving disease control (DC) and 33 experiencing disease progression (PD). Key findings included significant differences in treatment response and progression-free survival between the DC and PD groups. The radiomics model, using 14 selected features, achieved 73.1% accuracy and a receiver operating characteristic (ROC) area under the curve (AUC) of 0.635 for the test set. The clinical model, with 4 selected features, achieved 73% accuracy and a ROC AUC of 0.649 for the test set. The combined model showed improved performance, with 69% accuracy and a ROC AUC of 0.753 for the test set. Hyperparameter tuning further enhanced the accuracy of the combined model to 80.1% and the ROC AUC to 0.771 for the test set. In conclusion, the hybrid model combining clinical and radiological data outperformed individual models, providing improved predictions of response to atezolizumab/bevacizumab in patients with HCC.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40861099/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Immune cell transcriptional profiles from pre-vaccination peripheral blood predict immune response to preventative MUC1 cancer vaccine.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Daniel Y Yuan, Michelle L McKeague, Vineet K Raghu, Robert E Schoen, Olivera J Finn, Panayiotis V Benos</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A cancer preventative cancer vaccine for advanced colon cancer .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent advances in vaccine technology raise hopes for effective cancer preventative vaccines. The first clinical trials (single-arm NCT007773097; double-blind, placebo controlled randomized trial NCT02134925) of a non-viral cancer preventative vaccine were conducted in individuals with previous advanced colonic adenoma to test the safety and immunogenicity of the MUC1 tumor antigen vaccine. The vaccine was safe and strongly immunogenic in 43 %-25 % of participants. The lack of response in a significant number of participants suggested that the pre-malignant immune system may have already been exposed to some level of suppression, something previously reported only in cancer.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40815873/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Metabolic Reprogramming in Pheochromocytoma and Paraganglioma: Insights From Untargeted NMR Metabolomics Presurgical and Postsurgical intervention.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jashanpreet Kaur, Rakhi Pooja, Gurvinder Singh, Sanniya Middha, Ankit Tandon, Dinesh Kumar, Rama Walia</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that PPGLs undergo significant metabolic reprogramming following surgical resection, with partial normalization of metabolic pathways following tumor resection and the restorative effect of surgical intervention.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The aim of this study is to investigate the metabolic alterations associated with pheochromocytomas and paragangliomas (PPGLs) and the impact of surgical resection on the serum metabolome using untargeted nuclear magnetic resonance (NMR) metabolomics. For this, the study included 34 patients diagnosed with PPGLs. Pre-operative and postoperative serum samples were analyzed using 1D-proton NMR spectroscopy, and NMR spectral data were processed using Bruker software Topspin. The quantitative metabolic profiles were estimated using CHENOMX NMR-Suite, and multivariate data were analyzed using partial least squares discriminant analysis (PLS-DA) and orthogonal PLS-DA followed by random forest (RF) classification method (a machine learning approach). The multivariate analysis revealed distinct metabolic differences between pre-operative and postoperative samples with respect to normal control (NC) samples, indicating a metabolic shift following tumor resection. RF classification, with an out-of-bag error rate of 0.186, effectively distinguished between NC, presurgery, and postsurgery groups, underscoring the distinct metabolic states in PPGL and the restorative effect of surgical intervention. Pre-operative serum profiles of PPGL patients were characterized by decreased levels of key metabolites, including glucose, citrate, amino acids (glutamine, glycine, leucine, valine, tyrosine, and alanine), histidine, myo-inositol, and creatinine, suggesting altered energy metabolism, and amino acid catabolism induced by catecholamine excess. Postsurgical profiles showed partial metabolic restoration, with significant increases in proline, glutamate, and 3-hydroxybutyrate (3-HB) (p < 0.01), indicating normalization involving lipid oxidation and amino acid metabolism. Although plasma metanephrines normalized postsurgery, full biochemical recovery lagged, as metabolic profiles of postoperative patients remained distinct from healthy controls. In conclusion, the present untargeted NMR metabolomics revealed significant metabolic reprogramming in PPGL patients and captured the partial normalization of metabolic pathways following tumor resection. Metabolites such as proline, glutamate, and 3-HB emerged as potential biomarkers of treatment response. These findings underscore the utility of metabolomics to identify biomarkers for monitoring disease progression, assessing postsurgical recovery, and improving our understanding of PPGL pathophysiology.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40765177/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

