<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Wednesday, 25 June 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MetaGLIMPSE: Meta Imputation of Low Coverage Sequencing Data for Modern and Ancient Genomes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kumar, K. H., Rubinacci, S., Zoellner, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a novel meta-imputation method, MetaGLIMPSE, that combines estimates from multiple reference panels for low coverage sequencing imputation, which outperforms the best single panel imputation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The advent of efficient and accurate imputation for low coverage sequencing offers an unbiased alternative to SNP array imputation, increasing the accuracy of rare variant imputation across all populations. Since imputation accuracy generally increases with larger reference panel size and closer ancestry match between target and reference samples, leveraging imputation from multiple reference panels would facilitate better imputation accuracy; however, individual reference panel genotypes are often privacy protected. We present a novel meta-imputation method, MetaGLIMPSE, that combines estimates from multiple reference panels for low coverage sequencing imputation. We directly combine single panel imputation results using weights estimated for each individual and imputed marker. Across all our scenarios, MetaGLIMPSE outperforms the best single panel imputation for coverages 0.5x - 8x and across all minor allele frequencies, and for certain coverages and scenarios, performs equally to the combined panel imputation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.24.660721v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AlphaFold 3-enabled in silico exploration of PGAM1 interactions in cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Clarke, J. J., Colcombe, J. L., Rigden, D. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a model in which phosphorylation of PGAM1 at Y119 enables access to the active site for PEP, thus enhancing its enzymatic activation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cancer cell metabolism is commonly reprogrammed to favour glycolysis over oxidative phosphorylation, even under aerobic conditions, a phenomenon known as the Warburg effect. A key enzyme implicated in this shift is phosphoglycerate mutase 1 (PGAM1), which catalyses the conversion of 3-phosphoglycerate to 2-phosphoglycerate. The human enzyme is dependent on cofactor 2,3-bisphosphoglycerate which can phosphorylate and thereby activate the enzyme at histidine 11 (H11). A recently characterised moonlighting activity of pyruvate kinase M2 (PKM2), in its monomeric or dimeric forms, leads to phosphoenolpyruvate (PEP)-dependent phosphorylation of PGAM1 at the same position. Crucially, this phosphorylation is dependent on prior tyrosine 119 (Y119) phosphorylation of PGAM1 by Src kinase, itself activated by oncogenic growth factors. Using AlphaFold 3 (AF3), this study models the conformational changes induced by PGAM1 Y119 phosphorylation and investigates the molecular basis for its role in facilitating PEP-dependent phosphorylation at H11. Structural comparisons suggest that Y119 phosphorylation induces rearrangement of the C-terminal tail of PGAM1, opening the catalytic site around H11 to enable binding of phosphoenolpyruvate (PEP). Use of molecular docking (Webina, SwissDock, and DiffDock) found that AF3 generated models of PGAM1 with Y119 phosphorylation showed enhanced binding of PEP in comparison to non-phosphorylated PGAM1. However, extensive protein-protein docking (ClusPro, AF3 multimer generation) failed to identify configurations of PGAM1 and PKM2 where catalytic sites were proximal. Overall, this study supports a model in which phosphorylation of PGAM1 at Y119 enables access to the active site for PEP, thus enhancing its enzymatic activation. These findings underscore the critical role of post-translational modifications in modulating protein function and exemplify the utility of AF3 in predicting PTM-induced structural changes relevant to cancer metabolism.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.23.661104v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Corelative light and electron microscopy (CLEM) image registration by blood vessel axes alignment</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Van den Berghe, Y., Saeys, Y., Verhaege, D., Kremer, A., Vandenbroucke, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel method that focuses on aligning the medial axes of blood vessels in both the LM and EM modalities.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Correlative light and electron microscopy (CLEM) is a multi-modal microscopy approach that contains complementary information from light (LM) and electron microscopy (EM). CLEM image alignment can be a challenging task for both the biological expert as well as for existing automated methods. This work introduces a novel method that focuses on aligning the medial axes of blood vessels in both the LM and EM modalities. Evaluation of 3D CLEM image registration problems shows that this approach obtains an accurate alignment in an intuitive and robust way. This method holds potential as a valuable tool for global and local alignment of a subset of challenging CLEM image alignment problems, both as guidance in a manual setting or in a more automated workflow.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.23.661033v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Euktect: Enhanced Eukaryotic Sequence Detection and Classification in Metagenomes via the DNA Language Model</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Peng, Y., Ji, B., Wang, Y., Lu, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Euktect, a deep-learning-based toolbox for reliable, alignment-free classification of eukaryotic DNA sequences across different phylogenetic levels from metagenome datasets.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The current taxonomy classification of DNA sequences in metagenomics primarily relies on alignment against reference databases. However, the eukaryotic species in genome databases are insufficient, as there exist numerous unculturable eukaryotes within metagenomes. These limitations impede comprehensive functional and evolutionary analyses of eukaryotes across distinct environmental samples. To overcome these limitations, we created Euktect, a deep-learning-based toolbox for reliable, alignment-free classification of eukaryotic DNA sequences across different phylogenetic levels from metagenome datasets. Euktect achieves high accuracy in extracting eukaryotic sequences longer than 500bp from the assembled contigs of metagenomes, significantly outperforming existing methods. Furthermore, we developed an algorithm that integrates Euktect's predictions with existing tools to refine metagenome-assembled genomes (MAGs), substantially increasing the yield of high-quality eukaryotic MAGs for downstream analysis. Beyond eukaryotic detection, Euktect incorporates two specialized models: a high-precision classifier for fungal phyla and a hierarchical classifier that accurately identifies sequences from specific fungal genera (e.g., Candida) in diverse metagenomic samples. Significantly, this framework enables prediction of host disease status (e.g., inflammatory bowel disease) by linking eukaryotic sequence identification to clinical phenotypes through machine learning models. Collectively, Euktect enables accurate reconstruction and functional annotation of eukaryotic genomes from metagenomes at large scale, thereby empowering comprehensive utilization of sequenced eukaryotic species for downstream evolutionary and clinical research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.19.660294v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Giant viruses specific to deep oceans show persistent presence and activity</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liu, W., Nagasaka, K., Wu, J., Ban, H., Mimick, E., Meng, L., Neches, R. Y., Moniruzzaman, M., Yoshida, T., Nishimura, Y., Endo, H., Okazaki, Y., Ogata, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified 48 deep-sea-specific giant viruses (GVs) of the phyla Nucleocytoviricota and Mirusviricota, with six exclusively detected in the mesopelagic layer, which supports the scenario that adaptation to aphotic environments has occurred in multiple</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Giant viruses (GVs) of the phyla Nucleocytoviricota and Mirusviricota are large double-stranded DNA viruses that infect diverse eukaryotic hosts and impact biogeochemical cycles. Their diversity and ecological roles have been well studied in the photic layer of the ocean, but less is known about their activity, population dynamics, and adaptive strategies in the aphotic layers. Here, we conducted eight seasonal time-series samplings of the surface and mesopelagic layers at a coastal site in Muroto, Japan, and integrated 18S metabarcoding, metagenomic, and metatranscriptomic data to investigate deep-sea GVs and their potential hosts. The analysis identified 48 GV genomes including six that were exclusively detected in the mesopelagic layer. Notably, these mesopelagic-specific GVs showed persistent activity across seasons. To investigate the global deep-sea-specific GV distribution, we compiled GV reference genomic data from the OceanDNA MAG project and other resources, and analyzed 1,890 marine metagenomes. This revealed 101 deep-sea-specific GVs, distributed across the GV phylogenetic tree, indicating that adaptation to deep-sea environments has occurred in multiple lineages. One clade enriched with deep-sea-specific GVs included one GV identified in our Muroto sampling, which displayed a wide geographic distribution. Seventy-six KEGG orthologs and 74 Pfam domains were specifically enriched in deep-sea-specific GVs, encompassing functions related to the ubiquitin system, energy metabolism, and nitrogen acquisition. These findings support the scenario that distinct GV lineages have adapted to hosts in aphotic marine environments by altering their gene repertoire to thrive in this unique habitat.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.21.660676v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Sedimentary ancient DNA metagenomic analysis provides new insights into farming in central Norway from the Bronze Age to late Medieval period</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> La Torre, R., Kosel, P., Bieker, V. C., Bakke Westergaard, K., Lien Mykkeltvedt, J., Yildirim, K. L., Seiler, M., Philippsen, B., Macphail, R. I., Cannell, R. J., Halvorsen, L. S., Bronseth Lorentzen, A., Sauvage, R., Martin, M. D., Mjelva Breivik, H., Martin, S. L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of ancient genetic material from the archaeological site of Torgardsletta in central Norway, spanning from the Bronze Age through the Medieval period, reveals temporal shifts in plant, animal and microbial communities, reflecting human practices and climatic conditions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Sedimentary ancient DNA (sedaDNA) has been proposed as a key methodology for reconstructing paleoclimates and biodiversity over time. To a lesser extent, it has been explored as a complementary tool for reconstructing human-driven local environmental changes over time, such as those explored in open air archaeological sites. Our study employs a sedaDNA metagenomic approach to investigate land use and environmental change at the archaeological site of Torgardsletta in central Norway, spanning from the Bronze Age through the Medieval period. Stratigraphic sediment samples reveal temporal shifts in plant, animal and microbial communities, reflecting evolving human practices and climatic conditions. Progression through the layers indicate signs or land clearance, cultivation and animal husbandry in the region. Notably, a significant reduction in microbial and plant diversity during periods of climatic upheaval, such as the AD 536 to 540 volcanic event, correlates with landscape and societal adjustments. The findings demonstrate that sedaDNA complements traditional proxies, providing high-resolution insights into past land use, environmental interactions, and societal organization. The successful extraction of ancient genetic material underscores sedaDNA's potential to reconstruct dynamic prehistoric landscapes and anthropogenic impacts, offering further potential for understanding long-term human-environment relationships in Scandinavia.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.19.660509v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identifying Key Hub Genes that Attribute Varying Host Responses: A Longitudinal RNA-seq Analysis of SARS-CoV-2 Delta and Omicron Infections.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bajracharya, D., Jansen, R., Yan, C., Ramamoorthy, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study compares host immune responses to the Delta and Omicron variants of SARS-CoV-2, showing that Delta is associated with higher virulence but reduced severity.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Coronavirus disease 2019 (COVID19), caused by SARS-CoV-2, has severely impacted global health, with different Variants of Concern (VOCs) resulting in varied clinical outcomes. Among these, the Delta and Omicron variants have drawn significant attention due to their distinct characteristics. Delta is associated with higher virulence and Omicron has greater transmissibility but reduced severity. This study compares host immune responses to the Delta and Omicron variants using RNAseq data from infected primary human airway epithelial cells. Both variants triggered a robust antiviral innate immune response by 2 days post-infection (dpi). However, Omicron was found to elicit a more rapid immune response, showing pathway enrichment at 1 dpi. In contrast, Delta displayed no immune-related pathway activation within the first 24 hours, suggesting it may evade early immune detection, promoting increased viral replication. By 3 dpi, Delta induced a more aggressive immune response, particularly in pathways related to cell death and pro-inflammatory signaling, such as programmed cell death and regulation of cell death. Weighted gene co-expression network analysis revealed distinct immune-related genes: Delta infections were characterized by hub genes like MYD88 and IL1R1 involved in pro-inflammatory responses, HLA A & B, NLRC5 and PSMB9 involved in antigen presentation and TNSF10 and IFR1 involved in pro-apoptotic processes. Conversely, Omicron infections were marked by hub genes such as CXCL1 and CXCL8 involved in immune cells recruitment, MET and LYN involved in reducing hyper-inflammatory responses and maintaining immune balance, and IFI44, EIF2AK2 and IFIT5 responsible for the sensing of viral RNA among others.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.23.654375v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Host microRNA Target Prediction in SARS-CoV-2 and Hepatitis E Virus Genomes: Insights from RNAhybrid Analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tasnim, Z., Islam, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We predict the binding sites of three abundant human miRNAs within the genomes of SARS-CoV-2 (Bangladesh isolate OM967280.1) and Hepatitis E Virus genotype 1 (NC_001434.1).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> MicroRNAs (miRNAs) are key post-transcriptional regulators of gene expression that can influence viral replication and pathogenesis by binding to viral RNA genomes. This study uses RNAhybrid to predict binding sites of three abundant human miRNAs hsa-miR-155-5p, hsa-miR-21-5p, and hsa-let-7a-5p within the genomes of SARS-CoV-2 (Bangladesh isolate OM967280.1) and Hepatitis E Virus genotype 1 (NC_001434.1). Our results show stable hybridization events with minimum free energy (mfe) values ranging from -23.8 to -28.1 kcal/mol, indicating strong potential interactions. These findings suggest that host miRNAs may modulate viral RNA stability or translation, impacting viral pathogenesis. Further experimental validation is necessary to confirm these regulatory relationships and explore their therapeutic potential.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.23.661149v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Improving the Scalability of Bayesian Phylodynamic Inference through Efficient MCMC Proposals</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bouckaert, R., WeidemuÃàller, P., Esquivel Gomez, L., MuÃàller, N. F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose an improved proposal distribution to scale phylogenetic trees based on the intervals between consecutive nodes in the tree, and a set of operators informed by a pseudo parsimony score to explore different tree topologies.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In Bayesian phylodynamics, we jointly reconstruct the posterior distribution of timed phylogenetic trees, evolutionary and population dynamic parameters. Most approaches in the field use Markov chain Monte Carlo (MCMC) for inference. Inferring timed phylogenetic trees using MCMC relies on a set of proposal distributions -- operators -- to explore different phylogenetic tree topologies, node ranks, and heights. However, inefficiencies in these proposal distributions limit the speed and scale with which phylodynamic analyses can be performed. Here, we introduce two classes of operators that allow scaling of phylodynamic analyses by reducing the time needed to reach convergence. We propose an improved proposal distribution to scale trees based on the intervals between consecutive nodes in the tree, and a set of operators informed by a pseudo parsimony score to explore different tree topologies. Contrary to already existing operators that make large changes to areas of the tree with no mutations at all, our new parsimony-focused operators target areas of the tree with many mutations. We first demonstrate the correctness of these proposal distributions using a well-calibrated simulation study. We then demonstrate the improvements of using the updated proposal distributions using a number of large real-world datasets, showing a substantial increase in effective sample size per hour across viral datasets as well as a reduction in the number of samples required for the MCMC chain to reach stationarity. Finally, we apply the operators to a 10,000-sequence H3N2 dataset including phylogeography and finish the analysis within two weeks. The operators are implemented in the TargetedBeast package, which is available as an open-source package for BEAST2 and a user-friendly graphical user interface. The speed improvements in the TargetedBeast package are beneficial to any phylodynamics method implemented in BEAST2 that relies on standard phylogenetic trees, such as coalescent-skyline, birth-death skyline, and structured phylodynamic methods.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.18.660471v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Biologically Informative NA Deconvolution (BIND) excavates hidden features of the proteome from missing values in large-scale datasets</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> GUO, W., JIN, W., ZHENG, J., PAN, Y., WANG, R., ZHANG, J., FENG, X., CHEN, L., ZHANG, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a novel method for deconvolving missing values in single-cell proteomics that can be used to identify the biological significance of missing values.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The fast-advancing mass spectrometry and related technologies have greatly extended the depth of coverage in large-scale proteomics studies, including single-cell applications. As sample numbers grow rapidly, it is often challenging to interpret the proteins with missing values that are often presented as "NA" (not available). It could be the evidence of no expression, low expression below the detection threshold, or false negative detection due to technical issues. Existing methods for missing values imputation, while generally useful, rarely consider the non-random NA values that inform biological significance. In the current study, we developed Biologically Informative NA Deconvolution (BIND) that applies an adaptive neighborhood-based modeling to deconvolve the nature of NAs as "biological" (low/no expression) or technical (experimental errors). Applying to multiple cell line datasets and human tissue extracellular vesicle datasets, BIND excavated the NAs that indicated "hallmark absence" of unique proteins. This led to improvements in protein-protein interaction analysis and the identification of novel disease biomarkers. To facilitate its public accessibility, we compiled BIND into a web server that features functional online operations and interactive visualizations. Furthermore, we demonstrated that the BIND server could deconvolve the NAs and improve the analyses of single-cell proteomics datasets. Overall, BIND delineates the biological significance of missing values rather than treating them as a burden, providing a critical perspective for understanding the complex proteome in various biological contexts.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.19.660508v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genome-wide rules of transcription factor cooperativity revealed through in silico binding site ablation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> He, X., Einarsson, H., Palenikova, P., Rennie, S., Hu, D., Cui, R., Vaagenso, C., Lage, K., Krautz, R., Andersson, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> DeepCompARE, a lightweight model paired with an in silico ablation framework for genome-wide analysis of regulatory sequences, offers new insights into gene regulatory logic.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Transcription factor (TF) cooperativity plays a critical role in gene regulation. However, the underlying genomic rules remain unclear, calling for scalable methods to characterize the TF binding site (motif) syntax of regulatory elements. Here, we introduce DeepCompARE, a lightweight model paired with an in silico ablation (ISA) framework for genome-wide analysis of regulatory sequences. Our framework enables precise interpretation of the motif syntax governing chromatin accessibility, enhancer activity, and promoter function. We find that most TF motifs are pairwise independent, indicating a default additive behavior of TFs, and define a cooperativity score to quantify deviations from this baseline. This reveals synergy and redundancy as opposite effects along the same cooperative spectrum. TF redundancy is linked to promoter activity and broad expression, whereas TF synergy is associated with enhancer activity, physical interactions, and cell-type specificity. Our framework provides a quantitative model for TF cooperativity, offering new insights into gene regulatory logic.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.19.660093v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Ultra-fast and Efficient Network Embedding for Gigascale Biological Datasets</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Knight, R., Zhao, J., Pierre Both, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present GraphEmbed, a computer program for scalable, memory-efficient network embedding.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Graph/network representation learning (or graph/network embedding) is a widely used machine learning technique in industry recommending systems and has recently been applied in computational biology. Popular network representation learning algorithms include random walk and matrix factorization methods, but they do not scale well to large networks. To accommodate the fast growth of real-world network datasets, especially biological datasets, we engineered and improved several network embedding algorithms via intensive computational optimization (e.g., randomized-generalized singular value decomposition/SVD, efficient sketching via ProbMinHash including edge weights) and parallelization to allow ultra-fast and accurate embedding of large-scale networks. We present GraphEmbed, a computer program for scalable, memory-efficient network embedding. GraphEmbed can perform embedding for large-scale networks with several billion nodes in less than 2 hours on a commodity computing cluster. We benchmark it against standard datasets and demonstrate consistent speed and accuracy advantages over state-of-the-art techniques. We also propose centric AUC, a new metric for evaluating link-prediction accuracy in network embedding. It corrects the bias in conventional AUC caused by the highly skewed node degree distributions, which are typically found in real-world networks, especially biological networks. Taken together, GraphEmbed solves a major challenge in large-scale network representation learning for networks in general and biological networks in particular.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.18.660497v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

