<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Sunday, 31 August 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Breaking Free from the Acquisition Dogma for Volume Electron Microscopy</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chen, B., Wang, F., Wang, H., Zhang, Y., Zhao, Z. Z., Chen, H., Han, H., Chen, X., Hua, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Volume electron microscopy (VEM) enables three-dimensional (3D) visualization of thick specimens through serial sectioning and nanometer-resolution imaging. In an effort to overcome daunting challenges in subsequent data analysis, people tend to employ excessively slow imaging and high resolution, both of which significantly prolong the acquisition. Here, using authentic and synthetic VEM datasets through image stack re-registration, we demonstrate the implications of 3D oversampling in maintaining structural information fidelity against high background noise. This provides a fact-based argument for a higher priority of improving the spatial sampling rate, particularly in the most difficult axial direction, than suppressing the image noise during a VEM acquisition. To accelerate VEM acquisition by leveraging the oversampled 3D context, we explore distinct machine learning-based methods for restoring serial images that contain either low-contrast snapshots or skipped sections. On the datasets of similar acquisition consumptions, 3D context-based (volumetric) denoising outperforms state-of-the-art interpolation and 2D denoising approaches. Furthermore, the volumetric denoising models can proceed in a self-supervised manner, thereby no longer relying on specific training datasets. This work is not only instructive for planning efficient large-scale acquisition on commercial setups but also benchmarks the methodology of optimizing VEM acquisition to facilitate automated image processing.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.26.672334v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Determination of Copy Number Variations and Affected Gene Networks in Breast Cancer from Mexican Patients</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Larios-Serrato, V., Valdez-Salazar, H.-A., Torres, J., Camorlinga-Ponce, M., Pina-Sanchez, P., Mayani-Viveros, H., Ruiz-Tachiquin, M.-E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Triple-negative breast cancer (TNBC) is an aggressive subtype with limited treatment options and high molecular heterogeneity. In this study, we performed a genome-wide analysis of copy number variations (CNVs) using high-density microarrays in tumor tissue (TUM), tumor adjacent tissue (ADJ), and leukocytes (LEU) from five Mexican TNBC patients. We identified both unique and shared CNVs across tissues, including alterations in key chromosomal regions such as 1q23.3, 1q32.1, and 8q24.3, which harbor oncogenes like MYC, MCL1, and BCL9. Losses in 6q25.2 affecting ESR1 were also detected. CNVs were enriched in genes related to the Hallmarks of Cancer, with TUM samples showing profiles associated with proliferation, metastasis, and immune evasion; ADJ samples with growth suppression; and LEU samples with genomic instability. Pathway enrichment analyses revealed disrupted functions in DNA repair, extracellular matrix organization, and TP53 signaling in TUM. Notably, EGFR, ERCC4, and HSP90AB1 emerged as central nodes in interaction networks and may serve as markers or therapeutic targets. This is the first CNV profiling study of its kind in TNBC from Mexican patients, highlighting the importance of including underrepresented populations in genomic research to uncover distinct molecular signatures and potential diagnostic or therapeutic avenues. Implications: Molecular signatures of breast cancer (BC), predicted bioinformatically, involve common and distinct CNV-Hallmarks of Cancer genes, which are suitable candidates for screening as potential BC markers.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.28.672450v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Deciphering the role of autophagy under Cd toxicity in Arabidopsis thaliana</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Collado-Arenal, A. M., Perez-Gordillo, F. L., Espinosa, J., Moreno-Diaz, R., Shabala, S., Romero-Puertas, M. C., Sandalio, L. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cadmium (Cd) is a toxic pollutant in soil and water affecting plants, animals, and humans. Autophagy, a cellular recycling process, is crucial for different biotic and abiotic plant stress responses. This study explores the autophagy role in Arabidopsis under Cd stress, using wild-type, autophagy-deficient mutants (atg5, and atg7) and overexpressing lines (35S:ATG5, 35S:ATG7). Cd exposure induced autophagy, as evidenced by ATG8a and ATG8a-PE accumulation, GFP-ATG8a fluorescence, and upregulation of ATG genes and proteins. Responses differed between Col-0 and Ws backgrounds, with Ws showing higher Cd tolerance. atg5 mutants were more sensitive to Cd, indicating the autophagy protective role, whereas ATG5/ATG7 overexpression did not significantly enhance Cd tolerance. Although oxidative stress may activate autophagy, ATG5/ATG7 overexpression did not significantly change the oxidative stress response. Notably, atg5 mutants displayed marked disruptions in metal/ion homeostasis under control and Cd conditions, reinforcing autophagy's role in ion homeostasis. In contrast, atg7 showed no significant differences from its WT (Ws), suggesting genotype-specific effects. Transcriptional analysis of metal transporters and ion flux analyses indicates that autophagy can regulate metal/ion accumulation through transcriptional control and post-translational modifications (e.g., ROS) with a differential response being observed between Ws and Col-0 plants.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.27.672299v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Exploring the Repetitive DNA Diversity in Solanum betaceum (Solanaceae)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sader, M. A., Vaio, M., Trenchi, A., Zapata, M. J., Chiarini, F., Lopez, A., Urdampilleta, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The Solanaceae family, known for its diverse and economically important crops, includes the genus Solanum, which comprises 1,245 species. Solanum betaceum (tree tomato), native to the Andes and cultivated globally, is a promising species due to its nutritional value and market potential. The Cyphomandra clade, which includes the tree tomato, is characterized by huge genomes and chromosomes, with repetitive DNA elements (e.g., retrotransposons and satellite DNA) playing crucial roles in genomic and evolutionary studies. Despite its importance, genetic research on S. betaceum remains limited. This study addresses this knowledge gap by characterizing the repetitive DNA fraction to better understand intraspecific variation and develop molecular markers. Samples from five populations in northwestern Argentina were cultivated, and genome size was assessed via flow cytometry. Illumina HiSeq sequencing combined with RepeatExplorer analysis was used to identify repetitive DNA elements. Cytogenetic techniques, including CMA/DAPI staining and fluorescence in situ hybridization (FISH), were employed to detect satellite DNA patterns. Genome size analysis revealed slight variation among populations. Repetitive DNA accounted for 63.5% of the genome, with Ty3-gypsy retrotransposons being the most abundant (51.44%). Satellite DNA and rDNA were less prevalent, comprising 0.93% and 0.30% of the genome, respectively. Population comparisons showed consistent proportions of repetitive DNA overall, with notable differences in Ty3-gypsy-Tekay and satellite DNA fractions. This study provides a detailed profile of the repetitive DNA landscape in S. betaceum, uncovering intraspecific differences and delivering valuable genomic insights for future breeding and conservation efforts.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.26.672483v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Haplotype-resolved genomes of diverse oat crown rust isolates reveal both global dispersion of long-lived clonal haplotypes and limited recombination between haplotypes.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Henningsen, E., Nazareno, E. N., Spanner, R., Lewis, D., Lubega, J., Moreau, E. P., Moscou, M. J., Boesen, B., Stone, E., Kanyuka, K., Kianian, S. F., Steffenson, B. J., Dodds, P., Sperschneider, J., Figueroa, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genetic diversity of pathogen populations plays an important role in host adaptation. While single high quality genome references are a valuable resource, a compilation of genome references from individuals increases the breadth of genetic variation within a species that is captured. Puccinia coronata f. sp. avenae (Pca), a fungal pathogen causing crown rust of oat, demonstrates rapid virulence evolution and adaptation to newly released cultivars. To broaden the geographic and temporal distribution of available Pca genomes, we generated nuclear haplotype-resolved genome references for ten isolates from Europe, Africa and the Middle East and compared these with existing references for USA and Australian isolates. Of the full collection of 52 haplotypes, 40 were unique. Importantly, the presence of a nearly identical haplotype in a UK isolate collected in 1984 and in USA isolates from 1990 and 2017 supports the existence of long-lived clonal haplotypes in the global population that have been exchanged between lineages. Taken together with the identification of infrequent recombination between haplotypes from geographically dispersed isolates, this evidence reflects a globally mobile population of Pca that is mostly comprised of persistent clonal lineages with some influence from rare recombination events. Analysis of the core and non-core proteome suggests that while the core proteome is enriched for predicted secreted and effector proteins, sequence and expression variation are most prevalent in non-core orthogroups. We anticipate that this expanded collection of haplotypes will facilitate the development of new surveillance technologies and identification of virulence loci.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.26.672502v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MutAIverse: An AI-Powered, Mechanism-backed Platform for Discovering Novel DNA Adducts and their precursor Genotoxins</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Satija, S., Mohanty, S. K., Jorvekar, S., Verma, A., Solanki, S., Gautam, V., Arora, S., Mittal, A., Sharma, A., Chauhan, S., Duari, S., Kumar, S., Rai, A., Das, A., Kataki, K., Das, K., Sarma, A., Tayal, J., Sengupta, D., Mehta, A., Borkar, R. M., Ahuja, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genotoxin exposure leads to DNA adduct formation, potentially causing mutations if unrepaired. Current DNA adductomics platforms or analytical workflows are limited by incomplete spectral libraries, reliance on experimentally validated adducts, limited cellular contexts, and inefficient computational methodologies. We introduce MutAIverse, an advanced AI-driven DNA adductomics analysis platform that overcomes these limitations by leveraging intracellular mechanistic modeling of genotoxin bioactivation to construct a comprehensive DNA adduct library and offers advanced spectral mapping. MutAIverse integrates experimentally validated and chemically valid putative DNA adducts, enabling interpretable retrograde tracking to parental genotoxins. Validation of MutAIverse against experimental MS/MS datasets demonstrated the detection of both known and novel adducts. Furthermore, application to in-house generated DNA adductomics data from tissue biopsies of smokeless tobacco-induced head and neck cancer patients revealed selective enrichment of both novel and known DNA adducts; a subset of them was validated using MS/MS analysis. Collectively, MutAIverse provides a robust, end-to-end, and interpretable platform for advanced DNA adductomics analysis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.26.672508v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Whole-transcriptome-scale and isoform-resolved spatial imaging of single cells in complex tissues</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cohen, L., Halpern, A., Blosser, T. R., Che, P., Pan, X., Zhuang, X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cell and tissue functions arise from complex interactions among numerous genes, and a systematic understanding of these functions requires isoform-resolved, whole-transcriptome-scale analysis of single cells with high spatial resolution. Here, we introduce a spatial transcriptomics method based on a novel in situ RNA amplification strategy, enabling short RNA sequence detection and hence spatially resolved expression profiling of individual cells at the whole-transcriptome scale with splice-isoform resolution. Using this approach, we imaged ~33,000 distinct RNAs, including ~23,000 genes and ~10,000 isoform-defining transcripts, in the mouse brain. Our data enabled systematic analyses of region- and cell-type-specific gene programs and ligand-receptor-based cell-cell communications. These data further revealed a rich spatial diversity and cell-type specificity in isoform usage across numerous genes and identified brain structures particularly enriched for specific isoform usage. We anticipate broad application of this method for molecular and cellular analysis of tissues, unlocking previously inaccessible discoveries in cell and organismal biology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.27.672533v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Iterative, multimodal, and scalable single-cell profiling for discovery and characterization of signaling regulators</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Blair, J. D., Dalgarno, C., Satija, R., Grabski, I. N., Bradu, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cell signaling plays a critical role in regulating cellular state, yet uncovering regulators of signaling pathways and understanding their molecular consequences remains challenging. Here, we present an iterative experimental and computational framework to identify and characterize regulators of signaling proteins, using the mTOR marker phosphorylated RPS6 (pRPS6) as a case study. We present a customized workflow that uses the 10x Flex assay to jointly profile intracellular protein levels, transcriptomes, and CRISPR perturbations in single cells. We use this to generate a 'glossary' dataset of paired protein-RNA measurements across targeted perturbations, which we leverage to train a predictive model of pRPS6 levels based solely on transcriptomic data. Applying this model to a genome-wide Perturb-seq dataset enables in silico screening for pRPS6 and nominates novel regulators of mTOR signaling. Experimental validation confirms these predictions and reveals mechanistic diversity among hits, including changes in signaling output driven by anabolic activity, cellular proliferation and multiple stress pathways. Our work demonstrates how integrated experimental and computational approaches provide a scalable framework for multimodal phenotyping and discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.27.672635v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Dynamics of complex genomic regions in giant genomes: MHC evolution in newts</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Babik, W., Dudek, K., Palomar, G., Marszalek, M., Dubin, G., Yun, M., Migalska, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Major Histocompatibility Complex (MHC) molecules are central to vertebrate adaptive immunity and MHC genes serve are key models in evolutionary genomics, offering insight into birth-and-death evolution, gene duplication, and the maintenance of genetic diversity. However, the organization and evolution of the MHC in species with giant genomes, such as salamanders, remain poorly understood. Here, we use comparative genomics, ontogenetic, tissue expression and polymorphism data across seven newt species to investigate MHC evolution in this group. Contrary to earlier suggestions of a massively expanded MHC in salamanders, we find that the core MHC region remains relatively compact, demonstrating that genome gigantism does not scale proportionally in this region. Our finding also challenges the model of coevolution between a single classical MHC-Ia gene and antigen processing genes (APGs), revealing instead several polymorphic and highly expressed putative MHC-Ia located at varying distances from the APGs. MHC-I genes exhibit lineage-specific duplications and signs of concerted evolution, resulting in poorly resolved phylogenies. In contrast, MHC-II genes are more conserved and exhibit extensive trans-species polymorphism. Expression and polymorphism patterns identify putative nonclassical MHC-Ib genes, likely repeatedly derived from MHC-Ia genes - paralleling patterns seen in mammals, but contrasting with the situation in fish and Xenopus frogs. In all seven species, some MHC-Ib genes show high relative expression during the larval stage but not at adulthood, suggesting a role in larval immunity. Our results underscore the importance of salamanders for understanding the evolution of complex regions in giant genomes and the architecture of the tetrapod MHC.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.27.672527v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Ensemblify: a user-friendly tool for generating ensembles of intrinsically disordered regions of AlphaFold and user-defined models</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fernandes, N., Gomes, T. L., Cordeiro, T. N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: Intrinsically disordered proteins (IDPs) and regions (IDRs) challenge structural characterization due to their dynamic conformational ensembles and lack of stable structure. Existing computational approaches for modelling these ensembles are either computationally intensive, limited in flexibility, or inaccessible to non-experts, especially when dealing with multi-domain or multi-chain proteins. Results: We present Ensemblify, an open-source, user-friendly Python package for generating and analyzing conformational ensembles of IDPs/IDRs. Ensemblify uses a Monte Carlo algorithm coupled with neighbour-aware sampling of dihedral angles from curated or user-defined fragment libraries to explore conformational space. It directly incorporates information from AlphaFold confidence metrics as flexible energy restraints in PyRosetta to guide the sampling. It supports multi-chain and multi-domain proteins and can sample N-terminal, C-terminal, and inter-domain linkers while preserving folded regions. Ensemble quality can be validated and refined against experimental data such as SAXS via Bayesian/Maximum Entropy (BME) reweighting. Interactive dashboards provide in-depth structural analysis and comparison. Testing across 10 diverse proteins demonstrated Ensemblify's accuracy, flexibility, and ability to recover experimentally observed structural features. Incorporating AlphaFold confidence metrics shows potential to improve the ensemble-data agreement. Availability: Ensemblify is freely available at https://github.com/CordeiroLab/ensemblify, along with detailed installation instructions and usage tutorials. Ensemblify can be used for scripting through its Python API or directly through the provided command-line interface (CLI). Complete documentation is available within the source-code and CLI and on Ensemblify official documentation page (https://ensemblify.readthedocs.io).</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.26.672300v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Characterization of microbial dark matter at scale with MetaSBT and taxonomy-aware Sequence Bloom Trees</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cumbo, F., Blankenberg, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Metagenomics has become a powerful tool for studying microbial communities, allowing researchers to investigate microbial diversity within complex environmental samples. Recent advances in sequencing technology have enabled the recovery of near-complete microbial genomes directly from metagenomic samples, also known as metagenome-assembled genomes (MAGs). However, accurately characterizing these genomes remains a significant challenge due to the presence of sequencing errors, incomplete assembly, and contamination. Here we present MetaSBT, a new tool for organizing, indexing, and characterizing microbial reference genomes and MAGs. It is able to identify clusters of genomes at all seven taxonomic levels, from the kingdom all the way down to the species level, using the Sequence Bloom Tree (SBT) data structure that relies on Bloom Filters (BFs) to index massive amounts of genomes based on their k-mers composition. We have built an initial set of databases composed of over 190 thousand viral genomes from NCBI GenBank and public sources grouped into sequence consistent clusters at different taxonomic levels, making it the first software solution for the classification of viruses at different ranks, including still unknown ones. This results in the definition of over 40 thousand species clusters where ~80% do not match with any known viral species in reference databases to date. Furthermore, we show how our databases can be used as a new basis for existing quantitative metagenomic profilers to unlock the detection of unknown microbes and the estimation of their abundance in metagenomic samples. Finally, the framework is released open-source and, along with its public databases, is fully integrated into the Galaxy Platform enabling broad accessibility. Importance: The MetaSBT framework and its databases, together with its integration in the Galaxy Platform, provide a powerful resource for microbial research. MetaSBT provides a powerful and scalable approach for classifying microbial genomes, including previously unknown ones. This facilitates the discovery and characterization of novel taxa, a crucial feature for expanding our knowledge of microbial diversity and its implications within host health and environmental factors. Furthermore, MetaSBT databases can serve as a reference base for other state-of-the-art tools, enhancing their capabilities to identify, analyze, and classify unknown microbes in metagenomic samples.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.25.672238v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A taxonomic synopsis of Heterotis (Melastomataceae), including H. kupensis, a new threatened species endemic to Mt Kupe, Cameroon</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Svahnstrom, V., Cheek, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We provide a synopsis of the African genus Heterotis (Melastomataceae), recognising seven species and including the description of a new species to science Heterotis kupensis Sv. & Cheek (Melastomataceae) which is only known from Mt Kupe in the South West Region of Cameroon. In addition, we select a lectotype for H. prostrata (Thonn.) Benth., second step lectotype for H. buettneriana (Cogn. ex Buttner) Jacq.-F&egravel. and create the new combination H. welwitschii (Cogn.) Sv. & Cheek to replace H. cogniauxiana (A.Fern. & R.Fern.) Ver.-Lib. & G.Kadereit synon. nov. Distribution, habitat, phenology, taxonomic notes, and preliminary conservation assessments following IUCN Red List criteria are presented along with a key to the species.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.26.672404v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MoGAAAP: A modular Snakemake workflow for automated genome assembly and annotation with quality assessment</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> van Workum, D.-J. M., Dey, K. K., Kozik, A., Lavelle, D. O., de Ridder, D., Schranz, M. E., Michelmore, R. W., Smit, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> With the current speed of sequencing, there is a desire for standardised and automated genome assembly and annotation to produce high-quality genomes as input for comparative (pan)genomics. Therefore, we created a convenience pipeline using existing tools that creates annotated genome assemblies from HiFi (and optionally ultra-long ONT and/or Hi-C) reads for a set of related accessions as well as a related reference genome. Our pipeline is species-agnostic and generates an extensive quality assessment report that can be used for manual filtering and refinement of the assembly and annotation. It includes statistics for individual completeness and contamination assessments as well as a concise pangenome view. The pipeline is implemented in Snakemake and available with a GPLv3 license at GitHub under github.com/dirkjanvw/MoGAAAP and at Zenodo under doi.org/10.5281/zenodo.14833021.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.26.672321v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Microbial Named Entity Recognition and Normalisation for AI-assisted Literature Review and Meta-Analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Patel, D., Lain, A. D., Vijayaraghavan, A., Faghih Mirzaei, N., Mweetwa, M. N., Wang, M., Beck, T., Posma, J. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: Manual curation of biomedical literature is slow and error-prone and while large language models (LLMs) trained on general texts have shown to be useful for text summarisation, these methods lack the domain- specific expertise required to perform this task accurately. Here we describe the creation of the first microbiome- specific text corpus, use this to train deep learning algorithms for named-entity recognition (NER) and normalisation (NEN), and demonstrate their use to meta-analyse microbiome literature. Methods: We developed an automated pipeline to annotate all mentions of bacteria, archaea, and fungi in 1,410 full-text microbiome articles. We manually annotated (gold-standard) a separate test set of 288 documents. We trained different transformer-based language models for microbiome recognition and normalisation to taxonomic identifiers and evaluate their performance using the precision, recall, F1-score, and accuracy on the test set. The best models were used to automatically annotate all available Open Access, full-text microbiome articles (n=6,927) and unpick taxa significantly reported more often across 14 domains. Results: The training and validation set contained a total of 90,150 annotations (both full text and abbreviations). Using the gold-standard test set, with an inter-annotator agreement rate of 99.52% for NER and 88.31% for NEN, the models trained were evaluated and our fine-tuned BioBERT model achieved an F1-score of 96% for NER surpassing a rule- and dictionary-based annotation pipeline (94%). For NEN the accuracy obtained by the deep learning models greatly surpassed that of the pipeline (91% vs 69%). Evaluated across the entire available literature, our models annotate an entire full-text document in only 7 seconds. Conclusion: Our algorithms have near perfect precision and greatly speed up the process of annotating microbes in full-text articles. We demonstrated the capabilities of these methods by analysing the entire available literature and describe the taxa associated with each of the domains in our meta-analysis, and exemplify how these methods can be integrated into literature review workflows improving both the speed and accuracy of results.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.29.671515v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multilayered Defense Responses in Sugarcane Against Pratylenchus zeae  Revealed by Comparative Transcriptomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Confort, P. M. d. S., Souto, T. G., Ferreira, J., Creste, S., Vitorello, C. B. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The root-lesion nematode Pratylenchus zeae ranks among the most pervasive soilborne threats to global sugarcane (Saccharum spp.) production, however, the molecular basis of host resistance to this pathogen remains largely unexplored. Using comparative transcriptomics 15 days after inoculation, we profiled root response patterns of a resistant cultivar (RB966982) and a susceptible cultivar (CTC9001). Differential gene expression analysis identified 3,385 DEGs (1,426 up, 1,959 down) in the susceptible CTC9001 and 8,689 DEGs (5,334 up, 3,355 down) in the resistant RB966982 when comparing control and inoculated plants, revealing distinct genotype-specific defense strategies. The resistant genotype mounted a coordinated, multilayered defense marked by dramatic transcriptional reprogramming, including intensified glycolysis and fatty-acid biosynthesis, elevated oxidoreductase/ROS activity, and concurrent activation of jasmonic and salicylic acid associated pathways, with strong induction of multiple PR1 homologs. RB966982 also showed pre-primed and inducible resistance gene analogs and targeted cell-wall reinforcement through xyloglucan fucosylation. By contrast, CTC9001 displayed a delayed or attenuated response characterized by cell-proliferation signatures and broad activation of callose-related defenses (1,3-{beta}-D-glucan synthesis) that appear insufficient to limit nematode progression. GO enrichment analysis revealed that the resistant response was dominated by biological processes linked to carbohydrate and lipid metabolism, oxidoreductase activity, hormone signaling, and cell-wall organization, while the susceptible response was enriched in stress-related pathways lacking coordinated metabolic and structural reinforcement. Collectively, these findings indicate that durable resistance to P. zeae is unlikely to arise from a single mechanism, effective protection will require stacking complementary defense layers, including early metabolic reprogramming, robust hormone-mediated signaling, and reinforced cell-wall barriers. The candidate RGAs, PR1 homologs, and cell-wall remodeling enzymes identified here, together with the GO-enriched pathways, provide concrete targets for marker-assisted breeding and gene editing strategies aimed at developing sugarcane cultivars with durable nematode resistance.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.25.672178v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Development and characterization of Novel Small Molecule Inhibitors Targeting LAG-3 Protein</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jones, S. J. M., Yakovenko, O., Baradaran-Heravi, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Lymphocyte Activation Gene-3 (LAG-3) is a 503-amino acid transmembrane protein that modulates immune responses by negatively regulating the proliferation and activation of T cells - key effectors in adaptive immunity. The finely tuned expression of LAG-3, along with other immune checkpoints, prevents excessive immune activation and safeguards tissues from inflammation-induced damage. Importantly, the immune system also plays a critical role in tumor surveillance by recognizing and eliminating cells expressing neoantigens arising from somatic mutations. However, many cancers exploit immune checkpoint molecules like LAG-3 to dampen antitumor immunity. Elevated expression of LAG-3 within the tumor microenvironment contributes to immune evasion by suppressing cytotoxic T-cell activity. Consequently, inhibition of LAG-3 has emerged as a promising strategy for restoring immune function and enhancing anticancer immunity. This report presents the rational design and development of small-molecule inhibitors targeting LAG-3 through a novel semi-allosteric mechanism - a priori superior to classic (antibody) binding inhibitory - representing a next-generation therapeutic approach with potential applications in oncology and immune-related disorders.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.29.673196v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">METTL13 Promotes Pre-Leukemic Transformation and the Development of Pediatric Leukemia</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Enlund, S., Lim, C.-E., Hoang, I., Joshi, S., Ramilo Amor, A., Thomsson, C., Sinha, I., Shirazi Fard, S., Nilsson, A., Hermanson, O., Jiang, Q., Holm, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Post-transcriptional RNA modifications, such as N6-methyladenosine (m6A) methylation and adenosine to inosine (A-to-I) editing, are critical regulators of hematopoietic stem cell (HSC) self-renewal and differentiation, yet their precise contributions to malignant transformation are not fully elucidated. In this study, we uncovered the epitranscriptomic landscape caused by knockdown of genes from the methyltransferase (METTL)-family in hematopoietic stem and progenitor cells (HSPCs). We identified both converging and distinct roles of METTL3 and METTL14, known members of the m6A writer complex, as well as orphan gene METTL13. Notably, METTL13 was uniquely upregulated by adenosine deaminase acting on RNA 1 (ADAR1) overexpression, while other METTL genes were downregulated. Knockdown of METTL13 altered the expression of multiple genes involved in oncogenic development in HSPCs. Furthermore, METTL13 was associated with a high-risk profile in pediatric T-cell acute lymphoblastic leukemia (T-ALL), and functional studies confirmed that METTL13 is required for T-ALL cell proliferation and survival both in vitro and in vivo. Collectively, our results indicate a previously unrecognized, oncogenic role for METTL13 in pre-leukemic transformation and T-ALL pathogenesis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.29.673071v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ATR inhibitors synergize with mitomycin C to enhance cytotoxicity in patient-derived non-muscle invasive bladder cancer organoids</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zuidema, A., Nijland, L., Vosjan, M. M., van Megesen, K., Viergever, B. J., Kranenburg, O., Meijer, R. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;">Background and Objective: There is a high recurrence rate in non-muscle invasive bladder cancer (NMIBC) patients treated with intravesical chemotherapy or BCG, indicating the need for novel treatment options. Synergy between DNA damage response inhibitors and chemotherapy has been shown for different solid tumors. In this study, we explore whether combining chemotherapeutic agents with Ataxia telangiectasia and Rad3-related (ATR) kinase inhibitors is a suitable strategy to reduce recurrence in NMIBC. Methods: NMIBC patient-derived organoids (PDOs; n=6) were exposed to mitomycin C (MMC), gemcitabine, or epirubicin for 2h, to mimic intravesical instillation, and subsequently to ATR inhibitors (berzosertib, ceralasertib, tuvusertib) for 72h. Cell viability and PDO regrowth potential was determined by microscopy and CellTiter-Glo luminescence assays, during a period of 6 weeks post-treatment. Key Findings and Limitations: PDO viability was severely impaired (1-22% viable cells at t=6 weeks) after combination treatment with MMC and ATR inhibitors. This effect was observed directly after treatment and maintained during 6 weeks after drug exposure. In contrast, PDOs treated with only chemotherapy or ATR inhibitors showed similar proliferation rates as untreated controls in week 6. Limitations: PDO models were all chemotherapy naive (either BCG or no pre-treatment). It would be interesting to confirm these findings in MMC-resistant models. Conclusions and Clinical Implications: This study demonstrates synergy between different ATR inhibitors and MMC in a panel of six NMIBC PDOs. We propose that combining intravesical chemotherapy instillations with DNA damage response inhibitors should be further evaluated as a promising strategy to reduce recurrence rates for NMIBC.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.29.673024v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Diverse epigenomic mechanisms underpin transcriptional dysregulation in Polycomb-altered acute myeloid leukemia</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tudose, C., Jones, L., Grosu, T.-I., Fitzgerald, M.-C., Maziak, N., Ling, R., Roy, A., Vaquerizas, J. M., Ryan, C. J., Bond, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;">Background Polycomb Repressive Complex 2 (PRC2) modulates chromatin accessibility and architecture to direct tissue-specific gene expression. PRC2 function is frequently altered in cancer by loss-of-function mutation or deletion, but the downstream effects on transcriptional regulation are incompletely understood. Results To gain insights into these mechanisms, we performed a holistic analysis of epigenomic and transcriptional changes in an isogenic model of acute myeloid leukemia (AML) with heterozygous EZH2 deletion that mimics reduced PRC2 function in patient leukemias. PRC2-depleted cells had diverse gene expression changes, including a bias towards more immature monocyte-lineage transcriptional signatures. PRC2 depletion also correlated with marked increases in chromatin accessibility genome-wide, with 10-45% increases in ATAC-seq peaks in EZH2+/- clones. These changes were accompanied by decreased H3K27me3 and increased H3K27ac levels in CUT+RUN assays that were incompletely linked to transcriptional activity. Despite these generalised changes, 3D chromatin architecture assessed by Hi-C was largely maintained, with H3K27me3 preferentially lost in regions with low DNA-DNA contact frequency. Surprisingly, some regions gained broad H3K27me3 domains at heavily compacted chromatin. We notably saw compartmentalisation changes upstream of the transcriptionally upregulated fetal hematopoiesis gene LIN28B in EZH2+/- cells, with corresponding activation of a LIN28B-specific transcriptional program, including upregulation of the CDK6 oncogene. These results correlated with EZH2+/- cell phenotype, including decreased cellular proliferation and increased resistance to CDK6 inhibitor palbociclib. Conclusions Our findings suggest that PRC2 depletion pleiotropically affects AML transcriptional regulation to directly impact cell phenotype and treatment responsiveness, which may partially explain the aggressive biology seen in these cases.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.28.672871v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Phenotypic screening converges on CDK9 inhibition as a therapeutic strategy in translocation renal cell carcinoma</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Thapa, D. R., Deodhar, R., Skepner, A., Konda, P., Zhou, M., Bluck, J., Pichlo, C., Cui, Y., Stocker, P., Khanna, P., Li, J., Tsai, J., Leed, A., Gould, S., Alimova, M., Friedrich, D., Cheah, J. H., Rupaimoole, R., Guicherit, O., Viswanathan, S. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Translocation renal cell carcinoma (tRCC) is an aggressive kidney cancer driven by gene fusions of the TFE3 transcription factor. TFE3 is essential in tRCC but dispensable in normal cells, presenting an attractive but pharmacologically challenging therapeutic target. We show that the basic helix-loop-helix (bHLH) domain of TFE3 is crucial for chromatin binding and transcriptional function. Via a phenotypic screen of 25,000 compounds, we identified molecules that either displace or retain chromatin-bound TFE3. BRD6866, a compound trapping TFE3 on chromatin, emerged as a pan-CDK inhibitor. Mechanistically, its inhibition of CDK9-a key regulator of transcriptional elongation-was linked to impaired TFE3 fusion activity. These effects were recapitulated by the CDK9-selective inhibitor enitociclib, which downregulated TFE3 targets and suppressed tRCC cell growth. Our findings nominate CDK9 inhibition as a therapeutic strategy in tRCC and demonstrate the utility of mechanism-informed phenotypic screening for challenging targets.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.25.672235v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multi-omic assessment of mRNA translation dynamics in liver cancer cell lines.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Asier Gonz√°lez, Muskan Pandey, Niels Schlusser, Sayanur Rahaman, Meric Ataman, Nitish Mittal, Alexander Schmidt, Attila Becskei, Mihaela Zavolan</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The limited correlation between mRNA and protein levels within cells highlighted the need to study mechanisms of translational control. To decipher the factors that determine the rates of individual steps in mRNA translation, machine learning approaches are currently applied to large libraries of synthetic constructs, whose properties are generally different from those of endogenous mRNAs. To fill this gap and thus enable the discovery of elements driving the translation of individual endogenous mRNAs, we here report steady-state and dynamic multi-omics data from human liver cancer cell lines, specifically (i) ribosome profiling data from unperturbed cells as well as following the block of translation initiation (ribosome run-off, to trace translation elongation), (ii) protein synthesis rates estimated by pulsed stable isotope labeled amino acids in cell culture (pSILAC), and (iii) mean ribosome load on individual mRNAs determined by mRNA sequencing of polysome fractions (polysome profiling). These data will enable improved predictions of mRNA sequence-dependent protein output, which is crucial for engineering protein expression and for the design of mRNA vaccines.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40885748/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

