<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Saturday, 11 October 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Generation of synthetic scRNA-seq-like transcriptomes using a generative adversarial network from RNA-seq data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ruan, D., Armstrong, S. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a cycle-consistent generative adversarial network (cycleGAN) approach to generate synthetic single-cell-like transcriptomes from bulk RNA-seq data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Next-generation sequencing (NGS) technologies have become integral for high-throughput transcriptomic studies. Among these, single-cell RNA sequencing (scRNA-seq) is especially valuable for quantifying gene expression at the individual cell level, enabling the identification of rare cell populations and cellular differentiation pathways. However, the high cost of scRNA-seq often limits its broader application. Bulk RNA sequencing (RNA-seq) provides a more affordable alternative but lacks the single-cell resolution needed to elucidate cellular heterogeneity. Here, we present a cycle-consistent generative adversarial network (cycleGAN) approach to generate synthetic single-cell-like transcriptomes from bulk RNA-seq data. By adversarially training two sets of generators and discriminators, our framework attempts to learn the relationship between bulk and single-cell transcriptome distributions. Although this approach does not replace real scRNA-seq experiments, it can be a useful tool to generate synthetic single-cell-like data for preliminary exploratory investigations and other machine learning applications. We further discuss the performance, limitations, and ethical considerations of our method.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.09.681449v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Leveraging protein representations to explore uncharted fold spaces with generative models</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Miao, Y., Pacesa, M., georgeon, s., Schmidt, J., Lu, T., Huang, P., Correia, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a new coarse grained protein structure representation generative method, DiffTopo, a diffusion model which increases the sampling efficiency and diversity.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> A major challenge in computational de novo protein design is the exploration of uncharted areas within the protein structural space. However, the large degrees of freedom of protein backbones complicate the sampling process during protein design. Machine learning-based models have made great strides in this problem, however due to their nature they tend to exploit rather than explore the data distribution used for training the neural networks. To address some of these challenges, we propose a new coarse grained protein structure representation generative method, DiffTopo, a diffusion model which increases the sampling efficiency and diversity. Combined with a backbone level protein generative model like RFdiffusion, novel protein folds can be generated rapidly, allowing for efficient exploration of the designable topology space. Interestingly, we have discovered that by mirroring the topological organization of native proteins using a pipeline named MirrorTopo, we can readily expand the known fold space. We generated and experimentally characterized 30 different novel topologies from DiffTopo and 6 different novel mirror topologies from MirrorTopo. The developed framework relying on low resolution sampling provides new means for fold exploration challenges, which could in principle enhance our knowledge of the first principles of protein structure and folding, as well as create new opportunities for functional design.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.10.681606v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Sourdough starter-inspired living materials grown from probiotic consortia with engineered division-of-labor</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, S., Xu, Y., Zhan, Y., Echavarria Galindo, M., Chau, S. C. Y., Cui, L., Shaw, W. M., Xie, W. J., Tan, H. Y., Lin, C., Jiang, X., Bian, Z., Dedon, P., Church, G. M., Lu, T. K., Tang, T.-C., LAI, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We create a synthetic sourdough starter in which functional modules are distributed between the two constituent species, enabling in vitro biosensing and in vivo drug delivery and modulate the gut ecosystem in an animal disease model.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genetic circuits enable precise control over cellular behavior. As circuits become more complex, implementing them within a single cell population becomes difficult. Tasks can instead be distributed in engineered consortia. However, selecting appropriate consortia and harnessing the natural traits of each species for such a division of labor remain unsolved challenges. Here, we report the creation of SINERGY, an engineered synthetic sourdough starter in which functional modules are distributed between the two constituent species. These modules, consisting of sensing, communication, and response, enable in vitro biosensing and in vivo drug delivery and modulate the gut ecosystem in an animal disease model. SINERGY not only preserves the safety and health-promoting properties of the natural components but also endows microbes with programmable functionalities, facilitating broad applications in biomedicine.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.09.681537v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Uncertainty-Aware Tau Detection in Progressive Supranuclear Palsy Using Object Detection Models</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhou, G., Azevedo, T., Quaegebeur, A., Pansuwan, T., Rittman, T., Lio, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We fine-tuned YOLO and Faster R-CNN models on a dataset of 29,852 tau lesions from 16 PSP brain slides, achieving mAP@50 of 0.702. We integrated Monte Carlo Dropout into both architectures to estimate spatial and label uncertainty.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Abnormal tau accumulation is a hallmark of neurodegenerative diseases such as Progressive Supranuclear Palsy (PSP). Traditional post-mortem assessments rely on manual tau annotation, which is time-consuming and subjective. Existing machine learning methods typically involve multi-stage feature-based pipelines, resulting in limited scalability and reliance on handcrafted features. This work evaluates deep learning-based object detection as an efficient end-to-end alternative for automating tau lesion identification and classification. We fine-tuned YOLO and Faster R-CNN models on a dataset of 29,852 tau lesions from 16 PSP brain slides, achieving mAP@50 of 0.702. To address limitations of conventional object detection models with overconfident spatial predictions, we integrated Monte Carlo Dropout into both architectures to estimate spatial and label uncertainty. We evaluated detection uncertainty using Probability-based Detection Quality. Our results demonstrate that incorporating uncertainty offers a more comprehensive evaluation of detection reliability, complementing traditional metrics like mAP. For clinical application, we integrated a non-stochastic YOLO model into QuPath for whole-slide inference. Downstream clinical validation was conducted, including SHAP-based saliency analysis and correlation between model-predicted tau densities and clinical disease severity scores. The alignment of detection outputs with pathological features and clinical severity underscores the clinical utility of object detection models. Overall, this work highlights the complementary strengths of object detectors and uncertainty quantification, offering a path toward scalable, interpretable, and clinically meaningful tau pathology analysis. Code is available at https://github.com/Pidongg/dl_histopathology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.09.681479v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Chronic alcohol intake elicits distinct multi-omic profiles in the liver versus skeletal muscle of mice</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Willis, C., Ganjayi, M. S., Brown, A. M., Moser, S. E., Szewczyk, N. J., Clark, B. C., Baumann, C. W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A multi-omics study of liver versus skeletal muscle responses to chronic alcohol exposure, using a pre-clinical mouse model showing signs of alcohol-related liver dysregulation (diminished liver phosphatidylcholine-to-phosphatidylethanolamine lipid ratio) and alcohol-</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Alcohol-related liver disease and alcohol-related myopathy are widespread consequences of chronic alcohol use. However, understanding of the associated molecular mechanisms and effective treatments remains limited. To address this, we employed multi-omics to uncover molecular blueprints of liver versus skeletal muscle responses to chronic alcohol exposure, using a pre-clinical mouse model showing signs of alcohol-related liver dysregulation (diminished liver phosphatidylcholine-to-phosphatidylethanolamine lipid ratio) and alcohol-related myopathy (reduced muscle mass and strength). We found that the liver was more sensitive to chronic alcohol than muscle across the transcriptome, proteome and metabolome levels, but both tissues were equally sensitive at the lipidome level. The liver displayed an extensive and multi-layered metabolic molecular profile, while muscle was associated with upregulated inflammatory and matrisome responses and impaired mitochondrial energetics. Lipidome analyses also revealed a novel potential role for altered phospholipid remodelling in the aetiology of alcohol-related myopathy. Finally, computational drug repurposing identified several compounds for therapeutic targeting of alcohol-induced liver (e.g., saracatinib, GSK126) and muscle (e.g., metformin, trichostatin A) pathophysiology, perhaps working partly to counter metabolic dysregulation. Overall, our study provides a tractable list of therapeutic targets and treatments to help expedite the understanding of and countermeasures against alcohol-related liver disease and alcohol-related myopathy in humans.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.09.681415v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">OptSurvCutR: Validated Cut-point Selection for Survival Analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yau, T. O.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present OptSurvCutR (Optimal Survival Cut-Points in R programming), an R package designed to provide a complete, end-to-end workflow for robust and interpretable cut-point analysis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The stratification of subjects based on continuous predictors is a common yet challenging task in time-to-event analysis (or survival), particularly when relationships are non-linear and require multiple thresholds. Arbitrary cut-point selection can lead to overfitting and spurious conclusions, highlighting the need for a statistically rigorous and reproducible framework. We present OptSurvCutR (Optimal Survival Cut-Points in R programming), an R package designed to provide a complete, end-to-end workflow for robust and interpretable cut-point analysis. The package's methodology is structured into a logical three-step process. First, the find_cutpoint_number() function provides a data-driven approach to determine the optimal number of cut-points by comparing models using information criteria (e.g., AIC, AICc or BIC). Second, the find_cutpoint() function identifies the precise locations of these thresholds by optimising survival metrics, such as the log-rank statistic or hazard ratio, using either an exhaustive systematic search or an efficient genetic algorithm. Finally, the validate_cutpoint() function assesses the stability of the identified thresholds by generating 95% confidence intervals through bootstrap resampling. We demonstrate the package's full workflow using two case studies: a plant science example modelling the non-monotonic effect of temperature on rapeseed germination and a clinical bioinformatics analysis stratifying colorectal cancer patients by a microbial biomarker. These examples show how OptSurvCutR can uncover complex survival patterns that would be missed by traditional dichotomisation. OptSurvCutR provides a transparent and extensible framework for validated cut-point selection, making it broadly applicable for researchers working with time-to-event data. The package, including source code and documentation, is freely available from GitHub at https://github.com/paytonyau/OptSurvCutR.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.08.681246v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Monoaminergic neurons share transcriptional identity across Bilaterian animals</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Goulty, M., Lewis, C., Nicolini, F., Khalid, R., Syed, H., Oliveri, P., Thomas, M., Rosato, E., Feuda, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that monoaminergic neurons form consistent transcriptional clusters across Bilateria and show a shared set of transcription factors that define 'monoaminergic identity' across bilaterians.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The evolutionary conservation of cell types over deep time has long been theorised but remains difficult to demonstrate. Monoaminergic neurons, which produce molecules such as serotonin and dopamine, are central to animal behaviour and cognition, yet their evolutionary origins remain unresolved. Here, we analysed single-cell transcriptomes from 16 metazoan species spanning eight phyla. Using a novel integration method, Orthogroup Recoding, together with experimental validation, we show that monoaminergic neurons form consistent transcriptional clusters across Bilateria. These clusters share a regulatory signature involving conserved transcription factors, including homologues of Fev, Lmx1b, Fer2, Insm, and Isl, suggesting a common regulatory program. This signature extends beyond the brain including other monoaminergic cells, such as the gut enterochromaffin cells and larval sensory neurons. By contrast, non-bilaterian lineages lack the coordinated expression of the transcription factors and the biosynthetic machinery that regulate monoamine production. Our observations support the existence of a shared set of TFs that define 'monoaminergic identity' across in bilaterians.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.10.679534v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Physically-Realistic Simulator and Cosine-Based Decoder for MERFISH Spatial Transcriptomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tsui, J., Adam, N., Choi, W., Liu, L. Y., Flores, C., Ansari, S., Kong, E., Thapliyal, Y., Lee, H., Haider, S., Von Riedemann, I., O'Flanagan, C., IMAXT Cancer Grand Challenges Consortium,, Aparicio, S., Roth, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Serval, a modular framework for evaluating and comparing decoding methods in spatial transcriptomics, and develop a novel cosine decoder method that improves transcript recovery by optimizing the cosine similarity to known barcodes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Imaging based spatial transcriptomics technologies such as MERFISH have opened new avenues for studying cellular organization and gene expression within intact tissues. However, the accuracy of downstream analyses depends critically on the decoding step that reconstructs barcodes from fluorescence patterns and maps them to gene identities. Despite a growing number of decoding methods, limited systematic benchmarking has made it difficult to assess accuracy, sensitivity, and robustness under varying experimental conditions. Here, we introduce Serval, which is a modular framework for evaluating and comparing decoding methods in spatial transcriptomics. Serval separates key decoding stages into independently configurable modules, enabling flexible integration of alternative algorithms. Using the Serval framework we develop a novel decoding method that improves transcript recovery by optimizing the cosine similarity to known barcodes. To evaluate performance under controlled conditions, we developed a photorealistic multiplexed spot simulator that generates MERFISH-like images with ground-truth labels. Using both simulated and real datasets, including cultured cells and tumor sections, we show that the new cosine decoder achieves higher correlation with bulk and single-cell RNA-seq references compared to existing methods. Our results demonstrate that modular decoding frameworks combined with reproducible benchmarking tools can guide method development and support more accurate spatial transcriptomics analysis in complex biological samples.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.09.681318v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Modtector: Ultra-Fast Modification Signal Mining on Mapped Sequencing Reads</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhou, T., Hong, Y., Li, P., Liu, X., Li, A., Sun, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed Modtector, an efficient and versatile tool for unified extraction of modification signals from aligned sequencing data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Current tools for RNA epitranscriptomic modification and structural feature analysis are often fragmented, focusing on single signal types and struggling with processing efficiency, especially as sequencing data volumes increase and single-cell technologies advance. To address this challenge, we developed Modtector, an efficient and versatile tool for unified extraction of modification signals from aligned sequencing data. By integrating dual-signal recognition within a single framework, Modtector employs a "count-then-correct" approach to handle both mutation and stop signals, significantly reducing computational complexity. This results in a multi-fold performance improvement compared to existing tools, particularly when processing large-genome, high-coverage datasets, such as completing the analysis of HEK293 22G data in 5 minutes, demonstrating its potential for large-scale data analysis. Availability and implementation: The manual is available at Readthedocs (https://modtector.readthedocs.io/), and source code is available at GitHub (https://github.com/TongZhou2017/modtector) and Crate.io (https://crates.io/crates/modtector).</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.08.681300v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">TET CpG sequence context specific DNA demeth-ylation shapes progression of IDH-mutant gliomas</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hoogstrate, Y., Ghisai, S. A., van Hijfte, L., Head, R., de Heer, I., Padovan, M., de Wit, M., Vallentgoed, W. R., Dipasquale, A., Wijnenga, M. M. J., Weenink, B., Luning, R., Maas, S. L. N., Brzobohata, A., Weller, M., Weiss, T., Mair, M. J., Berghoff, A. S., Woehrer, A., Jeltsch, A., Koekkoek, J. A. F., Hazelbag, H. M., Kouwehoven, M. C. M., Kim, Y., Westerman, B. A., Ylstra, B., Niers, J. M., Johnson, K. C., Verhaak, R. G. W., Varn, F. S., Khasraw, M., van den Bent, M. J., Wesseling, P., French, P. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that DNA methylation changes in IDH-mutant oligodendrogliomas are linked to increased epigenetic aging, methylation of transcription factors and Ki-67+ cell density, and to large scale DNA demethylation.</p><p style="font-size: 16px; text-align: justify;">Background: Treatment decisions in IDH-mutant oligodendrogliomas are shaped by tumor aggressiveness, underscoring the need for objective grading of these malignant brain tumors. Material and Methods: We collected 302 primary and recurrent resections from oligodendrogliomas and performed Ki-67 staining, proteomics and DNA methylation profiling. Results & conclusion: During tumor progression, DNA methylation of oligodendrogliomas changed along a continuum. This continuum is linked to increased epigenetic aging, methylation of transcription factors and Ki-67+ cell density, and to large scale DNA demethylation. Demethylation was correlated with CpGs flanking sequences preferred by TET enzymes. We confirmed these findings in previously profiled astrocytomas, indicating IDH-mutant gliomas progress along a shared epigenetic axis. We developed an objective DNA methylation based prognostic continuous grading coefficient (CGC{psi}) that captured these changes and outperformed WHO grading for oligodendrogliomas. Our findings underscore the potential of DNA methylation-based grading to more accurately reflect tumor biology and inform clinical decision-making in IDH-mutant gliomas.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.09.681314v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Urinary Exosomal miRNA Profiling Reveals Sensitive Non-Invasive Diagnosis of Bladder Cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> singh, G., Kumar, A., Kumar, L., Mishra, N., Bhattacharjee, S., Yadav, K., Singh, Y., Kumar, U., Trivedi, S., Singh, S. K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that miRNAs in bladder cancer samples are robust and stable, with high sensitivity, and can be used as a biomarker for non-invasive detection and monitoring of UBC.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> miRNAs represent a transformative advancement in both research and clinical management of urinary bladder cancer (UBC), emerging as clinically significant molecular targets with the potential to revolutionize existing diagnostic standards. This study exploited the robust stability of miRNAs and profiled urinary miRNAs in UBC patients and controls through miRNA sequencing, revealing an expanded and altered miRNA repertoire in cancer samples. Validation in an independent cohort revealed fold changes for miR-6724-5p, miR-1273h-5p, miR-7704, miR-200-5p, and miR-10400-5p ranged from ~46 to ~2,777 (p 70%) and high sensitivity (>90%). These data demonstrate the potential of urinary mature miRNAs as robust biomarkers for non-invasive detection and monitoring of UBC in early stages, supporting their translation into clinical diagnostic assays pending larger prospective studies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.09.681526v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Pharmacological Induction of Irreversible Senescence in Drug-Tolerant Persister Cells Prevents Tumor Relapse</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, B., Zhi, Y., Wang, P., Guo, W., Gu, J., He, H., He, K., Zhou, W., Wang, T., Huang, N., Yang, H., Wang, D., Ma, T., Ding, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that a combination of SAHA and tilorone dihydrochloride can induce permanent senescence in melanoma persister cells to prevent residual disease.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Despite substantial advances in targeted therapies, most patients with advanced cancers ultimately relapse. This relapse is frequently seeded by drug-tolerant persister cells (DTPs), which persist as minimal residual disease (MRD) by adopting a reversible quiescent state and later reacquire proliferative capacity under ongoing treatment. To address this, we sought to convert quiescent DTPs into an irreversible, non-proliferative state by pharmacologically inducing senescence. Through large-scale chemical screening and combination optimization, we identified SAHA plus tilorone dihydrochloride (SIC) as a regimen that robustly induces senescence in melanoma DTPs and abolishes their proliferative potential in vitro. Mechanistically, SAHA relieves HDAC1-mediated repression of HMGA2, a key senescence driver, while tilorone synergistically inhibits autophagy to promote a fully senescent state. In a melanoma xenograft mouse model receiving MAPK-targeted therapy, SIC treatment significantly suppressed MRD progression. Importantly, SIC also induced robust senescence across diverse DTP models irrespective of tumor origin, mutational background, or prior treatment. Collectively, these findings provide proof-of-concept that converting DTPs from reversible quiescence to permanent senescence represents an effective strategy to prevent relapse under targeted therapy and position SIC as a broadly applicable pharmacological approach for durable control of residual disease in advanced cancers.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.10.681532v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identification and validation of cell senescence genes in recurrent spontaneous abortion via multiple bioinformatics algorithms.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yiyun Wei, Zhuolin Zhou, Changqiang Wei, Jinlian Cheng, Shanshan Lin, Zhiwei Zhu, Lihong Pang</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified three cellular senescence genes linked to recurrent spontaneous abortion (RSA) and identified potential small molecule therapeutics for RSA.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recurrent spontaneous abortion (RSA) represents a significant challenge in reproductive obstetrics, affecting approximately 5% of couples globally. Despite various treatments, the effectiveness of these interventions remains highly contentious. Emerging evidence suggests that cellular senescence plays a critical role in the pathogenesis of multiple diseases, potentially implicating its involvement in RSA as well. This study integrated two RNA sequencing datasets and employed Weighted Gene Co-Expression Network Analysis (WGCNA) alongside five machine learning algorithms (XGBoost, Boruta, LASSO, SVM-RFE, and Random Forest) to identify cellular senescence genes linked to RSA. Gene expression was validated at both the transcriptome and protein levels using qPCR, Western blot, and immunofluorescence techniques. An Artificial Neural Network (ANN) model was implemented to evaluate their diagnostic value. Additionally, functional enrichment and single-cell RNA sequencing analyses were conducted to investigate the biological functions of these genes. Three cellular senescence genes-TBX2, SRSF3, and TNRC6B-were identified and found to be upregulated in RSA patients. Functional enrichment analysis revealed these genes' involvement in the MAPK signaling pathway, ECM-receptor interaction, and cell-cell communication. Single-cell RNA sequencing demonstrated the distribution of these genes across various cell types, underscoring their significance in RSA. Furthermore, drug sensitivity analysis identified potential small molecule therapeutics for RSA. Cellular senescence may play a central role in the pathology of RSA, with TBX2, SRSF3, and TNRC6B emerging as potential diagnostic biomarkers. This research enhances our understanding of the molecular mechanisms underlying RSA and lays the groundwork for new diagnostic and therapeutic strategies. Nonetheless, further experimental studies are required to elucidate the specific roles and mechanisms of these genes in RSA, with the ultimate goal of achieving precise prevention and personalized treatment.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41073490/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

