<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Sunday, 07 September 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Pathologist-interpretable breast cancer subtyping and stratification from AI-inferred nuclear features</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Barman, R. K., Dhruba, S. R., Hoang, D.-T., Shulman, E. D., Campagnolo, E. M., Wang, A. T., Harmon, S. A., Hu, T. C., Papanicolau-Sengos, A., Nasrallah, M. P., Aldape, K. D., Ruppin, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Artificial intelligence (AI) is making notable advances in digital pathology but faces challenges in human interpretability. Here we introduce EXPAND (EXplainable Pathologist Aligned Nuclear Discriminator), the first pathologist-interpretable AI model to predict breast cancer tumor subtypes and patient survival. EXPAND focuses on a core set of 12 nuclear pathologist-interpretable features (NPIFs), composing the Nottingham grading criteria used by the pathologists. It is a fully automated, end-to-end diagnostic workflow, which automatically extracts NPIFs given a patient tumor slide and uses them to predict tumor subtype and survival. EXPANDs performance is comparable to that of existing deep learning non-interpretable black box AI models. It achieves areas under the ROC curve (AUC) values of 0.73, 0.79 and 0.75 for predicting HER2+, HR+ and TNBC tumor subtypes, respectively, matching the performance of proprietary models that rely on substantially larger and more complex interpretable feature sets. The 12 NPIFs demonstrate strong and independent prognostic value for patient survival, underscoring their potential as biologically grounded, interpretable biomarkers for survival stratification in BC. These results lay the basis for building interpretable AI diagnostic models in other cancer indications. The complete end-to-end pipeline is made publicly available via GitHub (https://github.com/ruppinlab/EXPAND) to support community use and reproducibility.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.04.674077v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Petri Net-Based Framework for Oxygen-Sensitive Regulation of Pyruvate Metabolism in M. tuberculosis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> SIDDIQUI, M. A., Dayal, S., Kapoor, N., Mahapatra, R. P., Jain, R. K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Pyruvate serves as a central hub of cellular metabolism, linking glycolysis with downstream pathways such as the tricarboxylic acid (TCA) cycle, gluconeogenesis, and fermentation. Its metabolic flexibility enables organisms to switch between oxidative and fermentative fates depending on oxygen availability and environmental stress. In this study, a Petri net (PN) model of pyruvate metabolism in Mycobacterium tuberculosis H37Rv was developed to capture and simulate these dynamic transitions. Using Snoopy 2.0 for model construction and COPASI for simulation, the framework incorporated key metabolites, cofactors, and enzymatic processes regulating aerobic and anaerobic states. The model demonstrated that oxygen availability acts as a regulatory switch, channeling pyruvate either toward acetyl-CoA for ATP generation under aerobic conditions or toward lactate production under hypoxia to regenerate NAD+. Structural validation confirmed boundedness, conservativeness, and deadlock-free behavior, underscoring the robustness of the framework. Sensitivity analyses highlighted enzymatic kinetics as critical determinants of flux distribution and system stability. Collectively, the PN model provides a scalable and biologically relevant computational framework for exploring oxygen-dependent metabolic reprogramming, offering insights into energy adaptation and potential therapeutic targets in pathogenic systems.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.02.673648v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">High-resolution metagenome assembly for modern long reads with myloasm</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shaw, J., Marin, M. G., Li, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Long-read metagenome assembly promises complete genomic recovery from microbiomes. However, the complexity of metagenomes poses challenges. We present myloasm, a metagenome assembler for PacBio HiFi and Oxford Nanopore Technologies (ONT) R10.4 long reads. Myloasm uses polymorphic k-mers to construct a high-resolution string graph and then leverages differential abundance for graph simplification. On real-world ONT metagenomes, myloasm assembled three times more complete circular contigs than the next-best assembler. Myloasm can make ONT and HiFi comparable for assembly: for a jointly sequenced gut metagenome, myloasm with ONT assembled more complete circular genomes than any assembler with HiFi. Myloasm recovers previously inaccessible within-species diversity; we recovered six complete Prevotella copri single-contig genomes from a gut metagenome and eight complete TM7 (Saccharibacteria) contigs with > 93% similarity from an oral metagenome. With this improved resolution, we resolved two 98% similar ermF antibiotic resistance genes spreading through distinct strain-specific mobile genetic elements in a human gut.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.05.674543v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PhageAI: a new approach to predicting the lifestyle of bacteriophages using proteinBERT and convolutional neural networks</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Urbanowicz, M., Guzinski, A., Szulc, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Bacteriophages are viruses that infect bacteria, including temperate, virulent and chronic phages. In the current times of increasing resistance to antibiotics (AMR), it is necessary to quickly find phages and determine their lifecycle, therefore computational methods in the first phase of composing phage preparations are necessary to find such sequences. The challenge in building such tools is the lack of good quality and correctly labeled data from multiple reference databases, which are necessary to select sequences to create cocktails that should contain complete and replication-capable phages. Current published models also do not support chronic phages, which constitute a small but significant group of phages. Another problem is explainability, current models are black-box models where we cannot observe what actually influences the model's predictions, this may result in the model overlooking features that are truly associated with the phage life cycle. In the presented tool, however, these issues have been explicitly addressed. The presented tool was also compared to other lifecycle prediction models: Phatyp, PhagePred, Bacphlip and Phacts.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.02.673651v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Instruction-tuned extraction of virus-host interactions from integrated scientific evidence</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, Z., Li, W., Ren, J., Chen, Y., He, B., Sun, Q., Wang, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: Viral infectious diseases continue to pose a major threat to global health. Understanding protein-protein interactions (PPIs) and RNA-protein interactions (RPIs) between viruses and hosts is essential for elucidating infection mechanisms. However, manual curation of these interactions from biomedical literature is inefficient, creating a pressing need for automated and scalable extraction methods. Large language models (LLMs), such as the generative pre-trained transformer (GPT) and bidirectional encoder representations from transformers (BERT), offer promising solutions. Yet, most existing datasets focus on abstracts, overlooking other information-rich sections. We aim to develop a data-efficient approach to extract virus-host interaction (VHI) entities from full-text biomedical articles, including Results, Methods and tables. To our knowledge, this is the first study to apply instruction tuning to full-text VHI extraction. Results: We curated a dataset containing 3, 395 PPI and 674 RPI entities from the Results, Materials and Methods sections, along with 566 PPIs and 793 RPIs from tables. Under low-resource conditions (4, 000 training examples), ChatMed-VHI maintained the highest overall performance, while PubMedBERT achieved slightly higher precision (92.3% vs. 91.3%). Notably, ChatMed-VHI improved F1 and recall by 2.79% but precision dropped by 4.20% with more training data, whereas PubMedBERT improved consistently across all metrics. These results demonstrate the effectiveness of instruction-tuned LLMs for full-text biomedical extraction tasks, and position ChatMed-VHI as a scalable, domain-adaptable solution for VHI mining.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.02.673691v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Clustering of plasmid genomes for genomic epidemiology by using rearrangement distances, with pling</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Frolova, D., Iqbal, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Integration of plasmids into genomic epidemiology is challenging, because there are no clearly defined evolving-units (equivalent to species), and because plasmids appear to evolve as much by structural change (rearrangements, insertions and deletions) as by mutation (1). Further, plasmids transfer horizontally between bacterial hosts (2), and thus a model beyond just a phylogeny is needed to integrate their genetic information with that of their hosts. Pling (3) is a tool designed to measure a genetic distance between plasmids that is related to how they empirically appear to evolve, by measuring the distance between two plasmids as the minimum number of structural changes needed to change one plasmid into the other (ignoring SNP differences). Having done this, it constructs a relatedness network of the plasmids under study, and then clusters them into groups that are credibly recently related. We give here a protocol for running pling, and how we integrate its information with plasmid typing and SNP information. Together, these provide a system for deciding which plasmids are worth treating as "the same plasmid" for the purposes of epidemiology, quantifying their relatedness in terms of rearrangements and SNPs, and then seeing how they are distributed across the host phylogeny.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.02.673752v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrative reverse vaccinology approach for identification of a vaccine antigen candidate with immunoprotective and anti-fibrotic potential against Schistosoma japonicum-associated liver fibrosis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yang, Q. X., Wang, J., Ming, Y. Z., Zhang, Y., Wong, L. P., Lee, H. Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Schistosoma japonicum-associated liver fibrosis (SSLF) is a major cause of morbidity in schistosomiasis, yet no licensed vaccines or specific therapies exist to prevent or treat this complication. Current vaccine development has focused mainly on infection prevention, with limited attention to pathology-driven outcomes such as fibrosis. To address this gap, we developed an in silico pipeline integrating host single-cell transcriptomic analysis, parasite antigen screening, and structure-based modeling. Single-cell RNA-seq of liver tissues from SSLF and controls identified three fibrosis-related hub proteins: FYN, BCL2, and AKT3. In parallel, parasite antigens curated from public databases were evaluated for immunogenicity and safety using reverse vaccinology principles. Among these, DRE2_SCHJA, an Anamorsin homolog involved in Fe-S cluster assembly, was prioritized as the top candidate. Docking analysis predicted the strongest interaction between DRE2_SCHJA and FYN ({Delta}G = -14.9 kcal/mol), linking the antigen to a central regulator of fibrosis. Unlike Saracatinib, which inhibits FYN at the ATP-binding pocket, DRE2_SCHJA bound to the SH2 domain through both canonical and non-canonical contacts, indicating a different regulatory mechanism. Taken together, these findings suggest DRE2_SCHJA as a potential vaccine candidate with both protective and anti-fibrotic potential against SSLF. While experimental validation is required, this study supports the early identification of SSLF vaccine candidates in a rapid and cost-effective manner and also provides a framework applicable to other neglected tropical diseases.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.02.673671v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">3DTubularVoronoi: a Voronoi-Based Framework for Estimating Equilibrium Cellular Packing in 3D Tubular Epithelia</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Guitou, L., Serrano, E. T., Conejero, J. A., Buceta, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Epithelial tube morphogenesis is fundamental to organ development and functioning, yet accurately modeling its intricate 3D organization and dynamics remains a challenge. In particular, current computational tools fail to capture the spontaneous emergence of non-transient scutoids, a geometric feature essential for accommodating curvature in epithelia. Here, we introduce 3DTubularVoronoi, a novel simulation framework for modeling epithelial tube morphogenesis and determining their stable cell packing organization depending on mechanical parameters. The model employs a Voronoi tessellation strategy combined with an energy minimization approach using the Metropolis-Hasting algorithm. 3DTubularVoronoi enables realistic epithelial tube simulations, accounting for scutoid formation and dynamic rearrangements. This tool provides an open-source, user-friendly solution for exploring gland morphogenesis and offering insights into developmental biology and tissue engineering.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.02.673513v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A chromosome-level genome resource for the lethal tapeworm Sparganum proliferum, with candidate genomic safe harbours for functional genetics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hikima, A., Sun, S., Okino, T., Kinoshita, K., Yoshida, A., Maeeda, Y., Ishiwata, K., Kanuka, H., Kikuchi, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Sparganum proliferum is an enigmatic and highly proliferative cestode known for its ability to form invasive larval infections in mammalian hosts. Here, we present the first chromosome-level genome assembly for a member of the order Diphyllobothriidea, generated using a hybrid sequencing approach that integrates Oxford Nanopore/PacBio long-read sequencing, Illumina short-read sequencing, and Hi-C scaffolding. The final genome assembly spanned approximately 681 megabases (Mb) across nine chromosomes. BUSCO analysis revealed 81.9% completeness, whereas repeat annotation identified 55.8% of the genome as repetitive elements. Gene annotation uncovered approximately 29,000 protein-coding genes including ~6200 transposon associated genes, highlighting the complex genomic landscape underlying parasitic lifestyles. Synteny analysis with other cestode linages including Echinococcus and Hymenolepis provided insights into the structural organisation and evolutionary trajectory of S. proliferum. In addition, we identified candidate genomic safe harbour (GSH) loci and promoters from housekeeping genes, offering potential for stable transgene integration. This high-quality genome serves as a critical resource for studying parasite evolution, host adaptation mechanisms, and the molecular basis of invasive proliferation in cestodes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.02.673767v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comprehensive analysis of pseudogene expression in human and macaque brains compared with other tissues</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jiang, Y., Yang, Y. T., Sisu, C., He, T., Won, H., Gerstein, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Although gene expression in the brain has been extensively investigated and compared to other tissues, the activity of pseudogenes has not been comprehensively surveyed. Here, leveraging large-scale RNA-seq data, we construct consistent pseudogene expression profiles in human and macaque brains and compare them to 29 other tissues. We further annotate pseudogenes with potential cellular roles based on co-clustering them with protein-coding genes. Notably, the majority of the expressed pseudogenes show elevated expression in the brain relative to other tissues, and these pseudogenes show broad and consistent expression patterns across brain subregions. Furthermore, spatiotemporal analyses reveal that pseudogenes in different brain subregions have greatly varying temporal trajectories (e.g., increasing vs. decreasing), in contrast to protein-coding genes that tend to be more uniform. Finally, we identify a set of pseudogenes exhibiting significant changes in neuropsychiatric disorders, some of which overlap with known brain eGenes (genes whose expression is associated with expression quantitative trait loci, eQTLs), as well as with genes implicated by genome-wide association studies (GWAS). Together, our study provides a public resource of pseudogene expression in human and macaque brains (in comparison to other tissues) and highlights aspects of pseudogene expression in brain development and pathogenesis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.02.673826v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Cooperativity between Ras pathway mutations in colonic tumorigenesis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Johnson, C. W., Idelevich, A., Shui, B., Tisdale, A., O'Donoghue, E., Cook, J., Hull, S., Yang, M.-H., Sheth, S., Terrell, E., Bovo Minto, S., Morrison, D. W., Kopetz, S., Yaeger, R., Haigis, K. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The Kirsten rat sarcoma (KRAS) gene is the most frequently mutated oncogene in colorectal cancer (CRC). We previously characterized two activating alleles of KRAS, A59T and A59E, that show impaired BRAF dimerization. These alleles of KRAS are enriched in CRC tumors with genetic alterations in genes that regulate MAPK signaling (e.g. EGFR, NF1). Using a new conditional mouse model of K-Ras (LSL-K-Ras A59E) we show that, despite its inability to universally activate RAF kinases, K-Ras A59E disturbs colon epithelium and decreases mouse survival in a tumor model. By combining LSL-K-Ras A59E mice with a conditional knockout of Nf1, we demonstrate cooperation between these alleles at multiple levels. Consistent with cooperation and clinical observations, we show that K-Ras A59E neither promotes EGF independence of organoid growth, nor confers intrinsic resistance to EGFR inhibition. Thus, our data provide a deeper understanding of the role of RAF isoforms in mutant KRAS driven CRC and argue for the use of anti-EGFR therapies against some Ala59 mutant alleles of KRAS.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.05.673994v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

