<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Tuesday, 13 May 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Unraveling the molecular mechanisms of paclitaxel in high-grade serous ovarian cancer through network pharmacology.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yihao Pei, Ziqi Yang, Ben Li, Xiping Chen, Yiming Mao, Yun Ding</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified differentially expressed genes (DEGs) between HGSOC and normal ovarian tissues using the GSE54388 dataset from the Gene Expression Omnibus database and identified 15 overlapping genes related to PTX targets.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> High-grade serous ovarian cancer (HGSOC) is the most common and aggressive subtype of epithelial ovarian cancer, often diagnosed at advanced stages with a poor prognosis. Paclitaxel (PTX), a standard chemotherapeutic agent for HGSOC, exerts cytotoxic effects on cancer cells and modulates the tumor microenvironment. This study aimed to elucidate the molecular mechanisms of PTX in HGSOC using bioinformatics, machine learning, network pharmacology, and molecular docking, to identify potential diagnostic biomarkers and therapeutic targets. We identified differentially expressed genes (DEGs) between HGSOC and normal ovarian tissues using the GSE54388 dataset from the Gene Expression Omnibus database. The intersection of these DEGs with PTX targets, identified from the Swiss Target Prediction database, yielded 15 overlapping genes. These genes were analyzed via protein-protein interaction (PPI) network analysis to identify significant interaction relationships. Kaplan-Meier survival analysis was then performed to assess the prognostic significance of these genes. Their protein expression patterns in HGSOC tissues were validated using the Human Protein Atlas (HPA) database. Functional enrichment analysis was conducted using Gene Ontology and the Kyoto Encyclopedia of Genes and Genomes. A combined diagnostic model was developed using LASSO regression and validated in two independent external datasets (GSE26712 and GSE12470). Molecular docking experiments were conducted to confirm the binding affinity of PTX to key proteins. Immune infiltration analysis was performed to assess the tumor microenvironment, revealing significant differences in immune cell composition between normal and tumor tissues. A total of 2267 DEGs were identified, with 15 overlapping genes related to PTX targets. After PPI network analysis, Kaplan-Meier survival analysis, and HPA validation, five key genes (AURKA, CBX7, CCNA2, HSP90AA1, and TUBB3) were identified as associated with HGSOC progression. The combined diagnostic model demonstrated high accuracy in distinguishing HGSOC from normal tissues, with AUC values of 0.9892 and 0.9465 in the GSE26712 and GSE12470 validation datasets, respectively. Molecular docking confirmed stable binding of PTX to these key proteins, suggesting their role in PTX's therapeutic effects. Immune infiltration analysis revealed significant differences in immune cell composition between normal and tumor tissues, highlighting the potential impact of these genes on the tumor microenvironment. In summary, our findings provide a theoretical basis for improving clinical diagnosis and elucidating the underlying mechanisms of HGSOC.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40355485/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The interictal transcriptomic map of migraine without aura.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Peter Petschner, Sahel Kumar, Duc A Nguyen, Dora Torok, Zsofia Gal, Daniel Baksa, Kinga Gecse, Gyongyi Kokonyei, Hiroshi Mamitsuka, Gabriella Juhasz</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a replicable transcriptomic map of migraine without aura and its comprehensive, genome- and drug discovery focused analysis to identify hypotheses for future research- and clinical attempts.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The present study aimed to deliver a replicable transcriptomic map of migraine without aura (MO) and its comprehensive, genome- and drug discovery focused analysis to identify hypotheses for future research- and clinical attempts.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40350427/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Discrimination of Normal from Slow-Aging Mice by Plasma Metabolomic and Proteomic Features</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Badenoch, B., Fiehn, O., Rappaport, N., Srivastava, P., Watanabe, K., Chandrasekaran, S., Miller, R. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We have applied a machine learning algorithm, XGBoost regression, to seek sets of plasma metabolites that can discriminate control mice from mice treated with an anti-aging diet (caloric restriction) or any of four anti- aging drugs.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Tests that can predict whether a drug is likely to extend mouse lifespan could speed up the search for anti-aging drugs. We have applied a machine learning algorithm, XGBoost regression, to seek sets of plasma metabolites that can discriminate control mice from mice treated with an anti-aging diet (caloric restriction) or any of four anti-aging drugs. When the model is trained on any four of these five interventions, it predicts significantly higher lifespan extension in mice exposed to the intervention which was not included in the training set. Plasma peptide data sets also succeed at this task. Models trained on drug-treated normal mice also discriminate long-lived mutant mice from their respective controls, and models trained on males can discriminate drug-treated from control females. Triglycerides are over-represented among the most influential features in the regression models. Triglycerides with longer fatty acid chains tend to be higher in the slow-aging mice, while triglycerides with shorter fatty acid chains tend to decrease. Plasma metabolite patterns may help to select the most promising anti-aging drugs in mice or in humans, and may give new leads into physiological and enzymatic targets relevant to discovery of new anti-aging drugs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.11.651908v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Efficient evidence-based genome annotation with EviAnn</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zimin, A. V., Puiu, D., Pertea, M., Yorke, J., Salzberg, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Evidence-based eukaryotic gene annotation system that builds the exon-intron structure of genes from transcript alignments or protein-sequence homology rather than from purely ab initio gene finding techniques.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> For many years, machine learning-based ab initio gene finding approaches have been the central components of eukaryotic genome annotation pipelines, and they remain so today. The reliance on these approaches was originally sustained by the high cost and low availability of gene expression data, a primary source of evidence for gene annotation along with protein homology. However, innovations in modern sequencing technologies have revolutionized the acquisition of abundant gene expression data, allowing us to rely more heavily on this class of evidence. In addition to gene expression data, proteins found in a multitude of well-annotated genomes represent another invaluable resource for gene annotation. Existing annotation packages often underutilize these data sources, which prompted us to develop EviAnn (Evidence-based Annotation), a novel evidence-based eukaryotic gene annotation system. EviAnn takes a strongly data-driven approach, building the exon-intron structure of genes from transcript alignments or protein-sequence homology rather than from purely ab initio gene finding techniques. We show that when provided with the same input data, EviAnn consistently outperforms current state-of-the-art packages including BRAKER3, MAKER2, and FINDER, while utilizing considerably less computer time. Annotation of a mammalian genome can be completed in less than an hour on a single multi-core server. EviAnn is freely available under an open-source license from https://github.com/alekseyzimin/EviAnn_release.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.07.652745v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">P3ANUT: An enhanced DNA sequencing analysis platform for uncovering and correcting errors in peptide phage display library screening</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tucker, L., Kolland, E., Gooding, J., Boudjelal, H., Ahmadipour, S., Field, R. A., Warren, D. T., Baker, D., Ma, Y., Marin, M. J., Wu, T., Morris, C. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A Python Pipeline for Phage Analysis through a Normative Unified Toolset (P3ANUT) which employs Levenshtein distance, k-mer approaches and a novel encoding scheme on paired-end sequencing outputs to correct sequencing errors from next-generation sequencing outputs of display library screens.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Display technologies are used extensively in the discovery of peptides and antibodies towards the development of new medicines and diagnostic tools. Phage display technology enables the filtering of <1010 random peptides/antibodies down to and enriched pool of candidates with favourable binding affinities. In recent years, next-generation DNA sequencing technologies have increased the precision and accuracy with which the peptide sequences of phage display library clones are identified. Inaccuracies in DNA sequencing such as substitutions, insertions and deletions in the library oligonucleotide region have the potential to result in the identification of erroneous candidate sequences. Here, we describe a Python Pipeline for Phage Analysis through a Normative Unified Toolset (P3ANUT) which employs Levenshtein distance, k-mer approaches and a novel encoding scheme on paired-end sequencing outputs to correct sequencing errors from next-generation sequencing outputs of display library screens. We introduce an easy-to-use and highly customisable computational tool with graphical user- and command line interfaces to process entire datasets within a single input, as well as visualisation tools for candidate analysis and data generation. P3ANUT shows significant improvements in read recovery, overall read quality, and runtime compared to a previously published pipeline.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.12.648809v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Molecular dynamics simulations of intrinsically disordered protein regions enable biophysical interpretation of variant effect predictors</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zafar, A., Hou, C., Amirani, N., Shen, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We develop a method that uses biophysical features extracted from molecular dynamics simulations of IDRs to predict pathogenicity of variants in Long IDRs.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Predictive models for missense variant pathogenicity offer little functional interpretation for intrinsically disordered regions, since they rely on conservation and coevolution across homologous sequences. To understand the extent to which biophysics modulates model performance compared to genomic conservation, we model biophysics of IDRs explicitly for improved interpretation of variant effects. We develop MDmis, a method that uses biophysical features extracted from molecular dynamics (MD) simulations of IDRs to predict pathogenicity. We find that pathogenic variants in Long IDRs manifest differently, with transient order and depleted solvent access, compared to those in Short IDRs. Using MD simulations of sequences with single missense variants, we identify stronger evidence for pathogenic effects in Long IDRs compared to Short IDRs. MDmis, when combined with conservation information, achieves strong predictive accuracy of pathogenicity of variants in Long IDRs. Overall, extracting information from MD simulations can help understand the drivers of predictive performance and elucidate biophysical behaviors affected by pathogenic variants.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.07.652723v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identification of HIV-Associated Gene Expression Biomarkers Using Machine Learning and Interpretable Artificial Intelligence</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mahmud, S., Hossain, M. F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a machine learning (ML) and interpretable artificial intelligence (AI) framework to identify transcriptomic biomarkers for HIV diagnosis using immune cell-specific gene expression data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Despite advances in antiretroviral therapy (ART), the early and accurate diagnosis of Human Immunodeficiency Virus (HIV) infection remains a significant public health challenge. Traditional biomarkers, such as CD4+ T cell counts and viral load, are limited in capturing the complex biological mechanisms underlying HIV pathogenesis. This study proposes a machine learning (ML) and interpretable artificial intelligence (AI) framework to identify transcriptomic biomarkers for HIV diagnosis using immune cell-specific gene expression data. We utilized the GSE6740 dataset, which includes microarray profiles from CD4+ and CD8+ T cells of HIV-positive (treatment-naive) and HIV-negative individuals. Feature selection was performed through differential gene expression analysis, weighted gene co-expression network analysis (WGCNA), and protein-protein interaction (PPI) network construction. Hub genes identified through these methods were used to train six supervised ML classifiers. The best-performing model was selected based on cross-validated metrics, including accuracy, Kappa, ROC, sensitivity, and specificity, and further interpreted using SHapley Additive exPlanations (SHAP) values. Seven co-expression modules were identified, with the red and green modules showing strong positive and negative correlations with HIV status, respectively. From the intersection of WGCNA modules, differentially expressed genes (DEGs), and PPI networks, ten hub genes were prioritized. Among the trained models, the regularized regression model (GLMNET) demonstrated the highest diagnostic performance (ROC = 0.97, accuracy = 91%). SHAP analysis highlighted GBP1, ISG15, OAS2, OAS1, and DDX60 as the most influential genes contributing to model predictions, thereby enhancing interpretability and biological relevance. By integrating transcriptomic profiling with interpretable ML, this study identifies novel gene-based biomarkers for HIV diagnosis and underscores the potential of explainable AI in advancing precision medicine approaches for infectious diseases.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.08.652807v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">BMDD: A Probabilistic Framework for Accurate Imputation of Zero-inflated Microbiome Sequencing Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhou, H., Chen, J., Zhang, X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce BMDD (BiModal Dirichlet Distribution), a novel probabilistic modeling framework for accurate imputation of microbiome sequencing data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Microbiome sequencing data are inherently sparse and compositional, with excessive zeros arising from biological absence or insufficient sampling. These zeros pose significant challenges for downstream analyses, particularly those that require log-transformation. We introduce BMDD (BiModal Dirichlet Distribution), a novel probabilistic modeling framework for accurate imputation of microbiome sequencing data. Unlike existing imputation approaches that assume unimodal abundance, BMDD captures the bimodal abundance distribution of the taxa via a mixture of Dirichlet priors. It uses variational inference and a scalable expectation-maximization algorithm for efficient imputation. Through simulations and real microbiome datasets, we demonstrate that BMDD outperforms competing methods in reconstructing true abundances and improves the performance of differential abundance analysis. Through multiple posterior samples, BMDD enables robust inference by accounting for uncertainty in zero imputation. Our method offers a principled and computationally efficient solution for analyzing high-dimensional, zero-inflated microbiome sequencing data and is broadly applicable in microbial biomarker discovery and host-microbiome interaction studies. BMDD is available at: https://github.com/zhouhj1994/BMDD.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.08.652808v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Two high-quality rose genomes underpin a novel Rosa pangenome to advance rose genomics, phylogenetics, and breeding</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yang, Z., Schijlen, E. G. W. M., Schranz, E., Zwaan, B. J., de Ridder, D., Visser, R. G. F., Arens, P. F. P., Smulders, M. J. M., van Velzen, R., Keurentjes, J. J. B., Bourke, P. M., Smit, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We have built a Rosaceae panproteome and a Rosa pangenome (spanning wild, traditional garden, and modern rose lineages) using a De Bruijn graph (DBG)-based approach.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Rosa, belonging to the family Rosaceae, encompasses more than 150 species which are widely distributed in the northern hemisphere. Renowned for their beauty, roses are cultivated throughout the world for ornamental purposes and the production of essential oils and perfumes. Despite their cultural and commercial significance, the genomic resources of wild Rosa species have not been studied comprehensively, hampering the understanding of their genetic diversity, evolutionary history, and breeding potential. Here we report on high-quality de novo genomes for Rosa sericea and Rosa rugosa. By integrating these two de novo genomes with existing public genomic resources, we have built a Rosaceae panproteome and a Rosa pangenome (spanning wild, traditional garden, and modern rose lineages) using a De Bruijn graph (DBG)-based approach. A maximum likelihood (ML) phylogeny of 18 Rosa haplotypes based on 4,367 single-copy core homology groups (genes) provided robust evolutionary inference, confirming the basal position of R. sericea, and enabled a gene-based macrosynteny analysis across the pangenome. Our analyses revealed significant genomic diversity among species, extensive variation in core gene content, and lineage-specific transposable element (TE) expansion patterns that contribute to the variation in Rosa genome size and to species-specific adaptations. The pangenome also revealed biased diversification of homology groups potentially linked to phenotypic plasticity in Rosa. Specifically, our analysis of the rose scent-related gene family, NUDX1, uncovered its evolutionary trajectory in Rosa, in which TEs insertions provided putative novel regulatory elements that facilitated adaptive evolution in metabolic pathways. This pangenomic study deepens our understanding of the genetic diversity and evolution of traits within the Rosa genus. In addition, the findings lay the foundation for future efforts to understand the genetic mechanisms driving trait evolution, which can support rose breeding.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.07.652600v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The Impact of Exogenous Mutagens on Human Mitochondrial DNA Ploidy: Analysis of Changes and Possible Protective Mechanisms.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Van Leiden, N., Potapova, N., Ree, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We study the dynamics of mitochondrial ploidy in iPSC cells in vitro following exposure to a range of chemical mutagens.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mitochondrial ploidy - the relative copy number of mitochondrial DNA per mitochondrion - may serve as an early marker of cellular stress and a potential signal of genotoxic damage. In this study, we investigate the dynamics of mitochondrial ploidy in iPSC cells in vitro following exposure to a range of chemical mutagens. Using a qPCR-based method allowing accurate quantification of mitochondrial DNA and mitochondria number, we reveal distinct changes in mitochondrial ploidy that correlate with the genotoxic impact of specific agents. Among the mutagens tested, MX, benzidine, cyclophosphamide, hydrogen peroxide, semustine and nickel (II) chloride induced the most pronounced alterations in mitochondrial DNA content and organization. These findings suggest that mitochondrial ploidy can be used as a sensitive molecular indicator of mutagenic stress, potentially reflecting mitochondrial genome maintenance and organelle adaptation mechanisms.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.09.648561v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Sex-Based Transcriptomic Differences in Psoriatic Lesions: A Comprehensive Meta-Analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Stemmer, E., Salmon-Divon, M., Sinberger, L. A., Lax, T., Shrem, G., Mor, I., Zahavi, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified 2,760 differentially expressed genes (DEGs) between sexes, highlighting gender-based molecular and cellular disparities in psoriasis, including male-enriched PI3K-Akt signaling and chemokine receptor activity, and female- enriched glycolysis and</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Psoriasis, a chronic inflammatory skin disease affecting men and women equally, presents distinct gender-based differences in severity and treatment response. While molecular mechanisms underlying psoriasis are well-studied, sex-specific differences remain largely unexplored. To address this, we conducted a meta-analysis of transcriptomic data from lesional psoriasis skin and healthy controls, comparing male and female cohorts. Our findings reveal 2,760 overlapping differentially expressed genes (DEGs) between sexes, highlighting shared pathways like IL-17 signaling and Th17 differentiation. However, sex-specific pathways emerged, including male-enriched PI3K-Akt signaling and chemokine receptor activity, and female-enriched glycolysis and AHR-NRF2 pathways. Upstream regulator analysis identified sex-specific drivers, including VEGFA activation and CFTR inhibition in males, and AHR activation and FGF21 inhibition in females. Notably, Tregs and neutrophil abundance differed by sex, aligning with disease severity trends. These results highlight critical molecular and cellular disparities, emphasizing the need for personalized, sex-specific therapeutic approaches in psoriasis management.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.07.652611v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Evaluating selection at intermediate scales within genes provides robust identification of genes under positive selection in M. tuberculosis clinical isolates</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ioerger, T. R., Shatby, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We evaluate positive selection of genes in the Mtb genome at all three levels, including individual codons, local regions within an open reading frame (clusters of residues), and the intermediate level (so-called "Window model").</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Several recent studies have reported that multiple genes in the M. tuberculosis (Mtb) genome are under positive selection, based on analysis of large collections of Mtb clinical isolates. It is inferred that mutations in these genes are adaptions to selection pressures associated with modern clinical treatment of TB. Many, but not all, of these genes under selection are related to drug resistance. Although the calculations are based on a variety of evolutionary principles, most of these studies have evaluated positive selection at the gene-level. However, positive selection can be evaluated on different scales, including individual codons, and local regions within an open reading frame (clusters of residues). In this paper, we use GenomegaMap to evaluate selection of genes in the Mtb genome at all three levels, and we present evidence that the intermediate level (so-called "Window model") provides the most credible list of candidate genes under selection. A further advantage of this approach is that it identifies specific regions within an ORF that harbor the selection signal (clusters of SNPs with a local excess of non-synonymous changes), which can be useful for structural and functional analysis. In an analysis of 2,057 Mtb clinical isolates from Moldova, we observe 53 statistically significant genes under selection, including expected drug-resistance genes, as well as resR and phoR (regulators of antibiotic tolerance, and virulence/stress response, respectively), that have previously been identified to be under selection in Mtb. To evaluate the generality of the results, the analysis was then repeated on a larger, global set of 5,195 clinical isolates from 15 other countries. While many of the same genes were under significant selection in both datasets, the global dataset also revealed positive selection of multiple ESX-1 genes, along with whiB6 and phoR, which have been shown to regulate expression of ESX-1, a type VII secretion system that plays an important role in modulating the host immune response. The selection signals for these ESX-1 genes lend support to the potential selective advantage of genetic variants affecting ESX-1 function, as adaptations to modern treatment of TB.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.07.652684v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Control with Practical Guarantees of Stationary Variance in Stochastic Chemical Reaction Networks</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> M. Zand, A., Gupta, A., Khammash, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A stochastic integral feedback controller that enables first-moment control while maintaining a tractable steady-state variance bound under specific parametric regimes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Biomolecular integral feedback controllers offer precise regulation of molecular species copy numbers, making them valuable for synthetic biology applications. Antithetic integral feedback controllers, in particular, can be effective in low-copy-number regimes with stochastic dynamics. In this work, we introduce a modified variant of this controller, called the antithetic dual-rein integral feedback motif, and analyze its performance from a stochastic perspective in the presence of intrinsic dynamic randomness. We demonstrate that our controller enables first-moment control while maintaining a tractable steady-state variance bound under specific parametric regimes. Notably, this variance bound is tunable, as it depends solely on the controller parameters. We derive these results using stochastic model-order reduction and validate them through numerical simulations. Our findings provide new insights into achieving both precise regulation and noise suppression in stochastic genetic circuits.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.12.653489v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multilayered regulation by RNA thermometers enables precise control of Cas9 expression</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Phillips, E. K., Klingeman, D. M., Guss, A. M., Eckert, C. A., Alexander, W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We demonstrate the utility of temperature sensitive RNA elements as a layer of post-transcriptional regulation to reduce the impact of promoter leak.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cas9-based genome editing technologies can rapidly generate mutations to probe a diverse array of mutant genotypes. However, aberrant Cas9 nuclease translation and activity can occur despite the use of inducible promoters to control expression, leading to extensive cell death. This background killing caused by promoter leakiness severely limits the application of Cas9 for generating mutant libraries because of the potential for population skew. We demonstrate the utility of temperature sensitive RNA elements as a layer of post-transcriptional regulation to reduce the impact of promoter leak. We observe significant temperature-dependent increases in cell survival when certain RNA thermometers (RNATs) are placed upstream of the cas9 coding sequence. We show that the most highly repressing RNAT, hsp17rep, significantly reduces population skew with a library of characterized guide RNAs (gRNAs). This information should be applicable to all Cas9-based methods and technologies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.12.651296v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Unraveling Disease-Associated PIWI-Interacting RNAs with a Contrastive Learning Methods.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xiaowen Hu, Hao Sun, Linchao Shan, Chenxi Ma, Hanming Quan, Yuanpeng Zhang, Jiaxuan Zhang, Ziyu Fan, Yongjun Tang, Lei Deng</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel computational model, iPiDA_CL, to predict potential piRNA-disease associations through contrastive learning methods, which do not require negative samples.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> PIWI-interacting RNAs (piRNAs) are a class of small, non-coding RNAs predominantly expressed in the germ cells of animals and play a crucial role in maintaining genomic integrity, mediating transposon suppression, and ensuring gene stability. Beyond their functions in reproductive cells, piRNAs also play roles in various human diseases, including cancer, suggesting their potential as significant biomarkers critical for disease diagnosis and treatment. Wet-lab methods to identify piRNA-disease associations require substantial resources and are often hit-or-miss. With advancements in computational technologies, an increasing number of researchers are employing computational methods to efficiently predict potential piRNA-disease associations. The sparsity of data in piRNA-disease association studies significantly limits model performance improvement. In this study, we propose a novel computational model, iPiDA_CL, to predict potential piRNA-disease associations through contrastive learning methods, which do not require negative samples. The model represents piRNA-disease association pairs as a bipartite graph and computes the initial embeddings of piRNAs and diseases using Gaussian kernel similarity, with features updated via LightGCN. Based on the siamese network framework, iPiDA_CL constructs online and target networks and employs data augmentation in the target network to build a contrastive learning objective that optimizes model parameters without introducing negative samples. Finally, cross-prediction methods are used to calculate specific piRNA-disease association scores. A series of experimental results demonstrate that iPiDA_CL surpasses state-of-the-art methods in both performance and computational efficiency. The application of iPiDA_CL to the miRNA-disease association dataset underscores its versatility across various ncRNA-disease association task. Furthermore, a case study highlights iPiDA_CL as an efficient and promising tool for predicting piRNA-disease associations.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40263714/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

