<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Thursday, 07 August 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Salivary Extracellular Vesicle RNA Profiling Reveals Biomarkers for Sjogrens</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chakrabortty, S. K., Xing, S., George, A., Sawicki, B., Lang, S., Nguyen, S., Cole, T. J., Mitsock, E., Ray, C. J., Zoukhri, D., Singh, M. L., Chatzis, L., Goules, A., Saridaki, M.-I., Gowrisankar, S., Tzioufas, A. G., Papas, A., Skog, J. K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A saliva-based liquid biopsy capable of diagnosing Sjogrens with high sensitivity and specificity .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Sjogrens is a chronic autoimmune disease affecting exocrine glands and is subclassified into SSA-positive (SSA+) and SSA-negative (SSA-) subtypes, with a complex diagnostic journey and an average diagnostic delay of almost 4 years. While SSA+ cases can be detected via serological testing, current assays lack specificity. For SSA- patients, no non-invasive diagnostic tools exist, and diagnosis often requires invasive lip biopsy. A saliva-based liquid biopsy capable of diagnosing both subtypes is therefore of high clinical interest. However, saliva poses challenges due to its abundant oral microbiome, which complicates unbiased biomarker discovery. In this study, we present a novel RNA sequencing workflow that efficiently depletes microbial content, enabling deep profiling of long RNAs within salivary extracellular vesicles (EVs). This approach identified both known and novel RNA biomarkers capable of diagnosing SSA+ and SSA- subtypes with high sensitivity and specificity. Moreover, we uncovered distinct RNA signatures that allow molecular stratification of Sjogrens subtypes. Pathway analysis in SSA+ cases revealed enrichment of immune and glandular pathways consistent with prior tissue-based studies, supporting the utility of salivary EVs as a non-invasive surrogate for tissue biopsy. Importantly, our data provides new molecular insights into the under-characterized SSA- subtype, laying the foundation for future mechanistic studies and facilitating their broader inclusion in clinical trials.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.05.668729v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Chronological age estimation from human microbiomes with transformer-based Robust Principal Component Analysis.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tyler Myers, Se Jin Song, Yang Chen, Britta De Pessemier, Lora Khatib, Daniel McDonald, Shi Huang, Richard Gallo, Chris Callewaert, Aki S Havulinna, Leo Lahti, Guus Roeselers, Manolo Laiola, Sudarshan A Shetty, Scott T Kelley, Rob Knight, Andrew Bartko</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Transformer-based Robust Principal Component Analysis improves age prediction accuracy from human microbiome samples, achieving the largest reduction in Mean Absolute Error for WGS skin (MAE: 8.03, 28% reduction) and 16S skin (MEE: 5.09, 14% reduction).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Deep learning for microbiome analysis has shown potential for understanding microbial communities and human phenotypes. Here, we propose an approach, Transformer-based Robust Principal Component Analysis(TRPCA), which leverages the strengths of transformer architectures and interpretability of Robust Principal Component Analysis. To investigate benefits of TRPCA over conventional machine learning models, we benchmarked performance on age prediction from three body sites(skin, oral, gut), with 16S rRNA gene amplicon(16S) and whole-genome sequencing(WGS) data. We demonstrated prediction of age from longitudinal samples and combined classification and regression tasks via multi-task learning(MTL). TRPCA improves age prediction accuracy from human microbiome samples, achieving the largest reduction in Mean Absolute Error for WGS skin (MAE: 8.03, 28% reduction) and 16S skin (MAE: 5.09, 14% reduction) samples, compared to conventional approaches. Additionally, TRPCA's MTL approach achieves an accuracy of 89% for birth country prediction across 5 countries, while improving age prediction from WGS stool samples. Notably, TRPCA uncovers a link between subject and error prediction through residual analysis for paired samples across sequencing method (16S/WGS) and body site(oral/gut). These findings highlight TRPCA's utility in improving age prediction while maintaining feature-level interpretability, and elucidating connections between individuals and microbiomes.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40770074/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Exploring the unique characteristics of genes with dual autosomal dominant and recessive inheritance: mechanisms, phenotypes and candidate identification.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shlomit Ezer, Tal Sido, Jonathan Rips, Ronit Hoffman Lipschuetz, Adina Fuchs, Bassam Abu-Libdeh, Elena Chervinsky, Nadirah S Damseh, Nada Danial-Farran, Ilham Morani, Ann Saada, Mohammed Al-Raqad, Somaya Salah, Shira Yanovsky-Dagan, Nadra Samra, Hanna Mandel, Stavit A Shalev, Hagar Mor-Shaked, Joël Zlotogora, Tamar Harel</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A list of genes that show both autosomal dominant and recessive inheritance patterns.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Autosomal dominant (AD) inheritance often arises through haploinsufficiency, dominant-negative or gain of function (GoF) effects, while autosomal recessive (AR) inheritance generally results from partial or complete loss of function (LoF). Yet, a subset of genes demonstrates both inheritance patterns. We aimed to curate a list of such 'AD/AR' genes and to propose additional candidates.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40769572/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Single Cell Multiomics Across Nine Mammals Reveals Cell Type Specific Regulatory Conservation in the Brain</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Anderson, A. G., Rogers, B. B., Barinaga, E. A., Loupe, J., WaMaina, E., Johnston, S. Q., Limbo, H. L., Gardner, E. A., Moyer, A. J., Gross, A. L., Martin, D. R., Thyme, S. B., Rizzardi, L. F., Myers, R. M., Cooper, G. M., Cochran, J. N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify and study enhancer conservation in the brain and demonstrate how regulatory evolution shapes human brain biology and disease susceptibility.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding the gene regulatory mechanisms underlying brain function is crucial for advancing knowledge of the genetic basis of neurologic diseases. Cis-regulatory elements (CREs) play a pivotal role in gene regulation, and their evolutionary conservation can offer valuable insights. Importantly, the function and evolution of CREs are affected not only by primary sequence, but also by the cis- and trans-regulatory context. However, comparative functional analyses across species have been limited, leaving how these regulatory landscapes evolve in the brain largely unresolved. Here, we generated single-nucleus multiomic (snRNA- and snATAC-seq) data from cortex tissue across nine mammalian species and identified candidate CREs (cCREs) in a cell type-specific manner. We developed a multidimensional framework of conservation to assess sites of shared function that integrates sequence, chromatin accessibility, and enhancer-gene associations. Using massively parallel reporter assays (MPRA) in human neural progenitor cells and neurons, we measured activity of cCREs including both conserved and human-specific regions. CRISPR interference validated conserved enhancer function, including at neurodevelopmentally important genes like FAM181B. Motif enrichment identified transcription factors distinguishing conserved versus recent cCREs. Linkage disequilibrium score regression indicated that both conserved and human-specific cCREs were enriched for neuropsychiatric GWAS risk, while neurodegenerative risk was confined to conserved elements. Our findings define functional dimensions of enhancer conservation and demonstrate how regulatory evolution shapes human brain biology and disease susceptibility.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.06.668931v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genome-wide chromatin recording resolves dynamic cell state changes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Takei, Y., Lay, J. A., Linton, J. M., Chadly, D. M., Ochiai, Y., Hadas, R., Perez, A. A., Blanco, M. R., Laurino, P., Guttman, M., Elowitz, M. B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present LagTag, a method for recovery of earlier and endpoint chromatin states in the same mammalian cells, which is able to record and recover dynamic chromatin state transitions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding how the chromatin state of a cell influences its future behavior is a major challenge throughout biology. However, most chromatin profiling methods are limited to endpoint assays. Here, we present LagTag, a method for recovery of earlier and endpoint chromatin states in the same mammalian cells. In this approach, transient expression of bacterial adenine methyltransferase fusions records the DNA binding profiles of chromatin-associated proteins of interest at earlier timepoints. Subsequent tagmentation and sequencing recovers the earlier chromatin profile from adenine methylation profiles, alongside endpoint profiles of endogenous chromatin-associated proteins. We verified that LagTag profiles aligned with those from established methods in mouse and human cells. More importantly, LagTag was able to record and recover dynamic chromatin state transitions during mouse embryonic stem cell differentiation, capturing transcriptional signatures from pre- and post-differentiation timepoints in the same cells. LagTag thus provides a foundation for temporally resolved chromatin profiling.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.05.668773v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Tracing spontaneously occurring mutations in Fusarium graminearum laboratory strains resulting in reduced virulence on wheat</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Svoboda, T., Kastner, F., Freitag, M., Strauss, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of Fusarium graminearum in the lab and the impact of mutations on virulence .</p><p style="font-size: 16px; text-align: justify;">Background: Fusarium graminearum is a well characterized plant pathogenic fungus which is able to infect a broad range of economically relevant crop plants. Besides yield reduction this fungus is also responsible for mycotoxin contamination of food and feed. Upon propagation under laboratory conditions, mutations may occur which would be disadvantageous for fungal fitness in nature but not in the lab where the strains are usually grown on nutrient-rich media under optimal growth conditions. In this study we characterized four phenotypically different Fusarium graminearum strains for fitness traits and compared their genomes to trace down mutations responsible for the phenotypes. Results: The four tested F. graminearum PH1-derived strains revealed differences in their phenotypic appearance and also in their secondary metabolite profiles expressed on different growth media. Also, two of the investigated strains (94 and 96) showed significantly reduced virulence on wheat upon point inoculation of flowering wheat. We identified one high impact mutation in each of the two strains. In strain 96 a loss of function mutation occurred in FGSG_00355 which has a high similarity to Ras GTPase activating proteins and consequently may have an impact on the cell cycle. Even though strain 96 showed enhanced DON production in vitro, the strain was no longer able to spread within the wheat ear in infection assays. In strain 94 we identified an insertion of an A rather at the end of FGSG_00052 leading to a frameshift and consequently mutation of the last three amino acids and a shift of the stop codon by seven amino acids. Even though knock-out of this putative transcription factor has been described by Son et al (2011) to have no impact on virulence, changes at the C-terminal region may result in changes of the binding affinity. Conclusions: We tracked down the mutations which might be responsible for the changes in phenotypic appearance, secondary metabolite profile as well as virulence. Yet, a closer biological characterization is necessary to determine the impact of these mutations on the fungus.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.06.668883v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Protein Structure Informed Bacteriophage Genome Annotation with Phold</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bouras, G., Grigson, S. R., Mirdita, M., Heinzinger, M., Papudeshi, B., Mallawaarachchi, V., Green, R., Kim, R. S., Mihalia, V., Psaltis, A. J., Wormald, P.-J., Vreugde, S., Steinegger, M., Edwards, R. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Phold, an annotation framework utilising protein structural information that combines the ProstT5 protein language model and structural alignment tool Foldseek.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Bacteriophage (phage) genome annotation is essential for understanding their functional potential and suitability for use as therapeutic agents. Here we introduce Phold, an annotation framework utilising protein structural information that combines the ProstT5 protein language model and structural alignment tool Foldseek. Phold assigns annotations using a database of over 1.36 million predicted phage protein structures with high quality functional labels. Benchmarking reveals that Phold outperforms existing sequence-based homology approaches in functional annotation sensitivity whilst maintaining speed, consistency and scalability. Applying Phold to diverse cultured and metagenomic phage genomes shows it consistently annotates over 50% of genes on an average phage and 40% on an average archaeal virus. Comparisons of phage protein structures to other protein structures across the tree of life reveals that phage proteins commonly have structural homology to proteins shared across the tree of life, particularly those that have nucleic acid metabolism and enzymatic functions. Phold is available as free and open-source software at https://github.com/gbouras13/phold.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.05.668817v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">On the Utility of Virtual Staining for Downstream Applications as it relates to Task Network Capacity</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sengupta, S., Xu, J., Nguyen, P., Brooks, F. J., Liu, Y., Anastasio, M. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the utility of virtual staining is largely dependent on the ability of the segmentation or classification task network to extract meaningful task-relevant information, which is related to the concept of network capacity.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Virtual staining, or in-silico-labeling, has been proposed to computationally generate synthetic fluorescence images from label-free images by use of deep learning-based image-to-image translation networks. In most reported studies, virtually stained images have been assessed only using traditional image quality measures such as structural similarity or signal-to-noise ratio. However, in biomedical imaging, images are typically acquired to facilitate an image-based inference, which we refer to as a downstream biological or clinical task. This study systematically investigates the utility of virtual staining for facilitating clinically relevant downstream tasks (like segmentation or classification) with consideration of the capacity of the deep neural networks employed to perform the tasks. Comprehensive empirical evaluations were conducted using biological datasets, assessing task performance by use of label-free, virtually stained, and ground truth fluorescence images. The results demonstrated that the utility of virtual staining is largely dependent on the ability of the segmentation or classification task network to extract meaningful task-relevant information, which is related to the concept of network capacity. Examples are provided in which virtual staining does not improve, or even degrades, segmentation or classification performance when the capacity of the associated task network is sufficiently large. The results demonstrate that task network capacity should be considered when deciding whether to perform virtual staining.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.04.668552v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Chromosome-Scale, Telomere-to-Telomere Assembly of Winged Bean (Psophocarpus tetragonolobus (L.) DC.) Genome Reveals Lipid Biosynthetic Hotspots and Gene Family Dynamics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tribhuvan, K. U., Singh, N. K., Singh, B. K., Pandey, A., Kumar, S., Bishi, S. K., K, K. K., Sahu, T. K., Pattanayak, A., Badhana, V. P., Rakshit, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We report a telomere-to-telomere, chromosome-scale genome assembly of the winged bean (Psophocarpus tetragonolobus) and its first chromosomal ideotype map for the species.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Winged bean (Psophocarpus tetragonolobus) is a nutritionally rich but cytologically and genomically underexplored legume with potential for climate-resilient agriculture. We report a telomere-to-telomere, chromosome-scale genome assembly (~697.69 Mb; N50 = 85.98 Mb; ~98.25% of the cytometric estimate) of the cultivar AKWB1 using a combination of PacBio HiFi, Illumina, BioNano, and Hi-C technologies. Cytogenetics revealed a diploid karyotype (2n = 18) with exclusively metacentric chromosomes, establishing the first chromosomal ideotype map for the species. The high-quality assembly anchors 98.28% of bases to nine pseudochromosomes and includes 53,745 annotated protein-coding genes, with 97.6% BUSCO completeness. Repeat elements comprise ~60% of the genome, dominated by LINEs, LTRs, and DNA transposons. KEGG and GO annotations reveal diverse biosynthetic capabilities across metabolic and stress-response pathways. Comparative genomics with nine legumes demonstrates conserved chromosomal synteny with Glycine max, absence of recent whole-genome duplication, and a contraction-dominated gene family history. Lipid metabolism analysis identified 750+ genes across 12 KEGG pathways, including fatty acid biosynthesis, elongation, and triacylglycerol assembly. Chromosomes 2, 3, and 5 showed enrichment of lipid biosynthetic loci, suggesting evolutionary hotspots. This reference genome positions P. tetragonolobus as a valuable model for studying genome evolution and engineering underutilized legumes for oil yield, protein enrichment, and climate adaptability.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.04.668583v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MOH: a novel multilayer multi-omics heterogeneous graph for single-cell clustering</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dong, Y., Chen, C., Shi, J., Hu, Y., Cao, X., Dong, Y., Zhang, X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce MOH, a novel single-cell clustering algorithm based on a multilayer multi-omics heterogeneous graph.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cell clustering is crucial in single-cell multi-omics research for identifying distinct cellular populations. Although there has been progress in integrating multi-omics data for clustering, combining more than two types of omics data remains challenging due to the diversity and heterogeneity of these datasets. Traditional approaches typically use heterogeneous graphs that integrate only two types of omics data, constructing graphs with genes and cells as nodes and a single type of edge representing their relationships. However, this method has limitations as it overlooks cell-cell interactions and struggles to capture complex cellular dynamics. Additionally, the graph structure must be redesigned whenever new omics data are introduced, limiting the scalability of these models. To address these issues, we introduce MOH, a novel single-cell clustering algorithm based on a multilayer multi-omics heterogeneous graph. MOH integrates three key single-cell omics types: scRNA-seq, scATAC-seq, and spatial transcriptomics. It constructs a multilayer heterogeneous graph to simultaneously extract and enhance representations from all three omics layers, incorporating both intra-layer and inter-layer edges to capture association and similarity relationships. This enriched representation leads to an accurate clustering results. Extensive experiments show that MOH outperforms six state-of-the-art methods on unsupervised clustering metrics, offering a precise and comprehensive analysis with consistent improvements across all evaluation criteria. Moreover, downstream analyses validate the results, revealing novel biological insights into immune disorder complications in cancer, cancer drug repurposing, and new signaling pathways, which merit further investigation and validation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.04.668248v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Metabolomics Ex/GWAS in the Amish reveals novel insights into cardiometabolic disease pathways</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jallow, M. W., Tzoneva, G., Ye, B., Backman, J., Zavala, V. A., Sosina, O., Kapoor, M., Verweij, N., Landheer, K., Geraghty, B., Marchini, J., Locke, A., Bovijn, J., Nielsen, J., Lotta, L., Regeneron Genetics Center,, Montasser, M. E., Lewis, J., Mitchell, B. D., Shuldiner, A. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We conducted an exome and genome-wide association study (Ex/GWAS) of 1,015 metabolites in serum samples from 5,981 Amish adults and identified 149 functional or likely functional genetic variants, based on CADD scores significantly associated (P 10-fold enriched in the Am</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We conducted an exome and genome-wide association study (Ex/GWAS) of 1,015 metabolites in serum samples from 5,981 Amish adults. We identified 149 functional or likely functional genetic variants, based on CADD scores significantly associated (P 10-fold enriched in the Amish versus Europeans in gnomAD. We discovered novel associations involving a PCK2 splice-donor variant (rs138881435) and metabolites important for energy metabolism and mitochondrial function. In UK Biobank participants, this variant was associated with increased longitudinal relaxation time (T1) values on liver MRI suggesting increased inflammation and fibrosis. Furthermore, we found novel associations involving ENPEP stop-gain variant (rs33966350) and peptide and amino acid metabolite levels, implicating the role of ENPEP in blood pressure regulation. Additional known and novel genetic variant-metabolite associations were identified including genes involved in Amish-enriched Mendelian diseases and cardiometabolic traits, providing insights into underlying mechanisms of these disorders.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.04.667775v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Gene model for the ortholog of tgo in Drosophila miranda</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jones, G. M., Giunta, A. A., Mastandrea, H., Wittke-Thompson, J. K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A gene model for the ortholog of tango (tgo) in the April 2013 (UC Berkeley DroMir_2.2) Genome Assembly (GenBank Accession: GCA_000269505.2).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Gene model for the ortholog of tango (tgo) in the April 2013 (UC Berkeley DroMir_2.2/DmirGB2) Genome Assembly (GenBank Accession: GCA_000269505.2) of Drosophila miranda. This ortholog was characterized as part of a developing dataset to study the evolution of the Insulin/insulin-like growth factor signaling pathway (IIS) across the genus Drosophila using the Genomics Education Partnership gene annotation protocol for Course-based Undergraduate Research Experiences.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.04.668519v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Transcriptional response to warm temperatures is confounded by organ-specificity</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hua, J., Gupta, S., Reis, R. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We found a surprisingly poor consensus among published whole seedling transcriptomic datasets for temperature-responsive thermomorphogenesis, with strong organ-specific transcriptional responses to warmth.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Climate change is causing increases in global average temperatures with detrimental consequences to food security, in part because most plants are sensitive to increases of even 1-2 degrees Celsius. Our molecular understanding of how plants perceive and acclimatise to mild increases in temperature (aka thermomorphogenesis) have largely been built upon transcriptomic analyses of whole seedling using the plant model Arabidopsis thaliana. In this work, we demonstrate that this widely used approach has significant limitations that may have skewed our understanding of gene regulation in thermomorphogenesis.We found a surprisingly poor consensus among published whole seedling transcriptomic datasets. On average, nearly 60% of reported differentially expressed genes (DEGs) are unique to a single study. To investigate the source of this variability, we performed a controlled comparison of RNA sequencing data from whole seedlings versus dissected organs (root, hypocotyl, and cotyledon), which revealed strong organ-specific transcriptional responses to warmth. Critically, we found that ~70% of DEGs identified in whole seedlings were not differentially expressed in any of the individual organs, suggesting that whole seedling data is confounded by the pooling of distinct, and sometimes opposing, organ-level responses. These findings are further supported by proteomic analysis. Our work provides a word of caution and clear guidance for the field to advance when producing and interpreting RNA sequencing data for plant response to warm temperatures.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.05.668523v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ZmCRY1s interact with GL2 in a blue light dependent manner to regulate epidermal wax composition in Zea mays</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhao, Z., Feng, F., Liu, Y., Liu, Y., Wang, F., Ni, Y., Liang, H., Hu, W., Wang, S., Hao, Y., Li, X., Li, J., Wang, J., Zhang, P., Liu, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We discovered that ZmCRYs mediated blue light-enhanced UV-B stress tolerance in Zea mays by upregulating the expression of genes involved in epidermal wax accumulation and UV- B stress tolerance-related metabolites such as phenylpropanoid, flavonoid</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cryptochromes (CRYs) are photolyase-like blue-light/ultraviolet-A (UV-A) receptors that regulate diverse aspects of plant growth. Maize (Zea mays), a major crop often grown under high UV-B radiation, possesses four copies of CRY. However, it remains unclear whether the multiple copies of CRY in maize have evolved to improve UV tolerance or to acquire new functions. In this study, CRISPR-Cas9-engineered Zmcry mutants were used to investigate the functions of four cryptochromes (ZmCRYs) in maize. The findings revealed that ZmCRYs play a redundant role in mediating blue light signaling and in inhibiting the elongation of the mesocotyl. The results also demonstrated that ZmCRYs mediated blue light-enhanced UV-B stress tolerance in Zea mays by upregulating the expression of genes involved in UV-B stress tolerance-related metabolites such as phenylpropanoid, flavonoid, and fatty acid biosynthesis. Furthermore, blue light was found to influence both the accumulation and composition of epidermal waxes, suggesting that blue light enhances epidermal wax accumulation for UV-B stress tolerance. Additionally, it was discovered that ZmCRY1 directly interacted with GLOSSY2 (GL2) in a blue light dependent manner to mediate blue light promoted C32 aldehyde accumulation, shedding new light on the enigma of aldehyde-forming pathway. These results highlight the critical roles of ZmCRY1s in mediating blue light regulated epidermal wax biosynthesis and UV-B tolerance in Zea mays.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.06.668858v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Single-nucleus transcriptome analyses uncover a dynamic transcriptional landscape of soybean roots in response to soybean cyst nematode infection</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bai, P., Jia, X., Chen, L., Han, Y., Han, S., Weng, L., Feng, X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified and transcriptionally profiled syncytial cells in soybean cyst nematode-infected roots, and validate the spatial expression of known SCN-resistant genes, and identify promising molecular targets for engineering nematodes-resistant soybean varieties.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Soybean cyst nematode (SCN, Heterodera glycines) causes substantial yield losses by inducing the formation of specialized feeding structure called syncytia, which arise from extensive reprogramming of host root cells. Despite their pivotal role in SCN parasitism, the molecular mechanisms governing syncytium formation and function remain poorly understood, largely due to the technique challenges in precisely isolating these rare cell types. Here, we employed single-nucleus RNA sequencing (snRNA-seq) to generate a comprehensive, high-resolution transcriptomic atlas of SCN-infected soybean roots, comparing two resistant cultivars (PI 88788 and Forrest) with a susceptible cultivar (Williams 82) across multiple infection stages. By leveraging the well-characterized syncytium marker GmSNAP18, we successfully identified and transcriptionally profiled syncytial cells, and validate the spatial expression of known SCN-resistant genes. Our analyses revealed strong induction of resistance-associated genes specifically enriched in syncytia from resistant cultivars. Functional validation further demonstrated that overexpression of three novel candidate genes significantly suppressed SCN development. Pseudotime trajectory analyses traced syncytium development back to procambial cell origins, uncovering distinct, cultivar-specific developmental pathways. Collectively, this work provides a foundational single cell transcriptomic resource, advances our understanding of syncytial biology and SCN resistance mechanisms, and identifies promising molecular targets for engineering nematode-resistant soybean varieties.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.06.668855v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AT-HOOK-MOTIF NUCLEAR LOCALIZED 15 extends plant longevity by binding at poorly accessible, epigenetic mark-depleted chromatin that surrounds transcribed regions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Offringa, R., Chouaref, J., Luden, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that induction of AHL15 activity causes strong and rapid changes in transcription, with the majority of the differentially expressed genes being downregulated but without directly affecting chromatin accessibility, resulting in developmental defects.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Members of the AT-HOOK MOTIF NUCLEAR LOCALIZED (AHL) gene family have been shown to play important roles in plant development. In Arabidopsis thaliana, one member of this family, AHL15, induces somatic embryogenesis and extends plant longevity when overexpressed - the latter through strong repression of several ageing-related developmental transitions. However, its direct target genes and the mechanisms by which it regulates their expression have remained elusive to date. In this study we identified the genome-wide DNA binding sites of AHL15, and show that AHL15 binds throughout the genome at AT-rich sequences near the transcription start- and end sites in regions depleted of epigenetic marks. We show that induction of AHL15 activity causes strong and rapid changes in transcription, with the majority of the differentially expressed genes being downregulated but without directly affecting chromatin accessibility, resulting in developmental defects. In addition, AHL15 binding to regions near the transcription start and end sites was enhanced at genes that were differentially expressed upon AHL15 induction, and was especially strong near the transcription start site of upregulated genes and near the transcription end site of downregulated genes. Finally, we show that AHL15 shares binding sites with the chromatin architectural protein GH1-HMGA2/HON5, which was previously shown to alter transcription by disrupting gene loop formation. Together, our findings suggest that AHL15 affects the expression of its target genes by regulating the 3D organization rather than the accessibility of chromatin.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.05.668696v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A scalable cell-free manufacturing platform for two-step bioproduction of immunogenic conjugate vaccines</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wong, D. A., Aw, R., Hulbert, S. W., Qin, Y., Shaver, Z. M., Myers, K. A., Karim, A. S., DeLisa, M. P., Jewett, M. C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present an optimized, modular cell-free platform for the synthesis and purification of conjugate vaccines and demonstrate strong immunogenicity of pneumococcal conjugates.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Rapid and decentralized vaccine production is essential to ensure global preparedness against emerging and re-emerging infectious diseases. Cell-free gene expression systems, which can be freeze-dried for long-term storage and re-activated for point-of-use synthesis, offer a promising solution to address this need. However, scalable cell-free production of conjugate vaccines-highly effective tools against bacterial infections-has been hindered by low yields and inefficient glycosylation. Here, we present an optimized, modular cell-free platform for the synthesis and purification of conjugate vaccines. By decoupling cell-free protein expression from in vitro glycosylation in a two-step approach, we achieve >85% glycosylation efficiency and up to ~450 mg/L of glycoprotein. We apply this platform to manufacturing protein-polysaccharide conjugates composed of vaccine carrier proteins covalently modified with polysaccharide antigens from enterotoxigenic Escherichia coli O78 and Streptococcus pneumoniae serotype 4. Our workflow produced conjugate vaccine candidates in under 5 days with >87% product purity and low endotoxin levels suitable for preclinical evaluation. Immunization of mice with the pneumococcal conjugate vaccine induced a strong IgG response against the S. pneumoniae serotype 4 capsular polysaccharide, confirming the immunogenicity of the conjugate. We anticipate that this cell-free platform will advance efforts in decentralized manufacturing and rapid response to bacterial disease threats.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.05.668792v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Federated Learning for ICU Mortality Prediction: Balancing Accuracy and Privacy in a Multi-Hospital Setting</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Benhammou, Y., Kalyan, S., Kumar, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A federated model for ICU mortality prediction using Federated Learning (FL), which enables distributed model training without sharing sensitive patient data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Intensive care units (ICUs) manage critically ill patients whose clinical outcomes rely on timely and accurate decision-making. Predictive modeling using electronic health records (EHRs) has shown promise in forecasting adverse events such as in-hospital mortality. However, constructing robust and generalizable models typically requires large-scale, multi-institutional datasets. Privacy regulations such as HIPAA and GDPR make centralized data aggregation across hospitals challenging, posing a barrier to collaborative healthcare research. In this study, we explore a privacy-preserving approach for ICU mortality prediction using Federated Learning (FL), which enables distributed model training without sharing sensitive patient data. Leveraging a curated subset of the eICU Collaborative Research Database---a multi-center dataset of over 200,000 ICU admissions---we simulate a real-world federated learning scenario using data from three distinct hospitals comprising 5,000 ICU stays. We design a federated logistic regression framework and evaluate its performance against independently trained local models. Our findings indicate that the federated model consistently outperforms local baselines, achieving improved accuracy and AUC while maintaining strict data privacy. Our work delivers a reproducible and scalable framework for privacy-preserving healthcare AI and demonstrates the practical feasibility of deploying federated models for critical outcome prediction in ICU settings.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.04.668527v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The ornithine-arginine cycle supported a toxic, metalimnic Planktothrix rubescens bloom</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zepernick, B. N., Niknejad, D. J., Chase, E. E., Abiodun, B. A., Adler, M. J., Houghton, K. A., Olavesen, J. L., Sarumi, Q., Truchon, A. R., Walton, J. L., Cheshire, J. H., Stanislawcyzk, K., Hart, L. N., Paerl, H. W., Chaffin, J. D., Boyer, G. L., Castro, H. F., Campagna, S. R., Bullerjahn, G. S., Wilhelm, S. W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a deep-discrete partitioning of the arginine deiminase-mediated ornithine-arginine cycle (OAC) in Planktothrix rubescens, which could influence the fitness of toxigenic cyanobacterial blooms in fresh waters</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Planktothrix rubescens is distinct from other cyanobacterial harmful algal bloom (cHAB) genera: the crimson-red cHAB thrives in the cold, low-light, nutrient-limited metalimnion. Studies have attributed this ecological success to buoyancy regulation, low-light adaptations, and the uptake of nitrogen-rich amino acids. Yet, it remains to be mechanistically determined how this cHAB maintains physiological nutrient quotas in the metalimnion due to limited in situ molecular studies. We employed metagenomics and metabolomics to investigate a toxigenic P. rubescens bloom in Meads Quarry (Knoxville, TN, USA) observed in two separate years. Our results suggest a perennial, metalimnic P. rubescens population may exist, with spring turnover facilitating seasonal migration to the epilimnion. Although P. rubescens dominated the epilimnion and metalimnion, intracellular metabolite pools grouped by depth and suggested depth-discrete partitioning of the arginine deiminase-mediated ornithine-arginine cycle (OAC, i.e., urea cycle). While the arginine influx driving the OAC is unclear, we hypothesize this input is provided via the uptake of urea or nitrogen-rich amino acids. Further, we demonstrate arginine deiminase (argE) is broadly distributed in Planktothrix genera and known microcystin producers, suggesting argE-mediated arginine recycling via the OAC may influence the fitness of toxigenic cHAB genera which require ample nitrogen to synthesize microcystins. Cumulatively, our results serve as a case study to provide insight on the metabolic pathways driving the ecological success of metalimnic P. rubescens blooms. On a broader scale, this work strengthens the case that alternative nitrogen metabolism (including urea utilization, amino acid uptake, and the OAC) is a driver of toxigenic cyanobacterial blooms in fresh waters.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.04.668501v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Evolutionary and Structural Bioinformatics Reveal GPR89 as a Conserved Solute Carrier Transporter</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Quercia-Raty, C. A., Robeson, L., Brauchi, S. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comprehensive bioinformatic analysis of GPR89 in Eukarya by integrating phylogenetic reconstruction, sequence conservation, and structural modeling.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Also called the Golgi pH regulator (GPHR), GPR89 is an orphan membrane protein found in nearly all eukaryotic lineages. Despite its broad phylogenetic distribution, the evolutionary history, structural diversity, and function of GPR89 remain poorly understood. In this study, we present a comprehensive bioinformatic analysis of GPR89 in Eukarya by integrating phylogenetic reconstruction, genomic synteny, sequence conservation, and structural modeling. While GPR89 is typically encoded as a single-copy gene, we identified lineage-specific duplications in both vascular plants and vertebrates. In contrast to the large sequence conservation, large differences can be observed in whether plants and animals preserve the gene structure flanking GPR89. Structural phylogenetic clustering places GPR89 within the solute carrier (SLC) superfamily, supporting previous reports claiming transporter-like activity rather than GPCR signaling. Predicted structures reveal a unique intracellular hairpin loop and a conserved transmembrane core compatible with transport activity. Our findings provide a unifying framework for interpreting the divergent functional roles reported for GPR89 and establish it as a structurally conserved, evolutionarily stable member of the SLC superfamily.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.04.668498v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">denim: an R package for deterministic compartmental models with flexible dwell time distributions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ong, T. P., Phan, A. T. Q., Ha, L. M., Choisy, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a deterministic, continuous-variable, discrete-time compartmental modelling approach that allows full flexibility on the dwell time distributions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Compartmental models are widely used for dynamical systems where states are discrete such as in infectious diseases epidemiology with the so-called SIR or Susceptible-Infectious-Recovered framework. For mathematical simplicity, rates of transition between compartments are generally assumed to be independent of the dwell time (a.k.a. secondary time scale in survival analysis): they are either constant or dependent on the epidemiological time (a.k.a. primary time scale in survival analysis) only, either directly (e.g. environmental or behavioral forcings in epidemiological models) or indirectly through dependence on other variables of the system (e.g. the force of infection in epidemiological models). In some domains of application, this memoryless assumption leads to distributions of dwelling times that are incompatible with those observed on data is this can lead to serious problems since the model prediction are highly sensitive on the exact shape of these distributions. Here we propose a deterministic, continuous-variable, discrete-time compartmental modelling approach that allows full flexibility on the dwell time distributions. The accompanying denim package provides a user-friendly interface to implement our proposed method through a dedicated language for model definition. The package is open-source and available on CRAN. With more detailed data on the clinical process of infections becoming available, the denim package will be extremely useful for building more realistic epidemiological models providing more accurate projections.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.05.668369v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multi-omic Profiling of Senescence Following Myocardial Infarction Reveals Dynamic Characteristics in Cardiac Remodeling</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Deng, J., Wang, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify a gradual increment number of senescent cells within the ischemic heart, peaking at day 7 post- MI, in both wild-type and p16Ink4a-CreERT2-mT/mG senescence reporter mice.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Premature senescence is essential for tissue remodeling. Myocardial infarction (MI) induces pathological cardiac remodeling through fibroblast-driven extracellular matrix (ECM) production. The role of senescence in MI-induced remodeling process remains elusive. Here we identify a gradual increment number of senescent cells within the ischemic heart, peaking at day 7 post- MI, in both wild-type and p16Ink4a-CreERT2-mT/mG senescence reporter mice. Lineage tracing shows that senescent cells transition to non-senescent state within 4 weeks after MI. We perform single-nucleus (sn) Multiome and fluorescence-based spatial transcriptomics analyses to profile senescent cells. We next generate a reference (query dataset) based on SPiDER- {beta}Gal/p16-EGFP positivity and map it back to the snMultiome dataset. We then deconvolute senescent cells in the integrated dataset using multiple computational algorisms. Through these approaches, we reveal that fibroblasts and its subpopulation-late myofibroblasts (MF)-constitute a major proportion of senescent cells, which functionally reduce ECM production. Importantly, ischemia-induced senescent MF show less soluble collagen production compared to TGF-{beta}1- induced non-senescent MF in vitro. At the functional level, depletion of senescent cells in vivo augments fibrosis and worsens cardiac myopathy post-MI. Our findings highlight the transient nature of senescent cells in the heart and underscore the importance of dynamic regulation of senescent cells post-MI.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.04.667980v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">BOLD-GPCRs: A Transformer-Powered App for Predicting Ligand Bioactivity and Mutational Effects Across Class A GPCRs</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Provasi, D., Konovalov, K., Riina, N., Cullen, O., Filizola, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce BOLD-GPCRs, a deep learning framework designed to enhance the prediction of ligand bioactivity across class A GPCR family, especially for poorly characterized receptors.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> G protein-coupled receptors (GPCRs) are important targets for drug discovery owing to their ability to respond to a broad range of stimuli and their involvement in numerous pathologies. Although traditional ligand-based and structure-based approaches have facilitated the development of effective therapeutics for many GPCRs, these approaches often fall short when applied to receptors with limited ligand or structural data. This limitation highlights the critical need for advanced strategies capable of accurately predicting ligand bioactivity across the entire GPCR family, especially for understudied receptor subtypes. In this study, we introduce BOLD-GPCRs (BERT-Optimized Ligand Discovery for GPCRs), a deep learning framework designed to enhance the prediction of ligand bioactivity across class A GPCRs. Accessible via a user-friendly web interface, BOLD-GPCRs employs transfer learning and leverages curated datasets of known class A GPCR ligands, receptor sequences, and signaling-relevant mutations. By integrating dense neural network classifiers with transformer-based protein language models, BOLD-GPCRs captures complex relationships between receptor sequence/function and ligand activity. Our results demonstrate that BOLD GPCRs achieves robust predictive performance for both ligand bioactivity and mutational effects across a broad range of class A GPCRs, underscoring its potential as a valuable tool for ligand discovery, especially for poorly characterized receptors.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.04.668547v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

