<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Monday, 13 October 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DEEPReditor-CMG: A deep-learning-based predictive RNA editor for crop mitochondria genomes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Qin, S., Xu, Z., Wang, Y., Wang, Z., Cao, Y., Hou, C., Yang, J., Liu, Y., Xiao, Y., Dai, Z., Zhou, W., Li, Q.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A deep neural network-based model for C-to-U RNA editing in crops .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> C-to-U RNA editing is a crucial post-transcriptional modification in plant organelles, yet accurately predicting its sites, especially in crop mitochondrial genomes, remains a significant challenge. Here, we present DEEPReditor-CMG, a novel deep-learning-based predictive model specifically designed for this task. We constructed a comprehensive dataset from 13 crop species and developed a rigorous hyperparameter optimization framework to build a convolutional neural network (CNN) model. DEEPReditor-CMG demonstrated superior performance, achieving the highest AUC of 0.968 and ACC of 0.919, significantly outperforming existing methods. Furthermore, by leveraging the model's features, we systematically analyzed the conservation of RNA editing mechanisms, revealing that phylogenetically closer species exhibit more similar editing patterns. This study not only provides a powerful and accurate tool (complete with user-friendly software) for the plant science community but also offers novel evolutionary insights into the RNA editing machinery in crops.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.09.681429v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrative Computational Analysis Reveals H. pylori GroEL as a Stabilizer of Neurotoxic Amyloid-β Oligomers</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Abdullah, W. M., Ahmad, I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel mechanism whereby H. pylori GroEL directly stabilizes soluble A{beta} oligomers and propose that this stabilization inhibits their aggregation into plaques while paradoxically prolonging the lifetime of neurotoxic species, potentially increasing neurodegeneration.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Alzheimer's disease (AD) is characterized by amyloid-{beta} (A{beta}) aggregation, with soluble oligomers implicated as the most neurotoxic species. Recent evidence suggests microbial infections, including Helicobacter pylori, contribute to AD pathogenesis. This study investigates the role of H. pylori GroEL, a conserved chaperonin found in bacterial outer membrane vesicles (OMVs), in stabilizing toxic A{beta} oligomers. A pan-genome analysis of 353 H. pylori strains identified GroEL as a highly conserved protein present in 83% of strains, which supported its widespread relevance. We structurally modelled a conserved 27-amino acid GroEL fragment and docked it against the A{beta} (1-42) tetramer. Interaction analysis revealed stabilizing salt bridges, hydrogen bonds, and extensive non-bonded contacts within the GroEL-A{beta} complex. Molecular dynamics simulations (50 ns) demonstrated that GroEL binding enhanced A{beta} oligomer stability, evidenced by reduced structural deviations and a more extensive hydrogen bonding network compared to A{beta} oligomer alone. These computational findings support a novel mechanism whereby H. pylori GroEL directly stabilizes soluble A{beta} oligomers. We hypothesize that this stabilization inhibits their aggregation into plaques while paradoxically prolonging the lifetime of neurotoxic species, potentially increasing neurodegeneration through pathways distinct from canonical amyloid deposition. This highlights the complex role of bacterial proteins in AD and underscores the need for experimental validation of GroEL-A{beta} interactions as a potential therapeutic target.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.12.679596v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">In silico analysis of the Chikungunya virus and SARS-CoV-2 Macrodomain</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tiwari, V., Rachman, M., Huang, K., Echeverria, I., Sali, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We rationalize this difference in the ligand binding affinity in terms of three differences in the binding site structure and dynamics in ChikV Mac1 by studying its conformational heterogeneity, water affinity, and smaller number of Mac1 residues.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The macrodomain of the Chikungunya virus NSP3 protein (ChikV Mac1) hydrolyzes mono-ADP ribose post-translational modifications on the human host proteins. Mutations in Mac1 reduce ChikV virulence. Thus, ChikV Mac1 is a viral drug target. While no potent ChikV Mac1 inhibitors are available, high-affinity inhibitors were developed for Mac1 in Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). Here, we rationalize this difference in the ligand binding affinity in terms of three differences in the binding site structure and dynamics. First, the apo ChikV Mac1 binding site exhibits substantially more conformational heterogeneity than that of SARS-CoV-2, based on microsecond-scale molecular dynamics simulations; it also has a less druggable binding site, based on program SiteMap. Second, water binding sites overlapping with the ligand binding site in ChikV Mac1 are predicted by program WaterMap to have a stronger affinity for water molecules than the corresponding sites in SARS-CoV-2 Mac1, thus decreasing ligand binding affinity. Finally, a smaller number of Mac1 residues interacts persistently with ADP- ribose in holo ChikV Mac1 than in SARS-CoV-2. With these rationalizations in hand, we designed ligands of ChikV Mac1 using a fragment growth strategy; subsequent molecular dynamics simulations of a representative ligand in complex with ChikV Mac1 substantiated our design.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.10.681726v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Sustainability During Instability: Long-Lived Life Science Databases and Science Funding Outlook in the United States</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Imker, H. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of online life science databases in the United States that had previously received support from U.S. federal agencies.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> For decades, life science researchers have had cost-free, unrestricted access to data through online databases. However, the sustainability of even well-established resources was already tenuous, and abrupt changes in science funding in the United States seems poised to exacerbate these challenges. This study employed a multiple mini case study approach, triangulating semi-structured interviews with supplemental documentation to investigate 9 diverse, long-standing databases that had previously received support from U.S. federal agencies. The research explored each database's purpose and use, examined current and emerging funding strategies, and considered the potential consequences if any of these databases were forced to shut down. Participating databases include: 1) BHL: Biodiversity Heritage Library, 2) MorphoBank, 3) OMIM: Online Mendelian Inheritance in Man, 4) ORDB: Olfactory Receptor Database, 5) rrnDB: ribosomal RNA operon copy number database, 6) VEuPathBD: Eukaryotic Pathogen, Vector, and Host Informatics Resources, 7) WormAtlas, and 8-9) two databases that wished to remain anonymous. Findings revealed anticipation of higher barriers to data access and reuse, loss of subject matter expertise now and into the future, and lost or interrupted opportunities. Additionally, real impacts have already begun through redirection of energy, abrupt reductions in support, and increased competition for funding. Sustainability models in light of the current funding outlook in the US are discussed.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.08.680785v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Structure-Guided Kinase-Transcription Factor Interactome Atlas Reveals Docking Landscapes of the Kinome</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kim, A.-R., Huang, K., Johnson, J. L., Yaron-Barir, T. M., Wang, K., Cantley, L. C., Hu, Y., Perrimon, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a structure-guided atlas of the human and Drosophila kinome, built by applying a new interface-aware scoring framework (iLIS) to AlphaFold-Multimer predictions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein kinases orchestrate cellular processes through phosphorylation, yet the structural basis for their specific binding partner interactions remains largely unmapped. Here, we present a structure-guided atlas of the human and Drosophila kinome, built by applying a new interface-aware scoring framework (iLIS) to AlphaFold-Multimer predictions. The resulting atlas recapitulates hallmark sequence preferences, confirms previously reported and functionally related protein-protein interactions (PPIs), and uncovers previously unrecognized docking interactions. Notably, our analysis predicts a potentially widespread docking motif on homeodomain (HD) transcription factors for interaction with basophilic kinases. We validate selected predictions in cells and in vivo and demonstrate the atlas utility by mechanistically dissecting a key pathway controlling lipid metabolism. This resource provides a blueprint for dissecting signaling networks and for the rational design of docking-site-specific kinase modulators.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.10.681672v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genetic elements promote retention of extrachromosomal DNA in cancer cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sankar, V., Hung, K. L., Gnanasekar, A., Wong, I. T.-L., Shi, Q., Kraft, K., Jones, M. G., He, B. J., Yan, X., Belk, J. A., Liu, K. J., Agarwal, S., Wang, S. K., Henssen, A. G., Mischel, P. S., Chang, H. Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify a family of human genomic elements, termed retention elements, that tether episomes to mitotic chromosomes to increase ecDNA transmission to daughter cells.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Extrachromosomal DNA (ecDNA) is a prevalent and devastating form of oncogene amplification in cancer. Circular megabase-sized ecDNAs lack centromeres and segregate stochastically during cell division yet persist over many generations. EcDNAs were first observed to hitchhike on mitotic chromosomes into daughter cell nuclei over 40 years ago with unknown mechanism. Here we identify a family of human genomic elements, termed retention elements, that tether episomes to mitotic chromosomes to increase ecDNA transmission to daughter cells. We develop Retain-seq, a genome-scale assay that reveals thousands of human retention elements conferring generational persistence to heterologous episomes. Retention elements comprise a select set of CpG-rich gene promoters and act additively. Live-cell imaging and chromatin conformation capture show that retention elements physically interact with mitotic chromosomes at regions which are mitotically bookmarked by transcription factors and chromatin proteins, intermolecularly recapitulating promoter-enhancer interactions. Multiple retention elements are co-amplified with oncogenes on individual ecDNAs in human cancers and shape their sizes and structures. CpG-rich retention elements are focally hypomethylated; targeted cytosine methylation abrogates retention activity and leads to ecDNA loss, suggesting that methylation-sensitive interactions modulate episomal DNA retention. These results highlight the DNA elements and regulatory logic of mitotic ecDNA retention. Amplifications of retention elements promote the maintenance of oncogenic ecDNA across generations of cancer cells, revealing the principles of episome immortality intrinsic to the human genome.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.10.681495v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Lineage-determining transcription factors constrain cohesin to drive multi-enhancer oncogene regulation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Faryabi, R. B., Zhou, Y., Jay, A., Burget, N., Friedrich, T., Yoon, S., Alsing, J., Nir, G., Grosschedl, R., Vahedi, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the B-cell-lineage-determining transcription factor EBF1 preferentially positions multiple enhancers at loci containing sparsely distributed genes essential for B- cell identity and oncogenesis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Multiple enhancers, often located across vast genomic distances, regulate key genes. However, how chromatin topology organization at individual alleles enables cell-type-restricted multi-enhancer gene regulation remains unclear. Using acute protein degradation and time-course population-average chromatin conformation capture in lymphoma, we found that the B-cell-lineage-determining transcription factor EBF1 preferentially positions multiple enhancers at loci containing sparsely distributed genes essential for B-cell identity and oncogenesis. Our time-resolved sub-diffraction optical chromatin architecture tracing of >100,000 alleles in individual lymphoma cells further revealed diverse topological conformations facilitating multi-enhancer interactions. Mechanistically, we found that positioning of enhancers at allelic topological centers is required for their interactions with target promoters, with EBF1 serving as a barrier to the loop-extruding cohesin on enhancers. These findings, which we demonstrate their generalizability to the T-cell-lineage-determining transcription factor TCF1 in T-cell leukemia, suggest that lineage-determining transcription factors radially position enhancers and promoters to enable multi-enhancer regulation of key oncogenes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.10.681717v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

