<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Wednesday, 27 August 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ACLPred: an explainable machine learning and tree-based ensemble model for anticancer ligand prediction.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Arvind Kumar Yadav, Jun-Mo Kim</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Several small molecules have been approved for cancer treatment, but the continuously growing cancer cases have further encouraged the identification of new anticancer drug compounds. Experimental methods are costly and time-consuming, thus rapid and cost-effective alternative method is much required. The effective identification of anticancer compounds using machine learning (ML) offers a promising solution, reducing both time and cost. In this study, small molecules with known inhibitory activities, both anticancer and non-anticancer were considered to train classification models. Molecular descriptors were calculated, and multistep feature selection was applied to identify significant features. Multiple ML algorithms were employed to build classification models and evaluated their performance using independent test and external datasets. The tree-based ensemble model, particularly Light Gradient Boosting Machine (LGBM), achieved the highest prediction accuracy of 90.33%, with an area under the receiver operating characteristic curve (AUROC) of 97.31%. Consequently, LGBM model was implemented in our proposed method, ACLPred. The ACLPred demonstrated superior prediction accuracy with good generalizability compared to existing methods. SHapley Additive exPlanations (SHAP) analysis provided model interpretability and revealed that topological features made major contributions to decision-making. ACLPred is available as an open-source, user-friendly graphical interface at https://github.com/ArvindYadav7/ACLPred for the screening of potential anticancer compounds.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40855112/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Senescent fibroblasts secrete CTHRC1 to promote cancer stemness in hepatocellular carcinoma.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hai Huang, Wang Peng, Qiaodan Zhou, Yuchong Zhao, Luyao Liu, Haochen Cui, Jingwen Liang, Mengdie Cao, Wei Chen, Ronghua Wang, Shiru Chen, Si Xiong, Bin Cheng, Shuya Bai</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cellular senescence plays a significant role in tumorigenesis and tumor progression. Substantial evidence indicates that senescence occurs in cancer-associated fibroblasts (CAFs), the predominant stromal component within the tumor microenvironment (TME), which profoundly impacts tumor biology. However, despite growing evidence of stromal cell involvement in cancer progression, the specific mechanisms and clinical implications of senescent CAFs (SCAFs) in hepatocellular carcinoma (HCC) have not been fully elucidated.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40855439/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Leveraging structure-informed machine learning for fast steric zipper propensity prediction across whole proteomes.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Samantha Zink, Songrong Qu, Thomas Holton, Eesha Shankar, Paulina Stanley, David S Eisenberg, Michael R Sawaya, Jose A Rodriguez</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Predicting the amyloid fold and the propensity of peptide segments to adopt amyloid-like structures remain a challenge. However, recent progress has facilitated structure-based prediction of steric zipper propensity and the use of machine learning to accelerate the calculation of predictive models across many scientific areas. Leveraging these advances, we have developed a new approach for rapid proteome-wide assessment of zipper profiles that is informed by four million steric zipper predictions collected over ten years. This collection is used to build a machine learning model capable of rapidly predicting steric zipper propensity, and allowing for the assessment of zippers at both the protein and proteome level. Our predictions show enrichment for zipper forming segments in proteins involved in cell wall reorganization in yeast, highlighting a potential category of interest for experimental characterization. Overall, our predictive model allows for the exploration of amyloid formation across the tree of life and provides a tool for assessment of both novel and designed sequences for zipper density.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40853992/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Preliminary screening of urinary host protein biomarkers for Schistosomiasis haematobium: A proteome profiling study identifying candidate diagnostic targets in school-aged children.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yiyun Liu, Saleh Juma, Qingkai Xue, Mingzhen He, Said Mohammed Ali, Khamis Seif Khamis, Mchanga Mohd Suleiman, Mayda Salim Hamad, Mgeni Abdalla Khamis, Hongxia Zhao, Xin Dong, Kun Yang, Yuzheng Huang</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Schistosomiasis is a major public health challenge and a globally neglected tropical disease. Schistosoma haematobium, the causative agent of urogenital schistosomiasis, is endemic in African countries; with school-aged children ages 7-15 years being the most vulnerable population. Current diagnostic methods rely on microscopy to identify parasite eggs in urine; which is labor-intensive, requires specialized skills, and often lacks sensitivity, especially in mild infections. To address these limitations, we explored host disease-related biomarkers as a promising avenue for advancing diagnosis and detection. We recruited 135 children ages 7-15 years from Zanzibar, a known transmission hotspot, and used data-independent acquisition (DIA) proteomics combined with machine learning to identify potential host protein biomarkers in urine samples from individuals infected with Schistosoma haematobium. Proteomic analysis identified 823 common host proteins in urine samples from the infected group. Machine learning algorithms highlighted candidate discriminative proteins; which were validated using enzyme-linked immunosorbent assays (ELISA). Machine learning emphasized SYNPO2, CD276, α2M, LCAT, and hnRNPM as the most discriminating biomarkers for Schistosoma haematobium infection. ELISA validation confirmed the differential expression trends of these proteins, while machine learning further validated LCAT and α2M, underscoring their diagnostic potential. Our study focused on host-derived proteins and identified key urinary protein biomarkers associated with Schistosoma haematobium infection, and offers new insights into host-parasite interactions and potential tools for non-invasive diagnostics. While validated in African pediatric populations from transmission hotspots, this host-protein approach inherently overcomes geographic limitations of parasite-based diagnostics; which is a critical advantage for surveillance in non-endemic regions where imported cases threaten gains toward elimination. These findings lay the groundwork for developing novel diagnostic approaches that could significantly improve the detection and surveillance of schistosomiasis, particularly in high-risk populations.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40853910/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The emerging sequence grammar of 3D genome organisation.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liezel Tamon, James Ashford, Matthew Nicholls, Marella F T R de Bruijn, Aleksandr B Sahakyan</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The multiplexed layers of regulatory processes and mechanisms within a cell are, to a degree, encoded in our genome. Unravelling the relationship between DNA sequence and molecular processes is crucial for understanding evolution, interpreting and predicting the consequences of genomic variation. Furthermore, understanding the extent to which DNA sequence contributes to the genome organisation can help reveal the aspects more influenced by other factors. Therefore, here we provide a succinct summary of the emerging genomic sequence code or "grammar" of genomic contact formation and 3D genome organisation. Drawing on different types of evidence from multiple disciplines, from large-scale genomic studies, biochemical in vitro assays, and computational analyses utilising machine learning and other modelling techniques, we aim to inform future research on the present associations between 3D genome organisation and sequence.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40853480/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Increasing the Scale of the Mass Spectrometry Query Language Compendium with Explainable AI.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Thomas V Harwood, Mingxun Wang, Trent R Northen, Benjamin P Bowen</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> A significant bottleneck in metabolomics data interpretation is the effective use of domain knowledge to assign structural information based on fragmentation patterns. The mass spectrometry query language (MassQL) aims to make this process accessible and applicable across multiple analysis platforms. While advanced computational methods are capable of predicting compound structures from fragmentation data, AI/ML approaches often rely on complex, opaque criteria that are difficult to interpret or modify. As a result, their predictive patterns cannot be readily translated into human-readable rules, such as those used in MassQL. In this study, we introduce ChemEcho, a machine learning embedding method that converts tandem mass spectrometry data into sparse feature vectors containing peak and neutral mass subformulae to enhance explainable AI/ML-based methods. An advantage of this approach is that decision trees trained using these feature vectors can be directly translated to MassQL. Using a battery of decision trees trained using ChemEcho embeddings to predict molecular attributes, we generated over 1500 MassQL queries for 765 molecular features and evaluated their precision and recall. From these queries, the 50 highest-performing queries were integrated into the MassQL compendium. This set of generated MassQL queries included environmentally and biologically relevant classes such as PFAS and molecules containing phosphate or sulfate substructures. To illustrate the impact these queries would have on a typical metabolomics experiment, these MassQL queries were applied to a public metabolomics data set─resulting in a marked increase in the structural information derived from tandem mass spectra. Access and reuse of these queries is expected to enhance structural annotation in untargeted experiments, leading to more specific claims and advancing many applications in metabolomics.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40851452/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Honeybee-Gilliamella synergy in carbohydrate metabolism enhances host thermogenesis in cold acclimation. </h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tang M, Li C, Ge X, Kuang Y, Qiu L, Chan KC, Chen Z, Guo L, Zhao Z, Zhang B, Liu S, Zhou X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #fedbf2; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, pan-genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> How gut symbionts contribute to host adaptation remains largely elusive. Studying co-diversified honeybees and gut bacteria across climates, we found cold-adapted species (Apis mellifera, A. cerana) exhibit enhanced genomic capacity for glucose, pyruvate, lipid and glucuronate production versus tropical species. Metagenomics revealed Gilliamella as the most enriched gut bacterium in cold-adapted bees. Germ-free honeybees inoculated with the Gilliamella from A. cerana showed increased activity, body temperature and fat storage upon cold exposure. Saccharide metabolomics demonstrated higher hindgut glucose levels in Gilliamella-colonized A. mellifera versus germ-free bees, and in A. cerana versus three sympatric tropical species. Although Gilliamella can hydrolyze β-glucan into glucose, cultural experiments suggest it preferentially degrades glucuronate to pyruvate. In turn, monocolonized bees upregulated hindgut glucose/pyruvate utilization while increasing glucuronate provision, suggesting nutritional complementarity. Gilliamella's transporter genes predominantly target ascorbate (a glucuronate derivative), which is elevated in inoculated hindguts. Accordingly, Gilliamella converts ascorbate to D-xylulose-5P (promoting lipogenesis), while showing reduced growth on glucuronate/ascorbate versus glucose, potentially minimizing glucose competition with hosts. We revealed a highly coordinated host-symbiont metabolic synergy enhancing host energy acquisition for cold adaptation.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40850953/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Epigenetic synthetic lethality as a cancer therapeutic strategy: synergy of experimental and computational approaches.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Maria Farina-Morillas, Laia Ollé-Monràs, Silvana Ce Maas, Isabel de Rojas-P, Miguel F Segura, Jose A Seoane</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cancer treatment is an ongoing challenge, as directly targeting oncogenic drivers is often unfeasible in many patients due to the lack of druggable targets. This has led to the exploration of alternative strategies, such as exploiting synthetic lethality (SL) relationships between genes. SL facilitates the indirect targeting of oncogenic drivers, as exemplified by the clinical success of PARP inhibitors against BRCA-mutated tumors. Advances in high-throughput perturbation screens and multi-omics technologies have deepened our understanding of SL relationships, while computational models enhance SL predictions to better reflect biological complexity. However, while numerous experimental and computational methods have been developed to identify SL interactions, difficulties remain in translating these findings into clinical applications.This review combines recent progress on SL relationships in cancer with emerging insights into epigenetic regulation, highlighting how epigenetic drugs (epidrugs) can provide new opportunities for targeted interventions, offering a way to minimize off-target effects and enhance therapeutic precision. To advance SL-based therapies, efforts must focus not only on identifying new SL interactions but also on consolidating existing knowledge and integrating experimental and computational approaches to characterize the vulnerabilities of cancer cells. Strengthening this foundation will be critical for the effective development of SL-based cancer treatments.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40851437/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Heat shock protein family H member 1 HSPH1 expression correlates with progression and prognosis of hepatocellular carcinoma. </h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Huang T, Xu S, Li M, Lu C, Zhang J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #fedbf2; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, pan-genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Heat shock proteins (HSPs) are evolutionarily conserved molecular chaperones that maintain cellular proteostasis under physiological and stress conditions. HSPH1 (Heat Shock Protein Family H Member 1, also known as HSP105 or HSP110) belongs to the HSP110 family and functions as a nucleotide exchange factor for HSP70, enhancing its folding activity. Beyond its canonical role, HSPH1 is increasingly recognized for its involvement in tumor progression. It has been reported to regulate cell proliferation, invasion, metastasis, and resistance to therapy in several cancers, including breast, lung, and liver cancer. Pan-cancer transcriptomic analyses have identified HSPH1 as frequently overexpressed and correlated with poor prognosis. In hepatocellular carcinoma (HCC), while other HSPs such as HSP70 and HSP90 are well-studied, the biological role of HSPH1 remains unclear. In this study, we systematically analyzed HSPH1 expression in HCC using TCGA and GEO datasets, and validated its clinical relevance in patient samples. HSPH1 was significantly upregulated in HCC tissues and associated with advanced tumor stage and worse overall survival. Functional enrichment and immune infiltration analyses suggested that HSPH1 participates in oncogenic pathways (e.g., p53, cell cycle) and modulates the tumor immune microenvironment. Knockdown of HSPH1 in HCC cell lines inhibited proliferation and colony formation. Together, our findings highlight HSPH1 as a potential prognostic biomarker and therapeutic target in HCC.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40851097/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">L‑type amino acid transporter 1 in enhancing boron neutron capture therapy: Mechanisms, challenges and future directions (Review). </h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zheng X, Pan J, Lin D, Shao W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> L‑type amino acid transporter 1 (LAT1) has emerged as a critical molecular target for advancing boron neutron capture therapy (BNCT), a promising treatment that leverages selective boron accumulation and neutron irradiation to eradicate cancer cells. The frequent upregulation of LAT1 in aggressive tumors (such as gliomas and specific subtypes of lung and breast cancer) underpins its essential role as the principal mediator of tumor‑selective boron compound uptake in BNCT. The present review comprehensively examines the structure and function of LAT1, the mechanistic principles of boron transport (including LAT1 mediation) and the key regulatory pathways governing BNCT efficacy. Building on this and given the role of LAT1 in reprogramming tumor metabolism through amino acid transport, advanced metabolomics tools, particularly liquid chromatography‑mass spectrometry and nuclear magnetic resonance, offer a novel approach for clarifying the contribution of LAT1 to BNCT. These techniques hold significant potential to map metabolic profiles altered by LAT1‑mediated boron compound uptake, thereby elucidating downstream biochemical consequences relevant to the therapeutic efficacy and resistance mechanisms. Synthesizing the dual role of LAT1 as both a vulnerability and therapeutic target in BNCT, the present review systematizes key challenges, including the need for selective boron compounds, resistance mechanisms and off‑target effects, while mapping actionable pathways to unlock its potential via refined regulation strategies, next‑generation delivery agents and personalized approaches. By addressing these knowledge gaps, this synthesis provides a foundational framework to harness LAT1‑targeted BNCT, offering potential to advance precision oncology paradigms and improve clinical outcomes for patients with LAT1‑enriched tumors.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40849804/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

