<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Friday, 27 June 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MassNet: billion-scale AI-friendly mass spectral corpus enables robust de novo peptide sequencing</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> A, J., Zhang, X., Zhang, X., Wei, J., Zhang, T., Deng, Y., Liu, P., Nie, Z., Chen, Y., Dong, N., Gao, Z., Sun, S., Guo, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present MassNet, the largest known corpus of data-dependent acquisition (DDA) mass spectrometry data, derived from ~30 TB of raw files and comprising 1.54 billion MS/MS spectra, resulting in 558 million peptide-spectrum matches (PSMs)</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Breakthroughs in artificial intelligence (AI) for natural language processing and computer vision have been largely driven by high-quality, large-scale datasets such as OpenWebText and ImageNet. Inspired by this, we present MassNet, a foundational resource for proteomics designed to accelerate deep learning applications. MassNet is the largest known corpus of data-dependent acquisition (DDA) mass spectrometry (MS) data, derived from ~30 TB of raw files and comprising 1.54 billion MS/MS spectra, resulting in 558 million peptide-spectrum matches (PSMs) across 35 species, including animals, plants, and microbes. Within the human subset, MassNet includes more than 1.7 million precursors and 19,966 proteins, covering 98% of annotated human proteins. To enable efficient AI training, we developed the Mass Spectrometry Data Tensor (MSDT), a structured format based on Parquet that enables standardized, high-performance batch access and seamless integration with GPU and TPU platforms for distributed training. We further extended MassNet to support de novo peptide sequencing, which infers peptide sequences directly from MS/MS spectra without reference databases, and is critical for discovering novel proteins, characterizing non-model organisms, and identifying post-translational modifications (PTMs). We introduce XuanjiNovo, a non-autoregressive Transformer model that leverages a curriculum learning strategy to enhance training stability. By dynamically adjusting learning difficulty based on model performance, XuanjiNovo achieves smooth convergence on complex, multi-distributional data without manual hyperparameter tuning. Trained on 100 million PSMs from the MassNet, it consistently outperforms state-of-the-art methods across diverse benchmarking tasks. Peptide recall exceeds 0.8 on the Bacteroides thetaiotaomicron and Zea mays datasets. On human data acquired using the Orbitrap Astral platform, XuanjiNovo achieves 38.8% to 144.3% improvement over existing models. MassNet represents the first large-scale, standardized foundational dataset in proteomics, marking a critical milestone in the integration of artificial intelligence into proteomics research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.20.660691v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ContrastQA: A Label-guided Graph Contrastive Learning-based approach for protein complex structure quality assessment</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, L., Ding, R., Chen, X., Hou, J., Si, D., Wang, Y., Lin, K., Cao, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A deep learning-based protein complex EMA method based on a label-guided graph contrastive learning framework .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: Recent advances in deep learning have significantly boosted the accuracy of protein complex structure prediction, with models like AlphaFold3 and RoseTTAFold All-Atom achieving remarkable performance. However, their predictions still fall short of true experimental resolution. A key challenge lies in accurately estimating the quality of predicted models - known as Estimation of Model Accuracy (EMA) - which is essential for selecting the most reliable structures from among numerous decoys. Efficient EMA methods can directly improve prediction accuracy and outcomes. Additionally, leveraging protein complex interface features can further enhance global accuracy estimation. In this work, we introduce ContrastQA, a deep learning-based EMA approach built on a label-guided graph contrastive learning framework. By using DockQ, a well-established interface scoring metric, as the supervisory signal and integrating geometric graph neural networks, our method provides an effective solution for assessing the structural quality of protein complexes. Results: We evaluated ContrastQA and other state-of-the-art (SOTA) protein complex EMA methods on the CASP16 and ABAG-AF3 datasets. Our method achieved the lowest ranking loss based on TMscore and GDT-TS on the CASP16 dataset, as well as the lowest TMscore ranking loss on hard targets. Our method also achieved the lowest ranking loss on the latest ABAG-AF3 dataset compared to SOTA method. Overall, ContrastQA demonstrated solid performance. Ablation studies confirmed that both the graph contrastive learning module and the contrastive loss function play a critical role in the model's effectiveness. These findings suggest that our graph contrastive learning framework serves as a valuable pre-training strategy for learning protein structure representations. Availability and implementation: The source code of ContrastQA is available at https://github.com/Cao-Labs/ContrastQA.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.20.660832v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MITScan: Score-Based Genome-Wide Association Analysis of the Microbiome and Host Transcriptome</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hao, Y., Kafri, T., Zou, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present MITScan, a genome-wide MIcrobiome-host Transcriptome Score-based association analysis, employing zero-inflated negative binomial models to accommodate extra zeros commonly observed in microbiome data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Microbiome research explores how microbial communities interact with the human body to influence health and well-being, offering insights for advancing personalized medicine and targeted treatments. Microbiome profiles can be measured, for example, by 16S rRNA sequencing, where sequencing reads are clustered into genus- or species-level operational taxonomic units (OTUs), yielding count-based abundance data. In addition, these data are often characterized by sparsity with extra zeros. Emerging evidence suggests that imbalances in microbial composition, potentially regulated by host gene expression, have been associated with various diseases. Investigating the associations between the microbiome and host transcriptome can help uncover the mechanisms underlying human health and disease. Current existing approaches for evaluating their associations largely rely on Pearson or Spearman correlations, ignoring the fact that microbiome data are sparse count data, leading to potentially biased results. To overcome this issue, we present MITScan, a genome-wide MIcrobiome-host Transcriptome Score-based association analysis, employing zero-inflated negative binomial models to accommodate extra zeros commonly observed in microbiome data. To address the large number of paired association tests arising from two high-dimensional omics datasets, we utilize score tests and matrix calculations for computational efficiency. We further apply an empirical permutation method with genomic control to effectively control the family-wise error rate in studies with small sample sizes. With real datasets, we demonstrate that MITScan achieves computational gains of three orders of magnitude compared to commonly used Wald or t tests, while identifying similar significant OTU-gene pairs. The MITScan R package is accessible on GitHub at https://github.com/yajing-hao/MITScan.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.24.661342v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">miRNA/mRNA analysis of increased TGF-Œ≤ pathways drive epithelial-mesenchymal transition and regulatory T cell differentiation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Darville, T., Sun, X., Zhang, Y., O'Connell, C. M., Mokashi, N., Tang, W., Bhardwaj, A., Duncan, B., Andrews, C. W., Weisenfeld, H., Zheng, X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of endometrial infection and miRNA-mRNA interactions in endometrium-only and endo+ tissue shows that chlamydia-induced dysregulation of microRNAs can promote tissue scarring while dampening inflammatory responses that could otherwise limit infection.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Chlamydia trachomatis genital tract infection is linked to severe reproductive complications in women, including ectopic pregnancy, infertility, and adverse pregnancy outcomes. Mouse models of infection suggest that chlamydia-induced dysregulation of microRNAs (miRNAs) can drive harmful cytokine responses, pathogenic epithelial-mesenchymal transition (EMT), and fibrosis. To investigate these mechanisms in humans, we profiled miRNA and mRNA expression in endometrial biopsies from women with endometrial infection (Endo+) and compared them to profiles from women with cervix-only infection (Endo-) or no infection. Ingenuity Pathway Analysis (IPA) revealed that Endo+ tissues had upregulated genes associated with innate and adaptive immune response pathways, as well as EMT regulation, while downregulated genes were linked to cell cycle control. An integrative miRNA-mRNA analysis, which combined a review of published miRNA regulation in human infections and immune responses with IPA's miRNA target filter, identified differentially expressed miRNAs that modulate these pathways in the endometrium of Endo+ women. Functional annotation of these miRNAs showed a predominance of downregulated miRNAs that typically suppress EMT and regulatory T cell (Treg) differentiation, along with miRNAs that usually enhance Th17 responses. Comparisons with previously identified mRNA pathways in blood samples from women with endometrial Chlamydia infection indicated that alterations in TGF-{beta} signaling and EMT were specific to the endometrium. Overall, the miRNA-mRNA interactions inferred from Endo+ tissue suggest increased activity in TGF-{beta} ; pathways that promote enhanced EMT and Treg differentiation, while reducing Th17 activation. These changes highlight a dual potential for promoting tissue scarring while dampening inflammatory responses that could otherwise limit infection.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.23.661210v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Expanding the utility of transcriptome analysis for mutation detection in high-risk childhood precision oncology</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mayoh, C., Barahona, P., Lin, A., Cui, L., Ajuyah, P., Altekoester, A., Lau, L. M., Khuong-Quang, D.-A., Sullivan, P., Senapati, A., Nagabushan, S., Sullivan, A., Omer, N., Moore, A. S., Nicholls, W., Endersby, R., Gottardo, N. G., McCowage, G. B., Dalla Pozza, L., Hansford, J. R., Khaw, S.-L., Wood, P. J., Trahair, T. N., Marshall, G. M., Ziegler, D. S., Tyrrell, V., Haber, M., Wong, M., Ekert, P. G., Cowley, M. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that whole transcriptome sequencing can identify all mutation classes found previously using whole genome sequencing (WGS) and provides additional functional insights into their pathogenicity.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In precision oncology, whole transcriptome sequencing (RNA-seq) excels at identifying oncogenic fusions. Here, using a cohort of 477 high-risk paediatric tumours, we demonstrate that RNA-seq can identify all mutation classes found previously using whole genome sequencing (WGS) and provides additional functional insights into their pathogenicity. By incorporating reference-guided fusion, and reference-free structural variant (SV) detection algorithms with RNA abundance assessment, RNA-seq identified 96% of SVs and resolved 33 complex SVs that WGS failed to identify. Furthermore, RNA-seq identified 92% of all single nucleotide variants and small insertions and deletions. Importantly RNA-seq informed the pathogenicity assessment in 22% of variants through identification of allele specific expression or the functional consequence of splice-altering variants. The utility of RNA-seq extends beyond fusion identification to the interpretation of mutation pathogenicity and the discovery of important mutations that would otherwise go undetected. We propose that RNA-seq is an indispensable companion to WGS in precision medicine.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.25.661445v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comprehensive Survey of Gut Microbiome Associations with Health Conditions in the Human Phenotype Project</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Weissglas-Volkov, D., Reisner, Y., Shor, T., Diament, A., Jankelow, A., Godneva, A., Dhir, R., Segal, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We explore associations between the gut microbiome and 37 health indications, including obesity, diabetes, hyperlipidemia, metabolic syndrome, inflammation, liver disease, cardiovascular and kidney conditions, immune and allergic diseases, mental health, and sleep-related traits.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The human gut microbiome is increasingly implicated with diverse health conditions, highlighting the importance of developing comprehensive resources to systematically map these associations. In this study, we leveraged the Human Phenotype Project (HPP) 10K cohort, a large, deeply phenotyped population study cohort with extensive metagenomic profiling, to explore associations between the gut microbiome and 37 health indications. These include conditions with established or emerging links to the microbiome, such as obesity, diabetes, hyperlipidemia, metabolic syndrome, inflammation, liver disease, cardiovascular and kidney conditions, immune and allergic diseases, gastrointestinal symptoms, mental health, and sleep-related traits. Using a curated set of approximately 200 refined phenotypic features and analyzing 1,184 microbial species, we performed robust statistical association analyses. We observed significant enrichment of associations in 14 health indications, with most associations reflecting increased microbial abundance in favorable health states, thereby suggesting potential microbial targets for intervention. This work introduces a publicly available, high-resolution resource to facilitate future research and support the development of microbiome-informed health strategies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.26.661672v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Mapping and reprogramming microenvironment-induced cell states in human disease using generative AI</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Akbarnejad, A., Steele, L., Jafree, D. J., Birk, S., Sallese, M. R., Rademaker, K., Boxall, A., Rumney, B., Tudor, C., Patel, M., Prete, M., Makarchuk, S., Li, T., Stanley, H., Foster, A. R., Roberts, K., Trinh, A. L., Villa, C. E., Testa, G., Mahil, S., Mehrjou, A., Smith, C., Vakili, S., Clatworthy, M. R., Mitchell, T., Bayraktar, O. A., Haniffa, M., Lotfollahi, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present MintFlow, a generative AI algorithm that learns how the tissue microenvironment influences cell states and predicts how tissue perturbations can change and reprogram them.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Tissue microenvironments reprogram local cellular states in disease, yet current computational spatial methods remain descriptive and do not simulate tissue perturbation. We present MintFlow, a generative AI algorithm that learns how the tissue microenvironment influences cell states and predicts how tissue perturbations can change and reprogram them. Applied to three human diseases, MintFlow uncovered distinct pathogenic spatial reprogramming in inflammatory and tumor microenvironments. In atopic dermatitis, MintFlow prioritized an epidermal resident T cell program and a dendritic cell activation niche within inflammatory hotspots within the superficial dermis. In melanoma, MintFlow identified fibrotic stroma resembling keloid scar tissue. In kidney cancer, MintFlow resolved immunosuppressive CD8+ T cell states within tertiary lymphoid structures. Furthermore, MintFlow enabled in silico perturbations of disease-relevant cell states and tissue environments. Regulatory T cell modulation in atopic dermatitis suppressed the pro-inflammatory tissue environment, supporting manipulation of these cells as a therapeutic target. Spatially-targeted macrophage depletion in kidney cancer reverted immunosuppressed T cell states, and the corresponding gene programs correlated with survival in large kidney cancer patient cohorts. These findings position MintFlow as a tool for unbiased prediction of disease mechanisms and in silico perturbations to accelerate translational hypothesis generation and guide therapeutic strategies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.24.661094v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The reference-free pangenome of Arabidopsis thaliana</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Obinu, L., Guarracino, A., Trivedi, U., Porceddu, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A pangenome-based study of intra-specific variation in the model plant Arabidopsis thaliana .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> For several decades, scientists relied on unique reference genomes as the basis for various genomic studies. This methodology presents multiple drawbacks, particularly for variation studies, often yielding unrepresentative and incomplete observations regarding the diversity of the species or group under study. Pangenomes are collections of genomic sequences from several individuals of a species or population, which allow to overcome the limitations of studies based on a single reference genome. In this study, we produced a 93-assembly pangenome of the model plant Arabidopsis thaliana using the reference-free method PanGenome Graph Builder (PGGB). The aim was to investigate the diversity within this species using this novel methodology, encompassing genomic sequences, genes, and pseudogenes. The pangenome exhibited a total length of 488.78 Mb, consisting of 30,391,243 nodes and 7,189,634 edges. We mapped a total of 2,325,577 genes and 23,894 pseudogenes across 93 assemblies, of which 36% and 0.9%, respectively, were classified as core. The observed variation in terms of genes and especially pseudogenes related to the geographical distance between sampling sites. Since the simulated pangenome growth curve based on gene did not reach a plateau, new accessions could potentially expand the represented diversity. This study enriches our understanding of intra-specific variation in A. thaliana and provides a new viewpoint on the potential applications of pangenomes in diversity research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.25.661508v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

