<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Tuesday, 20 May 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Robust inference and widespread genetic correlates from a large-scale genetic association study of human personality</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Schwaba, T., Clapp Sullivan, M. L., Akingbuwa, W. A., Ilves, K., Tanksley, P. T., Williams, C. M., Dragostinov, Y., Liao, W., Ackerman, L. S., Fealy, J. C. M., Hemani, G., de la Fuente, J., Gupta, P., Stein, M. B., Gelernter, J., Levey, D. F., Vosa, U., Ausmees, L., Realo, A., Estonian Biobank Research Team,, Vaht, M., Allik, J., Esko, T., Mottus, R., Vainik, U., Jonsdottir, G. A., Thorleifsson, G., Gunnarsson, A. F., Bjornsdottir, G., Thorgeirsson, T. E., Steffansson, H., Stefansson, K., Cheesman, R., Qin, Q., Corfield, E. C., Ask, H., Ask Torvik, F., Ystrom, E., Tesli, M., Boomsma, D. I.,</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify 1,257 lead genetic variants associated with personality, including 823 novel variants, and find minimal spousal assortment.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Personality traits describe stable differences in how individuals think, feel, and behave and how they interact with and experience their social and physical environments. We assemble data from 46 cohorts including 611K-1.14M participants with European-like and African-like genomes for genome-wide association studies (GWAS) of the Big Five personality traits (extraversion, agreeableness, conscientiousness, neuroticism, and openness to experience), and data from 51K participants for within-family GWAS. We identify 1,257 lead genetic variants associated with personality, including 823 novel variants. Common genetic variants explain 4.8%-9.3% of the variance in each trait, and 10.5%-16.2% accounting for measurement unreliability. Genetic effects on personality are highly consistent across geography, reporter (self vs. close other), age group, and measurement instrument, and we find minimal spousal assortment for personality in recent history. In stark contrast to many other social and behavioral traits, within-family GWAS and polygenic index analyses indicate little to no shared environmental confounding in genetic associations with personality. Polygenic prediction, genetic correlation, and Mendelian randomization analyses indicate that personality genetics have widespread, potentially causal associations with a wide range of consequential behaviors and life outcomes. The genetic architecture of personality is robust and fundamental to being a human.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.16.648988v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Application of Bioinformatics and Machine Learning Tools in Food Safety.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mahdi Soroushianfar, Goli Asgari, Fatemeh Afzali, Atiyeh Falahat, Mohammad Soroush, Mansoor Baghahi, Mohammad Javad Haratizadeh, Ghazaleh Khalili-Tanha, Elham Nazari</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Food safety is a fundamental challenge in public health and sustainable development, facing threats from microbial, chemical, and physical contamination.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Food safety is a fundamental challenge in public health and sustainable development, facing threats from microbial, chemical, and physical contamination. Innovative technologies improve our capacity to detect contamination early and prevent disease outbreaks, while also optimizing food production and distribution processes.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40388006/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Optimized GMP-grade production of non-viral Sleeping Beauty-generated CARCIK cells for enhanced fitness and clinical scalability.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ilaria Pisani, Giusi Melita, Patricia Borges de Souza, Stefania Galimberti, Angela Maria Savino, Jolanda Sarno, Beatrice Landoni, Stefano Crippa, Elisa Gotti, Carolina Cuofano, Olga Pedrini, Chiara Capelli, Giada Matera, Daniela Belotti, Stefania Cesana, Benedetta Cabiati, Michele Quaroni, Valentina Colombo, Massimiliano Mazza, Barbara Vergani, Anna Gaimari, Fabio Nicolini, Marcella Tazzari, Martine Bocchini, Marta Serafini, Alessandro Rambaldi, Benedetta Rambaldi, Giuseppe Dastoli, Andrea Biondi, Giuseppe Gaipa, Martino Introna, Josée Golay, Sarah Tettamanti</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Strict adherence to GMP guidelines and regulatory compliance is crucial when transitioning from research to clinical-grade production of ATMPs like CAR T cells.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Strict adherence to GMP guidelines and regulatory compliance is crucial when transitioning from research to clinical-grade production of ATMPs like CAR T cells. The success of CAR T cell therapy in treating hematological malignancies highlights the need for closed or automated systems to ensure quality and efficacy. Recent evidence also suggests that ex vivo culture conditions can significantly impact CAR T cell functionality.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40390044/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Assessing Large Language Model Alignment Towards Radio-logical Myths and Misconceptions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> West, C., Wang, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Large language models are susceptible to the same underlying sentiments and biases with regards to radiological topics as humans.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Humans are susceptible to believing in or espousing myths and misconceptions in the radiological field. Since the advent of large language models (LLMs)-with applications such as ChatGPT processing over one billion queries per day, and Gemini now integrated into Google's search engine-these models have steadily evolved into a major source of information for vast audiences. The aim of our study was to assess whether large language models, which are primarily trained on human-generated data, are susceptible to the same underlying sentiments and biases with regards to radiological topics. Furthermore, we assess the use of large language models in both detecting and analyzing trends in the circulation of radiological myths and misconceptions in online communities. Finally, we evaluate the use of large language models as supportive tools for improving communication on these controversial topics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.16.652427v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">De novo whole genome assembly of the globally invasive green shore crab Carcinus maenas (Linnaeus, 1758) via long-read Oxford Nanopore MinION sequencing</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Brons, J. K., Hackl, T., Iacovelli, R., Haslinger, K., Lequime, S. J. J., van der Meij, S. E. T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A genome assembly of the invasive green shore crab, Carcinus maenas, via long-read Oxford Nanopore Technology sequencing .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Invasive species are rapidly reshaping aquatic ecosystems worldwide at an accelerating pace, with profound ecological and economic impacts. Many crustacean species have demonstrated invasive potential or are already well-established invaders. The green shore crab, Carcinus maenas, native to Europe and North Africa, is one of the most successful global marine invaders and is now present on six continents. Although the role of genomics in invasion science is increasingly recognized, genomic resources for brachyuran crabs remain limited, including the notable absence of a reference genome for C. maenas. Here we report on a de novo whole genome assembly of C. maenas via long-read Oxford Nanopore Technology sequencing. The assembly spans 1.09 Gbp across 21,887 scaffolds (N50 = 15 Mbp) with a BUSCO completeness of 98.4%, providing a high-quality resource for future genomic analyses. Additionally, we provide a detailed protocol for obtaining high-quality DNA to successfully sequence brachyuran crabs using a long-read approach, including strategies to address nanopore blockage issues. This new resource expands available genomic data for the species-rich infraorder Brachyura, and provides a valuable foundation for understanding the genetic factors underlying the global invasion success of C. maenas, supporting future research in marine invasion genomics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.19.654951v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multiscale Segmentation using Hierarchical Phase-contrast Tomography and Deep Learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhou, Y., Aslani, S., Javanmardi, Y., Brunet, J., Standsby, D., Carroll, S., Bellier, A., Ackermann, M., Tafforeau, P., Lee, P. D., Walsh, C. L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a deep learning-based segmentation pipeline for multi-scale biomedical image segmentation using Hierarchical Phase-Contrast Tomography and applied it to low-resolution whole-organ scans.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Biomedical systems span multiple spatial scales, encompassing tiny functional units to entire organs. Interpreting these systems through image segmentation requires the effective propagation and integration of information across different scales. However, most existing segmentation methods are optimised for single-scale imaging modalities, limiting their ability to capture and analyse small functional units throughout complete human organs. To facilitate multi-scale biomedical image segmentation, we utilised Hierarchical Phase-Contrast Tomography (HiP-CT), an advanced imaging modality that can generate 3D multi-scale datasets from high-resolution regions of interest (ROIs) at ~2 m/voxel to whole-organ scans at ~20 m/voxel. Building on these hierarchical multi-scale datasets, we developed a deep learning-based segmentation pipeline that is initially trained on manually annotated high-resolution HiP-CT data and then extended to lower-resolution whole-organ scans using pseudo-labels generated from high-resolution predictions and multi-scale image registration. As a case study, we focused on glomeruli in human kidneys, evaluating four deep learning models for 3D biomedical segmentation on high-resolution data. Among them, nnUNet demonstrated the best performance and was subsequently used as the baseline model for multi-scale segmentation in the pipeline. Applying this model to full-organ low-resolution data enabled comprehensive morphological analyses, including cortical spatial statistics and glomerular distributions, which aligned well with previous anatomical studies. Our results highlight the effectiveness of the proposed pipeline for segmenting small functional units in multi-scale bioimaging datasets and suggest its broader applicability to other organ systems.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.15.654263v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Triad-LMF: A Hierarchical Low-Rank Multimodal Fusion Framework for Robust Cancer Subtype Classification Using Multi-Omics Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tan, X., Chen, X., Ren, Z., Tian, R., Jiang, M., Yang, D., Guo, X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Triad-LMF, a novel multi-omics integration framework, is introduced, leveraging a low-rank multimodal fusion mechanism to improve the Accuracy of cancer subtype classification.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cancer heterogeneity is recognized as a major barrier to precise molecular subtype classification. Conventional approaches inadequately leverage the complementary nature of multimodal data, resulting in overfitting when handling high-dimensional omics data and constraining the comprehensive characterization of cancer subtype heterogeneity. To address this, Triad-LMF, a novel multi-omics integration framework, is introduced, leveraging a low-rank multimodal fusion mechanism to improve the Accuracy of cancer subtype classification. Triad-LMF consolidates multi-omics datasets, mitigating feature dimension disparities through optimized preprocessing. To enable efficient multimodal information integration, Triad-LMF implements a two-stage hierarchical fusion strategy, where in Local Pairwise Fusion and Global Triadic Fusion are combined via the Two-Feature and Three-way LMF modules, facilitating a progressive transition from local modality interactions to global feature integration. Experimental results demonstrate that Triad-LMF outperforms traditional machine learning methods and previous published methods in classification performance. UMAP verifies that Global Triadic Fusion significantly enhances the ability to discriminate against subtypes in implicit space representation. We further illustrate the effectiveness of Triad-LMF in making full use of multimodal interactive information. Additionally, SHAP feature importance analysis was adopted for extracting the important features. Moreover, across independent datasets, Triad-LMF demonstrates superior generalization capabilities. Triad-LMF offers an efficient and robust framework for multi-omics-driven cancer subtype classification.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.15.653805v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">BEstimate: a computational tool for the design and interpretation of CRISPR base editing experiments</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dincer, C., Fussing, B., Garnett, M., Coelho, M. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a flexible computational pipeline that systematically specifies base editor gRNA target sites, generates gRNA off-target predictions, and rich functional, structural and clinical annotations of targeted edits.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> CRISPR base editors enable scalable targeted DNA mutagenesis and are a powerful tool for the interpretation of gene function, the study of variants of unknown significance, and disease modelling. Existing guide RNA (gRNA) design tools focus on engineering a small number of targeted variants in a given gene or lack comprehensive functional annotation of target sequences. Here we developed BEstimate, a flexible computational pipeline that systematically specifies base editor gRNA target sites, generates gRNA off-target predictions, and rich functional, structural and clinical annotations of targeted edits. BEstimate accommodates custom gRNA design against variant alleles, improving editing accuracy and facilitating the programmed reversion of disease variants. BEstimate is a freely available, versatile tool for designing gRNA libraries and analysing base editor screens.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.19.654892v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Fast, flexible gene cluster family delineation with IGUA</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Larralde, M., Blom, J., Gourle, H., Carroll, L. M., Zeller, G. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present IGUA (Iterative Gene clUster Analysis; https://github.com/zellerlab/IGUA), a scalable, flexible GCF delineation method for genomic segments with multi-gene architectures.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Prokaryotic genomes harbor a variety of functional elements encoded as contiguous multi-gene clusters, with biosynthetic gene clusters (BGCs, genetic determinants of secondary metabolite biosynthesis) serving as a notable example. In a typical workflow, BGCs are clustered into Gene Cluster Families (GCFs), units that group BGCs encoding similar biosynthetic pathways together. However, existing methods cannot readily scale to massive datasets and cannot be used for GCF delineation tasks beyond BGC clustering. Here, we present IGUA (Iterative Gene clUster Analysis; https://github.com/zellerlab/IGUA), a scalable, flexible GCF delineation method for genomic segments with multi-gene architectures. On a BGC clustering task, IGUA is [≥]10x faster than the state-of-the-art (BiG-SCAPE/BiG-SLiCE), without sacrificing accuracy. To highlight its scalability, we use IGUA to cluster >2.8 million BGCs from {approx}1 million prokaryotic genomes in <18 hours (n = 2,829,071 BGCs to 56,960 GCFs). To showcase its utility beyond BGC clustering, we use IGUA to cluster (i) secretion systems and (ii) prophages into GCFs (n = 10,576 and 356,776 gene clusters to 2,744 and 213,699 GCFs, respectively). Overall, IGUA represents a versatile GCF delineation tool with unmatched computational efficiency and flexibility, enabling (meta)genomic mining applications at unprecedented scales.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.15.654203v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Efficient Estimation of Nucleotide Diversity and Divergence using Depth Information</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mirchandani, C. D., Enbody, E., Sackton, T. B., Corbett-Detig, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Callable Loci And More (clam), a tool that leverages callable loci -- determined from depth information -- to estimate population genetic statistics using a variant-only VCF.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The increasing scale of population genomic datasets presents computational challenges in estimating summary statistics such as nucleotide diversity ({pi}) and divergence (dxy). Unbiased estimates of diversity require knowledge of missing data and existing tools require all-sites VCFs. However, generating these files is computationally expensive for large datasets. Here, we introduce Callable Loci And More (clam), a tool that leverages callable loci -- determined from depth information -- to estimate population genetic statistics using a variant-only VCF. This approach offers improvements in storage footprint and computational performance compared to contemporary methods. We benchmark clam using a large muskox dataset and demonstrate that it produces unbiased estimates of {pi} while reducing runtime and storage requirements, compared to an existing approach. clam provides an efficient and scalable alternative for population genomic analyses, facilitating the study of increasingly large and diverse datasets. clam is available as a standalone program and integrated into snpArcher for efficient reproducible population genomic analysis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.15.654353v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Chromatin interaction-based annotation of distal cis-regulatory elements reveals highly dynamic promoter-enhancer interactions in lymphocyte development</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tingvall-Gustafsson, J., Jensen, C. T., Ungerback, J., Sigvardsson, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that lineage-specific transcription factors target regulatory elements that are annotated to both lineage- specific and broadly expressed genes, revealing a highly complex interplay between promoters and DREs.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Stage- and lineage-specific gene expression patterns are controlled by a complex interplay between transcription factors, the epigenetic landscape, and the 3-dimensional (3D) structure of the DNA. The 3D structure allows for the formation of DNA loops that juxtaposition distal regulatory elements to the promoters, allowing for tight control of gene expression. These loops can span hundreds of thousands base pairs, making it challenging to link regulatory elements to the correct target genes using conventional proximity-based annotation methods. Using a novel tool to facilitate the exploration of complex gene regulatory networks based on chromosome configuration data in early lymphocytes, we show that lineage-specific transcription factors target regulatory elements that are annotated to both lineage-specific and broadly expressed genes. Targeted inactivation of a set of these genes revealed their importance for B-cell development. Several regulatory elements annotated to lineage-specific genes were also annotated to alternative promoters in a context dependent manner, revealing a highly complex interplay between promoters and DREs in early lymphocyte development. Investigating the chromatin configuration in EBF1 deficient pro-B cells identified this transcription factor as a key mediator in the establishment of the 3-D promoter enhancer landscape in early B-lymphocyte differentiation. These data highlight how efficient annotation procedures for linking distal regulatory elements to target genes provide valuable insights into gene regulatory networks.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.15.653418v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Coexisting with humans: genomic and behavioural consequences in a small and isolated bear population</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fabbri, G., Biello, R., Gabrielli, M., Torres Villaca, S., Sammarco, B., Fuselli, S., Santos, P., Ancona, L., Peretto, L., Padovani, G., Sollitto, M., Iannucci, A., Paule, L., Balestra, D., Gerdol, M., Ciofi, C., Ciucci, P., Mahan, C. G., Trucchi, E., Benazzo, A., Bertorelle, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that Apennine brown bears possess a unique genomic diversity pattern including selective signatures at genes associated with reduced aggressiveness, possibly involving alternative splicing mechanism, and hypothesise that human-induced selection has influenced these changes, reducing conflicts and contributing to the long-term persistence of the Apenn</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Climate and land use change have increased human-wildlife interactions, potentially reducing wild species density and prompting behavioural adaptations to urbanised environments. It is still debated if behavioural responses are mainly the result of phenotypic plasticity or if they were driven by anthropic selective pressures, especially in small populations. Our study focused on the Apennine brown bear population (Ursus arctos marsicanus), which has coexisted with humans in Central Italy for millennia. We characterised genomic diversity and identified adaptation signals distinctive to this population by comparing whole genome resequencing data across the Holarctic species range. We show that Apennine brown bears possess a unique genomic diversity pattern including selective signatures at genes associated with reduced aggressiveness, possibly involving alternative splicing mechanism. Our findings suggest that even in small and long-isolated populations, selection may shape behavioural traits. We hypothesise that human-induced selection has influenced these changes, reducing conflicts and contributing to the long-term persistence of the Apennine bear and its coexistence with humans.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.15.654188v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Speed breeding transgenic American chestnut trees toward restoration</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Klak, T., Pilkey, H., May, V. G., Matthews, D., Oakes, A. D., Tan, E. H., Newhouse, A. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> The American chestnut (Castanea dentata) was a dominant, foundational forest canopy tree in eastern North America until an imported chestnut blight (caused by Cryphonectria parasitica) rendered it functionally extinct across its native range.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The American chestnut (Castanea dentata) was a dominant, foundational forest canopy tree in eastern North America until an imported chestnut blight (caused by Cryphonectria parasitica) rendered it functionally extinct across its native range. Biotechnological approaches have the potential to help restore the species, but field-based breeding advances are hampered by long generation times, [≤]50% transgene inheritance, and regulatory restrictions on outdoor breeding of transgenic trees. Self-incompatibility and flowering phenology further limit generational advances and field testing of chestnuts. Our work here demonstrates that long generational times and field constraints can be circumvented by producing both male and receptive female flowers in controlled indoor environments. Additionally, we developed an embryo rescue protocol for both indoor and field conditions, in which developing embryos can be extracted and micropropagated from immature seeds between 6- and 8-weeks post pollination. These advances have enabled production of the first homozygous transgenic American chestnuts, which have produced pollen that was used for outdoor controlled pollinations and yielded nearly 100% transgene inheritance by offspring. This work also provides event-specific DNA markers to differentiate transgenic chestnut lines and identify homozygous individuals. We demonstrate that an obligate outcrossing forest tree can reach sexual maturity rapidly in controlled, indoor environments. When coupled with genomic analyses and other biotechnological advances, this procedure could facilitate the reintroduction of this iconic species.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.19.654928v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">STT3A is required for recognition of pathogen-derived sphingolipids in Arabidopsis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Choi, S., Shimizu, M., Abe, A., Ishihama, N., Ishikawa, Y., Takemoto, D., Shirasu, K., Takano, Y., Terauchi, R., Kato, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified three Arabidopsis mutants with altered responses to Phytophthora infestans ceramide D, each carrying mutations in STAUROSPORIN and TEMPERATURE SENSITIVE 3-LIKE A (STT3A), which encodes an oligosacch</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Plants recognize pathogen-associated molecular patterns (PAMPs) via pattern recognition receptors, leading to the activation of pattern-triggered immunity in response to pathogen attack. Phytophthora infestans ceramide D (Pi-Cer D) is a sphingolipid from the oomycete pathogen P. infestans. Pi-Cer D is cleaved by the plant extracellular ceramidase NEUTRAL CERAMIDASE 2 (NCER2), and the resulting 9-methyl-branched sphingoid base is recognized by the plant receptor RESISTANT TO DFPM-INHIBITION OF ABSCISIC ACID SIGNALING 2 (RDA2) at the plasma membrane to transduce a defense signal. However, additional components are likely involved in sphingolipid recognition, which remain to be identified. Here, we employed a screen based on Lumi-Map technology to look for Arabidopsis (Arabidopsis thaliana) mutants with altered defense responses to Pi-Cer D. We identified three mutants showing diminished responses to Pi-Cer D and elf18, each carrying mutations in STAUROSPORIN AND TEMPERATURE SENSITIVE 3-LIKE A (STT3A), which encodes an oligosaccharyltransferase. The stt3a mutants exhibited higher susceptibility to the pathogen Colletotrichum higginsianum than the wild type and displayed alterations in NCER2 protein modifications. These findings suggest that STT3A contributes to plant immunity via post-translational modification of proteins including NCER2.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.18.654772v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The Cell Wall Controls Stem Cell Fate in the Arabidopsis Shoot Apical Meristem</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zerin, T., Ruiz-Duarte, P., Schuerholz, A.-K., Prevedel, R., Bevilacqua, C., Ma, Y., Schlamp, T., Wenzl, C., El Arbi, N., Miotk, A., Greb, T., Lohmann, J. U., Wolf, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the cell wall and its mechanical properties are under surveillance, linking cell wall state to intracellular gene-regulatory networks, linking stem cell identity and SAM organization.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> At the basis of plant developmental plasticity are continuously active pluripotent stem cells, which fuel the life-long post-embryonic formation of new organs. Since plant cells are encased in cell walls and thus immotile, their individual fate-specification program is dependent on their relative position within the organism. It is assumed that the cell wall and its mechanical properties are under surveillance, linking cell wall state to intracellular gene-regulatory networks. However, the role of these cell wall signalling pathways in plant development and the contribution of cell wall properties to cell behaviour and identity are unclear. Here, we show that control of cell wall properties is essential for the perpetuation of stem cell populations and pattering in the shoot apical meristems. The expression of pectin methylesterases (PMEs), which modify homogalacturonan methylesterification and thereby modulate cell wall mechanics, is maintained at low levels in stem cells by the stem cell specifying transcription factor WUS. Low PME expression is required for stemness, auxin patterning and stem cell-specific mechanical properties. Conversely, WUS depletion reduces wall stiffness in the meristem centre, reinforcing its role in maintaining mechanical homeostasis. Together, our findings show that WUS-mediated control of cell wall-modifying enzymes is essential for sustaining stem cell identity and SAM organization. These results demonstrat that the plant cell wall is not only involved in cell differentiation, but also exerts feedback control on developmental transitions, contributing to our understanding how the immediate physical environment is able to guide cell fate decisions in plants.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.19.654883v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Physiological characterization in a somaclonal mutant for fruit shape derived from persimmon 'Hiratanenashi'</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Horiuchi, A., Matsuzaki, R., Onoue, N., Ushijima, K., Kubo, Y., Akagi, T., Minamikawa, M. F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comparative physiological analysis of fruit shape in persimmon cultivars in East Asia and the discovery of four candidate genes that could explain the differences in fruit shape between cultivars.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Fruit shape is an essential agronomic trait that contributes to the commercial value of fruit crops; however, the physiological mechanisms determining fruit shape have been studied only in a few model crops. Persimmon (Diospyros kaki Thunb.), a major fruit crop in East Asia, shows diversity in fruit shape among cultivars. The underlying molecular mechanisms that determine fruit shape remain unclear. A comparative physiological analysis was conducted to investigate the differences between a major Japanese cultivar, 'Hiratanenashi' with a ''flat'' fruit shape, and its somaclonal mutant cultivar, 'Koushimaru' with a ''round'' fruit shape. Through principal component analysis (PCA) with the elliptic Fourier descriptor, the fruit shape transitions in these two cultivars were measured over the fruit differentiation period (40-53 days after anthesis). Histological observation of the mesocarp cells in the fruit shape differentiation stage indicated that not only cell proliferation patterns but also multiple vectors involving cell shapes or sizes are possible determinants of differences in fruit shape. Transcriptome analysis at the fruit shape differentiation stage detected differentially expressed genes between the two cultivars. The hierarchical clustering with expression patterns throughout the fruit development/maturation stages defined four main clusters, including candidate genes determining fruit shapes. We focused on one of these genes, DKAch04a32073.t1, annotated as WUSCHEL related homeobox 13 (WOX13), which has been reported to be a key factor in callus formation (or stem cell proliferation) and the reconnection of organs in Arabidopsis thaliana. Future functional analyses of the candidate genes would provide a better understanding of the mechanisms underlying persimmon fruit shape.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.19.654774v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multiobjective learning and design of bacteriophage specificity</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Novy, N., Huss, P., Evert, S., Romero, P., Raman, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We apply deep learning to understand and design the multifunctional host-targeting landscape of the T7 bacteriophage receptor binding protein for enhanced infectivity, pre-defined specificity, and high generality in virulence toward unseen strains.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> To better understand and design proteins, it is crucial to consider the multifunctional landscapes on which all proteins exist. Proteins are often optimized for single functions during design and engineering, without considering the countless other functionalities that may contribute to or interfere with the intended outcome. In this work, we apply deep learning to understand and design the multifunctional host-targeting landscape of the T7 bacteriophage receptor binding protein for enhanced infectivity, pre-defined specificity, and high generality in virulence toward unseen strains. We compare several different model architectures and design approaches and experimentally characterize designed phages optimized for 26 diverse tasks. We demonstrate that with multiobjective machine learning, it is possible to design complex specificities at success rates that can enable low-throughput validation of predicted hits. Our results show that the targeting capabilities of T7 are highly plastic, with opposite specificities often separated by only a few mutations. This level of tunability underscores how models trained on multifunctional data can uncover key principles of phage biology and specificity. The same modeling framework can be applied to guide the multiobjective design of other proteins or mutable biological systems, offering a general strategy for navigating multifunctional landscapes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.19.654895v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">difFUBAR: Scalable Bayesian comparison of adaptive evolution</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sadiq, H., Truong, P., Danielsson, M., Kumar, V., Nordlinder, H. N., Martin, D. P., Murrell, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose difFUBAR, which allows the demarcation of two branch sets of interest and, optionally, a background set, and estimates joint site-specific posterior distributions over , {omega}.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> While many phylogenetic methods exist to characterize evolutionary pressure at individual codon sites, relatively few allow direct comparison between different a priori selected sets of branches. Such comparisons may be useful for pinpointing precisely the codon sites that are under differing selective pressures due to differing environmental contexts, or differing genomic contexts via epistatic interactions. Indeed, this was only recently addressed by an approach, developed in the frequentist framework, that proposes a site-wise likelihood ratio hypothesis test. Previously, we have demonstrated that approximate grid-based Bayesian approaches to characterizing site-wise variation in selection parameters can outperform individual site-wise likelihood ratio tests. Such grid-based approaches can exhibit poor computational scaling when the number of site-wise parameters expands, but here we show that this is still tractable up to four parameters, and that a simple subtree-likelihood caching strategy can provide efficiency improvements in some cases. We propose difFUBAR, which allows the demarcation of two branch sets of interest and, optionally, a background set, and estimates joint site-specific posterior distributions over , {omega}1, {omega}2, and {omega}BG using a Gibbs sampler. Evidence for hypotheses of interest can then be quantified directly from the posterior distribution, and we standardly report P({omega}1>{omega}2), P({omega}2>{omega}1), P({omega}1>1), and P({omega}2>1). We characterize the computational and statistical performance of this approach on previous simulations, comparing it to a site-wise likelihood ratio test approach, where it shows moderate statistical benefits, and substantial computational gains, typically being more than two orders of magnitude faster on the same datasets. We also demonstrate that it can scale to datasets of over ten thousand taxa, on a laptop in under ten minutes. difFUBAR is implemented in MolecularEvolution.jl - a Julia framework for phylogenetic model development - and can be run locally, or online via a Colab notebook.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.19.654647v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Immune Spatial Organization Predicts Metastasis Risk in Aggressive Localized Prostate Cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yang, D. D., Abdelnaser, A., Wala, J., Haas, A. J., Barney, A. A., Saad, E., Crowdis, J. P., Ricker, C. A., Park, J., King, M. T., Nguyen, P. L., Choueiri, T. K., Tewari, A. K., Salari, K., Taplin, M.-E., Wu, C.-L., Van Allen, E. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify, validate, and provide interpretability for an immune microenvironment-derived computational pathology biomarker for high-grade PCa and show that spatial clustering of immune clusters, but not immune cell abundance, is independently associated with reduced risk of distant metastasis for high and low-grade disease</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Risk stratification in localized prostate cancer (PCa) remains imprecise. Computational pathology has emerged as an attractive option for improving risk stratification, but current approaches either lack interpretability or focus solely on tumor morphology. Here, we identify, validate, and provide interpretability for an immune microenvironment-derived computational pathology biomarker for high-grade PCa. Using digitized hematoxylin and eosin-stained (H&E) slides from two independent cohorts of patients treated with radical prostatectomy (n=490), we found that spatial clustering of immune clusters, but not immune cell abundance, was independently associated with reduced risk of distant metastasis for high-grade disease but not low-grade. Joint analysis of H&E images and RNA sequencing (n=326) revealed that in high-grade disease, high-cluster samples were enriched in CD8+ T cells, activated memory CD4+ T cells, and Tregs, as well as clonal T cell populations, overall suggestive of an underlying T cell-mediated antitumor response. No such enrichment was found in low-grade tumors. These findings establish immune spatial architecture as a novel, interpretable computational pathology biomarker and provide insight into the immune landscape of high-grade PCa.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.15.654324v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SWANS: A highly configurable analysis pipeline for single-cell and single-nuclei RNA-sequencing data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Beigel, K., Wafula, E., Mitchelle, D. V., Pastor, S. J., Gong, M., Heuckeroth, R., Ricarte-Filho, J. C., Franco, A. T., Reichenberger, E. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce SWANS, a tailorable single cell data analysis pipeline that can employ multiple analysis methods, resolutions, and modifiable parameters.</p><p style="font-size: 16px; text-align: justify;">Background: Single-cell RNA sequencing (scRNA-seq) is a powerful technique that enables the analysis of gene expression at the individual cell level. Bioinformatic tools for scRNA-seq data analysis have many different options throughout the typical scRNA-seq workflow (normalization, integration, annotation, clustering, and visualization), and the choice of method(s) and parameter(s) at each stage can impact results. Results: Here, we introduce SWANS (v2.0), a configurable analysis pipeline that, in a single run, can employ multiple analysis methods, resolutions, and modifiable parameters. The resulting clustering arrangements, differential gene expression results, and other quantitative measurements can be dynamically visualized and compared in a Shiny interactive report to assist in choosing a single analysis schema for annotation and downstream analysis. Once a final approach is chosen, SWANS will perform differential gene expression (DGE) analysis based on experimental conditions and gene set enrichment analysis (GSEA) in addition to creating reports that display figures and interactive tables, quality control metrics, and benchmarking information. SWANS uses Snakemake as a workflow manager, Cell Ranger for alignment and gene expression quantification, Seurat for single cell data analysis, and additional single cell R packages for quality control and downstream single cell analysis. Conclusion: SWANS is a tailorable pipeline that provides options for quality control, dimensionality reduction, clustering, differential gene expression analysis, gene set enrichment analysis, and trajectory analysis. Additionally, SWANS generates a series of reports that facilitate sharing large volumes of complex data in a clear and concise manner with other investigators.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.14.654073v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

