<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Tuesday, 06 May 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genomics and integrative clinical data machine learning scoring model to ascertain likely Lynch syndrome patients.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ramadhani Chambuso, Takudzwa Nyasha Musarurwa, Alessandro Pietro Aldera, Armin Deffur, Hayli Geffen, Douglas Perkins, Raj Ramesar</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a unique machine learning scoring model to ascertain likely-LS cases from a cohort of colorectal cancer (CRC) patients.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Lynch syndrome (LS) screening methods include multistep molecular somatic tumor testing to distinguish likely-LS patients from sporadic cases, which can be costly and complex. Also, direct germline testing for LS for every diagnosed solid cancer patient is a challenge in resource limited settings. We developed a unique machine learning scoring model to ascertain likely-LS cases from a cohort of colorectal cancer (CRC) patients.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40325286/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Machine Learning for Protein Science and Engineering.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Peter K Koo, Christian Dallago, Ananthan Nambiar, Kevin K Yang</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new approach to protein design based on machine learning and protein structure prediction .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent years have seen significant breakthroughs at the intersection of machine learning and protein science. Tools such as AlphaFold have revolutionized protein structure prediction. They are also enabling variant effect prediction and functional annotation of proteins, as well as opening up new possibilities for protein design. However, these technological advances must be balanced with sustainable computing practices.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40324837/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The Implication of The Vaginal Microbiome in Female Infertility and Assisted Conception Outcomes.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xiuju Chen, Yanyu Sui, Jiayi Gu, Liang Wang, Ningxia Sun</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new understanding of the vaginal microbiota of infertile women and its influence on fertility and ART outcomes .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The rise in infertility rates has prompted research into the impact of vaginal microbiota on female fertility and assisted reproduction technology (ART) success. Our study compares the vaginal microbiome of fertile and infertile women and explores its influence on ART outcomes. We analyzed vaginal secretions from 194 infertile women and 100 healthy controls at Shanghai Changzheng Hospital using polymerase chain reaction (PCR) to amplify the 16S rRNA V3-V4 region. A machine learning model predicted infertility based on genus abundances, and the PICRUSt algorithm predicted metabolic pathways related to infertility and ART outcome. The results showed women with infertility exhibited a significantly different vaginal microbial composition compared to healthy women, with the infertility group showing higher microbial diversity. Burkholderia, Pseudomonas, and Prevotella levels were significantly elevated in the vaginal microbiota of the infertility group, while Bifidobacterium and Lactobacillus abundances were reduced. Recurrent implantation failure (RIF) within the infertile population showed even higher diversity of vaginal microbiota, with specific genera such as Mobiluncus, Peptoniphilus, Prevotella, and Varibaculum being more abundant. Eleven metabolic pathways were associated with RIF and infertility, with Prevotella demonstrating stronger correlations. The present study provides insights into the differences in vaginal microbiome between healthy and infertile women, offering a new understanding of how vaginal microbiota may impact infertility and ART outcomes. Our findings underscore the significance of specific microbial taxa in women with RIF, suggesting avenues for targeted interventions to enhance embryo transplantation success rates.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40323322/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Automatic Biomarker Discovery and Enrichment with BRAD.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Joshua Pickard, Ram Prakash, Marc Andrew Choi, Natalie Oliven, Cooper Stansbury, Jillian Cwycyshyn, Nicholas Galioto, Alex Gorodetsky, Alvaro Velasquez, Indika Rajapakse</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed the Bioinformatics Retrieval Augmented Digital (BRAD) agent software system.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Integrating Large Language Models (LLMs) with research tools presents technical and reproducibility challenges for biomedical research. While commercial artificial intelligence (AI) systems are easy to adopt, they obscure data provenance, lack transparency, and can generates false information, making them unfit for many research problems. To address these challenges, we developed the Bioinformatics Retrieval Augmented Digital (BRAD) agent software system.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40323323/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">mimicDetector: a pipeline for protein motif mimicry detection in host-pathogen systems</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rich, K. D., Wasmuth, J. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present mimicDetector, an optimized bioinformatic pipeline for systematically identifying protein-level molecular mimicry between pathogens and their hosts.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: Molecular mimicry is a widespread strategy used by pathogens to evade the host immune system and manipulate other host cellular processes. Detecting these events--where pathogen proteins resemble host molecules--is challenging due to limitations in the sensitivity, specificity, and scalability of current bioinformatics tools. The challenges are pronounced when identifying subtle similarities in short protein fragments. Results: We present mimicDetector, an optimized bioinformatic pipeline for systematically identifying protein-level molecular mimicry between pathogens and their hosts. mimicDetector builds on existing k-mer-based approaches with three key improvements: (i) improved sensitivity for short-sequence alignments using the PAM30 substitution matrix and tuned BLASTP parameters; (ii) a revised k-mer filtering strategy based on bitscore differences rather than percent identity; (iii) the removal of overly conservative homologue exclusion steps. Applied to 17 globally important pathogens, mimicDetector identified a broad and biologically plausible set of mimicry candidates, including helminth proteins mimicking components of the human complement system and a Leishmania infantum mimic of Reticulon-4, a regulator of immune cell recruitment. Availability and implementation: mimicDetector is freely available at https://github.com/Kayleerich/mimicDetector/, implemented in Python, and compatible with Unix-based systems.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.02.651971v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Uncovering Convergent Cell State Dynamics Across Divergent Genetic Perturbations Through Single-Cell High-Content CRISPR Screening</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xu, Z., Lu, Z., Ugurbil, A., Abdulraouf, A., Liao, A., Zhang, J., Zhou, W., Cao, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present PerturbFate, a high-throughput, cost-effective, combinatorial-indexing single-cell platform that enables systematic interrogation of massively parallel CRISPR perturbations across the full spectrum of gene regulation, from chromatin remodeling and nascent transcription to steady</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> High-throughput genomic studies have uncovered associations between diverse genetic alterations and disease phenotypes; however, elucidating how perturbations in functionally disparate genes give rise to convergent cellular states remains challenging. Here, we present PerturbFate, a high-throughput, cost-effective, combinatorial-indexing single-cell platform that enables systematic interrogation of massively parallel CRISPR perturbations across the full spectrum of gene regulation, from chromatin remodeling and nascent transcription to steady-state transcriptomic phenotypes. Using PerturbFate, we profiled over 300,000 cultured melanoma cells to characterize multi-modal phenotypic and gene regulatory responses to perturbations in more than 140 Vemurafenib resistance-associated genes. We uncovered a shared dedifferentiated cell state marked by convergent transcription factor (TF) activity signatures across diverse genetic perturbations. Combined inhibition of cooperative TF hubs effectively reversed cellular adaptation to Vemurafenib treatment. We further dissected phenotypic responses to perturbations in Mediator Complex components, linking module-specific biochemical properties to convergent gene activations. Together, we reveal common regulatory nodes that drive similar phenotypic outcomes across distinct genetic perturbations. We also delineate how perturbations in functionally unrelated genes reshape cell state. PerturbFate thus establishes a versatile platform for identifying key molecular regulators by anchoring multi-modal regulatory dynamics to disease-relevant phenotypes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.02.651939v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PyCycleBio: modelling non-sinusoidal-oscillator systems in temporal biology</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bennett, A. R., Birchenough, G., Bojar, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a new bioinformatics platform for biologically-resolved rhythmic patterns, using bounded-multi-component models and modulus operators alongside the harmonic oscillator equation, to model a diverse and interpretable array of rhythmic behaviours, including the regulation of temporal dynamics via amplitude coefficients.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein, mRNA, and metabolite abundances can exhibit rhythmic dynamics, such as during the day/night cycle. Leading bioinformatics platforms for identifying biological rhythms often utilise single-component models of the harmonic oscillator equation, or multi-component models based upon the Cosinor framework. These approaches offer distinct advantages: modelling either temporally-resolved regulatory behaviour via the extended harmonic oscillator equation, or complex rhythmic patterns in the case of Cosinor. Here, we have developed a new platform to combine the advantages of these two approaches. PyCycleBio utilises bounded-multi-component models and modulus operators alongside the harmonic oscillator equation, to model a diverse and interpretable array of rhythmic behaviours, including the regulation of temporal dynamics via amplitude coefficients. We demonstrate increased sensitivity and functionality of PyCycleBio compared to other analytical frameworks, and uncover new relationships between data modalities or sampling conditions with the qualities of rhythmic behaviours from biological datasets- including transcriptomics, proteomics, and metabolomics. We envision that this new approach for disentangling complicated temporal regulation of biomolecules will advance chronobiology and our understanding of physiology. PyCycleBio is available at: https://github.com/Glycocalex/PyCycleBio, and the Python package is available to install at: https://pypi.org/project/pycyclebio/. PyCycleBio can also be used at https://colab.research.google.com/github/Glycocalex/PyCycleBio/blob/main/PyCycleBio.ipynb with no installations necessary.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.30.651403v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Athlytics: A Computational Framework for Longitudinal Analysis of Exercise Physiology Metrics from Wearable Sensor Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> HE, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed and introduce texttt{Athlytics}, a computational framework specifically designed for processing Strava API data directly within R for longitudinal exercise physiology analysis, which substantially enhances the feasibility, efficiency, and reproducibility of quantitative exercise physiology research.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The proliferation of wearable sensors provides large-scale longitudinal physiological data collected in real-world settings, offering unprecedented opportunities to investigate dynamic human responses to exercise interventions. However, systematically quantifying key physiological indicators related to adaptation and fatigue from these dense time-series data, particularly from popular platform APIs like Strava, and performing standardized, reproducible integrative analyses currently lacks established open-source workflows in R, posing significant practical and computational barriers. Researchers often expend considerable effort on custom programming, limiting analytical scale and efficiency. To overcome this critical bottleneck and empower broader research applications, we developed and introduce texttt{Athlytics}, a computational framework specifically designed for processing Strava API data directly within R for longitudinal exercise physiology analysis. This framework uniquely (or "provides a dedicated means to") seamlessly integrates data acquisition with the calculation of key physiological indicators (such as ACWR reflecting training stress balance, EF assessing aerobic efficiency, and physiological decoupling indicating cardiovascular stability), based on activity summary data while carefully addressing necessary approximations for composite load metrics like TSS/HRSS, and multidimensional time-series visualization. By providing standardized function interfaces, texttt{Athlytics} significantly lowers the technical barrier for conducting complex longitudinal analyses (e.g., modeling individualized physiological responses, identifying training patterns in large cohorts), enabling researchers to efficiently test hypotheses regarding the dynamic interplay between training stimuli, physiological efficiency, and stress responses. This work provides an open-source computational tool that fills a critical gap, contributing by substantially enhancing the feasibility, efficiency, and reproducibility of quantitative exercise physiology research utilizing widely available physiological sensor data through standardization and automation. The package (url{https://github.com/HzaCode/Athlytics}) provides an important foundation for standardizing and applying computational methods in this field.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.01.651597v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Gene regulatory networks linked to GABA signalling emerge as relevant for glioblastoma pathogenesis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nissen, I., Dakhel, S., Chakraborty, C., Nygaard, A. H., Kirkeby, A., Hornblad, A., Erdem, C., Remeseiro, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of the role of GABA-related networks in glioblastoma pathogenesis and a potential therapeutic target in GB tumour pathogenesis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Gene regulatory networks (GRNs) are essential for precise control of gene expression, orchestrating cellular functions by coordinating interactions between transcription factors (TFs), enhancers, and gene promoters. Disruptions in these regulatory networks are known to contribute to oncogenic processes, particularly because of alterations in epigenetic modifications, chromatin accessibility and promoter-enhancer interactions within the 3D nucleus. Here, we integrated multi-omics datasets using the machine learning-based MOBILE (Multi-Omics Binary Integration via Lasso Ensembles) pipeline to uncover novel GRNs and pathways involved in glioblastoma (GB) pathogenesis, one of the most aggressive brain cancers. GABA-signalling emerged as a previously unrecognized potential driver of GB tumour pathogenesis, with the discovery of GABA-related networks and relevant transcription factors (TFs) such as ARX, GSX2 and members of the DLX family, which play key roles in GABAergic interneuron development. Furthermore, in vitro co-culture assays showed that GABAergic input promotes GB cell proliferation, potentially through non-synaptic pathways such as metabolic or paracrine mechanisms. Our study provides insights into novel GRNs in GB pathogenesis, positioning GABA signalling as a potential therapeutic vulnerability in GB, and offering new promising therapeutic targets aimed at blocking neuron-glioma interactions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.30.651564v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Root Contours Guided Design of a Multicellular PID Controller</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Martinelli, V., Fiore, D., Salzano, D., di Bernardo, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a multicellular feedback control architecture based on the root contours method and validate its robustness and modularity through in silico simulations in BSim.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Ensuring a stable and robust phenotype expression is a key challenge for the correct operation of synthetically engineered cells, and feedback has been highlighted as a key mechanism to achieve this goal. Biomolecular PID controllers have been extensively leveraged at a single cell level to regulate gene expression. However, single-cell architectures suffer from limited modularity and might pose a significant challenge for their in vivo implementation due to high metabolic load and possible incompatible reactions. To overcome these limitations, it has been proposed to distribute the control actions over different cell populations realizing a multicellular feedback control architecture. In this paper we provide design guidelines derived by means of the root contours method to tune the control gains of a multicellular PID controller. We then validate performance, robustness and modularity of the multicellular PID controller through in silico simulations in BSim.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.05.652201v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

