<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Friday, 02 May 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Artificial intelligence-driven microsatellite instability profiling reveals distinctive genetic features in patients with lung cancer.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Thomas QD, Vendrell JA, Khellaf L, Cavaillon S, Quantin X, Solassol J, Cabello-Aguilar S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new method for detecting microsatellite instability in lung cancer using next-generation sequencing data and bioinformatics tools .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Microsatellite instability (MSI) has emerged as a predictive biomarker for immunotherapy response in various cancers, but its role in non-small cell lung cancer (NSCLC) is not fully understood.The authors used the bioinformatics tool MIAmS to assess microsatellite status from next-generation sequencing (NGS) data using a tailored microsatellite score. Immunohistochemistry (IHC) assays were also performed to evaluate the correspondence between MSI and deficient mismatch repair (dMMR) status. A retrospective analysis of 1547 lung cancer patients was conducted, focusing on those with an MSI phenotype. Clinical characteristics, co-occurring molecular alterations, tumor mutation burden (TMB), and homologous recombination deficiency (HRD) status were evaluated in this subset.Of the 1547 patients analyzed, eight (0.52%) were identified as having MSI through MIAmS, with six (0.39%) of these cases also being dMMR on IHC. All patients with dMMR had an MS score ≥2 and a history of smoking. Most patients showed loss of MLH1 and PMS2 staining on IHC. No correlation was found between MSI status and programmed death-ligand 1 expression, although all MSI patients exhibited high TMB, averaging 21.4 ± 5.6 mutations per megabase.MSI/dMMR in lung cancer is exceedingly rare, affecting less than 1% of cases. NGS-based analysis combined with bioinformatics tools provides a robust method to identify MSI/dMMR patients, potentially guiding immunotherapy decisions. This comprehensive approach integrates molecular genotyping and MSI detection, offering personalized treatment options for lung cancer patients. NGS-based MSI testing is emerging as the preferred method for detecting microsatellite instability in various tumor types, including rare cancers.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40297960/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrative multi-omics modelling for cultivated meat production, quality, and safety</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mathieu, T., Legare, S., Nzekoue, A. F., Jaure, N., Lester, H., Dias, T., Kusters, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A network-based interactome model that integrates the transcriptomic, proteomic, and metabolomic layers, we achieve a system-level understanding of cellular metabolism and regulatory mechanisms, which can allow for precise monitoring and targeted interventions of critical quality and safety attributes associated with cellular biomass.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cell culture technology, which offers a promising solution for complementary food production, is slowly becoming a reality with the first wave of regulatory approvals in pioneering markets. However, significant challenges remain for large-scale cultivated meat commercialization, including effective scaling, cost efficiency, and product quality, but also scientific evidence to support regulatory approval and building consumer trust. In this paper, we discuss the potential of an integrative multi-omics approach to characterize and optimize cultivated meat production. By analyzing a network-based interactome model that integrates the transcriptomic, proteomic, and metabolomic layers, we achieve a system-level understanding of cellular metabolism and regulatory mechanisms. This approach can allow for precise monitoring and targeted interventions of critical quality and safety attributes associated with cellular biomass. We then describe a Target-Action-Metabolite (TAM) framework, which utilizes insights from the interactome to optimize cell culture conditions through actionable interventions. We illustrate the potential use of this framework through a case study involving Duck Embryonic Stem Cells (dESCs) for use in cell-cultured meat products, providing hypotheses for improving key metabolic pathways through targeted interventions on metabolites present in culture media. Finally, our paper highlights the potential of this interactome-based strategy to enhance bioprocess efficiency, improve product quality and ensure safety attributes, addressing regulatory challenges associated with cultivated meat production.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.30.651459v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Evaluation of epistasis detection methods for quantitative phenotypes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Listopad, S., Peng, Q., Renjith, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified six epistasis detection methods suitable for quantitative phenotype data: EpiSNP, Matrix Epistasis, MIDESP, PLINK Epistasia, QMDR, and REMMA, and evaluated their performance on the Adolescent Brain Cognitive Development (ABCD) dataset.</p><p style="font-size: 16px; text-align: justify;">Background Epistasis, or genetic interaction, has been increasingly recognized for its ubiquity and for its role in susceptibility to common human diseases, such as Alzheimers. A wide variety of epistasis detection tools are currently available with several studies comparing the performance of methods suitable for case-control data. However, there is limited understanding of how well these tools perform with quantitative phenotypes. Methods We identified six epistasis detection methods suitable for quantitative phenotype data: EpiSNP, Matrix Epistasis, MIDESP, PLINK Epistasis, QMDR, and REMMA. To evaluate these tools, we generated simulated datasets using EpiGEN. The datasets modeled various pairwise interactions between disease-associated SNPs, including dominant, multiplicative, recessive, and XOR interactions. Additionally, we assessed the BOOST and MDR algorithms on discretized (case-control) version of the datasets. These tools were then tested on the Adolescent Brain Cognitive Development (ABCD) dataset for the externalizing behavior phenotype. Results Each tool exhibited strong performance for certain interaction types, but weaker performance for others. MDR achieved the highest overall detection rate of 60%, while EpiSNP had the lowest overall detection rate of 7%. MDR and MIDESP performed best at detecting multiplicative interactions with detection rates of 54% and 41% respectively. Both MDR and MIDESP were also effective at detecting XOR interactions with detection rates of 84% and 50% respectively. PLINK Epistasis, Matrix Epistasis, and REMMA excelled at detecting dominant interactions, all achieving a 100% detection rate. On the other hand, EpiSNP was particularly effective at detecting recessive interactions with a detection rate of 66%. When analyzing the ABCD dataset, Plink Epistasis and Plink BOOST identified SNPs within the DRD2 and DRD4 genes, which have been previously linked to externalizing behavior. Conclusion Since no single method consistently outperforms others across all types of epistasis, and given that the specific types of epistasis present in a dataset are often unknown, it may be more effective to use multiple epistasis detection algorithms in combination to obtain comprehensive results.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.30.651312v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">From Pairwise Distances to Neighborhood Preservation: Benchmarking Dimensionality Reduction Algorithms for CyTOF, scRNA-seq, and CITE-seq</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bombina, P., Adams, Z. B., McGee, R. L., Coombes, K. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comprehensive evaluation of 16 dimensionality reduction methods for high-dimensional data visualization .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Dimensionality reduction algorithms are essential tools for visualizing high-dimensional biological data, such as single-cell transcriptomics, mass cytometry by time of flight, and cellular indexing of transcriptomes and epitopes. These algorithms map complex, high dimensional data into lower dimensions to reveal underlying structures and patterns. Despite the popularity of dimensionality reduction methods like t-SNE and UMAP, concerns have arisen regarding their ability to preserve critical aspects of high-dimensional data and their sensitivity to user-defined parameters. This study aims to evaluate the impact of extreme dimension reduction from hundreds or thousands of dimensions to just two dimensions, highlighting the resulting distortions to deeply understand their implications. Given the significance of dimensionality reduction in biological research, careful evaluation of these methods is necessary to ensure reliable and meaningful results. In this paper, we present a comprehensive evaluation of 16 dimensionality reduction methods. Our evaluation addresses several key factors, such as the preservation of pairwise distances and local neighborhood relationships between the original high-dimensional space and the low-dimensional projections.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.28.651069v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">iSHARC: Integrating scMultiome data for heterogeneity and regulatory analysis in cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zeng, Y., Bahl, S., Xu, X., Ci, X., Keshavarzian, T., Yang, L., Lee, H. S., Kossinna, P., Gaiti, F., Schwartz, G. W., He, H. H., Lupien, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present iSHARC, a streamlined pipeline for quality control, modality integration, clustering, cell type annotation, and regulatory mechanism analysis of individual scMultiome data, as well as for integrating multiple samples.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The 10x Genomics single cell Multiome (scMultiome) assay enables the simultaneous profiling of chromatin accessibility and gene expression from the same nucleus, and has increasingly been utilized in revealing cellular heterogeneity and gene regulation in cancers. However, a dedicated bioinformatics pipeline specifically designed for this type of data is still lacking. Here we present iSHARC, a streamlined pipeline for quality control, modality integration, clustering, cell type annotation, and regulatory mechanism analysis of individual scMultiome data, as well as for integrating multiple samples. The main advantages of iSHARC are: 1) easy implementation, execution and extension through a modular Snakemake workflow management system; 2) flexible analysis and parameters customization via a single configuration file; and 3) comprehensive accessibility by providing different access points to results and detailed summary reports from a single run.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.28.651068v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">OmiXAI: An Ensemble XAI Pipeline for Interpretable Deep Learning in Omics Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Alaeva, A., Lapteva, A., Mikhaylovskaya, N., Malkov, V., Herbert, A., Borevskiy, A., Poptsova, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed OmiXAI, a pipeline integrating ensemble model-aware XAI methods with gradient-based techniques including Integrated Gradients, InputXGradients, Guided Backpropagation, and Deconvolution (for CNNs and GNNs).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Deep learning methods have become methods of choice in the analysis of genomic data. The performance of deep learning models depends on the information available for training. A growing trend in deep learning applications involves leveraging multi-omics data spanning genomics, transcriptomics, epigenomics, proteomics, metabolomics, and other domains. When a deep learning model trained on omics data achieves high performance, the important question is to define factors that contribute to model predictive power. Explainable AI (XAI) methods can be categorized as model-aware and model-agnostic. Model-agnostic approaches, which rely on combinatorial feature perturbations to assess impact, are often computationally prohibitive for deep learning models. To address this, we developed OmiXAI, a pipeline integrating ensemble model-aware XAI methods. Our framework incorporates gradient-based techniques including Integrated Gradients, InputXGradients, Guided Backpropagation, and Deconvolution (for CNNs and GNNs) as well as Saliency Maps and GNNExplainer (specifically for GNNs). We evaluated OmiXAI on Z-DNA prediction using multi-omics features, demonstrating its efficacy through feature importance analysis and benchmarking of XAI methods. Notably, OmiXAI enabled feature engineering, reducing the critical feature set from almost 2,000 to just 50. Its modular design allows seamless integration of additional attribution methods, ensuring adaptability beyond omics to diverse problem domains. While testing the ensemble approach we benchmarked individual XAI methods and discuss their drawbacks and limitations. OmiXAI is freely available at https://github.com/aameliig/OmiXAI.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.28.651097v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The Consortium for Genomic Diversity, Ancestry, and Health in Colombia (CODIGO): building local capacity in genomics, bioinformatics, and precision medicine</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Marino-Ramirez, L., Sharma, S., Hamilton, J. M., Nguyen, T. L., Gupta, S., Natarajan, A. V., Nagar, S. D., Menuey, J. L., Chen, W.-A., Sanchez-Gomez, A., Satizabal-Soto, J. M., Martinez, B., Marrugo, J., Medina-Rivas, M., Gallo, J. E., Jordan, I. K., Valderrama-Aguirre, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> The Consortium for Genomic Diversity, Ancestry, and Health in Colombia (CODIGO) aims to build a community of Colombian researchers in support of local capacity in genomics, bioinformatics, and precision health.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The Consortium for Genomic Diversity, Ancestry, and Health in Colombia (CODIGO) aims to build a community of Colombian researchers in support of local capacity in genomics, bioinformatics, and precision health. Here, we present the first CODIGO data release and the consortium web platform, including annotations for more than 95 million genetic variants from 1,441 samples representing 14 populations from across the country. CODIGO samples show a wide range of African (16.7%), European (50.6%), and Indigenous American (32.8%) genetic ancestry components, with five distinct ancestry clusters. Thousands of ancestry-enriched variants, with divergent allele frequencies across clusters, show pharmacogenomic and clinical genetic associations. Examples include African ancestry-enriched variants associated with fast metabolism of the immunosuppressive drug tacrolimus and malaria resistance and European ancestry-enriched variants associated with nicotine dependence and hereditary hemochromatosis. CODIGO reveals the nexus between ancestry and health in Colombia and underscores the utility of collaborative genome sequence analysis efforts.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.28.651081v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ScIsoX: A Multidimensional Framework for Measuring Transcriptomic Complexity in Single-Cell Long-Read Sequencing Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wu, S., Schmitz, U.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce ScIsoX, a computational framework that integrates a novel hierarchical data structure, complexity metrics, and visualisation tools for isoform-level investigations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell long-read sequencing enables comprehensive characterisation of full-length transcript isoforms, but lacks robust analytical frameworks for measuring transcriptomic complexity. Here, we introduce ScIsoX, a computational framework that integrates a novel hierarchical data structure, complexity metrics, and visualisation tools for isoform-level investigations. ScIsoX enables systematic interrogation of global and cell-type-specific isoform expression patterns resulting from splicing, revealing distinct complexity signatures across diverse datasets that were previously inaccessible with short-read approaches.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.28.650897v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Precision Through Frequency-Aware Prioritization: Introducing the Bouziane Similarity Index (BSI) for Rare Variants.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> BOUZIANE, I., LOUAFI, K., LOUAFI, O., ALLOUCHE, M., SEKIOU, O., BOUMENDJEL, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A frequency-aware genetic similarity metric that prioritizes rare variant sharing in relationship inference.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Current genetic similarity metrics, including Identity-by-State (IBS) and Kinship coefficients, treat all alleles equally, ignoring allele frequency differences and partial genetic sharing (Purcell et al., 2007; Speed & Balding, 2015). Despite the recognized biological importance of rare variants (Lee et al., 2012), no widely adopted similarity metric prioritizes rare variant sharing in relationship inference. We introduce two novel metrics: the Bouziane Similarity Index (BSI), a frequency-weighted score emphasizing rare variant matches, and BSI-VAR, a variant that integrates allele dosage and inverse variance weighting to capture partial allele sharing and amplify rare variant signals. To evaluate these metrics, we first performed unsupervised principal component analysis (PCA) on simulated genotype similarity matrices, showing that BSI and BSI-VAR provide clearer genetic structure compared to IBS. We then conducted controlled simulations involving Parent-Child, Sibling, and Random genetic pairs to quantify discriminative performance. Through all evaluations, BSI and especially BSI-VAR consistently outperformed IBS in distinguishing genetic relationships. These findings establish BSI and BSI-VAR as robust, interpretable, and rare-variant-aware alternatives for modern genetic similarity analysis, with potential applications in population genetics, ancestry inference, and precision medicine (Ashley, 2016; Manolio et al., 2009). Keywords Rare variant prioritization; Frequency-aware genetic similarity; Bouziane Similarity Index (BSI); Diagnostic genomics; Precision medicine and genomics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.28.651048v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Mass balance approximation of unfolding boosts potential-based protein stability predictions.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ivan Rossi, Guido Barducci, Tiziana Sanavia, Paola Turina, Emidio Capriotti, Piero Fariselli</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that incorporating mass balance as a first approximation of the unfolded state significantly improves potential-like methods for predicting protein stability change predictions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Predicting protein stability changes upon single-point mutations is crucial in computational biology, with applications in drug design, enzyme engineering, and understanding disease mechanisms. While deep-learning approaches have emerged, many remain inaccessible for routine use. In contrast, potential-like methods, including deep-learning-based ones, are faster, user-friendly, and effective in estimating stability changes. However, most of them approximate Gibbs free-energy differences without accounting for the free-energy changes of the unfolded state, violating mass balance and potentially reducing accuracy. Here, we show that incorporating mass balance as a first approximation of the unfolded state significantly improves potential-like methods. While many machine-learning models implicitly or explicitly use mass balance, our findings suggest that a more accurate unfolded-state representation could further enhance stability change predictions.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40277391/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Progress and innovations of gene cloning in wheat and its close relatives.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Korchanová Z, Milovanov A, Švec M, Doležel J, Bartoš J, Valárik M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #fedbf2; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, pan-genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comprehensive overview of the genes cloned in wheat so far and discuss the strategies used for cloning these genes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Wheat and its close relatives have large and complex genomes, making gene cloning difficult. Nevertheless, developments in genomics over the past decade have made it more feasible. The large and complex genomes of cereals, especially bread wheat, have always been a challenge for gene mapping and cloning. Nevertheless, recent advances in genomics have led to significant progress in this field. Currently, high-quality reference sequences are available for major wheat species and their relatives. New high-throughput genotyping platforms and next-generation sequencing technologies combined with genome complexity reduction techniques and mutagenesis have opened new avenues for gene cloning. In this review, we provide a comprehensive overview of the genes cloned in wheat so far and discuss the strategies used for cloning these genes. We highlight the advantages and drawbacks of individual approaches and show how particular genomic progress contributed to wheat gene cloning. A wide range of new resources and approaches have led to a significant increase in the number of successful cloning projects over the past decade, demonstrating that it is now feasible to perform rapid gene cloning of agronomically important genes, even in a genome as large and complex as that of wheat.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40295316/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Gene expression profiling, prognosis, and immune microenvironment of KLF4 in malignancies.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yu S, Qian L, Tong T, Ma X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #fedbf2; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, pan-genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A systematic analysis of genetic alterations, dysregulated RNA expression, and prognostic implications of KLF4 in malignancies.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Gene expression profiling plays a crucial role in understanding the role of Krüppel-like factor 4 (KLF4) in the prognosis and tumor immune microenvironment of various malignancies. The transcription factor KLF4 plays a crucial role in various cellular processes, including cell differentiation, proliferation, and apoptosis. Genetic alterations and aberrant KLF4 expression have been observed in many malignancies, thus suggesting a potential role as a prognostic marker and therapeutic target. Herein, a systematic analysis of KLF4 genetic alterations revealed the mutation, amplification, and deletion frequencies across different cancer types. The genetic alteration patterns varied across malignancies, thus highlighting the diverse roles of KLF4 in different tumor contexts. Secondly, the prognostic significance of KLF4 expression was assessed in multiple cancers. High expression levels of KLF4 were associated with better clinical outcomes in kidney renal clear cell carcinoma, while low KLF4 expression correlated with a favorable prognosis in certain malignancies. In conclusion, the genetic alterations, dysregulated RNA expression, and prognostic implications of KLF4 in malignancies underscore its significance in cancer biology. The present findings will aid in understanding the role of KLF4 in tumor biology and its association with immune responses. Future investigations should focus on elucidating the functional roles and regulatory mechanisms of KLF4 to further assess its potential as a therapeutic target and predictive biomarker in cancer management.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40299916/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

