<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Tuesday, 19 August 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">BAV-LLPS: A database of bacterial, archaea and virus liquid-liquid phase separation proteins</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rodriguez, C. B., Tunque Cahui, R. R., Demitroff, N., Hirsh, L., Devos, D. P., Boccaccio, G., Parisi, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a hand-curated database of LLPS proteins from bacteria, archaea, and viruses (BAV-LLPS Database). This resource, extended through sequence similarity searches, comprises over 5,000 proteins and integrates diverse data including biological annotations, sequence features, predicted disordered regions,</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Liquid-liquid phase separation (LLPS) is a key process underlying the formation of biomolecular condensates (BMCs), such as membrane-less organelles (MLOs), that compartmentalize biochemical processes inside the cells. While LLPS has been extensively studied in eukaryotes, its role in bacteria, archaea, and viruses remains far less characterized. Recent studies in bacteria have revealed that LLPS-driven condensates play critical roles in RNA processing, stress response, and pathogenicity. Similarly, many viruses exploit LLPS to facilitate crucial steps in their infection cycles, including viral entry, genome replication, assembly, and host immune evasion. In this work, we introduce a hand-curated database of LLPS proteins from bacteria, archaea, and viruses (BAV-LLPS Database). This resource, extended through sequence similarity searches, comprises over 5,000 proteins and integrates diverse data including biological annotations, sequence features, predicted disordered regions, LLPS per site probability, and AlphaFold2-based structural models. Additionally, our web server enables users to explore both the curated and homologous derived datasets, providing a platform to uncover evolutionary relationships and intrinsic and differential properties of LLPS proteins across various taxonomic groups. This work seeks to deepen our understanding of LLPS mechanisms beyond eukaryotic organisms, emphasizing their significance across diverse life forms. It also aims to foster the development of specialized predictive tools that will facilitate the exploration and characterization of LLPS processes in a wide array of living organisms, thereby contributing to advancements in both fundamental biological research and applied biomedical sciences.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.15.670539v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Semi-supervised detection of natural selection with positive-unlabeled learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Arnab, S. P., Campelo dos Santos, A. L., Fumagalli, M., DeGiorgio, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce positive-unlabeled learning as a more flexible framework for detection of adaptive events, with the potential to generalize across a range of genomic landscapes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Identifying genomic regions shaped by natural selection is a central goal in evolutionary genomics. Existing machine learning methods for this task are typically trained using simulated genomic data labeled according to specific evolutionary scenarios. While effective in controlled settings, these models are limited by their reliance on explicit class labels. They can only detect the specific processes they were trained to recognize, making it difficult to interpret predictions for regions influenced by other evolutionary forces. This limitation is especially problematic when analyzing empirical genomes shaped by a mixture of adaptive, demographic, and ecological factors. One-vs.-rest strategies offer a potential alternative, but suffer from the inherent complexity of modeling all other evolutionary and demographic processes as a catch-all ``rest'' class. Here, we explore positive-unlabeled learning as a more flexible framework for detection of adaptive events. Positive-unlabeled learning is a semi-supervised approach that permits identification of samples of a target class using only positive labels and an unlabeled background, without requiring explicit modeling of negative samples. To assess the utility of this approach, we focus on a binary classification setting for detecting selective sweeps (positive samples) arising from positive natural selection against a mixed background composed of both unlabeled sweeps and neutrally-evolving regions. To accomplish this goal, we introduce PULSe, a method that employs only a set of labeled sweep observations for training while treating all remaining data as unlabeled. By avoiding assumptions about the composition of the background, PULSe enables robust sweep discovery in realistic genomic landscapes. We systematically evaluate its performance across a range of demographic, adaptive, and confounding contexts, including domain shift arising from misspecified demographic models, and find that PULSe delivers high performance and generalizability. To demonstrate a practical application of PULSe, we analyzed European and Bengali in Bangladesh human genomes, treating the empirical genomes as unlabeled sets, and recapitulating several previously-identified sweep candidates. Our results show that PULSe provides a powerful and versatile alternative for detecting adaptive regions, with the potential to generalize across a range of genomic landscapes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.15.670602v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Ancestral intronic splicing regulatory elements in the SCNα gene family</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chernyavskaya, E., Vorobeva, M., Spirin, S. A., Skvortsov, D. A., Pervouchine, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the nonsense mediated decay (NMD) system plays a role in maintaining mutually exclusive choice of human exons 5a and 5b, and characterize their splicing patterns across tissues.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> SCN genes encode components of voltage-gated sodium channels that are crucial for generating electrical signals in many organisms. Humans have ten paralogous SCN genes, some of which contain duplicated mutually exclusive exons 5a and 5b (MXE). In reconstructing their evolutionary history, we report multiple unannotated copies of exon 5 in distant species and show that exon 5 duplication likely goes back to a common ancestor of all SCN genes. We provide evidence from RNA-seq experiments that the nonsense mediated decay (NMD) system plays a role in maintaining mutually exclusive choice of human exons 5a and 5b, and characterize their splicing patterns across tissues and developmental stages. Comparative analysis of nucleotide sequences among paralogs identified two highly conserved complementary intronic elements, IRE1 and IRE2, possibly forming RNA structure that impacts splicing. Indeed, blocking of IRE2 by antisense oligonucleotides significantly changed the 5a/5b ratio without the loss of mutually exclusive pattern. IRE1 and IRE2 are conserved from human to Elephant shark and may represent an ancient, evolutionarily conserved regulatory mechanism that is shared across many SCN genes. These results demonstrate the power of comparative analysis of paralogs for elucidating conserved splicing regulatory mechanisms.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.16.670673v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identifying pyrogenic contaminants using transcriptomic profiling of monocyte activation test with machine learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Afanasyeva, T. A., de Albuquerque, B. F., Doodeman, P., Dieker-Meijer, M., Molenaar-de Backer, M., van Dam, T. J., ten Brinke, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the monocyte activation test for pyrogenicity can be used to distinguish between different types of pyrogenic contaminants.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The monocyte activation test is an in vitro pyrogenicity assessment method that can utilise human peripheral blood mononuclear cells to detect pyrogens in injectable drugs, providing a binary outcome that indicates the presence or absence of a pyrogen. The added ability to distinguish between different types of pyrogens would greatly expand the applicability of the test, for example, by allowing to pinpoint the source of a contaminating pyrogen in pharmaceutical products. Pyrogens activate a unique set of pattern recognition receptors (PRRs), which contribute to inflammation, yielding distinct transcriptomic activation signatures. In this paper, we capture the unique expression signatures of activated monocytes through bulk RNA sequencing and introduce a data preprocessing pipeline that allows the training of a machine-learning model to classify pyrogenic contaminants. Using a dataset of 108 samples stimulated with five classes of PRR agonists, we could differentiate between these classes with more than 97% F1 on test data. We further demonstrate the model's capacity to generalise on the previously unseen data using different ligands for the same PRRs as well as heat-killed Escherichia coli and Staphylococcus aureus.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.13.670109v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">RNA m6A Methylation Control Salt Response by Affecting Photosynthesis Capacity</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yang, Q., Wang, S., Si, J., Pang, Q., Zhang, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that Arabidopsis mutants deficient in m6A writers or readers exhibit heightened salt sensitivity at both growth and physiological levels and uncover the specific role of RNA metabolism in plant response to salinity .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Soil salinization is a major abiotic stress constraining global plant growth. Epigenetic regulation particularly RNA modifications like N6-methyladenosine (m6A) is crucial for plant stress responses, the specific role of m6A in salt tolerance remains unclearly defined. Here, we show that Arabidopsis mutants deficient in m6A writers or readers exhibit heightened salt sensitivity at both growth and physiological levels. Global m6A modification levels increase following salt stress which results in a significant elevation of m6A signature throughout the Arabidopsis transcriptome. Spatiotemporal analysis through standardized read density of MeRIP-seq uncovered a significant salt-induced redistribution of m6A modification patterns. More than 80% m6A reads locate around the stop codon within 3'-untranslated regions and tend to increase under salt stress. A total of 9,986 peaks aligned into 8,667 coding gene showing significant changes in m6A modification levels. By associating with gene expression profiling, 840 salt-responsive genes display significant alterations in m6A enriched level and negative regulation pattern happens in most of genes. The salt responsive genes showing increased m6A methylation but significantly decreased expression are focused which representatively enriched in photosynthesis pathway, and this suppression due to mRNA decay mediated by m6A modification. Photosynthetic capacity and chloroplast apparatus are impaired in m6A-dependent manner under salt stress. These results clarify the biological functions of m6A modification in plant response to salinity and uncover the specific role of RNA metabolism based on dynamic change pattern of m6A modification to cope with salt stress.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.13.670251v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Simulating the Impacts of Climate Change on UH Manoa Lettuce (Lactuca sativa) Growth by Modifying Air Temperature, Soil Water Availability, and Atmospheric CO2 Concentration</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yos, N., Mora, C., Webster, K., McDowell, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of the cumulative influences of temperature, water availability, and atmospheric CO2 concentration on lettuce growth .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Plant species are adapted to survive under specific ranges of temperature, water availability, and atmospheric CO2 concentration. Climate change-induced shifts in these environmental conditions have the potential to significantly affect nearly all terrestrial plants. A number of studies have explored the impacts of changing one or two of the conditions listed above, but few have examined the combined effects of all three. To study the cumulative influences of the three environmental conditions, 350 M[a]noa lettuce (Lactuca sativa) plants were grown in indoor growth chambers. Within the chambers, plants were grown under varying degrees of CO2 concentration, water availability, and temperature for 21 days. At the end of this period, the leaf mass (biomass) of each plant was cut, dried, and weighed. Percent mortality and nitrogen content were also measured. Across all combinations of temperature and water availability, elevated CO2 concentrations were associated with increased biomass production and survival rates. Survival and biomass decreased under high temperatures and both high and low water availability. The combination of environmental conditions that produced the largest amount of biomass was 750 ppm CO2, 80% of soil water capacity, and 24 {degrees}C while the treatment that produced the least amount was ambient CO2, 60% of soil water capacity, and 36 {degrees}C. Nitrogen content increased under high temperatures and water availability. These results suggest that although increased atmospheric CO2 levels have the potential to promote lettuce growth, lettuce yield is still likely to decrease in many regions due to the negative effects of high temperatures, drought, and flooding.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.13.670242v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">groovDB in 2026: A community-editable database of small molecule biosensors</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Love, J. D., Rafferty, B. M., Thomas, M., Springer, M., Silver, P. A., d'Oelsnitz, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A major update to groovDB, making the database community-editable and adding several advanced features.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The groovDB database (https://groov.bio) was launched in 2022 with the goal of organizing information on prokaryotic ligand-inducible transcription factors (TFs). This class of proteins is important in fundamental areas of microbiology research and for biotechnological applications that develop biosensors for diagnostics, enzyme screening, and real-time metabolite tracking. Uniquely, groovDB contains stringently curated, literature-referenced data on both TF:DNA and TF:ligand interactions. Here, we describe a major update to groovDB, making the database community-editable and adding several advanced features. Users can now add new TF entries and update existing entries using a simple online form. New user interface elements allow viewing of interactive TF protein structures and DNA-binding motifs. Updated query methods enable text-based search, chemical similarity search, and attribute-filtering searches. A new data architecture improves web interface responsiveness by over five-fold. Finally, the number of TF entries has more than doubled and all source code is now open-access.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.18.670880v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

