<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Friday, 30 May 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">scMODAL: a general deep learning framework for comprehensive single-cell multi-omics data alignment with feature links.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gefei Wang, Jia Zhao, Yingxin Lin, Tianyu Liu, Yize Zhao, Hongyu Zhao</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce scMODAL, a deep learning framework tailored for single-cell multi-omics data alignment using feature links.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent advancements in single-cell technologies have enabled comprehensive characterization of cellular states through transcriptomic, epigenomic, and proteomic profiling at single-cell resolution. These technologies have significantly deepened our understanding of cell functions and disease mechanisms from various omics perspectives. As these technologies evolve rapidly and data resources expand, there is a growing need for computational methods that can integrate information from different modalities to facilitate joint analysis of single-cell multi-omics data. However, integrating single-cell omics datasets presents unique challenges due to varied feature correlations and technology-specific limitations. To address these challenges, we introduce scMODAL, a deep learning framework tailored for single-cell multi-omics data alignment using feature links. scMODAL integrates datasets with limited known positively correlated features, leveraging neural networks and generative adversarial networks to align cell embeddings and preserve feature topology. Our experiments demonstrate scMODAL's effectiveness in removing unwanted variation, preserving biological information, and accurately identifying cell subpopulations across diverse datasets. scMODAL not only advances integration tasks but also supports downstream analyses such as feature imputation and feature relationship inference, offering a robust solution for advancing single-cell multi-omics research.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40442129/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Using supervised machine-learning approaches to understand abiotic stress tolerance and design resilient crops.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rajneesh Singhal, Paulo Izquierdo, Thilanka Ranaweera, Kenia Segura Abá, Brianna N I Brown, Melissa D Lehti-Shiu, Shin-Han Shiu</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce ML approaches and provide examples of how researchers use these approaches to predict molecular activities, gene functions and genotype responses under stressful conditions, which can be used to guide the design of resilient crops.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Abiotic stresses such as drought, heat, cold, salinity and flooding significantly impact plant growth, development and productivity. As the planet has warmed, these abiotic stresses have increased in frequency and intensity, affecting the global food supply and making it imperative to develop stress-resilient crops. In the past 20 years, the development of omics technologies has contributed to the growth of datasets for plants grown under a wide range of abiotic environments. Integration of these rapidly growing data using machine-learning (ML) approaches can complement existing breeding efforts by providing insights into the mechanisms underlying plant responses to stressful conditions, which can be used to guide the design of resilient crops. In this review, we introduce ML approaches and provide examples of how researchers use these approaches to predict molecular activities, gene functions and genotype responses under stressful conditions. Finally, we consider the potential and challenges of using such approaches to enable the design of crops that are better suited to a changing environment.This article is part of the theme issue 'Crops under stress: can we mitigate the impacts of climate change on agriculture and launch the 'Resilience Revolution'?'.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40439305/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Resolving the full set of human polymorphic inversions and other complex variants from ultra-long read data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Moreira-Pinhal, R., Karakostis, K., Yakymenko, I., Conchillo, O., Diaz-Ros, M., Santos, A., Senar, M. A., Martinez-Urtaza, J., Puig, M., Caceres, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify 612 candidate inversions between 197 bp and 4.4 Mb of length and flanked by long inverted repeats (IRs) and show that ultra-long reads (50-100 kb) and adaptive sampling are an efficient method to detect most human inversions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Inversions are a unique type of balanced structural variants (SVs) with important consequences in multiple organisms. However, despite considerable effort, this and other complex SVs remain poorly characterized due to the presence of large repeats. New techniques are finally allowing us to identify the full spectrum of human inversions, but the number of individuals analyzed is still quite limited. Here, we take advantage of Oxford Nanopore Technologies (ONT) long reads to characterize an exhaustive catalogue of 612 candidate inversions between 197 bp and 4.4 Mb of length and flanked by <190-kb long inverted repeats (IRs). For that, we developed a bioinformatic package to identify inversion alleles reliably from long read data. Next, using a combination of different DNA extraction, library preparation, and ONT sequencing protocols, we showed that ultra-long reads (50-100 kb) and adaptive sampling are an efficient method to detect most human inversions. Lastly, by analyzing ONT data from 54 diverse individuals, 87-99% of the inversions could be genotyped in each sample, depending mainly on read and IR length and genome coverage. Both orientations were observed for 155 of the analyzed regions (frequency 0.01-0.49), which multiplies by three the polymorphic IR-mediated inversions studied in detail so far. Moreover, we found more than 300 additional independent SVs in the studied regions and resolved several complex rearrangements. Our work therefore provides an accurate benchmark of those inversions that typically escape most analyses, improving existing resources, such as the Pangenome. In addition, it demonstrates the potential of nanopore sequencing to determine the functional impact of missing human genomic variation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.27.656315v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CNAttention: an attention-based deep multiple-instance method for uncovering CNA signatures across cancers</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yang, Z., Baudis, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce CNAttention, an attention-based deep multiple instance learning method designed to comprehensively analyze CNAs across different cancers and uncover specific CNA patterns within integrated gene-level CNA profiles of 30 cancer types.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Somatic copy number aberrations (CNAs) represent a distinct class of genomic mutations associated with oncogenetic effects. Over the past three decades, significant volumes of CNA data have been generated through molecular-cytogenetic and genome sequencing-based techniques. This data has been pivotal in identifying cancer-related genes and advancing research on the relationship between CNAs and histopathologically defined cancer types. However, comprehensive studies of CNA landscapes and disease parameters are challenging due to the vast diagnostic and genomic heterogeneity encountered in "pan-cancer" approaches. In this study, we introduce CNAttention, an attention-based deep multiple instance learning method designed to comprehensively analyze CNAs across different cancers and uncover specific CNA patterns within integrated gene-level CNA profiles of 30 cancer types. CNAttention effectively learns CNA features unique to each cancer type and generates CNA signatures for 30 cancer types using attention mechanisms, highlighting the distinctiveness of their CNA landscapes. CNAttention demonstrates high accuracy and exhibits stable performance even with the incorporation of external datasets or parameter adjustments, underscoring its effectiveness in tumor identification. Expanding these signatures to cancer classification trees reveals common patterns not only among physiologically related cancer types but also among clinico-pathologically distant types, such as different cancers originating from neural crest derived cells. Additionally, detected signatures also uncover genomic heterogeneity in individual cancer types, for instance in brain lower-grade glioma. Additional experiments with classification models underscore the efficacy of these signatures in representing various cancer types and their potential utility in clinical diagnosis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.26.656180v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrating genomics, machine learning, and computer vision to understand growth traits in selectively bred snapper (Chrysophrys auratus)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Blommaert, J., Bayer, P. E., Ashton, D. T., Samuels, G., Jesson, L., Wellenreuther, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify genetic variants associated with growth-related traits in snapper and demonstrate the utility of integrating computer vision-based phenotyping with GWAS and ML for trait prediction in aquaculture.</p><p style="font-size: 16px; text-align: justify;">Background: Understanding the genetic basis of growth-related traits is essential for optimising selective breeding programmes in aquaculture species. In this study, we analysed phenotypic and genomic data from a selectively bred population of snapper (Chrysophrys auratus) to identify genetic variants associated with key growth traits. We used a high-throughput, image-based phenotyping pipeline to extract 13 measurements rapidly and with minimal impact on the fish. These phenotypic measurements, together with manually measured weight and fork length, were analysed for correlations and principal component structure. Additionally, heritabilities were estimated for each trait. Then, genome-wide association studies (GWAS) were performed to identify growth-associated SNPs. To trial genomic prediction, we implemented machine learning (ML) models in XGBoost trained on SNP genotypes, with relatedness-based clustering used to minimise data leakage. Results: Via GWAS, we identifying 24 SNPs significantly associated with growth traits, with several mapping to genes involved in metabolic and developmental pathways. Despite the high-dimensionality of these data, the ML approach still achieved moderate levels of predictability. The top ML growth SNPs showed some congruency with the GWAS growth SNPs, and 75 % of the GWAS SNPs were used by the ML model to predict weight. Functional annotation identified putative gene-level effects, providing insights into potential biological mechanisms underlying growth variation. Conclusions: Our findings contribute to the development of genomic selection tools for snapper breeding and highlight the utility of integrating computer vision-based phenotyping with GWAS and ML for trait prediction in aquaculture species.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.29.656727v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MIC-Drop-seq: Scalable single-cell phenotyping of mutant vertebrate embryos</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Carey, C. M., Parvez, S., Brandt, Z. J., Bisgrove, B. W., Yates, C. J., Peterson, R. T., Gagnon, J. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We combined high-throughput gene disruption in zebrafish embryos via Multiplexed Intermixed CRISPR Droplets with phenotyping by multiplexed scRNAseq (MIC-Drop-seq).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Advances in genome engineering and single-cell RNA sequencing (scRNAseq) have revolutionized the ability to precisely map gene functions, yet scaling these techniques for large-scale genetic screens in animals remains challenging. We combined high-throughput gene disruption in zebrafish embryos via Multiplexed Intermixed CRISPR Droplets with phenotyping by multiplexed scRNAseq (MIC-Drop-seq). In one MIC-Drop-seq experiment, we intermixed and injected droplets targeting 50 transcriptional regulators into 1,000 zebrafish embryos, followed by pooled scRNAseq. Tissue-specific gene expression and cell abundance analysis of demultiplexed mutant cells recapitulated many known phenotypes, while also uncovering novel functions in brain and mesoderm development. We observed pervasive cell-extrinsic effects among these phenotypes, highlighting how whole-embryo sequencing captures complex developmental interactions. Thus, MIC-Drop-seq provides a powerful and scalable platform for mapping gene functions in vertebrate development with cellular resolution.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.27.656468v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A genomic catalog of Earth's bacterial and archaeal symbionts</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Villada, J. C., Vasquez, Y., Szabo, G., Walker, E. W., Gutierrez, M. R., Qin, S., Varghese, N., Eloe-Fadrosh, E. A., Kyrpides, N., SymGs data consortium,, Visel, A., Woyke, T., Schulz, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new model of microbial symbiosis that predicts the presence of 15-23% of uncultivated microbes that engage in symbiotic relationships with other organisms .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Microbial symbiosis drives the functional and phylogenomic diversification of life on Earth, yet remains underexplored due to culturing challenges. This study employed machine learning (ML) to predict symbiotic lifestyles in hundreds of thousands of microbial genomes from diverse environmental metagenome samples and reference genomes. Predictions were performed using symclatron, a novel ML framework developed here to identify genomic signatures of symbionts. Predictions were deposited in a novel Symbiont Genomes catalog (SymGs). The results indicated that 15-23% of uncultivated microbes likely engage in symbiotic relationships with other organisms and are present in half of all known bacterial and archaeal phyla. We also identified genomic signatures of symbiotic lifestyles, including the loss of certain metabolic functions and the differential presence of others that may enable host-dependent living. The symclatron software and the SymGs catalog represent valuable resources for studying symbioses, facilitating future mechanistic investigations of host-microbe associations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.29.656868v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Map functional insulators with cross-platform Hi-C meta-analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xu, W., Cui, J., Zhang, S., LU, L., Liu, X., Li, Y., Jin, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that functional insulators can form new loops upon CTCF depletion, but not TAD boundaries, but only at several hundred loci.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> CTCF forms the boundaries of topologically associating domains (TADs) but their insulating function in transcription is controversial. Here we report that functional insulators (FINs) shall be determined not only by their ability to form static loops or boundaries, but also by the dynamic context if their flanking sequences form new loops upon boundary loss. We performed a meta-analysis of nine independent Hi-C and micro-C datasets upon acute CTCF- or cohesin protein-depletion and found that only at several hundred loci, newly gained enhancers-promoters (E-P) loops can be reproducibly observed upon CTCF depletion. The new E-P loops are cohesin-dependent and associated with recurrent gene activation. These findings allow us to map FINs and define their direct target genes genome-wide. FINs are mostly present in euchromatin, do not overlap with TAD boundaries, and can be perturbed by Wapl-depletion or CTCF-displacement. We therefore conclude that FINs, but not TAD boundaries, are bona fide insulators.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.26.656183v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SNP calling, haplotype phasing and allele-specific analysis with long RNA-seq reads</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Huang, N., Human Pangenome Reference Consortium,, Li, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce longcallR for joint SNP calling, haplotype phasing, and allele-specific analysis, which achieves high accuracy on benchmark datasets.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Long-read RNA sequencing (lrRNA-seq) is a powerful technology to link transcript structures to genetic variants but such analysis is not often performed due to the lack of end-user tools. Here, we introduce longcallR for joint SNP calling, haplotype phasing, and allele-specific analysis, which achieves high accuracy on benchmark datasets. Applied to 202 human samples, longcallR identified 88 significant allele-specific splicing events per sample on average. 46% of them involved unannotated junctions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.26.656191v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">From Hype to Health Check: Critical Evaluation of Drug Response Prediction Models with DrEval</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bernett, J., Iversen, P., Picciani, M., Wilhelm, M., Baum, K., List, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present DrEval, a pipeline for unbiased and biologically meaningful evaluation of cancer drug response models. We find that deep learning models barely outperform a naive model predicting the mean drug and cell line effects, while no complex model significantly outperforms properly tuned tree-based ensemble baselines.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: Large-scale drug sensitivity screens have enabled training drug response prediction models based on cancer cell line omics profiles to facilitate personalized medicine. While model performances reported in the literature appear promising, no successful translation to the clinic has been reported. Results: We identify six primary obstacles to this: non-reproducible models, data leakage leading to poor generalization, pseudoreplication, biased evaluation metrics, missing ablation studies, and inconsistent viability data reducing comparability across models. Together, these issues lead to overly optimistic performance estimates of state-of-the-art models and make it challenging to track progress in the field. To address this, we present DrEval, a pipeline for unbiased and biologically meaningful evaluation of cancer drug response models. It includes baseline and literature models with consistent hyperparameter tuning, statistically sound evaluations, and cross-study benchmarks. DrEval enables ablation studies and publication-ready visualizations. It allows researchers to focus on model development without implementing their own evaluation protocol. We find that deep learning models barely outperform a naive model predicting the mean drug and cell line effects, while no complex model significantly outperforms properly tuned tree-based ensemble baselines in relevant settings. We advocate making our pipeline a standard benchmark for cancer drug response prediction, ensuring a clinically relevant and robust assessment. Availability and implementation: DrEval consists of a Python package, available on PyPI (drevalpy) and GitHub (github.com/daisybio/drevalpy), and an accompanying nf core pipeline (github.com/nf-core/drugresponseeval). All data is available on Zenodo (DOI: 10.5281/zenodo.12633909), preprocessing scripts on github.com/daisybio/preprocess_drp_data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.26.655288v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DoggifAI: a transformer based approach for antibodycaninisation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Grabarczyk, D., Kocikowski, M., Parys, M., Houston, D., Hupp, T., Alfaro, J. A., Cohen, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present DoggifAI, a transformer model that translates non-canine antibody sequences into canine ones by generating species-appropriate framework regions (FRs) based on desired CDR.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Antibody translation across species offers a compelling strategy to extend the vast and expensive investments in human therapeutic antibodies to veterinary oncology, with applications in both veterinary medicine and comparative oncology. While precise, low-immunogenic treatments are essential for canine cancer care, traditional species conversion methods rely on ad hoc bioinformatics modifications. These methods often implicitly decouple the framework (FR) and complementarity-determining regions (CDRs), ignoring how structural changes in FRs can affect the conformation and function of CDRs. This can compromise binding specificity and require costly high-throughput in vitro screening. To address this, we present DoggifAI, a transformer model that translates non-canine antibody sequences into canine ones by generating species-appropriate framework regions (FRs) based on desired CDRs. This allows the model to better preserve structural compatibility between FRs and CDRs. The model is pretrained in a T5-style text-to-text denoising task on a large multispecies antibody dataset, which allows further finetuning on a much smaller species-specific dataset. DoggifAI generates highly canine-like antibodies and shows promising results in preserving binding specificity. To support further progress in this field, we also release a curated dataset of over 430,000 unique canine antibody chain sequences, significantly expanding the public sequence repertoire.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.28.656573v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">mt.surv: Multi-Threshold Survival Analysis for Associating Continuous Predictor Variables with Time-to-Event Outcomes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Loncar, A. J., Hoyd, R., Liu, Y., Dravillas, C., Dhrubo, D., Spakowicz, D. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a framework for multi-threshold survival analyses for time-to-event models, where a continuous predictor variable is stratified into two groups at an arbitrary number of places, defined by the percentile of the distribution, and evaluate the strength and consistency of association with the event of interest.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Time-to-event models are a useful and common approach to infer the importance of biological variables. Most often, the predictor variable is binarized and associated with censored time to death, a so-called Kaplan-Meier curve. However, the threshold to binarize the continuous variable is often arbitrary. We sought a rigorous way to define thresholds and evaluate the strength and consistency of association with the event of interest. We present {mt.surv}, an R package for multi-threshold survival analyses. The primary function performs a time-to-event analysis, where a continuous predictor variable is stratified into two groups at an arbitrary number of places, defined by the percentile of the distribution. The result can be visualized with a function that creates a custom line plot with the -log of a log-likelihood p-value against the threshold percentile. A third function operates on this plot and calculates the area above a significance threshold, creating a scalar that can be used to rank biological variables for their association with the event of interest. This framework can be broadly applied to any continuous predictor variable including, but not limited to, gene expression and microbial abundances. Several helper functions are included for structuring input data and job submission in a cluster framework. We found that this method has value in discovery-type analyses where one lacks prior information about appropriate stratification thresholds. However, we found additional biological in-sight is possible, indeed, quite common, as many variables show different associations at differ-ent stratification thresholds.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.29.656703v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Making deep mutational scanning accessible: a cost-efficient approach to construct barcoded libraries for genes of any length</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jann, J., Gagnon-Arsenault, I., Pageau, A., Dube, A. K., Fijarczyk, A., Durand, R., Landry, C. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed an efficient, affordable and scalable library construction approach that relies on low-cost DNA synthesis and standard cloning technologies, which will increase accessibility to systematic mutational studies and help advance the field of protein science.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent developments in DNA synthesis and sequencing have allowed the construction of comprehensive gene variant libraries and their functional analysis. Achieving high-replication and thorough mutation characterization remains technically and financially challenging for long genes. Here, we developed an efficient, affordable and scalable library construction approach that relies on low-cost DNA synthesis and standard cloning technologies, which will increase accessibility to systematic mutational studies and help advance the field of protein science.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.29.656836v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AlphaFold modeling of polyubiquitin complexes and covalently linked proteins</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fabian, B., Stuke, J. F. M., Heinz, M., Hummer, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce correlated cysteine mutations to induce linkage-specific proximity of Ubs in complex with interacting proteins, and introduce short covalent linker groups in AlphaFold 3 calculations that mimic the isopeptide bonds between linked lysines.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cells use the covalent attachment of Ubiquitin (Ub) chains to mark proteins for degradation, alter their cellular localization or drive their association. Protein fate is encoded in the distinct poly-Ub linkages, exploiting the vast combinatorial space of linear and branched Ub modifications. AlphaFold has emerged as a powerful tool to predict the structure of protein-protein complexes. However, standard AlphaFold does not consider linkages between individual protein chains, limiting its applicability to Ub chains. The near complete conservation of the ubiquitin sequence and the large number of binding partners suppresses coevolutionary signals, further challenging the prediction of poly-Ub complex structures. We address this challenge, first, by introducing correlated cysteine mutations to induce linkage-specific proximity of Ubs in complex with interacting proteins. Second, we introduce short covalent linker groups in AlphaFold 3 calculations that mimic the isopeptide bonds between linked lysines and Ub C-terminal carboxylates. These two approaches enable the robust structural modeling of complexes involving poly-Ub chains with AlphaFold. The linker approach is general and can be used for other covalent inter-chain connections and to enforce distance restraints for integrative structural modeling.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.27.656350v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Hierarchical prediction and perturbation of chromatin organization reveal how loop domains mediate higher-order architectures</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wei, J., Xue, Y., Gao, Y. Q.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce HiCGen, a hierarchical and cell-type-specific generator based on Swin-transformer architecture that predicts genome organization across different spatial scales, including insulation degree, compartmentalization, and contact intensity over genomic distances exceeding 10 Mb.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The genome is folded within the dense cell nucleus in a hierarchical manner, resulting in complex interactions between distinct folding strategies at various length scales. To elucidate how short-range loop domains regulate higher-order structures of the chromatin, such as topologically associating domains (TADs) and compartments, we introduce HiCGen, a hierarchical and cell-type-specific generator based on Swin-transformer architecture. HiCGen predicts genome organization across different spatial scales utilizing DNA sequence and genomic features as inputs. The model enables in silico screening through genetic or epigenetic perturbations on genome architecture, with resolution down to 1 kb. Our analysis reveals unexpected linear correlations between loop properties and genome organization at various levels, including insulation degree, compartmentalization, and contact intensity over genomic distances exceeding 10 Mb. Regional or global perturbation conducted by HiCGen provides biological implications for such cross-scale correlations and their genome-function dependence. Notably, perturbation analysis of the human genome in sigmoid colon tissue demonstrates that modest activation of carcinogenesis-associated enhancers is sufficient to hijack nearby promoter, reshape TAD boundaries, and even flip compartment at mega-base scale.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.25.656045v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Circos Plots for Genome Level Interpretation of Genomic Prediction Models</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tomura, S., Powell, O. M., Wilkinson, M. J., Cooper, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A deeper understanding of genomic prediction models with enhanced interpretability can reveal several critical findings at the genome level from the inferred genetic architecture, providing insights into the improvement of genomic predicting for crop breeding programs.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Investigations of genomic prediction models have focused primarily on the level of metrics summarising the accuracy and error of predicted phenotypes. Analysis of the genomic regions contributing to the genomic prediction models can deepen our understanding of the trait genetic architecture. Here, we develop circos plots to visualise how different genomic prediction models quantify marker effects. The genetic architecture was inferred using interpretable genomic prediction models to characterise each method at the genome level. Application is demonstrated for the trait days to anthesis (DTA) in the TeoNAM dataset. The results indicate that genomic prediction models can capture different views of trait genetic architecture, even when their overall profiles of prediction accuracy are similar. Combinations of diverse views of the genetic architecture for the DTA trait in the TeoNAM study might explain the improved prediction performance achieved by ensembles of prediction models, aligned with the implication of the Diversity Prediction Theorem. In addition to identifying well-known genomic regions contributing to the genetic architecture of DTA in maize, the ensemble of genomic prediction models highlighted several new genomic regions that have not been previously identified for DTA. Finally, different views of trait genetic architecture were observed across populations, within the TeoNAM design, highlighting challenges for between-population genomic prediction. A deeper understanding of genomic prediction models with enhanced interpretability can reveal several critical findings at the genome level from the inferred genetic architecture, providing insights into the improvement of genomic prediction for crop breeding programs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.25.656055v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comprehensive biobanking strategy with clinical impact at the European Cancer Moonshot Lund Center.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Henriett Oskolas, Fábio C N Nogueira, Gilberto B Domont, Kun-Hsing Yu, Yevgeniy R Semenov, Peter Sorger, Erik Steinfelder, Les Corps, Lesley Schulz, Elisabet Wieslander, David Fenyö, Sarolta Kárpáti, Péter Holló, Lajos V Kemény, Balazs Döme, Zsolt Megyesfalvi, Krzysztof Pawłowski, Toshihide Nishimura, HoJeong Kwon, Sergio Encarnación-Guevara, A Marcell Szasz, Zoltán Veréb, Rolland Gyulai, István Balázs Németh, Roger Appelqvist, Melinda Rezeli, Bo Baldetorp, Peter Horvatovich, Johan Malmström, Indira Pla, Aniel Sanchez, Beatrice Knudsen, András Kiss, Johan Malm, György Marko-Varga, Jeovanis Gil</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> This white paper presents a comprehensive biobanking framework developed at the European Cancer Moonshot Lund Center that merges rigorous sample handling, advanced automation, and multi-omic analyses to accelerate precision oncology.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> This white paper presents a comprehensive biobanking framework developed at the European Cancer Moonshot Lund Center that merges rigorous sample handling, advanced automation, and multi-omic analyses to accelerate precision oncology. Tumor and blood-based workflows, supported by automated fractionation systems and standardized protocols, ensure the collection of high-quality biospecimens suitable for proteomic, genomic, and metabolic studies. A robust informatics infrastructure, integrating LIMS, barcoding, and REDCap, supports end-to-end traceability and realtime data synchronization, thereby enriching each sample with critical clinical metadata. Proteogenomic integration lies at the core of this initiative, uncovering tumor- and blood-based molecular profiles that inform cancer heterogeneity, metastasis, and therapeutic resistance. Machine learning and AI-driven models further enhance these datasets by stratifying patient populations, predicting therapeutic responses, and expediting the discovery of actionable targets and companion biomarkers. This synergy between technology, automation, and high-dimensional data analytics enables individualized treatment strategies in melanoma, lung, and other cancer types. Aligned with international programs such as the Cancer Moonshot and the ICPC, the Lund Center's approach fosters open collaboration and data sharing on a global scale. This scalable, patient-centric biobanking paradigm provides an adaptable model for institutions aiming to unify clinical, molecular, and computational resources for transformative cancer research.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40246065/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

