<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Saturday, 05 July 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">iACP-DPNet: a dual-pooling causal dilated convolutional network for interpretable anticancer peptide identification.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zimeng Zhang, Xin Wang, Wenhui Shang</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel deep learning predictive model for anticancer peptide discovery, which significantly outperforms all existing state-of-the-art methods in comparative analyses.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Anticancer peptides (ACPs) are acknowledged for their potential in cancer therapy, attributed to their safety, low side effects, and high target specificity. However, the discovery of ACPs is slowed by the high cost and labor-intensive nature of experimental validation, resulting in a limited number of confirmed ACPs. Although various computational methods have been proposed, existing models commonly suffer from three critical limitations: reliance on small-scale datasets, lack of interpretable feature learning mechanisms, and insufficient generalization capability. To address these challenges, this study constructs a larger and more diverse dataset by consolidating data from existing literature and databases, and proposes a novel deep learning predictive model named iACP-DPNet. The model utilizes the protein language model ProtBert with positional encoding to convert protein sequences into feature vectors, then applies a two-step feature selection process via LightGBM and MIC. The selected features undergo processing by a causal dilated convolution network. A dual-pooling mechanism is designed to enhance the model's ability to synergistically model local critical residues and global sequence contexts, integrating parallel GlobalAveragePooling and attention pooling layers. Compared to traditional single-pooling models (e.g., ACP‑MHCNN), this architecture significantly improves feature extraction capability. To understand the model's decision-making process, we employ t-SNE for visualizing key steps, ISM for interpreting sequence regions, and SHAP analysis for evaluating feature importance. These approaches significantly improve the model's interpretability. The model exhibits outstanding performances on the novel dataset, as evidenced by rigorous tenfold cross-validation. Achieving remarkable metrics-including Sp of 96.1%, Sn of 92.91%, Acc of 94.5% and MCC of 89.05%, it significantly outperforms all existing state-of-the-art methods in comparative analyses. Furthermore, to assess its generalizability, we evaluated iACP-DPNet on an additional dataset, where it outperformed other current models. In conclusion, the iACP-DPNet exhibits exceptional performance and generalizability, showcasing its advanced design and effectiveness in ACPs prediction. This research provides a robust and interpretable framework for advancing research in anticancer peptide discovery. HIGHLIGHTS: • We have established a larger and more diverse dataset for ACPs prediction, addressing the limitations of existing datasets and providing a robust foundation for model training and evaluation. • The implementation of a dual-pooling layer mechanism (GlobalAveragePooling and attention pooling) bolsters the model's capacity to learn diverse features, ultimately enhancing its prediction efficiency. • We employed t-SNE visualization and ISM-based interpretability analysis to provide insights into the model's decision-making process, highlighting key regions and amino acids critical for ACPs functionality. • The iACP-DPNet model demonstrates strong generalizability across diverse datasets, making it a reliable tool for ACPs prediction and potentially other peptide-related tasks.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40613943/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Breeding perspectives on tackling trait genome-to-phenome (G2P) dimensionality using ensemble-based genomic prediction.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mark Cooper, Shunichiro Tomura, Melanie J Wilkinson, Owen Powell, Carlos D Messina</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We explore the applications of G2P models for prediction-based breeding and the role of the trait genetic architecture on selection trajectories.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Trait Genome-to-Phenome (G2P) dimensionality and "breeding context" combine to influence the realised prediction skill of different whole genome prediction (WGP) methods. Theory and empirical evidence both suggest there is likely to be "No Free Lunch" for prediction-based breeding. Ensembles of diverse sets of G2P models provide a framework to expose and investigate the high G2P dimensionality of trait genetic architecture for WGP applications. Artificial Intelligence and Machine Learning (AI-ML) prediction algorithms contribute novel trait G2P model diversity to ensemble-based WGP. Prediction-based breeding leveraging ensembles of G2P models creates new opportunities to identify and design novel paths for genetic gain. Improving our understanding of trait genetic architecture is motivated by creating new opportunities to enhance breeding methodology, create new selection trajectories for crop improvement, and accelerate rates of genetic gain. With access to high-throughput sequencing, phenotyping and envirotyping technologies we can model the complex multidimensional relationships between sequence variation and trait phenotypic variation that are under the influences of selection. Using the framework of the diversity prediction theorem, we consider applications of ensembles of diverse trait genome-to-phenome (G2P) models. Crop growth models (CGM) are an example of a hierarchical framework for studying the influences of quantitative trait loci (QTL) within trait networks and their interactions with different environments to determine yield. Hybrid CGM-G2P models combine elements of CGMs, to understand how trait networks influence crop yield performance, with trait G2P models, to understand influences of trait genetic architecture on selection trajectories. We discuss hybrid CGM-G2P models and their potential applications to enhance ensemble-based prediction. Multi-environment trials conducted across breeding cycles can be designed to include contrasting environments to expose the different CGM-G2P dimensions of the trait by environment interactions that are influential on selection trajectories. Artificial intelligence and machine learning (AI-ML) algorithms can be applied as components of ensembles to improve gene discovery and quantification of allele effects for traits to enhance G2P prediction applications. We use the trait flowering time in the maize TeoNAM experiment to illustrate and motivate further investigations of how to leverage ensembles of G2P models for prediction-based breeding.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40613890/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Accurate Free Energy Calculation via Multiscale Simulations Driven by Hybrid Machine Learning and Molecular Mechanics Potentials.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xujian Wang, Xiongwu Wu, Bernard R Brooks, Junmei Wang</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a hybrid machine learning/molecular mechanics interface integrated into the AMBER molecular simulation package. We proposed a novel ML/MM-compatible thermodynamic integration framework that adequately addressed the challenge of applying MLIPs in TI calculations due to its indivisible nature of energy and force.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> This work develops a hybrid machine learning/molecular mechanics (ML/MM) interface integrated into the AMBER molecular simulation package. The resulting platform is highly versatile, accommodating several advanced machine learning interatomic potential (MLIP) models while providing stable simulation capabilities and supporting high-performance computations. Building upon this robust foundation, we developed new computational protocols to enable pathway-based and end point-based free energy calculation methods utilizing ML/MM hybrid potential. In particular, we proposed an ML/MM-compatible thermodynamic integration (TI) framework that adequately addressed the challenge of applying MLIPs in TI calculations due to its indivisible nature of energy and force. Our results demonstrated that the hydration free energies calculated using this framework achieved an accuracy of 1.0 kcal/mol, outperforming the traditional approaches. Moreover, ML/MM enables more precise sampling of conformational ensembles for improved end point-based free energy calculations. Overall, our efficient, stable, and highly compatible interface not only broadens the application of MLIPs in multiscale simulations but also enhances the accuracy of free energy calculations from multiple aspects. By introducing a novel ML/MM-compatible thermodynamic integration framework, we offered a novel foundation for combining advanced multiscale simulation methodologies with highly accurate free energy calculation techniques, thereby opening new avenues and providing a robust theoretical framework for future developments in this field.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40613671/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The effect of environmental enrichment on immune cell DNA methylation profiles depends on the parity of sows.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lopes, M. M., Monteiro Moreira, G. C., Chaulot-Talmon, A., Frambourg, A., Costes, V., Demars, J., MERLOT, E., JAMMES, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study on the epigenetic impact of parity class on individual immunity in multiparous pregnant sows .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The aim of this study was to identify epigenetic markers that reflected positive affective states in multiparous pregnant sows. The animals were housed during gestation in either a conventional (C) environment (2.4 square meter per sow), featuring a concrete slatted floor and minimal enrichment, or in an enriched (E) environment (3.5 m square meter per sow) with deep straw bedding. Peripheral blood mononuclear cells (PBMCs) were isolated at day 98 of gestation (G98) and day 12 of lactation (L12) for genomic DNA extraction and reduced representation bisulfite sequencing (RRBS). Significant effects of individual class of parity (low-parity (LP) versus high-parity (HP)) were observed with the identification of 1,358 and 680 differentially methylated cytosines (DMCs) at G98 and L12, respectively. Interestingly, some of them are organized into differentially methylated regions (DMRs). Some DMRs colocalized with or near a gene and displayed a continuous methylation distribution (44 at G98 and 15 at L12 including 4 in common): 5 targeted genes are related to epigenetic regulation (DNMT3A, KDM8, HDAC4, SIRT2 and U2) and 12 to immune function (CD2, CD5, CAMK4, SECTM1, URODL1, CMIP, SEC14L1, SKI, TNFRSF1B, CCND3, SGK1, PACS1). These results suggested a true epigenetic impact of parity class on individual immunity. Considering the two parity classes separately, a minor effect of environmental enrichment was observed (at G98: 60 and 42 DMCs; at L12: 35 and 81 DMCs, in the LP and HP groups, respectively). Remarkably, some DMC-associated genes had previously been linked to affective states in humans. In conclusion, unexpected DNA methylation changes associated with parity class were identified, suggesting a genome adaptation during reproductive life and modifying the response to housing conditions. Furthermore, specific CpG sites emerged as potential biomarkers of positive affective states in pigs. https://www.ebi.ac.uk/ena/browser/home - PRJEB90105 / ERP173119 https://doi.org/10.57745/0GWNEV - Recherche Data Gouv. and referenced under MERLOT, Elodie; JAMMES, Helene, 2025, "BEPPI project - data, tables and figures on sow peripheral white blood cell epigenome".</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.01.662510v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Sequence Analysis of P4-ATPases Reveals the Structural Determinants for the Stable Monomeric P4B-ATPase Phospholipid Transporters.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sai, K. V., Rajan, S. A. S., Lee, J.-Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A structure-based sequence analysis approach for the first time identified unique features of the P4-ATPase family of phospholipid flippases, as well as previously unidentified conserved properties of the group that supersedes substrate specificities.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The P4-ATPase family of phospholipid flippases plays a critical role in the maintenance of membrane asymmetry and consequently, various roles in cellular protein traffic and eukaryotic homeostasis. Currently, several structures of these (usually heterodimeric) phospholipid flippases have been resolved, along with extensive biochemical characterization of the substrate transport properties. However, an essential subfamily of monomeric phospholipid flippases, the P4B-ATPases, remains to be characterized in depth. While these P4B-ATPases appear to have similar lipid transport properties to their heterodimeric counterparts, the P4A-ATPases, the basis of their existence as monomers is currently unknown. In addition, the unique features of this group have yet to be comprehensively analyzed since the resolution of one P4B-ATPase member. In this study, we investigated the divergence of P4B-ATPases from other P-type ATPases using a structure-based sequence analysis approach. This led to identification of features unique to P4B-ATPases, as well as previously unidentified conserved properties of P4-ATPases. The results of this study provide a basis for further studies on P4-ATPases to characterize conserved properties of this group that supersedes substrate specificities.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.01.662597v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An Intrinsic-hoc Framework for Heterogeneous Cellular Senescence Elucidation Using Deep Graph Representation Learning and Experimental Validation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ma, A., Cheng, H., Vanegas, N. D. P., Ghobashi, A., Chen, H., Rodriguez, J., Rosas, L., Wang, C., Shao, J., Jiang, Y., Wang, X., Rahman, I., Lugo-Martinez, J., Li, D., Pryhuber, G. S., Bar-Joseph, Z., Eickelberg, O., Konigshoff, M., Chung, D., Chen, R., Finkel, T., Mora, A., Rojas, M., Ma, Q.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present DeepSAS, an intrinsic-hoc framework for elucidating the heterogeneity encoded in senescent cells and their associated genes from single-cell RNA-seq data using deep graph representation learning.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Characterizing senescent cells and identifying corresponding senescence-associated genes within complex tissues is critical for our understanding of aging and age-related diseases. We present DeepSAS, an intrinsic-hoc framework for elucidating the heterogeneity encoded in senescent cells and their associated genes from single-cell RNA-seq data using deep graph representation learning. Applied to both healthy eye cell atlas and in-house idiopathic pulmonary fibrosis datasets with Xenium spatial transcriptomics validation, DeepSAS reveals robust and biologically grounded senotypes and demonstrates superior benchmarking performance compared with existing methods.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.01.662364v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Unsupervised Variant Clustering Identifies Genetic Subtypes of Disease</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mizikovsky, D., Wray, N. R., Shah, S., Palpant, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A phenotype-free, unsupervised, phenotype- free framework that analyses the genetic co-occurrence of variants across individuals to identify variant clusters underpinning disease heterogeneity.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Current approaches to identify disease subtypes rely on prior knowledge or external phenotypic references that limit their generalisability across traits. This study presents an unsupervised, phenotype-free framework that analyses the genetic co-occurrence of variants across individuals to identify variant clusters underpinning disease heterogeneity. The approach is developed and validated using simulated combinations of real phenotypes then applied to dissect disease subtypes of type 2 diabetes and asthma. The approach captures clinically established mechanistic profiles in type 2 diabetes and asthma without requiring any reference phenotype data and enables identification of plasma proteins for novel subtype-specific biomarkers and drug targets. Lastly, we demonstrate that variants with conflicting effects on disease-relevant traits are resolved into distinct clusters, identifying subtypes with opposing phenotypic profiles that are lost by the aggregated genetic risk. Co-occurrence-based clustering provides a scalable strategy for understanding disease heterogeneity across diverse populations by identifying biologically meaningful subtype structure from genotype data alone.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.01.662666v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Spatial joint profiling of DNA methylome and transcriptome in mammalian tissues</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lee, C. N., Fu, H., Cardilla, A., Zhou, W., Deng, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a method for whole genome spatial co-profiling of DNA methylation and transcriptome of the same tissue section at near single-cell resolution.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The spatial resolution of omics dynamics is fundamental to understanding tissue biology. Spatial profiling of DNA methylation, which is a canonical epigenetic mark extensively implicated in transcriptional regulation, remains an unmet demand. Here, we introduce a method for whole genome spatial co-profiling of DNA methylation and transcriptome of the same tissue section at near single-cell resolution. Applying this technology to mouse embryogenesis and postnatal brain resulted in rich DNA-RNA bimodal tissue maps. These maps revealed the spatial context of known methylation biology and its interplay with gene expression. The two modalities concordance and distinction in spatial patterns highlighted a synergistic molecular definition of cell identity in spatial programming of mammalian development and brain function. By integrating spatial maps of mouse embryos at two different developmental stages, we reconstructed the dynamics of both epigenome and transcriptome underlying mammalian embryogenesis, revealing details in sequence, cell type, and region-specific methylation-mediated transcriptional regulation. This method extends the scope of spatial omics to DNA cytosine methylation for a more comprehensive understanding of tissue biology over development and disease.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.01.662607v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">OmniCorr: An R-package for visualizing putative host-microbiota interactions using multi-omics data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gupta, S., Lai, W., Kobel, C. M., Aho, V. T. E., Vera-Ponce de Leon, A., La Rosa, S. L., Sandve, S. R., Pope, P. B., Hvidsten, T. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present OmniCorr, an R package designed to: (1) manage the complexity of omics data by clustering similar observations into modules, (2) visualize correlations of these modules across different omics layers, host-microbiota interfaces, and meta-data, and (3</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Holo-omics leverages omics datasets to explore the interactions between hosts and their associated microbiomes. Although the generation of omics data from matching host and microbiome samples is steadily increasing, there remains a scarcity of computational tools capable of integrating and visualizing this data to facilitate the interpretation and prediction of host-microbiota interactions. We present OmniCorr, an R package designed to: (1) manage the complexity of omics data by clustering similar observations into modules, (2) visualize correlations of these modules across different omics layers, host-microbiota interfaces, and meta-data, and (3) identify statistically significant associations indicative of putative host-microbiota interactions. OmniCorr's utility is demonstrated using datasets from two systems: (i) Atlantic salmon, integrating host transcriptomics with metagenomics and metatranscriptomics to explore dietary impacts, and (ii) cattle, combining host proteomics with metaproteomics and metabolomics to investigate methane emission variability.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.01.662509v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Combining AI structure prediction and integrative modelling for nanobody-antigen complexes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sanchez-Marin, M., Giulini, M., Bonvin, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose an ensemble docking pipeline that achieves high success rates starting from nanobody structural models predicted by AlphaFold2 and ImmuneBuilder and demonstrate the effectiveness of physics-based complex prediction of immune proteins.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Nanobodies exhibit antigen binding affinities of the same order as those of antibodies, which, along with their small size and unique structural characteristics, makes them well-suited for therapeutic and diagnostic applications. The lack of coevolutionary signals in nanobody-antigen complexes together with the broad complementary determining region 3 loop (CDR3) conformational space poses a challenge for predicting the 3D structure of those complexes with computational modelling and artificial intelligence-based methods. In this context, physics-based information-driven docking can provide an alternative solution. This study evaluates the state-of-the-art of machine learning-based methods for nanobody structure prediction and benchmarks various HADDOCK workflows to model their interaction with antigens using different input nanobody ensembles and information scenarios. We propose an ensemble docking pipeline that achieves high success rates starting from nanobody structural models predicted by AlphaFold2 and ImmuneBuilder. Our results highlight the effectiveness of physics-based complex prediction of immune proteins when accurate input structures and sufficient information to guide the modelling are available.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.01.662355v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">NANOME: A Nextflow pipeline for haplotype-aware allele-specific consensus DNA methylation detection by nanopore long-read sequencing</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liu, Y., Taha, H. B., Zhang, Q., Pan, Z., Chatzipantsiou, C., Wade, E., Slocum, T., Karuturi, L., Zhao, Y., Karmakar, S., Li, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a consensus DNAme predictive model using XGBoost, which integrates the output of Megalodon, Nanopolish, DeepSignal, and Deepsignal for analyzing data obtained using Oxford Nanopore Technologies (ONT).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Nanopore long-read sequencing has expanded the capacity of long-range, single-base, and single-molecule DNA-methylation (DNAme) detection and haplotype-aware allele-specific epigenetic phasing. Previously, we benchmarked and ranked the robustness of seven computational tools for DNAme detection using nanopore sequencing. The top performers were Megalodon, Nanopolish, DeepSignal and Guppy. However, these algorithms exhibit lower performance at regions with discordant non-singleton DNAme patterns compared to genome-wide regions. Furthermore, long-read sequencing analysis of mammalian genomes requires higher computational resources than next-generation sequencing. To address these issues, we developed a NANOpore Methylation (NANOME) a consensus DNAme predictive model using XGBoost, which integrates the output of Megalodon, Nanopolish, and Deepsignal for analyzing data obtained using Oxford Nanopore Technologies (ONT). NANOME enhanced DNAme detection precision (mean square error) at single-base resolution by 11% and improved accuracy (F1-score) at single-molecule resolution by 2.4% for human B-lymphocyte European cell lines (NA12878). The consensus model also detected ~200,000 more CpGs than all three tools. Combing variant calling and long-read phasing, NANOME can detect haplotype-aware allele-specific DNAme in known imprinting controls in resolved and previously unresolved regions. We conducted haplotype-aware methylation detection on the T2T genome for dataset NA12878, revealing significant variations in differentially methylated region (DMR) density between gap and non-gap regions. Overall, NANOME represents a significant step forward in DNAme detection and long-range epigenetic phasing, offering a robust and accessible tool for researchers studying the epigenome.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.29.662079v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Uncertainty-Aware Discrete Diffusion Improves Protein Design</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mahbub, S., Feinauer, C., Ellington, C. N., Song, L., Xing, E. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose an uncertainty-aware discrete denoising diffusion model that employs a prior-posterior signaling mechanism to dynamically guide the denoise process.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein inverse folding involves generating amino acid sequences that adopt a specified 3D structure---a key challenge in structural biology and molecular engineering. While discrete diffusion models have demonstrated strong performance, existing methods often apply uniform denoising across residues, overlooking position-specific uncertainty. We propose an uncertainty-aware discrete denoising diffusion model that employs a prior-posterior signaling mechanism to dynamically guide the denoising process. Our approach further integrates learned priors from a pretrained protein large language model and a structure encoder within a modular framework, jointly optimized through multi-objective training. Across multiple benchmarks, our method achieves substantial improvements over state-of-the-art baselines, offering a principled framework for structure-conditioned sequence generation in proteins and beyond.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.30.662407v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Rapid and Reproducible Multimodal Biological Foundation Model Development with AIDO.ModelGenerator</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ellington, C. N., Li, D., Zou, S., Cole, E., Sun, N., Addagudi, S., Song, L., Xing, E. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present AIDO.ModelGenerator, an open-source toolkit for multimodal FM research in the life sciences, which achieves a new SOTA by combining DNA and RNA FMs that outperforms unimodal baselines by over 10%.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Foundation models (FMs) for DNA, RNA, proteins, cells, and tissues have begun to close long-standing performance gaps in biological prediction tasks, yet each modality is usually studied in isolation. Bridging them requires software that can ingest heterogeneous data, apply large pretrained backbones from various sources, and perform multimodal benchmarking studies at scale. We present AIDO.ModelGenerator, an open-source toolkit that turns these needs into declarative experiment recipes through a structured experimental framework. AIDO.ModelGenerator provides (i) 300+ datasets covering DNA, RNA, protein, cell, spatial, and multimodal data types; (ii) 30+ pretrained FMs ranging from 3M to 16B parameters; (iii) 10+ plug-and-play use-cases covering inference, adaptation, prediction, generation, and zero-shot evaluation; and (iv) YAML-driven experiment recipes that enable exact reproducibility. On a sequence-to-expression prediction task, AIDO.ModelGenerator systematically builds and tests unimodal and multimodal models, achieving a new SOTA by combining DNA and RNA FMs that outperforms unimodal baselines by over 10%. In a Crohn's disease case-study, the framework's simulated knockout protocol ranks the clinically implicated target SOX4 6,000 positions higher than differential-expression baselines, illustrating its utility for therapeutic target discovery. We release code, tutorials, checkpoints, datasets, and API reference to accelerate multimodal FM research in the life sciences.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.30.662437v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">STRAND: Structure Refinement of RNA-Protein Complexes via Diffusion</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Al-zeqri, M., Franke, J. K. H., Runge, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We develop a diffusion-based model for monomeric RNA-protein complex refinement that builds upon the success of DiffDock-PP in protein-protein docking.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> RNA-protein interactions play crucial roles in cellular processes, from gene regulation to viral replication. While recent advances in structure prediction have revolutionized our ability to model macromolecular complexes, achieving accurate predictions of RNA-protein binding poses remains challenging. In this work, we present STRAND, a diffusion-based model for monomeric RNA-protein complex refinement that builds upon the success of DiffDock-PP in protein-protein docking. Unlike traditional docking, we develop STRAND as a modular extension to existing RNA-Protein complex prediction tools to improve their backbone predictions. We study the effect of different transformations by training models to learn either translation, rotation, torsion, or combinations of these during the diffusion process and initialize the backward process with a complex prediction at test time. Our experiments with AlphaFold 3 and ProRNA3D-single reveal that STRAND can improve the backbones of a large fraction of RNA-protein complex predictions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.30.662415v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MarkerMatch: A Proximity-Based Probe-Matching Algorithm for Joint Analysis of Copy-Number Variants from Different Genotyping Arrays</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ivankovic, F., Yu, D., Shen, J., Zhan, L., Niarchou, M., Kaylor, A., Domenech, L., Miller-Fleming, T. W., Porras, L. M., Giusti-Rodriguez, P., Ophoff, R. A., Scharf, J. M., Mathews, C. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> MarkerMatch, a proximity-based algorithm that matches probes across different genotyping microarrays to maximize the number of probes considered in the CNV calling algorithm, thereby increasing the resolution and sensitivity while preserving precision.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Copy-number variants (CNVs) are a form of genetic structural variation with increasing importance in complex human disorders. Both DNA sequencing and microarray data can be used to call CNVs, which can be used in association tests, such as association between CNV number and disease status. Unlike genotypes, CNV detection in microarrays requires the use of observed intensity signals at each probe, which limits the imputability for analyses that span multiple array types. Thus far, a consensus set of probes (the intersection encompassing the probes that occur in common on all arrays) has been used to circumvent the problem of differing array-specific sensitivities. This has, however, led to excessive reduction in overall sensitivity of CNV calls as arrays can have an undesirably low overlap of probe sets. To overcome this limitation, we developed MarkerMatch, a proximity-based algorithm that matches probes across different genotyping microarrays to maximize the number of probes considered in the CNV calling algorithm, thereby increasing the resolution and sensitivity while preserving precision. By analyzing CNV calls from 4,906 individuals genotyped across three different arrays (Global Screening Array, Omni2.5 array, and Omni Express Exome array), we show that the MarkerMatch approach improves sensitivity by increasing the density of probes available for CNV calling while maintaining precision or improving it relative to the current practice (e.g., use of consensus probes only). We further demonstrate that MarkerMatch exceeds the output from current practice in terms of F1 score, Fowlkes-Mallows index, and Jaccard index. We also optimize MarkerMatch parameters, DMAX and Method, and find an optimal DMAX setting at 10kb, with no clear optimal candidate based on Method, indicating that parameters for this metric should be determined on a use case basis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.30.662249v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CENTRA: Knowledge-Based Gene Contexuality Graphs Reveal Functional Master Regulators by Centrality and Fractality</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hause, F., Wedler, A., Keil, R., Schian, L., Glass, M., Guenther, W., Sorokin, O., Hoehenwarter, W., Sinz, A., Huettelmaier, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a framework that models gene contextuality through topic-specific gene co-occurrence networks derived from curated gene sets and associated literature.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Deciphering gene function via context-aware approaches is limited by various means. Especially static gene sets used in enrichment analyses and the lack of single-gene resolution in such analyses restrains the flexible association of genes with specific context. Here, we introduce CENTRA (Centrality-based Exploration of Network Topologies from Regulatory Assemblies), a framework that models gene contextuality through topic-specific gene co-occurrence networks derived from curated gene sets and associated literature. Using Latent Dirichlet Allocation on 12,045 abstracts linked to MSigDB C2 gene sets, we uncovered 27 biological topics and constructed corresponding topic-specific networks that reflect distinct biological states, perturbation conditions, and disease-related regulatory programs. Graph-topological metrics, including centrality, local fractality, and perturbation sensitivity, were computed for each gene to capture structural relevance within these topic-specific contexts. We demonstrate that topological profiles distinguish well-characterized regulators, identify emerging functional candidates, and reveal context-specific roles. Thereby, our framework enables the prioritization of understudied genes by assessing the robustness of their topological signatures across topic-specific networks. To support exploration of these results, we developed a publicly accessible interactive browser application, CENTRA, which enables dynamic navigation of networks and their functional annotations. CENTRA provides an interpretable, scalable framework for investigating context-dependent gene function and hypothesis generation, offering a novel entry point beyond traditional enrichment approaches.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.30.662180v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

