<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Monday, 11 August 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Environmental adaptations in metagenomes revealed by deep learning.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Johanna C Winder, Simon Poulton, Taoyang Wu, Thomas Mock, Cock van Oosterhout</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We apply a transfer learning approach using the ESM-2 protein structure prediction model and our own smaller ANN to classify proteins containing the domain of unknown function 3494 by their source environments.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Deep learning has emerged as a powerful tool in the analysis of biological data, including the analysis of large metagenome data. However, its application remains limited due to high computational costs, model complexity, and difficulty extracting biological insights from these artificial neural networks (ANNs). In this study, we applied a transfer learning approach using the ESM-2 protein structure prediction model and our own smaller ANN to classify proteins containing the domain of unknown function 3494 (DUF3494) by their source environments. DUF3494 is found in a diverse group of putative ice-binding and substrate-binding proteins across a range of environments in prokaryotic and eukaryotic microorganisms. They present a compelling test case for exploring the balance between prediction accuracy and interpretability in sequence classification.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40784938/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">HomRank: Homogeneous RNA Ranking for 3D Structure Evaluation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Li, L., Yao, Y., Martinovic;, I., Kulkarni, M., Pan, Y., Huber, R. G., Sick, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose an improved pipeline for RNA 3D structure assessment and a novel ranking-based evaluation paradigm that directly optimizes the selection of near-native conformations from candidate sets, resembling expert evaluation strategies.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Leveraging rigorously curated RNA sequences and a novel ranking-based evaluation paradigm, we propose an improved pipeline for RNA 3D structure assessment. To enhance generalization, we construct a dataset comprising non-redundant single-chain RNA sequences from the PDB and apply unsupervised clustering to minimize data leakage. Based on this, we retrain the previous state-of-the-art method, ARES, to develop ARES+, which improves the top-1 success rate by 21% on the RNA Puzzles benchmark. To further boost near-native identification, we introduce HomRank, a homogeneous RNA ranking method that directly optimizes the selection of near-native conformations from candidate sets, resembling expert evaluation strategies. HomRank achieves 95% and 100% top-1 and top-5 success rates, respectively, significantly outperforming ARES+. These results demonstrate that carefully designed datasets and the expert-like selection paradigm can substantially improve the accuracy and robustness of RNA 3D structure assessment, offering a promising direction for deep learning-based RNA evaluation and near-native conformation selection.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.07.669087v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Predicting Loop Quality in Protein Structure Models</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wieser, F., Kaltenbrunner, S., Zechner, A., Zach, M., Pock, T., Oberdorfer, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce the Equivariant Loop Evaluation Network (ELEN), a local model quality assessment (MQA) method that is tailored towards evaluating the accuracy of protein loops at the per-residue level.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Typically, sequences designed de novo are assessed in silico using deep learning-based protein structure prediction methods prior to wetlab testing. While these deep learning (DL) models excel at predicting well-ordered regions, accurate prediction of loop regions, which often are flexible and crucial for protein function, remains a significant challenge. To address this, we introduce the Equivariant Loop Evaluation Network (ELEN), a local model quality assessment (MQA) method that is tailored towards evaluating the accuracy of protein loops at the per-residue level. ELEN jointly predicts three quality metrics, local Distance Difference Test (lDDT), Contact Area Difference Score (CAD-score), and Root Mean Squared Deviation (RMSD), by comparing predicted to experimental reference structures. Learning these metrics simultaneously enables ELEN to capture complementary structural insights, providing a richer assessment of model accuracy. The network operates at all-atom resolution and employs 3D equivariant group convolutions to learn the local geometric environment of each atom. By incorporating sequence embeddings from large language models (LLMs), such as SaProt, we enhance the sequence and evolutionary awareness of the model. Furthermore, by informing ELEN with per-residue physicochemical features, the model achieves competitive accuracy relative to state-of-the-art MQA methods on the Continuous Automated Model EvaluatiOn (CAMEO) benchmark. Although ELEN was primarily developed for assessing loop quality, its architecture also demonstrates strong potential for general MQA tasks. We used ELEN to perform detailed analysis, including identification of flexible or disordered regions and assessment of structural effects from single-residue mutations on three sets of redesigned enzymes. We show that for all sets ELEN successfully identifies poor design positions and thus serves as a powerful tool for advancing both the study and modeling of loops in protein structures.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.07.668890v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrative analysis of gene networks associated with adipose and muscle traits in Hanwoo steers</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hwang, S., Jeong, T., Lee, J. y., Park, W. c., Jang, S. s., Lim, D. j.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified seven co-expression modules and hub genes related to fat and muscle traits using Weighted Gene Co-expression Network analysis (WGCNA).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Objective This study aims to characterize tissue-specific expression patterns in Hanwoo steers by identifying co-expression modules, functional pathways, and hub genes related to fat and muscle traits using Weighted Gene Co-expression Network analysis (WGCNA). Methods RNA-Seq data were generated from three muscle tissues (longissimus muscle, tenderloin, and rump) and two fat tissues (back fat and abdominal fat) collected from six 30-month-old Hanwoo steers. Quality control of raw sequencing reads was performed using FastQC, and trimmed reads were aligned to the bovine reference genome (ARS-UCD1.3) using HISAT2. We also identified a gene co-expression network via WGCNA using normalized gene expression values. Modules were defined based on topological overlap and correlated with tissue-specific expression patterns. Modules with a significant association (p < 0.05) were used for functional enrichment based on Gene Ontology (GO) and KEGG pathways, as well as Protein-Protein Interaction Network analysis. Results A total of seven co-expression modules were identified by WGCNA and labeled in distinct colors (yellow, blue, red, brown, turquoise, green, black). Among them, the yellow and blue modules were positively associated with back fat, while the turquoise and green modules showed a negative correlation with abdominal fat. Additionally, the turquoise or green module was positively correlated with longissimus and rump tissues, indicating distinct gene expression patterns between fat and muscle. Conclusion This study identified key co-expression modules and hub genes associated with muscle and fat metabolism. Notably, ARPC5 (blue module) was involved in lipid metabolism and energy storage, whereas AGPAT5 (turquoise module) was linked to maintaining muscle cell structure and function. These findings reveal biological mechanisms for tissue-specific gene regulation, providing targets for enhancing meat quality in Hanwoo.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.07.669089v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Transfer learning enables discovery of sub-micromolar antibacterials for ESKAPE pathogens from ultra-large chemical spaces</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Garcia-Ortegon, M., Seal, S., Geddes, E., Littler, J. L., Guy, C. S., Whiteside, J., Rasmussen, C. E., Bender, A., Bacallado, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We use deep graph neural networks to identify antibacterial compounds from ultra-large chemical libraries using transfer learning, which significantly improved enrichment factors and predictive performance in cross-dataset benchmarks.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The rise of antimicrobial resistance, especially among gram-negative ESKAPE pathogens, presents an urgent global health threat. However, the discovery of new antibiotics is hampered by sparse publicly available antibacterial data, complex bacterial defenses, and weak economic incentives. Here, we introduce a transfer learning framework using deep graph neural networks (DGNNs) to identify antibacterials from ultra-large chemical libraries. DGNNs were first pre-trained on large molecular datasets of protein-ligand simulations, binding affinities, and physicochemical properties to learn generalizable chemical features, and then fine-tuned on limited antibacterial screening data. Compared to classical methods, transfer learning significantly improved enrichment factors and predictive performance in cross-dataset benchmarks. Applying this strategy to the ChemDiv and Enamine libraries, we virtually screened over a billion compounds and prioritized 156 candidates. Experimental testing against Escherichia coli revealed that 54% of compounds exhibited antibacterial activity (MIC <= micro g/ml), with several demonstrating sub-micromolar potency and broad-spectrum efficacy against gram-positive and gram-negative pathogens, including three ESKAPE species. Of 18 broad-spectrum candidates, 15 showed minimal cytotoxicity and no hemolytic activity. These results validate our approach for navigating underexplored chemical space and identifying potent, low-toxicity compounds with antibiotic activity. We release open-source models and a scalable workflow to accelerate antibacterial discovery in the face of data scarcity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.07.669197v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">RESM: Capturing sequence and structure encoding of RNAs by mapped transfer learning from ESM (evolutionary scale modeling) protein language model</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, Y., Zhang, H., Li, G.-W., Wang, H., Zhang, X., Hong, X., Zhang, T., Wen, L., Zhao, Y., Jiang, J., Chen, J., Chen, Y., Liu, L., Zhan, J., Zhou, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new RNA Evolution-Scale Model for RNA sequences that can generalize to unseen sequences beyond its 1,024-nucleotide training limit, with state-of-the-art results in sequence classification and secondary structure prediction.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> RNA sequences exhibit lower evolutionary conservation than proteins due to their informationally constrained four-letter alphabet, compared to the 20-letter code of proteins. More limited information makes unsupervised learning of structural and functional evolutionary patterns more challenging from single RNA sequences. We overcame this limitation by mapping RNA sequences to pseudo-protein sequences to allow effective transfer training from a protein language model (protein Evolution-Scale Model 2, protESM-2). The resulting RNA ESM (RESM) outperforms 12 existing RNA language models in zero-shot prediction, not only in sequence classification but also in RNA secondary structure and RNA-RNA interaction prediction. Further supervised fine-tuning demonstrates RESM's generalizability and superior performance over the existing models compared across multiple downstream tasks, including mRNA ribosome loading efficiency and gene expression prediction, despite RESM being trained exclusively on noncoding RNAs. Moreover, RESM can generalize to unseen sequences beyond its 1,024-nucleotide training limit, achieving 81.3% improvement over state-of-the-art methods in supervised secondary structure prediction for RNAs up to 4,000 nucleotides, limited only by the available GPU memory, while providing >1000-fold speedup compared to MSA-based approaches. RESM provides a robust foundation for deciphering RNA sequence-structure-function relationships, with broad implications for RNA biology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.09.669469v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The FORGENIUS genomic resources: new genotyping tools and genomic data for 23 forest tree species and their Genetic Conservation Units</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pinosio, S., Bagnoli, F., Avanzi, C., Castellani, M. B., Frascella, A., McEvoy, S. L., Olsson, S., Spanu, I., Vajana, E., FORGENIUS CONSORTIUM,, Gonzalez-Martinez, S. C., Pyhäjärvi, T., Scotti, I., Vendramin, G. G., Piotti, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed and applied targeted genotyping tools for genome-wide genetic assessments of forest tree species in the European H2020 FORGENIUS project.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genetic diversity is a critical but often overlooked component of biological diversity. The European H2020 FORGENIUS project is precisely aimed at increasing the quality and quantity of genetic data to start monitoring the European network of forest Genetic Conservation Units (GCUs). A first step in this direction was developing standardized genomic resources for 23 forest tree species, spanning from rare and scattered (e.g., Abies nebrodensis and Torminalis glaberrima) to widespread, economically relevant ones (e.g., Fagus sylvatica, Picea abies and Pinus sylvestris). Here, we describe the development and application of targeted genotyping tools, primarily based on Single Primer Enrichment Technology (SPET), along with existing SNP arrays for the selected species. The SPET panels developed in FORGENIUS were designed to capture 10,000 loci per species, balancing species-specific and randomly distributed regions to ensure broad genome coverage and minimize ascertainment bias. Across 7,192 genotyped trees, we identified over 1.8 million single nucleotide polymorphisms (SNPs) covering approximately 50 Mb of DNA sequence. SPET panels demonstrated high genotyping efficiency and cross-species transferability, especially within genera such as Quercus and Abies. They represent a cost-effective, flexible, and scalable solution for population-level genetic assessments across diverse taxa, enabling standardized, genome-wide characterization of the GCU network. These resources not only promote the establishment of genetic monitoring, support genetically informed conservation strategies and improve our understanding of adaptive responses in European forests, but also enhance species delimitation and hybrid detection, and enable the characterization of phylogenetically related but previously underexplored species.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.08.669074v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Replication rate-information storage trade-off shapes genome architecture across domains</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sahu, P., Barik, S., Ghosh, K., Subramanian, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Genome size varies widely among organisms, from compact genomes of bacteria to vast and complex genomes of eukaryotes, and we propose a connection between genome size and cellular energetics.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genome size varies widely among organisms, from compact genomes of bacteria to vast and complex genomes of eukaryotes. In this study, we theoretically identify the evolutionary pressures that may have driven this divergence in genome size. We use a parameter-free model to study genome size evolution under selection pressure to minimize replication time and maximize information storage capacity. We substantiate this choice by demonstrating a correlation between replication time and genome size using literature data. We show that bacteria tend to reduce genome size, constrained by a single replication origin, while eukaryotes expand their genomes by incorporating multiple replication origins. We propose a connection between genome size and cellular energetics, suggesting that endosymbiotic organelles, mitochondria and chloroplasts, evolutionarily regulate the number of replication origins, thereby influencing genome size in eukaryotes. We substantiate this claim by showing a correlation between organelle count and genome size data. We argue that nucleotide skews, present in nearly all genomes studied, and are used to identify replication origins, directly influence DNA unzipping kinetics, and hence its replication time. This connection enables us to derive nearly universal observations, such as Chargaff's second parity rule and replichore symmetrization, as adaptive consequences. We argue that high skews lead to faster replication, and substantiate it by showing a correlation between skews and replication speed in bacteria. The model reproduces more evolutionary genomic observations, such as a general preference for deletions over insertions across domains, and elongation and high variance of genome size under reduced selection pressure for replication rate, an integral component of C-value paradox. We highlight the possibility of regulation of the firing of latent replication origins in response to cues from the extracellular environment, leading to the regulation of cell cycle rates in multicellular eukaryotes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.07.669222v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Beyond polyA: scalable single-cell total RNA-seq unifies coding and non-coding transcriptomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Isakova, A., Liu, D. D., Cvijovic, I., Sinha, R., Eastman, A. E., Saul, S., Detweiler, A., Neff, N., Einav, S., Weissman, I. L., Quake, S. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that non-coding RNAs are key to cell identity and state, and show how expanded transcriptome coverage can reveal additional layers of gene control - now accessible through practical and scalable single-cell profiling.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Non-coding RNAs represent a widespread and diverse layer of post-transcriptional regulation across cell types and states, yet much of their diversity remains uncharted at single-cell resolution. This gap stems from the limitations of widely used single-cell RNA-sequencing protocols, which focus on polyadenylated transcripts and miss many short or non-polyadenylated RNAs. Here, we adapted single-cell RNA-sequencing on the 10x Genomics platform to capture a broad complement of coding and non-coding RNAs - including miRNAs, tRNAs, lncRNAs, histone RNAs, and non-adenylated viral transcripts. This approach enabled the discovery of rich, dynamic non-coding RNA programs across immune cells, virally infected hepatocytes, and the developing human brain. In dengue virus-infected hepatocytes, we detect non-adenylated viral transcripts and distinguish active from transcriptionally quiescent infected states, each with distinct host regulatory signatures. In brain tissue, we identify biotype-specific, cell-type-restricted non-coding RNAs, including miRNAs whose expression anticorrelates with predicted targets, consistent with post-transcriptional regulatory relationships. We show that MIR137, one of the strongest GWAS loci associated with schizophrenia and intellectual disability, is expressed specifically in Cajal-Retzius cells, an early-born but transient population that guides subsequent cortical neuron migration. These findings demonstrate the importance of non-coding RNAs in defining cell identity and state, and show how expanded transcriptome coverage can reveal additional layers of gene control - now accessible through practical and scalable single-cell profiling.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.08.669394v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

