<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Tuesday, 16 September 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Systematic identification and characterization of regulators of aryl hydrocarbon receptor signaling</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Verma, M., Desai, K., Ding, Y., Wang, X., Bae, M., Rakoff-Nahoum, S., Balskus, E. P., Fischbach, M. A., Jost, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify a large network of regulators of human aryl hydrocarbon receptor (AHR) signaling in hepatocytes and propose novel modes of regulation of AHR activation, with implications for our understanding of host-microbe interactions and integrative chemosensation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The human aryl hydrocarbon receptor (AHR) integrates chemical signals derived from the environment, gut microbes, and endogenous sources to regulate processes ranging from intestinal barrier integrity to xenobiotic detoxification. Despite strong evidence that dysregulation of AHR signaling is a causal factor in metabolic and autoimmune disorders, we currently lack a comprehensive understanding of the factors that regulate AHR activity in human cells. Here, we use genome-scale CRISPR screening to systematically identify regulators of AHR signaling in hepatocytes. The resulting datasets recapitulate the core AHR signaling pathway and identify a large network of regulators. Many of these factors have roles beyond AHR signaling, reflecting that AHR signaling is deeply integrated into human cell biology. We further dissect this network to reveal novel modes of regulation of AHR expression, protein levels, and signaling. For example, we find that the E3 ubiquitin ligase UBR5 sustains AHR signaling by counteracting degradation of ligand-bound AHR. Finally, we identify components of the AHR regulatory network that are specific to cell types and ligands as potential nodes to manipulate AHR signaling in a targeted manner for therapeutic benefit. Overall, our results define the regulatory network that underpins AHR activation, with implications for our understanding of host-microbe interactions and integrative chemosensation and the etiology of metabolic and inflammatory disorders.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.15.676391v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A role for the male germline in the expansion of the mammalian brain</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bush, S. J., Goriely, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a new theory of the testis-brain connection and show that thousands of proteins functionally associated with brain growth and development are found in male germ cells, but show particular enrichment in spermatogonia.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The brain and testis share a surprisingly high number of molecular and cellular similarities. We have previously hypothesised that, throughout evolution, many genetic variants contributing to brain size expansion first arose in spermatogonia where they conferred a selective advantage to the male germline stem cells via a process analogous to oncogenesis - known as 'selfish spermatogonial selection'. Upon transmission of the mutant sperm to the next generation, these selfish variants became constitutive, disproportionately accumulating in signalling pathways active in both spermatogenesis and neurogenesis and which regulate stem cell proliferation. While we believe these observations supporting a close association between germline and brain development are compelling, research in this area is stymied by the relative scarcity of spermatogonia and the inherent stochasticity of the single-cell transcriptomic methods used to profile them. Accordingly, the molecular signatures of spermatogonia are incompletely understood, and their similarity with those of the brain difficult to assess. To support further work on the testis-brain relationship, we combine re-analyses of 34 adult human single-cell testis datasets with data from the Human Protein Atlas to assess the extent to which genes functionally associated with brain growth and development are expressed within testicular cell types. Consistent with our hypothesis, we find that of thousands of proteins functionally associated with the brain, the majority are not only expressed in male germ cells, but show particular enrichment in spermatogonia. We contextualise these results with an extensive literature survey and conclude that further enquiry into the testis-brain connection may yield novel insight into the evolutionary processes that shaped the human condition.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.15.676184v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Development of an Integrated Single-Cell and Spatial Transcriptomics Atlas of Healthy Human Skin Focusing on the Pilosebaceous Unit</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Duz, T., Torocsik, D., Momnougui, S. M. E., Oubounyt, M., Al, B., Gallinat, S., Baumbach, J., Holzscheck, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed the healthy Human Skin Cell Atlas (HSCA), systematically integrating 34 public datasets and totaling 821,464 cells, with curated metadata and harmonized cell type nomenclature.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell and spatial transcriptomics have transformed our ability to chart human tissue organization at unprecedented resolution. These technologies enable the construction of high-quality reference atlases, essential for mapping healthy tissue architecture and identifying robust gene markers. We developed the healthy Human Skin Cell Atlas (HSCA), systematically integrating 34 public datasets and totaling 821,464 cells, with curated metadata and harmonized cell type nomenclature to ensure consistency. We place particular emphasis on the pilosebaceous unit, a key epithelial structure critical for both homeostasis and pathology. While prior studies captured the interfollicular epidermis and immune landscape in detail, deeper hair follicle regions remained under-characterized. By leveraging high-resolution spatial transcriptomics (Visium HD), we spatially resolved and transcriptionally defined the lower hair follicle compartments and pinpointed signalling hubs. Furthermore, the HSCA enables the detection of cell types not visible in standalone datasets, such as Merkel cells. Our results illustrate the value of the integrated single-cell atlas and spatial data in refining tissue organization and highlight the PSU as a complex and diverse epithelial niche.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.09.675235v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Meta-Analysis of Transcriptomic Datasets Reveals Key Immune Gene Profiles and Signaling Pathways in Bos taurus</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Marimuthu, V. K. D., Matheswaran, K., Thambiraja, M., Yennamalli, R. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A meta-analysis of four bovine transcriptomic datasets to identify immune-related differentially expressed genes (DEGs) in Bos taurus.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding immune mechanisms in cattle is vital for improving disease resistance through informed breeding and vaccine development. Meta-analysis enables the integration of multiple transcriptomic studies to identify consistent gene expression patterns and enhance statistical power. In this study, we performed a meta-analysis of four bovine transcriptomic datasets (GSE45439, GSE62048, GSE125964, and GSE247921) to identify immune-related differentially expressed genes (DEGs) in Bos taurus. These datasets covered infections with Mycobacterium bovis and Mycobacterium avium subsp. paratuberculosis, comparing diseased and healthy cattle. Our pipeline included FastQC, Trimmomatic, Bowtie2, SAMtools, FeatureCounts, DESeq2, and MetaRNASeq, leading to the discovery of 28 significant DEGs (12 upregulated and 16 downregulated). Comparison with an innate immune gene database identified five key immune-related genes IL1A, RGS2, RCAN1, and ZBP1 with known regulatory roles in immunity. KEGG enrichment analysis revealed involvement in Necroptosis, Osteoclast Differentiation, Oxytocin Signaling, and cGMP-PKG Signaling pathways, associated with inflammatory cell death, cytokine signaling, and immune cell differentiation. This meta-analysis enhances understanding of conserved immune signaling mechanisms in cattle and highlights genes that may serve as biomarkers for immune competence, disease susceptibility, and vaccine responsiveness, offering valuable insights for future bovine immunogenomics research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.13.676061v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">JointMap identified SERPINA3+ chondrocytes as a therapeutic target for osteoarthritis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wu, H., Yan, W., Sun, M., Fan, Y., Jia, S., Wang, J., Xu, B., Lu, B., Du, Y., Chen, L., Lin, J., Cheng, J., Ao, Y., Hu, X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified a novel OA-elevated chondrocyte subtype, SERPINA3+ chondrogenesis function and intercellular communication patterns as potential therapeutic targets for articular diseases.</p><p style="font-size: 16px; text-align: justify;">Background: Osteoarthritis (OA) is a prevalent degenerative joint disease that affects nearly half of the population aged over 65. Recent single-cell transcriptional studies have identified heterogeneous OA-specific chondrocyte subtypes, yet a unified subtype-classification remains incomplete. Methods: In this study, we integrated a total of 545,946 cells to develop 'JointMap', an interspecies atlas. The chondrocyte subtypes were analyzed using machine learning algorithms were validated via human samples, animal models, and in vitro cell culture. We performed KAS-seq and analyzed the spatial characters for JointMap. Results: Based on clinical diagnosis and cross-omic indexes, we identified a novel OA-elevated chondrocyte subtype, SERPINA3+ chondrocytes. Notably, we uncovered their upregulated chondrogenesis function and intercellular communication patterns as potential therapeutic targets. Conclusions: In summary, JointMap provides a unified framework for the systematic annotation of chondral tissues across species and helps in the identification of druggable networks for articular diseases.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.09.675251v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Convenient screening for drug resistance mutations from historical febrile malaria samples across Kenya</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Magudha, J., Ndwiga, L., Akinyi, M., Wamae, K., Osoti, V., Kandie, R., Kiplagat, R., Keitany, K., Bargul, J. L., Akala, H. M., Oyier, L. I. O.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study on the evolution of antimalarial resistance genetic markers in Kenya .</p><p style="font-size: 16px; text-align: justify;">Background Historically, chloroquine resistance emerged that was driven by mutations in the chloroquine resistance transporter (Pfcrt) gene. This led to the global withdrawal of chloroquine in 1998 and its subsequent replacement with sulfadoxine-pyrimethamine, whose efficacy was compromised by a high prevalence of mutations in the dihydrofolate reductase and the dihydropteroate synthase genes by 2004. Consequently, artemisinin-based combination therapies (ACTs) were introduced in 2006. Since then, thirteen mutations in the kelch 13 (Pfk13) propeller domain have emerged and validated by the World Health Organization (WHO) as markers of partial artemisinin resistance. This study aimed to characterize temporal trends in both established, Pfcrt and Pfk13 and less well-described potential markers, cysteine desulfurase (Pfnfs) and Pfcoronin, using febrile malaria samples collected across diverse regions of Kenya between 2013 and 2022. Methods The temporal trend of these markers of resistance were assessed by screening archived P. falciparum positive dried blood spots (DBS). A total of 1,750 DBS samples collected from Therapeutic Efficacy Studies (TES) conducted in: Kwale (2013, n=350), Kisumu (2015, n=314), Busia (2016, n=334), Kisii (2017, n=314), Kwale (2018, n=150), and a hrp2 study conducted Kisii (2022, n=288). Parasite genomic DNA was extracted using the Chelex-saponin method and confirmed by a Pf 18S RT-PCR. Pfk13, Pfcrt, Pfnfs and Pfcoronin PCR amplicons were sequenced using capillary electrophoresis, Illumina Miseq or the Oxford Nanopore (GridION) platform. Results The prevalence of Pfcrt mutations declined over time and no WHO validated Pfk13 mutations associated with artemisinin resistance were detected. However, synonymous substitutions at WHO-validated codons C469C and P553P were identified. In the PfCoronin gene, non-synonymous mutations distinct from those reported in West Africa were observed at high frequencies (>75%). Notably, the Pfnfs-K65Q mutation, previously associated with reduced lumefantrine sensitivity in West Africa, was detected in over 80% of samples. Our findings reveal differences in some antimalarial resistance genetic markers between observations made in The Gambia and Senegal (West Africa) and Kilifi (East Africa). Based on the convenient sample set, there were no WHO validated k13 mutations up until 2022, suggesting continued ACT efficacy in Kenya. This study underscores the importance of continued molecular surveillance and suggests that resistance may evolve through different pathways in East compared to West Africa and Southeast Asia.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.09.675152v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Experimenting with Two Recent Feature Selection Methods for High-Dimensional Biological Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, M., Yang, X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Nullstrap is a robust, accurate, and computationally efficient feature selection tool for modern omics data analysis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Feature selection in high-dimensional biological data, where the number of features far exceeds the number of samples, has long posed a significant methodological challenge. This study evaluates two recently developed feature selection methods, Stabl and Nullstrap, under a simulation framework designed to replicate regression, classification, and non-linear regression tasks across varying feature dimensions and noise levels. Our results demonstrate that Nullstrap consistently outperforms Stabl and other benchmarked methods across all evaluated scenarios. Furthermore, Nullstrap proved significantly faster and more scalable in high-dimensional settings, underscoring its suitability for large-scale omics data applications. These findings establish Nullstrap as a robust, accurate, and computationally efficient feature selection tool for modern omics data analysis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.09.675248v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Characterizing intra- and inter-tumor heterogeneity in ovarian high-grade serous carcinoma subtypes using single-cell and spatial transcriptomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Li, W., Grieshober, L., Gertz, J., Ivich, A., Doherty, J. A., Greene, C. S., Hicks, S. C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of molecular subtypes in ovarian high-grade serous carcinoma and their variability between patients.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Ovarian high-grade serous carcinoma (HGSC) is an aggressive ovarian cancer with a heterogeneous tumor microenvironment (TME). Advances in single-cell RNA sequencing (scRNA-seq) and spatially-resolved transcriptomics have enabled the study of complex TME. This study explores connections between molecular subtypes described from bulk transcriptomes and spatial domains characterized by distinct gene expression in HGSC and their variability between patients. We explored both intra- and inter-tumor heterogeneity across 2D space and identified differing spatial patterns of gene expression pertaining to immune pathways and vasculature development. Functional characterization of tumor spaces revealed potentially shared cell states across molecular subtypes, while correlation analysis underscored subtype-specific spatial anti-colocalization between spots exhibiting antigen-presenting functions and B cell-mediated immunity. Lastly, we performed spatially-aware cell-cell communication analysis on the spatial samples and identified a molecular subtype specific difference in total signaling activity and heterogeneity in Midikine signaling between the differentiated subtype. Our results suggest that generating multiple tissue slices per patient might be necessary to enable comprehensive characterization of HGSC spatial transcriptomes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.15.676244v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Phage-mediated lysis does not determine Cutibacterium acnes colonization on human skin</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tripp, A. D., Qu, E. B., Balogun, I., Brodsky, J., Baker, J. S., Mancuso, C. P., Roux, S., Hussain, F. A., Lieberman, T. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that phage-bacteria interactions in the human microbiome are a static arms race, with a limited, phylogenetically restricted defense repertoire under weak selective pressure to diversify or be maintained.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Despite Cutibacterium acnes being the most abundant and prevalent bacteria on human skin, only a single type of phage has been identified that infects this host. Here, we leverage this one-to-one system to systematically characterize how the phage-bacteria arms race shapes C. acnes evolution and community composition on individual people. Our analysis reveals a surprising lack of phage-mediated selection despite global prevalence of C. acnes phages. Analysis of anti-phage defense systems across 3,205 bacterial genomes revealed a limited, phylogenetically restricted defense repertoire under weak selective pressure to diversify or be maintained. Functional assays did not reveal alternative phage resistance mechanisms or fitness costs associated with defense gene carriage that could explain this limited immune arsenal. This lack of pressure to maintain phage resistance could not be explained by lack of phage colonization, as examination of 471 global human facial skin metagenomes demonstrated that even in samples with high virus-to-microbe ratio, phage-sensitive clades dominate on-person populations. Together, these findings indicate that phage pressure, while present, does not play a critical role in determining strain fitness and success within C. acnes populations on human skin. We propose that this observed weak phage-mediated selective pressure can be explained by the anatomy of skin: C. acnes growth is thought to occur at the bottom of pores, where exposure to phage may be limited by physical barriers. Together, this portrait of a static arms race provides a strong contrast with other microbial species in different ecosystems and expands understanding of phage-bacteria interactions in the human microbiome.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.09.675206v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Repurposing base editors for targeted knock-in and simultaneous knockouts to generate multiplex-edited allogeneic CAR T cells with minimal translocations</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Glaser, V., Becker, L. J., Fuster-Garcia, C., Huth, L., Nitulescu, A.-M., Pu, Y., Kassing, I., Hartmann, L. M., Flugel, C. L., Karklins, R., Shaji, S. M., Pouzolles, M., Stein, M., Andrieux, G., Cathomen, T., Volk, H.-D., Reinke, P., Kath, J., Wagner, D. L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed Base editor-mediated knock-in (BEKI), a non-viral platform that allows targeted transgene insertion in parallel with multiplex gene disruption using a single base editor.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The CRISPR-Cas system enables precise genome engineering of cell therapies. For allogeneic applications, multiplex editing is frequently required to improve efficacy, persistence, and safety. However, strategies involving multiple DNA double-strand breaks (DSBs) induce genotoxicity by provoking chromosomal aberrations. Base editors, which enable sequence changes without generating DSBs, are widely used for gene disruption, but their capacity for gene insertion remains unexplored. Here, we developed Base editor-mediated knock-in (BEKI), a non-viral platform that allows targeted transgene insertion in parallel with multiplex gene disruption using a single base editor. This is achieved by repurposing the Cas9 nickase domain of base editors to generate paired nicks that induce homology-directed repair (HDR). In human T cells, optimized guide RNA orientation and nick distance, together with HDR-enhancing modulators, enabled efficient transgene knock-in at the TRAC, CD3Z, B2M, and CD3E loci. Simultaneous base editing of multiple additional genes produced chimeric antigen receptor (CAR) T cells with increased cytokine secretion, drug resistance, and resistance to allo-rejection. Compared to multiplex editing with Cas9, BEKI markedly reduced chromosomal translocations. BEKI therefore provides a streamlined, scalable strategy for multiplex CAR T-cell engineering with a single enzyme, offering a safer route to clinical-grade manufacturing of off-the-shelf therapies for cancer and autoimmune diseases.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.15.676172v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SMTrackR: a R/Bioconductor package for mapping protein binding at individual DNA molecules</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bansal, A., Barmola, H., Yadav, S., Rao, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present, SMTrackR, a Bioconductor package to visualize protein-DNA binding states on individual sequenced DNA molecules.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-molecule assays like NOMe-seq, dSMF and Nanopore are superior to DNase-seq and ATAC-seq as they do not nibble down DNA. Thus, they enable quantification of all three i.e., protein-free, Transcription Factor-bound and histone-complex-bound states. But a user-friendly tool to visualize and quantify such states is lacking. Here, we present, SMTrackR, a Bioconductor package to visualize protein-DNA binding states on individual sequenced DNA molecules. SMTrackR queries the single-molecule footprint database we built and hosted at Galaxy Server. It comprises of BigBed files generated from NOMe-seq, dSMF, and Nanopore (SMAC-seq) datasets. SMTrackR exploits UCSC REST API to query a BigBed file and plot footprint heatmap categorized in different binding states as well as report their occupancies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.09.674732v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Intelligent Tool Orchestration for Rapid Mechanistic Model Prototyping: MCP Servers as AI-Biology Interfaces</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ruscone, M., Vazquez, M., Valencia, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present intelligent tool orchestration through Model Context Protocol (MCP) servers that enable Large Language Models (LLMs) agents to function as AI laboratory assistants for rapid model prototyping.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The construction of multicellular mechanistic models in systems biology typically requires months of literature research, programming expertise, and deep knowledge of specialized computational tools. Here we present intelligent tool orchestration through Model Context Protocol (MCP) servers that enable Large Language Models (LLMs) agents to function as AI laboratory assistants for rapid model prototyping. We demonstrate this approach by constructing a multiscale model of cancer cell fate in response to TNF, entirely through natural language interactions, using an AI agent connected to MCP servers that interface with three complementary modeling software: NeKo for constructing gene regulatory networks from prior-knowledge databases, MaBoSS for simulating and analyzing Boolean models, and PhysiCell for setting up multicellular agent-based models. Our architecture encompasses more than 60 specialized tools, covering the entire workflow from data collection to multicellular simulation setup. From this use case we derived three key design principles for biological AI-tool integration: first, optimal tool granularity is best defined at biological decision points, where domain knowledge guides modeling choices; second, comprehensive session management is essential for tracking complex workflows and ensuring reproducibility across long interactions; and third, effective orchestration must be established from the start to let LLMs combine tools flexibly while maintaining biological coherence. In applying this framework across three different LLMs and scenarios, we observed consistent end-to-end orchestration as well as model-dependent differences in network size, dynamical behaviors, and integration details. This variability highlights both the portability of our approach and the importance of careful reporting and validation when deploying different LLMs in scientific contexts. While the resulting models require refinement, this work establishes the foundation for AI-assisted rapid prototyping in systems biology, enabling researchers to explore computational hypotheses at unprecedented speed while maintaining biological fidelity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.10.675105v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multilayered Transcriptomic Reprogramming and Spliceosomal Divergence Shape Bipolaris sorokiniana Pathogenicity</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dayama, B. R., Shukla, A. K., Kondhare, K. R., Kadoo, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We performed a comprehensive RNA-sequencing analysis of Bipolaris sorokiniana during attack on spot blotch-resistant and susceptible wheat genotypes, integrating differentially expressed genes (DEGs), alternative splicing (AS) events, and long non-coding RNAs (lnc</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Bipolaris sorokiniana, the causative agent of spot blotch of wheat, significantly limits wheat productivity. Despite the pathogen's widespread impact, the in planta molecular mechanisms underpinning its virulence remain poorly characterized. We performed a comprehensive RNA-sequencing analysis of B. sorokiniana during attack on spot-blotch-resistant and susceptible wheat genotypes, integrating differentially expressed genes (DEGs), alternative splicing (AS) events, and identification of long non-coding RNAs (lncRNAs). The pathogen exhibited extensive host-genotype-dependent transcriptomic reprogramming, with 128 pathogen genes upregulated in the susceptible host. These genes were associated with ribosome biogenesis, RNA processing, and primary metabolic functions, supporting aggressive colonization. In contrast, pathogen attack on the resistant genotype triggered the upregulation of 58 pathogen genes associated with stress-responsive pathways, including sphingolipid and ceramide metabolism. This suggests a shift toward defensive metabolic reprogramming in the resistant host that restricted its proliferation. Our investigation uncovered five classes of AS events and 14 differentially expressed lncRNAs, revealing substantial post-transcriptional complexity. Notably, a subunit of the H/ACA small nucleolar ribonucleoprotein (snoRNP) complex emerged as a rare 'triple-hit' candidate simultaneously identified as a DEG, differentially alternatively spliced gene, and target of a differentially expressed lncRNA, highlighting its potential as a central regulatory node in host-responsive stress adaptation. This study reveals a multilayered regulatory landscape involving transcriptional plasticity, alternative splicing, and lncRNA-mediated control, enabling B. sorokiniana to fine-tune its infection strategy in response to host resistance. This work advances the understanding of fungal pathogenesis and identifies molecular vulnerabilities that could be exploited for targeted, host-specific disease control.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.09.675095v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DNA methylation and immune infiltration mediate the impact of tobacco exposure on pancreatic adenocarcinoma outcome: a high-dimensional mediation analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pittion, F., Amblard, E., Devijver, E., Samson, A., Varoquaux, N., Richard, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified four distinct pathways where tobacco-induced epigenetic modifications mediate survival effects through immune microenvironment alterations, involving the HIST1H4E, NAA11, SPTBN2, and CHL1 loci.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Pancreatic ductal adenocarcinoma (PDAC) ranks among the most aggressive malignancies, characterized by exceptionally poor prognosis due to late diagnosis and therapeutic resistance. While tobacco exposure is an established risk factor for PDAC incidence and influences patient survival, the underlying molecular mechanisms mediating these effects remain poorly understood. We aimed to characterize the causal pathways linking tobacco exposure to PDAC patient survival, focusing on DNA methylation and tumor immune infiltration as potential mediators. For this purpose, we developed HDMAX2-surv, an extension of high-dimensional mediation analysis method HDMAX2, adapted to survival outcomes. Through extensive simulations across multiple scenarios, we demonstrated superior performance compared to existing methods in handling complex DNA methylation data. We integrated this approach with causal discovery frameworks and immune deconvolution algorithms to dissect multi-pathway mediation mechanisms. Using The Cancer Genome Atlas (TCGA) cohort of 145 PDAC patients, we identified 37 aggregated methylated regions (AMRs) with significant mediation effects, exhibiting both protective and deleterious impacts on survival. Our causal discovery analysis revealed complex relationships where tobacco effects operate through sequential pathways involving DNA methylation and immune infiltration changes. Serial mediation analysis identified four distinct pathways where tobacco-induced epigenetic modifications mediate survival effects through immune microenvironment alterations, involving the HIST1H4E, NAA11, SPTBN2, and CHL1 loci. Remarkably, these effects were undetectable through gene expression analysis alone, highlighting the unique value of methylation-based approaches. This study provides dual contributions: methodologically, through the first integrated framework for high-dimensional survival mediation analysis with causal discovery; and biologically, by elucidating complex molecular mechanisms through which tobacco exposure influences PDAC outcomes. Our findings reveal the intricate interplay between environmental exposures, epigenetic modifications, and immune responses in cancer prognosis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.09.675033v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrative multi-omics combined with functional pharmacological profiling in patient-derived organoids identifies personalized therapeutic vulnerabilities of adult high-grade gliomas</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ermini, L., Lipsa, A., Hau, A.-C., Krokhmal, I., Nosirov, B., Toth, R., Klein, E., Oudin, A., Houben, L., Fritah, S., Herold-Mende, C., Frauenknecht, K. B. M., Mittelbronn, M., Berchem, G., Hertel, F., Malta, T. M., Nazarov, P. V., Niclou, S. P., Golebiewska, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified distinct subgroups of high-grade gliomas associated with IDH1 mutation and MYCN amplification, and a MYCN-amplified glioblastoma with selective sensitivity to histone deacetylase 3 inhibitors.</p><p style="font-size: 16px; text-align: justify;">Background: Precision medicine has transformed cancer treatment by tailoring therapies to specific molecular aberrations. Integrating high-resolution multi-omics with high-throughput functional profiling in patient-derived organoids offers a powerful strategy to further refine patient stratification. While (epi)genetic profiling has drastically improved the classification in diffuse adult gliomas, these advances have not yet translated into effective therapeutic interventions and precision medicine approaches remain to be established. Material and Methods: We investigated a panel of 48 patient-derived organoid and orthotopic xenograft models of adult high-grade gliomas, comprehensively characterized at genomic, epigenomic and transcriptomic levels. A functional drug screen was performed on 27 organoid models using a 202-compound library targeting cancer-related pathways and epigenetic regulators. Unsupervised multi-omics factor analysis was employed to identify patient-specific therapeutic vulnerabilities. Validation included dose-dependent drug efficacy assessments, as well as biomarker assessment in patient tumors across molecular subgroups. Results: Multi-omics analysis revealed a broad spectrum of molecular profiles capturing the genetic, epigenetic, and transcriptomic diversity of high-grade gliomas. Multi-omics factor analysis, integrating multi-omics and drug response profiles, identified distinct subgroups associated with IDH1 mutation and MYCN amplification. IDH1 mutant grade 4 astrocytomas showed selective sensitivity to histone deacetylase 3 inhibitors, while a MYCN-amplified glioblastoma responded preferentially to histone methyltransferase inhibitors. The differential drug responses were linked to specific (epi)genetic and transcriptomic biomarkers. While other glioblastomas exhibited heterogeneous treatment responses, no robust biomarker-defined responder subgroups were identified. Conclusion: Our findings highlight the value of integrating multi-omics and functional profiling to inform precision medicine strategies. This approach enables the stratification of distinct patient subgroups in preclinical models, paving the way for tailored therapeutic interventions. While we observed distinct pharmacogenomic profiles in IDH1 mu-tant grade 4 astrocytomas and a MYCN-amplified glioblastoma, implementing precision medicine in other glioblastoma subtypes remains a substantial challenge.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.09.675145v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Dual-Functional Biomimetic Nanoclusters for Sustained Antifibrotic Therapy and Vascular Protection in Fibrosis Management</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Monteiro, R. A., Oliveira, C. S., Ferreira, T. L., Duarte, M. C., Carvalho, P. H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a biomimetic nanocluster system designed to overcome therapeutic bottlenecks in fibrotic disease management and demonstrate the dual antifibrotic and vascular protective effects of nanoclusters.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Fibrosis represents a critical pathological hallmark underlying multiple chronic diseases, yet conventional pharmacological interventions are hampered by limited bioavailability, rapid systemic clearance, and insufficient vascular protection. In this study, we developed a biomimetic nanocluster system designed to overcome these therapeutic bottlenecks. The nanoclusters were fabricated via a self-assembly approach using PLGA-PEG and cholesterol matrices to co-encapsulate pirfenidone (antifibrotic) and resveratrol (vascular protective agent). Physicochemical characterization confirmed uniform particle morphology, stable size distribution, and sustained colloidal stability. Pharmacokinetic evaluation revealed that nanoclusters achieved a delayed peak time and prolonged plateau phase, maintaining therapeutic drug concentrations in serum and hepatic tissue for up to 72 h, in contrast to the rapid clearance observed in micelle and free-drug formulations. Mechanistic investigations suggested that biomimetic surface engineering and electrostatic coupling contributed to suppressed burst release, enhanced retention in fibrotic microenvironments, and receptor-mediated uptake. Functional assays further demonstrated significant inhibition of hepatic stellate cell activation, reduction of collagen deposition, and improved endothelial tube formation. In a CCl4-induced hepatic fibrosis mouse model, nanocluster treatment markedly decreased fibrotic area and enhanced CD31-positive vessel density without detectable systemic toxicity. The findings highlight the dual antifibrotic and vascular protective effects of biomimetic nanoclusters, which substantially outperform conventional nanocarriers in both therapeutic durability and safety. The present work not only introduces a long-acting drug delivery paradigm for fibrotic disease management but also provides an engineering framework adaptable to broader applications in translational nanomedicine.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.09.675075v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Systems Age: a single blood methylation test to quantify aging heterogeneity across 11 physiological systems.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Raghav Sehgal, Yaroslav Markov, Chenxi Qin, Margarita Meer, Courtney Hadley, Aladdin H Shadyab, Ramon Casanova, JoAnn E Manson, Parveen Bhatti, Ann Z Moore, Eileen M Crimmins, Sara Hagg, Themistocles L Assimes, Eric A Whitsel, Albert T Higgins-Chen, Morgan Levine</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Aging occurs at different rates across individuals and physiological systems, but most epigenetic clocks provide a single age estimate, overlooking within-person variation. Here we developed systems-based DNA methylation clocks that measure aging in 11 distinct physiological systems-Heart, Lung, Kidney, Liver, Brain,</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Aging occurs at different rates across individuals and physiological systems, but most epigenetic clocks provide a single age estimate, overlooking within-person variation. Here we developed systems-based DNA methylation clocks that measure aging in 11 distinct physiological systems-Heart, Lung, Kidney, Liver, Brain, Immune, Inflammatory, Blood, Musculoskeletal, Hormone and Metabolic-using data from a single blood draw. By integrating supervised and unsupervised machine learning with clinical biomarkers, functional assessments and mortality risk, we derived system-specific scores that outperformed existing global clocks in predicting relevant diseases and aging phenotypes. We also created a composite Systems Age score to capture overall multisystem aging. Clustering individuals based on these scores revealed distinct biological aging subtypes, each associated with unique patterns of health decline and disease risk. This framework enables a more granular and clinically relevant assessment of biological aging and may support personalized approaches to monitor and target system-specific aging processes.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40954326/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

