<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Wednesday, 04 June 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Gut microbiota and its influence on the Gut-Brain axis in comparison with chemotherapy patients and cancer-free control data in Breast cancer-A computational perspective.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tamizhini Loganathan, George Priya Doss C</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of gut microbiome diversity, community composition, and pathway analysis in women undergoing chemotherapy for breast cancer .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Breast cancer (BC) continues to be a major cause of cancer-related illness and death among women worldwide. Traditional treatments include surgery, radiation, hormone therapy, and chemotherapy, but these approaches often face challenges due to variability in patient response and adverse effects. This study investigated the relationship between gut microbiome diversity, community composition, and pathway analysis in women undergoing chemotherapy for BC (During Treatment-DT) compared to cancer-free controls (CFC). Using 16S rRNA amplicon sequencing, the study assessed alpha and beta diversity. Results showed differences in microbiome composition between DT and CFC samples, with Firmicutes being highly abundant in both groups. Core microbiome and correlation analysis at the phylum and genus levels identified significant microbiota. Specifically, the abundance of genera such as Pseudomonas and Akkermansia decreased, while Ruminococcus and Allistipes increased, as determined by statistical and machine learning approaches. Disease associations were examined based on KO abundance, identifying links to conditions such as autism spectrum disorder, Clostridium difficile infection, chronic kidney disease, and multiple sclerosis. Key KEGG pathways enriched in DT and CFC groups included the two-component system, tyrosine metabolism, and the pentose phosphate pathway. Conversely, dysbiosis or the presence of pathogenic bacteria (Ruminococcus) associated with the SOX8 gene could lead to chemoresistance, altered metabolic pathways, and increased toxicity. These findings underscore the potential implications for treatment outcomes and personalized medicine.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40460162/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Highly sensitive and scalable time-resolved RNA sequencing in single cells with scNT-seq2</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Qiu, Q., Zhang, H., Gao, W., Li, F., Liang, D., Wu, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present scNT-seq2, a highly sensitive and scalable method for time-resolved single-cell RNA sequencing.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding gene expression dynamics requires resolving newly synthesized RNAs from pre-existing pools at single-cell resolution. Here, we present scNT-seq2, a highly sensitive and scalable method for time-resolved single-cell RNA sequencing. By systematically optimizing the second-strand cDNA synthesis (2nd SS) step, we substantially improved read alignment rates, reduced background mutations, and enhanced library complexity compared to the original scNT-seq 1. Benchmarking in 4sU-labeled K562 cells demonstrated that scNT-seq2 accurately quantifies newly synthesized transcripts and preserves the gene level RNA turnover. The enhanced sensitivity enables robust detection of dynamic, cell-cycle state specific genes, such as S-phase regulators. Together, scNT-seq2 provides an efficient and versatile tool for dissecting transcriptional dynamics across diverse biological systems at single-cell resolution.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.03.657745v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genomic factors contributing to the resilience of Salmonella enterica on ready-to-eat muskmelon</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Esteban Cuesta, I., Führer, L., Porwollik, S., Chu, W., Fiddaman, S. R., Sah, I., McClelland, M., Guldimann, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Genes from Salmonella serovars are selected for survival and growth on muskmelon, a food safety risk.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Salmonella outbreaks have repeatedly been associated with muskmelons. This fruit poses a food safety risk due to its growth on the soil surface and because it is eaten raw. To identify genes under selection in S. enterica growing in this food matrix, barcoded transposon mutant libraries in three serovars of Salmonella, Typhimurium, Enteritidis, and Newport, were screened for survival and growth on muskmelon. The role of selected genes was subsequently verified in competition assays. Genes whose disruption caused a fitness effect in all three serovars were enriched for functions related to RNA degradation and ribosome biogenesis. The polyribonucleotide nucleotidyltransferase gene, pnp, and the D-3-phosphoglycerate dehydrogenase gene, serA, conferred an advantage when growing in muskmelon but not in a nutrition-rich control environment. This study provides a rare insight into genome-wide adaptation mechanisms of multiple Salmonella serovars to growth on food matrices, exemplified by muskmelons.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.02.657348v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Circular van Krevelen diagram for visualizing metabolic pathways</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Li, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a new approach based on circularized van Krevelen diagram for metabolic pathway visualization based on chemical principles, providing the consistency critical for community exchanges and collaborations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Emerging biochemical data require effective pathway visualization. However, traditional metabolic maps were based on manual layouts, falling behind new scientific discoveries. Automated visualization of metabolic pathways has been investigated heavily, but resulted in few consensuses. We report here a new approach based on circularized van Krevelen diagram. This method is based on chemical principles, providing the consistency critical for community exchanges and collaborations. Example applications include metabolomics data interpretation and a new set of metabolic maps.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.31.657198v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Optimizing Strain Selection for Association Studies Under Hard Cost Constraints</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rau, C. D., Bradley, P. H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that selecting for cost is most beneficial at low-to-moderate budget thresholds, while selecting for diversity is optimal in scenarios involving rare (MAF 5-10%) variants or higher total costs, or when accounting for additional costs per strain.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Quantitative genetics methods can be particularly powerful in model organisms and non-human populations, and we now have strain collections such as recombinant inbred lines, etc. that can be phenotyped. Natural diversity is also valuable in non-model systems that do not yet have reverse genetic tools. However, purchasing and phenotyping large collections can be cost-prohibitive. Strain or sample acquisition costs may also vary dramatically for different strains or isolates. Thus, investigators need efficient strategies to optimize experimental power for a given limited budget. In this study, we evaluate several approaches to optimally select subsets of the total cohort to best maintain power when performing genome-wide association studies. Some approaches focus solely on costs, others on genetic diversity, and some on both simultaneously. Through simulation studies across different minor allele frequencies and SNP effect sizes, we demonstrate that selecting for cost is most beneficial at low-to-moderate budget thresholds, while selecting for diversity is optimal in scenarios involving rare (MAF 5-10%) variants or higher total costs, or when accounting for additional costs per strain studied. We also evaluate these approaches on data from the Hybrid Mouse Diversity Panel (HMDP), and find that an approach that considers both cost and diversity is superior at recovering significant loci and maintaining statistical power under real-world conditions. This approach picks the strains that, for a given budget, minimize the total genetic distance to the strains that were not selected. This approach, which we term "ThriftyMD" (for "Thrifty Minimum Distance"), extends previous distance-based methods to pick a representative panel by explicitly adding a cost constraint. Overall, our results highlight the trade-offs between cost, diversity, and power in GWAS cohort design, and present the ThriftyMD algorithm as a versatile and robust approach for optimizing study design in resource-limited settings.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.31.657208v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Deep-Learning on condition specific expression profiles reveal critical cytosines in gene regulation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kesarwani, V., Kumar, A., Sharma, A., Gupta, S., Shankar, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A deep learning system and a large volume of bisulfite and RNA-seq data have been used to decode criticality of cytosines in plants.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Compared to other nucleotides, the cytosines stand as the most expressive one for gene regulation in plants due to its status as methylation-based epigenetic switch. Methylation of some of these cytosines may have higher impact on the downstream genes, making them critical ones. To this date not much has been done to decipher the criticality of such cytosines. Understanding them may open door-ways for minimal interventions led modification and control of gene expression. This is one such pioneering step in decoding the critical 'C's where a large volume of bisulfite and RNA-seq data, including 232 WGBS and 260 corresponding RNA-seq datasets from A. thaliana and rice, have been utilized to decode criticality of cytosines in plants. Using a deep learning system, strong relationship was established between the methylation states of cytosines in contextual manner with respect to the downstream genes expression levels. Using the same system, all the 2kb upstream regions for Arabidopsis and rice genes have been annotated. The developed approach has been shown to be a universal one and may be used to annotate other plant genomes to identify critical cytosines. The developed resource and the deep-learning system to decode criticality have been made publicly available as CritiCal-C portal, which is poised to accelerate the field of plant epigenomics and regulatory studies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.01.657178v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">De novo design of protein-binding aptamers through deep reinforcement learning assembly of nucleic acid fragments</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Guo, G., Guo, L., Qian, J., He, X., Qian, X., Wang, L., Huang, Q.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed an artificial intelligence agent capable of de novo design of protein-binding DNA aptamers for the first time.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Nucleic acid aptamers targeting proteins are becoming increasingly important in biopharmaceuticals and molecular diagnostics. Traditionally, aptamers are discovered through labor-intensive screening of nucleic acid libraries using the SELEX method. However, de novo design approaches without experimental screening remain a significant challenge. Here, we employed deep reinforcement learning to develop an artificial intelligence (AI) agent capable of de novo aptamer design, termed AiDTA (AI-driven Docking-Then-Assembling). First, nucleic acid fragments were docked to the target protein to identify target-binding fragments. Then, AiDTA automatically assembled these fragments into aptamers using the Monte Carlo tree search algorithm and a policy-value neural network to guide the agent in generating aptamers with secondary structures similar to the original constituent fragments. Experimental validation demonstrated that the AiDTA-designed DNA aptamers targeting disease-related proteins exhibited high binding affinities in the nanomolar range, achieving the de novo design of protein-binding aptamers for the first time. Our study establishes a new approach to obtaining protein-binding aptamers for potential applications in biopharmaceuticals and diagnostics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.01.657174v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Competing Subclones and Fitness Diversity Shape Tumor Evolution Across Cancer Types</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chen, H., Shu, J., Mudappathi, R., Li, E., Wang, P., Bergsagel, L., Yang, P., Sun, Z., Zhao, L., Shi, C., Townsend, J. P., Maley, C., Liu, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a novel computational framework that models tumors as mixtures of two competing cell populations: an ancestral clone with baseline fitness and a derived subclone with elevated fitness.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Intratumor heterogeneity arises from ongoing somatic evolution complicating cancer diagnosis, prognosis, and treatment. Here we present TEATIME (estimating evolutionary events through single-timepoint sequencing), a novel computational framework that models tumors as mixtures of two competing cell populations: an ancestral clone with baseline fitness and a derived subclone with elevated fitness. Using cross-sectional bulk sequencing data, TEATIME estimates mutation rates, timing of subclone emergence, relative fitness, and number of generations of growth. To quantify intratumor fitness asymmetries, we introduce a novel metric--fitness diversity--which captures the imbalance between competing cell populations and serves as a measure of functional intratumor heterogeneity. Applying TEATIME to 33 tumor types from The Cancer Genome Atlas, we revealed divergent as well as convergent evolutionary patterns. Notably, we found that immune-hot microenvironments constraint subclonal expansion and limit fitness diversity. Moreover, we detected temporal dependencies in mutation acquisition, where early driver mutations in ancestral clones epistatically shape the fitness landscape, predisposing specific subclones to selective advantages. These findings underscore the importance of intratumor competition and tumor-microenvironment interactions in shaping evolutionary trajectories, driving intratumor heterogeneity. Lastly, we demonstrate that TEATIME-derived evolutionary parameters and fitness diversity offer novel prognostic insights across multiple cancer types.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.31.657191v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Structome-AlignViewer: On Confidence Assessment in Structure-Aware Alignments</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Malik, A. J., Mao, S., Hugenholtz, P. J., Ascher, D. B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A web-based method for evaluating the quality of structure-aware alignments through both spatial mapping of alignment columns to protein structures and quantitative confidence scoring.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein structure-based comparison provides a framework for uncovering deep evolutionary relationships that can escape conventional sequence-based approaches. Encoding three-dimensional protein structures using a simplified structure-aware alphabet, can lead to compact, comparable strings that retain key spatial relationships. Although this enables comparison, structure-aware alignments can experience misaligned regions, particularly when comparing proteins with substantial divergence in fold architecture. To address this, a web-based resource, Structome-AlignViewer, is introduced in this work for evaluating the quality of structure-aware alignments through both spatial mapping of alignment columns to protein structures and quantitative confidence scoring. Confidence is computed from pairwise structural substitutions between adjacent inputs and normalized within each alignment to highlight relatively well-supported columns. To provide broader context, thousands of alignments from established structural classification systems were analysed, allowing for an empirical comparative statistic to be derived to assess alignment quality. Option to exclude gap-rich regions enable users to refine alignments and focus on conserved structural cores. This approach provides an interpretable method for assessing structural alignment quality and supports more robust comparative and evolutionary analyses. Structome-AlignViewer is freely available at https://biosig.lab.uq.edu.au/structome_alignviewer/.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.31.657027v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DeepLC introduces transfer learning for accurate LC retention time prediction and adaptation to substantially different modifications and setups</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bouwmeester, R., Nameni, A., Declercq, A., Devreese, R., Velghe, K., Gorshkov, V., Penanes, P. A., Kjeldsen, F., Rompais, M., CARAPITO, C., Gabriels, R., Martens, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that transfer learning can successfully overcome the limitations of LC retention time prediction by leveraging pre-trained model parameters and can even fit highly performant models to substantially different peptide modifications and LC conditions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> While LC retention time prediction of peptides and their modifications has proven useful, widespread adoption and optimal performance are hindered by variations in experimental parameters. These variations can render retention time prediction models inaccurate and dramatically reduce the value of predictions for identification, validation, and DIA spectral library generation. To date, mitigation of these issues has been attempted through calibration or by training bespoke models for specific experimental setups, with only partial success. We here demonstrate that transfer learning can successfully overcome these limitations by leveraging pre-trained model parameters. Remarkably, this approach can even fit highly performant models to substantially different peptide modifications and LC conditions than those on which the model was originally trained. This impressive adaptability of transfer learning makes it a highly robust solution for accurate peptide retention time prediction across a very wide variety of imaginable proteomics workflows.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.01.657225v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">scVIVA: a probabilistic framework for representation of cells and their environments in spatial transcriptomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Levy, N., Ingelfinger, F., Bakulin, A., Cinnirella, G., Boyeau, P., Nadler, B., Ergen, C., Yosef, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce scVIVA, a deep generative model that leverages both cell-intrinsic and neighboring gene expression profiles to output stochastic embeddings of cell states as well as normalized gene Expression profiles.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Spatial transcriptomics provides a significant advance over studies of dissociated cells in that it reveals the environment in which cells reside, thus opening the way for a more complete description of their state and function. However, most current methods for embedding and discovery of cell states rely only on the cells' own gene expression profile, thus raising the need for ways to account for the neighboring cells as well. Here, we introduce scVIVA, a deep generative model that leverages both cell-intrinsic and neighboring gene expression profiles to output stochastic embeddings of cell states as well as normalized gene expression profiles. We demonstrate that scVIVA produces informative fine-grained partitions of cells that reflect both their internal state and the surrounding tissue and that its generative model facilitates the testing of hypotheses of differential expression between tissue niches. We leverage these properties of scVIVA to uncover a spatially-restricted tumor-promoting endothelial population in breast cancer and niche-associated T cell states that are shared across multiple cancers. scVIVA is available as open source software within scvi-tools.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.01.657182v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Meta-Analysis of Wild Relatives and Domesticated Species of Rice, Tomato, and Soybean Using Publicly Available Transcriptome Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yumiya, M., Bono, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We performed a meta-analysis of public gene expression data of domesticated species of rice, tomato, and soybean and their presumed ancestral species.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The domesticated species currently available in the market have been developed through the breeding of wild relatives. Breeding strategies using wild relatives with high genetic diversity are attracting attention as an important approach for ad-dressing climate change and ensuring sustainable food supply. However, studies examining gene expression variation in multiple wild and domesticated species are few. Therefore, we aimed to investigate the changes in gene expression associated with domestication. We performed a meta-analysis of public gene expression data of domesticated species of rice, tomato, and soybean and their presumed ancestral species. In wild relatives, the expression of genes involved in osmotic, drought, and wound stress tolerance was upregulated. In domesticated species, upregulated expression was observed in genes related to auxin and those involved in the efflux of heavy metals and harmful substances. These findings provide insights into how domestication influences changes in crop traits. Thus, our findings may contribute to rapid breeding and development of new varieties capable of growing in harsh natural environments. Hence, a new cultivation method called "de novo domestication" has been proposed, which combines the genetic diversity of currently unused wild relatives and wild relatives with genome editing technologies that enable rapid breeding.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.01.656526v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">BifurcatoR: A Framework for Revealing Clinically Actionable Signal in Variance Masquerading as Noise</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Madaj, Z. B., Ding, M., Khoo, C. K., Tokarski, E., Fagnocchi, L., PERMUTE,, Li, Z., Riordan, J., Pospisilik, J. A., Nadeau, J., Lary, C., Triche, T. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed BifurcatoR, an open-source software platform that simplifies the detection, modeling, and interpretation of Variance heterogeneity (VH), a powerful but underused lens for detecting latent etiological structures in disease heterogeneity.</p><p style="font-size: 16px; text-align: justify;">Background: Disease heterogeneity is a persistent challenge in medicine, complicating both research and treatment. Standard analytical pipelines often assume patient populations are homogeneous, overlooking variance patterns that may signal biologically distinct subgroups. Variance heterogeneity (VH), including skewness, outliers, and multimodal distributions, offers a powerful but underused lens for detecting latent etiological structures relevant to prognosis and therapeutic response. Methods: A major barrier to VH analysis is the fragmented landscape of available methods, many of which rely on normality assumptions that biological data frequently violate. In addition, existing tools often require programming expertise, and clear guidance on study design considerations, such as sample size and method selection, is lacking. To address these issues, we developed BifurcatoR, an open-source software platform that simplifies the detection, modeling, and interpretation of VH. BifurcatoR integrates simulation-based method evaluation, study design recommendations, and a user-friendly web interface to support VH analysis across a range of data distributions. We benchmarked VH methods through simulation and applied BifurcatoR to two clinical datasets: acute myeloid leukemia (AML) and obesity. Results: Simulation studies revealed that VH method performance is highly context-specific, varying with distribution shape, mean-variance coupling, and underlying subgroup structure. In AML, BifurcatoR identified two molecularly distinct subgroups with different treatment responses, including an EVI1-high group with significantly poorer prognosis (p < 0.005) among KMT2A-rearranged cases. In a separate study, VH analysis uncovered immunophenotypic subgroups in obesity based on gene-level discordance across monozygotic twin pairs, highlighting latent variation in adipose immune cell composition.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.01.657083v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A bio-informatics approach to identify new drug targets in multidrug-resistant bacteria</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ramsden, I., de Jong-Hoogland, D., Chiam, A. J., Ulmschneider, M. B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A non-human analogues drug target that is present in 11 bacterial species and present across multiple bacterial species.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Antibiotic resistance poses a global health crisis. In order to develop new antibiotic agents, it is crucial to identify drug targets in multidrug-resistant bacteria. Criteria for such a target are an -helical, essential membrane protein, that is non-homologues with the human membrane proteome, and present across multiple bacterial species. Using a stepwise subtractive genomics approach, the membrane protein F0F1 ATP synthase subunit C was identified as a non-human analogues drug target that is present in 11 bacterial species.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.31.657076v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Predicting Protein Function in the AI and Big Data Era.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Riccardo Percudani, Carlo De Rito</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new method for extracting functional information from protein sequences using deep learning and artificial intelligence .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> It is an exciting time for researchers working to link proteins to their functions. Most techniques for extracting functional information from genomic sequences were developed several years ago, with major progress driven by the availability of big data. Now, groundbreaking advances in deep-learning and AI-based methods have enriched protein databases with three-dimensional information and offer the potential to predict biochemical properties and biomolecular interactions, providing key functional insights. This progress is expected to increase the proportion of functionally bright proteins in databases and deepen our understanding of life at the molecular level.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40380914/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

