<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Tuesday, 27 May 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MassiveFold data for CASP16-CAPRI: a systematic massive sampling experiment</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Raouraoua, N., Lensink, M. F., Brysbaert, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the MassiveFold CASP16-CAPRI dataset, a systematic, large-scale sampling of both monomeric and multimeric protein targets, by exploiting maximal parallelization, we produced up to 8,040 models per target and shared them with the community for collaborative selection and</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Massive sampling with AlphaFold2 has become a widely used approach in protein structure prediction. Here we present the MassiveFold CASP16-CAPRI dataset, a systematic, large-scale sampling of both monomeric and multimeric protein targets. By exploiting maximal parallelization, we produced up to 8,040 models per target and shared them with the community for collaborative selection and scoring. This collective effort minimizes redundant computation and environmental impact, while granting resource-limited groups - especially those focused on scoring - access to high quality structures. In our analysis, we define an interface-difficulty classification based on DockQ metrics, showing that massive sampling yields the greatest gains on most of the challenging interfaces. Crucially, this classification can be predicted from the median ipTM scores of a routine AF2 run, enabling users to selectively deploy massive sampling only when it is most needed. Such targeted strategies dramatically cut computation time and resource use with minimal loss of accuracy. Finally, we underscore the persistent challenge of choosing optimal models from massive sampling datasets, emphasising the need for more robust scoring methods. The MassiveFold datasets, together with AlphaFold ranking scores and CASP and CAPRI assessment metrics, are publicly available at https://github.com/GBLille/CASP16-CAPRI_MassiveFold_Data to accelerate further progress in protein structure prediction and assembly modeling.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.26.653955v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">VIProDesign: Viral Protein Panel Design for Highly Variable Viruses to Evaluate Immune Responses and Identify Broadly Neutralizing Antibodies</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bylund, T., El Anbari, M., Schaub, A., Zhou, T., Rawi, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present Viral Protein Panel Design (VIProDesign), a computational tool for designing viral protein panels that address the high sequence diversity of rapidly evolving viruses.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Highly mutable viruses continuously evolve, with some posing major pandemic risks. However, standardized neutralization assays and up-to-date viral panels are often lacking, limiting evaluation of immunogens and identification of broadly neutralizing antibodies. Closing these gaps is essential for guiding effective countermeasure development. In this study, we present Viral Protein Panel Design (VIProDesign), a computational tool for designing viral protein panels that address the high sequence diversity of rapidly evolving viruses. VIProDesign uses the Partitioning Around Medoids (PAM) algorithm to select representative strains and applies the elbow-point method based on cumulative Shannon entropy to balance diversity and panel size. We used VIProDesign to generate optimized panels for Betacoronavirus, human immunodeficiency virus-1 (HIV-1), Influenza virus, Norovirus, and Lassa virus. The tool also supports customizable panel sizes, making it suitable for both resource-limited contexts and early-stage research. This flexible approach streamlines viral panel design across diverse pathogens. Although VIProDesign was originally developed for viral proteins, its underlying framework is broadly applicable to the selection of representative protein panels across diverse taxa, including bacterial species, toxins, and other biologically relevant protein families.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.21.654924v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SCOT+: A Comprehensive Software Suite for Single-Cell alignment Using Optimal Transport</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Baker, C. D., Pham, T. M., Demetci, P., Tran, Q.-H., Redko, I., Sandstede, B., Singh, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Optimal Transport+ is a single-cell alignment software suite for multi-omics experiments that allows biologists to align cells and features across disparate datasets produced by separate assays without the need for any correspondence.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> New advances in single-cell multi-omics experiments have allowed biologists to examine how various biological factors regulate processes in concert on the cellular level. However, measuring multiple cellular features for a single cell can be quite resource-intensive or impossible with the current technology. By using optimal transport (OT) to align cells and features across disparate datasets produced by separate assays, Single Cell alignment using Optimal Transport+ (SCOT+), our unsupervised single-cell alignment software suite, allows biologists to align their data without the need for any correspondence. SCOT+ has a generic optimal transport solution that can be reduced to multiple different OT optimization procedures, each of which provide state-of-the-art single-cell alignment performance. With our user-friendly website and tutorials, this new package will help improve biological analyses by allowing for more accurate downstream analyses on multi-omics single-cell measurements.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.21.655322v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DynaRNA: Dynamic RNA Conformation Ensemble Generation with Diffusion Model</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chen, H., Li, Z., Zhu, J., Hong, X., Zheng, Z., Cui, T., Sun, Y., Wei, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present DynaRNA, a diffusion-based generative model for RNA conformation ensemble, with broad implications potential in RNA structural biology, synthetic biology, and therapeutic development.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> RNA plays a wide variety of roles in biological processes. In addition to serving as the coding messenger RNA (mRNA), the vast majority of RNAs function as non-coding RNAs (ncRNAs), where their dynamic structural ensemble is critical for mediating diverse biological functions. However, traditional experimental techniques and molecular dynamics (MD) simulations face significant challenges in characterizing the conformational dynamics of RNA, due to inherent methodological limitations and high computing power cost. We herein presented DynaRNA, a diffusion-based generative model for RNA conformation ensemble. DynaRNA employs denoising diffusion probabilistic model (DDPM) with equivariant graph neural network (EGNN) to directly model RNA 3D coordinates, enabling rapid exploration of RNA conformational space. DynaRNA enables end-to-end generation of RNA conformation ensemble reproducing experimental geometries without the need for Multiple Sequence Alignments (MSA) information. Our results demonstrate that DynaRNA effectively and accurately generate tetranucleotides ensemble with lower intercalation rate than molecular dynamics simulations. Besides, DynaRNA has the ability to capture rare excited states of HIV TAR(trans-activation response) element, and recapitulate de novo folding of tetraloops. DynaRNA serve as a versatile and efficient platform for modeling RNA structural dynamics, with broad implications potential in RNA structural biology, synthetic biology, and therapeutic development.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.22.655453v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Artemisia Database: A Comprehensive Resource for Gene Expression and Functional Insights in Artemisia annua</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Taheri, A., Almeida-Silva, F., Zhang, Y., Fu, X., Li, L., Wang, Y., Tang, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the Artemisia Database (Artemisia-DB), a high-resolution expression atlas constructed from an extensive integration of publicly available RNA-seq datasets, which provides transcript- and gene-level abundance estimates across major tissues and includes functional annotations such as Gene Ontology (GO) terms</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Artemisia annua is renowned for producing artemisinin, a compound that revolutionized malaria treatment and holds therapeutic promise for other diseases, including cancer and diabetes. However, low natural yields of artemisinin remain a major bottleneck, necessitating a deeper understanding of the genetic and regulatory networks involved in its biosynthesis. Although several transcriptomic studies on A. annua exist, they are often limited in scope, and a comprehensive, tissue-resolved gene expression resource has been lacking. Here, we present the Artemisia Database (Artemisia-DB), a high-resolution expression atlas constructed from an extensive integration of publicly available RNA-seq datasets. The database provides transcript- and gene-level abundance estimates across major tissues and includes functional annotations such as Gene Ontology (GO) terms, KEGG pathways, and InterPro domains. As a case study, we investigated the coexpression profile of HMGR (3-hydroxy-3-methylglutaryl-CoA reductase), a key enzyme in the mevalonate pathway and an early step in artemisinin biosynthesis. Coexpression analysis in leaf tissue revealed a subset of Auxin Response Factor (ARF) transcription factors strongly correlated to HMGR. This finding suggests a potential regulatory link between auxin signaling and artemisinin biosynthesis, providing new hypotheses for functional validation. Artemisia-DB is freely accessible at https://artemisia-db.com and offers an interactive interface for exploring expression data, functional annotations, transcription factors, CRISPR targets, and more. By combining high-quality transcriptome data with regulatory and functional insights, Artemisia-DB serves as a valuable resource for the plant research community and facilitates deeper investigation into the transcriptional dynamics and specialized metabolism of A. annua.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.21.655314v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SCALE: Unsupervised Multi-Scale Domain Identification in Spatial Omics Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yousefi, B., Schaub, D. P., Khatri, R., Kaiser, N., Kuehl, M., Ly, C., Puelles, V. G., Huber, T. B., Prinz, I., Krebs, C. F., Panzer, U., Bonn, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present SCALE, an unsupervised algorithm for multi-scale domain identification in spatial transcriptomics data, which achieves state-of-the-art performance in single- and multi- scale domain detection on simulated and murine brain Xenium and MERFISH data, as well as patient</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell spatial transcriptomics enables precise mapping of cellular states and functional domains within their native tissue environment. These functional domains often exist at multiple spatial scales, with larger domains encompassing smaller ones, reflecting the hierarchical organization of biological systems. However, the identification of these functional domain hierarchies has been hardly explored due to the lack of appropriate computational methods. In this work, we present SCALE, an unsupervised algorithm for multi-scale domain identification in spatial transcriptomics data. SCALE combines neural graph representation learning with an entropy-based search algorithm to detect functional domains at different scales. It reaches state-of-the-art performance in single- and multi-scale domain detection on simulated and murine brain Xenium and MERFISH data, as well as patient-derived kidney tissue, highlighting its robustness and scalability across diverse tissue types and platforms. SCALE's ease of use makes it a powerful aid for advancing our understanding of tissue organization and function in health and disease.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.21.653987v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrating bioinformatics and machine learning to identify biomarkers of branched chain amino acid related genes in osteoarthritis.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xiao-Zhi ZhaYang, Yan-Xiong Chen, Wen-Da Hua, Zheng-Lin Bai, Yun-Peng Jin, Xing-Wen Zhao, Quan-Fu Liu, Zeng-Dong Meng</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study to identify potential biomarkers and mechanisms of action of BCAA-RGs in OA tissues.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Branched-chain amino acids (BCAA) metabolism is significantly associated with osteoarthritis (OA), but the specific mechanism of BCAA related genes (BCAA-RGs) in OA is still unclear. Therefore, this research intended to identify potential biomarkers and mechanisms of action of BCAA-RGs in OA tissues.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40420260/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">TaxaCal: enhancing species-level profiling accuracy of 16S amplicon data.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Qingrong Shen, Xiaoqian Fan, Yangyang Sun, Hao Gao, Xiaoquan Su</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A 16S rRNA amplicon sequencing method for microbiome composition analysis .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> 16S rRNA amplicon sequencing is a widely used method for microbiome composition analysis due to its cost-effectiveness and lower data requirements compared to metagenomic whole-genome sequencing (WGS). However, inherent limitations in 16S-based approach often lead to profiling discrepancies, particularly at the species level, compromising the accuracy and reliability of findings.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40419960/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Machine Learning Analysis of Whole-Blood Transcriptomics Data in Rheumatoid Arthritis Patients Treated with Adalimumab Identifies Predictive Biomarkers of Response.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chuan Fu Yap, Nisha Nair, Ann W Morgan, John D Isaacs, Anthony G Wilson, Kimme Hyrich, Guillermo Barturen, María Riva-Torrubia, Marta Gut, Ivo Gut, Marta E Alarcón Riquelme, Anne Barton, Darren Plant</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study to identify biomarkers predictive of adalimumab treatment response using whole blood transcriptomics, leveraging machine learning models for data mining followed by targeted statistical analysis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Tumor necrosis factor inhibitors (TNFi) have significantly improved rheumatoid arthritis (RA) management, yet variability in patient response remains a substantial challenge, with approximately 40% of patients discontinuing TNFi due to non-response or adverse effects. This study aimed to identify biomarkers predictive of adalimumab treatment response using whole blood transcriptomics, leveraging machine learning models for data mining followed by targeted statistical analysis.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40415615/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Gene set optimization for cancer transcriptomics using sparse principal component analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Frost, H. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We have developed a bioinformatics approach based on sparse principal component analysis (PCA) for optimizing existing gene set collections to reflect the pattern of gene activity in dysplastic tissue and have used this technique to optimize the Molecular Signatures Database (MSigDB) Hallmark gene set collection</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> A common approach for exploring pathway dysregulation in cancer involves the gene set or pathway analysis of tumor transcriptomic data. Unfortunately, the effectiveness of cancer gene set testing is limited by the fact that most gene set collections model gene activity in normal tissue, which can differ significantly from gene activity found within tumors. To address this challenge, we have developed a bioinformatics approach based on sparse principal component analysis (PCA) for optimizing existing gene set collections to reflect the pattern of gene activity in dysplastic tissue and have used this technique to optimize the Molecular Signatures Database (MSigDB) Hallmark collection for 21 solid human cancers profiled via bulk RNA-seq by The Tumor Genome Atlas (TCGA). Demonstrating the biological utility of our approach, the average survival association of gene set members is improved after optimization for nearly all cancer types and Hallmark gene sets.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.21.655279v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CSFeatures improves identification of cell type-specific differential features in single-cell and spatial omics data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Li, R., Li, Y., Hua, H., Li, Z., Zhang, Y., Yao, Y., Chen, S., Xu, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a new differential analysis method for single-cell RNA-seq data that effectively identifies cell type-specific genes that are highly expressed in the target cell population while being lowly expressed in all other populations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent advancements in single-cell and spatial omics have enabled the analysis of gene expression patterns in cells and tissues with unprecedented precision. Genes that exhibit significant variation across different cell types or states are typically closely linked to cellular functional states and disease processes. However, current differential analysis methods often fail to adequately account for cell type specificity when identifying differentially expressed genes. CSFeatures effectively addresses this issue by comprehensively considering the expression level, the smoothness of expression distribution, and the proportion of gene expression in the target cell population as well as in each of the other cell populations. Through comprehensive experiments on eight single-cell RNA-seq datasets, it was demonstrated that CSFeatures can effectively identify cell type-specific genes that are highly expressed in the target cell population while being lowly expressed in all other populations. Moreover, it was validated that the underlying principle of CSFeatures can be generalized to other omics data, including single-cell ATAC-seq, spatial transcriptomics, and spatial ATAC-seq data. Furthermore, the differential features identified by CSFeatures are enriched in pathways, functional categories, and regulatory regions that are highly relevant to the specific functional states of the target cell population, providing insights into its gene regulation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.21.655244v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">TypeSINE: Genome-Wide Detection of SINE Retrotransposon Polymorphisms Reveals Functional Variants Linked to Body Size Variation in Pigs</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zheng, Y., Yang, N., Chen, C., Zong, W., Chen, H., Shi, S., Wang, X., Murani, E., Reyer, H., Gao, B., Zhang, L., Wimmers, K., Song, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed TypeSINE to genotype SINE-RIPs from short-read sequencing data, a scalable method for genome-wide Sine-RIP analysis, advancing population genomics research.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The genotyping of Short Interspersed Element (SINE) retrotransposon insertion polymorphisms (RIPs) from large-scale sequencing data remains technically challenging. To overcome this limitation, we developed TypeSINE to genotype SINE-RIPs from short-read sequencing data. Analyzing 362 porcine genomes, we identified 749,835 SINE-RIPs (>85% accuracy), including 65,917 common variants (5-95% frequency). These showed uniform genomic distribution (~22/Mb), with 40% in introns. Population analyses based on SINE-RIPs revealed independent domestication of Asian and European pigs from local wild populations, followed by introgression. GWAS detected body size-associated regions using SINE-RIPs. We created pigRIPdb (82,227 curated SINE-RIPs) with browsing and visualization tools. The method's cross-species applicability is supported by conserved SINE evolutionary patterns, enabling polymorphism discovery of SINEs in livestock and humans. TypeSINE represents an efficient, scalable solution for genome-wide SINE-RIP analysis, advancing population genomics research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.21.653756v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Frustratingly easy domain adaptation for cross-speciestranscription factor binding prediction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ebeid, M. M., Balci, A. T., Chikina, M., Benos, P. V., Kostka, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce MORALE, a direct and competitive approach that leverages domain adaptation techniques to improve cross-species TF binding site prediction.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: Sequence-to-function models interpret genomic DNA and predict functional outputs, successfully characterizing regulatory sequence activity. However, interpreting these models remains challenging, raising questions about the generalizability of inferred sequence functions. Cross-species prediction of transcription factor (TF) binding offers a promising approach to enhance model generalization by leveraging sequence variation across species, and it can contribute to the discovery of a conserved gene-regulatory code. However, addressing systematic differences between the genomes of various species is a significant challenge. Results: We introduce MORALE, a framework that utilizes a well-established domain adaptation approach that is "frustratingly easy." MORALE trains on sequences from one or more source species and predicts TF binding on a single target species where no binding data is available. To learn an invariant cross-species sequence representation, MORALE aligns the first and second moments of the data-generating distribution between all species. This direct approach integrates easily into representation learning models with an embedding layer. Unlike alternatives such as adversarial learning, it does not require additional parameters or other model design choices. We apply MORALE to two ChIP-seq datasets of liver-essential TFs: one comprising human and mouse, and another comprising five mammalian species. Compared to both a baseline and an adversarial approach termed gradient reversal (GRL), MORALE demonstrates improved performance across all TFs in the two-species case. Importantly, it avoids a performance degradation observed with the GRL approach in this study. Furthermore, feature attribution revealed that important motifs discovered by MORALE were closer to the actual TF binding motif compared with the GRL approach. For the five-species case, our method significantly improved TF binding site prediction for all TFs when predicting on human data, surpassing the performance of a human-only model -- a result not observed in the two-species comparison. Overall, MORALE is a direct and competitive approach that leverages domain adaptation techniques to improve cross-species TF binding site prediction.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.21.655414v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">OptiK: An Entropy-Driven Framework for Optimal k-mer Size Selection for Bacterial Genomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gutierrez, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> OptiK, a novel alignment-free tool that evaluates the information richness of k-mer encodings across a range of k values to identify the optimal k for comparative analysis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> K-mer-based approaches have become fundamental (Zielezinski et al., 2017) to modern computational genomics, underpinning tools for genome assembly, metagenomic classification, variant calling, and phylogenetic analysis. Despite their ubiquity, selecting an appropriate k-mer size (k) is often made arbitrarily or heuristically, with little consideration for the underlying signal quality relative to a given dataset. Here, I introduce OptiK, a novel alignment-free tool that evaluates the information richness of k-mer encodings across a range of k values to identify the optimal k for comparative analysis. OptiK operates by constructing k-mer frequency matrices from genome collections, reducing their dimensionality via truncated singular value decomposition (SVD), and evaluating clustering structure through unsupervised metrics including the Silhouette coefficient, Calinski-Harabasz index, and Davies-Bouldin index. We validate OptiK on a curated dataset of 1044 Helicobacter pylori genomes with well-characterized population structure. OptiK robustly identifies k = 8 as the optimal k-mer size, yielding latent structures in UMAP space that align with fineSTRUCTURE-defined subpopulations without relying on prior labels or reference alignments. These results demonstrate that OptiK provides a reproducible, alignment-free strategy for optimizing k-mer resolution in bacterial comparative genomics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.21.655412v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

