<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Tuesday, 24 June 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MF-ProtDisMap: protein real-valued distance prediction with fusion of sequence and coevolutionary features</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, Y., Zhong, S., Xu, S., Wang, Z., Xin, C., Ni, F., Yan, F., Lu, X., Sun, S., Wang, H., Zhang, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel protein distance prediction framework based on MSA-derived coevolutionary features and ESM2 sequence-level features that outperforms state-of-the-art real-valued distance prediction methods.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The precise estimation of protein inter-residue distances is essential for high-accuracy protein structure modeling. Currently, the prediction methods are predominantly based on MSA-derived coevolutionary features or language model-based sequence features. To effectively leverage the strengths of both methods, this study developed MF-ProtDisMap (Multi-Feature Protein Distance Map), an integrative framework that effectively combines both feature types to achieve superior real-valued distance prediction. Briefly, MSA Transformer is employed to extract the coevolutionary features from protein multiple sequence alignments, whereas ESM2 is used to capture long-range interactions and sequence-level features. To reduce the computational cost while maximally representing fused feature information, we implement group pooling for feature dimensionality reduction and introduce Diff-former - a novel module combining a diffusion model with a triangular attention mechanism to enhance representation learning. MF-ProtDisMap achieved a MAE of 2.20 Angstrom and an RMSE of 3.40 Angstrom in the protein real-valued distance prediction task. The predicted distances can be converted into contact results, achieving ROC and PR values of 84.56% and 81.01%, respectively. These results demonstrate that MF-ProtDisMap outperforms state-of-the-art real-valued protein distance methods.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.17.660102v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Landscape and mutational dynamics of G-quadruplexes in the complete human genome and in haplotypes of diverse ancestry</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chantzi, N., Liew, S. W., Wijaya, A., Chan, C., Mouratidis, I., Amaral, E. d. O. S., Uzun, Y., Hemberg, M., Vasquez, K. M., Kwok, C. K., Georgakopoulos Soares, I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> G-quadruplexes are dynamic and functionally significant elements of the human genome and highlight new avenues for investigating their contributions to human disease and evolution.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> G-quadruplexes (G4s) are alternative DNA structures with diverse biological roles, but their examination in highly repetitive parts of the human genome has been hindered by the lack of reliable sequencing technologies. Recent long-read based genome assemblies have enabled their characterization in previously inaccessible parts of the human genome. Here, we examine the topography and genomic instability of potential G4-forming sequences in the gap-less, reference human genome assembly and in 88 haplotypes of diverse ancestry. We report that G4s are highly enriched in specific repetitive regions, including in certain centromeric and pericentromeric repeat types, and in ribosomal DNA arrays, and experimentally validate the most prevalent G4s detected. G4s tend to have lower methylation than expected throughout the human genome and are genomically unstable, showing an excess of all mutation types, including substitutions, insertions and deletions and most prominently structural variants. Finally, we show that G4s are consistently enriched at PRDM9 binding sites, a protein involved in meiotic recombination. Together, our findings establish G4s as dynamic and functionally significant elements of the human genome and highlight new avenues for investigating their contributions to human disease and evolution.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.17.660256v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AdaGenes: A streaming processor for high-throughput annotation and filtering of sequence variant data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kurz, N. S., Drofenik, K., Kornrumpf, K., Reuter-Jessen, K., Dönitz, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce the Adaptive Genes processor (AdaGenes), a sequence variant streaming processor designed to efficiently annotate, filter, LiftOver and transform large-scale VCF files.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The amount of sequencing data resulting from whole exome or genome sequencing (WES / WGS) presents challenges for annotation, filtering, and analysis. We introduce the Adaptive Genes processor (AdaGenes), a sequence variant streaming processor designed to efficiently annotate, filter, LiftOver and transform large-scale VCF files. AdaGenes provides a unified solution for researchers to streamline VCF processing workflows and address common challenges in genomic data processing, e.g. to filter out non-relevant variants to focus on further processing of the relevant positions. AdaGenes integrates genomic, transcript and protein data annotations, while maintaining scalability and performance for high-throughput workflows. Leveraging a streaming architecture, AdaGenes processes variant data incrementally, enabling high-performance on large files due to low memory consumption and seamless handling of whole genome files. The interactive front end provides the user with the ability to dynamically filter variants based on user-defined criteria. It allows researchers and clinicians to efficiently analyze large genomic datasets, facilitating variant interpretation in diverse genomics applications, such as population studies, clinical diagnostics, and precision medicine. AdaGenes is able to parse and convert multiple file formats while preserving metadata, and provides a report of the changes made to the variant file. AdaGenes is available at href{https://mtb.bioinf.med.uni-goettingen.de/adagenes}{https://mtb.bioinf.med.uni-goettingen.de/adagenes}.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.17.659929v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DeepSeq: High-Throughput Single-Cell RNA Sequencing Data Labeling via Web Search-Augmented Agentic Generative AI Foundation Models</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Al Dajani, S. A., Sanchez, A., Williams, J. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose the use of agentic foundation models with real-time web search to automate the labeling of experimental data, achieving up to 82.5% accuracy.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Generative AI foundation models offer transformative potential for processing structured biological data, particularly in single-cell RNA sequencing, where datasets are rapidly scaling toward billions of cells. We propose the use of agentic foundation models with real-time web search to automate the labeling of experimental data, achieving up to 82.5% accuracy. This addresses a key bottleneck in supervised learning for structured omics data by increasing annotation throughput without manual curation and human error. Our approach enables the development of virtual cell foundation models capable of downstream tasks such as cell-typing and perturbation prediction. As data volume grows, these models may surpass human performance in labeling, paving the way for reliable inference in large-scale perturbation screens. This application demonstrates domain-specific innovation in health monitoring and diagnostics, aligned with efforts like the Human Cell Atlas and Human Tumor Atlas Network.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.17.660107v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CellMentor: Cell-Type Aware Dimensionality Reduction for Single-cell RNA-Sequencing Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hevdeli, O., Petrenko, E., Aran, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present CellMentor, a novel supervised non-negative matrix factorization (NMF) framework that leverages labeled reference datasets to learn biologically meaningful latent spaces that can be transferred across related datasets.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell RNA sequencing (scRNA-seq) enables high-resolution profiling of individual cells, yet transforming this high-dimensional data into biologically meaningful representations remains a critical challenge. Current dimensionality reduction methods often fail to effectively balance technical noise reduction with preservation of cell-type-specific biological signals, particularly when integrating data across multiple experiments. Here, we present CellMentor, a novel supervised non-negative matrix factorization (NMF) framework that leverages labeled reference datasets to learn biologically meaningful latent spaces that can be transferred across related datasets. CellMentor employs a loss function that preserves cell type identity by simultaneously minimizing variation within known cell populations while maximizing distinctions between different cell types. We evaluated CellMentor against state-of-the-art dimensionality reduction and integration methods using controlled simulations of increasing difficulty and diverse real tissue types, each consisting of a labeled reference dataset and an unlabeled query dataset. In simulations, CellMentor maintained near-perfect clustering performance even under challenging conditions where other methods failed. In real datasets from PBMC, pancreas, and melanoma tissues, CellMentor demonstrated superior cell type separation while effectively mitigating batch effects. CellMentor also excelled at detecting rare cell populations and maintained reasonable performance when encountering novel cell types absent from reference data. With its robust batch correction capabilities and ability to preserve biologically meaningful cell type distinctions, CellMentor is particularly valuable for integrative analyses across multiple experiments.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.17.660094v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Epilogos: information-theoretic navigation of multi-tissue functional genomic annotations</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Quon, J., Reynolds, A. P., Tripician, N., Rynes, E. T., Teodosiadis, A., Kellis, M., Meuleman, W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present Epilogos, a simple information-theoretic framework for large-scale visualization, navigation and interpretation of functional genomics annotations, and apply it to over 2,000 genome-wide chromatin state maps in human and mouse.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Functional genomics data, such as chromatin state maps, provide critical insights into biological processes, but are hard to navigate and interpret. We present Epilogos to address this challenge by offering a simple information-theoretic framework for large-scale visualization, navigation and interpretation of functional genomics annotations, and apply it to over 2,000 genome-wide chromatin state maps in human and mouse. We construct intuitive visualizations of multi-tissue chromatin state maps, prioritize salient genomic regions, identify group-wise differential regions, and enable rapid similarity search given a region of interest. To facilitate usability, we provide a purpose-built web-based browser interface (http://epilogos.net) alongside open-source software for community access and adoption.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.18.660301v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Med14 phosphorylation shapes genomic response to GLP-1 Agonist</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Van de Velde, S., Yu, J., Evensen, K. G., Pakhlevanyan, E., Williams, A. E., Shaw, R. J., Montminy, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that long-term exposure to GLP-1 receptor agonist Exendin-4 stimulates the expression of beta cell specific CREB target genes with delayed kinetics, which leads to a broad genomic response with salutary effects on beta cell function.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Under feeding conditions, release of glucagon-like peptide (GLP-1) from intestinal L cells promotes insulin secretion and pancreatic beta cell viability. Binding of GLP-1 to its cognate receptor on the beta cell surface results in induction of the cAMP signaling pathway, leading to the protein kinase A (PKA) mediated phosphorylation of CREB and induction of CREB target genes. By contrast with the acute effects of this pathway on immediate early CREB target genes, which attenuate the cAMP-CREB response, sustained exposure of beta cells to the stable GLP-1 receptor agonist Exendin-4 stimulates the expression of beta cell specific CREB target genes with delayed kinetics. In a proteomic screen for transcriptional co-regulators that mediate the long-term effects of GLP-1 analogs, we identified Med14, a backbone subunit of the Mediator complex. Exposure to Exendin-4 stimulates Med14 phosphorylation at Ser983, corresponding to a conserved PKA recognition site (RRXS) that is located within an intrinsically disordered region of Med14. Phosphorylation of Med14 by PKA is essential for maintenance of enhancers that drive induction of beta cell-specific and diabetes-linked genes. Mutation of Med14 at Ser983 to alanine decreased beta cell numbers and repressed Exendin-4 induced gene regulation in primary mouse beta cells. Our work reveals how phosphorylation of a general transcription factor in response to GLP-1 analogs triggers a broad genomic response with salutary effects on beta cell function.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.17.660196v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">GENETIC, MORPHOLOGICAL, AND PHYTOCHEMICAL DIFFERENTIATION OF CARNAVAL - A NEW HOP CULTIVAR ADAPTED TO TROPICAL ENVIRONMENT</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lima, R. B. d. S., Delgado, C. H. O., Raffaele, M. V., Croce, V. P., Pinheiro, J. B., Marques, M. O. M., Marroquin, J. A. M., Martin, G. A. d. S., Oliveira, F. R., Silva, L. T. P., Mondin, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new tropicalized hop cultivar developed over six years of breeding and genetic improvement to adapt the crop to tropical conditions, Carnaval, is a promising alternative for domestic hop production in Brazil.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The global hops market generates approximately US$8 billion annually, primarily driven by beer production, the world's most consumed beverage. Despite this global demand, hop cultivation remains concentrated in temperate regions. Brazil, the third-largest beer consumer, imports over 99% of its hops. This study aimed to technically validate a new tropicalized hop cultivar, Carnaval, developed over six years of breeding and genetic improvement to adapt the crop to Brazilian conditions. A comparative analysis was conducted between Carnaval and two commercial varieties, Comet and Cascade, evaluating morphological traits, production potential, phytochemical composition via GC-MS, and genotyping using 10,541 SNP markers obtained through dd-GBS on the Illumina NextSeq 1000/2000 platform. Carnaval exhibited clear genotypic and phenotypic distinctiveness, with notable differences in cone, leaf, and branch morphology, as well as in alpha (14.0%) and beta (6.9%) acid content and essential oil concentration (0.61%), comprising over 35 components - mainly myrcene, -humulene, selinene, and caryophyllene. Productivity reached 478 kg/ha, comparable to commercial cultivars. Genetic analyses revealed high diversity (HS = 0.35), and population structure showed Carnaval grouping with its female parent Zeus but with distinct genomic characteristics. These findings confirm Carnaval's unique identity and its adaptation to tropical conditions, making it a promising alternative for domestic hop production in Brazil.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.17.660195v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Proteomic analysis reveals distinct germination strategies in polymorphic fruits of Haloxylon ammodendron</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, Y. Z., Chen, Z. W., Zhang, H. X., Wang, Z., Jiang, M. L., Yimingniyazi, A., Ren, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that fruit polymorphism in Haloxylon ammodendron is an adaptive bet-hedging strategy in variable environments, and its molecular and ecophysiological mechanisms are linked to germination and germination.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Fruit polymorphism, the production of multiple fruit morphotypes within a species, is an adaptive bet-hedging strategy in variable environments. However, researches about perennial plant and the adaptive mechanisms are not well understood. Haloxylon ammodendron, a constructive desert shrub, exhibits three fruit morphotypes: YY (yellow wings with yellow pericarp), YP (yellow wings with pink pericarp), and PP (pink wings with pink pericarp). We investigated their ecophysiological and molecular mechanisms through germination assays under salt and drought stress, combined with proteomic analysis. YP consistently showed the highest germination percentage (GP) and germination rate index (GRI) under stress, while PP displayed well germination success under low salinity and well-watered conditions (GP=32.7%, 36.7%; GRI=0.018, 0.020), but significantly impaired viability under stress (GP=12.7%, 12.0%; GRI=0.006, 0.006). Proteomics identified 721 differentially expressed proteins (DEPs), with the most (662) between YP and PP, linked to stress response and germination. YP's high abundance of stress-resistant proteins enabled rapid germination, whereas PP's delayed germination aligns with a persistent seed bank strategy. This polymorphism promotes niche differentiation: YP ensures quick colonization, PP enhances long-term resilience, and YY offers an intermediate strategy. Our findings reveal molecular-ecological adaptations in H. ammodendron, aiding targeted germplasm use for desert restoration.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.17.660099v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Phytofabrication of gold nanoparticles (AuNPs) via Wedelia chinensis Internodal extract: Characterization, Antioxidant, Antimicrobial, Hemolytic assay and Methylene Dye degradation properties</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pandey, B. C., Gupta, A., Kushwaha, R., Yaseen, M., Nath, G., Kumari, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We synthesized gold nanoparticles from Wedelia chinensis using aqueous internodal extract using a range of analytical techniques, such as energy-dispersive X-ray spectroscopy (EDS, EDX), X-Ray diffraction (XRD), transmission electron microscopy</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Wedelia chinensis is a medicinal herb of Asteraceae family. Preliminary phytochemical screening of internodal extract was done by using its aqueous, methanolic and ethanolic extracts. Preliminary phytochemical screening showed the presence of several pharmaceutically important compounds such as alkaloids, tannins, anthraquinons, saponins, flavonoids, terpenoids, phenols, etc. Aqueous internodal extract was used for the biosynthesis of gold nanoparticles. The characterization of gold nanoparticles were done by using a range of analytical techniques, such as energy-dispersive X-ray spectroscopy (EDS, EDX), X-ray diffraction (XRD), transmission electron microscopy (TEM), dynamic light scattering (DLS), Fourier transformed infrared spectroscopy (FTIR), scanning electron microscopy (SEM). The stability and other characteristics were ascertained using a UV-Vis spectrophotometer. The plasmon resonance occurred at the wavelength of 530 nm, thus showing the formation of gold nanoparticles. X-ray diffraction demonstrated the existence of Au-rich (fcc) phases in gold nanoparticles. FTIR analysis clearly indicated the participation of active constituents of internodal extract in the conversion of gold ions into AuNPs. DLS showed the size distribution (99.71 nm) of the suspended particles. There was variations in the sizes and shapes of the nanoparticles. The zeta potential of synthesized AuNPs was observed as -5.12 mV, thus showing the stability of the synthesized nanoparticles. In TEM study, uniform distribution of nanoparticles was seen with some agglomerates and average size of nanoparticles was observed in between 30-40 nm. The antioxidant potential of internodal extract was slightly higher than the synthesized gold nanoparticles. The green- synthesized AuNPs along with antibiotics demonstrated strong synergistic effect against several bacterial strains such as Escherichia coli and Bacillus subtilis. Gold nanoparticles showed good hemolytic activity against rat erythrocytes (90%). Gold nanoparticles also showed catalytic activity that reduces methylene blue at low concentration (1 mg).</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.17.660090v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">TaPGS1 Driven Flavonol Accumulation Delays Endosperm Cellularization and Enlarges Wheat Grains</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, J., Guo, X., Liu, X., Jin, Y., zhao, s., Liang, W., Li, M., Cheng, M., Dong, H., Chen, Q., Chen, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a flavonol regulatory module that links auxin distribution to endosperm development and seed size control in wheat.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Flavonols play a crucial role in seed development by regulating multiple physiological processes, including seed coat pigmentation, dormancy, fertilization, and endosperm formation. Notably, flavonols influence the polar transport of auxin, thereby affecting seed growth dynamics. In our previous work, we found that overexpression of the wheat bHLH transcription factor TaPGS1 results in increased grain size; however, the underlying mechanism remained unclear. In this study, metabolomic and transcriptomic analyses of TaPGS1 overexpressing wheat lines revealed enhanced flavonol accumulation and upregulation of key flavonol biosynthetic genes. Further investigations suggested that flavonol accumulation in the seed coat may disrupt auxin transport, leading to localized auxin buildup, delayed endosperm cellularization, and an increase in endosperm cell number. These changes collectively contribute to grain enlargement. Our findings uncover a TaPGS1 flavonol regulatory module that links auxin distribution to endosperm development and seed size control in wheat.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.17.660139v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Establishing Vibrio natriegens as a high-performance host for acetate-based poly-3-hydroxybutyrate production</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Politan, R. J., Della Valle, S., Pineda, L., Joshi, J., Euler, C., Flematti, G., Fritz, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new strain of Vibrio natriegens that can efficiently convert acetate into bioplastic poly-3-hydroxybutyrate .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Acetate can be a sustainable and renewable carbon source which holds significant promise for biotechnological production but is underutilized industrially due to limited microbial efficiency. Vibrio natriegens, recognized for exceptionally fast growth rates, represents a compelling host for developing efficient acetate-based bioprocesses. In this study, adaptive laboratory evolution significantly enhanced V. natriegens' ability to grow on acetate as the sole carbon source, achieving an 89% increase in growth rate. Genetic and transcriptomic analyses revealed key adaptations improving acetate uptake and metabolism via increased salt tolerance, boosted Pta/AckA pathway activity, and rewired quorum sensing. Further metabolic engineering and bioprocess optimization enabled the evolved strain to reach high cell densities and efficiently convert acetate into the bioplastic poly-3-hydroxybutyrate (PHB), with productivities up to 0.27 g/L/h and PHB accumulation reaching 45.66% of cell biomass. These advances position V. natriegens as a highly promising microbial platform for sustainable, scalable, and cost-effective biomanufacturing using acetate as green feedstock.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.23.661028v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Straightforward and Robust Enzymatic Reporter System for Anaerobic Thermophiles</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Galindo, J. L., Tjo, H., Conway, J. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that a simple and robust enzymatic reporter system for thermophilic anaerobic organisms that eliminates background activity from endogenous galactosidases .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Thermophilic anaerobic organisms, particularly species that can naturally degrade lignocellulosic biomass, show great promise for next generation bioprocessing. This has led to the development of nascent genetic systems to metabolically engineer these non-model organisms. However, a major challenge remains a lack of reliable reporter systems compatible with the combination of thermophilic and anaerobic growth conditions. Additionally, native glycoside hydrolases in these organisms limit the usefulness of traditional glycosidic enzyme reporters (e.g. LacZ) because of the native background activity present on para-nitrophenyl glucoside substrates. Here we describe the development of a straightforward and robust enzymatic reporter system that overcomes these challenges in Anaerocellum (f. Caldicellulosiruptor) bescii, an anaerobic, extremely thermophilic (Topt ~78 {degrees}C), lignocellulolytic bacterium. Our method is based on heterologous expression of hyperthermophilic archaeal galactosidases: an -galactosidase from Pyroccous furiosus (Pfgal), and a {beta}-galactosidase from Caldivirga maquilingensis (Cm{beta}gal). We show that these reporters produce strong, orthogonal signals on colorimetric substrates at high temperatures ([≥]90 {degrees}C) that eliminate background activity from endogenous galactosidases. We then demonstrate the capability of Cm{beta}gal, the stronger of the two reporters, to distinguish differences in levels of expression between A. bescii promoter sequences, which we verify through qRT-PCR. With its high signal to noise ratio and ease of use, this reporter system offers a reliable method for assessing protein expression in anaerobic thermophilic organisms, opening doors to improved genetic tools and metabolic engineering applications for industrial biotechnology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.23.661153v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Rational Development of Recombinant ELP Bolaamphiphiles for the Controlled Construction of Multifunctionalized Globular Protein Vesicles</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> de la Maza, L. E. L., Deb, B., Powers, J., Hood, A., Jang, Y., Denard, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce the bottom-up construction of globular protein vesicles that incorporate elastin-like peptide (ELP) bolaamphiphiles as transmembrane components.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Synthetic biology has enabled the development of new strategies for creating artificial cells that can sense and respond to external stimuli. This study introduces the bottom-up construction of globular protein vesicles (GPVs) that incorporate elastin-like peptide (ELP) bolaamphiphiles as transmembrane components. To enable this strategy, we devised a Golden Gate-based cloning strategy to streamline the design, expression, and purification of ELP bolaamphiphiles. Three ELP bolamphiphiles with varying structural complexity were developed, incorporating fluorescent proteins to facilitate visualization and characterization. The self-assembly of these bolaamphiphiles into GPVs was optimized by varying the molar ratios of recombinant building blocks. Structural characterization confirmed vesicle formation, dynamic light scattering analysis revealed size distributions dependent on modular complexity, and atomic force microscopy demonstrated that the vesicles exhibited MPa-range Young moduli, indicative of high mechanical robustness. Our findings demonstrate that multifunctional ELP bolaamphiphiles can be incorporated into GPVs, enabling modular vesicle engineering. This work provides a foundation for designing synthetic cells with customizable bi-functionalities and modularity, advancing compartmentalized systems.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.23.660353v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

