<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Tuesday, 05 August 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">varCADD: large sets of standing genetic variation enable genome-wide pathogenicity prediction.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lusiné Nazaretyan, Philipp Rentzsch, Martin Kircher</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A large unbiased dataset for variant prioritization and effect predictions is rare as most of the available databases do not represent a broad ensemble of variant effects and are often biased towards the protein-coding genome.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Machine learning and artificial intelligence are increasingly being applied to identify phenotypically causal genetic variation. These data-driven methods require comprehensive training sets to deliver reliable results. However, large unbiased datasets for variant prioritization and effect predictions are rare as most of the available databases do not represent a broad ensemble of variant effects and are often biased towards the protein-coding genome, or even towards few well-studied genes.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40759979/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Longitudinal image-based prediction of surgical intervention in infants with hydronephrosis using deep learning: Is a single ultrasound enough?</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Adree Khondker, Stanley Bryan Z Hua, Jethro C C Kwong, Kunj Sheth, Daniel Alvarez, Kyla N Velaer, John Weaver, Alice Xiang, Gregory E Tasian, Armando J Lorenzo, Anna Goldenberg, Mandy Rickard, Lauren Erdman</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed machine learning models for hydronephrosis that incorporate multi-visit inference across multiple institutions but did not demonstrate superiority over single-visIT inference in terms of AUROC or AUPRC.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The potential of deep learning to predict renal obstruction using kidney ultrasound images has been demonstrated. However, these image-based classifiers have incorporated information using only single-visit ultrasounds. Here, we developed machine learning (ML) models incorporating ultrasounds from multiple clinic visits for hydronephrosis to generate a hydronephrosis severity index score to discriminate patients into high versus low risk for needing pyeloplasty and compare these against models trained with single clinic visit data. We included patients followed for hydronephrosis from three institutions. The outcome of interest was low risk versus high risk of obstructive hydronephrosis requiring pyeloplasty. The model was trained on data from Toronto, ON and validated on an internal holdout set, and tested on an internal prospective set and two external institutions. We developed models trained with single ultrasound (single-visit) and multi-visit models using average prediction, convolutional pooling, long-short term memory and temporal shift models. We compared model performance by area under the receiver-operator-characteristic (AUROC) and area under the precision-recall-curve (AUPRC). A total of 794 patients were included (603 SickKids, 102 Stanford, and 89 CHOP) with a pyeloplasty rate of 12%, 5%, and 67%, respectively. There was no significant difference in developing single-visit US models using the first ultrasound vs. the latest ultrasound. Comparing single-visit vs. multi-visit models, all multi-visit models fail to produce AUROC or AUPRC significantly greater than single-visit models. We developed ML models for hydronephrosis that incorporate multi-visit inference across multiple institutions but did not demonstrate superiority over single-visit inference. These results imply that the single-visit models would be sufficient in aiding accurate risk stratification from single, early ultrasound images.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40758672/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Demulticoder: An R Package for the Simultaneous Analysis of Multiplexed Metabarcodes.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sudermann MA, Foster ZSL, Dawson SCL, Phan H, Fieland VJ, Martin FN, Chang JH, Grünwald NJ.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Demulticoder, an R package automating the use of DADA2 to analyze data derived from multiple metabarcodes, has novel capabilities that streamline data analysis by reducing the number of manual input steps and enabling automated processing of multiplexed metabarcode types.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Metabarcoding is a widely used approach relying on short DNA sequences to identify organisms present in a community. Although established workflows exist for analysis of single metabarcodes, these are cumbersome when multiple metabarcodes are required to study diverse taxa, such as those in plant- and soil-associated microbial communities, or when analyzing newly developed metabarcodes. To address this, we developed demulticoder, an R package automating the use of DADA2 to analyze data derived from multiple metabarcodes. It has novel capabilities that streamline data analysis by reducing the number of manual input steps and enabling automated processing of multiplexed metabarcodes. Additionally, demulticoder modularizes data processing to allow for iterative quality control and reformats data for downstream analyses. We also updated the oomycete-specific rps10 barcode database by revising the taxonomic information of select entries based on updates to the classifications within the NCBI Taxonomy database. A multiplex sequenced dataset consisting of ITS1 and rps10 metabarcodes from 162 samples and 12 controls was analyzed to compare demulticoder against a standard analysis workflow. Demulticoder required manual input at only four steps in comparison with 28 steps required for the standard workflow. Data quality and results from downstream exploratory, diversity, and differential abundance analyses were comparable to those from the standard workflow. Demulticoder is versatile and can be used to analyze datasets consisting of single metabarcodes, multiplexed and pooled metabarcode types, and different metabarcode types generated in separate experiments. The demulticoder R package, example datasets, and instructions are publicly accessible and open source.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40280882/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Defining and cataloging variants in pangenome graphs</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Salehi Nowbandegani, P., Zhang, S., Hu, H., Li, H., O'Connor, L. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a new definition of a 'genetic variant' and amethod, pantree, to identify them.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Structural variation causes some human haplotypes toalign poorly with the linear reference genome, leading to 'reference bias'. Apangenome reference graph could ameliorate this bias by relating a sample to multiplereference assemblies. However, this approach requires a new definition of a 'genetic variant.' We introduce a definition of pangenome variants and amethod, pantree, to identify them. Our approach involves a pangenome reference tree which includes all nodes(sequences) of the pangenome graph, but only a subset of its edges;non-reference edges are variant edges.Our variants are biallelic and have well-defined positions. Analyzing theMinigraph-Cactus draft human pangenome reference graph, we identified 29.6million genetic variants. Most variants (99.2%) are small, and most smallvariants (73.9%) are SNPs. 3.5 million variants (11.7%) have a reference allele which is not on GRCh38;these variants are difficult to detect without a pangenome reference, or withexisting pangenome-based approaches. They tend to be embedded within tangled,multiallelic regions. We analyze two medically relevant regions, around theHLA-A and RHD genes, identifying thousands of small variants embedded within severallarge insertions, deletions, and inversions. We release an open-source softwaretool together with a VCF variant catalogue.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.04.668502v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Reengineering the antigen optimization process for superior neoantigen vaccine design</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, G., Fu, Y., Buratto, D., Chan, K. C., Wang, H., Huang, Y., Huang, L., Zhou, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present AlphaVacc, a deep reinforcement learning framework that integrates Monte Carlo Tree Search with a Transformer-based network to optimize antigenic peptides.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Identifying effective neoantigen sequences is essential for enhancing anti-tumor immunity. However, the vast sequence space (>10^9 possible peptides) and limited accuracy of existing immunogenicity predictors hinder efficient vaccine design for patient-specific human leukocyte antigens (HLAs). We present AlphaVacc, a deep reinforcement learning framework that integrates Monte Carlo Tree Search with a Transformer-based network to optimize antigenic peptides. AlphaVacc outperforms previous generative models in binding-affinity prediction. Experimental validation of 12 AlphaVacc-generated variants of the BING-4 peptide confirmed that 11 showed increased HLA-A*02:01 binding and 7 elicited significant T cell responses. Further testing of 16 single-mutation peptides confirmed computational predictions for 15 candidates, exhibiting a remarkable success rate. AlphaVacc thus provides a powerful tool for designing neoantigen-based cancer vaccines and may accelerate personalized immunotherapies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.03.668363v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Ribo-ITP expands the translatome of limited input samples</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ghatpande, V., Paul, U., Howard, M. A., Cenik, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A proof-of-concept study to identify non-canonical translation events from low input samples which can be applied to cell and tissue types inaccessible to conventional methods.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In the last decade, an unexpectedly large number of translated regions (translons) have been discovered using ribosome profiling and proteomics. Translons can regulate mRNA translation and encode micropeptides that contribute to multiprotein complex formation, Ca2+ regulation in muscle, and signaling during embryonic development. However, identification of translons has been limited to cell lines or large organs due to high input requirements for conventional ribosome profiling and mass spectrometry. Here, we address this challenge using Ribo-ITP on difficult-to-collect samples like microdissected hippocampal tissues and single pre-implantation embryos. Comparative analysis of more than a thousand ribosome profiling datasets across a wide range of cell types revealed distinct sample specific expression patterns of the detected translons. To test the translational capacity of the identified translons, we engineered a translon-dependent GFP reporter system and detected expression of translons initiating at near-cognate start codons in mouse embryonic stem cells (mESCs). Mutating the translons in mESCs identified a small proportion that negatively impacted growth. Taken together, we present a proof-of-concept study to identify non-canonical translation events from low input samples which can be applied to cell and tissue types inaccessible to conventional methods.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.04.668486v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A complete and near-perfect rhesus macaque reference genome: lessons from subtelomeric repeats and sequencing bias</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, S., Xu, N., Lu, Y., Nie, Y., Li, Z., de Gennaro, L., Fu, L., Zhang, Z., Chen, J., Ma, K., Yang, X., Zhang, J., Schmitz, M. T., Antonacci, F., Bakken, T. E., Ventura, M., Phillippy, A. M., Sun, Q., Mao, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a near-perfect T2T assembly of the rhesus macaque genome, representing the highest base-level accuracy reported in a primate genome to date, and reveal previously unrecognized features of subtelomeric genome structure and evolution.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> A truly complete, telomere-to-telomere (T2T), and error-free reference genome remains a foundational resource--and long-standing goal--for unbiased comparative and functional genomics. While recent T2T assemblies of humans and other primates have made substantial progress, most still contain thousands of base-level errors, particularly within highly repetitive regions. Here, we present T2T-MMU8v2.0, a near-perfect T2T assembly of the rhesus macaque (Macaca mulatta), representing the highest base-level accuracy reported in a primate genome to date. By employing an optimized ONT-only assembly strategy, we identify subtelomeric satellite-rich regions as the principal bottleneck to improving assembly quality, owing to technological biases in long-read platforms and limitations in current hybrid assembly frameworks. We discover 268 previously unannotated repeat families and resolve ~8 Mbp of SATR satellite arrays, with over 99-fold enrichment in historically misassembled subtelomeric regions. These satellites form four distinct genomic architectures, each with unique SATR satellite composition, segmental duplication organization, and epigenetic signatures, distinct from the subtelomeric architectures observed in hominid genomes. Notably, in contrast to the largely gene-poor subtelomeric regions in African hominids, the SATR architectures in macaques harbor 58 actively transcribed genes, supported by open chromatin and expression data, suggesting gene innovation within these repetitive regions. Functionally, T2T-MMU8v2.0 improves read mappability and accuracy across sequencing platforms, and results in a 19% improvement of transcription start site enrichment scores and 5,821 additional chromatin accessibility peaks on average, thereby enhancing variant detection, regulatory annotation, and transcriptomic resolution in population genetics or single-nucleus studies. Together, this work establishes a new benchmark for genomics, offers a roadmap for resolving complex repetitive regions, and reveals previously unrecognized features of subtelomeric genome structure and evolution.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.04.668424v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A mathematical model for inflammation and demyelination in multiple sclerosis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jenner, A. L., Weatherley, G. R., Frascoli, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a minimal model for MS disease onset and progression driven by inflammation and demyelination.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Multiple sclerosis (MS) is an incurable life-long disease caused by the demyelination of neurons in the brain and spine. MS is often characterised by relapses in inflammation and demyelination, that are then followed by periods of remittance. Symptoms can be highly debilitating and there are still many open questions about the origin and progression of the disease. Mathematical modelling is well-placed to capture the dynamics of MS and provide insight into disease aetiology. In this work, we present a minimal model for MS disease onset and progression driven by inflammation and demyelination. The model dynamics are capable of describing a typical evolution of the illness, with changes from a healthy state to a diseased scenario captured by certain ranges of parameter values. Our model also describes the non-uniform oscillatory nature of the disease, born from a Hopf bifurcation due to the strength of the inflammatory response. In particular, using experimental data for Contrast Enhancing Lesions obtained from MS patients, we are able to reproduce some of the typical relapsing-remitting behaviours of this disease. We hope that the model presented here can serve as a baseline for more complex approaches and as a tool to predict possible evolutions of the disease.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.31.667831v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The telomere-to-telomere genome and lifestyle transcriptome profiling of Discula destructiva Redlin provide modern molecular and genomic context to a historical epidemic</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Niece, I. S., Beever, J. E., Moisa, S. J., Trigiano, R. N., Gwinn, K. D., Klingeman, W. E., Staton, M. E., Nowicki, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We assemble the first telomere-to-telomere, chromosome-scale genome for Discula destructiva Redlin and identify virulence-associated genes that likely contributed to its rapid spread across native dogwoods.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Fungal pathogens have dramatically altered forests worldwide, yet the mechanisms underlying their virulence remain poorly understood. From the 1970s to the early 2000s, dogwood anthracnose, caused by Discula destructiva Redlin, devastated flowering and Pacific dogwoods Cornus florida L. and C. nuttallii Aud., respectively). Despite the impacts of D. destructiva and other phytopathogens on forest ecosystems, genomic resources remain limited, hindering efforts to understand pathogenicity. The goal of this study was to evaluate the historical D. destructiva epidemic through a modern genomics lens by uncovering virulence-associated genes that likely contributed to its rapid spread across native dogwoods. We therefore utilized PacBio HiFi and Proximo Hi-C sequencing to assemble the first telomere-to-telomere, chromosome-scale genome for D. destructiva isolate AS111. The resulting 46.655 Mb assembly comprised eight chromosomes with an overall BUSCO completeness of 97.64%. We also identified 10,373 predicted gene models with an overall BUSCO completeness of 96.45%. To investigate gene expression across distinct life cycle phases, reproductive (sporulating) and vegetative (nonsporulating), we conducted RNA sequencing and identified 240 differentially expressed genes (padj < 0.05). GO enrichment revealed 162 upregulated genes during sporulation linked to plant cell wall degradation and sugar metabolism, whereas 78 downregulated genes were linked to electron carrier activity and redox balance. Among these 240 genes, 117 genes had predicted protein sequences that were also identified as a candidate virulence factor, including signal peptides, carbohydrate-active enzymes (CAZymes), and effectors, highlighting the role of sporulation-associated gene expression in D. destructiva virulence. Together, these findings suggest that the reproductive phase primes D. destructiva for host invasion and ecological persistence, which may influence its success as a forest pathogen.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.04.668385v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An loss of independence: genomic insights into a pest fruit fly-bacterial mutualism</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hansen, T. E., Corpuz, R. L., Simmonds, T. J., Aldebron, C., Geib, S. M., Mason, C., Sim, S. B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We characterized the genome of E. dacicola to investigate functional pathways that facilitate its obligate symbiosis with the olive fly and found that many of its genes are pseudogenized.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Tephritid fruit flies are invasive species of ripening fruit and vegetables. The olive fly (Bactrocera oleae) a specialist of olives, harbors a symbiotic gut bacterium Candidatus Erwinia dacicola, which is crucial to the olive flys exploitation of Oleaceae. This study characterized the genome of Ca. E. dacicola to investigate functional pathways that facilitate its obligate symbiosis. We used an extremely high-quality Ca. E. dacicola genome produced in tandem with an olive fly genome and paired that reference with transcriptomics. Of 4,675 genes, 1,783 (38.13%) were classified as pseudogenes. Overall, many pathways are complete, mostly associated with nutritional components. The symbiont also has genes encoding hypothetical proteins to degrade the phenolics in olives. Some pathways were heavily pseudogeneized, suggesting that Ca. E. dacicola have impaired functions in generalized metabolism, carbon utilization, pathogenicity, and signaling systems. Gene selection analyses in Ca. E. dacicola compared to related organisms, indicated positive selection on genes related to amino acid metabolism, carbon utilization, secretion and transport, and energy production. Our results indicate that the main service likely provided by Ca. E. dacicola is the production of amino acids and metabolism of oleuropein. The high pseudogenization rate implies incipient erosion of a formerly free-living symbiont.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.03.667778v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Defining the molecular determinants of titin kinase by analysing distantly evolved fish orthologues</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dorendorf, T., Gravenhorst, P., Mayans, O.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a classification of titin gene duplicates from fish into a and b isoforms (ttna and ttnb) that can assist future comparative studies.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Titin kinase (TK) is a pseudokinase specific to the striated muscle of vertebrates. Embedded within the contractile sarcomere and flanked by extensible regulatory tails, TK is thought to sense mechanical signals arising from muscle function. Studies on TK to date have focused narrowly on the human representative. To investigate if a pseudokinase character is a hallmark of TK, we have studied sequences of distantly evolved fish representatives and rationalized conservation patterns by resolving the crystal structure of TK from medaka isoform b. We find that sequence alterations in catalytic motifs involved in ATP and magnesium binding, respectively {theta}xK ({theta}: bulky hydrophobic residue) and EFG, are evolutionarily conserved in TK. Beyond the kinase domain, N- and C-terminal flanking tails show remarkable structural similarity across human and medaka orthologues, even though sequence conservation is limited to individual residues and short motifs: a YD-motif in the N-terminal tail; a [R/K]H[R/K]RYY sequence, a R-7x-R motif and position -2 of the latter in the C-terminal tail. We observe that motifs in the C-terminal tail consistently covary with the divergent functional motifs of TK, being part of its pseudokinase signature. In contrast, the proposed inhibition of the catalytic aspartate by a tyrosine residue from the P+1 loop is not significantly conserved outside mammals. Finally, based on these features and sequence clustering analysis, we propose a classification of titin gene duplicates from fish into a and b isoforms (ttna and ttnb) that can assist future comparative studies. A curated genomic annotation is provided here.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.04.668337v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Mechanism of Tethered Agonist Binding to an Adhesion G-Protein-Coupled Receptor</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Joshi, K., Miao, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel Peptide-Gaussian accelerated Molecular Dynamics method for all-atom enhanced sampling on ADGRs and its dynamic mechanism of tethered agonist binding to an ADGR .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Adhesion GPCRs (ADGRs) contain a GPCR autoproteolysis-inducing (GAIN) domain that is proximal to the receptor N-terminus and undergoes autoproteolysis at a highly conserved GPCR proteolysis (GPS) site to generate the N-terminal fragment (NTF) and transmembrane C-terminal fragment (CTF). Dissociation of NTF reveals a peptide tethered agonist (TA) which is responsible for the activation of the ADGRs. The NTF-bound ADGRs contain the encrypted TA that assumes a {beta}-strand configuration within the GAIN domain, which is markedly different from a U-shaped -helical configuration of TA in the active cryo-EM structures of ADGRs. However, how the TA dramatically changes its configuration and binds to the ADGR CTF remains unknown. In this study, we have performed all-atom enhanced sampling simulations using a novel Peptide-Gaussian accelerated Molecular Dynamics (Pep-GaMD) method on TA binding to the ADGRD1. The Pep-GaMD simulations captured spontaneous binding of the TA into orthosteric pocket of the ADGRD1 and its large conformational transition from the extended {beta} strand to the U-shaped helical configuration. We were able to identify important low-energy conformations of the TA in the binding pathway, as well as different active and inactive states of ADGRD1, in the presence and absence of the Gs protein. Therefore, our Pep-GaMD simulations have revealed dynamic mechanism of the TA binding to an ADGR, which will facilitate rational design of peptide regulators of ADGRs. Keywords: Adhesion G-protein-coupled receptors; tethered agonist; binding; Peptide-Gaussian accelerated Molecular Dynamics; enhanced sampling.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.04.668378v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Interpreting the Effects of DNA Polymerase Variants at the Structural Level</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Arnaudi, M., Krzesinska, K., Beltrame, L., Tiberti, M., Nilbert, M., Rohlin, A., Papaleo, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We systematically characterize 60,000 missense variants within the structured domains of DNA polymerase enzymes POLE and POLD1 and identify hundreds of potentially damaging genetic variants.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genetic variants in the DNA polymerase enzymes POLE and POLD1 can profoundly impact protein function, potentially disrupting stability, catalysis, DNA-binding, and interactions with other molecules. Understanding the structural basis of these functional impacts at the atomic level is essential for comprehensively interpreting variant effects. In this study, we utilize MAVISp, a modular, structure-based framework, to systematically characterize over 60,000 missense variants within the structured domains of POLE and POLD1. By integrating predictions related to protein stability, local misfolding, active-site or DNA and protein binding alterations, we provide a detailed structural interpretation of variants reported across various databases, including ClinVar, COSMIC, and cBioPortal. Moreover, we predict the functional consequences of numerous variants not found in disease-related mutation databases, creating a comprehensive catalogue for future studies. Our approach identifies structural mechanisms underlying established pathogenic mutations and flags over 300 Variants of Uncertain Significance (VUS) as potentially damaging, integrating them with already published experimental data (https://poled-db.org/). Collectively, our findings underscore the effectiveness of integrative structural analyses in elucidating the functional impact of genetic variants. This approach not only advances our understanding of protein variant effects at the atomic scale but also facilitates future studies aimed at variant classification, prioritization of variants for experimental studies, and functional interpretation in diverse biological contexts.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.04.653638v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Dissecting genomic regions and candidate genes for pod borer resistance and component traits in pigeonpea minicore collection</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Moghiya, A., Munghate, R. S., Sharma, V., Mishra, S. P., Jaba, J., Singh Gaurav, S., S. Gangurde, S., Dube, N., Rangari, S., Roychowdhury, R., Gangashetty, P., Sharma, H. C., Pandey, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified 14 significant marker-trait associations (MTAs) and candidate genes associated with pod borer resistance and three component traits in pigeonpea.</p><p style="font-size: 16px; text-align: justify;">Background: Pigeonpea is an important leguminous food crop primarily grown in tropical and subtropical regions of the world and is a rich source of high-quality protein. The biotic (weed, disease, and insect-pests) and abiotic stresses have significantly reduced the production and productivity of pigeonpea. Helicoverpa armigera, also known as pod borer, is a major pest in pigeonpea. A substantial investigations needed for comprehending the genetic and genomic underpinnings of resistance to H. armigera. Genetic improvement by genomics-assisted breeding (GAB) is an effective approach for developing high yielding H. armigera resistance cultivars. Still yet, no genetic markers and genes linked to this key trait have been detected in pigeonpea. In this context, a set of 146 pigeonpea minicore accessions were evaluated for four H. armigera resistance component traits namely pod borer resistance (PBR), days to 50% flowering (DF), days to maturity (DM), and grain yield (GY) for three consecutive seasons under field conditions. Results: Phenotypic data of pod borer resistance and component traits, along with the whole-genome resequencing (WGRS) data for 4,99,980 SNPs, were utilized to perform multi-locus genome-wide association study (GWAS) analysis. Two models (SUPER, and FarmCPU) detected 14 significant marker-trait associations (MTAs) for PBR and three component traits. The MTAs with significant effect were mainly identified on chromosomes CcLG02, CcLG04, CcLG05, CcLG07 and CcLG11. These MTAs subsequently delineated with key candidate genes associated with pod borer resistance (Probable carboxylesterase 15, microtubule-associated protein 5, FAR1-RELATED SEQUENCE, and Omega-hydroxypalmitate O-feruloyl transferase 4), days to maturity (RING-H2 finger protein ATL7 and Leucine-rich repeat receptor-like protein kinase), and grain yield (Secretory carrier-associated membrane protein, and Glutaredoxin-C5 chloroplastic). Conclusion: These research findings reported significant MTAs and candidate genes associated with pod borer resistance and component traits. Further lab-based pod bioassay screening identified four minicore accessions, namely ICP 10503, ICP 655, ICP 9691 and ICP 9655 (moderately resistance genotypes), showing least damage rating and larval weight gain %, compared to the susceptible checks. After validating the significant MTAs, the associated SNP markers can be effectively utilised in indirect selection, which offers potential gains for such quantitative traits with low heritability and can improve insect management more sustainably. The significant MTAs, candidate genes and resistant accessions, reported in this study may be utilized for the development of pod borer-resistant pigeonpea varieties.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.04.668421v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Structural and functional characterisation of a reconstructed ancestral strigolactone receptor</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tuckey, A. J., Marshall, A. C., Rothzerg, E., Bennett, T., Bond, C. S., Waters, M. T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We study the evolution of strigolactone perception in seed plants and show that the ancestral D14 protein shows substrate specificity, and propose a robust scaffold for the application of D14-type proteins in synthetic biology.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Strigolactones are phytohormones that regulate shoot branching and promote root interactions with arbuscular mycorrhizal fungi. In seed plants, strigolactone perception begins with the enzyme-receptor DWARF14 (D14), an /{beta}-hydrolase that is believed to have evolved via gene duplication from the karrikin receptor, KARRIKIN INSENSITIVE2 (KAI2). D14 and KAI2 are highly homologous at the sequence and structural levels, and both proteins bind and cleave similar butenolide compounds. Nevertheless, the two signalling pathways are distinct, as the activity of one receptor cannot compensate for the loss of the other. Here, we apply ancestral sequence reconstruction to generate a D14 protein representative of seed plants to study the evolution of substrate specificity, and to explore desirable traits for protein engineering. We describe the structure, as well as the in vitro and in planta activity of ancestral D14, showing that substrate specificity does not meaningfully differ from that of Arabidopsis thaliana D14. We also demonstrate that ancestral D14 shows higher recombinant yields, greatly increased thermostability, and enhanced catalytic activity relative to D14 from A. thaliana. This work provides insight into the evolution of phytohormone signalling and presents a robust scaffold for the application of D14-type proteins in synthetic biology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.03.668049v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Phased Potato Genome Assembly and Association Genetics Enable Characterisation of the Elusive H1 Resistance Locus Against Potato Cyst Nematodes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cheung, Y. W., Brown, L. H., Adams, T. M., Harrower, B., Kaur, A., McKenzie, G., Orr, J., Price, J., Singh, V., Smith, M., Bayer, M., Hein, I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We have reconstructed the entire H1 resistance locus in potato cultivars from Solanum tuberosum spp. to map the complete structure of the H1 interval, including recombination points at both the 5' and 3' ends of the interval.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The complexity of potato genetics, characterised by tetrasomic inheritance, has contributed to slower genetic gain in potato compared to other major crops. Disease resistance genes, often found in large clusters of highly similar paralogs and alleles, further complicate genetic studies. The H1 resistance locus, introgressed into potato cultivars from Solanum tuberosum spp. andigena, has been successfully used for over 60 years to control Globodera rostochiensis in Europe. Although previous genetic studies mapped this resistance to chromosome 5, the complete structure of the locus remained elusive. To reduce genomic complexity, we generated a dihaploid of the cultivar 'Athlete', DH4_Athlete, carrying the H1 resistance locus, and produced a phased haplotype representation of the H1 interval using Oxford Nanopore sequencing. Combined with RenSeq-based association genetics, this approach allowed us to reconstruct the entire H1 locus, including recombination points at both the 5' and 3' ends of the interval.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.04.667887v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Epistatic mutations in ISC metabolism synergize with cell cycle regulation and the PPP to enhance xylose fermentation and acetic acid tolerance in industrial yeast</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Santos, L. V. d., Palermo, G. C. d. L., Costa, P. E. d. S., Almeida, L. D., Carazzolle, M. F., Pereira, G. A. G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A yeast strain engineered with a xylose isomerase pathway and a double mutant lacking both ISU1 and ZWF1, which exhibited improved fermentative performance and increased tolerance to acetic acid.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Efficient utilization of complex biomass-derived sugars and tolerance to inhibitors are key requirements for the viability of lignocellulosic-based biorefineries. In this study, a two-stage evolution of an industrial yeast strain engineered with a xylose isomerase pathway yielded strain AceY.14, which exhibited improved fermentative performance and increased tolerance to acetic acid. Whole-genome sequencing of the evolved strain identified SNPs in ZWF1, a component of the pentose phosphate pathway (PPP), and in the G1 cyclin gene CLN3, both of which were functionally validated through CRISPR and reverse engineering. The zwf1E191D mutation reduced xylitol accumulation, alleviating inhibition of xylose isomerase and enhancing flux through the non-oxidative branch of the PPP, while the frameshift cln3T556fs mutation unexpectedly improved acetic acid tolerance and xylose consumption in the evolved strain, also affecting cell size and growth. Genome sequencing of AceY.14 also revealed a significant reduction in the xylA gene copy number, likely decreasing the metabolic burden associated with high xylose isomerase expression. A synergistic effect was observed in the double mutant lacking both ISU1 and ZWF1, further boosting xylose consumption rates. A diploid derivative (AceY-2n) demonstrated high productivity and robustness in fermentations using hydrolysates from various lignocellulosic feedstocks, highlighting the strain potential for industrial-scale applications. These findings reveal novel metabolic targets for strain optimization and offer valuable insights for the rational engineering of yeast platforms for sustainable biofuel and bioproduct production.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.04.668411v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">OmniCellAgent: Towards AI Co-Scientists for Scientific Discovery in Precision Medicine</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Huang, D., Li, H., Li, W., Zhang, H., Dickson, P., Zhan, M., Miller, J. P., Cruchaga, C., Province, M., Chen, Y., Payne, P., Li, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We develop a novel agentic AI, OmniCellAgent, to empower non-computational-expert users-such as patients and family members, clinicians, and wet-lab researchers-to conduct scRNA-seq data-driven biomedical research like experts.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The convergence of large language models (LLMs), AI agents, and large-scale omic datasets-such as single-cell omics, marks the arrival of a critical inflection point in biomedical research, via autonomous data mining and novel hypothesis generation. However, there is no specifically designed agentic AI model that can systematically integrate large-scale single-cell (sc) RNAseq (covering diverse diseases and cell types), omic data analytic tools, accumulated biomedical knowledge, and literature search to facilitate autonomous scientific discovery in precision medicine. In this study, we develop a novel agentic AI, OmniCellAgent, to empower non-computational-expert users-such as patients and family members, clinicians, and wet-lab researchers-to conduct scRNA-seq data-driven biomedical research like experts, uncovering molecular disease mechanisms and identifying effective precision therapies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.31.667797v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Single-cell omics: moving towards a new era in ischemic stroke research.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jieqiong Zeng, Huifen Zhou, Haitong Wan, Jiehong Yang</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a new single-cell omics approach for understanding the molecular mechanisms of IS and its potential impact on precision medicine.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Ischemic stroke (IS) is a highly complex and heterogeneous disease involving multiple pathophysiological events. A better understanding of the pathophysiology of IS will enhance preventive, diagnostic and therapeutic strategies. Despite significant advances in modern medicine, the molecular mechanisms of IS are still largely unknown. The high-throughput omics approach opens new avenues for identifying IS biomarkers and elucidating disease pathogenesis mechanisms. Single-cell omics enables a more thorough and in-depth analysis of the cellular interactions and properties in IS. This will lead to a better understanding of the onset, treatment and prognosis of IS. In this paper, we first reviewed the disease signatures and mechanisms research of IS. Subsequently, the use of single-cell omics to comprehend the mechanisms of IS was discussed, along with some recent developments in the field. To further delineate the upstream pathogenic alterations and downstream molecular impacts of IS, we also discussed the current use of machine learning approaches to single-cell omics data analysis. Particularly, single-cell omics is being used to inform risk assessment, early patient diagnosis and treatment strategies, and their potential impact on precision medicine. Thus, we summarized the role of single-cell omics in precision medicine. Despite the relative youth of the field, the development of single-cell omics promises to provide a powerful tool for elucidating the pathogenesis of IS.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40350018/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

