<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Thursday, 08 May 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">IKZF1 as a potential therapeutic target for dendritic cell-mediated immunotherapy in IgA nephropathy.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fei Peng, Chunjia Sheng, Jiayi He, Yena Zhou, Yilun Qu, Shuwei Duan, Yinghua Zhao, Jikai Xia, Jie Wu, Guangyan Cai, Lingling Wu, Chuyue Zhang, Xiangmei Chen</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify novel genes as potential diagnostic and therapeutic targets in IgAN and identify immune mechanisms in the disease.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Immunoglobulin A nephropathy (IgAN) is the most common primary glomerulonephritis globally and a major cause of renal failure. Immune dysregulation drives its pathogenesis. This study identifies novel genes as potential diagnostic and therapeutic targets, elucidating immune mechanisms in IgAN.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40336063/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">EVA: a Foundation Model Advancing Translational Drug Development in Immuno-Inflammation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fouche, A., Bruley, A., Corney, M., Marschall, P., Bouget, V., Duquesne, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present EVA, the first pre-trained foundation model in complex inflammatory diseases tailored to support drug development, which predicts drug responses with high precision at both cohort and patient levels.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Drug development is a lengthy and high-risk process, with most investigational drug candidates failing in phase II randomized clinical trials (RCT) due to insufficient efficacy. It makes early prediction of trial outcomes crucial for reducing attrition and guiding strategic decisions, especially in immunology and inflammation (I&I) diseases. Herein, we present EVA, the first pre-trained foundation model in complex inflammatory diseases tailored to support drug development. EVA learns generalizable patterns from large-scale data of cell biology and immunology, enabling superior predictive performance and generalization compared to traditional approaches. EVA is pre-trained on tens of millions of single-cell RNA-seq samples and tens of thousands of bulk RNA-seq samples from I&I diseases patients, enabling it to learn disease-relevant transcriptomic patterns in this therapeutic area. By fine-tuning EVA in few-shot settings on both preclinical (mouse) and clinical (human) data and harnessing its wide pre-training knowledge, EVA predicts drug responses in I&I with high precision at both cohort and patient levels, as illustrated by accurate forecasting of anti-TNF therapeutic activity in ulcerative colitis. By deciphering its decision process, we further highlight that EVA's ability to stratify patients based on predicted drug response can also be leveraged to discover drug response biomarkers as early as preclinical stages. EVA's applications in precision immunology encompass therapeutic target validation prior to clinical entry, identification of patient subpopulations most likely to benefit from treatment, and comparative efficacy analysis against competitor compounds. EVA's versatility makes it an invaluable tool for strategic decision-making throughout the drug development pipeline: by leveraging it to prioritize the most promising drug candidates and optimize RCT designs, it can contribute to reduce late-stage failures and accelerate the delivery of effective therapies. Overall, this work represents a significant advancement in utilizing a pre-trained foundation model for precision drug development in complex inflammatory diseases.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.02.651839v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Downscaling local distribution of cattle over Guadeloupe archipelago: an adapted method for disaggregating census data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dufleit, V., Guerrini, L., Gilbert, M., Da Re, D., Etter, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose an adaptation of the Gridded Livestock of the World methodology that accounts for the spatial autocorrelation in observed cattle distribution, thereby better capturing the specific characteristics of geographically limited areas such as the Guadeloupean archipelago.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Gridded livestock distribution datasets have been produced for several years and are used in various fields, including epidemiology, livestock impact assessment, and territory management. Those datasets are based on census conducted at national/sub-national scale which are then downscaled using machine learning algorithm and relevant spatial-explicit environmental predictors. The most known dataset of livestock disaggregated observations is the Gridded Livestock of the World (GLW), which produces global maps of livestock density at 10 km spatial resolution for several livestock species. Though this spatial resolution can be appropriate to describe livestock distribution at the global scale, it inherently leads to a coarse representation of breeding species density for smaller territories such as the Caribbean Islands. In this study, we propose an adaptation of the GLW methodology that accounts for the spatial autocorrelation in observed cattle distribution, thereby better capturing the specific characteristics of geographically limited areas such as the Guadeloupean archipelago. Cattle census data were collected for the 32 municipalities of the archipelago and associated to environmental predictors derived from remote sensing and land cover datasets. Together with the Random Forest (RF) algorithm used in the standard GLW methodology, we tested the performance of a Geographical Random Forest (GRF), a novel methodology allowing for taking into account the spatial autocorrelation of the response variable. The GRF algorithm demonstrated significantly better performance compared to the RF algorithm, albeit with longer processing times, and allowed us producing cattle distribution maps for the entire Guadeloupe archipelago at a spatial resolution of 225 m using both algorithms. The approach developed holds potential for application to other small territories, including other islands in the Caribbean.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.02.651856v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genome graphs reveal the importance of structural variation in Mycobacterium tuberculosis evolution and drug resistance</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Canalda-Baltrons, A., Silcocks, M., Hall, M. B., Theys, D., Chang, X., Viberg, L. T., Sherry, N. L., Coin, L., Dunstan, S. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A long-read pangenome reference graph and genome graph tool for long-term analysis of Mycobacterium tuberculosis's structural variants .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Structural variants (SVs) are increasingly recognized as key drivers of bacterial evolution, yet their role has not been explored thoroughly. This is due to limitations in traditional short-read sequencing and linear reference-based analyses, which can miss complex structural changes. Tuberculosis (TB), a disease caused by Mycobacterium tuberculosis (Mtb), remains a major global health concern. In this study, we harness long-read sequencing technologies and genome graph tools to construct a Mtb pangenome reference graph (PRG) from 859 high-quality, diverse, long-read assemblies. To enable accurate genotyping of SVs leveraging the PRG, we developed miniwalk, a tool that outperforms a traditional linear genome-based approach in precision for SV detection. We characterize patterns of structural variation genome-wide, revealing a virulence-associated ESX-5 deletion to be recurrent across the phylogeny, and fixed in a sub-lineage of L4. Systematic screens for additional genes that are recurrently affected by SVs implicated those related to metal homeostasis, including a copper exporter fixed in the widely distributed L1.2.1 sub-lineage. Lastly, we genotyped 41,134 isolates and found SVs putatively associated with resistance to various first and second-line drugs. These findings underscore the broader role of SVs in shaping Mtb diversity, highlighting their importance in both understanding evolution and designing strategies to combat drug-resistant TB.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.07.652570v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">cytoGPNet: Enhancing Clinical Outcome Prediction Accuracy Using Longitudinal Cytometry Data in Small Cohort Studies</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, J., Sun, L., Ready, N. E., Guo, W., Lin, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce cytoGPNet, a novel approach that harnesses extensive cytometry data to predict individual-level outcomes in immunological studies.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cytometry data, including flow cytometry and mass cytometry, are now standard in numerous immunological studies, such as cancer immunotherapy and vaccine trials. These data enable the monitoring of an individual's peripheral immune status over time, providing detailed insights into immune cells and their role in clinical outcomes. However, traditional analyses relying on summary statistics, such as cell subset proportions and mean fluorescence intensity, may overlook critical single-cell information. To address this limitation, we introduce cytoGPNet, a novel approach that harnesses extensive cytometry data to predict individual-level outcomes. cytoGPNet is designed to address four key challenges: (1) accommodating varying numbers of cells per sample; (2) analyzing the longitudinal cytometry data to understand temporal relationships; (3) maintaining robustness under the constraints of limited individual samples in immunological studies; and (4) ensuring interpretability to facilitate biomarker identification. We apply cytoGPNet to data from multiple diverse studies, each with unique characteristics. Despite these differences, cytoGPNet consistently outperforms other popular methods in terms of prediction accuracy. Moreover, cytoGPNet provides interpretable results at multiple levels, offering valuable insights. Our findings underscore the effectiveness and versatility of cytoGPNet in enhancing the analysis of cytometry data and its potential to advance immunological research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.01.651729v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Bonferroni's correction not Tukey's should be used for making pairwise comparisons in experiments with few replicates.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> zweifach, a.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Tukey's test is not as effective as Bonferroni's correction in controlling the false positive rate in tests of means, and should not be used in laboratory experiments.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Statistical tests can be used to help determine whether experimental manipulations produce effects. In tests of means, when more than two groups are compared the number of false positive results increases unless a correction is used. Tukey's test is thought to offer good control of the false positive rate and high statistical power when all pairwise comparisons are made. However, the number of replicates in laboratory experiments is often quite low, and small sample sizes can undermine assumptions underlying statistical methods. I used simulations to investigate how well Tukey's test controls the false positive rate when there are 3- 6 experimental groups and 2- 6 experimental replicates, conditions that span the range of typical values, and found that it generates too many false positives. I investigated 11 other approaches to controlling the false positive rate and found that none is as effective as the simple Bonferroni correction or offers much more power. I conclude that researchers should not make all pairwise comparisons using Tukey's test but instead use Bonferroni's correction on a limited number of pre-selected comparisons.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.01.651705v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PanScan: A Tool for Tertiary Analysis of Human Pangenome Graphs</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Uddin, M., Balan, B., Hanif, M., Hashmi, A., Kumail, M., Bineshaq, S., Alyazeedi, T., Naveed, R., Murtaza, M., Jamalalail, B., Elsokary, H., Alobathani, M., Mohamed, N., Kondaramage, D., Advani, D., Ibrahim, A., Shiyas, S., Sares, N., Alkhnbashi, O., Duplessis, S., Almarri, M., Nassir, N., Alsheikhali, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A pangenome tertiary analysis tool for human genomic diversity .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The genomic representation of populations across the globe is critical to ensuring a comprehensive and equitable human reference. Constructing a pangenome graph reference for different populations is the best approach to addressing local genomic diversities. Although major initiatives across continents are underway to construct pangenome graph references, the field lacks the necessary toolsets for tertiary analysis to characterize telomere-to-telomere (T2T) assemblies and the complexity of haplotypes. PanScan is a bioinformatics software package developed for human pangenome tertiary analysis. It includes multiple modules designed to detect duplicated gene sets from T2T assemblies, identify novel variants and sequences, as well as detect and visualize complex genomic regions through pangenome graph haplotype loops. We have used multiple pangenomes across different populations to assess the tertiary analysis and their accuracy. The tool is designed to streamline tertiary analysis and is compatible with multiple pangenome graph construction algorithms. PanScan is freely available on GitHub (https://github.com/CATG-Github/panscan), where users can provide human pangenome assemblies or VCF files as inputs for automated analyses through command-line operations on Linux systems.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.01.651685v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Regenie.QRS: computationally efficient whole-genome quantile regression at biobank scale</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, F., Wang, C., Wang, T., Masala, M., Fiorillo, E., Devoto, M., Cucca, F., Ionita-Laza, I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel and computationally efficient whole-genome quantile regression technique, Regenie, for biobank-scale GWAS data with genetic structure.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genotype-phenotype associations can be context-dependent and dynamic in nature leading to heterogeneity of genetic effects across different parts of the phenotype distribution. Quantile regression, an alternative to linear regression for continuous phenotypes, is particularly well suited for detecting and characterizing heterogeneous genotype-phenotype associations. Here we propose a novel and computationally efficient whole-genome quantile regression technique, Regenie.QRS, for biobank-scale GWAS data with genetic structure. Our approach first estimates the polygenic effect, and then incorporates this effect as an offset in the non-mixed quantile regression model. Our simulations demonstrate robust control of type I error and higher power to detect heterogeneous associations relative to linear regression in GWAS, and improved power over the marginal quantile regression tests. We present applications using data from the UK Biobank and the ProgeNIA/SardiNIA project, where we show the advantages of Regenie.QRS in identifying and characterizing heterogeneous genetic effects. To cite just one interesting example, using quantile regression we are able to show that even though variants at the G6PC2 locus increase glucose levels, their effects are much stronger at lower quantiles of glucose level distribution than at higher quantiles, showing that G6PC2 serves as a guardian against low glucose levels without driving dangerous hyperglycemia, which may explain the lack of association with diabetes risk.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.02.651730v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Beyond Natural Antibodies: Scaffold-Based Generation of Novel Anti-3CLpro Nanobody Nb01</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhan, Z., Luo, C., Xiong, J., Jiang, Y., Li, Y., Huang, F., Cai, Y., Liu, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel nanobody directed against the SARS-CoV-2 3CLpro, which has a higher affinity than the current best performing nanobodies (S43, bn03, R14, and 3-2A2-4) for most of the variants, including</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In the post-pandemic era, continuous mutations and persistent infections of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) pose a significant threat to global health, making the normalization of preventive measures imperative. Due to its high conservation, the 3CLpro is relatively stable across all these variants without significant changes, suggesting that drugs targeting this enzyme could be effective against all variant viruses. And compared to traditional antibodies, nanobodies show significant advantages against SARS-CoV-2 and its variants. Traditional antibodies often lose their inhibitory activity due to viral mutation. Nanobodies are characterised by their small size, high stability, high affinity for antigen binding, and high water solubility, which enable them to capture viruses quickly and effectively address the continuous mutations and persistent infections of SARS-CoV-2. In this research, we devised a strategy to produce nanobodies by utilizing a fragment-generating large language model, resulting in the identification of a nanobody named Nb01. Nb01 exhibits potent and broad-spectrum neutralizing activity against numerous SARS-CoV-2 variants. The nanobody Nb01, directed against the SARS-CoV-2 3CLpro, demonstrated efficacy against the majority of prevalent SARS-CoV-2 variants. Notably, it has a higher affinity than the current best performing nanobodies (S43, bn03, R14, and 3-2A2-4) for most of the variants, including the Alpha (27.5%), Gamma (29.7%), Omicron BA.2 (32.2%), BA.4/5 (81.2%), BF.7 (64.2%) and XBB (5.5%) variants. For other variants, Nb01 displayed affinities that were on par with these benchmark nanobodies. In summary, the exceptional specificity, low toxicity, robust stability, and extensive spectrum of Nb01 indicate its potential to be developed as a nanobody therapeutic for the management of SARS-CoV-2 infections and its diverse variants.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.06.652338v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Medlib: A Feature-Rich C/C++ Library for Exact Alignment of Nanopore Sequences Using Multiple Edit Distance</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rafeie, M., Vafaee, F., Faridani, O. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Medlib (Multiple Edit Distance Library) that offers both high performance and novel, biologically motivated features.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Pairwise sequence alignment is a cornerstone of bioinformatics, driving applications from variant calling to barcode demultiplexing. However, the advent of long-read, high-error sequencing platforms--such as Oxford Nanopore Technologies--demands alignment tools that not only deliver exact results but also accommodate the elevated noise inherent in these data. Here, we introduce Medlib (Multiple Edit Distance Library) that offers both high performance and novel, biologically motivated features. Medlib's threshold mode allows users to retrieve every alignment with edit distance k [≤] kth and optionally alignment score s [≥] sth, ensuring comprehensive discovery of all plausible matches. Counterintuitively, this exhaustive retrieval outpaces traditional minimum-error searches. Medlib further provides the first exact implementations of overlap-aware reporting and motif boundary treatment. Medlib supports multiple input formats, predefined and custom alphabets, affine-gap scoring applied retrospectively, built-in dedicated multithreading schemes, and user-defined function callbacks for immediate post-processing. Benchmarking against Edlib demonstrates a 1.6x speedup in minimum mode and even greater gains (2.5x) in threshold mode--while delivering up to twice as many alignments. Medlib thus offers a robust, feature-rich engine for exact pairwise alignment in modern, high-throughput sequencing workflows. Availability and Implementation: Source code, installation instructions and test data are freely available for download at https://github.com/Mehdi-Rafeie/Medlib, under the MIT license.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.01.651420v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Separable Spatial Single-cell Transcriptome Representation Learning via Graph Transformer and Hyperspherical Prototype Clustering</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fu, L., Wang, P., Xu, G., Lu, J., Sun, H., Wu, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose S3RL (Separable Spatial Single-cell transcriptome Representation Learning via Graph Transformer and Hyperspherical Prototype Clustering), a unified framework that integrates gene expression, spatial coordinates, and histological image features via a graph neural network and hyperspherical prototype</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Spatial transcriptomics enables the exploration of cell types, spatial domain organization, and cell-cell communication at tissue scale, serving as a powerful link between tissue morphology and molecular function. However, existing methods still struggle with limited spatial resolution, poor robustness in multi-slice alignment, and suboptimal ligand-receptor (L-R) detection in complex tissues. To address these challenges, we propose S3RL (Separable Spatial Single-cell transcriptome Representation Learning via Graph Transformer and Hyperspherical Prototype Clustering), a unified framework that integrates gene expression, spatial coordinates, and histological image features via a graph neural network and hyperspherical prototype-based separable representation learning. In spatial clustering tasks, S3RL improves the average Adjusted Rand Index (ARI) by nearly 120% on the Nanostring lung cancer dataset and over 26% on the 10X DLPFC dataset compared to state-of-the-art methods. For multi-slice spatial alignment, S3RL achieves an average ARI improvement of over 65.4% in partial brain slices and over 48.3% in complete slices relative to GraphST, highlighting its superior robustness and alignment consistency across heterogeneous spatial inputs. Moreover, S3RL reveals more biologically meaningful ligand-receptor signaling relationships, enhancing the interpretability of cell-cell communication patterns. Together, these results demonstrate S3RL's effectiveness in enhancing spatial resolution, cell type identification, and biological insight across diverse spatial transcriptomics datasets.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.01.651634v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Transcriptome and metabolome analyses reveal novel genetic targets for L-tryptophan overproduction in Corynebacterium glutamicum</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dong, Y., Gao, R., Qin, N., Liu, K., Liu, Y., Chen, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified two novel genetic targets for L-tryptophan synthesis in Corynebacterium glutamicum and identified the underlying mechanisms and potential bottlenecks for the industrial production of L- tryptophans.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Corynebacterium glutamicum is a promising microbial chassis for the industrial production of L-tryptophan, which has exhibited increasing demand due to its diverse applications and high market value. In previous work, we developed an L-tryptophan-overproducing C. glutamicum strain TR26 through multiple rounds of rational metabolic engineering. Here, comparative transcriptome and metabolome analyses were conducted between TR26 and its progenitor strain MB001 to reveal the underlying mechanisms and potential bottlenecks for L-tryptophan production in TR26. Furthermore, by systematically down- and up-regulating differentially expressed genes of interest, two novel genetic targets, glnK and sugR, were identified as being associated with L-tryptophan synthesis. Specifically, the repression of glnK and overexpression of sugR in strain TR26 increased the titer of L-tryptophan by 6.7% and 20.9%, respectively. Gene transcription profiling and intracellular metabolite analysis further suggested that the observed improvements in L-tryptophan synthesis could be attributed to optimized nitrogen transport and metabolism, efficient reallocation of cellular resources and enhanced supply of phosphoenolpyruvate (PEP). This study advances our understanding of the regulation mechanisms governing L-tryptophan synthesis in C. glutamicum and provides valuable insights for further optimization of industrial cell factories.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.07.652659v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

