<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Sunday, 14 September 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Semi-automated biofoundry workflows for sequence coevolution-guided isoprene synthase engineering.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Georgii Emelianov, Dong-Uk Song, Aporva Kamath, Hyeongseop Kim, Geunyeong Lee, Ha-Neul Kim, Kil Koang Kwon, Bong Hyun Sung, Dae-Hee Lee, Nathan J Hillson, Haseong Kim, Sanguk Kim, Hyewon Lee, Seung-Goo Lee</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed scalable enzyme engineering workflows tailored for biofoundry applications, focusing on isoprene synthase (IspS) - a critical rate-limiting enzyme in the isophene biosynthesis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Biofoundries serve as transformative platforms for accelerating the engineering of enzymes and microorganisms toward biomanufacturing. In this study, we developed scalable enzyme engineering workflows tailored for biofoundry applications, focusing on isoprene synthase (IspS) - a critical rate-limiting enzyme in the isoprene biosynthesis. By integrating computational mutation design based on sequence coevolution analysis and laboratory automation, we conducted three rounds of site-directed mutagenesis and screening. Approximately 100 genetic mutants were synthesized per round and these workflows can be easily scaled up to thousands without extensive optimization. Moreover, this approach enabled the rapid identification of IspS variants with up to 4.5-fold improvement in catalytic efficiency and simultaneously enhanced thermostability. Additionally, introducing the engineered IspS into Methylococcus capsulatus Bath improved methane-to-isoprene bioconversion, achieving a titer of 319.6 mg/l. These scalable workflows establish a robust framework for enzyme engineering within biofoundries. This provides a basis for the development of innovative biotechnological advancements.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40946078/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genetic integration with cell-specific nucleosome positioning resolves causal relationships underlying chromatin accessibility profiles</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, X., Robertson, C. C., Varshney, A., Manickam, N., Orchard, P. C., Laakso, M., Tuomilehto, J., Lakka, T. A., Mohlke, K. L., Boehnke, M., Scott, L. J., Koistinen, H. A., Collins, F. S., Parker, S. C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We map cell type-specific genetic effects on NFR accessibility (nfrQTLs) and nucleosome occupancy (24,623 nucQTL) and identify regulatory mechanisms underlying GWAS associations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cell type-specific chromatin accessibility QTL (caQTL) mapping is a promising approach for understanding genetic control of chromatin landscapes and identifying regulatory mechanisms underlying GWAS associations. However, current caQTL studies do not distinguish nucleosome-free regions (NFR) from positioned nucleosomes. Here, we leverage ATAC-seq fragment position and length to decompose accessible chromatin into NFRs and phased nucleosomes. With snATAC-seq from 281 muscle biopsies, we map cell type-specific genetic effects on NFR accessibility (76,027 nfrQTLs) and nucleosome occupancy (24,623 nucQTLs) across five skeletal muscle cell types. We find abundant colocalization between nucQTLs and nearby nfrQTLs and show that nfrQTLs are substantially more likely to causally influence nucQTLs than the other way around. Hundreds of nfrQTLs colocalize with GWAS signals for muscle-related traits, including grip strength, atrial fibrillation, and fasting insulin, and the majority of colocalizing signals mapped to credible sets overlapping the corresponding nfrPeak. This method elucidates how variants underlying caQTLs and GWAS signals exert their cis regulatory effects by initially modifying NFR accessibility and subsequently shaping broader chromatin landscapes, gene expression, and ultimately higher-level traits and disease.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.09.674883v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">dioscRi enables transferable prediction of clinical outcomes in multi-parameter cytometry data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Willie, E., Rao, S., Figtree, G., Yang, J. Y. H., Fazekas de St Groth, B., McGuire, H., Patrick, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present dioscRi, a transferable deep learning framework that integrates a maximum mean discrepancy variational autoencoder for normalization and de-noising, enhancing cross-dataset compatibility.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Multi-parameter cytometry technologies enable high-dimensional analysis of immune cell populations at single-cell resolution. Deep learning has emerged as a transformative tool for analyzing these datasets, but existing methods often struggle with transferability across datasets due to technical variability, batch effects and identification of biologically relevant cell populations, limiting their utility in clinical research. We present dioscRi, a transferable deep learning framework that integrates a maximum mean discrepancy variational autoencoder for normalization and de-noising, enhancing cross-dataset compatibility. Changes in cell type proportions and marker expression are identified by structuring these features within biologically or empirically derived cell type hierarchies. These hierarchies are incorporated directly into an overlapping group LASSO model, improving the prediction of clinical outcomes. When applied to a coronary artery disease study, dioscRi recapitulated several known immune associations. Benchmarking across multiple datasets demonstrated dioscRi ability to generalize and outperform existing methods, establishing it as a versatile and interpretable tool for cytometry data analysis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.08.675022v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SpatialZoomer: multi-scale feature analysis of spatial transcriptomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Li, X., Fan, Y., Han, Y., Guo, W., Huang, J., Yan, N., Chen, Z., Wu, Y., Miao, Y., Hou, L., Zhang, X., Chen, Z., Gu, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> SpatialZoomer, a spectral graph-based method that applies a set of low-pass filters to efficiently extract spatial molecular features from single-cell resolution spatial transcriptomics data at multiple resolutions or scales, including the single cell scale, the niche scale with dozens of closely interacting cells, and</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell resolution spatial transcriptomics (ST) provides a great opportunity to explore the complex cellular contents in different tissues. In this study, we introduce SpatialZoomer, a spectral graph-based method that applies a set of low-pass filters to efficiently extract spatial molecular features from ST data at multiple resolutions or scales, including the single cell scale, the niche scale with dozens of closely interacting cells, and the domain scale with spatially-organized cell contents. The corresponding "critical" scales can be automatically identified by partitioning a cross-scale similarity map. Results show that SpatialZoomer can identify disease-progression signals at specific scales in Alzheimer's Disease, and spatial context dependent cell subtypes in tumor microenvironment. The extracted multi-scale features also uncovered spatially heterogeneous niches in cancer. SpatialZoomer also takes advantage of high computational efficiency and low hardware requirements.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.08.674870v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Unmasking Vibrio paracholerae: Genomic Reclassification and Epidemiology</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Morgado, S., Fonseca, E., Vicente, A. C. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified 193 unique genomes from the Cholerae clade, including Vibrio paracholerae, Vibria metoecus, and Vibro tarriae, which were long misclassified as V. cholerae.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The genus Vibrio encompasses globally relevant pathogens, of which Vibrio cholerae is the best known due to its role in cholera. Closely related species within the Cholerae clade, Vibrio paracholerae, Vibrio metoecus, and Vibrio tarriae, were long misclassified as non-O1/O139 V. cholerae. The objective of this study was to analyze all 13,000+ available V. cholerae genomes in GenBank to determine the presence of species from the Cholerae clade. Genome-wide analyses using Mash, Whole-genome based Average Nucleotide Identity (gANI), and digital DNA-DNA hybridization (dDDH) reclassified 193 unique genomes as V. paracholerae, while V. metoecus and V. tarriae were not identified. Phylogenomic analyses revealed that V. paracholerae forms distinct but relatively homogeneous lineages, spanning clinical, environmental, and animal sources over a period of more than a century. Virulence profiling revealed the absence of CTX and TCP; however, most genomes exhibited other virulence factors, including hemolysins, RTX toxins, colix toxin, and a conserved type VI secretion system. Resistome analysis revealed several antibiotic resistance genes, many of which were incorporated into superintegrons, highlighting the role of V. paracholerae as a reservoir of resistance determinants. Importantly, two genomic markers exhibited high discriminatory power for species identification, providing robust tools for epidemiological surveillance. These findings underscore the relevance of V. paracholerae for clinical and epidemiological monitoring.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.08.674929v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Variable information across SNPs in GWAS data can cause false rejections of colocalisation which can be resolved by proportional colocalisation tests</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wallace, C., Robins, C., Johnson, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that fine-mapping inference is inaccurate for large meta-analysis GWASs, and propose an adaptation of proportional colocalisation, which avoids any fine- mapping inference, as an alternative to fine- mapping-based coloc.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Fine-mapping is now a standard post-GWAS analysis, but it has been shown to be potentially inaccurate for large meta-analysis GWAS. We show how this can be caused by variable amounts of statistical information between variants, e.g. due to variable sample sizes, which arise widely in meta-analyses, or due to variable imputation accuracy or call rate. This effect becomes more pronounced as sample sizes become larger, such that small differences in information available at different variants can lead to large errors in credible set predictions. Here, we show this inaccuracy propagates to the colocalisation method coloc, yielding confident but incorrect inference. We propose an adaptation of the proportional colocalisation test, which avoids any fine-mapping inference as an alternative to fine- mapping based colocalisation. This method was originally superseded by coloc because the null hypothesis is colocalisation, and because it can have low power in genetic regions with many variants. Our adaptation resolves this second issue, and simulations demonstrate that our approach accurately resolves false coloc inferences. However, the first issue remains, such that we suggest proportional colocalisation is used as an accompaniment to fine- mapping-based coloc. Our findings also emphasize the need for new methodologies in fine- mapping, particularly as GWAS sample sizes grow, to ensure accurate fine-mapping inference.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.08.674910v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Assembly of the complete horse genome reveals variants associated with reproduction and temperamental traits</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liu, Y., Wang, X., Zhang, H., Yang, R., Feng, Y., Feng, M., Yang, T., Li, Y., Qian, S., Zhao, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We assembled a complete telomere-to-telomere horse genome with a size of 2.80 Gb and a base accuracy of more than 99.9999%, and added 180.30 Mb of previously unresolved regions and 77 new genes compared with the previous TB-T2T.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> A high-quality horse genome assembly is crucial for genetic analysis of equine. Benefiting from long-read sequencing techniques and trio-binning strategy, we assembled a complete telomere-to-telomere horse genome (T2T-horse 1.0) with a size of 2.80 Gb and a base accuracy of more than 99.9999%, comprising a 34.11-Mb entire Y chromosome. T2T-horse 1.0 filled the gaps and corrected errors in previous assemblies, and added 180.30 Mb of previously unresolved regions and 77 new genes compared with the previous reference genome TB-T2T. We identified five types of centromeric satellites and revealed the features of the horse centromeric regions. We characterized the structure of the Y chromosome, and determined the borders of PAR transposed region, and annotated 59 novel genes on Y chromosome of T2T-horse 1.0. Using SNPs in the X-degenerate region of the Y chromosome of T2T-horse 1.0, we performed selection signatures analysis between indigenous horses and horses from intensively selected breeds, and detected the selected genes ARSF and NLGN4Y, which are closely associated with reproduction and temperamental traits, respectively.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.10.675315v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Shared molecular consequences of epigenetic machinery disruption in neuronal progenitors</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ouyang, J., Okay, K., Moller, K., Wang, K., Petursson, S., Pierce, T., Amirfallah, A., Davidovich, A., Anderson, K., Hansen, K. D., Bjornsson, H. T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that loss of DNA methyltransferases causes the most profound DNAm alterations, whereas other genes induced more subtle effects.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The Mendelian disorders of the epigenetic machinery (MDEMs) are an emerging cause of intellectual disability and growth abnormalities. Hippocampal neurons have been repeatedly found to be an effected cell type for these disorders. To directly probe the effect of disruption of the epigenetic machinery, we established a murine neuronal hippocampal differentiation model which allows rapid and systemic knockout (KO) of diverse epigenetic machinery (EM) factors. This system allows us to interrogate the consequences of each KO using ONT DNA methylation sequencing and RNA sequencing. Upon KO of KMT2A (a histone methyltransferase) or DNMT1 (a DNA methyltransferase), we observed a striking convergence in gene expression. In contrast, KO of DNMT1 lead to extensive changes in DNA methylation whereas KO of KMT2A had little-to-no effect on DNA methylation. In both models, KO accelerated neuronal differentiation, and the convergence in gene expression appeared linked to this differentiation state. The transcription factor GSX1 was differentially methylated, differentially expressed and ~20% of its known down-stream targets were differentially expressed genes. Extending our methylation analysis to 46 EM genes, we found that loss of DNA methyltransferases caused the most profound DNAm alterations, whereas other genes induced more subtle effects. Clustering of promoter DNAm changes revealed two EM subgroups that collectively showed a marked enrichment of interactions with the DNAm machinery. Through allele-specific phasing, we identified a differentially methylated region (DMR) at Zic4 that was shared across the entire group of EM KOs, as well as 1074 haplotype-specific DMRs. Together, our results highlight shared mechanisms across diverse EM mutations and provide new insight into the molecular etiology of MDEMs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.12.675897v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Programmable multiplexed phosphosignaling networks in bacteria</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Song, X., Lim, B., Yung, M. C., Hou, S.-L., Jiao, Y., Childers, W. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We establish a bottom-up framework for phosphosignaling circuits using two histidine kinases that converge on a shared response regulator (RR).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Natural bacterial two-component systems use crosstalk between histidine kinases to integrate multiple environmental cues, enabling complex multi-signal decisions. In comparison, most synthetic bacterial phosphosignaling circuits remain confined to single-input designs. Here, we establish a bottom-up framework for phosphosignaling circuits using two histidine kinases (2HKs) that converge on a shared response regulator (RR). As a proof of concept, we built a synthetic 2HK-1RR circuit in Escherichia coli that consists of a blue-light sensor (YF1) and an indole-3-aldehyde sensor, integrating opto- and chemo-signals through their shared RR. By using distinct dimerization tags on each HK, tuning HK activities, and reversing HK response modes, we program a spectrum of logic behaviors. These include ternary gates (an inverse count gate, a count gate, and two context gates), as well as a binary NAND gate. This work establishes fundamental phosphosignaling circuit design rules based on two-component systems that expand the scope of synthetic gene circuit design and pave the way towards complex, multi-input environmental biosensors and programmable therapeutics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.09.674673v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MiniCARbids: Minimalistic human binding domains specifically tailored to CAR T applications</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Teufl, M., Mayer, J., Holzer, K. E., Emminger, D., Sylvander, E., Baik, H., Schubert, F., Maniaki, E., Bader, U., Munoz-Lopez, A., Mittelstaet, J., Dobersberger, M., Buri, M. C., Sutzl, L., Salzer, B., Putz, E. M., Zajc, C. U., Lehner, M., Traxlmayr, M. W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce the miniCARbid engineering platform, enabling the generation of small, human antigen-binding domains with high potency in CAR T cells against virtually any target antigen, while being stable and aggregation-resistant.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Traditionally, chimeric antigen receptor (CAR) T cells employ single-chain variable fragments (scFvs) as binding entities. While scFvs represent a convenient option due to their broad availability, they also come with drawbacks, in particular their tendency to cluster and their relatively large size. Moreover, most scFvs used in the CAR field are of non-human origin, potentially causing immunogenicity. Therefore, we established an engineering platform for minimalistic CAR binding domains (miniCARbids), which combine several critical advantages: (i) human origin, (ii) small size, (iii) efficient expression in T cells and (iv) single-domain architecture, among others. We demonstrate that miniCARbids can be engineered to recognize various antigens with antibody-like affinities, while being stable and aggregation-resistant. When miniCARbids are incorporated into CARs, they induce high anti-tumor potency in both adapter and conventional CAR formats. Remarkably, CD22-directed miniCARbid-based CARs showed similar or even more efficient tumor clearance in leukemia-bearing mice when compared with a CAR comprising the clinically tested m971-1xG4S scFv. Together, we introduce the miniCARbid engineering platform, enabling the generation of small, human antigen-binding domains with high potency in CAR T cells against virtually any target antigen.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.09.675083v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Chromatin landscape of budding yeast acquiring H3K9 methylation and its reader molecule HP1</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shimizu, T., Fukuda, K., Shimura, C., Nakayama, J.-i., Oki, M., Shinkai, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A synthetic biology approach to explore the role of H3K9 methylation and HP1 in the epigenetic regulation of heterochromatin formation .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Histone H3 lysine 9 (H3K9) methylation and heterochromatin protein 1 (HP1) are well conserved heterochromatin epigenome and its reader molecule. However, the details of how importance of them in heterochromatin formation still remain unclear. One of the reasons is the redundancy aspects, as there are multiple H3K9 methylation reader molecules, including HP1, and HP1 itself functions as a hub recruiting various effector molecules. To overcome those issues, we took a synthetic biology approach and introduced H3K9 methylation and HP1 into budding yeast Saccharomyces cerevisiae, which does not have this system, and examined its impact on transcription and chromatin compaction. We observed that the mammalian H3K9 methyltransferase can induce genome-wide H3K9 di- and tri-methylation (H3K9me2,3) in the budding yeast which mainly in the gene body region, and HP1 accumulates over the H3K9 methylated regions. The forced expression of H3K9 methyltransferase and HP1 had little impact on transcription. Furthermore, Micro-C-seq analysis revealed no significant effects on the chromatin 3D structure. These results suggest that although H3K9 methylation and recruitment of HP1 play essential roles in the epigenetic regulation of heterochromatin, they alone are not sufficient to alter the higher-order chromatin structure, at least in the gene body regions in budding yeast.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.11.675748v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Uncovering the Design Rules for Sustainable Growth of Mineralized Mycomaterials</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Moss, D. H., Pear, O., Guio, J., Libonati, A., Ducat, D., Bay, K., Khakhar, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We demonstrate that surface display of an enzyme from sea sponges, silicatein , on the hyphae of the filamentous fungus Aspergillus niger enables mineralization of polysilicate and that this does not impact fungal growth.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mycomaterials, materials made from filamentous fungi, have several advantages over traditional materials, such as their genetic programmability and self-healing properties. However, their lack of mechanical strength and cost of production often constrain the applications they can be used in. In this work, we take inspiration from natural systems to overcome these challenges by elucidating design principles for mineralization-based enhancement of mechanical strength and synthetic lichen-based low-cost growth. We demonstrate that surface display of an enzyme from sea sponges, silicatein , on the hyphae of the filamentous fungus Aspergillus niger enables mineralization of polysilicate and that this does not impact fungal growth. We also show that this strategy can be extended to other silicatein variants and characterize how the degree of mineralization can be modulated. We then demonstrate that mineralization enhances the mechanical properties of the mycelium, including its tensile strength, modulus, and toughness. Finally, we show how these reinforced mycelia can be grown without external carbon sources using a synthetic lichen-based co-culture to facilitate low cost biomanufacturing. Together, our results lay the groundwork for the sustainable production of mineralized mycomaterials and create a new model system to study how mineralization impacts growth and mechanical properties.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.12.675881v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Enhanced cellular and transdermal delivery of the modified chromatin using gH625 cell-penetrating peptide</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, X., Liang, Y., Zonta, F., Park, J. H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A modified histone H2A fused with a cell-penetrating peptide for non-invasive transdermal drug delivery .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The transdermal drug delivery system has been used as a noninvasive route for drug administration, but often requires specialized equipment or invasive treatment. To examine recombinant synthetic chromatin as a non-invasive transdermal delivery vehicle, a modified histone H2A fused with a cell-penetrating peptide (gH625) was examined for its utility. Both in vitro porcine skin-based test and in vivo mice studies showed significantly enhanced passive skin penetration of gH625-modified chromatin compared to control groups. The gH625-H2A chromatin was examined as a delivery vehicle of EGF by displaying it at the carboxy terminus of histone H2B to enhance stability and skin penetration. The experimental results showed that the biological activities of EGF-displayed chromatin are comparable to those of the isolated EGF peptide in the HaCa-T cell line. The enhanced skin penetration capabilities of the gH625-H2A-modified chromatin offer great potential for developing non-invasive transdermal delivery of therapeutic proteins or peptides.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.11.675732v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Tumor-myeloid crosstalk drives therapy resistance in localized bladder cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Carvalho, F. L., Lee, J., Kalavros, N., Zhou, Y., Michaud, D., Stelter, I., Siddiqui, H., Stawiski, K., Bel, J. P., Wang, S., Garza, A., Bi, K., Park, J., Egan, J. M., Hirohashi, Y., Epstein, I., Vlachos, I., Jia, L., Kibel, A. S., Hirsch, M., Bellmunt, J., Guerriero, J. L., Mouw, K. W., Van Allen, E. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that cancer cells and macrophages cooperate to promote cisplatin resistance and identify macrophage-directed PARP14 inhibition as a novel therapeutic strategy to sensitize MIBC to cisplatoon.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Neoadjuvant cisplatin-based chemotherapy results in pathologic complete response for only a minority of patients with muscle-invasive bladder cancer (MIBC), and mechanisms of resistance and the effects of chemotherapy on the MIBC microenvironment remain incompletely understood. Here, we defined the single-cell and spatial transcriptomes of cancer and immune cells from MIBC patients with extreme responses to cisplatin-based chemotherapy. Tumors with persistent MIBC after chemotherapy harbored cancer cells expressing epithelial-to-mesenchymal programs that were associated with worse overall survival in independent cisplatin-treated bladder cancer cohorts. These cisplatin-resistant tumor cells were infiltrated by macrophages that upregulated tumor permissive programs defined by increased PARP14 expression in spatially-resolved multicellular niches. Macrophage reprogramming through PARP14 inhibition sensitized tumors to cisplatin via downregulation of tumor cell pathways implicated in resistance. Our results demonstrate that cancer cells and macrophages cooperate to promote cisplatin resistance and identify macrophage-directed PARP14 inhibition as a novel therapeutic strategy to sensitize MIBC to cisplatin.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.08.674862v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Prostate-specific EV capture with sufficient RNA yield to enable transcriptomic profiling</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yang, Y., Doo, J., Perez, D., Franzen, K., Nguyen, S., Mirabal, J., Mitsock, E., Punnen, S., Gaston, S., Gerard, T., Pollack, A., Cortizas, E., Chakrabortty, S. K., Skog, J., Johnson, J. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a novel immunocapture-based method for isolating prostate-derived EVs with high specificity and a quantitative framework to track EV stoichiometry and assess depletion efficiency and yield, enabling rigorous optimization of the workflow.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Prostate-specific antigen (PSA) screening has reduced prostate cancer (PCa) mortality but suffers from limited specificity, contributing to unnecessary biopsies and overdiagnosis of indolent disease. There is a critical need for biofluid-based biomarkers that improve the precision of PCa detection. Extracellular vesicles (EVs) offer a promising platform for noninvasive diagnostics, as they carry molecular cargo reflective of their tissue of origin. The ExoDx Prostate IntelliScore (EPI) test, a urine-based EV assay, is currently the only commercial EV diagnostic for clinically significant (cs)PCa, but its performance may be constrained by contamination from renal and bladder-derived EVs. To address this, we developed Exosome Diagnostics Depletion and Enrichment (EDDE), a novel immunocapture-based method for isolating prostate-derived EVs with high specificity. By targeting Prostate Specific Membrane Antigen (PSMA), we optimized EDDE to selectively enrich prostate EVs from post-DRE urine and recover sufficient RNA for transcriptomic analysis. Throughout development, we implemented a quantitative framework to track EV stoichiometry and assess depletion efficiency and yield, enabling rigorous optimization of the workflow. Our findings demonstrate that PSMA EDDE enriches prostate-specific EVs and yields RNA quantities compatible with sequencing. This platform enhances the specificity of EV-based biomarker discovery and holds promise for determining if tissue-specific EV biomarkers contribute advantages over bulk EVs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.08.674937v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Serial Imaging of Tumor and microEnvironment (SITE) platform for live-cell ex vivo modeling of primary and metastatic cancer dynamics.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Murthy, V., Makkawi, R., Anderson, F., Halsey, C., Manalo-Hall, E., Srikanth, S., Tangkilisan, G., Zheng, T., McGann, J., Birtwistle, M. R., Copperman, J., Davies, A. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> SITE, Serial Imaging of Tumor and microEnvironment, a spatially and temporally integrated platform combining ex vivo culture, biosensors, live imagin,.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding cancer onset and progression is extremely challenging, in part due to experimental limitations in measuring and interpreting key signaling and tumor-host interactions that determine cancer behavior over time, across cell and tissue scales. Here we developed SITE, Serial Imaging of Tumor and microEnvironment, a spatially and temporally integrated platform combining ex vivo culture, biosensors, live imagin,, and computation. Applied to modeling primary and lung metastatic breast cancer, SITE revealed tissue-specific tumor-host interactions and ERK signaling patterns linked to distinct single-cell behaviors. We found that multicellular niche formation involved active protrusion and cell-cell contact driven by both cancer and host cells. Mathematical modeling showed ERK signaling was co-influenced by neighboring cancer and host cells. Paracrine signaling among cancer cells increased signaling in a cluster size-dependent manner, while disruption of cancer-cancer signaling loops amplified tissue-specific differences in tumor architecture. Applied specifically to breast cancer, we demonstrated the utility of the SITE platform, enabling quantitative exploration of ex vivo signaling and tumor-host interaction dynamics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.08.674915v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

