<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Monday, 08 September 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">BLASE: Bulk Linkage Analysis for Single Cell Experiments - Teasing Out the Secrets of Bulk Transcriptomics with Trajectory Analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> McCluskey, A., Kettlewell, T., Smith, A. M., Kundu, R., Gunn, D. A., Otto, T. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: scRNA-seq experiments can capture cell process trajectories. Bulk RNA-seq is more practical, however does not have the granularity to elucidate cell-type specific trajectories. Deconvolution methods can estimate cell-types in RNA-seq data, but there is a need for methods characterising their pseudotime. Results: We show that our method, BLASE, can identify the progress of an RNA-seq sample through a trajectory in a scRNA-seq reference. Conclusion: BLASE can be used to a) annotate scRNA-seq data from existing RNA-seq, b) identify progress of RNA-seq data through a process based on scRNA-seq data, and c) be used to correct developmental differences in RNA-seq differential expression analysis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.03.673925v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Ankh-score produces better sequence alignments than AlphaFold3</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Malec, J., Rusen, K., Golding, G. B., Ilie, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein sequence alignment is one of the most fundamental procedures in bioinformatics. Due to its many downstream applications, improvements to this procedure are of great importance. We consider two revolutionary concepts that emerged recently as candidates for improving the state-of-the-art alignment methods: AlphaFold and protein embeddings. Alignment improvements can come from structural alignment of AlphaFold-predicted structures or scoring based on similarity of protein embedding vectors, resp. Thorough comparison on many domains from BAliBASE and CDD demonstrates that the Ankh-score method produces much better sequence alignments than structural alignments using US-align of AlphaFold3-predicted structures. Both are better than the traditional method using BLOSUM matrices. This suggests that Ankh embedding vectors may possess certain information that is not available in the AlphaFold3-predicted structures.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.03.674029v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identification of Engineered IMGT Fc Variants in IMGT/mAb-DB Therapeutic Antibodies and Fusion proteins</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Manso, T., Sanou, G., Nousias, C., Maalem, I., Boutin, F., Giudicelli, V., Duroux, P., Lefranc, M.-P., Kossida, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Monoclonal antibodies (mAbs) and fusion proteins for immune applications (FPIA) play a crucial role in treating autoimmune diseases and cancers by targeting cell-surface proteins and triggering multiple immune mechanisms. These functions are mediated by the fragment crystallizable (Fc) region of mAbs and fusion proteins, whose interaction with Fc gamma receptors (FcyRs) can be modulated through Fc amino acid (AA) engineering. To address this, we developed the IMGT/FcVariantsExplorer tool (https://www.imgt.org/fcvariantsexplorer/) to identify AA changes within the Fc region in mAb and fusion proteins sequences from IMGT/2Dstructure-DB, the AA sequence database of IMGT, the international ImMunoGeneTics information system. We used the IMGT nomenclature of engineered Fc variants involved in antibody effector properties and formats, applying a standardized classification in five categories: Effector, Half-life, Physicochemical properties, Structure, and Hybrid. We analyzed sequences of 1,107 mAbs and fusion proteins, identifying 483 entries with Fc AA changes, resulting in 211 unique Fc variants in the dataset. We also used web scraping to retrieve associated biological data from literature. All data have been integrated into IMGT/mAb-DB, with links to sequences in IMGT/2Dstructure-DB, enabling users to query Fc variants by their Category or Effect. This curated dataset reveals key trends in antibody engineering.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.03.673706v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Computational study of heme b595 to heme d electron transfer in E. coli cytochrome bd-I oxidase</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Siddeeque, R., Etcheverry, B., Cattin, C., Deviers, J., Melin, F., Hellwig, P., Cailliez, F., de la Lande, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cytochrome bd is a distinctive family of terminal oxidases present in the respiratory chains of many prokaryotes. Despite its biological importance, the redox chemistry of these proteins remains poorly understood, largely due to the presence of two b-type hemes and one d-type heme. Here, we report the first computational study of inter-heme electron transfer in the cytochrome bd family. We performed 10 s of molecular dynamics simulations of E. coli cytochrome bd-I embedded in realistic membranes, combined with quantum chemical calculations to estimate the thermodynamic parameters of electron transfer from heme b595 to heme d within the framework of Marcus theory. We further identify the respective contributions of the hemes, protein scaffold, lipid bilayer, water, and counterions to the driving force and reorganization energy. The inter-heme electronic coupling was calculated using the Projected Orbital Diabatization (POD) method in a hybrid Quantum Mechanics/Molecular Mechanics scheme and rationalized through electron transfer pathway analysis. This study provides fundamental insights into how electron transfer steps are orchestrated in the catalytic cycle of E. coli cytochrome bd-I.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.03.673948v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">In Silico Design of APOE Œµ4 Interaction Inhibitor Peptides for Alzheimer's Disease</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ji, J., Han, E., Park, J., Son, A., Kim, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;">Background: Protein-protein interactions (PPIs) are essential for cellular functions, and their aberrant formation contributes to neurodegenerative diseases. AD, characterized by its complex pathogenesis, poses an increasing societal burden with population aging. The APOE e4 allele represents the strongest genetic risk factor for late-onset AD, yet its pathological mechanisms remain incompletely understood. Methods: We employed artificial intelligence-driven peptide design to elucidate the pathological interaction between APOE e4 and amyloid precursor protein (APP). Using advanced AI algorithms, we identified critical binding interfaces and designed mimetic peptides targeting the APOE e4 and APP interaction site. Peptide efficacy was evaluated through comprehensive molecular dynamics simulations. Results: Our analysis revealed key residues mediating APOE e4 and APP binding. The designed inhibitory peptides demonstrated stable interaction with target sites, favorable binding energetics, and sustained structural integrity throughout simulations. Lead candidate effectively disrupted APOE e4 and APP complex formation in silico. Conclusion: This study presents a novel AI-powered approach for developing PPI-targeted therapeutics against AD. Our computationally validated peptide inhibitors offer promising therapeutic candidates that warrant experimental validation. These findings demonstrate the potential of integrating artificial intelligence with structural biology for accelerating drug discovery in neurodegenerative diseases.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.03.673973v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Streamlining Spatial Transcriptomics for Human Kidney Tissue</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Vo, S., Meadows, K., Do, H., Chapman, K., Andonegui, G., Muruve, D. A., Pham, T., Chun, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell spatial technologies have emerged in recent years, enabling tissue architecture and organization characterization at unprecedented resolution. However, computational analysis of spatial transcriptomics data is often a bottleneck for scientific discoveries in the absence of a dedicated bioinformatician. Here, we describes a workflow to annotate cell types from a dataset generated using NanoString CosMx single-cell resolution spatial transcriptomic technology, enabling a comparison between healthy kidney biopsies and diseased tissue. We validated our pipeline's accuracy with both gene expression analysis and pathological changes associated with biopsy-proven diabetic kidney disease (DKD). Through precise cell type annotation, we observed significant changes in the proportions of podocytes and immune cells in DKD, with DKD tissue showing regional enrichment of immune cells and differential gene expression. Notably, injured proximal tubules had the expected increased expression of HAVCR1 and VCAM1 and genes associated with diabetes, including IL18, ITGA3 and ITGB1. The entire workflow, now fully integrated into the BioTuring SpatialX (Lens V2.0), is available as a platform designed for users with no formal bioinformatics training.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.02.673834v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Modeling the Relationship between the Capsid Spike Protein Stability and Fitness in {varphi}X174 Bacteriophage</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sapozhnikov, Y., Van Leuven, J. T., Patel, J. S., Miller, C. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;">Background: Protein function depends on the stable folding or binding of peptide chains, and the extent to which an amino acid substitution disrupts this stability can be a strong predictor of the mutation's impact on the protein's function and the organism's fitness. This study seeks to understand this relationship in bacteriophage {varphi}X174 using an experimental technique of deep mutational scanning and computationally predicted protein stability. Results: Analyzing the viability data using a newly developed multistage binomial model confirms that highly destabilizing mutations predict inviability. For single-site variants, the model predicts that {Delta}{Delta}G of folding greater than ~3 kcal/mol and {Delta}{Delta}G of binding greater than ~6 kcal/mol cuts the probability of survival by half of the maximum. The maximum probability, even for minimal {Delta}{Delta}G values, plateaus at around 87%, reflecting the influence of unobserved factors. Fitting of double-site variants data shows similar results. In contrast, a linear regression analysis of viable mutants reveals that the effect of protein stability is too weak to be a reliable predictor of fitness. Conclusion: In this study, we introduce a novel modeling technique to explore the relationship between protein stability and survival outcomes. While large destabilization often indicates inviability, most destabilizing mutations exert only small to moderate effects and do not reliably predict survival. Although protein stability is a crucial biophysical property influencing biological systems, it represents just one of multiple factors that contribute to fitness and survival.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.02.673881v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Hyperbolic Wavelet Interaction Network (HyWinNet): Multi-Scale Geometric Learning for Interpretable Protein-Protein Interaction Prediction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cheng, X., Yu, Q., Yan, S., Dai, W., Cheng, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding the multi-scale organization of protein-protein interactions (PPIs) is fundamental to deciphering cellular signaling, allosteric regulation, and disease mechanisms, yet existing computa- tional approaches fail to simultaneously resolve atomic-scale binding interfaces and pathway-level coordination. We present HyWinNet, a geometric deep learning framework that unifies Lorentzian hyperbolic graph neural networks with spectral graph wavelet transforms to intrinsically model the hierarchical architecture of biological interactions. The core innovation lies in three synergistic components: (1) Hyperbolic embeddings that preserve evolutionary-conserved topological relationships through negatively curved space projections, capturing scale-free properties of protein networks; (2) Multi-scale wavelet decomposition operating at biologically grounded resolutions to jointly analyze local residue contacts and global pathway dynamics; (3) Contrastive learning with stochastic feature dropout that mimics natural binding interface plasticity. Benchmarked against state- of-the-art methods, HyWinNet demonstrates superior performance in recovering known interactions while predicting previously unannotated functional sites validated through independent structural and biochemical studies. By bridging hyperbolic geometry with graph signal processing, this work establishes a new paradigm for analyzing multi-scale biological networks, offering both interpretable computational predictions and testable hypotheses for experimental validation. The frameworks ability to map hierarchical relationships from atomic details to system-level modules provides a transformative tool for drug discovery and mechanistic studies of complex diseases.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.01.673587v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Query Augmented Generation (QAG) from the Genomic DataCommons for Accurate Variant Statistics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Venkat, A., Wysocki, W. P., Lukowski, M., Song, S., Subramanyam, A., Zhang, Z., Grossman, R. L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In precision oncology, researchers often use public knowledgebases to check somatic variant frequencies against their cohort data. Large language models (LLMs) can quickly answer questions on somatic variant frequencies, but often hallucinate and give inaccurate results for factual data. Using synthetic queries, we show that somatic variant frequencies in baseline LLM responses are underestimated compared to the Genomic Data Commons (GDC), the worlds largest data commons for cancer research. We present a modular architecture called Query Augmented Generation (QAG) for integrating LLMs with high-quality data from a third party data source such as a data commons, knowledgebase or database. We apply QAG to the GDC to help researchers obtain accurate frequencies for somatic variants, copy number variants, and MSI status, even for complex queries requiring multiple steps in the GDC portal and API. Our software is deployed as a model context protocol (MCP) server on Hugging Face and available on GitHub.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.02.673797v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrated Blood Brain Transcriptomics Reveals CD79A and GRIA2 as Drug Repurposing Targets in Multiple Sclerosis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mahmoud, M. A., AlShehri, M. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Multiple sclerosis (MS) has concomitant immune and neurodegenerative mechanisms that are not tractable to single mechanism treatments. We speculated that simultaneous transcriptomic network analysis of peripheral blood mononuclear cells (PBMCs) and MS brain lesions would identify convergent, druggable hubs susceptible to combination repurposing approaches. Microarray data of PBMCs (GSE21942; 14 MS, 15 controls) and brain lesions (GSE38010; 5 MS, 2 controls) were analyzed using differential expression thresholds (adjusted P < 0.05; |log2 FC| [‚â•] 1.5). Protein protein-interaction networks were constructed with Search Tool for the Retrieval of Interacting Genes and subnetworks recognized by molecular complex detection. PBMC co-expression modules were recognized by Weighted Gene Co-expression Network Analysis. Functional enrichments of cluster genes were undertaken by gene ontology and gene set enrichment analysis. CytoHubba identified candidate genes first, then hubs were identified by Least Absolute Shrinkage and Selection Operator (LASSO) regression (PBMCs) and top three brain lesion CytoHubba genes were considered hub genes. We screened a ligand library against four hubs CD79A, GRIN2A, NRXN1, and GRIA2, and assessed binding stability by 200 ns molecular dynamics and MM/PBSA. We found 142 PBMC differentially expressed genes (DEGs) and 1493 lesion DEGs, which were mapped to four peripheral modules B-cell-receptor signalling, erythrocyte metabolism, spliceosome stress and dampened innate sensing, and four central modules glutamatergic synapse, axon ensheathment/sodium-channel stress, netrin-1 signalling and nucleocytoplasmic-transport. LASSO identified FCRL1, CD22, and CD79A, while GRIN2A, GRIA2, and NRXN1 were the chosen brain hubs based on CytoHubba Maximal Clique Centrality. Molecular docking and dynamics simulation identified icotinib and niraparib as dual-target ligands of CD79A and GRIA2. Our end-to-end discovery pipeline defines CD79A and GRIA2 as dual-compartment targets. Repurposing compounds like icotinib and niraparib to co-modulate B-cell activation and excitotoxic synaptic injury represents an actionable strategy for multipronged treatments of MS.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.02.673787v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Distributional Data Analysis Uncovers Hundreds of Novel and Heritable Phenomic Features from Temporal Cotton and Maize Drone Imagery</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> DeSalvio, A. J., Matabuena, M., Adak, A., Arik, M. A., DeSalvio, S. M., Murray, S. C., Wong, R. K. W., Edwards, J., de Leon, N., Kaeppler, S. M., Lima, D. C., Hirsch, C. N., Thompson, A., Stelly, D. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genomic and phenomic analyses suggest additional heritable phenomic features can improve modeling of important end traits like senescence or yield. Field phenotyping generally uses trait values averaged across individual experimental units (plants or numerous plants within plots), ignoring the full distributional pattern of collected measures. Images of plants or plots, as captured by drones (unoccupied aerial vehicles / UAVs / drones), can be viewed as individual distribution functions that capture biological information. This study introduces and validates distributional data analysis in two crops and experiment types - cotton (Gossypium hirsutum L.) single plant vegetation index (VI) analysis and maize (Zea mays L.) plot-level yield predictions. In both crops, the concept of within-day variance decomposition was demonstrated. In cotton, genotypes exerted significant influences on temporal quantile functions of VIs. Maize yield prediction using distributional data with elastic-net regression indicated improvements in yield prediction between 12.7%-21.6% with quantiles outside the conventionally used median responsible for added predictive power. A novel data visualization method for per-pixel heritability allowed distributional features to be explainable and interpretable. These results have implications for future plant phenomic studies, indicating that distributional data analysis applied across temporal imagery captures novel, heritable, and interpretable biological signal that is lost when working with conventional measures of central tendency such as mean or median summary values of experimental units.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.05.674557v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The Rapid Anatomics Tool (RAT): A low-cost root anatomical phenotyping pipeline reveals changes in root anatomy along the root axis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jones, D. H., Baca Cabrera, J. C., Behrend, D., Wells, D. M., Swift, J. F., Atkinson, J., Lobet, G., Hanlon, M. T., Schneider, H. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Root anatomical phenotyping has become a demonstrably essential part of investigating root physiology and in acquiring a holistic understanding of plant development. However, accessible high throughput methods for root anatomical analysis are still lacking. Here, we present the Rapid Anatomics Tool (RAT), a novel, low-cost system for high throughput root anatomical imaging with a shallow learning curve for obtaining high quality images suitable for comparative analysis across a number of plant species. Its efficiency comes from combining blockface-like imaging and stain-free imaging using near-ultraviolet (nUV) autofluorescence utilising a combination of low-cost commercial equipment, readily available mechanical components, and custom designed and 3D printed tools. Using this system, we investigated the anatomy of mature tissue along the axis of wheat crown roots, revealing a tendency of reduction in vascular complexity (expressed through a reduction in metaxylem number, area, and mean area per metaxylem file) from the basal to the distal region of the root. This study highlights the importance of thorough sampling strategies for investigating root anatomy in relation to organ function and introduces an accessible, relatively high-throughput method to support such research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.05.674482v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The 'Iceberg region of the Grapevine leafroll-associated virus 3 replicase polyprotein contains signals for mitochondrial targeting and outer membrane association</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fust, C., Li, C., Meng, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Positive-sense single-stranded (+ss) RNA viruses such as Grapevine leafroll-associated virus 3 (GLRaV3) replicate their genomes within membrane-bound viral replication complexes (VRCs). The biogenesis of such VRCs is driven by viral "replicase" polyproteins containing several replication-related domains. Past electron microscopy evidence suggests that GLRaV3 forms VRCs from the outer mitochondrial membrane of host grapevine plants. Here, we report the subcellular localization of the replicase polyprotein encoded by GLRaV3 ORF1a (PP1a) towards understanding its putative role in VRC formation. Through confocal laser scanning microscopy analysis of distinct EGFP-tagged PP1a domains, interdomain regions and truncations expressed in model plants, we report the molecular signals responsible for the targeting and association of PP1a to the mitochondria. This signal, located in the "Iceberg" region downstream from the methyltransferase-guanylyltransferase (M/GTase) domain, comprises an amphipathic a-helix and a downstream transmembrane domain (TMD). Mutagenesis studies suggest that the polar face of the amphipathic a-helix functions as the targeting signal, whereas the non-polar face, together with the TMD, act in membrane-anchoring. Microscopy observations are confirmed through mitochondrial isolation via gradient centrifugation and Western blotting. Structure prediction of the GLRaV3 M/GTase domain and its downstream TMD suggests a putative dodecameric oligomeric state. This dodecamer may gate the neck of GLRaV3 VRCs and contribute to their biogenesis, a hypothesis to be tested in follow-up studies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.05.674466v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Regenerative agriculture effects on biomass, drought resilience and 14C-photosynthate allocation in wheat drilled into ley compared to disc or ploughed arable soil</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Austen, N., Short, E., Tille, S., Johnson, I., Summers, R., Cameron, D. D., Leake, J. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Regenerative agriculture practices including leys and no-tillage facilitate biological reassembly of soil aggregates, increasing water, carbon and nutrient storage, but how this effects crop biomass, photosynthate partitioning, and drought resilience is unclear. To address this, we took monoliths growing semi-dwarf and taller wheat genotypes from 3-year plots that were ploughed or disc cultivated, or direct-drilled into a grass-clover ley and applied 35 kg N ha-1. Half the monoliths received a spring drought, then all were watered and the wheat 14CO2 pulse-labelled at stem elongation. The ley soil had lower bulk density, stored more water, and the taller wheat genotype maintained 8-fold higher proportion of water-stable macroaggregates despite unexpectedly having the smaller root biomass. Yields on the ley soil (3.74 t ha-1) were 77% -123% higher than on ploughed and disc cultivated soils, and unaffected by genotype or drought, despite its >70% reduction in root biomass. Of the 14C initially retained in wheat, 72% was in shoots, with root allocation decreasing by 75-90% in droughted ley soil, and at harvest soil retained <1% of the 14C, with significantly lower values for taller wheat, and ley. We conclude that soil health regeneration in the ley enhanced wheat yields, but reduced photosynthate allocation to root biomass under drought. Although taller wheat maintained better macroaggregation in ley soil, this was not explained by root biomass or photosynthate allocation and unexpectedly failed to increase soil 14C sequestration. We find no evidence that ley regeneration of macroaggregation enhances soil C sequestration under wheat, despite higher yields.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.04.674292v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The functional divergence of plant ESCRT components TOL3, SNF7.1, and VPS4 during salt stress response</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Schnurer, M., Schweighofer, A., Hilscher, J., Kapusi, E., Korbei, B., Naegele, T., Kang, L., Fuhrmann, P., Danninger, S., Scharl, L., Bock, A., Marino, G., Leister, D., Ibl, V.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Salinity, a major abiotic stressor, impacts protein trafficking and alters transport routes. The endosomal sorting complex required for transport (ESCRT)is a key player for membrane protein sorting and is increasingly recognized as pivotal element in abiotic stress responses. However, the functional role of ESCRT proteins in cereal crops under salt stress remains unexplored. Using the as near as possible to nature DRD-BIBLOX (Dark-Root device Brick Black Box), barley (Hordeum vulgare, L.), although relatively salt-tolerant, exhibits impaired germination, seedling growth, and an altered root system architecture (RSA) under high salinity. Shotgun proteomic analysis revealed spatiotemporal regulation of ESCRT proteins in shoots and roots during seedling development. We identified HvSNF7.1 and HvVPS4 as critical, but differently acting modulators of salt stress responses in barley tissues. In this study we further reveal novel functions of ESCRT-0-like TOM-1-like proteins (TOLs): Previously described as gatekeepers of degradative protein sorting and as regulators of abscisic acid signaling, growth, and heat stress responses in Arabidopsis (Arabidopsis thaliana), HvTOLs had not been characterized for their role in salinity adaptation during plant development. Functional analyses demonstrated lethality in Hvtoldouble and Hvtolquadruple mutants under standard growth conditions. In Arabidopsis, loss of AtTOL3 alone impaired germination and root growth under salt stress, underscoring the essential role of individual TOL proteins in the spatio-temporal regulation of abiotic stress adaptation. These findings highlight ESCRT-mediated trafficking as a key determinant of cereal resilience under saline conditions and provide a molecular framework for improving crop stress tolerance.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.06.674610v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">From Photoperiod Thresholds to Photoperiod sensitivity: Dual Strategies for Cost-Effective Speed Breeding and Climate-Ready Barley</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rossi, N., Powell, W., Halliday, K., Sharma, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Yield and the duration of the growing season are closely linked. Climate change may shorten growing seasons in certain European regions, and reducing the time to flowering could be an effective strategy to mitigate its effects. Therefore, exploring allelic combinations shape flowering time, is needed. Additionally, speed breeding (SB), characterized by extended photoperiods to accelerate generation time, can be energy-intensive, and the shortest day length needed to induce rapid flowering remains unknown. We present the first integrated study of how allelic variation at three key flowering time genes, PPD-H1, ELF3 and PHYC, modulates three parameters of the photoperiod response model: threshold photoperiod, photoperiod sensitivity, and intrinsic earliness. We recorded flowering under lengths of 16-24h in Near Isogenic Lines carrying PhyC-e or PhyC-I allele within ppd-H1 background, and in lines from HEB-25 combining wild and domesticated alleles of ELF3 and PPD-H1. The ELF3 allele in ppd-H1 background reduced intrinsic earliness, whereas PhyC-e reduced photoperiod sensitivity, opening opportunities for climate change adaptation. Remarkably, ppd-H1 lines flowered at a 20-h threshold, whereas Ppd-H1 lines showed no response, consequently we propose new SB photoperiods at 20 and 16h depending on PPD-H1 background. These photoperiods lower energy costs compared to the current 22h standard.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.04.674238v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Population structure and diversity of common bean (Phaseolus vulgaris L.) landraces in the Peruvian Amazon</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tobaru, J., Barrera-Lozano, M., Vecco-Giove, C. D., Pelaez, J., Pinchi, M., Vargas, Y., Chuquillanqui, J., Soto-Torres, J., Robles, R., Saravia, D., Petroli, C., Etchevers, J. D., Ortiz, R., Blas, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Common bean (Phaseolus vulgaris L.) is a globally important legume with significant nutritional and agronomic value. Genetic diversity in Amazonian landraces remains undercharacterized, limiting their utilization in breeding programs. Here we analyzed 647 accessions from the Peruvian Amazon using 18 morphological traits and 23,050 high-quality DArTseq single nucleotide polymorphisms (SNPs). Our results revealed two major genetic groups corresponding to Andean and Mesoamerican gene pools, each subdivided into distinct subgroups, exhibiting high allelic richness and structured diversity. Despite abundant polymorphic loci, observed heterozygosity was low, consistent with the species' self-pollinating nature. These findings highlight the Peruvian Amazon as a reservoir of unique genetic variation, underscoring the need for integrated in situ and ex situ conservation strategies. This germplasm offers valuable resources for breeding programs targeting tropical adaptation and resilience, contributing to sustainable agriculture and food security.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.04.674357v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Enhancer-driven random gene overexpression (ERGO): a method to study gene function in Chlamydomonas</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lihanova, Y., Craig, R. J., Pribbernow, M., Goss, R., Sasso, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Gene overexpression can be used to study gene function and is more suitable to characterize essential and redundant genes than gene knockout. A forward genetic approach based on random gene overexpression, also known as activation tagging, was previously used to study gene function in angiosperms. However, such an approach has never been applied to algae. Here, we present enhancer-driven random gene overexpression (ERGO), a forward genetic screen that we utilized to study genes involved in carotenoid metabolism in the green alga Chlamydomonas reinhardtii. We generated a library of over 33,000 insertional mutants in a yellow-in-the-dark background strain, which is incapable of producing chlorophyll in the dark. Each mutant contained a randomly inserted enhancer, Ehist cons, capable of activating gene expression in the C. reinhardtii nuclear genome. After visually screening the mutant colonies for a color change from yellow to orange, we isolated a mutant with increased carotenoid content and remarkable resistance to high-light stress. RNA-seq data analysis revealed substantial upregulation of a gene, that we name CMRP1, encoding a putative F-box protein. CRISPR-mediated knockout of this gene resulted in decreased carotenoid concentrations, confirming that CMRP1 is involved in the regulation of carotenoid metabolism. Our study shows that a gene overexpression screen can be successfully adapted to C. reinhardtii and potentially other plants and algae, thereby expanding the palette of genetic tools to study gene function.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.05.674469v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Modular Approach for the Optogenetic Control of Small GTPase Activity in Living Cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Faulkner, B., He, Y., Sitrin, D., Ziamanesh, L., Stains, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Small GTPases are critical regulators of cellular processes, such as cell migration, and comprise a family of over 167 proteins in the human genome. Despite the essential role these proteins play in maintaining cell homeostasis, there is no generalizable method for directly controlling the activity of these signaling enzymes with subcellular precision. Herein, we report the design and implementation of a modular, optogenetic platform for the spatial control of small GTPase activity within living cells. By combining split-small GTPases with improved Light-Inducible Dimerization (iLID) domains, we obtain spLIT-small GTPase constructs that enable spatially precise control of cytoskeletal dynamics such as filopodia formation (spLIT-Cdc42), membrane retraction (spLIT-RhoA), and directed cell migration (spLIT-Rac1). These results establish spLIT-small GTPases as a modular optogenetic platform for spatially controlled activation of small GTPase signaling within living cells and demonstrate the ability to probe spatiotemporal aspects of cell signaling using this approach.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.07.674731v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Anti-PD-1/PD-L1 Therapy Triggers Cognitive Deficits and Anxiety-Like Behaviors Through Tumor-Initiated Neuroinflammatory Niches in Male Mice</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nicola, C., Pedard, M., Dubois, M., Desrues, L., Neveu, P., Riou, G., Johnston, I., Dembele, P. K., Lecras, P., Vaudry, D., Adriouch, S., Joly, F., Hilber, P., Wurtz, O., Castel, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Checkpoint inhibitors are promising immunotherapy to treat cancer patients, but their cognitive impact has not been evaluated despite several neurological adverse events. We studied the impact of immune desert or inflamed cancers when combined with immune checkpoint inhibitors (ICI) anti-PD-1/anti-PD-L1 on mouse behaviors and brain immune cells infiltration/homeostasis, and neuroinflammation in male mice. We showed that systemic inflammation, brain-barriers permeability accompanying meningeal infiltration of peripheral macrophages and neuroinflammation as well as deficits in cognition or emotional reactivity, depending on immuno-inflammatory or immune-desert cancer type. Combined with cancers, anti-PD-1 and PD-L1 treatments exacerbated the decline in executive functions and hippocampal vascular inflammation. PD-L1 specifically relayed the infiltration of the Tgammadelta lymphocytes subpopulation in choroid plexus and leptomeninges implicated, whose systemic neutralization counteracted anti-PDL1-induced cognitive deficits and anxiety in mice bearing immune-inflamed cancer. Our findings highlight new systemic biomarkers of cold or hot cancer, treated with anti-PD-1/anti-PDL-1, and associated with cognitive and emotional alterations in mice; guiding ways of intervention to secure the cancer curation and improve patients quality of life under ICI treatment.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.03.673981v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Survivin as a Multifaceted Oncogenic Driver and Therapeutic Target in Renal Cell Carcinoma</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tuli, S., Inserra, G., Murakami, Y., Bae, Y., Gau, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Renal cell carcinoma (RCC) is a heterogeneous malignancy in which clear cell RCC (ccRCC) represents the most aggressive subtype. Survivin (BIRC5), an inhibitor of apoptosis and key regulator of mitosis, is frequently overexpressed in RCC and associated with poor prognosis, yet its broader role in kidney cancer biology remains poorly defined. Here, we analyzed transcriptomic data from the TCGA-KIRC cohort and found that advanced-stage ccRCC exhibits widespread dysregulation of cell cycle pathways, with 1,484 genes upregulated and 479 genes downregulated in stage IV compared to stage I tumors. To define survivins functional contribution, we performed loss-of-function and pharmacologic inhibition studies in RENCA cells. Survivin knockdown or treatment with the small molecule inhibitor YM155 significantly reduced proliferation, S-phase entry, and Cyclin D1 expression, while also impairing both collective and single-cell migration. Beyond cell cycle control, survivin depletion induced notable changes in mitochondrial morphology and bioenergetics, including increased mitochondrial content coupled with reduced oxygen consumption, suggesting accumulation of dysfunctional mitochondria due to impaired clearance. Collectively, these findings identify survivin as a multifaceted oncogenic driver in RCC that integrates cell cycle progression, cytoskeletal organization, and mitochondrial homeostasis. By revealing survivins dual roles in proliferative and metabolic adaptation, this work highlights survivin as both a prognostic biomarker and a therapeutic vulnerability, supporting future strategies that combine survivin inhibition with metabolic or cell cycle-directed therapies for advanced kidney cancer.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.02.673805v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Inhibition of PTCH1 drug efflux activity enhances chemotherapy efficacy against triple negative breast cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cogoluegnes, S., Kovachka, S., Dubois, T., Wurth, R., Donato, E., Trumpp, A., Franco, M., Luton, F., Azoulay, S., Mus-Veteau, I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Triple-negative breast cancer (TNBC) is the most aggressive breast cancer subgroup characterized by a high risk of resistance to chemotherapies and high relapse potential. High levels of mRNA from the Hedgehog receptor PTCH1 are associated with poor prognosis in TNBC. PTCH1 is overexpressed in many aggressive cancers. We previously reported that PTCH1 is a multidrug transporter that triggers resistance to chemotherapy of adrenocortical carcinoma and melanoma cells, and that inhibiting PTCH1 drug efflux strongly enhanced chemotherapy efficacy on these cell lines both in vitro and in vivo. In the present study, we found that PTCH1 inhibition also significantly inhibited doxorubicin efflux in three TNBC cell lines leading to a strong increase of the cytotoxic effect of doxorubicin and docetaxel, and an inhibition of cell migration. Altogether, our data highlight the therapeutic potential of targeting PTCH1 drug efflux activity using drug association strategies for the treatment of TNBC patients.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.03.673759v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

