<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Saturday, 16 August 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Functional and epitope specific monoclonal antibody discovery directly from immune sera using cryo-EM.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> James A Ferguson, Sai Sundar Rajan Raghavan, Garazi Peña Alzua, Disha Bhavsar, Jiachen Huang, Alesandra J Rodriguez, Jonathan L Torres, Maria Bottermann, Julianna Han, Florian Krammer, Facundo D Batista, Andrew B Ward</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We improved upon our streamlined approach combining structural analysis and bioinformatics to infer heavy and light chain sequences from cryo-EM (cryo-electron microscopy) maps of serum-derived polyclonal antibodies (pAbs) bound to antigens.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Antibodies are crucial therapeutics, comprising a substantial portion of approved drugs due to their safety and clinical efficacy. Traditional antibody discovery methods are labor-intensive, limiting scalability and high-throughput analysis. Here, we improved upon our streamlined approach combining structural analysis and bioinformatics to infer heavy and light chain sequences from cryo-EM (cryo-electron microscopy) maps of serum-derived polyclonal antibodies (pAbs) bound to antigens. Using ModelAngelo, an automated structure-building tool, we accelerated pAb sequence determination and identified sequence matches in B cell repertoires via ModelAngelo-derived hidden Markov models (HMMs) associated with pAb structures. Benchmarking against results from a nonhuman primate HIV vaccine trial, our pipeline reduced analysis time from weeks to under a day with higher precision. Validation with murine immune sera from influenza vaccination revealed multiple protective antibodies. This workflow enhances antibody discovery, enabling faster, more accurate mapping of polyclonal responses with broad applications in vaccine development and therapeutic antibody discovery.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40815640/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Uncovering Functional Sequence Gaps in Human Reference Genomes using African Pan Genome Contig Sequences</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Martini, R., Tijjani, A., Founta, K., Cha, D., Maurice, S., White, J., Balogun, O., Davis, M. B., Chambwe, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of African Pan-Genome contig sequences not represented in GRCh38 to assess their representation and functional potential in newer long-read assemblies.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Predominantly used human reference genomes, including GRCh38 and the gapless T2T-CHM13 references, remain limited in their representation of African genomic diversity. We analyzed African Pan-Genome (APG) contig sequences representing 296.5 Mb of African-ancestry-specific sequence not represented in GRCh38 to assess their representation and functional potential in newer long-read assemblies. Alignments to T2T-CHM13 and the 47 Human Pangenome Reference Consortium (HPRC) linear assemblies positioned 40% and 83% of APG contigs, respectively, with high identity and coverage. Most T2T-CHM13 placements corresponded to sequences absent from GRCh38 (94.5%) and were enriched in centromeric and satellite repeats (94.2%). Functional overlap included annotated genes (2.6%) and CpG islands (3.6%), with enrichment in immune, synaptic and intracellular signaling pathways. HPRC alignments revealed ancestry-associated patterns, with African and Admixed American genomes showing the highest numbers of unique contig placements and shared alignments. A subset of 742 APG contigs showed weak or no mapping to any of the reference assemblies. These contigs were not enriched for repeat elements, and ~58% showed predicted gene content. These findings highlight persistent gaps in even the most complete reference genomes and underscore the importance of incorporating ancestry-enriched sequences into future genome frameworks to reduce reference bias and advance equitable discovery in genomics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.15.670543v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Rapid turnover and recurrent structural variation at the 17q21.31 locus in modern and ancient human genomes, and primates</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sridharan, S., Lou, R. N., Ferguson, S., Rocha, J., Mitchell, M., Kililia, A., Borda, V., Medina-Munoz, S. G., Gravel, S., Henn, B., Sudmant, P. H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the 17q21.31 locus in humans harbors several complex structural haplotypes including a recurrent inversion and KANSL1 duplication, which is 2 million years younger than the human inversion, and that gorillas exhibit an independent duplication.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The 17q21.31 locus in humans harbors several complex structural haplotypes including a ~970kb inversion. Different inversion haplotypes have been associated with susceptibility to microdeletions causing Koolen-de Vries syndrome and variation in fecundity and recombination rates. Here, using 210 haplotype-resolved human genome assemblies and pangenome graph-based approaches we characterize 11 distinct structural haplotypes, several of which have not been previously described. Extending our analyses to a set of haplotype-resolved great-ape genomes, we characterize the structure of a recurrent inversion in chimpanzees which extends an additional 750kb and is ~2 million years younger than the human inversion. We further determine that gorillas exhibit an independent duplication of the KANSL1 gene, disruption of which in humans is the cause of Koolen-de Vries syndrome. Using short read sequencing data we characterize 17q21.31 haplotype diversity worldwide in ~5174 individuals from 107 populations finding increased frequencies of KANSL1 duplication-containing haplotypes in both European and South Asian populations as well as 8 double recombination events between inverted and non inverted haplotypes ranging in size from 20-180kb. Finally, using 626 ancient Eurasian human genomes we show haplotypes containing KANSL1 duplications have increased ~6-fold over the past 12 thousand years in Europe. Together, our results highlight the dynamics, complexity, and recurrent, independent evolution of a medically relevant locus across humans and great apes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.15.670618v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A laser capture microdissection-based method for high-sensitivity transcriptomics from archived FFPE tissue slides with single-cell resolution using LCM-FFPEseq</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Callens, E., Syryn, H., Deserranno, K., Cattebeke, G., Couckuyt, H., Van De Velde, J., Berrevoet, D., Verplancke, A., Deforce, D., De Spiegelaere, W., Van de Vijver, K., Cools, M., Van Nieuwerburgh, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel method combining LCM with the advanced Smart-seq3xpress protocol and FFPE-specific adaptations for spatial transcriptomics of formalin-fixed, paraffin-embedded (FFPE) tissues .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding gene expression within its spatial context is essential for unravelling biological processes. Laser Capture Microdissection (LCM) has emerged as a transformative technology, enabling targeted isolation of individual cells or regions from tissue sections while preserving spatial context. However, its application to formalin-fixed, paraffin-embedded (FFPE) tissues has been limited by RNA degradation, leaving the vast repository of clinical FFPE samples underutilized. To address this, we introduce LCM-FFPEseq, a novel method combining LCM with the advanced Smart-seq3xpress protocol and FFPE-specific adaptations for spatial transcriptomics of FFPE sections. Unlike traditional protocols requiring thousands of cells to generate high-quality libraries, LCM-FFPEseq achieves high sensitivity, reproducibility, and transcript coverage. With as few as 30 FFPE-embedded K562 cells and Sertoli cells, we detected over 14,000 protein-coding genes per sample, with no substantial gains when a higher number of cells were isolated. Even individual LCM-isolated cells yielded an average of 7,353 or 6,490 protein-coding genes per K562 or Sertoli single cell, respectively. To demonstrate its clinical utility, we applied LCM-FFPEseq to archived testicular FFPE samples from transgender females receiving gender-affirming hormone therapy. Transcriptomic profiling of isolated seminiferous tubules revealed tubular hyalinization to be associated with greater upregulation of extracellular matrix remodelling and inflammatory pathways, alongside stronger downregulation of spermatogenesis-associated pathways. These findings suggest that testicular fibrosis and/or tubular hyalinization may contribute to germ cell loss following inappropriate hormonal exposure. By enabling high-resolution transcriptomics in archived FFPE samples, LCM-FFPEseq unlocks new possibilities for investigating rare cell types, spatial heterogeneity, and therapy-induced tissue remodelling in vast FFPE repositories.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.13.669096v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Autosomal Allelic Inactivation: Variable Replication and Dosage Sensitivity</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Heskett, M. B., Vouzas, A., Johnstone, B., Freese, K. P., Yates, P., Copenhaver, P. F., Spellman, P. T., Gilbert, D. M., Thayer, M. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Inactivation/Stability Centers (I/SCs) are recently described autosomal loci that exhibit epigenetic regulation of allelic expression and replication timing, with differences that can be comparable to those observed between the active and inactive X chromosomes 1.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Autosomal monoallelic gene expression and asynchronous replication between alleles are well-established features of imprinted genes and genes regulated by allelic exclusion. Inactivation/Stability Centers (I/SCs) are recently described autosomal loci that exhibit epigenetic regulation of allelic expression and replication timing, with differences that can be comparable to those observed between the active and inactive X chromosomes 1. Here we characterize hundreds of autosomal loci with allele-specific epigenetic regulation of replication timing and gene expression, defining them as I/SCs. I/SCs are approximately 1 megabase in size and can contain both protein-coding and noncoding genes. In different single cell derived clones, these genes may be expressed from a single allele, the opposite allele, both alleles, or not expressed at all. This stochastic, yet mitotically stable, pattern indicates that the choice of which allele is expressed is independent of parent of origin and independent of the expression status of the other allele. Similarly, alleles within I/SCs show varying replication timing, either earlier or later, that is also independent of the other allele. Additionally, we find corresponding I/SCs in the mouse genome that display conserved synteny with human I/SCs. This allele-restricted regulation creates extensive cellular mosaicism through a stable epigenetic mechanism. This mosaicism impacts numerous dosage-sensitive genes associated with human diseases such as Parkinson, epilepsy, deafness, and impaired intellectual development.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.13.670061v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Massive proliferation of retrotransposons contributes to genome size expansion in species of the Pseudocercospora genus</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gonzalez Sayer, S.-M., Garcia, I. A., Traslavina, C. A., Zaccaron, A. Z., Stergiopoulos, I., Aristizabal, F. A., Oggenfuss, U., Croll, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of genome size expansions in the fungal genus Pseudocercospora and its species with the most expanded genomes and reference-quality genomes .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genome size expansions are common among eukaryotic lineages. Enlarged genomes can be bioenergetically demanding, and active mobile elements can trigger chromosomal rearrangements and loss of gene function. What triggers genome size expansions remains largely unexplored in many biological clades, particularly within the fungal kingdom. Activation of large transposable elements (TEs), such as long-terminal repeats (LTRs), is a common contributor. Yet the mechanisms of LTR activation remain poorly understood. Here, we focus on the fungal genus Pseudocercospora and closely related species with known variation in genome size. In using an assembly-free approach, we found that TE content is highly variable among species, with species-specific retrotransposon families being the main drivers of independent genome expansions. We further focused on the two species with the most expanded genomes and reference-quality genomes, P. fijiensis and P. ulei. We found that the P. ulei genome is compartmentalized, with highly variable TE densities among chromosomal regions, and a striking reduction in pathogenicity-associated genes. Overall, our study indicates that species of Pseudocercospora originally had reduced genome sizes, and genome expansions are species-specific, driven by heterogeneous sets of TE families. Furthermore, we found that in the species with the most expanded genome, TE activity might not have ceased yet as indicated by resequencing data analysis of six strains from diverse locations in Colombia. We discuss what might have caused TE activation and subsequent proliferation in the genus, including stress conditions and host adaptation. Surveys of clades with highly dynamic genome sizes are crucial for the investigation of causal factors driving long-term TE dynamics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.13.670038v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A functional annotation of a selected protein-coding gene in the Olimarabadopsis pumila genome: Oxysterol-binding protein-related-protein (ORP) 2B</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rubin, N. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A functional annotation of the genome of Olimarabidopsis pumila, a spring ephemeral from Asia, and its role in sterol transportation and division .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Olimarabidopsis pumila, a spring ephemeral from Asia, is closely related to the well-studied model organism Arabidopsis thaliana, though with various important differences. O. pumila has evolved to possess various adaptations to its arid environment, including a high salt-tolerance and utilization of extreme light conditions, which A. thaliana does not have. However, O. pumila is drastically understudied, despite the advantages of using natural genetic variation and broader contexts in researching the various fields of genomic, cellular, and plant biology. In this study, we attempted to address this lack of important research by contributing to the functional annotation of the genome of O. pumila. One gene was selected and analyzed using various genomic biology softwares -- particularly GenSAS, AUGUSTUS, NCBI BLASTP and MSA Viewer, PSIPRED, AlphaFold, InterPro, STRING, DeepTMHMM, SignalP, TargetP, and WoLF PSORT -- to determine its sequence conservation, three-dimensional structure, function, and subcellular localization. The gene, located on contig tig00000022 of the O. pumila genome, was found to encode oxysterol-binding protein-related protein (ORP) 2B. We concluded this protein is involved in sterol transportation from the endoplasmic reticulum, and possibly works in contribution to the important biological processes of cell proliferation and division.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.13.670020v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Characterization of interindividual DNA methylation variability in rainbow trout (Oncorhynchus mykiss)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lefort, G., Brionne, A., Piegu, B., Terrier, F., Pigeon, A., Beauclair, L., Roy, J., Marandel, L., Panserat, S., Lallias, D., Coustham, V.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A first description of the rainbow trout methylome variability using whole-genome bisulfite sequencing approach to minimize genetic variation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Interindividual epigenetic variability, particularly in DNA methylation, is now recognized as a significant contributor to phenotypic diversity in humans and mammals. These epivariable regions, which make up a small fraction of the genome, are strongly influenced by genetic factors and environmental factors, especially during early development. In this context, epigenetic variability of DNA methylation has been proposed as an adaptive force involved in various environmental responses. In fish and other vertebrates, environmental factors are known to influence the health, performance and welfare, likely through the alteration of the epigenetic landscape. However, whether interindividual epigenetic variability may contribute to the phenotypic plasticity of fishes is unknown. Here we provide a first description of the rainbow trout methylome variability using a whole-genome bisulfite sequencing approach in an isogenic line to minimize genetic variation. Variable methylation regions were identified in both liver and hypothalamus tissues of 12 replicate fishes and were found enriched at gene regulatory elements, such as promoters and first introns. Gene Ontology analysis revealed functional clusters related to cellular development, neural communication, metabolic balance, and immune response. Interestingly, some variably methylated regions are found at the same genomic loci in both tissues and showed a strong intraindividual correlation in methylation levels, suggesting establishment during early embryogenesis. Overall, our work demonstrates the existence of interindividual epigenetic variability in rainbow trout and provides valuable insights into the regulatory function of DNA methylation variation that is likely involved in developmental and physiological processes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.13.669306v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Nuclear Irregularity as a Universal Diagnostic Tool in Solid Tumors</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hamilton, F., Foster, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify cancerous and non-cancerous nuclei in breast cancer cells using imaging mass cytometry and a computational and machine learning-based pipeline for morphological heterogeneity of nuclei and protein expression, enabling patient-specific tumor phenotypes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> As tumors develop, cancer cells accumulate diverse genomic and phenotypic alterations to meet heightened demands for energy production and biosynthesis. Loss of lamina function and perturbations in energy production are associated with pronounced aberrations in cellular morphology, particularly within nuclear architecture and the plasma membrane. Systematic analysis of nuclear morphology can reveal conserved structures across diverse cancer types, enabling disease state stratification, biomarker discovery, and potential avenues for personalizing therapy to minimize recurrence risk. To this end, this study analyzes an imaging mass cytometry (IMC) breast cancer dataset, differentiating cancerous and non-cancerous nuclei with a p-value of 1.02e-06. In addition, this study achieves an accuracy of 78 percent and a f1 score of 72 percent using a computational and machine learning-based pipeline for analyzing the morphological heterogeneity of nuclei and protein expression, enabling characterization of patient-specific tumor phenotypes. Unlike traditional morphology analysis pipelines limited to specific imaging platforms, this workflow enables cross-cohort and cross-cancer comparison, capturing tumor-specific phenotypic deviations at a single-cell resolution. The resulting phenotypic profiles could inform prognosis, treatment, and monitoring of therapeutic response.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.12.669986v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Systematic feature and architecture evaluation reveals tokenized learned embeddings enhance siRNA efficacy prediction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Coffey, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A methodical approach to feature selection and hyperparameter tuning, particularly favoring learned embeddings, can yield a more accurate and reliable model for predicting siRNA efficacy, outperforming more complex architectures.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent advances in machine learning have improved the prediction of siRNA efficacy, with graph neural networks and transformer-based encodings leading the way. However, existing models still face challenges, including potential inaccuracies in thermodynamic feature calculations (such as incorrect strand selection for siRNA-mRNA Gibbs free energy), limited effective utilization of available datasets, and a lack of systematic model refinement. In this study, I systematically evaluated the predictive power of individual features and neural network architectures to identify the most effective configurations. This process led to the development of RN.Ai-Predict, a model built upon a tokenized learned embedding for nucleotide sequences. This work demonstrates that a methodical approach to feature selection and hyperparameter tuning, particularly favoring learned embeddings, can yield a more accurate and reliable model for predicting siRNA efficacy, outperforming more complex architectures in generalizability.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.12.669916v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SeqForge: A scalable platform for alignment-based searches, motif detection, and sequence curation across meta/genomic datasets</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Horvath, E. R. B., Winter, J. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed SeqForge, a scalable, modular command-line toolkit that streamlines alignment-based searches and motif mining across large genomic datasets.</p><p style="font-size: 16px; text-align: justify;">Background: The rapid increase in publicly available microbial and metagenomic data has created a growing demand for tools that can efficiently perform custom large-scale comparative searches and functional annotation. While BLAST+ remains the standard for sequence similarity searches, population-level studies often require custom scripting and manual curation of results, which can present barriers for many researchers. Results: We developed SeqForge, a scalable, modular command-line toolkit that streamlines alignment-based searches and motif mining across large genomic datasets. SeqForge automates BLAST+ database creation and querying, integrates amino acid motif discovery, enables sequence and contig extraction, and curates results into structured, easily parsed formats. The platform supports diverse input formats, parallelized execution for high-performance computing environments, and built-in visualization tools. Benchmarking demonstrates that SeqForge achieves near-linear runtime scaling for computationally intensive modules while maintaining modest memory usage. Conclusions: SeqForge lowers the computational barrier for large-scale meta/genomic exploration, enabling researchers to perform population-scale BLAST searches, motif detection, and sequence curation without custom scripting. The toolkit is freely available and platform-independent, making it suitable for both personal workstations and high-performance computing environments.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.12.669971v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Autophagic degradation of EIN3 ensures developmental plasticity and recovery from environmental stress in Arabidopsis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ansboel, J., Riis, I., Stub, M., Chaudhury, R., Nielsen, E., Chevalier, J., Straeten, D. v. d., Zuo, Z. T., Hartman, S., Rodriguez, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Autophagy regulates ethylene signaling in EIN3-dependent growth and stress adaptation, and is essential for EIN2-driven transcriptional reprogramming during submergence tolerance.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Ethylene signaling, mediated by the key transcription factor EIN3, regulates diverse developmental processes and stress adaptations, including hypocotyl growth, aging, and submergence tolerance. Autophagy, a cellular recycling process, also facilitates adaptation by reprogramming cellular components. While EIN3 degradation via the proteasome is well established, its connection to autophagy remains unclear. Here, we show that EIN3 turnover is directly regulated by ATG8-mediated autophagy. Consistently, autophagy-deficient plants exhibit impaired EIN3-dependent hypocotyl growth during light-to-dark transitions. Additionally, EIN3 accumulation contributes to the premature senescence observed in atg mutants. Beyond development, our combination of cell imaging, phenotypic analyses, and proteomics reveals that autophagy is essential for EIN3-driven transcriptional reprogramming during submergence. Together, our findings uncover a direct role for autophagy in regulating EIN3 stability, providing mechanistic insight into how this process fine-tunes ethylene responses in growth and stress adaptation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.13.669664v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Restricted transmission of Xanthomonas oryzae pv. oryzae from rice roots to shoots detected by a rapid root infection system</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Redzich, L., Arra, Y., Loo, E., Frommer, W. B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We establish a coleoptile crown root-based infection protocol to investigate whether gene-for-gene interactions between Xoo TAL effectors and SWEET sucrose uniporter susceptibility genes occur in the root xylem, and whether the disease can propagate from roots to seed</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Xanthomonas oryzae pv. oryzae (Xoo), the causal agent of bacterial blight in rice, is primarily studied in the context of foliar infections. However, infected stubble and irrigation water may serve as reservoirs and be responsible for seedling stage root infections in the field, especially during transplanting. Here, we established a coleoptile crown root-based infection protocol to investigate whether gene-for-gene interactions between Xoo TAL effectors and SWEET sucrose uniporter susceptibility genes occur in the root xylem, and whether the disease can propagate from roots to seedling shoots. Using translational SWEET11a-GUS reporter lines under control of the native SWEET11a promoter, we observed progressive infection as indicated by accumulation of SWEET11a-GUS fusion protein in infected coleoptile crown roots. However, we did not detect progression of GUS accumulation beyond the coleoptile node, nor did we detect blight symptoms on the young leaves. Notably, the xylem, at least during early stages of infection remained functional as shown by Rhodamine B tracer, consistent with transfer of xylem constituents via living cells at the coleoptile node that did not allow bacteria to pass. Furthermore, the root infection protocol is a [~]4x faster compared to standard leaf-clipping assays (roots assay: 11 days from sowing, compared to 39 days for clip infection), enabling more rapid assessment of TAL effector repertoire and plant defense responses with translational SWEET-GUS reporter lines. Our findings expand our understanding of Xoo infection routes and provide a valuable tool for resistance testing and pathogen surveillance.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.13.670017v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ESM-LoRA-Gly: Improved prediction of N- and O-linked glycosylation sites by tuning protein language models with low-rank adaptation (LoRA)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Feng, Z., Zhang, X., Wang, H., Hong, X., Zhan, J., Zhou, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce ESM-LoRA-Gly, a method that employs Low-Rank Adaptation (LoRA) to fine-tune the ESM2-3B protein language model for predicting both N- and O-linked glycosylation sites.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Glycosylation associates with many diseases ranging from cancer to neurodegeneration and understanding these disease mechanisms requires the precise identification of glycosylation sites. Computational prediction of glycosylation sites has been useful to complement laborious experimental methods, while existing tools lack sufficient accuracy and scalability. Here, we introduce ESM-LoRA-Gly, a method that employs Low-Rank Adaptation (LoRA) to fine-tune the ESM2-3B protein language model for predicting both N- and O-linked glycosylation sites. According to the evaluation on the benchmark datasets, ESM-LoRA-Gly outperforms existing state-of-the-art techniques. The improvement is particularly significant (>100% in Matthews correlation coefficient) for the O-linked dataset. By substantially reducing trainable parameters while maintaining predictive power, ESM-LoRA-Gly enables computationally efficient proteome-scale predictions. This approach should be instrumental for advancing glycoproteomic research and accelerating therapeutic discovery for glycosylation-related diseases.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.12.669850v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">VO: The Vaccine Ontology</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zheng, J., Lin, A. Y., Huffman, A., Masci, A. M., Racz, R., Wu, G., Roan, K., Ong, E., Sarntivijai, S., Hu, J., Asfaw, E., Kahn, H., Li, X., Zhang, X., Kosar, N., Li, J., Manuel, W., Abeysinghe, R., Rehana, H., Bansal, B., Pan, Y., Guo, J., He, V., Song, J., Seleznev, A. I., Hur, K., He, A., Davydov, A., Yang, Q., Vita, R., Peters, B., Ruttenberg, A., Diehl, A. D., Hoyt, C. T., Roncaglia, P., Huntley, R. P., Scheuermann, R. H., Courtot, M., Todd, T., Sayers, S., Chen, F., Li, X., Yeh, F.-Y., Xiang, Z., Ozgur, A., Whetzel, P. L., Musen, M. A., Mungall, C. J., Leitner, W. W., Cui, L., Colby, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed the community-based Vaccine Ontology (VO), which aligns with the Basic Formal Ontology and adheres to OBO Foundry principles, and greatly supports vaccine AI research in the Semantic Web era.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> With the widespread use of vaccines in research and clinical settings, there is an urgent need to standardize vaccine representation, integrate information across diverse vaccine types, and support computer-assisted reasoning. Accordingly, we have since 2007 developed the community-based Vaccine Ontology (VO), which aligns with the Basic Formal Ontology and adheres to OBO Foundry principles. VO models ontologically vaccines, vaccine components, vaccine immune responses, vaccine investigation studies and other vaccine-related topics. VO represents more than 10,000 vaccines targeting 289 infectious pathogens and cancers in humans and over 30 nonhuman animal species. VO provides mappings to external resources such as RxNorm, CVX, FDA, and USDA. Various VO use cases exist. VO facilitates vaccine standardization in resources such as the VIOLIN vaccine database, ImmPort, and the Vaccine Adjuvant Compendium (VAC). Semantic queries can be made to query VO. VO has been shown to enhance experimental and clinical vaccine data analysis and vaccine literature mining. Overall, VO standardizes vaccine modeling and representation and greatly supports vaccine AI research in the Semantic Web era.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.12.669998v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Prompt-to-Pill: Multi-Agent Drug Discovery and Clinical Simulation Pipeline</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Vichentijevikj, I., Mishev, K., Simjanoska Misheva, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> This study presents a comprehensive, modular framework for AI-driven drug discovery and clinical trial simulation, spanning from target identification to virtual patient recruitment.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> This study presents a comprehensive, modular framework for AI-driven drug discovery and clinical trial simulation, spanning from target identification to virtual patient recruitment. Synthesized from a systematic analysis of 51 LLM-based systems, the proposed Prompt-to-Pill architecture and corresponding implementation leverages a multi-agent system divided into preclinical and clinical phases, coordinated by a central Orchestrator. Each phase comprises specialized large language model (LLM) agents responsible for tasks such as molecular generation, toxicity screening, docking simulation, trial design, and patient matching. To demonstrate the full pipeline in practice, the well-characterized target Dipeptidyl Peptidase 4 (DPP4) was selected as a representative use case. The process begins with generative molecule creation and proceeds through ADMET evaluation, structure-based docking, and lead optimization. Clinical-phase agents then simulate trial generation, patient eligibility screening using EHRs, and predict trial outcomes. By tightly integrating generative, predictive, and retrieval-based LLM components, this architecture bridges preclinical discovery with virtual clinical development, offering an end-to-end demonstration of how LLM-based agents can operationalize the entire drug development workflow in silico.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.12.669861v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SALVE: prediction of interorgan communication with transcriptome latent space representation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pavelka, J., Voong, C. K., Schaal, P., Lam, M. P. Y., Lau, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A method to infer the associations between secretome-encoding transcripts and gene modules in a distal organ from RNA sequencing data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Massive transcriptomics data allow gene relationships to be discovered from their correlated expression. We describe SALVE, a method to infer the associations between secretome-encoding transcripts and gene modules in a distal organ from RNA sequencing data. This method builds upon similar bioinformatics approaches by introducing transcriptome latent space representations and transfer learning to simultaneously increase discovery power and predict downstream functional associations. Applied to GTEx v8 data, we show the method readily recapitulates canonical endocrine relationships, including insulin and adiponectin signaling, while inferring new candidate organokines and their signaling modality. We also explore its utility for generating new hypotheses on cardiokine candidates and finding distal factors that may affect cardiac protein synthesis and metabolism. The predictions suggest a potential role of circulating galectin-3 (LGALS3) in regulating cardiac protein synthesis and homeostasis, which can be recapitulated in part in human induced pluripotent stem cell (hiPSC)-derived cardiomyocytes. This method may aid in ongoing efforts to delineate interorgan communications and endocrine networks in various areas of study.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.12.669923v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">IMGT(R) at scale: FAIR, Dynamic and Automated Tools for Immune Locus Analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sanou, G., Zeitoun, G., Manso, T., Eidi, M., Grand, F., Kushwaha, A., Croze, M., Debbagh, C., Vaillant, A., Georga, M., Papadaki, A., Sideri, I., Batool, S., Samadova, T., Jabado-Michaloud, J., Folch, G., Giudicelli, V., Duroux, P., Kossida, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> IMGT(R), the international ImMunoGeneTics information system(R) has advanced its comprehensive platform for the analysis of immunoglobulin (IG) and T cell receptor (TR) genes through the development of new automated and scalable tools.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> IMGT(R), the international ImMunoGeneTics information system(R), has advanced its comprehensive platform for the analysis of immunoglobulin (IG) and T cell receptor (TR) genes through the development of new automated and scalable tools. This article presents major updates aligned with IMGT three axes of research. Axis I introduces dynamic resources such as IMGT/GeneTables, IMGT/AssemblyComparison, and IMGT/StatAssembly, enabling real-time access to annotated genomic data and quality assessment of assemblies. Axis II enhances repertoire analysis with a redesigned IMGT/GeneFrequency tool and new customization features in IMGT/V-QUEST, supporting flexible exploration of IG and TR gene expression. Axis III improves the accurate prediction of peptide-MHC thanks to IMGT/RobustpMHC. Additionally, the IMGT Knowledge Graph (IMGT-KG) and its therapeutic extension, IMGT/mAb-KG, provide semantically structured access to more than 100 million immunogenetic triplets, integrating IMGT databases and linking IMGT content to external biomedical resources. These developments promote standardization, interoperability, and integrative analysis across immunogenetics and clinical applications, reinforcing the role of IMGT as a core reference in the era of FAIR data and personalized medicine.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.12.669823v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Detection of Macrobrachium rosenbergii golda virus (MrGV) in giant river prawn larvae across Asia through NCBI sequence read archive (SRA) data mining</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hooper, C., van Aerle, R., Ryder, D., Coyle, N. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study to further characterise the geographical spread of MrGV globally, suggest the life stages of prawn that should be screened to prevent spread of the virus and demonstrate how the Logan database can be used to inform epidemiological studies.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Macrobrachium rosenbergii golda virus (MrGV) was first characterised in Macrobrachium rosenbergii larvae in 2020, associated with mass mortalities in multiple Southern Bangladesh prawn hatcheries. MrGV has since been detected in two metatranscriptomic datasets from healthy M. rosenbergii in China and in relation to a larval mortality event in India. The major objective of this study was to further characterise the geographical spread of the virus by mining the NCBI Sequence Read Archive (SRA) database. Utilising a new database, Logan, generated from assembling each SRA dataset, and a custom SnakeMake pipeline, we discovered MrGV sequence data in M. rosenbergii SRA datasets from China, Thailand, and India, and determined that presence and relative abundance of MrGV is mostly associated with the larval life stage of M. rosenbergii. These results provide insights into the current prevalence of MrGV globally, suggest the life stages of prawn that should be screened to prevent spread of the virus, and demonstrate how the Logan database can be used to inform epidemiological studies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.13.669918v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The Dawn of High-Throughput and Genome-Scale Kinetic Modeling: Recent Advances and Future Directions.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ilias Toumpe, Subham Choudhury, Vassily Hatzimanikatis, Ljubisa Miskovic</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A review of the state-of-the-art in kinetic modeling, highlighting the potential of these advances to drive progress in systems and synthetic biology, metabolic engineering, and medical research at an unprecedented scale and pace.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Researchers have invested much effort into developing kinetic models due to their ability to capture dynamic behaviors, transient states, and regulatory mechanisms of metabolism, providing a detailed and realistic representation of cellular processes. Historically, the requirements for detailed parametrization and significant computational resources created barriers to their development and adoption for high-throughput studies. However, recent advancements, including the integration of machine learning with mechanistic metabolic models, the development of novel kinetic parameter databases, and the use of tailor-made parametrization strategies, are reshaping the field of kinetic modeling. In this Review, we discuss these developments and offer future directions, highlighting the potential of these advances to drive progress in systems and synthetic biology, metabolic engineering, and medical research at an unprecedented scale and pace.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40262025/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

