<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Tuesday, 30 September 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Engineered antimicrobial-derived peptides to manipulate mixed microbial systems</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Trivedi, V. D., Van Deventer, J. A., Nair, N. U.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We develop molecular probes to manipulate specific subpopulations within multispecies microbial populations and validate these methods using model synthetic populations in vitro.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Due to complexity of microbial communities and coarseness of currently available manipulation techniques (e.g. transplantation, antibiotic-treatment), it is often difficult to fully elucidate the interactions between members that define community structure and function from the top-down. Thus, it is imperative to be able to observe and manipulate subpopulations within microbial communities to enable a fine-detail understanding of the full-spectrum and mechanism of community functions. However, there is a technological gap that prevents targeted manipulation of subpopulations within intact microbial mixtures and communities. In this work, we develop molecular probes to manipulate specific subpopulations within multispecies microbial populations and validate these methods using model synthetic populations in vitro. We leverage the narrow-spectrum of class II peptide bacteriocin (a bacterially-synthesized antimicrobial peptide), pediocin PA-1, as a model to develop molecular probes. We first demonstrate the narrow-spectrum activity of pediocin and quantify its potency against a panel of bacteria. Next, we conjugate chemical handles on the bacteriocin and show that the binding spectrum is largely unchanged. Finally, using truncated variants, also conjugated to chemical handles, we show functional non-bactericidal binders that largely maintain their specificity. Finally, with the unmodified and modified bacteriocins, we show that specific bacteria can be depleted through killing or cell sorting within a mixture of highly similar bacteria. While we developed the system with a single bacteriocin, we expect that the elucidated design rules may be applicable to a variety of natural bacteriocins to develop a generalized approach for manipulating specific bacterial members within a community. Development of such molecular probes would be transformative to advancing mechanistic underpinnings of microbial community and microbiota structure-function relationships.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.29.679234v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ChromPolymerDB: A High-Resolution Database of Single-Cell 3D Chromatin Structures for Functional Genomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chen, M., Du, L., Zhao, S., Ye, B., Delafrouz, P., Farooq, H., Chattopadhyay, D., Marai, G. E., Shao, Z., Liang, J., Czajkowsky, D. M., Chronis, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a polymer physics-based modelling framework, the Sequential Bayesian Inference Framework (sBIF), that deconvolutes bulk Hi-C data to reconstruct single-cell 3D chromatin conformations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The three-dimensional (3D) organization of chromatin plays a critical role in regulating gene expression and genomic processes like DNA replication, repair, and genome stability. Although these processes occur at the individual-cell level, most chromatin structure data are derived from population-averaged assays, such as Hi-C, obscuring the heterogeneity of single-cell conformations. To address this limitation, we developed a polymer physics-based modelling framework, the Sequential Bayesian Inference Framework (sBIF), that deconvolutes bulk Hi-C data to reconstruct single-cell 3D chromatin conformations. To support a broader use of sBIF, we created ChromPolymerDB, a publicly accessible, high-resolution database of single-cell chromatin structures inferred by sBIF. The database contains ~10^8 reconstructed 5 kb-resolution single cell structures, spanning over 60,000 genomic loci across 50 human cell types and experimental conditions. ChromPolymerDB features an interactive web interface with tools for 3D structural analysis and multi-omics integration. Users can explore associations between chromatin conformation and gene expression, epigenetic modifications, and regulatory elements. The platform also supports comparative analyses to identify structural changes across cell types, developmental stages, or disease contexts. ChromPolymerDB offers a unique resource for researchers studying the relationship between genome architecture and gene regulation, and for advancing comparative 3D genomics. ChromPolymerDB is available online at https://chrompolymerdb.bme.uic.edu/.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.27.678816v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Graph-Attentive GAN for Rare-Cell-Aware Single-Cell RNA-Seq Data Generation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ganguly, R., Aafrine, S., Hossain, S. M. M., Ray, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce GARAGE, a Graph-Attentive RAre-cell aware single-cell data GEneration that augments the generator's input with a small, attention-weighted 'leakage' of real cells in addition to prior noise.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> A central challenge in downstream single-cell RNA sequencing (scRNA-seq) analysis is the high-dimensional, small-sample (HDSS) regime, often compounded by class imbalance from rare cell types. These factors hinder robust feature (gene) selection and cell clustering and limit the realism of samples generated by existing simulators. We introduce GARAGE, a Graph-Attentive RAre-cell aware single-cell data GEneration that augments the generator's input with a small, attention-weighted 'leakage' of real cells in addition to prior noise. Specifically, we build a k-nearest-neighbour cell graph and use a graph attention network (GAT) to prioritize nodes that likely represent under-sampled (rare) subpopulations; these high-attention cell embeddings are injected into the generator input to steer synthesis toward biologically plausible regions of the data manifold while respecting cell-type proportions. This attention-guided leakage accelerates training, reduces mode dropping, and yields realistic synthetic cells that preserve rare-cell structure. Across real scRNA-seq benchmarks, GARAGE improves downstream feature selection and clustering compared with state-of-the-art baselines. In summary, GARAGE directly addresses HDSS and rarity in scRNA-seq by coupling graph attention with adversarial generation to produce high-fidelity synthetic cells that enhance downstream analyses.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.28.679012v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">scPhoenix: A Contrastive Learning-based Framework with Aux-Core Feature Disentanglement Enhances Sparse Cellular Multi-omics Translation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yan, C., He, Z., Ye, L., Zheng, S., Lin, N., Li, G., Li, W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present scPhoenix, a contrastive learning-based framework for single-cell cross-modality translation that preserves cellular heterogeneity during translation and demonstrates significant advantages for data with high sparsity.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent advances in single-cell multi-omics co-assays and spatiotemporal sequencing technologies have provided unprecedented opportunities for systematically characterizing cellular heterogeneity. However, severe sparsity and pronounced spatial heterogeneity - hallmark features of complex diseases and tumor microenvironment - remain major obstacles in deciphering cellular multi-omics data. Here, we present scPhoenix, a contrastive learning-based framework for single-cell cross-modality translation. scPhoenix adopts a two-stage Aux-Core strategy to disentangle modality-specific feature extraction from cross-modality feature interaction. Across diverse datasets, it preserves cellular heterogeneity during translation and demonstrates significant advantages for data with high sparsity. In addition, the framework integrates a contrastive learning framework with five effective data augmentation methods tailored to single-cell data. Moreover, scPhoenix's design supports extensions to unpaired data training and spatial multi-omics translation, enabling robust performance in scenarios with high spatial heterogeneity. scPhoenix is freely available at https://github.com/liwz-lab/scPhoenix.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.27.678942v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">BindPred: A Framework for Predicting Protein-Protein Binding Affinity from Language Model Embeddings</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Piao, H., Boorla, V., Santra, S., Maranas, C. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce BindPred, a structure-agnostic input framework that predicts affinities directly from amino acid sequences by combining embeddings from large protein language models with gradient boosting trees.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Reliable predictions of protein-protein binding affinities are essential for molecular biology and therapeutic discovery. However, most computational methods rely on three-dimensional structural models, which are often unavailable for many complexes. We introduce BindPred, a structure-agnostic input framework that predicts affinities directly from amino acid sequences by combining embeddings from large protein language models with gradient boosting trees. On the PPB-Affinity benchmark, which comprises 11,919 diverse complexes, BindPred achieves a Pearson correlation coefficient of 0.86 in random split five-fold cross-validation, where the training and test sets share <30% global sequence identity. Ablation analysis indicates that evolutionary embeddings alone capture most of the predictive signals, while augmenting with physics-based energy terms from PyRosetta and BindCraft increases the correlation by only 0.01. A more stringent protein-level split that places entire protein families (wild-type and all mutants) exclusively in either training or testing sets, results in only a modest decline in performance, demonstrating robust generalization to novel interaction pairs. Because BindPred operates exclusively on sequence input, it enables rapid inference (approximately 3 million complexes per GPU (T4) hour), making proteome-scale screening computationally feasible.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.27.678407v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Unveiling the evolutionary code of NOTCH3: mammalian bioinformatics sheds light on human pathogenicity.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Siddiqui, K. S., Ertan, H., Ren, Y., Plojak, A., Jayasena, T., Sachdev, P. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the first large-scale comparative bioinformatic analysis of a highly conserved transmembrane receptor implicated in CADASIL, a hereditary small vessel disease driven by mutations in its extracellular EGF-like repeats.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> NOTCH3 is a highly conserved transmembrane receptor implicated in CADASIL, a hereditary small vessel disease driven by mutations in its extracellular EGF-like repeats. The mechanism by which these mutations cause pathology remains unclear. We present the first large-scale comparative bioinformatic analysis of NOTCH3 across 113 mammalian species, uncovering three novel insights: i) a remarkable evolutionary conservation of all 204 cysteines, with the only exception being eight naturally occurring cysteine mutations in jaguar (EGFr13-15); ii) a unique deletion in Brandt's bat regulatory region, which may expose it to proteases, potentially altering signaling; iii) a rare human NOTCH3-X1 isoform, absent in most mammals but shared with select primates, a bat, and elephants, involving a cysteine-depleting deletion spanning EGFr20-22. These features provide novel evolutionary insights into human pathogenicity and suggest testable targets for in vivo experiments. Our study highlights the potential of comparative bioinformatics to identify previously hidden functional elements in disease-associated mammalian proteins.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.28.678708v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Horus: Reproducible Workflow Manager for Biomolecular Modelling</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dominguez-Dalmases, C., Canellas-Sole, A., Lambea-Jane, R., Municoy, M., Guallar, V.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Horus is a versatile, multi-platform reproducible-workflow management system designed for biomolecular modelling and computational life sciences.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Horus is a versatile, multi-platform reproducible-workflow management system designed for biomolecular modelling and computational life sciences. Available as both a desktop application and a web-based service, Horus offers researchers an intuitive graphical interface alongside a robust Python API, democratising the creation, execution, and management of complex scientific workflows. Its modular architecture uses individual blocks that can be easily customised and extended, allowing for an integration with diverse computational tools and resources. Horus supports remote execution via SSH to efficiently dispatch tasks to high-performance computing clusters in remote machines. The platform also features integrated 3D and 2D molecular visualizers for interactive structure analysis, which enhances the interpretability of simulation results. Its design emphasises reproducibility, scalability, and user-friendliness, positioning Horus as a valuable tool to advance research in structural bioinformatics and related fields.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.26.678751v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Recurrent enhancer-promoter interactions across samples</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Weston, M., Gunjala, S., Hu, H., Li, X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Most human enhancer-promoter interactions are already captured and reuse subsets of these shared EPIs across different cell types and conditions, with important implications for chromatin conformation studies.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Enhancer-promoter interactions (EPIs) are fundamental to gene regulation, and understanding their recurrence across diverse biological samples is key to deciphering chromatin architecture. In this study, we systematically analyzed the recurrence of EPIs across 49 Hi-C and 95 HiChIP datasets. We found that the majority of EPIs identified in a given sample were also present in other samples, regardless of the assay type (Hi-C or HiChIP) or the enhancer annotations used. Interestingly, EPIs that appeared unique to individual samples were typically surrounded by fewer neighboring EPIs, suggesting they may not represent truly sample-specific interactions. Our findings indicate that most human EPIs have already been captured and that cells primarily reuse subsets of these shared EPIs across different cell types and conditions. This study provides new insights into the pervasive and reusable nature of EPIs in the human genome, with important implications for chromatin conformation studies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.27.678855v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multi-omics integration at cell type resolution uncovers gene-metabolite mechanisms underlying osteoarthritis heterogeneity</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ojha, A., Chatterjee, P., Kippner, L. E., Stevens, H. Y., Yeago, C., Drissi, H., Mautner, K., Boden, S., Roy, K., Sinha, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We reconstruct networks of gene-metabolite associations in osteoarthritis patients and identify key molecular factors of osteo arthritis heterogeneity.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Metabolic dysregulation is an important factor for osteoarthritis pathogenesis, but comprehensive studies of underlying mechanisms and pathways are rare. We analyzed newly generated metabolomics data on bone marrow from 119 osteoarthritis patients, along with single-cell transcriptomics data to reconstruct networks of gene-metabolite associations at cell type resolution. Hubs of these networks, cell type-specific as well as pan-cell type hubs, revealed key molecular factors of osteoarthritis heterogeneity. Systems-level analysis of hubs revealed major roles for glycerophospholipid, glycerolipid and sphingolipid metabolism pathways, along with lipid signaling. We used Machine Learning models of gene-metabolite relationships to discover cell types most relevant to each metabolite. Integrative analysis of disease severity scores along with multi-omics data revealed a shift in specific immune cell subtypes in low versus high grade disease. We conclude that leveraging gene-metabolite covariation in a patient cohort can uncover underlying molecular mechanisms, overcoming the challenges posed by high dimensionality of multi-omics data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.26.678886v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Disruption of the Wnt-antagonist Apc in the pituitary stem cells drives the development of adamantinomatous craniopharyngioma.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Blackburn, J., Gomez-Corral, L., Nicholson, J. G., Rai, A., Xue, J., Tan, R., Gangaraju, K. C., Gualtieri, A., Hall, C., Dutta, P., Korbonits, M., Roncaroli, F., Davis, S. W., Gaston-Massuet, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that genetic disruption of the Wnt-antagonist and tumour suppressor, adenomatous polyposis coli (Apc), within the pituitary progenitors/stem cells, leads to pituitsary tumours that closely resemble human aCPs.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Adamantinomatous craniopharyngiomas (aCPs) are complex intracranial neoplasms that generally arise in the sellar and suprasellar region of the brain which affect the endocrine and nervous systems causing severe sequelae. Activating mutations resulting in degradation-resistant forms of {beta}-catenin (CTNNB1) have been shown to be the main driver for many of these neoplasms. However, the underlying genetic driver for a proportion of these tumours is still unknown. Using murine transgenic models, we show that genetic disruption of the Wnt-antagonist and tumour suppressor, adenomatous polyposis coli (Apc), within the pituitary progenitors/stem cells, leads to pituitary tumours that closely resemble human aCPs. These tumours present classic histopathological hallmarks of aCPs, such as large deposits of wet keratin, stellar-reticular-like cells, large cystic components and clusters of accumulating nucleo-cytoplasmic {beta}-catenin that are slow-dividing and exhibit a secretory phenotype. We show that a hypomorphic allele of Apc is sufficient for tumour development, indicating that disruption of Apc function can lead to aCP formation. Moreover, we identify that bi-allelic loss of Apc in the Sox2+ve pituitary stem cells is sufficient to initiate tumour formation, indicating that Sox2+ve stem cells are the cell origin of these Apc-driven aCPs. Transcriptomic analyses of early tumour-initiating cells revealed that Apc-driven clusters of accumulating {beta}-catenin undergo senescence-associated secretory phenotype (SASP), which is p21-mediated and results in secretion of inflammasome, angiosome and developmental growth factors. Our data unequivocally show a causal role for the disruption of the tumour suppressor Apc as a primary driver of aCPs independent of mutations in {beta}-catenin. We provide murine models representing a novel genetic subtype of human aCPs, offering insights into aCP pathogenesis. Our work reinforces the importance of genetic testing for mutations in APC in patients with aCPs and identifies a potential need to screen patients with familial adenomatous polyposis (FAP) or spectrum of APC-pathogenic syndromes for aCPs in early life.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.28.679021v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">High-Dimensional Spatiotemporal Single-Cell and 3D Atlas of the Bone Marrow Microenvironment during Leukemic Progression</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Li, L., Rottmann, I., Ivison, G., Wei, H., Schroeder, J. C., Dunkel, G., Goltsev, Y., Nolan, G. P., Mayer, A. T., Saeed, B. R., Weigelin, B., Schuerch, C. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A spatiotemporal single-cell atlas of the bone marrow microenvironment during leukemogenesis .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The bone marrow microenvironment (BMME) is essential for hematopoiesis and immunity, yet spatiotemporal single-cell analysis during leukemogenesis remains challenging. We characterized the BMME in femurs from wild-type and chronic myeloid leukemia (CML) mice at 7, 14 and 21 days post-induction by highly multiplexed and 3D microscopy. Using a 54-marker CODEX panel, we profiled 2,033,725 cells in 55 tissue regions of interest and identified 41 cell types through unsupervised clustering and supervised annotation. During leukemic progression, we observed an expansion of myeloid and progenitor cell populations, increased PD-L1+ leukemic cells, the upregulation of PD-1 on CD4+ and CD8+ T cells, and a profound loss of B cells, plasma cells and bone cells. Advanced CML exhibited a striking expansion of immature, pericyte-deficient vasculature that disrupted vascular niches and impaired hematopoietic stem/progenitor cell positioning. Spatial mapping revealed leukemia-specific cellular neighborhoods enriched in PD-1+CD8+ T cells, suggesting localized immune cell exhaustion. Early-stage CML showed increaseds between plasmacytoid dendritic cells and megakaryocytes, whereas advanced CML featured heightened megakaryocyte emperipolesis of non-leukemic granulocytes. Megakaryocytes were morphologically irregular in CML mice and BM trephine biopsies from CML patients. Laser-capture microdissected megakaryocytes from newly diagnosed CML patients had reduced expression of cytoskeleton genes, which was reversed in advanced cases treated with tyrosine kinase inhibitors. 3D imaging revealed vascular disorganization and depleted megakaryocytes in the diaphysis, underscoring region-specific pathology. Together, this study provides a spatiotemporal single-cell atlas of the BMME during leukemic progression, showing how leukemic cells reprogram the niche to support their expansion and immune evasion.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.27.678113v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Establishing a Canine Urothelial Organoid Repository: A Platform for Comparative and Translational Carcinoma Studies</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zdyrski, C., Pawlak, A., Nicholson, H. F., Corbett, M. P., Catucci, M., Cheville, J., Peng, J., Saba, C., Hamsher, H., Friedenberg, S. G., Woodward, A. P., Douglass, E., Mochel, J. P., Allenspach, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We describe the largest known canine urothelial carcinoma organoid bioarchive, with in-depth phenotypical and molecular characterization, with the potential to expand personalized medicine approaches.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Despite the increasing number of treatment options for patients with muscle-invasive bladder Cancer (MIBC), many of these patients ultimately have a poor prognosis. Drug responses vary considerably among patients and in many who respond initially, drug resistance may ultimately develop leading to tumor progression. Scope for improvement has been limited by the phenotypic and molecular diversity of MIBC, which impacts the selection pressures of therapy. To address this translational gap, we describe the largest known canine urothelial carcinoma organoid bioarchive, with in-depth phenotypical and molecular characterization. Immunohistochemistry confirmed the expression of key biomarkers including UPKIII, E-cadherin, Vimentin, and Ki-67 in organoids. Single-nucleus RNA sequencing revealed cellular heterogeneity, while bulk RNA sequencing showed canine patient-to-patient variability in transcriptomic profiling. Bulk RNA sequencing also displayed highly similar expression profiles between tissue- and urine-derived organoids, as well as similarity to previously published human MIBC transcriptome data. Whole genome sequencing in a subset of patients further supported the overall genomic fidelity of the organoids with their tissue of origin. Finally, for proof-of-concept of usefulness for drug testing studies, organoid cytotoxicity to vinblastine was investigated in several organoid lines. These canine bladder cancer organoid lines can be thawed, expanded, and screened for response to potential novel therapeutics to expand personalized medicine approaches. In addition, the organoid lines can serve as a valuable resource for comparative bladder cancer research, and as a pre-clinical screening tool to identify efficacious drugs before taking them into canine clinical trials to support future human clinical trials.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.27.678962v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Microbiota and Small Cell Lung Cancer.A casual bystander or a hidden culprit?</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rolim, I., Lopez-Beltran, A., Pantarotto, M., de Sousa, E., Sobral, J., Farver, C., Gil, N., Penha-Goncalves, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that small cell lung carcinoma has a distinct microbial signature that is distinct from other lung pathologies and uncovers a novel link between the biotrophic plant pathogenic Taphrina and human cancer.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The tumor-associated microbiome is a key player in cancer development, progression, prognosis, and therapeutic response. Notably, distinct microbial signatures have been identified across cancer types. Small cell lung carcinoma (SCLC) accounts for approximately 15% of all lung cancer, yet its microbiome remains unclear. Analyzing the bacteriome composition in tissue from ten SCLC cases and in 10 cases of a heterogenous lung pathology group, we found a distinct microbial signature associated with SCLC with significantly lower diversity and higher dissimilarity, characterized by a higher relative abundance of Firmicutes and Bacteroidota, and a markedly different set of dominant genera (Pseudomonas, Streptococcus and Haemophilus) resulting in an increased Proteobacteria-to-Actinobacteria ratio. Unexpectedly, mycobiome analysis comparing pooled samples of these SCLC cases with ten pooled lung adenocarcinoma (LUAD) cases revealed that the fungal genus Taphrina was uniquely represented in SCLC. Strikingly, mycobiome individual analysis of twenty-one additional SCLC cases compared with 10 LUAD cases showed an increased prevalence of Taphrina sp. in SLCL tissue. Overall, the results suggest that SCLC microbiome is distinct from other lung pathologies and uncovers a novel link between the biotrophic plant pathogenic Taphrina and human cancer.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.26.678903v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Therapeutic Blockade of Type 2 Cytokines and PD1 Unleashes Anti-Tumor Immunity Through Coordinated Reprogramming of Innate and Adaptive Immune Surveillance</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fabre, T., Hendrickson, E., McGourty, K., Marcovecchio, P., McMahon, J., Patti, S., Cage, K., Searles, S., Widjaja, C., Stairiker, C., Bound, K., Martino, R., Pyszczynski, E., Zhang, Y., Thomas, G., Stoner, S., Ching, K., Jin, F., Dillon, C., Wynn, T., Gieseck, R. L., Barron, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that IL-4, IL-13, TSLP and PD1 blockade can improve tumor growth inhibition in cancer patients alone and in combination with checkpoint inhibitors.</p><p style="font-size: 16px; text-align: justify;">Background: Checkpoint inhibitors improve survival in patients with several types of tumors. However, resistance to checkpoint inhibitors creates an opportunity for patients to benefit from novel immunotherapies. The type 2 cytokines IL-4, IL-13 and TSLP have been implicated in suppressing anti-tumor immune responses through T and myeloid cells. Our current study tested whether combined therapeutic blockade of IL-4, IL-13 and TSLP improved anti-tumor immunity alone and in combination with PD1 antagonism. Methods: We used in vitro experiments with primary cells to identify cell types likely to participate in controlling tumors upon IL-4, IL-13, TSLP and PD1 blockade. Therapeutic blockade in the subcutaneous CT26 model tested in vivo tumor growth inhibition and associated immunological changes. Bioinformatic analysis of human tumor bulk RNA sequencing data probed for survival associations with IL-4/IL-13 and TSLP transcriptional responses. Results: In vitro, IL-4 suppressed T cell-mediated tumor growth inhibition and reduced monocyte-derived dendritic cell expression of proteins associated with anti-tumor immunity. In vivo, blocking IL-4, IL-13, TSLP and PD1 improved tumor growth inhibition by creating hotter tumors. This was associated with repolarization of CD4 and CD8 T cells and shifts in monocyte, conventional type 1 and type 2 (or monocyte-derived) dendritic cell programs. Transcriptional responses to IL-4/IL-13 and TSLP were associated with poor survival outcomes across patients with several types of cancers. Conclusion: Therapeutic blockade of IL-4, IL-13 and TSLP may drive immunological tumor growth inhibition in subsets of cancer patients alone and in combination with checkpoint inhibitors. Improved tumor growth inhibition was likely driven through augmented cytotoxic T cell priming in secondary lymphoid organs and improved reactivation by repolarized monocytes and dendritic cells in tumors.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.26.678820v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Plasma proteome correlations with liver stiffness in pediatric cholestasis implicate epithelial to mesenchymal transition.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Benjamin L Shneider, Rupa S Kanchi, Sandra L Grimm, Sridevi Devaraj, Juliet Emamaullee, Jeremy Schraw, Philip J Lupo, Jorge A Bezerra, Kathleen M Loomes, John C Magee, Ronald J Sokol, Kasper S Wang, Alyssa Kriegmeier, Evelyn Hsu, Jean P Molleston, Philip Rosenthal, Rohit Kohli, Saul J Karpen, Simon P Horslen, M Kyle Jensen, Arianna Barbetta, Cristian Coarfa</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A cross-sectional analysis of vibration-controlled elastography in biliary atresia, alpha-1 antitrypsin deficiency, and Alagille syndrome to interrogate the plasma proteome relative to liver stiffness measurements .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Pediatric cholestatic liver diseases can be characterized by rapidly progressive fibrosis. A multicenter cross-sectional analysis of vibration-controlled elastography in biliary atresia (BA), alpha-1 antitrypsin deficiency (A1AT), and Alagille syndrome (ALGS) was leveraged to interrogate the plasma proteome relative to liver stiffness measurements (LSM).</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41021277/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multiomics combined with machine learning defines unique molecular subtypes of cholangiocarcinoma and identifies TNK1 as a therapeutic target.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dong-Gi Mun, Erik Jessen, Jennifer L Tomlinson, Danielle Carlson, Rohit Budhraja, Roberto Alva-Ruiz, Amro Abdelrahman, Ryan Watkins, Lindsey Gregory, Chantal McCabe, Chen Wang, Rondell P Graham, Kathryn Woods, Martin Golkowski, Marshall Baker, Gregory J Gores, Sumera I Ilyas, Caitlin Conboy, Ellen L Larson, Jack W Sample, Enis H Ozmert, Richard K Kandasamy, Mitesh J Borad, Lewis Roberts, Joshua Andersen, Akhilesh Pandey, Rory L Smoot</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We study the molecular heterogeneity of cholangiocarcinoma using multiomic characterization and machine learning to uncover new biological signals.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cholangiocarcinoma (CCA) is one of the most lethal cancers, characterized by molecular heterogeneity and treatment resistance. To uncover new biological signals and therapeutic opportunities, we employed multiomic characterization combined with machine learning.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41020779/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

