<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Thursday, 21 August 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SciToolAgent: a knowledge-graph-driven scientific agent for multitool integration.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Keyan Ding, Jing Yu, Junjie Huang, Yuchen Yang, Qiang Zhang, Huajun Chen</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present SciToolAgent, a large language model-powered agent that automates hundreds of scientific tools across biology, chemistry and materials science.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Scientific research increasingly relies on specialized computational tools, yet effectively utilizing these tools requires substantial domain expertise. While large language models show promise in tool automation, they struggle to seamlessly integrate and orchestrate multiple tools for complex scientific workflows. Here we present SciToolAgent, a large language model-powered agent that automates hundreds of scientific tools across biology, chemistry and materials science. At its core, SciToolAgent leverages a scientific tool knowledge graph that enables intelligent tool selection and execution through graph-based retrieval-augmented generation. The agent also incorporates a comprehensive safety-checking module to ensure responsible and ethical tool usage. Extensive evaluations on a curated benchmark demonstrate that SciToolAgent outperforms existing approaches. Case studies in protein engineering, chemical reactivity prediction, chemical synthesis and metal-organic framework screening further demonstrate SciToolAgent's capability to automate complex scientific workflows, making advanced research tools accessible to both experts and nonexperts.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40835791/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A single-cell transcriptomic atlas of the pigtail macaque placenta in late gestation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Li, A., Li, R. J., Huang, H., Zhao, H., Del Rosario, B. M., Li, M., Li, E., Vo, A. E., Manuel, G., Cervantes, O., Kapur, R. P., Munson, J., Orvis, A., Coleman, M., Berg, M. R., Curtis, B. C., Menz, B. A., Dai, J., Golez, I. D., Wangari, S. N., English, C., Baldessari, A. E., Rajagopal, L., Cornelius, J. A., Adams Waldorf, K. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A single-cell RNA-Seq atlas of the placenta from the pigtail macaque in the third trimester that provides a powerful resource for understanding experimental perturbations on the NHP placentas.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The placenta is a complex organ with multiple immune and non-immune cell types that promote fetal tolerance and facilitate the transfer of nutrients and oxygen. The nonhuman primate (NHP) is a key experimental model for studying human pregnancy complications, in part due to similarities in placental structure, which makes it essential to understand how single-cell populations compare across the human and NHP maternal-fetal interface. We constructed a single-cell RNA-Seq (scRNA-Seq) atlas of the placenta from the pigtail macaque (Macaca nemestrina) in the third trimester, comprising three different tissues at the maternal-fetal interface: the chorionic villi (placental disc), chorioamniotic membranes, and the maternal decidua. Each tissue was separately dissociated into single cells and processed through the 10X Genomics and Seurat pipeline, followed by aggregation, unsupervised clustering, and cluster annotation. Next, we determined the maternal-fetal origins of cell populations and analyzed single-cell RNA trajectory, Gene Ontology enrichment, and cell-cell communication. Single-cell populations in the pigtail macaque were strikingly similar in their identity and frequency to those found in the human placenta, including cells from trophoblast, stromal cell, immune, and macrophage lineages. An advantage of our approach was the deep sequencing of three tissues at the maternal-fetal interface, which yielded a rich diversity of common and rare single-cell populations. The third trimester pigtail macaque single-cell atlas enables the identification of cellular subclusters analogous to those in humans and provides a powerful resource for understanding experimental perturbations on the NHP placenta.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.15.669966v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A predicted structural interactome reveals binding interference from intrinsically disordered regions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Peng, J., Zhao, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We found that functional associations contribute significantly to high-confidence predictions of protein-protein interactions from both physical and functional datasets in Drosophila, and highlight the importance of intrinsically disordered regions in the predicted high- confidence interactions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Proteins function through dynamic interactions with other proteins in cells, forming complex networks fundamental to cellular processes. While high-resolution and high-throughput methods have significantly advanced our understanding of how proteins interact with each other, the molecular details of many important protein-protein interactions are still poorly characterized, especially in non-mammalian species, including Drosophila. Recent advancements in deep learning techniques have enabled the prediction of molecular details in various cellular pathways at the network level. In this study, we used AlphaFold2 multimer to examine and predict protein-protein interactions from both physical and functional datasets in Drosophila. We found that functional associations contribute significantly to high-confidence predictions. Through detailed structural analysis, we also found the importance of intrinsically disordered regions in the predicted high-confidence interactions. Our study highlights the importance of disordered regions in protein-protein interactions and demonstrates the importance of incorporating functional interactions in predicting physical interactions between proteins. We further compiled an interactive web interface to present the predictions, facilitating functional exploration, comparative analysis, and the generation of mechanistic hypotheses for future studies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.15.670535v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Gene model for the ortholog of Ilp3 in Drosophila eugracilis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> O'Brien, J., Dufur, R., Kotrba, M., Severn, J., Wittke-Thompson, J., Rele, C., Fleet, C. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Gene model for the ortholog of Insulin-like peptide 3 (Ilp3) in the Drosophila eugracilis .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Gene model for the ortholog of Insulin-like peptide 3 (Ilp3) in the D. eugracilis Apr. 2013 (BCM-HGSC/Deug_2.0) (DeugGB2) Genome Assembly of D. eugracilis (GenBank Accession: GCA_000236325.2) of Drosophila eugracilis. This ortholog was characterized as part of a dataset to study the evolution of the Insulin/ insulin-like growth factor signaling pathway (IIS) across the genus Drosophila using the Genomics Education Partnership gene annotation protocol for Course-based Undergraduate Research Experiences.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.16.670678v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">rbio1-training scientific reasoning LLMs with biological world models as soft verifiers</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Istrate, A.-M., Milletari, F., Castrotorres, F., Tomczak, J. M., Torkar, M., Li, D., Karaletsos, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce rbio-1, a reasoning model for biology that is post-trained from a pretrained LLM using reinforcement learning and uses learned models of biology to obtain biological knowledge for verification during training.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Reasoning Models are typically trained against verification mechanisms in formally specified systems such as code or symbolic math. However, in open domains like biology, we do not generally have access to exact rules facilitating formal verification at scale, and oftentimes resolve to testing hypotheses in the lab to assess the validity of a prediction. Verification by performing real experiments is slow, expensive, and inherently does not scale with computation. In this work, we show that one can use world models of biology or other prior knowledge as approximate oracles over biological knowledge to utilize as soft verification to train reasoning systems without the need for additional experimental data. We introduce rbio-1, a reasoning model for biology that is post-trained from a pretrained LLM using reinforcement learning and uses learned models of biology to obtain biological knowledge for verification during training. We show that soft verification successfully distills biology world models into rbio, at the example of achieving leading performance on perturbation prediction against the PerturbQA benchmark compared to state-of-the-art models; we demonstrate the benefits of compositions of verifiers to learn more general rbio models. We believe rbio provides a proof of concept that demonstrates that predictions from bio-models can be used to train powerful reasoning models using simulations, rather than experimental data, as a new training paradigm.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.18.670981v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Metabolomics-Guided Machine Learning Reveals Diagnostic and Mechanistic Biomarkers in CHB with MASLD</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, C., Chen, Y., Xiao, H., Cai, J., Wang, R., Zeng, X., Lin, M., Liu, W., Chi, X., Chen, Q.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify metabolic signatures of chronic hepatitis B-related steatotic liver disease (MASLD) and construct a predictive model using untargeted metabolomics integrated with machine learning.</p><p style="font-size: 16px; text-align: justify;">Background: Metabolic dysfunction-associated steatotic liver disease (MASLD) often coexists with chronic hepatitis B (CHB), yet early diagnosis remains challenging, particularly in non-obese patients or those with subclinical features. This study aimed to identify metabolic signatures of CHB-related MASLD and construct a predictive model using untargeted metabolomics integrated with machine learning. Methods: Serum metabolomics was performed on 160 subjects (80 CHB + MASLD and 80 healthy controls). Differential metabolites were identified and analyzed using KEGG enrichment and 4 machine learning algorithms (Random Forest, XGBoost, SVM, and LASSO). Metabolite-clinical correlations and diagnostic model performance were evaluated. Results: A total of 924 differential metabolites were identified, with significant enrichment in pathways related to the TCA cycle, sphingolipid metabolism, and amino acid turnover. Machine learning prioritized six robust and biologically relevant metabolites - L-aspartic acid, succinic acid, caproic acid, sebacic acid, monomenthyl succinate, and glycolaldehyde, which consistently distinguished CHB + MASLD patients from controls (AUC > 0.75). These metabolites reflect key disruptions in mitochondrial function, lipid oxidation, and redox homeostasis. Integrated models combining metabolomics with clinical indices achieved perfect classification (AUC = 1.000). Conclusion: CHB-associated MASLD is driven by systemic metabolic remodeling centered on mitochondrial overload, oxidative stress, and impaired amino acid metabolism. The identified metabolites provide mechanistic insights and hold promise for non-invasive MASLD screening in CHB patients. This study underscores the potential of multi-algorithmic metabolomics in advancing early diagnosis and personalized management of complex liver comorbidities.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.19.671181v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Three-dimensional spatial transcriptomics at isotropic resolution enabled by generative deep learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Li, B., Bao, F., Sun, Y., Wang, T., Li, F., Li, H., Dai, Q., Deng, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that 3D, organ-wide, isotropic-resolution 3D gene expression landscapes can be effectively inferred from sparse 2D spatial transcriptomics data using artificial intelligence, enabling the comprehensive characterization of tissue niches at full 3D resolution.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mapping the complete three-dimensional (3D), transcriptome-wide spatial architecture of tissues and organs remains a fundamental challenge in biology. Typically, studies approximate 3D structures by profiling serial two-dimensional (2D) tissue sections, resulting in incomplete and anisotropic reconstructions. Here, we demonstrate that organ-wide, isotropic-resolution 3D gene expression landscapes can be effectively inferred from sparse 2D spatial transcriptomics data using artificial intelligence. Our framework, isoST, leverages stochastic differential equations (SDEs) to model the intrinsic biological continuity of gene expression along tissue depth, enabling the integration of discrete 2D measurements into coherent 3D transcriptomic maps. isoST is applied across diverse biological systems, including the brain, kidney, spinal cord, and gastrulating embryos, accurately reconstructing anatomical structures and molecular patterns that are only partially resolved in 2D analyses. isoST enables the comprehensive characterization of tissue niches at full 3D resolution, uncovering radial and spherical gene expression gradients associated with disease progression that are often distorted or missed in 2D approaches. Furthermore, isoST is extended to integrate histological imaging for reducing experimental complexity and to reconstruct high-resolution developmental spatial trajectories over time. Together, isoST provides a practical and scalable solution toward comprehensive high-resolution 3D spatial transcriptomic atlases.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.15.670472v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Mathematical Foundations of Beta Diversity: Why Common Metrics Fail in Microbiome Analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhu, Z., Zhang, Y., Li, W., Greenacre, M., Satabdi, S., Shi, Y., Zhang, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We systematically evaluate the commonly used beta diversity measures for microbiome studies and establish a unified framework to ensure theoretical rigor and validity, including whether they are true metrics, conform to Euclidean geometry, and satisfy conditional negative definiteness.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In microbiome studies, beta diversity quantifies dissimilarity between samples and is often visualized using ordination techniques. It enables researchers to characterize ecological structure, compare microbial communities, assess environmental or host-driven heterogeneity, and track longitudinal shifts over time. Although many diversity indices were originally developed with practical goals in mind, they lack a unified framework to ensure theoretical rigor and validity. This gap makes it challenging for researchers to evaluate and select appropriate beta diversity measures for microbiome analyses, potentially leading to biased analyses and invalid conclusions. To bridge the persistent knowledge gaps, we systematically evaluate the commonly used beta diversity measures according to key mathematical properties, including whether they are true metrics, conform to Euclidean geometry, and satisfy conditional negative definiteness. We show that their violations can compromise downstream analyses such as PCoA, PERMANOVA, and kernel-based tests. In addition, drawing on mathematical consensus, we introduce a novel four-category classification of beta diversity measures: scale difference, difference scale, Hamming difference, and distribution difference. Complementing this framework, we build diagnostic tools for assessing Euclidean validity and develop remedial strategies that correct problematic dissimilarity matrices while preserving ordination structures. We demonstrate the effectiveness of these solutions using real-world microbiome datasets. These results establish a unified framework for evaluating beta diversity in microbiome research, supported by an R package, interactive Shiny app, and step-by-step tutorials. The framework provides a clear roadmap for selecting and refining dissimilarity metrics, paving the way for future methodological advances.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.14.670326v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Communication breakdown and evolution of the cancer cell</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nemati Fard, L. A., Arora, C., Miglionico, P., Varisco, M., Bisceglia, L., Vukotic, R., Raimondi, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We studied cell-cell interactions (CCI) in large scale transcriptomic datasets which showed higher co-expression in cancer compared to healthy tissues.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We studied cell-cell interactions (CCIs) in large scale transcriptomic datasets which showed higher co-expression in cancer compared to healthy tissues. CCIs are more co-expressed than any other type of intracellular interactions and, likewise, they are the protein-protein interaction (PPI) class that is more co-evolved in sequenced genomes. Similar trends of stricter regulation and evolutionary pressure are observed when comparing extracellular vs intracellular interactions mediated by G protein Coupled Receptors (GPCRs), whose ligand interactions are also characterized by a higher mutational burden in later tumor stages when considering somatic mutations associated with tumor clonal evolution. CCIs undergo the most extensive rewiring of their tumor co-expression networks relative to healthy tissues, more so than any other PPI type, with a set of CCI hubs highly conserved across multiple tumor tissues, and a higher diversity on healthy ones. Cancer rewiring is also associated with the formation of recurrent circuits of co-expressed CCI pairs, represented by enriched network motifs such as triad or tetrad cliques. These act as integrative hotspots to facilitate the crosstalk of distinct processes and the interaction of the cancer cell with its tumor microenvironment (TME). Remarkably, many CCI circuits are significantly associated with patient survival and are predictive of patient response to immunotherapy. CCI circuits mapping to Allograft rejection and Inflammatory response inform immunotherapy response prediction, while those related to Epithelial Mesenchymal Transition are associated with poorer prognosis. Overall, we show that CCIs expression signatures could be effectively exploited to stratify patients and, at the same time, they highlight new combination therapeutic opportunities in personalized medicine settings.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.15.670478v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Constructing and Using Cell Type Populations of the Human Reference Atlas</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bueckle, A. D., Herr, B. W., Chen, L., Bolin, D., Qaurooni, D., Ginda, M., Jain, Y., Puig-Barbe, A., Ardlie, K. G., Wang, F., Boerner, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> The Human Reference Atlas (HRA) v2.3 provides a quantitative 3D framework of CTs across 73 reference organs and 1,283 3D anatomical structures (ASs).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The human body contains ~27-36 trillion cells of up to 10,000 cell types (CTs) within a volume of ~62-120 liters (males) and 52-89 liters (females). The Human Reference Atlas (HRA) v2.3 provides a quantitative 3D framework of CTs across 73 reference organs and 1,283 3D anatomical structures (ASs). The HRA Cell Type Population (HRApop) effort quantifies CTs per AS using high-quality single-cell (sc) data processed through scalable, reproducible workflows and cell type annotation (CTann) tools. HRApop v1.0 includes reference CT populations for 73 ASs (112 when sex-specific) using 662 datasets spatially registered to 230 locations across 17 organs (31 when sex-specific). For 558 sc-transcriptomics datasets (11,042,750 cells), CTs and biomarker expression were computed using Azimuth, CellTypist, and popV. To test generalizability, 104 sc-proteomics datasets (16,576,863 cells) were integrated. In total, HRApop includes 27,619,613 cells. HRApop can be used to predict (1) CT populations for 3D volumes in the human body and (2) the spatial origin of a tissue block, given a CT population. Data and code are at cns-iu.github.io/hra-cell-type-populations-supporting-information.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.14.670406v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Optimized ChIP-exo for mammalian cells and patterned sequencing flow cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> James, D. Q., Mukherjee, S., Cannon, C. C., Mahony, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a Mammalian-Optimized ChIP-exo (MO-ChIP-Exo) protocol that builds upon previous ChIP immunoprecipitation protocols with systematic optimization of crosslinking, harvesting, and library construction.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> By combining chromatin immunoprecipitation (ChIP) with an exonuclease digestion of protein-bound DNA fragments, ChIP-exo characterizes genome-wide protein-DNA interactions at near base-pair resolution. However, the widespread adoption of ChIP-exo has been hindered by several technical challenges, including lengthy protocols, the need for multiple custom reactions, and incompatibilities with recent Illumina sequencing platforms. To address these barriers, we systematically optimized and adapted the ChIP-exo library construction protocol for the unique requirements of mammalian cells and current sequencing technologies. We introduce a Mammalian-Optimized ChIP-exo (MO-ChIP-exo) protocol that builds upon previous ChIP-exo protocols with systematic optimization of crosslinking, harvesting, and library construction. We validate MO-ChIP-exo by comparing it to previously published ChIP-exo protocols and demonstrate its adaptability to both suspension (K562) and adherent (HepG2, mESC) cell lines. This improved protocol provides a more robust and efficient method for generating high-quality ChIP-exo libraries from mammalian cells.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.14.670389v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Active learning-guided optimization of cell-free biosensors for lead testing in drinking water</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, B. M., Chiang, N., Ekas, H. M., Brown, D. M., Dildine, G., Lucci, T. J., Feng, S., Bly, V., Gaillard, J.-F., Lucks, J. B., Karim, A. S., Shukla, D., Jewett, M. C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We develop a multi-objective, machine learning-guided, cell-free gene expression workflow for engineering aTF-based biosensors and show how aTF mutations alter performance.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Point-of-use diagnostics based on allosteric transcription factors (aTFs) are promising tools for environmental monitoring and human health. However, biosensors relying on natural aTFs rarely exhibit the sensitivity and selectivity needed for real-world applications, and traditional directed evolution struggles to optimize multiple biosensor properties at once. To overcome these challenges, we develop a multi-objective, machine learning (ML)-guided cell-free gene expression workflow for engineering aTF-based biosensors. Our approach rapidly generates high-quality sequence-to-function data, which we transform into an augmented paired dataset to train an ML model using directional labels that capture how aTF mutations alter performance. We apply our workflow to engineer the aTF PbrR as a point-of-use diagnostic for lead contamination in water. We tune the sensitivity of PbrR to sense at the U.S. Environmental Protection Agency (EPA) action level for lead and modify the selectivity away from zinc, a common metal found in water supplies. Finally, we show that the engineered PbrR functions in freeze-dried cell-free reactions, enabling a diagnostic capable of detecting lead in drinking water down to ~5.7 ppb. Our ML-driven, multi-objective framework--powered by directional tokens--can generalize to other biosensors and proteins, accelerating the development of synthetic biology tools for biotechnology applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.20.671382v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Hypobaric hypoxia drives citrate cycle reprogramming to suppress tumor progression</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gao, Y., He, X., E, G., Wang, Y., Huang, S., Cai, D., Deng, Y., Gan, P., Feng, R., Liang, Q., Tang, Z., Xu, G., Chen, J., Xue, H., Wang, Q., Liu, H., Li, G., Zeng, S., Xie, M., Gao, Y., Gong, J., Zhang, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> High altitude residents present lower tumor incidence and mortality than plain residents, suggesting that hypobaric hypoxia displays protective effect against tumor development.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> High altitude residents present lower tumor incidence and mortality than plain residents, suggesting that hypobaric hypoxia displays protective effect against tumor development. The results of our studies showed that hypobaric hypoxia exposure significantly suppressed tumor progression in subcutaneous bearing and lung metastasis tumor models. With single-cell transcriptome analysis, we identified HIF-1 pathway was significantly downregulated in tumor tissues with high altitude hypoxia, leading to decreased glycolysis and angiogenesis and improved antitumor immune response. Mechanistically, systemic hypoxia rewired citric acid cycle, with enhanced CS and IDH2 expression and reduced OGDH and SUCLG2 expression, to induce -ketoglutarate accumulation and succinate decline in tumor microenvironment, further mediating HIF-1 pathway inhibition. Moreover, hypobaric hypoxia treatment significantly improved the antitumor effects of adjuvant therapies (including chemotherapy, anti-angiogenic therapy and immunotherapy). Our findings reveal the role and mechanism of hypobaric hypoxia for tumor regression and yield new insights into tumor therapy.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.15.669572v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Theoretical value of radiation priming for mitigating toxicity in radiation therapy</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shaki, Y. Y., Socol, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> An empirical mathematical model for adaptive biological response to time-dependent irradiation of arbitrary time-shapes .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The relevance of ionizing radiation effects is high in the field of radiation oncology. Namely, radiation toxicity is a crucial limiting factor in dose administration. To address this concern, by adopting an empirical approach it is possible to uncover certain truths that might not be attainable through rigorous biophysical analysis, as the human body represents a complex system. That is, even possessing complete knowledge of interactions between its elementary units (cells) may prove insufficient in explaining the overall system's reactions. An empirical mathematical model for adaptive biological response to time-dependent irradiation of arbitrary time-shapes was developed. The model was based on the damped-oscillator analogy from physics. The research reported here is a generalization of the damped-oscillator model by introducing a biologically relevant inverted Gompertzian-like shape, rather commonly employed in adaptive response modeling, with practical implications for biologists and oncologists. Based on the developed model, a practical approach is suggested to improve radiation therapy protocols by radiation training (priming) through administration of low, gradually increasing total-body irradiation doses. Following this priming, individuals should be capable of withstanding higher therapeutic doses, maybe up to five times the regular amount. Initial estimations propose a training time of approximately six weeks and a training dose of up to approximately 600 cGy. If the model is correct, effectiveness of radiation therapy can be significantly improved by radiation priming, since administration of considerably higher therapeutic doses would be enabled through priming procedures.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.15.670485v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Tumour genotype shapes blood biomarker expression for use in pancreatic cancer detection and diagnosis.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Canel, M., Lonergan, D. W., Ferguson, C., Gautier, P., Morton, j. P., Kriegsheim, A. v., Serrels, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that loss of Cdkn2a and Smad4, a genomic event that occurs following progression from PanIN to PDAC, substantially limits the availability of blood biomarkers and uncover a potential role for Smad 4 loss in limiting opportunities for the early detection of pancreatic cancer.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Typically diagnosed late, when systemic metastasis has already occurred, pancreatic ductal adenocarcinoma (PDAC) has one of the worst 5-year survival rates of any cancer type. For many patients with advanced disease, current chemotherapy regimens offer only modest benefit despite significant toxicity and surgical resection, the only treatment option with curative potential, is not possible. Therefore, while new treatments are much needed, diagnosing patients at an earlier disease stage when surgery remains possible and the window of opportunity for treatment response is longer will be critical to improving patient outcomes. In this regard, the identification of biomarkers from biospecimens that can be easily sampled from patients remains the focus of considerable research, however success has not been forthcoming. Using a suite of novel genetically defined murine isogenic models of early PDAC, engineered using CRISPR-Cas9 gene editing, we sought to address whether loss-of-function mutations in common driver genes, and thus the genetic heterogeneity inherent to the disease, may represent an important confounding factor in the identification of a one-size-fits-all biomarker suitable for early detection. Focusing on the multi-omics analysis of blood, we show that both loss of Cdkn2a and / or Smad4 on the background of a KrasG12D Trp53-/- genotype has profound effects on the profile of differentially expressed RNA species including protein coding RNAs, lncRNAs, snoRNAs, scRNAs, snRNAs and miRNAs, and on plasma protein expression, when compared to both healthy controls and chemically induced pancreatitis. In addition, we find that loss of Smad4, a genomic event that occurs following progression from PanIN to PDAC, substantially limits the availability of blood biomarkers. These findings identify the need to move towards genotype-specific biomarker signatures and uncover a potential role for Smad4 loss in limiting opportunities for the early detection of pancreatic cancer.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.15.670486v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrated spatial proteomics of human PDAC uncovers an expanded tumour-immune-stroma spectrum with genomic associations</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shakfa, N., Nowlan, F., Drissler, S., Tan, T. J., Sunnucks, E., Chen, E. L. Y., Wong, C. J., Seale, B., Lin, Z.-Y., Chan-Seng-Yue, M., Zhang, A., Chaudhary, S., Yu, C., Abazari, G., Geuenich, M. J., Watson, M., Peng, J., Afiuni-Zadeh, S., Borgida, A., Gonzalez, R., Liang, S.-B., Nowak, K., Mrkonjic, M., Dodd, A., Wilson, J. M., Campbell, K. R., Gorman, J. L., Grunwald, B. T., Grant, R. C., Knox, J. J., Notta, F., Gingras, A.-C., Gallinger, S., O'Kane, G. M., Jackson, H. W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the connections between tumour phenotypes, vascularization, immune response, and stromal biophysical state are reinforced by genomic aberrations, altered by treatment, and associated with patient outcome.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Distinctively, pancreatic ductal adenocarcinoma (PDAC) consists of sparse tumour lesions intertwined with extensive desmoplastic stroma. The complexity of tumour-microenvironment interactions within this desmoplasia poses a challenge for accurate tumour profiling and patient stratification, and characterizes a profoundly chemoresistant tumour. Here we mapped the spatial relationships between tumour, stroma, and immune cell compartments delineating tumour and microenvironment types that expand the classical to basal spectrum of human PDAC. We used imaging mass cytometry to profile the in situ multi-cellular organization of 81 cell types in resected cases with paired whole genome sequencing. Cell types, functions, and pathway activation were distributed as highly reproducible environments in discrete locations throughout these tumours, which we deep-profiled using laser-capture mass spectrometry. We show that the connections between tumour phenotypes, vascularization, immune response, and stromal biophysical state are reinforced by genomic aberrations, altered by treatment, and associated with patient outcome. Predictive machine-learning models showed that spatial single cell data outperformed genomic or clinical features but integrated multi-omics models provide the best prediction of patient survival with compressed models requiring only 10 non-redundant robust molecular measures associated with the phenotypic spectrum of PDAC. Together, these findings define a phenotypic and molecular framework of PDAC that captures tumour-microenvironment co-dependencies and offers a refined basis for patient stratification and therapeutic targeting.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.15.670565v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Decoding phospho-regulation and flanking regions in autophagy-associated short linear motifs.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mattia Utichi, Oana N Antonescu, Valentina Sora, Henri-Baptiste Marjault, Matteo Tiberti, Emiliano Maiani, Matteo Lambrughi, Elena Papaleo</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that specific phosphorylation events and flanking residues modulate binding to LC3 at the atomic level, and that disease-associated mutations alter these interactions in the phosphorylated context.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Short Linear Motifs (SLiMs) play a pivotal role in the interactions between intrinsically disordered proteins and their binding partners. SLiMs can undergo regulation through post-translational modifications, including phosphorylation. The flanking regions surrounding the core motifs also exert a crucial role for the interaction. While phosphorylation and flanking regions are known to influence SLiM function, the mechanistic basis of this regulation remains poorly understood. We integrate biomolecular simulations, in silico high-throughput mutational scans, and biophysical experiments to elucidate the phospho-regulation in SLiMs crucial for autophagy, i.e., the LC3 interacting regions (LIRs). We investigate the Optineurin LIR, which perfectly exemplifies a class of LIR with a complex interplay of phosphorylations and flanking regions. Here we show that specific phosphorylation events and flanking residues modulate binding to LC3 at the atomic level, and that disease-associated mutations alter these interactions in the phosphorylated context. Notably, we establish an approach based on Microfluidic Diffusional Sizing to investigate binding affinities of SLiMs to target proteins, complemented by Surface Plasmon Resonance, enabling precise measurements of dissociation constants and kinetics for a selection of variants. Our work provides a versatile toolkit to characterize phospho-regulated SLiMs, advancing the understanding of important cellular processes.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40835742/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multimodal analysis of cell-free DNA enhances differentiation of early-stage breast cancer from benign lesions and healthy individuals.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Thi Tuong Vi Van, Trung Hieu Tran, Thi Hue Hanh Nguyen, Van Thien Chi Nguyen, Dac Ho Vo, Giang Thi Huong Nguyen, Trong Hieu Nguyen, Kim Sang To, Anh Luan Nguyen, Cao Hong An Tran, Thanh Xuan Jasmine, Thi Loan Vo, Thi Huong Thoang Nai, Thuy Trang Tran, My Hoang Truong, Ngan Chau Tran, Thi Loc Le, Thi Hong Nhung Nguyen, Ngoc Hieu Tu, Thanh Son Tran, Bao Toan Le, Van Phong Tang, Pham Thanh Nhan Nguyen, Khac Tien Nguyen, Van Chien Ho, Xuan Vinh Nguyen, Nhu Nhat Tan Doan, Thi Trang Tran, Thi Minh Thu Tran, Vu Uyen Tran, Minh Phong Le, Thi Luyen Vu, Ba Linh Tieu, Huu Tam Phuc Nguyen, Luu Hong Dang Nguyen, Ngoc Minh Phan, Thi Van Phan, Thi Thanh Thuy Do, Thi Huyen Dao, Hung Sang Tang, Duy Sinh Nguyen, Hoa Giang, Minh Duy Phan, Hoai-Nghia Nguyen, Duc Hieu Vo, Le Son Tran</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new liquid biopsy technique for breast cancer diagnosis .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Breast cancer (BC) remains the second leading cause of cancer-related mortality among women worldwide. Liquid biopsy based on circulating tumor DNA (ctDNA) offers a promising noninvasive approach for early detection; however, differentiating malignant tumors from benign abnormalities remains a significant challenge.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40830871/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Why Protein Modifications Matter for Digestibility: The Case of Ara h 1 Peanut Allergen and Trypsin Cleavage.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Teodora Cvetiƒá, Aleksa Saviƒá, Vesna Jovanoviƒá, Ivana Prodiƒá, Jelena Radosavljeviƒá, Albert Sickmann, Katarina Smiljaniƒá</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of the impact of protein modifications on digestibility and allergenicity at trypsin sites .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Trypsin is the principal intestinal endopeptidase and proteomics digestion tool, yet the impact of protein modifications (PMs) on digestibility and allergenicity remains underexplored. We employed a proteomic approach to assess trypsin cleavage efficacy (TCE) at modified versus unmodified K/R residues in Ara h 1, a major peanut allergen. Seven of 17 PM sites showed ‚â•20% difference in TCE, with carbamoylation + methylation and dihydroxylation retaining significance after multiple-testing correction. The 20% threshold aligns with the 19 ¬± 1% baseline of porcine trypsin miscleavages. Molecular docking confirmed reduced binding affinity due to steric hindrance from methylation at R259. These findings suggest that impaired digestion at PM sites may enhance peptide sensitization potential. This study provides a basis for machine learning-driven models using public proteomic data sets to predict the influence of PMs on protease performance.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40743210/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

