<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Monday, 26 May 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ChemBERTaPolyPharm: Modeling polypharmacy side effects with ChemBERTa and PubMed Encoders</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gromova, A. A., Maida, A. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new method for detecting drug-drug interactions in drug interactions .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Polypharmacy side effects occur when drug combinations trigger unexpected interactions, altering therapeutic outcomes. During which the activity of one drug may change favorably or unfavorably if taken with the other drug. Drug interactions are rare and are only observed in clinical studies. Thus, the discovery and detection of polypharmacy side effects remains a challenge. Our approach achieves an impressive average F1 score of 0.93, demonstrating its efficacy in capturing drug interaction patterns.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.20.655109v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">RNA-sequencing Strain-specific Genome Alignment Increases Differential Expression Findings in Comparison of C57BL/6J and DBA/2J Nucleus Accumbens</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gnatowski, E., Zeliff, D., Dozmorov, M. G., Miles, M. F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Using strain-specific genome alignment to improve the detection of differentially expressed genes (DEGs) and differentially utilized exons (DUEs) in gene ontology functional analyses .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Alcohol Use Disorder (AUD) is a polygenic disease defined by the inability to regulate alcohol consumption despite adverse consequences. C57BL/6J (B6) and DBA/2J (D2) mice, the progenitor strains to the BXD recombinant inbred strain, exhibit differences in voluntary ethanol consumption and other ethanol behaviors, making them frequently used models for studying genetic influences on ethanol responses. The B6 genome is the standard reference genome for the majority of mouse RNA-sequencing (RNAseq) studies, including studies on D2 mice. We hypothesized that aligning B6 and D2 RNAseq data to their strain specific genome would allow improved detection of differentially expressed genes (DEGs) in comparison of brain gene expression between these two strains. RNA samples obtained from B6 and D2 nucleus accumbens (NAc) tissue were analyzed using a standard RNAseq analysis pipeline except from genome alignment. Following quality control, samples were aligned to either the B6 reference genome (Release 108) or the D2 samples were aligned to a recent homologous genome assembly (GCA_921998315.2). Alignment of D2 samples to the D2 genome showed significantly higher alignment compared to the B6 reference genome (93.82% vs 92.02%, p = 0.0272), but also showed a decrease in the number of total reads assigned (72.30%% vs 74.36%, p <0.0001). When comparing B6 and D2 expression, using the D2 alignment resulted in large increases in the number of differentially expressed genes (DEGs) (10,777 vs 6,191) and differentially utilized exons (DUEs) (81,206 vs 21,223) with resulting changes in gene ontology functional analyses. The gene ontology identified substantial overlap between the two analyses while also adding novel categories. These studies highlight the importance of using strain-specific alignment in increasing the number of reads aligned and the number of DEGs and DUEs identified. The use of strain-specific alignment in RNA-seq studies may provide greater accuracy in investigating gene expression and pathways regulated by ethanol in model organism studies on molecular mechanisms of AUD.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.25.656014v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genome assembly and annotation of the naked mole rat Heterocephalus glaber reared in Japan</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Toga, K., Oka, K., Tanaka, H., Itoh, T., Toyoda, A., Bono, H., Miura, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We generated a genome assembly of naked mole rat using PacBio long-read sequencing, and identified 417 gene models that were previously undetectable in the reference genome of this species.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The naked mole rat (NMR, Heterocephalus glaber) is a eusocial rodent that is native to northeastern Africa. NMRs exhibit extraordinary traits such as longevity, resistance to age-related decline, and remarkable hypoxia tolerance. Although the reference genome of this species has been determined because of its unique characteristics, the significance or role of intraspecific genomic variations remains unknown. In this study, we used PacBio long-read sequencing to generate a genome assembly of NMR reared in Japan. The assembled genome is 2.56 Gb. Benchmarking Universal Single-Copy Orthologs (BUSCO) revealed high completeness (95.2%). BRAKER3 estimated 26,714 protein-coding genes, and we successfully added functional annotations for 26,232 protein-coding genes using the functional annotation workflow. We identified 417 gene models that were previously undetectable in the reference genome of this species. We also identified structural and amino acid sequence variations between our assembly and the reference genome, suggesting the presence of intraspecific genomic variations. This new genomic resource could help uncover the molecular mechanisms underlying the behavioral and physiological traits of NMR.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.20.654782v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AI-Guided Discovery and Optimization of Antimicrobial Peptides Through Species-Aware Language Model</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bae, D., Kim, M., Seo, J., Nam, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A target species-aware AI model for antimicrobial peptide activity prediction that leverages pre-trained language models to predict minimum inhibitory concentration (MIC) values.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The rise of antibiotic-resistant bacteria drives an urgent need for novel antimicrobial agents. Antimicrobial peptides (AMPs) show promise solutions due to their multiple mechanisms of action and reduced propensity for resistance development. This study introduces LLAMP (Large Language model for AMP activity prediction), a target species-aware AI model that leverages pre-trained language models to predict minimum inhibitory concentration (MIC) values of AMPs. Analysis of attention values allowed us to pinpoint critical amino acid residues (e.g., Trp, Lys, and Phe). Our work demonstrates the potential of AI to expedite the discovery of peptide-based antibiotics to combat antibiotic resistance.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.20.654992v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A statistical model for quantitative analysis of single-molecule footprinting data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ozonov, E. A., Gaspa-Toneu, L., Peters, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a novel statistical framework designed for inference of footprint lengths and predictions of footprint positions for unbiased quantitative analysis and interpretation of single-molecule footprinting datasets.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The binding of sequence-specific TFs (TF) to genomic DNA is fundamental to gene regulation. Emerging single-molecule footprinting (SMF) technologies such as the NOMe-seq and Fiber-seq assays offer unique opportunities for acquiring quantitative information about binding states of TFs and nucleosomes at single-DNA-molecule resolution. Contrasting bulk epigenomic profiling methodologies, SMF enables better molecular characterization of inherently stochastic processes of protein-DNA interactions. Despite the many advantages that SMF technologies bring for studying mechanisms of gene regulation, rigorous statistical models for the analysis of datasets generated using these technologies are still missing. Here, we introduce a novel statistical framework designed for inference of footprint lengths and predictions of footprint positions for unbiased quantitative analysis and interpretation of SMF datasets. We carried out comprehensive computational simulations of SMF experiments and identified experimental parameters that are critical to footprint detection. Finally, we demonstrate the power of this statistical approach for the analysis of genome-wide and amplicon-based NOMe-seq datasets generated for mouse embryonic stem cells.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.20.655044v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Unmasking Human T Cell Receptor Germline Diversity: 335 Novel Alleles Identified in 47 Pangenome Reference Individuals Using the gAIRR Suite</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yang, Y.-H., Yao, C.-Y., Lin, M.-J., Huang, K.-T., Lin, Y.-H., Chiu, I.-H. C., Lai, S.-K., Chen, C.-Y., Yang, Y.-C., Hsu, C.-L., Hsu, J. S., Chen, C.-Y., Chen, P.-L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified 335 novel TR alleles in the adaptive immune receptor repertoire (gAIRR) using the gAIRR Suite and identified 335 new TR genes from 47 high-quality genomes from the Human Pangenome Reference Consortium (HPRC).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The adaptive immune receptor repertoire (AIRR), composed of V(D)J-recombined T cell receptors (TR) and immunoglobulins (IG), is central to adaptive immunity. Accurate AIRR profiling depends on a comprehensive and diverse germline gene set encoding AIRR (gAIRR). Furthermore, gAIRR alleles themselves are associated with immune-related phenotypes and diseases. However, for TR genes, the IMGT database - the primary gAIRR reference - has not been updated since 2020 and lacks sufficient population diversity and complete flanking sequences. To address these limitations, we investigated 47 high-quality genomes from the Human Pangenome Reference Consortium (HPRC) using the gAIRR Suite and identified 335 novel TR alleles - 305 TRV and 30 TRJ - representing 91.6% and 30.9% increases over IMGT records. All novel alleles were crosschecked using two orthogonal pipelines. We also established a comprehensive flanking sequence database, including recombination signal sequences (RSS). We have made all resources publicly available to support immunogenomics research and clinical applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.24.655452v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genome-wide screen uncovers novel host factors for L-A virus maintenance and a mutualistic-symbiosis relationship in yeast</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hsiao, W.-Y., Yeh, C.-S., Liu, H.-I., Tung, L., Chang, T.-H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified 96 host genes required for the maintenance of the L-A double-stranded RNA virus, a persistent and non-lytic resident of most laboratory yeast strains, and discovered a previously unrecognized mutualistic relationship between the virus and yeast.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Viruses are often regarded as obligate intracellular parasites that exploit host resources for their own propagation. However, emerging evidence suggests that virus-host interactions can be more complex than simple antagonism. Here, we performed a genome-wide screen in Saccharomyces cerevisiae to identify host factors required for the maintenance of the L-A double-stranded RNA virus, a persistent and non-lytic resident of most laboratory yeast strains. Using two complementary mutant collections encompassing ~6,000 yeast genes (~93% genome coverage), we identified 96 host genes essential for L-A maintenance, spanning diverse biological functions. Transcriptome profiling revealed that the presence of L-A virus alters the host stress-response gene expression program. Strikingly, competitive fitness assays under environmental stress conditions showed that L-A enhances host stress tolerance, revealing a previously unrecognized mutualistic relationship. Together, our findings redefine the L-A-yeast interaction as a form of stable mutualism and highlight the utility of functional genomics and systems-level approaches in uncovering hidden dimensions of virus-host coevolution.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.20.655038v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Siibra: A software tool suite for realizing a Multilevel Human Brain Atlas from complex data resources</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dickscheid, T., Gui, X., Simsek, A. N., Schiffer, C., Mangin, J.-F., Leprince, Y., Jirsa, V., Bjaalie, J. G., Leergaard, T. B., Bludau, S., Amunts, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present siibra, a tool suite that connects diverse data from cloud resources to reference atlases and coordinate spaces and a Multilevel Human Brain Atlas.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Computational technology opens new possibilities towards understanding the complexity of the human brain, but it requires integrating measurements from different modalities and scales in anatomical context and exposing them in interoperable, actionable form. Especially with growing big data resources, accessing information from different scales and modalities coherently for visual exploration, reproducible analysis and application development remains challenging. We present siibra, a tool suite that connects diverse data from cloud resources to reference atlases and coordinate spaces. It supports different use cases by making contents accessible through a web viewer, Python library and HTTP API. Using siibra we implemented a Multilevel Human Brain Atlas linking macro-anatomical concepts and their inter-subject variability with measurements of the microstructural composition and intrinsic variance of brain regions, building on cytoarchitecture as a reference and supporting MRI-based and microscopic templates. The atlas is integrated with the EBRAINS research infrastructure. All software and content are openly accessible.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.20.655042v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Improved open modification searching via unified spectral search with predicted libraries and enhanced vector representations in ANN-SoLo</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Arab, I., Laukens, K., Bittremieux, W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present an enhanced version of ANN-SoLo that combines strengths of both spectral library searching and sequence database searching, by integrating with Prosit to generate predicted spectral libraries from protein sequence databases.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The primary computational challenge in mass spectrometry-based proteomics is determining the peptide sequence responsible for generating each measured tandem mass spectrum. This task is traditionally addressed through sequence database searching, as well as alternative approaches such as spectral library searching. ANN-SoLo is a powerful spectral library search engine optimized for open modification searching, enabling the detection of peptides carrying any post-translational modification. Here, we present an enhanced version of ANN-SoLo that combines strengths of both spectral library searching and sequence database searching, by integrating with Prosit to generate predicted spectral libraries from protein sequence databases. Additionally, it provides functionality to generate decoys at both the spectrum and the peptide level, introduces an optimized internal file structure for large-scale analytics, and improves search accuracy by incorporating neutral loss information into spectrum vector representations. These advancements collectively address challenges associated with missing spectral libraries and enhance peptide identification in large-scale and complex proteomics workflows.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.20.655174v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Saliva-based Biomarkers for Predicting Gastric Cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Arora, A., Raghava, G. P. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified eight salivary exRNA biomarkers capable of distinguishing GC from normal samples.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Saliva-based biomarkers offer a non-invasive, convenient, and patient-friendly approach for gastric cancer (GC) diagnosis, eliminating the need for uncomfortable procedures. This study aimed to identify salivary extracellular RNA (exRNA) biomarkers capable of distinguishing GC from normal samples. We used GEO dataset containing exRNA expression profiles of 98 GC and 100 normal samples. We filtered the data and applied the Mann-Whitney U test to identify statistically significant salivary exRNAs (p < 0.05). Initially, a single gene-based classification method identified potential biomarkers including GSDMA, SMR3B, NPTXR, TBCD, KCNC4, and KRT26, achieving accuracies ranging from 61.61% to 64.14%. Then, we applied AI-based methods to identify GC biomarkers. We developed machine learning models on different sets of biomarkers extracted from the original set of statistically significant exRNA using feature selection methods. An optimal primary biomarker set of eight mRNAs (GSDMA, CCDC141, TBCD, STARD13, WRB, ARHGAP23, CDHR3, BX842679.1) was identified, yielding an AUC of 0.905 and MCC of 0.770 with an ensemble stacking classifier. We performed comparative analysis with existing GC biomarker studies which demonstrated that our identified primary biomarker set outperformed previously reported biomarker panels. Out of the eight identified genes, seven have been reported in the literature to be associated with Gastric Cancer. Our results demonstrate the potential of salivary exRNA biomarkers as a non-invasive tool for GC diagnosis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.20.655204v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Partitioned Multi-MUM finding for scalable pangenomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shivakumar, V. S., Langmead, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We extend Mumemto, which computes maximal unique matches (multi-MUMs) across pangenomes using compressed indexing, by introducing two new partitioning and merging strategies.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Pangenome collections are growing to hundreds of high-quality genomes. This necessitates scalable methods for constructing pangenome alignments that can incorporate newly-sequenced assemblies. We previously developed Mumemto, which computes maximal unique matches (multi-MUMs) across pangenomes using compressed indexing. In this work, we extend Mumemto by introducing two new partitioning and merging strategies. Both strategies enable highly parallel, memory efficient, and updateable computation of multi-MUMs. One of the strategies, called string-based merging, is also capable of conducting the merges in a way that follows the shape of a phylogenetic tree, naturally yielding the multi-MUM for the tree's internal nodes as well as the root. With these strategies, Mumemto now scales to 474 human haplotypes, the only multi-MUM method able to do so. It also introduces a time-memory tradeoff that allows Mumemto to be tailored to more scenarios, including in resource-limited settings.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.20.654611v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

