<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Friday, 06 June 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">GENERanno: A Genomic Foundation Model for Metagenomic Annotation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Li, Q., Wu, W., Zhu, Y., Feng, F., Ye, J., Wang, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce GENERanno, a compact yet powerful genomic foundation model specifically optimized for metagenomic annotation, which achieves superior accuracy compared to traditional HMM-based methods (e.g., GLIMMER3, GeneMarkS2, Prodigal).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The rapid growth of genomic and metagenomic data has underscored the pressing need for advanced computational tools capable of deciphering complex biological sequences. In this study, we introduce GENERanno, a compact yet powerful genomic foundation model specifically optimized for metagenomic annotation. Trained on an extensive dataset comprising 715 billion base pairs (bp) of prokaryotic DNA, GENERanno employs a transformer encoder architecture with 500 million parameters, enabling bidirectional attention over sequences up to 8192 nucleotides at single-nucleotide resolution. This design addresses key limitations of existing methods, including the inability of traditional Hidden Markov Models (HMMs) to handle fragmented DNA sequences, as well as the suboptimal tokenization schemes of current genomic foundation models that compromise fine-grained analysis. To evaluate the model performance, we curated the Prokaryotic Gener Tasks, a biologically meaningful benchmark encompassing gene fitness prediction, antibiotic resistance prediction, gene classification, and taxonomic classification. Across these tasks, GENERanno consistently outperforms its counterparts, establishing itself as a leading genomic foundation model in the prokaryotic domain. For metagenomic annotation, GENERanno achieves superior accuracy compared to traditional HMM-based methods (e.g., GLIMMER3, GeneMarkS2, Prodigal) and recent LLM-based approaches (e.g., GeneLM), while demonstrating exceptional generalization ability on archaeal genomes. Notably, GENERanno pioneers the prediction of pseudogenes based solely on sequence data, leveraging its contextual understanding to differentiate non-functional sequences from active coding regions. Overall, GENERanno represents a significant advancement in genomic foundation modeling, bridging the gap between large-scale sequence analysis and fine-grained biological insights. By providing a versatile tool for metagenomic annotation and broader genomic exploration, this work lays the groundwork for future research in functional genomics and related fields. Implementation details and supplementary resources are available at https://github.com/GenerTeam/GENERanno.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.04.656517v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Prevalence of dual-donating amines in key regions of functional RNAs</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Veenis, A. J., Saon, M. S., Bevilacqua, P. C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Dual-donating amines within structured RNAs may enhance interactions by reducing conformational entropy loss as well as strengthening nearby H-bonds.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> RNA performs many critical functions nearly all of which are enabled by complex hydrogen bonded structures. Nucleotides possess far fewer hydrogen bond donors than acceptors, and only the exocyclic amine can donate two H-bonds, suggesting a specialized role. To assess the prevalence and structural contexts of dual-donating amines within structured RNAs, we created a computational workflow that mines and analyzes experimental RNA-containing structures. We evaluated H-bonding in over 250,000 amines from more than 1,800 structures. Dual-donating amines were found most frequently in G's where they regularly interacted with diverse pairs of acceptors. In contrast, the dual-donating amines of A's and C's were less frequent and they interacted with a more select set of acceptors. For all three nucleobases, amines that were dual-donating had both reduced solvent accessibility and higher atom density relative to amines that were non-donating, indicating a tendency of dual donors to be more buried and help compact the RNA. Moreover, analysis of RNA pseudo-torsion angles revealed that dual-donating amines are enriched in two non A-form conformations, both of which are present in S-motifs found in the sarcin-ricin loop of rRNA. We find that dual-donating amines populate additional structural motifs including the GNRA tetraloop receptor, the kink-turn, and the WC/H A-minor motif, which are present in the self-splicing group I intron, the SAM riboswitch, and the poly(A)-bound ENE. We suggest that dual-donating amines may enhance interactions by reducing conformational entropy loss as well as strengthening nearby H-bonds.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.05.658080v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The Immunophenotype and Proviral Landscape of HIV-infected CD4 T Cells During Antiretroviral Therapy</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Delley, C. L., Shah, S., Joslin, K. M., Park, Y. P., Busch, M. P., Stone, M., Demaree, B., Deeks, S. G., Boritz, E. A., Abate, A. R., Clark, I. C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We applied DAb-seq, a high-throughput method that combines single-cell DNA and surface protein sequencing, to map the proviral landscape with diverse cellular phenotypes, revealing heterogeneity in surface protein expression and provirus maintenance.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In individuals on effective antiretroviral therapy, integrated HIV proviruses persist within CD4 T cells, forming a viral reservoir that rebounds if treatment is stopped. Identifying and targeting these rare, infected cells is critical for advancing therapies, but methods to study reservoir cells are limited and their unique properties remain largely unknown. We applied DAb-seq, a high-throughput method that combines single-cell DNA and surface protein sequencing, to profile over five hundred and twenty thousand CD4 T cells from the blood of six individuals on ART. Infected cells were unequally distributed in T cell subsets, and differential protein expression between infected and uninfected cells revealed significant heterogeneity across cell subsets. Attempts to identify surface markers that differentiate infected from uninfected cells found antigens that mirrored the enrichment of HIV in central memory subsets. However, while central memory T cells harbored the majority of HIV, cells with intact provirus were enriched relative to their defective counterparts in Naive and Regulatory T cell subsets, suggesting that they differentially maintain intact proviruses. In summary, we developed DAb-seq as a powerful open-source tool to map the proviral landscape with diverse cellular phenotypes, revealing heterogeneity in surface protein expression and provirus maintenance across subsets.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.05.657210v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Making AI accessible for forensic DNA profile analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> de Wit, A. K. J. G., Wagenaar, C. D., Janssen, N. A., Hoegen, B., van de Wetering, J., Hoofs, H., ArieÃàns, S., Benschop, C. C., Ypma, R. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We train a well-performing model using data gathered as part of casework, and employing a widely adopted architecture: the U-Net, and compare its performance with human annotations following standard procedures.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Deep learning has the potential to be a powerful tool for automating allele calling in forensic DNA analysis. Studies to date have relied on bespoke model architecture and painstaking manual annotations to train models, which makes it challenging for other researchers to work with these techniques. In this study, we explore the possibility of training a well-performing model using data gathered as part of casework, and employing a widely adopted architecture: the U-Net. In this approach, annotations are created from alleles called during casework. The model, dubbed 'DNANet', then classifies each scan point in the electropherogram (EPG) as part of an allele or non-allele, building on the task of segmentation in computer vision. We evaluate performance on unseen case data and on independent mixture research data, taking analyst annotations as ground-truth. We further compare DNANet's performance with analyst performance on the research data, taking actual donor alleles as ground-truth. DNANet reached an F1 score of 0.971 on analyst annotated alleles on case data not seen during training, and 0.982 on the research data. On actual donor alleles, DNANet reached an F1 score of 0.962, equal to the F1 score computed from analyst annotations. Our results show that DNANet's performance is comparable to human annotations following standard procedures. This illustrates the potential for obtaining good results with standard data and architecture. Future work may focus on what aspects of data, annotations or model architecture are key in shaping performance. We make our code, model weights and research data publicly available to aid the community. Lastly, we call for an effort to establish a standardized benchmark to aid in quantitative comparisons between allele calling systems.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.02.656996v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Two de novo transcriptome assemblies and functional annotations from juvenile cuttlefish (Sepia officinalis) under various metal and pCO2 exposure conditions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sol Dourdin, T., Minet, A., Pante, E., Lacoue-Labarthe, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present two annotated de novo transcriptome assemblies from recently hatched (whole body) and one-month-old (head) Sepia officinalis juveniles.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The cuttlefish Sepia officinalis is a precious model in behavioural and neurobiology studies. It is currently facing combined environmental changes related to the anthropogenic global change. However, genomic resources available to support investigations tackling this issue are still scarce. Therefore, we present two annotated de novo transcriptome assemblies from recently hatched (whole body) and one-month-old (head) Sepia officinalis juveniles. Both assemblies rely on an important read depth validated by a pseudo-rarefaction analysis, and gathered several individuals from various metal and pCO2 exposure conditions. After redundancy reduction, assemblies from newly hatched and one-month-old individuals comprised 244,772 and 397,569 transcripts with 39,804 and 49,402 putative ORFs, respectively, and an average annotation rate of 60%. Assemblies were compared to each other, revealing age-specific transcriptomic landscapes. These two assemblies constitute highly valuable genomic resources facilitating the investigation of transcriptomic endpoints in environmental studies considering coleoid cephalopods.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.03.657592v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A spatial transcriptomic atlas of acute neonatal lung injury across development and disease severity</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mallapragada, S., Lyu, R., Williams-Katek, A. L., Fischer, B. K., Vannan, A., Hadad, N., Mee, E. D., Shirazi, S. P., Jetter, C. S., Negretti, N. M., Hilgendorff, A., Eldredge, L. C., Deutsch, G. H., McCarthy, D. J., Kropski, J. A., Sucre, J. M. S., Banovich, N. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A molecular understanding of lung organogenesis requires delineation of the timing and regulation of the cellular transitions that ultimately form and support a surface capable of gas exchange, while the advent of single-cell transcriptomics has allowed for the discovery and identification of transcriptionally distinct cell populations present during lung development, the</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> A molecular understanding of lung organogenesis requires delineation of the timing and regulation of the cellular transitions that ultimately form and support a surface capable of gas exchange. While the advent of single-cell transcriptomics has allowed for the discovery and identification of transcriptionally distinct cell populations present during lung development, the spatiotemporal dynamics of these transcriptional shifts remain undefined. With imaging-based spatial transcriptomics, we analyzed the gene expression patterns in 17 human infant lungs at varying stages of development and lung injury, creating a spatial transcriptomic atlas of ~1.2 million cells. We applied computational clustering approaches to identify shared molecular similarities among this cohort, generating hypotheses about how tissue architecture and molecular spatial relationships are coordinated during normal development and disrupted in disease. Recognizing that all preterm birth represents an injury to the developing lung, we have moved away from the conventional paradigm of classifying an infant lung as belonging to 'disease' or 'control' groups in favor of a linear regression approach that accounts for the routinely collected objective measures of gestational age and life span. Within this new framework, we have identified cell type patterns across these variables that would likely be overlooked when using a binary, conventional 'diseased' vs. 'control' comparison. Together, these data represent an open resource for the lung research community, supporting discovery-based inquiry and identification of targetable molecular mechanisms in both normal and arrested human lung development.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.02.656433v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Learning Genetic Perturbation Effects with Variational Causal Inference</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liu, E., Zhang, J., Uhler, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a hybrid approach that combines a mechanistic causal model with variational deep learning, termed Single Cell Causal Variational Autoencoder (SCCVAE), for extrapolating to predict unseen perturbational responses.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Advances in sequencing technologies have enhanced the understanding of gene regulation in cells. In particular, Perturb-seq has enabled high-resolution profiling of the transcriptomic response to genetic perturbations at the single-cell level. This understanding has implications in functional genomics and potentially for identifying therapeutic targets. Various computational models have been developed to predict perturbational effects. While deep learning models excel at interpolating observed perturbational data, they tend to overfit and may not generalize well to unseen perturbations. In contrast, mechanistic models, such as linear causal models based on gene regulatory networks, hold greater potential for extrapolation, as they encapsulate regulatory information that can predict responses to unseen perturbations. However, their application has been limited to small studies due to overly simplistic assumptions, making them less effective in handling noisy, large-scale single-cell data. We propose a hybrid approach that combines a mechanistic causal model with variational deep learning, termed Single Cell Causal Variational Autoencoder (SCCVAE). The mechanistic model employs a learned regulatory network to represent perturbational changes as shift interventions that propagate through the learned network. SCCVAE integrates this mechanistic causal model into a variational autoencoder, generating rich, comprehensive transcriptomic responses. Our results indicate that SCCVAE exhibits superior performance over current state-of-the-art baselines for extrapolating to predict unseen perturbational responses. Additionally, for the observed perturbations, the latent space learned by SCCVAE allows for the identification of functional perturbation modules and simulation of single-gene knockdown experiments of varying penetrance, presenting a robust tool for interpreting and interpolating perturbational responses at the single-cell level.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.05.657988v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrating Multimodal Data for a Comprehensive Knowledge Graph to Advance Infectious Disease Research</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fan, H., Guo, L., Li, F., Yuan, Z., Deng, Y., Xiao, Y., Li, H., Li, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Infectious diseases remain a formidable threat to global public health, with their escalating morbidity and mortality rates compounded by recurrent epidemics and the alarming rise of antimicrobial resistance .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Infectious diseases remain a formidable threat to global public health, with their escalating morbidity and mortality rates compounded by recurrent epidemics and the alarming rise of antimicrobial resistance (AMR). These challenges have intensified the urgent demand for innovative therapeutic strategies that can accelerate drug development cycles and overcome traditional research bottlenecks. To address these critical needs, we present IDKG (Infectious Disease Knowledge Graph), a specialized large-scale biomedical knowledge network designed to bridge data fragmentation through multimodal data integration. The IDKG constructs comprehensive associations from 345 infectious diseases and 708 pathogens across heterogeneous biomedical sources systematically. The graph architecture comprises nearly 50,000 nodes (8 types, including Pathogen, Protein, etc.) and over 1.2 million edges (11 types, including treats, contains, etc.), establishing an interconnected framework that enables systematic interrogation of cross-disciplinary knowledge. The integrative approach effectively dismantles conventional data silos while preserving biological contextuality. We validated the IDKG's potential by applying graph neural network-based approaches for drug repurposing prediction in human metapneumovirus (hMPV) infection, a common acute respiratory infection for which effective specific antiviral drugs are currently absent. The successfully identification of established antiviral agents, such as ribavirin and emetine, by our M1 model demonstrated its predictive accuracy and biological relevance. IDKG unifies multimodal biomedical data into a network to accelerate drug discovery and bolster outbreak response. This establishes a data-driven, knowledge-based paradigm for infectious disease research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.02.657361v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Structural and temporal dynamics analysis on PANoptosis in sepsis: a bibliometric analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Li, Z., Nie, D., Yin, L., Qin, Q., Yang, C., Li, R., Gao, X., Yu, X., Wang, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comprehensive bibliometric analysis of PANoptosis in sepsis related research in the past 20 years .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> PANoptosis, as a new type of programmed cell death, is characterized by pyroptosis, apoptosis and necroptosis, and is a key mechanism causing a variety of inflammatory diseases. Despite the growing number of studies indicating the crucial role of PANoptosis in sepsis, there has been no bibliometric analysis of the research hotspots and trends in this field. Therefore, this study aims to explore the history, research hotspots and emerging trends of PANoptosis in sepsis related research in the past 20 years from the perspective of structure and temporal dynamics. The articles related to PANoptosis in sepsis were retrieved from the Web of Science Core Collection (WoSCC) database from 2000 to 2024. CiteSpace and HistCite were used to analyse the historical features, the evolution of active topics, and emerging trends about PANoptosis in sepsis. 6165 original articles and reviews on PANoptosis in sepsis were included in the bibliometric analysis. In the last 20 years, the number of published documents is increasing year by year and reaches a peak in 2022. At the same time, many activation themes have emerged at different times, as evidenced by a total of 96 categories, 865 keywords and 629 reference bursts. Keyword clustering anchored eight emerging research subfields, namely 0# oxidative stress, 1#pyroptosis, 2#sepsis-associated encephalopathy, 3#acute kidney injury, 4#immunosuppression, 5#necroptosis, 7#lung injury and 8#extracellular vesicles. The keyword alluvial map shows that the most persistent research concepts in this field are phosphatase. And the emerging keywords are toll_like_receptors, regulatory T cells, recognition, etc. In a timeline visualization based on the time span of the citations, we find five relevant emerging topics, namely 1# immunosuppression, 2# sepsis-induced cardiomyopathy, 3# pyroptosis, 4# acute kidney injury, and 13# COVID-19. Our study provides a comprehensive bibliometric analysis and summary about the current status and trends on PANoptosis in sepsis, which will aid researchers in conducting further scientific research in this field.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.02.657552v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">10 Years of Variational Autoencoder: Insights from Cancer Temporal Progression Studies, a Systematic Literature Review</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Prol-Castelo, G., Cirillo, D., Valencia, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose that applying the Variational Autoencoder (VAE) as a generative model to study cancer in time, for example, focusing on cancer staging, will lead to meaningful advancements in our understanding of cancer.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Deep Learning methods such as Deep Representation Learning (DRL) and, specifically, the Variational Autoencoder (VAE), have been widely used to handle the high dimensionality of available datasets. Hence, these methods have been applied to study cancer through omics data. Cancer is one of the leading causes of death worldwide, and its complex and dynamic nature makes it especially difficult to study. However, the temporal dimension of cancer progression is a target that has not been tackled. In this systematic literature review, we explore the use of DRL, particularly the VAE, in cancer studies with omics data in a temporal context. Our research reveals that the most common uses of the VAE in cancer studies are related to subtyping, diagnosis, and prognosis. Meanwhile, cancer's temporal aspect is often overlooked, namely, because of the lack of longitudinal data. We propose that applying the VAE as a generative model to study cancer in time, for example, focusing on cancer staging, will lead to meaningful advancements in our understanding of cancer.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.29.656750v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Wheat - black-grass competition is a non-zero-sum game influenced by root growth</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Clark, J., Tatnell, L., Bennett, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that black-grass grows significantly faster under winter conditions than wheat, and has much more vigorous root growth, collectively suggesting that prolonged root growth over winter is key to its competitive advantage over wheat.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Interactions with neighbouring plants have profound effects on plant growth, especially in crop-weed interactions, which can cause major crop-losses in agricultural systems. Black-grass (Alopecurus myosuroides) is the most problematic weed for UK agriculture, causing dramatic yield losses in winter wheat, but the basis of its competitive advantage is not clear. We aimed to fundamentally reappraise the nature of wheat - black-grass competition. Here, we show that black-grass is slow to establish, and requires long periods to demonstrate any competitive advantage over wheat. We show that black-grass grows significantly faster under winter conditions than wheat, and has much more vigorous root growth, collectively suggesting that prolonged root growth over winter is key to black-grass's competitive advantage over wheat. We identify significant variation in crop varietal competitiveness against black-grass, in both black-grass suppression and tolerance. We further identified that this competition is not a zero-sum game, with loss in crop biomass not necessarily equal to gain in black-grass biomass. Finally, we found that the black-grass suppression, but not tolerance, is correlated with crop root growth, supporting our hypothesis for black-grass competition. Overall, our results suggest that breeding for increased root growth and/or winter growth rate can create more competitive wheat cultivars.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.02.657396v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Gamma radiation-induced molecular toxicity and effects on pluripotent stem cells of the radiosensitive conifer Norway spruce (Picea abies)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bhattacharjee, P., Gillard, G. B., Hvidsten, T. R., Sandve, S. R., Lind, O. C., Brede, D. A., Olsen, J. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> In vitro-grown pluripotent stem cells of Norway spruce have been used as a model for molecular toxicity of gamma radiation in conifers.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Conifers are among the most radiosensitive plant species. Elevated, sublethal levels of ionising radiation result in reduced apical dominance in conifers, indicating a negative effect on shoot apical meristems (SAMs). The SAMs, harbouring the pluripotent stem cells, generate all the cells of the shoot, enabling growth and reproduction. However, knowledge on the effects of ionising radiation on such stem cells is scarce, but important for risk assessment and radioprotection of plants in contaminated ecosystems. Here, we assessed the sensitivity of in vitro-grown stem cells of Norway spruce to 144-h of gamma irradiation at 1-100 mGy h-1, using such cells as a model for molecular toxicity of gamma radiation in conifers. Although there were no visible effects of the gamma irradiation on cell proliferation and subsequent embryo formation, dose rate-dependent DNA damage was observed at [‚â•]10 mGy h-1, and comprehensive organelle damage at all dose rates. Massive dose rate-dependent transcriptome changes occurred, with downregulation of a range of genes related to cell division, DNA repair and protein folding but upregulation of stress-related hormonal pathways and several antioxidant-related genes. The upregulation of such genes, survival and continued proliferation of at least a subset of cells and the post-irradiation normalisation of expression of DNA repair and protein-folding genes together with somatic embryo formation suggest that stem cells are able to recover from gamma-irradiation-induced stress. Collectively, regardless of cellular abnormalities after gamma irradiation, and huge transcriptomic shifts towards stress management pathways, the pluripotent stem cell cultures were able to retain their stemness.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.02.653021v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Cytokinin overcomes spikelet-driven inhibition of tillering in wheat and barley by delaying meristem development</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wakeman, A., Bennett, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that high spikelet lines with spikes containing enhanced numbers of spikelets have reduced tillering and retarded development of later-initiated spikes, thus representing a form of correlative inhibition in grass reproductive architecture.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Correlative inhibition mechanisms allow plants to regulate the number of organs they produce, but act as a developmental limitation on crop yields. Previously, trade-offs have been observed in wheat and barley between the size of individual compound inflorescences (spikes) and the total number of spikes produced by each plant. We aimed to test the hypothesis that these trade-offs represent an uncharacterized form of correlative inhibition in grass reproductive architecture. We show that 'high spikelet' lines with spikes containing enhanced numbers of spikelets have reduced tillering and retarded development of later-initiated spikes. Moreover, we show that this effect is driven by the presence of developing, but not mature, spikelets in early-initiated spikes, thus representing a form of correlative inhibition. We show that cytokinin treatment delays the development of spike meristems and formation of spikelets, and as such, can promote the development of additional spikes in high spikelet lines, by providing a longer window for tillering to occur. Our results thus identify cytokinin as a potential target for breeding crops with enhanced yield. Furthermore, our results provide a developmental basis for previously observed yield trade-offs in cereal crops, and illustrate the importance of understanding developmental timing when attempting to breed higher-yielding crops.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.02.657399v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A continuous landscape of signaling encodes a corresponding landscape of CAR T cell phenotype</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cho, W., Liu, J. Y., Beckett, A. N., Lunger, J. C., Xu, P., Ho, K., Chen, E. E., Salcido-Alcantar, A., Sant'Anna, L. E., Obbad, K., Po, N., Sotillo, E., Mackall, C. L., Daniels, K. G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We map the structure of Janus kinase (JAK)/STAT signaling domains to cell signaling and resulting CAR T cell function and show a continuous spectrum of cell phenotype.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cytokine signaling is critical to the function of natural immune cells and engineered immune cell therapies such as chimeric antigen receptor (CAR) T cells. It remains unclear how the limited set of signal transducers and activators of transcription (STATs) and other proteins activated by these cytokine receptors can encode the observed diversity of immune cell phenotypes. To understand how signaling downstream of cytokines control immune cell phenotype, we sought to map the structure of Janus kinase (JAK)/STAT signaling domains to cell signaling and resulting CAR T cell function. We recombined 14 signaling motifs to construct a library of ~30,000 constitutively active synthetic cytokine receptors (SCRs) with intracellular domains composed of novel signaling motif combinations that activate different signaling cascades. We experimentally tested ~450 SCRs which generated a range of CAR T cell memory, cytotoxicity, and proliferation. SCRs with pSTAT1 and 3 signaling generated effector memory CAR T cells, while SCRs with strong pSTAT5 generated effector CAR T cells with potent anti-tumor activity. To map the structure-signaling-phenotype landscape we trained models to predict signaling and CAR T cell phenotype that result from varied motif combinations. From neural network predictions we identified features, including strong STAT5 and Shc signaling, that promote unsafe autonomous CAR T cell proliferation. Models also revealed a trade-off between memory and cytotoxicity, with a Pareto front encoded by a continuous change in signaling. These results demonstrate that recombination of a limited set of signaling motifs creates a continuous spectrum of signaling that encodes a corresponding spectrum of cell phenotype. This work synthetically expands the combinatorial space of JAK/STAT signaling and provides a foundation for rational design of CAR T cells with improved cytotoxicity, memory, and safety profiles.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.05.658149v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An experimentally verified mechanistic model for predicting quorum sensing-based switches</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> De Baets, J., De Paepe, B., De Mey, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a mathematical model to predict how varying expression levels of the transcription factor and synthase affect the response of the EsaI/EsaR quorum sensing system.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Quorum sensing-based genetic circuits are gaining traction in synthetic biology as they link population-level behaviour to individual cell responses. However, tuning these circuits remains challenging due to complex dynamics, particularly during the 'Learn' phase of the Design-Build-Test-Learn (DBTL) cycle. To accelerate this process, we developed a mathematical model to predict how varying expression levels of the transcription factor and synthase affect the response of the EsaI/EsaR quorum sensing system. A strain library was constructed, and experimental data were used to optimize the model. The final model could successfully differentiate between the effects of these expression levels on the response of the bidirectional promoter. It allowed visualization of all potential system outcomes and emphasized the transcription factor's critical role in tuning the circuit. This model offers a valuable tool for fine-tuning EsaI/EsaR-based systems for synthetic biology applications. Moreover, given the homology within the LuxR-family quorum sensing systems, this modelling approach may serve as a foundation for model-based tuning of other quorum sensing systems.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.05.658036v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Bit-Reproducible Phylogenetic Tree Inference under Varying Core-Counts via Reproducible Parallel Reduction Operators</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Stelz, C., Huebner, L., Stamatakis, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the first bit-reproducible parallel phylogenetic inference tool, as well as introduce a general algorithm and open-source code for conducting reproducible associative parallel reduction operations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: Phylogenetic trees describe the evolutionary history among biological species based on their genomic data. Maximum Likelihood (ML) based phylogenetic inference tools search for the tree and evolutionary model that best explain the observed genomic data. Given the independence of likelihood score calculations between different genomic sites, parallel computation is commonly deployed. This is followed by a parallel summation over the per-site scores to obtain the overall likelihood score of the tree. However, basic arithmetic operations on IEEE 754 floating-point numbers, such as addition and multiplication, inherently introduce rounding errors. Consequently, the order by which floating-point operations are executed affects the exact resulting likelihood value since these operations are not associative. Moreover, parallel reduction algorithms in numerical codes re-associate operations as a function of the core count and cluster network topology, inducing different round-off errors. These low-level deviations can cause heuristic searches to diverge and induce high-level result discrepancies (e.g., yield topologically distinct phylogenies). This effect has also been observed in multiple scientific fields, beyond phylogenetics. Results: We observe that varying the degree of parallelism results in diverging phylogenetic tree searches (high level results) for over 31 % out of 10 130 empirical datasets. More importantly, 8 % of these diverging datasets yield trees that are statistically significantly worse than the best known ML tree for the dataset (AU-test, p < 0.05). To alleviate this, we develop a variant of the widely used phylogenetic inference tool RAxML-NG, which does yield bit-reproducible results under varying core-counts, with a slowdown of only 0 to 12.7 % (median 0.8 %) on up to 768 cores. We further introduce the ReproRed reduction algorithm, which yields bit-identical results under varying core-counts, by maintaining a fixed operation order that is independent of the communication pattern. ReproRed is thus applicable to all associative reduction operations - in contrast to competitors, which are confined to summation. Our ReproRed reduction algorithm only exchanges the theoretical minimum number of messages, overlaps communication with computation, and utilizes fast base-cases for local reductions. ReproRed is able to all-reduce (via a subsequent broadcast) 4.1 {middle dot} 106 operands across 48 to 768 cores in 19.7 to 48.61 {micro}s, thereby exhibiting a slowdown of 13 to 93 % over a non-reproducible all-reduce algorithm. ReproRed outperforms the state-of-the-art reproducible all-reduction algorithm ReproBLAS (offers summation only) beyond 10 000 elements per core. In summary, we re-assess non-reproducibility in parallel phylogenetic inference, present the first bit-reproducible parallel phylogenetic inference tool, as well as introduce a general algorithm and open-source code for conducting reproducible associative parallel reduction operations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.02.656320v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

