<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Wednesday, 15 October 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">map3C: a computational tool for processing multiomic single-cell Hi-C data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Galasso, J., Wang, Y., Alber, F., Ernst, J., Luo, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present map3C, a software tool for processing multiomic single-cell Hi-C data for analysis and its utility for identifying structural variant locations in the genome.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The emergence of multiomic single-cell Hi-C methods, which simultaneously profile chromatin conformation and other modalities such as gene expression or DNA methylation, creates tremendous opportunities for studying the genome's structure-function relationships. Existing tools for processing multiomic single-cell Hi-C datasets have certain limitations for downstream bioinformatics analysis. We present map3C, a software tool designed to address these limitations. We demonstrate that map3C improves the quality of multiomic single-cell Hi-C data for analysis and its utility for identifying structural variant locations in the genome.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.10.681728v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comprehensive benchmarking of somatic single-nucleotide variant and indel detection at ultra-low allele fractions using short- and long-read data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ha, Y.-J. J., Maziec, D., Markowski, J., Georges, S. J., Parmalee, N. L., Berselli, M., Coorens, T. H., Dong, S., Gardiner, S., Kalra, D., Li, D., Miao, B., Musunuri, R., Xue, L., Yu, Z., Walker, K., Anderson, L., Au, N. Y., Cibulskis, C., Doddapaneni, H., Grochowski, C. M., Jensen, D. M., Lindsay, T., Loy, K., Narayan, A., Narzisi, G., Ou, J., Pham, M. M., Runnels, A. M., Stergachis, A. B., Sutherlin, L. M., Wang, T., Jin, H., Feng, W. C., Zhang, Y., Veit, A. D., Kim, C. T., Chun, H.-J. E., Ardlie, K., Fulton, R. S., Germer, S., Gibbs, R. A., Marth, G. T., Bennett, J. T., Park, P. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comprehensive framework for reliable detection of low-VAF mutations in non-cancer tissues and a valuable resource for the community.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mosaic mutations in normal tissues occur at low variant allele fractions (VAFs), complicating detection. To benchmark strategies, the SMaHT Network created a cell-line mixture (1:49) and produced ultra-deep whole-genome sequencing using short and long reads (five centers, 180-500x each). We assembled a reference of 44,008 mosaic SNVs and 2,059 Indels, cross-validation between platforms to expose limits of short-read analysis. We also partitioned the genome by mappability to examine the impact of genomic context, added a negative reference set, and accounted for culture-derived mutations. When seven institutions applied eleven algorithms to mixture data, call sets were largely discordant across tools and replicates, partly reflecting stochastic presence of low-VAF mutations in biological replicants. For >2% VAF SNVs, sensitivity and precision approached ~80% at > 300x, with little gain from additional sequencing. This work provides a comprehensive framework for reliable detection of low-VAF mutations in non-cancer tissues and a valuable resource for the community.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.13.681545v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Mapping the diverse topologies of protein-protein interaction fitness landscapes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dickinson, B. C., Lu, S. S., Styles, M. J., Gao, C. F., Nandy, A., Basile, C., Pixley, J. A., Tao, S., Dinner, A. R., Vaikuntanathan, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We use our PANCS-Binder technology to perform more than 1,300 independent selections of a randomized small protein to identify binders to a panel of 96 distinct target proteins.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> De novo binder discovery is unpredictable and inefficient due to a lack of quantitative understanding of protein-protein interaction (PPI) sequence-function landscapes. Here, we use our PANCS-Binder technology to perform >1,300 independent selections of various library sizes and compositions of a randomized small protein to identify binders to a panel of 96 distinct target proteins. For successful selections, we discovered reproducible fitness landscapes that group into a few, target-specific, clusters. Each cluster defines a minimal binding motif whose frequency is inversely proportional to the number of specified amino acids (~2-8) and determines selection success, which is quantifiable by the density of binders to the target within a theoretical sequence space. We leverage these data to develop a supervised contrastive learning approach that discriminates binders from non-binders and demonstrates generalization beyond a threshold amount of data. Together, this framework renders PPI landscapes measurable and predictive, accelerating de novo binder discovery and optimization.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.14.682342v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">deCYPher: Star Allele-Resolution Computational Framework of Pharmacogenes for Haplotype-Resolved Long-Read Assemblies</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chang, T.-Y., Liu, Y.-S., Lai, H.-S., Hung, T.-K., Lin, H.-F., Lin, Y.-H., Hsu, C.-L., Yang, Y.-C., Chen, C.-Y., Chen, P.-L., Hsu, J. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed deCYPher, a tool that generates personalized pharmacogenomic reports from haplotype-resolved assemblies.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Although existing next-generation sequencing (NGS) tools, such as Aldy and Cyrius, have been applied for allele typing, they cannot achieve complete accuracy due to various genomic challenges including pseudogenes, structural variations, hybrid genes, copy number variations, and gene deletions. These complexities make accurate pharmacogene interpretation more challenging, despite the crucial role pharmacogenomics plays in precision medicine. We developed deCYPher, a tool that generates personalized pharmacogenomic reports from haplotype-resolved assemblies. The tool enables analysis of all PharmVar 1A level genes, such as CYP2B6, CYP2C9, CYP2C19, CYP2D6, CYP3A5, CYP4F2, DPYD, NUDT15, and SLCO1B1. Applied to all HPRC haplotypes (including both release 1 and release 2 data), deCYPher demonstrated high accuracy in resolving complex gene structures. In the case of CYP2D6, release 1 identified 6% gene multiplications, 6% full gene deletions, and 4% CYP2D6/CYP2D7 hybrids. By contrast, release 2 demonstrated an increased prevalence of multiplications (14%) and hybrids (11%), while the frequency of full gene deletions remained comparable at 5%. Comparison with pb-StarPhase revealed discrepancies in 12 of 94 assemblies in the release 1 dataset. For instance, in sample HG02257, Aldy, Cyrius, and deCYPher consistently identified the genotype as *2/*35, whereas pb-StarPhase reported *2/*2. Notably, the *35-defining variants were present in the BAM and VCF files in the pb-StarPhase pipeline, but the local read depth over the *35-specific region was only 5x in HG02257-p, suggesting that the misclassification likely resulted from insufficient coverage - a known limitation of pb-StarPhase under low-depth conditions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.13.681303v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PepGlider: Property Regularized VAE for interpretable and controllable peptide design</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gwiazda, A., Szymczak, P., Szczurek, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present PepGlider, a continuous property regularization framework that enables direct control over peptide properties that often exhibit complex correlations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Computational peptide design requires precise control over peptide properties that often exhibit complex correlations. Existing generative models for peptide design rely on simplistic discrete conditioning mechanisms rather than precise targeting of specific property values. We present PepGlider, a continuous property regularization framework that enables direct control over their specific values. The method achieves structured latent space with superior disentanglement quality and displays smooth property gradients along regularized dimension. In silico experimental results demonstrate that PepGlider enables independent control of naturally correlated properties, and supports both de novo generation and targeted optimization of existing peptides. PepGlider applied to antimicrobial peptide design allows generation of candidates with desired antibacterial activity profile and maintained low toxicity profile. Unlike existing approaches, PepGlider provides precise control over continuous property distributions while maintaining generation quality, thus offering a generalizable solution for therapeutic and materials applications requiring exact property specifications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.13.682099v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Structured Chemical Reaction Modeling with Multitask Graph Neural Networks</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Astero, M., Li, A., Casiraghi, E., Rousu, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a multi-task architecture for chemical reaction modeling that achieves state-of-the-art performance on the USPTO-50K benchmark.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Modeling chemical reactions requires connecting fine-grained atom--bond edits with broader semantic categories. Yet, most machine learning approaches model these aspects in isolation: atom mapping, reaction center identification, and reaction classification are treated as separate problems. This separation limits accuracy, interpretability, and generalization. In this work, we argue that multitask learning provides a natural and powerful framework for reaction modeling. By jointly predicting mappings, centers, and classes within a single graph neural network, models can leverage structural dependencies between tasks. We proposed MARCC (Mapping-Assisted Reaction Center and Classification), a multitask architecture that achieves state-of-the-art performance on the USPTO-50K benchmark. MARCC demonstrates that multitask supervision not only improves accuracy across all tasks but also provides a structured representation of reaction mechanisms.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.13.682169v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PARSEbp: Pairwise Agreement-based RNA Scoring with Emphasis on Base Pairings</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tarafder, S., Bhattacharya, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a fast and effective multi-model RNA scoring method that integrates pairwise structural agreement across the conformational ensemble with base pairing consistency.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: High-fidelity scoring of RNA 3D structures remains a major challenge in RNA structure prediction and conformational sampling. While single-model methods for scoring RNA structures can capture individual structural features, they fail to capture the broader structural consensus within a conformational ensemble, limiting their effectiveness in ranking and model selection. Results: We present PARSEbp, a fast and effective multi-model RNA scoring method that integrates pairwise structural agreement across the conformational ensemble with base pairing consistency. By leveraging both alignment-based global structural agreement at the 3D level and base pairing consistency at the 2D level, PARSEbp efficiently constructs a consensus similarity matrix from which per-structure accuracy scores are computed. Tested on 16th Critical Assessment of Structure Prediction (CASP16) RNA targets, PARSEbp significantly outperforms existing single- and multi-model RNA scoring functions, including traditional statistical potentials, state-of-the-art deep learning methods, and consensus-based approaches, as well as a baseline variant of PARSEbp without the emphasis of base pairings, across a wide range of complementary assessment metrics. Availability: PARSEbp is freely available at https://github.com/Bhattacharya-Lab/PARSEbp.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.13.682106v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Biological versus Technical Reliability of Epigenetic Clocks and Implications for Disease Prognosis and Intervention Response</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sehgal, R., Borrus, D., Gonzalez, J. T., Markov, Y., ADNI,, Higgins-Chen, A. T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We comprehensively evaluate the technical and biological reliability of 18 epigenetic clocks, including chronological predictors, mortality predictors and pace-of-aging measures, reliable variants, and newer explainable clocks.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> DNA methylation based aging biomarkers, or epigenetic clocks, are increasingly used to estimate biological age and predict health outcomes. Their translational utility, however, depends not only on predictive accuracy but also on reliability, the ability to provide consistent results across technical replicates and repeated biological measures. Here, we leveraged the TranslAGE platform to comprehensively evaluate the technical and biological reliability of 18 Epigenetic clocks, including chronological predictors, mortality predictors, pace-of-aging measures, reliable variants, and newer explainable clocks. Technical reliability was quantified across four independent datasets. For standard replicate assays on EPIC and 450K arrays, nearly all clocks achieved excellent technical reproducibility. However, some clocks showed dramatic drops in technical reliability based on differences in slide position and DNA extraction protocol. PC-based clocks, especially PCGrimAge and SystemsAge remained technically reliable in all cases. In contrast, biological reliability, measured across repeated samples collected within hours, before and after meals, under acute stress, across environmental exposures, and over days, was markedly lower, with most clocks showing only moderate stability. PCGrimAge was the only clock with good ICC > 0.75 for biological reliability. Importantly, technical reproducibility did not predict biological reliability; clocks that were technically robust often proved biologically unreliable. We further demonstrated that reliability directly constrains downstream applications. Clocks with higher ICCs produced more stable prognostic associations with cognitive decline and more consistent responsiveness to a vegan diet intervention, whereas unreliable clocks yielded highly variable or spurious effects. Together, these findings reveal that technical reliability is not enough: biological reliability remains a critical limitation for many DNA methylation clocks that constrains their utility, and our work provides a roadmap for prioritising next-generation clocks most suited for clinical translation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.13.682176v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genotype-phenotype modeling of light ecotypes in Prochlorococcusreveals genomic signatures of ecotypic divergence</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Brenner, E. P., Vang, C. K., Johnson, C. G., Ravi, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We study Prochlorococcus' high-light/low-light ecotype preferences and show that a range of classical ML models trained to predict ecotype achieve exceptional binary classification accuracy even when predicting on partial genomes (Matthews Correlation Coefficient = 0.81 - 0.98).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Prochlorococcus is a cyanobacterial genus that exhibits photosynthetic capacity and remarkable genetic diversity. We analyze how Prochlorococcus genomics relate to high vs. low light environment adaptations, applying traditional comparative genomics and machine learning (ML) approaches to connect genotypes to phenotypes. We downloaded nearly 1,000 Prochlorococcus genomes from NCBI with information on their light adaptation ecotype (high-light/low-light) and depth of isolation using metadata through JGI. Average nucleotide identity analysis and traditional pangenome generation tools struggle to capture the cyanobacterial core genome, but despite its scant conservation, we clearly observe a sharp separation of the taxon by its light utilization preferences -- that is, its light ecotypes. A range of classical ML models trained to predict ecotype achieve exceptional binary classification accuracy even when predicting on partial genomes (Matthews Correlation Coefficient = 0.81 - 0.98), while regression models trained to predict the depth of isolation performed poorly, with relatively high root mean square error values (40.8 - 45.3m). For ecotype prediction, top features for the best-performing models included photosynthesis-associated genes and pathways, as well as some novel markers of unknown function. Our research findings recapitulate the extreme genetic versatility in Prochlorococcus and find that variable genetic markers allow excellent classification accuracy, and therefore, ecotype prediction, even for incomplete metagenomic assemblies, emphasizing the specialization and separation among these cyanobacterial ecotypes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.13.678797v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The long non-coding RNA, CyKILRb, augments oncogenic phenotypes via induction of PIK3R2 and activation of the PI3K/AKT axis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> xie, X., Macknight, H. P., Lu, A. L., Chalfant, C. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that downregulation of CyKILRb downregulation induced the loss of PIK3R2, an activator of PI3K, as well as RPS6KB2 and GNB2, two implicated tumor promotors, with a concomitant increase in the tumor</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Our laboratory recently identified a novel long noncoding RNA termed CyKILR that has two splice variants with distinct cellular localizations and opposing roles in tumorigenesis. The cytoplasmic variant, CyKILRb (exon 3 exclusion), promotes tumorigenesis, whereas the nuclear variant, CyKILRa (exon 3 inclusion), functions as a tumor suppressor. In this study, the molecular mechanism of the tumorigenic role of CyKILRb was characterized. Specifically, deep RNA sequencing analysis revealed that CyKILRb regulated the PI3K/AKT signaling pathway to block downstream tumor suppressors. In particular, downregulation of CyKILRb induced the loss of PIK3R2, an activator of PI3K, as well as RPS6KB2 and GNB2, two implicated tumor promotors, with a concomitant increase in the tumor suppressors, CDKN1A (p21) and CDKN1B (p27). In contrast, CyKILRb ectopic expression produced the opposite effect, and suppression of either PIK3R2, PI3K or AKT attenuated CyKILRb-induced cell proliferation and clonogenic survival. CyKILRb negatively regulated CyKILRa expression, which was blocked by inhibition of either PI3K or AKT. PIK3R2 ectopic expression overcame the cellular effects of CyKILRb downregulation, but not PI3K or AKT inhibition orienting the signaling pathway from CyKILRb[->]{uparrow}PIK3R2[->]PI3K[->]AKT[->]{downarrow}CyKILRa[->]enhanced oncogenicity. These findings highlight the critical role of CyKILRb in tumorigenesis and define a novel feed-forward regulatory mechanism linked to alternative RNA splicing.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.13.682173v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Transforming Esogastric Cancer Surgery Integrating SpiderMass Mass Spectrometry with Clinical and Microbiome Data for Margin Delineation and Prognosis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ledoux, L., Zirem, Y., Goossen, A., Renaud, F., Dufour, C., Viezant, J., Lion, C., Piessen, G., Salzet, M., FOURNIER, I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> SpiderMass, an ambient mass spectrometry technology, with clinical metadata and microbiome profiling couple along with Machine learning, provides a more precise and biologically informative intraoperative diagnostic tool, significantly enhancing surgical decision-makin, to improve patient outcomes and extend survival.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Esophageal-gastric cancers (EC) represent a significant global health concern, with esophageal cancer ranking seventh in terms of incidence and mortality worldwide. Gastric cancer is especially concerning, with an estimated one million new cases and 800,000 deaths annually. Late diagnoses often lead to poor outcomes, requiring critical interventions such as radical surgical resection with clear margins, in conjunction with chemotherapy, or radiotherapy to prevent recurrences and enhance survival. Thus, EC represents a significant clinical challenge, especially given the difficulty in achieving precise surgical margins in aggressive subtypes like poorly cohesive carcinoma (PCC). Moreover, pathological intraoperative margin assessment encounters significant issues, especially for PCCs, due to lacks of sensitivity for microscopic infiltration, potentially leading to recurrence and poorer patient outcomes. We address these critical limitations by integrating SpiderMass, an ambient mass spectrometry (MS) technology, with clinical metadata and microbiome profiling couple along with Machine learning. We demonstrate SpiderMass capability in real-time molecular margin delineation and identify distinct lipidomic and microbiome signatures correlating with tissue type and prognosis. Our integrative approach provides a more precise and biologically informative intraoperative diagnostic tool, significantly enhancing surgical decision-makin, to improve patient outcomes and extend survival.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.13.682035v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A comprehensive benchmark of single-cell Hi-C embedding tools.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dylan Plummer, Xiuyuan Lang, Shanshan Zhang, Yan Li, Jing Li, Fulai Jin</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> No single tool works best for single-cell Hi-C embedding and no single tool is better than another for deep learning.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Embedding is the key step in single-cell Hi-C (scHi-C) analysis which relies on capturing biological meaningful heterogeneity at various levels of genome architecture. To understand the strength and limitations of existing tools in various applications, here we use ten scHi-C datasets to benchmark thirteen embedding tools including Va3DE, a new convolutional neural network model that can accommodate large cell numbers. We built a software framework to decouple the preprocessing options of existing tools and found that no single tool works best across all datasets under default settings. The difficulty levels and preferred resolutions are different between benchmark datasets, and the choice of data representation and preprocessing strongly impact the embedding performance. Embedding cells from early embryonic stages relies on long-range compartment-scale contacts, but resolving cell cycle phases and complex tissue requires short-range loop-scale contacts. Both random-walk and inverse document frequency (IDF) transformation prefers long-range "compartment-scale" over short-range "loop-scale" embedding, while deep-learning methods better overcome sparsity at both scales and are more versatile with different resolutions. Finally, "diagonal integration" with independent data modal is a promising approach to distinguish similar cell subpopulations. Our findings underscore the significance of appropriate priors for scHi-C embedding and also offer insights into genome architecture heterogeneity.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41087414/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Deep learning detection of dynamic exocytosis events in fluorescence TIRF microscopy.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hugo Lachuer, Emmanuel Moebel, Anne-Sophie Macé, Arthur Masson, Kristine Schauer, Charles Kervrann</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We adapted DeepFinder, a U-Net originally designed for 3D noisy cryo-electron tomography data, for the detection of rare dynamic exocytosis events (termed ExoDeepFinder) observed in temporal series of 2D Total Internal Reflection Fluorescence Micro</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Segmentation and detection of biological objects in fluorescence microscopy is of paramount importance in cell imaging. Deep learning approaches have recently shown promise to advance, automatize and accelerate analysis. However, most of the interest has been given to the segmentation of static objects of 2D/3D images whereas the segmentation of dynamic processes obtained from time-lapse acquisitions has been less explored. Here we adapted DeepFinder, a U-Net originally designed for 3D noisy cryo-electron tomography (cryo-ET) data, for the detection of rare dynamic exocytosis events (termed ExoDeepFinder) observed in temporal series of 2D Total Internal Reflection Fluorescence Microscopy (TIRFM) images. ExoDeepFinder achieved good absolute performances with a relatively small training dataset of 12000 events in 60 cells. We rigorously compared deep learning performances with unsupervised conventional methods from the literature. ExoDeepFinder outcompeted the tested methods, but also exhibited a greater plasticity to the experimental conditions when tested under drug treatments and after changes in cell line or imaged reporter. This robustness to unseen experimental conditions did not require re-training demonstrating generalization capability of our deep learning model. ExoDeepFinder, as well as the annotated training datasets, were made transparent and available through an open-source software as well as a Napari plugin and can directly be applied to custom user data. The apparent plasticity and performances of ExoDeepFinder to detect dynamic events open new opportunities for future deep learning guided analysis of dynamic processes in live-cell imaging.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41086183/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A prognostic model for gastric cancer constructed by multiple machine learning algorithms.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xueli Yang, Xu Huang, Wang Ying, Tao Deng, Jun Zhang, Qianshan Ding</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A high-precision risk model based on random survival forest (RSF) and generalized boosted regression modelling (GBM) was constructed using 7 hub genes to predict prognosis of Gastric cancer patients.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Gastric cancer (GC) is a highly heterogeneous disease that requires highly accurate prognostic models. Machine learning is a powerful tool for identifying predictive biomarkers and developing prognostic models. Here, we aim to integrate bioinformatics and machine learning algorithms to construct a risk model to predict prognosis of GC patients. Transcriptome data and clinical information of GC patients were obtained from the Cancer Genome Atlas (TCGA) database. Microarray data (GSE84437 and GSE26253) were obtained from the Gene Expression Omnibus (GEO) database. Univariate Cox regression analysis was used to screen prognostic genes. The risk genes closely related to prognosis were screened by machine learning algorithms and the risk score was calculated. Kaplan-Meier survival curve, time-dependent receiver operating characteristic (ROC) curve, univariate and multivariate Cox regression analysis were used to verify the validity of the risk model. The protein expression of hub genes in GC tissues was evaluated by immunohistochemical staining. 7 hub genes (CGB5, FEM1A, MATN3, ZNF101, MARCKS, BRI3BP and APOD) were identified and correlated with GC prognosis. A high-precision risk model based on random survival forest (RSF) and generalized boosted regression modelling (GBM) was constructed using these 7 hub genes. The risk model has good predictive ability for GC patients' prognosis, and the risk score could be used as an independent prognostic factor for GC. In addition, the protein expression levels of CGB5, MATN3, MARCKS and APOD in GC tissues were significantly higher than those in normal tissues, and correlated with the pathological characteristics of GC patients. The risk model composed of 7 hub genes can accurately evaluate the prognosis of GC patients, which may contribute to the precise and personalized treatment of GC patients.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41085794/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrated Chemical Array and SERS Profiling of Plasma Small Extracellular Vesicles for Breast Cancer Diagnosis.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xin Feng, Wei Zhang, Simon Chang-Hao Tsao, Chao Shen, Lianghai Hu, Yuling Wang</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a high-throughput strategy integrating amphiphile-dendrimer supramolecular probe (ADSP)-based sEV capture with surface-enhanced Raman scattering (SERS) for sensitive, multiplexed detection of breast cancer-related surface proteins.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Small extracellular vesicles (sEVs) are nanoscale vesicles carrying biomolecules reflective of their cellular origin, making them attractive biomarkers for cancer diagnosis. In this study, we present a high-throughput strategy integrating amphiphile-dendrimer supramolecular probe (ADSP)-based sEV capture with surface-enhanced Raman scattering (SERS) for sensitive, multiplexed detection of breast cancer (BC)-related surface proteins. Plasma-derived sEVs from BC patients at different clinical stages were analyzed, focusing on CD9, EpCAM, and HER2 as key proteins linked to vesicle identity and tumor progression. Gold nanoparticle-based SERS nanotags conjugated with specific antibodies enabled precise detection. Statistical and machine learning analyses of protein profiles allowed accurate discrimination among healthy donors, ductal carcinoma in situ (Stage 0), early stage BC (Stage I-II), and metastatic-stage BC (Stage IV). This integrated platform provides a powerful tool for BC diagnosis and highlights the potential of sEV-based liquid biopsy strategies for clinical application.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41084822/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Correspondence regarding "Clinical parameters-based machine learning models for predicting intraoperative hemodynamic instability in hypertensive pheochromocytomas and paragangliomas patients".</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Emmanuel Pio Pastore</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present machine-learning models to predict intraoperative hemodynamic instability in hypertensive pheochromocytoma and paraganglioma surgery.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Zhao et al. present machine-learning models to predict intraoperative hemodynamic instability in hypertensive pheochromocytoma and paraganglioma surgery. The clinical motivation is sound and the reported discrimination and decision-curve metrics indicate promise. Yet the work illustrates two recurring challenges for clinical AI. First, models trained and tested within a single institution risk overestimation without external validation that reflects the intended deployment. Second, calibration and timing of prediction must be explicit to avoid information leakage and to ensure actionable probabilities. Attention to these methodological points will enhance interpretability and safe translation into peri-operative practice.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41083651/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Plasma proteomic profiles for early detection and risk stratification of non-small cell lung carcinoma: A prospective cohort study with 52,913 participants.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Renjia Zhao, Heyang Lu, Huangbo Yuan, Shuaizhou Chen, Kelin Xu, Tiejun Zhang, Zhenqiu Liu, Yanfeng Jiang, Chen Suo, Xingdong Chen</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified three plasma proteins with the highest predictive power for non-small cell lung cancer (NSCLC) and showed that CXCL17, CEACAM5, and WFDC2 are the most likely to be found in the early stage of the disease.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Early detection of non-small cell lung cancer (NSCLC) can improve survival rates, and plasma proteomics may provide effective tools for risk prediction. The population for this study included 52,913 participants and 2911 plasma proteomics from UK Biobank. The cohort was divided into discovery and validation cohorts based on their countries. Cox regression, XGBoost, and SHAP analysis were used to identify key NSCLC-associated proteins. Machine learning (ML) models were developed and validated across different timeframes. Risk stratification was performed using protein levels. Temporal change analysis and two-sample Mendelian Randomization (MR) were conducted to assess the early prediction ability and causal relationships, respectively. Twenty-five proteins were significantly associated with NSCLC. ML identified CXCL17, CEACAM5, and WFDC2 as having the highest predictive power. The three-protein panel plus epidemiological indicators exhibited superior performance in 5- and 10-year predictions, achieving AUROC of 0.904 (95%CI: 0.839-0.968) and 0.873 (95%CI: 0.815-0.931), respectively. Risk stratification identified a high-risk group with a 9.18-fold higher risk than the general population and a 16.75-fold higher risk than the low-risk group, respectively. Temporal change analysis revealed that protein expression levels in cases were globally higher than in controls up to 10 years before diagnosis. MR implied a suggestive causal relationship between CXCL17 and NSCLC. Our findings suggest three plasma proteins possess robust predictive capabilities for NSCLC, allowing for predictions up to 10 years in advance. Incorporating these biomarkers into risk models enhances early detection, providing a foundation for targeted screening and precision medicine in NSCLC.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40480837/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

