<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Thursday, 31 July 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identification of DNA damage response and crotonylation-related biomarkers for lung adenocarcinoma via machine learning and WGCNA.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kandi Xu, Yayi He</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study on the activation of DNA damage response and crotonylation in lung adenocarcinoma .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> DNA damage response (DDR) and crotonylation occur frequently in lung adenocarcinoma (LUAD), but their relationship is yet to be elucidated. RNA sequencing data from LUAD patients in GSE27262 and GSE140797 datasets were obtained. DDR-crotonylation-related differentially expressed genes were identified by differential analysis and weighted gene co-expression network analysis. Three machine learning algorithms were used to screen for foremost ones. Various analyses such as immune infiltration, genetic mutation, and drug sensitivity were carried out based on multiple databases and cytotoxicity tests. Three hub genes were extracted with the potential as diagnostic and predictive criteria. They were also related to immune infiltration, immune checkpoint expression and genome heterogeneity of LUAD. Additionally, drug sensitivity analysis pinpointed probable small molecule inhibitors targeting our hub genes. This study confirmed the activation of DDR and crotonylation in LUAD, which is characterized by abnormal expressions of the hub genes, and translated it into clinical applications. We also overviewed the relationships among those specific molecular features, metabolic reprogramming and immune evasion.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40736752/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Gene signatures characterizing driver mutations in lung squamous carcinoma are predictive of the progression of pre-cancer lesions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lin, Y., Mekala, V., Li, J.-r., Wang, X., Wu, J., Aminu, M., Zhang, J., Ripley, R. T., Amos, C. I., Cheng, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We derive tumor-derived gene expression signatures from lung squamous cell carcinoma tumors to predict progression risk in precancerous lesions and show that they are more strongly associated with patient prognosis in LUSC than the presence of genomic aberrations alone.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Patients with lung squamous cell carcinoma (LUSC) are often diagnosed at advanced stages, limiting opportunities for early intervention. LUSC develops through a multistep progression from low-grade lesions to high-grade lesions, including carcinoma in situ (CIS), of which approximately half progress to invasive cancer while the other half regress. Although frequent mutations and copy number alterations have been documented in LUSC and observed in precursor lesions, their prognostic significance in precancerous stages remains largely unexplored. In this study, we leveraged gene expression data from LUSC tumors in The Cancer Genome Atlas (TCGA) to derive transcriptional signatures corresponding to 34 key driver genomic aberrations, including mutations, amplifications, and deletions. These tumor-derived gene signatures were then applied to precancerous lesion datasets to assess their ability to characterize developmental stages and predict progression risk. We found that many of these signatures increased progressively across lesion stages, reflecting their roles in early tumorigenesis. In particular, several signatures accurately predicted which CIS lesions would progress to invasive cancer. Furthermore, these signature scores were more strongly associated with patient prognosis in LUSC than the presence of genomic aberrations alone. We also examined the relationship between driver-associated signatures and the tumor immune microenvironment. Signature scores were significantly correlated with immune features such as immune cell infiltration and immune checkpoint gene expression, including CD274 (PD-L1). Interestingly, these associations varied across lesion stages, indicating dynamic immune interactions during cancer evolution. Together, our findings demonstrate that tumor-derived driver gene expression signatures provide valuable insight into the biology and progression risk of precancerous lesions, offering potential utility for early detection and intervention strategies in LUSC.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.29.664792v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Structure-Based Computational Pipeline for Broad-Spectrum Antiviral Discovery</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Castellanos, M. A., Payne, A. M., Scheen, J., MacDermott-Opeskin, H., Pulido, I., Balcomb, B. H., Griffen, E. J., Fearon, D., Barr, H., Lahav, N., Cousins, D., Stacey, J., Robinson, R., Lefker, B., Chodera, J. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a structure-based computational pipeline designed to identify and evaluate broad-spectrum inhibitors across viral family members for a given target in order to support spectrum breadth assessment and prioritization in lead optimization programs.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The rapid emergence of viruses with pandemic potential continues to pose a threat to public health worldwide. With the typical drug discovery pipeline taking an average of 5-10 years to reach clinical readiness, there is an urgent need for strategies to develop broad-spectrum antivirals that can target multiple viral family members and variants of concern. We present a structure-based computational pipeline designed to identify and evaluate broad-spectrum inhibitors across viral family members for a given target in order to support spectrum breadth assessment and prioritization in lead optimization programs. This pipeline comprises three key steps: (1) an automated search to identify viral sequences related to a specified target construct, (2) pose prediction leveraging any available structural data, and (3) scoring of protein-ligand complexes to estimate antiviral activity breadth. The pipeline is implemented using the drugforge package: an open-source toolkit for structure-based antiviral discovery. To validate this framework, we retrospectively evaluated two overlapping datasets of ligands bound to the SARS-CoV-2 and MERS-CoV main protease (Mpro), observing useful predictive power with respect to experimental binding affinities. Additionally, we screened known SARS-CoV-2 Mpro inhibitors against a panel of human and non-human coronaviruses, demonstrating the potential of this approach to assess broad-spectrum antiviral activity. Our computational strategy aims to accelerate the identification of antiviral therapies for current and emerging viruses with pandemic potential, contributing to global preparedness for future outbreaks.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.29.667267v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A SNP panel for co-analysis of capture and shotgun ancient DNA data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fournier, R., Pearson Fulton, A., Reich, D. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a method to identify single nucleotide polymorphisms (SNPs) with minimal technology-specific bias with minimal spurious Z-scores due to different sequencing platforms, and which has been effectively assayed in a large fraction of ancient human DNA experiments.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Advances in technology have decreased the cost of generating genetic variation data from ancient people, resulting in exponentially increasing numbers of individuals with whole genome data. However, each technology comes with platform-specific biases, limiting co-analyzability of individuals sequenced with different technologies as well as joint analysis of modern and ancient individuals. We present a method to identify single nucleotide polymorphisms (SNPs) with minimal technology-specific bias. Leveraging data from over 17,000 ancient individuals, we apply this method to identify a set of more than a million SNPs that we call the "Compatibility" panel, and which has been effectively assayed in a large fraction of ancient human DNA experiments published to date. We also identify a subset of these SNPs, the "Compatibility-HO" panel, which further restricts to positions that have been assayed in more than ten thousand modern individuals from more than a thousand diverse populations using the Affymetrix Human Origins (HO) genotyping array. The Compatibility panel reduces spurious Z-scores due to different sequencing platforms by nearly an order of magnitude, while retaining around 70-90% of statistical power for f-statistic analysis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.30.667733v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An updated resource of 180K soybean SNP genotyping array based on the T2T reference genome</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hwang, J.-H., Lee, S., Lee, J. S., Kim, K. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We successfully remapped 180,000 SNP markers from the Korean soybean core collection to the Williams 82 reference genome version 1 (Wm82.v1) using sequence-based alignment of flanking regions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single nucleotide polymorphism (SNP) genotyping has revolutionized crop improvement by enabling high-resolution genomic analyses and accelerating breeding programs. In soybean (Glycine max (L.) Merr.), a globally important legume crop, existing genotyping data for 180,961 SNP markers from the Korean soybean core collection were generated using the outdated Williams 82 reference genome version 1 (Wm82.v1), which contains numerous assembly gaps and misassemblies that limit genomic resolution. While high-quality reference genomes including Wm82.v4 and Wm82.v6 (telomere-to-telomere assembly) are now available, the valuable existing SNP array data have not been integrated with these improved genomic resources. Here we show successful remapping of the 180K SNP array data to both Wm82.v4 and Wm82.v6 reference genomes through sequence-based alignment of flanking regions. We extracted flanking sequences from SNP marker positions in Wm82.v1 and mapped them to the newer reference versions based on sequence similarity, excluding markers with mapping failures, allele mismatches, low-identity alignments, or multiple mappings, which resulted in successful mapping of 175,202 and 175,763 markers to Wm82.v4 and Wm82.v6, respectively. We also remapped genotype data from 927 soybean accessions (497 USDA-GRIN accessions and 430 Korean core collection accessions) to both reference versions. This updated SNP dataset provides the soybean research community with a comprehensive genomic resource that leverages both existing genotyping investments and state-of-the-art reference genome assemblies for enhanced crop improvement and genomic studies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.29.667553v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Benchmarking Alignment Strategies for Hi-C Reads in Metagenomic Hi-C Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, Y., Zuo, W., Huang, J., Sun, F., Du, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We evaluate seven Hi-C alignment tools for metagenomics and compare their performance against each other on inter-contig read pairs and binning quality.</p><p style="font-size: 16px; text-align: justify;">Background: Metagenomics combined with High-throughput Chromosome Conformation Capture (Hi-C) offers a powerful approach to study microbial communities by linking genomic content with spatial interactions. Hi-C enhances shotgun sequencing by revealing taxonomic composition, functional interactions, and genomic organization from a single sample. However, aligning Hi-C reads to metagenomic contigs presents challenges, including the unique statistical distribution of Hi-C paired-end reads, multi-species complexity, and gaps in assemblies. Although many benchmark studies have evaluated general alignment tools and Hi-C data alignment, none have specifically addressed metagenomics Hi-C data. Results: Here, we selected seven alignment strategies that have been used in Hi-C analyses: BWA MEM -5SP, BWA MEM default, BWA aln default, Bowtie2 default, Bowtie2-very-sensitive-local, Minimap2 default, and Chromap default. We benchmarked them on one synthetic and seven real-world environments, and evaluated these tools based on the number of inter-contig Hi-C read pairs and their influence on downstream tasks, such as binning quality. Conclusion: Our findings show that BWA MEM -5SP consistently outperforms other tools across all environments in terms of inter-contig read pairs and binning quality, followed by BWA MEM default. Chromap and Minimap2, while less effective in these metrics, demonstrate the highest computational efficiency.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.30.667754v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Colorectal adenoma presence is associated with decreased menaquinone pathway functions in the gut microbiome of patients undergoing routine colonoscopy</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Vilkote, I., Silamikelis, I., Klovins, J., Tolmanis, I., Lejnieks, A., Runce, E., Cebere, K., Margole, K., Sjomina-Gubareva, O., Silamikele, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of gut microbiome composition and the presence of colorectal adenomas in patients undergoing routine colonoscopy .</p><p style="font-size: 16px; text-align: justify;">Background Colorectal cancer (CRC) is among the most common malignancies worldwide, with colorectal adenomas recognized as well-established precursors to CRC. Changes in gut microbiota appear to be linked to CRC by promoting chronic inflammation, immune dysfunction, and metabolic issues that drive tumor growth and progression. Objectives To explore the relationship between gut microbiome composition and the presence of colorectal adenomas in patients undergoing routine colonoscopy. Materials and methods Patients were selected from those receiving standard colonoscopy based on strict inclusion and exclusion criteria, to minimize potential confounding factors such as previous colorectal surgeries, inflammatory bowel diseases, and the use of antibiotics or probiotics. Fecal samples were collected before bowel preparation for the colonoscopy procedure, and metagenomic shotgun sequencing was used to analyze the composition and functions of the gut microbiome. Results Overall, 136 participants were recruited, and 56 of them had colorectal adenomas. Although no distinction was observed in alpha diversity, beta diversity analysis indicated significant differences between the adenoma-positive and adenoma-negative groups. Signs of dysbiosis were found in patients with adenomas: increased abundance of the genera Bacteroides and Prevotella and decreased abundance of Faecalibacterium and Anaerostipes species. Beta diversity analysis showed statistically significant differences in the structure of the microbiota. Significant differences in the relative abundance of UBA7597 sp003448195 were observed between groups. Functionally, decreased vitamin K2, SCFA (propionate) synthesis, along with lower Stickland fermentation activity was observed, indicating altered microbial metabolism. These changes may compromise epithelial barrier support, anti-inflammatory signaling, and energy metabolism in the colon. Conclusions The discovery of microbial taxa and functional pathways associated with the presence of adenomas underscores the potential of microbiota-based biomarkers and therapeutic strategies in the prevention and management of colorectal cancer.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.29.667347v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multi-Model LLM Architectures for Personalized Summarization and Relevance Ranking in Biomedical Literature</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pandey, A., Kuznetsov, A., Mukhopadhyay, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We develop and evaluate a personalized literature review system that efficiently processes and summarizes biomedical literature to provide timely, relevant insights for researchers.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Objective: To develop and evaluate a personalized literature review system that efficiently processes and summarizes biomedical literature to provide timely, relevant insights for researchers. Methods: The system integrates ontology-aware keyword extraction (MeSH/ACM constrained TF-IDF from CV/Research Statement), citation-informed retrieval (PubMed and NIH iCite API), and dual-model large language model (LLM) summarization (Google Gemini 2.0 flash, OpenAI GPT-4o-mini). These LLMs leverage advanced Transformer architectures, building on foundations such as BERT, BART, and BioBERT. A two-stage ranking algorithm combines Relative Citation Ratio (RCR) with cosine similarity. Summary quality was evaluated using ROGUE-1/2/L and BERTScore. The system is deployed as a Streamlit web application. Results: Across 20 biomedical queries, the system demonstrated strong average performance (BERT-F1 {approx} 0.86), with cosine similarity strongly correlating with summary quality. Human evaluation involving 10 users yielded average scores above 4.5/5 across summary fidelity and keyword relevance. Conclusion: Hybrid ranking and ensemble LLM summarization significantly accelerate scientific sense-making. These findings suggest broad applicability to various domains beyond biomedicine.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.29.667503v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Model of leaf biomass partitioning coefficient in different leaf ranks of rapeseed (Brassica napus L.) main-stem</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, W., Zhang, W., Wu, Q., Sun, C., Cao, J., Ge, D., Li, H., Cao, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed the leaf biomass partitioning coefficient model for the main-stem at different leaf ranks in rapeseed, which is a theoretical basis for functional-structural rapeseed model.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Leaf growth is a dynamic process, and understanding the number of main-stem leaves and the distribution of leaf biomass at different development stages can provide a theoretical basis for the development of functional-structural rapeseed model. To address the gap in research on the leaf biomass partitioning coefficient model in the crop growth model, we developed the leaf biomass partitioning coefficient model for the main-stem at different leaf ranks in rapeseed. The outdoor experiment with cultivars, Ningyou 18 (V1, conventional), Ningyou 16 (V2, conventional), and Ningza 19 (V3, hybrid), and designed treatment of cultivars x fertilizer x transplanting density in the 2012-2015 growth season. The model was fitted by calculating the ratio of leaf biomass at different leaf ranks on the main-stem to the total leaf biomass on the main-stem, and normalizing the leaf ranks into (0-1] interval. We parameterized the model variables to elucidate how cultivar and environmental conditions influence the leaf biomass partitioning coefficients for different leaf ranks. The descriptive models were validated with independent experimental data, the correlation (r) of simulation and observation values all greater than 0.9 with significant levels at p < 0.001. The absolute values of the average absolute difference (da) are all less than 0.011 g g-1. The ratio of da to the average observation (dap) for the normalized leaf rank, and leaf biomass partitioning coefficient at seedling stage, and budding and bolting stage, and blooming stage, and mature stage are 13.599%, 11.236%, 11.312%, 3.077%, and 8.661%, respectively. The RMSE values are all less than 0.032 g g-1. The models developed in this paper have good performance and reliability for predicting leaf biomass partitioning coefficient of main-stem at different leaf ranks in rapeseed. This study integrates the leaf morphology model of rapeseed with the rapeseed growth model through biomass partitioning, contributing to the development of the functional-structural rapeseed model.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.27.667074v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ADCY3 Ser107Pro links difficulty awakening in the morning to adiposity through circadian regulation of adipose thermogenesis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tchio, C., Maher, M., Moth, C., Meiler, J., Lane, J., Taylor, H., Williams, J., Saxena, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify a missense variant in ADCY3 as a pleiotropic regulator of circadian preference and adiposity, and show that morning difficulty awakening is linked to morningness, BMI, and fat mass in European and African ancestry groups.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Modern lifestyles often disturb circadian rhythms, yet the genetic circuits that convert this stress into metabolic dysfunction remain poorly defined. Here, we identify a missense variant in ADCY3 (rs11676272; Ser107Pro) as a pleiotropic regulator of circadian preference and adiposity. Using genome-wide pleiotropy analysis in ~480,000 UK Biobank participants, we show that the G risk allele (Pro107) increases morningness, BMI, and fat mass in European (N=451,324) and African (N=8,738) ancestry groups, with behavioral amplification by morning difficulty awakening observed in Europeans, power limited interaction modeling in other populations. Structural modeling and transcriptomic analysis suggest this allele destabilizes ADCY3 and alters adipose-specific splicing and expression. In mice, Adcy3 is rhythmically expressed in adipose tissue, with the conserved Pro107 site showing BMAL1 binding and cold-inducible activation. Human adipose ADCY3 expression also increases after weight loss. Together, these findings reveal a genotype-dependent, behaviorally modifiable axis linking difficulty awakening to adipose thermogenesis and obesity risk.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.28.667339v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Spatial Needleman-Wunsch: A Deterministic Dynamic Programming Framework for Molecular Docking</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Curry, J. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Spatial Needleman-Wunsch, a deterministic dynamic programming framework adapted from sequence alignment and applied to three-dimensional molecular structures.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Most molecular docking tools today rely on stochastic sampling or heuristic optimization. They can be effective, but reproducibility is often hit or miss, and there's no guarantee the best binding pose has been found. This paper introduces Spatial Needleman-Wunsch, a deterministic dynamic programming framework adapted from sequence alignment and applied to three-dimensional molecular structures. Ligands and protein cavities are voxelized into grids; chemical interactions are scored with compatibility matrices inspired by BLOSUM and PAM; and optimal alignments are computed exhaustively. The result is a docking method that's reproducible, interpretable, and mathematically grounded. The framework supports Boltzmann ensemble analysis, Pareto trade-off modeling, conformational sampling, and adaptive scoring. Proof-of-concept tests on synthetic systems and real targets like HIV protease show perfect reproducibility and meaningful chemical discrimination. This approach eliminates random variability while making interaction scoring fully transparent. It opens the door to principled docking exploration, something essential in drug discovery and structural biology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.24.666489v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SPEECHLESS duplication in grasses expands potential for environmentalregulation of stomatal development</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Erberich, J. M., Bennett, B., Bergmann, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that paralogs of the stomatal regulator SPEECHLESS (SPCH) remain under selection and show that they can be duplicated in response to environmental change.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Plants acquire atmospheric carbon dioxide for photosynthesis while minimizing water loss and do so by regulating stomatal function and development. The ancestral basic helix-loop-helix transcription factor (TF) gene that drove stomata production in early land plants diversified in sequence and function to become paralogs SPEECHLESS (SPCH), MUTE, and FAMA. Extant angiosperms use these three TFs and their heterodimer partners to regulate stomatal cell identities. Grasses exhibit a particularly interesting set of duplications and losses of SPCH. Using phylogenetic methods, we tracked the duplication of SPCH to the Poaceae-specific rho whole genome duplication and demonstrated that both paralogs remain under selection. By following responses to environmental change in B. distachyon plants bearing mutations in either BdSPCH1 or BdSPCH2, we reveal paralog-specific divergence in response to light or temperature shifts, and further show this behavior is conserved O. sativa SPCH paralogs. Plausible molecular mechanisms underpinning paralog divergence, and cellular mechanisms driving the stomatal phenotypes are supported by analyses of RNA and protein expression in B. distachyon and sequence variation among grasses. These studies suggest ways in which a duplication of a key stomatal regulator enables adaptation and could inform genetic strategies to mitigate anticipated stressors in agronomically important plants.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.29.667563v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Assessment of Spring wheat responses to late season heat stress under Mediterranean conditions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Awad-Allah, S. M., Amer, S. M., Abdel-Moneim, M. H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Late sowing of spring wheat cultivars with reduced number of grains/spike, resulting in 42% yield penalty under heat stress, and a decline in biological yield over that of grain yield.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In Mediterranean climatic regions, wheat yield is significantly influenced by sowing date and cultivar selection, particularly where production could be limited by heat waves during critical growth stages. This investigation was undertaken to determine the effect of shifting sowing date on a cohort of six high yielding spring wheat cultivars (Triticum aestivum L.), by delaying sowing date 15 and 30 days in a two-year field trial, aiming to highlight agronomic traits contributing to sustained yield potential under heat stress induced by late sowing, with a focus on identifying tolerant cultivars. Our findings showed that heat stress during grain filling was less detrimental compared to its occurrence during anthesis. Late sowing shifted the timing of heat stress, resulting in floral abortion and significantly reduced number of grains/spike, consequently imposing 42% yield penalty, underscoring number of grains/spike stability as key factor to sustain yield under heat stress during anthesis. On the contrary, harvest index of sowing delayed by 30 days was significantly higher compared to both 15 days late sowing and the optimum sowing date, suggesting that decline in biological yield exceeded that of grain yield. Upon shifting sowing date, the strength of traits association is altered, grain yield associations strongest positive correlation in the 30 day late sowing was plant height (r=0.86), in the 15 days late sowing was number of total tillers/m2 (r=0.63) and in optimum sowing date was biological yield (r= 0.62), suggesting that grain yield enhancement through selected traits with direct or indirect contributions should be evaluated independently across different growing conditions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.29.667146v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Scalable, fast and accurate differential gene expression testing from millions of cells of multiple patients</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Santacatterina, G., Tosato, N., Milite, S., Davydzenka, K., Insaghi, E., Cozzini, S., Sanguinetti, G., Egidi, L., Caravagna, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new Bayesian framework for single-cell differential expression analysis that can finally integrate datasets from large clinical cohorts, atlas projects, or drug-response screens.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Since the development of DNA microarrays and later RNA bulk sequencing, testing with statistically independent samples has been the standard method for detecting genes with different transcription patterns. Single-cell assays challenge these assumptions because individual cells are statistically dependent, and all proposed methodologies present mathematical limitations or computational bottlenecks that prevent a seamless integration of data from many cells and patients simultaneously. In this work, we solve this crucial limitation by introducing a Bayesian framework that retrieves the independence structure at the level of individual patients, separating differences across individuals from actual transcriptional differences. Leveraging multi-GPU and variational inference, our approach excels across different experimental designs and, for the first time, scales to analyse over 10million cells in less than 2hours. This new framework enables single-cell differential expression analysis that can finally integrate datasets from large clinical cohorts, atlas projects, or drug-response screens with thousands of samples and millions of cells.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.24.666556v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Pathway-Centric Visualization of Cell-Cell Communication in Single-Cell Transcriptomics Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Maryam, S., Manolov, M., Kramann, R., Hayat, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed scVizComm, a ShinyApp based web-portal that can be used to interactively visualize ligand-receptor networks across cell-cell communication between different cell-types and different conditions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Objective: Single-nuclei transcriptomics enables investigating ligand-receptor mediated cell-cell communication between different cell-types. However, current tools do not allow for non-programmatic means to access this analysis. Additionally, most methods can not account for molecular pathways involved in downstream receptor signaling in cell-cell communication. Methods: We developed scVizComm, a ShinyApp based web-portal that can be used to interactively visualize ligand-receptor networks across cell-types and different conditions. Results: scVizComm can be used to host pathway centric pre-calculated cell-cell communication results. Using human kidney and heart single-nuclei transcriptomics data, we showcase the utility of scVizComm in understanding ligand-receptor interactions involved in fibrosis related pathways. Conclusion: scVizComm allows interactive visualisation and pathway centric prioritisation of cell-cell communication analyses.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.25.666732v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

