<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Sunday, 08 June 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genetic modulation of immune gene co-expression in the aged mouse hippocampus by the Apbb1ip locus</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tomkins, J. E., Methi, A., Bouland, G., Doludda, B., Murphy, K., Miller, J. A., Overall, R. W., Mozhui, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified a key age-associated co-expression module enriched in immune related genes, and quantitative trait locus mapping of this module uncovered a genetic regulatory locus in which Apbb1ip was the primary candidate gene.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Ageing is a major risk factor for many neurodegenerative diseases, and the hippocampus is particularly vulnerable to the effects of ageing. To define the transcriptomic changes related to ageing and the impact of genetic variation, we analysed hippocampal gene expression data generated from a genetically diverse panel of inbred mouse strains belonging to the BXD Family. We applied a combination of differential expression, differential correlation, and weighted gene co-expression network analyses, followed by genetic mapping to delineate age-associated transcriptomic patterns and genetic modulators. This study revealed an upregulation in immune response and microglial genes. We identified a key age-associated co-expression module enriched in immune related genes, and quantitative trait locus mapping of this module uncovered a genetic regulatory locus in which Apbb1ip was the primary candidate gene. Additionally, gene level differential correlation analysis identified a substantial restructuring of the hippocampal transcriptome during ageing. Notably, Ywhab, an important signalling chaperone displayed altered gene expression correlations with >70 genes, between young and aged mice. These results provide novel insights into transcriptional dynamics of hippocampal ageing and identified the immune gene Apbb1ip as a potential modulator of immune response and microglial gene upregulation with implications for neurodegenerative disease pathogenesis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.04.657838v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Lizards on a sky archipelago: Genomic approaches to the evolution of the mountain genus Iberolacerta</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Talavera, A., Burriel-Carranza, B., Mochales-Riano, G., Estarellas, M., Arribas, O., Tejero-Cicuendez, H., Salces-Ortiz, J., Fernandez-Guiberteau, D., Amat, F., Niell, M., Fernandez, R., Mikheyev, A. S., Carranza, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a chromosome-level genome assembly for Iberolacerta aurelioi, an Endangered rock lizard endemic to the Pyrenees, and provide insights into homozygosity burdens, which offer valuable information for conservation efforts.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The mountain-dwelling lizards of the genus Iberolacerta inhabit several isolated massifs across central and southwestern Europe. Their restricted and fragmented ranges, coupled with high altitude specialization in most species, entail a significant threat in the context of climate change for this group of lizards that has attracted interest from different fields. On the one hand, the alpine confinement of these relict species precedes the Pleistocene glacial cycles, and a few hypotheses have been proposed to explain it: from competitive exclusion by the wall lizards of the genus Podarcis, to adaptations to either cold or hypoxia, that would prevent them from expanding into lowlands. On the other hand, extensive research on chromosome evolution has shown Iberolacerta karyotypes to fairly differ from other lacertid lizards, exhibiting reductions in chromosome numbers and multiple sex chromosome determination systems. Here we present a chromosome-level genome assembly for Iberolacerta aurelioi, an Endangered rock lizard endemic to the Pyrenees. This genome has shed light on a genome architecture shaped by chromosome fusions, whose adaptive potential we discuss, as well as on expression shifts towards a hemoglobin isoform of enhanced oxygen affinity, as an adaptation to altitudinal hypoxia. In addition, medium coverage whole-genome sequencing data from 12 representatives encompassing all species and subspecies within the genus allowed us to address phylogenomic relationships, unveiling introgression events, gathering evidence in support of the competitive exclusion hypothesis through past demographic inference, and providing insights into homozygosity burdens, which offer valuable information for conservation efforts.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.04.657811v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Exploring the Potential of Large Language Models in Differential Abundance Analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Franco-Alba, R., Goryanin, I., Goryanin, I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Large Language Models are not yet reliable enough to replace human expertise in complex scientific tasks, including differential abundance analysis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The rapid development of Large Language Models (LLMs) has opened new possibilities in various fields, including microbiome research. This dissertation explores the applica- tion of LLMs for differential abundance analysis, a crucial method for understanding the relationship between microbial communities and health conditions. Through a series of experiments, we assessed the capabilities and limitations of LLMs in automating and enhancing the differential abundance analysis process. Our findings reveal that while LLMs can effectively extract relevant information from scientific literature and assist in generating reports, they also face significant chal- lenges, including data inaccuracies, hallucinations, and issues with reference generation. These challenges highlight the importance of integrating LLMs with human oversight to ensure scientific rigor. The study suggests that while LLMs can increase research efficiency, they are not yet reliable enough to replace human expertise in complex scientific tasks. This research contributes to the ongoing dialogue on the role of AI in scientific research, emphasizing the need for further development of LLMs and the exploration of hybrid models that combine AI capabilities with human judgment. The dissertation concludes with recommendations for improving the accuracy and consistency of LLMs, and expanding their applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.04.657820v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genome-wide characterization of upper Fraser River sockeye salmon identifies a region of major differentiation between run timing groups in the Stuart River Watershed</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sutherland, B. J. G., Williamson, C., Toth, B., Sterritt, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Early Stuart sockeye salmon Oncorhynchus nerka undergo one of the most intensive and far-reaching migrations in the Fraser River (British Columbia, Canada) that has likely exerted selective pressures shaping their genetics and physiology.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Early Stuart sockeye salmon Oncorhynchus nerka undergo one of the most intensive and far-reaching migrations in the Fraser River (British Columbia, Canada) that has likely exerted selective pressures shaping their genetics and physiology. To support upper Fraser River sockeye management objectives, there is a need to further understand the unique genetics of these populations. Based on neutral genetic markers, Early Stuart sockeye are considered a metapopulation with little to no genetic differentiation among their approximately 50 spawning streams. In this study we make use of existing resources to evaluate genome-wide genetic differentiation in this metapopulation and put it into the context of other nearby populations. We confirm that genetic differentiation is low or negligible in the metapopulation at a whole-genome level, although we identify a slight increase in differentiation at greater distances within the watershed compared to more proximal streams (e.g., FST = 0.001-0.002 compared with undifferentiated). Even the greatest differentiation detected is very low, being nine times lower than that between run timing groups (Early Stuart vs. Stuart-Summer; e.g., FST = 0.0091). Notably, a region of major differentiation was detected between the run timing groups on Chr18 between 56.3-58.0 Mbp (average FST = 0.661). This genomic region appears to contain a unique haplotype specific to the Stuart-Summer, Nadina River, and Stellako River populations (i.e., upstream to the Nechako River). By contrast, Early Stuart and other upper Fraser River sockeye outside of the Nechako Watershed do not possess this haplotype. The genomic region is within ancestral chromosome 15.2, the homeolog to a chromosome arm of Chr12 (i.e., 15.1); this arm of Chr12 is known to contain a major differentiated region associated with run timing and life history variation in Alaska and the Okanagan. This finding provides additional evidence for the importance of this genomic region in both homeologous chromosomes within different populations of the same species, and points to the uniqueness of upper Fraser River sockeye in this potential major effect locus.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.04.657877v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">STARR-CRAAVT: A platform to design cell type-specific regulatory elements for next-generation gene therapy</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Becker, R., Choudhury, P., Oti, M., Fouani, Y., Strobel, B., Ketterer, S., Rumpel, S., Park, J., Kreuz, S., Maier, U., Michelfelder, S., Schön, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a novel STARR-Seq-derived platform to screen for enhancers in the AAV context and demonstrate that the level of enhancer activity is highly dependent on the position in the genome.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Precise control of transgene expression through novel enhancer-promoter combinations is a promising strategy for advancing gene therapy mediated by adeno-associated virus (AAV). We present STARR-CRAAVT, a novel STARR-Seq-derived platform to screen for enhancers in the AAV context. Using HepG2 and HaCaT cells as screening models, we applied STARR-CRAAVT for the identification of cell type-specific enhancers. We integrated epigenetic datasets into an in-silico library of putative HepG2 enhancers and captured corresponding fragments from genomic DNA. The fragments were processed into AAV libraries and applied to HepG2 and HaCaT cells. STARR-CRAAVT analysis revealed a selective activity of the libraries confirming the HepG2-directed in-silico design and the specificity of single enhancer-promoter combinations could be validated using luciferase reporter assays. In addition, we scrutinized the impact of key experimental parameters on enhancer identification and found that the used promoter type had significant influence on the ability of candidates to act as enhancers. Furthermore, switching the location of identified enhancers in reporter assays revealed that the level of enhancer activity is highly dependent on the position in the AAV genome. Taken together, our study yields novel insights into enhancer function and demonstrates that STARR-CRAAVT can be employed to identify cell type-specific enhancers, highlighting promoter preference and enhancer positioning as key considerations for enhancer screening campaigns.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.04.657008v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Positional frequency chaos game representation for machine learning-based classification of crop lncRNAs</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Papastathopoulos-Katsaros, A., Liu, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose positional frequency chaos game representation (PFCGR), a novel encoding that improves the traditional frequency chaosGame representation (FCGR) by incorporating four statistical moments of k-mer positions: mean, standard deviation, skewness, and kurtosis, enabling machine learning models such as Log</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: Alignment-based methods are fundamental for sequence comparison but are often computationally prohibitive for large-scale genomic analyses. This limitation has spurred the development of quicker, alignment-free alternatives, such as k-mer analysis, which are crucial for studying long non-coding ribonucleic acids (lncRNAs) in plants. These lncRNAs play critical roles in regulating gene expression at both the epigenetic and transcriptomic levels. However, existing alignment-free approaches typically lose positional information, which can be vital for achieving accurate classification. Results: We propose positional frequency chaos game representation (PFCGR), a novel encoding that improves the traditional frequency chaos game representation (FCGR) by incorporating four statistical moments of k-mer positions: mean, standard deviation, skewness, and kurtosis. This creates a multi-channel image representation of genomic sequences, enabling machine learning models such as Logistic Regression, Random Forests, and Convolutional Neural Networks to classify plant lncRNAs directly from raw genomic sequences. Tested on seven major crop species, our PFCGR-based classifiers achieve classification accuracies comparable to or exceeding those of the computationally intensive DNABERT-based model, while requiring significantly less computational resources. These results demonstrate PFCGR's potential as an efficient and accurate tool for plant lncRNA identification, as well as its ability to facilitate large-scale computational studies in genomics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.03.657533v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Paired-Sample and Pathway-Anchored MLOps Framework for Robust Transcriptomic Machine Learning in Small Cohorts</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shabanian, M., Pouladi, N., Wilson, L., Prosperi, M., Lussier, Y. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel method for high-dimensional transcriptomic classification of infrequent diseases that leverages intra-subject dynamics, N-of-1 pathway-based analytics, and reproducible machine learning operations for continuous model refinement.</p><p style="font-size: 16px; text-align: justify;">Background. Ninety percent of the 65,000 human diseases are infrequent, collectively affecting about 400 million people, substantially limiting cohort accrual. This low prevalence constrains the development of robust transcriptome-based machine learning (ML) classifiers. Standard data-driven classifiers typically require cohorts of over 100 subjects per group to achieve clinical accuracy while managing high-dimensional input (about 25,000 transcripts). These requirements are infeasible for micro-cohorts of about 20 individuals, where overfitting becomes pervasive. Objective. To overcome these constraints, we developed a classification method that integrates three enabling strategies: (i) paired-sample transcriptome dynamics, (ii) N-of-1 pathway-based analytics, and (iii) reproducible machine learning operations (MLOps) for continuous model refinement. Methods. Unlike ML approaches relying on a single transcriptome per subject, within-subject paired-sample designs - such as pre- versus post-treatment or diseased versus adjacent-normal tissue - effectively control intra-individual variability under isogenic conditions and within-subject environmental exposures (e.g. smoking history, other medications, etc.), improve signal-to-noise ratios, and, when pre-processed as single-subject studies (N-of-1), can achieve statistical power comparable to that obtained in animal models. Pathway-level N-of-1 analytics further reduces each sample s high-dimensional profile into about 4,000 biologically interpretable features, annotated with effect sizes, dispersion, and significance. Complementary MLOps practices - automated versioning, continuous monitoring, and adaptive hyperparameter tuning - improve model reproducibility and generalization. Results. In two case studies - human rhinovirus infection versus matched healthy controls (n=16 training; 3 test) and breast cancer tissues harbouring TP53 or PIK3CA mutations versus adjacent normal tissue (n=27 training; 9 test) - this approach achieved 90% precision and recall on an unseen breast cancer test set and 92% precision with 90% recall in rhinovirus fivefold cross-validation. Incorporating paired-sample dynamics boosted precision by 8.8% and recall by 6%, while the MLOps workflow yielded additional gains of 14.5% and 12.5%, respectively. Moreover, our method identified 42 critical gene sets (pathways) for rhinovirus response and 21 for cancer mutation status. Conclusions. These proof-of-concept results support the utility of integrating intra-subject dynamics, biological knowledge -based feature reduction (pathway-level feature reduction grounded in prior biological knowledge; e.g., N-of-1-pathways analytics), and reproducible MLOps workflows can overcome cohort-size limitations in infrequent disease, offering a scalable, interpretable solution for high-dimensional transcriptomic classification. Future work will extend these advances across various therapeutic and small-cohort designs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.03.657721v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Single-cell transcriptomic integrated with machine learning reveals retinal cell-specific biomarkers in diabetic retinopathy</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> LIN, S., Yang, L., TAO, Y., PAN, Q., CAI, T., Ye, Y., Liu, J., Zhou, Y., Shao, Y., Yi, Q., Lu, Z. H., Chen, L., McKay, G., Rankin, R., Li, F., Meng, W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A high-resolution single-cell transcriptomic atlas of 297,121 retinal cells from 20 Chinese donors, including non-diabetic controls, diabetic without retinopathy and DR .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Diabetic retinopathy (DR) remains a principal cause of vision impairment worldwide, involved complex retinal cellular pathophysiology that remains incompletely understood. To elucidate cell-type-specific molecular signatures underlying DR, we generated a high-resolution single-cell transcriptomic atlas of 297,121 retinal cells from 20 Chinese donors, including non-diabetic controls (26.4%), diabetic without retinopathy (23.4%) and DR (50.2%). Following rigorous quality control, batch-effect correction, and clustering and annotation, 10 major retinal cell populations were delineated. Differential expression analyses across disease states within each cell type yielded candidate gene sets, which were further refined via a multi-stage machine-learning pipeline combining L1-regularized logistic regression and recursive feature elimination with cross-validation, alongside bootstrap stability selection. Resulting cell-type-specific classifiers achieved high accuracy (79-95%) and AUCs (0.85-0.99) in distinguishing DR disease states. Enrichment analyses implicated immune activation, oxidative stress, neurodegeneration and synaptic dysfunction pathways across multiple cell types in retina. Integrating 567 unique marker genes from all cell types, a general multilayer perceptron classifier achieved 95.31% overall accuracy on held-out test data, demonstrating the translational potential of these signatures for non-diabetic controls, diabetic without retinopathy and DR classification. This high-resolution atlas and the accompanying analytic framework provide a robust computational framework for biomarker discovery, mechanistic insight and targeted intervention strategies in diabetic retinal diseases.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.03.657753v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PreDSLpmo V2.0: A deep learning-based prediction tool for functional annotation of lytic polysaccharide monooxygenases</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Arulselvan, K. S., Pulavendran, P., Saravanan, V., Yennamalli, R. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present an improved machine learning framework designed for classifying and annotating LPMOs across the eight families (AA9, AA10, AA11, AA13, AA14, AA15, AA16, and AA17).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Lytic polysaccharide monooxygenases (LPMOs) are redox enzymes in the oxidative cleavage of recalcitrant polysaccharides, facilitating efficient biomass deconstruction for biofuel production. While computational annotation tools have accelerated LPMO discovery, the existing model is limited to the AA9 and AA10 families, leaving the broader diversity of LPMOs underexplored. In this study, we present an improved machine learning framework designed for classifying and annotating LPMOs across the eight families (AA9, AA10, AA11, AA13, AA14, AA15, AA16, and AA17). We curated a high-quality dataset by integrating sequences from GenBank, Joint Genome Institute, and the CAZy database, followed by data cleaning and redundancy reduction using CD HIT, consisting of 33691 LPMO sequences. Feature extraction was performed using both Python (iFeature) and R-based pipelines, generating over 13526 descriptors per sequence to capture compositional, physicochemical, and structural properties. Ensemble feature selection was used to identify the significant features for both binary and multiclass classification. An independent dataset was used to estimate the trained models performance. To address data imbalance, we maintained a 1:1 ratio for all positive and negative sets during both training and validation. A range of machine learning models were systematically trained and evaluated. The multiclass Bi-LSTM model demonstrated higher accuracy, robustness, and generalizability, outperforming traditional approaches. We compared the performance with dbCAN3, a CAZyme predictor. The final model was deployed as a user-friendly web server (https://predlpmo.in), enabling high-throughput, sequence-based functional annotation of LPMOs for the research community. This work bridges a critical gap in LPMO annotation, providing a scalable and reliable solution for enzyme discovery in bioenergy and industrial biotechnology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.05.658061v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comparing phenotypic manifolds with Kompot: Detecting differential abundance and gene expression at single-cell resolution</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Otto, D. J., Arriaga-Gomez, E., Thieme, E., Yang, R., Lee, S. C., Setty, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A statistical framework for holistic comparison of multi-condition single-cell datasets, supporting both differential abundance and differential expression.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Kompot is a statistical framework for holistic comparison of multi-condition single-cell datasets, supporting both differential abundance and differential expression. Differential abundance captures changes in how cells populate the phenotypic manifold across conditions, while differential expression identifies condition-specific changes in gene regulation that may be localized to particular regions of that manifold. Kompot models the distribution of cells and gene expression as continuous functions over a low-dimensional representation of cell states, enabling single-cell resolution inference with calibrated uncertainty estimates. Applying Kompot to aging murine bone marrow, we identified a continuum of shifts in hematopoietic stem cell and mature cell states, transcriptional remodeling of monocytes independent of compositional changes, and divergent regulation of oxidative stress response genes across cell types. By capturing both global and cell-state specific effects of perturbation, Kompot reveals how aging reshapes cellular identity and regulatory programs across the hematopoietic landscape. This framework is broadly applicable to dissecting condition-specific effects in complex single-cell landscapes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.03.657769v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Maize root growth, Oxygen and N availability drive formation of N2O hotspots in soil</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rummel, P. S., Rasmussen, M. R., Saghai, A., Merl, T., Hallin, S., Mueller, C. W., Koren, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> In-situ measurements of N2O concentrations in the rhizosphere and O2 in the soil .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Plant roots modify all major controls of denitrification in soils, particularly the availability of the main substrates (NO3- and Corg), soil moisture, soil O2 content, and root-associated microbial communities, and thus play an important role in N2O formation. Direct in-situ measurements of N2O concentrations in the rhizosphere are lacking, yet crucial to understanding how rhizosphere denitrification contributes to overall N2O emissions from soil. We equipped rhizoboxes with O2-sensitive planar optodes to simultaneously monitor root growth and rhizosphere/soil O2 concentrations. We measured soil surface N2O fluxes and linked them to root growth, soil moisture, and root/soil O2 concentrations. Based on root growth and O2 concentrations, we identified regions of interest (ROI) and sampled small soil volumes, which were analyzed for C, N, abundance of microbial denitrifiers (nirK, nirS) and N2O reducers (nosZI, nosZII), and soil N2O concentrations. Plant roots determined depth gradients of nutrients and denitrification gene abundances in the soil of the rhizoboxes with higher resource availability (NO3-, DOC) and lower soil moisture in the upper soil layers, which also had higher abundances of total bacteria, nirK and nosZII. We anticipate that these uppermost soil layers largely contributed to N2O formation. For the first time we were able to show high in-situ N2O concentrations with distinct depth profiles around roots, and O2 and N availability controlling N2O production at the process scale.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.03.657623v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Single-cell characterization of bacterial optogenetic Cre recombinases</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Huang, H., Jafarbeglou, F., Dunlop, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Optogenetic recombinase performance can be reporter-dependent, suggesting that this is an important consideration in system design.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Microbial optogenetic tools can regulate gene expression with high spatial and temporal precision, offering excellent potential for single-cell resolution studies. However, bacterial optogenetic systems have primarily been deployed for population-level experiments. It is not always clear how these tools perform in single cells, where stochastic effects can be substantial. In this study, we focus on optogenetic Cre recombinase and systematically compare the performance of three variants (OptoCre-REDMAP, OptoCre-Vvd, and PA-Cre) for their population-level and single-cell activity. We quantify recombination efficiency, expression variability, and activation dynamics using reporters which produce changes in fluorescence or antibiotic resistance following light-induced Cre activity. Our results indicate that optogenetic recombinase performance can be reporter-dependent, suggesting that this is an important consideration in system design. Further, our single-cell analysis reveals highly heterogeneous activity across cells. Although general trends match expectations for mean levels of light-dependent recombination, we found substantial variation in this behavior across individual cells. In addition, our results show that the timing of recombinase activity is highly variable from cell to cell. These findings suggest critical criteria for selecting appropriate optogenetic recombinase systems and indicate areas for optimization to improve the single-cell capabilities of bacterial optogenetic tools.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.06.658346v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Engineering lung-sensing T cells using synthetic receptors targeting RAGE</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jacob, A., Whitty, C., Lupica, G., Badwal, E., Ren, X., Qiu, W., Yu, W., Tonai, Y., Camara Serrano, J. A., Simic, M., Lim, W. A., Allen, G., Sheppard, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that RAGE is a lung-specific target and that T cells expressing anti-RAGE receptors can sense and activate lung- specific therapeutic transcriptional programs.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Our goal was to leverage synthetic biology approaches to engineer lung-sensing T cells that trigger synthetic transcriptional programs only when in the lung. First, we identified a lung-specific cell surface protein, receptor for advanced glycation endproducts (RAGE), expressed at high levels exclusively in the lung. Then, we engineered chimeric antigen receptors (CAR) and synthetic notch receptors (SynNotch) to bind this protein. We showed that anti-RAGE CAR-T cells traffic to and proliferate in the lung exclusively. Anti-RAGE SynNotch receptors activate transcription of a fluorescent reporter only when co-cultured with RAGE+ cell lines or primary lung cells. Finally, we tested an anti-RAGE SynNotch to anti-CD19 CAR circuit in vivo in mice implanted with lung and flank tumors and found that this approach cleared CD19-expressing lung tumors without affecting CD19-expressing flank tumors. Thus, we demonstrate that RAGE is a lung-specific target and that T cells expressing anti-RAGE receptors can sense and activate lung-specific therapeutic transcriptional programs. This approach could be extended to allow cell-based therapies that target and deliver genetically encoded payloads to treat lung diseases while avoiding systemic toxicity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.06.658315v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Examining the Role of Extrachromosomal DNA in 1,216 Lung Cancers</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Khandekar, A., Hoang, P. H., Luebeck, J., Diaz-Gay, M., Zhao, W., McElderry, J. P., Hartman, C., Miraftab, M., Lee, O. W., Barnao, K. M., Bergstrom, E. N., Yang, Y., Nowak, M. A., Rothman, N., Homer, R., Yang, S.-R., Lan, Q., Wedge, D. C., Yang, L., Chanock, S. J., Zhang, T., Alexandrov, L. B., Landi, M. T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that whole-genome-sequenced lung cancers harboring ecDNA or other focal amplifications are associated with poor survival, and suggest that this instability reflects genomic instability in treatment-naive lung cancer.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The role of extrachromosomal DNA (ecDNA) in lung cancer, particularly in subjects who never smoked (LCINS), remains unclear. Examination of 1,216 whole-genome-sequenced lung cancers identified ecDNA in 18.9% of patients. Enrichment of MDM2 and other oncogenes amplification via ecDNA possibly drives a LCINS subset. Tumors harboring ecDNA or other focal amplifications showed similarly poor survival. A strong association with whole-genome doubling suggests most ecDNA reflects genomic instability in treatment-naive lung cancer.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.03.657117v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Ion exchange- and enrichment-based technology applied to large-scale plasma proteomic analysis of breast cancer neoadjuvant chemotherapy.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xiaozhen Cui, Zhihua Zhong, Sen Xu, Yini Pan, Xi Wang, Luobin Zhang, An He, Xueting Ye, Hua Cao, Weibing Zhang, Ruijun Tian</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-03-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a plasma proteomics workflow based on two kinds of magnetic metal-organic frameworks modified with ion exchange functional groups (denoted as MHP-UiO-66-SAX and MHP -HKUST-1-SCX) and a Liquid-handling System, which</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mass spectrometry (MS) based proteomics provides unbiased quantification of all proteins in plasma, which can dynamically reflect individual health states in real time. However, large-scale proteomics studies are constrained by the excessive dynamic range of plasma proteome and low throughput. Herein, two kinds of magnetic metal-organic frameworks (MOFs) modified with ion exchange functional groups (denoted as MHP-UiO-66-SAX and MHP-HKUST-1-SCX) were designed and fabricated to exhibit large protein adsorption capability, which were combined with an automated Liquid-handling System, thus realizing in-depth, high-throughput and automated proteomics studies. The constructed workflow could automatically complete the sample preparation before MS within only six hours and nearly a thousand protein groups per sample could be quantified. In the cohort study of nearly one hundred breast cancer neoadjuvant chemotherapy (NC) plasma samples, two differentially expressed proteins previously reported as biomarkers were related with the pathological complete response (PCR) of the breast cancer, demonstrating the feasibility of the developed technology for preparing large-scale clinical samples and exhibiting the potential application in monitoring the effect of chemotherapy.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40188783/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

