<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Friday, 15 August 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Predicting Experimental Success in De Novo Binder Design: A Meta-Analysis of 3,766 Experimentally Characterised Binders</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Overath, M. D., Rygaard, A., Jacobsen, C. P., Brasas, V., Morell, O., Sormanni, P., Jenkins, T. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that interface-focused metrics, most notably the AF3-derived interaction prediction Score from Aligned Errors (ipSAE) outperform commonly used scores such as ipAE and ipTM, with a significant 1.4-fold increase in average precision compared to ipAE.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Designing high-affinity de novo protein binders has become increasingly tractable, yet in vitro prioritisation continues to depend on heuristics in the absence of systematic analysis. Here, we present a large-scale meta-analysis of 3,766 experimentally tested binders across 15 structurally diverse targets. Using a unified, high-throughput pipeline that re-predicts each binder-target complex with AF2 (initial guess and ColabFold), AF3 and Boltz-1, we extract over 200 structural, energetic and confidence features per design. We show that interface-focused metrics, most notably the AF3-derived interaction prediction Score from Aligned Errors (ipSAE) outperform commonly used scores such as ipAE and ipTM, with a significant 1.4-fold increase in average precision compared to ipAE. We further show that combining these metrics with orthogonal physicochemical interface descriptors, including Rosetta {Delta}G/{Delta}SASA and interface shape complementarity, improves predictive performance. While overall performance varies by target, simple linear models trained on a small number of AF3-derived features generalize well across datasets. We propose interpretable, target-agnostic filtering strategies, such as combining AF3 ipSAE_min rankings with structural filters, to improve precision in selecting binders for testing. Finally, we release the complete dataset establishing a community resource to benchmark and accelerate de novo binder discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.14.670059v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Evolutionary and methodological considerations when interpreting gene presence-absence variation in pangenomes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bruna, T., Sreedasyam, A., Harder, A. M., Lovell, J. T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that pangenome gene model annotations can be highly error prone, and show that sequence evolution and presence-absence variation are the most important factors in genome annotation and pangene set inconsistency.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> While graph-based pangenomes have become a standard and interoperable foundation for comparisons across multiple reference genomes, integrating protein-coding gene annotations across pangenomes in a single 'pangene set' remains challenging, both because of methodological inconsistency and biological presence-absence variation (PAV). Here, we review and experimentally evaluate the root of genome annotation and pangene set inconsistency using two polyploid plant pangenomes: cotton and soybean, which were chosen because of their existing diverse high-quality genomic resources and the known importance of gene presence-absence variation in their respective breeding programs. We first demonstrate that building pangene sets across different genome resources is highly error prone: PAV calculated directly from the genome annotations hosted on public repositories recapitulates structure in annotation methods and not biological sequence differences. Re-annotation of all genomes with a single identical pipeline largely resolves the broadest stroke issues; however, substantial challenges remain, including a surprisingly common case where exactly identical sequences have different gene model structural annotations. Combined, these results clearly show that pangenome gene model annotations must be carefully integrated before any biological inference can be made regarding sequence evolution, gene copy-number, or presence-absence variation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.14.670405v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identification of Prospective PETases Across Prokaryotes Using an in silico Approach</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rathod, Y., Biswas, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comprehensive profiling of prokaryotic sequences has been undertaken to look for the presence of similar plastic-degradation abilities across the kingdom.</p><p style="font-size: 16px; text-align: justify;">Background: Plastics count as one of the most potent threats to the habitats and survival of global flora and fauna. Reports keep accumulating globally about the ever-exploding load of plastic wastes, but the need and economics of multiple industrial and household processes compels the production of more plastic materials. It has always been imperative to look for natural sources of degradation of plastic. The identification of plastic-degrading microbes, therefore remains a major focus of the microbial fraternity. While the discoveries of Ideonella sakaiensis or later, Rhizobacter gummiphilus were more out of providence, the structure determination of the enzyme responsible for PET degradation does provide a fillip to efforts towards identification of more such prokaryotic entities. Results: In this work, a comprehensive profiling of prokaryotic sequences has been undertaken to look for the presence of similar plastic-degradation abilities across the kingdom. The identification of twenty-seven such hits across different bacterial species led us to believe in the natural diversity of plastic-degradation enzymes. Moreover, there seems to be conservation of the structural motif that renders such ability as has been observed from the constructed models and analysis of their interfaces. Docking of BHET, one of the key products of PET, against these 27 entities showed considerable interactions with the above and pointed towards the possible roles of these bacteria as natural plastic degradation models. Eight of these proteins have very close similarity in binding interactions and surface properties to the PETase from I. sakaiensis and were shortlisted as prospective candidates. Conclusions: Of these eight, three PETases from Halopseudomonas pertucinogena Halopseudomonas bauzanensis and Ketobacter sp. revealed significant similarity in structure and conformational stability to the PETase from I. sakaiensis as was evident from the analysis of their molecular dynamics parameters. Principal Component Analysis and the free energy landscape during binding to BHET also validated the hypothesis and these three PETases could be immediately explored for possible plastic degradation activity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.14.670228v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The Therapeutic Nanobody Profiler: characterising and predicting nanobody developability to improve therapeutic design</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gordon, G. L., Gervasio, J., Souders, C., Deane, C. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Developability optimisation is an important step for successful biotherapeutic design, and we have developed an open-source computational tool for predicting nanobody developability.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Developability optimisation is an important step for successful biotherapeutic design. For monoclonal antibodies, developability is relatively well characterised. However, progress for novel biotherapeutics such as nanobodies is more limited. Differences in structural features between antibodies and nanobodies render current antibody computational methods unsuitable for direct application to nanobodies. Following the principles of the Therapeutic Antibody Profiler (TAP), we have built the Therapeutic Nanobody Profiler (TNP), an open-source computational tool for predicting nanobody developability. Tailored specifically for nanobodies, it accounts for their unique properties compared to conventional antibodies for more efficient development of this novel therapeutic format. We calibrate TNP metrics using the 36 currently available clinical-stage nanobody sequences. We also collected experimental developability data for 108 nanobodies and examine how these results are related to the TNP guidelines. TNP is available as a web application at opig.stats.ox.ac.uk/webapps/tnp.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.11.669635v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Deciphering chromatin architecture and dynamics in Plasmodium falciparum using the nucDetective pipeline</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Holzinger, S., Schmutterer, L., Rothe, V. M., Watzlowik, M. T., Schwartz, U., Laengst, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We demonstrate that the chromatin structure of the malaria-causing parasite Plasmodium falciparum is highly regular, with a unique chromatin landscape, with stage-specific structural alterations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> High-resolution analysis of cellular chromatin structure is crucial for uncovering developmental and cell-type-specific regulatory networks. We developed the nucDetective pipeline to provide a comprehensive evaluation of chromatin organisation. This involves assessing nucleosome positioning, occupancy, fuzziness, and array regularity. The pipeline was benchmarked by analysing the chromatin structure of the malaria-causing parasite Plasmodium falciparum (Pf) during its erythrocytic development cycle. Pf is characterised by a unique chromatin landscape, exhibiting unstable nucleosomes and a genomic AT-content exceeding 80%, which presents challenges for standard MNase-seq analysis of chromatin. The nucDetective pipeline provides specific, high-resolution nucleosome profiles for the different asexual stages of Pf, monitoring the dynamics of individual nucleosomes. In contrast to the current view, suggesting an irregular chromatin structure and lack of nucleosomal DNA, we demonstrate that the transcription start sites exhibit typical eukaryotic features, including +1 nucleosomes, nucleosome-free regions upstream, and phased nucleosome arrays downstream of the TSSs. The global mean nucleosome repeat length varies from 176 bp to 185 bp depending on the developmental stage. Stage specific changes in nucleosome positioning occur locally in intergenic regulatory regions, which are characterized by specific histone modifications and variants. Dynamic nucleosomes correlate with DNA accessibility, gene expression and determine the access to transcription factor binding sites in Pf. The highly regular chromatin structure, with stage-specific structural alterations, emphasises the important role of epigenetic mechanisms in regulating the complex life cycles of Pf.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.13.670116v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">XenoSignal: Investigating Intra- and Inter-Species Ligand-Receptor Interactions Using AlphaFold3</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Abdelkareem, A. O., Hall, C. F., Marutharaj, A., Narta, K., Verhey, T. B., Morrissy, A. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a computational framework that integrates AlphaFold3 structural predictions to assess the plausibility of intraspecies and interspecies LR interactions and develop a high-confidence cross-species LR resource.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Xenograft models, in which human tumors are implanted into mouse hosts, offer a unique platform to study tumor-microenvironment (TME) interactions. Ligand-receptor (LR) interactions are central to intercellular communication in this context, but most inference tools rely on single-species databases and assume conserved function across orthologues. To test whether this assumption holds given sequence and structural divergence, we developed XenoSignal, a computational framework that integrates AlphaFold3 structural predictions to assess the plausibility of intraspecies and interspecies LR interactions. The pipeline predicts whether human-mouse orthologues retain or lose their functional interaction, enabling construction of a high-confidence cross-species LR resource. XenoSignal facilitates the interpretation of tumor-TME crosstalk in xenografts and prioritizes LR pairs for experimental validation. This resource enhances our ability to dissect interspecies signaling, uncover mechanisms of immune evasion, and identify candidate therapeutic targets in translational cancer models.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.13.670200v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Single-molecule landscape of DNA replication pausing</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Thiyagarajan, S., Rogers, A. M., Muller, C. A., Nieduszynski, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify single-molecule sites of replisome perturbation across a eukaryotic genome using long-read nanopore sequencing and show that replication pauses can be the molecular trigger behind epigenetic switching.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> A pause in DNA synthesis that occurs when the replisome encounters an obstacle could lead to genome instability. Although important, systematic identification of replication pause sites is challenging due to their low frequency and delocalized nature. Here we present the first single-molecule identification of sites of replisome perturbation across a eukaryotic genome using long-read nanopore sequencing. For each single-molecule replication pause we determine the direction of replication, leading/lagging-strand identity, location and approximate duration, and whether the replisome resumed synthesis. Although pauses are largely diffuse over the genome, they are significantly enriched over transcribed features and correlate with transcription and R-loop levels. Transcription-replication conflicts are more numerous when head-on than co-directional. Finally, we identified genomic loci with a strong bias towards leading over lagging strand pauses, consistent with uncoupling of the helicase from polymerase epsilon. Our data support helicase-polymerase uncoupling resulting from replication pausing as the molecular trigger behind epigenetic switching.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.14.670160v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Temporal control of sgRNA library activation unlocks large-scale in vivo CRISPR screens</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fenoglio, S., Yu, Y., Tepper, J., Grove, L., Bejnood, A., Meier, S. R., Choi, A. H., Wu, H.-J., Devault, A., Liu, S., Shen, B., Khendu, T., Stowe, H., Uijttewaal, E. C., Zhang, M., Haines, B. B., Wilker, E., Huang, A., Schramek, D., Elling, U., Pan, X., Andersen, J. N., Teng, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel inducible CRISPR-StAR screening platform for large-scale in vivo functional genomics screens that reduces animal use by up to 7-fold compared to conventional in vivo dropout screens.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Functional genomics screens have illuminated genetic dependencies in cancer, but conventional in vitro approaches fail to capture vulnerabilities shaped by the tumor microenvironment. Here, we implement CRISPR-StAR (Stochastic Activation by Recombination), a next-generation inducible CRISPR screening platform for large-scale in vivo applications. The system uses a dual lox-based recombination system to enable guide-level normalization and clonal knockout phenotyping. To analyze the rich (barcode-embedded) sequencing output, we developed UMIBB, a superior Bayesian statistical framework for quantifying gene-level dropout and enrichment compared to conventional software packages. Screening a 30,000-sgRNA library in A549 xenografts, followed by clone representation and dropout correlation analyses, showed high fidelity and reproducibility with dropout phenotypes resolvable using as few as 30 tumors for this size library. Validation across multiple tumor models demonstrated that a single tumor can provide reliable, functional annotation for ~1,000 genes leveraging intra-tumor library controls for normalization. Comparing in vivo and in vitro screens revealed that a substantial subset of tumor suppressor genes exerts strong phenotypic effects only observable in vivo. For example, single-gene knockout and transcriptomic profiling confirmed that KMT2C and KMT2D have contrasting impacts on tumor growth - an insight that would have been overlooked in standard cell culture. Looking ahead, CRISPR-StAR screening, combined with our user-friendly analysis pipeline available on GitHub (R-package), offer an integrated framework for creating in vivo dependency maps that can complement existing vitro datasets like DepMap and Achilles. Critically, our approach reduces animal use by up to 7-fold compared to conventional in vivo dropout screens. This represents a significant ethical and methodological advancement - achieving genome-scale resolution with far fewer animals and greater reproducibility.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.14.670372v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Insights gained into Vascular Streak Dieback, an Emerging Disease of Woody Ornamentals in the United States, using Metagenomic Sequencing</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Belay, K., Abdelrazek, S., Bily, D., Gyatso, T., Avin, F. A., Bonkowski, J., Liyanapathiranage, P., Salamanca, L. R., Heath, L. S., Baysal-Gurel, F., Vinatzer, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a single genetic lineage of vascular streak dieback in the United States from Southeast Asia that is characterized by a wide host range and that quickly spread geographically.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> [bullet] Woody ornamentals are an integral part of home gardens and urban landscapes and are key to the economic vitality of the nursery industry. Movement of plants infected with emerging pathogens can quickly lead to wide geographic spread of new diseases impacting ecosystem health and leading to economic losses. [bullet] Using metagenomic sequencing, we investigated the microbial species associated with vascular streak dieback (VSD), an emerging disease of woody ornamentals in the United States, analyzing 106 samples obtained from 34 different species in seven states. [bullet] The fastidious fungal pathogen Ceratobasidium sp. (Csp), previously found to be associated with VSD, was the only putative pathogen consistently identified. Assembly of 17 high quality draft genomes followed by phylogenetic and pangenome analyses revealed Csp to form a tight genetic cluster closely related to, yet distinct in gene content, from genomes of Ceratobasidium theobromae, a pathogen of cacao, avocado, and cassava in Southeast Asia. [bullet] These results reveal the unparalleled ability of metagenomics for identification and characterization of fastidious fungal pathogens and suggest the recent introduction of a single genetic lineage of Csp into the U.S., possibly from Southeast Asia, that is characterized by a wide host range and that quickly spread geographically.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.11.669776v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Elucidating design principles for Ribozyme-Enabled Tissue Specificity (RETS) to allow precise expression without specialized promoters.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Combest, M. M., Conlin, J., Van De Mark, V., Boisen, M., Colwell, H., Khakhar, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We develop a novel ribozyme-based method for tissue-specific and dose-dependent expression of transgenes in Arabidopsis thaliana and demonstrate its utility for spatiotemporal control of transgene expression.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Tissue- or cell type- specific expression of transgenes is often essential for interrogation of biological phenomenon or predictable engineering of multicellular organisms but can be stymied by cryptic enhancers that make identification of promoters that generate desired expression profiles challenging. In plants the months-to-years long timeline associated with prototyping putative tissue-specific promoters in transgenic lines deepens this challenge. We have developed a novel strategy called Ribozyme Enabled Tissue Specificity (RETS) that leverages the knowledge of where and when genes are expressed derived from transcriptomic studies to enable tissue specific expression without needing characterized promoters. It uses a split self-splicing ribozyme based on a group I intron from Tetrahymena thermophila to enable conditional reconstitution of a transgene mRNA in the presence of a secondary tissue-specific mRNA of choice. We elucidate the design features that enable flexible swapping of transgenes and targets, enhancing transgene expression, and circumventing host RNA interference responses. We then show that these innovations enable tissue-specific and dose-dependent expression of transgenes in Arabidopsis thaliana. Finally, we demonstrate the utility of RETS both for creating genetically encoded biosensors to study the spatiotemporal patterns of gene expression in planta as well as for engineering tissue-specific changes in organ size. RETS provides a novel avenue to study expression patterns of native loci with non-destructive imaging, complementing the weakness of existing approaches. Additionally, the spatiotemporal control of transgene expression afforded by RETS enables precision engineering of plant phenotypes which will facilitate enhancing crops without the trade-offs associated with constitutive expression.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.14.670357v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The melanoma MEGA-study: Integrating proteogenomics, digital pathology, and AI-analytics for precision oncology.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jessica Guedes, Leticia Szadai, Nicole Woldmar, Ágnes Judit Jánosi, Klára Koroncziová, Blanka Míra Lengyel, Bella Kelemen, Eszter Boltas, Rolland Gyulai, Elisabet Wieslander, Krzysztof Pawłowski, Peter Horvatovich, Lazaro Betancourt, A Marcell Szasz, Zoltan Vereb, Peter Horvath, Henriett Oskolás, Roger Appelqvist, Johan Malm, Gyorgy Marko-Varga, István Balázs Németh, Jeovanis Gil</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A multi-center initiative to unravel the complex molecular landscape of melanoma by integrating advanced proteogenomic profiling, AI-driven digital pathology and machine learning analytics.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Melanoma remains the most aggressive form of skin cancer, characterized by high metastatic potential, genetic heterogeneity, and resistance to conventional therapies. The Melanoma MEGA-Study is a multi-center initiative designed to address these clinical challenges by integrating advanced proteogenomic profiling, clinical metadata, with AI-driven digital pathology and machine learning analytics, aiming to enhance personalized treatment strategies and improve patient outcomes. Between 2013 and 2022, a cohort of 1653 melanoma patients each contributed a primary tumor sample, with 361 providing 819 metastatic tumor samples. Clinical data collection for this cohort continued until May 2023. Comprehensive analyses using high-resolution mass spectrometry, optimized workflows for formalin-fixed paraffin-embedded tissues, and advanced digital pathology platforms enabled precise mapping of the tumor microenvironment, identification of metabolic reprogramming, and characterization of immune evasion signatures. The European Cancer Moonshot Lund Center's MEGA-Study, under the academic umbrella of Lund and Szeged universities, marks a significant advancement in its collaborative efforts with the National Institutes of Health (NIH) under the Cancer Moonshot partnership. This initiative exemplifies the center's dedication to pioneering cancer research and underscores the strength of its international collaborations. SIGNIFICANCE: The significance of this study lies in its pioneering integration of high-resolution proteomics, AI-driven digital pathology, and comprehensive clinical annotation to unravel the complex molecular landscape of melanoma. By leveraging a robust, population-based cohort of 1653 patients, including extensive analyses of both primary and metastatic tumor specimens, our approach provides unprecedented insights into the proteogenomic alterations that underpin tumor progression, immune evasion, and therapeutic resistance. The preliminary application of advanced mass spectrometry techniques to formalin-fixed paraffin-embedded tissues, combined with state-of-the-art digital pathology and machine learning, has enabled the identification of novel protein biomarkers and metabolic signatures that hold promise for refining patient stratification and informing personalized treatment strategies. This integrative framework not only deepens our understanding of melanoma biology but also establishes a scalable model for precision oncology that can be extended to other complex malignancies. Ultimately, our findings have the potential to transform clinical practice by facilitating earlier risk stratification, improving prognostication, and guiding the development of targeted therapeutic interventions for this highly aggressive cancer.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40532957/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Emerging blood biomarkers in Alzheimer's disease: a proteomic perspective.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Neeraj Patel, Neetu Agrawal, Rakhi Mishra, M M Rekha, Priya Priyadarshini Nayak, Mandeep Kaur, Anil Khachi, Kavita Goyal, A Rekha, Mohit Rana, Ghala Alnuaimi, Rashi Kulshrestha</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comprehensive synthesis comparing mass-spectrometry and ultrasensitive immunoassay platforms across multiple blood-based AD biomarkers and the first to integrate these findings into a unified roadmap for clinical implementation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Early detection of Alzheimer's disease (AD) remains a formidable clinical challenge, but emerging blood-based assays show promise for identifying at-risk individuals long before cognitive symptoms arise. This is the first comprehensive synthesis comparing mass-spectrometry and immunoassay platforms across multiple blood-based AD biomarkers and the first to integrate these findings into a unified roadmap for clinical implementation. In this review, we compare high-throughput mass spectrometry and ultrasensitive immunoassays for quantifying circulating amyloid-β isoforms, phosphorylated tau species (p-tau181, p-tau217), neurofilament light chain (NfL), glial fibrillary acidic protein (GFAP), YKL-40 and selected inflammatory markers. Individual biomarkers demonstrate diagnostic accuracies (AUC) up to 0.90, and integrating these protein signatures with APOE ε4 genotype, brief cognitive assessments and neuroimaging via machine-learning models boosts discrimination of preclinical AD from normal aging to over 80% accuracy. We trace the path from initial discovery through analytical validation to clinical implementation, emphasizing critical hurdles such as variability in sample collection, limited cohort diversity and regulatory requirements. Future work must standardize preanalytical protocols, extend validation across populations, refine ultrasensitive detection techniques, and combine proteomic data with genomics and other "omics" layers to move toward routine blood-based screening. These coordinated efforts provide a clear roadmap for transforming early AD diagnosis and enabling timely, personalized interventions.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40441344/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

