<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Wednesday, 10 September 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrated Multi-Omics Analysis for Molecular Subtyping in NSCLC: A Cohort Study</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yan, B., Wang, Z., Peng, Y., Liu, K., Zhou, J.-G., Wang, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Lung cancer remains the leading cause of cancer-related morbidity and mortality worldwide, with non-small cell lung cancer (NSCLC) accounting for approximately 85% of cases. Current histopathological classification and driver gene testing provide limited prognostic and therapeutic guidance due to intra-tumoral heterogeneity and incomplete characterization of the tumor microenvironment (TME). Here, we constructed single- and multi-omics molecular classification systems for NSCLC by integrating transcriptomic, genomic, epigenomic, proteomic, and TME data from TCGA LUAD and LUSC cohorts. Single-omics analyses revealed distinct molecular patterns, including DNA methylation subtypes associated with sex and histology. Multi-omics integration identified five consensus subtypes closely corresponding to histology, with three immune-activated (CS2-CS4) and two immunosuppressive (CS1, CS5) subtypes. While overall survival did not differ significantly, progression-free survival analysis highlighted CS4 as a high-risk subtype with KRAS-driven genomic alterations. These findings provide a framework for NSCLC molecular stratification and highlight molecular and immune features that could guide future research on targeted therapies and immunotherapy.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.05.674586v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Computational pathology annotation enhances the resolution and interpretation of breast cancer spatial transcriptomics data.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tianyi Li, Qiao Yang, Balazs Acs, Emmanouil G Sifakis, Hosein Toosi, Camilla Engblom, Kim Thrane, Qirong Lin, Jeff E Mold, Wenwen Sun, Ceren Boyaci, Sanna Steen, Jonas Frisén, Jens Lagergren, Joakim Lundeberg, Xinsong Chen, Johan Hartman</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Breast cancer is a highly heterogeneous disease with diverse outcomes, and intra-tumoral heterogeneity plays a significant role in both diagnosis and treatment. Despite its importance, the spatial distribution of intra-tumoral heterogeneity is not fully elucidated. Spatial transcriptomics has emerged as a promising tool to study the molecular mechanisms behind many diseases. It offers accurate measurements of RNA abundance, providing powerful tools to correlate the morphologies of cellular neighborhoods with localized gene expression patterns. However, the spot-based spatial transcriptomic tools, including the most widely used platform, Visium, do not achieve single-cell resolution readouts, which hinders data interpretability. In this study, we present a computational pathology image analysis pipeline (i.e., computational tissue annotation, CTA) that utilizes machine learning algorithms to accurately map tumor, stroma, and immune compartments within Visium-assayed tumor sections. Using a cohort of 23 breast tumor sections from four patients, we demonstrate that CTA can provide high-resolution annotations on the hematoxylin-and-eosin-stained images alongside the paired sequencing data, support the evaluation of deconvolution methods, deepen insights into intra-tumoral heterogeneity by increasing data analysis resolution, assist with spatially resolved intrinsic subtyping, and enhance the visualization of lymphocyte clones at single-cell resolution. The proposed pipeline provides valuable insights into the complex spatial architecture of breast cancer, contributing to more personalized diagnostics and treatment strategies.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40925915/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Recent advances in research on gene polymorphisms in Kawasaki disease.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yang ZY, Pan Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Kawasaki disease (KD) is a systemic vasculitis primarily affecting children, and represents a major cause of acquired heart disease in this population. Although the etiology of KD remains incompletely understood, existing genome-wide association studies and genome-wide linkage studies have uncovered various susceptibility genes and their associated chromosomal regions as closely related to the onset and progression of KD. With the rapid advancement of high-throughput DNA sequencing technology, an increasing amount of genomic information pertinent to KD has been discovered, offering new perspectives to investigate the pathogenesis of KD. In particular, genetic polymorphisms play a pivotal role in the immune response, coronary artery lesions, and treatment responsiveness in KD, providing fresh insights into optimizing diagnostic and therapeutic strategies. This article aimed to review and summarize the crucial role of genetic polymorphisms in the pathogenesis of KD, analyze the latest advancements in current research, and discuss the potential applications of gene polymorphism studies in the future diagnosis and treatment of KD.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40881086/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Exploring the Frontiers of Computational NMR: Methods, Applications, and Challenges.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Susanta Das, Kenneth M Merz</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Computational methods have revolutionized NMR spectroscopy, driving significant advancements in structural biology and related fields. This review focuses on recent developments in quantum chemical and machine learning approaches for computational NMR, emphasizing their role in enhancing accuracy, efficiency, and scalability. QM methods provide precise predictions of NMR parameters, enabling detailed structural characterization of diverse systems. ML techniques, leveraging extensive data sets and advanced algorithms, complement QM by efficiently automating spectral assignments, predicting chemical shifts, and analyzing complex data. Together, these approaches have transformed NMR workflows, addressing challenges in metabolomics, protein structure determination, and drug discovery. This review highlights recent progress, emerging tools, and future directions in computational NMR, underscoring its critical role in modern structural science.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40923454/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Accurate characterization of CRISPR-Cas9 genome editing outcomes and mosaicism with near-perfect long reads</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Höijer, I., van Schendel, R., Emmanouilidou, A., Östlund, R., Bunikis, I., Tijsterman, M., den Hoed, M., Ameur, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;">Background: Genetic mosaicism is a consequence of CRISPR-Cas9 genome editing that is difficult to study, especially when it involves structural variants occurring at low frequency. A comprehensive analysis of mosaicism requires deep and unbiased sequencing of the target loci, with accurate single molecule reads. Here we performed amplification-free PureTarget PacBio sequencing to investigate CRISPR-Cas9 outcomes at on-target and off-target sites in germline edited zebrafish and their offspring. Results: Thirty samples from pooled larvae and individual zebrafish were successfully sequenced, resulting in >1100x average target coverage. The PacBio reads reached an exceptional accuracy (QV39) over the target regions, with every read originating from a unique DNA molecule. The two haplotypes of the target loci displayed a balanced depth of coverage, while long-range PCR of the same samples resulted in skewed data. Further analysis of the PureTarget data revealed widespread genetic mosaicism in individual founder (F0) fish, with up to 18 distinct on-target events and 11 off-target events present in a single adult founder. Several CRISPR-Cas9 editing outcomes, including large structural variants and off-target mutations, were inherited to the F1 generation. Notably, as many as seven unique editing events were found among F1 juvenile offspring from a single founder pair, thereby confirming the presence of genetic mosaicism in germ cells of founder zebrafish. This implies that some consequences of CRISPR-Cas9 editing may emerge only in the second generation. We also analyzed DNA methylation but did not observe altered 5mC CpG signals in genome edited samples. Conclusions: PureTarget enables efficient, accurate and unbiased analysis of the full spectrum of genetic mosaicism in a sample. The potential risks of introducing germ cell mosaicism in founder individuals should be carefully evaluated when designing germline genome editing experiments.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.08.674810v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Empirical optimization of dual-sgRNA design for in vivo CRISPR/Cas9-mediated exon deletion in mice</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lee, S.-Y., Ma, S., Jeon, S. D., Kim, H., Jo, B., Han, S.-H., Jang, E., Lee, J., Lee, Y.-K., Lee, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> CRISPR/Cas9 has transformed gene editing, enabling precise genetic modifications across species. However, existing sgRNA design prediction models based on in vitro data are difficult to generalize to in vivo contexts. In particular, approaches based on single-sgRNA design require additional filtering of in-frame mutations, which is inefficient in terms of both time and cost. In this study, we developed the first mammalian in vivo-trained prediction model to evaluate the efficiency of a dual-sgRNA-based exon deletion strategy. Using 230 editing outcomes of postnatal viable individuals, eight prediction models were constructed and evaluated based on generalized linear models and random forests. The final selected model, a Combined GLM, integrated the DeepSpCas9 score with k-mer sequence features, achieving an AUC of 0.759 (95% Confidence Interval: 0.697-0.821). Motif analysis revealed that CC sequences were associated with high efficiency and TT sequences were associated with low editing efficiency. This study demonstrates that integrating sequence-based features with existing design scores can improve sgRNA efficiency prediction in vivo. The proposed framework can be applied to the development of next-generation sgRNA design tools, with direct implications for gene therapy, effective animal model generation, and precision genome engineering.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.08.675005v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comparative Analysis of Single-Nucleus and Single-Cell RNA Sequencing in Human Bone Marrow Mononuclear Cells: Methodological Insights and Trade-offs</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ghamsari, R., de Graaf, C. A., Thijssen, R., You, Y., Lovell, N. H., Alinejad-Rokny, H., Ritchie, M. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Bone marrow mononuclear cells (BMMCs) are a heterogeneous pool of hematopoietic progenitors and mature immune cells that collectively sustain hematopoiesis and coordinate immune responses. The bone marrow serves not only as the primary site for blood cell production but also as a niche for various disorders, including blood cancers. Advances in single-cell RNA sequencing (scRNA-seq) and single-nucleus RNA sequencing (snRNA-seq) have significantly enhanced our understanding of the cellular biology and molecular dynamics within this complex microenvironment. The choice between these two approaches, however, is often shaped or constrained by the study design, such as research objectives, sample type, and preservation conditions. Consequently, methodological differences in library preparation and transcript capture efficiency can introduce systematic biases that complicate downstream analyses and interpretation, underscoring the need to identify and account for method-specific features. In this study, we conducted a comparative analysis of matched snRNA-seq and scRNA-seq datasets from 11 pairs of healthy donor bone marrow mononuclear cell samples, generated using the popular 10x Genomics platform. We evaluated method-specific biases using multiple quality metrics and compared cell type proportions and transcriptomic signatures captured by each approach. Integrative analysis of these datasets is feasible but not advisable due to systematic gene length biases that were observed between these approaches. Our results showed that despite inherent differences in library complexity, both protocols reliably captured all major cell types. This comparative analysis highlights intrinsic differences between snRNA-seq and scRNA-seq data, providing valuable insights into their respective advantages, limitations, and trade-offs. These findings can assist researchers in selecting the optimal method tailored to specific biological questions and sample characteristics, and also enable more method-aware data analysis and interpretation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.08.675012v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Efficient Search of Ultra-Large Synthesis On-Demand Libraries with Chemical Language Models</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Heyer, K., Yang, D., Diaz, D. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Ultra-large building block catalogs provide inexpensive access to billions of synthesis- on-demand molecules, but the combinatorial scale renders conventional virtual screening impractical. We present Vector Virtual Screen (VVS), a score-function-agnostic machine learning framework for efficient navigation of combinatorial libraries and rapid identifi- cation of promising molecules for experimental validation. VVS comprises four key innovations: (i) the Embedding Decomposer, which factors molecules into building blocks in latent space; (ii) ChemRank, a correlation-based loss that improves retrieval precision; (iii) BBKNN, an algorithm for nearest-neighbor search directly in building block space; and (iv) a multi-scale hill-climbing algorithm for gradient-based navi- gation of molecular embedding vector databases. Across diverse scoring functions, VVS consistently outperforms existing methods in retrieving high-scoring molecules while evaluating only a fraction of the library, achieving orders-of-magnitude run- time improvements. By turning ultra-large libraries into tractable search spaces, VVS enables virtual screening to keep pace with the rapid expansion of chemical space and adapt seamlessly to future advances in scoring functions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.04.674350v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Human-scATAC-Corpus: a comprehensive database of scATAC-seq data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chen, X., Gao, Z., Li, K., Wang, Z., Jiang, Q., Cui, X., Li, Z., Jiang, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell Assay for Transposase-Accessible Chromatin using sequencing (scATAC-seq) profiles chromatin accessibility at cellular resolution, making it possible to reveal epigenomic landscapes that govern gene regulation in a variety of cells. Nevertheless, heterogeneous feature spaces and complex processing pipelines have impeded the construction of an ensemble resource capable of supporting diverse downstream analytical scenarios. To address this gap, we present Human-scATAC-Corpus (https://health.tsinghua.edu.cn/human-scatac-corpus/), a comprehensive database of human scATAC-seq comprising 5,407,621 cells from 35 datasets across 37 tissues or cell lines. To support complementary use cases, each dataset is distributed in three aligned formats: cell-by-candidate cis-regulatory element matrices for cross-dataset integration, raw fragment files for flexible processing, and cell-by-peak matrices for within-dataset analyses. This resource spans diverse biological contexts and includes rich metadata, enabling method benchmarking and development, as well as pretraining of foundation models. The website offers searchable browsing, detailed dataset pages, on-demand downloads, and tutorials. EpiAgent, a foundation model pretrained on Human-scATAC-Corpus, is further integrated to provide online analyses, including reference mapping, embedding extraction, and cell type annotation. Human-scATAC-Corpus establishes a unified and scalable substrate for single-cell epigenomics and is intended to accelerate discovery while standardizing evaluation across tasks.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.05.674505v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Generative genomics accurately predicts future experimental results</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Koytiger, G., Walsh, A. M., Marar, V., Johnson, K. A., Highsmith, M., Abbas, A. R., Stirn, A., Brumbaugh, A. R., David, A., Hui, D., Kahn, J. M., Niu, S.-Y., Ray, L. J., Savonen, C., Setvik, S., Leek, J. T., Bradley, R. K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Realizing AI's promise to accelerate biomedical research requires AI models that are both accurate and sufficiently flexible to capture the diversity of real-life experiments. Here, we describe a generative genomics framework for AI-based experimental prediction that mirrors the process of designing and conducting an experiment in the lab or clinic. We created GEM-1 (Generate Expression Model-1), an AI system that effectively models the enormous range of bulk and single-cell gene expression experiments performed by scientists and benchmarked its performance across multiple biological axes. GEM-1's prediction of future gene expression experiments--RNA-seq data deposited in public archives after our training data cutoff--yielded accuracy comparable to the best-possible performance estimated by comparing the results of matched lab experiments. Overall, our approach illustrates the transformative potential of generative genomics for applications ranging from predicting cellular perturbations in vitro to de novo generation of data from large clinical cohorts.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.08.674753v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">High-precision Biomedical Text Corpora for Multi-Entity Recognition: A CoDiet study</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lain, A. D., Go, S., Mahmud, A., Rajendra, S., Cano San Jose, A., Loupasaki, K., Theodoridis, G., Bizkarguenaga Uribiarte, M., Gu, Y., Deda, O., Conde, R. D. A., Embade, N., de Diego Rodriguez, A., Burguera, N., Rossiou, D., Gil Redondo, R., Gallou, D., Tueros, I., Velmurugan, R., Gkanali, V., Caro Burgos, M., Pousinis, P., Alektoridis, G., Arranz, S., Nikolopoulos, N., Yan, X., Fernandez Carrion, R., Rowlands, T., Choi, D., Rei, M., Cave-Ayland, C., D Alessandro, A., The CoDiet consortium,, Beck, T., Posma, J. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We present here four biomedical, multi-entity corpora that can be used as benchmarks for named-entity recognition (NER), targeted to literature on metabolic syndrome. The CoDiet-Gold corpus (348,413 annotations) contains 500 re-distributable full-text publications, of which each document was independently annotated by two human experts, with disagreements fully adjudicated by a third expert. The CoDiet-Electrum corpus (2,349,499 annotations) contains 3,688 publications that were annotated using the entities from CoDiet-gold. Finally, for the same 3,688 documents, two fully machine-annotated corpora CoDiet-Bronze (2,399,647 annotations) and CoDiet-Silver (1,868,422 annotations), were created by utilising existing NER algorithms to annotate these. These corpora contain categories (organisms, disease, genes, proteins, metabolites) that add depth to existing corpora, as well as new categories that do not have other corpora (food, dietary methods, sample types, computational methods, study methodology, population characteristics, data types, and microbiome).</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.04.673740v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Insilico Investigation of Terpenoid efficacy on Cannabinoid Receptors using QSAR models and fragment-based Pharmacophore modelling</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mukherjee, A., Pandey, K. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cannabis Sativa a medical plant rich in phytochemicals have a range of terpenoids and cannabinoids with a wide range of therapeutic uses. The possible interaction of terpenoids with cannabinoid receptors implicated in analgesic pathways has not received enough attention. This work used in silico methods to target Cannabinoid Receptors Type I (CNR1) and II (CNR2) in order to find novel terpenoid compounds from Cannabis sativa that may have pain-relieving properties. QSAR modelling based on known cannabinoid receptor inhibitors was used to build and screen a curated library of 119 terpenoids. The terpenoid library was screened using pharmacophore models that were created. Molecular docking was performed on the top candidates using CNR1 (PDB: 5U09) and CNR2 (PDB: 5ZTY). To evaluate the stability of receptor-ligand complexes over 100 ns, molecular dynamics simulations were run. SwissADME was used for ADMET profiling in order to assess drug-likeness and pharmacokinetic characteristics. Both receptors' QSAR models showed strong predictive power (CNR1: r2 = 0.854; CNR2: r2 = 0.798). {gamma}-Eudesmol and Bisabolol were the top hits for CNR1 and CNR2, respectively, according to pharmacophore screening. Strong binding affinities were demonstrated by molecular docking ({gamma}-Eudesmol: -7.9 kcal/mol; Bisabolol: -8.5 kcal/mol), and simulations verified the stable connections. Both compounds were drug-like, according to ADMET analysis, while {gamma}-Eudesmol had better synthetic accessibility and fewer structural alarms. According to the results, bisabolol and {gamma}-Eudesmol show promise as cannabinoid receptor-targeted analgesics. The finding encourages more biological validation and emphasises the potential of terpenoids obtained from Cannabis sativa in medicinal medication discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.04.674146v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Machine Learning Models Based on Histological Images from Healthy Donors Identify ImageQTLs and Predict Chronological Age</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Meng, R., Zhu, W. Z., Cameron, C. C., Ni, P., Zhou, X., Ulammandakh, T., Gerstein, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Histological images offer a wealth of data. Mining these data holds significant potential for enhancing disease diagnosis and prognosis, though challenges remain, especially in non-cancer contexts. In this study, we developed a statistical framework that links raw histological images and their derived features to the genotype, transcriptome, and chronological age of the samples. We first demonstrated an association between image features and genotypes, identifying 906 image quantitative trait loci (imageQTLs) significantly associated with image features. Next, we identified differentially expressed (DE) genes by stratifying samples into image-similar groups based on image features and performing DE comparisons between groups. Additionally, we developed a deep-learning model that accurately predicts gene expression in specific tissues from raw images and their features, highlighting gene sets associated with observed morphological changes. Finally, we constructed another deep-learning model to predict chronological age directly from raw images and their features, revealing relationships between age and tissue morphology, especially aspects derived from nucleus features. Both models are supported by a computational approach that greatly compresses gigapixel whole-slide images and extracts interpretable nucleus features, integrating both large-scale tissue morphology and smaller local structures. We have made all interpretable nucleus features, imageQTLs, DE genes, and deep-learning models available as online resources for further research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.04.674328v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Unveiling the Terra Cognita of Sequence Spaces using Cartesian Projection of Asymmetric Distances</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ramette, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In the vast and ever-expanding realm of biological sequences, effectively visualizing relationships among numerous, potentially very long sequences is crucial. We introduce Cartesian Projections of Asymmetric Distances (CAPASYDIS), a novel approach designed to map and visualize the 'terra cognita' - the already explored regions - of a given sequence space. CAPASYDIS uniquely calculates asymmetric sequence distances, which account for the directional nature of sequence information, and then associates each distinct sequence to a unique location in a low-dimensional projection space, called a "seqverse". The approach is by design computationally scalable, and allows a standardized exploration of the seqverses of massive datasets, comprising hundreds to millions of potentially very long sequences. The approach is illustrated by analyzing the seqverses of rRNA sequences from the three domains of life. We show that the seqverses of Bacteria, Archaea and Eukaryota are not only spatially distinct, but also differ in shapes, sizes and patterns of variation around their centroids. asymdist values retain most of the taxonomic signal present in the sequences, when analyzed at different taxonomic levels from the domain to the single base difference levels. Overall, CAPASYDIS represents a novel mapping strategy that helps define the shape and boundaries of seqverses, allowing their exploration in a reproducible and scalable manner.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.04.674223v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Unravelling the anti-cancer mechanisms elicited by non-covalent thioredoxin reductase inhibitors for triple negative breast cancer therapy.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rullo, A., Flowers, B., Chang, K., Zhang, A., Petukhova, V. Z., Harding, L., Aboagye, S. Y., Bocchetta, M., Qiu, W., Williams, D. L., Angelucci, F., Petukhov, P. A., Kastrati, I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Thioredoxin reductases (cytosolic TXNRD1 and mitochondrial TXNRD2) are antioxidant enzymes often overexpressed in tumors, including triple negative breast cancer (TNBC), making them promising targets for cancer therapy. Inhibiting these enzymes may worsen the already elevated oxidative stress in cancer cells, ultimately leading to cell death through a pro-oxidant mechanism. However, selectively targeting TXNRDs has been challenging due to the traditional reliance on covalent inhibition strategies. Recent studies have identified a druggable allosteric pocket in this enzyme family, paving the way for the development of novel non-covalent inhibitors, referred to as TXNRD(i)s. These inhibitors have been tested in TNBC models and have demonstrated a range of anti-cancer effects. To understand the molecular and cellular consequences of TXNRD(i)s, we conducted unbiased transcriptomic analyses and found that the gene expression changes induced by TXNRD(i) treatment closely mirror those resulting from TXNRD1 silencing, reinforcing TXNRD1 as the primary therapeutic target. While TXNRD(i) treatment increases redox stress in TNBC cells, this is not the main driver of the anti-cancer effect. Instead, TXNRD(i)s potently inhibit cell proliferation and induce G1 phase cell cycle arrest. Notably, supplementing cells with exogenous deoxynucleotides restores cell viability, cell cycle progression and partially reverses cell death. These findings indicate that TXNRD(i)s deplete endogenous deoxynucleotide pools and impair ribonucleotide reductase activity as the main mechanism of anti-cancer effects. We further demonstrate that TXNRD(i)s inhibit both TXNRD1 and TXNRD2, and that dual inhibition is more effective in suppressing TNBC cell growth. In vivo, TXNRD(i) treatment significantly impairs TNBC xenograft tumor growth and reduces proliferation-related genes. Collectively, these findings challenge the prevailing paradigm that all TXNRD inhibitors function through prooxidant mechanism, instead highlighting that non-covalent TXNRD(i)s exert their effects by blocking proliferation offering a compelling therapeutic strategy for TNBC and potentially other cancers with elevated TXNRD expression.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.04.674327v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Systematic Growth Factor Profiling Platform for 3D Tumor Models Reveals Estradiol-Responsive Cellular Mechanisms of Immunotherapy Resistance</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lee, J.-K., Lee, K., Kim, M., Lim, S. O., Shin, D.-J., Shin, Y., Choi, J.-J., Lee, M., Kang, H. J., Lee, J.-W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Current organoid culture systems face critical limitations: standardized growth factor formulations fail to capture patient-specific signaling requirements, and single-cell-type approaches overlook tumor-stromal interactions essential for understanding immunotherapy resistance. To address these challenges, we developed an innovative biofabrication platform that systematically integrates patient-derived three-dimensional (3D) cultures with comprehensive growth factor profiling across 128 combinations. Through systematic screening of 23 ovarian cancer patient samples and single-cell RNA sequencing, we identified two estradiol-responsive cellular populations that coordinate immunosuppression: a malignant cell fraction (MAL.PDCD5) and a cancer-associated fibroblast (CAF) fraction (FB.TNFSF10). MAL.PDCD5 cells suppress immune infiltration by downregulating antigen presentation, while FB.TNFSF10 cells promote immunosuppression through enhanced TGF-{beta} signaling. Spatial transcriptomic analysis revealed striking mutual exclusivity between FB.TNFSF10 cells and T/NK cells, providing evidence of active immune cell exclusion. Most significantly, FB.TNFSF10 abundance emerged as a robust predictor of immune checkpoint inhibitor therapy resistance across multiple cancer cohorts, independent of conventional biomarkers. This biofabrication platform provides a scalable framework for drug screening and biomarker discovery, with immediate applications in precision medicine for patient stratification and combination therapy development.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.04.674374v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">EZH2 Inhibition Induces an Integrated Stress Response Driving Glutamine-Dependent Vulnerability in TNBC</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Deblois, G., Porras, L., Fukano, M., Gironne, A.-S., Quadri, E., Alzial, G., Philippeau, H., Alaessa, Y., Monast, A., Gorse, F., St-Arnaud, M., De Sa Tavares Russo, M., MADER, s., Avizonis, D., Lemieux, S., Park, M. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> EZH2, the catalytic subunit of Polycomb Repressive Complex II, is highly expressed and associated with poor prognosis in triple-negative breast cancer (TNBC). Despite inducing significant changes in chromatin profiles and gene expression, EZH2 inhibition in TNBC models has limited impact on growth, suggesting adaptive compensatory mechanisms. Here, we demonstrate that EZH2 inhibition induces accumulation of double-stranded RNA and misfolded proteins in TNBC, activating an integrated stress response (ISR) via the PKR/PERK-eIF2 pathway. We identify Activating Transcription Factor 4 (ATF4) as a key effector upon EZH2 inhibition, driving metabolic changes characterized by increased amino acid uptake and glutamine dependency. Targeting this ISR-ATF4-mediated metabolic response using glutaminase inhibitor in combination with EZH2 inhibition significantly impairs TNBC cell proliferation and tumor progression. These findings reveal a stress-driven metabolic adaptation that enables TNBC survival upon EZH2 blockade, highlighting inhibition of this pathway as a strategy to enhance the efficacy of EZH2 inhibitors in TNBC.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.04.674332v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Transcriptomic landscape of Gallbladder cancer reveals altered pathways related to cell cycle and Aurora kinase</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> SARKAR, S. K., NAYEK, A., MINOCHA, R., GILL, G. S., KUMAR, D., BHARDWAJ, N., DAS, P., DASH, N. R., KURDIA, K. C., DHAR, R., KARMAKAR, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;">Background: Gallbladder cancer (GBC) is a rare but aggressive biliary tract malignancy. This study explores the transcriptomic profile of GBC to identify differentially expressed genes (DEGs) and dysregulated pathways involved in its pathogenesis. Methods: RNA sequencing was performed on 13 GBC tumors and 6 matched controls. Functional enrichment analysis as well as WGCNA were used to identify dysregulated pathways, functionally relevant gene modules and hub genes. Key targets were validated in patient tissues and cell lines. Results: A total of 621 DEGs were identified (247 upregulated, 374 downregulated). Gene set enrichment analysis revealed activation of E2F targets and G2/M checkpoint, with downregulation of bile acid metabolism and estrogen response pathways. A tumor grade-correlated WGCNA module was enriched in cell cycle genes. TPX2 emerged as a central hub gene. Inhibitors of aurora kinase, TPX2 dependent enzyme, significantly reduced proliferation, migration, and invasion in GBC cells. High-grade tumors confirmed elevated Aurora kinase expression. Conclusions: This first transcriptomic analysis of GBC in South-East Asian Indians uncovers key drivers like TPX2 and Aurora kinases in disease progression. The study highlights cell cycle dysregulation and sex-linked signatures, offering insights for biomarker discovery and targeted therapies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.04.674273v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">STmiR: A Novel XGBoost-based framework for spatially resolved miRNA activity prediction in cancer transcriptomics.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jiaqi Yuan, Peng Xu, Zheng Ye, Wenbin Liu</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> MicroRNAs (miRNAs) are critical regulators of gene expression in cancer biology, yet their spatial dynamics within tumor microenvironments (TMEs) remain underexplored due to technical limitations in current spatial transcriptomics (ST) technologies. To address this gap, we present STmiR, a novel XGBoost-based framework for spatially resolved miRNA activity prediction. STmiR integrates bulk RNA-seq data (TCGA and CCLE) with spatial transcriptomics profiles to model nonlinear miRNA-mRNA interactions, achieving high predictive accuracy (Spearman's ρ > 0.8) across four major cancer types (breast, lung, ovarian, prostate), with performance further confirmed through direct comparison with experimentally measured miRNA expression in an independent spatial transcriptomics dataset. Applied to 10X Visium ST datasets from nine cancers, STmiR identifies six pan-cancer conserved miRNAs (e.g., hsa-miR-21, hsa-let-7a) consistently ranked in the top 40 across malignancies, and uncovers cell-type-specific regulatory networks in fibroblasts, B cells, and malignant cells. A breast cancer case study demonstrates STmiR's utility in uncovering biologically relevant miRNA-target relationships and their association with key cancer pathways. By enabling spatial mapping of miRNA activity, STmiR provides a transformative tool to dissect miRNA-mediated regulatory mechanisms in cancer progression and TME remodeling, with implications for biomarker discovery and precision oncology.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40924781/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Predicting mortality dynamics in cancer patients: A machine learning approach to pre-death events.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tatsuki Yamamoto, Minoru Sakuragi, Yuzuha Tuji, Yuji Okamoto, Eiichiro Uchino, Motoko Yanagita, Manabu Muto, Mayumi Kamada, Yasushi Okuno</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Capturing the dynamic changes in patients' internal states as they approach death due to fatal diseases remains a major challenge in understanding individual pathologies and improving end-of-life care. However, existing methods primarily focus on specific test values or organ dysfunction markers, failing to provide a comprehensive view of the evolving internal state preceding death. To address this, we analyzed electronic health record (EHR) data from a single institution, including 8,976 cancer patients and 77 laboratory parameters, by constructing continuous mortality prediction models based on gradient-boosting decision trees and leveraging them for temporal analyses. We applied Shapley Additive exPlanations (SHAP) to assess the contribution of individual features over time and employed a SHAP-based clustering approach to classify patients into distinct subtypes based on mortality-related feature dynamics. Our analysis identified three distinct clinical patterns in patients near death, with key laboratory parameters-including albumin, C-reactive protein, blood urea nitrogen, and lactate dehydrogenase-playing a critical role. Dimensionality reduction techniques demonstrated that SHAP-based patient stratification effectively captured hidden variations in terminal disease progression, whereas traditional stratification using raw laboratory values failed to do so. These findings suggest that machine learning-driven temporal analysis can reveal clinically meaningful state transitions that conventional approaches overlook, offering new insights into the heterogeneous nature of terminal disease progression. This framework has the potential to enhance personalized risk stratification and optimize individualized end-of-life care strategies by identifying distinct patient trajectories that may inform more targeted interventions.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40924724/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Utilizing artificial intelligence as an arbitrary tool in managing difficult COVID-19 cases in critical care medicine.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chervenkov L, Miteva DG, Velikova T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> This opinion review paper explores the application of artificial intelligence (AI) as a decisive tool in managing complex coronavirus disease 2019 (COVID-19) cases within critical care medicine. Available data have shown that very severe cases required intensive care, most of which required endotracheal intubation and mechanical ventilation to avoid a lethal outcome if possible. The unprecedented challenges posed by the COVID-19 pandemic necessitate innovative approaches to patient care. AI offers significant potential in enhancing diagnostic accuracy, predicting patient outcomes, and optimizing treatment strategies. By analyzing vast amounts of clinical data, AI can support healthcare professionals in making informed decisions, thus improving patient outcomes. We also focus on current technologies, their implementation in critical care settings, and their impact on patient management during the COVID-19 crisis. Future directions for AI integration in critical care are also discussed.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40880575/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

