<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Wednesday, 28 May 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Feasibility of multiomics tumor profiling for guiding treatment of melanoma.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nicola Miglino, Nora C Toussaint, Alexander Ring, Ximena Bonilla, Marina Tusup, Benedict Gosztonyi, Tarun Mehra, Gabriele Gut, Francis Jacob, Stephane Chevrier, Kjong-Van Lehmann, Ruben Casanova, Andrea Jacobs, Sujana Sivapatham, Laura Boos, Parisa Rahimzadeh, Manuel Schuerch, Bettina Sobottka, Natalia Chicherova, Shuqing Yu, Rebekka Wegmann, Julien Mena, Emanuela S Milani, Sandra Goetze, Cinzia Esposito, Jacobo Sarabia Del Castillo, Anja L Frei, Marta Nowak, Anja Irmisch, Jack Kuipers, Monica-Andreea Baciu-Drăgan, Pedro F Ferreira, Franziska Singer, Anne Bertolini, Michael Prummer, Ulrike Lischetti, Rudolf Aebersold, Marina Bacac, Gerd Maass, Holger Moch, Michael Weller, Alexandre P A Theocharides, Markus G Manz, Niko Beerenwinkel, Christian Beisel, Lucas Pelkmans, Berend Snijder, Bernd Wollscheid, Viola Heinzelmann, Bernd Bodenmiller, Mitchell P Levesque, Viktor H Koelzer, Gunnar Rätsch, Reinhard Dummer, Andreas Wicki</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> TuPro-based molecular tumor profiling and data-guided clinical decision-making for advanced melanoma .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> There is limited evidence supporting the feasibility of using omics and functional technologies to inform treatment decisions. Here we present results from a cohort of 116 melanoma patients in the prospective, multicentric observational Tumor Profiler (TuPro) precision oncology project. Nine independent technologies, mostly at single-cell level, were used to analyze 126 patient samples, generating up to 500 Gb of data per sample (40,000 potential markers) within 4 weeks. Among established and experimental markers, the molecular tumor board selected 54 to inform its treatment recommendations. In 75% of cases, TuPro-based data were judged to be useful in informing recommendations. Patients received either standard of care (SOC) treatments or highly individualized, polybiomarker-driven treatments (beyond SOC). The objective response rate in difficult-to-treat palliative, beyond SOC patients (n = 37) was 38%, with a disease control rate of 54%. Progression-free survival of patients with TuPro-informed therapy decisions was 6.04 months, (95% confidence interval, 3.75-12.06) and 5.35 months (95% confidence interval, 2.89-12.06) in ≥third therapy lines. The proof-of-concept TuPro project demonstrated the feasibility and relevance of omics-based tumor profiling to support data-guided clinical decision-making. ClinicalTrials.gov identifier: NCT06463509 .</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40425842/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">hu.MAP3.0: atlas of human protein complexes by integration of >25,000 proteomic experiments.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Samantha N Fischer, Erin R Claussen, Savvas Kourtis, Sara Sdelci, Sandra Orchard, Henning Hermjakob, Georg Kustatscher, Kevin Drew</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show our map of protein complexes is highly accurate and more comprehensive than previous maps, placing nearly 70% of human proteins into their physical contexts. hu.MAP3.0 generates testable functional hypotheses for 472 uncharacterized proteins which we support using AlphaFold modeling.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Macromolecular protein complexes carry out most cellular functions. Unfortunately, we lack the subunit composition for many human protein complexes. To address this gap we integrated >25,000 mass spectrometry experiments using a machine learning approach to identify >15,000 human protein complexes. We show our map of protein complexes is highly accurate and more comprehensive than previous maps, placing nearly 70% of human proteins into their physical contexts. We globally characterize our complexes using mass spectrometry based protein covariation data (ProteomeHD.2) and identify covarying complexes suggesting common functional associations. hu.MAP3.0 generates testable functional hypotheses for 472 uncharacterized proteins which we support using AlphaFold modeling. Additionally, we use AlphaFold modeling to identify 5871 mutually exclusive proteins in hu.MAP3.0 complexes suggesting complexes serve different functional roles depending on their subunit composition. We identify expression as the primary way cells and organisms relieve the conflict of mutually exclusive subunits. Finally, we import our complexes to EMBL-EBI's Complex Portal ( https://www.ebi.ac.uk/complexportal/home ) and provide complexes through our hu.MAP3.0 web interface ( https://humap3.proteincomplexes.org/ ). We expect our resource to be highly impactful to the broader research community.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40425816/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Beyond performance: How design choices shape chemical language models</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fender, I., Gut, J. A., Lemmin, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that molecular representation format, tokenization strategy, and model architecture influence chemical language model performance and chemical understanding.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Chemical language models (CLMs) have shown strong performance in molecular property prediction and generation tasks. However, the impact of design choices, such as molecular representation format, tokenization strategy, and model architecture, on both performance and chemical interpretability remains underexplored. In this study, we systematically evaluate how these factors influence CLM performance and chemical understanding. We evaluated models through fine-tuning on downstream tasks and probing the structure of their latent spaces using simple classifiers and dimensionality reduction techniques. Despite similar performance on downstream tasks across model configurations, we observed substantial differences in the structure and interpretability of their internal representations. SMILES molecular representation format with atomwise tokenization strategy consistently produced more chemically meaningful embeddings, while models based on BART and RoBERTa architectures yielded comparably interpretable representations. These findings highlight that design choices meaningfully shape how chemical information is represented, even when external metrics appear unchanged. This insight can inform future model development, encouraging more chemically grounded and interpretable CLMs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.23.655735v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Updated chromosome-level genome assembly of Sporisorium scitamineum with improved accuracy and completeness</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Taniguti, L. M., Vilanova, P. F., Kitajima, J. P., Monteiro-Vitorello, C. B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present an updated genome assembly of Sporisorium scitamineum, the fungal pathogen responsible for sugarcane smut disease, using newly generated high-quality Illumina short-read data and state-of-the-art variant-calling workflows.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In 2015, we published the complete genome assembly of Sporisorium scitamineum, the fungal pathogen responsible for sugarcane smut disease, generated primarily from PacBio long-read sequencing and polished with Illumina short reads. Since then, the tools for genome assembly have improved considerably, including advances in sequencing technologies and bioinformatics workflows, motivating us to revisit our original assembly. Here, we present an updated genome version, re-polished using newly generated high-quality Illumina short-read data and state-of-the-art variant-calling workflows, resulting in corrections at approximately 5,500 genomic positions, predominantly correcting SNP and indel errors. BUSCO benchmarking demonstrated increased genome completeness from the original 94.2% to 99.3%, indicating substantial improvements in both accuracy and gene annotation reliability. This enhanced genome assembly provides a significantly improved reference resource, enabling more robust genetic, evolutionary, and functional studies of this economically important sugarcane pathogen.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.23.649816v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comparative genomic analysis of Cluster AZ Arthrobacter phages</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Freise, A. C., Furlong, K. P., Klyczek, K. K., Beyer, A. R., Chong, R. A., Edgington, N. P., Gibb, B. P., Swerdlow, S. J., Bendele, M. G., Cobb, I. D., Mitchell, Z. J., Cresawn, S. G., Garcia Costas, A. M., Rudner, A. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We describe a group of 32 Arthrobacter phages grouped in Cluster AZ, isolated on four different Arth Robacter strains all with siphoviral morphologies.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Bacteria in the Arthrobacter genus belong to the phylum Actinobacteria and are primarily soil-dwelling. Over 600 bacteriophages infecting Arthrobacter hosts have been isolated and sequenced, and genomic analyses show these phages to be highly diverse with mosaic genome architectures. We describe here a group of 32 Arthrobacter phages grouped in Cluster AZ, isolated on four different Arthrobacter strains all with siphoviral morphologies. The Cluster AZ phages exhibit a spectrum of diversity and can be subdivided into four subclusters. The intracluster diversity was analyzed in-depth at the whole genome level and through individual genes. The diversity in minor tail protein and endolysin genes correlates partly with isolation host strain and may be predictive of the host range of these phages. Most of the Cluster AZ phages are temperate, form stable lysogens, and encode an integrase; however, an immunity repressor gene has not been identified. As more Arthrobacter phages are isolated and analyzed they continue to provide new insights into phage evolution.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.26.655371v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Neurotranscriptomic profiling of DWV-infected honey bee foragers with different cognitive abilities</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Loughran, S. E., Manfredini, F., Bowman, A. S., Dingle, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We examined the transcriptomic response of the honey bee mushroom bodies in naturally occurring deformed wing virus infections and found significant changes in the immune response, including antimicrobial peptides, lysozyme, PPO, and other genes associated with responses to responses to a range of stressors.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Honey bees (Apis mellifera) provide important ecosystem services to both natural and hu-man-managed environments, but are increasingly threatened by a variety of pathogens, the most common of which is deformed wing virus (DWV). DWV is known to replicate in the honey bee brain and has been documented as both improving and impairing olfactory learning and memory. We examined the transcriptomic response of the honey bee mushroom bodies--an area of the insect brain associated with higher cognitive functions--in bees with naturally occurring DWV infections who varied in their ability to perform an associative learning task. RNA-seq analysis detected increased expression of genes involved in the immune response, including important antimicrobial peptides (AMPs) such as hymenoptaecin, apidaecin, and abaecin, and the down-regulation of lysozyme, PPO, and other genes associated with responses to a range of stressors. Additionally, gene ontology (GO) enrichment analysis revealed overrepresentation of key bio-logical processes which form part of the immune response. We also noted significant differential expression of long non-coding RNAs (lncRNAs) presumed to be acting in a regulatory manner, and used these lncRNAs to construct gene regulatory networks (GRNs). Strikingly, in contrast to previous studies on bees with artificially-induced infections that have examined viral loads in the abdomen and non-specific areas of the brain, no correlation between DWV load in the mushroom bodies and cognitive function was noted. This highlights the complexity of host-pathogen inter-actions in honey bee neural tissues and the benefits of a spatially-refined approach to brain transcriptomics in naturally-occurring infections.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.23.655587v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Mobius: Mixture-Of-Experts Transformer Model in Epigenetics of ME/CFS and Long COVID</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Acharya, P., Jacoby, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present Mobius, a transformer-based model that uses autoencoder-derived features from blood DNA methylation to distinguish ME/CFS, Long COVID, and healthy controls.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Myalgic Encephalomyelitis/Chronic Fatigue Syndrome (ME/CFS) and Long COVID are chronic debilitating post-infectious illnesses that collectively affect up to 470 million individuals. Unlike illnesses of comparable scale, there are no validated blood or imaging tests for the clinical diagnosis of these conditions. Currently, these conditions are diagnosed through clinical exclusion, resulting in approximately 90% of ME/CFS patients being incorrectly diagnosed as Long COVID patients. This misdiagnosis contributes to delayed care and millions of dollars in healthcare burdens. We present Mobius, a transformer-based model that uses autoencoder-derived features from blood DNA methylation to distinguish ME/CFS, Long COVID, and healthy controls. Using 852 samples from 14 distinct datasets, our method employs three innovations: (i) self-supervised masked pretraining to learn epigenetic patterns, (ii) a sparsely-gated mixture-of-experts architecture to handle heterogeneous data, and (iii) an adaptive computation time mechanism for dynamic inference. Mobius achieved 97.06% accuracy (macro-F1 0.95, AUROC 0.96), outperforming current symptom-based diagnostics (58%) and baseline models such as XGBoost (82%). Ablation experiments showed that pretraining added 6% accuracy and that the gating and adaptive depth contributed an additional 7%. Our open-source pipeline could enable a much-needed objective blood test for these conditions and guide targeted precision medicine therapies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.25.656018v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Alignment-driven Cross-Graph Modeling for 3D RNA Inverse Folding</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, S., Shen, H.-B., Pan, X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present AlignIF, a pipeline designed to generate RNA sequences conditioned on the target RNA 3D structure and its multiple structure alignment (MStA), explicitly incorporating structural evolutionary conservation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The 3D RNA inverse folding task aims to design RNA sequences that can fold into a target tertiary structure. This task is complicated by the inherent structural complexity and flexibility of RNA molecules. Considering that conserved RNA structures often exhibit sequence similarity, we present AlignIF, a pipeline designed to generate RNA sequences conditioned on the target RNA 3D structure and its multiple structure alignment (MStA), explicitly incorporating structural evolutionary conservation. AlignIF introduces a novel MStA backbone encoder designed to model multiple geometric graphs derived from MStA, which can better capture meaningful information from MStA structures. Additionally, AlignIF uses a random-ordered autoregressive sequence decoder to generate candidate RNA sequences. In our constructed benchmark dataset, AlignIF achieves a perplexity of 2.32 and a recovery rate of 57. 08%, respectively, a relative improvement of 17.8% and 11.8% over the state-of-the-art methods. In addition, AlignIF excels in diversity and ranking capability. We further demonstrate the added value of the incorporated MStA structures along with the proposed encoder architecture.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.23.655885v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">First full-genome alignment representative for the genus Pestivirus</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Triebel, S., Eulenfeld, T., Ontiveros-Palacios, N., Sweeney, B., Tautz, N., Marz, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We provide the first genome-wide multiple sequence alignment (MSA) of all known pestivirus species and identify 29 new RNA secondary structures in the protein-coding region, with so far unresolved functional importance.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The members of the genus Pestivirus in the family Flaviviridae comprise economically important pathogens of life stock like classical swine fever (CSFV) and bovine viral diarrhea virus (BVDV). Intense research over the last years revealed that at least 11 recognized and eight proposed pestivirus species exist. The single-stranded, positive-sense RNA genome encodes for one large polyprotein which is processed by viral and cell-derived proteases into 12 mature proteins. Besides its protein-coding function, the RNA genome also contains RNA secondary structures with critical importance for various stages of the viral life cycle. Some of those RNA secondary structures, like the internal ribosome entry site (IRES) and a 3' stem-loop essential for genome replication, had already been studied for a few individual pestiviruses. In this study, we provide the first genome-wide multiple sequence alignment (MSA) including all known pestivirus species (accepted and tentative). Moreover, we performed a comprehensive analysis of RNA secondary structures phylogenetically conserved across the complete genus. While showing well-described structures, like a 5' stem-loop structure, the IRES element, and the 3' stem loop SLI to be conserved between all pestiviruses, other RNA secondary structures in the 3' untranslated region (UTR) were only conserved in subsets of the species. We identified 29 novel phylogenetically conserved RNA secondary structures in the protein-coding region, with so far unresolved functional importance. The microRNA binding site for miR-17 was previously known in species A, B, and C; in this study, we identified it in ten additional species, but not in species K, S, Q, and R. Another interesting finding is the identification of a putative long-distance RNA interaction between the IRES and the 3' end of the genome. These results together with the now available comprehensive multiple sequence alignment including all 19 pestivirus species, represent a valuable resource for future research and diagnostic purposes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.22.655560v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Fast and Accurate Draft Genome Patching with GPatch</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Diehl, A., Boyle, A. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce GPatch, a tool that constructs chromosome-scale pseudoassemblies from fragmented drafts using alignments to a reference genome.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent advancements in sequencing technologies have yielded numerous long-read draft genomes, promising to enhance our understanding of genomic variation. However, draft genomes are typically highly fragmented, posing significant challenges for functional genomics. We introduce GPatch, a tool that constructs chromosome-scale pseudoassemblies from fragmented drafts using alignments to a reference genome. GPatch produces complete, accurate, gap-free assemblies preserving over 95% of nucleotides from draft genomes. We show that GPatch assemblies can be used as references for Hi-C data analysis, whereas draft assemblies cannot. Until complete genome assembly becomes routine, GPatch presents a necessary tool for maximizing the utility of draft genomes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.22.655567v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">JMod: Joint modeling of mass spectra for empowering multiplexed DIA proteomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> McDonnell, K., Wamsley, N., Derks, J., Sipe, S., Yeh, M., Specht, H., Slavov, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> JMod is an open source software for Joint Modeling of mass spectra that enables increasing the throughput of sensitive proteomics by multiplexing in the mass and time domains.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The throughput of mass spectrometry (MS) proteomics can be increased substantially by multiplexing that enables parallelization of data acquisition. Such parallelization in the mass domain (plexDIA) and the time domain (timePlex) increases the density of mass spectra and the overlap between ions originating from different precursors, potentially inhibiting analysis. To enhance sequence identification and quantification from such spectra, we developed an open source software for Joint Modeling of mass spectra: JMod. It uses the intrinsic structure in the spectra and explicitly models overlapping peaks as linear superpositions of their components. This modeling enabled performing 9-plexDIA using 2 Da offset PSMtags by deconvolving the resulting overlapping isotopic envelopes in both MS1 and MS2 space. The results demonstrate 9-fold higher throughput with preserved quantitative accuracy and coverage depth. This support for smaller mass offsets increases multiplexing capacity and thus proteomic throughput for a given plexDIA tag, and we demonstrate this generalizability with diethyl labeling. By supporting enhanced decoding of DIA spectra multiplexed in the mass and time domains, JMod provides an open and flexible software that enables increasing the throughput of sensitive proteomics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.22.655512v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identifying Positional Orthologs by Linear Programming</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gafurov, A., Swenson, K. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present two novel integer linear programming solutions for the identification of positional orthologs within a family of homologous genes, which are those pairs of orthologous gene copies that descend from the same ancestor, not having been produced by an intervening duplication event.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Studying the evolution of gene orders is essential to our understanding of the link between gene regulation and phenotype. Comparative studies concerning the full complement of genes in large numbers of genomes are now possible, thanks to the advent of affordable long-read sequencing and whole chromosome assembly, combined with automated genome annotation. The study of gene orders remains cumbersome, however, due to a lack of a streamlined, and standardized, process meant for the identification of positional orthologs. Within a family of homologous genes, these are those pairs of orthologous gene copies that descend from the same locus in their most recent ancestor, not having been produced by an intervening duplication event. This article approaches the detection of positional orthologs by way of the Maximum Colorful Graph Partition problem. We present two novel integer linear programming solutions to this problem, evaluate their efficacy on simulated and real data, while comparing them to several new and existing heuristics, as well as to the Double Cut and Join based ILP called FFGC.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.22.655535v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Advancing the Discovery of Phage-Host Interactions and Disease Classification from Metagenomic Profiles Using Deep Learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yang, Y., Wang, T., Huang, D., Wang, X.-W., Weiss, S. T., Korzenik, J., Liu, Y.-Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed phiNODE (phage-host interaction predictor using Neural Ordinary Differential Equations), a deep learning method for predicting phage- host interactions directly from metagenomic profiles.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Microbiomes, the communities of microorganisms, are essential for sustaining ecosystem functions in diverse environments, including the human gut. Bacteriophages (phages) interact dynamically with their prokaryotic hosts and play a crucial role in shaping the structure and function of microbial communities. Previous approaches for inferring phage-host interactions from metagenomic data (e.g., assembly-based methods) are constrained by high computational demands, limited sensitivity, and the inability to accurately capture ecological relationships. To address these issues, we developed phiNODE (phage-host interaction predictor using Neural Ordinary Differential Equations), a deep learning method for predicting phage-host interactions directly from metagenomic profiles. We first validated phiNODE using synthetic datasets generated by ecological models and found that it outperformed both alternative deep-learning and co-abundance-based methods in inferring phage-host interactions. We then applied phiNODE to a large-scale metagenomic dataset comprising 7,016 stool samples from healthy individuals and identified 90% more genus-level phage-host interactions than traditional assembly-based methods. Finally, we demonstrated that the latent representations learned by phiNODE served as more powerful features than taxonomic abundance-based features for disease classifications. In summary, phiNODE offers a novel framework for inferring phage-host interactions from shotgun metagenomic data, representing an innovative approach for advancing microbiome research and clinical applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.26.656232v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">FoldBench: An All-atom Benchmark for Biomolecular Structure Prediction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xu, S., Feng, Q., Qiao, L., Wu, H., Shen, T., Cheng, Y., Zheng, S., Sun, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce FoldBench, an extensive benchmark dataset for all-atom structure prediction models, which provides crucial insights and benchmarks to inform future model development and refinement.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurate prediction of biomolecular complex structures is fundamental for understanding biological processes and rational therapeutic design. Recent advances in deep learning methods, particularly all-atom structure prediction models, have significantly expanded their capabilities to include diverse biomolecular entities, such as proteins, nucleic acids, ligands, and ions. However, comprehensive benchmarks covering multiple interaction types and molecular diversity remain scarce, limiting fair and rigorous assessment of model performance and generalizability. To address this gap, we introduce FoldBench, an extensive benchmark dataset consisting of 1,522 biological assemblies categorized into nine distinct prediction tasks. Our evaluations reveal critical performance dependencies, showing that ligand docking accuracy notably diminishes as ligand similarity to the training set decreases, a pattern similarly observed in protein-protein interaction modeling. Furthermore, antibody-antigen predictions remain particularly challenging, with current methods exhibiting failure rates exceeding 50%. Among evaluated models, AlphaFold 3 consistently demonstrates superior accuracy across the majority of tasks. In summary, our results highlight significant advancements yet reveal persistent limitations within the field, providing crucial insights and benchmarks to inform future model development and refinement.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.22.655600v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Database for extended ligand-target analyses (DELTA): a new balanced resource for AI applications in drug discovery</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pisati, A., Pozzi, A., Giuntini, F., Pedretti, A., Vistoli, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We here present the DELTA resource, a database including balanced and annotated datasets of ligands for about 500 therapeutically relevant targets specifically collected for developing AI-based predictive models.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We here present the DELTA resource, a database including balanced and annotated datasets of ligands for about 500 therapeutically relevant targets specifically collected for developing AI-based predictive models. For each target, DELTA comprises an optimized protein structure plus 200 experimentally tested ligands equally distributed between active and inactive molecules. All ligands are prepared by considering unspecified isomeric elements and combining semi-empirical calculations with MD simulations to explore their conformational space. The so-collected molecules allowed extended analyses of both ligands and targets, and the study presents some preliminary results. The performed analyses revealed that on average active ligands are larger than inactive molecules, while possessing a similar polarity. The scaffold analysis emphasized the expected and crucial role of aromatic systems, even though with some relevant differences between active and inactive molecules. Moreover, similar targets often show conserved binding sites and there is a limited but not negligible relationship between the similarity of binding sites and ligands suggesting that similar pockets tend to bind rather similar ligands. Finally, the collected biological data also allowed the analysis of the polypharmacological profile of the ligands endowed with more than one biological value. Most ligands bind two or three targets with diverse activities and almost always the bound targets belong to the same biological class. All the collected data are available for download at delta.unimi.it.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.22.655476v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

