<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Monday, 22 September 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">3D multi-omic mapping of whole nondiseased human fallopian tubes at cellular resolution reveals a large incidence of ovarian cancer precursors</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Forjaz, A., Queiroga, V., Li, Y., Hernandez, A., Crawford, A., Qin, X., Zhong, M., Tsapatsis, M., Joshi, S., Kramer, D., Nizet, O., Bea, H., Li, Y., Qin, S., O Flynn, R., Yang, M., Pratt, B., Wu, F., Gensbigler, P., Blecher, M., Wu, P.-H., Kagohara, L. T., Shih, I. M., Zwicker, D., Atkinson, M. A., Shi, L., Fan, R., Kiemen, A. L., Wirtz, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A spatially resolved, multi-omics framework that integrates whole-organ 3D imaging at cellular resolution with targeted proteomic, metabolomic, and transcriptomic profiling to detect and characterize microscopic tubal lesions and their presumed precursors.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Uncovering the spatial and molecular landscape of precancerous lesions is essential for developing meaningful cancer prevention and early detection strategies. High-Grade Serous Carcinoma (HGSC), the most lethal gynecologic malignancy, often originates from Serous Tubal Intraepithelial Carcinomas (STICs) in the fallopian tubes, yet their minute size and our historical reliance on standard 2D histology contribute to their underreporting. Here, we present a spatially resolved, multi-omics framework that integrates whole-organ 3D imaging at cellular resolution with targeted proteomic, metabolomic, and transcriptomic profiling to detect and characterize microscopic tubal lesions. Using this platform, we identified a total of 99 STICs and their presumed precursors that harbor TP53 mutations in morphologically unremarkable tubal epithelium in all five specimens obtained from cancer-free organ donors with average-risk of developing ovarian cancer. Although these lesions comprised only 0.2% of the epithelial compartment, they displayed geographic diversity, immune exclusion, metabolic rewiring, and DNA copy number changes among lesions and normal fallopian tube epithelium discovered alterations in STIC-associated genes and the pathways they control. In sum, this platform provides a comprehensive 3D atlas of early neoplastic transformation, yielding mechanistic insights into tumor initiation and informing clinical screening strategies for detecting cancer precursors in whole organs at cellular resolution.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.21.677628v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Designing smart spatial omics experiments with S2Omics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yuan, M., Jin, K., Yan, H., Schroeder, A., Luo, C., Yao, S., Dumoulin, B., Levinsohn, J., Luo, T., Clemenceau, J., Jang, I., Kim, M., Liu, Y., Deng, M., Furth, E. E., Wilson, P., Nayak, A., Lubo, I., Soto, L. M. S., Wang, L., Park, J. H., Susztak, K., Hwang, T. H., Li, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present S2Omics, an end-to-end workflow that automatically selects ROIs from histology images with the goal of maximizing molecular information content in the ROIs.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Spatial omics technologies have transformed biomedical research by enabling high-resolution molecular profiling while preserving the native tissue architecture. These advances provide unprecedented insights into tissue structure and function. However, the high cost and time-intensive nature of spatial omics experiments necessitate careful experimental design, particularly in selecting regions of interest (ROIs) from large tissue sections. Currently, ROI selection is performed manually, which introduces subjectivity, inconsistency, and a lack of reproducibility. Previous studies have shown strong correlations between spatial molecular patterns and histological features, suggesting that readily available and cost-effective histology images can be leveraged to guide spatial omics experiments. Here, we present S2Omics, an end-to-end workflow that automatically selects ROIs from histology images with the goal of maximizing molecular information content in the ROIs. Through comprehensive evaluations across multiple spatial omics platforms and tissue types, we demonstrate that S2Omics enables systematic and reproducible ROI selection and enhances the robustness and impact of downstream biological discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.21.677634v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Binding Affinities of Disheveled PDZ Protein-Ligand Complexes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Singh, A., Jubintoro, A., Kancharla, H., Blankenberg, P., Zheng, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We test the validity of a computational molecular dynamics simulator, Binding Free Energy Estimator 2 (BFEE2), in determining peptide and small molecule ligand affinity for Wnt/beta-catenin signaling.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Wnt/beta-catenin signaling is critical for cell growth and development, with its hyperactive dysregulation implicated in the development of cancer. Current therapeutic research on inhibition of Wnt/beta-catenin signaling is impeded by the high cost of experimentally determining binding affinities. Consequently, interest has risen in screening potential inhibitors binding affinities with computational tools to reduce costs. Here, we test the validity of a computational molecular dynamics simulator, Binding Free Energy Estimator 2 (BFEE2), in determining peptide and small molecule ligand affinity for Wnt/beta-catenin signaling. We focus on the Dishevelled (DVL) PDZ domain, a key mediator in WNT signaling through its ability to bind to various peptide ligands. We analyze the binding affinities of several DVL PDZ domain-peptide complexes against previously established results to determine the validity of computational analysis. We conclude that computational molecular dynamics simulations are moderately effective in determining DVL peptide ligand complex affinity with an R2 of 0.44; however, the simulation was not effective with small molecule drug inhibitors with an R2 of 0.06.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.21.672002v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Predicting functional constraints across evolutionary timescales with phylogeny-informed genomic language models</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ye, C., Benegas, G., Albors, C., Li, J. C., Prillo, S., Fields, P. D., Clarke, B., Song, Y. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce GPN-Star, a biologically grounded genetic language model that leverages whole-genome alignments and species trees to model evolutionary relationships explicitly.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genomic language models (gLMs) have emerged as a powerful approach for learning genome-wide functional constraints directly from DNA sequences. However, standard gLMs adapted from natural language processing often require extremely large model sizes and computational resources, yet still fall short of classical evolutionary models in predictive tasks. Here, we introduce GPN-Star (Genomic Pretrained Network with Species Tree and Alignment Representation), a biologically grounded gLM featuring a phylogeny-aware architecture that leverages whole-genome alignments and species trees to model evolutionary relationships explicitly. Trained on alignments spanning vertebrate, mammalian, and primate evolutionary timescales, GPN-Star achieves state-of-the-art performance across a wide range of variant effect prediction tasks in both coding and non-coding regions of the human genome. Analyses across timescales reveal task-dependent advantages of modeling more recent versus deeper evolution. To demonstrate its potential to advance human genetics, we show that GPN-Star substantially outperforms prior methods in prioritizing pathogenic and fine-mapped GWAS variants; yields unprecedented enrichments of complex trait heritability; and improves power in rare variant association testing. Extending beyond humans, we train GPN-Star for five model organisms - Mus musculus, Gallus gallus, Drosophila melanogaster, Caenorhabditis elegans, and Arabidopsis thaliana - demonstrating the robustness and generalizability of the framework. Taken together, these results position GPN-Star as a scalable, powerful, and flexible new tool for genome interpretation, well suited to leverage the growing abundance of comparative genomics data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.21.677619v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A complete diploid human genome benchmark for personalized genomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hansen, N. F., Dwarshuis, N., Ji, H. J., Rhie, A., Loucks, H., Logsdon, G. A., Vollger, M. R., Storer, J. M., Kim, J., Adam, E., Altemose, N., Antipov, D., Asri, M., Barreira, S., Bohaczuk, S. C., Bzikadze, A. V., Carioscia, S. A., Carroll, A., Chao, K.-H., Chu, Y., Das, A., Ebert, P., English, A., Fleharty, M., Fleming, L. E., Formenti, G., Guarracino, A., Hartley, G. A., Jenike, K., Kalleberg, J., Kang, Y., King, R., Lipovac, J., Mastoras, M., Mitchell, M. W., Negi, S., Olson, N. D., Oshima, K. K., Paulin, L. F., Pickett, B. D., Porubsky, D., Ranchalis, J., Ranjan, D., Rautiainen, M., Riethm</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a telomere-to-telomere genome benchmark that achieves near-perfect accuracy (i.e. no detectable errors) across 99.4% of the complete HG002 genome.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Human genome resequencing typically involves mapping reads to a reference genome to call variants; however, this approach suffers from both technical and reference biases, leaving many duplicated and structurally polymorphic regions of the genome unmapped. Consequently, existing variant benchmarks, generated by the same methods, fail to assess these complex regions. To address this limitation, we present a telomere-to-telomere genome benchmark that achieves near-perfect accuracy (i.e. no detectable errors) across 99.4% of the complete, diploid HG002 genome. This benchmark adds 701.4 Mb of autosomal sequence and both sex chromosomes (216.8 Mb), totaling 15.3% of the genome that was absent from prior benchmarks. We also provide a diploid annotation of genes, transposable elements, segmental duplications, and satellite repeats, including 39,144 protein coding genes across both haplotypes. To facilitate application of the benchmark, we developed tools for measuring the accuracy of sequencing reads, phased variant call sets, and genome assemblies against a diploid reference. Genome-wide analyses show that state-of-the-art de novo assembly methods resolve 2 to 7% more sequence and outperform variant calling accuracy by an order of magnitude, yielding just one error per 100 kb across 99.9% of the benchmark regions. Adoption of genome-based benchmarking is expected to accelerate the development of cost-effective methods for complete genome sequencing, expanding the reach of genomic medicine to the entire genome and enabling a new era of personalized genomics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.21.677443v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MicroNucML: A machine learning approach for micronuclei segmentation and the refinement of nuclei-micronuclei relationships</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, Y., Boev, N., Garcia, U., MacDonald, K., Harding, S., Kumar, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We develop a generalizable Mask-RCNN model for small object detection and apply it to DNA damage data to gain insights into genome instability.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Micronuclei (MN) are structures containing small fragments of DNA, arising from mitotic errors or failed DNA repair attempts. Therefore, MN serve as markers of genomic instability and are typically quantified either manually or through threshold-based methods, which can be tedious and inaccurate, leading to varying degrees of success and throughput. By employing a two-phase labeling approach that utilizes polygon and brush segmentation, along with refinement using SAM2, we developed a high-quality MN segmentation tool. Subsequent data augmentation, which captured heterogeneity in image quality and color diversity, enabled us to train a generalizable Mask-RCNN model optimized for small object detection, achieving state-of-the-art performance in MN detection. Finally, we applied our model to immunofluorescence data obtained from cell lines exposed to DNA damage conditions to gain biological insights into MN dynamics and their role in inducing genome instability. In summary, this work establishes an accessible resource for systematically studying genome instability with significantly greater fidelity and sensitivity, enabling insights into damage biology that were previously unresolved.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.20.677550v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Transformer based method for the Cap Analysis of Gene Expression and Gene Expression Tag associated 5' cap site prediction in RNA</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Haldar, D. K., Pramanick, A., Mukherjee, C., Mitra, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We have developed a transformer based method for computational prediction of 5' capping from DNA sequences.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> 5' RNA capping is one of the major post-transcriptional modifications for the mobility and stability of RNA molecules. Measuring 5' caps of RNAs can help quantify expression levels of mRNAs and lncRNAs. One of the most successful RNAseq methods that have used capping as a tool to quantify expression of transcription is Cap Analysis of Gene Expression(CAGE). Computational prediction of capping can therefore be used as a precursor to the prediction of transcriptional expression. Unfortunately, there is hardly any computational technique that has focused purely on predicting 5' capping. We have developed a transformer based method for computational prediction of capping from DNA sequences. Our Llama and ReLoRA-based pre-training model, and Llama and LoRA-based fine-tuning model predict 5' cap sites. We have used Leave-one-chromosome-out-cross-validation for our model. The average accuracy, and F1-score after fine-tuning the human genome hg19(mouse genome mm9) for sequence classification is 79.12%(78.09%), and 78.11%(76.17%), respectively. We noted attention peak-based motifs having an aggregate Wilcoxon rank-sum p-value of 1.075e-10 between the attention peak region and the entire context window for the predicted positive motifs; an aggregate p-value of 7.17e-18 for the predicted negative motifs; and an aggregate p-value of 6.70e-08 between the attention peaks of the predicted positive and the predicted negative motifs. Our Llama-based approach aims to create a sequence-based framework to identify 5' capping sites corresponding to CAGE peaks. Our analysis reveals statistically significant motifs from the regions of peak attention scores, which demonstrates biological relevance for some through their resident sites matching with known TF motifs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.21.677558v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Parallel Evolution of a Drimenol Biosynthetic Gene Cluster Unique to Marine Bacteria</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Vo, N. N. Q., Nomura, Y., Takahashi, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show the unique evolution of drimenol biosynthesis in marine bacteria, and propose a new bacterial mono-{beta}-domain sesquiterpene synthase with similar architecture to HAD-like and terpene-like drimane-type synthase {beta}</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Natural bacterial communities represent a rich resource for the discovery of enzymes and natural compounds including terpenoids. However, the biosynthetic potential of marine bacteria remains poorly explored due to limitations in studying fastidious microorganisms. In a recent study, we discovered unusual haloacid dehalogenase (HAD)-like drimenol synthases (DMSs) of marine bacterial origin, including Aquimarina spongiae AsDMS and Flavivirga eckloniae FeDMS, which produce drimenol in vitro. In the present study, we conducted an in vivo investigation of the biosynthesis of this drimane-type sesquiterpene in marine bacteria. To verify drimenol accumulation in cultures of F. eckloniae, we mined the drimenol biosynthetic genes encoded in the F. eckloniae genome and found a putative AraC family transcriptional regulator in the drimenol gene cluster. Upon the addition of arabinose, we observed drimenol biosynthesis in F. eckloniae, demonstrating the importance of genomics-driven strategies in investigating natural products. Syntenic comparisons of drimenol gene clusters across bacteria revealed the unique evolution of drimenol biosynthesis in marine bacteria. Moreover, the distinct nature of drimenol pathways and relevant enzymes in bacteria compared to those in plants and fungi suggests parallel evolution. Remarkably, we describe two individual proteins from Sorangium cellulosum, HAD and terpene synthase with {beta} domain architecture, similar to HAD-like and terpene synthase {beta} domains of AsDMS, respectively, suggesting the fusion of monodomain proteins during the evolution of bacterial bifunctional DMSs. Our findings shed light on the evolutionary trajectory of the drimenol gene cluster unique to marine bacteria, while also demonstrating the new bacterial mono-{beta}-domain sesquiterpene synthase.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.20.677545v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DNA shape and epigenomics distinguish the mechanistic origin of human genomic structural variations</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Boev, N., Gerstein, M., Kumar, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify homology-driven SVs that retain ancestral recombination patterns and propose repair mechanisms that may influence the choice of repair mechanisms linked to SV origins.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The recent advent of long-read whole genome sequencing has enabled us to create an accurate telomere-to-telomere reference genome, construct pangenome graphs, and compile precise catalogs of genomic structural variations (SVs). These comprehensive SV repositories provide an excellent opportunity to explore the role of SVs in genotype-phenotype associations and examine the mechanisms by which SVs are introduced through double-strand break (DSB) repair. Here, we employed comprehensive SV catalogs identified through various short- and long-read whole genome sequencing efforts to infer the underlying mechanisms of SV introduction based on their genomic and epigenomic profiles. Our findings indicate that high local DNA methylation and DNA shape-related features, such as low variations in propeller twist, support the origins of homology-driven SVs. Subsequently, we utilized an active-learning-based unsupervised clustering approach, revealing that the homology-dependent SVs show greater evidence of retaining ancestral recombination patterns compared to their homology-independent counterparts. Finally, our comparison of inherited and de novo SVs from healthy populations and rare disease cohorts showed distinct upstream H3K27me3 levels in de novo SVs from individuals with ultra-rare disorders. These findings highlight genome-wide characteristics that may influence the choice of repair mechanisms linked to heritable SV origins.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.20.677549v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Automated FRET Analysis for Enhanced Characterization of Protein-Protein Interactions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> NURSOY, A. Z., CEVHEROGLU, O., SON, C. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the SONLab FRET Analysis Tool, an open-source and automated software that achieves FRET efficiencies comparable to those of manual methods but with reduced bias, enabling robust and high-throughput analysis of protein interactions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> F[o]rster Resonance Energy Transfer (FRET) analysis is a powerful technique for studying protein-protein interactions; however, manual methods often introduce variability and user dependency. We present the SONLab FRET Analysis Tool, an open-source and automated software that integrates Cellpose for cell segmentation with standardized pipelines for bleed-through correction and FRET efficiency calculation. By minimizing human intervention, the tool improves reproducibility and comparability between experiments. The results demonstrate that the tool achieves FRET efficiencies comparable to those of manual methods but with reduced bias, enabling robust and high-throughput analysis of protein interactions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.20.674865v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Progressive oxygenation of developing leaves directs morphogenesis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Panicucci, G., Shukla, V., Voloboeva, V., Jo, L., van Kollenburg, K., Buti, S., Dalle Carbonare, L., Licausi, F., Weits, D. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that oxygen acts as a positional cue in normal growth, guiding developmental trajectories, and highlight opportunities to harness oxygen gradients or sensing to direct plant form and function.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Oxygen availability underpins energy production in multicellular organisms, yet internal oxygen gradients arise in both plants and animals. Plants sense these variations via the PLANT CYSTEINE OXIDASE branch of the N-degron pathway, which regulates the stability of key transcription factors. Originally linked to metabolic control, this pathway recently emerged as a development regulator. While the shoot apical meristem was shown to be hypoxic, the oxygen dynamics of organs originating from this low-oxygen niche remain unknown. Here we show that developing leaves form a spatiotemporal oxygen gradient that is sensed through the oxygen sensing machinery. This pathway integrates local oxygen availability to regulate leaf morphogenesis: early hypoxia restricts cell expansion, while subsequent distal-to-proximal oxygenation enables specialized cell fates acquisition. Our findings reveal that oxygen acts as a positional cue in normal growth, guiding developmental trajectories. Our work highlights opportunities to harness oxygen gradients or sensing to direct plant form and function.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.21.677629v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Soybean aphids exploit abscisic acid signaling to suppress jasmonate defense responses</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hohenstein, J. D., Kanobe, C., Natukunda, M. I., Gallardo, P., Zhang, D., Kovinich, N., Helms, A. M., Tooker, J. F., MacIntosh, G. C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that aphid feeding attenuates jasmonate responses in soybean plants and exploit this ABA-JA antagonism to suppress plant defenses.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Soybean aphids (Aphis glycines) can induce susceptibility on soybean (Glycine max) during colonization. However, the mechanism for this process is not known. Based on previous transcriptome analyses, we hypothesized that aphids block effective jasmonate (JA) defenses by inducing an antagonistic abscisic acid (ABA) signal. To test this hypothesis, we used a combination of gene expression analyses, measurements of hormone levels, and aphid bioassays on plants with reduced expression of ABA-related genes. Aphid feeding attenuated JA responses in soybean plants and facilitated the growth of a chewing herbivore. Aphid-treated plants had increased levels of cis-JA but not biologically active JA-isoleucine, and aphid feeding induced expression of genes associated with JA-Ile catabolism. In parallel, aphid-feeding induced higher levels of ABA. ABA treatment and knockdown lines impaired in ABA biosynthesis (aba2-RNAi) or signaling (scof-1-RNAi), showed that ABA suppressed wound-induced JA responses. Aphid populations were significantly reduced on ABA-deficient plants and aphid-regulated attenuation of JA signaling was abolished in these lines. Remarkably, plants defective in ABA signaling had increased JA signaling in the absence of stressors. Our results indicate that, in soybean, the ABA pathway is necessary to control basal levels of JA and soybean aphids exploit this ABA-JA antagonism to suppress plant defenses.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.21.677601v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Phosphoinositide-specific Phospholipase C 2 (SlPLC2) Facilitates Vesicle Formation and Modulates Immune Signaling in Tomato Phytophthora infestans Interactions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Perk, E. A., D'Ambrosio, J. M., Cerrudo, I., Robuschi, L., Juarez, M., Velez, P., Mary, V., Theumer, M., Segretin, M. E., Laxalt, A. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify SlPLC2 as a susceptibility factor associated with enhanced P. infestans colonization and show its positive role in colonization.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Phosphoinositide-specific phospholipase C (PI-PLC) is a signaling enzyme that hydrolyzes membrane phosphoinositide to generate lipid- and lipid-derived second messengers. In plants, PI-PLCs have been implicated in various physiological processes, including immunity. Tomato SlPLC2 was previously shown to be implicated in susceptibility to the necrotrophic fungus Botrytis cinerea. However, whether this observation extends to pathogens with different lifestyles and evolutionary origins, remains unknown. Here, we investigated SlPLC2 function during infection with Phytophthora infestans, a globally devastating oomycete and causal agent of potato and tomato late blight. CRISPR-Cas9 knockout SlPLC2 plants showed significantly reduced disease symptoms, lower pathogen biomass, and decreased sporangia production, indicating impaired colonization. Upon infection, SlPLC2 knockout lines displayed attenuated expression of salicylic acid (SA)- and jasmonic acid (JA)-responsive genes suggesting disrupted hormone signaling. Consistent with PLC role during plant defense, early immune responses such as hydrogen peroxide accumulation and callose deposition were reduced in knockout plants. At the cellular level, these plants allow fewer infection vesicles formation by P. infestans, accompanied by an increase in expression of a biotrophy-associated effector gene PiAvrblb2, suggesting impaired establishment of infection. In Nicotiana benthamiana SlPLC2-GFP localizes predominantly at the plasma membrane. Upon P. infestans inoculations, SlPLC2-GFP localizes to the membrane surrounding infection vesicles, where we also detected phosphoinositides PI4P and PI(4,5)P2. Overexpression of SlPLC2 in Nicotiana benthamiana enhanced susceptibility to P. infestans, reinforcing its positive role in colonization. Together, these findings identify SlPLC2 as a susceptibility factor associated with enhanced P. infestans colonization.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.21.677499v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Evolutionary Rewiring of Anthocyanin Biosynthesis Pathway in Poaceae</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Khatun, N., Jones, A., Rahe, A., Choudhary, N., Pucker, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that an independently evolved MYB lineage (ncaMYB) is responsible for the regulation of the anthocyanin biosynthesis in Poaceae, followed by a loss of the Anthocyanins in the genus Brachypodium.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Poaceae is a species-rich family of plants in the monocots harboring many important crop species. As grasses, these plants lack striking flowers that are typically pigmented by anthocyanins. Here, we investigated the anthocyanin biosynthesis genes in Poaceae and characterized several striking patterns. (1) An independently evolved MYB lineage (ncaMYB) is responsible for the regulation of the anthocyanin biosynthesis in Poaceae. (2) The independently evolved BZ2 lineage replaced the otherwise widespread TT19/An9 lineage as anthocyanin-related glutathione S-transferase in Poaceae. (3) The anthocyanin biosynthesis genes ANS, arGST (BZ2), and the anthocyanin biosynthesis activating MYB (ncaMYB) appear to be absent from the genus Brachypodium. These observations could be explained by a loss of the anthocyanin biosynthesis at the basis of the Poaceae, followed by an evolutionarily independent regain and again a loss of the anthocyanin biosynthesis in Brachypodium.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.21.677584v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genome-wide Identification, Structural Features and Single-Cell Expression Atlas of the Carbonic Anhydrase Gene Family in Maize (Zea mays L.)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gao, Y., Zhao, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A genome-wide census and a single-cell-resolved expression atlas of the maize CA family.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Maize is a cornerstone C crop underpinning food, feed and bioenergy. Carbonic anhydrases (CAs) are zinc metalloenzymes that catalyze the interconversion of CO2 and HCO-, underpinning photosynthesis, pH homeostasis and stress acclimation. Despite extensive work on plant CA families, a maize-focused synthesis integrating genome-wide cataloging, structural divergence, regulatory logic, and cell-type specificity is still lacking. Here we present a genome-wide census and a single-cell-resolved expression atlas of the maize CA family. We identified 18 CA genes partitioning into (8), {beta} (5) and {gamma} (5) subfamilies. Proteins span 206-452 aa, pI 6.09-10.58 and 22.67-49.36 kDa, with predominantly hydrophilic profiles. Phylogeny resolved three robust clades (UFboot >90%) and indicated closer affinity to rice than to Arabidopsis; synteny recovered 15 maize-rice and 3 maize-Arabidopsis orthologs. Promoters are enriched for ABRE, TCA-element and ARE, suggesting ABA, SA and redox regulation. Bulk RNA-seq across 21 tissues revealed strong tissue specificity ({tau} >0.7 for 12 genes), with {beta}-CAs elevated in leaves and -CAs in roots. Single-cell analyses further showed Zm{beta}CA3 enriched in mesophyll cells and ZmCA2 enriched in vascular/root-cap cells. Together, these data position {beta}-CAs as chloroplast-centered hubs for C4 photosynthesis, while /{gamma}-CAs support ion/pH buffering and mitochondrial metabolism. We nominate cell-type-specific CA members as candidates for genetic improvement toward stress resilience in maize.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.21.677582v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Hollow-fibre biomanufacturing and cell-free engineering of HEK293 extracellular vesicles</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kelwick, R. J. R., Webb, A. J., Heliot, A., Freemont, P. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We demonstrate cell-free engineering of extracellular vesicles with membrane fusion proteins in vitro, which could help accelerate future EV discoveries.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Extracellular vesicles (EVs) are lipid-delineated nanoparticles that are produced by most cell types. EVs contain complex molecular cargoes that can have useful therapeutic or vaccine immunological effects. Cell-free gene expression systems can be used to produce membrane proteins in vitro, that can co-localise and integrate with exogenously added EVs. To advance this type of cell-free EV engineering we manufactured, isolated and characterised HEK293 cell EVs. These EVs were successfully cell-free engineered with several CD63-based membrane fusion proteins. In our most optimal conditions, up to 4.83 x1011 /ml of HEK293 EVs were successfully cell-free engineered with a fusion membrane protein incorporating CD63 I-shaped membrane-insertion topology transmembrane helix 3 (CD63ITM3) and monomeric green lantern (mGL). Finally, we also demonstrated that nano flow cytometry is a powerful tool for assessing cell-free EV engineering efficiency. In the future cell-free EV engineering could help accelerate future EV discoveries and the development of EV translational applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.20.677537v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Undersampling techniques for large datasets</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chen, L., Miranda Quintana, R. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We explore different undersampling strategies for the majority class of DNA-Encoded Libraries (DELs) to improve training of machine learning models.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> DNA-Encoded Libraries allow for an efficient approach to synthesize and screen billions of small molecules against a target of interest. With more real-world binding data, this can improve training of machine learning models. However, one key challenge in DELs is the severe imbalances between the classes, in other words, there are much more inactive than active compounds against any given target. This can heavily skew the training process. In this study, we explore different undersampling strategies for the majority class. These different techniques are benchmarked against random selection and prototyped on two different DEL datasets with three different machine learning models. Overall, the max_sim strategy shows the best scores, and the general pipeline is implemented in the DELight package.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.20.677527v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CryoFSL: an annotation-efficient, few-shot learning framework for robust protein particle picking in cryo-EM micrographs</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Poudel, B., Gyawali, R., Dhakal, A., Cheng, J., Xu, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present CryoFSL (Cryo-EM Few Shot-Learning), a novel few-shot learning framework built upon Segment Anything Model 2 (SAM2) with lightweight adapters, enabling robust particle picking using as few as five labeled micrographs, significantly reducing annotation burden and performance.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurate identification of protein particles in cryo-electron microscopy (cryo-EM) micrographs is crucial for high-resolution structure determination, but remains challenging due to the heavy reliance on extensive annotated datasets and the difficulty of ensuring robustness under low signal-to-noise ratio (SNR) conditions. Current approaches require large annotations and exhibit poor generalization to new protein targets. We present CryoFSL (Cryo-EM Few Shot-Learning), a novel few-shot learning framework built upon Segment Anything Model 2 (SAM2) with lightweight adapters, enabling robust particle picking using as few as five labeled micrographs, significantly reducing annotation burden. The framework's hierarchical adapter design supports dynamic feature modulation for low-SNR and heterogeneous conditions, resolving the trade-off between annotation burden and performance. CryoFSL surpasses both traditional template-based methods and state-of-the-art deep learning models across diverse proteins in the few-short learning setting, achieving superior recall, precision and 3D reconstruction resolution with minimal supervision. It maintains stability across heterogeneous micrographs and consistently detects high-quality particles with fewer false positives. Notably, CryoFSL achieves competitive density map reconstruction resolution with just a fraction of the particles picked by other methods, redefining efficiency and quality in cryo-EM analysis. This work paves the way for scalable, generalizable, and annotation-efficient particle picking pipelines.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.19.677446v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Automated Eight-Stage Classification of Drosophila melanogaster Using Transfer-Learning CNNs with Mobile Live-Inference Deployment</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bhatt, R., Prasad, R., S, A., Paul, S., Gupta, I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> An automated eight-class classifier for Drosophila staging, which achieves 85% accuracy (F1 = 0.85) on a held-out validation set).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Drosophila melanogaster is a foundational model organism whose rapid development and genetic tractability underpin research in genetics, development, and disease. Manual staging of its embryonic, larval, and pupal phases is slow and error-prone. An automated eight-class classifier is introduced to distinguish egg, first-, second-, and third-instar larvae, as well as white, brown, eye, and black pupae, from stereo-microscope images. By fine-tuning ImageNet-pretrained CNNs (ResNet-50, InceptionV3, ConvNeXtSmall) on a balanced dataset (~300 images per class), the best-performing model (ResNet-50) achieves 85% accuracy (F1 = 0.85) on a held-out validation set, significantly outperformed alternatives. Primary misclassifications align with subtle morphological transitions between adjacent stages. To facilitate broad adoption, the ResNet-50 model has been deployed in a lightweight Streamlit app offering live-camera inference ({approx}8 FPS on mobile). All code, pretrained weights, and data are publicly available, enabling scalable, high-throughput Drosophila staging for diverse experimental workflows.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.19.677285v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SRF fusion oncogenes encode constitutively activated chimeric transcription factors in myoid soft tissue tumors</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pirson, C., Sablon, A., Loriot, A., Autin, P., Braat, K. A., Karanian, M., Antonescu, C. R., Tirode, F., Demoulin, J.-B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel family of oncogenes in muscle-related soft tissue tumors, including SRF fusion genes, which recapitulate transcriptional signatures of human tumors.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> SRF fusion genes drive the pathogenesis of muscle-related soft tissue tumors, including subsets of perivascular tumors, inflammatory myofibroblastic tumor, and rhabdomyosarcoma. SRF encodes Serum Response Factor, a well-characterized transcription factor that regulates muscle development. We characterized four fusion genes: SRF::RELA, SRF::FOXO1, SRF::ICA1L, and SRF::PDGFRB. All localized to the nucleus and dimerized through the SRF MADS box. SRF::RELA, SRF::FOXO1, and SRF::ICA1L acted as constitutively active transcription factors independent of canonical cofactors, binding SRF target promoters and driving transcription via the partner transactivation domain (TAD). A cryptic TAD was uncovered in ICA1L. These fusions promoted mesenchymal cell growth and upregulated muscle-related genes in mesenchymal stem cells, recapitulating transcriptional signatures of human tumors. In contrast, SRF::PDGFRB acted through its kinase domain, was imatinib-sensitive, activated STAT1 and stimulated inflammation genes, consistent with its tumor phenotype. SRF fusions thus define a novel family of oncogenes in human myoid soft tissue tumors.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.18.676997v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Selective degradation of oncogenic extrachromosomal DNA by type I-E CRISPR</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Poursani, E., Pham, V. V., Ehteda, A., Vittorio, O.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the type I-E CRISPR system can specifically target and degrade ecDNA in human cancer cell lines (COLO320DM and GBM39). By designing guide RNAs targeting ecDNA-specific breakpoints absent from chromosomal DNA, we achieved efficient depletion of ecDNA</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Extrachromosomal DNA (ecDNA) is a major driver of oncogene amplification, intratumoral heterogeneity and therapy resistance across multiple cancer types, yet there are currently no effective strategies to selectively eliminate it. Here, we show that the type I-E CRISPR system can specifically target and degrade ecDNA in human cancer cell lines (COLO320DM and GBM39). By designing guide RNAs targeting ecDNA-specific breakpoints absent from chromosomal DNA, we achieved efficient depletion of MYC- and EGFR-containing ecDNAs in COLO320DM and GBM39 cells. Loss of ecDNA was accompanied by diminished oncogene signaling, disrupted ecDNA architecture, and impaired tumor cell proliferation, without detectable chromosomal off-target activity. These findings establish a proof-of-concept framework for directly targeting oncogenic ecDNAs and highlight type I-E CRISPR as a promising platform for therapeutic development in ecDNA-driven cancers.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.21.677573v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Spatially organized lymphocytic microenvironments in high grade primary prostate tumors</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Amiryousefi, A., Wala, J., Lin, J.-R., Labadie, B. W., Atmakuri, A., Maliga, Z., Toye, E., Chaudagar, K., Torcasso, M. S., Coy, S., Fanelli, G. N., Kobs, B., Socciarelli, F., Gagne, A., Van Allen, E., Patnaik, A., Sorger, P. K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A subset of high-grade primary prostate cancer (PCa) tumors harbor organized immune clusters that may play a role in tumor control and contain therapeutically targetable T and B cells.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The spatial organization and composition of the tumor-immune microenvironment (TME) play a critical role in shaping the progression of many solid cancers, but the organization of the TME in primary prostate cancer (PCa) remains poorly characterized. We therefore profiled the abundance and spatial distributions of major cell types involved in adaptive immunity in 29 radical prostatectomy specimens stratified into high (HGG; n=14) and low Gleason-grade (LGG; n=15). Compared to LGG, HGG PCa exhibited significantly greater B and T cell infiltration with many immune cells organized into clusters, some of which resembled tertiary lymphoid structures (TLSs). In HGG tumors, these clusters were dense, symmetric, rich in PD-1+ T cells, and frequently proximate to the tumor compartment. LGG clusters were less well organized, and T cell depleted. Thus, a subset of high-grade PCa harbor organized immune clusters that may play a role in tumor control and contain therapeutically targetable T and B cells.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.21.677465v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Tumor Organoids Modeling Reveals Timed Responses and Interplay of Radiotherapy and Chemotherapy in Pancreatic Cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yang, C., Keepers, Z., Shukla, H., Ren, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We proposed novel mathematical forecasting frameworks based on logistic growth ordinary differential equations (ODEs) and the characteristics of tumor responses to model the therapeutic effects of chemo- and radiation-induced killing.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Purpose: Model the therapeutic effects of chemotherapy, radiotherapy, and chemoradiotherapy to study the heterogeneous responses of patient-derived tumor organoids (PDTOs). Methods: We proposed novel mathematical forecasting frameworks based on logistic growth ordinary differential equations (ODEs) and the characteristics of tumor responses to model the therapeutic effects of chemo- and radiation-induced killing. To validate the models, we cultured PDTOs from different patients and treated them with radiation (4 Gy and 8 Gy), chemotherapy (FOLFIRINOX at a previously determined organoid-specific IC50 dose), and combined regimens, respectively. The diameters of 20-40 organoids per patient were tracked and measured using brightfield images up to 7 or 9 days following each treatment to capture organoid growth dynamics data, which was used for model fitting. The accuracy of the modeling was evaluated by the average normalized mean squared errors (NMSE) of data fitting. Results: The proposed mathematical modeling frameworks accurately captured the observed growth dynamics of three PDTO samples after chemotherapy, radiotherapy, and chemoradiotherapy, as reflected by the average NMSE of data fitting, which are all close to zero (less than 0.0045). The fitted parameters, including killing strength, effect window, and peak killing timing, revealed significant heterogeneities in treatment responses across different PDTOs. Chemotherapy shows pronounced effectiveness in the early stage, while radiotherapy exhibits the effect later in the first week, and a significantly stronger secondary response occurs one week after radiation. Chemoradiotherapy combines the strengths of both modalities, producing pronounced effects in both response phases, with modeling results suggesting that the radiation-induced killing effect may play a dominant role in the synergistic interaction. Conclusions: Our modeling frameworks demonstrated high accuracy in modeling the heterogeneous therapeutic responses of PDTOs and provided insights into the dynamic killing effects and interplay between chemotherapy and radiotherapy. The modeling of therapeutic responses of PDTOs provides a valuable tool for optimizing treatment regimens and informing clinical trial design, thereby improving the efficacy of personalized medicine.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.21.677544v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">FOXO1 inhibition and FADD knockdown have opposing effects on anticancer drug-induced cytotoxicity and p21 expression in osteosarcoma</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Walker, D., Hall, A., Bonwell, A., Gordon, N., Robinson, D., Hollomon, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that FOXO1 has a tumor suppressor function while FADD has a tumour promoting function in osteosarcoma following anticancer drug treatment.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Forkhead box class O1 (FOXO1) and fas-associated death domain (FADD) regulate cell death pathways and homeostatic processes such as cell cycle progression and apoptosis. FADD phosphorylation promotes nuclear localization of FOXO1 and FOXO1 regulates FADD expression. Therefore, it is plausible that FOXO1 and FADD have synergistic or antagonistic effects on cell cycle regulation and the response to anticancer drug treatment in cancer cells. In the present study we report that AS1842856-mediated inhibition of FOXO1 reverses anticancer drug-induced cytotoxicity while FADD knockdown increases anticancer drug-induced cytotoxicity in osteosarcoma (OS). Reversed anticancer drug-induced cytotoxicity was ac-companied with G2/M cell cycle arrest and increased expression of p21. The anti-cancer function of FOXO1 was further supported by the observation that OS cells that express higher basal levels of FOXO1 had increased sensitivity to camptothecin-induced cytotoxicity. FADD knockdown reversed FOXO1 inhibition-induced increase in p21 expression. The results presented in this study indicate that FOXO1 has a tumor suppressor function while FADD has a tumor promoting function in OS following anticancer drug treatment. The experimental approach used in this investigation also indicates that FADD antagonists the effect of FOXO1 on p21 expression in OS.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.20.677161v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

