<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Friday, 18 July 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Hybrid classical and quantum computing for enhanced glioma tumor classification using TCGA data.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Emine Akpinar, Murat Oduncuoglu</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel hybrid classical and quantum computing model to distinguish LGGs from HGGs using data from The Cancer Genome Atlas (TCGA).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Gliomas are the most prevalent malignant primary brain tumors and present diagnostic challenges due to varying survival rates and treatment responses between low-grade gliomas (LGGs) and high-grade gliomas (HGGs). Accurate classification is crucial for effective treatment and prognosis. While classical AI methods have shown promise in glioma classification, the growing volume of medical data, inherent noise, and limitations of classical vector spaces present significant challenges. However, quantum computing-based AI methods have the potential to process data in parallel by leveraging quantum properties such as superposition and entanglement, analyze higher-dimensional data more efficiently, and solve certain problems that classical methods struggle with more rapidly and effectively. This study introduces a novel hybrid classical and quantum computing model to distinguish LGGs from HGGs using data from The Cancer Genome Atlas (TCGA). In the classical part, an ensemble feature selection method was employed to identify the most informative molecular markers and clinical features in the TCGA. In the quantum component, six variational quantum classifier (VQC) models with varying hyperparameters were evaluated. These classifiers utilize selected features to differentiate LGGs from HGGs. Among these, the VQC-1 model, which employs [Formula: see text] and CX gates in the feature map and [Formula: see text] [Formula: see text] and CY gates in the parameterized quantum circuit, achieved the highest classification accuracy of 0.74 using the AQCD optimization method. Additionally, VQC-1 identified IDH1, age at diagnosis, PTEN, EGFR, and ATRX, in descending order of importance, as the most informative features distinguishing LGGs from HGGs. Compared to classical machine learning models, VQC-1 demonstrated performance comparable to that of XGBoost and GBM, while outperformed KNN, SVC, DTC, and RFC in five-fold cross-validation experiments. This study provides a novel perspective on glioma classification by integrating classical and quantum computing, offering valuable insights into hybrid computational approaches.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40676161/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Automated web-based typing of Clostridioides difficile ribotypes via MALDI-TOF MS.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mario Blázquez-Sánchez, Alejandro Guerrero-López, Ana Candela, Albert Belenguer-Llorens, José Miguel Moreno, Carlos Sevilla-Salcedo, María Sánchez-Cueto, Manuel J Arroyo, Mark Gutiérrez-Pareja, Vanessa Gómez-Verdejo, Pablo M Olmos, Luis Mancera, Patricia Muñoz, Mercedes Marín, Luis Alcalá, David Rodríguez-Temporal, Belén Rodríguez-Sánchez</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a MALDI-TOF MS-based workflow coupled with machine learning to distinguish epidemic toxigenic ribotypes (RT027 and RT181) from other strains in real time.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Clostridioides difficile is a major cause of hospital-acquired diarrhea and a driver of nosocomial outbreaks, yet rapid, accurate ribotype identification remains challenging. We sought to develop a MALDI-TOF MS-based workflow coupled with machine learning to distinguish epidemic toxigenic ribotypes (RT027 and RT181) from other strains in real time.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40676564/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SAME: Topology-flexible transforms enable robust integration of multimodal spatial omics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pratapa, A., Mansouri, S., Nikulina, N., Matuck, B., Schneider, M., Byrd, K., Savai, R., Tata, P. R., Singh, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Spatial omics technologies provide complementary and layered molecular insights that span proteins, transcripts, and metabolites. We present SAME (Spatial Alignment of Multimodal Expression) that introduces space-tearing transforms, a framework for controlling localized topological disruptions during cross-sectional alignment.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Spatial omics technologies provide complementary and layered molecular insights that span proteins, transcripts, and metabolites. However, aligning and integrating these modalities across serial tissue sections remains a computational challenge. Existing alignment methods are primarily unimodal and assume preserved topology, often failing with tissue distortions like tears, folds, or anatomical changes. Here, we present SAME (Spatial Alignment of Multimodal Expression) that introduces space-tearing transforms, a framework for controlling localized topological disruptions during cross-sectional alignment. Using integer linear programming to maximize cell type matches across the modalities, we enhance cell-type alignment accuracy by 20% compared to existing methods while preserving biologically meaningful spatial relationships. Applied to protein-RNA integration in healthy tongue tissue and lung adenocarcinoma, SAME revealed cryptic immune subpopulations that were otherwise missed by RNA-only or protein-only classification. In a separate lung adenocarcinoma study, we assayed and integrated protein and metabolomic profiles, uncovering localized mevalonic acid upregulation specifically within tumor-macrophage spatial niches and identifying targeted metabolic crosstalk invisible to single-modality approaches. SAME enables experimental designs that leverage each modality independently while computationally recovering cross-modal spatial structure, unlocking multimodal discoveries in complex tissues.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.12.664419v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Peer victimization and subsequent cognitive-affective biases in threat attention, reactivity, and interpretation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Heumann, J., Eisner, M., Ribeaud, D., Shanahan, M. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that peer victimization in adolescence leads to a victim profile characterized by impaired attentional processing with biased social threat perception, a more hostile view of the social world, and blunted affect.</p><p style="font-size: 16px; text-align: justify;">BACKGROUND: Peer victimization (PV) in adolescence--verbal, social, and physical aggression--may disrupt normative developmental processes and is associated with psychosocial difficulties. Although previous research has established associations between early adversity and altered attention and cognitive-affective processing, the specific mechanisms of information-processing through which adolescent PV may affect later social functioning remain unclear. METHODS: We conducted a counterfactual multimethod study in a sample of young adults (age 22; n = 191; 47.1% female), comparing individuals with and without a history of PV between ages 11-20. Specifically we employed: (1) an eye tracking task using a free-viewing paradigm to assess spontaneous threat-related attention; (2) a longitudinal analysis of changes in hostile attribution and reactive aggression using baseline data collected prior to PV exposure and follow-up data in young adulthood; and (3) a morph-based task assessing facial emotion discrimination capabilities between happy and angry expressions. RESULTS: Relative to controls, victims exhibited a vigilance-avoidance attentional pattern and blunted affective responses; they showed reduced initial gaze allocation, repeated returns to angry faces, and a tendency to rate more angry faces as neutral. Over adolescence, a flatter decline in hostile attribution and a steeper drop in reactive aggression led to elevated hostile attribution and reduced reactive aggression. CONCLUSIONS: These results suggest a victim profile characterized by impaired attentional processing with biased social threat perception, a more hostile view of the social world, and blunted affect. Our study offers preliminary evidence that adolescent PV may induce enduring changes in social information processing, with implications for maladaptive social cognition and increased risk for psychopathology. KEYWORDS: peer victimization; adolescence; attention to threat; facial emotion processing; hostile attribution; aggression; eye tracking.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.12.664449v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Conservation of chromatin states and their association with transcription factors in land plants</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shukla, V., Axelsson, E., Hisanaga, T., Haseloff, J., Romani, F., Berger, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify and characterize chromatin states in the flowering plant Arabidopsis thaliana and the bryophyte Marchantia polymorpha, showing a large degree of functional conservation over more than 450 million years of land plant evolution.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The complexity of varied modifications of chromatin composition is integrated in archetypal combinations called chromatin states that predict the local potential for transcription. The degree of conservation of chromatin states has not been established amongst plants, and how they interact with transcription factors is unknown. Here we identify and characterize chromatin states in the flowering plant Arabidopsis thaliana and the bryophyte Marchantia polymorpha, showing a large degree of functional conservation over more than 450 million years of land plant evolution. We used this new resource of conserved plant chromatin states to understand the influence of chromatin states on gene regulation. We established the preferential association of chromatin states with binding sites and activity of transcription factors. These associations define three main groups of transcription factors that bind upstream of the transcription start site, at the +1 nucleosome or further downstream of the transcription start site and broadly associate with distinct biological functions. The association with the +1 nucleosome defines a list of candidate pioneer factors we know little about in plants, compared to their important roles in animal stem cells and early development.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.12.664529v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Complete Telomere-to-Telomere Diploid Reference Genome for Indian Population</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sarashetti, P., Lipovac, J., Jia, Q., Wang, L., Li, Z., Vrcek, L., Yao, F., Sia, Y., Lin, D., Zhang, X., Muliaditan, D., Low, H. M., Leong, S. T., Khor, C. C., Wong, E., Zheng, W., Krizanovic, K., Chambers, J. C., Hwang, W. Y. K., Tan, P., Sikic, M., Liu, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the first gapless, telomere-to-telomere (T2T) diploid genome assembly created by using a trio sample set of Indian ancestry (I002C), with NG50 of 154.89 Mb and 146.27 Mb for the maternal and paternal</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Human reference genomes have been instrumental in advancing genomic and biomedical research, but South and Southeast Asian populations are underrepresented, despite accounting for a large proportion of world population. As a part of effort on generating reference genomes for these populations, we present the first gapless, telomere-to-telomere (T2T) diploid genome assembly created by using a trio sample set of Indian ancestry (I002C), with NG50 of 154.89 Mb and 146.27 Mb for the maternal and paternal haplotypes, including the fully assembled rDNA array for the maternal chromosome 21 and Y chromosome. With the Merqury QVs of 82.05, 83.08 and 82.64 for the maternal, paternal and haploid assemblies respectively, I002C represents the highest-quality human genome assembled in both diploid and haploid forms to date. Compared to CHM13, the I002C genome displays substantial sequence diversity, resulting in 14,943 structural variants, including 3,236 novel variants absent from public databases. Analysis of trio-phased haplotypes further revealed elevated inter-haplotype divergence within centromeric and subtelomeric regions, along with identification of differentially methylated regions (DMRs) as candidates for novel imprinting loci. As a result of substantial SVs between them, I002C is a more suitable reference than CHM13 for the genomic analysis of South Asian samples with less reference bias and better performance in mapping and variant calling, particularly for long read sequencing data. As the first high-quality T2T diploid reference genome for Indian, the largest world's population, I002C contributes to the growing set of population-specific reference genomes and helps to overcome a significant gap in human genome diversity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.12.664550v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">High-Accuracy, Ultrafast DNA Barcode Identification via Statistical Sketching and Approximate Nearest Neighbor Search</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Boone, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> DNA-Sketch resolves the classic speedversus- accuracy trade-off, demonstrating that by pairing robust feature extraction with highperformance ANN indexing, it is possible to achieve the accuracy of gold-standard alignment at a fraction of the computational cost.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> High-throughput DNA barcoding, a cornerstone of modern biodiversity and environmental genomics, is critically limited by the computational cost of traditional, alignment-based identification methods. While faster alignment-free approaches have been proposed, first-generation techniques based on k-mer hashing are fundamentally unreliable due to their inherent sensitivity to insertions and deletions (indels), a common form of sequence variation. Here, we introduce DNA-Sketch, a novel alignment-free framework that overcomes this limitation. DNA-Sketch transforms a DNA sequence into a robust statistical fingerprint by vectorizing its binned dinucleotide frequencies. These high-dimensional "sketches" are then indexed for ultrafast similarity search using an Approximate Nearest Neighbor (ANN) library. We benchmarked a single-pass sketch and a "Multi-Sketch Ensemble" against the state-ofthe- art aligner VSEARCH on a large, challenging benchmark simulating real-world noise and intra-species variation. The Multi-Sketch Ensemble achieved 100% accuracy, perfectly matching VSEARCH, while delivering a 31-fold speed improvement. The single-pass sketch achieved 99.98% accuracy with a 95-fold speedup. DNA-Sketch resolves the classic speedversus- accuracy trade-off, demonstrating that by pairing robust feature extraction with highperformance ANN indexing, it is possible to achieve the accuracy of gold-standard alignment at a fraction of the computational cost, providing a powerful and highly scalable solution for modern bioinformatics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.13.664560v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Bacteriocin Prediction Through Cross-Validation-Based and Hypergraph-Based Feature Evaluation Approaches</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Akhter, S., Miller, J. H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a new model for bacteriocin prediction using hypergraph-based feature evaluation and cross-validation feature evaluation, which outperforms existing approaches.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Bacteriocins offer a promising solution to antibiotic resistance, possessing the ability to target a wide range of bacteria with precision. Thus, there is an urgent need for a computational model to predict new bacteriocins and aid in drug development. This work centers on constructing predictive models with XGBoost machine learning algorithm, using physicochemical structural properties and sequence profiles of protein sequences. We employed correlation analyses, cross-validation, and hypergraph-based techniques to select features. Cross-validation feature evaluation (CVFE) partitions the dataset, selects features within each partition, and identifies common features, ensuring representativeness. On the contrary, hypergraph-based feature evaluation (HFE) focuses on minimizing hypergraph cut conductance, leveraging higher-order data relationships to precisely utilize information regarding feature and sample correlations. The XGBoost models were built using the selected features obtained from these two feature evaluation methods. Our HFE-based approach achieved 99.11% accuracy and an AUC of 0.9974 on the test data, overall outperforming the CVFE-based feature evaluation method and yielding results comparable to existing approaches. We also analyzed the feature contributions directly from the best model using SHapley Additive exPlanations (SHAP). Our web application, accessible at https://shiny.tricities.wsu.edu/bacteriocin-prediction/, offers prediction results, probability scores, and SHAP plots using both cross-validation- and hypergraph-based methods, along with previously implemented approaches for feature selection.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.12.664537v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Evaluating large language models in biomedical data science challenges through a classroom experiment</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ma, H., BIOSTAT 824 Student Consortium,, Ji, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Large language models have shown remarkable capabilities in algorithm design, but their effectiveness in solving data science challenges remains poorly understood.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Large language models have shown remarkable capabilities in algorithm design, but their effectiveness in solving data science challenges remains poorly understood. We conducted a classroom experiment in which graduate students used large language models (LLMs) to solve biomedical data science challenges on Kaggle. While their submissions did not top the leaderboards, their prediction scores were often close to those of leading human participants. LLMs frequently recommended gradient boosting methods, which were associated with better performance. Among prompting strategies, self-refinement, where the LLM improves its own initial solution, was the most effective, a result validated using additional LLMs. These findings demonstrate that LLMs can design competitive machine learning solutions, even when used by non-experts.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.12.664517v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Single-cell profiling of EZH2-mediated immune signaling perturbations in NSCLC</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Plaugher, D. R., Childress, A. R., Gosser, C. M., Esoe, D.-P., Liu, J., Brainson, C. F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that EZH2 inhibition shifts the TME to immunogenic signaling patterns conducive to increased T cell response, including antigen presentation and homing, and improved interferon response, altered myeloid and B cell differentiation, and apoptotic markers.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Lung cancer remains a significant public health burden. One of the most personalized treatments uses a patient's own tumor infiltrating lymphocytes (TILs), and TIL activity is also essential for immune checkpoint inhibitor (ICI) effectiveness. Responses to immunotherapies vary due to immune-suppressive tumor microenvironments (TMEs) and limited antigen presentation. In this study, we computationally examine cell-cell signaling and transcriptional activity using single-cell RNA sequencing of lung cancer treated by inhibiting methyltransferase EZH2. We show that EZH2 inhibition shifts the TME to immunogenic signaling patterns conducive to increased T cell response, including antigen presentation and homing. T cells also showed more stem-like phenotypes. Transcriptional activity was quieter with EZH2 inhibition but revealed better interferon response, altered myeloid and B cell differentiation, and apoptotic markers. Importantly, inferred EZH2 activity showed it could perform non-methyltransferase duties vital for T cell differentiation. These results indicate that EZH2 inhibition could improve immunotherapies for lung cancer patients.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.12.663845v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Characterisation of β-tubulin isotypes in Uncinaria stenocephala and implications for benzimidazole resistance in hookworms</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Stocker, T., Slapeta, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified the coding sequences for key residues in the {beta}-tubulin isotype-1 gene of the hookworm U. stenocephala and confirmed its potential benzimidazole resistance.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Uncinaria stenocephala is a widespread hookworm of dogs across Europe, Canada, southern Australia, and other temperate regions, where it often outnumbers infections caused by Ancylostoma caninum. Although a putative {beta}-tubulin isotype-1 mutation associated with resistance has been detected in U. stenocephala, clinical resistance to benzimidazoles has not yet been confirmed. Benzimidazole resistance is linked to single nucleotide polymorphisms (SNPs) in the {beta}-tubulin isotype-1 gene; however, the {beta}-tubulin genes of U. stenocephala have not been fully characterised. We aimed to identify {beta}-tubulin genes and confirm the coding sequences for key residues (Q134, F167, E198, and F200) in the {beta}-tubulin isotype-1 gene of the U. stenocephala genome. Two U. stenocephala specimens were subjected to Illumina sequencing, and species identity was confirmed through morphological and molecular analysis using ITS rDNA and cox-1 markers. Genome assembly revealed the presence of {beta}-tubulin isotype-1 (10 exons) and isotype-2 (9 exons), both homologous to {beta}-tubulins from other hookworms (A. caninum, A. ceylanicum, A. duodenale and Necator americanus). The {beta}-tubulin isotype-1 protein sequence of U. stenocephala contained two variable residues (S37Q and G441A) compared to other hookworm sequences. While the isotype-2 protein sequence was conserved among Ancylostoma species, U. stenocephala exhibited six distinct polymorphisms (E39D, T40S, N115S, L130I, A287S, T439G). The benzimidazole-susceptible residues (Q134, F167, E198, F200) were present in the {beta}-tubulin isotype-1 protein sequence. Characterisation of the complete coding regions of {beta}-tubulin isotypes 1 and 2 enables population-level screening for benzimidazole resistance-associated SNPs and provides a foundation for future epidemiological studies in U. stenocephala.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.12.664546v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Mapping the Metalloproteome of Deinococcus indicus DR1 through Integrative Structure and Function Annotation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ramesh, S. D., Vasan, G., Senthilkumar, S., Thambiraja, M., Sethi, D., Priyadarshini, R., Yennamalli, R. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A proteome-wide structural and functional survey of the deinococcus indicus DR1 bacterium that tolerates arsenic and other heavy metals.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Deinococcus indicus DR1 is a rod-shaped bacterium isolated from the Dadri wetlands (Uttar Pradesh, India) that tolerates ionizing radiation and arsenic. The molecular basis of its wider heavy-metal resilience, particularly among the 1017 out of 4128 proteins still annotated as hypothetical, remains unclear. We performed a proteome-wide structural and functional survey to address this gap. All the 4128 proteins were modeled with AlphaFold2, yielding very-high-confidence structures (pLDDT [≥] 90) for 2145 sequences. CATH/InterPro analysis assigned domains to 2735 proteins. Functions were predicted by combining DeepFRI (graph neural-network), MorphologFinder (Foldseek plus EggNOG-Mapper), and existing GenBank annotations. The integrated workflow suggests that more than 100 previously uncharacterized proteins may bind or transport arsenic, chromium, cobalt, copper, iron, manganese, molybdenum, nickel, or zinc, indicating a metal-handling capacity that extends beyond the known ars operon. Recurrent domain architectures that include P-loop NTPases, Rossmann folds, GNAT acetyl-transferases, and sensor modules (CHASE, PAS, GAF) point to coordinated redox regulation and efflux pathways. Twenty high-confidence metal-binding candidates have been prioritized for experimental validation through expression, mutagenesis, and knockout studies. All structural models, domain assignments, and query tools are available at https://deinococcus.in, providing a resource for future investigations of heavy-metal tolerance in this organism.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.11.664487v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Breaking the Salinity-Nitrogen Fixation Trade-Off: Engineering A Synthetic Nitrogen-Fixing Vibrio natriegens Strain</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wu, W., Tang, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a nitrogen-fixing Vibrio natriegen bacteria that enhances plant-microbe interactions through reprogrammed nitrogen metabolism.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Inefficient nitrogen fixation in legumes under saline stress threatens food security, while conventional symbiotic nitrogen fixation is limited by host specificity and saline stress inhibition. This study engineered Vibrio natriegens with nitrogen fixation genes, creating a saline-tolerant, broad-host-range nitrogen-fixing bacteria that enhances plant-microbe interactions through reprogrammed nitrogen metabolism. We demonstrated that engineered nitrogen-fixing V. natriegen significantly enhanced soybean growth and nodulation under saline stress. It also activated nitrogenase gene (nifHDK) expression in rhizosphere bacteria and increased the abundance of rhizosphere diazotrophs. This study presents a novel approach for developing stress-resilient crop-microbe symbioses, offering a sustainable solution to improve crop growth under saline stress.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.16.665069v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Harnessing Dynamic Cell Responses to Optimise Pluripotency Protocols</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> La Regina, A., Pedone, E., Marucci, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We explore the possibility to avoid continuous drug exposure to pluripotent stem cells by controlling the expression of a pluripotency reporter, and then harnessing such delays to design novel cell culture protocols with limited cell exposure to small-molecule inhibitors.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Pluripotent stem cells have a high potential for research and development applications in biomedicine. They are undifferentiated cells capable of unlimited self-renewal and maintenance of pluripotency, yet can be induced to differentiate into specific cell types through defined cues. Various protocols have been proposed to maintain mouse embryonic stem cells (mESCs) in the pluripotent state; state of the art cell culture protocols are based on serum-free media, and continuous provision of drugs inhibiting pathways associated to differentiation. We explore here the possibility to avoid continuous drug exposure. First, we employ an external feedback control microfluidics/microscopy platform to steer the expression of a pluripotency reporter. Control experiments show a significant delay in cell response to drugs; these dynamics are confirmed by flow-cytometry open-loop experiments. We then harness such delays to design novel culture protocols with limited cell exposure to small-molecule inhibitors; in so doing, we reduce cell exposure to drug, but we maintain pluripotency. Our results suggest new approaches to design cell culture protocols based on the quantitative analysis and control of cellular dynamics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.17.665308v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Synthetic Aptamer Mechanoreceptors Enable Cell-SpecificForce Sensing and Temporal Control via DNA Circuits</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xu, T., Sethi, S., Drees, C., Walther, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce an all-DNA mechanosensing platform based on aptamers that transmit force through noncanonical surface receptors, enabling the design of reversible and temporally programmable mechanoresponses.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cells interpret mechanical cues from their microenvironment with spatiotemporal precision to guide adaptive behaviors. However, engineering synthetic mechanosensing systems with both cell-specificity and programmability remains challenging, especially when targeting ubiquitous classical mechanoreceptors. Here, we introduce an all-DNA mechanosensing platform based on aptamers that transmit force through noncanonical surface receptors. Aptamer-receptor recognition acts as a molecular gate for force transduction, enabling the design of mechanoprobes with dual selectivity; by force magnitude and cell type. These probes interpret diverse mechanical inputs via distinct mechanisms, including actomyosin-driven contractility relayed through focal adhesions and membrane ruffling during macropinocytosis. By integrating aptamer mechanoprobes with upstream DNA reaction networks, we achieve reversible and temporally programmable mechanoresponses. This modular, all-nucleic-acid system offers a general framework for constructing tunable mechanotransduction circuits. It expands the design space for synthetic mechanobiology and provides new opportunities for autonomous, multi-layered mechanical-biochemical regulation in tissue engineering, morphogenesis, and dynamic cell programming.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.16.665263v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

