<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Thursday, 03 July 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Deep learning deciphers the related role of master regulators and G-quadruplexes in tissue specification.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Artem Bashkatov, Andrey Andreasyan, Dmitry Konovalov, Alan Herbert, Maria Poptsova</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a deep learning-based framework for the genome-wide G-flipon predictions across 14 human tissue types.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> G-quadruplexes (GQs) are non-canonical DNA structures encoded by G-flipons with potential roles in gene regulation and chromatin structure. Here, we explore the role of G-flipons in tissue specification. We present a deep learning-based framework for the genome-wide G-flipon predictions across 14 human tissue types. The model was trained using high-confidence experimental maps of GQ-forming sequences and ATAC-seq peaks, conjoined with the location of RNA polymerase, histone marks, and transcription factor binding sites. The training dataset for the DeepGQ model was derived from EndoQuad level 4-6 GQs. Model predictions were subsequently validated against the comprehensive EndoQuad dataset (levels 1-6) to optimize the whole-genome prediction threshold. To identify tissue-specific regulatory patterns, we classified GQ promoter predictions as either 'core' or 'tissue-specific'. We identified a notable overlap between predicted unique tissue-specific GQ sites and master regulatory genes (MRGs), tissue-specific DNase-hypersensitivity sites, and proteins that modulate R-loop formation. Collectively, the findings highlight the transactions between MRG and G-flipons intermediated by RNA: DNA hybrids associated with tissue specification.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40603503/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Shap-Cov: An Explainable Machine Learning Based Workflow for Rapid Covariate Identification in Population Modeling.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Logan Brooks, Rashed Harun, Jin Y Jin, James Lu</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a formal framework for covariate identification in population pharmacokinetic/pharmacodynamic (popPK/PD) modeling and a flexible set of functions (shap-cov) to support covariate screening.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Covariate identification in population pharmacokinetic/pharmacodynamic (popPK/PD) modeling is a key component in model development that is often prone to bias, time-consuming, and even intractable when too many covariates or complicated models are being considered. Early work leveraging machine learning (ML) for covariate screening has shown promising results over traditional methods. In this work, we expand this effort by integrating explainable machine learning facilitated by Shapley Additive Explanations (SHAP) analysis and covariate uncertainty quantification as well as a formal framework for establishing statistical significance of covariate relationships. Finally, we have packaged the proposed methodology into a flexible set of functions (shap-cov) to support popPK/PD modeling covariate identification.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40601439/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Leveraging protein language models for cross-variant CRISPR/Cas9 sgRNA activity prediction.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yalin Hou, Yiming Li, Ruiqing Zheng, Fuhao Zhang, Fei Guo, Min Li, Min Zeng</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a new method to predict single-guide RNA activity in the Cas9 gene-editing system .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurate prediction of single-guide RNA (sgRNA) activity is crucial for optimizing the CRISPR/Cas9 gene-editing system, as it directly influences the efficiency and accuracy of genome modifications. However, existing prediction methods mainly rely on large-scale experimental data of a single Cas9 variant to construct Cas9 protein (variants)-specific sgRNA activity prediction models, which limits their generalization ability and prediction performance across different Cas9 protein (variants), as well as their scalability to the continuously discovered new variants.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40600900/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrating machine learning and bioinformatics approaches to identify novel diagnostic gene biomarkers for diabetic mice.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Weibin Wu, Zheng Peng, Mingyi Chen, Yi Yu, Caisheng Wu, Qiang Xie</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified 45 overlapping genes associated with streptozotocin-induced diabetes in aged diabetic mice and identified three genes as diagnostic genes for the diagnosis and treatment of diabetes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Diabetes is a complex metabolic disorder, and its pathogenesis involves the interplay of genetic, environmental factors, and lifestyle choices. With the rising prevalence and increasing associated chronic complications, identifying and understanding the molecular mechanisms of diabetes has become an important direction in bioinformatics research. The aim of this study is the identification of diagnostic genes associated with streptozotocin (STZ)-induced diabetic mice. GSE179717 and GSE179718 gene expression datasets were downloaded from the Gene Expression Omnibus (GEO) database. Differentially expressed genes (DEGs) were identified by screening diabetic mice and controls. A total of 45 overlapping genes were recognized as potential DEGs across the two datasets. Next, Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) analysis were performed on the overlapping genes. Protein-protein interaction (PPI) networks were constructed to identify hub genes in the DEGs. After the application of Least absolute shrinkage and selection operator (LASSO), Random Forest analysis, three genes (Scd1, Sirt1 and Hmgb1) were identified as diagnostic genes. Real-time quantitative polymerase chain reaction (RT-qPCR) was supplied to detect the expressions of diagnostic genes in aged diabetic mice. In conclusion, the results of this study suggest that focusing on these genes may provide new targets for the diagnosis and treatment of diabetes.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40595011/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Benchmarking Large Language Models for Pathogen-Disease Classification in Post-Acute Infection Syndromes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Khalid, S. M., Woelker, T., Molano, L.-A. G., Graf, S., Keller, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Large language models can be used to identify pathogen-disease associations in the literature of post-Acute Infection Syndromes, but few-shot, few- shot, and Chain of Thought methods can improve classification performance.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Post-Acute Infection Syndromes (PAIS) are medical conditions that persist following acute infections from pathogens such as SARS-CoV-2, Epstein-Barr virus, and Influenza virus. Despite growing global awareness of PAIS and the exponential increase in biomedical literature, only a small fraction of this literature pertains specifically to PAIS, making the identification of pathogen-disease associations within such a vast, heterogeneous, and unstructured corpus a significant challenge for researchers. This study evaluated the effectiveness of large language models (LLMs) in extracting these associations through a binary classification task using a curated dataset of 1,000 manually labeled PubMed abstracts. We benchmarked a wide range of open-source LLMs of varying sizes (4B-70B parameters), including generalist, reasoning, and biomedical-specific models. We also investigated the extent to which prompting strategies such as zero-shot, few-shot, and Chain of Thought (CoT) methods can improve classification performance. Our results indicate that model performance varied by size, architecture, and prompting strategy. Zero-shot prompting produced the most reliable results, with Mistral-Small-Instruct-2409 and Nemotron-70B achieving strong balanced accuracy scores of 0.81 and macro F1 scores of up to 0.80, while maintaining minimal invalid outputs. While few-shot and CoT prompting often degraded performance in generalist models, reasoning models such as DeepSeek-R1-Distill-Llama-70B and QwQ-32B demonstrated improved accuracy and consistency when provided with additional context.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.30.662395v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">XtractPAV: An Automated Pipeline for Identifying Presence-Absence Variations Across Multiple Genomes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ahmad, R. S., Sadaqat, M., Tahir ul Qamar, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed XtractPAV, an end-to-end pipeline that automates the extraction, annotation, and interactive visualization of presence-absence variations across large-scale genomic datasets.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: Presence-absence variations (PAVs) significantly influence phenotypic diversity across and within species by modulating functional modules involved in stress responsiveness, adaptation, and developmental processes. This modulation ultimately contributes to genetic diversity at both inter- and intra-species levels. However, existing tools for detecting PAVs offer limitations in achieving optimal analysis because they lack scalable workflows for multi-genome comparisons and frequently necessitate manual integration. To address these challenges, we developed XtractPAV, an end-to-end pipeline that automates the extraction, annotation, and interactive visualization of PAVs across large-scale genomic datasets. Results: XtractPAV was evaluated using assembled genomes of both eukaryotic and prokaryotic organisms, including Pyrus communis, Arabidopsis thaliana, Mus musculus, and Salmonella enterica, to assess its ability to detect the genomic variations across diverse species. The performance of XtractPAV was benchmarked against other established pipelines, demonstrating superior precision and a more comprehensive extraction of PAV segments. Notably, our pipeline not only identified the known PAVs from the reference set but also revealed novel variations in genes associated with various functions such as flowering time regulation and disease resistance. Furthermore, the automated report generation feature of XtractPAV produces publication-ready summaries of PAV distributions and related metrics. Availability: XtractPAV is freely accessible at https://github.com/SherazAhmadd/XtractPAV and on the XtractPAV webpage. The Package includes all requisite files, a user manual, test data, and a license permitting non-commercial use.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.27.661953v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MORPH Predicts the Single-Cell Outcome of Genetic Perturbations Across Conditions and Data Modalities</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> He, C., Zhang, J., Dahleh, M. A., Uhler, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present MORPH, a MOdular framework for predicting Responses to Perturbational cHanges.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Modeling cellular responses to genetic perturbations is a significant challenge in computational biology. Measuring all gene perturbations and their combinations across cell types and conditions is experimentally challenging, highlighting the need for predictive models that generalize across data types to support this task. Here we present MORPH, a MOdular framework for predicting Responses to Perturbational cHanges. MORPH combines a discrepancy-based variational autoencoder with an attention mechanism to predict cellular responses to unseen perturbations. It supports both single-cell transcriptomics and imaging outputs and can generalize to unseen perturbations, combinations of perturbations, and perturbations in new cellular contexts. The attention-based framework enables inference of gene interactions and regulatory networks, while the learned gene embeddings can guide the design of informative perturbations. Overall, we envision MORPH as a flexible tool for optimizing perturbation experiments, enabling efficient exploration of the perturbation space to advance understanding of cellular programs for fundamental research and therapeutic applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.27.661992v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">nf-core/viralmetagenome: A Novel Pipeline for Untargeted Viral Genome Reconstruction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Klaps, J., Lemey, P., nf-core community,, Kafetzopoulou, L. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present Viralmetagenome, a comprehensive viral metagenomic pipeline for untargeted genome reconstruction and variant analysis of eukaryotic DNA and RNA viruses.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: Eukaryotic viruses present significant challenges for genome reconstruction and variant analysis due to their extensive diversity and potential genome segmentation. While de novo assembly followed by reference database matching and scaffolding is a commonly used approach, the manual execution of this workflow is extremely time-consuming, particularly due to the extensive reference curation required. Here, we address the critical need for an automated, scalable pipeline that can efficiently handle viral metagenomic analysis without manual intervention. Results: We present nf-core/viralmetagenome, a comprehensive viral metagenomic pipeline for untargeted genome reconstruction and variant analysis of eukaryotic DNA and RNA viruses. Viralmetagenome is implemented as a Nextflow workflow that processes short-read metagenomic samples to automatically detect and assemble viral genomes, while also performing variant analysis. The pipeline features automated reference selection, consensus quality control metrics, comprehensive documentation, and seamless integration with containerization technologies, including Docker and Singularity. We demonstrate the utility and accuracy of our approach through validation on both simulated and real datasets, showing robust performance across diverse viral families in metagenomic samples. Availability: nf-core/viralmetagenome is freely available at https://github.com/nf-core/viralmetagenome with comprehensive documentation at https://nf-co.re/viralmetagenome.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.27.661954v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Uncovering smooth structures in single-cell data with PCS-guided neighbor embeddings</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ma, R., Li, X., Hu, J., Yu, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce NESS, a principled and interpretable machine learning approach to improve NE representations by leveraging algorithmic stability and to enable robust inference of smooth biological structures.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell sequencing is revolutionizing biology by enabling detailed investigations of cell-state transitions. Many biological processes unfold along continuous trajectories, yet it remains challenging to extract smooth, low-dimensional representations from inherently noisy, high-dimensional single-cell data. Neighbor embedding (NE) algorithms, such as t-SNE and UMAP, are widely used to embed high-dimensional single-cell data into low dimensions. But they often introduce undesirable distortions, resulting in misleading interpretations. Existing evaluation methods for NE algorithms primarily focus on separating discrete cell types rather than capturing continuous cell-state transitions, while dynamic modeling approaches rely on strong assumptions about cellular processes and specialized data. To address these challenges, we build on the Predictability--Computability--Stability (PCS) framework for reliable and reproducible data-driven discoveries. First, we systematically evaluate popular NE algorithms through empirical analysis, simulation, and theory, and reveal their key shortcomings such as artifacts and instability. We then introduce NESS, a principled and interpretable machine learning approach to improve NE representations by leveraging algorithmic stability and to enable robust inference of smooth biological structures. NESS offers useful concepts, quantitative stability metrics, and efficient computational workflows to uncover developmental trajectories and cell-state transitions in single-cell data. Finally, we apply NESS to six single-cell datasets, including those about pluripotent stem cell differentiation, organoid development, and multiple tissue-specific lineage trajectories. Across these diverse contexts, NESS consistently yields useful and verifiable biological insights, such as identification of transitional and stable cell states and quantification of transcriptional dynamics during development. Notably, NESS resolves distinct neuronal subpopulations during embryoid formation and provides a deeper understanding of their cell-state dynamics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.27.661958v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Confidence: A Web App for Cross-Platform Differential Gene Expression Analysis, Gene Scoring, and Enrichment Analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shastry, A., Ott, B., Paterson, A., Simpson, M., Dunham-Snary, K. J., Hindmarch, C. C. T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Confidence is a web-based, 'wide-net' approach to differential gene expression analysis that provides a new strategy for target prioritization in RNA-seq analysis and the generation of publication-quality figures.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> RNA sequencing (RNA-seq) is used to quantify transcript levels through measurement of nucleotide sequences. To evaluate statistically significant changes in gene expression, transcript counts between samples are compared using differential expression analysis methods. However, three of the most pressing challenges in transcriptomics analyses are: 1) analytical packages produce a distinct number of differentially expressed genes with varied P-value and fold-change values; 2) the effective use of these analytical packages requires substantial knowledge of programming and bioinformatics; and 3) there are a lack of intuitive methods to select target genes for further investigation in an unbiased manner. To address these challenges, we developed Confidence, a web-based application to perform simultaneous statistical analysis of RNA-seq count data. Confidence incorporates the Confidence Score (CS), ranging from 1 to 4 to aid in gene prioritization, where 1 represents low confidence and 4 represents high confidence. The Confidence web-based application was designed for rapid and intuitive analysis of standard experimental metadata and gene count inputs. Confidence provided a web-based, 'wide-net' approach to differential gene expression analysis. Gene scoring allows for unbiased gene selection and identification of novel genes strongly associated with disease and treatment models across multiple species. Additionally, pathway analysis tools have been integrated so that highly confident genes can be placed into biological context in terms of functions and pathways. Confidence provides a new strategy for target prioritization in RNA-seq analysis and the generation of publication-quality figures.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.27.661997v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Persistent Activation of Endothelial Cells is Linked to Thrombosis and Inflammation in Cerebral Cavernous Malformation Disease</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gallego-Gutierrez, H., Frias-Anaya, E., Bui, C., Zhao, L., Hsu, E., Indralingam, H. S., Korbelin, J., Trejo, J., Steinberg, J., Zemke, N. R., Lopez-Ramirez, M. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We found that epigenetics significantly influences the subtype identity and function of brain endothelial cells within the arteriovenous axis within Cerebral cavernous malformations (CCM) disease and contribute to the progression of chronic neuroinflammation.</p><p style="font-size: 16px; text-align: justify;">BACKGROUND: Cerebral cavernous malformations (CCM) are neurovascular lesions that affect both children and adults, and morbidity often results from thrombosis, bleeding, and neurological dysfunction. Studies indicate that inflammation-related activation of endothelial cells contributes significantly to the worsening of CCM disease. This suggests that ongoing vascular inflammation and endothelial dysfunction are key factors associated with thrombosis and bleeding in CCM disease. However, the inflammatory mechanisms leading to altered brain endothelial cell function with a high propensity for thrombosis, inflammation, and dysfunction are not fully understood. METHODS: Multi-omic analyses was conducted by performing simultaneous high-throughput single-nucleus RNA sequencing (snRNA-seq) and single-nucleus transposase-accessible chromatin sequencing (snATAC-seq) with the 10x Genomics multiome platform in combination with immunofluorescence to study CCM pathogenesis in both female and male mice with CCM (Slco1c1-CreERT2; Pdcd10fl/fl) disease. The analysis was complemented with bulk RNA-seq, bulk ATAC-seq, and ChIP-seq (Chromatin immunoprecipitation sequencing) using an in vitro human CCM model. An AAV-BR1 viral system selectively upregulates the activator protein-1 (AP-1) transcription factor JUNB in brain endothelial cells was used to evaluate its effectiveness in maintaining a persistent activated cell state during the pathogenesis of CCM. RESULTS: We found that epigenetics significantly influences the subtype identity and function of brain endothelial cells within the arteriovenous axis. Through multi-omic analyses, specific regulatory elements and enhancers (cis-Regulatory Elements, cCREs) in mouse brain endothelial cells were identified that influence subtype-specific transcriptional programs and the transcription factors responsible for establishing the various subtypes of brain endothelial cells. Additionally, large-scale epigenomic reprogramming of brain endothelial cell subtypes was observed during the pathogenesis of CCM disease. Among the most significant changes were alterations in the chromatin state of endothelial cells, along with transcriptional processes associated with a persistently activated endothelial cell state, which renders them susceptible to inflammation and thrombosis. The activator AP-1 transcription factor JUNB was identified as a key regulator of the persistently activated endothelial state during chronic neuroinflammation. Moreover, both trans- and cis-regulatory factors conserved between mice and humans were discovered and contribute to the progression of chronic CCM disease. CONCLUSIONS: Epigenetics plays a crucial role in determining the transcription patterns and functions of brain arteriovenous endothelial cells. The activator JUNB is identified as a driver of chronic brain vascular inflammation by inducing a persistent activated endothelial cell state from epigenome reprogramming.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.29.662238v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multi-season analysis reveals hundreds of drought-responsive genes in sorghum</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cole, B., Zhang, W., Shi, J., Wang, H., Baker, C., Varoquaux, N., Hollingsworth, J., Hutmacher, R., Dahlberg, J., Pierroz, G., Barry, K. W., Singan, V., Yoshinaga, Y., Daum, C., Zane, M., Blow, M., OMalley, R., Shu, S., Jenkins, J. W., Lovell, J. T., Schmutz, J., Taylor, J. W., Coleman-Derr, D., Visel, A., Lemaux, P. G., Purdom, E., Vogel, J. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified a set of drought-responsive genes that responded similarly in all three years of our field study and identified a pan-gene set for characterizing conserved and genotype-specific expression.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Persistent drought affects global crop production and is becoming more severe in many parts of the world in recent decades. Deciphering how plants respond to drought will facilitate the development of flexible mitigation strategies. Sorghum bicolor L. Moench (sorghum), a major cereal crop and an emerging bioenergy crop, exhibits remarkable resilience to drought. To better understand the molecular traits that underlie sorghum's remarkable drought tolerance, we undertook a large-scale sorghum gene expression profiling effort, totaling nearly 1,500 transcriptome profiles, across a 3-year field study with replicated plots in California's Central Valley. This study included time-resolved gene expression data from roots and leaves of two sorghum genotypes, BTx642 and RTx430, with different pre-flowering and post-flowering drought-tolerance adaptations under control and drought conditions. Quantification of genotype-specific drought tolerance effects was enabled by de novo sequencing, assembly, and annotation of both BTx642 and RTx430 genomes. These reference-quality genomes were used to construct a pan-gene set for characterizing conserved and genotype-specific expression. By integrating time-resolved transcriptomic responses to drought in the field across three consecutive years, we identified a set of drought-responsive genes that responded similarly in all three years of our field study. This expansive dataset represents a unique resource for sorghum and drought research communities and provides a methodological framework for the integration of multi-faceted time-resolved transcriptomic datasets.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.27.662006v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Title: KymoTip: High-throughput Characterization of Tip-growth Dynamics in Plant Cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kang, Z., Kimata, Y., Nonoyama, T., Ikeuchi, T., Kuchitsu, K., Satoru, T., Ueda, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a powerful and user-friendly tool called KymoTip with an available format for live imaging data analysis of cell dynamics .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Live imaging data analysis often requires an objective, local, and accurate way of quantification of cell dynamics. In the research field of polarized tip-growth, the cell fluctuations and/or fluctuations in tip position and growth direction hampers automated analyses of huge amounts of imaging sequences. The fluctuated nature in data makes it unclear how cell shape and growth are linked to intracellular events that could be the actual driving force of cell growth. To overcome these difficulties, we developed a powerful and user-friendly tool called KymoTip with an available format. In this software, novel functions such as coordinate normalization, tip-bottom detection, and signal kymograph were implemented. We confirmed that not only plasma membrane-labeled fluorescent images, but also images such as bright-field and cortical microtubule markers -so long as the cell contours can be identified- are amenable to KymoTip. Furthermore, by combining markers for cell contours with those that visualize intracellular structures, it becomes possible to quantitatively analyze various intracellular events, such as nuclear migration and calcium wave, in conjunction with cellular growth dynamics. Since KymoTip can be handled by non-specialist, it is expected to promote understanding of what happens at the sub- and cellular level with high throughput outcomes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.27.661917v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ATG8i Autophagy activation is mediated by cytosolic Ca2+ under osmotic stress in Arabidopsis thaliana</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Castillo-Olamendi, L., Gutierrez-Martinez, J., Jimenez-Nopala, G., Galindo, A., Barrera-Ortiz, S., Rosas-Santiago, P., Cordoba, E., Leon, P., Porta, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Autophagy is a highly conserved catabolic process in eukaryotic cells that enables the degradation of damaged or unnecessary cytoplasmic components.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Autophagy is a highly conserved catabolic process in eukaryotic cells that enables the degradation and recycling of damaged or unnecessary cytoplasmic components. It plays essential roles in both development and responses to environmental stress. In this study, we investigated the regulation of autophagy in response to osmotic stress, focusing on the dynamics of the RFP-tagged ATG8i protein and the potential involvement of cytosolic calcium ion (Ca) in this process. Our findings indicate that both osmotic stress and (Ca); signaling modulate the accumulation of RFP-ATG8i-labeled autophagosomes in a plant organ-specific manner. Furthermore, the observed colocalization of RFP-ATG8i with the endoplasmic reticulum (ER) marker HDEL suggests a significant role for ATG8i in ER-phagy, highlighting its potential contribution to ER turnover under stress conditions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.02.662772v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Resolving the Foliar Calcium Mobility Paradox: Enhancing Foliar Calcium Transport in Tomato Using Osmotic Regulators</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Santos, E., Montanha, G. S., da Silva, H. J. F. A., da Silva, N. G. d. C., Franco, F. S., Brandao, J. R., Huatatoca, K. A. C., Virgilio, A., Karunakaran, C., Tanino, K. K., Lavres, J., Pereira De Carvalho, H. W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We studied the transport and penetration of foliar-applied Ca using strontium as a physiological tracer, and the influence of osmotic regulators, sucrose, mannitol, glycerol, and potassium, on Ca transport.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Calcium (Ca) deficiency can impair fruit development even under optimal soil Ca levels due to its transpiration-dependent transport. Reduced fruit transpiration may limit Ca delivery to fruits, leading to lower Ca content in the fruit and diminished quality. Foliar application of Ca offers a potential strategy to mitigate these effects; however, its low mobility in the phloem often limits treatment efficacy. To address this, we employed X-ray fluorescence spectroscopy (XRF) to investigate the penetration and transport of foliar-applied Ca, using strontium (Sr) as a physiological tracer. Additionally, we evaluated the influence of osmotic regulators, sucrose, mannitol, glycerol, and potassium, on Ca transport. Results showed that Sr was effectively translocated to distal tissues. While potassium and mannitol had no significant impact on transport kinetics, sucrose and glycerol enhanced Sr movement. XRF imaging of leaf tissue revealed that Sr was primarily transported through the apoplast toward the leaf margin. Moreover, foliar application of Sr combined with sucrose significantly increased Sr accumulation in seeds and in the apical portion of tomato fruits. These findings suggest that, contrary to the common assumption of limited foliar Ca mobility, sucrose, can acts as an effective osmotic regulator, enhancing both short-range movement within leaf tissue and long-distance translocation to fruit.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.02.662739v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Machine learning-based structural classification of lytic polysaccharide monooxygenases</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Manikandan, A., Yennamalli, R. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We will train a machine learning model to identify and classify LPMO with the help of structural features by either a neural network algorithm or other suitable methods.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Lytic polysaccharide monooxygenase (LPMO) is a copper-dependent redox enzyme and according to CAZy is classified either as cellulolytic or chitinolytic. According to CAZy, there are eight families of LPMO namely AA9, AA10, AA11, AA13, AA14, AA15, AA16, and AA17, where AA stands for Auxiliary Activity. Previously, using the sequence information machine learning-based functional annotation was successfully completed using neural network and LSTM models. This was done for AA9 and AA10 as the number of sequences was large enough to train a model with high-performance indices. Here, the goal is to use existing 3D structures and AI-based models of the remaining LPMO sequences and train a machine learning model to identify and classify LPMO with the help of structural features by performing either a neural network algorithm or other suitable methods. Using the features of LPMO such as surface depth, accessible area, electrostatic charge distribution, and geometric features (independent features that define the shape and are not based on enzyme reaction mechanism) we will identify the features with high signal-to-noise ratio/significance using ensemble feature selection method. The features will be extracted using Pymol, MdTraj, Biopython, Open3d, and Bio3D tools from OBJ and STL format. The model thus trained using structural features will enable identifying and annotating newer LPMO sequences belonging to one of the eight AA families.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.02.662747v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

