<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Wednesday, 24 September 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Next-generation translational genomics for developing future crops.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Udita Basu, Swarup K Parida</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Translational genomics is a rapidly evolving field of genetic and transcriptomic research that is enabling rapid and efficient crop improvement.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Advancements in translational genomics have revolutionized crop breeding, driving us from traditional breeding methods towards next-generation strategies that integrate genomic, transcriptomic, and phenotypic data to expedite crop improvement. There has been a shift from single genomes to pan-genomes, which better capture intraspecific diversity, and from bulk transcriptome analyses to single-cell transcriptomics, enabling cell-specific insights into gene regulation and functional genomics. Both high throughput genopyting and phenotyping approaches are now possible due to rapid technological advancement in the field of translational genomics. Large-scale phenotyping data from multi-environment field trials is now possible due to AI-enabled digital and drone-based scanning. In the era of artificial intelligence and machine learning we have developed flexible models to handle complex genetic architecture of trait regulation using various tools and approaches. These genetic and genomic resources are the foundation for generating novel, adaptable, and high-yielding varieties, accelerating trait discovery and mapping. This review explores the comprehensive landscape of modern translational genomics, highlighting key shifts and innovations that enhance our capacity to address agricultural challenges. Integrative pipelines that unify these next-generation approaches could facilitate faster, more precise, and sustainable crop improvement, ultimately meeting the growing demands for future-ready crops.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40987990/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The Implications of Alternative Splicing Regulation for Maximum Lifespan</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zheng, S., Chen, L., Jiang, W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A cross-species analysis of alternative splicing across six tissues in 26 mammals, identifying hundreds of conserved AS events significantly associated with MLS, with the brain containing twice as many tissue-specific events as peripheral tissues.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mammalian maximum lifespan (MLS) varies over a hundred-fold, yet the molecular mechanisms underlying this diversity remain unclear. We present a cross-species analysis of alternative splicing (AS) across six tissues in 26 mammals, identifying hundreds of conserved AS events significantly associated with MLS, with the brain containing twice as many tissue-specific events as peripheral tissues. MLS-AS events are enriched in pathways related to mRNA processing, stress response, neuronal functions, and epigenetic regulation, and are largely distinct from genes whose expression correlates with MLS, indicating that AS captures unique lifespan-related signals. The brain exhibits certain associations divergent from peripheral tissues and reduced overlap with body mass (BM)-associated splicing; neither is observed at the gene expression level. While MLS- and age-associated AS events show limited overlap, the shared events are enriched in intrinsically disordered protein regions, suggesting a role in protein flexibility and stress adaptability. Furthermore, MLS-associated AS events display stronger RNA-binding protein (RBP) motif coordination than age-associated ones, highlighting a more genetically programmed adaptation for lifespan determination, in contrast to the more variable splicing changes seen with chronological aging. These findings suggest alternative splicing as a distinct, transcription-independent axis of lifespan regulation, offering new insights into the molecular basis of longevity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.23.678147v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Effectiveness of CRISPR-Cas in Sensitizing Bacterial Populations with Plasmid-Encoded Antimicrobial Resistance.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Johannes Kippnich, Fabienne Benz, Hildegard Uecker, Franz Baumdicker</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A model to test the effectiveness of CRISPR-Cas in sensitizing bacterial populations carrying resistance on non-transmissible plasmids and assess the success probability of a subsequent treatment with conventional antibiotics.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The spread of bacteria resistant to antibiotics poses a serious threat to human health. Genes that encode antibiotic resistance are often harbored on plasmids, extra-chromosomal DNA molecules found in bacteria. The emergence of multiresistance plasmids is particularly problematic and demands the development of new antibiotics and alternative strategies. CRISPR-Cas derived tools with their sequence specificity offer a promising new approach to combating antibiotic resistance. By introducing CRISPR-Cas encoding plasmids that %specifically target antibiotic resistance genes on plasmids, the susceptibility of bacteria to conventional antibiotics can be restored. However, genetic variation within bacterial populations can hinder the effectiveness of such CRISPR-Cas tools by allowing some mutant plasmids to evade CRISPR-mediated cleaving or gene silencing. In this study, we develop a model to test the effectiveness of CRISPR-Cas in sensitizing bacterial populations carrying resistance on non-transmissible plasmids and assess the success probability of a subsequent treatment with conventional antibiotics. We evaluate this probability according to the target interference mechanism, the copy number of the resistance-encoding plasmid, and its compatibility with the CRISPR-Cas encoding plasmid. Our results identify promising approaches to revert antibiotic resistance with CRISPR-Cas encoding plasmids: A DNA-cleaving CRISPR-Cas system on a plasmid incompatible with the targeted plasmid is most effective for low copy numbers, while for resistance plasmids with higher copy numbers gene silencing by CRISPR-Cas systems encoded on compatible plasmids is the superior solution.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40985758/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Illuminating the Virosphere's Dark Matter using Hierarchical Deep Learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cao, C., He, L., Li, C., Jiang, Y., Tang, C., Li, Y., He, Y., Min, Y., Liu, H., Qin, T., Liu, T.-Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> DeepVirus integrates deep learning with genome-aware open-set discovery to illuminate viral dark matter, providing a foundation for systematic viral taxonomy and advancing exploration of the global virosphere.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Systematic discovery of novel viruses is essential for pandemic preparedness, understanding tumor-associated viruses, developing viral delivery systems, and advancing biomedical applications. Yet, the majority of sequences in metagenomic datasets lack close relatives in existing references, representing a vast viral "dark matter" whose biology and evolution remain largely unknown. The central task is threefold: 1) to determine whether a genome is viral or non-viral, 2) to correctly assign viral genomes to known lineages when possible, and, critically, 3) to recognize when no existing lineage applies and thereby identify candidates for entirely novel viral groups. Existing approaches, which depend on sequence homology or narrow markers, struggle to capture this uncharted viral space. Here we present DeepVirus, a hierarchical transformer-based framework that models viral genomes as structured sequences of protein-coding genes. By combining protein-level embeddings from a foundation model with genome-aware representations, DeepVirus not only achieves accurate classification across deep taxonomic hierarchies, but also extends beyond conventional classification to detect and organize candidate novel viral lineages through open-set recognition. Applied to large-scale metagenomic resources, DeepVirus uncovered extensive viral diversity, including previously uncharacterized RNA-dependent RNA polymerases (RdRps), thereby expanding the known evolutionary space of RNA viruses. DeepVirus integrates deep learning with genome-aware open-set discovery to illuminate viral dark matter, providing a foundation for systematic viral taxonomy and advancing exploration of the global virosphere, with broad implications for safeguarding human health.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.677955v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PRIME: A Multi-Agent Environment for Orchestrating Dynamic Computational Workflows in Protein Engineering</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhou, Y., Su, J., Zhang, J., Hu, W., Tao, T., Li, G., Zhou, X., Fan, L., Yuan, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We have developed PRIME, an autonomous protein engineering multi-agent system that automates the complete pipeline for developing specialized AI models, from data acquisition through training.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Artificial intelligence (AI) is revolutionizing protein engineering, yet its practical application is often hindered by a fragmented toolchain and the specialized expertise required to orchestrate complex computational workflows. To address these challenges, we have developed PRIME, an autonomous protein engineering multi-agent system. The core innovation of PRIME is dynamic workflow synthesis: it interprets high-level engineering objectives, reasons over a curated library of more than 65 validated protein tools, and autonomously constructs custom computational pathways. Crucially, by grounding every step in verifiable tool execution, PRIME mitigates the risk of model hallucination. It also automates the complete pipeline for developing specialized AI models, from data acquisition through training. In a benchmark of 213 multi-step protein engineering tasks, PRIME exhibited superior performance, successfully completing the majority of tasks where state-of-the-art, general-purpose AI agents fail. We validated its capabilities in demanding real-world applications, including the fully autonomous training of a machine learning classifier and the de novo design of a therapeutic antibody against SARS-CoV-2. By abstracting technical complexity, PRIME empowers scientists to execute sophisticated computational experiments with unprecedented flexibility, establishing a new paradigm for autonomous scientific discovery. The PRIME agent and source code will be made publicly available.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.677756v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Long-read single-cell RNA sequencing uncovers cell-type specific transcript regulation in COVID-19</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Koehler, K., Morris, L., Rakszewska, A., Chua, R. L., Balura, M., Jechow, K., Loske, J., Eils, R., Lehmann, I., Robinson, P. N., Conrad, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Long-read single-cell RNA sequencing on nasopharyngeal swabs from SARS-CoV-2 patients and healthy controls reveals cell-type specific alternative splicing, undetectable with short-read sequencing.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> SARS-CoV-2 infection leads to extensive host transcriptomic changes, but the role of alternative splicing in shaping the immune response remains underexplored. Here, we present the first application of long-read single-cell RNA sequencing on nasopharyngeal swabs from COVID-19 patients and healthy controls to resolve transcript-level changes across cell types. Our analysis identified major epithelial cell types and pronounced immune infiltration, with cell-type annotations concordant with those from short-read data. By enabling isoform-level resolution, our nanopore sequencing approach revealed cell-type specific alternative splicing, undetectable with short-read sequencing. For example, although gene-level expression of the key immune and apoptosis regulators, IFNAR2 and FAIM, did not differ between COVID-19 patients and healthy controls we identified marked shifts in isoform usage. Between moderate and critical cases, we observed cell-type specific differential transcript usage in the T cell signaling kinase FYN and the immune-regulatory transcription factor IRF2. As some of these splicing alterations yield functionally distinct isoforms, we hypothesize that alternative splicing modulates immune signaling and apoptosis, fine-tuning the host response to SARS-CoV-2 infection. Our study demonstrates the unique power of long-read single-cell transcriptomics to uncover isoform-resolved regulatory changes, offering novel insights into the role of alternative splicing in shaping immune responses to viral infections.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.23.676511v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Draft genome of Trichoderma gamsii strain T035 a promising beneficial fungus in agriculture</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Colou, J., Robin, A., Bastide, F., Guillemette, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We report the high-quality draft genome of T. gamsii strain T035, a promising biocontrol strain with strong antibiosis and parasitic activities.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Trichoderma gamsii is a filamentous fungus widely recognized for its beneficial roles in agriculture, particularly for its ability to suppress plant pathogens and enhance crop health. However, genomic resources for this species remain scarce, limiting functional and applied studies. Here, we report the high-quality draft genome of T. gamsii strain T035, a promising biocontrol strain with strong antibiosis and parasitic activities against diverse pathogens. The assembly consisted of 18 scaffolds, with an N50 value of 7.2 Mb and a total assembly length of 38.8 Mb. This draft genome represents the most complete T. gamsii assembly to date and will provide a valuable resource to facilitate the exploration of molecular mechanisms underlying biocontrol and support the development of sustainable plant protection strategies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.677720v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Water body size shapes body size evolution via subgenomic selection in allopolyploid fish</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ren, L., Zeng, Y., Dai, H., Tai, Y., Luo, K., He, S., Gao, X., Cui, J., Luo, M., Zhang, X., Zhang, H., Li, M., Liu, L., Zhou, H., Zhang, J., Han, X., Wang, S., Liu, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that water body size variation influences body size in allopolyploid fish through subgenomic selection, impacting genetic variation, gene regulation, and epigenetic mechanisms.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Allopolyploidization is a major force in species evolution, and studying allopolyploids in diverse environments provides insights into their genetic mechanisms. Here, we investigated how water body size variation influences body size in allopolyploid fish. We used a nascent allopolyploid lineage derived from goldfish (Carassius auratus) and common carp (Cyprinus carpio) and established two populations in small and large water bodies for two generations. Genomic, transcriptomic, and epigenomic analyses revealed that the larger population had higher genetic diversity in subgenome R, while the smaller population had higher diversity in subgenome C. Potential selection regions were associated with growth regulatory genes, suggesting a role for gene regulation in body size adaptation. Dynamic gene expression patterns in various tissues indicated potential mechanisms underlying muscle development and adaptation to smaller water bodies. Furthermore, selection regions were linked to changes in chromatin accessibility, with subgenome R being more susceptible to these changes during genetic divergence. Our findings demonstrate that water body size variation drives body size in allopolyploid fish through subgenomic selection, impacting genetic variation, gene regulation, and epigenetic mechanisms.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.677753v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Sensitive detection of copy number alterations in low-pass liquid biopsy sequencing data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Eriksson, L., Lakatos, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We develop a novel method to improve signal extraction from low-pass liquid biopsy sequencing data, by utilizing a Bayesian changepoint detection algorithm.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Liquid biopsies, coupled with analysis of copy number alterations (CNAs), have emerged as a promising tool for non-invasive monitoring of cancer progression and tumor composition. However, methods utilizing CNA data from liquid biopsies are limited by the low signal in the samples, caused by a low percentage of cancer DNA in the blood, and inherent noise introduced in the sequencing. To address this challenge, we developed BayesCNA, a method designed to improve signal extraction from low-pass liquid biopsy sequencing data, by utilizing a Bayesian changepoint detection algorithm. We use information of the posterior changepoint probabilities to identify likely changepoints, where a changepoint indicates a shift in the copy number state. The signal is then reconstructed using the identified partition. We show the effectiveness of the method on synthetically generated datasets and compare the method with state-of-the-art bioinformatics tools under noisy conditions. Our results show that this novel approach increases sensitivity in detecting CNAs, particularly in low-quality cases.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.677749v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">EWS::WT1 Isoform-Dependent Regulation of Neogenes in Desmoplastic Small Round Cell Tumors</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Truong, D. D., Magrath, J. W., Murgas, K., Fan, J., Shamsutdinova, D., Ingram, D. R., Lazar, A., Lee, S. B., Ludwig, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that DSRCT-specific "neogenes" (NGs) are directly transcriptional outputs of the EWS::WT1 fusion protein (FP), an oncogenic chimeric transcription factor (OCTF) resulting from the t(11;22) translocation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Desmoplastic small round cell tumor (DSRCT) is a rare, aggressive sarcoma characterized by the pathognomonic EWS::WT1 fusion protein (FP), an oncogenic chimeric transcription factor (OCTF) resulting from the t(11;22)(p13;q12) translocation. Recent studies have identified "neogenes" (NGs), genes normally silent in normal tissues but transcriptionally activated by OCTFs, as potential tumor-specific markers in fusion-driven cancers. In this study, we investigated the expression and regulation of DSRCT-specific NGs (DSRCT_NGs) using multimodal data across different cohorts of patients, PDX, and cell line data. We evaluated bulk and single-nucleus RNA sequencing of patient specimens from MD Anderson Cancer Center, revealing the robust ability for DSRCT_NGs to distinguish FP-positive DSRCT from samples failing detection of the EWS::WT1 FP. To elucidate the regulatory role of the EWS::WT1 FP in driving NG expression, we performed knockdown experiments in four DSRCT cell lines. This consistently resulted in a reduction of DSRCT_NG expression. Isoform-specific expression of EWS::WT1 in LP9 and MeT-5A mesothelial cells revealed that the E-KTS isoform of EWS::WT1 predominantly drives DSRCT_NG expression. Mechanistically, ATAC-seq and ChIP-seq analyses demonstrated that EWS::WT1 directly binds to accessible chromatin regions near NG transcription start sites, enriched for WT1 motifs and active histone marks. Integration of Hi-ChIP data further revealed that EWS::WT1 facilitates long-range enhancer-promoter looping at DSRCT_NG loci, promoting the expression of nearby genes. Collectively, these findings establish DSRCT_NGs as direct transcriptional outputs of the EWS::WT1 FP and implicate their loci as regulatory regions of the DSRCT transcriptome. Their fusion-dependent expression, chromatin accessibility, and promoter-enhancer connectivity underscore their potential utility as highly specific biomarkers and therapeutic targets in DSRCT.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.676906v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Novel chromosome-length genome assemblies of three distinct subspecies of pine marten, sable, and yellow-throated marten (genus Martes, family Mustelidae)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tomarovsky, A. A., Khan, R., Dudchenko, O., Beklemisheva, V. R., Perelman, P. L., Totikov, A. A., Serdyukova, N. A., Bulyonkova, T. M., Pobedintseva, M., Abramov, A. V., Weisz, D., Yakupova, A., Zhuk, A., Graphodatsky, A. S., Powell, R., Aiden, E. L., Koepfli, K.-P., Kliver, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We report chromosome-length genome assemblies for three subspecies of marten, each representing a different marten species: the Tobol sable, the Ural pine marten (M. martes uralensis), and the Far East yellow-throated marten.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The genus Martes consists of medium-sized carnivores within the family Mustelidae that are commonly known as martens, many of which exhibit extensive geographic variation and taxonomic uncertainty. Here, we report chromosome-length genome assemblies for three subspecies, each representing a different marten species: the Tobol sable (M. zibellina zibellina), the Ural pine marten (M. martes uralensis), and the Far East yellow-throated marten (M. flavigula aterrima). Using linked-read sequencing and Hi-C scaffolding, we generated assemblies with total lengths of 2.39-2.45 Gbp, N50 values of 137-145 Mbp, and high BUSCO scores (93.6-96.4%). We identified 19 chromosomal scaffolds for sable and pine marten, and 20 for yellow-throated marten, which agrees with the known karyotypes of these species (2n=38 and 2n=40, respectively). Annotation predicted ~20,000 protein-coding genes per genome, of which >90% were assigned functional names. Repeats encompass 36.9-40.4% of the assemblies, with a prevalence of LINEs and SINEs, and is conservative across the genus. Synteny analysis of our generated and available marten genome assemblies revealed assembly artifacts in previously published assemblies, which we confirmed through investigation of Hi-C contact maps. Among other rearrangements, we verified a sable-specific inversion on chromosome 11 using the published cytogenetic data. Our assemblies broaden the genomic resources available for Martes, extending coverage to geographically distant and taxonomically significant subspecies. Together, they provide a robust framework for assessing intraspecific genetic diversity, identifying signatures of hybridization, and refining the complex taxonomy of the genus. Beyond conservation and evolutionary applications, these references will facilitate comparative genomics across Mustelidae and other carnivorans.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.677678v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A century of breeding has preserved genetic variation, accumulated favorable alleles, and shaped the Rht gene portfolio in North American spring wheat</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gill, H. S., Blecha, S., Conley, E., Brault, C., Fiedler, J., Cook, J., Glover, K., Green, A., Read, A. C., Anderson, J. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of HRSW genetic diversity and the effect of Rht genes on plant height (PH).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Hard red spring wheat (HRSW) is an important market class of wheat in North America with a rich history of breeding. We genotyped 1,013 HRSW lines representing a century of breeding material using a SNP array and targeted KASP genotyping to assess the changes in genetic diversity and Rht genes in HRSW from North America. Results show that early breeding efforts broadened an initially narrow gene pool derived from a few founders and subsequently maintained the genetic diversity. Analysis of Rht genes revealed that Rht-D1b was the predominant semi-dwarf allele from introduction of the Green Revolution genes (Rht-B1 and Rht-D1) in 1960s until the FHB epidemics of 1990s, when most breeding programs shifted to Rht-B1b. However, adoption of Rht-B1b and Rht-D1b remains low in some regions of the Great Plains. The dwarf alleles at GA-sensitive Rht24 and Rht25 have persisted in the region at low frequencies before the introduction of Green Revolution genes and have increased in frequency in recent decades. We also analyzed the effect of Rht genes on plant height (PH) in the HRSW growing region and found that Rht-B1b or Rht-D1b results in ~12% reduction in PH, which decreases to ~9% in dry environments. Notably, combining dwarfing alleles at Rht24 and Rht25 reduced height by ~6% in dry environments, suggesting an alternative approach for reducing PH. Our analysis also revealed significant genetic interactions between Rht-B1 and Rht25, as well as between Rht-D1 and Rht24, which could help fine-tune PH for different environments by employing appropriate Rht combinations. Furthermore, our results demonstrate positive selection for photoperiod sensitivity in recent decades, particularly in high-latitude regions. This study provides a valuable genomic resource, and the findings enhance our understanding of past breeding efforts while offering insights to guide future plant breeding strategies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.677111v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Growth and Yield Response of Soft White Common Spring Wheat (SWCSW) Varieties under Different Nitrogen Fertilizations and Plant Growth Regulators Applications</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hussain, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We observed a significant seasonal impact and variations in the performance of PGRs on soft white common spring wheat varieties, and we observed a variable but positive response particularly under application of CC-C, TE, CC-BC and CC-AB.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Application of plant growth regulators (PGRs) is often reported beneficial in wheat to achieve various targets including lodging resistance. However, research evidence on the application of PGRs in Soft White Common Spring Wheat (SWCSW) varieties is limited particularly in Columbia Basin. Therefore our objectives were to evaluate the effects of PGRs on SWCSW varieties exposed to different nitrogen (N) application levels. Two widely grown SWCSW varieties Louise and Diva were planted for two growing seasons using split plot design with four replications. Crop growth (stem height and diameter), yield and yield attributes (grain numbers per head, head numbers per fit, grain weight), quality characteristics (test weight, protein content) and lodging score were recorded following the application of PGRs at tillering (GS21-26), stem elongation (GS30-32), and flag leaf emergence (GS37-39) stages combined with high N fertilization. No lodging was observed during first season whereas it was witnessed during second season , and we observed a variable but positive response particularly under application of CC-C, TE, CC-BC and CC-AB. An increase in stem diameter and reduction in stem height were observed under PGR application which indirectly contributed to stem strength and reduced lodging. Although impact of PGRs application on grain numbers, head numbers grain yield and grain weight were non-significant and variable, we observed an improved performance of these attributes such as higher grain yield was observed under application of CC-B and CC-BC. The grain yield of Louise and Diva was similar at low and high N input. These results indicate that application of 168 kg N / ha is sufficient for an acceptable grain productivity and to gain agronomic returns. As we observed a significant seasonal impact and variations in the performance of SWCSW varieties, future research should consider long term evaluations to evaluate the impacts of PGRs on soft white common spring wheat varieties.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.676075v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Intracellularly Coupled Oscillators for Synthetic Biology</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hollo, G., Park, J. H., Evard, R., Schaerli, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We model the interactions of intracellularly coupled oscillators and propose novel oscillator-based circuit designs and outline potential applications, including oscillator based computers that integrate processing and memory functions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Synthetic biology aims to engineer or re-engineer living systems. To achieve increasingly complex functionalities, it is beneficial to use higher-level building blocks. In this study, we focus on oscillators as such building blocks and model the interactions of intracellularly coupled oscillators. We classify these oscillators on the basis of coupling strength: independent, weakly or strongly coupled, and intrinsically coupled. We predict a range of fascinating phenomena to arise in these systems, such as beat phenomena, amplitude and frequency modulation, period doubling, higher-period oscillations, chaos, resonance, and synchronization, with the aim of guiding future experimental work in bacterial synthetic biology. Finally, we propose novel oscillator-based circuit designs and outline potential applications, including oscillator-based computers that integrate processing and memory functions, offering multistate and nonlinear processing capabilities.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.23.678067v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Synthetic Horizontal Gene Transfer for Ecosystem Restoration</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Maull, V., Aguade-Gorgorio, G., de Lorenzo, V., Sole, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We explore the impacts of synthetic horizontal gene transfer in models of ecological consortia.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Restoring endangered ecosystems has become a pressing issue as the effects of global warming continue to harm existing communities. Over the last decade, several studies have suggested the use of synthetic biology as a tool to protect these biodiverse communities by augmenting their functionality. A critical example concerns soil microbiome communities in drylands, where increasing water retention by some of the constituent species could effectively protect the ecosystem from abrupt degradation. However, how to effectively deploy a functional synthetic construct that can scale its impact to the community level remains an open question. Recent experimental research has designed recombinant gene plasmids with the capacity to horizontally transfer across soil microbial communities. Here, we explore the impacts of synthetic horizontal gene transfer in models of ecological consortia. By coupling consumer-resource dynamics with multispecies gene transfer, our work identifies the design conditions that promote biodiversity while regulating gene propagation and the spread of engineered organisms.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.23.678013v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ABaCo: Addressing Heterogeneity Challenges in Metagenomic Data Integration with Adversarial Generative Models</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Vidal, E., Phanthanourak, A. L., Gharib, A., Webel, H. E., Assis, J., Ayala Ruano, S., Ferreira Cunha, A., Santos, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce ABaCo, a generative model that combines a Variational Autoencoder (VAE) with an adversarial discriminator specifically designed to handle the unique characteristics of metagenomic data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The rapid advancement of high-throughput metagenomics has produced extensive and heterogeneous datasets with significant implications for environmental and human health. Integrating these datasets is crucial for understanding the functional roles of microbiomes and the interactions within microbial communities. However, this integration remains challenging because of the technical heterogeneity and inherent complexity of these biological systems. To address these challenges, we introduce ABaCo, a generative model that combines a Variational Autoencoder (VAE) with an adversarial discriminator specifically designed to handle the unique characteristics of metagenomic data. Our results demonstrate that ABaCo effectively integrates metagenomic data from multiple studies, corrects technical heterogeneity, outperforms existing methods, and preserves taxonomic-level biological signals. We have developed ABaCo as an open-source, fully documented Python library to facilitate, support and enhance metagenomics research in the scientific community.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.677692v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">OTMalign: A Fast and Robust TM-score-driven Protein Structure Alignment Algorithm using Optimal Transport</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hu, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present OTMalign, a novel algorithm that iteratively optimizes the TM-score in a fast and robust manner.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein structure alignment is a fundamental task in computational biology. While the TM-score is a gold standard for evaluating structural similarity, its direct optimization is challenging. Many algorithms, therefore, rely on heuristic or combinatorial approaches. In this work, we present OTMalign, a novel algorithm that iteratively optimizes the TM-score in a fast and robust manner. OTMalign leverages the theory of Optimal Transport (OT) to establish a soft correspondence between protein structures, guided by a TM-score-inspired reward matrix. This avoids the pitfalls of combinatorial searches and provides a global perspective at each iteration. The optimal rigid body transformation is then determined analytically using a weighted Kabsch algorithm, where the weights are derived from the OT correspondence matrix (the P matrix). We demonstrate that this P matrix effectively captures the structural similarity, and its use in a weighted covariance matrix allows for a direct SVD-based solution for the rotation and translation. We evaluated OTMalign on the RPIC benchmark dataset, demonstrating its rapid convergence and ability to produce high-quality alignments. OTMalign offers a practical and theoretically sound framework for TM score optimization, distinguished by its speed and iterative nature.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.21.677676v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SpaceSequest: A unified pipeline for spatial transcriptomics data analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sun, Y. H., Piya, S., Ouyang, Z., Chen, Y., Gagnon, J., Cao, S., Zhang, H., Song, B., Zhu, J., Chandratre, K., Yu, H., Hu, W., Ryals, M., Casey, F., Huh, D., Zhang, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present SpaceSequest, a unified and comprehensive pipeline designed for the analysis, visualization, and publication of spatial transcriptomics data from various platforms.</p><p style="font-size: 16px; text-align: justify;">Background Spatial transcriptomics has emerged as one of the most powerful tools for gaining biological insights, enabling researchers to uncover intricate relationships between gene expression patterns and tissue architecture. Recent advances in the field have resulted in a variety of new platforms, including Visium, Visium HD, and Xenium from 10x Genomics, as well as GeoMx and CosMx from NanoString Technologies, which has now been acquired by the Bruker Corporation. However, the existence of diverse spatial transcriptomics platforms and various data formats poses challenges in standardizing data analysis. Thus, there remains a critical gap in the availability of a comprehensive pipeline capable of conducting end-to-end analysis that is necessary to extract biological insights from multiple spatial transcriptomics platforms. Results Here, we present SpaceSequest, a tailored pipeline that utilizes cutting-edge computational methodologies to conduct a thorough analysis, enabling the extraction of crucial biological insights from five major spatial transcriptomics technologies. SpaceSequest performs (1) standardized quality control and general data processing, (2) key analyses customized for each spatial platform, (3) automated cell type annotation and deconvolution, and (4) high-quality figure and analysis result generation. In addition, SpaceSequest allows for smooth integration with cellxgene VIP and Quickomics for user-friendly data access and interactive visualization. Conclusions: SpaceSequest is a unified and comprehensive pipeline designed for the analysis, visualization, and publication of spatial transcriptomics data from various platforms. The source code is available at https://github.com/interactivereport/SpaceSequest. To facilitate seamless installation and usage, we have also created a detailed Bookdown tutorial that can be accessed through https://interactivereport.github.io/SpaceSequest/tutorial/docs/index.html.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.15.676389v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Signatures of Micropeptides Encoded by lncRNAs in Cancer Progression and Metastasis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zok, S., Linial, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that cancer progression-associated lncRNAs with strong statistical evidence for MP-encoding potential are a promising new class of biomarkers and therapeutic targets in oncology.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Long non-coding RNAs (lncRNAs) are key regulators of gene expression, chromatin remodeling, and signaling. Recent estimates suggest that the human genome contains more than 35,000 lncRNA genes, with roughly 20% predicted to encode micropeptides (MPs) with unknown functions. In this study, we focused on the subset of lncRNAs with strong statistical evidence for MP-encoding potential, accounting for approximately 8% of the unfiltered MPs collection. Our analysis centered on 1,782 high-confidence lncRNA-MPs derived from 478 genes expressed across 17 cancer types from The Cancer Genome Atlas (TCGA). We show that lncRNA-MPs display distinct amino acid compositions and unique 4-mer patterns compared to the human coding proteome. A few genes (9) with exceptionally long transcripts are characterized by [‚â•]20 MPs each. Functional interference confirmed that most of the lncRNA-MPs are unstructured. Only a third of the genes display some phylogenetic conservation, and only 4 genes display canonical N-terminal signal peptides characteristic of secreted proteins. We focused on cancer progression-associated lncRNAs that show differential expression (z-score >|3|) across consecutive tumor stages and metastatic states (transitional lncRNAs, Tr-lncRNAs). A collection of 72 genes encoding 314 MPs (Tr-lncRNA-MPs) was detected, with 76% of the MPs being [‚â•]30 amino acids long. Prediction by AlphaFold 2.0 and homology modeling tools revealed dozens of MPs with well-defined secondary structures and recognizable 3D motifs. Among the longer Tr-lncRNA-MPs (>60 amino acids), we confirmed the presence of ubiquitin-like, RNase H-related, and other conserved foldable motifs. Known cancer lncRNAs containing high-confidence MPs (XIST, UCA1, HOXA11-AS, LINC01234, and HAND-AS1) overlap with 50 pan-cancer lncRNAs associated with tumor stage or metastasis transitions. Together, these findings demonstrate that integrating sequence motifs (e.g., signal peptides, k-mers) with structural foldability offers a multifaceted view of lncRNA-MPs in cancer. We argue that the capacity to produce MPs may reinforce the oncogenic impact dominated by the lncRNA entity. We propose that Tr-lncRNA-MPs represent a promising new class of biomarkers and therapeutic targets in oncology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.677872v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ACSS2-Mediated Metabolic-Epigenetic Crosstalk Drives Fulvestrant Resistance and Represents a Novel Therapeutic Target</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mogol, A. N., Yoo, J. Y., Eve, A. A., Goel, M., Dutton, D. J., Schane, C. P., Lam, A., Dutta, D., Barnick, B., Erdogan, E. D., Nelson, E. R., Grosse-Perdekamp, M., Madak-Erdogan, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Fulvestrant (Fulv) and acetyl-CoA synthase inhibitor (ACSI) targeting Acyl- CoA Synthetase Short Chain Family Member 2 (ACSS2) contributed to endocrine therapy resistance through nuclear acetyl CoA provision for epigenetic alterations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Endocrine therapies target hormone-dependent cancer cells, primarily through estrogen receptor alpha (ERa), expressed in ~70% of breast cancers (ER+). Despite treatment advances, 30-40% of ER+ breast cancer patients experience recurrence and metastasis, with 5-year survival rates of only 31.9%. We validated poor outcomes for liver metastasis patients treated with Fulvestrant (Fulv) using the local Carle Foundation Hospital cohort and examined metabolic pathways in liver metastatic patient-derived xenograft (PDX) models, revealing upregulated lipid and acetyl-CoA production. Our previous work demonstrated that combining Fulv with acetyl-CoA synthase inhibitor (ACSI) targeting Acyl-CoA Synthetase Short Chain Family Member 2 (ACSS2), synergistically reduced ER+ metastatic breast cancer (MBC) cell viability in vitro. Using multiple analytical approaches, isotope tracing, CUT&RUN sequencing, immunofluorescence, western blot, and RNA sequencing, we characterized the effects of acetyl-CoA synthesis inhibition on Fulv-induced alterations. Fulv treatment of MBC cells increased ACSS2 expression and acetate utilization. Isotope tracing revealed that Fulv decreased acetate flux to the TCA cycle while promoting fatty acid synthesis. Importantly, ACSS2 was predominantly nuclear and CUT&RUN sequencing showed that Fulv treatment increased ACSS2 chromatin occupancy and ER/ACSS2/H3K27ac overlapping sites near genes associated with tumor progression, which was eliminated by combination of ACSI and Fulv. RNA sequencing revealed reduction of Fulv-induced expression of genes involved in cancer cell metabolism and key signaling pathways in cancer with the Fulv+ACSI combination. In a therapy-resistant xenograft model, combining Fulv and ACSI reduced Fulv-dependent increase in metastatic burden. Our findings indicate ACSS2 contributes to endocrine therapy resistance through nuclear acetyl-CoA provision for epigenetic alterations. Targeting these cancer cell adaptations represents a novel therapeutic approach potentially reducing metastasis-related mortality and improving breast cancer treatment outcomes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.23.678089v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Dual Modes of Gene Regulation by CDK12</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, Y., Baluapuri, A., Manokaran, C., Ni, J., Jiang, T., Janssens, R., Marteijn, J., Adelman, K., Zhao, J., Roberts, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that loss or inhibition of CDK12 also significantly upregulates a set of coding and non-coding loci, whose activation could contribute to the anti-proliferative effects of the CDK 12 inhibitors.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The process of transcription is driven forward by the activity of kinases including CDK7, CDK9 and CDK12. Accordingly, acute inhibition of any of these kinases results in profound downregulation of gene expression. Here, we discover that loss or inhibition of CDK12 also significantly upregulates a set of coding and non-coding loci, whose activation could contribute to the anti-proliferative effects of CDK12 inhibitors. Mechanistically, CDK12 inhibition impairs transcription elongation, leading to increased RNA polymerase II termination or arrest in long genes. However, short genes such as MYC and enhancer RNAs are highly transcribed in the absence of CDK12 activity. Indeed, in HER2+ breast cancer, a malignancy where CDK12 is co-amplified with HER2 and its expression correlates with disease status, CDK12 inhibition markedly elevates MYC expression to induce lethality. The dual effects of CDK12 inhibition elucidated herein clarify its role in transcriptional control and have significant translational implications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.677923v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Understanding Cancer Survivorship Care Needs Using Amazon Reviews: Content Analysis, Algorithm Development, and Validation Study.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liwei Wang, Qiuhao Lu, Rui Li, Taylor B Harrison, Heling Jia, Ming Huang, Heidi Dowst, Rui Zhang, Hoda Badr, Jungwei W Fan, Hongfang Liu</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Amazon consumer reviews are a valuable real-world data source for understanding cancer survivorship care needs.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Complementary therapies are being increasingly used by cancer survivors. As a channel for customers to share their feelings, outcomes, and perceived knowledge about the products purchased from e-commerce platforms, Amazon consumer reviews are a valuable real-world data source for understanding cancer survivorship care needs.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40986859/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Revisiting the biophysical aspects of extracellular-matrix-mimicking hydrogels: what cells see &lt;i&gt;vs.&lt;/i&gt; what cells feel.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sui J, Pragnere S, Kurniawan NA.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We review key ECM-mimicking hydrogel properties to replicate physiological conditions, examine the impact of biochemical, structural, and mechanical features on cell behavior, and explore how structural and mechanical properties-what cells "see" and "feel"-are interrelated and jointly affect cell function.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The extracellular matrix (ECM) is critical in regulating cell behavior and tissue function. This recognition has driven the development of ECM surrogates to better understand cell-ECM interactions and advance biomedical applications. Hydrogels are promising candidates for this purpose due to their biocompatibility, tunability, and ability to embed cells in 3D environments. While early efforts in the design of ECM-mimicking hydrogels focused on macromolecular type, it is now clear that their biophysical parameters, such as polymer molecular weight, fibrous versus non-fibrous structures, pore size, and mechanical properties, significantly influence cell behavior. Understanding the interplay of these factors is crucial for the rational design of biomaterials, but remains challenging given the complexity of hydrogel systems and the rapid pace of new findings. This review critically evaluates hydrogels as ECM mimics for 3D cell cultures. We revisit key ECM properties to replicate, examine how hydrogel design can meet these needs, and summarize the impact of biochemical, structural, and mechanical features on cell behavior. We also explore how structural and mechanical properties-what cells "see" and "feel"-are interrelated and jointly affect cell function. Our analysis concludes that strategic combinations of polymeric materials will play an important role for next-generation hydrogels to replicate physiological conditions and independently enable precise control over key parameters. These advancements will enhance our understanding of cell-ECM interactions and support the development of innovative biomaterials for tissue engineering and regenerative medicine.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40878170/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A New Approach to Large Multiomics Data Integration.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Alex Dexter, Spencer A Thomas, Rory T Steven, Kenneth N Robinson, Adam J Taylor, Efstathios A Elia, Chelsea Nikula, Andrew D Campbell, Yulia Panina, Arafath K Najumudeen, Bin Yan, Piotr Grabowski, Gregory Hamm, John Swales, Aurelien Tripp, George Poulogiannis, Mariia O Yuneva, Simon Barry, Richard J A Goodwin, Owen J Sansom, Zoltan Takats, Josephine Bunch</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We use deep learning on subsampled nonlinear dimensionality reduction using t-SNE and UMAP to extract features from large complex data sets including mass spectrometry imaging and chromosome conformation capture.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Data reduction and data mining are common practices for handling large-scale data from wide-ranging sources, but high-dimensional omics and imaging data sets present difficult challenges for feature extraction and data mining due to the large number of features that cannot be simultaneously examined. The sample numbers and variables in these methods are constantly growing as new technologies are developed, and computational analysis needs to evolve to keep up with growing demand. In recent years, there has been a rapid uptake of nonlinear dimensionality reduction via methods such as t-distributed stochastic neighbor embedding and uniform manifold approximation and projection. These approaches have revolutionized our ability to visualize and interpret high-dimensional data and have rapidly become preferred methods for analysis of data sets containing an extremely high number of variables. Further to this is the emerging interest in combining information from multiple omics sources to gain a more holistic view of systems biology. Current state-of-the-art algorithms can perform data mining, visualization, and classification on routine data sets but struggle when data sets grow above a certain size. We present a new approach to large and multiomic data integration to extract, mine, and integrate large multiomics data sets that were previously considered prohibitively large. Here, we demonstrate the use of deep learning on subsampled nonlinear dimensionality reduction using t-SNE and UMAP to extract features from large complex data sets including mass spectrometry imaging and chromosome conformation capture. We then go on to demonstrate how this method can be used to learn embeddings from the fusion of different omics data, allowing metabolomics data to be projected into a reduced transcriptomics representation.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40934376/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

