<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Friday, 20 June 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Biome-specific genome catalogues reveal functional potential of shallow sequencing</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Escobar-Zepeda, A., Ruuskanen, M. O., Beracochea, M., Lu, J., Mongad, D., Richardson, L., Finn, R. D., Lahti, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We used deep shotgun metagenomic and 16S rRNA metabarcoding data for taxonomic and functional benchmarking, using only a fraction of the original deep sequencing data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The use of 16S rRNA metabarcoding for functional prediction is limited by several biases. Shallow shotgun sequencing is a cost-effective and taxonomically high-resolution alternative to 16S rRNA metabarcoding, but the low sequencing depth limits functional inference. Our BioSIFTR tool maps shallow shotgun sequencing reads against single-biome databases and extrapolates into their precalculated functional profiles. We used three datasets from red junglefowl, mice, and human gut, containing matched deep shotgun metagenomic and 16S rRNA metabarcoding data for taxonomic and functional benchmarking. An additional human gut deep shotgun sequencing data set was subsampled to 1 M reads and analysed with BioSIFTR to replicate previously obtained results. BioSIFTR taxonomic and functional profiles closely agree with the results of the full deep sequencing data in all biomes. We also replicated differences in the human gut microbiome between high and low trimethylamine N-oxide producing participants, using only < 2 % of the original deep sequencing data. The BioSIFTR tool is a powerful approach which approximates the functional information of a deep-sequenced metagenome while using only a fraction of the data. Shallow shotgun sequencing combined with BioSIFTR could be a stand-in replacement for 16S rRNA metabarcoding with an increased taxonomic and functional resolution, and lower bias.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.16.659887v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Circadian disruption beneath the surface: a screening approach to identify actigraphy-based markers of split shift work in submariners</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Marques, D., Fernandes, C., Barbosa-Morais, N. L., Reis, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a minimal yet robust exploratory feature-screening approach to identify candidate actigraphy markers of split shift work during a three-week military submarine mission.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Submarines pose significant physiological and behavioral challenges due to their dimly lit, isolated, confined, and extreme (DICE) environment, which deprives individuals from exposure to a natural light-dark cycle and limits opportunities for physical activity. In this study, we developed a minimal yet robust exploratory feature-screening approach to identify candidate actigraphy markers of split shift work during a three-week military submarine mission. Actigraphy data was collected across the pre-mission, mission and post-mission periods. During the mission, crew members followed a 6-hours-on/6-hours-off split shift schedule, except for submariners with on-call duty or extended hours. Forty-one actigraphy features were screened for their ability to differentiate between study periods and shifts. A Monte Carlo permutation test was used for this initial screening, followed by receiver operating characteristic (ROC) curve analysis for pairwise comparisons. Light-related features, the slopes of regression lines fitted to the daily predicted dim light melatonin onset (DLMO) trajectories, and Pearson correlation coefficients between each recording day and its corresponding predicted DLMO demonstrated the greatest potential to distinguish between study periods. Among shifts, most high-performing features discriminated between split shift work schedules and on-call or extended-hours schedules. Only the relative amplitude (RA) and the average light intensity during the five least-illuminated hours of the day (L5) reliably distinguished between the shift performing the bulk of nighttime work and its counterpart. Given its robustness under conditions of low activity and light exposure, our methodology may be applicable to other operational or clinical populations with reduced activity levels, such as individuals with depression or dementia.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.16.659930v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Exploring Mycobacterium tuberculosis rRNA Transcriptomic Signatures asResponse to Anti-TB Treatment in Whole Blood RNA-seq Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lopez-Exposito, P., Calvet Seral, J., Ferrer, S., Mendoza-Losana, A., M. Gordaliza, P., Vaquero, J. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We detected 16S and 23S rRNA transcripts in blood RNA-seq and suggest dynamic transcriptomic changes during treatment.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Current clinical tools for monitoring tuberculosis (TB) treatment response rely mostly on sputum culture and chest X-ray, which are unreliable in patients with low bacterial loads and lack the desirable promptness. There is a need for biomarkers able to provide earlier and more accurate insights into pathogen viability and response to therapy. We analyzed publicly available whole-blood RNA-seq data from 79 TB patients sampled at diagnosis, and weeks 1, 4, and 24 of standard anti-TB treatment. After aligning human reads and filtering, remaining reads were mapped to the Mycobacterium tuberculosis H37Rv genome to quantify rRNA subunit transcripts (16S, 23S, 5S, ITS1, ITS2). Microbiome profiles were assessed using Kraken2/Bracken, with alpha/beta diversity analyses and differential abundance (ANCOM-BC2). 16S and 23S rRNA transcripts were consistently detected across all treatment times, with 23S reads dominating in diagnosis and early stages of treatment shifting toward a significant predominance of 16S reads at week 24. 5S and ITS1 were inconsistently detected, whereas ITS2 was undetectable. Alpha diversity (Shannon index) increased during treatment (significant at weeks 1 and 4), while beta diversity showed significant shifts over time despite no significant differences in total M. tuberculosis abundance. Our findings suggest that it may be feasible to detect M. tuberculosis rRNA signatures in blood RNA-seq and suggest dynamic transcriptomic changes during treatment. The 16S/23S ratio and minor rRNA units may serve as complementary biomarkers for treatment monitoring and other transcriptomic-based biomarkers. Future work should validate these findings in larger cohorts using optimized RNA-seq protocols focusing on the pathogen.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.16.659901v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Small uncultured viral-like entities redraw the origin of viruses and cellular compartments</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Aguilar, A., Miranda, I., Nadel, O., Nayfach, S., Segall, A., Rohwer, F., Roux, S., White, S., Luque, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified HK97-fold viruses as a possible evolutionary bridge between encapsulins and procapsids, and proposed a new evolutionary mechanism for the transition between the two.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The HK97-fold is a conserved three-dimensional protein structure specialized in forming protein shells. It is the primary building block of capsids that protect the genome of viruses in the Duplodnaviria realm, infecting organisms across all domains of life. It is also the primary building block of encapsulins, compartments that confine biochemical reactions in prokaryotes. Recent genomic studies have hypothesized that encapsulins evolved from HK97-fold viruses. However, this evolutionary pathway is challenging to justify biophysically because HK97-fold viruses form larger and more complex protein shells than encapsulins. We addressed this paradox by searching for smaller and simpler HK97-fold viral entities in metagenomes across ecosystems. The investigation yielded a well-defined group of viral entities, which encode HK97-fold proteins displaying significant molecular similarities with encapsulins and lacking tail genes. The structural phylogenetic analysis of HK97-fold entities revealed likely bidirectional evolutionary transitions between encapsulins and HK97-fold viruses. A potential evolutionary mechanism responsible for such transitions was identified based on the presence of lysogeny-associated genes in the viral entities and the structural and molecular parallels between encapsulins and procapsids, the immature state of viral capsids before genome packaging. Structural comparisons indicated that procapsids are akin to the common HK97-fold protein shell ancestor and might still facilitate transitions between modern viruses and encapsulins. This also implies that viral procapsids may retain the ability to carry out biochemical reactions during infections or when released in the environment, which, if confirmed, would require a fundamental revision of the role of viral capsids in the biosphere.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.18.659913v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Signatures of selection and mechanisms of insecticide resistance in Ugandan Anopheles funestus: Insights from embedding translational genomics into the LLINEUP cluster randomised trial.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kayondo, L. N., Nagi, S. C., Njoroge, H., Hernandez-Koutoucheva, A., McDermott, D. P., Gonahasa, S., Lynd, A. R., Oruni, A., Sebuguzi, C. M., Opigo, J., Yeka, A., Katureebe, A., Kyohere, M., Kamya, M., Dorsey, G., Hemingway, J., Staedke, S. G., Sam, N., Nankabirwa, J., Kayondo, J., Clarkson, C. S., Miles, A., Lawniczak, M. K. N., Lucas, E. R., Donnelly, M. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that An. funestus populations were relatively unaffected by the interventions of pyrethroid-piperonyl butoxide (PBO) and pyrethroids-only long-lasting insecticidal nets (LLINs) in Uganda, but An. gambiae s.l</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In response to the emerging threat of insecticide resistance in malaria vectors, insecticides are being repurposed for vector control or developed de novo. Good stewardship of these finite new resources is essential if disease control programmes are to remain effective. This is dependent on timely data to help guide evidence-based decision-making for National Malaria Control programmes (NMCPs). By embedding genomics into cluster randomized control trials (cRCTs), we can perform surveillance and early detection of insecticide resistance variants to new and repurposed chemicals in natural field conditions, supporting effective stewardship. The LLIN Evaluation Uganda Project (LLINEUP) trial evaluated the efficacy of pyrethroid-piperonyl butoxide (PBO) and pyrethroids-only long-lasting insecticidal nets (LLINs). It was conducted in Uganda between 2017-2020 and was the largest cRCT to date, covering 40% of the country in 104 health sub-districts. We embedded genomic surveillance within LLINEUP to detect and track insecticide resistance variants. At baseline and throughout the trial, we sampled Anopheles mosquitoes with Prokopack aspirators and performed Illumina whole-genome sequencing. We show that An. funestus populations were relatively unaffected by the interventions, compared to An. gambiae s.l., which were markedly reduced six months following LLIN deployment. Standard approaches for describing genetic diversity and population structure e.g. fixation index (FST), Principal Component Analysis (PCA) and Neighbour-Joining (NJ) trees, were consistent with the density observations and suggestive of a single large An. funestus population in Uganda with little genetic differentiation. Genome-wide selection scans revealed strong signals of selection at the Resistance to pyrethroid-1 (RP1) locus and Cyp9k1, both loci previously implicated in pyrethroid resistance. We report two additional loci, eye diacylglycerol kinase (Dgk) ({cong} 13.5Mb on the X chromosome) and O-mannosyl-transferase (TMC-like) ({cong} 67.9Mb on 3RL) that showed signals of selection. Known DDT and permethrin resistance-associated variants at the Gste2 locus, L119F and L119V, were also identified. Over the trial period, changes in haplotype frequencies were observed in regions under selection, with more pronounced shifts in the PBO arm. Notably, there were significant reductions in the frequencies of swept haplotypes (measured by delta ({Delta}) H12) in the Dgk and Cyp6p9a regions, while significant increases in haplotype frequency were observed at Gste2 and Cyp9k1 loci. Our findings reveal the differential impact of the trial on An. gambiae s.l. and An. funestus densities and the differing responses of An. funestus populations to pyrethroid and pyrethroid-PBO selection pressure. These insights underscore the potential value of tailored, species- and region-specific vector control strategies, supported by regional genetic surveillance, to better control insecticide resistance evolution and spread. By embedding genomic surveillance in cRCTs we can facilitate the discovery of putative resistance variants and can provide evidence of their impact on vector control tool efficacy; both of crucial importance to evidence-based deployment of vector control tools by NMCPs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.17.659961v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">FastGA: Fast Genome Alignment</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Myers, E. W., Durbin, R., Zhou, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> FastGA finds alignments between two genome sequences more than an order of magnitude faster than previous methods that have comparable sensitivity.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> FastGA finds alignments between two genome sequences more than an order of magnitude faster than previous methods that have comparable sensitivity. Its speed is due to (a) a carefully engineered architecture involving only cache-coherent MSD radix sorts and merges, (b) a novel algorithm for finding adaptive seed hits in a linear merge of sorted k-mer tables, and (c) a variant of the Myers adaptive wave algorithm to find alignments around a chain of seed hits that detects alignments with up to 25-30% variation. It further does not require pre-masking of repetitive sequence, and stores millions of alignments in a fraction of the space of a conventional CIGAR-string using a trace-point encoding that is further compressed by the ONEcode data system introduced here. As an example, two bat genomes of size 2.2Gbp and 2.5Gbp can be compared in a little over 2 minutes using 8 threads on an Apple M4 Max laptop using 5.7GB of memory and producing 1.05 million alignments totaling 1.63Gbp of aligned sequence that cover about 60% of each genome. The output ALN-formated file occupies 66MB. This file can be converted to a PAF file with CIGAR strings in 6 seconds, where the PAF representation is a significantly larger 1.03GB file. FastGA is freely available at github: http://www.github.com/thegenemyers/FASTGA, along with utilities for viewing inputs, intermediate files, and outputs and transforming outputs into other common formats. Specifically, FastGA can, in addition to its highly efficient ONEcode representation, output PSL-formatted alignments, or PAF-formatted alignments with or without CIGAR strings explicitly encoding the alignments. There is also a utility to chain FastGA's alignments and display them in a dot-plot like view in Postscript files, and an interactive viewer is in development.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.15.659750v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PathoEye: a deep learning framework for the whole-slide image analysis of skin tissue</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lin, Y., Lin, F., Zhang, Y., Wen, J., Li, G., Zeng, X., Sun, H., Jiang, H., Yan, T., Xue, R., Sun, H., Yang, B., Zhou, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present PathoEye for WSI analysis in dermatology, which integrates epidermis-guided sampling, deep learning and radiomics and extracts distinct features associated with various skin conditions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Objective: To provide an interpretable computational framework for the examination of whole-slide images (WSI) in skin biopsies, PathoEye focuses on the dermis-epidermis junctional (DEJ) areas, also known as the basement membrane zone (BMZ), to enrich the pathological features of various skin diseases. Methods: We presented PathoEye for WSI analysis in dermatology, which integrates epidermis-guided sampling, deep learning and radiomics. It enables unsupervised semantic segmentation of the BMZ and extracts distinct features associated with various skin conditions. Results: PathoEye performs comparably with existing methods in binary classification tasks, while outperforming them in multi-classification tasks involving different skin conditions. It enables the investigation of histopathological aberrations in aged skin compared with young skin. Additionally, it highlighted the texture changes in the BMZ of young skin compared with aged skin. Further experimental analyses revealed that senescence cells were enriched in the BMZ, and the turnover of basement membrane (BM) components, including COL17A1, COL4A2, and ITGA6, was increased in aged skin. Conclusion: PathoEye is a comprehensive tool for characterizing the unique features of different skin conditions associated with BMZ, thereby ensuring its potential applications in skin disease diagnosis and treatment planning.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.16.659818v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Predicting Spatial Transcriptomics from H&E Image by Pretrained Contrastive Alignment Learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zou, J., Xiao, K., Chen, Z., Pei, J., Xu, J., Chen, T., Hou, L., Wu, C., She, Y., Yuan, Z., Chen, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel method for accurately predicting high-dimensional spatial transcriptomics from H&E (Hematoxylin and Eosin) stained images, which can be used to identify subtle pathological features in cancer, and reconstruct 3D spatial transcriptomes from images alone.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The intricate molecular landscape within tissues holds crucial information about cellular behavior and disease progression, yet capturing this complexity at a spatial level remains challenging. While Spatial transcriptomics (ST) offers valuable insights into gene expression patterns within their native tissue context, its widespread adoption is hindered by high costs and limited gene detection capabilities. Here we introduce CarHE (Contrastive Alignment of gene expRession for hematoxylin and eosin image), a method that overcomes these limitations by accurately predicting high-dimensional ST data (over 10,000 genes) solely from readily available H&E (Hematoxylin and Eosin) stained images. This novel pre-trained architecture employs contrastive learning through two mechanisms: cell-type-based transcriptomics information transfer and image-based histology information transfer. These mechanisms precisely align image features with spatial single-cell gene expressions, achieving prediction accuracies exceeding 0.7 (up to 1.7 folds compared to second best) across diverse tissue types and species. CarHE's superior performance extends to identifying subtle pathological features such as tertiary lymphoid structures in various cancers, including breast cancer, lung cancer, melanoma and ccRCC (clear cell renal cell carcinoma), and reconstructing 3D spatial transcriptomics from images alone, offering a cost-effective and robust alternative for large-scale spatial transcriptomics. We further validated CarHE's effectiveness by predicting DFS (Disease-Free Survival) from >1,600 lung cancer patients HE images, achieving a significantly higher AUC (Area Under the Receiver Operating Characteristic Curve) of 0.73 compared to state-of-the-art alternatives (0.58-0.64).</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.15.659438v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">LadybirdBase: a comprehensive biology, ecology and omics resource for ladybird beetles (Coccinellidae)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sun, Y.-F., Yang, K.-Y., Li, H., Liang, Y.-S., Cai, L.-Q., Xie, J.-Y., Zhang, Y.-W., Liang, J.-Y., Mou, Q., Wang, Y.-M., Chen, D., Qi, M.-X., Carlos Ramos Aguila, L., Li, H.-S., Pang, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed LadybirdBase, a comprehensive database that compiles both published and self-collected data on ladybird beetles, including their classification, morphology, diet, distribution, and omics data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Ladybird beetles (Coleoptera: Coccinellidae) represent a highly diverse insect group comprising over 6000 species. Recently, an increasing amount of omics data on ladybirds has accumulated. To better understand their roles and adaptations in biological control, we developed LadybirdBase (http://www.ladybirdbase.com/), a comprehensive database that compiles both published and self-collected data on ladybird beetles. It contains five main modules that encompass: (1) the Biology Module summarizing information on over 6000 species, including their classification, morphology, diet, distribution, and omics data; (2) the Ecology Module offering diet and distribution records; (3) the Genomics Module integrating published and unpublished resources such as genomes, transcriptomes, mitogenomes, orthogroups, and variance; (4) the Microbiomics Module presenting 16S rRNA, ITS, and metagenome data; and (5) the Lab Test Module covering diet ranges, life histories under different treatments, and stress tolerance. Additionally, LadybirdBase offers analysis tools for species detection using morphology or barcodes, BLAST searches against the genome resources, differential gene expression analysis, and PCR primer search. LadybirdBase can greatly simplify species identification, enhance the efficiency of omics studies, and provide a theoretical foundation for the use of ladybirds in biological control.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.15.659824v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">One Sea, Different Whales: Genomics Sheds Light on a Small Population of Fin Whales</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Biello, R., Iannucci, A., Fuselli, S., Desiato, E., Urban R., J., Fossi, M. C., Panti, C., Mancia, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We sequenced and analysed the whole genomes of the Mediterranean fin whale to provide a comprehensive genomic characterization of this population, assessing its genomic variability, genetic load, population structure and potential for adaptation to environmental stressors.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Whales play a crucial role in marine ecosystems, yet many populations face growing environmental pressures. Among them, the fin whale, Balaenoptera physalus, from the Mediterranean Sea remains poorly characterized at the genomic level despite its classification as Endangered and decreasing in population size due to habitat degradation, climate change, and human-induced disturbances. Previous studies based on bioacoustics and telemetry data suggest the presence of resident and migratory subgroups, but the genetic isolation of this population remains uncertain. Here, we sequenced and analysed for the first time whole genomes of the Mediterranean fin whale to provide a comprehensive genomic characterization of this population, assessing its genomic variability, genetic load, population structure and potential for adaptation to environmental stressors. By comparing genomic data from Mediterranean fin whales with populations from the North Atlantic, North Pacific, and the Sea of Cortez (also sequenced in the present study), we aim to determine the degree of genetic isolation and contextualize the Mediterranean population within a broader evolutionary and conservation framework. Our results showed that while Mediterranean fin whales form a distinct genetic cluster, they are not entirely isolated from North Atlantic populations, with gene flow persisting at low levels. Furthermore, we identified a complex substructure within the Mediterranean, supporting the existence of a resident subpopulation. Although this population preserves moderate levels of genomic diversity and adaptive potential, it remains vulnerable to genomic erosion due to continued demographic decline, limited connectivity, and increasing environmental stressors. These findings underscore the urgency of targeted conservation actions and long-term genetic monitoring, especially as climate change accelerates and introduces increasingly unpredictable selective pressures that may exacerbate the risk of genomic erosion and threaten population viability.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.16.659691v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Development of a Nested Association Mapping (NAM) population for untangling complex traits in lentil (Lens culinaris Medik.)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Neupane, S., Stonehouse, R., Ramsay, L., Baker, M., Haile, T., Konkin, D. J., Sidebottom, C., Bett, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A multi-parental Nested Association Mapping (NAM) design in lentil that can detect minor-effect loci often overlooked by traditional biparental or diversity panel approaches.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding the genetic basis of complex traits remains a key challenge in crop improvement. This study aimed to develop a structured, multi-parental mapping population to enhance the resolution of quantitative trait dissection, based on the hypothesis that a Nested Association Mapping (NAM) design would enable the detection of minor-effect loci often overlooked by traditional biparental or diversity panel-based approaches. A lentil (Lens culinaris Medik.) NAM population was developed by crossing the Canadian cultivar CDC Redberry with 32 diverse genotypes sourced globally from three major lentil-growing macro-environments: Northern temperate, Mediterranean, and South Asia. The resulting recombinant inbred lines were phenotyped for key phenological traits, days to emergence (DTE), flowering (DTF), and maturity (DTM), under field conditions, and genotyped using exome capture sequencing. Genome-wide association studies for DTF identified 14 significant loci across six chromosomes, including the known FTb locus and novel associations near AP3a, HUB2a, and PIF6 genes. These results demonstrate the utility of the NAM design in detecting both major and minor-effect loci that underlie complex trait variation. To our knowledge, this is the first publicly available NAM population in lentil. It provides a high-resolution, globally representative platform for trait discovery, pre-breeding, and collaborative genetic improvement of this nutritionally and agronomically important legume.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.17.660011v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Safeguarding evolutionary history in a biodiversity hotspot: downscaling the EDGE metric to a regional application</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ramos-Gutierrez, I., Pipins, S., Molina-Venegas, R., Fernandez-Mazuecos, M., Mejias, P. J., Moreno-Saiz, J. C., Forest, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of evolutionary history in the Iberian Peninsula and the global conservation relevance of phylogenetic singularity with extinction risk .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In the face of accelerating biodiversity loss, phylogenetically informed approaches offer critical insights for conservation planning from an evolutionary perspective. The EDGE (Evolutionarily Distinct and Globally Endangered) metric combines phylogenetic singularity with extinction risk to identify taxa that represent irreplaceable evolutionary history under threat. In this study, we focus on the global conservation relevance of angiosperms native to a regional biodiversity hotspot, the Iberian Peninsula. Our aim was to identify areas with a high concentration of EDGE species and to evaluate how effectively the current protected area network encompasses them. Our findings reveal that threatened evolutionary history is primarily concentrated in mountainous and coastal regions. While several EDGE zones -areas containing unique and jeopardized evolutionary lineages- overlap with existing protected areas, particularly in mountains, others harbouring few but evolutionarily unique and highly threatened taxa remain largely unprotected. This study highlights the value of applying global conservation metrics such as EDGE at regional scales. Our results provide a foundation for integrating evolutionary history into conservation prioritization in the Iberian Peninsula and offer a replicable framework for implementing the EDGE approach in other biodiversity-rich regions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.16.659859v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Metabolite-based genome-wide association studies enable the dissection of the genetic bases of bioactive compounds in Chickpea seeds</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ruan, S., Rocchetti, L., Bitocch, E., Wendenburg, R., Santamarina, C., Di Vittori, V., Istanbuli, T., Hamwieh, A., Fernie, A. R., Papa, R., Alseekh, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A genetic analysis of chickpea seeds' metabolite diversity and the effects of domestication process on chickpeas' metabolome .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Chickpea, is the second most consumed food legume, and significantly contributes to the human diet. Chickpea seeds are rich in a wide range of metabolites including bioactive specialized metabolites influencing nutritional qualities and human health. However, the genetic basis underlying the metabolite-based nutrient quality in chickpea remains poorly understood. Here we dissected the genetic architecture of seed metabolic diversity and explored how domestication shaped the chickpea metabolome. Through UPLC-MS we quantify over 3400 metabolic features in 509 chickpea seeds accessions from three independent multi-location field trials. The metabolite genome-wide association study (mGWAS) detected around 130,000 leading SNPs corresponding to 1890 metabolites across different environments. We further found and functionally validated a gene cluster of three CabHLH transcription factors that regulate soyasaponin biosynthesis in chickpea seeds. Our results reveal new insights on the effects of domestication process on chickpea metabolome. and provide valuable resources for the genetic improvement of the bioactive compounds in chickpea seeds.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.16.659862v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Microbial diversity of Atlantic Rainforest ponds assessed by nanopore sequencing</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Atum, S. V., Soares, D. M., Santos, I., Johnson, G. A., Domingos, A. H., Bechara, E. J., Setubal, J. C., Urban, L., Stevani, C. V., Freire, R. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A metagenomic survey of microbial diversity in three ponds in the Atlantic Rainforest biome in Brazil.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Despite their limited size, ponds are ecologically important habitats that harbor rich microbial communities and play key roles in global biodiversity and biogeochemical cycling. In this study, we conducted a metagenomic survey of microbial diversity in three ponds - Vermelha, Grande, and Furnas - situated within the Atlantic Rainforest biome in Brazil. Vermelha and Grande are natural, remote, and minimally impacted by human activity, whereas Furnas is an artificial pond with greater human accessibility and long-term anthropogenic influence. Using a long-read nanopore shotgun metagenomics approach, we sequenced DNA extracted from pond water, assembled metagenomes, and performed taxonomic classification and functional annotation. From these datasets, 21 metagenome-assembled genomes (MAGs) were recovered. Our results reveal pond-specific microbial community compositions and functional profiles. Although a direct temporal comparison would require synchronized sampling, the current data offer compelling snapshots of microbial diversity. Vermelha was characterized by high cyanobacterial diversity and exclusive Planctomycetes, while Furnas displayed broader taxonomic diversity, and more notable Actinobacteria and Patescibacteria. Viral communities, including dsDNA bacteriophages and eukaryotic viruses, also varied across sites, with Prasinovirus consistently detected in all ponds. Functional annotation identified putative antibiotic resistance genes (ARGs), most of them in Furnas, potentially reflecting human impact. Besides ARGs, cyanotoxin biosynthesis genes-especially those linked to microcystin production- were predominantly found in Vermelha. Our findings underscore the ecological distinctiveness of each pond and demonstrate the utility of nanopore-based metagenomics for investigating microbial biodiversity in understudied freshwater ecosystems. This work provides knowledge on microbial community structure and anthropogenic influence in tropical pond environments, with implications for conservation and biotechnological exploration.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.17.660208v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Interactive delineation and quantification of anatomical structure with virtual reality</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Garkov, D., Lein, E., Kielkopf, N., Dullin, C., Klein, K., Sommer, B., Jordan, A. L., Schreiber, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We apply virtual reality as a method to visualise and derive higher-level anatomical features from low-level descriptors.</p><p style="font-size: 16px; text-align: justify;">Background: Full tissue segmentation is laborious, especially for non-model organisms, whereas accurate and reliable delineation still requires much firsthand visual inspection. A virtual environment can be equipped with suitable data representations, interaction techniques, and method interfaces as to enable the interactive delineation and quantification of anatomical structure. Situated in such an environment, analysts can benefit from reduced pre-processing, but also from in-situ learning and collaboration. Results: Therefore, we apply virtual reality as a method to visualise and derive higher-level anatomical features from low-level descriptors. Following voxel-size calibration, scalable delineations and measurements are performed in virtual reality. The data representation for delineation is volume visualisation: a volume rendering or an isosurface mesh. Two delineation techniques are proposed for the placement and editing of points and segments in virtual reality. For quantification, different measures and metrics can be computed for each delineated region. To mitigate some of the fundamental challenges of virtual reality, e. g., mid-air interaction affecting precision at a distance, different virtual-reality affordances were considered as part of the design. As a result, we present Brainacle, a virtual reality application, and make its usage freely available. We incorporate Brainacle in a synchrotron tomography reconstruction pipeline to delineate and quantify the gross brain regions of 20 individuals from six species of African cichlid fish. Conclusion: Brainacle, an editor for the interactive delineation and quantification of anatomical structures in virtual reality, is applicable to different biological pipelines and workflows. In particular, Brainacle can be used to quickly gain an overview of structure, ease repetitive delineation and measurement, and visually inspect and communicate findings.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.17.659041v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

