<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Tuesday, 09 September 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Long-read single-cell genome, transcriptome and open chromatin profiling links genotype to phenotypes.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pancikova, A., Cools, R., Eftychiou, M., Aertgeerts, M., Vande Velde, J., Segers, H., Cools, J., Harbers, L., Demeulemeester, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Current single-cell multiomics methods typically provide limited genomic information, constraining genotype-phenotype studies. To address this gap, we developed SPLONGGET (Single-cell Profiling of LONG-read Genome, Epigenome, and Transcriptome), which integrates 10X Genomics barcoding with Oxford Nanopore sequencing to simultaneously profile genome, chromatin accessibility, and full-length transcriptomes in thousands of single cells. By retaining all tagmentation fragments during library preparation, SPLONGGET delivers whole-genome coverage, supports target enrichment for effective single-cell genotyping, and remains backwards compatible with existing short-read workflows. SPLONGGET enables comprehensive calling of small variants, structural variants, and copy number alterations. Applying SPLONGGET to paediatric B-cell acute lymphoblastic leukaemia revealed clonal dynamics and the phenotypic effects of somatic variants. Notably, we evidence parallel evolution of immune escape variants with four distinct splice site mutations and loss of heterozygosity in the CAR T-cell therapy CD19 target. In conclusion, SPLONGGET enables integrated high-throughput analysis of genetic variation and molecular phenotypes using off-the-shelf kits, offering a timely and powerful tool to study genetically heterogeneous samples, such as tumours but also ageing normal tissues.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.08.674950v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Changes in sea ice alter genetic structure of an iconic Arctic apex predator in less than three decades</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Vanderluit, S. N., Fortin, M.-J., Van Coeverden de Groot, P., Clemente-Carvalho, R. B. G., Jensen, E. L., Gomez-Sanchez, A., Sun, Z., Dyck, M., Branigan, M., Lougheed, S. C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Climate change is having profound effects on biodiversity and species distributions worldwide. Nowhere are these effects potentially more pronounced than in the Arctic, where warming is almost two times the global average, and where year-round sea ice extent has significantly decreased, affecting many ice-dependent species. The polar bear (Ursus maritimus) is a circumpolar, apex Arctic predator, a sentinel of climate change, and a symbol of conservation. It is of immense cultural and spiritual importance to Inuit peoples and is hunted across the Arctic. Declines in sea ice have caused habitat fragmentation and loss, disrupting movement and prey access, potentially altering genetic structure and influencing polar bear's potential to persist. Using samples collected from 1997 to 2020 by Inuit across much of the Canadian Arctic, 322 genome-wide autosomal DNA markers specifically designed to quantify polar bear genetic structure and a very stringent spatial-temporal method, we compare polar bear spatial genetic structure and landscape features between two periods across a consistent distribution: 1997-2008 and 2009-2020. We observe marked changes in spatial genetic structure across the Arctic Archipelago over just two decades, shifting the boundaries between polar bear genetic clusters by ~250 km. Landscape resistance models reveal the importance of sea ice and land cover type for each period, with spatial lag models showing that genetic change is best predicted by sea ice shifts between periods. Our study reveals rapid changes in genetic structure of polar bears in the Canadian Arctic, helps to inform conservation and management, and offers insight on future polar bear persistence across its immense, remote distribution.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.06.674276v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Revolution and advances in gene editing and genomics technology for developing climate-resilient legume crops: developments and prospects.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gayatri Mishra</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Legumes are essential for agriculture and food security. Biotic and abiotic stresses pose significant challenges to legume production, lowering productivity levels. Most legumes must be genetically improved by introducing alleles that give pest and disease resistance, abiotic stress adaptability, and high yield potential. The quickest way to develop high-yielding elite legume varieties with long-lasting resistance is to tap into potential resistance alleles present in landraces and wild relatives and exploit them in legume resistance breeding programs using next-generation molecular breeding methods. Most of the reviews focus on the advancements made by genome editing technologies in generating climate-tolerant legumes for breeding. This review discusses the challenges of genome-based editing tools and how the integration of other popular breeding methodologies, such as QTLs and GWAS, as well as computational techniques, can aid in the development of climate-tolerant legume crops. This review highlights genomics-based methodologies and recent advances that make it easier to assess genetic diversity and uncover adaptive genes in legumes. Computational approaches, such as machine learning, are important in mining the breeding-related genes identified by CRISPR and other genomic tools, as well as detecting the key elements and factors that regulate the expression of these genes, which addresses the challenge of developing climate-resilient legume crops.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40920217/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Improving cryo-EM maps by resolution-dependent and heterogeneity-aware deep learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cao, H., Li, T., Chen, J., He, J., Huang, S.-Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cryo-electron microscopy (cryo-EM) has emerged as a leading technology for determining the structures of biological macromolecules. However, map quality issues such as noise and loss of contrast hinder accurate map interpretation. Traditional and deep learning-based postprocessing methods offer improvements but face limitations particularly in handling map heterogeneity. Here, we present EMReady2, a substantial evolution of our previously developed deep learning-based methodology, towards a generalist model for improving cryo-EM maps. EMReady2 introduces a local resolution-dependent training strategy, allowing it to effectively address map heterogeneity. Additionally, EMReady2 extends its applicability to cryo-EM maps with nucleic acids, intermediate-resolution maps, and cryo-electron tomography (cryo-ET) maps. Extensive evaluations on diverse test sets of 94 maps at 2.0-10.0 [A] resolutions demonstrate that EMReady2 is a universally applicable post-processing model for improving EM density maps in terms of both quality and interpretability, and significantly outperforms existing methods. EMReady2 is freely available at http://huanglab.phys.hust.edu.cn/EMReady2/.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.03.674102v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Large-language-model-based antibiotic resistance gene prediction and resistomes mining in cyanobacterial blooms</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Li, H., Qi, M., Wang, X., Liu, H., Yang, Q., Chen, H., Jiang, R., Chen, T., Yang, Y., Zhu, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The rapid spread of antibiotic resistance genes (ARGs) in aquatic ecosystems poses a serious public health threat. Conventional ARG detection methods, including sequence-alignment and machine learning, are limited by high false-negative or false-positive rates, especially due to sequence diversity and class imbalance in metagenomic data. Moreover, few tools offer comprehensive solutions for ARG identification, classification, and functional annotation. To address these limitations, we developed ESMARG, a novel protein language model framework based on ESM1v, for accurate ARG identification, classification, and annotation. Trained with both ARG and abundant non-ARG sequences from real-world metagenomes to enhance robustness and reduce false positives, ESMARG significantly outperformed traditional alignment-based (BLAST, DIAMOND) and recent deep learning models (ARGNet, ARG-SHINE), achieving 0.998 precision, 0.939 recall, and 0.968 F1-score for ARG identification. Functional and mechanism classification modules also demonstrated high accuracy (0.986 and 0.987) and computational efficiency. We applied ESMARG to analyze 26 cyanobacterial aggregate (CA) metagenomes collected from Lake Taihu across a full annual cyanobacterial harmful algal bloom (CyanoHAB) cycle. A total of 110 ARGs, spanning 24 drug classes and multiple resistance mechanisms, were detected, with seasonal shifts in ARG abundance and composition. Strong correlations were identified between CA resistomes and microbial community structure, especially with cyanobacteria. Further, ARG abundance was shaped by both biotic factors-such as cyanobacterial dominance and bacterial functional profiles-and abiotic variables, including biochemical oxygen demand and water temperature. Our findings demonstrate the power of ESMARG for high-resolution ARG profiling and reveal complex ecological interactions between resistomes, microbiomes, and environmental factors in CyanoHAB.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.03.674132v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Treasure: A Sensitive Pipeline for Species-Level and Functional Microbiome Profiling</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Avelar, D. d. S., Teixeira, E. B., Casseb, S. M. M., Moreira, F., Assumpcao, P. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Next Generation Sequencing (NGS) methods, such as 16S rRNA amplicon sequencing and Whole Genome Sequencing (WGS), enable taxonomic analyses but have limitations. This project proposes the development of a computational tool capable of performing functional analysis of the most abundant microorganisms within a microbiome based on taxonomic analysis. The proposed method integrates the tools Kraken, Gffread, and Salmon. Compared to Samsa 2, a commonly used pipeline for RNA-Seq Total samples, the new approach demonstrated superior performance across all evaluated scenarios (p < 0.01). The tool aims to functionally characterize the microbiome of regions affected by Gastric Cancer (GC) and adjacent areas, assess associations between survival, expression/abundance, and identify potential microbial biomarkers for GC.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.03.674071v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A multifaceted approach for obstructive sleep apnea classification from ECG signal using deep learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Varghese, A. J., Gatsonis, A., Agraz, M., Oommen, V., Parulkar, A., Chu, A., Karniadakis, G. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Obstructive sleep apnea (OSA) is a common sleep disorder associated with increased cardiovascular and neurocognitive risks. While polysomnography remains the clinical gold standard for diagnosis, it is costly and unsuitable for large-scale or real-time screening. Electrocardiogram (ECG) signals offer a non-invasive, low-cost alternative for sleep apnea detection. We present a holistic new framework for OSA detection and forecasting using ECG data on two datasets: PhysioNet Apnea-ECG datasets (healthy patients with apnea), and OSASUD dataset (patients in a stroke unit). Our framework integrates feature engineering methods rooted in dynamical systems theory and statistical analysis. These features are used across a range of models, from conventional machine learning algorithms to novel deep learning architectures. To improve generalization and personalization, we incorporate transfer learning in two ways: across datasets to adapt models trained on large cohorts to smaller clinical datasets, and at the patient level to personalize models using limited individual data, hence demonstrating the use of precision medicine.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.03.673993v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Guide RNA library representation results in gene essentiality prediction bias in genome-wide CRISPR screens</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Metz, P., Alves-Vasconcelos, S., Wallbank, R., Riepsaame, J., Hassan, A. B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genome wide CRISPR-based perturbation screens are powerful discovery tools enabling the identification of novel gene dependencies through either gain or loss of function. While genome wide guide RNA (gRNA) libraries have advantages when using enAsCas12a, such as multiplex single gRNAs per gene, they may be subject to similar confounding factors that can affect the interpretation of large genome-wide datasets. Here, we examine the impact of these variables in over twenty enASCas12a multiple gRNA based perturbation screens performed using Humagne C, Humagne D and Inzolia libraries in human cells. We demonstrate that the choice of CRISPR library is often the most significant factor that influences genetic perturbation results, outweighing other variables such as either target cell lines or culture media conditions. A major contributor to this effect is gRNA representation bias within a given CRISPR library, where lower gRNA representation can lead to variable and more pronounced gene effect scores using either log fold change or Chronos analysis. These effects may be mitigated by using either multiple gRNA constructs per gene, by optimisation of CRISPR library production processes or by targeting with multiple independent gRNA libraries. Importantly, we also consider gRNA representation bias during CRISPR screen hit prioritisation. CRISPR library gRNA representation bias remains a major challenge in the interpretation of gene essentiality in perturbation screens.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.07.673082v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Knowledge-Driven Interpretable Neural Networks for Mechanistic Insight</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ke, Y., Yu, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Analyzing omics data based on pathway knowledge is critical for understanding the molecular mechanism behind pathological changes, but current pathway analysis methods do not model the detailed mechanistic nature of biological interactions, limiting the understanding of pathway behavior to a relatively shallow level. To address this issue, we present a knowledge-driven machine learning framework that embeds features into pathway graphs and models reactions analytically, producing interpretable feature hierarchies and sub-networks where functional associations are estimated to model biological interactions. The approach is agnostic to feature selection, enabling the use of full omics datasets without discarding weak signals. Applications to breast cancer microRNA-gene regulation data and COVID-19 metabolomics data highlight immune and metabolic pathways relevant to disease progression. This framework bridges predictive modeling with mechanistic interpretation, offering a foundation for integrative pathway analysis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.08.674894v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">peakScout - a user-friendly and reversible peak-to-gene translator for genomic peak calling results</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lin, A. L., Cartailler, L. A., Cartailler, J.-P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> peakScout is a command line and web-based bioinformatics tool designed to quickly and easily bridge the gap between genomic peak data and gene annotations, enabling researchers to understand the relationship between measurements of regulatory elements and their target genes. At its core, peakScout processes genomic peak files obtained through various means chromatin profiling and maps them to nearby genes using reference genome annotations. The workflow begins with input processing, where peak files are standardized and reference GTF files are decomposed into chromosome-specific feature collections. The core analysis modules then perform bidirectional mapping: peak-to-gene identifies which genes are potentially regulated by specific genomic regions, while gene-to-peak reveals which regulatory elements might influence particular genes of interest. Throughout this process, nearest-feature detection algorithms handle the complex spatial relationships between genomic elements, considering factors like distance constraints and feature overlaps. Finally, the results are formatted into researcher-friendly CSV and Excel outputs, providing a comprehensive view of the genomic landscape that connects regulatory elements to their potential gene targets.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.07.671934v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MMonitor: Software for Real-Time Monitoring of Microbial Communities Using Long Reads</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lucas, T. N., Biehain, U., Gautam, A., Gemeinhardt, K., Lass, T., Konzalla, S., Ley, R. E., Angenent, L. T., Huson, D. H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Real-time monitoring of microbial communities offers valuable insights into microbial dynamics across diverse environments. However, many existing metagenome analysis tools require advanced computational expertise and are not designed for monitoring. We present MMonitor, an open source software platform for the real-time analysis and visualization of metagenomic Oxford Nanopore Technologies (ONT) sequencing data. MMonitor includes two components: a desktop application for running bioinformatics pipelines through a graphical user interface (GUI) or command-line interface (CLI), and a web-based dashboard for interactive result inspection. The dashboard provides taxonomic composition over time, quality scores, diversity indices, and taxonomy-metadata correlations. Integrated pipelines enable automated de-novo assembly and reconstruction of metagenome-assembled genomes (MAGs). To validate MMonitor, we tracked human gut microbial populations in three bioreactors using 16S rRNA gene sequencing, and applied it to whole-genome sequencing (WGS) data to generate high-quality annotated MAGs and reveal functional insights. We also compare MMonitor to other software for real-time metagenomic analysis, highlighting the strengths and limitations of each tool for this use case. By performing automated time-series analyzes, sample management, and updating of reference databases, MMonitor addresses current limitations and supports dynamic microbiome research in various fields.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.08.674848v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Unified Protein-Small Molecule Graph Neural Networks for Binding Site Prediction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, J., Dokholyan, N. V.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Predicting small molecule binding sites on proteins remains a key challenge in structure-based drug discovery. While AlphaFold3 has transformed protein structure prediction, accurate identification of functional sites such as ligand binding pockets remains a distinct and unresolved problem. Graph neural networks have emerged as promising tools for this task, but most current approaches focus on local structural features and are trained on relatively small datasets, limiting their ability to model long-range protein-ligand interactions. Here, we develop YuelPocket, a new graph neural network that addresses these limitations through an innovative global graph design featuring a virtual joint node connecting all protein residues and small molecule atoms to capture long-range interactions while maintaining linear computational complexity. Trained on the comprehensive MOAD dataset containing over 38,000 protein-small molecule complexes, YuelPocket achieves exceptional performance with AUC-ROC values of 0.85 and 0.89 on COACH420 and Holo4k datasets, respectively.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.03.674017v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genomic heterogeneity inflates the performance of variant pathogenicity predictions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lu, B., Liu, X., Lin, P.-Y., Brandes, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent studies have reported unprecedented accuracy predicting pathogenic variants across the genome, including in noncoding regions, using large AI models trained on vast genomic data. We present a comprehensive evaluation of these frontier models, showing that performance is inflated by differences in the prevalence of pathogenic variants across genomic contexts. We identify the best-performing models for each variant type and establish a benchmark to guide future progress.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.05.674459v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Phylogenetic Analysis of TET2 Gene Variants in Pakistani Acute Myeloid Leukemia Patients</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Abakar, M. E. A., Abaker, M. A. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;">Background: Acute myeloid leukemia (AML) is a heterogeneous malignancy caused by the proliferation of neoplastic myeloid progenitor cells. Mutations in ten-eleven translocation methylcytosine dioxygenase 2 (TET2), a key regulator of DNA demethylation, are common in AML and play a role in its progression and response to treatment. Methods: We performed a phylogenetic analysis of 25 TET2 gene sequences (12 AML and 13 normal) from Pakistani individuals using multiple sequence alignment, phylogenetic tree construction, and distance matrix evaluation. We also identified population-specific expression patterns and mutational hotspots. Results: AML samples showed significant deletions in the catalytic domain (positions 693-876) and lower sequence conservation compared to controls. These deletions may impair TET2 enzymatic activity and disrupt the epigenetic regulation. The Phylogenetic tree also showed two significant clades supported by high bootstrap values, differentiating AML from control individuals in the evolutionary direction. Conclusion: Our findings reveal distinct TET2 mutational hotspots associated with acute myeloid leukemia (AML) in the Pakistani population. These results have implications for targeted epigenetic therapy and motivate large-scale, cross-population research to unravel the global effects of TET2 variations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.03.674098v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MondoA mediates transcriptional coordination between the MYC network and the integrated stress response in pancreatic ductal adenocarcinoma</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ramsey, E. L., Dobersch, S., Freie, B., Hong, N. H., Wu, X., Kugel, S., Eisenman, R. N., Carroll, P. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> MYC amplification contributes to poor survival and outcome in pancreatic ductal adenocarcinoma (PDAC). Here we show that in PDAC cell lines with amplified MYC, MondoA is required for viability, facilitating proliferation while suppressing apoptosis in vitro and in vivo. Transcriptional and genomic profiling demonstrates that loss of MondoA leads to altered expression of direct MondoA targets as well as MYC target genes and is accompanied by shifts in genomic occupancy of MYC, MNT, and the MondoA paralog ChREBP. This altered genomic binding by MYC network members is associated with transcriptional perturbation of multiple metabolic and stress pathways, as well as global changes in N6-methyladenosine modification (m6A) of mRNA. MondoA inhibition disrupts coordination between MYC network members and the Integrated Stress Response (ISR), resulting in decreased translation of ATF4 mRNA, discordant gene regulation of shared targets of MYC and ATF4 and, ultimately, apoptosis. Re-establishing ATF4 protein expression rescues the diminished viability due to loss of MondoA expression or activity, providing direct evidence of a link between deregulated MYC and the transcriptional machinery of the ISR. Lastly, we find that small-molecule inhibition of MondoA is lethal in a subset of PDAC cell lines, including patient-derived organoids, suggesting that the ability to target MYC via chemical inhibition of MondoA transcriptional activity may have broad efficacy.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.03.674106v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

