<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Friday, 16 May 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Exon-intron boundary detection made easy by physicochemical properties of DNA.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dinesh Sharma, Danish Aslam, Kopal Sharma, Aditya Mittal, B Jayaram</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present ChemEXIN, an innovative novel method for predicting exon-intron boundaries, with notable accuracy and precision, with potential implications for understanding gene expression, regulation, and cellular functions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genome architecture in eukaryotes exhibits a high degree of complexity. Amidst the numerous intricacies, the existence of genes as non-continuous stretches composed of exons and introns has garnered significant attention and curiosity among researchers. Accurate identification of exon-intron (EI) boundaries is crucial to decipher the molecular biology governing gene expression and regulation. This includes understanding both normal and aberrant splicing, with aberrant splicing referring to the abnormal processing of pre-mRNA that leads to improper inclusion or exclusion of exons or introns. Such splicing events can result in dysfunctional or non-functional proteins, which are often associated with various diseases. The currently employed frameworks for genomic signals, which aim to identify exons and introns within a genomic segment, need to be revised primarily due to the lack of a robust consensus sequence and the limitations posed by the training on available experimental datasets. To tackle these challenges and capitalize on the understanding that DNA exhibits function-dependent local physicochemical variations, we present ChemEXIN, an innovative novel method for predicting EI boundaries. The method utilizes a deep-learning (DL) architecture alongside tri- and tetra-nucleotide-based structural and energy features. ChemEXIN outperforms existing methods with notable accuracy and precision. It achieves an accuracy of 92.5% for humans, 79.9% for mice, and 92.0% for worms, along with precision values of 92.0%, 79.6%, and 91.8% for the same organisms, respectively. These results represent a significant advancement in EI boundary annotations, with potential implications for understanding gene expression, regulation, and cellular functions.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40094442/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Tissue-specific restriction of TE-derived regulatory elements safeguards cell-type identity</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Milovanovic, D., Duc, J., Matsushima, W., Hamelin, R., Planet, E., Offner, S., Rosspopoff, O., Trono, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A noncanonical, TRIM28-independent role for KZFPs in shaping cell-type-specific regulatory landscapes and emphasize the importance of repressing alternative regulatory programs alongside activating lineage-specific ones to safeguard cell identity.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Transposable elements (TEs) have extensively reshaped the cis-regulatory landscape of mammalian genomes, yet the mechanisms that govern their context-specific activity remain incompletely understood. KRAB zinc finger proteins (KZFPs), a large family of transcription factors specialized in TE recognition, are known repressors of TE-derived regulatory activity through TRIM28-mediated H3K9me3 deposition. Here, we expand this paradigm by uncovering noncanonical relationships between TEs and KZFPs. By generating a comprehensive epigenomic map of KZFP-bound TEs, we find that the regulatory activity of ancient mammalian L2/MIR elements is broadly delineated by KZFP binding patterns, despite low H3K9me3 enrichment. We further dissect this relationship by investigating the function of ZNF436, a non-canonical KZFP, highly expressed during in vivo human fetal heart development. Using loss-of-function approaches, we show that ZNF436 preserves cardiomyocyte function by promoting cardiac gene expression while restricting the activation of alternative lineage programs. Mechanistically, ZNF436 recruits specialized SWI/SNF chromatin remodeling complexes to limit the accessibility of L2/MIR-derived enhancers, many of which are active in non-cardiac tissues. These findings reveal a noncanonical, TRIM28-independent role for KZFPs in shaping cell-type-specific regulatory landscapes and emphasize the importance of repressing alternative regulatory programs alongside activating lineage-specific ones to safeguard cell identity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.13.653700v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identification of a sex-determining locus potentially involved in a conflict over sex-ratio</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lheraud, B., Dussert, Y., Chebbi, M. A., Giraud, I., Cordaux, R., Peccoud, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a new model of sex determination in terrestrial isopods and propose a candidate genomic region of 34 annotated genes, including one that encodes the androgenic gland hormone, a protein involved in male sexual differentiation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Sex-determining genes remain largely uncharacterized outside classical models in vertebrates and insects, leaving a gap in our understanding of their evolutionary emergence and sex chromosome formation. Terrestrial isopods, particularly the common pill bug Armadillidium vulgare, provide an excellent model for investigating these processes due to the rapid turnover in sex-determination mechanisms they undergo. In A. vulgare, multiple genetic determinants coexist. Notably, a feminizing factor is transmitted at a rate exceeding Mendelian expectations, resulting in female-biased populations. Some lineages possess a masculinizing allele at a locus referred to as the "M gene", which is functionally analogous to an XY system. Its masculinizing dominant allele is hypothesized to have been selected due to the deficit of males caused by the feminizing factor. The existence of the M gene was inferred from crosses carried out in the 1990s, but its molecular nature remains unresolved. Here, we conducted a genome-wide SNP analysis combining pooled sequencing of male and female progenies with sequencing of individual parents across two families. Bayesian estimation of haplotype frequencies in progenies enabled us to delimit a candidate genomic region of approximately two megabases containing 34 annotated genes. Most notably, one of these genes encodes the androgenic gland hormone, a protein involved in male sexual differentiation. Our findings lay the groundwork for detailed genetic and functional investigations of the M gene, offering novel insights into the dynamics of sex determination in terrestrial isopods and into the turnover of sex chromosomes in response to sex-ratio distortion.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.14.654164v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Deconstructing the Cancer Epigenome using Reduced Representation Bisulfite Sequencing (RRBS)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Glendinning, N., Chung, D., Hardiman, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> This chapter presents a step-by-step pipeline for the processing of raw RRBS reads including quality control, trimming, alignment, methylation extraction, differential methylation analysis, annotation, and downstream analysis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Reduced Representation Bisulfite Sequencing (RRBS) represents a relatively inexpensive methodology for investigating the influence of DNA methylation on the development and progression of diseases including cancer. RRBS is a targeted approach which combines the use of restriction enzyme selection, bisulfite conversion, and high throughput sequencing technologies. The computational processing and analysis of RRBS data therefore requires modified approaches compared to typical DNA sequencing, due to deviations from standard laboratory protocols. This chapter presents a step-by-step pipeline for the processing of raw RRBS reads including quality control, trimming, alignment, methylation extraction, differential methylation analysis, annotation, and downstream analysis. This pipeline is designed to ensure optimal processing for accurate results and to provide downstream analysis tools that can extract biological insights from the data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.12.653548v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrating the ENCODE blocklist for machine learning quality control of ChIP-seq with seqQscorer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Albrecht, S., KraÃàmer, C., RoÃàchner, P., Mayer, J. U., Rothlauf, F., Andrade-Navarro, M. A., Sprang, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We improve the quality assessment of next-generation sequencing data by up to 4.4%, when using ENCODE blocklist features, which are a set of genomic regions known to be related to the quality of NGS data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: Quality assessment of next-generation sequencing (NGS) data is a complex but important task to ensure correct conclusions from experiments in molecular biology, biomedicine, and biotechnology. While there are multiple software tools to support this process, it is still mainly driven by domain experts manually curating quality reports. This time-consuming effort is expected to increase with the future capacity to generate more sequencing samples, at lower costs, using advanced technologies. By leveraging quality-labelled data from large databases and automatically created quality-related features derived from raw sequencing samples, supervised machine learning shows great potential for supporting quality assessment, as shown by the seqQscorer tool. This tool shows outstanding results for automated quality assessment and has been used to investigate several quality-related aspects, integrating large-scale sequencing datasets. To improve seqQscorer in terms of accuracy and processing time, we explored the potential of creating features that are more informative and faster in generation than those conventionally used by seqQscorers models. For this purpose, we used the ENCODE blocklist*, a set of problematic genomic regions known to be related to the quality of NGS data. Results: Our results show that we can improve the quality assessment for ChIP-seq samples derived from human tissues and cell lines when using features based on the ENCODE blocklist by up to 4.4%, when using a generic dataset containing single-end and paired-end sequencing samples. For other assays, these new blocklist features allow for highly accurate quality assessment, in general, but are less accurate in comparison to using the conventional seqQscorer features. Outlook: These results strongly encourage an extension of the seqQscorer tool that automatically integrates the new blocklist features. Users investigating human protein-DNA interactions would benefit from this extension, as it not only improves model performance during quality assessment but also enhances usability by simplifying the installation procedure and reducing the computational resources required for feature generation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.12.653555v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A computational strategy to uncover fusion genes in prostate cancer cell lines</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Morgan, R. A., Hardiman, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Fusion genes, chimeric transcripts formed by the merging of two distinct genes due to chromosomal structural changes (e.g., inversions or trans/cis-splicing).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Fusion genes, chimeric transcripts formed by the merging of two distinct genes due to chromosomal structural changes (e.g., inversions or trans/cis-splicing), are established cancer drivers. Advances in genomic technologies, particularly RNA sequencing and improved fusion gene prediction algorithms, have significantly expanded our understanding of fusion genes in cancer. This chapter explores computational methods for identifying fusion genes through RNA sequencing data, using the TMPRSS2::ERG fusion in prostate cancer cell lines as a case study, and includes analysis of both fusion-positive and fusion-negative cell lines. To achieve high-confidence detection, three open-source fusion prediction tools, STAR-Fusion, FusionCatcher, and JAFFA are investigated. These tools were selected for their accessibility, active maintenance, and strong performance in benchmarking studies. Their sensitivity and accuracy in detecting TMPRSS2::ERG is systematically evaluated and validated, ensuring robust and high-resolution detection of fusion events.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.12.653554v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Systematic Background Selection for Enhanced Contrastive Dimension Reduction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Park, K., Sun, Z., Liao, R., Bresnick, E. H., Keles, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce BasCoD, a novel testing framework grounded in the spectral theory of subspace inclusion, to enable rigorous evaluation and optimal selection of backgrounds for contrastive dimension reduction.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Contrastive dimension reduction enhances the analysis of high-dimensional data by generating target-specific low-dimensional representations relative to a background. Emerging methods for contrastive dimension reduction have demonstrated their utility in extracting target-specific signals across domains involving high-dimensional observations, including genomics, transcriptomics, and pattern recognition. However, even though choosing an appropriate background is critical to the success of contrastive dimension reduction, no established criterion currently exists for selecting such backgrounds. To address this gap, we introduce BasCoD, a novel testing framework grounded in the spectral theory of subspace inclusion, to enable rigorous evaluation and optimal selection of backgrounds. Extensive application of BasCoD across diverse single-cell datasets demonstrates its effectiveness in systematically identifying suitable backgrounds, thereby significantly improving the contrast and interpretability of the derived target representations. We further illustrate how BasCoD can facilitate design of appropriate backgrounds in large-scale single cell experiments under heterogeneous conditions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.12.653541v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Defining conservation units in a highly diverse species: A case on Arctic charr</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fenton, S., Bean, C. W., Martin, S. A. M., Poultney, S. J., Smith, A., de Eyto, E., Elmer, K. R., Adams, C. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A genetic study of Arctic charr populations across Britain and Ireland with a focus on Scottish populations (N=64 populations; 24,878 SNPs; 410 individuals) and a genetic offset study of genetic vulnerability to climate change.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Defining appropriate conservation units is crucial to the protection and management of biodiversity. These delineations deliver further benefit when they encompass assessments of population vulnerability to extinction from pressures such as climate change. However, delineations and vulnerability assessments are particularly difficult within highly diverse species, such as the salmonid fish Arctic charr (Salvelinus alpinus), that show extensive phenotypic and genetic variation within and across locations, variable and complex life histories, and broad geographic distributions. As yet, the nature and scope of Arctic charr diversity has not been characterised at the scale needed to delineate key conservation units in Scotland. To identify evolutionarily significant and vulnerable populations to prioritise conservation needs, we conducted a genomic study of Arctic charr populations across Britain and Ireland with a focus on Scottish populations (N=64 populations; 24,878 SNPs; 410 individuals). We found that most lake populations represented distinct genetic clusters, with limited gene flow between them resulting in high levels of genetic differentiation. Higher level groupings of genetic similarity across catchments likely reflect historic anadromy and migration, with populations primarily grouping east or west of the central watershed divide in Scotland. Analysing genetic offset, also known as genomic vulnerability, we identified strong inverse correlations between genetic vulnerability with latitude and distance to the sea suggesting that more southern and more inland populations are more vulnerable to loss due to climate change. Additionally, patterns of vulnerability across several additional metrics identified further populations that may be at high vulnerability to loss. We further used our genetic data, along with phenotypic and geographic information, to identify populations of greatest evolutionary significance. This highlighted the most important ones to protect are those in locations with multiple ecotypes, a key facet of functional Arctic charr biodiversity, and populations that are the only ones in their Hydrometric Area.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.12.653098v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The plant-time-bender model: predicting yield through wheat's perception of time</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Roth, L., Herrera, J. M., Levy HaÃàner, L., Pellet, D., Fossati, D., Boss, M., Chen, X., Nousi, P., Volpi, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A deep learning model that learns genotype-specific responses to temperature and vapor pressure deficit to predict wheat yield.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> To address challenges in food security, a better understanding of crop performance under varying environmental conditions is required. Plant Time Bender (PTB) is a deep learning model that integrates high-throughput field phenotyping data with genomic and environmental information to predict wheat yield. PTB leverages image time series, genetic markers, and environmental covariates to learn genotype-specific responses to temperature and vapor pressure deficit. Compared to mere genomic prediction models, PTB demonstrates superior performance when predicting yield in unseen environments across 48 year-locations in Europe. The model captures non-linear growth responses varying with phenological stages and identifies distinct patterns associated with yield performance and stability. Specifically, varieties with higher yield stability exhibit reduced sensitivity to vapor pressure deficit around 1.5 kPa and distinctive temperature responses during emergence and senescence. PTB enables retrospective yield predictions across 20 years, providing a foundation for location-specific variety recommendations and targeted breeding strategies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.12.653430v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ngsAMOVA: A Probabilistic Framework for Analysis of Molecular Variance, dXY and Neighbor-Joining Trees with Low Depth Sequencing Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Altinkaya, I., Zhao, L., Nielsen, R., Korneliussen, T. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce ngsAMOVA, a novel probabilistic framework for analyzing molecular variation in population hierarchies with low- and medium-depth sequencing data, and demonstrate that it is more accurate than genotype calling at low and medium read depths.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: Next-generation sequencing (NGS) has transformed population genetics and evolutionary biology, but the data produced in studies of non-model organisms, ancient DNA, and environmental DNA often consist of low- or medium-depth sequencing. Analyses of these data rely on computational methods that utilize genotype likelihoods (GLs) to account for genotype uncertainty. Nevertheless, many widely-used analysis methods, such as analysis of molecular variance (AMOVA) and methods for estimating phylogenetic trees using nucleotide divergence (dXY) still lack the probabilistic frameworks necessary to accommodate GLs. Results: We introduce ngsAMOVA, a novel probabilistic framework for analyzing molecular variation in population hierarchies with low- and medium-depth sequencing data. It employs an Expectation Maximization algorithm to first estimate the joint genotype probabilities for pairs of individuals, accounting for genotype uncertainty using GLs. It then uses these estimates to generate a pairwise distance matrix, which can be used for AMOVA, estimation of dXY, and for estimating phylogenetic trees using Neighbor-Joining. Hypothesis testing is facilitated using genomic block-bootstrapping. Through extensive simulations, we demonstrate that ngsAMOVA provides more accurate results compared to genotype calling at low and medium read depths. Overall, ngsAMOVA represents a methodological advance in the analysis of molecular variance and divergence under sequencing uncertainty. It provides a robust framework, opening up numerous possibilities for gaining insights into the evolutionary histories through its applications. ngsAMOVA is available as a fast, efficient, and user-friendly program written in C/C++.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.12.653431v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Community-promoted antibiotic resistance genes show increased dissemination among pathogens</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lund, D., Johnning, A., HolmstroÃàm, M., Varghaei, L., Inda-Diaz, J. S., Bengtsson-Palme, J., Kristiansson, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of mobile antibiotic resistance genes (ARGs) in human gut and wastewater microbiomes and their presence in human pathogens .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Antibiotic resistance is increasing among bacterial pathogens, posing one of the most severe threats to future public health. A major contributor to the increasing resistance is the dissemination of mobile antibiotic resistance genes (ARGs) among bacterial communities. These genes are ubiquitously present in various environments and are especially diverse in the human gut and in wastewater. Despite this, the clinical implications of the prevalence of ARGs in these bacterial communities remain unclear. In this study, we aimed to investigate how the abundance of ARGs in human gut and wastewater microbiomes reflects their dissemination among important bacterial pathogens. To do this, we estimated the abundance of >30,000 ARGs, including both well-known (established) and computationally predicted (latent) genes, in >6,000 metagenomic samples. From their prevalence in human gut and wastewater, we identified four major categories of ARGs: co-promoted, human gut (HG)-promoted, wastewater (WW)-promoted, and non-promoted. Our results showed that co-promoted ARGs were, by far, most promiscuous, with an increased frequency of being carried by multiple bacterial phyla, and higher co-localization with broad host range conjugative elements. Co-promoted ARGs were also found to be overrepresented among genes identified in multiple pathogenic species and exhibited an overall higher genetic compatibility, both with pathogens and other typical residents of the human gut and wastewater microbiomes. Taken together, our results highlight the link between the promotion of ARGs in the human gut and wastewater microbiomes and their presence in human pathogens, and, thereby, their potential risk to human health.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.12.653433v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Systematic Mapping of Bacterial CRISPRa Design Rules and Implications for Synergistic Gene Activation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kiattisewee, C., Karanjia, A. V., Cardiff, R. A. L., Olander, K. E., Leejareon, P., Alvi, S. S., Carothers, J. M., Zalatan, J. G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We found that optimal target sites for different activators could vary by up to 200 bases in the region upstream of the transcription start site (TSS), and we found that many combinations were antagonistic.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> CRISPR gene activation (CRISPRa) tools have shown great promise for bacterial strain engineering but often require customization for each intended application. Our goal is to create generalizable CRISPRa tools that can overcome previous limitations of gene activation in bacteria. In eukaryotic cells, multiple activators can be combined for synergistic gene activation. To identify potential effectors for synergistic activation in bacteria, we systematically characterized bacterial activator proteins with a set of engineered synthetic promoters. We found that optimal target sites for different activators could vary by up to 200 bases in the region upstream of the transcription start site (TSS). These optimal target sites qualitatively matched previous reports for each activator, but the precise targeting rules varied between different promoters. By characterizing targeting rules in the same promoter context, we were able to test activator combinations with each effector positioned at its optimal target site. We did not find any activator combinations that produced synergistic activation, and we found that many combinations were antagonistic. This systematic investigation highlights fundamental mechanistic differences between bacterial and eukaryotic transcriptional activation systems, and suggests that alternative strategies will be necessary for strong bacterial gene activation at arbitrary endogenous targets.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.14.654134v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Peptide nucleic acid (PNA) and DNA hybrid three-way junctions and mesojunctions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Madhanagopal, B. R., Patel, A., Talbot, H., Geary, J. A., Ganesh, K. N., Chandrasekaran, A. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the PNA/DNA hybrid three-way junctions and mesojunctions can be assembled in magnesium-free buffers containing low concentrations of calcium or sodium salts.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Three-way junctions are simple and fundamental structural motifs that impart the typical branching property in most DNA nanostructures. While conventional three-way DNA junctions are well-known, mesojunctions are relatively unexplored. Here, we report the synthesis of peptide nucleic acid (PNA)/DNA hybrid three-way conventional and mesojunctions, containing a 14 bp duplex DNA arm and two PNA/DNA hybrid arms. We show that the PNA/DNA mesojunction can be assembled in magnesium-free buffers containing low concentrations of calcium or sodium salts. PNA/DNA hybrid junctions and mesojunctions showed higher thermal stability compared to the DNA versions. Further, PNA/DNA hybrid junctions assembled in sodium exhibited higher nuclease resistance against DNase I. Our results pave the way for PNA/DNA hybrid three-way junctions to be integrated into complex DNA nanostructure designs to improve their structural and enzymatic stability.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.15.653195v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Unusually Broad-spectrum small-molecule sensing using a single protein scaffold</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tian, H., Beltran, J., George, W., Lenert-Mondou, C., Seder, N., Davis, Z. I., Swift, S. D., Whitehead, T. A., Girke, T., Wheeldon, I., Cutler, S. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A small-molecule sensor system based on mutant abscisic acid receptors with nanomolar in vivo responses from ligand-ligand interactions .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Small-molecule sensing in plants is dominated by chemical-induced dimerization modules. In the abscisic acid (ABA) system, allosteric receptors recruit phosphatase effectors and achieve nanomolar in vivo responses from {micro}M receptor-ligand interactions. This sensitivity amplification could enable ABA receptors to serve as generic scaffolds for designing small-molecule sensors. To test this, we screened collections of mutant ABA receptors against 2,726 drugs and other ligands and identified 569 sensors for 6.7 % of these ligands. The mutational patterns indicate strong selection for ligand-specific binding pockets. We used these data to develop a sensor-design pipeline and isolated sensors for multiple plant natural products, 2,4,6-trinitrotoluene (TNT), and forever per- and polyfluoroalkyl substances (PFAS). Thus, the ABA sensor system enables design and isolation of small-molecule sensors with broad chemical scope and antibody-like simplicity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.15.654352v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Sustainable production of plastic-degrading enzymes in Chlamydomonas pacifica</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Diaz, C. J., Dutra Molino, J. V., Saucedo, B., Kang, K., do Espirito Santo, E., Tessman, M., Gupta, A., Burkart, M. D., Simkovsky, R., Mayfield, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new extremophile alga that can produce functional PHL7 enzymes in addition to producing the valuable components that inherently exist in the C. pacifica algae biomass.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The discovery of a new extremophile alga, Chlamydomonas pacifica, provides an opportunity to expand on heterologous protein expression beyond the traditional Chlamydomonas reinhardtii. C. pacifica is a unicellular extremophile capable of surviving at high pH, high temperatures, and high salinity. These various growth conditions allow C. pacifica to outcompete any invading contaminants in open-air environments. Developing this novel species as a platform for recombinant protein production could significantly advance commercial microalgal recombinant protein production. We have previously shown that C. reinhardtii can secrete a plastic-degrading enzyme: a PETase known as PHL7. This PETase is capable of cleaving ester bonds and has been used commercially for the degradation of PET plastics. However, the expression of such an enzyme has yet to be done in open raceway ponds and on a large scale. Here, we describe the culturing of PHL7 transgenic C. pacifica strain in three 80L raceway ponds and the measurements of recombinant enzymatic expression and activity found in the culture media. Our work provides proof of concept that this new organism can produce functional PHL7 enzymes in addition to producing the valuable components that inherently exist in the C. pacifica algae biomass.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.14.654053v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Probing as a new technique to assess single-cell RNA-seq batch correction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Codice', F., Fariselli, P., Raimondi, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce the Batch Probing Score (BPS), a supervised metric quantifying residual batch signal after correction.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell RNA sequencing (scRNA-seq) is vulnerable to batch effects that could confound biological interpretation. We introduce the Batch Probing Score (BPS), a supervised metric quantifying residual batch signal after correction. Unlike existing unsupervised metrics, BPS directly measures the remaining batch signal that could affect downstream analysis. Evaluated on six datasets and six correction methods, BPS has the highest sensitivity among existing metrics used to evaluate batch correctors.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.12.653389v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

