<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Saturday, 10 May 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ChromActivity: integrative epigenomic and functional characterization assay based annotation of regulatory activity across diverse human cell types.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tevfik Umut Dincer, Jason Ernst</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce ChromActivity, a computational framework for predicting and annotating regulatory activity across the genome through integration of multiple epigenomic maps and various functional characterization datasets.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We introduce ChromActivity, a computational framework for predicting and annotating regulatory activity across the genome through integration of multiple epigenomic maps and various functional characterization datasets. ChromActivity generates genomewide predictions of regulatory activity associated with each functional characterization dataset across many cell types based on available epigenomic data. It then for each cell type produces ChromScoreHMM genome annotations based on the combinatorial and spatial patterns within these predictions and ChromScore tracks of overall predicted regulatory activity. ChromActivity provides a resource for analyzing and interpreting the human regulatory genome across diverse cell types.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40346707/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">High-content screening (HCS) workflows for FAIR image data management with OMERO.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Riccardo Massei, Wibke Busch, Beatriz Serrano-Solano, Matthias Bernt, Stefan Scholz, Elena K Nicolay, Hannes Bohring, Jan Bumberger</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show how using Workflow Management Systems (WMS) to HCS data management enables us to consistently transfer images across different locations in a structured and reproducible manner, reducing the risk of errors and increasing data consistency and reproducibility.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> High-content screening (HCS) for bioimaging is a powerful approach to studying biological processes, enabling the acquisition of large amounts of images from biological samples. However, it generates massive amounts of metadata, making HCS experiments a unique data management challenge. This data includes images, reagents, protocols, analytic outputs, and phenotypes, all of which must be stored, linked, and made accessible to users, scientists, collaborators, and the broader community to ensure sharable results. This study showcases different approaches using Workflow Management Systems (WMS) to create reusable semi-automatic workflows for HCS bioimaging data management, leveraging the image data management platform OMERO. The three developed workflows demonstrate the transition from a local file-based storage system to an automated and agile image data management framework. These workflows facilitate the management of large amounts of data, reduce the risk of human error, and improve the efficiency and effectiveness of image data management. We illustrate how applying WMS to HCS data management enables us to consistently transfer images across different locations in a structured and reproducible manner, reducing the risk of errors and increasing data consistency and reproducibility. Furthermore, we suggest future research direction, including developing new workflows and integrating machine learning algorithms for automated image analysis. This study provides a blueprint for developing efficient and effective image data management systems for HCS experiments and demonstrates how different WMS approaches can be applied to create reusable, semi-automated workflows for HCS bioimaging data management using OMERO.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40346117/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Mini-review on Immunoblotting Technique in Vaccine Development: Current Innovations.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Roshan Kumar Dubey</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Immunoblotting, commonly known as Western blotting, remains a pivotal technique in the identification and quantification of specific proteins, offering critical insights into antigen-antibody interactions essential for vaccine development.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Immunoblotting, commonly known as Western blotting, remains a pivotal technique in the identification and quantification of specific proteins, offering critical insights into antigen-antibody interactions essential for vaccine development. This mini-review highlights the evolving role of immunoblotting in modern vaccinology, focusing on its application in evaluating immune responses, verifying antigen expression, and screening candidate vaccine components. Current innovations, including enhanced detection systems, high-throughput formats, and integration with proteomics, have significantly improved the sensitivity and specificity of this method. Moreover, advancements in automation and data analysis are streamlining workflows, enabling faster and more reliable vaccine research. This review underscores the continued relevance of immunoblotting in the post-genomic era, particularly as new vaccine platforms, such as mRNA and vector-based vaccines, demand precise immunological validation.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40340799/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Isoswitching drives the aging process in human brains</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Erdogdu, B., Ji, H. J., Rudnick, Z. C., Pertea, M., Salzberg, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present isoswitching as a major force in brain development, capable of accurately predicting human brain age from prenatal stages through late adulthood and beyond eighty years of age.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Learning, reasoning, and working memory functions are attributed to the dorsolateral prefrontal cortex (DLPFC), a distinctive region of the human brain that is highly evolved in primates and exhibits notable variability among individuals. Environmental and genetic factors likely contribute to this variability, but little is known about how they influence changes within an individual brain across the lifespan as different cognitive tasks and challenges arise. Most genetic studies focus on DNA mutations or changes in overall gene expression levels. However, genes can also alter the form in which they are expressed through alternative splicing. Using RNA sequencing data from prenatal and postnatal human DLPFCs, we observed that many genes undergo dramatic shifts in their isoform preferences around the time of birth. We further found that thousands of genes continue to undergo gradual, temporally regulated changes in their preferred isoforms, a phenomenon we term 'isoswitching'. In this study, we present isoswitching as a major force in brain development, capable of accurately predicting human brain age from prenatal stages through late adulthood and beyond eighty years of age. This represents the first demonstration of brain age prediction based solely on RNA sequencing data. We also report isoswitching in the brain of a closely related primate, the rhesus macaque.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.05.652255v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CRIS: A Centralized Resource for High-Quality RNA Structure and Interaction Data in the AI Era</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lee, W. H., Li, K., Bai, J., Zhang, M., Sharma, A., Lu, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> The Crosslinking-based RNA Interactomes and Structurome (CRIS) database is a unified platform for RNA structure and interaction data, combining rigorously curated datasets, standardized workflows, and user-friendly tools.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The Crosslinking-based RNA Interactomes and Structurome (CRIS) database is a unified platform for RNA structure and interaction data, combining rigorously curated datasets, standardized workflows, and user-friendly tools. While advancements in experimental and computational methods expand our understanding of RNA structure and function, challenges in reproducibility, quality control, and data accessibility remain prevalent due to inconsistencies in data processing. As a result, many existing databases lack breadth in their data sources, are inconsistent in their processing pipelines, and limit accessibilities for end users. Designed for the AI-driven era, CRIS addresses critical gaps in reproducibility and accessibility by harmonizing crosslinking-based sequencing data (e.g., PARIS, SHARC) with built-in quality metrics, enabling robust comparative analyses and machine learning applications. Its structured workflow, built-in quality metrics, and detailed visualization guides ensure reproducibility and transparency while integrating seamlessly with existing experimental pipelines. By bridging complex RNA datasets with accessible computational tools, CRIS accelerates RNA biology and drug development, establishing a new standard for the field.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.05.652292v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Super-Loop Extrusion Mechanism Shapes the 3D Mitotic Chromosome Folding</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Norouzi, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a hierarchical three-layer model in which a third set of megabase-scale "super-loops," extruded by a small number of high-processivity motors, compacts chromatin to its native dimensions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Despite significant advances in understanding mitotic chromosome folding, existing loop-extrusion models - based on two nested tiers of Condensin I- and II-mediated loops - fail to fully reproduce the structural features of chromosomes observed in vivo. We propose a hierarchical three-layer model in which a third set of megabase-scale "super-loops," extruded by a small number of high-processivity motors, compacts chromatin to its native dimensions. Polymer simulations of human chromosome 1 (250 Mb) show that 25-200 consecutive or interlaced super-extruders forming ~10 Mb loops are sufficient to collapse a 25m bottle-brush structure into the empirically observed 5-8*1.5m rod. This model simultaneously reproduces the second diagonal in mitotic Hi-C maps, the double-helical scaffold seen in 3D-SIM microscopy, and the alternating-handedness, zero-mean helicity reported by high-resolution imaging. Interlaced super-loops enhance structural robustness against extruder loss and naturally account for the centromere as a region devoid of super-extruders. Although our simulations remain agnostic to molecular identity, we hypothesize that specialized forms of Condensin II, aided by auxiliary chromosome-associated proteins, act as super-extruders, while Topoisomerase II and other cross-linkers stabilize accumulated stress by converting transient DNA crossings into durable catenations. This three-layered mechanism resolves longstanding discrepancies in chromosome size, shape, contact-map profile, and scaffold ultrastructure within a unified physics-based framework. The next step is to identify or distinguish super-extruders experimentally and to design Hi-C, imaging, or mechanical perturbation assays to validate their existence and dynamics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.05.652288v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CellTune: An integrative software for accurate cell classification in spatial proteomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bussi, Y., Shainshein, D., Ovits, E., Posner, S., Azulay, N., Maimon, N., Keidar Haran, T., Ben-uri, R., Brown, C., Schuldiner, N., Yaniv, E., Van Valen, D. A., Milo, I., Elhanani, O., Schiemann, R., Keren, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce CellTune, an integrated software for analysis of large spatial proteomics datasets, which streamlines precise cell classification through an optimized human-in-the-loop active learning workflow.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Spatial proteomics measures multiple proteins in situ, capturing tissue complexity. However, cell classification in densely packed tissues remains challenging due to the lack of efficient classification algorithms, annotation tools, and high-quality labeled datasets to benchmark computational methods. We introduce CellTune, an integrated software for analysis of large spatial proteomics datasets, which streamlines precise cell classification through an optimized human-in-the-loop active learning workflow. It advances core capabilities across within a unified, intuitive, and code-free interface. To evaluate CellTune, we created CellTuneDepot, a resource of 40k manually-annotated cells and 3.5 million high- quality labeled cells across 60 cell types. CellTune outperforms alternative methods, achieving accuracy comparable to human performance while enabling increased classification resolution and discovery of novel cell types. Together, CellTune and CellTuneDepot provide researchers with a tool for state-of-the- art classification accuracy and resolution at scale to drive biological insights.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.05.652215v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genome Dynamics and Chromosome Structural Variations in Histoplasma ohiense, a fungal pathogen of humans</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Heater, S., Voorhies, M., Sil, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the first telomere-to-telomere genome assemblies for Histoplasma ohiense and demonstrate genome plasticity in the genus Histoplasmas.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Histoplasma is a clinically important but understudied genus of thermally dimorphic human fungal pathogens. Histoplasma species normally transition between a multicellular sporulating hyphal form in the soil and a unicellular pathogenic yeast form in a mammalian host. Little is known about genome plasticity of Histoplasma, which we address in this study with the ultimate goal of increasing our understanding of its pathogenicity. This study addresses the Histoplasma genome and its plasticity to further our understanding Histoplasmas ability to cause severe disease. Here we present the first telomere-to-telomere genome assemblies for Histoplasma ohiense. To analyze genome alterations between isolates, we develop an analysis tool to identify genome discontinuities relative to a reference genome. We utilize this analysis to interrogate the genome of laboratory strains and natural Histoplasma isolates and discover that the previously published reference genome does not completely match the chromosome structure of the majority of isolates, instead harboring reciprocal chromosome translocations. We identify the telomere-to-telomere Histoplasma ohiense reference genome that is most representative of clinical isolates. Additionally, to determine the rate of Histoplasma genomic changes, we sequence 46 passaged isolates and calculate the mutation rate to be 2.6 x10-10 SNP/base/doubling--the first such measurement to our knowledge within the order Onygenales, which encompasses several critical fungal pathogens. Finally, we sequence populations of cells to assess genome stability over the course of a month under both yeast and hyphal growth conditions. Interestingly, we observe that transposon signal is not static over time and instead increases during growth in both the yeast and hyphal forms as well as through morphologic transitions, suggesting an increase in transposon copy number. Taken together, this work highlights the plasticity of the Histoplasma genome and presents a comprehensive genome assembly that is representative of Histoplasma ohiense natural isolates.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.05.652209v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Streamlining Multiplexed Tissue Image Analysis with PIP{Sigma}X: An Integrated Automated Pipeline for Image Processing and EXploration for Diverse Tissue Types</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mardamshina, M., Ballllosera Navarro, F., Martinez Casals, A., Avenel, C., Wahlby, C., Lundberg, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed PIP{Sigma}X (Pipeline for Image Processing and EXploration), a user-friendly, end-to-end open-source software designed to make complex image analysis approachable, even for those with little or no programming skills.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Spatial proteomics via multiplexed tissue imaging is transforming how we study biology, enabling researchers to investigate dozens of markers in a single tissue section and explore how cells behave in their native habitat. While imaging technologies have advanced rapidly, data analyses remain a bottleneck. To address this, we developed PIP{Sigma}X (Pipeline for Image Processing and EXploration), a user-friendly, end-to-end open-source software designed to make complex image analysis approachable, even for those with little or no programming skills. PIP{Sigma}X combines robust automation with an intuitive graphical user interface, guiding users through each step of the analysis, from image preprocessing and membrane-aware cell segmentation to signal quantification and spatial data exploration. Each feature includes built-in explanations, recommendations, and quality controls to help users make confident choices throughout the process. PIP{Sigma}X is compatible with a wide range of multiplexed imaging platforms, and its outputs integrate seamlessly with visualization tools like TissUUmaps and QuPath. Also, it supports downstream applications by enabling direct export of selected cell coordinates for laser microdissection. This functionality facilitates precise isolation of target cell populations for deep proteomic or transcriptomic profiling. With PIP{Sigma}X, researchers can extract meaningful biological insights from multiplexed images more easily and robustly, helping to bridge the gap between powerful imaging technologies and real-world scientific discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.04.652145v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Application of spatial transcriptomics across organoids: a high-resolution spatial whole-transcriptome benchmarking dataset</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nucera, M. R. R., Charitakis, N., Leung, R., Leichter, A., Tuano, N., Walkiewicz, M., Sawant, V., Rowley, L., Scurr, M., Er, P., Tan, K., Sutton, R., Ahmad, F., Saxena, R., Maytum, A., Turner, D., Voges, H., Nim, H., Sun, X., Yang, B., Li, K., Ball, G., Elefanty, A., Lamande', S., Lawlor, K., Vanslambrouck, J., Mills, R., Ng, E., Stanley, E., Werder, R., Little, M., Elliott, D., Porrello, E., Faux, M., Eisenstat, D., Velasco, S., Rossello, F., Ramialison, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the first systematic profiling of multiple organoids (brain, heart muscle,heart valve, kidney, lung, cartilage, and blood) using Stereo-seq.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Stem cell-derived organoid models hold great promise to model tissue-specific disease. To enable this, it is crucial to determine how their composition compares to endogenous organs. However, technologies such as spatial transcriptomics (STs) that can inform on regional molecular identity have been challenging to apply to organoids. Here we present the first systematic profiling of multiple organoids (brain, heart muscle,heart valve, kidney, lung, cartilage, and blood) using Stereo-seq. We describe the optimisation of ST with multiple organoids on a single chip. We reveal differences in data capture efficiency compared to reference tissues hindering traditional downstream analyses. To overcome such challenges, we developed a bespoke regional analysis method to enable detection of transcriptional changes. Together, these developments form a platform to inform future work to investigate organoids using ST, both in terms of optimising data capture of multiple organoids across a chip and novel methods for regional analysis of organoids.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.04.651803v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A comprehensive water buffalo pangenome reveals extensive structural variation linked to population specific signatures of selection</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Arshad, F., Jayaraman, S., Talenti, A., Owen, R., Mohsin, M., Mansoor, S., Asif, M., Prendergast, J. G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a comprehensive pangenome that integrates four newly generated, highly contiguous assemblies of Pakistani river buffalo with available assemblies from both sub-species, which doubles the number of accessible high-quality river buffalo genomes and offers valuable insights into how integrating assemblies can transform analyses of water buffalo.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Water buffalo is a cornerstone livestock species in many low- and middle-income countries, yet major gaps persist in its genomic characterization, complicated by the divergent karyotypes of its two sub-species (swamp and river). Such genomic complexity makes water buffalo a particularly good candidate for the use of graph genomics, which can capture variation missed by linear reference approaches. However, the utility of this approach to improve water buffalo has been largely unexplored. We present a comprehensive pangenome that integrates four newly generated, highly contiguous assemblies of Pakistani river buffalo with available assemblies from both sub-species. This doubles the number of accessible high-quality river buffalo genomes and provides the most contiguous assemblies for the sub-species to date. Using the pangenome to assay variation across 711 global samples, we uncovered extensive genomic diversity, including thousands of large structural variants absent from the reference genome, spanning over 140 Mb of additional sequence. We demonstrate the utility of these data by identifying putative functional indels and structural variants linked to selective sweeps in key genes involved in productivity and immune response across 26 populations. This study represents one of the first successful applications of graph genomics in water buffalo and offers valuable insights into how integrating assemblies can transform analyses of water buffalo and other species with complex evolutionary histories. We anticipate that these assemblies, and the pangenome and putative functional structural variants we have released, will accelerate efforts to unlock water buffalo's genetic potential, improving productivity and resilience in this economically important species.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.04.652079v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Spatially varying graph estimation for spatial transcriptomics cancer data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Acharyya, S., Kang, J., Baladandayuthapani, V.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel spatial graph model to infer spatially varying graphs for high-resolution multivariate spatial data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Modern spatial transcriptomic profiling techniques facilitate spatially resolved, high-dimensional assessment of cellular gene transcription across the tumor domain. The characterization of spatially varying gene networks enables the discovery of heterogeneous regulatory patterns and biological mechanisms underlying cancer etiology. We propose a textit{spatial Graphical Regression} (texttt{sGR}) model to infer spatially varying graphs for high-resolution multivariate spatial data. Unlike existing graphical models, texttt{sGR} explicitly incorporates spatial information to infer non-linear conditional dependencies through Gaussian processes. It conducts sparse estimation and selection of spatially varying edges, at both spatial and sub-spatial levels. Extensive simulation studies illustrate the profitability of texttt{sGR} for spatial graph structural recovery and estimation accuracy. Our methods are motivated by and applied to two spatial transcriptomics data sets in breast and prostate cancer, to investigate spatially varying gene connectivity patterns across the tumor micro-environment. Our findings reveal several novel spatial interactions between genes related to immune activation and carcinogenesis regulation such as CD19 in breast cancer and ARHGAP family in prostate cancer. We also provide a modular software package for fitting and visualization of spatially varying graphs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.04.652097v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

