<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Wednesday, 11 June 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">spaMGCN: a graph convolutional network with autoencoder for spatial domain identification using multi-scale adaptation.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tianjiao Zhang, Hongfei Zhang, Zhongqian Zhao, Saihong Shao, Yucai Jiang, Xiang Zhang, Guohua Wang</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present spaMGCN, an innovative approach specifically designed for identifying spatial domains, especially in discrete tissue distributions, and demonstrate its effectiveness in spatial transcriptomics analysis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Spatial domain identification is crucial in spatial transcriptomics analysis. Existing methods excel with continuous and clustered distributions but struggle with discrete ones. We present spaMGCN, an innovative approach specifically designed for identifying spatial domains, especially in discrete tissue distributions. By integrating spatial transcriptomics and spatial epigenomic data through an autoencoder and a multi-scale adaptive graph convolutional network, spaMGCN outperforms baseline methods. Our evaluations demonstrate its effectiveness in recognizing discrete T cell zones in mouse spleens and follicular cells in human lymph nodes, as well as effectively distinguishing capsule structures from surrounding tissues.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40495225/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SRPS: Survival Reinforced Transfer Learning for Multicentric Proteomic Subtyping and Biomarker Discovery.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Linhai Xie, Pei Jiang, Cheng Chang</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a molecular subtyping algorithm, named Survival Reinforced Patient Stratification (SRPS), to adapt the known subtypes from a discovery cohort to another by simultaneously preserving the distinct prognosis and molecular characteristics of each subtype.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Omics-based molecular subtyping in large-scale and multicentric cohort studies is a prerequisite for proteomics-driven precision medicine (PDPM). However, keeping the subtypes with robust molecular features and significant associations with prognosis across different cohorts is challenging due to the biological heterogeneity and technical inconsistency. Herein, we propose a subtyping algorithm, named Survival Reinforced Patient Stratification (SRPS), to adapt the known subtypes from a discovery cohort to another by simultaneously preserving the distinct prognosis and molecular characteristics of each subtype. SRPS has been benchmarked on simulated and real-world datasets, where it shows a 12% increase in classification accuracy and possesses the best prognostic discrimination. Moreover, based on the calculated subtype significance score, an "unpopular" protein, Peptidylprolyl Isomerase C (PPIC), was identified as the top-1 remarkable protein for subtyping the hepatocellular carcinoma (HCC) patients with the worst prognosis. Eventually, PPIC was experimentally proved to be a pro-cancer protein in HCC, confirming our work as a demonstration of interpretable machine learning guided biological discovery in PDPM research. SRPS is publicly available at https://github.com/PHOENIXcenter/SRPS and https://ngdc.cncb.ac.cn/biocode/tool/BT007770.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40493767/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genome-resolved metagenomics from short-read sequencing data in the era of artificial intelligence.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hajra Qayyum, Zaara Ishaq, Amjad Ali, Masood Ur Rehman Kayani, Lisu Huang</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We reviewed recent AI-based tools specifically developed for short-read metagenomic data, and their underlying models for genome-resolved metagenomics. We believe this study will provide insights into the strengths and limitations of currentAI-based approaches, serving as a valuable resource for selecting appropriate tools</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genome-resolved metagenomics is a computational method that enables researchers to reconstruct microbial genomes from a given sample directly. This process involves three major steps, i.e. (i) preprocessing of the reads (ii) metagenome assembly, and (iii) genome binning, with (iv) taxonomic classification, and (v) functional annotation as additional steps. Despite the availability of multiple bioinformatics approaches, metagenomic data analysis encounters various challenges due to high dimensionality, data sparseness, and complexity. Meanwhile, integrating artificial intelligence (AI) at different stages of data analysis has transformed genome-resolved metagenomics. Though the application of machine learning and deep learning in metagenomic annotation started earlier, the emergence of better sequencing technologies, improved throughput, and reduced processing time have rendered the initial models less efficient. Consequently, the number of AI-based metagenomics tools is continuously increasing. The recent AI-based tools demonstrate superior performance in handling complex and multi-dimensional metagenomics data, offering improved accuracy, scalability, and efficiency compared to traditional models. In this paper, we reviewed recent AI-based tools specifically developed for short-read metagenomic data, and their underlying models for genome-resolved metagenomics. It also discusses the performance of these tools and overviews their usability in metagenomics research. We believe this study will provide researchers with insights into the strengths and limitations of current AI-based approaches, serving as a valuable resource for selecting appropriate tools and guiding future advancements in genome-resolved metagenomics.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40493087/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A spatial long-read approach at near-single-cell resolution reveals developmental regulation of splicing and polyadenylation sites in distinct cortical layers and cell types.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Foord, C., Prjibelski, A. D., Hu, W., Michielsen, L., Vandelli, A., Narykov, O., Evans, B., Hsu, J., Belchikov, N., Jarroux, J., He, Y., Ross, M. E., Hajirasouliha, I., Tartaglia, G. G., Korkin, D., Tomescu, A. I., Tilgner, H. U.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Spatial ISOform sequencing for long reads with near-single-cell resolution and post-puberty splicing changes in cortical layers .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genome-wide single-cell and spatial long-read approaches have gained traction, but mostly lack single-cell resolution - and yield limited read lengths. Here, we introduce spatial ISOform sequencing (Spl-ISO-Seq), which reveals exons and polyadenylation sites from long reads with near-single-cell resolution. Spl-ISO-Seq selects long cDNAs and doubles to triples read lengths compared to standard preparations. Adding a highly specific software tool (Spl-ISOquant) and comparing human post-mortem pre-puberty samples of the visual cortex (8-11 years) to post-puberty samples (16-19 years), we find that cortical layers harbor stronger splicing and poly(A)-site regulation than the adjacent white matter, with enrichment of multiple protein-domain types. For oligodendrocytes however, developmental splicing changes are stronger in white matter. Among cortical layers, layer 4 has the most developmental changes in alternative-exon inclusion in excitatory neurons and in poly(A) sites. We also find many repeat elements, especially ERV1 long terminal repeats downstream of developmentally-regulated layer 4 exons. Overall, alternative splicing changes are linked to synapses -- specifically at the post-synapse. Age-linked splicing changes in layers 1-3 and 4 are associated with autism spectrum disorder but not with schizophrenia, amyotrophic lateral sclerosis and Alzheimer's disease. These results root developmental splicing changes during puberty and the resulting protein changes in specific layers and cell types. More generally, our new technologies enable new observations for any complex tissue.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.10.658877v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Quantitative trait loci mapping of gene expression and chromatin accessibility in primary fibroblast reveals shared allelic effects between Latin American and European ancestries</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Boltz, T., Bot, M., Lapinska, S., Schwarz, T., Hou, K., Garske, K. M., Freund, M. K., Bearden, C. E., Macaya, G., Lopez-Jaramillo, C., Freimer, N. B., Boks, M. P., Kahn, R. S., Pasaniuc, B., Ophoff, R. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We used gene expression and chromatin accessibility data from human primary fibroblasts to investigate quantitative trait loci (QTLs) in cohorts ascertained for bipolar disorder of European and Latin American ancestries.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Quantitative Trait Locus (QTL) analysis of molecular data has identified genetic variants associated with traits such as gene expression, and colocalization of these functional QTL with GWAS risk loci has offered insights into the genetic basis of human disease. We employed gene expression (RNA-seq) and chromatin accessibility (ATAC-seq) obtained from human primary fibroblasts to investigate quantitative trait loci (QTLs) in cohorts ascertained for bipolar disorder of European (n=150) and Latin American (n=96) ancestries. Leveraging data from three countries of origin (The Netherlands, Colombia, Costa Rica) within our cohort, we characterized differences among individuals at the SNP, gene, and accessible-chromatin levels to compute ancestry-specific expression (e)QTLs and chromatin-accessibility (ca)QTLs. Across ancestries, we observed R2 >= 0.93 for eQTL effect sizes and R2 >= 0.95 for caQTLs, indicating a high degree of concordance. Integrating chromatin data with expression and genotype information enabled precise fine-mapping of eQTLs, yielding 203 high-confidence (posterior probability > 90 %) regulatory pathways. In downstream analyses, transcriptome-wide (TWAS) and chromatin-wide (CWAS) association studies with brain- and skin-related GWAS identified 36 TWAS-significant genes and 77 CWAS-significant open chromatin regions. These findings underscore the shared genetic regulatory mechanisms across European and Latin American ancestries, while demonstrating that ancestry-specific reference panels enhance the accuracy of TWAS and CWAS in diverse populations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.09.658613v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Generalizable Machine Learning Model for Early Detection of Hepatocellular Carcinoma Using Bisulfite Sequencing Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Subharam, M., Koehler, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a generalizable machine learning framework for HCC detection using cfDNA methylation profiles derived from bisulfite sequencing data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Hepatocellular carcinoma (HCC) is a leading cause of cancer-related mortality worldwide, with early detection significantly improving survival outcomes. However, both current screening methods (e.g., ultrasound and AFP) and emerging molecular diagnostics have struggled to achieve the sensitivity and specificity required for reliable early-stage detection. In this study, we present a generalizable machine learning framework for HCC detection using cfDNA methylation profiles derived from bisulfite sequencing data. We trained separate XGBoost models on three cfDNA datasets from previously published studies, each based on a different bisulfite sequencing methodology: MCTA Seq, targeted bisulfite sequencing (using methylation-correlated blocks), and low-pass whole-genome bisulfite sequencing (WGBS). For each model, we selected features based on biologically relevant methylation markers reported in the original study, ensuring that feature selection was grounded in experimentally validated signals. A meta-classifier was trained on shared CpG site features to route incoming samples to the most appropriate model, eliminating the need for forced feature harmonization. To evaluate generalizability, we used an independent moderate-depth WGBS liver tissue dataset for blind validation. The dataset was filtered separately to match the feature set of each training model. Models trained on MCTA Seq and WGBS generalized well (blind validation accuracy up to 92%), while the padlock probe based MCB model failed to transfer, underscoring the importance of platform aware modeling. Across datasets, methylation patterns separating cancerous from non-cancerous samples remained stable, even when absolute values shifted, as in tissue vs. blood. Threshold adjustment compensated for this shift without compromising classification. Intra dataset k-fold accuracy remained consistently high, confirming the robustness of methylation based classification across patient cohorts and disease stages. Overall, we demonstrate a generalizable machine learning framework capable of predicting HCC with over 90% accuracy using cfDNA methylation data from any of three distinct bisulfite sequencing assays, including off-the-shelf whole genome bisulfite sequencing (WGBS) at moderate depth. By aligning assay specific models with a meta classifier for sample routing, we provide a modular strategy that avoids harmonization pitfalls while preserving accuracy. The next logical step is clinical validation using biobank-derived plasma samples sequenced with WGBS and passed through this framework to confirm real world performance.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.09.658649v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">TaxonReportViewer: Parsing and Visualizing Taxonomic Hierarchies in Metagenomic Datasets</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Razzolini, E., de Souza, C. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed TaxonReportViewer (TRV), an open-source, cross-platform software with a graphical user interface that facilitates the exploration, comparison, and visualization of Kraken2 classification reports.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Shotgun metagenomics enables comprehensive analysis of microbial communities by classifying sequencing reads directly from environmental samples, yet interpreting the resulting taxonomic classification reports, especially those generated by tools like Kraken2, remains a challenge due to their hierarchical, text-based format. Existing visualization tools such as Krona and Pavian either rely on web infrastructure or provide limited interactivity and flexibility for comparative analysis. To address this gap, we developed TaxonReportViewer (TRV), an open-source, cross-platform software with a graphical user interface that facilitates the exploration, comparison, and visualization of Kraken2 classification outputs. Implemented in Python 3, TRV includes a robust parser capable of accurately reconstructing taxonomic hierarchies even from inconsistently formatted reports. The tool offers dynamic search and filtering of taxa, exportable abundance matrices, one-click generation of comparative bar charts and heatmaps, and direct links to the NCBI Taxonomy database. TRV supports analysis of multiple samples in parallel, allowing users to identify taxonomic patterns across conditions without requiring programming skills. Its visualization features enable immediate recognition of taxon-specific abundance shifts and clustering of samples by taxonomic profile. Tested on a range of Kraken2 reports, TRV demonstrated consistent performance, rapid search capabilities, and accurate representation of lineage structures. It operates efficiently on standard consumer hardware and requires no installation beyond Python, ensuring accessibility for researchers in diverse computational environments. TaxonReportViewer fills a critical niche between raw taxonomic classification and biological interpretation, empowering researchers to derive meaningful insights from complex metagenomic datasets with ease.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.07.658440v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Chromosome-level assembly and annotation of the grey reef shark (Carcharhinus amblyrhynchos) genome</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dahms, C., Vigliola, L., Cheung, L. H. T., Hui, J. H. L., Momigliano, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We assembled the first nuclear reference genome of the endangered grey reef shark (Carcharhinus amblyrhynchos) using long-read PacBio HiFi and Omni-C sequencing.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> To date only four of nine shark orders have nuclear reference genomes, despite next-generation sequencing advances. Particularly for threatened shark species, there is a lack of reliable genomes which are crucial in facilitating research and conservation applications. We assembled the first nuclear reference genome of the endangered grey reef shark (Carcharhinus amblyrhynchos) using long-read PacBio HiFi and Omni-C sequencing to reach chromosome-level contiguity (36 pseudo chromosomes; 2.9 Gbp) and high completeness (94% complete BUSCOs). BRAKER3 annotated 16,522 protein-coding genes after masking repetitive elements which accounted for 59% of the genome. We identified potential X and Y sex chromosomes on pseudo chromosomes 36 and 57, respectively. The quality and completeness of the draft genome of C. amblyrhynchos suggest that it has the potential to facilitate comprehensive comparative genomics, enabling researchers to investigate genetic variations and adaptations specific to this population and will help advance conservation genetic applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.07.658398v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Cut-Detector: A Tool for Automated Temporal Analysis of Late Cytokinetic Events</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bonte, T., Dubois, L., Gagna, P., Dibsy, R., Petrovic, A., Advedissian, T., Serres, M., Cuvelier, F., Crouigneau, M., Sassoon, N., Gupta-Rossi, N., Fremont, S., Echard, A., Walter, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Cut-Detector, an open-source tool for the automatic analysis of late cytokinesis timing from time-lapse microscopy movies.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cytokinesis is the final step of cell division, resulting in the physical separation of the two daughter cells. Despite its fundamental importance in cell biology, biologists currently lack automatic tools for large-scale profiling of cytokinesis dynamics. In particular, the timing of the first microtubule cut in the intercellular bridge connecting the daughter cells is crucial, as it marks a critical step preceding abscission. Here, we introduce Cut-Detector, an open-source tool for the automatic analysis of late cytokinesis timing from time-lapse microscopy movies. Cut-Detector employs an AI approach to carry out the multiple tasks required to monitor cytokinesis: cell segmentation and tracking, detection of cell division events, localization of the intercellular bridge, and detection of the microtubule cuts. Cut-Detector will facilitate large-scale analyses to uncover new cytokinetic genes, a task that would be impractical without automation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.06.658046v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Decoding PDI diversity: insights into structure, domains, and functionality in sorghum</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lopez Gomez, C. F., Morris, M. T., Massel, K., Smith, M., Crisp, P., Schenk, G., Godwin, I. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified and characterized Protein Disulfide Isomerases (PDI) in sorghum to explore their domain architecture, three-dimensional structure and functionality.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Proteins play indispensable roles in cellular function, acting as both structural components and catalysts for essential biological processes. Their proper folding into three-dimensional structures is critical for functionality. To ensure correct folding, proteins interact with chaperones and folding catalysts such as Protein Disulfide Isomerases (PDIs), which assist in the formation and rearrangement of disulfide bonds that stabilize proteins by linking cysteine residues. PDIs are part of the thioredoxin (TRX) superfamily and are characterized by a conserved CXXC motif that contributes to their redox potential. They exhibit isomerase and oxidoreductase activities, that enable them to rearrange and form new disulfide bonds. PDI family members in sorghum (SbPDI) present a broad and largely unexplored diversity in domain order, structure, and architecture between or even within species. To shed light on this diversity, we identified and characterized PDI family members in sorghum in silico to explore their domain architecture, three-dimensional structure and functionality.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.06.658324v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PhenoGraph: A Multi-Agent Framework for Phenotype-driven Discovery in Spatial Transcriptomics Data Augmented with Knowledge Graphs</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Niyakan, S., Qian, X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present PhenoGraph, a large language model (LLM) based multi-agent system that automates the full pipeline for phenotype-driven ST data analysis, augmented by biological knowledge graphs for enhanced interpretability.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Spatial transcriptomics (ST) provides powerful insights into gene expression patterns within tissue structures, enabling the discovery of molecular mechanisms in complex tumor microenvironments (TMEs). Phenotype-based discovery in ST data holds transformative potential for linking spatial molecular expression patterns to clinical outcomes; however, appropriate ST data analysis remains fundamentally fragmented and highly labor-intensive. Due to its limited scalability considering the size of typical ST data in large cohorts, researchers must rely on other phenotype-annotated omics data modalities (e.g. bulk RNA-sequencing) and align them with ST data to extract clinically meaningful spatial patterns. Yet, this process requires manually identifying relevant cohorts, aligning multi-modal data, selecting and tuning analysis pipelines, and interpreting results, typically without any built-in support for biologically context-aware reasoning. In this paper, we present PhenoGraph, a large language model (LLM) based multi-agent system, that automates the full pipeline for phenotype-driven ST data analysis, augmented by biological knowledge graphs for enhanced interpretability. Built on a modular agent architecture, PhenoGraph dynamically selects, executes, and corrects phenotype analysis pipelines based on user-defined queries. We showcase the flexibility and effectiveness of PhenoGraph across a variety of TME ST datasets and phenotype classes, highlighting its potential to enhance biological discovery efficacy.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.06.658341v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Monalisa: An Open Source, Documented, User-Friendly MATLAB Toolbox for Magnetic Resonance Imaging Reconstruction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Leidi, M., Jia, Y., Helbing, D., Barranco, J., Acikgoz, B. C., Peper, E., Ledoux, J.-B., Bastiaansen, J. A. M., Franceschiello, B., Milani, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Monalisa, an open-source, user-friendly MATLAB framework designed to simplify the MRI reconstruction process.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Magnetic Resonance Imaging (MRI) is a pivotal tool in modern diagnostics and research, yet image reconstruction is often used by clinicians and researchers as a black box. In this work, we introduce Monalisa, an open-source, user-friendly MATLAB framework designed to simplify the MRI reconstruction process. Monalisa decomposes the reconstruction pipeline into clear modular steps, including raw data reading with flexible file-type abstraction, trajectory computation, density compensation, advanced coil sensitivity mapping, and tailored binning strategies through its mitosius preprocessing stage. The framework supports a suite of reconstruction methods, including iterative-SENSE (also named CG-SENSE), GeneRalized Autocalibrating Partial Parallel Acquisition (GRAPPA) reconstructions, and Compressed Sensing (CS) supporting both spatial and temporal regularization using l1 and l2 techniques, accommodating both Cartesian and non-Cartesian acquisitions. Benchmark experiments comparing Monalisa with established frameworks such as Berkeley Advanced Reconstruction Toolbox (BART) on simulated 2D radial acquisitions demonstrate competitive performance, with trade-offs observed between Structural Similarity Index (SSIM) and l2 error metrics. In some benchmark reconstructions, Monalisa produced images with fewer visible artifacts than BART, although these differences were not always reflected in SSIM or l2 distance metrics. By providing comprehensive documentation and modular design, Monalisa serves not only as a powerful tool for research and clinical imaging but also as an educational platform to facilitate innovation in MRI reconstruction.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.06.658299v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Traditional uses, population, threats and conservation of the bansouman or gingerbread plum Neocarya macrophylla (Chrysobalanaceae) in Republic of Guinea (West Africa)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Molmou, D. N., Cheek, M., Konomou, G., Magassouba, S., Simoes, A. R., Larridon, I., Couch, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> This study documents the local names, harvesting methods, uses and seed sale prices of Neocarya macrophylla (Chrysobalanaceae) in Guinea, and assesses its global conservation status as Near Threatened using the 2012 IUCN Red List categories and criteria.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The flora of the Republic of Guinea (West Africa) includes nearly 4000 vascular plants species, of which c. 10% have been documented to have local socio-economic uses. These socio-economic plants provide among others, foods, medicines, materials and a source of income for the people of Guinea. One example is the bansouman or gingerbread plum, Neocarya macrophylla (Chrysobalanaceae) which is locally valued for its large, delicious seeds ('nuts'), edible fruit flesh, and medicinal bark (among other medicinal parts). Through a field survey of 17 local communities adjacent to Neocarya populations and the Neocarya populations themselves (780 mature individuals measured), in seven of the ten Guinean prefectures from which the species is recorded, this study documents (1) the local names, harvesting methods, uses and seed sale prices at different points in the value chain, partitions the species by habitat, records perceived threats to trees, key elements of the population structure and relative frequency of Neocarya macrophylla in Guinea, (2) map and re assesses its global conservation status as Near Threatened using the 2012 IUCN Red List categories and criteria, and (3) construct a species conservation action plan. Analysis of our survey data suggests that the largest number of mature trees (75% of the total) is in the three northern prefectures (Gaoual, Koundara, Pita) at the western foot of the Fouta massif whence seeds are exported internationally. Trees mainly occurred in shrub savannah (45% of mature individuals), with lower numbers in three other habitats. Human set bush fires appear to be the most ubiquitous threat (55%) in Guinea, but sand quarrying (18%), agricultural conversion and debarking are also key threats. Natural regeneration levels seem low compared with data from Niger with juveniles ( 11 cm circumference) recorded. This is the first study focused on Neocarya macrophylla in the Republic of Guinea. In contrast the species has been the subject of numerous other papers elsewhere in its range, and we document 14 studies from Niger alone in the last 15 years. While the species is well known as an important Indigenous Fruit Tree with multiple uses in some countries, it is currently not among the top 10 such species listed for Guinea. Compared with Nigeria where numerous other uses including industrial are derived from the species, Neocarya macrophylla appears under utilised in Guinea. We suggest that studies of the chemistry and nutritional value of the seeds of Guinean Neocarya are made, since there are indications that these differ between Guinea and e.g. Niger. Follow up studies to map tree distribution in Guinea in more detail, quantify tree density, fruit yield per tree and per Ha, and duration between germination and fruiting are advised. Field trials to select elite individuals are recommended. Development of a machine to extract seeds from the endocarp (decortication) is a priority as this is the major factor limiting wild harvesting and availability of seed for human consumption.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.08.655547v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Analysis of Wheat Spike Morphological Traits Using 2D Imaging</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sun, F., Zheng, S., Li, Z., Gao, Q., Jiang, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a pipeline for precise measurement of wheat spike phenotypes and studied their correlations with thousand grain weight (TGW) and spike yield. Through hierarchical clustering based on spike morphology, we categorized wheat spikes into six classes and identified phenotypic differences between these classes and their impact on TGW</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The morphological structure of wheat spikes plays a central role in wheat yield. Wheat spike morphology, closely associated with crop yield, has attracted considerable attention in the fields of genetics and breeding. However, traditional measurement methods can only measure simple traits, and precise phenotypes remain difficult to obtain, constraining the study and improvement of complex spike-related traits. This study utilized deep learning technologies to develop a pipeline, called SpikePheno, for the acquisition of precise wheat spike phenotypes. Our pipeline demonstrated high accuracy in spike segmentation, achieving a mean Intersection over Union (mIoU) of 0.948. Additionally, our method accurately identified spikelet counts, achieving an R2 of 0.9923. Using experimental data of 221 wheat cultivars from various regions of China grown in Zhao County, Hebei Province, our pipeline extracted 45 different phenotypes and studied their correlations with thousand grain weight (TGW) and spike yield. Our findings indicate that precise measurement of spike area, spikelet area, and other phenotypic traits enables a clearer understanding of the correlation between spike morphology and wheat yield. Through hierarchical clustering based on spike morphology, we categorized wheat spikes into six classes and identified phenotypic differences between these classes and their impact on TGW and yield. Furthermore, this study revealed phenotypic differences between wheat cultivars from different geographical regions and over different decades, with an increase in large-spike cultivars over time, especially in southern China. This research may help breeders understand the relationship between wheat spike morphology and yield, providing an important basis for future wheat breeding efforts.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.06.658159v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Resin canal leakage underlying xylem dysfunction in pine wilt disease</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Umebayashi, T., Fukuda, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We studied the development of xylem dysfunction around the inoculation site of one-year-old stems of PWN-infected seedlings using a combination of compact MRI and cryo-SEM, and found that embolisms and resin leakage occur from damages of resin canals.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> During the course of pine wilt disease (PWD), xylem dysfunction is induced (in the absence of water stress) by invasion of pinewood nematodes (PWN) into resin canals, and the infected pine trees wilt. When PWNs are inoculated to Japanese black pine seedlings, massive embolisms develop at the inoculation site. The developmental process of massive embolization to form an embolized zone is not well understood. We studied the development of xylem dysfunction around the inoculation site of one-year-old stems of PWN-infected seedlings using a combination of compact MRI and cryo-SEM. We visualized pathological changes in tracheids and resin canals in the xylem, and found that embolisms occurred in PWN-inoculated seedlings at a negative hydraulic pressure of -0.5 MPa. Resin canals in healthy trees were filled with resin. In PWN-infected seedlings, abnormal embolisms and resin were observed around resin canal near the inoculation wound. Empty tracheids were observed more extensively than resin-filled tracheids. We suggest that abnormal embolisms and resin leakage occur from damages of resin canals. Resin may seep into the tracheids when cell wall of the axial and radial resin canals is crushed. These embolisms under high water potential are unavoidable under natural conditions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.06.656100v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">OligoSeq: Rapid nanopore-sequencing of single-stranded oligonucleotides</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ramirez Garcia, R., Heinis, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We develop OligoSeq, an innovative approach that integrates two complementary techniques: AmpliSeq (based on PCR and reverse complementation of either sequence-specific or random primers) to facilitate sequencing of single-stranded oligonucleotides.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Nanopore-based DNA sequencing technology has achieved remarkable success in sequencing increasingly long DNA strands for genomics research and biotechnology applications. However, the same level of progress has not been achieved for shorter sequences, such as RNA transcripts and DNA oligonucleotides. Oligonucleotides play a crucial role in automated genome engineering through oligo library generation and in DNA data storage, where they encode binary information in DNA libraries. The accurate sequencing of large and diverse oligonucleotide libraries hence is essential, however, accessible sequencing solutions for oligonucleotides, particularly DNA primers for PCR, oligo DNA libraries for mutagenesis or RNA libraries used in gene expression analysis, remain inadequate. Current biological research platforms still rely on expensive and cumbersome quality control methods such as high-performance liquid chromatography and mass spectrometry. To address this gap, we develop OligoSeq, an innovative approach that integrates two complementary techniques: AmpliSeq (based on PCR) and RevSeq (based on reverse complementation of either sequence-specific or random primers) to facilitate sequencing of single-stranded oligonucleotides. OligoSeq uses nanopore technology and can be used as a reference for other sequencing platforms requiring double-stranded adapters, offering a practical and scalable alternative for standard quality control in single-stranded oligonucleotide synthesis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.10.658886v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multi-criteria decision making and its application to in silico discovery of vaccine candidates for Toxoplasma gondii.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> John T Ellis, Paul J Kennedy</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new in silico vaccine discovery approach based on metrics on the physicochemical properties of proteins.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Vaccine discovery against eukaryotic parasites is not trivial and few exist. Reverse vaccinology is an in silico vaccine discovery approach, designed to identify vaccine candidates from the thousands of protein sequences encoded by a target genome. Previously, we produced the Vacceed bioinformatics pipeline for identification of parasite membrane and excreted/secreted proteins that were likely be exposed to the hosts immune system. More recently, we improved upon machine learning as the final decision-making process to identify parasite proteins that induce a protective response in an animal model. Subsequently, we combined Vacceed with metrics on B and T cell epitope types to produce a new in silico discovery workflow. In this study we extend this in silico workflow to the developability of proteins as vaccines by the incorporation of metrics on the physicochemical properties of proteins. To demonstrate this process, every Toxoplasma gondii protein was ranked in its capacity to provide exposure to the immune system (Vacceed exposure score), presence of epitopes and solubility characteristics by several multicriteria decision making (MCDM) tools (such as TOPSIS, VIKOR and MABAC). A consensus rank was subsequently generated from the results of these tools using a variety of aggregate ranking methods. Levels of uncertainty in the aggregate protein rankings was assessed by conformal interval prediction in association with a machine learning model. Several of the top ranked proteins identified by this approach were novel, uncharacterized membrane transporters or proteins associated with RNA metabolism. In conclusion, MCDM automated the decision making using well known algorithms while conformal prediction intervals varied significantly across the 8000+ proteins of T. gondii. Highly ranked proteins (e.g. the top 100) typically generated low prediction intervals, providing high levels of confidence in their ranks.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40378548/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

