<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Tuesday, 12 August 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Machine Learning-Based Predictive Modeling Maximizes the Efficacy of mTOR/p53 Co-Targeting Therapy Against AML.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jingmei Li, Emi Sugimoto, Keita Yamamoto, Yutong Dai, Wenyu Zhang, Yu-Hsuan Chang, Jakushin Nakahara, Tomohiro Yabushita, Toshio Kitamura, Sung-Joon Park, Kenta Nakai, Susumu Goyama</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that mTOR inhibition by rapamycin and TP53 activation by DS-5272 collaboratively induced the downregulation of MYC and MCL1 partly through miR-34a, thereby inducing cell cycle arrest and apoptosis in AML cells.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Although mTOR signaling plays a key role in acute myeloid leukemia (AML), mTOR inhibitors have shown limited efficacy against AML in clinical trials. In this study, we found that the anti-leukemic effect of mTOR inhibition was mediated in part through the TP53 pathway. mTOR inhibition by rapamycin and TP53 activation by DS-5272 collaboratively induced the downregulation of MYC and MCL1 partly through miR-34a, thereby inducing cell cycle arrest and apoptosis in AML cells. Joint non-negative matrix factorization (JNMF) and statistical regression analysis using public AML databases revealed that monocytic AMLs with distinctive gene expression profiles were highly sensitive to mTOR inhibition, leading to the generation of an 11-gene score (Rapa-11) to predict the rapamycin sensitivity of each monocytic AML. Consistent with our in silico prediction, mouse AML cells expressing MLL-AF9, the monocytic AML with a low Rapa-11 score, were highly sensitive to rapamycin, whereas those expressing RUNX1-ETO or SETBP1/ASXL1 mutations were not. Co-treatment with rapamycin and DS-5272 had a dramatic in vivo effect on MLL-AF9-driven AML, curing 85% of the leukemic mice. Thus, machine learning-based predictive approaches identified monocytic AML with wild-type TP53 and low Rapa-11 score as a rapamycin-sensitive AML subtype and an ideal target for mTOR/p53 co-targeting therapy.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40785506/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A novel deep learning-based framework reveals a continuum of chromatinsensitivities across transcription factors</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Burger, L., Gil, N., Schubeler, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that chromatin sensitivity is a continuum across transcription factors, and that regulatory regions are activated neither by a purely hierarchical nor purely cooperative mechanism, but by a mixture of both mechanisms.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The genome-wide binding of many transcription factors (TFs) depends not only on the presence of their recognition motifs, but also on the surrounding chromatin context. This raises the question of how binding specificity is achieved, which is essential for the proper activation of regulatory regions during developmental and differentiation processes. Two competing hypotheses have been proposed: either opening is a hierarchical process, whereby chromatin-insensitive TFs bind initially and make the DNA accessible for other TFs; or multiple chromatin-sensitive TFs bind cooperatively, enabling them to overcome the chromatin barrier. Here, using complementary genomic- and deep-learning-based techniques, we provide a quantitative framework to assess chromatin sensitivity of individual TFs, and hone in on the definitions and implications of such sensitivity. Our findings demonstrate that the degree of chromatin sensitivity is a continuum across TFs, showing that regulatory regions are activated neither by a purely hierarchical nor purely cooperative mechanism, but by a mixture of both mechanisms. The presented framework allows for a locus-specific as well as global, quantitative assessment of chromatin sensitivity and should help in understanding the binding behaviour of individual TFs as well as their potential contribution to gene regulation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.11.669605v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comparative genomic analysis of DNA motif frequency across the animal kingdom</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> DuFriend, D., Valderrama, E., Annor, E., Rosler, C., Davis, E., Anthem, Y. L., Velasquez, C., Wheat, S., Goelzer, M. H., Goelzer, J. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the number of regulatory motifs in an organism's genome increases with genome size and show that this increases with the size of the genome.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The rapid expansion of genomic sequencing has produced vast datasets, yet the functional significance of non-coding DNA remains largely unexplored. Eukaryotic genomes contain numerous short, recurring DNA sequences known as motifs, which are crucial for gene regulation. In this study, it was hypothesized that the quantity of these regulatory motifs scales with the overall size of an organism's genome. To test this, a high-performance computing approach was employed using a word count frequency algorithm to quantify 32 universal regulatory motifs across 51 diverse animal genomes. The analysis revealed a strong, positive linear correlation between the total motif count and genome size (R^2 = 0.9852). Regression analysis identified several motifs as significant predictors of genome size. Further investigation into key developmental motifs such as SOX2, SRY, POU5F1, and POU1F1 showed distinct distribution patterns across species. Together, these findings support the hypothesis that regulatory complexity, as approximated by motif frequency, increases with genome size and highlights the utility of computational methods in comparative genomics to uncover large-scale organizational properties of genomes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.10.669586v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Condensins-mediated mitotic folding persists as a structural echo in interphase chromatin</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yunusova, A., Tsoi, E., Smirnov, A., Shnaider, T., Pristyazhnuk, I., Gridina, M., Kazakov, E., Kireev, I., Myakinkov, I., Nuriddinov, M., Fishman, V., Battulin, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that condensin II is the major driver of chromatin folding in interphase nuclei following cohesin depletion, and show that the contact patterns established by condensins during mitosis are preserved in the interphase nucleus, suggesting a structural memory of chromosome folding.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The Structural Maintenance of Chromosomes (SMC) protein complexes, cohesin and condensins, orchestrate cell cycle-dependent transitions in chromosome folding: from the relatively decondensed interphase chromatin, organized into dynamic DNA loops by cohesin, to the highly condensed arrays of DNA loops in mitotic chromosomes formed by condensins. Here, using Hi-C data from mouse embryonic stem cell lines with auxin-inducible degron-tagged subunits of SMC complexes, we clarify the role of condensins in shaping chromatin contacts within the interphase nucleus. We found that depletion of condensin I leads to weakened segregation of chromatin into A and B compartments. We also show that the contact patterns established by condensins during mitosis are preserved in interphase, suggesting a structural memory of mitotic chromosome folding. Condensins influence chromocenter organization: in the absence of condensin II during mitosis, centromeres tend to cluster, forming hyperclusters of pericentric heterochromatin. In contrast, depletion of condensin I reduces the number of heterochromatin clusters, consistent with enhanced chromosome territoriality. By combining a cohesin-degron system with Mcph1 knockout, we established an artificial model in which condensin II becomes the major driver of chromatin folding in interphase nuclei following cohesin depletion. Unlike extrusive cohesin, condensin II does not stall at sequence-specific sites and forms large, randomly positioned loops on interphase chromatin. Moreover, condensin II activity during interphase does not interfere with cohesin-dependent structures such as TADs and chromatin loops.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.08.669285v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comparative analysis of the EF-1Œ± Intergenic Region in Babesia divergens isolates: Insights into TA Repeat Variation and Potential Regulatory Implications</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ozubek, S., Sanchez-Flores, A., Montero, E., Alzan, H., Suarez, C., Grande, R., Gil-Iglesias, A., Bastos, R., Aktas, M., Gonzalez, L. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comparative analysis of the EF-1-IG region in Babesia divergens, a zoonotic tick-borne pathogen that causes bovine and human babesiosis in Europe.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Babesia divergens, a zoonotic tick-borne pathogen, causes bovine and human babesiosis in Europe. The Elongation Factor 1 alpha (EF-1) protein is important in many cellular processes and has emerged as a possible target for subunit vaccine development against parasitic infections, and its intergenic region (IG) is an important tool for genetic manipulation of Babesia parasites. While the EF-1 locus of B. divergens has been described, structural variation between isolates was poorly defined. In order to fill this gap, we performed a comparative analysis of the EF-1-IG region in B. divergens human (Rouen 87, and Spanish sample), and bovine (Turkiye) host isolates. Our findings revealed both conserved and variable elements, particularly in TA nucleotide repeat numbers and IG sequence length. The Spanish isolate exhibited the highest TA repeat expansion, whereas the Rouen 87 strain had the shortest IG region. Given the known role of repeat-rich promoter elements in gene regulation, these differences may influence EF-1 transcription. Additionally, these findings provide insights into the evolutionary divergence of B. divergens and its host adaptation mechanisms. This study establishes a foundation for future gene editing and transfection strategies, where selecting intergenic sequences with varying TA repeats could optimize transfection efficiency and explain phenotypic differences between isolates from different hosts or regions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.08.669274v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Compound Delivery of eVLPs Enhances Prime Editing for Targeted Genome Engineering and High-Throughput Screening</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Langley, J., Baudrier, L., Curry, J., Narta, K., Todesco, H. M., Potts, K., Morrissy, S., Mahoney, D. J., Billon, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present PRIME-VLP (Progressive Repeated Infections for Maximized Editing via Virus-Like Particles), a delivery strategy that enhances prime editing efficiency for both targeted genome engineering and high-throughput prime editing screening.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Engineered virus-like particles (eVLPs) enable transgene-free ribonucleoprotein delivery for genome editing applications, yet optimized delivery strategies for high-throughput applications remain unexplored. Prime editing enables precise genomic modifications but suffers from limited efficiency that constrains its widespread adoption. Here, we present PRIME-VLP (Progressive Repeated Infections for Maximized Editing via Virus-Like Particles), a delivery strategy that enhances prime editing efficiency for both targeted genome engineering and high-throughput prime editing screening. PRIME-VLP leverages the temporal dynamics of eVLP-mediated editing through multiple sequential transductions with sub-saturating eVLP doses delivered at optimal intervals. This approach achieves 1.5 to 2.8-fold improvements in editing efficiency across diverse genomic targets and cell types. PRIME-VLP maintains high specificity without increasing off-target effects, compromising cellular viability or causing transcriptional perturbations. By decoupling pegRNA and editor delivery through pegRNA-free eVLPs, PRIME-VLP enables pooled prime editing screens, circumventing transgene silencing limitations of conventional lentiviral-based screens. Using a 6,000-pegRNA library targeting TP53, PRIME-VLP achieved 2.8-fold higher editing efficiency and improved reproducibility compared to conventional lentiviral delivery. An eVLP-based screen identified functional TP53 loss-of-function variants that confer resistance to MDM2 inhibition by Nutlin-3. This work expands the versatility of eVLPs beyond their current in vivo therapeutic applications, demonstrating their promise for high-throughput functional genomics by overcoming the delivery limitations of lentiviral systems.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.11.669692v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Protein Function Prediction via Contig-Aware Multi-Level Feature Integration</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yang, L., Du, K., Lu, Y., Wang, M., Zhang, H., Yang, S., Lin, Y., Zhuo, J., Zhang, D., Jiang, Y., Zhang, X., Li, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel deep learning model that integrates intra-protein features including sequence and predicted structure with inter- protein features capturing functional linkages among CDSs in contigs, and achieve improvements in accuracy, precision, recall and F1-score.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Proteins play a central role in biological processes, and accurately predicting their functions is crucial for biomedical research. While computational methods have advanced significantly, most approaches rely solely on sequence or structure, neglecting critical inter-protein relationships, such as the topological arrangement of coding sequences (CDSs) within contigs. To address this gap, we propose CAML, a novel deep learning model that integrates intra-protein features including sequence and predicted structure with inter-protein features capturing functional linkages among CDSs in contigs. Specifically, CAML employs a Graph Isomorphism Network (GIN) to extract structural features from predicted protein contact graphs and ESM-2 for sequence embeddings. Additionally, it leverages k-mer frequencies and a Bidirectional Long Short-Term Memory (BiLSTM) network to model functional relationships among co-localized CDSs within contigs, capturing operon-like associations. Extensive experiments demonstrate that CAML outperforms the state-of-the-art methods in accuracy, precision, recall and F1-score, achieving improvements of 11.24%, 12.43%, 13.59%, and 13.30%, respectively over the second-best model. Ablation studies further confirm the critical contribution of CAML's multi-level biological feature integration in enhancing functional annotation accuracy. Our study demonstrates the importance of integrating structural, sequential, and CDSs topological features for accurate protein function prediction, providing a robust computational framework for genomics research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.07.669053v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SpaFoundation: a visual foundation model for spatial transcriptomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, N., Long, Y., Xia, S., Jia, P., Jin, Y., Chen, S., Chen, J., Li, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present SpaFoundation, a versatile visual foundational model with 80 million trainable parameters, pre-trained on 1.84 million histological image patches to learn general-purpose imaging representations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Spatial transcriptomics has revolutionized biological research but is limited by trade-offs between spatial resolution and gene coverage. Here we present SpaFoundation, a versatile visual foundational model with 80 million trainable parameters, pre-trained on 1.84 million histological image patches to learn general-purpose imaging representations. Using histology images alone, SpaFoundation accurately infers spatial gene expression and demonstrates strong robustness and transferability across tasks such as super-resolution inference, tumor detection, and spatial clustering.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.07.669202v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Predictive Gene Discovery with EPCY: A Density-Based Alternative to DE analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Audemard, E. O., Spinella, J.-F., Lavallee, V.-P., Hebert, J., Sauvageau, G., Lemieux, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present EPCY, a method that ranks genes based on their predictive power using cross-validated classifiers and density estimation, without relying on null hypothesis testing.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Identifying predictive genes from high-throughput data remains a key challenge in biomedical research. Most current approaches rely on statistical tests to select differentially expressed genes (DEGs), which may not align with the goal of predicting outcomes. We present EPCY, a method that ranks genes based on their predictive power using cross-validated classifiers and density estimation, without relying on null hypothesis testing. Applied to both bulk and single-cell RNA sequencing datasets, EPCY consistently outperforms benchmark DEG-based methods in selecting robust candidate genes. It also demonstrates greater stability across varying cohort sizes, enabling reproducible gene prioritization even in large, heterogeneous datasets. EPCY provides interpretable predictive scores, facilitating candidate selection aligned with downstream validation goals.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.07.668357v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Automated Annotation and Validation of Human Respiratory Virus Sequences using VADR</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Furlong, J., Goya, S., Nawrocki, E. P., Calhoun, V., Hatcher, E., Yankie, L., Greninger, A. L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed and validated 12 new reference sequence based VADR models targeting key human respiratory viruses: measles virus, mumps virus, rubella virus, human metapneumovirus, human parainfluenza virus types 1-4, and seasonal coronaviruses.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurate annotation of viral genomes is essential for reliable downstream analysis and public data sharing. While NCBI Viral Annotation DefineR (VADR) pipeline provides standardized annotation and quality control, it only supports six viral groups to date. Here, we developed and validated 12 new reference sequence based VADR models targeting key human respiratory viruses: measles virus, mumps virus, rubella virus, human metapneumovirus, human parainfluenza virus types 1-4, and seasonal coronaviruses (229E, NL63, OC43, HKU1). Model construction was guided by a comprehensive analysis of intra-species genomic and phylogenetic diversity, enabling the development of genotype-specific models associated with reference genomes that defined expected genome structure and annotation. Models were trained on 5,327 publicly available complete viral genomes and tested on 372 viral genomes not yet submitted to GenBank. VADR passed 96.3% of publicly available viral genomes and 98.1% of viral genomes not in the training set, correctly identifying overlapping ORFs, mature peptides, and transcriptional slippage as well as genome misassemblies. VADR detected novel viral biology including the first reported HCoV-OC43 NS2 knockout in a human infection and novel G and SH coding sequence lengths in human metapneumovirus. These VADR models are publicly available and are used by NCBI curators as part of the GenBank submission pipeline, supporting high-quality, scalable viral genome annotation for research and public health.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.07.669219v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">misosoup: A metabolic modeling tool for identifying minimal microbial communities reveals pervasive cross-feeding-driven niche expansion</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ochsner, N., San Roman, M., Bonhoeffer, S., Pascual-Garcia, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce misosoup, a Python package for identifying minimal communities (MInimal Supplying cOmmunity Search) that enables the systematic identification of communities that support microbial growth in environments where individual species fail to survive alone.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Microbial survival and function often depend on metabolic interactions within communities. Therefore, a central question to disentangle microbial organization is what minimal groups of species are able to thrive in a given medium -referred to as "minimal communities". Answering this question is essential for understanding microbial distribution, enhancing laboratory cultivation, or designing synthetic consortia. Here, we introduce misosoup, a Python package for identifying minimal communities (MInimal Supplying cOmmunity Search). Through genome-scale constraint-based metabolic modeling, misosoup enables the systematic identification of communities that support microbial growth in environments where individual species fail to survive alone. We validate misosoup against experimentally verified minimal communities, demonstrating its ability to predict known cooperative interactions. Applying misosoup to a set of 60 marine microbes, we observe widespread metabolic niche expansion, with approximately 66% of species capable of colonizing more environments in communities than in isolation. Our results highlight the frequent role of metabolic cross-feeding in niche expansion, whereby certain species depend on bidirectional exchange to utilize available resources. The detailed outputs provided by misosoup facilitate research on key questions such as the identification of functional groups. By efficiently identifying minimal communities and elucidating mechanisms underlying species interactions, misosoup provides a powerful tool for microbial ecology and community design, with potential applications in both research and biotechnological innovation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.07.669121v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">STARComm Scalably Detects Emergent Modules of Spatial Cell-Cell Communication in Inflammation and Cancer.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Huynh, K., Matuck, B. F., de Souza, D., Zhang, X., Fatobene, G., Soares Junior, L. A. V., Ferraz da Silva, L. F., Rocha, V., Byrd, K., Liu, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A scalable, scalable, and spatially resolved computational method that identifies Multicellular Communication Interaction Modules (MCIMs) by detecting spatially co-located receptor-ligand activity from high-plex spatial transcriptomics in 2D and 3D.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In humans, cell-cell communication orchestrates tissue organization, immune coordination, and repair, yet spatially mapping these interactions remains a challenge for biology. We introduce STARComm, a scalable-interpretable computational method that identifies Multicellular Communication Interaction Modules (MCIMs) by detecting spatially co-located receptor-ligand activity from high-plex spatial transcriptomics in 2D and 3D. Applied to an atlas of >14million cells across 8 cancers, STARComm revealed 24 conserved and tumor-specific MCIMs, including a fibro-immune module with targetable axes linked to immune exclusion and immunotherapy resistance. In chronic graft-versus-host disease, STARComm identified three salivary gland MCIMs predictive of patient death and two druggable axes (CXCL12-CXCR4, CCL5-SDC4), both with FDA-approved therapeutics. STARComm demonstrated that peripheral tissue profiling can forecast fatality nearly 3 years in advance using minor salivary glands. By enabling scalable biomarker discovery, drug targeting, and spatially resolved precision profiling, STARComm bridges the gap between spatial biology and clinical translation, advancing the field of spatial medicine.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.07.669133v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Semantic Framework for Predicting Herbal-Drug Biotransformation Conflicts via Biomedical Literature Mining</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Verandani, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a semantic framework that synthesizes knowledge from biomedical ontologies, curated databases, and full-text literature to model potential biotransformation conflicts between natural compounds and pharmaceutical agents.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The widespread use of herbal supplements alongside conventional medicines increases the risk of unpredictable interactions affecting drug absorption and metabolism. This study introduces a comprehensive semantic framework that synthesizes knowledge from biomedical ontologies, curated databases, and full-text literature to model potential biotransformation conflicts between natural compounds and pharmaceutical agents. Leveraging advanced relation extraction systems and graph-based inference techniques, we constructed an enriched knowledge graph capable of highlighting mechanistic pathways involving enzymes, transporters, and drug constituents. Case studies with compounds like green tea and kratom demonstrate the framework's potential to surface both known and previously underexplored interactions. The proposed system offers a scalable, hypothesis-generating platform for early-stage pharmacokinetic safety analysis in the context of natural product co-administration.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.07.668963v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">KeySDL: Sparse Dictionary Learning for Keystone Microbe Identification</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gordon, M., Akyol, T. Y., Amos, B. K., Andersen, S. U., Williams, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a keystone identification approach called KeySDL which reduces the amount of data required by incorporating assumptions about the type of microbial dynamics present in the experimental system.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Identification of microbes with large impacts on their microbial communities, known as keystone microbes, is a topic of long-standing interest in microbiome research. However, many approaches to identify keystone microbes are limited by the inherent nonlinearity and state-dependence of microbial dynamics. Machine learning approaches have been applied to address these shortcomings but often require more data than is available for a given microbial system. We propose a keystone identification approach called KeySDL which reduces the amount of data required by incorporating assumptions about the type of microbial dynamics present in the experimental system. The data are modeled as originating from a Generalized Lotka-Volterra (GLV) model, an architecture commonly used to simulate microbial systems. The parameters of this model are then estimated using Sparse Dictionary Learning (SDL) Compared to existing methods, this approach allows accurate prediction of keystone microbes from small numbers of samples and provides an output interpretable as reconstructed system dynamics. We also propose a self-consistency score to help evaluate whether the assumption of GLV dynamics is reasonable for a given dataset, either through the application of KeySDL or other analysis tools validated using GLV simulation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.07.669165v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

