<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Thursday, 01 May 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">YME1L1 Dysfunction Associated With 3-Methylglutaconic Aciduria.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Demetriadou A, Grafakou O, Georgiou T, Burska D, Malekkou A, Krizova J, Paramera E, Mavrikiou G, Dionysiou M, Theodosiou A, Sismani C, Anastasiadou V, Ioannou I, Papakonstantinou E, Hansikova H, Drousiotou A, Petrou PP.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #fedbf2; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, pan-genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We report a new type of secondary 3-methylglutaconic aciduria in two siblings with sensorineural hearing loss and neurological abnormalities associated with a novel, homozygous missense variant in the YME1L1 gene which encodes a mitochondrial ATP-dependent metalloprote</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> 3-methylglutaconic aciduria (3-MGCA) is a biochemical finding in a diverse group of inherited metabolic disorders. Conditions manifesting 3-MGCA are classified into two major categories, primary and secondary. Primary 3-MGCAs involve two inherited enzymatic deficiencies affecting leucine catabolism, whereas secondary 3-MGCAs comprise a larger heterogeneous group of conditions that have in common compromised mitochondrial energy metabolism. Here, we report 3-MGCA in two siblings presenting with sensorineural hearing loss and neurological abnormalities associated with a novel, homozygous missense variant (c.1999C>G, p.Leu667Val) in the YME1L1 gene which encodes a mitochondrial ATP-dependent metalloprotease. We show that the identified variant results in compromised YME1L1 function, as evidenced by abnormal proteolytic processing of substrate proteins, such as OPA1 and PRELID1. Consistent with the aberrant processing of the mitochondrial fusion protein OPA1, we demonstrate enhanced mitochondrial fission and fragmentation of the mitochondrial network in patient-derived fibroblasts. Furthermore, our results indicate that YME1L1L667V is associated with attenuated activity of rate-limiting Krebs cycle enzymes and reduced mitochondrial respiration, which may explain the build-up of 3-methylglutaconic and 3-methylglutaric acid due to the diversion of acetyl-CoA, not efficiently processed in the Krebs cycle, towards the formation of 3-methylglutaconyl-CoA, the precursor of these metabolites. In summary, our findings classify YME1L1 deficiency as a new type of secondary 3-MGCA, thus expanding the genetic landscape and facilitating the diagnosis of inherited metabolic disorders featuring this biochemical phenotype.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40255048/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Tri-omic single-cell mapping of the 3D epigenome and transcriptome in whole mouse brains throughout the lifespan.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chai H, Huang X, Xiong G, Huang J, Pels KK, Meng L, Han J, Tang D, Pan G, Deng L, Xiao Q, Wang X, Zhang M, Banecki K, Plewczynski D, Wei CL, Ruan Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #fedbf2; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, pan-genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A single-cell method to map chromatin accessibility, chromatin interactions and RNA expression simultaneously, which is remarkably correlated with transcriptional activity, open chromatin state and genome folding density.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Exploring the genomic basis of transcriptional programs has been a long-standing research focus. Here we report a single-cell method, ChAIR, to map chromatin accessibility, chromatin interactions and RNA expression simultaneously. After validating in cultured cells, we applied ChAIR to whole mouse brains and delineated the concerted dynamics of epigenome, three-dimensional (3D) genome and transcriptome during maturation and aging. In particular, gene-centric chromatin interactions and open chromatin states provided 3D epigenomic mechanism underlying cell-type-specific transcription and revealed spatially resolved specificity. Importantly, the composition of short-range and ultralong chromatin contacts in individual cells is remarkably correlated with transcriptional activity, open chromatin state and genome folding density. This genomic property, along with associated cellular properties, differs in neurons and non-neuronal cells across different anatomic regions throughout the lifespan, implying divergent nuclear mechano-genomic mechanisms at play in brain cells. Our results demonstrate ChAIR's robustness in revealing single-cell 3D epigenomic states of cell-type-specific transcription in complex tissues.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40301621/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">How genomics can help unravel the evolution of endophytic fungi.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jefferson Brendon Almeida Dos Reis, Andrei Stecca Steindorff, Adriana Sturion Lorenzi, Danilo Batista Pinho, Helson Mario Martins do Vale, Georgios Joannis Pappas</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #fedbf2; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, pan-genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comprehensive review of the evolution of endophytic fungal genomes, focusing on the evolutionary processes that shape their genomes, and the complex interdependencies governing their symbiotic interactions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Endophytic fungi (EFs) form intimate associations with plants, residing within their tissues without causing apparent harm. Understanding the evolution of endophytic fungal genomes is essential for uncovering the mechanisms that drive their symbiotic relationships with host plants. This review explores the dynamic interactions between EFs and host plants, focusing on the evolutionary processes that shape their genomes. We highlighted key genomic adaptations promoting their endophytic lifestyle, including genes involved in plant cell wall degradation, secondary metabolite production, and stress tolerance. By combining genomic data with ecological and physiological information, this review provides a comprehensive understanding of the coevolutionary dynamics between EFs and host plants. Moreover, it provides insights that help elucidate the complex interdependencies governing their symbiotic interactions.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40289066/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Expression and immunological role of FUNDC2 in pan-cancer.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Qiu X, Wang S, Li C, Wang Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #fedbf2; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, pan-genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We investigate the possible role and mechanism of FUNDC2 in pan-cancer and the relationship between level of the protein and prognosis of the patients with pan- cancer.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> FUNDC2 is a novel mitochondrial protein and is highly involved in various cancers. However, expression pattern and possible role and mechanism of FUNDC2 in pan-cancer remain to be investigated. TIMER 2.0 was used to investigate the expression patterns and immune infiltration of FUNDC2. GEPIA was applied to study the relationship between level of FUNDC2 and prognosis of the patients with pan-cancer. STRING was employed to analyze the potential interacting proteins of FUNDC2. The phosphorylation sites were predicted by cBioPortal and PhosphoNet. Furthermore, variations of FUNDC2 in cancers were investigated by cBioPortal. Finally, AlphaFold was used to predict the structure of FUNDC2. The data show that there were significant differences in the expression levels of FUNDC2 between cancer tissues and controls. Specifically, the levels of FUNDC2 in 8 cancers were significantly lower than the respective controls. The survival time of the cancer patients with higher levels of FUNDC2 was longer than that of lower FUNDC2 in most different types of cancers. The pattern of FUNDC2 was significantly related to immune infiltration of B cells of cancer patients. STRING analysis revealed that FUNDC2 can interact with FUNDC1, et al. Fifteen phosphorylation sites were predicted by PhosphoNet and cBioPortal, of which the S167 also overlapped with the mutation sites of FUNDC2. These data collectively show that the mitochondrial protein FUNDC2 may serve as a possible prognostic biomarker across various cancers and the mechanism may include immune infiltration.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40294153/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Are you with me? Co-occurrence tests from community ecology can identify positive and negative epistasis between inversions in Mimulus guttatus.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Madrigal-Roca LJ, Kelly JK.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #fedbf2; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, pan-genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show how co-occurrence metrics used in community ecology can provide additional insight regarding coupling and repulsion of inversions at genotypic level.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Chromosomal inversions are structural genetic variants that can play a crucial role in adaptive evolution and speciation. Patterns of attraction and repulsion among unlinked inversions - whether they tend to be carried by the same or different individuals- can indicate how selection is acting on these polymorphisms. In this study, we compare analytical techniques using data from 64 inversions that segregate among 1373 F2 plants of the yellow monkeyflower Mimulus guttatus. Mendelian assortment provides a strong null hypothesis for [Formula: see text] contingency tests. Here, we show how co-occurrence metrics used in community ecology can provide additional insight regarding coupling and repulsion of inversions at genotypic level. The centered Jaccard/Tanimoto index and the affinity score describe the specific way that inversions interact to generate epistasis for plant survival. We further explore the use of network analysis to visualize inversion interactions and to identify essential third and fourth order interactions, which expand the traditional pairwise scope of the co-occurrence metrics. We suggest that a combination of different statistical approaches will provide the most complete characterization of the fitness effects, both for inversions and other polymorphisms essential to adaptation and speciation.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40294049/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

