<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Saturday, 09 August 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">FusionProt: Fusing Sequence and Structural Information for Unified Protein Representation Learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kalifa, D., Singer, U., Radinsky, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce FusionProt, a novel protein representation learning framework designed to learn unified representations of both the 1D and 3D structures of proteins simultaneously.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Proteins play a crucial role in biological processes and understanding the complex mechanisms within living organisms. Accurate protein representation is vital for advancements in drug development, synthetic biology, and understanding disease mechanisms. Current methodologies primarily focus on one-dimensional (1D) structures, emphasizing amino acid sequences. However, this approach neglects the essential three-dimensional (3D) structural details that determine protein functionality and interactions. Initial attempts to leverage both the 1D and 3D modalities have shown limited benefits compared to single modality representations. In this work, we introduce FusionProt, a novel protein representation learning framework designed to learn unified representations of both the 1D and 3D structures of proteins simultaneously. FusionProt incorporates an innovative learnable fusion token that serves as an adaptive bridge, enabling an iterative exchange of information between a protein language model and the protein's 3D structure graph. This token is integrated into the training process of both modalities, enabling seamless propagation of information and facilitating comprehensive representation through iterative learning cycles. Empirical evaluations across various biological tasks demonstrate that FusionProt achieves state-of-the-art performance, offering a significant advancement in protein representation learning.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.06.668973v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Advancing Target Discovery Through Disease-Specific Integration of Multi-Modal Target Identification Models and Comprehensive Target Benchmarking System</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Leung, H., Duan, C., Gou, W., Chen, J., Xin, Y., Zheng, Z., Naumov, V., Gennert, D., Zhang, M., Aliper, A., Ren, F., Izumchenko, E., Pun, F. W., Zhavoronkov, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a unified platform combining machine learning-based target identification with comprehensive benchmarking.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Target identification is crucial for drug development. AI-driven approaches leveraging multi-omics and computational modeling can accelerate this process. However, integrating multi-modal data for disease-specific target identification and predicting translational potential remains challenging. Moreover, the absence of a systematic evaluation framework for model performance limits confidence in target reliability. We present a unified platform combining machine learning-based target identification with comprehensive benchmarking. As a testbed, we developed Target Identification Pro (TID-Pro), a disease-specific model spanning 38 diseases across oncology, metabolic, immune, fibrotic, and neurological categories. TID-Pro shows strong predictive performance for clinical-stage targets and reveals disease-specific patterns, underscoring the need for tailored target detection models. We next created Target Identification Benchmark (TargetBench 1.0) to rigorously assess target identification systems, including large language models, based on their ability to recover established targets and find high-quality novel candidates. This integrated approach offers a streamlined strategy to evaluate target discovery models, ultimately improving drug development efficiency.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.06.668866v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PBMCpedia: A Harmonized PBMC scRNA-seq Database With Unified Mapping and Enhanced Celltype Annotation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hoffmann, E. S., Hombrecher, L., Diks, I. F., Flotho, M., Keller, A., Grandke, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present PBMCpedia, a unified PBMC atlas comprising over 4.3 million single cells from 519 samples across 24 publicly available scRNA-seq studies, enabling reproducible, metadata-aware comparisons across biological contexts.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Reproducibility in single-cell transcriptomics remains limited by inconsistent preprocessing, heterogeneous cell type annotations, and study-specific batch effects. This challenge is particularly pronounced in peripheral blood mononuclear cell (PBMC) datasets, which are central to immunological research but rarely harmonized across studies. We present PBMCpedia, a unified PBMC atlas comprising over 4.3 million single cells from 519 samples across 24 publicly available scRNA-seq studies. Unlike prior efforts, PBMCpedia reprocesses all raw sequencing data using a single, standardized pipeline with consistent quality control, batch correction, and multi-resolution cell type annotation. The dataset spans 14 diseases, including autoimmune, infectious, and neurodegenerative conditions, alongside healthy controls, enabling reproducible, metadata-aware comparisons across biological contexts. In addition to transcriptomes, PBMCpedia includes TCR/BCR repertoire data for 75 samples and surface protein measurements for 56 samples, supporting integrative immune profiling at the transcriptomic and proteogenomic levels. To support exploration and accessibility, we provide an interactive web interface (https://web.ccb.uni-saarland.de/pbmcpedia/) for querying gene expression, marker genes, and pathway enrichment across cell types, conditions, sexes, and age groups. PBMCpedia fills a critical gap by offering a transparent, harmonized, and disease-diverse PBMC resource designed for cross-study immune profiling and discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.06.668843v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MoCETSE: A mixture-of-convolutional experts and transformer-based model for predicting Gram-negative bacterial secreted effectors</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shi, H., Lin, Y., Liu, D., Zou, Q.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a deep learning model named MoCETSE to predict Gram-negative bacterial effector proteins by using pre-trained protein language models for learning and a target preprocessing network based on convolutional experts.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Identifying effector proteins of Gram-negative bacterial secretion systems is crucial for understanding their pathogenic mechanisms and guiding antimicrobial strategies. However, existing studies often directly rely on the outputs of protein language models for learning, which may lead to difficulties in accurately recognizing complex sequence features and long-range dependencies, thereby affecting prediction performance. In this study, we propose a deep learning model named MoCETSE to predict Gram-negative bacterial effector proteins. Specifically, MoCETSE first uses the pre-trained protein language model ESM-1b to transform raw amino acid sequences into context-aware vector representations. Then, by employing a target preprocessing network based on a mixture of convolutional experts, multiple sets of convolutional kernel "experts" process the data in parallel to separately learn local motifs and short-range dependencies as well as broader contextual information, generating more expressive sequence representations. In the transformer module, MoCETSE incorporates relative positional encoding to explicitly model the relative distances between residues, enabling the attention mechanism to precisely recognize the sequential relationships and long-range functional dependencies among amino acids, thereby achieving high-accuracy prediction of secreted effectors. MoCETSE has demonstrated outstanding predictive ability in 5-fold cross-validation and independent testing. Benchmark test shows that the performance of MoCETSE surpasses existing excellent binary and multi-class classifiers.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.06.668857v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ICEPIC: A Toolkit to Discover Ice Binding Proteins from Sequence</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, J., Suresh, S., Gleizer, S., Ewens, S., Venkat, A., Zulkower, V., Biernacki, T., Wen, D., Li, C., Eslami, M., Buckhout-White, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a sequence-based model that leverages contextual embeddings from protein language models to predict ice binding potential, as well as the expression and activity potential of candidate proteins.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Ice binding proteins, such as antifreeze proteins (AFPs) and ice nucleation proteins (INPs), are critical for survival in subzero environments and have wide-ranging applications in biotechnology, agriculture, and materials science. Current discovery methods for these proteins are constrained by low throughput and limited datasets that are not conducive for engineering. Here, we present a high-throughput, sequence-based model that leverages contextual embeddings from protein language models to predict ice binding potential, as well as the expression and activity potential of candidate proteins. Using a curated data corpus of over 18,000 ice binding proteins - far larger than previous datasets - we fine-tuned a ProtBERT-based model, achieving 99% accuracy for prediction of ice binding potential. Sensitivity analyses through targeted mutagenesis (alanine and threonine substitutions) confirmed the model's biological significance, revealing functionally important residues and sequence patterns. Additionally, we developed an expression prediction model that achieved an R2 score of 0.64 and low false-negative rates in identifying highly expressible candidates in Pichia pastoris. An additional regression model trained to predict ice activity as measured by thermal hysteresis achieved an R2 score of at least 0.79 with a clear difference in prediction between ice binding and non-ice binding proteins. Our toolkit advances the predictive accuracy, interpretability, and scalability of ice binding protein discovery, offering a powerful tool for protein engineering in cold-environment applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.08.669420v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

