<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Wednesday, 30 July 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Single-Sample, Multiomic Mass Spectrometry for Investigating Drug Effects and Mechanisms.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Iqbal Mahmud, Wai-Kin Chan, Karen Yannell, Cate Simmermaker, Genevieve C Van de Bittner, Linfeng Wu, Daniel Chan, Sheher Banu Mohsin, Yiwei Liu, John Sausen, John N Weinstein, Philip L Lorenzi</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We provide a streamlined, three-in-one LC-MS/MS workflow for targeted metabolomics, lipidomics, and proteomics and, as a proof of principle, demonstrate its ability to generate new hypotheses about mechanisms of drug toxicity.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Poor therapeutic indexes are a principal cause of drug attrition during development. To develop multiomic methods for elucidating potentially targetable mechanisms of drug toxicity, we performed profiling of the response to subtoxic and toxic doses of l-Asparaginase (ASNase) in immune-compromised mice. ASNase is an enzyme-drug approved for the treatment of pediatric acute lymphoblastic leukemia (ALL) but too toxic for use in adults, making it an ideal test case. We collected 20-μL whole blood samples longitudinally, processed them to plasma, and extracted three molecule types (metabolites, lipids, and proteins) from each sample. We then analyzed the extracts using multiple reaction monitoring (MRM) of 500+ water-soluble metabolites, 750+ lipids, and 375 peptides on a triple quadrupole LC-MS/MS platform. Metabolites, lipids, and peptides that were modulated in a dose-dependent manner appeared to converge on antioxidation, inflammation, autophagy, and cell death pathways, prompting the hypothesis that inhibiting one or more of those pathways might decrease ASNase toxicity while preserving anticancer activity. The present studies were not designed to address therapeutic index directly, because efficacy was not studied. We provide here a streamlined, three-in-one LC-MS/MS workflow for targeted metabolomics, lipidomics, and proteomics and, as a proof of principle, demonstrate its ability to generate new hypotheses about mechanisms of ASNase toxicity.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40726195/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">HVRLocator: A Computationally Efficient Tool for Identifying Hypervariable Regions in 16S rRNA Big Datasets</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Arboleda-Baena, C., Borim Correa, F., Saraiva, J. P., Castillo, S., Kasmanas, J. C., Chatzinotas, A., Jurburg, S. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the HVRLocator, a computational tool that reliably identifies hypervariable regions of 16S rRNA amplicon data, enhancing metadata quality and enabling more robust large-scale microbiome studies.</p><p style="font-size: 16px; text-align: justify;">Background: Amplicon sequencing of the 16S rRNA gene is widely used to assess microbial diversity due to its cost-effectiveness and efficiency. However, public 16S rRNA datasets often lack standardized metadata, particularly information on the sequenced hypervariable regions or primers used, which are critical for accurate analysis and data reuse. To address this, we present the HVRLocator, a computational tool that reliably identifies sequenced hypervariable regions, enhancing metadata quality and enabling more robust large-scale microbiome studies. Results: The HVRLocator tool processed samples at an average rate of 0.147 per minute. Validation confirmed 100% accuracy in predicting alignment positions, correctly matching sequences to the expected primer regions based on literature. We demonstrated how to use the tool to select appropriate and comparable sequences for building a global bacterial database from V4 region amplicons of the 16S rRNA gene. Using HVRLocator, we selected 36,217 valid samples out of 45,882 runs, enabling us to identify cases where metadata incorrectly labeled sequences as targeting the V4 region. Conclusion: Even when metadata is available, it can be inaccurate or misleading. HVRLocator offers a reliable and efficient method to identify the exact hypervariable sequenced region, ensuring accurate processing of large-scale 16S rRNA amplicon data. By bypassing inconsistent metadata and literature, it streamlines data curation and enhances the reliability of microbial studies, syntheses, and meta-analyses. Its use is essential for critically evaluating published data and enabling accurate and reproducible research in microbial ecology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.24.666487v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Impact of integration on persistent homology clustering and biological signal detection in scRNA-seq data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Daneshmand, J., Chariker, J. H., Mistry, A., Rouchka, E. C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We investigate the impact of data integration on topological features and biological interpretability of single-cell RNA sequencing data and show how topological summaries can help disentangle biological signal from batch-related noise in single- cell data.</p><p style="font-size: 16px; text-align: justify;">Background: As the availability of single-cell RNA sequencing (scRNA-seq) data expands, there is a growing need for robust methods that enable integration and comparison across diverse biological conditions and experimental protocols. Persistent homology (PH), a technique from topological data analysis (TDA), provides a deformation-invariant framework for capturing structural patterns in high-dimensional data. Methods: In this study, PH was applied to a diverse collection of scRNA-seq datasets spanning eight tissue types to investigate how data integration affects the topological features and biological interpretability of the resulting representations. Clustering was performed based on PH-derived pairwise distances and global topological structure was assessed through Betti curves, Euler characteristics, and persistence landscapes. By comparing these summaries across raw, normalized, and integrated datasets, we examined whether integration enhances the detection of biologically meaningful patterns, or, conversely, obscures fine-scale structure. Results: This approach demonstrates that PH can serve as a powerful complementary strategy for evaluating the impact of integration and reveals how topological summaries can help disentangle biological signal from batch-related noise in single-cell data. This work establishes a framework for using topological methods to assess integration quality and highlights new avenues for interpreting complex transcriptomic landscapes beyond conventional clustering.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.24.666637v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Aridity shapes adaptive genomic divergence and population connectivity in a Southern African rodent</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Keilani, H. S., Fiston-Lavier, A.-S., Caminade, P., Loiseau, A., Galan, M., Parrinello, H., Brouat, C., Leblois, R., Avenant, N., Pillay, N., Ganem, G., Smadja, C. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that aridity shapes population connectivity and adaptation, with implications for climate resilience.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Elucidating the drivers of evolution in dry environments is central to understanding how organisms respond to climate change. While research on the genomics of adaptation is growing, aridity-driven intraspecific divergence remains poorly quantified. Here, we address this gap by using genomic data from 230 individuals of the arid-adapted four-striped mouse Rhabdomys bechuanae, sampled across an aridity gradient in southern Africa, a region facing increasing aridification. Combining these data with palaeoclimatic reconstructions and present-day aridity indices, we investigate, from a spatio-temporal perspective, how intraspecific genetic variation relates to aridity. Inference of past effective population size revealed a sharp decline around 15,000 years BP, coinciding with regional aridification and likely reflecting reduced connectivity during dry periods. Population structure followed a pattern of isolation by distance and mirrored the aridity gradient. Genotype-Environment Association analyses identified SNPs and genes significantly associated with aridity and genetically differentiated among populations, with functions related to water and energy conservation - as expected under arid conditions - as well as neurotransmission. These findings highlight the underappreciated role for neurological processes in coping with water and resource scarcity. More broadly, our integrative population genomics approach strongly suggests that aridity shapes population connectivity and adaptation, with implications for climate resilience.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.29.667317v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Computationally Driven Top-Down Mass Spectrometry of Ubiquitinated Proteins</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shestoperova, E. I., Ivanov, D. G., Zhong, J., Chien, L., Strieter, E. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the first integrated strategy that enables simultaneous determination of ubiquitin site and chain architecture using top-down mass spectrometry (TD-MS). Central to this approach is a custom computational platform that predicts Ub chain topology from tandem MS (MS2) fragmentation data by utilizing Bayesian</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Ubiquitination regulates numerous cellular processes through the attachment of polyubiquitin (Ub) chains that vary in linkage type, length, and branching topology. However, current mass spectrometry approaches cannot simultaneously define both the site of ubiquitination and the topology of the attached Ub chain on intact protein substrates. Here, we present the first integrated strategy that enables simultaneous determination of ubiquitin site and chain architecture using top-down mass spectrometry (TD-MS). Central to this approach is UbqTop, a custom computational platform that predicts Ub chain topology from tandem MS (MS2) fragmentation data by utilizing Bayesian-like scoring algorithm. To address the challenge of analyzing complex substrates, we combine this with selective Asp-N proteolysis, which digests the substrate while preserving intact Ub chains. This enables direct, site-resolved mapping of Ub chain topology on proteins. We demonstrate the broad utility of this method on both free Ub chains and multiply ubiquitinated protein substrates, including the resolution of isomeric chains and branched architectures. Together, this work establishes a powerful new framework for proteoform-level analysis of ubiquitin signaling with unprecedented structural resolution.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.24.666707v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MViewEMA: Efficient Global Accuracy Estimation for Protein Complex Structural Models Using Multi-View Representation Learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liu, D., Zhao, X., Zhang, T., Xie, L., Ye, E., Liang, F., Wang, H., Zhang, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present MViewEMA, a single-model EMA method that leverages a multi-view representation learning framework to integrate residue-residue interaction features from micro- environment, meso-environment, and macro-environment levels for global accuracy assessment of protein complex models.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Estimation of model accuracy (EMA) is crucial for the reliable application of protein structure models in biological research. With the rapid advancement of protein structure prediction techniques and the explosive growth of predicted structural data, existing EMA methods struggle to balance computational efficiency with estimation performance. Here, we present MViewEMA, a single-model EMA method that leverages a multi-view representation learning framework to integrate residue-residue interaction features from micro-environment, meso-environment, and macro-environment levels for global accuracy assessment of protein complex models. Benchmark results demonstrate that MViewEMA outperforms state-of-the-art EMA methods in global accuracy assessment, achieving more than a 10-fold improvement in computational efficiency compared to our previous method, DeepUMQA3. This method enables efficient selection of high-quality protein complex models from large-scale structural datasets and achieved top performance in model selection tracks during the CASP16 blind test, demonstrating its potential to enhance the accuracy of complex structure prediction when integrated into modern frameworks such as AlphaFold-Multimer, AlphaFold3, and DiffDock-PP.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.25.666906v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">From Library to Landscape: Integrative Annotation Workflows for Compound Libraries in Drug Repurposing</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Reinshagen, J., Seashore-Ludlow, B., Gadiya, Y., Gustavsson, A.-L., Tanoli, Z., Aittokallio, T., Huchting, J., Jenmalm-Jensen, A., Gribbon, P., Zaliani, A., Ballante, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel annotation pipeline for drug discovery and repurposing that streamlines the collection of harmonized data and facilitates analyses that are critical in drug repurpose efforts, while remaining versatile for broader applications.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In the rapidly advancing landscape of drug discovery and repurposing, efficient access and integration of chemical and bioactivity data from public repositories has become essential. We implemented two complementary annotation pipelines (KNIME- and Python-based) designed to automate the extraction and integration of curated chemical and bioactivity data from public repositories. These pipelines are adaptable to any user-provided compound library, allowing reproducible workflows to integrate data from heterogeneous sources (e.g., ChEMBL and PubChem). As part of the REMEDi4ALL project, which aims to establish a European platform for drug repurposing, we validated our framework on a harmonized subset of the Specs repurposing collection (over 5000 compounds, available in-house). Additionally, we developed two interactive dashboards that support multilayered analyses and visualization by integrating chemical properties, bioactivity profiles, and relational data. We show how this framework streamlines the collection of harmonized data and facilitates analyses that are critical in drug repurposing efforts, while remaining versatile for broader applications in drug discovery. Both pipeline protocols are publicly available online, and the dashboards are open access.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.23.666341v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">TopoLa: A Universal Framework to Enhance Cell Representations for Single-cell and Spatial Omics through Topology-encoded Latent Hyperbolic Geometry</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zheng, K., Wang, S., Xu, Y., Lei, Q., Zhao, Q., Liang, X., Feng, Q., Li, Y., Li, M., Xu, J., Wang, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose Topology-encoded Latent Hyperbolic Geometry (TopoLa), a computational framework enhancing cell representations by capturing fine-grained intercellular topological relationships.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent advances in cellular research demonstrate that scRNA-seq characterizes cellular heterogeneity, while spatial transcriptomics reveals the spatial distribution of gene expression. Cell representation is the fundamental issue in the two fields. Here, we propose Topology-encoded Latent Hyperbolic Geometry (TopoLa), a computational framework enhancing cell representations by capturing fine-grained intercellular topological relationships. The framework introduces a new metric, TopoLa distance (TLd), which quantifies the geometric distance between cells within latent hyperbolic space, capturing the network's topological structure more effectively. With this framework, the cell representation can be enhanced considerably by performing convolution on its neighboring cells. Performance evaluation across seven biological tasks, including scRNA-seq data clustering and spatial transcriptomics domain identification, shows that TopoLa significantly improves the performance of several state-of-the-art models. These results underscore the generalizability and robustness of TopoLa, establishing it as a valuable tool for advancing both biological discovery and computational methodologies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.23.666288v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genotypic Diversity of Hepatitis B and Delta Viruses in Peru: A Phylogenetic Study Using Next-Generation Sequencing.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Balbuena-Torres, J., Gavilan-Chavez, R., Cardenas-Bustamante, F., Terrazas-Aranibar, M., Martinez-Gago, M., Cabezas, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A cross-sectional study of hepatitis B virus and hepatitis delta virus in Peru using nextgeneration sequencing (NGS).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Objective: To characterize the molecular epidemiology and genomic diversity of hepatitis B virus (HBV) and hepatitis delta virus (HDV) in Peru using nextgeneration sequencing (NGS). Materials and Methods: A cross-sectional observational study was conducted using 209 serum/plasma samples from various regions of Peru, stored at the National Institute of Health. Detection of HBV and HDV was performed by PCR and RT-PCR, respectively. Nucleic acids were purified and quantified for library preparation and sequencing using the Illumina platform. Sequencing reads were aligned to reference genomes to determine genotypes, subgenotypes, and clinically relevant mutations. Geneious Prime was used for assembly and phylogenetic analysis, and STATA v.14.0 for statistical analysis. Results: HBV genotype F was the most prevalent (96.1%), with subgenotype F1b being predominant, followed by genotypes H, G, and C. Mutations associated with virulence and immune escape were identified, particularly A1762T/G1764A (66.7%) and A1762T/G1764A + G1896A (45.7%). All HDV sequences corresponded to genotype 3 (HDV-3), which was detected in six regions of the country. Conclusions: The findings confirm the predominant circulation of HBV genotype F and HDV genotype 3 in Peru, with genotypic diversity that may influence clinical outcomes, viral load, and response to treatment. Sustained molecular surveillance programs are recommended in endemic areas to guide control, treatment, and prevention strategies. Keywords: Hepatitis B, Hepatitis Delta, Molecular epidemiology, Next-generation sequencing, Genotypes, Peru.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.25.666658v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Machine Learning and Network Analysis to Predict Hypothetical Protein Functions of Aeromonas hydrophila</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> pirim, h., Rahman, Z., Saei, S., Gyawali, S., Marufuzzaman, M., Tajik, N., Tekedar, H. C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Aeromonas hydrophila, antibiotic resistant gram negative bacteria, is a major fish pathogen, which is considered to cause 13% of gastroenteritis cases in the United States.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Aeromonas hydrophila, antibiotic resistant gram negative bacteria, is a major fish pathogen. Moreover, A. hydrophila is considered to cause 13% of gastroenteritis cases in the United States. Therefore, it is important to identify groups of proteins that are effective in antibiotic resistance and causing mortality in aquaculture. We train machine learning models on existing A. hydrophila genomes to predict functions of 83 carefully filtered hypothetical proteins. Network analysis is conducted to cluster these proteins based on their similarities. Both ML and network analysis inform about possible roles of these proteins in vaccine candidacy and fish mortality.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.22.666223v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Mink by mink: stitching together signatures of subspecies adaptation through a pangenome of threatened mustelids</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Affini, A., Baranowski, H., Forbes, S., Foust, E. N., Hatley, K., Ni, E. L., McGuire, A., Paist, K., Rutter, M., Smith, R. N., Vargas, N., Akella, H., Castellano, K., Pauloski, N., Reid, N., O'Neill, R., King, T., Jockusch, E. L., Wegrzyn, J. L., Hapeman, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A pangenome for the American mink (Neogale vison), a semi-aquatic Mustelidae carnivoran with broad ecological range across North America, to investigate the evolutionary history and adaptive signatures of mink subspecies.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The American mink (Neogale vison), a semi-aquatic Mustelidae carnivoran with broad ecological range across North America, includes several putative subspecies of conservation concern. To investigate the evolutionary history and adaptive signatures of mink subspecies, chromosome-scale genome assemblies were generated for six individuals representing three southern subspecies: N. vison evergladensis, N. vison vulgivaga, and N. vison lutensis. Genomes were assembled using Illumina short reads, scaffolded with Oxford Nanopore long reads, and aligned to the phased N. vison reference genome. Assemblies ranged from 75.9% to 97.8% completeness, with five meeting thresholds for pangenome construction. A reference-free pangenome revealed an open architecture, highlighting considerable subspecies diversity. Subspecies-specific gene enrichment reflected adaptation: N. vison evergladensis showed enrichment in traits related to reproduction and sensory function; N. vison vulgivaga in cytoskeletal remodeling and oxidative stress; and N. vison lutensis in neuronal development, synaptic plasticity and cellular stress pathways. Assessment of the mitogenomes resolved N. vison lutensis as a distinct lineage, while nuclear data supported broader subspecies divergence but lacked fine scale resolution. N. vison evergladensis showed multiple signatures of small population size, including inbreeding coefficients (FROH) above 0.5, and displayed consistent population decline over time via demographic inference. Our findings support evergladensis as a distinct subspecies, supported by both the mitogenome phylogeny, and significant functional differentiation. As the first pangenome for Mustelidae, this study demonstrates the power of integrating cross-platform sequencing with natural history specimens to improve the resolution on signatures of adaptation and inform conservation policy and management of threatened populations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.25.666817v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Aphid infestation induces plant-sex-specific changes in floral chemistry and pollinator behaviour in Silene latifolia</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zill, K. B., Stegemann, T., Kaltenegger, E., Demetrowitsch, T. J., Bilger, W. J., Berndt, H., Schrieber, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A aphid infestation alters floral traits in a plant-pollinator-herbivore system, with stronger declines in female floral attractiveness, suggesting sex-specific defence strategies and possible involvement of indirect defences.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Pollinators consistently share the complex information and resource landscape of their host plants with herbivores. Yet, how sap feeders affect floral attractiveness to pollinators remains poorly understood, despite the critical role of this tripartite interaction in natural and agricultural ecosystems. These intricate interactions may be further enriched by pronounced sexual dimorphisms in dioecious species. In this study, we examined how infestation by the oligophagous aphid Brachycaudus lychnidis affects sex-specific interactions among the dioecious plant Silene latifolia and its specialist moth pollinator Hadena bicruris. We exposed male and female plants to aphid herbivory and evaluated its effects on floral traits (visual cues, floral scent, and nectar chemistry) and pollinator behaviour. Contrary to sexual selection theory, female plants were not less attractive than males under control conditions. Instead, our results suggest that females exhibit plasticity in floral trait expression depending on pollination success. Moreover, both sexes produced a diverse array of potentially bioactive nectar metabolites, with females showing higher abundances of many compounds that may influence female pollinator behaviour. Aphid infestation altered floral traits differently between sexes, with stronger declines in female floral attractiveness, suggesting sex-specific defence strategies and possible involvement of indirect defences in stabilizing this specialized plant-pollinator-herbivore system.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.22.666187v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Supervised Learning Approach to Classify Clinical Outcomes in Helicobacter pylori-positive Patients</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Narasimhan, V., Warrier, S. P., Jacob, J. J., Priya T, M., Varadaraj, N., Thomas, G. G., Veeraraghavan, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A supervised learning framework for gastric cancer prediction using H. pylori genomes and a combination of clinical and genome-derived features.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Helicobacter pylori (H. pylori) is a gastric pathogen with global prevalence that is implicated in a lot of clinical outcomes, ranging from gastritis to gastric cancer. While chronic gastritis is a nearly universal outcome among infected individuals, only a small subset progresses to severe diseases like gastric cancer, with these outcomes being governed by a mixture of bacterial, host and environmental factors. This study presents a supervised learning framework in order to predict whether an H. pylori infection will result in gastric cancer or a non-malignant outcome with a combination of clinical and genome-derived features. A dataset with 1,363 H. pylori genomes containing host metadata and annotated genomic features was curated. Feature extraction was performed using gene presence/absence profiles, various sequence descriptors, and variant-level annotation features. A white-box logistic regression model and black-box eXtreme Gradient Boosting (XGBoost) classifier and Random Forest models were trained and Synthetic Minority Over-sampling Technique for Nominal and Continuous (SMOTE-NC) was used for addressing class imbalance. SHapely Additive exPlanations (SHAP) values were used to assess model interpretability. The baseline model showed an Area Under Receiver Operating Characteristic curve (AUROC) of 0.89 while the XGBoost model and Random Forest models displayed an AUROC of 0.95. The black-box models improved recall for the gastric cancer group with 0.83 for XGBoost and 0.82 for Random Forest while the baseline model had a recall of 0.73. Feature importance analyses were also used to identify the key drivers of gastric cancer prediction including host age, geographic origin and specific disruptive mutations. These findings support the use of machine learning in biomarker-guided decision-making in assessing gastric disease risk.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.28.667137v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Ab-VS: Evaluating Large Language Models for Virtual Antibody Screening via Antibody-Antigen Interaction Prediction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, Y., Tsuda, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present Ab-VS-Bench, a new benchmark for evaluating large language models (LLMs) on antibody virtual screening tasks through natural language instructions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We present Ab-VS-Bench, a new benchmark for evaluating large language models (LLMs) on antibody virtual screening (VS) tasks through natural language instructions. Unlike prior efforts that focus on structural annotation, binding prediction, or developability using frozen antibody models, Ab-VS-Bench targets the end-to-end VS workflow and leverages LLMs' instruction-following capabilities. The benchmark comprises three core tasks inspired by small-molecule VS: (1) emph{Scoring}-predicting antibody-antigen binding affinity, (2) emph{Ranking}-ordering antibodies by affinity or thermostability, and (3) emph{Screening}-identifying high-affinity binders from large antibody libraries. We convert experimental datasets into instruction-tuning format and evaluate multiple model variants, including zero-shot LLMs, instruction-finetuned LLMs, and multimodal models enhanced with antibody-specific embeddings. Ab-VS-Bench provides a unified framework to benchmark LLMs for antibody discovery, aiming to catalyze progress in language-guided biotherapeutic design.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.26.666985v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

