<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Saturday, 06 September 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genetic analysis of iris pigmentation in Swiss pig breeds identifies a missense KITLG variant as a potential causal factor for pale and heterochromatic irises</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gorssen, W., Kadri, N. K., Khayatzadeh, N., Leonard, A. S., He, Q., Mehrotra, A., Neuenschwander, S., Pausch, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Iris pigmentation is a heritable trait with a complex genetic architecture. While the genetic basis of iris pigmentation has been extensively studied in humans, little is known about iris pigmentation in pigs. Iris pigmentation in pigs varies from different shades of brown or pale irises to heterochromia manifesting either as different colors between both irises (heterochromia iridum) or multiple colors within a single iris (heterochromia iridis). This study investigates the genetics of iris pigmentation variability in the Swiss Landrace and Swiss Large White pig breeds. Iris pigmentation was phenotyped in 837 Swiss Landrace and 328 Swiss Large White pigs of which the majority also had array-derived genotypes. A high prevalence of heterochromia iridum (18.6%) was observed in the Swiss Landrace breed. Heritability estimates for iris pigmentation were high in both breeds (heritability = 56.0 to 57.2%). Iris pigmentation was not genetically correlated with production traits. Genome-wide association analysis identified several loci associated with iris pigmentation, including regions near functional candidate genes such as TYR, ALX4 and DCT. The strongest association was detected near the KITLG gene, which was identified as a candidate gene for iris pigmentation in a previous study on Italian Large White pigs. Fine-mapping identified a highly significantly associated (P=3.8x10-11) missense variant in KITLG (5 94084790 G>A, rs342599807, p.R124K) as a potential causal variant for pale and heterochromatic iris pigmentation in Swiss pigs. Our findings provide new insights into the genetic architecture of iris pigmentation in pigs and indicate that KITLG plays a key role. The identification of a putative causal missense variant offers a foundation for further functional studies aiming to better understand pigmentation traits in pigs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.02.673690v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Adipose tissue-derived fibroblasts engage in immune-stromal crosstalk during obesity-aggravated atherosclerosis in mice</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mikkola, L., Boluk, A., Piipponen, M., Mikocziova, I., Valkonen, M., Fagersund, J., Hakovirta, H., Saraste, A., Hernandez de Sande, A., Palani, S., Ord, T., Roivainen, A., Ruusuvuori, P., Heinaniemi, M., Kaikkonen, M. U., Lonnberg, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;">Background Atherosclerosis is a common disorder where changes in vascular wall and in the surrounding perivascular adipose tissue are evident. However, how multiple cell types contribute to the disease at the cellular level remains poorly understood. Moreover, obesity can exacerbate atherogenesis, but it remains to be revealed which cell types drive the aggravation. Our objectives were to define the key cell populations across tissues in a highly atherogenic mouse model of atherosclerosis under obese and normal-weight conditions, and to explore tissue-specific cellular changes and interactions associated with obesity. Methods We employed 5-prime single-cell RNA sequencing combined with antibody staining in Ldlr-/- Apob100/100 male mice, comparing two groups: animals fed a high-fat diet to induce obesity and those maintained on regular chow as non-obese controls. We harvested aorta, epididymal and perivascular adipose tissues, and spleen from each animal. Aorta samples were CD45+ enriched, and a panel of 138 antibodies was used to generate CITE-seq libraries. Spatial transcriptomics of human carotid arteries and aorta were used to confirm the expression of top targets from the mouse model. Immunohistochemistry and multiplexed immunofluorescence were used to validate the findings. Results We generated data from almost 46000 cells enabling analyses of cell phenotypes, gene set enrichment, regulon activity, and cell interactions. We found fibroblast subpopulations derived from the adipose tissues to be involved in local immune responses and potential contributors to atherosclerosis in obesity. Pi16+ progenitor fibroblasts were significantly reduced and PVAT morphology was remarkably changed in obesity. We also observed clear spatial patterning of the top differentially expressed genes from our mouse model in human aorta and carotid artery. Conclusions We present a comprehensive multi-tissue single-cell analysis of the immune-stromal landscape in a murine model of atherosclerosis in obesity and normal state. Fibroblasts show distinct immune activation especially in perivascular adipose tissue.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.02.673686v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Tech Note: Simplified protocol for SMARTer Pico kit</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Contreras-Lopez, O., Masvidal Sanz, L., Wang, J., Einarsdottir, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The SMARTer(R)Stranded Total RNA-Seq Kit v2 - Pico Input Mammalian kit from Takara(R)(SMARTer Pico) has proved successful and reliable in generating stranded RNA Illumina libraries from degraded total RNA and ultra-low input amounts of total RNA below detection level. Here we attempted to streamline and simplify the library prep protocol at the key fragmentation step bottleneck. Our key findings were that reduced fragmentation times neither affect the depletion efficiency, nor the library complexity. Skipping the fragmentation resulted in longer libraries when examined in the capillary electrophoresis but this was compensated for during sequencing, as long fragments are less likely to form clusters during sequencing. Skipping the fragmentation also affected the gene body coverage, where a bias to the 5' end was observed though this compromised neither the data quality, complexity nor reproducibility. Additionally, using 16 PCR cycles seems to have little effect on the library complexity. Overall, we can see that sample input is the key to library complexity and reproducibility, while fragmentation time has less impact on data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.02.673416v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Systematic identification of oscillatory gene expression in single cell types</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Weinreb, A., Basavaraju, M., Hammarlund, M., Heiman, M. G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Many biological cycles are driven by oscillatory gene expression coordinated across cell types. For example, larval development in Caenorhabditis elegans involves coordinated cyclic changes in cell division, behavior, and growth, the latter requiring production of a structured extracellular matrix called the cuticle. Here, we combine single-cell RNA sequencing and novel computational approaches to identify oscillatory gene expression in individual cell types. We find that many cell types exhibit looping structures in PCA and UMAP space that correspond to transcriptional oscillations at each larval stage. Oscillatory gene expression is found in all cuticle-producing cell types, including glia, but not detected in neurons or muscle. We develop rigorous statistical approaches for de novo identification of oscillatory genes and cell types, yielding >5,000 genes. While many oscillatory genes relate to cuticle production, each cell type expresses largely distinct genes, suggesting that cuticle production is a patchwork of cell-type-specific programs. Finally, we derive a potential set of regulatory transcription factors that can explain coordinated oscillatory gene expression and find that shared upstream factors likely control gene timing across cell types. Together, our results suggest that shared regulators control cell-type-specific oscillatory gene expression, including in previously overlooked cell types such as glia.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.02.673125v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">TSdb: A Curated Database for Terpene Synthases and Their Application in Natural Product Mining</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gu, T., Ming, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Terpenoids constitute nature's most chemically diverse metabolite family with vital pharmaceutical and industrial applications, yet existing databases lack systematic integration of precursor metabolic enzymes (HMGR, DXS) and mechanistic insights into terpene diversification. To bridge this gap, we developed the Terpene Synthase Database (TSDB), distinguishing itself through three key innovations: (1) comprehensive integration of MVA/MEP pathway enzymes with downstream terpenoid synthases, (2) enhanced functional annotation via InterProScan domain mapping and phylogenetics to decode catalytic plasticity, and (3) unprecedented taxonomic breadth spanning 456,142 non-redundant sequences across 30,491 taxa. By consolidating data from BRENDA, UniProt, TeroKit, and ocean gene clusters through rigorous BLASTp deduplication (95% identity cutoff), TSDB reveals 3,499 Gene Ontology terms highlighting core functions like isoprenoid biosynthesis (GO:0019288) and metalloenzyme catalysis. Validation against MIBiG gene clusters (e.g., BGC0001324) demonstrates precise identification of terpene cyclases, P450 monooxygenases, and prenyltransferases with residue-level active site annotations. As the first resource connecting precursor metabolism to structural diversity, TSDB enables accurate gene-enzyme-product prediction for enzyme engineering and natural product discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.01.673588v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Reference-based variant detection with varseek</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rich, J. M., Luebbert, L., Sullivan, D. K., Rosa, R., Pachter, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Variant detection from sequencing data is fundamental for genomics and is the first step in a wide range of applications, ranging from genome-wide association studies to disease diagnosis. Widely used tools for variant detection utilize a de novo approach that is based on a combination of read mapping algorithms and statistical methods for identifying genetic variation from error-prone sequencing data. This approach has been successful, although the detection of insertion and deletion variants, as well as the detection of variants from low-coverage data, remain challenging problems. We introduce varseek, a reference-based approach to variant detection that provides large improvements in performance in these challenging cases. The varseek approach utilizes a k-mer pseudoalignment approach, which provides the ability to identify variants at single-cell resolution in single-cell transcriptomics data. We showcase the versatility and performance of varseek for detecting tumor-specific COSMIC variants in glioblastoma single-cell sequencing.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.03.674039v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Merits and challenges of plasma proteomics on association replicability</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jiao, Z., Zhang, Y., Lai, Y., Kang, J., Ma, L., Zhao, W., You, J., Cheng, W., Feng, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Developments in proteomic platforms have enabled the generation of large-scale high-throughput plasma proteomics data [1-3]. With recent breakthroughs in AI modelling, these data have significantly enhanced our understanding of molecular mechanisms underlying human behaviors and diseases [4-6]. However, the replicability of associations between plasma proteomics and phenotypes remains underexplored. Here, we systematically assessed the replicability of associations with recent plasma proteomics data in the UK biobank. Over 75% of cognitive function and mental health traits demonstrated high overall (proteomics-wide) replicability when brain-related traits were considered as phenotypes. Although mean cortical thickness (CT) as phenotype exhibited clearly reduced replicability, total cortical surface area (CSA) and cortical volume (CV) showed high overall replicability across hemispheres and over twenty brain regions. In comparative multi-omics analyses based on the same cohort of participants, proteomics outperformed genomics across all brain-related traits, and exceeded metabolomics for over half of traits where metabolomics also exhibited high overall replicability. Furthermore, we developed a predictive framework to estimate the replicability for potential future proteomics panels based on the crucial influential factors including dilution level, proportion of samples below the limit of detection (LOD), and sample size. Moreover, we constructed an individual replicability index for proteins and identified eleven proteins with highly replicable associations across cognitive function and mental health traits, which was in line with the recent identifications of pleiotropic proteins in large-scale population studies. Collectively, our results revealed the challenges in the association replicability of plasma proteomics under reduced data quality (from "Explore" to "Expansion" assay panels), and we further explored how to sustain high replicability in potential future panels. Fundamentally, our findings affirm the merits of plasma proteomics: this molecular omics platform enables highly replicable associations for mapping biomedical signatures.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.01.673490v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Mechanistic dissection of GRHL2 and PR transcriptional co-regulation in breast cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Aarts, M. T., Nordin, A., Cantu, C., van Boxtel, A. L., van Amerongen, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Gene expression is controlled by complex transcriptional networks in which transcription factors and their cognate enhancer elements integrate developmental and environmental cues. The progesterone receptor (PR), a hormone-activated transcription factor, is essential for breast development and physiology, yet how it engages with the chromatin and lineage-specific cofactors remains unclear. Using an unbiased approach, we identify the epithelial transcription factor grainyhead-like 2 (GRHL2) as a key co-regulator of PR activity in hormone responsive breast cancer cells. We show that GRHL2 interacts with PR in a progesterone-independent manner. Upon progesterone stimulation, GRHL2 and PR are both recruited to distal enhancer elements of target genes. Furthermore, GRHL2- and PR-bound elements connect spatially through chromatin looping to regulate shared targets. These findings uncover a previously unrecognized mechanism by which GRHL2 and PR coordinate gene regulation through both chromatin binding and 3D genome architecture modification, positioning GRHL2 as a crucial modulator of steroid hormone receptor function.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.01.673508v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">LieOTDock: A Protein Docking Framework via Lie Group Optimization and Optimal Transport</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hu, Y., Cao, Z., Liu, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein-protein interactions (PPIs) are fundamental to cellular function, and structurally characterizing them is a key goal in molecular biology. Computational docking, a crucial tool for predicting the structure of protein complexes, faces significant challenges in search efficiency and scoring accuracy. Here, we introduce LieOTDock, a novel framework that leverages the mathematics of Lie groups and Optimal Transport (OT) for highly efficient protein docking pose generation. The core of our method is a data-driven surface sampling technique that identifies salient geometric features for initial pairing. These initial poses are then refined via a gradient-based optimization directly on the Lie group SE(3), the natural space of rigid-body motions. This optimization is guided by a differentiable score based on Sinkhorn-regularized Optimal Transport, which measures the geometric complementarity of surface patches. We tested LieOTDock on the 1AVW benchmark, where it successfully generated a near-native pose of 3.43 Angstrom RMSD from a 5000-candidate run in under three minutes on a consumer-grade Apple M4 Mini. Our results demonstrate that LieOTDock, by its principled use of Lie theory and OT, provides a powerful and efficient framework for generating high-quality candidate structures. This work decouples the challenge of geometric sampling from final scoring, offering a robust and modular approach to the protein docking problem, though future work is needed to integrate more physically realistic scoring functions for improved candidate ranking.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.04.674197v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Production Status, Breeding Priorities and Genetic Resources of Cowpea in Post-Civil War Somalia</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Warsame, A. O., Isse, Y. A., Sh. Abdi, A. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Over the past three decades, Somalia has faced recurrent civil conflicts and prolonged droughts, leading to widespread displacement of farming communities. These disruptions have weakened traditional seed systems and potentially damaged local crop diversity. In this context, to strengthen the country's food security and climate change adaptation of smallholder farmers, there is an urgent need to revive agricultural research and breeding programs for the main food crops grown in the country. Cowpea (Vigna unguiculata (L.) Walp.) is the most important food crop after maize and sorghum in Somalia, providing inexpensive protein for humans and biologically fixed nitrogen for low-input crop production systems in the country. This study assessed the current status of cowpea production, identified breeding priorities across three agroecological zones in southern Somalia, and established a core germplasm collection for future breeding programs. Farmer surveys were conducted in Baidoa, Afgoie, and Jowhar, involving interviews with 150 farmers using a semi-structured questionnaire. The results showed that over 60% of farmers cultivate cowpea for both household consumption and market purposes. While the crop is predominantly grown as an intercrop, nearly 45% of farmers in Afgoie grow it as a sole crop. Marked regional differences were observed in varietal preferences, with farmers in Afgoie favouring erect, uniformly maturing cultivars with a dark-red seed colour, whereas other regions demonstrated more diverse preferences. Production constraints also varied by location, with drought identified as the primary limitation in Baidoa, and pests and diseases as the major challenges in Afgoie and Jowhar. Accordingly, farmers ranked early maturity as the top breeding priority in Baidoa, whereas disease and pest resistance were the primary targets in Afgoie. Furthermore, we established a germplasm collection representing Somalia's cowpea diversity from both the pre- and post-war periods. The collection also includes a subset of global cowpea diversity and accessions with known desirable agronomic traits. These genetic resources, together with farmer survey data, provide a foundation for targeted breeding efforts aimed at improving cowpea productivity and enhancing food security in Somalia in the future.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.02.673644v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Qualifying the effectiveness of Cannabis smoke filtration via Bong water using GC-MS and risk assessment of inhaling smoke compounds</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kiatchaipipat, S., Vimolmangkang, S., Jinta, P., Khasitanon, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cannabis is widely consumed worldwide, yet its health impacts remain contested due to the presence of both potentially harmful carcinogens and bioactive compounds with therapeutic promise. This study investigated the chemical composition of cannabis smoke generated from three strains when consumed via joints and bongs, with a focus on whether bong water effectively filters harmful compounds. Smoke samples were analyzed using gas chromatography mass spectrometry (GCMS). Across all samples, the smoke composition between bong and joint methods was largely consistent, with no compounds between 5 and 350 g/mol found to be completely removed by bong water. Several compounds were detected in all or nearly all samples, including {beta}-cis-Caryophyllene, phenol, p-cresol, and naphthalene, representing substances with potentially beneficial, harmful, unclear, or benign effects. Notably, {beta}-cis-Caryophyllene was consistently present in the highest quantities, suggesting possible physiological importance despite limited research compared to THC and CBD. Methodological limitations, including sample size, smoke loss during collection, and GCMS inability to detect larger particles, aerosols, and metal ions, restricted definitive conclusions regarding bong filtration effectiveness. However, the findings highlight that cannabis smoke contains a reproducible profile of compounds, both harmful and potentially beneficial. Establishing standardized analytical approaches could support more accurate assessments of cannabis quality, health risks, and therapeutic potential, while enabling comparisons across strains, cultivation methods, and global research efforts.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.01.673549v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Conserved transcriptional reprogramming in nematode infected root cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Saura-Sanchez, M., Gomez Rojas, A., Deveux, M., Minne, M., Grones, C., Eekhout, T., Abril-Urias, P., Van Bel, M., Tenorio Berrio, R., Vandepoele, K., Escobar, C., Beeckman, T., De Rybel, B., Kyndt, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Plant-parasitic nematodes are responsible for important annual losses in crop productivity worldwide. Although the formation of feeding organs within the roots is essential for successful sedentary parasitism, the molecular mechanisms underlying their development are poorly understood. This is partly because these organs originate from a limited number of root cells, making difficult to capture the transcriptional reprogramming that occurs during the early stages of the infection. Here, we first developed a comparative host-pathogen framework to study the nematode infection process in Arabidopsis and rice. Using a cross-species single-cell transcriptomics approach, we identified a unique molecular signature in infected root cells and show that the cellular reprogramming during these early stages is highly conserved across both host-pathogen interactions. This transcriptional cell reprogramming is associated with stemness acquisition related to de novo organogenesis process. By cell-type specific gene regulatory network analysis, we identified AtATHB2/OsHOX28 as an evolutionary conserved and key regulator of the nematode infection process. Loss-of-function of this regulator in both species results in nematode resistance without affecting root growth. This discovery opens up new avenues for the development of sustainable nematode control strategies that could be translated across crop species.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.05.674408v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Rapid assay optimization by plug-and-play MAPPIT</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Maes, S., Delhaye, L., Cooreman, R., Soetens, E., Verhee, A., De Sutter, D., Van de Steene, T., Nguyen, L., Audenaert, D., Saelens, X., Eyckerman, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Studying protein-protein interactions, which are crucial for all cellular processes in both health and disease, offers valuable insights into underlying mechanisms and reveals promising therapeutic targets. While efforts have been made to vastly increase the scale of MAPPIT assays, covering the full protein interactome remains challenging, in particular when considering the availability of different assay configurations. In this study, we expanded our Golden Gate modular cloning platform to enable rapid, customizable construct assembly for MAPPIT assays. We demonstrate the strength of this expanded toolbox for multiple distinct protein-protein interactions, including a MAPPIT assay for six different intracellular VHH targeting the anti-viral effector protein MXA.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.05.674518v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Streptomyces Autoregulator Biosensors from Natural Product Cluster-Situated Regulators</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wilbanks, L. E., Roberts, C. B., Frias-Gomez, M., Hennigan, H. E., Castator, K. G., Budimir, Z. L., Zu, C., Parkinson, E. I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The soil dwelling bacteria Streptomyces is an abundant producer of numerous anticancer, antifungal, and antibiotic compounds (i.e. Natural Products, NPs). The sophisticated cellular machinery required to produce NPs is frequently regulated by quorum-sensing systems, consisting of cluster situated regulators (CSRs), such as TetR-like repressors, and small-molecule autoregulator (AR) ligands. Only a small fraction of bioinformatically predicted quorum-sensing AR circuits have been experimentally determined, and fewer still have been engineered as inducible expression systems for synthetic biology. This research details the development of eight CSR-based AR biosensors and the synthetic routes to their AR ligands. Overall, the AR biosensors exhibit a range of maximum activation, AR affinity, and AR selectivity. We examined crosstalk between noncognate CSRs and ARs, as well as the ability of CSRs to regulate alternative operators. Additionally, we establish these biosensors can be cocultured with Streptomyces for rapid analysis of AR production. Finally, we demonstrate the CSR-based biosensor vectors can be combined to create orthogonal signaling systems in bacterial coculturing or multi-input genetic circuits. Longterm, these Streptomyces AR biosensors will contribute to the elucidation of small molecule quorum sensing circuits employed by Streptomyces as well as increasing the complexity of genetic circuits used in industrial or agricultural settings.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.05.673737v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Uncovering directionally and temporally variable genes with STAVAG</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shen, Q., Gai, K., Li, S., Yang, F., Cui, H., Zhang, S., Zhang, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Spatially variable genes (SVGs) are crucial for understanding spatial heterogeneity in spatial transcriptomics. Yet, SVGs are challenging to align with established biological axes and lack straightforward mechanistic interpretation, thereby limiting their ability to inform downstream experiments or clinical applications. Here, we introduce the concept of directionally variable genes (DVGs) and temporally variable genes (TVGs). Also, we propose a unified framework, STAVAG, that models spatial or temporal information to identify DVGs and TVGs. STAVAG effectively identifies biologically meaningful DVGs for uncovering prognostically relevant gene modules that align with the malignant-immune-stroma structure in the tumor microenvironment, identifying marker genes in three-dimensional brain data, and detecting 12 polarization genes regulating planarian regeneration. Furthermore, by tracking the progression of myocardial infarction and tissue development, STAVAG reveals critical TVGs that can serve as diagnostic biomarkers for the ultra-acute phase of myocardial infarction. STAVAG also identifies both global and local gene modules involved in mouse placentation, as well as tissue-specific and interactive TVG modules that dynamically evolve during mouse embryonic development.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.02.673732v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">UALCAN Mobile, an app for cancer proteogenomic data analysis.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Varambally, S., Rubey, D., Shimoga Chandrashekar, D., Shovon, A. R., Puli, G. C., Karthikeyan, S. K., Manne, U., Creighton, C. J., Kumar, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cancer is a complex disease affecting various organs and is a major cause of death worldwide. During cancer initiation, disease progression, and tumor metastasis, various genomic and proteomic alterations are observed. Recent technological advances have led to the generation of large amounts of molecular data, including genomics and transcriptomics. These large-scale datasets can be utilized to analyze and identify sub-class-specific cancer biomarkers and targets. However, there is a need for the development of user-friendly tools for large-scale data analysis, disseminating the analyzed data in a visualizable format to cancer researchers with no programming skills. We developed UALCAN, a comprehensive platform that allows users to integrate disparate data to better understand the genes, proteins, and pathways perturbed in cancer and make discoveries of potential biomarkers and targets. In the current study, we describe the development of the UALCAN Mobile application (app) that will provide cancer transcriptomic data obtained from The Cancer Genome Atlas (TCGA) project to evaluate protein-coding gene expression based on various stratifications, including stage, grade, race, gender, and molecular-subtypes across over 30 types of cancers. In addition, the UALCAN mobile provides data analysis options for epigenetic changes due to DNA promoter methylation and Clinical Proteomic Tumor Analysis Consortium (CPTAC) cancer proteomic data. The app provides access to large cancer molecular datasets on the go. To find changes in the expression of causative genes and proteins and to identify biomarkers and therapeutic targets, UALCAN mobile app will be extremely valuable. The UALCAN Mobile app is free to use and can be downloaded from both the iOS/Apple and the Android Play Store and has been downloaded over 100 times in each of iOS and android app stores.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.04.674198v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DART: Deep learning for the Analysis and Reconstruction of Transcriptional dynamics from live-cell imaging data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ma, M., Grima, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Transcriptional bursting, characterized by stochastic switching between promoter states, underlies cell-to-cell variability in gene expression. Accurately inferring promoter activity from live-cell imaging data remains challenging because the fluorescence signal at any given point is influenced by the history of promoter states. Here, we present DART (Deep learning for the Analysis and Reconstruction of Transcriptional dynamics), a deep learning framework that infers promoter on- and off-states from fluorescence intensity traces, enabling the estimation of activation and inactivation rates and the selection of the most appropriate promoter-switching model. DART utilizes a neural network architecture that combines convolutional neural networks and long short-term memory layers to binarize fluorescence traces. Using extensive synthetic datasets spanning a wide range of transcriptional bursting levels, we demonstrate that DART outperforms current binarization methods, including conventional and augmented hidden Markov models, in both accuracy and robustness. Furthermore, a reanalysis of published experimental data using DART reveals a strong linear coupling between activation and inactivation rates, contradicting previous claims of independence. Our approach provides a powerful and generalizable tool for quantitative analysis of transcriptional kinetics from live-cell imaging data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.02.673499v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Selective mRNA translation determines adaptative mutability of melanoma cells to anti-BRAF/MEK combination therapy</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fabbri, L., Lagadec, L., Guerin, E., Lecourt, H., Baille, D., Besse, L., Messaoudi, C., Desaubry, L., Abou-Hamdan, H., Roy, S., Lombard, B., Loew, D., Scoazec, J.-Y., Robert, C., Vagner, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> During their inevitable evolution towards acquired resistance to anti-cancer targeted therapies, cancer cells adopt distinct gene expression profiles that allow them to transiently adapt to and tolerate the treatment. Similar to bacterial cells that transiently tolerate antibiotics, cancer cells surviving therapy can increase their mutation rate, enhancing the likelihood of acquiring resistance-conferring mutations and evolving into resistant cells. This adaptive mutability has been linked to transcriptional reprogramming of DNA damage repair mechanisms and effective therapeutic strategies to target such mechanisms are currently lacking. Here we show that translational control mediates the adaptive mutability of melanoma drug-tolerant cells by regulating the translation of the error-prone non-homologous end joining (NHEJ) component 53BP1. The specific inhibition of 5 prime-UTR-driven 53BP1 mRNA translation was sufficient to impair NHEJ and mutability. We found that the eIF4A RNA helicase, a key component of the eIF4F translation complex, regulates 53BP1 mRNA translation. Consequently, targeting the eIF4A with two small molecule inhibitors significantly delays the acquisition of resistance to combination of BRAF and MEK inhibitors (BRAFi/MEKi) in BRAFV600-mutant melanoma xenograft models and cell lines by reducing the mutability of drug-tolerant cells. Our results demonstrate that a standard-of-care therapy for melanoma, by engaging non-genetic adaptation driven at the translational level, contributes to the evolution of drug-tolerant melanoma cells toward acquired resistance.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.02.673634v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Stacked Waves and Qualitative Analysis of an Oncolytic Virus-Immune Model</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mohammadnejad, N., Hillen, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Oncolytic virotherapy (OVT) represents an innovative and promising therapeutic method for cancer treatment. This approach involves the introduction of oncolytic viruses into the patient, which are engineered to selectively target and lyse tumor cells. Based on previous mathematical modelling of oncolytic viruses, we consider a mathematical model that describes the intricate interactions between the oncolytic virus, cancer cell populations, and the immune system. Our study includes a detailed qualitative and quantitative analysis of the model to explain why, despite their promise, oncolytic viruses alone rarely lead to complete and lasting regression of established tumors in vivo. We use parameter sensitivity analysis to support our findings. Furthermore, we consider the spatial version of the model in the form of a system of reaction-diffusion equations. A travelling wave analysis shows an unexpected phenomenon. The solution components do not necessarily evolve into a single traveling front; rather, they develop into stacked fronts, where each front propagates at a different speed. We give explicit formulas for these different invasion speeds, confirm those through numerical simulations, and discuss their significance for OVT.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.04.674336v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">HIF-1Œ± and RhoA Drive Enhanced Motility and Aerotaxis of Polyaneuploid Prostate Cancer Cells in Hypoxia</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hosny, N., Li, S., Amend, S. R., Gatenby, R., Pienta, K., Brown, J., Qu, J., Austin, R. H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Most cancer deaths result from metastasis, yet only a rare subset of tumor cells can complete this process. Among these, polyaneuploid cancer cells (PACCs), which arise via endoreplication under stressors such as hypoxia, are implicated as metastatic drivers, but how they acquire this potential is poorly understood. Here, we show that prostate cancer-derived PACCs exhibit features predictive of invasion and intravasation. Time-lapse fluorescence microscopy and single-cell tracking under hypoxia revealed that PACCs migrated significantly farther than Non-PACCs, consistent with local invasion. PACC trajectories showed a strong tendency to migrate toward oxygen, consistent with aerotaxis and predictive of intravasation. siRNA-mediated knockdown demonstrated that the enhanced motility and aerotaxis of hypoxic PACCs require both HIF-1 and RhoA, with RhoA expression suppressed upon HIF-1 inhibition. Altogether, we propose a HIF-1 [->] RhoA [->] motility/aerotaxis mechanism enabling PACCs to escape hypoxic cores, invade tissue, and access vasculature, highlighting them as a uniquely invasive subpopulation with implications for anti-metastatic therapies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.03.673887v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">OSDR2.0 infers microenvironment-driven cell-state transitions and population dynamics from a single spatial biopsy</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ben Shalom, I., Somer, J., Miyara, S., Mayo, A., Mannor, S., Alon, U.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cell populations in human tissues change over time by cell division, death and transitions between functional states. In the tumor microenvironment (TME), such dynamics are central to immune evasion, stromal remodeling and therapeutic response. However, it is difficult to measure such dynamics in vivo because usually only a single biopsy is available providing a static snapshot. To obtain cell population dynamics from a snapshot we previously developed One Shot Dynamic Reconstruction (OSDR1.0), an algorithm that reconstructs cell population dynamics over weeks to months from a spatial biopsy using cell-type information and a proliferation marker. OSDR1.0 however does not include transitions between cell states. Here we present OSDR2.0, an extension that incorporates transitions between cell states inferred from the local cellular neighborhood. The algorithm OSDR2.0 models the probability of a cell being in a given state (e.g., PD1+ vs. PD1- T cell, or cancer-associated vs. resting fibroblast) as a function of its surrounding cell types. These state-transition rules are then integrated into simulations of population dynamics. After the cell population is advanced by a timestep, the cell states are adjusted according to the new neighborhoods, using the fact that cell state transitions, which take hours, are typically much more rapid than cell population changes on the scale of weeks, and can thus be considered at quasi-steady-state. Applying OSDR2.0 to spatial proteomics data from triple-negative breast cancer (TNBC), we find that cell state is strongly associated with local microenvironment composition. Incorporating state transitions significantly improves the ability to predict treatment response - OSDR2.0 accurately separates responders from nonresponders to chemotherapy and immunotherapy based on early post-treatment biopsies, outperforming OSDR1.0 that does not include cell state transitions. This work highlights the importance of cell state plasticity in shaping tumor response and opens a way to infer both population dynamics and cell state transitions from static clinical samples.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.02.673328v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Pancreatic Cancer Induces Population-Specific Switching of Myosin Isoforms and Discrete Activation of Cachexia Genes in Skeletal Muscle Myocytes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Counts, B. R., Jean, S., Gaafer, O., Ota, S. K., Inabathini, I., Guo, T., Guttridge, D. C., Ostrowski, M. C., Koniaris, L. G., Cao, S., Roh, H. C., Zimmers, T. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Skeletal muscle loss in pancreatic cancer is a significant cause of morbidity and mortality for patients. In order to understand myocytes changes we examined myonuclei- and myofiber-specific dynamics during pancreatic cancer cachexia progression. Single-nucleus RNA-seq was used to interrogate myonuclear gene expression, and RNAscope and immunofluorescence characterized myofiber-specific changes. Bulk RNA-seq of skeletal muscle provided a whole-muscle transcriptomic profile. Cachexia induces a progressive loss of muscle differentiation factor Maf and its target Myh4, accompanied by increased expression of Myh1 and Myh2. This myofiber dedifferentiation occurs without evidence for fiber type shifting, regeneration, or proliferation. Single-nuclei analysis reveals global shifts in myofiber gene expression identity including the identification of a cachexia only myonuclear subpopulation. Cachexia gene expression was not restricted solely to this PDAC-specific myonuclear subpopulation and did not overlap with Myh1 and Myh2 expressing myonuclei early in cachexia. Altogether, PDAC cachexia elicits distinct transcriptional responses across different myonuclear populations. These results reveal population-specific heterogeneity in cachexia gene activation, rather than a uniform upregulation of cachexia mediators across muscle tissue. Our data suggest that myonuclei fate occurs prior to overt muscle wasting when cachexia gene expression only modestly overlaps with differentiation factors, with a strong association after irreversible muscle wasting. These findings explain the challenge of effectively targeting skeletal muscle wasting in cancer cachexia requires addressing the changing cell population induced through non overlapping mechanisms.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.01.673500v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Syntenin orchestrates matrix degradation by controlling MT1-MMP secretion in small extracellular vesicles and invadopodia formation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> HUBER, M., GHOSSOUB, R., Leblanc, R., David, G., Zimmermann, P., THUAULT, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Extracellular matrix (ECM) remodelling is essential for tumour progression. The metalloproteinase MT1-MMP promotes cell invasion by degrading ECM components. MT1-MMP accumulates at invadopodia, actin-rich structures mediating ECM degradation, and is also present on the surface of small extracellular vesicles (sEV), key players in cell-to-cell communication. The role of sEV-associated MT1-MMP in ECM degradation and the mechanisms supporting MT1-MMP loading into sEV are unknown. We previously established that the syntenin PDZ protein, in association with the syndecan (SDC), is essential for sEV from endosomal origin biogenesis. In the present study, we demonstrate that syntenin contributes to ECM degradation via invadopodia in MDA-MB-231 triple negative breast cancer (TNBC) cells, whereas SDC limit it. Moreover, we identify that syntenin colocalizes with MT1-MMP in late endosomes and directly interacts with MT1-MMP. Furthermore, we decipher that both syntenin and SDC are essential for the specific loading of MT1-MMP into sEV. Additionally, we show that sEV-mediated ECM degradation depends on MT1-MMP activity and syntenin. Our findings show how syntenin-SDC-MT1-MMP complexes orchestrate ECM degradation and pave the way for innovative rational approaches to control TNBC invasion.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.01.673493v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Modifying inhibitor specificity for homologous enzymes by machine learning.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dor S Gozlan, Reut Meiri, Gili Shapira, Matt Coban, Evette S Radisky, Yaron Orenstein, Niv Papo</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Selective inhibitors are essential for targeted therapeutics and for probing enzyme functions in various biological systems. The two main challenges in identifying such protein-based inhibitors lie in the extensive experimental effort required, including the generation of large libraries, and in tailoring the selectivity of inhibitors to enzymes with homologous structures. To address these challenges, machine learning (ML) is being used to improve protein design by training on targeted libraries and identifying key interface mutations that enhance affinity and specificity. However, such ML-based methods are limited by inaccurate energy calculations and difficulties in predicting the structural impacts of multiple mutations. Here, we present an ML-based method that leverages HTS data to streamline the design of selective protease inhibitors. To demonstrate its utility, we applied our new method to find inhibitors of matrix metalloproteinases (MMPs), a family of homologous proteases involved in both physiological and pathological processes. By training ML models on binding data for three MMPs (MMP-1, MMP-3, and MMP-9), we successfully designed a novel N-TIMP2 variant with a differential specificity profile, namely, high affinity for MMP-9, moderate affinity for MMP-3, and low affinity for MMP-1. Our experimental validation showed that this novel variant exhibited a significant specificity shift and enhanced selectivity compared with wild-type N-TIMP2. Through molecular modeling and energy minimization, we obtained structural insights into the variant's enhanced selectivity. Our findings highlight the power of ML-based methods to reduce experimental workloads, facilitate the rational design of selective inhibitors, and advance the understanding of specific inhibitor-enzyme interactions in homologous enzyme systems.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40912922/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Backyard Proteomics: A Case Study with the Black Widow Spider.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tarsh Shah, Jackson A Fitzpatrick, Benjamin C Orsburn</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Nearly all methods of mass-spectrometry-based proteomics rely on knowing the proteome of the species. In less studied organisms without annotated genomes, it can seem impossible to perform proteomic analysis. In this study, we sought to answer the question: does enough information exist to do proteomics on any organism we want? As a case study, we started with material available due to an infestation of a home with black widow spiders. Thanks to the recent publication of an annotated genome for one species of black widow spider, we were able to identify 5502 protein groups and assign putative annotations using ortholog mapping. We also demonstrate that had we not had this resource, over 2000 proteins could be identified using other available spider genome annotations, despite their unrelatedness. Moreover, regardless of the spider proteome used, proteins annotated as toxins were almost exclusively observed in the main body of the mature female black widow spider. Overall, these results provide a draft proteome map for the black widow spider and valuable data for validating machine learning models while also suggesting that the door to insightful quantitative proteomics may already be open for millions of less studied organisms. All raw and processed proteomic data are available through the ProteomeXchange repository as accession PXD051601.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40721992/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

