<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Thursday, 04 September 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">GWAS SVatalog: a visualization tool to aid fine-mapping of GWAS loci with structural variations</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chirmade, S., Wang, Z., Mastromatteo, S., Sanders, E., Thiruvahindrapuram, B., Nalpathamkalam, T., Pellecchia, G., Lin, F., Keenan, K., Patel, R. V., Sung, W. W., Roshandel, D., Whitney, J., Allana, S., Avolio, J., Eckford, P., Ratjen, F., Strug, L. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;">Background: Genome-wide association studies (GWAS) have been successful in identifying single nucleotide polymorphisms (SNPs) associated with phenotypic traits. However, SNPs form an incomplete set of variation across the genome and since a large percentage of GWAS-significant SNPs lie in non-coding regions, their impact on a given trait is difficult to decipher. Recognizing whether these SNPs are tagging other polymorphisms, like structural variations (SV), is an important step towards understanding the putative causal variation at GWAS loci. Results: Here, we develop GWAS SVatalog (https://svatalog.research.sickkids.ca/), a novel open-source web tool that computes and visualizes linkage disequilibrium (LD) between SVs and GWAS-associated SNPs throughout the human genome. The tool combines GWAS Catalog's SNP-trait association data across 14,479 phenotypes with LD statistics calculated between 35,732 SVs and 116,870 SNPs identified in 101 whole-genome long-read sequences. We use GWAS SVatalog to identify SVs that may explain GWAS loci for iron levels, refractive error, and Alzheimer's disease, where previously SNPs were unable to provide a causal explanation. Conclusions: GWAS SVatalog advances the fine-mapping of GWAS loci with structural variations, enabling researchers to associate 35,732 common SVs with 14,479 phenotypes, accelerating the understanding of disease etiology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.03.674075v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Capturing the Mutational Dynamics of SARS-CoV-2 with Graphs</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Das, B., Heath, L. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The rapid evolution of SARS-CoV-2 presents significant challenges for modeling viral dynamics, driven by lineage diversification and region-specific mutation patterns. While phylogenetic trees are traditionally used for evolutionary inference, the massive volume of SARS-CoV-2 genomic data, with many similar sequences and few distinguishing mutations, poses computational and methodological limitations. The quasispecies theory instead models viral evolution as a cloud of mutants, motivating a graph-based representation that better captures the complexity of mutational events. Geographic variation adds another critical layer to this complexity. Mutation trends often differ across regions due to local transmission dynamics, host population structures, and selective pressures. In this study, we present the Mutation Learning Graph (MLG), a directed graph framework that organizes SARS-CoV-2 variants based on their cumulative mutation profiles relative to the reference genome (NC_045512.2), thereby capturing the dynamics of mutation propagation. This structure captures fine-grained mutational transitions and encodes plausible evolutionary relationships among variants. To construct these graphs, we introduce an alignment-aware mutation profiling method and a novel ANCESTOR JOINING algorithm, which incorporates ancestral variants as inferred intermediate nodes to connect observed genomes through biologically coherent mutational paths. We generate MLG datasets for ten geographically and epidemiologically diverse regions and benchmark them on two graph-based tasks: node-level lineage classification and edge-level mutational transition prediction. Using baseline graph neural network architectures (GCN, GraphSAGE, GAT, GGNN, VGAE), we demonstrate how mutation-centric graph structures expose key biological challenges, such as lineage imbalance and location-specific mutation spectra. For node classification, GraphSAGE and GGNN consistently achieved high accuracy (up to 0.96) and AUROC (up to 0.98). In contrast, VGAE and GraphSAGE led the way in link prediction, with AUPRCs of up to 0.96. These results highlight the effectiveness of MLG for capturing biologically meaningful mutation patterns and underscore the importance of localized, mutation-aware modeling for predicting viral mutations and future variant emergence.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.02.673804v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Accurate and scalable genome-wide ancestry estimation using haplotype clusters</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Meisner, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Unsupervised genome-wide ancestry estimation has been a staple in population and medical genetics for decades, and its importance continues to grow with the increasing number of large genetic cohorts of mixed ancestries. We propose an extension to the hapla framework that scales model-based ancestry estimation to unprecedented sample sizes by leveraging inferred haplotype clusters from phased genotype data. Our haplotype cluster-based approach is approximately 5x faster than the fastest model-free SNP-based approach on the harmonized Human Genome Diversity Project and 1000 Genomes Project dataset, while we further demonstrate it to be the most accurate method to date in an extensive simulation study. Our accurate ancestry estimates can help reduce health disparities and accelerate precision medicine efforts in the growing number of biobanks globally.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.02.673718v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">User-friendly scheduler Using a hybrid architecture and supercomputing for big data processing</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> McKeever, P., Mittal, V., Fukuda, B., Yeung, K. Y., Hung, L.-H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The exponential growth of omics data requires novel strategies for storage, transfer, and processing of said data. We present a scheduler based on the Temporal.io workflow framework which enables two key optimizations of bioinformatics workflows. Firstly, we enable users to transparently map workflow steps to diverse execution environments, including high-performance computing (HPC) resources managed by the SLURM resource manager through an easy-to-use graphical user interface. Secondly, we enable asynchronous execution of workflows, a feature which guarantees that workflows will achieve reasonable resource utilization even when the scheduler cannot make use of a system's full RAM and CPU resources. Thirdly, we propose a universal, platform agnostic JSON representation of workflows that allows platform-specific execution details to be abstracted away from the core scientific logic. Our work includes a custom executor plugin that supports translation of workflows from an external language, such as Nextflow, to our universal JSON format. Finally, we develop a graphical user interface to make our scheduler easy-to-use for non-technical users. When benchmarked on a bulk RNA sequencing workflow, these features reduced the cost and time requirements. We illustrated the merits of our cross-platform method using credit allocations from federally funded supercomputers.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.01.673517v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CMDdemux: an efficient single cell demultiplexing method</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, J., Chen, L., Brown, D. V., Chiu, C., Speed, T. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Multiplexing technologies label cells with molecular tags, allowing cells from different donors to be pooled together for sequencing. Although this approach enhances cell throughput, eliminates batch effects, and enables doublet detection, limitations of hashtag-based labelling can still lead to low-quality data. Existing demultiplexing methods can accurately assign donor identities in high-quality datasets, but they often fail on low-quality data. To address this, we developed CMDdemux, a method comprising three key steps: within-cell centered log-ratio (CLR) normalization of hashtag count data, K-medoids clustering, and classification of cells based on Mahalanobis distance. By integrating both hashing and mRNA data, CMDdemux achieves high accuracy in distinguishing singlets, doublets, and negatives. It also provides visualization tools to help users inspect potentially misclassified droplets. We benchmarked CMDdemux against existing methods using a range of high- and low-quality datasets. Results show that CMDdemux consistently outperforms other approaches, demonstrating robust performance on both high- and low-quality data where other methods fail. CMDdemux is particularly effective in handling diverse types of low-quality multiplexing data across different multiplexing technologies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.28.672983v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Generation of a global freshwater algal taxonomic database by application of PCR-free rbcL gene detection and machine-learning-based taxonomic classification to public metagenome datasets</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Murdoch, R. W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> While it has become increasingly evident that microalgae are critical components of aquatic ecosystems, comprehensive taxonomic analysis of the microalgal component of microbial communities has lagged behind recent innovations in prokaryotic sequencing and bioinformatics. Microalgae are sequenced alongside prokaryotes in metagenomic whole genome shotgun (WGS) sequencing efforts but remain overlooked. In this study, an analytic pipeline for detecting a key algal taxonomic barcode, rbcL, from WGS datasets. This approach allowed for near-perfect detection of rbcL sequences from eight major algal phyla, whereas in silico PCR utilizing popular phyla-specific rbcL primer sets for four algal phyla suggests that such approaches may miss much algal diversity. Further, a machine learning classification model was built and validated to allow for taxonomic classification of rbcL sequences of eight algal phyla to genus or species level with high accuracy and recall values (>90%). Benchmarking of this approach against microscopic algal identifications showed accurate identifications for algae with higher relative abundance values and may offer additional detections of algae not readily identified by microscopy. This pipeline was applied against over 4,000 freshwater WGS metagenomes to generate a global taxonomic occurrence and relative abundance database of freshwater algae composed of over 14,000 accessions. WGS-based algal detection offers broad, unbiased taxon-agnostic algal detection in a single sample analysis. Furthermore, it allows for simultaneous identification of the prokaryotic community, offering new opportunities for studying inter-domain relationships as evidenced by a case-study presented herein. The rbcL taxonomic classifier and metagenome-derived algal detection database are available for download and use at https://doi.org/10.6084/m9.figshare.29996962.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.28.672922v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">EnrichMET: R Package for Integrated Pathway and Network Analysis for Metabolomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mekonnen, Y. A., Dhake, N., Jaiswal, S., Rubio, V., Narvaez-Bandera, I., Ackerman, H., Flore, E., Stewart, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Advances in metabolomics have significantly improved our understanding of cellular processes by enabling the identification of hundreds of metabolites in a single experiment. These developments provide valuable insights into complex metabolic networks. Pathway Enrichment Analysis (PEA) is essential for connecting identified metabolites with biological pathways, uncovering meaningful relationships, and revealing underlying metabolic mechanisms. However, existing tools, often require multiple steps, rely on web-based interfaces, and require R packages that can be challenging to install due to outdated or missing dependencies. This hampers both reproducibility and efficiency. To overcome these limitations, we introduce EnrichMet, an R package designed for efficient and reproducible pathway, metabolite set enrichment analysis (MetSEA). The package integrates fgsea for rapid MetSEA and igraph for topology-based metrics, enabling comprehensive integration of metabolomics data. EnrichMet utilizes curated pathway data from the Kyoto Encyclopedia of Genes and Genomes (KEGG) and employs Fisher Exact Test to identify significantly enriched pathways. Through a case study on experimental metabolomics datasets, we demonstrate the robustness and efficiency of EnrichMet, benchmarking it against existing tools. The results show that EnrichMet provides accurate and comprehensive pathway enrichment outcomes while reducing computational time and user effort. Moreover, its flexible framework allows EnrichMet to be adaptable to a broad range of data types and applications beyond KEGG. By streamlining the pathway analysis process, EnrichMet enables researchers to derive deeper insights from metabolomics data, promoting reproducible and efficient research practices.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.28.672951v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Hierarchical Analysis of RNA Secondary Structures with Pseudoknots Based on Sections</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Masuki, R., Liew, D., Yong, E. H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Predicting RNA structures containing pseudoknots remains computationally challenging due to high processing costs and complexity. While standard methods for pseudoknot prediction require O(N6) time complexity, we present a hierarchical approach that significantly reduces computational cost while maintaining prediction accuracy. Our method analyzes RNA structures by dividing them into contiguous regions of unpaired bases derived from known secondary structures ("sections"). We examine pseudoknot interactions between sections using a nearest-neighbor energy model and dynamic programming. The algorithm scales as O(n2{ell}4), offering substantial computational advantages over existing global prediction methods. Our analysis of 726 transfer messenger RNA and 455 Ribonuclease P RNA sequences reveals that biologically relevant pseudoknots are highly concentrated among section pairs with large minimum free energy gains. Over 90% of connected section pairs appear within just the top 3% of section pairs ranked by MFE gain. For 2-clusters, our method achieves high prediction accuracy with sensitivity exceeding 0.90 and positive predictive value above 0.80. For 3-clusters, we discovered asymmetric behavior where "former" section pairs (formed early in the sequence) predict accurately, while "latter" section pairs show different formation patterns. The hierarchical section-based approach demonstrates that local energy considerations can effectively predict pseudoknot formations between unpaired regions. Our work provides strong evidence for the effectiveness of local energy calculations in pseudoknot prediction and offers insights into the dynamic processes governing RNA structure formation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.29.672997v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">scAgeClock: a single-cell transcriptome based human aging clock model using gated multi-head attention neural networks</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xie, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Aging Clock models have emerged as a crucial tool for measuring biological age, with significant implications for anti-aging interventions and disease risk assessment. However, human aging clock models that offer single-cell resolution and account for cell and tissue heterogeneities remain underdeveloped. This study introduces scAgeClock, a novel gated multi-head attention (GMA) neural network-based single-cell aging clock model. Leveraging a large-scale dataset of over 16 million single-cell transcriptome profiles from more than 40 human tissues and 400 cell types, scAgeClock demonstrates improved age prediction accuracy compared to baseline methods. Notably, the mean absolute error for the best-performing cell type is remarkably low at 2 years. Feature importance analysis reveals enrichment of aging clock genes related to ribosome, translation, defense response, viral life cycle, programmed cell death, and COVID-19 disease. A novel metric, the Aging Deviation Index (ADI) proposed by this study, revealed deceleration of ages in cells with higher differentiation potencies and tumor cells in higher phases or under metastasis, while acceleration of ages was observed in skin cells. Furthermore, scAgeClock is publicly available to facilitate future research and potential implementations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.29.673183v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">BRD2 Upregulation as a Pan-Cancer Adaptive Resistance Mechanism to BET Inhibition</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Archasappawat, S., Jacques, J., Lee, E., Hwang, C.-I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Bromodomain and extraterminal motif (BET) inhibitors, such as JQ1, are promising cancer therapeutics that target epigenetic regulators, particularly BRD4. However, resistance to BET inhibitors (BETi) limits their clinical utility, necessitating a better understanding of adaptive mechanisms. We identified BRD2 upregulation as a conserved response to BET inhibition across multiple cancer types and hypothesized that BRD2 compensates for BRD4 loss, sustaining essential transcriptional programs upon treatment. Consistent with this, BRD2 knockdown sensitized cancer cells to BETi in vitro, and combining BRD2 depletion and JQ1 treatment significantly impaired tumor growth in vivo. At the chromatin level, BRD2 and BRD4 ChIP-seq analysis of pancreatic cancer cells showed consistent BRD4 loss from chromatin after JQ1 treatment, while BRD2 displacement differed by sensitivity. Resistant cells maintained higher BRD2 occupancy than sensitive cells, suggesting a link between BRD2 retention and drug response. Mechanistically, NFYA mediates BRD2 up-regulation as NFYA depletion abrogated BRD2 upregulation upon BETi treatment. Collectively, our findings establish BRD2 as a critical mediator of adaptive resistance to BETi in pan-cancer and identify NFYA as a novel transcriptional regulator of this process. Co-targeting BRD2 or its regulatory network offers a rational strategy to enhance the durability and efficacy of BET-based therapies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.30.673282v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Upregulation of GREB1 in colorectal cancer ovarian metastases may be a potential therapeutic target</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ruff, S. M., La Ferlita, A., Palavalli, M., Beane, J., Tsai, S., Eng, O., Kim, A. C., Esnakula, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Introduction: Classification of ovarian metastases (OM) in colorectal cancer (CRC) as peritoneal metastasis (PM) remains controversial. OM demonstrate resistance to systemic therapy, suggesting distinct molecular tumorigenesis. Gene expression profiling and transcriptomic analysis were performed to distinguish OM, PM, and primary CRC (pCRC) and identify potential therapeutic targets. Methods: RNA sequencing data were obtained from the Total Cancer Care database. After filtering out low-expressed genes, raw counts of retained genes were normalized in trimmed mean of M-values and then log2-transformed. Genes with a |Log2FC|>0.58 and adjusted p-value <0.05 were considered dysregulated. In silico motif enrichment analysis of estrogen receptor 1 (ESR1) on growth regulating estrogen receptor binding 1 (GREB1) was conducted. Results: There were 115 patients with tissue from OM (n=6), PM (n=4), and pCRC (n=105). Among upregulated genes, LAMC3, SCUBE1, and GREB1 had the highest differential expression in OM compared to PM, and PEG3, C7, and GREB1 had the highest differential expression in OM compared to pCRC (all p<0.001). GREB1 was upregulated in OM compared to PM and pCRC. There are two estrogen response element (ERE) sites within the promoter region of GREB1. ESR1 transcription factor was shown to bind to these ERE (p<0.001). Conclusions: Transcriptomic analysis demonstrated clear molecular distinction between OM, PM and pCRC. We identified upregulation of GREB1 in OM compared to PM and pCRC. GREB1 contains both ERE and ESR1 in the upstream promoter regions implying potential upstream transcription regulation mediated by ESR1. GREB1 may represent a novel, hormonal target in treatment of OM in CRC.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.31.673371v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Mitochondrial ACSS1 Links Acetate Metabolism to Pyrimidine Biosynthesis in Nutrient-Stressed B-Cell Lymphomas</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Basappa, J., Goldman, A., Lobello, C., Wang, S., Rushmore, D., Melnikov, O., Neil V. Sen, N., Mallikarjuna, V., Jain, P., Edalati, M., Cai, K., Lu, P., Nejati, R., Borghaei, H., Wellen, K., Wasik, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Acetate acts as an alternative carbon source in nutrient-scarce tumors, but its role in supporting nucleotide biosynthesis is still not well understood. Here, we identify the mitochondrial enzyme ACSS1 as a crucial metabolic driver in mantle cell lymphoma, diffuse large B-cell lymphoma, and chronic lymphocytic leukemia.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.29.673065v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">High-resolution spatial analysis reveals pregnenolone metabolism as a potential target for immuno-suppressive microenvironment of BRAF mutant colorectal cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, X., Zhao, Z., Liu, L., Cheng, J., Ba, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> BRAF mutant colorectal cancer (CRC) is highly aggressive and often resistant to traditional therapies, presenting a significant challenge in cancer immunotherapy. Understanding the tumor microenvironment (TME) and identifying novel immune checkpoints are crucial for improving treatment outcomes. Here, we employed high-resolution spatial analysis and single-cell RNA sequencing to map the TME of BRAF mutant CRC at single-cell resolution, focusing on the interactions between exhausted CD8+ T cells and C1QC+ macrophages. Mechanistically, we discovered that C1QC+ macrophages drive CD8+ T cell exhaustion through pregnenolone synthesis via the steroidogenic enzyme CYP11A1. Our findings identify pregnenolone metabolism as a previously unrecognized immunosuppressive pathway in BRAF mutant CRC, and suggest that steroid biosynthesis is a generalizable mechanism of immune evasion beyond endocrine-related malignancies. Targeting this pathway offers a promising therapeutic strategy for enhancing anti-tumor immunity in BRAF mutant CRC and potentially other cancers.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.01.673435v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Revealing the Paper Mill Iceberg: AI-Based Screening of Cancer Research Publications.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Scancar, B., Byrne, J. A., Causeur, D., Barnett, A. G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Paper mills are fraudulent organisations that publish fake manuscripts for profit. Paper mill-supported papers have been identified in the cancer research literature, where some papers show features of template use, with seemingly pre-formulated sentences. We trained a machine learning model to distinguish between paper mill papers and genuine cancer research papers published between 1999 and 2024. Using titles and abstracts, the model achieved a prediction accuracy of 0.91. When applied to the cancer research literature, it flagged 9.87% (95% CI 9.83 to 9.90) of papers and revealed a large increase in flagged papers from 1999 to 2024, both across the entire corpus and in the top 10% of journals by impact factor. Most publishers had substantial numbers of flagged papers. Over 170,000 papers by authors from Chinese institutions were flagged. These results indicate that paper mills are a large and growing problem in the cancer literature and are not restricted to low-impact journals.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.29.673016v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The Ubiquitin-Proteasome System is an Important Driver of EBV-Associated Nasopharyngeal Carcinoma Progression: A Meta-Analysis of Transcriptomics Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ratnawati, H., Sanjaya, A., Christiandy, A., Young, L. S., Ott, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;">Background: Epstein-Barr virus (EBV)-associated nasopharyngeal carcinoma (NPC) is characterized by extensive immune infiltration, yet immune evasion remains a hallmark of the disease. In this study, we aimed to leverage publicly available datasets to identify EBV-host gene interactions and re-map their expression at single-cell resolution. Methods: We conducted a meta-analysis of transcriptomic datasets to identify differentially expressed genes in NPC, mapped these genes to EBV-host interaction data, and constructed a network. Network clustering and pathway enrichment were performed, and single-cell RNA sequencing datasets were used to map these genes at cellular resolution. We analysed differences in cell cycle, immune signaling, cell-cell interactions, and assessed prognostic associations of a 12-gene UPS signature using the GEPIA2 platform. Results: We identified 85 EBV-interacting DEGs, most regulated by lytic-phase proteins. MCODE clustering highlighted genes related to the ubiquitin-proteasome system (UPS). Single-cell analyses confirmed elevated UPS-related gene expression in NPC compared to EBV-negative oropharyngeal cancer. UPS-High cells exhibited lower proliferative activity, enriched stemness signaling, and reduced antigen presentation and immune activation, whereas UPS-Low cells showed marked upregulation of growth-promoting pathways. Survival analysis revealed that high UPS expression was associated with poorer outcomes in both pan-cancer and head and neck squamous cell carcinoma datasets. The proportion of UPS-High cells varied widely across patients. Conclusion: UPS activity, influenced by EBV lytic proteins, is linked to immune evasion, stemness, proliferation, and adverse prognosis. These findings support UPS as a potential biomarker and therapeutic target in EBV-associated NPC, warranting NPC-specific validation and functional studies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.30.673297v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Metabolic Adaptation Study of Tumor Cells during Lung Cancer Bone Metastasis in Mice Based on Single-Cell Metabolome Analysis.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xuesen Hu, Tianrun Xu, Shengkai Xia, Yang Xu, Yao Chen, Liliang Wen, Wangshu Qin, Xianzhe Shi, Xinyu Liu, Qi Wang, Chunxiu Hu, Guowang Xu</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Lung cancer metastasis, the leading cause of patient mortality, is driven by circulating tumor cells (CTCs), which act as direct mediators of metastatic spread. To elucidate the metabolic heterogeneity across lung cancer metastatic stages, a panoramic single-cell metabolomics study in a mouse lung cancer bone metastasis model was performed using a concentric hybrid nanoelectrospray ionization-atmospheric pressure chemical ionization source. This platform enables high-coverage detection of polar and nonpolar metabolites, overcoming limitations in sensitivity and metabolite diversity. Unsupervised clustering and dimensionality reduction (t-SNE) of single-cell metabolic profiles distinguished primary tumor cells, CTCs, and bone metastatic cells, revealing stage-specific metabolic reprogramming. Machine learning identified key metabolites (e.g., aminobenzoic acid, 2-methyl-3-ketovaleric acid, pantothenic acid) that robustly discriminated metastatic stages with high accuracy (AUC > 0.96). CTCs exhibited dynamic metabolic adaptions at different stages: during blood circulation, amino acid and glutamine metabolism dominated to counteract nutrient deprivation, while during bone colonization, the tricarboxylic acid cycle and one-carbon metabolism were upregulated to support proliferation. This study provides important data to shed light on the metabolic heterogeneity of tumor cells and the metastasis mechanism of lung cancer.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40768566/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

