<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Monday, 21 July 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Network-Based Discovery of Prognostic Markers in Recurrent IDH wild-type Gliomas</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liu, Y., Huse, J. T., Kannan, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comprehensive analysis of gene expression data from 180 recurrent IDH wild-type glioma samples from the Glioma Longitudinal AnalySiS (GLASS) Consortium.</p><p style="font-size: 16px; text-align: justify;">Background: Isocitrate dehydrogenase wild-type (IDH wild-type) gliomas represents the most aggressive subtype of diffuse gliomas, characterized by therapeutic resistance and dismal prognosis. Despite advances in molecular classification, reliable prognostic biomarkers for these tumors remain limited, particularly for recurrent disease. This study aims to identify gene expression signatures associated with survival outcomes in recurrent IDH wild-type gliomas, with the goal of improving patient stratification and potential therapeutic targeting. Methods: We analyzed gene expression data from 180 recurrent IDH wild-type glioma samples from the Glioma Longitudinal AnalySiS (GLASS) Consortium. Using multiple computational approaches including a novel network-based method (netSurvival) and various survival analysis techniques, we identified genes associated with patient survival outcomes. Results: Our comprehensive analysis identify several gene expression markers that are significantly associated with survival outcomes in recurrent IDH wild-type gliomas. The AFT log normal model revealed that FN1, HIF3A, and EIF4B are associated with poorer survival (hazard ratios of 1.40, 1.49, and 1.54, respectively; p < 0.05), while PTK2, CCND2, RAD51L3 RFFL, and MAX demonstrated protective effects (hazard ratios of 0.76, 0.78, 0.79, and 0.79, respectively; p < 0.05). Five genes (KIF5C, LINC00632, B4GALNT3, HIF3A, and RAD51L3 RFFL) show significant differential expression between primary and recurrent tumors, with four having established functional roles in glioma pathobiology. Conclusions: This study identifies a panel of gene expression markers with significant prognostic value in recurrent IDH wild-type gliomas. The differential impacts of these genes on survival outcomes provide insights into the biological heterogeneity underlying clinical behavior in these aggressive tumors. Particularly significant are the biomarkers associated with both survival outcomes and recurrence patterns, which may represent key drivers of disease progression. These findings represent an important step toward improved prognostic stratification and therapeutic targeting in IDH wild-type gliomas, addressing a critical unmet need in neuro-oncology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.17.665414v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Aberrant splicing prediction during human organ development</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wagner, N., Neverov, A. M., Martin-Geary, A. C., Londhe, S., Schröder, C., Yepez, V. A., Whiffin, N., Gagneur, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We extend the AbSplice framework to incorporate splicing dynamics across developmental stages and improve the accuracy of splice-disruptive variant prediction and provide developmental context.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Developmental disorders constitute a major class of genetic diseases, yet tools to identify splicing-disruptive variants during development are lacking. To address this need we extended the AbSplice framework to incorporate splicing dynamics across developmental stages. Moreover, we introduced several improvements including a refined ground truth from the aberrant splicing caller FRASER2, a continuous representation of splice site usage, and integration of the rich set of predictions from the sequence-based model Pangolin. These advances double the precision and recall of the original model at predicting aberrant splicing events and enable predictions across embryonic, childhood, and adult tissues. Genome-wide scores for all single-nucleotide variants and a web interface to score indels are available to facilitate the exploration of the predictions. Our genome-wide predictions reveal a class of variants with splicing-disruptive effects confined to early development, particularly abundant in the brain (>18,000 variants). Analysis of genomes of individuals with a suspected Mendelian disorder from Genomics England's National Genomics Research Library identified 26 unique variants in disease-linked genes, with stronger predicted effects during development than in adulthood, including a candidate new diagnosis in the gene FGFR1. Altogether, these results improve the accuracy of splice-disruptive variant prediction and provide developmental context to aid interpretation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.16.665183v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Kinetic parameter prediction using neural networks identifies limitations to C4 photosynthesis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wendering, P., Ferguson, J., Xu, R., Kromdijk, J., Nikoloski, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present C4TUNE, an artificial neural network, which can efficiently predict parameters of a large-scale photosynthesis model from photosynthesis response curves.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Large-scale kinetic models of photosynthesis enable time-resolved predictions of traits related to this key process, and provide the means to identify factors limiting photosynthesis. However, their use is currently limited by the lack of efficient approaches to estimate the hundreds of genotype-specific kinetic parameters. Here, we present C4TUNE, an artificial neural network, which can efficiently predict parameters of a large-scale photosynthesis model from photosynthesis response curves. C4TUNE was trained on a biologically-relevant synthetic dataset comprising matched samples of parameters and response curves obtained using a C4 photosynthesis kinetic model. To speed up the training of C4TUNE, we devised a surrogate neural network to predict photosynthesis response curves directly from the model parameters and environmental inputs. Given response curves as input, we showed that over 99% of the parameter vectors predicted by C4TUNE could be used directly in simulation of the kinetic model and resulted in excellent fits. Finally, we applied C4TUNE to predict parameters for a population of 68 maize genotypes across two seasons. The predicted genotype-specific parameters allowed pinpointing factors that limit photosynthetic efficiency, validated using simulations. Therefore, the use of C4TUNE presents a fast and precise approach for parameter prediction based on minimal datasets.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.16.665120v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Fold first, ask later: structure-informed function annotation of Pseudomonas phage proteins</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Longin, H., Bouras, G., Grigson, S. R., Edwards, R. A., Hendrix, H., Lavigne, R., van Noort, V.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that up to 43% of truly unannotated proteins can be functionally annotated when combining structure-informed approaches with UniProt-derived annotations, and offer insights into optimizing structure-based annotation pipelines for viral proteins.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Phages, the viruses of bacteria, harbor an incredibly diverse repertoire of proteins capable of manipulating their bacterial hosts, inspiring many medical and biotechnological applications. However, to date, only a limited subset of that repertoire can be exploited, due to the difficulties in functionally elucidating these proteins. In this study, we investigated several structure-informed approaches to annotate hypothetical proteins from Pseudomonas infecting phages. We curated a representative dataset of over 10,000 proteins derived from NCBI, for which we predicted protein structures with ColabFold and assessed structural similarity via FoldSeek against the PDB, AlphaFold, and Phold databases. We evaluated multiple annotation strategies, including sequence-based (Pharokka), and structure-based (FoldSeek, Phold) methods. Our results show that up to 43 % of truly unannotated proteins can be functionally annotated when combining structure-informed approaches with UniProt-derived annotations. We highlight the complementarity of different databases and the importance of annotation quality filtering. This work provides a valuable resource of predicted structures and annotations, and offers insights into optimizing structure-based annotation pipelines for viral proteins, paving the way for deeper exploration of phage biology and its applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.17.665397v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Pervasive noise in human splice site selection</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Khokhar, E. S., Brokaw, K., Kartje, Z. J., Sanabria, V., Javeed, N., Kumar, A., Watts, J. K., Pai, A. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We use ultra-deep sequencing across RNA compartments to track splicing intermediates in human cells and see abundant cryptic splicing associated with genomic features that promote splicing noise.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> RNA splicing has historically been thought to be highly efficient and accurate, with little opportunity for deviation from regulated alternative splicing decisions. This dogma has been challenged by recent observations that suggest that biological noise may contribute substantially to transcriptome diversity. However, quantitative understanding of stochastic variations in splicing is challenging because these transcripts are likely subject to rapid degradation. Here, we use ultra-deep sequencing across RNA compartments to track splicing intermediates in human cells and see abundant cryptic splicing associated with genomic features that promote splicing noise. We observe pervasive usage of low-fidelity splice sites, likely due to stochasticity in recruitment or binding of the spliceosome. These sites are most likely degraded in the nucleus rather than targeted by translation-dependent degradation processes, suggesting widespread surveillance and rapid quality control of non-productive RNA transcripts. Our findings provide unprecedented insights into the propensity for error in RNA processing mechanisms and the regulation of alternative splice sites across a gene.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.16.665169v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">European ash pangenome reveals widespread structural variation and a diverse genetic basis for low ash dieback susceptibility</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wood, D. P., Vatanparast, M., Galanti, D., Wheeler, K., Curran, E., Yant, L., Whittet, R., Nichols, R. A., Buggs, R. J. A., Kelly, L. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a new chromosomal-level pangenome for the European Ash tree and used it to identify loci associated with low susceptibility to Ash Dieback disease.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> European Ash (Fraxinus excelsior) is a keystone forest tree species, whose populations are being decimated by Ash Dieback disease (ADB). Uncovering the genetic basis of low susceptibility to this devastating disease relies on a comprehensive understanding of genetic variation present in F. excelsior. A linear reference genome from a single individual cannot contain the total sequence variability within a species, including its genic regions; a pangenome more fully captures total sequence content. In this study, we developed a F. excelsior pangenome reference using a new chromosomal-level phased linear reference genome, and de novo assemblies and long-read data from a geographically diverse set of fifty F. excelsior samples. We identified 362,965 structural variants (SVs), including 174Mb of sequence absent from the linear reference genome (22% of the linear reference size). We demonstrated that failing to explicitly link SVs with gene sequences can lead to substantial overestimation of dispensable genes (those that vary in their presence/absence between individuals) due to variability in the annotation process. Controlling for this, we identified 3,065 dispensable genes including 133 with gene ontologies associated with low disease susceptibility. We used the pangenome to analyse existing genomic data from over 1,200 individuals to identify loci associated with reduced susceptibility to ADB. This analysis revealed that different genetic backgrounds harbour distinct sets of loci associated with low susceptibility, highlighting the complexity of this trait but also the potential benefits of combining variants from different genetic backgrounds to boost efforts to ensure the future survival of F. excelsior.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.16.665114v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">RNALens: Study on 5' UTR Modeling and Cell-Specificity</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mao, L., Tian, Y., Qian, K.-w., Song, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose RNALens, a 5' untranslated region-based model for RNA expression and translation prediction that is efficient in silico and generalizes across diverse cell types and can be used to guide the design of mRNA therapeutics.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recently, the Transformer architecture has been applied to predict the structure, function, and regulatory activity of biological sequences. Predicting the cell-specific regulatory impact of 5' untranslated regions (5' UTRs) on mRNA expression and translation remains a key challenge for rational mRNA design. Existing studies such as UTR-LM, RNABERT, and RNA-FM train transformer-based models solely on 5' UTR sequences with fixed nucleotide tokenization schemes and auxiliary structural features. These models pay less attention to the integration of broader genomic context and thermodynamic objectives, which limits their ability to generalize across diverse cell types and accurately predict both mRNA expression level (EL) and translation efficiency (TE). In this paper, we propose RNALens, a foundation model pre-trained in two stages on multispecies genomic sequences and curated 5' UTR data using masked language modeling augmented with secondary structure prediction and minimum free energy regression. RNALens employs byte-pair encoding to capture variable-length nucleotide motifs. It is then fine-tuned on high-throughput reporter assay datasets from HEK293T, PC3, and muscle tissues to yield specialized predictors for EL and TE in each cellular context. Experiment results on benchmark datasets demonstrate that RNALens achieves superior performance than existing machine learning methods for both expression and translation predictions across cell-specific and cross-context tests, offering an efficient in silico platform for guiding the design of mRNA therapeutics with precise cellular targeting.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.20.665722v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The Impact of Reference Genome Divergence on Ancient DNA Damage Detection in Metagenomic Contexts</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Guinet, B., Sharif, B. M., van der Valk, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A correction matrix that adjusts C-to-T damage profiles for reference divergence, improving damage signal recovery, improving the detectability and intensity of characteristic damage patterns.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The reliability of ancient DNA (aDNA) authentication depends on detecting characteristic damage patterns, particularly cytosine deamination at fragment ends. However, in ancient metagenomic studies, sequence divergence between aDNA reads and available reference genomes may obscure such damage signals. We systematically evaluated how reference genome divergence, read count, read length, and damage levels affect aDNA damage profiles using both empirical datasets and controlled simulations. Using ancient Yersinia pestis and Hepatitis B virus data, we show that mapping to divergent reference genomes significantly reduces the detectability and intensity of characteristic damage patterns, particularly at low read counts. Simulations further revealed that reference genome identity is the strongest predictor of damage intensity, while read count primarily influences damage stochasticity. We introduce a correction matrix that adjusts C-to-T damage profiles for reference divergence, improving damage signal recovery. Our findings highlight methodological considerations for authenticating aDNA in metagenomic contexts, particularly when closely related reference genomes are unavailable.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.16.665190v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Systematic investigation of Epstein-Barr virus transcriptional regulator interactions with the human genome</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dexheimer, P. J., Hass, M. R., Edsall, L. E., Diouf, A. A., Jones, S. H., Donmez, O., Yin, C., Dunn, K. A., Forney, C., Hesse, H. K., VonHandorf, A., Chen, X., Parameswaran, S., Gittens, O. E., Viel, K. C. M. F., Ernst, K., Razavi, R., Gewurz, B. E., Zhao, B., Lawson, L. P., Hughes, T. R., Kottyan, L., Weirauch, M. T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We systematically investigate interactions between Epstein-Barr virus (EBV) transcriptional regulators (vTRs) and the human genome, including >100,000 vTR binding events impacting almost a quarter of all human genes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We systematically investigate interactions between Epstein-Barr virus (EBV) transcriptional regulators (vTRs) and the human genome. Starting with 16 known and candidate vTRs, we identify nine whose introduction into human cells results in substantial alterations to human gene expression. Genome-scale determination of vTR genomic binding events and alterations to chromatin accessibility reveals a detailed map of EBV's functional interactions with the human genome, including >100,000 vTR binding events impacting almost a quarter of all human genes. Notably, BMRF1 emerges as a potent regulator, impacting >7,000 genes and altering >37,000 chromatin regions. Our results provide new evidence that EBV RTA interacts with and stabilizes the binding of human RBPJ. Network analysis reveals that many human genes are targeted by multiple EBV vTRs, highlighting the vast coordinated impact of EBV on human gene expression. Collectively, this study provides a valuable resource for examining EBV-induced alterations to human gene regulation, with data available on multiple platforms.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.18.665561v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Global cis-regulatory landscape of double-stranded DNA viruses</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Taslim, T. H., Finkelberg, J. A., Kales, S., Soto-Ugaldi, L., D'Elia, B., Engin, B., Munoz-Esquivel, G., Morara, E., Purinton, J., Chandok, H., Rottenberg, J. T., Castro, R., Martinez-Cuesta, L., Paz, M. A., Tewhey, R., Bass, J. I. F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify thousands of viral cis-regulatory elements (CREs) that regulate viral replication, reactivation, evolution, and viral vector design.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Most double-stranded DNA (dsDNA) viruses use the host transcriptional machinery to express viral genes for replication and immune evasion. This is mediated by viral cis-regulatory elements (CREs) regulated by host and viral transcription factors (TFs). Although some viral CREs and their regulatory mechanisms have been determined, most remain unidentified. Here, we used massively parallel reporter assays to identify ~2,000 CREs across 27 dsDNA viruses from the Adenovirus, Herpesvirus, Polyomavirus and Papillomavirus families. Viral genomes have a higher CRE density than the human genome, with most viral CREs having promoter-like features and overlapping protein coding sequences. Using saturation mutagenesis and machine learning models, we report viral CRE regulators, including SP, ETS, bZIPs, and TFs acting downstream of signal-activated pathways. Altogether, we present a comprehensive functional CRE map of human-infecting dsDNA viruses that serves as a blueprint for further studies in viral regulation, reactivation, evolution, and viral vector design.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.20.665756v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Investigating Cannabis sativa L. gene expression through housekeeping genes and gene coexpression networks</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Barbosa-Xavier, K., Venancio, T. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify and prioritize housekeeping genes for qRT-PCR normalization and the construction of a genome-wide gene coexpression network (GCN) using publicly available RNA-Seq data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cannabis sativa L. is a versatile crop with applications ranging from medicinal products to industrial materials. Despite growing interest in cannabis transcriptomics, comprehensive studies of gene expression in this species remain limited. Here, we explore cannabis transcriptomics through two complementary approaches: the identification and prioritization of housekeeping (HK) genes for qRT-PCR normalization and the construction of a genome-wide gene coexpression network (GCN) using publicly available RNA-Seq data. Based on expression stability criteria (i.e. log mean TPM [≥] 6, log variance < 0.3), we proposed the optimal HK genes and designed qRT-PCR primers for them. Primer specificity was verified in silico using the Jamaican Lion mother + Y genome. In parallel, to explore the non HK gene expression, we constructed a GCN comprising 32 coexpression modules, which was then analyzed for tissue-specific expression patterns, transcription factor (TF), and functional enrichment. Several modules were significantly correlated with cannabis tissues such as flowers and bast fibres and were linked to key processes, including photoperiod sensitivity, stress responses, and vegetative-to-reproductive phase transitions. Together, our results help understand the functions and maximize the use of HK genes in molecular biology and reveal coexpression modules that constitute critical genetic circuits underlying cannabis physiology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.16.665086v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Early seed priming with closely related Bacillus strains induces divergent physiological and defense responses in melon</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Carregalo-Rios, L., Molina-Santiago, C., Berlanga-Clavero, M. V., Petras, D., Hierrezuelo, J., Pineda, M., Alba, J. M., de Vicente, A., Baron, M., Dorrestein, P. C., Romero, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that melon perceives inoculum dose and microbial identity, translating them into distinct metabolic and defense programs that converge on stress resilience.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Early microbial seed priming is a promising strategy to improve crop resilience, yet it remains unclear whether plants can discriminate among closely related beneficial strains and integrate dose-dependent microbial cues. We primed melon (Cucumis melo) seeds with two phylogenetically similar Bacillus strains (B. subtilis NCIB3610 and B. velezensis FZB42) and combined transcriptomic, metabolomic and physiological analyses across development. Despite comparable colonization, the strains provoked contrasting host programs and distinct dose responses. B. subtilis promoted radicle elongation, chloroplastic starch storage and drought tolerance irrespective of inoculum level, together with L-tryptophan and palatinose accumulation. By contrast, B. velezensis displayed a clear dose effect: low inoculum sustained normal radicle growth, whereas high inoculum transiently repressed it, coinciding with retrotransposon activation, suppression of AOS and proteasome genes, and enrichment of flavonoids and glutathione in leaves. Chemical assays showed that radicle inhibition requires the combination of surfactin, produced by both strains, with bacillomycin D, an iturin-type lipopeptide unique to FZB42; neither molecule alone reproduced the effect. This synergy links the strain-specific lipopeptide repertoire to the dose-dependent growth response. Although their early trajectories diverged, both primings converged on improved above-ground performance. 3610-primed plants restricted Botrytis cinerea via caffeic- and rosmarinic-acid accumulation, whereas FZB42-primed plants curtailed jasmonate-sensitive Tetranychus urticae mites through early JA-pathway activation. Our results demonstrate that melon perceives inoculum dose and microbial identity, translating them into distinct metabolic and defense programs that converge on stress resilience. These mechanistic insights-linking lipopeptide fingerprints, sentinel metabolites and defense transcripts-provide a framework for precision seed treatments in horticultural crops.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.16.665077v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Omics in IgG4-related disease.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shaozhe Cai, Yu Chen, Ziwei Hu, Shengyan Lin, Rongfen Gao, Bingxia Ming, Jixin Zhong, Wei Sun, Qian Chen, John H Stone, Lingli Dong</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2024-10-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We review the current progress of omics research in IgG4-RD and discuss the value of machine learning methods in analyzing the data with high dimensionality.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Research on IgG4-related disease (IgG4-RD), an autoimmune condition recognized to be a unique disease entity only two decades ago, has processed from describing patients' symptoms and signs to summarizing its critical pathological features, and further to investigating key pathogenic mechanisms. Challenges in gaining a better understanding of the disease, however, stem from its relative rarity-potentially attributed to underrecognition-and the absence of ideal experimental animal models. Recently, with the development of various high-throughput techniques, "omics" studies at different levels (particularly the single-cell omics) have shown promise in providing detailed molecular features of IgG4-RD. While, the application of omics approaches in IgG4-RD is still at an early stage. In this paper, we review the current progress of omics research in IgG4-RD and discuss the value of machine learning methods in analyzing the data with high dimensionality.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/39450944/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

