<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Sunday, 27 April 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A computational framework for identifying cytoskeletal genes associated with age-related diseases.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Reem A Elghaish, Nayera E Attallah, Hesham Khaled, Asmaa S Mekawy, Menattallah Elserafy, Eman Badr</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> This study aims to investigate the transcriptional changes of the cytoskeletal genes and their regulators in five age-related diseases: Hypertrophic Cardiomyopathy (HCM), Coronary Artery Disease (CAD), Alzheimer's disease (AD), Idiopathic Dilated</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The cytoskeleton comprises polymers from protein filaments shaped in an organized structure. This structure contributes significantly to the cell's function and viability. Decades of research have implicated that the cytoskeleton's dynamic nature is associated with downstream signaling events that further regulate cellular activity and control aging and neurodegeneration. This study aims to investigate the transcriptional changes of the cytoskeletal genes and their regulators in five age-related diseases: Hypertrophic Cardiomyopathy (HCM), Coronary Artery Disease (CAD), Alzheimer's disease (AD), Idiopathic Dilated Cardiomyopathy (IDCM), and Type 2 Diabetes Mellitus (T2DM). An integrative approach of machine learning-based models and differential expression analysis was employed to identify potential biomarkers based on the cytoskeletal genes. Multiple machine-learning algorithms were used, where the Support Vector machines (SVM) classifier achieved the highest accuracy. The study highlighted 17 genes involved in the cytoskeleton's structure and regulation associated with age-related diseases. The results provide a holistic overview of the role of transcriptionally dysregulated cytoskeletal genes in age-related diseases. This study pinpoints cytoskeletal genes and regulators of the cytoskeleton that can be utilized as potential markers and drug targets.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40287491/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">metaTP: a meta-transcriptome data analysis pipeline with integrated automated workflows.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Limuxuan He, Quan Zou, Yansu Wang</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> The accessibility of sequencing technologies has enabled meta-transcriptomic studies to provide a deeper understanding of microbial ecology at the transcriptional level.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The accessibility of sequencing technologies has enabled meta-transcriptomic studies to provide a deeper understanding of microbial ecology at the transcriptional level. Analyzing omics data involves multiple steps that require the use of various bioinformatics tools. With the increasing availability of public microbiome datasets, conducting meta-analyses can reveal new insights into microbiome activity. However, the reproducibility of data is often compromised due to variations in processing methods for sample omics data. Therefore, it is essential to develop efficient analytical workflows that ensure repeatability, reproducibility, and the traceability of results in microbiome research.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40287646/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Popcorn: prediction of short coding and noncoding genomic sequences in prokaryotes.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Alison Kyrouz, Lian Liu, Lixin Qin, Brian Tjaden</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present Popcorn, a novel machine learning method for determining whether prokaryotic sequences are coding or noncoding, including coding sORFs and nonc coding RNAs.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The most challenging prokaryotic genes to identify often correspond to short ORFs (sORFs) encoding small proteins or to noncoding RNAs. RNA-seq experiments commonly evince small transcripts that do not correspond to annotated genes and are candidates for novel coding sORFs or small regulatory RNAs, but it can be difficult to accurately assess whether the numerous small transcripts are coding or not. We present Popcorn (PrOkaryotic Prediction of Coding OR Noncoding), a novel machine learning method for determining whether prokaryotic sequences are coding or noncoding. We find that Popcorn is effective in distinguishing coding from noncoding sequences, including coding sORFs and noncoding RNAs.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40279271/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Directing stem cell differentiation by chromatin state approximation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Montano-Gutierrez, L. F., Mueller, S., Kutschat, A. P., Seruggia, D., Halbritter, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We found that greedy selection by chromatin approximation can be a viable optimisation strategy for the generation of erythroblasts from haematopoietic stem cells, and discovered transcriptional regulators linked to roadblocks in differentiation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> A prime goal of regenerative medicine is to replace dysfunctional cells in the body. To design protocols for producing target cells in the laboratory, one may need to consider exponentially large combinations of culture components. Here, we investigated the potential of iteratively approximating the target phenotype by quantifying the distance between chromatin profiles (ATAC-seq) of differentiating cells in vitro and their in-vivo counterparts. We tested this approach on the well-studied generation of erythroblasts from haematopoietic stem cells, evaluating a fixed number of components over two sequential differentiation rounds (8x8 protocols). We found that the most erythroblast-like cells upon the first round yielded the most erythroblast-like cells at the second round, suggesting that greedy selection by chromatin approximation can be a viable optimisation strategy. Furthermore, by analysing regulatory sequences in incompletely reprogrammed chromatin regions, we uncovered transcriptional regulators linked to roadblocks in differentiation and made a data-driven selection of ligands that further improved erythropoiesis. In future, our methodology can help craft notoriously difficult cells in vitro, such as B cells.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.24.650451v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DNA Methylation Dynamics of Dose-dependent Acute Exercise, Training Adaptation, and Detraining</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hariharan, M., Patel, S., Song, H., Rehman, A., Barragan, C., Bartlett, A., Castanon, R., Nery, J., Rothenberg, V., Chen, H., Tian, W., Ding, W., Wang, W., McAdam, J., Graham, Z., Lavin, K., Bamman, M., Broderick, T., Ecker, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We study the dynamics of DNA methylation in blood and muscle derived from males and females, and show that exercise- and training-induced epigenetic changes alter immune surveillance, mitochondrial function, and inflammatory response, and underscore the relevance of epigenetic plasticity to health monitoring and wellness.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Exercise and diet are direct physical contributors to human health, wellness, resilience, and performance [1-5]. Endurance and resistance training are known to improve healthspan through various biological processes such as mitochondrial function [6-8], telomere maintenance [9], and inflammaging [10]. Although several training prescriptions have been defined with specific merits [1,10-20], the long-term effects of these in terms of their molecular alterations have not yet been well explored. In this study, we focus on two combined endurance and resistance training programs: (1) traditional moderate-intensity continuous endurance and resistance exercise (TRAD) and (2) a variation of high-intensity interval training (HIIT) we refer to as high intensity tactical training (HITT), to assess the dynamics of DNA methylation (DNAm) in blood and muscle derived from males (N=23) and females (N=31), over a 12-week period of training followed by a 4-week period of detraining, sampled at pre-exercise and acute time points, totaling 528 samples. Due to its rapid responsiveness to stimuli and its stability, DNAm has been known to facilitate regulatory cascades that significantly affect various physiological processes and pathways. We find that several thousand differentially methylated regions (DMRs) associated with acute exercise in blood, many of which are shared across males and females. This trend is reversed when comparing the baseline (pre-exercise) time points or post-exercise timepoints at the untrained state with those at the post-conditioned state. Here, muscle shows majority of DNAm changes, with most of those being unique. We also find several hundred memory DMRs in muscle that maintain the gain or loss of methylation after four weeks of inactivity. Comparing phenotypic measurements, we find specific DMRs that correlate significantly with mitochondrial function and myofiber switching. Using machine learning, we select a subset of DMRs that are most characteristic of training modalities, sex and timepoint. Most of the DMRs are enriched in pathways associated with immune function, cell differentiation, and exercise adaptation. These findings reveal mechanisms by which exercise- and training-induced epigenetic changes alter immune surveillance, mitochondrial function, and inflammatory response, and underscore the relevance of epigenetic plasticity to health monitoring and wellness.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.22.650067v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MiT4SL: multi-omics triplet representation learning for cancer cell line-adapted prediction of synthetic lethality</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tao, S., Feng, Y., Yang, Y., Wu, M., Zheng, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose MiT4SL, a multi-omics triplet representation learning model for cell line-adapted SL prediction, which is the first deep learning model designed specifically for cancer cell-line- adapted SL prediction.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Synthetic lethality (SL) offers a promising approach for targeted cancer therapies. Current SL prediction models heavily rely on extensive labeled data for specific cell lines to accurately identify SL pairs. However, a major limitation is the scarcity of SL labels across most cell lines, which makes it challenging to predict SL pairs for target cell lines with limited or even no available labels in real-world scenarios. Furthermore, gene interactions could be opposite between training and test cell lines, i.e. SL vs. non-SL, which further aggravates the challenge of generalization among cell lines. A promising strategy is to transfer knowledge learned from cell lines with relatively abundant SL labels to those with limited SL labels for the discovery of novel SL pairs, i.e., cell line-adapted SL prediction. Here, we propose MiT4SL, a multi-omics triplet representation learning model for cell line-adapted SL prediction. The core idea of MiT4SL is to model cell lineage information as embeddings, which are generated by combining a protein-protein interaction network representation tailored to each cell line with the corresponding protein sequence embeddings. We then combine these cell line embeddings with gene pair representations derived from a biomedical knowledge graph and protein sequences. This triplet representation learning strategy enables MiT4SL to capture both shared biological mechanisms across cell lines and those unique to each cell line, effectively mitigating distribution shift and improving generalization to target cell lines. Additionally, explicit cell line embeddings provide the necessary signals for MiT4SL to effectively differentiate between cell line contexts, enabling it to adjust predictions and mitigate possible label conflicts for the same gene pair across different cell lines. Experimental results across various cell line-adapted scenarios show that MiT4SL outperforms six state-of-the-art models. To the best of our knowledge, MiT4SL is the first deep learning model designed specifically for cancer cell line-adapted SL prediction.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.20.649694v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Mapping the rapid growth of multi-omics in tumor immunotherapy: Bibliometric evidence of technology convergence and paradigm shifts.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Huijing Dong, Xinmeng Wang, Yumin Zheng, Jia Li, Zhening Liu, Aolin Wang, Yulei Shen, Daixi Wu, Huijuan Cui</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A systematic analysis of omics-driven tumor immunotherapy research through a bibliometric lens.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> This study aims to fill the knowledge gap in systematically mapping the evolution of omics-driven tumor immunotherapy research through a bibliometric lens. While omics technologies (genomics, transcriptomics, proteomics, metabolomics)provide multidimensional molecular profiling, their synergistic potential with immunotherapy remains underexplored in large-scale trend analyses. A comprehensive search was conducted using the Web of Science Core Collection for literature related to omics in tumor immunotherapy, up to August 2024. Bibliometric analyses, conducted using R version 4.3.3, VOSviewer 1.6.20, and Citespace 6.2, examined publication trends, country and institutional contributions, journal distributions, keyword co-occurrence, and citation bursts. This analysis of 9,494 publications demonstrates rapid growth in omics-driven tumor immunotherapy research since 2019, with China leading in output (63% of articles) yet exhibiting limited multinational collaboration (7.9% vs. the UK's 61.8%). Keyword co-occurrence and citation burst analyses reveal evolving frontiers: early emphasis on "PD-1/CTLA-4 blockade" has transitioned toward "machine learning," "multi-omics," and "lncRNA," reflecting a shift to predictive modeling and biomarker discovery. Multi-omics integration has facilitated the development of immune infiltration-based prognostic models, such as TIME subtypes, which have been validated across multiple tumor types, which inform clinical trial design (e.g. NCT06833723). Additionally, proteomic analysis of melanoma patients suggests that metabolic biomarkers, particularly oxidative phosphorylation and lipid metabolism, may stratify responders to PD-1 blockade therapy. Moreover, spatial omics has confirmed ENPP1 as a potential novel therapeutic target in Ewing sarcoma. Citation trends underscore clinical translation, particularly mutation-guided therapies. Omics technologies are transforming tumor immunotherapy by enhancing biomarker discovery and improving therapeutic predictions. Future advancements will necessitate longitudinal omics monitoring, AI-driven multi-omics integration, and international collaboration to accelerate clinical translation. This study presents a systematic framework for exploring emerging research frontiers and offers insights for optimizing precision-driven immunotherapy.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40275437/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A hybrid machine learning model with attention mechanism and multidimensional multivariate feature coding for essential gene prediction.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wu Yan, Fu Yu, Li Tan, Li Mengshan, Xie Xiaojun, Zhou Weihong, Sheng Sheng, Wang Jun, Wu Fu-An</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a new approach for essential gene prediction based on machine learning, which can help to identify the most important genes in the genome.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Essential genes are crucial for the development, inheritance, and survival of species. The exploration of these genes can unravel the complex mechanisms and fundamental life processes and identify potential therapeutic targets for various diseases. Therefore, the identification of essential genes is significant. Machine learning has become the mainstream approach for essential gene prediction. However, some key challenges in machine learning need to be addressed, such as the extraction of genetic features, the impact of imbalanced data, and the cross-species generalization ability.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40275343/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identification of pivotal genes and regulatory networks associated with SAH based on multi-omics analysis and machine learning.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Haoran Lu, Teng Xie, Xiaohong Qin, Shanshan Wei, Zilong Zhao, Xizhi Liu, Liquan Wu, Rui Ding, Zhibiao Chen</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified five SAH-related feature genes and 1336 differentially expressed proteins in the immune microenvironment, which could serve as potential therapeutic targets and provide clues for exploring therapeutic options.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Subarachnoid hemorrhage (SAH) is a disease with high mortality and morbidity, and its pathophysiology is complex but poorly understood. To investigate the potential therapeutic targets post-SAH, the SAH-related feature genes were screened by the combined analysis of transcriptomics and metabolomics of rat cortical tissues following SAH and proteomics of cerebrospinal fluid from SAH patients, as well as WGCNA and machine learning. The competitive endogenous RNAs (ceRNAs) and transcription factors (TFs) regulatory networks of the feature genes were constructed and further validated by molecular biology experiments. A total of 1336 differentially expressed proteins were identified, including 729 proteins downregulated and 607 proteins upregulated. The immune microenvironment changed after SAH and the changement persisted at SAH 7d. Through multi-omics and bioinformatics techniques, five SAH-related feature genes (A2M, GFAP, GLIPR2, GPNMB, and LCN2) were identified, closely related to the immune microenvironment. In addition, ceRNAs and TFs regulatory networks of the feature genes were constructed. The increased expression levels of A2M and GLIPR2 following SAH were verified, and co-localization of A2M with intravascular microthrombus was demonstrated. Multiomics and bioinformatics tools were used to predict the SAH associated feature genes confirmed further through the ceRNAs and TFs regulatory network development. These molecules might play a key role in SAH and may serve as potential biological markers and provide clues for exploring therapeutic options.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40274967/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identification and verification of mitochondria-related genes biomarkers associated with immune infiltration for COPD using WGCNA and machine learning algorithms.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Meijuan Peng, Chen Jiang, Ziyu Dai, Bin Xie, Qiong Chen, Jianing Lin</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified five mitochondrial-related genes associated with COPD and its immune microenvironment and identified a potential therapeutic target for androstenol.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mitochondrial dysfunction plays a pivotal role in the pathogenesis of chronic obstructive pulmonary disease (COPD). This study combines bioinformatics analysis with machine learning to elucidate potential key mitochondrial-related genes associated with COPD and its immune microenvironment. We utilized the limma package and Weighted Gene Co-expression Network Analysis (WGCNA) to analyze datasets from the Gene Expression Omnibus (GEO) database (GSE57148), identifying 12 key differentially expressed mitochondrial genes (MitoDEGs). Using 12 distinct machine learning algorithms (comprising 143 predictive models), we identified the optimal diagnostic model, which includes five pivotal MitoDEGs: ERN1, FASTK, HIGD1B, NDUFA7 and NDUFB7. The diagnostic specificity and sensitivity of each gene, as well as the diagnostic model itself, were evaluated using Receiver operating characteristic (ROC) curves. This model demonstrated high specificity in the validation cohorts (GSE76925, GSE151052, GSE239897). Expression analysis revealed upregulation of ERN1 and downregulation of FASTK, HIGD1B, NDUFA7 and NDUFB7 in COPD patients. Spearman's correlation analysis indicated a significant association between MitoDEGs and immune cell infiltration, with ERN1 expression positively correlated with neutrophil infiltration and the other genes negatively correlated. The GABA receptor modulator androstenol was identified as a potential therapeutic candidate. In vivo studies confirmed reduced mRNA expression of HIGD1B and NDUFB7 in COPD mice. These findings elucidate mitochondrial-immune interactions in COPD and highlight novel diagnostic and therapeutic targets.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40274954/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Definer: A computational method for accurate identification of RNA pseudouridine sites based on deep learning.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bo Han, Sudan Bai, Yang Liu, Jiezhang Wu, Xin Feng, Ruihao Xin</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a deep learning-based computational method, Definer, to accurately identify RNA pseudouridine modification sites from high-throughput RNA sequence data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Pseudouridine is an important modification site, which is widely present in a variety of non-coding RNAs and is involved in a variety of important biological processes. Studies have shown that pseudouridine is important in many biological functions such as gene expression, RNA structural stability, and various diseases. Therefore, accurate identification of pseudouridine sites can effectively explain the functional mechanism of this modification site. Due to the rapid increase of genomics data, traditional biological experimental methods to identify RNA modification sites can no longer meet the practical needs, and it is necessary to accurately identify pseudouridine sites from high-throughput RNA sequence data by computational methods. In this study, we propose a deep learning-based computational method, Definer, to accurately identify RNA pseudouridine loci in three species, Homo sapiens, Saccharomyces cerevisiae and Mus musculus. The method incorporates two sequence coding schemes, including NCP and One-hot, and then feeds the extracted RNA sequence features into a deep learning model constructed from CNN, GRU and Attention. The benchmark dataset contains data from three species, H. sapiens, S. cerevisiae and M. musculus, and the results using 10-fold cross-validation show that Definer significantly outperforms other existing methods. Meanwhile, the data sets of two species, H. sapiens and S. cerevisiae, were tested independently to further demonstrate the predictive ability of the model. In summary, our method, Definer, can accurately identify pseudouridine modification sites in RNA.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40273178/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Reference-guided genome assembly at scale using ultra-low-coverage high-fidelity long-reads with HiFiCCL</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jiang, Z., Pan, W., Gao, R., Hu, H., Gao, W., Zhou, M., Yin, Y.-H., Qian, Z., Jin, S., Wang, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose HiFiCCL, the first assembly framework specifically designed for ultra-low-coverage high-fidelity reads, using a reference-guided, chromosome-by-chromosome assembly approach.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Population genomics using short-read resequencing captures single nucleotide polymorphisms and small insertions and deletions but struggles with structural variants (SVs), leading to a loss of heritability in genome-wide association studies. In recent years, long-read sequencing has improved pangenome construction for key eukaryotic species, addressing this issue to some extent. Sufficient-coverage high-fidelity (HiFi) data for population genomics is often prohibitively expensive, limiting its use in large-scale populations and broader eukaryotic species and creating an urgent need for robust ultra-low coverage assemblies. However, current assemblers underperform in such conditions. To address this, we propose HiFiCCL, the first assembly framework specifically designed for ultra-low-coverage high-fidelity reads, using a reference-guided, chromosome-by-chromosome assembly approach. We demonstrate that HiFiCCL improves ultra-low-coverage assembly performance of existing assemblers and outperforms the state-of-the-art assemblers on human and plant datasets. Tested on 45 human datasets (~5x coverage), HiFiCCL combined with hifiasm reduces the length of misassembled contigs relative to hifiasm by an average of 21.19% and up to 38.58%. These improved assemblies enhance germline structural variant detection, reduce chromosome-level mis-scaffolding, enable more accurate pangenome graph construction, and improve the detection of rare and somatic structural variants based on the pangenome graph under ultra-low-coverage conditions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.20.649739v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Attention-Based Solution for Synergistic Virus Combination Therapy</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Majidifar, S., Hooshmand, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> This paper proposes AI-based models to predict novel drug combinations that can synergistically treat viral diseases.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Computational drug repurposing is vital in drug discovery research because it significantly reduces both the cost and time involved in the drug development process. Additionally, combination therapy--using more than one drug for treatment--can enhance efficacy and minimize the side effects associated with individual drugs. However, there is currently limited research focused on computational approaches to combination therapy for viral diseases. This paper proposes AI-based models to predict novel drug combinations that can synergistically treat viral diseases. To achieve this, we have compiled a comprehensive dataset containing information on viruses, drug compounds, and their approved interactions. We introduce two attention-based models and compare their performance with traditional machine learning and deep learning models in predicting synergistic drug pairs for treating viral diseases. Among all the methods tested, the random forest algorithm and one of the attention-based models utilizing a customized dot product as a predictor showed the highest performance. Notably, two predicted combinations--acyclovir + ribavirin and acyclovir + Pranobex Inosine--have been experimentally validated to produce a synergistic antiviral effect against the herpes simplex virus type 1, as reported in existing literature.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.22.649915v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Task Splitting and Prompt Engineering for Cypher Query Generation in Domain-Specific Knowledge Graphs</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Soleymani, S., Gravel, N. M., Kochut, K., Kannan, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel approach, Prompt2Cypher (P2C), which leverages task splitting and prompt engineering to decompose user queries into manageable subtasks, enhancing LLMs ability to generate accurate Cypher queries that align with the underlying graph database schema.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The integration of large language models (LLMs) with knowledge graphs (KGs) holds significant potential for simplifying the process of querying graph databases, especially for non-technical users. KGs provide a structured representation of domain-specific data, enabling rich and precise information retrieval. However, the complexity of graph query languages, such as Cypher, presents a barrier to their effective use by non-experts. This research addresses the challenge by proposing a novel approach, Prompt2Cypher (P2C), which leverages task splitting and prompt engineering to decompose user queries into manageable subtasks, enhancing LLMs ability to generate accurate Cypher queries that align with the underlying graph database schema. We demonstrate the effectiveness of P2C in two biological KGs (protein kinase and ion-channel) that differ in size, schema and complexity. Compared to a baseline approach, our method improves query accuracy, as demonstrated by higher Precision, Recall, F1-score, and Jaccard similarity metrics. This work contributes to the ongoing efforts to bridge the gap between domain-specific knowledge graphs and user-friendly graph database query interfaces.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.23.649790v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comparative analysis of genomic prediction approaches for multiple time-resolved traits in maize</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hobby, D., Lindner, R., Mbebi, A. J., Tong, H., Nikoloski, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We compared and contrasted the performance of MegaLMM and dynamicGP as well as their hybrid variants that can handle high-dimensional temporal data for multi-trait genomic prediction.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Ability to accurately predict multiple growth-related traits over plant developmental trajectories has the potential to revolutionize crop breeding and precision agriculture. Despite increased availability of time-resolved data for multiple traits from high-throughput phenotyping platforms of model plants and crops, genomic prediction is largely applied to a small number of traits, often neglecting their dynamics. Here, we compared and contrasted the performance of MegaLMM and dynamicGP as well as their hybrid variants that can handle high-dimensional temporal data for multi-trait genomic prediction. The comparative analysis made use of time series for 50 geometric, colour, and texture traits in a maize multiparent advanced generation inter-cross (MAGIC) population. The performance of the approaches was assessed using snapshot accuracy and longitudinal accuracy, providing insight into the ability to predict multiple traits at a single time point or the dynamics of individual traits over the considered time domain, respectively. We found that MegaLMM outperforms dynamicGP in terms of snapshot accuracy, while dynamicGP proved superior in terms of longitudinal accuracy. This study paves the way for careful investigation of factors that affect the capacity to predict dynamics of multiple traits from genetic markers alone.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.22.649925v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">BioMiner: A Multi-modal System for Automated Mining of Protein-Ligand Bioactivity Data from Literature</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yan, J., Zhu, J., Yang, Y., Liu, Q., Zhang, K., Zhang, Z., Liu, X., Zhang, B., Gao, K., Xiao, J., Chen, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce BioMiner, a multi-modal system designed to automatically extract protein-ligand bioactivity data from thousands to potentially millions of publications.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein-ligand bioactivity data published in literature are essential for drug discovery, yet manual curation struggles to keep pace with rapidly growing literature. Automated bioactivity extraction is challenging due to the multi-modal distribution of information (text, tables, figures, structures) and the complexity of chemical representations (e.g., Markush structures). Furthermore, the lack of standardized benchmarks impedes the evaluation and development of extraction methods. In this work, we introduce BioMiner, a multi-modal system designed to automatically extract protein-ligand bioactivity data from thousands to potentially millions of publications. BioMiner employs a modular, agent-based architecture, leveraging a synergistic combination of multi-modal large language models, domain-specific models, and domain tools to navigate this complex extraction task. To address the benchmark gap and support method development, we establish BioVista, a comprehensive benchmark comprising 16,457 bioactivity entries and 8,735 chemical structures curated from 500 publications. On BioVista, BioMiner validates its extraction ability and provides a quantitative baseline, achieving F1 scores of 0.22, 0.45, and 0.53 for bioactivity triplets, chemical structures, and bioactivity measurement with high throughput (14s/paper on 8 V100 GPUs). We further demonstrate BioMiner's practical utility via three applications: (1) extracting 67,953 data from 11,683 papers to build a pre-training database that improves downstream models performance by 3.1%; (2) enabling a human-in-the-loop workflow that doubles the number of high-quality NLRP3 bioactivity data, helping 38.6% improvement over 28 QSAR models and identification of 16 hit candidates with novel scaffolds; and (3) accelerating the annotation of the protein-ligand structures in PoseBusters benchmark with reported bioactivity, achieving a 5-fold speed increase and 10% accuracy improvement over manual methods. BioMiner and BioVista provide a scalable extraction methodology and a rigorous benchmark, paving the way to unlock vast amounts of previously inaccessible bioactivity data and accelerate data-driven drug discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.22.648951v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Assemblies of long-read metagenomes suffer from diverse errors</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Trigodet, F., Sachdeva, R., Banfield, J. F., Eren, A. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that erroneous reporting is pervasive among long-read assemblers and can take many forms, including multi-domain chimeras, prematurely circularized sequences, haplotyping errors, excessive repeats, and phantom sequences.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genomes from metagenomes have revolutionised our understanding of microbial diversity, ecology, and evolution, propelling advances in basic science, biomedicine, and biotechnology. Assembly algorithms that take advantage of increasingly available long-read sequencing technologies bring the recovery of complete genomes directly from metagenomes within reach. However, assessing the accuracy of the assembled long reads, especially from complex environments that often include poorly studied organisms, poses remarkable challenges. Here we show that erroneous reporting is pervasive among long-read assemblers and can take many forms, including multi-domain chimeras, prematurely circularized sequences, haplotyping errors, excessive repeats, and phantom sequences. Our study highlights the need for rigorous evaluation of the algorithms while they are in development, and options for users who may opt for more accurate reads than shorter runtimes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.22.649783v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">High-quality metagenome assembly from nanopore reads with nanoMDBG</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Benoit, G., James, R., Raguideau, S., Alabone, G., Goodall, T., Chikhi, R., Quince, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present nanoMDBG, an evolution of the metaM DBG HiFi assembler, designed to support newer ONT sequencing data through a novel pre-processing step that performs fast and accurate error correction in minimizer-space.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Third-generation long-read sequencing technologies, have been shown to significantly enhance the quality of metagenome assemblies. The results obtained using the highly accurate reads generated by PacBio HiFi have been particularly notable yielding hundreds of circularized, complete genomes as metagenome-assembled genomes (MAGs) without manual intervention. Oxford Nanopore Technologies (ONT) has recently improved the accuracy of its sequencing reads, achieving a per-base error rate of approximately 1-2%. Given the high-throughput, convenience and low-cost of ONT sequencing this could accelerate the uptake of long read metagenomics. However, current metagenome assemblers are optimized for PacBio HiFi data and underperform on the latest ONT data and do not scale to the large data sets that it enables. We present nanoMDBG, an evolution of the metaMDBG HiFi assembler, designed to support newer ONT sequencing data through a novel pre-processing step that performs fast and accurate error correction in minimizer-space. Across a range of ONT datasets, including a large 400 Gbp soil sample sequenced specifically for this study, nanoMDBG reconstructs up to twice as many high-quality MAGs as the next best ONT assembler, metaflye, while requiring a third of the CPU time and memory. As a result of these advances, we show that the latest ONT technology can now produce results comparable to those obtained using PacBio HiFi sequencing at the same sequencing depth.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.22.649928v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Engineering microbial consortia for distributed signal processing</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Duncker, K. E., Shende, A. R., Shyti, I., Ruan, A., D'Cunha, R., Ma, H. R., Venugopal Lavanya, H., Liu, S., Gottel, N., Anderson, D. J., Gunsch, C. K., You, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new, generalizable approach that leverages microbial consortia to distribute sensory functions and computational methods to disentangle signals.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> A critical goal in biology is deducing input signals from measurable readouts. Genetic circuits have successfully been designed to respond to specific analytes and produce quantifiable outputs; however, multiplexed signal processing remains challenging. This limitation is partially due to crosstalk, or sensors' non-specific responses to unintended signals. While strategies to achieve orthogonality are promising, they are time-intensive and context-dependent. Here, we introduce a new, generalizable approach that leverages microbial consortia to distribute sensory functions and computational methods to disentangle signals. Compartmentalizing sensor circuits within distinct populations simplifies experimental optimization by allowing individual populations to be exchanged without requiring genetic modifications. Our computational pipeline combines mechanistic modeling with machine learning to decode microbial communities' unique temporal responses and predict multiple input concentrations. We demonstrated this platform's versatility in a variety of contexts: measuring signals with high crosstalk, detecting antibiotics with natural microbial communities, and quantifying chemicals in hospital sink wastewater. Our findings highlight how combining microbial engineering with computational strategies can produce robust, scalable biosensors for diverse applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.23.650302v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AI-Based Antibody Design Targeting Recent H5N1 Avian Influenza Strains</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Santolla, N. F., Ford, C. T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show the utility of artificial intelligence in the discovery of novel antibodies against specific H5N1 strains of interest, which bind similarly to known therapeutic and elicited antibodies.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In 2025 alone, H5N1 avian influenza is responsible for thousands of infections across various animal species, including avian and mammalian livestock such as chickens and cows, and poses a threat to human health due to avian-to-mammalian transmission. There have been 70 human cases of H5N1 influenza in the United States since April 2024 and, as shown in recent studies, our current antibody defenses are waning. Thus, it is imperative to discover new therapeutics in the fight against more recent strains of the virus. In this study, we present the Frankies framework for automated antibody diffusion and assessment. This pipeline was used to automate the generation of 30 novel anti-HA1 Fv antibody fragment sequences, fold them into 3-dimensional structures, and then dock against a recent H5N1 HA1 antigen structure for binding evaluation. Here we show the utility of artificial intelligence in the discovery of novel antibodies against specific H5N1 strains of interest, which bind similarly to known therapeutic and elicited antibodies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.24.650061v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Lipid flip flop regulates the shape of growing and dividing synthetic cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lira, R. B., Dekker, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We demonstrate that lipid flip flop relaxes curvature stresses and yields more symmetrically sized buds, which leads to bud scission.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cells grow their boundaries by incorporating newly synthesized lipids into their membranes as well as through fusion of intracellular vesicles. As these processes yield trans-bilayer imbalances in lipid numbers, cells must actively flip lipid molecules across the bilayer to enable growth. Using giant and small unilamellar vesicles (GUVs and SUVs, respectively), we here recapitulate cellular growth and division under various conditions of transmembrane flip flop of lipids. By dynamically monitoring the changes in reduced volume and spontaneous curvature of GUVs that grow by fusion of many small SUVs, the morphology of these growing synthetic cells is quantified. We demonstrate that lipid flip flop relaxes curvature stresses and yields more symmetrically sized buds. Further increasing the neck curvature is shown to lead to bud scission. The mechanisms presented here offer fundamental insights into cell growth and division, which are important for understanding early protocells and designing synthetic cells that are able to grow and divide.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.23.650179v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">High-throughput engineering and modification of non-ribosomal peptide synthetases based on Golden Gate assembly</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Podolski, A., Lindeboom, T. A., Praeve, L., Kranz, J., Schindler, D., Bode, H. B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel method for efficient high-throughput generation of novel and engineered NRPS libraries utilising the XUT concept, which allows the efficient modular assembly of different natural NRPS fragments to form hybrid NRPS that produce defined peptides.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Non-ribosomal peptide synthetases (NRPS) are multimodular enzymes that produce complex peptides with diverse biological activities, potentially being used as clinical drugs. However, the pharmaceutical applications of such natural peptides often require further derivatisation and modification of the peptide backbone, mainly performed by chemical synthesis. A sustainable alternative resembles the in vivo engineering of NRPS to change and modify the enzyme properties rationally and, thus, the produced products. The novel NRPS engineering concept, the eXchange Unit Thiolation domain (XUT), allows the efficient modular assembly of different natural NRPS fragments to form hybrid NRPS that produce defined peptides. In this study, we describe a Golden Gate assembly (GGA) method for efficient high-throughput generation of novel and engineered NRPS libraries utilising the XUT concept. This method was applied to generate over 100 novel NRPS with the possibility of changing starter, elongation, and termination modules, respectively. Additionally, we applied this method for targeted modification of the xenoamicin biosynthetic gene cluster (BGC) XabABCD from Xenorhabdus doucetiae, resulting in the generation of 25 novel xenoamicin derivatives.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.23.650154v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Conformation-specific Design: a New Benchmark and Algorithm with Application to Engineer a Constitutively Active MAP Kinase</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Stern, J., Alharbi, S., Sandholu, A., Arold, S. T., Della Corte, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A general method for designing proteins with high conformational specificity is desirable for a variety of applications, including enzyme design and drug target redesign.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> AO_SCPLOWBSTRACTC_SCPLOWA general method for designing proteins with high conformational specificity is desirable for a variety of applications, including enzyme design and drug target redesign. To assess the ability of algorithms to design for conformational specificity, we introduce MotifDiv, a benchmark dataset of 200 conformational specificity design challenges. We also introduce CSDesign, an algorithm for designing proteins with high preference for a target conformation over an alternate conformation. On the MotifDiv benchmark, CSDesign designs protein sequences that are predicted to prefer the target conformation. We apply this method in vitro to redesign human MAP kinase ERK2, an enzyme with active and inactive conformations. Out of two designs for the active conformation, one increased activity sufficiently to retain activity in the absence of activating phosphorylations, a property not present in the wild type protein.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.23.650138v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Harnessing Coding Sequence Cleavage: Theophylline Aptazymes as Portable Gene Regulators in Bacteria</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yurdusev, E., Nicole, Y., Pan Du, F., Suslu, N. E., Perreault, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a novel "semi-trans" aptazyme-based system for gene regulation in bacteria, expanding the range of gene regulation tools, and demonstrate ligand-controlled regulation of tetracycline resistance and nickel sensitivity.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Nucleic acid-based regulatory elements capable of modulating gene expression in response to specific molecular cues have gained increasing attention in synthetic biology. These systems, which include riboswitches, allosteric DNAzymes, and aptazymes, function as Gene Expression Nucleic Allosteric actuators (GENAs) by coupling molecular recognition with genetic regulation. Their versatility can enable applications in diagnostics, therapeutics, and metabolic engineering. This study presents a novel "semi-trans" aptazyme-based system for gene regulation in bacteria, expanding the range of GENAs. The system employs theophylline-responsive hammerhead ribozyme aptazymes positioned in the 5' UnTranslated Region (UTR), designed to cleave within the coding sequence of the target gene, thereby modulating gene expression in a ligand-dependent manner. Using the tetA gene in Escherichia coli (E. coli) as a proof of concept, we demonstrate ligand-controlled regulation of tetracycline resistance and nickel sensitivity. The system's effectiveness is validated through in vitro cleavage assays and in vivo phenotypic studies in two E. coli strains, highlighting its portability across genetic backgrounds. Furthermore, the ability to design multiple aptazymes targeting different coding regions enables complex and fine-tuned regulation. This work broadens the landscape of synthetic gene regulation tools, facilitating the development of new aptazymes based on this approach.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.24.650125v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

