<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Sunday, 11 May 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An evolutionary approach to predict the orientation of CRISPR arrays</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fehrenbach, A., Mitrofanov, A., Alkhnbashi, O., Backofen, R., Baumdicker, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce CRISPR-evOr, a method that leverages evolutionary history to predict the acquisition orientation for (a group of)CRISPR arrays by reconstructing and comparing the likelihood of their evolutionary history with respect to both possible acquisition orientations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> CRISPR-Cas is a defense system of bacteria and archaea against phages. Parts of the foreign DNA, called spacers, are incorporated into the CRISPR array which constitutes the immune memory. The orientation of CRISPR arrays is crucial for analyzing and understanding the functionality of CRISPR systems and their targets. Several methods have been developed to identify the orientation of a CRISPR array. To predict the orientation, different methods use different features such as the repeat sequences between the spacers, the location of the leader sequence, the Cas genes, or PAMs. However, those features are often not sufficient to predict the orientation with certainty, or different methods disagree. Remarkably, almost all CRISPR systems have been found to insert spacers in a polarized manner at the leader end of the array. We introduce CRISPR-evOr, a method that leverages the resulting patterns to predict the acquisition orientation for (a group of) CRISPR arrays by reconstructing and comparing the likelihood of their evolutionary history with respect to both possible acquisition orientations. The new method is independent of Cas type, leader existence and location, and transcription orientation. CRISPR-evOr is thus particularly useful for arrays that other CRISPR orientation tools cannot predict confidently and to verify or resolve conflicting predictions from existing tools. CRISPR-evOr currently confidently predicts the orientation of 28.3% of the arrays in the considered subset of CRISPRCasdb, which other tools like CRISPRDirection and CRISPRstrand cannot reliably orient. As our tool leverages evolutionary information we expect this percentage to grow in the future when more closely related arrays will be available. Additionally, CRISPR-evOr provides confident decisions for rare subtypes of CRISPR arrays, where knowledge about repeats and leaders and their orientation is limited.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.09.653049v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MettleRNASeq: Complex RNA-Seq Data Analysis and Gene Relationships Exploration Based on Machine Learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Salles, S., Brustolini, O., Ciapina, L. P., Ocana, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present MettleRNASeq, a robust alternative for complex RNA-Seq data analysis that integrates machine learning techniques - a tailored classification approach, association rule mining, and complementary correlation analysis - to accurately identify key genes that distinguish experimental conditions and emphasize gene relationships.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Typical differential gene expression (DGE) analysis might struggle when RNA-Seq datasets possess characteristics that hinder the power of statistical analyses and the obtention of accurate conclusions, such as a limited number of replicates and high variability. We present MettleRNASeq, a robust alternative for complex RNA-Seq data analysis that integrates machine learning techniques - a tailored classification approach, association rule mining, and complementary correlation analysis - to accurately identify key genes that distinguish experimental conditions and emphasize gene relationships. This approach provides full control over critical parameters, making it versatile for transcriptomic analyses and enhancing the comprehension of disease mechanisms, treatments, and their progression. MettleRNASeq was applied for the analysis of complex radiotherapy datasets. While popular DGE tools showed an inability to accurately differentiate the distinct radiotherapy treatments, MettleRNASeq effectively and consistently indicated relevant genes for condition discrimination and identified meaningful gene relationships related to radiotherapy, highlighting condition-specific and shared gene relationships. MettleRNASeq is implemented as an R package and available on GitHub at https://github.com/SamellaSalles/MettleRNASeq.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.06.652387v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Human Messenger RNA Harbors Widespread Noncoding Splice Isoforms</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shi, Q., Li, H., Song, J., Wu, H., Zeng, Y., Jiang, J., Li, S., Chen, Z., He, X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We discovered 15,836 noncoding splice isoforms in human mRNA from a protein-coding gene, and highlight the dual characters of mRNA including protein-translation isoforms and nonc coding splices.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Pre-messenger RNA transcribed from canonical protein-coding genes is frequently alternatively spliced into diverse mature mRNA isoforms, thus translating into various protein products. However, it remains unknown whether human mRNA from a protein-coding gene harbors noncoding splice isoforms in the genome. Herein, we discovered 15,836 mRNA noncoding splice isoforms across 7,298 protein-coding genes in human. mRNA noncoding isoforms are mainly produced by alternative splicing and polyadenylation, which display tissue-specific distributions and involve in RNA processing pathways. Notably, mRNA noncoding isoforms are frequently upregulated in cancer. Differentially expressed mRNA noncoding isoforms are associated with the cancer hallmarks and can independently predict patient survival. These findings unraveled human mRNA harbors widespread noncoding splice isoforms, and highlight the dual characters of mRNA including protein-translation isoforms and noncoding splice isoforms, providing a new dimension to our understanding of mRNA functional property.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.06.652571v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Unraveling Tyrosine-Kinase Inhibitor Resistance in NSCLC Cells via Same-cell Measurement of RNA, Protein, and Morphological Responses</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dien, V., Thompson, C., Rammel, T., Roquero, J., Krivit, M., Yeo, G., Honigfort, D., Bhadha, P., Brown-Greeves, C., Dawood, M., Elango, K., Kwon, H. J., Lopez, T., Lo, A., Manjunath, M., Ren, P., Song, B., Sundkamp, M., Tippana, R., Van IJcken, W., Wong, J., Yao, J., Zhou, D., Arslan, S., Kellinger, M., Kia, A., Kruglyak, S., Previte, M., He, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A single-cell, multiomic analysis of drug-sensitive and drug-resistant NSCLC cells using the Teton detection assay on the AVITI24 platform to measure concurrently RNA, protein, and cellular morphology in the same cells.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Non-small cell lung cancer (NSCLC) frequently develops resistance to tyrosine kinase inhibitors (TKIs), limiting the long-term success of targeted therapies. A deeper understanding of resistance mechanisms at the molecular and cellular levels may enable the development of more effective treatment strategies. Here, we applied the Teton detection assay on the AVITI24 platform to measure concurrently RNA, protein, and cellular morphology in NSCLC cell lines treated with the TKIs gefitinib and osimertinib. This single-cell, multiomic analysis revealed distinct expression and morphological profiles between drug-sensitive and resistant cells, including differences in MAPK-related pathway activity. Stratifying responses at the single-cell level uncovered subtle responses not detectable in bulk measurements. We identified CDK4/6 activity as a route of cell survival under TKI treatment and demonstrated that co-treatment with the CDK4/6 inhibitor palbociclib enhanced TKI efficacy. The ability to measure multiomics and cellular morphology in the same cells opens new avenues for future studies aimed at improving personalized treatment strategies in NSCLC and overcoming the obstacles posed by drug resistance.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.06.652479v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Alignment-free integration of single-nucleus ATAC-seq across species with sPYce</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zeitler, L., Berthelot, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a cross-species snATAC-seq integration method that relies on sequence composition similarities through k-mer histograms to uncover conserved cellular identities and compare gene regulatory evolution across species.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Changes in gene regulation largely contribute to differences in cellular identities and phenotypes between species. Single-nucleus assays for transposase-accessible chromatin with sequencing (snATAC-seq) are an efficient strategy to identify putative gene regulatory elements active in a given tissue at single-cell resolution, and have unprecedented potential to provide new insight into evolutionary divergence of regulatory programs. However, no dedicated framework exists to integrate and compare snATAC-seq data across species, whilst methods designed for single-cell gene expression data have serious limitations. Here, we present sPYce, a cross-species snATAC-seq integration method that relies on sequence composition similarities through k-mer histograms. In contrast to other approaches, sPYce does not require orthologous genes or genome alignments to anchor data from different species. Instead, it uses similarity in non-coding regulatory sequence motifs to uncover conserved cellular identities. sPYce can embed datasets from multiple species into the same mathematical space and permits further downstream analysis steps, such as dimensionality reduction, visualisation, clustering, cell type annotation transfer, and motif enrichment. We benchmarked sPYce against existing approaches on two publicly available datasets spanning more than 160 million years of evolution, demonstrating that it successfully uncovers conserved cellular programs whilst preserving biologically relevant species-specific differences. sPYce also implements a significance test for divergence of regulatory motifs between species. By comparing cerebellar development in mouse and opossum, we discovered regulatory divergence in granule cell differentiation programs, particularly driven by Nuclear Factor 1 (NF1). Our extensive evaluation suggests that sPYce is the first easy-to-use, alignment-free cross-species snATAC-seq integration approach, opening novel perspectives to compare gene regulatory evolution across species.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.07.652648v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">mtgRNA-db: An annotated database of multi-target CRISPR-Cas9 guide-RNAs in the human genome</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Modrek, A. S., Huynh, K., Do, C., Karp, J., Deng, Y., Zhang, Z., Ezhilarasan, R., Valor, B., Cova, G., Skok, J. A., Sulman, E. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We generate a database of guide-RNAs that target multiple sites in the genome at once.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> CRISPR-Cas9 guide-RNA design tools can be used to identify guide-RNAs that target single human genomic loci. However, these approaches limit their effect to a single locus. Here, we generate a database of potential individual guide-RNAs that target multiple sites in the genome at once. All guide-RNAs in this database are curated with on- and off-target quantification and enrichment scores for genomic elements such as gene elements, regulatory elements, repetitive elements, and transcription factor motifs. This tool enables rationale mass targeting of genomic loci with a single guide for functional studies. We created a web-app for user-friendly guide-RNA selection (https://modreklab.shinyapps.io/guiderna/).</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.07.652656v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MetagenBERT: a Transformer Architecture using Foundational DNA Read Embedding Models to enhance Disease Classification</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Roy, G., Belda, E., Prifti, E., Chevaleyre, Y., Zucker, J.-D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present MetagenBERT, a Transformer-based framework to embed metagenomes that relies on the foundational models DNABERT-2 and DNAberT-S for the embedding of DNA sequencing reads.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Microbial ecosystems constitute complex yet information-rich environments whose characterization is crucial for understanding host health and disease. Among them, the human gut microbiome has emerged as a key "super-integrator", owing to its dense interactions with host physiology and its established associations with a wide spectrum of pathologies. Driven by advances in high-throughput sequencing technologies and the continuous decline in associated costs, metagenomic studies have expanded exponentially, generating massive amounts of sequencing data and opening new avenues for data-driven disease modeling. Conventional approaches to microbiome analysis predominantly rely on the alignment of DNA sequencing reads against reference databases to infer microbial composition and profiling at the species level. While effective, these methods are inherently constrained by reference bias and limited taxonomic resolution. Recent advances in artificial intelligence - particularly in Natural Language Processing (NLP) offer new methodological perspectives for metagenomic data representation. In this study, we present MetagenBERT, a Transformer-based framework to embed metagenomes that relies on the foundational models DNABERT-2 and DNABERT-S for the embedding of DNA sequencing reads. Our approach encodes gut microbiome metagenome in a taxonomy-agnostic manner, enabling direct downstream application to disease classification tasks. We demonstrate that MetagenBERT reaches similar performance to state-of-the-art abundance-based models for cirrhosis prediction and surpasses them in the more challenging context of type 2 diabetes. Furthermore, we introduce an alternative representation of metagenomes based on read-level embeddings aggregated into abundance vectors, demonstrating their complementarity with conventional species-level abundance metrics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.06.652444v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">eLaRodON: identification of large genomic rearrangements in Oxford Nanopore sequencing data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mikheeva, R., Koryukov, M., Leonov, P., Mukhametdinova, V., Borobova, V., Laktionov, P., Tarkhov, A., Filipenko, M., Kechin, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present eLaRodON, a novel tool for identifying large genomic rearrangements in Oxford Nanopore sequencing data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Long-read sequencing enables more accurate detection of large genomic rearrangements (LGRs) compared to short-read technologies. However, existing tools for LGR calling continue to evolve and require further optimization. In this study, we present eLaRodON, a novel tool for identifying LGRs in Oxford Nanopore sequencing data. Using publicly available datasets--including Mycobacterium tuberculosis genomes with extensive complete genome references and the human cell line NA12878--we demonstrate that eLaRodON outperforms existing tools (NanoSV, Sniffles2, NanoVar, and SVIM), achieving an AUC of 0.61 (vs. 0.13-0.43) for M. tuberculosis and 0.86 (vs. 0.40-0.72) for the human genome. Validation against gold-standard LGR sets (derived from de novo Flye assemblies and Mauve alignments) confirmed the tool's high accuracy. Notably, we identified recurrent false-positive LGR patterns across diverse datasets (M. tuberculosis, NA12878, and {lambda} phage controls). Orthogonal validation by targeted NGS and Sanger sequencing yielded a variant verification rate of 67-100% for different LGR types--including those supported by a single read. These results represent a significant advancement in LGR detection accuracy, with implications for genomic research and clinical applications. eLaRodON is an open-source program, and its code can be freely accessed on GitHub: https://github.com/aakechin/eLaRodON/.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.07.652628v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Expanding the definition of MHC Class I peptide binding promiscuity to support vaccine discovery across cancers with CARMEN</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Waleron, M. M., Kallor, A. A., Palkowski, A., Daghir-Wojtkowiak, E., Borole, P., Kocikowski, M., Polom, K., Marino, F., Monterde, B., Mastromattei, M., Venditti, D., Stares, M., The KATY Consortium,, Singh, A., Hupp, T., Battail, C., Laird, A., Pesquita, C., Zapata, L., Symeonides, S. N., Rajan, A., Zanzotto, F. M., Alfaro, J. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We extend the concept of promiscuity in T-cell antigen landscapes to encompass peptides, MHC alleles, individuals, populations, and genomic regions, with a focus on cancer.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Promiscuity in T-cell antigen landscapes refers to the dual flexibility of peptides binding multiple MHC alleles and MHC alleles presenting diverse arrays of peptides. By understanding how neoantigens are shared across varied HLA backgrounds, promiscuity analysis can inform the selection of cancer-vaccine targets that reach a wider segment of the population and help refine patient stratification for diverse immunotherapies. We expand the concept of promiscuity to encompass peptides, MHC alleles, individuals, populations, and genomic regions. Our CARMEN database release harmonizes data from 72 publications (2,323 samples) across tissue types, with a focus on cancer. Using Gibbs clustering and dimensionality reduction (UMAP), we systematically map promiscuity and immunological versatility across these biological levels. Gene and mutation analysis reveals recurrent cancer mutations in highly promiscuous genomic regions, highly mutated cancer genes that avoid presented regions, and sheds light on genomic regions important to response to immunotherapy.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.08.651510v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">GWAs reveals SUBER GENE1-mediated suberization via Type One Phosphatases</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Han, J.-P., Lefebvre-Legendre, L., Capitao, M. B., Gully, K., Shukla, V., Wu, Y., Nawrath, C., Barberon, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify a previously unknown regulatory module linking suberin formation to TOPP activity and ABA signaling and provide a framework for improving plant stress resilience through targeted manipulation of root barrier properties.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Suberin deposition in the root endodermis is critical for plant nutrient acquisition and environmental adaptation. Here, we used an unbiased forward genetic approach based on natural variation across 284 Arabidopsis thaliana accessions to identify novel regulators of suberization. This screen revealed striking diversity in suberin levels and patterns, uncovering broader roles for suberin beyond those observed in the reference accession Col-0. A genome-wide association study pinpointed SUBER GENE1 (SBG1), a previously uncharacterized gene encoding a 129-amino acid protein, as a key regulator of suberin deposition. SBG1 acts through physical interaction with type one protein phosphatases (TOPPs) via conserved SILK and RVxF motifs. Disrupting this interaction abolishes SBG1 function, while topp mutants exhibit enhanced endodermal suberization, mirroring SBG1 overexpression. Our findings uncover a previously unknown regulatory module linking suberin formation to TOPP activity and ABA signaling and provide a framework for improving plant stress resilience through targeted manipulation of root barrier properties.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.06.652434v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Ultrasmall chemogenetic tags with group-transfer ligands</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Vedagopuram, S., Sindi, S. H., Chaudhary, S. K., Pergu, R., Singh, P., Karaj, E., Fung, J. E., Deb, A., DeCarlo, S. J., Mercer, J. A. M., Yamada, K., Rodriguez, D., Liu, D. R., Choudhary, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We report two ultrasmall chemogenetic tags (mgTag and cTag) of 36 and 50 amino acids that, to the best of our knowledge, are the smallest reported.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Chemogenetic tags are valuable tools for studying functions of a given protein-of-interest (POI) lacking small-molecule ligands, but most tags are too large for several POIs. Here, we report two ultrasmall chemogenetic tags (mgTag and cTag) of 36 and 50 amino acids (aa) that, to the best of our knowledge, are the smallest reported. These tags exhibit transferase-type reactivity with their ligands to append any moiety of interest to the tag. cTag utilizes an engineered C1 domain-bearing cysteine that undergoes group-transfer reaction with its ligand. Likewise, mgTag utilizes an engineered zinc-finger domain-bearing cysteine that undergoes group-transfer reaction with its molecular-glue ligand in the presence of cereblon (CRBN). We applied these tags in the context of cell signaling and proximity induction. While the fusion of HaloTag (297 aa) to the KRASG12D (188 aa) disrupted its ability to activate the growth-signaling pathway, fusion of mgTag or cTag did not. Group-transfer of BRD4 binder to tags appended to Abelson kinase (ABL) induced proximity between ABL and BRD4, resulting in the phosphorylation of the latter. Deletion of the transferase-type reactivity reduced phosphorylation levels, suggesting that proximity-inducing chimeras with group-transfer design may be more efficacious in certain scenarios. We envision these ultrasmall tags to have wide-ranging applications, including in basic science, biotechnology, and medicine.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.10.653252v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">De Novo Protocell Membrane Formation Fueled by Primitive Metabolites</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Khanal, S., Fracassi, A., Harjung, A., Burkart, M., Devaraj, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We demonstrate the de novo generation of lipid bilayers in the absence of any preexisting membranes, membrane-bound proteins, or lipid nanostructure templates.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Lipid membranes define cell boundaries, acting as gatekeepers for transport and signaling. A central principle in biology is that all cellular membranes descend from a common ancestral membrane, as they cannot be generated in the absence of preexisting lipid structures. It is thus unclear how the first protocell membranes originated from membrane-less precursors. Here we demonstrate the de novo generation of lipid bilayers in the absence of any preexisting membranes, membrane-bound proteins, or lipid nanostructure templates. Using acetate as a two-carbon precursor, lipid tails are constructed by soluble enzymes and spontaneously conjugate to cysteine backbones, forming diacyl lipids that assemble into vesicles. Pore-forming peptides facilitate precursor transport into vesicles, allowing the continuous generation of new lipids. Formation of glycolipid membranes creates compartments that can maintain proton gradients. The de novo formation of membrane bilayers using primitive chemical building blocks may have been an intermediary step between primitive prebiotic biochemistry and the emergence of cellular life.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.10.653202v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multiplexed, scalable analog recording of gene regulation dynamics over weeks using intracellular protein tapes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zheng, L., Yan, Y., Zhou, B., Lim, J., Shi, D., An, B., Ko, B., Klyder, E., Pitchiaya, S., Cai, D. J., Boyden, E. S., Wei, D., Lio, P., Linghu, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present CytoTape-a genetically encoded, modular, and scalable analog recorder for continuous, multiplexed in situ recording of gene regulation dynamics over multiple days and weeks at single-cell resolution.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Gene expression is constantly regulated by gene regulatory networks that consist of multiple regulatory components to mediate cellular functions. An ideal tool for analyzing gene regulation processes would provide simultaneous measurements of the dynamics of many components in the gene regulatory network, but existing methodologies fall short of simultaneously tracking the dynamics of components over long periods of time. Here, we present CytoTape-a genetically encoded, modular, and scalable analog recorder for continuous, multiplexed in situ recording of gene regulation dynamics over multiple days and weeks at single-cell resolution. CytoTape consists of a flexible, thread-like, elongating intracellular protein self-assembly engineered via AI-guided rational design. Gene regulation dynamics, together with timestamps for reconstruction of the continuous time axis, are directly encoded via distinct molecular tags distributed along single CytoTape assemblies in live cells, to be readout at scale after fixation via standard immunofluorescence imaging. CytoTape recorders are modularly designed to record gene expression driven by a variety of activity-dependent promoters. We demonstrated the utility of CytoTape in mammalian embryonic kidney cells, cancer cells, glial cells, and neurons, achieving simultaneous recording of five cell plasticity-associated transcription factor activities and immediate early gene expression levels, namely CREB, c-fos, Arc, Egr1, and Npas4 activities, within single cells in a spatiotemporally scalable manner. CytoTape revealed complex waveforms and nonlinear temporal couplings among these cellular activities, enabling investigations of how gene regulation histories and intrinsic signaling states shape transcriptional logics. We envision CytoTape to have broad applications in both basic and disease-related cell biology research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.10.653182v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">De novo production of prenylnaringenin compounds by a metabolically engineered Escherichia coli</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gomes, D., Rodrigues, J. L., Scrutton, N. S., Rodrigues, L. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We validate a biosynthetic pathway for de novo production of PN compounds using E. coli as a chassis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Prenylnaringenin (PN) compounds, namely 8-prenylnaringenin (8-PN), 3'-prenylnaringenin (3'-PN), and 6-prenylnaringenin (6-PN), are reported to have several interesting bioactivities. This study aimed to validate a biosynthetic pathway for de novo production of PN in Escherichia coli. A previously optimized E. coli chassis capable of efficiently de novo producing naringenin was used to evaluate eleven prenyltransferases (PTs) for the production of PN compounds. As PT reaction requires dimethylallyl pyrophosphate (DMAPP) as extended substrate that has limited availability inside the cells, clustered regularly interspaced short palindromic repeats (CRISPR) and CRISPR-associated protein 12a (Cas12a) (CRISPR-Cas12a) was used to construct ten boosted DMAPP-E. coli strains. All the PTs, in combination with the naringenin biosynthetic pathway, were tested in these strains. Experiments in 96-well deep well plates identified twelve strains capable of producing PN. E. coli M-PAR-121 with the integration of the 1-deoxy-D-xylulose-5-phosphate synthase (DXS) gene from E. coli (EcDXS) into the lacZ locus of the genome (E. coli M-PAR-121:EcDXS) expressing the soluble aromatic PT from Streptomyces roseochromogenes (CloQ) and the naringenin biosynthetic pathway was selected as the best producer strain. After optimizing the production media in shake flasks, 160.57 M of 3'-PN, 4.4 M of 6-PN, and 2.66 M of 8-PN were obtained. The production was then evaluated at the bioreactor scale and 397.57 M of 3'-PN (135.33 mg/L) and 25.61 M of 6-PN (8.72 mg/L) were obtained. To the best of our knowledge, this work represents the first report of de novo production of PN compounds using E. coli as a chassis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.09.653005v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

