<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Friday, 22 August 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">GNODEVAE: a graph-based ODE-VAE enhances clustering for single-cell data.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zeyu Fu, Chunlin Chen, Song Wang, Junping Wang, Shilei Chen</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Single-cell RNA sequencing analysis with high dimensionality, sparsity, and complex topological relationships between cells.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell RNA sequencing analysis faces critical challenges including high dimensionality, sparsity, and complex topological relationships between cells. Current methods struggle to simultaneously preserve global structure, model cellular dynamics, and handle technical noise effectively.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40841610/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Liquid biopsy diagnostics for non-small cell lung cancer via elucidation of tRNA signatures.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhuokun Feng, Masaki Nasu, Gehan Devendra, Ayman A Abdul-Ghani, Owen T M Chan, Jeffrey A Borgia, Zitong Gao, Hanqiu Zhang, Yu Chen, Ting Gong, Gang Luo, Hua Yang, Lang Wu, Yuanyuan Fu, Youping Deng</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A non-invasive liquid biopsy approach for early lung cancer diagnosis .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Lung cancer, particularly non-small cell lung cancer (NSCLC), accounts for about 85% of all lung cancer cases and remains a major global health challenge. Traditional diagnostic methods, such as chest X-rays and low-dose CT scans, have limitations, including high false-positive rates, radiation risks, and the invasiveness of tissue biopsies. This study aims to develop a non-invasive liquid biopsy approach for early NSCLC diagnosis.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40841456/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PDP-Miner: an AI/ML tool to detect prophage tail proteins with depolymerase domains across thousands of bacterial genomes.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jeff Gauthier, Irena Kukavica-Ibrulj, Roger C Levesque</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Phage tail proteins can be used to develop a new class of antibiotics that are resistant to antibiotics and can also be used as prophages.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Antibiotic resistance is predicted to become the leading cause of human mortality by 2050. Despite this, no other major antibiotic class has been approved for medical use since 1987. Nevertheless, phage tail proteins offer a promising alternative, given their depolymerase activity toward outer membrane polysaccharides. Several pathogenic bacteria harbor prophages, thus making these prophages' molecular target already known.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40839215/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Unlocking the microbial mechanisms in fermented bean products: advances and future directions in multi-omics and machine learning approaches.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Li Jiang, Yunhao Lu, Qiang He</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> This review examines the key microbial communities involved in the fermentation of various FBPs and explores the application of multi-omics technologies, such as genomics and metabolomics, to elucidate the microbiological mechanisms influencing product quality.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Fermented bean products (FBPs) are integral to diverse global culinary traditions, prized for their distinctive flavors, nutritional benefits, and functional properties. Despite their importance, the interplay between microbial fermentation and the quality of FBPs remains incompletely understood, necessitating further investigation. This review examines the key microbial communities involved in the fermentation of various FBPs and explores the application of multi-omics technologies, such as genomics and metabolomics, to elucidate the microbiological mechanisms influencing product quality. It highlights the potential of integrating bioinformatics and machine learning to identify critical fermentation pathways and their impact on the final product. Moreover, the review anticipates the role of emerging smart technologies in enhancing the quality and efficiency of FBPs. By applying multi-omics approaches, it is possible to pinpoint core microbial consortia linked to high-quality FBPs, offering new avenues for engineering synthetic microbial communities. Such strategies may permit more precise control over fermentation processes, optimizing both the safety and consistency of FBPs. Finally, the review outlines future research directions focused on exploiting technological innovations to improve the quality, sustainability, and safety of FBPs.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40838853/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">COMPARATIVE ANALYSIS OF THE RNA-CHROMATIN INTERACTOME DATA</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ryabykh, G., Nikolskaya, A., Garkul, L., Mironov, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> The article presents a comparative analysis of data from two types of experiments to study RNA-chromatin interactions, and shows that the overrepresentation of contacts in ATA data compared to the expected value obtained from RNA sequencing data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Two types of experiments are used to study RNA-chromatin interactions: a search for the interactome of individual RNAs ("one-to-all" or OTA) and a genome-wide search for contacts of all RNAs ("all-to-all" or ATA). The article presents a comparative analysis of these data. The concept of "chromatin potential" is introduced, which shows the overrepresentation of contacts in ATA data compared to the expected value obtained from RNA sequencing data. It is shown that the use of this characteristic allows one to select RNAs that interact with chromatin more specifically. The consistency of the replicas is studied, and data from experiments of different types are compared. As a result of the analysis, estimates of the completeness and specificity of the contacts found are obtained.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.16.668492v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SnakeAltPromoter Facilitates Differential Alternative Promoter Analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tan, J., Wu, Y., Barve, R., Li, F., Payne, P., Kong, N., Jin, S. C., Head, R., Sun, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a reproducible framework that pairs comparative method evaluation with scalable differential alternative promoter analysis, enabling high-throughput mining of promoter-level regulation from public RNA-seq repositories.</p><p style="font-size: 16px; text-align: justify;">Background: Alternative promoter usage regulates isoform diversity in mammals, playing critical roles in development, disease, and cellular reprogramming. While Cap Analysis of Gene Expression (CAGE) enables precise transcription start site mapping, its high cost and limited coverage hinder scalability. In contrast, RNA-seq is abundant across biological contexts; several algorithms (ProActiv, Salmon, DEXSeq) infer promoter activity from these data, yet no unified, reproducible framework exists to execute, benchmark, and compare them or to scale alternative promoter analyses across large compendia. Results: We developed snakeAltPromoter, an end-to-end Snakemake workflow that ingests raw FASTQ files, performs quality control and alignment, quantifies promoter activity using three complementary strategies (junction-based, transcript-based, and first-exon-based), classifies promoters into major, minor/alternative, and inactive categories, and conducts both differential promoter activity and usage analysis. Crucially, snakeAltPromoter integrates a systematic benchmarking module against matched CAGE profiles to reveal method-specific strengths and limitations. ProActiv showed the highest concordance with CAGE in promoter classification, activity and differential analysis, Salmon was robust at low coverage and intronless cases. Overall, the complete workflow recovered a majority of CAGE-validated active promoters and processed a 50 M-read RNA-seq sample in around 2h on a 32-core node, demonstrating both accuracy and scalability. Conclusions: snakeAltPromoter is, to our knowledge, the first reproducible framework that pairs comparative method evaluation with scalable differential alternative promoter analysis. It provides concrete guidance for method selection under different experimental scenarios and enables high-throughput mining of promoter-level regulation from public RNA-seq repositories. Code and example data are freely available at https://github.com/YidanSunResearchLab/SnakeAltPromoter.git.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.16.669128v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Fine scale structural information substantially improves multivariate regression model for mRNA in-vial degradation prediction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yi, S., Ali, S., Jadeja, Y., Davis, J. W., Metkar, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that base-pairing probability, in terms of log odds, provide fine-scale, orthogonal insight that complements global metrics and improves stability modeling.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The success of COVID-19 mRNA vaccines has made optimizing mRNAs for in-vial stability a key objective. However, we still lack a complete understanding of the sequence metrics that influence mRNA stability in solution. RNA secondary structure plays a central role in protecting against hydrolysis, the primary degradation pathway under storage conditions. Yet, the structural metrics that best guide stability-focused mRNA design remain unclear. Global metrics like minimum free energy and average unpaired probability have improved mRNA stability but fail to capture local structural variation relevant to degradation. We show that base-pairing probability, in terms of log odds, provide fine-scale, orthogonal insight that complements global metrics and improves stability modeling. By combining local and global features into a four-feature regression model, dubbed STRAND (Stability Regression Analysis using Nucleotide-Derived features), we achieve substantial gains in predictive performance over current methods. This compact and interpretable model provides a practical framework for designing mRNAs with enhanced in-solution stability.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.15.670605v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Interpretable deep generative ensemble learning for single-cell omics with Hydra</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wagle, M. M., Liu, C., Liu, Z., Wang, Y., Kellis, M., Patrick, E., Yang, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose Hydra, a deep generative framework based on an ensemble of variational autoencoders for effective utilization of diverse unimodal and multimodal single-cell omics data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell omics technologies enable the dissection of cellular heterogeneity, yet the high dimensionality, inherent noise, and sparsity present significant analytical challenges. As the technologies evolve to measure multiple molecular features, such as chromatin accessibility and surface protein expression, they introduce additional complexity to the integrative analysis of multimodal single-cell omics data. Here, we propose Hydra, a deep generative framework based on an ensemble of variational autoencoders for effective utilization of diverse unimodal and multimodal single-cell omics data. Hydra implements interpretable learning modules for capturing cell type-specific molecular signatures. The ensemble of such interpretable modules improves feature selection reproducibility and robust annotation of cell types in query datasets. We benchmarked Hydra on a rich repertoire of 21 datasets, including unimodal and multimodal single-cell omics data spanning multiple tissue types and technologies. Our results demonstrate that Hydra offers comparable to superior performance against several state-of-the-art methods. Finally, we highlight the utility of Hydra in learning and robustly annotating brain cellular subtypes using our previously published dataset that profiles early and late stages of Alzheimer's disease.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.15.670517v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Cytoplasmic dynamics are overlooked in single nuclei RNA-seq but can be rescued by CytoRescue, a generative AI model to recover cytoplasm enriched gene</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Deng, W., Zhao, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present CytoRescue, a novel generative AI model designed to recover attenuated cytoplasmic signals in snRNA-seq data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-nucleus RNA sequencing (snRNA-seq) generates single cell data from nuclei. It provides valuable compatibility with frozen or difficult-to-dissociate tissues while avoiding stress responses in fresh samples. However, cytoplasmic depletion inherently limits quantification of cytoplasm-enriched genes. Here, we present CytoRescue, a novel generative AI model designed to recover attenuated cytoplasmic signals in snRNA-seq data. Our results demonstrate that CytoRescue effectively restores expression of cytoplasm-enriched genes while preserving underlying gene expression signatures. Taking advantaging of the raw-in-raw-out design, CytoRescue can be easily integrated into the existing pipelines for single-cell sequencing analysis. Notably, CytoRescue successfully recovers EGF signaling pathway components, a critical cell-cell communication pathway in lung cancer, in an independent dataset. CytoRescue addresses a fundamental limitation of snRNA-seq technology, enhancing its utility for comprehensive transcriptomic profiling while maintaining the advantages of single nucleus-based approaches.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.15.670239v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SpatialFinder: A Human-in-the-Loop Vision-Language Framework for Prioritizing High-Value Regions in Spatial Transcriptomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xu, J., Jiang, M., Koga, S., Zhang, N., Huang, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> SpatialFinder, a framework that combines a biomedical vision-language model (VLM) with a human-in-the-loop optimization pipeline to predict gene expression heterogeneity and rank high-value ROIs across routine H&E tissue slides.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Sequencing an entire spatial transcriptomics slide can cost thousands of dollars per assay, making routine use impractical. Focusing on smaller regions of interest (ROIs) based on adjacent routine H&E slides offers a practical alternative, but there is (i) no reliable way to identify the most informative areas from standard H&E images alone; and (ii) limited solutions for clinicians to prioritize the microenvironment of their own interests. Here we introduce SpatialFinder, a framework that combines a biomedical vision-language model (VLM) with a human-in-the-loop optimization pipeline to predict gene expression heterogeneity and rank high-value ROIs across routine H&E tissue slides. Evaluated across four Visium HD tissue types, SpatialFinder consistently outperforms baseline VLMs in selecting regions with high cellular diversity and tumor presence, achieving up to 89% correlation with ground truth rankings. These results demonstrate the potential of human-AI collaboration to make spatial transcriptomics more cost-effective and clinically actionable.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.16.670684v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">StrainFacts accurately quantifies both endogenous and live biotherapeutic product strain abundances in simulated and clinical vaginal microbiota samples</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shih, J., Bloom, S. M., Xu, J., Mitchell, C. M., Elsherbini, J., Kwon, D. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> StrainFacts, a computational tool for inferring strains from short-read metagenomic data, can accurately assess LBP and endogenous strain abundance, abundance, and dynamics in simulated and experimental microbiota samples, supporting its utility as an analysis tool for vaginal LBP therapeutic trial data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Live biotherapeutic products (LBPs) deliver microbial strains to modulate the host microbiota in order to promote health or treat and prevent disease. Since endogenous strains are already present, accurately evaluating LBP efficacy and mechanism of action requires distinguishing administered from endogenous strains. Although computational tools exist for inferring strains from short-read metagenomic data, few have been rigorously tested in the context of LBP treatment. Here, we assess the ability of StrainFacts, a computational tool for inferring strains from short-read metagenomic data, to estimate strain abundances and genotypes of endogenous and administered strains. We performed a simulation study of a single-strain LBP trial, modeling serial samples across a range of administered strain abundance, co-occurring endogenous strains, and sequencing depths. We found that StrainFacts accurately estimated both LBP and endogenous strain abundances and genotypes within simulated samples. We further validated methods using human vaginal microbiota samples spiked with CTV-05, the Lactobacillus crispatus strain contained in the LBP LACTIN-V, which has been shown to reduce recurrent bacterial vaginosis. Our findings demonstrate that StrainFacts can robustly assess LBP and endogenous strain colonization, abundance, and dynamics in simulated and experimental microbiota samples, supporting its utility as an analysis tool for vaginal LBP therapeutic trial data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.15.670563v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Missing data in single-cell transcriptomes reveals transcriptional shifts</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chen, R., Chung, Y.-C., Kelly, B., Moore, H., Lvovs, D., Basu, S., Gueguen, P. M., Sanin, D. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new scoring approach for cell prioritization based on response to biological perturbations .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Profiling thousands of single cell transcriptomes is routine, yet cell prioritization based on response to biological perturbations is challenging and confounded by clustering, normalization and dimensionality reduction strategies. We developed a scoring approach independent of these obstacles that unbiasedly identifies distinct transcriptomes within a set based on missing data patterns, allowing cell prioritization and feature selection for downstream analysis. Our method applied to D. discoideum reveals a metabolic shift that marks the transition between the amoeboid and aggregated states of this model organism.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.15.669765v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The Limitations of TabPFN for High-Dimensional RNA-seq Analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhou, S., Agarwal, V., Gopinath, A., Kassis, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We investigate multiple approaches to adapt TabPFN for transcriptomic analysis using two benchmark datasets: Age-ARCHS4, a regression dataset derived from the ARCHS4 dataset (57,873 samples, 10,000 genes), and Inflammatory Bowel Disease, a classification dataset encomp</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Tabular Prior-Data Fitted Networks (TabPFN) demonstrate remarkable performance on small-to-medium tabular datasets through in-context learning, but struggle with high-dimensional genomic data such as RNA-seq with tens of thousands of features. We investigate multiple approaches to adapt TabPFN for transcriptomic analysis using two benchmark datasets: Age-ARCHS4, a regression dataset derived from the ARCHS4 dataset (57,873 samples, 10,000 genes), and an Inflammatory Bowel Disease (IBD) classification dataset encompassing Crohn's Disease and Ulcerative Colitis samples (2,490 samples, 10,000 genes). Our experimental design proceeds in two phases: first evaluating existing optimization methods, then testing novel adaptations including (1) self-supervised embedding learning and (2) BulkFormer integration. We demonstrate that when constrained to equal training conditions (500 features, 10,000 samples), TabPFN outperforms classical baselines like random forest and XGBoost. However, when classical methods utilize full feature sets while TabPFN adaptations attempt to handle higher-dimensional data, all TabPFN variants consistently underperform the naive baseline. Our findings reveal fundamental limitations in current approaches to adapting TabPFN for genomic applications, showing that architectural modifications paradoxically degrade performance, while intelligent metadata-based subgrouping emerges as the most effective strategy for deploying TabPFN on biological data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.15.670537v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CRISPR-Cas13d-mediated targeting of a context-specific essential gene enables selective elimination of uveal melanoma</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Stauber, D., Sosnick, L., Ma, Y., Pimcharoen, S., Lawanprasert, A., Murthy, N., Myung, D., Qi, L. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a CRISPR-Cas13d RNA-targeting therapeutic that specifically knocks down RasGRP3 mRNA for uveal melanoma and show that this approach outperforms conventional Cas9 and siRNA methods in potency without inducing permanent genomic alterations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Uveal melanoma, the most common eye cancer in adults, remains limited to surgical intervention and chemotherapy, with a dismal survival rate that has not improved in over 50 years. To address this therapeutic impasse, we systematically analyzed public gene expression, RNAi, and CRISPR knockout datasets and identified RASGRP3 as an essential gene specifically for uveal melanoma. RasGRP3 is uniquely overexpressed and essential for survival in uveal melanoma cells, but dispensable in healthy cells. RasGRP3 remains undruggable due to its intracellular localization and lack of targetable binding pockets. To overcome this, we developed a CRISPR-Cas13d RNA-targeting therapeutic that specifically knocks down RasGRP3 mRNA. This Cas13d-based therapeutic mediates selective uveal melanoma killing through two synergistic mechanisms: (i) direct silencing of the essential RasGRP3 transcript, and (ii) collateral RNA degradation triggered by the cleavage of overexpressed RasGRP3. When delivered via optimized lipid nanoparticles encoding Cas13d mRNA and guide RNA, this strategy eliminated >97% of uveal melanoma cells while sparing healthy cells, including retinal pigment epithelial cells. This approach outperformed conventional Cas9 and siRNA methods in potency without inducing permanent genomic alterations. Our findings establish an RNA-targeting therapeutic for uveal melanoma and a framework for Cas13d-based interventions against broad undruggable cancers.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.21.671629v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A PBD-dimer containing antibody drug conjugate targeting CCRL2 for high-risk MDS/AML</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Naji, N. S., Ahmedna, T., Peske, J. D., Zeng, X., Perkins, B., Thompson, Z., Nichakawade, T. D., Lee, B. S., Watson, E., Chatzilygeroudi, T., Luo, L., Paun, B., Klausner, M., An, Y., Supeanu, T., Gojo, I., Ghiaur, G., DeZern, A. E., Levis, M. J., Resar, L., Jones, R. J., Karanika, S., Paul, S., Karantanos, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed an anti-CCRL2 antibody-drug conjugate (ADC) by conjugating an antibody with the cytotoxic drug pyrrolobenzodiazepine (PBD) which causes DNA double-strand breaks leading to cancer cell death.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Patients with myelodysplastic syndrome (MDS)/acute myeloid leukemia (AML) with high-risk features including TP53 mutations and deletions have poor outcomes due to lack of effective therapies. The atypical chemokine surface receptor C-C motif chemokine receptor-like 2 (CCRL2) is overexpressed in MDS and secondary AML (sAML) compared to healthy hematopoietic cells and we recently found that TP53-mutated MDS/AML and AML with erythroid features express the highest levels of this receptor across MDS/AML subtypes. To illustrate the therapeutic potential of CCRL2 as a therapeutic target, we developed an anti-CCRL2 antibody-drug conjugate (ADC) by conjugating an anti-CCRL2 antibody with the cytotoxic drug pyrrolobenzodiazepine (PBD), which causes DNA double-strand breaks leading to cancer cell death. The anti-CCRL2 ADC demonstrated strong CCRL2-selective cytotoxicity against cell lines derived from MDS/AML patients with TP53 mutations and erythroid features, surpassing the cytotoxic effects observed with gemtuzumab and PBD-conjugated anti-CD33 and anti-CD123 ADCs. It also induced apoptosis and suppressed the clonogenicity of primary MDS/AML bone marrow samples without affecting the survival, differentiation and clonogenicity of healthy hematopoietic stem and progenitor cells. This agent also suppressed the leukemic growth of TP53-mutated MDS/AML cell line xenografts, improving mice survival and decreasing the leukemic burden in patient-derived TP53-mutated MDS/AML xenografts. In conclusion, our study introduces CCRL2 as a potential new therapeutic target in high-risk MDS/AML.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.17.670714v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Endocytic control of cell-autonomous and non-cell-autonomous functions of p53</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cacciatore, R., Basile, A., Freddi, S., Schiano Lomoriello, I., Zucca, C. R., Ciossani, G., Scietti, L., Cuomo, A., Ronzoni, S., Pelicci, S., Faretta, M., Zaccheroni, E., Pelicci, G., Matafora, V., Bachi, A., Gunby, R. H., Pece, S., Sigismund, S., Lanzetti, L., Colaluca, I. N., Di Fiore, P. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that endocytic endocytosis and exosomal p53 transfer contribute to the establishment of a tumor-suppressive microenvironment.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> NUMB is an endocytic protein with tumor suppressor activity, largely mediated by its ability to inhibit p53 degradation. This function depends on the inclusion of a short alternatively spliced exon (Ex3) in NUMB, although the mechanistic link between endocytosis and p53 regulation remains unclear. Here, we show that the Ex3-encoded sequence directs NUMB to the plasma membrane, where it forms a complex with the endocytic adaptor SNX9. This complex recruits p53 in a SNX9-dependent manner and is internalized and trafficked to multivesicular bodies, culminating in exosomal secretion, in a process requiring both SNX9 and NUMB. Exosomal p53 is taken up by recipient cells and translocated to the nucleus, where it activates p53-dependent transcriptional and phenotypic programs. These findings suggest that exosome-mediated p53 transfer may contribute to the establishment of a tumor-suppressive microenvironment.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.16.670648v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DFFB suppresses interferon to enable cancer persister cell regrowth</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Williams, A. F., Gervasio, D. A. G., Turkal, C. E., Stuhlfire, A. E., Wang, M. X., Mauch, B. E., Plawat, R., Nguyen, A. H., Paw, M. H., Hairani, M., Lathrop, C. P., Harris, S. H., Page, J. L., Hangauer, M. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that residual cancer persister cells that survive oncogene targeted therapy are growth arrested by drug stress-induced intrinsic Type I interferon (IFN) signaling.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Oncogene targeted cancer therapies can provide deep responses but frequently suffer from acquired resistance. Therapeutic approaches to treat tumours which have acquired drug resistance are complicated by continual tumour evolution and multiple co-occurring resistance mechanisms. Rather than treating resistance after it emerges, it may possible to prevent it by inhibiting the adaptive processes which initiate resistance but these are poorly understood. Here we report that residual cancer persister cells that survive oncogene targeted therapy are growth arrested by drug stress-induced intrinsic Type I interferon (IFN) signaling. To escape growth arrest, persister cells leverage apoptotic machinery to transcriptionally suppress interferon-stimulated genes (ISGs). Mechanistically, persister cells sublethally engage apoptotic caspases to activate DNA endonuclease DNA Fragmentation Factor B (DFFB, also known as Caspase-Activated DNase (CAD)) which induces DNA damage, mutagenesis, and stress response factor Activating Transcription Factor 3 (ATF3). ATF3 limits Activator Protein-1 (AP1)-mediated ISG expression sufficiently to allow persister cell regrowth. Persister cells deficient in DFFB or ATF3 exhibit high ISG expression and are consequently unable to regrow. Therefore, sublethal apoptotic stress paradoxically promotes regrowth of residual cancer cells that survive drug treatment.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.15.670603v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Enhancing B-mode-based breast cancer diagnosis via cross-attention fusion of H-scan and Nakagami imaging with multi-CAM-QUS-Driven XAI.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mondol SS, Hasan MK.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose an automated breast lesion classification network based on B-mode and/or RF ultrasound images, combined with machine-generated B- mode images, seamlessly integrated through a multi modal cross attention fusion mechanism to extract complementary information.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Objective.B-mode ultrasound is widely employed for breast lesion diagnosis due to its affordability, widespread availability, and effectiveness, particularly in cases of dense breast tissue where mammography may be less sensitive. However, it disregards critical tissue information embedded in raw radiofrequency (RF) data. While both modalities have demonstrated promise in computer-aided diagnosis (CAD), their combined potential remains largely unexplored.Approach.This paper presents an automated breast lesion classification network that utilizes H-scan and Nakagami parametric images derived from RF ultrasound signals, combined with machine-generated B-mode images, seamlessly integrated through a multi modal cross attention fusion mechanism to extract complementary information. The proposed architecture also incorporates an attention-guided modified InceptionV3 for feature extraction, a knowledge-guided cross-modality learning module for inter-modal knowledge sharing, and attention-driven context enhancement modules to improve contextual understanding and fusion with the classification network. The network employs categorical cross-entropy loss, a Multi-CAM-based loss to guide learning toward accurate lesion-specific features, and a Multi-QUS-based loss to embed clinically meaningful domain knowledge and effectively distinguishing between benign and malignant lesions, all while supporting explainable artificial intelligence principles.Main results.Experiments conducted on multi-center breast ultrasound datasets-BUET-BUSD, ATL, and OASBUD-characterized by demographic diversity, demonstrate the effectiveness of the proposed approach, achieving classification accuracies of 92.54%, 89.93%, and 90.0%, respectively, along with high interpretability and trustworthiness. These results surpass those of existing methods based on B-mode and/or RF data, highlighting the superior performance and robustness of the proposed technique.Significance.By integrating complementary RF-derived information with B-mode imaging with pseudo-segmentation and domain-informed loss functions, our method significantly boosts lesion classification accuracy-enabling fully automated, explainable CAD and paving the way for widespread clinical adoption of AI-driven breast screening.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40780251/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

