<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Saturday, 12 July 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CNA detection from spatial transcriptomics using SpaCNA</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, Z., Wang, X., Xuan, H., Jin, Z., Xi, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Spatial-aware computational framework that integrates multi-modal spatial information to improve CNA detection and tumor region identification .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Spatial transcriptomics (ST) enables genome-wide profiling of gene expression while preserving spatial context, yet accurate detection of copy number alterations (CNAs) in tumor ST data remain challenging. Here we present SpaCNA, a spatial-aware computational framework that integrates multi-modal spatial information to improve CNA detection. SpaCNA constructs adjacency graphs using spatial proximity and H&E stained image similarity, refines the raw gene expressions, and implements a Hidden Markov Random Field model for robust CNA state inference. Through extensive benchmarking on simulated data and multi-cancer cohorts, SpaCNA demonstrates superior accuracy and outperforms existing methods in CNA detection and tumor region identification. In real-world applications to breast cancer and colorectal cancer, SpaCNA reveals spatially distinct subclones with context-dependent interactions within the microenvironment. Additionally, applied to a 3D ST dataset of head and neck squamous cell carcinoma, SpaCNA uncovers clone-specific CNAs associated with therapeutic resistance biomarkers across multiple slices. By facilitating precise spatial mapping of CNAs and tumor architecture, SpaCNA significantly enhances our understanding of intratumoral heterogeneity and spatial evolutionary patterns.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.08.663688v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CAT-APP: Contamination Analysis and Tempering--An Automated Online Platform for Plasma Proteomics with Data Rescuing Capabilities</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Niu, M., Zhang, D., Zhou, Z., Zhang, J., Zhang, R., Shen, L., Wang, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce CAT-APP (Contamination Analysis and Tempering-An Automated Online Platform for Plasma Proteomics). CAT- APP tackles contamination through three key modules: multi-dimensional contamination assessment and adaptive contamination indexing, mathematical model-based contamination tempering, and data recovery evaluation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Plasma and serum proteome profiling have become central to biomarker discovery for medical research. The advent of high-throughput plasma proteomics pipelines has enabled the generation of massive datasets. However, biomarker discovery is frequently compromised by sample contamination, primarily from erythrocytes, platelets, and coagulation factors. Such contamination can lead to the false identification of contaminants as biomarkers in over 50% of studies or obscure genuine biomarkers due to contamination-induced variability. Currently, no tools are available to salvage such compromised data, and the default approach is often to discard the affected samples. To address this gap, we introduce CAT-APP (Contamination Analysis and Tempering-An Automated Online Platform for Plasma Proteomics). CAT-APP tackles contamination through three key modules: multi-dimensional contamination assessment and adaptive contamination indexing, mathematical model-based contamination tempering, and data recovery evaluation with visualization. We applied CAT-APP to 28 independent plasma proteomics datasets and found that 82% exhibited contamination issues. CAT-APP markedly improved data quality by removing false-positive biomarkers introduced by contamination and restoring true biological signals that were previously obscured in representative datasets. To promote widespread adoption, we provide CAT-APP as an freely accessible, user-friendly, and robust web platform at https://www.bloodecosystem.com/tools/CAT-APP/.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.08.663798v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Synthesis errors in oligonucleotides are propagated into the genome by homology-directed repair</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wyman, S. K., Romero, Z., Heo, S.-J., Navarrete, M., Krishnappa, N., Kohn, D. B., Martin, D. I., Walters, M. C., Boffelli, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A deep sequencing study of single-stranded oligonucleotides used as donor templates for gene editing by targeted endonuclease cleavage and homology directed repair .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-stranded oligonucleotides (ssODNs) are used as donor templates for gene editing by targeted endonuclease cleavage and homology directed repair (HDR). We probed their sequence fidelity by deep sequencing ssODNs from three manufacturers, and genomes derived from editing using these ssODNs as templates for HDR. The ssODNs carry single-nucleotide and small deletion synthesis errors in proportions differing widely among manufacturers; these are propagated into the genome by HDR; therapeutic gene correction strategies will need to this additional source of editing errors.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.08.663753v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genome sequence data and integrative RNA-Seq data of Black Soldier Fly (Hermetia illucens) (A Japanese strain)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Takeda, K., Uehara, T., Liu, C.-M., Kobayashi, T., Shimoda, M., Yokoi, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We established an H. illucens field-collected strain to develop an industrial system for recycling organic waste.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The black soldier fly, Hermetia illucens, is a dipteran insect with a distinctive black body in the adult stage. The larvae of this species possess biological features which can be used for industrial applications. We established an H. illucens field-collected strain to develop an industrial system for recycling organic waste. We constructed whole-genome sequences of the strain of H. illucens and prepared a gene set and its functional annotation data to promote research on the social implementation of H. illucens and genetically characterise this strain. In addition, time-course transcriptome data of the larvae and transcriptome expression data of multiple tissues from 6th instar larvae and male and female adults were prepared. The genome size and its N50 value were approximately 1.05 Gbp and 191 bp, respectively. The results of the validation analyses of the genome and transcriptome expression data showed that these data were reliable as reference data, suggesting that they can be used in research on the industrial usage of evolutionary H. illucens or evolutionary or comparative biology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.08.663698v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MIMIC: a flexible pipeline to register and summarize IMC-MSI experiments</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gerber, R., Griner, J., Guglietta, S., Krieg, C., Robinson, M. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present MIMIC - a reproducible, semi-automated workflow that integrates Mass Spectrometry Imaging and Imaging Mass Cytometry for joint downstream analysis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Spatial omics is transforming our ability to interrogate local tissue microenvironments by enabling spatially resolved measurement of biomolecules such as transcripts, proteins, and metabolites. However, capturing the full biological complexity of tissues often requires combining multiple modalities, which introduces both experimental as well as computational challenges. To address computational difficulties due to differences in resolution, noise levels, and available channels, we present MIMIC - a reproducible, semi-automated workflow that integrates Mass Spectrometry Imaging and Imaging Mass Cytometry for joint downstream analysis. MIMIC incorporates rigorous quality control, including registration error assessment, and supports pixel-level modeling to delineate analyte-cell type associations. We demonstrate the power of our approach with a proof-of-concept study on artificial tissue and apply it to human liver tissue affected by metabolic dysfunction-associated steatotic liver disease. Despite integration challenges, MIMIC provides a robust framework that successfully recovers known molecular associations and reveals novel spatial relationships across modalities.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.08.663623v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">resLens: genomic language models to enhance antibiotic resistance gene detection</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mollerus, M., Dittmar, K., Crandall, K. A., Rahnavard, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present resLens, a family of genomic language models (gLM) leveraging latent genomic representations for ARG detection and analysis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The pace of antibiotic resistance necessitates advanced tools to detect and analyze antibiotic resistance genes (ARGs). We present resLens, a family of genomic language models (gLM) leveraging latent genomic representations for ARG detection and analysis. Unlike alignment-based methods constrained by reference databases, resLens fine-tunes pre-trained gLMs on curated ARG datasets, achieving superior performance across several evaluation scenarios, including when ARGs exhibit dissimilar sequences and mechanisms to those in reference databases.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.08.663767v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Convergent DNA Methylation Abnormalities at Bivalent Chromatin in Human Growth Disorders</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Strauss, M. E., Takahashi, Y., Lee, J., Perez, C., Chen, X., Lee, Y., Pope, Z. S., Yun, H., Weirauch, M. T., Byun, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify DNA methylation defects as a common molecular feature and nominate dysregulated DNA methylations at bivalent chromatin as a potential driver of abnormal growth phenotypes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Loss-of-function mutations in DNMT3A, a DNA methyltransferase, or NSD1, a histone methyltransferase, cause overgrowth syndromes. Conversely, disruption of the DNMT3A domain that binds NSD1-deposited H3K36 dimethylation (H3K36me2) results in growth restriction. To investigate the molecular basis of these opposing growth outcomes, we generated isogenic human embryonic stem cells carrying growth syndrome-associated mutations in DNMT3A and NSD1. Unexpectedly, both overgrowth- and growth restriction-associated DNMT3A mutations led to DNA hypomethylation in a shared subset of active enhancers, implicating H3K36me2 in directing enhancer methylation maintenance. In contrast, bivalent chromatin--marked by both active and repressive chromatin modifications--showed divergent DNA methylation changes: hypermethylation in growth restriction-associated DNMT3A mutants and hypomethylation in overgrowth-associated DNMT3A or NSD1 loss-of-function mutants. These findings identify locus-specific DNA methylation defects as a common molecular feature and nominate dysregulated DNA methylation at bivalent chromatin as a potential driver of abnormal growth phenotypes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.08.663614v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Weighted sliced inverse regression for scalable supervised dimensionality reduction of spatial transcriptomics data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Woollard, M., Panwar, P., Harland, L., Mo, J., Zhang, H., Swarbrick, A., Gottgens, B., Ghazanfar, S., Nghiem, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Weighted sliced inverse regression (wSIR) is a low-dimensional dimensionality reduction technique that retains as much predictive power of the spatial coordinates as possible, and is applicable to multiple distinct spatial transcriptomic datasets.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> With the dramatic take-up of spatially resolved transcriptomics biotechnologies, performing spatially-aware analysis of the resulting data is crucial to maximise advances in biological understanding. Dimensionality reduction is a first step in almost any analysis of spatial transcriptomics data, regardless of whether the data is collected at the single-cell or spot level. While common approaches, such as principal component analysis, aim to identify low-dimensional scores that preserve total variances of gene expression features, such variances do not usually correspond to biologically relevant spatial variation. To this end, we have developed weighted sliced inverse regression (wSIR), a sufficient dimension reduction technique that performs dimensionality reduction and retains as much predictive power of the spatial coordinates as possible. As a linear dimensionality reduction approach, wSIR is applicable to multiple distinct spatial transcriptomic datasets, and is extremely scalable due to the algorithm and our Rcpp implementation, with over 100,000 cells processed in under 2 minutes on a standard laptop. The feature loadings are interpretable, and new non-spatial data can be projected into the wSIR low-dimensional space for further downstream analyses. We examine wSIR's performance through benchmarking and demonstrate its capability of biological discovery through two case studies in breast cancer and early embryonic development.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.08.663651v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Cell Type Enrichment Analysis Tool for Brain DNA Methylation Data (CEAM)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Muller, J., Laroche, V., Imm, J., Weymouth, L., Harvey, J., Smith, A. R., van den Hove, D., Lunnon, K., Cavill, R., Pishva, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce CEAM (Cell-type Enrichment Analysis for Methylation), a robust and interpretable framework for cell type enrichment analysis in DNA methylation data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> DNA methylation signatures are highly cell type-specific, yet most epigenome-wide association studies (EWAS) are performed on bulk tissue, potentially obscuring critical cell type-specific patterns. Existing computational tools for detecting cell type-specific DNAm changes are often limited by the accuracy of cell type deconvolution algorithms. Here, we introduce CEAM (Cell-type Enrichment Analysis for Methylation), a robust and interpretable framework for cell type enrichment analysis in DNA methylation data. CEAM applies over-representation analysis with cell type-specific CpG panels from Illumina EPIC arrays derived from nuclei-sorted cortical post-mortem brains from neurologically healthy aged individuals. The constructed CpG panels were systematically evaluated using both simulated datasets and published EWAS results from Alzheimer''s disease, Lewy body disease, and multiple sclerosis. CEAM demonstrated resilience to shifts in cell type composition, a common confounder in EWAS, and remained accurate across a wide range of differentially methylated positions, underscoring its flexibility. Application to existing EWAS findings generated in neurodegenerative diseases revealed enrichment patterns concordant with established disease biology, confirming biological relevance of the enrichment tool. The workflow is publicly available as an interactive Shiny app (https://um-dementia-systems-biology.shinyapps.io/CEAM/) enabling rapid, interpretable analysis of cell type-specific DNAm changes from bulk EWAS.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.08.663671v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MicrobiomePrime: A primer pair selection tool for microbial source tracking validated on a comprehensive collection of animal gut and fecal waste microbiomes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zlender, T., Brezocnik, L., Podgorelec, V., Rupnik, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present MicrobiomePrime, a tool for identifying primer pairs for amplification of source-associated markers by analyzing amplicon sequences.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Identifying primer pairs for use in microbial source tracking (MST) can be challenging. The primer pairs must amplify markers with high sensitivity and specificity for a particular microbiota source, while factors such as melting temperature, secondary structure formation, and amplicon length must be carefully considered to ensure efficient amplification. To accomplish this, previous MST studies relied on complex, multi-step analysis, which was labor-intensive and time-consuming. In this paper, we present MicrobiomePrime, a tool for identifying primer pairs for amplification of source-associated markers by analyzing amplicon sequences. Our approach is based on splitting amplicon sequences from target sources into K-mers, which are treated as potential primers in an in silico PCR. As proof of concept, we applied the tool to identify markers associated with pig feces, generating thousands of primer pairs. Among nine primer pairs selected for laboratory validation, six primer pairs exhibited in vitro specificity higher than 93 %. Among them, four were 100 % specific. While originally developed for MST, MicrobiomePrime has broader applicability, enabling the design of primers targeting any molecular marker based on amplicon sequencing data. MicrobiomePrime is available on GitHub: https://github.com/tanjazlender/MicrobiomePrime.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.08.663663v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An efficient beet severe curly top virus-based VIGS vector in Beta vulgaris</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nie, Z., Guo, Z., Qin, X., Liu, Q., Zhou, M., Ye, J., Zhang, Z., Han, C., Wang, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We generated an infectious clone of beet sever curly top virus and engineered it into a VIGS vector that can induce strong and persistent bleached PDS-silencing phenotype in 8 sugar beet varieties.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Sugar beet (Beta vulgaris L.) is one of the most important sugar crops worldwide. However, studies on sugar beet gene functions are still lagging compared to other crops due to the lack of an effective genetic manipulation. In this study, we generated an infectious clone of beet sever curly top virus (BSCTV) and engineered it into a VIGS vector. BSCTV-BvPDS can induce strong and persistent bleached PDS-silencing phenotype in 8 tested sugar beet varieties. We further efficiently silenced B. vulgaris MYB1 gene that regulating the betalain pathway in sugar beet by BSCTV-based VIGS system. Finally, we used this system to silence the effector CbNip1 of Cercospora beticola through cross-kingdom RNAi, resulting in reduced disease symptoms in sugar beet plants. Therefore, the BSCTV VIGS system can be used to study sugar beet-pathogen interactions. In summary, we have established an easy and efficiency BSCTV-based VIGS vector in sugar beet. This system will be an attractive tool for functional genomic studies for sugar beet.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.11.663803v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comparing LigandMPNN and Directed Evolution for Altering the Effector-Binding Site in the RamR Transcription Factor</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Clark-ElSayed, A., Creed, E., Nayvelt, K., Ellington, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We used directed evolution and computational redesign to design a transcriptional repressor that was not functional in Escherichia coli, and showed that different protein design methods may be needed.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recently, the number of ML-based tools for protein design has greatly expanded. Although there have been many successful uses of these tools for improved stability, solubility, and ligand binding, there have been fewer uses of these tools for designing proteins that have intrinsic allosteric mechanisms. In this regard, allosteric transcription factors (aTFs) are a class of regulatory proteins that includes repressors and activators that respond to environmental signals by allosteric communication to regulate their binding with DNA elements. The data exist for evaluating design algorithms for their ability to take allostery into account, as many aTFs have previously been engineered to respond to new ligands, enabling their use as biosensors. In particular, previous work from our lab used directed evolution to change the effector specificity of the transcriptional repressor, RamR, from cholic acids to each of five benzylisoquinoline alkaloids (BIAs). We wanted to see to what extent we could recapitulate these results by instead using LigandMPNN to design the ligand binding pocket. The wild-type RamR structure was predicted in complex with the five BIAs, and the binding pocket was then targeted for computational redesign. However, there was little overlap between the results of directed evolution and computational redesign, and in fact the nine redesigned protein variants tested proved not to be functional in Escherichia coli. Overall, these and other results suggest that different protein design methods may be needed to advance the computational design of allosteric or conformationally flexible proteins.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.10.663684v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Rewiring DNA Repair Activity into CRISPR Signal Transduction via Synthetic DNA Transducers</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bagheri, N., Bertucci, A., Merlo, R., Porchetta, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a synthetic transduction platform that directly links endogenous DNA repair activity to CRISPR-Cas12a activation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Harnessing the precision of CRISPR systems for diagnostics has transformed nucleic acid detection. However, the integration of upstream cellular signals into CRISPR-based circuits remains largely underexplored. Here, we introduce a synthetic transduction platform that directly links endogenous DNA repair activity to CRISPR-Cas12a activation. By coupling base excision repair (BER) events to a programmable DNA-based transducer, our system converts the activity of DNA glycosylases, such as uracil-DNA glycosylase (UDG) and human 8-oxoguanine glycosylase (hOGG1), into a robust fluorescence signal via Cas12a-mediated trans-cleavage. This one-step CRISPR-based assay operates directly in cell lysates, enabling rapid and sensitive readout of enzymatic activity with high specificity. Additionally, it also enables in 15 minutes the throughput screening of novel potential inhibitors with high sensitivity. The modular design allows adaptation to diverse repair enzymes, offering a generalizable strategy for transforming intracellular repair events into programmable outputs. This approach lays the foundation for activity-based molecular diagnostics, synthetic gene circuits responsive to cellular states, and new tools for monitoring DNA repair in real time and drug screening.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.11.664310v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">EPEPDI: prediction of binding free energy changes from missense mutations in double and single-stranded DNA-binding proteins</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> YU, X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed EPEPDI, a deep learning framework that integrates sequence-based embeddings from pre-trained protein language models, including ESM2, ProtTrans, and ESM1v, to predict mutation impacts on DNA-binding proteins.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Predicting changes in binding free energy due to missense mutations (MMs) in protein-DNA interactions (PDIs) is vital for understanding disease mechanisms and advancing therapeutic strategies. However, many existing models fail to account for the unique characteristics of MMs in double-stranded DNA binding proteins (DSBs) and single-stranded DNA binding proteins (SSBs). To address this, we constructed a comprehensive dataset from diverse sources, clearly distinguishing between DSBs and SSBs. Using sequence-based embeddings from pre-trained protein language models, including ESM2, ProtTrans, and ESM1v, we developed EPEPDI, a deep learning framework that integrates these embeddings through a multi-channel architecture. To refine predictive accuracy, we introduced an information entropy-based algorithm, determining 181 residues as the optimal sequence length where amino acid contributions and entropy dynamics balance. This approach boosts both precision and computational efficiency, enabling scalable analysis of mutation impacts on DNA-binding proteins. Ablation studies validated optimal feature combinations, demonstrating that EPEPDI outperforms existing approaches, achieving an average Pearson correlation coefficient of 0.755 on the MPD276 dataset via ten-fold cross-validation and 0.632 on independent tests for both DSBs and SSBs. This work highlights the importance of distinguishing DSBs and SSBs in PDIs and shows the potential of advanced machine learning in biological research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.08.663633v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ALPAR: Automated Learning Pipeline for Antimicrobial Resistance</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yurtseven, A., Joeres, R., Kalinina, O. V.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce ALPAR (Automated Learning Pipeline for Antimicrobial Resistance), a comprehensive AMR data analysis tool covering the entire process from processing of raw genomic data to training machine learning models to interpretation of results.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The field of machine learning in antimicrobial resistance (AMR) research has experienced rapid growth, fueled by advancements in high-throughput genome sequencing and growing capacity of computational resources. However, the complexity and lack of standardized data preparation and bioinformatic analyses present significant challenges, especially to newcomers to the domain. In response to these challenges, we introduce ALPAR (Automated Learning Pipeline for Antimicrobial Resistance), a comprehensive AMR data analysis tool covering the entire process from processing of raw genomic data to training machine learning models to interpretation of results. Our method relies on a reproducible pipeline that integrates widely used bioinformatics tools, presenting a simplified, automatic workflow specifically tailored for single-reference AMR analysis. Accepting genomic data in the form of FASTA files as input, ALPAR facilitates generation of machine learning-ready data tables and both training of machine learning and execution of genome-wide association studies (GWAS) experiments. Additionally, our tool offers supplementary functionalities such as phylogeny-based analysis of distribution of mutations, enhancing its utility for researchers. Our tool is accessible through the ALPAR GitHub page (https://github.com/kalininalab/ALPAR) and installable via conda (https://anaconda.org/kalininalab/ALPAR).</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.08.663126v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Systematic benchmarking of basecalling models for RNA modification detection with highly multiplexed nanopore sequencing</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Diensthuber, G., Milenkovic, I., Llovera, L., Milovanovic, A., Pelizzari, F., Novoa, E. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that modification-aware basecalling models can reliably detect diverse RNA modification types across a broad range of sequence contexts, but are prone to elevated false positive rates and exhibit notable cross-reactivity with other RNA modifications.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Nanopore direct RNA sequencing (DRS) holds promise for advancing our understanding of the epitranscriptome by detecting RNA modifications in native RNA molecules. Recently, Oxford Nanopore Technologies (ONT) has released basecalling models capable of detecting several RNA modifications. However, their accuracy, sensitivity and specificity, as well as cross-reactivity against other modification types, remains largely unexplored. Here, we systematically benchmark modification-aware models by evaluating their performance on a highly-multiplexed panel of synthetic molecules covering all possible sequence contexts, as well as on biological samples from a diverse set of species. We find that modification-aware models reliably detect diverse RNA modification types across a broad range of sequence contexts. However, they are prone to elevated false positive rates and exhibit notable cross-reactivity with other RNA modification types. We show that the use of modification-free controls allows significant, yet incomplete removal of false positives, thus constituting an essential control. Finally, we demonstrate that basecalling error patterns and alterations in current features can identify differentially modified sites, for modifications for which modification-aware models are absent. Overall, our results underscore the utility and accuracy of modification-aware basecalling models for RNA modification detection, while highlighting the importance of including diverse control samples to mitigate false positive rates.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.11.663424v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

