<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Thursday, 24 April 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Benchmarking foundation cell models for post-perturbation RNA-seq prediction.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gerold Csendes, Gema Sanz, Kristóf Z Szalay, Bence Szalai</p>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We found that even the simplest baseline model-taking the mean of training examples-outperformed scGPT and scFoundation, and that basic machine learning models that incorporate biologically meaningful features outperformed the latter by a large margin.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurately predicting cellular responses to perturbations is essential for understanding cell behaviour in both healthy and diseased states. While perturbation data is ideal for building such predictive models, its availability is considerably lower than baseline (non-perturbed) cellular data. To address this limitation, several foundation cell models have been developed using large-scale single-cell gene expression data. These models are fine-tuned after pre-training for specific tasks, such as predicting post-perturbation gene expression profiles, and are considered state-of-the-art for these problems. However, proper benchmarking of these models remains an unsolved challenge. In this study, we benchmarked two recently published foundation models, scGPT and scFoundation, against baseline models. Surprisingly, we found that even the simplest baseline model-taking the mean of training examples-outperformed scGPT and scFoundation. Furthermore, basic machine learning models that incorporate biologically meaningful features outperformed scGPT by a large margin. Additionally, we identified that the current Perturb-Seq benchmark datasets exhibit low perturbation-specific variance, making them suboptimal for evaluating such models. Our results highlight important limitations in current benchmarking approaches and provide insights into more effectively evaluating post-perturbation gene expression prediction models.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40269681/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Longitudinal prediction of DNA methylation to forecast epigenetic outcomes.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Arthur Leroy, Ai Ling Teh, Frank Dondelinger, Mauricio A Alvarez, Dennis Wang</p>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new method for measuring epigenetic changes in DNA methylation in early life .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Epigenetic changes in early life play an important role in the development of health conditions in children. Longitudinally measuring and forecasting changes in DNA methylation can reveal patterns of ageing and disease progression, but biosamples may not always be available.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40267756/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The Signed Two-Space Proximity Model for Learning Representations in Protein-Protein Interaction Networks.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nikolaos Nakis, Chrysoula Kosma, Anastasia Brativnyk, Michail Chatzianastasis, Iakovos Evdaimon, Michalis Vazirgiannis</p>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the Signed Two-Space Proximity Model (S2-SPM) for signed PPI networks, which explicitly incorporates both positive and negative interactions, reflecting the complex regulatory mechanisms within biological systems.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurately predicting complex protein-protein interactions (PPIs) is crucial for decoding biological processes, from cellular functioning to disease mechanisms. However, experimental methods for determining PPIs are computationally expensive. Thus, attention has been recently drawn to machine learning approaches. Furthermore, insufficient effort has been made toward analyzing signed PPI networks, which capture both activating (positive) and inhibitory (negative) interactions. To accurately represent biological relationships, we present the Signed Two-Space Proximity Model (S2-SPM) for signed PPI networks, which explicitly incorporates both types of interactions, reflecting the complex regulatory mechanisms within biological systems. This is achieved by leveraging two independent latent spaces to differentiate between positive and negative interactions while representing protein similarity through proximity in these spaces. Our approach also enables the identification of archetypes representing extreme protein profiles.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40266854/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Custom CRISPR-Cas9 PAM variants via scalable engineering and machine learning.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rachel A Silverstein, Nahye Kim, Ann-Sophie Kroell, Russell T Walton, Justin Delano, Rossano M Butcher, Martin Pacesa, Blaire K Smith, Kathleen A Christie, Leillani L Ha, Ronald J Meis, Aaron B Clark, Aviv D Spinner, Cicera R Lazzarotto, Yichao Li, Azusa Matsubara, Elizabeth O Urbina, Gary A Dahl, Bruno E Correia, Debora S Marks, Shengdar Q Tsai, Luca Pinello, Suk See De Ravin, Qin Liu, Benjamin P Kleinstiver</p>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> The first generalist CRISPR-Cas enzymes were developed in the 1970s, and are used in the discovery of new proteins.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Engineering and characterizing proteins can be time-consuming and cumbersome, motivating the development of generalist CRISPR-Cas enzymes</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40262634/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The Dawn of High-Throughput and Genome-Scale Kinetic Modeling: Recent Advances and Future Directions.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ilias Toumpe, Subham Choudhury, Vassily Hatzimanikatis, Ljubisa Miskovic</p>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A review of the state-of-the-art in kinetic modeling, highlighting the potential of these advances to drive progress in systems and synthetic biology, metabolic engineering, and medical research at an unprecedented scale and pace.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Researchers have invested much effort into developing kinetic models due to their ability to capture dynamic behaviors, transient states, and regulatory mechanisms of metabolism, providing a detailed and realistic representation of cellular processes. Historically, the requirements for detailed parametrization and significant computational resources created barriers to their development and adoption for high-throughput studies. However, recent advancements, including the integration of machine learning with mechanistic metabolic models, the development of novel kinetic parameter databases, and the use of tailor-made parametrization strategies, are reshaping the field of kinetic modeling. In this Review, we discuss these developments and offer future directions, highlighting the potential of these advances to drive progress in systems and synthetic biology, metabolic engineering, and medical research at an unprecedented scale and pace.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40262025/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrating machine learning models with multi-omics analysis to decipher the prognostic significance of mitotic catastrophe heterogeneity in bladder cancer.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Haojie Dai, Zijie Yu, You Zhao, Ke Jiang, Zhenyu Hang, Xin Huang, Hongxiang Ma, Li Wang, Zihao Li, Ming Wu, Jun Fan, Weiping Luo, Chao Qin, Weiwen Zhou, Jun Nie</p>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Mitotic catastrophe is well-known as a major pathway of endogenous tumor death, but the prognostic significance of its heterogeneity regarding bladder cancer (BLCA) remains unclear.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mitotic catastrophe is well-known as a major pathway of endogenous tumor death, but the prognostic significance of its heterogeneity regarding bladder cancer (BLCA) remains unclear.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40259382/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrated multiomics analysis identifies potential biomarkers and therapeutic targets for autophagy associated AKI to CKD transition.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yaojun Wang, Qiang Li</p>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified 14 autophagy-related genes and their immune infiltration during the transition from acute kidney injury to chronic kidney disease (CKD).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> This study explored the relationship between acute kidney injury (AKI) and chronic kidney disease (CKD), focusing on autophagy-related genes and their immune infiltration during the transition from AKI to CKD. We performed weighted correlation network analysis (WGCNA) using two microarray datasets (GSE139061 and GSE66494) in the GEO database and identified autophagy signatures by Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG), and GSEA enrichment analysis. Machine learning algorithms such as LASSO, random forest, and XGBoost were used to construct the diagnostic model, and the diagnostic performance of GSE30718 (AKI) and GSE37171 (CKD) was used as validation cohorts to evaluate its diagnostic performance. The study identified 14 autophagy candidate genes, among which ATP6V1C1 and COPA were identified as key biomarkers that were able to effectively distinguish between AKI and CKD. Immune cell infiltration and GSEA analysis revealed immune dysregulation in AKI, and these genes were associated with inflammation and immune pathways. Single-cell analysis showed that ATP6V1C1 and COPA were specifically expressed in AKI and CKD, which may be related to renal fibrosis. In addition, drug prediction and molecular docking analysis proposed SZ(+)-(S)-202-791 and PDE4 inhibitor 16 as potential therapeutic agents. In summary, this study provides new insights into the relationship between AKI and CKD and lays a foundation for the development of new treatment strategies.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40258914/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrated bioinformatic analysis of immune infiltration and disulfidptosis related gene subgroups in type A aortic dissection.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Duo Wang, Chengwen Wang, Haoyue Liu, Zhenzhu Zhang, Mixia Li, Xinpeng Ge, Anqi Bi, Chenguang Gao, Xue Tian, Kexiang Liu, Zhicheng Zhu</p>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study investigates the role of disulfidptosis-related genes in immune infiltration in type A aortic dissection and identifies potential therapeutic targets for TAAD.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Type A aortic dissection (TAAD) is a lethal cardiovascular disease characterized by the separation of the layers within the aortic wall. The underlying pathological mechanisms of TAAD requires further elucidation to develop effective prevention and pharmacological treatment strategies. Inflammation plays a crucial role in TAAD pathogenesis. Disulfidptosis, an emerging type of cell death, may shed light on disease mechanisms. This study investigates the role of disulfidptosis-related genes in immune infiltration in TAAD. TAAD gene expression datasets were obtained from the Gene Expression Omnibus (GEO) database. Immune cell infiltration analysis assessed immune cell dysregulation in TAAD. Differentially expressed genes (DEGs) between TAAD samples and controls were identified and intersected with known disulfidptosis-related gene sets to obtain relevant DEGs. Hub genes were identified using machine learning algorithms. A diagnostic model was constructed using Least Absolute Shrinkage and Selection Operator (LASSO) regression on 25 TAAD samples. Consensus clustering classified TAAD samples based on disulfidptosis-related gene expression. Functional enrichment analyses, including Gene Ontology (GO) terms and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway analyses, elucidated associated biological processes and pathways. A total of 13,316 DEGs were identified, among which 11 disulfidptosis-related genes were screened: INF2, CD2AP, PDLIM1, ACTN4, MYH10, MYH9, FLNA, FLNB, TLN1, MYL6, ACTB, CAPZB, DSTN, and IQGAP1. Most of these genes exhibited lower expression levels in TAAD samples, except CAPZB, and were correlated with immune cell infiltration. Cluster-specific DEGs were found in one cluster, involving several immune response processes. Co-clustering analysis based on disulfidptosis-related genes classified TAAD samples into two clusters, with higher gene expression levels observed in cluster C2 compared to cluster C1. Three key hub genes were identified, and potential therapeutic mechanisms for TAAD were explored. Immuno-infiltration results revealed significant differences in immune profiles, with higher immunological scores and more extensive immune infiltration in TAAD. Disulfidptosis occurs in TAAD and is associated with immune cell infiltration and metabolic activity, influencing immune cell function and responses. These findings suggest that disulfidptosis may promote TAAD progression through the induction of immune responses and metabolic activities. This research provides new insights into the pathogenesis and identifies potential therapeutic targets for TAAD.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40258895/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Deconvolution of cell types and states in spatial multiomics utilizing TACIT.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Khoa L A Huynh, Katarzyna M Tyc, Bruno F Matuck, Quinn T Easter, Aditya Pratapa, Nikhil V Kumar, Paola Pérez, Rachel J Kulchar, Thomas J F Pranzatelli, Deiziane de Souza, Theresa M Weaver, Xufeng Qu, Luiz Alberto Valente Soares Junior, Marisa Dolhnokoff, David E Kleiner, Stephen M Hewitt, Luiz Fernando Ferraz da Silva, Vanderson Geraldo Rocha, Blake M Warner, Kevin M Byrd, Jinze Liu</p>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> TACIT, an unsupervised algorithm for cell annotation using predefined signatures that operates without training data, identifies under- and overrepresented immune cell types and states in regions of interest, and discovers new phenotypes in inflammatory gland diseases.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Identifying cell types and states remains a time-consuming, error-prone challenge for spatial biology. While deep learning increasingly plays a role, it is difficult to generalize due to variability at the level of cells, neighborhoods, and niches in health and disease. To address this, we develop TACIT, an unsupervised algorithm for cell annotation using predefined signatures that operates without training data. TACIT uses unbiased thresholding to distinguish positive cells from background, focusing on relevant markers to identify ambiguous cells in multiomic assays. Using five datasets (5,000,000 cells; 51 cell types) from three niches (brain, intestine, gland), TACIT outperforms existing unsupervised methods in accuracy and scalability. Integrating TACIT-identified cell types reveals new phenotypes in two inflammatory gland diseases. Finally, using combined spatial transcriptomics and proteomics, we discover under- and overrepresented immune cell types and states in regions of interest, suggesting multimodality is essential for translating spatial biology to clinical applications.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40258827/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Design of nanobody targeting SARS-CoV-2 spike glycoprotein using CDR-grafting assisted by molecular simulation and machine learning.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Matheus Vf Ferraz, W Camilla S Adan, Tayná E Lima, Adriele Jc Santos, Sérgio O de Paula, Rafael Dhalia, Gabriel L Wallau, Rebecca C Wade, Isabelle Ft Viana, Roberto D Lins</p>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a complementarity-determining region (CDR) grafting approach for designing nanobodies (Nbs) that target specific epitopes, with the aid of computer simulation and machine learning.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The design of proteins capable effectively binding to specific protein targets is crucial for developing therapies, diagnostics, and vaccine candidates for viral infections. Here, we introduce a complementarity-determining region (CDR) grafting approach for designing nanobodies (Nbs) that target specific epitopes, with the aid of computer simulation and machine learning. As a proof-of-concept, we designed, evaluated, and characterized a high-affinity Nb against the spike protein of SARS-CoV-2, the causative agent of the COVID-19 pandemic. The designed Nb, referred to as Nb Ab.2, was synthesized and displayed high-affinity for both the purified receptor-binding domain protein and to the virus-like particle, demonstrating affinities of 9 nM and 60 nM, respectively, as measured with microscale thermophoresis. Circular dichroism showed the designed protein's structural integrity and its proper folding, whereas molecular dynamics simulations provided insights into the internal dynamics of Nb Ab.2. This study shows that our computational pipeline can be used to efficiently design high-affinity Nbs with diagnostic and prophylactic potential, which can be tailored to tackle different viral targets.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40257976/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">High-throughput screening data generation, scoring and FAIRification: a case study on nanomaterials.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gergana Tancheva, Vesa Hongisto, Konrad Patyra, Luchesar Iliev, Nikolay Kochev, Penny Nymark, Pekka Kohonen, Nina Jeliazkova, Roland Grafström</p>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a HTS-driven FAIRifed computational assessment tool for hazard analysis of multiple materials hazards rapidly and simultaneously, aligning with regulatory recommendations and addressing industry needs.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In vitro-based high-throughput screening (HTS) technology is applicable to hazard-based ranking and grouping of diverse agents, including nanomaterials (NMs). We present a standardized HTS-derived human cell-based testing protocol which combines the analysis of five assays into a broad toxic mode-of-action-based hazard value, termed Tox5-score. The overall protocol includes automated data FAIRification, preprocessing and score calculation. A newly developed Python module ToxFAIRy can be used independently or within an Orange Data Mining workflow that has custom widgets for fine-tuning, included in the custom-developed Orange add-on Orange3-ToxFAIRy. The created data-handling workflow has the advantage of facilitated conversion of the FAIR HTS data into the NeXus format, capable of integrating all data and metadata into a single file and multidimensional matrix amenable to interactive visualizations and selection of data subsets. The resulting FAIR HTS data includes both raw and interpreted data (scores) in machine-readable formats distributable as data archive, including into the eNanoMapper database and Nanosafety Data Interface. We overall present a HTS-driven FAIRifed computational assessment tool for hazard analysis of multiple agents simultaneously, including with broad potential applicability across diverse scientific communities.Scientific Contribution Our study represents significant tool development for analyzing multiple materials hazards rapidly and simultaneously, aligning with regulatory recommendations and addressing industry needs. The innovative integration of in vitro-based toxicity scoring with automated data preprocessing within FAIRification workflows enhances the applicability of HTS-derived data application in the materials development community. The protocols described increase the effectiveness of materials toxicity testing and mode-of-action research by offering an alternative to manual data processing, enrichment of HTS data with metadata, refining testing methodologies-such as for bioactivity-based grouping-and overall, demonstrates the value of reusing existing data.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40269950/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">OSaMPle workflow for salivary metaproteomics analysis reveals dysbiosis in inflammatory bowel disease patients.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jinhui Yuan, Boyan Sun, Murong Li, Congyi Yang, Lingqiang Zhang, Ning Chen, Feng Chen, Leyuan Li</p>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present an Optimized Salivary MetaProteomic sample analysis workflow (OSaMPle) to enrich salivary bacteria and reduce host-derived interferences for in-depth analysis of the oral metaproteome.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The human oral microbiome has been associated with multiple inflammatory conditions including inflammatory bowel disease (IBD). Identifying functional changes in oral microbiome by metaproteomics helps understanding the factors driving dysbiosis related to intestinal diseases. However, enriching bacterial cells from oral samples (such as saliva and mouth rinse) rich in host proteins is challenging. Here, we present an Optimized Salivary MetaProteomic sample analysis workflow (OSaMPle) to enrich salivary bacteria and reduce host-derived interferences for in-depth analysis of the oral metaproteome. Compared to a conventional approach, OSaMPle improved the identification of bacterial peptides and proteins by 3.2 folds and 1.7 folds, respectively. Furthermore, applying OSaMPle to analyze mouth rinse samples from IBD patients revealed significant alterations in bacterial protein expressions under disease conditions. Specifically, proteins involved in the fatty acid elongation pathway in Peptostreptococcus were significantly less abundant in IBD patients, whereas proteins associated with the TCA cycle in Neisseria were significantly more abundant. The OSaMPle workflow is capable of processing small-volume oral samples and adaptable to high-throughput automation. It holds promise as a strategy for investigating the functional responses of oral microbiomes under disease conditions and identifying disease-associated microbes with their proteins, providing critical insights for detecting disease-related biomarkers within the oral microbiome.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40268913/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Ibaqpy: A scalable Python package for baseline quantification in proteomics leveraging SDRF metadata.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ping Zheng, Enrique Audain, Henry Webel, Chengxin Dai, Joshua Klein, Marc-Phillip Hitz, Timo Sachsenberg, Mingze Bai, Yasset Perez-Riverol</p>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present ibaqpy, a Python package designed to compute iBAQ values efficiently for experiments of any scale while incorporating experimental metadata into the quantification workflow.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Intensity-based absolute quantification (iBAQ) is essential in proteomics as it allows for the assessment of a protein's absolute abundance in various samples or conditions. However, the computation of these values for increasingly large-scale and high-throughput experiments, such as those using DIA, TMT, or LFQ workflows, poses significant challenges in scalability and reproducibility. Here, we present ibaqpy (https://github.com/bigbio/ibaqpy), a Python package designed to compute iBAQ values efficiently for experiments of any scale. Ibaqpy leverages the Sample and Data Relationship Format (SDRF) metadata standard to incorporate experimental metadata into the quantification workflow. This allows for automatic normalization and batch correction while accounting for key aspects of the experimental design, such as technical and biological replicates, fractionation strategies, and sample conditions. Designed for large-scale proteomics datasets, ibaqpy can also recompute iBAQ values for existing experiments when an SDRF is available. We showcased ibaqpy's capabilities by reanalyzing 17 public proteomics datasets from ProteomeXchange, covering HeLa cell lines with 4921 samples and 5766 MS runs, quantifying a total of 11,014 proteins. In our reanalysis, ibaqpy is a key component in automating reproducible quantification, reducing manual effort and making quantitative proteomics more accessible while supporting FAIR principles for data reuse. SIGNIFICANCE: Proteomics studies often rely on intensity-based absolute quantification (iBAQ) to assess protein abundance across various biological conditions. Despite its widespread use, computing iBAQ values at scale remains challenging due to the increasing complexity and volume of proteomics experiments. Existing tools frequently lack metadata integration, limiting their ability to handle experimental design intricacies such as replicates, fractions, and batch effects. Our work introduces ibaqpy, a scalable Python package that leverages the Sample and Data Relationship Format (SDRF) to compute iBAQ values efficiently while incorporating critical experimental metadata. By enabling automated normalization and batch correction, ibaqpy ensures reproducible and comparable quantification across large-scale datasets. We validated the utility of ibaqpy through the reanalysis of 17 public HeLa datasets, comprising over 200 million peptide features and quantifying 11,000 proteins across thousands of samples. This comprehensive reanalysis highlights the robustness and scalability of ibaqpy, making it an essential tool for researchers conducting large-scale proteomics experiments. Moreover, by promoting FAIR principles for data reuse and interoperability, ibaqpy offers a transformative approach to baseline protein quantification, supporting reproducible research and data integration within the proteomics community.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40268243/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Rapid assay development for low input targeted proteomics using a versatile linear ion trap.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ariana E Shannon, Rachael N Teodorescu, No Joon Song, Lilian R Heil, Cristina C Jacob, Philip M Remes, Zihai Li, Mark P Rubinstein, Brian C Searle</p>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show consistent quantification across three orders of magnitude in a matched-matrix background of low-level proteins such as transcription factors and cytokines in a 1 ng sample without requiring stable isotope-labeled standards.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Advances in proteomics and mass spectrometry enable the study of limited cell populations, where high-mass accuracy instruments are typically required. While triple quadrupoles offer fast and sensitive low-mass specificity measurements, these instruments are effectively restricted to targeted proteomics. Linear ion traps (LITs) offer a versatile, cost-effective alternative capable of both targeted and global proteomics. Here, we describe a workflow using a hybrid quadrupole-LIT instrument that rapidly develops targeted proteomics assays from global data-independent acquisition (DIA) measurements without high-mass accuracy. Using an automated software approach for scheduling parallel reaction monitoring assays (PRM), we show consistent quantification across three orders of magnitude in a matched-matrix background. We demonstrate measuring low-level proteins such as transcription factors and cytokines with quantitative linearity below two orders of magnitude in a 1 ng background proteome without requiring stable isotope-labeled standards. From a 1 ng sample, we found clear consistency between proteins in subsets of CD4</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40263265/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">INSIGHT: In Silico Drug Screening Platform using Interpretable Deep Learning Network</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shi, X., Ramathal, C., Dezso, Z.</p>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A biologically informed deep learning framework for drug sensitivity and drug response in cancer cell lines .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The large-scale multiplexed drug screening platforms like PRISM and GDSC facilitate the screening of drug treatments over 1,000 cancer cell lines. The cancer cell lines are well characterized by multiomics screening in CCLE and DepMap, enabling the application of AI and machine learning techniques to study the association between drug sensitivity and the underlying molecular profiles. The large scale and variety of data modalities enabled us to build an interpretable deep learning framework, INSIGHT, integrating the multiomics data and the drug's molecular structure to predict drug response. We trained our model on the PRISM screen for single treatments and on the DrugComb screen database for combination treatments. Our method enables the in-silico extension of current screens by predicting drug response in cancer cell lines not included in the screen, as well as the drug response to novel single agent or combination therapy by leveraging the drug's molecular structure. Furthermore, the deep learning framework was built to enable biological interpretation. The connections between the hidden layers of the neural network incorporate prior biological knowledge such as signaling pathways. This enables the model, in addition to predicting the drug sensitivity profiles, to prioritize the pathways predictive of drug response and identify pathways related to the mechanism of action (MOA) and potential off target effects of novel drugs. The evaluation of our model using cross-validation on the PRISM and DrugComb dataset showed an improved performance compared to previously developed biologically informed deep learning methods and traditional state of the art machine learning methods like elastic-net and XGBoost. We illustrated with examples the value of incorporating biological knowledge into INSIGHT by relating the pathway activity of the predictive models to the MOA.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.21.649855v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SIMIT-seq: a bead-free, scalable microfluidic platform for single cell mRNA sequencing with ultrahigh cell-indexing rate</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Duan, M., Gao, W., Li, G., Cai, Y., Zhao, Z., Lan, X., Wang, D., Xing, X., Luo, Y.</p>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a bead-free, scalable single-cell RNA sequencing platform that eliminates the need for beads entirely by employing a deterministic, orthogonal barcoding strategy within a two-dimensional micro-well array.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> SIMIT-seq is a bead-free, scalable microfluidic platform designed for high-efficiency single-cell mRNA sequencing. Conventional microfluidic-based single-cell RNA sequencing platforms rely heavily on barcoded beads and intricate co-encapsulation schemes, often constrained by double Poisson limitations and the complexities of bead synthesis. In contrast, SIMIT-seq eliminates the need for beads entirely by employing a deterministic, orthogonal barcoding strategy within a two-dimensional micro-well array. This platform achieves an impressive single-cell indexing rate of 96.6% without the need for complex microfluidic operations. Here, we describe the design and fabrication of the SIMIT-seq platform, outline its workflow for transcript capture and library preparation, and demonstrate its application in profiling K562 cells. Our results validate both the high fidelity of molecular barcode immobilization and the platform capability to support downstream single-cell mRNA sequencing. SIMIT-seq offers a cost-effective and scalable alternative for single-cell transcriptomics, providing a promising foundation for future single-cell omics applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.23.650129v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SCassist: An AI Based Workflow Assistant for Single-Cell Analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nagarajan, V., Shi, G., Arunkumar, S., Liu, C., Gopalakrishnan, J., Nath, P. R., Jang, J., Caspi, R.</p>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We have developed SCassist, an R package that leverages the power of the large language models (LLM's) to guide and enhance scRNA-seq analysis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell RNA sequencing (scRNA-seq) data analysis often involves complex iterative workflow, requiring significant expertise and time. To navigate this complexity, we have developed SCassist, an R package that leverages the power of the large language models (LLM's) to guide and enhance scRNA-seq analysis. SCassist integrates LLM's into key workflow steps, to analyze user data and provide relevant recommendations for filtering, normalization and clustering parameters. It also provides LLM guided insightful interpretations of variable features and principal components, along with cell type annotations and enrichment analysis. SCassist provides intelligent assistance using popular LLM's like Google's Gemini and Meta's Llama3, making scRNA-seq analysis accessible to researchers at all levels. The SCassist package, along with the detailed tutorials, is available at GitHub - https://github.com/NIH-NEI/SCassist</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.22.650107v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Shiny-Calorie: A context-aware application for indirect calorimetry data analysis and visualization using R</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Grein, S., Elschner, T., Kardinal, R., Bruder, J., Strohmeyer, A., Gunasekaran, K., Witt, J., Hermannsdottir, H., Behrens, J., U-Din, M., Yu, J., Heldmaier, G., Schreiber, R., Rozman, J., Heine, M., Scheja, L., Worthmann, A., Heeren, J., Wachten, D., Wilhelm-Juengling, K., Pfeifer, A., Hasenauer, J., Klingenspor, M.</p>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present Shiny-Calorie, an open-source interactive web application for transparent data and metadata integration, comprehensive statistical data analysis, and visualization of indirect calorimetry datasets.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Indirect calorimetry is a cornerstone technique for metabolic phenotyping of animal models in preclinical research, with well-established experimental protocols and platforms. However, a flexible, extensible, and user-friendly software suite that enables standardized integration of data and metadata from diverse metabolic phenotyping platforms - followed by unified statistical analysis and visualization - remains absent. We present Shiny-Calorie, an open-source interactive web application for transparent data and metadata integration, comprehensive statistical data analysis, and visualization of indirect calorimetry datasets. Shiny-Calorie is compatible with data formats from widely used commercial metabolic phenotyping platforms, such as TSE and Sable Systems, and includes functionality for exporting processed data in these formats. Built using GNU R and a Shiny-based reactive interface, Shiny-Calorie enables intuitive exploration of complex, multi-modal longitudinal datasets comprising categorical, continuous, ordinal, and count variables. The platform incorporates state-of-the-art statistical methods for robust hypothesis testing, thereby facilitating biologically meaningful interpretation of energy metabolism phenotypes, including resting metabolic rate and energy expenditure. Overall, Shiny-Calorie streamlines routine analysis workflows and enhances reproducibility and transparency in metabolic phenotyping studies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.24.648116v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SieveAI: Development of an Automated extensible and customisable drug discovery pipeline and its validation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sahu, V. K., Sand, A., Ballav, S., Raman, V., Nagar, S., Ranjan, A., Basu, S.</p>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> SieveAI is an automated drug discovery pipeline developed to enhance the efficiency of virtual screening and computer-aided drug discovery processes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Systematic Interaction Evaluation and Virtual Enhancement Analysis Interface (SieveAI) is an automated drug discovery pipeline developed to enhance the efficiency of virtual screening and computer-aided drug discovery processes. The molecular docking workflow encompasses acquiring, modeling, and pre-processing of molecular structure files, conducting docking with various algorithms, and subsequent analysis and interpretation of the outcomes by visualising or tabulating the results. While several open-source software tools are available to assist these operations at different steps of molecular docking, they often necessitate manual user intervention at every stage. To streamline and automate this extensive manual process and develop a comprehensive solution, we have developed an innovative, fully extensible, molecular docking pipeline SieveAI (Copyright Number L-129927/2023). The same has been demonstrated in this manuscript. SieveAI works with a range of open-source libraries, packages, and programs to facilitate automated drug discovery using established programs and software. The package is accessible at https://miRNA.in/SieveAI.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.20.648820v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Advancing the understanding of the Bombyx mori genome through satellite DNA analysis: low strain differentiation and relationship with transposable elements</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rico-Porras, J. M., Mora Ruiz, P., Palomeque, T., Lorite, P., Cabral-de-Mello, D. C.</p>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the first comprehensive analysis of the satellite DNA (satDNA) fraction of the Bombyx mori satellitome, revealing its low abundance, high monomer size variability, and predominantly dispersed chromosomal organization.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Despite the economic and scientific importance of Bombyx mori, its satellite DNA (satDNA) fraction remains poorly characterized. Here, we present the first comprehensive analysis of the B. mori satellitome, revealing its low abundance (~0.76%), A+T enrichment, high monomer size variability, and predominantly dispersed chromosomal organization. Comparative analysis with B. mandarina, the wild ancestor of B. mori, demonstrated high satDNA conservation in abundance and sequence composition, despite speciation and domestication events. Divergence landscapes of satDNA sequences indicate predominantly ancient amplifications with few recent homogenization events. SatDNA families exhibited low strain differentiation across five B. mori strains, exemplified by the consistent presence and abundance of BmorSat01-575 and BmorSat29-84. The W chromosome was identified as a hotspot for satDNA accumulation, especially BmorSat24-753, which showed female-specific amplification and higher sequence homogeneity, suggesting functional or structural roles in W chromosome evolution. In contrast, the Z chromosome displayed similar satDNA content to autosomes, with two families being exclusive to this chromosome and conserved in both Bombyx species. Remarkably, some of B. mori satDNAs were found to be derived from transposable elements (TEs), supporting the hypothesis of TE-driven satDNA origin via multiple mechanisms, including tandemization of TE fragments. Our findings underscore the dynamic interplay between satDNAs and TEs in shaping genome architecture, the stability of the satellitome through domestication, and the sex chromosome-specific accumulation patterns. These insights lay the groundwork for future studies investigating the functional roles of satDNAs in genome regulation and chromatin dynamics in Lepidoptera.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.21.649856v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DeltaBreed: A BrAPI-centric breeding data information system</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yarnes, S. C., Palladino, N., Meidlinger, D. J., Philips, D. R., Sweeney, H. M., Mustafa, S., Mandych, M. L., Bouabane, S., Parsons, T. E., Slonecki, T. J., Zhao, D., Rife, T. W., Ellerbrock, B. J., Courtney, C., Selby, P., Flores-Gonzalez, M., Mueller, L. A., Aparicio, J. S., Al-Sham'aa, K., Raubach, S., Beil, C. T., Sheehan, M. J., Meng Lin, Edwin Reidel, Tyr Wiesner-Hanks, Arlyn J Ackerman, Alexandra M Casa, Alexander M Sandercoc</p>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> DeltaBreed is a unified breeding data management system designed by Breeding Insight (BI, Cornell University) for USDA-ARS specialty crop and animal breeding programs.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> DeltaBreed is a unified breeding data management system designed by Breeding Insight (BI, Cornell University) to serve the wide diversity of USDA-ARS specialty crop and animal breeding programs. DeltaBreed has a RESTful microservice architecture that utilizes the BrAPI Java Test Server as its primary database. The system is interoperable with many BrAPI-compliant applications (BrApps), including Field Book, and is continually aligned with the most recent BrAPI specifications. Here, we describe the features of DeltaBreed and provide several interoperability test cases to illustrate successes and challenges encountered during development. We also discuss future expansion and enhancement plans for DeltaBreed, as well as outline possible solutions to known limitations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.23.650200v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Transcription-based identification of uncharacterized genes in the human immune response</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Vorsteveld, E. E., Kersten, S., Kaffa, C., Simons, A., 't Hoen, P. A. C., Netea, M. G., Hoischen, A.</p>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify 901 variants in uncharacterized genes that could underly the human immune response and identify 5 candidate variants in 4 genes that underly rare immune disorders.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Host-pathogen interactions are shaped by the nature of the pathogen and by host-related factors. The human host responses can be characterized in microbe-stimulated immune cells using transcriptomics. We set out to characterize gene expression changes as a result of microbial in vitro stimulation in RPMI-cultured human primary immune cells, using four pathogen ligands representing bacteria (LPS and S. aureus), viruses (Poly(I:C)) and fungi (C. albicans) for 4 and 24 hours, resulting in a total of 52 samples for analysis. We analyze the transcriptional changes on gene- and pathway level, highlighting common and distinct effects of pathogen stimulation. We highlight genes without a known function that were differentially expressed as a common effect of pathogen stimulation. We find uncharacterized genes such as KIAA0040 and CYRIA to be co-expressed with genes involved in innate immunity and downstream signaling from the PRRs, respectively. We identify 901 variants in uncharacterized genes in a cohort of patients with rare immune disorders, prioritizing 5 candidate variants in 4 genes that could underly these diseases. Our results therefore indicate the potential role of these genes the human immune response and provides candidate variants with potential relevance in rare immune-mediated diseases.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.22.646545v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Dynamic Contrastive Learning with Pretrained Deep Language Model Enhances Metagenome Binning for Contigs</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zou, B., Zhang, Z., Tao, R., Gu, N., Zhang, L.</p>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose CompleteBin, an innovative pretrained deep language model enhanced by dynamic contrastive learning, which uniquely integrates the sequence context of contigs into metagenome-assembled genomes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The recovery of microbial genomes from large metagenomic datasets is a critical step for identifying uncultivated microbial populations and explaining their functional roles. This process requires metagenomic binning, which involves clustering assembled contigs into metagenome-assembled genomes (MAGs). However, existing computational binning methods often fail to leverage contigs' sequence context effectively. Most of them cannot incorporate taxonomic knowledge from external databases, and insufficient training data frequently limits the deep learning-based binning methods. To address these challenges, we propose CompleteBin, an innovative pretrained deep language model enhanced by dynamic contrastive learning. CompleteBin trains a pretrained deep language model using both long and short contigs and clusters the contigs based on their embeddings generated by the language model. This approach uniquely integrates the sequence context of contigs, incorporates taxonomic knowledge from external databases, and leverages unlimited data during training to significantly improve binning accuracy. Compared to state-of-the-art binning tools such as CONCOCT, MetaBAT2, SemiBin2, and COMEBin, CompleteBin achieves higher performance across both simulated datasets (CAMI I and II) and diverse real-world datasets, including ocean, plant, freshwater, and human fecal samples. On average, CompleteBin achieves a 39.7% improvement on simulated datasets and a 58.9% improvement on real-world datasets in terms of the number of near-complete MAGs when compared to the best-performing results of other binning methods. These results highlight CompleteBin's robust capability to recover high-quality MAGs from long and short contigs, establishing it as a powerful tool for advancing metagenomic research. CompleteBin is open-source and available at https://github.com/zoubohao/CompleteBin.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.20.649691v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An interpretable alphabet for local protein structure search based on amino acid neighborhoods</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zerefa, S., Cool, J., Singh, P., Petti, S.</p>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We design a structural alphabet that encodes the local neighborhoods around each amino acid in an interpretable way in a way that improves protein structure search accuracy. We provide software tools that enable the exploration of novel alphabets and combinations of alphabeets for protein structuresearch.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent advancements in protein structure prediction methods have vastly increased the size of databases of protein structures, necessitating fast methods for protein structure comparison. Search methods that find structurally similar proteins can be applied to find remote homologs, study the functional relationships among proteins, and aid in protein engineering tasks. The structure comparison method Foldseek represents each protein structure as a sequence of "3Di" characters and uses highly optimized sequence comparison software to search with this alphabet. An alternate alphabet encoding richer features has the potential to improve search accuracy while leaving the underlying search algorithm unchanged. We design a "3Dn" structural alphabet that encodes the local neighborhoods around each amino acid in an interpretable way. In a search benchmark task, a combination of our alphabet and Foldseek's 3Di alphabet, outperforms each alphabet individually and ranks best among local search methods that do not require amino acid identity information. We provide software tools that enable the exploration of novel alphabets and combinations of alphabets for protein structure search.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.21.649886v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Label-free quantification in the Crux toolkit</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Acquaye, F. L. N. A., Wen, B., Grant, C. E., Noble, W. S., Kertesz-Farkas, A.</p>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We demonstrate an extension to the Crux MS/MS analysis toolkit to enable label-free quantification of peptides.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Ultimately, most tandem mass spectrometry (MS/MS) proteomics experiments aim to not just detect but also quantify the proteins in a given complex sample. Here, we describe an extension to the Crux MS/MS analysis toolkit to enable label-free quantification of peptides. We demonstrate that Crux's new quantification command, which is modeled after the algorithms implemented in the widely used FlashLFQ software, is both efficient and accurate. In particular, we achieve a 1.9-fold speedup while reducing the memory usage by 26%. The new crux-lfq command is available in Crux v4.3.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.21.649897v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ProtHGT: Heterogeneous Graph Transformers for Automated Protein Function Prediction Using Biological Knowledge Graphs and Language Models</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ulusoy, E., Dogan, T.</p>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a heterogeneous graph transformer-based model that integrates diverse biological datasets into a unified framework using knowledge graphs for accurate and interpretable protein function prediction.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: The rapid accumulation of protein sequence data, coupled with the slow pace of experimental annotations, creates a critical need for computational methods to predict protein functions. Existing models often rely on limited data types, such as sequence-based features or protein-protein interactions (PPIs), failing to capture the complex molecular relationships in biological systems. To address this, we developed ProtHGT, a heterogeneous graph transformer-based model that integrates diverse biological datasets into a unified framework using knowledge graphs for accurate and interpretable protein function prediction. Results: ProtHGT achieves state-of-the-art performance on benchmark datasets, demonstrating its ability to outperform current graph-based and sequence-based approaches. By leveraging diverse biological entity types and highly representative protein language model embeddings at the input level, the model effectively learns complex biological relationships, enabling accurate predictions across all Gene Ontology (GO) sub-ontologies. Ablation analyses highlight the critical role of heterogeneous data integration in achieving robust predictions. Finally, our use-case study has indicated that it's possible to interpret ProtHGT's predictions via exploring the related parts of our input biological knowledge graph, offering plausible explanations to build or test new hypotheses. Availability and Implementation: ProtHGT is available as a programmatic tool on https://github.com/HUBioDataLab/ProtHGT and as a web service on https://huggingface.co/spaces/HUBioDataLab/ProtHGT. Contact: To whom the correspondence should be addressed: Tunca Do[g]an (tuncadogan@gmail.com)</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.19.649272v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

