<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Thursday, 25 September 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CCDC137 knockdown suppresses bladder cancer progression by downregulating SCD.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Haiyu Zhang, Weisheng Huang, Zhimao Cai, Shuanzhu Mou, Yuhan Liu, Bing Yan, Wendong Jiang, Yue Liu, Qiaolin Mei, Xinhui Liao, Yuchen Liu, Hongbing Mei</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> The Coiled-coil domain-containing (CCDC) family, due to its unique protein structural domain and broad involvement in diverse biological processes, has emerged as a focus in oncology research, but its clinical significance and function in bladder cancer (BLCA) remain poorly defined.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The Coiled-coil domain-containing (CCDC) family, due to its unique protein structural domain and broad involvement in diverse biological processes, has emerged as a focus in oncology research. Nevertheless, its clinical significance and function in bladder cancer (BLCA) remain poorly defined.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40993629/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">aMLProt: an automated machine learning library for protein applications.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ruite Xiang, Christian Domínguez-Dalmases, Albert Cañellas-Solé, Victor Guallar</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We train pre-trained large language models using AutoML, a method that streamlines the entire model development pipeline and improves the performance of models.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Machine learning tools have become increasingly common in biological research, driven by the emergence of pre-trained large language models. However, training effective models remains a complex task, since many choices influence their performance. AutoML (automated machine learning) approaches help address these challenges by streamlining the entire model development pipeline.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40991316/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Insights into the patterns of molecular evolution and functional diversification of NAC gene family in land plants</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yuxin, L., Zelong, L., Yilu, Z., Yu, W., Liyuan, G., Xiaohong, C., Zhaoxue, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified 5052 NAC genes in 46 species of land plants and reconstructed evolutionary framework of NAC proteins in land plants, and identified 13 ancestral genes which showed distinct expansion patterns.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> NAC proteins are involved in various aspects of plant development and stress responses. However, the evolutionary history and diversification patterns of NAC genes have not been examined systematically in land plants. By identifying 5052 NAC genes in 46 species of land plants and constructing their phylogenies, we found that NAC genes fell into three categories of evolutionary clades. An integrative analysis of evolutionary clades and synteny network revealed that NAC syntelogs exist widely across angiosperms, and ancient tandem duplications and multiple lineage-specific transposition events contributed to expansion and divergence of NAC family. Based on the reconstructed phylogenies, a potential evolutionary framework of NAC genes in land plants was presented, indicating that 13 ancestral genes which showed distinct expansion patterns evolved all current NAC repertoires of Arabidopsis and maize. ZmNAC genes exhibit divergent expression diversification patterns, which are generally in agreement with the expected clade- and ancestral gene specific expression characterization. The subcellular localization analysis of 14 ZmNAC proteins demonstrated that they are predominantly localized in the nucleus. Transcriptional activity analysis revealed their functional divergence in transcriptional activity. Through ZmNACs overexpression Arabidopsis, we identified the roles of ZmNACs under drought and salt stresses, thereby substantiating functional differentiation of NAC transcription factors in abiotic stress responses. Overall, the synteny network analysis and the reconstructed evolutionary framework of NAC genes in this study increased our understanding for the evolutionary history and functional consequences of NAC genes, and in return, ZmNAC expression profiles are also crucial for understanding the evolutionary sequence divergence from a functional perspective. Our results deepen the knowledge of the evolutionary mechanisms of NAC genes and the function of maize NAC genes in abiotic stress responses.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.677750v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A near gap-free haplotype-resolved genome assembly of Zoysia japonica uncovers intra-subgenomic gene expression and regulatory variation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lee, S. H., Purushotham, P., Chandra, A., Jayakodi, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A fully phased genome of Zoysia japonica cv. Palisades .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Zoysiagrass is a warm-season allotetraploid turfgrass valued for stress tolerance, but high heterozygosity and polyploidy have impeded accurate genome assembly. Here, a near gap-free, allele-aware, fully phased genome of Zoysia japonica cv. Palisades was assembled using PacBio HiFi and Hi-C data. The assembly resolves 20 chromosome pairs (317.2 and 317.0 Mb for haplotypes 1 and 2) with telomeric repeats at both ends of all chromosomes, attains 98.3% BUSCO completeness and 99% completeness by Merqury, and comprises approximately 50% repetitive DNA and about 40,000 protein-coding genes per haplotype. Comparative analysis identified roughly 2.9 million SNPs and numerous structural variants. Hemizygous genes were widespread (4,863 in hap-1 and 4,519 in hap-2), enriched within insertion regions, highly divergent regions, and haplotype-unique sequences, and showed elevated expression relative to homozygous genes. Among 539 one-to-one coding-identical homologs, 29 exhibited haplotype-biased expression associated with upstream promoter variants. This phased genome enables marker development, genetic mapping, and trait dissection in heterozygous polyploids.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.676383v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ppIRIS: deep learning for proteome-wide prediction of bacterial protein-protein interactions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Piochi, L. F., Tang, D., Malmstrom, J., Karami, Y., Khakzad, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce ppIRIS (protein-protein Interaction Regression via Iterative Siamese networks), a lightweight deep learning model that integrates evolutionary and structural embeddings to predict PPIs directly from sequence.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein-protein interactions (PPIs) are central to cellular processes and host-pathogen dynamics, yet bacterial interactomes remain poorly mapped, especially for extracellular effectors and cross-species interactions. Experimental approaches provide only partial coverage, while existing computational methods often lack generalizability or are too resource-intensive for proteome-scale application. Here, we introduce ppIRIS (protein-protein Interaction Regression via Iterative Siamese networks), a lightweight deep learning model that integrates evolutionary and structural embeddings to predict PPIs directly from sequence. Trained on curated bacterial datasets, ppIRIS achieves state-of-the-art accuracy across benchmarks while enabling proteome-wide screening in minutes. Applied to Group A Streptococcus (GAS), ppIRIS revealed functional clusters linked to virulence pathways, including nutrient transport, stress response, and metal scavenging. For host-pathogen predictions, ppIRIS recovered 56.2% of known GAS-human plasma interactions with enrichment in complement, coagulation, and protease inhibition pathways. Experimental validation confirmed novel predictions, demonstrating the applicability of ppIRIS for systematic discovery of bacterial and cross-species PPIs. The software is freely available at github.com/lupiochi/ppIRIS.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.677885v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Insights from farming Macrocystis pyrifera offshore: phenotypic analysis, genome-wide association studies, genomic selection</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kovalev, M., Arismendi, G. M., Wade, R. M., Molano, G., Miller, R., Reed, D., Alberto, F., Nuzhdin, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We found that outplanting season and farm design have significant effects on giant kelp biomass, carbon content and total biomass, and we identified three statistically significant SNPs, located near or within genes involved in carbohydrate metabolism and cytoskeletal functions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Seaweed farming, as a part of aquaculture, offers a sustainable alternative to modern agricultural practices; however, genetic enhancement and breeding programs for most species are underdeveloped. We aimed to advance seaweed domestication by focusing on giant kelp (Macrocystis pyrifera), the fastest-growing haplodiplontic brown alga, which has significant ecological and commercial importance. We analyzed phenotypic data from two offshore experimental farms conducted in 2019 and 2020, which involved hundreds of outplanted genetically diverse sporophytes. We found that outplanting season and farm design had significant effects on giant kelp biomass. Broad-sense heritability estimates showed moderate (0.27-0.50) genetic contributions to two phenotypes, carbon content and total biomass. Genome-wide association studies for these phenotypes resulted in three statistically significant SNPs, located near or within genes involved in carbohydrate metabolism and cytoskeletal functions. In addition, we applied genomic selection models that integrated sporophyte phenotypes and parental gametophyte genotypes. These models utilized reduced sets of GWAS-ranked SNPs obtained by a procedure based on linkage disequilibrium estimations. Model testing yielded cross-validation accuracy values of up to 0.84 and predictive accuracy values of up to 0.40, demonstrating the potential of marker-assisted breeding for phenotype improvement. Our results provide foundational genomic resources and tools for domesticating and breeding M. pyrifera, offering a basis for developing giant kelp varieties with desirable traits.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.677618v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AMICI: Attention Mechanism Interpretation of Cell-cell Interactions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hong, J., Desai, K., Nguyen, T. D., Nazaret, A., Levy, N., Ergen, C., Plitas, G., Azizi, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present AMICI, an interpretable attention framework that recovers ground-truth interactions in semi-synthetic data, uncovers gene programs linked to cell communication in the mouse cortex, and reveals length-scale-dependent tumor-immune signaling in breast cancer.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Spatial transcriptomic data enable study of cell-cell communication, yet current analysis tools often fail to provide dynamic, interpretable estimates of interactions and their spatial range across tissue. We present AMICI, an interpretable attention framework that jointly estimates interaction length scales, adaptively resolves sender-receiver subpopulations, and links communication to downstream gene programs. AMICI recovers ground-truth interactions in semi-synthetic data, uncovers gene programs linked to cell communication in the mouse cortex, and reveals length-scale-dependent tumor-immune signaling that reinforces estrogen receptor (ER) programs in breast cancer.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.677860v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Parallel algorithms for phylogenetic inference under a structured coalescent approximation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shao, Y., Suchard, M. A., Rambaut, A., Ji, X., Lemey, P., Vasylyeva, T. I., Baele, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a comprehensive algorithmic restructuring of the structured coalescent likelihood that eliminates redundancies, optimizes memory access, and exposes parallelization opportunities.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> While advances in molecular epidemiology and computational modeling have enhanced our capacity to track pathogen evolution, the accurate reconstruction of spatiotemporal transmission dynamics remains essential for developing epidemic preparedness frameworks and implementing outbreak response measures. Structured coalescent models offer a phylogeographic framework by restricting lineage coalescence events to geographically proximate host populations. Although the Bayesian structured coalescent approximation (BASTA) provides a tractable approach, contemporary phylogeographic analyses involving dozens of geographic localities and hundreds to thousands of viral genomes substantially exceed the computational capacity of existing implementations. The BASTA likelihood scales cubically with deme count and quadratically with sequence count due to matrix exponentiation and pairwise coalescent probability calculations. Here, we introduce a comprehensive algorithmic restructuring of the structured coalescent likelihood that eliminates redundancies, optimizes memory access, and exposes parallelization opportunities. Our approach reorganizes computations along three dimensions: (i) independent calculation of deme-transition probability matrices across time intervals; (ii) simultaneous evaluation of partial likelihood vectors within temporal slices; and (iii) concurrent aggregation of coalescent probabilities. Algorithmic restructuring cuts average coalescent likelihood computation by 7 to 8 fold, and parallelization further boosts performance to 10 to 26 fold, enabling joint phylogeographic analyses of dengue virus across 10 South American countries and H5N1 avian influenza across 20 Eurasian regions to finish in a fraction of prior time. This computational efficiency also enables comparison between backward-in-time structured coalescent approximations and forward-in-time phylogeographic methods, revealing that the former provides appropriately conservative posterior estimates, particularly at intermediate phylogenetic depths. We integrate our implementation into the popular BEAST X and BEAGLE software packages, with an accompanying interface in BEAUti X to easily set up the analyses, providing researchers with an accessible and scalable tool for real-time phylogeographic surveillance of rapidly evolving pathogens.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.677844v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Mapping Risk and Resilience Across Indo-Pacific Reefs with Shark Genomescapes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Johri, S., Araujo, G., Nevill, J., Daly, R., Reimer, T. E., Tyabji, Z., Jabado, R. W., Ichsan, M., Sani, I., Schallert, R., Curnick, D., Yu, C.-J., Block, B. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the Reefshark Genomescape, the first genome-wide reference database for Indo-Pacific reef sharks, an assessment of genetic diversity, structure, and connectivity of two key species across their Indo- Pacific range and geographic assignment of fished individuals using population-specific genetic signatures.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Overfishing has severely depleted marine populations worldwide, including within protected areas. Illegal and unreported fishing are major contributors to this decline. Large-bodied apex predators such as sharks are among the most affected, with overfishing causing dramatic species declines and ecosystem destabilization due to trophic downgrading. Key barriers to effective marine conservation and management include: Data deficiencies that hinder population benchmarks and impact assessments, limited surveillance, allowing illegal fisheries to disproportionately affect apex predators, and insufficient capacity in vulnerable nations to monitor and protect species within their waters. Our study addresses these challenges through a novel genomic framework that enables assessment of shark population diversity and health, while also improving fisheries traceability by detecting instances of illegal fishing across the Indian and Pacific Oceans. We present the Reefshark Genomescape, the first genome-wide reference database for Indo-Pacific reef sharks, an assessment of genetic diversity, structure, and connectivity of two key species across their Indo-Pacific range and geographic assignment of fished individuals using population-specific genetic signatures. We show that grey reef shark (Carcharhinus amblyrhynchos) populations exhibit high genetic diversity, strong population structure, and elevated Fst values, with previously unknown connectivity between the central and western Indian Ocean and clear isolation of populations in the Andaman Sea. In contrast, silvertip sharks (Carcharhinus albimarginatus) display high connectivity, but show genomic signals of declining population health, supporting a reassessment of their IUCN status. Using supervised machine learning with Monte Carlo cross-validation, we assigned geographic origins to fished grey reef sharks with 96% accuracy. These findings provide critical insights into population structure, connectivity, and health of two ecologically important reef shark species, while establishing a robust method for assigning geographic origin. We anticipate this framework will support regional conservation assessments and targeted management. Moreover, by enabling the identification of fishing hotspots and detection of IUU fishing, it lays the groundwork for a broader traceability system in marine ecosystems. Much like the landmark elephant ivory tracing study, our approach has the potential to transform marine conservation globally.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.676358v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">NEST-Scoring, a Novel miRNA Target Gene Profile Network Analysis of Placenta-Derived Extracellular Vesicles in a Transgenic Rat Model of Preeclampsia</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Westwood, B. M., Ray, J., Sun, X., Su, Y., Cruz-Diaz, N., Singh, S., Lewis, N., Kumar, A., Deep, G., Pulgar, V., Yamaleyeva, L. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Preeclampsia is a multisystem hypertensive disorder of pregnancy and a leading cause of maternal morbidity and mortality, the underlying mechanisms remain incompletely understood.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Preeclampsia is a multisystem hypertensive disorder of pregnancy and a leading cause of maternal morbidity and mortality. Despite its increasing incidence and the debilitating nature of its cerebrovas-cular complications, the underlying mechanisms remain incompletely understood. The goals of this study were to 1) determine whether middle cerebral artery (MCA) hemodynamics are altered in late gestation (LG) or two months postpartum in transgenic rats with preeclampsia compared to normal pregnant Sprague-Dawley (SD) rats, and 2) evaluate the microRNA (miRNA) profiles of circulating pla-cental-derived extracellular vesicles (EVPD) using the Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway framework using NEST-Scoring [Node, Edge, System, Topology], a novel target gene profile network analysis. Our data showed elevated blood pressure in transgenic preeclampsia rats at both LG and two months postpartum (PP-2M) compared to normal pregnant SD rats in the setting of hypertension. The heart rate was also significantly higher in preeclampsia rats at LG. No differences were observed in left MCA (LMCA) velocities between strains or time points. However, LMCA re-sistance and pulsatility indexes were lower at PP-2M in transgenic preeclampsia rats compared to SD rats. Exposure to preeclampsia differentially altered the miRNA profiles of circulating EVPD in transgenic preeclampsia rats at both LG and PP-2M. The identified miRNA targets were associated with key vascu-lar and cellular regulatory pathways, suggesting a mechanistic link between placental dysfunction and postpartum vascular alterations in this preeclampsia model. Overall, this study suggests that the TGA-PE rat model could be a valuable platform to study the mechanistic link between preeclampsia and cerebrovascular disease.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.675363v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Single Cell Foundation Models Evaluation (scFME) for In-Silico Perturbation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Boylan, J., Solovyeva, E., Bouiller, T., Liu, X., Hoersch, S., Ataman, B., Jenkins, J., Devarakonda, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose Single-Cell Foundation Model Evaluation (scFME), a systematic method designed to benchmark fine-tuned foundation models for insilico perturbation (ISP).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Foundation models pre-trained on large single-cell RNA atlases offer a compelling alternative to in-vitro experimentation for understanding gene regulatory networks and conducting gene perturbation analyses, with significant implications for target identification. Numerous foundation models have been developed, building upon early efforts such as Geneformer and scGPT. Hyperparameter optimization also results in multiple variants which require comparative analysis. Current benchmarking approaches focus on feature-based assessments or intuitive biological and statistical tasks, which may not align with the models' training objectives. A recent study proposed a systematic benchmarking framework; however, its scope was limited to pre-trained (zero-shot) models. To address these limitations, we propose Single-Cell Foundation Model Evaluation (scFME)-a systematic method designed to benchmark fine-tuned foundation models for insilico perturbation (ISP). scFME ensures comprehensive and robust assessment by requiring sufficient separation between control and perturbed cells at the outset and by quantifying ISP accuracy against zero and random perturbation baselines. Furthermore, scFME enables exploration of model performance across distinct gene categories, facilitating biological interpretation and functional relevance. Using this framework, we evaluated several commonly used models (and some of their variants) and demonstrated that the methodology effectively characterizes their performance in ISP studies. Our results position scFME as a versatile and rigorous methodology for evaluating and comparing current and future foundation models.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.677811v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comparative transcriptomics reveals an extracellular worm argonaute as an ancestral regulator of LTR retrotransposons.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Martinez-Ugalde, I., Neophytou, K., Villagrana-Pacheco, Y., Orrego Duranona, A., Stevens, L., Du, X., Bancroft, R., Hall, J. L., Pedersen, A. B., Blaxter, M., Buck, A. H., Abreu-Goodger, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that exWAGO is part of an evolutionarily conserved pathway for LTR retrotransposon regulation, while preferentially utilising degenerated elements as sources of secreted sRNAs.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Safeguarding the genome from non-self elements is essential for reproduction, development, and ageing. One of the major threats to genome integrity is Transposable Elements (TEs), which can be post-transcriptionally silenced through small RNAs (sRNAs) and argonaute proteins. Recent work suggests TE-derived sRNAs may also act as virulence factors in host-pathogen interactions. During infection, the intestinal parasite Heligmosomoides bakeri secretes a single argonaute protein (exWAGO) and a wide variety of TE-derived sRNAs. Although exWAGO is highly expressed, conserved, and secreted by parasitic nematodes, its function and sRNA guide preference remain unclear. Using comparative transcriptomics of the sRNAs bound to exWAGO within parasites of rodents, livestock and humans, and its orthologs in C. elegans, we found that exWAGO is capable of loading sRNAs produced from all classes of TEs in addition to some protein-coding and non-coding transcripts. However, our results suggest that the ancestral endogenous function of exWAGO was likely linked to LTR retrotransposon regulation. To understand how this relates to potential extracellular functions of exWAGO we also examined the sRNAs bound to exWAGO secreted by H. bakeri in both vesicular and non-vesicular forms. Extracellular exWAGO preferentially loads sRNA guides derived from non-autonomous and fragmented LTRs, suggesting the existence of adaptable reservoirs of regulatory sRNAs with potential roles in cross-species RNA communication. Together, our results show that exWAGO is part of an evolutionarily conserved pathway for LTR retrotransposon regulation, while preferentially utilising degenerated elements as sources of secreted sRNAs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.674508v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genetic diversity and variation in heterozygosity in Macadamia</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Manatunga, S., Furtado, A., Topp, B., Alam, M., Mason, P., Henry, R. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A genome-wide study of the genetic diversity and heterozygosity of macadamia genotypes .</p><p style="font-size: 16px; text-align: justify;">Background Macadamia is a diploid (2n =28) rainforest tree of the Proteaceae family, native to Australia. Single-nucleotide polymorphisms (SNPs) have proven to be effective tools for evaluating genetic diversity and heterozygosity. SNP from the whole genome were used to determine allelic diversity within different species and breeding populations and to investigate the heterozygosity of genotypes Results A total of 349 samples including wild and domesticated genotypes were whole genome sequenced, generating 4,180,786 to 7,661,269, 2,446,762 to 7,356,187, 2,829,944 to 4,391,600, 1601519 to 3,182,089 and 3,123,715 to 9,620,640 single nucleotide polymorphism (SNP) positions at the whole genome level for M. integrifolia, M. tetraphylla, M. ternifolia, M. jansenii and the domesticated cultivars and breeding selections respectively. The overall SNP diversity for wild M. integrifolia, M. tetraphylla, M. ternifolia and M. jansenii was 4.43%, 4.33%, 1.80% and 1.1%, respectively. Domesticated M. integrifolia showed an overall SNP diversity of 3.4%, 1.03% lower than wild M. integrifolia, providing evidence of a genetic bottleneck during domestication. Genetic diversity at the individual level for the domesticated gene pool varied from 0.4% to 1.24%, with an overall diversity of 6.10%. Results across different breeding groups revealed that the Australian breeding selections (NMBPA) had the highest diversity, and HAES cultivars had the lowest diversity. Wild M. integrifolia and M. tetraphylla exhibited heterozygosity ranging from 0.36% to 0.71% and 0.30% to 0.71% respectively. The two non-edible species, M. ternifolia and M. jansenii, displayed relatively low heterozygosity (From 0.29% to 0.42% and 0.19% to 0.34% respectively). Domesticated macadamia exhibited a wide range of heterozygosity (0.33% to 1.15%). Additionally, heterozygosity correlated with yield efficiency (kernel yield/tree volume), cumulative kernel yield at age 8 and cumulative nut in shell (NIS) yield at age 8 and showed that performance increased with heterozygosity to an optimal level and declined thereafter. Conclusion These findings confirmed that there is substantial genetic variation among the different macadamia genotypes. The results also revealed that the greatest diversity is present in wild germplasm. The data will greatly support future genome-wide studies and breeding programs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.677927v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Computational method for mapping mass signatures along developmental gradients reveals a novel role for a monosaccharide tetrose in maize salt-stress response.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sama, A. M., Cahill, S. B., Luo, S., Tripka, A. L., Meng, Y., Noll, S. E., Zare, R. N., Shah, P., Dickinson, A. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We used desorption electrospray ionization mass spectrometry imaging (DESI-MSI) to map metabolites in the developing maize root, which has a well-characterized longitudinal gradient that encompasses developmental transitions from quiescence through proliferation and maturation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Metabolic processes are essential for regulating and maintaining developmental transitions, from stem cell quiescence through differentiation. However, the distinct metabolite-driven mechanisms that are critical for development remain poorly characterized due to inherent challenges in measuring their production, localization, and function in situ. We employed desorption electrospray ionization mass spectrometry imaging (DESI-MSI) to map metabolites in the developing maize root, which has a well-characterized longitudinal gradient that encompasses developmental transitions from quiescence through proliferation and maturation. DESI-MSI enables in situ analysis of the chemical composition of tissue sections with high spatial resolution (~50-100 {micro}m). To identify metabolites with specific developmental enrichment patterns, we developed a computational approach called Developmental Imaging Mass Spectrometry Pipeline for Linear Evaluation (DIMPLE). DIMPLE processes mass signatures along linear gradients, generating clusters of metabolites with specific developmental enrichment patterns in the maize root. We employed this method to compare developmental enrichment of metabolites in a salt-resilient maize variety, Oaxacan Green, and a salt-susceptible variety, B73. DIMPLE identifies specific differences in the mass signatures and the overall enrichment patterns between these varieties. DIMPLE also revealed a metabolite, D-erythrose, that had different localization patterns in these varieties. We found that in salt-sensitive maize varieties, treatment with D-erythrose improves stress tolerance by increasing primary root length. Overall, DIMPLE enables comprehensive and rapid analysis of metabolite patterns along a linear gradient, revealing new biology in plant growth and stress response.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.677919v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The Arabidopsis nucleoporins 98A/B are essential for NLP7 nuclear accumulation in response to nitrate</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kelemen, Z., Duville, G., Brehaut, V., Krapp, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that nitrate-triggered nuclear accumulation of the master transcriptional regulator NLP7, a member of the RWP-RK family of transcription factors, triggers a major gene expression cascade.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The nuclear compartment is an essential evolutionary gain in eukaryotes. Stimulus-induced nuclear accumulation of transcription factors allows rapid and reversible regulation of gene expression. Nitrate-regulated nuclear accumulation of the master transcriptional regulator NLP7, a member of the RWP-RK family of transcription factors, triggers a major gene expression cascade. Whether and how this process is regulated at the level of the nuclear pore remains elusive. Here we show that the mobile nucleoporin NUP98A and its close homolog NUP98B interact with NLP7 in the nucleus and that the nitrate-triggered nuclear accumulation of NLP7 is impaired in their absence. We also showed that the simultaneous loss of NLP7 and NUP98A strongly and specifically alters the expression of nitrate-regulated genes and drastically reduces plant growth. Our results indicate that plant NUP98 acts on nitrate use by plants through its interaction with the top-tier NLP7 transcription factor.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.677877v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genetically encoded RNA strand exchange circuits for programmable protein expression and computation in cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Schaffter, S. W., Vasilyeva, O. B., Wintenberg, M. E., Hurley, J. M., Alperovich, N. Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show the first demonstration of genetically encoded RNA strand exchange circuits, designed analogously to state-of-the-art TMSE circuits, in living cells.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Programmable cellular information processing could advance biomanufacturing of chemicals and medicine, and enable smart, living therapeutics and diagnostics. Nucleic acids circuits based on toehold mediated strand exchange (TMSE) show tremendous potential for cellular programming due to their scalable, composable, and biocompatible parts. However, these circuits are constrained primarily to in vitro applications because genetically encoding them is challenging and the principles of TMSE in cells remain unknown. Here we show the first demonstration of genetically encoded RNA strand exchange circuits, designed analogously to state-of-the-art TMSE circuits, in living cells. To elucidate the design principles of TMSE in cells, we develop toehold exchange riboregulators, which convert TMSE to protein expression, enabling precise control of protein translation rate. We find many of the design principles and parts used for TMSE in vitro transfer to Escherichia coli, allowing construction of multi-layer cascades and logic elements. We also identify caveats where strand exchange in cells differ substantially from cell-free systems, even in lysate from the same bacterial strain, suggesting active cellular processes are involved. Our results further highlight bounds on strand exchange circuit architectures feasible in cells. We anticipate this study will lay the groundwork for developing advanced cellular circuits, bringing nucleic acid computing from the test tube to the cell and enabling new applications by connecting TMSE to gene expression. More broadly, our results have implications for RNA:RNA interactions and gene regulation in bacteria and provide a synthetic system for exploring such phenomena.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.24.678369v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Synthetic microbial co-cultures for modular bioelectronic sensing in diverse environments</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Li, S., Zhu, D., Saha, K., Kundu, B. B., Sonkusale, S., Britton, R. A., Ajo-Franklin, C. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the electroactive co-culture sensing system (e-COSENS), a plug-and-play platform for bioelectronic sensor development and a centimeter-sized bioelectronics device that enables low-cost, portable signal readout using a household digital multimeter.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Human disruption of ecosystems poses a significant threat to global health, driving the need for low-cost, low-power, and easily deployable sensors for environmental and health monitoring. Microbial bioelectronic sensors are particularly well-suited as they generate electrical signals and can be integrated into compact electronic devices for field deployment over extended periods. However, current engineering strategies for bioelectronic sensors lack modularity, are limited to a few microbial chassis, and depend on specialized instruments for signal detection. Here, we present the electroactive co-culture sensing system (e-COSENS), a plug-and-play platform for bioelectronic sensor development. This system comprises a 'sender' bacterium that produces electron mediators in response to analytes and a 'receiver' bacterium that utilizes the electron mediators to generate electrical signals via extracellular electron transfer (EET). By modularly swapping the sender bacterium and its associated genetic sensing elements, we achieved bioelectronic sensing of metals, small molecules, and peptides in distinct environmental, food, and human-relevant settings. Moreover, we designed a centimeter-sized bioelectronic device that enables low-cost, portable signal readout from e-COSENS using a household digital multimeter. The e-COSENS platform greatly simplifies the bioelectronic sensor design and opens unprecedented potential for bioelectronic sensor applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.24.678173v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Inferring Dynamic Information from Protein Structures by Gaussian Integrals and Deep Learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Vilicich, F., Yin, S., Su, Z., Wu, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A deep learning framework that predicts protein flexibility directly from static structural descriptors, bypassing the need for molecular dynamics simulations, with potential applications in structural bioinformatics, drug design, and functional annotation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein conformational flexibility underlies a wide range of biological functions, yet experimentally probing dynamics at atomic resolution remains costly and low-throughput. Here, we present a deep learning framework that predicts protein flexibility directly from static structural descriptors, bypassing the need for molecular dynamics (MD) simulations. Using the ATLAS database of standardized all-atom MD trajectories, we encoded 1,374 protein chains as 30-dimensional Gaussian integral (GI) vectors, global shape and topology invariants of the protein backbone. Principal component analysis of GI profiles revealed four structural clusters with distinct secondary structure compositions and flexibility distributions. We trained an attention-based one-dimensional convolutional neural network (1D-CNN) to classify proteins as flexible or non-flexible based on their root-mean-square fluctuation (RMSF) relative to the dataset-wide mean. The classifier achieved an AUC of 0.772 (95% CI: 0.712-0.826) on an independent test set, with balanced sensitivity and specificity, and identified a small subset of GI components as the most predictive. In a regression setting, a recurrent neural network outperformed other architectures, attaining an R2 of 0.537, though high-flexibility values were systematically underestimated. Cluster-specific analyses indicated that coil-rich and {beta}-sheet-dominated proteins were more amenable to flexibility prediction than -helical proteins, likely due to greater structural heterogeneity. Our results demonstrate that compact GI descriptors preserve sufficient information to recover MD-derived flexibility trends, offering a computationally efficient complement to simulation-based approaches. This framework enables large-scale screening of protein dynamics from structural data alone, with potential applications in structural bioinformatics, drug design, and functional annotation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.677755v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SpliceUp: Predicting SF3B1 mutations in individual cells via aberrant splice site activation from scRNA-seq data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kholmatov, M., Prummel, K. D., Guezguez, B., Lindberg, E. H., Moura, P. L., Zaugg, J. B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed SpliceUp, a computational tool that identifies splice factor-mutant cells through their aberrant RNA splicing signatures.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Myeloid neoplasms (MN) are clonal heterogeneous disorders initiated by somatic driver mutations in hematopoietic stem and progenitor cells (HSPCs). Among the most common are mutations in RNA splicing factors, which exert pleiotropic effects but are difficult to study due to altered hematopoietic differentiation and the inability to specifically isolate mutant HSPCs. Single-cell transcriptomics offers a powerful framework to dissect mutant cell states, yet direct genotyping is hampered by data sparsity and often requires costly, labor-intensive approaches. To overcome this limitation, we developed SpliceUp, a computational tool that identifies splice factor-mutant cells through their aberrant RNA splicing signatures. Applied to SF3B1-mutant samples, SpliceUp exploits cryptic 3' splice site usage and exon-skipping events to achieve more than a 7-fold increase in mutant cell detection from low-coverage 10x Genomics datasets. Differential expression analysis revealed cell type-specific programs, including MHC-II upregulation in SF3B1-mutated HSCs and enhanced RNA translation in SF3B1-mutated erythroid precursors.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.677806v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Biomechanical 3D tumor models on a micro-milled high-throughput force sensor array</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Emon, B., Kashefi, A., Rahman, M. H., Adnan, D., Ospina-Munoz, N., Mellican, S., Santiago, S., Drennan, W. C., Joy, M. S. H., Phan, A. A., Afrin, T., Han, B., Smith, K. C., Bishehsari, F., Saif, M. T. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a 3D tumor model integrated with a high-throughput biomechanical sensor array that enables simultaneous measurement of cellular forces, matrix remodeling, and molecular transport.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The tumor microenvironment plays a critical role in drug resistance, with extracellular matrix (ECM) mechanics, cell-cell crosstalk, and transport barriers contributing to poor therapeutic outcomes. Traditional two-dimensional (2D) cultures fail to capture these features, and drug efficacy in 2D often does not translate to three-dimensional (3D) models or in vivo tumors. Here, we introduce a 3D tumor model integrated with a high-throughput biomechanical sensor array that enables simultaneous measurement of cellular forces, matrix remodeling, and molecular transport. Fabricated using a scalable and cost-effective micro-milling approach, the platform allows parallel generation of multiple tumor constructs within a single dish. Using patient-derived pancreatic ductal adenocarcinoma (PDA) cells and stromal fibroblasts, we show that responses to gemcitabine and all-trans retinoic acid (ATRA) in 3D differ markedly from 2D cultures, consistent with clinical observations. By integrating biochemical and biomechanical readouts, this technology provides a more physiologically relevant tumor model and a powerful tool for preclinical drug testing and personalized medicine.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.677940v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Breast Cancer Subtyping with HyperCLSA: A Hypergraph Contrastive Learning Pipeline for Multi-Omics Data Integration</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bhole, G., HC, P., Jayachandran, M., PK, V., Bhimalapuram, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose HyperCLSA (Hypergraph Contrastive Learning with Self-Attention), a novel deep learning framework for efficient multi-omics integration in breast cancer subtyping.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurate molecular subtyping of cancer is crucial for advancing personalized medicine. Although multiomics data contain valuable predictive information, effectively integrating them is challenging due to differences across modalities, high dimensionality, and complex cross modal biological interactions. We propose HyperCLSA (Hypergraph Contrastive Learning with Self-Attention), a novel deep learning framework for efficient multi-omics integration in breast cancer subtyping. HyperCLSA combines hypergraph-based sample encoding, supervised contrastive learning for latent space alignment, and multi-head self-attention for adaptive fusion of omics modalities. Evaluated on The Cancer Genome Atlas (TCGA) Breast Cancer dataset for PAM50 subtype classification, HyperCLSA achieves a significant performance improvement over state-of-the-art baselines. Our results demonstrate HyperCLSAs effectiveness in extracting complementary information across heterogeneous omics sources, providing a robust framework for molecular characterization of breast cancer.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.677517v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Role of tankyrase scaffolding in the β-catenin destruction complex and WNT signaling</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, Q., Li, L., You, L., Wang, S., Han, L., Wang, B., Yao, L., Lu, Y., Mender, I., Flusche, A. M., Kim, C., Yarravarapu, N., Lemoff, A., Lum, L., Shay, J. W., Yu, Y., Chen, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Aberrant WNT/{beta}-catenin signaling drives tumorigenesis and metastasis in cancer, and the induction of AXIN puncta represents a major barrier to achieving therapeutic efficacy.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Aberrant WNT/{beta}-catenin signaling drives tumorigenesis and metastasis in cancer. Although enzymatic inhibitors of tankyrase (TNKS) effectively block AXIN degradation and stabilize the {beta}-catenin destruction complex (DC), they have demonstrated limited efficacy in various cancer models. Here we demonstrate that, unexpectedly, the induction of AXIN puncta represents a major barrier to achieving therapeutic efficacy. Mechanistically, catalytic inhibition of TNKS prevents TNKS turnover and drives its accumulation in the DC, wherein the scaffolding function of TNKS induces AXIN puncta formation, rigidifies the DC, and impedes {beta}-catenin turnover. Chemically induced degradation of TNKS overcomes this limitation by stabilizing AXIN without puncta formation, providing a deeper suppression of the WNT/{beta}-catenin pathway activity and the proliferation of colorectal cancer cells harboring dysfunctional APC mutations. Collectively, these findings provide an explanation for the unsatisfactory outcomes of drugging the WNT/{beta}-catenin signaling pathway by TNKS inhibitors and highlight TNKS degradation as a promising approach to treat WNT/{beta}-catenin-driven cancers.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.677768v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Full-length protein classification via cysteine fingerprinting in solid-state nanopores.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Neeraj Soni, Zohar Rosenstock, Navneet C Verma, Krishnan Siddharth, Noam Talor, Jingqian Liu, Barak Marom, Anatoly B Kolomeisky, Aleksei Aksimentiev, Amit Meller</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a click-chemistry-based strategy for covalently attaching short oligonucleotides to cysteine residues on denatured proteins across a broad range of molecular weights.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent advances in single-molecule technologies are transforming the field of protein analysis. Solid-state nanopores provide an effective method to linearize and thread full-length proteins in a single file. However, slowing their rapid translocation remains a challenge for accurate, time-resolved ion-current-based fingerprinting. In this work, we present a click-chemistry-based strategy for covalently attaching short oligonucleotides to cysteine residues on denatured proteins across a broad range of molecular weights. The negatively charged oligonucleotides increase the capture rate by a factor of ten compared with native proteins and induce a distinct 'stick-slip' motion that slows protein passage through the nanopore by more than 20-fold. These oligonucleotide tags also produce characteristic, time-resolved ion current pulses that serve as unique protein-specific signatures. To uncover the physical mechanism responsible for the protein translocation dynamics, we model our system using all-atom molecular dynamics and finite element simulations. By leveraging a supervised machine learning classifier, we demonstrate that a small number of translocation events is sufficient to identify individual proteins, achieving near-perfect classification accuracy. To demonstrate the robustness of the method, we successfully distinguish between VEGF-A isoforms (VEGF-165 and VEGF-121), which are relevant to cancer diagnostics, within a mixed protein sample. Our nanopore-based fingerprinting technique eliminates the need for affinity reagents, such as protein-specific antibodies, or motor proteins, offering a rapid, direct and cost-effective approach for single-molecule protein identification and classification.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40993352/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Accessible, uniform protein property prediction with a scikit-learn based toolset AIDE.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Evan Komp, Kristoffer E Johansson, Nicholas P Gauthier, Japheth E Gado, Kresten Lindorff-Larsen, Gregg T Beckham</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Artificial Intelligence Driven Protein Estimation (AIDE) for protein property prediction with and without labeled data .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein property prediction via machine learning with and without labeled data is becoming increasingly powerful, yet methods are disparate and capabilities vary widely over applications. The software presented here, "Artificial Intelligence Driven protein Estimation (AIDE)," enables instantiating, optimizing, and testing many zero-shot and supervised property prediction methods for variants and variable length homologs in a single, reproducible notebook or script by defining a modular, standardized application programming interface (API) that is drop-in compatible with scikit-learn transformers and pipelines.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40991335/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Radiomics-based artificial intelligence (AI) models in colorectal cancer (CRC) diagnosis, metastasis detection, prognosis, and treatment response prediction.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Reza Elahi, Parsa Karami, Mohammadreza Amjadzadeh, Mahdis Nazari</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the latest evidence on the clinical applications of radiomics models based on different imaging modalities in CRC and discuss the challenges facing clinical translation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Colorectal cancer (CRC) is the third most common cause of cancer-related morbidity and mortality in the world. Radiomics and radiogenomics are utilized for the high-throughput quantification of features from medical images, providing non-invasive means to characterize cancer heterogeneity and gain insight into the underlying biology. Such radiomics-based artificial intelligence (AI)-methods have demonstrated great potential to improve the accuracy of CRC diagnosis and staging, to distinguish between benign and malignant lesions, to aid in the detection of lymph node and hepatic metastasis, and to predict the effects of therapy and prognosis for patients. This review presents the latest evidence on the clinical applications of radiomics models based on different imaging modalities in CRC. We also discuss the challenges facing clinical translation, including differences in image acquisition, issues related to reproducibility, a lack of standardization, and limited external validation. Given the progress of machine learning (ML) and deep learning (DL) algorithms, radiomics is expected to have an important effect on the personalized treatment of CRC and contribute to a more accurate and individualized clinical decision-making in the future.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40991016/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrating of serum pharmacochemistry, metabolomics, and experimental validation to explore the mechanism of Erzhi Pill against Triple-negative breast cancer.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yu Qi, Pei Li, Binqing Li, Panpan Lv, Hairui Hu, Chengkai Tan, Wenyan Wang, Yuewen Jiang, Jinrong He</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new treatment for triple-Negative Breast Cancer using Erzhi Pill, a traditional Chinese medicine used for various cancers.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Erzhi Pill (EZP), a traditional Chinese medicine used for various cancers, exhibits antitumor potential, though its role and mechanisms in Triple-Negative Breast Cancer (TNBC) remain unclear.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40659139/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

