<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Monday, 01 September 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Personalized Genotype-Based Approach for Treatment of Phenylketonuria.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gundorova P, Yousefi B, Woidy M, Rose-Heine MS, Khatri R, Kasten V, Bonn S, Muntau AC, Gersting SW.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Extensive studies have examined the clinical manifestations, pathogenic mechanisms, and genetic variations of phenylketonuria (PKU) across different populations, resulting in a substantial collection of molecular genetic data on the phenylalanine hydroxylase (PAH) gene and its variants. However, many genotypes are associated with a range of clinical phenotypes, as well as variable responsiveness to sapropterin, presenting ongoing challenges for effective treatment. To address this, we enhanced the PAH activity landscapes method by incorporating high-throughput techniques, including automated pipetting, integrated data processing via Gaussian modeling of 3D surfaces, and bioinformatics analyses with robust quality control. Using PAH activity landscapes, we visualized PAH enzymatic function across 99 common PAH genotypes under varying metabolic and therapeutic conditions. This deep functional phenotyping approach enabled us to identify distinct genotype subpopulations by using consensus clustering, correlate them with clinical phenotypes, and propose subpopulation-specific treatment protocols. Our findings suggest that clinical phenotypes can be predicted and treatment regimens can be adjusted based on residual PAH function profiles. To further support personalized treatment strategies, we revised our publicly accessible PAH genotype & activity landscapes database to share the latest insights into PAH function and patient phenotypes-namely residual enzyme activity and responsiveness to sapropterin as conveyed by two alleles. This resource underscores the translational significance of functional research in PKU and offers a practical tool to support personalized treatment in clinical settings.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40730740/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Seed protein electrophoresis in plant genetics: Commemorating the pioneering contributions of Prof. Chittaranjan Kole and team to the foundation of plant proteomics.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pandey S, Tiwari S, Bhattacharjee P, Anand A, Pazhamala LT, Yasin JK, Yadav RK.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #fedbf2; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, pan-genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Despite the rapid progress of proteomics in human and other model organisms, plant proteomics has advanced at a comparatively slower pace. This review aims to highlight the pioneering work on seed protein markers detected by employing gel electrophoresis primarily by a team of Indian scientists that paved the way for elucidation of intervarietal and interspecific variation, evolution, and phylogenetic relationship of species and their association with resistance to pest and diseases. Far from being replaced, gel electrophoresis remains as an excellent supporting and different approach, offering a pathway to a more profound visualization and understanding of the cell proteome. This review focuses on how, from a historical standpoint, gel electrophoresis has significantly contributed to plant proteomics and other biological research. Acknowledging the pioneering work on seed storage proteins, this review serves as both a congratulatory gesture and a tribute to the eminent scientist Prof. Chittaranjan Kole and his team who pioneered the strategy of seed protein electrophoresis in crop biology research. Their findings, both directly and indirectly, have proven invaluable, particularly for those who ventured into proteomics without easy reach to sophisticated and expensive instruments/equipment to pursue DNA-based genomics research. This gel electrophoresis-based plant proteomics review includes the evolution of gel-based proteomics, their contribution to crop biology research, and future directions. It stands not only as a retrospective analysis but also as a testament to the enduring significance of gel electrophoresis in shaping the landscape of crop proteomics.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40646713/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Modern Approaches for Thoracic Image Registration and Respiratory Motion Management in Oncology.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jank E, Liu EY, Delery W, Boyle P, Miller C, Andosca R, Naumann L, Upadhyay R, Kadambi A, Low D, Lauria M, Savjani RR.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> This review explores modern image registration techniques in the context of managing respiratory-induced motion in the thoracic region. The respiratory cycle introduces anatomic variability that poses challenges for radiation therapy, breathing dynamics modeling, and diverse research applications. Conventional and advanced motion management strategies are presented, emphasizing the critical role of robust image registration. Emerging approaches such as multimodal and interpatient registration are discussed, alongside metrics for assessing registration quality. This review aims to enhance understanding of recent developments in image registration and thoracic motion management. Continued progress in modeling respiratory dynamics is essential to support advanced research applications and clinical innovation. Thorax, Deep Learning, Machine Learning, Radiation Therapy Supplemental material is available for this article. ¬© RSNA, 2025.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40815223/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Current progress of digital twin construction using medical imaging. </h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhao F, Wu Y, Hu M, Chang CW, Liu R, Qiu R, Yang X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Medical imaging is fundamental to digital twin technology, enabling patient-specific virtual models of anatomy and physiology. By integrating high-resolution modalities (Magnetic Resonance Imaging (MRI), Computed Tomography (CT), Positron Emission Tomography (PET), ultrasound) with computational frameworks, recent imaging advances now support real-time simulation, predictive modeling, and earlier disease detection. Such capabilities directly inform individualized treatment planning and contribute to more precise, personalized care. Despite remaining challenges-complex anatomical modeling, multimodal integration, and high computational demands-recent advances in imaging and machine learning have significantly enhanced the accuracy and clinical utility of digital twins. The main contributions of our review are: (1) a system-by-system classification of methodologies; (2) evidence that advanced imaging modalities have improved diagnostic accuracy, treatment effectiveness, and patient outcomes beyond conventional approaches; and (3) identification of remaining technical bottlenecks. We further analyze key technical barriers-such as data scarcity and computational complexity-and outline future directions (e.g., AI-driven data augmentation, real-time model optimization) to unlock digital twins' full potential in precision medicine.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40841176/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Accurate Paediatric Brain Tumour Classification Through Improved Quantitative Analysis of &lt;sup&gt;1&lt;/sup&gt;H MR Imaging and Spectroscopy.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhao T, L Rose HE, Grist JT, MacPherson L, Li H, Arvanitis TN, Apps JR, Peet AC.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Multimodality imaging is an emerging research topic in neuro-oncology for its potential of being able to demonstrate tumours in a more comprehensive manner. Diffusion-weighted magnetic resonance imaging (dMRI) and proton magnetic resonance spectroscopy (1H-MRS) allow inferring tissue cellularity and biochemical properties, respectively. Combining dMRI and 1H-MRS may provide more accurate diagnosis for paediatric brain tumours than only one modality. This retrospective study collected 1.5-T clinical 1H-MRS and dMRI from 32 patients to assess paediatric brain tumour classification with combined dMRI and 1H-MRS. Specifically, spectral noise of 1H-MRS was suppressed before calculating metabolite concentrations. Extracted radiomic features were apparent diffusion coefficient (ADC) histogram features through dMRI and metabolite concentrations through 1H-MRS. These features were put together and then ranked according to the multiclass area under the curve (mAUC) and selected for tumour classification through machine learning. Tumours were precisely typed by combining noise-suppressed 1H-MRS and dMRI, and the cross-validated accuracy was improved to be 100% according to na√Øve Bayes. The finally selected radiomic biomarkers, which showed the highest diagnostic ability, were ADC fifth percentile (mAUC = 0.970), myo-inositol (mAUC = 0.952), combined glutamate and glutamine (mAUC = 0.853), total creatine (mAUC = 0.837) and glycine (mAUC = 0.815). The study indicates combining MR imaging and spectroscopy can provide better diagnostic performance than single-modal imaging.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40702777/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">RENOVO-NF1 accurately predicts NF1 missense variant pathogenicity.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Emanuele Bonetti, Serena Pellegatta, Nayma Rosati, Marica Eoli, Luca Mazzarella</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Identification of a pathogenic variant in NF1 is diagnostic for neurofibromatosis, but is often impossible at the moment of variant detection due to many factors including allelic heterogeneity, sequence homology, and the lack of functional assays. Computational tools may aid in interpretation but are not established for NF1. Here, we optimized our random forest-based predictor RENOVO for NF1 variant interpretation. RENOVO was developed using an approach of "database archaeology": by comparing versions of ClinVar over the years, we defined "stable" variants that maintained the same pathogenic/likely pathogenic/benign/likely benign (P/LP/B/LB) classification over time (n = 3579, the training set), and "unstable" variants that were initially classified as Variants of Unknown Significance (VUS) but were subsequently reclassified as P/LP/B/LB (n = 57, the test set). This approach allows to retrospectively measure accuracy on prediction with insufficient information, reproducing the scenario of maximal clinical utility. We further validated performance on: (i) validation set 1: 100 NF1 variants classified as VUS at the time of RENOVO development and subsequently reclassified as P/LP/B/LB in ClinVar; (ii) validation set 2: 15 de novo variants discovered in a prospective clinical cohort and subsequently reclassified per ACMG criteria. RENOVO obtained consistently high accuracy on all datasets: 98.6% on the training test, 96.5% in the test set, 82% in validation set 1 (but 96.2% for missense variants) and 93.7% on validation set 2. In conclusion, RENOVO-NF1 accurately interprets NF1 variants for which information at the time of detection is insufficient for ACMG classification and may overcome diagnostic challenges in neurofibromatosis.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40887659/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multiple origins of a sex ratio supergene in Formica ants</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lagunas-Robles, G., Purcell, J., Shimota, E., Brelsford, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Many known supergenes, regions of the genome containing two or more functional mutations with little or no recombination between them, formed in recent evolutionary time to control complex traits. However, when supergenes persist through millions of years of evolution, they may take on additional functions in different lineages. Here, we investigate how an ancient supergene has evolved to gain novel function. In Formica ants, most species have an ancient "social" supergene with M and P haplotypes which determine whether a colony is headed by a single queen or multiple queens, respectively. At least three Formica species have an additional haplotype on the same chromosome, termed MD, associated with female-biased offspring sex ratio at the colony level. Is an MD haplotype present in additional species? Do the MD haplotypes share a common origin or did they evolve convergently? With whole-genome resequencing, we identify MD haplotypes in three additional species and use variation across six Formica species to examine the evolutionary history of the MD haplotype. We find that the MD haplotype did not originate from a rearrangement of the ancestral M haplotype. Instead, the MD haplotype likely originated from recombination between the M and P haplotypes. Strikingly, we identified two putative origins of the MD haplotype, one in the ancestor of five Nearctic species, and a second in the Palearctic species F. cinerea. The discovery of two MD haplotypes convergently evolving from distinct recombination events between two ancestral supergene haplotypes illustrates how supergenes can diversify and gain additional phenotypes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.27.672632v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Efficient detection and typing of phage-plasmids</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ilchenko, K., Bonnin, R., Rocha, E. P. C., Pfeifer, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Phage-plasmids are temperate phages that replicate as plasmids during lysogeny. Despite their high diversity, they carry genes similar to phages and plasmids. This leads to gene exchanges, and to the formation of hybrid or defective elements, which limits accurate detection of phage-plasmids. To address this challenge, we present tyPPing, an easy-to-use method that efficiently detects and types phage-plasmids with high accuracy. It searches for distinct frequencies and sets of conserved proteins to separate phage-plasmids from plasmids and phages. tyPPing's strength comes from both its precise predictions and its ability to systematically type phage-plasmids, including the assignment of confidence levels. We tested tyPPing on current databases and a collection of incomplete (draft) genomes. While predictions rely on the quality of assemblies, we detected high-quality phage-plasmids and experimentally proved them to be functional. Compared to other classification methods, tyPPing is designed to detect distinct types of phage-plasmids, and surpasses other tools in terms of sensitivity and scalability. Phage-plasmids are highly diverse, making the systematic identification of new types a difficult task. By combining tyPPing with other tools, however, we show a valuable foundation for addressing this challenge.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.29.673033v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">IDEAL GENOM: Integrated Downstream Analytical Pipeline for Genomic Analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gonzalez Ricardo, L. G., Tenghe, A. M. M., Ashok Kumar Sreelatha, A., Sharma, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;">Background: Since the release of the first human genome draft, there has been an exponential increase in the development of technologies that have offered unprecedented opportunities to unravel the genetic underpinnings of rare and complex diseases. However, despite the progress, the challenge persists. This includes the need for developing efficient computational tools to analyse large genomic datasets. The availability of clinical information, along with the development of cost-effective arrays that can scan hundreds of thousands of patients and healthy subjects to identify underlying risk variants, necessitates the development of tools that can streamline and automate the processing of millions of genetic markers. Results: To address this challenge, we present IDEAL-GENOM (Integrated Downstream Analytical Pipeline for Genomic Analysis), a Python-based framework designed to streamline the analytical workflow commonly implemented in genome-wide association studies (GWAS) settings. IDEAL-GENOM integrates widely used tools such as PLINK and GCTA along with custom-developed functionalities, enabling reproducible results through parameter sharing. Initial testing demonstrates that our framework reduces quality control processing time by 10%, requiring minimal user intervention. Additionally, IDEAL-GENOM supports both local and cloud-based environments, enhancing its accessibility and scalability. Conclusions: IDEAL-GENOM pipeline offers a simplified GWAS pipeline that will allow beginners and advanced users to leverage the in-built functionalities to perform GWAS analysis for complex diseases.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.27.672528v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Novel examples of NMD escape through alternative intronic polyadenylation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Vlasenok, M., Kuznetsova, A., Skvortsov, D. A., Pervouchine, D. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The nonsense-mediated mRNA decay (NMD) surveillance system detects and selectively degrades transcripts with premature stop codons (PTC). A stop codon is recognized as premature if it is followed by an exon-exon junction in >50 nucleotides downstream. Pruning of the 3'-untranslated region containing such junctions by alternative polyadenylation (APA) could provide a mechanism of NMD escape. Here, we systematically examine exons that carry a PTC (poison exons) for the presence of polyadenylation sites (PAS) in the downstream intron. Using the GTEx consortium data, we found that poison exons are followed by an active PAS more frequently than cassette exons, and identified tissue-specific switches between NMD-target and NMD-escape isoforms in a number of human genes. Among them are the tropomyosin 2 (TPM2) gene switching between NMD-target and NMD-escape isoforms in skeletal muscle and heart, and the vaccinia-related kinase 3 (VRK3) with a similar kidney-specific switch. Blocking the polyadenylation signal and the cleavage site in VRK3 by antisense oligonucleotides in the A549 cell line led to a switch from NMD-escape to NMD-target isoform that was accompanied by a drop in the expression level. This study suggests that NMD escape via alternative polyadenylation could be a widespread and currently overlooked post-transcriptional regulatory mechanism.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.27.672552v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Chromosome-scale genomes of two wild flowering cherrys (Cerasus itosakura and C. jamasakura) provide insights into structural evolution in Prunus</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fujiwara, K., Toyoda, A., Katsuki, T., Sato, Y., Biswa, B. B., Kishida, T., Tsuruta, M., Nakamura, Y., Mochizuki, T., Kimura, N., Kawamoto, S., Ohta, T., Nonomura, K.-I., Niki, H., Yano, H., Umehara, K., Suzuki, C., Koide, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Flowering cherries (genus Cerasus) are iconic trees in Japan, celebrated for their cultural and ecological significance. Despite their prominence, high-quality genomic resources for wild Cerasus species have been limited. Here, we report chromosome-level genome assemblies of two representative Japanese cherries: Cerasus itosakura, a progenitor of the widely cultivated C. xyedoensis 'Somei-yoshino', and Cerasus jamasakura, a traditional popular wild species endemic to Japan. Using deep PacBio long-read and Illumina short-read sequencing, combined with reference-guided scaffolding based on near-complete C. speciosa genome, we generated assemblies of 259.1 Mbp (C. itosakura) and 312.6 Mbp (C. jamasakura), with both >98% BUSCO completeness. Consistent with their natural histories, C. itosakura showed low heterozygosity, while C. jamasakura displayed high genomic diversity. Comparative genomic analyses revealed structural variations, including large chromosomal inversions. Notably, the availability of both the previously published C. speciosa genome and our new C. itosakura genome enabled the reconstruction of proxy haplotypes for both parental lineages of 'Somei-yoshino'. Comparison with the phased genome of 'Somei-yoshino' revealed genomic discrepancies, suggesting that the cultivar may have arisen from genetically distinct or admixed individuals, and may also reflect intraspecific diversity. Our results offer genomic foundations for evolutionary and breeding studies in Cerasus and Prunus.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.30.673203v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Diversity and composition of microbial communities in shrimp ponds sediments: A metagenomic approach</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Escobar, T. A., Palma, J. J., Flores, J. A., Lopez, K. M., Zambrano, S. A., Mestanza, C. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The cultivation of Litopenaeus vannamei is of great economic importance to Ecuador, yet it faces significant challenges due to microbial diseases. This study analyzes the microbial metagenome in white shrimp culture ponds sediments, assessing the diversity and communities of both pathogenic and non-pathogenic bacteria. Sediment samples were collected from ponds in three provinces with varying salinity levels: province of Guayas (6.28 g/L), El Oro (21 g/L), and Santa Elena (48 g/L). Through sequencing the V3-V4 region of the 16S rRNA gene, diverse microbial communities were identified, significantly influenced by salinity. Results revealed that microbial communities vary among locations, highlighting the presence of pathogenic bacteria such as Proteobacteria and beneficial ones like Firmicutes. Shannon and Simpson indices indicated high diversity and evenness in Guayas, El Oro, and Santa Elena (Shannon: 5.547, 5.810, and 4.326; Simpson: 0.9953, 0.9957, and 0.982, respectively). This analysis provides crucial information for improving management and sustainability of shrimp aquaculture, offering insights into the complex microbial ecosystems in culture environments and their potential impact on shrimp health and production.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.29.672896v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Benchmarking sketching methods on spatial transcriptomics data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gingerich, I. K., Goods, B. A., Frost, H. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> High-throughput spatial transcriptomics (ST) now profiles hundreds of thousands of cells or locations per section, creating computational bottlenecks for routine analysis. Sketching, or intelligent sub-sampling, addresses scale by selecting small, representative subsets. While used in scRNA-seq, most sketching methods optimize coverage in expression space alone and ignore physical location, risking spatial bias. We systematically benchmark uniform sampling, leverage-score sampling, Geosketch (minimax/Hausdorff), and scSampler (maximin) across multiple real ST datasets (mouse ovary, MERFISH brain, human breast cancer, lung) and simulations, using three input representations: PCA embeddings, spatial coordinates, and spatially smoothed embeddings. We show that expression-only designs capture global transcriptomic heterogeneity but distort tissue architecture by over-sampling high-variability regions and under-sampling homogeneous areas. Coordinate-only sampling restores tissue coverage but misses transcriptional extremes. A simple spatially aware extension, computing leverage scores from a randomized SVD basis smoothed by a spatial weights matrix, strikes a favorable balance, recovering rare cell states while maintaining uniform tissue coverage and avoiding edge effects. Across robust Hausdorff distances, clustering stability (ARI), PCA loading drift, and local cell-type MSE, spatially smoothed leverage scores match or outperform alternatives. These results motivate joint spatial-transcriptomic sketching objectives to enable fast, unbiased analyses of increasingly large ST datasets.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.26.672376v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Histolytics: A Panoptic Spatial Analysis Framework for Interpretable Histopathology</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lehtonen, O., Nordlund, N., Salloum, S., Kalliala, I., Virtanen, A., Hautaniemi, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Quantifying spatial organization in hematoxylin and eosin (H&E)-stained whole-slide images (WSIs) is essential for uncovering tissue-level patterns relevant to pathology. We present Histolytics, an open-source, scalable Python framework for interpretable, WSI-scale histopathological analysis. Histolytics integrates panoptic segmentation with spatial querying, morphological profiling, and graph-based analytics to enable high-resolution, quantitative characterization of nuclei, tissue compartments, and the extracellular matrix (ECM). Designed to align with diagnostic reasoning, Histolytics supports segmentation with state-of-the-art deep learning models and provides modular tools for extracting biologically grounded features across entire WSIs. By leveraging spatially contextualized measurements at cellular and tissue levels, Histolytics addresses a critical gap in explainable computational pathology, offering an interpretable alternative or complement to black-box predictive models. The framework is compatible with the broader Python data science ecosystem and includes extensive documentation and pretrained models to promote widespread adoption.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.26.672346v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Fast Multi-objective RNA Optimization with Autoregressive Reinforcement Learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Huang, J., Bai, H., Fang, Y., Liu, X., Wang, S., Yuan, Y., Yan, J., Shen, H.-B., Hu, R., Pan, X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Codon optimization is essential in mRNA vaccine development, while existing tools face limitations in the computational efficiency, sequence diversity and universality. To address these challenges, we develop RNAJog (RNA Joint Optimization with autoregressive Generative model), a framework integrating autoregressive generation with reinforcement learning to optimize codon sequences for minimum free energy (MFE) and codon adaptation index (CAI), even enabling sequence design without requiring annotated training data. Evaluations in both in silico and wet-lab experiments have confirmed RNAJog's effectiveness and efficiency, with two orders of magnitude faster than LinearDesign for long RNA sequence and about a 10-fold increase in antibody titre compared to the wild-type mRNA for Influenza virus hemagglutinin (HA) mRNA vaccine design in mouse.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.26.672486v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">gnSPADE: a reference-free deconvolution method incorporating gene network structures in spatial transcriptomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xie, A., Cui, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Spatial transcriptomics (ST) technologies have offered unprecedented insights into the spatial organization of gene expression, allowing for the study of tissue architecture, domain boundaries, and cell-cell interactions. However, most ST data generated so far are at multicellular resolution, where each spot captures transcripts from a mixture of diverse cells of different cell types. While reference-based deconvolution approaches offer robust solutions, they rely heavily on the availability and quality of external single-cell reference data, which may be incomplete, unavailable, or poorly matched to the spatial data. Moreover, even when such references are available, they often represent only broad cell types, potentially obscuring finer subpopulation structures and masking intra-type heterogeneity. To overcome these limitations, we introduce gnSPADE, a reference-free spatial deconvolution method that incorporates gene network structures via a Markov random field within a latent Dirichlet allocation (LDA) modeling framework. gnSPADE jointly infers cell type-specific transcriptional profiles and spatial compositions without external references. Applied to synthetic and real ST datasets, gnSPADE achieves improved accuracy, spatial resolution, and biological interpretability compared to other methods, highlighting the power of reference-free deconvolution in resolving complex tissues.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.26.672457v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Quantitative Analysis of Meloidogyne incognita Population Density Using Real-Time PCR and Its Correlation with Root-Knot Disease Index in Tabacco (Nicotiana tabacum)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cheng, Z., Chai, J., Pu, H., Zhao, Z., Duan, X., Zheng, W., Xu, J., Miao, P., Hou, W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Tobacco root-knot disease represents a significant threat to tobacco production, particularly in the western Henan region, where Meloidogyne incognita is the predominant species. This study collected samples of M. incognita and soil from Luoyang, Henan, and designed specific primers MiF and MiR based on the amplified 735 bp sequence of the ITS1-5.8S-ITS2 region. These primers exhibit mismatches with related species, including M. javanica, M. minor, M. hapla, and M. arenaria, showing 1, 6, 10, and 10 base differences in the forward direction, respectively, 3, 10, and 9 base mismatches in the reverse direction. Although the primers were used to detect corn-wheat soil samples, no amplification was observed. Additionally a real-time quantitative PCR curve for M. incognita in soil was constructed, revealing a negative correlation between the Ct value (Y) and the log-transformed number of nematodes (x) per 20 g of dry soil, represented by the equation y = -0.9757x + 35.565; (R2 = 0.9999, P 2000 individuals/20 g soil, r = 0.69, P<0.05). This study showed that as the RKI increases, the rate of increase in nematode density in the soil diminishes. These findings provide valuable insights for the development of effective scientific strategies for nematode control.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.27.672540v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Keetia tinka sp. nov. & K. kounounkan sp.nov. (Rubiaceae - Vanguerieae) new threatened forest climbers and shrubs of sandstone plateau of the Republic of Guinea.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Simbiano, F. J., Couch, C., Magassouba, S., Plummer, J., Cheek, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Two new species of Keetia are described from recent botanical collections for conservation management made in surviving submontane forest areas of the sandstone plateau areas of the Republic of Guinea. Keetia kounounkan Cheek & Simbiano is a shrub of the Kounounkan Plateau towards the border of Sierra Leone, so far with a single location in gallery forest. K. tinka Cheek & Simbiano is an evergreen rainforest climber of the main part of the Fouta Djalon Highlands with two locations both in degraded forest. Both species are described, illustrated and provisionally assessed for their conservation status, the first as Critically Endangered, the second as Endangered.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.27.672669v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Phage-mediated just-in-time circuit amplification and delivery for recombinant expression of toxic proteins</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Baecker, L. E., Baert, F., Broux, K., Oome, D., Simoens, K., Waldherr, S., Bernaerts, K., Aertsen, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In order to address outstanding challenges in bacterial bioreactor bioprocessing, we have developed an approach that alleviates the need for chemical inducers and selective agents (such as antibiotics), and mitigates leaky expression of the production circuit. More specifically, temperate bacteriophage Lambda ({lambda}) was engineered as a chassis to just-in-time amplify and deliver a production circuit to a wild-type Escherichia coli population. Since the expression of the production pathway is engineered to be dependent on {lambda}'s central regulatory switch towards the lysogenic state, {lambda} will first lytically amplify in the background without significantly altering overall bioreactor growth dynamics or gene expression. Only when {lambda} eventually starts outnumbering the host population, its natural switch to lysogeny will actively install and trigger the production circuit. As such, the {lambda} chassis serves as an intrinsic self-amplifying inducer that meanwhile rules out any leaky expression. The impact of the initial phage-to-host ratio on population dynamics and production efficiency was assessed experimentally and through ODE modeling, revealing that it can be used to fine-tune the trade-off between gene dosage and biomass conversion into virions. We demonstrate that this leakproof approach is particularly promising for the expression of toxic proteins such as Benzonase, a highly potent DNase and RNase.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.31.673379v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Synthetic yeast genome SCRaMbLEing uncovers a new role for ribosomal proteins in genetic code expansion</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sanders, J. E. J., Hoffmann, S. A., Moody, E. R., McCullough, M., Green, A. P., Cai, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The translation of proteins with non-canonical amino acids (ncAAs) has emerged as a powerful technology for embedding new functional elements into proteins, enabling the development of novel enzymes, materials, and biopharmaceuticals. However, the utility of this approach has been hindered by weak translation efficiencies. To address this challenge, we sought to substantially improve orthogonal translation in Saccharomyces cerevisiae. We first evaluated recently described {triangleup}NPylRS-class pyrrolysyl-tRNA synthetase systems and identified a homolog with[~]5.4-fold higher activity than the best previously reported pyrrolysyl system. Building on this advance, we leveraged the SCRaMbLE system in the semi-synthetic yeast strain Syn6.5 to generate structural genomic variation, and identified strains with enhanced ncAA incorporation. Pooling genomic alterations across enhanced strains, we identified an association for the deletion of several ribosomal protein genes with the increased production of ncAA-containing protein. These findings demonstrate a previously unrecognized role for ribosomal proteins in enabling alternate genetic codes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.31.673356v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identification of Distinct Topological Structures From High-Dimensional Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xu, B., Braun, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell RNA sequencing allows the direct measurement of the expression of tens of thousands of genes, providing an unprecedented view of the transcriptomic state of a cell. Within each cell, different biological processes such as differentiation or cell cycle take place simultaneously, each providing a different characterization of cell state. To identify gene sets that govern these processes for the purpose of disentangling convolved biological processes, we present ``Identification of Distinct topological structures'' (ID). ID works by constructing an alternative low-dimensional parametrization of the high-dimensional system, applying a finite perturbation to this alternative parametrization, and looking for genes that respond similarly. With this approach, we demonstrate that ID is capable of identifying structures within the data that will otherwise be missed. We further demonstrate the utility of ID in scRNA-seq datasets collected under various backgrounds, delineating cellular differentiation, characterizing cellular response to external perturbation, and dissecting the effect of genetic knock-outs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.26.672446v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Niche Net Based Dissection of CD14+ Monocyte Crosstalk with Memory CD4+ T Cells in Human PBMCs</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> kota, t.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Intercellular communication between monocytes and T cells is essential for immune regulation within peripheral blood mononuclear cells (PBMCs). To examine how CD14+monocytes regulate memory CD4+ T cells at the gene expression level, we applied NicheNet to PBMC single-cell RNA-seq data. We performed quality control, normalization, principal component analysis, and UMAP-based visualization. CD14+ monocytes and memory CD4+ T cells were identified by canonical markers and designated as sender and receiver populations, respectively. NicheNet analysis revealed both classical cytokine signaling and checkpoint-like interactions. Prominent ligands included TNF, IFNG, IL10, and CXCL9, predicted to regulate inflammatory target genes through receptors such asTNFRSF1A/B and CXCR3. In addition, the SIRPG - CD47 axis emerged as a potential costimulatory pathway modulating T cell responses. These results suggest that monocytes regulate memory CD4+ T cells via a combination of inflammatory, costimulatory, and regulatory signals, providing mechanistic hypotheses for immune modulation in health and disease.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.26.672385v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Loss of the Y chromosome drives cancer metabolic reprogramming</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Subramaniyan, B., Acharya, B. R., Anand, P., Mani, S. K. K., Mohapatra, S., Chen, X., Qu, F., Fstkchyan, Y., Gutierrez, C., Abdel-Hafiz, H., Costello, J., Theodorescu, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Metabolic reprogramming promotes cancer aggressiveness and an immune-suppressive tumor microenvironment. Loss of the Y chromosome (LOY) drives both phenotypes in bladder cancer (BC). We investigated the hypothesis that LOY leads to metabolic reprogramming using untargeted metabolomic profiling of human BC cells and analysis of pan-cancer transcriptomic datasets. This revealed that aerobic glycolysis is activated in LOY BC cells. Since prior work showed that expression of collagen receptor DDR2 drives BC progression and DDR2 is a regulator of tumor metabolism, we investigated if DDR2 is implicated in metabolic reprogramming of LOY-tumors. Analysis of scRNAseq data from 251 patients with 12 tumor types found that LOY and DDR2 expression promote aerobic glycolysis, and this was confirmed by metabolomics. Deletion of DDR2 in LOY BC cells reduced glycolytic flux, inhibited cell proliferation, reduced EMT and stemness features, and promoted apoptosis. Our data provide a rationale for using LOY as a tumor selection biomarker for DDR2 targeted therapeutics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.26.672471v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A proteomic study of the dual oncogenic and tumor- suppressive roles of SIRT3 in lung and breast cancer cell lines</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ayala Reyes, M., Fernandez Coto, D. L., Alonso Bastida, R., Marko-Varga, G., Gil, J., Encarnacion-Guevara, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mitochondria play a crucial role in metabolism and energy production by generating adenosine triphosphate (ATP) through oxidative phosphorylation. They also help maintain intracellular calcium levels, facilitate communication between the nucleus and cytoplasm, detoxify reactive oxygen species (ROS), and regulate apoptosis (programmed cell death). Reversible acetylation of mitochondrial proteins is a key post-translational modification influencing these processes, with the NAD-dependent deacetylase SIRT3 being a major regulator. While SIRT3 has been described as a tumor suppressor in some contexts and as a tumor promoter in others, its role appears to be tissue- and metabolism-specific. Here, we compared the proteomic and acetylomic responses of lung adenocarcinoma (A549) and breast adenocarcinoma (MCF7) cell lines to SIRT3 inhibition by 3-(1H-1,2,3-triazol-4-yl)pyridine (3-TYP). The two lines were selected based on distinct metabolic phenotypes and reported differences in basal SIRT3 abundance. Total proteome and mitochondrial-enriched fractions were analyzed separately for each cell line to avoid cross-line normalization bias. We identified 6,457 proteins and 4,199 acetylated peptides, revealing divergent pathway enrichments and acetylation patterns following SIRT3 inhibition. A549 cells showed enrichment of oxidative metabolism pathways, whereas MCF7 cells exhibited features consistent with metabolic reprogramming. These findings suggest that the cellular context determines the proteomic consequences of SIRT3 modulation and highlight the importance of metabolism in shaping SIRT3-associated phenotypes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.26.672455v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Human iPSC-derived prostate organoids with germline BRCA2 mutation undergo tumorigenic transformations</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Acharya, B. R., Lawless, G., Avalos, P., Anand, P., Fstkchyan, Y., Bell, S., Otero, M. G., Guillemette, S., Myers, Z., Workman, M., Catalona, W. J., Theodorescu, D., Svendsen, C. N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The lack of physiologically relevant in vitro prostate models has impeded studies of organ development and prostate tumorigenesis. We reprogrammed peripheral blood mononuclear cells (PBMCs) from individuals with and without pathogenic-germline BRCA2 mutation (MUT_BRCA2, CON_BRCA2) into induced pluripotent stem cells (iPSCs), which showed no differences in morphology, proliferation, or pluripotency markers. Differentiation of MUT_BRCA2 iPSCs into prostate organoids (iPROS) using defined growth factors and signaling molecules resulted in disrupted morphology, impaired polarity, increased proliferation, and elevated prostate-specific antigen (PSA) secretion compared to CON_BRCA2 iPROS. Transcriptomic profiling revealed early prostate cancer (PCa) signatures. Upon exposure to dietary carcinogens, MUT_BRCA2 iPROS showed further PSA elevation, enhanced proliferation, AMACR upregulation, p63 reducetion are markers of aggressive PCa. In vivo, MUT_BRCA2 iPROS formed tumors in immunodeficient mice. This patient-derived iPROS-platform recapitulates human-prostate mopphology and function, models early tumorigenesis events, and provides a valuable tool for studying PCa biology and enabling personalized drug discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.26.672478v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Loss of Sox10 in a HER2+ model prevents tumor initiation and induces luminal-to-basal reprograming in cancer cells.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Garland, B., Delisle, S., Abou-Hamad, J., De Souza, C., Zuccarini, R., Auer, R. C., Sabourin, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The SRY-HMG-Box transcription factor SOX10 plays a critical role in neural crest development, but its function in epithelial tumorigenesis remains unclear. Here, we identify SOX10 as a key regulator of tumor-initiating activity in Neu-driven mammary cancers. Genetic ablation of Sox10 in the luminal compartment of MMTV-Neu (NIC) mice resulted in delayed but normal mammary gland development. Sox10 deletion conferred a dose-dependent delay in tumor onset, with a complete loss of tumor initiation in Sox10-deficient luminal cells. CRISPR/Cas9-mediated Sox10 inactivation in Neu-transformed tumor cells led to reduced 3D invasion and diminished self-renewal in mammosphere assays. Established Sox10-null cell lines exhibited markedly impaired growth in orthotopic transplant models and failed to colonize lung tissue following tail vein injection, suggesting a loss of tumor-initiating capacity. Transcriptomic profiling revealed that Sox10-deficiency in Neu+ tumor cells induces a luminal-to-basal/stem-like shift and the downregulation of several genes associated with genetic networks regulating stemness. Collectively, these findings demonstrate that Sox10 is required for a permissive cell progenitor state for Neu-driven tumor initiation and that it is critical to sustain the invasive and self-renewing traits that drive tumor progression and metastasis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.26.672423v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Lipid metabolites as biomarkers and therapeutic targets in oral squamous cell carcinoma.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hexin Ma, Chang Liu, Xibo Li, Lihua Zuo, Chunshen Li, Xiaohui Xu, Shilong Zhang, Xiang Ma, Erli Yue, Bin Qiao, Yifei Wang, Wantao Chen, Zhi Sun, Hongyu Zhao</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> This study explores the association of lipid metabolism disruption and Oral Squamous Cell Carcinoma (OSCC). We aim to identify specific lipid biomarkers and therapeutic targets for OSCC. We included 78 OSCC patients and 80 healthy controls, and applied non-target lipidomics and transcriptomics for comprehensive analysis. Using ultra-high-performance liquid chromatography quadrupole-Orbitrap high-resolution accurate mass spectrometry (UHPLC/Q-Orbitrap HRMS) coupled with machine learning for diagnostic modeling, we identified potential lipid biomarkers. Transcriptomic analysis helped in pinpointing genetic and metabolic targets relevant to lipid metabolism in OSCC. Notably, we observed 70 differential lipid metabolites in the OSCC group, with nine achieving an AUC > 0.95, suggesting high potential as biomarkers. A diagnostic model based on 10 differentiated lipids yielded accuracy rates of 98.2% in a training cohort and 95.7% in a validation cohort. Additionally, the overexpression of DGKG, linked to poor prognosis, was noted to enhance migration and invasion of OSCC cells, marking it a potential target for therapy. This research underscores the critical role of lipid metabolic alterations in OSCC and highlights innovative diagnostic and therapeutic avenues.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40887599/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrated multi-omics analysis identifies SELENOP and PKMYT1 as immune-metabolic hub genes in breast cancer.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Guohui Tang, Zheng Zhang, Bo Pang, Ruonan Li, Yuting Liu, Haotian Cai, Wenrui Wang, Changjie Chen, Yurong Ou, Qingling Yang</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Metabolic reprogramming and immune evasion synergistically drive breast carcinogenesis, but their combined impact remains unclear.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40821907/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Interpretable bioinformatics approaches for pheochromocytoma bioactivity and protein interaction analysis.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> ƒ∞lhan Uysal</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Pheochromocytoma (PCC) is a rare neuroendocrine tumor driven by complex molecular mechanisms, notably involving the oncogenic c-Myc/Max and c-Myc/c-Max protein complexes. Despite their pivotal role in tumor progression, the molecular interactions and bioactive compounds specifically targeting these complexes remain inadequately characterized. This study presents an integrative computational pipeline combining interpretable bioinformatics, network biology, and machine learning to elucidate key molecular mechanisms and bioactive motifs associated with PCC. A curated dataset of 5000 bioactive molecules was obtained from ChEMBL, and structural motifs associated with bioactivity were identified using a genetic programming-based approach. Random Forest, Support Vector Machines, and Gradient Boosting classifiers were trained and cross-validated using 10-fold cross-validation to predict pIC50 values, achieving high performance (mean accuracy: 0.98, AUC >0.97). Feature importance analysis consistently identified pIC50, molecular weight (MW), lipophilicity (LogP), and hydrogen-bonding properties as primary determinants of bioactivity. PPI networks were built using STRING's experimentally validated interactions and refined using BioGRID and literature cross-validation. Network centrality analysis and community detection using the Girvan-Newman algorithm revealed MYC, MAX, and EP300 as central hubs, with associated protein modules significantly enriched for biological processes including transcriptional regulation, cell cycle control, ubiquitination, and apoptosis. To enhance model interpretability, explainable artificial intelligence (XAI) methods, including SHAP and DALEX, were employed to elucidate the contribution of individual molecular descriptors, mechanistically elucidating compound-target interactions. Despite its robustness, this computational framework lacks experimental validation and independent external datasets. Additionally, STRING's uniform confidence scores limited edge-weight precision in network visualizations during network analyses. Nevertheless, this study demonstrates the potential of a multi-layered computational approach to deepen the understanding of MYC-driven oncogenesis in PCC. By integrating motif discovery, network biology, and interpretable machine learning, the work identifies actionable molecular signatures and critical protein targets, providing a foundation for future experimental validation and the development of targeted therapies in pheochromocytoma as well as other rare cancers.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40753949/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A multivariate cell-based liquid biopsy for lung nodule risk stratification: Analytical validation and early clinical evaluation.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jason D Berndt, Fergal J Duffy, Mark D D'Ascenzo, Leslie R Miller, Yijun Qi, G Adam Whitney, Samuel A Danziger, Anil Vachani, Pierre P Massion, Stephen A Deppen, Robert J Lipshutz, John D Aitchison, Jennifer J Smith</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The Indicator Cell Assay Platform (iCAP) is a novel tool for blood-based diagnostics that uses living cells as biosensors to integrate and amplify weak, multivalent disease signals present in patient serum. In the platform, standardized cells are exposed to small volumes of patient serum, and the resulting transcriptomic response is analyzed using machine learning tools to develop disease classifiers.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40831733/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Utilizing protein structure graph embeddings to predict the pathogenicity of missense variants.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Martin Danner, Matthias Begemann, Miriam Elbracht, Ingo Kurth, Jeremias Krause</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genetic variants can impact the structure of the corresponding protein, which can have detrimental effects on protein function. While the effect of protein-truncating variants is often easier to evaluate, most genetic variants that affect the protein-coding region of the human genome are missense variants. These variants are mostly single nucleotide variants, which result in the exchange of a single amino acid. The effect on protein function of these variants can be challenging to deduce. To aid the interpretation of missense variants, a variety of bioinformatic algorithms have been developed, yet current algorithms rarely directly use the protein structure as a feature to consider. We developed a machine learning workflow that utilizes the protein-language-model ESMFold to predict the protein structure of missense variants, which is subsequently embedded using graph autoencoders. The generated embeddings are used in a classifier model, which predicts pathogenicity. We provide evidence that graph embeddings can be used for pathogenicity prediction and that they can be used to enhance the widely applied CADD score. Additionally, we explored different levels of abstraction of the graph embeddings and their influence on the classifier. Finally, we compare the utility of graph embeddings from different protein-folding models.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40708850/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">HTGAnalyzer: An accessible R package with a web interface for enhanced transcriptomic analysis in precision medicine.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Laia D√≠ez-Ahijado, Aar√≥n Marc√©n Del Rinc√≥n, Lorena Marim√≥n, Adela Saco, Marta Del Pino, Aureli Torn√©, Katarzyna Darecka, Lia Sisuashvili, N√∫ria Pe√±uelas, Pau Pascual-Mas, N√∫ria Carreras-Dieguez, Oriol Ordi, Natalia Rakislova, Robert Albero</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Transcriptomics generates promising data for personalized medicine, but its clinical utility is hindered by technical barriers associated with formalin-fixed, paraffin-embedded samples. Advances in automation and simplified workflows are key for integrating these technologies into routine care. We have developed HTGAnalyzer, an R package that enables bulk transcriptomics analysis from multiple data sources, such as HTG EdgeSeq and RNA-seq. Designed with clinical and molecular diagnostics in mind, HTGAnalyzer includes a fully automatic pipeline that performs sample quality control, data normalization, and a set of transcriptomic and clinical analyses. Importantly, HTGAnalyzer provides quality control and analysis tailored for HTG data, filling the gap left by HTG closure with no alternative solutions. Designed for ease of use without knowledge of bioinformatics, it features a user-friendly Shiny app, standardized functions for easy Docker setup, and enhanced connectivity to clinical data management systems. It delivers a complete analytical workflow in less than 20 min on a standard PC (Intel i7-7500U, 16 GB RAM). We applied HTGAnalyzer to diverse datasets, including a new vulvar cancer cohort, and generated interpretable reports with rich visual outputs. In summary, HTGAnalyzer offers robust quality control and advanced analytics in a single, unified tool, supporting the clinical use of transcriptomics in precision medicine.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40706503/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AI-powered liquid biopsy for early detection of gastrointestinal cancers.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Md Sadique Hussain, Mokhtar Rejili, Amna Khan, Saud O Alshammari, Ching Siang Tan, Faouzi Haouala, Sumel Ashique, Qamar A Alshammari</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Gastrointestinal cancers (GICs) are a leading cause of cancer-related mortality worldwide, largely due to late-stage diagnosis. Liquid biopsy has emerged as a promising non-invasive diagnostic tool, utilizing circulating tumor DNA (ctDNA), circulating tumor cells, exosomal RNA (exoRNA), and tumor-educated platelets for early cancer detection. Challenges, including data complexity, low biomarker abundance, and detection variability, require advanced computational solutions. Artificial intelligence (AI), particularly machine learning (ML) and deep learning (DL), has significantly improved the accuracy and clinical utility of liquid biopsy by enabling high-throughput biomarker discovery, multi-omics integration, and predictive modelling. AI-driven algorithms have enhanced ctDNA mutation profiling, methylation analysis, and fragmentomics, offering superior sensitivity and specificity for early GIC detection. Additionally, AI-based analysis of exoRNA and platelet-derived biomarkers provides novel insights into tumor progression and patient stratification. Despite these advances, key challenges remain, including data standardization, bias mitigation, and regulatory validation. The implementation of federated learning and ethical AI frameworks can further refine AI-powered liquid biopsy models, paving the way for precision oncology applications. This review highlights advances in AI-powered liquid biopsy for early GIC detection, emphasizing its potential and the need for validation, collaboration, and regulatory alignment for clinical adoption.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40669686/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Moving past multidisciplinary discussions and Gender-Age-Physiology model: precision medicine through biological phenotyping in interstitial lung disease.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Manoj V Maddali, Justin M Oldham, Catharina C Moor</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Interstitial lung disease (ILD) presents significant diagnostic and therapeutic challenges due to underlying biological heterogeneity and variable clinical course. Traditional diagnostic and prognostic tools are limited in their ability to capture this heterogeneity or guide personalized treatment. Advances in biological phenotyping have set the stage for precision medicine in ILD, improving diagnosis, prognosis, and therapeutic decision-making in ILD. This review highlights recent advances in biological phenotyping technologies and their potential to reshape ILD care.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40643570/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A comprehensive targeted panel of 295 genes: Unveiling key disease initiating and transformative biomarkers in multiple myeloma.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Vivek Ruhela, Ritu Gupta, Rupin Oberoi, Anubha Gupta</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Multiple myeloma (MM) is a hematological malignancy that progresses from a benign precursor stage known as Monoclonal Gammopathy of Undetermined Significance (MGUS). Distinguishing MM from MGUS at the molecular level by identifying key biomarkers, genomic alterations, and gene interactions is critical for early detection and deeper insight into MM pathogenesis.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40617081/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">EnsemPred-ACP: Combining machine and deep learning to improve anticancer peptide prediction.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Minjun Kwon, Yong Eun Jang, Ji Su Hwang, Seok Gi Kim, Nimisha Pradeep George, Shaherin Basith, Gwang Lee</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Anticancer peptide (ACP) has emerged as potent therapeutic agents owing to its ability to selectively target cancer cells while minimising toxicity to healthy cells. However, the accurate computational prediction of ACP remains challenging because of the complex molecular mechanisms underlying cancer. In this study, we introduce EnsemPred-ACP, an innovative ensemble framework that combines machine learning (ML) and deep learning (DL) approaches to enhance ACP prediction. Our primary innovation is the introduction of binary profile features (BPF) to augment pre-trained protein embeddings, thereby capturing position-specific patterns crucial for ACP identification. The framework used a dual-pipeline architecture; ML models processed handcrafted sequence features and embeddings, whereas DL models handled BPF-enhanced embeddings. Upon evaluation with independent datasets, EnsemPred-ACP achieved an accuracy of 0.863, sensitivity of 0.897, and specificity of 0.830, notably outperforming existing methods. The model demonstrated a strong generalisation performance, achieving an area under the receiver operating characteristic curve of 0.93. Ablation studies on independent datasets further highlighted the substantial impact of BPF, enhancing the prediction accuracy by 2.5 % and 11.1 % when integrated with ESM2 and ProtT5 embeddings, respectively. These results demonstrate the effectiveness of our integrated approach in accurately identifying potential therapeutic peptides, thereby contributing to the advancement of peptide-based cancer therapeutics.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40614519/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Innovative Strategy for Identifying Subtypes Through the Analysis of Multi-Omics Data with Adversarial Autoencoders.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xia Chen, Hao Nie, Quanwei Chen, Xiang Zhang, Zixing He, Xiuxiu Chao, Weihao Ou, Xiangzheng Fu, Haowen Chen</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cancer is a disease that is both complex and diverse, and effective diagnosis and treatment require an accurate depiction of tumor subtypes. Traditional methods of cancer identification, which rely on clinical and histopathological criteria, have limitations in identifying key molecular subtypes. With the advancement of high-throughput genomics technologies, the field of cancer research has undergone a transformation, enabling detailed analysis of tumor molecular characteristics on a large scale. The integration of multiple types of genomic data is expected to provide a more comprehensive understanding of the molecular mechanisms of cancer and to promote the discovery of new diagnostic and therapeutic targets. However, achieving this requires the development of new computational techniques. In order to facilitate more efficient feature extraction and dimensionality reduction of multi-omics data, we present MultiDAAE (Multi-omics Double Adversarial Autoencoder), a novel technique that combines autoencoders with two discriminators to form two generative adversarial networks. On several cancer datasets, our method shows outstanding clustering performance when compared to state-of-the-art techniques. To sum up, MultiDAAE can help identify possible molecular pathways and provide information for the development of tailored cancer treatments.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40566761/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Differential expression and hypermethylation of apoptotic genes in circulating nucleic acids are associated with clinical phenotypes of prostate cancer and benign prostatic hyperplasia in Nigerians.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Olayiwola Akanji Popoola, Titilola Aderonke Samuel, Comfort Folorunso, Idowu Taiwo, Olubunmi Magbagbeola, Oluyemi Akinloye</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> During carcinogenesis, cells have an increased ability to survive in the face of death-inducing mechanisms (apoptosis). Subversion of apoptosis is regarded as one of the primary factors helping the evolution of tumor cells. This study described the gene expression and DNA methylation of selected genes associated with apoptosis in prostate cancer (PCa) and benign prostatic hyperplasia (BPH).</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40579334/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identification of hub genes involved in the pathogenesis of diabetic nephropathy: A multi-omics study integrating machine learning, mendelian randomization and mediation analysis.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kang Li, Huidi Tang, Yanqing Wang, Xiaojie Wang</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Diabetic nephropathy (DN), affecting 30%-40% of diabetic patients, is the leading cause of end-stage renal disease worldwide. This study aims to identify diagnostic biomarkers and explore potential gene-metabolite interactions in DN pathogenesis through integrated bioinformatics approaches and experimental validation.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40555691/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Radiogenomic insights suggest that multiscale tumor heterogeneity is associated with interpretable radiomic features and outcomes in cancer patients.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Peng Lin, Jin-Mei Zheng, Chang-Wen Liu, Quan-Quan Tang, Jin-Shu Pang, Qiong Qin, Zhen-Hu Lin, Hong Yang</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> To develop radiogenomic subtypes and determine the relationships between radiomic phenotypes and multiomics molecular characteristics.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40544543/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multi-omics reveals the polyethylene terephthalate carcinogenicity: Cancer progression and immune microenvironment.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kangping Yang, Xianhuan Zhou, Kai Wu, Jiaqiang Wu, Chunxia Huang, Liang Yang</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Polyethylene terephthalate (PET), a polymer widely used in consumer products, has recently been implicated in cancer progression, though its mechanistic underpinnings remain elusive. This multidisciplinary study systematically investigates PET's carcinogenic mechanisms by integrating network toxicology, machine learning, molecular docking simulations, single-cell RNA sequencing, and spatial transcriptomics. We identified PET-associated oncogenic targets and conducted pan-cancer expression profiling with prognostic analyses of 22 core genes, subsequently employing machine learning algorithms to quantify their prognostic weights across malignancies. Molecular docking revealed six high-affinity PET-binding targets (PIK3CA, TOP2A, PARP1, SRC, EP300, CREBBP; binding energies <-6.0 kcal/mol) predominantly expressed in T-cells and phagocytic cells, with significant infiltration into malignant tumor regions. Mechanistically, PET exposure may drive carcinogenesis through three synergistic pathways: dysregulation of the PI3K-Akt-mTOR signaling axis via PIK3CA, compromised DNA repair mechanisms through TOP2A/PARP1 interactions, and epigenetic reprogramming mediated by the EP300/CREBBP transcriptional complex. These molecular perturbations collectively induce genomic instability, immune dysregulation, and tumor evolution by disrupting cellular homeostasis, DNA integrity maintenance, and chromatin remodeling processes. Finally, we confirmed that PET exposure promotes tumor cell proliferation by experiments such as clone formation and western blotting, that the effect increases with exposure concentration, and that PET exposure increases TOP2A protein levels. Our findings not only delineate PET's role in tumorigenesis but also establish a computational-experimental framework for environmental toxicant evaluation. This work provides critical insights for developing prevention strategies against pollutant-driven malignancies, bridging molecular toxicology with clinical cancer biology.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40532600/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

