<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Sunday, 15 June 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PHLAME: A benchmark for continuous evaluation of host phenotype prediction from shotgun metagenomic data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Barak, N., Bhattacharya, H., Asnicar, F., Sung, J., Segata, N., Yassour, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce PHLAME, a standardized benchmark for evaluating host phenotype prediction methods using gut metagenomic data, and show that classic machine learning methods perform competitively, offering both ease of use and state-of-the-art results.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Predicting host phenotypes from shotgun metagenomic data is essential for translating microbiome research into clinical practice. Despite the development of numerous computational tools for this task, researchers often default to traditional machine learning methods such as Random Forest. This hesitance to adopt newer methods stems from the complexity of newer tools and the lack of standardized evaluations, as most tools are assessed on different datasets and compared against a limited set of methods. To address this, we introduce PHLAME, a standardized benchmark for evaluating host phenotype prediction methods using gut metagenomic data. PHLAME features a diverse range of prediction tasks and enables consistent, comparative assessments across prediction tools. Our systematic evaluation of existing tools shows that microbiome-based phenotype prediction remains a challenging problem. However, we find that classic machine learning methods perform competitively, offering both ease of use and state-of-the-art results. PHLAME is publicly available for ongoing benchmarking at https://phlame.yassourlab.com/.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.12.658885v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Simulating Iron Deficiency in Plant Plastids With a Flexible Physics-Informed Neural Network Approach</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> El Alaoui, S., Henry, C. S., Paape, T., Xie, M., Seaver, S. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a new model for flux balance analysis of plant tissues that incorporates transient metabolic states, at the cost of violating the steady-state assumption, in order to enable more accurate flux estimation in plant tissues.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Flux balance analysis has proven to be a successful approach in metabolic engineering and systems biology, for predicting intracellular fluxes of large genome-scale networks and the essentiality of genes encoding enzymes and regulatory factors. Flux balance analysis relies on a key assumption of a metabolic state being persistent ("steady") over a given timeframe. This assumption works well for microbial growth because of the ease with which microbial media can be fixed, biomass can be decomposed, and growth rates can be measured. However, the assumption is far less tenable for the cells and tissues of complex multicellular organisms, particularly if any integrated data is sampled from a heterogeneous collection of developing cells continually interacting between and across tissues. These will likely exhibit transient metabolic states equilibrating over varying timescales, and many FBA studies in complex organisms typically either ignore time as a parameter, or integrate data taken over long timescales (days/weeks). In this work, we adopt and modify a previously published machine learning approach that hybridized several aspects of a constraint-based approach with machine-learning in order to predict growth. This study introduces a ML-FBA framework for plant tissues that accommodates transient state dynamics, at the cost of violating the steady-state assumption, in order to enable more accurate flux estimation in plant tissues. For our case study we reconstruct the metabolism of the plastid of Poplar and Sorghum, integrating data sampled from leaf tissue under varying levels of iron bioavailability. We show that the approach gives us more realistic insights into plastidial metabolism, indicates where our metabolic reconstruction could be improved, and still allows us to draw novel hypotheses on the impact of metal bioavailability on plant leaves.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.10.658179v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PhyloControl: a phylogeny visualisation platform for risk analysis in weed biological control</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chen, S. H., Stevens, L., Gooden, B., Rafter, M. A., Knerr, N., Thrall, P. H., Ord, L., Schmidt-Lebuhn, A. N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a user-friendly visualisation tool called PhyloControl to aid researchers in their decision-making during biocontrol risk analysis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> 1. Phylogenetic distance is a key measure used to develop species lists for host specificity tests that delimit the fundamental and realised host range of candidate weed biocontrol agents to meet the assumptions of the centrifugal phylogenetic method. Plant pathogens and insects, even those with broad host ranges, exhibit some degree of phylogenetic conservatism in their host plant associations. Thorough host-specificity testing is crucial to minimise the risk of off-target damage by biocontrol agents to native and economically important plant species. To facilitate this, host test lists need to be developed from an understanding of evolutionary relationships, usually visualised as a phylogenetic tree generated from genetic data, together with plant functional traits and geospatial information. Currently, the process of obtaining a host test list is not standardised, and the manual steps are time-consuming and challenging. 2. We introduce a user-friendly visualisation tool called PhyloControl to aid researchers in their decision-making during biocontrol risk analysis. PhyloControl integrates taxonomic data, molecular data, spatial data, and plant traits in an intuitive interactive interface, empowering biocontrol practitioners to summarise, visualise and analyse data efficiently. 3. Comprehensively sampled phylogenetic trees are often unavailable, and older published phylogenies often lack branch resolution and support, which increases uncertainty. PhyloControl includes a workflow implemented through Quarto notebooks in R that allows users to download publicly available DNA sequences and perform phylogenetic analyses. The modular workflow also incorporates species distribution modelling to predict the current and potential extent of target weed species. 4. PhyloControl will streamline the development of biocontrol host tests lists to support risk analysis and decision making in classical weed biological control.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.11.658203v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Artificial Intelligence in cancer epigenomics: a review on advances in pan-cancer detection and precision medicine.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Karishma Sahoo, Prakash Lingasamy, Masuma Khatun, Sajitha Lulu Sudhakaran, Andres Salumets, Vino Sundararajan, Vijayachitra Modhukur</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We explore the synergy between AI and DNA methylation profiling to advance precision oncology, including early cancer detection and accurate classification, and how AI-powered epigenetic diagnostics transform clinical practice.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> DNA methylation is a fundamental epigenetic modification that regulates gene expression and maintains genomic stability. Consequently, DNA methylation remains a key biomarker in cancer research, playing a vital role in diagnosis, prognosis, and tailored treatment strategies. Aberrant methylation patterns enable early cancer detection and therapeutic stratification; however, their complex patterns necessitates advanced analytical tools. Recent advances in artificial intelligence (AI) and machine learning (ML), including deep learning networks and graph-based models, have revolutionized cancer epigenomics by enabling rapid, high-resolution analysis of DNA methylation profiles. Moreover, these technologies are accelerating the development of Multi-Cancer Early Detection (MCED) tests, such as GRAIL's Galleri and CancerSEEK, which improve diagnostic accuracy across diverse cancer types. In this review, we explore the synergy between AI and DNA methylation profiling to advance precision oncology. We first examine the role of DNA methylation as a biomarker in cancer, followed by an overview of DNA profiling technologies. We then assess how AI-driven approaches transform clinical practice by enabling early detection and accurate classification. Despite their promise, challenges remain, including limited sensitivity for early-stage cancers, the black-box nature of many AI algorithms, and the need for validation across diverse populations to ensure equitable implementation. Future directions include integrating multi-omics data, developing explainable AI frameworks, and addressing ethical concerns, such as data privacy and algorithmic bias. By overcoming these gaps, AI-powered epigenetic diagnostics can enable earlier detection, more effective treatments, and improved patient outcomes, globally. In summary, this review synthesizes current advancements in the field and envisions a future where AI and epigenomics converge to redefine cancer diagnostics and therapy.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40517231/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Capture-C MPRA: A high-throughput method to simultaneously characterize promoter interactions and regulatory activity</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Arnould, C., Keukeleire, P., Inoue, F., Cui, X., An, K., Murray, E., Zhang, X., Drmanac, R., Peters, B., Shendure, J., Shen, Y., Kircher, M., Ahituv, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a novel technology called Capture-C MPRA (ccMPRA) that combines both technologies and can simultaneously obtain chromatin interactions and regulatory activity and measure CRE activity alongside their target promoters.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cis regulatory elements (CREs) interact with their target promoters over long genomic distances and can be identified using chromatin conformation capture (3C) assays. Their regulatory activity can be functionally characterized in a high-throughput manner using massively parallel reporter assays (MPRAs) that generally test an enhancer alongside a minimal promoter. Here, we developed a novel technology called Capture-C MPRA (ccMPRA) that combines both technologies and can simultaneously obtain chromatin interactions and measure CRE activity alongside their target promoters. We utilized ccMPRA to analyze the regulatory activity of 650 promoters interacting with 42,719 sequences. As C-based techniques also capture isolated promoters, we were able to obtain promoter baseline activity, enabling the identification of both enhancers and silencers. Analysis of CREs interacting with more than one promoter showed significant activity differences depending on the promoter. In summary, ccMPRA can simultaneously characterize chromatin interactions and regulatory activity, allowing to further dissect regulatory grammar.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.11.658967v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Breaking Through Biology's Data Wall: Expanding the Known Tree of Life by Over 10x using a Global Biodiscovery Pipeline</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Vince, O., Oldach, P., Pereno, V., Leung, M. H. Y., Greco, C., Minto-Cowcher, G., Ur-Rehman, S., Kam, K. Y. K., Chow, W., Bolton, E., Mwambingu, B. R., Greenhalgh, N. L., Knot, I. E., Christoffersen, L., Clark, M., Pecoraro, R., Kollasch, A. W., Bohnuud, T., Bakalar, M., Lorenz, P., Gowers, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce BaseData: the largest and fastest-growing biological sequence database ever built, and the first purpose-built for training foundation models.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Advancements in the life sciences have always been built upon our collective understanding of life on Earth. Now, the rise of generative biology - the use of AI foundation models to design, generate, and annotate proteins, pathways and therapeutics - is creating unprecedented demand for large, diverse biological sequence datasets. While a limited subset of such data can be generated in clinical or laboratory settings, the vast majority of the training data for unsupervised models must be sourced from the natural world - the product of nearly four billion years of evolutionary history. However, the public databases that currently supply this data, while foundational to research, were established to aggregate results from academic experiments, not as training datasets for machine learning. Their human-centric data structure limits model performance due to redundancy, taxonomic and geographic bias, limited biological context, and inconsistent provenance. With 68% of all sequence data in the SRA database coming from just 5 species, this is one of the most severe class imbalance problems ever encountered in AI. Legal and infrastructural constraints further exacerbate this bottleneck. To address these limitations and support scalable model training, we introduce BaseData: the largest and fastest-growing biological sequence database ever built, and the first purpose-built for training foundation models. As of late 2024, BaseData contained 9.8 billion novel genes across more than 1 million newly discovered species, representing more than a 10-fold expansion in known protein diversity after accounting for redundancy. Its partnership-driven data supply chain across 26 countries and autonomous regions enables growth of over 2 billion novel genes per month, far exceeding public repositories. All data is collected under benefit sharing agreements using standardized protocols and structured using graph-based, ontology-rich metadata that preserves evolutionary context. BaseData represents a new, ethically grounded infrastructure for training biological foundation models, complementing public efforts and enabling the next era of generative biology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.11.658620v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MLMarker: A machine learning framework for tissue inference and biomarker discovery</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Claeys, T., van Puyenbroeck, S., Gevaert, K., Martens, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present MLMarker, a machine learning-based tool trained on healthy human tissue proteomic data to compute continuous tissue similarity scores for new datasets.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mass spectrometry-based proteomics enables high-throughput profiling of protein expression across tissues but interpreting complex or sparse datasets remains challenging. We here present MLMarker, a machine learning-based tool trained on healthy human tissue proteomic data to compute continuous tissue similarity scores for new datasets. Using a Random Forest model trained across 34 tissue types, MLMarker predicts probabilistic tissue scores and explains these predictions using SHAP values at the single protein level for biological interpretability. To address missingness, a penalty factor can be used to correct for absent proteins contributing to predictions. We demonstrate MLMarker's utility across three public datasets: (i) cerebral melanoma metastases, where brain-like proteomic profiles were linked to poor treatment response and invasive phenotypes, revealing 241 differentially expressed proteins; (ii) a pan-cancer FFPE dataset, where tissue origin predictions reached 79% accuracy using broad tissue mappings despite the absence of cancer-specific training data; and (iii) cerebrospinal fluid and plasma, where the model identified brain and pituitary origin in biofluid samples. MLMarker offers an explainable, robust framework for hypothesis generation in proteomics, applicable across tumours, biofluids, and multi-tissue studies. It is available as a Python package and Streamlit GUI at https://mlmarker.streamlit.app.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.10.658642v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Intratumoral heterogeneity in microsatellite instability status at single cell resolution</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Anthony, H., Seoighe, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We found evidence of heterogeneity in MSI status both in individuals originally classified as MSI-H and MSS. These results raise questions about the current practice of treating MSI as a binary biomarker, ignoring heterogeneity between cancer subclones.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Subclonal diversity within a tumor is highly relevant for tumor evolution and treatment. This diversity is often referred to as intratumoral heterogeneity and is known to complicate the interpretation of single-test biomarkers. Microsatellite instability (MSI) is one such biomarker, which is used to help guide immune checkpoint inhibitor treatment through the classification of samples as either having high microsatellite instability (MSI-H) or as being microsatellite stable (MSS). One area that has yet to be addressed in depth is whether MSI itself is a heterogeneous phenomenon, such that only some subclones are MSI-H. To investigate heterogeneity in MSI status, we curated and analyzed data from several single-cell RNA sequencing studies that had paired clinical MSI status and developed a computational pipeline to infer MSI-H cells and quantify heterogeneity In MSI status. We found evidence of heterogeneity in MSI status both in individuals originally classified as MSI-H and MSS. Out of 49 individuals, 15 showed evidence of divergence in MSI status between distinct clusters of cancer cells and most had distinct MSI-H and MSS subclones. These results raise questions about the current practice of treating MSI as a binary biomarker, ignoring heterogeneity between cancer subclones. Accounting for heterogeneity may lead to improved biomarker performance and, potentially, help explain reports of intrinsic treatment resistance and low overall responder rate in MSI-H cancers. Further studies are warranted to determine the frequency of heterogeneity in this biomarker at the population level, and whether the presence of both MSI-H and MSS subclones can have clinical implications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.11.658021v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Gene and cell line efficiency of CRISPR computed by tensor decomposition in genome-wide CRISPR-Cas9 knockout screens</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Taguchi, Y.-h., Turki, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that tensor decomposition (TD) is the ever first method that can integrate multiple sgRNAs and profiles simultaneously and can achieve comparative performances with one of SOTA.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genome-wide CRISPR-Cas9 knockout screens is often used to evaluate gene functions experimentally. Nevertheless, the efficacy of individual sgRNAs that target unique gene varies and is hard to integrate. In this study, tensor decomposition (TD) was used to integrate multiple sgRNAs as well as sgRNA profiles simultaneously. As a result, TD can discriminate between essential and non-essential genes with the competitive performances of one of SOTA, JACKS, which previously outperformed various SOTA. In addition to this, in spite of that TD is nothing but the simple linear algebra, TD can achieve the performance even without control samples without which JACKS even cannot be performed. Moreover, since raw and logarithmic values can achieve the similar performances through TD for the largest dataset among those tested, taking logarithmic values as has been done frequently is questioned. In conclusion, TD is the ever first method that can integrate multiple sgRNAs and profiles simultaneously and can achieve comparative performances with JACKS, one of SOTA.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.12.659265v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">All-Atom Protein Sequence Design using Discrete Diffusion Models</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Villegas-Morcillo, A., Admiraal, G. J., Reinders, M. J. T., Weber, J. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> All-Atom Protein Sequence Generation using D3PM and D2PM .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Advancing protein design is crucial for breakthroughs in medicine and biotechnology. Traditional approaches for protein sequence representation often rely solely on the 20 canonical amino acids, limiting the representation of non-canonical amino acids and residues that undergo post-translational modifications. This work explores discrete diffusion models for generating novel protein sequences using the all-atom chemical representation SELFIES. By encoding the atomic composition of each amino acid in the protein, this approach expands the design possibilities beyond standard sequence representations. Using a modified ByteNet architecture within the discrete diffusion D3PM framework, we evaluate the impact of this all-atom representation on protein quality, diversity, and novelty, compared to conventional amino acid-based models. To this end, we develop a comprehensive assessment pipeline to determine whether generated SELFIES sequences translate into valid proteins containing both canonical and non-canonical amino acids. Additionally, we examine the influence of two noise schedules within the diffusion process-uniform (random replacement of tokens) and absorbing (progressive masking)-on generation performance. While models trained on the all-atom representation struggle to consistently generate fully valid proteins, the successfully generated proteins show improved novelty and diversity compared to their amino acid-based model counterparts. Furthermore, the all-atom representation achieves structural foldability results comparable to those of amino acid-based models. Lastly, our results highlight the absorbing noise schedule as the most effective for both representations. Data and code are available at https://github.com/Intelligent-molecular-systems/All-Atom-Protein-Sequence-Generation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.13.659451v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MitoDelta: identifying mitochondrial DNA deletions at cell-type resolution from single-cell RNA sequencing data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nakagawa, H., Shima, Y., Sasagawa, Y., Nikaido, I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present MitoDelta, a computational pipeline that enables the detection of mtDNA deletions at cell-type resolution solely from single-cell RNA sequencing data.</p><p style="font-size: 16px; text-align: justify;">Background Deletion variants in mitochondrial DNA (mtDNA) are associated with various diseases, such as mitochondrial disorders and neurodegenerative diseases. Traditionally, mtDNA deletions have been studied using bulk DNA sequencing, but bulk methods average signals across cells, thereby masking the cell-type-specific mutational landscapes. Resolving mtDNA deletions at single-cell resolution is beneficial for understanding how these mutations affect distinct cell populations. To date, no specialized method exists for detecting cell-type-specific mtDNA deletions from single-cell RNA sequencing data. Notably, mtDNA possesses unique molecular features: a high copy number, stable transcription, and compact structure of the mitochondrial genome. This results in a relatively high abundance of mtDNA-derived reads even in single-cell RNA sequencing data, suggesting the possibility of detecting mtDNA deletion variants directly from transcriptomic data. Results Here, we present MitoDelta, a computational pipeline that enables the detection of mtDNA deletions at cell-type resolution solely from single-cell RNA sequencing data. MitoDelta combines a sensitive alignment strategy with robust statistical filtering based on a beta-binomial model, allowing accurate identification of deletion events even from noisy single-cell transcriptomes. To capture cell-type-specific deletion patterns, MitoDelta analyzes reads pooled by annotated cell types, enabling quantification of deletion burden across distinct cellular populations. We benchmarked MitoDelta against existing mtDNA deletion detection tools and demonstrated superior overall performance. As a practical application, we applied MitoDelta to a published single-nucleus RNA sequencing dataset for Parkinson's disease and revealed distinct mtDNA deletion burdens across neuronal subtypes. Conclusions MitoDelta enables the transcriptome-integrated, cell-type-specific detection of mtDNA deletions from single-cell RNA sequencing data alone, offering a valuable framework for reanalyzing public datasets and studying mitochondrial genome alterations at cell-type resolution. This integrated approach enables insights into how mtDNA deletions are distributed across specific cell types and cellular states, providing new opportunities to investigate the role of mtDNA deletions in cell-type-specific disease mechanisms. The tool is available at https://github.com/NikaidoLaboratory/mitodelta.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.10.658806v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Cell-free RNA reveals host and microbial correlates of broadly neutralizing antibody development against HIV</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kowarsky, M., Moufarrej, M. N., Okamoto, J., Xie, Y., Neff, N. N., Abdool Karim, S. S., Garrett, N. J., Moore, P. L., Camunas-Soler, J., Quake, S. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that early immune activation is associated with a transcriptomic signature of HIV broadly neutralizing antibodies (bNAbs) in HIV-positive people, and show that cfDNA/cfRNA sequencing can reveal relationships between a protective immunogenic response to HIV infection, the host immune system, and microbiome</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> An effective broadly reactive antibody response is crucial to prevent viral infections. A small number of people living with HIV (PLWH) develop broadly neutralizing antibodies (bNAbs) targeting multiple HIV strains. Although several viral and immune factors contribute to bNAb development, the genetic and environmental factors driving this response remain largely unknown. We performed combined cell-free DNA (cfDNA) and cell-free RNA (cfRNA) sequencing in 42 plasma samples from a longitudinal cohort of 14 PLWH (7 who develop bNAbs and 7 matched controls). This approach enabled us to non-invasively monitor the host transcriptome, viral genetic variation, and microbiome composition during HIV infection, and to identify molecular correlates of bNAb development. We find that development of bNAbs is associated with a transcriptomic signature of early immune activation that involves MHC class I antigen presentation. This signature is independent of viral load or CD4 count and declines over time. In addition to host features, we recovered sufficient viral reads to reconstruct HIV consensus sequences, supporting the utility of cfRNA for viral genotyping. Finally, we also identified an enrichment of several commensal microbiota in bNAb producers and increased coinfection with GB virus C. Our findings suggest a distinct early immune activation profile in PLWH who develop bNAbs. More broadly, we show that combined cfDNA/cfRNA sequencing can reveal relationships between a protective immunogenic response to HIV infection, the host immune system, and microbiome, highlighting its potential for biomarker discovery in future vaccine and therapeutic studies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.10.658831v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Synthetic Rewiring of Virus-Like Particles via Circular Permutation Enables Modular Peptide Display and Protein Encapsulation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liang, S., Butaney, K., de Castro Assumpcao, D., Jung, J., Kennedy, N. W., Tullman-Ercek, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We demonstrate the power of circular permutation in self-assembling protein nanoparticles and demonstrate new features via covalent bonding.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Virus-like particles (VLPs) are self-assembling nanoparticles derived from viruses with potential as scaffolds for myriad applications. They are also excellent testbeds for engineering protein superstructures. Engineers often employ techniques such as amino acid substitutions and insertions/deletions. Yet evolution also utilizes circular permutation, a powerful natural strategy that has not been fully explored in engineering self-assembling protein nanoparticles. Here, we demonstrate this technique using the MS2 VLP as a model self-assembling, proteinaceous nanoparticle. We constructed, for the first time, a comprehensive circular permutation library of the fused MS2 coat protein dimer construct. The strategy revealed new terminal locations, validated via cryo-electron microscopy, that enabled C-terminal peptide tagging and led to a stable protein encapsulation strategy via covalent bonding - a feature the native coat protein does not permit. Our systematic study demonstrates the power of circular permutation for engineering new features as well as quantitatively and systematically exploring VLP structural determinants.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.13.659512v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Inducible genome-wide mutagenesis for improvement of pDNA production by E. coli</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Li, Z., Sun, G., Zhou, Y.-H., Crook, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A genome-wide mutagenesis strategy to increase plasmid copy numbers in E. coli with significant increases in pDNA production .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Plasmid DNA (pDNA) is a cost-driving reagent for the production of gene therapies and DNA vaccines. Improving pDNA production in the most common production host (E. coli) has faced obstacles arising from the complex network of genes responsible for pDNA synthesis, with the specific enzyme(s) limiting pDNA yield remaining unidentified. To address this challenge, we employed an inducible genome-wide mutagenesis strategy, combined with fluorescent screening, to isolate E. coli NEB 5 strains with enhanced pDNA production. Following selection, we successfully isolated an E. coli strain (M3) with elevated plasmid copy numbers (PCNs) across multiple origin types. Specifically, we observed a 5.93-fold increase in PCN for the GFP reporter plasmid, a 1.93-fold increase for the gWiz DNA vaccine plasmid, and an 8.7-fold increase for the pAAV-CAGG-eGFP plasmid, all of which contain pUC origins. In addition, plasmids with p15A and pSC101 origins showed 1.44-fold and 1.68-fold increases in PCN, respectively. Whole-genome sequencing of the adapted strain M3 identified 85 mutations, including one in recG, which encodes an ATP-dependent DNA helicase. Replacement of the mutant recG with its wild-type counterpart in the mutant strain resulted in a 63% reduction in PCN, but the recG mutation alone was insufficient to increase PCN in the wild-type strain. These findings suggest that the recG mutation plays a synergistic role with other genomic mutations to drive PCN increases. Taken together, this study presents the development of a pDNA hyperaccumulating E. coli strain with promising applications in industrial and therapeutic pDNA production, while also offering important insights into key genes involved in pDNA production.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.13.659583v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Ibaqpy: A scalable Python package for baseline quantification in proteomics leveraging SDRF metadata.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ping Zheng, Enrique Audain, Henry Webel, Chengxin Dai, Joshua Klein, Marc-Phillip Hitz, Timo Sachsenberg, Mingze Bai, Yasset Perez-Riverol</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present ibaqpy, a Python package designed to compute iBAQ values efficiently for experiments of any scale while incorporating experimental metadata into the quantification workflow.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Intensity-based absolute quantification (iBAQ) is essential in proteomics as it allows for the assessment of a protein's absolute abundance in various samples or conditions. However, the computation of these values for increasingly large-scale and high-throughput experiments, such as those using DIA, TMT, or LFQ workflows, poses significant challenges in scalability and reproducibility. Here, we present ibaqpy (https://github.com/bigbio/ibaqpy), a Python package designed to compute iBAQ values efficiently for experiments of any scale. Ibaqpy leverages the Sample and Data Relationship Format (SDRF) metadata standard to incorporate experimental metadata into the quantification workflow. This allows for automatic normalization and batch correction while accounting for key aspects of the experimental design, such as technical and biological replicates, fractionation strategies, and sample conditions. Designed for large-scale proteomics datasets, ibaqpy can also recompute iBAQ values for existing experiments when an SDRF is available. We showcased ibaqpy's capabilities by reanalyzing 17 public proteomics datasets from ProteomeXchange, covering HeLa cell lines with 4921 samples and 5766 MS runs, quantifying a total of 11,014 proteins. In our reanalysis, ibaqpy is a key component in automating reproducible quantification, reducing manual effort and making quantitative proteomics more accessible while supporting FAIR principles for data reuse. SIGNIFICANCE: Proteomics studies often rely on intensity-based absolute quantification (iBAQ) to assess protein abundance across various biological conditions. Despite its widespread use, computing iBAQ values at scale remains challenging due to the increasing complexity and volume of proteomics experiments. Existing tools frequently lack metadata integration, limiting their ability to handle experimental design intricacies such as replicates, fractions, and batch effects. Our work introduces ibaqpy, a scalable Python package that leverages the Sample and Data Relationship Format (SDRF) to compute iBAQ values efficiently while incorporating critical experimental metadata. By enabling automated normalization and batch correction, ibaqpy ensures reproducible and comparable quantification across large-scale datasets. We validated the utility of ibaqpy through the reanalysis of 17 public HeLa datasets, comprising over 200 million peptide features and quantifying 11,000 proteins across thousands of samples. This comprehensive reanalysis highlights the robustness and scalability of ibaqpy, making it an essential tool for researchers conducting large-scale proteomics experiments. Moreover, by promoting FAIR principles for data reuse and interoperability, ibaqpy offers a transformative approach to baseline protein quantification, supporting reproducible research and data integration within the proteomics community.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40268243/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Intersecting impact of CAG repeat and huntingtin knockout in stem cell-derived cortical neurons.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jennifer T Stocksdale, Matthew J Leventhal, Stephanie Lam, Yu-Xin Xu, Yang Oliver Wang, Keona Q Wang, Reuben Thomas, Zohreh Faghihmonzavi, Yogindra Raghav, Charlene Smith, Jie Wu, Ricardo Miramontes, Kanchan Sarda, Heather Johnston, Min-Gyoung Shin, Terry Huang, Mikelle Foster, Mariya Barch, Naufa Amirani, Chris Paiz, Lindsay Easter, Erse Duderstadt, Vineet Vaibhav, Niveda Sundararaman, Dan P Felsenfeld, Thomas F Vogt, Jennifer Van Eyk, Steve Finkbeiner, Julia A Kaye, Ernest Fraenkel, Leslie M Thompson</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel feature-based pipeline and multi-omics approach to study the impact of the CAG repeat expansion in Huntington's Disease and its associated loss-of-function mechanisms.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Huntington's Disease (HD) is caused by a CAG repeat expansion in the gene encoding huntingtin (HTT). While normal HTT function appears impacted by the mutation, the specific pathways unique to CAG repeat expansion versus loss of normal function are unclear. To understand the impact of the CAG repeat expansion, we evaluated biological signatures of HTT knockout (HTT KO) versus those that occur from the CAG repeat expansion by applying multi-omics, live cell imaging, survival analysis and a novel feature-based pipeline to study cortical neurons (eCNs) derived from an isogenic human embryonic stem cell series (RUES2). HTT KO and the CAG repeat expansion influence developmental trajectories of eCNs, with opposing effects on growth. Network analyses of differentially expressed genes and proteins associated with enriched epigenetic motifs identified subnetworks common to CAG repeat expansion and HTT KO that include neuronal differentiation, cell cycle regulation, and mechanisms related to transcriptional repression, and may represent gain-of-function mechanisms that cannot be explained by HTT loss of function alone. A combination of dominant and loss-of-function mechanisms are likely involved in the aberrant neurodevelopmental and neurodegenerative features of HD that can help inform therapeutic strategies.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40258535/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identification and taste presentation characteristics of umami peptides from soybean paste based on peptidomics and virtual screening.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Guoyang Pan, Feiyu An, Junrui Wu, Yaqi Wang, Li Deng, Xing Tong, Shuaiqi Ji, Jinhui Jiang, Qu Liu, Ning Yang, Dongbing Tao, Rina Wu</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-02-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> This research concentrated on soybean paste fermented with Tetragenococcus halophilus, employing peptidomics and machine learning methodologies to screen for novel umami peptides.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> This research concentrated on soybean paste fermented with Tetragenococcus halophilus, employing peptidomics and machine learning methodologies to screen for novel umami peptides. Taste characteristics of umami peptides were evaluated through sensory evaluation and electronic tongue analysis. The mechanism of taste presentation of the umami peptides was investigated through T1R1/T1R3 molecular docking techniques. Four peptides were identified: LLYGKVVKKT, DKKVSVGT, TRKQALLN, and QKNSHQ, with umami thresholds ranging from 0.02 to 0.14 mmol/L. Hydrogen bonds and electrostatic interactions are the key forces between umami peptides and receptors, and the length of hydrogen bonds is between 2.94 and 3.30 √Ö. Molecular docking analyses revealed that electrostatic and hydrogen bonding interactions are crucial, with ARG 248 and ALA 282 serving as key binding sites on T1R1 and T1R3 receptors, significantly influencing umami intensity. These findings aid in further understanding the flavor properties of umami peptides in soybean paste.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40023041/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

