<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Friday, 13 June 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multi-omics integration of TPX2 in prognostic prediction in resectable hepatocellular carcinoma.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Haoran Zhao, Weiwei Duan, Tong Qu, Xu Tong, Chao Zhan, Yubao Zhang</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a multi-omics prognostic model for HCC and explore the role of TPX2 gene in HCC using bioinformatics and cellular experiments, assessing its feasibility as a potential therapeutic target.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Hepatocellular carcinoma (HCC) is a malignant tumor with high incidence and mortality rates globally, significantly affecting patient prognosis and quality of life. Currently, predicting postoperative recurrence risk and survival in HCC patients remains challenging, as precise and effective quantitative indicators are lacking, limiting clinicians' ability to make individualized treatment decisions. Multi-omics models, as an emerging predictive tool, combine tumor imaging features, molecular markers, and clinical data, offering a new direction for prognostic evaluation in HCC. Multi-omics models integrating radiomics, genomics, and clinical factors have shown potential in predicting prognosis and treatment response in HCC patients. This study aims to develop a multi-omics prognostic model that integrates radiomic features, the genetic biomarker Targeting Protein for Xklp2 (TPX2), and clinical data to accurately predict postoperative overall survival (OS) and recurrence-free survival (RFS) in patients with resectable HCC. Additionally, this study explores the role of the TPX2 gene in HCC using bioinformatics and cellular experiments, assessing its feasibility as a potential therapeutic target. Through these investigations, we aim to provide new theoretical insights and application prospects for the precision diagnosis and treatment of HCC.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40506577/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Sparse deconvolution of cell type medleys in spatial transcriptomics.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nuray Sogunmez Erdogan, Deniz Eroglu</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Weight-Induced Sparse Regression (WISpR), a machine learning algorithm that integrates spot-specific hyperparameters and sparsity-driven modeling, which accurately predicts cell-type distributions while preserving biological coherence, even in unmatched datasets.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mapping cell distributions across spatial locations with whole-genome coverage is essential for understanding cellular responses and signaling However, current deconvolution models aim to estimate the proportions of distinct cell types in each spatial transcriptomics spot by integrating reference single-cell data. These models often assume strong overlap between the reference and spatial datasets, neglecting biology-grounded constraints such as sparsity and cell-type variations, as well as technical sparsity. As a result, these methods rely on over-permissive algorithms that ignore given constraints leading to inaccurate predictions, particularly in heterogeneous or unmatched datasets. We introduce Weight-Induced Sparse Regression (WISpR), a machine learning algorithm that integrates spot-specific hyperparameters and sparsity-driven modeling. Unlike conventional approaches that neglect biology-grounded constraints, WISpR accurately predicts cell-type distributions while preserving biological coherence, i.e., spatially and functionally consistent cell-type localization, even in unmatched datasets. Benchmarking against five alternative methods across ten datasets, WISpR consistently outperformed competitors and predicted cellular landscapes in both normal and cancerous tissues. By leveraging sparse cell-type arrangements, WISpR provides biologically informed, high-resolution cellular maps. Its ability to decode tissue organization in both healthy and diseased states highlights WISpR's practical utility for spatial transcriptomics, particularly in challenging settings involving noise, sparsity, or reference mismatches.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40505018/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Single-cell Analysis of Intracellular Transport and Expression of Cell Surface Proteins</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mudappathi, R., Bhardwaj, V., Liu, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel computational frame for Modeling Protein Expression and Transport (MPET) that evaluates the contribution of ICT activity to differential surface protein expression using CITE-seq data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Intracellular protein transport (ICT) is a tightly regulated process that orchestrates protein localization and expression, ensuring proper cellular function. Dysregulated ICT can lead to aberrant expression of surface proteins involved in cell-cell communication, adhesion, and immune responses, contributing to disease progression and therapeutic resistance. Cellular Indexing of Transcriptomes and Epitopes by Sequencing (CITE-seq) enables the simultaneous measurement of mRNA and surface protein levels in the same cell, providing a powerful opportunity to investigate the molecular mechanisms underlying surface protein regulation. In this study, we introduce a novel computational frame for Modeling Protein Expression and Transport (MPET) that evaluates the contribution of ICT activity to differential surface protein expression using CITE-seq data. MPET comprises three modules for identification of ICT and surface protein regulatory circuits across biological scales and their contributions to phenotypic variation. We applied MPET to analyze single-cell data from COVID-19 patients with varying disease severity. Our analysis revealed context-dependent recruitment of ICT genes and pervasive rewiring of ICT pathways throughout the course of disease progression. Notably, we found that even when the transcriptional levels of key immune response proteins remained stable, their expression on cell surface were significantly altered due to dysregulated ICT. MPET provides a valuable new tool for dissecting complex regulatory networks and offers mechanistic insight into post-transcriptional regulation of cell surface proteins in diseases.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.09.658704v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A survey of dietary effects on tRNA abundance and modifications</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liu, S., Holmes, A. D., Zhao, R., Herbert, C., Limbach, P. A., Lowe, T. M., Sharma, U.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We investigate the effects of dietary interventions on tRNA abundance and modification profiles across multiple mouse tissues and show that low-protein and high-fat diets can significantly alter the abundance and modifications of tRNAs.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Transfer RNAs (tRNAs) play a central role in protein translation and are increasingly recognized as dynamic regulators of gene expression. Both physiological and environmental signals can modulate tRNA abundance and chemical modifications, yet the impact of dietary cues on the tRNA landscape remains poorly understood. Here, we investigated the effects of two distinct dietary interventions--low-protein and high-fat diets--on tRNA abundance and modification profiles across multiple mouse tissues. We conducted a comprehensive analysis of tRNA abundance and modification changes in response to these nutritional challenges using RNA mass spectrometry and Ordered Two-Template Relay sequencing (OTTR-seq), a modified-base-sensitive tRNA sequencing method. Our results reveal both shared and tissue-specific alterations in abundance and modifications of specific nuclear and mitochondrial genome-encoded tRNAs in response to dietary conditions at isotype, isoacceptor, and isodecoder levels. As many of the tissue-specific or diet-responsive tRNA modifications have been previously reported to affect decoding efficiency or translational fidelity, these results have implications for understanding translational adaptation in response to dietary conditions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.10.658430v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Direct RNA nanopore sequencing reveals rapid RNA modification changes following glucose stimulation of human pancreatic beta-cell lines</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mulroney, L., Taylor, H. J., Lee, A., Swift, A. J., Zdravkov, M., Bonnycastle, L. L., Brooks, S. Y., Lee, B. N., Fitzgerald, T., Narisu, N., Biesecker, L. G., Erdos, M. R., Nicassio, F., Birney, E., Collins, F. S., Taylor, D. L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We studied RNA modifications and expression levels in human pancreatic beta-cell lines after one hour of glucose stimulation and identified more than 1,700 differentially modified sites across all modifications.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> RNA modifications are critical regulators of gene expression and cellular processes; however, the epitranscriptome is less well studied than the epigenome. Here, we studied transcriptome-wide changes in RNA modifications and expression levels in two human pancreatic beta-cell lines, EndoC-BH1 and EndoC-BH3, after one hour of glucose stimulation. Using direct RNA nanopore sequencing (dRNA-seq), we measured N6-methyladenosine (m6A), 5-methylcytosine (m5C), inosine, and pseudouridine concurrently across the transcriptome. We developed a differential RNA modification method and identified 1,697 differentially modified sites (DMSs) across all modifications. These DMSs were largely independent of changes in gene expression levels and enriched in transcripts for type 2 diabetes (T2D) genes. Our study demonstrates how dRNA-seq can be used to detect and quantify RNA modification changes in response to cellular stimuli at the single-nucleotide level and provides new insights into RNA-mediated mechanisms that may contribute to normal beta-cell response and potential dysfunction in T2D.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.12.659352v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Analysis of 3'-seq data from multiple E. coli studies identifies diverging results sets and raw data characteristics despite similar collection conditions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Furumo, Q., Meyer, M. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified large disparities in the characteristics of raw 3'-seq data between each of the studies, despite all three studies using the same strain and very similar reported experimental conditions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> 3-prime end sequencing (3'-seq) is a high-throughput sequencing technique that is used to specifically quantify the changes in 3'-end formation of transcripts in bacterial cells, which is increasingly being utilized to address fundamental questions regarding transcription termination and pausing across a range of different bacterial species. However, the growing number of 3'-seq studies is accompanied by an increase in study-specific 3'-seq data analysis approaches. Thus, differences in a number of factors including: experimental design, data collection approaches, analysis methodologies, and interpretation decisions, make it challenging to confidently compare results derived from different studies, even those that were performed on the same organism. To assess the potential severity of these discrepancies, we used PIPETS, a statistically robust and genome-annotation agnostic 3'-seq analysis package, to study Escherichia coli 3'-seq data sets from three different groups collected under similar conditions. By using a consistent analysis and results interpretation approach, we identified large disparities in the characteristics of the raw 3'-seq data between each of the studies, despite all three studies using the same strain and very similar reported experimental conditions. Additionally, we found strand-specific inconsistencies, with some data sets having reference strand 3'-seq read coverage distributions that differed greatly from the complement strand within the same replicate. Finally, when the 3'-seq distribution profiles of the three E. coli studies are compared to studies from four additional bacteria, we identified 3'-seq results clustering patterns that are not explained by phylogenetic similarity between organisms. With the large differences seen between data sets from the same organism as well as the inconsistencies seen between replicates from the same data sets, we urge the field to reconsider the assumptions around 3'-seq data homogeneity and move towards consistent analysis approaches, and cautious interpretation of the data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.12.658996v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrating structural homology with deep learning to achieve highly accurate protein-protein interface prediction for the human interactome</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xiong, D., Torres, M., Murray, D., Li, L., Naravane, A. C., Fragoza, R., Honig, B., Yu, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A significant portion of disease-causing mutations occur at protein-protein interfaces and for these cases, PIONEER2.0 significantly outperforms AlphaFold3 in predicting their interface residues.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> A significant portion of disease-causing mutations occur at protein-protein interfaces however, the number of structurally resolved multi-protein complexes is extremely small. Here we present a computational pipeline, PIONEER2.0, that integrates 3D structural similarity with geometric deep learning to accurately predict protein binding partner-specific interfacial residues for all experimentally observed human binary protein-protein interactions. We estimate that AlphaFold3 fails to produce high-quality structural models for about half of the human interactome; for these challenging cases, PIONEER2.0 significantly outperforms AlphaFold3 in predicting their interface residues, making PIONEER2.0 an excellent alternative and complementary tool in real-world applications. We further systematically validated PIONEER2.0 predictions experimentally by generating 1,866 mutations and testing their impact on 5,010 mutation-interaction pairs, confirming PIONEER-predicted interfaces are comparable in accuracy as experimentally determined interfaces using PDB co-complex structures. We then used PIONEER2.0 to create a comprehensive multiscale structurally informed human interactome encompassing all 352,124 experimentally determined binary human protein interactions in the literature. We find that PIONEER2.0-predicted interfaces are instrumental in prioritizing disease-associated mutations and thus provide insight into their underlying molecular mechanisms. Overall, our PIONEER2.0 framework offers researchers a valuable tool at an unprecedented scale for studying disease etiology and advancing personalized medicine.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.09.658393v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A fast TMT-based proteomic workflow reveals neural enrichment in neurospheres of hiPSC-derived neural stem cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Muniz, P. d. L., Souza, L. R. Q., Martins, M. R., Valadares, W., Nogueira, F. C. S., Rehen, S., Nugue, G., Junqueira, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a streamlined and scalable TMT-based quantitative proteomics workflow to contrast the proteomic landscapes of hiPSC-derived neural stem cells cultured in 2D monolayers versus 3D neurospheres, and demonstrate a proteomic shift toward a more mature neural phenotype.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Three-dimensional (3D) neural spheroids, or neurospheres, generated from human induced pluripotent stem cell (hiPSC)-derived neural stem cells (NSCs) more accurately recapitulate the microenvironmental cues of neural tissue compared to traditional two-dimensional (2D) monolayers. However, comparative omics-based characterizations of these models remain limited. Here, we present a streamlined and scalable TMT-based quantitative proteomics workflow to contrast the proteomic landscapes of hiPSC-derived NSCs cultured in 2D monolayers versus 3D neurospheres. A total of 1,576 proteins were identified in an unfractionated LC-MS/MS of 68 minutes, with 542 showing significant differential abundance between groups. Neurospheres exhibited enrichment in neural-related pathways, such as synaptic signaling, neurotrophin signaling, cytoskeletal organization and vesicle trafficking, while monolayers enriched for multipotency features, such as general metabolic activity. Cell-type enrichment analyses confirmed increased neuronal identity in neurospheres, including elevated levels of markers associated with neuronal maturation. Our results demonstrate that 3D culture of NSCs induces a proteomic shift toward a more mature neural phenotype. This rapid, multiplexed proteomic approach enables high-content molecular profiling suitable for drug screening and personalized medicine applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.09.658417v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CSLAN: Cross-Species Latent Alignment Network for Trauma-Related Cell-Type Classification</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wu, R., Shi, A., Liu, Y., Duprey, A., Chen, T., Li, R., Cao, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce the Cross-Species Latent Alignment Network (CSLAN), a transfer learning framework that effectively bridges this gap for robust cell type classification.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding cellular identity and function across species is fundamental to deciphering conserved biological principles and translating insights from model organisms to human health. However, integrating single-cell transcriptomic data, particularly from resource-limited human studies, with comprehensive datasets from model organisms like mice remains a significant challenge due to evolutionary divergence and technical variability. Here, we introduce the Cross-Species Latent Alignment Network (CSLAN), a transfer learning framework that effectively bridges this gap for robust cell type classification. CSLAN employs a novel strategy involving focused feature selection on source (mouse) data followed by pre-training an encoder-decoder architecture. Crucially, for transfer to human data, only the encoder is fine-tuned while the latent space processor and the decoder, which encapsulate learned, conserved cell type signatures, remains fixed. This targeted adaptation mechanism allows CSLAN to achieve exceptional accuracy (e.g., 96.67% for human immune cells in a trauma context) by efficiently aligning human gene expression profiles to a biologically informed latent space derived from mouse data. Our findings establish a powerful and broadly applicable paradigm for cross-species knowledge transfer, enabling deeper biological insights from limited human scRNA-seq data by leveraging the wealth of information from model systems, with significant implications for translational research and the construction of comprehensive, cross-species cell atlases.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.09.655966v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Biologically Guided Variational Inference for Interpretable Multimodal Single-Cell Integration and Mechanistic Discovery</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Arnoldt, L., Upmeier zu Belzen, J., Herrmann, L., Nguyen, K., Theis, F. J., Wild, B., Eils, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> NetworkVI is a sparse deep generative model designed for the integration and interpretation of multimodal single-cell data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Multi-omics technologies allow for a detailed characterization of cell types and states across multiple omics layers, helping to identify features that differentiate biological conditions, such as chemical or CRISPR-based perturbations. However, current tools employing variational inference on single-cell datasets, including methods for paired and mosaic integration, transfer learning, and modality imputation, typically act as black boxes. This lack of interpretability makes it challenging to evaluate whether biological variation is preserved, which can compromise downstream analyses. Here, we introduce NetworkVI, a sparse deep generative model designed for the integration and interpretation of multimodal single-cell data. NetworkVI utilizes biological prior knowledge as an inductive bias, specifically it relies on gene-gene interactions inferred from topologically associated domains and structured ontologies like the Gene Ontology to aggregate gene embeddings to cell embeddings, enhancing the interpretability at the gene and subcellular level. While achieving state-of-the-art data integration, modality imputation, and cell label transfer via query-to-reference mapping benchmarks across bimodal and trimodal datasets, NetworkVI additionally excels in providing biologically meaningful modality- and cell type-specific interpretations. NetworkVI aids researchers in identifying associations between genes and biological processes and uncovers immune evasion mechanisms in a Perturb CITE-seq dataset of melanoma cells. NetworkVI will support researchers in interpreting cellular disease mechanisms, guiding biomarker discovery, and ultimately aiding the development of targeted therapies in large-scale single-cell multimodal atlases. NetworkVI is available at http://github.com/LArnoldt/networkVI .</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.10.657924v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Deacon: fast sequence filtering and contaminant depletion</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Constantinides, B., Lees, J., Crook, D. W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present Deacon, an efficient and versatile sequence filter for raw sequence files and streams, which is able to accurately classify and discard diverse human sequences from long reads at over 250Mbp/s with a commodity laptop.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: Realising the value of large DNA sequence collections demands efficient search and extraction of sequences of interest. Search queries may vary in size from short gene sequences to multiple whole genomes that are too large to fit in computer memory. In microbial genomics, a routine search application involving both large queries and large collections is the removal of contaminating host genome sequences from microbial (meta)genomes. Where the host is human, sensitive classification and excision of host sequences is usually necessary to protect host genetic information. Precise classification is also critical in order to retain microbial sequences and permit accurate microbial genomic analysis. While human pangenomes have been shown to increase sensitivity of human sequence classification, existing bioinformatic host depletion approaches have either limited precision when used with metagenomes or large computing resource requirements. Results: We present Deacon, an efficient and versatile sequence filter for raw sequence files and streams. We demonstrate its leading accuracy for the task of host depletion, using less computing resource than existing approaches. By querying a human pangenome index for minimizers contained in each input sequence, Deacon is able to accurately classify and discard diverse human sequences from long reads at over 250Mbp/s with a commodity laptop. We present validation of classification sensitivity, specificity and speed with simulated short and long reads for diverse catalogues of human, bacterial and viral genomes alongside existing methods. Beyond host depletion, Deacon is well suited to common sequence search and filtering applications, particularly those involving large queries. Capable of indexing a human genome in under 30s, Deacon is equipped to rapidly compose custom minimizer indexes using set operations, facilitating efficient search and filtering of massive sequence datasets using gigabase queries. Availability and implementation: Deacon is implemented as an MIT-licensed command line tool written in Rust and packaged with Bioconda. Code is available from https://github.com/bede/deacon.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.09.658732v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multiplets in scRNA-seq data: extent of the problem and efficacy of methods for removal</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ttoouli, D., Hoffmann, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that heuristics are systematically underestimating the number of multiplets in scRNA-seq analysis, and that existing tools-despite optimized parameters-detect only a small subset of cell-hashing multiplets.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Multiplets-droplets that capture more than one cell-are a known artefact in droplet-based single-cell RNA sequencing (scRNA-seq), yet their prevalence and impact remain underestimated. In this study, we assess the frequency of multiplets across diverse publicly available datasets and evaluate how well commonly used detection tools are able to identify them. Using cell hashing data to determine a lower bound of the true multiplet rate, we demonstrate that commonly used heuristic estimations systematically underestimate multiplet rates, and that existing tools-despite optimized parameters-detect only a small subset of cell-hashing multiplets. We further refine a Poisson-based model to estimate the true multiplet rate, revealing that actual rates can exceed heuristic predictions by more than twofold. Downstream analyses are significantly affected by multiplets: they are not confined to isolated clusters but are distributed throughout the transcriptional landscape, where they distort clustering and cell type annotation. Using both quantitative and qualitative approaches, we visualize these effects and show that cell-hashing-informed multiplet removal eliminates artefactual clusters and improves annotation clarity, whereas computationally detected multiplets fail to fully remove artefacts in the most common experimental contexts. Our findings confirm that multiplet contamination remains a pervasive and under-addressed issue in scRNA-seq analysis. Since most datasets lack multiplexing, researchers must often rely on heuristics and limited tools, leaving many multiplets unidentified. We advocate for more robust multiplet-detection strategies, including multimodal validation, to ensure more accurate and interpretable scRNA-seq results.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.09.658708v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Analysis of protein aging reveals rates of subcellular organelle renewal and selective post-translational modification in Arabidopsis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tivendale, N., Liu, X., Millar, A. H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We use pulse chase biorthogonal amino acid tagging and affinity purification to capture and analyse snapshots of the cellular proteome as it ages over a two-week period.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The cellular proteome represents a mixture of older and newer copies of each protein type and turnover of this mixture occurs by cycles of protein synthesis and degradation. There is considerable research on new protein synthesis, the nature of nascent proteins and cellular machinery of protein degradation. However, we have limited insights into older proteins or the protein aging process in plants at scale. In this study we use pulse chase biorthogonal non-canonical amino acid tagging (BONCAT) in Arabidopsis cells coupled to affinity purification to capture and analyse snapshots of the cellular proteome as it ages over a two-week period. Each snapshot was subjected to peptide mass spectrometry-based identification, quantitation and characterisation. We show that there are a broad range of lifespans among the 1688 proteins studied and that their subcellular location correlates strongly with protein longevity. Mitochondria, plastids and the extracellular environment contained the longest lived sub-proteomes while the vesicular pathway to ER, PM and peroxisomes contained the shortest-lived protein sets. Abundant primary metabolic enzymes have considerable longevity, while kinases and ubiquitination machinery do not. Through analysis of the aging profiles, we demonstrate that many proteins selectively accumulate posttranslational modifications (PTMs) as they age and that these are mostly oxidative in nature. We show by analysis of exemplar proteins that distinct PTM profiles and proportional changes with age exist between proteins, likely dictated by differences in subcellular environment and protein function. Implications of these insights for understanding cellular function and for biotechnological modification of the plant proteome are discussed.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.09.658759v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Organellar data sets confirm overall angiosperm relationships if problematic RNA-edit sites are accounted for in mitochondrial genomes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gerelle, W. K., Jost, M., Marques, I., Les, D., Vallejos, R., Little, S., Sinn, B. T., Sokoloff, D. D., Macfarlane, T. D., Iles, W., Feild, T., Mathews, S., Moore, M., Couvreur, T. L. P., Sauquet, H., Wanke, S., Graham, S. W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of phylogenetic inferences of early flowering-plant relationships using both organellar genomes and filtered mitochondrial data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Premise: Plastid-based data sets continue to play a major role in our understanding of early flowering-plant relationships, although organellar genomes of major lineages outside the monocots and eudicots remain under-sampled. A tendency of mitochondrial RNA-edit sites to mislead phylogenetic analysis in mixed transcriptomic/genomic data sets needs attention in angiosperm-wide studies, which only rarely consider mitochondrial genomes. Methods: We compared mitochondrial- vs. plastid-based phylogenomic inferences, examined the effect of removing putative RNA-edit sites from mitochondrial data, and performed combined organellar analysis (plastid plus filtered mitochondrial genomes). We expanded taxon sampling for multiple angiosperm lineages for phylogenomic analysis using both organellar genomes, representing several poorly sampled lineages (in particular Degeneriaceae, Trimeniaceae) with smaller (few-gene) data sets. Results: Plastid-based inferences recover well-supported relationships that align with and build upon previous studies, and recover well-supported internal relationships for two ANA-grade families (Hydatellaceae, Trimeniaceae) sampled for nearly all species. By contrast, unfiltered mitochondrial inferences of angiosperm phylogeny are generally poorly supported, and recover anomalous relationships compared to plastid-based inferences. However, removing putative mitochondrial RNA-edit sites dramatically reduces inter-organellar conflict and improves overall branch support. Conclusions: We accounted for phylogenomic discordance between the two organellar genomes regarding overall angiosperm-wide relationships and filled in taxonomic gaps (poorly sampled lineages). Removing RNA edit sites substantially improves congruence in interorganellar inferences by effectively correcting a systematic bias in mitochondrial data. Uncertain relationships persist among five major mesangiosperm lineages in plastid-based inferences, but a clade comprising Chloranthales, Ceratophyllales and eudicots is well supported by filtered mitochondrial data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.09.658674v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Phage-Assisted Evolution of Allosteric Protein Switches</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Southern, N. T., von Bachmann, A.-L., Hovsepyan, A., Griebl, M.-L., Wolf, B., Lemmen, N., Kroell, A.-S., Westermann, S., Mathony, J., Niopek, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a phage-assisted evolution platform for in vivo optimization of allosteric proteins for programmable cellular control.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Allostery, the transmission of locally induced conformational changes to distant functional sites, is a key mechanism for protein regulation. Artificial allosteric effectors enable remote manipulation of cell function; their engineering, however, is hampered by our limited understanding of allosteric residue networks. Here, we introduce a phage-assisted evolution platform for in vivo optimization of allosteric proteins. It applies opposing selection pressures to enhance activity and switchability of phage-encoded effectors and leverages retron-based recombineering to broadly explore fitness landscapes, covering point mutations, insertions and deletions. Applying our pipeline to the transcription factor AraC yielded optogenetic variants with light-controlled activity spanning ~1000-fold dynamic range. Long-read sequencing across selection cycles revealed adaptive trajectories and corresponding allosteric interactions. Our work facilitates phage-assisted evolution of allosteric proteins for programmable cellular control.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.12.659331v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Knowledge Graph-Based Framework for Detecting Horizontal Gene Transfer Events Driving Antimicrobial Resistance</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Islam, M. R., Summers, A. O., Arpinar, I. B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a comprehensive knowledge graph-based framework that integrates genomic, environmental, biochemical, and ontological datasets to detect and analyze HGT events contributing to antimicrobial resistance.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Horizontal gene transfer (HGT) is a key mechanism driving the rapid dissemination of antimicrobial resistance (AMR) among bacterial populations, posing a severe threat to global public health. Traditional detection methods for HGT, such as phylogenetic incongruence and compositional analysis, are often limited in their scope and accuracy. To address these challenges, we propose a comprehensive knowledge graph (KG)-based framework that integrates genomic, environmental, biochemical, and ontological datasets to detect and analyze HGT events contributing to AMR. Our KG captures relationships among genes, plasmids, species, mobile genetic elements, antibiotics, and ecological factors using curated sources like NCBI, CARD, ICEberg, MGnify, and KEGG. We enrich nodes and edges with biological (e.g., GC content, codon bias) and chemical (e.g., binding affinities, environmental exposure) signatures and apply ontology-based semantic enrichment using ARO and GO. A Graph Neural Network (GNN) trained on the KG achieves high accuracy (92%) and F1-score (0.90) in predicting HGT events, outperforming conventional methods. Community detection identifies AMR dissemination hubs, while environmental analyses reveal strong correlations between antibiotic concentrations and resistance gene prevalence. Our framework demonstrates strong biological and chemical validation and provides a scalable, interpretable, and data-rich tool for AMR surveillance, with implications for clinical, agricultural, and environmental microbiology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.09.658534v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

