<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Sunday, 18 May 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Quantitative electroencephalogram and machine learning to predict expired sevoflurane concentration in infants.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rachit Kumar, Justin Skowno, Britta S von Ungern-Sternberg, Andrew Davidson, Ting Xu, Jianmin Zhang, XingRong Song, Mazhong Zhang, Ping Zhao, Huacheng Liu, Yifei Jiang, Yunxia Zuo, Jurgen C de Graaff, Laszlo Vutskits, Vanessa A Olbrecht, Peter Szmuk, Allan F Simpao, Fuchiang Rich Tsui, Jayant Nick Pratap, Asif Padiyath, Olivia Nelson, Charles D Kurth, Ian Yuan</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We proposed that machine learning combined with qEEG can accurately classify expired sevoflurane concentrations in young infants.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Processed electroencephalography (EEG) indices used to guide anesthetic dosing in adults are not validated in young infants. Raw EEG can be processed mathematically, yielding quantitative EEG parameters (qEEG). We hypothesized that machine learning combined with qEEG can accurately classify expired sevoflurane concentrations in young infants. Knowledge from this may contribute to development of future infant-specific EEG algorithms. Frontal EEG collected from infants ≤ 3 months were time-matched as one-minute epochs to expired sevoflurane (eSevo). Fifteen qEEG parameters were extracted from each epoch and eight machine learning models combined the qEEG to classify each epoch into one of four eSevo levels (%): 0.1-1.0, 1.0-2.1, 2.1-2.9, and > 2.9. 64 epochs formed the post hoc SHAP dataset to determine the qEEG that contributed most to the model. The remaining epochs were randomly split 50 times into 80/20 training/testing sets. Accuracy and F1-score determined model performance. 42 infants provided 4574 epochs. The top classifiers K-nearest neighbors, default multi-layer perceptron, and support vector machine achieved 67.5-68.7% accuracy. Burst suppression ratio and entropy β were the top contributors to the models. Post hoc analysis performed without burst suppression ratio yielded similar prediction performance. In young infants, machine learning applied to qEEG predicted eSevo levels with moderate success. Burst suppression ratio, the most important contributor, represented an efficient EEG feature that encapsulated underlying EEG changes seen on other qEEG features. These results provided insight into EEG parameter selection and optimal machine learning models used for future development of infant-specific EEG algorithms.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40381151/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Case Study of Using AI as Co-Pilot in Biotech Research: Functional Network Analysis of Invasive Cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sinitskiy, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Using AI systems as co-pilots in biological research, focusing on functional protein networks in invasive colorectal cancer, we identified cytoskeleton- and signaling-related networks, aligning with known biology.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> This study presents a case analysis of using AI systems as co-pilots in biological research, focusing on functional protein networks in invasive colorectal cancer. We used public proteomic data alongside ChatGPT, GitHub Copilot, and PaperQA to automate parts of the workflow, including literature review, code generation, and network analysis. While AI tools improved efficiency, they required expert guidance for tasks involving complex metadata, domain-specific parsing, and reproducibility. Our analysis identified cytoskeleton- and signaling-related networks in invasive cancer, aligning with known biology, but attempts to distinguish invasive from non-invasive cases produced inconclusive results. An attempt to conduct fully automated research using Agent Laboratory failed due to hallucinated data, misinterpretation of research goals, and instability as the complexity of the underlying LLM increased. These findings show that current AI can assist but not replace human researchers in complex biotech studies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.14.654152v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Systems Biology Approach to Unveil Shared Therapeutic Targets and Pathological Pathways Across Major Human Cancers</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> ALAM, A., Siddiqui, M. F., Hamoudi, R. A., Kishore, U., Fernandez Cabezudo, M., Al Ramadi, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified 26 cancer hallmark genes across 18 different cancer types and identified 179 shared signature genes across the SCTs, which can be used to identify pan-cancer diagnostic tools and therapeutic targets.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cancer is a leading global cause of mortality, responsible for nearly 10 million deaths annually, with breast, lung, colorectal, and prostate cancers among the most prevalent. Despite extensive research on individual cancer types, identifying shared molecular signatures could unlock pan-cancer diagnostic tools and therapeutic targets. This study leveraged RNA-seq data from The Cancer Genome Atlas (TCGA) to analyze our selected cancer types (SCTs), breast, lung, colorectal, and prostate, employing a multi-step bioinformatics pipeline. Differentially expressed genes (DEGs) between tumors and normal tissues were identified and validated using an Elastic Net regression model. Weighted Gene Co-expression Network Analysis (WGCNA) revealed highly correlated gene modules linked to clinical traits, pinpointing 179 shared signature genes across the SCTs. Protein-protein interaction (PPI) network and clustering analyses further refined these to 26 hub genes, enriched in cancer hallmark pathways. Nine hub genes KIF18B, RRM2, MYBL2, IQGAP3, TPX2, SLC7A11, RHPN1, HJURP, and SKA3 are stood out due to consistent upregulation in metastatic tumors (breast, colorectal, prostate) and their high expression across more than 18 different cancer types, suggesting roles as oncogenes, prognostic markers, or therapeutic targets. The expression patterns of hub genes were further validated across larger cancer patient cohorts of SCTs, confirming relevance across multiple datasets, and their prognostic significance was assessed by their influence on overall survival (OS). Notably, these hub genes also correlated with immune-related functions, potentially influencing tumor microenvironment modulation. This integrative approach provides a strong framework for identifying cross-cancer potential biomarkers, advancing pan-cancer insights, and supporting improved diagnosis, prognosis, and therapy development.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.14.653945v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">LSD1 serine 166 is a phosphorylation switch for chromatinlandscaping, gene activation, and tissue remodeling</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sharma, M., Aakula, A., Denisova, O., Ceccacci, E., Hayatigolkhatmi, K., Caroli, J., Natkin, R., Laisne, M.-C., Huhtaniemi, R., Ghimire, S., Merilahti, J., Noberini, R., Kamila, J., Kauko, O., Mattevi, A., Bonaldi, T., Jacquemet, G., Nykter, M., Minucci, S., Westermarck, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that phosphorylation of serine 166 of LSD1 is a phosphorylated switch for epigenetic gene activation in non-hormonal cancer cells, and modify tumor stroma, and immune cell content.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> LSD1 is a histone 3 (H3) demethylase that can either repress or activate gene expression. We discover here that the so far enigmatic balance between these two activities in non-hormonal cancer cells is regulated by phosphorylation of serine 166 (S166) on LSD1. SET-mediated Protein Phosphatase 2A (PP2A) inhibition in KRAS mutant cells promotes S166 phosphorylation. Endogenous LSD1 S166 alanine mutant (S166A) cells display H3 lysine 9 demethylation and acetylation, euchromatin, and gene activation. Mechanistically this is explained by the impaired interaction of S166A mutant LSD1 with repressor proteins SNAI2 and MYBP1. Functionally LSD1 S166A mutant cells display augmented beta1 integrin activity and stress fiber formation, and the mutant xenograft tumors have altered tumor microenvironment associated with increased macrophage recruitment. Collectively, PP2A-regulated S166 of LSD1 is a phosphorylation switch for epigenetic gene activation in non-hormonal cancer cells. Conceptually we demonstrate how dephosphorylation of one amino acid on a non-histone protein shapes chromatin landscape in cancer cells, and modify tumor stroma, and immune cell content.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.14.653937v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genome reorganization and its functional impact during breast cancer progression</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Reed, K. S. M., Fritz, A., Greenyer, H., Heselmeyer-Haddad, K., Frietze, S., Stein, J., Stein, G., Misteli, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that chromatin structure and epigenetic parameters of genome architecture changes during cancer progression at a fine scale and genome-wide, and show a functionally relevant connection between gene regulation and genome remodeling at many key genes.</p><p style="font-size: 16px; text-align: justify;">Background: Cancer cells undergo widespread changes in epigenetic patterns that mediate cancer compromised gene expression programs during cancer progression. However, the alterations in higher-order genome organization in which these changes occur and their functional implications are less well understood. To explore how chromatin structure and epigenetic parameters of genome architecture changes during cancer progression at a fine scale and genome-wide, we generated high-resolution Micro-C contact maps in non-malignant, pre-cancerous, and metastatic MCF10 breast cancer epithelial cells. Results: We profiled progression-associated reorganization of chromatin compartments, topologically associated domains (TADs), and chromatin loops, and also identified invariable chromatin features. We find large-scale compartmental shifts occur predominantly in early stages of cancer development, with more fine-scale structural changes in TADs and looping accumulating during the later transition to metastasis. We related these structural features to changes in gene expression, histone marks, and potential enhancers and found a large portion of differentially expressed genes physically connected to distal regulatory elements. While changes in chromatin loops were relatively rare during progression, differential loops were enriched for progression-associated genes, including those involved in proliferation, angiogenesis, and differentiation. Changes in either enhancer-promoter contacts or distal enhancer activity were accompanied by differential gene regulation, suggesting that changes in chromatin contacts are not necessary but can be sufficient for gene regulation. Conclusions: Together, our results demonstrate a functionally relevant connection between gene regulation and genome remodeling at many key genes during cancer progression.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.14.654144v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genome complexity, not ploidy, dictates long-read variant-calling accuracy</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fukasawa, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that genotyping accuracy for long-read variant analysis is significantly reduced with increasing ploidy level and identify a critical mechanism impacting variant discovery: structural variations between the reference and sample genomes, particularly those containing repetitive elements, induce spurious read mapping, which leads to false variant calls.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurate characterization of genetic variation is fundamental to genomics. While long-read sequencing technologies promise to resolve complex genomic regions and improve variant detection, their application in polyploid and complex genomes remains challenging. Here, we systematically investigate the factors influencing variant calling accuracy using long reads. Using human trio data with known variants to simulate variable ploidy levels (diploid, tetraploid, hexaploid), we demonstrate that while variant sites can often be identified accurately, genotyping accuracy significantly decreases with increasing ploidy due to allelic dosage uncertainty. This highlights a specific challenge in assigning correct allele counts in polyploids even with high depth, separate from the initial variant discovery. We then assessed variant detection performance in genomes with varying complexity: the relatively simple diploid Fragaria vesca, the tetraploid Solanum tuberosum, and the highly repetitive diploid Zea mays. Our results reveal that overall variant calling accuracy correlates more strongly with inherent genome complexity (e.g., repeat content) than with ploidy level alone. Furthermore, we identify a critical mechanism impacting variant discovery: structural variations between the reference and sample genomes, particularly those containing repetitive elements, induce spurious read mapping. This leads to false variant calls, constituting a distinct and more dominant source of error than allelic-dosage uncertainty. Our findings underscore the multifaceted challenges in long-read variant analysis and highlight the need for ploidy-aware genotypers, complexity-informed variant callers, and bias-aware mapping strategies to fully realize the potential of long reads in diverse organisms.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.14.653922v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Hi-Compass resolves cell-type chromatin interactions by single-cell and spatial ATAC-seq data across biological scales</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sun, Y.-C., Jiang, W.-J., Cai, K.-W., Wu, H.-J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present Hi-Compass, a generalizable deep learning model that accurately predicts chromatin organization across diverse biological contexts, from bulk to single-cell samples.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Computational prediction of three-dimensional (3D) genome organization provides an alternative approach to overcome the cost and technical limitations of Hi-C experiments. However, current Hi-C prediction models are constrained by their narrow applicability to studying the impact of genetic variation on genome folding in specific cell lines, significantly restricting their biological utility. We present Hi-Compass, a generalizable deep learning model that accurately predicts chromatin organization across diverse biological contexts, from bulk to single-cell samples. Hi-Compass outperforms existing methods in prediction accuracy and is generalizable to unseen cell types through chromatin accessibility data, enabling broad applications in single cell omics. Hi-Compass successfully resolves cell-type-specific 3D genome architectures in complex biological scenarios, including immune cell states, organ heterogeneity, and tissue spatial organization. Furthermore, Hi-Compass enables integrative analysis of single-cell multiome data, linking chromatin interaction dynamics to gene expression changes across cell clusters, and mapping disease variants to pathogenic genes. Hi-Compass also extends to spatial multi-omics data, generating spatially resolved Hi-C maps that reveal domain-specific chromatin interactions linked to spatial gene expression patterns.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.14.654019v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Deep Learning approach for time-consistent cell cycle phase prediction from microscopy data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bonte, T., Pourcelot, O., Safieddine, A., Slimani, F., Mueller, F., Weil, D., Bertrand, E., Walter, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose to infer the cell cycle phase from a widely used fluorescent reporter: SiR-DNA.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The cell cycle consists of four phases and impacts most cellular processes. In imaging assays, the cycle phase can be identified using dedicated cell-cycle markers. However, such markers occupy fluorescent channels that may be needed for other reporters. Here, we propose to address this limitation by inferring the phase from a widely used fluorescent reporter: SiR-DNA. Our method is based on a variational auto-encoder, enhanced with two auxiliary tasks: predicting the intensity of phase-specific markers and enforcing the latent space temporal consistency. Our model is freely available, along with a new dataset comprising over 600,000 annotated HeLa Kyoto nuclear images.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.16.654306v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Chromosomal fusions and subsequent rearrangements shaped octopus genomes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yoshida, M.-a., Toth, E., Kon-Nanjo, K., Kon, T., Hirota, K., Toyoda, A., Toh, H., Miyazawa, H., Terauchi, M., Noguchi, H., Setiamarga, D. H., Simakov, O.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We provide a draft genome assembly of the vampire squid Vampyroteuthis sp., the largest cephalopod genome sequenced to-date (over 10 gigabasepairs), which shows partial retention of the chromosomal complement of Decapodiformes, indicating its more</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Why some animal groups retain ancestral chromosomal complements while others change significantly is a fundamental question in evolutionary genomics. Few systems exist where accumulations of chromosomal changes can be studied in the context of morphological innovation. In coleoid cephalopods (octopus, squid, cuttlefish), an ancient coleoid chromosomal rearrangement event (ACCRE) has led to a substantial increase in the chromosome number and a new set of chromosomal homologies. Compared to typical molluscan or bilaterian genomes, ACCRE has enabled the origin of many novel regulatory regions in coleoid cephalopods. However, the discrepancies between extant octopodiform (octopus, ~30 chromosomes) and decapodiform (squid and cuttlefish, ~46 chromosomes) karyotypes and the direction of these evolutionary changes remain unexplained. Here we provide a draft genome assembly of the vampire squid Vampyroteuthis sp., the largest cephalopod genome sequenced to-date (over 10 gigabasepairs). Through syntenic comparisons, we infer that this basally branching octopodiform species shows partial retention of the chromosomal complement of Decapodiformes, indicating its more ancestrality, while at the same time also suggest the derived nature of the octopod karyotype. Together with the analysis of a new chromosome-level assembly of the pelagic octopod Argonauta hians, we identified irreversible chromosomal fusion-with-mixing events followed by inter-chromosomal translocations in octopods after their separation from the vampire squid lineage. This secondary reduction and mixing within octopod chromosomes has enabled the origin of a more entangled genomic configuration, shedding light onto the early evolutionary transitions within the clade. Our results offer broader insights into general patterns of chromosomal evolution following large-scale rearrangement events in animal genomes, and clarify the evolutionary trajectory leading to present-day coleoid diversity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.16.652989v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Visual analytics framework for survival analysis and biomarker discovery</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kokosar, J., Turkay, C., Avsec, L., Stajdohar, M., Zupan, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a visual analytics methodology for survival analysis, and propose a framework that defines a reusable set of visualization and modeling components to support exploratory and hypothesis-driven biomarker discovery.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We introduce a visual analytics methodology for survival analysis, and propose a framework that defines a reusable set of visualization and modeling components to support exploratory and hypothesis-driven biomarker discovery. Survival analysis--essential in biomedicine--evaluates patients survival rates and the onset of medically relevant events, given their clinical and genetic profiles and genetic predispositions. Existing approaches often require programming expertise or rely on inflexible analysis pipelines, limiting their usability among biomedical researchers. The lack of advanced, user-friendly tools hinders problem solving, limits accessibility for biomedical researchers, and restricts interactive data exploration. Our methodology emphasizes functionality-driven design and modularity, akin to combining LEGO bricks to build tailored visual workflows. We (1) define a minimal set of reusable visualization and modeling components that support common survival analysis tasks, (2) implement interactive visualizations for discovering survival cohorts and their characteristic features, and (3) demonstrate integration within an existing visual analytics platform. We implemented the methodology as an open-source add-on to Orange Data Mining and validated it through use cases ranging from Kaplan-Meier estimation to biomarker discovery. The resulting framework illustrates how methodological design can drive intuitive, transparent, and effective survival analysis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.14.654136v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Sequestration-Based Neural Networks That Operate Out of Equilibrium</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nakamura, E., Britto Bisso, F., Gispert, I., Moghimianavval, H., Okuda, S., Przybyszewska-Podstawka, A., Chisholm, S., Hemanth Kumar, V. S., Perez Medina, V., Wu, M.-R., Cuba Samaniego, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose and experimentally validate a molecular sequestration-based neural network based on the molecular sequestring reaction, which overcomes the DNA-based system's limitation and behaves as a linear classifier even before it reaches its steady state (or, equivalently, out-of-equilibrium).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Classification of high-dimensional information is a ubiquitous computing paradigm across diverse biological systems, including organs such as the brain, down to signaling between individual cells. Inspired by the success of artificial neural networks in machine learning, the idea of engineering genetic circuits that operate as neural networks emerges as a strategy to expand the classification capabilities of living systems. In this work, we design these biomolecular neural networks (BNNs) based on the molecular sequestration reaction, and experimentally characterize their behavior as linear classifiers for increasing levels of complexity. Initially, we demonstrate that a static, DNA-based system can effectively prototype a linear classifier, though we also identify its limitations to easily tune the slope of the decision boundary it generates. We then propose and experimentally validate a BNN at the protein level using a cell-free transcription-translation (TXTL) system, which overcomes the DNA-based system's limitation and behaves as a linear classifier even before it reaches its steady state (or, equivalently, out-of-equilibrium). Ultimately, we test a CRISPR-based design and its out-of-equilibrium behavior in a biological context by successfully constructing a linear classifier within mammalian cells. Overall, by leveraging mathematical modeling and experimental automation, we establish molecular sequestration as a universal scheme for implementing neural networks within living systems, paving the way for transformative advances in synthetic biology and programmable biocomputing systems.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.16.654484v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ProtFun: A Protein Function Prediction Model Using Graph Attention Networks with a Protein Large Language Model</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Talo, M., Bozdag, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a multi-modal deep learning architecture called ProtFun to predict protein functions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding protein functions facilitates the identification of the underlying causes of many diseases and guides the research for discovering new therapeutic targets and medications. With the advancement of high throughput technologies, obtaining novel protein sequences has been a routine process. However, determining protein functions experimentally is cost- and labor-prohibitive. Therefore, it is crucial to develop computational methods for automatic protein function prediction. In this study, we propose a multi-modal deep learning architecture called ProtFun to predict protein functions. ProtFun integrates protein large language model (LLM) embeddings as node features in a protein family network. Employing graph attention networks (GAT) on this protein family network, ProtFun learns protein embeddings, which are integrated with protein signature representations from InterPro to train a protein function prediction model. We evaluated our architecture using textcolor{red}{three} benchmark datasets. Our results showed that our proposed approach outperformed current state-of-the-art methods for most cases. An ablation study also highlighted the importance of different components of ProtFun. The data and source code of ProtFun is available at https://github.com/bozdaglab/ProtFun under Creative Commons Attribution Non Commercial 4.0 International Public License.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.13.653854v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A novel bioinformatics approach reveals key transcription factors regulating type 1 diabetes-associated transcriptomes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sheikh, A., Parvin, S., Dipto, S., Hossain, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a novel bioinformatics pipeline that reveals key transcription factors (TFs) regulating type 1 diabetes transcriptomes by combining automated Python workflows, differential expression analysis, motif enrichment, and genome-wide TF abundance profiling, a rare but powerful strategy that deepens our understanding of disease mechanisms</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We developed a novel bioinformatics pipeline that reveals key transcription factors (TFs) regulating type 1 diabetes transcriptomes by combining automated Python workflows, DESeq2-based differential expression analysis, motif enrichment, and genome-wide TF abundance profiling, a rare but powerful strategy that deepens our understanding of disease mechanisms. This approach not only uncovers TFs driving pathology but also quantifies TF abundance across multiple genomic loci, enabling rapid and precise monitoring of their occupancy. Analyzing 21 RNA sequencing datasets, including 13 from early-stage T1D patients and 8 matched controls, we identified nearly 6 000 differentially expressed genes; 1 900 met strict significance and fold-change criteria and 211 are previously uncharacterized transcripts that distinguish T1D from healthy samples. Pathway analysis highlighted disruptions in beta-cell signaling, ALK-linked drug responses, neurodegenerative processes, and cytoskeletal organization. Upstream motif analysis revealed enrichment of Myc/Max, AP-1, SP-1, TATA-box, and NF-{kappa}B binding sites in upregulated genes, confirming their central role in the T1D transcriptome. By placing TFs at the core of our discovery platform, this work uncovers novel molecular drivers of T1D and identifies actionable biomarkers and therapeutic targets for personalized treatment.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.13.653885v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

