<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Sunday, 13 July 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Machine learning-based prediction of antimicrobial resistance and identification of AMR-related SNPs in Mycobacterium tuberculosis.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yi Xu, Ying Mao, Xiaoting Hua, Yan Jiang, Yi Zou, Zhichao Wang, Zubi Liu, Hongrui Zhang, Lingling Lu, Yunsong Yu</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a new method for studying the genomic mechanisms of drug resistance in tuberculosis using whole-genome sequencing data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mycobacterium tuberculosis (MTB) is a human-specific pathogen that primarily infects humans, causing tuberculosis (TB). Antimicrobial resistance (AMR) in MTB presents a formidable challenge to global health. The employment of machine learning on whole-genome sequencing data (WGS) presents significant potential for uncovering the genomic mechanisms underlying drug resistance in MTB.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40646483/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">LegionProfiler: A computational tool for the identification of virulence factors and classification of Legionella pneumophila serogroup 1 isolates.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Oluwafemi A Sarumi, Wilhelm Bertrams, Oliver Schwengers, Jan-Paul Herrmann, Torsten Hain, Laurine Kieper, Markus Petzold, Alexander Goesmann, Bernd T Schmeck, Dominik Heider</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed LegionProfiler, a computational tool that swiftly identifies virulence factor protein domains within genome assemblies of Legionella pneumophila serogroup 1 isolates and classifies them into high- or low-virulence groups.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Legionella pneumophila has significantly contributed to multiple cases of pneumonia with a high rate of mortality globally. Its ability to exploit host mechanisms through several expressed virulence factors poses challenges for diagnosis, treatment, and outbreak control. To address this, we developed LegionProfiler, a computational tool that swiftly identifies virulence factor protein domains within genome assemblies of Legionella pneumophila serogroup 1 isolates and classifies them into high- or low-virulence groups. LegionProfiler automates the probing of genome assemblies for virulence-associated protein domains and determines the isolate's potential to cause severe pneumonia infection. The LegionProfiler workflow is made available through a user-friendly interface to enhance technical control of infectious sources and adds important insights to the general epidemiology of clinical isolates. It could also support the development of targeted therapeutic strategies that will improve patient treatment.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40650928/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Closing the loop: Teaching single-cell foundation models to learn from perturbations</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pershad, Y., Nandi, T. N., Van Amburg, J. C., Parker, A. C., Ostrowski, L. H., Giannini, H. K., Ong, D. A., Heimlich, J. B., Obeng, E., Ericson, K., Agarwal, A., Madduri, R. K., Bick, A. G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a closed-loop framework that extends single-cell foundation models by incorporating perturbation data during model fine-tuning.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The application of transfer learning models to large scale single-cell datasets has enabled the development of single-cell foundation models (scFMs) that can predict cellular responses to perturbations in silico. Although these predictions can be experimentally tested, current scFMs are unable to close the loop and learn from these experiments to create better predictions. Here, we introduce a closed-loop framework that extends the scFM by incorporating perturbation data during model fine-tuning. Our closed-loop model improves prediction accuracy, increasing positive predictive value in the setting of T-cell activation three-fold. We applied this model to RUNX1-familial platelet disorder, a rare pediatric blood disorder and identified two therapeutic targets (mTOR and CD74-MIF signaling axis) and two novel pathways (protein kinase C and phosphoinositide 3-kinase). This work establishes that iterative incorporation of experimental data to foundation models enhances biological predictions, representing a crucial step toward realizing the promise of "virtual cell" models for biomedical discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.08.663754v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">easyClock: A User-Friendly Desktop Application for Circadian Rhythm Analysis and Visualization</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wu, B., Ja, W. W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present easyClock, an intuitive and interactive application designed to streamline circadian rhythm analysis and visualization.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Circadian rhythms regulate a wide range of biological processes, and their precise characterization is essential for understanding behavioral and physiological fluctuations. However, existing tools to analyze circadian data often require coding expertise or rely on specific data acquisition software, limiting their general applicability. Here, we present easyClock, an intuitive and interactive application designed to streamline circadian rhythm analysis and visualization. The easyClock application enables simultaneous processing of multiple files, allowing users to preview, batch-analyze, and plot diverse sets of time series generated from high-throughput experiments. To enhance data analysis efficiency and provide comparable results, this application integrates four different analysis methods for handling data with various waveforms. Results are easily viewed and exported for any selected range of data. This free-to-use application is open-source and can be deployed on macOS computers by users without coding experience.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.08.663749v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Microenvironmental information significantly improves the recognition of cell types in human lung cancer patients</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Toth, T., Reiniger, L., Migh, E., Bauer, D., Moldvay, J., Szallasi, Z., Horvath, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We used a fisheye transformation technique to improve single-cell phenotypic classification in histopathological tissue sections, with potential implications for more robust and clinically applicable AI-driven diagnostics.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurate single-cell phenotypic classification in histopathological tissue sections is essential for understanding tumor behavior, identifying potential therapeutic targets, and improving prognostic assessments in cancer research. In this study, we applied deep learning to classify lung cancer cell phenotypes in hematoxylin and eosin-stained tissue sections. Using 11 whole slide images from 11 patients, we annotated nearly 20,000 cells into seven distinct phenotypes for training and validation. We used a fisheye transformation technique, which modifies images to mimic fisheye camera effects in order to incorporate cellular microenvironment information to enhance deep learning models. We evaluated its effectiveness on lung cancer tissue sections, optimizing transformation parameters and assessing classification performance through multiple cross-validation strategies. Our results demonstrate that the transformation significantly improves classification accuracy, approaching human level performance, particularly for phenotypes that rely on subtle morphological differences. The approach enhances model generalizability across patient samples, highlighting the importance of integrating spatial context in computational pathology. These findings suggest that incorporating adaptive image transformations can significantly improve automated histopathological analysis, with potential implications for more robust and clinically applicable AI-driven diagnostics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.09.663829v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">High Throughput Information Extraction of Printed Specimen Labels from Large-Scale Digitization of Entomological Collections using a Semi-Automated Pipeline</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Belot, M., Tuberosa, J., Preuss, L., Svezhentseva, O., Claessen, M., Bölling, C., Schuster, F., Leger, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a semi-automated pipeline, ELIE (Entomological Label Information Extraction), which combines computer vision, convolutional neural networks (CNNs), optical character recognition (OCR), and clustering algorithms to streamline label data extraction.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> 1. Natural history museums curate billions of insect specimens, forming a vast but underutilized resource for biodiversity research. While digitization efforts have increased the availability of high-resolution specimen images, extracting metadata from labels remains a major bottleneck, often requiring manual transcription. 2. We developed a semi-automated pipeline, ELIE (Entomological Label Information Extraction), which combines computer vision, convolutional neural networks (CNNs), optical character recognition (OCR), and clustering algorithms to streamline label data extraction. Our pipeline operates in three stages: (1) label detection and classification (printed vs. handwritten), (2) OCR-based text extraction from printed labels using Tesseract or Google Vision, and (3) clustering of extracted text for human validation of outliers. 3. Benchmarking on diverse datasets from multiple museum collections showed that our approach successfully extracted and clustered up to 98% of printed labels, significantly reducing manual effort. The pipeline improves efficiency in digitization workflows while maintaining high accuracy in label data capture. 4. Our approach demonstrates the potential of integrating AI-driven methods with human validation to accelerate specimen digitization. By reducing manual transcription workload and enabling scalable extraction of insect label metadata, it unlocks biodiversity data for research in ecology, systematics, and conservation globally.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.09.663724v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identification of novel protective loci for executive function using the trail making test part B in the Long Life Family Study</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, L., Tanner, K., Andersen, S. L., Cosentino, S., Moghaddam, V. A., Daw, E. W., Anema, J. A., Lin, S. J., Acharya, S., Province, M., Wojczynski, M. K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A family-based study design to identify the genetic variants associated with the TMT-B incorporating both genome-wide linkage analysis (GWLS) and whole genome sequencing (WGS).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The Trail Making Test (TMT) Part B (TMT-B), a well-established assessment of cognitive function, is a frequent component of diagnostic assessments for Mild Cognitive Impairment and dementia in older adults. Identifying the genetic variants associated with the TMT-B will not only gain insights of genetic determinants of cognitive function, but also the molecular mechanisms for dementia. Published GWAS to date for TMT-B suffer from relatively low power due to the use of population level data and imputation methods. To address these deficits, we used a family-based study design to identify the genetic variants associated with the TMT-B incorporating both genome-wide linkage analysis (GWLS) and whole genome sequencing (WGS). As such, we examined the sequenced genetic determinants of TMT-B using GWLS in over 2000 participants from Long Life Family Study (LLFS). In GWLS, the estimated heritability of TMT-B was 0.29. We detected one significant linkage peak at 15q25 (LOD>3.0). Statistical fine-mapping nominated five variants including three SNPs (NTRK3-rs74031103, protective CEMIP-rs2271159, and protective AGBL1-rs4134376) and two INDELs (protective KLHL25-15:85882445:IND, and protective CEMIP-15:80893381:IND) contributing to the linkage peak. Four out of these five variants are protective for TMT-B. The rs2271159 SNP influences CEMIP expression in cerebellum and hippocampus, while the 15:80893381:IND modulates CEMIP expression in blood. Additionally, the variant rs4134376 is a basal ganglia-specific eQTL for AGBL1. In conclusion, we utilized GWLS, leveraged multi-omics data (whole genome sequence genomic data, transcriptomic data, and lipidomic data), and identified novel protective variants and genes for TMT-B performance.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.08.663708v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Impacts of projected elevated temperature on estimated chilling accumulation and bud break of rabbiteye blueberries</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lawas, L. M., Rossi, G., Salazar Gutierrez, M., Tian, D., Leisner, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the projected impacts of mid 21st century warming on chilling accumulation and phenological development in rabbiteye blueberry (Vaccinium virgatum) cultivars Krewer and Titan.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Climate change poses a significant threat to perennial fruit crops, including blueberries, by altering temperature dependent developmental processes such as dormancy and bud break. This study investigates the projected impacts of mid 21st century warming on chilling accumulation and phenological development in rabbiteye blueberry (Vaccinium virgatum) cultivars Krewer and Titan. Using downscaled climate projections from 20 global climate models under the RCP 8.5 scenario, we estimated future temperature trends for a key southeastern U.S. blueberry production region. We quantified changes in chilling hours, chilling portions, and freeze event probabilities between historical (1981 to 2000) and projected (2041 to 2070) periods. Results indicate a substantial reduction in chilling accumulation, chilling hours decreased by 66.6% and chilling portions by nearly half, alongside a delayed onset and shortened duration of chilling periods. Despite warming, freezing events are still expected during critical phenological windows, potentially increasing frost damage risk. Growth chamber experiments simulating ambient and elevated temperature regimes revealed significant cultivar-specific shifts in the timing and rate of bud break and bloom stages. Logistic models developed from these data provide predictive tools for assessing phenological responses under future climates. These findings highlight the vulnerability of rabbiteye blueberry to climate-induced dormancy disruption and underscore the need for adaptive management strategies and cultivar selection to sustain productivity in a warming climate.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.08.663765v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Structural basis for size-selective perception of chitin in plants</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gysel, K., Hansen, S. B., Rübsam, H., Alsarraf, H. M. A. B., Madland, E., Cheng, J. X. J., Baadegaard, C., Poulsen, E. C., Vinther, M., Fort, S., Stougaard, J., Andersen, K. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a mechanistic model for chitin perception and the formation of receptor complexes critical for immune signaling and identify a hallmark bridge domain characteristic for the CHIP receptor class.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Plants detect microbes through pattern recognition receptors that perceive conserved microbial surface motifs known as microbe-associated molecular patterns (MAMPs). LysM receptors recognize and mediate downstream responses to chitinous MAMPs. Here, we elucidate the mechanism for the specific recognition of long-chain chitin oligomers and identify a hallmark bridge domain characteristic for the CHIP receptor class. Structural analysis of receptor-ligand complexes, biochemistry, and in planta functional studies using inhibitory nanobodies reveal the mechanism of size-selective, high affinity chitin perception in Lotus japonicus and Medicago truncatula. Additionally, we identify CERKs as low-affinity, yet essential co-receptors and propose a mechanistic model for a ligand-induced core signaling heterocomplex. Our findings provide mechanistic insights into plant chitin perception and the formation of receptor complexes critical for immune signaling.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.09.663820v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Overcoming extensive redundancy in the arabidopsis TREHALOSE-6-PHOSPHATE PHOSPHATASE gene family reveals connections to development</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Skopelitis, T., Swentowsky, K., Goldshmidt, A., Feil, R., Lunn, J., Jackson, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the trehalose-6-phosphate PHOSPHATASE (TPP) genes control shoot branching, sugar metabolism, iron homeostasis and photosynthesis in arabidopsis, and support a role for TPP-mediated Tre6P signaling in coordinating developmental and physiological</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Arabidopsis encodes ten TREHALOSE-6-PHOSPHATE PHOSPHATASE (TPP) genes, homologous to maize RAMOSA3 (I), which controls shoot branching. To explore the roles of the arabidopsis TPPs, we analyzed their expression in shoot apices and found distinct spatial patterns, including TPPI and TPPJ expressed in shoot meristem boundaries, reminiscent of RA3 expression. Single and double TPP mutants lacked dramatic phenotypes, however a CRISPR-Cas9 knockout of all ten TPP genes resulted in increased branching, mirroring ra3 mutants in maize, as well as reduced size and earlier flowering. Expression of GFP-tagged TPPI under its native promoter partially complemented these defects, with protein localization in meristems, vascular tissues and in nuclei. Metabolite profiling revealed higher trehalose 6-phosphate (Tre6P), lower trehalose, and altered sugar and iron-associated metabolites. The mutants also developed chlorosis and grew poorly on low-nutrient media, linked to low iron levels, and reversible with iron supplementation. Consistent with these findings, developmental and iron-responsive genes were up-regulated in the mutants, while photosynthesis-related genes were repressed. Our findings suggest that TPP genes redundantly regulate shoot architecture, sugar metabolism, iron homeostasis and photosynthesis in arabidopsis, and support a role for TPP-mediated Tre6P signaling in coordinating developmental and physiological pathways.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.09.663849v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Glycosylation-dependent sorting of an Arabinogalactan protein SLEEPING BEAUTY mediates apical tip growth and osmosensing in Physcomitrium patens</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yu, C.-Y., Maharjan, M., Liu, C.-H., Teh, O.-k.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose that a wall-associated Arabinogalactan proteins (AGP) from the bryophyte Physcomitrium patens is intrinsically disordered and rigidifies the cell wall to facilitate ordered depositions of cellulose microfibril and osmosensing.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Plant terrestrialization is a key event that profoundly transforms the Earth surface, during which early land plants have acquired cell walls with sophisticated modifications to overcome osmotic stress. Here, we describe the cell wall rigidifying function of SLEEPING BEAUTY (SB), a wall-associated Arabinogalactan proteins (AGP) from the bryophyte Physcomitrium patens. The highly-glycosylated SB is intrinsically disordered and rigidifies the cell wall to facilitate ordered depositions of cellulose microfibril, which when affected compromised the cell wall integrity and rendered a soften cell wall. We further pinpointed critical glycosylation sites on the SB that contributes to its disordered propensity and osmosensing function. Pharmacological inhibition of endocytic trafficking pathways showed that SB is actively endocytosed and trafficked to tonoplast, which was enhanced under mild osmotic stress. Importantly, sorting of SB to tonoplast requires functional glycosylations at Prolines 55, 92 and 94. Finally, we showed that the wall-rigidifying function of SB is conserved in the flowering plant Arabidopsis, thus highlighting a common cell wall organization across land plants. We propose that the highly disordered AGP may act as a cell wall scaffold protein to mediate wall materials biosynthesis/deposition and play a role in apical tip growth as well as extracellular osmosensing.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.08.663806v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Cytosolic water potential as a mechanistic driver of leaf airspace unsaturation and non-stomatal control of transpiration</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Marquez, D. A., Cernusak, L. A., Busch, F. A., Farquhar, G. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that low plasma membrane conductance (Lp) is the primary physiological signal regulating water loss in mesophyll cells, and propose a predictive, physiologically grounded model linking Lp to cytosolic water potential ({psi}cy).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mesophyll cells exhibit a previously underappreciated capacity to regulate water loss via low plasma membrane conductance (Lp), offering a non-stomatal mechanism for transpiration control. However, the structural basis and regulation of Lp remain poorly understood, limiting its integration into predictive models. In this study, we show that Lp responds dynamically to changes in cytosolic water potential ({psi}cy), decreasing as {psi}cy approaches the turgor loss point. This identifies {psi}cy as the primary physiological signal regulating Lp. We introduce a predictive, physiologically grounded model linking Lp to {psi}cy. Our model establishes a mechanistic connection between internal water status, leaf hydraulics, substomatal cavity unsaturation, and gas exchange. This framework opens new avenues for understanding and modelling plant water use under stress.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.08.663815v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Deep Learning for Sorghum Yield Forecasting using Uncrewed Aerial Systems and Lab-Derived Imagery</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bari, M. A., Bakshi, A., Choton, J. C. A., Pramanik, S., Witt, T. D., Caragea, D., Bean, S., Jagadish, K., Felderhoff, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> This research trained YOLO and the Faster R-CNN (Detectron2) models to harness yield attributes from UAS field and lab images to improve yield prediction performance.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The AI revolution, advanced Graphics Processing Units (GPUs), and open-source platforms have enabled Machine Learning (ML) and Deep Learning (DL) algorithms to rapidly and accurately extract phenotypic features from Uncrewed Aerial System (UAS)-derived imagery. Such advancement leads to phenotypic digitization and sorghum yield forecasting. Yield analytics are critical for breeding programs to assess the genetics and breeding potential of genotypes to enhance cultivar development. This trial followed a three-replicated Randomized Complete Block Design (RCBD) with 36 diverse sorghum genotypes in 2023 at Ashland Bottoms, Kansas. The field images were captured 6 meters above using a DJI M300 drone equipped with the P1 sensor at nadir (90 degrees) and oblique (45 degrees) angles. This research trained YOLO and the Faster R-CNN (Detectron2) models to harness yield attributes from UAS field and lab images. The YOLO models outperformed the Faster R-CNN model in detecting sorghum panicles, achieving a mean average precision at 50% Intersection over Union (IoU) ranging from 0.92 to 0.98, compared to 0.61 to 0.89. Panicle detection from field imagery correlated at 0.86 with ground truth. Lab imagery analyses measured panicle size, seed counts, and seed area with correlation coefficients of 0.71, 0.95, and 0.25, respectively. Three machine learning models: Support Vector Regression (SVR), Decision Tree Regression (DTR), and Random Forest Regression (RFR) are used to predict yield with correlation coefficients of 0.58, 0.76, and 0.70, respectively. We observed that YOLO models are well-suited for extracting yield-attributing traits from images, which are then incorporated into ML regression models to improve yield prediction performance.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.11.663520v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Modulating Cardiac-Gut Microbiome Interaction Post-Myocardial Infarction with Engineered Bacteria</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Madaan, T., Nieman, M., Rakheja, T., Siddiqui, N., Gherardini, D., Sertorio, M., Kamble, N., Thomas, S., Hartman, A., Koch, S., Knipper, L., Marino, V. A., Sadayappan, S., Lorenz, J. N., Konhilas, J. P., Kotagiri, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We demonstrate that acute myocardial infarction results in the translocation and colonization of live, intact gut bacteria in the MI heart, a novel aspect of the heart-gut axis, and highlight the potential of microbiome-based interventions in MI management.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The gut microbiome plays a critical role in the pathophysiology of acute myocardial infarction (MI). MI events significantly impact intestinal integrity which results in leakage of bacterial products into the systemic circulation. We demonstrate that MI not only compromises intestinal integrity, leading to systemic leakage of bacterial products like LPS, but also results in the translocation and colonization of live, intact gut bacteria in the MI heart, a novel aspect of the heart-gut axis. Our initial findings with natural murine gut microbiome were substantiated using orally administered E. coli Nissle 1917 (EcN), as a tracer bacterium. Furthermore, we engineered EcN to express the microbial anti-inflammatory molecule (MAM) derived from the probiotic Faecalibacterium prausnitzii. Treatment with this engineered strain, EcN-MAM, led to significantly improved survival and cardiac function in MI mice. This was attributed to enhanced gut barrier integrity, resulting in reduced systemic bacterial permeation and subsequent inflammation. These findings shed light on a previously unrecognized dimension of the heart-gut axis and highlight the potential of microbiome-based interventions in MI management.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.10.664025v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

