<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Tuesday, 07 October 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Learning Protein Representations with Conformational Dynamics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kalifa, D., Horvitz, E., Radinsky, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present DynamicsPLM, a protein language model conditioned on ensembles of computationally generated conformations to derive state-aware representations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Proteins change shape as they work, and these changing states control whether binding sites are exposed, signals are relayed, and catalysis proceeds. Most protein language models pair a sequence with a single structural snapshot, which can miss state-dependent features central to interaction, localization, and enzyme activity. Studies also indicate that many proteins assume multiple, functionally relevant shapes, motivating approaches that learn from this variability. Here we present DynamicsPLM, a protein language model conditioned on ensembles of computationally generated conformations to derive state-aware representations. DynamicsPLM improves predictive performance across protein-protein interaction, subcellular localization, enzyme classification, and metal-ion binding. On a widely used protein-protein interaction benchmark, it achieves a four-point accuracy gain over the strongest baseline. On a curated test set enriched for proteins with multiple conformational states, the margin increases to eleven points. These findings argue for a shift from static to dynamics-aware modeling, in which conformational variability is treated as informative. By elevating conformational state to a central element of machine learning in protein biology, this work advances modeling toward mechanisms that better reflect how proteins operate in cells and provides a route to actionable hypotheses about when and how binding, signaling, and catalysis occur.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.06.680789v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SHARK: a specialized host for assembling R6K plasmids</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Joshi, S. H.-N., Jenkins, C., Ulaeto, D., Gorochowski, T. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a series of R6K plasmid maintenance strains from DH10B derivative Marionette-Clo and show that they are more efficient than commercial Pir strains for large and complex cloning reactions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> R6K plasmids are commonly used for a wide range of genome engineering applications due to their ability to support transient delivery of genetic cargoes in many hosts. The maintenance of R6K plasmids requires specific strains. Unfortunately, many of these have obscure backgrounds, limited availability and were not built for efficient cloning. To address this issue, we present the construction and characterisation of a series of Pir E. coli strains called SHARK that are built from the DH10B derivative, Marionette-Clo. All SHARK strains have a genome encoded Pir gene for stable R6K plasmid maintenance and a {lambda}CI gene for tight unconditional repression of specific genes on plasmids. We show that SHARK strains are >100-fold more efficient than a commercial Pir strain for large and complex cloning reactions. SHARK is intended to help facilitate the cloning of large and complex R6K plasmids for challenging genome engineering projects, with all strains and genetic tools for their assembly being made publicly available.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.06.680659v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ceQTL: A co-expression QTL model to detect a variant that affects transcription factor binding and its target regulation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, P., Liu, L., Wang, J., Yang, P., Sun, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a new method called differential co-expression QTL (ceQTL) among different alleles using Chow statistics to specifically detect eQTLs that are modulated by a particular TF.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Expression quantitative trait locus (eQTL) mapping is used to identify a functional link between a genomic variant, such as single nucleotide polymorphism (SNP), and gene expression (often close-by pair for cis-eQTL) by linear regression, commonly done when matching genotype and expression data from same individuals are available. Millions of significant eQTLs have been reported by both individual studies and coordinated large consortiums such as Genotype-Tissue Expression project (GTEx). A significant eQTL association does not establish a causal relationship or provide any underlying mechanism so further investigation is needed to understand how a SNP impacts gene expression. One of the plausible explanations for eQTL is that a genomic variant affects transcription factor (TF) binding and thus impacts its regulation on its target genes (TGs). However, data-driven or formal statistical methods to prove that hypothesis are still lacking. To address the gap, we propose a new method called differential co-expression QTL (ceQTL) among different alleles using Chow statistics to specifically detect eQTLs that are modulated by a particular TF. We start with building a trio of TF, its TG, and related SNP and then test the significant coefficient difference among different levels of SNP in terms of TF and TG correlation. We applied this ceQTL model to simulated data and the lung tissue datasets from the GTEx project. The simulated data results showed that the model was robust to detect true ceQTLs at variable sample sizes and different minor allele frequencies as measured by Area Under the Curve (AUC). In normal lung tissue, a small fraction of eQTLs were found to have strong ceQTLs, i.e., eQTLs where SNP affects gene expression though TF binding. Some ceQTLs may not be detected by traditional eQTL analysis. Our tool also performed a TF binding affinity analysis to add another layer of evidence for functional interpretation. Comparisons with other similar tools were also presented. In summary, ceQTL analysis provides a more interpretable and biological insight into the mechanism of eQTL, which would help us better understand how genomic variants affect phenotypes and diseases.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.06.680553v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Fully Automated, Data-Driven Approach for Dimensionality Reduction and Clustering in Single-Cell RNA-seq Analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kim, H., Ardi, F. R., Spinicci, K., Kim, J. K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a fully automated clustering approach for single-cell RNA sequencing using scLENS-a method for optimal PC selection-with scLens-a for optimal DR and a new method for data-driven DR .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell RNA sequencing (scRNA-seq) provides deep insights into cellular heterogeneity but demands robust dimensionality reduction (DR) and clustering to handle high-dimensional, noisy data. Many DR and clustering approaches rely on user-defined parameters, undermining reliability. Even automated clustering methods like ChooseR and MultiK still employ fixed principal component defaults, limiting their full automation. To overcome this limitation, we propose a fully automated clustering approach by integrating scLENS-a method for optimal PC selection-with these tools. Our fully automated approach improves clustering performance by ~14% for ChooseR and ~10% for MultiK and identifies additional cell subtypes, highlighting the advantages of adaptive, data-driven DR.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.06.680609v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Addressing multiple facets of ligand-receptor network inference including single-cell proteomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Villemin, J.-P., Giroux, P., Maillard, M., Colombo, P.-E., Larbouret, C., Colinge, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We extend the SingleCellSignalR Bioconductor package to provide an integrated framework that incorporates alternative scoring strategies and adjustable analytical depth, enabling the first direct comparison of LRI inference across single-cell proteomics and transcriptomics data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Distinct ligand-receptor interaction (LRI) inference tools often produce markedly different results, and their performance can vary considerably across datasets. Indeed, performance is influenced by differences in experimental designs and dataset-specific features, making it difficult to establish a universal LRI tool. To address this challenge, we expanded our SingleCellSignalR Bioconductor package to provide an integrated framework that incorporates alternative scoring strategies and adjustable analytical depth. We motivate this choice through the analysis of two single-cell transcriptomics datasets that exemplify contrasting experimental designs. Leveraging the new framework flexibility, we present a detailed analysis of paired single-cell proteomics and transcriptomics data, providing, to our knowledge, the first direct comparison of LRI inference across these complementary modalities at single-cell resolution. Finally, we demonstrate how the same framework seamlessly accommodates additional underexplored data types from the LRI perspective, including patient-derived mouse xenografts and bulk RNA sequencing of upstream-sorted cell populations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.05.680519v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Uncertainty-aware transcription factor activity and perturbation inference without additional training</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sugimoto, H., Tsuyuzaki, K., Zou, Z., Oki, S., Ohta, T., Kawakami, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce TFActProfiler, an integrated resource and computational framework that learns signed TF-mRNA effects by pretraining on prior TF- mRNA interaction knowledge and large RNA-seq atlases.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding how transcription factors (TFs) control gene expression is essential for deciphering cellular regulatory programs. However, estimating TF activity remains challenging. Current methods rely on either broad TF-target databases, which increase false positives, or on curated sets with limited coverage. Most methods lack uncertainty quantification. Here, we introduce TFActProfiler, an integrated resource and computational framework that learns signed TF-mRNA effects by pretraining on prior TF-mRNA interaction knowledge and large RNA-seq atlases. Given an expression profile as input, TFActProfiler infers per-sample TF activity together with a confidence score that quantifies agreement between predicted and observed mRNA levels. Across TF knockdown datasets spanning multiple human cell types, TFActProfiler consistently improves TF activity inference over approaches based on unfiltered priors or narrowly curated target sets while retaining broad TF and mRNA coverage. The learned effect map further enables training-free prediction of transcriptome responses to TF perturbations. By coupling TF activity inference with training-free perturbation modeling and confidence estimates, TFActProfiler offers a tool for dissecting regulatory programs across diverse cellular states.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.05.680506v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multimodal AI agents for capturing and sharing laboratory practice</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Skowronek, P., Nawalgaria, A., Mann, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a multimodal AI laboratory agent that captures and shares tacit experimental practice by linking written instructions with hands-on laboratory work through the analysis of video, speech, and text.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We present a multimodal AI laboratory agent that captures and shares tacit experimental practice by linking written instructions with hands-on laboratory work through the analysis of video, speech, and text. While current AI tools have proven effective in literature analysis and code generation, they do not address the critical gap between documented knowledge and implicit lab practice. Our framework bridges this divide by integrating protocol generation directly from researcher-recorded videos, systematic detection of experimental errors, and evaluation of instrument readiness by comparing current performance against historical decisions. Evaluated in mass spectrometry-based proteomics, we demonstrate that the agent can capture and share practical expertise beyond conventional documentation and identify common mistakes, although domain-specific and spatial recognition should still be improved. This agentic approach enhances reproducibility and accessibility in proteomics and provides a generalizable model for other fields where complex, hands-on procedures dominate. This study lays the groundwork for community-driven, multimodal AI systems that augment rather than replace the rigor of scientific practice.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.05.680425v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Optimal transport modeling uncovers spatial domain dynamics in spatiotemporal transcriptomics studies</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ma, W., Hou, S., Shang, L., Lu, J., Zhou, X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Spatiotemporal transcriptomics is an emerging and powerful approach that adds a temporal dimension to spatial transcriptomics, enabling the characterization of dynamic changes in tissue architecture during development or disease progression.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Spatiotemporal transcriptomics is an emerging and powerful approach that adds a temporal dimension to spatial transcriptomics, enabling the characterization of dynamic changes in tissue architecture during development or disease progression. Here, we present SpaDOT (Spatial Domain Transition detection), a computational method designed to identify spatial domains and infer their temporal dynamics across time points for spatiotemporal transcriptomics studies. SpaDOT employs a variational autoencoder (VAE) framework to capture a low-dimensional representation of the data, incorporating hidden clustering variables to define distinct spatial domains. In the process, SpaDOT integrates a Gaussian Process prior and graph neighbor information to explicitly model both global spatial continuity and local structural heterogeneity among spatial locations within each time point, while using optimal transport (OT) to derive time-varying embeddings and infer the relationships between spatial domains across time points. Through simulation and real data applications, we demonstrate the superiority of SpaDOT in spatial domain detection, latent space preservation, and domain transition tracking over time. In real data applications, SpaDOT accurately captures dynamic domain transitions, including the disappearance and re-emergence of domains due to technical variation. Notably, SpaDOT uncovers key aspects of valvulogenesis in the developing heart, revealing the splitting of a single valve structure into two distinct functional valves - the atrioventricular valve and the semilunar valve.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.06.680672v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Aardvark: Sifting through differences in a mound of variants</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Holt, J. M., Kronenberg, Z., Saunders, C. T., Dolzhenko, E., Krusche, P., Olson, N. D., Zook, J. M., Eberle, M. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Aardvark, a variant benchmarking tool that introduces the basepair score to directly compare haplotype sequences, reducing representation biases while allowing partial credit scoring.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Variant benchmarking is critical in assessing the accuracy of genomic secondary pipelines. However, traditional benchmarking tools that require exact genotype matches inject biases from variant representation and are ill-suited for tandem repeat or structural variation. We describe Aardvark, a variant benchmarking tool that introduces the basepair score to directly compare haplotype sequences, reducing representation biases while allowing for partial credit scoring. The tool also includes a traditional genotype score and supports separate or joint benchmarking of small variants, tandem repeats, and structural variants (<10 kb). Aardvark accepts standard inputs, runs {approx}16x faster than hap.py, and is freely available and open source (https://github.com/PacificBiosciences/aardvark).</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.03.680257v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MS2extract: an R Package for a Scalable LC-MS/MS Library Creation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Quiroz Moreno, C. D., Cooperstone, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We pre-sent MS2extract, an R package facilitating the user-led creation and automation of MS/MS spectral libraries, and PhenolicsDB, a phenolics-focused MS/ MS library with fragmentation patterns of 71 authentic analytical standards.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Reference MS/MS libraries are useful to aid in metabolomics compound identification. However, creating these libraries can be challenging because of the complexity of MS/MS data handling, the need for manual spectrum inspection, and requirement for custom-designed bioinformatics tools for this purpose. Although licensed and open-source alternatives to create MS/MS reference libraries exist, most of these pipelines require manual extraction of each reference spectrum, which is a time-consuming and error-prone process. Here, we pre-sent MS2extract, an R package facilitating the user-led creation and automation of MS/MS spectral libraries. The core MS2extract workflow consists of importing MS/MS raw data, detecting specific precursor ions, extracting MS/MS spectra, removing low-intensity signals, and exporting a spectral database. In addition, libraries created with MS2extract are compatible with GNPS2 batch upload to maximize MS/MS library reuse. We used MS2extract to create PhenolicsDB, a phenolics-focused MS/MS library with fragmentation patterns of 71 authentic analytical standards collected in positive and negative polarity, and multiple collision energies, with 320 MS/MS reference spectra. Finally, we employed PhenolicsDB to identify apple fruit phenolics. MS2extract and PhenolicsDB are free and publicly available for download and use, and PhenolicsDB is also available in GNPS2.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.05.680545v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Age distinguishes selection from causation in cancer genomes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cheek, D., Blohmer, M., Nowak, M. A., Antal, T., Naxerova, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A mathematical and statistical framework for identifying cancer-promoting mutations in cancer genomes and those that are not.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cancer genome sequencing efforts have revealed hundreds of genes under positive selection, many of which are now being developed as therapeutic targets. However, positively selected mutations also populate our aging tissues in the absence of cancer. For most mutations, it is currently unknown whether they are recurrently found in cancer genomes because they cause cancer or because they expand during normal tissue evolution and are passively inherited. Here, we develop a mathematical and statistical framework that distinguishes these two factors. We discover - across thousands of cancer and normal tissue genomes - that mutations that most strongly increase cancer risk are enriched in younger patients' cancers, whereas mutations that are positively selected in normal tissue without causing cancer are enriched in older patients. Focusing on a particularly data-rich cancer type, acute myeloid leukemia, we show that genetic differences between young- and adult-onset cancers can largely be explained by the cumulative effects of normal tissue evolution, contradicting the long-standing notion that childhood cancers require a distinct set of causal mutations. Our framework establishes patient age as a powerful resource for clarifying whether positively selected mutations in cancer genomes are truly disease-promoting.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.06.680730v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Differential Impact of Lymphocytes on Radiation Response in Autochthonous versus Transplant Sarcomas in Syngeneic Mice</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mowery, Y. M., Sobti, A., Bassil, A. M., Wisdom, A. J., Kent, C. L., Su, C., Himes, J. E., Xu, E. S., Williams, N. T., Luo, L., Kirsch, D. G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Autochthonous tumor models show high cure rates with combined radiotherapy (RT) and immunotherapy, while transplant tumor onset is significantly slower in immunocompetent mice.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Preclinical studies in transplant tumor models showing high cure rates with combined radiotherapy (RT) and immunotherapy have rarely translated to clinical success, and studies in autochthonous tumor models are limited. We hypothesized that lymphocytes differentially affect RT response in transplant versus autochthonous tumor models. Here, we compared tumor onset and growth delay after 0 or 20 Gy for autochthonous versus transplant soft tissue sarcomas in Rag2-/- mice lacking an adaptive immune system versus immune-intact Rag2+/- mice. While time to tumor onset did not differ between Rag2-/- and Rag2+/- mice in the autochthonous model, transplant tumor onset was significantly slower in immunocompetent mice. No transplant tumors in Rag2-/- mice or autochthonous tumors were cured by RT, whereas 30.4% of Rag2+/- mice with transplant tumors were cured. These results highlight the importance of including autochthonous tumors as complementary model systems to study the interplay between the immune system and RT response.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.05.680584v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Leveraging WNT Hyperactivation to Kill Colorectal Cancer While Rejuvenating Healthy Intestine.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Eng, G., Braverman, J., Gala, M., Yilmaz, O. H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that colorectal cancer can be caused by over-WNTing in APC-mutant tumor cells, a discovery that leads to the first cancer therapy that simultaneously enhances normal tissue function while eliminating tumors.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> WNT signaling maintains intestinal homeostasis yet drives colorectal cancer (CRC) when constitutively activated by APC mutations. We overturn the fundamental assumption that APC-mutant tumors exist at maximal WNT activation, revealing instead that cancer cells occupy a precarious "WNT-just-right" zone along a signaling continuum. This discovery exposes an unprecedented therapeutic vulnerability: while normal intestinal epithelium thrives with enhanced WNT signaling, APC-mutant tumor cells undergo apoptosis when pushed beyond their oncogenic setpoint, a phenomenon we term "over-WNTing." Through systematic organoid-based screening, we identified that WNT hyperactivation through multiple approaches: GSK3 inhibition, concentrated WNT proteins, or APC knockdown in GSK3-null backgrounds, selectively kills tumor cells by hyperactivating the driving pathway of CRC. Mechanistically, over-WNTing in APC-mutant cells triggers spillover into non-canonical planar cell polarity signaling, where RHOC upregulation induces ROCK1/2-mediated apoptosis. We demonstrate therapeutic efficacy across the neoplastic continuum, from adenomas to metastatic CRC, including patient-derived tumors, validating GSK3 inhibition with a novel nanoparticle formulation. This discovery enables the first cancer therapy that simultaneously enhances normal tissue function while eliminating tumors. "Over-WNTing" effectively treats adenomas and both mouse and patient-derived CRC, establishing a therapeutic paradigm that exploits fundamental differences in cellular WNT biology to achieve the dual benefit of eliminating cancer while promoting healthy tissue regeneration.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.05.680591v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Modulating p38MAPK recalibrates sensitivity of drug-resistant pancreatic adenoductal carcinoma cells toward chemotherapy and correlate with improved outcome in PDAC patients: Experimental and metadata evidence</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mehra, V., Gandhi, K., Das, P., Sharma, G., Bhardwaj, M., Gongora, C., Prakash, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We believe that p38 MAPK based approach is potential approach for changing the faith of PDAC patients toward chemo and immunotherapy and believed to improve PDAC burden effectively in the host.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Pancreatic cancer is one of the deadliest cancers and has very limited therapeutic options and a dismal prognosis. Among various signaling pathways which are activated during tumor development, hyperactivation of Mitogen-Activated Protein Kinase (MAPK) is responsible for high grade angiogenesis, polarization of Tumor Associated Macrophages, unfolded protein responses and exhaustion of T cells, which together contributes towards this therapeutic resistance. We therefore believe that MAPK targeting is expected to enhance sensitivity of highly resistant PDAC cells toward various cancer directed interventions. In this context, we investigated the impact of modulating p38MAPK on the sensitivity of pancreatic cancer cells towards gemcitabine. Supporting our hypothesis, our results convincely, demonstrated that indeed, p38 inhibition sensitizes both KRAS positive Panc-1 and MiaPaCa2 pancreatic carcinoma cells towards gemcitabine induced death. Interestingly p38MAPK targeting significantly reduced the cell viability, clonogenic potential of these cells and enhanced the early apoptosis. Our in-silico studies, supporting our in vitro data, potentially correlated that that high expression of p38 MAPK14 in PDAC patients is associated with poor prognosis and disease free survival. Deep miming of in silico data further demonstrated that MAPK14, in association with, hypoxia inducible factor-1 alpha and vascular endothelial growth factor signaling pathways promote angiogenic programming of PDAC which render these tumors refractory for cancer directed interventions. Based on our preliminary data, we believe that p38 MAPK based approach is potential approach for changing the faith of PDAC patients toward chemo and immunotherapy and believed to improve PDAC burden effectively in the host.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.04.680430v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AI-driven chemotoxicity prediction in colorectal cancer: impact of race, SDOH, and biological aging.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Claire Han, Christin Burd, Jesse Plascak, Fode Tounkara, Ashley Rosko, Anne Noonan, Alai Tan, Diane Von Ah, Xia Ning</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study to develop effective artificial intelligence (AI)/machine learning (ML) models, integrating racialized group, social determinants of health (SDOH), and biological aging (Levine Phenotypic Age) to predict overall, gastrointestinal (GI), and hematological chemotoxicity.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Patients with colorectal cancer (CRC) often experience chemotoxicity that impacts treatment adherence, survival, and quality of life. Early screening for chemotoxicity risk is vital, yet comprehensive predictive models are lacking. The objective of this study was to develop effective artificial intelligence (AI)/machine learning (ML) models, integrating racialized group, social determinants of health (SDOH) (Area Deprivation Index [ADI], employment status), and biological aging (Levine Phenotypic Age) to predict overall, gastrointestinal (GI), and hematological chemotoxicity.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41053663/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The Road to Precision Nanomedicine: An Insight on Drug Repurposing and Advances in Nanoformulations for Treatment of Cancer.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yasmina Elmahboub, Rofida Albash, Sadek Ahmed, Salwa Salah</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> This article analyzes various types of nanoparticles encapsulating different classes of drugs, such as phenelzine, fexofenadine, telmisartan, metformin, canagliflozin, atorvastatin, and fenbendazole,</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cancer remains one of the most significant global health challenges, with its burden continuing to rise. The limitations of conventional anticancer therapies caused by the lack of tissue selectivity, demands urgent development of safer and more selective therapies to target tumors. Identifying the fundamental cancer hallmarks provided a comprehensive understanding of cancer biology for effective tumor targeting, encompassing tumor-promoting inflammation, metabolic reprogramming, immune evasion, genomic instability, phenotypic plasticity, epigenetic reprogramming, and polymorphic microbiomes. Moreover, drug repurposing is a cost-effective and time-saving method for cancer therapy that accelerates the drug discovery process by reusing drugs for new indications. Current research is focusing on combining drug repurposing with nanocarriers that enhance tumor targeting, reduce the side effects, and improve the bioavailability of the drug in a single nanoformulation. This article analyzes various types of nanoparticles encapsulating different classes of drugs, such as phenelzine, fexofenadine, telmisartan, losartan, metformin, canagliflozin, atorvastatin, and fenbendazole, highlighting their anticancer effects and the influence of nanocarriers on the drug's therapeutic effect. Results revealed that drug-encapsulated nanoparticles enhanced antitumor effects compared to the free drug solutions. This is attributed to the synergism from the nanocarrier's functionalization, sustained drug release, and improved cellular uptake within tumors that leads to targeting multiple cancer hallmarks. Additionally, this review highlights the present challenges in the clinical translation of nanoformulation and demonstrates how artificial intelligence may facilitate drug screening and identification, therapeutic optimization, and large-scale manufacture. Finally, using these technologies in combination with drug repurposing presents a promising direction for advancing cancer treatment.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41053454/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Single-cell RNA-sequencing of circulating tumour cells: A practical guide to workflow and translational applications.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Francis Yew Fu Tieng, Learn-Han Lee, Nurul-Syakima Ab Mutalib</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a 12-step CTC-specific scRNA-seq workflow to overcome methodological inconsistencies in metastasis research and propose a new method to investigate rare and hybrid populations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The global burden of cancer is rising, with treatment failures often due to the metastatic nature of late-stage malignancies. Circulating tumour cells (CTCs) are metastatic precursors shed from primary tumours, which survive in circulation, extravasate and colonise distant organs. The advent of high-throughput single-cell RNA sequencing (scRNA-seq) has revolutionised the investigation of transcriptomic landscape at single-cell resolution, enabling deep transcriptomic profiling, re-stratifying CTC subtypes and improving the detection of rare new subpopulations. Applications extend to understanding tumour microenvironments, characterising cellular heterogeneity, uncovering metastasis molecular mechanisms and improving prognosis and diagnostic strategies. A timeline of key milestones in CTC scRNA-seq research is also provided. Nevertheless, a knowledge gap remains due to unstandardised protocols and fragmented resources in CTC scRNA-seq research. We address this gap by proposing a 12-step CTC-specific scRNA-seq workflow to overcome methodological inconsistencies. This workflow spans the entire process from enrichment, single-cell sorting and sequencing to data pre-processing and downstream analyses, with a detailed compilation of data analysis tools. An in-depth discussion of the pros and cons of commonly used scRNA-seq tools is also included, specifically evaluating their suitability for CTC research. Additionally, emerging research frontiers, including the discovery of hybrid cells-fusion products of tumour and normal cells-and the integration of machine learning (ML) into scRNA-seq workflows, are explored. Future research should prioritise CTC scRNA-seq workflow standardisation, integrate ML-driven analysis and investigate rare and hybrid populations to advance metastasis research. This review supports these goals by guiding methods, informing tool selection and promoting data sharing for reproducibility.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41053409/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Combining Oral fluid aMMP-8, calprotectin and CCAAs in dental panoramic radiography for periodontal disease and systemic disease risk assessment: a point-of-care diagnostic approach.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ismo T R√§is√§nen, Miika Penttala, Vaibhav Sahni, Julie Toby Thomas, Andreas Grigoriadis, Dimitra Sakellari, Shipra Gupta, Pirjo P√§rn√§nen, Tommi P√§til√§, Timo Sorsa</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Calcifying carotid artery atheromas (CCAAs) identified on standard dental panoramic radiographs (DPRs) have been presented as potential disease markers for cardiovascular disease (CVD) and diabetes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Calcifying carotid artery atheromas (CCAAs) identified on standard dental panoramic radiographs (DPRs) have been presented as potential disease markers for cardiovascular disease (CVD). CCAAs are further linked to several systemic disease processes (i.e. diabetes) that are also associated with periodontitis. The active matrix metalloproteinase-8 (aMMP-8) mouthrinse point-of-care-test has been multiply globally validated for periodontitis disease diagnostics. Calprotectin can inhibit matrix metalloproteinases and also exert significant anti-microbial activities. Recently, calprotectin has been suggested as a potential biomarker of endovascular inflammation.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41048171/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Enabling Open Machine Learning of Deoxyribonucleic Acid-Encoded Library Selections to Accelerate the Discovery of Small Molecule Protein Binders.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> James Wellnitz, Shabbir Ahmad, Nabin Bagale, Xuemin Cheng, Jermiah Joseph, Hong Zeng, Albina Bolotokova, Aiping Dong, Shaghayegh Reza, Pegah Ghiabi, Elisa Gibson, Guiping Tu, Xianyang Li, Jian Liu, Dengfeng Dou, Jin Li, Timothy L Foley, Anthony R Harris, Jacquelyn L Klug-McLeod, Jisun Lee, Zsofia Lengyel-Zhand, Justin I Montgomery, Sylvie Sakata, Jinzhi Zhang, Hongyao Zhu, Dafydd R Owen, Rachel J Harding, Aled M Edwards, Benjamin Haibe-Kains, Levon Halabelian, Alexander Tropsha, Rafael M Cou√±ago</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the first fully open, automated DEL-ML framework using public DNA-encoded library data sets and chemical fingerprints to enable reproducible, accessible drug discovery.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Machine learning (ML) is increasingly used in DNA-encoded library (DEL) screening for ligand discovery, but its success depends on access to suitable data sets, which are often proprietary and costly. To overcome this, we present the first fully open, automated DEL-ML framework using public DEL data sets and chemical fingerprints to enable reproducible, accessible drug discovery. Our workflow‚îÄfrom model training to virtual screening and compound selection‚îÄrequires no human intervention. As a proof of concept, we identified binders for WDR91 by training ML models on the HitGen OpenDEL library (3B molecules) and screening the Enamine REAL Space library (37B molecules), yielding 50 candidates. Experimental testing confirmed seven novel binders with dissociation constants between 2.7-21 ŒºM. Our open-source approach matches the performance of proprietary methods, demonstrating that public DEL data can support robust ML-driven ligand discovery and fostering transparency and broader community participation in drug development.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41047897/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

