<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Thursday, 17 April 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!</p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Disease candidate genes prediction using positive labeled and unlabeled instances.</h2>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Identifying disease genes and understanding their performance is critical in producing drugs for genetic diseases. Nowadays, laboratory approaches are not only used for disease gene identification but also using computational approaches like machine learning are becoming considerable for this purpose. In machine learning methods, researchers can only use two data types (disease genes and unknown genes) to predict disease candidate genes. Notably, there is no source for the negative data set. The proposed method is a two-step process: The first step is the extraction of reliable negative genes from a set of unlabeled genes by one-class learning and a filter based on distance indicators from known disease genes; this step is performed separately for each disease. The second step is the learning of a binary model using causing genes of each disease as a positive learning set and the reliable negative genes extracted from that disease. Each gene in the unlabeled gene's production and ranking step is assigned a normalized score using two filters and a learned model. Consequently, disease genes are predicted and ranked. The proposed method evaluation of various six diseases and Cancer class indicates better results than other studies.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40241088/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Can interpretability and accuracy coexist in cancer survival analysis?</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Survival analysis refers to statistical procedures used to analyze data that focuses on the time until an event occurs, such as death in cancer patients. Traditionally, the linear Cox Proportional Hazards (CPH) model is widely used due to its inherent interpretability. CPH model help identify key disease-associated factors (through feature weights), providing insights into patient risk of death. However, their reliance on linear assumptions limits their ability to capture the complex, non-linear relationships present in real-world data. To overcome this, more advanced models, such as neural networks, have been introduced, offering significantly improved predictive accuracy. However, these gains come at the expense of interpretability, which is essential for clinical trust and practical application. To address the trade-off between predictive accuracy and interpretability in survival analysis, we propose ConSurv, a concept bottleneck model that maintains state-of-the-art performance while providing transparent and interpretable insights. Using gene expression and clinical data from breast cancer patients, ConSurv captures complex feature interactions and predicts patient risk. By offering clear, biologically meaningful explanations for each prediction, ConSurv attempts to build trust among clinicians and researchers in using the model for informed decision-making.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.11.648380v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CellFlow enables generative single-cell phenotype modeling with flow matching</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">High-content phenotypic screens provide a powerful strategy for studying biological systems, but the scale of possible perturbations and cell states makes exhaustive experiments unfeasible. Computational models that are trained on existing data and extrapolate to correctly predict outcomes in unseen contexts have the potential to accelerate biological discovery. Here, we present CellFlow, a flexible framework based on flow matching that can model single cell phenotypes induced by complex perturbations. We apply CellFlow to various phenotypic screens, accurately predicting expression responses to a wide range of perturbations, including cytokine stimulation, drug treatments and gene knockouts. CellFlow successfully modeled developmental perturbations at the whole-embryo scale and guided cell fate and organoid engineering by predicting heterogeneous cell populations arising from combinatorial morphogen treatments and by performing a virtual organoid protocol screen. Taken together, CellFlow has the potential to accelerate discovery from phenotypic screens by learning from existing data and generating phenotypes induced by unseen conditions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.11.648220v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Global and cross-omics feature aggregation improves single-cell multi-omics integration and clustering</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Sequencing technology advancements enable the simultaneous measurement of multiple omics within single cells, thereby facilitating single-cell multi-omics analysis. Most existing deep multi-omics integration methods learn shared or omics-specific representations from multiple omics by omics-wise aggregation. However, these approaches often fail to account for the structural relationships among all cells. Here, we introduce scCODI, an innovative cross-omics deep learning model to integrate single-cell multi-omics data for cell clustering. This model processes paired omics datasets, combining single-cell transcriptomics with either epigenomics or proteomics data to facilitate a more comprehensive exploration of cellular heterogeneity. Specifically, the shared representation of two omics is derived through cross-omics feature aggregation, fully leveraging the complementarity of different omics within the same cell. Additionally, we align the omic-specific and shared representations of each cell using a global relationship-guided contrastive learning module, which enhances the similarity between the shared and omic-specific representations of the same cell. Extensive experiments demonstrate that scCODI outperforms previous methods in single-cell multi-omics data integration and cell clustering, as evidenced by its performance across nine paired multi-omics datasets. This makes it a promising tool for decoding complex biological mechanisms.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.10.648152v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PETScan: Score-Based Genome-Wide Association Analysis of RNA-Seq and ATAC-Seq Data</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">High-dimensional sequencing data, such as RNA-Seq for gene expression and ATAC-Seq for chromatin accessibility, are widely used in studying systems biology. Accessible chromatin allows transcription factors and regulatory elements to bind to DNA, thereby regulating transcription through the activation or repression of target genes. The association analysis of RNA-Seq and ATAC-Seq data provides insights into gene regulatory mechanisms. Most existing analytic tools exclusively focus on cis-associations, despite regulatory elements being able to physically interact with distant target genes. Furthermore, conventional approaches often utilize Pearson or Spearman correlations, which ignore the count-based nature of RNA-Seq data. To address these limitations, we introduce PETScan, a computationally efficient genome-wide PEak-Transcript Score-based association analysis, utilizing negative binomial models to better accommodate RNA-Seq data. We leverage score tests and matrix calculations for improved computational efficiency, and combine an empirical permutation method with genomic control to ensure valid p-value calculations in studies with limited sample sizes. In real-world datasets, PETScan achieved three orders of magnitude faster than Wald tests, while identifying similar significant gene-peak pairs. The PETScan R package is available on GitHub at https://github.com/yajing-hao/PETScan.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.10.648278v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Biologically-informed Interpretable Deep Learning Framework for Phenotype Prediction and Gene Interaction Detection</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">The detection of epistatic effects has significant potential to enhance understanding of the genetic basis of complex traits, but statistical epistatic analysis methods are complex and labour intensive. In recent years, Deep Neural Networks (DNNs) have emerged as a powerful tool for modelling arbitrarily complex genetic interactions in relation to a phenotype; however, their utility is often limited by the challenge of interpreting their predictive reasoning. Although DNN interpretation methods exist, they are typically not designed for genomic applications, leading to hard-to-understand outputs with limited relevance to the field. To address this gap, we introduce GENEPAIR - a novel DNN interpretation framework designed specifically for genomic data, aimed at detecting putative associated gene-gene interactions for a phenotype of interest. Our approach offers several key advantages including model agnositicity, robustness to sample- and variant-level data variance, and flexibility to integrate varied domain knowledge into interpretable features. We demonstrate the efficacy of our method by applying it to a DNN trained on genetic variant data to predict Body Mass Index (BMI). The results of the analysis not only reveal single gene influences in close alignment with literature but also uncover previously unreported gene-gene interactions, demonstrating its significant potential for genomic discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.11.648325v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Large Language Model Consensus Substantially Improves the Cell Type Annotation Accuracy for scRNA-seq Data</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Different large language models (LLMs) have the potential to complement one another. We introduce an iterative multi-LLM consensus framework for annotating single-cell RNA sequencing data. This framework outperforms the best state-of-the-art method by nearly 15% in mean accuracy (77.3% vs 61.3%) across 50 diverse datasets from 26 tissues, encompassing over 8 million cells. By leveraging cross-model deliberation, our framework quantifies uncertainty, identifies ambiguous clusters for expert review, provides transparent reasoning chains, and minimizes the effort and expertise needed for cell type annotation in large-scale studies. Additionally, our framework enables users to seamlessly integrate new LLMs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.10.647852v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">BRIDGE: A Coarse-Grained Architecture to Embed Protein-Protein Interactions for Therapeutic Applications</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">In this work, we present Biophysical Representation of Interfaces via Delaunay-based Graph Embeddings (BRIDGE), a coarse-grained graph neural network that captures embeddings containing meaningful information about protein-protein interactions. The BRIDGE model takes as input graphs defined by the Delaunay tesselation of the individual chains and is pre-trained to predict the Delaunay adjacency at the protein-protein interface. The model achieves state-of-the-art performance in this task. The biophysical information captured by the BRIDGE embedding layer due to this pre-training task can further be used for downstream tasks, including for therapeutically relevant property prediction. We demonstrate the use of these embeddings by training models to predict antibody-antigen (Ab-Ag) affinity and antibody viscosity. These predictors, which require only the structures of the individual molecules in isolation, are especially well suited to large molecule therapeutic applications where complex structures are rarely determined until the very latest stages of a project.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.11.648307v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">GeST: Towards Building A Generative Pretrained Transformer for Learning Cellular Spatial Context</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Learning spatial context of cells through pretraining on spatial transcriptomics (ST) data may empower us to systematically decipher tissue organization and cellular interactions. Yet, transformer-based generative models often focus on modeling individual cells, neglecting the intricate spatial relationships within them. We develop GeST, a deep transformer model that is pretrained by a novel spatially informed generation task: Predict cellular expression profile of a given location based on the information from its neighboring cells. We propose a spatial attention mechanism for efficient pretraining, a flexible serialization strategy for converting ST data into sequences, and a cell tokenization method for quantizing gene expression profiles. We pretrained GeST on large-scale ST datasets of different ST technologies and demonstrated its superior performance in generating unseen spatial cells. Our results also show that GeST can extract spatial niche embeddings in a zero-shot way and can be further fine-tuned for spatial annotation tasks. Furthermore, GeST can simulate gene expression changes in response to perturbations of cells within spatial context, closely matching existing experimental results. Overall, GeST offers a powerful generative pre-training framework for learning spatial contexts in spatial transcriptomics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.09.648072v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Expression and In-silico Structural analysis of N-acetylglucosamine transporter, Ngt1 in model fungal pathogen, Candida albicans</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Introduction: The commensal yeast, Candida albicans, has the enormous metabolic flexibility to utilize alternative carbon sources such as N-acetylglucosamine (GlcNAc) at the infection sites, thereby facilitating its adaptive strategies to colonize and thrive within diverse host niches. In Candia, GlcNAc specific transporter (Ngt1) imports GlcNAc at cell surface to induce signaling through the master regulator Ngs1 at the chromatin level. Thus, GlcNAc signaling is responsible for changing its habitat from commensal to pathogenic life style. Objective: The current study aims at understanding the expression, structural and functional intricacies of Ngt1 that determines to GlcNAc sensitivity, specificity and its transport which is a prerogatory step in the GlcNAc signaling process in Candida. Method: The 3-D structure of Ngt1 was predicted using AlphaFold. We have carried out docking studies using AutoDoc suite for native and Site directed mutant versions of Ngt1 with GlcNAc to determine role of critical amino acids and further, tunnel analysis to reveal the transport mechanism. These findings lead to the designing structure-based Ngt1blockers via virtual screening of natural compound libraries. Results: Our docking studies on Ngt1 with GlcNAc revealed the useful insights in to mechanism and the role of critical amino acids (Ala 324, Ile 325, Tyr 329, and Asp 332) responsible for interaction. In our analysis, Asn 195, located within transmembrane helix and part of the UNC-93-like transmembrane regulatory protein domain, was identified as critical residue for GlcNAc interaction compared with other mutated residues. Interestingly, the tunnel analysis of Ngt1 transporter revealed presence potential pathways in Ngt1 protein for effective GlcNAc transport. Conclusion: This study provides basic insights in to the structural aspects of Ngt1 and its interaction with GlcNAc that can be further explored for the development effective therapeutic agents for control of infection caused by the Candida.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.14.648653v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">STM is required for fate establishment of productive shoot progenitor cells in Arabidopsis tissue culture</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">De novo shoot regeneration via the two-step tissue culture method is widely used for vegetative reproduction of plants and for gene transformation and editing. A key step in tissue culture is the fate transition from shoot progenitor cells (SPCs) to productive SPCs (prSPCs), which are capable of differentiating into a shoot. In this study, we identified the class I KNOTTED1-like homeobox (KNOX) transcription factor gene SHOOT-MERISTEMLESS (STM) as the key controller of, and marker gene for, the fate transition from SPCs into prSPCs, indicating that STM might serve as a candidate of molecular tools to improve de novo shoot regeneration in the agricultural applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.14.648630v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

