<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Saturday, 04 October 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Prediction of Polygenic Risks by Screening Thousands of Polygenic Scores</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chen, W.-M., Manichaikul, A., Onengut-Gumuscu, S., Worrall, B. B., Rich, S. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the existing PGS models for binary trait prediction are not broadly available for certain traits due to statistical power limitations, and propose a method of developing PGS model that can be used in studies of any size with genome-wide SNP data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Rationale: A polygenic score (PGS) summarizes the genetic information of a person in a single number for a trait, and its utility in genomic medicine is well-recognized. Although PGS models have been generated for many traits, they are not broadly available for certain traits due to limited sample sizes in studies of infrequent outcomes. Often, prediction of these not well-studied traits (e.g., treatment/drug responses) would be of great clinical utility and have been underutilized due to statistical power limitations. To support more versatile trait prediction, we present a method of developing PGS models that can be used in studies of any size with genome-wide SNP data. Method: We first generate thousands of PGSs for each study participant in a given data set using their genome-wide SNP data and public resources. A PGS-wide scan involves evaluating the Area Under the Curve (AUC) of prediction for a binary trait (or the R-squared of association for a quantitative trait) at each PGS. We present two methods for the PGS model development, SECRET-Best, which selects the most predictive PGS from the PGS scan for prediction, and SECRET-WTSUM, which considers a combined score from multiple correlation-pruned PGSs. This algorithm is scalable and implemented in a user-friendly software tool, SECRET (Screen and Evaluate Catalogued Risk scores to Enhance Trait predictions). Results: We applied SECRET to a binary outcome (type 1 diabetes [T1D]) and a dataset of 2,100 samples, each with 12 laboratory test-related continuous traits. For nine traits with existing PGSs available in the PGS catalog, eight of the traits were predicted correctly, with the same trait-related PGS identified as the top predictor. We showed that the existing PGS methods had rather limited power to predict trait values in a validation set when only 1,500 samples were used to develop a PGS model, while the SECRET methods were able to maintain the prediction power for under-powered GWAS studies even when the sample size of the study was in hundreds. Conclusion: The SECRET methods and tool provide a valuable resource for studies with genomic data but of limited sample sizes. This approach enables systematic development and evaluation of PGS models.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.03.680330v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">NLTD 2.0: A Nonlinear Framework for Robust and Customizable Color Deconvolution in Histopathology</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Selaru, F., Phillip, J., Wirtz, D., Wu, P.-H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> An intuitive and robust color deconvolution method that effectively and accurately separates more than three stain signals, does not rely on predetermined color vectors, and doesnot rely on identifying optimal stain vectors.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Advancements in computational approaches have enabled robust utilization of histological tissue data. A crucial step in the development of computational tools for the objective and quantitative analysis of tissue sections has been color deconvolution. Color deconvolution functions by separating the absorption of colors corresponding to stained molecular or tissue compartments. The most widely used color deconvolution method in digital pathology, linear color deconvolution as described in Ruifrok 2001 et al., decomposes color images according to the absorbance values for individual stains. However, linear color deconvolution assumes that stains are linearly decomposable, and it relies heavily upon identifying optimal color vectors of stains, which is often challenging. Furthermore, linear deconvolution methods cannot deconvolve the image with more than three stains, further limiting their broader applicability. To combat the limitations of previous methods, we developed an intuitive and robust color deconvolution method that effectively and accurately separates more than three stain signals, does not rely on predetermined color vectors, and does not rely on identifying optimal stain vectors. The proposed method, NLTD 2.0, presents a robust and efficient solution to tackle color variations in histopathology images, enhancing the reliability and precision of computational pathology. Additionally, incorporating the method as an ImageJ plugin amplifies accessibility and usability, enabling researchers and pathologists to leverage its capabilities without specialized programming skills. The intuitive interface streamlines the application, fostering broader acceptance within the computational pathology community.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.02.679756v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Modular Genetic Code Expansion Approach to Site-Specific Lysine Acylations</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Knecht, P., Nguyen, T.-A., Lang, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a modular strategy that combines genetic code expansion (GCE) with a chemoselective amide bond-forming reaction to install a wide range of lysine acylations directly onto folded proteins.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Lysine acylations are a diverse class of post-translational modifications (PTMs) that dynamically regulate protein function. Access to homogenous, site-specifically modified proteins is essential for dissecting their molecular roles, yet remains challenging with traditional methods. Here we present a modular strategy that combines genetic code expansion (GCE) with a chemoselective amide bond-forming reaction to install a wide range of lysine acylations directly onto folded proteins. Key to this approach is the genetic encoding of N{epsilon}-methoxylysine, which reacts efficiently with N-methyliminodiacetyl (MIDA) acylboronates to generate diverse acyl lysine modifications from a single protein precursor, including several acyl PTMs not previously accessible through GCE. This mild and broadly applicable methodology enables systematic studies of lysine acylation on multiple target proteins. We demonstrate its utility by probing the effects of defined acyl modifications on enzymatic activity, protein-RNA interactions and deacylase-mediated regulation. Together, this platform establishes a versatile strategy to interrogate the functional consequences of this wide-spread class of PTMs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.03.680225v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Rapid assembly of functional modules for generating human artificial chromosome constructs compatible with epigenetic centromere seeding</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Birchak, G. J., Gibson, D. G., Allu, P. K., Kashyap, P., Glass, J. I., Black, B. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed an in vitro assembly method for multi-module HACs that is efficient, efficient, and fast, and can be used to quickly incorporate diverse genetic cargoes to engineer HAC with specific design features.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The ongoing development of human artificial chromosomes (HACs) will permit investigation into essential centromere processes and the means to deliver large genetic cargoes to target cells. Starting with large (~750 kb) yeast artificial chromosome (YAC)-based constructs limits the rampant multimerization that has complicated many prior types of HACs. Large YAC construction is accomplished using transformation-associated recombination (TAR) strategies that can become unwieldly when several functional modules are to be incorporated and tested. To address this issue, we developed an approach where modules are built using high-fidelity in vitro assembly strategies in a bacterial artificial chromosome (BAC) format. Then, the assembled modules are transferred in a simplified TAR step into a recipient YAC harboring the prokaryotic stuffer DNA that comprises a large portion of the final HAC construct. This approach is highly efficient with two-thirds of all screened yeast clones harboring the correct TAR product. Further, whole-genome Oxford Nanopore Technologies (ONT) sequencing/alignments, de novo assembly of the final YAC using a single ONT sequencing run, and close inspection of highly repetitive regions are all streamlined to rapidly validate clones that match the design. The fully sequenced, verified strain harboring a multi-module construct was then fused to human cells, where it efficiently formed functional HACs upon initial seeding with CENP-A-containing nucleosomes. We envision that the rapid assembly steps will be useful to quickly incorporate different functional modules, including diverse genetic cargoes, to engineer HACs with specific design features.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.02.680129v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DREAMER-S: Deep leaRning-Enabled Attention-based Multiple-instance approaches with Explainable Representations for Spatial Biology</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rafsanjani, M. R., Dooney, A., Suresh, R., OFarrell, A. C., Jarzabek, M. A., Shiels, L., Byrne, A. T., Prehn, J. H. M., Meade, A. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce an attention-based multiple-instance learning (MIL) framework, DREAMER-S, that learns, from slide-level labels alone, to pinpoint regions in 3D imaging hypercube datasets whose signatures are most predictive for a downstream target classification, acting as a filter over massive</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We introduce an attention-based multiple-instance learning (MIL) framework, DREAMER-S, that learns, from slide-level labels alone, to pinpoint regions in 3D imaging hypercube datasets whose signatures are most predictive for a downstream target classification, acting as a filter over massive, multi-channel image sets. We demonstrate DREAMER-S in its application to tissue images captured by Quantum Cascade Laser IR imaging, where the model assigns attention weights to each spectrum, producing spatial "importance maps" that localize class-relevant regions without requiring manual annotation, and automatically surfaces the most informative spatial instances for downstream analysis. As the DREAMER-S MIL attention layer yields ROI maps and an instance-level filter, the technique is broadly applicable across spatial biology tasks to narrow regions-of-interest in high-content imaging datasets. We demonstrate the approach on a chemotherapy-response task in colorectal cancer patient-derived xenograft (PDX) models, separating a chemo-sensitive class (CRC0344) from a less responsive class (CRC0076) with a best validation F1-score of ~0.95 using a compact, residual-free linear network tuned by grid search. The attention mechanism both accelerates computation and functions as a data-reduction stage, enabling targeted interpretation with SHAP that highlights biochemically meaningful spectral features (nucleic acids, lipids, amide features) while excluding sample artifacts such as paraffin wax. In validating DREAMER-S from an explainability perspective, we connect model saliency to cellular physiology, finding: (1) unsupervised embeddings of the high-attention spectra stratify samples by treatment (FOLFOX, ABT-199, combination, vehicle), and (2) selected spectral markers correlate with pro-apoptotic proteins (Bim, Puma) measured independently in the same PDX system, overall supporting a mechanistic link between spectral signals and apoptosis pathways. Overall DREAMER-S supports efficient, explainable and interpretable analysis of high-content spatial-biology imaging sets.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.02.679953v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Pretrained protein language models choose between sequence novelty and structural completeness</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Subramanian, A. M., Martinez, Z. A., Thomson, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that although protein language models exhibit a prodigious latent capacity to access novel amino-acid sequences, they struggle to approach the structural variation that exists on plain display in nature.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein language models (PLMs) have gained increasing acceptance in tasks ranging from variant effect prediction in disease to optimization and de novo design of proteins with improved stability, target-binding affinity, and catalytic performance. Despite encouraging performance in such applications, little is understood as far as the degree to which PLM-generated sequences -- putative novel protein outputs -- recapitulate the broad biophysical rules and diversity of sequence, structure, and function that defines natural protein-space, vital knowledge for boosting the design capacity of PLMs in ever-more-complex systems. Towards this end, we computationally profile and characterize the sequence and structure statistics and properties of hundreds of thousands of potential small proteins proposed through free unconstrained generation from architecturally distinct PLMs. We show that although these models exhibit a prodigious latent capacity to access novel amino-acid sequences, they struggle to approach the structural variation that exists on plain display in nature. Moreover, we uncover a stark tradeoff between prioritizing sequence novelty or structural breadth, exemplified by a "helical bundle trap" that dominates model output when aiming outside the comfortable bounds and evolutionary organization of natural sequences. These findings underscore a critical need for strategies that can rapidly guide PLMs into unlocking through generation the full richness of protein sequence, structure, and function that is consistent with governing biophysics but tantalizingly untapped as of yet in design contexts.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.01.679905v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Long-Read epigenetic clocks identify improved brain aging predictions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Grant, S. M., Makarious, M. B., Meredith, M., Moller, A., Grant-Peters, M., Hicks, A., Mandal, A., Auluck, P., Leonard, H., Kuznetsov, N., Weller, C., Reed, X., Jain, M., Ferrucci, L., Cookson, M. R., Ryten, M., Nalls, M. A., Billingsley, K. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present an ancestry-aware aging clock trained using GenoML, an automated machine learning platform for clinical and multi-omics data, on DNA methylation from long-read sequencing.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Epigenetic clocks are widely used to estimate biological aging, yet most are built from array-based data from peripheral tissues of predominantly European-ancestry individuals, limiting generalizability. Here, we present aging clocks trained using GenoML, an automated machine learning platform for clinical and multi-omics data, on DNA methylation from Oxford Nanopore long-read sequencing. These models leverage over 28 million CpG sites across individuals of African and European ancestry. Our findings highlight the power of long-read methylation data for constructing accurate, ancestry-aware aging clocks and emphasize the importance of inclusive training datasets.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.30.679553v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">EasyFigAssembler: Enhance Omics Data Storytelling through Effective Figure Assembly</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Long, N. P., Quang Thu, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> EasyFigAssembler, a user-friendly web application that significantly simplifies and accelerates the making of multi-panel omics-based figures for scientific publications.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: An effective figure helps communicate the data and findings clearly. The assembly of multi-panel omics-based figures for scientific publications is a time-consuming and error-prone process that often requires specialized software and skills. However, there is a lack of a quick and effective tool for assembling complex omics-based figures. Results: We present EasyFigAssembler, a user-friendly web application that significantly simplifies and accelerates the making of multi-panel omics-based figures. The app supports non-destructive panel editing, multi-figure project management, and high-resolution export to TIFF, PDF, PNG, and JPEG formats, providing a complete solution for figure assembly without requiring programming expertise. Its Smart Layout feature automates the initial arrangement of panels into optimal, publication-ready grids. The advanced controls for column and row spanning enable the rapid assembly of complex, non-uniform layouts that are typically time-consuming to create. The workflow is ensured by an integrated Quality Advisor module that provides real-time feedback on effective resolution and color-blind-friendly palettes to ensure journal compliance. Finally, users can manage multiple figures and save their projects effortlessly. Availability and implementation: The source code of EasyFigAssembler is available at https://github.com/Pharmaco-OmicsLab/EasyFigAssembler. The web interface for users can be accessed at https://easyverse.app/easyfig/.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.01.679913v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SpacGPA: annotating spatial transcriptomes through de novo interpretable gene programs</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xu, Y., Chen, L., Ma, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present SpacGPA, a segmentation-free, single-cell-independent framework that identifies de novo interpretable gene programs--co-expressed gene modules capturing key biological processes--from spatial transcriptomes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Spatial transcriptomes, especially those from high-density platforms (e.g., Visium HD, Stereo-seq, Xenium Prime 5K), enable detailed tissue mapping but remain challenging to annotate for spatial domains and spatially variable genes (SVGs). Most existing methods depend on companion single-cell datasets and are sensitive to cell segmentation. We present SpacGPA, a segmentation-free, single-cell-independent framework that identifies de novo interpretable gene programs--co-expressed gene modules capturing key biological processes--from spatial transcriptomes. Leveraging GPU-acceleration and block-matrix operations, SpacGPA efficiently detects gene programs based on a graphical Gaussian model in ultra-large datasets. These programs exhibit domain-specific spatial expression patterns and cell type-specific expression in independent single-cell datasets, serving as robust markers for spatial domains. Applied to four mouse and human spatial datasets, SpacGPA identified 58-109 gene programs per dataset, revealing refined tissue structures. Hub genes within these programs correlate with canonical cell type markers and SVGs, enabling systematic SVG identification. The programs are also functionally enriched and transferable across datasets and platforms. Built on AnnData, SpacGPA provides a scalable, low-hardware solution for high-resolution annotation of tissue organization and gene regulation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.01.679918v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Protein Language Model-Aligned Spectra Embeddings for De Novo Peptide Sequencing</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> NaderiAlizadeh, N., Dallago, C., Soderblom, E. J., Soderling, S. H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a constrained learning framework for de novo peptide sequencing in tandem mass spectrometry that leverages pre-trained protein language models to guide the training process.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We consider the problem of de novo peptide sequencing in tandem mass spectrometry, where the goal is to predict the underlying peptide sequence given a spectrum's fragment peaks and precursor information. We present PLMNovo, a constrained learning framework that leverages pre-trained protein language models (PLMs) to guide the training process. In particular, we cast peptide-spectrum matching as a constrained optimization problem that enforces alignment between spectrum and peptide embeddings produced by a spectrum encoder and a PLM, respectively. We use a Lagrangian primal-dual algorithm to train the spectrum encoder and the peptide decoder by solving the proposed constrained learning problem, while optionally fine-tuning the pre-trained PLM. Through numerical experiments on established benchmarks, we demonstrate that PLMNovo outperforms several state-of-the-art deep learning-based de novo sequencing algorithms.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.01.679857v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Characterising cancer-stroma interactions through high-content phenotyping from microscopy time-lapses</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wiggins, L., Malcolm, J. R., Hogg, K., O'Toole, P. J., Wilson, J., Brackenbury, W. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the CellPhe toolkit enables label-free identification and phenotypic characterisation of different cell types within complex live-cell imaging datasets.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding how cancer-stromal interactions shape cancer progression requires tools that can capture dynamic phenotypic changes in physiologically relevant conditions. Traditional approaches for studying co-culture interactions, such as transcriptomics and flow cytometry, provide valuable insights but are limited by their static nature and reliance on fixed or dissociated cells. In contrast, label-free time-lapse microscopy preserves temporal and spatial context, enabling observation of live-cell behaviours over time. A major challenge, however, lies in the analysis of the resulting high-dimensional datasets. Using co-cultures of breast cancer cells and cancer-associated fibroblasts (CAFs) as a model system, we show that the CellPhe toolkit enables label-free identification and phenotypic characterisation of different cell types within complex live-cell imaging datasets. Our analysis shows that exposure to CAFs drives marked phenotypic shifts in breast cancer cells, including elongation, loss of cell-cell adhesion, and redistribution of intracellular components - hallmarks of epithelial-mesenchymal transition (EMT). To probe the underlying mechanisms, we performed a Luminex immunoassay on CAF-conditioned media and identified secreted analytes strongly associated with EMT induction. Together, these results highlight how automated phenotyping can be integrated with molecular profiling to identify and characterise cellular processes shaped by stromal interactions and reveal the signalling mediators that drive them.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.02.680037v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Drosophila storage proteins promote both the rate and the duration of tumor growth</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Valzania, L., Blanco Obregon, D., Alami, A., Leopold, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Amino acid storage proteins, such as serum albumin in mammals, are known to accumulate within tumors, but their precise contribution to tumor biology remains unclear. We show that fat body-derived hexamerins accumulate in the tumors through a selective uptake process.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Amino acid storage proteins, such as serum albumin in mammals, are known to accumulate within tumors, but their precise contribution to tumor biology remains unclear. Using both cachectic and non-cachectic tumor models in Drosophila, we observed that fat body-derived hexamerins accumulate in the tumors through a selective uptake process. Disabling this uptake led to a marked reduction in tumor growth, demonstrating that hexamerins are used as a nutrient source to support cancer progression. Hexamerin uptake also supports expression of the relaxin-like Dilp8 by the tumor, inhibiting ecdysone production and extending the growth period. This coupling between nutrient uptake by the tumor and inhibition of the developmental progression exerts a full diversion of the host resources. Functional parallels with mammalian albumins suggest evolutionarily conserved mechanisms with potential implications for cancer biology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.02.680002v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Persistent interferon signaling and clonal expansion mark early events in DNA methylation damage-induced liver cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pribyl, L. J., Kay, J. E., Corrigan, J. J., Volk, L. B., Norales, M., Owiti, N. A., Kowal, E. A., Kohale, I. N., Nazari, I. S., Swanson, M. R., Moise, A. C., Ma, D., Levine, S. S., Michelsen, E., Croy, R. G., Ragan, T., Torous, D. K., Avlasevich, S. L., Dertinger, S. D., Carrasco, S. E., Samson, L. D., Essigmann, J. M., White, F. M., Engelward, B. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that acute early-life NDMA exposure leads to a cascade of molecular events triggered by DNA damage, culminating in cancer development months later.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> N-Nitrosodimethylamine (NDMA), a probable human carcinogen, induces toxic and mutagenic O-methylguanine lesions that are repaired by the O6-methylguanine methyltransferase (MGMT). To elucidate mechanisms of NDMA-induced liver cancer progression, we performed longitudinal analyses of phenomic, transcriptomic, and phosphoproteomic changes in wild-type and MGMT-deficient mice, observing amplified responses in the deficient genotype. Early molecular rewiring indicative of a DNA damage response was detected by phosphoproteomic and transcriptomic profiling within days post-exposure. Transcriptomic analyses identified a persistent and robust interferon response as the dominant activated pathway. This chronic interferon signaling, which remained unresolved, correlated with extensive clonal expansion, an early hallmark of oncogenesis. Spatial transcriptomics further revealed pathway alterations favoring tumorigenesis within clonally expanded cells. These findings delineate the cascade of molecular events triggered by acute early-life NDMA exposure, culminating in cancer development months later. Our study unveils potential predictive biomarkers and strategies for disease mitigation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.01.679856v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Collagen-Based Tumor Spheroid Model for Investigating Tumor-Macrophage Interactions through Extracellular Matrix Remodeling</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, L., Abdi, S., Szafraniec, H., Provenzano, P., Schwertfeger, K. L., Wood, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We develop a 3D collagen-based tumor spheroid model to study the impact of peripheral blood mononuclear cell-derived macrophages on cancer cell-ECM and cancer-macrophage interactions within the TME.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Macrophages in the tumor microenvironment (TME) can constitute up to 50% of tumor mass and play a critical role in cancer cell proliferation, invasion, and metastasis. While their contribution to extracellular matrix (ECM) degradation through matrix metalloproteinases (MMPs) has been explored, the role of other macrophage-derived factors in ECM remodeling and their impacts beyond degradation remain poorly understood. Here, we describe the development of a 3D collagen-based tumor spheroid model to investigate the impact of peripheral blood mononuclear cell (PBMC)-derived macrophages on cancer cell-ECM and cancer cell-macrophage interactions within the TME. We observed that cancer cells stimulated PBMC-derived macrophages into an M2-like phenotype and that tumor spheroid conditioned macrophages (TSCMs) shifted cancer cell populations toward phenotypes with greater invasion distances and reduced circularity, indicative of increased malignancy. Such observations can be explained by macrophage-mediated ECM remodeling. Specifically, we demonstrate that TSCMs secreted a variety of soluble factors that are known to contribute to ECM remodeling, including ECM degradation and fiber realignment. These processes collectively create a tumor-favoring environment by loosening the collagen matrix and aligning fibers that serve as invasion tracks for migrating tumor cells that facilitate cancer cell migration and invasion. This model provides a robust platform to study the interactions between cellular and non-cellular components in the TME and to identify the molecular mechanisms underlying cancer progression. These insights may aid in the development of novel therapeutic strategies targeting macrophage-mediated processes in cancer.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.01.679587v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

