<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Monday, 05 May 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Synergistic bioinformatics and sophisticated machine learning unveil ferroptosis-driven regulatory pathways and immunotherapy potential in breast carcinoma.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lei Xia, Zhen Ye, Man Zheng, Zhaofeng Tan</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> The intersection of aberrant iron metabolism and the rapidly advancing field of immunotherapy has emerged as a critical focus in breast cancer treatment.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The intersection of aberrant iron metabolism and the rapidly advancing field of immunotherapy has emerged as a critical focus in breast cancer (BRCA) therapeutics. Ferroptosis, a distinct form of iron-dependent cell death driven by lipid peroxidation, has garnered increasing attention for its pivotal role in cancer progression.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40320501/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Quantification of single cell-type-specific alternative transcript initiation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fu, S., Wilson, P., Zhang, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed TSRdetector, a novel bioinformatic method specifically designed to detect the significant usage alteration of gene TSRs in different tissues, cell types, or diseases.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cells can transcribe different isoforms of a gene by using distinct Transcriptional Start Regions (TSRs), which are recognized by RNA-Polymerase II and regulated by cell-type-specific expressed transcription factors, eventually forming tissue and cell-type-specific expression during development. However, how the distinct TSRs are selectively activated in different tissues and cell types remains largely uncharacterized. To better explore the alternative usage of gene TSRs, we developed TSRdetector, a novel bioinformatic method specifically designed to detect the significant usage alteration of gene TSRs in different tissues, cell types, or diseases. TSRdetector can process either scRNA-seq or bulk-RNA-seq transcriptome data, define dominant TSRs, and compute the differential usage of gene TSRs between given conditions. To demonstrate the capacity of TSRdetector, we applied TSRdetector to analyze a 10X snRNA-seq dataset of healthy and diabetic human kidneys, a Smart-seq2 dataset of mouse B-cell differentiation, and bulk-RNA-seq data of pluripotency transition in human ESCs. In all three analyses, TSRdetectors discovered significant alterations in TSR usage, accompanied by the significant remodeling of the epigenetic landscape. Alteration of TSR usage can change the dominant transcript isoform and further affect the major protein products. Interestingly, a large proportion of these alterations of TSR usage in different cell types did not change the overall gene expression, revealing unique transcription regulations that are independent of expression level. In summary, TSRdetector is a user-friendly package to analyze the differential usage of gene TSRs by using both scRNA-seq and bulk-RNA-seq data, and can be used to explore the alternative transcription initiations of genes at the single cell-type level.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.29.651292v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Gene-based calibration of high-throughput functional assays for clinical variant classification</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zeiberg, D., Tejura, M., McEwen, A. E., Fayer, S., Pejaver, V., Rubin, A. F., Starita, L. M., Fowler, D. M., O'Donnell-Luria, A., Radivojac, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce and evaluate a method for calibrating continuous high-throughput functional data as a line of evidence in clinical variant classification.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> High-throughput functional assays measure the effects of variants on macromolecular function and can aid in reclassifying the rapidly growing number of variants of uncertain significance. Under the current clinical variant classification guidelines, using functional data as a line of evidence to assert pathogenicity relies on determining assay score thresholds that define variants as functionally normal or functionally abnormal. These thresholds are designed to maximize the separation of variants with known clinical effects (benign, pathogenic) and often incorporate expert opinion. However, this approach lacks the rigor of calibration, in which a variant's posterior probability of pathogenicity must be estimated from the raw experimental score and mapped to discrete evidence strengths. To build upon the existing guidelines, we introduce and evaluate a method for calibrating continuous high-throughput functional data as a line of evidence in clinical variant classification. Assay score distributions of synonymous variants and variants appearing in gnomAD for a given functional scoreset are jointly modeled with score distributions of known pathogenic and benign variants using a multi-sample skew normal mixture of distributions. This model is learned using a constrained expectation-maximization algorithm that provably preserves the monotonicity of pathogenicity posteriors and is subsequently used to calculate variant-specific evidence strengths for use in the clinic. Using 24 datasets from 14 genes, we first assess the model's ability to capture assay score distributions. We then demonstrate its potential impact on reclassifying variants by comparing the evidence strengths assigned at the variant-level with those assigned uniformly to all functionally normal and abnormal variants under the existing ClinGen guidelines. An improved classification of variants will directly improve the accuracy of genetic diagnosis and subsequent medical management for individuals affected by Mendelian disorders.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.29.651326v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Localized Reactivity on Protein as Riemannian Manifolds:A Geometric and Quantum-Informed Basis for Deterministic, Metal-Aware Reactive-Site Prediction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Park, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a unified framework for protein reactive-site prediction that couples a rigorous geometric, quantum-informed model with a metal-aware, fully deterministic implementation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We present a unified framework for protein reactive-site prediction that couples a rigorous geometric, quantum-informed model with a metal-aware, fully deterministic implementation. Proteins are treated as smooth Riemannian manifolds; each residue is equipped with a localized fiber, whose environment vector encodes geometric and physico-chemical features. Motivated by Density Functional Theory - which links global electron density to reactivity - we hypothesize that a sufficiently rich local density stencil captures the same information at the residue scale. On this foundation we build a single-file Python toolkit, reactive site predictor.py, which ranks candidate reactive residues in both metallo- and apo-proteins. The score combines (i) a geometry-based environment term, (ii) relativistic ZORA distance factors for metal centers, (iii) coordination-number and angle-deviation corrections, and (iv) an optional pocket-proximity flag. A fixed global random seed removes all stochastic elements, guaranteeing reproducibility. Benchmarks on three representative assemblies - Rubisco, GroEL, and SecA - achieve top-10 recall between 90 and 50 percent for documented functional residues. While promising, the current validation is limited to three static PDB structures; extension to dynamic ensembles and explicit quantum layers is left to future work. The framework thus offers an interpretable, physics-grounded, and deterministic paradigm for protein-function prediction. Beyond residue-level reactivity, the same manifold-fiber formalism is ultimately intended for purely geometric analysis of protein-protein interfaces: instead of assigning scalar scores, we construct an explicit contact submanifold and study its curvature, topology, and fibre-overlap signatures to deduce interaction logic without additional learning.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.29.651260v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MIST-Explorer: The Comprehensive Toolkit for Spatial Omic Analysis and Visualization of Single-Cell MIST Array Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fischer, C., Chen, J., Meah, A., Wang, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present MIST-Explorer, a comprehensive, user-friendly toolkit for the visualization and analysis of single-cell spatial MIST array data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent advances in spatial proteomics have enabled high-dimensional protein analysis within tissue samples, yet few methods accurately detect low-abundance functional proteins. Spatial MIST (Multiplex In Situ Tagging) is one such technique, capable of profiling over 100 protein markers spatially at single-cell resolution on tissue sections and cultured cells. However, despite the availability of various open-source tools for image registration and visualization, no dedicated software exists to align the images and analyze spatial MIST data effectively. To address this gap, we present MIST-Explorer, a comprehensive, user-friendly toolkit for the visualization and analysis of single-cell spatial MIST array data. Developed in Python with a PyQt6-based graphical interface, MIST-Explorer streamlines the spatial omics workflow-from image preprocessing and registration to cell segmentation and protein quantification. The software supports two workflows: one for preprocessed datasets and another for raw image inputs, ensuring broad compatibility across experimental designs. Key features include tile-based image registration using Astroalign and PyStackReg, deep learning-based segmentation with StarDist, multi-channel visualization with layer controls, and an interactive analysis module offering ROI selection along with histograms, heatmaps, and UMAP plots. MIST-Explorer generates spatially resolved expression tables readily compatible with downstream single-cell analysis pipelines. By integrating all major steps into a single platform, MIST-Explorer empowers researchers to derive biological insights from complex spatial omics datasets without requiring extensive computational expertise.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.29.650640v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AI-assisted Drug Re-purposing for Human Liver Fibrosis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> GUAN, Y., Inchai, J., Fang, Z., Law, J., Brito, A. A. G., Pawlosky, A., GottWeis, J., Daryin, A., Myaskovsky, A., Palepu, A., Rong, k., Xu, Y., Tu, T., Peltz, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A hypothesis generating multi-agent AI system (AI co-scientist) could assist in re-purposing drugs for treatment of liver fibrosis and direct their experimental characterization.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Liver fibrosis is a severe disease with few treatment options due to the poor quality of the available animal and in vitro models. To address this, we investigated whether a hypothesis generating multi-agent AI system (AI co-scientist) could assist in re-purposing drugs for treatment of liver fibrosis and direct their experimental characterization. A multi-parameter image analysis workflow, which enabled anti-fibrotic efficacy and drug toxicity to be serially assessed in multi-lineage human hepatic organoids grown in microwells (i.e., microHOs), was used to assess the effects of 14 drugs. Remarkably, two of the three AI co-scientist-recommended drugs that targeted epigenomic modifiers exhibited significant anti-fibrotic activity. Analysis of the anti-fibrotic effects of five drugs indicated that two inhibited TGFb;-induced intracellular signaling and three drugs altered TGFb;-induced mesenchymal cell differentiation. Since all five of the anti-fibrotic drugs reduced TGFb;-induced chromatin structural changes, epigenomic changes play an important role in the pathogenesis of liver fibrosis. One AI co-scientist recommended drug is an FDA-approved anti-cancer treatment (Vorinostat) that reduced TGFb;-induced chromatin structural changes by 91% and promoted liver parenchymal cell regeneration in microHOs. Hence, the use of AI co-scientist and this microHO platform identified a potential new generation of liver fibrosis treatments that also promote liver regeneration.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.29.651320v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Evaluation of Nanopore direct RNA sequencing updates for modification detection</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Esfahani, N. G., Stein, A. J., Akeson, S., Tzadikario, T., Jain, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We demonstrate that Nanopore Direct RNA sequencing can simultaneously detect pseudouridine, N6-methyladenosine, 5-methylcytosine, and inosine modifications on individual mRNA strands.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Nanopore technology can directly sequence intact RNA molecules, offering a unique capability to read native modifications. Oxford Nanopore Technologies recently updated its direct RNA sequencing technology from RNA002 to RNA004 chemistry. This update included an improved basecaller (Dorado) for increased sequencing accuracy, and ionic current models for de novo detection of four RNA modifications. Using a single RNA extraction from GM12878 B-lymphocyte cell line, we compared RNA002 and RNA004 sequencing chemistries and evaluated Dorado modification calling accuracy. We computed U-to-C mismatches, previously used to identify putative pseudouridine sites, and ran m6anet for identifying putative N6-methyladenosine sites. Dorado results for each respective modification showed both global and site-specific differences when compared to RNA002 results. We used DRS data from in vitro transcription of GM12878 genomic DNA as well as synthetic oligonucleotides to evaluate Dorado modification calling performance. Dorado pseudouridine model achieved 96-98% for both accuracy and F1-score. Similarly, Dorado N6-methyladenosine model achieved 94-98% accuracy, 96-99% F1-score. Our results demonstrate that Nanopore Direct RNA sequencing could simultaneously detect pseudouridine, N6-methyladenosine, 5-methylcytosine, and inosine modifications on individual mRNA strands. It is critical to validate these calls using orthogonal methods (e.g., Liquid Chromatography with Tandem Mass Spectrometry) for increased confidence.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.01.651717v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genome-wide expression gradient estimation based on local pseudotime in single cell RNA sequencing</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tjaernberg, A., Jackson, C., Bonneau, R., Christiaen, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present pseudovelocity, an alternative method for calculating a local rate of change in transcription based on the kNN-G and diffusion-based pseudotime, to derive RNA velocity estimates for individual genes, and demonstrate how it improves the gene-based estimation of RNA velocity for downstream analysis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single cell genomics measures the internal state of individual cells and makes it possible to describe biological phenomena such as cell type heterogeneity, developmental progression, and dynamic differentiation using computational methods. A prominent approach to facilitate downstream analysis, for large collection of cells, is to connect individual cells by a k-Nearest Neighbor Graph (kNN-G). The kNN-G is among other things used by methods that derive pseudotime, a temporal ordering of cells. However, pseudotime methods require knowledge of initial states of the trajectories created. Alternatives have been invented that estimate gene rate of transcription e.g. by RNA velocity but these methods don't generalize to all individual genes for estimating transcriptional rate of change along pseudotime. While these methods can derive pseudo progression de novo by extrapolating transcription velocity vectors they are limited to subsets of genes where intronic reads are captured with sufficient magnitude. Here we present pseudovelocity, an alternative method for calculating a local rate of change in transcription based on the kNN-G and diffusion-based pseudotime, to derive RNA velocity estimates for individual genes, and demonstrate how it improves the gene-based estimation of RNA velocity for downstream analysis. This package can be accessed here: https://gitlab.com/Xparx/pseudovelocity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.01.650773v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Understanding the population structure of Moraxella catarrhalis using core genome multilocus sequence typing (cgMLST) and a life identification number (LIN) code classification system</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yassine, I., Jolley, K. A., Bray, J. E., Jansen van Rensburg, M. J., Patel, F., Sheppard, A. E., Zar, H. J., Allen, V., Ah Tow, L., Maiden, M. C. J., Nicol, M. P., Brueggemann, A. B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a core-genome multilocus sequence typing (cgMLST) scheme using 1,319 core genes, and a life identification number (LIN) barcode classification system to characterise M. catarrhalis, and found two divergent SR and SS lineages with</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Moraxella catarrhalis is an important cause of infectious exacerbations of chronic obstructive pulmonary disease and otitis media. Previously, seroresistant (SR) and serosensitive (SS) lineages that differed in virulence potential were described, which raised questions about their evolutionary relationship and species classification. To investigate the population structure of M. catarrhalis, we developed a core-genome multilocus sequence typing (cgMLST) scheme using 1,319 core genes, and a life identification number (LIN) barcode classification system. Whole-genome analyses of nearly 2,000 genomes confirmed two divergent SR and SS M. catarrhalis lineages with distinct evolutionary trajectories. SR genomes were more conserved, while SS genomes exhibited greater genetic variability. Virulence gene analyses revealed lineage-specific variations in ubiquitous surface proteins (UspA1 and UspA2) and lipooligosaccharide (LOS) types, and SR genomes had more diverse LOS variants. The bro {beta}-lactamase gene, and the mcb bacteriocin cluster, were more common in SR, which suggested different selective pressures and ecological adaptation. This cgMLST scheme and LIN code system provides a robust method for characterising M. catarrhalis, distinguishes between SR and SS lineages, and offers a unified framework for population structure analyses. Implemented within PubMLST, this open-access resource facilitates high-resolution genomic studies and supports the scientific community in understanding the evolutionary complexity of M. catarrhalis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.30.651387v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SarcAsM: AI-based multiscale analysis of sarcomere organization and contractility in cardiomyocytes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Haertter, D., Hauke, L., Driehorst, T., Nishi, K., Long, J., Tiburcy, M., Berecic, B., Primessnig, A., Bao, G., Cyganek, L., Schmidt, C. F., Zimmermann, W. H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce the Sarcomere Analysis Multitool (SarcAsM), which combines deep learning and graph-based methods for automated, fast, and unbiased structural assessment of sarcomeric Z-bands, sarcomeres, and myofibrils.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cardiomyocyte function critically depends on sarcomere dynamics and their organization in myofibrils. To uncover how cardiomyocyte function emerges from individual sarcomere dynamics, comprehensive analysis of Z-bands (nanometer scale), sarcomeres (~2 micrometer scale), and myofibrils (~10 to 100 micrometer scale) is required. Tools for such multiscale analyses are presently lacking. Here we introduce the Sarcomere Analysis Multitool (SarcAsM), which combines deep learning and graph-based methods for automated, fast, and unbiased structural assessment of sarcomeric Z-bands, sarcomeres, and myofibrils as well as their organization in larger myofibril domains. SarcAsM features a generalist deep learning model, pre-trained on a broad range of experimental and published images, ensuring its versatile application and immediate usability across diverse datasets. Finally, we demonstrate the versatile utility of SarcAsM in analyzing sarcomere structure and dynamics under acute and chronic drug exposure in hiPSC-derived cardiomyocytes with fluorescently tagged Z-bands. SarcAsM is available as open-source Python package and stand-alone application.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.29.650605v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Design of a Modular Inducible Circuit for Pulsatile Population Control of Engineered Bacteria</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Roy, R., Mayalu, M. N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A genetic circuit design that leverages AHL (acyl-homoserine lactone)-based quorum sensing and a transcriptional cascade regulated by an external inducer to drive synchronized proliferation and lysis events.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Synthetic biology enables the precise control of bacterial population dynamics through engineered genetic circuits, paving the way for innovative therapeutic and biosensing applications. Here, we present genetic circuit design that leverages AHL (acyl-homoserine lactone)-based quorum sensing and a transcriptional cascade regulated by an external inducer to drive synchronized proliferation and lysis events. In this system, the absence of the inducer arabinose prevents CinR mediated repression, enabling activation of a transcriptional cascade that drives the expression of the lysis protein, fluorescent protein, and a drug resistance gene. The accumulation of AHL acts as a proxy for population density and modulates gene expression through a quorum-sensing mechanism. This inducible circuit provides tunable and pulsatile control over bacterial population dynamics, enabling cycles of population growth and collapse. Such behavior prevents premature population clearance while ensuring functional bacterial densities for extended durations. Our system represents a foundational step toward implementing more complex multi-stable architectures for self-regulation of bacterial population. This dynamic and modular framework offers significant advantages for therapeutic interventions, such as self-regulating drug delivery, inflammation-responsive probiotic therapies, and targeted bacterial therapy.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.03.652077v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

