<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Wednesday, 07 May 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">HADDOCK3: A modular and versatile platform for integrative modelling of biomolecular complexes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Giulini, M., Reys, V., Teixeira, J. M. C., Jimenez-Garcia, B., Honorato, R. V., Kravchenko, A., Xu, X., Versini, R., Engel, A., Verhoeven, S., Bonvin, A. M. J. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce HADDOCK3, the new modular version of the program, in which the original, parameterisable albeit rigid pipeline has been first broken down in a catalogue of independent modules and then enriched with powerful analysis tools and third-party integrations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> HADDOCK is a widely used resource for integrative modelling of a variety of biomolecular complexes that is able to incorporate experimental knowledge into physics-based calculations during complex prediction, refinement, scoring and analysis. Here we introduce HADDOCK3, the new modular version of the program, in which the original, parameterisable albeit rigid pipeline has been first broken down in a catalogue of independent modules and then enriched with powerful analysis tools and third-party integrations. Thanks to this increased flexibility, HADDOCK3 can now handle multiple integrative modelling scenarios, providing a valuable, physics-based tool to enrich and complement the predictions made by machine learning algorithms in the post-AlphaFold era. We present examples of successful applications of HADDOCK3 that were not feasible with the previous versions of HADDOCK, highlighting its expanded capabilities. The HADDOCK3 software source code is freely available from the GitHub repository (https://github.com/haddocking/haddock3) and comes with an online user guide (www.bonvinlab.org/haddock3-user-manual). All example data described in this manuscript are available at https://github.com/haddocking/haddock3-paper-data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.30.651432v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Polygraph: a software framework for the systematic assessment of synthetic regulatory DNA elements.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Avantika Lal, Laura Gunsalus, Anay Gupta, Tommaso Biancalani, Gokcen Eraslan</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Polygraph, a Python framework that evaluates synthetic DNA elements, based on features like diversity, motif and k-mer composition, similarity to endogenous sequences, and screening with predictive and foundational models.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The design of regulatory elements is pivotal in gene and cell therapy, where DNA sequences are engineered to drive elevated and cell-type specific expression. However, the systematic assessment of synthetic DNA sequences without robust metrics and easy-to-use software remains challenging. Here, we introduce Polygraph, a Python framework that evaluates synthetic DNA elements, based on features like diversity, motif and k-mer composition, similarity to endogenous sequences, and screening with predictive and foundational models. Polygraph is the first instrument for assessing synthetic regulatory sequences, enabling faster progress in therapeutic interventions and improving our understanding of gene regulatory mechanisms.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40329378/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Cell-free synthetic biology for natural product biosynthesis and discovery.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rice AJ, Sword TT, Chengan K, Mitchell DA, Mouncey NJ, Moore SJ, Bailey CB.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We review the advances in cell-free biosynthesis of ribosomal peptides, analyse the rapid prototyping of natural product biosynthetic enzymes and pathways, highlight advances in novel antimicrobial discovery, and discuss the rising use of cell- free technologies in industrial biotechnology and synthetic biology.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Natural products have applications as biopharmaceuticals, agrochemicals, and other high-value chemicals. However, there are challenges in isolating natural products from their native producers (e.g. bacteria, fungi, plants). In many cases, synthetic chemistry or heterologous expression must be used to access these important molecules. The biosynthetic machinery to generate these compounds is found within biosynthetic gene clusters, primarily consisting of the enzymes that biosynthesise a range of natural product classes (including, but not limited to ribosomal and nonribosomal peptides, polyketides, and terpenoids). Cell-free synthetic biology has emerged in recent years as a bottom-up technology applied towards both prototyping pathways and producing molecules. Recently, it has been applied to natural products, both to characterise biosynthetic pathways and produce new metabolites. This review discusses the core biochemistry of cell-free synthetic biology applied to metabolite production and critiques its advantages and disadvantages compared to whole cell and/or chemical production routes. Specifically, we review the advances in cell-free biosynthesis of ribosomal peptides, analyse the rapid prototyping of natural product biosynthetic enzymes and pathways, highlight advances in novel antimicrobial discovery, and discuss the rising use of cell-free technologies in industrial biotechnology and synthetic biology.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40104998/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">dsOMOP: Bridging OMOP CDM and DataSHIELD for Secure Federated Analysis of Standardized Clinical Data.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> David Sarrat-Gonz√°lez, Xavier Escrib√†-Montagut, Jared Houghtaling, Juan R Gonz√°lez</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A tool that enables interoperability between federated data analysis platforms like DataSHIELD to perform remote analyses without transferring individual-level data between centers.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Collaborative clinical research projects face several challenges related to data sharing. The disparity between data standards and strict privacy regulations become more relevant as the number of involved institutions increases. To address these challenges, the scientific community has progressively adopted common data models like the OMOP CDM for multicenter data standardization and implemented federated data analysis platforms like DataSHIELD to perform remote analyses without transferring individual-level data between centers, thus mitigating disclosure risks. However, there is no native implementation that automatically combines both solutions, revealing the need for a tool that enables interoperability between these systems.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40327502/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">NMNAT1 Binding at Promoters and Enhancers Couples NAD+ Synthesis to RNA Polymerase II Engagement</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cedeno-Cedeno, Y., Taylor, S. J., Gamble, M. J., Steidl, U., Coleman, R. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A genome-wide study of the role of NAD+-synthetase NMNAT1 in gene expression and chromatin-modifying enzymes .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Gene expression relies on transcriptional bursts driven by dynamic chromatin-modifying enzymes that depend on metabolites like NAD+. Depletion of NAD+ contributes to cancer, metabolic disorders, and aging, emphasizing the significance of tight regulation of NAD+ production in the cell. NMNAT1, a nuclear NAD+ -synthetase enzyme, supports chromatin-modifying enzymes such as PARP1 and SIRT1; however, its direct role in transcriptional regulation remains unclear. Using integrated multi-omics, we present the first high-resolution, genome-wide study of NMNAT1s regulatory functions. We demonstrate that NMNAT1 binds to the promoters and enhancers of actively transcribed genes involved in DNA replication, cell cycle progression, and chromatin regulation. RNA-seq and CUT&Tag analyses indicate reduced RNA Polymerase II occupancy at downregulated genes in NMNAT1 knockout cells, implicating NMNAT1 in transcriptional activation through Pol II engagement. These findings position NMNAT1 as a key node linking localized NAD+ production to gene-specific transcription, offering new insights into metabolic regulation of gene expression.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.30.651499v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Sample-specific CpG loci are important for accurate long-read methylation analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hansen, S. B., Hansen, K. D., Limborg, M. T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a likelihood-based method that identifies sample-specific CpG loci at high precision in long-read sequencing data from Nanopore and PacBio platforms.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Methylated CpG loci mutate at high rates, but it remains challenging to identify sample-specific non-reference CpG loci for methylation analyses. Here, we present a likelihood-based method that identifies sample-specific CpG loci at high precision in long-read sequencing data from Nanopore and PacBio platforms. Inclusion of non-reference CpG loci increased the number of loci and reduced reference bias, and is important for accurate estimation of global methylation levels across samples.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.30.651558v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Single-cell alternative polyadenylation analysis reveals mechanistic insights of COVID-19-associated neurological and psychiatric effects</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ning, W., Chen, Q., Gu, Y., Liu, S., Li, X., Xu, R., Ye, R., Yang, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comprehensive analysis of APA in neural cells from COVID-19 patients at the single-cell level elucidated changes in APA levels in the brains of SARS-COV-2-infected patients and confirmed that these changes impair the function of the nervous system.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> COVID-19 is associated with increased risks of neurological and psychiatric sequelae. Alternative polyadenylation (APA) is ubiquitous in human genes, resulting in mRNA diversity, and has been validated to play a pivotal regulatory role in the onset and progression of a variety of diseases, including viral infections. Here, we analyzed the APA usage across different cell types in frontal cortex cells from non-viral control group and COVID-19 patients, and identified functionally related APA events in COVID-19. According to our study, the poly(A) site (PAS) usage is different among cell types and following SARS-COV-2 infection. Moreover, we found the genes with significant PAS level changes affected pathways related to RNA splicing, and neuronal development and function, suggesting that survivors of COVID-19 will have a high risk of these diseases and that alternative splicing functions cause these changes. Additionally, APA usage and its correlation with gene expression levels varied across genes, some prefer short isoform that is more stable to produce more proteins, while others may be regulated by different mechanisms. A total of 267 risk genes targeted by microRNAs for common neurological and psychiatric disorders were found to undergo significant changes in APA following infection. In conclusion, our comprehensive analysis of APA in neural cells from COVID-19 patients at the single-cell level elucidated changes in APA levels in the brains of SARS-COV-2-infected patients and confirmed that these changes impair the function of the nervous system, providing important insights for COVID-19-associated sequelae.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.02.651855v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Quantifying uncertainty in Protein Representations Across Models and Task</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Prabakaran, R., Bromberg, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that low-quality protein embeddings often fail to capture meaningful biology, displaying vector properties indistinguishable from those of randomly generated sequences. Our novel, model-agnostic scoring framework is the first to quantify protein sequence embedding reliability.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Embeddings, derived by language models, are widely used as numeric proxies for human language sentences and structured data. In the realm of biomolecules, embeddings serve as efficient sequence and/or structure representations, enabling similarity searches, structure and function prediction, and estimation of biophysical and biological properties. However, relying on embeddings without assessing the model's confidence in its ability to accurately represent molecular properties is a critical flaw - akin to using a scalpel in surgery without verifying its sharpness. In this study, we propose a means to evaluate the ability of protein language models to represent proteins, assessing their capacity to encode biologically relevant information. Our findings reveal that low-quality embeddings often fail to capture meaningful biology, displaying vector properties indistinguishable from those of randomly generated sequences. A key contributor to this performance issue is the models' failure to learn the underlying biology from unevenly distributed sequence spaces in the training data. Our novel, model-agnostic scoring framework is, to the best of our knowledge, the first to quantify protein sequence embedding reliability. We believe that our robust approach to screening embeddings prior to making biological inferences, stands to significantly enhance the reliability of downstream applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.30.651545v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Pathogenic Missense Mutations in Intrinsically Disordered Regions Reveal Functional and Clinical Signals</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Deutsch, N., Erdos, G., Dosztanyi, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A deep learning model for variant effect prediction reveals pathogenicity patterns within disordered protein regions, including disease-relevant mutations in disordered proteins.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Intrinsically disordered regions (IDRs) play essential roles in signaling and regulation but remain under-characterized in the context of human disease. We analyzed nearly one million ClinVar missense variants, focusing on the subset within IDRs defined by curated and predicted annotations. Pathogenic variants were significantly enriched in short linear motifs (SLiMs) and disordered binding regions, consistent with their functional importance. To extend these insights beyond existing annotations, we applied AlphaMissense, a deep learning model for variant effect prediction, revealing localized 'island-like' pathogenicity patterns within IDRs. Based on these signals, we developed a classifier to prioritize predicted ELM motifs (PEMs), identifying motifs linked to pathogenicity and major disease classes, including neurological, cardiovascular, and cancer-associated genes. Case studies in POLK, FOXP2, and LMOD3 demonstrate how this framework yields mechanistic insight into disease-relevant mutations within disordered protein regions, advancing clinical interpretation in the disordered proteome.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.01.651640v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CRISPRi perturbation screens and eQTLs provide complementary and distinct insights into GWAS target genes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ghatan, S., Panten, J., Oliveros, W., Sanjana, N. E., Morris, J. A., Lappalainen, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that CRISPRi and eQTL studies are highly complementary in terms of gene regulation, and show the potential for a powerful toolkit for interpretation of disease-associated loci.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Most genetic variants associated with human traits and diseases lie in noncoding regions of the genome 1, and a key challenge is to determine which genes they affect 2,3. A common approach has been to leverage associations between natural genetic variation and gene expression to identify expression quantitative trait loci (eQTLs) in the population 4,5, while a newer method uses pooled CRISPR interference (CRISPRi) perturbations of noncoding loci with single-cell transcriptome sequencing 6,7. Here, we systematically compared the results from these approaches across hundreds of genomic regions associated with blood cell traits. We find that while the two approaches often identify the same target genes, they also capture distinct features of gene regulation. CRISPRi tends to detect genes that are physically closer to regulatory variants and more constrained, whereas eQTL studies are sensitive to detecting multiple, often distal, genes. Comparing these discoveries to a gold-standard set of genes linked to blood traits, the two approaches provide highly complementary insights with distinct strengths and caveats. Our results provide insights for improved design of CRISPRi and eQTL studies and indicate the potential for a powerful toolkit for interpretation of disease-associated loci.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.05.651929v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">XVCF: Exquisite Visualization of VCF Data from Genomic Experiments</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Almuneef, G., Aljouie, A., Bokhari, Y., Almazroa, A., Rashid, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We have developed a Shiny App, interactive tool using the R programming language that utilizes other R packages like 'vcfR' and 'maftools' to visualize and generate quality control metrics for genetic data effectively and exquisitely.</p><p style="font-size: 16px; text-align: justify;">Background High-throughput genomic analyses of germline and cancer genomes facilitate the identification of causal and actionable genetic variants. The recent advances in next-generation sequencing technology generated large-scale genomic and/or multi-omics datasets from different disease types or models. Owing to the huge volume of data coming out of genomic experiments, scientists are facing challenges in handling, manipulating, visualizing, and interpreting the data. Currently, available tools to visualize genetic variants from VCF files are not very user-friendly as most of them require knowledge of command line tools or scripts to install and run those software. Moreover, biologists or clinicians lack this knowledge of computer programming. Therefore, graphical user interface (GUI) based tools or software are needed to effectively summarize and visualize the huge volume of genomic data such as VCF data. Methods We have developed a Shiny App, interactive tool using the R programming language that utilizes other R packages like 'vcfR' and 'maftools' to visualize and generate quality control metrics for genetic data effectively and exquisitely. A key improvement is the addition of a user-friendly interface, providing researchers with an interactive way to explore VCF or Cancer genomics data. Our tool is powered by Shiny, making it even easier for researchers to analyze and visualize genomic variation data using a GUI. Researchers can upload their datasets and customize the analysis parameters to suit their specific research needs. Results A user-friendly interactive tool has been developed for the summarization and visualization of data related to genomic variation research. The application features an easy and friendly interface, allowing users to perform various functions such as data loading, summarization, and visualization interactively. XVCF offers an easy-to-use GUI platform to read genetic variant data (annotated or unannotated) and extract useful information such as read depth, mapping quality, genotype, quality control summary, and allele frequency from unannotated data. In the second module of XVCF, the cancer genomic data (annotated, so far supported by ANNOVAR) is analyzed using 'maftools' to produce oncoplot, comparison of mutational load across different TCGA datasets, gene summary, etc. XVCF is available for free download from https://github.com/rashidma/XVCF. Conclusion XVCF Shiny web application can serve as a robust visualization and quality control GUI platform for the germline and cancer genomics dataset. We expect this tool will be immensely useful for researchers with less computational or technical knowledge. Being a shiny R package, XVCF can be installed across different operating systems and utilize different computer hardware configurations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.30.651450v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SAKPE: A Site Attention Kinetic Parameters Prediction Method for Enzyme Engineering</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Qiu, J.-H., Lin, Z., Chen, K.-W., Sun, T.-Y., Zhang, X., Yuan, L., Tian, Y., Wu, Y.-D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce SAKPE (Site Attention Kinetic Parameters Prediction Method for Enzyme Engineering), a novel machine-learning framework designed to predict enzyme kinetic parameters with enhanced accuracy in practical application scenarios.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The quantitative determination of enzyme kinetic parameters traditionally relies on experimental methods that are both time-intensive and costly. Machine learning models have demonstrated significant potential for predicting enzyme kinetic parameters in recent years. Despite this promise, these methods face challenges, including limited training data, inadequate sensitivity to subtle mutations, and poor alignment with practical enzyme engineering contexts. Here, we introduce SAKPE (Site Attention Kinetic Parameters Prediction Method for Enzyme Engineering), a novel machine-learning framework designed to predict enzyme kinetic parameters with enhanced accuracy in practical application scenarios. By incorporating protein representation, substrate representation, and protein representation with weights for important sites, SAKPE significantly outperforms existing methods in predicting enzymatic kinetic parameters, including turnover number (kcat), Michaelis constant (Km), and inhibition constant (Ki). Incorporating protein representation with weights for important sites enables SAKPE to effectively capture the impact of mutations, especially mutations of important sites and their surrounding amino acids of interest in enzyme engineering, on enzyme kinetics parameters. SAKPE offers a robust and practical tool for predicting enzyme kinetic parameters, providing a superior tool for enzyme engineering scenarios such as enzyme design, directed evolution, and industrial applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.30.651216v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Mechanisms and Plasticity in Leaves and Leaflets in a Creeping Legume, Mimosa pudica</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kellogg, M. T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Mechanical exhaustion explanation for the primary pulvinus behavior over repeated stimuli, where the rapid decline and sustained low responses suggest energy depletion and slow ion channel reset.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mimosa pudica (Fabaceae) is a creeping plant known for its rapid thigmonastic movement upon touch, facilitated by specialized joint-like thickenings called pulvini. This study examines the activation behavior of the primary pulvinus (P1) in response to repeated touch stimuli, providing evidence for a mechanical exhaustion mechanism underlying the response. Experiments were conducted on M. pudica in Cuajiniquil, Costa Rica. Petiole angle change was recorded following repeated P1 stimuli, both with (P3 group) and without (NS-P3 group) concurrent tertiary pulvinus (P3) activation. Results showed the highest mean petiole angle change and P1 activations at the first stimulus, with a significant decline at the second stimulus and sustained lower responses thereafter. Both the NS-P3 and P3 groups exhibited similar overall behavior, characterized by a sharp decline in petiole angle change and P1 activation counts after the first stimulus. However, the P3 group had a lower initial petiole angle change compared to the NS-P3 group, and exhibited significant wave-like behavior, suggesting a more pronounced refractory period due to the combined activation of both P1 and P3 pulvini. The findings support a mechanical exhaustion explanation for the primary pulvinus behavior over repeated stimuli, where the rapid decline and sustained low responses suggest energy depletion and slow ion channel reset.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.03.652067v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Liquid-phase determination of Arabidopsis respiration and photosynthesis using Clark-type O2 electrodes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sena, F., Couture, C., Berais-Rubio, A., Millar, A. H., Signorelli, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We validate and characterize the use of Clark-type oxygen electrodes, specifically the Hansatech Oxytherm+P system, to quantify both photosynthetic and respiratory activity in intact arabidopsis seedlings and demonstrate that oxygen consumption and production correspond to mitochondrial respiration and photosynthesis, respectively.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Photosynthesis and respiration are fundamental metabolic processes in plants, tightly connected through shared substrates, energy dynamics, and redox balance. Arabidopsis is the key genetic model for plants but monitoring these sorts of physiological processes presents significant challenges using traditional gas-exchange or fluorescence-based techniques due to the small size of intact Arabidopsis thaliana (arabidopsis) seedlings. Here, we validate and characterize the use of Clark-type oxygen electrodes, specifically the Hansatech Oxytherm+P system, to quantify both photosynthetic and respiratory activity in intact arabidopsis seedlings. By monitoring oxygen evolution in dark and light phases, we demonstrate that oxygen consumption and production correspond to mitochondrial respiration and photosynthesis, respectively. These processes were modulated by tissue biomass, light intensity, developmental stage, and stress conditions. Specific inhibitors such as potassium cyanide and paraquat confirmed that the recorded changes in oxygen concentrations reflected mitochondrial cytochrome oxidase activity and photosystem electron transport-dependent oxygen production, respectively. Moreover, oxygen evolution increased significantly with bicarbonate supplementation, validating the system's sensitivity to carbon fixation. We further showed that photosynthetic activity measured with this method correlates with a quantitative green index and responds dynamically to de-etiolation, abiotic stress (salt, osmotic, oxidative), and temperature shifts. Our study lays the groundwork for measuring photosynthesis based on oxygen evolution and respiration in arabidopsis knockout mutants, CRISPR lines, overexpression lines and ecotypes using Clark-type oxygen electrodes and highlights key considerations and limitations to consider when applying this approach. This platform could also be adapted for many other small tissue plant samples.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.04.652138v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Phylogenetic Analysis and Machine Learning Identify Signatures of Selection and Predict Deleterious Mutations in Common Bean</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cordoba-Novoa, H. A., Buckler, E. S., Romay, C., Berthel, A., Johnson, L., Balasubramanian, P., Hoyos Villegas, V.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of mildly deleterious mutations in common bean breeding populations and its impact on flowering time, maturity, and yield.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mutations are continuous source of new alleles and genetic diversity in populations. Domestication and selection influence the accumulation of alleles occurring across a range of deleteriousness. Evidence suggests that mildly deleterious mutations (DelMut) can be purged out of breeding populations, increasing favorable allele accumulation. We used phylogeny-based analyses among 36 legume genomes to identify selection signatures and predict DelMut in common bean. We also developed a multiparent advanced generation intercrossed (MAGIC) population of black beans to characterize DelMut. Genes involved in nitrogen metabolism showed signs of positive selection in the Middle American genome, whereas genes related to phosphorylation were positively selected in the Andean genome. By combining conservation and protein information with machine learning (ML) for high-dimensional feature analysis, we characterized 82,442 sites in the MAGIC founders (36,558 polymorphic) and 4,753 sites evenly sequenced among RILs that could be potentially deleterious. Variation in the number of highly DelMut (high predicted deleterious scores) among lines was observed and later correlated with agronomic traits. Phenotypic analyses showed that calculated genetic load (and number of highly DelMut) was negatively correlated with flowering time, maturity, and yield. A detailed in-silico analysis of predicted mutations showed presence in highly conserved protein regions, which is likely to affect protein functionality. Our results show that variation in genetic load can be observed in breeding populations and potentially impact plant performance. These results contribute to understanding the genome-wide accumulation patterns of DelMut in breeding populations. Our study supports future development of strategies to reduce genetic load in promising germplasm and accelerate breeding programs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.05.652309v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Engineering Artificial 5' Regulatory Sequences for Thermostable Protein Expression in the Extremophile Thermus thermophilus</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wong, C. F. A., Zhang, S., Tietze, L., Dahiya, G. S., Lale, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We extend the genetic toolkit for recombinant protein production by developing novel artificial 5' regulatory sequences in the thermophilic bacterium Thermus thermophilus.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The utilisation of biocatalysts in biotechnological applications often necessitates their heterologous expression in suitable host organisms. However, the range of standardised microbial hosts for recombinant protein production remains limited, with most being mesophilic and suboptimal for certain protein types. Although the thermophilic bacterium Thermus thermophilus has long been established as a valuable extremophile host, thanks to its high-temperature tolerance, robust growth, and extensively characterised proteome, its genetic toolkit has predominantly depended on a limited set of native promoters. To overcome this bottleneck, we have expanded the available regulatory repertoire in T. thermophilus by developing novel artificial 5' regulatory sequences. In this study, we applied our Gene Expression Engineering platform to engineer 53 artificial 5' regulatory sequences (ARES) in T. thermophilus. These ARES, which comprise both promoter and 5' untranslated regions (UTRs), were functionally characterised in both T. thermophilus and Escherichia coli, revealing distinct host-specific expression patterns. Furthermore, we demonstrated the utility of these ARES by demonstrating high-level expression of thermostable proteins, including {beta} -galactosidase, a superfolder citrine fluorescent protein, and phytoene synthase. A bioinformatic analysis of the novel sequences has also being carried out indicating that the ARES possess markedly lower GC content compared to native promoters. This study contributes to expanding the genetic toolkit for recombinant protein production by providing a set of functionally validated ARES, enhancing the versatility of T. thermophilus as a synthetic biology chassis for thermostable protein expression.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.06.652417v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

