<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Saturday, 23 August 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Patient-derived organoids reveal hypoxia-driven plasticity and therapeutic vulnerabilities in pheochromocytomas and paragangliomas</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, X., Calucho, M., Estrada-Zuniga, C. M., Noureen, M., Gonzalez-Cantu, H., Mossuto, S., Levi, J. N., Landry, B., Guo, Q., Rutherford, K. A., Cheng, Z.-M., Sannajust, A., Raghunathan, S., Balaji, S., Nascimento da Conceicao, V., Norris, S., Munawar, S. Y., Nguyen, H. T.-L., De Rosa, J., Tomlinson, G., Wu, J. X., Yeh, M. W., Livhits, M. J., Vaidya, A., Powers, J., Lechan, R., Tischler, A., Zhang, M., Graham, P., Jimenez, C., Peitzsch, M., Bechmann, N., Eisenhofer, G., Ding, Y., Kitano, M., Soragni, A., Dahia, P. L. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Pheochromocytomas and paragangliomas (PPGLs) are rare chromaffin cell-derived neuroendocrine tumors of sympathetic (catecholamine producing) or parasympathetic (nonsecretory) origin, frequently driven by dysregulation of hypoxia-inducible factor (HIF) signaling, particularly HIF-2a. Although often benign, PPGLs can metastasize unpredictably, with limited therapeutic options once disseminated. Progress has been hindered by the lack of robust preclinical models, especially those that capture their molecular complexity and microenvironmental influences. To address this gap, we established patient-derived tumor organoids (PDOs) from 35 PPGLs, encompassing a broad spectrum of clinical and molecular phenotypes. The organoids retained key immunohistochemical, genomic, transcriptomic, and catecholamine-secretory features of their parental tumors. PPGL organoids cultured under hypoxic conditions generally exhibited enhanced viability, supporting hypoxia as a driver of cell survival. Hypoxia activated HIF-1a and expanded ASCL1+ cell populations, suggesting a lineage shift toward an immature chromaffin state. In contrast, long-term normoxic cultures activated hypoxia inducible factor 2a (HIF-2a) and acquired a hybrid sympathoblast-mesenchymal identity in subpopulations with upregulation of extracellular matrix and cell cycle markers, independent of genotype. These features resemble high-risk neuroblastoma subtypes and establish a molecular parallel suggestive of shared lineage plasticity and pathogenic programs, detectable in primary PPGLs. Drug screening across a library of up to 51 drugs and combinations revealed both shared and unique vulnerabilities, with response rates to approved therapies matching clinical observations. The CDK4/6 inhibitor abemaciclib, previously unexplored in PPGLs, elicited the strongest activity. Abemaciclib responsive PDOs and their matched tumors, including a metastatic sample, exhibit ed epithelial mesenchyme transition enrichment, nominating potential biomarkers for patient stratification. Our results establish PDOs as a novel platform for modeling neuroendocrine tumor biology, reveal microenvironment-driven plasticity in PPGLs, with potential translational relevance, and identify actionable vulnerabilities in a disease with few effective systemic therapies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.22.671868v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Time-Resolved Crystallography Reveals the Mechanisms of GTP hydrolysis for N-RAS and the Oncogenic Mutants G12C, G12V and Q61L</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lin, G., Zinser-Peniche, P., Zhou, X., Santiago, U., Kurnikov, I., Russi, S., Sharpe, M. E., Stegmann, D. P., Heinig, S. A., Goyal, R. K., Nagarajan, V., Calero, D., Vergara, S., Deiters, A., Burns, T. F., Kurnikova, M. G., Cohen, A. E., Calero, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The RAS family of small GTPases are molecular switches that convey downstream signals regulating cell proliferation, differentiation, and apoptosis1. The signaling competent GTP-bound RAS transitions to its inactive GDP-bound form through {gamma}-phosphate hydrolysis. Oncogenic RAS mutations hamper GTP hydrolysis and are present in up to 30% of all human cancers2. Structural studies of RAS proteins bound to non-hydrolysable GTP analogs have revealed snapshots of the enzyme in its possibly active form. Yet, the mechanism of GTP hydrolysis has not been structurally resolved3. To visualize this reaction in real time, we performed time-resolved crystallographic experiments employing a photolabile caged-GTP substrate. Fifty-seven distinctive reaction intermediates were captured during hydrolysis of a live GTP for N-RAS, the oncogenic mutants G12C, G12V and Q61L; and Y32R, a fast hydrolytic mutant. The reaction mechanisms and rates for the native and each of the mutants differed significantly; however, they shared common elements: an initially catalytically-defective open state, which transitions into the closed Michaelis complex state with solvent-assisted O3B-P{gamma} bond lengthening and breaking, followed by the release of the Mg2+ stabilized PO3-/PO4-3 species and unfolding of the switch loops. Given the conserved nature of GTP- and ATP-ases active sites, this structural work lays the basis to understand the universal mechanism of {gamma}-phosphate hydrolysis4. Furthermore, search for cryptic binding sites during GTP hydrolysis in G12C, G12V, and Q61L mutants reveals the presence of distinctive state-dependent binding pockets that could be targets for structure-based drug discovery of experimentally resolved intermediates states.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.21.670574v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">GraphVelo allows for accurate inference of multimodal velocities and molecular mechanisms for single cells.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yuhao Chen, Yan Zhang, Jiaqi Gan, Ke Ni, Ming Chen, Ivet Bahar, Jianhua Xing</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> RNA velocities and generalizations emerge as powerful approaches for extracting time-resolved information from high-throughput snapshot single-cell data. Yet, several inherent limitations restrict applying the approaches to genes not suitable for RNA velocity inference due to complex transcriptional dynamics, low expression, or lacking splicing dynamics, or data of non-transcriptomic modality. Here, we present GraphVelo, a graph-based machine learning procedure that uses as input the RNA velocities inferred from existing methods and infers velocity vectors lying in the tangent space of the low-dimensional manifold formed by the single cell data. GraphVelo preserves vector magnitude and direction information during transformations across different data representations. Tests on synthetic and experimental single-cell data, including viral-host interactome, multi-omics, and spatial genomics datasets demonstrate that GraphVelo, together with downstream generalized dynamo analyses, extends RNA velocities to multi-modal data and reveals quantitative nonlinear regulation relations between genes, virus, and host cells, and different layers of gene regulation.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40847023/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">GhostParser: A highly scalable phylogenomic approach for the identification of ghost introgression</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tolman, E. R., Suvorov, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> A growing body of empirical research shows that interspecific gene flow is a widespread biological force that shapes evolutionary histories across the Tree of Life. Computational approaches designed to detect introgression either employ full likelihood, including Bayesian frameworks, to directly estimate phylogenetic networks or utilize summary statistics derived directly from locus sequence alignments or estimated gene trees to map gene flow events onto the species tree. Many current methods currently have major shortcomings. The computationally scalable summary statistics and pseudo-likelihood-based techniques may provide erroneous results in the presence of so-called ghost introgression and rate variation between lineages. On the other hand, full likelihood methods are more accurate, but are not computationally tractable for large phylogenomic datasets. Here, we develop a novel summary statistic, based on tree heights of different gene tree topologies to reliably distinguish between sampled and ghost introgression events. We implemented this approach in the publicly accessible bioinformatic pipeline GhostParser. We demonstrate that GhostParser can accurately distinguish between scenarios of sampled and ghost introgression, even in the presence of rate variation between lineages. Our methodology concurs in accuracy with the full likelihood software Bayesian Phylogenetics and Phylogeography (BPP) on empirical datasets, and outperforms BPP in our simulation conditions, both in a small fraction of the computational time. We show that GhostParser is a scalable tool for the identification of different introgression patterns in phylogenomic datasets.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.21.671585v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A new subgenome of the Camelina genus reveals genome dominance is controlled by chromosomal proximity</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chaudhary, R., Koh, K., Gao, P., Perumal, S., Higgins, E., Horner, K., Robinson, S., Wang, Z., Eynck, C., Bandi, V., Sharpe, A. G., Parkin, I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Camelina sativa is an oilseed of the Brassicaceae, whose close relatives vary in ploidy number, providing a novel platform for studying plant genome evolution. The availability of diploid, tetraploid and hexaploid species of Camelina allow the evolutionary trajectory and fate of duplicated genes in the neopolyploid Camelina species to be elucidated. Here we report an improved assembly of the widely used C. sativa reference DH55 and three new genome assemblies of Camelina microcarpa; one tetraploid CN119243 (2n = 26), and two hexaploids with divergent chromosome numbers, Type 1 - CN119205 (2n=40) and Type 2 - CN120025 (2n=38). The tetraploid represents the first step in the evolutionary path to form C. sativa, while the hexaploids suggest three divergent lineages in the formation of higher ploidy Camelina species. The previously uncharacterized fourth subgenome found in C. microcarpa Type 2, although showing some homology to the C. sativa diploid progenitor genome, C. neglecta, showed numerous unique chromosomal rearrangements differentiating it from other subgenomes present in known Camelina species. Although this species was recently formed, the second subgenome showed gene expression dominance, which was in contrast to both 2n=40 Camelina species where the third subgenome was dominant. The expression dominance in Type 2 C. microcarpa contradicted the accepted two-step evolutionary process which led to the generation of related Brassicaceae species. However, the observed genome dominance in all Camelina species was negatively correlated with inter-subgenome chromatin interaction frequencies, suggesting that chromosome confirmation and proximity in the nucleus contributes to this mechanism of genome evolution. Despite the differences in genome structure, successful inter-specific hybridization provided evidence of chromosomal exchange between the divergent third sub-genomes of C. sativa and C. microcarpa Type 2, opening up a novel avenue to new diversity in the established oilseed.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.18.670935v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Adaptive Reconstruction of Pupil Dynamics During Blink Data Loss: A Physiologically-Informed Exponential Recovery and Noise Modelling Framework</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Khodami, M. A., Hosseini, S. M., Kireeva, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> This paper presents a novel, physiologically inspired framework for reconstructing missing pupil data during blink-induced interruptions. Pupillometry has become integral to studies of cognition, emotion, and neural function, yet the frequent data gaps caused by blinks can compromise the accuracy of inferences drawn from pupil measurements. This approach addresses these challenges by introducing an exponential recovery model that dynamically adjusts its time constant in proportion to blink duration, thus capturing the non-linear dynamics of pupil responses more effectively than traditional interpolation methods. A localised noise estimation procedure further enhances realism by incorporating both signal-dependent and baseline noise derived from statistical properties of the data surrounding each blink. Additionally, a Savitzky-Golay filter is selectively applied to the reconstructed intervals, preserving key physiological features while mitigating high-frequency artefacts. To facilitate the implementation of our framework, we introduce PRPIP, a Python package that implements our methodology. This work demonstrates the effectiveness of this method using empirical pupil data from visual tasks, illustrating that the reconstructed signals maintain physiological fidelity across a range of blink durations. These findings underscore the potential of combining physiological principles with data-driven noise modelling to generate robust, continuous pupil traces. By offering a more accurate reconstruction of pupil size, this framework stands to advance pupillometric research in fields ranging from cognitive psychology to clinical neuroscience.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.18.670842v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">IsoPairFinder: A tool for biochemical pathway discovery using stable isotope tracing metabolomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhou, Z., Liu, Y., Wang, M., Dodd, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Summary: The functional annotation of microbial genes lags far behind genome sequencing, leaving critical gaps in our knowledge of metabolic pathways. While integrating genetic manipulation with stable isotope tracing (SIT) metabolomics holds promise for pathway discovery, existing tools lack specialized capabilities for gene perturbation experiments. To address this need, we developed IsoPairFinder, a computational tool that identifies pathway intermediates by analyzing paired unlabeled (12C) and isotope-labeled (13C) metabolomics data from gene-edited microbes. By prioritizing substrate-specific feature pairs, IsoPairFinder efficiently prioritizes biologically relevant intermediates. Implemented as an open-source R package and integrated into the GNPS2 ecosystem, IsoPairFinder provides an accessible platform for the research community to accelerate novel pathway discovery and validation. Availability and implementation: IsoPairFinder is freely available as an open-source R package (https://github.com/DoddLab/IsoPairFinder) and as a web-based workflow in the GNPS2 ecosystem (https://gnps2.org/workflowinput?workflowname=isopairfinder_nextflow_workflow). Comprehensive tutorials are provided at https://doddlab.github.io/IsoPairFinder_Tutorials. These implementations support both command-line and graphical analysis workflows.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.18.670916v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">On the use of variational autoencoders for biomedical data integration</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pielies Avelli, M., Hernandez Medina, R., Webel, H. E., Rasmussen, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Variational Autoencoders (VAEs) are a widely used framework to integrate diverse biomedical data modalities, create representations that capture the underlying structure of the datasets, and obtain insights about the relations between variables. Here we describe how this is achieved from an empirical point of view in our VAE-based framework MOVE, providing an intuitive perspective on the inner workings of multimodal VAEs in biomedical contexts. We explore how the models' emerging dynamics shape their performance and how in silico perturbations can be leveraged to bring to light potential associations between variables. To do that, we extend our framework to handle perturbations of continuous variables, introduce a new approach to better capture associations between them, and create synthetic datasets to benchmark the proposed methods against well defined ground truth associations. We finally showcase our findings in a real biomedical scenario using a multimodal dataset of inflammatory bowel disease.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.18.670835v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Testing for positive selection in multi-copy gene families using a reconciliation approach</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pellen, Y., Privman, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Multi-copy gene families evolve through dynamic processes of duplication, loss, and sequence divergence, often exhibiting complex paralogy and orthology relationships that complicate the detection of adaptive evolution. Traditional dN/dS-based positive selection analyses are typically limited to single-copy genes, overlooking most of the adaptive signal in multi-copy families. We present a novel pipeline that integrates existing tools to map branch-specific positive selection events from gene trees onto species phylogenies. Our approach incorporates a normalization procedure that corrects for biases caused by variable branch lengths, reducing false enrichment signals on longer branches. Applied to the odorant receptor repertoires of ants, the framework successfully identified branches of the species tree enriched for positive selection, revealing clade-specific adaptive patterns obscured by conventional methods. This reconciliation-based strategy enables detection of adaptive hotspots in complex gene families, providing a versatile tool for linking molecular evolution to species-level diversification.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.18.670524v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Chromosome-scale Genome Assemblies of Two Allopolyploid Cuscuta Species Uncover Genomic Signatures of Parasitic Lifestyle and Polyploid Evolution</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Segawa, T., Yoshizumi, S., Toyonaga, H., Shiraishi, A., Sato, K., Yamabe, T., Takagi, M., Takagawa, M., Yokoyama, R., Itoh, T., Ono, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Dodders (Cuscuta spp.) is an obligated parasitic plant, which lost a large part of photosynthetic genes but gained host genes through parasitism-mediated horizontal gene transfer (HGT). Their migratory ecology across the world would contribute to complexity of the speciation via geographic isolation. Here we report the de novo genome assemblies of the two phylogenetically distinct dodders; C. chinensis (2n=4x=60) and C. campestris (2n=4x=60) that are classified into Clade B and Clade H of subgenus Grammica, respectively. Relatively low completeness of BUSCO genes ca.87% indicated progressive gene loss after evolution of parasitic lifestyle due to release from functional constraints as photosynthesis and organ development. Comparative genomics uncovered that both species are the genome size is completely different regardless of the same cytotypes and allopolyploid through the independent ancient hybridization between different parents. Various genomic rearrangements including 1) gene gain and loss events, 2) homoeologous recombination between two subgenomes and 3) the lineage-specific proliferation of transposable elements likely contribute to their genomic diversity and sexual isolation of the two lineages partly sharing their habitats. Our findings not only provide genomic basis to survey parental species for allopolyploidization but also help to understand unique speciation of parasitic dodders through these chromosomal natures.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.18.670985v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multi-omics reveals lipid metabolism, mitochondrial and extracellular matrix dysregulation of diabetic cardiomyopathy in the human heart</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lu, Q., Tang, S., Wu, Y., Chen, L., Liang, M., Fang, S., Chen, J., Wen, P., Jin, L., Yu, J., Jiao, F., Wu, Y., Jiang, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Diabetic cardiomyopathy (DbCM) is a condition characterized by myocardial dysfunction in diabetes. Although the clinical recognition of DbCM is increasing, its underlying molecular mechanisms remain poorly understood. We aimed to conduct a comprehensive multi-omics analysis of heart tissue from cardiomyopathy patients (both with and without diabetes), alongside tissue from healthy donors, by integrating transcriptomic, 4D-DIA proteomic, and full-spectrum widely targeted metabolomic data. Differential analyses revealed a significant metabolic pattern in patients with DbCM, characterized by increasing utilization of triglyceride-derived fatty acids. Correlation analyses highlighted impaired BNIP3L-mediated mitophagy as a potential contributor to the disruption of fatty acid metabolism and mitochondrial dysfunction in DbCM. Protein interaction and fibrotic trichrome analyses showed that patients with DbCM had impaired cardiac matrix repair compared to patients with cardiomyopathy without diabetes. These findings provide a system-level understanding of DbCM and reveal molecular signatures driving its progression, offering insights for targeted therapeutic strategies against diabetic heart disease.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.21.671651v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Whole genome sequencing reveals a novel Renibacterium salmoninarum lineage and suggests geographic endemism combined with anthropogenic spread in the North-East Atlantic area</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Villamil-Alonso, A., Moldal, T., Boe, C. A., Bragason, B. T., Nilsen, H., Olsen, A. B., Garseth, A. H., Gulla, S., Vendramin, N., Madsen, L., Cuenca, A., Lagesen, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Renibacterium salmoninarum is the causative agent of bacterial kidney disease (BKD) and has been isolated from northern European salmonid farms since the 1960s. The bacterium has been detected only sporadically in Norway during the last decades, but the country experienced several geographically spread outbreaks since December 2022. The phylogenomic relationships of R. salmoninarum isolates associated with the epizootics ongoing on the West Coast and in Mid-Norway were explored using a whole genome sequencing approach. A broad overview of the phylogeography of this pathogen was gained through sequencing and analysis of a collection of isolates from Norway (n=67), Iceland (n=12), Denmark (n=12), and the Faroe Islands (n=1), along with a collection of 109 publicly available sequences. We identified two distinct contemporary clades of R. salmoninarum causing BKD in Norway in the period 2022-2024. Both clades belong to the expanding, aquaculture-associated Lineage 1. The epidemiological picture appears consistent with contemporary aquaculture operations, raising questions on the effectiveness or practice of current biosecurity practices towards R. salmoninarum. Our work also describes a hitherto undescribed lineage (Lineage 3), predominantly from Iceland, where BKD is considered endemic. The detection of endemic reservoirs of R. salmoninarum in European water systems underscores the potential for ongoing pathogen circulation independent of acute outbreaks. This finding emphasizes the importance of further investigation of the mechanisms of pathogen persistence, particularly within environments related to aquaculture, where chronic infection reservoirs could compromise disease management and biosecurity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.18.670871v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Tracking blaKPC Plasmid Dissemination within and between Enterobacterales across Michigan Over a Decade</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shankar, C., Narayanan, D. B., Miihlbach, A., McNamara, S., Brennan, B., Miles-Jay, A., Blankenship, H. M., Bahr, A., Snitkin, E. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> blaKPC is endemic among Enterobacterales in the USA. While present on diverse plasmids, blaKPC burden is often associated with the clonal spread of multi-drug resistant (MDR) epidemic lineages. In this study we sought to determine the relative contributions of clonal spread and plasmid transfer to blaKPC burden across Michigan healthcare facilities over a decade. To this end we performed whole-genome sequencing of 1,058 KPC-producing isolates collected from 47 Michigan healthcare facilities between 2013 and 2022, including long-read sequencing for 527 isolates to enable precise plasmid tracking. Analysis with MOB-suite identified 64 distinct KPC plasmid types ("secondary clusters"), with the AK975 broad-host range plasmid being the most prevalent, found in 27% of isolates, spanning 20 species and 92 sequence types. Among genomes with AK975, 30% were from epidemic and 70% non-epidemic lineages, highlighting its broad role in regional blaKPC spread. Epidemic lineages of various species constituted 46% of the study population. Epidemic lineages differed in their primary plasmids, and even within epidemic lineages there were clonal expansions with distinct blaKPC plasmids, including in some cases AK975. These findings highlight two patterns of KPC spread: transmission of epidemic lineages harboring broad-range and lineage-specific KPC plasmids; and broader spread of AK975 among diverse species. Traditional surveillance studies often focus on common MDR lineages, potentially overlooking rare species and lineages that mediate the spread of plasmid-borne antimicrobial resistance (AMR) genes. Here we show how longitudinal studies tracking plasmids across species are essential to understand the pathways leading to AMR infections in hospitals.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.18.670900v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Actin Counters Geometry to Guide Plant Cell Division</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Goldy, C., Moulin, S., Shimizu, Y., Cerutti, G., Bayle, V., Grison, M., Jaillais, Y., Bouchez, D., Boutte, Y., Pastuglia, M., Andrey, P., Caillaud, M.-C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In multicellular organisms, cell division shapes tissue architecture, cell identity, and function. In walled organisms like plants, division plane orientation irreversibly defines tissue topology and is tightly regulated. While divisions often follow the shortest path enclosing cell volume, certain cells deviate, dividing perpendicular to the growth axis. Tissue-scale mechanical stress has been proposed to guide such orientation, but how these cues are integrated remains unclear. Here, we re-examine the role of the actin cytoskeleton in orienting cell division in Arabidopsis root epidermis. Combining cell biology, genetics, pharmacological treatments, 3D segmentation, and modelling, we show that actin is a central molecular actor required to establish cell division orientation against the geometrical rules, highlighting its role in integrating spatial information.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.18.670899v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Suite of Stains: Characterization of four fluorophores as complementary tools for visualizing neutral lipids in an extremophilic green alga</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Osmers, P., Boisvert, E.-J. Y., Boddy, C. N., Fogg, D. E., Cvetkovska, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding lipid metabolism in algae is critical to advancing our knowledge on fundamental algal physiology and for harnessing these organisms as platforms for the sustainable production of high-energy lipids. BODIPY is the most prevalently used fluorescent dye for the visualization of lipid droplets (LDs) in algae; however, its limitations warrant exploration of alternatives. Here we evaluate and compare four lipophilic fluorophores (BODIPY, DAF, Cou, DPAS) for their effectiveness in staining LDs in the extremophilic green alga Chlamydomonas priscui. We assess each dye photophysical properties, synthetic accessibility, LD specificity, cellular toxicity, and suitability for microscopy and flow cytometry. All four dyes successfully stain LDs, but their performance diverges under different experimental conditions. BODIPY permits long-term incubation allowing quantification in time-course studies but exhibits poor LD specificity and high susceptibility to photobleaching. DAF enables polarity-sensitive staining but is highly toxic on prolonged exposure or during cellular stress. Cou and DPAS yield strong LD-specific signals with low cytotoxicity, making them ideal for studies involving environmental stress. However, DPAS requires room-temperature incubation, pointing toward greater potential utility for non-extremophilic algae. These results expand the toolbox for lipid biotechnology research in extremophiles and underscore the importance of tailoring dye selection and experimental conditions to algal physiology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.18.670973v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Searching for Acrolein-Scavenging Compounds in Plants: Isoalliin from Onion as an Excellent Scavenger</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hada, A., Nozaki, C., Tamura, N., Matsui, K., Matsuoka, Y., Shibata, D., Mano, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Reactive carbonyl species (RCS), such as acrolein (Acr), are generated via the degradation of lipid peroxides and function both as signaling molecules and as cytotoxins downstream of reactive oxygen species (ROS). The aim of this study was to identify RCS-scavenging compounds, acting as secondary antioxidants, in plants. We established an HPLC-based method to sensitively evaluate the RCS-scavenging ability of compounds, using Acr as a representative RCS. Among 80% ethanol extracts of 46 angiosperm species, the highest scavenging activities were observed in garlic, spinach, avocado, broccoli, and lotus, representing a diverse range of plant families. The active ingredient from garlic cloves was identified as S-allyl cysteine sulfoxide (alliin), an amino acid typical of Allium species, which trapped up to two Acr molecules at its amino group. Notably, S-(1E)-propenyl cysteine sulfoxide (isoalliin), a structural analog of alliin and a major constituent of onion, showed even stronger Acr-scavenging activity. Isoalliin outperformed known Acr scavengers such as carnosine, epigallocatechin gallate, resveratrol, and hesperetin. These findings highlight isoalliin as a representative plant-derived RCS scavenger and emphasize the importance of S-alk(en)yl cysteine sulfoxides as potent Acr-scavenging compounds in plants. This study further suggests that structurally diverse RCS scavengers remain to be discovered among specialized metabolites in the plant kingdom.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.18.670978v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Augment rice ANNEXIN expression to counter planthopper NlAnnexin-like5 as an anti-virulence strategy against a major crop pest</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, X.-Y., Li, S., Opachaloemphan, C., Zhang, C.-X., Jiang, Y., he, s. y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The brown planthopper (BPH) is the most devastating insect pest in rice, posing a serious threat to global rice production. One attractive control strategy would be based on the understanding of the virulence mechanisms of BPH at the molecular level and then designing targeted methods to neutralize such mechanisms. Salivary proteins of BPH are important players in mediating rice-BPH interactions. Here, we describe a pivotal role of a watery saliva protein, NlAnnexin-like5 (NlANX5), in the rice-BPH interaction. RNA interference (RNAi) of NlANX5 greatly compromised BPH feeding performance and survival rate on rice plants. NlANX5-RNAi BPH triggered a rapid calcium ion influx in rice cells. The feeding and survival defects of NlANX5-RNAi BPH can be restored in NlANX5-expressing transgenic rice plants. NlANX5 targets rice annexin (OsANN) proteins, including OsANN2 and OsANN8. Further analysis with NlANX5 and OsANN2 as well as OsANN8 showed that NlANX5 displaces OsANN2 and OsANN8 from the rice cell membrane. The osann2 osann8 mutant rice plants are hypersusceptible to BPH infestation. In contrast, enhanced expression of OsANN2 and OsANN8 gene resulted in robust rice resistance against BPH. This study highlights a successful example of identifying and augmenting the expression of the host targets of a major BPH virulence effector as a promising anti-virulence strategy against an important crop pest.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.18.670976v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Lateral flow cell-free transcriptional assay for contaminant detection</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dujovne, M. V., Silva, C., Rocco, F. A., Ghiglieri, G., Riesgo, M., Capdevila, D. A., Peinetti, A. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cell free transcriptional biosensors are emerging as a powerful technology, offering enhanced capabilities for detecting chemical contaminants in settings where traditional analytical techniques fall short of societal needs. However, the sensitivity of these biosensors for many critical chemical contaminants often remains insufficient to meet regulatory detection limits, while maintaining portability and ensuring selectivity. In this work, an in vitro transcription (IVT) based lateral flow assay (LFA) is reported, which is 1000 times more sensitive than reported IVT portable detection methods. Through combining the amplification power of IVT with nucleic acid LFA-based avidity effects, we develop rapid, ultra-sensitive turn-on sensors. Our sensors meet regulatory detection limits for lead, a particularly challenging metal ion contaminant, in untreated spring water samples collected from a basin that exemplifies the environmental monitoring challenges. Additionally, we show that by optimizing the IVT conditions, we can tune the limit of detection to produce an on-off signal reaching the WHO regulation threshold and others. Finally, we evaluate the selectivity of different metal-responsive repressors in the context of in vitro biosensing and discuss how to identify potentially hazardous samples with an easy-to-use and point-of-care assay.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.20.670534v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">BOTany Methods: Accessible Automation for Plant Synthetic Biology</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Qiande, M., Lin, A., Larson, L., Voiniciuc, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Most members of the synthetic biology community, particularly plant scientists, lack access to liquid handling robots to scale up experiments, enhance reproducibility, and accelerate the Design, Build, Test, Learn cycle. Biofoundries enable high throughput data acquisition to train AI models and to develop new bioproducts, but they are capital-intensive to set up and not widely distributed. Entry-level, 3D-printed robots suffer from more affordable alternatives, but suffer from a shortage of validated protocols that can be modified without prior coding experience. To enhance access to biological automation, we developed a collection of modular BOTany Methods using Opentrons OT-2 robots to streamline the most common methods for molecular biology research and education. Our comprehensive workflow offers automation for a variety of procedures, ranging from simple but repetitive tasks (such as primer dilution and PCR setup) to more complex operations, including Plant Modular Cloning (MoClo), bacterial transformation, and plasmid extraction. Our BOTany Methods enable undergraduate students and other early career researchers to run designer experiments using table-based inputs, without editing the custom Phyton scripts. This pipeline enables end-to-end molecular cloning with minimal user intervention, enhancing throughput and traceability for synthetic biology applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.21.671538v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genetically encoded SpyTag enables modular AAV retargeting via SpyCatcher-fused ligands for targeted gene delivery.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Armbruster, A., Hoerner, M., Wagner, H. J., Fink-Straube, C., Weber, W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recombinant adeno-associated viral (rAAV) vectors are a leading platform of in vivo gene therapy, valued for their excellent safety, broad serotype diversity, and scalable production. Targeted delivery through capsid display of ligands holds great promise, yet current retargeting strategies often rely on extensive capsid re-engineering and restrict the use of ligands incompatible with intracellular expression systems. Here, we present a modular AAV retargeting platform that, for the first time, employs the SpyTag/SpyCatcher system via genetic integration into the AAV2 capsid. SpyTag is a small peptide that forms a covalent, irreversible bond with its protein partner, SpyCatcher, allowing site-specific ligand coupling under physiological conditions. Inserting SpyTag into surface-exposed capsid sites enabled post-assembly functionalization of AAVs with SpyCatcher-fused targeting proteins. As proof-of-concept, we used SpyCatcher fusions with designed ankyrin repeat proteins (DARPins) specific for EGFR, EpCAM, and HER2. This conferred highly specific transduction of corresponding cancer cell lines with minimal off-target activity. Therapeutic potential was demonstrated by delivering a suicide gene, inducing selective cancer cell killing upon prodrug administration. This one-fits-all platform allows rapid and flexible retargeting without significantly altering the underlying vectors genome or production process. It supports the incorporation of large or complex ligands not amenable to genetic fusion and facilitates high-throughput preclinical evaluation strategies. By uniting capsid engineering with modular ligand display, our approach provides a scalable and versatile framework for precision gene delivery, broadening the applicability of rAAV in both therapeutic and discovery settings.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.22.671696v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Controlled reassociation of multistranded, polycrossover DNA molecules into double helices</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kabbara, N., Anderson, L., Chandrasekaran, A. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Shape-changing DNA nanostructures have found applications in biosensing, drug delivery, cell modulation and data storage. A key aspect of this reconfiguration is the interaction of DNA nanostructures with other biomolecules or chemical stimuli such as pH and ionic conditions. Sequence-based nanostructure reconfiguration is largely achieved by strand displacement which is based on single stranded toeholds. In this work, we use sequence and temperature-controlled reassociation of one type of a DNA nanostructure into another structure. We demonstrate this strategy using the paranemic crossover (PX) DNA, a four-stranded structure with two adjacent double helical domains connected by six strand crossovers. In the presence of an anti-PX structure that is composed of strands that are each complementary to those in PX DNA, the structures reassociate at specific temperatures to form duplexes. Using the denaturing agent formamide, we decreased the temperature required for this reassociation. We extend the strategy to other polycrossover DNA molecules such as a double crossover motif (2 crossovers) and a juxtaposed DNA motif (4 crossovers), showing controlled reassociation of different DNA motifs into duplexes. Our study highlights the potential for DNA motifs to function as switchable molecular systems, offering new insights for responsive DNA-based materials and devices.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.22.671170v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Renewable Self-Folding Origami Constructed from Bioengineered Bacterial Cellulose</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tseo, Y., Guempel, M., Hogan, C., Hunter, I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> This work presents the first genetically engineered cellulose actuator where the structural basis for actuation is directly encoded in the producing organism's DNA. By modifying Komagataeibacter rhaeticus to secrete BslA protein and treating with NaOH, we create surface-activated bacterial cellulose that exhibits water contact angles 2.4x greater than and water retention 8.2x less than unmodified cellulose. Layering this hydrophobic BslA-activated cellulose with untreated bacterial cellulose produces a biofabricated actuator driven by differential strain during dehydration. The actuation follows the Timoshenko bilayer beam equation adapted for hydrogels and can be fabricated either by shaping mature pellicles or through direct growth in 3D-printed autoclavable molds. We demonstrate two applications with a self-folding bacterial cellulose origami crane and a biomimetic bacterial cellulose gripper capable of supporting more than 360 times its own weight. This approach represents a significant advance in sustainable soft robotics through the creation of fully renewable, biodegradable, and genetically programmable actuators.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.22.671751v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Paranemic cohesion of DNA molecules in different counter ions at room and physiological temperatures</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Anderson, L., Talbot, H., Chandrasekaran, A. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Isothermal assembly allows the construction of DNA nanostructures at constant moderate temperatures. Further, assembly of DNA nanostructures in different counter ions expands their utility in various applications. In this work, we demonstrate the isothermal assembly of paranemic crossover (PX) DNA motifs in magnesium (Mg2+), calcium (Ca2+) and strontium (Sr2+) at 20 deg C and 37 deg C. Further, we study how differences in design affect the assembly of 4-stranded and 2-stranded PX molecules.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.21.671517v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Construction and Functional Characterization of a Heterologous Quorum Sensing Circuit in Clostridium sporogenes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sadr, S., Zargar, B., Aucoin, M. G., Ingalls, B. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Quorum sensing (QS) is a bacterial communication mechanism that regulates gene expression in a population density-dependent manner. Substantial progress has been made in the engineering of QS systems in Gram-negative bacteria, but development of engineered QS systems in Gram positive bacteria remains limited. In this study, obligate anaerobic Gram-positive bacterium Clostridium sporogenes was engineered with the Staphylococcus aureus agr-QS system to enable density-dependent gene regulation. Using LC-MS/MS, we confirmed production of autoinducing peptides in the engineered C. sporogenes strain. A QS-regulated GFP reporter demonstrated activation of expression in response to both exogenous AIP addition and increasing cell density, confirming functional integration of the agr operon. A media refreshment experiment showed that replacing the culture supernatant delayed QS activation, highlighting the importance of signal accumulation. Moreover, we observed that a non-cognate AIP from another agr specificity group acts as a competitive antagonist, inhibiting gene expression under the QS promoter. To our knowledge, this study presents the first successful engineering of the agr quorum sensing system in an obligate anaerobe, expanding the synthetic biology toolkit and offers new opportunities for bacterial therapies and metabolic engineering.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.21.671393v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrating Human Genetics and Protective Genome Editing to Enable ADGRE2-Directed AML Therapy</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Keschner, Y., Etchin, J., Silva, M., Mager, H., Hoyt, H., Xavier-Ferrucio, J., Manalo, N., Halfond, A., Qiu, H., Wang, R., Lin, M. I., Ge, H. G., Scherer, J., Chakraborty, T., Lydeard, J. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Acute myeloid leukemia (AML) remains a major therapeutic challenge due to extensive disease heterogeneity and lack of cancer-specific antigens. ADGRE2 has emerged as a promising AML target with broad expression in AML patient blast and leukemic stem cell-enriched populations. However, comparable expression in healthy hematopoietic stem and progenitor cells (HSPCs) and myeloid lineages suggests a high susceptibility to on-target, off-tumor myelotoxicity with ADGRE2-targeted therapies. Guided by human genetics data identifying loss-of-function variants, we evaluated whether ADGRE2 is dispensable in hematopoietic stem cells as a protective approach for transplant-based shielding from ADGRE2-directed therapies. Using CRISPR-Cas9 and adenine base editors, we achieved high-efficiency ADGRE2 knockout (>94%) in HSPCs with corresponding protein loss without impairing cell viability, differentiation, and cytokine release in vitro, or long-term engraftment, multilineage differentiation, and persistence of gene editing in mouse xenografts. We also developed novel ADGRE2-specific chimeric antigen receptor (CAR) T cells that demonstrated potent cytotoxicity against AML cells, even at low antigen levels. Together, these findings establish ADGRE2 as a compelling AML target and provide a framework for hematopoietic stem cell transplant with protective gene editing to enable ADGRE2-directed immunotherapies while minimizing myelotoxicity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.21.671614v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">BACH2 dosage establishes the hierarchy of stemness and finetunes antitumor immunity in CAR T cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hu, T., Zhu, Z., Luo, Y., Wizzard, S., Hoar, J., Shinde, S. S., Yihunie, K., Yao, C., Wu, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Self-renewing stem-like T cells promote the efficacy of cancer immunotherapy and are a heterogeneous population with sub-lineages demonstrating different degrees of stemness. At the apex of this hierarchy is long-term (LT) stem-like T cells with the highest capacity of persistence, repopulation and response to immune checkpoint inhibitors (ICI). However, the pathway that establishes the hierarchy of stemness in chimeric antigen receptor (CAR) T cells and its role in antitumor efficacy of CAR T cells are unclear. Here, we demonstrate that LT stem-like differentiation and antitumor immunity of CAR T cells are dose-dependently regulated by BACH2. Pre-infusion LT stem-like CAR T cells showed epigenetic activation of BACH2 and superior antitumor response in mice and humans. After clearing tumor in vivo, LT stem-like T cells emerged as a CAR subset that transcriptionally and epigenetically upregulated BACH2 and downregulated TOX. Loss-of-function experiments revealed an essential role of BACH2 in the antitumor response of CAR T cells and the transcriptional program of LT stem-like CAR T cells. In exhaustion-prone GD2 CAR T cells with high tonic signaling, we showed that quantitative control of BACH2 protein level by a small molecule drug finetuned the degree of stemness and exhaustion in CAR T cells. Chemically inducing temporal BACH2 activation during manufacture of GD2 CAR T cells imprinted greater antitumor immunity against solid tumor after infusion. Together, we show that BACH2 dosage establishes the hierarchy of stem-like CAR T cells and can be temporally and tunably controlled in CAR T cells to optimize differentiation and antitumor immunity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.18.670909v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Pyrimethamine inhibits mutant NRF2 as a molecular glue</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xiong, Z., Paiboonrungruang, C., Wang, H., Subramaniyan, B., Bui-Linh, C., Li, Y., Li, H., Spitz, F., Chen, X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Esophageal squamous cell carcinoma (ESCC) is a deadly disease and one of the most aggressive cancers of the gastrointestinal tract. As a master transcription factor regulating the stress response, nuclear factor erythroid 2-related factor 2 (NRF2) is often mutated, loses negative regulation by Kelch-like ECH-associated protein 1 (KEAP1), becomes hyperactive, and thus causes chemo-radioresistance and poor survival in human ESCC. Our previous research identified pyrimethamine (PYR) as an NRF2 inhibitor and demonstrated that inhibition of dihydrofolate reductase (DHFR) is the major mechanism of its action. Based on these findings, PYR has advanced into a Phase I window-of-opportunity clinical trial. In this study, using isogenic ESCC cell lines, we aimed to investigate the phenotypic consequences of NRF2 activation in ESCC cells and further elucidate the mechanism of action of PYR. While overexpression of wild-type NRF2 (NRF2WT) promoted squamous differentiation and suppressed proliferation, expression of mutant NRF2 (NRF2Mut) produced mixed effects on differentiation markers and promoted proliferation. NRF2 activation reduced sensitivity to chemotherapy and radiation, whereas PYR treatment restored chemosensitivity and radiosensitivity in NRF2Mut ESCC cells. Co-immunoprecipitation and proximity ligation assays revealed that PYR selectively enhanced the interaction between NRF2W24C and KEAP1, but not between NRF2WT and KEAP1. Isothermal titration calorimetry (ITC) confirmed the direct binding of PYR to KEAP1 and surface plasmon resonance (SPR) showed that PYR modestly facilitated the interaction between a DLGW24C peptide and the Kelch fragment. Molecular docking suggested that PYR bound to a pocket within the Kelch domain near Arg415. In summary, NRF2 activation promotes cell proliferation and therapy resistance in ESCC in a context-dependent manner. PYR functions as a modest molecular glue that selectively restores KEAP1 binding to NRF2W24C, providing a potential therapeutic strategy for NRF2Mut ESCC.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.18.670902v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">O-mannosylation and protein maturation check-points represent therapeutic opportunities in BRAF fusion protein oncogenesis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Misek, S., Borgenvik, A., Christen, D., Kyrila, G., Zhang, A., Reel, S., Boisvert, M., Cai, K., Zhou, K., Gonzales, E., Goodale, A., LazodelaVega, L., Ragnoni, T., Zohra, K., Persky, N., Abid, T., Miglietta, E., Vakhrushev, S., Stumpe, M., Jones, J., Malinowski, S., Elsadek, L., Parikh, J., Condurat, A., Fultineer, A., Choi, S. H., Ozdemir, M., Eisenbies, Z., Lee, J., Novikov, D., Bahadur, S., Apfelbaum, A., Robinson, J., Tang, K., Maldera, A., Bondeson, D., Kwon, J., Vallurupalli, M., Jeon, H., Pal, S., Golub, T., Hahn, W., Fischer, E., Boehm, J., Dengjel, J., Clausen, H., Jabado, N., Milde</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Fusions between protein-coding genes are common oncogenic drivers across cancers, typically pairing a proto-oncogene with partner that does not independently drive cancer. In all therapeutically actionable fusions, the proto-oncogene is the drug target, the contributions to oncogenicity of the fusion partner have largely been ignored. We studied the role of BRAF fusion partners and found that they are necessary for transformation. In the setting of KIAA1549::BRAF, the most common fusion protein across brain tumors, we found that KIAA1549 is necessary for the oncogenicity of KIAA1549::BRAF and engenders a striking and specific dependency on the protein O-mannosyltransferase complex (POMT1/2). Specifically, we show that genetic silencing or pharmacologic inhibition of the protein O-mannosyltransferase complex (POMT1/2) reverses fusion-induced transformation, thereby representing a novel and MAPK independent therapeutic target. Furthermore, POMT1/2 is required to glycosylate and enable maturation of the K::B fu-sion protein. These findings represent a proof-of-concept for targeting the partners in oncogenic fusions as a potential cancer therapeutic strategy.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.18.670743v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Circulating proteins and metabolites panel for noninvasive preoperative diagnosis of epithelial ovarian cancer.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yan Jia, Li Yuan, Weijia Wen, Linna Chen, Xueyuan Zhao, Qiong Wu, Yan Liao, Caixia Shao, Chaoyun Pan, Chunyu Zhang, Shuzhong Yao</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Existing biomarkers for epithelial ovarian cancer (EOC) have demonstrated limited sensitivity and specificity. This study aimed to investigate plasma protein and metabolite characteristics of EOC and identify novel biomarker candidates for noninvasive diagnosis and differential diagnosis.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40846961/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">KYNU is a potential metabolic-related biomarker for nasopharyngeal carcinoma by Raman spectroscopy, metabolomics, and transcriptomics analysis.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ziman Wu, Haiyan Yang, Yafei Xu, Xiang Ji, Dayang Chen, Chuang Zhang, Mingjie Liang, Xinying Li, Xiuming Zhang, Dan Xiong</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Nasopharyngeal carcinoma (NPC) is a malignant tumor with high incidence in Southeast Asia and Southern China, characterized by difficulties in early diagnosis and high recurrence rates after treatment. Metabolic reprogramming plays a crucial role in the development and progression of tumors. In-depth studies on the metabolic characteristics and molecular mechanisms of NPC are essential to identify novel diagnostic and therapeutic targets.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40844682/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

