<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Monday, 07 July 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PIMENTO: A PrIMEr infereNce TOolkit to facilitate large-scale calling of amplicon sequence variants</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Atallah, C., Richardson, L., Beracochea, M., Finn, R. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce PIMENTO, a Python package which uses a dual-strategy approach for identifying primers that are present in sequencing reads to enable the large-scale calling of amplicon sequence variants from publicly available nucleotide datasets.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The calling of amplicon sequence variants from DNA metabarcoding data is a common method of revealing the taxonomic makeup of environmental samples. A significant hurdle to the large-scale calling of amplicon sequence variants from publicly available nucleotide datasets is the presence of primer sequences in reads, the removal of which is a necessary pre-processing step for this form of analysis. Further, as the details of which primers were used is rarely associated with the sequence records, there is a need for a method that can automatically infer the presence and identity of primers in sequencing data. In this work, we introduce PIMENTO, a Python package which uses a dual-strategy approach for identifying primers that are present in sequencing reads to enable their removal, and therefore facilitate amplicon sequence variant calling at scale.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.04.663168v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multi-omics time-series analysis in microbiome research: a systematic review</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sherwani, M. K., Ruuskanen, M., Feldner-Busztin, D., Firbas, P. N., Boza, G., Moreh, A., Borman, T., Erawijantari, P. P., Scheuring, I., Gopalakrishnan, S., Lahti, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A systematic literature review on data science methods for longitudinal multi-omics, with a particular focus on microbiome research.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent developments in data generation have opened up unprecedented insights into living systems. It has been recognized that integrating and characterizing temporal variation simultaneously across multiple scales, from specific molecular interactions to entire ecosystems, is crucial for uncovering biological mechanisms and understanding the emergence of complex phenotypes. With the increasing number of studies incorporating multi-omics data sampled over time, it has become clear that integrated approaches are pivotal for these efforts. However, standard data analytical practices in longitudinal multi-omics are still shaping up and many of the available methods have not yet been widely evaluated and adopted. Thus, we performed a systematic literature review on data science methods for longitudinal multi-omics, with a particular focus on microbiome research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.03.659054v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Structural motif search across the protein-universe with Folddisco</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kim, H., Kim, R. S., Mirdita, M., Steinegger, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed Folddisco, which overcomes this through an index of position-independent geometric features, including side-chain orientation, combined with a rarity-based scoring system, to detect protein structural motifs in large structure collections.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Detecting similar protein structural motifs, functionally crucial short 3D patterns, in large structure collections is computationally prohibitive. Therefore, we developed Folddisco, which overcomes this through an index of position-independent geometric features, including side-chain orientation, combined with a rarity-based scoring system. Folddisco indexes 53 million AFDB50 structures into 1.45 terabyte within 24 hours, enabling rapid detection of discontinuous or segment motifs. Folddisco is more accurate and storage-efficient than state-of-the-art methods, while being an order of magnitude faster. Folddisco is free software available at folddisco.foldseek.com and a webserver at https://search.foldseek.com/folddisco.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.06.663357v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Why do some predicted protein structures fold poorly? Benchmarking AlphaFold, ESMFold, and Boltz in maize</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Haley, O. C., Tibbs-Cortes, L., Hayford, R. K., Harding, S., Woodhouse, M. R., Cannon, E. K., Gardiner, J. M., Portwood, J. L., Sen, T. Z., Kim, H.-S., Andorf, C. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We benchmarked five structure-prediction programs for protein folding and found that plant proteins are underrepresented in sequence and structural datasets used to train these programs.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein structure prediction tools have significantly reduced the time and cost to generate protein structures and accelerated protein discovery and design. However, plant proteins are underrepresented in sequence and structural datasets used to train these programs. To quantify the downstream impact of this deficiency, we benchmarked five structure-prediction programs (AlphaFold 2, AlphaFold 3, ESMFold, Boltz-1, and Boltz-2) across 417 well-characterized Zea mays genes. These "classical" genes represent a set of well-studied genes with known genetic and phenotypic effects. We generated structures for each gene using these programs and compared how sequence, structural, and evolutionary conservation impacted the structures' confidence and geometric features. Proteins lacking conserved sequence and/or structural domains had on average 25% to 43% lower confidence scores than proteins having both domains. Proteome-wide phylostratigraphy revealed that species-specific proteins had substantially lower confidence scores than proteins conserved amongst angiosperms and Eukaryotes. Boltz-1 and ESMFold structures had the highest occurrence of structures with severe geometry issues, including overlapping atoms and unlikely bond angles. We also compared computational and experimental alignments of Arabidopsis, maize, rice, wheat, and soybean proteins from the Protein Data Bank and identified structures showing incongruences with experimental data. This study challenges the assertion that protein folding has been completely 'solved', and urges more investigation into benchmarking and standardized evaluation frameworks to improve model performance and assessment in agricultural crops.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.05.663230v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Product-stabilized filamentation by human glutamine synthetase allosterically tunes metabolic activity</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Greene, E., Muniz, R., Yamamura, H., Hoff, S. E., Bajaj, P., Lee, D. J., Thompson, E. M., Arada, A., Lee, G. M., Bonomi, M., Kollman, J. M., Fraser, J. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that human glutamine synthetase can reversibly polymerize into filaments aided by a composite binding site formed at the filament interface by the product, glutamine.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> To maintain metabolic homeostasis, enzymes must adapt to fluctuating nutrient levels through mechanisms beyond gene expression. Here, we demonstrate that human glutamine synthetase (GS) can reversibly polymerize into filaments aided by a composite binding site formed at the filament interface by the product, glutamine. Time-resolved cryo-electron microscopy (cryo-EM) confirms that glutamine binding stabilizes these filaments, which in turn exhibit reduced catalytic specificity for ammonia at physiological concentrations. This inhibition appears induced by a conformational change that remodulates the active site loop ensemble gating substrate entry. Metadynamics ensemble refinement revealed >10 [A] conformational range for the active site loop and that the loop is stabilized by transient contacts. This disorder is significant, as we show that the transient contacts which stabilize this loop in a closed conformation are essential for catalysis both in vitro and in cells. We propose that GS filament formation constitutes a negative-feedback mechanism, directly linking product concentration to the structural and functional remodeling of the enzyme.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.04.663231v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Generalizability of machine learning models for plant traits using hyperspectral reflectance data: The case of maize</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xu, R., Ferguson, J. N., Kromdijk, J., Nikoloski, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We use hyperspectral reflectance data to systematically compare the performance of representative ML models for different traits, including slow fluorescence kinetics whose predictability by hyperspectrals has not yet been investigated, (2) evaluate the ML model performance in prediction scenarios concerning unseen genotypes, unseen seasons,</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Hyperspectral reflectance provides rapid and precise phenotyping of plants in a non-destructive manner both in field and well-controlled settings. The resulting data have been used to devise machine learning (ML) models for paired measurements of different traits in diverse plants and crops. Yet, despite advances in using of hyperspectral data to reliably predict crop traits of interest, there are pressing issues concerning the training of ML models, the aggregation of data from crop field trials, and the generalizability of the models in different prediction settings. We collected hyperspectral reflectance data along with 25 anatomical, gas exchange, and chlorophyll fluorescence traits from 320 recombinant inbred lines of a maize Multi-Parent Advanced Generation Inter-Cross population grown across three consecutive seasons. We use these data to systematically: (1) compare the performance of representative ML models for different traits, including slow fluorescence kinetics whose predictability by hyperspectral data has not yet been investigated, (2) evaluate the ML model performance in prediction scenarios concerning unseen genotypes, unseen seasons, and the combination thereof, (3) investigate the effects of data aggregation of ML model performance. These problems are addressed in a rigorous nested cross-validation setting that provides a template for adequate assessment of performance of ML models for diverse crop traits considering the particularities of the experimental design.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.03.661070v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Secretory carrier membrane proteins assist with aquaporin trafficking in Arabidopsis.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jiang, Q., Vandorpe, M., fox, a. R., Vermeersch, M., Mylle, E., Cuadrado, A. F., Kraus, J., Liu, H., Eeckhout, D., Navarre, C., Courtoy, A., Jacobs, T. B., Dragwidge, J. M., De Smet, I., Pleskot, R., Chaumont, F., Van Damme, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose that the drought tolerance phenotype of the scamp135 mutants is at least partially caused by altered PIP trafficking and link it to the secretory carrier membrane proteins.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Secretory carrier membrane proteins are evolutionarily conserved multi-spanning transmembrane proteins. In animal cells, they function in secretion, endocytosis and autophagy. Knowledge on their role in plants is restricted to heterologous localization and trafficking experiments as well as to genetic perturbation experiments in lily and poplar. Here, we analysed the five members of the Arabidopsis SCAMP family. Using SCAMP5, the most prominently expressed member of the family as hallmark, we identified three conserved tyrosine motifs that assist in anterograde transport from the Trans Golgi Network to the plasma membrane, while N-terminally located NPF motifs are crucial for endocytosis. SCAMPs dimerize both at the plasma membrane and endosomes, and dimerization is required for internalization. Functionally, SCAMP5 interacts with several plasma membrane associated aquaporins (PIPs), and is required for their intracellular trafficking as mCherry-PIP2;1 accumulated differentially and specifically in the root meristematic vacuoles independent of the endocytic or secretory trafficking capacity of SCAMP5. Triple and quintuple scamp mutants show only mild developmental delay under standard growth conditions, but are more tolerant to drought. In conclusion, our research links SCAMP proteins to PIPs and we propose that the drought tolerance phenotype of the scamp135 mutants is at least partially caused by altered PIP trafficking.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.03.662988v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ERAD machinery controls the conditional turnover of PIN-LIKES in plants</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Noura, S., Ferreira Da Silva Santos, J., Feraru, E., Hoernstein, S. N. W., Feraru, M. I., Montero-Morales, L., Roessling, A.-K., Scheuring, D., Strasser, R., Huesgen, P. F., Waidmann, S., Kleine-Vehn, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Auxin is a crucial phytohormone that regulates plant development and facilitates dynamic responses to environmental changes through subcellular control mechanisms.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Auxin is a crucial phytohormone that regulates plant development and facilitates dynamic responses to environmental changes through subcellular control mechanisms. PIN-LIKES (PILS) are auxin transport facilitators at the endoplasmic reticulum (ER) that mediate nuclear auxin abundance and signalling. While the posttranslational regulation of PILS is important for acclimating growth responses, the molecular mechanisms involved remain largely unknown. This study demonstrates that components of the ER-associated degradation (ERAD) machinery regulate the proteasome-dependent degradation of functional PILS proteins under non-stressed conditions. We further reveal that both internal and external signals utilise the ERAD complex to differentially modulate the turnover rates of PILS proteins. Our findings uncover an additional physiological role of the ERAD complex in regulating PILS protein turnover. This finding uncovers the interplay between protein homeostasis at the ER and growth regulation, opening new molecular avenues into how plants acclimate to internal and external cues.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.05.663279v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Evidence for plastome loss in the holoparasitic Mystropetalaceae</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yu, R., Jost, M., Wanke, S., Bruy, D., Li, P., Nickrent, D. L., Zhou, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Plastome loss is an extremely unusual phenomenon in land plants, even for those that lose their photosynthetic ability. Here we propose plastite loss in holoparasitic Mystropetalaceae (Santalales), which consists of three monotypic genera: D</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Plastome loss is an extremely unusual phenomenon in land plants, even for those that lose their photosynthetic ability. To date, evidence for plastome loss has only been presented for two holoparasitic angiosperm lineages: two genera (Rafflesia and Sapria) of Rafflesiaceae and one section (Subulatae) of Cuscuta (Convolvulaceae). Here we propose plastome loss in holoparasitic Mystropetalaceae (Santalales), which consists of three monotypic genera: Dactylanthus, Hachettea, and Mystropetalon. Using Illumina DNA sequencing data, we successfully assembled the complete mitochondrial genomes of all three genera; however, we failed to recover any plastome sequences from them using reference-based and de novo assembly approaches. Illumina RNA sequencing data for Hachettea revealed the absence of plastid gene transcripts and losses of many nuclear-encoded genes with plastid function, including genes involved in DNA transcription, translation and protein degradation. Convergent loss of these key genes has also been documented in other plastome-less non-photosynthetic plant lineages. Taken together, we provided compelling evidence for plastome loss in the family Mystropetalaceae.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.03.662983v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multipartite coevolution shapes plant apoplastic immunity against rice blast fungus</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Takeda, T., Shimizu, M., Kodan, A., Utsushi, H., Kanzaki, E., Natsume, S., Imai, T., Oikawa, K., Abe, A., Terauchi, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that a GBP of the causal agent for blast fungus, Magnaporthe oryzae, is involved in multipartite coevolutionary arms race with rice (Oryza sativa) and propose a general defense mechanism against fungal pathogens.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Beta-1,3-glucan-binding proteins (GBPs) play crucial roles in cell wall integrity, host-pathogen interactions, and stress responses in fungi, making them essential for fungal survival and virulence. Here, we show that a GBP of the causal agent for blast fungus, Magnaporthe oryzae, is involved in multipartite coevolutionary arms race with rice (Oryza sativa). The rice thaumatin-like pathogenesis-related protein 5 (OsPR5) binds to M. oryzae GBP and sequesters it from the fungal cell wall to confer immunity, but this interaction is counteracted by thaumatin-binding proteins secreted by the fungus to suppress apoplastic immunity. Moreover, a rice receptor kinase located at the cell surface has evolved a thaumatin-like domain to detect the fungal GBP and activate immunity. Our findings reveal that GBPs and thaumatin-domain-containing proteins are engaged in intricate coevolutionary dynamics in the rice apoplast. Given the conservation of thaumatin across plants, we propose a general defense mechanism against fungal pathogens.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.03.663104v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Zero-shot antibody design in a 24-well plate</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chai Discovery,, Boitreaud, J., Dent, J., Geisz, D., McPartlon, M., Meier, J., Qiao, Z., Rogozhnikov, A., Rollins, N., Wollenhaupt, P., Wu, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Chai-2, a multimodal generative model that achieves a 16% hit rate in fully de novo antibody design, representing an over 100-fold improvement compared to previous computational methods.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Despite breakthroughs in protein design enabled by artificial intelligence, reliably designing functional antibodies from scratch has remained an elusive challenge. Recent works show promise but still require large-scale experimental screening of thousands to millions of designs to reliably identify hits. In this work, we introduce Chai-2, a multimodal generative model that achieves a 16% hit rate in fully de novo antibody design, representing an over 100-fold improvement compared to previous computational methods. We prompt Chai-2 to design <=20 antibodies or nanobodies to 52 diverse targets, completing the workflow from AI design to wet-lab validation in under two weeks. Crucially, none of these targets have a preexisting antibody or nanobody binder in the Protein Data Bank. Remarkably, in just a single round of experimental testing, we find at least one successful hit for 50% of targets, often with strong affinities and favorable drug-like profiles. Beyond antibody design, Chai-2 achieves a 68% wet-lab success rate in miniprotein design -- routinely yielding picomolar binders. The high success rate of Chai-2 enables rapid experimental validation and characterization of novel antibodies in under two weeks, paving the way toward a new era of rapid and precise atomic-level molecular engineering.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.05.663018v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CloneCoordinate: Open-source software for collaborative DNA construction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jeon, E., Shen, Z., Christ, S., Qi, E., Fan, I., Lee, N., Morgan, O., Ohl, M., Millson, D., Laker, M., Pierson, A., Villegas Garcia, E., Zhang, Y., Choi, A., Iyengar, A., Kim, R., Lee, J., Niedeck, L., Oeien, V., Rashid, M., Seetharaman, N., Singh, A., Soble, D., Yu, J., Yu, K., Berdy, S., Chang, E., Kitazono, R., Ortiz, S., Taylor, D., Thuronyi, B. W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> CloneCoordinate is a free, open-source electronic laboratory notebook specifically designed for cloning and fully implemented in Google Sheets.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Custom DNA constructs have never been more common or important in the life sciences. Many researchers therefore devote substantial time and effort to molecular cloning, aided by abundant computer-aided design tools. However, support for managing and documenting the construction process, and for effectively handling and reducing the frequency of setbacks, is lacking. To address this need, we developed CloneCoordinate, a free, open-source electronic laboratory notebook specifically designed for cloning and fully implemented in Google Sheets. By maintaining a real-time, automatically prioritized task list, a uniform physical sample inventory, and standardized data structures, CloneCoordinate enables productive, collaborative cloning for individuals or teams. We demonstrate how the information captured by CloneCoordinate can be leveraged to troubleshoot assembly problems and provide data-driven insights into cloning efficiency. CloneCoordinate offers a new and uniquely accessible model for how to carry out, and iteratively improve on, real-world DNA assembly.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.04.663221v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Production of Glycolic acid through Whole-Cell Bioconversion from PET Monomer Ethylene Glycol Using Engineered Corynebacterium glutamicum</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mohammad, G. R., Wong, F. T., Lim, Y. H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a biotransformation platform using Corynebacterium glutamicum to produce glycolic acid (GA) from EG.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In the last decade, the global warming and plastic pollution issue have driven research on developing a more sustainable platform for chemicals production from alternative feedstocks. Ethylene glycol (EG), a monomer of polyethylene terephthalate (PET) plastic, has a potential to become a renewable substrate for microbial production of value-added chemicals. This study presents a biotransformation platform using Corynebacterium glutamicum to produce glycolic acid (GA) from EG. C. glutamicum was engineered to express a heterologous EG oxidation pathway. Subsequent promoter engineering yielded strain FA4, producing 10.6 g/L GA from EG in 48 h. Implementation of a two-stage biotransformation strategy using resting cells further enhanced the GA production, reaching a cumulative GA titer of 98.8 g/L after a 72-h production. Finally, applying this platform to a simulated EG mixture from PET-degradation achieved a cumulative GA titer of 67.3 g/L over 72 h, highlighting the potential for valorizing plastic waste through this biotransformation platform. These findings establish C. glutamicum as an efficient biotransformation chassis for sustainable GA production from EG and offer a promising route for PET waste valorization into value-added chemicals.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.06.663337v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

