<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Monday, 14 July 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">TATAT: a containerized software for generating annotated coding transcriptomes from raw RNA-seq data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Brown, A. J., Seifert, S. N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed TATAT, a modular, Dockerized software that contains all the tools necessary to generate an annotated coding transcriptome from raw RNA-seq data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: Many transcriptome creation workflows are not standardized, are difficult to install or share, prone to breaking as dependencies update or cease to be maintained and are resource intensive. Due to a lack of authoritative literature, many also overlook potentially important steps, such as thinning contig over-assembly or identifying transcript consensus across samples, which reduce resource demands during annotation and increase the accuracy of final transcripts. Results: We developed TATAT, a modular, Dockerized software that contains all the tools necessary to generate an annotated coding transcriptome from raw RNA-seq data. The tools remain in a static state and can be coordinated with bash and python scripts provided therein, making TATAT a standardized, reproducible workflow that can easily be shared and installed. We preferentially incorporate tools that are not only accurate, but are fast and require less RAM, and subsequently show TATAT can generate a comprehensive transcriptome for a non-model organism, the Egyptian rousette bat (Rousettus aegyptiacus), in ~8 hours in a high-performance computing (HPC) environment. Availability and implementation: The TATAT code, instructions, and tutorial are available at https://github.com/viralemergence/tatat.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.09.663867v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Orthogonal chemical genomics approaches reveal genomic targets for increasing anaerobic chemical tolerance in Zymomonas mobilis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Eckmann, J., Enright, A. L., Davies, M., Whelan, E., Myers, K. S., Robinson, M. L., Banta, A. B., Lal, P. B., Coon, J. J., Sato, T. K., Kiley, P. J., Peters, J. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify all known genes in the cy tochrome bc1 and cytochrome c synthesis pathway as potential targets for engineering resistance to phenolic acids under anaerobic conditions, and show that exposure of Zymomonas mobilis to ferulic acid causes substantial remodeling of the cell envelope prote</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genetically-engineered microbes have the potential to increase efficiency in the bioeconomy by overcoming growth-limiting production stress. Screens of gene perturbation libraries against production stressors can identify high-value engineering targets, but follow-up experiments needed to guard against false positives are slow and resource-intensive. In principle, the use of orthogonal gene perturbation approaches could increase recovery of true positives over false positives because the strengths of one technique compensate for the weaknesses of the other, but, in practice, two parallel screens are rarely performed at the genome-scale. Here, we screen genome-scale CRISPRi (CRISPR interference) knockdown and TnSeq (transposon insertion sequencing) libraries of the bioenergy-relevant Alphaproteobacterium, Zymomonas mobilis, against growth inhibitors commonly found in deconstructed plant material. Integrating data from the two gene perturbation techniques, we established an approach for defining engineering targets with high specificity. This allowed us to identify all known genes in the cytochrome bc1 and cytochrome c synthesis pathway as potential targets for engineering resistance to phenolic acids under anaerobic conditions, a subset of which we validated using precise gene deletions. Strikingly, this finding is specific to the cytochrome bc1 and cytochrome c pathway and does not extend to other branches of the electron transport chain. We further show that exposure of Z. mobilis to ferulic acid causes substantial remodeling of the cell envelope proteome, as well as the downregulation of TonB-dependent transporters. Our work provides a generalizable strategy for identifying high-value engineering targets from gene perturbation screens that is broadly applicable.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.09.663894v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Telomere-to-telomere assemblies reveal complex adaptive variation of 3-ketoacyl-CoA-synthases in Populus trichocarpa likely driven by helitrons</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kainer, D., Martin, S., Hopp, D., Mosher, S., Tschaplinski, T. J., Hyatt, P. D., Martin, M. Z., Leboldus, J. M., Sondreli, K. L., Busby, P. E., Shu, M., Barry, K., Schmutz, J., Furches, A., Zhao, N., Jacobson, D., Chen, J.-G., Pavicic, M., Ranjan, P., Muchero, W., Tuskan, G. A., Garvin, M. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Long-read sequencing and telomere-to-telomere assembles revealed large-effect loci critical to genetic studies that are unattainable from short-reads, such as helitrons.</p><p style="font-size: 16px; text-align: justify;">Background The model woody plant Populus trichocarpa displays an atypical alkene-diverse wax cuticle likely driven by copy number variation (CNV) of 3-ketoacyl-CoA synthases (KCS), which has been difficult to confirm based on short-read assemblies. New long-read sequencing provides opportunities to develop telomere-to-telomere resources to detect cryptic variation, including CNVs, which are currently missed in traditional analyses. Integrating this information can improve genomic prediction for breeding and provide insights into the evolutionary basis of important traits. Results Our analysis of 78 telomere-to-telomere long-read haplotypes identified more than twice as many KCS genes as previously reported, along with numerous intragenic non-synonymous substitutions. Random forest predictive models highlighted the importance of Potri.010G079500 in producing very long chain alkenes; however, its absence did not predict previously reported alkene-deficient phenotypes. Instead, alkene levels are best predicted by the combinations of KCS copies. Amino acid substitutions clustered around ligand and donor binding pockets, suggesting they contribute to differing wax cuticle composition. Finally, each KCS gene and copy was linked to a helitron transposon. A phylogenetic analysis indicates they are the evolutionary mechanism for generating KCS tandem arrays. Conclusions Long-read sequencing and telomere-to-telomere assembles revealed large-effect loci critical to genetic studies that are unattainable from short-reads. These approaches also have the potential to reveal novel insights into genome structure and function, such as the helitrons identified here. Our results highlight that, given current challenges in annotation and assembly, detailed and focused long-read sequences are key to interpreting complex genomic regions that contain tandem copy number variants.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.10.664019v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SCOPE: Revealing Hidden Mechanisms in Phenotypic Screens Through Target and Pathway Enrichment</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kapoor, A., Kelleher, K., Underhill, S., Jain, S., Harvey, B. K., Henderson, M. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed SCOPE (Screening Compound Ontology for Pathway Enrichment), a KNIME-based computational framework that resolves the molecular drivers of phenotypic activity by linking compound-level screening data to annotated targets and pathways.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Phenotypic screening enables discovery of small molecules without requiring predefined targets, but mechanistic interpretation remains challenging due to polypharmacology and pathway complexity. We developed SCOPE (Screening Compound Ontology for Pathway Enrichment), a KNIME-based computational framework that resolves the molecular drivers of phenotypic activity by linking compound-level screening data to annotated targets and pathways. SCOPE integrates multi-source target annotations and performs statistical enrichment to identify shared mechanisms of action. Applied to a high-throughput screen for modulators of ER-stress induced secretion of endoplasmic reticulum (ER) resident proteins, a process known as exodosis, SCOPE identified calcium signaling as the most enriched KEGG pathway without prior biological context. Target enrichment revealed G protein-coupled receptors (GPCRs) involved in inositol 1,4,5-trisphosphate receptors (IP3Rs)- mediated signaling, with widespread antagonism among hit compounds implicating this pathway in the regulation of exodosis. Notably, SCOPE uncovered a novel role for the histamine receptor HRH1, which was validated by RNAi knockdown and pharmacological inhibition, implicating HRH1 as a potential therapeutic target in ER stress-related disorders. These results highlight SCOPE's potential to deconvolute phenotypic screens and uncover actionable mechanisms in complex cellular systems.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.11.664427v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Learned Geometry, Predicted Binding: Structurally-BasedPrediction of Peptide:MHC Binding Using AlphaFold 3 EnablesCD4 T Cell Epitope Prediction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lahouel, K., Mulazimuglu, M., Soria-Bustos, J., Bates, K., Kelley, E. J., Woods, L. J., Upadhyaya, K. M., Acevedo, G. R., Penisson, S., Munini, M., Variani, E., Feeney, M. E., Altin, J., Tomasetti, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present MHCIIFold-GNN, an entirely orthogonal solution to the CD4 T cell epitope prediction problem that combines three new elements: (i) a highly-multiplexed peptide:MHCII binding assay, (ii) generalizable structural modeling using</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The accurate prediction of T cell epitope peptides within proteins of interest has a wide range of applications, but is complicated by the multiple determinants of antigenicity, the polymorphism of the Major Histocompatibility Complex (MHC) locus, and the great diversity of possible peptide antigens. Leading in silico methods use a variety of statistical approaches to learn from sequences identified through both in vitro MHC binding and peptide elution studies, but their performance remains imperfect, particularly for MHC II-restricted responses. Here we present MHCIIFold-GNN, an entirely orthogonal solution to this problem that combines three new elements: (i) a highly-multiplexed peptide:MHCII binding assay, (ii) generalizable structural modeling using AlphaFold3, and (iii) transfer learning with a Graph-based Neural Network. Trained exclusively on newly-generated in vitro binders, we show that MHCIIFold-GNN enables state-of-the-art prediction of CD4 T cell epitopes presented by diverse MHC II proteins, on par with non-structural methods that rely on much larger datasets including naturally-processed ligands. Moreover, when MHCIIFold-GNN and a leading non-structural method are combined, we observe unparalleled performance on a held-out test set (11 % boost), underscoring the orthogonality of the methods. These results highlight the power of a new class of structure-informed approaches to the CD4 T cell epitope prediction problem.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.10.664203v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Souporcell3: Robust Demultiplexing for High-Donor Single-Cell RNA-seq Datasets</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Weerakoon, M., Vu, H., Behboudi, R., Heaton, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present an enhanced version of souporcell capable of demultiplexing up to 64 donors, with improved accuracy and scalability.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation Accurate demultiplexing of pooled single-cell RNA-seq (scRNAseq) data is critical for large-scale studies. However, existing methods like vireo, while effective up to ~16 donors, often struggle with poor clustering due to local optima as donor numbers rise. In high-donor scenarios, overlapping genotypes, a dense genotype space, and increased doublet formation make demultiplexing challenging, requiring methods that are robust to sparse, high-dimensional data and maintain reliable accuracy even as sample complexity grows. Results We present an enhanced version of souporcell capable of demultiplexing up to 64 donors. The method uses 10x merge for initialization, K-Harmonic Means for robust clustering, and iterative refinement with reinitialization of low-quality clusters and locking of high-quality ones. Compared to vireo, vireo with overclustering, and the original souporcell, our approach completely eliminates duplicate clusters and achieves consistently high Adjusted Rand Index (ARI) scores across various doublet rates, demonstrating improved accuracy and scalability.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.10.664218v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AlphaFlex: Accuracy modeling of protein multiple conformations via predicted flexible residues</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ge, L., Cui, X., Zhao, K., Zhou, X., Zhang, Y., Zhang, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present AlphaFlex, an innovative framework that integrates flexible residue prediction with a directed masking strategy for multiple sequence alignments.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding protein conformational dynamics is critical for elucidating disease mechanisms and facilitating structure-based drug design. While AlphaFold2-based methods have advanced static structure prediction, modeling multiple conformation states with both accuracy and efficiency remains challenging. We present AlphaFlex, an innovative framework that integrates flexible residue prediction with a directed masking strategy for multiple sequence alignments. By selectively relaxing co-evolutionary constraints in dynamic regions, AlphaFlex enables accurate prediction of biologically relevant states while preserving structural fidelity. Benchmarking on 69 apo-holo pairs from the CoDNaS dataset, including enzymes, binding proteins, and chaperones, reveals AlphaFlex's superior performance in both flexible residue identification and multiple conformation prediction. Comparative analysis against AF-Cluster, AF2-conformations, and AlphaFlow demonstrates AlphaFlex's significant advantage, achieving the highest success rate (42%) in accurately predicting apo/holo structures (TM-score >0.95 for both states) while consistently outperforming these methods in reproducing both apo and holo conformations. Notably, AlphaFlex maintains robust performance for membrane proteins, resolving inward- and outward-facing conformations with equal fidelity. Case studies further reveal its unique ability to sample physically plausible intermediate states along transition pathways. These results establish AlphaFlex as a pivotal computational tool for mapping protein conformational landscapes, with promising potential to accelerate structure-based therapeutic development.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.11.664327v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">EZpred: improving deep learning-based enzyme function prediction using unlabeled sequence homologs</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, C., Liu, Q., Freddolino, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose EZpred, the first deep learning model to use unlabeled sequence homologs for protein function prediction.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Features extracted from sequence homologs significantly enhance the accuracy of deep learning-based protein structure prediction. Indeed, models such as AlphaFold, which extracts features from sequence homologs, generally produce more accurate protein structures compared to single sequence-based methods like ESMfold. In contrast, features from sequence homologs are seldom employed for deep learning-based protein function prediction. Although a small number of models also incorporate function labels from sequence homologs, they cannot utilize features extracted from sequence homologs that lack function labels. To address this gap, we propose EZpred, which is the first deep learning model to use unlabeled sequence homologs for protein function prediction. Starting with the target sequence and homologs identified by MMseqs2, EZpred extracts sequence features using the ESMC protein language model. These features are then fed into a deep learning model to predict the Enzyme Commission (EC) numbers of the target protein. For 753 enzymes, the F1-score of EZpred EC number prediction is 4% higher than a similar model that does not use sequence homologs and at least 10% higher that state-of-the-art EC number prediction models. These results demonstrate the strong positive impact of sequence homologs in deep learning-based enzyme function prediction.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.09.663945v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Bridging Molecular Simulations and Experimental Observables: Predicting FRET Signals and Conformational Transitions Using MD and Deep Learning in Cas9</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Leyva, A. G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A method to predict single-molecule Forster Resonance Energy Transfer efficiency from structural features derived from molecular dynamics simulations .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> This study applies machine learning to predict single-molecule Forster Resonance Energy Transfer (smFRET) efficiency from structural features derived from molecular dynamics simulations. Using Cas9 as a model system, inter-residue distances, center-of-mass displacements, radius of gyration, interdomain angles, and linker flexibility were extracted and used to train Hidden Markov Models and Long Short-Term Memory networks. These models were applied to characterize transitions between conformational states associated with Cas9 activation. Predicted FRET efficiencies and state transitions were compared to experimental smFRET data, with qualitative similarities in transition patterns and histograms. Due to differences in simulation and experimental timescales, direct comparison of kinetics was not performed. This approach explores the relationship between simulated structural dynamics and FRET-relevant observables, providing a framework for further studies that integrate computational models with experimental conformational analysis. The method may contribute to understanding how structural features influence conformational transitions in dynamic proteins.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.10.664034v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Hierarchical Robust Linear Model for Cryo-EM Map Analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tu, I.-P., Zheng, S.-C., Lien, Y.-H., Lin, S. H., Lin, P.-C., Chang, W.-H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a hierarchical robust linear model to estimate key atom-specific parameters, specifically the amplitude and width of Gaussian functions, that are utilized to assess the consistency between paired cryo-EM maps and their corresponding atomic models, or to improve their refinement.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cryo-electron microscopy (cryo-EM) has become a pivotal tool for determining the atomic structures of biological macromolecules. In this paper, we propose a hierarchical robust linear (HRL) model to estimate key atom-specific parameters, specifically the amplitude and width of Gaussian functions, that are utilized to assess the consistency between paired cryo-EM maps and their corresponding atomic models, or to improve their refinement. Our HRL modeling leverages minimum density power divergence estimates (MDPDE) to construct a heteroscedastic framework that could potentially mitigate the influence of outliers. We demonstrate the robustness of our method through both simulation studies and real data applications, showing its effectiveness in reducing the impact of outliers and ensuring reliable parameter estimates. Applying our HRL model to analyze real cryo-EM data from human apoferritin (PDB ID: 6Z6U, EMDB ID: 11103) reveals that the Gaussian parameters remain stable across most amino acids, with nitrogen atoms consistently exhibiting lower amplitude and width estimates than expected based on the commonly used Gaussian modeling. These findings highlight the need for a more systematic analysis of paired cryo-EM map and atomic model from the EMDB and PDB to further understand the atom-specific properties in cryo-EM data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.10.664269v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A nuclear CobW/WW-domain factor represses the CO2-concentrating mechanism in the green alga Chlamydomonas reinhardtii</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shimamura, D., Yasuda, J., Yamahara, Y., Nakano, H., Ozawa, S.-I., Tokutsu, R., Yamagami, A., Matsushita, T., Takahashi, Y., Nakano, T., Fukuzawa, H., Yamano, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify a nuclear repressor of the CO2-concentrating mechanism in the green alga Chlamydomonas reinhardtii and propose a regulatory mechanism potentially linking zinc-dependent protein chemistry to CCM gene repression.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Microalgae induce a CO2-concentrating mechanism (CCM) to maintain photosynthesis when dissolved CO2 is limited, but how this energy-intensive system is suppressed when CO2 levels rise has remained unclear. The CCM consumes 15-30% of photosynthetically-generated ATP, making its regulation critical for cellular energy balance. Here, we identify a nuclear repressor of the CCM in the green alga Chlamydomonas reinhardtii. A pull-down screen for interacting partners of the master activator CCM1/CIA5 revealed an uncharacterized protein that co-purifies with CCM1 even after high-salt washes. This protein, designated CCM1-binding protein 1 (CBP1), combines a CobW/CobW_C GTP-binding metallochaperone module with a WW domain characteristic of protein-protein interactions. CBP1 colocalizes with CCM1 in the nucleus regardless of CO2 conditions, and the two proteins interact in vivo. CRISPR/Cas9-based disruption of CBP1 does not affect growth or CCM induction under CO2 limitation but derepresses 27 of 41 CCM1-dependent low-CO2 inducible genes under high-CO2 conditions. These include the periplasmic and intracellular carbonic anhydrases (CAH1, LCIB) and inorganic carbon transporters/channels (LCIA, LCI1, BST1, BST3). Consistently, cbp1 mutants accumulate higher levels of CAH1 and LCIB proteins and exhibit a 40% increase in inorganic carbon affinity under high-CO2 conditions; this phenotype is rescued by CBP1 complementation or by acetazolamide treatment. These results demonstrate that CBP1 prevents unnecessary CCM activity when CO2 is abundant, acting upstream of both transporter/channel and carbonic anhydrase modules. Our findings suggest a regulatory mechanism potentially linking zinc-dependent protein chemistry to CCM gene repression, providing insights into energy-efficient CO2 sensing in aquatic photosynthetic organisms.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.09.663896v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An extended Œ≤-finger motif is a near-universal feature of kinase fusion proteins</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Powell, O. R., Gilbert, D., Wulff, B. B. H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the KFPs that confer disease-resistance in wheat and barley share an extended {beta}-finger motif, a feature that originated in the Poaceae, and show that elevated transcript abundance is a hallmark of disease-resistant activity.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Kinase fusion proteins (KFPs) are immune receptors conferring disease resistance in wheat and barley. We found that those with a confirmed function typically share an extended {beta}-finger motif, a feature that originated ~98 million years ago in the Poaceae. Moreover, these functionally active KFPs are among the most highly expressed within a plant's full KFP repertoire, suggesting that elevated transcript abundance is a hallmark of disease-resistance activity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.14.664643v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Protein Abundance Inference via Expectation Maximization in Fluorosequencing</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kipen, J., Smith, M. B., Blom, T., Zhou, S. B., Marcotte, E. M., Jalden, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a probabilistic framework that adapts expectation maximization to the fluorosequencing measurement process, estimating relative protein abundances with peptide inference results delivered by previously developed peptide-classification tools.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Fluorosequencing produces millions of single-peptide reads, yet a principled strategy for converting these data into quantitative protein abundances has been lacking. We introduce a probabilistic framework that adapts expectation maximization to the fluorosequencing measurement process, estimating relative protein abundances with peptide inference results delivered by previously developed peptide-classification tools. The algorithm iteratively updates protein abundances, maximising the likelihood of the observed reads by obtaining more accurate protein abundance estimations. We first assess performance on simulated five-protein mixtures that reflect realistic labelling and system errors. A simple Python implementation processes one million reads in under ten seconds on a standard workstation and lowers the mean absolute error in relative abundance by more than an order of magnitude compared with a uniform-abundance guess, demonstrating robustness in protein inference for small-scale settings. Scalability is then evaluated with simulations of the complete human proteome (20 642 proteins). Ten million reads are processed in less than four hours on a NVIDIA DGX system using one Tesla V100 GPU, confirming that the method remains tractable at proteome scale. Using error rates characteristic of current fluorosequencing, the algorithm produces marginal improvements in relative abundance accuracy. However, when error rates were artificially lowered, estimation error decreased significantly. This result suggests that improvements in fluorosequencing chemistry could directly translate into substantially more accurate quantitative proteomics with this computational framework. Together, these results establish EM-based inference as a scalable model-driven bridge between peptide-level classification and protein-level quantification in fluorosequencing, laying computational groundwork for high-throughput single-molecule proteomics. Furthermore, the proposed protein inference framework can also be used as a refinement step within other inference methods, enhancing their protein abundance estimates.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.10.664057v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Assessing large-scale genomic language models in predicting personal gene expression: promises and limitations</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Li, S., Luo, R., Huang, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A framework to benchmark large-scale genomic language models for predicting personal gene expression .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Large-scale genomic language models (gLMs) hold promise for modeling gene regulation, yet their ability to predict personal gene expression remains largely unexplored. We developed a framework, gLM2X-Tower, to benchmark gLMs and sequence-to-function (S2F) models on this task with paired personal genome-transcriptome data. With individual-level training, we found that similar to S2F models (e.g., AlphaGenome), gLMs (e.g., Evo2) remain incapable of predicting the inter-person variability on held-out genes. However, such training improves prediction for seen genes in new individuals, particularly by gLMs, highlighting the potential applications in few-shot settings like for rare variants.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.09.664024v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Predicting Biological Activity from Biosynthetic Gene Clusters Using Neural Networks.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hemant Goyat, Dalwinder Singh, Sunaina Paliyal, Shrikant Mantri</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a new tool, NPBdetect, that can detect multiple bioactivities and has been designed through rigorous experiments.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Microorganisms such as bacteria and fungi have been used for natural products that translate to drugs. However, assessing the bioactivity of extract from culture to identify novel natural molecules remains a strenuous process due to the cumbersome order of production, purification, and assaying. Thus, extensive genome mining of microbiomes is underway to identify biosynthetic gene clusters or BGCs that can be profiled as particular natural products, and computational methods have been developed to address this problem using machine learning. However, existing tools are ineffective due to a small training data set, dependence on old genome mining tools, lack of relevant genomic descriptors, and prevalent class imbalance. This work presents a new tool, NPBdetect, that can detect multiple bioactivities and has been designed through rigorous experiments. First, we composed a larger training set using the MIBiG database and a test set through literature mining to build and assess the model, respectively. Second, the latest antiSMASH genome mining tool was used to obtain BGCs and introduced new sequence-based descriptors. Third, neural networks are used to build the model by dealing with class imbalance issues through the class weighting technique. Finally, we compared the NPBdetect tool with an existing tool to show its efficacy and real-world utility in detecting several bioactivities with high confidence.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40524326/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

