<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Wednesday, 18 June 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PRCFX-DT: a new graph-based approach for feature selection and classification of genomic sequences.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Amin Khodaei, Sania Eskandari, Hadi Sharifi, Behzad Mozaffari-Tazehkand</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> In recent years, viral diseases have exhibited a significant incidence of infections and fatalities.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In recent years, viral diseases have exhibited a significant incidence of infections and fatalities. The analysis of viral genomic sequences can be efficacious in evaluating the present and potentially forthcoming condition of viruses. Considering the importance of the internal structure of the cell and the nucleotide sequences within it, analyzing nucleotide sequences can provide a range of discussable features. On the other hand, it has been demonstrated that the use of graph algorithms and machine learning in the analysis and examination of virus samples and even viral variants can yield beneficial results.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40528202/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multimodal CustOmics: A unified and interpretable multi-task deep learning framework for multimodal integrative data analysis in oncology.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hakim Benkirane, Maria Vakalopoulou, David Planchard, Julien Adam, Ken Olaussen, Stefan Michiels, Paul-Henry Cournède</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel deep-learning-based approach designed to represent multi-omics & histopathology data for precision medicine in a readily interpretable manner.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Characterizing cancer presents a delicate challenge as it involves deciphering complex biological interactions within the tumor's microenvironment. Clinical trials often provide histology images and molecular profiling of tumors, which can help understand these interactions. Despite recent advances in representing multimodal data for weakly supervised tasks in the medical domain, achieving a coherent and interpretable fusion of whole slide images and multi-omics data is still a challenge. Each modality operates at distinct biological levels, introducing substantial correlations between and within data sources. In response to these challenges, we propose a novel deep-learning-based approach designed to represent multi-omics & histopathology data for precision medicine in a readily interpretable manner. While our approach demonstrates superior performance compared to state-of-the-art methods across multiple test cases, it also deals with incomplete and missing data in a robust manner. It extracts various scores characterizing the activity of each modality and their interactions at the pathway and gene levels. The strength of our method lies in its capacity to unravel pathway activation through multimodal relationships and to extend enrichment analysis to spatial data for supervised tasks. We showcase its predictive capacity and interpretation scores by extensively exploring multiple TCGA datasets and validation cohorts. The method opens new perspectives in understanding the complex relationships between multimodal pathological genomic data in different cancer types and is publicly available on Github.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40526743/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">miRNA-Mediated Regulation of Gene Expression During Early Activation in Jurkat Cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mukherjee, P., Leal-Calvo, T., Ferguson, L., Cate, J. H. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified 9 differentially expressed miRNAs that are differentially expression (DE) during early T cell activation, with most changes occurring as early as 5 hours post-activation, with translational upregulation predominating over downregulation.</p><p style="font-size: 16px; text-align: justify;">Background: T cell activation induces substantial changes in gene expression by rapidly increasing transcription and translation. Additionally, microRNAs play a crucial role in regulating protein expression in T cell physiology, adding a layer of complexity by fine-tuning protein levels. While various miRNAs have been implicated in T cell function, a systematic analysis of differentially expressed miRNAs during early T cell activation and identification of their mRNA targets remains mostly unknown. Results: We investigated dynamic changes in global gene expression during early T cell activation using a multi-omics approach combining small RNA-seq, mRNA-seq and ribosome profiling. Our results show that most differential expression changes occur by 5 hours post-activation, with translational upregulation predominating over downregulation. From 5 to 12 hours, we observed modest transcriptional and translational reprogramming. We identified 9 miRNAs that are differentially expressed (DE) during early activation, with most changes occurring as early as 5 hours. We calculated translation efficiency (TE) and classified genes based on changes in both mRNA abundance and ribosome-protected fragments (RPFs). By integrating TE and miRNA expression data, we examined the relationship between TE group- specific regulation patterns and the number of miRNA binding sites. Interestingly, rather than observing a uniform downregulation of targets with 4 or more predicted DE miRNA binding sites, we found distinct regulatory patterns that varied with both activation time point and TE category. Conclusions: Our data provide new insights into how genes associated with key events in T cell activation such as translation, cell proliferation, and immune signaling are regulated at both the transcriptional and translational levels. The observation that most regulatory changes occur within the first 5 hours post-activation highlights the rapid and coordinated nature of T cell responses. The differential patterns of target regulation, based on translation efficiency groups and miRNA binding site density, suggest a context-dependent role for miRNAs in shaping protein output. Future experiments will be required to functionally validate specific miRNA--target interactions and to explore their relevance in primary T cells in vivo. This study also lays the groundwork for identifying miRNA-based regulatory circuits for therapeutic modulation of T cell activity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.15.659805v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A eukaryote without tRNA introns</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Van Hoof, A., Furuta, T., Arur, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that some nematode species have lost the introns in Tyr-GUA, Ile-UAU and Leu-CAA tRNAs, and one species, Levipalatum texanum, completely lacks tRNA introns.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> One of the striking characteristics of eukaryotic genomes is the presence of three types of introns: spliceosomal introns, tRNA introns, and a unique intron in the XBP1 mRNA. Exceptional eukaryotic genomes that lack spliceosomal or XBP1 introns have been described. However, tRNA introns and the tRNA endonuclease that is required for their splicing are thought to be universal in eukaryotes. The introns in three tRNAs are widely conserved across Metazoa: Tyr-GUA, Ile-UAU and Leu-CAA. This study shows that some nematode species have lost the introns in Tyr-GUA and Ile-UAU tRNAs, and one species, Levipalatum texanum, completely lacks tRNA introns. The loss of the intron from Leu-CAA tRNA is accompanied by an unusual A-C mismatched basepair in the anticodon stem loop and a triplication of a tRNA deaminase that could potentially restore basepairing. These changes may be an adaptation to the loss of the intron. L. texanum also lacks the tRNA endonuclease, one of two enzymes required for tRNA splicing. The other key enzyme in tRNA splicing, tRNA ligase, is bifunctional and is also required for XBP1 mRNA splicing. L. texanum retains tRNA ligase and the XBP1 intron. This eukaryote without tRNA introns has the potential to be a valuable tool for disentangling the functions of tRNA splicing, XBP1 splicing and tRNA modification enzymes, and is the only animal known to have lost one of the three intron types.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.17.660180v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SARS-CoV-2 ORF8 sequence conservation and mutational analysis -- insight into the influence of dataset size on identifying top mutations</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kandwal, S., Cmelo, I., Fayne, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We have performed conservation and mutational analysis on SARS-CoV-2 ORF8 protein sequences to identify the conserved and mutated residues and identified top mutations across each of them.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Given how quickly the SARS-CoV-2 virus mutates, the COVID-19 pandemic has been a major source of concern. The ORF8 accessory protein is one such protein, which is reported to have undergone many mutations. This makes ORF8 an intriguing protein to investigate how these mutations might play a role in overall ORF8 activity. In this study, we have performed conservation and mutational analysis on SARS-CoV-2 ORF8 protein sequences to identify the conserved and mutated residues. We have also split the ORF8 sequence data into SARS-CoV-2 variant datasets to further identify top mutations across each of them. The mutated and conserved residues were visualised on the available structure of ORF8 to highlight the conserved and mutated sites, which might hold some biological significance. Finally, our study also investigated the significance of sequence dataset size in capturing top mutations following multiple sequence alignments.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.16.659836v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Super-giga and tiny orchid genomes illuminate evolution of Orchidaceae</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lan, S., Liu, K.-W., Li, Z., Hsiao, Y.-Y., Liu, Y., Zhang, D., Zhao, X., Sun, W.-H., Liu, D.-K., Huang, M.-Z., Zhou, C.-Y., Zeng, M., Wang, L., Zheng, R., Zhao, Z., Gao, X., Lai, J.-H., Yeh, K.-L., Zhang, L.-S., Chen, J., Ma, X., Li, Y., Chen, D., Li, J., Zhang, M.-M., He, X., Huang, Y., Zhang, C., Wu, X., Chen, C., Ma, L., Zhai, J., Ai, Y., Li, M.-H., Zhou, Y., Zhou, Z., Wu, S., Zhao, K., Guan, Y., Tu, X.-D., Zeng, D., Ji, X., Liu, N., Zou, S., Chen, Y.-Y., Lin, S.-T., Su, W.-Y., Wang, Z.-W., Luo, Y.-B., Zhang, W., Guo, Y.-Y., Tian, Y.-Q., Zou, L.-H., Lv, X., Zhuo, X., Zhu, J., Peng, D.-H., Y</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We generated chromosome-level reference genomes for two orchids using a combination of PacBio, Illumina, and Hi-C sequencing, Cypripedium singchii has the largest genome and chromosomes among the sequenced species so far, with a genome size of 43.19 Gb</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Orchidaceae (orchids) is commonly known as one of the largest families of seed plants, and grow in an extensive range of habitats worldwide. In the present study, we generated chromosome-level reference genomes for two orchids using a combination of PacBio, Illumina, and Hi-C sequencing, Cypripedium singchii has the largest genome and chromosomes among the sequenced species so far, with a genome size of 43.19 Gb (1C) with ten chromosomes, and Apostasia fujianica has the smallest known genome and chromosomes in Orchidaceae, with a genome size of 340.90 Mb (1C) with 35 chromosomes. We predicted a total of 32,412 and 21,724 protein-coding genes for C. singchii and A. fujianica, respectively. The overall BUSCO score was 85.01% for C. singchii and 91.80% in A. fujianica. Based on protein-coding sequences from 55 conserved single-copy families across 21 plant species, we constructed a high-confidence phylogenetic tree and estimated the divergence times. The high-quality genomes of super-giga and tiny orchids offer key insight for future evolutionary researches.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.13.659439v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Inference of multi-enhancer interactions in T lymphocytes using Hi-Cociety</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yoon, S., Vahedi, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Hi-Cociety - a graph-based computational framework and accompanying R package that infers 3D enhancer communities directly from 3D chromatin data, without relying on histone modification or chromatin accessibility measurements.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Three-dimensional (3D) enhancer communities are key regulators of gene expression, shaping cell fate decisions and contributing to disease pathogenesis. Assays such as H3K27ac HiChIP have been used to map enhancer-enhancer interactions and define enhancer communities; however, their reliance on antibody-based enrichment restricts scalability and cross-cell-type applicability. In contrast, Hi-C provides an unbiased, genomewide view of chromatin architecture but lacks direct annotation of regulatory elements, limiting its utility for enhancer-focused analyses. To bridge this gap, we introduce Hi-Cociety - a graph-based computational framework and accompanying R package that infers 3D enhancer communities directly from Hi-C data, without relying on histone modification or chromatin accessibility measurements. Hi-Cociety constructs a network of significant interactions and applies clustering algorithms to define chromatin interaction modules. Applying Hi-Cociety to Hi-C measurements in T lymphocytes, we identified highly connected modules enriched for active transcription, chromatin accessibility, and histone acetylation. Notably, modules identified in T cells pinpoint critical genes central to T cell biology. Hi-Cociety also detects cell-type-specific differences in chromatin organization, highlighting dynamic regulatory rewiring across T cell states. Our findings underscore the importance of network properties - connectivity, transitivity, and centrality - in shaping gene regulation through 3D genome organization. Hi-Cociety provides a scalable and versatile tool for mapping enhancer communities at scale, advancing our understanding of immune cell identity and the regulatory logic encoded in 3D chromatin structure.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.12.659372v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CrebA regulation of secretory capacity: Genome-wide transcription profiling coupled with in vivo DNA binding studies</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Andrew, D. J., Jackson, D. M., Peng, D., Shinde, S., Holenarasipura, A., Cahan, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that CrebA binds far more genes than it regulates, that SPCGs are the major class of genes regulated by Creb a transcription factor, and that the different methods for capturing regulated transcripts identify overlapping but distinct sets of Creba targets.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> DNA binding assays, expression analyses, and binding site mutagenesis revealed that the Drosophila CrebA transcription factor (TF) boosts secretory capacity in the embryonic salivary gland (SG) through direct regulation of secretory pathway component genes (SPCGs). The mammalian orthologues of CrebA, the Creb3L-family of leucine zipper TFs, not only activate SPCG expression in a variety of mammalian tissues but can also activate SPCG expression in Drosophila embryos, suggesting a highly conserved role for this family of proteins in boosting secretory capacity. However, in vivo assays reveal that CrebA binds far more genes than it regulates, and it remains unclear what distinguishes functional binding. It is also unclear if CrebA is the major factor driving SPCG gene expression in all Drosophila embryonic tissues and/or if CrebA also regulates other tissue-specific functions. Thus, we did single cell RNA sequencing (scRNA-seq) of wild-type (WT) and CrebA null embryos to explore the relationship between CrebA binding and gene regulation. We find that CrebA binds the proximal promoters of its targets, that SPCGs are the major class of genes regulated by CrebA across tissues, and that CrebA is sufficient to activate SPCG expression even in cells that do not normally express the protein. A comparison of scRNA-Seq to other methods for capturing regulated transcripts reveals that the different methodologies identify overlapping but distinct sets of CrebA targets.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.12.659381v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Disentangling the role of chromosome inversions in the structure of flat oyster (Ostrea edulis): origin, adaptation and phylogeography</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sambade, I. M., Hlordzi, V., Blanco, A., Hermida, M., Vera, M., Culloty, S., Bean, T., Lynch, S., Martinez, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified three large chromosomal inversions in the European flat oyster (Ostrea edulis), a species of ecological and commercial relevance, and identified the genetic features, frequency, and evolutionary significance of the inversions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Chromosomal inversions are large-scale structural variants that disrupt recombination and contribute to population differentiation and local adaptation. In this study, we characterized the genomic features, frequency, and evolutionary significance of chromosomal inversions in the European flat oyster (Ostrea edulis), a species of ecological and commercial relevance. Using a low-density SNP array (~5K), we obtained confident genotyping data across the native European distribution of the species. The breakpoints of the inversions at chromosome (C) 2, C5 and C8, spanning 27, 7 and 35 Mb, respectively, were precisely mapped by combining linkage disequilibrium in a large sample and long-read sequencing. This enabled us to identify the SNPs and genes within the inversions for comparative population genomics and functional evaluation. A north to south cline was observed for the three inversions, although with low-moderate correlation between them. Strong signals for divergent selection were detected for the three inversions, although with specific profiles suggestive of adaptive peaks around highly divergent SNPs across the inversion length. The three inversions seem to be old considering genetic diversity and differentiation between arrangements. Phylogeographic events and adaptation to the current environment seem underlying the structural pattern observed. Gene content and enrichment analyses within inversions suggest that these polymorphisms may play a role in the immune system and response to different stressors including parasite resistance. Our results provide new insights into the genomic architecture of adaptation in O. edulis and highlight the potential of structural variants as targets for conservation and selective breeding.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.12.659260v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SurfacOmics: an R shiny application integrating variable Feature Selection for gene biomarker discovery using Elastic-Net Regularization</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tripathi, S., Kajla, P., Abbasi, B. A., Kota, K. P., Bailey, A., Varma, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a user-friendly R application, SurfacOmics, which allows the user to perform biomarker identification via penalized regression algorithms, which enables researchers to choose a crucial binary variable, such as treatment or group, from the study design metadata to anticipate potential biomarkers.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Affordable sequencing technologies have resulted in a rapid rise in genomic and proteomic data. As a result, a massive amount of data is being analyzed, and the outcomes must be summarized as relevant clinical biomarkers. One of the major challenges is enabling wet-lab researchers to make meaningful inferences from the data, even in the absence of expertise in pipeline development and statistical training. We present a user-friendly R shiny application, SurfacOmics, which allows the user to perform biomarker identification via penalized regression algorithms. It also enables researchers to choose a crucial binary variable, such as treatment or group, from the study design metadata to anticipate potential biomarkers. We have introduced a novel concept of scoring scheme to characterize potential biomarkers for prediction, prioritization, and ranking. The tool integrates Gene Ontology information for sub-cellular localization together with a manually curated knowledgebase to provide valuable insights on biological processes, molecular functions and antibody resources, offering a comprehensive view in a single interface.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.12.659259v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Biological Function Assignment Across Taxonomic Levels in Mass-Spectrometry-Based Metaproteomics via a Modified Expectation Maximization Algorithm</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Alves, G., Ogurtsov, A. Y., Yu, Y.-K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A major challenge in mass-spectrometry-based metaproteomics is accurately identifying and quantifying biological functions across the full taxonomic lineage of microorganisms.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> A major challenge in mass-spectrometry-based metaproteomics is accurately identifying and quantifying biological functions across the full taxonomic lineage of microorganisms. This issue stems from what we refer to as the "shared confidently identified peptide problem". To address this issue, most metaproteomics tools rely on the lowest common ancestor (LCA) algorithm to assign biological functions, which often leads to incomplete biological function assignments across the full taxonomic lineage of identified microorganisms. To overcome this limitation, we implemented an expectation-maximization (EM) algorithm, along with a biological function database, within MiCId workflow. Using synthetic datasets, our study demonstrates that the enhanced MiCId workflow achieves better control over false discoveries and improved accuracy in microorganism identification and biomass estimation compared to Unipept and MetaGOmics. Additionally, the updated MiCId offers improved accuracy and better control of false discoveries in biological function identification compared to Unipept, along with reliable computation of function abundances across the full taxonomic lineage of identified microorganisms. Reanalyzing human oral and gut microbiome datasets using the enhanced MiCId workflow, we show that the results are consistent with those reported in the original publications, which were analyzed using the Galaxy-P platform with MEGAN5 and the MetaPro-IQ approach with Unipept, respectively.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.12.659309v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Transcriptomic analysis of genotypes derived from Rosa wichurana unveils molecular mechanisms associated with quantitative resistance to Diplocarpon rosae</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lambelin, L., Thouroude, T., Jeauffre, J., Chameau, J., Vilfroy, C., Boursier, C., Aubourg, S., Pelletier, S., Lopez Arias, D. C., Hibrand-Saint Oyant, L., Soufflet-Freslon, V., Foucher, F., Paillard, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A hybrid of Rosa wichurana and Rosa chinensis 'Old Blush' exhibits quantitative resistance to black spot disease, and the molecular mechanisms of this resistance are better understood.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Black spot disease, caused by the hemibiotrophic fungus Diplocarpon rosae, is a major foliar disease of garden roses. Resistant cultivars offer an alternative to fungicides, but the genetic basis of resistance is not well known. Understanding these mechanisms is essential for the effective and durable deployment of resistant cultivars. A hybrid of Rosa wichurana (RW) exhibits quantitative resistance to black spot. Analysis of an F1 progeny from a cross between RW and the susceptible cultivar Rosa chinensis 'Old Blush' showed that RW's resistance mainly involves two Quantitative Trait Loci (QTLs), on linkage groups 3 (B3) and 5 (B5). This study aimed to better characterize the molecular mechanisms underlying these QTLs. RNA sequencing was performed on inoculated and non-inoculated samples of RW and four F1 genotypes with different QTL combinations, at 0, 3, and 5 days post-inoculation (dpi). The comparison of inoculated and non-inoculated samples revealed that most genes were differentially expressed at 3 dpi. For all the genotypes, we observed shifts in expression of genes involved in plant defense, as well as genes participating in hormonal pathways and calcium-mediated signaling. Genotypes harboring QTL B3 showed "classic" defense responses involving pathogen recognition, signaling, ROS production, callose deposition, and localized cell death. In contrast, QTL B5 was associated with few shared DEGs, making it harder to define its role. The quantitative nature of RW's resistance may result in this complex regulation of gene expression.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.13.659350v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Discovery of iridoid cyclase completes the iridoid pathway in asterids</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Colinas, M., Tymen, C., Wood, J. C., David, A., Wurlitzer, J., Morweiser, C., Gase, K., Alam, R. M., Titchiner, G. R., Hamilton, J. P., Heinicke, S., Dirks, R. P., Lopes, A. A., Caputi, L., Buell, C. R., O'Connor, S. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the iridoid biosynthesis pathway in asterid flowering plants involves the cyclization of a reactive enol intermediate to form the two major iridoids stereoisomers found in plants.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Iridoids are specialized monoterpenes ancestral to asterid flowering plants (Albach et al, 2001; Stull et al, 2018). Iridoids play key roles in plant defense and are also essential precursors for pharmacologically important alkaloids (Dobler et al, 2011; Eisner, 1964). The biosynthesis of all iridoids involves the cyclization of a reactive enol intermediate. While this cyclization occurs spontaneously at low yields, it has long been hypothesized that a dedicated enzyme is involved in this process (Geu-Flores et al, 2012; Lichman et al, 2019b). Here, we report the discovery of asterid iridoid cyclases (ICYC). We show that these enzymes catalyze cyclization of the reactive intermediate to form the two major iridoid stereoisomers found in plants. Our work uncovers the last missing key step in the otherwise well-characterized iridoid biosynthesis pathway in asterids. This discovery unlocks the possibility to generate previously inaccessible iridoid stereoisomers, which will enable metabolic engineering for the sustainable production of valuable iridoid and iridoid-derived compounds.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.12.659277v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">RNAGym: Large-scale Benchmarks for RNA Fitness and Structure Prediction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Arora, R., Angelo, M., Choe, C. A., Shearer, C., Kollasch, A., Qu, F., Weitzman, R., Gazizov, A., Gurev, S., Xie, E., Marks, D. S., Notin, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce RNAGym, a large-scale benchmarking framework specifically designed for three core tasks - RNA fitness, secondary structure, and tertiary structure prediction.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding RNA structure and predicting the functional consequences of mutations are fundamental challenges in computational biology with broad implications for therapeutic development and synthetic biology. Current evaluation of machine learning-based RNA models suffers from disparate experimental datasets and inconsistent performance assessments across different RNA families. To address these challenges, we introduce RNAGym, a large-scale benchmarking framework specifically designed for three core tasks - RNA fitness, secondary structure, and tertiary structure prediction. The framework integrates extensive datasets, including 70 standardized deep mutational scanning assays covering over a million mutations across diverse RNA types; 901k chemical-mapping reactivity profiles for secondary structure; and 215 diverse tertiary structures curated from the PDB. RNAGym is designed to facilitate a systematic comparison of RNA models, offering an essential resource to enhance the understanding and development of these models.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.16.660049v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A generalized and efficient approach for complete mRNA design improves translation, stability and specificity</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Riley, A. T., Vlasity, M., Huang, J. Z., Becicka, W. M., Wong, W. W., Grinstaff, M. W., Green, A. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel, model-informed mRNA design framework that unifies the strengths of predictive neural networks with the vast quantities of diverse RNA data that has been accumulating for decades across transcriptomic profiling, CLIP-seq, and individual experimental findings.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> End-to-end, machine-learning based design of mRNA molecules offers a powerful means to tailor their properties for specific tasks. mRNA expression level, immunogenicity, tissue specificity, stability, and localization, among others strongly depend on sequence, thus providing a rich set of properties amenable to optimization. Despite this potential, the various components of mRNA are governed by distinct grammatical and functional rules that hinder the development of a unified algorithmic approach for complete mRNA design. While machine learning and generative AI techniques demonstrate substantial benefits for individual sequence design tasks, adapting these tools to new domains or out-of-distribution tasks remains challenging. In this work, we describe a simple and powerful alteration to integrated gradients (Design by Integrated Gradients- DIGs) that serves as the foundation for several mRNA design tasks. Using this technique, we demonstrate complete model-informed mRNA design and reveal the underexplored rules governing the assembly of mRNA components into complete, high-performance transcripts. This framework closes the loop on mRNA design by unifying the strengths of predictive neural networks with the vast quantities of diverse RNA data that has been accumulating for decades across transcriptomic profiling, CLIP-seq, and individual experimental findings. By linking the benefits of neural network-based design with diverse alternative datasets, we demonstrate the design of complete mRNA sequences in out-of-distribution settings that exhibit dramatically improved translational capacity, enhanced cell-type-specific expression, and improved stability.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.17.659751v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genome-wide Pervasiveness and Localized Variation of k-mer-based Genomic Signatures in Eukaryotes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sadjadi, N., de Souza, C. P. E., Randhawa, G. S., Hill, K. A., Kari, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> This study qualitatively and quantitatively analyzes the variations of the genomic signature along an entire genome along different chromosomes of a genome.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genomic signatures are taxon-specific patterns in nucleotide sequence composition observed across different regions of a genome, used in the taxonomic classification of organisms and in inferring their evolutionary relationships. However, the nature and extent of the pervasiveness of a genomic signature across the expanse of a Telomere-to-Telomere (T2T) assembly, especially across the functionally diverse sequence elements and highly repetitive regions, remain counterintuitive and underexplored. This study aims to bridge this knowledge gap by systematically investigating the pervasiveness and variation of the genomic signature across the human genome and the genome of each of three other eukaryotic species from different kingdoms. Using the alignment-free k-mer-based Frequency Chaos Game Representation (FCGR) of DNA sequences, this study qualitatively and quantitatively analyzes the variations of the genomic signature along an entire genome. Qualitative analysis is first performed through visual inspection of FCGR patterns across different chromosomes of a species. In parallel, a quantitative analysis evaluates the variation of the genomic signature within a genome by comparing eight distance measures to identify the optimal one for the datasets in this study. By taking an intragenomic perspective with detailed analysis of chromosome landscapes these analyses reveal that, while the genomic signature is preserved in most genomic regions, exceptions exist in localized regions, such as tandem repetitions of short and long repeat units. Upon determining this pervasiveness, we assemble novel pipelines aimed at selecting a short contiguous representative genomic segment that encapsulates the sequence composition patterns characteristic of the entire genome. These representative segments are then used to assess intragenomic variation of the genomic signature, demonstrating that only a small proportion of segments (namely those characterized by regional density of short and long tandem repeats) show high distance values from the representative. Notably, in the human genome, 80% of the segments have a distance of less than 0.24 (on a [0,1] DSSIM scale) from the representative. Moreover, we demonstrate that using these representative segments improves downstream tasks, e.g., increasing one-nearest-neighbor (1-NN) taxonomic classification accuracy by 7% compared to selecting a random genomic segment to serve as a proxy of the genome. Lastly, this study presents a special-purpose graphical user interface (GUI) software tool, CGR-Diff, designed to provide both visual and quantitative comparisons of FCGRs of sample or user-provided DNA sequences, thereby facilitating intragenomic variation analysis of genomic signature within and across species.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.12.659337v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Efficient Cell Factory Design by Combining Meta-Heuristic Algorithm with Enzyme Constrained Metabolic Models</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liao, W., Gao, G., Fan, X., Wang, H., He, S., Xiang, G., Yan, X., Lu, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a novel strain design algorithm-MetaStrain that integrates enzyme-constrained models (ecModels) with efficient meta-heuristic algorithms to identify non-intuitive combinatorial gene targets for improving product yields.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The design of high-performance microbial cell factories is essential for advancing sustainable biomanufacturing. However, the intricate nature of metabolic networks complicates the prediction of genetic interventions, making strain optimization a challenging combinatorial problem. Here we present a novel computational strain design algorithm-MetaStrain that integrates enzyme-constrained models (ecModels) with efficient meta-heuristic algorithms to identify non-intuitive combinatorial gene targets for improving product yields. Our algorithm employs a modified enforced objective flux scanning algorithm to reduce the dimensionality of gene candidates and annotate editing strategies. Subsequently, binary and symbolic genetic algorithms and the adaptive differential evolution algorithm with external archives (JADE) are employed for gene target selection, with product yield as the fitness function. We adapted minimization of metabolic adjustment strategy for ecModels to evaluate the productive phenotypes of mutant strains. Computational validation using Saccharomyces cerevisiae as a host demonstrates significant enhancements in 2-phenylethanol production. Further generalization to spermidine biosynthesis confirms the framework's robustness. The proposed algorithm achieves high computational efficiency, stable convergence, and strong adaptability across diverse metabolic targets. This study provides a powerful computational tool for metabolic engineering, addressing the complexity of strain design and highlighting the potential of meta-heuristic optimization in synthetic biology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.12.659423v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DiffusionST: A deep generative diffusion model-based framework for enhancing spatial transcriptomics data quality and identifying spatial domains</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cui, Y., Wang, R., Wei, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a method for imputing spatial transcriptomics data and clustering the imputed data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent advancements in spatial transcriptomics technology have generated substantial volumes of spatial transcriptome data. However, the quality of this data is often compromised due to the limitations of current sequencing technologies. To address this issue, DiffusionST proposes a method for imputing spatial transcriptomics data and clustering the imputed data. The method employs a graph convolutional network (GCN) model combined with a newly designed loss function, denoising data using the zero-inflated negative binomial (ZINB) distribution, and data enhancement through a diffusion model to improve clustering accuracy. DiffusionST demonstrates superior clustering accuracy compared to six of the most popular spatial transcriptomics clustering algorithms. DiffusionST also excels in data imputation when compared to five single-cell RNA sequencing (scRNA-seq) imputation algorithms. Additionally, DiffusionST's robustness against noise is quantitatively validated by manually introducing random dropout noise into the dataset, where our model significantly enhances the quality of spatial transcriptomic data. Moreover, DiffusionST is well-suited for high-resolution spatial transcriptomics data and has been demonstrated, through survival analysis and cell-cell communication studies, to dissect spatial domains within breast cancer tissues. These findings provide strong evidence of DiffusionST's efficacy in handling spatial transcriptomic data especially with strong noise, making it a valuable tool in this field.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.12.659243v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Optimizing fungal DNA extraction and purification for Oxford Nanopore untargeted shotgun metagenomic sequencing from simulated hemoculture specimens.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nattapong Langsiri, Wieland Meyer, Laszlo Irinyi, Navaporn Worasilchai, Nuttapon Pombubpa, Thidathip Wongsurawat, Piroon Jenjaroenpun, J Jennifer Luangsa-Ard, Ariya Chindamporn</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A DNA extraction protocol for long-read metagenomics based fungal identification .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Long-read metagenomics provides a promising alternative approach to fungal identification, circumventing methodological biases, associated with DNA amplification, which is a prerequisite for DNA barcoding/metabarcoding based on the primary fungal DNA barcode (Internal Transcribed Spacer (ITS) region). However, DNA extraction for long-read sequencing-based fungal identification poses a significant challenge, as obtaining long and intact fungal DNA is imperative. Comparing different lysis methods showed that chemical lysis with CTAB/SDS generated DNA from pure fungal cultures with high yields (ranging from 11.20 ± 0.17 µg to 22.99 ± 2.22 µg depending on the species) while preserving integrity. Evaluating the efficacy of human DNA depletion protocols demonstrated an 88.73% reduction in human reads and a 99.53% increase in fungal reads compared to the untreated yeast-spiked human blood control. Evaluation of the developed DNA extraction protocol on simulated clinical hemocultures revealed that the obtained DNA sequences exceed 10 kb in length, enabling a highly efficient sequencing run with over 80% active pores. The quality of the DNA, as indicated by the 260/280 and 260/230 ratios obtained from NanoDrop spectrophotometer readings, exceeded 1.8 and 2.0, respectively. This demonstrated the great potential of the herein optimized protocol to extract high-quality fungal DNA from clinical specimens enabling long-read metagenomics sequencing.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40197053/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

