<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Thursday, 22 May 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Accurate prediction of virulence factors using pre-train protein language model and ensemble learning.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Guanghui Li, Jian Zhou, Jiawei Luo, Cheng Liang</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new method for studying virulence factors in bacterial infections .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> As bacterial pathogens develop increasing resistance to antibiotics, strategies targeting virulence factors (VFs) have emerged as a promising and effective approach for treating bacterial infections. Existing methods mainly relied on sequence similarity, and remote homology relationships cannot be discovered by sequence analysis alone.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40399812/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Immunopeptidomics-guided discovery and characterization of neoantigens for personalized cancer immunotherapy.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yangyang Cai, Manyu Gong, Mengqian Zeng, Feng Leng, Dezhong Lv, Jiyu Guo, Hao Wang, Yapeng Li, Quan Lin, Jing Jing, Ying Zhang, Juan Xu, Yongsheng Li</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a pan-cancer peptide atlas and proposed neoantigen screening pipeline for cancer immunotherapies .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Neoantigens have emerged as ideal targets for personalized cancer immunotherapy. We depict the pan-cancer peptide atlas by comprehensively collecting immunopeptidomics from 531 samples across 14 cancer and 29 normal tissues, and identify 389,165 canonical and 70,270 noncanonical peptides. We reveal that noncanonical peptides exhibit comparable presentation levels as canonical peptides across cancer types. Tumor-specific peptides exhibit significantly distinct biochemical characteristics compared with those observed in normal tissues. We further propose an immunopeptidomic-guided machine learning-based neoantigen screening pipeline (MaNeo) to prioritize neo-peptides as immunotherapy targets. Benchmark analysis reveals MaNeo results in the accurate identification of shared and tumor-specific canonical and noncanonical neo-peptides. Last, we use MaNeo to detect and validate three neo-peptides in cancer cell lines, which can effectively induce increased proliferation of active T cells and T cell responses to kill cancer cells but not damage healthy cells. The pan-cancer peptide atlas and proposed MaNeo pipeline hold great promise for the discovery of canonical and noncanonical neoantigens for cancer immunotherapies.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40397742/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">esMPRA: an easy-to-use systematic pipeline for MPRA experiment quality control and data analysis.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jiaqi Li, Pengcheng Zhang, Xi Xi, Xiaowo Wang</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a new method for reproducibility and quality control of large-scale MPRAs, which is based on the use of high-throughput library processing and high-quality sequencing.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Massively Parallel Reporter Assays (MPRAs) have emerged as pivotal tools for systematically profiling cis-regulatory element activity, playing critical roles in deciphering gene regulation mechanisms and synthetic regulatory element engineering. However, MPRA experiments involve multi-step library processing procedures coupled with high-throughput sequencing. Operational errors during these complex workflows can lead to substantial resource depletion and experimental delays. Thus robust and user-friendly quality control methods are essential to minimize experimental failures and ensure reproducibility between replicates.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40397077/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Translational buffering tunes gene expression in mouse and human</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rao, S., Le, A. Y., Persyn, L., Cenik, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A global study of genes that exhibit translational buffering, which is a conserved property of human and mouse genes across cell types.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Translational buffering refers to the regulation of ribosome occupancy to offset the effects of transcriptional variation. While previous work systematically analyzed translational buffering in yeast, it remains unknown whether this is an intrinsic property of genes in mouse and human across cell types. To identify this phenomenon on a global scale and across different experimental conditions, we uniformly analyzed 1515 matched ribosome profiling and RNA-seq datasets from human and mouse tissues or cell lines. This resource enabled us to assess the potential of genes to exhibit translational buffering by comparative analysis of variation at ribosome occupancy and the RNA levels across cell types as well as by examining the relationship between mRNA abundance and translation efficiency. We demonstrate that translational buffering is a conserved property of genes using homologous gene pairs from humans and mice. Although the identified buffered genes show association with some intrinsic sequence features, our modeling results suggest that these alone are insufficient to predict translational buffering, highlighting the importance of cellular context in determining buffering. Further, genes exhibiting translational buffering have lower variation in protein abundance in cancer cell lines, primary human tissues and mouse samples. We also observed that translationally buffered genes are more likely to be haploinsufficient and triplosensitive suggesting a demand for stringent dosage limits in these genes. We hypothesize two models of translational buffering, namely "differential accessibility model" and "change in translation initiation rate model". Our experiment suggests that some transcripts conform to the former and others align with the alternate model. Overall, our work broadens the catalog of genes subjected to translational buffering, underscores the characteristics of genes that demonstrate this phenomenon and additionally provides an insight into the rationale driving this effect.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.16.654561v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Change point based dynamic functional connectivity estimation outperforms sliding window and static estimation for classification of early mild cognitive impairment in resting-state fMRI</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ondrus, M., Cribben, I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that dynamic functional networks are dynamic, multiscale, and that different FC methods capture distinct information for classification efficacy.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The most widely used inputs in classification models of brain disorders such as early mild cognitive impairment (eMCI) or Alzheimer's disease are estimates of static-based functional connectivity (SFC) and sliding window dynamic functional connectivity (swDFC). Although these methods are convenient for estimation and computational purposes, as it keeps the estimation tractable, they present a simplified version of a highly integrated and dynamic phenomenon. Change point dynamic functional connectivity (cpDFC) methods, which are far less commonly used, offer an alternative to swDFC approaches. In this study, we consider a classification task between controls and patients with eMCI using resting-state functional magnetic resonance imaging (fMRI) data from the Alzheimer's Disease Neuroimaging Initiative (ADNI) studies, ADNI2 and ADNIGO. Our results indicate that the DFC methods are generally superior to the SFC methods when used as inputs into the classification model. Most importantly, we find that the cpDFC methods are generally superior to the widely used swDFC methods. We discuss how cpDFC methods offer many distinct advantages over swDFC methods, namely, the parsimony of network features and ease of interpretability. We validate the robustness and consistency of our results by testing the methods on an additional resting-state fMRI dataset of mild cognitive impairment patients. These findings call into question the validity of numerous fMRI studies that have utilized inferior approaches, such as SFC and swDFC, as inputs to classification models to predict various brain disorders. Finally, we present an ensemble model of the best models, which achieves an accuracy of 91.17% from leave-one-out cross-validation of subjects with eMCI. Our results suggest that the underlying functional networks are dynamic, multiscale, and that different FC methods capture distinct information for classification efficacy.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.16.654552v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ChromMovie: A Molecular Dynamics Approach for Simultaneous Modeling of Chromatin Conformation Changes from Multiple Single-Cell Hi-C Maps</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Banecki, K. H., Chai, H., Ruan, Y., Plewczynski, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present ChromMovie, a comprehensive molecular dynamics framework for modeling 3D chromatin structure changes in the context of dynamic cellular processes, such as cell cycle or cell maturation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The development of 3C-based techniques for analyzing three-dimensional chromatin structure dynamics has driven significant interest in computational methods for 3D chromatin reconstruction. In particular, models based on Hi-C and its single-cell variants, such as scHi-C, have gained widespread popularity. Current approaches for reconstructing the chromatin structure from scHi-C data typically operate by processing one scHi-C map at a time, generating a corresponding 3D chromatin structure as output. Here, we introduce an alternative approach to the whole genome 3D chromatin structure reconstruction that builds upon existing methods while incorporating the broader context of dynamic cellular processes, such as the cell cycle or cell maturation. Our approach integrates scHi-C contact data with single-cell trajectory information and is based on applying simultaneous modeling of a number of cells ordered along the progression of a given cellular process. The approach is able to successfully recreate known nuclear structures while simultaneously achieving smooth, continuous changes in chromatin structure throughout the cell cycle trajectory. Although both Hi-C-based chromatin reconstruction and cellular trajectory inference are well-developed fields, little effort has been made to bridge the gap between them. To address this, we present ChromMovie, a comprehensive molecular dynamics framework for modeling 3D chromatin structure changes in the context of cellular trajectories. To our knowledge, no existing method effectively leverages both the variability of single-cell Hi-C data and explicit information from estimated cellular trajectories, such as cell cycle progression, to improve chromatin structure reconstruction.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.16.654550v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Accounting for uncertainty in residual variances improves calibration of the Sum of Single Effects model for small sample sizes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Denault, W. R. P., Carbonetto, P., Li, R., Consortium, T. A. s. D. F. G., Wang, G., Stephens, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that a simple alternative fitting procedure, which takes account of uncertainty in the residual variance, notably improves performance in small sample studies.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The Sum of Single Effects (SuSiE) model, and a fitting procedure based on variational approximation, were recently introduced as a way to capture uncertainty in variable selection in multiple linear regression. This approach is particularly well-suited to cases where variables are highly correlated, and it has become quickly adopted for genetic fine-mapping studies. Here, we show that in small studies (<50 samples) the original fitting procedure can produce substantially higher rates of false positive findings than it does in larger studies. We show that a simple alternative fitting procedure, which takes account of uncertainty in the residual variance, notably improves performance in small sample studies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.16.654543v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Large Scale Cell Painting Guided Compound Selection Reveals Activity Cliffs and Functional Relationships</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sanchez, M., Bourriez, N., Bendidi, I., Cohen, E., Svatko, I., Del Nery, E., Tajmouati, H., Bollot, G., Calzone, L., Genovesio, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a training-free, transfer learning-based method for large scale compound preselection that leverages deep phenotypic profiling of human cells, and show that this approach captures subtle, atom-level determinants of bioactivity that cannot be accessed by structure-based models.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Traditional structure-based pre-screen compound selection relies on the assumption that chemical similarity implies similar biological activity. This paradigm narrows the exploration of chemical space and often fails to account for functional convergence, where structurally diverse compounds act through distinct targets to produce similar phenotypic effects. As a result, compounds with therapeutic potential may be overlooked. To overcome this constraint, we introduce a training-free, transfer learning-based method for large scale compound preselection that leverages deep phenotypic profiling of human cells. Notably, this enables robust pairwise comparison of phenotypic signatures across any source of the entire JUMP-CP, the largest publicly available cell painting dataset (112,480 compounds), preserving biological signals while mitigating batch effects. Validated across 65 high-throughput assays, including in vitro and in cellulo systems, our method provides efficient pre-screen enrichment of biologically active compounds, bypassing the blind spots of structure-centric approaches. Interestingly, because it is large scale, it also allows for a comprehensive analysis of structure-phenotypic activity relationships, revealing potentially thousands of compound activity cliffs, where minimal chemical changes in structure may result in profound phenotypic shifts. We show that these cliffs capture subtle, atom-level determinants of bioactivity that cannot be accessed by structure-based models. Furthermore, we demonstrate that structurally diverse compounds targeting different genes in the same biological pathway can induce either convergent or opposite phenotypes, a phenomenon validated across 30 pathways, hundreds of genes, and thousands of compounds. Finally, to support the broader community, we propose Phenoseeker, a web-based tool enabling instant retrieval of JUMP-CP compounds with similar phenotypic profiles. Together, these findings position phenotypic profiling not merely as a complementary tool, but as a transformative and scalable framework for navigating chemical space through a biological lens. By capturing rich morphological signatures that reflect functional outcomes, regardless of structural similarity, this approach enables the discovery of bioactive compounds, novel mechanisms of action, and unexpected target-pathway relationships. Applied at the scale of the JUMP-CP dataset, phenotypic profiling emerges as a powerful strategy for prioritizing compounds, illuminating activity cliffs, and accelerating the identification of therapeutically relevant candidates across diverse biological contexts.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.16.654292v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">acmgscaler: An R package and Colab for standardised gene-level variant effect score calibration within the ACMG/AMP framework</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Badonyi, M., Marsh, J. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A genome-wide variant effect calibration method for evidence-based variant classification .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> A genome-wide variant effect calibration method was recently developed under the guidelines of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology (ACMG/AMP), following ClinGen recommendations for variant classification. While genome-wide approaches offer clinical utility, emerging evidence highlights the need for gene- and context-specific calibration to improve accuracy. Building on previous work, we have developed an algorithm tailored to converting functional scores from both multiplexed assays of variant effects (MAVEs) and computational variant effect predictors (VEPs) into ACMG/AMP evidence strengths. Our method is designed to deliver consistent performance across different genes and score distributions, with all variables adaptively determined from the input data, preventing selective adjustments or overfitting that could inflate evidence strengths beyond empirical support. To facilitate adoption, we introduce `acmgscaler`, a lightweight R package and a plug-and-play Google Colab notebook for the calibration of custom datasets. This algorithmic framework bridges the gap between MAVEs/VEPs and clinically actionable variant classification.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.16.654507v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Oncogenic drivers shape the tumor microenvironment in human gliomas</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhao, B., Cho, C. Y., Ye, L., Keal, T., Mitchell, T. J., Martin-Barrio, I., Weinberg, J. S., Huse, J. T., Dixon, J. R., Lang, F. F., Akdemir, K. C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We study the role of oncogenic mutations in glioma microenvironment, revealing subtype-specific cellular ecosystems that could inform targeted therapeutic strategies.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Gliomas are aggressive and heterogeneous brain tumors with limited treatment options. While oncogenic mutations in gliomas have been well-characterized, their impact on the tumor microenvironment remains poorly understood. To investigate how genomic alterations may influence the glioma microenvironment, we performed an integrative multiomic and spatial transcriptomic analysis of 93 glioma samples (46 IDH-mutant gliomas and 47 IDH-wildtype glioblastoma) from 69 patients representing both primary and recurrent stages. Using whole-genome sequencing, chromatin conformation capture (Hi-C), RNA-seq, and single-cell spatial transcriptomics (Xenium), we defined how major driver mutations influence spatial tumor organization. We found that IDH-mutant gliomas frequently harbored inflammatory microglia expressing CX3CR1 specifically within their astrocyte-like malignant neighborhoods. In contrast, glioblastomas demonstrated relatively higher T-cell infiltration and enrichment of immunosuppressive myeloid cell populations. We further compared glioblastomas harboring EGFR amplifications due to extrachromosomal DNA (ecDNA) amplifications versus linear chromosomal 7 gains. Tumors with EGFR ecDNA displayed increased presence of mesenchymal-like malignant cells, and higher interactions between pericytes and mesenchymal-like malignant cells, likely driven by hypoxia-associated vascular proliferation. Our findings reveal that the mode of oncogene amplification (linear versus ecDNA) shapes distinct tumor architectures and transcriptional dynamics. Taken together, our study highlights the role of oncogenic drivers in shaping the glioma microenvironment, revealing subtype-specific cellular ecosystems that could inform targeted therapeutic strategies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.16.654515v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">From Movement to METs: A Validation of ActTrust(R) for Energy Expenditure Estimation and Physical Activity Classification in Young Adults</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> dos Santos Batista, E., Basilio Gomes, S. R., Bruno de Morais Ferreira, A., Franca, L. G. S., Fontenele Araujo, J., Mortatti, A. L., Leocadio-Miguel, M. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a general linear model to estimate energy expenditure (EE) from movement count of combinations of ActTrust(R) devices placed at hip or wrist and compared activity counts to metabolic equivalents derived from indirect calorimetry during treadmill walking and running.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Physical activity (PA) is recognised for providing several health benefits in humans, mainly preventing and controlling chronic non-communicable diseases. However estimating PA is a challenging and expensive task. An alternative would be to devise a model to estimate PA using actigraphy devices calibrated from an initially validated model. This has been previously done to a number of devices, including ActiGraph(R) GT3X+. In this study we aimed at validating ActTrust(R) against the widely used GT3X+ and comparing activity counts to metabolic equivalents (METs) derived from indirect calorimetry during treadmill walking and running. Fifty-six young adults (34 men, 22 women) participated in controlled effort exercises including light, moderate, vigorous, and very vigorous activity intensities. We developed a general linear model to estimate energy expenditure (EE) from movement count of combinations of GT3X+ and ActTrust devices placed at hip or wrist. We then estimated cut-off points for each intensity range. Our results showed correlations between treadmill speed, METs, and movement counts across all devices and placements combinations. Our proposed model performed well with balanced accuracies above 0.77 for all intensity ranges and over 0.9 for light and moderate activity. This is the first study to model estimate and validate PA intensity thresholds on ActTrust(R) devices. Our findings support the use of ActTrust(R) devices in PA estimation as a low complexity and cost approach to allow 24-hour assessments of EE.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.16.654458v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Accelerate the discovery of genetic variants in mitochondrial diseases with VIOLA: Variant PrIOritization using Latent space</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Labory, J., Boulaimen, Y., Singh, J., AIT-EL- MKADEM SAADI, S., Paquis, V., Bannwarth, S., BOTTINI, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a new method for detecting complex diseases with complex phenotypes using variational autoencoder and a new pathogenicity score.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Interpreting variants from whole-exome sequencing remains a major challenge, particularly for heterogeneous disorders such as mitochondrial diseases (MD). To help find a diagnosis for complex cases, we have developed VIOLA (Variant prIoritizatiOn using Latent spAce). The pipeline includes a variational autoencoder to embed functional annotations into a low-dimensional space, followed by DBSCAN-based outlier detection. To prioritize potential pathogenic variants, filtering steps and phenotype integration via HPO terms are then applied. Two scores are associated to the selected variants: the VIOLA score (Vscore), which combines variant features, transcriptomic co-expression data, and MD-specific annotations, and the VIOLA Aggregated score (VAscore) that merges Vscore with Exomiser pathogenicity score. Finally, we also provide the ARrank, specifically designed for variants compatible with autosomal recessive inheritance. We illustrate the functionalities of VIOLA on a novel cohort of patients with suspected MD with complex phenotypes. VIOLA systematically ranked causal variants among the top, outperforming existing methods. Overall, VIOLA is a patient-specific approach to help discover novel variants in complex MD.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.16.654430v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Fusion transcription factor dosage controls cell state in rhabdomyosarcoma</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hoffman, R. A., Wang, M., Sunkel, B. D., Nguyen, T. H., Lopez-Nava, J., Chatterjee, B., Sun, W., Barr, F. G., Stanton, B. Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A human myoblast system for inducible expression of PAX3::FOXO1 protein for fusion-positive rhabdomyosarcoma and its epigenetic and phenotypic functions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In the fusion-positive subset of rhabdomyosarcoma, the PAX3::FOXO1 oncoprotein is the most common fusion driver. We previously established a human myoblast system for inducible expression of PAX3::FOXO1. In the current study, we modulate PAX3::FOXO1 protein expression to understand the epigenetic and phenotypic functions at different PAX3::FOXO1 levels. Proliferative and oncogenic outcomes depend on PAX3::FOXO1 dosage in this system with transformation dominant at intermediate levels and growth suppression dominant at high levels. After prolonged PAX3::FOXO1 expression, there is dosage-dependent heterogeneity in single cell gene expression profiles. We observe a dosage-specific effect for PAX3::FOXO1 chromatin recognition and identify factors that modulate PAX3::FOXO1 chromatin binding. PAX3::FOXO1 dosage affects expression signatures related to cell cycle, epithelial-mesenchymal transition, and myogenesis. Whereas intermediate PAX3::FOXO1 expression maximizes chromatin binding to modulate gene expression, high PAX3::FOXO1 expression alters S phase progression and increases accessibility behind the replication fork. We conclude that PAX3::FOXO1 exerts dosage-dependent functions to influence epigenetic heterogeneity in fusion-positive rhabdomyosarcoma.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.16.654522v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integration of strain and process optimization to increase autotrophic growth of engineered Komagataella phaffii</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Baumschabl, M., Lutz, L., Jecmenica, M., Ata, O., Mattanovich, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new round of adaptive laboratory evolution and the optimal cultivation temperature for autotrophic growth of the methylotrophic yeast Komagataella phaffii .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Synthetic autotrophs are a promising platform for sustainable bioproduction using CO2 as substrate. The methylotrophic yeast Komagataella phaffii has been engineered to use CO2 as the sole carbon source by integration of the Calvin-Benson-Bassham (CBB) cycle, based on its native methanol assimilating xylulose monophosphate pathway (XuMP) cycle. Initial growth rates were low, but could be doubled by adaptive laboratory evolution (ALE). Beneficial mutations led to a decrease of CBB cycle reactions, indicating further limitations. During this study, temperature was identified as one of the key process parameters to improve autotrophic growth. For this reason, a new round of adaptive laboratory evolution was performed at the identified optimal cultivation temperature of 25{degrees}C, resulting in isolates growing up to 50 % faster compared to the control strain. Whole genome resequencing followed by reverse engineering helped to identify first key mutations of the evolved strains. In addition, targeted engineering was performed by increasing the copy number of the key gene of the CBB cycle RuBisCO, which is the bottleneck of carbon fixation. Combining this with the optimal cultivation temperature boosted maximum specific growth rates of the autotrophic K. phaffii strain. In comparison to ALE, the targeted engineering still is lagging behind a bit. Starting from the initial condition, growth was boosted more than 2.5-fold in this study to a maximum of 0.025 h-1.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.21.655372v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Unveiling the hidden window of prime editing</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chen, P.-R., Wei, Y., Yuan, X.-Z., Wang, S.-G., Xia, P.-F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Prime editing on the opposite 5' direction of the nicked DNA strand through a hidden reverse transcription pathway, where a designed edit is integrated upstream of the nick.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Prime editing enables versatile and precise genomic modifications, following a fundamental principle that the editing extends from the nick site towards the 3' direction of the cleaved DNA strand. Here, we report the unknown competency of prime editing on the opposite 5' direction of the nicked DNA strand through a hidden reverse transcription pathway, where a designed edit can be integrated upstream of the nick.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.20.655067v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

