<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Thursday, 29 May 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Hybrid Deep Learning with Protein Language Models and Dual-Path Architecture for Predicting IDP Functions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liang, J., Peng, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present IDPFunNet, a hybrid deep learning model combining convolutional neural network, bidirectional LSTM, residual MLP, and the protein language model ProtT5 to predict six IDR functional classes: five binding subtypes and disorded flexible linkers.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Intrinsically disordered regions (IDRs) drive essential cellular functions but resist conventional structural-function annotation due to their dynamic conformations. Current computational methods struggle with cross-dataset generalization and functional subtype discrimination. We present IDPFunNet, a hybrid deep learning model combining convolutional neural network, bidirectional LSTM, residual MLP, and the protein language model ProtT5 to predict six IDR functional classes: five binding subtypes and disorded flexible linkers (DFLs). Its dual-path architecture decouples binding IDR prediction and DFL identification using evolutionary embeddings from ProtT5, which outperformed ESM-family models and AlphaFold2 structural features by [≥]1.3% average AUC and [≥]12.7% average APS. Validated across six benchmarks including CAID2/3 blind tests, IDPFunNet achieves superior protein-binding prediction (AUC 0.832-0.866) with significant improvements ([≥]1.5%, p-value<0.05) over existing methods, while matching specialized DFL predictors. Multi-task learning enhances protein/lipid/small molecule-binding performance (3.1-35.1% AUC gains) with BiLSTMs proving optimal DFL identification, despite self-attention's potential for nucleic acid-binding (AUC 0.831). This framework overcomes methodological limitations through interpretable, generalizable IDR functional mapping.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.25.655984v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">House of Clocks: On the Misuse of Ageing Composite Measures</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hu, I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> This article examines the structural implications of composite variables developed in the field of "ageing" and shows how the property of feature convergence arises in practice.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> This article examines the structural implications of composite variables developed in the field of "ageing". First, using data from the Baltimore Longitudinal Study of Aging (BLSA), we illustrate how the property of feature convergence arises in practice. Second, we show how this constrains both cross-sectional stratification and longitudinal monitoring. Third, we extend the analysis to molecular implementations and demonstrate that similar structural issues persist. Fourth, we examine how these properties impact common objectives - intervention evaluation, prediction, and investigation - and show that the limitations are not confined to linear models. Finally, we consider additional properties that may help explain the widespread appeal and persistence of these models, despite clear conflicts with their stated objectives.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.24.655934v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identifying Molecular Determinants and Therapeutic Targets in Luminal B Breast Cancer: A Systems Biology Approach</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Saeidi, Y., Ghorbani, M., Najafi, A., Moosazadeh Moghaddam, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> In silico analysis of LBBC cell lines using the Gene Expression Omnibus (GEO) microarray dataset, we identified LBBC-associated genes, TFs, miRNAs, and pathways as prospective biomarkers, providing insights into LBBC diagnosis and therapeutic approaches.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Luminal B breast cancer (LBBC) is on the rise worldwide, with both incidence and mortality rates steadily increasing. Early detection proves difficult due to its aggressive characteristics, most notably its heightened proliferation rate and the complex interplay of molecular biomarkers, particularly in more advanced stages. In the present study, we conducted an in silico analysis of LBBC cell lines using the Gene Expression Omnibus (GEO) microarray dataset, which includes 30 LBBC and 11 normal samples. Differentially expressed genes (DEGs) were identified using RStudio. A series of analyses, including cancer data interrogation via pan-cancer analysis, eXpression2Kinases (X2K), and the Cancer Dependency Map (DepMAP), was carried out to elucidate the underlying signaling pathways, transcription factors (TFs), and kinases, as well as to perform stemformatics analysis. Protein-protein interaction (PPI) networks were reconstructed using STRING and Cytoscape, enabling the identification of co-expressed and hub genes through the cytoHubba plug-in. Of note, FGF2, EGFR, ADIPOQ, LIPE, MET, IGF1, FGF1, EGF, FGF7, and PPARG were identified as the top 10 driver genes; RELA, PPARG, CTCF, EGR1, and NFE2L2 were detected as predominant TFs in LBBC; and CDK1, CDK2, MAPK3, CSNK2A1, and MAPK14 were identified as potential biomarkers through hub gene clustering. Further analysis indicated hsa-mir-221-3p and hsa-mir-29a-3p as key miRNAs targeting LBBC-related genes. Collectively, our findings highlighted LBBC-associated genes, TFs, miRNAs, and pathways as prospective biomarkers, providing insights into LBBC diagnosis and therapeutic approaches.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.25.656027v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The genomic legacy of selectively breeding rhesus macaques for HIV/AIDS-related research</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lyke, M. M., Bagwell, A., Newman, D., Galindo, S., Church, T., Christensen, C., Gray, S. B., Cox, L. A., Ross, C. N., Kaushal, D., Cheeseman, I. H., Cole, S. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that SIV-refractory MHC haplotypes and genetic ancestry in Rhesus macaques can be reduced by colony management strategies without reducing genetic diversity of the MHC or exonic regions.</p><p style="font-size: 16px; text-align: justify;">Background: Rhesus macaques are the preferred non-human primate model for HIV/AIDS-related research. Specific major histocompatibility complex (MHC) haplotypes and genetic ancestry are linked to slow disease progression of Simian Immunodeficiency Virus (SIV) infections in macaques. To maximize their utility for HIV/AIDS research, the Southwest National Primate Research Center (SNPRC) has introduced targeted breeding strategies to reduce the prevalence of SIV-refractory MHC haplotypes and levels of admixture between Indian- and Chinese-origin macaques. Results: We characterized the MHC region in SNPRC macaques by targeted deep sequencing of 1,458 animals born over a ten-year period. Following the implementation of MHC management strategies, the prevalence of SIV-refractory MHC haplotypes reduced significantly while overall haplotype diversity was maintained. We investigated the impact of management strategies on admixture, population genetic structure, genetic diversity and inbreeding using whole exome sequencing of founding and colony-born animals (n=488). Admixture analysis of founders showed some Chinese ancestry, though population substructure more closely reflected primate research center source than geographical origin. The levels of Chinese ancestry declined significantly over time, though genetic diversity remains high. Finally, we performed genome-wide scans for genetic selection over time. We identify numerous genomic regions where allele frequencies have shifted significantly, supporting the presence of short-term adaptation within the colony. Conclusions: We show that colony management strategies have been successful without reducing genetic diversity of the MHC or exonic regions. We also show that colony genetic substructure is related to animal colony source and that mergers and migrations have reduced inbreeding and increased overall genetic diversity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.26.654976v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Decoupled Information Theoretic Feature Selection for Rapid Protein Key Tuning Residue Identification</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Saeed, H., Huang, W., Yang, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose an information-theoretic approach to identify key residues in protein sequences, inspired from signal processing methods, and show competitive performance compared to current state-of-the-art computational tools.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Proteins contain key residues that critically influence their function, stability, and interactions. However, accurately identifying these "key tuning" residues remains challenging. In this study, we present an information-theoretic approach, inspired from signal processing methods, to identify key residues in protein sequences. We propose a feature selection framework that integrates both unsupervised and supervised approaches, using Shannon entropy and mutual information as initial priors, respectively. A key aspect of our method is the use of Cramer's V and Thiel's U to decouple coevolved residues. The supervised approach employed mutual information to assess residue relationships, followed by evolutionary decoupling with Cramer's V to better decouple individual contributions. This two-step process demonstrated statistically significant improvement compared to DeepSequence, a leading tool (p-value=0.0149). The unsupervised approach, based solely on Shannon entropy prior to decoupling, showed competitive performance despite a slightly lower statistical significance (pvalue=0.14). Although the supervised method yields higher accuracy given sufficient training data, the unsupervised technique is simpler, requiring only a multiple sequence alignment (MSA) that can be obtained via BLAST. When applied to a system with well-characterised structural information, specifically nanobody-antigen interactions, our methods not only effectively identified the binding residues but also accurately reconstructed a contiguous binding surface, surpassing current state-of-the-art computational tools. Collectively, our methods provide a flexible and interpretable solution for various protein engineering applications. This work contributes to the broader field of protein structure-function mapping by offering a computationally efficient and transparent approach for identifying key tuning residues, paving the way for new strategies in rational protein design and functional modulation with potential applications in drug discovery, enzyme optimization, and novel protein development.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.28.653817v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comparative assessment of co-folding methods for molecular glue ternary structure prediction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liao, Y., Zhu, J., Xie, J., Lai, L., Pei, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that current co-folding methods struggle to accurately model large protein-protein interaction interfaces, and highlight the need for further improved in novel E3 ligase systems.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Molecular glues (MGs) represent an emerging therapeutic paradigm capable of inducing or stabilizing protein-protein interactions, with broad applications in creating neomorphic interactomes and targeted protein degradation. However, current discovery efforts remain largely confined to experimental screening, while in silico rational design of MGs persists as a formidable challenge. A critical step toward rational design lies in accurate ternary complex modeling. Given the scarcity of such complexes in the Protein Data Bank for training specialized predictive models, we tested the ability of recently developed co-folding models, including AlphaFold 3, Boltz-1, Chai-1, Protenix, and RoseTTAFold All-Atom in building the complex models. We systematically curated a dataset, named MG-PDB, with 221 non-covalent MG-engaged ternary complexes. MGBench were further introduced as a comprehensive benchmark set, which comprises 88 ternary structures excluded from co-folding models' training data through rigorous time-based partitioning. Our benchmark results demonstrated that AlphaFold3 achieved the best overall performance among co-folding methods, in terms of both protein-protein interaction interface prediction (50.6% success rate) and MG-protein interaction recovery (32.9% success rate). However, our homology study showed that most of their successful predictions actually stemmed from memorization. Further analysis revealed three phenomena of current co-folding methods for MG ternary structure prediction. Firstly, these methods struggle to accurately model large interaction interfaces. Secondly, their predictive accuracy is notably reduced for domain-domain complexes compared to domain-motif interactions. Lastly, they face specific challenges in modeling MG degrader complexes with sufficient accuracy. We showcased they relied on the existing interaction patterns, and highlighted the need for further improved in novel E3 ligase systems. These findings reveal fundamental gaps in existing methods to learn atomic-level interaction rules for MG-engaged ternary complex modeling. MG-PDB and MGBench provide critical resources and mechanistic insights to advance computational MG discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.25.655997v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Spatiotemporal transcriptomic analysis during cold ischemic injury to the murine kidney reveals compartment-specific changes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Singh, S., Patel, S. K., Matsuura, R., Velazquez, D., Sun, Z., Noel, S., Rabb, H., Fan, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Spatial transcriptomics for cold ischemia injury and a molecular basis for how cold ischemic injury may negatively affect kidney outcomes .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Kidney transplantation is the gold standard treatment strategy for end-stage renal disease. Deceased donor kidneys usually undergo cold storage until kidney transplantation, leading to cold ischemia injury that may contribute to poor graft outcomes. However, the molecular characterization of potential mechanisms of cold ischemia injury remains incomplete. To bridge this knowledge gap, we leveraged spatial transcriptomics technology to perform full transcriptome characterization of cold ischemia injury (0-48 hours) using a murine model. We developed a computational workflow to identify spatiotemporal transcriptomic changes that accompany the injury pathophysiology in a compartment-specific manner. We identified potential metabolic reprogramming preferentially within the kidney inner medulla displaying strong oxidative phosphorylation signature in an ischemic environment. We found commonalities between the spatiotemporal transcriptomic presentation of cold ischemia and warm ischemia-reperfusion injury, including an induction of an anti-viral like immune response throughout the renal tissue. Altogether, these systems-level biological insights enabled by our full transcriptome temporal characterization unveil a molecular basis for how cold ischemia injury may negatively affect kidney outcomes. Moreover, our spatial analyses highlight pathological developments deep within the renal tissue, suggesting potential opportunities for new insights beyond biopsy-focused superficial tissue examinations. We also developed an interactive online browser at https://jef.works/vitessce-cold-ischemia/ to facilitate exploration of our results by the broader scientific and clinical community.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.25.654911v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Differential expression of mitochondria-associated genes in clinical samples of Plasmodium falciparum showing severe manifestations</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gujarati, S., Gupta, D., Kochar, S. K., Kochar, D. K., Das, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified disease-specific sense and Natural Antisense Transcripts (NATs) encoded by the mitochondrial genome, along with those encoded in the nucleus and targeted to the parasite mitochondria, and propose these NATs against protein-coding genes specific to the observed disease severity.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The malaria parasite mitochondrial proteins are critical targets for antimalarial drugs, however, the emergence of drug resistance against the existing protein targets necessitates novel therapeutic approaches. In this study, using custom-designed, multi-probe microarrays and pooled RNA sequencing experiments in patient samples of Plasmodium falciparum, we identified disease-specific sense and Natural Antisense Transcripts (NATs) encoded by the mitochondrial genome, along with those encoded in the nucleus and targeted to the parasite mitochondria. These findings span two clinical manifestations of severe malaria - hepatic dysfunction (HD) and cerebral malaria (CM). Although mitochondrial activity is known to be reduced in blood-stage malaria, upregulation of genes linked to the TCA cycle in the CM cluster indicates disrupted mitochondrial bioenergetics which might contribute to severe disease pathogenesis. Profiling of sense and antisense mitochondrial transcripts reveal a striking correlated expression of sense-antisense transcript pairs in both the disease manifestations, indicating a potential regulatory role of NATs in mitochondrial function. This study provides direct evidence for the origin of NATs from the mitochondrial genome. It further provides a dynamic gene regulation profile of key sense and antisense transcripts responsible for mitochondrial translation and bioenergetics. We propose these differentially regulated NATs against protein-coding genes specific to the observed disease severity as potential novel targets to disrupt key mitochondrial functions of the parasite.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.24.655897v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">LlamaAffinity: A Predictive Antibody Antigen Binding Model Integrating Antibody Sequences with Llama3 Backbone Architecture</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hossain, D., Saghapour, E., Song, K. M., Chen, J. Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present an advanced antibody-antigen binding affinity prediction model (LlamaAffinity), leveraging an open-source Llama 3 backbone and antibody sequence data employed from the Observed Antibody Space (OAS) database.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Antibody-facilitated immune responses are central to the body's defense against pathogens, viruses, and other foreign invaders. The ability of antibodies to specifically bind and neutralize antigens is vital for maintaining immunity. Over the past few decades, bioengineering advancements have significantly accelerated therapeutic antibody development. These antibody-derived drugs have shown remarkable efficacy, particularly in treating Cancer, SARS-Cov-2, autoimmune disorders, and infectious diseases. Traditionally, experimental methods for affinity measurement have been time-consuming and expensive. With the realm of Artificial Intelligence, in silico medicine has revolutionized; recent developments in machine learning, particularly the use of large language models (LLMs) for representing antibodies, have opened up new avenues for AI-based designing and improving affinity prediction. Herein, we present an advanced antibody-antigen binding affinity prediction model (LlamaAffinity), leveraging an open-source Llama 3 backbone and antibody sequence data employed from the Observed Antibody Space (OAS) database. The proposed approach significantly improved over existing state-of-the-art (SOTA) approaches (AntiFormer, AntiBERTa, AntiBERTy) across multiple evaluation metrics. Specifically, the model achieved an accuracy of 0.9640, an F1-score of 0.9643, a precision of 0.9702, a recall of 0.9586, and an AUC-ROC of 0.9936. Moreover, this strategy unveiled higher computational efficiency, with a five-fold average cumulative training time of only 0.46 hours, significantly lower than previous studies. LlamaAffinity defines a new benchmark for antibody-antigen binding affinity prediction, achieving advanced performance in the immunotherapies and immunoinformatics field. Furthermore, it can effectively assess binding affinities following novel antibody design, accelerating the discovery and optimization of therapeutic candidates.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.28.653051v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Acoustic individual identification in a species of field cricket using deep learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kabuga, E., Nandi, D., Burrell, S., Dlamini, G., Balakrishnan, R., Bah, B., Durbach, I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We demonstrate the feasibility of identifying individual field crickets from their calls using deep learning using wild recordings of an insect species, and further highlight the potential of deep learning-based AIID for non-invasive monitoring of animal populations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Individual animal identification is essential for wildlife conservation and management, helping to estimate animal abundance and related parameters. We assessed the feasibility of identifying individual field crickets (Plebeiogryllus guttiventris) from their calls using deep learning. In a closed population of known individuals, the best models recognized crickets with up to 99.9% accuracy when trained and tested on calls from the same night, and 63.2% when tested on calls from nights not used in training. When matching pairs of calls, without knowledge of all individuals in the population, models identified pairs of calls from the same night with 97% accuracy, falling to 87.1% if calls were from different nights. Accuracy remained high when tested on individuals not observed during training (within night: 94.1%; across nights 78.8%). Pooling training data across multiple nights improved test accuracy for all models. Deep learning outperformed random forests, particularly on harder tasks, although both were able to discriminate individuals. Analysis of temperature effects on cricket calling behavior showed that higher temperatures were associated with shorter chirp durations and higher carrier frequencies. However, adjusting spectrograms for chirp duration alone, or for both duration and frequency, did not improve performance. The results are the first to demonstrate AIID using wild recordings of an insect species, and further highlight the potential of deep learning-based AIID for non-invasive monitoring of animal populations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.24.655958v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Developing a versatile technology for agroinfiltration in multiple plants</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Du, H., Wang, T., Ge, J., Qu, C., Chen, H., Li, J., Wang, S., Guan, X., Chen, T., Si, Z., Hu, K., Ding, X., Liu, L., Zhang, J., Mishin, A. S., Yampolsky, I. V.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We develop a fungal bioluminescence pathway-based reporter system that utilizes caffeic acid to generate self-sustained luminescence to enhance transient transformation efficiency and broaden our understanding of gene function in plant kingdom.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Agrobacterium-mediated transient transformation methods are widely used in plant science and molecular pharming. However, plant immune response and post-transcriptional gene silencing (PTGS) can compromise gene expression, limiting the adaptability of these methods across plant species. The lack of a robust reporter system further hinders advancements in agroinfiltration technology. To address these challenges, we developed a fungal bioluminescence pathway (FBP)-based reporter system that utilizes caffeic acid to generate self-sustained luminescence. By incorporating NahG and P19, our FBP assay enhances transient transformation efficiency in over 20 plant families, including major crops, vegetables, ornamental plants, and trees. Furthermore, we demonstrate its utility for assessing protein localization, interactions, and transcriptional regulation in multiple plant species, thereby broadening our understanding of gene function throughout the plant kingdom.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.25.656011v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A non-rectifying potassium channel increases cassava drought tolerance and storage root yield</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zierer, W., Fritzler, M., Chiu, T. J., Anjanappa, R. B., Chang, S.-H., Metzner, R., Quiros, J., Lamm, C. E., Thieme, M., Koller, R., Huber, G., Muller, O., Rascher, U., Sonnewald, U., Neuhaus, H. E., Gruissem, W., Bellin, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A non-rectifying Arabidopsis thaliana potassium (K+) channel gene version, AKT2var, in the vascular tissue of cassava plants, which leads to improved storage root yield and increased source-sink carbohydrate transport.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cassava (Manihot esculenta) is an important crop for food security in the tropics, particularly for smallholder farmers in sub-Saharan Africa, where yields are often severely limited by pathogen pressure, nutrient deficiency, and water scarcity. We expressed a non-rectifying Arabidopsis thaliana potassium (K+) channel gene version, AKT2var, in the vascular tissue of cassava plants. The transgenic cassava plants had higher electron transport and CO2 assimilation rates, a higher bulk flow velocity, and increased source-sink carbohydrate transport, as demonstrated by comparative 11C-positron emission tomography and tissue-specific metabolite profiling. Cassava storage root yield was significantly increased in greenhouse experiments and in a multi-year field trial conducted under subtropical conditions. AKT2var plants were also more tolerant to drought stress and had higher storage root yield. Targeted alteration of K+ transport is therefore a promising strategy to improve cassava productivity without additional fertilizer input and in climate-adverse growing conditions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.26.655271v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Characterization of recombinase-based genetic parts and circuits using nanopore sequencing</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Greco, F. V., Cameron, S. K., Joshi, S. H.-N., Guiziou, S., Brophy, J. A. N., Grierson, C. S., Gorochowski, T. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a semi-automated workflow that combines low-cost liquid handling robotics, multiplexed long-read nanopore sequencing, and a supporting computational analysis tool to enable the high-throughput and detailed characterization of recombinase parts and circuits.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recombinases are versatile enzymes able to perform the precise insertion, deletion, and rearrangement of DNA and can act as a foundation for programmable genetic logic and memory. Fundamental to their use are accurate measurements of function. However, these are often laborious, time-consuming, and costly to collect. To address this, we developed a semi-automated workflow that combines low-cost liquid handling robotics, multiplexed long-read nanopore sequencing, and a supporting computational analysis tool to enable the high-throughput and detailed characterization of recombinase parts and circuits when used in a variety of contexts and organisms. Our approach overcomes the limitations of typically used fluorescence-based assays and is able to monitor temporal dynamics, observe structural changes at a nucleotide resolution, and unravel the internal workings of complex multi-state circuits. The ability to scale-up and automate genetic circuit characterization is an essential step towards more rigorous biological metrology that can support the construction of predictive models for efficiently engineering biology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.27.656450v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multi-scale early warning system for influenza A spillovers</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Alfonsi, T., Bernasconi, A., Chiara, M., Ceri, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce FluWarning, a highly efficient and elegant computational method based on anomaly detection of codon bias and dinucleotide composition for early flagging of divergent viral HA segments.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Spillovers of Influenza A viruses into farmed animals and humans have the potential to trigger epidemics or even global pandemics. We introduce FluWarning, a highly efficient and elegant computational method based on anomaly detection of codon bias and dinucleotide composition for early flagging of divergent viral HA segments. Applied to H1N1 specimens collected between 2008-2009, FluWarning accurately identifies the emergence of pdm09 - the virus that caused the 2009 flu pandemic; warnings precede the observed progression of the pandemic. Applied to H5N1 specimens collected between 2019 and 2025, FluWarning flagged the emergence of the B3.13 genotype, linked to a spillover event in dairy cows in the United States. In summary, FluWarning is an effective multi-scale warning system for IAVs, detecting spillovers with few available sequences.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.24.655955v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

