<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Saturday, 31 May 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The genetic architecture of an allosteric hormone receptor</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Stammnitz, M. R., Lehner, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We quantify the evolution of allosteric switches by quantifying the dose-response of a receptor and reveal the remarkable evolutionary malleability of an allosterIC switch.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Many proteins function as switches, detecting chemicals and transducing their concentrations into cellular responses. Receptor switches are key to the integration of environmental signals, yet it is not well understood how their signal processing is genetically encoded. Here, to address these questions, we present a complete map of how mutations quantitatively alter the input-output function of a receptor. Using a massively parallel approach, GluePCA, we quantify more than 40,000 binding measurements and 3,500 dose-response curves for variants of the allosteric phytohormone receptor PYL1. Strikingly, >91% of missense variants tune the dose-response of the receptor. Many variants drive changes in sensitivity, basal activity, maximum response and induction steepness in a correlated manner, suggesting a shared underlying biophysical mechanism, which we identify as stability changes. Beyond this, signalling parameters can be independently tuned by mutations, with surprisingly large effects in interface-distal positions and a modular genetic architecture across the receptor's structure. A small subset of missense variants confers qualitative phenotypic innovation, producing hypersensitive and inverted response patterns. Our data demonstrate the feasibility of dose-response profile quantification at massive scale and reveal the remarkable evolutionary malleability of an allosteric switch.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.30.656975v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MLOmics: Cancer Multi-Omics Database for Machine Learning.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ziwei Yang, Rikuto Kotoge, Xihao Piao, Zheng Chen, Lingwei Zhu, Peng Gao, Yasuko Matsubara, Yasushi Sakurai, Jimeng Sun</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce MLOmics, an open cancer multi-omics database aiming at serving better the development and evaluation of bioinformatics and machine learning models.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Framing the investigation of diverse cancers as a machine learning problem has recently shown significant potential in multi-omics analysis and cancer research. Empowering these successful machine learning models are the high-quality training datasets with sufficient data volume and adequate preprocessing. However, while there exist several public data portals, including The Cancer Genome Atlas (TCGA) multi-omics initiative or open-bases such as the LinkedOmics, these databases are not off-the-shelf for existing machine learning models. In this paper, we introduce MLOmics, an open cancer multi-omics database aiming at serving better the development and evaluation of bioinformatics and machine learning models. MLOmics contains 8,314 patient samples covering all 32 cancer types with four omics types, stratified features, and extensive baselines. Complementary support for downstream analysis and bio-knowledge linking are also included to support interdisciplinary analysis.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40447627/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A robust multiplex-DIA workflow profiles protein turnover regulations associated with cisplatin resistance and aneuploidy.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Barbora Salovska, Wenxue Li, Oliver M Bernhardt, Pierre-Luc Germain, Qinyue Wang, Tejas Gandhi, Lukas Reiter, Yansheng Liu</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a robust workflow that integrates a machine learning algorithm and channel-specific statistical filtering, enabling dynamic adaptation to channel ratio changes across multiplexed experiments and enhancing both coverage and accuracy of protein turnover profiling.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Quantifying protein turnover is fundamental to understanding cellular processes and advancing drug discovery. Multiplex-DIA mass spectrometry (MS), combined with dynamic SILAC labeling (pulse-SILAC, or pSILAC) reliably measures protein turnover and degradation kinetics. Previous multiplex-DIA-MS workflows have employed various strategies including leveraging the highest isotopic labeling channels to enhance the detection of isotopic signal pairs. Here we present a robust workflow that integrates a machine learning algorithm and channel-specific statistical filtering, enabling dynamic adaptation to channel ratio changes across multiplexed experiments and enhancing both coverage and accuracy of protein turnover profiling. We also introduce KdeggeR, a data analysis tool optimized for pSILAC-DIA experiments, which determines and visualizes peptide and protein degradation profiles. Our workflow is broadly applicable, as demonstrated on 2-channel and 3-channel DIA datasets and across two MS platforms. Applying this framework to an aneuploid cancer cell model before and after cisplatin resistance, we uncover strong proteome buffering of key protein complex subunits encoded by the aneuploid genome mediated by protein degradation. We identify resistance-associated turnover signatures, including mitochondrial metabolic adaptation via accelerated degradation of respiratory complexes I and IV. Our approach provides a powerful platform for high-throughput, quantitative analysis of proteome dynamics and stability in health and disease.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40447611/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrated plasma and vegetation proteomic characterization of infective endocarditis for early diagnosis and treatment.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shiman He, Xuejiao Hu, Jiajun Zhu, Weiteng Wang, Chi Ma, Peng Ran, Oudi Chen, Fanyu Chen, Hongkun Qing, Jianhong Ma, Danni Zeng, Yunzhi Wang, Weijiang Liu, Jinwen Feng, Lixi Gan, Zhaoyu Qin, Subei Tan, Sha Tian, Chen Ding, Xuhua Jian, Bing Gu</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a machine learning-based diagnostic and prognostic model for infective endocarditis, with area under the curve values of 0.98 and 0.87, respectively. Leucine-rich alpha-2 glycoprotein 1 and NADH:ubiquinone oxidoreduct</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Infective endocarditis, a life-threatening condition, poses challenges for early diagnosis and personalized treatment due to insufficient biomarkers and limited understanding of its pathophysiology. Here, we performed proteomic profiling of plasma and vegetation samples from 238 patients with infective endocarditis and 100 controls, with validation in two external plasma cohorts (n = 328). We developed machine learning-based diagnostic and prognostic models for infective endocarditis, with area under the curve values of 0.98 and 0.87, respectively. Leucine-rich alpha-2-glycoprotein 1 and NADH:ubiquinone oxidoreductase subunit B4 are potential biomarkers associated with infection severity. Pathologically, protein networks characterized by glycometabolism, amino acid metabolism, and adhesion are linked to adverse events. Liver dysfunction may exacerbate the condition in patients with severe heart failure. Neutrophil extracellular traps emerge as promising therapeutic targets in Streptococcus or Staphylococcus aureus infections. Our findings provide insights into biomarker discovery and pathophysiological mechanisms in infective endocarditis, advancing early diagnosis and personalized medicine.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40447567/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Harnessing Machine Learning for Agnostic Biodetection.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sarah H Sandholtz, Camilo Valdes, Nisha Mulakken, Marisa W Torres, Aram Avila-Herrera, Jeffrey A Drocco, Jose Manuel Mart√≠, Jonathan E Allen, Uttara Tipnis, Crystal J Jaing, Nicholas A Be</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We provide a systematic review of existing ML platforms and discuss anticipated development efforts needed to achieve effective ML-enabled, agnostic biodetection.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The United States' current list-based approach to biodefense is limited because it considers only known biological agents. Alternatively, developing and adopting a system based on agent-agnostic signatures would enable detection and characterization of both known and novel agents, thereby engendering greater adaptability in the face of an evolving threat landscape. Machine learning (ML) could aid in such a transition, as it can recognize and encode highly complex patterns from multiple input data modalities and has already demonstrated success in many healthcare and defense applications. Functionalizing ML for environmental biodetection requires understanding current technical capabilities. In this article, we provide a systematic review of existing ML platforms and discuss anticipated development efforts needed to achieve effective ML-enabled, agnostic biodetection.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40443394/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Finding an optimal sequencing strategy to detect short and long genetic variants in a human genome</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Eveleigh, R. J. M., Reiling, S. J., Galvez, J. H., Bourgey, M., Ragoussis, J., Bourque, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that while SRS combined with DeepVariant or DRAGEN offers excellent small variant detection in well-mapped regions, LRS technologies significantly outperform SRS in complex regions and SV detection.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Advances in DNA sequencing have transformed genomics, enabling comprehensive insights into human genetic variation. While short-read sequencing (SRS) remains dominant due to its high accuracy and affordability, its limitations in complex genomic regions have spurred the adoption of long-read sequencing (LRS) platforms, such as those from Pacific Biosciences (PacBio) and Oxford Nanopore Technologies (ONT). Despite these advances, there is still a lack of systematic, large-scale benchmarking of variant calling performance across diverse platforms, variant types, sequencing depths, and genomic contexts. Here, we present a comprehensive benchmark of sequencing technologies and variant calling algorithms, evaluating their performance in detecting single nucleotide polymorphisms (SNPs), insertions/deletions (indels), and structural variants (SVs). We show that while SRS combined with DeepVariant or DRAGEN offers excellent small variant detection in well-mapped regions, LRS technologies significantly outperform SRS in complex regions and SV detection. PacBio achieves high SNP and smaller SV accuracy even at moderate coverage, while ONT excels in detecting large SVs. The SV callers Dysgu and SVIM emerged as top performers across LRS datasets. Our results highlight that no single platform is optimal for all variant types or regions: SRS remains optimal for high-throughput small variant detection in accessible regions, whereas LRS is critical for capturing SVs and resolving difficult-to-map loci. These findings offer practical guidance for selecting sequencing technologies, coverage and variant calling strategies tailored to specific research or clinical goals, contributing to more accurate and cost-effective genomic analyses.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.30.656631v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Missense variants in human forkhead transcription factors reveal determinants of forkhead DNA bispecificity</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> King, J., Gisselbrecht, S. S., Dias, J.-A., Jeong, R. L., Rothman, E., Bulyk, M. L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that non-DNA-contacting amino acid residues in forkhead proteins control mono- versus bispecificity of DNA binding activity and may play a role in the evolution of TF DNA binding activities.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recognition of specific DNA sequences by transcription factors (TFs) is a key step in transcriptional control of gene expression. While most forkhead (FH) TFs bind either an FKH (RYAAAYA) or an FHL (GACGC) recognition motif, some FHs can bind both motifs. Mechanisms that control whether a FH is monospecific versus bispecific have remained unknown. Screening a library of 12 reference FH proteins, 61 naturally occurring missense variants including clinical variants, and 22 designed mutant FHs for DNA binding activity using universal ("all 10-mer") protein binding microarrays (PBMs) revealed non-DNA-contacting residues that control mono- versus bispecificity. Variation in non-DNA-contacting amino acid residues of TFs is associated with human traits and may play a role in the evolution of TF DNA binding activities and gene regulatory networks.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.27.656303v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Unpaired Image-to-Image Translation for Segmentation and Signal Unmixing</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Andrejic, N., Spasic, M., Mihajlovic, I., Milosavljevic, P., Pavlovic, D., Milisavljevic, F., Milivojevic, U., Delibasic, D., Mikic, I., Todorovic, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel model for unpaired image-to-image translation, trained on content-wise unpaired datasets to enable style transfer across domains while preserving content.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> This work introduces Ui2i, a novel model for unpaired image-to-image translation, trained on content-wise unpaired datasets to enable style transfer across domains while preserving content. Building on CycleGAN, Ui2i incorporates key modifications to better disentangle content and style features, and preserve content integrity. Specifically, Ui2i employs U-Net-based generators with skip connections to propagate localized shallow features deep into the generator. Ui2i removes feature-based normalization layers from all modules and replaces them with approximate bidirectional spectral normalization - a parameter-based alternative that enhances training stability. To further support content preservation, channel and spatial attention mechanisms are integrated into the generators. Training is facilitated through image scale augmentation. Evaluation on two biomedical tasks - domain adaptation for nuclear segmentation in immunohistochemistry (IHC) images and unmixing of biological structures superimposed in single-channel immunofluorescence (IF) images - demonstrates Ui2i's ability to preserve content fidelity in settings that demand more accurate structural preservation than typical translation tasks. To the best of our knowledge, Ui2i is the first approach capable of separating superimposed signals in IF images using real, unpaired training data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.26.656226v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Neurons accumulate disease-specific somatic genomic changes across taupathologic states in Alzheimer's disease</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jin, B., Brown, K. S.- M., Smirnov, D. S., Naik, S. M., Kirkham, S. L., Hennessey, E. L., Dong, G., Mao, S., Wijethunga, S. P., Stewart, T. C., Selkoe, D. J., Frosh, M. P., Oakley, D. H., Hyman, B. T., Huang, A. Y., Miller, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that somatic mutations are associated with tissue-wide tau pathology and that non-tangle components drive somatic mutation in AD, suggesting that tangles do not confer cell-autonomous genotoxicity to neurons.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Tau deposition within neurons marks Alzheimer's disease (AD) neuropathology, suggesting that tau may contribute to cellular dysfunction and death. In AD, somatic mutations accumulate in neurons, with features that suggest deleterious effects on cellular function. To examine the relationship between tau and somatic mutation, we isolated neurons according to tau pathology and performed single-cell whole-genome sequencing on tau+, tau-, and tau-agnostic neurons from 13 individuals with AD, as well as neurons from 15 control individuals. We found that AD neurons, regardless of tau status, exhibited an increased burden of somatic single-nucleotide variants (sSNVs) and insertions and deletions (sIndels). Mutational signature analyses revealed disease-specific patterns, including sIndels characterized by two-basepair (2bp) deletions, indicating shared mutagenic mechanisms in AD neurons across tau pathologic cell states. Somatic mutations are associated with tissue-wide tau pathology, suggesting that tangles do not confer cell-autonomous genotoxicity to neurons and that non-tangle components drive somatic mutation in AD.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.26.656152v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">scCoBench: Benchmarking single cell RNA-seq co-expression using promoter-reporter lines</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chau, T. N., Ryu, K. H., Alajoleen, R., Bargmann, B. O. R., Schiefelbein, J., Li, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We systematically benchmark correlation metrics, pseudo bulk analysis, and imputation methods using promoter-reporter and native gene pairs as internal controls to evaluate the performance of ten widely used gene-gene co-expression measurements.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell RNA sequencing (scRNA-seq) has become a powerful tool for uncovering transcriptomic heterogeneity and reconstructing gene regulatory networks in complex tissues. However, the sparsity, high noise levels, and dropout events inherent to scRNA-seq data pose challenges for accurate inference of gene-gene relationships. In this study scCoBench, we systematically benchmark correlation metrics, pseudo bulk analysis, and imputation methods using promoter-reporter and native gene pairs as internal controls to evaluate the performance of ten widely used gene-gene co-expression measurements. Interestingly, we found that commonly used data scaling and normalization approaches lead to lower correlation between promoter reporter and native gene pairs in most of the co-expression methods. Moreover, we assess the impact of five popular imputation techniques, including scImpute, SAVER, Autoencoder (AE), Variational Autoencoder (VAE), and Generative Adversarial Network (GAN) on recovering biologically relevant co-expression patterns. Our results demonstrate that imputation models not only markedly enhance correlation between each promoter-reporter and native gene pair but also increase the number of cells co-expressing both genes. Imputation also improved transcription factor target gene correlations and revealed stronger associations among genes within the same protein complex. This work highlights the utility of promoter-reporter systems for benchmarking computational methods and underscores the potential of deep learning-based imputation to improve the biologically relevant signals of scRNA-seq data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.26.656221v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">How Good is AlphaFold3 at Ranking Drug Binding Affinities?</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hong, X., Gao, B., Jia, Y., Zhu, W., Chen, Q., Tian, X., Zhong, Z., Wang, J., Lan, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that structure prediction models like AlphaFold3, pretrained on protein-ligand interactions, can address the affinity ranking task of small molecules and demonstrate the emergent potential of AlphaRank, a deep learning-based model for affinity ranking tasks.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurate affinity ranking of small molecules is pivotal for drug discovery. We investigate whether structure prediction models like AlphaFold3, pretrained on protein-ligand interactions, can address this task. Zero-shot evaluation of Protenix (an AlphaFold3-like model) demonstrates superior prioritization of active compounds over conventional scoring functions and state-of-the-art deep learning models. By further fine-tuning Protenix on structure-agnostic protein-ligand bioactivity data from ChEMBL and BindingDB, we develop AlphaRank that predicts pairwise affinity relationships. AlphaRank achieves prediction accuracy comparable to computationally intensive free energy perturbation (FEP+) workflows on standard benchmarks, while requiring substantially less computational resources. Our findings highlight the emergent potential of AlphaFold3-derived models in affinity ranking tasks and emphasize the necessity for targeted methodological exploration to fully harness their capabilities in drug discovery applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.27.656341v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genomic and evolutionary factors influencing the prediction accuracy of optimal growth temperature in prokaryotes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Toki, S., Matsui, M., Tominaga, K., Suzuki, T. K., Tsuchimatsu, T., Iwasaki, W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a genome composition-based phenotype prediction model for optimal growth temperatures (OGT) and found that rapid OGT shifts from their ancestors, including psychrophiles, showed less accuracy in genome composition based models, while those with phylogenetically conserved OGT are more accurate.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Bacteria and archaea have evolved diverse genomic adaptations to thrive across various temperatures. These adaptations include genome sequence optimizations, such as increased GC content in rRNA and tRNA, shifts in codon and amino acids usage, and the acquisition of functional genes conferring adaptation for specific temperatures. Since the experimental determination of optimal growth temperatures (OGT) is only possible for cultured species, predicting OGT from genomic information has become increasingly important given the exponential increase in genomic data. Although previous studies developed prediction models integrating multiple features based on genome composition using machine learning, the accuracy was variable depending on the target species, with models performing well for thermophiles but less accurately for psychrophiles. In this study, we curated the OGT and genomic data of 2,869 bacterial species to develop a novel prediction model incorporating features reflecting genomic adaptation toward lower temperatures. We found that species with rapid OGT shifts from their ancestors, including psychrophiles, showed less accuracy in genome composition-based models. Incorporating the gene presence/absence information associated with the rapid changes in OGT improved the prediction accuracy for psychrophiles. We also observed that OGT in archaea is phylogenetically more conserved than in bacteria, which may lead to the long-term optimization of the genome composition and explain high predictability of OGT in archaea. These findings highlight the importance of integrating long- and short-term evolutionary adaptations for phenotype prediction models.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.30.656958v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Phased epigenomics and methylation inheritance in a historical Vitis vinifera hybrid</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cochetel, N., Vondras, A., Figueroa-Balderas, R., Liou, J., Peluso, P., Cantu, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A phased sequence graph for epigenetic methylation in perennial grapevine cultivars .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Epigenetic modifications, such as DNA methylation, regulate transcription and influence key biological traits. While many efforts were made to understand their stability in annual crops, their long-term persistence in clonally propagated plants remains poorly understood. Grapevine (Vitis vinifera) provides a unique model, with cultivars vegetatively propagated for centuries. Here, we assembled the phased genomes of Cabernet Sauvignon and its parental lineages, Cabernet Franc and Sauvignon Blanc, using HiFi long-reads and a gene map tenfold denser than existing maps. Using three clones per cultivar, we quantifyied methylation with very consistent short- and long-read sequencing and ensured both varietal representativeness and assessment of clonal variability. We leveraged the parent-progeny sequence graph to highlight allele-specific methylation and conserved transcriptomic patterns for genes and small RNA. Such a format was essential to integrate multi-omics data and revealed that despite less clonal conservation than genetic polymorphisms, methylation marks were remarkably inherited. By further demonstrating the linear-reference limitations, we determined the correct representation of genetic variants by the sequence graph is crucial for the accurate allelic quantification of the methylome. These findings reveal the remarkable stability of epigenetic marks in a model propagated by asexual reproduction. Using a phased sequence graph, we introduce a scalable framework that accounts for genomic variation, accurately quantifies allele-specific methylation, and supports multi-omics integration such as our evaluation of the transcriptional impact of epigenetic inheritance. This approach has broad implications for perennial crops, where epigenetic variation could influence traits relevant to breeding, adaptation, and long-term agricultural sustainability.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.27.656431v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Diffusion-Based Framework for Designing Molecules in Flexible Protein Pockets</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, J., Dokholyan, N. V.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce YuelDesign, a novel diffusion-based framework specifically developed to address the design of molecules for flexible protein pockets, as proteins often undergo conformational changes upon ligand binding.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The design of molecules for flexible protein pockets represents a significant challenge in structure-based drug discovery, as proteins often undergo conformational changes upon ligand binding. While deep learning-based approaches have shown promise in molecular generation, they typically treat protein pockets as rigid structures, limiting their ability to capture the dynamic nature of protein-ligand interactions. Here, we introduce YuelDesign, a novel diffusion-based framework specifically developed to address this challenge. YuelDesign employs a new protein encoding scheme with a fully connected graph representation to encode protein pocket flexibility, a systematic denoising process that refines both atomic properties and coordinates, and a specialized bond reconstruction module tailored for de novo generated molecules. Our results demonstrate that YuelDesign generates molecules with favorable drug-likeness and low synthetic complexity. The generated molecules also exhibit diverse chemical functional groups, including some not even present in the training set. Redocking analysis reveals that the generated molecules exhibit docking energies comparable to native ligands. Additionally, a detailed analysis of the denoising process shows how the model systematically refines molecular structures through atom type transitions, bond dynamics, and conformational adjustments. Overall, YuelDesign presents a versatile framework for generating novel molecules tailored to flexible protein pockets, with promising implications for drug discovery applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.27.656443v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Changing the initiation unit of nonribosomal peptide synthetases to access underexplored biosynthetic potential</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bai, X., Zhong, L., Chen, H., Liu, Y., Shi, X., Wang, X., Yang, Q., Diao, X., Wu, D., Zhang, Y., Bian, X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel strategy to recover silent NRPS biosynthetic gene clusters by replacing the initiation region of endopyrrole pathway with a Cs-containing initiation unit for generation of a lipo-endopyr role derivate.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Nonribosomal peptide synthases (NRPSs) are large multimodular enzymes, capable of synthesizing nonribosomal peptides (NRPs) with diverse structures and bioactivities. Genome sequencing revealed a large number of uncharacterized NRPS biosynthetic gene clusters (BGCs) and their products are underexplored. The majority of NRPSs remain silent potentially attributed to factors such as the low activity of the initiation unit or insufficient precursor supply. Exchanging the starter condensation (Cs) domains within initiation unit can change the length of acyl chains of NRPs, hinting at a promising strategy through swapping of a well-studied Cs domain to activate the initiation unit and harness primary metabolites as precursors, which may offer a new option to access silent BGCs. Here, we first pinpointed two highly efficient fusion sites for initiation unit exchanges. Subsequently, they were validated by replacing the initiation region of endopyrrole pathway with a Cs-containing initiation unit for generation of a lipo-endopyrrole derivate. We promptly leveraged this strategy of changing initiation unit to target six previously silent NRPS BGCs, three BGCs were successfully activated and five novel lipopeptides were identified, demonstrating its application to recover silent BGCs. Furthermore, we extended this strategy in more BGCs from different bacteria. Utilizing a heterologous Cs-containing unit to replace the initiation region of chitinimide biosynthetic pathway led to successful incorporation of N-terminal fatty acid chains into chitinimide to create artificial lipo-chitinimides. This study provides a feasible strategy to rationally recover silent BGCs and add fatty acid chains to NRPs, enriching the genome mining and combinatorial biosynthesis approach for bacterial natural products.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.29.656725v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SenolyticSynergy: An Attention-Based Network for Discovering Novel Senolytic Combinations via Human Aging Genomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ye, Y., Ming, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify and predict effective senolytic drug combinations for anti-aging drugs, and show that the combination of temsirolimus and nitazoxanide may be the most promising candidate.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Senolytics, a category of drugs targeting aging processes, have garnered significant attention since their emergence in 2015. Unlike traditional drug development approaches that rely on randomized screening, research on aging-related pharmaceuticals has employed mechanism-based strategies, resulting in the discovery of the pioneering combination therapy of dasatinib (D) and quercetin (Q). Although preliminary studies with senolytic drug combinations have shown promising outcomes, the predictive capabilities of the research in this field remain limited by the extensive experimental data requirements. In this study, we employed differential gene expression analysis and machine learning techniques to investigate the combinatorial effects of senolytic drugs. We identified 1624 core aging-related genes and used this dataset to retrain a multimodal attention mechanism model, creating a specialized framework: SenolyticSynergy for predicting effective senolytic drug combinations. We then utilized 63 established senolytic compounds as starting points for combination testing, developing a comprehensive dataset of 1953 potential drug combinations for aging interventions. Following rigorous filtration, we identified 190 high-confidence drug combinations and predicted their synergistic scores. Among these combinations, ten demonstrated exceptionally high synergistic scores, exceeding 8. The combination of temsirolimus and nitazoxanide ranked first and may be the most promising candidate. The analysis of literature data and computational studies of molecular structures using 3D modeling validated the accuracy of these predictions. This framework paves the way for large-scale research into anti-aging drug combinations, advancing research capabilities in this field.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.28.655258v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">De novo design of hundreds of functional GPCR-targeting antibodies enabled by scaling test-time compute</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nabla Bio,, Biswas, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present significant advances in de novo antibody design against G protein-coupled receptors (GPCRs) enabled by scaling the test-time compute used by our generative protein design system, JAM.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We present significant advances in de novo antibody design against G protein-coupled receptors (GPCRs) enabled by scaling the test-time compute used by our generative protein design system, JAM. We de novo design hundreds of VHH (single domain) antibodies against CXCR4 and CXCR7, with top designs showing picomolar to low-nanomolar affinities, high selectivity, and favorable early-stage developability profiles, matching or outperforming clinical-stage molecules in these dimensions. Further, high affinity designs potently modulate receptor function, with most acting as antagonists (inhibitors) and, strikingly, a subset functioning as agonists (activators) of CXCR7 - the first antibody agonists reported for this receptor, and the first computationally designed antibody GPCR agonists of any kind. Using a single experimentally validated agonist to further prompt JAM, we generate over 300 additional diverse agonists with superior properties, including a design with agonism EC50 rivaling that of CXCR7s natural ligand, SDF1. These results show that increasing the reasoning capacity of biomolecular generative models by scaling test-time computation will enable them to solve increasingly difficult problems in drug design.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.28.656709v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Parameter-Efficient Fine-Tuning of a Supervised Regulatory Sequence Model</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yuan, H., Linder, J., Kelley, D. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A transfer learning framework based on parameter-efficient fine-tuning (PEFT) approaches for supervised regulatory sequence models for gene regulation and genetic variant effects .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> DNA sequence deep learning models accurately predict epigenetic and transcriptional profiles, enabling analysis of gene regulation and genetic variant effects. While large-scale training models like Enformer and Borzoi are trained on abundant data, they cannot cover all cell states and assays, necessitating new model training to analyze gene regulation in novel contexts. However, training models from scratch for new datasets is computationally expensive. In this study, we systematically evaluate a transfer learning framework based on parameter-efficient fine-tuning (PEFT) approaches for supervised regulatory sequence models. We focus on the recently published state-of-the-art model Borzoi and fine-tune to new RNA-seq datasets, enabling accurate and efficient analysis of gene regulation across different biological systems. Our results demonstrate that PEFT substantially improves memory and runtime efficiency while achieving high accuracy. The transferred models effectively predict held-out gene expression changes, identify regulatory drivers of differentially expressed genes, and predict cell-type-specific variant effects. Our findings underscore the potential of PEFT to enhance the utility of large-scale sequence models like Borzoi for broader application to study genetic variants and regulatory drivers in any functional genomics dataset. Code for efficient Borzoi transfer is available in the Baskerville codebase https://github.com/calico/baskerville.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.26.656171v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Sarcomere analysis in human cardiomyocytes by computing radial frequency spectra</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Habeck, M., Saleem, H. N., Plota, D., Cheruiyot, C., Kohl, T., Jakobs, S., Lehnart, S. E., Ebert, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that radial frequency spectra computing (RFSC) method can efficiently analyze sarcomere properties in a human cardiomyocyte model, iPSC-CMs, and compare RFSC to a different method.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In cardiomyocytes, the basic contractile unit are sarcomeres, which are organized in a regular manner facilitating their function. Here, we present a new analysis approach to assess the functional properties of sarcomeres at the nanoscale level in human cardiac cells, induced pluripotent stem cell-derived cardiomyocytes (iPSC-CMs). We combined our analysis mode for application to different types of high-resolution imaging data, structured illumination microscopy (SIM), stimulated emission depletion (STED) microscopy-based imaging, as well as confocal microscopy data. We show that our radial frequency spectra computing (RFSC) method can efficiently analyze sarcomere properties in a human cardiomyocyte model, iPSC-CMs, and compare RFSC to a different method. Moreover, the RFSC method is suitable to quantify molecular differences of sarcomeres such as present in severe cardiac diseases, such as dilated cardiomyopathy (DCM). Defects in the sarcomere organization as occur in the presence of inherited DCM mutations in sarcomere proteins are efficiently recapitulated by our analysis approach. This new approach may facilitate streamlined analysis of molecular disease-specific phenotypic imaging data of cardiac cells, aiding our deeper understanding of the molecular basis of cardiac diseases.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.28.655290v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SIREN: Suite for Intelligent RNAi Design and Evaluation of Nucleotide Sequences</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Vargas Mejia, P., Vega-Arreguin, J. C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present SIREN, an open-source Python pipeline designed to streamline RNAi construct design.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: RNA interference (RNAi) is a powerful tool for gene silencing across biological research, therapeutics, and agriculture. While siRNA design has benefited from advances in thermodynamic modeling and machine learning, comprehensive tools for designing long double-stranded RNAs (dsRNAs) with minimized off-target effects remain limited. Results: Here, we present SIREN, an open-source Python pipeline designed to streamline RNAi construct design. SIREN integrates siRNA generation, thermodynamically-informed off-target prediction, scoring of dsRNA candidates based on cumulative off-target effects, and primer design for in vitro synthesis. It accepts user-defined transcriptomes for context-specific analysis and provides adjustable sensitivity settings balancing accuracy and computational demands. Benchmarking with plant, oomycete, and human transcriptomes demonstrates SIRENs efficient scalability and the practical utility of medium sensitivity, recovering over 75% of high-sensitivity targets with significantly reduced computing times. Experimental validation in Phytophthora capsici confirms that SIREN effectively identifies highly specific RNAi constructs with no detectable off-target phenotypes in host plants. Availability and implementation: SIREN is implemented in Python 3 and available under an open-source license at https://github.com/pablovargasmejia/SIREN; installer via PyPI: https://pypi.org/project/siren-rnai/.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.26.656188v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

