<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Friday, 23 May 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrated transcriptomic analyses identifies host-targeting repurposing drugs for hepatitis C virus infection and related hepatocellular carcinoma</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kamboj, S., Kumar, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> This study identified prospective repurposing drugs using host-targeting approach for HCV infection and related HCC treatment.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Hepatitis C virus (HCV) infection is a major risk factor in developing hepatocellular carcinoma (HCC). Drug development for HCV infection and HCC treatment is needed owing to treatment failure due to viral resistance and tumor heterogeneity. We analysed transcriptomic data of human liver tissues from multiple studies to identify differentially expressed genes in four study groups: (HCV+ vs Control); (HCV+HCC+ vs Control); (HCV+ AND HCV+HCC+ vs Control); and (HCV+HCC+ vs HCV+) respectively. We identified 597, 1321, 923, and 1526 upregulated and 474, 558, 336, and 989 downregulated genes for above respective studied groups. We performed functional, pathway enrichment, and protein-protein interaction analyses for identified upregulated genes. We identified and prioritized 39, 57, 58, and 110 repurposing drugs for 09, 17, 14, and 33 non-essential upregulated target genes based on drug group, action, and literature evidence. We also performed mutational analysis of key target genes. We found several dysregulated genes such as CTLA4, GPC3, MDK, TUBB2B etc. and many novel target genes like CACNB4, HTR7, IGHG1, SLC22A12, etc. We identified several potential repurposing drugs namely asenapine, cabergoline, dequalinium, epinastine, methysergide, loxapine, lurasidone, etc. This study identified prospective repurposing drugs using host-targeting approach for HCV infection and related HCC treatment.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.17.654645v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Enhancing healthcare AI stability with edge computing and machine learning for extubation prediction.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kuo-Yang Huang, Ying-Lin Hsu, Che-Liang Chung, Huang-Chi Chen, Ming-Hwarng Horng, Ching-Hsiung Lin, Ching-Sen Liu, Jia-Lang Xu</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present an edge-based framework that incorporates machine learning algorithms to predict ventilator extubation success using real-time data collected directly from ventilators.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The advancement of the Internet of Medical Things (IoMT) has revolutionized data acquisition and processing in critical care settings. Given the pivotal role of ventilators, accurately predicting extubation outcomes is essential to optimize patient care. This study presents an edge computing-based framework that incorporates machine learning algorithms to predict ventilator extubation success using real-time data collected directly from ventilators. The system was deployed on edge devices to enable on-site inference with minimal latency. Among the evaluated models, Random Forest and XGBoost, the latter demonstrated superior predictive performance under both holdout and tenfold cross-validation schemes. Notably, the edge-based architecture reduced server data transmissions by 83.33%, while improving system stability, resilience, and sustainability. This paper details the model evaluation and demonstrates the feasibility and efficiency of edge intelligence in ventilator weaning decision support.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40404764/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Imputing single-cell protein abundance in multiplex tissue imaging.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Raphael Kirchgaessner, Cameron Watson, Allison Creason, Kaya Keutler, Jeremy Goecks</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We apply machine learning to impute single-cell protein abundance using multiplex tissue imaging data from a breast cancer cohort, using regularized linear regression, gradient-boosted trees, and deep learning autoencoders.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Multiplex tissue imaging enables single-cell spatial proteomics and transcriptomics but remains limited by incomplete molecular profiling, tissue loss, and probe failure. Here, we apply machine learning to impute single-cell protein abundance using multiplex tissue imaging data from a breast cancer cohort. We evaluate regularized linear regression, gradient-boosted trees, and deep learning autoencoders, incorporating spatial context to enhance imputation accuracy. Our models achieve mean absolute errors between 0.05-0.3 on a [0,1] scale, closely approximating ground truth values. Using imputed data, we classify single cells as pre- or post-treatment, demonstrating their biological relevance. These findings establish the feasibility of imputing missing protein abundance, highlight the advantages of spatial information, and support machine learning as a powerful tool for improving single-cell tissue imaging.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40404617/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PCAGP, a parallel convolutional attention network-based method for crop genomic prediction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Peng, W., Sheng, Y., Zhou, Y., Chai, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel deep learning framework for crop genomic prediction based on a parallel convolutional attention network, which significantly improves the accuracy of phenotypic prediction.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genomic selection (GS) has emerged as a transformative breeding paradigm, leveraging genome-wide marker data to predict phenotypic outcomes and accelerate crop improvement cycles. Traditional statistical models face fundamental limitations in modeling the higher-order, non-linear genotype-phenotype relationships inherent in complex agronomic traits. Deep learning architectures have recently shown remarkable predictive performance in GS applications. Current deep learning approaches in GS predominantly employ a serial structure convolution kernel, this may lead to gradient vanishing and computational efficiency challenges. In this study, we propose a method based on a parallel convolutional attention network for crop genomic prediction (PCAGP), a novel deep learning framework for crop genomic prediction. The architecture uses a convolution kernel to transmit information through a parallel structure, combined with a coordinate attention mechanism that effectively integrates inter-channel relationships and spatial genomic information. This approach significantly improves the accuracy of phenotypic prediction. We compared PCAGP with six widely used GS methods on four benchmark datasets, including statistical method (GBLUP), machine learning approaches (SVR, RF), and deep learning models (DeepGS, DNNGP, and SoyDNGP). The experimental results demonstrate that PCAGP outperforms all baseline methods, and the average prediction accuracy is improved (0.4%-53.1%).</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.17.654636v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AttentionAML: An Attention-based Deep Learning Framework for Accurate Molecular Categorization of Acute Myeloid Leukemia</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Li, L., Khoury, J. D., Wang, J., Wan, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose AttentionAML, a novel attention-based deep learning framework for accurately categorizing AML subtypes based on transcriptomic profiling only.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Acute myeloid leukemia (AML) is an aggressive hematopoietic malignancy defined by aberrant clonal expansion of abnormal myeloid progenitor cells. Characterized by morphological, molecular, and genetic alterations, AML encompasses multiple distinct subtypes that would exhibit subtype-specific responses to treatment and prognosis, underscoring the critical need of accurately identifying AML subtypes for effective clinical management and tailored therapeutic approaches. Traditional wet lab approaches such as immunophenotyping, cytogenetic analysis, morphological analysis, or molecular profiling to identify AML subtypes are labor-intensive, costly, and time-consuming. To address these challenges, we propose AttentionAML, a novel attention-based deep learning framework for accurately categorizing AML subtypes based on transcriptomic profiling only. Benchmarking tests based on 1,661 AML patients suggested that AttentionAML outperformed state-of-the-art methods across all evaluated metrics (accuracy: 0.96, precision: 0.96, recall of 0.96, F1-score: 0.96, and Matthews correlation coefficient: 0.96). Furthermore, we also demonstrated the superiority of AttentionAML over conventional approaches in terms of AML patient clustering visualization and subtype-specific gene marker characterization. We believe AttentionAML will bring remarkable positive impacts on downstream AML risk stratification and personalized treatment design. To enhance its impact, a user-friendly Python package implementing AttentionAML is publicly available at https://github.com/wan-mlab/AttentionAML.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.20.655179v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A complete reference genome assembly and annotation of the Black Redstart (Phoenicurus ochruros)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ghimire, P., Wang, N., Lamichhaney, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the first reference genome of the Black Redstart (Phoenicurus ochruros), generated using PacBio HiFi long-read sequencing.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The Black Redstart (Phoenicurus ochruros) is one of the most widely distributed species, occupying diverse habitats and exhibiting remarkable altitudinal migration, making it suitable model for studying altitudinal migration and high-altitude adaptation. In this study, we present the first reference genome of Phoenicurus ochruros, generated using PacBio HiFi long-read sequencing. The nuclear genome is 1.37 Gb in length, assembled into 296 scaffolds with a scaffold N50 of 29.9 Mb. Multiple complementary genome validation approaches - including BUSCO analysis, mapping of raw PacBio HiFi reads, and comparative genomics analysis - confirmed the high contiguity and gene-space completeness of the genome. The Black Redstart genome contains one of the highest proportions of transposable elements among passerines (30.58%), second only to Bell's Sparrow (Artemisiospiza belli), 31.2%. We further integrated RNA-seq data, protein homology evidence, and ab initio gene prediction to build high-quality genome annotation of 18,609 genes in the P. ochruros genome. This genomic resource provides invaluable resource for evolutionary genomics studies of passerines and explore genetic basis of high-altitude adaptation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.21.655187v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">cellSight: Characterizing dynamics of cells using single-cell RNA-sequencing</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chatterjee, R., Gohel, C., Shook, B. A., Rahnavard, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose the automated workflow cellSight, which integrates high-throughput sequencing in a user-friendly platform. We reduce researcher workload, improve reproducibility, and accelerate discoveries in single-cell biology.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell analysis has transformed our understanding of cellular diversity, offering insights into complex biological systems. Yet, manual data processing in single-cell studies poses challenges, including inefficiency, human error, and limited scalability. To address these issues, we propose the automated workflow cellSight, which integrates high-throughput sequencing in a user-friendly platform. By automating tasks like cell type clustering, feature extraction, and data normalization, cellSight reduces researcher workload, promoting focus on data interpretation and hypothesis generation. Its standardized analysis pipelines and quality control metrics enhance reproducibility, enabling collaboration across studies. Moreover, cellSight's adaptability supports integration with emerging technologies, keeping pace with advancements in single-cell genomics. cellSight accelerates discoveries in single-cell biology, driving impactful insights and clinical translation. It is available with documentation and tutorials at https://github.com/omicsEye/cellSight.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.16.654572v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Age-informed, attention-based weakly supervised learning for neuropathological image assessment</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Li, S., Malamut, M., McKee, A., Cherry, J., Tian, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose an age-informed, attention-based multiple instance learning (MIL) pipeline to predict AT8 density, a key marker of p-tau aggregation in CTE.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Chronic Traumatic Encephalopathy (CTE) and other neurodegenerative disorders (NDs) pose diagnostic challenges due to their diffuse and subtle pathological changes. Traditional diagnostic methods relying on manual histopathological slide inspection are labor-intensive and prone to variability, often missing subtle structural alterations. This study introduces an age-informed, attention-based multiple instance learning (MIL) pipeline to predict AT8 density, a key marker of p-tau aggregation in CTE. Using Luxol Fast Blue and Hematoxylin & Eosin (LH&E) stained images, our model identifies critical pathological regions and generates interpretable attention maps highlighting structural changes linked to tau pathology. Incorporating patient age enhances predictive accuracy and contextual understanding, addressing aging's confounding effects. We also develop quantitative evaluation procedures for foundation models (FMs), assessing attention map smoothness, faithfulness, and robustness to perturbations like stain variability and noise. These benchmarks facilitate informed FM selection and optimization for neuropathological tasks. By enabling scalable, automated whole-slide image (WSI) analysis, our approach advances digital neuropathology, supporting earlier and more precise ND diagnoses and uncovering subtle markers with potential applications in clinical imaging.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.16.654510v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Deep-learning-enabled multi-omics analyses for prediction of future metastasis in cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ma, Q., Wang, X., Duan, M., Su, P.-L., Li, J., Krull, J., Jin, J., Chen, H., Sun, Y., Wu, W., He, K., Carpenter, R., Zhang, C., Cao, S., Xu, D., Wang, G., Li, L., Xin, G., Carbone, D. P., Li, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present EmitGCL, a deep-learning framework that accurately predicts future metastasis and its corresponding biomarkers and a potential therapeutic target for deterring metastasis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Metastasis remains the leading cause of cancer-related mortality, yet predicting future metastasis is a major clinical challenge due to the lack of validated biomarkers and effective assessment methods. Here, we present EmitGCL, a deep-learning framework that accurately predicts future metastasis and its corresponding biomarkers. Based on a comprehensive benchmarking comparison, EmitGCL outperformed other computational tools across six cancer types from seven cohorts of patients with superior sensitivity and specificity. It captured occult metastatic cells in a patient with a lymph node-negative breast cancer, who was declared to have no evidence of disease by conventional imaging methods but was later confirmed to have a metastatic disease. Notably, EmitGCL identified HSP90AA1 and HSP90AB1 as predictable biomarkers for future breast cancer metastasis, which was validated across five independent cohorts of patients (n=420). Furthermore, we demonstrated YY1 transcription factor as a key driver of breast cancer metastasis which was validated through in-silico and CRISPR-based migration assays, suggesting that YY1 is a potential therapeutic target for deterring metastasis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.16.654579v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Trophic and temporal dynamics of macrophage biology in human inner ear organogenesis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Deng, Y., Ehiogu, B., Luca, E., Dabdoub, A., Le Cao, K.-A., Wells, C., Nayagam, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify seven distinct macrophage subtypes in the human inner ear and show that they are seeded from both the yolk sac and fetal liver, corroborating studies in mouse.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent single-cell transcriptomic approaches are uncovering the breadth and depth of cell diversity within the mammalian inner ear. Macrophages, detected from gestational week seven in the human inner ear, persist into adulthood, and yet remain poorly understood in terms of their origin and function. Using self-generated and public scRNA-seq data, we identify seven distinct macrophage subtypes spanning fetal weeks 7.5 to 16.4 and adulthood. Each macrophage subtype is linked to specific developmental stages and displays a unique gene expression profile. These findings corroborate earlier histological evidence of resident and non-resident macrophages in both the developing and adult human cochlea. We also show that the human inner ear is seeded by macrophages from both yolk sac and fetal liver, corroborating studies in mouse. By analyzing ligand-receptor interactions, we highlight potential macrophage contributions to inner ear organogenesis. This research provides new insights into the diverse roles of human inner ear macrophages.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.16.654631v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Guided Co-clustering Transfer Across Unpaired and Paired Single-cell Multi-omics Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Li, H., Liu, Y., ZENG, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce textbf{Guided Co-clustering Transfer (GuidedCoC)}, a novel unsupervised framework that transfers structural knowledge from unpaired scRNA-seq source data to improve both cell clustering and feature alignment in paired sc RNA-seq/scATAC-</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell multi-omics technologies enable the simultaneous profiling of gene expression and chromatin accessibility, providing complementary insights into cellular identity and gene regulatory mechanisms. However, integrating paired scRNA-seq and scATAC-seq data remains challenging due to inherent sparsity, technical noise, and the limited availability of high-quality paired measurements. In contrast, large-scale unpaired scRNA-seq datasets often exhibit robust and biologically meaningful cell cluster structures. We introduce textbf{Guided Co-clustering Transfer (GuidedCoC)}, a novel unsupervised framework that transfers structural knowledge from unpaired scRNA-seq source data to improve both cell clustering and feature alignment in paired scRNA-seq/scATAC-seq target data. GuidedCoC jointly co-clusters cells and features across modalities and domains via a unified information-theoretic objective, aligning gene expression modules with regulatory elements while implicitly performing cross-modal dimensionality reduction to reduce noise. Additionally, it automatically aligns cell types across unpaired and paired datasets without requiring explicit annotations. Extensive experiments on multiple benchmark datasets demonstrate that GuidedCoC achieves superior clustering accuracy and biological interpretability compared to existing methods. These results highlight the promise of structure-guided, unsupervised transfer learning for robust, scalable, and interpretable integration of single-cell multi-omics data. Code is available at ~url{https://github.com/No-AgCl/GuidedCoC}.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.16.654635v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Optimizing Oyster Breeding with Machine Learning and BigData for Superior Quality</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nayak, P., Mishra, D. R., Swain, R., Sahoo, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel approach combining machine learning with multi-omics data genomics, transcriptomics, and proteomics to optimize oyster breeding for resilience and quality.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Oyster aquaculture is a vital component of global marine ecosystems and food production, yet winter mortality events threaten both ecological stability and economic viability. Traditional selective breeding methods, reliant on phenotypic traits and slow generational cycles, struggle to address these challenges efficiently. This study introduces an innovative approach integrating machine learning with multi-omics data genomics, transcriptomics, and proteomics to optimize oyster breeding for resilience and quality. By analyzing high-resolution datasets encompassing genetic markers, environmental stressors, and survival metrics, our ML models identified key SNPs linked to cold tolerance and disease resistance. Marker-assisted selection (MAS) accelerated breeding cycles, while predictive algorithms achieved 92.4% accuracy in forecasting survival and growth traits. Controlled trials demonstrated a 30% reduction in winter mortality and a 25% improvement in growth rates among ML-selected oyster lineages compared to traditional methods. Additionally, a smartphone-based diagnostic tool was developed to enable real-time monitoring of oyster health, empowering farmers to adapt feeding and environmental strategies dynamically. This research bridges the gap between conventional aquaculture and computational innovation, offering a scalable framework to enhance genetic diversity, sustainability, and yield. By replacing trial-and-error practices with data-driven precision, our approach not only mitigates immediate industry challenges but also establishes a pathway for climate-resilient aquaculture. The fusion of ML with multi-omics technologies marks a transformative shift, enabling breeders to make rapid, evidence-based decisions that harmonize ecological stewardship with commercial demands.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.16.654565v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Combinatorial mutagenesis of N-terminal sequences reveals unexpected and expanded stability determinants of the Escherichia coli N-degron pathway</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sen, S., Corrado, N., Tiso, A., Kho, K. K., Kunjapur, A. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the deepest sequence coverage screen of the bacterial N-degron pathway using combinatorial site-saturation mutagenesis, fluorescence-activated cell sorting (FACS), and next-generation sequencing (NGS).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Although it is known that penultimate N-terminal residues can influence protein stability by the N-degron pathway, there has not been a comprehensive effort to document nor predict these effects for natural and synthetic sequences in prokaryotes. Here, we present the deepest sequence coverage screen of the bacterial N-degron pathway using combinatorial site-saturation mutagenesis, fluorescence-activated cell sorting (FACS), and next-generation sequencing (NGS). Our results reveal new categories of exceptions and nuance to the N-degron pathway observed in bulk and context-specific analyses. For example, we find cases where penultimate residues can reinforce or shift the stability expected based only on the identity of position 1 (P1). We observe lowered stability for P2-P5 motifs rich in bulky residues and Gln as well as heightened stability for P2-P5 motifs rich in negatively charged residues, Gly, and Pro, particularly when these residues are at P2. We find that P1 Cys can be a component of an N-degron in a sequence-specific manner. Furthermore, we employ machine learning to predict protein stability, identifying several motifs with unexpected fates. Our work expands the sequence determinants of the bacterial N-degron pathway with unprecedented granularity, providing a set of sequence guidelines for proteomic investigation and protein engineering.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.22.655665v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Peptide Barcodes for miRNA activity assessment in mammalian cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cheras, V., Rousounelou, E., Aschenbach, J.-L., Panke, S., Benenson, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that genetically encoded peptide reporters (peptide barcodes) can be used to analyze multiple post-transcriptional pathways in parallel.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Studies of gene regulation require measurements of mRNA and protein levels of a regulated gene. Recently, parallel reporter assays have been introduced to study the regulation of multiple genes at once. While transcriptional regulation can be probed by next generation sequencing, post-transcriptional regulation requires the ability to measure multiple proteins in the same experiment. Multiplexing with the help of fluorescent proteins limits the addressable diversity of simultaneous measurements due to spectral overlap. Inspired by the utility of proteotypic peptides in targeted proteomics, here we show that genetically encoded peptide reporters (peptide barcodes) can be used to analyze multiple post-transcriptional pathways in parallel. We use RNA interference as an exemplary regulatory mechanism that occurs on both transcriptional and post-transcriptional levels. We measure the activity of multiple microRNAs in parallel using a peptide barcode-based miRNA sensor library. Fluorescent reporters are used to validate the accuracy of the miRNA activities reported via the peptide barcodes. Several assay optimization steps are explored leading to the robust activity profiling of nine miRNAs across three different cell lines. Overall, this study underlines the multiplexing potential of peptide barcodes to rapidly and quantitatively measure the post-transcriptional regulation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.22.655559v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

