<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Friday, 03 October 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">scGPD: single-cell informed gene panel design for targeted spatial transcriptomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Guo, Y., Zhao, J., Chang, R. B., Zhao, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce scGPD, a deep learning-based framework for gene panel design that leverages single-cell RNA-seq data to identify compact, nonredundant sets of genes for spatial profiling.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In targeted spatial transcriptomics technologies, a key challenge is to select an informative gene panel that captures the complexity of cellular and spatial heterogeneity within tissues. Many existing methods use prior knowledge or heuristic selection rules, such as selecting highly variable genes, which overlook gene-gene correlations and may consequently result in suboptimal coverage. To address the limitations of the existing methods, we introduce scGPD, a deep learning-based framework for gene panel design that leverages single-cell RNA-seq data to identify compact, nonredundant sets of genes for spatial profiling. scGPD uses a gene-gene correlation-aware gating mechanism to extract informative features from data, encouraging diversity among selected genes and eliminating redundancy. Across diverse single-cell datasets, scGPD outperforms existing gene panel design methods in recovering transcriptome-wide expression using a limited number of genes. When applied to spatial transcriptomics data, it achieves superior cell type classification accuracy, demonstrating strong generalization across modalities. The gene panels selected by scGPD further exhibit well-defined spatial expression patterns, highlighting their robustness and relevance for spatial analysis. The scGPD framework is flexible and can be adapted to multiple use cases, enabling the prioritization of genes relevant to specific diseases or phenotypes. Together, these results demonstrate that scGPD provides a robust and adaptable solution to design efficient gene panels for spatial transcriptomics, with broad applicability to tissue mapping and disease characterization.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.02.680117v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">KNexPHENIX: A PHENIX-Based Workflow for Improving Cryo-EM and Crystallographic Structural Models</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nandi, S., Conn, G. L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a semi-automated, semi-automatic method for refining atomic models from cryo-EM and X-ray crystallography data .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The number of structures deposited in the Protein Data Bank (PDB) has increased exponentially in recent years due to the availability of new methods to visualize increasingly complex macromolecules and their complexes in atomic detail. However, accurately refining atomic models from experimental maps remains a challenge due to limitations of current approaches for efficient model refinement. Standard PHENIX refinement can partially address these limitations with its speed and broad accessibility but often fails to yield the best model compared to those generated by more computationally intensive approaches. We therefore developed KNexPHENIX, a customized semi-automated PHENIX-based workflow to support optimal macromolecular model building. KNexPHENIX can be used to refine macromolecular structures obtained from both cryo-electron microscopy (cryo-EM) and X-ray crystallography and is independent of the nature or size of the molecule. We evaluated KNexPHENIX on deposited structures as well as on de novo models chosen based on their visual fit to the deposited map or generated by molecular replacement. Compared to other approaches-PHENIX with default settings, REFMAC Servalcat, REFMAC, and CERES-KNexPHENIX consistently produced models with lower MolProbity scores, indicating improved model stereochemistry. Importantly, this was accomplished while maintaining model-to-map correlation for cryo-EM datasets and maintaining or reducing the Rfree-Rwork difference below accepted thresholds for X-ray crystallographic structures, limiting overfitting while preserving refinement accuracy. These results establish the KNexPHENIX workflow as a practical, accessible approach for refining both cryo-EM and crystallographic structures, enabling the generation of models with enhanced quality metrics suitable for deposition and to guide further experimental studies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.02.680106v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Machine learning-assisted selection of informative loci for strain-level phylogenetics of Neisseria gonorrhoeae</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kochubei, E., Dobrovolskii, S., Zenchenko, Z., Rayko, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose and evaluate a machine learning algorithm for the automated selection of a minimal set of informative genetic loci for accurate strain classification of Neisseria gonorrhoeae.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Epidemiological surveillance of Neisseria gonorrhoeae is hindered by the limitations of existing molecular typing methods, such as NG-MAST and MLST, which either suffer from excessive variability or insufficient resolution. In this study, we propose and evaluate a machine learning (ML) algorithm for the automated selection of a minimal set of informative genetic loci for accurate strain classification. Using a collection of 29 reference genomes of N. gonorrhoeae, we developed a pipeline that integrates Random Forest models and DNABERT embeddings to generate optimized gene panels. The results demonstrate that ML-selected panels substantially outperform traditional schemes, yielding markedly improved phylogenetic accuracy and branch support consistently above 90%. The proposed approach significantly reduces computational costs compared to whole-genome analysis and represents a promising resource-efficient tool for routine epidemiological monitoring, tracking transmission pathways, and identifying antibiotic-resistant strains.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.02.679713v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Split inteins for generating combinatorial non-ribosomal peptide libraries</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gonschorek, P., Wilson, C. S., Schelhas, C., Bozhueyuek, K. A. J., Gruen, P., Bode, H. B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that an NRPS can be split into three parts and reconstituted via trans-splicing using two orthogonal inteins at four different engineering sites, enabling the expression of single, covalently linked NRPS proteins from multiple plasmids.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Engineering Non-Ribosomal Peptide Synthetases (NRPS) is a promising strategy for discovering new bioactive compounds, which can serve as valuable leads for drug development, such as new antibiotics. However, their engineering is hampered by the limited availability of molecular tools for the efficient heterologous expression of their large biosynthetic gene clusters. In fact, a single NRPS gene can already exceed the size limits of standard cloning vectors. In this study, we establish split inteins as a novel tool for NRPS engineering to enable the expression of single, covalently linked NRPS proteins from multiple plasmids and to perform cloning-free module swapping. Using the xenotetrapeptide synthetase as model system, we show that an NRPS can be split into three parts and reconstituted via trans-splicing using two orthogonal inteins at four different engineering sites. Based on this tripartite platform we build a library comprising 21 plasmids and generated 324 hybrid NRPS by combinatorial transformation. More than half were catalytically active, producing over 200 novel peptides. This intein-based technology provides a modular platform for generating natural product-like peptide libraries, expanding biocatalytically accessible chemical space.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.02.680031v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Finishing a complete giraffe genome from telomere to telomere with Verkko-Fillet</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kim, J., Rosen, B. D., Fumagalli, S. E., Kuhn, K. L., Long, A., Schoenebeck, J. J., Wu-Cavener, L., Zimin, A. V., Cavener, D., Smith, T. P. L., Phillippy, A. M., Koren, S., Rhie, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed Verkko-Fillet, a Python-based interactive framework for genome graph inspection, editing, and curation that also provides step-by-step guidelines for downstream polishing.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> High-quality genome assemblies are essential for understanding speciation, evolution, and for building reference genomes and pan-genomes. Despite major advances in long-read sequencing technologies and graph-based assembly algorithms, current assemblies often remain incomplete and require refinement, including correction of haplotype switching and resolution of gaps, particularly within repetitive regions such as ribosomal DNA clusters and recent segmental duplications. These limitations largely stem from challenges in graph curation and are not adequately addressed by conventional polishing methods, which focus primarily on nucleotide-level corrections. To overcome these issues, we developed Verkko-Fillet, a Python-based interactive framework for genome graph inspection, editing, and curation that also provides step-by-step guidelines for downstream polishing. Verkko-Fillet takes as input Verkko output files, including the assembly graph, haplotype paths, Hi-C contacts, and additional ONT reads or alternative assemblies aligned to the graph, and offers tools for visualizing, modifying, and exporting curated assembly graphs. It enables users to track changes, resolve complex structural features, fill gaps, and enhance assembly quality beyond conventional polishing. As a case study, we applied Verkko-Fillet to the giraffe genome (Giraffa tippelskirchi), refining a draft assembly (QV 61.5) to a telomere-to-telomere (T2T) reference genome (QV 73.6) by improving both contiguity and completeness. Along with the T2T diploid giraffe genome, we provide gene annotations on this assembly as a valuable resource. Our results highlight the critical role of graph-based curation in producing high-quality, gapless, T2T assemblies that are ready for downstream biological analyses.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.01.679366v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Systematic Fairness Evaluation of Racial Bias in Alzheimers Disease Diagnosis Using Machine Learning Models</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Baddam, N. G., Pijani, B. A., Bozdag, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study investigates the fairness of ML models in AD diagnosis and shows that models trained on a single racial group perform well within that group but poorly in others.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> INTRODUCTION: Alzheimers disease (AD) is a major global health concern, expected to affect 12.7 million Americans by 2050. Machine learning (ML) algorithms have been developed for AD diagnosis and progression prediction, but the lack of racial diversity in clinical datasets raises concerns about their generalizability across demographic groups, particularly underrepresented populations. Studies show ML algorithms can inherit biases from data, leading to biased AD predictions. METHODS: This study investigates the fairness of ML models in AD diagnosis. We hypothesize that models trained on a single racial group perform well within that group but poorly in others. We employ feature selection and model training techniques to improve fairness. RESULTS: Our findings support our hypothesis that ML models trained on one group underperform on others. We also demonstrated that applying fairness techniques to ML models reduces their bias. DISCUSSION: This study highlights the need for racial diversity in datasets and fair models for AD prediction.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.30.678854v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A structure-guided pipeline to uncover the underexplored beta-lactamases from Antarctic and Subantarctic soil microbiota</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Coche-Miranda, J., Arros, P., Berrios-Pasten, C., Lagos, R., Orlando, J., Chavez, F. P., Marcoleta, A. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We found several classes and families of beta-lactamases, with a notable predominance of class-B metallo-beta-l lactamases in Antarctica and Subantarctic islands, including proposed novel families and variants of known families.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> As the One Health approach points out, the environment can significantly affect human health, particularly having diverse and profound implications in the ongoing antimicrobial resistance crisis. One central aspect is the potential role of environmental microorganisms as a source of resistance genes that could emerge among pathogens aided by horizontal transfer. In this context, previous reports showed that the Antarctic soil microbiota hosts a rich resistome, including putative beta-lactamases conferring resistance to beta-lactams, the most widely used antibiotics for treating bacterial infections worldwide. However, the diversity of beta-lactamases across different areas of Antarctica and Subantarctic islands and their associated microbiota remains largely unexplored. In this study, we analyzed an extensive collection of Antarctic soil metagenomes, applying a novel bioinformatic pipeline based on combining sequence identity and structural similarity criteria to search for bona fide distant beta-lactamase orthologs. We found several classes and families, with a notable predominance of class-B metallo-beta-lactamases, including proposed novel families and variants of known families. The beta-lactamase diversity and dominant classes varied across sites following changes in the microbial community, observing three main sample groups: 1) Subantarctic islands, 2) Antarctic Peninsula and surrounding islands; and 3) Cold-desert environments. The most prevalent beta-lactamases corresponded to subclass B3. We reconstructed more than 3500 metagenome-assembled genomes (MAGs) and searched for beta-lactamases among them. The main taxa encoding the beta-lactamases were Pseudomonadota and Bacteroidota. Moreover, a certain proportion of beta-lactamases was associated with mobile genetic elements, especially integrons and insertion sequences, suggesting their potential dissemination by horizontal transference. This evidence reinforces the role of the environment, and especially (sub)Antarctic soils, as a reservoir of resistance genes, in particular, beta-lactamases.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.30.679653v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CytokineFindeR: an R-package for benchmarking methods and databases for identifying cytokines</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tang, J. S., Singh, A., Singh, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We benchmark various cytokine identification methods and found little congruency between gene sets for the same ligand, and variable performance in identifying cytokines based on curated ligand-receptor interactions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cytokines play a central role in disease but are hard to study due to their short half-life and low abundance. Computational approaches have utilized gene-expression for pinpointing cytokine drivers of diseases. We benchmark various cytokine identification methods and found little congruency between gene sets for the same ligand, and variable performance in identifying cytokines based on curated ligand-receptor interactions. CytoSig, a model-based approach, was generally better at identifying the correct disease cytokine but failed for certain cytokines such as IL-13. We developed CytokinefindeR, an R package that enables comparative analysis of cytokine detection across 10 databases and four computational approaches.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.30.679635v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Qimai: a multi-agent framework for zero-shot DNA-protein interaction prediction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, W., Liu, c., Yao, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Qimai, a modular AI agent framework that integrates deep learning predictions with biological evidence using Large Language Model (LLM) as reasoning engine.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurate prediction of DNA-protein interactions, a fundamental task in genomics, is limited by the poor generalization of existing models to novel proteins not seen during training. To address this challenge, we introduce Qimai, a modular AI agent framework that integrates deep learning predictions with biological evidence using Large Language Model (LLM) as reasoning engine. Qimai combines direct motif evidence from the query protein, indirect motif evidence from its interactors, and quantitative prediction from a new transformer-based DPI model to produce explainable predictions with confidence scores. On a benchmark of 78 unseen proteins, Qimai consistently outperforms standalone deep learning models across all metrics, increasing the Area Under Curve of the Precision-Recall (AUC-PR), the Area Under Curve of the Receiver Operating Characteristic (AUC-ROC), and Matthews Correlation Coefficient (MCC) by 17.6%, 15.6%, and 244% respectively compared to the best standalone model. Ablation analyses reveal that this gain is driven by the LLM's ability to dynamically weigh diverse evidence, with indirect motif evidence of co-factors particularly critical for unseen proteins. Qimai establishes a generalizable and interpretable paradigm for integrating heterogeneous data in predictive genomics. This framework is accessible via the Qimai web portal (https://qimai.wanglab.ucsd.edu/).</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.30.679628v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Oncogenic PTPN11/SHP2 drives immune escape in juvenile myelomonocytic leukemia (JMML) through activation of ectonucleotidase/adenosine signaling</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rajak, J., Meng, K., Stippel, A. L., Wang, J., Koleci, N., Bohler, S., Schlaak, A. E., Xiao, H., Braun, L. M., Masetti, R., Zeiser, R., Flotho, C., Lipka, D. B., Niemeyer, C. M., Hannibal, L., Bengsch, B., Erlacher, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We study the mechanisms of oncogenic PTPN11-mutated JMML and propose a new immunomodulatory approach to enhance T cell-mediated control of JMML, particularly in the context of HSCT and relapse prevention.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Juvenile myelomonocytic leukemia (JMML) is a myelodysplastic/myeloproliferative neoplasm of early childhood driven by RAS pathway mutations. Allogeneic hematopoietic stem cell transplantation (HSCT) is the therapy of choice for most patients. However, relapse rate is high, in patients with adverse features, frequently noted in PTPN11-mutated JMML, or in patients without evidence of graft-versus-host disease (GvHD). Here we set out to understand the mechanisms associated with oncogenic PTPN11 immune escape. Analyzing primary PTPN11-mutated JMML samples and MxCre;Ptpn11D61Y/+ mice, we observed elevated expression of immune checkpoint molecules, including ectonucleotidases CD39 and CD73 - key mediators of the adenosine pathway - on monocytic and granulocytic leukemic cells. Stimulation with GM-CSF, a central mediator of JMML pathogenesis, induced ectonucleotidases expression on granulocytes and monocytes. In contrast, MEK inhibition downstream of Ptpn11D61Y/+ reduced ectonucleotidases expression. Functionally, Ptpn11D61Y/+-mutated myeloid cells suppressed activation and proliferation of wild-type (WT) T lymphocytes, an effect recapitulated by adenosine and reversed by pharmacological CD39 inhibition with POM-1. In vivo, POM-1 treatment of MxCre;Ptpn11D61Y/+ mice presenting with myeloproliferation reduced spleen size and partially restored immune responsiveness. Moreover, POM-1 induced apoptosis in murine Ptpn11D61Y/+ myeloid cells, highlighting a dual therapeutic benefit of CD39 inhibition in JMML. Together, these findings suggest that targeting the adenosine pathway may represent an immunomodulatory approach to enhance T cell-mediated control of JMML, particularly in the context of HSCT and relapse prevention.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.01.679664v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">TP53-mutant AML with ribosomal gene loss exhibits impaired protein translation and sensitivity to HSP90 inhibition</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Spinella, J.-F., Chagraoui, J., Boivin, I., Richard Carpentier, G., Moison, C., Mayotte, N., Beliveau, F., Lavallee, V.-P., Hebert, J., Sauvageau, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that a previously unappreciated subset of acute myeloid leukemia is marked by recurrent deletions on the short arm of chromosome 3 (del(3p), > 20% TP53-mutated cases) and a global downregulation of several ribosomal protein genes (RPG</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> TP53-mutated acute myeloid leukemia (AML) represents a particularly aggressive and therapeutically refractory subtype of the disease. While recurrent chromosomal abnormalities such as -5/del(5q), -7/del(7q), and del(17p) are well studied in this context, additional co-occurring events remain less well defined. Using the multi-dimensional Leucegene dataset (~700 primary AML specimens), we identified and comprehensively characterized a distinct subset of TP53-altered AML marked by recurrent deletions on the short arm of chromosome 3 (del(3p), > 20% TP53-mutated cases). These deletions frequently co-occur with del(5q) and encompass several ribosomal protein genes (RPGs), leading to a global downregulation of the ribosomal network and reduced protein synthesis. We show that this ribosomopathy-like phenotype is most pronounced in TP53-mutated cases with combined RPG deletions on chromosomes 3p and 5q, suggesting a cooperative oncogenic mechanism. Importantly, chemical screening identified HSP90 inhibition as a selective vulnerability in AML with low RPG expression. These findings highlight a previously unappreciated TP53-altered AML subset characterized by converging genomic and translational defects, and suggest that ribosomal stress may serve as a therapeutic entry point for targeted intervention of this patient subgroup.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.01.679539v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The immune and genetic heterogeneity of Hepatocellular Carcinoma, with a focus on multifocal disease</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Connor, A. L., Chang, L., West, E., Scott, K. J., Ismail, F., Cratchley, A., Newton, D. J., Samson, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study characterises the genetic and immune heterogeneity within a population of 56 tumours from 28 patients, 10 of whom have multifocal disease, obtained from patients undergoing surgery at the Leeds Teaching Hospital NHS Trust.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding the genetic and immune heterogeneity in hepatocellular carcinoma is crucial as treatments advance towards personalisation. This study characterises the genetic and immune heterogeneity within a population of 56 tumours from 28 patients, 10 of whom have multifocal disease, obtained from patients undergoing surgery at the Leeds Teaching Hospital NHS Trust. These samples represent the most common aetiological groups found in the UK, alcohol-related liver disease, non-alcoholic fatty liver disease and Hepatitis C. Whole exome sequencing was performed on DNA from these tumour and background liver samples to determine tumour-specific variants. Tissue sections were stained for both proliferation and immune cell markers. Tumours across aetioligical groups showed significant levels of variation, with high inter-individual variation. Analyses of multifocal tumours revealed significant discordance in genetic and immune cell profiles, both between multi-centric primary and metastatic tumours. This work emphasises the genetic and aimmuneheterogeneity in HCC across HCC subtypes, between and within individuals, highlighting mechansisms for therapeutic resistance and the need for personalised medicine.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.01.678727v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrated machine learning analysis of proteomic and transcriptomic data identifies healing associated targets in diabetic wound repair.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Brandon J Sumpio, Kyongmin Yeo, Georgios Theocharidis, Ikram Mezghani, Achille Fokoue, Alexander Gray, Jessica Gilman, Nikolaos Kalavros, Ioannis Vlachos, Samuel Sia, Tengfei Ma, Aristidis Veves</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified biomarkers that could be leveraged to classify the state of healing in diabetic wounds and identified molecular and protein deficits associated with impaired diabetic wound repair.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> This study identified biomarkers that could be leveraged to classify the state of healing in diabetic wounds. Firstly, by collecting wound samples from diabetic mice at different time points and generating their protein profiles using standard techniques, we set to interrogate whether a small number of biomarkers could serve as sensors to monitor the healing stage. Least absolute shrinkage and selection operator (LASSO) was applied. Large-scale analysis of wound tissue proteins integrated with the respective wound sizes allowed to establish a correlation between the observed protein profile and wound closure. We further evaluated human subjects' systemic serum proteomics for biomarkers. An additional wound healing model in diabetic mice was employed for microRNA quantitation at the same time points and similarly analyzed. Our analysis highlighted markers MMP-2, HGF, miR-1b and miR-107-3p in mice and Fractalkine and FGF-2 in humans that could correctly identify the extent of healing. By using proteomics from mice and human patients and complementary microRNA mouse data with computer regression models we can better predict molecular and protein deficits associated with impaired diabetic wound repair.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41038874/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

