<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Sunday, 04 May 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">nanoFOLD : sequence design of nanobodies via inverse folding</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chomicz, D., Janusz, B., Wrobel, S., Dudzic, P., Polasa, A., Martin, K., Darnell, S., Comeau, S. R., Krawczyk, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We develop an inverse folding method for nanobodies devoid of light chains and demonstrate its application in nanobody-engineering scenarios of enriching binders from next-generation sequencing experiments and novel binder design.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Antibodies devoid of light chains are a promising class of biotherapeutics. Computational methods that address these molecules are crucially needed to accelerate the traditional, long and expensive experimental process of their discovery. Inverse folding, wherein one is tasked to predict a sequence given molecular coordinates, is an established method in scaffold-based protein design. Here we develop an inverse folding method specific to nanobodies. We demonstrate its application in nanobody-engineering scenarios of enriching binders from next-generation sequencing experiments and novel binder design.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.29.651236v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Improving RNA Secondary Structure Prediction Through ExpandedTraining Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Langeberg, C. J., Kim, T., Nagle, R., Meredith, C., Garuadapuri, D. A., Doudna, J. A., Cate, J. H. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A large and diverse dataset for RNA secondary structure modeling and a new deep learning model for the task .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In recent years, deep learning has revolutionized protein structure prediction, achieving remarkable speed and accuracy. RNA structure prediction, however, has lagged behind. Although several methods have shown moderate success in predicting RNA secondary and tertiary structures, none have reached the accuracy observed with contemporary protein models. The lack of success of these RNA structure prediction models has been proposed to be due to limited high-quality structural information that can be used as training data. To probe this proposed limitation, we developed a large and diverse dataset comprising paired RNA sequences and their corresponding secondary structures. We assess the utility of this enhanced dataset by retraining two deep learning models, SincFold and MXfold2. We find that SincFold exhibited improved generalization to some previously unseen RNA families, enhancing its capability to predict accurate de novo RNA secondary structures. By contrast, retraining MXfold2 proved too computationally expensive for the large RNASSTR dataset and did not achieve high performance on the testing set. The RNASSTR dataset provides a substantial advance for RNA structure modeling, laying a strong foundation for the development of future RNA secondary structure prediction algorithms.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.03.652028v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Enriching stabilizing mutations through automated analysis of molecular dynamics simulations using BoostMut</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Korbeld, K. T., Fürst, M. J. L. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce BoostMut (Biophysical Overview of Optimal Stabilizing Mutations), a computational tool that standardizes and automates mutation filtering by analyzing dynamic structural features from molecular dynamics simulations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Thermostability is a critical goal in protein engineering for applications of biocatalysts and biomedicines. Despite striking advances in biomolecular predictive modeling, reliably identifying stabilizing mutations remains challenging. Previously, molecular dynamics (MD) simulations and visual inspection have been used as secondary filter to improve the success rate of mutations pre-selected by thermostability algorithms. However, this approach suffers from low throughput and subjectivity. Here, we introduce BoostMut (Biophysical Overview of Optimal Stabilizing Mutations), a computational tool that standardizes and automates mutation filtering by analyzing dynamic structural features from MD. BoostMut formalizes the principles guiding manual verification, providing a consistent and reproducible stability assessment. Rigorous benchmarking across multiple datasets showed that integrating BoostMut's biophysical analysis improves prediction rate regardless of the initial thermostability predictor. Given a modest amount of existing mutant stability data, BoostMut's performance can be further enhanced with a lightweight machine learning model. Upon experimentally validating BoostMut predictions on the enzyme limonene-epoxide hydrolase, we identified stabilizing mutations previously overlooked by visual inspection, and achieved a higher overall success rate. We foresee BoostMut being used for mutation filtering, as an integrated step in thermostability prediction workflows, and for labelling data to train future predictors.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.29.651183v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Strain-level metagenomic profiling using pangenome graphs with PanTax</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Luo, X., Zhang, W., Liu, Y., Li, G., Xu, J., Chen, E., Schonhuth, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present PanTax, a pangenome graph-based taxonomic profiler that overcomes the shortcomings of sequence-based approaches to strain-level taxonomic classification, and significantly outperforms state-of-the-art approaches.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Microbes are omnipresent, thriving in a range of habitats, from oceans to soils, and even within our gastrointestinal tracts. They play a vital role in maintaining ecological equilibrium and promoting the health of their hosts. Consequently, understanding the diversity in terms of strains in microbial communities is crucial, as variations between strains can lead to different phenotypic expressions or diverse biological functions. However, current methods for taxonomic classification from metagenomic sequencing data have several limitations, including their reliance solely on species resolution, support for either short or long reads, or their confinement to a given single species. Most notably, most existing strain-level taxonomic classifiers rely on the sequence representation of multiple linear reference genomes, which fails to capture the sequence correlations among these genomes, potentially introducing ambiguity and biases in metagenomic profiling. Here, we present PanTax, a pangenome graph-based taxonomic profiler that overcomes the shortcomings of sequence-based approaches, because pangenome graphs possess the capability to depict the full range of genetic variability present across multiple evolutionarily or environmentally related genomes. PanTax provides a comprehensive solution to taxonomic classification for strain resolution, compatibility with both short and long reads, and compatibility with single or multiple species. Extensive benchmarking results demonstrate that PanTax drastically outperforms state-of-the-art approaches, primarily evidenced by its significantly higher precision or recall at strain level, while maintaining comparable or better performance in other aspects across various datasets. PanTax is an open-source user-friendly tool that is publicly available at https://github.com/LuoGroup2023/PanTax.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.29.651271v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">multideconv - Integrative pipeline for cell type deconvolution from bulk RNAseq using first and second generation methods</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hurtado, M., Essabbar, A., Khajavi, L., Pancaldi, V.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present multideconv, an R package that reduces dimensionality and eliminates redundancy through unsupervised filtering and iterative correlation analyses, and harmonizes outputs across methods to identify robust cell subgroups and mitigate signature-driven heterogeneity.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Summary: Computational methods for cell type deconvolution from bulk RNA-seq data have been increasing in the last years, but their high feature complexity and variability across methods and signatures limit their utility and effectiveness for patient stratification. Applying multiple combinations of deconvolution methods and signatures often results in hundreds of redundant or contradictory cell features describing the tumor microenvironment (TME). Benchmarking efforts of deconvolution methods to look for the best one are inherently limited by the lack of bias-free ground truth, often yielding inconsistent results or no consensus. To address this, we present multideconv, an R package that reduces dimensionality and eliminates redundancy through unsupervised filtering and iterative correlation analyses. Built on top of existing frameworks like immunedeconv (Sturm et al. 2019) and omnideconv (Dietrich et al. 2024), multideconv harmonizes outputs across methods to identify robust cell subgroups and mitigate signature-driven heterogeneity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.29.651220v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Lazypipe3: Customizable Virome Analysis Pipeline Enabling Fast and Sensitive Virus Discovery from NGS data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Weinstein, I., Vapalahti, O., Kant, R., Smura, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that a simple two-round annotation strategy designed for speed and low false positive rate required in virus diagnostics, can reduce execution time to a fraction of BLAST search without any loss in accuracy.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Metagenomic next-generation sequencing (mNGS) has revolutionized virus detection across clinical, animal, and wildlife samples, enabling the discovery of both known and novel viruses. However, efficient identification of viromes from mNGS data depends heavily on bioinformatics pipelines. To contribute to publicly available pipelines, we introduce here Lazypipe3 with multiple new features including flexible annotation strategies that can be adjusted to match different datasets and research goals. Using a synthetic benchmark, we show that a simple two-round annotation strategy designed for speed and low false positive rate required in virus diagnostics, can reduce execution time to a fraction of BLAST search without any loss in accuracy. Additionally, using real data from mosquito samples we show that annotation strategies based on combinations of fast-to-accurate homology searches reduce execution time from 5 to 20-fold compared to BLASTN/BLASTP baseline. Using results from one of these "annotation chains" we were able to discover and characterise multiple novel whole-genome viral sequences that were missed by previous Lazypipe1 and BLASTX analysis. In Lazypipe3 we also implemented and benchmarked an improved background filtering and novel interactive reports for visualising coverage and placement of contigs relative to reference genomes. Lazypipe3 represents a highly efficient and versatile tool for virome analysis, offering customizable and transparent workflows that can facilitate virus discovery and diagnostics in diverse mNGS applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.29.651217v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AbBFN2: A flexible antibody foundation model based on Bayesian Flow Networks</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Guloglu, B., Braganca, M., Graves, A., Cameron, S., Atkinson, T., Copoiu, L., Laterre, A., Barrett, T. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present AbBFN2, a generative antibody foundation model trained on paired antibody sequences as well as genetic and biophysical metadata.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Antibody engineering is marked by diverse data and desiderata, making it a prime candidate for multi- objective design, but is commonly tackled as a series of sequential optimisation tasks. Here, we present AbBFN2, a generative antibody foundation model trained on paired antibody sequences as well as genetic and biophysical metadata. This is achieved using the Bayesian Flow Network paradigm, which allows unified modelling of diverse data sources and flexible conditional generation at inference time. By virtue of its rich set of features and architectural flexibility, AbBFN2 can be adapted to a number of tasks commonly tackled by individual models, consolidating traditional computational pipelines into a single step. We demonstrate the adaptability of AbBFN2 using sequence inpainting, humanisation, biophysical property optimisation, and conditional de novo library generation of antibodies with rare attributes as example tasks. By removing the need for task-specific training, we hope that AbBFN2 will accelerate machine learning-based antibody design and development workflows.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.29.651170v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Differential cell signaling testing for cell-cell communication inference from single-cell data by dominoSignal</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mitchell, J. T., Stapleton, O., Krishnan, K., Nagaraj, S., Lvovs, D., Cherry, C., Poissonnier, A., Horton, W., Adey, A., Rao, V., Huff, A., Zimmerman, J. W., Kagohara, L. T., Zaidi, N., Coussens, L. M., Jaffee, E. M., Elisseeff, J. H., Fertig, E. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed the dominoSignal software to innovate upon Domino and extend its functionality to test statistically differential cellular signaling.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Algorithms for ligand-receptor network inference have emerged as commonly used tools to estimate cell-cell communication from reference single-cell data. Many studies employ these algorithms to compare signaling between conditions and lack methods to statistically identify signals that are significantly different. We previously developed the cell communication inference algorithm Domino, which considers ligand and receptor gene expression in association with downstream transcription factor activity scoring. We developed the dominoSignal software to innovate upon Domino and extend its functionality to test statistically differential cellular signaling. This new functionality includes compilation of active signals as linkages from multiple subjects in a single-cell data set and testing condition-dependent signaling linkage. The software is applicable for analysis of single-cell data sets with multiple subjects as biological replicates as well as with bootstrapped replicates from data sets with few or pooled subjects. We use simulation studies to benchmark the number of subjects in compared groups and cells within an annotated cell type sufficient to accurately identify differential linkages. We demonstrate the application of the Differential Cell Signaling Test (DCST) in the dominoSignal software to investigate consequences of cancer cell phenotypes and immunotherapy on cell-cell communication in tumor microenvironments. These applications in cancer studies demonstrate the ability of differential cell signaling analysis to infer changes to cell communication networks from therapeutic or experimental perturbations, which is broadly applicable across biological systems.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.02.651747v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Spatialproteomics - an interoperable toolbox for analyzing highly multiplexed fluorescence image data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Meyer-Bender, M. F., Voehringer, H. S., Schniederjohann, C., Koziel, S. P., Chung, E. K., Popova, E., Brobeil, A., Held, L.-M., Munir, A., Scverse Community,, Dietrich, S., Bruch, P.-M., Huber, W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present spatialproteomics, a Python package that addresses the end-to-end image analysis challenges of highly multiplexed immunofluorescence imaging.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Highly multiplexed immunofluorescence imaging is a recent method to characterize tissues at single-cell resolution on the protein level, offering low cost, high scalability, and the ability to analyze paraffin-embedded tissue samples. However, the analysis of these data involves a sequence of steps, including segmentation, image processing, marker quantification, cell type classification, and neighborhood analysis, each of which involves a multitude of method and parameter choices that need to be adapted to the data and analytical objective at hand. Moreover, variations in data quality can be high and unpredictable, which necessitates further flexibility and interactivity. While individual components exist, there is an unmet need for a coherent toolbox that offers end-to-end coverage of the workflow, flexibility, and automation. We present spatialproteomics, a Python package that addresses these challenges. Built on top of xarray and dask, spatialproteomics can process images that are larger than the working memory. It supports synchronization of shared coordinates across data modalities such as images, segmentation masks, and expression matrices, which facilitates easy and safe subsetting and transformation. We demonstrate spatialproteomics on a set of images of reactive lymph nodes or different forms of B cell Non-Hodgkin lymphomas (BNHL) from 132 patients. We showcase an end-to-end analysis from raw images to statistical characterization of cell type composition and spatial distribution across indolent and aggressive lymphomas. Furthermore, we show how spatialproteomics can process gigapixel whole slide images. Altogether, we propose spatialproteomics as an easy-to-install, easy-to-learn, comprehensive toolbox for constructing powerful end-to-end image analysis solutions for highly multiplexed immunofluorescence imaging.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.29.651202v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Deciphering Biosynthetic Gene Clusters with a Context-aware Protein Language Model</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kang, Z., Zhang, H., Liang, C., Yang, R., Ye, Y., Bai, H., Zhang, Y., Ning, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present CoreFinder, a deep learning framework that integrates protein language models (pLMs) and genomic contexts to predict product class and decipher gene functions within BGCs without alignment.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Microbial secondary metabolites, synthesized by biosynthetic gene clusters (BGCs), play critical roles in ecological interactions and offer vast potential for biotechnological and pharmaceutical applications. Despite advances in computational BGC detection, current methods face challenges, including time-consuming sequence alignments, dependence on known homologs and manually defined rules, limiting its robustness and generalizability. To address these, we present CoreFinder, a deep learning framework that integrates protein language models (pLMs) and genomic contexts to predict product class and decipher gene functions within BGCs without alignment. CoreFinder demonstrated higher precision of 0.945 (842/891) and recall of 0.821 (842/1,025) than antiSMASH for core gene annotation in over 700 experimentally validated fungal BGCs. Built on CoreFinder, we introduced an end-to-end scalable workflow for BGC screening and deciphering, which is about 240 times faster than antiSMASH. Applied to 256 genomes spanning 197 taxa, CoreFinder identified 6,414 core genes within 4,585 BGCs. Further analysis indicates that a non-ribosomal peptide synthetase (NRPS) family likely existed prior to the divergence of Fusarium and Aspergillus and evolved into function-specific gene clusters. These findings emphasize the potential of CoreFinder as a powerful tool for accelerating natural product discovery and driving innovation in synthetic biology by unlocking novel biosynthetic pathways for biotechnological and pharmaceutical advancements.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.29.651206v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Cross-order detection of bacteriophage transduction in communities using ribosomal RNA barcoding</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> LaTurner, Z. W., Dysart, M. J., Schwartz, S. K., Zeng, E., Chappell, J., Silberg, J. J., Stadler, L. B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A synthetic ribozyme that stores information about DNA uptake in 16S ribosomal RNA (rRNA) was used to identify phage-host interactions by integrating the ribo enzyme into phage P1 and performing targeted 16S rRNA sequencing following transduction.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Bacteriophages (phages) facilitate gene transfer and microbial evolution in all ecosystems and have applications as tools for engineering microbiomes and as antimicrobials. Historic efforts to map phage hosts, such as plaque assays, are limited to culturable bacteria, are low throughput, and are hard to apply in environmentally-relevant contexts. To overcome these limitations, a synthetic ribozyme that stores information about DNA uptake in 16S ribosomal RNA (rRNA) was used to identify phage-host interactions by integrating the ribozyme into phage-plasmid P1 and performing targeted 16S rRNA sequencing following transduction. Experiments in synthetic and wastewater communities revealed Aeromonadales as a novel P1 host order and transduction of P1 into pathogens. Host range varied across phagemids with different origins of replication and phage particles with different tail fibers. This work shows how autonomous barcoding can be used in phages to identify the molecular controls on their host range in communities.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.03.652062v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

