<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Friday, 04 July 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">From Orphan Genes to Cryptogenic Gene Candidates: Reassessing Uniqueness</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Maan, S. K., Butzin, X. Y., Ge, S. X., Butzin, N. C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We re-analyzed a repository of putative orphan genes from over 80,000 bacterial species and show that their prevalence has been overestimated.</p><p style="font-size: 16px; text-align: justify;">Background Orphan genes lack recognizable homologues outside a given taxonomic unit; thus, they have elusive evolutionary origins. They have long been invoked to explain lineage specific traits in medicine and evolutionary biology. Here, we re-analyzed a 2023 repository of putative orphan genes compiled from over 80,000 bacterial species. Using iterative homology-based analyses, we reassessed each gene's taxonomic distribution across a broader genomic landscape. Results These results challenge the long-held assertion that species truly harbor large numbers of orphan genes and instead demonstrate that their prevalence has been overestimated. Many orphan genes from 2023 now match homologues in other bacterial taxa (2025), demonstrating that sparse sampling had inflated earlier orphan counts. To better reflect these findings, we propose the term Cryptogenic Gene Candidates (CGCs). This offers a more precise alternative to orphan genes, as it more accurately reflects the provisional and potentially temporary nature of these genes' apparent uniqueness. This revised terminology acknowledges that many genes identified as having a unique origin may lose this status as more genomes are sequenced. Although our analyses substantially reduced the number of false-positive CGCs, it cannot determine whether a given CGC truly encodes a functional protein or is an artifact of bioinformatic analysis. To minimize annotation artifacts, we then applied additional computational filters to prioritize candidates most likely to encode bona fide proteins. Conclusions This work revises the understanding of orphan gene prevalence, establishes CGCs as a more accurate classification, and provides a prioritized set of candidates to guide future experimental studies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.28.662130v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Spliformer-v2 predicts multi-tissue RNA splicing and reveals functional genomic links with neurodegenerative diseases</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tang, X., Lei, H., Guo, J., Shen, Y., Zhang, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Spliformer-v2, a deep learning model based on SegmentNT architecture to predict multi-tissue RNA splicing across the most comprehensive set of human brain and spinal cord tissues to date.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Precise regulation of pre-mRNA splicing underpins molecular diversity and is linked to aging and disease. Genetic variants are key drivers of RNA mis-splicing, yet how they induce tissue-specific splicing remains largely unclear. Here, we introduce Spliformer-v2, a deep learning model based on SegmentNT architecture to predict multi-tissue RNA splicing. Spliformer-v2 is trained on paired genome/transcriptome data from 18 human tissues, including 12 central nervous system and 6 peripheral tissues. Spliformer-v2 accurately predicts the effects of heterozygous and homozygous variants on splicing, outperforming existing models (such as SpliceTransformer) across tissues for both regression evaluations (R2: 0.83-0.89 vs 0.10-0.41) and classification evaluations (AUPRC: 0.90-0.95 vs 0.45-0.61). Analysis of ClinVar dataset indicated the link of tissues-specific splicing variants with disease pathogenicity. Analysis of genome-wide association study hits from neurological diseases (such as Amyotrophic Lateral Sclerosis, Alzheimer's disease and Parkinson's disease) identified tissue-specific splicing variants in genes like CREB3, SCFD1, MAPT and TOMM40, suggesting potential splicing mechanisms in disease. In summary, Spliformer-v2 is a powerful tool for predicting RNA splicing across the most comprehensive set of human brain and spinal cord tissues to date. This advancement enhances our understanding of tissue-specific splicing variants and their roles in neurological and other complex diseases, thereby advancing the discovery of novel drug targets and biomarkers in precision medicine.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.28.662167v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multimodal deep learning integration of cryo-EM and AlphaFold3 for high-accuracy protein structure determination</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gyawali, R., Dhakal, A., Cheng, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce MICA, a fully automatic and multimodal deep learning approach combining cryo-EM density maps with AlphaFold3-predicted structures at both input and output levels to improve cryo/EM protein structure modeling.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cryo-electron microscopy (cryo-EM) is a key technology for determining the structures of proteins, particularly large protein complexes. However, automatically building high-accuracy protein structures from cryo-EM density maps remains a crucial challenge. In this work, we introduce MICA, a fully automatic and multimodal deep learning approach combining cryo-EM density maps with AlphaFold3-predicted structures at both input and output levels to improve cryo-EM protein structure modeling. It first uses a multi-task encoder-decoder architecture with a feature pyramid network to predict backbone atoms, C atoms and amino acid types from both cryo-EM maps and AlphaFold3-predicted structures, which are used to build an initial backbone model. This model is further refined using AlphaFold3-predicted structures and density maps to build final atomic structures. MICA significantly outperforms other state-of-the-art deep learning methods in terms of both modeling accuracy and completeness and is robust to protein size and map resolution. Additionally, it builds high-accuracy structural models with an average template-based modeling score (TM-score) of 0.93 from recently released high-resolution cryo-EM density maps, showing it can be used for real-world, automated, accurate protein structure determination.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.03.663071v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PROFET Predicts Continuous Gene Expression Dynamics from scRNA-seq Data to Elucidate Heterogeneity of Cancer Treatment Responses</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cheng, Y.-C., Gu, H., McDonald, T. O., Wu, W., Tripathi, S., Guarducci, C., Russo, D., Abravanel, D. L., Bailey, M., Wang, Y., Zhang, Y., Pantazis, Y., Levine, H., Jeselsohn, R., Katsoulakis, M. A., Michor, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed PROFET (Particle-based Reconstruction Of Generative Force-matched Expression Trajectories), a computational framework that reconstructs continuous, nonlinear single-cell gene expression trajectories from sparsely sampled scRNA-seq data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell RNA sequencing captures static snapshots of gene expression but lacks the ability to track continuous gene expression dynamics over time. To overcome this limitation, we developed PROFET (Particle-based Reconstruction Of generative Force-matched Expression Trajectories), a computational framework that reconstructs continuous, nonlinear single-cell gene expression trajectories from sparsely sampled scRNA-seq data. PROFET first generates particle flows between time-stamped samples using a novel Lipschitz-regularized gradient flow approach and then learns a global vector field for trajectory reconstruction using neural force-matching. The framework was developed using synthetic data simulating cell state transitions and subsequently validated on both mouse and human in vitro datasets. We then deployed PROFET to investigate heterogeneity in treatment responses to palbociclib, a CDK4/6 inhibitor, in hormone receptor positive breast cancer. By comparing newly generated scRNA-seq data from a palbociclib-resistant breast cancer cell line with published patient-derived datasets, we identified a subpopulation of patient cells exhibiting profound phenotypic shifts in response to treatment, along with surface markers uniquely enriched in those cells. By recovering temporal information from static snapshots, PROFET enables inference of continuous single-cell expression trajectories, providing a powerful tool for dissecting the heterogeneity of cell state transitions in treatment responses.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.27.662030v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Allosteric Site Prediction Using Protein Language Models and Orthosteric Conditioning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Eccleston, R. C., Furnham, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We evaluate three strategies to predict allosteric binding sites using pre-trained protein language models, Ankh, ProtT5 and ESM-2, which are trained on sequence information alone.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Allosteric modulators as therapeutics offer many advantages over orthosteric modulators, including improved selectivity and tunability. However, identifying and characterising allosteric sites remains a major challenge both experimentally and computationally. Accurate prediction of allosteric binding sites is critical to facilitate allosteric drug discovery. Here, we evaluate three strategies to predict allosteric sites using pre-trained protein language models (pLMs), Ankh, ProtT5 and ESM-2, which are trained on sequence information alone. First, a classifier was trained using static embeddings extracted from each pLM. Second, parameter-efficient fine-tuning was implemented using LoRA with focal loss to account for the sparse positive labels and lack of data. Finally, a structure-aware conditioning mechanism was introduced, whereby orthosteric binding sites are encoded and integrated directly into the input embeddings to enable the model to predict allosteric binding sites, conditioned on the knowledge of the orthosteric binding pocket. This approach, which captures functional dependencies between the othosteric and allosteric binding sites, improves upon the performance of the first two methods and achieves results comparable to the leading structure-based allosteric site predictors.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.27.662060v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">tugMedi: simulator of cancer-cell evolution for personalized medicine based on the genomic data of patients</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nagornov, I., Furukawa, E., Nagai, M., Yagishita, S., Shibata, T., Kato, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present tugMedi, a cancer-cell evolution simulator designed for cancer genome medicine, enabling real-time predictions of clonal dynamics under virtual drug regimens.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cancer comprehensive genomic profiling tests are increasingly used, but drug response rates remain limited. Simulations forecasting cancer progression could aid targeted therapies; however, existing simulations focus mainly on basic biology. We present tugMedi, a cancer-cell evolution simulator designed for cancer genome medicine. By integrating patient-specific genomic and imaging data, tugMedi reconstructs each tumor's genomic features and growth behavior, enabling real-time predictions of clonal dynamics under virtual drug regimens. tugMedi explicitly models copy number alterations and SNVs on parental chromosomes in recessive and dominant modes, capturing loss-of-heterozygosity and yielding precise variant allele frequencies and tumor contents. It handles mutations in cancer-related genes with real exon-intron structures by a fast algorithm. For TCGA samples, it provided ensemble predictions of clonal dynamics, allowing extraction of drug response, tumor-size shrinkage rates, and time to recurrence under specific drug conditions. tugMedi represents a first step toward simulation-driven genome medicine based on a patient-derived virtual tumor.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.27.661855v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Human protein interactome structure prediction at scale with Boltz-2</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ille, A. M., Markosian, C., Burley, S. K., Pasqualini, R., Arap, W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present de novo computed models of 1,394 binary human protein interaction structures predicted using Boltz-2.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In humans, protein-protein interactions mediate numerous biological processes and are central to both normal physiology and disease. Extensive research efforts have aimed to elucidate the human protein interactome, and comprehensive databases now catalog these interactions at scale. However, structural coverage of the human protein interactome is limited and remains challenging to resolve through experimental methodology alone. Recent advances in artificial intelligence/machine learning (AI/ML)-based approaches for protein interaction structure prediction present opportunities for large-scale structural characterization of the human interactome. One such model, Boltz-2, which is capable of predicting the structures of protein complexes, may serve this objective. Here, we present de novo computed models of 1,394 binary human protein interaction structures predicted using Boltz-2. These structural predictions were based on biochemically determined interaction data sourced from the IntAct database, together with corresponding protein sequences obtained from UniProt. We then assessed the predicted interaction structures through various confidence metrics, which consider both overall structure and the interaction interface. These analyses indicated that prediction confidence tended to be greater for smaller complexes, while increased multiple sequence alignment (MSA) depth tended to improve prediction confidence. Additionally, we performed a limited evaluation against experimentally determined human protein complex structures not included in the Boltz-2 training regimen, which indicated prediction accuracy consistent with AlphaFold3. This work demonstrates the utility of Boltz-2 for in silico structural modeling of the human protein interactome, while highlighting both strengths and limitations. Ultimately, such modeling is expected to yield broad structural insights with relevance across multiple domains of biomedical research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.03.663068v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An Enhancement of Extrachromosomal Circular DNA Enrichment and Amplification to Address the Extreme Low Overlap Between Replicates</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Burnham, C. M., Kurilung, A., Wanchai, V., Regenberg, B., Delgado-Calle, J., Basnakian, A. G., Nookaew, I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study revisits the Circle-seq approach for enrichment of eccDNA to address if these limitations, hypothesizing that experimental procedures significantly contribute to the observed low eccDNA overlap.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Extrachromosomal circular DNA (eccDNA) of chromosomal origin is commonly present in all eukaryotic organisms and tissue tested so far. EccDNA populations exhibit immense diversity and a characteristically low degree of overlap between samples, suggesting low inherence of eccDNA between cells or a deficiency the methods by which eccDNA is detected. This study revisits the Circle-seq approach for enrichment of eccDNA to address if these limitations, hypothesizing that experimental procedures significantly contribute to the observed low eccDNA overlap. We optimized the protocol by reducing the time. Linear DNA is digested by increasing exonuclease V activity. We employed CRISPR-Cas9 for mitochondrial linearization, which proved superior to restriction enzymes. A key finding is the critical role of random hexamer primer concentration and genomic DNA input in Rolling Circle Amplification (RCA) for generating high-quality long amplicons from eccDNA (concatemeric tandem copy, CTC), essential for confident de novo eccDNA construction from long-read sequencing data. Lower primer concentrations substantially increased the percentage of CTC-derived eccDNA and improved the overlap of identified eccDNAs in technical replicates. Applying this revisited approach to human myeloma and breast cancer cell lines, as well as xenograft models, demonstrated that the optimized conditions enhanced the overlap of detected eccDNA up to over 50% overlap which substantially improved over previous studies (less than 1%). These findings provide guidelines for developing standardized procedures for eccDNA profiling, advancing our understanding of eccDNA biology and its potential clinical applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.28.662146v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">In Silico Investigation Reveals a Potential Functional Role for Human Microbiome in Chronic Obstructive Pulmonary Disease</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jana, N., Dhara, O., Bhattacharya, S. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study explores the structural and functional relations between microbial proteins and human proteins involved in COPD using in-silico bioinformatic tools.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Chronic Obstructive Pulmonary Disease (COPD) is a progressive enervating lung disease characterized by chronic inflammation, airway inhibition and unrecoverable structural damage to the lungs. While traditionally associated with environmental factors similar as cigarette smoke and air pollution as well as genetic factors, recent revelations has increasingly indicative of the role of microbiomes in the modulation of this disease. This study explores the structural and functional relations between microbial proteins and human proteins involved in COPD using in-silico bioinformatic tools. Several critical human proteins involved in COPD pathogenesis such as MMP 1, MMP 2, MMP 14, TNF-, TGF-{beta} were evaluated for the presence of microbial homologous proteins using BLAST analysis in the microbial database. The results revealed significant structural homology with microbial proteins from species including E. coli, K.pneumonia, Bacillus thuringiensis, Bifidobacterium samirii. This microbial protein may impact COPD progression through mechanisms similar as molecular belittlement and modulation of host vulnerable responses. Further molecular docking simulations were conducted using herbal drugs (oleanolic acid, gingerol, nobiletin, menthol among others) known for their anti-inflammatory effect and can be potentially used in the treatment of COPD. Results demonstrated favourable interactions between these composites and both human and microbial proteins, indicating therapeutic potential of the compounds. For case oleanolic acid showed strong interaction with TNF-- and its microbial homolog while menthol effectively interacted with MMP-14 and its bacterial counterparts. Also phylogenetic tree construction using Clustal Omega handed perceptivity into the evolutionary connections between the bacteria hosting homologous proteins, suggesting possible ancestral links and participated functional pathway applicable to COPD pathology . The finding emphasize the possibility of microbiome deduced proteins to act as modifiers in COPD progression as well as the therapeutic potential of the herbal compounds in targeting both host and microbial factors, which need to be further ascertained by in vitro , in vivo, as well as clinical studies. This novel in silico approach offers a unique prospective on COPD treatment by integrating microbiome and phytotherapy laying the groundwork for future personalized therapeutic strategies targeting both microbial and host inflammatory pathways.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.29.660899v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Divergence between transcriptomes and chromatin accessibility during differentiation from a bipotential progenitor cell population to erythroblasts and megakaryocytes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mishra, T., Giardine, B. M., Morrissey, C. S., Keller, C. A., Heuston, E. F., Anderson, S. M., Paralkar, V. R., Pimkin, M., Weiss, M. J., Bodine, D. M., Hardison, R. C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We used deep sequencing of polyA+ RNA to map the transcriptomes of the megakaryocyte-erythroid progenitor (MEP) and cells of its two daughter lineages, erythroblasts (ERY) and Megakaryocytes (MEG) in mice</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Changes in gene expression drive differentiation along distinct cell lineages, and these shifts in gene expression are associated with alterations in chromatin accessibility and modifications reflecting activation or repression. We used deep sequencing of polyA+ RNA to map the transcriptomes of the megakaryocyte-erythroid progenitor (MEP) and cells of its two daughter lineages, erythroblasts (ERY) and megakaryocytes (MEG) in mice to reveal insights into differentiation. Transcriptome comparisons revealed that MEPs already expressed much of the MEG program while continuing to express genes associated with parallel myeloid lineages. By contrast, ERY underwent an extensive program of gene induction along with repression of pan-hematopoietic and MEG genes. Maps of transcription factor (TF) occupancy also indicated distinct modes of regulation for the MEG and ERY programs, with MEG genes preferentially occupied by hematopoietic TFs in multipotent progenitors and continued occupancy post-commitment, in contrast to erythroid genes that were primarily occupied in committed ERY. Previous work had indicated a surprising discordance in the clustering of MEP with other hematopoietic cell types by RNA-seq versus chromatin states. We combined the differential expression data with chromatin accessibility across blood cell types to identify trends that contribute to this discordance. Specifically, candidate cis-regulatory elements (cCREs) in some ERY-specific genes were precociously actuated in the bipotential cell populations, and some other genes were expressed in both the MEP population and MEG but their cCREs have less chromatin accessibility in MEP. This discordance in cell type clustering by different modalities of functional genomics may reflect the different contributions of subpopulations in the MEP to the different modalities measured.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.30.662383v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An ancient X chromosomal region harbours three genes potentially controlling sex determination in Cannabis sativa</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Toscani, M., Riera-Begue, A., Malik, A., Dowling, C., Schilling, S., Melzer, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose that the combinatorial interaction of three genes can explain not only the molecular control of monoecy vs. dioecy but also of male vs. female individuals.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The development of different sexes has fascinated humankind for millennia. In plants unisexual flowers evolved multiple times independently, yet the molecular mechanisms governing sex determination remain largely elusive. Here, we analysed sex determination in Cannabis sativa, one of the world's oldest and most versatile crops. C. sativa is primarily dioecious, with separate male and female individuals, but some cultivars are monoecious, bearing both male and female flowers on the same plant. Using QTL mapping we identified Monoecy1, a locus on the X chromosome controlling monoecy vs dioecy. Strikingly, transcriptomic and genomic analyses revealed that Monoecy1 contains three genes within just 60,000 bp which have distinct male-, female- and monoecy-specific expression patterns and encode for putative developmental regulators. We propose that the combinatorial interaction of those genes can explain not only the molecular control of monoecy vs. dioecy but also of male vs. female individuals.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.03.663031v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

