<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Wednesday, 03 September 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Towards the genome-scale discovery of bivariate monotonic classifiers.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Océane Fourquet, Martin S Krejca, Carola Doerr, Benno Schwikowski</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Bivariate monotonic classifiers (BMCs) are based on pairs of input features. Like many other models used for machine learning, they can capture nonlinear patterns in high-dimensional data. At the same time, they are simple and easy to interpret. Until now, the use of BMCs on a genome scale was hampered by the high computational complexity of the search for pairs of features with a high leave-one-out performance estimate.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40898061/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Leveraging genome and transcriptome sequencing to decipher fungicide resistance mechanisms in crop pathogenic fungi: current status and prospects.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Alexander Dumbai Joe, Runze Liu, Xiao Luo, Xianhang Meng, Xiaojie Fang, Ziting Ding, Meng Liu, Zhitian Zheng</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Fungicide resistance in crop-pathogenic fungi presents a major threat to global agriculture, jeopardizing food security and sustainable crop production. This review thoroughly explores the use of genomic and transcriptomic sequencing technologies to uncover the molecular and genetic mechanisms underlying fungicide resistance in various phytopathogens. Resistance develops through both target-site mutations, such as changes in CYP51 and β-tubulin genes, and nontarget-site mechanisms, including metabolic detoxification, efflux pump overexpression and stress response modulation. Advances in whole-genome sequencing (WGS), genome-wide association studies (GWAS) and RNA-seq have led to the discovery of novel resistance alleles, differentially expressed genes (DEGs) and critical regulatory networks. Combining multi-Omics approaches improves our understanding of resistance mechanisms and aids the development of targeted diagnostics, breeding of resistant crop varieties and precise application of fungicides. Furthermore, machine learning and bioinformatics tools offer predictive insights into the development of resistance and potential countermeasures. This review highlights the importance of using genomic and transcriptomic data to design sustainable and integrated strategies for managing crop diseases and combating fungicide resistance in agriculture. © 2025 Society of Chemical Industry.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40891462/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CLAW-MRM: </h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Connor Beveridge, Sanjay Iyer, Caitlin E Randolph, Matthew Muhoberac, Palak Manchanda, Katherine A Walker, Shane Tichy, Gaurav Chopra</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Lipidomic profiling generates vast datasets, making manual annotation and trend interpretation complex and time-intensive. The structural and chemical diversity of the lipidome further complicates the analysis. While existing tools support targeted lipid identification, they often lack automated workflows and seamless integration with statistical and bioinformatics tools. Here, we introduce the comprehensive lipidomics automated workflow for multiple reaction monitoring (CLAW-MRM), a platform designed to automate lipid annotation, statistical analysis, and data parsing using custom multiple reaction monitoring (MRM) precursor product ion transitions. CLAW-MRM employs trimmed mean of m-value (TMM) normalization to account for lipid load differences, enabling robust cross-sample comparisons. To evaluate CLAW-MRM's performance, we analyzed lipid profiles in liver tissues of Alzheimer's disease (AD) mice and age-matched wild-type controls under conditions of constant and variable tissue mass, assessing the impact of normalization strategies on TMM-normalized lipidomic outcomes. Additionally, we isolated and profiled lipid droplets from individual brain regions of 18- to 24-month-old AD male mice and controls, leveraging nearly 1,500 MRM transitions across 11 lipid classes. Enhancing biological relevance, CLAW-MRM integrates LIGER (lipidome gene enrichment reactions), linking lipid expression with gene activation and suppression patterns. Through CLAW-MRM-based LIGER, we identified metabolic pathways enriched in differentially expressed lipids, offering insights into altered lipid metabolism in AD. To improve usability, CLAW-MRM incorporates a natural language interface powered by large language models, enabling artificial intelligence (AI)-driven user interaction for statistical and bioinformatics analyses. By automating lipid structural identification and integrating AI-assisted bioinformatics, CLAW-MRM provides an end-to-end workflow from data acquisition to interpretation, streamlining high-throughput lipidomics.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40892027/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SynGlue: AI-Driven Designer for Clinically Actionable Multi-Target Therapeutics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Solanki, S., Mohanty, S. K., Satija, S., Chauhan, S., Bandaru, N. V. M. R., Dukare, S., Tiwari, N. K., R, N. K., B, A. A., Mukherjee, S., Gautam, V., Arora, S., Kumar, S., Duari, S., Sharma, A., Shome, R., Sengupta, D., Abbineni, C., Samajdar, S., Ahuja, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The rational design of multi-targeting compounds and targeted protein degraders, such as PROTACs, remains a significant challenge in drug discovery. Here, we introduce SynGlue, a modular generative AI platform that integrates fragment-based reasoning with data-driven and structure-guided approaches to design multi-target therapeutics. Its flexible four-module architecture enables the generation of chemically diverse, target-specific, and polypharmacologically optimized compounds. The platform supports user-defined scoring functions and external docking tools, offering an end-to-end pipeline from target structure to synthesizable candidate. Using this platform, we designed and synthesized novel BRD4-VHL PROTACs featuring previously unreported and uniquely structured warheads. In vitro profiling confirmed potent nanomolar growth inhibition across multiple cell lines, driven by selective BRD4 degradation and subsequent c-Myc downregulation. In vivo, the lead compound established favorable pharmacokinetics, pharmacodynamics, and tolerability profiles. Importantly, this lead PROTAC demonstrated significant tumor burden reduction in mouse xenograft models via its targeted degradation mechanism. In summary, this work validates SynGlue as a powerful generative AI platform for the rapid and effective design of multi-targeting therapeutics with promising preclinical anti-cancer activity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.28.672835v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The genome of the Glacier lanternfish shows loss of MHC I and II function and provides insight into evolution of lanternfish immune systems</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Solbakken, M. H., Torresen, O. K., Tooming-Klunderud, A., Skage, M., Bakken, T., Jakobsen, K. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Lanternfishes (Myctophidae) are one of the most abundant and species diverse orders inhabiting the mesopelagic zone. Exploitation of marine resources has recently attracted increased interest. It is therefore essential to improve our understanding of the meso- and deep pelagic ecosystems with respect to conservation and management strategies. Genomic resources are paramount to enable in-depth studies of population structuring and understanding the inherent genetic diversity, but also to enable studies of processes such as local adaptation and selection. Here we present a genome assembly of the Glacier lanternfish (Benthosema glaciale) generated with long-read PacBio and Hi-C contact map data. By comparing additional available genomes from the Myctophiformes order we explore their adaptive immune strategies. Our findings reveal that multiple lineages within this order have lost a significant proportion of genes related to adaptive immunity (Electrona, Protomyctophum and Benthosema). We find simultaneous loss of both classical MHC class I and class II function coinciding with reduction in genetic diversity with respect to immunoglobulin and T-cell receptors. In contrast, the sister branch represented by Gymnoscopelus and Nannobrachium appear to have maintained the standard configuration of the jawed vertebrate adaptive immune system apart from large gene expansions in MHC class II. Our results demonstrate that the Benthosema-belonging lineage (Myctophinae) has completely lost the core functions of the adaptive immune system. How, when and why this occurred warrants further investigations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.28.672920v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An epigenomic investigation of atrial fibrillation in a matched left and right atrial human cohort</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rodriguez-Gonzalez, A., Frost, S., Thomas, A. M., Fernandez-Aroca, D. M., Choudhury, J., Tikkas, G., Tinker, A. M., Munroe, P. B., Bell, C. G., Villar, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> As the most prevalent cardiac arrythmia, atrial fibrillation is an important contributor to cardiovascular morbidity and mortality. Human population findings increasingly support its complex genetic architecture, with most genetic association signals for atrial fibrillation found in the non-coding genome. In this study, we integrated genome-wide histone modification, gene expression and DNA methylation levels in a paired left and right atrial cohort comprising permanent atrial fibrillation patients and sinus rhythm controls. First, we first identified epigenomic regions enriched in histone H3 lysine 27 acetylation (H3K27ac) across left and right atria from patients and controls, and associated them with differentially expressed genes to derive a set of dysregulated candidate loci, including NPPB and SCX. Second, by incorporating an independent replication cohort, we were able to validate gene expression and epigenomic differences for a subset of these candidate loci. Third, we profiled base-resolution DNA methylation levels and identified differentially-methylated regions (DMRs) between atrial fibrillation and sinus rhythm samples. Integration of these data with histone modification levels and gene expression allowed us to propose epigenetic mechanisms underlying transcriptomic and epigenomic changes across dysregulated loci, such as disruption of transcription factor binding by DNA methylation at the LRRC4B locus. The data and analyses we report constitute a systematic investigation of gene regulatory alterations across the left and right atria in permanent atrial fibrillation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.29.673028v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Mitochondrial genome reduction and accelerated evolution in planktonic foraminiferans</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lai, H.-T., Lai, M.-W., Wang, T.-H., Ren, H., Ku, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The evolution of mitochondria provides crucial insights into the diversification of eukaryotes, with complex events of gene losses revealed through comparative analyses of mitochondrial genomes (mitogenomes) across eukaryotic lineages. However, the mitogenomes of many microbial eukaryotes remain underexplored due to challenges in their isolation and cultivation. Particularly understudied are Foraminifera (Rhizaria, SAR), unicellular calcifyers that are widely distributed across global oceans and important paleoenvironmental proxies. Through single-cell genomic sequencing, here we report a 22-kb complete mitogenome of a planktonic foraminiferan from tropical seawater, which is the smallest known to date among all sequenced mitogenomes of Rhizaria, a major lineage of eukaryotes. It contains only six protein-coding genes and fragmented ribosomal RNA genes, and has lost most genes in oxidative phosphorylation and all genes in mitochondrial translation. Such genome reduction is associated with accelerated evolutionary rates and a lower GC content than that of benthic foraminiferan and other rhizarian mitogenomes. These findings highlight the unique trajectory of mitogenome reduction during rhizarian evolution and the use of single-cell approaches for recovering microbial eukaryotic genomes and expanding our understanding of mitochondrial evolution.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.28.672841v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Tech Note- Increased Complexity of Amplicon Libraries using Phased Primers</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bonath, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Phased primers are used during the first PCR of amplicon library preparation in order to increase the complexity of the library. This increased complexity leads to improved sequencing quality even when sequencing with a 2x300 paired-end setup. We can show that libraries prepared using a phased primer approach are complex enough to reduce the amount of PhiX added to the sequencing reaction to that of other, complex libraries.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.26.672315v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Population genomics highlight the vulnerability of coral-dwelling gobies to ecological losses due to climatic disturbances</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Froehlich, C. Y. M., Kirby, R., Greaves, S., Heatwole, S., Klanten, O. S., Hing, M., Wong, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Predicting species resilience to climate change generally focuses on species life history and ecology, overlooking genomic changes following disturbances. After consecutive cyclones and heatwaves, coral-dwelling Gobiodon gobies have experienced extreme population and group size reductions, and slower recovery rates than their coral hosts. To assess genomic contributions to this vulnerability, we analyzed multiple Gobiodon species for population structure (1) among three locations in western Oceania and (2) pre- and post-disturbances at the central location that underwent the same cyclones and heatwaves that reduced populations and group sizes. Species compared among locations all exhibited population structure. Post-disturbances, two species (G. fuscoruber, G. rivulatus) were extirpated, two species (G. histrio, G. quinquestrigatus) exhibited genetic bottlenecks, one species had no change to genetic diversity (G. brochus), and another species had consistently low diversity (G. erythrospilus). Our results highlight that coral-dwelling gobies face compounding ecological and genomic losses from climatic disturbances, warranting targeted conservation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.28.672959v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Artificial Intelligence Powered Biomarker Discovery: A Large-Scale Analysis of 236 Studies Across 19 Therapeutic Areas and 147 Diseases</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Abbasi, A. F., Naveed, S., Asim, M. N., Sajjad, M., Dengel, A., Vollmer, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Biomarkers are the molecular signatures that drive and reflect disease states and are indispensable for disease diagnosis, therapeutic target identification, and drug development. The landscape of biomarker discovery has undergone a transformative shift with the emergence of AI-powered predictive pipelines that can integrate complex, high-dimensional datasets. However, the field still lacks a comprehensive, cross-disciplinary foundation that unites AI pipelines with disease-specific biological insights. Together, a combined scattered knowledge of 15 review articles fails to provide a unified framework encompassing data availability, methodological trends, and disease-specific biomarker discoveries across therapeutic areas. Most prior efforts have concentrated on narrow aspects, i.e., either focusing on disease-specific AI models or individual stages of the biomarker discovery pipelines, leaving a substantial gap in translational utility. This study addresses this gap by systematically consolidating and analyzing findings from 236 AI-driven biomarker discovery studies. We systematically map the trends of datasets, data modalities, preprocessing strategies, feature engineering methods, AI models, and explainability methods across 147 diseases, which we further organize into 19 therapeutic areas. By doing so, we aim to provide a comprehensive resource that not only highlights current trends and gaps but also lays the groundwork for future advancements, including the design of multi-task learning models and multimodal AI frameworks tailored to complex biomedical data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.28.672795v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Long-read sequencing of single cell-derived melanoma subclones reveals divergent and parallel genomic and epigenomic evolutionary trajectories</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liu, Y., Goretsky, A., Keskus, A., Malikic, S., Ahmad, T., Gertz, M., Mehrabadi, F. R., Kelly, M. C., Hernandez, M., Seibert, C., Caravaca, J. M., Kline, K., Zhao, Y., Wu, Y., Shrestha, B., Tran, B., Ghosh, A., Cui, X., Sassano, A., Malik, L., Baker, B., Blauwendraat, C., Billingsley, K. J., Perez-Guijarro, E., Merlino, G., Molloy, E., Sahinalp, S. C., Day, C.-P., Kolmogorov, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Tumor evolution is driven by various mutational processes, ranging from single-nucleotide variants (SNVs) to large structural variants (SVs) to dynamic shifts in DNA methylation. Current short-read sequencing methods struggle to accurately capture the full spectrum of these genomic and epigenomic alterations due to inherent technical limitations. To overcome that, here we introduce an approach for long-read sequencing of single-cell derived subclones, and use it to profile 23 subclones of a mouse melanoma cell line, characterized with distinct growth phenotypes and treatment responses. We develop a computational framework for harmonization and joint analysis of different variant type in the evolutionary context. Uniquely, our framework enables detection of recurrent amplifications of putative driver genes, generated by independent SVs across different lineages, suggesting parallel evolution. In addition, our approach revealed gradual and lineage-specific methylation changes associated with aggressive clonal phenotypes. We also show our set of phylogeny-constrained variant calls along with openly released sequencing data can be a valuable resource for the development of new computational methods.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.28.672865v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Linking stomatal function with photosynthetic light reactions and stress response in faba bean</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shapiguzov, A., Punkkinen, M., Laine, T., Engstrom, S., Aphalo, P. J., Khazaei, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Faba bean (Vicia faba L.) is a key protein crop, but its cultivation and yield stability are hindered by a number of environmental stresses. Stomata regulate gas exchange between the plant and atmosphere, playing a central role in photosynthesis and mediating plant responses to a wide range of environmental stressors. This study aimed to investigate variations in leaf temperature as a proxy for stomatal function and photosynthetic regulations in faba bean, and to examine the response of this crop to short-term acute ozone (O) exposure stress. Here, we used a high-throughput plant phenotyping (HTPP) platform to screen 196 faba bean genotypes for photosynthetic and stomatal function under controlled conditions. A subset of extreme genotypes, identified based on relative leaf tempreture from the initial screening, was exposed to a 450 ppb O treatment. Our results revealed strong positive relationship between photosynthetic efficiency and relative leaf temperature. A three-fold difference in relative leaf temperature was observed among genotypes. The O treatment caused signicantly less damage in genotypes with higher leaf temperature compared to those with lower leaf temperature (p<0.001). By combining a HTPP platform with elevated O stress treatment, we identified faba bean genotypes with contrasting stomatal responses to the O exposure. Our results advance understanding of the regulation mechanisms of photosynthetic light reactions and the role of stomatal function in modulating faba bean responses to environmental stressors.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.29.673109v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A cytokine receptor-targeting chimera (kineTAC) toolbox for expanding extracellular targeted protein degradation.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kumru, K., Yao, Z., Holmes, B. B., Zhao, F., Zhang, Y., Ferrara, E., Peters-Clarke, T. M., Leung, K. K., Wells, J. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Extracellular targeted protein degradation (eTPD) is as an important new modality for manipulating the extracellular proteome. However, most eTPD receptors are expressed broadly or are restricted to the liver. Cytokine receptor targeting chimeras (kineTACs) are genetically encoded bispecifics for eTPD that fuse a natural ligand like CXCL12 to an antibody, directing soluble or membrane proteins for lysosomal degradation using the widely expressed chemokine receptor CXCR7 (Pance K, Gramespacher JA., Byrnes, JR, Salangsang F., Serrano JAC, Cotton AD, Steri V, and Wells JA, Nat. Biotechnol. 2023, 41, 273-281). Here, we dramatically expand the kineTAC toolbox by constructing 81 new kineTACs based on an unbiased list of cytokines, chemokines and growth factors. Remarkably, 55 of these expressed at suitable levels for analysis without any optimization. Many of these kineTACs bind receptors that have unique cell-type expression profiles, allowing for eTPD in specific cells and tissues and some were more potent than the original CXCL12-based kineTAC. We further show the internalizing capability of a kineTAC can enhance the performance of antibody drug conjugates. We believe these simple, genetically encoded tools will be useful for expanding the applications for optimized or cell type-selective eTPD.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.02.673832v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The tyrosine kinase inhibitor GNF-7 targets senescent cells through allosteric activation of GCN2.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lopez-Pernas, G., Murga, M., Ahmed, W., Haggblad, M., Jimenez-Ortega, E., Serrano, A. G., Cardona, C., Lopez-Prieto, M., Fisicaro, S., Mota-Pino, J., Navarro-Gonzalez, B., Zarzuela, E., Anton, M. E., Lidemalm, L., Matinez, S., Isasa, M., Pastor, J., Fernandez-Leiro, R., Huhn, D., Fernandez-Capetillo, O.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The one-two-punch approach refers to the sequential administration of two different chemotherapies, the second of which targets cancer cells that resisted the initial treatment. To find such a second punch, we performed a chemical screen to find drugs that are preferentially toxic for cells with an activated DNA damage response (DDR). This screen identified the tyrosine kinase inhibitor GNF-7 as a top hit. Subsequent work revealed that GNF-7 is a potent senolytic, even when senescence is triggered by therapies that do not activate the DDR. Consistently, GNF-7 is highly efficacious to kill cancer cells previously treated with CDK4/6 inhibitors, including in patient-derived organoids and mouse xenografts. Surprisingly, the senolytic effect of GNF-7 is not mediated by the inhibition of a tyrosine kinase (TK), but rather by the activation of GCN2, an effect previously reported for other TK inhibitors. Together, our study reports the discovery of a novel senolytic agent that strongly synergizes with CDK4/6 inhibitors when applied sequentially and expands our understanding of the mechanisms behind the anticancer effects of TK inhibitors.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.29.673062v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

