<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Saturday, 13 September 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AI-Supported Extraction of Functional Tissue Unit Properties for Human Reference Atlas Construction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kong, Y., Börner, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Large-Model Retrieval-Augmented Generation for HRA FTU-relevant properties extraction from scholarly publications .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The Human Reference Atlas (HRA) effort brings together experts from more than 25 international consortia to capture the multiscale organization of the human body-from large anatomical organ systems (macro) to the single-cell level (micro). Functional tissue units (FTU, meso) in 10 organs have been detailed and 2D illustrations have been created by experts. Comprehensive FTU property characterization is essential for the HRA, but manual review of the vast number of scholarly publications is impractical. Here, we introduce Large-Model Retrieval-Augmented Generation for HRA FTUs (HRAftu-LM-RAG), an AI-driven framework for scalable and automated extraction of FTU-relevant properties from scholarly publications. This validated framework integrates Large Language Models for textual reasoning, Large Vision Models for visual interpretation, and Retrieval Augmented Generation for knowledge grounding, offering a balanced trade-off between accuracy and processing efficiency. We retrieved 244,640 PubMed Central publications containing 1,389,168 figures for 22 FTUs and identified 617,237 figures with microscopy and schematic images. From these images and associated text, we automatically extracted 331,189 scale bars and 1,719,138 biological entity mentions, along with donor metadata such as sex and age. The resulting data facilitates the design, review, and approval of future FTU illustrations during HRA construction.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.11.675730v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">OmniExtract: An automatic data extraction tool based on Large Language Model and Prompt Engineering</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, Y., Tang, B., Wu, S., Meng, Y., Kong, D., Zhao, W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose OmniExtract, an automatic data extraction tool with user-friendly configuration files which can adapt to various data extraction tasks which can support a comprehensive data extraction including text and tables.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Extracting structured information from documents or scientific papers is crucial for data sharing and retrieval. Recently, Large Language Model (LLM) has shown its impressive ability in text understanding and several tools based on LLM has been developed. However, it's still difficult to find a universal and user-friendly tool for various practical extraction tasks. To address this challenge, we propose OmniExtract, an automatic data extraction tool with user-friendly configuration files which can adapt to various data extraction tasks. OmniExtract uses a prompt optimized engineering to improve prompt and obtain high performance, and it can support a comprehensive data extraction including text and tables. Evaluation results show that OmniExtract obtains a high accuracy over 80% for 3 datasets. Furthermore, two additional data extraction applications using OmniExtract have been provided, achieving an accuracy of 92.21% and an average F1 score of 0.83 respectively. The data reliability performance shows that OmniExtract is a valuable tool for database updating.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.11.675332v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">OmniPath: integrated knowledgebase for multi-omics analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Turei, D., Schaul, J., Palacio-Escat, N., Bohar, B., Bai, Y., Ceccarelli, F., Cevrim, E., Daley, M., Darcan, M., Dimitrov, D., Dogan, T., Domingo-Fernandez, D., Dugourd, A., Gabor, A., Gul, L., Hall, B. A., Hoyt, C. T., Ivanova, O., Klein, M., Lawrence, T., Mananes, D., Modos, D., Muller-Dott, S., Olbei, M. L., Schmidt, C., Sen, B., Theis, F. J., Unlu, A., Ulusoy, E., Valdeolivas, A., Korcsmaros, T., Saez-Rodriguez, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed OmniPath, a database combining diverse molecular knowledge from 168 resources, which can be used for omics analysis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Analysis and interpretation of omics data largely benefit from the use of prior knowledge. However, this knowledge is fragmented across resources and often is not directly accessible for analytical methods. We developed OmniPath (https://omnipathdb.org/), a database combining diverse molecular knowledge from 168 resources. It covers causal protein-protein, gene regulatory, miRNA, and enzyme-PTM (post-translational modification) interactions, cell-cell communication, protein complexes, and information about the function, localization, structure, and many other aspects of biomolecules. It prioritizes literature curated data, and complements it with predictions and large scale databases. To enable interactive browsing of this large corpus of knowledge, we developed OmniPath Explorer, which also includes a large language model (LLM) agent that has direct access to the database. Python and R/Bioconductor client packages and a Cytoscape plugin create easy access to customized prior knowledge for omics analysis environments, such as scverse. OmniPath can be broadly used for the analysis of bulk, single-cell and spatial multi-omics data, especially for mechanistic and causal modeling.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.11.675512v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Fine-scale recombination rates inferred using the canFam4 assembly are strongly correlated with previous maps of dog recombination</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kidd, J. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We describe genetic maps based on the canFam4 assembly which will be useful for future studies of genetic variation that rely on updated dog genome references.</p><p style="font-size: 16px; text-align: justify;">Background: Genetic maps are a key resource for the analysis of genetic variation. Prior maps in dogs were based on the canFam3.1 genome assembly, which contains thousands of sequence gaps that include functional elements. Although long-read technologies have enabled the generation of updated reference assemblies, updated maps are required to take full advantage of these new resources. Methods: We inferred fine-scale recombination maps that consider population demographic history for four populations of village dogs sequenced by the Dog10K Consortium. We compare properties among the newly generated and previously published maps and assess the effects of genetic map choice on genotype phasing accuracy. Results: The newly inferred maps are broadly consistent with previous studies of recombination in dogs. At small scales, stronger correlations are observed among maps inferred using linkage disequilibrium than with a pedigree-based map. The position of recombination hotspots showed significant overlap among inferred maps and were enriched for sequences that were gaps in a previous canine genome assembly. When using the Dog10K samples as a reference panel, a phase error rate of approximately 2% was found with the choice of genetic map having little effect. Conclusions: We describe genetic maps based on the canFam4 assembly which will be useful for future studies of genetic variation that rely on updated dog genome references.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.11.675639v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genome assembly and annotations of a Cyprinidae fish, Sinibrama wui</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, Y., Liu, J., Liu, J., Zhang, D., Wang, L., Xie, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We assembled a chromosome-level genome assembly of Sinibrama wui, a freshwater fish found in the Yangtze River in China.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Sinibrama wui is a freshwater fish belonging to the Cyprinidae family, primarily found in the middle and upper Yangtze River of China. However, no reference genome sequence for S. wui has yet to be established. The present study presents a high-quality, chromosome-level genome assembly of S. wui through PacBio Biosciences sequencing and Hi-C technologies. The genome size was 1028.95 Mb, with a contig N50 length of 37.21 Mb and scaffold N50 of 40.74 Mb. We assembled 124 contigs, within which 68 were organized into 24 chromosomes. In total, 25,967 protein-coding genes were predicted, with 99.10% functionally annotated. We further analyzed the evolutionary relationships, gene families expansion, and positive selection of S. wui. Our data showed that S. wui diverged from Megalobrama amblycephala approximately 10.1 million years ago, and the expanded gene families had enriched in the lipid metabolism and nervous system. These datasets offer valuable genomic resources for future investigations studies on the genetic conservation, selective breeding, and evolution of S. wui.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.11.675668v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">RNA-tailor: accurate gene-level identification of transcript isoform diversity from long reads</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Marchand, L., Touzet, H., Varre, J.-S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that RNA-tailor is an easy-to-use tool to perform single gene resolution of alternative splicing isoform repertoire with high accuracy.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurate splicing isoform identification is an essential need for progress of modern medicine and biological knowledge. The advent of long-read sequencing technologies opened up the possibility of sequencing full length transcripts. Most of the existing methods aims to predict isoforms at genome scale. However, solving all isoforms at genome scale is not always needed, mostly when one is interested in a single gene or a few genes. It also brings algorithmic constraints, encouraging the use of less accurate alignment algorithms and downgrading sensitivity of methods to determine the repertoire of isoforms. The aim of RNA-tailor is to make available an easy-to-use tool to perform single gene resolution of alternative splicing isoform repertoire with high accuracy. To leverage such precision, RNA-tailor uses a combination of exact alignment algorithm and context aware alignment corrections. By analyzing both real and simulated datasets, we show that RNA-tailor is able to achieve higher sensitivity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.09.673982v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Whole genome sequencing reveals the phylogenetic relationship and reticulate evolution of Phrynocephalus lizards in Eurasia</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cui, L., Nazarov, R. A., Chen, Y., Qi, Y., Guo, X., Xia, Y., Yang, W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A phylogenetic analysis of Phrynocephalus lizards from the Caspian Basin to the Qinghai-Tibetan Plateau and the Gobi Desert .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Toad-headed agama (genus Phrynocephalus) is a widely distributed group of lizards across Central Eurasia, inhabiting diverse arid and semi-arid environments from the Caspian Basin to the Qinghai-Tibetan Plateau (QTP). Despite considerable research efforts over the past decades, a robust phylogenetic framework for Phrynocephalus has remained elusive. In this study, we collected 26 samples representing a substantial portion of the distribution of this genus and established a well-supported phylogeny based on whole-genome sequencing and phylogenomic approaches. We found that the Phrynocephalus lizards comprised of three major clades, including the Western Asia, Central Asia and viviparous clades. Furthermore, we provided compelling evidence for an ancient hybridization event that led to the speciation of P. mystaceus, which likely contributed to the observed phylogenetic discordance across the genome. Divergence time estimation and ancestral area reconstruction indicated that Phrynocephalus lizards originated approximately 31.57 million years ago in the Central Asia and Iranian Plateau region, followed by Miocene dispersals into the Arabian Peninsula, the Gobi Desert, and the QTP. Our results illuminate how hybridization and historical range dynamics have jointly driven the origin and diversification of Phrynocephalus, providing new insights into the formation of biodiversity in Eurasia.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.09.675123v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Unique molecular identifiers don't need to be unique: a collision-aware estimator for RNA-seq quantification</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Agyemang, D., Irizarry, R. A., Baharav, T. Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A method-of-moments estimator that accounts for UMI collisions, accurately quantifying gene expression and preserving downstream biological insights.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> RNA-sequencing (RNA-seq) relies on Unique Molecular Identifiers (UMIs) to accurately quantify gene expression after PCR amplification. Longer UMIs minimize collisions---where two distinct transcripts are assigned the same UMI---at the cost of increased sequencing and synthesis costs. However, it is not clear how long UMIs need to be in practice, especially given the nonuniformity of the empirical UMI distribution. In this work, we develop a method-of-moments estimator that accounts for UMI collisions, accurately quantifying gene expression and preserving downstream biological insights. We show that UMIs need not be unique: shorter UMIs can be used with a more sophisticated estimator.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.08.674884v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Fundamental Limits of Inferring Dynamical Gene Regulatory Models from Single-Cell Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Balubaid, A., Bernal Tamayo, J. P., Kumar, R., Agboola, H., Cabrero, D. G., Kiani, N., tegner, j.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A long-standing goal in systems biology is to construct computational models that capture gene regulatory dynamics. We evaluate several computational methods to infer Jacobian eigenvalues from single-cell data, but find that all methods are susceptible to data quality, rendering them unreliable under real-world conditions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> A long-standing goal in systems biology is to construct computational models that capture gene regulatory dynamics. On the one hand, graph-based regulatory network models rely on prior knowledge but are constrained by unknown kinetic parameters. On the other hand, recent advancements in single-cell sequencing technologies, particularly RNA velocity and pseudotime analysis, offer data-driven approaches to infer kinetic parameters. However, accurate modeling of biological dynamics requires the estimation of the Jacobian matrix, which captures signal propagation within a regulatory system. Several computational methods have recently emerged to infer Jacobians from single-cell data, including SpliceJAC, scMomentum, and Dynamo. Here, we evaluate these methods by analyzing Jacobian eigenvalues, assessing their ability to capture stability and oscillatory properties. Using simulated ground truth data, we find that all methods are susceptible to data quality, rendering them unreliable under real-world conditions. In conclusion, beyond improvements in data quality, there is a critical need to develop more robust computational methods for estimating Jacobians from single-cell data. Addressing these challenges will enable the derivation of more accurate dynamical models of gene regulation at scale.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.12.674509v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Reverse Engineering the Evolutionary Logic of Cytoskeletal Dynamics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Larios, D., Najma, B., Miao, J., Lee, H. J., Thomson, M., Phillips, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We develop a cell-free approach to reconstitute microtubule dynamics driven by genetically encoded kinesin motor variants in bacterial lysate droplets, enabling quantitative characterization of motor-induced cytoskeletal behavior in the cell-like context.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Living cells perform tasks by generating mechanical force through networks of protein filaments that are actively organized by molecular motors. Eukaryotic genomes encode diverse motor protein variants with distinct biochemical properties, which cells deploy for specific processes such as spindle formation during division or cilia for navigation. Despite extensive characterization of motor-filament systems by in vitro and in vivo approaches, how variations in genomic sequence and structure translate into filament organization dynamics remains poorly understood. Here we develop ActiveDROPS, a cell-free approach to reconstitute microtubule dynamics driven by genetically encoded kinesin motor variants in bacterial lysate droplets, enabling quantitative characterization of motor-induced cytoskeletal behavior in a cell-like context. We screened a library of uncharacterized kinesin-like sequences and identified 12 functional motors that can be grouped into three behavioral classes: "slow-sustained" (prolonged motion >12 h, velocity 10-50 nm/s), "fast-burst" (brief motion 12 h, velocity >100 nm/s). This revealed domain sets controlling microtubule motion lifespan and velocity, with computational analysis of AlphaFold-predicted structures suggesting a0 and L12 as the respective determinants. Incorporation of microtubule-associated proteins such as PRC1 and Tau, together with motor co-expression, further broadened the range of cytoskeletal dynamics. These results provide a framework to reverse engineer how nature generates mechanical diversity, offering insight into how cells transform genetic variation into emergent behavior.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.10.674348v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ibSLS: A Biobank for Democratizing Access to Multi-Omics Data and Biospecimens from Spaceflight Research</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Otsuki, A., Aoki, Y., Okada, R., Kamimura, D., Shiba, D., Hishinuma, E., Koshiba, S., Katsuoka, F., Kinoshita, K., Suzuki, T., Uruno, A., Yamamoto, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed the integrated biobank for Space Life Sciences (ibSLS; https://ibsls.megabank.tohoku.ac.jp), a data-visualization and sample-sharing platform that provides access to spaceflight-derived data and biological resources.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> During spaceflight, astronauts are exposed to extreme conditions such as microgravity, cosmic radiation, and confinement, which can cause a wide range of health problems. To elucidate the molecular mechanisms underlying these issues and to develop intervention strategies for maintaining physiological homeostasis during space missions, space life science research using mouse models is actively conducted on the International Space Station. However, because of the high cost and technical complexity of spaceflight experiments, it is critical to maximize the scientific value of each mission by ensuring broad accessibility to both data and biospecimens. To this end, we developed the integrated biobank for Space Life Sciences (ibSLS; https://ibsls.megabank.tohoku.ac.jp), a data-visualization and sample-sharing platform that provides access to transcriptomic and metabolomic datasets generated from Mouse Habitat Unit (MHU) missions. The platform features a user-friendly interface, tools for cross-mission analysis, and integration with human multi-omics databases, enabling cross-species comparisons. In addition, ibSLS facilitates biospecimen requests to support downstream research. By promoting open access to spaceflight-derived data and biological resources, ibSLS encourages the participation of researchers from diverse fields in space life science. We believe that ibSLS will make a valuable contribution to both biomedical research on spaceflight-related health issues and the study of diseases on Earth.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.08.675003v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CD155 regulates tumor growth and immune evasion in diffuse midline glioma.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tzaridis, T. D., Fernandez, E. C., Eisemann, T., Andrade, A. F., de Biagi-Junior, C. A. O., Hope, J. L., Becher, O. J., Jabado, N., Larson, J. D., Baker, S. J., Califano, A., Bagchi, A., Filbin, M. G., Bradley, L. D., Adams, P. D., Pavisic, J., Wechsler-Reya, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that CD155 regulates immune evasion and tumor growth in DMG, and suggest that targeting CD155 could be a valuable two-pronged therapeutic strategy for this disease.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Diffuse midline glioma (DMG) is a devastating pediatric brain tumor with an unmet need for novel therapies. Immune checkpoint inhibitors have failed to prolong survival for DMG patients. In this study, we analyzed the expression of immune checkpoint molecules in human and murine DMG cells, as well as primary brain tumor samples, and identified CD155 as the most highly expressed. When murine DMG cells were co-cultured with CD8+ T cells, silencing of CD155 led to a marked increase in T cell-mediated killing. Strikingly, CD155-deficient DMG cells failed to grow in immunocompetent mice, and depletion of CD8+ T cells allowed these tumors to grow. CD155 also exerted cell-autonomous effects on tumor cells: silencing of CD155 led to induction of apoptosis of DMG cells and to delayed tumor growth in immunodeficient mice. Transcriptomic analyses identified FOXM1 as a key target of CD155. Notably, FOXM1 silencing also led to reduced proliferation of DMG cells in vitro and in vivo. Finally, treatment of DMG-bearing mice with Thiostrepton, a FOXM1-targeting agent, delayed tumor growth and prolonged survival. These studies demonstrate that CD155 regulates immune evasion and tumor growth in DMG, and suggest that targeting CD155 could be a valuable two-pronged therapeutic strategy for this disease.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.09.674382v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Whole genome CRISPR knockout screen reveals ID3 as a key regulator of myeloma cell survival via TCF3 and c-MYC</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Andersson-Rusch, C., Hanif, M. A., Quist-Lokken, I., Satrom, P., Rolinski, M., Nordstrand Moen, J. F., Campos Roman, N., Misund, K., Beisvag, V., Aas, P. A., Van Loon, B., Holien, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We found that ID3, TCF4, and TCF12 were among genes that potentially protected cells from apoptosis in BMP-induced myeloma cells and that depletion of ID3 protected them from c-MYC downregulation and apoptosis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Bone morphogenetic proteins (BMP) induce apoptosis in myeloma cells and the mechanism behind this could point to new therapeutic targets. Here, we did a whole genome CRISPR/Cas9 knockout screen using the INA-6 myeloma cell line. Apoptosis was induced with BMP9 and the relative amounts of sgRNAs in treated versus control cells were determined with next-generation sequencing. We identified key positive control genes and a substantial number of novel genes that could be involved in BMP-induced apoptosis. One of the overrepresented genes was the known BMP target gene ID3. We found that ID3 was potently induced by BMP9 treatment and that depletion of ID3 protected cells from c-MYC downregulation and apoptosis. ID3 is known to heterodimerize with basic helix-loop-helix (bHLH) TCF transcription factors. In the screen, TCF3, TCF4, and TCF12 were among genes that potentially protected cells from apoptosis. Knockdown of TCF3, and to some extent TCF12, led to lower basal c-MYC levels and lower cell viability, and this was more pronounced after BMP9-treatment. Our results suggest that ID3 plays an important role in regulating the survival of myeloma cells, at least in part by forming heterodimers with TCF3 and thus preventing expression of the c-MYC oncogene.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.09.675036v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Highly multiplexed imaging recovers immune and metabolic niches in multiple myeloma associated with disease progression and bone involvement</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Roseth, I. A., Hatscher, L., Schiller, C., Johnstuen, H. S. S., Slordahl, T. S., Hov, H., Schapiro, D., Standal, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Spatial interactions between PCs and immune cells in multiple myeloma are a key factor in disease pathogenesis and the potential for spatial analyses to improve MM risk stratification.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Multiple myeloma (MM) is a cancer arising from genetically aberrant plasma cells (PCs) that remain dependent on the bone marrow (BM) microenvironment for disease establishment and progression. Here, we spatially mapped BM biopsies from patients with MM, smoldering MM and monoclonal gammopathy of undetermined significance by imaging mass cytometry. We found that PCs near bone surfaces were quiescent and demonstrated a distance-to-bone-dependent upregulation of IL-32 and HIF1A; in patients with bone disease, consistent with their role in promoting bone loss in MM. We further identified two distinct PC neighborhoods termed PC_OXPHOS characterized by focal PC growth, enrichment of endothelial cells and elevated oxidative phosphorylation; and PC_MYELOID, characterized by PCs interspersed with immune cells and featuring a glycolytic phenotype. Spatial interactions between PCs and immune correlated with shorter progression free survival. Notably, a strong neighbor preference between PCs and CD4+ T cells was an independent predictor for disease progression. Our work underscores spatial context as a key factor in understanding MM pathogenesis and the potential for spatial analyses to improve MM risk stratification.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.08.674473v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">TET2 loss impairs MPLA-induced innate immune memory during infection and disrupts hematopoiesis via RIPK1 in mice</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jarabek, A. N. L., Davis, X., Oliver, M., wang, y., Chen, H., Bohannon, J. K., Zinkel, S. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We found that TET2 loss impairs innate immune memory and infection responses, and MPLA with infection promotes hematopoietic skewing through RIPK1.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Ten-eleven translocation protein 2 (TET2) is commonly mutated in hematologic disorders of bone marrow failure such as myelodysplastic syndrome (MDS), and loss of TET2 is associated with poor prognosis. TET2 loss is also associated with augmented inflammation, as well as impaired innate immune responses. However, many individuals harboring mutations do not develop hematologic disorders, indicating that additional factors, such as inflammation, cooperate with TET2 loss to promote disease progression. Innate immune memory is a phenomenon in which pre-treatment with microbial ligands, including monophosphoryl lipid A (MPLA), improves innate immune function while minimizing inflammation during subsequent infection. Given the previously reported innate immune impairments attributed to TET2 loss, we tested whether MPLA could induce an innate immune response in TET2-deficient mice infected with P. aeruginosa. Moreover, due to the inflammatory role of TET2 loss, we investigated whether MPLA with infection would promote disease-related phenotypes. We found that TET2-deficient mice display impaired infection response to P. aeruginosa which is partially improved with MPLA pretreatment. Assessments of pathogenic clearance functions further showed decreased capacity in TET2-deficient innate immune cells. Moreover, TET2-defcient cells also exhibit impaired differentiation. MPLA pretreatment in infected mice promotes myeloid-biased hematopoiesis at the expense of erythroid- and megakaryocyte-biased hematopoiesis, resembling MDS-like disease progression. Intriguingly, inhibition of receptor-interacting serine/threonine-protein kinase 1 (RIPK1) dampens the effects of TET2 loss on hematopoietic perturbation. Collectively, we find that TET2 loss impairs innate immune memory and infection responses, and MPLA with infection promotes hematopoietic skewing through RIPK1.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.08.674943v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

