<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Wednesday, 17 September 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Telomere-to-telomere gap-free genome assembly of a male donkey and the identification of novel SVs associated with functional genes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> yang, t., yang, R., Liu, Y., feng, m., li, y., zhang, H., wang, x., yao, r., wu, j., xing, w., qian, s., zhao, c.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A telomere-to-telomere gap-free donkey genome, including a 29.78-Mb Y chromosome, aided by ONT and trio-binning approach.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Previous assemblies of the donkey genome remain with gaps and structural errors, and a complete donkey genome will greatly facilitate genetic research related to donkeys. In the present study, a 2.78-Gb telomere-to-telomere gap-free donkey genome (CAU_T2T_donkey) was assembled, including a 29.78-Mb Y chromosome, aided by ONT and trio-binning approach. CAU_T2T_donkey corrected the structural errors of previous assemblies and added a total of 153.8-Mb previously unresolved regions and 354 genes to the reference genome EquAss-T2T_v2. We identified a 1.9-Mb PAR on CAU_T2T_donkey- chromosome Y, and added 17.1Mb regions and 75 new genes to the chromosome Y of the previous reference genome ASM1607732v2. Multi-copy genes, such as TSPY, L1RE, ETY, HSFY, and ETSTY were also identified in CAU_T2T_donkey- chromosome Y. Totally 6 types of repetitive sequences in centromeric regions were identified, and the features of the centromeric regions were revealed, and satellite-free centromeres were identified. We aligned HiFi long-read sequences of donkeys from six breeds against CAU_T2T_donkey and identified SVs in previously unresolved regions, and some of the novel SVs were located in functional genes, such as AOX1 (Chr4:DEL61), ASIC2 (Chr13:INS954), and Twist2 (Chr19:DEL98).</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.12.675839v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">BRCA2-deficiency Causes Global Transcriptomic Alterations in Endothelial Cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nikfarjam, S., Singh, A., Bu, S., Nguyen, H. C., Singh, K. K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of gene expression in endothelial cells following BRCA2 loss in endothelium .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> This study aims to explore the alterations in gene expression following BRCA2 loss in endothelial cells and to investigate the potential contribution of BRCA2 in regulating the endothelial proteincoding transcriptome. Cultured human umbilical vein endothelial cells (HUVECs) were transfected with siBRCA2 or non-targeting scrambled RNA as control. Total RNA was extracted and used for RNA sequencing following purification and quality check. Transcription profiles and differentially expressed genes (DEGs) were identified. Enrichment of functions and signaling pathways analysis were performed based on Gene Ontology (GO) and the Kyoto Encyclopedia of Genes and Genomes (KEGG) database. In total, 3196 DEGs with (cut-off 1.5-fold) were identified, of which 1852 genes were upregulated, and 1344 genes were downregulated in BRCA2-deficient HUVECs. Cell cycle exit and neuronal differentiation 1 (CEND1) and mannose receptor C-type 1 (MRC1) were the most significantly up- and downregulated genes, respectively, in BRCA2deficient endothelial cells. The expression of top DEGs were further validated by RT-qPCR in HUVECs and in human coronary artery endothelial cells. The GO and KEGG analysis suggest that cell adhesion molecules and cytokine-cytokine receptor interaction may play an important role in BRCA2-deficient endothelial cells. Overall, these findings contribute to a greater understanding of the mechanisms involved in BRCA2-mediated endothelial function.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.14.676113v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Evolutionary remodeling of non-canonical ORF translation in mammals</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chang, Y., Lei, T., Zhou, F., Jiang, J., Huang, Y., Zhu, Z., Zhang, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comprehensive atlas of mammalian non-canonical open reading frames (ncORF) and their evolutionary dynamics .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Non-canonical open reading frames (ncORFs) are pervasive within transcripts annotated as "non-coding" or "untranslated regions" of mRNAs, yet their landscape under normal physiological conditions remains to be fully resolved, particularly outside humans. Here we applied a stringent and standardized pipeline to hundreds of high-quality ribosome profiling libraries from normal mammalian tissues and cell types, identifying 11,623 human and 16,485 mouse ncORFs. Evolutionary analyses revealed that thousands of ncORFs are subject to coding constraint and exhibit lineage-specific conservation, underscoring their functional potential. Ancient ncORFs are preferentially highly translated, broadly expressed, and enriched for lineage-specific conservation. Co-expression patterns further indicate that many ncORFs, especially ancient ones, are cotranslated with canonical coding sequences, consistent with functions mediated through protein-protein interactions. Together, these findings establish a comprehensive atlas of mammalian ncORFs and provide fundamental insights into their evolutionary dynamics and functional integration within the proteome.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.16.676541v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SomaMutDB 2.0: A comprehensive database for exploring somatic mutations and their functional impact in normal human tissues</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shea, A., Sun, S., Kennedy, J., Zhang, L., Vijg, J., Dong, X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present SomaMutDB 2.0, the most comprehensive resource currently available for characterizing somatic mosaicism and functional interpretation in human health and aging.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent advances in ultra-accurate sequencing technologies have revealed that somatic mutations accumulate across the human lifespan and may contribute to both normal aging and disease. These mutations are highly diverse, often non-recurrent, and functionally heterogeneous, which makes their biological impact difficult to evaluate systematically. Although many studies have profiled somatic mutations in individual tissues or limited cohorts, a centralized and scalable platform that integrates discoveries and supports functional interpretation has been lacking. To address this gap, we present SomaMutDB 2.0 (https://somamutdb.org/SomaMutDB/), a substantially expanded database that catalogs 8.9 million mutations (8.57 million SNVs and 0.29 million INDELs) from 10,852 samples of 607 human subjects across 47 studies. Beyond expanded data coverage, SomaMutDB 2.0 introduces a comprehensive functional annotation framework that applies 22 predictive models, encompassing coding, regulatory, expression-based, and ensemble predictors, to systematically assess mutational impact. Users can browse pre-annotated variants through an interactive interface or upload their own variants for real-time analysis, with results contextualized against all mutations from normal, non-diseased tissues in the database. Together, these advances establish SomaMutDB 2.0 as the most comprehensive resource currently available for characterizing somatic mosaicism and functional interpretation in human health and aging.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.14.676124v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MOLECULAR STRATIFICATION OF INTESTINAL AND DIFFUSE SUBTYPES OF GASTRIC ADENOCARCINOMA AND ITS PROGNOSTIC IMPACT</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mourao, R. M. d. S., da Silva, J. M. C., da Silva, V. C. S., Moreira, F. C., de Assumpcao, P. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We tested whether transcriptomic features refine risk beyond histology and whether biomarker signals generalize across populations. We found that the utility of biomarkers in gastric cancer depends on the population context.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Gastric adenocarcinoma is routinely classified by Lauren histology into Intestinal and Diffuse types; however, its prognostic use is limited by molecular heterogeneity within each subtype. We tested whether transcriptomic features refine risk beyond histology and whether biomarker signals generalize across populations. The premise is that when underlying programs diverge across tumors and cohorts, a single biomarker is unlikely to be universally informative; accordingly, integrating molecular signatures with histology may improve risk stratification. We analyzed TCGA-STAD and GTEx gene expression data for discovery and validated the findings in ACRG. Cell-program markers guided unsupervised clustering, and a predictive signature was learned using recursive feature elimination with a random forest and cross-validation. Pathway context was assessed with GSEA, and clinical relevance was evaluated within each Lauren subtype. Clustering of canonical gastric and intestinal programs yielded two molecular clusters that did not explain survival. However, Diffuse tumors with stronger intestinal-like programs tended to experience better outcomes. Consistently, the derived signature was associated closely with Lauren categories and defined two molecular groups. Nevertheless, neither histology nor the signature alone stratified overall survival at conventional thresholds. Moreover, gene-level survival analyses indicated subtype specificity rather than universality. In TCGA, 28 signature genes were associated with overall survival (P < 0.05; HR, 0.79-1.63). ROC analyses for Lauren typing were heterogeneous, with CRLF1 achieving an AUC of 0.92, followed by CYP1B1 and MMRN2. ACRG gene expression revealed an inversion of the TCGA pattern, with prognostic risk shifting toward the Diffuse subtype. Specifically, CRLF1, ERG, and FRZB shifted from Intestinal risk to Diffuse risk, while CDX1 shifted from Intestinal to Diffuse protection. Taken together, these findings suggest that the utility of biomarkers in gastric cancer depends on histology and the population context. Consequently, a contextual strategy that integrates molecular signatures with the Lauren classification has the potential to improve risk stratification beyond either source alone.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.14.676054v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Time in sympatry correlates with the strength of reproductive isolation in hybridising Typha</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Aleman, A., Freeland, J., Dorken, M., Shafer, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We reconstructed their demographic histories and characterised barrier loci between Typha angustifolia and T. x glauca in North America and Europe, and show that prolonged sympatry can promote the evolution of reproductive barriers, whereas prolonged allopatry reduces their development.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In North America, hybridisation between Typha angustifolia and T. latifolia results in the highly impactful T. x glauca. In Europe, where its parental taxa are also sympatric, T. x glauca is scarce, suggesting stronger reproductive isolation on this continent. Using genomic data from the two species in North America and Europe, we reconstructed their demographic histories and characterised barrier loci between them. Demographic modelling suggests that the initial contact between the two species in Europe occurred ~800,000 years ago, indicating sympatry in that region since the Middle Pleistocene. In North America, their contact likely happened within the last centuries and was potentially driven by the human-mediated dispersal of T. angustifolia. We identified 47 candidate barrier loci between species in Europe, 6 of which are associated with reproductive functions. No barrier loci were found in North America. Our results suggest that prolonged sympatry can promote the evolution of reproductive barriers, whereas prolonged allopatry can reduce the likelihood of their development. Future studies could help determine whether time in sympatry is a predictor of the strength of reproductive isolation between hybridising taxa. Preventing invasions by hybrid taxa will require limiting the human-mediated dispersal of Typha (and other allopatric species) lacking reproductive isolation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.12.675895v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A haplotype-phased male genome sequence of the stinging nettle, Urtica dioica ssp. dioica</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hirabayashi, K., Percy, D., Gonzalez-Segovia, E. G., Deyholos, M., Cronk, Q. C., Todesco, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a chromosome-level, haplotype-resolved sequence of a diploid male plant; since the male is believed to be the heterogametic sex in Urtica dioica, this assembly represents a first step towards elucidating the control of sex determination in this</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Stinging nettle (Urtica dioica L.) is a widespread weed of economic significance with a dioecious mating system. Previously, we generated a high-quality genome assembly of a diploid female plant, which showed extreme levels of structural variation between haplotypes. Here, we present a chromosome-level, haplotype-resolved sequence of a diploid male plant; since the male is believed to be the heterogametic sex in Urtica dioica, this assembly represents a first step towards elucidating the control of sex determination in this species. This completely independently assembled genome also allows confirmation of previously reported features of the nettle genome, including (1) a high degree of structural variation between haplotypes, including large inversions, (2) the likely existence of polycentric centromeres, and (3) the presence of urticaceous pain peptide sequences. Chromosome 8 stands out for its multiple large, nested inversions and high levels of repetitive sequences, features that are often associated with sex determining regions (SDRs). This chromosome is therefore a candidate for further investigations to characterize the sex determination in nettle.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.12.675976v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Universal Duplex Sequencing Approach for Accurate Detection of Somatic Mutations</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nandi, S., Cheng, Y., Al-Azzam, S., Saeed, S., Kristin, A., Sunico, N., Stuewe, I., Jiang, Z., Culibrk, L., Zhivagui, M., Yang, X., Wise, R., Jacobs, F., Chavanel, B., Korenjak, M., Petljak, M., Balbo, S., Hudson, L. G., Liu, K. J., Zavadil, J., Gleeson, J. G., Alexandrov, L. B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present UDSeq, a duplex sequencing protocol combining random fragmentation, efficient UMI ligation, and quantitative input control to achieve near-complete genome/exome representation from as little as 100 pg DNA.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Ultra-accurate detection of rare somatic mutations is critical for understanding mutational processes in human disease, aging, and environmental exposures, yet current methods are limited by error rates, restricted genome coverage, and high DNA input. We present UDSeq, a duplex sequencing protocol combining random fragmentation, efficient UMI ligation, and quantitative input control to achieve near-complete genome/exome representation from as little as 100 pg DNA. Benchmarking in human sperm estimates a UDSeq error rate of ~2.5x10^-9 per base pair. UDSeq captures mutational signatures from heterogeneous populations without clonal expansion, reproduces exposure-specific patterns in cell lines and rodent models, and enables cross-species profiling. Compared with prior duplex methods, UDSeq yields up to fourfold more usable duplex molecules, improves library conversion, and remains cost-effective. We include a step-by-step protocol with quality-control checkpoints for fragment size, ligation yield, library conversion, and duplication rate. UDSeq provides a scalable, low-input platform for accurate profiling of somatic mutagenesis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.14.676103v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Ecological and evolutionary insights into the diversification of Atlantic bluefin tuna</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mikles, C., Pagniello, C. M., Bigal, E., Moran, B. M., Rooker, J., Ortega, A., De la Gandara, F., Maxwell, H., Schallert, R., Castleton, M., Schumer, M., Stokesbury, M., Block, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify adaptive variation in the Atlantic bluefin tuna genome that will be consequential for management decisions and critical for preserving locally adapted populations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Identifying and preserving biological diversity is fundamental for the management of wild populations. The Atlantic bluefin tuna (Thunnus thynnus; ABT) is a vital species to the North Atlantic Ocean ecosystem that is now rebounding from decades of overfishing. This teleost fish is notable for its large body size, unique form of endothermy, and trans-oceanic migrations that enable individuals to move rapidly between spawning and foraging locations. Here we combine high-resolution whole genome sequencing data with spatial and environmental data from electronic tagging to improve our understanding of population structure in ABT. We analyzed whole genome sequences (n=82) from both larvae and adult fish representing the two recognized stocks (western and eastern) of ABT, which originate from separate spawning grounds in the Gulf of Mexico and Mediterranean Sea. Analyzing these data, we identified 11,181,223 single nucleotide polymorphisms (SNPs), a dataset of unprecedented size and resolution. Coverage across the entire genome resulted in increased power in analyses of population structure, patterns of selection, and demographic history between the two recognized stocks. Notably, we report that both neutral and putatively adaptive SNPs are differentiated between populations, and through analyses of FST outlier SNPs, we discovered candidate genes with potentially adaptive roles. We suggest that both demographic history and oceanographic variation of the spawning grounds have contributed to shaping bluefin tuna genomic diversity. Our results characterize adaptive variation that will be consequential for management decisions and critical for preserving locally adapted populations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.15.675172v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An AI for an AI: identifying zoonotic potential of avian influenza viruses via genomic machine learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Brierley, L., Mould-Quevedo, J., Baylis, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We develop a novel zoonotic prediction model for avian influenza viruses, building upon "host-predictor" machine learning methods that estimate host potential given only a viral genome sequence.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Avian influenza remains a serious risk to human health via zoonotic transmission, as well as a feasible pandemic threat. Although limited zoonotic cases have resulted from the current epizootic outbreak, the wide diversity of influenza viruses in avian hosts means the emergence of new strains that could transmit to humans more readily cannot be ruled out. There is therefore a need to anticipate zoonotic potential before spillover occurs. Here, we develop a novel zoonotic prediction model for avian influenza viruses, building upon "host-predictor" machine learning methods that estimate host potential given only a viral genome sequence. We construct a machine learning framework combining individual sub-models of influenza genome segments, each trained on many genomic and proteomic traits (e.g., k-mer composition, codon biases, protein physicochemistry). To prevent over-fitting to heavily sampled lineages and ensure models generalise to phylogenetically distant viruses, we pre-process training data by considering clusters of shared sequence identity. Curated training sets cover ~4,000 representative, complete genome sequences of avian influenza from 120 subtypes including 9 containing known zoonotic viruses. We combine best-performing models into a single ensemble that can distinguish zoonotic capability of sequences held out from training with strong performance (AUROC = 0.95, F1 score = 0.90), including sequences of rarely-sampled subtypes, e.g., H10N8. Interrogating ensemble model decisions also allows us to identify influential genomic motifs most associated with human infection. These findings suggest specific genomic traits that are key to understanding and monitoring evolution of influenza viruses that circulate within bird populations. Our ensemble model can estimate zoonotic potential for new sequence inputs, offering a means to quickly risk-assess emerging avian influenza strains as soon as a sequence becomes available.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.16.676011v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Phylogenomic challenges in polyploid-rich lineages: Insights from orthology inference and reticulation methods using the complex genus Packera (Asteraceae: Senecioneae)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Moore-Pollard, E. R., Ellestad, P., Mandel, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We examine the impact of orthology inference and paralog processing on phylogenetic reconstruction in the complex genus Packera, and show that no method performs the best overall over others.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Phylogenomic discordance is pervasive and cannot be fully addressed through increased data alone. Biological processes such as polyploidy, hybridization, and incomplete lineage sorting are major contributors to discordance and must be accounted for to avoid misleading evolutionary interpretations. To better understand how these processes influence phylogenetic reconstruction, we conducted a comprehensive phylogenomic study in the complex genus Packera. With over 90 species and varieties (40% of which exhibit polyploidy, aneuploidy, or other cytological complexities), Packera presents significant challenges for phylogenetic reconstruction. Given these complexities, we assessed the impact of different published orthology inference methods on the resulting evolutionary relationships and phylogenetic support of this group. We then applied three paralog-processing methods to evaluate their impact on tree topology and our understanding of Packera's evolutionary history by constructing a time-calibrated phylogeny, reconstructing historical biogeography, and testing for ancient reticulation. Unsurprisingly, phylogenetic outcomes varied depending on the paralog processing method used, with no method performing the best overall over others. Our findings highlight the profound impact of orthology inference and paralog processing on phylogenomic analyses, particularly in polyploid-rich groups such as Packera, and we offer guidance on methodological impacts along with practical recommendations. We note that gaining a robust understanding of Packera's evolutionary history requires more than computational approaches alone and while technological advancements have greatly expanded our ability to analyze genomic data, effective phylogenomic research still relies on strong taxon sampling and detailed species knowledge. Without careful attention to the biological context, such as species boundaries, cytological variation, and ecological interactions, phylogenomic studies risk misinterpreting evolutionary history and processes. By accounting for these factors, we can begin to improve the accuracy of evolutionary reconstructions and gain deeper insights into the complex history of plant diversification.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.14.676143v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A plant-centric investigation of Class B Flavin-dependent Monooxygenase evolution and structural diversity</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Christensen, J. M., Neilson, E. H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comprehensive phylogenetic investigation of class B FMOs to uncover lineage-specific evolutionary patterns and structural diversification.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Class B Flavin-dependent monooxygenases (FMOs) are ancient and ubiquitous enzymes, present throughout diverse kingdoms of life. These enzymes utilize flavin-based coenzymes to incorporate an oxygen atom into their substrate, thereby altering its chemical properties. In plants, FMOs perform crucial catalytic functions essential for plant health, particularly within hormone biosynthesis, defense compound production, and immunogenic responses. Despite the evolutionary significance of class B FMOs across life, their evolution within the plant lineage remains underexplored and underrepresented. Here we present a comprehensive, plant-centric phylogenetic investigation of class B FMOs to uncover lineage-specific evolutionary patterns and structural diversification. Using known Class B FMOs as baits, a large selection of flavin-related proteins was assembled from species representing key lineages across Viridiplantae. Structural domain architecture and motif analysis was used to accurately define different Class B FMOs, resulting in eight distinct class B FMO families. Three families include the canonical YUCCA, N-OX and S-OX FMOs, which remain the most abundant and prevalent across the plant kingdom. Three families are novel, encompassing a small selection of bryophyte FMOs. The analysis also expanded the BVMO family to include monilophyte and angiosperm members, and a potential YUCCA-related family is reclassified as the evolutionary distinct "Seedless FMO" family. Considerable structural diversification within the NADPH-binding domain is observed across the eight families, and by assessing structurally conserved folds rather than amino acid sequence, a refined set of conserved FMO-specific motifs is defined. Overall, this phylogenetic and structural analysis provides new insights into FMO evolution and provides an important foundational framework to aid functional characterization of class B FMOs in plants.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.16.676513v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DNA Data Storage Architecture via Ligation of Dynamic DNA Bytes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jia, L., Shi, Y., Yang, J., Li, S., Yang, W., Li, W., Zhang, M., Li, Q., Zhang, Y., Wang, X., Li, L., Duan, B., Bu, D., Chen, F., Liu, H., Yang, H., Shi, Y., Liu, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a modular DNA storage architecture based on dynamic DNA bytes (DynaBytes) that can be ligated into reconfigurable information units. We stored 210,776 bits (26,347 bytes) of digital information organized within a file-system, and demonstrated CRUD (Create-Read</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The explosive growth of digital data is overwhelming conventional storage media, creating an urgent need for more efficient solutions. DNA offers immense potential for digital data storage, yet most systems remain static and archival. Here, we present a modular DNA storage architecture based on dynamic DNA bytes (DynaBytes)--pre-fabricated DNA segments that can be ligated into reconfigurable information units. Utilizing core, functional and control DynaBytes, we stored 210,776 bits (26,347 bytes) of digital information organized within a file-system, and demonstrated CRUD (Create-Read-Update-Delete)-like operations, hierarchical access and nanopore-based realtime retrieval. Robust data recovery was achieved under ~100x error-prone sequencing using streamlined error correction and fuzzy decoding. By relying on in vitro ligation of standardized components, DynaBytes reduces cost, scales efficiently, and allows interactive, rewritable storage. These features advance DNA storage beyond passive archiving toward a reconfigurable framework, opening new possibilities for dynamic, practical and large-scale DNA-based data systems.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.15.676197v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genomic REWRITE of Immune Interfaces Reveals Heritable Epigenetic Memory in Human Stem Cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Generoso, S. F., Levovitz, S., Jaramillo, S., Kim, M., Dara, S., Phen, S. F., Yi, B., Yanagi, T., DesMarais, T. L., Agmon, N., Hogan, M. S., Mitchell, L. A., Truong, D. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A modular platform for iterative, scar-minimized genome writing of synthetic constructs in human pluripotent stem cells, and a heritable epigenetic memory of an active state in key lineages .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Human leukocyte antigen (HLA) polymorphism hinders allogeneic therapies. We developed REWRITE, a modular platform for iterative, scar-minimized genome writing of synthetic constructs >100 kb in human pluripotent stem cells (hPSCs). Using REWRITE, we deleted 105-209 kb of the HLA locus and installed synthetic 24 kb or 100 kb HLA haplotypes at native and ectopic sites, and a 62 kb antigen-processing locus. This uncovered heritable epigenetic memory of an active state, whose resolution toward a silenced state is driven by native intergenic DNA. These loci restored inducible expression in key lineages, reducing NK-mediated cell killing and establishing HLA-matched T-cell tolerance. REWRITE provides an extensible platform for programming multigenic behaviors in human cells -- from immune interface design to discovery of principles governing human genome architecture.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.16.676382v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrase-On-Demand: Bioprospecting integrases for targeted genomic insertion of genetic cargo</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> McClain, H. M., Lowrey, L. C., Quinto, L. B., Torrance, E. L., Gagliano, T. R., Isaacs, F. J., Schoeniger, J. S., Williams, K. P., Mageeney, C. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that integrases from diverse bacteria and archaea can integrate into available attB sites in any target genome and propose a method to enable single-step site-specific genome editing in a broad spectrum of prokaryotes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Integrases recognize defined sites within DNA and facilitate homologous recombination-independent transfer of genetic fragments. These innate capabilities render integrases as powerful biotechnology tools that mediate efficient site-specific integration of large genetic cargos. Given that genomes often lack the integration sites recognized by frequently utilized integrases, integrase technology has largely been restricted to genetic engineering of model organisms into which attB sites can be synthetically introduced. To enable single-step site-specific integrase-mediated genome editing in a broad spectrum of prokaryotes, we have devised the Integrase-On-Demand (IOD) method. IOD systematically identifies integrases, from bacteria and archaea, that can integrate into available attB sites in any target prokaryote. Computational results show that diverse bacteria generally have multiple potentially useable native attB/integrase pairs. We confirmed the functionality of predicted integrase and attB pairs for mediating site-specific genomic integration of heterologous DNA into the genomes of Pseudomonas putida S12 and KT2440 and Synechococcus elongatus UTEX 2973 and measured efficiency of integration using suicide constructs. By eliminating the requirement to introduce non-native attB sites into the target genome, IOD may, when suitable transformation methods exist, allow facile genome integration of large constructs in non-model and possibly non-culturable bacteria.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.16.676606v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Modanovo: A Unified Model for Post-Translational Modification-Aware de Novo Sequencing Using Experimental Spectra from In Vivo and Synthetic Peptides</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Klaproth-Andrade, D., Bruns, Y., Gabriel, W., Nix, C., Bergant, V., Pichlmair, A., Wilhelm, M., Gagneur, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Modanovo, an extension of the Casanovo transformer architecture for de novo peptide sequencing, has been shown to outperform database-based methods for detecting modified and unmodified peptides.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Post-translational modifications (PTMs) play a central role in cellular regulation and are implicated in numerous diseases. Database searching remains the standard for identifying modified peptides from tandem mass spectra, but is hindered by the combinatorial expansion of modification types and sites. De novo peptide sequencing offers an attractive alternative, yet existing methods remain limited to unmodified peptides or a narrow set of PTMs. Here, we curated a large dataset of spectra from endogenous and synthetic peptides from ProteomeTools spanning 19 biologically relevant amino acid-PTM combinations, covering phosphorylation, acetylation, and ubiquitination. We used this dataset to develop Modanovo, an extension of the Casanovo transformer architecture for de novo peptide sequencing. Modanovo achieved robust performance across these amino acid-PTM combinations (median area under the precision-coverage curve 0.92), while maintaining performance on unmodified peptides (0.93), nearly identical to Casanovo (0.94). The model outperformed {pi}-PrimeNovo-PTM and showed increased precision and complementarity to the database search tool MSFragger. Robustness was confirmed across independent datasets, particularly at peptide lengths frequently represented in the curated dataset. Applied to a phosphoproteomics dataset from monkeypox virus-infected cells, Modanovo recovered numerous confident peptides not reported by database search, including new viral phosphosites supported by spectral evidence, thereby demonstrating its complementarity to database-driven identification approaches. These results establish Modanovo as a broadly applicable model for comprehensive de novo sequencing of both modified and unmodified peptides.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.12.675784v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PAQman: reference-free ensemble evaluation of long-read eukaryotic genome assemblies</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> O'Donnell, S., Li, N., Steenwyk, J. L., Geiser, D., Martin, F. N., Gluck-Thaler, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed the Post-Assembly Quality manager (PAQman), a tool that evaluates seven reference-free features of genome quality: Contiguity, Gene content, Completeness, Accuracy, Correctness, Coverage, and Telomerality.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Advances in long-read sequencing and assembly algorithms have made it easier and more cost effective to generate high-quality genome assemblies. However, assessing assembly quality remains challenging, as existing tools frequently rely on few metrics and/or require a reference assembly for comparison. To address this, we developed the Post-Assembly Quality manager (PAQman), a tool that evaluates seven reference-free features of genome quality: Contiguity, Gene content, Completeness, Accuracy, Correctness, Coverage, and Telomerality. PAQman integrates multiple commonly used tools alongside custom scripts, requiring users to provide only the genome assembly and its underlying long-read data, while providing a streamlined and consistent framework for quality assessment across datasets and organisms.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.11.675652v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Geno2pheno: Interpretable and accurate HIV antibody resistance prediction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pirkl, M., Schommers, P., Boehm, M., Buech, J., Kaiser, R., Lengauer, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a web service for the prediction of antibody resistance (geno2pheno[bNAbs]) to HIV-1, which can be extended to other viruses, like Sars-Cov2, in the future.</p><p style="font-size: 16px; text-align: justify;">Background: Antiretroviral therapy (ART) is a life saving option for people living with HIV-1 (PLWH) and is effective against many viral strains. The most common ARTs involve combinations of drugs targeting viral or cellular proteins. Most of these drugs have to be taken daily. An alternative to ARTs with established inhibitors comprises broadly neutralizing antibodies (bNAbs). However, bNAbs share the problem of viral resistance with protein inhibitors. We developed a web service geno2pheno[bNAbs] that allows users to upload viral genotypes and estimates the respective resistance to many common bNAbs. The service uses trained statistical models to classify the virus into sensitive and resistant, respectively or to regress the IC50. Methods: We used two linear models as well as two neural nets for each task and multi-task (MT) learning to train both models for IC50 prediction and classification simultaneously. During multi-task learning we penalize divergence of class and IC50 score in addition to the loss individual to each of the models. Findings: We compared the linear models of geno2pheno[bNAbs] to other state-of-the-art methods like recurrent neural nets and self-attention, and found them to be competitive in regard to accuracy and have the benefit of fast computation and being easily interpretable in regard to features, i.e., positions on the envelope. Interpretation: We developed a web service for the prediction of antibody resistance (geno2pheno[bNAbs]) to HIV-1, which is free to use and can be extended to other viruses, like Sars-Cov2, in the future.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.11.675561v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Accounting for longitudinal peak quality metrics with MSstats+ enhances differential analysis in proteomic experiments with data-independent acquisition</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kohler, D., Dogu, E., Bhattacharya, M., Karayel, O., Magana, M., Wu, A., Anania, V. G., Vitek, O.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce MSstats+, a computational workflow that takes as input not only peak intensities reported by tools such as Spectronaut, but also quality metrics such as peak shape and retention time, and longitudinal run order profiles of these metrics.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mass spectrometry-based proteomics with data-independent acquisition benefits from advanced instrumentation and computational analysis. Despite continued improvements, the quality of quantification may be poor for some measurements. As the scale of proteomic experiments increases, these poor-quality measurements are challenging to characterize by hand, yet they undermine the detection of differentially abundant proteins and the downstream biological conclusions. We introduce MSstats+, a computational workflow that takes as input not only peak intensities reported by tools such as Spectronaut, but also quality metrics such as peak shape and retention time, and longitudinal run order profiles of these metrics. MSstats+ translates these metrics into a single measure of quality, and downweights poor quality measurements when detecting differentially abundant proteins. The method offers a natural treatment of missing values, weighting the imputed values according to the quality metrics in the run. We demonstrate the accuracy of the resulting differential analysis in four experiments: two custom benchmarking studies with intentionally induced anomalies, a controlled mixture of proteomes, and a large-scale clinical investigation. MSstats+ is implemented in the family of open-source R/Bioconductor packages MSstats.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.11.675573v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Using cross-species co-expression to predict metabolic interactions in microbiomes.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Medema, M., Belzer, C., Koetsier, R., Handelsman, J., Chevrette, M., Reitz, Z., Zhu, Y., van der Hooft, J. J. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Using cross-species co-expression to predict microbial activity data to identify metabolic interactions .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In microbial ecosystems, metabolic interactions are key determinants of species relative abundance and activity. Given the immense number of possible interactions in microbial communities, their experimental characterization is best guided by testable hypotheses generated through computational predictions. However, widely adopted software tools - such as those utilizing microbial co-occurrence - typically fail to highlight the pathways underlying these interactions. Bridging this gap will require methods that utilize microbial activity data to infer putative target pathways for experimental validation. In this study, we explored approaches applying cross-species co-expression to predict interactions from microbial co-culture RNA-sequencing data. Specifically, we investigated the extent to which co-expression between genes and pathways of different bacterial species can predict competition, cross-feeding, and specialized metabolic interactions. Our analysis of the Mucin and Diet-based Minimal Microbiome (MDb-MM) data yielded results consistent with previous findings and demonstrated the methods potential to identify pathways that are subject to resource competition. Our analysis of the Hitchhikers of the Rhizosphere (THOR) data showed links between related specialized functions, for instance, between antibiotic and multidrug efflux system expression. Additionally, siderophore co-expression and further evidence suggested that increased siderophore production of the Pseudomonas koreensis koreenceine BGC deletion-mutant drives siderophore production in the other community members. In summary, our findings confirm the feasibility of using cross-species co-expression to predict pathways potentially involved in microbe-microbe interactions. We anticipate that the approach will also facilitate the discovery of novel gene functions through their association with other species metabolic pathways, for example, those involved in antibiotic response.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.11.675599v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DiscERN: An Automated Genome Mining Tool for the Discovery of Evolutionarily Related Natural Products</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Owen, J. G., Woolly, E. F., Lai, H.-E., Woolner, V. H., Little, R. F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Discoverer of Evolutionarily Related Natural Products, a user-friendly tool designed to address the gap between in silico prediction and bioactive compound discovery.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Targeted genome mining to expand known families of natural products is a powerful strategy for discovering bioactive compounds, yet it remains a significant bioinformatics challenge. While tools exist for de novo biosynthetic gene cluster identification and large-scale unsupervised clustering, dedicated methods for the targeted, hypothesis-driven expansion of user-defined BGC families are lacking. Here, we present DiscERN (Discoverer of Evolutionarily Related Natural products), a user-friendly tool designed to address this gap. DiscERN leverages a multi-modal ensemble method that integrates four complementary algorithms classifying biosynthetic gene clusters based on Pfam content, sequence homology, and predicted product structure. This approach allows users to strategically balance discovery sensitivity with predictive precision to suit diverse research goals. We demonstrate DiscERN's utility by applying it to a large collection of actinomycete genomes and validating its predictive power through the successful isolation of discomycin A, a new calcium-dependent lipopeptide antibiotic, from a silent biosynthetic gene cluster. DiscERN provides a robust and accessible platform that streamlines the path from genomic data to a prioritised list of candidate biosynthetic gene clusters, effectively bridging the gap between in silico prediction and bioactive compound discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.10.675446v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Uncertainty Modeling Outperforms Machine Learning for Microbiome Data Analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Konnaris, M. A., Saxena, M., Lazar, N., Silverman, J. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that published machine learning models fail to generalize: on average they perform worse than a naive baseline that always predicted the training set mean. We show that Bayesian partially identified models consistently outperformed normalization and machine learning approaches, providing a principled and reproducible foundation for microbiome inference.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Microbiome sequencing measures relative rather than absolute abundances, providing no direct information about total microbial load. Normalization methods attempt to compensate, but rely on strong, often untestable assumptions that can bias inference. Experimental measurements of load (e.g., qPCR, flow cytometry) offer a solution, but remain costly and uncommon. A recent high-profile study proposed that machine learning could bypass this limitation by predicting microbial load from sequencing data alone. To evaluate this claim, we assembled mutt, the largest public database of paired sequencing and load measurements, spanning 35 studies and over 15,000 samples. Using mutt, we show that published machine learning models fail to generalize: on average they perform worse than a naive baseline that always predicted the training set mean. These failures stem from covariate shift-limited shared taxa between studies, differences in community composition, and differences in preprocessing pipelines-that silently derail model inputs. In contrast, Bayesian partially identified models do not attempt to impute microbial load, but instead propagate scale uncertainty through downstream analyses. Across 30 benchmark datasets, Bayesian partially identified models consistently outperformed normalization and machine learning approaches, providing a principled and reproducible foundation for microbiome inference.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.12.675956v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">TRPML1 signaling at lysosomes-mitochondria nexus drives triple-negative breast cancer mitophagy, metabolic reprogramming and chemoresistance</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Syeda, A. K., Almasi, S., Benson, C. J., Kennedy, B. E., Yoast, R. E., Emrich, S. M., Slade, L., Pakkiriswami, S., Vijayan, V. V., Gujar, S., Pulinilkunnil, T., Trebak, M., El Hiani, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We demonstrate the lysosomal Ca2+;-release channel TRPML1 as a master regulator of mitochondrial bioenergetics in triple-negative breast cancers and highlight its potential as a therapeutic target to exploit the metabolic vulnerabilities of TNBC cells.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Inter-organelle signaling mechanisms, particularly those at the lysosomes-mitochondria interface, are critical for cancer cell metabolism, mitophagy and survival. However, the incomplete understanding of these mechanisms has limited the development of effective therapies, especially for triple-negative breast cancers (TNBC). Here, we demonstrate the lysosomal Ca2+;-release channel TRPML1 as a master regulator of mitochondrial bioenergetics in TNBC cells. TRPML1 knockdown (ML1-KD) in TNBC cells selectively compromises mitochondrial respiration, reprograms cell metabolism, and induces mitochondrial fragmentation without impacting non-cancerous cells. Mitochondria of ML1-KD TNBC cells sequester around the endoplasmic reticulum (ER), increasing mitochondria-ER contact sites at the expense of mitochondria-lysosomes contacts. Mechanistically, ML1-KD reduces lysosomal acidification, thus hindering autophagic flux and completion of autophagy. ML1-KD inhibits TFEB-mediated mitophagy and oxidative defense mechanisms while causing mitochondrial Ca2+ overload, further impairing mitochondrial function. These alterations render ML1-KD TNBC cells highly sensitive to doxorubicin and paclitaxel at low doses that are typically ineffective on their own. Together, our findings establish TRPML1 as a critical inter-organelle regulator and highlight its potential as a therapeutic target to exploit the metabolic vulnerabilities of TNBC cells.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.11.675705v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Epigenomic Heterogeneity of Non-Functional Pancreatic Neuroendocrine Tumors Uncovered by Single nucleus and Spatial ATAC Profiling</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, D., Di, x., Gao, f., Li, g., Lin, l., He, s., Zhang, d., Xu, j., Jin, y., Liang, y., Cecchini, m., Lacy, j., Kunstman, j., Kunz, p., Du, j., Liu, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A spatially resolved epigenomic framework for dissecting NF-PanNET heterogeneity and evolution, offering new biomarkers for molecular stratification and precision therapy.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Non-functional pancreatic neuroendocrine tumors (NF-PanNETs) account for the majority of neuroendocrine neoplasms arising in the pancreas and exhibit substantial clinical and biological heterogeneity, yet their epigenetic regulation and spatial architecture remain poorly understood. Here, we present an integrative study of NF-PanNETs across multiple tumor grades using single-nucleus ATAC-seq (snATAC-seq) and spatial ATAC-seq. snATAC-seq delineates the chromatin accessibility landscapes of distinct tumor subtypes, immune cells, and cancer-associated fibroblasts (CAFs), revealing key transcription factor (TF) programs that drive tumor progression and shape microenvironmental interactions. Spatial ATAC-seq further identifies two distinct tumor-stroma ecological niches: a proliferative niche marked by MYC and FOX family, and an invasive niche enriched for Snail family TFs and KRAS pathway activity. These findings demonstrate that cellular behavior in NF-PanNETs is governed not only by intrinsic epigenetic states but also by spatial context. Together, our study provides a spatially resolved epigenomic framework for dissecting NF-PanNET heterogeneity and evolution, offering new biomarkers and regulatory axes for molecular stratification and precision therapy.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.11.675640v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Insulin-Like Growth Factor 1 Receptor Regulates Breast Cancer Cell Adhesion through Beta-1 Integrin</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Galifi, C. A., Dogan, E., Almansa, L. F., Maingrette, K., Shah, S. S., Bulatowicz, J. J., Ebenezer, K., Miri, A. K., Wood, T. L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that IGF1R itself regulates cancer cell adhesion, and that IGF-1 stimulation and knockdown of IGF1r promote cell adjection, which seems paradoxical.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Introduction: The insulin-like growth factor (IGF-1/IGF1R) pathway has been implicated in breast cancer aggressiveness; however, inhibition of this pathway has not been successful in clinical trials, indicating a lack of understanding about its role in TNBC metastasis. Recent studies have explored IGF1R involvement in integrin function and cancer cell adhesion dynamics. The goal of this study was to test the hypothesis that IGF1R itself regulates cancer cell adhesion. Methods: We use MDA-MB-231 and Hs578T TNBC cell lines, siRNA-mediated knockdown, and adhesion assays to assess how IGF1R and integrin knockdowns impact cancer cell adhesion. Using xCELLigence E-plates, we quantify the effect of IGF-1 ligand stimulation versus IGF1R knockdown on functional cell adhesion. We also use HUVEC human endothelial cells to determine how IGF1R regulates adhesion to the endothelium. Results: We found that IGF-1 stimulation increased MDA-MB-231 TNBC adhesion, which was reversed by the IGF1R tyrosine kinase inhibitor BMS-754807 and the ligand-dependent receptor internalization inhibitor dansylcadaverine. Unexpectedly, IGF1R knockdown also potently stimulated cell adhesion. Concomitant {beta}1 integrin knockdown reversed the increased cell adhesion after both IGF-1 stimulation or IGF1R knockdown, indicating that the increased adhesion is {beta}1 integrin dependent. This was also seen via immunocytochemistry when cells were seeded on fibronectin. Finally, inhibiting IGF1R signaling also reduced MDA-MB-231 cell adhesion to HUVEC endothelial cells. Discussion: Both IGF-1 stimulation and IGF1R knockdown in TNBC cells promote cell adhesion, which seems paradoxical. However, the commonality of both interventions is removal of IGF1R from the cell surface, since IGF-1 stimulation causes IGF1R internalization and intracellular trafficking. Blocking IGF1R signaling using a tyrosine kinase IGF1R inhibitor preserves IGF1R on the cell surface. Thus, we propose a model whereby surface-bound IGF1R inhibits {beta}1 integrin function and blocks cell adhesion. This model is supported further by our finding that treatment of MDA-MB-231 cells with dansylcadaverine, which inhibits ligand-mediated receptor internalization, blocked the effect of IGF-1 on adhesion. These findings may explain why selective IGF1R receptor antagonists, which downregulate IGF1R protein upon chronic administration, were unsuccessful in the clinical setting.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.11.674989v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Glabridin from &lt;i&gt;Glycyrrhiza glabra&lt;/i&gt; L. Exhibits Anti-Triple-Negative Breast Cancer Effects by Regulating Various Cell Cycle Genes.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yuan C, Lin Z, Zhang L, Tao Q, Zheng H, Feng Y, Zhong W, Cheng H, Shao J, Yu M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Glabridin (GLA), a characteristic polyphenolic extracted from the renowned sweet plantGlycyrrhiza glabra L., demonstrates potent anticancer effects.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Glabridin (GLA), a characteristic polyphenolic extracted from the renowned sweet plantGlycyrrhiza glabra L., demonstrates potent anticancer effects. This study aims to investigate the effects of GLA on triple-negative breast cancer (TNBC) and its underlying mechanisms. The IC50 values of GLA in MDA-MB-231 and MDA-MB-468 cells were calculated as 62.48 ¬± 2.62 and 64.77 ¬± 1.86 ŒºM, respectively. In vivo experimental results confirmed that GLA (10 and 30 mg/kg) induced no hepatorenal toxicity in mice but significantly suppressed TNBC growth (P MCM8, MCM3, CDC6, EXO1, and ATAD2) involved in cell cycle pathways and two key metabolites (N,N-dimethylarginine and Kuwanon K) were identified. Microarray data and molecular biology experiments demonstrated that GLA significantly downregulated these key genes, thereby inhibiting the cell cycle progression in TNBC. Notably, CDC6 was identified as a target protein of GLA using a drug affinity responsive target stability and cellular thermal shift assay. Molecular docking results revealed that, compared to norcantharidin, GLA exhibited distinct binding energy (-7.6 vs -6.4 kcal/mol) and a different binding mode with CDC6, supporting its potential as a novel CDC6 inhibitor. These findings provide valuable insights into the potential therapeutic application of GLA for TNBC treatment.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40955780/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

