<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Wednesday, 09 July 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Machine learning assists regulated cell death crucial biomarker selection in adenocarcinoma of the lung: biological data testing and cell assay determination.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Han Ning, Ying Jiang, Mengli Zheng, Gao Yang, Lianjun Ma, Yachao Zhao</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A prognostic signature for lung cancer and prognostic programmed cell death (PCD) related genes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Lung cancer is a highly aggressive and lethal cancer requiring prognostic and predictive biomarkers for improving patient outcomes. Here, a prognostic signature for lung cancer was developed and prognostic programmed cell death (PCD)-related genes were identified.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40629109/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AncestryGeni: A novel genetic ancestry classification pipeline for small and noisy sequence data.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Eran Elhaik, Sara Behnamian, Michael Howe, Hongwei Tang, Huihuang Yan, Shulan Tian, Suganti Shivaram, Cinthya Zepeda Mendoza, Kylee MacLachlan, Saad Usmani, Mehdi Pirooznia, Gareth Morgan, Patrick Blaney, Francesco Maura, Linda B Baughn</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that genetic ancestry informative markers can be used to estimate genetic similarity to continental groups, and propose a method to identify continental groups using germline DNA.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Efforts to address health disparities are often limited by the lack of robust computational tools for inferring genetic ancestry by calculating an individual's genetic similarity to continental groups. We have already shown that a preferred alternative to self-described race is using ancestry informative markers (AIMs) that can be classified into ancestral components and used to estimate their similarity to those of known populations to identify continental groups. However, real-world genomic data can present challenges, including limited availability of germline DNA, a small number of AIMs for each sample, and the use of different variant calling software, limiting the application of existing solutions.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40627371/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The phylodynamic threshold of measurably evolving populations</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Weber, A., Kende, J., Oeversti, S., Duchene, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that a lack of temporal signal due to a narrow sampling window results in a prior that is overly informative relative to the data, such that a wider sampling window is more important than the outcome of tests of temporal signals.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The molecular clock is a fundamental tool for understanding the time and pace of evolution, requiring calibration information alongside molecular data. Sampling times are often used for calibration since some organisms accumulate enough mutations over the course of their sampling period. This practice ties two key concepts: measurably evolving populations and the phylodynamic threshold. Current dogma suggests that populations meeting these criteria are suitable for molecular clock calibration via sampling times. However, the definitions and implications of these concepts remain unclear. Using Hepatitis B virus-like simulations and analyses of empirical data, this study shows that determining whether a population is measurably evolving or has reached the phylodynamic threshold does not only depend on the data, but also on model assumptions and sampling strategies. In Bayesian applications, a lack of temporal signal due to a narrow sampling window results in a prior that is overly informative relative to the data, such that a prior that is potentially misleading typically requires a wider sampling window than one that is reasonable. In our analyses we demonstrate that assessing prior sensitivity is more important than the outcome of tests of temporal signal. Our results offer guidelines to improve molecular clock inferences and highlight limitations in molecular sequence sampling procedures.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.07.663495v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Evaluating microbial network inference methods: Moving beyond synthetic data with reproducibility-driven benchmarks</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ghaeli, Z., Aghdam, R., Eslahchi, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that bootstrap resampling and low-level noisy datasets (<=10% perturbation) effectively preserve key statistical properties of real microbiome data, such as diversity indices, abundance distributions, and sparsity patterns.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Microbial network inference is an essential approach for revealing complex interactions within microbial communities. However, the lack of experimentally validated gold standards presents a significant obstacle in evaluating the accuracy and biological relevance of inferred networks. This study delivers a comprehensive comparative assessment of six widely used microbial network inference algorithms--gCoda, OIPCQ, S-E(glasso), S-E(mb), SPRING, and SparCC--using four diverse real-world microbiome datasets alongside multiple types of generated data, including synthetic, noisy, and bootstrap-derived datasets. Our evaluation framework extends beyond conventional synthetic benchmarking by emphasizing reproducibility-focused assessments grounded in biologically realistic perturbations. We show that bootstrap resampling and low-level noisy datasets (<=10% perturbation) effectively preserve key statistical properties of real microbiome data, such as diversity indices, abundance distributions, and sparsity patterns. Conversely, synthetic datasets generated via the widely used SPIEC-EASI method exhibit substantial divergence from real data across these metrics. We find that while SparCC demonstrates superior robustness across varying data conditions, other methods tend to produce inflated performance metrics when evaluated on unrealistic synthetic networks. Notably, several algorithms fail to distinguish between structured and random networks, highlighting issues of structural insensitivity and the limitations of overreliance on synthetic benchmarks. To address these challenges, we propose a reproducibility-centered benchmarking framework that prioritizes real-data-derived perturbations and mandates rigorous statistical validation of synthetic datasets before their use. This work provides critical insights and practical guidance for the microbiome research community, aiming to foster more reliable and ecologically meaningful microbial network inference in the absence of a true ground truth.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.05.663212v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A conserved long-range RNA interaction in SARS-CoV-2 recruits ADAR1 to enhance virus proliferation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yang, S. L., DeFalco, L., Wang, S., Wong, Y. H., Han, J., Mok, C. K., Tan, K. Y., Lim, S. Y., Zhao, Z., Zhang, Y., Lim, J. J. A., Xiang, J. S., Sobota, R., Wang, L.-f., Chu, J. J. H., Merits, A., Huber, R. G., Wan, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that mutations that disrupt long-range RNA-RNA pairing reduce viral fitness at later stages of its infection cycle, while compensatory mutations partially restore virus fitness.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Long-range RNA-RNA pairing impacts the genome structure and function of SARS-CoV-2 variants. To understand the structure and function relationships of different SARS-CoV-2 variants that have emerged during the COVID-19 pandemic, we performed high-throughput structure probing and modelling of the genomic structures of the wildtype (WT), Alpha, Beta, Delta and Omicron variants of SARS-CoV-2. We observed that genomes of SARS-CoV-2 variants are generally structurally conserved, and that single-nucleotide variations and interactions with RNA binding proteins can impact RNA structures across the viruses. Importantly, using proximity ligation sequencing, we identified many conserved ultra-long-range RNA-RNA interactions, including one that spans more than 17 kb in both the WT virus and the Omicron variant. We showed that mutations that disrupt this 17 kb long-range interaction reduce viral fitness at later stages of its infection cycle, while compensatory mutations partially restore virus fitness. Additionally, we showed that this ultra-long-range RNA-RNA interaction structure binds directly to ADAR1 to alter the RNA editing levels on the viral genome. These studies deepen our understanding of RNA structures in the SARS-CoV-2 genome and their ability to interact with host factors to facilitate virus infectivity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.07.663401v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CAPTAIN: A multimodal foundation model pretrained on co-assayed single-cell RNA and protein</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ji, B.-Y., Hu, T.-T., Wang, J.-W., Liu, M.-M., Xu, L.-W., Zhang, Q.-H., Zhong, S.-Y., Qiao, L.-B., Zhang, Y., Peng, S.-L., Yu, F.-L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A multimodal single-cell foundation model for cellular understanding and virtual cell construction .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Proteins act as the terminal effectors of cellular function, encoding the phenotypic consequences of genomic and transcriptomic programs. Although transcriptomic profiles serve as accessible proxies, they remain incomplete surrogates for the proteomic landscape that ultimately defines cellular phenotypes. Current single-cell foundation models, however, are trained exclusively on transcriptomes, resulting in biased and partial characterizations of cellular states. To address this limitation, we introduce CAPTAIN, a multimodal foundational model pretrained on over four million single cells with concurrently measured transcriptomes and a curated repertoire of 382 surface proteins across diverse human and mouse tissues. Our results show that CAPTAIN learns unified multimodal representations by modeling cross-modality dependencies and capturing the diversity of cellular states across complex biological contexts. CAPTAIN generalizes robustly across both fine-tuning and zero-shot settings, excelling in core downstream tasks such as protein imputation and expansion, cell type annotation, and batch harmonization. Beyond improved accuracy in multi-omics integration, CAPTAIN uncovers previously inaccessible mechanisms of protein-driven intercellular dynamics, including immune interaction patterns linked to COVID-19 severity. CAPTAIN establishes a new paradigm for multimodal single-cell modeling, laying the foundation for comprehensive cellular understanding and virtual cell construction.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.07.663366v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">BAGEL: Protein Engineering via Exploration of an Energy Landscape</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lala, J., Al-Saffar, A., Angioletti-Uberti, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce BAGEL, a modular, open-source framework for programmable protein engineering, enabling flexible exploration of sequence space through model-agnostic and gradient-free exploration of an energy landscape.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Despite recent breakthroughs in deep learning methods for protein design, existing computational pipelines remain rigid, highly specific, and ill-suited for tasks requiring non-differentiable or multi-objective design goals. In this report, we introduce BAGEL, a modular, open-source framework for programmable protein engineering, enabling flexible exploration of sequence space through model-agnostic and gradient-free exploration of an energy landscape. BAGEL formalizes protein design as the sampling of an energy function, either to optimize (find a global optimum) or to explore a basin of interest (generate diverse candidates). This energy function is composed of user-defined terms capturing geometric constraints, sequence embedding similarities, or structural confidence metrics. BAGEL also natively supports multi-state optimization and advanced Monte Carlo techniques, providing researchers with a flexible alternative to fixed-backbone and inverse-folding paradigms common in current design workflows. Moreover, the package seamlessly integrates a wide range of publicly available deep learning protein models, allowing users to rapidly take full advantage of any future improvements in model accuracy and speed. We illustrate the versatility of BAGEL on four archetypal applications: designing de novo peptide binders, targeting intrinsically disordered epitopes, selectively binding to species-specific variants, and generating enzyme variants with conserved catalytic sites. By offering a modular, easy-to-use platform to define custom protein design objectives and optimization strategies, BAGEL aims to speed up the design of new proteins. Our goal with its release is to democratize protein design, abstracting the process as much as possible from technical implementation details and thereby making it more accessible to the broader scientific community, unlocking untapped potential for innovation in biotechnology and therapeutics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.05.663138v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A novel subtyping method for TNBC with implications for prognosis and therapy</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mesrizadeh, Z., Mukund, K., Zabaleta, J., Valle, L. D., Tomsic, J., Neuhausen, S. L., Ding, Y. C., Seewaldt, V., Ochoa, A., Miele, L., Subramaniam, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified three major subtypes and three distinct groups exhibiting unique cell-type composition and mechanisms of triple-negative breast cancer.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The biological heterogeneity of triple-negative breast cancer (TNBC) poses significant challenges for diagnosis, prognosis, and treatment. While prior TNBC subtype classifications exist, they are not widely used clinically. Here, we aimed to subtype TNBC based on transcriptomic profiles using cell type and state heterogeneity in tumor tissue from 250 pre-treatment women (127 African-American and 123 European-American). We identified three major subtypes and three distinct groups exhibiting unique cell-type composition and mechanisms: Subtype-1 immune signaling/T-cell response; Subtype-2 pro-fibrotic and immune desert; Subtype-3 fatty acid and nuclear receptor signaling. Subtype-1 showed potential responsiveness to immunotherapy, while Subtypes-2 and 3 suggested alternative therapeutic targets. In Subtype-3, which contained a patient group with high ESR1, (but not high ER protein expression) we identified putative mutations in the gene that are unique to these patients. This framework provides a path toward personalized TNBC treatment and is accessible through a user-friendly RShiny application for clinical use.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.04.663242v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Decoding the peripheral transcriptomic and meta-genomic response to music in Autism Spectrum Disorder via saliva-based RNA sequencing</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cavenaghi, A., Mallah, N. E. Z., Navarro, L., Martinon-Torres, F., Gomez-Carballa, A., Salas, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that music exposure can influence both host gene expression and oral microbiota, potentially affecting neuroimmune processes via the microbiota-brain axis, and that music-induced transcriptional changes relevant to neurodevelopmental and immune processes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Saliva-based RNA sequencing (RNA-seq) poses technical challenges due to high bacterial content, RNA degradation, and sample heterogeneity. This study investigates the transcriptional effects of music exposure in individuals with autism spectrum disorder (ASD) using this non-invasive approach. To address saliva-specific limitations, we employed two complementary library preparation methods, Poly-A selection and Human-Enriched protocols, allowing us to maximize human transcript detection and ensure reproducibility. By merging them, we ensured reproducibility and captured both host and microbial signals. While each dataset individually revealed a limited number of differentially expressed genes (DEGs), their integration enhanced biological resolution. Among the consistently modulated genes were HERC6, TSPAN5, and REM2, pointing to music-induced transcriptional changes relevant to neurodevelopmental and immune processes. Functional enrichment analyses highlighted pathways involved in immune regulation, oxidative phosphorylation, and epithelial differentiation. These findings align with evidence of immune dysregulation, mitochondrial dysfunction, and altered cellular communication in ASD. Importantly, co-expression network analysis identified modules significantly correlated with music exposure. Notably, the AKNA module, previously associated with ASD risk, was downregulated and enriched for Ras-related GTPase signaling and immune pathways, suggesting that music may modulate intracellular signaling and inflammation. Conversely, upregulation of the UBE2D3 module pointed to activation of endoplasmic reticulum stress response mechanisms, a known contributor to ASD neurodevelopmental deficits. These results suggest that music engages specific stress-adaptive and immunomodulatory networks in buccal cells, potentially reflecting systemic effects. Our exploratory metagenomic analysis highlights 15 microbial species with consistent abundance shifts across both methods. Notably, Acidipropionibacterium acidipropionici and Propionibacterium freudenreichii, associated with propionic acid production, emerged as music-responsive taxa. Elevated propionic acid has been implicated in ASD-like behaviors and neuroinflammation, suggesting a microbiota-mediated pathway. Music may influence both host gene expression and oral microbiota, potentially affecting neuroimmune processes via the microbiota-brain axis. Although exploratory, the results support the feasibility of using saliva for integrated molecular profiling in ASD.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.04.663204v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genome sequence of Talaromyces trachyspermus, a biocontrol fungus isolated from broomrape</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hemmati, R., Dolatabadian, A., Saeedi, S., Batley, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We analysed the genome of a biocontrol isolate of Talaromyces trachyspermus for the first time to investigate its secondary metabolites and cell wall-degrading enzymes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Applying antimicrobial compounds derived from microorganisms for plant disease management is one of the objectives of sustainable agriculture. The genus Talaromyces is known for its species' ability to produce a diverse group of antimicrobial compounds. For example, T. trachyspermus has been reported to produce secondary metabolites, cell wall-degrading enzymes, and plant growth-promoting factors. Identification of novel promising metabolites and enzymes from T. trachyspermus is still in its infancy. Also, there is a lack of information about the genomic resources for its secondary metabolites and hydrolytic enzymes. Therefore, this study aimed to analyse the genome of a biocontrol isolate of this species to investigate its biocontrol mechanisms at the genomic level, focusing on secondary metabolites and cell wall degrading enzymes. The whole genome of T. trachyspermus isolate IRAN 3054C, obtained from necrotic Orobanch ramosa stems in Iran with biocontrol ability, was sequenced using the Illumina platform. We performed both de novo and resequencing analyses of the genome, obtaining a 31.3 Mb assembly. The abundance of protein groups associated with biocontrol activities was assessed in the studied genome. Fungismash was used to detect and annotate secondary metabolites. The analysis revealed the presence of several secondary metabolite biosynthesis gene clusters (BGCs), with a high frequency of polyketide synthases (T1PKS) and nonribosomal peptide synthetases (NRPS), which are known to produce bioactive compounds with antimicrobial properties. Among the identified secondary metabolites, Fusarin, YWA1, Dimethylcoprogen, and Squalestatin S1 exhibited the highest similarity to known compounds. Furthermore, sequences similar to Phyllostictine A/B and Cornexistin indicate potential herbicidal properties. The genome also had domains for enzymes involved in phosphate solubilisation, siderophore production, and fungal cell wall degradation, which are essential for biocontrol and plant growth promotion. Our findings highlight the genomic richness of T. trachyspermus IRAN 3054C for biocontrol. Further metabolomics studies are needed to validate the actual production of these secondary metabolites and explore their functional roles in biocontrol.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.07.663553v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">mRNABench: A curated benchmark for mature mRNA property and function prediction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shi, R., Dalal, T., Fradkin, P., Koyyalagunta, D., Chhabria, S., Jung, A., Tam, C. L., Ceyhan, D., Lin, J., Laverty, K. U., Baali, I., Wang, B., Morris, Q.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a comprehensive benchmarking suite for mature mRNA biology that evaluates the representational quality of mature mRNA embeddings from self-supervised nucleotide foundation models.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Messenger RNA (mRNA) is central in gene expression, and its half-life, localization, and translation efficiency drive phenotypic diversity in eukaryotic cells. While supervised learning has widely been used to study the mRNA regulatory code, self-supervised foundation models support a wider range of transfer learning tasks. However, the dearth and homogeneity of standardized benchmarks limit efforts to pinpoint the strengths of various models. Here, we present mRNABench, a comprehensive benchmarking suite for mature mRNA biology that evaluates the representational quality of mature mRNA embeddings from self-supervised nucleotide foundation models. We curate ten datasets and 59 prediction tasks that broadly capture salient properties of mature mRNA, and assess the performance of 18 families of nucleotide foundation models for a total of 135K experiments. Using these experiments, we study parameter scaling, compositional generalization from learned biological features, and correlations between sequence compressibility and performance. We identify synergies between two self-supervised learning objectives, and pre-train a new Mamba-based model that achieves state-of-the-art performance using 700x fewer parameters. mRNABench can be found at: https://github.com/morrislab/mRNABench.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.05.662870v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Decoding functional genes across species with annotation-independent machine learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xie, Y., Li, L., Zhu, G., Lu, P., Ma, J., Tao, W., Tang, J., Wang, C., Han, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a computational framework that integrates cross-species comparative genomics and machine learning to identify functional genes for shared phenotypes, independent of prior annotation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Dissecting the genetic basis of complex traits across species is a challenge for traditional low-throughput, species-specific methods. To overcome these limitations, we introduce a computational framework that integrates cross-species comparative genomics and machine learning to identify functional genes for shared phenotypes, independent of prior annotation. Our method links orthologous gene group (OG) profiles to specific traits, enabling the powerful discovery of functional genes across diverse evolutionary clades. For plant-arbuscular mycorrhizal symbiosis, a notoriously difficult system, our model pinpointed a regulatory network by identifying 27 core plant-AM symbiosis genes among its top 50 candidate OGs, including the critical receptor SYMRK. The approach also proved highly effective for identifying functional genes related to the tested vertebrate skeletal development and multiple bacterial traits. Most notably, for bacterial motility, our model not only identified 63 known motility genes from the top 100 candidate OGs (of which 78 are present in Escherichia coli) but also guided the experimental validation of three novel essential genes. This annotation-independent strategy represents a paradigm shift in functional genomics, offering a scalable and universal engine to decode the genetic architecture of complex traits and illuminate the vast 'functional dark matter' across the tree of life.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.04.662730v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An effector from the potato late blight pathogen bridges ENTH-domain protein TOL9a to an activated helper NLR to suppress immunity</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Madhuprakash, J., Toghani, A., Pai, H., Harvey, M., Bentham, A. R., Seager, B. A., Yuen, E. L. H., De la Concepcion, J. C., Lawson, D. M., Stevenson, C. E. M., Vergara-Cruces, A., Derevnina, L., Bozkurt, T. O., Banfield, M. J., Kamoun, S., Contreras, M. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that an effector from the potato late blight pathogen Phytophthora infestans bridges the host protein NbTOL9a, a putative member of the host ESCRT pathway, to a helper NLR to suppress immunity.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Pathogens counteract central nodes of NLR immune receptor networks to suppress immunity. However, the mechanisms by which pathogens hijack helper NLR pathways are poorly understood. Here, we show that an effector from the potato late blight pathogen Phytophthora infestans bridges the host protein NbTOL9a, a putative member of the host ESCRT pathway, to a helper NLR to suppress immunity. In this work, we solved the crystal structure of the RXLR-LWY effector AVRcap1b in complex with the ENTH domain of NbTOL9a. The structure revealed that unlike other RXLR-LWY effectors, AVRcap1b has a novel L-shaped fold and defines a new structural family of effectors in the Phytophthora genus. Moreover, we defined the AVRcap1b/NbTOL9a binding interface and designed effector mutants that don't bind NbTOL9a, impairing immune suppression. This indicates that ENTH binding is required for full virulence activity of this effector. Lastly, we show that AVRcap1b associates specifically with activated NbNRC2 independently of NbTOL9a binding. This suggests that the effector functions as a bridge that interconnects NbNRC2 with the NbTOL9a pathway. These results illustrate an unprecedented pathogen mechanism to hijack helper NLR pathways and suppress immunity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.06.663370v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A shotgun approach for highly multiplexed mammalian metabolic engineering</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Trolle, J., Sessa, S., Rodrick, T., Jones, D. R., Fenyo, D., Pinglay, S., Boeke, J. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed Shotgun Genetic Engineering (SGE), a scalable approach that exploits the ease of delivering many barcoded small constructs - rather than a single large one - into mammalian cells to engineer complex biosynthetic traits in mammalian systems.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mammalian metabolic engineering is critical to advancing basic biology, bioproduction, and cell therapy. However, as pathway complexity increases, so does the size of both the combinatorial design space and the DNA constructs required, rendering unbiased screens intractable. To address this, we developed Shotgun Genetic Engineering (SGE), a scalable approach that exploits the ease of delivering many barcoded small constructs - rather than a single large one - into mammalian cells. This allows each cell to serve as an independent experiment, carrying a unique synthetic metabolic pathway that explores combinations of gene content, stoichiometry, and organellar localization. Functional pathways are identified by sequencing barcodes from cells exhibiting the desired phenotype. Using SGE, we screened millions of pathway combinations to engineer essential amino acid biosynthesis in two mammalian cell lines (CHO and Jurkat), achieving near-wild-type growth in valine-free medium and, for the first time, enabling isoleucine prototrophy in CHO cells. Successful solutions favored mitochondrial localization and required integration of 23-52 kb of synthetic DNA - lengths that are impractical to screen by conventional methods. The resulting datasets are compatible with machine learning frameworks, positioning SGE as a powerful platform for decoding and engineering complex biosynthetic traits in mammalian systems.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.08.663766v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genetically encoded control of in vitro transcription-translation coupled DNA replication</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Barthel, S., Hoffmann-Becking, M., Karimov, I. G., Erb, T. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We control DNA-encoded information within in vitro transcription-translation coupled DNA replication using TetR and a modified PURE (Protein synthesis Using Recombinant Elements) IVTT system.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The bottom-up reconstruction of cellular functions has gained increasing attention for studying biological complexity, and for developing advanced biotechnological tools, including synthetic cells. A fundamental challenge is the ability to control and replicate DNA-encoded information within basic in vitro transcription-translation (IVTT) systems. Here, we constructed a transcription-translation coupled DNA replication (TTcDR) system that is based on a modified PURE (Protein synthesis Using Recombinant Elements) IVTT system and {Phi}29 DNA polymerase, which is controlled by external signals. To this end, we first established and characterized a PUREfrex 1.0-based TTcDR system. We then constructed and optimized TetR-based control of TTcDR activity, either by DNA-encoding TetR or by supplying purified TetR. Our final DNA-encoded TetR circuit allows ~1000-fold DNA replication, ~100-fold repression, and ~4-fold induction with anhydrotetracycline. Our results demonstrate the potential and challenges of controlling in vitro DNA replication, for example for the evolution of in vitro systems.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.08.663768v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Expression of nano-engineered RNA organelles in bacteria</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ng, B., Fan, C., Dordevic, M., Knirsch, A., Malouf, L., Fabrini, G., Nuccio, S. P., Rubio-Sanchez, R. M., Christie, G., Takinoue, M., Cicuta, P., Di Michele, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We design and express synthetic biomolecular condensates from branched RNA motifs, and by embedding a protein-binding aptamer we achieve selective protein recruitment.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Designing synthetic biomolecular condensates, or membrane-less organelles, offers insights on the functions of their natural counterparts, and is equally valuable for cellular and metabolic engineering. Choosing E. coli for its biotechnological relevance, we deploy RNA nanotechnology to design and express non-natural membrane-less organelles in vivo. The designer condensates assemble co-transcriptionally from branched RNA motifs interacting via base-pairing. Exploiting binding selectivity we express orthogonal, non-mixing condensates, and by embedding a protein-binding aptamer we achieve selective protein recruitment. Condensates can be made to dissolve and re-assemble upon thermal cycling, thereby reversibly releasing and re-capturing protein clients. The synthetic organelles are expressed robustly across the cell population and remain stable despite enzymatic RNA processing. Compared with existing solutions based on peptide building blocks or repetitive RNA sequences, these nanostructured RNA motifs enable algorithmic control over interactions, affinity for clients, and condensate microstructure, opening new directions in synthetic biology and biotechnology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.08.663582v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">GAME: Genomic API for Model Evaluation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Luthra, I., Priyadarshi, S., Guo, R., Mahieu, L., Kempynck, N., Dooley, D., Penzar, D., Vorontsov, I., Sheng, Y., Tu, X., Klie, A., Drusinsky, S., Floren, A., Armand, E., Alasoo, K., Seelig, G., Tewhey, R., Koo, P., Agarwal, V., Gosai, S., Pinello, L., White, M. A., Lal, A., Zeitlinger, J., Pollard, K. S., Libbrecht, M., Carter, H., Mostafavi, S., Kulakovskiy, I., Hsiao, W., Aerts, S., Zhou, J., de Boer, C. G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We have created a system for large-scale, community-led standardized model benchmarking on diverse evaluation tasks.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The rapid expansion of genomics datasets and the application of machine learning has produced sequence-to-activity genomics models with ever-expanding capabilities. However, benchmarking these models on practical applications has been challenging because models are evaluated in heterogenous and ad hoc ways. To address this, we have created GAME, a system for large-scale, community-led standardized model benchmarking on diverse evaluation tasks. GAME uses Application Programming Interfaces (APIs) to communicate between containerized pre-trained models and benchmarking tasks, enabling uniform model evaluations and cross-platform compatibility. We also developed a Matcher module powered by a large language model (LLM) to automate ambiguous task alignment between prediction requests and tasks models can perform. The community can easily contribute GAME modules, leading to an ever-expanding and evolving set of models and benchmarks. GAME will accelerate genomics research by illuminating the best models for a given task, motivating novel benchmarks, and providing a nuanced understanding of model abilities.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.04.663250v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Nanoparticle-Protein Corona Boosted Cancer Diagnosis with Proteomic Transfer Learning.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Haoxiang Guo, Baichuan Jin, Zhenjie Zhu, Xin Dai, Mengjie Wang, Yueli Xie, Chenlu Xu, Zongping Wang, Yuan Liu, Weihong Tan</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> ProteoTransNet integrates serum and urine proteome databases with proteomic transfer learning, significantly enhancing the diagnostic accuracy through minimizing biases and errors caused by variations in proteomic data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Keeping pace with the rapid growth of proteomic data, the integration of multiproteomic data can improve biomarker identification and cancer diagnosis. However, the data integration needs to overcome substantial challenges owing to considerable variability among diverse data set sources and the extensive range of protein expression levels. In this study, with serum and urine from the same individuals, we established two in-depth paired proteome databases, including 956 serum proteins and 4730 urine proteins. To integrate multiproteomic data, we developed a proteomic-based transfer learning neural network (ProteoTransNet) to enhance the accuracy of bladder cancer diagnosis and progression monitoring. Using random forest analysis on the integrated database, we selected two panels comprising the top 10 key proteins, achieving a diagnostic AUC of 0.996 and a stage classification AUC of 0.914. ProteoTransNet integrates serum and urine proteome databases with proteomic transfer learning, significantly enhancing the diagnostic accuracy through minimizing biases and errors caused by variations in proteomic data. Our study provides insights that transfer learning of sophisticated biological information may solve complicated biological problems in disease diagnosis, prognosis, and treatment.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40560746/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Label-free single-cell phenotyping to determine tumor cell heterogeneity in pancreatic cancer in real time.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Katja Wittenzellner, Manuel Lengl, Stefan R√∂hrl, Carlo Maurer, Christian Klenk, Aristeidis Papargyriou, Laura Schmidleitner, Nicole Kabella, Akul Shastri, David E Fresacher, Farid Harb, Nawal Hafez, Stefanie B√§rthel, Daniele Lucarelli, Carmen Escorial-Iriarte, Felix Orben, Rupert √ñllinger, Ellen Emken, Lisa Fricke, Joanna Madej, Patrick Wustrow, I Ekin Demir, Helmut Friess, Tobias Lahmer, Roland M Schmid, Roland Rad, G√ºnter Schneider, Bernhard Kuster, Dieter Saur, Oliver Hayden, Klaus Diepold, Maximilian Reichert</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We have developed a single-cell phenotypic model for intratumoral heterogeneity in pancreatic ductal adenocarcinoma that can be used to assess and quantify ITH in the treatment response of the disease.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Resistance to chemotherapy of pancreatic ductal adenocarcinoma (PDAC) is largely driven by intratumoral heterogeneity (ITH) due to tumor cell plasticity and clonal diversity. To develop alternative strategies to overcome this defined mechanism of resistance, tools to monitor and quantify ITH in a rapid and scalable fashion are needed urgently. Here, we employed label-free digital holographic microscopy (DHM) to characterize ITH in PDAC. We established a robust experimental and machine learning analysis pipeline to perform single-cell phenotyping based on DHM-derived phase images of PDAC cells in suspension. Importantly, we were able to detect dynamic changes in tumor cell differentiation and heterogeneity of distinct PDAC subtypes upon induction of epithelial-mesenchymal transition and under treatment-imposed pressure in murine and patient-derived model systems. This platform allowed us to assess phenotypic ITH in PDAC on a single-cell level in real time. Implementing this technology into the clinical workflow has the potential to fundamentally increase our understanding of tumor heterogeneity during evolution and treatment response.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40424076/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

