<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Saturday, 30 August 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">IQUP identifies quantitatively unreliable spectra with machine learning for isobaric labeling-based proteomics.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yi-Yun Chou, Jing-Chen Yang, Tun-Chu Tsai, Ching-Tai Chen</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mass spectrometry‑based proteomics using isobaric labeling technology has become popular for proteomic quantitation. Existing approaches rely on the mechanism of target-decoy search and false discovery rate control to examine whether a peptide-spectrum match (PSM) is utilized for quantitation. However, some PSMs passing the examination may still exhibit high quantitation errors, which can deteriorate the overall quantitation accuracy. We present IQUP, a machine learning-based method to identify quantitatively unreliable PSMs, termed QUPs. PSMs were characterized by 16 spectral and distance-based features for machine learning. Independent test results reveal that the best-performing models for the three datasets achieve accuracies of 0.883-0.966, AUCs of 0.924-0.963, and MCCs of 0.596-0.691. Notably, the distributions of relative errors for QUPs and quantitatively reliable PSMs (QRPs) exhibit significant differences. By using only the predicted QRPs for peptide-level quantitation, the proportions of peptides with larger relative errors decrease significantly, with a range between 15.3 and 83.3% for the three datasets; in the meantime, the proportions of peptides with smaller relative errors increase by 3.1-25.5%. Our experimental results demonstrate that IQUP provides robust performance and strong generalizability across multiple datasets and has great potential in improving proteomic quantitation accuracy at PSM and peptide levels for isobaric labeling experiments.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40883461/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Deep learning in chromatin organization: from super-resolution microscopy to clinical applications.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mikhail Rotkevich, Carlotta Viana, Maria Victoria Neguembor, Maria Pia Cosma</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The 3D organization of the genome plays a critical role in regulating gene expression, maintaining cellular identity, and mediating responses to environmental cues. Advances in super-resolution microscopy and genomic technologies have enabled unprecedented insights into chromatin architecture at nanoscale resolution. However, the complexity and volume of data generated by these techniques necessitate innovative computational strategies for effective analysis and interpretation. In this review, we explore the transformative role of deep learning in the analysis of 3D genome organization, highlighting how deep learning models are being leveraged to enhance image reconstruction, segmentation, and dynamic tracking in chromatin research. We provide an overview of deep learning-enhanced methodologies that significantly improve spatial and temporal resolution of images, with a special focus on single-molecule localization microscopy. Furthermore, we discuss deep learning's contribution to segmentation accuracy, and its application in single-particle tracking for dissecting chromatin dynamics at the single-cell level. These advances are complemented by frameworks that enable multimodal integration and interpretability, pushing the boundaries of chromatin biology into clinical diagnostics and personalized medicine. Finally, we discuss emerging clinical applications where deep learning models, based on chromatin imaging, aid in disease stratification, drug response prediction, and early cancer detection. We also address the challenges of data sparsity, model interpretability and propose future directions to decode genome function with higher precision and impact.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40879746/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Prediction of virulence factors in bacterial hypothetical proteins.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Anchita Prasad, Vijayaraghava Seshadri Sundararajan, JayaramanValadi, Vinod Kumar Nigam, Prashanth Suravajhala</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> A large number of genes in bacterial genomes encode hypothetical proteins (HPs) with unknown functions that require annotation. These HPs have recently been linked to pathogenicity due to potential virulence factors. Understanding the key elements influencing their virulence is a crucial goal. In our study, we analyzed protein domain families from the Pfam database in select Gram-positive and Gram-negative bacteria to identify virulent factors using a scoring system. We employed computational tools such as VICMpred and VirulentPred to identify virulent domains in different bacteria and used regression and Support Vector Machine (SVM) with Weka to find the best candidates. Genotypes with higher adherence suggest that diverse adhesin expression contributes to varying virulence potential, making strains with higher adhesin expression more aggressive. We believe our methods for predicting bacterial protein functions can aid in developing drugs and vaccines by enabling the prediction of virulence proteins.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40879738/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SAGA (Simplified Association Genomewide Analyses): a user-friendly Pipeline to Democratize Genome-Wide Association Studies</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cieza, B., Pandey, N., Ruhela, V., Ali, S., tosto, g.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genome-wide association studies (GWAS) have enabled clinicians and researchers to identify genetic variants linked to complex traits and diseases(1). However, conducting GWAS remains technically challenging without bioinformatics expertise due to required data preprocessing, software installation, and analysis scripting (2,3). SAGA is a BASH-based, open-source, fully automated pipeline that integrates three widely adopted tools -PLINK, GMMAT and SAIGE- for accessible, robust, and reproducible GWAS. After installation, users only provide standard genotype and phenotype files. The pipeline automates preprocessing, association testing, and visualization, outputting summary statistics, Manhattan plots, and quantile-quantile plot. SAGA enables robust GWAS for users with no scripting experience, expanding access to complex genetic analyses.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.25.672146v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SEA CDM: Study-Experiment-Assay Common Data Model and Databases for Cross-Domain Data Integration and Analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Huffman, A., Yeh, F.-Y., Hur, J., Zheng, J., Masci, A. M., Tao, C., Athey, B. D., He, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> With the increasing volume of biomedical experimental data, standardizing, sharing, and integrating heterogeneous experimental data across domains has become a major challenge. To address this challenge, we have developed an ontology-supported Study-Experiment-Assay (SEA) common data model (CDM), which includes 10 core and 3 auxiliary classes based on object-oriented modeling. SEA CDM uses interoperable ontologies for data standardization and knowledge inference. Building on the SEA CDM, we developed the Ontology-based SEA Network (OSEAN) relational database and knowledge graph, along with a set of ETL (Extract, Transform, Load) and query tools, and further applied them to represent 1,278 immune studies with over two million samples from three resources: VIGET, ImmPort, and CELLxGENE. Using simple, robust queries and analyses, our research identified multiple scientific insights into sex-specific immune responses, such as neutrophil degranulation and TNF binding to physiological receptors, following live attenuated and trivalent inactivated influenza vaccination. The novel SEA CDM system lays a foundation for establishing an integrative biodata ecosystem across biological and biomedical domains.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.26.671804v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">QTL mapping, breeding, and debugging Saccharomyces cerevisiae strains through Reiterated Mass Selection and backcrosSing (ReMaSSing)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> de Bem, L. S., Barreto, J. A., de Souza, D. T., Jordan, D. O., de Souza, S. F. D., Marin, D. C., Da Silva, J. J., Brienzo, M., dos Santos, L. V., Rabelo, S. C., Rodrigues Alves Margarido, G. V. d., Gross, J., Jacobus, A. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;">Background Producing second-generation ethanol from lignocellulosic hydrolysates (LCHs) poses significant challenges for Saccharomyces cerevisiae due to the presence of fermentation inhibitors. Quantitative trait loci (QTL) mapping of stress-tolerant S. cerevisiae strains is important for identifying adaptive alleles that can enhance yeast fermentation of LCHs. However, the QTL mapping process is labor-intensive, requiring the screening of numerous recombinants and repeated crossings to improve mapping resolution. Results We developed Reiterated Mass Selection and backcrosSing (ReMaSSing) to facilitate the identification of adaptive alleles through QTL mapping and to enhance LCH tolerance in yeast strains. ReMaSSing was applied to populations obtained by crossing the stress-resistant yeast PE-2_H4 with the laboratory strain S288C. Using alternative protocols, we selected haploid or diploid populations with dominant markers, enriching millions of segregants carrying adaptive alleles by propagating them in standard or LCH-supplemented media. The enriched pools were then bulk backcrossed with S288C, and germination of millions of spores generated new recombinant populations for subsequent selection cycles. After five rounds of ReMaSSing, whole-genome sequencing and QTL mapping identified key alleles associated with LCH tolerance, linked to VPS70, CAT5, GCY1, UBP2, MKT1/SAL1, HAP1, and PHO84, which influence growth and mitochondrial function in S288C. Mutations in IRA1 and HTA1, unique to our S288C strain, were also mapped, highlighting ReMaSSings ability to detect and correct deleterious alleles (bugs). Allele swapping and competition assays confirmed that the identified QTL improved LCH tolerance and growth, with strains combining adaptive alleles performing over 20% better than the parental S288C. Finally, applying ReMaSSing to breed an LCH-tolerant yeast with a xylose-consuming strain produced recombinants with improved fermentation of xylose-enriched LCH. Conclusion ReMaSSing offers a practical protocol for generating QTL mapping populations to identify adaptive alleles in tolerant strains and correct genetic defects in inferior ones. Notably, recombinant populations and clones derived from ReMaSSing outperformed both parental strains in LCH tolerance and growth. Furthermore, we applied ReMaSSing to breed strains with enhanced LCH tolerance, efficient xylose catabolism, and robust ethanol production. Together, these results demonstrate that ReMaSSing is a powerful tool for engineering industrial yeast strains that integrate desirable traits from multiple parental backgrounds.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.25.672184v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Common tissue-specific expressions and regulatory mechanisms of c-KIT isoforms with and without GNNK and GNSK sequences across five mammals</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Goto, R., Sakamoto, N., Awazu, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> c-KIT is a transmembrane receptor tyrosine kinase involved in various signaling pathways. Alternative pre-mRNA splicing of KIT results in isoforms that differ in the presence or absence of four amino acid sequences in the extracellular juxtamembrane region, such as, isoforms with and without the GNNK sequence (GNNK+ and GNNK-, respectively) in humans and mice, and those with and without GNSK (GNSK+ and GNSK-, respectively) in domestic dogs, cats, and sheep. These isoforms have been extensively studied as disease-associated splice variants with differing kinase activities. However, the expression patterns and regulatory mechanisms of each isoform in various animal species remain poorly understood. In the present study, a comprehensive expression analysis was conducted using tissue-wide transcriptome data from humans, mice, domestic dogs, domestic cats, and domestic sheep. We found that the expression ratio of c-KIT isoforms differs across tissues, and such features are conserved across animal species: GNNK+ and GNSK+ isoforms have high expression ratios in the central nervous system, while GNNK- and GNSK- predominate in other tissues. Furthermore, the RNA-binding proteins NOVA1/2 were involved in regulating such tissue-specific isoform expressions through alternative 5' splice donor site selections.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.25.672062v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The multiscale distribution of radiation-induced DNA damage and its impact on local genome structure</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Li, H., Xue, T., McCord, R. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The three-dimensional genome structure is critical for the regulation of gene expression and repair of DNA damage. While previous work has characterized genome-wide sites of DNA damage caused by etoposide or nucleases, the distribution of double-strand breaks (DSBs) caused by external radiation and how these interact with the 3D genome organization is less well understood. Here, we measure the genomic landscape of radiation-induced DNA damage using END-seq in fibroblasts and lymphoblasts after exposure to 5 Gy X-rays. We identify frequently broken regions and investigate the 3D genome properties around these breaks with Hi-C data. We observe that the distribution of robust breaks correlates with transcriptional and chromatin features of the genome. Transcriptionally active and decondensed regions, such as chromosome 19, the A compartment, and topologically associating domain (TAD) boundaries, show pronounced break probability. We also find evidence of DSB-induced loop formation in the vicinity of frequent radiation-induced breaks. Our data reveal that pre-existing 3D genome architecture influences the distribution of radiation-induced DSBs and that these breaks reshape local chromatin landscapes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.25.672161v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Partitioning of nighttime transpiration and stem water refilling using VPD and dendrometer data: insights into baselining and nighttime sap flux interpretation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, M., Rasanen, M., Holtta, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The baseline for thermal dissipation probes is a reference for simultaneous cessation of both nighttime transpiration and refilling of trees internal water stores. Finding this moment is challenging as the baseline is dynamically changing, and the complete sap flux cessation may not even occur. We proposed a novel framework using VPD and dendrometer measurements to partition transpiration and refilling from nighttime sap flux density, and introduced a night-by-night baselining scheme that does not rely on identifying the zero-flow moment. One conventional baselining method using the maximum probe voltage (or temperature) difference substantially underestimated sap flux density by 204.56% for nighttime, 28.30% for daytime, and 45.20% for full-day flux in P. sylvestris, and by 344.77%, 45.68%, and 54.76%, respectively, in A. glutinosa. Baseline calibrations using commonly available variables are thus provided. Evaluation through gas exchange and evapotranspiration data confirmed the accuracy of partitioning between nighttime transpiration and refilling, demonstrating that nighttime transpiration can drive measurable nighttime evapotranspiration responses. Nightly variations in water allocation within stem tissue add complexity to stem swelling. Seasonally, sap flux density showed an initial increase in both transpiration and refilling, followed by a decline, with transpiration dominating until mid-summer before refilling gradually took precedence.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.25.672109v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Antibacterial and Antiproliferative Properties of Portulaca oleracea Essential Oil and its Targeted Delivery to MCF-7 Cells via Spermine-PLA-PEG-FA Nanocapsules</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Asghari Zakaria, R., Noruzpuor, M., Bourang, S., Yaghoubi, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Portulaca oleracea essential oil is plant-derived product with documented antimicrobial and antiproliferative activities. Its clinical use is limited by poor water solubility, chemical instability, and lack of targeting. Encapsulation in polymeric nanocarriers can enhance solubility, stability, and selective delivery. This study investigated the antibacterial activity of P. oleracea essential oil against Staphylococcus aureus and Escherichia coli, and its antiproliferative effects on human breast cancer (MCF-7) cells using folic acid-modified spermine-polylactic acid-polyethylene glycol (Spermine-PLA-PEG-FA) nanocapsules. Gas chromatography mass spectrometry (GC MS) identified major components such as [alpha] pinene, limonene, and phytol. The essential oil and quercetin were co-encapsulated in the nanocapsules. Characterization was conducted by proton nuclear magnetic resonance spectroscopy (1H-NMR), Fourier transform infrared spectroscopy (FTIR), thermogravimetric analysis (TGA), differential thermogravimetric analysis (DTG), and transmission electron microscopy (TEM). Antibacterial properties were assessed using disk diffusion, minimum inhibitory concentration (MIC), and minimum bactericidal concentration (MBC) assays. Antioxidant capacity was measured via 2,2 diphenyl 1 picrylhydrazyl (DPPH) assay. Cytotoxicity was evaluated by 3 (4,5 dimethylthiazol 2 yl) 2,5 diphenyltetrazolium bromide (MTT) assay and flow cytometry. Nanocapsules had spherical shape, size of 115 to 237 nm, and pH-responsive release. Quercetin loaded nanocapsules induced apoptosis in MCF7 cells with an IC50 of 11.21 mg/microlitre. This delivery system improved the antibacterial and anticancer efficacy of P. oleracea essential oil.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.25.672136v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multiple geographic breakdown events of the Rpv1/Rpv3.1 pyramided resistance in grapevine by Plasmopara viticola.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pelissier, R., Delmotte, F., Deliere, L., Martinez, J. R., Marolleau, L., Mazet, I. D., Fabre, F., Miclot, A. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In 2024, three vineyard sites located within a radius of 40 km in southeastern France, each planted with Artaban, a grapevine variety carrying pyramided resistance (Rpv1 and Rpv3.1), exhibited severe downy mildew symptoms following climatic conditions highly conducive to disease development. Twenty-one Plasmopara viticola isolates were collected from these vineyards and, after monosporangium isolation, tested under controlled conditions on a panel of susceptible and resistant grapevine varieties to assess their virulence. Among the 21 tested field strains, 19 overcame Rpv3.1 mediated resistance, 10 overcame Rpv1 alone, and four genetically distinct strains from two separate vineyards were overcoming the pyramided Rpv1/Rpv3.1 combination. The genetic characterization of the strains suggests that this adaptation likely resulted from convergent evolution. To our knowledge, these represent the first documented cases in Europe of a breakdown in resistance conferred by Rpv1, whether deployed alone or in combination with Rpv3.1.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.25.672106v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Bending and Squeezing: Gradual Potentials Encode Mechanical Stimuli in Poplar</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tinturier, E., Badel, E., Leblanc-Fournier, N., Guena, G., Forterre, Y., JULIEN, J.-L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mechanical stimuli such as wind elicit rapid electrical signals in plants, yet the mechanisms driving these responses remain poorly understood. Here, we investigated the electrophysiological responses of young poplar trees to controlled stem bending. We identified a gradual potential (GP) distinct from classical action potentials, whose amplitude attenuation and propagation distance were strongly influenced by stimulus speed and intensity. Although maximal GP amplitude near the bending site remained stable across stimulation setups, slower and gentler flexions led to more rapid spatial decay and reduced signal propagation. Similar GP responses were observed following either stem bending or direct root pressurization, suggesting hydraulic-electrical coupling. These results suggest that mechanical stress induces a transient hydraulic pressure wave that triggers the GP. While the GP's attenuation and propagation dynamics resemble those of a diffusive pressure signal, key features--such as constant peak amplitude regardless of stimulus strength and progressive signal narrowing--point to a complex, non-linear transduction mechanism. Altogether, our findings reveal a finely tuned system where spatial and temporal dynamics of this non-action potential electrical signal encode detailed information about mechanical stimuli, potentially allowing plants to adaptively respond to fluctuating environmental forces like wind.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.25.672115v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">THE ROOT-KNOT NEMATODE EFFECTOR MJ-MSP18: A SWISS ARMY KNIFE FOR REPROGRAMMING PLANT IMMUNITY</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chen, Y., Maulana, M. I., Oklestkova, J., Siroka, J., Jonckheere, V., De Gernier, H., Bauters, L., Zhang, N., Strnad, M., Van Damme, P., Goossens, A., Gheysen, G., Stojilkovic, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Root-knot nematodes (Meloidogyne spp.) are obligatory plant root parasites whose effector proteins play a critical role in suppressing plant immunity. However, the effectors direct host targets and underlying molecular mechanisms remain poorly understood. Using TurboID-mediated proximity labeling in tomato (Solanum lycopersicum) hairy roots, we identified an interaction between the nematode effector Mj-MSP18 and the tomato BRASSINOSTEROID-SIGNALING KINASE 7 (Sl-BSK7). Yeast two-hybrid (Y2H) assays confirmed that this interaction is conserved in Arabidopsis thaliana and Nicotiana benthamiana. Additionally, yeast three-hybrid and luciferase complementation assays demonstrated that Mj-MSP18 disrupts the interaction between BSK7/8 and FLS2 in both yeast and in planta. Given that the BSK7/8-FLS2 interaction is essential for flg22-induced pattern-triggered immunity (PTI), this disruption likely accounts for the suppression of reactive oxygen species (ROS) production and callose deposition observed upon transient expression of Mj-MSP18 in flg22-treated N. benthamiana leaves. Correspondingly, Arabidopsis and tomato bsk7 mutants exhibited increased susceptibility to root-knot nematode infection. Furthermore, RNA-seq analysis of tomato hairy roots expressing Mj-MSP18 revealed extensive transcriptional reprogramming, including the downregulation of defence-related genes and hydrogen peroxide response pathways. In addition, Y2H screening identified Sl-MYB and Sl-MYC2 as additional interactors, linking Mj-MSP18 to phytohormone biosynthesis, particularly the brassinosteroid (BR) and salicylic acid (SA) pathways, as validated by targeted metabolite analysis. The conservation of Mj-MSP18 across Meloidogyne species suggests a broadly conserved mechanism for host immune suppression and phytohormone modulation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.28.672664v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Dual role of the receptor kinase FERONIA in regulating tissue mechanics and growth</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Muller, E., Ropitaux, M., Durambur, G., LAPLAUD, V., Gebhard, L., Lehner, A., Drevensek, S., Boudaoud, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> During morphogenesis, growing organisms adapt to internal and external mechanical stress. In plants, the receptor kinase FERONIA is involved in a broad range of responses to perturbations of the cell wall, but it is unclear which cues are sensed and how this sensing controls growth. Here, we addressed these questions in Marchantia vegetative propagules. We combined culture in microfluidic chips to quantify growth and mechanical properties of propagules, perturbations with osmotic treatments, characterisation of cell wall polysaccharides, and a mathematical model of cell wall expansion that incorporates sensing. We found that FERONIA independently regulates tissue mechanical properties and growth rates. We propose that the regulation of growth by the FERONIA pathway relies on both a positive feedback from elastic deformation of the wall and a negative feedback from wall expansion. Altogether, we expect our quantitative framework to be broadly relevant to the investigation of how mechanical cues guide development.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.28.672783v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Growth, dissolution and segregation of genetically encoded RNA droplets by ribozyme catalysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Giessler, F., Verstraeten, W., Abele, T., Maurer, S. J., Monari, L., Göpfrich, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Active droplets, membraneless compartments driven by internal chemical reactions, are compelling models for protocells and synthetic life. A central challenge is to program their dynamic behaviors using heritable genetic information, which would grant them the capacity to evolve. Here, we create transiently active RNA droplets by integrating sites for ribozyme catalysis directly into the sequence of self-assembling, four-arm RNA nanostars. To enable perfusion and observe the resulting dynamics over time, we develop a method for trapping individual droplets in hydrogel cages by targeted in situ photopolymerization. This enables us to quantify the sequence-programmable droplet dissolution and to control the degradation kinetics by choosing between fast (hammerhead) and slow (hairpin) ribozymes. Furthermore, we trigger the segregation of mixed droplet populations via the sequence-specific cleavage of a chimeric linker RNA. The droplet-encapsulated DNA templates code for the regrowth of new droplets, establishing the proof-of-principle for a minimal, genetically encoded cycle of dissolution and regrowth. By directly linking RNA sequence to droplet stability, composition, and life-cycle dynamics, our work provides a robust platform for engineering evolvable materials and advancing the bottom-up construction of synthetic cells.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.29.673008v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Demonstration of SLU7 as a new pan-cancer target</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rojo, C., Otero, A., Elizalde, M., Azkona, M., Barbero, R., Latasa, M. U., Uriarte, I., Gutierrez-Uzquiza, A., Alignani, D., Guembe, L., Lujambio, A., Pastor, F., Fernandez-Barrena, M. G., Avila, M. A., Arechederra, M., Berasain, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cancer treatment remains challenging due to heterogeneous responses to immunotherapy across patients and tumor types. Innovative strategies are required to overcome immune evasion. We have identified the splicing factor SLU7 as essential for the survival of cancer cells from diverse origins. SLU7 knockdown induces R-loop accumulation, transcription-dependent genomic instability, DNA damage, and replication catastrophe, together with aberrant splicing and inhibition of nonsense-mediated mRNA decay (NMD) and/or DNA methylation. These alterations lead to the expression of neoantigens, interferon B1, endogenous retroviruses, and cancer-testis antigens, which would enhance tumor immunogenicity. Therefore, we propose SLU7 targeting as a dual-action therapy, combining direct tumor suppression with immune activation. Using various murine cancer models, including orthotopic liver tumors, and multiple molecular strategies, such as inducible CRISPR/Cas9, systemic delivery of chimeric siSLU7-nucleolin aptamers (APTASLU), and intratumoral injection of siSLU7-loaded nanoparticles, we show that distinct siSLU7 sequences and delivery platforms effectively inhibit tumor growth. Furthermore, SLU7 silencing synergizes with immune checkpoint inhibitors, amplifying anti-tumor responses. Our in vivo data demonstrate that SLU7 is a promising, versatile target for diverse cancers. Its multimodal mechanism offers potential to overcome tumor heterogeneity, reverse immune tolerance, and enhance immunotherapy efficacy.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.25.672085v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Pan-Cancer Single-Cell Profiling Uncovers the Biological Characteristics of Cancer-Testis Genes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fu, C., Liu, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cancer-testis genes (CTGs), characterized by their restricted expression in testicular and neoplastic tissues, have emerged as pivotal modulators of tumor progression and immunogenicity. Here, leveraging single-cell and single-nucleus RNA sequencing (scRNA-seq and snRNA-seq) data from 828 tumor samples spanning 13 cancer types, we systematically identified 407 CTGs based on their expression level in malignant cells. Compared to previously reported CTG sets, our curated genes exhibited broader expression among tumor samples and were predominantly enriched on autosomes. These CTGs displayed pronounced co-expression patterns, with a subset consistently activated in proliferating malignant cells, whereas non-cell cycle CTGs were highly correlated with epigenetic regulatory pathways. Functional analyses pinpointed chromatin-modifying genes, including ARID4B and YEATS2, as putative upstream regulators, whose knockdown resulted in widespread CTG downregulation. Moreover, we demonstrated that high CTG-count constitutes a robust and scalable feature for malignant cell identification across diverse omics data, including scRNA-seq, single-cell ATAC sequencing, and spatial transcriptomics. By quantifying transcriptomic similarity to a high CTG-count reference population, we developed a streamlined classification framework that enables rapid and accurate annotation of malignant cells in tumor sc/snRNA-seq datasets across diverse cancer types. Clinically, CTG expression levels were associated with patient prognosis and immune microenvironment composition. In hepatocellular carcinoma, elevated CTG expression defined a stem-like, immune-suppressive malignant cell state enriched for regulatory T cells, linked to advanced disease and poor survival. Collectively, our study refines the understanding of CTGs in cancer biology and provides a valuable resource to facilitate future research and therapeutic targeting.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.25.672079v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CDK4/6 inhibition induces a DNA damage-independent senescence-associated secretory phenotype driven by delayed activation of NF-κB</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yeung, J. L.-H., Rendleman, J., Anderson, L., Pressler, M., Pagane, N., Duba, I., Hosuru, R., Tumenbayar, B.-I., Koff, A., Risca, V. I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cellular senescence protects against aberrant cancer proliferation by enforcing stable cell cycle arrest. The best-studied triggers of senescence involve DNA damage signaling, while DNA damage-independent senescence is less understood. In the cancer context, senescence has recently been shown to be induced by CDK4/6 inhibitors (CDK4/6i), a class of targeted therapies currently applied to breast cancer and sarcoma, among other tumors. The senescence-associated secretory phenotype (SASP) can exert either pro- or anti-tumorigenic effects depending on its composition, making it a critical determinant of therapy outcome. The dependence of the SASP on the senescence trigger has been a controversial question, which we investigate here, particularly the involvement of NF-{kappa}B signaling. Using a multi-omics approach, we profiled the temporal dynamics of transcriptional and epigenetic state rewiring during therapy-induced senescence in CDK4/6i-treated liposarcoma cells over 28 days; we compared this to the effects of the DNA damaging chemotherapeutic, doxorubicin. Doxorubicin triggers an early and robust activation of p53 and NF-{kappa}B, as expected. Surprisingly, a delayed NF-{kappa}B-driven SASP signature also emerges in CDK4/6i-induced senescence after a 10-day exposure, in the absence of additional DNA damage. We replicated this effect in breast cancer cells over similar time courses. Inhibition of NF-{kappa}B blunts pro-inflammatory SASP expression without reversing cell cycle arrest in both treatment contexts, showing that NF-{kappa}B is a conserved SASP regulator, regardless of trigger and cancer type. Instead, the nature of the trigger regulates differential dynamics of the NF-{kappa}B response and additional SASP gene signatures, including a CDK4/6i-specific gene module driven by TEAD factors. Overall, our 28-day multi-omics time course reveals previously unknown dynamics of the SASP and demonstrates that both common and trigger-specific components of therapy-induced SASP can be separately tuned and uncoupled from cell cycle arrest, with implications for the rational design of adjuvant therapies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.25.672139v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

