<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Thursday, 18 September 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identification of a novel CLPX variant in a mixed breed dog with anemia and spinocerebellar ataxia</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> de Cecco, B. S., Blake, J. M., Kim, N. J., Coffey, M. C., Johnston, A., Miller, A. D., Ekenstedt, K. J., Lee, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A homozygous 4-base-pair frameshift deletion in caseinolytic mitochondrial matrix peptidase chaperone subunit X (CLPX) is associated with spinocerebellar ataxia in a dog.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Spinocerebellar ataxia (SCA) or hereditary ataxia is a progressive neurodegenerative disorder primarily manifesting as cerebellar or spinocerebellar dysfunction, resulting in the loss of motor control and voluntary muscle coordination. SCAs are typically inherited conditions, with causative genetic variants identified in multiple genes in people and across various dog breeds. Recently, an atypical case of SCA was documented in a mixed breed dog. In addition to the classic clinical signs and spinocerebellar lesions of SCA, the dog had retinal and optic nerve degeneration and severe, non-regenerative anemia. Whole-genome sequence (WGS) of the affected dog did not reveal any previously identified canine SCA-associated variants. Subsequent variant filtering against a control cohort of over 700 unaffected dog genomes identified a homozygous 4-base-pair frameshift deletion in caseinolytic mitochondrial matrix peptidase chaperone subunit X (CLPX) [XM_038580726.1:c.1723_1726del]. CLPX encodes a subunit of the ATP-dependent ClpXP protease, a molecular chaperone involved in mitochondrial protein degradation. The variant is predicted to cause a frameshift and a premature stop codon within 17 amino acids, truncating approximately 6.64% of the protein. Our study is the first to explore the association of CLPX variants with SCA in any species. Given the high evolutionary conservation of CLPX, this report of a CLPX variant associated with SCA in a dog may have relevance for understanding CLPX-related neurodegeneration and/or anemia in other species.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.15.676223v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Learning the cellular origins across cancers using single-cell chromatin landscapes.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mohamad D Bairakdar, Wooseung Lee, Bruno Giotti, Akhil Kumar, Paula Stancl, Elvin Wagenblast, Dolores Hambardzumyan, Paz Polak, Rosa Karlic, Alexander M Tsankov</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We predict the pre-malignant cell of origin of 37 cancer subtypes with high robustness and accuracy, confirming both the known anatomical and cellular origins of numerous cancers, often at cell subset resolution.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Deciphering the pre-malignant cell of origin (COO) of different cancers is critical for understanding tumor development and improving diagnostic and therapeutic strategies in oncology. Prior work demonstrates that somatic mutations preferentially accumulate in closed chromatin regions of a cancer's COO. Leveraging this information, we combine 3,669 whole genome sequencing patient samples, 559 single-cell chromatin accessibility cellular profiles, and machine learning to predict the COO of 37 cancer subtypes with high robustness and accuracy, confirming both the known anatomical and cellular origins of numerous cancers, often at cell subset resolution. Importantly, our data-driven approach predicts a basal COO for most small cell lung cancers and a neuroendocrine COO for rare atypical cases. Our study also highlights distinct cellular trajectories during cancer development of different histological subtypes and uncovers an intermediate metaplastic state during tumorigenesis for multiple gastrointestinal cancers, which have important implications for cancer prevention, early detection, and treatment stratification.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40962882/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Opportunities for machine learning to predict cross-neutralization in FMDV serotype O.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dennis N Makau, Jonathan Arzt, Kimberly VanderWaal</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a machine learning approach to estimate cross-neutralization between serotype O foot-and-mouth disease viruses using VP1 sequence data and published virus neutralization titer results.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurately estimating cross-neutralization between serotype O foot-and-mouth disease viruses (FMDVs) is critical for guiding vaccine selection and disease management. In this study, we developed a machine learning approach to estimate r1 values-an established measure of antigenic similarity-using VP1 sequence data and published virus neutralization titer (VNT) results. Our dataset comprised 108 serum-virus pairs representing 73 distinct FMDV strains. We applied Boruta feature selection and random forest classifiers, optimizing model performance through tenfold cross-validation and sub-sampling to address class imbalance. Predictors included pairwise amino acid distances, site-specific polymorphisms, and differences in potential N-glycosylation sites. Using a 0.3 r1 threshold to define cross-neutralization, the final model achieved high accuracy (0.96), sensitivity (0.93), and specificity (0.96) in training, and performed robustly on independent test sets - accuracy was 0.75 (95% CI 0.60 and 0.90), F1 score 0.86% and PPV 0.77. Importantly, key VP1 residues-positions 48, 100, 135, 150, and 151-emerged as strong predictors of antigenic relationships. Our results demonstrate the utility of integrating routinely generated genomic data with machine learning to inform vaccine candidate selection and anticipate immune interactions among circulating FMDV strains. This approach offers a practical tool for accelerating vaccine decision-making and can be adapted to other FMDV serotypes. The latest version of the r1 predictive model is available for access via a Shiny dashboard (https://dmakau.shinyapps.io/PredImmune-FMD/).</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40961118/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An extensive evaluation of single-cell RNA-Seq contrastivelearning generative networks for intrinsic cell-typesdistribution estimation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Alsaggaf, I., Buchan, D., Wan, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A large-scale empirical evaluation to investigate the generative encoder networks that are learned by five state-of-the-art single-cell RNA-Seq contrastive learning methods.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Contrastive learning has already been widely used to handle single-cell RNA-Seq data due to its outstanding performance in transforming original data distributions into hypersphere feature spaces. In this work, we conduct a large-scale empirical evaluation to investigate the generative encoder networks that are learned by five different state-of-the-art single-cell RNA-Seq contrastive learning methods. Unlike the conventional discriminative model-based cell-type prediction studies, this work is focused on the performance of contrastive learning-based generative encoder networks in terms of their capacity to estimate the intrinsic distributions of different cell-types - a fundamental property that directly affects the performance of any downstream single-cell RNA-Seq data analytics. The experimental results confirm that supervised contrastive learning-based encoder networks lead to better performance than self-supervised contrastive learning-based encoder networks, and the recently proposed Gaussian noise augmentation-based single-cell RNA-Seq contrastive learning method shows the best performance on estimating the intrinsic distribution of different cell-types.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.15.675691v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Protein tyrosine phosphatase receptor type D - A convergent risk gene for MDD, BPD, and ADHD and regulator of dopaminergic neuroplasticity in reward associated circuits; A multivariate GWAS, brain tissue and cell type enrichment, and gene finemapping</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lawrence, C., Hansen, T. F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose that the loss of PTPRD D1 phosphatase domain results in hyperactivated tropomyosin receptor kinase B (TrkB) and rearranged tyrosine-protein kinase receptor (RET) signaling in dopamine neurons, potentiating excessive downstream phosphorylation and activation</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Major depressive disorder (MDD), bipolar disorder (BPD), and attention-deficit hyperactivity disorder (ADHD), are prevalent, highly heritable psychiatric disorders with significant degrees of genetic overlap. Using open-sourced summary statistics from the Psychiatric Genomics Consortium and 1000 Genomes European reference panel, we utilize a plethora of statistical frameworks to estimate shared genetic liability across MDD, BPD, and ADHD. We observed significant enrichment of latent genetic variables in dopamine neurons, and identify protein tyrosine phosphatase receptor delta (PTPRD), as a candidate gene. Literature review and gene fine-mapping suggest that PTPRD indirectly regulates DAT transporter surface trafficking and neuronal excitability. We propose that the loss of PTPRDs D1 phosphatase domain results in hyperactivated tropomyosin receptor kinase B (TrkB) and rearranged during transfection proto-oncogene tyrosine-protein kinase receptor (RET) signaling in dopamine neurons, potentiating excessive downstream phosphorylation and activation of VAV2, ERK1/2, and PKC. We hypothesize that these mechanisms contribute to the significantly altered reward-related behaviors such as cocaine-conditioned place preference, cocaine self-administration, motivation for cocaine, and goal oriented behavior that are consistently observed across PTPRD KO/inhibition mice studies. With the inclusion of literature review, we present the first integrated multivariate genetic and mechanistic analysis aimed to identify pleiotropic, cell type-specific mechanisms driving psychiatric comorbidity at the biochemical level.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.17.676918v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A ratio-based framework using Quartet reference materials for integrating long- and short-read RNA-seq</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chen, Q., Guo, X., Wang, D., Zhao, J., Xu, Y., You, Y., Mai, Y., Duan, S., Liu, Y., Zhang, Y., Li, X., Chen, H., Hou, W., Yu, Y., Dong, L., Li, J., Ritchie, M. E., Zhang, R., Shi, L., Zheng, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We first establish that ratio-based quantification against built-in reference samples effectively removes technical noise, revealing underlying biological signals, revealing the highest quantification accuracy for both isoforms and AS events, and provide a comprehensive benchmark using these ground truths.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Long-read RNA sequencing (lrRNA-seq) enables full-length transcript profiling but is confounded by technical batch effects that compromise quantification and prevent data integration across platforms, protocols, and laboratories. The lack of a transcriptome-wide biological ground truth has hindered objective benchmarking. To address these dual challenges, we leveraged certified Quartet reference materials to generate one of the largest multi-center lrRNA-seq resources to date: over one billion long reads from 144 libraries across four PacBio and Nanopore protocols in four independent laboratories. We first establish that ratio-based quantification against built-in reference samples effectively removes technical noise, revealing underlying biological signals. We then constructed the first ratio-based reference datasets for full-length transcripts-comprising 10,218 isoforms and 6,032 alternative splicing (AS) events-and orthogonally validated them with RT-qPCR. Finally, a comprehensive benchmark using these ground truths reveals that a hybrid strategy integrating long- and short-read data (hybrid-seq) achieves the highest quantification accuracy for both isoforms and AS events. Our work provides a foundational framework and resource for evaluating lrRNA-seq technologies and accelerating the standardization of full-length transcriptomics for research and clinical applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.15.676287v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Discovery and development of single-nucleotide polymorphism markers for resistance to Striga gesnerioides in cowpea (Vigna unguiculata)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ongom, P. O., Fatokun, C. A., Boukar, O. O.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A genome-wide scan uncovered a primary gene effect signal on chromosome Vu11 and minor regions on chromosomes Vu02, Vu03, Vu07, Vu09 and Vu10, previously implicated in race-specific resistance to S. gesnerioides in cowpea.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The parasitic weed [Striga gesnerioides (Willd.) Vatke] is a principal biotic constraint to cowpea [Vigna unguiculata (L.) Walp.] production in West and Central Africa, causing severe yield reductions. Multiple races of S. gesnerioides exist across the cowpea-growing areas of the sub-region. Past efforts identified some resistant sources and race-specific genes underpinning Striga resistance, but deployment of associated markers in breeding is limited. Here, we utilized a 51K cowpea iSelect single-nucleotide polymorphisms (SNPs) to decipher genomic regions underlying Striga resistance and explore marker conversion and validation for easy deployment. The study used two-year phenotypic data on a minicore panel of 368 cowpea genotypes screened at two sites in Northern Nigeria. SNPs performances were verified and validated using two independent sets of 60 and 20 diverse genotypes respectively. The minicore displayed apparent differences in response to the S. gesnerioides attack. A genome-wide scan uncovered a primary gene effect signal on chromosome Vu11 and minor regions on chromosomes Vu02, Vu03, Vu07, Vu09 and Vu10. The major effect region on Vu11 harbored a coil-coil nucleotide-binding site leucine-rich repeat (CC-NBS-LRR) protein, encoded by the RSG3-301 gene, previously implicated in race-specific resistance to S. gesnerioides in cowpea. The associated SNPs were successfully converted into Kompetitive Allele-Specific PCR (KASP) assays and validated using 20 independent diverse cowpea genotypes. Five KASP markers, snpVU00075, snpVU00076, snpVU00077, snpVU00078, and snpVU00079, depicted consistent and significant associations with the phenotype in the validation set. The markers provide valuable tools for efficient marker-assisted selection (MAS) in breeding programs focused on developing Striga-resistant cowpea varieties.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.15.676209v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Nanopore long-read only genome assembly of clinical Enterobacterales isolates is complete and accurate</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nagy, D., Pennetta, V., Rodger, G., Hopkins, K., Jones, C. R., The NEKSUS Consortium,, Hopkins, S., Crook, D., Walker, A. S., Robotham, J., Hopkins, K. L., Ledda, A., Williams, D., Hope, R., Brown, C. S., Stoesser, N., Lipworth, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Nanopore long-read only bacterial genome assembly with Autocycler combined with Medaka polishing (using un-subsampled reads) is similarly accurate and possibly more complete than hybrid assemblies, representing a viable alternative for incorporating high-quality genomic data, including plasmids, into</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Whole bacterial genome sequence reconstruction using Oxford Nanopore Technologies ('Nanopore') long-read only sequencing may offer a lower-cost, higher-throughput alternative for pathogen surveillance to 'hybrid' assembly with recent improvements in Nanopore sequencing accuracy. We evaluated the accuracy, including plasmid reconstruction, of Nanopore long-read only genome assemblies of Enterobacterales. We sequenced 92 genomes from clinical Enterobacterales isolates, collected in England under a national surveillance program, with long-read Nanopore (R10.4.1, Dorado v5.0.0 super-high-accuracy basecalled) and short-read Illumina (NovaSeq) sequencing approaches. Genomes were assembled using three long-read only (Flye; Hybracter long; Autocycler), and three hybrid assemblers (Hybracter hybrid; Unicycler normal; bold). Three polishing modalities (Medaka v2 with subsampled or un-subsampled long-reads; Polypolish + Pypolca with short-reads) were investigated. Autocycler circularised the most chromosomes (87/92 [95%]). Plasmid sequence reconstruction was comparable between all assemblers except Flye, all recovering 90-96% of plasmids, although the 'ground truth' was uncertain. Flye performed worse than other assemblers on almost all metrics. Autocycler + Medaka (un-subsampled long-reads) was the most accurate long-read only assembler/polisher combination, comparable to hybrid assemblies (median 0 [IQR:0-0] SNPs and 0 [IQR:0-1] indels per genome; quality value/Q score, 100 [IQR: 64-100]), with only 4/92 genome sequences having >10 SNPs/indels. Medaka polishing with un-subsampled long-reads resulted in small improvements in indels but not SNPs for both Flye and Autocycler assemblies. Seven-locus MLST, antimicrobial resistance, virulence, and stress gene annotation was equivalent across assembler/polisher combinations. Nanopore long-read only bacterial genome assembly with Autocycler combined with Medaka polishing (using un-subsampled reads) is similarly accurate and possibly more complete than hybrid assemblies, representing a viable alternative for incorporating high-quality genomic data, including plasmids, into Enterobacterales surveillance.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.15.676237v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The small ORFeome of Tribolium castaneum reveals deeply conserved small ORFs broadly expressed across tissues</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Guerra-Almeida, D., Nery, M. F., Mucherino-Munoz, J. J., Pessoa-Costa, E., Tschoeke, D. A., Nunes-da-Fonseca, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Small open reading frames (small ORF/smORFs/sORFs) were historically overlooked due to their short length (1) in at least one sample, and 16.5% expressed in five or more libraries.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Small open reading frames (small ORFs/smORFs/sORFs) were historically overlooked due to their short length (1) in at least one sample, and 16.5% expressed in five or more libraries. Conserved small ORFs showed broader and more stable expression, while lineage-specific ones displayed restricted expression, mostly in gonads or embryos. Isoformic and small CDSs were the most widely expressed classes. Notably, 29 small ORFs exhibited transcriptional and translational support (Swiss-Prot and/or Ribo-seq), 28 being small CDSs. A total of 390 small ORFs had potential paralogs, supporting gene duplication as a mechanism for small ORF emergence. Altogether, our findings reinforce that small ORFs, contrary to initial assumptions, can be deeply conserved and potentially perform key roles in evolutionary processes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.15.675444v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">HUBMet: An integrative database and analytical platform for human blood metabolites and metabolite-protein associations</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, X., Qiao, X., Zenere, A., Barde, S., Wang, J., Zhong, W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed HUBMet, an open-access web server that includes 3,950 metabolites and 129,814 metabolite-protein associations, with four analytical modules: Over-Representation Analysis (ORA) for enrichment analysis; Metabolite Set Enrichment Analysis (MSEA)</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding human blood metabolites is essential for deciphering systemic physiology and disease mechanisms, yet remains challenging due to diverse origins and dynamic regulation. In this study, we developed HUBMet (https://hubmet.app.bio-it.tech/home), an open-access web server that includes 3,950 metabolites and 129,814 metabolite-protein associations, with four analytical modules: Over-Representation Analysis (ORA) for enrichment analysis; Metabolite Set Enrichment Analysis (MSEA) for quantitative data analysis; Tissue Specificity Analysis (TSA) for assessing metabolite-tissue relevance; Metabolite-Protein Network Analysis (MPNet) for identifying key metabolite-protein associations and functional modules. HUBMet's utility is demonstrated through a COVID-19 case study revealing metabolic signatures associated with disease severity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.16.676477v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Systematic evaluation of the impact of promoter proximal short tandem repeats on expression</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, X., Lingzhi, Z., Wang, E., Benton, S., Modolo, E., Maksimov, M., Shleizer-Burko, S., Gong, Q., Wang, C., Lamkin, M., Mendenhall, E., Gymrek, M., Goren, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We study the impact of variation in repeat copy number on expression at thousands of short tandem repeats (STRs), which consist of consecutive repeated sequences of 1-6bp, and provide a framework for understanding how dynamic form of genetic variation shapes gene expression.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genetic variation at thousands of short tandem repeats (STRs), which consist of consecutive repeated sequences of 1-6bp, has been statistically associated with gene expression and other molecular phenotypes in humans. However, the causality and regulatory mechanisms for most of these STRs remains unknown. Massively parallel reporter assays (MPRA) enable testing the regulatory activity of a large number of synthesized variants, but have not been applied to STRs due to experimental and computational challenges. Here, we optimized an MPRA framework based on random barcoding to study the impact of variation in repeat copy number on expression. We first performed an MPRA on sequences derived from 30,516 promoter-proximal STR loci along with up to 152bp of genomic context, testing 3-4 variants with differing repeat copy numbers for each locus in HEK293T cells. We identified 1,366 loci with significant associations between repeat copy number and expression, which were enriched for positive effect sizes (P=2.08e-110). We then designed a second MPRA in which we performed deeper perturbations, including systematic manipulation of the repeat unit sequence, orientation, and copy number, with 200-300 perturbations for each of the 300 loci with the strongest signals. Our results revealed that the repeat unit sequence is the primary driver of differences in the relationship between copy number and expression across loci, whereas orientation and flanking sequence have weaker effects, primarily for AT-rich repeat units. The high resolution of these perturbations enabled us to detect non-linear effects, most notably for AAAC/GTTT repeats, which emerge only beyond a certain copy number threshold. Finally, we observed that a subset of STRs in our library show expression levels that are tightly linked with predicted DNA secondary structure formation. We repeated our perturbation MPRA in HeLa S3 cells under wildtype and RNase H1 knockdown conditions, which, via reduction in RNase H1 activity, are expected to hinder resolution of R-loops. This demonstrated that associations between copy number and expression at G-quadruplex-forming CCCCG/CGGGG repeats are particularly sensitive to loss of RNase H1, providing support for an R-loop mediated mechanism for these repeats. Altogether, we establish STRs as a critical component of the non-coding regulatory grammar and provide a framework for understanding how this dynamic form of genetic variation shapes gene expression.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.14.676153v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">TypeAssembly: Copy number estimation and allele typing for haplotype assemblies</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lai, H.-S., Liu, Y.-S., Chiu, H.-C., Chang, T.-Y., Lin, H.-F., Tseng, Y.-H., Hung, T.-K., Lai, S.-K., Hsu, J. S., Hsu, C.-L., Yang, Y.-C., Chen, P.-L., Chen, C.-Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed TypeAssembly, a local alignment-based framework for copy number estimation and allele typing.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurately annotating complex genes in the human genome, particularly from haplotype assemblies, remains a significant challenge. To overcome this, we developed TypeAssembly, a local alignment-based framework for copy number estimation and allele typing. Operating in two modes, mode-FASTA and mode-VCF, TypeAssembly can define alleles by either sequence or variant information. We successfully applied it to annotate 41 genes in the MHC locus, 17 KIR genes, and, for the first time, 15 pharmacogenes across 466 haplotype assemblies. This study establishes TypeAssembly as a robust method for accurately annotating complex genomic regions and provides an evaluation of existing gene annotations and callers.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.13.675996v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Establishing an RNA sensor with high sensitivity and dynamic range utilizing a signal amplifier platform</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lim, H. E., Chappell, J., Segatori, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We demonstrate the utility of a combined RADAR-amplifier platform for real-time monitoring of subtle changes in the abundance of endogenous transcripts under physiological conditions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Precise control of gene expression in a cell-state-specific manner is essential for effective therapeutic interventions in complex and dynamic disease microenvironments. Traditional targeting strategies that rely on surface markers or cell type-specific promoters often assume static cellular identities, limiting effectiveness in contexts such as cancer and inflammation, where cell states are highly heterogeneous and dynamic. RNA sensors, such as RADAR (RNA sensing using Adenosine Deaminases Acting on RNA), provide a modular, programmable, and non-integrating platform for classifying cell states. However, it is also characterized by low sensitivity and dynamic range, which limits its applications in detecting low-abundance transcripts. In this work, we integrate RADAR sensors with a signal amplification circuit to enhance sensitivity and dynamic range. We demonstrate that this combined RADAR-amplifier platform enables real-time monitoring of subtle changes in the abundance of endogenous transcripts under physiological conditions. Our results demonstrate the utility of this platform for fundamental biological studies and the development of precision therapeutic strategies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.17.675666v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">vsgseq2: an updated pipeline for analysis of the diversity and abundance of population-wide Trypanosoma brucei VSG expression</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Oldrieve, G. R., Larcombe, S., Krasilnikova, M., Mugnier, M. R., Matthews, K. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> VSGSeq is an amplicon sequencing approach that enables surveillance of the population-wide diversity and abundance of expressed variant surface glycoprotein (VSG) coats.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Trypanosoma brucei is an extracellular eukaryotic parasite that causes sleeping sickness in humans and Nagana, Surra and Dourine in livestock, game animals and horses. The parasite displays an extensive immune evasion mechanism, utilising the expression and ability to switch antigenically distinct variant surface glycoprotein (VSG) coats. VSG encoding genes account for ~10% of the T. brucei genome, and mosaic VSGs, assembled from distinct incomplete VSG gene copies, can be produced from this VSG library, generating an almost infinite VSG repertoire, which enables chronic infections. Each parasite expresses just one VSG at a time, but within a host, many VSGs can be expressed simultaneously. VSGSeq is an amplicon sequencing approach that enables surveillance of the population-wide diversity and abundance of expressed VSGs. vsgseq2 is an updated bioinformatics pipeline that enhances the reproducibility, accuracy, and efficiency of VSGseq analysis, utilising publicly available analytical tools.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.12.675796v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Species-Agnostic and Salmonella-Specific Models for Antimicrobial Resistance Prediction Using FCGR and ResNet-18</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ismail, S. M., Fayed, S. H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed deep learning models combining Frequency Chaos Game Representation (FCGR) with a ResNet-18 architecture to classify resistance phenotypes from bacterial genomes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Antimicrobial resistance (AMR) prediction from whole genome sequencing is a major challenge for clinical microbiology and surveillance. We developed deep learning models combining Frequency Chaos Game Representation (FCGR) with a ResNet-18 architecture to classify resistance phenotypes from bacterial genomes. Using a homology-aware clustering strategy to prevent genomic data leakage, we compared a species-specific model trained on Salmonella enterica to a species-agnostic model trained across S. enterica, Escherichia coli, and Staphylococcus aureus. The Salmonella model achieved reliable performance on the independent test set (Balanced Accuracy = 0.82, MCC = 0.66), though results varied by antibiotic: cephalosporins reached near-perfect levels (Balanced Accuracy up to 0.98, MCC up to 0.92), while tetracycline and ampicillin remained more difficult (Balanced Accuracy = 0.7-0.79, MCC = 0.55-0.58). In contrast, the species-agnostic model showed weaker generalization overall, reflecting dataset heterogeneity and missing labels, but provided clear gains for underrepresented species. Notably, tetracycline prediction for E. coli improved compared to the corresponding single-species model. These findings demonstrate that FCGR-based deep learning can produce robust AMR predictors in species-focused settings, and that species-agnostic models, while currently limited, may enhance predictive power for poorly sampled pathogens.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.13.675356v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Protein Language Models Capture Structural and Functional Epistasis in a Zero-Shot Setting</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nambiar, A., Littlefield, S. B., Cuellar, C., Khorana, R., Maslov, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We use epistasis, the dependence of a mutation's effect on its sequence context, as a lens to probe what PLMs capture about proteins, and that epistasis provides a powerful window into the biological principles embedded in their representations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein language models (PLMs) learn from large collections of natural sequences and achieve striking success across prediction tasks, yet it remains unclear what biological principles underlie their representations. We use epistasis, the dependence of a mutation's effect on its sequence context, as a lens to probe what PLMs capture about proteins. Comparing PLM-derived scores with deep mutational scanning data, we find that epistasis emerges naturally from pretrained models, without supervision on experimental fitness. Raw model scores align with residue-residue contacts, indicating that PLMs internalize structural proximity. Applying a nonlinear transformation to bring model outputs onto the experimental scale, however, shifts the signal toward functional couplings between distant sites. These findings show that PLMs capture both structural and functional dependencies from sequence data alone, and that epistasis provides a powerful window into the biological principles embedded in their representations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.14.676130v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrative Computational and Experimental Analysis of Curly Su Mutations in Drosophila melanogaster</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sobitan, A., Jalal, M. S., Yao, Q., Constantin, M., Xu, J., Hunter, G., Duttaroy, A., Teng, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified single-point mutations that disrupt protein stability and alter gene expression in Drosophila melanogaster and show significant changes in wing morphology and lifespan.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The Curly Su (dMPO) protein, a homolog of the human myeloperoxidase (hMPO), is critical for wing development in Drosophila melanogaster. Like human peroxidases, dMPO is involved in various cellular and physiological processes, producing significant quantities of reactive oxygen species that contribute to both development and immunity in the fruit fly. Given the significant sequence and structural similarities between dMPO and hMPO, dMPO serves an ideal model for studying peroxidase functions and related pathologies. We performed saturated computational mutagenesis on dMPO, analyzing the effects of 11,191 missense mutations on its stability. Notably, the G378W mutation exhibited the greatest destabilizing effect, while the W621R, potentially pathogenic, also reduced dMPO stability. To investigate these effects in vivo, we used genome editing to generate the transgenic Drosophila with G378W, W621R, and deletion of residues 305-687. Remarkably, G378W mutants displayed significant alterations in wing morphology and reduced lifespan. RNA-seq analysis of transgenic and wild-type flies revealed differentially expressed genes (DEGs), as interpreted through gene ontology analysis. Our integrated computational and genetic approach identified dMPO mutations that disrupt protein stability and alter gene expression. These findings offer new insight into how single-point mutations can lead to systemic biological changes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.15.676221v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Optimizing Polygenic Scores for Complex Morphological Traits: A Case Study in Nasal Shape Prediction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yuan, M., Goovaerts, S., Claessens, N., Devine, J., Becelaere, S., Cleynen, I., Claes, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel framework to enhance polygenic prediction for complex morphological traits by leveraging multivariate GWAS summary statistics for improved SNP selection, (2) defining genetically informative phenotypes, and (3) benchmarking PGS methods to select the optimal model.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Polygenic scores (PGS) facilitate the prediction of an individual's phenotype from their genotype. Typically, PGS methods apply regularization or use a clumping and thresholding (C+T) approach to handle SNP inclusion. To achieve good prediction accuracy, these approaches rely on effect size estimates from well-powered genome-wide association studies (GWAS). However, this is currently not feasible for morphological shape when phenotyped as univariate traits. Here, we introduce a novel framework to enhance polygenic prediction through three key components: (1) leveraging multivariate GWAS summary statistics for improved SNP selection, (2) defining genetically informative phenotypes, and (3) benchmarking PGS methods to select the optimal model. Our approach integrates multivariate GWAS, which performs an omnibus test against all phenotypic variables jointly with increased power. Specifically, our approach leverages P values from multivariate GWAS to improve SNP selection while maintaining the effect size estimates for the univariate trait under investigation, allowing the use of current PGS tools. We evaluated our proposed method for predicting 3D nasal morphology using a dataset of 52,896 individuals of European ancestry from the UK Biobank. Using the C+T method, SNP selection based on multivariate GWAS resulted in significantly improved phenotypic prediction (P = 9.74e-5) for eigen-shapes, with a mean variance explained of 3.88% (SD = 1.59%) compared to 2.02% (SD = 1.10%) using a traditional univariate approach in the test set (n = 2,896). We also tested whether heritability-optimized phenotypes were more predictable than eigen-shapes derived from principal component analysis (PCA). On average, with SNP selection based on multivariate GWAS using the C+T method, heritability-optimized phenotypes yielded greater predictive performance, with PGS scores explaining 2.72%-10.37% of phenotypic variance, compared to 1.05%-6.84% for eigen-shapes. Furthermore, benchmarking several PGS methods revealed that LDpred2 consistently achieved the best performance for predicting nasal morphology. Our results demonstrate that combining multivariate GWAS P values with optimized phenotypes and advanced PGS models leads to more accurate polygenic prediction for complex morphological traits.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.14.676081v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PAN-CANCER ANALYSES IDENTIFY ONCOGENIC DRIVERS, EXPRESSION SIGNATURES, AND THERAPEUTIC VULNERABILITIES IN RHO GTPase PATHWAY GENES</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fernandez, R., Lorenzo-Martin, L. F., Quesada, V., Bustelo, X. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a pan-cancer analysis of 484 genes encoding RHO GTPases, regulators, proximal effectors, distal downstream signaling elements, and components of their proximal interactomes using data from over 10,000 tumor samples and 33 tumor types.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> RHO family GTPases are key regulators of cancer-related processes such as cytoskeletal dynamics, cell migration, proliferation, and survival. Despite this, a comprehensive understanding of RHO signaling alterations across tumors is still lacking. Here, we present a pan-cancer analysis of 484 genes encoding RHO GTPases, regulators, proximal effectors, distal downstream signaling elements, and components of their proximal interactomes using data from over 10,000 tumor samples and 33 tumor types present in The Cancer Genome Atlas (TCGA). In addition, we have utilized available data from genome-wide functional dependency screens performed in more than 1,000 gene-edited cancer cell lines. This study has uncovered positively selected mutations in both well-known and previously uncharacterized RHO pathway genes. Transcriptomic profiling reveals widespread and tumor-specific differential expression patterns, some of them correlating with copy number changes. Interestingly, certain regulators exhibit consistent expression profiles across tumors opposite to those predicted from their canonical roles. Coexpression and gene set enrichment analyses highlight coordinated transcriptional programs involving some RHO GTPase pathway genes and their linkage to key cancer hallmarks, including extracellular matrix reorganization, cell motility, cell cycle progression, cell survival, and immune modulation. Functional screens further identify context-specific dependencies on several deregulated RHO GTPase pathway genes. Altogether, this study provides a comprehensive map of RHO GTPase pathway alterations in cancer and identifies new oncogenic drivers, expression-based signatures, and therapeutic vulnerabilities that could guide future mechanistic and translational research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.14.676083v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Antimicrobial Peptides Induce Cell Death in Marginal Zone Lymphoma Models Resistant to Targeted Therapies</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Spriano, F., Arribas, A. J., Zhang, F., Mangoni, M. L., Buonocore, F., Bertoni, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that natural antimicrobial peptides, including Antarctic fish-derived trematocines and chionodracine variants, and amphibian temporins, possess selective cytotoxicity against malignant cells by exploiting tumor-specific membrane alterations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Marginal zone lymphoma (MZL) is an indolent yet incurable B-cell malignancy in which targeted agents such as BTK and PI3K inhibitors frequently fail due to resistance or toxicity. Antimicrobial peptides (AMPs), evolutionarily conserved effectors of innate immunity, possess selective cytotoxicity against malignant cells by exploiting tumor-specific membrane alterations. We evaluated the antitumor activity of seven natural AMPs, including Antarctic fish-derived trematocines and chionodracine variants, and amphibian temporins, against MZL cell lines (VL51, Karpas1718) and derivatives resistant to BTK, PI3K{delta}, or PI3K/{delta} inhibitors. Among them, W-trematocine and temporin L demonstrated potent dose-dependent cytotoxicity with IC values of 5.7-10 M, maintaining full activity in all resistant models. Other peptides showed moderate activity, while chionodracine-1 was inactive. Notably, W-trematocine displayed minimal toxicity toward non-malignant cells in prior studies, underscoring its selectivity. AMP-mediated killing, driven by membrane disruption and non-apoptotic death pathways, bypassed conventional resistance mechanisms, suggesting therapeutic potential in relapsed/refractory disease. These findings highlight natural AMPs as promising candidates for development in drug-resistant MZL, warranting further optimization and preclinical validation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.13.674604v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Longitudinal antibody titers measured after COVID-19 mRNA vaccination can identify individuals at risk for subsequent infection.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hyeongki Park, Naotoshi Nakamura, Sho Miyamoto, Yoshitaka Sato, Kwang Su Kim, Kosaku Kitagawa, Yurie Kobashi, Yuta Tani, Yuzo Shimazu, Tianchen Zhao, Yoshitaka Nishikawa, Fumiya Omata, Moe Kawashima, Toshiki Abe, Yoshika Saito, Saori Nonaka, Morihito Takita, Chika Yamamoto, Hiroshi Morioka, Katsuhiro Kato, Ken Sagou, Tetsuya Yagi, Takeshi Kawamura, Akira Sugiyama, Aya Nakayama, Yudai Kaneko, Risa Yokokawa Shibata, Kazuyuki Aihara, Tatsuhiko Kodama, Akifumi Kamiyama, Tomokazu Tamura, Takasuke Fukuhara, Kenji Shibuya, Tadaki Suzuki, Shingo Iwami, Masaharu Tsubokura</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A key issue in the post-COVID-19 pandemic era is the ongoing administration of COVID- 19 vaccines, and we found that spike protein-specific IgA titers are lower during the early stage after booster vaccination in participants who went on to become infected with SARS-Co</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> A key issue in the post-COVID-19 pandemic era is the ongoing administration of COVID-19 vaccines. Repeated vaccination is essential for preparing against currently circulating and newly emerging severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) variants. However, optimizing vaccination strategies is crucial to efficiently manage medical resources and establish an effective vaccination framework. Therefore, a strategy to identify poor responders with lower sustained antibody titers would be beneficial because these individuals should be considered high priority for revaccination. We investigated longitudinal antibody titer data in a cohort of 2526 people in Fukushima, Japan, collected between April 2021 and November 2022. Using mathematical modeling and machine learning, we stratified the time-course patterns of antibody titers after two primary doses and one booster dose of COVID-19 messenger RNA vaccines. We identified three populations, which we refer to as the durable, the vulnerable, and the rapid-decliner populations, and approximately half of the participants remained in the same population after the booster dose. The rapid-decliner population experienced earlier infections than the others. Furthermore, when comparing spike protein-specific immunoglobulin G (IgG) titers, spike protein-specific IgA titers, and SARS-CoV-2-specific T cell responses between participants who experienced subsequent infections after booster vaccination and those who did not, we found that spike protein-specific IgA titers were lower during the early stage after booster vaccination in participants who went on to become infected with SARS-CoV-2. This approach could be used to inform policy decisions on vaccine distribution to maximize population-level immunity both in future pandemics and in the post-COVID-19 pandemic era.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40961221/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Differentiating Gastric Cancers from Acid Peptic Diseases through Integrative Targeted Proteomics and Machine Learning Approaches.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Poornima Ramesh, Shubham Sukerndeo Upadhyay, Sonet Daniel Thomas, Chandrashekar Jeevaraj Sorake, Ganesh M K, Vijith Vittal Shetty, Prashant Kumar Modi, Rohan Shetty, Manavalan Vijayakumar, Jalaluddin Akbar Kandel Codi, Thottethodi Subrahmanya Keshava Prasad</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel Deep Neural Network-based pattern recognition scoring architecture, integrated with a model explainability tool (SHAP), has been developed to score and categorize GCs.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Gastric cancers (GCs) are often diagnosed in advanced stages owing to nonspecific early symptoms resembling Acid Peptic Diseases (APDs). Despite recent efforts, a simple, liquid biopsy-based multiprotein panel prediagnostic assay capable of differentiating GCs from APDs is lacking. Mass spectrometry (MS)-based targeted proteomics methods, including Multiple Reaction Monitoring (MRM), are utilized as the method of choice to develop Laboratory Developed Tests (LDTs) that revolutionize GC early diagnosis and screening. In this study, a 22-min MS-MRM LDT was developed and tested to quantify a serum protein panel in 135 serum samples from treatment-naive cases of GCs, APDs, and healthy individuals. Notably, a novel Deep Neural Network (DNN)-based pattern recognition scoring architecture, integrated with a model explainability tool (SHAP), was developed to score and categorize GCs. The MRM-MS assay produced minimal carryover and matrix effects, with adequate limits of detection/quantification. Quantities of SAA1 and IGFBP2, as determined through ELISA, demonstrated similar sensitivity compared to the LDT. Importantly, the DNN-based scoring architecture efficiently differentiated GCs from the rest of the samples (AUROC = 0.95), with average precision marking >0.90 and minimal bias in protein expression affecting model performance. This LDT can serve as a prediagnostic screening method to distinguish GCs from APDs, guiding clinicians and patients in proceeding with a confirmatory diagnosis.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40958687/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

