<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Monday, 16 June 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Fetal sex is associated with the maternal gut microbiota composition and its metabolic function</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Williams, B. M., Liu, J., Bernabe, B. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of the composition of the maternal gut microbiota in male and female pregnancies and the role of fetal sex in the composition.</p><p style="font-size: 16px; text-align: justify;">Background: Pregnancy affects almost every organ and system, including the microorganisms that resides in the gastrointestinal track, the gut microbiota. Pregnancy hormones, such as progesterone and estradiol, alter the composition of the maternal gut microbiota, directly and indirectly, and in turn, the gastrointestinal microorganism can modify the activity of reproductive hormones. While the levels of pregnancy hormones depend on the fetal sex, the role of fetal sex in the maternal gut microbiota composition and function has not been explored yet. Methods: Using publicly available microbial sequencing data from a cohort of 485 pregnant women in Guangzhou, China, we identified taxonomical and predicted metabolic signatures associated with fetal sex and gestational age using zero-inflated Gaussian models and linear models, respectively, correcting for pre-pregnancy BMI, maternal age, parity, and residency status. Further, we determined sex-specific co-abundance microbial networks using SPIEC-EASI. Results: We observed higher diversity levels in male than in female pregnancies and identified multiple microbial species and predicted microbial enzymes that were differentially abundant between the fetal sex groups, such as increased levels of Tyzerrella and {beta}-glucuronidase in female pregnancies, respectively. {beta}-glucuronidase deconjugates glucuronide-bound estrogens into free estrogens and glucuronic acid. Further, the microbial co-abundance networks between the male and female pregnancies were distinct, with efficient information transfer being representative of the female network while the male network was characterized by a larger number of competitive or exclusion interactions between microbial species. While the two networks had similar core communities, there were distinct connections between hormone metabolizing microbes, with {beta}-glucuronidase expressing microbes being more highly connected in the female network. Conclusions: Our study suggests that differences in the maternal-fetal interactions, such as fetus/placenta-produced hormone levels, may affect and be affected by the composition of the gut microbiota of pregnant individuals in a sex-specific manner. A better understanding of the relationship between fetal sex and the maternal gut microbiota may improve maternal health and fetal outcomes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.15.659758v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">VAMPIRE: Analyzing variation and motif pattern in tandem repeats</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yang, Z., Zhang, S., Logsdon, G. A., Mao, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce VAMPIRE, a k-mer-based computational tool for comprehensive TR discovery, annotation, and quantification, with broad utility across human genetics, evolutionary biology, and the study of complex TRs.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Tandem repeats (TRs) are pervasive in eukaryotic genomes and play key roles in genome organization, evolution, and function, particularly in complex regions such as centromeres and subtelomeres. Although long-read sequencing technologies have improved the resolution of these regions, existing methods remain limited in their ability to systematically and accurately characterize large-scale TRs. Here, we introduce VAMPIRE, a k-mer-based computational tool for comprehensive TR discovery, annotation, and quantification. Unlike previous methods, VAMPIRE enables reference-free, fine-grained decomposition of both simple and complex TRs, capturing motif variation in sequence, length, and structure with high sensitivity and scalability. Applied to complete telomere-to-telomere (T2T) human and nonhuman primate (NHP) genome assemblies, VAMPIRE reveals previously unrecognized high-order repeat inversions within human centromeres - an underappreciated evolutionary mechanism contributing to centromere diversity. Additionally, the tool identifies lineage-specific and expanded TRs, including human-specific STR/VNTR expansions and NHP-specific subtelomeric heterochromatin (e.g., pCht/StSat), underscoring their dynamic turnover and structural complexity. VAMPIRE provides a robust and scalable framework for TR analysis in the era of long-read sequencing, with broad utility across human genetics, evolutionary biology, and the study of complex TRs in non-model organisms.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.15.659631v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Histo-LCM-Hi-C reveals the 3D chromatin conformation of spatially localized rare cells in tissues at high resolution</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liu, Y., Chen, M., Liu, X., Xu, Z., Li, X., Guo, Y., Czajkowsky, D. M., Shao, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Histological Laser Capture Microdissection Hi-C (Histo-LCM-Hi-C) to enable the characterization of chromatin architecture of phenotype-defined, spatially localized cells within intact tissue sections from as few as 300 cells.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> It is now well established that an understanding of the chromatin structure is essential to delineate the mechanisms underlying genomic processes. However, while methods to obtain this information from cells in vitro are widely available, there is presently a significant lack of techniques that can acquire this data from cells in the tissue. Such a capability is critical to determine the dependence of the local tissue environment on cell functioning. Further, this ability is particularly necessary for cells that are a significant minority of the total tissue population, which are often obscured in data dominated by more abundant tissue cells. Here we have developed Histological Laser Capture Microdissection Hi-C (Histo-LCM-Hi-C) to enable the characterization of chromatin architecture of phenotype-defined, spatially localized cells within intact tissue sections from as few as about 300 cells. We demonstrate the effectiveness of this approach with the generation of the first 3D Hi-C map of the tissue-resident macrophages of the liver, the Kupffer cells (KC), which are a minor cell population in the normal liver. As expected, owing to their relative rarity, these KC maps are significantly different from those obtained from whole liver, revealing distant contacts between putative enhancers and genes involved in key KC functions as well as significant differences with that of in vitro induced bone-marrow derived macrophages. We anticipate that this method will prove to be an indispensable technique in the growing repertoire of methodologies used for the characterization of the genomic properties of cells within their native environment.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.10.658807v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Applying Self-Supervised Tissue Patch Encoders to Enable Cellular Graph Encoding</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ding, Z., Baras, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We demonstrate that a pretrained tissue patch encoder, originally trained in the context of the token used in conventional vision transformer frameworks, can also serve effectively as a cellular graph encoder for cell-based patches.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent advances in computation pathology have seen the development of various forms of foundational models that have enabled high-quality, general-purpose feature extraction from tissue patches. However, most of these models are somewhat limited in their ability to capture cell-to-cell spatial relationships essential for understanding tissue microenvironments. To address this issue, self-supervised learning (SSL) applied to cellular graph models is a promising avenue of investigation as this would afford one the ability to capture cell-level interpretability of the tissue microenvironment and learn relationships from the cellular topology. Compared to SSL on tissue patches, developing cellular graph models requires considerable computation and algorithmic effort, starting with accurate cell detection/segmentation from tissue and then subsequently requiring adequate cell-level feature extraction. In this study, we demonstrate that a pretrained tissue patch encoder, originally trained in the context of the token used in conventional vision transformer frameworks can also serve effectively as a cellular graph encoder for cell-based patches. We report three key findings: 1) the pretrained tissue patch encoder acts as a powerful graph-level feature extractor for cellular graphs in tissue patches; 2) it also can function as a cell-level feature extractor, allowing each cell to be contextualized within its local tissue microenvironment environment; 3) the cell patch size and positional embeddings are critical factors for high-quality cellular graph feature extraction. To validate and support these insights, we evaluate our approach on various tissue region classification tasks from three different types of cancers. Experimental results demonstrate that our method achieves performance comparable to the current state-of-the-art foundation models in computational pathology. These results highlight the potential of pretrained tissue patch encoders in cellular graph modeling, which can enable insights into how the cellular features and topology are related to various biological processes and phenomena.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.10.658651v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Functional Atlas of the Tardigrade Resistome Reveals a Diverse Molecular Toolkit for Extremotolerance</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ito, T., Ota, K., Hishinuma, H., Shimizu, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We develop an AI-driven engine that systematically discovers and prioritizes novel 'guardian' proteins constituting the tardigrade molecular shield, revealing hundreds of novel protein families organized into a diverse and compartmentalized molecular toolkit.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> How organisms survive environmental extremes that push life to its physical limits is a fundamental question in biology. Tardigrades are a paradigm for this resilience, entering a state of suspended animation called cryptobiosis to withstand near-complete desiccation and intense radiation. While a few key effectors, including the DNA-shielding protein Dsup and various tardigrade-specific intrinsically disordered proteins (TDPs) have been identified, they constitute only a fraction of a vast, uncharacterized proteome, fundamentally limiting a systems-level understanding of this remarkable biology. Here we overcome this limitation by developing AEGIS, an AI-driven engine that systematically discovers and prioritizes novel 'guardian' proteins constituting the tardigrade molecular shield. Applying AEGIS to the tardigrade proteome, we construct the first functional atlas of the tardigrade 'resistome', revealing hundreds of novel protein families organized into a diverse and compartmentalized molecular toolkit. This atlas pinpoints new Dsup-like nuclear proteins, suggesting multi-layered genome protection, and a rich cohort of cytoplasmic proteins poised to form protective biological glasses. Our work provides a foundational resource for understanding the evolution of extremotolerance and serves as a blueprint for engineering novel biomolecules with transformative biotechnological potential.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.10.658754v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Systematic characterization of the ovarian landscape across mouse menopause models</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kim, M., Bhala, R., Wang, J., Lu, R. J., Lee, E. H., Alvarenga, J. L., Williams, R. G., Benayoun, B. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We compare three mouse models of menopause, including intact aging, chemical ovarian follicle depletion, and Foxl2 haploinsufficiency, a genetic model linked to human premature ovarian failure, and uncover hormonal and immune alterations not captured by intact aging alone.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Menopause not only affects fertility but also has widespread impact on systemic health. Yet, the molecular mechanisms underlying this process are not fully understood, partly due to the absence of robust, age-relevant preclinical models with comprehensive molecular and phenotypic characterization. To address this, we systematically compared three candidate mouse models of menopause: (1) intact aging, (2) chemical ovarian follicle depletion using 4-vinylcyclohexene diepoxide (VCD) administered at multiple ages, and (3) Foxl2 haploinsufficiency, a genetic model based on a transcription factor linked to human premature ovarian failure. Through histology, serum hormone profiling, single-cell transcriptomics and machine-learning approaches, we uncovered both shared and model-specific features of follicle loss, endocrine disruption, and transcriptional remodeling. The VCD and Foxl2 haploinsufficiency models revealed distinct patterns of hormonal and immune alterations not captured by intact aging alone. This comparative framework enables informed selection of context-appropriate preclinical rodent models to study menopause and the broader physiological consequences of ovarian aging.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.10.658920v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Unraveling cooperative and competitive interactions within protein triplets in the human interactome</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Vagiona, A.-C., Mier, P., Andrade, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A computational framework to predict cooperative and competitive interactions among protein triplets in the human Protein Interaction Network (hPIN). By embedding the hPIN into two-dimensional hyperbolic space using the LaBNE+HM algorithm, we extracted latent geometric coordinates for each protein.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Knowledge about protein-protein interactions (PPIs) is crucial for our understanding of cellular functions. An important contribution to these comes from experimental high-throughput techniques such as yeast two-hybrid (Y2H), which provide evidence for direct, binary PPIs. Consequently, although protein function often emerges from interactions within multi-protein assemblies, most PPI networks focus on binary relationships. Higher-order motifs, such as protein triplets, can reflect cooperative or competitive binding events crucial for cellular function. However, distinguishing these interaction types systematically from a network of binary PPIs remains challenging. To address this issue, we present a computational framework to predict cooperative and competitive interactions among protein triplets in the human Protein Interaction Network (hPIN). By embedding the hPIN into two-dimensional hyperbolic space using the LaBNE+HM algorithm, we extracted latent geometric coordinates for each protein. Combined with topological features and biological annotations, these were used to train a Random Forest classifier on structurally validated triplets from Interactome3D. Our model achieved strong performance (AUC = 0.88), identifying angular and hyperbolic distances as two of the most predictive features. Cooperative triplets were characterized by longer common proteins and a lower frequency of paralogous partners, indicating distinct structural and evolutionary features compared to competitive cases. Structural evaluation using AlphaFold 3 confirmed that cooperative triplets exhibit distinct binding sites, whereas competitive ones share overlapping interfaces. Hyperbolic embeddings, in combination with machine learning, offer a powerful approach to characterize higher-order interaction motifs in protein networks. Our findings provide mechanistic insights into the organization of protein complexes and a predictive framework to support future studies in network biology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.10.658265v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Dissecting the biological impact of GBA1 mutations using multi-omics in an isogenic setting</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Alvarez Jerez, P., Wild Crea, P. A., Patel, D., Glasstetter, L. M., Alvarez, C., Makarious, M. B., Lara, E., Chen, Y., Pantazis, C. B., Paquette, K., Malik, L., Nalls, M. A., Reed, X., Singleton, A. B., Billingsley, K. J., Mcdonough, J. A., Ning, G., Skarnes, W. C., Ryten, M., Sidransky, E., Cookson, M. R., Beilina, S., Blauwendraat, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We examine the downstream consequences of GBA1 loss of function in iPSC and identify limited overlapping results across cell types suggesting different downstream effects.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> GBA1 is a risk gene for multiple neurodegenerative diseases, including Lewy Body Dementia and Parkinsons disease, and biallelic pathogenic variants in the gene result in the lysosomal storage disorder Gaucher disease. GBA1 encodes the enzyme glucocerebrosidase (GCase), and alterations in the gene result in reduced enzymatic activity, which affects lysosome function downstream. Induced pluripotent stem cells (iPSCs) are a useful tool for testing the functional consequences of gene variants in an isogenic setting. Additionally, they can be used to perform multiomic studies to explore biological effects independent of disease mechanisms. Using CRISPR-edited isogenic KOLF2.1J iPSC lines containing pathogenic GBA1 variants D409H (p.D448H), D409V (p.D448V) and GBA1 knockout line generated by the iPSC Neurodegenerative Disease Initiative (iNDI), we examined potential molecular mechanisms and downstream consequences of GCase reduction. In this study, we confirm that this isogenic series behaves as expected for loss of function variants, despite the known difficulties with GBA1 editing. We identified that there are limited overlapping results across cell types suggesting potential different downstream effects caused by GBA1 variants. Additionally, we note that RNA-based quantitation may not be the best method to characterize GCase mechanisms, but protein and metabolomic analyses may be used to evaluate differences across genotypes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.12.656971v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Cryo-EM Structure of Photosystem I from mangroves in their natural environment</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Malavath, T., Heilmann, E., Charras, Q., Yuan, X., Al-Amoudi, A., Naschberger, A., Gholami, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a cryo-EM structure of the PSI-LHCI supercomplex from Avicennia marina, a halotolerant, dominant mangrove species native to the Arabian Gulf that thrives in harsh coastal environments.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Photosystem I (PSI) is a multi-subunit pigment-protein supercomplex responsible for light driven electron transport in photosynthesis. Its structural plasticity under natural environmental stress remains poorly understood. Here, we present a 2.1 angstrom cryo-EM structure of the PSI-LHCI supercomplex from Avicennia marina, a halotolerant, dominant mangrove species native to the Arabian Gulf that thrives in harsh coastal environments. Unlike structures derived from laboratory grown specimens, our analysis captures PSI directly from wild-grown plant leaves, offering an in-situ snapshot of plant adaptation. The core architecture comprising twelve protein subunits and four Lhca antenna proteins coordinating 157 chlorophylls, 36 carotenoids and 24 lipids is highly conserved relative to model angiosperms yet exhibits subtle shifts in antenna core coupling and pigment organization. Red-shifted chlorophylls in Lhca3, conserved among far red adapted plants, suggest an ancestral photoprotective mechanism retained in high light, high-salinity environments. Lipid mediated stabilization at interfaces likely counteracts salinity-induced membrane fluidity changes enhancing resilience under fluctuating light and salt stress. Comparative analysis with PSI structures from other vascular plants highlights both evolutionary conservation and distinct ecological tuning of LHCI organization and cofactor binding in mangroves. These insights reveal molecular strategies enabling A. marina to maintain efficient photosynthesis under extreme conditions, offering strategies for engineering photosystems in crops.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.10.658896v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">EARLY RAINY SEASON PATTERNS OF FERN DIVERSITY AND ECOLOGICAL DISTRIBUTION IN AMURUM FOREST RESERVE, JOS PLATEAU.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tumba, C. Y., Samuel, G., Ibrahim, I., Abok, C. J., Cleophas, A. B., Chaskda, A. A., Diffa, A. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Ferns are crucial to ecosystem functioning and biodiversity conservation, yet there is a significant gap in comprehensive data regarding its diversity and distribution in Amurum forest reserve.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Ferns are crucial to ecosystem functioning and biodiversity conservation, yet there is a significant gap in comprehensive data regarding its diversity and distribution in Amurum forest reserve. This research unveils the distribution and diversity of ferns in Amurum forest reserve. Systematically, 20 plots of 10 X 10 meters were established in the study sites with each plot 50 m away from each other and three sub-quadrants were established in every 10 x10 meter plots making a total of 60 sub-quadrants. Ferns species within reach were counted, photographed, recorded and identified. Coordinates of each plot were recorded using GPS (Global Positioning System). Seven species belonging to five families were recorded in the study sites with a Shannon-Weiner diversity Index of 0.6325. Nephrolepis undulate was the most abundant ferns species. All fern species recorded in this study were found and distributed only in the gallery forest of the Amurum forest reserve and not the other habitat types. The study has revealed a rich and varied fern flora during early rainy season, with significant ecological and conservation value in Amurum forest reserve. The study underscores the importance of preserving fern-rich habitats, particularly in the face of environmental change. By collaborating with conservation authorities and conducting further research, we can better understand and protect these ancient plants, ensuring their continued contribution to the region's biodiversity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.10.659003v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Structural Determinants and Biochemical Characterization of LORELEI as a GPI-Anchored Protein</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Palanivelu, R., Wang, Y., Liu, X., Desnoyer, N. J., Howard, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We demonstrate that LORELEI, a putative glycosylphosphatidylinositol-anchored protein involved in pollen tube reception and early seed development in Arabidopsis thaliana, has lacked direct biochemical evidence confirming its glycosymptom.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Glycosylphosphatidylinositol-anchored proteins play critical roles in plant development, reproduction, and environmental responses. However, their biochemical characterization in plants remains limited. LORELEI, a putative glycosylphosphatidylinositol-anchored protein involved in pollen tube reception and early seed development in Arabidopsis thaliana, has lacked direct biochemical evidence confirming its glycosylphosphatidylinositol anchoring. This study employed biochemical approaches to validate LORELEI as a glycosylphosphatidylinositol-anchored protein. We demonstrate that ectopic expression of wild-type LORELEI fused to a reporter protein in vegetative tissues associates with a detergent-resistant membrane fraction and is sensitive to glycosylphosphatidylinositol-specific cleavage enzymes, thereby confirming that it is a glycosylphosphatidylinositol-anchored protein. Moreover, we show that mutations in the {omega}-sites or the glycosylphosphatidylinositol attachment signal of LORELEI disrupt its membrane localization, highlighting the necessity of these structural elements for proper glycosylphosphatidylinositol anchoring. Analysis of a reporter-fused LORELEI protein in the loss of function mutant of glycosylphosphatidylinositol 8, which encodes the catalytic subunit of glycosylphosphatidylinositol transamidase complex involved in glycosylphosphatidylinositol anchor addition, results in the appearance of prominent higher molecular weight bands, further supporting its role in glycosylphosphatidylinositol anchoring of the LORELEI protein. This study provides direct biochemical evidence confirming LORELEI as a glycosylphosphatidylinositol-anchored protein and sheds light on the structural determinants required for its glycosylphosphatidylinositol anchoring. Additionally, our study demonstrates that heterologous ectopic expression of LORELEI in vegetative tissues provides a viable alternative for the biochemical characterization of glycosylphosphatidylinositol-anchored proteins predominantly expressed in hard-to-access and small-sized female gametophytes. Our findings underscore the role of GPI anchoring in membrane localization and biosynthesis of GPI-anchored proteins in plants.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.10.659000v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">NanoBondy reacting through NeissLock anhydride allows covalent immune cell decoration</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Vankayala, L. R., Adoni, K. R., Lim, S. Y., Dushek, O., Thalassinos, K., Howarth, M. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the NanoBondy, a nanobody modified for covalent ligation to an endogenous protein target at the cell-surface.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cell-surface conjugation has enormous therapeutic and research potential. Existing technologies for cell-surface modification are usually reversible, non-specific, or rely on genetic editing of target cells. Here we present the NanoBondy, a nanobody modified for covalent ligation to an endogenous protein target at the cell-surface. The NanoBondy utilizes the 20 naturally occurring amino acids, harnessing NeissLock chemistry engineered from Neisseria meningitidis. We evaluated binding and specificity of a panel of nanobodies to CD45, a long-lived surface marker of nucleated hematopoietic cells. We demonstrated conversion of existing nanobodies to covalently reacting NanoBondies using a disulfide clamp to position the self-processing module of FrpA close to the nanobody antigen-binding site. Addition of calcium induces anhydride formation at the NanoBondy C-terminus and proximity-directed ligation to surface amines on CD45. We optimized NanoBondy reaction by fine-tuning linkers and disulfide clamp sites to modulate anhydride positioning. Tandem mass spectrometry mapped reaction sites between the NanoBondy and CD45. NanoBondy ligation was robust to buffer, pH and temperature and was detected within 2 minutes. We established reaction specificity of NanoBondies to endogenous CD45 at the surface of NK cells and T cells. NanoBondy technology provides a modular approach for targeted, inducible and covalent cell-surface modification of immune cells.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.14.659678v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

