<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Sunday, 20 April 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">EBMGP: a deep learning model for genomic prediction based on Elastic Net feature selection and bidirectional encoder representations from transformer's embedding and multi-head attention pooling.</h2>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Enhancing early selection through genomic estimated breeding values is pivotal for reducing generation intervals and accelerating breeding programs. Recently, deep learning (DL) approaches have gained prominence in genomic prediction (GP). Here, we introduce a novel DL framework for GP based on Elastic Net feature selection and bidirectional encoder representations from transformer's embedding and multi-head attention pooling (EBMGP). EBMGP applies Elastic Net for the selection of features, thereby diminishing the computational burden and bolstering the predictive accuracy. In EBMGP, SNPs are treated as "words," and groups of adjacent SNPs with similar LD levels are considered "sentences." By applying bidirectional encoder representations from transformers embeddings, this method models SNPs in a manner analogous to human language, capturing complex genetic interactions at both the "word" and "sentence" scales. This flexible representation seamlessly integrates into any DL network and demonstrates a marked improvement in predictive performance for EBMGP and SoyDNGP compared to the widely used one-hot representation. We propose multi-head attention pooling, which can adaptively assign weights to features while learning features from multiple subspaces through multi-heads for a high level of semantic understanding. In a comprehensive comparative analysis across four diverse plant and animal datasets, EBMGP outperformed competing models in 13 out of 16 tasks, achieving accuracy gains ranging from 0.74 to 9.55% over the second-best model. These results underscore EBMGP's robustness in genomic prediction and highlight its potential for deep learning applications in life sciences.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40253568/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Microbiome data management in action workshop: Atlanta, GA, USA, June 12-13, 2024.</h2>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Microbiome research is revolutionizing human and environmental health, but the value and reuse of microbiome data are significantly hampered by the limited development and adoption of data standards. While several ongoing efforts are aimed at improving microbiome data management, significant gaps still remain in terms of defining and promoting adoption of consensus standards for these datasets. The Strengthening the Organization and Reporting of Microbiome Studies (STORMS) guidelines for human microbiome research have been endorsed and successfully utilized by many research organizations, publishers, and funding agencies, and have been recognized as a consensus community standard. No equivalent effort has occurred for environmental, synthetic, and non-human host-associated microbiomes. To address this growing need within the microbiome research community, we convened the Microbiome Data Management in Action Workshop (June 12-13, 2024, in Atlanta, GA, USA), to bring together key decision makers in microbiome science including researchers, publishers, funders, and data repositories. The 50 attendees, representing the diverse and interdisciplinary nature of microbiome research, discussed recent progress and challenges, and brainstormed actionable recommendations and paths forward for coordinated environmental microbiome data management and the modifications necessary for the STORMS guidelines to be applied to environmental, non-human host, and synthetic microbiomes. The outcomes of this workshop will form the basis of a formalized data management roadmap to be implemented across the field. These best practices will drive scientific innovation now and in years to come as these data continue to be used not only in targeted reanalyses but in large-scale models and machine learning efforts.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40253432/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Large-scale identification of microsatellite loci from multiple otter (Mammalia, Carnivora, Lutrinae) species using whole genome sequence data</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">The development of molecular studies on elusive, rare, and/or poorly known species faces challenges due to the lack of suitable markers. Species-specific microsatellite markers minimize bias, offer better performance, and are cost-effective, aiding the development of population genetic studies. The use of whole-genome sequences allows for the development of species-specific microsatellite markers and their survey in closely related species, enabling the discovery of shared markers that can facilitate comparative studies. Lutrinae includes 14 extant species of otters. Despite their worrisome conservation status, due to inherent characteristics of these species that make their study difficult by traditional methods, many of them lack reliable population genetic data, limiting conservation efforts. In this study, we employed a multi-taxon approach to identify a large number of novel microsatellite loci for 11 of the 14 otter species, assessing whether the identified loci were shared among different taxa. We identified a total of 23,320 microsatellite loci across 11 species, which were reduced to 12,573 after stringent filtering criteria. Primer design was completed successfully for 420 and 259 unique loci, considering two minimum melting temperatures. We validated marker efficiency by testing the 81 loci designed for two Asian species. Of these, 51 loci yielded reliable microsatellite genotypes in both species, with 34 showing allelic variation in at least one of them. These results demonstrate that these markers are applicable in empirical genotyping for both their target species and closely related ones.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.14.648774v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MegaPX: fast and space-efficient peptide assignment method using IBF-based multi-indexing</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">A central problem for metaproteomic analysis is the often-unknown taxonomic composition of the analyzed microbiomes. Using a database search, the standard approach requires prior knowledge of which proteins and taxa to include in the protein reference database or to use tailored metagenome-derived databases, which are expensive and error-prone in their generation. A possible strategy to circumvent this database search issue is de novo sequencing, where peptide sequences are directly identified from mass spectra. However, these sequences must still be mapped back to potentially extensive databases. Here, alignment-based approaches enable robust and precise results, with the potential drawback of high memory usage and long run times. We present MegaPX, a software for rapidly classifying de novo peptide sequences against large protein databases. MegaPX implemented as a C++-based tool, uses an alignment-free, k-mer-based approach as a taxonomic classification method with the possibility of generating mutated reference databases for error-tolerant searching. It uses various algorithms, including interleaved Bloom filters, to efficiently compute approximate membership queries, ensuring fast processing times while querying and indexing large databases in a multi-indexing fashion. We demonstrate the potential of MegaPX by analyzing different samples, including metaproteomics, against extensive reference databases, highlighting its use as a fast screening tool.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.14.648734v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Plasmodium Repetome: A mysterious space with a wealth of information</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Eukaryotic proteomes harbour repetitive stretches of amino acids that may play critical roles in the biology of that organism. While several tandem repeats (TR) are known to contribute to protein structure and function, information about the vast majority of repeat regions remains obscure. In this article, we have analysed the repeat content of different Plasmodium species and found the leading human malaria-causing P. falciparum (Pf) and P. vivax to be exceptionally rich in TR regions (>40% TR containing proteins). Detailed analysis of Pf repetome showed this intracellular parasite to carry longer TRs, several of which were present in exported proteins important for parasite survival and immune evasion. The repeat regions of Pf were enriched in acidic amino acids and asparagine (Asn), where Asn was more abundant in short and intermediate TRs, suggesting an evolutionary bias influenced by replication slippage and positive selection. Gene ontology analysis of TR containing Pf proteins helped us to understand their cellular localization along with the molecular and biological processes they are involved in. The Pf variable surface antigen families with roles in important pathogenic processes like cytoadherence, immune evasion etc. had low repeat content present within seroreactive peptides. Three-dimensional structure predictions of TR regions revealed several repeats to adopt ordered super-secondary conformations that are known to facilitate intermolecular interactions. Overall, this is a comprehensive study attempting to gain insights on the importance of TRs in malaria parasite biology and suggests a novel route to understanding protein function through the characterization of repeat content.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.14.647515v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">HemeFinder: a Computational Predictor for Heme-Binding Sites in Proteins</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">HemeFinder has been developed to predict heme-binding sites in natural and heme-dependent de novo enzymes. This tool relies on the structural and physicochemical characteristics of heme-binding sites, including shape, residue composition, and geometric descriptors. HemeFinder benchmarks more than 94% accuracy in identifying the correct heme location, considering the complete set of solutions, and 72% accuracy for the proper location with the correct iron-coordinating residues among the three best-ranked solutions. HemeFinder performs within seconds for monomeric systems and takes minutes for larger multimeric ones, demonstrating that its speed does not compromise its performance. An illustrative case of its potential is provided. HemeFinder is applied to the heme carrier protein 1 (HCP1), a transmembrane protein involved in heme recruitment in evolved organisms, for which no ligand-bound structures have been revealed. HemeFinder provides a relevant prediction of porphyrin binding and, when combined with protein-ligand docking, offers the first evidence of low-energy Heme-HCP1 complexes and unveils possible heme pathways. HemeFinder is an interesting, fast, and accurate tool for identifying heme-binding sites in proteins.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.14.648773v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">mspms: A Comprehensive R Package and Graphic Interface for Multiplex Substrate Profiling by Mass Spectrometry Analysis</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Multiplex Substrate Profiling by Mass Spectrometry (MSP-MS) is a powerful method for determining the substrate specificity of proteolytic enzymes, knowledge key for developing protease inhibitors, diagnostics, and protease-activated therapeutics. However, the complex datasets generated by MSP-MS pose significant analytical challenges. To address this, we developed mspms, a Bioconductor R package complemented by an intuitive graphical interface. Mspms streamlines MSP-MS data analysis by standardizing workflows for data preparation, processing, statistical analysis, and visualization. Designed for accessibility, it serves both advanced users via the R package and broader audiences through the web interface. We validated mspms by profiling the substrate specificity of four well-characterized cathepsins (A-D), demonstrating its ability to reliably capture expected substrate specificities. As the first publicly available platform for MSP-MS data analysis, mspms delivers comprehensive functionality, transparency, and ease of use, making it a valuable resource for the protease research community. Access to mspms is available through the Bioconductor project at https://bioconductor.org/packages/mspms, and a graphic interface is available at https://gonzalezlab.shinyapps.io/mspms_shiny/.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.14.648679v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Estimating drivers of breast tissue transitions from normal to tumor state</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Tumor tissues are characterized by dysregulated gene expression patterns leading to altered cellular pathways and molecular functions as a result of their transition from normal to tumor state. Further, tumor-adjacent normal tissues (TANTs), utilized as a control in cancer research, are not molecularly normal and differ from healthy normal tissues. These TANTs represent a distinctive transitional state betwixt normal and tumor states. However, the mechanism underlying this state transition, expression dysregulation, and perturbed regulation remain largely unexplored and hence elusive. Herein, the transitions of breast tissues from normal and TANT to tumor states were modeled using gene expression and regulation data to estimate key drivers underlying these transitions. As a result, we identified 645 shared driver genes underlying the transitions of breast tissues from the healthy normal state to the adjacent normal and tumor states. Besides, we identified 635 shared driver genes underlying the transitions of TANTs to different subtypes. When we intersected both lists of shared driver genes, a total of 615 commonly shared driver genes across the state transitions were observed. Subsequently, functional annotations of these driver genes revealed their involvement in the growth and maintenance-related activity of cells. Additionally, key pathways associated with cancer pathogenesis, such as Wnt signaling, Notch signaling, NF-kappa B signaling, and PD-L1 expression and PD-1 checkpoint pathway in cancer, were found significantly enriched with these shared driver genes. Thus, the shared driver genes identified across tissue transitions provide ways forward to devise more efficient diagnostic and therapeutic strategies for early and effective disease management.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.14.648655v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Bulk-based hypothesis weighing to increase power in single-cell differential expression analysis</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Due to the costs of single-cell sequencing, sample sizes are often relatively limited, sometimes leading to poorly reproducible results. In many contexts, however, larger bulk RNAseq data is available for the same conditions or experimental paradigm, which can be used as additional evidence of a generalizable differential expression pattern. Here, we show how such data can be used, via bulk-based hypothesis weighing (bbhw), to increase the power and robustness of single-cell differential state analysis. We find that all methods improve performance, with the best results obtained by applying a grouped Benjamini-Hochberg procedure on bins based on proportion-adjusted significance (PAS). These methods are implemented in the muscat package, and should be applicable to a broader range of scenarios.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.15.648932v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Uncovering Superior Alleles and Genetic Loci for Yield-Related Traits in Mungbean (Vigna radiata L. Wilczek) Through Genome-Wide Association Study</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Mungbean is an important legume crop in South and Southeast Asia and Australia in terms of area coverage and production. However, productivity remains low due to limited genetic diversity, necessitating the dissection of the genetic basis of quantitatively inherited yield-related traits to develop stable and high-yielding varieties. In the current study, a total of 296 mungbean minicore germplasm accessions from the World Vegetable Centre were evaluated over three years in Bangladesh to assess their genetic diversity and local adaptation. Out of the 296 accessions, 206 produced flowers and yield, showing significant genetic variation in six yield-related traits: days to flowering (DF), days to maturity (DM), plant height (PH), pods per plant (PODS), 100-seed weight (HSW), and seed yield (YLD). Moderate to high broad-sense heritability was exhibited for all phenotypic traits, including DF (72%) and HSW (62%). Genome-wide association study (GWAS) was conducted using the 4,307 high-quality SNPs obtained from the genotyping by sequencing method and found 16 genetic loci across the six mungbean chromosomes associated with the six traits. Further, we selected 21 superior germplasm based on multi-trait stability index including four accessions with a higher number of favorable alleles (10). We also employed genomic prediction models and found a moderate prediction accuracy (>30%) for the HSW and YLD. These results will assist in incorporating important alleles into the elite mungbean germplasm through marker-assisted breeding and/or genomic prediction for improving mungbean yield.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.15.648935v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Before the Black Box</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Ecdysteroid hormones are responsible for regulating arthropod moulting, but unlike other taxa which used sterol-derived hormones (e.g. vertebrate sex-determining hormones), arthropods cannot synthesis sterols de novo. Ecdysteroid synthesis is thus dependent on funnelling diverse dietary sterols into the conserved reactions of ecdysteroidogenesis. While the later steps of ecdysteroid synthesis are dominated by cytochrome P450s, earlier stages of this pathway (i.e. those acting in or before the Black Box in which 7-dehydrocholesterol is converted to 5,22,25-trideoxyecdysone) involve more diverse enzymes. While numerous comparative analysis of arthropod cytochrome P450s have been conducted, little comparable work has been done on the pre-Black Box FAD-dependent oxidoreductase DHCR24 and the Rieske domain oxygenase Neverland and the Black Box short-chain dehydrogenase/reductase Shroud. However, there is some evidence for evolutionary shifts among these enzymes across arthropods, e.g. the absence of DHCR24 from Drosophila melanogaster. We have sought to systematically identify such changes in copy number in DHCR24, Neverland and Shroud across arthropods, as well as to identify correlations between these changes and dietary factors with reference to the literature. Among other results, we find that Neverland is absent from all beetles (providing support for mycophagy as an important element of the beetle ancestral diet) and that losses of DHCR24 correlate with production of alkylated ecdysteroids.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.14.648839v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">THLANet: A Deep Learning Framework for Predicting TCR-pHLA Binding in Immunotherapy Applications</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Adaptive immunity is a targeted immune response that enables the body to identify and eliminate foreign pathogens, playing a critical role in the anti-tumor immune response. Tumor cell expression of antigens forms the foundation for inducing this adaptive response. However, the human leukocyte antigens (HLA)-restricted recognition of antigens by T-cell receptors (TCR) limits their ability to detect all neoantigens, with only a small subset capable of activating T-cells. Accurately predicting neoantigen binding to TCR is, therefore, crucial for assessing their immunogenic potential in clinical settings. We present THLANet, a deep learning model designed to predict the binding specificity of TCR to neoantigens presented by class I HLAs. THLANet employs evolutionary scale modeling-2 (ESM-2), replacing the traditional embedding methods to enhance sequence feature representation. Using scTCR-seq data, we obtained the TCR immune repertoire and constructed a TCR-pHLA binding database to validate THLANets clinical potential. The models performance was further evaluated using clinical cancer data across various cancer types. Additionally, by analyzing divided complementarity-determining region (CDR3) sequences and simulating alanine scanning of antigen sequences, we unveiled the 3D binding conformations of TCRs and antigens. Predicting TCR-neoantigen pairing remains a significant challenge in immunology, THLANet provides accurate predictions using only the TCR sequence (CDR3{beta}), antigen sequence, and class I HLA, offering novel insights into TCR-antigen interactions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.14.648678v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Ploidy variation modulates outbreeding response and promotes mating system evolution in a selfing plant lineage.</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Outbreeding response, the phenotypic differences observed between selfed parental lines and their outcrossed offspring, can influence the evolution of selfing strategies. However, such effect remains poorly understood in non-crop species. We investigated the phenotypic outbreeding response variation across ploidy levels in Erysimum incanum, a predominantly selfing plant complex with diploid, tetraploid, and hexaploid populations distributed across the Iberian Peninsula and Morocco. We performed controlled within-population crosses to generate offspring with varying heterozygosity levels across ploidy types. We quantified individual, flower, and reproductive traits, and we estimated fitness components, and assessed trait modularity and phenotypic integration to see how heterozygosity affects trait coordination. Tetraploids showed the strongest and most consistently positive outbreeding responses, particularly in gamete production. Trait-specific outbreeding responses were positively associated with fitness across ploidy levels. Increasing heterozygosity was linked to a reduction in phenotypic integration, suggesting a loosening of trait correlations. Our results show that outbreeding response is ploidy-dependent and functionally connected to fitness. This suggests it may act as a selective force promoting outcrossing in highly inbred lineages. We suggest that outbreeding response is a dynamic and evolvable trait, with implications for mating system transitions and diversification in selfing plant populations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.15.648901v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">TestNet: A Testing Method for Inferring Microbial Networks with False Discovery Rate Control, for Clustered and Unclustered Samples</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Most existing methods for inferring microbial networks generate only point estimates of Pearson's correlations without assessing their significance, and none can account for clustered samples. In this article, we introduce TestNet, a novel method that delivers well calibrated results by controlling the false discovery rate (FDR). We developed a permutation-based procedure to generate valid null replicates that account for compositional effects and extensive zeros in microbiome data, and clustering structures within the samples when present. Our results demonstrate that TestNet is the only method that effectively controls the FDR while maintaining high power across a wide range of scenarios.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.14.648862v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Characterizing Clinical Toxicity in Cancer Combination Therapies</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Predicting synergistic cancer drug combinations through computational methods offers a scalable approach to creating therapies that are more effective and less toxic. However, most algorithms focus solely on synergy without considering toxicity when selecting optimal drug combinations. In the absence of combinatorial toxicity assays, a few models use toxicity penalties to balance high synergy with lower toxicity. However, these penalties have not been explicitly validated against known drug-drug interactions. In this study, we examine whether synergy scores and toxicity metrics correlate with known adverse drug interactions. While some metrics show trends with toxicity levels, our results reveal significant limitations in using them as penalties. These findings highlight the challenges of incorporating toxicity into synergy prediction frameworks and suggest that advancing the field requires more comprehensive combination toxicity data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.13.648641v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">JarrVis: Visualising Taxa-function relationships from meta-omic data</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">High-throughput sequencing has necessitated and facilitated the development of various computational tools for dissecting the taxonomic structure and molecular function of microbial communities residing in diverse ecological niches. A wide variety of tools and protocols have been developed to process amplicon sequencing as well as meta-omic (metagenomic, metatranscriptomic and genomic) data to generate relative abundances of taxonomic groups or functional categories on a per-sample basis. A key output from many of these protocols is a stratified relative abundance table that specifies the taxonomic contribution to each function. However, there are a very few tools which can effectively visualize the different taxa-function relationships from these stratified outputs. Here we introduce an R Shiny application called JarrVis (Just Another stRatified Rpkm VISualizer) which can visualize the taxa-function relationships resulting from different types of microbiome data. JarrVis can visualize relationships between samples (which can be combined based on metadata categories), the taxa that are detected in these samples (at any given taxonomic level) and the functions encoded by these taxa in an interactive interface. We utilized JarrVis to visualize and examine taxa-function relationships in 1) a 16S amplicon time series spanning 4 years with samples collected weekly, 2) functions associated with microbial cobalamin biosynthesis and uptake in metagenome assembled genomes from marine metagenomes and 3) a set of metatrascriptomes from gut samples of patients with Crohns Disease, Ulcerative Colitis and non-IBD controls. Our analysis was able recapitulate already published taxa-function relationships as well as discover novel insights from these publicly available datasets. JarrVis and related scripts and data are available at https://github.com/dhwanidesai/JarrVis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.13.648596v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

