<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Wednesday, 20 August 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A plasma proteomics-based candidate biomarker panel predictive of amyotrophic lateral sclerosis.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ruth Chia, Ruin Moaddel, Justin Y Kwan, Memoona Rasheed, Paola Ruffo, Natalie Landeck, Paolo Reho, Rosario Vasta, Andrea Calvo, Cristina Moglia, Antonio Canosa, Umberto Manera, Allison Snyder, Sara Saez-Atienzar, Maurizio Grassano, Maura Brunetti, Federico Casale, Anindita Ray, Kumar Arvind, Betul Comertpay, Min Zhu, J Raphael Gibbs, Camille Alba, Ted M Dawson, Liana S Rosenthal, Anna J Hall, Alexander Y Pantelyat, Derek P Narendra, Debra J Ehrlich, Keenan A Walker, Peter Kosa, Bibiana Bielekova, Josephine M Egan, Julián Candia, Toshiko Tanaka, Luigi Ferrucci, Clifton L Dalgard, Sonja W Scholz, Adriano Chiò, Bryan J Traynor</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We found that plasma proteins are differentially abundant in the plasma of patients with ALS and controls, and used this information to develop a model that diagnosed ALS with high accuracy.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Identifying a reliable biomarker for amyotrophic lateral sclerosis (ALS) is crucial for clinical practice. Here, in this cross-sectional study, we used the Olink Explore 3072 platform to investigate plasma proteomics as a biomarker tool for this neurodegenerative condition. Thirty-three proteins were differentially abundant in the plasma of patients with ALS (n = 183) versus controls (n = 309). We replicated our findings in an independent cohort (n = 48 patients with ALS and n = 75 controls). We then applied machine learning to create a model that diagnosed ALS with high accuracy (area under the curve, 98.3%). By analyzing plasma samples from individuals before ALS symptoms emerged, we estimated the age of clinical onset and showed that the disease process-impacting skeletal muscle, nerves and energy metabolism-occurs years before symptoms appear. Our research suggests that plasma proteins can be a biomarker for this fatal disease and offers molecular insights into its prodromal phase.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40830661/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Serological proteomic characterization for monitoring liver fibrosis regression in chronic hepatitis B patients on treatment.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mengyang Zhang, Shuyan Chen, Xiaoning Wu, Jialing Zhou, Bingqiong Wang, Tongtong Meng, Rongxuan Hua, Yameng Sun, Hong You, Wei Chen</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Longitudinal serological proteomic dynamics during antiviral therapy (AVT) in chronic hepatitis B (CHB) patients with liver fibrosis remain poorly characterized.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Longitudinal serological proteomic dynamics during antiviral therapy (AVT) in chronic hepatitis B (CHB) patients with liver fibrosis remain poorly characterized. Here, using four-dimensional data-independent acquisition mass spectrometry (4D-DIA-MS), paired liver biopsy (LBx)-proven serum samples from 130 CHB liver fibrosis patients undergoing short-term (78 weeks) or long-term (260 weeks) AVT are analyzed. Our findings show that prolonged AVT drives progressive serological proteomic remodeling in fibrosis regressors, characterized by a temporal inversion in the activation of the complement and coagulation cascades. Using machine learning algorithms trained on the 4D-DIA-MS discovery cohort, we develop a logistic regression model incorporating a seven-protein panel for short-term AVT and a three-protein panel for long-term AVT, respectively, both of which demonstrate moderate discriminatory capabilities for fibrosis regression. Subsequent external validation in an independent cohort (n = 54) with serial LBx assessments at baseline, 78 weeks, and 260 weeks, where serological proteins are quantified using parallel reaction monitoring mass spectrometry (PRM-MS), further confirms their generalizability. Furthermore, our longitudinal trajectory analysis highlights that the long-term proteomic signature exhibits greater stability compared to the short-term panel. This study proposes and validates duration-adapted serological proteomic panels as non-invasive tools for monitoring histological fibrosis regression in on-treatment CHB patients.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40830346/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A deep learning model to predict glioma recurrence using integrated genomic and clinical data.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jessica A Patricoski-Chavez, Seema Nagpal, Ritambhara Singh, Jeremy L Warner, Ece D Gamsiz Uzun</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new model for predicting early glioma recurrence, which is critical for optimizing patient outcomes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Gliomas account for approximately 25.5% of all primary brain and central nervous system (CNS) tumors and 80.8% of malignant brain and CNS tumors. The prognosis varies considerably; patients with low-grade gliomas (LGGs) have 5-year survival rates of up to 80%, while patients with higher-grade gliomas (HGGs) often experience rates below 5%. Recurrence is a common challenge, occurring in 52% to 62% of patients with LGGs and 90% of patients with HGGs, complicating clinical management and treatment planning. Currently, no widely available models exist for reliably predicting early glioma recurrence, which is critical for optimizing patient outcomes. Machine learning (ML) and deep learning (DL) techniques have shown promise in predicting recurrence for various cancers, with those utilizing multimodal data sources showing increasing promise.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40830273/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">TrimNN: characterizing cellular community motifs for studying multicellular topological organization in complex tissues.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yang Yu, Shuang Wang, Jinpu Li, Meichen Yu, Kyle McCrocklin, Jing-Qiong Kang, Anjun Ma, Qin Ma, Dong Xu, Juexin Wang</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the triangulation cellular community motif neural network (TrimNN), a graph-based deep learning framework designed to identify conserved spatial cell organization patterns, termed cellular community (CC) motifs, from spatial transcriptomics and proteomics data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The spatial organization of cells plays a pivotal role in shaping tissue functions and phenotypes in various biological systems and diseased microenvironments. However, the topological principles governing interactions among cell types within spatial patterns remain poorly understood. Here, we present the triangulation cellular community motif neural network (TrimNN), a graph-based deep learning framework designed to identify conserved spatial cell organization patterns, termed cellular community (CC) motifs, from spatial transcriptomics and proteomics data. TrimNN employs a semi-divide-and-conquer approach to efficiently detect overrepresented topological motifs of varying sizes in a triangulated space. By uncovering CC motifs, TrimNN reveals key associations between spatially distributed cell-type patterns and diverse phenotypes. These insights provide a foundation for understanding biological and disease mechanisms and offer potential biomarkers for diagnosis and therapeutic interventions.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40830137/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The chronODE framework for modelling multi-omic time series with ordinary differential equations and machine learning.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Beatrice Borsari, Mor Frank, Eve S Wattenberg, Ke Xu, Susanna X Liu, Xuezhu Yu, Mark Gerstein</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a novel framework for modeling gene-expression and chromatin kinetics from longitudinal studies, based on ordinary differential equations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Many genome-wide studies capture isolated moments in cell differentiation or organismal development. Conversely, longitudinal studies provide a more direct way to study these kinetic processes. Here, we present an approach for modeling gene-expression and chromatin kinetics from such studies: chronODE, an interpretable framework based on ordinary differential equations. chronODE incorporates two parameters that capture biophysical constraints governing the initial cooperativity and later saturation in gene expression. These parameters group genes into three major kinetic patterns: accelerators, switchers, and decelerators. Applying chronODE to bulk and single-cell time-series data from mouse brain development reveals that most genes (~87%) follow simple logistic kinetics. Among them, genes with rapid acceleration and high saturation values are rare, highlighting biochemical limitations that prevent cells from attaining both simultaneously. Early- and late-emerging cell types display distinct kinetic patterns, with essential genes ramping up faster. Extending chronODE to chromatin, we find that genes regulated by both enhancer and silencer cis-regulatory elements are enriched in brain-specific functions. Finally, we develop a bidirectional recurrent neural network to predict changes in gene expression from corresponding chromatin changes, successfully capturing the cumulative effect of multiple regulatory elements. Overall, our framework allows investigation of the kinetics of gene regulation in diverse biological systems.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40830112/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">HPVPool-Seq: a genotype-guided pooling strategy for cost-effective next-generation sequencing detection of HPV integration in cervical samples.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wenyan Guan, Siyuan Liu, Yiqiang Chen, Chengzhuo Chu, Guanghao Peng, Jinhuan Wang, Qiao Weng, Yali Fu, Jingjing Li</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed HPVPool-Seq, an innovative pooling strategy that leverages the inherent diversity of HPV genotypes as natural barcodes, enabling cost-effective, scalable integration detection, balancing cost, scalability, and clinical precision.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Integration of high-risk human papillomavirus (hrHPV) DNA is a critical event in carcinogenesis and a promising biomarker for risk stratification. However, the high cost of next-generation sequencing (NGS) limits its widespread clinical adoption. We developed HPVPool-Seq, an innovative pooling strategy that leverages the inherent diversity of HPV genotypes as natural barcodes, enabling cost-effective, scalable integration detection. Samples were pooled based on qPCR-derived HPV genotypes and viral loads prior to targeted NGS and bioinformatic decoding. A web-based automation tool was implemented to streamline pooling and decoding workflows. In a proof-of-concept study of 175 clinical specimens, HPVPool-Seq achieved 77.1% exact genotype concordance and 97.1% combined sensitivity compared with qPCR. Cost simulations demonstrated a 60% reduction in per-sample sequencing expenses. Self-correcting capability through targeted retesting further enhanced reliability. HPVPool-Seq offers a novel, traceable, and economically viable solution for high-throughput HPV integration profiling, balancing cost, scalability, and clinical precision. This strategy sets a new framework for molecular screening in HPV-associated cancers.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40827881/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrated QTL mapping and CRISPR screening in pooled iPSC-derived microglia reveals genetic drivers of neurodegenerative risk</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Perez-Alcantara, M., Washer, S., Chen, Y., Steer, J., Gonzalez-Padilla, D., McWilliam, J., Wille, D., Panousis, N., Kolberg, P., Navarro Guerrero, E., Alasoo, K., Hall-Roberts, H., Williams, J., Cowley, S. A., Trynka, G., Bassett, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comprehensive framework for interpreting the function of risk loci in neurodegenerative disorders .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mounting evidence implicates microglia in neurodegeneration, but linking disease-associated genetic variants to target genes and cellular phenotypes is hindered by the inaccessibility of these cells. We differentiated 261 human iPSC lines into microglia-like cells (iMGL) in pools with phenotypic (differentiation, phagocytosis and migration) and single-cell transcriptomic readouts. Burden analysis of deleterious variants detected 36 genes influencing microglial phenotypes. Expression quantitative trait locus (eQTL) analysis found 7,121 eGenes, and 79 colocalizations across four neurodegenerative disease GWAS, half of which had limited prior evidence of causality. Integration of eQTL and phenotypic associations highlighted the role of disease-relevant variants including LRRK2 and TREM2 acting via microglial phagocytosis. A coupled CRISPR screen identified a role of TREM2 in phagocytosis and highlighted the importance of cellular state in directionality of phenotype. By contextualizing variant effects within disease-relevant microglial states, we provide a comprehensive framework for interpreting the function of risk loci in neurodegenerative disorders.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.18.670767v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">spa-ChIP-seq enables the comprehensive profiling of chromatin</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cao, Y., Patel, L., Alcoser, L., Mendenhall, E., Benner, C., Heinz, S., Goren, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A fully automated ChIP-seq protocol for chromatin immunoprecipitation followed by sequencing that is scalable from 8 to 96 samples, and a robust, cost-efficient protocol that can be incorporated into antibody evaluation procedures, compound screening, population genomics and diagnostic frameworks.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Chromatin immunoprecipitation followed by sequencing (ChIP-seq) is a well-established method for studying the genomic localization of DNA-associated proteins. Yet, while useful, most ChIP-seq protocols include multiple manual steps that can introduce inconsistency and make it burdensome to analyze large sample sets, limiting the inclusion of appropriate replicates and other controls. Although some of these challenges were addressed by incorporation of liquid handling platforms, most of those previous efforts have automated only a subset of the ChIP-seq steps. Further, using automation for efficiently mapping non-histone proteins, such as chromatin regulators, has been challenging. We recently developed a single-pot ChIP-seq protocol1. Here we established a liquid handler operation for this protocol and created an end-to-end fully automated version that is scalable from 8 to 96 ChIP-seq reactions. Our single-pot automated (spa-ChIP-seq) protocol requires about 3 days from cross-linked cells to library, and the costs to produce indexed libraries is approximately $70 per sample. We first benchmarked spa-ChIP-seq against manual ChIP-seq that was performed in parallel, showing a nearly indistinguishable signal-to-noise ratio between the two workflows. Next, we used spa-ChIP-seq to systematically evaluate multiple parameters including shearing and crosslinking conditions, buffer compositions, and antibody to cell-number ratios. Our method allowed us to identify optimal conditions for double crosslinked chromatin as well as easily and robustly conduct titrations and screening of different antibodies and reagents, eliminating many laborious and costly steps. We show, for the first time to our knowledge, that the effect of the ratio of antibody to cell-number is most pronounced in detecting weak genomic localization signals. In particular, while strong signal seems to be unimpacted by low antibody to cell-number ratio, weaker signals are sensitive to this ratio. We note the importance of maintaining a consistent antibody to cell-number ratio, especially when conducting comparative studies, e.g., between treatments such as small molecules, or individuals such as for chromatin-QTL mapping. Our automated ChIP-seq protocol is publicly available, including specific deck setups, software files and parameters. Lastly, we envision that our robust, cost-efficient protocol can advance research via multiple fronts, e.g., by (i) allowing the scaling up the number of replicates and conditions tested, (ii) improving quantification precision when using spike-in normalization in ChIP-seq experiments2, (iii) enabling core facilities to provide high-throughput ChIP-seq as a service and (iv) being incorporated into antibody evaluation procedures, compound screening, population genomics and diagnostic frameworks.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.14.670415v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Accelerating Insight Discovery in Large Biomedical Text with Scalable Processing Framework</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kim, D., Hauptman, M., Patrick, M. T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Quanta, a hybrid system that combines large language models with established evaluation metrics, such as cosine similarity, to enable efficient summarization and interpretation of curated research corpora.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Large language models are increasingly being used by dermatology professionals to support diagnostic investigation, patient education, and medical research. While these models can help manage information overload and improve efficiency, concerns persist regarding their accuracy and potential reliance on dubious sources. We introduce Quanta, a hybrid system that combines large language models with established evaluation metrics, such as cosine similarity, to enable efficient summarization and interpretation of curated research corpora. This methodology ensures that synthesized insights remain domain-specific and contextually relevant, thereby supporting clinicians and researchers in navigating the expanding digital landscape of dermatology literature. Deployed within an interactive chatbot, the tool delivers direct answers to user queries, provides cross-publication insights, and can suggest new directions for research. Comparative evaluations on benchmark datasets demonstrate improvements in accuracy, efficiency, and computational cost, with the curated document approach enhancing reliability and reducing misinformation risk.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.14.670384v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">TissueFormer: a neural network for labeling tissue from grouped single-cell RNA profiles</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Benjamin, A. S., Zador, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce TissueFormer, a Transformer-based neural network that analyzes groups of single-cell RNA profiles to infer population-level labels while retaining single- cell resolution.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell RNA sequencing technologies have enabled unprecedented insights into gene expression and are poised to transform clinical diagnostics. At present, most computational approaches for interpreting single-cell data operate at the level of individual cells, predicting labels or properties based on isolated transcriptomic profiles. This approach overlooks a key class of signals: the composition of cells within a sample or defined population. Such signals are often critical for inferring tissue identity, disease state, or other sample-level phenotypes. To address this limitation, we introduce TissueFormer, a Transformer-based neural network that analyzes groups of single-cell RNA profiles to infer population-level labels while retaining single-cell resolution. Applied to predict the cortical area of groups of cells sampled from spatial transcriptomic data from mouse brains, TissueFormer outperformed both single-cell foundation models and machine learning methods applied to pseudobulk and cell type composition. This higher performance enables the automated construction of high-resolution brain region maps in individual animals directly from spatial transcriptomic data. More broadly, TissueFormer provides a framework for predicting any population-level phenotypes which are influenced by cellular diversity and tissue-level organization.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.17.670735v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Extracellular Vesicle Gene Expression Enables Sensitive Detection of Colorectal Neoplasia</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gowrisankar, S., Ray, C. J., Nguyen, S., Benway, C. J., Xing, S., George, A., Miller, D. L., Manning, K., Yang, Y., Cole, T. J., Mitsock, E., Perez, D., Dealmeida, L., Doo, J., Upadhyay, P. K., Yao, Y., Johnson, J. M., Haynes, B. C., Chakrabortty, S. K., Skog, J. K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We found that EV-derived gene expression on long RNA and small RNA transcriptomes is the most promising modality for cancer detection and identified a new, blood-based CRC screening tool for advanced adenomas.</p><p style="font-size: 16px; text-align: justify;">BACKGROUND & AIMS: Extracellular vesicles (EVs), including exosomes, are emerging as promising carriers of disease-specific biomarkers due to their molecular cargo reflective of cellular origin. While cell-free DNA (cfDNA) methylation assays have recently been developed for colorectal cancer (CRC) screening and perform well for cancer detection, they show limited sensitivity for advanced adenomas (AA), a key precursor in the CRC pathway. To address this gap, we sought to determine whether other blood-based analytes, specifically EV-derived long and small RNA transcriptomes and EV proteomes, could improve detection of AA and early-stage CRC. In a prospective cohort, we performed a head-to-head comparison of EV transcriptomics, EV proteomics, and their combinations against cfDNA methylation, all measured from the same patient cohort, enabling a direct performance benchmark and identification of the most promising modality for larger-scale CRC screening studies. METHODS: We prospectively collected pre-colonoscopy plasma samples from 220 participants across three clinical sites. EVs were isolated and profiled using long RNA-seq, small RNA-seq, and Olink-based proteomics. cfDNA was analyzed for methylation patterns. Analyses were conducted according to a statistical analysis plan pre-specified before unblinding. Machine learning models were developed under nested cross-validation to evaluate sensitivity for detecting AA and CRC at a fixed specificity of 91%, consistent with clinical screening benchmarks. RESULTS: EV-derived gene expression on long RNA demonstrated the highest sensitivity for detecting AA as well as CRC: 54.8% (95% CI, 26.4%-75.6%) for AA and 94.1% (95% CI, 79.2%-100%) for CRC. This outperformed cfDNA methylation (33.6% [95% CI, 9.7%-60.3%] for AA, 81.3% [95% CI, 64.6%-93.9%] for CRC) and other EV-based modalities. In addition, for small RNA the sensitivities were 42.6% (95% CI, 35.9%-47.4%) for AA, and 78.9% (95% CI, 69.2%-83.0%) for CRC, while for proteomics the sensitivities were 30.0% (95% CI, 13.9%-40.0%) for AA, and 64.2% (95% CI, 43.6%-87.2%) for CRC. Transcriptomic profiles revealed progressive enrichment of hallmarks of cancer pathways, including apoptosis and epithelial-mesenchymal transition, across disease stages. Multiomic integration did not improve performance beyond EV transcriptomics alone. CONCLUSIONS: By directly comparing multiple EV-based and cfDNA analytes within the same patient cohort, we found that EV transcriptomics delivers the strongest diagnostic performance for both AA and CRC. This rigorous benchmarking approach allows clear prioritization of the most promising modality guiding the design of larger validation studies and accelerating development of next-generation, blood-based CRC screening tools.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.14.670394v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Preservation and Clonal Behavior of Extrachromosomal DNA in Patient-Derived Xenograft Models of Childhood Cancers</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kenkre, R., Larson, J. D., Chapman, O. S., Luebeck, J., Lo, Y. Y., Paul, M., Zhang, W., Bafna, V., Wechsler-Reya, R., Chavez, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show strong selection or de novo formation of ecDNA-positive clones in PDXs, consistent with the selective growth advantage conferred by ecDNA during tumor evolution.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Extrachromosomal DNA (ecDNA) is a powerful oncogenic driver linked to poor prognosis in pediatric cancers. Whole-genome sequencing of 338 patient-derived xenograft (PDX) samples and 127 matched primary tumors across multiple childhood cancer types was used to compare ecDNA prevalence, sequence conservation, and clonal dynamics. ecDNA in PDX models frequently mirrored oncogene amplifications observed in patient tumors (e.g., MYCN, MYC, MDM2) and showed high sequence conservation. Medulloblastoma and neuroblastoma PDXs exhibited significantly higher ecDNA prevalence, consistent with strong selection or de novo formation during tumor propagation. Although ecDNA copy numbers were generally preserved, some neuroblastoma PDXs displayed marked MYCN copy gains. Single-cell multiome profiling revealed that ecDNA-positive clones either persisted or expanded dramatically in PDXs, in one case growing from a minor subpopulation to nearly all tumor cells. These findings establish PDX models as valuable systems for ecDNA research and underscore the selective growth advantage conferred by ecDNA during tumor evolution.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.14.670153v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CBM KG: A Comorbidity-Centric Knowledge Graph Uncovering Causal Pathomechanisms Between COVID-19 and Neurodegenerative Diseases</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Atas Guvenilir, H., Sethumadhavan, P., Kaladharan, A., Vieira de Sa, R., Sridhar, A., Haferkamp, U., Pless, O., Marti-Sarrias, A., Acosta, S., Letoha, T., Hudak, A., Ohnmacht, J., Hefeng, F. Q., Hofmann-Apitius, M., Tom Kodamullil, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present CBM KG (Causal Biological Mechanisms Knowledge Graph), a manually curated, comorbidity-centric resource developed within the EU-funded COMMUTE project, which represents causal biological mechanisms spanning both infectious and neurodegenerative processes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> COVID-19 is increasingly recognized as a potential trigger or accelerator of neurodegenerative diseases such as Alzheimer`s and Parkinson`s. To systematically explore the putative molecular and clinical associations between them, we present CBM KG (Causal Biological Mechanisms Knowledge Graph), a manually curated, comorbidity-centric resource developed within the EU-funded COMMUTE project. CBM KG integrates over 2,800 cause-and-effect or correlative relationships from 63 peer-reviewed publications, highlighting key mechanisms such as viral entry routes, blood-brain barrier alteration, microglial activation, neuroinflammation, and APOE e4-associated susceptibility. Each relationship in the graph is fully traceable to its source evidence, ensuring transparency and reproducibility. Unlike general-purpose or single disease-focused knowledge graphs, CBM KG is specifically designed to represent causal biological mechanisms spanning both infectious and neurodegenerative processes. By encoding directional, cause-and-effect relationships, it supports the interpretation of clinical co-occurrences through plausible mechanistic links between overlapping disease pathways, offering high-resolution insights at both molecular and clinical levels.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.18.670790v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DeepDiff-SHAP: Interpretable deep learning for subgroup-specific causal inference using conditional SHAP</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sriram, A., Park, H. J., Kim, S., Carcillo, J. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce DeepDiff-SHAP, a novel framework that combines regression-based and deep learning-based differential causal inference to detect changes in causal relationships across patient subgroups.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Precision medicine aims to tailor healthcare strategies to individual differences in genetic, clinical, and environmental factors. However, identifying subgroup specific causal relationships in complex biomedical data remains a major challenge, especially when standard causal inference methods average over population heterogeneity. We introduce DeepDiff-SHAP, a novel framework that combines regression-based and deep learning-based differential causal inference to detect changes in causal relationships across patient subgroups. DeepDiff-SHAP integrates SHapley Additive exPlanations (SHAP) to estimate conditional dependencies and perform nonlinear differential causal inference in a principled, interpretable manner. Applying DeepDiff-SHAP to two population-scale datasets, the CDC Diabetes Health Indicators Dataset and a UK Biobank sepsis cohort stratified by hypertension, we identified clinically meaningful and subgroup-specific causal changes in relationships between features such as age, general health, alkaline phosphatase, and cholesterol. Our results demonstrate that deep learning enhances sensitivity to complex interaction patterns overlooked by linear models, providing new biological insights into disease progression and comorbidity-specific risk mechanisms. DeepDiff-SHAP offers a scalable and interpretable solution to uncover individualized causal pathways, advancing the goal of truly personalized medicine.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.14.670190v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">GRIDGEN: Guided Region Identification based on Density of GENes - a transcript density-based approach to characterize tissues by spatial transcriptomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sequeira, A. M., Ijsselsteijn, M., Rocha, M., Roelands, J., de Miranda, N. F. C. C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed GRIDGEN (Guided Region Identification based on Density of GENes), an approach for defining regions of interest based on transcript density, which enables flexible analysis of spatial omics data, supporting the study of biological processes across diverse tissue structures and microenvironments.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Spatial omics brought unprecedented power to study biological processes within tissues while preserving spatial context and morphology. Most spatial proteomics and transcriptomics analyses methods are cell-centric, relying on cell segmentation to identify and characterize individual cells before downstream tasks. However, certain biological questions may be better addressed using cell-free approaches, which also eliminate unnecessary computations when cell segmentation is not essential. To address this need, we developed GRIDGEN (Guided Region Identification based on Density of GENes), an approach for defining regions of interest based on transcript density. GRIDGEN enables the identification of biologically relevant tissue compartments, including interfaces between regions, phenotype-enriched areas, and zones defined by specific gene signatures, supporting analyses such as pathway enrichment. We demonstrated the utility of GRIDGEN by applying it to spatial transcriptomics data from CosMx and Xenium platforms in colorectal cancer (CRC) samples. By bypassing cell segmentation, our approach enables flexible analysis of spatial omics data, supporting the study of biological processes across diverse tissue structures and microenvironments. Nevertheless, GRIDGEN can be easily integrated with cell segmentation strategies for complementary analyses. GRIDGEN thus broadens the analytical toolkit for spatial omics, enabling both cell-free and cell-based insights.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.14.670318v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Causal single-cell RNA-seq simulation, in silico perturbation, and GRN inference benchmarking using GRouNdGAN-Toolkit</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zinati, Y., Emad, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a causal implicit generative model capable of simulating realistic observational and interventional data following a user-defined GRN on any biological system of interest.</p><p style="font-size: 16px; text-align: justify;">Background: Rapid advances in high-throughput single-cell sequencing technologies, coupled with the development of computational methods capable of leveraging large datasets, have led to the emergence of numerous approaches for deciphering regulatory interactions in the form of Gene Regulatory Networks (GRNs). However, in the absence of context-specific gold-standard ground truths, particularly those containing causal interactions, systematically benchmarking GRN inference methods remains a challenge. Thus, we previously developed GRouNdGAN, a causal implicit generative model capable of simulating realistic observational and interventional scRNA-seq data following a user-defined GRN on any biological system of interest. Importantly, we demonstrated that GRouNdGAN generates datasets that bridge the gap between experimentation and simulation for GRN inference benchmarking. Method: Building upon the GRouNdGAN framework, we developed an extended toolkit that offers additional features, including interactive model visualization, training monitoring, more customizable GRN creation options, synthetic data similarity and GRN inference benchmarking metrics, and an intuitive TF knockout prediction module. Here, we provide a step-by-step procedure for implementing the protocol from start to finish and introduce alternative variations to adapt GRouNdGAN to studies with different experimental setups. GRouNdGAN-Toolkit is publicly available as a python code repository and containerized application and is accompanied by a tutorial website featuring a collection of simulated datasets. Model training largely depends on graphic hardware and the size and density of the input GRN, and typically takes around 75h to complete on a single GPU. Excluding model training, this protocol typically takes less than 25min to complete. Discussion: GRouNdGAN-Toolkit is a versatile simulator with user friendly interface that does not assume advanced computational genomics expertise, enhancing its usability and accessibility across a wide range of users.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.14.670294v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Urochloa trichopus (Hoch.) Stapf (Poaceae: Paniceae): An addition to the Flora of Karnataka</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pulla, S., Mande, M., Sankaran, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Urochloa trichopus (Hoch.) Stapf, a grass species native to Africa, the Arabian Peninsula, and Madagascar, is reported here for the first time from Karnataka (India).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Urochloa trichopus (Hoch.) Stapf, a grass species native to Africa, the Arabian Peninsula, and Madagascar, is reported here for the first time from Karnataka (India). A detailed taxonomic description with photographs is provided to facilitate its identification.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.14.670282v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Engineering compact Physalis peruviana (goldenberry) to promote its potential as a global crop</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Santo Domingo, M., Fitzgerald, B., Robitaille, G. M., Ramakrishnan, S., Swartwood, K., Karavolias, N., Schatz, M., Van Eck, J., Lippman, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A compact version of the goldenberry based on a novel gene-editing technique .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Goldenberry (Physalis peruviana) produces sweet, nutritionally-rich berries, yet like many minor crops, is cultivated in limited geographical regions and has not been a focus of breeding programs for trait enhancement. Leveraging knowledge of plant architecture-related traits from related species, we used CRISPR/Cas9-mediated gene editing to generate a compact ideotype to advance future breeding efforts and agricultural production. Goldenberry growers will benefit from these compact versions because it optimizes per plot yield, facilitating larger-scale production to meet rising consumer popularity and demand.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.15.670557v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Yeast MoClo secretion and surface display toolkit 2.0: improvements and applications for analysis of protein-protein interactions and whole-cell biocatalysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Juric, V., Erwin, L. G., O'Riordan, N. M., Maher, E., Holmes, J. D., Young, P. W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A modular cloning toolkit for S. cerevisiae that facilitates rapid optimization of signal peptides and anchor proteins for efficient secretion and/or surface display of heterologous proteins of interest.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Saccharomyces cerevisiae is an invaluable model organism for both fundamental biological research and biotechnological applications including recombinant protein production as well as protein and metabolic engineering. We previously developed a modular cloning (MoClo) based toolkit for S. cerevisiae that facilitates rapid optimization of signal peptides and anchor proteins for efficient secretion and/or surface display of heterologous proteins of interest. Here we describe further improvements and applications of this yeast secretion and display (YSD) toolkit. New parts encoding anchor proteins based on N-terminal fusion to a truncated Aga1 and C-terminal fusion to Aga2, each with three possible epitope tag options, are described. We also added parts that facilitate high throughput detection of secreted proteins of interest through GFP fluorescence complementation and parts encoding secretion boosting yeast proteins, whose overexpression has previously been reported to enhance secretion of heterologous proteins. In addition, two surface display applications of the toolkit are showcased. We demonstrate that yeast surface display of an anti-GFP nanobody allows cost-effective evaluation of the interactions of GFP-tagged proteins of interest, either by flow cytometry or yeast-based co-immunoprecipitation. In addition, using yeast cells as whole-cell catalysts, we show that co-display of the poly(ethylene terephthalate) (PET) degrading enzyme leaf-branch compost cutinase with hydrophobin1 enhances the breakdown of PET plastic, while triple co-display of these proteins with MHETase causes complete conversion of the intermediary monohydroxy-ethyl-terephthalate (MHET) to terephthalic acid. The diverse applications described herein demonstrate the broad applications of the updated MoClo YSD toolkit 2.0 in both synthetic biology and other research fields.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.19.671047v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Human single-cell atlas analysis reveals heterogeneous endothelial signaling</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhu, Z., Zheng, R., Yu, Y., Zhang, L., Chen, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We systematically investigate EC heterogeneity and provide insights into how EC heterogeneity was regulated across diverse tissue microenvironments.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Endothelial cells (ECs) play complex roles across tissues and vessel types. Yet, systematic investigations of EC heterogeneity in the combined context of vessel type and tissue microenvironment are still largely lacking. We integrated over three million single cells of scRNA-seq datasets in 15 human tissues and found that ECs in some tissues (e.g., heart and kidney) exhibited greater tissue specificity, while others displayed more substantial vessel specificity. We developed a computational pipeline to analyze cell-cell communications (CCC) mediated by metabolites or proteins to explore microenvironmental regulation. Interestingly, our results showed that CCC events involving ECs varied vastly across tissues, highlighting tissue-specific EC interactions. Using topic modeling, we identified CCC patterns, termed CCC topics, representing metabolite- and protein-mediated interactions between ECs and other tissue-resident cells. Most CCC topics exhibited high tissue specificity, potentially explaining microenvironmental regulations for EC heterogeneity. The work systematically investigates EC heterogeneity and provides insights into how EC heterogeneity was regulated across diverse tissue microenvironments.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.15.670370v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Novel natural vector with asymmetric covariance for classifying biological sequences.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Guoqing Hu, Tao Zhou, Piyu Zhou, Stephen Shing-Toung Yau</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose an improved version of the natural vector method that incorporates k-mer information alongside covariance computations with asymmetric properties between base positions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The genome sequences of organisms form a large and complex landscape, presenting a significant challenge in bioinformatics: how to utilize mathematical tools to describe and analyze this space effectively. The ability to compare relationships between different organisms depends on creating a rational mapping rule that can uniformly encode genome sequences of varying lengths as vectors in a measurable space. This mapping would enable researchers to apply modern mathematical and machine learning techniques to otherwise challenging genomic comparisons. The natural vector method has been proposed as a concise and effective approach to accomplish this. However, its various iterations have certain limitations. In response, we carefully analyze the strengths and weaknesses of these natural vector methods and propose an improved version-an asymmetric covariance natural vector method (ACNV). This new method incorporates k-mer information alongside covariance computations with asymmetric properties between base positions. We tested ACNV on microbial genome sequence datasets, including bacterial, fungal, and viral sequences, evaluating its performance in terms of classification accuracy and convex hull separation. The results demonstrate that ACNV effectively captures sequence characteristics, showcasing its robust sequence representation capabilities and highlighting its elegant geometric properties.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40367998/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

