<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Wednesday, 23 April 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Custom CRISPR-Cas9 PAM variants via scalable engineering and machine learning.</h2>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> The first generalist CRISPR-Cas enzymes were developed in the 1970s, and are used in the discovery of new proteins.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Engineering and characterizing proteins can be time-consuming and cumbersome, motivating the development of generalist CRISPR-Cas enzymes</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40262634/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The Dawn of High-Throughput and Genome-Scale Kinetic Modeling: Recent Advances and Future Directions.</h2>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A review of the state-of-the-art in kinetic modeling, highlighting the potential of these advances to drive progress in systems and synthetic biology</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Researchers have invested much effort into developing kinetic models due to their ability to capture dynamic behaviors, transient states, and regulatory mechanisms of metabolism, providing a detailed and realistic representation of cellular processes. Historically, the requirements for detailed parametrization and significant computational resources created barriers to their development and adoption for high-throughput studies. However, recent advancements, including the integration of machine learning with mechanistic metabolic models, the development of novel kinetic parameter databases, and the use of tailor-made parametrization strategies, are reshaping the field of kinetic modeling. In this Review, we discuss these developments and offer future directions, highlighting the potential of these advances to drive progress in systems and synthetic biology, metabolic engineering, and medical research at an unprecedented scale and pace.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40262025/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrating machine learning models with multi-omics analysis to decipher the prognostic significance of mitotic catastrophe heterogeneity in bladder cancer.</h2>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Mitotic catastrophe is well-known as a major pathway of endogenous tumor death, but the prognostic significance of its heterogeneity regarding bladder cancer (B</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mitotic catastrophe is well-known as a major pathway of endogenous tumor death, but the prognostic significance of its heterogeneity regarding bladder cancer (BLCA) remains unclear.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40259382/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrated multiomics analysis identifies potential biomarkers and therapeutic targets for autophagy associated AKI to CKD transition.</h2>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified 14 autophagy-related genes and their immune infiltration during the transition from acute kidney injury to chronic kidney disease (CKD).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> This study explored the relationship between acute kidney injury (AKI) and chronic kidney disease (CKD), focusing on autophagy-related genes and their immune infiltration during the transition from AKI to CKD. We performed weighted correlation network analysis (WGCNA) using two microarray datasets (GSE139061 and GSE66494) in the GEO database and identified autophagy signatures by Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG), and GSEA enrichment analysis. Machine learning algorithms such as LASSO, random forest, and XGBoost were used to construct the diagnostic model, and the diagnostic performance of GSE30718 (AKI) and GSE37171 (CKD) was used as validation cohorts to evaluate its diagnostic performance. The study identified 14 autophagy candidate genes, among which ATP6V1C1 and COPA were identified as key biomarkers that were able to effectively distinguish between AKI and CKD. Immune cell infiltration and GSEA analysis revealed immune dysregulation in AKI, and these genes were associated with inflammation and immune pathways. Single-cell analysis showed that ATP6V1C1 and COPA were specifically expressed in AKI and CKD, which may be related to renal fibrosis. In addition, drug prediction and molecular docking analysis proposed SZ(+)-(S)-202-791 and PDE4 inhibitor 16 as potential therapeutic agents. In summary, this study provides new insights into the relationship between AKI and CKD and lays a foundation for the development of new treatment strategies.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40258914/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrated bioinformatic analysis of immune infiltration and disulfidptosis related gene subgroups in type A aortic dissection.</h2>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study investigates the role of disulfidptosis-related genes in immune infiltration in type A aortic dissection (TAAD)</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Type A aortic dissection (TAAD) is a lethal cardiovascular disease characterized by the separation of the layers within the aortic wall. The underlying pathological mechanisms of TAAD requires further elucidation to develop effective prevention and pharmacological treatment strategies. Inflammation plays a crucial role in TAAD pathogenesis. Disulfidptosis, an emerging type of cell death, may shed light on disease mechanisms. This study investigates the role of disulfidptosis-related genes in immune infiltration in TAAD. TAAD gene expression datasets were obtained from the Gene Expression Omnibus (GEO) database. Immune cell infiltration analysis assessed immune cell dysregulation in TAAD. Differentially expressed genes (DEGs) between TAAD samples and controls were identified and intersected with known disulfidptosis-related gene sets to obtain relevant DEGs. Hub genes were identified using machine learning algorithms. A diagnostic model was constructed using Least Absolute Shrinkage and Selection Operator (LASSO) regression on 25 TAAD samples. Consensus clustering classified TAAD samples based on disulfidptosis-related gene expression. Functional enrichment analyses, including Gene Ontology (GO) terms and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway analyses, elucidated associated biological processes and pathways. A total of 13,316 DEGs were identified, among which 11 disulfidptosis-related genes were screened: INF2, CD2AP, PDLIM1, ACTN4, MYH10, MYH9, FLNA, FLNB, TLN1, MYL6, ACTB, CAPZB, DSTN, and IQGAP1. Most of these genes exhibited lower expression levels in TAAD samples, except CAPZB, and were correlated with immune cell infiltration. Cluster-specific DEGs were found in one cluster, involving several immune response processes. Co-clustering analysis based on disulfidptosis-related genes classified TAAD samples into two clusters, with higher gene expression levels observed in cluster C2 compared to cluster C1. Three key hub genes were identified, and potential therapeutic mechanisms for TAAD were explored. Immuno-infiltration results revealed significant differences in immune profiles, with higher immunological scores and more extensive immune infiltration in TAAD. Disulfidptosis occurs in TAAD and is associated with immune cell infiltration and metabolic activity, influencing immune cell function and responses. These findings suggest that disulfidptosis may promote TAAD progression through the induction of immune responses and metabolic activities. This research provides new insights into the pathogenesis and identifies potential therapeutic targets for TAAD.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40258895/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Deconvolution of cell types and states in spatial multiomics utilizing TACIT.</h2>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> TACIT, an unsupervised algorithm for cell annotation using predefined signatures that operates without training data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Identifying cell types and states remains a time-consuming, error-prone challenge for spatial biology. While deep learning increasingly plays a role, it is difficult to generalize due to variability at the level of cells, neighborhoods, and niches in health and disease. To address this, we develop TACIT, an unsupervised algorithm for cell annotation using predefined signatures that operates without training data. TACIT uses unbiased thresholding to distinguish positive cells from background, focusing on relevant markers to identify ambiguous cells in multiomic assays. Using five datasets (5,000,000 cells; 51 cell types) from three niches (brain, intestine, gland), TACIT outperforms existing unsupervised methods in accuracy and scalability. Integrating TACIT-identified cell types reveals new phenotypes in two inflammatory gland diseases. Finally, using combined spatial transcriptomics and proteomics, we discover under- and overrepresented immune cell types and states in regions of interest, suggesting multimodality is essential for translating spatial biology to clinical applications.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40258827/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Design of nanobody targeting SARS-CoV-2 spike glycoprotein using CDR-grafting assisted by molecular simulation and machine learning.</h2>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a complementarity-determining region (CDR) grafting approach for designing nanobodies (Nbs) that target specific</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The design of proteins capable effectively binding to specific protein targets is crucial for developing therapies, diagnostics, and vaccine candidates for viral infections. Here, we introduce a complementarity-determining region (CDR) grafting approach for designing nanobodies (Nbs) that target specific epitopes, with the aid of computer simulation and machine learning. As a proof-of-concept, we designed, evaluated, and characterized a high-affinity Nb against the spike protein of SARS-CoV-2, the causative agent of the COVID-19 pandemic. The designed Nb, referred to as Nb Ab.2, was synthesized and displayed high-affinity for both the purified receptor-binding domain protein and to the virus-like particle, demonstrating affinities of 9 nM and 60 nM, respectively, as measured with microscale thermophoresis. Circular dichroism showed the designed protein's structural integrity and its proper folding, whereas molecular dynamics simulations provided insights into the internal dynamics of Nb Ab.2. This study shows that our computational pipeline can be used to efficiently design high-affinity Nbs with diagnostic and prophylactic potential, which can be tailored to tackle different viral targets.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40257976/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Rapid assay development for low input targeted proteomics using a versatile linear ion trap.</h2>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show consistent quantification across three orders of magnitude in a matched-matrix background of low-level proteins such as transcription factors and cytokines</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Advances in proteomics and mass spectrometry enable the study of limited cell populations, where high-mass accuracy instruments are typically required. While triple quadrupoles offer fast and sensitive low-mass specificity measurements, these instruments are effectively restricted to targeted proteomics. Linear ion traps (LITs) offer a versatile, cost-effective alternative capable of both targeted and global proteomics. Here, we describe a workflow using a hybrid quadrupole-LIT instrument that rapidly develops targeted proteomics assays from global data-independent acquisition (DIA) measurements without high-mass accuracy. Using an automated software approach for scheduling parallel reaction monitoring assays (PRM), we show consistent quantification across three orders of magnitude in a matched-matrix background. We demonstrate measuring low-level proteins such as transcription factors and cytokines with quantitative linearity below two orders of magnitude in a 1‚Äâng background proteome without requiring stable isotope-labeled standards. From a 1‚Äâng sample, we found clear consistency between proteins in subsets of CD4</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40263265/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">GOALS: Gene Ontology Analysis with Layered Shells for Enhanced Functional Insight and Visualization</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Gene Ontology Analysis using Layered Shells (GOALS), a novel tool that discretizes GOs into optimal layers and reveals</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Gene Ontologies (GOs) are standardized descriptions of gene functions in terms of biological processes, molecular functions, and cellular components, capturing their Parent-Child relationships in a structured framework and advancing cancer biological modeling to provide consistent and meaningful insights into functional genomics analysis. The conventional GO hierarchical structure is defined by human curation experts, with levels determined by the shortest path to the root term. However, grouping GOs poses challenges due to the uneven distribution of gene members within GO terms and inconsistencies in the level of detail across terms at the same GO level. In this work, we introduce Gene Ontology Analysis using Layered Shells (GOALS), a novel tool that discretizes GOAs into optimal layers. GOALS creates scalable GO layers while maintaining a balanced number of genes across GOs in each layer. Unlike existing tools, the GOALS framework organizes GO terms using a bottom-up approach based on their co-membership network, discretizing GOs to achieve an exponential fit with GO's gene member size. Meanwhile, GOALS reveals clusters or supersets reflecting biological relevance by unsupervised clustering of GO's latent projections. In a case study on mouse natural killer (NK) cell development, GOALS identified distinct GO functional clusters with multi-GO layers to reveal multiple levels of detail from specific to abstract contexts to maximize signal discovery and uncover those signals' associations with trajectory divergence. More importantly, GOALS enhances enrichment analysis by introducing additional GO stratification and latent GO map that enables more accurate classification of functional differences. GOALS offers a robust and innovative framework for exploring disordered GO clusters, mining GO activities, and analyzing potential GO-GO interplays. By addressing critical challenges in functional genomics, GOALS provides a powerful tool for advancing our understanding of cell heterogeneity and potentially uncovering actionable insights for therapeutic development.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.22.650095v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ShinyCell2: An extended library for simple and sharable visualisation of spatial, peak-based and multi-omic single-cell data</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present ShinyCell2, an enhanced R package for interactive visualisation of single-cell multi-omics and spatial transcriptomics data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell technologies now span multiple modalities, generating large, complex datasets that challenge analysis and sharing. We present ShinyCell2, an enhanced R package for interactive visualisation of single-cell multi-omics and spatial transcriptomics data. ShinyCell2 retains the simplicity and lightweight deployment of its predecessor while introducing advanced visualisations, cross-modality comparisons, and statistical tools tailored to spatial and multi-omic data. It enables intuitive, rapid exploration of high-dimensional data without requiring extensive computational expertise.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.22.650045v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">GAT-HiC: Efficient Reconstruction of 3D Chromosome Structure via Residual Graph Attention Neural Networks</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We come up with graph attention and residual network-based GAT-HiC to predict three-dimensional chromosome structure from Hi-C interactions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Hi-C is an experimental technique to measure the genome-wide topological dynamics and three-dimensional (3D) shape of chromosomes indirectly via counting the number of interactions between distinct sets of loci. One can estimate the 3D shape of a chromosome over these indirect interaction datasets. Here, we come up with graph attention and residual network-based GAT-HiC to predict three-dimensional chromosome structure from Hi-C interactions. GAT-HiC is distinct from the existing 3D chromosome shape prediction approaches in a way that it can generalize to data that is different than train data. So, we can train GAT-HiC on one type of Hi-C interaction matrix and infer on a completely dissimilar interaction matrix. GAT-HiC combines the unsupervised vertex embedding method Node2vec with an attention-based graph neural network when predicting each genomic locis three-dimensional coordinates from Hi-C interaction matrix. We test the performance of our method across multiple Hi-C interaction datasets, where a trained model can be generalized across distinct cell populations, distinct restriction enzymes, and distinct Hi-C resolutions over human and mouse. GAT-HiC can reconstruct accurately in all these scenarios. Our method outperforms the existing approaches in terms of the accuracy of three-dimensional chromosome shape inference over interaction datasets.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.18.649477v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">EpitopeTransfer: a Phylogeny-aware transfer learning framework for taxon-specific linear B-cell epitope prediction</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a phylogeny-aware transfer learning strategy that results in noticeable gains in expected predictive performance for the identification of linear B-cell epitopes</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The identification of linear B-cell epitopes (LBCEs) plays a pivotal role in the development of immunological products such as immunodiagnostic, vaccines and therapeutic antibodies. This criticality has led to the development of computational approaches for the prediction of LBCEs, aiming at accelerating discovery and prioritising targets for experimental assessment. Existing LBCE predictors rely on having access to large volumes of data from a wide range of pathogens for training predictive models, which can result in biases towards widely studied organisms and compromise performance when focusing on neglected or emerging pathogens. We introduce a phylogeny-aware transfer learning strategy that results in noticeable gains in expected predictive performance. This is achieved by fine tuning large protein language models using abundant data from higher-level taxa and applying this refined feature embedder for fitting pathogen- or lower taxon-specific models. We report substantially increased performance in comparison to state-of-the-art approaches for LBCE prediction, and show that these gains can be directly attributed to the use of phylogeny-aware fine tuning of the feature embedder coupled with pathogen- or taxon-optimized modelling.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.17.649425v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Mechanistic Insights into Proteomic Mutation-Phenotype Linkages from Tiling Mutagenesis Screens</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present ProTiler-Mut, a versatile computational framework that harnesses tiling mutagenesis screens, which introduce variants across entire protein sequences</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> High-throughput mutagenesis screens are powerful tools for mapping mutations to phenotypes. However, deciphering the molecular mechanisms that link mutations to phenotypic outcomes remains a significant challenge. Here, we present ProTiler-Mut, a versatile computational framework that harnesses tiling mutagenesis screens, which introduce variants across entire protein sequences, to facilitate investigation of mutation-to-phenotype associations at multiple levels, including individual residues, protein substructures, and protein-protein interactions (PPIs). As demonstrated through our analyses of base editing (BE) screens targeting DNA Damage Response (DDR) proteins and T cell regulators, ProTiler-Mut provides novel insights into the mutation-phenotype linkages, including: i) refined classification of mutation that reveals separation-of-function (SOF) category beyond the conventional binary classification of loss-of-function (LOF) and gain-of-function (GOF); ii) definition of phenotype-associated hotspot substructures that enable the inference of the function of unscreened pathogenic mutations; and iii) identification of phenotype-associated PPIs disrupted by functional mutations. Through ProTiler-Mut analyses, we identified a substructure harboring pathogenic GOF mutations that disrupt interactions between the kinases MAPK1 and RSK1, leading to MAPK1 activation and elevated expression of the immune checkpoint receptor PD-1. Furthermore, we demonstrate the applicability of ProTiler-Mut to various mutagenesis screening platforms, highlighting its broad utility and generalizability.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.17.649336v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Fine-Tuning Protein Language Models Unlocks the Potential of Underrepresented Viral Proteomes</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that fine-tuning pre-trained protein language models on viral protein sequences significantly enhances representation quality and improves performance on downstream tasks.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein language models (pLMs) have revolutionized computational biology by generating rich protein vector representations -- or embeddings -- enabling major advancements in de novo protein design, structure prediction, variant effect analysis, and evolutionary studies. Despite these breakthroughs, current pLMs often exhibit biases against proteins from underrepresented species, with viral proteins being particularly affected -- frequently referred to as the "dark matter" of the biological world due to their vast diversity and ubiquity, yet sparse representation in training datasets. Here, we show that fine-tuning pre-trained pLMs on viral protein sequences, using diverse learning frameworks and parameter-efficient strategies, significantly enhances representation quality and improves performance on downstream tasks. To support further research, we provide source code for fine-tuning pLMs and benchmarking embedding quality. By enabling more accurate modeling of viral proteins, our approach advances tools for understanding viral biology, combating emerging infectious diseases, and driving biotechnological innovation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.17.649224v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Measurement and prediction of unmixing-dependent spreading in spectral flow cytometry panels</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that unmixing-dependent spreading (UDS) is a mathematical phenomenon which leads to a progressive deterioration of unmixed signal-to</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Advances in spectral cytometry instrumentation and fluorescent reagents have led to the possibility of ultra-high-parameter panels exceeding 50 colors. However, panel size is limited in practice by unmixing-dependent spreading (UDS), a mathematical phenomenon which leads to a progressive deterioration of unmixed signal-to-noise ratios in panels that contain fluorochrome combinations with significant spectral overlap. Choosing spectrally compatible sets of fluorochromes that avoid UDS is a complex and labor-intensive task involving substantial trial-and-error experimentation. Here, we provide a detailed explanation of UDS and practical strategies for handling UDS in large spectral panels. We describe the empirical hallmarks of UDS, demonstrate how to quantify its impact, and dissect its underlying mathematical cause in terms of spectral collinearity. We present novel computational metrics that can be used to select optimal combinations of fluorochromes in a platform-agnostic fashion based on publicly available reference data, providing a general tool for spectral panel design.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.17.649396v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">scStudio: A User-Friendly Web Application Empowering Non-Computational Users with Intuitive scRNA-seq Data Analysis</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed scStudio, a user-friendly, comprehensive and modular web-based application designed to democratize scRNA-seq data analysis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell RNA sequencing (scRNA-seq) has revolutionized our understanding of cellular heterogeneity by providing detailed insights into gene expression at the individual cell level. Despite its potential, the complexity of scRNA-seq data analysis often poses challenges for researchers without computational expertise. To address this, we developed scStudio, a user-friendly, comprehensive and modular web-based application designed to democratize scRNA-seq data analysis. scStudio is equipped with a suite of features designed to streamline data retrieval and analysis with both flexibility and ease, including automated dataset retrieval from the Gene Expression Omnibus (GEO). Users can also upload their own datasets in a variety of formats, integrate multiple datasets, and tailor their analyses using a wide range of flexible methods with options for parameter optimization. The application supports all the essential steps required for scRNA-seq data analysis, including in-depth quality control, normalization, dimensionality reduction, clustering, differential expression and functional enrichment analysis. scStudio also tracks the history of analyses, supports session data storage and export, and facilitates collaboration through data sharing features. By developing scStudio as a user-friendly interface and scalable architecture, we address the evolving needs of scRNA-seq research, making advanced data analysis accessible and manageable while accommodating future developments in the field.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.17.649161v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MTBseq-nf: Enabling Scalable Tuberculosis Genomics "Big Data" Analysis through a User-Friendly Nextflow Wrapper for MTBseq pipeline</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose the MTBseq-nf wrapper pipeline, a user-friendly pipeline optimized for hardware efficiency, scalability for larger datasets, and</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The MTBseq pipeline, published in 2018, was designed to address bioinformatics challenges in tuberculosis research using whole-genome sequencing data. It was the first publicly available pipeline on Github to perform full analysis of WGS data for M. tuberculosis encompassing quality control through mapping, variant calling for lineage classification, drug resistance prediction, and phylogenetic inference. However, the pipeline's architecture is not optimal for high-performance computing or cloud computing environments, which often require large datasets. To optimize the pipeline, a Nextflow wrapper MTBseq-nf, was created which offers shorter execution times through the parallel mode along with multiple other thematic improvements. The MTBseq-nf wrapper, as opposed to the linear batched analysis of samples in TBfull step of MTBseq pipeline, can execute multiple instances of the same step in parallel and therefore makes full use of the provided computational resources. For evaluation of scalability and reproducibility, we used 90 M. tuberculosis genomes (ENA accession PRJEB7727) for the benchmarking analysis on a dedicated computing server. In our experiments the execution time of MTBseq-nf parallel analysis mode is at least twice as fast as the standard MTBseq pipeline for more than 20 samples. Furthermore, the MTBseq-nf wrapper facilitates reproducibility using the nf-core, bioconda, and biocontainers projects for platform independence. The proposed MTBseq-nf wrapper pipeline is a user-friendly pipeline optimized for hardware efficiency, scalability for larger datasets, and improved reproducibility.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.17.649337v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The ISW1 and CHD1 chromatin remodelers suppress global nucleosome dynamics in living yeast cells</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We compare DNA methylation rates in wild type cells and chromatin remodeler mutants by normalizing nuclear methylation rate to the non-n</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The budding yeast genome is globally accessible to DNA methyltransferases in living cells, unlike in isolated nuclei, where it is mostly inaccessible. Here, we assess the roles of the RSC, ISW1 and CHD1 ATP-dependent chromatin remodelers in generating nucleosome dynamics in vivo. We compare DNA methylation rates in wild type cells and chromatin remodeler mutants by normalizing nuclear methylation rates to the non-nucleosomal mitochondrial DNA methylation rate in each strain. Depletion of subunits of the RSC, ISW1 or CHD1 ATP-dependent chromatin remodelers has little effect on normalized methylation rate. A decaying sine wave model used to fit nucleosome phasing data shows that nucleosome dynamics decrease with distance from the promoter in an Isw1/Chd1-dependent manner. Depletion of both Isw1 and Chd1 increases the methylation rate, suggesting that these remodelers act together to suppress nucleosome dynamics. Furthermore, the TFIIIB and TFIIIC transcription factors exhibit differential dynamics at tRNA genes. Our analysis provides insight into nucleosome and transcription factor dynamics in vivo.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.17.649351v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Benchmarking circRNA Detection Tools from Long-Read Sequencing Using Data-Driven and Flexible Simulation Framework</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the first comprehensive benchmark comparison of three bioinformatic tools for circRNA detection from long-read Oxford Nanopore sequencing.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Circular RNAs (circRNAs) are unique non-coding RNAs with covalently closed loop structures formed through backsplicing events. Their stability, tissue-specific expression patterns, and potential as disease biomarkers have garnered increasing attention. However, their circular structure and diverse size range pose challenges for conventional sequencing technologies. Long-read Oxford Nanopore (ONT) sequencing offers promising capabilities for capturing entire circRNA molecules without fragmentation, yet the effectiveness of bioinformatic tools for analyzing this data remains understudied. This study presents the first comprehensive benchmark comparison of three specialized tools for circRNA detection from ONT long-read data: CIRI-long (Zhang et al., 2021), IsoCIRC (Xin et al., 2021), and circNICK-Irs (Rahimi et al., 2021). To address the lack of standardized evaluation frameworks, we developed a novel computational pipeline, open-source and freely available, to generate realistic simulated circRNA ONT long-read datasets. Our pipeline integrates several molecular features of circRNAs extracted from established databases and real datasets into NanoSim tool (Hafezqorani et al., 2020) and outputs FASTQ reads reflecting therefore biological diversity and technical properties. We systematically assessed tool performance across key metrics, including precision, recall, specificity, accuracy, and F1 score. Our analysis revealed distinct performance profiles: while all tools exhibited high specificity, they varied in precision and their ability to detect different circRNA subtypes, often showing limited sensitivity and precision. Notably, the overlap in detected circRNAs among tools was relatively low. Additionally, computational efficiency varied significantly across the tools. This suggests that relying on a single tool might not be ideal, and combining tools or improving algorithms could be necessary for more accurate circRNA detection from ONT data. This benchmark provides valuable insights for researchers selecting appropriate tools for circRNA studies using ONT sequencing. Furthermore, our customizable simulation framework, offering a resource to optimize detection approaches and advance bioinformatic tool development for circRNA research is freely available at: https://gitlab.com/bioinfog/circall/nano-circ.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.17.649290v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

