<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Thursday, 26 June 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">rbpTransformer: A novel deep learning model for prediction of piRNA and mRNA bindings.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ahmet Gürhanlı</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a deep learning model for the binding of piRNA and mRNA sequences with an AUC value of 94.38%.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> An important issue in biotechnology is predicting whether a piRNA and an mRNA will or will not bind. Research and treatment of diseases, drug discovery, and the silencing and regulation of genes, transposons, and genomic stability may all benefit from accurate binding predictions. The literature offers numerous deep-learning models for piRNA and mRNA binding prediction. However, a proper adjustment of the effective transformer model and the impact of important design alternatives has not been evaluated thoroughly. This paper summarizes the models available in the literature, briefly introduces transformers, then offers a novel deep learning model and evaluates various design alternatives, including k-mer size, number of core modules, choice of optimization algorithm, and whether to use self-attention. The results show that rbpTransformer can be a good candidate for building deep AI models to predict the binding of piRNA and mRNA sequences with an AUC value of 94.38%. The test results also reveal how the design affects the model's accuracy.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40561121/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Context-aware sequence-to-activity model of human gene regulation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Aksu, E. D., Vingron, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose Corgi, a new sequence-to-activity model that accurately predicts genome-wide gene expression and epigenetic signals, even in previously unseen cell types.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Sequence-to-activity models have been very successful in predicting gene expression, chromatin accessibility, and epigenetic marks from DNA sequences alone. However, current state-of-the-art models have a fundamental limitation: they cannot extrapolate beyond the cell types and conditions included in their training dataset. Here, we introduce a new approach that is designed to overcome this limitation: Corgi, a new context-aware sequence-to-activity model that accurately predicts genome-wide gene expression and epigenetic signals, even in previously unseen cell types. We designed an architecture that strives to emulate the cell: Corgi integrates DNA sequence and trans-regulator expression to predict the coverage of multiple assays including chromatin accessibility, histone modifications, and gene expression. We define trans-regulators as transcription factors, histone modifiers, transcriptional coactivators, and RNA binding proteins, which directly modulate chromatin states, gene expression, and mRNA decay. Trained on a diverse set of bulk and single cell human datasets, Corgi has robust predictive performance, approaching experimental-level accuracy in gene expression predictions in previously unseen cell types, while also setting a new state-of-the-art level for joint cross-sequence and cross-cell type epigenetic track prediction.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.25.661447v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Nanopore duplex sequencing reveals patterns of asymmetric states of 5hmC and 5mC in the medaka brain genome</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Santana Garcia, W., Fitzgerald, T., Loosli, F., Wittbrodt, J., Birney, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We use Nanopore duplex sequencing, i.e, the sequencing of both strands of DNA from a single double stranded molecule, to profile the co-occurrence of 5hmC with 5mC in CpG dinucleotides of medaka brain tissues and show that 5</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The nucleotide modification 5-methylcytosine (5mC), has been extensively described in terms of its genomic distribution and function. However, the distribution and functional contributions of its oxidised state, 5-hydroxymethylcytosine (5hmC), and its interplay with 5mC remain to be further characterised. Moreover, the hemi-methylation and hemi-hydroxymethylation of CpG dinucleotides in the genome also remain largely unexplored. Here, we use Nanopore duplex sequencing, i.e, the sequencing of both strands of DNA from a single double stranded molecule, to profile the co-occurrence of 5hmC with 5mC in CpG dinucleotides of medaka (japanese rice paddy fish, Oryzias latipes) brain tissues. We characterised asymmetrical patterns of hemi-modified states of CpGs and show using duplex resolution that in the asymmetrical state 5hmC/5mC, 5hmC is preferentially found in the antisense strand of DNA in intron regions, in particular in splice sites, whereas the exons have an enrichment in 5hmC in the sense strand. Furthermore, we found that in the 5hmC/5mC state, 5hmC preferentially occurs along one strand of the DNA duplex in specific families of transposable elements. Altogether these results shed more light on understanding the function and distribution of 5hmC and its mutual role with 5mC in the genome.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.24.661039v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Predictive Analysis of Methylation Patterns in Oral Squamous Cell Carcinoma (OSCC) Using Machine Learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sarkar, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A highly accurate and generalizable machine learning model for DNA methylation in oral squamous cell carcinoma (OSCC).</p><p style="font-size: 16px; text-align: justify;">Background: Oral and oropharyngeal cancers are the most common types of head and neck cancers, with over 90% originating from squamous cells in the mouth and throat. Chronic tobacco and alcohol use, inflammation, viral infections, betel quid chewing, and genetic predisposition are major risk factors for OSCC, which kills over 100,000 patients annually. Epigenetic mechanisms, such as DNA methylation, can silence tumour suppressor genes, contributing to cancer progression and patient outcomes in oral squamous cell carcinoma (OSCC). Objective: This study aimed to predict prominent methylation signatures that can distinguish OSCC from normal cells. Methods: Machine learning algorithms like Support Vector Machine (SVM), Random Forest (RF), and Multilayer Perceptron (MLP) were implemented using R packages and a balanced training dataset consisting of M-values of methylated CpG sites from 46 matched OSCC and normal adjacent tissue samples. Results: MLP model demonstrated the highest accuracy of 92% on the training dataset and 100% on the blind dataset, even with a reduced feature set of just 10 significantly differentially methylated CpG sites. Conclusion: A highly accurate and generalizable machine learning model was developed using the Multi-Layer Perceptron with multiple layers (MLP-ml) algorithm, which achieved an accuracy of 95% on an independent validation dataset of 15 OSCC tumors and 7 non-tumor adjacent tissue samples.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.19.660641v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CagY sequence and structural motifs are associated with ancestry and 1 disease in world-wide Helicobacter pylori strains</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lopez-Luis, M. A., Torres, R. C., Solnick, J. V., Camargo, M. C., Mendez-Tenorio, A., Torres, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We comprehensively analysed cagY in a worldwide collection of strains to unmask any correlation between ancestry and/or disease and sequence diversity, including an analysis of critical domains and associated structural changes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Helicobacter pylori (Hp) interacts with gastric epithelial cells using a multi-protein type 4 secretion system (T4SS) whose function is regulated by CagY. CagY is the T4SS backbone, connecting the bacterial cytoplasm with the epithelial cytoplasm and interacting with most other T4SS proteins. However, its mechanisms of action are unknown. We aimed to comprehensively analyse cagY in a worldwide collection of strains to unmask any correlation between ancestry and/or disease and sequence diversity, including an analysis of critical domains and associated structural changes. We used 674 cagY sequences derived from1,012 Hp genomes, sequenced with Single Molecule Real-Time from the Hp Genome Project. Phylogenetic and principal component analyses (PCA) revealed a strong population structure among European, Asian, African and American clusters, which interfered with other analyses. To address this heterogeneity, dimensional reduction analysis with PCA and Linear Discriminant Analyses were used to minimize interference before studying possible association of cagY diversity with disease. For this analysis, cagY genes were fragmented using unitig-caller with a designed analytical pipeline that clearly separated the non-atrophic gastritis (NAG), advanced intestinal metaplasia and gastric cancer (GC) groups. Sixty-four unitigs significantly separated GC from NAG based on cagY sequences, and the most GC-associated unitigs were localized in the middle repeat region of the protein (MRR), particularly in the A module, including the cysteine-containing motif YLDCVSQ. Proline was enriched in the MRR and VirB10 regions. The consequences of the diversity on the protein structure were analysed using ChimeraX. The validated and stabilized 14-multimer model showed major structural changes along the multimer-structure, particularly in the cysteine-rich MRR region, that might be one mechanism by which CagY regulates T4SS function.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.20.660796v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">16S-23S-5S rRNA Database: a comprehensive integrated database of archaeal and bacterial rRNA sequences, alignments, intragenomic heterogeneity, and secondary structures</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chodvadiya, J., Shukla, G., Sharma, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce the 16S-23S-5S rRNA gene database, a comprehensive and intuitive platform encompassing around 5000 complete bacterial and archaeal genomes, and a foundational resource for large-scale investigations in microbial systematics, genome evolution, and ribosomal biology, benefiting taxonom</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The 16S-23S-5S ribosomal RNA (rRNA) gene operon is a cornerstone of ribosomal architecture and function, making it an indispensable molecular marker for microbial taxonomy and phylogenetics. Despite the extensive development of 16S rRNA gene databases, a significant void persists in comprehensive resources that integrate all three rRNA genes (16S, 23S, and 5S), their precise alignments, intragenomic heterogeneity, and intricate structural details. To address this critical research gap, we introduce the 16S-23S-5S rRNA Database, a meticulously curated and intuitive platform encompassing around 5000 complete bacterial and archaeal genomes. This database provides granular operon-level annotations, robust alignment data, quantitative copy number statistics, percent identity matrices, and predicted secondary structures for each rRNA component. Our rigorous methodology integrates covariance models from RFAM with BLAST-based analyses for sequence comparisons, Clustal Omega for multiple sequence alignment, and Circos for genomic visualization, thereby elucidating the genomic architecture, sequence diversity, and structural conservation of rRNA genes. Crucially, the database systematically accounts for intragenomic heterogeneity, a factor paramount for accurate microbial classification and the inference of evolutionary trajectories. Our analysis reveals prevalent phyla such as Pseudomonadota (1,891 genomes), Actinomycetota (846 genomes), and Bacillota (773 genomes), with Bacillota exhibiting the highest average rRNA gene copy numbers. By integrating these comprehensive elements, the 16S-23S-5S rRNA Database not only fills a substantial lacuna in the field of rRNA data analysis but also establishes a foundational resource for large-scale investigations in microbial systematics, genome evolution, and ribosomal biology, benefiting taxonomists, microbial genomics, and microbiome researchers alike.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.23.661116v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">NucleoBench: A Large-Scale Benchmark of Neural Nucleic Acid DesignAlgorithms</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shor, J., Strand, E., McLean, C. Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A large-scale comparison of sequence design algorithms across 16 biological tasks (such as transcription factor binding and gene expression) and 9 design algorithms.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> One outstanding open problem with high therapeutic value is how to design nucleic acid sequences with specific properties. Even just the 5' UTR sequence admits 2*10^120 possibilities, making exhaustive exploration impossible. Although the field has focused on developing high-quality predictive models, techniques for generating sequences with desired properties are often not well benchmarked. Lack of benchmarking hinders production of the best molecules from high-quality models, and slows improvements to design algorithms. In this work, we performed the first large-scale comparison of modern sequence design algorithms across 16 biological tasks (such as transcription factor binding and gene expression) and 9 design algorithms. Our benchmark, NucleoBench, compares design algorithms on the same tasks and start sequences across more than 400K experiments, allowing us to derive unique modeling insights on the importance of using gradient information, the role of randomness, scaling properties, and reasonable starting hyperparameters on new problems. We use these insights to present a novel hybrid design algorithm, AdaBeam, that outperforms existing algorithms on 11 of 16 tasks and demonstrates superior scaling properties on long sequences and large predictors. Our benchmark and algorithms are freely available online (https://github.com/move37-labs/nucleobench).</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.20.660785v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Mapping Fusarium Wilt and Sterility Mosaic Disease Resistance-Associated Genomic Regions and Haplotype Variants in Pigeonpea</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rangari, S. K., Krishnappa, S., Sharma, M., Sharma, V. K., Dube, N., Gangurde, S. S., Sharma, V., Abbai, R., Bomma, N., Belliappa, S., Bhutia, K. L., Thudi, M., Gangashetty, P. I., Pandey, M. K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A genome-wide association study on 176-genotype panel to identify genomic regions associated with FW and SMD resistance in pigeonpea.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The occurrence of fusarium wilt and sterility mosaic disease in genial conditions causes significant yield losses in pigeonpea. The present genome-wide association study (GWAS) was employed on 176-genotype panel to identify candidate genomic regions associated with FW and SMD resistance in pigeonpea. A total of 869,447 filtered SNP markers were used for GWAS analysis. GWAS analysis identified significant genomic regions for FW on chromosomes 05, 10, and 11 and for SMD on chromosome 09. Four resistant sources, namely, ICP20096, ICP20097, ICPL87119 and ICP13304 were identified as a resistance source for both FW and SMD. Total 12 markers from chromosome 11 for FW and 4 markers from chromosome 09 for SMD were validated using the KASP genotyping approach. We have identified resistant haplotypes for FW on chromosome 10 and 11 and for SMD on chromosome 09. Moreover, we have identified four InDels for FW on chromosome 5 in Two component response regulator gene (Cc_11098) and four InDels for SMD on chromosome 9 in genes, namely, LRR and NB-ARC domain disease resistance protein (Cc_21069), Flavonoid 3', 5'-hydroxylase (Cc_21070) respectively. For FW, Cc_10507586 region from chromosome 5 is identified in ICPL87119 and ICPL20096 donors. A region Cc_21637656 from chromosome 11 is identified in the ICPL87119 but not in the ICPL20096. Similarly, a region Cc_4438493 from chromosome 10 is identified in the ICPL20096 but not in the ICPL87119. For SMD, Cc_24131177 region is identified in both the SMD resistance donors (ICPL20096 and ICPL87119). Ccsmd04 region from chromosome 4 has been identified in ICPL87119 but not in the ICPL20096. These genomic regions can be brought together into single elite genetic background with the help of the markers identified in this study. The validated markers and donor lines in this study offer valuable tools and resources for the genetic improvement of pigeonpea cultivars with enhanced FW and SMD resistance.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.19.660531v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genomic analysis reveals the interplay between ABA-GA in determining dormancy duration in groundnut</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mohinuddin, D. K., Gangurde, S. S., Khan, H., Bomireddy, D., Sharma, V., Shah, P., Sagar, U. N., Dube, N., Senthil, R., Tembhurne, B. V., Nayak, V. H., Amaregouda, A., Babu, K., Singh, K., Janila, P., Guo, B., Liao, B., Varshney, R. K., Pandey, M. K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A mini-core collection of 184 accessions was phenotyped to identify donors in each agronomic type, in addition to analysing data on whole genome re-sequencing and multi-season phenotypic evaluations to identify stable marker-trait associations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Groundnut is an important leguminous crop however; its productivity and seed quality are frequently reduced due to lack of fresh seed dormancy (FSD). To address this challenge, a mini-core collection of 184 accessions was phenotyped to identify donors in each agronomic type, in addition to analysing data on whole genome re-sequencing and multi-season phenotypic evaluations to identify stable marker-trait associations (MTAs) associated with FSD. Phenotypic analysis revealed substantial variability in dormancy durations, with days to 50% germination (DFG) ranging from 1 to 30 days. Multi-locus genome-wide association studies (ML-GWAS) identified 27 MTAs in individual seasons and 12 MTAs in pooled seasons data, respectively. Key candidate genes identified included Cytochrome P450 superfamily proteins, protein kinase superfamily proteins, and MYB transcription factors involved in the Abscisic acid (ABA) pathway, as well as F-box interaction domain proteins, ATP-binding ABC transporters, associated with the Gibberellic acid (GA) pathway. SNP-based KASP (Kompetitive Allele-Specific Polymerase chain reaction) markers for 12 SNPs were developed and validated, of these 6 markers (snpAH00577, snpAH00580, snpAH00582, snpAH00585, snpAH00586 and snpAH00588) showed polymorphism between dormant and non-dormant lines. Incorporating favourable dormant alleles into breeding strategies could enable the development of high-yielding cultivars with a dormancy period of 2-3 weeks.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.20.660118v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">R-loops shape chromatin architecture to promote balanced lineage allocation during differentiation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chao, C.-H., Fazzio, T. G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that R-loop loss leads to a striking reduction in H2A.Z occupancy at both active and bivalent promoters, accompanied by increased nucleosome density, revealing a previously unrecognized role for R-loops in maintaining promoter architecture and preserving balanced lineage trajectories.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> R-loops, RNA:DNA hybrids that often form co-transcriptionally, are emerging as key regulators of genome function, yet their roles in shaping chromatin architecture and developmental potential remain incompletely defined. Here, we use inducible Rnaseh1 expression in mouse embryonic stem cells (mESCs) to achieve acute, global R-loop depletion and systematically interrogate their impact on chromatin structure and lineage specification. We find that R-loop loss has minimal effect on steady-state gene expression or self-renewal. Instead, it leads to a striking reduction in H2A.Z occupancy at both active and bivalent promoters, accompanied by increased nucleosome density, revealing a previously unrecognized role for R-loops in maintaining promoter architecture. During gastruloid differentiation, R-loop-depleted mESCs exhibit a pronounced bias toward ectodermal fates, along with dysregulation of lineage-specific transcription factors and impaired cell-cell signaling. Consistent with these alterations, R-loop-depleted cells show widespread perturbations in gene regulatory networks across several early cell types. These findings uncover a critical role for R-loops in shaping the H2A.Z chromatin landscape and preserving balanced lineage trajectories during early development, offering new insights into the epigenomic regulation of stem cell fate.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.23.661125v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Facilitating Analysis and Dissemination of Proteomics data through Metadata Integration in MaxQuant</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Viegener, W., Urazbakhtin, S., Ferretti, D., Cox, J., Xiao, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the metadata integration in MaxQuant, which provides a user-friendly way to export metadata as SDRF, the standard format that maps sample properties to proteomics data files.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Metadata plays an essential role in the analysis and dissemination of proteomics data. It annotates sample information for output tables from library searches and displays sample information from data files in public repositories. However, integrating metadata into data analysis can be time-consuming and is not well standardized. Then inconsistent metadata formats in public repositories would make it even more difficult for other researchers to reproduce and reuse these public datasets. Here we present the metadata integration in MaxQuant, which provides a user-friendly way to export metadata as SDRF, the standard format that maps sample properties to proteomics data files. We have also implemented the annotation of output tables with the SDRF file, enabling users to perform seamless downstream data analysis with annotated output tables. These new features provide a simple and standardized approach to creating and leveraging standardized metadata. They will greatly facilitate data analysis and improve the reusability and reproducibility of public proteomics datasets.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.20.660677v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SpliceImpactR maps alternative RNA processing events driving protein functional diversity</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wakefield, Z. P., Fiszbein, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> SpliceImpactR, an open-source tool that systematically identifies RNA isoform switches, including alternative first and last exons, exon skipping, intron retention, hybrid exons and splice site variation, across the human transcriptome, and predicts their impact on encoded proteins.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Alternative RNA processing is a key regulator of gene expression, driving transcript and proteomic diversity essential for cell function. However, the precise impacts of different alternative RNA processing events on protein function remain poorly understood. Here, we introduce SpliceImpactR, an open-source tool that systematically identifies RNA isoform switches, including alternative first and last exons, exon skipping, intron retention, hybrid exons, and splice site variation, across the human transcriptome, and predicts their impact on encoded proteins. We find that intron retention and hybrid exons frequently alter the coding potential of transcripts. Strikingly, when both isoforms remain protein-coding, 87% of alternative RNA processing events result in substantial changes to the protein sequence. Among these, alternative last exons drive the largest changes, frequently disrupting protein-protein interactions. Across human tissues, alternative first exons drive the largest relative changes in proteins, often resulting in tissue-specific protein domains. Notably, frameshifts introduced by alternative splicing are often rescued by co-regulated alternative downstream exons, suggesting a buffering mechanism in isoform regulation. Moreover, we show that alternative splicing exhibits gradual, tissue-specific variation, rather than binary on/off behavior, enabling conserved regulation of protein domains across tissues. Together, our results provide a proteome-wide view of splicing regulation, uncovering widespread, context-dependent impacts of alternative splicing on the human proteome.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.20.660706v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Novel algal modular LOV domain proteins expand opto-biotechnological avenues for controlling of eukaryotic riboswitching, translational and proteolytic processes.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Manisha,, Singh, R., KATERIYA, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We have identified 13 novel algal modular LOV domain-containing proteins across the algal system, which can be used as light-controlled translational, ribogenetic and proteolytic switches, and pave the way for opto-biomanufacturing of valuable bioactive.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Light, Oxygen, or Voltage (LOV) domains mediate blue light-gated signal transduction, regulating diverse opto-biological functions. LOV domain functions either standalone or fused with effector domains, regulating the downstream signalling process. The current repertoire of LOV domain-based tools is limited to a relatively small number of naturally occurring proteins. In this study, we have identified novel algal LOV domain fused with different effector domains as potential light-mediated translational, ribogenetic and proteolytic switches, highlighting their unexplored avenues of opto-biotechnology. LOV-domain fusion with eIF4E suggests its potential as a light-controlled translational switch and as an opto-ribogenetic regulator. LOV-SppA might be used as a light-gated proteolytic switch. Additionally, LOV coupled with UFD1, UbiH, mannosyl-oligosaccharide glucosidase and biosynthetic gene cluster (BCG) molecular chassis pave the way for opto-biomanufacturing strategies of relevant valuable algal bioactive. Here, we report the discovery of 13 novel algal modular LOV domain-containing proteins across the algal system through comprehensive bioinformatics, bio-curation and systems biology approaches. It offers important insights into the structural and functional diversity of LOV photoreceptors in algae. Hence, these newly identified modular LOV domain-containing proteins expand the platform of opto-biotechnology applications. These finding lay the foundation for future research on the mechanistic basis for light-driven signalling cascade of RNA, translational and protein homeostasis in algae, and potentiate development of next-generation opto-biotechnological tools for synthetic biology, optogenetics and opto-biomanufacturing of valuable bioactive via regulation of biosynthetic gene cluster (BGC) in green lineage.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.20.660589v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A pre-menopausal single-cell atlas for ovarian drug discovery</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> VanBenschoten, H., Nugmanova, Z., Olmsted, A. M., Gingerich, I., Kratka, C., Huang, R., Duncan, F., Kholod, O., Goods, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a single-cell atlas of ovarian cells from pre-menopausal women and identify ovarian gene targets for fertility management and contraceptive target discovery.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell atlas efforts have reshaped the way we understand cells across the human body. Despite their power, they have not been effectively used to study ovarian biology in the context of other tissues, nor have they comprehensively incorporated healthy tissues from pre-menopausal donors. A focused pre-menopausal single-cell atlas could both advance our understanding of this life stage and support identification of ovarian gene targets for indications prevalent among younger demographics, such as fertility management. Here, we present an integrated resource of single-cell datasets from pre-menopausal women (PreMeno Atlas), comprising 511,365 cells from 14 tissues, including the ovary. This unified resource enables transcriptomic comparisons across cell types, tissues, and organs within the pre-menopausal context. Our analysis revealed distinct ovarian cell gene signatures, with theca and stromal cells exhibiting the most unique transcriptional profiles among ovarian cell types. We further leveraged the PreMeno Atlas to prioritize ovary-specific genes with potential druggability, particularly G-protein coupled receptors (GPCRs), identifying GPR78, ADRB3, GPR20, and GPR101 as candidate targets. Finally, we assessed species homology of ovarian cell marker genes using a harmonized mouse ovulation and spatial transcriptomics dataset. Collectively, this work establishes the PreMeno Atlas as a resource for ovarian biology research and contraceptive target discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.20.660779v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The AlphaFold Database Ages</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tsitsa, I., Conev, A., David, A., Islam, S. A., Sternberg, M. J. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> The AlphaFold database provides 200M protein structures predicted from UniProt in 2022 from sequences in UniProt (version 2024_05), but only 575 of the 20,504 full-length human structures in the AlphaFolding database conflict with UniProt.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The AlphaFold database provides 200M protein structures predicted by AlphaFold2 in 2022 from sequences in UniProt. However, of the 20,504 full-length human structures in the AlphaFold database, 575 entries conflict with UniProt (version 2024_05); and there is a similar discrepancy for other species. This highlights how bioinformatics resources as exemplified by the AlphaFold database can rapidly age.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.22.660930v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MUDflow: Combining Neural Networks, UMAP and DBM Clustering to Identify Cell Populations Accurately, Quickly and Easily in Mass and Fluorescence Cytometry</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Meehan, C., Meehan, S., Ebrahimian, J., Moore, W., Parks, D., Walther, G., Herzenberg, L. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce an easily trainable multilayer perceptron neural network for automatic gating (MLPgater) and show that it is the fastest and uniquely able to replace manual gating except for training purposes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Identifying cell populations in flow cytometry data is mostly done via a "manual gating" method that often lacks verifiability and reproducibility, even in the hands of experienced investigators. Recently developed automatic gating methods have been shown to have good performance in cell population identification, but may require fine-tuned setup from experts or struggle to identify small populations. Here, we introduce an easily trainable multilayer perceptron neural network for automatic gating (MLPgater). Compared to the three popular automatic gating methods LDA, FlowSOM and PhenoGraph on three mass and six fluorescence cytometry datasets, MLPgater is most accurate by a substantial margin, tied with LDA as the fastest and uniquely able to replace manual gating except for training purposes. Furthermore, we show that combining MLPgater with UMAP's guided dimensionality reduction feature and DBM's clustering (MUDflow) effectively detects new populations that did not exist or were not identified in the training set.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.19.660626v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A k-mer-based estimator of the substitution rate between repetitive sequences</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wu, H., Blanca, A., Medvedev, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel estimator for the mutation rate between low-repetitive and high-rePETitive k-mer sets and show that it remains accurate for repetitive genomic sequences, such as the centromere of a human chromosome.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> K-mer-based analysis of genomic data is ubiquitous, but the presence of repetitive k-mers continues to pose problems for the accuracy of many methods. For example, the Mash tool (Ondov et al 2016) can accurately estimate the substitution rate between two low-repetitive sequences from their k-mer sketches; however, it is inaccurate on repetitive sequences such as the centromere of a human chromosome. Follow-up work by Blanca et al. (2021) has attempted to model how mutations affect k-mer sets based on strong assumptions that the sequence is non-repetitive and that mutations do not create spurious k-mer matches. However, the theoretical foundations for extending an estimator like Mash to work in the presence of repeat sequences have been lacking. In this work, we relax the non-repetitive assumption and propose a novel estimator for the mutation rate. We derive theoretical bounds on our estimator's bias. Our experiments show that it remains accurate for repetitive genomic sequences, such as the alpha satellite higher order repeats in centromeres. We demonstrate our estimator's robustness across diverse datasets and various ranges of the substitution rate and k-mer size. Finally, we show how sketching can be used to avoid dealing with large k-mer sets while retaining accuracy. Our software is available at https://github.com/medvedevgroup/Repeat-Aware_Substitution_Rate_Estimator.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.19.660607v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

