<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Friday, 25 April 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multi-omics analysis constructs a novel neuroendocrine prostate cancer classifier and classification system.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shen J, Lu L, Chen Z, Guo W, Wang S, Liu Z, Gong X, Qi Y, Jin R, Zhang C.</p>
                <p style="font-size: 16px;"><em>Source: MED</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel classifier and classification system for neuroendocrine prostate cancer (NEPC) with poor prognosis and high heterogeneity .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Neuroendocrine prostate cancer (NEPC), a subtype of prostate cancer (PCa) with poor prognosis and high heterogeneity, currently lacks accurate markers. This study aims to identify a robust NEPC classifier and provide new perspectives for resolving intra- tumoral heterogeneity. Multi-omics analysis included 19 bulk transcriptomics, 14 single-cell transcriptomics, 1 spatial transcriptomics, 16 published NE signatures and 10 cellular experiments combined with multiple machine learning algorithms to construct a novel NEPC classifier and classification. A comprehensive single-cell atlas of prostate cancer was created from 70 samples, comprising 196,309 cells, among which 9% were identified as NE cells. Within this framework and in combination with bulk transcriptomics, a total of 100 high-quality NE-specific feature genes were identified and differentiated into NEPup sig and NEPdown sig. The random forest (RF) algorithm proved to be the most effective classifier for NEPC, leading to the establishment of the NEP100 model, which demonstrated robust validation across various datasets. In clinical settings, the use of the NEP100 model can greatly improve the diagnostic and prognostic prediction of NEPC. Hierarchical clustering based on NEP100 revealed four distinct NEPC subtypes, designated VR_O, Prol_N, Prol_P, and EMT_Y, each of which presented unique biological characteristics. This allows us to select different targeted therapeutic strategies for different subtypes of phenotypic pathways. Notably, NEP100 expression correlated positively with neuroendocrine differentiation and disease progression, while the VR-NE phenotype dominated by VR_O cells indicated a propensity for treatment resistance. Furthermore, AMIGO2, a component of the NEP100 signature, was associated with chemotherapy resistance and a poor prognosis, indicating that it is a pivotal target for future therapeutic strategies. This study used multi-omics analysis combined with machine learning to construct a novel NEPC classifier and classification system. NEP100 provides a clinically actionable framework for NEPC diagnosis and subtyping.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40263498/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Basal Xenobot transcriptomics reveals changes and novel control modality in cells freed from organismal influence.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pai VP, Pio-Lopez L, Sperry MM, Erickson P, Tayyebi P, Levin M.</p>
                <p style="font-size: 16px;"><em>Source: MED</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We investigate the effects of morphology and nascent emergent life history on gene expression in the basal (no engineering, no sculpting) form of Xenobots -autonomously motile constructs derived from Xenopus embryo ectodermal cell explants.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Would transcriptomes change if cell collectives acquired a novel morphogenetic and behavioral phenotype in the absence of genomic editing, transgenes, heterologous materials, or drugs? We investigate the effects of morphology and nascent emergent life history on gene expression in the basal (no engineering, no sculpting) form of Xenobots -autonomously motile constructs derived from Xenopus embryo ectodermal cell explants. To investigate gene expression differences between cells in the context of an embryo with those that have been freed from instructive signals and acquired novel lived experiences, we compare transcriptomes of these basal Xenobots with age-matched Xenopus embryos. Basal Xenobots show significantly larger inter-individual gene variability than age-matched embryos, suggesting increased exploration of the transcriptional space. We identify at least 537 (non-epidermal) transcripts uniquely upregulated in these Xenobots. Phylostratigraphy shows a majority of transcriptomic shifts in the basal Xenobots towards evolutionarily ancient transcripts. Pathway analyses indicate transcriptomic shifts in the categories of motility machinery, multicellularity, stress and immune response, metabolism, thanatotranscriptome, and sensory perception of sound and mechanical stimuli. We experimentally confirm that basal Xenobots respond to acoustic stimuli via changes in behavior. Together, these data may have implications for evolution, biomedicine, and synthetic morphoengineering.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40263484/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Definer: A computational method for accurate identification of RNA pseudouridine sites based on deep learning.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bo Han, Sudan Bai, Yang Liu, Jiezhang Wu, Xin Feng, Ruihao Xin</p>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a deep learning-based computational method, Definer, to accurately identify RNA pseudouridine modification sites from high-throughput RNA sequence data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Pseudouridine is an important modification site, which is widely present in a variety of non-coding RNAs and is involved in a variety of important biological processes. Studies have shown that pseudouridine is important in many biological functions such as gene expression, RNA structural stability, and various diseases. Therefore, accurate identification of pseudouridine sites can effectively explain the functional mechanism of this modification site. Due to the rapid increase of genomics data, traditional biological experimental methods to identify RNA modification sites can no longer meet the practical needs, and it is necessary to accurately identify pseudouridine sites from high-throughput RNA sequence data by computational methods. In this study, we propose a deep learning-based computational method, Definer, to accurately identify RNA pseudouridine loci in three species, Homo sapiens, Saccharomyces cerevisiae and Mus musculus. The method incorporates two sequence coding schemes, including NCP and One-hot, and then feeds the extracted RNA sequence features into a deep learning model constructed from CNN, GRU and Attention. The benchmark dataset contains data from three species, H. sapiens, S. cerevisiae and M. musculus, and the results using 10-fold cross-validation show that Definer significantly outperforms other existing methods. Meanwhile, the data sets of two species, H. sapiens and S. cerevisiae, were tested independently to further demonstrate the predictive ability of the model. In summary, our method, Definer, can accurately identify pseudouridine modification sites in RNA.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40273178/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Azurify integrates cancer genomics with machine learning to classify the clinical significance of somatic variants</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bigdeli A, Chandrashekar DS, Chitturi A, Rushton C, Mackinnon AC, Segal J, Harada S, Sacan A, Faryabi RB.</p>
                <p style="font-size: 16px;"><em>Source: PPR</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Azurify - a computational tool that integrates machine learning, public resources recommended by professional societies, and clinically annotated data to classify the pathogenicity of variations in precision cancer medicine.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurate classification of somatic variations from high-throughput sequencing data has become integral to diagnostics and prognostics across various cancers. However, the classification of these variations remains highly manual, inherently variable, and largely inaccessible outside specialized laboratories. Here, we introduce Azurify - a computational tool that integrates machine learning, public resources recommended by professional societies, and clinically annotated data to classify the pathogenicity of variations in precision cancer medicine. Trained on over 15,000 clinically classified variants from 8,202 patients across 138 cancer phenotypes, Azurify achieves 99.1% classification accuracy for concordant pathogenic variants in data from two external clinical laboratories. Additionally, Azurify reliably performs precise molecular profiling in leukemia cases. Azurify's unified, scalable, and modular framework can be easily deployed within bioinformatics pipelines and retrained as new data emerges. In addition to supporting clinical workflows, Azurify offers a high-throughput screening solution for research, enabling genomic studies to identify meaningful variant-disease associations with greater efficiency and consistency.</p>
                <p><a href="https://doi.org/10.1101/2025.04.18.649588" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A review of machine learning methods for imbalanced data challenges in chemistry</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jiang J, Zhang C, Ke L, Hayes N, Zhu Y, Qiu H, Zhang B, Zhou T, Wei G.</p>
                <p style="font-size: 16px;"><em>Source: PMC</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A look at some of the best-loved movies of the past few years.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> </p>
                <p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12013631/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Learning and teaching biological data science in the Bioconductor community.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Drnevich J, Tan FJ, Almeida-Silva F, Castelo R, Culhane AC, Davis S, Doyle MA, Geistlinger L, Ghazi AR, Holmes S, Lahti L, Mahmoud A, Nishida K, Ramos M, Rue-Albrecht K, Shih DJH, Gatto L, Soneson C.</p>
                <p style="font-size: 16px;"><em>Source: MED</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We provide an overview of key resources and best practices available within the Bioconductor project-an open-source software community focused on omics data analysis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Modern biological research is increasingly data-intensive, leading to a growing demand for effective training in biological data science. In this article, we provide an overview of key resources and best practices available within the Bioconductor project-an open-source software community focused on omics data analysis. This guide serves as a valuable reference for both learners and educators in the field.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40261894/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Characterization of lactylation-based phenotypes and molecular biomarkers in sepsis-associated acute respiratory distress syndrome.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang Y, Wei A, Su Z, Shi Y, Li X, He L.</p>
                <p style="font-size: 16px;"><em>Source: MED</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified two distinct lactylation-based phenotypes in patients with sepsis-associated acute respiratory distress syndrome (ARDS) and identified potential biomarkers to facilitate the application of these phenotypes to clinical practice.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Sepsis-associated acute respiratory distress syndrome (ARDS) is a heterogeneous disease with high morbidity and mortality. Lactylation plays a crucial role in sepsis and sepsis-induced lung injury. This study aimed to identify distinct lactylation-based phenotypes in patients with sepsis-associated ARDS and determine relevant molecular biomarkers. We analyzed blood transcriptome and clinical data from patients with sepsis-associated ARDS and calculated the lactylation activity. KEGG pathway analysis, drug sensitivity prediction, and immune cell infiltration analysis were performed. Candidate molecular biomarkers were identified by intersecting the feature genes extracted from four machine learning models. Lactylation activity showed significant heterogeneity among patients with sepsis-associated ARDS, which enabled the classification into low- and high-lactylation activity phenotypes. Patients with high-lactylation experienced longer hospital stays and higher mortality rates, as well as distinct signaling pathways, drug responses, and circulating immune cell abundances. Six key markers (ALDOB, CCT5, EP300, PFKP, PPIA, and SIRT1) were identified to differentiate the two lactylation activity phenotypes, all significantly correlated with circulating immune cell populations. This study revealed significant heterogeneity in lactylation activity phenotypes among patients with sepsis-associated ARDS and identified potential biomarkers to facilitate the application of these phenotypes in clinical practice.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40263316/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An approach of molecular-fingerprint prediction implementing a GAT.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Deng C, Zhou C, Shi L, Wang B.</p>
                <p style="font-size: 16px;"><em>Source: MED</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel approach to compound identification based on the graph attention network (GAT) model.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In the domain of metabolomics, the accurate identification of compounds is paramount. However, this process is hindered by the vast number of metabolites, which poses a significant challenge. In this study, a novel approach to compound identification is proposed, namely a molecular-fingerprint prediction method based on the graph attention network (GAT) model. The method involves the processing of fragmentation-tree data derived from tandem mass spectrometry (MS/MS) data computation and the subsequent processing of fragmentation-tree graph data with a technique inspired by natural language processing. The model is then trained using a 3-layer GAT model and a 2-layer linear layer. The results demonstrate the method's efficacy in molecular-fingerprint prediction, with the prediction of molecular fingerprints from MS/MS spectra exhibiting a high degree of accuracy. Firstly, this model achieves excellent performance in receiver operating characteristic (ROC) and precision-recall curves. The factors that have the most influence on the resultant performance are identified as edge features using different training parameters. Then, better performance is achieved for accuracy and <i>F</i> <sub>1</sub> score in comparison with MetFID. Secondly, the model performance was validated by querying the molecular libraries through methods commonly used in related studies. In the results based on precursor mass querying, the proposed model achieves comparable performance with CFM-ID; in the results based on molecular formula querying, the model achieves better performance than MetFID. This study demonstrates the potential of the GAT model for compound identification tasks and provides directions for further research.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40264881/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Exploring the latent space of transcriptomic data with topic modeling.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Valle F, Caselle M, Osella M.</p>
                <p style="font-size: 16px;"><em>Source: MED</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> This paper leverages the statistical analogies between text and transcriptomic datasets to compare different topic modeling methods when applied to gene expression data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The availability of high-dimensional transcriptomic datasets is increasing at a tremendous pace, together with the need for suitable computational tools. Clustering and dimensionality reduction methods are popular go-to methods to identify basic structures in these datasets. At the same time, different topic modeling techniques have been developed to organize the deluge of available data of natural language using their latent topical structure. This paper leverages the statistical analogies between text and transcriptomic datasets to compare different topic modeling methods when applied to gene expression data. Specifically, we test their accuracy in the specific task of discovering and reconstructing the tissue structure of the human transcriptome and distinguishing healthy from cancerous tissues. We examine the properties of the latent space recovered by different methods, highlight their differences, and their pros and cons across different tasks. We focus in particular on how different statistical priors can affect the results and their interpretability. Finally, we show that the latent topic space can be a useful low-dimensional embedding space, where a basic neural network classifier can annotate transcriptomic profiles with high accuracy.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40264683/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An updated comparison of microarray and RNA-seq for concentration response transcriptomic study: case studies with two cannabinoids, cannabichromene and cannabinol.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gao X, Yourick MR, Campasino K, Zhao Y, Sepehr E, Vaught C, Sprando RL, Yourick JJ.</p>
                <p style="font-size: 16px;"><em>Source: MED</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We provide an updated comparison between microarray and RNA-seq using two cannabinoids, cannabichromene (CBC) and cannabinol (CBN), as case studies.</p><p style="font-size: 16px; text-align: justify;"><h4>Background</h4>Transcriptomic benchmark concentration (BMC) modeling provides quantitative toxicogenomic information that is increasingly being used in regulatory risk assessment of data poor chemicals. Over the past decade, RNA sequencing (RNA-seq) is gradually replacing microarray as the major platform for transcriptomic applications due to its higher precision, wider dynamic range, and capability of detecting novel transcripts. However, it is unclear whether RNA-seq offers substantial advantages over microarray for concentration response transcriptomic studies.<h4>Results</h4>We provide an updated comparison between microarray and RNA-seq using two cannabinoids, cannabichromene (CBC) and cannabinol (CBN), as case studies. The two platforms revealed similar overall gene expression patterns with regard to concentration for both CBC and CBN. However, in spite of the many varieties of non-coding RNA transcripts and larger numbers of differentially expressed genes (DEGs) with wider dynamic ranges identified by RNA-seq, the two platforms displayed equivalent performance in identifying functions and pathways impacted by compound exposure through gene set enrichment analysis (GSEA). Furthermore, transcriptomic point of departure (tPoD) values derived by the two platforms through BMC modeling were on the same levels for both CBC and CBN.<h4>Conclusions</h4>Considering the relatively low cost, smaller data size, and better availability of software and public databases for data analysis and interpretation, microarray is still a viable method of choice for traditional transcriptomic applications such as mechanistic pathway identification and concentration response modeling.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40264021/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrating Multiplex Immunohistochemistry and Machine Learning for Glioma Subtyping and Prognosis Prediction.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xu H, Fan Z, Jiang S, Sun M, Chai H, Zhu R, Liu X, Wang Y, Chen J, Wei J, Mao Y, Shi Z.</p>
                <p style="font-size: 16px;"><em>Source: MED</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel framework integrating multiplex immunohistochemistry (mIHC) and machine learning for glioma subtyping and prognosis prediction, which identified two distinct gliomas subtypes with significant differences in prognosis, clinical characteristics, and molecular profiles.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Glioma subtyping is crucial for treatment decisions, but traditional approaches often fail to capture tumor heterogeneity. This study proposes a novel framework integrating multiplex immunohistochemistry (mIHC) and machine learning for glioma subtyping and prognosis prediction. 185 patient samples from the Huashan hospital cohort were stained using a multi-label mIHC panel and analyzed with an AI-based auto-scanning system to calculate cell ratios and determine the proportion of positive tumor cells for various markers. Patients were divided into two cohorts (training: <i>N</i>¬†=¬†111, testing: <i>N</i>¬†=¬†74), and a machine learning model was then developed and validated for subtype classification and prognosis prediction. The framework identified two distinct glioma subtypes with significant differences in prognosis, clinical characteristics, and molecular profiles. The high-risk subtype, associated with older age, poorer outcomes, astrocytoma/glioblastoma, higher tumor grades, elevated mesenchymal scores, and an inhibitory immune microenvironment, exhibited IDH wild-type, 1p19q non-codeletion, and MGMT promoter unmethylation, suggesting chemotherapy resistance. Conversely, the low-risk subtype, characterized by younger age, better prognosis, astrocytoma/oligodendroglioma, lower tumor grades, and favorable molecular profiles (IDH mutation, 1p19q codeletion, MGMT promoter methylation), indicated chemotherapy sensitivity. The mIHC-based framework enables rapid glioma classification, facilitating tailored treatment strategies and accurate prognosis prediction, potentially improving patient management and outcomes.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40264576/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Prioritization of novel anti-infective stilbene derivatives by combining metabolomic data organization and a stringent 3R-infection model in a knowledge graph‚Ä† ‚Ä† Electronic supplementary information (ESI) available: Figures, NMR data, ECD spectra and HRMS data for all isolated compounds. See DOI: https://doi.org/10.1039/d4ra08421g</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kirchhoffer O, Quir√≥s-Guerrero L, Nitschke J, Nothias L, Burdet F, Marcourt L, Hanna N, Mehl F, David B, Grondin A, Queiroz E, Pagni M, Soldati T, Wolfender J.</p>
                <p style="font-size: 16px;"><em>Source: PMC</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A look at some of the best-loved movies of the past few years.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> </p>
                <p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12015462/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Reference-guided genome assembly at scale using ultra-low-coverage high-fidelity long-reads with HiFiCCL</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jiang, Z., Pan, W., Gao, R., Hu, H., Gao, W., Zhou, M., Yin, Y.-H., Qian, Z., Jin, S., Wang, G.</p>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose HiFiCCL, the first assembly framework specifically designed for ultra-low-coverage high-fidelity reads, using a reference-guided, chromosome-by-chromosome assembly approach.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Population genomics using short-read resequencing captures single nucleotide polymorphisms and small insertions and deletions but struggles with structural variants (SVs), leading to a loss of heritability in genome-wide association studies. In recent years, long-read sequencing has improved pangenome construction for key eukaryotic species, addressing this issue to some extent. Sufficient-coverage high-fidelity (HiFi) data for population genomics is often prohibitively expensive, limiting its use in large-scale populations and broader eukaryotic species and creating an urgent need for robust ultra-low coverage assemblies. However, current assemblers underperform in such conditions. To address this, we propose HiFiCCL, the first assembly framework specifically designed for ultra-low-coverage high-fidelity reads, using a reference-guided, chromosome-by-chromosome assembly approach. We demonstrate that HiFiCCL improves ultra-low-coverage assembly performance of existing assemblers and outperforms the state-of-the-art assemblers on human and plant datasets. Tested on 45 human datasets (~5x coverage), HiFiCCL combined with hifiasm reduces the length of misassembled contigs relative to hifiasm by an average of 21.19% and up to 38.58%. These improved assemblies enhance germline structural variant detection, reduce chromosome-level mis-scaffolding, enable more accurate pangenome graph construction, and improve the detection of rare and somatic structural variants based on the pangenome graph under ultra-low-coverage conditions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.20.649739v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Attention-Based Solution for Synergistic Virus Combination Therapy</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Majidifar, S., Hooshmand, M.</p>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> This paper proposes AI-based models to predict novel drug combinations that can synergistically treat viral diseases.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Computational drug repurposing is vital in drug discovery research because it significantly reduces both the cost and time involved in the drug development process. Additionally, combination therapy--using more than one drug for treatment--can enhance efficacy and minimize the side effects associated with individual drugs. However, there is currently limited research focused on computational approaches to combination therapy for viral diseases. This paper proposes AI-based models to predict novel drug combinations that can synergistically treat viral diseases. To achieve this, we have compiled a comprehensive dataset containing information on viruses, drug compounds, and their approved interactions. We introduce two attention-based models and compare their performance with traditional machine learning and deep learning models in predicting synergistic drug pairs for treating viral diseases. Among all the methods tested, the random forest algorithm and one of the attention-based models utilizing a customized dot product as a predictor showed the highest performance. Notably, two predicted combinations--acyclovir + ribavirin and acyclovir + Pranobex Inosine--have been experimentally validated to produce a synergistic antiviral effect against the herpes simplex virus type 1, as reported in existing literature.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.22.649915v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Task Splitting and Prompt Engineering for Cypher Query Generation in Domain-Specific Knowledge Graphs</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Soleymani, S., Gravel, N. M., Kochut, K., Kannan, N.</p>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel approach, Prompt2Cypher (P2C), which leverages task splitting and prompt engineering to decompose user queries into manageable subtasks, enhancing LLMs' ability to generate accurate Cypher queries that align with the underlying graph database schema.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The integration of large language models (LLMs) with knowledge graphs (KGs) holds significant potential for simplifying the process of querying graph databases, especially for non-technical users. KGs provide a structured representation of domain-specific data, enabling rich and precise information retrieval. However, the complexity of graph query languages, such as Cypher, presents a barrier to their effective use by non-experts. This research addresses the challenge by proposing a novel approach, Prompt2Cypher (P2C), which leverages task splitting and prompt engineering to decompose user queries into manageable subtasks, enhancing LLMs' ability to generate accurate Cypher queries that align with the underlying graph database schema. We demonstrate the effectiveness of P2C in two biological KGs (protein kinase and ion-channel) that differ in size, schema and complexity. Compared to a baseline approach, our method improves query accuracy, as demonstrated by higher Precision, Recall, F1-score, and Jaccard similarity metrics. This work contributes to the ongoing efforts to bridge the gap between domain-specific knowledge graphs and user-friendly graph database query interfaces.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.23.649790v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comparative analysis of genomic prediction approaches for multiple time-resolved traits in maize</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hobby, D., Lindner, R., Mbebi, A. J., Tong, H., Nikoloski, Z.</p>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We compared and contrasted the performance of MegaLMM and dynamicGP as well as their hybrid variants that can handle high-dimensional temporal data for multi-trait genomic prediction.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Ability to accurately predict multiple growth-related traits over plant developmental trajectories has the potential to revolutionize crop breeding and precision agriculture. Despite increased availability of time-resolved data for multiple traits from high-throughput phenotyping platforms of model plants and crops, genomic prediction is largely applied to a small number of traits, often neglecting their dynamics. Here, we compared and contrasted the performance of MegaLMM and dynamicGP as well as their hybrid variants that can handle high-dimensional temporal data for multi-trait genomic prediction. The comparative analysis made use of time series for 50 geometric, colour, and texture traits in a maize multi-parent advanced generation inter-cross (MAGIC) population. The performance of the approaches was assessed using snapshot accuracy and longitudinal accuracy, We found that MegaLMM outperforms dynamicGP in terms of snapshot accuracy, while dynamicGP proved superior in terms of longitudinal accuracy. This study paves the way for careful investigation of factors that affect the capacity to predict dynamics of multiple traits from genetic markers alone. providing insight into the ability to predict multiple traits at a single time point or the dynamics of individual traits over the considered time domain, respectively.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.22.649925v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">BioMiner: A Multi-modal System for Automated Mining of Protein-Ligand Bioactivity Data from Literature</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yan, J., Zhu, J., Yang, Y., Liu, Q., Zhang, K., Zhang, Z., Liu, X., Zhang, B., Gao, K., Xiao, J., Chen, E.</p>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce BioMiner, a multi-modal system designed to automatically extract protein-ligand bioactivity data from thousands to potentially millions of publications.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein-ligand bioactivity data published in literature are essential for drug discovery, yet manual curation struggles to keep pace with rapidly growing literature. Automated bioactivity extraction is challenging due to the multi-modal distribution of information (text, tables, figures, structures) and the complexity of chemical representations (e.g., Markush structures). Furthermore, the lack of standardized benchmarks impedes the evaluation and development of extraction methods. In this work, we introduce BioMiner, a multi-modal system designed to automatically extract protein-ligand bioactivity data from thousands to potentially millions of publications. BioMiner employs a modular, agent-based architecture, leveraging a synergistic combination of multi-modal large language models, domain-specific models, and domain tools to navigate this complex extraction task. To address the benchmark gap and support method development, we establish BioVista, a comprehensive benchmark comprising 16,457 bioactivity entries and 8,735 chemical structures curated from 500 publications. On BioVista, BioMiner validates its extraction ability and provides a quantitative baseline, achieving F1 scores of 0.22, 0.45, and 0.53 for bioactivity triplets, chemical structures, and bioactivity measurement with high throughput (14s/paper on 8 V100 GPUs). We further demonstrate BioMiner's practical utility via three applications: (1) extracting 67,953 data from 11,683 papers to build a pre-training database that improves downstream models performance by 3.1%; (2) enabling a human-in-the-loop workflow that doubles the number of high-quality NLRP3 bioactivity data, helping 38.6% improvement over 28 QSAR models and identification of 16 hit candidates with novel scaffolds; and (3) accelerating the annotation of the protein-ligand structures in PoseBusters benchmark with reported bioactivity, achieving a 5-fold speed increase and 10% accuracy improvement over manual methods. BioMiner and BioVista provide a scalable extraction methodology and a rigorous benchmark, paving the way to unlock vast amounts of previously inaccessible bioactivity data and accelerate data-driven drug discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.22.648951v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Assemblies of long-read metagenomes suffer from diverse errors</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Trigodet, F., Sachdeva, R., Banfield, J. F., Eren, A. M.</p>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that erroneous reporting is pervasive among long-read assemblers and can take many forms, including multi-domain chimeras, prematurely circularized sequences, haplotyping errors, excessive repeats, and phantom sequences.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genomes from metagenomes have revolutionised our understanding of microbial diversity, ecology, and evolution, propelling advances in basic science, biomedicine, and biotechnology. Assembly algorithms that take advantage of increasingly available long-read sequencing technologies bring the recovery of complete genomes directly from metagenomes within reach. However, assessing the accuracy of the assembled long reads, especially from complex environments that often include poorly studied organisms, poses remarkable challenges. Here we show that erroneous reporting is pervasive among long-read assemblers and can take many forms, including multi-domain chimeras, prematurely circularized sequences, haplotyping errors, excessive repeats, and phantom sequences. Our study highlights the need for rigorous evaluation of the algorithms while they are in development, and options for users who may opt for more accurate reads than shorter runtimes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.22.649783v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">High-quality metagenome assembly from nanopore reads with nanoMDBG</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Benoit, G., James, R., Raguideau, S., Alabone, G., Goodall, T., Chikhi, R., Quince, C.</p>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present nanoMDBG, an evolution of the metaM DBG HiFi assembler, designed to support newer ONT sequencing data through a novel pre-processing step that performs fast and accurate error correction in minimizer-space.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Third-generation long-read sequencing technologies, have been shown to significantly enhance the quality of metagenome assemblies. The results obtained using the highly accurate reads generated by PacBio HiFi have been particularly notable yielding hundreds of circularized, complete genomes as metagenome-assembled genomes (MAGs) without manual intervention. Oxford Nanopore Technologies (ONT) has recently improved the accuracy of its sequencing reads, achieving a per-base error rate of approximately 1-2%. Given the high-throughput, convenience and low-cost of ONT sequencing this could accelerate the uptake of long read metagenomics. However, current metagenome assemblers are optimized for PacBio HiFi data and underperform on the latest ONT data and do not scale to the large data sets that it enables. We present nanoMDBG, an evolution of the metaMDBG HiFi assembler, designed to support newer ONT sequencing data through a novel pre-processing step that performs fast and accurate error correction in minimizer-space. Across a range of ONT datasets, including a large 400 Gbp soil sample sequenced specifically for this study, nanoMDBG reconstructs up to twice as many high-quality MAGs as the next best ONT assembler, metaflye, while requiring a third of the CPU time and memory. As a result of these advances, we show that the latest ONT technology can now produce results comparable to those obtained using PacBio HiFi sequencing at the same sequencing depth.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.22.649928v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Advancing Pediatric and Longitudinal DNA Methylation Studies with CellsPickMe, an Integrated Blood Cell Deconvolution Method</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fu, M. P., Edwards, K., Navarro-Delgado, E. I., Merrill, S. M., Kitaba, N. T., Konwar, C., Mandhane, P., Simons, E., Subbarao, P., Moraes, T. J., Holloway, J. W., Turvey, S. E., Kobor, M. S.</p>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a new method for blood cell deconvolution of longitudinal birth cohorts using DNA methylation data from the Canadian Healthy Infant Longitudinal Development (CHILD) cohort and a new algorithm for cross-time-point integration of longitudinal DNAme studies.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Prospective birth cohorts offer the potential to interrogate the relation between early life environment and embedded biological processes such as DNA methylation (DNAme). These association studies are frequently conducted in the context of blood, a heterogeneous tissue composed of diverse cell types. Accounting for this cellular heterogeneity across samples is essential, as it is a main contributor to inter-individual DNAme variation. Integrated blood cell deconvolution of pediatric and longitudinal birth cohorts poses a major challenge, as existing methods fail to account for the distinct cell population shift between birth and adolescence. In this paper, we critically evaluated the reference-based deconvolution procedure and optimized its prediction accuracy for longitudinal birth cohorts using DNAme data from the Canadian Healthy Infant Longitudinal Development (CHILD) cohort. The optimized algorithm, CellsPickMe, integrates cord and adult references and picks DNAme features for each population of cells with machine learning algorithms. It demonstrated improved deconvolution accuracy in cord, pediatric, and adult blood samples compared to existing benchmark methods. CellsPickMe supports blood cell deconvolution across early developmental periods under a single framework, enabling cross-time-point integration of longitudinal DNAme studies. Given the increased resolution of cell populations predicted by CellsPickMe, this R package empowers researchers to explore immune system dynamics using DNAme data in population studies across the life course.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.22.649907v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Chromosome-scale genome assembly of the European common cuttlefish Sepia officinalis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rencken, S., Tushev, G., Hain, D., Ciirdaeva, E., Simakov, O., Laurent, G.</p>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We sequenced and assembled a chromosome-scale genome for the common cuttlefish Sepia officinalis, a model organism for the evolution, brain organization, information processing, development, and behavior in Coleoid cephalopod clade.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Coleoid cephalopods, a subclass of mollusks, exhibit remarkable adaptations, including the largest brains among invertebrates, camera-like eyes, and a distinctive embryonic development. They possess an advanced behavioral repertoire including dynamic camouflage. The common cuttlefish Sepia officinalis has served as a key model organism in various research fields, spanning biophysics, neurobiology, behavior, evolution, ecology and biomechanics. More recently, it has become a model to investigate the neural mechanisms underlying cephalopod camouflage, using quantitative behavioral approaches alongside molecular techniques to characterize the identity, evolution and development of neuronal cell types. Despite significant interest in this animal, a high-quality, annotated genome of its species is still lacking. To address this, we sequenced and assembled a chromosome-scale genome for S. officinalis. The final assembly spans 5.68 billion base pairs and comprises 47 repeat-rich chromosomes. Gene linkage analysis confirms the existence of 47 chromosomes, revealing clear homologies with related species such as Euprymna scolopes and Doryteuthis pealeii. Our work includes a comprehensive gene annotation and full-length transcript predictions that should be helpful for further evolutionary and single-cell expression studies. This genome provides a valuable resource for future research on the evolution, brain organization, information processing, development, and behavior in this important clade.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.22.649952v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Engineering microbial consortia for distributed signal processing</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Duncker, K. E., Shende, A. R., Shyti, I., Ruan, A., D'Cunha, R., Ma, H. R., Venugopal Lavanya, H., Liu, S., Gottel, N., Anderson, D. J., Gunsch, C. K., You, L.</p>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new, generalizable approach that leverages microbial consortia to distribute sensory functions and computational methods to disentangle signals.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> A critical goal in biology is deducing input signals from measurable readouts. Genetic circuits have successfully been designed to respond to specific analytes and produce quantifiable outputs; however, multiplexed signal processing remains challenging. This limitation is partially due to crosstalk, or sensors' non-specific responses to unintended signals. While strategies to achieve orthogonality are promising, they are time-intensive and context-dependent. Here, we introduce a new, generalizable approach that leverages microbial consortia to distribute sensory functions and computational methods to disentangle signals. Compartmentalizing sensor circuits within distinct populations simplifies experimental optimization by allowing individual populations to be exchanged without requiring genetic modifications. Our computational pipeline combines mechanistic modeling with machine learning to decode microbial communities' unique temporal responses and predict multiple input concentrations. We demonstrated this platform's versatility in a variety of contexts: measuring signals with high crosstalk, detecting antibiotics with natural microbial communities, and quantifying chemicals in hospital sink wastewater. Our findings highlight how combining microbial engineering with computational strategies can produce robust, scalable biosensors for diverse applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.23.650302v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AI-Based Antibody Design Targeting Recent H5N1 Avian Influenza Strains</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Santolla, N. F., Ford, C. T.</p>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show the utility of artificial intelligence in the discovery of novel antibodies against specific H5N1 strains of interest, which bind similarly to known therapeutic and elicited antibodies.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In 2025 alone, H5N1 avian influenza is responsible for thousands of infections across various animal species, including avian and mammalian livestock such as chickens and cows, and poses a threat to human health due to avian-to-mammalian transmission. There have been 70 human cases of H5N1 influenza in the United States since April 2024 and, as shown in recent studies, our current antibody defenses are waning. Thus, it is imperative to discover new therapeutics in the fight against more recent strains of the virus. In this study, we present the Frankies framework for automated antibody diffusion and assessment. This pipeline was used to automate the generation of 30 novel anti-HA1 Fv antibody fragment sequences, fold them into 3-dimensional structures, and then dock against a recent H5N1 HA1 antigen structure for binding evaluation. Here we show the utility of artificial intelligence in the discovery of novel antibodies against specific H5N1 strains of interest, which bind similarly to known therapeutic and elicited antibodies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.24.650061v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Lipid flip flop regulates the shape of growing and dividing synthetic cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lira, R. B., Dekker, C.</p>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We demonstrate that lipid flip flop relaxes curvature stresses and yields more symmetrically sized buds, which leads to bud scission.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cells grow their boundaries by incorporating newly synthesized lipids into their membranes as well as through fusion of intracellular vesicles. As these processes yield trans-bilayer imbalances in lipid numbers, cells must actively flip lipid molecules across the bilayer to enable growth. Using giant and small unilamellar vesicles (GUVs and SUVs, respectively), we here recapitulate cellular growth and division under various conditions of transmembrane flip flop of lipids. By dynamically monitoring the changes in reduced volume and spontaneous curvature of GUVs that grow by fusion of many small SUVs, the morphology of these growing synthetic cells is quantified. We demonstrate that lipid flip flop relaxes curvature stresses and yields more symmetrically sized buds. Further increasing the neck curvature is shown to lead to bud scission. The mechanisms presented here offer fundamental insights into cell growth and division, which are important for understanding early protocells and designing synthetic cells that are able to grow and divide.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.23.650179v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">High-throughput engineering and modification of non-ribosomal peptide synthetases based on Golden Gate assembly</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Podolski, A., Lindeboom, T. A., Praeve, L., Kranz, J., Schindler, D., Bode, H. B.</p>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel method for efficient high-throughput generation of novel and engineered NRPS libraries utilising the XUT concept, which allows the efficient modular assembly of different natural NRPS fragments to form hybrid NRPS that produce defined peptides.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Non-ribosomal peptide synthetases (NRPS) are multimodular enzymes that produce complex peptides with diverse biological activities, potentially being used as clinical drugs. However, the pharmaceutical applications of such natural peptides often require further derivatisation and modification of the peptide backbone, mainly performed by chemical synthesis. A sustainable alternative resembles the in vivo engineering of NRPS to change and modify the enzyme properties rationally and, thus, the produced products. The novel NRPS engineering concept, the eXchange Unit Thiolation domain (XUT), allows the efficient modular assembly of different natural NRPS fragments to form hybrid NRPS that produce defined peptides. In this study, we describe a Golden Gate assembly (GGA) method for efficient high-throughput generation of novel and engineered NRPS libraries utilising the XUT concept. This method was applied to generate over 100 novel NRPS with the possibility of changing starter, elongation, and termination modules, respectively. Additionally, we applied this method for targeted modification of the xenoamicin biosynthetic gene cluster (BGC) XabABCD from Xenorhabdus doucetiae, resulting in the generation of 25 novel xenoamicin derivatives.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.23.650154v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Conformation-specific Design: a New Benchmark and Algorithm with Application to Engineer a Constitutively Active MAP Kinase</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Stern, J., Alharbi, S., Sandholu, A., Arold, S. T., Della Corte, D.</p>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A general method for designing proteins with high conformational specificity is desirable for a variety of applications, including enzyme design and drug target redesign.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> A general method for designing proteins with high conformational specificity is desirable for a variety of applications, including enzyme design and drug target redesign. To assess the ability of algorithms to design for conformational specificity, we introduce MotifDiv, a benchmark dataset of 200 conformational specificity design challenges. We also introduce CSDesign, an algorithm for designing proteins with high preference for a target conformation over an alternate conformation. On the MotifDiv benchmark, CSDesign designs protein sequences that are predicted to prefer the target conformation. We apply this method in vitro to redesign human MAP kinase ERK2, an enzyme with active and inactive conformations. Out of two designs for the active conformation, one increased activity sufficiently to retain activity in the absence of activating phosphorylations, a property not present in the wild type protein.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.23.650138v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Harnessing Coding Sequence Cleavage: Theophylline Aptazymes as Portable Gene Regulators in Bacteria</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yurdusev, E., Nicole, Y., Pan Du, F., Suslu, N. E., Perreault, J.</p>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a novel "semi-trans" aptazyme-based system for gene regulation in bacteria, expanding the range of gene regulation tools, and demonstrate ligand-controlled regulation of tetracycline resistance and nickel sensitivity.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Nucleic acid-based regulatory elements capable of modulating gene expression in response to specific molecular cues have gained increasing attention in synthetic biology. These systems, which include riboswitches, allosteric DNAzymes, and aptazymes, function as Gene Expression Nucleic Allosteric actuators (GENAs) by coupling molecular recognition with genetic regulation. Their versatility can enable applications in diagnostics, therapeutics, and metabolic engineering. This study presents a novel "semi-trans" aptazyme-based system for gene regulation in bacteria, expanding the range of GENAs. The system employs theophylline-responsive hammerhead ribozyme aptazymes positioned in the 5' UnTranslated Region (UTR), designed to cleave within the coding sequence of the target gene, thereby modulating gene expression in a ligand-dependent manner. Using the tetA gene in Escherichia coli (E. coli) as a proof of concept, we demonstrate ligand-controlled regulation of tetracycline resistance and nickel sensitivity. The system's effectiveness is validated through in vitro cleavage assays and in vivo phenotypic studies in two E. coli strains, highlighting its portability across genetic backgrounds. Furthermore, the ability to design multiple aptazymes targeting different coding regions enables complex and fine-tuned regulation. This work broadens the landscape of synthetic gene regulation tools, facilitating the development of new aptazymes based on this approach.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.24.650125v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

