<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Monday, 12 May 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">UniCell: Towards a Unified Solution for Cell Annotation, Nomenclature Harmonization, Atlas Construction in Single-Cell Transcriptomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hu, L., Chen, Q., Qiu, P., Qin, H., Zhang, Y., Cao, L., Xia, T., Deng, Z., Zhang, Y., Fang, S., Li, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present UniCell, a hierarchical annotation framework that combines Cell Ontology structure with transcriptomic data for scalable, interpretable, and ontology-aware cell identity inference.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Standardizing cell type annotations across single-cell RNA-seq datasets remains a major challenge due to inconsistencies in nomenclature, variation in annotation granularity, and the presence of rare or previously unseen populations. We present UniCell, a hierarchical annotation framework that combines Cell Ontology structure with transcriptomic data for scalable, interpretable, and ontology-aware cell identity inference. UniCell leverages a multi-task architecture that jointly optimizes local and global classifiers, yielding coherent predictions across multiple levels of the ontology-defined hierarchy. When benchmarked across 20 human and mouse datasets, UniCell consistently outperformed state-of-the-art tools, including CellTypist, scANVI, OnClass, and SingleR, in annotation performance, and sensitivity to low-abundance populations. In disease settings, UniCell effectively identified previously unseen cell types through confidence-guided novelty detection. Applied to 45 human and 23 mouse tissue atlases, UniCell enabled cross-dataset and cross-species harmonization by embedding cells into a unified latent space aligned with Cell Ontology structure. Moreover, when used to supervise single-cell foundation models, UniCell substantially improved downstream annotation accuracy, rare cell detection, and hierarchical consistency. Together, these results establish UniCell as a generalizable framework that supports high-resolution annotation, nomenclature standardization, and atlas-level integration, providing a scalable and biologically grounded solution for single-cell transcriptomic analysis across diverse biological systems.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.06.652331v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multimodal Bonds Reconstruction Towards Generative Molecular Design</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, J., Dokholyan, N. V.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present YuelBond, a multimodal graph neural network framework for robust bonds reconstruction across three key scenarios: (i) recovery of bonds from accurate 3D atomic coordinates, (ii) reconstruction of chemically valid bonds in crude de novo generated compounds with perturbed geometries,</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Generative models such as diffusion-based approaches have transformed de novo drug design by enabling rapid generation of novel molecular structures in both 2D and 3D formats. However, accurate reconstruction of chemical bonds, especially from distorted geometries produced by generative models, remains a critical challenge. Here, we present YuelBond, a multimodal graph neural network framework for robust bonds reconstruction across three key scenarios: (i) recovery of bonds from accurate 3D atomic coordinates, (ii) reconstruction of chemically valid bonds in crude de novo generated compounds (CDGs) with perturbed geometries, and (iii) reassignment of bond orders in 2D topological graphs. YuelBond outperforms traditional rule-based methods such as RDKit, achieving 98.4% F1-score on standard 3D structures and maintaining strong performance (92.7% F1-score) on distorted CDGs, even when RDKit fails on most cases. Our results demonstrate that YuelBond enables accurate and reliable bond reconstruction from imperfect molecular data, bridging a critical gap in generative drug discovery pipelines.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.06.652517v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">GeoFlow-V2: A Unified Atomic Diffusion Model for Protein Structure Prediction and De Novo Design</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> BioGeometry Team, T., Tang, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce GeoFlow-V2, a unified atomic diffusion model that seamlessly integrates structure prediction and de novo protein design across multiple biological modalities, including proteins, nucleic acids (DNA/RNA), and small molecules.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We introduce GeoFlow-V2, a unified atomic diffusion model that seamlessly integrates structure prediction and de novo protein design across multiple biological modalities, including proteins, nucleic acids (DNA/RNA), and small molecules. The model's core innovation lies in its unified architecture that natively handles both structure prediction (when provided with complete input sequences) and generative design (for partially or fully masked inputs) through a shared atomic diffusion process. Through the integration of structure conditioning constraints and the sequence design module, GeoFlow-V2 operates as a fully bidirectional framework, accepting both sequence and structural inputs while generating corresponding outputs. GeoFlow-V2 can also accommodate diverse experimental constraints and prior knowledge, which boosts performance and enables precise control over the folding and design process. Benchmarking against state-of-the-art methods demonstrates GeoFlow-V2's strong performance in both structure prediction and de novo antibody design. We showcase GeoFlow-V2's versatility through protein design cases conditioned on diverse target modalities. To maximize accessibility, we engineer key functionalities of method and provide a user-friendly web server at https://prot.design for non-commercial research use.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.06.652551v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Diverse horizontally transferred cellulose biosynthesis gene clusters in Escherichia coli strains</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hosseinpourlamardi, S., Labiba, S., Li, L., Dadvar, A., Romling, U.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We report that homologs of the bcsABC core genes of the core genome bcs cellulose biosynthesis operon and even entire bcs operons can be mobilized on plasmids and occasionally manifested on E. coli chromosomes at alternative locations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The phosphoethanolamine modified exopolysaccharide cellulose is a major extracellular matrix component of Escherichia coli and Salmonella typhimurium. Upon enhanced acute virulence, however, cellulose production can be diminished or entirely abolished. Here, we report that homologs of the bcsABC core genes of the core genome bcs cellulose biosynthesis operon and even entire bcs operons can be mobilized on plasmids and are occasionally manifested on E. coli chromosomes at alternative locations. While BcsA2 and BcsA3 cellulose synthases are restricted to the genus Escherichia, the BcsA4 cellulose synthase and the entire associated bcs gene cluster are highly similar to one of the two chromosomal bcs gene clusters harboured by Klebsiella pneumoniae. Cyclic di-GMP turnover proteins known to post-translationally regulate cellulose biosynthesis are frequently localized on bcs operon bearing plasmids. In thermotolerant meat derived E. coli 740v1 harbouring a type 4 bcs operon on a plasmid, chemical and genetic evidence indicates that cellulose is produced in M9 minimal medium and upon activation by the second messenger cyclic di-GMP, an allosteric activator of the cellulose synthase, respectively. With gene duplication and horizontal gene transfer to contribute to the multiplication of bcs operons in a number of different bacterial species, the ecological role(s) of multiplication, transfer and replacement of cellulose biosynthesis operons also in species other than E. coli still need to be unraveled.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.09.653004v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">TurbOmics: a web-based platform for the analysis of metabolomics data using a multi-omics integrative approach</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Barrero-Rodriguez, R., Rodriguez, J. M., Naake, T., Huber, W., Juarez-Fernandez, M., R. Ramiro, A., Vazquez, J., Mastrangelo, A., Ferrarini, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present TurbOmics, a user-friendly web-based platform that enables researchers with diverse backgrounds to analyze metabolomics, proteomics, and transcriptomics data using advanced algorithms for multi-omics integration, while addressing key challenges associated with metabolomics data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In recent years, multi-omics integration has proven highly effective for the holistic characterization of biological systems, with the development of numerous bioinformatic platforms. However, these tools face limitations when incorporating metabolomics data, including absence of support for untargeted metabolomics annotation and dependence on predefined knowledge bases. Furthermore, the advanced algorithms required for multi-omics integration typically demand programming skills and statistical background, restricting their use to specialized users. To advance towards resolving these challenges, we present TurbOmics, a user-friendly web-based platform that enables researchers with diverse backgrounds to analyze metabolomics, proteomics, and transcriptomics data using advanced algorithms for multi-omics integration, while addressing key challenges associated with metabolomics data. Users can upload quantitative data and include additional information, such as metabolite identification or lipid classes, that streamline the interpretation of complex results. TurbOmics can be used sequentially with our previously published tool, TurboPutative, to simplify, reduce and prioritize the list of putative annotations from untargeted metabolomics datasets. Thanks to its flexible and interactive interface, researchers can perform exploratory data analysis and multi-omics integration using the Multi-Omics Factor Analysis, Pathway Integrative Analysis and Enrichment Analysis modules. We believe that TurbOmics will make multi-omics analysis more accessible to the research community. The platform is freely available at https://proteomics.cnic.es/TurboPutative/TurbOmicsApp.html.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.09.653072v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Non-Intestinal Microbial Signatures in Stool as Predictors of Cancer Immunotherapy Outcome</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kanaeva, V. A., Olekhnovich, E. I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A large-scale genome-resolved metagenomic analysis of 624 stool samples from 11 cohorts spanning five cancer types from three continents to identify microbial biomarkers of immunotherapy response.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Despite advances in immunotherapy, clinical responses remain heterogeneous, underscoring the need to elucidate the gut microbiome's role in modulating therapeutic efficacy. We conducted a large-scale genome-resolved metagenomic analysis of 624 stool samples from 11 cohorts spanning five cancer types from three continents to identify conserved microbial biomarkers of immunotherapy response. Responders (R; n = 362) exhibited enrichment of Blautia species, linked to acetate production from CO2 via the Wood-Ljungdahl (WL) pathway, glycoside hydrolases (GH), and amino acid biosynthesis. Non-responders (NR; n = 262) showed increased oral (e.g. Veillonella parvula, Dialister invisus) and food-derived taxa included opportunistic species e.g. Citrobacter freundii, Klebsiella spp., alongside depletion of gut-resident microbes and enriched lipopolysaccharide biosynthesis and flagellar assembly pathways. Interestingly, part of NR-associated microbes originated from non-intestinal niches (oral/food), which may correlate with reduced gut microbial load in the large intestine. These findings redefine dietary interventions as dual-edged: while carbon sources such as fiber processed by GH or CO2 processed by WL pathway supports beneficial taxa and immune system, pathogen-free food sourcing is critical to avoid introducing opportunistic microbes. Our work establishes the resident-to-transient microbial ratio as a novel predictive biomarker and positions Blautia as a keystone genus for next-generation probiotics targeting carbon metabolism-immune crosstalk. By integrating taxonomic, functional, and ecological axes, this study provides a roadmap for microbiome-guided precision nutrition to optimize immunotherapy outcomes. While immunotherapy non-responders exhibit gut microbiota enriched with food/oral commensals, Faecalibacterium prausnitzii - shows context-dependent associations, with strain-level variations potentially linked to A blood group antigen N-acetylgalactosamine utilization, highlighting its role as a conditional biomarker and the prognostic value of metagenomics in personalized therapy.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.07.652660v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Estimating the changing risks of low crop yield using non-stationary generalized Pareto distributions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Onogi, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A data-driven approach based on extreme value theory to estimate changes in the risk of low crop yield in a changing climate .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Estimating the changing risk of low crop yield in a changing climate is an important task in various fields of agricultural research. According to the extreme value theory, the probability of extreme events can be approximated using generalized Pareto distributions. In this study, non-stationary generalized Pareto distributions were used to estimate the changing risk of low crop yield. The proposed methods were applied to global yield data for maize, wheat, rice, and soybean collected from 1961 to 2022, as well as local yield data for wheat, rice, and soybean in Japan, from a start date of either 1948 or 1958 and running to 2020. The results illustrated exacerbated trends of low-yield risk in maize crops in Africa; maize, wheat, and rice in Americas; maize and wheat in Western, Central, and Southern Asia; maize and wheat in Europe; and soybean in Japan. Only wheat in Japan showed trends of mitigating the risk of low yields. The proposed models were also validated through simulations. The results showed that the models can generally estimate the changing risks accurately, and the precision depends on the size of the data set. Although there is still room for improvement in the models, the present study demonstrates that it is possible to estimate changes in the risk of low yield using a data-driven approach based on extreme value theory without assumptions about climate and crop physiology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.10.653234v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Two long non-coding RNAs, SVALKA and SVALNA, regulate CBF1 and CBF3 via multiple mechanisms</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rosenkranz, I., Mermet, S., Zacharaki, V., Kindgren, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We investigate the roles of the long non-coding RNAs SVALKA and SVALNA in regulating CBF1 and CBF3 gene expression in Arabidopsis under cold stress conditions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Long non-coding RNAs (lncRNAs) are emerging as key regulatory players of coding gene expression in eukaryotes. Here, we investigate the roles of the lncRNAs SVALKA (SVK) and SVALNA (SVN) in regulating CBF1 and CBF3 gene expression in Arabidopsis under cold stress conditions. We used Native Elongation Transcript Sequencing, CRISPR-Cas9 deletion strategies, and RT-qPCR to analyze the transcriptional dynamics and regulatory mechanisms of SVK and SVN. Our results demonstrate that SVK functions as a cis- and trans-acting lncRNA, regulating both CBF1 and CBF3 through RNAPII collision and chromatin remodeling, while SVN serves a cis role by negatively regulating CBF3 via a RNAPII collision mechanism. We identified isoforms of SVK, originating from distinct transcription start sites and undergo alternative splicing to adapt structural stability, crucial for their regulatory functions. This study elucidates the complex interplay of lncRNAs in gene regulation, highlighting their essential roles in modulating responses to environmental stresses. Our findings contribute to a deeper understanding of the mechanisms underlying lncRNA functionality and their significance in gene regulatory networks in eukaryotes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.09.653013v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrative analysis of plant responses to a combination of water deficit, heat stress and eCO2 reveals a role for OST1 and SLAH3 in regulating stomatal responses</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pelaez-Vico, M. A., Sinha, R., Ghani, A., Lopez-Climent, M. F., Joshi, T., Fritschi, F. B., Zandalinas, S. I., Mittler, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We study plant physiological and molecular responses to conditions of Global Change Factor combination (GCFc), or stress combination, and show high specificity in plant responses to GCFc.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The frequency, intensity and duration of global change factors and/or environmental stressors, such as droughts, heat waves, floods, and pollution, are increasing due to anthropogenic activities, subjecting plants to compound conditions of Global Change Factor combination (GCFc), or stress combination. These can have contrasting or additive effects on plant physiological performance and impact different ecosystems and agro-ecosystems worldwide. Using an integrative physiological, genetic, hormonal, and transcriptomic analysis, we studied the response of Arabidopsis thaliana (L.) Heynh to a combination of water deficit (WD), heat stress (HS) and/or elevated levels of CO2 (eCO2). Our findings reveal high specificity in plant responses to GCFc. We further reveal that stomatal aperture regulation under conditions of GCFc is controlled by a blend of unique and shared regulators, that together determine the specific aperture size under each different set of GCFc conditions. Under a combination of WD, HS, and eCO2 (WD+HS+eCO2), for example, stomatal closure required the function of nitric oxide, OPEN STOMATA 1 and the S-type anion channel SLAH3, but not the S-type anion channel SLAC1, or many other regulators typically found to control stomatal aperture under less complex conditions, including HS+WD. Our findings highlight the complexity of plant physiological and molecular responses to conditions of stress combination and open the way for future studies of plant responses to stress combination, crucial to our understanding of how GCFc impacts different ecosystems and agro-ecosystems worldwide. In addition, our study provides an initial definition for a stomatal hierarchical stress code that could apply to future studies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.07.652739v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multi-layer encapsulation of plant protoplasts.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lintilhac, P. M., Grasso, M. S., Floreani, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We report here on an attempt to create an engineered structure which can reproduce the physical and mechanical environment of the land plant sporangium.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We report here on an attempt to create an engineered structure which can reproduce the physical and mechanical environment of the land plant sporangium. This work is part of a broader effort to understand the stress mechanics of the land plant sporangium and its role in initiating reproductive development. A second purpose is to extend the range of experimental methods available for the study of the physical environment of plant cell growth and the mechanics of trans-cellular signaling in plant development. We describe an experimental protocol, based on the microfluidic encapsulation of living plant protoplasts in multilayered microbeads composed of tunable hydrogel materials whose mechanical properties can be modified to mimic the stress-mechanics of the living sporangium. Our results demonstrate the successful encapsulation of living plant protoplasts in dual-layered microbeads consisting of an inner layer of gelled agarose and an outer layer of cross-linked alginate/methacrylate.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.08.652860v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

