<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Saturday, 17 May 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Alcohol Use Disorder Associated Gene FNDC4 Alters Glutamatergic and GABAergic Neurogenesis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhu, X., John, A. J., Wang, L., Kim, S., Ding, E., Saleh, A., Marin-Goni, I., Jomaa, A., Gao, H., Wai, C. M., Moon, I., Coombes, B. J., Kerr, T. M., Suto, N., Wang, L., Frye, M. A., Biernacka, J. M., Karpyak, V. M., Li, H., Weinshilboum, R. M., Liu, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We found that genetic loci which are splicing quantitative trait loci for the fibronectin III domain containing 4 (FNDC4) gene in the brain were identified by genome-wide association studies for both AUD drug treatment outcomes and AUD risk.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Large-cohort genome-wide association studies (GWAS) for alcohol use disorder (AUD) and AUD-related phenotypes have identified more than one hundred genetic loci. Functional study of those GWAS-identified loci might represent an important step toward understanding AUD pathophysiology. We found that genetic loci which are splicing quantitative trait loci (sQTLs) for the fibronectin III domain containing 4 (FNDC4) gene in the brain were identified by GWAS for both AUD drug treatment outcomes and AUD risk. However, FNDC4 function in the brain and how it might contribute to AUD pathophysiology remain unknown. In the present study, we characterized GWAS locus-associated FNDC4 splice isoforms, studies which suggested that FNDC4 alternative splicing results in loss-of-function for FNDC4. We also investigated FNDC4 function using CRISPR/cas9 gene editing, and the creation of human induced pluripotent stem cell (iPSC)-derived neural organoids joined with single-nucleus RNA sequencing. We observed that knock-out (KO) of FNDC4 resulted in a striking shift in the relative proportions of glutamatergic and GABAergic neurons in iPSC-derived neural organoids, suggesting a possible important role for FNDC4 in neurogenesis. We also explored potential mechanism(s) of FNDC4-dependent neurogenesis with results that suggested a role for FNDC4 in mediating neural cell-cell interaction. In summary, this series of experiments indicates that FNDC4 plays a role in regulating cerebral cortical neurogenesis in the brain. This regulation may contribute to the response to AUD pharmacotherapy as well as the effects of alcohol on the brain.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.15.654319v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Biophysical Modeling Uncovers Transcription Factor and Nucleosome Binding on Single DNA Molecules</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dalakishvili, L., Managori, H., Bardet, A., Slaninova, V., Bertrand, E., Molina, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> HiddenFoot, a probabilistic modeling framework based on statistical mechanics that quantitatively infers TF, nucleosome, and Pol II occupancy profiles on single DNA molecules by systematically evaluating all thermodynamically plausible binding configurations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Gene regulation in eukaryotes emerges from a dynamic interplay between transcription factors (TFs), nucleosomes, and RNA Polymerase II (Pol II), whose competitive and cooperative binding shapes DNA accessibility and transcriptional output. Single-molecule footprinting (SMF) and long-read chromatin accessibility assays such as Fiber-seq now capture these interactions at nucleotide resolution on individual DNA molecules. However, existing computational tools remain insufficient to decode complex binding events from sparse methylation data. Here, we introduce HiddenFoot, a probabilistic modeling framework based on statistical mechanics that quantitatively infers TF, nucleosome, and Pol II occupancy profiles on single DNA molecules by systematically evaluating all thermodynamically plausible binding configurations. Applying HiddenFoot to SMF and Fiber-seq data from mouse, Drosophila, and human cells, we recovered known TF footprints, precisely resolved Pol II pausing, and identified extensive heterogeneity in nucleosome positioning driven by TF binding. HiddenFoot further distinguishes direct TF--TF cooperativity from nucleosome-mediated co-dependency by estimating pairwise interaction energies and comparing to null models under equilibrium. By integrating biophysical modeling with high-resolution single-molecule data, HiddenFoot offers a general, interpretable framework for dissecting regulatory logic in native chromatin with base-pair precision.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.13.653852v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Structure-based identification of GIRK2-PIP2 modulators among known drugs and metabolites using docking, MM-GBSA, ADMET, and molecular dynamics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jeremic, D., Jimenez-Diaz, L., Navarro-Lopez, J. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study aimed to advance the targeting of GIRK2 channels at the PIP2-binding site at the molecular dynamics simulations .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> G protein-gated inwardly rectifying potassium (GIRK) channels are key regulators of neuronal excitability, making them promising therapeutic targets for central nervous system disorders. Their activation depends on phosphatidylinositol-4,5-bisphosphate (PIP2), which stabilizes the channel s open state. A deeper understanding of GIRK-PIP2 interactions could uncover new physiological roles and pave the way for therapies that modulate channel function. This study aimed to advance the targeting of GIRK channels at the PIP2-binding site. Over one million compounds were screened against GIRK2 (PDB ID: 4KFM) using high-throughput virtual screening. A core constraint with a root-mean-square deviation (RMSD) < 2 A was applied to assure the accuracy and binding close to PIP2-binding site. The top-scoring ligands were redocked with Glide (SP, XP) and binding free energy was estimated using Molecular Mechanics Generalized Born Surface Area method. The most promising compounds were analyzed for pharmacokinetic/physicochemical properties, followed by molecular dynamics (MD) simulations over 200 ns in membrane bilayer. MD analysis revealed three known compounds (Rosuvastatin, CID: 54365126 and 7304563) as potential competitive GIRK2 modulators, exhibiting stable interactions with residues critical for binding endogenous activators (PIP2, cholesterol), and GIRK-acting drugs. Docking analyses also revealed strong binding to GIRK2 for various metabolites, including leukotrienes, resolvins, acyl-CoAs, and polyphosphates, including adenosine-triphosphate (ATP) and thiamine-triphosphate. Notably, some of the identified compounds can affect similar ion channels, indicating potential cross-reactivity with GIRK2. Furthermore, the binding modes of acyl-CoAs and polyphosphates closely resemble PIP2 s hydrophobic and phosphate group engagement. Together, these findings offer promising candidates for experimental validation and therapeutic development.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.13.653795v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Mitochondrial Genome of the lesser known - Madras Hedgehog (Paraechinus nudiventris): Genomic characterization and comparative analysis within Erinaceidae</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Vijayan, T., Meimberg, H., Kumar, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the complete mitochondrial genome of the Madras hedgehog (Paraechinus nudiventris), annotated and analyzed in comparison with other Erinaceidae species, providing insights into the diversification of hedgehogs in South Asia.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The Madras hedgehog (Paraechinus nudiventris) is an understudied species endemic to southern India, with limited genetic data available for understanding its evolutionary history. Here, we present the complete mitochondrial genome (17,232 bp), annotated and analyzed in comparison with other Erinaceidae species. The mitogenome retains the typical vertebrate structure, comprising 14 protein-coding genes, 22 tRNAs, 2 rRNAs, and a non-coding control region, with an A+T rich nucleotide composition. Phylogenetic analyses using Bayesian and maximum likelihood approaches confirmed the species' placement within Paraechinus, with divergence from P. micropus estimated during the Pleistocene. Molecular evolutionary analyses revealed selective constraints on oxidative phosphorylation genes, with atp8 showing the highest variability. Our divergence time estimates, incorporating fossil calibrations, refine the evolutionary timeline of Erinaceidae, providing insights into the diversification of hedgehogs in South Asia. By making this annotated mitochondrial genome publicly available, we provide a foundational resource for future studies on the genetic diversity, phylogeography, and conservation of P. nudiventris and its relatives. The study also demonstrates the utility of non-invasive sampling methods in generating high-quality genomic data for cryptic mammalian species.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.13.653780v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Poxvirus targeted by RFdiffusion peptide-binders</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Coll, J. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Cyclic peptide-binders for small-drug screening and/or co-evolution .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Peptide-binders were generated by computational RFdiffusion to the F13L-homodimer interface sequences of poxviruses such as vaccinia and monkeypox. Compared to small-drug screenings and/or co-evolutions, peptide-binders have the advantages of higher affinities and easier chemical/recombinant synthesis. Among the hundreds of 20-30-mer peptide-binders randomly generated by RFdiffusion, some targeted vaccinia (VACV) F13L homodimer interface predicting low nanoMolar affinities. To improve its physiological stability, additional peptide sequences were computationally generated by cyclization/ hallucination. The resulting de novo cyclic peptide sequences predicted picoMolar affinities not only targeting the VACV F13L homodimer interfaces but also inner cavities previously identified by small-drug dockings to Tecovirimat-resistant monkeypox (MPXV) mutants. Because their targeting to numerous highly-conserved amino acids among poxviruses, and improved physiological stability against proteases, cyclic peptide-binders may be more adequate against resistant mutants of VACV and MPXV poxviruses.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.14.654163v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Graph-Attention-Based Deep Learning Network for Predicting Biotech-Small-Molecule Drug Interactions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nasiri, F., Hooshmand, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel graph attention network-based deep learning framework that improves interaction prediction between biotech and small-molecule drugs.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The increasing demand for effective drug combinations has made drug-drug interaction (DDI) prediction a critical task in modern pharmacology. While most existing research focuses on small-molecule drugs, the role of biotech drugs in complex disease treatments remains relatively unexplored. Biotech drugs, derived from biological sources, have unique molecular structures that differ significantly from those of small molecules, making their interactions more challenging to predict. This study introduces BSI-Net, a novel graph attention network-based deep learning framework that improves interaction prediction between biotech and small-molecule drugs. Experimental results demonstrate that BSI-Net outperforms existing methods in multi-class DDI prediction, achieving superior performance across various evaluation types, including micro, macro, and weighted assessments. These findings highlight the potential of deep learning and graph-based models in uncovering novel interactions between biotech and small-molecule drugs, paving the way for more effective combination therapies in drug discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.13.653666v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PEPlife2: A Updated Repository of the Half-life of Peptides</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Alam, U., Chaudhary, K., Kumar, N., Tomer, R., Patiyal, S., Raghava, G. P. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present an updated version of PEPlife, a manually curated database that provides comprehensive information on peptide half-life.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> This manuscript presents an updated version of PEPlife, a manually curated database that provides comprehensive information on peptide half-life. The updated version, PEPlife2, includes 4,412 entries, compared to 2,229 entries in the previous version. Each entry contains detailed information that include experimental methods used to determine half-life, chemical modifications, biological activity, routes of administration, and other relevant data. The database encompasses 402 proteins and 1,781 peptides, derived from 105 unique assays. In addition to natural peptide sequences, PEPlife2 includes cyclic peptides and chemically modified peptides, such as those with N- and C-terminal modifications. To provide structural insights, peptide and protein structures are sourced from the Protein Data Bank (PDB) or predicted using PEPstrMOD. To support the scientific community, we have developed a user-friendly interface and integrated advanced analytical tools, including BLAST, Smith-Waterman, GGSEARCH, CLUSTALW, and MUSTANG. The updated database is accessible at: https://webs.iiitd.edu.in/raghava/peplife2/.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.13.653654v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">NuclePhaser: a YOLO-based framework for cell nuclei detection and counting in phase contrast images of arbitrary size with support of fast calibration and testing on specific use cases</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Voloshin, N., Putlyaev, E., Chechekhina, E., Usachev, V., Karagyaur, M., Bozov, K., Grigorieva, O., Tyurin-Kuzmin, P., Kulebyakin, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed and tested a set of deep learning object detection YOLO models that detect cell nuclei in phase contrast images, which allows cell count and tracking without staining.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Microscopy is an essential method in modern biology, and brightfield microscopy methods (phase contrast, differential interference contrast, etc.) are being widely used and actively developed since they do not require sample fixation and staining. But they produce low contrast images, where cells have similar intensity to the background. In this work we developed and tested a set of deep learning object detection YOLO models that detect cell nuclei in phase contrast images, which allows cell count and tracking without staining. We created a large dataset consisting of more than 100,000 640x640 pixels images with more than 3 million nuclei of 4 different cell cultures (CHO, HEK293, iPSCs, and MSCs). Using images from various microscopes and cameras, as well as full-scale augmentations, we developed a set of highly generalized models that can detect nuclei in images across different cell types and imaging conditions, including different microscopes and contrast methods. Combined with sliced inference methods, these algorithms can be applied to images of any size, allowing studies of large quantities of cells. Moreover, we developed a training-free calibration and testing algorithm based on confidence threshold optimization. It allows for fine-tuning of models for specific cell types and/or imaging options and evaluating the accuracy of the calibrated model. This provides a highly controllable and reliable method for studying cell proliferation rate, single cell tracking and other scenarios. Additionally, we developed a NuclePhaser plugin for Napari (https://github.com/nikvo1/napari-nuclephaser), which allows users to calibrate, test and apply our models in code-free manner. Given that the YOLO models are fast and can run at sufficient speeds even on CPUs, this makes our work highly accessible to a wide range of researchers.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.13.653705v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ATaRVa: Analysis of Tandem Repeat Variation from Long Read Sequencing data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sivakumar, A. K., Sudarsanam, S., Sharma, A., Avvaru, A. K., Sowpati, D. T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present ATaRVa, a sequencing technology-agnostic genotyper that outperforms existing long-read TR genotypers while running an order of magnitude faster.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Long-read sequencing propelled comprehensive analysis of tandem repeats (TRs) in genomes. Current long-read TR genotypers are either inaccurate, platform-specific, or computationally inefficient. Here we present ATaRVa, a sequencing technology-agnostic genotyper that outperforms existing tools while running an order of magnitude faster. ATaRVa also supports short-read data, multi-threading, consensus sequence derivation, and motif decomposition, making it an invaluable tool for population scale TR analyses.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.13.653434v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Rapid ethylene-triggered protein complex remodeling in dark grown hypocotyls</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lee, Y., Park, H. L., Yoon, G. M., Szymanski, D. B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel approach to gene discovery identifies potential targets for ethylene-mediated growth regulation and enhanced seedling adaptability during early development.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein-level cellular dynamics, including multimerization, play a crucial role in rapid adaptation of plants during developmental transitions and environmental stresses. The gaseous hormone ethylene is a key regulator of these rapid cellular adjustments, especially in growth inhibition. During soil emergence, etiolated seedlings undergo crucial morphological changes to their apical hooks and hypocotyls, with ethylene inhibiting hypocotyl axial elongation while promoting radial expansion. Ethylene triggers these growth responses within two hours; however, the protein machineries and cellular processes that mediate morphogenesis control are not known. Here, we used quantitative proteomics and co-fractionation mass spectrometry to test for rapid ethylene-dependent changes in protein abundance and protein complex composition. Protein multimerization responses were numerous and diverse. There were instances of protein complex assembly and disassembly, with varying degrees of completeness. Small-scale validation tests indicate that the identified proteins play a role in hypocotyl development and suggest that this novel approach to gene discovery identifies potential targets for ethylene-mediated growth regulation and enhanced seedling adaptability during early development.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.15.654145v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Efficient CRISPR-Cas9-mediated genome editing of the cane toad (Rhinella marina)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Clark, M., Funk, A., Paporakis, A., Brown, G. P., Beach, S. J., Tay, A., Deering, S., Cooper, C., Tizard, M., Jolly, C. J., Ward-Fear, G., Waddle, A. W., Shine, R., Maselko, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We demonstrate CRISPR-Cas9 genome editing in the cane toad (Rhinella marina), one of Australia's most notorious invasive species, by targeting the tyrosinase gene to produce albino phenotypes that provide clear visual markers for assessing editing efficiency.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Invasive species inflict major ecological, economic, social, and cultural harm worldwide, highlighting the urgent need for innovative and effective control strategies. Genome editing offers exciting possibilities for creating highly targeted control methods for invasive species. Here, we demonstrate CRISPR-Cas9 genome editing in the cane toad (Rhinella marina), one of Australia's most notorious invasive species, by targeting the tyrosinase gene to produce albino phenotypes that provide clear visual markers for assessing editing efficiency. Microinjection of Cas9 protein and guide RNAs into one-cell zygotes resulted in 87.6% of mosaic larvae displaying nearly complete albinism, with 2.3% exhibiting complete albinism. For completely albino individuals, genomic analysis confirmed predominantly frameshift mutations or large deletions at the target site, with no wild-type alleles detected. Germline transmission rates reflected the extent of albinism in the mosaic adult, where we achieved maternal germline transmission rates of almost 100%. This technology, representing the first application of CRISPR-Cas9 in the Bufonidae family, opens possibilities for exploring both basic research questions and strategies for population control.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.15.654396v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Saccharomyces boulardii synthetic biotic platform for delivery of therapeutic nanobodies to ameliorate gastrointestinal inflammation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Palou, R., van der Sloot, A. M., Fiebig, A. A., Zangara, M. T., Sangwan, N., Sanchez-Osuna, M., Ilyas, B., Zubyk, H., Cook, M., Wright, G. D., Coombes, B. K., Tyers, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We describe a cell microfactory platform based on the probiotic yeast Saccharomyces boulardii for the production of nanobody biologics in the gastrointestinal (GI) tract.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein-based pharmaceuticals, such as engineered antibodies, form a major drug class of steadily increasing market share. However, these biologic medicines are costly to manufacture, are subject to strict supply chain and storage constraints and often require invasive administration routes. Engineered microbes that secrete bioactive products directly within the microbiome milieu may mitigate these challenges. Here, we describe a cell microfactory platform based on the probiotic yeast Saccharomyces boulardii for the production of nanobody biologics in the gastrointestinal (GI) tract. High-level secretion of nanobodies by S. boulardii was achieved by optimizing promoters, secretion signals, and antibody formats. In mice, oral gavage of S. boulardii allowed efficient and transient colonization of the colonic compartment and in situ production of a therapeutic nanobody directed against tumor necrosis factor (TNF). In a mouse model of chemical-induced colitis, GI-delivery of anti-mTNF nanobody via live S. boulardii improved both survival and disease severity without causing overt perturbation of microbiome composition. These results position S. boulardii as a synthetic biotic platform for the in situ production and delivery of protein-based therapeutics to the GI tract.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.15.652971v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Experimental Evaluation of AI-Driven Protein Design Risks Using Safe Biological Proxies</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ikonomova, S. P., Wittmann, B. J., Piorino, F., Ross, D. J., Schaffter, S. W., Vasilyeva, O. B., Horvitz, E., Diggans, J., Strychalski, E. A., Lin-Gibson, S., Taghon, G. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a testing, evaluation, validation, and verification (TEVV) framework for AI-assisted protein design (AIPD), using safe proteins as SOC proxies. We show that current AIPD systems are not yet powerful enough to reliably rewrite the sequence of a given protein, while</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Advances in machine learning are providing new abilities for engineering biology, promising leaps forward with beneficial applications. At the same time, these advances raise concerns about biosecurity. Recently, Wittmann et al. described an in silico pipeline of generative AI tools to demonstrate how amino acid sequences encoding sequences of concern (SOCs) could be reformulated as synthetic homologs that may evade detection by biosecurity screening software (BSS) used by nucleic acid synthesis providers. While the discovered vulnerability has been mitigated, its true severity remains unclear, as the study was performed without experimental testing of the synthetic homologs. Here, we present a testing, evaluation, validation, and verification (TEVV) framework for AI-assisted protein design (AIPD), using safe proteins as SOC proxies. Our findings indicate AIPD can generate synthetic homologs whose predicted structures are similar to a native template, but without necessarily retaining activity. We determine that current AIPD systems are not yet powerful enough to reliably rewrite the sequence of a given protein, while both maintaining activity and evading detection by BSS. We further conclude that TEVV of generated sequences requires significant investment of time, technical skill, and resources. We offer our framework and test proteins as a general benchmark of AIPD capability.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.15.654077v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CRISPRi-Linked Multi-Module Negative Feedback Loops to Address Winner-Take-All Resource Competition</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rijal, S., Standage-Beier, K., Zhang, R., Stone, A., Youssef, A., Wang, X., Tian, X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel strategy for regulating resource competition in synthetic gene circuits that reduces Winner-Takes-All behavior and improves modularity.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cellular resource limitations create unintended interactions among synthetic gene circuit modules, compromising circuit modularity. This challenge is particularly pronounced in circuits with positive feedback, where uneven resource allocation can lead to Winner-Takes-All (WTA) behavior, favoring one module at the expense of others. In this study, we experimentally implemented a Negatively Competitive Regulatory (NCR) controller using CRISPR interference (CRISPRi) and evaluated its effectiveness in mitigating WTA behavior in two gene circuits: dual self-activation and cascading bistable switch. We chromosomally integrated a tunable dCas9 gene and designed module-specific gRNAs, with each module encoding its own gRNA to self-repress via competition for limited dCas9. This configuration introduces strong negative feedback to the more active module while reallocating resources to the less active one, promoting balanced module activation. Compared to the control group lacking dCas9-mediated repression, the NCR controller significantly increased module coactivation and suppressed WTA behavior. Our quantitative results demonstrate that NCR provides an effective strategy for regulating resource competition and improving the modularity of synthetic gene circuits.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.15.654351v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ECCFP: a consecutive full pass based bioinformatic analysis for eccDNA identification using Nanopore sequencing data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, T., Li, W., Zeng, Q., Zhang, J., Miao, B., Li, M., Luo, J., Liu, T., Chen, S., Wan, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose ECCFP, a bioinformatic analysis pipeline that detects eccDNA from long-read sequencing data and outputs eccDNA genomic coordinates and consensus sequences.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> It is commonly known that extrachromosomal circular DNA (eccDNA) has the potential as a molecular marker because of its close relationship with cancer progress and its prevalent existence in eukaryotic organisms. The mainstream technique of eccDNA detection is using high-throughput sequencing supported by bioinformatics analysis. Although these have various analysis pipelines for sequencing data, they are restricted by sequencing platforms or have shortcomings in accuracy and efficiency. To address these limitations, we design ECCFP, a bioinformatic analysis pipeline that detects eccDNAs amplified by rolling circle amplification (RCA) from long-read sequencing data and outputs eccDNA genomic coordinates and consensus sequences. This pipeline proposes a rigorous algorithm to retain all consecutive full passes derived from individual reads to obtain candidate eccDNAs, followed by systematic consolidation of candidate eccDNAs to detect unique eccDNAs. Using simulated datasets and experimental eccDNA sequencing datasets, we estimated ECCFP in several aspects and compared it with other existing pipelines. It exhibits a marked reduction in false positive rates compared with eccDNA_RCA_nanopore and superior sensitivity relative to CReSIL and FLED. Besides, inverse PCR and Sanger sequencing further validated the existence and accuracy of the position of detected eccDNAs by ECCFP. Collectively, ECCFP provides a more efficient choice for eccDNA detection from long-read sequencing data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.13.653627v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

