<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Tuesday, 22 July 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Machine learning in Alzheimer's disease genetics.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Matthew Bracher-Smith, Federico Melograna, Brittany Ulm, Céline Bellenguez, Benjamin Grenier-Boley, Diane Duroux, Alejo J Nevado, Peter Holmans, Betty M Tijms, Marc Hulsman, Itziar de Rojas, Rafael Campos-Martin, Sven van der Lee, Atahualpa Castillo, Fahri Küçükali, Oliver Peters, Anja Schneider, Martin Dichgans, Dan Rujescu, Norbert Scherbaum, Jürgen Deckert, Steffi Riedel-Heller, Lucrezia Hausner, Laura Molina-Porcel, Emrah Düzel, Timo Grimmer, Jens Wiltfang, Stefanie Heilmann-Heimbach, Susanne Moebus, Thomas Tegos, Nikolaos Scarmeas, Oriol Dols-Icardo, Fermin Moreno, Jordi Pérez-Tur, María J Bullido, Pau Pastor, Raquel Sánchez-Valle, Victoria Álvarez, Mercè Boada, Pablo García-González, Raquel Puerta, Pablo Mir, Luis M Real, Gerard Piñol-Ripoll, Jose María García-Alberca, Eloy Rodriguez-Rodriguez, Hilkka Soininen, Sami Heikkinen, Alexandre de Mendonça, Shima Mehrabian, Latchezar Traykov, Jakub Hort, Martin Vyhnalek, Nicolai Sandau, Jesper Qvist Thomassen, Yolande A L Pijnenburg, Henne Holstege, John van Swieten, Inez Ramakers, Frans Verhey, Philip Scheltens, Caroline Graff, Goran Papenberg, Vilmantas Giedraitis, Julie Williams, Philippe Amouyel, Anne Boland, Jean-François Deleuze, Gael Nicolas, Carole Dufouil, Florence Pasquier, Olivier Hanon, Stéphanie Debette, Edna Grünblatt, Julius Popp, Roberta Ghidoni, Daniela Galimberti, Beatrice Arosio, Patrizia Mecocci, Vincenzo Solfrizzi, Lucilla Parnetti, Alessio Squassina, Lucio Tremolizzo, Barbara Borroni, Michael Wagner, Benedetta Nacmias, Marco Spallazzi, Davide Seripa, Innocenzo Rainero, Antonio Daniele, Fabrizio Piras, Carlo Masullo, Giacomina Rossi, Frank Jessen, Patrick Kehoe, Tsolaki Magda, Pascual Sánchez-Juan, Kristel Sleegers, Martin Ingelsson, Mikko Hiltunen, Rebecca Sims, Wiesje van der Flier, Ole A Andreassen, Agustín Ruiz, Alfredo Ramirez, Ruth Frikke-Schmidt, Najaf Amin, Gennady Roshchupkin, Jean-Charles Lambert, Kristel Van Steen, Cornelia van Duijn, Valentina Escott-Price</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We apply machine learning to genome-wide data from the largest European consortium on Alzheimer's disease to investigate the effectiveness of various ML algorithms in replicating known findings, discovering novel loci, and predicting individuals at risk.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Traditional statistical approaches have advanced our understanding of the genetics of complex diseases, yet are limited to linear additive models. Here we applied machine learning (ML) to genome-wide data from 41,686 individuals in the largest European consortium on Alzheimer's disease (AD) to investigate the effectiveness of various ML algorithms in replicating known findings, discovering novel loci, and predicting individuals at risk. We utilised Gradient Boosting Machines (GBMs), biological pathway-informed Neural Networks (NNs), and Model-based Multifactor Dimensionality Reduction (MB-MDR) models. ML approaches successfully captured all genome-wide significant genetic variants identified in the training set and 22% of associations from larger meta-analyses. They highlight 6 novel loci which replicate in an external dataset, including variants which map to ARHGAP25, LY6H, COG7, SOD1 and ZNF597. They further identify novel association in AP4E1, refining the genetic landscape of the known SPPL2A locus. Our results demonstrate that machine learning methods can achieve predictive performance comparable to classical approaches in genetic epidemiology and have the potential to uncover novel loci that remain undetected by traditional GWAS. These insights provide a complementary avenue for advancing the understanding of AD genetics.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40691194/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A deep ensemble framework for human essential gene prediction by integrating multi-omics data.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xue Zhang, Weijia Xiao, Brent Cochran, Wangxin Xiao</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a snapshot ensemble deep neural network method, DeEPsnap, to predict human essential genes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Essential genes are necessary for the survival or reproduction of a living organism. The prediction and analysis of gene essentiality can advance our understanding of basic life and human diseases, and further boost the development of new drugs. We propose a snapshot ensemble deep neural network method, DeEPsnap, to predict human essential genes. DeEPsnap integrates the features derived from DNA and protein sequence data with the features extracted or learned from four types of functional data: gene ontology, protein complex, protein domain, and protein-protein interaction networks. More than 200 features from these biological data are extracted/learned which are integrated together to train a series of cost-sensitive deep neural networks. The proposed snapshot mechanism enables us to train multiple models without increasing extra training effort and cost. The experimental results of 10-fold cross-validation show that DeEPsnap can accurately predict human gene essentiality with an average AUROC of 96.16%, AUPRC of 93.83%, and accuracy of 92.36%. The comparative experiments show that DeEPsnap outperforms several popular traditional machine learning models and deep learning models, while all those models show promising performance using the features we created for DeEPsnap. We demonstrated that the proposed method, DeEPsnap, is effective for predicting human essential genes.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40691502/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Deconvolution of Single Homeologous Polymorphism (SHP) drives phylogenetic analysis of allopolyploids</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sancho, R., Catalan, P., Vogel, J. P., Contreras-Moreira, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We have developed AlloSHP, a command-line tool for detecting and extracting single homeologous polymorphisms (SHPs) from the subgenomes of allopolyploid species.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The genomic and evolutionary study of allopolyploid organisms involves multiple copies of homeologous chromosomes, making their assembly, annotation, and phylogenetic analysis challenging. Bioinformatics tools and protocols have been developed to study polyploid genomes, but sometimes require the assembly of their genomes, or at least the genes, limiting their use. We have developed AlloSHP, a command-line tool for detecting and extracting single homeologous polymorphisms (SHPs) from the subgenomes of allopolyploid species. This tool integrates three main algorithms, WGA, VCF2ALIGNMENT and VCF2SYNTENY, and allows the detection SHPs for the study of diploid-polyploid complexes with available diploid progenitor genomes, without assembling and annotating the genomes of the allopolyploids under study. AlloSHP has been validated on three diploid-polyploid plant complexes, Brachypodium, Brassica, and Triticum-Aegilops, and a set of synthetic hybrid yeasts and their progenitors of the genus Saccharomyces. The results and congruent phylogenies obtained from the four datasets demonstrate the potential of AlloSHP for the evolutionary analysis of allopolyploids with a wide range of ploidy and genome sizes. AlloSHP combines the strategies of simultaneous mapping against multiple reference genomes and syntenic alignment of these genomes to call SHPs, using as input data a single VCF file and the reference genomes of the known or closest extant diploid progenitor species. This novel approach provides a valuable tool for the evolutionary study of allopolyploid species, both at the interspecific and intraspecific levels, allowing the simultaneous analysis of a large number of accessions and avoiding the complex process of assembling polyploid genomes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.17.665301v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Gene Duplication, Horizontal Gene Transfer, and Trait Trade-offs Drive Evolution of Post-Fire Resource Acquisition in Pyrophilous Fungi</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sari, E., Enright, D. J., Ordonez, M., Allison, S. D., Homyak, P. M., Wilkins, M. J., Glassman, S. I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comparative genomics framework for 18 pyrophilous fungi from five Ascomycota and three Basidiomycota orders isolated from burned soils shows a dramatic trait trade-off between fast growth and number of genes responsible for aromatic C degradation, implying that these metabolically costly genes are</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Wildfires significantly alter soil carbon (C) and nitrogen (N), reducing microbial richness and biomass, while selecting for fire-loving pyrophilous microbes that drive post-fire nutrient cycling. However, the genomic strategies and functional trade-offs (balancing gains in one trait with costs in another) underlying the traits that enable pyrophilous microbes to survive and thrive post-fire are virtually unknown. We hypothesized that pyrophilous fungi employ specialized genomic adaptations for C and N cycling, with evolutionary trade-offs between traits governing aromatic C degradation, N acquisition pathways, and rapid growth. To test these hypotheses, we assessed hyphal extension rates and constructed a comparative genomics framework for 18 pyrophilous fungi from five Ascomycota (Eurotiales, Pleosporales, Sordariales, Coniochaetales, and Pezizales) and three Basidiomycota (Agaricales, Holtermaniales, and Geminibasidiales) orders isolated from burned soils. We found a dramatic trait trade-off between fast growth and number of genes responsible for aromatic C degradation, implying that these metabolically costly genes are selected for in burned environments despite their evolutionary cost. We used the comparative genomics framework to evaluate the genomic signatures of evolution and found that either gene duplication and somatic mutation, or recombination via sexual reproduction, were the primary drivers of fungal genomic variation in aromatic C degradation and N acquisition genes. Finally, we identified rare cross-kingdom bacterial to fungal horizontal gene transfer as a third strategy producing novel aromatic C degradation genes. Overall, we found that trait trade-offs and genome evolutionary strategies are key drivers of the rapid colonization and persistence of pyrophilous fungi in post-fire ecosystems.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.17.665399v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Enhanced Formulation of Precision Probiotics through Active Machine Learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Panda, A., Adhikari, M., Nasker, S. S., Nayak, A. K., Nayak, S. K., Dash, S. K., Nayak, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We apply an advanced active machine learning approach to predict how excipients affect the growth of Lactobacillus plantarum, a commonly used probiotic.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The human gut microbiome is crucial to health, with dysbiosis increasingly linked to disease. Precision probiotics offer a promising approach to restoring microbial balance, but ensuring probiotic viability through gastrointestinal transit remains a challenge. This study applies an advanced active machine learning (ML) approach to predict how excipients affect the growth of Lactobacillus plantarum, a commonly used probiotic. State-of-the-art experiments were carried out to complement the ML study. Starting with five known excipient-probiotic interactions, we apply active ML over three rounds to predict the effects of 116 excipients, iteratively refining model certainty and accuracy. Five ML models; Neural Networks, Gradient Boosting, Logistic Regression, Random Forest, and Support Vector Machines were trained and evaluated, with the final model achieving certainty levels close to 90%. Unlike previous methods, which retrained new models per iteration, our approach continuously optimized a single model, enhancing prediction stability and reducing error. These results highlight the potential of active ML to support accurate excipient selection in probiotic formulations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.17.665298v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Interventionally-guided representation learning for robust and interpretable AI models in cancer medicine</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kirkham, D., Masina, R., Sammut, S.-J., Mukherjee, S., Rueda, O. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a new class of models for high-dimensional molecular data that is guided by causal information to learn latent representations that are informative, interpretable-by-design and, as we show, allow for strong generalization to new data distributions and settings.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Machine learning models in cancer medicine are powerful but can suffer from a lack of robustness and interpretability that limits their medical and scientific utility. Here, we introduce a new class of models for high-dimensional molecular data that is guided by interventional auxiliary information to learn latent representations that are informative, interpretable-by-design and, as we show, allow for strong generalization to new data distributions and settings. Specifically, the proposed models use causal information from genetic loss-of-function screens to guide learning and thereby discover representations that are generally useful, even far beyond the original training context. Empirical results using cancer cell line datasets support the hypothesis that causal guidance permits strong generalization and we find that the models can be applied in an entirely "zero-shot" fashion--with no fine-tuning or adaptation whatsoever--to entire cancer types that were not encountered during training. Using several large patient cohorts, we show that this strong generalization applies also in the "bench-to-bedside" context of transferring cell line-based models to clinical data. Here, we find that our models, with all representation learning steps trained only on cell line data, can be efficiently transferred to patient data, hence opening up new ways to drive clinical modelling using tractable laboratory assays. Our results provide a new and very general way of leveraging causal and interventional information to construct data-efficient and robust AI-based predictive models for molecular medicine.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.21.662350v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Subphase-Labeled Mitotic Dataset for AI-powered Cell Division Analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ivan, Z. Z., Hirling, D., Grexa, I., Ammeling, J., Micsik, T., Dobra, K., Kuthi, L., Sukosd, F., Aubreville, M., Miczan, V., Horvath, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a new mitotic figure dataset annotated with subphase information based on the MIDOG++ dataset as well as a previously unrepresented tumor domain to enhance the diversity and applicability of the dataset.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mitosis detection represents a critical task in the field of digital pathology, as determination of the mitotic index (MI) plays an important role in the tumor grading and prognostic assessment of patients. Manual determination of MI is a labor-intensive and time-consuming task for practitioners with rather high interobserver variability, thus, automation has become a priority. There has been substantial progress towards creating robust mitosis detection algorithms in recent years, primarily driven by the Mitosis Domain Generalization (MIDOG) challenges. In parallel, there has been growing interest in the molecular characterization of mitosis with the goal of achieving a more comprehensive understanding of its underlying mechanisms in a subphase-specific manner. Here, we introduce a new mitotic figure dataset annotated with subphase information based on the MIDOG++ dataset as well as a previously unrepresented tumor domain to enhance the diversity and applicability of the dataset. We envision a new perspective for domain generalization by improving the performance of models with subtyping mitotic cells into the 5 main stages of normal mitosis, complemented with an atypical mitotic class. We believe that our work broadens the horizon in digital pathology: subtyping information could provide useful help for mitosis detection, while also providing promising new directions in answering biological questions, such as molecular analysis of the subphases on a single cell level.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.17.665280v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The History of Enzyme Evolution Embedded in Metabolism</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Corlett, T., Smith, H. B., Smith, E., Goldford, J., Longo, L. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the history of enzyme discovery and reuse is embedded within the web of interdependencies that constitute contemporary, biosphere-scale metabolism, and that the protein universe responded to a major transition by preferentially reusing pre-existing enzyme domains.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Phylogenetic reconstructions are a primary record of protein evolution. But what other records can attest to the deep history of enzymes, and what tools are needed to decode their meaning? Here, we demonstrate that the history of enzyme discovery and reuse is embedded within the web of interdependencies that constitute contemporary, biosphere-scale metabolism. Using a simple network analysis approach, we reconstruct both the relative temporal ordering of enzyme domain emergence and, where possible, the first reactions that they catalyzed. These network-based histories were found to be broadly concordant with phyletic information, suggesting that the two approaches reflect related generative processes. When enzyme emergence is initiated after the discovery of nucleotide cofactors, a predominantly stepwise trajectory of domain discovery is recovered. We find that the earliest enzyme-mediated metabolisms were dominated by /{beta} domains, likely due to their high discoverability and functional potential under constraint. Finally, we quantify how the protein universe responded to a major transition, the biological production of molecular oxygen, by preferentially reusing pre-existing enzyme domains. This work presents a self-consistent model of metabolic and enzyme evolution, essential progress towards integrating multiple, independent records into a unified history of protein evolution.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.16.665256v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Base modification analysis in long read sequencing data using Minimod</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Samarasinghe, S., Deveson, I. W., Gamaarachchi, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Minimod is a new vendor-agnostic tool designed to extract and analyse any type of base modification from sequencing data generated by any platform that supports MM/ML tags.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent advances in long read sequencing technologies have enabled the detection of various DNA and RNA base modifications in addition to standard nucleotide sequences. Both major vendors in this space - Oxford Nanopore Technologies (ONT) and Pacific Biosciences (PacBio) - now include base modification information in their sequencing outputs using MM/ML tags embedded in unaligned BAM files. Each vendor also provides dedicated tools for extracting and analysing these tags, such as ONT's modkit and PacBio's pb-CpG-tools. This work presents minimod, a new vendor-agnostic tool designed to extract and analyse any type of base modification from sequencing data generated by any platform that supports MM/ML tags. Benchmarking demonstrates that for DNA data, minimod is ~1.25X faster on a server and ~4X on a laptop compared to modkit and pb-CpG-tools. For RNA data, minimod achieves even greater speedups compared to modkit, ~12X on the server and ~55X on the laptop. Minimod is a free, open-source application written in C and is available at https://github.com/warp9seq/minimod.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.16.665072v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Clonal lineage tracing of innate immune cells in human cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liu, V., Sandor, K., Yan, P. K., Miao, M., Yin, Y., Stickels, R. R., Chen, A. Y., Hiam-Galvez, K., Gutierrez, J., Zhang, W., Sajjath, S. M., Valbuena, R., Wang, S., Daniel, B., Ludwig, L. S., Howitt, B. E., Lareau, C. A., Satpathy, A. T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Clonal tracing of innate immune cells from tumor and tissue sites and malignancies .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Innate immune cells constitute the majority of the tumor microenvironment (TME), where they mediate both natural anti-tumor immunity and immunotherapy responses. While single-cell T- and B-cell receptor sequencing has provided fundamental insights into the clonal dynamics of human adaptive immunity, the lack of appropriate tools has precluded similar analysis of innate immune cells. Here, we describe a method that leverages somatic mitochondrial DNA (mtDNA) mutations to reconstruct clonal lineage relationships between single cells across cell types in native human tissues. We jointly sequenced single-cell transposase-accessible chromatin and mtDNA to profile n=124,958 cells from matched tumor, non-involved lung tissue (NILT), and peripheral blood of early-stage non-small cell lung cancer (NSCLC) patients, as well as n=93,757 cells from matched tumor and peripheral blood of ovarian cancer patients. Single-cell concomitant profiling of lineage and cell states of thousands of immune cells resolved clonality across cell types, tissue sites, and malignancies. Clonal tracing of innate immune cells demonstrates that TME-resident myeloid subsets, including macrophages and type 3 dendritic cells (DC3), are clonally linked to both circulating and tissue-infiltrating monocytes. Further, we identify distinct DC-biased and macrophage-biased myeloid clones, enriched in the tumor and NILT, respectively, and find that their circulating monocyte precursors exhibit distinct epigenetic profiles, suggesting that myeloid differentiation fate may be predetermined before TME infiltration. These results delineate the clonal pathways of intratumoral myeloid cell recruitment and differentiation in human cancer and suggest that remodeling of the tumor myeloid compartment may be peripherally programmed.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.16.665245v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The reference genome for the northeastern Pacific bull kelp, Nereocystis luetkeana</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Alves-Lima, C., Montecinos, G., Escalona, M., Calhoun, S., Marimuthu, M., Nguyen, O., Berault, E., Lipzen, A., Grigoriev, I. V., Raimondi, P., Nuzhdin, S. V., Alberto, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We report the complete and annotated genome of Nereocystis luetkeana, a large-body kelp with severe population declines in northern California, the Salish Sea, and recently in some populations in Oregon.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Bull kelp, Nereocystis luetkeana, is a northeastern Pacific kelp with broad distribution from Alaska to central California. Its population declines have caused severe concerns in northern California, the Salish Sea in Washington, and recently in some populations in Oregon. Despite bull kelp's accumulated ecological and physiological studies, an assembled and annotated genomic reference was still unavailable. Here, we report the complete and annotated genome of Nereocystis luetkeana, produced by the California Conservation Genomics Project (CCGP), which aims to reveal genomic diversity patterns across California by sequencing the complete genomes of approximately 150 carefully selected species. The genome was assembled into 1,562 scaffolds with 449.82 Mb, 80x of coverage and 22,952 gene models. BUSCO assembly showed a completeness score of 72% for the stramenopiles gene set. The mitochondria and chloroplast genome sequences have 37 Mb and 131 Mb, respectively. The orthology analysis between 10 Phaeophycean genomes showed 1,065 expanded and 286 unique orthogroups for this species. Pairwise comparisons showed 542 orthogroups present only in N. luetkeana and M. pyrifera, another large-body kelp. The enrichment analysis of these orthogroups showed important functions related to central metabolism and signaling due to ATPases enrichment in these two species. This genome assembly will provide an essential resource for the ecology, evolution, conservation, and breeding of bull kelp.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.16.665147v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Temporal Shifts in Gene Expression Drive Quantitative Resistance to a Necrotrophic Fungus in a Tomato Crop Wild Relative</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Einspanier, S., Tobian Herreno, A., Rangegowda, N., Groenenberg, L., van der Lee, T., Stam, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the lag-phase duration of lesion growth rate is regulated by the host's regulatory dynamics, and propose a new model for QDR breeding .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Resistance breeding against generalist necrotrophic pathogens heavily relies on quantitative disease resistance (QDR). Lesion growth dynamics involve distinct phases (e.g., lag phase duration or lesion doubling time), each independently affecting the overall symptom severity. While the genetic and regulatory basis of lesion growth rate has been studied, the host-derived regulation of the lag-phase duration remains largely uncharacterised. In this study, we tested the regulatory response of Solanum pennellii genotypes exhibiting different lag-phase durations. We conducted a time-series gene expression profiling experiment dissecting genotype-specific regulatory responses to Sclerotinia sclerotiorum inoculation. We observed genotype-specific regulatory trajectories, with resistant plants displaying early activation of defence-related genes during the asymptomatic phase. These genes, regulated by a WRKY6-centered gene regulatory network, exhibited elevated basal expression in resistant genotypes and a fine-tuned longitudinal expression with induction before lesion onset. In contrast, susceptible genotypes lacked this early response, showing gene induction only post-infection. This study is the first to link host regulatory dynamics to lag-phase duration, suggesting that elevated basal expression of receptor genes and a WRKY6-mediated gene regulatory network may enhance QDR. These findings provide insights into the regulatory foundation of QDR and establish a functional basis for more focused breeding of QDR traits.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.17.665297v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Copper Stress Trigger Organelles Communication and Chromatin Condensation Leading to Cell Death in Solanum lycopersicum</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chouhan, S., Chandra, S., Salam, A., Nandi, C. K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that copper exposure in the root apex of Solanum lycopersicum leads to profound nuclear alterations, including shrinkage, lobulation, peripheral chromatin tethering, and global condensation events tightly correlated with stress signaling.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Copper (Cu) is a vital micronutrient for plants but becomes highly toxic when present in excess, disrupting redox balance and damaging cellular structures. While the physiological and mitochondrial responses to Cu toxicity are well-documented, the nuclear-level consequences, particularly chromatin remodeling and gene regulatory changes, remain poorly understood. In this study, we used the root apex of Solanum lycopersicum as a model system to explore how increasing copper concentrations affect organelle integrity, stress signaling, and nuclear architecture. Using confocal and super-resolution imaging with organelle-specific markers and immunostaining, we observed that mitochondria was the earliest affected to Cu stress, exhibiting fragmentation, membrane depolarization, and cytochrome c release led to reactive oxygen species (ROS) accumulation, activation of AMPK, suppression of mTOR signaling, and nuclear translocation of NRF2. Critically, we found that copper exposure induced profound nuclear alterations, including shrinkage, lobulation, peripheral chromatin tethering, and global condensation events tightly correlated with stress signaling. H3K4me3 immunostaining revealed a shift from active euchromatin to condensed, transcriptionally silent states, leading to membrane rupture and cell death in root tip cells. Our findings show that the nucleus actively integrates organelle-derived stress signals, with chromatin remodeling as a key marker of copper toxicity. This highlights potential for chromatin-based diagnostics and stress-resilient crop breeding.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.17.665307v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Thylakoids reorganization enables driving photosynthesis under far-red light in the microalga Nannochloropsis gaditana</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liistro, E., Battistuzzi, M., Storti, M., Boccia, B., Cocola, L., Perin, G., Morosinotto, T., La Rocca, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel mechanism of acclimation to far-red light in the marine microalga Nannochloropsis gaditana, that resulted capable of growth relying solely on far- red light.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> [bullet] Oxygenic photosynthesis is driven by visible light in most photosynthetic organisms, with exceptions in few cyanobacteria and microalgae species, that can extend the light absorption to far-red wavelengths, by synthesizing new pigments or shifting the antennae absorption range beyond 700 nm. [bullet] In this work, we describe a novel mechanism of acclimation in the marine microalga Nannochloropsis gaditana, that resulted capable of growth relying solely on far-red light. Unexpectedly, the response did not involve the synthesis of red-shifted absorption forms, rather a peculiar reorganization of chloroplasts. [bullet] The abundance of photosynthetic complexes changed, with an increased accumulation of all pigment binding proteins and photosystem II. Chloroplasts became bigger and thylakoid membranes increased in number occupying almost all the organelle volume, showing also newly observed structures, composed of a central superstack with perpendicular electrondense interconnections, that we propose to name thylakoidal bodies. [bullet] To the best of our knowledge, these results describe a novel acclimation strategy to far-red light, overall highlighting that the biodiversity of responses to far-red light is currently underestimated.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.17.665317v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The transcriptional mechanism behind Mimosa pudica leaf-folding in response to mechanical disturbance</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Buti, M., Checcucci, A., Vergata, C., Renna, L., Pollastri, S., Loreto, F., Mancuso, S., Martinelli, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We study the gene regulatory networks underlying leaf folding in Mimosa pudica in response to mechanical disturbances, which may help design sustainable strategies of plant stress defense.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mimosa pudica is a plant known for its ability to fold leaves in response to mechanical disturbances, which serves as a visible phenotypic stress marker. Leaf folding occurs with a response timing and intensity that varies depending on the stimulus. This adaptive behavior may function as a defense mechanism, helping plant resist herbivores and environmental stressors. However, the molecular and genetic mechanisms that are involved in leaf folding are still not totally understood. In this study, the gene regulatory networks underlying M. pudica leaf closure following single and multiple mechanical disturbances (whole pot drops) were investigated. Chlorophyll fluorescence was measured as fast phenotypic indicator of transient or permanent photochemical damage, and transcriptional responses were measured to identify the key genes regulating phenotypic changes after single or multiple drops. A progressive reduction of the quantum yield of photosystem 2 revealed a lower electron transport rate in leaves subjected to one or more drops, which may indicate the onset of energy shortage, perhaps caused by low ATP availability, limiting both leaf movement and photosynthesis. The transcriptomic profiles revealed larger differences when plants were subjected to multiple drops than to a single drop, with respect to unstressed controls. Interestingly, following a single drop, the majority of up-regulated genes were associated with the flavonoid biosynthetic pathway. After multiple drops, however, genes associated with biotic and abiotic stress resistance pathways were predominantly up-regulated. These results provide a basis for developing a gene regulatory network model of stress-induced movements in M. pudica leaves, which may help design sustainable strategies of plant stress defense.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.17.665295v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Graph pangenome reveals structural variation dynamics during cucumber breeding</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhao, X., Yu, J., Zhang, J., Sun, H., Wu, S., Zhao, J., Zhou, Y., Hammar, S., Lin, Y.-C., Zhang, Z., Huang, S., Dymerski, R., Chen, F., Weng, Y., Grumet, R., Xu, Y., Fei, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a graph-based pangenome for cucumber, which captures 171,892 high-confidence SVs and enables genotyping of these SVs across 443 cucumber accessions, representing diverse geographic origins and spanning wild progenitors, landraces, and modern cultiv</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Structural variants (SVs) represent a significant yet underexplored component of plant genome diversity. Analyzing SV dynamics is essential for understanding their contributions to phenotypic variation and their influence on genome evolution. Here, we present a graph-based pangenome for cucumber, constructed from 39 reference-level genome assemblies. The pangenome captures 171,892 high-confidence SVs and enables genotyping of these SVs across 443 cucumber accessions, representing diverse geographic origins and spanning wild progenitors, landraces, and modern cultivars. Our analyses reveal that during cucumber domestication, a substantial portion of mildly deleterious SNPs are retained, whereas SVs are consistently purged, highlighting their potentially highly deleterious nature. During the geographic expansion of cucumber, a reduction in SV burden and a younger age of SVs than SNPs were observed, suggesting stronger purifying selection acting on SVs. Additionally, notable gene flow from wild population to African and European populations was detected, resulting in an increased SV burden, potentially due to hitchhiking effects. Importantly, incorporating SV burden into genomic prediction models improved the prediction accuracy for several agronomically important traits. This study illuminates the dynamics of SVs during cucumber domestication and improvement, highlights the complex interplay between SVs and selection pressures, and underscores the extensive implications of SVs for future cucumber breeding.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.17.665377v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A conserved small RNA-generating gene cluster undergoes sequence diversification and contributes to plant immunity</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Feng, L., Hou, Y., Toghani, A., Wang, Z., Tang, B., Atkinson, N., Li, H., Qiao, Y., Wang, Y., Hua, J., Zhai, J., Ma, W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that siRNA production is an ancient function of an evolutionarily conserved clade of PPR genes that undergo extensive within-species diversification and highlights the potential of P PR-siRNA-based engineering for enhancing broad-spectrum disease resistance.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Small RNA-mediated gene silencing contributes to plant immunity. The secondary small interfering RNA (siRNA) pathway promotes defense by silencing target genes in invading fungal and oomycete pathogens. Many secondary siRNAs derive from transcripts potentially encoding pentatricopeptide repeat (PPR) proteins. Here, we report that siRNA production is an ancient function of an evolutionarily conserved clade of PPR genes that undergo extensive within-species diversification. In Arabidopsis thaliana, siRNA-source PPRs are physically clustered in one locus on Chromosome 1. These sequences are diversified through gene duplication followed by sequence diversification as well as accumulation of high-impact variations including pseudogenization. This diversity leads to the accumulation of a diverse PPR-siRNA pool, consistent with an engagement in a co-evolutionary arms race with the pathogens. This study defines siRNA-producing PPRs as a family of defense genes and highlights the potential of PPR-siRNA-based engineering for enhancing broad-spectrum disease resistance.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.20.665670v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Metabolic engineering approach to boost γ-decalactone production in S. cerevisiae identifies a critical role for the peroxisomal Fatty Acyl-CoA Synthetase Faa2</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chia, D. B., Chow, J. Y., Alfatah, M., Hoon, S., Arumugam, P., Lim, H. H., Surana, U.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A metabolic flux-engineering approach to produce {gamma}-decalactone from oleic acid in Saccharomyces cerevisiae .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Lactones constitute a family of aroma compounds found in fruits, flowers and vegetables and are in high demand in the food industry. Previous studies have reported biotransformation of castor beans-extracted ricinoleic acid to {gamma}-decalactone using oleaginous yeast Yarrowia lipolytica. Given the potential toxicities associated with castor beans, we have used a metabolic flux-engineering approach to produce {gamma}-decalactone from oleic acid in Saccharomyces cerevisiae. Intracellular conversion of oleic acid to ricinoleic acid was achieved by the expression of oleate hydroxylase Fah12 from ergot fungus Claviceps purpurea. Glycerol-3-phosphate dehydrogenase-mediated glycerol synthesis was identified as the major metabolic diversion of oleic acid that negatively impacts {gamma}-decalactone yields. Chemogenomic profiling analysis revealed that the tryptophan biosynthetic pathway provides resistance to {gamma}-decalactone-mediated toxicity in yeast. Overexpression of tryptophan transporter Tat1 enhanced {gamma}-decalactone production by about 3-fold. Deficiency of genes encoding the cytoplasmic fatty acyl CoA synthetases FAA1 or FAA4 alone did not significantly influence {gamma}-decalactone production. However, deficiency of peroxisomal FAA2 drastically diminished the yield of {gamma}-decalactone. Thus, this study uncovers the metabolic barriers to oleic acid-to-{gamma}-decalactone conversion and identifies Faa2 as an essential element in this biotransformation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.21.665937v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

