<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Monday, 30 June 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The human transcription factor occupancy landscape viewed using high-resolution in situ base-conversion strand-specific single-molecule chromatin accessibility mapping</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Marinov, G. K., Doughty, B. R., Schaepe, J. M., Wang, T., Smith, M. M., Chen, M., Kundaje, A., Sun, Z., Greenleaf, W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A single-molecule footprinting approach for chromatin accessibility profiling based on the application of highly active sequence context-independent DNA deaminases.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Chromatin accessibility profiling is a key tool for mapping the location of cis-regulatory elements (cREs) in the genome and tracking chromatin state dynamics during development, in response to various external and internal stimuli, and in disease contexts. Single-molecule footprinting (SMF) methods that rely on the labeling of individual accessible DNA bases have emerged in recent years as a powerful chromatin accessibility mapping approach, as they provide not just an average readout of accessibility over a given genomic position but also the distribution of accessibility states within a population at the level of individual original DNA molecules. However, SMF approaches have been limited either in their resolution or in labeling readout accuracy. To address these limitations, we have developed a high-resolution strand-specific single molecule footprinting approach (C[->]U/T-ssSMF) based on the application of highly active sequence context-independent endogenous methylation-insensitive double-strand DNA (dsDNA) deaminases (CseDa01 and LbDa02), which convert accessible cytosines into uracils (and in turn into thymine after amplification). We demonstrate the application of the method to mapping fine-grained single-molecule accessibility states in human cell lines in both a short and a long-read format, and quantifying the occupancy states of individual transcription factors (TFs) as well as TF co-accessibility and strand-specific accessibility patterns.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.27.662080v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MetaNet: a scalable and integrated tool for reproducible omics network analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Peng, C., Huang, Z., Wei, X., Jiang, L., Zhu, X., Liu, Z., Chen, Q., Shen, X., Gao, P., Jiang, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed MetaNet, a high-performance R package designed to construct, visualize, and analyze biological networks from multi-omics datasets.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Network analysis is a powerful strategy for uncovering complex relationships in high-throughput omics datasets. However, current tools often lack scalability, flexibility, and native support for multi-omics integration, posing significant barriers for exploring complex biological networks. To address these limitations, we developed MetaNet, a high-performance R package designed to construct, visualize, and analyze biological networks from multi-omics datasets. MetaNet supports highly efficient correlation-based network construction, scalable to datasets with over 10,000 features, and includes extensive layout algorithms and visualization options compatible with both static and interactive platforms. It also provides a comprehensive suite of topological and stability metrics for in-depth network characterization. Benchmarking results show that MetaNet outperforms existing R packages by up to 100-fold in computation time and reduces memory usage by up to 50-fold. We demonstrate its utility through two case studies: (1) a longitudinal analysis of microbial co-occurrence networks showing the dynamics of the airborne microbiome, and (2) an integrative exposome-transcriptome network of more than 40,000 features, uncovering distinct regulatory impacts of biological and chemical exposures. MetaNet bridges the gap between network theory and omics application by offering a robust, reproducible, and biologically informed framework for large-scale, interpretable, and integrative network analyses across diverse omics platforms, advancing systems-level understanding in modern life sciences. MetaNet is available on the Comprehensive R Archive Network (https://cran.r-project.org/web/packages/MetaNet).</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.26.661636v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Haplomatic: A Deep-Learning Tool for Adaptively Scaling Resolution in Genetic Mapping Studies</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Douglas, T., Tarvin, R., Long, A. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Haplomatic, a novel deep-learning-based tool that adaptively adjusts genomic resolution by predicting haplotype frequency estimation error.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genomic mapping studies face a fundamental trade-off between accuracy and resolution: increasing resolution improves localization of genetic signals but typically reduces the accuracy of frequency estimates due to increased statistical noise. In pooled sequencing studies, this trade-off impacts the accuracy of haplotype frequency estimates, a primary statistic used to identify genetic associations.To mitigate this trade-off we introduce Haplomatic, a novel deep-learning-based tool that adaptively adjusts genomic resolution by predicting haplotype frequency estimation error. Haplomatic generates simulated population data from known recombinant inbred line populations, predicts error through a transformer-based neural network, and adjusts resolution until a target error is achieved. Haplomatic achieves significant resolution gains (up to 13%) over previous methods without sacrificing accuracy across multiple evaluated sequencing depths (10x, 50x, and 100x). To our knowledge, this is the first instance of applying deep learning to directly predict estimation error and dynamically scale resolution in trait mapping studies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.25.661582v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DiPPER2 - a user-friendly pipeline for picking and evaluating taxon-specific PCR primers</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wacker, T., Studholme, D. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a user-friendly, reliable and comprehensive tool DiPPER2 (Diagnostic Primer Picking and Evaluation pipeline for Reproducibility and Reliability), which finds unique diagnostic targets, designs end-point or qPCR primers for them, evaluates the primers in sil</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Rapid, specific and sensitive detection is essential for pathogenic microorganisms in contexts ranging from phytosanitary-regulated diseases in agriculture to sepsis management in a clinical setting. Well-established, cost-effective, genomic-sequence-based methods are replacing or complementing laborious, slow and costly culture-based diagnostics. These methods include: end-point PCR and quantitative real-time PCR (qPCR), as well as digital PCR (dPCR) and droplet digital PCR (ddPCR). All these methods require oligonucleotide primers whose nucleotide sequences confer high specificity and sensitivity for the intended target. The most challenging steps in designing these assays include finding genomic sequences that are unique to the target pathogen and absent from off-target organisms, as well as designing primers for those sequences. Here, we present user-friendly, reliable and comprehensive tool DiPPER2 (Diagnostic Primer Picking and Evaluation pipeline for Reproducibility and Reliability), which finds unique diagnostic targets, designs end-point or qPCR primers for them, evaluates the primers in silico for specificity and sensitivity, characterizes the diagnostic target and produces a comprehensive and intuitive report. The pipeline has been developed according to FAIR (findable, accessible, interoperable and reproducible) principles and allows for flexible adjustment of primer parameters. It designs primers compatible with qPCR-, dPCR- and ddPCR. DiPPER2 can also evaluate previously designed primer sequences in silico for specificity and sensitivity. We use Pseudomonas amygdali pv. morsprunorum as an example to show that a clade-based, phylogeny-informed approach to diagnostic primer design ensures effective target discovery and specificity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.26.661695v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">OocystMeter, a machine-learning algorithm to count and measure Plasmodium oocysts, reveals clustering patterns in the Anopheles midgut</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Peng, D., Marrogi, E., Nelson, E., Liu, Q., Rinvee, T. A., de Vries, L. E., Thornburg, K., Singh, N., Shaw, W. R., Catteruccia, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present OocystMeter, a machine learning-based software developed to automate the segmentation of malaria oocysts from images of mosquito midguts stained with mercurochrome.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We present OocystMeter, a machine learning-based software developed to automate the segmentation of malaria oocysts from images of mosquito midguts stained with mercurochrome. Existing bioimage analysis tools, including machine learning-based ones, often struggle with the unique staining patterns, complex midgut backgrounds, and variable morphology of oocysts, making the determination of oocyst size and numbers cumbersome. To overcome these challenges, we curated a high-quality dataset comprised of 11,178 Plasmodium falciparum oocysts in Anopheles gambiae midguts annotated by expert parasitologists. Using this dataset, we fine-tuned a Mask R-CNN object detection model to achieve segmentation accuracy comparable to human parasitologists (Spearman's correlation of 0.998 for oocyst counts and 0.978 for size measurements). Applying this tool in conjunction with spatial analysis, we uncovered a non-random, clustered spatial distribution of oocysts independent of the midgut's anatomical regions or geometric axes, particularly in infections with fewer than 75 oocysts/midgut. Our workflow significantly accelerates malaria oocyst intensity and size analysis, reduces human bias, and provides spatial coordinates for advanced parasitology studies. OocystMeter is freely available at https://github.com/duopeng/OocystMeter, and as a web tool at http://Oocystmeter.org/, offering a valuable resource for researchers investigating the oocyst stage of malaria development.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.28.662088v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Design and Optimization of Multifunctional Peptide Candidates for Cosmeceutical Applications: Combining Anti-Inflammatory and Collagen-Boosting Properties</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Natarajan, A., Javvadi, A., Kolli, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We design and evaluate multifunctional cosmeceutical peptides that can simultaneously modulate collagen remodelling, exhibit anti-inflammatory responses, and maintain extracellular matrix stability.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Skin aging is multifactorial and involves inflammation, oxidative stress, collagen degradation, and impaired metal ion homeostasis, which requires holistic approaches for dermal rejuvenation. Conventional cosmeceutical peptides often target singular pathways, limiting their efficacy. In this study, we present a computational strategy for designing and evaluating multifunctional peptides that can simultaneously modulate collagen remodelling, exhibit anti-inflammatory responses, and maintain extracellular matrix (ECM) stability. We focused on three important skin-associated proteins: TGF- {beta} receptor II, Integrin 5 {beta} 1, and MMP-1. and generated peptide sequences specific to them through a de novo sequence generation method that utilized protein-specific profiles alongside machine learning-based property screening. Over 15,000 peptides were generated and filtered using physicochemical and synthesis-based criteria (charge, hydrophobicity, pI, solubility, and synthesis rules). A multi-parametric composite score was developed to visualize peptide favourability in a 3D physicochemical space. The top peptides from each target set were subjected to structure prediction and docking, resulting in binding affinities that ranged from -19.8 to -34.5 kcal/mol. Cross-target docking analysis demonstrated a significant potential for dual or pan-target applications, indicating the broad-spectrum utility of the cosmeceutical. This study firmly establishes a foundation for multifunctional peptide candidates that combine collagen-boosting, anti-inflammatory, and MMP-inhibitory effects. These candidates are poised to lead the way for innovative skincare formulations, paving the path for future validation and experimental evidence that will undeniably strengthen our findings.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.26.661667v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CRISPR/Cas9 editing of the wheat iron sensor TaHRZ1 confirms its conserved role in iron homeostasis and allocation in grains</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tyagi, D., Banoo, H., Meena, V., Joon, R., Agrwal, K., Kumar, A., Satbhai, S. B., Long, T., Pandey, A. K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of wheat HRZ homoeologs and CRISPR-Cas9 editing of the conserved HHE3 domain in all the TaHRZ1 homoeology, coupled with GRF4-GIF1 chimeric protein, improved iron loading in wheat cultivar C306</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Plants rely on specialised sensing systems, including transcriptional regulators, to maintain iron (Fe) homeostasis. Among these, Hemerythrin RING Zinc finger (HRZ) proteins have emerged as key regulators of Fe homeostasis. In this study, six wheat HRZ homoeologs TaHRZ1 and TaHRZ2 were identified from rice HRZ sequences and mapped to chromosomes 1 and 3. These encode proteins with conserved N-terminal Hemerythrin (HHE) domains and C-terminal CHY-RING and Zn-ribbon motifs. Phylogenetic analysis grouped these genes into distinct clades, while expression profiling revealed strong root-specific and Fe-responsive expression patterns, indicating roles in nutrient sensing. Functional conservation was demonstrated by complementation of the Arabidopsis bts-1 mutant, where both wheat genes restored normal Fe regulation. Full-length TaHRZ1 and TaHRZ2 interacted with members of wheat bHLH IVc transcription factors, while truncated versions lacking the RING domain did not, emphasising their conserved role in protein interactions. CRISPR-Cas9 editing of the conserved HHE3 domain in all the TaHRZ1 homoeologs, coupled with GRF4-GIF1 chimeric protein, achieved ~9% regeneration efficiency in wheat cultivar C306. Grain ICP-MS analysis indicated enhanced iron loading in the edited lines, particularly in the scutellum, suggesting improved iron partitioning compared to the wild type. Additionally, qRT-PCR revealed upregulation of FIT and IRO3, and downregulation of IDEF1 in edited lines, supporting a central role for TaHRZ1 in Fe homeostasis signalling. These findings position TaHRZ1 as a valuable target for biofortification strategies to enhance Fe content in wheat grains.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.29.662202v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

