<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Sunday, 06 July 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Population differences of chromosome 22q11.2 duplication structure predisposes differentially to microdeletion and inversion.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Porubsky, D., Yoo, D., Dishuck, P. C., Koundinya, N., Souche, E., Harvey, W. T., Munson, K. M., Hoekzema, K., Chan, D. D., Leung, T. Y., Santos, M. S., Meynants, S., Swillen, A., Breckpot, J., Tsapalou, V., Hasenfeld, P., Korbel, J. O., Lansdorp, P. M., Vermeesch, J. R., Eichler, E. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that high-order segmental duplications known as low copy repeats (LCR) drive the most common genomic disorder, chromosome 22q11.2 microdeletion syndrome, while recurrent large-scale inversions help explain why this syndrome is less prevalent among individuals of African descent.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The most common genomic disorder, chromosome 22q11.2 microdeletion syndrome (22q11.2DS), is mediated by highly identical and polymorphic segmental duplications (SDs) known as low copy repeats (LCRs; regions A-D) that have been challenging to sequence and characterize. Here, we report the sequence-resolved genomic architecture of 135 chromosome 22q11.2 haplotypes from diverse 1000 Genomes Project samples. We find that more than 90% of the copy number variation is polarized to the most proximal LCR region A (LCRA) where 50 distinct structural configurations are observed (~189 kbp to ~2.15 Mbp or 11-fold length variation). A higher-order SD cassette structure of 105 kbp in length, flanked by 25 kbp long inverted repeats, drives this variation and emerged in the human-chimpanzee ancestral lineage later expanding in humans ~1.0 [0.8-1.2] million years ago. African LCRA haplotypes are significantly longer (p=0.0047) when compared to non-Africans yet are predicted to be more protected against recurrent microdeletions (p=0.00053) due to a preponderance of flanking SDs in an inverted orientation. Conversely, we identified nine distinct inversion polymorphisms, including five recurrent ~2.28 Mbp inversions extending across the critical region (LCRA-D) and four smaller inversions (two LCRA-B, one LCRC-D, and one LCRB-D); 7/9 of these events were identified in haplotypes of African and admixed American ancestry. Finally, we sequence and assemble four families and show that LCRA-D deletion breakpoints map to the 105 kbp repeat unit while inversion breakpoints associate with the 25 kbp repeats adjacent to palindromic AT-rich regions. In one family, we observe evidence of more complex unequal crossover events associated with gene conversion and multiple breakpoints. Our findings suggest that specific haplotype configurations are protective and susceptible to chromosome 22q11.2DS while recurrent large-scale inversions help to explain why this syndrome is less prevalent among individuals of African descent.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.04.662981v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Image-based, pooled phenotyping reveals multidimensional, disease-specific variant effects</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pendyala, S., Partington, K., Bradley, N., McEwen, A. E., Straub, G., Kim, H.-J., Fayer, S., Holmes, D. L., Sitko, K. A., Vandi, A. J., Powell, R. L., Friedman, C. E., McDermot, E., Kishore, N., Roth, F. P., Rubin, A. F., Yang, K.-C., Starita, L. M., Noble, W. S., Fowler, D. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A pooled, image-based method that measures variant effects on molecular and cellular phenotypes in diverse cell types.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genetic variants often produce complex phenotypic effects that confound current assays and predictive models. We developed Variant in situ sequencing (VIS-seq), a pooled, image-based method that measures variant effects on molecular and cellular phenotypes in diverse cell types. Applying VIS-seq to ~3,000 LMNA and PTEN variants yielded high-dimensional morphological profiles that captured variant-driven changes in protein abundance, localization, activity and cell architecture. We identified gain-of-function LMNA variants that reshape the nucleus and autism-associated PTEN variants that mislocalize. Morphological profiles predicted variant pathogenicity with near-perfect accuracy and distinguished autism-linked from tumor syndrome-linked PTEN variants. Most variants impacted a multidimensional continuum of phenotypes not recapitulated by any single functional readout. By linking protein variation to cell images at scale, we illuminate how variant effects cascade from molecular to subcellular to cell morphological phenotypes, providing a framework for resolving the complexity of variant function.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.03.663081v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SpatialMap: A Scalable Deep Learning Method for Cell Typing in Subcellular Spatial Transcriptomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jia, P., Long, Y., Zhao, Y., Zhang, N., Fang, Z., Chen, S., Li, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present SpatialMap, a deep weakly supervised cell typing framework that combines graph neural networks (GNN) with transfer learning to annotate cellular identities in subcellular spatial transcriptomics (ST).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Subcellular imaging transcriptomics has offered unprecedented resolution of spatial mapping of gene expressions at subcellular level. However, accurate and robust cell type annotation at cellular level remains a challenge. Here we present SpatialMap, a deep weakly supervised cell typing framework that combines graph neural networks (GNN) with transfer learning to annotate cellular identities in subcellular spatial transcriptomics (ST). SpatialMap consistently outperforms state-of-the-art annotation methods on both simulated and real-world ST datasets. It demonstrates strong scalability across diverse tissue types and platforms, including NanoString CosMx, MERFISH and 10x Genomics Xenium. When applied to a mouse primary motor cortex dataset, SpatialMap successfully identified Micro, L4/5 IT, and L6 IT Car3 absent from the reference data, showcasing its capability to discover novel cell types. Furthermore, SpatialMap accurately annotated malignant cells in human liver cancer samples. Its design also enables seamless extension to cross-omics cell typing and cross-platform annotation within proteomics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.03.662904v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Gain of Function p53 mutant R273H confers distinct methylation profiles and consequent partial or full EMT states to colon tumour</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rani, H., Subhadarshini, S., Jolly, M. K., Mahadevan, V.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that p53 R273H mutation associates with a partial epithelial-mesenchymal transition (pEMT) state and metastatic progression in colorectal cancer.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> p53 is the second most frequently mutated gene in colorectal cancer. While different p53 mutations have been correlated with metastasis, the distinct phenotypes exhibited by site-specific mutations of p53 are not well elucidated. Here, we analyse transcriptomic and methylation data from TCGA-COAD cohort to understand the epigenetic impact of three most prevalent hotspot mutations of p53 (R175H, R273H and R282W). We observed that p53 R273H mutation associates with a partial epithelial-mesenchymal transition (pEMT) state and metastatic progression. In vitro ChIP-seq experiments conducted on p53R27H harbouring HT29 cells revealed an enrichment of mutant p53 R273H at pEMT or mesenchymal gene sets. Further, simulations from a gene regulatory network incorporating the interactions of p53R273H with EMT regulators explain how this mutation shapes the phenotypic landscape accessible to cancer cells. Finally, single-cell transcriptomic analysis of colorectal tumours reveals R273H-linked enrichment of partial and mesenchymal EMT phenotypes across tumour subpopulations in CRC. Overall, we identified distinct epigenetic regulation regulating partial EMT and consequent aggressive behaviour triggered by p53R273H. These findings can help devise effective therapeutic strategies for p53 mutant specific colon tumours.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.03.662954v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Computational Workflow for Structure-Guided Design of Potent and Selective Kinase Peptide Substrates</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yekeen, A. A., Meyer, C. J., McCoy, M., Posner, B., Westover, K. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present Subtimizer (substrate optimizer), a streamlined computational pipeline for structure-guided kinase peptide substrate design using AlphaFold-Multimer for structure modeling, ProteinMPNN for sequence design, andAlphaFold2-based interface evaluation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Kinases are pivotal cell signaling regulators and prominent drug targets. Short peptide substrates are widely used in kinase activity assays essential for investigating kinase biology and drug discovery. However, designing substrates with high activity and specificity remains challenging. Here, we present Subtimizer (substrate optimizer), a streamlined computational pipeline for structure-guided kinase peptide substrate design using AlphaFold-Multimer for structure modeling, ProteinMPNN for sequence design, and AlphaFold2-based interface evaluation. Applied to five kinases, four showed substantially improved activity (up to 350%) with designed peptides. Kinetic analyses revealed >2-fold reductions in Michaelis constant (Km), indicating improved enzyme-substrate affinity. Two designed peptides exhibited >5-fold improvement in selectivity. This study demonstrates AI-driven structure-guided protein design as an effective approach for developing potent and selective kinase substrates, facilitating assay development for drug discovery and functional investigation of the kinome.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.04.663216v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SSAlign: Ultrafast and Sensitive Protein Structure Search at Scale</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, L., Zhang, X., Wang, Y., Xue, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We have developed SSAlign, a protein structure retrieval tool that leverages protein language models to jointly encode sequence and structural information, and adopts a two-stage alignment strategy optimized with multi-GPU and multi-process parallelization.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The advent of highly accurate structure prediction techniques such as AlphaFold3 is driving an unprecedented expansion of protein structure databases. This rapid growth creates an urgent demand for novel search tools, as even the current fastest available methods like Foldseek face significant limitations in sensitivity and scalability when confronted with these massive repositories. To meet this challenge, we have developed SSAlign, a protein structure retrieval tool that leverages protein language models to jointly encode sequence and structural information, and adopts a two-stage alignment strategy optimized with multi-GPU and multi-process parallelization. On large-scale datasets such as AFDB50, SSAlign outpaces Foldseek by two to three orders of magnitude in search speed, offering unmatched scalability for high-throughput structural analysis. Compared to Foldseek, SSAlign retrieves substantially more high-quality matches on Swiss-Prot and achieves marked performance improvements on SCOPe40, with relative AUC increases of +20.2% at the family level and +33.3% at the superfamily level, demonstrating significantly enhanced sensitivity and recall. In sum, SSAlign achieves TM-align-comparable accuracy with Foldseek-surpassing speed and coverage, offering an efficient, sensitive, and scalable solution for large-scale structural biology and structure-based drug discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.03.662911v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Transformer-based Deep Learning for Glycan Structure Inference from Tandem Mass Spectrometry</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Abtheen, E. A., Singh, A., Sriram, S., Chen, C., Neelamegham, S., Gunawan, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present GlycoBERT and GlycoBART, novel transformer-based models for glycan structure prediction from MS/MS data, surpassing the current state-of-the-art deep learning model, CandyCrunch.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Glycans play critical roles in diverse biological processes, but their structural analysis by tandem mass spectrometry (MS/MS) remains a major challenge due to their branched structure and stereochemistry. Traditional computational methods, such as database searching, are constrained by the scope of existing libraries and can be computationally intensive. While recent deep learning models have advanced the field, they often struggle to capture the complex, long-range dependencies within MS/MS spectra required for accurate inference. To address these challenges, we present GlycoBERT and GlycoBART, novel transformer-based models for glycan structure prediction from MS/MS data. GlycoBERT, a sequence classifier, achieves 95.1% structural accuracy on test data, surpassing the current state-of-the-art deep learning model, CandyCrunch. However, classification-based methods are inherently limited to predicting structures present in the training data. To overcome this, we developed GlycoBART, a generative sequence-to-sequence model capable of de novo glycan inference. On independent validation datasets, both models demonstrate robust performance. Critically, when applied to an MS/MS dataset from human embryonic kidney cells, GlycoBART generated two de novo glycan structures absent from the training set, one of which is a novel structure not catalogued in major glycan databases. Together, GlycoBERT and GlycoBART establish a new benchmark for glycan analysis, offering a powerful framework that enables more accurate and comprehensive exploration of glycan diversity and discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.02.662857v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">RNA liquid biopsy via nanopore sequencing for novel biomarker discovery and cancer early detection</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Peddu, V., Hill, A., Velandi Maroli, S. L., Mattingly, C., Gardner, J. M. V., Miga, K. H., Fitzgerald, R. C., Kim, D. H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that long-read nanopore sequencing of full-length cell-free RNA in plasma from healthy individuals, precancerous Barretts esophagus patients with high-grade dysplasia, or patients with esophageal adenocarcinoma reveals a diverse cell-</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Liquid biopsies detect disease noninvasively by profiling cell-free nucleic acids that are secreted into the circulation. However, existing methods exhibit low sensitivity for detecting early stages of diseases such as cancer. Here we show that long-read nanopore sequencing of full-length cell-free RNA in plasma from healthy individuals, precancerous Barretts esophagus patients with high-grade dysplasia, or patients with esophageal adenocarcinoma reveals a diverse cell-free RNA transcriptome that can be leveraged for detecting and treating disease. We discovered 270,679 novel, intergenic cell-free RNAs, which we used to build a custom transcriptome reference for quantification, feature selection, and machine learning to accurately classify both precancer and cancer. Moreover, we found potential therapeutic targets, including metabolic, signaling, and immune checkpoint pathways, that are highly upregulated in both precancer and cancer patients. Our findings highlight the utility of our RNA liquid biopsy platform technology for discovering and targeting early stages of disease with molecular precision.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.02.662774v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Evidence supporting the first secondary chromosome in actinobacteria as a hallmark of the Embleya genus</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gomez-Escribano, J. P., Dorai-Raj, S., Baker, D., Lacey, E., Wilkinson, B., Booth, T. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose EEC1 as a secondary chromosome, distinct from previously described plasmid-like replication mechanisms (chromids) and the largest secondary replicon reported in bacteria, to date.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Embleya is a genus within the family Streptomycetaceae, a group of actinobacteria with outstanding capacity for production of specialised metabolites and a strikingly complex life cycle. In this work, we sequenced the complete genome of the new species Embleya australiensis MST-11070 and validated the assembly using optical mapping. The genome of E. australiensis MST-11070 consists of a 7.1 Mb linear chromosome and three additional replicons, including a 4.2 Mb linear replicon, EEC1, significantly larger than all previously described secondary replicons from bacteria. EEC1 is typified by its similar composition to the chromosome in terms of GC-content, codon usage and gene functions. It also carries terminal inverted repeats identical to the chromosome. EEC1 is enriched in biosynthetic gene clusters (BGCs), including the only copy of the BGCs for the spore pigment and the surfactant peptide SapB, metabolites essential for the organism's lifecycle. EEC1 contains an origin of replication with at least some chromosomal properties, and its replication is likely to depend on functions provided by chromosomally located genes. Further comparison of Embleya spp. genomes suggests that EEC1-like replicons are conserved across the genus, in contrast to other known large linear extrachromosomal replicons (megaplasmids) in the order. EEC1 is thus a hallmark of the Embleya genus and is central to its evolution within the Streptomycetaceae family. We propose EEC1 as a secondary chromosome, distinct from previously described secondary chromosomes that utilise plasmid-like replication mechanisms (chromids) and the largest secondary replicon reported in bacteria, to date.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.03.662523v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Epigenetic adaptation of beta cells across lifespan and disease: age-related demethylation is advanced in type 2 diabetes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Manduchi, E., Descamp, H., Liu, J., Schug, J., Da, T., Lahori, D., El-Mekkoussi, H., Lee, M., Feleke, E., Liu, C., Naji, A., Glaser, B., Kaestner, K. H., Avrahami, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We study the epigenetic basis of beta cell adaptation to metabolic demand in type 2 diabetes and show that age-related demethylation in healthy beta cells is accelerated as part of a compensatory response that ultimately fails under sustained insulin resistance.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Although the prevalence of type 2 diabetes (T2D) increases with age, most adults maintain normoglycemia despite rising insulin resistance, largely due to the adaptive capacity of pancreatic beta cells to meet increased metabolic demand. However, persistent insulin resistance can lead to beta cell dysfunction and T2D onset. Here, leveraging cell-type-specific methylome data from the Human Pancreas Analysis Program (HPAP), we investigate the epigenomic basis of beta cell adaptation by mapping genome-wide DNA methylation (DNAm) patterns across the human lifespan. In healthy donors, we identify progressive age-related demethylation enriched in cis-regulatory elements at beta cell identity and function genes, suggesting that epigenetic remodeling supports functional adaptation to metabolic demand over time. In contrast, alpha cells show the opposite trajectory, with subtle, age-related hypermethylation. In T2D beta but not alpha cells we observed further demethylation compared to healthy controls, underscoring a unique capacity of beta cells to respond to changes in metabolic demand. Together, our findings suggest that DNAm remodeling in healthy beta cells reflects a long-term adaptation to metabolic demand, which in T2D is accelerated as part of a compensatory response that ultimately fails under sustained insulin resistance.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.02.662814v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Expression patterns of Arabidopsis bZIP19 and bZIP23 elucidate their individual contribution to the regulation of the Zn deficiency response</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Campilho, A., Lilay, G. H., Benvindo, M., Castro, P. H., Assuncao, A. G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We analysed the phenotypes of bZIP19, bzip23 and bzip19bzip23 single and double mutants in response to Zn deficiency, and histochemical GUS staining in pbZIP 19::GUS and pb Zn-dependent lines.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Plants control the availability of nutrients through tightly regulated homeostasis networks. In Arabidopsis thaliana, the transcription factors bZIP19 and bZIP23 regulate the homeostasis of the micronutrient Zn, acting as the key regulators of the Zn deficiency response and as sensors of cellular Zn status. Although there is a degree of functional redundancy between bZIP19 and bZIP23, their individual contributions in this regulatory process are not yet understood. To gain insight on the individual contributions of bZIP19 and bZIP23, we analysed the phenotypes of bzip19, bzip23 and bzip19bzip23 single and double mutants in response to Zn deficiency, and histochemical GUS staining in pbZIP19::GUS and pbZIP23::GUS lines. Analysis of mutant phenotypes indicated unequal redundancy between bZIP19 and bZIP23. Expression of GUS in pbZIP19::GUS was observed in differentiated root tissues including root hairs, in the vascular tissue of hypocotyl, cotyledons and petals, in leaf trichomes and stamen filament. In pbZIP23::GUS, the expression was predominant in root and shoot apical meristems, being also observed in leaves, root vasculature, stamen filament and anthers. In summary, bZIP19 and bZIP23 have different and mostly non-overlapping tissue-specific expression patterns. This may explain the unequal redundancy between bZIP19 and bZIP23 and their evolutionary retention as paralogs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.03.662962v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Gene family expansions and nodule-specific expression patterns reveal the recruitment of Beta-Glucosidases and Cytochrome P450 genes to nodulation in soybean</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cardoso-Silva, C. B., Quintanilha-Peixoto, G., Turquetti-Morares, D. K., Almeida-Silva, F., Venancio, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A genome-wide analysis of gene family expansions in legumes and integrated expression data from over 5,000 soybean RNA-Seq samples to identify genes involved in nodulation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Nodulation is a symbiotic interaction between legumes and rhizobia that is essential for biological nitrogen fixation and regulated by complex gene networks, which are partially shaped by whole-genome duplication (WGD) events. Although many core nodulation genes have been identified, the contribution of lineage-specific gene expansions and WGD-derived duplicates remains poorly understood. In this study, we performed a genome-wide analysis of gene family expansions in legumes and integrated expression data from over 5,000 soybean RNA-Seq samples to identify genes involved in nodulation. Our analysis not only recovered well-characterized nodulation genes among the most expanded families but also revealed several differentially expressed genes in nodules, including nodulins, aspartic proteases, NLP proteins, and GRAS transcription factors. Notably, we identified previously unreported {beta}-glucosidase and cytochrome P450 genes with high nodule-specific expression, suggesting their involvement as novel components of the nodulation machinery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.03.662952v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">In Situ Inference of Copy Number Variations in Image-Based Spatial Transcriptomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jensen, A. E. V., Crowell, H., Pascual Reguant, A., Ruano, I., Tejpar, S., Heyn, H., Nilsson, M., Marco Salas, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a new approach that adapts CNV inference to iST data, enabling spatial mapping of malignant clones and the tumor microenvironment, at single-cell resolution.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Copy number variations (CNVs) drive cancer progression. So far, spatial CNV inference has relied on whole transcriptome-based sequencing technologies. However, advances in image-based spatial transcriptomics (iST) now enable high-plex gene measurement in situ. Here, we introduce an approach that adapts CNV inference to iST data, enabling spatial mapping of malignant clones and the tumor microenvironment, at single-cell resolution. Additionally, we assess how panel size and detection efficiency influence CNV inference.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.02.662761v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">OMIDIENT: Multiomics Integration for Cancer by Dirichlet Auto-Encoder Networks</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Safinianaini, N., Valimaki, N., Bresson, R., Gorbonos, A., Rajamaki, K., Aaltonen, L. A., Marttinen, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a novel multi-omics integrative method that performs unsupervised representation learning, referred to as OMIDIENT: multiOMics Integration for cancer by DIrichlet auto-ENcoder neTworks.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> To achieve a more comprehensive understanding of cancer, novel computational methods are required for the integrative analysis of data from different molecular layers, such as genomics, transcriptomics, and epigenomics. Here, we present a novel multi-omics integrative method that performs unsupervised representation learning, referred to as OMIDIENT: multiOMics Integration for cancer by DIrichlet auto-ENcoder neTworks. OMIDIENT provides a natural framework for modeling sparse and compositional latent representations by employing a deep generative model, where the latent space is distributed as the product of Dirichlet distributions. Applied to five different cancers, we demonstrate that OMIDIENT outperforms the top state-of-the-art unsupervised multi-omics integrative analysis approaches in clustering, classification, and reconstruction of missing data using mRNA expression data, DNA methylation data, and microRNA expression data. Furthermore, we provide interpretability analyses for OMIDIENT that not only support its improved performance, but also offer valuable insights into the underlying structure captured by the learned representations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.02.662608v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">OmniPert: A Deep Learning Foundation Model for Predicting Responses to Genetic and Chemical Perturbations in Single Cancer Cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Taj, F., Stein, L. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a deep learning, transformer-based architecture to model the effects of genetic and chemical perturbations on single-cell transcriptomes, which allows for more granular analysis of cellular responses, thereby facilitating downstream applications in cell-specific gene-gene and gene-drug interaction networks, biomark</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In cancer, intra- and inter-patient heterogeneity presents a significant challenge for therapeutic management, as patients with apparently similar profiles often exhibit divergent responses to the same therapies. This heterogeneity is primarily attributed to genetic and molecular variations among individuals and their tumors. Understanding the impact of these differences on treatment outcomes is widely believed to be a key step for developing effective precision medicine strategies. However, the complexity of most biological pathways makes it difficult to predict the effect of genetic variation on cells and tissues, let alone predict a patient's response to therapy. As a result, high-throughput genetic and chemical perturbation screens have emerged as valuable tools for precision medicine-related tasks, such as disease modeling, target discovery, cellular programming, and pathway reconstruction. This approach is fundamentally limited, however, because the number of possible combinations of cell types, cell states, perturbation targets, and perturbation types is huge and cannot be exhaustively tested experimentally. This calls for computational approaches that can simulate such experiments in silico, guiding in vitro experiments towards perturbations that are more likely to produce the desired effect. Here we describe OmniPert, a novel generative AI tool, which utilizes a deep learning, transformer-based architecture to model the effects of genetic and chemical perturbations on single-cell transcriptomes. Trained on millions of diverse cellular profiles, this approach allows for more granular analysis of cellular responses, thereby facilitating downstream applications in cell-specific gene-gene and gene-drug interaction networks, biomarker and drug target discovery, drug repurposing, and in silico perturbation reverse-engineering. In the context of oncology, OmniPert promises to facilitate the discovery of novel cell type- and state-specific targets, ultimately contributing to more effective and personalized cancer treatments.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.02.662744v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Mechanistic modeling and machine learning identifies optimum radiotherapy schedules to prevent treatment-induced metastasis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Graser, C., Zhou, Z., SchuÃàrch, M., Moorhead, G., Dean, J., Lin, C., Dean, J., Kozono, D., Lahav, G., Michor, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified optimal fractionation schedules and then performed extensive in silico clinical trials, establishing that our optimized schedules substantially reduce metastatic seeding relative to the standard of care.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Lung cancer patients often experience increased metastasis formation after radiotherapy. However, it is incompletely understood whether radiation affects the migratory behavior of tumor cells and how altered radiotherapy schedules might mitigate this risk. To address these questions, we performed live-cell microscopy experiments to profile changes in cell migration during radiation across 12 cancer cell lines and developed a predictive computational modeling platform describing tumor volume and dissemination during radiotherapy. Using this platform, we identified optimal fractionation schedules and then performed extensive in silico clinical trials, establishing that our optimized schedules substantially reduce metastatic seeding relative to the standard of care schedule. Training transformer models on the in silico clinical trial data enabled us to recover mechanistic parameters with high accuracy, demonstrating that the features determining optimal radiotherapy can be inferred from longitudinal tumor data. Our integrative predictive approach enables the rational design of optimum clinical trials across indications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.02.662373v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">STELLA: Self-Evolving LLM Agent for Biomedical Research</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jin, R., ZHANG, Z., Wang, M., Cong, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a self-evolving AI agent that learns from experience and improves its own capabilities over time to accelerate the pace of biomedical discovery.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The rapid growth of biomedical data, tools, and literature has created a fragmented research landscape that outpaces human expertise. While AI agents offer a solution, they typically rely on static, manually-curated toolsets, limiting their ability to adapt and scale. Here, we introduce STELLA, a self-evolving AI agent designed to overcome these limitations. STELLA employs a multi-agent architecture that autonomously improves its own capabilities through two core mechanisms: an evolving Template Library for reasoning strategies and a dynamic Tool Ocean that expands as a Tool Creation Agent automatically discovers and integrates new bioinformatics tools. This allows STELLA to learn from experience. We demonstrate that STELLA achieves state-of-the-art accuracy on a suite of biomedical benchmarks, scoring approximately 26% on Humanity's Last Exam: Biomedicine, 54% on LAB-Bench: DBQA, and 63% on LAB-Bench: LitQA, outperforming leading models by up to 6 percentage points. More importantly, we show that its performance systematically improves with experience; for instance, its accuracy on the Humanity's Last Exam benchmark almost doubles with increased trials. STELLA represents a significant advance towards AI Agent systems that can learn and grow, dynamically scaling their expertise to accelerate the pace of biomedical discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.01.662467v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

