<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Friday, 12 September 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A framework to detect positive selection using variant effect predictions reveals widespread adaptive evolution of human neurons</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Starr, A. L., Palmer, M. E., Gao, J., Nichols, C. G., Fraser, H. B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Detecting positive selection is essential to understanding evolution. Many methods to detect positive selection use simple classifications of genetic variants (e.g. synonymous/nonsynonymous). Here, we propose that these methods can be considered special cases of a more general framework to detect positive selection with any variant effect prediction method. Using evolutionary conservation and deep learning-based variant effect predictions, we apply this framework genome-wide in the human lineage and identify positive selection on protein sequences of genes involved in brain and heart development, chromatin accessibility in binding sites of dozens of transcription factors, and non-coding substitutions in compact genomic regions with reinforcing cell type-specific effects on cis-regulatory activity. Consistently, the dominant theme was positive selection on genes regulating neuronal connectivity, suggesting that fine-scale changes in brain wiring were essential to the evolution of human cognition. Overall, this framework represents a powerful, versatile tool to investigate adaptive evolution across the tree of life.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.11.675696v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Transcontinental Spread of HPAI H5N1 from South America to Antarctica via Avian Vectors</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> xu, r., Gao, M., Zhang, N., Wei, Z., Wang, Z., Zhang, L., Liu, Y., Zheng, Z., Chen, L., Ding, H., Wang, W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Our study has for the first time identified H5N1 strains (clade 2.3.4.4b, genotype B3.2) in brown skuas from the Fildes Peninsula, South Shetland Islands, Antarctica. These findings indicate that highly pathogenic avian influenza H5N1 viruses is now actively circulating in Antarctic ecosystems, representing a significant expansion of its geographic range.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.06.674605v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Unveiling the Human Nasopharyngeal Microbiome Compendium: Systematic Characterization of Community Architecture and Function Through a Comprehensive Meta-Analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Song, K., Brochu, H. N., Bustos, M. L., Zhang, Q., Icenhour, C. R., Letovsky, S. N., Iyer, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The nasopharyngeal microbiome acts as a dynamic interface between the human body and environmental exposures, modulating immune responses and helping to maintain respiratory health. Despite its clinical significance, the nasopharyngeal microbiome remains poorly characterized with few resources available. A robust standardized framework for distinguishing healthy from dysbiotic communities is needed to advance nasopharyngeal microbiome research. We conducted a comprehensive meta-analysis that yielded 28 independent nasopharyngeal microbiome studies encompassing 7,790 uniformly reprocessed samples from public repositories. Further, we developed a newly introduced, three-stage large language model-assisted background decontamination pipeline that mitigates study-specific technical and environmental variation while preserving biological signal integrity. We identified six reproducible nasopharyngeal community state types (NPCSTs), each dominated by distinct genera: Moraxella (I), Corynebacterium-Dolosigranulum (II), Staphylococcus (III), Haemophilus (IV), mixed (V), and Streptococcus (VI). NPCSTs were key drivers of the sample diversity rather than disease status, requiring NPCST-stratified downstream analyses. Co-occurrence network analysis and functional pathway predictions revealed NPCST-specific microbial interactions and metabolic signatures. To bridge the translational gap from research to clinic, we developed the Nasopharyngeal Microbiome Health Index (NMHI), an independent continuous wellness score that achieves an AUC of 0.90 and 0.92 in internal and external validations, respectively. Unlike binary disease classifiers, the NMHI quantifies nasopharyngeal health along a spectrum, enabling risk stratification for respiratory infections, treatment monitoring, and dysbiosis identification in PCR-negative symptomatic patients. Our comprehensive framework establishes reproducible NPCSTs, validated decontamination protocols, and publicly available classification models, providing the analytical framework for advancing nasopharyngeal microbiome research toward clinical implementation. The NMHI exemplifies international best practices for microbiome diagnostics through rigorous external validation, standardized pipelines, and clinically interpretable wellness metrics. It further delivers an evidence-based framework for translating nasopharyngeal microbiome research into precision respiratory medicine, enabling future development of microbiome-based respiratory therapeutics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.06.673381v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">S3R: Spatially Smooth and Sparse Regression Reveals High-Dimensional Regulatory Networks in Spatial Transcriptomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> zhou, x., dang, p., Tang, H., Peng, L. X., Yeh, J. J., Sears, R. C., Neelon, B., Zimmers, T., Zhang, C., Cao, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Spatial transcriptomics (ST) data demands models that recover how associations among molecular and cellular features change across tissue while contending with noise, collinearity, cell mixing, and thousands of predictors. We present Spatially Smooth Sparse Regression (S3R), a general framework that estimates location specific coefficients linking a response feature to high dimensional spatial predictors. S3R unites structured sparsity with a minimum spanning tree guided smoothness penalty, yielding coefficient fields that are coherent within neighborhoods yet permit sharp boundaries. S3R enables large scale data analysis with an efficient implementation using a reduced MST graph, multi GPU training, and parallel hyperparameter search. In synthetic data, S3R accurately recovers spatially varying effects, selects relevant predictors, and preserves known boundaries. Applied to human dorsolateral prefrontal cortex, S3R recapitulates layer-specific target TF associations with concordant layer wise correlations in matched single cell data. In acute Haemophilus ducreyi skin infection, S3R converts spot level mixtures into cell type attributed expression fields and reveals spatial gradients; applying SVG tests to these fields increases concordance and recovers gradients missed by spot level methods. In pancreatic ductal adenocarcinoma, S3R constructs cross cell type, cross gene co variation tensors that prioritize interactions among cell types with interacting genes enriching pathways consistent with known biology. Because responses and predictors in S3R are user defined, it could flexibly address diverse ST questions within a single, scalable, and interpretable regression framework.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.06.674629v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Drift and isolation drive genomic erosion and island speciation in a lineage of macaques</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Qi, j., Liu, s., Zhang, l., Zhou, z., Wang, r., Shen, y., Teng, y., Zhang, m., Liu, g., Hu, c., Wang, x., Zhao, q., Rusitanmu, D., You, y., Liu, z., Zhou, x., Rogers, J., Roos, C., Li, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Allopatric speciation, especially on large islands and archipelagos, is a significant driver of evolutionary diversification, as geographic isolation fosters the independent evolution of populations. In these isolated populations, lineage sorting and genetic drift dominate, accelerating allele fixation and reducing shared genetic variation. Here, we investigated how sea-level transgression during the Early Holocene triggered rapid speciation in large vertebrates by studying macaques isolated on Dangan Island (DGD), located just 30 km from present-day Hong Kong. Whole-genome sequencing revealed that ~10,000 years of isolation drove the macaques' evolution into a distinct species, as indicated by pronounced genomic divergence (mean Fst > 0.462 vs. mainland), 1.94 million lineage-specific variants, and complete ancestral differentiation with no evidence of post-isolation gene flow. A severe demographic collapse (effective population size, Ne ~ 40) led to substantial genomic erosion (65.8% loss of genetic diversity). Paradoxically, this also enhanced resilience through drift-mediated genetic triage. Increased homozygosity exposed and purged lethal recessive alleles in lipid metabolism pathways (68% reduction in genetic load), while simultaneously fixing mildly deleterious variants, such as a splice-site mutation in SKAP2, thereby generating a form of genomic "burden" alongside rapid immune adaptation via 251 fixed missense mutations. These findings demonstrate that island isolation can drive vertebrate speciation within a few thousand years, with genetic drift playing a dominant role in shaping genomic architecture. Accordingly, conservation strategies should prioritize monitoring loss-of-function (LoF) variants in essential pathways and prescreening for deleterious allele combinations between donors and recipients prior to implementing genetic rescue in small, drift-sensitive populations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.06.674399v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">TESSA: A unified model to detect trajectory-preserved and spatially-variable genes in spatial transcriptomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wu, Y., Su, H., Steele, N., Xie, Y., Cui, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Identifying spatially variable genes (SVGs) has been an essential task in spatial transcriptomics. In addition to SVGs detection, there are genes exhibiting expression patterns that are associated with cellular developmental stages or lineage fates across a tissue section. Identifying such genes could provide novel insights into tumor metastasis. Here, we introduce a unified statistical model, termed TrajEctory-preServed and SpAtially-variable gene detection (TESSA), to detect both types of genes. Moreover, we propose a novel strategy to address the inherent double-dipping issue commonly encountered when assessing temporal gene effects in transcriptomics analysis. We demonstrate the testing performance through extensive simulation studies and real applications to several publicly available datasets. Downstream analyses further highlight the potential of our method in identifying genes associated with tumor progression and enhancing spatial domain detection.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.06.674654v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A network pharmacology-based approach and molecular docking study to explore the therapeutic potential of a nutraceutical formula (Vernolac) in the treatment of cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> De Vass Gunawardane, S., Muhammadh Milhan, M., Chanuka Rathnayaka, P., Garusinghe, P. S., Gunaratne, K. S., Kanagasundaram, A., Darshanamala, T. M. D., Senevirathne, D. C., Wijerathne, S. K., Perera, R. P. C. D., Senathilake, K. S., Rajagopalan, U., Tennakoon, K. H., Samarakoon, S. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Vernolac is a commercially available polyherbal nutraceutical capsule comprised of Vernonia zeylanica aerial parts, Nigella sativa seeds, Hemidesmus indicus roots, Leucas zeylanica aerial parts, and Smilax glabra rhizome. Different herbal formulations, organic extracts, and many isolated phytochemicals of the above plants have been reported to exhibit anticancer properties. However, the anticancer mechanisms of action of Vernolac, as a polyherbal formulation, remain unexplored. This study employed an integrative network pharmacology-based approach, complemented by in vitro experiments, to investigate the anticancer potential of Vernolac. Phytochemicals in Vernolac were retrieved from databases, screened for drug-likeness and oral bioavailability using SwissADME, yielding 155 drug-like phytochemicals, and their protein targets were predicted via SwissTargetPrediction. The intersection of targets of phytochemicals and cancer-related targets from GeneCards yielded 137 common targets. Protein-protein interaction analysis in STRING and Cytoscape identified key hub nodes, including AKT1, BCL2, CASP3, CTNNB1, EGFR, ESR1, GAPDH, HSP90AA1, HSP90AB1, IL6, JUN, SRC, STAT3, and TNF. Clustering, topology, and formula-herb-compound-target-disease and target-pathway networks highlighted key phytochemicals, including vernolactone, thymoquinone, quercetin, nigellidine, -hederin, and carvacrol. Gene ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) enrichment revealed that the identified targets are significantly enriched in multiple cancer pathways. Molecular docking and dynamics simulations identified novel target-ligand interactions. Overall, network analysis suggests that Vernolac may exert anticancer effects through apoptosis induction, immune modulation, antioxidant, anti-inflammation, antiproliferative, and chemoradiosensitizing mechanisms. Moreover, Vernolac may exhibit chemoradioprotective potential by alleviating therapy-induced toxicity, supporting its promise as a potential adjunct to conventional cancer treatments. The Sulforhodamine B assay demonstrated selective antiproliferative activity of Vernolac against cancerous cells MCF-7 (IC50 = 54.01 {+/-} 0.02 g/mL), Caco-2 (IC50 = 85.52 {+/-} 0.13 g/mL), NTERA-2 cl.D1 (IC50 = 42.41 {+/-} 0.06 g/mL), and non-cancerous MCF-10A (IC50 = 803.5 {+/-} 0.03 g/mL). Novel target-ligand interactions identified via molecular docking and dynamics simulations, and the underlying mechanisms of Vernolac predicted in this study, require further validation through in vitro and in vivo experiments.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.06.674609v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">OmniBio: An Easy-to-Use Web App for Kinetic Growth Parameter Calculation from Microplate Reader Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dehnhardt-Amengual, S., Ardiles, C., Guarda, I., Larrondo, L. F., Mardones, W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> OmniBio is a user-friendly web application designed to streamline the analysis of microbial growth curves from microplate reader data, eliminating the need for coding skills or cumbersome file preprocessing. Built in R, using the gcplyr package and Shiny framework, OmniBio supports data outputs from both Gen5 and iControl software, handling multiple sheets within a single file. Users can upload raw data along with experimental metadata to the platform, compute kinetic parameters such as maximum growth rate, lag time, ODmax, and area under the curve, and promptly visualize the results in real-time. The app produces plots for each parameter and a global heatmap analysis for comparing microbial performance across strains and culture media. Importantly, all the primary and processed data can be readily downloaded in a summarized Excel report. In contrast to existing tools, OmniBio optimizes the analysis process by automating repetitive computations and offering an intuitive, step-by-step workflow that does not require bioinformatics training. OmniBio is freely accessible at https://sdehnhardt.shinyapps.io/OmniBio/, and can be modified from open-source code on GitHub (https://github.com/sidehnhardt/OmniBio). The tool enables researchers to conduct high-throughput kinetic analysis in an efficient and user-friendly manner, optimizing both time and resource utilization.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.05.674595v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Extracellular matrix stiffness modulates nuclear lamina organisation and sets nuclear conditions for PRC2 repression.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pigeot, A., Rey-Millet, M., Zine El Aabidine, A., Trullo, A., Costes, L., Manghi, M., Bystricky, K., Andrau, J.-C., Esnault, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Capability of cells to respond to tissue-level elasticity has important physiological and pathological implications. Stiffening of the extracellular matrix (ECM) promotes invasive behaviour of cancer cells, supports the transformation of fibroblasts into cancer-associated fibroblasts and primes stem cell differentiation programs. Here, we investigated how ECM stiffness modulates the Nuclear Lamina (NL) and its impact on gene expression programs, epigenetic marking and 3D genome organisation. By combining hydrogel cell culturing of primary fibroblasts, genomics and super-resolution microscopy, we found that ECM stiffness modifies composition of the NL, modulates long range chromatin interactions, induces changes in chromatin motion and regulates thousands of genes. We identified a specific set of genes coding proteins involved in pathways related to mechanical adaptation such as adhesion and signalling. These genes harbour an apparent bivalent chromatin signature and are expressed under soft condition while repressed in stiff condition through Polycomb Repressive Complex 2 (PRC2). We found that this stiffness-specific repression is tempered by mechano-transduction and the NL. This work uncovers mechano-dependent NL composition, changes in 3D genome organisation and in chromatin motion which underlie adaptative gene expression programs controlled through PRC2.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.07.674693v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Dissecting the epigenome dynamics in human immune cells upon viral and chemical exposure by multimodal single-cell profiling</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Guenduez, I. B., Wei, B., Chen, D. C., Wang, W., Hariharan, M., Norell, T., Broderick, T. J., McClain, M. T., Satterwhite, L. L., Burke, T. W., Petzold, E. A., Shen, X., Woods, C. W., Fowler, V. G., Ruffin, F., Panuwet, P., Barr, D. B., Wilk, A. J., Lee, M. J., Blish, C., Castellino, F., Walley, A. M., Evans, T., Ecker, J. R., Mueller, F., Greenleaf, W. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Pathogen and chemical exposures lead to profound remodeling of the gene-regulatory landscape across human immune cell populations. Exposure-induced epigenetic signatures in cell states can provide insights into gene regulation driven by these cell-extrinsic stimuli. Here, we present a single-nucleus chromatin accessibility atlas of human immune cells of individuals exposed to HIV, COVID-19, Influenza, organophosphates as well as healthy controls. This atlas comprises 271,299 cells and 319,420 candidate regulatory elements that exhibited dynamic accessibility associated with gene regulation across immune cell states. Our longitudinal HIV cohort revealed epigenetic signatures of T cell exhaustion, manifested in changes in the accessibility of binding sites for the FOXP transcription factor family. We further identified changes in the accessibility of candidate regulatory elements in CD14 monocytes upon SARS-CoV-2 exposure that are associated with a switch in NF-{kappa}B to AP-1-based regulation of cytokine networks. By integrating single-cell profiles of DNA methylation from matched samples, we create a multimodal epigenome atlas of human immune cells across various exposure states using the accessibility-derived candidate regulatory elements. Both modalities reveal complementary epigenetic signatures at transcription factor binding sites associated with cell state, as exemplified in the process of memory formation in T-cells. Finally, by linking potentially regulatory DNA methylation signatures to changes in chromatin in monocytes, we identify AP1 motifs exhibiting epigenetic dynamics, indicating selective remodeling in TF networks in cases of severe COVID-19.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.09.675101v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Versatile and efficient non-viral integration of large transgenes in human T cells via CRISPR knock-in and engineered integrases</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kassing, I., Kath, J., Nitulescu, A.-M., Glaser, V., Hartmann, L. M., Pu, Y., Huth, L., Karklins, R., Shaji, S., Ringel, A., Pouzolles, M., Stein, M., Ibrahim, D. M., Wagner, D. L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Current gene transfer methods often lack the precision, versatility, or efficiency when integrating large transgenes, limiting the ability to engineer therapeutic T-cells with more complex payloads. Here, we report one-pot PASTA (Programmable and Site-specific Transgene Addition), a non-viral genome engineering strategy for large gene insertion that combines CRISPR-Cas-mediated homology-directed repair (HDR) and site-specific recombination via serine integrases. Using one-pot PASTA with the Bxb1 integrase, we demonstrate efficient integration of transgenes at multiple genomic loci relevant for T-cell engineering (e.g., TRAC, B2M, CD3E, CD3Z, GAPDH). For constructs larger than 8 kb, one-pot PASTA outperforms conventional HDR by 19-fold on average and prime-editing-assisted site-specific integrase gene editing (PASSIGE) by 5-fold. This enables the delivery of multi-cistronic cargo to generate dual-antigen targeting CAR T-cells with a safety-switch that overcome antigen escape in lymphoma models. Finally, one-pot PASTA can be further optimized with improved integrase enzymes, such as engineered variants of Pa01 or Bxb1, and plasmids with minimized backbones. In summary, one-potnPASTA represents a versatile and scalable platform for precise, non-viral gene insertion in T-cells.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.10.675267v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Engineered histones reshape chromatin in human cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jena, S. G., Nagaraja, S., Earl, A. S., Driller-Colangelo, A. R., Quezada, M. A., Oreskovic, E., Horlbeck, M. A., Zhang, R., Gomarga, W., Buenrostro, J. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Histone proteins and their variants have been found to play crucial and specialized roles in chromatin organization and the regulation of downstream gene expression; however, the relationship between histone sequence and its effect on chromatin organization remains poorly understood, limiting our functional understanding of sequence variation between distinct subtypes and across evolution and frustrating efforts to rationally design synthetic histones that can be used to engineer specified cell states. Here, we make the first advance towards engineered histone-driven chromatin organization. By expressing libraries of sequence variants of core histones in human cells, we identify variants that dominantly modulate chromatin structure. We further interrogate variants using a combination of imaging, proteomics, and genomics to reveal both cis and trans-acting mechanisms of effect. Functional screening with transcription factor libraries identifies transcriptional programs that are facilitated by engineered histone expression. Double mutation screens combined with protein language models allow us to learn sequence-to-function patterns and design synthetic histone proteins optimized to drive specific chromatin states. This work establishes a foundation for the high-throughput evaluation and engineering of chromatin-associated proteins and positions histones as tunable nodes for understanding and modulating mesoscale chromatin organization.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.10.674980v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Unlocking biological insight from single-cell data with an interpretable dual-stream foundation model</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Guo, H., Cui, Q., Zhang, X., Chen, C., Zheng, W., Cai, C., Wang, X., Wang, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Deep learning foundation models are revolutionizing single-cell biology, yet learning holistic and discriminative representations from complex, high-dimensional data remains a central challenge. Although Transformer-based single-cell language models have shown significant progress, they typically rely on a single input-encoding scheme, a practice that results in the loss of critical gene expression information and hinders the effective learning of global cellular representations. To address these challenges, we introduce scDMC, an innovative dual-stream contrastive pre-training framework designed to synergistically optimize information fidelity at both the gene and cell levels. Pre-trained on only 2 million cells far fewer than the datasets used by mainstream models, scDMC sets a new state-of-the-art in multiple benchmark tasks, including cell annotation, clustering, and data integration. More importantly, we demonstrate that scDMC can uncover functional gene modules, infer cell-type-specific regulatory networks in a data-driven manner, and exhibits a high degree of biological interpretability. This work demonstrates an efficient pre-training approach that paves the way for the next generation of powerful and interpretable single-cell foundation models, promising to accelerate the pace of biological discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.05.674596v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comparative Analysis of Primary and Liver Fibroblasts Reveals MET as a Potent Target in Pancreatic Cancer Metastasis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Singh, R., Yousefian, N., Allen, W., Anaraki, C., Beutel, A., Calderon, S., Loveless, I., Mcdonald, O., Morton, J., Imagawa, D., Jutric, Z., Martinez, T., Steele, N., Halbrook, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The tumor microenvironment drives many malignant features of pancreatic ductal adenocarcinoma (PDAC). The fibroblasts within pancreatic tumors promote tissue remodeling, immune suppression, and resistance to therapy. However, the interactions between stromal populations and pancreatic cancer cells are less understood in the liver, the most frequent site of PDAC metastasis. To address this, we employ single cell transcriptomics to compare primary pancreatic vs. liver PDAC lesions. Here, we identify the expression of hepatocyte growth factor (HGF) in fibroblasts and its receptor MET in cancer cells are both markedly increased in the PDAC liver niche. Using functional assays, we validate that mitogenic MET signaling is activated in PDAC cells by liver-derived fibroblasts. Importantly, the inhibition of MET signaling leads to reduced tumor growth in immune competent mouse models. Collectively, our data demonstrates that liver stromal-epithelial crosstalk networks engage in signaling pathways distinct from primary pancreatic tumors, highlighting opportunities to develop new treatments for metastatic disease.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.06.674373v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrated in silico and in vitro approaches identify SNX.2112 as a drug vulnerability in t(7;12) AML stem-like cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ciciro, Y., Ragusa, D., Johnson, E. M., Johns, A., Kanannejad, S., Gurung, R., Oporto Espuelas, M., Suen, C.-W., Torregrosa-Cortes, G., Giustacchini, A., Tosi, S., Pina, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The t(7;12) translocation is a chromosomal rearrangement characteristic of infant Acute Myeloid Leukemia (AML) that results in ectopic overexpression of the homeobox gene MNX1. The resulting AML originates in utero and MNX1 transforms fetal, but not adult, hematopoietic progenitors. Using a 3-dimensional (3D) gastruloid model of blood development, we recently showed that t(7;12)-AML originates at the endothelial-to-hematopoietic transition, explaining its characteristic gene expression signature. Herein, we employ that signature to interrogate the transcriptional profiles of hundreds of human cell lines against the GDSC database of drug sensitivities and identify 12 candidate drugs with putative activity against t(7;12)-AML. We employ a cell line in which we engineered the t(7;12) translocation and systematically test the candidate drugs cell surface phenotype and clonogenic assays. Importantly, we identify SNX.2112 as a potential therapeutic agent against t(7;12)-AML. SNX.2112 selectively eliminates leukemia progenitors and colony-initiation in vitro, and selectively decreases MNX1 expression. The same effects are recapitulated in the 3D gastruloid model. Critically, the signature targeted by SNX.2112 uniquely maps to t(7;12) progenitors in single-cell RNA-seq analysis of leukaemia patients. Combinatorial treatment of t(7;12)-engineered cells with cytarabine or mitoxantrone reveals synergy with conventional chemotherapeutic agents, suggesting the potential of SNX.2112 as a targeted and cytotoxicity-sparing therapeutic approach. Overall, we successfully use an integrated computational and multi-model experimental approach to identify a drug vulnerability of t(7;12)-AML which can be explored for clinical potential.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.06.674620v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MIF-induced CD74+ microglia/macrophages are clinically relevant disease-associated subpopulations in brain metastasis and other CNS disorders</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Alvaro-Espinosa, L., Marquez-Galera, A., Priego, N., Garcia-Calvo, V., Perea-Garcia, M., Hernandez-Oliver, C., Retana, D., Sanchez, O., de Pablos-Aragoneses, A., Garcia-Gomez, P., Grana-Castro, O., Lapuente-Santana, O., Al-Shahrour, F., Cayuela, A., Peset, I., Megias, D., Ola, M., Vareslija, D., Young, L., Marti-Mateos, Y., Enriquez, J. A., Hernandez-Encinas, E., Blanco-Aparicio, C., Soengas, M., Bernhagen, J., Anton-Fernandez, A., Avila, J., Marchena, M. A., Torres, M., de Castro, F., Marquez-Ropero, M., Sierra, A., Lopez-Atalaya, J. P., RENACER,, Valiente, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The upregulation of CD74, a chaperone involved in MHC-II antigen processing1,2, has been broadly reported in virtually all brain disorders analyzed by single-cell RNA sequencing3-6. However, its expression is usually interpreted as indicative of antigen presentation. In parallel, CD74 expression has also been described in cancer cells across multiple tumor types, but interestingly in glioma its expression has been mainly identified in the microenvironment. However, the functional contribution of CD74 to disease progression in the brain, and specifically in secondary brain tumors, has not been directly addressed. Here we described that, in contrast to what it has been assumed, the presence of CD74+ microglia/macrophages, which is induced by increased levels of interferon gamma in the brain affected by metastases, does not relate to its canonical pathway. Instead, CD74's alternative function as cytokine receptor is pivotal. Rewired by increasing levels of its ligand MIF, produced by proliferating cancer cells, the CD74 receptor, upon binding to this ligand, translocates to the nucleus activating a NF-{kappa}B-dependent program promoting metastasis progression. A brain metastasis-associated CD74 signature involves a more aggressive progression of the local disease in patients, while it has no clinical correlation with the matched primary tumor. Furthermore, we identified the CD74+ myeloid population in additional brain disorders including Alzheimer's disease and multiple sclerosis, which shared a pan-disease non-canonical signature with clinical relevance. The brain-penetrant drug ibudilast, which prevents the binding of MIF to CD74, decreases brain metastases in experimental models in vivo and in patient-derived organotypic cultures ex vivo in a primary tumor-agnostic manner. Our findings suggest that MIF/CD74-induced reprogramming of myeloid cells in brain disorders is a novel vulnerability that could be exploited therapeutically against brain metastases, and possibly other brain disorders, guided by a non-invasive molecular strategy.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.06.673623v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Development of an EMT-related exosomal miRNA signature that can predict prognosis in hepatocellular carcinoma</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Missaghimamaghani, O., Nehri, L. N., Bakhshi, S., Karaosmanoglu, O., Sivas, H., Acar, A. C., Banerjee, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Chemoresistance and epithelial-mesenchymal transition (EMT) are associated with failure of cancer chemotherapy and poor survival of patients. We have previously shown that chemoresistance and stemness in hepatocellular carcinoma (HCC) cells was accompanied by the development of partial EMT (p-EMT) and identified a number of EMT-associated proteins that are released from exosomes. In this study, we aimed to identify and classify the differentially expressed (DE) exosomal miRNAs from chemoresistant HuH7 cells undergoing p-EMT. Out of the fifty-four miRNAs that were enriched in the exosomes from these cells compared to controls, thirteen were identified in the exosomes isolated from the serum of HCC patients. These miRNAs targeted genes that were associated with cell-cell junctions, extracellular matrix, cytoskeleton, transcription and signal transduction. Univariate Cox regression analysis indicated that 11/13 miRNAs were associated with either favorable (n=4) or worse (n=7) prognosis. A machine learning algorithm indicated that seven miRNAs (miR 215-5p, miR 340-5p, miR 210-3p, miR 19a-3p, miR19b-3p, miR 1266-5p and miR 25-3p) could predict worse prognosis in multiple datasets with 64-68% accuracy. A Bayesian Inference network analysis with the thirteen miRNAs and their key target proteins, along with EMT and survival as the nodes indicated that the common denominator was transcription, suggesting that the exosomal miRNAs released from cells undergoing p-EMT can mediate phenotypic changes in cells through transcriptional regulation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.06.674651v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Tumor Genotype Dictates Mitochondrial and Immune Vulnerabilities in Liver Cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Unlu, G., Millet, A., Wangdu, K., Donne, R., Erdal, R., DelGaudio, N. L., Uygur, B., Shah, V., Cho, K., Fecke, A., Cansiz, F., Tarcan, Z. C., Isay-Del Viscio, M., Kilic, E., Kurth, I., Molina, H., Sickmann, A., Basturk, O., Patti, G., Beyaz, S., Smith, K. W., Lujambio, A., Tasdogan, A., Tavazoie, S. F., Birsoy, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Although oncogenic alterations influence tumor metabolism, how they impose distinct metabolic programs within a shared tissue context remains poorly defined. Here, we developed a rapid mitochondrial profiling platform to compare metabolites and proteins in genetic models of primary liver cancer (PLC). Analyses of six genetically distinct PLCs revealed that mitochondrial energy metabolism is largely dictated by oncogene identity. Kras-driven tumors required creatine metabolism to buffer energy demands during early tumorigenesis, whereas c-MYC-driven tumors relied on oxidative phosphorylation. Among c-MYC-driven PLCs, Pten-deficient tumors accumulated mitochondrial phosphoethanolamine, a precursor for phosphatidylethanolamine (PE) synthesis. Inhibition of PE synthesis selectively impaired the growth of Pten-deficient tumors and extended survival, in part through enhanced infiltration of CD8+ T cells and sensitization to TNF-mediated cytotoxicity. Mechanistically, loss of PE elevated surface TNF receptor 2 (TNFR2), promoting TNF signaling and pro-inflammatory response. These findings uncover genotype-specific mitochondrial metabolic liabilities and establish PE synthesis as a tumor-intrinsic mechanism of immune evasion in PLC.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.10.675369v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">HIF2-driven PTHrP Causes Cachexia and Hypercalcemia in Kidney Cancer: Treatment with HIF2 Inhibitors</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Abu-Remaileh, M., Stransky, L. A., Bhalerao, N., Shirole, N. H., Jiang, Q., Saad, E., Machaalani, M., Vigeant, S. H., Woldemichael, H., Xu, C., Lu, J., Wei, H., Liu, Z., Sun, W., Enomoto, K., Choueiri, T. K., Pitarresi, J. R., Carr, S. A., Udeshi, N. D., Kaelin, W. G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Kidney cancer frequently causes paraneoplastic syndromes, including hypercalcemia and cachexia, but the underlying mechanisms are incompletely understood. The most common form of kidney cancer, clear cell renal cell carcinoma, is frequently caused by loss of the pVHL tumor suppressor protein and the resulting upregulation of the HIF2 transcription factor. We show that PTHLH, which resides on a ccRCC amplicon on chromosome 12p, is a direct HIF2 transcriptional target in ccRCC. Further, we show that the increased PTHLH expression is both necessary and sufficient for the induction of hypercalcemia and cachexia in preclinical orthotopic cell line tumor models. Consistent with these observations, two different allosteric HIF2 inhibitors, belzutifan and NKT2152, rapidly ameliorated hypercalcemia and cachexia in ccRCC patients, including in some patients who did not exhibit objective tumor shrinkage.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.09.675147v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genotoxic antibody-drug conjugates combined with Bcl-xL inhibitors enhance therapeutic efficacy in metastatic castration-resistant prostate cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Semenova, G., Frank, S., Dumpit, R., Han, W., Coleman, I., Gulati, R., Morrissey, C., Haffner, M., Nelson, P. S., Lee, J. K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Metastatic castration-resistant prostate cancer (mCRPC) is an aggressive subtype of prostate cancer (PC) without curative treatments. Antibody-drug conjugates (ADCs) emerged as promising cancer therapeutics that selectively deliver cytotoxic agents (payloads) to the tumors. Although ADCs have been successfully applied in the treatment of hematological and solid tumors, ADC monotherapy has not demonstrated durable responses in mCRPC and the mechanisms of PC resistance to ADCs have not been thoroughly investigated. Our study aimed to improve ADC efficacy using a new integrated approach for custom ADC design and multiplexing. To nominate rational combinations of ADC targets and ADC payloads, we (1) examined protein co-expression of three clinically relevant surface antigens--B7 homolog 3 (B7-H3), prostate specific membrane antigen (PSMA), and six-transmembrane epithelial antigen of prostate-1 (STEAP1)--in a series of human mCRPC samples and (2) screened established ADC payloads and their combinations in mCRPC cell lines with different phenotypes. We identified synergistic interactions between DNA-damaging payloads and the Bcl-xL inhibitor A-1331852 as well as their coordinated induction of the intrinsic apoptosis pathway. The functional relevance of isolated p53 loss and impaired PC responses to three genotoxic ADCs (B7-H3-seco-DUBA, PSMA-SG3249, and STEAP1-DXd) and their combinations with A-1331852 was established using genetic knockout models. Lastly, we found enhanced in vivo antitumor activity in mCRPC by combining the clinically relevant agents B7-H3-seco-DUBA (vobramitamab duocarmazine) and A-1331852. Collectively, our findings provide rationale for the development of ADC therapies combining genotoxic payloads with Bcl-xL inhibitors for mCRPC.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.05.674562v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

