<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Saturday, 28 June 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Long-read transcriptome assembly reveals vast isoform diversity in the placenta associated with metabolic and endocrine function</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bresnahan, S. T., Yong, H., Wu, W. H., Lopez, S., Chan, J. K. Y., White, F., Jacques, P.-E., Hivert, M.-F., Chan, S.-Y., Love, M. I., Huang, J. Y., Bhattacharya, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comprehensive placenta transcript reference with largest-in-class long-read RNA sequencing yields 37,661 high-confidence isoforms across 12,302 genes, with extensive diversity in genes involved in obesity and growth control.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The placenta plays a critical role in fetal development and mediates maternal metabolic effects on offspring health outcomes. Despite its importance, the placenta remains understudied in large-scale genomic initiatives, with most analyses focusing on gene-level expression based on annotations from adult tissue references that obscure isoform diversity particularly vital to understanding function in developmental tissues. Here, we employed largest-in-class long-read RNA sequencing (N = 72) to generate a comprehensive placental transcript reference, yielding 37,661 high-confidence isoforms across 12,302 genes. Our assembly revealed 14,985 novel isoforms of known genes and 2,759 transcripts representing previously unannotated genes, with extensive diversity in genes involved in obesity, placental lactogen production, and growth control. We demonstrate this approach has two immediate practical advantages: First, the improved reference reduced inferential uncertainty in isoform quantification by approximately 37%, reduced the frequency of low-confidence annotations and increased the yield (read depth) of high-confidence annotations. Second, in analyses of gestational diabetes mellitus (GDM) with short-read RNA-seq datasets from two independent, multi-ancestry birth cohorts (N = 344) as an exemplar, we were able to identify novel isoforms of chorionic somatomammotropin hormone 1 (CSH1) mediating the effect of maternal hyperglycemia on birth weight that was not apparent in conventional gene-level analyses. These findings demonstrate that isoform-level annotation of placental transcriptomics provides greater sensitivity and specificity than gene-level approaches. This enhanced precision may be essential for understanding placental function in developmental programming of metabolism and reveal ancestry- and context-specific variation in placental function and responses to environmental exposures.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.26.661362v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrating multi-omics and machine learning for disease resistance prediction in legumes.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shameela Mohamedikbal, Hawlader A Al-Mamun, Mitchell S Bestry, Jacqueline Batley, David Edwards</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Multi-omics assisted prediction of disease resistance mechanisms using machine learning has the potential to accelerate the breeding of resistant legumes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Multi-omics assisted prediction of disease resistance mechanisms using machine learning has the potential to accelerate the breeding of resistant legume varieties. Grain legumes, such as soybean (Glycine max (L.) Merr.), chickpea (Cicer arietinum L.), and lentil (Lens culinaris Medik.) play an important role in combating micronutrient malnutrition in the growing human population. However, plant diseases significantly reduce grain yield, causing 10-40% losses in major food crops. The genetic mechanisms associated with disease resistance in legumes have been widely studied using genomic approaches. Multi-omics data encompassing various biological layers such as the transcriptome, epigenome, proteome, and metabolome, in addition to the genome, enables researchers to gain a deeper understanding of these complementary layers and their roles in complex legume-pathogen interactions. Genomic prediction, used to select the best genotypes with desirable traits for breeding, has largely relied on genome-wide markers and statistical approaches to estimate the breeding values of individuals. Integrating multi-omics data into genomic prediction can be achieved using machine learning models, which can capture nonlinear relationships prevalent in high-dimensional data better than traditional statistical methods. This integration may enable more accurate predictions and identification of resistance mechanisms for breeding resistant legumes. Despite its potential, multi-omics integration for disease resistance prediction in legumes has been largely unexplored. In this review, we explore omics studies focusing on disease resistance in legumes and discuss how machine learning models can integrate multi-omics data for disease resistance prediction. Such multi-omics assisted prediction has the potential to reduce the breeding cycle for developing disease-resistant legume varieties.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40579624/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">3Mont: A multi-omics integrative tool for breast cancer subtype stratification.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Miray Unlu Yazici, J S Marron, Burcu Bakir-Gungor, Fei Zou, Malik Yousef</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel approach called 3-Multi-Omics Network and Integration Tool (3Mont), which integrates various -omics data by applying a grouping function, detecting pro-groups, and assigning scores to each pro-group using Feature importance scoring (FIS) component.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Breast Cancer (BRCA) is a heterogeneous disease, and it is one of the most prevalent cancer types among women. Developing effective treatment strategies that address diverse types of BRCA is crucial. Notably, among different BRCA molecular sub-types, Hormone Receptor negative (HR-) BRCA cases, especially Basal-like BRCA sub-types, lack estrogen and progesterone hormone receptors and they exhibit a higher tumor growth rate compared to HR+ cases. Improving survival time and predicting prognosis for distinct molecular profiles is substantial. In this study, we propose a novel approach called 3-Multi-Omics Network and Integration Tool (3Mont), which integrates various -omics data by applying a grouping function, detecting pro-groups, and assigning scores to each pro-group using Feature importance scoring (FIS) component. Following that, machine learning (ML) models are constructed based on the prominent pro-groups, which enable the extraction of promising biomarkers for distinguishing BRCA sub-types. Our tool allows users to analyze the collective behavior of features in each pro-group (biological groups) utilizing ML algorithms. In addition, by constructing the pro-groups and equalizing the feature numbers in each pro-group using the FIS component, this process achieves a significant 20% speedup over the 3Mint tool. Contrary to conventional methods, 3Mont generates networks that illustrate the interplay of the prominent biomarkers of different -omics data. Accordingly, exploring the concerted actions of features in pro-groups facilitates understanding the dynamics of the biomarkers within the generated networks and developing effective strategies for better cancer sub-type stratification. The 3Mont tool, along with all supporting materials, can be found at https://github.com/malikyousef/3Mont.git.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40577268/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Interpreting Attention Mechanisms in Genomic Transformer Models: A Framework for Biological Insights</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Consens, M. E., Diaz-Navarro, A., Chu, V., Stein, L. D., he, h. h., Moses, A., Wang, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A method to interpret attention heads in genomic transformers by correlating per-token attention scores with curated biological annotations, and a framework to trace how biological features are learned from random initialization to pre-training to fine-tuning.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Transformer models have shown strong performance on biological sequence prediction tasks, but the interpretability of their internal mechanisms remains underexplored. Given their application in biomedical research, understanding the mechanisms behind these models' predictions is crucial for their widespread adoption. We introduce a method to interpret attention heads in genomic transformers by correlating per-token attention scores with curated biological annotations, and we use GPT-4 to summarize each head's focus. Applying this to DNABERT, Nucleotide Transformer, and scGPT, we find that attention heads learn biologically meaningful associations during self-supervised pre-training and that these associations shift with fine-tuning. We show that interpretability varies with tokenization scheme, and that context-dependence plays a key role in head behaviour. Through ablation, we demonstrate that heads strongly associated with biological features are more important for task performance than uninformative heads in the same layers. In DNABERT trained for TATA promoter prediction, we observe heads with positive and negative associations reflecting positive and negative learning dynamics. Our results offer a framework to trace how biological features are learned from random initialization to pre-training to fine-tuning, enabling insight into how genomic foundation models represent nucleotides, genes, and cells.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.26.661544v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Hybrid-metagenomics reveal off-target effects of albendazole, ivermectin-albendazole and moxidectin-albendazole on the human gut bacteria.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dommann, J., Sprecher, V. P., Beisel, C., Ballmer, D., HuÃàrlimann, E., Coulibaly, J. T., Keiser, J., Schneeberger, P. H. H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> ivermectin-albendazole-moxidectin combination therapies induce taxonomic shifts in the gut microbiota with only mild functional consequences, while the resistome is largely spared.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The human whipworm parasite, Trichuris trichiura, poses a critical public health problem, affecting over 400 million people globally and responding poorly to the standard-of-care - benzimidazole chemotherapy. Recent efforts aimed at developing improved treatment options, the two macrolide-benzimidazole combination therapies ivermectin-albendazole and moxidectin-albendazole. Recent reports suggest that ivermectin and moxidectin possess antibacterial properties in vitro, hence there is a need to comprehensively characterize their off-target effects on the gut microbiome. In the framework of a randomized-controlled trial we collected stool samples of 204 T. trichiura-infected individuals in Cote d'Ivoire receiving albendazole (400mg), albendazole-ivermectin (400mg/200{micro}g/kg) and albendazole-moxidectin (400mg/8mg). Using a state-of-the-art hybrid sequencing approach that combines Illumina short read and Oxford Nanopore long read shotgun sequencing, we recovered over 800 high-quality metagenome-assembled genomes. Our analyses reveal that albendazole and albendazole-moxidectin induce taxonomic shifts in the gut microbiota with only mild functional consequences. In contrast, in individuals receiving higher quantities of albendazole-ivermectin (ivermectin >12mg), based on their weight, we observed profoundly modulated taxonomic composition and microbial function, while the resistome was largely spared. These findings robustly confirm ivermectin's antibacterial properties in the human gut, extending beyond previously reported in vitro effects. Given that ivermectin is a cornerstone in parasitic disease control, it is crucial to evaluate its broader impacts and considering more targeted, evidence-based treatment strategies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.27.660518v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">STRESS: Spatial Transcriptome Resolution Enhancing Method based on the State Space Model</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, X., Ye, F., LIU, X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose STRESS, the first deep learning method designed specifically for resolution enhancement tasks using only SRT data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The widespread application of spatial resolved transcriptomics (SRT) has provided a wealth of data for characterizing gene expression patterns within the spatial microenvironments of various tissues. However, the inherent resolution limitations of SRT in most published data restrict deeper biological insights. To overcome this resolution limitation, we propose STRESS, the first deep learning method designed specifically for resolution enhancement tasks using only SRT data. By constructing a 3D structure that integrates spatial location information and gene expression levels, STRESS identifies interaction relationships between different locations and genes, predicts the gene expression profiles in the gaps surrounding each spot, and achieves resolution enhancement based on these predictions. STRESS has been trained and tested on datasets from multiple platforms and tissue types, and its utility in downstream analyses has been validated using independent datasets. We demonstrate that this resolution enhancement facilitates the identification and delineation of spatial domains. STRESS holds significant potential for advancing data mining in spatial transcriptomics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.25.661624v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Chromosome-scale genome assembly for Yellow Wood sorrel, Oxalis stricta</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wood, J. C., Hamilton, J. P., Vaillancourt, B., Brose, J., Edger, P. P., Buell, C. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a chromosome-scale assembly for yellow wood sorrel, revealing its allotetraploid nature and synteny within its two subgenomes as well as synteny with Averrhoa carambola and Oxalis articulata.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Yellow wood sorrel (Oxalis stricta L.), also known as sourgrass, juicy fruit, or sheep weed, is a member of the Oxalidaceae family. Yellow wood sorrel is commonly considered a weed and while native to North America, it is distributed across Europe, Asia, and Africa. To date, only two other genomes from the Oxalidaceae family have been published, star fruit (Averrhoa carambola L.) and Oxalis articulata Savingy. Here, we present a chromosome-scale assembly for O. stricta, revealing its allotetraploid nature and synteny within its two subgenomes as well as synteny with A. carambola and O. articulata. Using Oxford Nanopore Technologies long-read sequences coupled with chromatin capture sequencing, we generated a 436 Mb chromosome-scale assembly of O. stricta with a scaffold N50 length of 36.2 Mb that is anchored to 12 chromosomes across the two subgenomes. Assessment of the final genome assembly using the Long Terminal Repeat Assembly Index yielded a score of 13.12 and assessment of Benchmarking Universal Single Copy Orthologs revealed 99.3% complete orthologs; both metrics are suggestive of a high-quality reference genome. Total repetitive sequence content in the O. stricta genome was 39.7% with retroelements being the largest class of transposable elements. Annotation of protein-coding genes yielded 61,550 high confidence genes encoding 115,089 gene models. Synteny between the two O. stricta subgenomes was present in 91 syntenic blocks containing 40,705 genes, of which, 76.6% were present in a 1:1 syntenic relationships between the two subgenomes. The availability of an annotated chromosome-scale high quality genome assembly for O. stricta will provide launching point to understand the high fecundity of this weed and provide further foundation for comparative genomics within the Oxalidaceae.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.26.661616v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Protein Dynamics at Different Timescales Unlock Access to Hidden Post-Translational Modification Sites</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lang, B., Streit, J. O., Kriwacki, R. W., Christodoulou, J., Babu, M. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We found nearly a fifth of protein modification sites to be buried within proteins and found that some sites remained fully buried in the native structure, enabling signal transduction across diverse timescales.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Post-translational modifications (PTMs) alter the proteome in response to intra- and extracellular signals, providing fundamental information processing in development, homeostasis and disease. Here, we used proteome-wide structure predictions to investigate the structural context of protein modification sites and found nearly a fifth to be buried within proteins. These cryptic sites within folded domains showed evolutionary conservation across species, as well as significantly less variability within the human population. We also found significantly more modification sites to be affected by recurrent cancer mutations and disease variants, with some variants appearing to mimic the modified or unmodified state, supporting their functionality and disease relevance. To understand how these buried sites become modified, we studied 23 phosphorylation sites across seven proteins using all-atom molecular dynamics simulations. The simulations revealed a range of mechanisms spanning broad timescales, ranging from picosecond fluctuations to seconds for proline isomerisation-gated phosphorylation sites. Notably, some sites remained fully buried in the native structure. Some conformational states with exposed cryptic sites resembled structures of co-translational protein folding intermediates formed on the ribosome, indicating an interplay between protein biogenesis, folding, modifications, and signalling. We propose more broadly that protein conformational fluctuations may serve as structure-specific regulatory mechanisms by kinetically burying or exposing the unmodified, and in some cases even modified, sidechains, thereby expanding the functional versatility of proteins and enabling signal transduction across diverse timescales.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.25.661537v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multi-omics guided pathway and network analysis of clinical metabolomics and proteomics data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Schmidt, C., Naake, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> This chapter provides a guide to performing metabolomics data analysis in *R*, focusing on enrichment analysis and network-based approaches, as well as enrichment analysis using prior knowledge.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Metabolomics, the study of small molecules in biological systems, is a powerful tool for understanding biochemical pathways, discovering biomarkers, and elucidating disease mechanisms. This chapter provides a guide to performing metabolomics data analysis in *R*, focusing on enrichment analysis and network-based approaches. It covers essential steps in data processing, quality control (QC), differential expression analysis, integration with proteomics using multi-omics factor analysis (MOFA) and statistical network analysis, as well as enrichment analysis using prior knowledge. The methods outlined provide a framework for biomarker discovery and advancing systems-level understanding of disease processes using metabolomics data in combination with prior knowledge and proteomics data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.26.661095v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Task-Specific Transfer Learning Approach to Enhancing Small Molecule Retention Time Prediction with Limited Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hong, Y., Tang, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a task-specific transfer learning (TSTL) strategy that pre-trains multiple models on distinct large-scale datasets, optimizing each for fine-tuned performance on the specific target task, then integrates them into a single model.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Liquid chromatography (LC) is an essential technique for separating and identifying compounds in complex mixtures across various scientific fields. In LC, retention time (RT) is a crucial property for identifying small molecules, and its prediction has been extensively researched over recent decades. The wide array of columns and experimental conditions necessary for effectively separating diverse compounds presents a challenge. Consequently, advanced deep learning for retention time prediction in real-world scenarios is often hampered by limited training data that spans these varied experimental setups. While transfer learning (TL) can leverage knowledge from upstream datasets, it may not always provide an optimal initial point for specific downstream tasks. We consider six challenging benchmark datasets from different LC systems and experimental conditions (100-300 compounds each) where TL from RT datasets under standard condition fails to achieve satisfactory accuracy (R2 [‚â•] 0.8), highlighting the need for more sophisticated TL strategies that can effectively adapt to the unique characteristics of target chromatographic systems under specific experimental conditions. We present a task-specific transfer learning (TSTL) strategy that pre-trains multiple models on distinct large-scale datasets, optimizing each for fine-tuned performance on the specific target task, then integrates them into a single model. Evaluated on five deep neural network architectures across these six datasets through 5-fold cross-validation, TSTL demonstrated significant performance improvements with the average R2 increasing from 0.587 to 0.825. Furthermore, TSTL consistently outperformed conventional TL across various sizes of training datasets, demonstrating superior data efficiency for RT prediction under various experimental conditions using limited training data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.26.661631v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">On the use of generative models for evolutionary inference of malaria vectors from genomic data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Eneli, A. A., Siu, P. C., Perez, M., Burt, A., Fumagalli, M., Mathieson, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We first quantify the genetic structure of mosquito populations in sub-Saharan Africa using unsupervised machine learning and apply an innovative generative deep learning algorithm to infer the joint evolutionary history of populations sampled in Guinea and Burkina Faso, West Africa.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Malaria in sub-Saharan Africa is transmitted by mosquitoes, in particular the Anopheles gambiae complex. Efforts to control the spread of malaria have often focused on these vectors, but relatively little is known about the relationships between populations and species in the Anopheles complex. Here, we first quantify the genetic structure of mosquito populations in sub-Saharan Africa using unsupervised machine learning. We then adapt and apply an innovative generative deep learning algorithm to infer the joint evolutionary history of populations sampled in Guinea and Burkina Faso, West Africa. We further develop a novel model selection approach and discover that an evolutionary model with migration fits this pair of populations better than a model without post-split migration. For the migration model, we find that our method outperforms earlier work based on summary statistics, especially in capturing population genetic differentiation. These findings demonstrate that machine learning and generative models are a valuable direction for future understanding of the evolution of malaria vectors, including the joint inference of demography and natural selection. Understanding changes in population size, migration patterns, and adaptation in hosts, vectors, and pathogens will assist malaria control interventions, with the ultimate goal of predicting nuanced outcomes from insecticide resistance to population collapse.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.26.661760v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Advancing in silico drug design with Bayesian refinement of AlphaFold models</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sen, S., Hoff, S. E., Morozova, T. I., Schnapka, V., Bonomi, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce bAIes, an integrative method that bridges the gap between physics-based force fields with data-driven predictions through Bayesian inference, and enhances the structural accuracy of predicted models in structure-based drug discovery.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Virtual screening has become an indispensable tool in modern structure-based drug discovery, enabling the identification of candidate molecules by computationally evaluating their potential to bind target proteins. The accuracy of such screenings critically depends on the quality of the target structures employed. Recent advances in protein structure prediction, particularly AlphaFold2, have revolutionized this field with unprecedented accuracy. However, AlphaFold2 models often exhibit limitations in local structural details, especially within binding pockets, which limit their utility for small molecule docking. In contrast, molecular dynamics (MD) simulations with accurate atomistic force fields can refine protein structures, but lack the ability to leverage the structural information provided by deep learning approaches. Here, we introduce bAIes, an integrative method that bridges this gap by combining physics-based force fields with data-driven predictions through Bayesian inference. bAIes enhances the structural accuracy of predicted models in regions most critical for ligand recognition. Crucially, bAIes demonstrates a superior ability to discriminate between binders and non-binders in virtual screening campaigns, outperforming both AlphaFold2 and MD-refined models. By enhancing the usability of AlphaFold2 models without requiring extensive experimental or computational resources, bAIes offers a convenient solution to a longstanding challenge in structure-based drug design, potentially accelerating the early phases of drug discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.25.661454v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Papaver S-determinants trigger an integrated network of mitochondrially derived ROS and disruption of energy metabolism in incompatible pollen tubes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, L., Hsiao, A.-S., Carli, J., Raza, A., Lin, Z., Arnaud, D., Davies, J., Franklin-Tong, V. E., Smirnoff, N., Bosch, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that mitochondrial disruption is an early and central event in the SI-response, likely driven by positive feedback loops involving altered Ca2+ signalling, cytosolic pH and ROS.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Many plants use self-incompatibility (SI) mechanisms to prevent inbreeding by inhibiting pollen tube growth in self pistils. In Papaver rhoeas, SI is triggered by allele-specific interaction between the pollen and pistil S-determinants, activating a Ca2+-dependent signalling network that leads to rapid reactive oxygen species (ROS) production and eventual programmed cell death (PCD) in incompatible pollen tubes. Expression of the Papaver pollen S-determinant (PrpS) in Arabidopsis thaliana recapitulates Papaver SI when challenged with the cognate pistil ligand (PrsS). Using roGFP2-Orp1, a genetically encoded hydrogen peroxide (H2O2) sensor, combined with measurements of mitochondrial metabolism we have revealed a complex SI response. Within minutes, elevated cytosolic Ca2+ ([Ca2+]cyt) and cytosolic acidification converge to trigger mitochondrial H2O2 production (with consequent cytosolic and plastid H2O2 increase), membrane depolarisation, decreased respiration rate, and ATP depletion. In parallel, oxidative inactivation of cytosolic GAPDH inhibits glycolytic flux, resulting in decreased TCA cycle intermediates and providing a feedback loop to enhance mitochondrial disruption. Preceding production of mitochondrially-sourced ROS, we show that SI rapidly arrests pollen tube growth via inactivation of plasma membrane localised NADPH oxidase (RBOH) mediated superoxide production. Our findings provide new insights into how different ROS signatures from NADPH oxidase and mitochondria drive distinct processes. We demonstrate that mitochondrial disruption is an early and central event in the SI-response, likely driven by positive feedback loops involving altered Ca2+ signalling, cytosolic pH and ROS. Our findings provide novel insights into how SI rapidly disrupts energy metabolism in pollen tubes through interconnected signalling networks centred on mitochondria prior to PCD.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.26.661469v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A new molecular seed assay to predict Ustilago nuda field infection levels</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Panzetti, C., Jenny, E., BaÃànziger, I., KaÃàgi, A., Vogelgsang, S., Hebeisen, T., BuÃàttner, P., Croll, D., Widmer, F., Sullam, K. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a multiplex qPCR method targeting U. nuda and H. vulgare DNA for the detection of Ustilago nuda seed infection in barley and found that it is better correlated with field infections than the number of infected embryos.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Seed health tests are performed to prevent sowing untreated seeds with problematic pathogen levels. The detection of internal seedborne pathogens like Ustilago nuda, causing loose smut in barley (Hordeum vulgare), is challenging because symptoms only appear when teliospores replace barley inflorescences and smutted ears develop. Currently, U. nuda seed infection levels are determined from the visual inspections of extracted embryos or fields of plants used for seed production, both of which are laborious and can be unreliable. To improve U. nuda detection, we developed a multiplex qPCR method targeting U. nuda and H. vulgare DNA. Naturally infected seed lots were tested using this qPCR method and the visual analysis of embryos. We grew the same seed lots in the field over two seasons and used the observed smutted ears as the reference infection level. The qPCR results, U. nuda DNA normalized to H. vulgare DNA, better correlated with observed field infections than the number of infected embryos. Our qPCR method more accurately distinguished seed lots with infections above and below the field tolerance threshold. It offers a reliable alternative to the visual analysis of extracted embryos. The integration of our qPCR method with field observations can enhance U. nuda management and reduce unnecessary prophylactic seed treatments, thereby bolstering integrated pest management strategies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.25.661478v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Inactivating a Œ≤1 subunit of SnRK1 promotes broad-spectrum disease resistance in rice</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yuan, G., Lu, X., Wang, X., Li, M., Wang, S., Huang, Z., Zhang, F., Zhang, X., Yang, J., Guo, H., Bhadauria, V., Zhu, W., Zhao, W., Peng, Y.-L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that a subunit of SnRK1, SnRk1{beta}1A, confers broad-spectrum susceptibility to multiple fungal diseases, including blast disease caused by Magnaporthe oryzae, sheath blight by Rhizoctonia solani,</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Rice is a staple crop for over half of the worlds population, and its sustainable production is vital to ensure global food security. However, rice is highly susceptible to several dev-astating fungal diseases, including blast disease caused by Magnaporthe oryzae, sheath blight by Rhizoctonia solani, false smut by Ustilaginoidea virens, brown spot by Bipolaris oryzae, bakanae by Fusarium fujikuroi, and head blight by F. graminearum. The mechanisms underlying the susceptibility of rice to these fungal diseases remain unclear. Here, we report that the {beta} subunit of SnRK1, SnRK1{beta}1A, confers broad-spectrum susceptibility to these fungal diseases. Our findings reveal that diverse rice fungal pathogens have convergently evolved an effector, Gas2, which interacts with SnRK1{beta}1A to prevent its ubiquitination-mediated degradation and promotes its nuclear translocation. While SnRK1{beta}1A is expressed at low levels in healthy plants, its expression is markedly induced upon fungal infections, facilitating susceptibility by inhibiting SnRK11, the subunit of SnRK1, whose overexpression has been shown to enhance broad-spectrum resistance in rice. Notably, rice lines with disrupted SnRK1{beta}1A are resistant to multiple fungal diseases without compromising growth and yield in the field. Together, this study demonstrates that broad-spectrum disease resistance in crops can be achieved by disrupting inducible susceptibility genes whose encoded proteins are targeted by effectors conserved in multiple pathogens.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.26.661071v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Simulation and experimental study on the strategy of soil auxiliary heating in a solar greenhouse in northeast China</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liu, X., Wang, C., Wang, T., Zhang, F., Xu, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A soil temperature prediction model for solar greenhouses in north eastern China .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> When solar greenhouses in north eastern China experience extreme weather, the internal soil requires varying auxiliary heating to ensure the normal growth of crops. In this study, considering the effects of four kinds of extreme weather on solar greenhouses, four different types of heating requirements were obtained through experiences in a Liaoshen-type solar greenhouse. Four distinct heating strategies were simulated using a soil temperature prediction model. The initial temperatures for these strategies were 13{degrees}C, 11{degrees}C, 13{degrees}C, and 14{degrees}C, respectively, while the target temperature was uniformly set at 15{degrees}C. The heating durations for the respective strategies were 432,000 seconds, 432,000 seconds, 864,000 seconds, and 1,728,000 seconds. To ensure a minimum soil temperature of 15{degrees}C within the solar greenhouse, the temperature of the heating layer located 400 mm underground needed to be maintained at or above 20{degrees}C, 28{degrees}C, 18{degrees}C, and 19{degrees}C, respectively. The simulation results were validated through experimental verification, with maximum and minimum errors of 3.1% and 0.5%, respectively. Several targeted promotional suggestions were also proposed. The model exhibited high accuracy and is capable of developing optimal and precise soil heating strategies for various types of solar greenhouses in north eastern China.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.25.661648v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Optimization of regulatory DNA with active learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shen, Y., Kudla, G., Oyarzun, D. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We explore the use of active learning as a strategy to improve expression levels through iterative rounds of measurements, model training, and sequence sampling-and-selection, with potential for leveraging data across laboratories, strains or growth conditions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Many biotechnology applications rely on microbial strains engineered to express heterologous proteins at maximal yield. A common strategy for improving protein output is to design expression systems with optimized regulatory DNA elements. Recent advances in high-throughput experimentation have enabled the use of machine learning predictors in tandem with sequence optimizers to find regulatory sequences with improved phenotypes. Yet the narrow coverage of training data, limited model generalization, and highly nonconvex nature of genotype-phenotype landscapes can limit the use of traditional sequence optimization algorithms. Here, we explore the use of active learning as a strategy to improve expression levels through iterative rounds of measurements, model training, and sequence sampling-and-selection. We explore convergence and performance of the active learning loop using synthetic data and an experimentally characterized genotype-phenotype landscape of yeast promoter sequences. Our results show that active learning can outperform one-shot optimization approaches in complex landscapes with a high degree of epistasis. We demonstrate the ability of active learning to effectively optimize sequences using datasets from different experimental conditions, with potential for leveraging data across laboratories, strains or growth conditions. Our findings highlight active learning as an effective framework for DNA sequence design, offering a powerful strategy for phenotype optimization in biotechnology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.27.661924v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Detecting and Quantifying Networks of Biological Kinship via Exponential Family Random Graph Models</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rohrlach, A. B., Gnecchi-Ruscone, G. A., Hofmanova, Z., Roughan, M., Haak, W., Tuke, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We use Exponential Random Graph models as a method to explore demographic and contextual parameters that may help to explain the significant drivers of the topology of mating networks, as well as to quantify their effects.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genetic relatedness between ancient humans can help to identify close and distant connections between groups and populations, uncovering signatures of demographic histories such as identifying mating networks or long-range migration. Critical to researchers are the characteristics that connected individuals, or groups of individuals, share, and how these characteristics interact and are correlated. Here we use Exponential Random Graph models as a method to explore demographic and contextual parameters that may help to explain the significant drivers of the topology of mating networks, as well as to quantify their effects. We show through simulations that model selection and coefficient estimators facilitate the exploration of such networks, and apply the method to individuals from a collection of Avar-associated cemeteries from the Carpathian Basin dating to the 6th to the 9th centuries CE.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.25.661441v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">EVscope: A Comprehensive Bioinformatics Pipeline for Accurate and Robust Analysis of Total RNA Sequencing from Extracellular Vesicles</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhao, Y., Chintalapudi, H., Xu, Z., Liu, W., Hu, Y., Grassin, E., Song, M., Hong, S., Lee, L. P., Dong, X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present an open-source bioinformatics pipeline designed specifically for processing EV RNA-seq datasets.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: Extracellular vesicle (EV) RNA sequencing has emerged as a powerful approach for studying RNA biomarkers and intercellular communication. Nevertheless, the extremely low abundance, fragmented nature and ubiquitous tissue origin of EV RNAs, alongside potential contamination from co-isolated materials, such as free DNA and bacterial RNA, pose substantial analytical challenges. These complexities highlight a pressing need for a standardized, computational workflow that ensures robust quality control and EV RNA characterization. Results: Here, we present EVscope, an open-source bioinformatics pipeline designed specifically for processing EV RNA-seq datasets. EVscope employs an optimized genome-wide expectation-maximization (EM) algorithm that significantly improves multi-mapping read assignment at single-base resolution by effectively leveraging alignment scores (AS) and local read coverage, specifically tailored for fragmented and low-abundance EV RNAs. Notably, EVscope uniquely generates EM-based BigWig files for downstream analysis, a capability currently unavailable in existing EM-based BigWig quantification tools. The pipeline systematically integrates 27 major steps, including quality control, analysis of library structure, contamination assessment, read alignment, read strandedness detection, UMI-based deduplication, RNA quantification, genomic DNA (gDNA) contamination correction, cellular and tissue source inference and visualization with a comprehensive HTML report. EVscope incorporates a comprehensive, updated annotation covering 19 distinct RNA biotypes, encompassing protein-coding genes, lncRNAs, miRNAs, piRNAs, retrotransposons (LINEs, SINEs, ERVs), and additional non-coding RNAs (tRNAs, rRNAs, snoRNAs). Furthermore, it leverages two highly balanced circRNA detection algorithms for robust circular RNA identification. Notably, a downstream module enables the inference of the tissue/cellular origins of EV RNAs using bulk and single-cell RNA-seq reference datasets. EVscope is implemented as a convenient, single-command Bash pipeline leveraging Conda-managed standard software packages and custom scripts, ensuring reproducibility and straightforward deployment. Availability and implementation: Code, documentation, and tutorials are available at GitHub (https://github.com/TheDongLab/EVscope) and archived on Zenodo (https://zenodo.org/records/15577789).</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.24.660984v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Accelerating Virtual Directed Evolution of Proteins via Reinforcement Learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mi, T., Wang, Y., Zhao, J., Wang, W., Shen, Y., Xiao, N., Chen, L., Chen, G.-Q., Zhang, S., Zhang, W.-B., Gong, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a virtual directed evolution pipeline, RelaVDEP, for the rapid identification of mutational variants with explicit enhancement in the desired property of the target protein.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> With the advancement of machine learning methods, the protein fitness landscape can be predicted, providing reliable guidance in the selection of advantageous mutations for the directed evolution of proteins. However, the potential multiple mutational variants derived from the simple combination of a limited number of advantageous single mutations may not represent superior choices. Moreover, the exploration and selection of the astronomical number of multiple mutational variants remain a highly challenging task. In this study, we introduce a virtual directed evolution pipeline, RelaVDEP, for the rapid identification of mutational variants with explicit enhancement in the desired property of the target protein. By adapting and fine-tuning a pre-trained fitness predictor to improve sequence-based protein functional prediction and by designing a model-based reinforcement learning framework to efficiently explore the vast combinatorial space of protein mutations, this pipeline is capable of effectively accelerating the directed evolution process for a broad spectrum of proteins with versatile functional profiles. According to a series of experimental validations, the diversified mutational variants identified by our method exhibit notable improvements in desirable protein functional properties. In particular, by integrating RelaVDEP with active learning, we successfully optimized the sequence of a PETase, enhancing its catalytic activity through previously unknown mutations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.25.661516v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

