<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Saturday, 20 September 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Federated Learning for the pathogenicity annotation of genetic variants in multi-site clinical settings.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nigreisy Montalvo, Francisco Requena, Emidio Capriotti, Antonio Rausell</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new approach for the pathogenicity scoring of human genetic variants .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Rare diseases collectively affect 5% of the population. However, fewer than 50% of rare disease patients receive a molecular diagnosis after whole genome sequencing. Supervised machine Learning is a valuable approach for the pathogenicity scoring of human genetic variants. However, existing methods are often trained on curated but limited central repositories, resulting in poor accuracy when tested on external cohorts. Yet, large collections of variants generated at hospitals and research institutions remain inaccessible to machine-learning purposes because of privacy and legal constraints. Federated learning (FL) algorithms have been recently developed enabling institutions to collaboratively train models without sharing their local datasets.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40973651/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Phylogenetic Methods Meet Deep Learning.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Svitlana Braichenko, Rui Borges, Carolin Kosiol</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A short perspective on the application of deep learning in phylogenetics, introducing prevalent DL architectures and highlighting potential problems in the field, including reproducibility and robustness in computational estimates.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Deep learning (DL) has been widely used in various scientific fields, but its integration into phylogenetics has been slower, primarily due to the complex nature of phylogenetic data. The studies that apply DL to sequencing data often limit analyses to 4-taxon trees. Many of these studies serve as "proof of principle" and perform similarly to traditional phylogeny reconstruction methods. New ways of using training data, such as encoding with compact bijective ladderized vectors or transformers, enable the handling of much larger trees and genomic data sets. This short perspective focuses on the application of DL in phylogenetics, introducing prevalent DL architectures. We highlight potential problems in the field by discussing the risks of using simulation-based training data and emphasize the importance of reproducibility and robustness in computational estimates. Finally, we explore promising research areas, including the combination of phylogenetics and population genetics in DL, the analysis of neighbor dependencies, and the potential to significantly reduce computational cost compared to traditional methods. This perspective illustrates the potential of DL in complementing traditional phylogeny reconstruction methods and aiding the advancement of phylogenetic analysis, especially in performing computationally demanding tasks such as model selection or estimating branch support values.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40973626/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Drug and Clinical Candidate Drug Data in ChEMBL.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fiona M I Hunter, Harris Ioannidis, A Patrícia Bento, Nicolas Bosc, Sybilla Corbett, Eloy Felix, M Paula Magarinos, Emma Manners, Ines A Smit, Marleen de Veij, Noel M O'Boyle, Barbara Zdrazil, Andrew R Leach</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> This paper is a reference guide to present the principles of why the ChEMBL drug data has been curated in a particular manner so that data users can better understand the nature of the data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> ChEMBL is a large-scale, open-access, FAIR database of bioactive molecules with drug-like properties. ChEMBL 35 contains 17,500 approved drugs, and drugs that are progressing through the clinical development pipeline. Drug curation has formed an integral part of the core offering of the ChEMBL database since its inception. The paper is a reference guide to present the principles of why the ChEMBL drug data has been curated in a particular manner so that data users can better understand the nature of the data. The drug data include information on: names, synonyms and trade names, chemical structure or biological sequence, data sources, indications, mechanisms, warnings and drug properties such as maximum phase of development, type of molecule, prodrug status and first approval. The integrated nature of the drug data within the context of a bioactivity resource enables the wide use of the data set in drug discovery, AI and machine learning.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40971497/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">LieMap: A Fast and Transparent Gradient-Based Method for Cryo-EM Structure Fitting</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hu, Y., Cao, Z., Liu, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present LieMap, a straightforward, fast, and transparent gradient-based fitting protocol designed for practical applications. We demonstrate the effectiveness of LieMap by fitting the apo-state structure of E. coli GroEL (PDB: 1aon) into its ATP-bound state cryo</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Fitting high-resolution atomic models into lower-resolution cryo-electron microscopy (cryo-EM) maps is a critical yet challenging step in structural biology. We present LieMap, a straightforward, fast, and transparent gradient-based fitting protocol designed for practical applications. Our method parameterizes rigid-body motion on the Lie algebra se(3) to provide a minimal and singularity-free representation. The optimization objective is to directly maximize a real-space cross-correlation score, calculated as the mean density value sampled from the map at the atomic coordinates. This score is computed differentially using trilinear interpolation and optimized with the Adam optimizer in a multi-stage fashion. We demonstrate the effectiveness of LieMap by fitting the apo-state structure of E. coli GroEL (PDB: 1aon) into its ATP-bound state cryo-EM map (EMD-1046). The protocol successfully navigates the large conformational change, achieving a inal RMSD of 3.23 Angstrom. LieMap's simplicity, computational efficiency, and transparent optimization process make it a valuable and practical tool for structural biologists.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.16.676663v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genome size and nucleotide skews as predictors of bacterial growth rate</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sahu, P., Barik, S., Ghosh, K., Subramanian, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that genome size, replichore organization, and nucleotide compositional asymmetry are reasonable predictors of bacterial doubling times, and propose a potential adaptive role of nucleotide skew in genome replication.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Bacterial growth rates are constrained by genome replication, yet the role of replication kinetics in bacterial growth rates remains incompletely understood. Here, we examine if genome size, replichore organization, and nucleotide compositional asymmetry are reasonable predictors of bacterial doubling times. In free-living bacteria, both genome size and the length of the longest replichore are found to correlate positively with doubling time, pointing to an influence of replication dynamics on bacterial growth rates. Moreover, fast-growing bacteria are shown to exhibit stronger nucleotide compositional skew. Incorporating skew into the model substantially improves predictive accuracy, suggesting that compositional asymmetry in genomes may facilitate replication fork progression and thereby enhance growth rates. These findings point to a potential adaptive role of nucleotide skew in bacterial genome replication.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.17.676822v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Gene-Family Encoding Boosts Domain-Adapted Single-Cell Language Models</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ma, H., Xu, C., Ho, S. W. T., Zhao, J. J., Chu, Y., Tan, A., Sundar, R., Tan, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce gene-family encoding, where expressed genes are grouped into functionally defined families, and ranking performed within each family. GF models outperformed RG models across both tasks and backbones.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Transformer-based single-cell foundation models often rely on ranked-gene (RG) sequences where genes, ranked by expression, are often not functionally related, weakening next-token learning and the structure of learned embeddings. Here, we introduce gene-family (GF) encoding, where expressed genes are grouped into functionally defined families, and ranking performed within each family. Using 100,000 gastric-cancer (GC) cells, we domain-adapted 8-billion-parameter Llama and Qwen backbones with either RG or GF sentences and benchmarked zero-shot performance on embedding-based and generative tasks. GF models outperformed RG models across both tasks and backbones. We scaled GF-Llama to 1.3 million GC cells to obtain GF-Llama-GC and applied it to two applications: resolving fine-grained cellular heterogeneity and discovering cell populations associated with disease progression. GF-Llama-GC revealed immune-cell subclusters not resolved by standard expression-based analyses. Applying in-silico cell removal/transplantation on a chemotherapy responder/non-responder scRNA-seq dataset, GF-Llama-GC highlighted not only epithelial cells but also neutrophils as key cells associated with chemotherapy response.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.16.676672v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">RobPicker: A Meta Learning Framework for Robust Identification of Macromolecules in Cryo-Electron Tomograms</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hosseini, R., Liang, Y., Singh, D., Rahmani, H., Choe, S., Lee, J., Xu, M., Segal, E., Zou, J., Williamson, J., Grotjahn, D. A., Villa, E., Xie, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed RobPicker, a meta-learning framework that effectively mitigates these issues by automatically learning deep neural networks to correct label errors and give greater emphasis to underrepresented macromolecule species.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurate particle picking of macromolecules of all kinds, shapes, and sizes in cryogenic electron tomograms (cryo-ET) is critical for understanding the molecular architecture of biological systems in their native state. Current deep learning methods have shown potential in identifying macromolecules from tomograms, but they are vulnerable to issues of noise in the training dataset obtained through human or automatic labeling and imbalanced distribution of macromolecule species. To address these limitations, we developed RobPicker, a meta-learning framework that effectively mitigates these issues by automatically learning deep neural networks to correct label errors and give greater emphasis to underrepresented macromolecule species. In evaluations across diverse cryo-ET datasets with noisy labels and imbalanced species distributions, RobPicker substantially outperforms state-of-the-art methods, particularly in identifying small and rare macromolecules. The efficiency and robustness of RobPicker can also be used for rapid fine-tuning of the tilt-series alignment, leading to improved tomogram reconstruction and enabling high-resolution cellular structural biology analysis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.16.676650v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Predicting RNA:DNA Triplex Structures from Sequence Features Using Deep Learning Architectures</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tsenum, J. L., Poptsova, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Deep neural networks can accurately predict RNA:DNA triplex formation potential, providing a scalable in silico framework for genome-wide triplex biology.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Long non-coding RNAs (lncRNAs) can perform their regulatory roles by forming triple helices through RNA:DNA interactions. Although this has been verified by a few in vivo and in vitro methods, robust in silico approaches that predict the potential of lncRNAs and DNA sites to form triplex structures are still required. Tools such as Triplexator have predicted vast numbers of lncRNAs and DNA sites with triplex forming potential, yet there remains a pressing need for advanced computational methods that can refine and extend these predictions. In this study, we developed ten (10) deep neural network models that predict the potential of lncRNAs and DNA sites to form triple helices on a genome-wide scale. To prepare our dataset, we first used Triplexator to screen out lncRNAs and DNA sites with low triplex-forming potential. We then trained different deep learning architectures, including two-layer convolutional neural networks (CNN), residual neural networks (ResNN), long short-term memory recurrent neural networks (LSTM-RNN), and multilayer perceptron (MLP). Among these architectures, our lncRNA_CNN and LSTM3-RNN both achieved a mean AUC of 0.99 for lncRNA features at a kernel size of 32 and a learning rate of 1e-3. For DNA site features, our DNA_CNN achieved the best performance with a mean AUC of 0.98 under the same conditions. In conclusion, we demonstrate that deep neural network architectures can effectively learn sequence features of lncRNAs and DNA to accurately predict RNA:DNA triplex formation potential, providing a scalable in silico framework for studying genome-wide triplex biology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.16.676176v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Molecular Contrastive Learning with Graph Attention Network (MoCL-GAT) for Enhanced Molecular Representation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dalkiran, A., Rifaioglu, A. S., Cetin-Atalay, R., Acar, A. C., Dogan, T., Atalay, V. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce MoCL-GAT, a novel contrastive and transfer learning-based SSL framework that addresses this gap by simultaneously learning from two complementary objectives by combining a local contrastive task on molecular subgraphs with a global predictive task to learn holistic molecular descriptors.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Learning the representation of molecules is crucial for drug discovery but is often hindered by the scarcity of labeled experimental data, which limits the performance of supervised machine learning models. While self-supervised learning (SSL) offers a solution by leveraging vast unlabeled chemical databases, many existing methods focus on learning from either local structural information or global molecular properties, but not both simultaneously. We introduce MoCL-GAT, a novel contrastive and transfer learning-based SSL framework that addresses this gap by simultaneously learning from two complementary objectives. It combines a local contrastive task on molecular subgraphs to capture fine-grained chemical environments with a global predictive task to learn holistic molecular descriptors. This dual-objective approach, powered by a Graph Attention Network, is designed to create more robust, versatile, and transferable molecular representations. Pre-trained on 1.9 million compounds, MoCL-GAT was fine-tuned on diverse benchmarks. It achieved state-of-the-art performance on molecular property prediction tasks, with an AUROC of 0.928 on BBBP and 0.749 on SIDER, and top-ranking RMSEs of 0.570 for ESOL and 1.818 for FreeSolv. Critically, fine-tuned models consistently and significantly outperformed models trained from scratch, confirming the value of pre-training. These results validate that MoCL-GAT's dual-objective approach learns highly effective and transferable representations, enabling more accurate and data-efficient predictions for key cheminformatics challenges.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.17.676724v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Chromoanagenesis is a driver of structural variationin the smallest photosynthetic eukaryote</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bugnot, C., Alioto, T., Cruz, F., Gomez-Garrido, J., Gut, M., Yau, S., Piganeau, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that resistance to OmV2 is not directly linked to any specific structural variant or single nucleotide polymorphism, but instead, the high rate of localized genomic structural variations points to a distinct mechanism of chromosome evolution.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Marine microalgae populations can rapidly evolve resistance to viruses upon infection. In Ostreococcus mediterraneus resistance to the prasinovirus OmV2 emerged within five days in all virus-exposed populations. Whole-genome sequencing of pairs of resistant and susceptible cell lines revealed extensive structural genomic changes, particularly on the Small Outlier Chromosome (SOC). SOC alterations included large deletions, duplications, rearrangements, and whole chromosome duplication, yet no consistent structural variant or single nucleotide polymorphism could be directly associated with resistance. Hybrid de novo assemblies confirmed the unique SOC assembly of each strain, with a highly polymorphic ~2 kb tandem repeat region exhibiting an "accordion-like" pattern of expansion and contraction. No new viral insertions were found, though endogenous viral elements were conserved across lines. Two interchromosomal translocations between the SOC and chromosomes 2 and 17 provide evidence for chromoplexy, thereby offering novel insights into the mechanisms underlying the distinctive evolutionary path of this chromosome. Together, these findings demonstrate that resistance to OmV2 evolves rapidly and consistently but cannot yet be linked to any specific structural variations; instead, the high rate of localized genomic structural variations points to a distinct mechanism of chromosome evolution.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.17.676765v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">RIPPLET: Mutation-Only Gene and Pathway Profiling for Precision Oncology</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wiesmann, F., Litchfield, C., Mayoux, M., Tumor Profiler Consortium,, Wicki, A., Becher, B., Koelzer, V. H., Moch, H., Buljan, M., Tugues, S., Sobottka, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce RIPPLET, a DNA-only framework converting somatic variants into quantitative gene-impact scores and topology-aware pathway-perturbation profiles.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Clinical implementation of comprehensive genomic profiling, via whole-genome (WGS) or whole-exome sequencing (WES), is constrained by sparse mutation burdens and analytic pipelines reliant on matched transcriptomes. Currently, gene-centric analysis prevails, but overlooks the complex, multigene and pathway perturbations shaping tumor biology. We introduce RIPPLET, a DNA-only framework converting somatic variants into quantitative gene-impact scores and topology-aware pathway-perturbation profiles. By integrating tissue-specific protein-protein interaction networks with cohort-informed reweighting, RIPPLET prioritizes likely functionally relevant alterations. Applied across 33 TCGA cancer types, RIPPLET surpasses four state-of-the-art multi-omic driver-prioritization tools in recovering cancer type-specific drivers. In a cohort of metastatic cutaneous melanomas, it identifies pathway signatures that predict drug response, provide prognostic insight and distinguish immune-infiltration phenotypes without RNA data, independently validated on an in-house cohort. RIPPLET enables DNA-only inference of tumor-specific gene and pathway dysregulation, aligning with clinical sequencing workflows and offering a scalable precision-oncology strategy in transcriptome-limited settings.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.17.676675v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Investigating the native functions of -carbon monoxide dehydrogenase through genomic context analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Land, H., Böhm, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A molecular framework for predicting CODH phenotypes and biochemical roles .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Carbon monoxide dehydrogenases containing nickel-iron active sites ([NiFe]-CODHs) catalyze the reversible oxidation of CO to CO2, representing key targets for biocatalytic CO2 reduction. Despite dramatic differences in catalytic rates and O2 tolerance between CODH variants, the molecular basis for this functional diversity remains poorly understood. We applied comparative genomics and synteny analysis to investigate the biochemical roles of CODH clades A-F using 1,376 CODH and 1,545 hybrid cluster protein sequences. Around ~30% of genomes encode multiple CODH isoforms. Analysis revealed distinct gene clustering patterns correlating with biochemical function. Clades A, E, and F exhibit a degree of distributional exclusivity. Clades C and D frequently co-occur with active CODHs, suggesting auxiliary roles. Operon architecture analysis revealed functional specialization: clade A links to acetyl-CoA synthase; clades A, E, F contain essential maturation machinery (CooC, CooJ, CooT) correlating with catalytic activity; clade B associates with transporters; clade C with electron transfer partners; clade D with transcriptional regulators. High CODH-HCP co-occurrence (except clade A) suggests environmental interdependency. These findings establish clades A, E, F as primary biocatalyst targets while defining regulatory functions for clades C, D, providing a genomics framework for predicting CODH phenotypes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.14.676152v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A surface morphology-based inference method for the cell wall elasticity profile in tip-growing cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xu, R., Vidali, L., Wu, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a new surface morphology-based method to measure the elasticity of the cell wall in tip-growing cells.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Plant cell morphology and growth are essential for plant development and adaptation, in particular the cell wall material deposition and rearrangement. As the wall is extended due to turgor pressure, the wall mechanical response, specifically the walls elastic properties, are not well understood. In this study, we introduce a new surface morphology-based method to measure the elasticity of the cell wall in tip-growing cells. Previous work is based on the wall meridional outline on the assumption the cell is axisymmetric, an idea that does not align well with the reality of the cells. Instead, we developed a way to triangulate the surface of the cell using marker point locations that could be achieved experimentally with fluorescent labeling. After measuring the elastic stretch and curvatures from our triangulation, a mechanical model is used to measure the tensions and surface bulk modulus distribution. We simulated moss tip cells from experimental Physcomitrium patens plants to challenge the method with noise estimated from experiments. We discovered that increasing the discretization of the triangulation improved robustness against noise. With multiple cell sampling, we found that around 10 cells were sufficient to recover the elasticity distribution with noise, but only when the elastic stretches were high enough at the tip. With low elastic stretch at the tip, the appearance of non-physical modulus values led to underestimated inference values there, causing false-positive gradient results. This compelled us to study a range of cases, leading us to a dimensionless global phase map showing areas of good inference. This technique will open the field to more comprehensive measurements of tip cell elasticity, providing a key step in understanding tip cell growth and morphogenesis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.17.676748v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Finding stable clusterings of single-cell RNA-seq data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Klebanoff, V. F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A sampling-based method that can identify stable (replicable) clusterings of cells for data presented as UMI counts is described.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> A sampling-based method that can identify stable (replicable) clusterings of cells for data presented as UMI counts is described. The structure of the processing pipeline is conventional: filter and transform counts, restrict to data for highly variable genes, reduce dimensionality, and cluster cells. Divisive (binary) hierarchical spectral clustering is used. We propose what may be a novel method to map a clustering tree to a set of nested clusterings. For spectral clustering, non-zero affinities are defined for points that are k-nearest neighbors (k is an input parameter). The affinity equals the inverse of the distance between the points. This led to exploration of the variation of the distance between points (that represent cells in low-dimensional Euclidean space) that are k-nearest neighbors. Variation can be large, ranging over three orders of magnitude for one data set studied. This may have implications for other clustering schemes. Given a set of points, Ng, Jordan, and Weiss' algorithm is used to divide it into two clusters. Repeating for each daughter cluster and its descendants generates a clustering tree. Because the algorithm splits a set of points into two subsets, the points are mapped to two-dimensional Euclidean space for clustering. The clusters' separation is measured by a quantity, H, calculated in two dimensions, which is formally identical to the F-statistic, equal to the between-cluster sum of squares divided by the within-cluster sum of squares, scaled by degrees of freedom. The larger H is, the greater the separation between the clusters. Each cluster corresponds to a node of the clustering tree. Dividing a set of points into two subsets corresponds to defining two daughter nodes. Assign the length of the branch between a node and each of its daughters to equal 1/H. That is, the larger the separation between the daughter clusters, the closer they are (viewed as nodes) to their parent node in the tree. Nodes' distances from the root define the mapping of the tree to a set of nested clusterings. Analysis is performed for all cells and for multiple pairs of complementary samples of cells. For a given number of clusters, each sample's clustering and clusters are compared to those of the full data set (restricted to the sample). If differences are small for all samples, the clustering may be considered stable. The method supports single-factor batch correction. Preliminary analysis not discussed here suggests that differential expression can contribute to evaluating stability.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.17.672302v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MOAST: Mechanism of Action Similarity Tool</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lohith, A., Terciano, D., Murray, A., MacMillan, J., Lokey, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel computational workflow that bridges traditional phenotypic screening and modern computational approaches, making MOA determination feasible for researchers with limited resources while maintaining statistical rigor and interpretability.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Determining the mechanism of action (MOA) for natural products remains a significant bottleneck in drug discovery, particularly for researchers with limited computational resources or small compound libraries. Traditional approaches require screening large numbers of annotated compounds alongside unknowns, which is cost-prohibitive, or depend on complex machine learning models that need substantial computational resources and large datasets. Here, we present a dissertation chapter excerpt: MOAST (Mechanism of Action Similarity Tool), a BLAST-inspired computational workflow that addresses these limitations by providing rapid MOA hypotheses for newly screened compounds. This chapter investigates two complementary approaches: a kernel density estimation (KDE) method providing statistical significance measures and E-values for MOA class membership, and a CatBoost machine learning classifier for multi-class prediction with ranked outputs. Using cytological profiling data from HeLa and A549 cell lines, MOAST achieved 22% accuracy for the top 5 predictions among ~300 MOA classes, with the CatBoost classifier reaching 10% balanced accuracy - significantly better than the ~3% reported in literature. The tool suggests a 0.8 prediction probability threshold for trustworthy results and demonstrates robust performance across multiple feature reduction strategies. MOAST provides a practical, accessible solution that bridges traditional phenotypic screening and modern computational approaches, making MOA determination feasible for researchers with limited resources while maintaining statistical rigor and interpretability.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.15.676411v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An Investigation of mini-GRID Radiation therapy with Immune Checkpoint Blockade in a Murine Tumor Model</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sansone, P., Rickard, A. G., Williams, N. T., Castillo, R. J., Brundage, S., Mowery, Y., Oldham, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Mini-GRID pencil-beam radiotherapy combined with anti-PD-1 immune checkpoint blockade in subcutaneous LLC1 flank tumors .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Spatially fractionated radiotherapy has shown potential to improve therapeutic outcomes possibly with an immunogenic mechanistic component. Here we report on in vivo mouse studies investigating mini-GRID pencil-beam radiotherapy combined with anti-PD-1 immune checkpoint blockade. Methods: GRID therapy was delivered at 225kV using the XStrahl Small Animal Radiation Research Platform with two custom lead mini-GRIDs, each consisting of an array of equally spaced holes: 1 mm diameter with 1mm spacing and 254 {micro}m diameter with 508 {micro}m spacing. GRID dosimetry was characterized using EBT3 film to determine peak-to-valley dose ratios and output. Two studies were performed with C57BL/6J mice bearing subcutaneous LLC1 flank tumors. In the first, mice (n=5/group) were treated in 3 groups with a single fraction: 15 Gy open field, 15 Gy 1 mm GRID, or 24 Gy 1 mm GRID. In the second, mice (n=6-7/group) were treated with fractionated GRID radiotherapy in 5 groups: 15 Gy open field x 3 fractions, 15 Gy hemi-irradiation x 3 fractions, (15 Gy 1 mm GRID x 3 fractions, or 15 Gy 254 {micro}m GRID x 3 fractions. All mice were treated with 200 g anti-PD-1 antibody on days 0, 3, and 6, then weekly until humane endpoint (tumor >15 mm in any dimension or ulceration). Results: Peak to valley ratios were 24.5 {+/-} 0.6 and 19.8 {+/-} 0.7 for the 1 mm and 254 {micro}m GRIDs, respectively. Tumor growth and mean survival times in both studies were significantly shorter for all non-open field arms (p < 0.05; Log Rank for survival; 2-way ANOVA for tumor growth). Conclusions: Two novel mini-GRIDs were characterized and tested in combination with anti-PD-1 therapy. In this study, neither single dose nor fractionated GRID therapy with anti-PD-1 improved tumor growth delay or survival. Similarly, hemi-irradiation resulted in worse tumor control compared to conventional open field radiotherapy.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.14.676104v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Clonal dynamics shaped by diverse drug-tolerant persister states in melanoma resistance</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Li, H., Chen, Y., Kaster, J., Dunne, M., Xiao, M., Li, L., Thomas, M., Promi, N., Fingerman, D., Brown, G. S., Zheng, Q., Zhu, X., Reale, M., Patterson, A., Gao, L., Zhang, X., Jiang, S., Hu, T., Fang, H., Ren, J., Qi, C., Wang, L., Mou, H., Thacker, G., Salazar, E. R., Villanueva, J., Raj, A., Hoon, D. S., Bin, T., Madzo, J., Wei, Z., Auslander, N., Herlyn, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed MeRLin, a high-resolution lineage tracing platform integrating cellular barcoding, single-cell transcriptomics, RNA fluorescence in situ hybridization (FISH), and computational analyses to track clonal and transcriptional dynamics in patient-derived melanoma models during prolonged therapy.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Most advanced melanomas initially respond to targeted therapy but eventually relapse. Rather than acquiring new mutations, resistance is driven by drug-tolerant persister cells that enter a reversible drug-refractory state. We developed MeRLin, a high-resolution lineage tracing platform integrating cellular barcoding, single-cell transcriptomics, RNA fluorescence in situ hybridization (FISH), and computational analyses to track clonal and transcriptional dynamics in patient-derived melanoma models during prolonged therapy. Clonal dynamics revealed that persister subpopulations first responded to treatment but persisted and expanded during minimal residual disease, ultimately leading to tumor recurrence. Pre-treatment melanoma populations diversified into four conserved persister states characterized by stress-like, lipid metabolism, PI3K signaling, and extracellular matrix remodeling programs associated with adaptive resistance. Spatial transcriptomics showed the organization of these adaptive programs and a complex signaling network of autocrine and paracrine interactions among persister subpopulations. Barcoded RNA-FISH enabled spatial mapping of clonal identity and gene expression, revealing in situ co-localization of a dominant resistant clone with SLC2A1 expression. MeRLin provides a robust framework for dissecting cancer heterogeneity and identifying vulnerabilities in persister populations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.16.676608v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

