<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Sunday, 05 October 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Towards functional annotation with latent protein languagemodel features</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Silberg, J., Simon, E. P., Zou, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a rapid and scalable protein annotation pipeline using SAE features that can detect missing annotations by finding proteins that display recognizable structural motifs but lack corresponding database labels.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein Language Models (PLMs) create high-dimensional embeddings that can be transformed into interpretable sparse features using Sparse Autoencoders (SAEs), where each feature activates on specific protein elements or patterns. However, scalably identifying which features are cohesive and reliable enough for protein annotation remains challenging. We address this by developing a validation pipeline combining three complementary methods: (1) expanded database matching across 20+ annotation sources including hierarchical codes, (2) feature-guided local structural alignment to identify structurally consistent activation regions, and (3) LLM-based feature description generation. Our annotation pipeline demonstrates three key properties of SAE features that make them a useful source of functional annotation complementary to existing methods. First, they can represent more granular patterns than existing protein databases, enabling the identification of sub-domains. Second, they can detect missing annotations by finding proteins that display recognizable structural motifs but lack corresponding database labels. Here, we automatically identify at least 491 missing CATH topology annotations with our pipeline. Third, they can maintain structural consistency across unseen proteins. Of our 10,240 SAE features, we find 615 that are consistently structurally similar in unannotated metagenomic proteins, allowing us to structurally match at least 8,077 metagenomic proteins to characterized proteins. This provides a rapid annotation pipeline with constant time search regardless of database size, that automatically includes structural and function information about the feature that triggered the match.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.02.680154v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">De Novo Design of Anti-Aging Peptides</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pan, Y., Fan, L., Zhang, N., Zheng, J., Jin, Z., Cai, H., Ma, H., Zhang, W., Zhu, J., Xu, W., Gong, Y., Li, R., Liang, R., Li, G., Jing, H., Ning, J., Gao, S., Xian, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present ElixirSeeker2, the first computational framework for de novo design of anti-aging peptides, and establish a strategy for the development of next-generation biologics.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Aging, a universal biological process in complex organisms, is increasingly recognized to be driven by progressive loss of epigenetic information, as proposed in the Information Theory of Aging (ITOA). However, research on anti-aging peptides remains scarce, with most existing efforts confined to derivatives of natural proteins, while systematic design attempts are virtually absent. This limitation not only restricts discovery within the evolutionary sequence space but also hampers the identification of candidates with novel mechanisms and improved efficacy. Here, we present ElixirSeeker2, the first computational framework for de novo design of anti-aging peptides. By integrating modeling of known anti-aging peptides, activity scoring from the IC50 database, and penalty constraints from toxic peptides, ElixirSeeker2 enables large-scale virtual screening and identification of novel peptide candidates. Several lead peptides demonstrated significant effects in delaying cellular senescence, restoring cellular functions in vitro and in enhancing locomotor activity of aged Caenorhabditis elegans. This study not only validates the feasibility of de novo design in anti-aging interventions but also establishes a strategy for the development of next-generation biologics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.04.680435v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Improving the benchmark of variant calling in clonal bacteria using more realistic in silico genomes, the case of Mycobacterium tuberculosis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Le Meur, A., Rodriguez de la Vega, R. C., Zein Eddine, R., Refregier, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Maketube, a method for evolving realistic genomes of the Mycobacterium tuberculosis complex with the full diversity of variants verified in natural isolates, and describe benchmarking results using MaketTube-evolved genomes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The democratisation of Whole Genome Sequencing data in bacterial genomics requires the benchmarking of associated analytical methodologies such as reference-based variant calling. Current variant calling benchmarks rely either on de novo assembled natural genomes, for which true variants are inferred using a genome aligner, or on genomes evolved in silico by incorporating short variants on reference genomes. We introduce Maketube, a method for evolving realistic genomes of the Mycobacterium tuberculosis complex with the full diversity of variants verified in natural isolates, and describe benchmarking results using Maketube-evolved genomes. We document that Maketube-evolved genomes satisfyingly mimic Mtbc complex genomes. Using Maketube-evolved genomes, we show that genome aligners miss up to 7.5% of the variants, which implies that benchmarkings with natural de novo assembled genomes are biased. Second, we show that recall of popular variant calling pipelines MTBseq, TB-Profiler, and our in-house genomic pipeline genotube, was overestimated by 1 to 10% in benchmarkings relying on simplistic in silico-evolved genomes, and that slight but significant differences in performance exist between pipelines. Finally, we provide evidence that variants are missed in duplicated regions and in regions flanking sequences absent in the reference (displaced insertion sequences or sequences deleted during the evolution of the reference). Altogether, realistic in silico-evolved genomes such as Maketube-derived ones are precious tools for reliable genomic tools benchmarking. We provide new evidence that structural variants interfere with variant-calling, both because of the additional sequences they contain, but also because of misalignments around insertions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.04.680280v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Helical Reconstruction of Amyloids in cryoSPARC</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Schaefer, J.-H., O'Neill, R. T., Donnelly, J. P., Powers, E., Kelly, J. W., Lander, G. C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present comprehensive processing guidelines for helical reconstruction of helical amyloid structures using cryoSPARC and identify key limitations in cryo-EM workflows.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Amyloid-mediated proteotoxicity underlies more than 50 human diseases. Cryo-electron microscopy (cryo-EM) analyses have yielded hundreds of in vitro and patient-derived amyloid structures, establishing direct links between filament morphologies and specific pathological conditions. Despite the growing popularity of the processing software cryoSPARC for single-particle analyses, RELION remains the dominant software platform for performing helical reconstruction of amyloid structures, highlighting an area for further development. Here, we present comprehensive processing guidelines for helical reconstruction of helical amyloids using cryoSPARC. Through systematic re-processing and validation of publicly deposited datasets, we demonstrate current capabilities and identify key limitations, emphasizing the need for amyloid-specific parameter optimization within cryoSPARC workflows. Our findings showcase a potential for developing unsupervised processing workflows to meet the demanding throughput requirements of time-resolved in vitro studies and large-scale compound screening initiatives, thereby accelerating therapeutic drug development. Ultimately, our goal is to shift the focus of amyloid cryo-EM from computationally intensive processing challenges toward addressing fundamental biological questions that enhance our capacity for treatment discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.03.680389v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AstraPTM2: A Context-Aware Transformer for Broad-Spectrum PTM Prediction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bozkurt, C., Vasilyeva, A., Goteti, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present AstraPTM2, a transformer-based model that predicts 39 distinct PTM types on full-length proteins without truncating context.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Post-translational modifications (PTMs) are covalent changes in proteins after biosynthesis that shape stability, localization, and function. While numerous computational tools exist for PTM site prediction, most struggle to handle full-length proteins without truncating context, focus on only a limited number of PTM types, and perform unevenly on rare modifications. We present AstraPTM2, a transformer-based model that predicts 39 distinct PTM types on full-length sequences. By combining ESM-2 embeddings, AlphaFold2-derived structural features, and protein-level descriptors, AstraPTM2 captures both short-range motifs and long-range dependencies. Training uses a three-stage curriculum and adaptive focal loss to balance rare and common PTMs, followed by per-label affine calibration and optimized thresholds for well-calibrated probabilities. In hold-out tests, AstraPTM2 achieves AUROC = 0.99 and macro-F1 = 59% across 39 PTM types, with particularly strong performance on rare motif-driven PTMs such as O-linked glycosylation and sumoylation. Results are available through the Orbion web platform, which offers synchronized 2D and 3D visualizations, dual prediction modes (calibrated and exploratory), and reproducible exports to support downstream experimental planning. AstraPTM2 can be accessed at https://www.orbion.life.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.03.680341v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">GECSI: Large-scale chromatin state imputation from gene expression</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fu, J., Ernst, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We develop Gene Expression-based Chromatin State Imputation (GECSI), which uses a multi-class logistic regression model trained using a large compendium of gene expression and chromatin state annotations, and apply it to IHEC data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Compendiums of chromatin state annotations based on integrating maps of multiple epigenetic marks such as from ChromHMM have become a powerful resource. While these compendiums have coverage of many biological samples, there are many additional biological samples that have gene expression data but lack epigenetic mark data and chromatin state annotations. The EpiAtlas resource of the International Human Epigenome Consortium (IHEC) contains a large compendium of chromatin state annotations for which many samples have matched gene expression data, which provides the opportunity to use it to train models to predict chromatin state annotations in additional biological samples with only gene expression data available. To address this, we develop Gene Expression-based Chromatin State Imputation (GECSI), which uses a multi-class logistic regression model trained using a large compendium of gene expression and chromatin state annotations, and apply it to IHEC data. Using cross-validation, we find that GECSI accurately predicts chromatin state assignments and generates probability estimates that are predictive of observed chromatin states, overall outperforming multiple other alternative and baseline methods. GECSI-predicted chromatin states reflect relationships among biological samples and show similar transcription factor and gene annotation enrichments as observed chromatin states. Using available IHEC gene expression data, we apply GECSI to predict chromatin state annotations for 449 additional epigenomes. We expect these predicted annotations and the GECSI software will be a useful resource for chromatin state analyses in many additional biological samples.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.02.679828v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Towards Human Inverse Dynamics from Real Images: A Dataset and Benchmark for Joint Torque Estimation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chen, C., Su, W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the vision-based inverse dynamics dataset (VID), the first dataset tailored for the joint torque prediction from real human images, and introduce a comprehensive benchmark for the state-of-the-art performance.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Human inverse dynamics is an important technique for analyzing human motion. Previous studies have typically estimated joint torques from joint pose images, marker coordinates, or EMG signals, which severely limit their applicability in real-world scenarios. In this work, we aim to directly predict joint torques during human movements from real human images. To address this gap, we present the vision-based inverse dynamics dataset (VID), the first dataset tailored for the joint torque prediction from real human images. VID comprises 63,369 frames of synchronized monocular images, kinematic data, and dynamic data of real human subjects. All data are carefully synchronized, refined, and manually validated to ensure high quality. In addition, we introduce a comprehensive benchmark for the vision-based inverse dynamics of real human images, consisting of a new baseline method and a new evaluation criteria with three levels of difficulty: (i) overall joint torque estimation, (ii) joint-specific analysis, and (iii) action-specific prediction. We further compare the baseline result of our VID-Network with other representative approaches, our baseline method achieves the state-of-the-art performance on almost all the evaluation criteria. By releasing VID and the accompanying evaluation protocol, we aim to establish a foundation for advancing biomechanics from real human images and to facilitate the exploration of new approaches for human inverse dynamics in unconstrained environments. The project website is https://github.com/MedVisionKitchen/VID-Benchmark.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.03.679772v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Ragnarok: a flexible and RApid GeNe Annotation (ROcKs) pipeline deployed through Nextflow</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kuster, R. D., Smith, Z. C., Whitt, L., Staton, M. E., Mansfeld, B. N., Gottschalk, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed RAGNAROK (RApid GeNe Annotation ROcKs), a modular annotation pipeline built on Nextflow and Apptainer that integrates ab initio prediction, transcriptome and protein evidence, repeat annotation, functional assignment, and quality assessment into a single reproducible workflow</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> High-quality genome assemblies and pangenomes are increasingly accessible and achievable due to advances in third-generation sequencing and assembly algorithms, but genome annotation remains a critical bottleneck. Existing gene annotation pipelines often require complex installations, multiple steps, long runtimes, and produce variable results, which impede quality and the downstream usage of the annotations. We developed RAGNAROK (RApid GeNe Annotation ROcKs), a modular annotation pipeline built on Nextflow and Apptainer that integrates ab initio prediction, transcriptome and protein evidence, repeat annotation, functional assignment, and quality assessment into a single reproducible workflow. Moreover, RAGNAROK utilizes GPU acceleration and parallelization to produce high-quality gene annotations. We benchmarked against BRAKER3 across five diverse, reference-quality plant genomes and demonstrated that RAGNAROK achieved higher sensitivity, precision, and F1 scores at the exon, transcript, and gene levels. Furthermore, we demonstrated RAGNAROK's improvement in re-annotating a suite of five Rosaceae genomes that were previously annotated using MAKER. Overall, RAGNAROK produced more ideal mono:multi-exonic gene ratios, improved BUSCO completeness scores, and reduced missing gene content compared to other pipelines. Additionally, RAGNAROK consistently outperformed BRAKER3 in runtime, scaling efficiently from small to gigabase-scale genomes. RAGNAROK provides a flexible, rapid, scalable, and accurate solution for de novo and re-annotation of plant genomes. Its modular design and workflow scalability lay the foundation for future extensions to animal, fungal, and other eukaryotic genomes. RAGNAROK is available as a GitHub repository at https://github.com/ryandkuster/ragnarok.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.03.680343v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Analyzing the Naming Conventions of Life Science Data Resources to Inform Human and Computational Findability</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Imker, H. J., Ou, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify suboptimal naming practices and a wide-spread opaqueness in common names, which poses challenges to resource identification and retrieval by both human- and computationally-centric methods.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> This study aimed to evaluate the names of life science data resources and consider the impacts on findability, a core feature of the FAIR (Findability, Accessibility, Interoperability, and Reusability) Principles. Utilizing a previously published list of unique data resources, we identified and validated data resources with both common and full names available (n = 1153). From this set, we analyzed characteristics of resource names to identify if any naming conventions have emerged organically. Additionally, since common names are often used in the absence of a resource's full name, we performed a test to evaluate our ability to infer any meaning from common names. Our results highlight suboptimal naming practices and a wide-spread opaqueness in common names, which poses challenges to resource identification and retrieval by both human- and computationally-centric methods. These results are informative for those who establish and promote data resources as well as for those who search for data to use in individual research projects, develop data discovery systems, analyze the scientific literature, or assess research infrastructure. The findings underscore the value of findability in the FAIR Principles and the current efforts to develop infrastructure that supports more efficient communication and global connectedness.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.02.680112v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Automated Specimen Triage for Dark Taxa: Deep Learning Enables Orientation, Sex Identification, and Anatomical Segmentation from Robotic Imaging</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shirali, H., Wuehrl, L., Lee, L., Klug, N., Meier, R., Pylatiuk, C., Hartop, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We demonstrate that deep learning and robotic imaging can be leveraged by deep learning to efficiently extract key biological information and guide targeted specimen processing, maximising efficiency and utility.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Robotic specimen processing is transforming biodiversity discovery by replacing labor-intensive handling with scalable systems that can simultaneously generate high-quality specimen images. We demonstrate that these images can be leveraged by deep learning to efficiently extract key biological information and guide targeted specimen processing. Using a model dark taxon, the Phoridae (Diptera), the workflow performs three core tasks: sex identification, specimen orientation classification, and anatomical segmentation. Sex identification allows selective retention of diagnostically informative specimens, avoiding wasted effort on non-diagnostic individuals. Orientation classification enables specimens in the desired orientation to proceed immediately, while suboptimally oriented specimens can be repositioned for informative processing. Anatomical segmentation allows targeted processing of specimens displaying diagnostic characters or targeted analysis of specific anatomical regions in subsequent workflow steps. Comparative analysis of model architectures shows task-specific selection is crucial: a Convolutional Neural Network achieved an accuracy of 0.94 for orientation, a Vision Transformer achieved 0.88 for sex, and a U-Net precisely segmented nine anatomical regions with a mean IoU of 0.78. These results demonstrate that robotic imaging combined with deep learning provides a validated foundation for high-throughput, targeted specimen processing, maximising efficiency and utility for taxonomic and trait-based analyses, and supporting scalable, sustainable biodiversity workflows.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.02.680063v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Activation of the cGAS-STING pathway contributes to cancer-related fatigue in a murine model of head and neck cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chelette, B., Bashaw, A., Bryant, J. D., Chidomere, C. L., Scott, K., West, A. P., Dantzer, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Cancer-related inflammation and metabolic alterations extend beyond the tumor microenvironment, exerting systemic effects that disrupt energy homeostasis and contribute to chronic fatigue.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cancer-related inflammation and metabolic alterations extend beyond the tumor microenvironment, exerting systemic effects that disrupt energy homeostasis and contribute to reduced physical function and chronic fatigue. The cGAS-STING pathway has emerged as a key regulator of innate immunity and inflammation; however, its role in cancer-associated fatigue remains poorly understood. In this study, we investigated the contribution of cGAS-STING-mediated inflammation to cancer- and/or its treatment-induced fatigue using a mouse model of human papillomavirus-related head and neck cancer. Wheel running activity, along with inflammatory and metabolic changes in tumor and liver tissues, were assessed following chemoradiotherapy and pharmacological inhibition of STING in tumor-bearing and tumor-free control mice. The results revealed that tumor growth and chemoradiotherapy activated the cGAS-STING pathway, together with an upregulation of proinflammatory mediators and alterations of mitochondrial and metabolic gene expression in the liver. To inhibit STING activation, mice were administered H-151, a specific STING antagonist. This intervention attenuated hepatic inflammatory signatures and mitigated tumor and/or chemoradiotherapy-associated behavioral fatigue measured by decreased voluntary wheel running. These findings implicate for the first time the cGAS-STING signaling pathway and metabolic homeostasis in cancer- and cancer therapy-related fatigue.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.02.680158v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Photoproximity labeling of c-Myc reveals SLK as a cancer specific co-regulator</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Milione, R. R., Tong, F., Wolfe, K. L., Linker, S. B., Mu, X. J., Oakley, J. V., Nager, A. R., Janiszewska, M., Seath, C. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that cancer-specific protein-protein interactions can be mined to identify druggable vulnerabilities in androgen receptor dependent and independent prostate cancer cell lines, and that a PC-3 specific Myc interactor, SLK, selectively promotes c-Myc stabilization at the protein level, drives</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Transcription factors (TFs) have long been aspirational therapeutic targets for the treatment of diseases, as their dysregulation is a common mechanism for altered cell states. Despite this, many TFs implicated in disease have disordered structures and lack canonical binding pockets, rendering them non-trivial targets for small molecule-based therapies. Directly inhibiting TF function has proven difficult, but indirect inhibition by targeting the effector molecules that modulate TF function is a promising alternative strategy. Here we report a strategy for capturing cancer-specific protein-protein interactions using context-dependent uMap photoproximity labeling. Using an intein-based strategy for catalyst conjugation in biochemically intact nuclei, we demonstrate that we can capture unique protein interactomes of c-Myc in androgen receptor dependent and independent prostate cancer cell lines, and that these unique interactors can be mined to identify druggable vulnerabilities. We find that a PC-3 specific Myc interactor, SLK, selectively promotes c-Myc stabilization at the protein level, drives epithelial morphology and is essential for tumorigenesis, validating it as a viable therapeutic target. Importantly, this stabilization is driven by a change in SLK splicing rather than expression at the protein or RNA levels. Furthermore, analysis of cancer patient data shows a strong correlation between this splice isoform and expression of c-Myc targets, suggesting this novel regulatory axis is operative across human cancer.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.02.680136v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An ERα-Dependent Hypoxia Response Defines EMT-Adjacent Tumour Regions and Suppresses the Pro-survival Effects of Amiloride in Estrogen Receptor-Positive Breast Cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Malcolm, J. R., Stenning, J., Pope, J., Lukaszonek, J., Rose, S. F., Smith, T. E., Gilbert, L., James, S. R., Bridge, K. S., Brackenbury, W. J., Holding, A. N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We demonstrate that activation of hypoxia-inducible factors (HIFs) dramatically remodels ER chromatin localisation in ER+ breast cancer cells, and present functional evidence that ER reprogramming offers novel therapeutic opportunities that bypass the need to directly target the hypoxic response.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Estrogen receptor-positive (ER+) breast cancer carries a lifelong risk of recurrence and disease progression, with hypoxia-associated transcriptional signatures linked to poor prognosis and therapy resistance. While the effects of hypoxia on tumour progression are well studied, the impact on ER epigenomic regulation remains poorly characterised. Here, we demonstrate that activation of hypoxia-inducible factors (HIFs) dramatically remodels ER chromatin localisation in ER+ breast cancer cells. Transcripts of genes located near hypoxia-induced ER binding sites are significantly associated with reduced recurrence-free survival in breast cancer patients. Transcriptomic profiling under hypoxic conditions (1% oxygen), with and without ER depletion by fulvestrant, revealed a hypoxia-induced, ER-dependent gene expression programme, including upregulation of epithelial sodium channel (ENaC) regulatory subunits that results in acquired sensitivity to the ENaC inhibitor amiloride. Notably, this transcriptional response is spatially correlated with the epithelial-to-mesenchymal hallmark in patient tumours. Our findings establish an interdependence between ER signalling and the hypoxic response, and present functional evidence that ER reprogramming offers novel therapeutic opportunities that bypass the need to directly target the hypoxic response.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.03.680339v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

