<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Sunday, 29 June 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Fundamental Limitations of Foundation Models in Single-Cell Transcriptomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Atti, S., Subramaniam, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We found that foundation models for single-cell tasks consistently underperform statistical models due to a lack of essential biological context.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent applications of foundation models in biology have focused on pretraining using large-scale single-cell datasets comprising millions of cells, across diverse patho-physiological states. These models are then fine-tuned for downstream tasks such as cell-type classification. In this study, we evaluated the performance of three widely-used foundation models in biology (scGPT, SCMAMBA-2, and Geneformer) and a statistical baseline (Seurat v5) on cell-type classification under Gaussian noise perturbation. We used two curated datasets, referred to as Myeloid (13k cells) and hPancreas (15k cells). Surprisingly, we found that the baseline performance of the foundation models was inferior to that of the statistical model, even without any added perturbation. Although we note that model size can affect performance, Geneformer accuracy outperformed scGPT and SCMAMBA-2 by ~5% on average across all datasets despite having ~40% fewer trainable parameters. Nonetheless, the statistical baseline still outperformed Geneformer by ~9% in accuracy. Based on these findings, we hypothesized that the conventional training paradigm used by foundation models for single-cell tasks consistently underperforms statistical models due to a lack of essential biological context. To investigate this, we evaluated whether performance degradation stems from early data embedding steps (such as binning or gene normalization) during tokenization, and sampling bias. First, to better understand tokenization-related artifacts, we introduced controlled Gaussian noise to gene expression values before tokenization, amplifying downstream distortions introduced by the tokenization process (all models were trained for identical step durations with identical hyperparameters). On the Myeloid dataset, following the introduction of Gaussian noise perturbation to 20% of cells, both scGPT and SCMAMBA-2 saw an ~11% decrease in accuracy while Geneformer saw an ~8% decrease in accuracy. This difference in performance may be due to the difference in encoding methods used by both models. scGPT and SCMAMBA-2 use a bin-based tokenization strategy, in contrast to Geneformer rank-value encoding, which normalizes gene expression values using a predetermined encoding constant. Although binning captures general trends in count data, it fails to preserve relative expression at the gene level, resulting in significant information loss. Applying the three models to the Myeloid dataset also revealed that the scGPT prediction distribution is biased toward overrepresented cell types in the training data, while underrepresenting rarer classes. CD14 cells are overpredicted by ~16% (among most abundant cell types) by scGPT. Geneformer, however, maintains a more stable prediction distribution with a ~6% (CD14) increase and outperforms scGPT and SCMAMBA-2 by ~26% in macro F1 score (unweighted metric). Based on our findings we assert that the gap in contextual encoding in bin-based tokenization is what contributes to the less-nuanced learning. Recent research that integrated cellular-ontology during training showed improved performance to both scGPT and Geneformer. Our results underscore a fundamental issue, that foundation models lack critical biological context that would allow for them to make the nuanced inferences required for complex biological analyses. The compression of single-cell data from raw counts to embedding vectors can span several orders of magnitude, and lead to significant loss of information. As a result, methods must adapt to prioritize contextual integration during tokenization to ensure sufficient information for the model.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.26.661767v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multimodal Benchmarking of Foundation Model Representations for Cellular Perturbation Response Prediction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hasanaj, E., Cole, E., Mohammadi, S., Addagudi, S., Zhang, X., Song, L., Xing, E. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We conduct the first systematic pan-modal study of perturbation embeddings, with an emphasis on those derived from biological FMs.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The decreasing cost of single-cell RNA sequencing (scRNA-seq) has enabled the collection of massive scRNA-seq datasets, which are now being used to train transformer-based cell foundation models (FMs). One of the most promising applications of these FMs is perturbation response modeling. This task aims to forecast how cells will respond to drugs or genetic interventions. Accurate perturbation response models could drastically accelerate drug discovery by reducing the space of interventions that need to be tested in the wet lab. However, recent studies have shown that FM-based models often struggle to outperform simpler baselines for perturbation response prediction. A key obstacle is the lack of understanding of the components driving performance in FM-based perturbation response models. In this work, we conduct the first systematic pan-modal study of perturbation embeddings, with an emphasis on those derived from biological FMs. We benchmark their predictive accuracy, analyze patterns in their predictions, and identify the most successful representation learning strategies. Our findings offer insights into what FMs are learning and provide practical guidance for improving perturbation response modeling.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.26.661186v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Molecular dynamics reveals a novel Regulatory Ca+2-Binding Site in the Mitochondrial Calcium Uniporter</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sommese, L. M., Palopoli, N., Fornasari, M. S., Parisi, G., Gabaldon, T., Rueda, A. J. V.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify a previously unknown pseudo-EF-hand motif in MICU1 that acts as an early calcium sensor and provide new mechanistic insights into the regulation of the mitochondrial Ca+2; uniporter.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The mitochondrial Ca+2; uniporter (MCU) plays essential roles in energy production, cytosolic Ca+2; signaling, and regulation of cell death. The activity of this channel complex is modulated by Mitochondrial Calcium Uptake 1 (MICU1) and Mitochondrial Calcium Uptake 2 (MICU2) proteins in response to Ca+2; levels. In cardiomyocytes, MICU1 inhibits the uniporter in resting cells while it activates it during contraction when bound to Ca+2;. Despite its importance, the structural mechanisms driving MICU1's open-close switch remain unclear. In this study, we used structural modeling combined with molecular dynamics simulations to investigate MICU1's conformational dynamics under varying Ca+2; concentrations. Our analysis identified a previously uncharacterized pseudo-EF-hand motif (pEF-h) that acts as an early calcium sensor, initiating conformational transitions that prime canonical EF-hand sites for subsequent calcium binding. Calcium binding induces substantial structural changes in MICU1, including a decrease in isoelectric point, shifts in surface charge distribution, and significant reductions in cavity volume, length, and hydrophobicity. Notably, a 53.4% reduction in intermolecular distances between key domains was observed during the transition from the apo to the Ca+2;-bound state. These findings uncover a previously overlooked hierarchical activation of MICU1's calcium-binding sites and provide new mechanistic insights into the regulation of MCU. This enhanced understanding of mitochondrial calcium signaling paves new ways to potential novel therapeutic strategies targeting calcium dysregulation in metabolic, cardiovascular, and neurodegenerative diseases.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.25.661581v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Mammal placental phenotypes are predictable from microRNA repertoires.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fenn, J., Edge, J. C., Ovchinnikov, V., Amelkina, O., Sinclair, M., De Bem, T. H. C., Bridi, A., Malo-Estepa, I., Tinning, H., Gonella-Diaza, A., Perecin, F., da Silveira, J. C., Pugliesi, G., Meirelles, F. V., Butt, Z., Fromm, B., McInerney, J. O., Forde, N., O'Connell, M. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that placental morphological diversification is constrained to predictable genetic pathways and that miRNA-mediated regulation of placental phenotype constrains placental diversification into reproducible programs.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Despite >100 million years of mammal diversification, similar placental morphologies have independently evolved multiple times, presenting a long-standing evolutionary puzzle: what genetic mechanisms lead to convergent forms? MicroRNAs (miRNAs), despite their diversity in mammals and known roles as developmental regulators, remain under-examined as drivers of morphological evolution. We identified presence-absence patterns for 429 miRNA gene families across 300 eutherian genomes and discovered that placental phenotype is highly predictable from genomic miRNA repertoires (classification accuracy >94-97%). We identified 30 miRNA gene families associated with placentation type, whose gene targets are enriched for developmental processes. Notably, convergent placental morphologies consistently involve identical miRNA families, revealing that evolution is constrained to predictable genetic pathways. We demonstrate evidence of mechanism through MIR-11968, associated with cotyledonary placentation and displaying tissue-restricted expression in cows. MiRNA-mediated regulation therefore constrains placental morphological diversification into reproducible programs, offering insights into how genetic architecture shapes the predictability of convergent evolution.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.27.660101v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">EC-Bench: A Benchmark for Enzyme Commission NumberPrediction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Davoudi, S., Henry, C. S., Miller, C. S., Banaei-Kashani, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce EC-Bench, an open-source benchmark for enzyme number prediction, which provides a framework for evaluating and studying the performance of existing methods objectively under uniform conditions, but also to introduce and effectively evaluate performance of new methods in a comparative framework.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Enzymes are proteins that catalyze specific biochemical reactions in cells. Enzyme Commission (EC) numbers are used to annotate enzymes in a four-level hierarchy that classifies enzymes based on the specific chemical reactions they catalyze. Accurate EC number prediction is essential for understanding enzyme functions. Despite the availability of numerous methods for predicting EC numbers from protein sequences, there is no unified framework for evaluating and studying such methods systematically. This gap limits the ability of the community to identify the most effective approaches for enzyme annotation. We introduce EC-Bench, a benchmark for EC number prediction, consisting of 1) an initial representative set of existing methods (including homology-based, deep learning, contrastive learning, and language model methods), 2) exist-ing and novel accuracy and efficiency performance metrics, and 3) selected datasets to allow for comprehensive comparative study. EC-Bench is open-source and provides a framework for researchers to not only compare among existing methods objectively under uniform conditions, but also to introduce and effectively evaluate performance of new methods in a comparative framework. To demonstrate the utility of EC-Bench, we perform extensive experimentation to compare the existing EC number prediction methods and establish their advantages and disadvantages in a variety of prediction tasks, namely "exact EC number prediction", "EC number completion" and (partial or additional) "EC number recommendation". We find wide variation in the performance of different methods, but also subtle but potentially useful differences in the performance of different methods across tasks and for different parts of the EC hierarchy.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.25.661207v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Navigating the Lipid Universe with LipidLibrarian: A Cross-Linked Database for Lipidomics Data Integration</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Niedermaier, F., Mechteridis, K., Pelz, K., Wuerf, V., Koehler, N., Pauling, J. K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present LipidLibrarian, a lipid search engine that enables a combined search of all major lipid databases by aggregating the available information and presenting it in a unified manner.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> There are numerous public resources and guidelines available for lipidomics research, including standard nomenclatures, classification systems, and lipid databases. However, these resources are not always aligned with one another, making it difficult to find and compare information on the same lipid across different databases. To tackle these challenges we present LipidLibrarian, a lipid search engine that enables a combined search of all major lipid databases by aggregating the available information and presenting it in a unified manner. The three main sources of information that build the foundation of LipidLibrarian as a comprehensive search-engine are SwissLipids, LIPID MAPS and ALEX123. Furthermore, various secondary resources such as LION/web, LINEX, LipidLynxX, and Goslin were incorporated to enhance the results and conduct name and hierarchy conversions. LipidLibrarian is accessible via a user-friendly website, allowing the user to query lipids using their trivial names, shorthand notations, database identifiers, or their masses. Alternatively, LipidLibrarian can be accessed as a Python package for integration into high-throughput lipidomics pipelines. The output of a LipidLibrarian query is split into multiple categories, such as nomenclature, database identifiers, masses, adducts, fragments, ontology terms, and reactions. For each of these categories, LipidLibrarian aggregates the results from all databases and provides the source from which each value originates. This enables the user to quickly assess if the databases contain differing or conflicting information. In summary, LipidLibrarian provides an effortless, comprehensive and automated search for lipid information, thereby accelerating the research workflow and making it a meaningful tool for the scientific community.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.26.661298v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Unveiling the genetic basis of the low pH response in the acidophilic yeast Maudiozyma bulderi as a potential host for biorefinery</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Balarezo-Cisneros, L. N., Hanak, A., Zeef, L., Mironov, A., Valle, F., Delneri, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified four uncharacterised genes, unique to Maudiozyma bulderi, and predicted function as transporters, upregulated at low pH.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Non-conventional yeasts represent a great genetic and phenotypic diversity with potential for industrial strain development in the bio-production of green chemicals. In recent years, mass genome sequencing of non-conventional yeasts has opened avenues to improved understanding of transcriptional networks and phenotypic plasticity and gene function, including the discovery of novel genes. Here, we investigated the expressional and morphological changes at low-pH in three strains of the acidophilic yeast Maudiozyma bulderi (previously Kazachstania bulderi and Saccharomyces bulderi): CBS 8638, CBS 8639 and NRRL Y-27205. The comparison of the transcriptome of cells growing in a bioreactor at pH=5.5 vs pH= 2.5, primarily showed dysregulation of genes involved in cell wall integrity, with NRRL Y-27205 the least acidophilic strain, showing the largest transcriptional response when compared to the other two strains. We identified four uncharacterised genes, unique to M. bulderi, and predicted function as transporters, upregulated at low pH. Microscopy studies showed that M. bulderi cell wall is not damaged in acidic environment, and the membrane lipid composition remains stable at low pH, unlike S. cerevisiae. Overall, our data on transcriptional variability in M. bulderi highlights genes and cellular pathways involved in the acidophilic adaptation of this species and can aid further strain development.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.27.661920v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Asian pika populations track local glaciation events through the Pleistocene</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kumar, H., Vijay, N., Rajamani, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of high-elevation specialist pikas on the Indo-Tibetan Plateau and their historical demography .</p><p style="font-size: 16px; text-align: justify;">Background The Pleistocene glaciation cycles (2.6 mya - 11 kya) were major climatic events that shaped diversity and community assemblages on Earth. Cold-adapted, high-elevation specialists are expected to have responded negatively to warm interglacial periods, with populations contracting and being pushed up the elevational gradient, leading to isolated populations on mountain tops. During the cool glacial maximums, they are expected to have been distributed at both low and high elevations, with populations expanding and facilitating gene flow across mountains. Pikas (Ochotonidae) are poorly studied high-elevation specialist Lagomorphs being extirpated at alarming rates due to climate change. Insights into their historical demography and current effective population size are crucial in determining their current population status to inform conservation planning. Results We use de novo assembly to construct partial genomes of three Asian pika species (Ochotona ladacensis, O. nubrica, and O. macrotis) that differ in their life history, lifestyle, and social behaviors. We use a combination of species distribution modeling (SDM) projected back in time, in addition to inferring historical demographic patterns using a pairwise sequentially Markovian coalescent (PSMC) approach. Our SDMs predicted the largest bio-climatic niches for all three species during the last glacial maximum (LGM ~20 kya), and the smallest niches at the last interglacial (LIG~120 kya), agreeing well with the elevational shift hypothesis. Our PSMC models for the populations in the Changthang Biotic province (CBP), revealed largely synchronous oscillations of populations sampled in geological timescales but displayed contrasting temporal patterns of population spikes when compared with SDMs. The largest effective population sizes of all species examined on the CBP were inferred to pre-date both the LGM and LIG and were placed around the MIS-6 (~ 191 kya). After the MIS-6 local glaciation event, populations were inferred to have been through a deep bottleneck, correlating well with local glaciation patterns on this plateau. Conclusion Our study illustrates the power of using complementary approaches to infer historical demography in mountainous landscapes with complex glaciation histories. While SDMs appropriately describe species responses to historical climate across the geographic range of species, PSMCs are more informative about local population dynamics when subject to large degrees of population differentiation. All three species of pikas responded similarly to historical glaciation events on the CBP that differ from the rest of the world and other parts of the Himalayas and likely have very low effective population sizes at current timescales. This is one of the first few studies to examine the population demography of small mammals on the Indo-Tibetan Plateau and has the potential to inform species and population-specific conservation planning in the Himalayas. It is very likely that small mammals such as pikas, marmots and voles on the CBP have low effective population sizes due to historical inbreeding during the LIG and the lack of a glaciation rescue during the LGM. This highlights the need for conservation measures for small mammals on the landscape.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.25.661461v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Tomato leaf transcriptomic changes promoted by long-term water scarcity stress can be largely prevented by a fungal-based biostimulant</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lopez-Serrano, L., Ferez-Gomez, A., Romero-Aranda, R., Jaime Fernandez, E., Leal Lopez, J., Fernandez Baroja, E., Almagro, G., Dolezal, K., Novak, O., Diaz, L., Bautista, R., Leon Morcillo, R. J., Pozueta Romero, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Foliar application of fungal culture filtrates to tomato plants improves yield and long-term water scarcity tolerance by preventing a large portion of the stress-induced transcriptional response, rendering plants less sensitive to water stress.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Water availability is by far the leading environmental factor limiting crop productivity worldwide. The use of cell-free microbial culture filtrates (CF) as biostimulant is gaining ground as a safe and ecologically sound approach to improving crop yields while reducing anthropogenic pressure. However, their action mechanisms remain unknown. Here we found that foliar application of CF enhanced fruit yield, root growth and photosynthesis in plants of a commercial tomato cultivar grown under long-term water deficit in Mediterranean greenhouse conditions. To investigate the biochemical and molecular mechanisms underlying this phenomenon, we adopted an integrative and systems biology approach to characterize plants grown under optimal and suboptimal irrigation conditions (OIC and SOIC, respectively) with or without the fungal CF treatment. Water shortage promoted changes in the levels of drought stress-related signalling molecules, and in the transcriptome of leaves that potentially accounted for the physiochemical differences recorded between OIC- and SOIC-grown plants. Notably, many of these changes were largely prevented by the foliar application of fungal CF to SOIC-grown plants, including the expression of ca. 50% of the expression of water scarcity-responsive genes. These genes did not respond to CF in OIC-grown plants, indicating that the transcriptomic response to CF is strongly determined by the water status of the plant. Taken together, the data provided evidence that foliar application of fungal CF improves yield and long-term water scarcity tolerance by preventing a large portion of the stress-induced transcriptional response, rendering plants , or less sensitive to water stress.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.26.661737v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A core genetic mechanism integrates growth hormone signals to control meristem fate and inflorescence architecture in setaria and maize</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sandhu, J., Yang, J., Braud, M., Bertolini, E., Eveland, A. L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A model of inflorescence architecture in grass Setaria viridis and a model for maize where a determinacy factor ramosa1 acts upstream of BR-mediated regulation of SM identity and determinacy .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Inflorescence architecture in cereals is a major determinant of grain yield and harvestability. Architecture is determined by the number, arrangement and order of lateral branches, which are formed from clusters of stem cells called meristems. Variation in branching patterns arises from regulation of meristem determinacy, i.e., indeterminate meristems maintain meristematic activity allowing for higher order branching and determinate meristems form a terminal organ. The timing by which an indeterminate branch meristem acquires a determinate fate and transitions to a spikelet meristem (SM), ultimately forming the grain-bearing spikelet, defines architecture and grain yield potential. This study leverages the unique inflorescence morphology of model grass Setaria viridis (setaria) to identify factors controlling SM fate and determinacy and characterize their interactions. In setaria, axillary branches terminate either in a spikelet or sterile bristle, where the latter loses SM identity due to brassinosteroid (BR)-mediated disruption of organ boundary formation. A forward genetics screen identified two mutants with only bristles on their panicles; one defective in gibberellic acid (GA) signaling caused extreme elongation and disruption of spikelet development and the other homeotically converted spikelets to bristles. The latter named spikeletless (spkl) was mapped and validated as an ortholog of the maize determinacy factor ramosa1 (ra1). Genetic analyses showed that SvRa1 acts upstream of BR-mediated regulation of SM identity and determinacy. A model is proposed where SvRA1 maintains SM fate by modulating GA homeostasis, spatially restricting BR signaling and preserving boundary gene expression and determinacy. In bristle primordia, which are generally paired with spikelets, GA and BR act synergistically. Comparative analyses in maize highlight pathway conservation in inflorescence architecture with some specific variations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.27.660957v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Unsupervised Multi-scale Segmentation of Cellular Cryo-electron Tomograms with Stable Diffusion Foundation Model</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Uddin, M. R., Nguyen, T.-H., Tabib, H. M. S., Gandhi, K., Xu, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce an unsupervised approach for segmenting multiscale subcellular objects in 3D volumetric cryo-electron tomography (cryo-ET) images, addressing key challenges such as large data volumes, low signal-to-noise ratios, and the heterogeneity</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We introduce an unsupervised approach for segmenting multiscale subcellular objects in 3D volumetric cryo-electron tomography (cryo-ET) images, addressing key challenges such as large data volumes, low signal-to-noise ratios, and the heterogeneity of subcellular shapes and sizes. The method requires users to select a small number of slabs from a few representative tomograms in the dataset. It leverages features extracted from all layers of a Stable Diffusion foundation model, followed by a novel heuristic-based feature aggregation strategy. Segmentation masks are generated using adaptive thresholding, refined with CellPose to split composite regions, and then utilized as pseudo-ground truth for training deep learning models. We validated our pipeline on publicly available cryo-ET datasets of S. Pombe and C. Elegans cell sections, demonstrating performance that closely approximates expert human annotations. This fully automated, data-driven framework enables the mining of multi-scale subcellular patterns, paving the way for accelerated biological discoveries from large-scale cellular cryo-ET datasets.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.25.661425v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

