<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Sunday, 12 October 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">GeDi: Simplifying Gene Set Distances for Enhanced Omics Interpretation in R/Bioconductor</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nedwed, A., Ustjanzew, A., Abassi, N., Dammer, L., Schulze, A., Helbich, S., Delacher, M., Strauch, K., Marini, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed GeDi, an R/Bioconductor package designed to streamline and standardize the interpretation of functional enrichment results by combining clustering and network context.</p><p style="font-size: 16px; text-align: justify;">Background: Functional enrichment analysis is a standard component in many omics data analysis workflows, supported by a variety of methods and algorithms. However, despite their utility and wide application, these methods often return the results as an extensive and redundant list of gene sets, impeding interpretation and hypothesis generation. Moreover, network based information can provide additional biological context through functional interaction data, yet this is often overlooked by existing tools. Results: We developed GeDi, an R/Bioconductor package designed to streamline and standardize the interpretation of functional enrichment results. GeDi aggregates gene sets into biologically meaningful clusters using a suite of gene set distance metrics and clustering algorithms, aimed to reduce redundancy and improve clarity. GeDi also enables the integration of protein-protein interaction (PPI) data, through the implementation of a weighted distance metric, providing a richer biological context by capturing functional connectivity between pathways and their components. The package offers visualizations, aggregation, and automated reporting, and is available as both a stand-alone R-package and an interactive Shiny application. Conclusion: GeDi facilitates clearer, faster interpretation of enrichment results by combining clustering and network context. Application to a public RNA-seq dataset revealed coherent biological themes, supporting both experimental and computational research. GeDi is freely available in the Bioconductor project under the MIT license (https://bioconductor.org/packages/GeDi), and a demo instance is accessible on the Shiny server (http://shiny.imbei.uni-mainz.de:3838/GeDi).</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.10.681592v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Unified Framework for TCR-pMHC Structural Model Assessment</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ascunce-Paris, A., Farriol-Duran, R., Romero-Durana, M., Valencia, A., Guallar, V.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a framework that enables quality assessment of TCR-pMHC structural models without requiring experimental reference structures, and demonstrate the applicability of our quality tier stratification in three settings: 1) filtering validating vs. non-validating TCRs, 2) enrichment of biologically validated</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> T cell receptor (TCR) recognition of peptide-MHC complexes (pMHCs) is a central determinant of cellular immunity. Structural insights are key to understanding TCR-pMHC interactions, however progress is limited by the scarcity of experimental structures (PDB = 232 TCR-pMHC class I complexes, date 01/2025). Although protein modelling tools have rapidly advanced, their accuracy on TCR-pMHC complexes is hampered by CDR loop hypervariability, conformational flexibility, and the lack of reliable quality assessment without experimental references. To evaluate the performance of recent protein modelling algorithms on TCR-pMHC complexes, we benchmarked three general-purpose (AlphaFold3, Boltz-2, Chai-1) and two TCR-specific (tFold-TCR, TCRmodel2) modelling tools using a benchmark set of 20 experimentally determined PDB structures, demonstrating AlphaFold3 superior performance. To expand structural coverage beyond the existing experimental landscape, we present a framework that enables quality assessment of TCR-pMHC structural models without requiring experimental reference structures. To this end, we integrate multiple modelling and interface confidence metrics (pLDDT, ipTM, iPAE, pDockQ) into explainable random forest classifiers trained on 1160 models of 232 experimentally determined PDB TCR-pMHC class I complexes. This approach reliably distinguishes low-, acceptable-, medium-, and high-quality models, outperforming the combination of these metrics using literature-established thresholds. Leveraging our quality tier framework, we used AlphaFold3 to generate and evaluate the largest synthetic dataset of TCR-pMHC class I structural models to date from VDJdb-annotated sequences, comprising 33,820 complexes (169,100 models, 5 models per complex) and representing a >70-fold expansion of available structures. We demonstrate the applicability of our quality tier stratification in three settings: 1) filtering validating vs. non-validating TCR-pMHC interaction in sequence databases (VDJdb), 2) enrichment of biologically validated interactions versus synthetic negatives amongst higher quality complexes, and 3) enhancing the predictive performance of TCR-pMHC pairing models. Altogether our TCR-pMHC structural quality tier framework provides a scalable and interpretable approach for improving modelling and functional analyses of TCR-pMHC class I complexes, with translational applications in T-cell and TCR-based immunotherapies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.09.681411v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DeepCEF: A Deep Causal Estimation Framework for Complex Biological Systems Integrating Local Scores, Independence Tests, and Relation Attributes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fan, Z., Zhang, M., Han, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel causal estimation framework that leverages the powerful classification capabilities of deep neural networks (DNNs) to identify causal patterns in pairwise relationships.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Causal relationship identification is a fundamental and complex research challenge that spans multiple disciplines, including biology, epidemiology, economics, and philosophy. Various scoring techniques and independence tests, such as local scores (e.g., Degenerate Gaussian (DG) and Bayesian Information Criterion (BIC)) and independence tests (e.g., Fisher's Z), have been employed in causality estimation. However, these local scores often excel in specific data types or application areas but falter in others, limiting their ability to capture the complexity and heterogeneity of underlying causal mechanisms. For instance, a method may perform well on linear relationships or continuous variables but struggle with discrete variables or non-linear relationships. Real-world observational datasets, particularly those generated in complex biological systems, often contain diverse data types and relationships, making it essential to develop a more comprehensive approach. To address this challenge, we propose a novel causal estimation framework that leverages the powerful classification capabilities of deep neural networks (DNNs) to identify causal patterns in pairwise relationships. Our framework integrates multiple local causality estimation scores, independence tests, and variable attributes, allowing it to capture a wide range of causal mechanisms. To ensure the framework's robustness and generalizability, we incorporate a diverse range of simulation data and 10 curated real-world datasets into the training procedure. Furthermore, our framework is designed to be extensible, enabling users to easily integrate their own data and additional scores and tests. Our validation results demonstrate that our framework outperforms existing methods in terms of estimation accuracy and precision on both simulation data and real-world biological datasets. By providing a more comprehensive and adaptable approach to causal relationship identification, our framework has the potential to advance research in various fields and improve our understanding of complex biological systems.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.09.681515v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Somatostatin-expressing neurons regulate sleep deprivation and recovery</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kobayashi, K. T., Taguchi, Y.-h.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We re-analyzed gene expression profiles from the male mouse cortex under conditions of sleep deprivation using tensor decomposition-based unsupervised feature extraction.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We re-analyzed gene expression profiles from the male mouse cortex under conditions of sleep deprivation using tensor decomposition-based unsupervised feature extraction. The analysis revealed that the selected genes were enriched in various brain regions, as identified through clustering in the Allen Brain Atlas. Somatostatin (Sst)-expressing neuronal clusters were among the most highly enriched. These finding suggests that Sst-expressing neurons may play a key role in sleep deprivation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.10.681551v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Kidney Tissue Characterization using Normalized Raman Imaging and Segment-Anything</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Karmakar, R., Trim, W., Kirschner, M., Noerrelykke, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a novel, modular computational pipeline for automated segmentation and quantification of kidney tissue structures imaged with NoRI.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Normalized Raman Imaging (NoRI) enables high-resolution, label-free quantification of protein and lipid concentrations in biological tissues. Because NoRI provides rich molecular information, the analysis of its large, multi-channel datasets turns into a significant computational bottleneck. In this work, we introduce a novel, modular computational pipeline for automated segmentation and quantification of kidney tissue structures imaged with NoRI. The pipeline integrates classical image processing with state-of-the-art machine learning tools, including the Segment Anything Model (SAM) and ilastik, to segment key anatomical and biochemical features, such as tubules, nuclei, brush borders, and lumens. A custom contrast-enhancement strategy was developed to create a third SAM input channel from NoRI data, leading to a substantial improvement in segmentation performance (F1 score: 0.9226). Our framework enables accurate cytoplasm-resolved quantification of protein and lipid concentrations and reveals distinct biochemical signatures across renal tubule subtypes and experimental conditions. This method offers a robust, scalable foundation for quantitative tissue analysis and enhances the utility of NoRI imaging for biomedical research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.09.681509v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A General Transformer-Based Multi-Task Learning Framework for Predicting Interaction Types between Enzyme and Small Molecule</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Atabaigielmi, V., Joeres, R., Kalinina, O. V.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a transformer-based multi-task learning framework designed to learn pairwise interaction signals between enzymes and small molecules, that can well generalise to out-of-distribution data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Predicting enzyme-small molecule interactions is critical for drug discovery and generally understanding the biochemical processes of life. While recent deep learning approaches have shown promising results, several challenges remain: the lack of a comprehensive training dataset, architectures lacking communication between representations of enzymes and small molecules, the tendency to simplify the problem as enzyme-substrate vs. enzyme-non-interacting, and thereby, misclassify enzyme-inhibitor pairs as substrates, and the negligence of the true impact of data leakage on the model's performance. To address these issues, we present EMMA (Enzyme-small Molecule interaction Multi-head Attention), a transformer-based multi-task learning framework designed to learn pairwise interaction signals between enzymes and small molecules, that can well generalise to out-of-distribution data. EMMA operates directly on the SMILES string representation of small molecules and enzyme sequences, with two classification heads that distinguish enzyme-non-interacting, enzyme-substrate, and enzyme-inhibitor pairs. By evaluating EMMA under five distinct data-splitting regimes that control for different types of data leakage, we demonstrate that EMMA achieves a strong and robust performance, particularly for previously unseen combinations of enzymes and small molecules. Further, a deeper analysis highlights that the topological properties of the enzyme-small molecule interaction network are crucial for the model performance and its ability to generalise, yet again stressing the decisive role of well-designed training datasets for successful model training.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.09.681419v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The global, regional, and national burden of cancer, 1990-2023, with forecasts to 2050: a systematic analysis for the Global Burden of Disease Study 2023.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> </p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We used the Global Burden of Diseases, Injuries, and Risk Factors Study (GBD) 2023 framework to generate and analyse estimates of cancer burden for 47 cancer types or groupings by age, sex, and 204 countries and territories from 1990 to 2023, cancer burden attributable to selected</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cancer is a leading cause of death globally. Accurate cancer burden information is crucial for policy planning, but many countries do not have up-to-date cancer surveillance data. To inform global cancer-control efforts, we used the Global Burden of Diseases, Injuries, and Risk Factors Study (GBD) 2023 framework to generate and analyse estimates of cancer burden for 47 cancer types or groupings by age, sex, and 204 countries and territories from 1990 to 2023, cancer burden attributable to selected risk factors from 1990 to 2023, and forecasted cancer burden up to 2050.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41015051/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

