<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Tuesday, 29 July 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">protPheMut: An Interpretable Machine Learning Tool for Classification of Cancer and Neurodevelopmental Disorders in Human Missense Mutations.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jingran Wang, Miao Yang, Chang Zong, Yuan Li, Gennady Verkhivker, Fei Xiao, Guang Hu</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed protPheMut (Protein Phenotypic Mutations Analyzer, http://netprotlab.com/protPhemut), leveraging interpretable machine learning approaches and enhancing model transparency through SHAP explanations, to integrate diverse biophysical and network dynamics-based signatures for predicting whether</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent advances in human genomics have revealed that missense mutations in a single protein can lead to distinctly different phenotypes. In particular, some mutations in oncoproteins like MEK1, MEK2, PI3Kα, PTEN, SHAP2, and RAS are linked various cancers and neurodevelopmental disorders (NDDs). While numerous tools exist for predicting the pathogenicity of missense mutations, linking these mutations to certain phenotypes remains a major challenge, particularly in the context of personalized medicine. To fill this gap, we developed protPheMut (Protein Phenotypic Mutations Analyzer, http://netprotlab.com/protPheMut), leveraging interpretable machine learning approaches and enhancing model transparency through SHAP explanations, to integrate diverse biophysical and network dynamics-based signatures for predicting whether mutations in the same protein promote cancer or NDDs. Overall, proPheMut achieved an AUCROC of 0.9118 in cross-validation and 0.8925 on an independent test set for discriminating cancer- versus NDDs-related mutations. We further illustrate its utility in phenotype (cancer/NDDs) prediction by mutation analyses of two protein cases, PI3Kα and PTEN. Compared to seven other predictive tools, protPheMut demonstrated exceptional accuracy in forecasting phenotypic effects, achieving an AUROC of 0.8501 for PI3Kα mutations related to cancer and Cowden syndrome. For multi-phenotype prediction of PTEN mutations related to cancer, PHTS, and HCPS, protPheMut achieved an AUROC of 0.9349 through micro averaging. Using SHAP model explanations, protPheMut highlights the strength of network and dynamic features in deeper uncovering of the effects of pathogenic mutations, thus classifying different disease phenotypes.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40721360/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">In silico prediction of variant effects: promises and limitations for precision plant breeding.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Janek Sendrowski, Thomas Bataillon, Guillaume P Ramstein</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We discuss state-of-the-art machine learning methods for predicting variant effects in plants across both coding and noncoding regions, contrasting supervised approaches in functional genomics with unsupervised methods in comparative genomics.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Sequence-based AI models show great potential for prediction of variant effects at high resolution, but their practical value in plant breeding remains to be confirmed through rigorous validation studies. Plant breeding has traditionally relied on phenotyping to select individuals with desirable traits-a process that is both costly and time-consuming. Increasingly, breeding strategies are shifting toward precision breeding, where causal variants are directly targeted based on their effects. To predict the effects of causal variants, in silico methods are emerging as efficient alternatives or complements to mutagenesis screens. Here, we review state-of-the-art machine learning methods for predicting variant effects in plants across both coding and noncoding regions, contrasting supervised approaches in functional genomics with unsupervised methods in comparative genomics. We discuss challenges in validating predictions, and compare these methods with traditional association and comparative genomics techniques. We argue that modern sequence models extend traditional methods by generalizing across genomic contexts, fitting a unified model across loci rather than a separate model for each locus. In doing so, they address inherent limitations of traditional quantitative and evolutionary comparative genetics techniques. However, the accuracy and generalizability of sequence models heavily depend on the training data, highlighting the need for validation experiments. We point to successful applications of sequence models, especially with protein sequences, and identify areas for further improvement, especially in modeling regulatory sequences. While not yet mature for in silico-driven precision breeding, sequence models show strong potential to become an integral part of the breeder's toolbox.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40719915/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A knowledge-based distance metric highlights underperformance of variant effect predictors on gain-of-function missense variants</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gerasimavicius, L., Marsh, J. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that Extent of Disease Clustering (EDC) and Spatial Proximity to Disease Variants (SPDV) can be used to identify non-loss-of-function (non-LOF) disease variants, which arise due to a gain of function (</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Most current variant effect predictors (VEPs) are less effective at identifying non-loss-of-function (non-LOF) disease variants, which arise due to a gain of function (GOF) or dominant-negative (DN) mechanism. Previously, we showed that Extent of Disease Clustering (EDC), a structure-derived metric, can be used to prioritize putative non-LOF genes due to the tendency of their pathogenic missense variants to cluster. We also introduced a structure-derived spatial distance metric, Spatial Proximity to Disease Variants (SPDV), which proved useful for scoring RyR1 variants. Here, we apply EDC and SPDV across thousands of human disease genes. In comparison with 72 variant effect predictors (VEPs), SPDV exceeds or is at least as effective as half of the tested models at identifying dominant disease variants. When applied to GOF variants, SPDV performs similarly to top VEPs. We show that SPDV performance improves with increasing target EDC values and identify an EDC threshold after which SPDV consistently surpasses all other predictors. Its use as an analytical metric enables the identification of a subset of putative GOF disease genes where current VEP methods underperform relative to SPDV, indicating they may be improved by incorporating spatial distance information.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.23.666325v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PInteract: Detecting Aromatic-Involving Motifs in Proteins and Protein-Nucleic Acid Complexes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Li, D., Pucci, F., Rooman, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present PInteract, an algorithm that detects non-covalent interactions in input structures based on geometric criteria, including {pi}-{pi}, cation-{PI}, amino-{Pi}, His-{ pi}, and sulfur-{2}.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> With the recent development of accurate protein structure prediction tools, virtually all protein sequences now have an experimental or a modeled structure. It has therefore become essential to develop fast algorithms capable of detecting non-covalent interactions not only within proteins but also in protein-protein, protein-DNA, protein-RNA, and protein-ligand complexes. Interactions involving aromatic compounds, particularly their {pi} molecular orbitals, hold unique significance among molecular interactions due to the electron density delocalization, which is known to play a key role in processes such as protein aggregation. In this paper, we present PInteract, an algorithm that detects {pi}-involving interactions in input structures based on geometric criteria, including {pi}-{pi}, cation-{pi}, amino-{pi}, His-{pi}, and sulfur-{pi} interactions. In addition, it is capable of detecting chains and clusters of {pi} interactions as well as particular recurrent motifs at protein-DNA and protein-RNA interfaces, called stair motifs, consisting of a particular combination of {pi}-{pi} stacking, cation/amino/His-{pi} and H-bond interactions. PInteract is freely available for download in our repository https://github.com/3BioCompBio/PInteract.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.23.666328v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AlignmentFold and AlignmentPartition: Improving the align-then-fold approach for RNA secondary structure prediction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mittal, A., Mathews, D. H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a new free energy minimization method for structure prediction using a fixed alignment of homologous RNA sequences and improve the prediction accuracy by using sequence covariation analysis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The combination of free energy minimization with sequence covariation analysis is a popular approach for predicting conserved secondary structure using a fixed alignment of homologous RNA sequences. In this work we revisit this approach by developing the new AlignmentFold and AlignmentPartition programs. AlignmentFold and AlignmentPartition predict consensus minimum free energy structures and base pairing probabilities, respectively. We determined new nearest neighbor thermodynamic parameters for gaps and non-canonical base pairs that align with base pairs. These frequently occur for structure prediction with a fixed input alignment. We assessed the impact of alignment prediction method and size on the prediction accuracy. We also assessed the improvement in prediction accuracy conferred by sequence covariation analysis. In contrast to previous solutions, we do not include covariation in the scoring of structure quality and achieve prediction accuracy as good as other tools. AlignmentFold and AlignmentPartition are freely available as part of the RNAstructure software package at https://rna.urmc.rochester.edu/RNAstructure.html .</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.23.666478v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Barley BODYGUARD controls cuticular specialisations regulated by SHINE transcription factors</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> McAllister, T., Campoli, C., Liu, L., Chia, T., Fisher, S. R., Prescott, A. R., Horsnell, R., Shoesmith, J., Eskan, M., Iredale, A., Nuter, M., Ramsay, L., Bayer, M. M., Schreiber, M., Milne, L., Wahl, V., Waugh, R., Cockram, J., Mckim, S. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify the barley BODYGUARD1 (HvBDG1) gene encoding an alpha/beta-hydrolase essential for leaf cuticular integrity and wax bloom deposition, and show that NUD and HvWIN1 are essential for strong hull to caryopsis adhesion.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The outer epidermis of land plants secretes a cuticular layer, a hydrophobic diffusion barrier which minimises water loss into the atmosphere and protects from pests, ultraviolet light and organ fusion. Cuticles typically comprise a polyester cutin matrix embedded and overlaid with cuticular waxes, but their exact chemical make-up, structure and functions can vary widely depending on the tissue and species. Barley shows two such cuticular specialisations: (1) deposition of a thick beta-diketone-rich wax bloom on multiple organs at reproductive stage, common in other Poeceae species and linked to yield; and, (2) secretion of a sticky layer on the grain fruit (caryopsis) pericarp cuticle which adheres to inner floral hulls, leading to the distinctive covered barley grain used in animal feed and malting. Two SHINE/WAX-INDUCER transcription factors in barley, HvWIN1 and NUD, promote the wax bloom and hull to caryopsis adhesion, respectively, yet little is understood about other genes involved. Leveraging near-isogenic lines of wax-deficient mutants, we identify the barley BODYGUARD1 (HvBDG1) gene encoding an alpha/beta-hydrolase essential for leaf cuticular integrity and wax bloom deposition. Modelling of functional and defective alleles suggests that HvBDG1 N-terminal region control of protein flexibility is important for HvBDG1 function. In addition to their role in controlling barley epicuticular wax deposition, we show that both HvBDG1 and HvWIN1 are essential for strong hull to caryopsis adhesion. Along with NUD, these gene products differentially contribute to ultrastructural changes on the pericarp associated with a cuticular building programme driven by NUD and HvWIN1 regulation of cuticle metabolism and transport and cell wall-related genes, and correlate with shifts in pericarp surface chemistry. We also show that the previously asserted grain-specific role of NUD should be revised, as our findings reveal that it is essential for maintaining leaf cuticle integrity. Our analyses in barley suggest that NUD and HvWIN1 control cuticular specialisations and cuticle integrity in part via promotion of HvBDG1 expression, while HvWIN1 and NUD likely act independently from each other. Lastly, mining tetraploid wheat mutant populations followed by crossing to combine mutated homoeologues demonstrated that BDG1 and WIN1 orthologues also control wax bloom in wheat. Taken together, our work greatly expands the genetic networks and molecular activities important for cuticle development in cereals and the underlying mechanisms for both shared and species-specific cuticular specialisations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.28.666421v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The chloroplast ionome shines new light on organellar Fe homeostasis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Holzner, L. J., Oestergaard Frank, L., Muehlbauer, S., Mueller, A., Schroeder, L., DeTar, R. A., Philippar, K., Mendoza-Cozatl, D., Kaemer, U., Naegele, T., Boelter, B., Kunz, H.-H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that chloroplasts can be turned into large iron storages and show that plastid ionomes can be genetically manipulated.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Annually, chloroplasts fix 258 billion tons of CO2 through photosynthesis. Photosynthesis and other biochemical pathways require specific amounts of metal ions in the organelle. Transport proteins in the plastid inner envelope maintain the organellar ion homeostasis. Despite substantial progress over the last decades, many genes encoding for plastid ion channels and ion carriers or their regulators remain unknown. To fill this knowledge gap, detailed information on the elemental composition of chloroplasts i.e., a plastid ionome, is needed. This will allow to compare mutants of transporter candidates with wild-types. Here, we provide quantitative descriptions of chloroplast ionomes from Arabidopsis thaliana, the metal hyperaccumulator Arabidopsis halleri, Pisum sativum, and Nicotiana benthamiana and analyze similarities and distinctions. Using A. thaliana, we show that plastid ionomes can be genetically manipulated. Chloroplasts of oligopeptide transporter3 (opt3)-deficient mutants contain 14-fold more iron, which they deposit into stromal FERRITIN. The removal of FERRITIN in opt3 mutants leads to a substantial decrease in plastid and leaf iron pointing to important signaling linked to the chloroplast ionome. Our study reveals that chloroplasts can be turned into large iron storages. Since crop biofortification to fight hidden hunger has become a global mission, this research provides groundwork to reach this goal.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.28.667122v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Activation-Free Upgrading of Carboxylic Acids to Aldehydes and Alcohols</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Black, W. B., Saleh, S., Perea, S., Luu, E., Cui, Y., Sun, J., Wang, Z., Lambrecht, S., Awachi, S., Hayworth, D., Wang, A., Chuayiuso, C., Hagerty, R., Gilcrease, P. C., Jiao, F., He, Z., Siegel, J. B., Li, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that aldehyde dehydrogenases can directly reduce carboxylic acids to aldehydes without prior activation, a process previously thought to be biologically inaccessible.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Advances in organic and gas waste valorization have enabled high-yield production of carboxylic acids, positioning them as promising feedstocks for the bioeconomy. However, carboxylic acids must be activated before downstream use, typically requiring ATP, CoA, or reduced ferredoxin to overcome unfavorable thermodynamics. These activators are costly to generate and divert carboxylic acids into CO2-releasing pathways, reducing carbon efficiency. Here, we demonstrate that aldehyde dehydrogenases (ALDHs) can directly reduce carboxylic acids to aldehydes without prior activation, a process previously thought to be biologically inaccessible. Screening 133 ALDHs revealed that this activity is remarkably widespread within the protein family, enabling production of aliphatic aldehydes and alcohols, diols, and aromatic alcohols, at titers >1 g/L, in some cases, after optimization of thermodynamic driving forces. Additionally, we applied this system to upgrade waste-derived carboxylic acid effluent streams from wastewater sludge, food waste, and waste gas (CO2). This activation-free process, termed ''reverse aldehyde oxidation'' (rAOX), establishes a broadly applicable, energy-efficient platform for carboxylic acid valorization at 100% carbon yield. Analogous to the reverse tricarboxylic acid cycle (rTCA) and reverse {beta}-oxidation (rBOX), rAOX exemplifies that metabolic reactions classically defined as unidirectional may have unexpected plasticity to operate in reverse and open new avenues in biomanufacturing.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.28.667276v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

