<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Monday, 23 June 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">JASMINE: A powerful representation learning method for enhanced analysis of incomplete multi-omics data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ballard, J. L., Dai, Z., Shen, L., Long, Q.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose JASMINE, a self-supervised representation learning method for incomplete multi-omics data that preserves both modality-specific and joint information and enhances sample similarity structure.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Integrative analysis of multi-omics data provides a more comprehensive and nuanced view of a subject's biological state. However, high-dimensionality and ubiquitous modality missingness present significant analytical challenges. Existing methods for incomplete multi-omics data are scarce, do not fully leverage both modality-specific and shared information, and produce task-biased representations. We propose JASMINE, a self-supervised representation learning method for incomplete multi-omics data that preserves both modality-specific and joint information and enhances sample similarity structure. JASMINE produces embeddings that achieve superior performance across multiple tasks for two different incomplete multi-omics datasets while requiring only a single round of training per dataset.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.16.659949v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Cell-Type Specific Single-Cell Signatures Reveal Nephrotoxic Drug Effects</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kuchi, A. S., Acitores Cortina, J. M., Liu, H., Fatapour, Y., Berkowitz, J., Tatonetti, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a single cell model of acute kidney injury and show that only particular subtypes of kidney cells may be responsible for observed nephrotoxicity and explain prediction challenges.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Drug-induced acute kidney injury (AKI) affects about 20% of hospitalized AKI patients, a significant contributor to morbidity and mortality. The lack of understanding of the kidney system and functioning of nephrotoxic drugs contributes to hospital-acquired AKI cases. AKI is difficult to predict because of its complex injury mechanism and the numerous pathways through which it manifests. Traditional toxicity biomarkers, like elevated creatinine levels, detect AKI only after significant kidney injury has occurred. Concurrently, advancements in single cell RNA sequencing (scRNAseq) have improved our ability to map cellular heterogeneity within tissues, potentially enabling the study of drug effects at a single cell level. We hypothesized that only particular subtypes of kidney cells may be responsible for observed nephrotoxicity and explain prediction challenges. To test this, we generated cellular response scores for 32 kidney cell types from the Human Cell Atlas and estimated drug effects. We identified significant expression differences in 6 cell types (e.g. Indistinct intercalated cell p = 0.009, Epithelial Progenitor cell, p = 0.04). We also developed an ensemble model that achieved an AUROC of 0.6 across different kidney cell populations - a significant improvement over using traditional bulk RNA sequencing alone. The single-cell transcriptomic signatures we identified potentially reveal unexplained molecular mechanisms of nephrotoxicity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.17.660070v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Learning Human T Cell Behaviors through Generative AI Embeddings of T Cell Receptors</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chen, D. G., Su, Y., Heath, J. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> T cell receptors are distinct with divergent physiochemical properties, and Tarpon is a computational framework to create interpretable TCR embeddings, via physicochemical associations, that have broad implications for the field.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> T cells interact with the world through T cell receptors (TCRs). The extent to which TCRs determine T cell behavior has not been comprehensively characterized. Our Tarpon model leverages advances in generative artificial intelligence to synthesize large-scale (>1M sequences) TCR atlases across human development and diseases into actionable insights. Tarpon creates: 1) bespoke sampling functions generating realistic Ag-specific TCRs, 2) embeddings revealing CD4+ and CD8+ single-positive TCR repertoires as distinct with divergent physiochemical properties, and 3) cross-dataset mappings of T cell states that validate fetal CD4+ versus CD8+ TCR differences in adults and find fetal type I innate T cells to map to MAIT and KIR+ adult CD8+ T cells which we verify via whole transcriptome analysis. Tarpon is a resource as a reference of TCRs across human physiological states and as a computational framework to create interpretable TCR embeddings, via physicochemical associations, that have broad implications for the field.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.16.660016v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Count your bits: more subtle similarity measures using larger radius count vectors</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Huber, F., Pollmann, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of 2D fingerprinting and similarity metrics in cheminformatics, highlighting critical issues such as fingerprint duplication, mass-dependent score biases, and high bit collision rates.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Quantifying molecular similarity is a cornerstone of cheminformatics, underpinning applications from virtual screening to chemical space visualization. A wide range of molecular fingerprints and similarity metrics, most notably Tanimoto scores, are employed, but their effectiveness is highly context-dependent. In this study, we systematically evaluate several 2D fingerprint types, including circular, path-based, and distance-encoded variants, using both binary and count representations. We highlight the consequences of fingerprint choice, vector folding, and similarity metric selection, revealing critical issues such as fingerprint duplication, mass-dependent score biases, and high bit collision rates. Sparse and count-based fingerprints consistently outperform fixed-size binary vectors in preserving structural distinctions. Furthermore, we introduce percentile-based normalization, propose inverse-document-frequency (IDF) weighting, and benchmark all methods against graph-based MCES similarities. Our results offer practical guidance for selecting molecular similarity measures, emphasizing the need for conscious, task-aware fingerprinting choices in large-scale chemical analyses.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.16.659994v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The chromatin remodeler LET-418/Mi-2 regulates the intracellular pathogen response in the C. elegans intestine</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rajopadhye, S. A., Lazetic, V., Rodriguez Crespo, D., Troemel, E., Meister, P., Wicky, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A chromatin remodeler that is involved in the balance between growth and defense mechanisms and a modulator of the intracellular pathogen response .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Chromatin remodeling provides essential transcriptional regulation for all biological processes. In Caenorhabditis elegans, the chromatin remodeler LET-418, a homolog of the human Mi-2 protein, plays a critical role in regulating development, organogenesis, tissue maintenance, stress resistance and lifespan. LET-418 is part of several chromatin remodeling complexes and contributes significantly to the balance between growth and defense mechanisms, yet its target genes remain unclear. Using DNA methylation profiling, we identified genomic binding sites and associated target genes of LET-418 and its MEC-complex-specific interactor MEP-1 in the intestine. Consistent with their presence in the same complex, the two proteins shared more than half of their target genes. Functional analysis revealed that LET-418 and MEP-1 target genes are highly active in the intestine and are involved in repressing innate immune responses, including the intracellular pathogen response (IPR). Consistently, in let-418 mutants, IPR-induced genes, such as pals-5 or pals-2 are strongly upregulated, in a manner dependent on ZIP-1, a major transcription factor for IPR. Additionally, we found pathogen levels of the natural intracellular intestinal pathogen Nematocida parisii significantly reduced in let-418 mutants, supporting the observation of increased IPR in this mutant. Altogether, these findings reveal a crucial role for LET-418 as a modulator of the IPR, aligning with its role in maintaining the balance between development and defense.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.17.659900v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Phytochemical Content and Antimicrobial Activity of Aloe Vera extract of Marathwada Region, India</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Barik, S. K., Nagargoje, M., Chaudhari, N., Harke, S. N., Khemnar, A. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Ethanol is a more efficient solvent for extracting phytochemicals from Aloe vera than methanol for the extraction of bioactive compounds, and the study indicates the significance of solvent selection in optimizing the therapeutic utility of Aloe barbadensis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Aloe barbadensis is widely recognized for its therapeutic properties due to its rich phytochemical composition, including proteins, carbohydrates, and steroids. The study aimed to evaluate the efficiency of solvents such as ethanol and methanol for extracting bioactive compounds and to analyse the phytochemical contents of Aloe vera extracts using various biochemical assays. The dinitro salicylic acid test quantified reducing sugars with ethanol extracts yielding 870 mg/ml compared to 450 mg/ml for methanol extracts. The Biuret test for proteins revealed concentrations of 13.8 mg/ml in ethanol extracts and 13.03 mg/ml in methanol extracts. The Barfoeds test indicated carbohydrate concentrations of 960 mg/ml in ethanol extracts. A qualitative test for steroids confirmed their presence in both solvents, with ethanol showing a more intense reaction, suggesting a higher concentration of steroidal compounds. The results establish ethanol as a more efficient solvent for extracting phytochemicals from Aloe vera. Antimicrobial activity test against E. coli demonstrated superior inhibition zones for ethanol extract, highlighting their potential as natural antibacterial agents. The research indicates the significance of solvent selection in optimizing the therapeutic utility of Aloe vera. It provides a foundation for further exploration into its bioactive compounds and mode of action.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.17.660060v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Molecular Bonsai: Elucidating the design principles for engineering plant organ size</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bull, T. A., Van Blair, J., Sutton, S., Colwell, H., Khakhar, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We explore if mathematical modeling can guide SynTF-based expression modulation of genes in the signaling pathway of the phytohormone, gibberellin (GA), which is a central regulator of cell expansion, to elucidate the design principles for engineering organ size.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Enhancements to crop morphology, such as the semi-dwarfing that helped drive the green revolution, are often driven by changes in gene dosage. These changes are challenging to translate across varieties and species, which slows the pace of crop improvement. Synthetic transcription factors (SynTFs) offer a rapid alternative to generate targeted alterations to gene dosage. However, the complexity of developmental pathways makes it unclear how to best apply them to predictably engineer morphology. In this work, we explore if mathematical modeling can guide SynTF-based expression modulation of genes in the signaling pathway of the phytohormone, gibberellin (GA), which is a central regulator of cell expansion, to elucidate the design principles for engineering organ size. We demonstrate that modulation of GA signaling gene expression can generate consistent dwarfing across tissues in both controlled and variable environments in the model plant Arabidopsis thaliana, and that the degree of dwarfing is dependent on the strength of regulation as predicted by modeling. We further validate the predictive power of the model by demonstrating its capacity to accurately predict the qualitative impacts of different regulatory architectures for both increasing and decreasing organ size. Finally, we show that these insights can be generalized for engineering organ size in the crop Solanum lycopersicum (tomato). This work creates a framework for predictable engineering of an agriculturally important trait, organ size, across tissues and plant species. It also serves as a proof-of-concept for how mathematical models can guide SynTF-based alterations in gene dosage to enable bottom-up design of plant phenotypes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.22.660808v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Structure-guided design of protein attachment points for functional augmentation of complex molecular machines</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sorensen, G. H., James, J., Steel, H. E., Parmeggiani, F., Gorochowski, T. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A computational workflow that supports the design of optimal insertion sites for protein-protein conjugation tags based on the structure of the proteins involved and performs localised residue redesign where needed.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein-protein conjugation systems are a powerful way of creating fusion proteins and enable the dynamic combination of protein domains with diverse functionalities. However, the insertion of these systems into enzymes is often performed with little consideration of the structural impact they might have. This is particularly relevant when modifying complex molecular machines that transition between numerous conformational states. Here, we address this issue by developing SIMPLIFE, a computational workflow that supports the design of optimal insertion sites for conjugation tags based on the structure of the proteins involved and performs localised residue redesign where needed. We demonstrate how SIMPLIFE can be used to effectively augment the function of T7 RNA polymerase using the DogCatcher-DogTag system, enabling diverse and dynamically varying mutations within a targeted region of DNA. This work demonstrates the power of combining biophysical and machine learning based approaches for protein structure prediction to efficiently augment the function of molecular machines, accelerating our ability to combine complex biochemical functionalities in new ways.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.22.660945v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An M13 phagemid toolbox for engineering tuneable DNA communication in bacterial consortia</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pujar, A., Sharma, A., Jbara, H., Kushwaha, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a library of five phagemid variants with distinct replication origins, including those based on the Standard European Vector Architecture (SEVA) family, designed to tune the growth and secretion dynamics of sender strains.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Intercellular communication is essential for distributed genetic circuits operating across cells in multicellular consortia. While diverse signalling molecules have been employed--ranging from quorum sensing signals, secondary metabolites, and pheromones to peptides, and nucleic acids--phage-packaged DNA offers a highly programmable method for communicating information between cells. Here, we present a library of five M13 phagemid variants with distinct replication origins, including those based on the Standard European Vector Architecture (SEVA) family, designed to tune the growth and secretion dynamics of sender strains. We systematically characterize how intracellular phagemid copy number varies with cellular growth physiology and how this, in turn, affects phage secretion rates. In co-cultures, these dynamics influence resource competition and modulate communication outcomes between sender and receiver cells. Leveraging the intercellular CRISPR interference (i-CRISPRi) system, we quantify phagemid transfer frequencies and identify rapid-transfer variants that enable efficient, low-burden communication. The phagemid toolbox developed here expands the repertoire of available phagemids for DNA-payload delivery applications and for implementing intercellular communication in multicellular circuits.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.22.660937v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

