<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Thursday, 12 June 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Machine learning and bioinformatics analysis to identify and validate diagnostic model associated with immune infiltration in rheumatoid arthritis.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jiayang Jin, Xiaohong Xiang, Xuanlin Cai, Yuke Hou, Zhaoqi Zhang, Jing Li</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> This study focuses on identifying immune infiltration-related diagnostic biomarkers of RA patients.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Rheumatoid arthritis (RA) is a chronic autoinflammatory condition that can result in significant disability. This study focuses on identifying immune infiltration-related diagnostic biomarkers of RA patients.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40500570/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Predicting cell type-specific coverage profiles from DNA sequence</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Linder, J., Yuan, H., Kelley, D. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We extend the capabilities of our recently developed Borzoi model by training on single-cell 3'-seq expression profiles from the Tabula Sapiens, Tabula Muris, and the Adult Brain Atlas, aggregated by cell type.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Predicting expression profiles from RNA-seq experiments provides a powerful approach for universal sequence-based variant effect prediction, enabling researchers to score variants that affect total gene expression and relative isoform abundances. These models can be repurposed for new prediction tasks through transfer learning. However, current base models train primarily on bulk RNA-seq profiles derived from tissues and cell lines, overlooking the wealth of single-cell 3'-seq data that captures cell type-specific gene regulation. Here, we extend the capabilities of our recently developed Borzoi model by training on single-cell 3'-seq expression profiles from the Tabula Sapiens, Tabula Muris, and the Adult Brain Atlas, aggregated by cell type. This new model, Borzoi Prime, enables accurate variant interpretation across diverse cells, spanning erythrocytes to microglia. Training on 3'-seq profiles improves the model's ability to predict cell- and tissue-specific alternative polyadenylation, even in the original bulk RNA-seq data. Through UTR-wide mutagenesis experiments of alternatively polyadenylated genes, we highlight determinants of cell type-specific 3' UTR regulation learned by the model. This cell type-resolved approach opens new possibilities for understanding genetic variant effects via multiple layers of regulation in specific cellular contexts.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.10.658961v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identification of Novel Marine-based Inhibitors against Tetracycline Destructase in Acinetobacter Baumannii using Computational Approaches</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Marimuthu, S. K., Chockalingam, P., Nagarajan, V., Subbiah, T., Ramakrishnan, V.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify novel inhibitors for tetracycline destructase in Acinetobacter Baumannii, utilizing compounds derived from marine environments, which hold promise for the development of novel and efficacious antibiotics against bacterial infections.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Tetracyclines are indispensable antibiotics employed to treat a broad spectrum of bacterial infections. However, the emergence of clinical pathogens exhibiting significant resistance to these drugs has posed a formidable challenge in managing bacterial diseases effectively. This resistance is attributed to the proliferation and diversity of resistance genes and various mechanisms that render tetracyclines inactive. A prevalent mechanism of resistance is enzymatic inactivation by tetracycline destructases, which compromises the efficacy of tetracyclines. Consequently, there is a pressing need to discover novel molecules capable of inhibiting tetracycline destructases activity and thereby enhancing the drug's potency. In this study, we identify novel inhibitors for tetracycline destructase (TDase) in Acinetobacter Baumannii, utilizing compounds derived from marine environments. Through a high-throughput virtual screening approach, we have investigated the potent natural marine compounds interaction and dynamic behaviour of tetracycline destructase through molecular simulations. These findings hold promise for the development of novel and efficacious therapeutics against bacterial infections.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.08.658539v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Targeting Listeria virulence-regulon glutathion-cavity by evolutecas</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Coll, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We generate small peptide and acetic-polymer evolutecas targeting one of the virulence regulons of Listeria monocitogenes (Lm).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Computational explorations are described generating small peptide and acetic-polymer evolutecas targeting one of the virulence regulons of Listeria monocitogenes (Lm). Lm are among the most important multiple-resistant food-borne infective bacteria whose antibiotic resistances raise worldwide concerns. The glutathion cavity of the Lm positive regulatory factor A (prfA) virulence regulon, was computationally targeted here because of the importance of the glutathion tri-peptide-dependent prfA activation of many virulent Lm genes. Computationally screening of alternative tri-,tetra-, and penta-mer peptides and innovative-acetic polymers and evolutecas generated from the top-screened molecules, were employed to optimize their fitting to glutathion-prfA cavities. Combining library-screening and children generation by co-evolution, several novel top-peptides and acetic-derived polymers were generated predicting low-toxicities (to reduce any known undesirable side-effects) and low nanoMolar consensus affinities (to maximize their specificities and accuracies). However, their possible inhibition of prfA virulence remains to be demonstrated. To continue these explorations, additional chemical synthesis and in vitro experimental validation, are strictly required to demonstrate any desirable interference with Lm virulence and/or biological functions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.09.658548v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Random survival forests identify myocardial gene signatures associated with survival in heart failure with preserved ejection fraction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kannan, S., Knutsdottir, H., Vaishnav, J., Bader, J. S., Kass, D. A., Sharma, K., Hahn, V. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified 33 genes associated with event-free survival in HFpEF, including genes previously implicated in heart failure, including ADAMTSL2, ADRB1, BMP6, and METRNL.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Heart failure with preserved ejection fraction (HFpEF) continues to be poorly understood at the molecular level. While previous studies have identified gene expression signatures unique to HFpEF, genes associated with clinical decompensation have not been determined. Here, we performed exploratory analysis of myocardial RNA-seq data to identify genes associated with event-free survival in HFpEF. We analyzed previously published RNA sequencing data of right ventricular septal endomyocardial biopsies from HFpEF patients (n=41) with paired clinical, echocardiographic, and outcome data (including mortality and heart failure hospitalizations). We constructed random survival forests with forward stepwise regularization (the "variable hunting" method) to determine genes associated with time to first event using a combined end-point of heart failure hospitalization or all-cause death. Selection of candidate forest variables was tested with both random and weighted sampling methods. We identified 33 genes that are predictive of survival in HFpEF. This set includes genes previously implicated in heart failure, including ADAMTSL2, ADRB1, BMP6, and METRNL. Survival forests constructed using these genes outperform those constructed from clinical and hemodynamic parameters alone (out-of-bag C-index 0.894 vs 0.545). Moreover, in survival forests constructed from both gene data and hemodynamic measurements, individual survival genes consistently showed higher variable importance than clinical/hemodynamic parameters. Our study shows that random survival forests identify myocardial gene signatures that may better model HFpEF prognosis than clinical measurements. Further studies are warranted to validate findings in independent cohorts.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.08.658488v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Better antibodies engineered with a GLIMPSE of human data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hepler, N. L., Hill, A. J., Jaffe, D. B., Gibbons, M. C., Pfeiffer, K. A., Hilton, D. M., Freeman, M., McDonnell, W. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A protein language model trained solely on paired human antibody sequences that can be used for humanization and development of antibodies.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> GLIMPSE-1 is a protein language model trained solely on paired human antibody sequences. It captures immunological features and achieves best-in-class performance in humanization benchmarks. We demonstrate the utility of GLIMPSE-1 in humanization; engineering of antibodies for affinity, species cross-reactivity, and key developability parameters; and the creation of highly divergent functional variants with <90% sequence identity to a marketed antibody. Learning exclusively from human antibody data enables GLIMPSE-1 to enhance therapeutics and native antibodies based on patterns in the human repertoire.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.08.658113v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Distinct Transcriptomic Biomarkers and Pathways Associated with Cardiovascular and Neurovascular Dysregulation in Long COVID-19 Brain Fog</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ashfaque, O., Kumar, H., Mazumder, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify distinct biomarkers and dysregulated pathways in long COVID-19 patients with brain fog, with a focus on cardiovascular and neurovascular involvement, suggesting novel biological mechanisms that contribute to cognitive impairment.</p><p style="font-size: 16px; text-align: justify;">Background: Long COVID-19 often manifests as persistent cognitive impairments, commonly referred to as brain fog, with poorly understood biological mechanisms. Previous studies highlighted neuroinflammation and blood-brain barrier disruption. Here, we identify distinct biomarkers and dysregulated pathways, with a focus on cardiovascular and neurovascular involvement. Methods: We analyzed public RNA-sequencing data (GEO: GSE251849) comparing long COVID-19 patients with brain fog, long COVID-19 patients without cognitive symptoms, convalescent patients, and healthy controls. Differential gene expression, principal component analysis, and pathway enrichment analyses were performed using DESeq2, KEGG, GO, and Enrichr tools. Results: Our analysis revealed a distinct gene expression profile in patients with brain fog, identifying exclusive biomarkers such as GRIN2C, NOS2, and EDNRB. Pathway analysis revealed significant dysregulation in cardiovascular, calcium signaling, and relaxin signaling pathways, suggesting novel biological mechanisms that contribute to cognitive impairment. These findings expand beyond neuroinflammation and highlight potential cardiovascular involvement in long COVID-related cognitive dysfunction. Conclusion: Our study reveals novel transcriptomic signatures that highlight cardiovascular and neurovascular dysregulation, providing promising diagnostic biomarkers and therapeutic targets. Future research should validate these findings and investigate therapeutic interventions addressing these unique pathways.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.10.656785v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SARS-CoV-2-induced dysregulation in ADAR editing patterns persists post viral clearance in individuals with mild COVID-19</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nair, A. M., Piontkivska, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We explored temporal changes in ADAR expression and editing patterns, across three distinct stages of SARS-CoV-2 infection, and examined whether infection-triggered dysregulation in ADar editing persists or returns to pre-infection states post-viral clearance.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infections remain a public health concern worldwide. Viral antigen triggered innate immune response leads to induction of interferons (IFNs) and interferon stimulated genes (ISGs) including ADAR1 p150 isoform, which edits adenosine (A) residues within double stranded RNAs in both the virus and the host. Such RNA editing mediated by ADARs plays a crucial role in innate immune responses during viral infections through modulation of host-virus interactions. Additionally, ADAR editing acts post-transcriptionally, and serves as a mechanism of dynamic regulation of transcriptome and proteome diversity. While evidence points to changes in ADAR editing during infection, we do not know whether editing targets change over the course of the infection. Here, we explored temporal changes in ADAR expression and editing patterns, across three distinct stages of SARS-CoV-2 infection. Furthermore, we examined whether infection-triggered dysregulation in ADAR editing persists or returns to pre-infection states post-viral clearance. We addressed this question by analyzing publicly available whole blood RNA sequencing samples from forty-five, age-matched individuals. The individuals selected had no documented comorbidities, developed mild COVID-19, and were sampled across three distinct stages of SARS-CoV-2 infection: pre-, mid-, and post-infection. Our results demonstrate dynamic changes in ADAR expression and editing across the three stages. We further identified unique editing sites resulting from SARS-CoV-2 infection, across all three stages of infection, within genes involved in immune response pathways. Noteworthy, genes within neutrophil degranulation pathway appear to be edited, suggesting they may play a role in inflammation and sequelae observed post-SARS-CoV-2 infection. Our results demonstrate a consistent trend of elevated ADAR expression and reduced overall ADAR editing within each individual mid-infection. Subsequently, in some post-infection samples ADAR expression returns to approximately pre-infection levels, while in others it remains dysregulated. These differences may be contributing to heterogeneity in disease outcomes seen in individuals post-SARS-CoV-2 infection.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.09.658648v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Optimization of Genome-wide CRISPR Screens using dual-guide RNA Infection with Cas9 Electroporation (DICE)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Stender, J. D., Purman, C., Lu, C., Modi, A., Vijaykumar, V., Flister, M., den Hollander, A., Kadri, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce dual-guide RNA infection with Cas9 protein electroporation, which expresses two guides from the same lentiviral construct that target the same gene.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-guide RNA lentiviral infection with Cas9 protein electroporation (SLICE) enables CRISPR screening in primary cell types that require transient Cas9 expression yet is limited by scalability and robustness. Here, we introduce dual-guide RNA infection with Cas9 electroporation (DICE), which expresses two guides from the same lentiviral construct that target the same gene. In genome-wide screens, DICE outperformed SLICE in defining essential genes and modulators of PD-L1 expression in IFN gamma activated THP1 cells. Collectively, these data demonstrate that DICE can be utilized for reduced-scale CRISPR screens in cell types with transient Cas9 protein expression without sacrificing screening quality.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.09.658313v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Insilico investigation of novel Plasmodium Falciparum Glycogen Synthase Kinase(pfGSk3Œ≤) inhibitors for the treatment of malaria infection</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Adebambo, K. F., Ottify, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose three new drug molecules that can be used to inhibit pfGSK3{beta}, of all these three molecules compound S56 was found to behave better insilico than S1- (3,6-diamino-4-chlorophenyl)thieno[</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Malaria, a parasitic disease, remains a global health burden with over 260 million cases worldwide in 2023. With the increasing resistance to existing antimalarials, there is an increasing demand in the continuous search for novel therapeutic targets and strategies. Glycogen synthase kinase (pfGSK{beta}) is a key enzyme involved in metabolic processes such as glycogen synthase phosphorylation, cell differentiation and cell proliferation of the malaria parasite. An attempt to exploit this enzyme as a potential target for drug repurposing was investigated. The insilico method investigated in this research work involved the use of Python coding to mine data from the CHEMBL database for possible GSK3{beta} inhibitors. The data mining using Python coding returned 53 potential molecules ranked according to their 1C50 value. The in silico analysis of these compounds using molecular docking, molecular dynamics simulations showed that three new molecules, labelled S20 -CHEMBL ID 1910196 (4-[5-(6-hydroxy-1H-indol-2-yl)pyridin-3-yl]benzonitrile), S39- CHEMBL ID 2321945- (2-(7-bromo-2-hydroxy-1H-indol-3-yl)-3-oxoindole-6-carboxylic acid) and S56-CHEMBL ID 2321951 (methyl 2-(2-hydroxy-1H-indol-3-yl)-3-nitroso-1H-indole-5-carboxylate), are potential molecules that can be used to inhibit pfGSK3{beta}, of all these three molecules compound S56 were found to behave better insilico than S1- (3,6-diamino-4-(2-chlorophenyl)thieno[2,3-b]pyridine-2,5-dicarbonitrile)-the co-crystallised ligand in pfGSK3{beta} which was used as a control for the insilico drug repurposing research because it has already been tested to inhibit pfGSK3{beta}, Finally, Protox III-an online webserver tool for insilico toxicity study was used to give the potential toxicity of the molecules, this will serve as a tool to guide further research in toxicity of the repurposed molecules. In conclusion, we have been able to propose that three new drug molecules, S39, S20 and S56, have the potential of being repurposed for the treatment of Malaria infection.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.07.658456v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Gender-specific MetS prediction using pathophysiological determinants: Beyond diagnostic constraints</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Desai, K., Kundu, I., Verma, V., S, A. M., Sudhakar, M., Menon, J., Idicula-Thomas, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A multivariate logistic regression model for MetS progression based on three pathophysiological determinants of inflammation, endothelial dysfunction, and hepatic dysregulation .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Metabolic syndrome (MetS), constituted by obesity, hyperglycemia, hypertension, and dyslipidemia, is a growing public health concern due to its associated risk with cardiovascular and other metabolic disorders. Early-stage detection and prevention of MetS are key factors in an effective management strategy. Thus, the current study aimed to develop a risk score for MetS progression based on features pertaining to its pathophysiology, that can facilitate early detection and treatment strategies. A multivariate logistic regression model was developed using a representative feature from MetS pathophysiological pathways of inflammation, endothelial dysfunction, and hepatic dysregulation. The model was built on NHANES dataset and validated in Chinese and Indian datasets. The model performance evaluated using ROC resulted in an AUC of 0.81 in training and 0.87 and 0.79 for Chinese and Indian validation datasets, respectively. In conclusion, a MetS predictive model built on three MetS pathophysiological determinants was developed and validated in distinct datasets.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.10.658812v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Undermining the cry for help: The phytopathogenic fungus Verticillium dahliae secretes an antimicrobial effector protein to undermine host recruitment of antagonistic Pseudomonas bacteria</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kraege, A., Punt, W., Doddi, A., Zhu, J., Schmitz, N., Snelders, N., Thomma, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that vascular wilt fungus Verticillium dahliae secretes Av2 to suppress the cry for help by inhibiting the recruitment of antagonistic Pseudomonas spp. to pave the way for successful plant invasion.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> During pathogen attack, plants recruit beneficial microbes in a cry for help to mitigate disease development. Simultaneously, pathogens secrete effectors to promote host colonization through various mechanisms, including targeted host microbiota manipulation. Here, we characterize the Av2 effector of the vascular wilt fungus Verticillium dahliae as a suppressor of the cry for help. Inspired by in silico antimicrobial activity prediction, we investigated the antimicrobial activity of Av2 in vitro. Furthermore, its role in V. dahliae virulence was assessed through microbiota sequencing of inoculated plants, microbial co-cultivation assays, and inoculations in a gnotobiotic plant cultivation system. We show that Av2 inhibits bacterial growth, and acts as a virulence factor during host colonization. Microbiota sequencing revealed involvement of Av2 in suppression of Pseudomonas spp. recruitment upon plant inoculation with V. dahliae, suggesting that Av2 suppresses the cry for help. We show that several Pseudomonas spp. are antagonistic to V. dahliae and sensitive to Av2 treatment. We conclude that V. dahliae secretes Av2 to suppress the cry for help by inhibiting the recruitment of antagonistic Pseudomonas spp. to pave the way for successful plant invasion.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.09.658588v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Heterologous expression of microbial nitroreductases for TNT degradation in transgenic animals</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Perusse, D., Tepper, K., Sychla, A., Beach, S., Smanski, M., Maselko, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Herbivores engineered to detoxify TNT from unexploded ordnance .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> TNT (2,4,6-trinitrotoluene) from unexploded ordinances is a common environmental pollutant near munitions factories, military training sites, and areas of armed conflict. As physical and chemical approaches for TNT remediation are costly and difficult to scale, in situ bioremediation is an attractive alternative. TNT is a phytoaccumulative pollutant. Herbivores engineered to detoxify TNT are a potentially cost-effective platform to bioremediate large areas of contaminated sites while grazing or browsing. As a first step towards engineering herbivores with metabolic capabilities for TNT degradation, we engineered the animal genetic model, Drosophila melanogaster, to screen a library of microbial nitroreductases that can catalyse the initial degradation steps required for TNT degradation. We find strong, cofactor-dependent activity in fly lysates engineered to express NsfA from Escherichia coli, and NsfI from Enterobacter cloacae.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.11.659045v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Detect de novo expressed ORFs in transcriptomes withDESwoMAN</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Grandchamp, A., Lebherz, M. K., Dohmen, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a software tool designed to identify and characterise novel, unannotated open reading frames from a transcriptome, and analyse their mutations and fixation patterns within or across species.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> De novo gene emergence refers to the process by which new genes arise from mutations in previously non-coding genomic regions. Prior to becoming fixed in a species, newly expressed open reading frames (neORFs) undergo significant turnover within their species of origin. Investigating these early stages of de novo gene emergence is essential for understanding the mechanisms that enable gene formation from scratch. No software currently exists that can identify and characterise novel, unannotated open reading frames from a transcriptome, and analyse their mutations and fixation patterns within or across species. To address this gap, we introduce DESwoMA (De novo Emergence Study With Outgroup MutAtioNs), a software tool designed to: (1) detect neORFs in transcriptomes, (2) filter neORFs with no homology to outgroup genes, and (3) search for syntenic sequences homologous to neORFs in outgroup genomes (and optionally transcriptomes) and analyse mutations in coding features between these sequences. We applied DESwoMAN with two different strategies to three setups, using twice human and once fruit fly as query species. Our results highlight the tool's capabilities and demonstrate its potential for elucidating the early stages of de novo gene emergence. DESwoMAN is available at https://github.com/AnnaGrBio/DESWOMAN. It is implemented in Python3 and comes with a docker image on DockerHub for easy installation and execution including all (non-Python) dependencies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.10.658796v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

