<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Tuesday, 29 April 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">How genomics can help unravel the evolution of endophytic fungi.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jefferson Brendon Almeida Dos Reis, Andrei Stecca Steindorff, Adriana Sturion Lorenzi, Danilo Batista Pinho, Helson Mario Martins do Vale, Georgios Joannis Pappas</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #fedbf2; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, pan-genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comprehensive review of the evolution of endophytic fungal genomes, focusing on the evolutionary processes that shape their genomes, and the complex interdependencies governing their symbiotic interactions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Endophytic fungi (EFs) form intimate associations with plants, residing within their tissues without causing apparent harm. Understanding the evolution of endophytic fungal genomes is essential for uncovering the mechanisms that drive their symbiotic relationships with host plants. This review explores the dynamic interactions between EFs and host plants, focusing on the evolutionary processes that shape their genomes. We highlighted key genomic adaptations promoting their endophytic lifestyle, including genes involved in plant cell wall degradation, secondary metabolite production, and stress tolerance. By combining genomic data with ecological and physiological information, this review provides a comprehensive understanding of the coevolutionary dynamics between EFs and host plants. Moreover, it provides insights that help elucidate the complex interdependencies governing their symbiotic interactions.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40289066/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Discovering governing equations of biological systems through representation learning and sparse model discovery.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sadria M, Swaroop V.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a novel end-to-end computational framework designed to uncover parsimonious dynamical models and identify active gene programs from single-cell RNA sequencing data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding the governing rules of complex biological systems remains a significant challenge due to the nonlinear, high-dimensional nature of biological data. In this study, we present CLERA, a novel end-to-end computational framework designed to uncover parsimonious dynamical models and identify active gene programs from single-cell RNA sequencing data. By integrating a supervised autoencoder architecture with Sparse Identification of Nonlinear Dynamics, CLERA leverages prior knowledge to simultaneously extract related low-dimensional representation and uncover the underlying dynamical systems that drive the processes. Through the analysis of both synthetic and biological data, CLERA demonstrates robust performance in reconstructing gene expression dynamics, identifying key regulatory genes, and capturing temporal patterns across distinct cell types. CLERA's ability to generate dynamic interaction networks, combined with network rewiring using Personalized PageRank to highlight central genes and active gene programs, offers new insights into the complex regulatory mechanisms underlying cellular processes.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40290314/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">YEATS2: a novel cancer epigenetic reader and potential therapeutic target.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ji K, Chen G, Wang Y, Li Y, Chen J, Feng M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #fedbf2; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, pan-genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present mechanistic insights into the role of YEATS2 in epigenetic regulation, evaluates its therapeutic potential, and proposes future directions: elucidating full-length complex topologies, mapping synthetic lethal interactors, and optimizing selective inhibitors.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> YEATS2, an evolutionarily conserved reader of histone acylation marks (H3K27ac, H3K27cr, H3K27bz), functions as a central oncogenic driver in diverse cancers, including non-small cell lung cancer (NSCLC), pancreatic ductal adenocarcinoma (PDAC), and hepatocellular carcinoma (HCC). Its structurally plastic YEATS domain bridges acyl-CoA metabolism to chromatin remodeling, amplifying transcription of survival genes such as MYC, BCL2, and PD-L1. YEATS2 orchestrates malignancy-specific programs-sustaining ribosome biogenesis in NSCLC through ATAC complex recruitment, enhancing NF-κB-dependent immune evasion in PDAC, and activating PI3K/AKT-driven metabolic rewiring in HCC. Structural studies demonstrate a unique aromatic cage architecture that selectively engages diverse acylated histones. Although pyrazolopyridine-based inhibitors targeting the YEATS domain show preclinical efficacy, developing isoform-selective agents remains challenging. Clinically, YEATS2 overexpression correlates with therapy resistance and may synergize with immune checkpoint blockade. This review integrates mechanistic insights into the role of YEATS2 in epigenetic regulation, evaluates its therapeutic potential, and proposes future directions: elucidating full-length complex topologies, mapping synthetic lethal interactors, and optimizing selective inhibitors. Disrupting YEATS2-mediated epigenetic adaptation presents novel opportunities for precision cancer therapy.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40287757/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">af3cli: Streamlining AlphaFold3 Input Preparation.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Philipp Döpner, Stefan Kemnitz, Mark Doerr, Lukas Schulig</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present af3cli, an open-source tool designed to facilitate the generation of AlphaFold3 input files, specifically tailored to the standalone version of the standalone modeler.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> With the release of AlphaFold3, modeling capabilities have expanded beyond protein structure prediction to embrace the inherent complexity of biomolecular systems, including nucleic acids, ions, small molecules, and their interactions. The increased complexity of these assemblies is reflected in the input file generation process, presenting a significant hurdle for researchers without advanced computational expertise. While AlphaFold Server comes with a user-friendly graphical user interface, it supports only a subset of the features of AlphaFold3. To address this, we present af3cli, an open-source tool designed to facilitate the generation of AlphaFold3 input files, specifically tailored to the standalone version of AlphaFold3 and its unrestricted functionality. Featuring a user-friendly command-line interface and an accompanying Python library, af3cli simplifies the input generation process while maintaining flexibility and customization, which makes af3cli especially useful for fast (automated) generation of a large number of input files since it enables direct incorporation of FASTA files, keeps track of IDs, and validates the JSON file. Through practical examples, we demonstrate its capabilities for constructing input data for diverse biological structures, ranging from simple proteins to complex systems, and demonstrate its seamless integration into both manual and automated workflows.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40202494/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

