<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Thursday, 05 June 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Gene specificity landscapes for comparative transcriptomic analysis across tissues, cell types, and species</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bot, E., Davila-Velderrain, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a computational strategy to globally analyse and compare the specificity of genes and groups of related genes across different contexts.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Gene expression specificity is a biological parameter relevant for understanding the molecular basis of evolutionary constraints and tissue-selective pathogenesis. Many efforts have tried to quantify the degree of tissue specificity of individual genes. The growing availability of single-cell transcriptomic data greatly expands the context in which expression specificity can be assessed. We present a computational strategy to globally analyse and compare the specificity of genes and groups of related genes across different contexts. By representing expression profiles in terms of expression level-breadth (L-B) relationships, we are able to quantify and construct 2D landscapes that provide a globally consistent coordinate system to map specificity patterns. We characterize these landscapes at different levels of resolution and across species to demonstrate simple strategies for comparative transcriptomics. We use this approach to investigate the tissue, cell type, and neuronal specificity of human genes and generate reference specificity landscapes. Finally, by comparing the specificity of brain cell subtypes across 4 primate species we find that their degree of conservation mirrors evolutionary divergence times. Our analysis framework and data resources are available in the R package GeneSLand.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.04.657813v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The mutational dynamics of the Arabidopsis centromeres</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dong, X., Jiao, W.-B., Campoy, J. A., Rabanal, F. A., Ton, J., Smith, L. M., Weigel, D., Schneeberger, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Centromere evolution is driven by a unique mutational spectrum of point mutations and structure-preserving, in-frame indels spanning a few kilobases, which shape and maintain the large-scale architecture of centromeric DNA.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Centromeres are specialized chromosome regions essential for sister chromatid cohesion and spindle attachment during mitosis. Many centromeres comprise highly variable, megabase-scale satellite DNA arrays, yet the mutation spectrum driving this variability remains poorly understood. Using replicated genome assemblies of six Arabidopsis mutation accumulation lines, we identified centromeric mutations consisting almost exclusively of point mutations and structure-preserving, in-frame indels spanning a few kilobases. Centromeric point mutations occurred at a ninefold higher rate (6.1x10^-8/bp/gen) than in chromosome arms, frequently introduced by non-allelic gene conversions from closely linked repeat units. Forward-in-time simulations based on the observed mutation spectrum recapitulated the emergence of megabase-scale higher-order repeat (HOR) structures, including long-range sequence similarities, without requiring large-scale rearrangements, closely mirroring centromeric divergence among natural genomes. Our results show that centromere evolution is driven by a unique mutational spectrum, providing a quantitative framework to understand how small-scale mutations shape and maintain the large-scale architecture of centromeric DNA.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.02.657473v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Oligo-FISH Validates Genome Assemblies and Delivers the Most Precise Karyotype for Lens Mill. Species</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Silvestrini, A. J. A., Ramsay, L., von Wettberg, E. B., Bett, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We used oligo-FISH probes for the karyotyping of the chromosomes of Lens culinaris and the assembly of a genus pan-genome .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Chromosome structural rearrangements play a significant role in karyotype evolution and speciation. These rearrangements pose challenges for precise karyotyping, leading to asymmetric chromosomes and complicating the assembly of a genus pan-genome for crops and their wild relatives. Lens culinaris, an important cool-season legume primarily cultivated in India and Canada, is the cultivated species among six wild relatives. All seven species of Lens face significant challenges due to chromosomal rearrangements, ranging from introgression issues to difficulties in developing a precise karyotype and advancing genomic studies. Using the gene synteny analysis between the cultivated Lens species and six wild relatives, we developed cross-species oligo-FISH (Fluorescent in situ hybridization) probes aiming to further attest to the genome assembly and synteny analysis of Lens species. Roots of seven Lens spp. accessions were harvested and used for chromosome spread preparations. Those slides were then used for Oligo-FISH experiments, where the DNA present in the slides was denatured, and a set of red and green oligo probes was hybridized to the chromosomes. Pictures were taken using a fluorescence microscope. The combination of both oligo sets/probes resulted in a distinct pattern for each Lens spp. chromosome, allowing the inference of the most precise karyotype to date for six Lens species. The number of oligo probe signals reflects the species' phylogenetic proximity, while the distribution of those signals changed drastically within the same gene pool. The karyotyping of Lens confirmed the proper assignment of chromosomes in the genome assemblies and validated the rearrangements detected in the synteny analysis. Differences in the assembly probe prediction and the oligo-FISH results were used to improve the assemblies. The results attest to a higher sequence-level similarity among the closest related species despite the occurrence of several structural changes among them. The oligo-FISH probes can be used in conjunction with plant genome assembly projects, supporting the delivery of a precise representation of their physical chromosomes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.02.657502v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comparative transcriptomics of iPSC-derived cardiomyocytes in a high and a low altitude population</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gray, O. A., Witonsky, D. B., Di Rienzo, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Tibetan adaptation to high-altitude hypoxia remains a classic example of Darwinian selection in humans, and the study of iPSC-derived endothelial cells from Tibetan and Han Chinese populations shows that several pathways are candidate targets of natural selection due to exposure to the high-</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Tibetan adaptation to high-altitude hypoxia remains a classic example of Darwinian selection in humans. To identify adaptive traits that might have evolved in Tibetans in response to long-term exposure to hypoxia, we previously established a library of induced pluripotent stem cells (iPSCs), derived from Tibetan and Han Chinese individuals, as a robust model system for the exploration of condition-specific molecular and cellular responses. We used this system to characterize and compare the transcriptome of iPSC-derived endothelial cells and found that angiogenesis, energy metabolism and immune pathways differ between the cell lines from these two populations. Here, we harness the same experimental system to characterize and compare the transcriptome of iPSC-derived cardiomyocytes in Tibetan and Han Chinese in hypoxia. We find that several pathways, such as the hypoxia, myogenesis and glycolysis pathways, are significantly enriched for differentially expressed genes across populations. These pathways are candidate targets of natural selection due to exposure to the high-altitude hypoxic environment and point to adaptive cardiac traits such as sustained cardiac function in hypoxia. A better understanding of these adaptations may offer insights into novel therapeutic strategies for hypoxia-related cardiovascular conditions, such as pulmonary hypertension and ischemic heart disease.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.04.657917v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Persistent cortical excitatory neuron dysregulation in adult Chd8 haploinsufficient mice.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Canales, C. P., Lozano, S., Frost, N. A., Cichewicz, K., Amaral, W., Seban, N., Fenton, E., Wade, A., Chu, N., Smith, E., Ardekani, C., Frank, S., Bennett, J., Lavenex, P., Kopley-Smith, A., Rahbarian, D., Corea, M., Perla, D., Davis, L., Zhu, J., Ortiz, R., Beauregard, P., Moyer, C., Baker, J., Sun, J., Ma, B., Lu, J., Sohal, V. S., Amaral, D. G., Zuo, Y., Nord, A. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We found Chd8+/- mouse models exhibit macrocephaly and transcriptional pathology, with inconsistent findings regarding neurogenesis, neuron function, and behavior.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> CHD8 mutations cause autism spectrum disorder, cognitive deficits, and macrocephaly. Chd8+/- mouse models exhibit macrocephaly and transcriptional pathology, with inconsistent findings regarding neurogenesis, neuron function, and behavior. Via stereology and single nuclei transcriptomics (snRNA-seq), we found increased Chd8+/- cortical volume was not explained by increase in neuron number. Differential expression (DE) was present across cortical cell types, with excitatory neurons exhibiting high DE burden and shared and subclass-specific DE signatures. Bulk RNA-seq DE of constitutive Chd8+/- and conditional Camk2a-Cre Chd8+/- mice identified shared transcriptional pathology. DE in synaptosomal versus nuclear mRNA identified overlapping DEGs, but also significant differences and exaggerated synaptosomal changes. Building on DE findings implicating glutamatergic neurons, we found Chd8+/- mice exhibited altered excitatory neuron spine density and dynamics, decreased GCaMP activity correlation, and sleep perturbation. Thus, Chd8 haploinsufficiency causes lasting excitatory neuron dysfunction, perturbs RNA regulation beyond transcription, and impacts neuronal properties, cortical microcircuits, and behavior.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.04.657776v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Single-nucleus epigenomic dysregulation unmasks genetic risk-associated neurodegenerative glia states</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Han, X., Zhang, T., Lee, C.-Y., Ravi, A., Spina, S., Li, A. N., Grinberg, L. T., Seeley, W. W., Huckins, L. M., Raj, T., Brennand, K. J., Rexach, J. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that human genetic variants affecting microglial gene regulation converge into distinct and co-regulated modules affecting specific cellular functions in tau-related dementias.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The accumulation of abnormal tau protein selectively affects distinct brain regions and specific populations of neurons and glial cells in tau-related dementias, such as Alzheimer's disease (AD), Pick's disease (PiD), and progressive supranuclear palsy (PSP). Although the three disorders share the feature of tau protein pathology, the regulatory circuitry of non-coding genetic variants underlying risk-associated cell states remains to be elucidated. Using paired single-nucleus profiling of chromatin accessibility and gene expression across AD, PiD, and PSP, we define cell-type-specific cis-regulatory elements (CREs) across six cell types and fifty subclasses. Comparing disease-dynamic CREs across three disorders, we find that glia overrepresent disorder-specific gene regulation related to dynamic cellular response to stress. We show that human genetic variants affecting microglial gene regulation converge into distinct and co-regulated modules affecting specific cellular functions. Moreover, polygenic risk modifiers are maximally co-accessible in disorder-specific glial states, modifying distinct pathways such as sphingomyelin regulation in PiD. Our study informs glial regulators linked to polygenic modifiers of primary tauopathy, introducing modifiable pathways governing resilience for therapeutic consideration.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.02.657512v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">RPcontact: Improved prediction of RNA-protein contacts using RNA and protein language models</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jiang, J., Zhang, X., Zhan, J., Miao, Z., Zhou, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce RPcontact, a novel computational method designed to predict RNA-protein contacts using large language models tailored for RNA (ERNIE-RNA) and proteins (ESM-2).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Determining key contacts between RNA-protein interactions is essential for understanding the molecular mechanisms of numerous biological processes, including transcription, splicing, and translation. However, progress in this area has been impeded by the scarcity of RNA-protein complex structures in the Protein Data Bank (PDB) and the challenges posed by traditional structural determination techniques. Recent computational advancements, including deep learning methods like AlphaFold 3 and RoseTTAFoldNA, have improved contact prediction but are still limited by the availability of homologous sequences and templates. Here, we introduce RPcontact, a novel computational method designed to predict RNA-protein contacts using large language models tailored for RNA (ERNIE-RNA) and proteins (ESM-2). Despite being trained entirely on ribosomal RNA-protein (rRNA-protein) complexes, RPcontact demonstrates robust and generalized performance in predicting contacts for both dimeric and multimeric non-rRNA-protein complexes. The performance of RPcontact on contact predictions significantly improves over the binary contacts inferred from RNA-protein complex structures predicted by AlphaFold 3 and RoseTTAFoldNA, highlighting its potential in RNA-protein complex structure and function prediction.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.02.657171v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CellVoyager: AI CompBio Agent Generates New Insights by Autonomously Analyzing Biological Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Alber, S., Chen, B., Sun, E., Isakova, A., Wilk, A. J., Zou, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce CellVoyager, an AI agent that autonomously explores scRNA-seq datasets in novel directions conditioned on prior user-ran analyses.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Modern biology increasingly relies on complex, high dimensional datasets such as single-cell RNA sequencing (scRNA-seq). However, the richness of such data means that conventional analyses may only scratch its surface. Extracting meaningful insights from these datasets often requires advanced computational methods and domain expertise. Current AI agents for biology are primarily focused on executing user-specified commands and are therefore limited by the user's creativity and familiarity with which kinds of analyses are useful. Furthermore, these agents do not account for prior analyses already attempted by researchers, reducing their ability to build upon existing work. To address these limitations, we introduce CellVoyager, an AI agent that autonomously explores scRNA-seq datasets in novel directions conditioned on prior user-ran analyses. Built on large language models, CellVoyager ingests both the dataset and a record of prior analyses to generate and test new hypotheses within a Jupyter notebook environment. We evaluate CellVoyager on CellBench, a new benchmark based on 50 published scRNA-seq studies encompassing 483 analyses. Given only the background sections of these papers, CellVoyager outperformed GPT-4o and o3-mini by up to 20% in predicting which analyses the authors eventually conducted. We then carried out three in-depth case studies where CellVoyager is given previously published papers with their scRNA-seq datasets and conducts analyses to generate new findings. The original authors of each study evaluated these findings and consistently rated them as creative and sound; 80% of the agent's hypotheses were deemed scientifically interesting. For example, in one case study, the agent found that CD8+ T cells in COVID-19 infection are more primed for pyroptosis, which was not explored by the original researchers. CellVoyager also reanalyzed a brain aging dataset to discover a previously unreported association between increased transcriptional noise and aging in the subventricular zone of the brain. These results demonstrate that CellVoyager can act both autonomously and collaboratively to accelerate hypothesis generation and computational biology. It also highlights the potential of agents like CellVoyager to unlock new biological insights by reanalyzing the vast existing biological data at scale.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.03.657517v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A novel transformer-based platform for the prediction and design of biosynthetic gene clusters for (un)natural products</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kawano, T., Shiraishi, T., Kuzuyama, T., Umemura, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a transformer-based framework that models functional domains as linguistic units to capture and predict their positional relationships within genomes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Biosynthetic gene clusters (BGCs), comprising sets of functionally related genes responsible for synthesizing complex natural products, are a rich source of bioactive compounds with pharmaceutical potential. Here, we present a transformer-based framework that models functional domains as linguistic units to capture and predict their positional relationships within genomes. Using a RoBERTa architecture, we trained models on four progressively broader datasets: bacterial BGCs, Actinomycetes genomes, bacterial genomes, and bacterial plus fungal genomes. Evaluation using 2,492 experimentally-validated BGCs from the MIBiG database showed that more than 60% of true domains were ranked first and over 80% within the top 10 candidates. Our models also achieved classification accuracies exceeding 70% for major compound classes including polyketides (PKs) and terpenes. To explore model-guided BGC design, we compared predictions from the BGC-trained and genome-trained models using the BGC for the bacterial diterpenoid cyclooctatin as a case study. The genome-trained model uniquely predicted several domains absent from both the original BGC and the prediction by the BGC-trained model. Heterologous expression of one of those predicted domains in Streptomyces albus, together with the biosynthetic genes for cyclooctatin, yielded an unknown cyclooctatin derivative. This framework not only provides a novel BGC prediction method using machine learning but also facilitates rational design of artificial BGCs. Future integration of transcriptomic, protein structural, and phylogenetic data will enhance the models predictive and generative capabilities, supporting accelerated discovery and engineering of natural products.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.02.657346v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Cell-ECM Graphs: A Graph-Based Method for Joint Analysis of Cells and the Extracellular Matrix</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ghafoor, M., Parkinson, J. E., Sutherland, T. E., Rattray, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present Cell-ECM Graphs, a computational framework that integrates ECM and cellular components into a unified graph structure, enabling joint modelling of cell-cell, ECM- ECM, and cell-ECm interactions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The extracellular matrix (ECM) provides essential structural and biochemical support to tissues, yet its spatial organization is often underrepresented in computational analyses of spatial proteomics. We present Cell-ECM Graphs, a computational framework that integrates ECM and cellular components into a unified graph structure, enabling joint modelling of cell-cell, ECM-ECM, and cell-ECM interactions. Using both simulated and real data, we evaluate performance across interaction profiling, node classification, spatial clustering, and explainable graph neural networks. Incorporating ECM enhances node classification and reveals biologically meaningful spatial patterns, including disrupted immune-ECM connectivity in allergen-exposed lungs. Cell-ECM Graphs offer a novel approach to analysing ECM architecture and its role in shaping the tissue microenvironment in health and disease.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.04.657781v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Defense-Suppressive Fragments of RIN4 generated by AvrRpt2 Participate in NDR1-dependent Activation of RPS2</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Afzal, A. J., Alam, M., Huang, J., Agha, M., Da Cunha, L., Iqbal Rai, M., Tahir, J., Jamroze, A., Jones, J. D., Mackey, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that non-membrane-tethered RIN4 derivatives are potent cytosolic activators of RPS2, a pathogen effector that promotes virulence.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Plant nucleotide-binding, leucine-rich-repeat (NLR) immune receptors recognize pathogen effectors and activate immunity. The NLR RPS2 recognizes AvrRpt2, a Pseudomonas effector that promotes virulence by proteolytically cleaving a membrane-tethered host protein, RIN4. RIN4 cleavage by AvrRpt2 generates fragments that activate RPS2. A model for RPS2 activation by RIN4 destruction is consistent with the ectopic activity of RPS2 in plants lacking RIN4 but does not explain the link between the virulence activity of AvrRpt2 and RPS2 activation. We found that non-membrane-tethered RIN4 derivatives are potent cytosolic activators of RPS2. Activation of RPS2 by these RIN4 derivatives, like AvrRpt2-induced activation, and unlike ectopic activation in the absence of RIN4, requires the defense signaling protein NDR1. Cleavage products of RIN4 produced by AvrRpt2 play contrasting roles in the activation of RPS2, with the membrane-tethered C-terminal fragment suppressing RPS2 and the non-membrane-tethered internal fragment, dependent on compatibility with the C-terminal fragment, overcoming its suppression of RPS2.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.02.656948v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Tuning viscoelasticity and fine structure of living materials via synthetic adhesion logic and rheological perturbations</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Costan, S., Hallerbach, K., Kim, S., Camp, C., Kim, M., Riedel-Kruse, I. H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show how rheological and viscoelastic properties of multicellular materials made from living bacteria can be tuned via adhesin strength, cell size and shape, and adhesion logic, and show that rheology is a powerful tool for actively shaping microscopic structure of ELMs.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Engineered living materials (ELMs) at the multicelluar level represent an innovation that promises programmable properties for biomedical, environmental, and consumer applications. However, the rational tuning of the mechanical properties of such ELMs from first principles remains a challenge. Here we use synthetic cell-cell adhesins to systematically characterize how rheological and viscoelastic properties of multicellular materials made from living bacteria can be tuned via adhesin strength, cell size and shape, and adhesion logic. We confirmed that the previous results obtained for non-living materials also apply to bacterial ELMs. Additionally, the incorporation of synthetic adhesins, combined with the adaptability of bacterial cells in modifying various cellular parameters, now enables novel and precise control over material properties. Furthermore, we demonstrate that rheology is a powerful tool for actively shaping the microscopic structure of ELMs, enabling control over cell aggregation and particle rearrangement, a key feature for complex material design. These results deepen our understanding of tuning the viscoelastic properties and fine structure of ELMs for applications like bioprinting and microbial consortia design including natural systems.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.04.657808v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Cell-free genome-wide transcriptomics through machine learning optimization</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wagner, L., HOANG, A., RUE, O., Delumeau, O., Loux, V., Faulon, J.-L., Jules, M., Borkowski, O.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We performed cell-free transcriptomics by developing a platform using an active learning workflow to explore over 1,000,000 buffer conditions to identify a buffer that increased mRNA yield by 20-fold.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Despite advances in transcriptomics, understanding of genome regulation remains limited by the complex interactions within living cells. To address this, we performed cell-free transcriptomics by developing a platform using an active learning workflow to explore over 1,000,000 buffer conditions. This enabled us to identify a buffer that increased mRNA yield by 20-fold, enabling cell-free transcriptomics. By employing increasingly complex conditions, our approach untangles the regulatory layers controlling genome expression.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.04.657821v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Pasta, an age-shift transcriptomic clock, maps the chemical and genetic determinants of aging and rejuvenation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Salignon, J., Tsiokou, M., Marques, P., Rodriguez-Diaz, E., Ang, H., Pietrocola, F., Riedel, C. G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Pasta, a robust and broadly applicable TAC based on a novel age-shift learning strategy, accurately predicts relative age from bulk, single-cell, and microarray data, capturing senescent and stem-like cellular states through signatures enriched in p53 and DNA damage response pathways.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> As the prevalence of age-related diseases rises, understanding and modulating the aging process is becoming a priority. Transcriptomic aging clocks (TACs) hold great promise for this endeavor, yet most are hampered by platform or tissue specificity and limited accessibility. Here, we introduce Pasta, a robust and broadly applicable TAC based on a novel age-shift learning strategy. Pasta accurately predicts relative age from bulk, single-cell, and microarray data, capturing senescent and stem-like cellular states through signatures enriched in p53 and DNA damage response pathways. Its predictions correlate with tumor grade and patient survival, underscoring clinical relevance. Applied to the CMAP L1000 dataset, Pasta identified known and novel age-modulatory compounds and genetic perturbations, and highlighted mitochondrial translation and mRNA splicing as key determinants of the cellular propensity for aging and rejuvenation, respectively. Supporting Pasta's predictive power, we validated pralatrexate as a potent senescence inducer and piperlongumine as a rejuvenating agent. Strikingly, chemotherapy drugs were highly enriched among pro-aging hits. Taken together, Pasta represents a powerful and generalizable tool for aging research and therapeutic discovery, distributed as an easy-to-use R package on GitHub.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.04.657785v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">REnformer, a single-cell ATAC-seq predicting model to investigate open chromatin sites</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Riva, S. G., Sanders, E., Wilson, T., Stranieri, N., Gur, R., Baxter, M., Hughes, J. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We used transfer learning with high quality single cell ATAC datasets to develop REnformer, a model to predict chromatin accessibility and interrogate genome variation in health and disease.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genome regulatory elements are fundamental to cellular identity and cell type specific gene expression. Understanding how the underlying genetic code is differentially utilised by different cell types is central to understanding human health and disease. To better understand how DNA encodes genome regulatory elements such as promoters, enhancers, and boundary elements, we leverage the Enformer gene expression and epigenetic prediction model. We used transfer learning with high quality single cell ATAC datasets to develop REnformer, a model to predict chromatin accessibility. By introducing a benchmark for comparing performances against Enformer model, REnformer significantly outperformed Enformer in terms of higher prediction outcomes and lower error rates in all extensive analyses shown; introducing these benchmarks allowed us, and possible future works, to fairly compare such models. We further tested REnformer by predicting the effects of a well characterised -thalassemia variant and found that the prediction aligned with the observed change in genome regulatory element, previously validated. We conclude that REnformer is and can be a state-of-the-art tool to predict cell type specific regulatory elements, and interrogate the effect of genome variation in health and disease.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.04.657786v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

