<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Saturday, 03 May 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Altered Gut Microbiome Composition and Function in Individuals with Complex Regional Pain Syndrome.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Emmanuel Gonzalez, Tali Sahar, May Haddad, Sylvie Toupin, Ramzi Zioud, Muhammad Zoabi, Lilach Eyal Waldman, Zohar Tal Leshinsky, Maayan Ben Sasson, Vibhu Kumar, Yosefa Marom, Ayelet Midbari, Nicholas J B Brereton, Yoram Shir, Amir Minerbi</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We investigate the composition and function of the gut microbiome in complex regional pain syndrome and propose a new treatment strategy for the syndrome.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Complex regional pain syndrome is a chronic pain syndrome typically affecting a limb. It is characterized by severe spontaneous and evoked pain, along with vasomotor, autonomic, and motor signs and symptoms. Although dysregulation in several physiologic systems has been suggested in complex regional pain syndrome (CRPS), including aberrant inflammatory and immune responses, vasomotor dysfunction, and nervous system changes, the pathophysiologic mechanisms underlying the syndrome remain elusive. Effective treatment options are also limited. Previous research has highlighted the role of the gut microbiome in chronic pain, prompting us to investigate the composition and function of the gut microbiome in CRPS.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40042519/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Novel Multilayer Cultivation Strategy Improves Light Utilization and Fruit Quality in Plant Factories for Tomato Production</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Furuta, H., Qu, Y., Ishizuka, D., Kawabata, S., Sano, T., Yamori, W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel S-shaped cultivation system for tomatoes in a multilayer plant factory that promotes earlier fruit maturation and improved fruit quality.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Plant factories using artificial lighting are a promising solution to food security and urban agricultural challenges. However, cultivation of fruit-bearing crops such as tomatoes remains limited due to their high light demands, long growth periods, and tall plant structure. In this study, we aimed to develop an efficient cultivation system for tomatoes in a multilayer plant factory. Mini-tomatoes were hydroponically cultivated using white LEDs in a five-tier shelf system under two different cultivation methods. The conventional I-shaped method involved vertical growth on the top tier with downward lighting, while the novel S-shaped method trained each plant horizontally across the 2nd to 4th tiers with lateral lighting on each level. The S-shaped method enabled even light distribution, resulting in consistent photosynthetic rates throughout the canopy. In contrast, the I-shaped method suffered from strong light attenuation in the lower tiers, leading to reduced photosynthetic efficiency in shaded parts. Although total yield did not differ significantly between the two methods, the S-shaped method promoted earlier fruit maturation and improved fruit quality, including higher sugar content. Compared with greenhouse cultivation, plant factory conditions ensured stable temperature and lighting, leading to compact plant morphology, shorter internodes, and higher SPAD values. Moreover, fruit quality was more consistent year-round, with higher lycopene and sugar contents. This study demonstrates that the S-shaped cultivation system offers significant advantages in light use efficiency, plant management, and fruit quality. It represents a scalable approach for enhancing tomato production in plant factories and may facilitate the introduction of other high-light-demanding fruit crops into vertical farming systems.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.02.651818v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Analysis of leaf CO2 Assimilation, CO and CH4 Release Under Different Environmental Settings</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Casanova, D., Bruhn, D., Mikkelsen, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study on photosynthesis and the release of CO and CH4 from plant leaves .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Many studies have found plant leaves to be emitters of CO and CH4. Consensus indicates that CH4 emissions are stimulated by heat and UV, while CO release is additionally stimulated by visible light. The mechanisms producing these emissions are yet to be discovered. To get closer to finding these mechanisms, this study examined whether photosynthesis might influence CO and CH4 leaf emissions. Five plant species of different photosynthesis pathways were analysed for their photosynthesis performance, as well as CO and CH4 emissions under different temperatures and visible light intensities. Findings reveal CO release rates to be positively correlated with light intensity and temperature but suggest a separate dark metabolism. CH4 rates were independent of light intensity and temperature. Much lower CH4 release from excised leaves compared to their connected counterparts, indicates that such is dependent on stomatal opening, supporting the hypothesis that CH4 is dissolved in transpired water. CO release rates are similar between attached and detached leaves, suggesting that CO is produced at epidermal level. Photosynthesis appears to be unrelated to the release of either of these gases.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.30.651537v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Effect of plant tissue culture parameters on the ploidy level of Physalis grisea, Solanum lycopersicum, and Solanum prinophyllum regenerants</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Van Eck, J., Swartwood, K., Green, Y., Gentile, I., Lippman, Z. B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Plants regenerated from seedling explants (hypocotyls and cotyledons) of the Solanaceae family members Physalis grisea, Solanum lycopersicum, and Solanuma prinophyllum were used to determine the in vitro culture</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Plants regenerated from seedling explants (hypocotyls and cotyledons) of the Solanaceae family members Physalis grisea (groundcherry), Solanum lycopersicum (tomato), and Solanum prinophyllum (forest nightshade) were used to determine the in vitro culture parameters that contribute to the incidence in polyploidization of tissue culture-derived plants (regenerants) from these species. We examined the possible effects of zeatin concentration in the plant regeneration medium, explant source, and species. Plants were grown to maturity under greenhouse conditions, pollen was collected and germinated. Flow cytometry analysis verified the utility of the pollen germination method for determining differences in ploidy, which was based on the number of pollen tubes produced with one tube representing diploid and two indicating polyploid. As for zeatin concentration, we assessed the effect of our standard method of initiation on medium containing 2 mg/l followed by 1 mg/l 2 weeks after culture initiation in comparison with 0.25, 0.5, and 1 mg/l throughout the culture lifetime. There were no major correlations for zeatin concentration on ploidy status across the species except for plants regenerated from S. lycopersicum hypocotyl explants where the percentage of polyploid regenerants increased with increasing concentrations. As for species and explant effects, P. grisea plants regenerated from hypocotyl explants had the highest percentage of polyploid plants at 81% compared to 43% and 35% for S. lycopersicum and S. prinophyllum, respectively. From cotyledons, 8% of S. lycopersicum and 20% of S. prinophyllum were polyploid. A comparison with P. grisea could not be made because cotyledon explants do not regenerate on zeatin-containing medium. The results indicated the incidence of polyploidization cannot be generalized for zeatin concentration, however, an influence of explant type and species was observed. Effects of increased ploidy on plant morphology were primarily larger flower and seed size; however, no significant differences were observed in plant or fruit size.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.01.651681v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genetic, epigenetic and metabolite variation in peripheral European Yew (Taxus baccata L.) populations at an unexplored part of the species natural distribution</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Aravanopoulos, F. A., Dalmaris, E., Avramidou, E., Sarrou, E., Xanthopoulou, A., Multari, S., Martens, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of taxanes in the yew tree (Taxus baccata L).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Taxanes form effective anticancer agents, which are found in the leaves and barks of the yew tree (Taxus L.). Taxol(R) (also known as paclitaxel), 10-diacetylbacatin III, 10-deacetyltaxol III, baccatin III and cephalommanine are anti-neoplastic taxanes used for cancer treatment. Due to the high demand of taxanes, it is of great pharmaceutical interest to investigate unexplored to date population diversity. In this context, three peripheral Greek Taxus baccata L. populations (Mt Cholomon, Mt Olympus and Mt Vourinos) were investigated to identify the extent and structure of their genetic (based on microsatellite markers), epigenetic (based on methylation sensitive amplified markers) and chemodiversity (based on liquid chromatography mass spectrometry) profiles. Results showed that the concentration of taxanes varied considerably in relation to population and harvest season. The main taxane in T. baccata needles was 10-deacetylbacatin III (DAB), with concentrations ranging from 267.8 (Mt Vourinos) to 517.6 (Mt Olympus) mg kg-1 dw. Besides metabolite variation, notable levels of genetic diversity and significant population differentiation were revealed. These results, in conjunction to the high levels of total methylation found in all populations, indicate their potential adaptability under climatic change. The findings of this study pave the way for prospective breeding and conservation strategies of these important Taxus baccata L. populations for artificial selection of highly producing taxane trees and protection of local germplasm.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.30.651400v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Sustainable Edamame Production in an Artificial Light Plant Factory with Improved Yield and Quality</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Takano, T., Wakabayashi, Y., Wada, S., Sano, T., Kawabata, S., Yamori, W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A plant factory utilizing artificial light is an innovative agricultural model that enables efficient and sustainable crop production, but its application to a diverse range of crops remains limited.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> A plant factory utilizing artificial light is an innovative agricultural model that enables efficient and sustainable crop production. However, its application to a diverse range of crops remains limited. Edamame, a highly nutritious legume, has gained global popularity, yet its long-term storage is challenging due to quality deterioration, restricting its market distribution to seasonal availability. In this study, we successfully cultivated edamame using three hydroponic systems--nutrient film technique (NFT), rockwool, and aeroponics (mist culture)--within an LED plant factory. Among these, NFT demonstrated the highest fresh seed yield, which was comparable to or exceeded that of conventional field cultivation. The high yield was attributed to enhanced pod formation and biomass production under controlled conditions. Furthermore, although free amino acid content was lower in NFT cultivation, NFT-grown edamame exhibited significantly higher total sugar and isoflavone contents than field-grown counterparts, suggesting superior eating quality and nutritional benefits. These results indicate that NFT hydroponics is a promising method for stable, high-yield edamame production with enhanced nutritional properties. This study provides a foundation for expanding edamame cultivation in controlled environments, contributing to year-round supply and its potential application in urban and space agriculture.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.01.651627v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Heterologous expression of the cyanobacterial fructose-1,6-/sedoheptulose-1,7-bisphosphatase in Chlamydomonas reinhardtii causes increased cell size and biomass productivity in mixotrophic conditions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bussola, M., Perozeni, F., Meloni, M., Pivato, M., Zaffagnini, M., Ballottari, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A cyanobacterial dual-function FBP/SBPase enzyme, which catalyzes both reactions in the Calvin-Benson-Bassham cycle, can be overexpressed to improve biomass accumulation in microalgae.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The Calvin-Benson-Bassham (CBB) cycle is the metabolic pathway responsible for CO2 assimilation in oxygenic photosynthetic organisms. Two key rate-limiting steps in this cycle are catalyzed by the enzymes fructose-1,6-bisphosphatase (FBPase) and sedoheptulose-1,7-bisphosphatase (SBPase), making them promising targets for genetic enhancement to improve carbon fixation. A potential strategy involves overexpressing a cyanobacterial dual-function FBP/SBPase, which catalyzes both reactionsOverexpression of this enzyme in tobacco plants or in other plants led to an increase in growth rate and biomass accumulation. Here, the overexpression of the same enzyme was achieved in Chlamydomonas reinhardtii. The recombinant cyanobacterial FBP/SBPase isolated from C. reinhardtii exhibited the expected catalytic activity, being Mg2+ dependent and strongly activated in the presence of a reducing agent. The FBP/SBPase expressing lines exhibited an increased photosynthetic activity at the cell level and decreased production of singlet oxygen upon exposure to high irradiances, suggesting improved capacity to manage high excitation pressure of the photosynthetic apparatus. Increased cell volume was measured in FBP/SBPase-expressing lines under different growth conditions. However, increased growth and biomass productivity were observed only in mixotrophy when light and CO2 were limiting, leading to increased starch, protein, and lipid content on a cellular basis. This phenotype caused an increased sedimentation rate in the transformant lines: the expression of FBP/SBPase enzyme could thus be considered as a strategy to improve the cell harvesting process. These findings provide new insights into carbon metabolism in microalgae, and could, in the future, support improved biomass accumulation, paving the way for effective domestication and industrial use.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.02.651950v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">How social and economic policies have affected the genome of mezcal agaves: the contrasting stories of Bacanora and Espadin</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Klimova, A., Gutierrez Rivera, J. N., Aguirre-Planter, E., Eguiarte, L. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We found evidence suggesting that Agave angustifolia in Oaxaca and Bacanora agave in Sonora are genetically distinct from wild populations, and that social and political decisions may have important impacts on crop genomic diversity and differentiation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Plant domestication in Mesoamerica gave the world crops of global significance, such as maize, beans, squashes, cocoa, and cotton. Additionally, it has introduced species of regional economic importance, which still display intermediate levels of domestication, including Agave, Opuntia, columnar cacti, Amaranthus, and various ornamental species. Agaves, in particular, hold immense cultural and economic significance in Mexico and play a crucial ecological role in wild plant communities. However, current agricultural practices have negatively impacted both wild populations of agave through overexploitation and habitat destruction, as well as cultivated plants by reducing available landraces and promoting the use of homogeneous plant material. Using genomic data (RADseq) and over 50,000 SNPs, we aimed to assess how local social and political decisions may have influenced genomic diversity and differentiation in intensively managed, clonally propagated Agave angustifolia (Espadin) in the state of Oaxaca, as well as in the cultivated A. angustifolia used to produce mezcal known as Bacanora in the state of Sonora and their wild counterparts from both regions. We found evidence suggesting that Espadin recently aroused through farmer selection of a clonal lineage with desirable mezcal production attributes (i.e., hybrid vigor), apparently from a cross between genetically distinct wild populations or by hybridization between wild and cultivated varieties. Espadin samples were represented by closely related heterozygous genotypes, with considerable genetic differentiation from wild plants. On the other hand, the genomic composition of cultivated Bacanora agave apparently was influenced by a recently lifted ban (in 1992) on its cultivation and distillation, which, along with the relatively lower popularity of this beverage, allowed cultivated Bacanora agave to maintain the genetic diversity found in wild populations of the regions. We found that social and political decisions may have important impacts on crop genomic diversity and differentiation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.30.651404v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Vernalization regulatory network identifies potential novel functions for ZCCT genes within HvVRN2</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Montardit-Tarda, F., Puyo, I., Contreras-Moreira, B., Karsai, I., Borrill, P., Casas, A. M., Igartua, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of HvVRN2 in barley shows its role in flowering and vernalization, and proposes new candidate genes for flowering regulation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Flowering is a key yield determinant, controlled by environmental and genetic factors. Vernalization requirement, a prolonged cold period to promote flowering, is determined in barley (Hordeum vulgare L.) by the allele in HvVRN1 and the presence of locus HvVRN2, which consists of two ZCCT genes: HvVRN2a and HvVRN2b. While the effect of HvVRN2 presence is well-known, its outcome on the transcriptome and other genes outside the flowering/vernalization pathway remains unknown. Near-isogenic lines with presence/absence of HvVRN2, were subjected to different lengths of vernalization treatments, identifying the expression dynamics of both HvVRN2 genes and wider transcriptional responses controlled by VRN2. Our work revealed previously unknown differences in expression levels of the two HvVRN2 genes, in responses to its main repressor, HvVRN1, and an effect on tillering. Besides, we shed new light on the repressing mechanism of VRN2 on flowering, using protein modelling. A regulatory network analysis pointed at new candidate genes for flowering regulation related to the vernalization pathway, and suggested additional biological roles for HvVRN2b. This study re-establishes HvVRN2 as a central gene for understanding barley responses to environmental cues, beyond its previously accepted simple role, and expands the catalogue of genes related to the vernalization pathway, which may be targeted in barley breeding.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.29.651237v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identification of immune response genes of cassava during early phases of cassava brown streak virus infection</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sheat, S., Winter, S., Lillenthal, J., Pucker, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A time-course RNA-Seq study of cassava brown streak disease and its resistance to CBSV and UCBSV shows significant differences in the number of differentially expressed genes (DEGs) in susceptible and resistant genotypes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cassava brown streak disease (CBSD) caused by the ipomoviruses CBSV and UCBSV, poses a significant threat to cassava crops and the destruction of tuberous roots leads to substantial yield losses. Virus resistance identified in some South American cassava germplasm lines inhibits virus replication and movement and, is a complete immunity of some lines or, a restriction of the virus replication to phloem companion cells and root infection in others. To further explore the resistance responses and identify potential resistance genes, a time-course RNA-Seq study was conducted comparing gene ex-pression of a virus-susceptible cassava TMS 96/0304 with a cassava line restricting CBSV to the roots and a cassava DSC 167 (COL2182), immune to the virus. Differential gene expression patterns across leaves, stems, and roots revealed substantial differences between susceptible and resistant genotypes. Temporal analysis highlighted significant variations in the number of differentially expressed genes (DEGs), particularly in stem tissues at one day after inoculation (DAI), with notable overexpression observed in the resistant lines DSC 167 (436 DEGs) and DSC 260 (335 DEGs). In contrast, significant expression changes were noted only at 10 DAI in the virus infected TMS 96/0304 (737 DEGs). Several DEGs in resistant genotypes were associated with proteins involved in hormonal regulation, defense and immunity, transcriptional regulation, stress response, cell structure, and signal transduction. The analysis indicates that virus resistance in the studied lines is driven by early-stage regulation of genes for which specific functions remain to be elucidated.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.29.651173v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SNRK3.15 is a crucial component of the sulfur deprivation response in Arabidopsis thaliana</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Apodiakou, A., Heyneke, E., Alseekh, S., Pinsorn, P., Metzger, S., Kopriva, S., Schulze, W., Hoefgen, R., Whitcomb, S. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the serine/threonine protein kinase CIPK14/SNRK3.15 plays a regulatory role in the -S response in Arabidopsis thaliana seedlings.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Sulfate deprivation (-S) results in numerous metabolic and phenotypic alterations in plants. Kinases are often key players in transducing nutrient status signals to molecular components involved in metabolic and developmental program regulation, but despite the physiological importance of sulfur, to date, no signaling kinases have been identified in sulfur-deficiency signaling response programs. Here we show that the serine/threonine protein kinase CIPK14/SNRK3.15 plays a regulatory role in the -S response in Arabidopsis thaliana seedlings. Multiple molecular and physiological responses to -S are attenuated in snrk3.15 mutants, including both early adaptive responses and later emergency salvage processes including nutrient deficiency induced senescence. When grown in soil with sufficient sulfur supply, snrk3.15 mutants showed no clear phenotypes, including no difference in seed sulfur content. Lastly, the proteome dataset generated from Col-0 and snrk3.15.1 Arabidopsis seedlings under -S conditions for this project is the first of its kind and will be a valuable research resource. Proteomics data are available via ProteomeXchange with identifier PXD046612.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.29.651231v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A unified analysis of atlas single-cell data.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hao Chen, Nam D Nguyen, Matthew Ruffalo, Ziv Bar-Joseph</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present GIANT, an integration method designed for atlas-scale gene analysis across cell types and data modalities.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent efforts to generate atlas-scale single-cell data provide opportunities for joint analysis across tissues and modalities. Existing methods use cells as the reference unit, hindering downstream gene-based analysis and removing genuine biological variation. Here we present GIANT, an integration method designed for atlas-scale gene analysis across cell types and tissues. GIANT converts data sets into gene graphs and recursively embeds genes without additional alignment. Applying GIANT to two recent atlas data sets yields unified gene-embedding spaces across human tissues and data modalities. Further evaluations demonstrate GIANT's usefulness in discovering diverse gene functions and underlying gene regulation in cells from different tissues.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/39965934/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A telomere to telomere phased genome assembly and annotation for the Australian central bearded dragon Pogona vitticeps</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Patel, H. R., Alreja, K., Reis, A. M., Chang, J. K., Chew, Z. A., Jung, H., Hammond, J. M., Deveson, I. W., Ruiz-Herrera, A., Marin-Gual, L., Holleley, C. E., Zhang, X., Lister, N. C., Whiteley, S. L., Xiong, L., Dissanayake, D. S. B., Waters, P. D., Georges, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We assemble the genome of a female ZW central bearded dragon that undergoes sex reversal from ZZ genotypic males to phenotypic females at high incubation temperatures.</p><p style="font-size: 16px; text-align: justify;">Background: The central bearded dragon (Pogona vitticeps) is widely distributed in central eastern Australia and adapts readily to captivity. Among other attributes, it is distinctive because it undergoes sex reversal from ZZ genotypic males to phenotypic females at high incubation temperatures. Here, we report an annotated telomere to telomere phased assembly of the genome of a female ZW central bearded dragon. Results: Genome assembly length is 1.75 Gbp with a scaffold N50 of 266.2 Mbp, N90 of 28.1 Mbp, 26 gaps and 42.2% GC content. Most (99.6%) of the reference assembly is scaffolded into 6 macrochromosomes and 10 microchromosomes, including the Z and W microchromosomes, corresponding to the karyotype. The genome assembly exceeds standard recommended by the Earth Biogenome Project (6CQ40): 0.003% collapsed sequence, 0.03% false expansions, 99.8% k-mer completeness, 97.9% complete single copy BUSCO genes and an average of 93.5% of transcriptome data mappable back to the genome assembly. The mitochondrial genome (16,731 bp) and the model rDNA repeat unit (length 9.5 Kbp) were assembled. Male vertebrate sex genes Amh and Amhr2 were discovered as copies in the small non-recombining region of the Z chromosome, absent from the W chromosome. This, coupled with the prior discovery of differential Z and W transcriptional isoform composition arising from pseudoautosomal sex gene Nr5a1, suggests that complex interactions between these genes, their autosomal copies and their resultant transcription factors and intermediaries, determines sex in the bearded dragon. Conclusion: This high-quality assembly will serve as a resource to enable and accelerate research into the unusual reproductive attributes of this species and for comparative studies across the Agamidae and reptiles more generally.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.01.651798v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PhysDock: A Physics-Guided All-Atom Diffusion Model for Protein-Ligand Complex Prediction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, K., Ma, Y., Yu, J., Luo, H., Lin, J., Qin, Y., Li, X., Jiang, Q., Bai, F., Dou, J., Zheng, J., Yu, J., Sun, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> PhysDock, a physics-guided diffusion model that uniquely integrates all-atom diffusion to model ligand flexibility and protein precision-flexibility; (ii) physical priors as diffusion conditioning; and (iii) two-phase physics guidance during the denoising diffusion to ensure physical plausibility</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurate prediction of protein-ligand complexes remains a central challenge in structural biology. Traditional methods are computationally inefficient and prone to local minima, whereas deep learning approaches struggle to capture molecular flexibility and physical plausibility. We introduce PhysDock, a physics-guided diffusion model that uniquely integrates (i) all-atom diffusion to model ligand flexibility and protein precision-flexibility; (ii) physical priors as diffusion conditioning, alongside two-phase physics guidance during the denoising diffusion to ensure physical plausibility. PhysDock demonstrates state-of-the-art performance in redocking benchmarks and excels in the more challenging cross-docking assessments involving protein structures resolved with different ligands. For practical utility, PhysDock (i) resolves cannabinoid receptor selectivity across diverse molecules, achieving accuracy comparable to experiments; (ii) effectively distinguishes most drug candidates from weak binders in virtual screening of NTRK3 kinase, while also uncovering novel candidate molecules with structural insights. These results highlight PhysDock as a versatile tool for protein-ligand complex prediction, with substantial potential to accelerate structure-based drug discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.28.650887v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Chromatix: a differentiable, GPU-accelerated wave-optics library</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Deb, D., Both, G.-J., Bezzam, E., Kohli, A., Yang, S., Chaware, A., Allier, C., Cai, C., Anderberg, G., Eybposh, M. H., Schneider, M. C., Heintzmann, R., Rivera-Sanchez, F. A., Simmerer, C., Meng, G., Tormes-Vaquerano, J., Han, S., Shanmugavel, S. C., Maruvada, T., Yang, X., Kim, Y., Diederich, B., Joo, C., Waller, L., Durr, N. J., Pegard, N. C., La Riviere, P. J., Horstmeyer, R., Chowdhury, S., Turaga, S. C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present Chromatix: an open-source, GPU-accelerated differentiable wave optics library that speeds up optical simulation and optimization from 2-6x on a single GPU to up to 22x on 8 GPUs (depending on the particular system being modeled).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Modern microscopy methods incorporate computational modeling of optical systems as an integral part of the imaging process, either to solve inverse problems or enable optimization of the optical system design. These methods often depend on differentiable simulations of optical systems, yet no standardized framework exists - forcing computational optics researchers to repeatedly and independently implement simulations that are prone to errors, difficult to reuse in other applications, and often computationally suboptimal. These common problems limit the potential impact of computational optics as a field. We present Chromatix: an open-source, GPU-accelerated differentiable wave optics library. Chromatix builds on JAX to enable fast simulation of diverse optical systems and inverse problem solving, scaling these simulations from single-CPU laptops to multi-GPU servers. The library implements various optical elements (e.g., lenses, polarizers and spatial light modulators) and multiple light propagation models (e.g., Fresnel approximation, angular spectrum and off-axis propagation) that can be flexibly combined to model various computational optics applications such as snapshot microscopy, holography, and phase retrieval of multiple scattering samples. These simulations can be automatically parallelized to scale across multiple GPUs with a single-line change to the modeling code, enabling simulation and optimization of previously impractical optical system designs. We demonstrate Chromatix's capacity to substantially accelerate optics simulation and optimization on existing methods in computational optics, speeding up optical simulation and optimization from 2-6x on a single GPU to up to 22x on 8 GPUs (depending on the particular system being modeled) compared to the original implementations. Chromatix establishes a standard for wave optics simulations, democratizing access to and expanding the design space of computational optics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.29.651152v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Case-Based Explainable Graph Neural Network Framework for Mechanistic Drug Repositioning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cavazos, A. C. G., Tu, R., Sinha, M., Su, A. I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Drug-Based Reasoning Explainer (DBR-X), an explainable GNN model that combines a link prediction module and a path-identification module to generate interpretable and faithful explanations for drug repositioning.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Drug repositioning offers a cost-effective alternative to traditional drug development by identifying new uses for existing drugs. Recent advances leverage Graph Neural Networks (GNN) to model complex biological data, showing promise in predicting novel drug-disease associations. However, these frameworks often lack explainability, a critical factor for validating predictions and understanding drug mechanisms. Here, we introduce Drug-Based Reasoning Explainer (DBR-X), an explainable GNN model that combines a link prediction module and a path-identification module to generate interpretable and faithful explanations. When benchmarked against other GNN link prediction frameworks, DBR-X achieves superior performance in identifying known drug-disease associations, demonstrating higher accuracy across all evaluation metrics. The quality of DBR-X biological explanations was assessed through multiple approaches: comparison with manually-curated drug mechanisms, evaluation of explanation faithfulness through deletion and insertion studies, and measurement of stability under graph perturbations. Together, our model not only advances the state-of-the-art in drug repositioning predictions but also provides multi-hop explanations that can accelerate the translation of computational predictions into clinical applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.28.651120v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Explainable deep learning for identifying cancer driver genes based on the Cancer Dependency Map</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yin, Q., Chen, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a biologically informed, supervised deep learning model that links the mutation status of a candidate cancer driver gene to genome-wide dependency scores.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Detecting cancer driver genes and mutations is critical but challenging in cancer biology, essential for understanding tumor progression and developing targeted therapies. In this study, we leveraged the Cancer Dependency Map (DepMap) to identify potential cancer driver genes and infer multi-driver mutations within tumor samples. We developed xNNDriver, a biologically informed, supervised deep learning model that links the mutation status of a candidate cancer driver gene to genome-wide dependency scores. The model's fitness provides a quantifiable measure of a gene's likelihood to act as a cancer driver. Our model successfully recovers well-established drivers, including NRAS, KRAS, HRAS, and BRAF, and identifies novel candidates such as VDR. The model's interpretability further illuminates downstream pathways regulated by driver genes. To complement the gene-by-gene analysis, we built xAEDriver, an unsupervised explainable autoencoder that infers multiple driver variant representations (DVRs) simultaneously. These DVRs capture the mutation patterns across multiple potential driver variants in tumor samples. Notably, DVR-based stratification of cell lines uncovers distinct drug response profiles, further demonstrating the model's translational potential. Overall, our framework offers a robust and interpretable approach to cancer driver gene discovery and mutation pattern analysis, offering new insights into cancer biology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.28.651122v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Modular Engineering of Thermo-Responsive Allosteric Proteins</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hoffmann, K. H., Kroell, A.-S., Motzkus, N. A., Lemmen, N., Happ, N., Wolf, B., von Bachmann, A., Southern, N., Vogd, F., Aschenbrenner, S., Niopek, D., Mathony, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a generalizable strategy for thermosensitive allosteric proteins through the insertion of optimized Avena sativa LOV2 domain variants.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Thermogenetics enables non-invasive spatiotemporal control over protein activity in living cells and tissues, yet its applications have largely been restricted to transcriptional regulation and membrane recruitment. Here, we present a generalizable strategy for engineering thermosensitive allosteric proteins through the insertion of optimized Avena sativa LOV2 domain variants. Applying this approach to a diverse set of structurally and functionally unrelated proteins in Escherichia coli, we generated potent, thermo-switchable chimeric variants that can be tightly controlled within narrow temperature ranges (37-41{degrees}C). Extending this strategy to mammalian systems, we engineered the first CRISPR-Cas genome editors directly modulated by subtle temperature changes within the physiological range. Finally, we showcase the incorporation of a chemoreceptor domain as an alternative thermosensing module, suggesting thermo-sensitivity to be a widespread feature in receptor domains. This work expands the toolkit of thermogenetics, providing a blueprint for temperature-dependent control of virtually any protein of interest.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.02.651844v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The nanosd integral gene drive enables population modification of the malaria vector Anopheles gambiae</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yen, P.-S., Verkuijl, S. A., Capriotti, P., Del Corsano, G., Hoermann, A., Inghilterra, M. G., Aramburu-Gonzalez, I., Kahn, M. A., Christophides, G. K., Vlachou, D., Windbichler, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the nanosd gene drive can be used for gene drive in malaria vector control, and outline steps towards its further optimization.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The modification of mosquito populations at scale through CRISPR-Cas9-mediated homing gene drives is a promising route for malaria vector control. Integral gene drives (IGDs) are designed to utilize the regulatory sequences of endogenous genes to express only the minimal set of components required for gene drive. In this study, we describe the creation and characterization of the nanosd IGD targeting and inserted into the nanos gene of the malaria vector Anopheles gambiae and show that it achieves high rates of gene drive (97.7% in females, 99.0% in males). We find that homozygous nanosd females but not males show impaired fecundity, and a variable loss of ovary phenotype. Transcriptomic analysis of ovaries points to decreased transcript levels of the nanos gene when harbouring Cas9. As a minimal genetic modification, nanosd does not induce widespread transcriptomic perturbations, and its susceptibility to Plasmodium spp. and O'nyong nyong virus infection remains similar to wild-type mosquitoes. Importantly, we find that nanosd propagates efficiently in caged mosquito populations and is maintained as a source of Cas9 after the emergence of drive resistant alleles whilst also mobilising a non-autonomous antiparasitic effector modification. The nanosd gene drive shows promise as a genetic tool for malaria vector control via population modification, and we outline steps towards its further optimization.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.02.651860v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Escherichia coli with a 57-codon genetic code</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Robertson, W. E., Rehm, F. B. H., Spinck, M., Schumann, R. L., Tian, R., Liu, W., Gu, Y., Kleefeldt, A. A., Day, C. F., Liu, K. C., Christova, Y., Zurcher, J. F., Boege, F. L., Birnbaum, J., van Bijsterveldt, L., Chin, J. W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We replace every known occurrence of six sense codons and a stop codon with synonymous codons to create an organism with a 57-codon genetic code; this organism - which we name Syn57 - uses 55 codons for protein synthesis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The near-universal genetic code of living organisms uses 64 codons to encode the 20 canonical amino acids in protein synthesis. Here we design and generate a variant of Escherichia coli with a 4 Mb synthetic genome in which we replace every known occurrence of six sense codons and a stop codon with synonymous codons. We thereby recode 105 codons to create an organism with a 57-codon genetic code; this organism - which we name Syn57 - uses 55 codons to encode the 20 canonical amino acids.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.02.651837v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrative multi-omics modelling for cultivated meat production, quality, and safety</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mathieu, T., Legare, S., Nzekoue, A. F., Jaure, N., Lester, H., Dias, T., Kusters, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A network-based interactome model that integrates the transcriptomic, proteomic, and metabolomic layers, we achieve a system-level understanding of cellular metabolism and regulatory mechanisms, which can allow for precise monitoring and targeted interventions of critical quality and safety attributes associated with cellular biomass.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cell culture technology, which offers a promising solution for complementary food production, is slowly becoming a reality with the first wave of regulatory approvals in pioneering markets. However, significant challenges remain for large-scale cultivated meat commercialization, including effective scaling, cost efficiency, and product quality, but also scientific evidence to support regulatory approval and building consumer trust. In this paper, we discuss the potential of an integrative multi-omics approach to characterize and optimize cultivated meat production. By analyzing a network-based interactome model that integrates the transcriptomic, proteomic, and metabolomic layers, we achieve a system-level understanding of cellular metabolism and regulatory mechanisms. This approach can allow for precise monitoring and targeted interventions of critical quality and safety attributes associated with cellular biomass. We then describe a Target-Action-Metabolite (TAM) framework, which utilizes insights from the interactome to optimize cell culture conditions through actionable interventions. We illustrate the potential use of this framework through a case study involving Duck Embryonic Stem Cells (dESCs) for use in cell-cultured meat products, providing hypotheses for improving key metabolic pathways through targeted interventions on metabolites present in culture media. Finally, our paper highlights the potential of this interactome-based strategy to enhance bioprocess efficiency, improve product quality and ensure safety attributes, addressing regulatory challenges associated with cultivated meat production.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.30.651459v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Evaluation of epistasis detection methods for quantitative phenotypes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Listopad, S., Peng, Q., Renjith, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified six epistasis detection methods suitable for quantitative phenotype data: EpiSNP, Matrix Epistasis, MIDESP, PLINK Epistasia, QMDR, and REMMA, and evaluated their performance on the Adolescent Brain Cognitive Development (ABCD) dataset.</p><p style="font-size: 16px; text-align: justify;">Background Epistasis, or genetic interaction, has been increasingly recognized for its ubiquity and for its role in susceptibility to common human diseases, such as Alzheimers. A wide variety of epistasis detection tools are currently available with several studies comparing the performance of methods suitable for case-control data. However, there is limited understanding of how well these tools perform with quantitative phenotypes. Methods We identified six epistasis detection methods suitable for quantitative phenotype data: EpiSNP, Matrix Epistasis, MIDESP, PLINK Epistasis, QMDR, and REMMA. To evaluate these tools, we generated simulated datasets using EpiGEN. The datasets modeled various pairwise interactions between disease-associated SNPs, including dominant, multiplicative, recessive, and XOR interactions. Additionally, we assessed the BOOST and MDR algorithms on discretized (case-control) version of the datasets. These tools were then tested on the Adolescent Brain Cognitive Development (ABCD) dataset for the externalizing behavior phenotype. Results Each tool exhibited strong performance for certain interaction types, but weaker performance for others. MDR achieved the highest overall detection rate of 60%, while EpiSNP had the lowest overall detection rate of 7%. MDR and MIDESP performed best at detecting multiplicative interactions with detection rates of 54% and 41% respectively. Both MDR and MIDESP were also effective at detecting XOR interactions with detection rates of 84% and 50% respectively. PLINK Epistasis, Matrix Epistasis, and REMMA excelled at detecting dominant interactions, all achieving a 100% detection rate. On the other hand, EpiSNP was particularly effective at detecting recessive interactions with a detection rate of 66%. When analyzing the ABCD dataset, Plink Epistasis and Plink BOOST identified SNPs within the DRD2 and DRD4 genes, which have been previously linked to externalizing behavior. Conclusion Since no single method consistently outperforms others across all types of epistasis, and given that the specific types of epistasis present in a dataset are often unknown, it may be more effective to use multiple epistasis detection algorithms in combination to obtain comprehensive results.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.30.651312v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">From Pairwise Distances to Neighborhood Preservation: Benchmarking Dimensionality Reduction Algorithms for CyTOF, scRNA-seq, and CITE-seq</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bombina, P., Adams, Z. B., McGee, R. L., Coombes, K. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comprehensive evaluation of 16 dimensionality reduction methods for high-dimensional data visualization .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Dimensionality reduction algorithms are essential tools for visualizing high-dimensional biological data, such as single-cell transcriptomics, mass cytometry by time of flight, and cellular indexing of transcriptomes and epitopes. These algorithms map complex, high dimensional data into lower dimensions to reveal underlying structures and patterns. Despite the popularity of dimensionality reduction methods like t-SNE and UMAP, concerns have arisen regarding their ability to preserve critical aspects of high-dimensional data and their sensitivity to user-defined parameters. This study aims to evaluate the impact of extreme dimension reduction from hundreds or thousands of dimensions to just two dimensions, highlighting the resulting distortions to deeply understand their implications. Given the significance of dimensionality reduction in biological research, careful evaluation of these methods is necessary to ensure reliable and meaningful results. In this paper, we present a comprehensive evaluation of 16 dimensionality reduction methods. Our evaluation addresses several key factors, such as the preservation of pairwise distances and local neighborhood relationships between the original high-dimensional space and the low-dimensional projections.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.28.651069v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">iSHARC: Integrating scMultiome data for heterogeneity and regulatory analysis in cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zeng, Y., Bahl, S., Xu, X., Ci, X., Keshavarzian, T., Yang, L., Lee, H. S., Kossinna, P., Gaiti, F., Schwartz, G. W., He, H. H., Lupien, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present iSHARC, a streamlined pipeline for quality control, modality integration, clustering, cell type annotation, and regulatory mechanism analysis of individual scMultiome data, as well as for integrating multiple samples.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The 10x Genomics single cell Multiome (scMultiome) assay enables the simultaneous profiling of chromatin accessibility and gene expression from the same nucleus, and has increasingly been utilized in revealing cellular heterogeneity and gene regulation in cancers. However, a dedicated bioinformatics pipeline specifically designed for this type of data is still lacking. Here we present iSHARC, a streamlined pipeline for quality control, modality integration, clustering, cell type annotation, and regulatory mechanism analysis of individual scMultiome data, as well as for integrating multiple samples. The main advantages of iSHARC are: 1) easy implementation, execution and extension through a modular Snakemake workflow management system; 2) flexible analysis and parameters customization via a single configuration file; and 3) comprehensive accessibility by providing different access points to results and detailed summary reports from a single run.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.28.651068v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">OmiXAI: An Ensemble XAI Pipeline for Interpretable Deep Learning in Omics Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Alaeva, A., Lapteva, A., Mikhaylovskaya, N., Malkov, V., Herbert, A., Borevskiy, A., Poptsova, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed OmiXAI, a pipeline integrating ensemble model-aware XAI methods with gradient-based techniques including Integrated Gradients, InputXGradients, Guided Backpropagation, and Deconvolution (for CNNs and GNNs).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Deep learning methods have become methods of choice in the analysis of genomic data. The performance of deep learning models depends on the information available for training. A growing trend in deep learning applications involves leveraging multi-omics data spanning genomics, transcriptomics, epigenomics, proteomics, metabolomics, and other domains. When a deep learning model trained on omics data achieves high performance, the important question is to define factors that contribute to model predictive power. Explainable AI (XAI) methods can be categorized as model-aware and model-agnostic. Model-agnostic approaches, which rely on combinatorial feature perturbations to assess impact, are often computationally prohibitive for deep learning models. To address this, we developed OmiXAI, a pipeline integrating ensemble model-aware XAI methods. Our framework incorporates gradient-based techniques including Integrated Gradients, InputXGradients, Guided Backpropagation, and Deconvolution (for CNNs and GNNs) as well as Saliency Maps and GNNExplainer (specifically for GNNs). We evaluated OmiXAI on Z-DNA prediction using multi-omics features, demonstrating its efficacy through feature importance analysis and benchmarking of XAI methods. Notably, OmiXAI enabled feature engineering, reducing the critical feature set from almost 2,000 to just 50. Its modular design allows seamless integration of additional attribution methods, ensuring adaptability beyond omics to diverse problem domains. While testing the ensemble approach we benchmarked individual XAI methods and discuss their drawbacks and limitations. OmiXAI is freely available at https://github.com/aameliig/OmiXAI.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.28.651097v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Massive experimental quantification allows interpretable deep learning of protein aggregation.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mike Thompson, Mariano Mart√≠n, Trinidad Sanmart√≠n Olmo, Chandana Rajesh, Peter K Koo, Benedetta Bolognesi, Ben Lehner</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We train a convolution-attention hybrid neural network that accurately predicts aggregation from sequence by experimentally quantifying the aggregation of >100,000 protein sequences.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein aggregation is a pathological hallmark of more than 50 human diseases and a major problem for biotechnology. Methods have been proposed to predict aggregation from sequence, but these have been trained and evaluated on small and biased experimental datasets. Here we directly address this data shortage by experimentally quantifying the aggregation of >100,000 protein sequences. This unprecedented dataset reveals the limited performance of existing computational methods and allows us to train CANYA, a convolution-attention hybrid neural network that accurately predicts aggregation from sequence. We adapt genomic neural network interpretability analyses to reveal CANYA's decision-making process and learned grammar. Our results illustrate the power of massive experimental analysis of random sequence-spaces and provide an interpretable and robust neural network model to predict aggregation.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40305601/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Machine Learning-Enhanced Cerebrospinal Fluid N-Glycome for the Diagnosis and Prognosis of Primary Central Nervous System Lymphoma.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tao Liu, Huaizu Guo, Qing Li, Kun Chen, Jin Xu, Yan Ma, Zhiguang Lin, Xinli Zhou, Bobin Chen</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study on the potential use of machine learning algorithms in diagnosing and predicting the prognosis for PCNSL based on cerebrospinal fluid (CSF) N-glycomics .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The diagnosis and prognosis of Primary Central Nervous System Lymphoma (PCNSL) present significant challenges. In this study, the potential use of machine learning algorithms in diagnosing and predicting the prognosis for PCNSL based on cerebrospinal fluid (CSF) N-glycomics was investigated. First, CSF samples obtained from a cohort of 60 PCNSL patients and 30 controls were analyzed by hydrophilic interaction-based ultra performance liquid chromatography (HILIC-UPLC)-fluorescence mass spectrometry. Subsequently, nine machine learning models were established to diagnose PCNSL based on the changes of CSF N-glycome, with the Random Forest algorithm proving to be the most effective, achieving an accuracy of 100% in the training set and 89.3% in the test set. Moreover, a COX proportional-hazard model and a nomogram incorporating CSF N-glycome (GP6 and GP27) along with clinical data (age) were crafted. This nomogram's discrimination capacity was considered satisfactory, as evidenced by a C-index of 0.804 (95% CI: 0.68, 0.927). The study reveals that machine learning models based on CSF N-glycome offer a valuable approach for diagnosing and prognosticating PCNSL, demonstrating high accuracy and sensitivity in both classification and survival analysis. These findings may offer new insights into the molecular mechanisms underlying PCNSL and contribute to the advancement of personalized medicine for patients with this disease.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40259603/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Deep Learning-Based Prediction of Decoy Spectra for False Discovery Rate Estimation in Spectral Library Searching.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chak Ming Jerry Chan, Dominik Madej, Chun Kit Jason Chung, Henry Lam</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We explore the shuffle-and-predict decoy library generation approach, which can generate decoy spectra without the need for template spectra.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> With the advantage of extensive coverage, predicted spectral libraries are becoming an attractive alternative in proteomic data analysis. As a popular false discovery rate estimation method, target decoy search has been adopted in library search workflows. While existing decoy methods for curated experimental libraries have been tested, their performance in predicted library scenarios remains unknown. Current methods rely on perturbing real spectra templates, limiting the diversity and number of decoy spectra that can be generated for a given library. In this study, we explore the shuffle-and-predict decoy library generation approach, which can generate decoy spectra without the need for template spectra. Our experiments shed light on decoy method performance for predicted library scenarios and demonstrate the quality of predicted decoys in FDR estimation.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40252226/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Perspectives on the role of "-Omics" in predicting response to immunotherapy.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Anna Maria Di Giacomo, Sumit Subudhi, Wim Vos, Massimo Andreatta, Santiago Carmona, Will McTavish, Barbara Seliger, Ramy Ibrahim, Michael Lahn, Michael Smith, Alexander Eggermont, Bernard A Fox, Michele Maio</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-03-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> The annual Immuno-Oncology "Think Tank" held in October 2023 in Siena reviewed the rapidly evolving systems-biological approaches which are now providing a deeper understanding of tumor and tumor microenvironment heterogeneity.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The annual Immuno-Oncology "Think Tank" held in October 2023 in Siena reviewed the rapidly evolving systems-biological approaches which are now providing a deeper understanding of tumor and tumor microenvironment heterogeneity. Based on this understanding opportunities for novel therapies may be identified to overcome resistance to immunotherapy. There is increasing evidence that malignant disease processes are not limited to purely intracellular or genetic events but constitute a dynamic interaction between the host and disease. Tumor responses are influenced by many host tissue determinants across different cellular compartments, which can now be investigated by high-throughput molecular profiling technologies, often labelled with a suffix "-omics". "Omics" together with ever increasing computational power, fast developments in machine learning, and high-resolution detection tools offer an unrivalled opportunity to connect high-dimensional data and create a holistic view of disease processes in cancer. This review describes advances in several state-of-the-art "-omics" approaches with perspectives on how these can be applied to the clinical development of new immunotherapeutic strategies and ultimately adopted in clinical practice.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40168935/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

