<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Tuesday, 15 July 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Library-based virtual match-between-runs quantification in GlyPep-Quant improves site-specific glycan identification.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> He Zhu, Zheng Fang, Lei Liu, Yan Wang, Hongqiang Qin, Yongzhan Nie, Mingming Dong, Mingliang Ye</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present GlyPep-Quant, a tool for sensitive quantification and identification of site-specific glycans at proteomics scale, enabling over eightfold more site- specific glycan identification/quantification than conventional MS2-based methods.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Glycosylation changes are closely related to various diseases, including cancer. The quantitative analysis of site-specific glycans at proteomics scale remains challenging due to low glycopeptide spectra interpretation. Here, we present GlyPep-Quant, a tool for sensitive quantification and identification of site-specific glycans. Using a well-trained machine learning model, GlyPep-Quant quantified 25.1%-178.9% more site-specific glycans without missing values than pGlycoQuant, MSFragger-Glyco, and Skyline. To utilize identified information from previous large-scale dataset, an MS1 feature library-based "virtual match-between-runs" quantification scheme was developed, enabling over eightfold more site-specific glycan identification/quantification than conventional MS2-based methods. Enhanced coverage prompted the development of a glycoproteomic biomarker discovery method, involving calculation of site-specific glycan abundances ratios at the same glycosylation site, minimizing individual expression and experimental condition variability. Two pairs of site-specific glycan ratios on sites P01011-N127 and P08185-N96, were selected as high-performance biomarkers to classify gastric cancer (GC) from healthy controls (AUC > 0.95). Moreover, the two ratios performed well in distinguishing GC using an independent cohort by the library-based quantification strategy with diagnostic accuracy up to 85%. GlyPep-Quant is poised for broader glycoproteomic applications.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40659630/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Influential landmarks</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dujardin, J.-P., Sriwichai, P., Samung, Y., Ruangsittichai, J., Sumruayphol, S., Dujardin, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that small subsets of landmarks can outperform the full landmark sets in species discrimination.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Geometric morphometrics based on two-dimensional landmarks is a powerful tool for distinguishing morphologically similar or cryptic taxa of medically and veterinary important arthropods. While it is commonly assumed that increasing the number of landmarks should improve discriminatory power by capturing more shape information, our findings challenge this assumption. We demonstrate that small subsets of landmarks can outperform the full landmark sets in species discrimination. Examples are given in 6 insect families: Culicidae, Glossinidae, Muscidae, Psychodidae, Reduviidae and Tabanidae. We compared the classification accuracy of complete landmark configurations (from 10 to 22 landmarks) with smaller subsets of them. In all of these examples where landmark-based geometric morphometry was effective in separating morphologically close taxa, the total number of landmarks was not as effective as some significantly smaller subsets. To find such performing subsets, we used a random approach. Thus, for each number of landmarks (subsets), we examined a random sample of their many possible combinations. This random search was compared to a simpler approach, called the hierarchical method, based on the contribution of each landmark to the overall distance between shapes. Both procedures have been integrated into the XYOM online software, providing accessible tools for efficient landmark selection and improved morphometric analysis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.14.664645v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

