<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Monday, 21 April 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Knowledge-driven annotation for gene interaction enrichment analysis</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Gene Set Enrichment Analysis (GSEA) is a cornerstone for interpreting gene expression data, yet traditional approaches overlook gene interactions by focusing solely on individual genes, limiting their ability to detect subtle or complex pathway signals. To overcome this, we present GREA (Gene Interaction Enrichment Analysis), a novel framework that incorporates gene interaction data into enrichment analysis. GREA replaces the binary gene hit indicator with an interaction overlap ratio, capturing the degree of overlap between gene sets and gene interactions to enhance sensitivity and biological interpretability. It supports three enrichment metrics: Enrichment Score (ES), Enrichment Score Difference (ESD) from a Kolmogorov-Smirnov-based statistic, and Area Under the Curve (AUC) from a recovery curve. GREA evaluates statistical significance using both permutation testing and gamma distribution modeling. Benchmarking on transcriptomic datasets related to respiratory viral infections shows that GREA consistently outperforms existing tools such as blitzGSEA and GSEApy, identifying more relevant pathways with greater stability and reproducibility. By integrating gene interactions into pathway analysis, GREA offers a powerful and flexible tool for uncovering biologically meaningful insights in complex datasets. The source code is available at https://github.com/compbioclub/GREA.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.15.649030v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Widespread discordance between mRNA expression, protein abundance and de novo lipogenesis activity in hepatocytes during the fed-starvation transition</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">The mammalian liver plays a critical role in maintaining metabolic homeostasis during fasting and feeding. Liver function is further shaped by sex dimorphism and zonation of hepatocytes. To explore how these factors interact, we performed deep RNA-sequencing and label-free proteomics on periportal and pericentral hepatocytes isolated from male and female mice under fed and starved conditions. We developed a classification system to assess protein-mRNA relationship and found that gene products (mRNA or protein) for most zonation markers showed strong concordance between mRNA and protein. Although classical growth hormone regulated sex-biased gene products also exhibited concordance, ~60% of sex-biased gene products showed protein-level enrichment without corresponding mRNA differences. In contrast, transition between feeding and starvation triggered widespread changes in mRNA expression without significantly affecting protein levels. In particular, key lipogenic mRNAs (e.g. Acly, Acaca, and Fasn) were dramatically induced by feeding, but their corresponding proteins (ACLY, ACC1, and FAS) showed little to no change even as functional de novo lipogenic activity increased ~28-fold in the fed state. To facilitate further exploration of these findings, we developed Discorda (https://shinoda-lab.shinyapps.io/discorda/), a web database for interactive analysis. Our findings reinforce the principle that mRNA changes do not reliably predict corresponding protein levels (and vice versa), particularly in the context of sex and acute metabolic regulation of hepatocytes, and that de novo lipogenesis activity can be completely uncoupled from changes in protein expression.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.15.649020v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CArP - CApture-based sequencing for Pathogen surveillance in complex matrices</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">This study highlights the development and application of a novel capture-based long-read sequencing approach, CArP, designed to benefit pathogen surveillance. This combines targeted gene enrichment with the contextual insights of long-read sequencing, providing a highly efficient and flexible approach for pathogen detection. In this initial version, antimicrobial resistance (AMR) genes are utilized as marker genes that improve sequencing sensitivity. This allowed the detection of bacteria in broiler caecal content samples that carried resistance genes for critical antibiotics, including macrolides and quinolones. These specific AMR genes would have remained undetected when traditional metagenomic shotgun sequencing would have been applied, as it failed to detect these genes and therefore putative pathogens. This demonstrates the potential to detect and characterize pathogens more effectively compared to traditional sequencing strategies. The modular design of CArP furthermore demonstrates expansion with additional marker genes, such as viral or fungal markers, that can enable broader pathogen coverage. Taken together, this approach allows to advance pathogen detection and can contribute to global efforts in pandemic preparedness and the subsequent response.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.20.648581v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Topography Aware Optimal Transport for Alignment of Spatial Omics Data</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Understanding the spatial organization of tissues is essential for uncovering cellular communication, developmental processes, and disease mechanisms. Recent advances in spatial omics technologies have provided unprecedented insight into tissue spatial organization, but challenges remain in aligning spatial slices and integrating complementary single-cell and spatial data. Here, we propose TOAST (Topography-aware Optimal Alignment of Spatially-resolved Tissues), a novel OT-based framework that extends the classical Fused Gromov-Wasserstein (FGW) objective to more comprehensively model the heterogeneity of local molecular interactions. By introducing spatial coherence, quantified through the entropy of local neighborhoods, and neighborhood consistency, which preserves the expression profiles of neighboring spots, TOAST's objective function significantly improves the alignment of spatially resolved tissue slices and the mapping between single-cell and spatial data. Through comprehensive evaluations on both simulated and real-world datasets, including human brain cortex Visium data, Axolotl brain Stereo-seq data, mouse embryo seqFISH data, and Imaging Mass Cytometry from multiple cancer types, we demonstrate that our method consistently outperforms traditional FGW and other OT-based alignment methods. Specifically, TOAST improves the accuracy of spatial slice alignment, better preserves cell type compositions, recovers lineage trajectories in developmental brain data, and reconstructs spatial relationships in spatial transcriptomics mouse embryo data. By integrating spatial constraints into OT, our framework provides a principled approach to enhance the biological interpretability of spatially resolved omics data and facilitate multimodal data integration.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.15.648894v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Accelerated, Accurate, Hybrid Short and Long Reads Alignment and Variant Calling</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify;">Background: Integrating short-read and long-read sequencing technologies has become a promising approach for achieving accurate and comprehensive genomic analysis. While short-read sequencing (Illumina, etc.) offers high base accuracy and cost efficiency, it struggles with structural variation (SV) detection and complex genomic regions. In contrast, long-read sequencing (PacBio HiFi) excels in resolving large SVs and repetitive sequences but is limited by throughput, higher error rates (especially Indels), and sequencing costs. Hybrid approaches may combine these technologies and leverage their complementary strengths and different sources of error to provide higher accuracy, more comprehensive results, and higher throughput by lowering the coverage requirement for the long reads. Methods: This study benchmarks the DNAscope Hybrid pipeline, a novel integrated alignment and variant calling framework that combines short- and long-read data sequenced from the same sample. We evaluate its performance across multiple human genome reference datasets (HG002-HG004) using the draft Q100 and Genome in a Bottle v4.2.1 benchmarks. The pipeline's ability to detect small variants (SNPs/Indels), structural variants (SVs), and copy number variations (CNVs) is assessed using data from the Illumina and Pacbio sequencing systems at varying read depths (5x-30x). Benchmark results are compared to DeepVariant. Results: The DNAscope Hybrid pipeline significantly improves SNP and Indel calling accuracy, particularly in complex genomic regions. At lower long-read depths (e.g., 5x-10x), the hybrid approach outperforms standalone short- or long-read pipelines at full sequencing depths (30x-35x). Additionally, the DNAscope Hybrid outperforms leading open-source tools for SV and CNV detection, enhancing variant discovery in challenging genomic regions. The pipeline also demonstrates clinical utility by identifying disease-associated variants. Moreover, DNAscope Hybrid is highly efficient, achieving less than 90 minutes runtimes at single standard instance. Conclusion: The DNAscope Hybrid pipeline is a computationally efficient, highly accurate variant calling framework that leverages the advantages of both short- and long-read sequencing. By improving variant detection in challenging genomic regions and offering a robust solution for clinical and large-scale genomic applications, it holds significant promise for genetic disease diagnostics, population-scale studies, and personalized medicine.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.15.648987v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Harmonizing Heterogeneous Single-Cell Gene Expression Data with Individual-Level Covariate Information</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">The growing availability of single-cell RNA sequencing (scRNA-seq) data highlights the necessity for robust integration methods to uncover both shared and unique cellular features across samples. These datasets often exhibit technical variations and biological differences, complicating integrative analyses. While numerous integration methods have been proposed, many fail to account for individual-level covariates or are limited to discrete variables. To address these limitations, we propose scINSIGHT2, a generalized linear latent variable model that accommodates both continuous covariates, such as age, and discrete factors, such as disease conditions. Through both simulation studies and real-data applications, we demonstrate that scINSIGHT2 accurately harmonizes scRNA-seq datasets, whether from single or multiple sources. These results highlight scINSIGHT2's utility in capturing meaningful biological insights from scRNA-seq data while accounting for individual-level variation. The scINSIGHT2 method has been implemented as an R package, which is available at https://github.com/yudimu/scINSIGHT2/.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.15.649009v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Facilitating genome annotation using ANNEXA and long-read RNA sequencing</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">With the advent of complete genome assemblies, genome annotation has become essential for the functional interpretation of genomic data. Long-read RNA sequencing (LR-RNAseq) technologies have significantly improved transcriptome annotation by enabling full-length transcript reconstruction for both coding and non-coding RNAs. However, challenges such as transcript fragmentation and incomplete isoform representation persist, highlighting the need for robust quality control (QC) strategies. This study presents an updated version of ANNEXA, a pipeline designed to enhance genome annotation using LR-RNAseq data while also providing QC for reconstructed genes and transcripts. ANNEXA integrates two transcriptome reconstruction tools, StringTie2 and Bambu, applying stringent filtering criteria to improve annotation accuracy. It also incorporates deep learning models to evaluate transcription start sites (TSSs) and employs the tool FEELnc for the systematic annotation of long non-coding RNAs (lncRNAs). Additionally, the pipeline offers intuitive visualizations for comparative analyses of coding and non-coding repertoires. Benchmarking against multiple reference annotations revealed distinct patterns of sensitivity and precision for both known and novel genes and transcripts and mRNAs and lncRNAs. To demonstrate its utility, ANNEXA was applied in a comparative oncology study involving LR-RNAseq of two human and eight canine cancer cell lines. The pipeline successfully identified novel genes and transcripts across species, expanding the catalog of protein-coding and lncRNA annotations in both species. Implemented in Nextflow for scalability and reproducibility, ANNEXA is available as an open-source tool: https://github.com/IGDRion/ANNEXA.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.16.648718v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Fast and accessible morphology-free functional fluorescence imaging analysis</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Optical calcium imaging is a powerful tool for recording neural activity across a wide range of spatial scales, from dendrites and spines to whole-brain imaging through two-photon and widefield microscopy. Traditional methods for analyzing functional calcium imaging data rely heavily on spatial features, such as the compact shapes of somas, to extract regions of interest and their associated temporal traces. This spatial dependency can introduce biases in time trace estimation and limit the applicability of these methods across different neuronal morphologies and imaging scales. To address these limitations, the Graph Filtered Temporal Dictionary Learning (GraFT) uses a graph-based approach to identify neural components based on shared temporal activity rather than spatial proximity, enhancing generalizability across diverse datasets. Here we present significant advancements to the GraFT algorithm, including the integration of a more efficient solver for the L1 least absolute shrinkage and selection operator (LASSO) problem and the application of compressive sensing techniques to reduce computational complexity. By employing random projections to reduce data dimensionality, we achieve substantial speedups while maintaining analytical accuracy. These advancements significantly accelerate the GraFT algorithm, making it more scalable for larger and more complex datasets. Moreover, to increase accessibility, we developed a graphical user interface to facilitate running and analyzing the outputs of GraFT. Finally, we demonstrate the utility of GraFT to imaging data beyond meso-scale imaging, including vascular and axonal imaging.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.15.648462v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A comprehensive pipeline for genome annotation across species: a case study on Schmidtea mediterranea</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify;">Background Despite advancements in genome annotation tools, challenges persist for non-classical model organisms with limited genomic resources, such as Schmidtea mediterranea. Findings To address these challenges, we developed a flexible and scalable genome annotation pipeline that integrates short-read (Illumina) and long-read (PacBio) sequencing technologies. The pipeline combines reference-based and de novo assembly methods, effectively handling genomic variability and alternative splicing events. Functional annotation is conducted to filter out low-confidence transcripts and ensure biological relevance. Applying this pipeline to the asexual strain of S. mediterranea revealed thousands of previously undescribed putative genes and transcripts, and improved the existing gene models, highlighting its utility in annotating complex, underexplored genomes. Conclusions The modularity and comprehensiveness of our pipeline ensure its adaptability for genome annotation across diverse species, making it a valuable tool for annotating genomes of non-model organisms and supporting broader genomic research. The source code and implementation details are available at https://github.com/Norreanea/SmedAnno.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.16.649076v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Annotation of 200 Insect Genomes with BRAKER for Consistent Comparisons across Species</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">The annotation of genomes lags behind their sequencing and assembly. For example, of 5,092 insect species in GenBank, as one of the most widely used databases, only 375 currently have annotated genomes within this database. Additionally, species that were previously annotated can benefit from reannotation using RNA-Seq and protein data that have been added since their last annotation, as well as from state-of-the-art annotation methods, whose accuracy has improved. Heterogeneous annotations performed with different tools and protein databases can introduce artifactual differences when comparing gene sets or gene structures between species. Recently, the BRAKER3 annotation pipeline was introduced that integrates evidence from RNA-Seq and from a protein database. It was benchmarked as one of the most accurate annotation methods. Here, we introduce an automated genome annotation workflow that allows to annotate a list of species with BRAKER3 and VARUS for RNA-Seq retrieval, or in the absence of transcriptome data, with BRAKER2, with minimal manual intervention. We selected a diverse set of 200 insect species from different families, including 85 species previously lacking annotations in GenBank. Using currently available RNA-Seq and protein sequence data, we applied our automated workflow to annotate these genomes and conducted downstream analyses typically performed in comparative genomics studies. We present the resulting gene structures, protein sequences, gene ontology terms, orthologous gene groups and a species tree.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.17.649312v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

