<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Sunday, 28 September 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Fusing Sequence Motifs and Pan-Genomic Features: Antimicrobial Resistance Prediction using an Explainable Lightweight 1D CNN - XGBoost Ensemble</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Siddiqui, M. S. B., Tarannum, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose AMR-EnsembleNet, an ensemble framework that synergistically combines sequence-based and feature-based learning, overcoming the limitations of using either an order-agnostic or a standalone sequence model.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Antimicrobial Resistance (AMR) is a rapidly escalating global health crisis. While genomic sequencing enables rapid prediction of resistance phenotypes, current computational methods have limitations. Standard machine learning models treat the genome as an unordered collection of features, ignoring the sequential context of Single Nucleotide Polymorphisms (SNPs). State-of-the-art sequence models like Transformers are often too data-hungry and computationally expensive for the moderately-sized datasets that are typical in this domain. To address these challenges, we propose AMR-EnsembleNet, an ensemble framework that synergistically combines sequence-based and feature-based learning. We developed a lightweight, custom 1D Convolutional Neural Network (CNN) to efficiently learn predictive sequence motifs from high-dimensional SNP data. This sequence-aware model was ensembled with an XGBoost model, a powerful gradient boosting system adept at capturing complex, non-local feature interactions. We trained and evaluated our framework on a benchmark dataset of 809 E. coli strains, predicting resistance across four antibiotics with varying class imbalance. Our 1D CNN-XGBoost ensemble consistently achieved top-tier performance across all the antibiotics, reaching a Matthews Correlation Coefficient (MCC) of 0.926 for Ciprofloxacin (CIP) and the highest Macro F1-score of 0.691 for the challenging Gentamicin (GEN) AMR prediction. We also show that our model consistently focuses on SNPs within well-known AMR genes like fusA and parC, confirming it learns the correct genetic signals for resistance. Our work demonstrates that fusing a sequence-aware 1D CNN with a feature-based XGBoost model creates a powerful ensemble, overcoming the limitations of using either an order-agnostic or a standalone sequence model.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.27.678993v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Augmented prediction of multi-species protein--RNA interactions using evolutionary conservation of RNA-binding proteins</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> He, J., Zhou, T., Hu, L.-F., Jiao, Y., Wang, J., Yan, S., Jia, S., Chen, Q., Wang, Y., Yang, Y. T., Sun, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce MuSIC (Multi-Species RBP--RNA Interactions using Conservation), a deep learning-based framework for predicting cross-species RBP-RNA interactions by leveraging label smoothing and evolutionary conservation of RBPs across 11 diverse species ranging from human to yeast.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> RNA-binding proteins (RBPs) play critical roles in gene expression regulation. Recent studies have begun to detail the RNA recognition mechanisms of diverse RBPs. However, given the array of RBPs studied so far, it is implausible to experimentally profile RBP-binding peaks for hundreds of RBPs in multiple non-model organisms. Here, we introduce MuSIC (Multi-Species RBP--RNA Interactions using Conservation), a deep learning-based framework for predicting cross-species RBP--RNA interactions by leveraging label smoothing and evolutionary conservation of RBPs across 11 diverse species ranging from human to yeast. MuSIC outperforms state-of-the-art computational methods, and provides predicted RBP-binding peaks across species with high accuracy. The prediction confidence is higher in the closely related species, partially due to the RBP conservation patterns. Finally, the effects of homologous genetic variants on RBP binding can be computationally quantified across species, followed by experimental validations. The target transcripts with disrupted binding events are enriched with the ubiquitination-associated pathways. To summarize, MuSIC provides a useful computational framework for predicting RBP--RNA interactions cross-species and quantifying the effects of genetic variants on RBP binding, offering novel insights into the RBP-mediated regulatory mechanisms implicated in human diseases.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.27.678913v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Conserved Filovirus Proteins as Targets of Broad-Spectrum Antivirals</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Scotti, M., Kelestemur, E., Martin, H.-J., Novy, B., Melo-Filho, C., Barros, R., Herrera-Acevedo, C., Tropsha, A., Muratov, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify and characterize conserved binding sites within key filoviral proteins to support the development of broad-spectrum, direct-acting antiviral agents. We validated the significance of these conserved regions for drug discovery using existing experimental data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Filoviruses are enveloped, non-segmented, negative-strand RNA viruses belonging to the Filoviridae family, which includes five genera: Ebolavirus, Marburgvirus, Cuevavirus, Striavirus, and Thamnovirus. Members of this family cause severe and, often, fatal hemorrhagic fevers in humans and non-human primates, with high mortality rates. To date, only two filoviruses, Ebola virus (EBOV) and Marburg virus (MARV), are known to infect humans and are listed as priority pathogens by the World Health Organization due to their potential for re-emergence and the current lack of effective vaccines and antiviral treatments. In this study, we identify and characterize conserved binding sites within key filoviral proteins to support the development of broad-spectrum, direct-acting antiviral agents. We validated the significance of these conserved regions for drug discovery using existing experimental data. Our analysis revealed notably high sequence similarity among proteins from filoviruses capable of infecting humans (EBOV, TAFV, BDBV, SUDV, MARV, and RAVV) compared to those from non-zoonotic species, with the highest conservation observed in the L and VP40 proteins - both critical for viral genome transcription and replication. Furthermore, we compiled and analyzed available experimental data on known antiviral compounds targeting these proteins, identifying several agents with cross-filovirus activity, including Galidesivir, Remdesivir, and Favipiravir. The integrated approach described here - combining sequence and structural conservation analysis with chemical structure and antiviral activity data - demonstrates a strategy that could be extended to the development of broad-spectrum therapeutics across multiple viral families.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.26.678902v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An evolved strain of Spathaspora passalidarum produces ethanol from sugarcane bagasse and switchgrass lignocellulosic hydrolysates</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Barros, K. O., Fisher, K. J., Crandall, J. G., Magni, G., Serate, J., Xie, D., Zhang, Y., Silva, S. S., Sato, T. K., Hittinger, C. T., Rosa, C. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study aimed to enable the growth and ethanol production by the xylose-fermenting yeast Spathaspora passalidarum in the presence of lignocellulosic hydrolysate inhibitors, particularly acetic acid.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Lignocellulosic hydrolysates, derived from plant biomass, contain various inhibitors that can hinder microbial growth. This study aimed to enable the growth and ethanol production by the xylose-fermenting yeast Spathaspora passalidarum in the presence of lignocellulosic hydrolysate inhibitors, particularly acetic acid. Ultraviolet (UV)-induced mutagenesis and adaptive laboratory evolution (ALE) were used to select for mutants with higher tolerance to these inhibitors. The initial mutant strain, MT01, was selected for increased growth in medium containing xylose and acetic acid. This strain underwent further evolution, resulting in the strain ME3.5.5, which showed significant improvements in both growth and ethanol production compared to the parental strain when tested in sugarcane bagasse hemicellulosic hydrolysate (SBHH). Genomic analysis identified non-synonymous and frameshift mutations in four genes, including CYR1 (encoding adenylate cyclase). These findings suggest that genetically optimized S. passalidarum strains could play a crucial role in advancing industrial bioethanol production from lignocellulosic biomass by overcoming the inhibitory effects of compounds found in lignocellulosic hydrolysates. Keywords: mutagenesis, Adaptive Laboratory Evolution (ALE), Spathaspora passalidarum, acetic acid, hemicellulosic hydrolysates.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.26.678828v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Retrocopy formation and domestication shape genome evolution in sloths and other xenarthrans</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Uliano-Silva, M., Beatriz da Conceicao, H., L. V. Mercuri, R., Winkler, S., D. A. Guardia, G., Myers, E. W., McCarthy, S., Tracey, A., Suh, A., Blaxter, M., A.F. Galante, P., J. Mazzoni, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present chromosome-level genomes for the two-toed sloth and southern anteater, and investigate how retrotransposon-mediated gene duplications (retrocopies) have shaped xenarthran genome evolution.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Xenarthrans, comprising sloths, anteaters, and armadillos, represent one of the most morphologically and physiologically specialised mammalian clades, yet the genomic basis of their adaptations remains poorly understood. Here, we present chromosome-level genomes for the two-toed sloth () and the southern anteater (), and investigate how retrotransposon-mediated gene duplications (retrocopies) have shaped xenarthran genome evolution. Comparative analyses revealed that xenarthran genomes harbor the highest number of retrocopies reported among mammals, with lineage-specific insertion dynamics. Anteater and armadillo genomes contain older LINE1 repertoires and species-specific older retrocopy insertions. In contrast, sloths retain both an abundance of young LINE1s and thousands of young retrocopies, alongside a large shared set that originated from an evolutionary burst of retroduplication in the branch leading to their last common ancestor (~30 Mya). In C. didactylus, approximately 50% of retrocopies are expressed, compared with 24% in Dasypus novemcinctus. Evolutionary analyses identified 38 retrocopies with strong hallmarks of domestication in C. didactylus. Many of these retrocopies derive from parental genes involved in mitochondrial and metabolic processes, suggesting a genomic mechanism underlying the physiological specialisations of sloths. Altogether, our findings identify retrotransposition as a major contributor to the genomic architecture of Xenarthra and highlight retrocopy origination as a mechanism for generating lineage-specific novelty and distinctive biological specialisations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.25.678567v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Aphid-derived cross-kingdom RNA dynamics underpin maize resistance</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jiang, S., Zhang, Z., Liu, C., Zhu, Y., Kou, Y., Yang, P., Hu, Z., Wu, J., Wang, Y., Wan, F., Wu, G., Chen, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Aphid-specific long non-coding RNAs from the Ya gene family from the aphid- specific Ya genefamily from the cereal-specialist aphids Rhopalosiphum maidis and R. padi.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Aphids are major pests of maize (Zea mays), yet the molecular mechanisms underlying their interactions with host plants remain poorly understood. Here, we identified and functionally characterized long non-coding RNAs (lncRNAs) from the aphid-specific Ya gene family in the cereal-specialist aphids Rhopalosiphum maidis and R. padi. We showed that Ya genes formed lineage-specific clusters and were transcriptionally active across both species. Multiple Ya transcripts secreted into maize tissues exhibited remarkable stability compared with rapidly degraded aphid mRNAs and migrated systemically, as visualized in planta using an RNA switch-controlled RNA-triggered fluorescence system. RNA interference of Ya genes significantly reduced aphid fecundity, while ectopic expression of Ya lncRNAs in maize enhanced aphid colonization. Importantly, maize genotypes differing in aphid resistance selectively influenced the persistence of aphid mRNAs but not Ya lncRNAs, indicating a decoupled fate of distinct aphid RNA classes in planta. These findings establish Ya lncRNAs as cross-kingdom effectors critical for aphid virulence, and suggest new molecular markers for breeding aphid-resistant maize.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.25.678037v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The reproductive microbiome inhibits pollen germination in milkweed</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dalgleish, H. J., Puzey, J., Barlow, K., Cunningham, O., Davies, E., Machiorlete, H., Waynick, G., Williamson, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the stigmatic microbiome in milkweeds influences the success of pollination, with many microbial taxa being neutral, but some being deleterious.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We know very little about the reproductive microbiomes of plants. Microbes may play important roles in shaping pollination, fertilization, and seed production - processes which are important evolutionarily, ecologically, and agriculturally. Through a series of field and laboratory experiments, we show that the stigmatic microbiome in milkweeds influences the success of pollination. Isolation of individual bacterial and fungal taxa from stigmatic secretions allowed us to experimentally test their effects on pollen germination. These experiments demonstrate that individual taxa impact pollen differently, with many microbial taxa being neutral, but some being deleterious. Through isolation of microbes from the legs of pollinator insects we found that pollinators are a likely source for pollen-harmful bacterial taxa. Next, by utilizing a natural hybrid zone, we demonstrate species-specific responses to the stigmatic microbiome that be driving asymmetric patterns of gene-flow between species - with Asclepias exaltata being a better pollen host than A. syriaca. This study demonstrates that the reproductive microbiome is an underappreciated player in sexual reproduction of plants.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.25.678039v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Adverse Effects of UV-Exposure on DNA Strand Displacement Reactions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sesli, K., Zhao, Y., Kallish, N. L., Scalise, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> UV-shadowing (a standard technique for visualizing DNA during PAGE purification) is a previously overlooked cause of DSD circuit errors, and we introduce a simple and practical purification protocol that avoids UV exposure, and show that the errors no longer occur.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> DNA strand displacement (DSD) reactions are widely used in molecular computing and nanotechnology due to their programmability and precise control over molecular interactions. However, experimental DSD systems often underperform relative to theoretical models, due in part to poorly characterized sources of reactant impurities. Here, we identify UV-shadowing (a standard technique for visualizing DNA during PAGE purification) as a previously overlooked cause of DSD circuit errors. Counterintuitively, existing purification protocols with UV-shadowing can increase impurities that disrupt circuit behavior. Specifically, we demonstrate that UV exposure (i) reduces DSD reaction percent yield and (ii) leads to a phenomenon we call "negative leak," where high-concentration double-stranded complexes that are designed to be inert can sequester functional signal strands and thus disrupt circuit performance. Additionally, we find that UV damage is sequence-dependent, with adjacent pyrimidine-pair-rich strands being more susceptible to errors. To circumvent these errors, we introduce a simple and practical purification protocol that avoids UV exposure, and show that the errors no longer occur. Our results highlight UV-induced damage as a critical factor in DSD circuit performance, and suggest that UV-shadowing may have contributed to significant reproducibility and scaling challenges in the broader DSD literature.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.27.673911v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A platform of robust patient-derived leukemia models covering subgroups for which no cell lines exist</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Vick, B., Jurinovic, V., Kuhbandner, K., Lagally, L., Latzko, L., Arnreich, C., HaÃànel, G., Muth, A., Rothenberg-Thurley, M., Dufour, A., Schneider, S., Holdt, L. M., Mura, L., Klein, F., Frank, A., Andre, M. C., Baldus, C. D., Carroll, M., Dierks, C., Ebinger, M., GoÃàtze, K. S., Menendez, P., Recher, C., Sahal, A., Sarry, J.-E., Thiede, C., Velasco-Hernandez, T., Wei, X., Klusmann, J.-H., von Bergweld-Baildon, M., Hiddemann, W., Metzeler, K. H., Greif, P. A., Subklewe, M., Vosberg, S., Herold, T., Spiekermann, K., Jeremias, I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We generate patient-derived xenografts from acute myeloid leukemia (AML) patients that can be used in preclinical cancer research and accelerate translational research.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Preclinical cancer research requires robust model systems, especially for poor prognosis entities like acute myeloid leukemia (AML), a highly aggressive blood cancer. Here, primary tumor cells from 137 AML patients of all age groups were transplanted into immune compromised mice to generate patient-derived xenografts (PDX). From these, 23 models enable robust, virtually endless serial re-transplantation and are amenable to lentiviral genetic engineering (*PDX AML models). These models primarily originate from patients with highly aggressive, relapsed disease. Comprehensive genomic, transcriptomic, and epigenomic analyses confirmed that they replicate primary AML biology more faithfully than conventional cell lines. Notably, *PDX AML models include AML subgroups that are underrepresented or absent in existing model systems, such as cytogenetically normal or IDH1/2-mutant AML. They withstand freeze-thaw cycles, making them suitable for broad distribution and reproducibility across research institutions. Luciferase-based in vivo imaging enables real-time monitoring of tumor progression and treatment responses in preclinical trials. Surprisingly, long-term treatment, including repeated cytarabine therapy over a period of one year, showed a gradual reduction in leukemia cell proliferation, which decreased continuously after each treatment block. Collectively, our *PDX models represent a robust, versatile, and relevant platform that holds great promise to accelerate translational research for the benefit of cancer patients.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.26.677299v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Machine learning and statistical inference in microbial population genomics.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Samuel K Sheppard, Nicolas Arning, David W Eyre, Daniel J Wilson</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We argue that the combination and synthesis of machine learning and statistics has potential for pathogen research in the big data era.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The availability of large genome datasets has changed the microbiology research landscape. Analyzing such data requires computationally demanding analyses, and new approaches have come from different data analysis philosophies. Machine learning and statistical inference have overlapping knowledge discovery aims and approaches. However, machine learning focuses on optimizing prediction, whereas statistical inference focuses on understanding the processes relating variables. In this review, we outline the different aspirations, precepts, and resulting methodologies, with examples from microbial genomics. Emphasizing complementarity, we argue that the combination and synthesis of machine learning and statistics has potential for pathogen research in the big data era.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41015769/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SoluProtMut: Siamese Deep Learning for Predicting Solubility Effects of Protein Mutations with Experimental Validation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Velecky, J., Faldynova, H., Hermosilla-Casajus, P., Sendlerova, N., Doerr, M., Egersdorfova, S., Bornscheuer, U., Damborsky, J., Prokop, Z., Mazurenko, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present SoluProtMut, a Siamese deep geometric neural network trained to predict the impact of mutations on protein solubility.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein solubility is an attractive engineering target because it is a critical property influencing the scalability of protein production and the success of therapeutic proteins in biomedical applications. However, predicting solubility changes upon mutation in silico is challenging due to data heterogeneity and protein bias. Here, we explore how different sources of solubility data can be used for machine learning and present SoluProtMut, a Siamese deep geometric neural network trained to predict the impact of mutations on protein solubility. Our final model was trained exclusively on deep mutational scanning data. We compare our model with five established solubility prediction methods. The model achieves state-of-the-art performance on an independent dataset of various proteins, especially in predicting the effects of multipoint mutations (informedness of 26.5 %). Our findings also reaffirm that the scarcity of solubility data continues to hamper progress in this field. To address this limitation, we experimentally quantified solubility changes for hundreds of single-point and multipoint mutants of haloalkane dehalogenase. Complemented with recent deep-mutational-scanning data on myoglobin, we employed both these data for external validation. Although the generalization to unseen proteins remains limited, our findings demonstrate the potential of integrating high-throughput assays with deep learning to improve the accuracy and scope of solubility prediction.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.26.676459v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identification of KKL-35 as a novel carnosine dipeptidase 2 (CNDP2) inhibitor by in silico screening</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Homma, T., Shinbara, K., Osaki, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel small molecule that inhibits CNDP2-mediated cancer cell metabolism and is a promising lead compound for the development of more selective therapeutics.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Extracellular glutathione (GSH) is degraded on the cell surface, in which the {gamma}-glutamyl residue is removed to generate cysteine-glycine (Cys-Gly) dipeptides that are subsequently transported to the cytoplasm. Carnosine dipeptidase II (CNDP2) is a cytoplasmic enzyme that hydrolyzes Cys-Gly and plays an important role in maintaining intracellular cysteine (Cys) homeostasis. CNDP2-mediated hydrolysis of Cys-Gly promotes Cys mobilization and contributes to the replenishment of intracellular GSH levels. CNDP2 is frequently overexpressed in various cancers and has been implicated in tumor cell proliferation and progression. This mechanism may enhance cancer cell survival by causing resistance to oxidative stress, which indicates that CNDP2 is a potential therapeutic target for cancer treatment. Although bestatin (BES) has been identified as a CNDP2 inhibitor, its limited specificity and suboptimal drug-like properties have limited its therapeutic potential. In this study, we performed an in silico screen of a small-molecule compound library and identified KKL-35 as a novel CNDP2-binding molecule. Molecular dynamics (MD) simulations suggested that KKL-35 interacts within the catalytic pocket. Biochemical assays confirmed that it inhibits CNDP2 enzymatic activity, albeit with lower potency compared with BES. Despite its modest intrinsic activity, KKL-35 exhibits favorable physicochemical and pharmacokinetic properties, which are characterized by a low topological polar surface area (TPSA), reduced molecular flexibility, and well-balanced lipophilicity. This positions it as an attractive and tractable starting point for lead optimization. Taken together, these findings establish KKL-35 as a validated CNDP2 inhibitor and a promising lead compound for the development of more selective therapeutics targeting CNDP2-mediated cancer cell metabolism.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.27.674371v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Machine Learning-Driven Prediction of TLR4 Binding Affinity: A Comprehensive Molecular Feature Analysis for Drug Discovery</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yee, B., Rutowski, M., Collins, W., Yee Collins Research Group</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a machine learning approach for predicting TLR4 binding affinity using comprehensive molecular descriptors. We identified key molecular scaffolds, including flavonoids and terpenoids, aligning with known natural productTLR4 modulators.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Toll-like receptor 4 (TLR4) represents a promising therapeutic target for inflammatory diseases and cancer, but developing selective modulators remains challenging. We present a machine learning approach for predicting TLR4 binding affinity using comprehensive molecular descriptors. Our ensemble learning pipeline extracted 53 physicochemical features from a curated dataset of 49 unique TLR4 ligands, achieving cross-validation R^2 of 0.74(+/-)0.10 with statistical significance confirmed by permutation testing (p < 0.01). The most predictive features were Bertz complexity (importance: 0.173), molecular shape descriptors (0.159), and molar refractivity (0.145), while traditional drug-like properties such as LogP showed lower importance (0.056). This suggests TLR4 binding follows distinct structure-activity patterns compared to conventional drug targets. Compounds with intermediate structural complexity (Bertz complexity: 400-600) demonstrated optimal binding affinity. The model successfully identified key molecular scaffolds, including flavonoids and terpenoids, aligning with known natural product TLR4 modulators. This work provides the first comprehensive machine learning analysis of TLR4 binding determinants and offers a computational framework for rational design of TLR4-targeted therapeutics, with identified molecular features providing actionable insights for developing next-generation immunomodulatory drugs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.26.676276v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Old Goats: 3,000 years of genetic connectivity of the domestic goat in Ireland</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Findlater, J., Erven, J. A. M., Siekmann, A., Mattiangeli, V., The Vargoats Consortium,, Murphy, E., Daly, K. G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A genetic analysis of Late Bronze and medieval goat remains from Haughey's Fort and Carrickfergus reveals a three-millenia genetic connection between herds in the Late Bronze Age, medieval period, and the indigenous Irish breed extant today, the Old Irish Goat.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The domestic goat likely first arrived to the island of Ireland as part of the introduction of agriculture approximately 5,900 years ago, and remains a part of the island's biocultural heritage. However, due to the challenges of differentiating goat remains from that of sheep using traditional archaeozoological approaches, there are few specimens specifically identified as goat. To address this we employed genetic, proteomics, and archaeozoological techniques to assess faunal remains from the Late Bronze assemblage of Haughey's Fort (Armagh) and medieval assemblage of Carrickfergus (Antrim). We identify these specimens as goats using proteomics and genetics, and additionally determine their molecular sex. Genomic data recovered from a Haughey's Fort goat reveals a three-millenia genetic connection between herds in the Late Bronze Age, medieval period, and the indigenous Irish breed extant today, the Old Irish Goat. We additionally find varying levels of inbreeding within goats from the settlement of Carrickfergus, suggesting possible mixed use of herds within medieval Irish society. Our results demonstrate the continuing potential of combining archaeological and biomolecular techniques to clarify existing ambiguities and at the same time reveal new facets of the past.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.26.678852v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Multi-Layered Atlas of Spatial Regulatory Programs and Therapeutic Vulnerabilities in Glioblastoma</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, L., Lu, M., Ma, X., Osmanbeyoglu, H. U.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce an integrated, multi layered framework that unifies transcription factor and pathway activity inference (STAN and SPAN) with ligand/receptor and drug target mapping, thereby constructing a systems-level regulatory atlas of GBM across 26 tumors.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Glioblastoma (GBM) is marked by profound spatial heterogeneity. This complexity fuels tumor progression and therapy resistance. Although recent spatial transcriptomics (ST) studies have defined transcriptional niches, or metaprograms, the systems level regulatory logic linking upstream signaling, transcriptional programs, and therapeutic vulnerabilities remains unresolved. Here, we introduce an integrated, multi layered framework that unifies transcription factor and pathway activity inference (STAN and SPAN) with ligand/receptor and drug target mapping, thereby constructing a systems-level regulatory atlas of GBM across 26 tumors. This atlas uncovers spatial niches organized around transcriptional hubs and signaling programs, including hypoxic mesenchymal regions coordinated by HIF1A and SOX2 with mTORC1 signaling, macrophage rich areas driven by STAT3 mediated immune modulation, and astrocyte-like states shaped by lipid metabolic regulators such as SREBF2. By coupling this regulatory atlas to drug target predictions, we identify niche specific therapeutic vulnerabilities, such as VEGF blockade in HIF1A high regions and MAO-B inhibition in mesenchymal states. Finally, we provide an open, interactive web resource to make these data broadly accessible. This work provides a generalizable blueprint for linking spatial regulation to therapeutic hypotheses across diverse human diseases.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.27.674707v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">multiDEGGs: a multi-omic differential network analysis package for biomarker discovery and predictive modeling</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sciacca, E., Wang, S., Pitzalis, C., Lewis, M. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present multiDEGGs, a CRAN R package that enables differential network analysis in multi-omic scenarios.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Modern clinical trials increasingly leverage high-throughput omic data for patient stratification and biomarker discovery. While traditional differential gene expression analysis disregards the networked nature of molecular entities and produces extensive gene lists with limited interpretability, differential network analysis has emerged as a crucial complementary analysis for comparative studies. Here we present multiDEGGs, a CRAN R package that enables differential network analysis in multi-omic scenarios. multiDEGGs uses a multi-layer graph framework to model omic data by leveraging an internal network of over 10 000 literature-validated biological interactions. For each data type, differential networks are generated, and the statistical significance of each link (p-values or adjusted p-values) is evaluated through robust linear regression with interaction terms. These networks are then integrated into a comprehensive visualisation that allows interactive exploration of cross-omic patterns. Beyond network visualization and exploration, multiDEGGs extends its utility to predictive modelling applications. The package facilitates seamless integration into cross-validation machine learning pipelines, serving as feature selection and augmentation tool. We validated multiDEGGs using two cohorts of rheumatoid arthritis patients who underwent tocilizumab and rituximab therapy, respectively. For each treatment group, multi-layer differential interactions were identified, and seven machine learning models were trained to predict treatment resistance using synovial RNA-seq data. We systematically compared multiDEGGs against five traditional feature selection methods. On average, AUC values obtained with multiDEGGs showed an improvement of 0.10 compared to conventional filters.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.25.678475v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Generative design of synthetic gene circuits for functional and evolutionary properties</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gallup, O., Steel, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that even simple generative models like the conditional variational autoencoder (CVAE) can produce novel genetic circuits that match complex dynamic functions such as signal adaptation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In the past decades, a wide suite of design tools for biological systems have been developed, but using these to create biotechnologies that achieve reliable and predictable behaviour remains challenging. Modelling approaches have enabled researchers to traverse the vast search space of genetic circuits more efficiently, while machine learning has proven useful for designing parts and predicting their function or evolutionary properties. Generative algorithms have the potential to leverage these features to design entire genetic circuits from the sequence level, but have only recently begun to be applied to synthetic biological applications. Here, we show that even simple generative models like the conditional variational autoencoder (CVAE) can produce novel genetic circuits that match complex dynamic functions such as signal adaptation. Using in silico RNA simulation, we construct a dataset of RNA sequences and convert them to circuits via RNA interaction predictors, allowing us to estimate functional features alongside evolutionary stability and interpret model-learned features. Our model generates diverse distributions of circuits that match their target adaptation specification well, even when limited to small training data sets. Structures in the embedding space correspond to motifs previously identified as crucial for adaptation and reflect the design rules for adaptable circuits. Framing adaptation as a single design objective outperforms other input representations, reflecting the importance of choosing the correct data encoding for generating genetic circuits. Finally, we show that functional and evolutionary properties can be prompted simultaneously, providing a proof-of-concept for the combined design of phenotype and evotype.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.26.678595v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">IntegrateALL: an end-to-end RNA-seq analysis pipeline for multilevel data extraction and interpretable subtype classification in B-precursor ALL</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wolgast, N., Beder, T., Mondal, M., Walter, W., Hutter, S., Bendig, S., Kaessens, J. C., Hansen, B.-T., Iben, K., Wolf, S., Cremer, A., Barz, M., Neumann, M., Goekbuget, N., Haferlach, C., Brueggemann, M., Baldus, C. D., Hartmann, A. M., Bastian, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce KaryALL, a machine-learning classifier that uses normalized expression and minor-allele-frequency features (RNASeqCNV) to distinguish near haploid, hypodiploid and high hyperdiploid B-ALL and chromosome-21 gains/iAMP21</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Transcriptome sequencing (RNA-seq) is emerging as a diagnostic standard for B-cell precursor acute lymphoblastic leukemia (B-ALL). Expression-based classifiers reach ~95% accuracy, but reproducible end-to-end solutions that also integrate transcript-derived genomic drivers and quantitative virtual karyotyping are lacking. We developed IntegrateALL, a Snakemake pipeline that standardizes RNA-seq analysis from FASTQ to rule-based subtype assignment across 26 WHO-HAEM5/ICC entities by integrating expression-based subtype prediction, gene fusion- / hotspot SNV calling and virtual karyotyping. We introduce KaryALL, a machine-learning classifier that uses normalized expression and minor-allele-frequency features (RNASeqCNV) to distinguish near haploid, hypodiploid and high hyperdiploid B-ALL and chromosome-21 gains/iAMP21 (accuracy: 0.98 / F1-score: 0.96 on 615 independent test samples). SNP-array concordance supported RNA-based karyotyping. Applied to 774 unselected B-ALL cases, IntegrateALL yielded unambiguous subtype assignments in 81.5%, based on concordance of gene expression class with a defining driver (75.3% of all cases) or, in selected cases, high-confidence expression-based classification alone (6.2%); the remainder (18.5%) were flagged for manual curation. Independent validation (3 cohorts; n=436, including pediatric cases) reproduced these distributions. Across all patients (n=1,210), 2.6% harbored two subtype defining drivers, including hyperdiploidy in fusion-driven subtypes where it was not expected or subtype-defining SNVs (e.g., PAX5 P80R / IKZF1 N159Y) co-occurring with BCR::ABL1-positive/-like, KMT2A- or DUX4-fusions. In most dual-driver cases, one subtype gene expression signature predominated, indicating a hierarchy of oncogenic control and the value of systematic driver screening alongside expression-based calls. IntegrateALL provides an adaptable fully reproducible workflow for molecular B-ALL characterization by systematically integrating genomic drivers and downstream gene regulation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.25.673987v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Analysis and characterization of extrachromosomal circular DNA in prostate cancer: Potential biomarker discovery from urine, plasma, and tumor samples.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mingxin Zhang, Zhe Xu, Yuxuan Cao, Chengwen Gao, Guangdi Chu, Guofeng Xia, Yuqing Tian, Nian Liu, Anqi Wang, Weimin Ma, Pengcheng Yang, Mingxuan Wu, Yihong Lian, Xiangzhong Zhao, Qian Zhang, Peng Han, Yonghua Wang, Zhiqiang Li, Haitao Niu</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> EccDNA profiles across urine, plasma, and tumor tissues reflect disease states and progression, suggesting its utility in clinical applications.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Extrachromosomal circular DNA (eccDNA) may contribute to genomic rearrangements and tumor heterogeneity, playing a role in cancer development and progression. This study evaluates eccDNA as a biomarker for prostate cancer by characterizing its profiles in urine, plasma, and tumor tissues from patients at different disease stages. We studied 49 prostate cancer patients (23 early-stage; 26 late-stage, including 19 with metastasis), 23 patients with prostatitis, and 21 healthy individuals. EccDNA was extracted from plasma, urine, and tumor tissues using the Circle-Map workflow. We analyzed eccDNA abundance, genomic origin, GC content, length distribution, and repetitive sequence content. Differences among these groups were assessed with the Wilcoxon rank-sum test, and five machine learning models classified cancer vs. non-cancer based on eccDNA features. Significant variations in eccDNA levels were observed among sample types and disease states. Prostate cancer patients exhibited higher eccDNA abundance in tumor tissues compared to plasma and urine samples. Metastatic patients had significantly elevated plasma eccDNA levels compared to nonmetastatic patients and controls. Tumor-derived eccDNA showed higher GC content and distinct length distributions. Shared eccDNA molecules across tissue types suggest common origins and potential systemic roles in cancer progression. Classification models achieved strong performance, especially in plasma, where a Neural Network model reached an AUC of 0.91, and in urine, where a Random Forest model reached 0.77. Limitations include the relatively small cohort size and the need for functional studies to clarify eccDNA's role in cancer biology. This study highlights eccDNA's potential as a noninvasive biomarker for prostate cancer diagnosis and monitoring. The distinct eccDNA profiles across urine, plasma, and tumor tissues reflect disease states and progression, suggesting its utility in clinical applications.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40516903/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

