<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Sunday, 20 July 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Plasma proteomics for biomarker discovery in childhood tuberculosis.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Andrea Fossati, Peter Wambi, Devan Jaganath, Roger Calderon, Robert Castro, Alexander Mohapatra, Justin McKetney, Juaneta Luiz, Rutuja Nerurkar, Esin Nkereuwem, Molly F Franke, Zaynab Mousavian, Jeffrey M Collins, George B Sigal, Mark R Segal, Beate Kampman, Eric Wobudeya, Adithya Cattamanchi, Joel D Ernst, Heather J Zar, Danielle L Swaney</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A non-sputum-based TB biomarker that could reduce delays in TB diagnosis and improve the detection and management of TB in children.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Failure to rapidly diagnose tuberculosis disease (TB) and initiate treatment is a driving factor of TB as a leading cause of death in children. Current TB diagnostic assays have poor performance in children, thus a global priority is the identification of novel non-sputum-based TB biomarkers. Here we use high-throughput proteomics to measure the plasma proteome for 511 children, with and without HIV, and across 4 countries, to distinguish TB status using standardized definitions. By employing a machine learning approach, we derive four parsimonious biosignatures encompassing 3 to 6 proteins that achieve AUCs of 0.87-0.88 and which all reach the minimum WHO target product profile accuracy thresholds for a TB screening test. This work provides insights into the unique host response in pediatric TB disease, as well as a non-sputum biosignature that could reduce delays in TB diagnosis and improve the detection and management of TB in children worldwide.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40683862/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Modelling the impacts of imports of non-native honeybees into the native Apis mellifera mellifera populationin Ireland</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> de Carlos, I., Strachan, L., McCormack, G. P., Gorjanc, G., Obsteter, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> The human-mediated import of non-native queens is threatening the unique adaptations of native honey bees in Ireland through hybridisation and introgression, and the best way forward is to invest in holistic selective breeding of the native honey bee population in its native environment.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The human-mediated import of non-native queens is threatening the unique adaptations of native honey bees in Ireland through hybridisation and introgression. To study the impacts of this import, we built a SIMplyBee stochastic model of hybridisation between a native Apis mellifera mellifera population in Ireland and a non-native (C lineage) honey bee population. The model included colony honey yield and fitness as complex polygenic traits. We modelled the effects of varying the percentage of imports, the genetic correlations between fitness in native and non-native environment, and the spatial distribution of introgression over 20 years. We measured the levels and rate of introgression and the changes in genetic means for honey yield and fitness. The results revealed that increased imports expectedly increase the rate of introgression. This introgression induced a trade-off between increased honey yield and decreased fitness. Changing the genetic correlation between fitness in native and non-native environment pointed to a larger fitness decrease with negative correlations. The study showed the spread of introgression across the whole simulated area, indicating that even colonies far from the import zones could be affected over time. Furthermore, halting imports generated a desirable direction of change, however the purging of introgressed material was very slow and varied substantially across replicates. These results underscore the risks and long-term impacts of import on complex polygenic traits. We argue that the best way forward to manage the problem of import and associated introgression is to invest in holistic selective breeding of the native honey bee population in its native environment rather than import invasive subspecies that are bred to be successful in a different climate. The study also highlights the lack of fundamental quantitative genetic parameters for complex polygenic traits of different honey bee populations in different environments.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.16.664955v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Improving archaeological metadata reporting in human paleogenomicstudies</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Staniuk, R., Lee, V. Y. K., Timpson, A., Kristiansen, K., Shennan, S., Racimo, F., Thomas, M. G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We examine the extent of archaeological metadata reporting in 30 ancient DNA studies, based on genetic sequences from 3911 ancient humans, published between 2013 and 2024. We show that the reporting of archaeological data is inconsistent across studies, and seek to identify the driving factors.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> While dramatically increasing our understanding of past demographic and adaptive processes, paleogeneomic research has also been criticized for a perceived disconnect between geneticists and other parties involved in the study of the past. For interdisciplinary research to be productive, contextual metadata associated with paleogenomic samples should be accessible in the same publication. Here, we examined the extent of archaeological metadata reporting in 30 peer-reviewed ancient DNA studies, based on genetic sequences from 3911 ancient humans, published between 2013 and 2024. Our findings show that the reporting of archaeological data is inconsistent across studies, and we have sought to identify the driving factors. Overall, we found no strong explanatory variables, though some metadata fields - like Geolocation - have improved in completeness over time. These inconsistencies mean that metadata reporting is often insufficient to directly investigate the relationship between patterns of cultural and genetic change on the basis of published data alone. We propose a minimum information checklist as a best-practice guideline for collecting and reporting archaeological data in the future.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.16.665041v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Long-read sequencing of trios reveals increased germline and postzygotic mutation rates in repetitive DNA</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Noyes, M. D., Sui, Y., Kwon, Y., Koundinya, N., Wong, I., Munson, K. M., Hoekzema, K., Kordosky, J., Garcia, G. H., Knuth, J., Lewis, A. P., Eichler, E. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Long-read sequencing increases the number of de novo mutations discovered in autism by 20-40% over previous studies, and more than doubles the discoverable number of PZMs that emerged early in embryonic development.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Long-read sequencing (LRS) has improved sensitivity to discover variation in complex repetitive regions, assign parent-of-origin, and distinguish de novo germline from postzygotic mutations (PZMs). Most studies have been limited to population genetic surveys or a few families. We applied three orthogonal sequencing technologies--Illumina, Oxford Nanopore Technologies, and Pacific Biosciences--to discover and validate de novo mutations (DNMs) in 73 children from 42 autism families (157 individuals). Assaying 2.77 Gbp of the human genome using read-based approaches, we discover on average 95 DNMs per transmission (87.5 de novo single-nucleotide variants and 7.8 indels), including sex chromosomes. We estimate that LRS increases DNM discovery by 20-40% over previous Illumina-based studies of the same families, and more than doubles the discoverable number of PZMs that emerged early in embryonic development. The strict germline mutation rate is 1.30x10-8 substitutions per base pair per generation, strongly driven by the father's germline (3.95:1), while PZMs increase the rate by 0.23x10-8 with a modest but significant bias toward paternal haplotypes (1.15:1). We show that the mutation rate is significantly increased for classes of repetitive DNA, where segmental duplication (SD) mutation shows a dependence on the length and percent identity of the SD. We find that the mutation rate enrichment in repeats occurs predominantly postzygotically as opposed to in the germline, a likely result of faulty DNA repair and interlocus gene conversion.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.18.665621v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Protein structure alignment significance is often exaggerated</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Edgar, R. C., Sahakyan, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that unrelated proteins have a universal tendency towards convergent evolution of secondary and tertiary motifs, causing an excess of high-scoring false positive alignments, and propose a novel method for estimating statistical significance.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Machine learning has generated millions of high-quality predicted protein structures, creating a need for computationally efficient structure search algorithms and robust estimates of statistical significance at this scale. We show that unrelated proteins have a universal tendency towards convergent evolution of secondary and tertiary motifs, causing an excess of high-scoring false positive alignments. To address this excess, and to accommodate recent innovations in search algorithm design, we describe a novel method for estimating statistical significance. We implement our approach in Reseek, showing that its E-values are accurate, scale successfully with database size, and are robust against the (generally unknown) diversity of folds in the database. We investigate popular structure search and alignment algorithms, finding that previous methods routinely overestimate significance by up to six orders of magnitude.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.17.665375v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Evaluating a SNP calling pipeline for Mycobacterium leprae</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cox, K., Whiley, D., Chilengue, A., Domingo Sananes, M. R., Meehan, C. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that SNP calling from short reads is robust and highly accurate from M. leprae, showing promise as a tool for molecular epidemiology studies to increase case detection and inform leprosy control strategies.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mycobacterium leprae is the causative agent of leprosy, a persisting disease characterised by skin lesions and peripheral nerve numbness. Attempts to sequence the genome have been challenged by its inability to be cultured on artificial media. Advances in DNA extraction and sequencing technologies have enabled molecular epidemiology to be introduced for M. leprae. However, a validated SNP calling pipeline does not currently exist. A ground truth of pairwise SNP differences was first created using Minimap2 and three complete genomes of M . leprae. Using simulated reads from each of these genomes and the others as a reference, we evaluated the precision and recall of short-read SNP calling using Snippy. Approximately 80% of SNPs were called with false positives only due to ambiguous bases in certain genomes. Repeat region masking was found to be unnecessary for M. leprae SNP calling, unlike for M. tuberculosis. We find that SNP calling from short reads is robust and highly accurate from M. leprae, showing promise as a tool for molecular epidemiology studies to increase case detection and inform leprosy control strategies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.18.665481v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Uniquely high spontaneous mutational load in blood cells of XP-C patients</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fan-Huang, G., Thompson, E. L., Lee, M., Sun, S., Boull, C., Maguiness, S. M., Vijg, J., Niedernhofer, L. J., Maslov, A. Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We discovered a uniquely high spontaneous somatic mutational load in peripheral blood mononuclear cells of Xeroderma Pigmentosum group C (XP-C) patients and demonstrate that the exceptionally high mutation burden in XP-C leukemia originates from mutations already present prior to malignant transformation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We discovered a uniquely high spontaneous somatic mutational load in peripheral blood mononuclear cells (PBMCs) of Xeroderma Pigmentosum group C (XP-C) patients, characterized by elevated single nucleotide variants associated with mutation signatures SBS8 and SBS32, as well as an enrichment of single-nucleotide cytosine deletions. This hypermutability was markedly lower in fibroblasts, suggesting a replication-dependent mechanism of mutagenesis due to deficient global genome nucleotide excision repair (GG-NER). Our findings reveal distinct molecular subtypes within XP defined by spontaneous mutational load in normal blood cells and demonstrate that the exceptionally high mutation burden in XP-C leukemia originates from mutations already present prior to malignant transformation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.16.665113v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Learning the native-like codons with a 5'UTR and secondary RNA structure aided species-informed transformer model</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hu, Q., Tian, X., Li, Y., Meng, J., Wang, S., Guo, J., Zheng, L., Wei, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present TransCodon, a Transformer-based deep learning model that leverages both 5' untranslated regions (5'UTRs) and coding sequences (CDS), along with explicit species identifiers and RNA secondary structure information, to learn nuanced codon usage patterns across diverse organisms.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Efficient protein expression across heterologous hosts remains a major challenge in synthetic biology, largely due to species-specific differences in codon usage and regulatory sequence context. A key difficulty lies in reconstructing the codon landscape of the target expression system within a foreign host with a native-like codon preference. To address this, we present TransCodon, a Transformer-based deep learning model that leverages both 5' untranslated regions (5'UTRs) and coding sequences (CDS), along with explicit species identifiers and RNA secondary structure information, to learn nuanced codon usage patterns across diverse organisms. By incorporating multisource genomic data and modeling sequence dependencies in a masked language modeling paradigm, TransCodon effectively captures both local and global determinants of codon preference. Our experiments demonstrate that integrating species-level information during training significantly improves the model's ability to predict optimal synonymous codons for enhanced cross-species protein expression. TransCodon consistently outperforms existing codon optimization tools in multiple evaluation metrics. More importantly it identifies native-like codons with less divergence from natural sequences compared to other methods. For a high abundant protein, TransCodon could capture more low-frequency codons which are often omitted by other deep learning-based method. The results thus indicate that TransCodon as a robust codon language model has the potential for designing genes with high translational efficiency in target hosts.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.19.665668v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">De novo design of functionally diverse and druggable antimicrobial peptides by diffusion and multimodal deep learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> ouyang, a., Ge, F., Song, J., Yu, D.-J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce AMP-D3, a comprehensive framework that integrates AMP generation, identification, and screening into a cohesive workflow.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The escalating crisis of antibiotic resistance underscores the urgent need for innovative anti-infective agents, such as antimicrobial peptides (AMPs), though their discovery and optimization remain challenging. To address this, we introduce AMP-D3, a comprehensive framework that integrates AMP generation, identification, and screening into a cohesive workflow. Central to AMP-D3 is PACD, an advanced generator that utilizes contrastive diffusion to produce diverse, functionally tailored AMPs. For precise recognition, the CAST model combines ESM-2 embeddings with localized sequence features through cross-modal attention, yielding an accuracy of 0.891 - a 2.2% enhancement over current state-of-the-art approaches. Additionally, a robust multi-attribute prediction module evaluates key properties, including antifungal and anticancer potential, across twenty-two distinct characteristics. Furthermore, a meticulously designed screening pipeline identifies high-efficacy AMP candidates against critical pathogens, such as C. albicans, E. coli, P. aeruginosa, and S. aureus. This process assessed 10,000 peptides generated by PACD alongside 43,000 from prior studies, selecting the top 100 candidates per pathogen for further analysis. Experimental validation confirms that these peptides exhibit structural diversity and binding affinities comparable to naturally occurring AMPs, as evidenced by AlphaFold2 structural predictions and molecular docking studies. Thus, AMP-D3 represents a transformative approach to the design and expedited discovery of clinically viable AMPs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.16.665042v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Three-State Gene Expression Model Parameterized for Single-Cell Multi-Omics Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Peyric, T., Lepoutre, T., Crombach, A., Guyet, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a novel three-state gene expression model designed to elucidate the underlying mechanisms of mRNA transcription and its regulation. Our model incorporates gene regulatory processes by explicitly including a transcription factor-bound state, thereby capturing the dynamic interplay between transcription activation and chromatin dynamics.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We present a novel three-state gene expression model designed to elucidate the underlying mechanisms of mRNA transcription and its regulation. Our model incorporates gene regulatory processes by explicitly including a transcription factor-bound state, thereby capturing the dynamic interplay between transcription activation and chromatin dynamics. We fit the model to paired single-cell ATAC-seq and single-cell RNA-seq data, as these data give us simultaneous information on a gene's transcriptional state and its accompanying chromatin state. Working at the pseudo-bulk level, we extract biologically meaningful high-level descriptors from homogeneous cell (sub)populations, such as the mean and variance of gene expression as well as the fraction of accessible chromatin. Crucial to the computational feasibility of our approach, these descriptors can be analytically related to our model parameters. Despite the increased complexity needed to capture regulatory processes in our model, it remains sufficiently parsimonious to infer parameters reliably from experimental data. Each parameter has a clear biological interpretation, reflecting properties such as burst frequency, chromatin opening and closing dynamics, and basal or regulated expression. Fitting the model to a large collection of genes allows us to analyze the parameters and distinguish so-called gene expression strategies. The model parameters reveal a small number of distinct expression strategies among gene clusters, providing data-driven novel insight into context-dependent regulation of gene expression.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.16.665109v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Floral temperatures of the southern-most flowering Amorphophallus titanum and seedling growth following self-pollination</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lord, J. M., Lord, K. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Amorphophallus titanum (Becc.) Becc. titanum can self-pollinate with pollen stored at -80{ring}C for at least several years.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Amorphophallus titanum (Becc.) Becc. (Araceae) is native to the rainforests of Sumatra, and famously produces a single, massive, thermogenic and highly odoriferous inflorescence. Despite having both male and female flowers in close proximity, A. titanum does not readily self-pollinate as it is strictly dichogamous, with female flowers receptive on the first night of flowering and pollen released on the second night. We report on external and internal temperatures reached by the most southerly plant in cultivation which flowered in February 2018 in Dunedin, New Zealand. We also report on successful self-pollination of the same plant in January 2021 with pollen collected in 2018 that had been stored at -80{ring}C. External spathe and internal spadix temperatures exceeded ambient temperatures on both flowering nights in 2018. Self-pollination in 2021 resulted in copious fruits that matured in 8 months and showed 75% germination success. More than a fifth of all seedings to germinate were achlorophyllous and died within four months, which is consistent with inbreeding depression. Seedling height, leaf production, length of active growth period and corm size varied markedly among the surviving chlorophyllous seedlings, with some producing two leaves in their first year. While we would not recommend the use of self-pollination for propagation of this species due to the likely presence of inbreeding depression, our study does prove that pollen can retain its viability at -80{ring}C for at least several years which is an important step towards a pedigree-informed breeding programme among plants in cultivation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.15.665007v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Contrasting needle physiological strategies to soil nutrient scarcity in radiata pine plantations</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ruiz de Larrinaga, L., San Miguel-Oti, F., Monasterio, A., Artetxe, U., Munne-Bosch, S., Mesa, T., Moreaux, C., Ortega-Barrueta, U., Sertutxa, U., Pena, L., Ametzaga, I., Yuste, J. C., Esteban, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We investigated the influence of soil nutrient scarcity on tree physiological responses during drought, particularly in Pinus radiata D. Don plantations, where it may impair needle function and overall tree health.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Intensive European forestry practices contribute to soil degradation and nutrient depletion, compromising tree health and ecosystem stability. However, the influence of soil nutrient scarcity on tree physiological responses during drought remains unclear, particularly in Pinus radiata D. Don plantations, where it may impair needle function and overall tree health. We investigated how soil nutrient availability influences leaf-level physiological strategies during drought by comparing same aged needles from two P. radiata stands with contrasting management stages and ages: (i) trees (20 years), likely to be clear cut at 30 or 35 years (herein managed), and (ii) trees (<45 years) not clear-cut at the typical rotation age (herein abandoned). During the severe summer drought of 2022, both stands exhibited downregulation of the photosynthetic apparatus, indicating impaired photosynthetic performance under drought. However, their leaf physiological responses to nutrient scarcity diverged. Managed trees exhibited dependence on soil nutrients, with reduced photosynthetic performance under nutrient-poor conditions. In contrast, abandoned trees showed relative independence from soil nutrients, maintaining photosynthetic function even under nutrient-poor conditions. This highlights a trade-off: younger, managed trees may enhance photosynthesis under optimal nutrient conditions but are more susceptible to nutrient imbalances and environmental stress. In turn, older, abandoned trees appeared to buffer the effects of drought and nutrient scarcity through age-related physiological traits. Our findings underscore the physiological value of mature stands. Furthermore, higher soil organic carbon and vegetation diversity in abandoned stands suggest that management cessation may enhance long-term ecosystem resilience. These results emphasise the importance of integrating soil-leaf interactions into sustainable forest management.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.15.664982v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

