<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Thursday, 24 July 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Continental-scale genomic surveillance of Plasmodium falciparum malaria with rapid nanopore sequencing</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mwenda, M., Mosler, K., Bohmeier, B., Chomba, M., van Loon, W., Mambwe, B., Gaye, A., Olorunfemi, A., Suliman, S., Yanogo, N. J., Sow, D., Ngom, B., Zoure, O. A. Z., Tibiri, Y. N. G., Ojeniyi, F., Zongo, A., Anoh, E. A., Achi, V., Chiyesu, C., Otieno, S., Ogola, B., Niangaly, M., Carrasquilla, M., Goboto, D. K., Feldt, T., Tufa, T. B., Oliveira, R., Schallenberg, E., Sogoba, Y., Ntalla, C., Ouedraogo, O., Otieno, K., Opaleye, O., Olowe, A., Gibbons, M., Drakeley, C., Schubert, G., Mockenhaupt, F. P., Portugal, S., Deme, A. B., Soulama, I., Ndiaye, D., Ojurongbe, O., Kariuki, S., Shi, Y. P., Sc</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed an approach to enable a decentralized scale-up of malaria genomic surveillance and established it in six African countries in one year, locally sequencing 1,065 samples.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In sub-Saharan Africa, continental-scale genomic surveillance of Plasmodium falciparum malaria is needed to track the spread of antimalarial drug resistance and diagnostic test evasion, as well as to monitor parasite evolutionary responses to vaccine rollout. Yet implementation of malaria genomic surveillance at a continental-scale is hindered by resource constraints, the vastness of the continent, and the lack of sequencing protocols suitable for most local laboratories. To address this, we developed an approach to enable a decentralized scale-up of P. falciparum genomic surveillance and established it in six African countries in one year, locally sequencing 1,065 samples. The approach includes a rapid ({approx}5 hours) and cost-efficient (< $25 USD/sample) nanopore sequencing protocol that provides surveillance of a panel of drug resistance-associated genes, hrp2/3 gene deletions, the vaccine target csp, and the highly diverse gene ama1. We coupled this to a bioinformatics dashboard that runs offline on a laptop and displays mapping and variant calling results in real-time. We demonstrate robust sequencing coverage across parasitemia levels and laboratories, accurate detection of hrp2/3 deletions in field samples and accurate identification of antimalarial drug resistance markers. Our approach will accelerate genomic surveillance of P. falciparum malaria across sub-Saharan Africa at a time of urgent need.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.23.666274v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identification of RNA binding proteins that mediate a quality control mechanism of splicing</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Arafat, M., Gotea, V., Umar, M. I., Muharram, A., Kamhi, E., Agami, R., Sperling, J., Hafner, M., Elnitski, L., Sperling, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A genetic screen targeting RNA-binding proteins (RBPs) with an siRNA library and a luminescence reporter for latent splice site activation. Our findings reveal that nuclear RBPs play a key role in suppressing LSS activation and suggest that ALYREF has a novel role in maintaining spl</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurate splicing, which involves the controlled removal of non-coding sequences (introns) from precursor messenger RNAs (pre-mRNAs), is essential for producing correct mature mRNAs that encode functional proteins. Within pre-mRNAs, latent splice sites (LSSs) resemble proper splice sites but are usually not used because their activation can introduce in-frame STOP codons. The nuclear suppression of splicing (SOS) mechanism prevents the use of LSSs. Although the SOS mechanism is not fully understood, recent studies have identified initiator-tRNA and the NCL protein as key components. To discover additional regulators, we performed a genetic screen targeting RNA-binding proteins (RBPs) with an siRNA library and a luminescence reporter for latent splice site activation. This identified five RBPs, ALYREF (THOC4), PPIE, DDX41, DHX38, and HNRNPA2B1, whose knockdown significantly increased LSS usage in the reporter. RNA-Seq analysis after knocking down each of these RBPs confirmed these results, showing widespread LSS activation in hundreds of mRNAs. Among these, we focused on ALYREF, a conserved protein involved in mRNA export and splicing. Using fPAR-CLIP, we found that U5 snRNA is ALYREF's main binding partner. Overexpressing ALYREF deletion mutants activated latent splicing, and affinity purification confirmed its interaction with U5 snRNA. These mutants exhibited different binding properties, highlighting the importance of specific structural elements within ALYREF in SOS regulation. Our findings reveal that nuclear RBPs play a key role in suppressing LSS activation and suggest that ALYREF has a novel role in maintaining splicing accuracy within the spliceosome, advancing our understanding of the SOS mechanism.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.20.665773v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Topoisomerase II inhibitors CX-5461 and Doxorubicin differ in their cardiotoxicity profiles</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Paul, S., Gutierrez, J. A., Bogar, A. R., Ward, M. C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> CX-5461 (CX) is a chemotherapeutic drug currently under investigation for the treatment of late-stage cancers, but it is unclear whether CX will similarly cause cardiotoxicity.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> CX-5461 (CX) is a chemotherapeutic drug currently under investigation for the treatment of late-stage cancers. While CX was first described as an RNA polymerase I inhibitor, it has recently been shown to primarily inhibit the beta isoform of topoisomerase II. This isoform is also inhibited by anthracycline drugs including Doxorubicin (DOX) and mediates the toxic effects of these drugs on the heart. It is unclear whether CX will similarly cause cardiotoxicity. We therefore designed a study to test the effects of CX compared to DOX on iPSC-derived cardiomyocytes from six individuals. While both CX and DOX induce cell death in cardiomyocytes, CX is 20-fold less cytotoxic than DOX. At sub-lethal doses, DOX induces DNA damage, while CX does not. Transcriptome profiling following treatment with two sub-micromolar concentrations of both drugs over time reveals that DOX induces thousands of gene expression changes compared to hundreds induced by CX. Comparison of gene expression trajectories across drugs reveals that genes that respond to CX also respond to DOX, while most DOX response genes are drug specific. Shared response genes correspond to pathways related to chromosome segregation and DNA replication. CX does not affect the expression of any of the genes in functionally-validated loci associated with DOX-induced cardiotoxicity. Our data demonstrate that CX treatment of cardiomyocytes induces gene expression changes that mirror a subset of those induced by DOX; however these changes do not coincide with the cardiotoxicity observed with DOX treatment.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.20.665626v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Inbreeding and resultant homozygosity across key inflammation and DNA repair genes linked to chlamydial infection in New South Wales koalas</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> McLennan, E. A., Vogelnest, L., Silver, L. W., Wright, B. R., Casteriano, A., Chen, Z., Mella, V. S. A., Higgins, D. P., Krockenberger, M. B., Belov, K., Jessop, T. S., Hogg, C. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Inbreeding and resultant homozygosity can reduce genetic diversity and increase disease susceptibility. We identify potential links between inbreeding, genome-wide variation and chlamydial infection.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Inbreeding and resultant homozygosity can reduce genetic diversity and increase disease susceptibility. Koalas (Phascolarctos cinereus) are one species suffering genomic diversity loss and inbreeding and concurrent significant disease pressure (particularly chlamydiosis). Using 259 whole genomes with a pathogen sampling regime we identify potential links between inbreeding, genome-wide variation and chlamydial infection. We found a general trend of reduced genomic diversity and increased inbreeding from north to south across six sites in New South Wales. A genome-wide association study of 153 individuals from sites with known Chlamydia pecorum presence were used to investigate the potential relationship between inbreeding and infection. Chlamydia positive individuals (average FH = 0.026) were significantly more inbred than Chlamydia negative individuals (average FH= -0.0051) (t = -2.31, df = 151, p-value = 0.022). We identified several genes involved in host-pathogen interactions and DNA mismatch repair within in runs of homozygosity that were unique to Chlamydia positive individuals. Interestingly, populations considered putatively Chlamydia-free had similar allele frequencies across candidate loci as Chlamydia positive individuals. Combined with gene flow analyses, this result suggests that isolation may have protected these populations more than harbouring alleles conferring infection resilience and supports the concept that disease should be carefully considered in any conservation measures that increase connectivity or translocations. Our genome-wide approach has identified several avenues for investigations into the pathogenesis of Chlamydia infection and chlamydiosis. We showcase the value of high-quality re-sequenced genomes for understanding the implications of inbreeding, genomic diversity loss, and infection susceptibility, all universal problems for threatened species.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.22.666211v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DNA Conformational Flexibility Descriptors Improve Transcription Factor Binding Prediction Across the Protein Families</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dey, U., Yella, V. R., Kumar, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We systematically integrate experimentally and computationally derived DNA flexibility descriptors into predictive models of TF-DNA binding specificity, and demonstrate that DNA flexibility-augmented models consistently outperform sequence based models, and DNA shape augmented models to an extent.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Precise binding of transcription factors (TFs) to specific DNA sequences is fundamental to gene regulation, yet the molecular principles underpinning TF-DNA specificity remain incompletely understood. While nucleotide sequence and DNA shape are known determinants of TF binding, the role of DNA flexibility encompassing axial, torsional, and stretching dynamics remains largely unexplored, particularly across diverse TF families. Here, we systematically integrate experimentally and computationally derived DNA flexibility descriptors into predictive models of TF-DNA binding specificity. Through extensive analyses of large-scale in vitro datasets from HT-SELEX, SELEX-Seq, protein binding microarrays encompassing mammalian and Drosophila TFs, we demonstrate that flexibility-augmented models consistently outperform sequence based models, and DNA shape augmented models to an extent. These improvements are robust across diverse experimental platforms and scale of the datasets, underscoring the importance of DNA conformational dynamics in indirect readout. Quantitative analyses of position-specific flexibility contributions reveal distinct "flexibility hotspots" within transcription factor binding sites and their flanking regions. This is exemplified by structural insights into the homeodomain TF MSX1, where localized DNA bendability directly correlates with enhanced binding a[ff]inity and precise recognition specificity. Finally, leveraging in vivo ChIP-Seq and DNase-Seq data from ENCODE, we further validate that DNA flexibility substantially enhances the identification of functional TF binding sites across various TF families and cellular contexts. Collectively, current findings substantiate DNA flexibility as a fundamental element of the cis-regulatory code and significantly advance predictive frameworks of gene regulatory networks.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.20.665724v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SimSpace: a comprehensive in-silico spatial omics data simulation framework</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhao, T., Zhang, K., Hollenberg, M., Zhou, W., Fenyo, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Simulation of spatial organization of biological tissues using a Markov Random Field model and a flexible simulation framework .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Tissue function is tightly linked to the spatial organization, and our knowledge of the relationship between tissue structure and function has increased due to the rapid technological development of spatial transcriptomics approaches. A bottle-neck is, however, the analysis of the complex and high-dimensional spatial transcriptomics data. Many computational tools have been proposed for spatial analysis but evaluating them remains challenging due to the scarcity of datasets with known ground truth. To address this, we developed SimSpace, a flexible simulation framework that can generate synthetic spatial cell maps with categorical cell type labels and biologically meaningful organization. Cell type spatial patterns are simulated using a Markov Random Field model, enabling the control of spatial autocorrelation and interaction between cell types. This approach captures a broad range of tissue architectures, from well-separated niches to spatially mixed environments. We evaluate the simulated data using spatial statistics metrics (eg. entropy, Moran's I) and benchmark the results against real spatial transcriptomics and proteomics datasets. Our findings show that the simulations closely mirror key spatial properties observed in biological tissues. Additionally, we demonstrate the utility of our framework as a testbed for benchmarking spatial gene detection and cell type deconvolution methods. By providing reproducible, ground-truth-controlled datasets, SimSpace supports the development, validation, and comparison of computational methods in spatial omics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.18.665587v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Prediction of bacterial protein-compound interactions with only positive samples</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kim, K.-H., Yaganapu, A., Kosaraju, S. C., Bhatt, A., Luo, Y., Parsa, S. P., Park, J., Lee, H., Lee, J. H., Oh, T.-J., Kang, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel Positive-Unlabeled (PU) learning framework, named BIN-PU, to address the lack of curated negative interaction samples for CPI prediction in bacteria.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Prediction of Compound-Protein Interactions (CPI) in bacteria is crucial to advance various pharmaceutical and chemical engineering fields, including biocatalysis, drug discovery, and industrial processing. However, current CPI models cannot be applied for bacterial CPI prediction due to the lack of curated negative interaction samples. This paper introduces a novel Positive-Unlabeled (PU) learning framework, named BIN-PU, to address this limitation. BIN-PU generates pseudo positive and negative labels from known positive interaction data, enabling effective training of deep learning models for CPI prediction. We also propose a weighted positive loss function that weights to truly positive samples. We have validated BIN-PU with multiple CPI backbone models, comparing the performance with the existing PU model using bacterial cytochrome P450 (CYP) data. Extensive experiments demonstrate the superiority of BIN-PU over the benchmark model in predicting CPIs with only truly positive samples. Furthermore, we have validated BIN-PU on additional bacterial proteins obtained from literature review, human CYP datasets, and uncurated data for its reproducibility. We have also validated the CPI prediction for the uncurated CYP data with biological and biophysical experiments. BIN-PU represents a significant advancement in CPI prediction for bacterial proteins, opening new possibilities for improving predictive models in related biological interaction tasks.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.18.665643v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">XL-Ranker: A Computational Workflow for Prioritizing Protein-Protein Interactions from Cross-Linking Mass Spectrometry Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chen, W., Elizarraras, J. M., Jiao, F., Huang, L., Zhang, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed XL-Ranker, a computational framework that combines a set cover graph algorithm and machine learning to systematically resolve peptide-mapping ambiguity and infer high-confidence PPIs from XL-MS data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein-protein interactions (PPIs) are central to virtually all biological processes, and their disruption can lead to a wide spectrum of human diseases. Cross-linking mass spectrometry (XL-MS) enables proteome-scale detection of interacting peptide pairs, from which PPIs can be inferred. However, interpretating XL-MS data to assign homologous cross-linked peptide pairs to specific protein interactions can be challenging. A major hurdle arises when a single peptide can map to multiple proteins, and when two such peptides are paired, the number of possible protein-protein interactions increases combinatorially. This mapping ambiguity can lead to inflated interaction networks that could compromise downstream analysis and biological interpretation. However, this problem has often been overlooked or addressed using heuristic approaches in previous XL-MS studies. To tackle this challenge, we developed XL-Ranker, a computational framework that combines a set cover graph algorithm and machine learning to systematically resolve peptide-mapping ambiguity and infer high-confidence PPIs from XL-MS data. Applied to an XL-MS dataset from HEK293 cells, XL-Ranker identified a high-confidence network with 880 PPIs involving 964 unique genes. Among all possible PPIs, the ones selected by XL-Ranker for inclusion in the final network had significantly higher interaction scores in the STRING database than the excluded ones. Network analysis further demonstrated that these interactions form biologically meaningful clusters, supporting the accuracy of our approach. In summary, XL-Ranker provides a practical solution to a key analytical challenge in XL-MS data interpretation, enhancing the reliability of PPI discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.18.665625v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SHARP: Generating Synthesizable Molecules via Fragment-based Hierarchical Action-space Reinforcement Learning for Pareto Optimization</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kim, J., Ryu, S., Park, H., Seok, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a molecular generator that addresses the optimization challenges of drug discovery using a fragment-based hierarchical action space and reinforcement learning.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Designing drug-like molecules that satisfy multiple objectives, such as high binding affinity, synthesizability, and drug-likeness, poses a complex global optimization problem over an astronomically large chemical space. Existing deep learning-based molecular generative models often treat this task as distribution modeling, relying on atom-level autoregressive actions with less consideration of explicit optimization feedback. Consequently, they frequently generate invalid structures, converge to local optima, or produce synthetically infeasible candidates. Here, we introduce SHARP (Synthesizable Hierarchical Action-space Reinforcement learning for Pareto optimization), a molecular generator that addresses these limitations via a fragment-based hierarchical action space and reinforcement learning. SHARP ensures synthetic accessibility by applying action masks guided by a pretrained Synthesizability Estimation Model (SEM). The reinforcement learning (RL) policy is trained using a composite reward function integrating docking scores, pharmacophore matching, and solvent accessibility to generate functionally relevant and experimentally tractable molecules. Furthermore, across four lead optimization tasks, fragment growing, linker design, scaffold hopping, and sidechain decoration on a diverse receptor set, SHARP consistently outperforms prior methods in producing molecules at high affinity and synthesizability. These results demonstrate that reinforcement learning with a chemically intuitive action space design can be an effective solution to the optimization challenges in AI-driven drug discovery, offering a robust framework for rational molecular design in structure-based applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.18.665529v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrated multi-omics analysis reveals divergent molecular responses in Palmer amaranth (Amaranthus palmeri) biotypes susceptible and resistant to glyphosate</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sandhu, P. K., Kumar, R., Nandula, V., Tharayil, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We captured transcriptomic, proteomic, and metabolomic perturbation of glyphosate-resistant and glyphosate-susceptible Palmer amaranth biotypes 24 hours after herbicide treatment, revealing complex stress responses hidden from single-omic analysis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Environmental stress triggers coordinated changes across genetic, transcriptomic, proteomic, and metabolomic levels in plants, yet the extent of synchronization across these omic layers remains underexplored. We captured transcriptomic, proteomic and metabolomic perturbation of glyphosate-resistant (GR) and glyphosate-susceptible (GS) Palmer amaranth (Amaranthus palmeri) biotypes 24 hours after herbicide treatment, quantifying 30,371 transcripts, 5,606 proteins, and 220 metabolites. Glyphosate perturbed threefold more transcripts and proteins in GS than in GR and caused the accumulation of shikimate intermediates in both biotypes. In GS, glyphosate severely disrupted primary metabolism, including photosynthesis and carbon fixation, leading to a collapse of energy production and impairment of phenylpropanoid and terpenoid biosynthesis, compromising defense and detoxification. In contrast, GR maintained cellular homeostasis, with minimal perturbation in carbon metabolism and upregulation of detoxifying pathways, indicating metabolic rerouting. Integrated multi-omics analysis captured stress responses hidden from single-omic analysis, including elevated glutathione metabolism, perturbation of the phenylpropanoid pathway and elevated raffinose family oligosaccharide metabolism in GR, and perturbation of taurine-hypotaurine metabolism in GS. Transcript and protein changes were broadly correlated, but GS exhibited signs of translational inhibition under glyphosate stress, indicating reduced protein synthesis. These findings reveal pervasive perturbation of glyphosate beyond the shikimate pathway within 24 hours after exposure, and underscore the importance of multi-omics integration to elucidate complex stress responses in plants.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.20.665775v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Study on physiological and ecological responses of alfalfa during the germination stage under drought, bicarbonate, and combined stress conditions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, Y., Hang, H., Wan, X., Guo, G., Li, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Alfalfa varieties under drought and salinity stress .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Alfalfa (Medicago sativa L.) is frequently constrained by drought and salinity during cultivation and production. This study aimed to investigate the differential response characteristics and underlying physiological and ecological mechanisms of alfalfa during the germination stage under drought stress, bicarbonate stress, and their combined effects. Twelve alfalfa varieties were subjected to germination tests using PEG-6000 to simulate drought stress (0~20%) and NaHCO to simulate bicarbonate stress (0~30 mM). Based on the semi-inhibitory concentrations and phenotypic indices observed under single-stress conditions, a combined stress treatment was designed. The subordinate function method was employed to comprehensively evaluate the variations in stress responses among the twelve alfalfa varieties during the germination stage. Additionally, this study explored the physiological and ecological response mechanisms of drought-tolerant, salt-tolerant, and drought-salt-sensitive varieties. Among the 12 alfalfa varieties, WL363HQ has the strongest drought and salt tolerance, while WL319HQ is the most sensitive to drought and salt stress. Based on this, the varieties can be classified into drought-tolerant, salt-tolerant, and sensitive categories. At low concentrations, the drought-salt combination shows antagonism (salt alleviates drought inhibition); at high concentrations, it turns into synergy, and the inhibitory effect is amplified.WL363HQ adapts to the combined stress by maintaining stable SOD activity and gradually accumulating soluble sugar; WL319HQ shows sensitivity due to fluctuating SOD, membrane lipid peroxidation, and uncontrolled soluble sugar.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.20.665730v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An NLP-inherited motif confers broad DNA-binding specificity to NIN in root nodule symbiosis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nosaki, S., Noda, M., Onoda, H., Ito, M., Suzaki, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify a previously undescribed motif in Lotus japonicus NIN, located downstream of the RWP-RK domain, which broadens the DNA-binding specificity of NIN by stabilizing the R WP-Rk dimer interface.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Nitrogen-fixing root nodule symbiosis (RNS) occurs in some eudicots, including legumes, and is regulated by the transcription factor NODULE INCEPTION (NIN), derived from the NIN-LIKE PROTEIN (NLP) family. However, how the NIN protein acquired RNS-specific functions remains unclear. We identify a previously undescribed motif in Lotus japonicus NIN, located downstream of the RWP-RK domain, which we term the FR. This motif broadens the DNA-binding specificity of NIN by stabilizing the RWP-RK dimer interface. nin mutants lacking the FR motif show defective nodulation and impaired nitrogen fixation. Arabidopsis NLP2 carries a NIN-type FR and shares key features with NIN. Furthermore, the NIN-type FR likely originated as early as gymnosperms, suggesting that the molecular feature of NIN for RNS regulation was inherited from ancestral NLPs before RNS emerged.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.20.665785v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Impact of elevated temperature on immunity-related hormone signaling in tomato plants</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liu, K., Shivnauth, V., Castroverde, C. D. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We studied tomato plants infected with the bacterial pathogen Pseudomonas syringae and found that tomato plants exhibited temperature-sensitive expression of marker genes associated with salicylic acid (SA), jasmonic acid (JA) and abscisic Acid (ABA) pathways,</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Molecular mechanisms governing the plant-pathogen-environment "disease triangle" are starting to emerge, although less so in agriculturally important species like tomato (Solanum lycopersicum). Here we analyzed defence hormone responses of tomato plants infected with the bacterial pathogen Pseudomonas syringae pv. tomato (Pst) DC3000 under two different temperatures. Our results showed that tomato plants exhibited temperature-sensitive expression of marker genes associated with salicylic acid (SA), jasmonic acid (JA) and abscisic acid (ABA) pathways, but not ethylene (ET). Our findings highlight the complexity of plant-microbe interactions and the importance of considering environmental conditions when studying plant defence responses.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.20.665745v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Non-classical Nuclear Signal PY-NLS Mediates Karyopherinβ2-Dependent Nuclear Import and Condensate Formation in Plants</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Saddhe, A. A., Pejchar, P., Potocky, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the first systematic characterization of non-classical PYNLS motifs in plants, demonstrating their functionality and divergence from known eukaryotic models.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Nuclear import is a fundamental cellular process regulated by nuclear localization signals (NLSs) and their cognate transport receptors. While classical NLSs have been extensively characterized in plants, the functionality of non-classical PYNLS (proline-tyrosine NLS) motifs - previously described only in animals and fungi - remains unexplored. Here, we present the first systematic characterization of PYNLS motifs in plants, demonstrating their functionality and divergence from known eukaryotic models. Using transient expression in tobacco and stable Arabidopsis lines, we show that PYNLS motifs from the jasmonate signaling protein JAZ1 and a previously uncharacterized PLA2-like protein mediate Karyopherin {beta}2 (Kap{beta}2)-dependent nuclear import. Through site-directed mutagenesis, structural modeling, and interaction assays, we define key sequence and structural determinants of functional PYNLS activity, including an extended linker between binding epitopes and a short -helix upstream of the conserved PY dipeptide. These features distinguish plant PYNLS motifs from their counterparts in animals and fungi. Leveraging these criteria, we identified 125 Arabidopsis proteins with predicted functional PYNLSs, encompassing transcription factors, RNA-binding proteins, and components involved in calcium signaling. Notably, Kap{beta}2 interaction also modulated condensate formation by cytoplasmic JAZ1, indicating a dual role for Kap{beta}2 in nuclear import and phase separation. Our findings reveal a plant-specific PYNLS architecture and highlight Kap{beta}2 as a multifunctional regulator of nuclear organization and biomolecular condensates, bridging a key gap in our understanding of plant nucleocytoplasmic transport.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.20.665762v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

