<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Monday, 29 September 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A rational approach for multicopy delta integration in Saccharomyces cerevisiae targeting a conserved Ty1 LTR sequence</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Seguchi, G., Vargas, B. d. O., Pereira, G. A. G., de Mello, F. d. S. B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel consensus {delta} sequence for efficient multicopy gene integration in S. cerevisiae, and show that it can be used to target retrotransposon-associated DNA elements in a single step.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genomic integration is a foundation in yeast-based cell factories, ensuring phenotypic stability and adequate gene dosage to sustain desired metabolic fluxes. Retrotransposon-associated sequences are attractive genomic integration targets for metabolic engineering, as they are present in multiple copies throughout the genome and their disruption should not jeopardize the cell. In S. cerevisiae, multiple strategies have been proposed to target Ty elements, but little attention has been given to the homology sequence itself. Through analysis of all long terminal repeats in S. cerevisiae, we propose a novel consensus {delta} sequence for efficient multicopy gene integration. Reducing homology length to favor a more conserved sequence greatly increases the number of potential genomic integration sites -- reaching 85 targets in this work. The designed integration sequence was validated by introducing a non-native xylose fermentation pathway into this yeast, achieving a single-step simultaneous integration of 15 xylose isomerase copies, enabling efficient ethanol production in the engineered strain.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.26.678811v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Programming Microbe-Computer Interaction for Customized Life-Nonlife and Cross-Life Communication</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Feng, Z., Zhang, C., Huang, X., Zhang, Y., Liu, H., Cui, M., Zhang, H., Song, Z., Li, M., Li, H., Ma, B., Lu, S., Li, M., Bai, G., Sun, D., Zhang, X., Zou, Z., Zhou, Y., Ye, B., Gao, M., Yin, H., Tao, M., Liu, P., Liu, D., Luo, H., Wang, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> This extension of human-computer interaction (HCI) paradigm to the non-intelligent microbe- computer interaction (MCI) is attractive for building generalized life-nonlife communication.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> This extension of human-computer interaction (HCI) paradigm to the non-intelligent microbe-computer interaction (MCI) is attractive for building generalized life-nonlife communication. Main challenges lie in how to construct biohybrid interface within individual MCI unit and thereafter program cross-life communication based on these MCI units. Here, we propose the design and construction of opto-interfaced MCI unit composed of engineered microbes and computer terminals, for enabling quantifiable transduction between microbial signals and digital signals. These units were able to serve as biohybrid sensors, actuators and transceivers, supporting biomonitoring, bio-actuation and microbe-centric bidirectional operations. Based on these functional units, we established programmable cross-life communication architectures capable of unicast, multicast, and logic-gate-driven communication with temporal dynamic control. The architectures were validated in biomedical scenarios including tumor hypoxia biomarking, epidermal heavy metal detoxification, and colitis intervention, demonstrating robust performance in closed-loop diagnostic-therapeutic applications. Additionally, we integrated MCI with an unmanned surface vessel for environmental monitoring, highlighting its cross-domain potential. Collectively, we provide a scalable strategy for building customized life-nonlife and cross-life communication, which is expected as an important complement to internet of things (IoT).</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.27.679003v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Directed evolution of a compact TranC11a system for efficient genome editing</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhu, Z., Gao, Q., Gao, Q., Jia, H., Wang, Z., He, M., Li, L., Zhang, L., Li, S., Jin, S., Gao, C., Zhao, K. T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We evolved TranC11a to improve genome editing efficiency in human and plant cells and breeding traits in maize.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The recently discovered TranC systems represent programmable RNA-guided DNA endonucleases of transposon origin with compact protein sizes ideal for therapeutic delivery. However, their editing efficiency in human and plant cells is limited. Here, we evolved TranC11a and engineered its sgRNA to enhance overall editing efficiencies. TranC11a systems exhibit up to 9.2-fold higher editing activity than its parent and achieves efficiency comparable to SpCas9 across multiple human genome endogenous sites, significantly outperforming compact editors NovaIscB and enTnpB1c. TranC11a enables efficient editing of disease-relevant genes in human cells and breeding traits in maize. With its high editing activity and compact size (574 aa), TranC11a demonstrates strong potential for future in vivo genome editing and crop engineering.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.28.678669v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A synthetic bacterium that degrades and assimilates poly(ethylene terephthalate)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Freund, D., Cherukuri, K. P., Mireles, R., Kippen, J., Shossel, M., Noda-Garcia, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We demonstrate that PET fermentation is feasible, opening the door to the production of valuable chemicals from waste.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Polyethylene terephthalate (PET) is the fourth most commonly used plastic worldwide. Like all plastics, post-consumer PET is poorly managed and accumulates in the environment, posing significant ecological threats. After 70 years of accumulation, microorganisms capable of degrading and assimilating PET have been isolated, demonstrating that PET can be broken down and converted into valuable cellular biomass or metabolic products. These natural isolates, however, are poorly characterized and challenging to genetically manipulate, which limits their further optimization and applicability. Here, we engineer a well-established synthetic biology chassis for the biodegradation and assimilation of PET. We modified the bacterium Pseudomonas putida KT2440 to heterologously express an active PET-hydrolytic enzyme extracellularly and to metabolize PET biodegradation products. The resulting strain, named PETBuster, was capable of growing on PET as the sole carbon source on solid and liquid media. We achieved 91% PET degradation after 21 days of culture, with a doubling time of 3.6 days, under mesophilic conditions. In this way, we demonstrate that PET fermentation is feasible, opening the door to the production of valuable chemicals from waste.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.28.673679v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">mBER: Controllable de novo antibody design with million-scale experimental screening</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Swanson, E., Nichols, M., Ravichandran, S., Ogden, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present mBER, an open-source protein binder design system capable of designing antibody-format binders with state-of-the-art experimental success rates.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent machine learning approaches have achieved high success rates in designing protein binders that demonstrate in vitro binding to their targets. While open models for unconstrained "minibinder" design have shown great experimental promise, methods for designing binders in specific formats, such as antibodies, have lagged behind in experimental success rates. In this work, we present mBER, an open-source protein binder design system capable of designing antibody-format binders with state-of-the-art experimental success rates. mBER builds on the ColabDesign framework, achieving successful antibody design primarily through the inclusion of informative sequence and structure conditioning information. Using mBER, we designed two libraries comprising over 1 million VHH binders against 436 diverse targets. We experimentally screened the two libraries against 145 of these targets, resulting in a dataset of over 100 million binding interactions. We achieved specific and significant design success against 45% of targets. In a filtered set of designs, we detect binding rates to specific epitopes as high as 38%. Through mBER, we demonstrate that format-specific binder design is possible with no additional training of underlying folding and language models. This work represents the largest reported de novo protein design and validation campaign, and one of the first open-source methods to demonstrate double-digit percentage experimental success rates for antibody binder design.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.26.678877v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The two sides of resistance: aggressiveness and mitotic instability as the Achilles heel of Osimertinib-resistant NSCLC</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Armanetti, P., Cabrera San Millan, E., Mercatelli, R., Cavallini, C., Chiodi, A., Mosca, E., Consiglio, A., Losito, N., Menichetti, L., Bramanti, E., Ali, A., Levantini, E., Maroni, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the Polycomb protein BMI1 is a central mediator of Osimertinib resistance and provide a mechanistic and therapeutic rationale for targeting BMI1 and mitotic weaknesses in refractory EGFR-mutant NSCLC.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Non-small cell lung cancer (NSCLC) represents majority of lung cancer cases and remains a leading cause of cancer mortality worldwide. Tumors carrying activating mutations in the epidermal growth factor receptor (EGFR) are highly sensitive to EGFR tyrosine kinase inhibitors (TKIs), with third-generation inhibitors such as Osimertinib now established as standard of care. However, acquired resistance to Osimertinib inevitably develops, involving both genetic and non-genetic mechanisms, the latter playing a major role in sustaining cellular plasticity and promoting tumor aggressiveness. Among regulators of adaptive programs, the Polycomb protein BMI1 has emerged as a key factor driving stemness, epithelial-to-mesenchymal transition (EMT), and therapy resistance in multiple cancers, yet its role in Osimertinib resistance remains poorly defined. Here, we show that Osimertinib-resistant H1975 cells, which display greater aggressiveness than their parental counterparts, are enriched in BMI1 target genes and mitotic cell-cycle pathways, establishing a dependency on microtubule dynamics and mitotic control. Functionally, BMI1 drives migration, invasiveness, and tumor progression in resistant cells. This mitotic dependency creates a therapeutic vulnerability that can be exploited with Unesbulin (PTC596), a BMI1 inhibitor that destabilizes microtubules and induces mitotic catastrophe, thereby effectively suppressing tumor growth in vitro and in vivo. Our findings establish BMI1 as a central mediator of Osimertinib resistance and provide a mechanistic and therapeutic rationale for targeting BMI1 and mitotic weaknesses in refractory EGFR-mutant NSCLC.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.25.678204v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Single-Cell Comparison of Schwann and Fibroblast Cells Reveals Distinct Transcriptional Programs in NF1-Associated Neurofibromas</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Swanson, G., Arenas-Hernandez, M., Gurdziel, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified distinct Schwann cell states that separated tumor from non-tumor populations and further distinguished cNFs from pNFs, highlighting both shared disease-associated features and subtype-specific adaptations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Neurofibromatosis type 1 (NF1) is an inherited tumor predisposition syndrome characterized by the development of benign peripheral nerve sheath tumors, most commonly cutaneous neurofibromas (cNFs) and plexiform neurofibromas (pNFs). Although both tumor types arise from Schwann cells and share NF1 loss as a genetic driver, they differ markedly in growth behavior, microenvironmental context, and clinical outcomes, with pNFs carrying risk of malignant transformation. To define transcriptional programs that underlie these differences, we performed an integrative single-cell RNA sequencing analysis of Schwann cells and fibroblasts from cNF and pNF tumors, alongside NF1-expressing reference populations derived from human skin and nerve. This approach enabled us to disentangle NF1-dependent transcriptional changes from tissue-of-origin effects. We identified distinct Schwann cell states that separated tumor from non-tumor populations and further distinguished cNFs from pNFs, highlighting both shared disease-associated features and subtype-specific adaptations. These findings establish a framework for understanding how NF1 loss interacts with developmental origin and tissue context to shape divergent tumor phenotypes and may inform strategies for therapeutic targeting in NF1-associated neurofibromas.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.26.677858v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Somatic evolution of prostate cancer: mutation, selection, and epistasis across disease stages</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rajaei, M., Yang, A., Glasmacher, K. A., Cross, C. N., Fisk, N., Perry, E. B., Mandell, J. D., Gaffney, S. G., Yamaguchi, T. N., Livingstone, J., Costa, J., Humphrey, P., Cannataro, V. L., Boutros, P. C., Townsend, J. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comprehensive map of the evolving selective and epistatic forces that shape prostate cancer progression across clinical stages, and a therapeutic rationale for combined inhibition of PI3K/AR pathway in PTEN-deficient prostate cancers.</p><p style="font-size: 16px; text-align: justify;">Background Somatic mutations involved in prostate cancer tumorigenesis and disease progression have been identified, but their evolutionary dynamics, including differential selective pressures across oncogenesis and metastatic spread, remain poorly understood. No prior study has systematically quantified the adaptive landscape from prostate organogenesis through tumor initiation and progression to metastatic castrate-resistant prostate cancer (mCRPC), nor characterized the selective epistatic interactions that structure this evolutionary trajectory. Methods and Findings To address this gap, we analyzed 2,704 low- and high-risk primary tumors and metastatic castration-resistant prostate cancers to quantify the mutation rates, mutational processes, and scaled selection coefficients of somatic mutations across disease stages. Trinucleotide mutational patterns were stable, but both mutation load and mutation rates increased with progression. In parallel, selective pressures on specific somatic mutations changed substantially, revealing a dynamic adaptive landscape. Stage-specific selective effects were associated with significant synergistic and antagonistic selective epistasis among key driver genes. Early selection on SPOP mutations in the BRD3 binding domain were under strong positive selection, and they increased selection for subsequent RHOA mutations while decreasing selection for TP53 mutations. Antagonistic selective epistasis was evident between mutations of CUL3 and both SPOP and PIK3CA. Mutations in KMT2C increased the selection for mutations in TP53, consistent with their frequent co-occurrence. Synergistic epistatic interactions between mutations of PTEN and both PIK3CA and AR support a strong therapeutic rationale for combined inhibition of PI3K/AR pathway in PTEN-deficient prostate cancers. Conclusions These findings provide a comprehensive map of the evolving selective and epistatic forces that shape prostate cancer progression across clinical stages. By distinguishing shifts in selection from changes in mutation rate and revealing the extents of cooperative and conflicting relationships among driver mutations, our work identifies critical points of vulnerability and informs that design of therapeutic strategies that anticipate and intercept the somatic evolutionary trajectory of prostate cancer.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.25.678387v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

