<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Friday, 10 October 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">STRIDE: A Sequencing Depth-Insensitive Metric for Robust Comparison between Sparse Chromosome Conformation Capture Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xu, B., Gao, X., Tao, W., Zhang, Z., Li, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a novel computational framework based on the mean first passage time (MFPT) in Markov chain theory, which transforms chromatin contact matrices into a robust, distance-based representation, which is inherently insensitive to sequencing depth.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The study of three-dimensional (3D) genome organization has been revolutionized by high-throughput chromatin conformation capture technologies (Hi-C) or its derivatives. However, the dependency on sequencing depth severely restricts the reliability and accuracy of existing Hi-C tools, especially in single-cells. To address this issue, we introduce a novel computational framework based on the mean first passage time (MFPT) in Markov chain theory, which transforms chromatin contact matrices into a robust, distance-based representation. We demonstrate that MFPT representation is inherently insensitive to sequencing depth. Leveraging this transformation, we develop STRIDE (Spatial Topological Representation of Interaction Distance Evaluation), a parameter-free metric for library similarity. STRIDE is resilience to sparse and noisy data, insensitive to technical variabilities, and facilitates unsupervised embedding from single-cell Hi-C, enabling accurate delineation of cellular states and developmental trajectories. In conclusion, as a reliable computational framework for sparse and noisy data, STRIDE may serve as a base for wide range single-cell 3D genome analysis that used to be inconceivable.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.08.681271v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Predicting enhancer-gene links from single-cell multi-omics data by integrating prior Hi-C information</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liang, X., Miao, Y., Han, D., Li, Y., Zhang, W., Wang, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present SCEG-HiC, a machine learning method based on weighted graphical lasso, which decodes enhancer-gene links from single-cell multi-omics data by integrating bulk average Hi-C as prior knowledge.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Enhancers play an important role in transcriptional regulation by modulating gene expression from distal genomic locations. Although single-cell ATAC and RNA sequencing (scATAC/RNA-seq) data have been leveraged to infer enhancer-gene links, establishing regulatory links between enhancers and their target genes remains a challenge due to the absence of chromatin conformation information. Here, we present SCEG-HiC, a machine learning method based on weighted graphical lasso, which decodes enhancer-gene links from single-cell multi-omics data by integrating bulk average Hi-C as prior knowledge. Comprehensive evaluation across ten single-cell multi-omics datasets from both humans and mice demonstrates that SCEG-HiC outperforms existing single-cell models, regardless of using paired scATAC/RNA-seq or scATAC-seq data alone. Application of SCEG-HiC to COVID-19 datasets illustrates its capacity to more reliably reconstruct gene regulatory networks underlying disease severity, and elucidate functional associations between non-coding variants and their putative target genes. SCEG-HiC is freely available as an open-source and user-friendly R package, facilitating broad applications in regulatory genomics research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.09.681330v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CSCN: Inference of Cell-Specific Causal Networks Using Single-Cell RNA-Seq Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, M., Yang, J., Lyu, L., Chen, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose the Cell-Specific Causal Network (CSCN) framework, which infers directed, cell-specific gene regulatory relationships by explicitly modeling causality.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding gene regulation is fundamental to deciphering the coordinated activity of genes within cells. Although single-cell RNA sequencing (scRNA-seq) enables gene expression profiling at cellular resolution, most gene network inference methods operate at the tissue or population level, thereby overlooking regulatory heterogeneity across individual cells. Recent approaches, such as Cell-Specific Network (CSN) and its extension c-CSN, attempt to construct gene networks at single-cell resolution, providing a more detailed view of the regulatory logic underlying individual cellular states. However, these methods remain limited by high false positive rates due to indirect associations and lack of directionality or causal interpretability. To address these issues, we propose the Cell-Specific Causal Network (CSCN) framework, which infers directed, cell-specific gene regulatory relationships by explicitly modeling causality. CSCN combines causal discovery techniques with efficient computation using kd-trees and bitmap indexing to perform conditional independence testing, yielding sparse and interpretable causal graphs for each cell that effectively suppress indirect and spurious associations. We demonstrate through simulations that CSCN significantly reduces false positives compared to existing methods. Furthermore, we evaluate the quality of the inferred causal networks via clustering on the Causal Katz Matrix (CKM), and CSCN outperforms CSN and c-CSN in distinguishing cellular states.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.09.681381v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A UNIQUOME BASED METHOD FOR THE PROTEIN IDENTIFICATION BY MASS SPECTROMETRY</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pierros, V., Kontopodis, E., Stravopodis, D. J., Tsangaris, G. T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed an alternative method for protein identification using Core Unique Peptides (CrUPs) and the Uniquome, termed as uniquome-Based Protein Identification Method (UB-PIM).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein identification by mass spectrometry is a pivotal step in proteomics. Numerous methods have been developed to securely and effectively identify proteins derived from experimentally detected peptides by mass spectrometry. The dominant approach is based on the assumption that each experimentally identified peptide can be matched with a peptide included in a database of peptide sequences, generated by in silico digestion of proteins with a specific proteolytic enzyme. In this way, the protein containing the peptide can be identified. In a more advanced approach, the proteins and their in silico digested peptides in the database are transformed into theoretical mass spectrometry spectra, and search engines match the experimentally obtained spectra to these theoretical spectra generated from protein and peptide sequences. We developed an alternative method for protein identification using Core Unique Peptides (CrUPs) and the Uniquome, termed as Uniquome-Based Protein Identification Method (UB-PIM). According to this method, instead of searching for peptides in the database of in silico_digested peptides, we search for CrUPs within the experimentally obtained peptides by mass spectrometry. If a peptide contains at least one CrUP, it can be directly correlated to the protein from which the CrUP is derived. Because of the unique nature of CrUPs, peptides obtained by MS can securely and uniquely identify the protein of origin. This provides a reference space in which even single-peptide identifications can achieve high specificity, reducing the ambiguity caused by shared or homologous sequences and improving the interpretability of MS data. Furthermore, UB-PIM can be applied to any type of peptide and is effective with both Data-Independent Acquisition (DIA) and Data-Dependent Acquisition (DDA) approaches, as well as with top-down and bottom-up proteomics. This allows confident protein identification from minimal evidence, expands the scope of detectable proteins, and remains computationally efficient, rapid, and universally applicable.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.08.681213v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Chryseobacterium indologenes mediates resistance to osimertinib by activating the IGF1R pathway in EGFR-mutant lung adenocarcinoma</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Li, W., Zhang, K., Raul, M., Rotter-Maskowitz, A., Nejman, D., Potempa, J., Straussman, R., Raz, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that intra-tumoral bacteria can influence response and resistance to osimertinib in EGFR-mutant lung adenocarcinoma by activating the IGF1R signaling pathway.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Introduction: Lung cancers harboring sensitizing epidermal growth factor receptor (EGFR) mutations typically exhibit an initial response to osimertinib; however, the development of resistance is inevitable, and there are currently no approved targeted therapies available once resistance emerges. Accumulating evidence indicates that intra-tumoral bacteria can influence tumor biology and contribute to therapy resistance in several cancer types, including pancreatic and colorectal cancer. Despite these findings, whether intra-tumoral bacteria play a role in modulating response and resistance to osimertinib in EGFR-mutant lung adenocarcinoma remains largely unexplored. This study aims to investigate the contribution of intra-tumoral bacteria to osimertinib resistance, thereby providing new insights into resistance mechanisms and identifying potential therapeutic strategies. Methods: Bacteria previously identified in lung cancer tissue samples were cultured in liquid growth medium, and their preconditioned medium (PCM) was collected. PC9 cells were treated with PCM in the presence or absence of osimertinib to screen for bacteria capable of mediating resistance. Cell viability was assessed using Cell Counting Kit-8 assays. To investigate potential mechanisms, Western blotting and receptor tyrosine kinase (RTK) phosphorylation arrays were performed. Control groups included cells treated with osimertinib alone, PCM alone, or vehicle. Statistical analyses were conducted using t test as appropriate, with p < 0.05 considered statistically significant. Data represent mean +/- standard deviation from at least three independent experiments. Results: The addition of PCM from Chryseobacterium indologenes restored cell viability in EGFR-mutant lung adenocarcinoma cells treated with osimertinib. PCM exposure markedly increased insulin-like growth factor-1 receptor (IGF1R) phosphorylation levels. PCM did not enhance cell viability when IGF1R was silenced or inhibited with linsitinib, demonstrating the essential role of this pathway. Proteinase K treatment abolished the ability of PCM to protect cells from osimertinib, and removal of proteins potentially interacting with IGF1R further diminished its efficacy. Finally, PCM also conferred resistance to osimertinib in patient-derived EGFR-mutant lung adenocarcinoma cell lines. Conclusion: Our study demonstrates that the intra-tumoral bacteria C. indologenes may significantly influence sensitivity to osimertinib or impart resistance in EGFR-mutant lung adenocarcinoma by activating the IGF1R signaling pathway.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.08.681060v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Targeting Tumor-intrinsic TAK1 triggers anti-tumor immunity and sensitizes pancreatic cancer to checkpoint blockade</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bansod, S. P., Lim, K.-H., Chen, H.-P., Somani, V., Li, L., Modekurty, S., Knolhoff, B., Fields, R. C., Ruzinova, M. B., DeNardo, D. G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify TGF beta activated kinase 1 (TAK1 or MAP3K7) as an aberrantly activated kin enzyme in human and mouse PDAC tissues that is associated with T cell dysfunction and highlights its potential as a therapeutic target to induce an inflamed TME that sensitiz</p><p style="font-size: 16px; text-align: justify;">Background and Aims: Targeting the Transforming Growth Factor beta (TGF beta) pathway to reverse the immunologically cold tumor microenvironment (TME) of pancreatic ductal adenocarcinoma (PDAC) remains clinically unsuccessful, warranting novel therapeutic strategies. Methods: We developed a novel tumor CD8 T cell co-culture to interrogate the TGF beta signaling pathways that promotes T cell mediated cytotoxicity. We performed multiplex immunohistochemistry (mIHC) on human PDAC samples to correlate cell type specific TGF beta pathway activation and CD8 T cell abundance. We employed specific pathway inhibitor and newly generated genetically-engineered mouse models (GEMMs) and confirmed our findings using single cell RNA sequencing, flow cytometry and mIHC. We performed proteomics and various in vitro assays to establish the molecular mechanisms. Results: We identify TGF beta activated kinase 1 (TAK1 or MAP3K7) as an aberrantly activated kinase in human and mouse PDAC tissues that is associated with T cell dysfunction. Pharmacological inhibition of TAK1 with Takinib, or genetic deletion of MAP3K7 in autochthonous p48 Cre;TP53flox/flox;LSL KRASG12D GEMM, enhances intratumoral CD4+ and CD8+ effector T cell infiltration and renders immune checkpoint blockade (ICB) effective. Mechanistically, TAK1 inhibition induces DNA damage and cytoplasmic DNA leakage, which activates the cyclic GMP-AMP synthase Stimulator of Interferon Genes (cGAS STING) DNA sensing pathway, triggering inflammatory responses that promote adaptive immune cell infiltration. At the molecular level, TAK1 phosphorylates Ephrin Receptor A2 (EphA2) at Serine 897, which in turn phosphorylates RAD51 at Tyrosine 315, a key DNA repair protein involved in homologous recombination. Conclusions: We uncover TAK1 as a critical mediator in maintaining genomic integrity and highlights its potential as a therapeutic target to induce an inflamed TME that sensitizes PDAC to ICB.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.08.681226v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Unifying Mechanism for Shared Splicing Aberrations in Splicing Factor Mutant Cancers</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Boddu, P. C., Roy, R., Hutter, S., Baumgartner, F., Li, W., Todisco, G., Ficara, F., Della Porta, M. G., Liu, Y., Haferlach, T., Pillai, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that SF-mutant cancers share a trans-acting, stress-driven AS signature wherein DDR signaling rewires SRSF1 activity impacting AS.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cancer-associated splicing factor (SF) mutations in SF3B1, U2AF1, and SRSF2 induce distinct changes in alternative splicing (AS). Yet these mutations are strikingly mutually exclusive, pointing to a convergent downstream mechanism. We hypothesized this would be reflected in the AS transcriptome. By analyzing transcriptomes of 395 patients with clonal myeloid disorders and 64 healthy donors, we found most AS alterations to be mutation-specific. However, a robust subset, enriched in the retained intron (RI) program, was shared across mutants. These RI events were bidirectional but highly concordant, and mirrored the effects of SRSF1 loss. SF-mutant states induced hypophosphorylation of RS domains in SRSF1, reducing its function. This arose from an altered AMPK-AKT balance impairing the AKT-SRPK1-SRSF1 axis. A common upstream trigger was activation of DNA damage response (DDR) by transcriptional R-loops, which increased AMPK signaling and reduced AKT activity. Pharmacologic DDR activation recapitulated reduced AKT/SRPK1 activity and SRSF1 hypophosphorylation, while relieving DDR restored SRSF1 phosphorylation and corrected RI defects. Thus, beyond cis-acting, mutation-specific changes, SF-mutant cancers share a trans-acting, stress-driven AS signature wherein DDR signaling rewires SRSF1 activity impacting AS. Our results link replication stress, kinase signaling, and RNA processing across genetically diverse clonal states, highlighting potential therapeutic approaches at these nodes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.08.679601v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identifying potential therapeutic targets for T-cell acute lymphoblastic leukemia using malignant networks and topological analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ramos, R. H., Carels, N., Scardini, R., Franca, L. L., Simao, A., Ferreira, C. d. O. L., Carneiro, F. R. G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified overexpressed genes in T-ALL cell lines, built subnetworks using the IntAct interactome, and evaluated the topological role of each gene. Clustering, largest connected component, and especially Betti 1 effectively distinguished control genes from others.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> T-cell acute lymphoblastic leukemia (T-ALL) is an aggressive and heterogeneous disease requiring new therapeutic targets. We identified overexpressed genes in T-ALL cell lines, built subnetworks using the IntAct interactome, and evaluated the topological role of each gene. An attack strategy, removing one gene at a time, was applied with eleven network measures and persistent homology. A positive control group of essential genes for T-ALL tumorigenesis was included. Clustering, largest connected component, and especially Betti 1 effectively distinguished control genes from others, showing that topological metrics are valuable for target identification. Notably, NPM1 emerged as a key gene for maintaining network integrity, promoting proliferation, and ensuring survival, highlighting its potential as a promising therapeutic target in T-ALL.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.08.681235v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrated single-cell whole genome sequencing and spatial transcriptomics reveal latent intra-tumoral heterogeneity in ovarian cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bassiouni, R., Jin, Y., Gibbs, L. D., Qian, J., Rotimi, S. O., Miller, H., Webb, M. G., Rajpara, S., Arias-Stella, J., Craig, D. W., Roman, L., Carpten, J. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of intra-tumoral heterogeneity in ovarian cancer that leads to molecular diversity, cell proliferation, and inflammation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The mortality rate of ovarian cancer remains disproportionately high compared to its incidence. This is partly due to a high level of intra-tumoral heterogeneity that promotes disease recurrence and treatment failure. In this study, we describe degrees of heterogeneity revealed by single-cell whole genome sequencing and spatial transcriptomics of five epithelial ovarian carcinomas. At the cellular level, we describe pseudo-diploid cells that match the malignant cell population in both somatic variant and copy number patterns. At the clonal and subclonal levels, we describe diversification associated with copy number gains and whole genome doubling. In multi-clonal samples, we infer evolutionary relationships from single cell copy number, loss of heterozygosity analysis, and somatic variant detection, and correlate these with tissue histology and gene expression programs. In one sample, we identify functionally consequential copy number alterations that contribute to molecular diversity, cell proliferation, and inflammation in a minor clone that persisted without major expansion alongside a more complex major clone. In another, we describe a complex evolutionary history including a spontaneous reversion of a driver mutation in a secondary clone, which correlated with a switch in oncogenic expression programs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.08.676897v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrating functional genomics and proteomics identifies Folate Carrier SLC19A1 as a predictor of pralatrexate sensitivity in T-cell lymphoma</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pantazis, J. C., Pomeroy, A. E., Palmer, A. C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify genes whose knockdown directly increases or decreases drug sensitivity in T-Cell Lymphomas, and show that pralatrexate sensitivity is a causal determinant of drug response.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cancer therapies are typically effective in subsets of patients, reflecting the molecular diversity of cancers and motivating the need for predictive biomarkers of response. Biomarker-guided therapy is increasingly useful in oncology, yet biomarker discovery remains complicated by the large number of molecular features that make it difficult to distinguish causal determinants from spurious associations. To address this challenge, we combined functional genomic screening, proteomics, and drug sensitivity profiling to discover response biomarkers for a number of therapies used in the treatment of Peripheral T-Cell Lymphomas (PTCL). First, we used genome-wide CRISPR-dCas9 interference screens in PTCL cells under drug treatment to identify a shortlist of genes whose knockdown directly increases or decreases drug sensitivity. Next, we profiled drug responses across a diverse panel of 30 PTCL cultures and, from the shortlist, identified genes whose protein abundance correlated with drug sensitivity. Genes detected by both approaches are causal determinants of drug response and correlates of drug response across the panel of lymphoma cultures, making them promising candidates for predictive biomarkers. Basal expression of the reduced folate carrier SLC19A1 was a strong predictor of pralatrexate sensitivity, consistent with its role as the primary transporter for pralatrexate uptake. Simulated clinical trials predicted that biomarker-guided patient selection could improve the power to detect significant benefit of adding pralatrexate to frontline chemotherapy in PTCL. These findings illustrate how the causal insights of functional genetic screens can augment correlative studies to identify biomarkers of drug response, and suggest the potential for precise use of pralatrexate for PTCL.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.08.681217v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Rapid Gene Expression Profiler Classifies AML Tumor Responsiveness to Standard Therapies</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kurtz, S. E., Eide, C. A., Kaempf, A., Long, N., Miller, A., Bottomly, D., McWeeney, S., Anand, S., Ng, S. W., Wang, J. C., Dick, J. E., Tyner, J. W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a rapid qPCR-based platform to profile cell state, stemness, and BCL2 family gene expression as a companion diagnostic test for acute myeloid leukemia (AML).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The emergence of transcriptional signatures that define cell types and pathways has made it possible to guide cancer therapy selection through gene expression profiling. We developed a rapid qPCR-based platform to profile cell state, stemness, and BCL2 family gene expression as a companion diagnostic test for acute myeloid leukemia (AML). We validated the stability and utility of the signatures across multiple measurement platforms and using patient samples from two centers. Integrating these signatures with clinical features enables an expedient means to predict the likelihood of patient responses to two standard-of-care therapies: intensive chemotherapy and hypomethylating agent plus venetoclax (HMA+Ven). For patients treated with HMA+Ven, expression levels of the promonocyte-like signature and BCL2 add predictive value for response and overall survival in multivariable models that include genetic features. The incorporation of the rapid profiler into the prospective evaluation of newly diagnosed AML patients may enhance treatment stratification and improve outcomes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.08.681215v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Pan-cancer and machine learning reveal the role of cell adhesion molecules in prognosis, immune remodeling, and anti-tumour therapy in lower-grade gliomas.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xu Zheng, Jianpeng Liu, Jiheng Zhang</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel CAM-related prognostic signature for lower-grade glioma with improved prognostic accuracy compared to some traditional biomarkers such as IDH mutation and 1p/19q co-deletion .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cell adhesion molecules (CAMs) have emerged as pivotal regulators of tumor biology, influencing invasion, metastasis, and the tumor microenvironment (TME) through their roles in cell-cell and cell-matrix interactions. Despite their critical roles, the prognostic value and clinical potential of CAMs in lower-grade gliomas (LGGs) remain underexplored. In this study, we utilized machine learning algorithms and multi-omics data from the Cancer Genome Atlas (TCGA) and the Chinese Glioma Genome Atlas (CGGA) to systematically evaluate CAM-associated features in LGGs. A novel CAM-related prognostic signature (CAMSig) was developed, incorporating 13 key genes, including CD58, ITGB1, and VCAM1. The CAMSig score effectively stratified patients into distinct risk groups with varying survival outcomes, showed improved prognostic accuracy compared to some traditional biomarkers such as IDH mutation and 1p/19q co-deletion in prognostic accuracy. Patients in the high-risk CAMSig group exhibited significant activation of tumor progression pathways, such as epithelial-mesenchymal transition (EMT) and PI3K-AKT signaling, along with substantial remodeling of the TME. These patients showed increased infiltration of immunosuppressive cells, including regulatory T cells (Tregs) and macrophages, as well as upregulation of immune checkpoint molecules such as PD-L1 and PD-L2, suggestive of an immunosuppressive microenvironment. Importantly, the CAMSig score also revealed differential therapeutic sensitivities. Patients with high CAMSig scores demonstrated reduced responsiveness to immune checkpoint blockade (ICB) therapy but increased sensitivity to chemotherapeutic agents, such as temozolomide. This study highlights the multifaceted role of CAMs in LGG and their potential impact across cancers, establishing the CAMSig score as a robust tool for prognostic assessment and personalized therapy guidance. By integrating insights into tumor progression, immune modulation, and therapeutic response, this research offers novel strategies for advancing the clinical management of LGG.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41066011/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Screening for Tumor Microtube-Targeting Drugs Identifies PKC Modulators as Multipotent Inhibitors of Glioblastoma Progression.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Daniel D Azor√≠n, Dirk C Hoffmann, Nils R Hebach, Erik Jung, David Hausmann, Miriam Ratliff, Ling Hai, Sandra Horschitz, Ammar Jabali, Matthias Osswald, Matthia A Karreman, Tobias Kessler, Susann Wendler, Chant√© D Mayer, Cathrin L√∂b, Pascal Lehnert, Gina Cebulla, Denise Reibold, Rajiv K Khajuria, Pino Bordignon, Andreas E Moor, Tim Holland-Letz, Jill Reckless, Nigel Ramsden, David Grainger, Anna Kreshuk, Philipp Koch, Wolfgang Wick, Sophie Heuer, Frank Winkler</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a combined, comprehensive in vitro/in vivo anti-TM drug screening approach, including machine-learning-based analysis tools, and a novel screening pipeline for anti- TM drug development.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Glioblastomas are incurable primary brain tumors that depend on neural-like cellular processes, tumor microtubes (TMs), to invade the brain. TMs also interconnect single tumor cells to a communicating multicellular network that resists current therapies. Here, we developed a combined, comprehensive in vitro/in vivo anti-TM drug screening approach, including machine-learning-based analysis tools. Two Protein Kinase C (PKC) modulators robustly inhibited TM formation and pacemaker tumor cell-driven, TM-mediated glioblastoma cell network communication. Since TM-unconnected tumor cells exhibited increased sensitivity to cytotoxic therapy, the PKC activator TPPB was combined with radiotherapy, and long-term intravital 2-photon microscopy paired with spatially resolved multiomics revealed anti-TM and anti-tumor effects. TPPB treatment also decreased the expression of tweety family member 1 (TTYH1), a key driver of invasive TMs. Our study establishes a novel screening pipeline for anti-TM drug development, identifies a TM master regulator pathway, and supports the approach of TM targeting for efficient brain tumor therapies.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41065276/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The Promise of Liquid Biopsies in Prostate Cancer: a potential point-of-care modality for Precision Oncology.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gayatri Asawa, Bhakti Basu, Sandip Basu</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Blood-based liquid biopsies represent a transformative, minimally-invasive, and patient-friendly approach to cancer detection and real-time monitoring of treatment effectiveness.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Blood-based liquid biopsies represent a transformative, minimally-invasive, and patient-friendly approach. Tumor-derived components in the bloodstream, are at the forefront of active research for diagnosis and prediction of prognosis. Early cancer detection and real-time monitoring of treatment effectiveness are the most relevant from the perspective of both the clinicians and the patients. A demand for non-invasive markers is ably powered by sustained advancements in high-throughput technologies such as Next Generation Sequencing and mass spectrometry, paired with artificial intelligence and machine learning, which are requisites in expanding the power of liquid biopsies through multi-analyte tests.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41064992/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

