<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Saturday, 21 June 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Differential DNA methylation in blood in nuclear genes that encode mitochondrial proteins in mild cognitive impairment and Alzheimers disease</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Boruch, A. E., Madrid, A., Papale, L. A., Bergmann, P. E., Renteria, I. S., Faasen, S., Cook, D. B., Alisch, R. S., Hogan, K. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that nuclear genes that encode proteins that participate in mitochondrial function are differentially methylated between cognitively unimpaired participants (CU) and those with MCI and late-onset dementia due to Alzheimers disease (AD).</p><p style="font-size: 16px; text-align: justify;">Background: Altered mitochondrial function contributes to the pathogenesis of mild cognitive impairment (MCI) and late-onset dementia due to Alzheimers disease (AD). Objective: To test for differential methylation in nuclear genes that encode proteins that participate in mitochondrial function between cognitively unimpaired participants (CU) and those with MCI and AD. Methods: Recently published whole genome methylation sequencing (WGMS) in blood from CU participants (N=174), and those with MCI (N=99) and AD (N=109) was used to test for differential methylation in 1121 nuclear genes that encode proteins that participate in mitochondrial function in the Human Protein Atlas. Results: Seventy-four nuclear genes that encode proteins that participate in mitochondrial function were differentially methylated between persons with MCI and CU. Seventy-one genes were differentially methylated between persons with AD and CU, and 132 genes were differentially methylated between persons with MCI and AD. Thirteen differentially methylated genes shared between the 3 comparisons support contributions from disrupted metabolism and oxidative stress pathways in AD pathogenesis. Conclusions: Nuclear genes that encode proteins that participate in mitochondrial glucose metabolism, fatty acid metabolism and oxidative stress pathways are differentially methylated between persons who are CU and those with MCI and AD.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.16.659556v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PAH-former: Transfer Learning for Efficient Discovery of Pulmonary Arterial Hypertension-Associated Genes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kawakami, T., Hosokawa, S., Masamichi, I., Kurozumi, A., Tanaka, R., Minatsuki, S., ishida, J., Isagawa, T., Kodera, S., Takeda, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A deep learning algorithm for single-cell RNA sequencing of pulmonary artery hypertension and fine-tuning it with public data to identify the disease-relevant genes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell RNA sequencing (scRNA-seq) of patient samples holds promise for understanding disease mechanisms, but faces the challenge of excessive cost and effort in acquisition, processing, and data analysis, making it essential to leverage existing data. Pulmonary artery hypertension (PAH) is a refractory disease characterized by pulmonary vascular remodeling, and access to patient specimens is limited due to difficulties in tissue collection. In this study, we employed transfer learning with Geneformer, a deep learning algorithm pre-trained with scRNA-seq datasets and fine-tuned it with public PAH lung tissue data to identify the disease-relevant genes. The resulting algorithm, which we named PAH-former, demonstrated that its prediction accuracy varied significantly depending on the dataset used for fine-tuning. PAH-former enabled us to perform in silico perturbation analysis and identified PAH related genes. Loss-of-function PAH related genes in human pulmonary artery endothelial cells increased the expression of SOX18, a signature gene of PAH. This integration of artificial intelligence and biological experiments can significantly advance our understanding of molecular mechanisms of PAH.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.16.660027v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">BiGER: Bayesian Rank Aggregation in Genomics with Extended Ranking Schemes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, K., Yang, Y., Xia, Y., Xiao, G., Lim, J., Wang, X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose BiGER, a fast and accurate Bayesian rank aggregation method for the inference of latent global rankings.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> With the rise of large-scale genomic studies, large gene lists targeting important diseases are increasingly common. While evaluating each study individually gives valuable insights on specific samples and study designs, the wealth of available evidence in the literature calls for robust and efficient meta-analytic methods. Crucially, the diverse assumptions and experimental protocols underlying different studies require a flexible but rigorous method for aggregation. To address these issues, we propose BiGER, a fast and accurate Bayesian rank aggregation method for the inference of latent global rankings. Unlike existing methods in the field, BiGER accommodates mixed gene lists with top-ranked and top-unranked genes as well as bottom-tied and missing genes, by design. Using a Bayesian hierarchical framework combined with variational inference, BiGER efficiently aggregates large-scale gene lists with high accuracy, while providing valuable insights into source-specific reliability for researchers. Through both simulated and real datasets, we show that BiGER is a useful tool for reliable meta-analysis in genomic studies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.17.660022v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Uncovering the Pathophysiological Pattern of Expression from Integrated Analysis across Uniformly Processed RNA Sequencing COVID-19 Datasets</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cole, A., Fukuda, B., Dahlstrom, T. J., Hung, L.-H., Yeung, K. Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comprehensive molecular framework for post-acute sequelae of SARS-CoV-2 infection, with profound clinical implications.</p><p style="font-size: 16px; text-align: justify;">Background: Post-acute sequelae of SARS-CoV-2 infection (PASC) affects millions globally, yet the molecular mechanisms underlying acute COVID-19 and its chronic sequelae remain poorly understood. Methods: We performed an integrative transcriptomic analysis of three independent RNA-seq datasets, capturing the complete COVID-19 pathophysiology from health through acute severe infection to post-acute sequelae and mortality (n=142 total samples). We implemented a containerized analytical pipeline from data download, quantification, differential gene expression to uniformly process these three RNA-seq datasets. Results: Our analysis reveals striking molecular dichotomies contrasting disease phases with profound clinical implications. Acute severe/critical COVID-19 reveals predominant enrichment of TNF- signaling via NF-{kappa}B pathways (normalized enrichment score >2.5, FDR 2.2, FDR <0.005), indicating important shifts toward cellular adaptation. Pathway signature analysis identifies core differentially expressed genes that reliably distinguish disease phases, thereby offering objective biomarkers for precision diagnosis and monitoring. Conclusions: These findings establish a comprehensive molecular framework distinguishing acute inflammatory from chronic metabolic COVID-19 phases, with potential clinical applicability. TNF-/NF-{kappa}B pathway signatures identify patients at risk for severe disease progression, while Myc/OXPHOS signatures allow objective PASC diagnosis, addressing current reliance on subjective and eliminative diagnosis. This integrative analytical framework has utility beyond COVID-19, offering an applicable approach for precision medicine implementation across other diseases processes. Clinical Significance: This study transforms COVID-19 from a symptom-based to a molecularly-defined disease spectrum, enabling precision diagnosis, prognostic monitoring, classification, and targeted therapeutic possibilities based on pathway-specific biomarkers rather than subjective clinical assessments.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.18.660454v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">NKX2-5 congenital heart disease mutations show diverse loss and gain of epigenomic, biochemical and chromatin search functions underpinning pathogenicity</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ward, A. O., Schonrock, N., McCann, A. J., Phanor, S. K., Kock, K. H., Kurland, J. V., Wu, F., Murray, N. J., Walshe, J., Alankarage, D., Dunwoodie, S. L., Meunier, F. A., Francois, M., Bulyk, M. L., Ramialison, M., Harvey, R. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We report an integrated functional-epigenomics approach to understanding congenital heart defects, focusing on the cardiac homeodomain (HD) family transcription factor NKX2-5, mutations in which cause diverse congenitals heart structural and conduction defects.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Congenital heart defects (CHD) occur in ~1% of live births, with both inherited and acquired mutations and environmental factors known to contribute to causation. However, network perturbations and epigenetic changes in CHD remain poorly characterised. We report an integrated functional-epigenomics approach to understanding CHD, focusing on the cardiac homeodomain (HD) family transcription factor NKX2-5, mutations in which cause diverse congenital heart structural and conduction defects. We selected twelve NKX2-5 CHD-associated variants affecting different residue classes - DNA base-contacting, backbone-contacting, and helix-stabilizing residues of the homeodomain, and those affecting other conserved protein:protein interaction (PPI) domains. In HL-1 cardiomyocytes, we profiled DNA targets of NKX2-5 wild type (WT) and variant proteins genome-wide using DamID, and their DNA binding affinity and specificity using comprehensive protein binding microarrays. We also profiled changes to PPI with known NKX2-5 cofactors using yeast 2-hybrid assay, and chromatin search and binding dynamics using single molecule tracking. Variants showed highly diverse but also class-specific behaviours with a range of severities. All variants failed to bind many WT targets but retained binding to a variable subset of core cardiomyocyte-related WT NKX2-5 targets, as well as hundreds of unique "off-targets", in part via a regulatory logic that included changes to DNA binding site specificity, homodimerization and/or cofactor interactions. All variants tested showed altered chromatin search functions to varying degrees. Our data suggest that complex residue-to-residue scale chromatin search and epigenomic perturbations, involving both loss- and gain-of-function, contribute to CHD phenotypes. These findings may inform precision molecular therapeutic approaches in patients with CHD.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.20.659510v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Monotherapy cancer drug-blind response prediction is limited to intraclass generalization</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Herbert, W., Chia, N., Jensen, P. A., Walther-Antonio, M. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Drug-blind prediction is a poor benchmark for cancer drug response prediction, and drug-blind performance is the poor benchmark, it does not describe model behavior, it describes dataset behavior. We first identify that drug overfitting is inhibiting generalization and preventing accurate analysis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Monotherapy cancer drug response prediction (DRP) models predict the response of a cell line to a given drug. Analyzing these models performance includes assessing their ability to predict the response of cell lines to new drugs, i.e. drugs that are not in the training set. Drug-blind prediction displays greatly diminished performance or outright failure across a wide range of model architectures and different large pharmacogenomic datasets. Drug-blind failure is hypothesized to be caused by the relatively limited set of drugs present in these datasets. The time and cost associated with further cell line experiments is significant, and it is impossible to predict beforehand how much data would be enough to overcome drug-blind failure. We must first define how current data contributes to drug-blind failure before attempting to remedy drug-blind failure with further data collection. In this work, we quantify the extent to which drug-blind generalizability relies on mechanistic overlap of drugs between training and testing splits. We first identify that the majority of mixed set DRP model performance can be attributed to drug overfitting, likely inhibiting generalization and preventing accurate analysis. Then, by specifically probing the drug-blind ability of models, we reveal the sources of generalizable drug features are confined to shared mechanisms of action and related pathways. Furthermore, we observed that, for certain mechanisms, we can significantly improve performance by limiting the training of models to a single mechanism compared to training on all drugs simultaneously. We conclude that drug-blind performance is a poor benchmark for DRP as it does not describe model behavior, it describes dataset behavior. Our investigation displays that deep learning models trained on large, monotherapy cell line panels can more accurately describe mechanism of action of drugs rather than their advertised connection to broader cancer biology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.16.659838v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Frequency-Blended Diffusion Models for Synthetic Generation of Biologically Realistic Splice Site Sequences</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kabanga, E., Jee, S., Van Messem, A., De Neve, W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a frequency-blended diffusion framework for generating biologically realistic splice site sequences in Arabidopsis thaliana and Homo sapiens.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We present a frequency-blended diffusion framework for generating biologically realistic splice site sequences. Our approach combines a U-Net-based denoising diffusion probabilistic model with conditional nucleotide frequency priors derived from real donor (5') and acceptor (3') splice site sequences in Arabidopsis thaliana and Homo sapiens. By guiding the generative process with position-specific empirical base frequencies, the model captures both local sequence motifs and long-range dependencies that are critical for realistic splice site representation. We evaluate the synthetic sequences through direct assessments (sequence logos, GC content, nucleotide conservation) and indirect functional tests using state-of-the-art splice site classifiers (SpliceRover, SpliceFinder, DeepSplicer, Spliceator). Our results show that frequency blending substantially improves motif conservation, compositional fidelity, and model transferability. This work establishes frequency-blended diffusion as a promising strategy for generating high-quality nucleotide sequences for modeling, benchmarking, and data augmentation in genomics research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.16.660047v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comparative study of manganese catalase monomers, interfaces and cage architecture within the ferritin superfamily</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Amrita, A., Chakraborti, S., Dey, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We study the structure of 6-mer manganese catalase protein cage with focus on its ability to form a cage architecture and show efficient catalytic activity.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Manganese catalase protein is an example of a protein cage within the ferritin superfamily that focuses on enzymatic catalysis rather than on storage which other ferritin proteins are known for. Formed of 6 homomeric chains, it shows large subunit-subunit sidewise contacts and special interfacial interactions ensuring cage generation with only 6 subunits. We aim to explore manganese catalase at the monomer, subunit-subunit and cage level. As compared to other ferritin subtypes, we found manganese catalase monomers to have greater fraction of {beta}-turns at the secondary structure level and thermophilic manganese catalase monomers to have greater non-polar to polar residue ratio. At the interface level, the placement of subunits in manganese catalase cage with sidewise and angular interface was found to be distinctly different from the parallel and perpendicular interfaces present in other ferritin subtypes. Our study highlights the contribution of sidewise interfaces and terminal extensions in making the cage architecture possible in 6-mer manganese catalases. We also studied the quaternary structure of manganese catalase cage with respect to Classical ferritin (C-ferritin) and found manganese catalase to have smaller cavity volume and cavity surface area than C-ferritin but higher cavity surface to volume ratio. This observation along with the smaller distance between substrate entry point and active site as compared to C-ferritin highlights the structural distinction between catalytic enzymes like manganese catalase and storage proteins like ferritins. Thus, the study gives structural insight into manganese catalase protein cage with focus on its ability to form a cage architecture and show efficient catalytic activity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.17.660058v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Regulation Flow Analysis discovers molecular mechanisms of action from large knowledge databases</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Roca, C. P., Sysoev, O., Eyre, E., Galan, S., Sinibaldi, D., Tedder, P., Mangion, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel method, called Regulation Flow Analysis (RFA), which is based on the principles of biological regulation, causal graphs, and graph flow.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Drug development is a long and expensive process, with only a small fraction of potential drugs being finally approved. The challenge of drug development is rooted in our limited understanding of biological systems and the disease processes that drugs are trying to modulate. We propose a novel method, called Regulation Flow Analysis (RFA), which is based on the principles of biological regulation, causal graphs, and graph flow. RFA is able to generate causal hypotheses of mechanism of action, using large Knowledge Graphs (KG) of molecular regulation. Our numerical experiments demonstrate that the generated hypotheses, in the form of regulation pathways, often summarize mechanisms of drug action in a manner both understandable and actionable. Thus, RFA can greatly improve our understanding of the biological processes underlying health and disease, and therefore substantially facilitate drug development. Our examples illustrate how RFA recovers known mechanisms of action and can be used for target and biomarker discovery and validation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.15.659785v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Accelerating k-mer-based sequence filtering</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Martayan, I., Vandamme, L., Constantinides, B., Cazaux, B., Paperman, C., Limasset, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose efficient streaming search methods to recover specific relevant sequences directly, without exhaustive pre-indexing, and demonstrate how minimizer-based sketching techniques, alongside SIMD acceleration, can enhance the performance of such streaming searches.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The exponential growth of global sequencing data repositories presents both analytical challenges and opportunities. While k-mer-based indexing has improved scalability over traditional alignment for identifying potentially relevant documents, pinpointing the exact sequences matching a query remains a hurdle, especially when considering the false positives and false negatives caused by sparse k-mer sampling or probabilistic data structures. Furthermore, searching for numerous k-mers with a large query or multiple distinct ones strains existing exact matching tools, whose performance scales poorly with an increasing number of patterns. Indexing entire vast datasets for infrequent or ad-hoc searches is often resource-prohibitive. This paper addresses the renewed need for efficient streaming techniques to recover specific relevant sequences directly, without exhaustive pre-indexing. Having discussed the relevance of streaming search methods in bioinformatics, we propose diverse use cases as benchmarks to encourage further research into tailored solutions for these challenges, and demonstrate how minimizer-based sketching techniques, alongside SIMD acceleration, can enhance the performance of such streaming searches. These contributions highlight practical approaches and identify avenues for future theoretical and empirical investigation in managing and querying vast genomic datasets. Availability: https://github.com/Malfoy/K2Rmini.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.16.659853v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Distribution and Effect of Echium Plantagineum L. on Floral Species Diversity in the Dejen District, Northern Ethiopia</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tamiria, G. K., Meteko, K. B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study on the distribution and effect of E. plantagineum on the diversity of floral in the area of its occurrences.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Invasive plant species are considered the greatest global threat to the loss of biodiversity. Echium plantagineum L. an invasive species, is primarily found along roadsides and grazing lands in the Amhara Regional State of the Dejen district. The purpose of this study was to investigate the distribution and effect of E. plantagineum on the diversity of floral in the area of its occurrences. Reconnaissance survey was conducted to observe the distribution, spread, and effect of E. plantagineum on plant and animal species. As a result, 116 systematically established sample plots from the study were taken (58 invaded and non-invaded plots for each). To gather information on trees (shrubs) 10m x 20m sample plots were created on grazing land types. At each corner, a 1m2 subplot was taken to collect herbaceous plants. A 4m2 plot on roadside land units was created to gather information on the herbaceous flora, tree or shrub seedlings. Plants in each plot were recorded and named. Visual estimation was used to determine the percentage cover of herbaceous species. The study assessed species diversity using species evenness, Shannon Diversity Index, and Simpson Index of Dominance measures. These metrics were analyzed using PAST software. Similarity between invaded and non-invaded areas was evaluated using Jaccard's similarity index and beta diversity. The relationship between the abundance of E. plantagineum and floral species abundance was examined through linear regression and correlation analyses. A total of 85 plant species from 32 families were discovered in the study areas. In non-invaded land units, 80 plant species from 30 families were recorded. In comparison, the invaded area contained 54 plant species grouped into 24 families. Thus, the number of plants was reduced by 32.5% in the E. plantagineum-invaded area compared to the non-invaded area. The findings also showed that the non-invaded areas have a higher Shannon Diversity index value was observed in plant (3.537) and also higher plant species evenness value (0.429) compared to the E. plantagineum invaded areas. The study also showed a negative relationship between the abundance of E. plantagineum invasives and the abundance of native plant species per study plot. It was concluded that E. plantagineum, the invasive plant species in the study area, has reduced floral species diversity. Therefore, it is strongly advised to implement adequate planning and strategies to detect or stop the spread and effects of E. plantagineum. This can be achieved by creating communication channels between the regional, zonal, and district agricultural offices.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.16.660053v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Exogenous Hormone Treatments Reveal Species-Specific Regulation of Individual Components of Root Architecture and Salt Ion Accumulation in Cultivated and Wild Tomatoes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rahmati-Ishka, M., Craft, E., Pineros, M., Julkowska, M. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A detailed analysis of root architecture and ion accumulation under salt stress in tomato with species-specific hormonal effects .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Hormonal signaling shapes plant architecture and salt stress responses, but its effects on root architecture and ion accumulation remain unclear. Here, we conducted a detailed analysis of how individual hormone treatments affect root architecture and ion accumulation under salt stress in tomato. The study focused on three tomato accessions with varying responses to salt stress. Our findings revealed distinct, species-specific hormonal effects. Auxin, ethylene, and gibberellin promoted lateral root development, yet their impacts on ion accumulation, particularly in Na/K ratio, varied considerably. To explore the molecular basis of these differences, we examined Arabidopsis mutants for ethylene- and auxin-related genes, revealing novel components of hormone signaling involved in the salt stress response. Further, analysis of tomato mutants with impaired ethylene perception demonstrated that the nr mutant exhibits increased root growth and a higher shoot Na/K ratio, largely due to reduced K retention. Our integrated physiological and genetic analysis reveals species-specific hormonal strategies can boost crop performance under salt stress.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.17.660158v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Cellulase secretion by engineered Pseudomonas putida enables growth on cellulose oligomers.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Smith, M. R., Moffitt, K., Holdsworth, W., Luna-Flores, C. H., Goyal, M., Beliaev, A., Speight, R. E., Behrendorff, J. B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We have developed a new method for engineering P. putida and other gram-negative bacteria to secrete non-native enzymes for extracellular feedstock depolymerisation, and this work supports progress towards consolidated bioprocessing of cellulosic materials using P. Putida.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Pseudomonas putida is an attractive synthetic biology platform organism for chemical synthesis from low-grade feedstocks due to its high tolerance to chemical solvents and lignin-derived small molecules that are often inhibitory to other biotechnologically relevant microorganisms. However, there are few molecular tools available for engineering P. putida and other gram-negative bacteria to secrete non-native enzymes for extracellular feedstock depolymerisation. In this study P. putida was transformed to secrete cellulase enzymes and evaluated for growth on polymeric or oligomeric cellulose substrates. Active exo- and endocellulase enzymes were secreted into the culture supernatant, and a preferred set of twin-arginine translocase secretion signal peptides were identified. Extracellular cellulase activity was sufficient to support growth of P. putida using cellotriose or cellotetraose as the sole source of carbon and energy. This work supports progress towards consolidated bioprocessing of cellulosic materials using P. putida, and advances the state of engineered protein secretion in gram negative bacteria.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.19.659671v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Mind the Gap: An Embedding Guide to Safely Travel in Sequence Space</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wu, A., Trolliet, Q., Rajendran, A., Lala, J., Angioletti-Uberti, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a hybrid approach combining a protein language model (pLM) with Monte Carlo sampling for generating enzyme mutants free of mutations deleterious for structural preservation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We present a hybrid approach combining a protein language model (pLM) with Monte Carlo (MC) sampling for generating enzyme mutants free of mutations deleterious for structural preservation. Given the amino acid sequence of the original enzyme and a set of residues for which the local environment should be conserved, i.e., the catalytic site, our approach generates mutants that differ vastly in the overall sequence while retaining the geometry of the conserved region, thereby representing promising candidates for further experimental screening. Unlike end-to-end deep learning approaches, whose results are harder to interpret and control, the use of a well-established, classic technique such as MC sampling allows us to easily interpret the generative process as the sampling of an energy landscape determined by the pLM. In turn, such an interpretation enables us to steer this generative process and control its outcome by making use of robust statistical mechanics concepts, e.g., temperature, thereby explicitly guaranteeing certain properties of the generated mutants. Given the increasing relevance of generative algorithms in the design and search for novel, optimised enzymes, we believe that our results constitute an important step for the future development of this class of techniques. To facilitate experimental verification, we finally provide hundreds of sequences for 13 different enzymes involved in catalytic processes ranging from carbon dioxide conversion to DNA replication.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.19.660524v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Temporal gene regulation enables controlled expression of gas vesicles and preserves bacterial viability</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Li, Z., Jeong, S., Lu, G. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A dual-inducer transcriptional regulation system that enables orthogonal control of GV assembly factor proteins and the shell protein over a range of stoichiometries.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Gas vesicles (GVs) are genetically encodable, air-filled protein nanostructures that have rapidly emerged as a versatile platform for biomedical imaging, cell tracking, and therapeutic delivery. However, their heterologous expression in non-native hosts such as Escherichia coli can be challenging due to the complex assembly process, which often involves around ten different proteins and can lead to proteotoxic stress and impair cell growth. Here, we report the observation of a drop in cell density occurring 8 to 16 hours after GV induction in E. coli. To address these, we developed a dual-inducer transcriptional regulation system that enables orthogonal control of GV assembly factor proteins and the shell protein over a range of stoichiometries. Sequential induction in time, in which assembly factor expression is initiated before shell protein expression, restored normal bacterial growth and prevented lysis without compromising GV production. Further analysis revealed that varying the interval between the two induction steps affected both GV yields and cellular stress. By preserving cell integrity with GV expression, our approach enhances the utility of GV-expressing bacteria in applications that demand population-wide cellular stability and facilitates their broader application in biomedical engineering and synthetic biology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.20.660610v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">How F repeats help in peptide binding with VIM-2 metallo-beta-lactamase and destabilizing the enzyme</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Anand, A. A., Anwar, S., Mondal, R. K., Samanta, S. K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We explore the role of phenylalanine and lysine residues, especially their repeat patterns, in modulating peptide binding affinity and structural interactions with VIM-2.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The global surge in antimicrobial resistance is a major public health concern, largely driven by the dissemination of beta-lactamases. Among them, VIM-2 metallo-beta-lactamase poses a significant therapeutic challenge. In this study, we explore the role of phenylalanine and lysine residues, especially their repeat patterns, in modulating peptide binding affinity and structural interactions with VIM-2. Docking, molecular dynamics simulations, free energy decomposition analysis and residue analysis were conducted on gut metagenome-derived AMPs, PolyF and a control peptide (PolyR). Binding affinity with VIM-2 decreased with declining F content, yet PolyF alone showed unexpectedly low binding. However, PolyF exhibited very low polar solvation energy in free energy decomposition analysis. This indicates that F repeats play a role in stabilizing a VIM-2-peptide complex in presence of a solvent. PolyF peptide lacks favorable electrostatic or dynamic interactions, likely due to the absence of K residues. Interestingly, loop1 and loop2 analysis of VIM-2 revealed that F repeats interfere with the loops that play a role in functioning of substrate binding at the active site. Additionally, F repeats were found to bind to residues lying adjacent to or nearby active site residues, which points towards the fact that F helps other residues like K to bind more stably with active site residues of VIM-2. Our findings suggest that a balance of aromatic (F) and positively charged (K) residues could enhance peptide binding with VIM-2. This work informs the rational design of next-generation peptide-based inhibitors targeting MBLs like VIM-2.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.15.659760v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

