<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Saturday, 24 May 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Deconstructing the GWAS library - Next Generation GWAS.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Weirui Zhang, Svenja Koslowski, Marouane Benzaki, Chang Jie Mick Lee, Yike Zhu, Michelle C E Mak, Yonglin Zhu, Shaun S E Loong, Guillaume Lettre, Chukwuemeka George Anene-Nzelu, Roger Foo</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We provide an overview of the various GWAS performed in heart disease and describe the various methods employed to prioritize disease-relevant variants from these studies, including bioinformatic and experimental approaches.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genome-wide association studies (GWAS) have identified numerous common genetic variants associated with cardiovascular traits and diseases. These studies have increased our understanding of the genetic architecture of cardiac diseases and have facilitated the identification of genetic risk factors in patients. Furthermore, they have spurred the development of novel effective therapies by targeting the causal disease pathways. Despite the demonstrated clinical utility of GWAS, the mechanism of action of many of these variants remains unstudied, and this has hindered the full potential of GWAS. Various high-throughput screening and machine-learning technologies have been developed to assist with predicting and prioritizing pathogenic variants for experimental validation. These technologies can potentially unravel novel pathways in disease pathogenesis and accelerate the development of new therapies. In this review, we provide an overview of the various GWAS performed in heart disease and describe the various methods employed to prioritize disease-relevant variants from these studies, including bioinformatic and experimental approaches. We highlight relevant examples that have applied these tools to identify the specific variants in each identified locus and how some of these variants have spurred novel therapies. Finally, we discuss the outstanding challenges facing research in this field and how they can be addressed.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40408169/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">POCALI: Prediction and Insight on CAncer LncRNAs by Integrating Multi-Omics Data with Machine Learning.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ziyan Rao, Chenyang Wu, Yunxi Liao, Chuan Ye, Shaodong Huang, Dongyu Zhao</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new algorithm for cancer lncRNAs by integrating 44 multi-omics features across six categories.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Long non-coding RNAs (lncRNAs) are receiving increasing attention as biomarkers for cancer diagnosis and therapy. Although there are many computational methods to identify cancer lncRNAs, they do not comprehensively integrate multi-omics features for predictions or systematically evaluate the contribution of each omics to the multifaceted landscape of cancer lncRNAs. In this study, an algorithm, POCALI, is developed to identify cancer lncRNAs by integrating 44 omics features across six categories. The contributions of different omics are explored to identifying cancer lncRNAs and, more specifically, how each feature contributes to a single prediction. The model is evaluated and benchmarked POCALI with existing methods. Finally, the cancer phenotype and genomics characteristics of the predicted novel cancer lncRNAs are validated. POCALI identifies secondary structure and gene expression-related features as strong predictors of cancer lncRNAs, and epigenomic features as moderate predictors. POCALI performed better than other methods, especially in terms of sensitivity, and predicted more candidates. Novel POCALI-predicted cancer lncRNAs have strong relationships with cancer phenotypes, similar to known cancer lncRNAs. Overall, this study facilitates the identification of previously undetected cancer lncRNAs and the comprehensive exploration of the multifaceted feature contributions to cancer lncRNA prediction.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40405764/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">De Bruijn Graph Partitioning for Scalable and Accurate DNA Storage Processing</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> De Moor, F., Boulle, O., Lavenier, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a novel method based on de Bruijn graph partitioning, enabling fast and accurate processing of sequencing data regardless of the underlying sequencing technology and without requiring prior knowledge of the information encoded in the oligonucleotides.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivations: DNA-based data storage offers a compelling solution for long-term, high-density archiving. In this framework, accurately reconstructing high-quality encoded sequences after sequencing is critical, as it has a direct impact on the design of error-correcting codes optimized for DNA storage. Furthermore, efficient and scalable processing is essential to manage the large volumes of data expected in such applications Results: We introduce a novel method based on de Bruijn graph partitioning, enabling fast and accurate processing of sequencing data regardless of the underlying sequencing technology and without requiring prior knowledge of the information encoded in the oligonucleotides. Evaluated on both synthetic and real datasets, the method achieves excellent precision and recall. It is implemented in C++ within the software ConCluD and optimized for multi-core servers. Our experiments demonstrated that a dataset of 55 million reads, corresponding to a 135 MB binary file, can be processed in less than 10 minutes on a 16 hyper-threaded core server.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.19.654814v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PEARL: Integrative multi-omics classification and omics feature discovery via deep graph learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhao, Q., Du, J., Zhou, M., Wang, X.-W., Sun, Q., Chen, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present PEARL, a novel deep graph learning method for biomedical classification and functional important omics features identification.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Integrating multi-omics data provides valuable insights into biological processes by capturing information across multiple molecular layers, enabling a comprehensive understanding of complex diseases and driving advancements in precision medicine. However, existing computational methods for multi-omics integration face significant challenges, such as low reliability and poor generalizability, due to the high dimensionality and low sample size nature of omics data. To address these challenges, we present PEARL (Pearson-Enhanced spectrAl gRaph convoLutional networks), a novel deep graph learning method for biomedical classification and functional important omics features identification. PEARL leverages a simple yet effective learning architecture to achieve superior and robust performance in high-dimensional, low-sample-size multi-omics settings. Our results demonstrate that PEARL significantly outperforms existing state-of-the-art methods on both synthetic and real biomedical datasets. Furthermore, applied to Alzheimer's disease (AD) brain multi-omics data, features prioritized by PEARL lead to functionally important genes that demonstrate significant enrichment in AD-related pathways. These findings highlight PEARL's practical utility in biomedical research and its potential to enhance biological interpretability in multi-omics studies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.19.654754v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Prevalence of Group II Introns in Phage Genomes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Merk, L. N., Jones, T. A., Eddy, S. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that group II introns are widely prevalent in bacteriophage genomes, from diverse phylogenetic backgrounds, from endosymbiont phage to jumbophage.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Although bacteriophage genomes are under strong selective pressure for high coding density, they are still frequently invaded by mobile genetic elements (MGEs). Group II introns are MGEs that reduce host burden by autocatalytically splicing out of RNA before translation. While widely known in bacterial, archaeal, and eukaryotic organellar genomes, group II introns have been considered absent in phage. Identifying group II introns in genome sequences has previously been challenging because of their lack of primary sequence similarity. Advances in RNA structure-based homology searches using covariance models has provided the ability to identify the conserved secondary structures of group II introns. Here, we discover that group II introns are widely prevalent in phages from diverse phylogenetic backgrounds, from endosymbiont phage to jumbophage.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.22.655115v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">TEITbase: a database for transposable element (TE)-initiated transcripts in human cancers</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, Y., She, J., Hu, X., Jin, Y., Tao, C., Du, M., Yang, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a deep learning-based method and identified 38,995 TE-initiated transcripts across 33 cancer types from The Cancer Genome Atlas, and created a user-friendly database that provides researchers with tools to conduct various downstream analyses and investigations for TE-Initiated Transcripts.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Transposable elements (TEs) are abundant and play a crucial regulatory role in the human genome. Functioning as alternative promoters, TEs can be reactivated to produce TE-initiated transcripts. In cancer, TE-initiated transcription may upregulate oncogene expression and generate novel tumor-specific antigens, which could serve as potential targets for immunotherapy. However, there remains a lack of comprehensive databases that systematically investigate TE-initiated transcription in cancer. To address this gap, we developed a deep learning-based method and identified 38,995 TE-initiated transcripts across 33 cancer types from The Cancer Genome Atlas. Among these, 6,203 were tumor-specific expressed and strongly associated with the tumorigenesis. Using these annotations, we created TEITbase (http://teitbase.medbioinfo.org/), a user-friendly database that provides researchers with tools to conduct various downstream analyses and investigations for TE-initiated transcripts, including 546 novel onco-exaptation events. The establishment of TEITbase provides new insights into transcriptional reprogramming in cancers and enables further investigation into the potential roles of TE-initiated transcripts in cancer diagnostics and therapeutic strategies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.19.654796v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">EYKTHYR reveals transcriptional regulators of spatial gene programs</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Krieger, S., Haber, E., Ma, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce EYKTHYR, a computational framework that integrates gene expression and chromatin accessibility within a spatially aware model to identify spatial transcriptional regulators and decode how TFs interpret local intercellular signaling to shape tissue structure.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding how transcription factors (TFs) orchestrate gene regulatory networks that define complex tissue structures is central to uncovering tissue organization and disease mechanisms. Although spatial multiome technologies now enable in situ measurement of both transcriptional activity and chromatin accessibility, existing computational methods either overlook spatial tissue context or are hindered by the high dropout rates characteristic of such data. Here, we introduce EYKTHYR, a computational framework that integrates gene expression and chromatin accessibility within a spatially aware model to identify TFs driving spatial gene programs. EYKTHYR mitigates dropout effects by leveraging interpretable, low-dimensional embeddings of gene expression and chromatin accessibility -- both linear with respect to their input -- enabling robust identification and scalable inference of spatial transcriptional regulators. Applied across diverse spatial multiome datasets, EYKTHYR consistently outperforms existing approaches, accurately identifying TFs that coordinate spatial gene programs in mouse brain development and regulate T-cell states within tumor microenvironments. EYKTHYR establishes a foundation for decoding how TFs interpret local intercellular signaling to shape tissue structure, offering insights into the regulatory logic underlying spatial organization in health and disease.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.19.654884v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">stTrace: Detecting Spatial-Temporal Domains from spatial transcriptome to Trace Developmental Path</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Song, Z., Zheng, C., Chen, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose an algorithm for detecting spatial-temporal domains in tissue to trace developmental path (stTrace) using spatial transcriptomics using spatial location, gene expression, and spatial location to identify regions where cells share similar functions and developmental stages.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Development is essential for the growth and functional maintenance of organisms. Investigating the development process is vital for uncovering the formation of complex biological systems. However, current approaches to studying development from gene expression rely primarily on single-cell gene expression data to infer developmental trajectories, neglecting the spatial distribution of cells within tissues and their interactions. Although spatial transcriptomics provides spatial context for gene expression, existing algorithms focus mainly on identifying spatial regions without further exploring their developmental connections. In this study, we propose an algorithm for detecting spatial-temporal domains in tissue to trace developmental path (stTrace) using spatial transcriptomics. stTrace integrates the degree of cell development, gene expression, and spatial location to identify "spatial-temporal domains", regions where cells share similar functions and developmental stages within the tissue. Moreover, hierarchical relationships exist among these regions, reflecting developmental connections between cells in the tissue. Our experiments on mouse embryo and human breast cancer data revealed that stTrace can detect more refined regions than traditional spatial domain identification algorithms. Furthermore, the directions of developmental paths inferred from hierarchical relationships are consistent with the dynamic trajectories derived from single-cell velocity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.19.654812v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">upsAI: A high-accuracy machine learning classifier for predicting Plasmodium falciparum var gene upstream groups</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pangilinan, E. A., Quenu, M., Claessens, A., Otto, T. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed upsAI, a machine learning-based classifier trained on 2,530 curated var genes, to accurately assign upstream groups using sequence features from different partial gene regions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Plasmodium falciparum erythrocyte membrane protein 1 (PfEMP1), encoded by the hypervariable var gene family, is central to malaria pathogenesis, influencing both disease severity and immune evasion. Classifying var genes into upstream groups (upsA, upsB, upsC, upsE) is important for understanding parasite biology and clinical outcomes, but remains challenging, especially with partial sequences, such as the DBL tag or RNA-Seq assemblies. We developed upsAI, a machine learning-based classifier trained on 2,530 curated var genes, to accurately assign upstream groups using sequence features from different partial gene regions. We compared seven different methods, including support vector machines, random forest, XGB boost and HMMer models. The best model of upsAI for DBL-tags sequences achieves an overall accuracy of 83%, 92% and for full-length var genes, therefore significantly outperforming existing tools. Further, we propose a new model to distinguish between internal and subtelomeric var genes with high accuracy and scalability. upsAI is available at https://github.com/sii-scRNA-Seq/upsAI, providing a robust and efficient resource for large-scale var gene analysis. It can classify var genes from 20 genomes in under one second.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.19.654848v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">RBPSignal: A deep learning approach for predicting RNA-Protein binding signals</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ding, X. S., Shu, Z. A., Wang, S., Shi, L., Zhou, Y., Zhang, B., Shen, H.-B., Liu, X., Pan, X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a deep learning-based computational tool designed to predict RBP binding signals on RNAs and identify potential sequence motifs associated with these interactions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> RNA-binding proteins play critical roles in post-transcriptional regulation by interacting with RNA to regulate various cellular processes. Inferring the binding signals and sequence patterns of these interactions is essential for elucidating the mechanisms underlying gene expression regulation. In this study, we present RBPSignal, a deep learning-based computational tool designed to predict RBP binding signals on RNAs and identify potential sequence motifs associated with these interactions. RBPSignal leverages the deep learning framework and trains on comprehensive eCLIP datasets, demonstrating an enhanced predictive accuracy. Furthermore, the integration of model interpretability through Integrated Gradients enables the detailed analysis of binding motif syntax. We validate the efficacy of RBPSignal on chromosome data and compare the discovered motifs with existing motif database, showcasing its ability not only to predict binding signals but also to uncover sequence patterns correlated with binding signals. Our findings provide insights into the sequence specificity of RBPs and explore the protein-protein interaction networks. RBPSignal serves as a valuable tool for exploring RBP-RNA interaction landscapes, facilitating further investigations into the regulatory networks underlying gene expression. The web server is freely available at http://www.csbio.sjtu.edu.cn/bioinf/RBPSignal/.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.19.654811v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multiscale analysis and optimal glioma therapeutic candidate discovery using the CANDO platform</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> XU, S., Mangione, W., Norden, M. V., Elefteriou, K., Hu, Y., Falls, Z., Samudrala, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We used the Computational Analysis of Novel Drug Opportunities (CANDO) platform for multiscale therapeutic discovery to predict new glioma therapies.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Glioma is a highly malignant brain tumor with limited treatment options. We employed the Computational Analysis of Novel Drug Opportunities (CANDO) platform for multiscale therapeutic discovery to predict new glioma therapies. We began by computing interaction scores between extensive libraries of drugs/compounds and proteins to generate "interaction signatures" that model compound behavior on a proteomic scale. Compounds with signatures most similar to those of drugs approved for a given indication were considered potential treatments. These compounds were further ranked by degree of consensus in corresponding similarity lists. We benchmarked performance by measuring the recovery of approved drugs in these similarity and consensus lists at various cutoffs, using multiple metrics and comparing results to random controls and performance across all indications. Compounds ranked highly by consensus but not previously associated with the indication of interest were considered new predictions. Our benchmarking results showed that CANDO improved accuracy in identifying glioma-associated drugs across all cutoffs compared to random controls. Our predictions, supported by literature-based analysis, identified 23 potential glioma treatments, including approved drugs like vitamin D, taxanes, vinca alkaloids, topoisomerase inhibitors, and folic acid, as well as investigational compounds such as ginsenosides, chrysin, resiniferatoxin, and cryptotanshinone. Further functional annotation-based analysis of the top targets with the strongest interactions to these predictions identified vitamin D3 receptor, thyroid hormone receptor, acetylcholinesterase, cyclin-dependent kinase 2, tubulin alpha chain, dihydrofolate reductase, and thymidylate synthase. These findings indicate that CANDO's multitarget, multiscale framework is effective in identifying glioma drug candidates, thereby informing new strategies for improving treatment.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.19.654757v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Using domain insertion to create sulfite reductases that present chemical-dependent activities</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Windham, E., Myerscough, D., Schwartz, S., Carpenter, M., Ajo-Franklin, C., Silberg, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We study the effect of domain insertion on the function of NADPH-dependent sulfite reductase from Escherichia coli, a dodecameric protein containing eight hemop Protein subunits and four flavoprotein subunits.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Domain insertion can be used to create oxidoreductase switches whose charge transfer is dependent upon analyte binding. To date, most domain insertion studies have targeted relatively small proteins of known structure, so it remains unclear how to effectively use this protein engineering approach with large oligomeric oxidoreductases that require dynamic conformational changes for catalysis. To address this question, we studied the effect of domain insertion on the function of NADPH-dependent sulfite reductase (SiR) from Escherichia coli, a dodecameric protein containing eight hemoprotein subunits and four flavoprotein subunits. SiR mutational tolerance was first mapped using systematic peptide insertion, and a subset of variants retaining activity were subjected to domain insertion. When a ligand-binding domain was inserted at locations tolerant to peptide insertion, including sites proximal and distal from the intersubunit interfaces, more than half of the resulting variants presented cellular activity that is enhanced by an endocrine disruptor. This ligand-dependent synthesis of a redox-active metabolite could be monitored electrochemically from cells, illustrating how a single protein complex can be used to convert chemical information in the environment into a metabolite within cells that diffuses across the cell membrane and can be detected electrochemically.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.23.655854v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

