<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Saturday, 27 September 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multi-omics analysis of parthanatos related molecular subgroup and prognostic model development in stomach adenocarcinoma.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xiangxin Wu, Lianming Cai, Yanping Liu, Bowen Wang, Tianyi Xia, Zhenhua Liu</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel, novel, and robust prognostic signature for parthanatos-associated genes in gastric cancer .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Stomach adenocarcinoma (STAD), the most prevalent histological subtype of gastric cancer, exhibits high heterogeneity and poor prognosis, posing significant therapeutic challenges. Parthanatos, a distinct form of regulated cell death mediated by poly (ADP-ribose) polymerase-1 (PARP-1), has been implicated in tumor biology and therapeutic resistance; however, the role of parthanatos-associated genes (PRGs) in STAD remains largely unexplored. In this study, we performed a comprehensive multi-omics analysis integrating transcriptomic, genomic, and clinical data from public databases to delineate the molecular landscape of PRGs in STAD. Unsupervised clustering revealed distinct PRG-related molecular subtypes with significant differences in clinical outcomes, immune infiltration profiles, and biological pathway activation. Based on machine learning algorithms, we established and validated a novel PRG-based prognostic signature, which demonstrated robust predictive performance. Moreover, single-cell RNA sequencing and in vitro functional assays were conducted to explore cellular heterogeneity and gene function. Notably, in vitro experiments, including western blot, colony formation, CCK-8, and Transwell assays, confirmed that one key PRG, COL8A1, promotes STAD cell proliferation and migration. Collectively, our findings highlight the clinical and biological significance of PRGs in STAD, offering novel biomarkers and potential therapeutic targets for STAD precision treatment.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41004487/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multi-omics based consensus subtypes, development of prognostic signature, and identification of INHBB as a potential therapeutic target in colorectal cancer.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xu Wang, Yuanmin Xu, Rui Sun, Sheng Wang, Xiang Wei</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A multi-omics study on colorectal cancer (CRC) and a machine learning-based prognostic score (CMLS).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> This study aims to refine molecular subtypes via multi-omics data, develop a prognostic signature, and identify novel biomarkers in colorectal cancer (CRC). On the basis of the multi-omics data, the MOVICS R package was used to divide patients with CRC into three consensus subtypes (CSs). Among the 3 CSs, CS1 had higher immune scores, immune checkpoint expression, infiltration of immune cells, and sensitivity to chemotherapy drugs. Specific markers of CS1 were identified, and 101 combinations of machine learning methods were applied for calculating the consensus machine learning-based score (CMLS) and constructing the CMLS signature for predicting patient survival. CMLS showed high efficiency in predicting the outcome of patients across multiple CRC datasets, and the results demonstrated that CMLS was an independent prognostic factor in CRC. The high- and low-CMLS groups presented distinct immune landscapes. CMLS was linked to malignant cancer features, suggesting its potential as a predictor of malignant progression in diverse cancers. Among the genes used to calculate CMLS, the role of inhibin subunit beta B (INHBB) in CRC remains unexplored. Expression analysis of INHBB across different cancer types revealed its upregulation in CRC, which was further validated by western blot and immunohistochemistry (IHC) experiments. INHBB silencing significantly inhibited tumor cell proliferation and migration, decreased phosphorylated AKT and N-cadherin levels, and increased E-cadherin expression. INHBB potentially suppresses CRC development and progression by suppressing the AKT signaling pathway and the EMT process.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41003812/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">GRUMB: A Genome-Resolved Metagenomic Framework for Monitoring Urban Microbiomes and Diagnosing Pathogen Risk.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Suleiman Aminu, AbdulAziz Ascandari, Rachid Benhida, Rachid Daoud</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present GRUMB (Genome-Resolved Urban Microbiome Biosurveillance), an open-source, SLURM-compatible pipeline that reconstructs high-quality metagenome-assembled genomes (MAGs) from shotgun sequencing reads and integrates taxonomic/functional annotation</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Urban infrastructure hosts dynamic microbial communities that complicate biosurveillance and AMR monitoring. Existing tools rarely combine genome-resolved reconstruction with ecological modeling and batch-aware analytics tailored to infrastructure-scale studies. We present GRUMB (Genome-Resolved Urban Microbiome Biosurveillance), an open-source, SLURM-compatible pipeline that reconstructs high-quality metagenome-assembled genomes (MAGs) from shotgun sequencing reads and integrates taxonomic/functional annotation (CARD, VFDB), batch-aware normalization, ecological diagnostics and machine learning classification of environment types with uncertainty and risk scoring. GRUMB accepts either SRA project accessions or paired-end FASTQ files with metadata, and produces assemblies, MAGs, taxonomic and functional profiles, ecological outputs and risk-informed classification. Its modular design enables reproducible, infrastructure-scale biosurveillance across diverse environments.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41002268/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MITE: the Minimum Information about a Tailoring Enzyme database for capturing specialized metabolite biosynthesis.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Adriano Rutz, Daniel Probst, César Aguilar, Daniel Y Akiyama, Fabrizio Alberti, Hannah E Augustijn, Nicole E Avalon, Christine Beemelmanns, Hellen Bertoletti Barbieri, Friederike Biermann, Alan J Bridge, Esteban Charria Girón, Russell Cox, Max Crüsemann, Paul M D'Agostino, Marc Feuermann, Jennifer Gerke, Karina Gutiérrez García, Jonathan E Holme, Ji-Yeon Hwang, Riccardo Iacovelli, Júlio César Jeronimo Barbosa, Navneet Kaur, Martin Klapper, Anna M Köhler, Aleksandra Korenskaia, Noel Kubach, Byung T Lee, Catarina Loureiro, Shrikant Mantri, Simran Narula, David Meijer, Jorge C Navarro-Muñoz, Giang-Son Nguyen, Sunaina Paliyal, Mohit Panghal, Latika Rao, Simon Sieber, Nika Sokolova, Sven T Sowa, Judit Szenei, Barbara R Terlouw, Heiner G Weddeling, Jingwei Yu, Nadine Ziemert, Tilmann Weber, Kai Blin, Justin J J van der Hooft, Marnix H Medema, Mitja M Zdouc</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present MITE, the Minimum Information about a Tailoring Enzyme database, a comprehensive set of parameters to annotate tailoring enzymes, defining substrate and reaction specificity by the expressive reaction SMARTS (Simplified Molecular Input Line Entry System Arbitrary Target Specification).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Secondary or specialized metabolites show extraordinary structural diversity and potent biological activities relevant for clinical and industrial applications. The biosynthesis of these metabolites usually starts with the assembly of a core 'scaffold', which is subsequently modified by tailoring enzymes to define the molecule's final structure and, in turn, its biological activity profile. Knowledge about reaction and substrate specificity of tailoring enzymes is essential for understanding and computationally predicting metabolite biosynthesis, but this information is usually scattered in the literature. Here, we present MITE, the Minimum Information about a Tailoring Enzyme database. MITE employs a comprehensive set of parameters to annotate tailoring enzymes, defining substrate and reaction specificity by the expressive reaction SMARTS (Simplified Molecular Input Line Entry System Arbitrary Target Specification) chemical pattern language. Both human and machine readable, MITE can be used as a knowledge base, for in silico biosynthesis, or to train machine-learning applications, and tightly integrates with existing resources. Designed as a community-driven and open resource, MITE employs a rolling release model of data curation and expert review. MITE is freely accessible at https://mite.bioinformatics.nl/.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41001881/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Small Molecule Approach to RNA Targeting Binder Discovery (SMARTBind) Using Deep Learning Without Structural Input</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jiang, S., Taghavi, A., Wang, T., Meyer, S. M., Childs-Disney, J. L., Li, C., Disney, M. D., Li, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Small Molecule Approaches to RNA Targeting Binder Discovery (SMARTBind), a structure-agnostic ligand discovery framework that combines an RNA large language model with contrastive learning and a ligand-specific decoy enhancement strategy.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurate identification of small molecule binders to RNA is critical for chemical probes and therapeutics. Computational approaches offer a cost-effective strategy to identify small molecules targeting RNA but are often limited by poor predictive accuracy and high computational demands. Here, we introduce Small Molecule Approaches to RNA Targeting Binder Discovery (SMARTBind), a structure-agnostic ligand discovery framework that combines an RNA large language model with contrastive learning and a ligand-specific decoy enhancement strategy. The RNA language model, pre-trained on millions of RNA sequences, together with the decoy enhancement strategy, addresses data scarcity and improves model generalizability. SMARTBind uses only RNA primary sequence to identify small molecule binders and their binding sites accurately. Across multiple benchmarks and case studies, SMARTBind outperforms existing data-driven and docking-based methods while significantly reducing computational cost. In a real-world application, SMARTBind identified novel small molecules targeting the precursor of oncogenic microRNA-21, validated by in vitro and cellular assays. These results highlight SMARTBind's potential as a scalable, accurate, and structure-independent platform for RNA-targeted small molecule discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.24.678312v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The regulatory landscape of nascent transcription in human health and disease</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shah, S. R., Chen, Y., Leung, A. K., Navarro, P. V. C., Paramo, M. I., Gupta, J., Gurumurthy, A., Fite, R. F., Weimer, A. K., Mohyeldin, A. M., Du, Q., Egli, D., Creusot, R. J., Ryan, R. J. H., Snyder, M. P., Clark, A. G., Lis, J. T., Yu, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A high-resolution atlas of nascent transcriptional regulatory elements, which reveals how patterns of transcription initiation and pausing encode regulatory logic, sheds light on the genetic architecture of human traits and diseases.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Transcriptional regulatory elements (TREs) orchestrate gene expression programs fundamental to cellular identity and transitions between physiological and pathological states. Decoding the regulatory logic of human biology requires resolving where, when, and how these elements are transcriptionally engaged. Here, we profiled the active transcriptional regulatory landscape across all major organ systems and a broad spectrum of developmental and disease states using PRO-cap, a high-resolution method that captures nascent transcription start sites with unprecedented sensitivity and specificity. This atlas of active TREs highlights elements shaped by their cellular contexts and evolutionary constraints, sheds light on the genetic architecture of human traits and diseases, and reveals how patterns of transcription initiation and pausing encode regulatory logic. In cancer, nascent transcription enables the delineation of lineage-specific regulatory states, metastatic adaptations, and the co-option of pre-existing programs. Together, these findings establish nascent transcription as a core dimension of gene regulation, illuminating principles that govern development, physiology, and disease.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.24.676871v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Nanopore genome skimming with Illumina polishing yields highly accurate mitogenome sequences: a case study of Niphargus amphipods</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Salussolia, A., Lecoquierre, N., Stoch, F., Flot, J.-F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the first complete mitogenome sequence of Niphargus Schiodte, 1849, a deep-sea amphipod genus with a rich diversity of species, and a first step towards a phylogeny of the genus.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> With over 430 species currently described, the amphipod genus Niphargus Schiodte, 1849 is the most species-rich crustacean genus in subterranean waters. Previous phylogenetic studies of this genus have relied mainly on mitochondrial COI and nuclear 28S sequences, which do not resolve all the nodes in its phylogeny. As a first step towards a mitogenome-based phylogeny of niphargids, we present here the first complete mitogenome sequence of Niphargus. To obtain high-accuracy mitogenome sequences and annotations, genome skims of three individuals of the Niphargus dolenianensis Lorenzi,1898 species complex were generated using both short, accurate reads (Illumina) and long, noisier reads (Nanopore). Whereas direct assembly of Illumina sequences yielded structurally incorrect mitogenome sequences, assembly of Nanopore reads produced highly accurate sequences that were corroborated by mapping of Illumina reads. Polishing of the Nanopore consensus using Illumina reads corrected a handful of errors at the homopolymer level. The resulting mitogenome sequences ranged from 14,956 to 15,199 bp and shared the same arrangement of 13 protein-coding genes, two ribosomal RNA genes, 22 transfer RNA genes, and a putative control region. Phylogenetic analyses based on protein-coding genes confirmed that the Niphargidae family is sister to Pseudoniphargidae, resolving their relationships with other amphipod families. This highlights the utility of mtDNA genome sequences for studying the evolution, diversification, and taxonomy of this groundwater genus.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.24.678374v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Construction of a reference genome for Starmerella batistae and annotation of Starmerella species reveal a close evolutionary relationship with Schizosaccharomyces pombe and suggest an alternative pathway for sophorolipid production.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Timouma, S., Hanak, A., Balarezo-Cisneros, L. N., Donaldson, I., Valle, F., Delneri, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We isolated a yeast contaminant from the laboratory environment, identified as Starmerella batistae, able to thrive under low pH, including high concentrations of lactic acid, and relative high temperatures.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The Starmerella clade is known for displaying osmotolerant and acidophilic traits from their association with bees. Several species in this genus can produce sophorolipids, which are commercially produced as biosurfactants. Here, we isolated a yeast contaminant from the laboratory environment, identified as Starmerella batistae, able to thrive under low pH, including high concentrations of lactic acid, and relative high temperatures. We have sequenced and conducted a de novo genome assembly in three chromosomes and a mitochondrial genome for S. batistae (ca 9.3 Mb). Based on this reference genome we functionally annotated 29 different Starmerella species, using the publicly available sequences. Phylogenetic analysis across different yeast clades revealed a close relationship of Starmerella species with Schizosaccharomyces yeasts. Fifteen genes were uniquely shared between Sz. pombe and S. batistae, of which twelve were involved in cell morphology. Interestingly, the shape of S. batistae cells is elongated rather than round, similar to the Sz. pombe. Additionally, we found that all the Starmerella strains capable of producing sophorolipids shared a last common ancestor. Such clustering can help identify other sophorolipid-producing Starmerella yeasts that have not yet been characterised. We did not find the one-to-one orthologs of S. bombicola sophorolipid pathway in any of the Starmerella sp. with the exception of S. kuoi, S. powellii and S. floricola. In S. etchelsii, the antisense and telomeric pair UGTA1/CYP52M1 was found to be structurally conserved although not functionally. These findings support the notion that alternative pathways for the production of sophorolipids have evolved in different Starmerella lineages.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.24.678367v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comprehensive Transcriptome Annotation of Thousands of HIV-1 Genomes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Varabyou, A., Artamonov, M., Cheng, S., Bolton, D. L., Salzberg, S. L., Pertea, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present HIV Atlas, the first reference-quality annotation of HIV-1 and SIV transcriptional diversity, and demonstrate near-perfect conservation of splice sites, revealing near- perfect preservation of major donors and acceptors.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Alternative splicing in HIV-1 has been a central focus of decades of research, uncovering key mechanisms of viral gene regulation, immune evasion, and therapeutic response - yet no reference resource has existed to support transcriptome-wide analysis, limiting adoption of modern computational methods. We present HIV Atlas (https://ccb.jhu.edu/HIV_Atlas), the first reference-quality annotation of HIV-1 and SIV transcriptional diversity. We manually curated transcriptomes for HIV-1HXB2 and SIVmac239 and developed Vira, an automated annotation-transfer method specifically designed to address unique challenges of viral genome biology, to generate high-quality annotations for 2,077 complete HIV-1 genomes. Using the resources presented in our work, we evaluated conservation of splice sites, revealing near-perfect preservation of major donors and acceptors. Furthermore, using several public datasets, we demonstrate how HIV Atlas enhances methodology, improves the quality and novelty of results, and opens novel avenues for research, supporting more accurate and comprehensive analyses of bulk, single-cell, and spatial RNA-seq in HIV-1 studies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.24.675449v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Exploring the genetic basis of cluster architecture-related traits in grapevine through a Genome-wide Association Study</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> de Oliveira, G. L., Francisco, F. R., de Moura, Y. A., Niederauer, G. F., Fritsche-Neto, R., de Souza, A. P., Furlan, M. F. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A genome-wide association study of grape berry size and cluster architecture using a diverse panel of 288 Vitis spp.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Berry and cluster size are pivotal determinants of grapevine productivity and consumer preferences and remain major targets in grapevine breeding. However, given their complexity as quantitative traits under polygenic control, a deeper understanding of their genetic determinants is needed. The gene pool of the Brazilian grapevine has made a significant contribution to enhancing grapevine performance in tropical and subtropical regions. In this study, we conducted a genome-wide association study (GWAS) using a diverse panel of 288 Vitis spp. accessions from the Agronomic Institute Germplasm Bank, Brazil. This panel was phenotyped for six critical cluster architecture traits over 12 years and genotyped using the Vitis18k SNP array. Using two different algorithms, the GWAS identified 56 significant SNPs distributed across 17 chromosomes, validating previously identified QTLs and revealing novel associations. Four closely spaced markers on Chr1 suggest the presence of a QTL influencing five traits simultaneously. A strong association signal, explaining approximately 29-35% of the phenotypic variance (PVE), indicated a major QTL for berry length (BL) and width (BWi) on Chr14. Additionally, major-effect SNP loci were identified for cluster weight (CW) on Chr1, cluster length (CL) on Chr7 and 14, cluster width (CWi) on Chr6 and 18, and berry weight (BW) on Chr4, with PVE values ranging from 18-27%. Furthermore, 80 genes associated with berry traits and 52 genes associated with cluster traits were identified as putative candidate genes in the genomic regions associated with significant SNPs. These candidate genes are involved in the regulation of growth and development, hormone regulation, protein synthesis, stress response, and other physiological processes essential for cell health and functionality. Our results provide valuable insights into the genetic determinants of grape berry size and cluster architecture, offering critical data to support future functional studies and enhance the efficiency of related breeding programs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.24.678328v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">High-throughput SNP discovery, development and validation of a 30 K target SNP genotyping tool for cultivated flax (Linum usitatissimum) breeding and germplasm characterization</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> DEMENOU, B. B., PINEAU, C. P., Le CLAINCHE, I., BERARD, A., FAIVRE RAMPANT, P., HINSINGER, D. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed and validated a target SNPs genotyping tool named AT-SNP-30K using Allegro targeted SNPs technology for the cultivation of cultivated flax.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The cultivated flax (Linum usitatissimum L.) is an industrial crop widely cultivated for fiber and seeds, in broad geographical regions around the world. This crop faces many challenges (yield, quality, biotic and abiotic stresses) linked to climate change as almost all other crops. Among solution for improving or breeding new adapted cultivars, marker-assisted selection has been widely applied in plant breeding to enhance crop yield, quality, and tolerance to biotic or abiotic stresses. Recent advance of targeted genotyping-by-sequencing (GBS) offers an ultimate MAS tool to accelerate plant breeding and crop improvement. To facilitate the utility of SNP-based genotyping, we developed and validated in this study a target SNPs genotyping tool named AT-SNP-30K using Allegro targeted SNPs technology. A total of 41k SNPs were selected from 4.78 million and 3.73 million SNPs identified in two different accessions panels, respectively. Probe design for all these markers was successfully achieved for 35,791 of these markers, representing a 86% conversion success rate. The set of markers was then validated by genotyping a diversity panel comprising 384 individuals, including 376 accessions and eight replicates of the fiber flax Ideo cultivar. The validated genotyping tool includes 35,791 SNPs, covering the fifteen chromosomes with 24,951 high-quality SNPs (MAF> 5%, average low rate of missing data) and 27,247 SNPs having a MAF greater than 1%; demonstrating high polymorphism and excellent genotyping accuracy. The repeatability of genotyping in the validation experiment, reached 99.00% of SNPs for the eight Ideo replicate controls. The AT-SNP-30K genotyping tool is a robust resource for genetic studies, germplasm characterisation and cultivated flax marker-assisted selection studies. It can be used to enhance the breeding of new flax cultivars adapted to the context of climate change.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.24.678307v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">High-depth Whole Genome Sequencing of Single Human Colon Crypts Uncovers New View on Crypt Clonality</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Manojlovic, Z., Okitsu, C., Okitsu, T., Wlodarczyk, J., Lieber, M. R., Loh, Y. H. E., Hsieh, C.-L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We comprehensively profiled somatic mutations in 71 single human colon crypts with matched bulk controls from 14 individuals.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> It is generally accepted that each colon crypt is monoclonal and is populated by a single stem cell lineage after early human life. It has been impossible to profile somatic mutations genome-wide because high-depth and high-quality whole genome sequencing (WGS) of single cells is unachievable without tissue culture or whole genome amplification (WGA). Therefore, the cell-to-cell heterogeneity in each individual remains mostly unknown. Applying our novel WGA-free WGS method to obtain >30X post-alignment depth, we comprehensively profiled somatic mutations in 71 single human colon crypts with matched bulk controls from 14 individuals. Analysis reveals that colon crypts are commonly of multiple lineages in human adults. Our study is the first to demonstrate that an appropriately designed WGS approach can determine cell-to-cell heterogeneity in natural cell clones. The much higher sensitivity of WGS than the previous methods in lineage tracing can unlock the complex stem cell dynamics in the colon crypt.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.24.677335v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Long-term clonal analysis using stochastic models reveals heterogeneity and quiescence of hematopoietic stem cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Garcia Vilela, Y., Thielecke, L., Cesar Fassoni, A., Glauche, I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a stochastic, two-compartment model in which HSCs transition reversibly between an actively cycling state and a quiescent, potentially niche-bound state.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Hematopoietic stem cells (HSCs) maintain lifelong production of blood by balancing self-renewal and differentiation. However, certain aspects of their divisional dynamics, namely the role of quiescence and the intrinsic heterogeneity of the HSC pool, are not completely understood. High-resolution clonal tracking provides a powerful resource to investigate such dynamics as the data captures patterns of clonal persistence, dilution and late clonal emergence. Here, we apply mechanistic mathematical modeling to longitudinal clonal data from non-human primates to explore structural requirements that underlie the observed dynamical patterns. We show that models treating HSCs as a single, homogeneous population can explain the gradual loss of clonal diversity, but fail to reproduce clone size distributions and the long-term persistence of small and late-appearing clones. To address this, we propose a stochastic, two-compartment model in which HSCs transition reversibly between an actively cycling state and a quiescent, potentially niche-bound state. Compared to the simpler one-compartment model, this advanced framework provides a substantially improved fit for different metrics, consistently captures clone size distributions and explains the delayed activation and sustained coexistence of small and large clones. These results provide quantitative evidence that heterogeneity within the HSC pool, particularly the existence of a reversible quiescent state, is critical to account for clonal aspects of long-term hematopoiesis. Our findings highlight how clonal data can uncover underlying regulatory mechanisms and supports a central role for niche-mediated HSC quiescence in maintaining stable and diverse blood production over time.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.24.677849v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">HRDPath: An Explainable Multi-Model Deep Learning Architecture for Predicting Homologous Recombination Deficiency from Histopathology Images</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wu, C.-i., Banda, K., Arora, R., Soong, T. R., Radke, M., Dillon, M., Nik-Zainal, S., Swisher, E., Sailem, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a novel patient-level deep learning architecture that combines the strengths of two complementary models with a multi-task design, to predict genomically derived HRD status from whole slide images in high-grade serous tubo-ovarian carcinoma.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Homologous recombination deficiency (HRD) is a critical biomarker for guiding treatment decisions in high-grade serous tubo-ovarian carcinoma (HGSOC), a cancer with few reliable biomarkers. However, existing genomic-based tests for HRD are variable, expensive, and time-consuming. To this end, we developed HRDPath, a novel patient-level deep learning architecture that combines the strengths of two complementary models with a multi-task design, to predict genomically derived HRD status from whole slide images in HGSOC. HRDPath was comprehensively validated across three datasets and benchmarked against leading deep learning models. It achieved an AUC of 0.846, surpassing previously reported H&E-based HRD prediction results for HGSOC images by 0.09, and for the first time, reporting a specificity of 0.938, where accuracy significantly increased when multiple slides per patient were used. Our proposed patient-level approach and interpretability pipeline enhance model trustworthiness and reveal important clinical and biological insights into HRD-positive cancers, highlighting the associated morphological and pathological changes at the cellular and tissue levels. HRDPath is a potentially accessible and scalable digital biomarker that could improve ovarian cancer diagnosis and therapy selection.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.24.678258v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genomic and Transcriptomic Insights into the Evolution and Parasitic Strategy of the Woody-Plant Nematode Pratylenchus vulnus</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dai, D., Zhang, Y., Latina, R., Yang, X., Williamson, V. M., Niels Groen, S., Shamsullah, S., Leslie, C. A., Castro, B., Siddique, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a high-quality, chromosome-level genome assembly of the root-lesion nematode Pratylenchus vulnus (61.7 Mb across six chromosomes). Comparative genomic analysis revealed high collinearity in protein-coding genes between P. vuln</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The root-lesion nematode Pratylenchus vulnus parasitizes a wide range of hosts including woody perennials such as walnut (Juglans regia) and grapevine (Vitis vinifera), significantly damaging roots and reducing yields. Here, we present a high-quality, chromosome-level genome assembly of P. vulnus (61.7 Mb across six chromosomes). Comparative genomic analysis revealed high collinearity in protein-coding genes between P. vulnus and the root-knot nematode Meloidogyne graminicola, indicating a closer evolutionary relationship with this sedentary endoparasite. Large chromosomal regions in P. vulnus lack synteny with other nematode genomes, have comparatively low GC content (<30%), and are enriched in genes with unique or lineage-specific functions. Transcriptome analysis highlighted dynamic, stage-specific expressions of genes involved in parasitism, development, and metabolism. Additionally, we identified an extensive repertoire of putative effector genes and characterized lineage-specific expansions of cell wall-degrading enzyme families. Overall, these findings provide insight into the genome organization, chromosome evolution, and parasitism-related gene repertoire in a woody-plant parasitizing nematode.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.24.678327v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Techniques and challenges in studying photosynthetic diversity in freshwater aquatic vascular plants</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Loew-Mendelson, E., Wickell, D., Mrozinski, C., Doan, E., Hupp, J., Vath, R. L., Yuan, L., Heyduk, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a method for targeted molecular assays of phosphoenolpyruvate carboxylase (PEPC) expression that could provide a sensitive tool for characterizing photosynthetic diversity in a wide range of aquatic plants.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> While the photosynthetic diversity of aquatic plants rivals that of terrestrial species, the environmental conditions underlying that diversity fundamentally differ. Despite these environmental differences, aquatic and terrestrial plants have convergently evolved Carbon Concentrating Mechanisms (CCMs). However, characterization of these pathways in submerged plants has lagged behind terrestrial systems due to methodological constraints of the aquatic environment. Here we review and evaluate contemporary methods for detecting CCMs in aquatic plants. Physiological methods including gas exchange and isotopic analyses provide valuable insights in terrestrial plants but face significant challenges in aquatic systems. Biochemical assays of organic acid accumulation reliably detect CCMs in aquatic species but may struggle to detect weak or non-canonical CCMs in submerged plants. Finally, we propose a method for targeted molecular assays of phosphoenolpyruvate carboxylase (PEPC) expression that could provide a sensitive tool for characterizing photosynthetic diversity in a wide range of aquatic species. Our results show that methods and frameworks developed for terrestrial plants do not necessarily directly translate to aquatic systems. However, by extending these methods and integrating multiple lines of evidence we can improve our ability to characterize photosynthetic diversity in aquatic plants.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.24.677520v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Expression screening and functional verification of recombinant hirudin from novel chassis cell Chlamydomonas reinhardtii</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Feng, L., Xing, J., Tian, A., Wang, W., Zhang, Y., Wang, W., Hu, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A green, scalable biomanufacturing and oral delivery platform for therapeutics, highlighting transformative potential at the nexus of agricultural bioengineering, functional foods, and biomedicine.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Hirudin is a potent anticoagulant peptide whose clinical application is limited by scarce natural supplies and the suboptimal activity of recombinant versions from conventional microbial hosts lacking key post-translational modifications. Here, for the first time, we establish the "Generally Regarded as Safe" (GRAS) microalga, Chlamydomonas reinhardtii, as a novel biological chassis for producing fully functional hirudin. Using high-density heterotrophic fermentation, we generated a bioactive lyophilized algal powder suitable for oral delivery. Crucially, the algal-derived hirudin underwent proper tyrosine sulfation-a key modification absent in prokaryotic hosts-conferring exceptionally high thrombin-inhibitory activity (up to 20,000 ATU/mg). When orally administered to a murine thrombosis model, this hirudin-loaded alga demonstrated potent antithrombotic efficacy. It also exhibited a superior safety profile, showing no signs of the thrombocytopenia or bleeding associated with traditional anticoagulants. This study establishes a green, scalable biomanufacturing and oral delivery platform for therapeutics, highlighting transformative potential at the nexus of agricultural bioengineering, functional foods, and biomedicine.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.25.678210v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CytoScan: Automated detection of technical anomalies for cytometry quality control</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mocking, T. R., Zwolle, F., Park, Y., Kelder, A., Saeys, Y., Cloos, J., van Gassen, S., Bachas, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a novel R package, CytoScan, that evaluates inter-measurement variation in cytometry datasets and allows for detecting anomalous measurements after data acquisition.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Studies evaluating cellular phenotypes by cytometry techniques are increasingly facing analytical challenges due to the multitudes of samples and parameters that are evaluated concurrently. Spurious technical effects resulting from a lack of standardization can affect marker distributions and further complicate multi-sample analyses. User-friendly tools for exploratory data analysis to identify such technical effects in large datasets are lacking. To fill this gap, we present a novel R package, CytoScan, that evaluates inter-measurement variation in cytometry datasets and allows for detecting anomalous measurements after data acquisition. CytoScan can detect two types of anomalies: files with limited similarity to others within a dataset (outliers) and files with limited similarity to previously acquired high-quality reference data (novelties). Using simulations of skewed marker distributions and real-life technical effects we demonstrate that CytoScan can accurately detect such anomalies. CytoScan can be applied to large cytometry datasets on consumer-grade hardware with informative visualizations, providing accessible quality-control for more reliable analyses.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.24.678276v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integration of Multiomic and Multi-phenotypic Data Identifies Biological Pathways Associated with Physical Fitness</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Alizadeh, A., Graf, J., Misner, M. J., Burns, A. A., Ginty, F., O'Donovan, K. J., Wickiser, J. K., Barringer, N., Freisinger, G., Hermansen, N., McDonough, J. E., Davis, B. M., Loghin, E. R., Surrette, C., Tu, P., Welch, J., Boomhower, O., Lenigk, R., Sorell, R., Hammond, T., Peterson, S., Caron, A., Safa, L., Chadwick, C., Stacey, S., Jobin, J., Evans, S. C., Xue, R., Khinda, G. S., Williams, E. D., Chhabra, S., Huynh, N., Fraenkel, E., Marinelli, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A bioinformatic framework and computational approach to discover biological drivers of phenotypic characteristics that reduces multi-omic data dimensionality by generating expression circuits via graph theory constrained by prior biological knowledge of molecular interactions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Unraveling the complex associations between human phenotypes and molecular pathways can pave the way to improved health and performance, but faces a fundamental challenge: the measurable genes, proteins, and metabolites vastly outnumber the participants in even the largest studies, yielding spurious correlations. To address this imbalance, we have developed a bioinformatic framework and computational approach (PhenoMol) to discover biological drivers of phenotypic characteristics that integrates all available phenotypic data predictive of outcomes and reduces multi-omic data dimensionality by generating expression circuits via graph theory constrained by prior biological knowledge of molecular interactions. We applied PhenoMol to analyze causal patterns and predict elite physical performance in a healthy cohort with deep physiological, physical, behavioral, cognitive, and molecular characterization. PhenoMol outperforms regression models based on equivalent analytic methodologies that do not employ network biology for dimensionality reduction. The PhenoMol software is provided for future studies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.24.677381v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ProteinDJ: a high-performance and modular protein design pipeline</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Silke, D., Iskander, J., Pan, J., Thompson, A. P., Papenfuss, A. T., Lucet, I. S., Hardy, J. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> ProteinDJ is a synthetic protein design workflow that is deployable on HPC systems using the Nextflow portable workflow management system and Apptainer containerisation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Leveraging artificial intelligence and deep learning to generate proteins de novo (a.k.a. 'synthetic proteins') has unlocked new frontiers of protein design. Deep learning models trained on protein sequences and experimental structures can generate novel protein designs that explore structural landscapes unseen by evolution. This approach enables the development of bespoke binders that target specific proteins and domains through new protein-protein interactions. However, successful binder generation can suffer from low in silico success rates, often requiring thousands of designs and hundreds of GPU hours to obtain enough hits for experimental testing. While commercial web-apps and workstation implementations are available for binder design, these are limited in both scalability and throughput. There is a lack of efficient open-source protein design pipelines for high-performance computing (HPC) systems that can maximise hardware resources and parallelise the workflow efficiently to generate successful binders. Here, we present 'ProteinDJ' - an implementation of a synthetic protein design workflow that is deployable on HPC systems using the Nextflow portable workflow management system and Apptainer containerisation. It automatically batches and parallelises the workload across both GPUs and CPUs, facilitating generation and testing of hundreds of designs per hour, dramatically accelerating the discovery process. Importantly, it allows central deployment of the workflow on shared HPC resources, rather than workstations restricted to individual research groups. ProteinDJ is designed to be modular and currently includes RoseTTAFold Diffusion (RFdiffusion) for fold generation, ProteinMPNN or Full-Atom MPNN (FAMPNN) for sequence design, and AlphaFold2 or Boltz-2 for prediction and validation of designs and binder-target interfaces, with supporting packages for structural evaluation of designs. We have included structure-based filters to reject problematic designs mid-stream to minimise manual inspection of designs by the user. ProteinDJ democratises protein binder design through its robust and user-friendly implementation and provides a framework for future protein design software pipelines.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.24.678028v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Differential analysis of translation efficiency and usage of open reading frames using DOTSeq</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lim, C. S., Chieng, G. S. W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a differential analysis of translational control events within genes using multiple open reading frames (ORF) and a statistical framework that enables systematic discovery of cis-regulatory events.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein synthesis is a key cellular process in which mRNAs are translated into proteins by ribosomes. This process is tightly regulated, enabling cells to control protein output in response to specific cellular states. Ribosome profiling captures translatomic landscapes across conditions, but existing computational tools for differential translation analysis operate at the gene level, overlooking translational control at the level of multiple open reading frames (ORFs). Here, we present DOTSeq, a Differential ORF Translation statistical framework that enables systematic discovery of translational control events within genes. DOTSeq offers differential analyses of ORF usage and translation efficiency across biological conditions. These modules allow global detection of cis-regulatory events, such as upstream ORF (uORF)-mediated translational control. Benchmarking on simulated datasets demonstrates DOTSeq's sensitivity to subtle regulatory signals, particularly in modest effect sizes with realistic batch effects, where all tools show limited sensitivity. DOTSeq provides a flexible and powerful approach for dissecting the complexity of translational control.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.24.678201v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Convection-enhanced delivery of dexamethasone in glioma suppresses myeloid inflammation while avoiding systemic toxicities</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rolfe, N. W., Dadario, N. B., Lei, L., Teasley, D. E., Amini, M., Chabot, P. J., Ifediora, N., Winans, N. J., Yoh, N., Tang, A., Khoury, N., Furnari, J., Kotidis, C., Stucke, C. H., Urena, N. M., Sun, Y., Upadhyayula, P., Argenziano, M. G., Sperring, C. P., Brand, A., Viswanathan, A., Humala, N., Sims, P. A., Gill, B. J., Canoll, P., Bruce, J. N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that local administration of dexamethasone via convection-enhanced delivery (CED) to maximize intratumoral anti-inflammatory effects by increasing local corticosteroid exposure while minimizing systemic exposure.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Dexamethasone is widely used to control cerebral edema and inflammation in glioblastoma, but its benefits are limited by systemic toxicities and adverse prognostic associations. We evaluated local administration via convection-enhanced delivery (CED) to maximize intratumoral anti-inflammatory effects by increasing local corticosteroid exposure while minimizing systemic exposure. In a syngeneic glioma mouse model, continuous intraparenchymal infusion of dexamethasone was well tolerated and associated with a modest but significant survival benefit versus vehicle. Single-nucleus RNA sequencing (snRNA-seq) and immunohistochemistry showed attenuation of glioma-associated inflammation with downregulation of reactive microglial/macrophage programs and reduced tumor-infiltrating myeloid cells with a morphology consistent with a less activated state. Experiments in human induced pluripotent stem cell (iPSC)-derived microglia confirmed that dexamethasone directly suppresses inflammatory gene expression, indicating a conserved mechanism across species. Pharmacokinetic analyses supported preferential intratumoral distribution and reduced systemic exposure with CED compared with systemic dosing. These findings suggest that localized dexamethasone delivered by CED reprograms the glioma immune microenvironment and achieves steroid-sparing control of inflammation without the systemic adverse effects associated with standard therapy. This clinically translatable strategy may improve symptom management and provide a platform for integrating local immunomodulation with future glioblastoma therapies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.24.677899v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CK2 inhibitor CX-4945 targets EWS-FLI1 protein abundance and shows anti-tumor activity in metastatic mouse models of Ewing Sarcoma</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Daniyal, M., Rajaiah, R., Pandiyan Shanmugam, M., Chloe, S., Jeremey, H., Beeravally Nagulapally, A., Saulnier Sholler, G., Uzun, Y., Valensi, H., Behura, C. G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show cytotoxic activity of single agent CX-4945 in Ewing sarcoma and show decreased lung metastases in mice treated with CX_4945.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Ewing sarcoma (ES) is a highly aggressive bone cancer. EWS-FLI oncogenic fusion protein is detected in more than 85% of cases and is indispensable for ES tumor survival and progression. Casein kinase II (CK2) is a serine/threonine kinase that plays an essential role in apoptosis, DNA damage repair, and the cell cycle. CSNK2A1 is highly expressed in ES and associated with metastatic disease and poor 5-year overall survival. CK2 is shown to phosphorylate and stabilize several transcription factors, including EWS-FLI1. We demonstrate that genetic and pharmacological inhibition of CK2 decreases protein abundance of EWS-FLI1 by increased ubiquitination and proteasomal degradation. Clinical-grade inhibitor of CK2, CX-4945 (silmitasertib), shows in vitro cytotoxic activity in a series of ES tumor organoids and patient-derived xenograft cells. We also demonstrate anti-tumor activity of single agent CX-4945 using metastatic xenograft models of Ewing sarcoma. We also show decreased lung metastases in mice treated with CX-4945. CX-4945 showed synergistic cytotoxic activity with Temozolamide and Irinotecan. CX-4945 is currently being tested in a Phase 1 study to evaluate the safety and tolerability in combination with chemotherapy for the treatment of pediatric colloid tumors, including Ewing sarcoma. The preclinical studies reported here support the clinical studies evaluating the efficacy of CX-4945 for the treatment of Ewing sarcoma.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.24.677357v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Harnessing the CD2 axis to broaden and enhance the efficacy of CAR T cell therapies</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Carturan, A., Angelos, M. G., Guruprasad, P., Patel, R. P., Pajarillo, R., Lee, A., Zhang, Y., Chiang, Y.-H., Xie, W., Rodriguez, J. L., Harris, J., Devi, P., Afolayan-Oloye, O. I., Xu, J., Sussman, J. H., Elghawy, O., Yang, A., Barsouk, A., Cho, J. H., Shaw, C. E., Singh, E., Ugwuanyi, O., Espie, D., Paruzzo, L., Stella, F., Liu, S., Nason, S., Imparato, A., Rotolo, A., Lemoine, J., Barrett, D. M., Posey, A. D., Rook, A. H., Pillai, V., Bagg, A., Pileri, S. A., Liu, D., Tan, K., Schuster, S. J., Teachey, D. T., Porazzi, P., Ruella, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel PD-1:CD2 switch receptor that rescues intracellular CD2 signaling, particularly when PD-L1 is engaged, resulting in improved in vivo outcomes, and a novel CD2:CD58 axis in CAR T-cells.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Patients with T-cell lymphomas and leukemias have overall poor outcomes due to the lack of targeted and effective treatments, particularly in the relapsed and refractory settings. Development of chimeric antigen receptor (CAR) T-cells against T-cell neoplasms is limited by a lack of discriminating T-cell antigens that allow for effective anti-tumor responses while preventing CAR T-cell fratricide. We hypothesized that targeting CD2, a pan-T-cell antigen, using anti-CD2 CAR T-cells engineered without CD2 expression (CART2), would support CAR T-cell manufacturability and preclinical efficacy. Optimized CD2-knockout CART2, generated using CRISPR-Cas9, eradicated primary patient-derived CD2+ hematological neoplasms in vitro and in vivo, secreted effector cytokines, and exhibited adequate proliferative capacity. Nevertheless, CD2 has a key costimulatory function, and its deletion could lead to CAR T-cell dysfunction. Therefore, we tested the role of the CD2:CD58 axis in CAR T-cells, using the anti-CD19 CART models. We demonstrate that CD2 loss attenuates CART19 efficacy by reducing avidity for tumor antigen, co-stimulation, and ultimately in vivo activity. Analogously, we show that tumor CD58 loss reduces CART19 efficacy. To overcome this issue, we developed a novel PD-1:CD2 switch receptor that rescues intracellular CD2 signaling, particularly when PD-L1 is engaged, resulting in improved in vivo outcomes. Collectively, we studied the role of CD2 both as a target for CAR T cell therapy and as a critical costimulatory protein, whose signaling can be rescued using the PD-1:CD2 switch receptor. This receptor can be incorporated into CAR T-cells and provides an effective strategy to overcome CD2-signaling deficiencies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.24.677515v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The impact of normal tissue density on tumor growth and evolution in a 3D whole-tumor model of lung cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bravo, R. R., Robertson-Tessi, M., Antonia, S., Gray, J., Beg, A. A., Gatenby, R. A., Schabath, M. B., Anderson, A. R. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that lung tissue density affects tumor growth and evolution in non-small cell lung cancer and show that tumors spread more rapidly through denser tissues when they upregulate glycolytic pathways and acid production.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Serial low-dose computed tomography (LDCT) scans in patients who are diagnosed with lung cancer during screening offer a history of the densities of tumors and the tissues that surround them during carcinogenesis and cancer progression. We built a CT-scan resolution computational model to explore how variations in lung tissue density impact tumor growth and evolution in non-small cell lung cancer (NSCLC). Our findings indicate that tumors spread more rapidly through denser tissues when they upregulate glycolytic pathways and acid production, whilst tumors spread more rapidly through sparser tissues when they upregulate angiogenesis. We used data and images from the National Lung Screening Trial to calibrate our model for untreated lung cancer growth in patients and corroborated our findings in low-density environments.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.25.676637v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An injury-associated lobular microniche is associated with the classical tumor cell phenotype in pancreatic cancer.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sara Söderqvist, Annika Viljamaa, Natalie Geyer, Anna-Lena Keller, Kseniya Ruksha, Carina Strell, Neda Hekmati, Alexandra Niculae, Jennie Engstrand, Ernesto Sparrelid, Caroline Salmén, Tânia D F Costa, Miao Zhao, Staffan Strömblad, Argyro Zacharouli, Poya Ghorbani, Sara Harrizi, Yousra Hamidi, Olga Khorosjutina, Stefina Milanova, Bernhard Schmierer, Béla Bozóky, Carlos Fernández Moro, Marco Gerling</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Pancreatic cancer is an aggressive disease with a dense fibrotic stroma and is often accompanied by chronic inflammation. We show that the inflamed pancreatic lobules are frequently invaded by tumor cells, forming a distinct, non-fibrotic tumor niche.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Pancreatic cancer is an aggressive disease with a dense fibrotic stroma and is often accompanied by chronic inflammation. Peritumoral inflammation is typically viewed as a reaction to nearby tumor growth. Here, we report that the inflamed pancreatic lobules are frequently invaded by tumor cells, forming a distinct, non-fibrotic tumor niche. Using a semi-supervised machine learning approach for annotations of clinical samples and multiplex protein profiling, we show that tumor cells at the invasion front are closely associated with acinar cells undergoing damage-induced changes, and with activated fibroblasts expressing markers of injury. The invaded lobules are linked to classical tumor phenotypes, in contrast to fibrotic areas where tumor cells display a more basal profile, highlighting microenvironment-dependent tumor subtype differences. In female mice, lobular invasion similarly aligns with the classical tumor phenotype. Together, our data reveal that pancreatic tumors colonize injured lobules, creating a unique niche that shapes tumor characteristics and contributes to disease biology.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41006303/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Evaluating Sex Disparity in Pancreatic Ductal Adenocarcinoma Risk in the UK Biobank Cohort.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Giulia Peduzzi, Roberto Pellungrini, Livia Archibugi, Federico Canzian, Cosmeri Rizzato, Gabriele Capurso, Daniele Campa</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> The incidence of pancreatic ductal adenocarcinoma is lower in females than males (19.0 vs. 20.2 for 100,000 individuals in Europe).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The incidence of pancreatic ductal adenocarcinoma (PDAC) is lower in females than males (19.0 vs. 20.2 for 100,000 individuals in Europe). This disparity is commonly attributed to differences in exposure to lifestyle risk factors such as smoking and alcohol consumption; however, hormonal activity may also play a role.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41004304/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Metabolic remodeling and its hidden heterogeneity in uterine fibroids: comprehensive metabolomic profiling and mass spectrometry imaging.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jialin Liu, Maokun Liao, Jingchao Liu, Shuo Liang, Jin Xie, Dandan Liang, Mingzhao Du, Honghui Shi, Wei Song</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new treatment for uterine fibroids that can lead to infertility, recurrent miscarriage, and obstetric complications.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> As the most common benign gynecological tumor in women, uterine fibroids not only pose a serious threat to reproductive health but also directly impair fertility. The structural abnormalities of the uterus and metabolic disturbances they induce have become critical pathological contributors to infertility, recurrent miscarriage, and obstetric complications in reproductive-aged women. However, the underlying metabolic mechanisms of uterine fibroids remain poorly understood.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41003804/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

