<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Monday, 02 June 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Effect of Chronic Stress on Whole Blood Transcriptome: A Meta-Analysis of Publicly Available Datasets from Rodent Models</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Flandreau, E. I., Nguyen, D. M., Hagenauer, M. H., Nguyen, M., Kim, H., Duan, T., Bader, A., Watson, S., Akil, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We performed a meta-analysis to examine the impact of chronic stress on the whole blood transcriptome and found down-regulation and up-regulation of genes related to B cells, immune response, ribosomal activity, translation, and catabolic cellular processes.</p><p style="font-size: 16px; text-align: justify;">Background: Chronic stress increases risk for neuropsychiatric disorders in humans. By modeling stress-induced changes in animals, we may improve diagnosis or treatment of these disorders. Successful translation benefits from studies with sufficient statistical power and outcome measurements that can be directly compared across species. We performed a meta-analysis to examine the impact of chronic stress on the whole blood transcriptome. Methods: Datasets were systematically identified in Gemma, a database of reprocessed public transcriptional profiling studies; datasets GSE68076, GSE72262, and GSE84185 met inclusion/exclusion parameters. Each study exposed eight-week old mice to chronic stress (5-10 days social defeat stress or 6-8 weeks chronic mild stress). The final sample size was n=92 (n=45 Non-Stress/n=47 Stress). Stress-related differential expression in each dataset was quantified using the Limma pipeline followed by empirical Bayes moderation. For the 9,219 genes represented in all three datasets, we ran a meta-analysis of Log(2) Fold Changes using a random effects model and corrected for false discovery rate (FDR). Functional patterns were assessed with fast Gene Set Enrichment Analysis. Cell type specific enrichment for each of the differentially expressed genes was further explored using a public 10x genomics scRNA-Seq dataset from mouse peripheral blood mononuclear cells. Results: Findings included 23 downregulated and 16 upregulated transcripts in stress-exposed mice (FDR<0.05). Results indicated a down-regulation in gene sets related to B cells, immune response, DNA and chromatin regulation, ribosomal activity, translation, and catabolic cellular processes. Upregulated gene sets related to erythrocytes and oxygen binding. Conclusion: Our results provide molecular insight into stress-related immune dysregulation and add weight to the hypothesis that environmental stress escalates cellular aging, supporting the use of blood transcriptome as a bridge between human and rodent models.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.30.657043v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Non-canonical enhancers control gene expression and cell fate in human pluripotent stem cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cao, Y., Dovgusha, O., Ebrahimi, A., Bevan, S., Widmer, L., Torabi, F., Kurlovich, J., Rodriguez-Polo, I., Khan, S., Patel, M., Pradeepa, M. M., Guenesdogan, U., Schoenfelder, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify a class of gene regulatory elements that had until now only been associated with a neutral or inactive status, challenging current models of enhancer function.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Enhancers are key gene regulatory elements that ensure the precise spatiotemporal execution of developmental gene expression programmes. However recent findings indicate that approaches to identify enhancers may not capture the full repertoire of active enhancers in mammalian genomes. Here, we combine massively parallel enhancer assays with chromatin structure and transcriptome profiling to functionally annotate enhancers genome-wide in human induced pluripotent stem cells. We find that a substantial fraction (~40%) of accessible chromatin regions with enhancer function lack key features associated with active enhancers, including the active enhancer mark histone H3 lysine 27 acetylation and enhancer-associated RNAs. Perturbation of this class of non-canonical enhancers by CRISPR-mediated epigenome editing results in decreased levels of target gene expression and, in one instance, loss of pluripotent stem cell characteristics. Collectively, our data demonstrate enhancer activity for a class of gene regulatory elements that had until now only been associated with a neutral or inactive status, challenging current models of enhancer function.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.01.657118v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Immunomodulatory Biomaterials Enhancing Implant Osseointegration: Knowledge Mapping of Research Evolution from March 2005 to March 2025</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Li, Y., Liu, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify pivotal theoretical breakthroughs and technological advancements in immunomodulatory biomaterials in implant osseointegration, and identify shifting frontiers toward clinical translation (3D printing, precision modulation, and precision modulation).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Objective To delineate the evolutionary trajectory of immunomodulatory biomaterials in implant osseointegration through bibliometric analysis, identifying pivotal theoretical breakthroughs and technological advancements. Methods A total of 419 articles (2005-2025) from the Web of Science Core Collection were analyzed using a multi-tool framework. Current research status and hotspots were evaluated by co-occurrence analysis of keywords and institutions using VOSviewer. The evolution and bursts of the knowledge base were assessed through co-citation analysis of references, authors, and journals via CiteSpace. Thematic evolution and keyword trends were mapped using the bibliometrix package in R. Results The field exhibited "intermittent-explosive" growth (32.7% annual increment), with China leading global contributions (69.4%). The osteoimmunomodulation (OIM) theory emerged as the cornerstone, emphasizing spatiotemporal macrophage polarization (M1/M2 balance) and multi-signal crosstalk (BMP-2/VEGF/OSM). Key technological pathways included: 1. Surface engineering (nanotopography, ion-doped coatings); 2. Smart materials (3D-printed scaffolds, pH/ROS-responsive carriers); 3. Antibacterial-immunomodulatory synergy. Burst detection revealed shifting frontiers toward clinical translation (2023-2025 burst: "3D printing", strength=4.05) and precision modulation ("macrophage polarization", strength=9.02). Conclusion Immunomodulatory biomaterials are transitioning from mechanistic exploration to clinical adaptation. Future development requires integrating dynamic microenvironment-responsive designs with multi-omics validation to address macrophage heterogeneity, ultimately enabling personalized osseointegration therapies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.29.656932v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A network medicine framework for multi-modal data integration in therapeutic target discovery</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Baltusyte, G., Toleman, I., Jones, J., Saeb-Parsy, K., Han, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Network medicine-based machine learning framework that integrates single-cell transcriptomics, bulk multi-omic profiles, genome-wide CRISPR perturbation screens, and protein-protein interaction networks to systematically prioritise disease-specific targets.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The high cost and attrition rate of drug development underscore the need for more effective strategies for therapeutic target discovery. Here, we present a network medicine-based machine learning framework that integrates single-cell transcriptomics, bulk multi-omic profiles, genome-wide CRISPR perturbation screens, and protein-protein interaction networks to systematically prioritise disease-specific targets. Applied to clear cell renal cell carcinoma, the framework successfully recovered established targets and predicted five therapeutic candidates, with subsequent in vitro validation demonstrating that among these, ENO2 inhibition had the strongest anti-tumour effect, followed by LRRK2, a repurposing candidate with phase III Parkinson's disease inhibitors. The proposed approach advances target discovery by moving beyond single-feature, single-modality heuristics to a scalable, machine learning-driven strategy that is generalisable across diseases.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.29.656813v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Automated registration of spatial expression data scales multimodal integration to large cohorts</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Harrigan, C. F., Lam, C. Y., Chen, D., Lai, C., Bremner, R. F., Jackson, H. W., Campbell, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce the largest dataset to date of cross-modality imaging of both cell line and tissue slides suitable for benchmarking registration methods and introduce Twocan, a Bayesian optimization framework that enables robust automated registration between immunofluorescence imaging and highly multiplex spatial proteomics data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent advances in spatial proteomics enable quantification of the spatial distribution of protein expression across a variety of scales, resolutions, and multiplexing. Registering images from such technologies across modalities is an essential task that enables both the integration of complementary imaging technologies and validation of biological findings. This can be particularly challenging when the modalities capture fundamentally different types of data, such as light intensity, probe counts, or heavy metal counts. However, few datasets and methods address this problem at scale. Here, we introduce the largest dataset to date of cross-modality imaging of both cell line and tissue slides suitable for benchmarking registration methods. We further present Twocan, a Bayesian optimization framework that enables robust automated registration between immunofluorescence imaging and highly multiplex spatial proteomics data. Our method achieves significantly higher registration success rates compared to existing approaches across our comprehensive dataset of 954 image pairs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.29.656851v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Temporal Single-Cell Multi-Omics Atlas of Murine Pancreatic Islet Remodeling During Hyperglycaemia Progression</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Singh, S., Pavan, M. K., Barella, L. F., Telang, J., Shree, A., Agarwal, S., Goel, A., Thaleshwar, S. i., Wess, J., Zafar, H., Pydi, S. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comprehensive high-resolution atlas of islet remodeling under metabolic stress, identifying key communication nodes and transcriptional programs pertinent to T2D pathogenesis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Pancreatic islets undergo coordinated cellular remodeling during obesity-induced insulin resistance (IR). However, the associated molecular changes across endocrine and non-endocrine compartments remain largely unexplored. Here, using longitudinal single-cell RNA sequencing (scRNA-seq) and single-cell ATAC sequencing (scATAC-seq) on islets from C57BL/6 mice subjected to high-fat diet (HFD) feeding for 8, 16, and 24 weeks, along with age-matched controls on regular chow, we mapped dynamic changes in islet cell composition and transcriptional states. Beta cells demonstrated pronounced stress-induced reprogramming, with the emergence of proliferative and dysfunctional subsets. Alpha and delta cell fractions declined under HFD, despite increased polyhormonal biosynthesis, suggesting functional rather than numerical adaptation. Immune profiling showed robust expansion of proinflammatory M1 macrophages and upregulation of NF-{kappa}B and chemotaxis pathways, particularly at 16 weeks. Notably, cell-cell communication analyses revealed diet-specific disruption in signaling networks. Under HFD conditions, intercellular communication among beta cells, macrophages, and delta cells was markedly altered, leading to the disruption of key signaling pathways such as the gastric inhibitory polypeptide receptor (GIPR) and major histocompatibility complex-I (MHC-I). Notably, C-C motif chemokine ligand 27A (Ccl27a) expression and chromatin accessibility were significantly altered in a distinct subpopulation of beta cells under HFD condition, indicative of a niche-specific regulatory mechanism. Integration with human islet datasets from obese and type 2 diabetes (T2D) donors confirmed conserved shifts in beta cell identity and immune activation. This study presents a comprehensive high-resolution atlas of islet remodeling under metabolic stress, identifying key communication nodes and transcriptional programs pertinent to T2D pathogenesis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.29.656754v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">EXTENDING PROTEIN LANGUAGE MODELS TO A VIRAL GENOMIC SCALE USING BIOLOGICALLY INDUCED SPARSE ATTENTION</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dejean, T. L., Ferrell, B. D., Sawhney, R., Harrigan, W., Polson, S. W., Wommack, K. E., Belcaid, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a long-context protein language model, trained on entire viral genomes, leveraging a sparse attention mechanism based on protein-protein interactions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The transformer architecture in deep learning has revolutionized protein sequence analysis. Recent advancements in protein language models have paved the way for significant progress across various domains, including protein function and structure prediction, multiple sequence alignments and mutation effect prediction. A protein language model is commonly trained on individual proteins, ignoring the interdependencies between sequences within a genome. However, biological understanding reveals that protein-protein interactions span entire genomic regions, underscoring the limitations of focusing solely on individual proteins. To address these limitations, we propose a novel approach that extends the context size of transformer models across the entire viral genome. By training on large genomic fragments, our method captures long-range interprotein interactions and encodes protein sequences with integrated information from distant proteins within the same genome, offering substantial benefits in various tasks. Viruses, with their densely packed genomes, minimal intergenic regions, and protein annotation challenges, are ideal candidates for genome-wide learning. We introduce a long-context protein language model, trained on entire viral genomes, leveraging a sparse attention mechanism based on protein-protein interactions. Our semi-supervised approach supports long sequences of up to 61,000 amino acids (aa). Our evaluations demonstrate that the resulting embeddings significantly surpass those generated by single-protein models and outperform alternative large-context architectures that rely on static masking or non-transformer frameworks.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.29.656907v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Trainable subnetworks reveal insights into structure knowledge organization in protein language models</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Vinod, R., Amini, A. P., Crawford, L., Yang, K. K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A framework for studying feature entanglement within pretrained protein language models and how structural factorization in PLM representations influences downstream structure prediction.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein language models (PLMs) pretrained via a masked language modeling objective have proven effective across a range of structure-related tasks, including high-resolution structure prediction. However, it remains unclear to what extent these models factorize protein structural categories among their learned parameters. In this work, we introduce trainable subnetworks, which mask out the PLM weights responsible for language modeling performance on a structural category of proteins. We systematically train 36 PLM subnetworks targeting both sequence- and residue-level features at varying degrees of resolution using annotations defined by the CATH taxonomy and secondary structure elements. Using these PLM subnetworks, we assess how structural factorization in PLMs influences downstream structure prediction. Our results show that PLMs are highly sensitive to sequence-level features and can predominantly disentangle extremely coarse or fine-grained information. Furthermore, we observe that structure prediction is highly responsive to factorized PLM representations and that small changes in language modeling performance can significantly impair PLM-based structure prediction capabilities. Our work presents a framework for studying feature entanglement within pretrained PLMs and can be leveraged to improve the alignment of learned PLM representations with known biological concepts.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.29.656902v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Cell Type Specific Aging Transcriptional Signatures of Human Retina Through Integrated Machine Learning and Single-Cell Transcriptomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yang, L., Lin, S., Tao, Y., Pan, Q., Cai, T., Ye, Y., Liu, J., Zhou, Y., Shao, Y., Yi, Q., Lu, Z. H., Chen, L., McKay, G., Rankin, R., Li, F., Meng, W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified age-discriminatory genes within each major retinal cell type and observed aging-associated shifts across the adult lifespan in a Chinese retinal population.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell RNA sequencing (scRNA-seq) has significantly advanced our understanding of retinal aging, yet the specific molecular characteristics within cell populations remain incompletely defined. We profiled 223,612 single cells from 18 unfrozen human retinas obtained from 13 Chinese donors aged 34-92 years, providing an ethnically specific atlas across the adult lifespan. A sparsity-driven machine-learning (ML) pipeline (L1-regularized logistic regression, recursive feature elimination with cross-validation) identified age-discriminatory genes within each major retinal cell type, complemented by gene set scoring for cellular senescence and metabolic pathways. Using integrated differential expression and ML feature selection, we identified eleven major retinal cell populations and observed aging-associated shifts. ML classifiers achieved high accuracy (80-96%), particularly for microglia (96%), revealing mitochondria-centric aging signatures in rods and bipolar cells, proteostasis and retinoid metabolism in cones, and structural-RNA maintenance signatures in horizontal cells. This study delivers the first ML-derived, cell-type-specific aging gene signatures for the human retina in a Chinese cohort, offering a reference for population-tailored biomarker discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.29.656930v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">EnrichMap: Spatially informed enrichment analysis for functional interpretation of spatial transcriptomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Celik, C., Secrier, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present EnrichMap, a Python package designed to compute gene set enrichment scores in spatial transcriptomics data by integrating batch correction, smoothing and spatial confounder adjustment.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Advances in spatial transcriptomics (ST) allow high-resolution mapping of gene expression within intact tissues, revealing spatially regulated biology. However, accurately quantifying gene set activity in spatial contexts remains inadequate. We present EnrichMap, a Python package designed to compute gene set enrichment scores in ST data by integrating batch correction, smoothing and spatial confounder adjustment. EnrichMap generates enrichment scores for signatures or pathways, with an optional framework for incorporating gene weights. To assess performance, we benchmark EnrichMap against established scoring techniques using spatial autocorrelation metrics such as Moran's I. Our results show that EnrichMap improves spatial coherence, producing scores that capture both local and global spatial patterns, while conventional approaches tend to exhibit higher heterogeneity or diminished spatial structure. EnrichMap is demonstrated to be generalisable across a variety of ST platforms, including Visium, Xenium, MERFISH and imaging mass cytometry, maintaining consistent performance despite differences in spatial resolution and detection chemistry. Our method offers a platform-agnostic, spatially and biologically meaningful representation of gene set activity across complex tissue landscapes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.30.656960v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">GNC is a regulator of metabolic and productivity responses to elevated CO2 in Arabidopsis thaliana</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Quebedeaux, J. C., Kannan, K., Marshall-Colon, A., Leakey, A. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A CO2-responsive transcription factor that may be a compelling target for adapting crops to future growing conditions after further characterization.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Despite established understanding of plant physiological responses to elevated [CO2], the underlying genes are poorly understood. Soybean transcriptomics previously identified a GATA transcription factor, involved in carbon and nitrogen metabolism, as responsive to elevated [CO2]. Supported by in silico modeling, we therefore hypothesized that this gene plays a previously unrecognized role in responding to elevated [CO2]. Wildtype and a T-DNA insertion line of Arabidopsis thaliana for GNC (GATA, Nitrate Inducible, Carbon Metabolism Involved) were grown under three treatments: sustained ambient [CO2], sustained elevated [CO2], and transfer from ambient to elevated [CO2], to assess changes in their physiology, biochemistry, and transcriptome. Photosynthetic and biomass responses to elevated [CO2] and transfer [CO2] in plants lacking GNC were significantly weaker than WT. A lag of 25-73 hrs in transcriptomic responses after transfer to elevated [CO2] was consistent with indirect sensing, presumably via sugar signals. The breakdown of the gene expression network around GNC was most pronounced in the transfer treatment and suggests targets for further study of interactions between elevated [CO2] and sulfur and nitrogen metabolism. This work provides a case study of a CO2-responsive transcription factor that may be a compelling target for adapting crops to future growing conditions after further characterization.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.29.656866v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Voltage-dependent anion channels are mitophagy receptors mediating the recycling of depolarized mitochondria in Arabidopsis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> MA, W., MA, J., ZHANG, K., ZHENG, X., WANG, P., FENG, L., MING, S., ZHUANG, X., ZHOU, J., Gao, C., KANG, B. H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified a unique mitophagy mechanism in plant, that three voltage-dependent anion channel (VDAC) family proteins in the mitochondria outer membrane - specifically VDAC1, VDac2, and VD AC3 - as mitophapy receptors in Arabidopsis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The mitochondrion is an essential organelle in eukaryotic cells, playing crucial roles in cellular respiration and intracellular signaling pathways. To maintain a healthy population of mitochondria, dysfunctional and excess mitochondria are selectively removed through an autophagic process known as mitophagy. Over the past few decades, various autophagy-related (ATG) proteins involved in mitophagy have been well characterized in yeast and mammalian cells since it has significance to the survival of eukaryotes. While the core autophagy machinery responsible for autophagosome formation is conserved among eukaryotes, the homologs of key regulators of mammalian system is absent in plant. In this study, we identified a unique mitophagy mechanism in plant, that three voltage-dependent anion channel (VDAC) family proteins in the mitochondria outer membrane - specifically VDAC1, VDAC2, and VDAC3 - as mitophagy receptors in Arabidopsis. These proteins were required for translocation of ATG8 from the cytosol to the mitochondria surface, when Arabidopsis cells were treated with an uncoupler, 2,4-dinitrophenol (DNP). The VDACs interacted directly with ATG8 through an ATG8-interacting motif (AIM) located in their amino (N) termini. Furthermore, vdac mutants exhibited impaired uncoupler-induced mitophagy and accumulated damaged mitochondria. These mitophagy-related phenotypes were more pronounced in vdac double and triple mutant lines. Altogether, our results indicated that VDAC1, 2, and 3 recruit ATG8 to depolarized mitochondria, facilitating the formation of mitophagosomes, presenting a distinguishing mitophagy pathway with mammalian system.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.29.656919v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Systematic measurements of dose-dependent responses for combinatorial treatments of SA and JA led to the development of transcriptomic biomarkers</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tomita, A., Maeda, T., Mori-Moriyama, N., Nomura, Y., Kurita, Y., Kashima, M., Betsuyaku, S., Nagano, A. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show transcriptional responses specific to combinations of SA and JA concentrations and develop transcriptomic biomarkers to estimate SA/JA response states.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Salicylic acid (SA) and jasmonic acid (JA) regulate plant defense against pathogens and herbivores. The SA-JA pathways often interact antagonistically with the induction of one, leading to the suppression of the other. However, the effect of combined SA/JA concentrations on plant responses is unknown. Here, we show transcriptional responses specific to combinations of SA and JA concentrations and develop transcriptomic biomarkers to estimate SA/JA response states. Combinatorial treatments with eight concentrations of SA and JA, combined with large-scale transcriptome analysis, revealed 43 different expression patterns, including not only well-known antagonistic responses but also responses specific to combinations of SA/JA concentrations. Notably, the expression of CYP94B3, a key enzyme involved in JA-Ile turnover, was finely tuned by specific SA/JA concentration combinations being induced under high SA conditions in a JA dose-dependent manner. In addition, the expression peaks of indole glucosinolate biosynthesis genes were found at approximately 30-fold higher JA concentrations than aliphatic glucosinolate biosynthesis genes. Furthermore, to quantitatively estimate SA/JA response states, we developed transcriptomic biomarkers using a machine learning approach. The marker specifically estimated response states to target phytohormones. Through analysis of npr3/4 double mutants, we confirmed that the biomarkers were applicable to the analysis of mutants. The biomarkers will allow us to extract insights into phytohormone responses from various large-scale transcriptome data, e.g., field transcriptomics and single-cell RNA-Seq.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.29.656841v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Developmentally-specific physiological and metabolic responses support drought resilience in switchgrass and constrains biofuel yield</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Basyal, B., Li, X., Pargulski, V., Fu, X., Nightingale, N., Overmyer, K., Coon, J., Zhang, Y., Benucci, G. M. N., Last, R., Sato, T., Walker, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We study the developmental stage-specific physiological and metabolic responses of switchgrass to drought stress and its implications for biofuel production using a custom-built programmable irrigation system.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Switchgrass (Panicum virgatum) is a promising bioenergy crop due in part to its resilience to drought stress. However, the significance of drought timing remains poorly understood, both from a plant biology perspective and its impact on downstream biofuel production. This study determines the developmental stage-specific physiological and metabolic responses of switchgrass to drought stress and its implications for biofuel production using a custom-built programmable irrigation system. Vegetative, flowering, and senescence-stage drought significantly reduced CO2 assimilation, and stomatal conductance without affecting biomass yield. Metabolic profiling revealed significant accumulation of glucose, fructose, quinic acid, shikimate and GABA during vegetative-stage drought, while flowering and senescence stages exhibited limited metabolic changes. Similarly, specialized metabolites also displayed distinct developmental patterns, with vegetative-stage drought driving the most pronounced metabolic alterations. Thermochemically-treated and hydrolyzed switchgrass biomass from vegetative-stage drought showed elevated lignocellulose-derived compounds and saponins with the latter most positively correlating with fermentation lag times. Conversely, senescence-stage drought enhanced ethanol yields while lowering saponin levels in the hydrolysates. While vegetative-stage drought enhanced physiological resilience, it compromises downstream biofuel production by introducing fermentation inhibitors, particularly saponins.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.29.656847v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">TSSpredator-Web: A web-application for transcription start site prediction and exploration</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Witte Paz, M., Herbig, A., Nieselt, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce TSSpredator-Web; an interactive web application that enhances the usability of the established yet unpublished TSS prediction tool, TS Spredator, by combining user-friendly accessibility with interactive data exploration.</p><p style="font-size: 16px; text-align: justify;">Background: With the rapid development of high-throughput RNA-seq technologies, the transcriptome of prokaryotes can now be studied in unprecedented detail. Transcription start site (TSS) identification provides critical insights into transcriptional regulation, yet current command-line tools for the prediction of TSS remain challenging with respect to their usability and lack of integrated exploration features. Results: We introduce TSSpredator-Web; an interactive web application that enhances the usability of the established yet unpublished tool, TSSpredator. TSSpredator-Web facilitates TSS prediction from non-enriched and enriched RNA-seq data, classifies TSS relative to annotated genes, and allows users to explore results through dynamic visualizations and interactive tables. For the visualizations we provide an UpSet plot summarizing TSS distribution across experiments or classes, and a genome viewer that integrates transcriptomic and genomic data that contextualizes the insights of the TSS predictions. To illustrate the usage of TSSpredator-Web, we provide a use case with Cappable-seq data from Escherichia coli. TSSpredator-Web is available at the TueVis visualization web-server at https://tsspredator-tuevis.cs.uni-tuebingen.de/. Conclusions: By combining user-friendly accessibility with interactive data exploration, TSSpredator-Web significantly facilitates genome-wide TSS analysis and interpretation in prokaryotes, empowering a broader range of researchers to generate biological insights from transcriptomic data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.29.656934v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Plastimos: a live cell imaging-based framework to study dynamics of EMT-mediated cellular plasticity in breast cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hosseini, H., Sameri, S., Das, D., Materna-Reichelt, S., Stojanovic Guzvic, N., WoÃàhrl, L., Hoffmann, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We study the EMT-mediated plasticity state of cancer cells in response to the well-known EGF and TGF-{beta}1.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cancer cell plasticity, primarily mediated by the process of epithelial-mesenchymal transition (EMT), plays a critical role in promoting therapeutic resistance, tumor heterogeneity, and metastasis. EMT-mediated plasticity enables cancer cells to undergo molecular and phenotypic changes, which alter their invasiveness and resistance to treatments. This study aims to develop a quantitative, high-throughput system to assess EMT-mediated plasticity, which can inform therapeutic strategies for metastatic cancers. To accomplish this, we developed Plastimos, a semi-automated imaging analysis pipeline. This framework utilizes time-series images to track live cells through deep learning-based segmentation and employs a greedy algorithm to map cell trajectories, enabling the extraction of cellular phenotypic features. These features are used to study the EMT-mediated plasticity state of cells in response to the well-known EMT-inducing factors EGF and TGF-{beta}1. We selected two breast cancer cell lines, MCF7 and MDA-MB-231, representing classical epithelial-like and mesenchymal-like cell. The pipeline assigns a Plasticity Index based on various parameters, including motility, morphology, and proliferation, thus providing a quantitative estimate for the plasticity of deviating from the epithelial state. Our results indicate that epithelial-like cells respond to EMT-inducing factors at both molecular and phenotypic levels, while mesenchymal-like cells' responses are only seen phenotypically. The Plasticity Index classifies cells along the EMT spectrum, converting to a Plasticity Score that reflects mesenchymal proportions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.28.656591v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">KODAMA enables self-guided weakly supervised learning in spatial transcriptomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Abdel-Shafy, E. A., Kassim, M., Vignoli, A., Mamdouh, F., Tyekucheva, S., Ahmed, D., Ojo, D., Price, B., Socciarelli, F., Duarte-Delgado, N. P., Ocharo, M., Okeke, C. J., Triboli, L., MacIntyre, D. A., Loda, M., Sarkar, D., Gupta, D., Piazza, S., Zerbini, L. F., Tenori, L., Cacciatore, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce the third version of the KODAMA algorithm, specifically tailored for spatial transcriptomics analysis, which effectively reduces data dimensionality while preserving spatial information.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Spatial transcriptomics provides researchers with a powerful tool to investigate gene expression patterns within their native positional context in tissues, revealing intricate cellular relationships. However, analyzing spatial transcriptomics data presents unique challenges due to their high dimensionality and complexity. Several competitive tools have emerged, each aiming to integrate spatial dependencies into their workflow. Yet, the overall performance of these approaches remains constrained by limitations in prediction accuracy and compatibility with specific data types. Here, we introduce the third version of the KODAMA algorithm, specifically tailored for spatial transcriptomics analysis. This upgraded version incorporates a novel approach that effectively reduces data dimensionality while preserving spatial information. At its core, the KODAMA algorithm employs parallel iterations that enforce spatial constraints throughout an embedded clustering process. Our method provides the flexibility to simultaneously analyze multiple samples, streamlining analysis workflows. Additionally, it extends beyond traditional 2D analysis to accommodate multidimensional datasets, including 3D spatial information. KODAMA seamlessly integrates into various pipelines, such as Seurat and Giotto. Extensive evaluations of KODAMA on 10x Visium, Visium HD, and image-based 3D datasets demonstrate its high accuracy in spatial domain prediction. Comparative analyses against alternative dimensionality reduction techniques and spatial analysis tools consistently validate and highlight KODAMA's superior performance in unraveling the spatial organization of cellular components across both single and integrated tissue samples.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.28.656544v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

