<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Saturday, 19 July 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">FAMSA2 enables accurate multiple sequence alignment at protein-universe scale</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gudys, A., Zielezinski, A., Notredame, C., Deorowicz, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce FAMSA2, an algorithm that produces high-accuracy multiple protein sequence alignments with unprecedented speed with state-of-the-art accuracy.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We introduce FAMSA2, an algorithm that produces high-accuracy multiple protein sequence alignments with unprecedented speed. Across structural, phylogenetic, and functional benchmarks, FAMSA2 matches or exceeds the accuracy of state-of-the-art tools while running up to 400 times faster and outperforming the fastest existing aligners. The ability of FAMSA2 to process 12 million sequences in 40 minutes on a 64 GB RAM workstation enables evolutionary and structural analyses at previously unattainable scales.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.15.664876v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Oral microbial signatures of head and neck cancer patients highlight diverse longitudinal patterns of oral mucositis severity</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kodikara, S., Mao, J., San Valentin, E. M. D., Do, K.-A., Reyes-Gibby, C., Le Cao, K.-A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify clinically relevant patient clusters based on oral mucositis severity trajectories; identify microbial signatures specific to these clusters; and identify cluster-specific microbial signatures.</p><p style="font-size: 16px; text-align: justify;">Background: Oral mucositis is a painful complication commonly observed in head and neck cancer patients receiving cancer treatment. Emerging evidence suggests that changes in the oral microbiome can contribute to oral mucositis development, making microbial signatures potential targets for therapeutic interventions. This study aimed to: (1) characterize longitudinal microbial patterns of oral mucositis severity among head and neck cancer patients; (2) determine clinically relevant patient clusters based on oral mucositis severity trajectories; and (3) identify microbial signatures specific to these clusters. Results: We derived a calibrated oral mucositis score by applying non-negative sparse principal component analysis to seven oral mucositis related symptom ratings, using longitudinal microbiome data from 140 head and neck cancer patients. Functional data analysis and hierarchical clustering identified three distinct patient clusters with differing microbial trajectories of oral mucositis progression. One cluster exhibited patients with a rapid increase in oral mucositis severity following treatment initiation, while the other clusters displayed more gradual increase. Demographic comparisons revealed significant differences in age and weight distributions between clusters, with older, lighter patients more common in clusters experiencing more gradual oral mucositis progression. Partial least squares knockoff analysis identified cluster-specific microbial signatures: notably, Prevotella spp. positively associated with calibrated oral mucositis score across all clusters, while Alloprevotella} was significantly enriched only in patients experiencing rapid oral mucositis progression. Conversely, genera associated with oral health, including Haemophilus, Rothia, and Actinomyces, were negatively correlated with calibrated oral mucositis score. Conclusions: Distinct trajectories of oral mucositis scores in head and neck cancer patients are linked to specific oral microbial profiles and demographic factors. The identification of cluster-specific microbial profiles highlights the potential for microbiome-targeted interventions to manage oral mucositis severity. While most taxa were cluster-specific, Prevotella consistently ranked among the top taxa positively associated with the calibrated oral mucositis score across clusters, suggesting it may not differentiate between patient groups but rather reflects overall disease severity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.15.665024v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">TaxTriage: An Open-Source Metagenomic Sequencing Data Analysis Pipeline Enabling Putative Pathogen Detection</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Merritt, B., Ratcliff, J., Ta, S., Osis, G., Mauldin, M., Thielen, P. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> TaxTriage is a comprehensive pathogen identification workflow designed for both short- and long-read untargeted DNA and RNA sequencing data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: TaxTriage is a comprehensive pathogen identification workflow designed for both short- and long-read untargeted DNA and RNA sequencing data. Combining read classification, mapping, and de novo assembly approaches, putative pathogens are identified through comparisons to curated pathogens and abundance expectations from healthy cohort data. Flexible installation options are enabled using Nextflow (NF), including cloud deployment via NF Tower (Seqera Platform) and local installation on a variety of systems, including standalone installations without external internet access. Final analysis summaries are compiled into an Organism Discovery Report, which lists likely pathogens and supporting data, including a custom confidence score. Results: Evaluation of published in silico, clinical, and outbreak datasets identified performance comparable to alternative cloud-based processing pipelines for expected pathogen and co-infection detection with similar sensitivity and increased specificity. To support both public health and veterinary diagnostics communities, customization options have been incorporated to enable improved performance for host species of interest. Availability and Implementation: Source code for TaxTriage is freely available at https://github.com/jhuapl-bio/taxtriage.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.16.664785v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Detecting Foldback Artifacts in Long Reads</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Heinz, J. M., Meyerson, M., Li, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed the open-source Breakinator tool to flag putative foldback artifact reads, as well as previously known chimeric artifacts in long-read data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Long-read sequencing data is useful for detecting large and complex structural variations; however, technical artifacts can lead to false structural variant calls. In our analyses, we became aware of a foldback artifact in long-read data. Therefore, we developed the open-source Breakinator tool to flag putative foldback artifact reads, as well as previously known chimeric artifacts. Through an alignment-based approach, Breakinator can detect artifacts missed by existing quality control tools. We profiled the occurrences of foldbacks and chimeric reads in both nanopore and single-molecule real-time sequences across a range of specimens, library types, sequencing chemistries, sequencing machines, and base-calling software.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.15.664946v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An atlas of positive selection in the genomes of major malaria vectors</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nagi, S. C., Harding, N. J., Lawniczak, M. K. N., Donnelly, M. J., Miles, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a comprehensive atlas of recent positive selection in natural populations of the major African malaria vectors; the An. gambiae species complex and An. funestus.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Insecticide resistance poses a major threat to malaria control efforts in Africa, yet our understanding of the genomic basis of adaptation in malaria vectors remains incomplete. Here we present a comprehensive atlas of recent positive selection in natural populations of the major African malaria vectors; the An. gambiae species complex and An. funestus. By analyzing whole-genome sequence data from 4,306 mosquitoes collected across twenty-two countries, we identify and characterize both known and novel genomic regions under strong positive selection. We develop an innovative approach for identifying and localising selection signals, and find extensive evidence for parallel evolution between species, with all important loci in An. funestus also under selection in An. gambiae s.l. We confirm intense selection at established resistance loci including known target site proteins (Vgsc, Rdl, Ace1) and metabolic enzymes (Cyp6p, Cyp9k1, Gste, Coeaexf) and describe novel signals at a diacylglycerol kinase on the X chromosome, and in An. gambiae s.l, at the metabolic regulator Keap1. To support real-time monitoring of emerging variants threatening vector control, we present an open-source web resource containing detailed selection scan results for all cohorts. The resource is driven by an automated computational workflow, which enables continuous updates as new whole-genome data are generated. The selection atlas provides a foundation for understanding vector adaptation to insecticide-based interventions and we hope will inform evidence-based resistance management strategies that are critical for sustaining malaria control.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.16.664900v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Quality control of single-cell ATAC-seq data without peak calling using Chromap</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ahmed, O., Zhang, H., Langmead, B., Song, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We extend Chromap, an ultrafast method for single-cell ATAC-seq data alignment, to directly report peak-based quality control (QC) metrics, such as the fraction of reads in peaks, without calling peaks.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In this work, we extend Chromap, an ultrafast method for single-cell ATAC-seq data alignment, to directly report peak-based quality control (QC) metrics, such as the fraction of reads in peaks, without calling peaks. Recent single-cell ATAC-seq analysis methods like SnapATAC2 utilize the genome-interval-based feature for data analysis, which disables filtering low-quality cells using common peak-based QC metrics. We show that Chromap's QC metrics capture additional low-quality cells missed by SnapATAC2 and improve downstream analysis results without sacrificing computational efficiency.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.15.664951v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Ensemble-based genomic prediction for maize flowering time reveals novel insights into trait genetic architecture and improves prediction for breeding applications</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tomura, S., Powell, O. M., Wilkinson, M. J., Cooper, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> An ensemble-based approach that combines multiple genomic prediction models for genetic gain .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> While many genomic prediction models have been evaluated for their potential to accelerate genetic gain for multiple traits, no individual genomic prediction model has outperformed others across all applications. This problem aligns with the implications of the No Free Lunch Theorem, stating that the average performance of individual prediction models becomes equivalent across the state space of diverse prediction scenarios. Ensembles of multiple individual genomic prediction models can be a potential alternative approach. The framework of the Diversity Prediction Theorem suggests the potential for a reduction in prediction error with the inclusion of diverse prediction models in the ensemble. We investigated the performance of an ensemble approach that combines multiple genomic prediction models. We demonstrate the results using flowering time traits measured in two maize Nested Association Mapping datasets. For both datasets, the ensemble-based prediction approach achieved the highest prediction accuracy and lowest prediction error across traits. Multiple genomic regions containing key flowering time-related genes were captured by the different genomic prediction models with diverse weights, demonstrating different views of the trait genetic architecture. The combination of such diverse views contributed to the improvement of prediction performance by the ensemble-based approach over the individual prediction models. Exploiting the expectations of the Diversity Prediction Theorem, the ensemble can overcome some limitations proposed by the No Free Lunch Theorem when applying individual genomic prediction models.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.15.664852v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identifying intervention strategies from machine learning models with COALA: a counterfactual optimization framework</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Han, B., Duan, Q., Hu, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A framework for interpretable interpretable models in biomedicine by analyzing counterfactual profiles of features rather than individual features, which reveals holistic patterns.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: Machine learning models in biomedicine have become increasingly complex, often functioning as black boxes. However, understanding contributors to disease and making actionable health interventions requires interpretable models. Common explainable AI methods like SHAP focus on feature importance but fall short in explaining why features contribute in certain patterns or what interventions to take. Counterfactual explanations address this by proposing "what if" scenarios but current tools focus on individual predictions and fail to generalize complex trends. Results: We introduce the framework Counterfactual Optimization for Actionable interpretabiLity in AI (COALA). COALA interprets models by identifying optimal counterfactuals across user-defined mutable feature subsets and constraining remaining features to reveal how constraint features determine what interventions are optimal. By analyzing counterfactual profiles of features rather than individual features, COALA reveals holistic patterns. Using synthetic and real datasets, COALA reveals simple and complex model trends and provides more intuitive, multi-feature interventions than SHAP. Availability and Implementation: Code for COALA implementation, synthetic data, models trained on synthetic data, and code to replicate results and figures are available at https://github.com/brt-solo/COALA.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.18.664723v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identifying the Minimal Number of Protein Markers for Cell Type Annotation Using MiniMarS</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Parikh, D., Xue, A., Liao, H.-C., Wishart, C., Ashhurst, T. M., Putri, G., Luciani, F., Naik, S., Salim, A., Marsh-Wakefield, F., Louie, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present MiniMarS (Minimal Marker Selection), an R package that serves as a wrapper for 10 different algorithms for identifying the optimal number of markers that will delineate cell populations in their dataset.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Over the past decade, there has been an explosion in the characterisation and discovery of cell populations using single-cell technologies. Single-cell multi-omics data, particularly those incorporating gene and protein expression, are increasingly commonplace and can lead to more refined characterisation of cell types. A common challenge for biologists is to isolate cells of interest using a minimal number of markers for cytometry experiments. Although several methods exist for marker selection, there is limited guidance on the relative performance of these methods, and a wrapper package that combines multiple methods is lacking. The method that performs best can vary depending on the dataset and it can be challenging for researchers to test multiple methods for a given dataset. To address these issues, we present MiniMarS (Minimal Marker Selection), an R package that serves as a wrapper for 10 different algorithms. It allows users to determine the best-performing algorithm for identifying the optimal number of markers that will delineate cell populations in their dataset. MiniMarS uses pre-annotated cells with protein features from CyTOF or sequencing-based assays such as CITE-seq and Abseq as input. Outputs include 1) the minimum number of protein markers required to identify the annotated cell populations using a range of marker selection algorithms, and 2) a range of metrics to evaluate the performance of each algorithm. MiniMarS effectively differentiated populations across various datasets, including those from human blood, bone marrow, thymus, mouse spleen, and lymph nodes, even after subsampling over 41,000 cells to 2,500 cells. MiniMarS also identified 15 markers from CITE-seq data, which were then used to successfully identify the same 11 cell subsets in a CyTOF dataset (F1 score>0.9). Additionally, we showed that by appropriately combining clusters, MiniMarS improves the F1 score of a rare population identification (<1% of total cells) by 28.7%. Together, these findings highlight the broad applicability of MiniMarS in identifying appropriate markers for distinguishing cell populations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.14.664825v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Exploring the diversity of the CO2-concentrating mechanism (CCM) in different C4 subtypes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Baccolini, C., Ishihara, H., Feil, R., Perez de Souza, L., Alseekh, S., Fernie, A. R., Stitt, M., Lunn, J. E., Arrivault, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We study the relationship between malate and aspartate in C4 pathway fluxes in four C4 grass species and demonstrate the feasibility of using non-radioactive 13CO2 in pulse-chase labelling experiments to study C4 photosynthesis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> C4 plants have traditionally been classified into NADP-malic enzyme (NADP-ME), NAD-malic enzyme (NAD-ME) and PEP carboxykinase (PEPCK) subtypes based on the predominant C4-acid decarboxylating enzyme. To investigate the relative contributions of malate and aspartate to C4-pathway fluxes in each subtype, we performed 13CO2 pulse and pulse-chase labelling experiments on four C4 grass species: Zea mays and Setaria viridis (NADP-ME), Panicum miliaceum (NAD-ME) and Megathyrsus maximus (PEPCK). Only a proportion (8-50%) of the total malate pool in the leaves is photosynthetically active whereas essentially all of the aspartate pool is photosynthetically active. Estimates of metabolic fluxes indicate that approximately two thirds of the C4 pathway flux is via malate in Z. mays and the remaining third via aspartate, while in S. viridis 50% of the flux is via malate and 50% via aspartate. In P. miliaceum and M. maximus, 91% and 85% of the flux is via aspartate and the remaining 5% and 15% via malate, respectively. The results reveal greater complexity of C4 pathway fluxes than is usually represented in textbook diagrams, and demonstrate the feasibility of using non-radioactive 13CO2 in pulse-chase labelling experiments to study C4 photosynthesis and to detect C4 pathway fluxes in C3 plants engineered to perform C4 photosynthesis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.15.664927v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Nucleotide sequence analysis reveals the presence of PVY-Tam isolates affecting tamarillo in Colombia</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Martinez-Moncayo, C., Burbano-Lagos, T. C., Ontanon, C., Ferriol, I., Lopez-Moya, J. J., Uranga, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified four different virus species belonging to the genera Torradovirus, Potyvirus and Polerovirus in symptomatic tamarillo plants and identified a novel isolate of potato virus Y-Tamarillo in Narino.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Tamarillo or tree tomato (Solanum betaceum Cav.) is a fruit tree species of Andean origin with cultural and economic relevance in Colombia. However, the high incidence of complex viral diseases termed virosis in all tamarillo-growing regions of the country leads to huge production losses and seriously threatens its cultivation. The lack of effective treatments implies eradication as the only alternative in severe cases. In this work, we characterized the virome of eight tamarillo-growing locations across the Department of Narino (Colombia). By in-depth sequence analysis of RNA libraries, we confirmed the presence of up to four different virus species belonging to the genera Torradovirus, Potyvirus and Polerovirus in symptomatic tamarillo plants. These results represent the first report of torradovirus infection in tamarillo. Additionally, we identified a novel isolate of potato virus Y-Tamarillo (PVY-Tam) in Narino that could have originated in South America by a recent divergence of the PVYN lineage. We propose that length variability in the P3N-PIPO protein, which in the case of PVY-Tam contains two premature stop codons not identified in other PVY isolates, might be involved with host-specific adaptations. Our findings broaden the knowledge of tamarillo virosis in the Andean region, and overall, worldwide, thus offering new possibilities for developing effective diagnostic and control strategies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.18.664922v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Screening and machine-learning assisted prediction of translation-enhancing peptides reducing ribosomal stalling in Escherichia coli</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ojima-Kato, T., Yokoyama, G., Nakano, H., Hamada, M., Motono, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We previously reported that the nascent SKIK peptide enhances translation and alleviates ribosomal stalling caused by arrest peptides (APs) such as SecM and polyproline when positioned immediately upstream of the APs in both in vivo and in vitro translation systems.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We previously reported that the nascent SKIK peptide enhances translation and alleviates ribosomal stalling caused by arrest peptides (APs) such as SecM and polyproline when positioned immediately upstream of the APs in both Escherichia coli in vivo and in vitro translation systems. In this study, we performed a comprehensive screening of translation-enhancing peptides (TEPs) using a randomized artificial tetrapeptide library. The screening was based on the ability of the peptides to suppress SecM AP-induced translational stalling in E. coli cells. Various TEPs exhibiting a range of translation-enhancing activities were identified. In vitro translation analysis suggested that the fourth amino acid in the tetrapeptide plays a key role in reducing SecM AP-mediated stalling. Furthermore, we developed a machine learning model using a random forest algorithm to predict TEP activity. The predicted values showed a strong correlation with experimentally measured activities. These findings offer a compact peptide toolkit and a data-driven approach for mitigating AP-induced ribosome stalling, with potential applications in synthetic biology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.17.665026v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

