<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Sunday, 21 September 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Deep Neurite Analysis Tool (DeNAT): A machine-learning framework for precise automated neurite outgrowth measurement</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kumaran, M., Narayan, A. P., Sahu, Y., Banerjee, S., Menon, A. S., Soni, S., Venkatesh, I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed the Deep Neurite Analysis Tool (DeNAT), an accessible deep-learning based platform that automatically measures neurite outgrowth after injury.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurate quantification of neurite sprouting after injury is a critical step in axon regeneration research. Yet it remains a major bottleneck, as the current gold standard relies on manual counting by multiple blinded observers. This process is slow, labor intensive, and prone to variability. While some software can measure total neurite length, they aren't made to specifically measure new growth in complicated images from real-life injury models, like the thoracic crush and pyramidotomy model. Existing software can measure total neurite length in culture, but it is not designed to capture new growth in complex images from injury models, such as thoracic crush or pyramidotomy. Crucially, these tools lack the ability to selectively analyze growth within user-defined regions, a key requirement for injury paradigms. To address this need, we developed the Deep Neurite Analysis Tool (DeNAT), an accessible deep-learning based platform that automatically measures neurite outgrowth after injury. DeNAT allows users to define regions of interest to specifically quantify sprouting in images from common spinal cord injury paradigms. We benchmarked its performance against manual scoring and conventional automated approaches. DeNAT achieved 87 percent accuracy in detecting neurite sprouts relative to manual counts, while reducing variability and labor. By combining user-guided region selection with automated deep learning analysis, DeNAT offers an accurate, reproducible, and efficient solution for measuring neurite outgrowth in injury models.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.18.676567v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Developmental temperature drives distinct transcriptomic responses to acute temperatures and correlated differences in thermal tolerance</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hall, A., Rege-Colt, M., Pespeni, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Developmental plasticity is a key contributor to thermal tolerance in thermal tolerance, and the molecular basis of plasticity and potential resilience to climate change.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Marine invertebrate populations exhibit varying capacities to withstand rising environmental temperatures, but the genetic basis of this differential tolerance remains an important area of investigation. Plasticity can be a valuable tool in the arsenal of an animal trying to maintain physiological function under rapidly changing conditions and can be an important contributor to thermal tolerance. In this work, we characterized the transcriptomic response to elevated developmental temperature, and subsequent acute exposure to two higher temperatures using the widespread and ecologically important copepod, Acartia tonsa. Using a split brood experimental design, we found that copepods that developed at 22C had higher upper lethal temperatures compared to those reared at 18C, demonstrating developmental plasticity in thermal tolerance. Transcriptomic analyses revealed that developmental temperature strongly influenced gene expression both at baseline and in response to acute thermal stress. Exposure to a moderate heat challenge (28C) elicited divergent transcriptional responses between developmental treatments, suggesting developmental preconditioning, whereas exposure to extreme heat (33C) triggered a more conserved stress response across groups. Weighted gene co-expression network analysis (WGCNA) identified gene modules associated with upper lethal temperature, highlighting stress response, cellular regulation, and metabolic pathways as key contributors to thermal tolerance. Together, our results reveal how developmental environments shape gene expression patterns and thermal phenotypes, providing insight into the molecular basis of plasticity and potential resilience to climate change.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.18.676804v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Spectral network analysis illuminates coordinated trait adaptation across plant populations</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ray, R., Quarles-Chidyagwai, B., Ashlock, S., Lyons, J., Gremer, J. R., Maloof, J., Magney, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We combined hyperspectral data, inverse modeling, and network analysis to investigate population-level adaptations in Streptanthus tortuosus .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding how plant populations respond to environmental variation through functional leaf traits remains challenging due to limitations of traditional phenotyping approaches. Hyperspectral reflectance offers a rapid, non-destructive and high-throughput method to capture functional trait variation and detect signatures of local adaptation across populations. We combined hyperspectral data, inverse modeling, and network analysis to investigate population-level adaptations in Streptanthus tortuosus. Using a common garden experiment with four geographically distinct populations, we applied partial least square discriminant analysis (PLS-DA) and ridge regression for population discrimination, inverse PROSPECT modeling to estimate leaf biochemical traits, and canonical correlation analysis to examine trait-climate relationships across historical (1900-1994) and recent (1995-2024) periods. We developed a novel spectral network approach treating wavelength correlations as biologically meaningful trait networks. Populations showed distinct, heritable spectral signatures with high classification accuracy. Significant population differences emerged in anthocyanins, carotenoids, chlorophyll, and water content. Trait-climate correlations shifted between time periods, consistent with historical climate adaptation. Network analysis revealed population-specific integration patterns, with more variable environments displaying greater spectral modularity. Hyperspectral signatures provide a high-throughput tool for detecting population-level adaptation and trait coordination. Our findings suggest plant populations respond to climate change through evolved shifts in trait networks rather than isolated traits alone.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.18.676927v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A symbiotic MLO gene regulates root development via RALF34-triggered Ca2+ signalling in Lotus japonicus</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Binci, F., Guarneri, G., Somoza, S. C., Vascon, F., Capparotto, A., Di Nuzzo, E., Rago, G., Cendron, L., Navazio, L., Giovannetti, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that LjMLO4 expression is strongly induced in arbuscular mycorrhizal fungus-colonized root cells, yet loss-of-function mutants exhibit only subtle AM-associated phenotypes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mildew Locus O (MLO) genes, initially identified as powdery mildew susceptibility factors, are increasingly recognized as multifunctional regulators implicated in diverse processes including plant reproduction, root thigmotropism, and interactions with beneficial microbes. Recent studies have revealed that MLO proteins can act as Ca2+-permeable channels activated by Rapid Alkalinization Factors (RALF) peptides in reproductive cells, pointing to broader roles in Ca2+-mediated signalling. In this study, we investigate the symbiotic clade IV member LjMLO4 in the model legume Lotus japonicus, focusing on its role in root development and responsiveness to LjRALF34 peptides. We show that LjMLO4 expression is strongly induced in arbuscular mycorrhizal (AM) fungus-colonized root cells, yet loss-of-function mutants exhibit only subtle AM-associated phenotypes. Instead, we uncover a previously uncharacterized function of LjMLO4 as a regulator of primary root growth and lateral root formation, acting independently of AM symbiosis but in a Ca2+-dependent manner. Heterologous expression in E. coli confirms that LjMLO4 facilitates Ca2+ transport, while genetic and physiological assays demonstrate its contribution to LjRALF34-triggered root growth responses and Ca2+ signalling. Together, these findings identify LjMLO4 as a key molecular hub between peptide signalling, Ca2+ transport and root system architecture, highlighting MLO integration of developmental, nutrition and symbiotic cues.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.18.676995v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A protocol for high-throughput microplate-based CUT&Tag</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Johnson, B. B., Boyle, G. E., Federico, A. J., Ma, J., Hemani, H., Spencer, D. H., Sarthy, J. F., Meers, M. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a method for simultaneous parallel processing of 96 CUT&Tag samples in a standard microplate.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cleavage Under Targets & Tagmentation (CUT&Tag) is a versatile method for measuring genomic occupancy of chromatin-associated proteins with high sensitivity and specificity. CUT&Tag has low sequencing requirements and is therefore suitable for highly multiplexed experiments, but methods to process samples at throughput without specialized equipment are lacking. Here we present a method for simultaneous parallel processing of 96 CUT&Tag samples in a standard microplate. Plate-CUT&Tag can be carried out in a similar time frame to benchtop CUT&Tag and yields data of comparable quality. We present data from cell culture and patient leukemia samples processed with Plate-CUT&Tag to illustrate its utility in large-scale preclinical and translational studies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.18.676951v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MetaFold-RNA: Accurate prediction of RNA secondary structure using a meta-learning-guided deep network</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, L., Zhang, H., Li, X., Wang, Y., Xue, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present MetaFold-RNA, a novel meta-learning framework that sets a new state of the art in RNA secondary structure prediction, outperforming all SOTA methods, particularly in cross-family generalization.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurately predicting the secondary structure of ribonucleic acid (RNA) is a critical step toward deciphering its biological roles and engineering novel RNA-based technologies. However, achieving high accuracy and generalization, especially for unseen RNA families, has long remained a central challenge in computational biology. Here we present MetaFold-RNA, a novel meta-learning framework that sets a new state of the art in RNA secondary structure prediction. Instead of relying on a single predictive paradigm, MetaFold-RNA synergizes insights from disparate models through a meta-learning approach and refines predictions via a co-evolutionary optimization process. On the challenging bpRNA-new benchmark, which specifically tests performance on unseen RNA families, MetaFold-RNA surpasses all SOTA methods, outperforming the best previous approach by 7.3%. This breakthrough performance is consistently maintained across other stringent assessments, including the PDB-new dataset and blind CASP predictions. Furthermore, our method demonstrates high robustness by accurately resolving the complex topologies of artificial RNA nanostructures. This work establishes a new performance benchmark, particularly in cross-family generalization, and provides a powerful computational tool for accelerating the study and design of RNA.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.18.676970v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comprehensive benchmarking of somatic structural variant detection at ultra-low allele fractions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, Y., English, A. C., Paulin, L. F., Grochowski, C. M., Maheshwari, S., Mack, T., Berselli, M., Veit, A. D., Fu, Y., SMAHT SV working group,, Park, P. J., Sedlazeck, F. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present comprehensive benchmark data and best practices for non-cancer somatic SV detection, including a high-confidence benchmark set of pseudo-somatic insertions and deletions of 50bp or larger and a synthetic mosaic sample.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Postzygotic mosaicism gives rise to somatic structural variants (SVs) at ultra-low variant allele fractions (VAFs), which pose challenges for detection due to the high-coverage sequencing required and noise introduced by sequencing artifacts. Although somatic SV detection has been extensively studied in cancer, these studies are not directly applicable to the study of tissue mosaicism, as they rely on matched normals, target higher VAF ranges, and are enriched for different types of SVs. We present comprehensive benchmark data and best practices for non-cancer somatic SV detection. We created a synthetic mosaic sample by combining six HapMap individuals at varying proportions, generating allele fractions as low as 0.25%. This sample was sequenced to ~2,300x total coverage using Illumina, PacBio, and Nanopore technologies across multiple sequencing centers. A high-confidence benchmark SV set containing over 21,000 pseudo-somatic insertions and deletions of 50bp or larger was derived from haplotype-resolved assemblies. We evaluated 12 SV discovery pipelines and identified caller-specific strengths and sequencing platform-specific shortcomings. We find that short read-based approaches show reduced recall for insertions and repeat-associated SVs, whereas long-read sequencing achieves high accuracy throughout the genome, increasing linearly with coverage. The best algorithm's sensitivity exceeded 80% for VAFs >4% and 15% for VAFs of 0.5-1% with 60x coverage. The publicly available benchmarking data and comparative analysis of current methods provide a foundation for robust discovery of SV mosaicism in non-cancer tissues.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.18.677206v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Target capture sequencing provides insights into hybridogenetic water frogs</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> van Veldhuijzen, L., Pabijan, M., Stark, T., Struijk, R. P. J. H., France, J. M., Wielstra, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that a previously developed target capture bait set, FrogCap, can be used to answer genomic questions within the genus Pelophylax, by applying it to the three taxa of Pelphylax native to the Netherlands.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The European water frogs of the genus Pelophylax, with their propensity to hybridize and a non-orthodox inheritance system in some hybrids, present a genetic puzzle that has previously lacked strong genomic tools. This has limited investigation into the remarkable hybridogenetic system observed in the edible frog P. esculentus as well as inhibited monitoring of invasive populations in a genus that is critically vulnerable to this threat. In this study we explore whether a previously developed target capture bait set, FrogCap, can be used to answer genomic questions within the genus, by applying it to the three taxa of Pelophylax native to the Netherlands. We observe that the target capture data cleanly separates the three taxa, confirming significant introgression of P. lessonae mtDNA into P. ridibundus from the Netherlands. We also show that target capture is an efficient method for identifying polyploidy in P. esculentus and is effective at determining ancestral contribution within our sample population. Targeted sequence capture using FrogCap is a useful tool to unravel the intricate evolution of Pelophylax water frogs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.18.677054v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SpaPheno: Linking Spatial Transcriptomics to Clinical Phenotypes with Interpretable Machine Learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Duan, B., Cheng, X., Zhou, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present SpaPheno, an interpretable machine learning framework that integrates spatial transcriptomics with clinically annotated bulk RNA seq to identify spatially resolved biomarkers predictive of patient outcomes, including survival, tumor stage, and immunotherapy response.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Linking spatial transcriptomic data to clinically relevant phenotypes is essential for advancing spatially informed precision oncology. Here, we present SpaPheno, an interpretable machine learning framework that integrates spatial transcriptomics with clinically annotated bulk RNA seq to identify spatially resolved biomarkers predictive of patient outcomes, including survival, tumor stage, and immunotherapy response. SpaPheno provides multi-scale interpretability from tissue regions to cell types and individual spatial spots, enabling clear biological insights from complex spatial data. We validate SpaPheno through extensive simulations and applications to multiple cancer cohorts: primary liver cancer, clear cell renal cell carcinoma, breast cancer, and melanoma, demonstrating robust predictive performance alongside biologically meaningful spatial patterns. SpaPheno offers a generalizable strategy to translate spatial omics data into clinically actionable knowledge, facilitating precision oncology informed by tumor spatial architecture.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.18.676993v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Mapping transcriptional responses to cellular perturbation dictionaries with RNA fingerprinting</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Grabski, I. N., Lee, J., Blair, J., Dalgarno, C., Mascio, I., Bradu, A., Knowles, D. A., Satija, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce RNA fingerprinting, a statistical framework that maps transcriptional responses from new experiments onto reference perturbation dictionaries, and demonstrate its broad utility across diverse biological settings.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell perturbation dictionaries provide systematic measurements of how cells respond to genetic and chemical perturbations, and create the opportunity to assign causal interpretations to observational data. Here, we introduce RNA fingerprinting, a statistical framework that maps transcriptional responses from new experiments onto reference perturbation dictionaries. RNA fingerprinting learns denoised perturbation "fingerprints" from single-cell data, then probabilistically assigns query cells to one or more candidate perturbations while accounting for uncertainty. We benchmark our method across ground-truth datasets, demonstrating accurate assignments at single-cell resolution, scalability to genome-wide screens, and the ability to resolve combinatorial perturbations. We demonstrate its broad utility across diverse biological settings: identifying context-specific regulators of p53 under ribosomal stress, characterizing drug mechanisms of action and dose-dependent off-target effects, and uncovering cytokine-driven B cell heterogeneity during secondary influenza infection in vivo. Together, these results establish RNA fingerprinting as a versatile framework for interpreting single-cell datasets by linking cellular states to the underlying perturbations which generated them.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.19.676866v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">LieOTMap-FFT: A Differentiable Fitting Framework Combining Lie Algebra and FFT-accelerated Optimal Transport for Cryo-EM Maps</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hu, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce LieOTMap-FFT, a novel, fully differentiable framework for rigid body map to map alignment.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Interpreting cryo-electron microscopy (cryo-EM) density maps often requires the accurate placement of atomic models, a challenging high-dimensional fitting problem. We introduce LieOTMap-FFT, a novel, fully differentiable framework for rigid body map to map alignment. Our framework synergistically combines four key mathematical and computational techniques. First, we parameterize the SE(3) rigid body transformation using Lie algebra, ensuring a continuous and singularity-free optimization landscape suitable for gradient descent. Second, we treat the mobile and target density maps as probability distributions on a 3D grid and measure their similarity using an Optimal Transport (OT) score based on the Sinkhorn algorithm. Third, to make this computationally feasible for large maps, we leverage the Fast Fourier Transform (FFT) to accelerate the core convolution operations within the Sinkhorn iterations from O(N2) to O(NlogN). Finally, a TM-align-inspired similarity kernel provides a robust score landscape for global search. We demonstrate the power of LieOTMap-FFT by fitting a large ribosomal subunit (PDB: 1AON) into a cryo-EM map (EMD-1046), refining a random initial placement to a final RMSD of 4.81 Angstrom. This result showcases the framework's ability to achieve high-accuracy global convergence by effectively integrating these four core technologies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.18.676986v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Development of a differential panel to identify Peronospora belbahrii races in sweet basil (Ocimum basilicum)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Allen, K. S., Barrett, A. J., Ben Naim, Y., Mattera, R., Wyenandt, C. A., Cohen, Y., Simon, J. E., Ma, L.-J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A differential panel of distinct basil cultivars with varying and differential sources of genetic resistance tested against downy mildew, and the evolution of new races under selective pressure.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> First reported in the United States in 2007, downy mildew of sweet basil (Ocimum basilicum), caused by the oomycete pathogen Peronospora belbahrii, has become a major challenge in global basil production. Downy mildew-resistant cultivars have been developed and commercially available since 2018, but observations of isolates overcoming resistance were reported after each new introduction. The rapid development of pathogens overcoming genetic resistance suggests the evolution of new races under selective pressure. In this study, we report on the development of a differential panel of distinct basil cultivars with varying and differential sources of genetic resistance tested against P. belbahrii strains obtained from geographically diverse multi-year collections from basil-producing locations in the US, Israel, and Italy. Screening multiple P. belbahrii isolates revealed three distinct virulence profiles that distinguished race 0, race 1, and race 2. Races 1 and 2 were defined by virulence in cultivars with single gene resistance loci Pb1 and Pb2, respectively. Evaluation of cultivars with quantitative resistance revealed more variation in susceptibility or partial resistance, suggesting broad spectrum (or race-nonspecific) resistance. With continued isolate collection and screening efforts, the denomination of P. belbahrii races may be further expanded, and newly developed resistant cultivars can be added to the proposed panel to definitively evaluate novel P. belbahrii strains. This study lays the groundwork to further explore dynamics of race evolution and mechanisms of resistance breakdown, which will be essential for discovering new sources of resistance for controlling basil downy mildew.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.18.674471v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Root growth in Arabidopsis depends on the amount of glutathione and not the glutathione redox potential</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bangash, S. A. K., Safi, M. T., Ugalde, J. M., Wagner, S., Nagel, K. A., Jansen, M., Krueger, S., Schwarzlander, M., Moseler, A., Reichheld, J.-P., Kopriva, S., Meyer, A. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that plant growth is dependent on the availability of sufficient amounts of GSH, and not affected by pronounced changes in EGSH, which is a decisive step towards understanding the mechanisms underpinning the proposed role of glutathione in cell cycle and growth control.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cell cycle activity and hence plant growth strictly depends on glutathione homeostasis. Despite compelling evidence for this dependency based on glutathione depletion, the cause for the cell cycle arrest had remained unclear. Cell cycle control may depend on either the absolute amount of glutathione or on the glutathione redox potential (EGSH). To unambiguously distinguish those two options, we characterized an allelic series of six Arabidopsis mutants affected in glutamate-cysteine ligase, which catalyses the first step for the biosynthesis of reduced glutathione (GSH). When grown under the same conditions, even mutants with 20% and 50% of wild-type GSH amounts are slightly stunted. The most severely compromised mutants, zir1 and rml1, were crossed with either gr1, which lacks cyto-nuclear glutathione disulfide reductase and was used to induce a pronounced shift in EGSH, or with bir6, which has a diminished glutathione consumption and thus exhibits slightly increased levels of GSH. Based on theoretical considerations, these levels are not expected to shift the EGSH to any significant extent. Our study shows that deleting GR1 in the zir1 or rml1 background does not result in an obvious phenotypic change. By contrast, deleting BIR6 was sufficient to suppress the growth arrest in rml1 and to attenuate the growth in zir1. These findings demonstrate that root growth is dependent on the availability of sufficient amounts of GSH, and not affected by pronounced changes in EGSH. This insight provides a decisive step towards understanding the mechanisms underpinning the proposed role of glutathione in cell cycle and growth control.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.18.677034v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Yorzoi: Predicting RNA-seq coverage from DNA sequence in yeast</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Schneider, T., Rafi, A. M., Jensen, C., Liao, D., Zhao, Y., de Boer, C., Ellis, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present Yorzoi, a sequence-to-expression model for bakers' yeast that predicts RNA-seq coverage for a 5 kilobase, multi-gene window of DNA at 10 base pair (bp) resolution.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Yeast is one of the principal model organisms in synthetic biology and a widely used chassis for testing heterologous DNA sequences. However, designing DNA constructs correctly is currently limited by our incomplete quantitative understanding of how sequences are transcribed and translated, especially non-native sequences. Here, we present Yorzoi, a sequence-to-expression model for bakers' yeast that predicts RNA-seq coverage for a 5 kilobase, multi-gene window of DNA at 10 base pair (bp) resolution. To extend its predictive ability beyond native DNA, Yorzoi has been pretrained on a comprehensive dataset not only including native yeast sequences but also human sequences expressed in yeast and structurally rearranged synthetic yeast chromosomes. We demonstrate that our model has learned general rules of yeast transcription by achieving high predictive performance on various downstream tasks. Yorzoi is a powerful tool for in-silico testing of DNA sequences and directly applicable for sequence design. A web application to use our model is available at yorzoi.eu and the code open source on GitHub.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.20.677345v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Advanced Deep Learning Enables Prediction of Allogeneic Stem Cell Mobilization Success</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Adil, A., Xiang, J., Piccirillo, N., Harris, H. G., Sica, S., DiPersio, J. F., Hurwitz, S. N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A machine-learning framework to identify poor donor CD34+ cells in HSPC transplantation and early identification of poor donors .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Hematopoietic stem and progenitor cell (HSPC) transplantation offers a potentially curative therapy for aggressive hematologic malignancies and bone marrow failure syndromes. Successful transplantation depends on effective mobilization of donor CD34+ cells, yet some healthy donors fail to achieve adequate CD34+ yields despite standard granulocyte colony-stimulating factor (G-CSF)-based regimens. Early identification of such donors enables timely intervention, improving transplantation outcomes and reducing healthcare costs. We analyzed demographic and pre- and post-G-CSF laboratory data from 1,160 healthy donors from across multiple institutions and developed two complementary machine-learning frameworks to predict mobilization outcome. A transformer-based probabilistic model (TabPFN) trained on baseline complete blood counts (CBCs) rigorously discriminates poor from good mobilizers. Applying the same architecture to donor data after mobilization attains near-perfect discrimination. To unify the predictions across time points, we introduce an attention-aware neural network that ingests either baseline or post-mobilization data via a ''lab-type'' context flag, enabling accurate prediction of poor mobilizers both before and after G-CSF mobilization. We further validated the framework on data from over 19,000 healthy donors compiled by the Center for International Blood and Marrow Transplant Research. These interpretable models enable early triage and ''just-in-time'' rescue interventions, providing a data driven foundation for personalized donor mobilization strategies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.17.676674v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Constructing Gene Regulatory Network using Chatterjee's Rank Correlation with Single-cell Transcriptomic Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gupta, S., Chaudhuri, A., Raghuraman, V., Ni, Y., Cai, J. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a multiple testing framework for inference of gene regulatory networks from single-cell RNA sequencing data by using Chatterjee's rank correlation coefficient, a nonparametric measure of dependence, and propose a new test for directed regulation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Discovering gene regulatory networks (GRNs) from single-cell RNA sequencing (scRNA-seq) data is critical for understanding cellular function, but existing methods are limited by strong theoretical assumptions or high computational complexity. We introduce a multiple testing framework for inference of GRN by using Chatterjee's rank correlation coefficient, a nonparametric measure of dependence. Our approach overcomes the limitations of traditional methods while offering a transparent, scalable, and computationally efficient alternative to recent black-box machine learning models. Crucially, we address the challenge of non-independent observations in scRNA-seq by developing a data-driven algorithm for estimating robust testing cutoffs. Furthermore, we exploit the asymmetric nature of Chatterjee's correlation to propose a new test for directed regulation, enabling the construction of biologically meaningful and directionally informed GRNs. We demonstrate that our method consistently outperforms state-of-the-art approaches in recovering true regulatory links from both simulated and real datasets, providing a powerful tool for dissecting complex GRNs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.17.676530v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">X-intNMF: A Cross- and Intra-Omics Regularized NMF Framework for Multi-Omics Integration</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bui, T.-T., Xie, R., Zhang, W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose X-intNMF, a network-regularized non-negative matrix factorization (NMF) model that explicitly incorporates both cross-omics and intra-omics feature interaction networks during multi-omics integration.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: The rapid accumulation of multi-omics data presents a valuable opportunity to advance our understanding of complex diseases and biological systems, driving the development of integrative computational methods. However, the complexity of biological processes - spanning multiple molecular layers and involving intricate regulatory interactions - requires models that can capture both intra- and cross-omics relationships. Most existing integration methods primarily focus on sample-level similarities or intra-omics feature interactions, often neglecting the interactions across different omics layers. This limitation can result in the loss of critical biological information and suboptimal performance. To address this gap, we propose X-intNMF, a network-regularized non-negative matrix factorization (NMF) model that explicitly incorporates both cross-omics and intra-omics feature interaction networks during multi-omics integration. By modeling these multi-layered relationships, X-intNMF enhances the representation of biological interactions and improves integration quality and prediction accuracy. Results: For evaluation, we applied X-intNMF to predict breast cancer phenotypes and classify clinical outcomes in lung and ovarian cancers using mRNA expression, microRNA expression, and DNA methylation data from TCGA. The results show that X-intNMF consistently outperforms state-of-the-art methods. Ablation studies confirm that incorporating both cross-omics and intra-omics interactions contributes significantly to the model's improved performance. Additionally, survival analysis demonstrates that the integrated multi-omics representation offers strong prognostic value for both overall survival and disease-free status. These findings highlight X-intNMF's ability to effectively model multi-layered molecular interactions while maintaining interpretability, robustness, and scalability within the NMF framework.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.17.676796v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Mevalonate Biosynthesis is a Metabolic Vulnerability of Gemcitabine-resistant Pancreatic Cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Beutel, A., Calderon, S., Nghiem, E., Gulay, K., Santana, L., Zimmer, E., Singh, R., Anaraki, C., Yousefian, N., Allgoewer, C., Tong, G., Geller, C., Chao, A., Juarez, D., Seufferlein, T., Muir, A., Jang, C., Edinger, A., Seldin, M., Martinez, T., Kleger, A., Chang, D., Tiriac, H., Valerin, J., Fruman, D., Halbrook, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the mevalonate biosynthesis pathway represents a promising therapeutic target in gemcitabine resistance and may serve as a biomarker to stratify treatment selection in PDAC patients.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Pancreatic ductal adenocarcinoma (PDAC) is a lethal malignancy with a devastating prognosis. Gemcitabine, a pyrimidine anti-metabolite, is a cornerstone in PDAC therapy. However, resistance remains a major hurdle in clinical care. Resistance can arise from microenvironmental metabolites or through direct metabolic reprogramming of pancreatic cancer cells. Here, we generated PDAC models of acquired gemcitabine resistance to determine the relationship between these mechanisms. We observed that physiological levels of exogenous pyrimidines have a diminished ability to impact gemcitabine response in PDAC cells with acquired resistance. This occurs as the metabolic reprogramming of PDAC cells in response to gemcitabine treatment forces a suppression of the pyrimidine salvage pathway. Importantly, this metabolic rewiring renders gemcitabine-resistant PDAC cells highly susceptible to inhibition of the rate limiting enzyme of the mevalonate biosynthesis pathway, 3-hydroxy-3-methylglutaryl coenzyme A reductase (HMGCR), using statins. Notably, statin treatment inhibits the growth of gemcitabine-resistant tumors in immunocompetent mouse models. Through metabolite rescue experiments, we identified geranylgeranyl pyrophosphate as the critical metabolite lost during statin treatment, resulting in reduced protein geranylation in PDAC cells. Finally, as downregulation of the HMGCR is gradually acquired during gemcitabine resistance, we observed that HMGCR expression predicts patient response to gemcitabine. Collectively, these data demonstrate that the mevalonate biosynthesis pathway represents a promising therapeutic target in gemcitabine resistance and may serve as a biomarker to stratify treatment selection in PDAC patients.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.19.675739v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Exploring Resistance to ETS Targeting Agents in Diffuse Large B-Cell Lymphoma</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Spriano, F., Cascione, L., Tarantelli, C., Sartori, G., Arribas, A. J., Velasova, A., Napoli, S., Havranek, O., Toretsky, J. A., Bertoni, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that multiple, heterogeneous mechanisms drive resistance to ETS inhibition in DLBCL and highlight therapeutic strategies to overcome it.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Diffuse large B-cell lymphoma (DLBCL) remains a challenging disease with limited therapeutic options beyond standard immunochemotherapy. ETS transcription factors, including SPIB and SPI1, are implicated in lymphoma pathogenesis and can be targeted by the small molecule TK216, which disrupts ETS-DHX9 interactions. To explore mechanisms of resistance, we generated stable TK216-resistant clones from the ABC-DLBCL line U2932. Resistant clones displayed a 4-5-fold increase in IC50 values and lost G2-M arrest upon treatment. Transcriptomic and mutational analyses revealed three resistance patterns: (i) MDR1/ABCB1 overexpression, leading to multidrug efflux; (ii) Cluster A, enriched for proliferation, Wnt, and transcriptional programs, with mutations in ESR2, USP24, and SFSWAP; and (iii) Cluster B, characterized by actin/microtubule remodeling, altered metabolism, and mutations in SRSF11 and PATJ. Pharmacologic screening identified increased sensitivity of resistant cells to BCL2, MCL1, and XPO1 inhibitors, while showing reduced sensitivity to aurora kinase and microtubule-targeting agents. Venetoclax and selinexor retained activity in resistant models, supporting their potential for rational combinations with TK216. These findings demonstrate that multiple, heterogeneous mechanisms drive resistance to ETS inhibition in DLBCL and highlight therapeutic strategies to overcome it.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.18.675797v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Quercetin as a Bitter Taste Receptor Agonist with Anticancer Effects in Head and Neck Cancer Cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Turner, G., Sywanycz, S., Buchler, B. L., Wardlow, R. D., Lee, R. J., Carey, R. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Quercetin is a bitter compound with demonstrated anticancer effects in preclinical models of head and neck squamous cell carcinoma, which is likely mediated by T2R14, a group of G protein-coupled receptors.</p><p style="font-size: 16px; text-align: justify;">Background/Objectives: Quercetin is a bitter compound with demonstrated anticancer effects in preclinical models of head and neck squamous cell carcinoma (HNSCC). In taste transduction, bitter compounds activate bitter taste receptors (T2Rs), a group of G protein-coupled receptors with downstream signaling that includes cytosolic calcium (Ca2+) release. T2Rs are expressed in HNSCC cells, where their activation induces apoptosis in vitro. Increased T2R expression in HNSCC also correlates with improved patient survival. The objective of this study was to investigate the role of quercetin as an anticancer T2R agonist in HNSCC cells in vitro and ex vivo. Methods: Quercetin-mediated Ca2+ responses were assessed using live cell Ca2+ imaging in the presence of the T2R14 antagonist LF1 and G-protein inhibitor YM-254980 (YM) in UM-SCC-47 and FaDu HNSCC cell lines. Cell viability was evaluated using crystal violet assays in cell lines and MTS assays in patient-derived tumor slices. Mitochondrial depolarization was measured with TMRE in the presence and absence of T2R pathway inhibitors. Results: Quercetin induced a Ca2+ response in HNSCC cells, which was significantly reduced by LF1 and YM. Quercetin also decreased cell viability in vitro and showed a potential decrease in viability in tumor slices but was not statistically significant. Finally, quercetin caused mitochondrial depolarization, which was reduced in the presence of LF1 but not by YM. Conclusions: In HNSCC cells, quercetin causes a Ca2+ response that is likely mediated by T2R14, decreases viability, and causes mitochondrial depolarization.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.17.676947v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Loss of the tumor suppressor PTEN activates cell-intrinsic interferon signaling to drive immune resistance</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, Y., Manokaran, C., Chen, J., Gu, H., Liu, D., Bousso, P., Zhao, J., Roberts, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that PTEN deficiency renders cancer cells resistant to the cytotoxic effects of immune cells and interferon-{gamma}.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Loss of the tumor suppressor PTEN is strongly associated with a lack of response to immune checkpoint blockade therapies in cancer patients, but the underlying mechanisms are not fully understood. We have developed a transformation model where knocking out PTEN in human mammary epithelial cells drives dependence on the p110{beta} subunit of PI3K and, notably, elicits a robust induction of endogenous retroviral elements (ERVs) and activation of the interferon signaling. This constitutive cell-intrinsic interferon response, hallmarked by hyperactivated STAT1, is also observed in human tumors with a PTEN-low status. We further find that PTEN deficiency renders cancer cells resistant to the cytotoxic effects of immune cells and interferon-{gamma}. Notably, PTEN loss also results in a dependency on an activated DNA damage response pathway, leading to an exquisite vulnerability to CDK12 inhibition. Our study suggests an interferon adaptation model in which tumors driven by PTEN deficiency inherently activate the interferon response, enabling them to adapt to interferon cytotoxicity and gain resistance to immunotherapies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.17.676669v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Beckwith-Wiedemann syndrome multiomic analysis of hepatoblastoma uncovers unique tumor heterogeneity and cellular landscapes, including transition cells leading to tumor formation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nirgude, S., Tichy, E. D., Zhang, Y., Pradieu, R., Xie, M., Bernt, K. M., MacFarland, S. P., Kalish, J. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified a unique population of transition cells in BWS hepatoblastoma with unique molecular profiles, with unique Molecular profiles, which likely promote the precancer to cancer neoplastic transition in the disease.</p><p style="font-size: 16px; text-align: justify;">Background: Beckwith-Wiedemann syndrome (BWS) is an overgrowth and cancer predisposition syndrome caused by epigenetic alterations on chromosome 11p15 that predisposes children to multiple cancer types, including hepatoblastoma. Hepatoblastoma is heterogenous in nature, and the 11p15 changes that cause BWS can also be found as a somatic alteration in nonBWS hepatoblastomas, further adding complexity to this disease. Methods: To understand the impact of the predisposition molecular cues in BWS hepatoblastoma, we interrogated BWS and nonBWS hepatoblastomas, as well as adjacent normal liver, using a multiomic approach [single nuclei RNA-sequencing (snRNA-seq) + single nuclei assay for transposable-accessible chromatin sequencing (snATAC-seq)]. Results: Our approach identified an enrichment of the WNT signaling pathway in BWS hepatoblastoma. Despite similar histology, we found greater tumor heterogeneity and embryonic transcriptional signatures in BWS hepatoblastoma. Furthermore, using pseudotime analysis, we identified a population of transition cells in BWS, with unique molecular profiles, which likely promote the precancer to cancer neoplastic transition in BWS. Conclusions: This study highlights key signaling pathways, particularly WNT, and identifies a unique population of transition cells that may drive neoplastic transformation in BWS hepatoblastoma. These findings provide new insights into the molecular events leading to cancer in BWS and suggest potential targets for early intervention and prevention strategies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.17.676802v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

