<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Monday, 19 May 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Towards foundation models that learn across biological scales</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kalfon, J., Cantini, L., Peyre, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a framework and architecture called Xpressor that enables cross-scale learning by (1) using a novel cross-attention mechanism to compress high-dimensional gene representations into lower-dimensional cell-state vectors, and (2) implementing a multi-scale fine-tuning approach that</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We have reached a point where many bio foundation models exist across 4 different scales, from molecules to molecular chains, cells, and tissues. However, while related in many ways, these models do not yet bridge these scales. We present a framework and architecture called Xpressor that enables cross-scale learning by (1) using a novel cross-attention mechanism to compress high-dimensional gene representations into lower-dimensional cell-state vectors, and (2) implementing a multi-scale fine-tuning approach that allows cell models to leverage and adapt protein-level representations. Using a cell Foundation Model as an example, we demonstrate that our architecture improves model performance across multiple tasks, including cell-type prediction (+12%) and embedding quality (+8%). Together, these advances represent first steps toward models that can understand and bridge different scales of biological organization.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.16.653447v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Amira: gene-space de Bruijn graphs to improve the detection of AMR genes from bacterial long reads</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Anderson, D., Lima, L., Le, T., Judd, L. M., Wick, R. R., Iqbal, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present Amira, a tool to detect antimicrobial resistance (AMR) genes directly from unassembled long-read sequencing data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurate detection of antimicrobial resistance (AMR) genes is essential for the surveillance, epidemiology and genotypic prediction of AMR. This is typically done by generating an assembly from the sequencing reads of a bacterial isolate and running AMR gene detection tools on the assembly. However, despite advances in long-read sequencing that have greatly improved the quality and completeness of bacterial genome assemblies, assembly tools remain prone to large-scale errors caused by repeats in the genome, leading to inaccurate detection of AMR gene content and consequent impact on resistance prediction. In this work we present Amira, a tool to detect AMR genes directly from unassembled long-read sequencing data. Amira leverages the fact that multiple consecutive genes lie within a single read to construct gene-space de Bruijn graphs where the k-mer alphabet is the set of genes in the pan-genome of the species under study. Through this approach, the reads corresponding to different copies of AMR genes can be effectively separated based on the genomic context of the AMR genes, and used to infer the nucleotide sequence of each copy. Amira achieves significant improvements in genomic copy number recall and nucleotide accuracy, demonstrated through objective simulations and comparison with alternative read and assembly-based methods on samples with manually curated truth assemblies. Applied to a dataset of 32 Escherichia coli samples with diverse AMR gene content, Amira achieves a mean genomic-copy-number recall of 98.4% with precision 97.9% and nucleotide accuracy 99.9%. Finally, we compare the number of fully (>90%) present genes with good read support by Amira and Flye with AMRFinderPlus across all E. coli, K. pneumoniae and E. faecium nanopore datasets from the ENA (n=8593, 2469 and 420 respectively), quantifying the improvement in recall when handling heterogeneous data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.16.654303v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">InstaNovo-P: A de novo peptide sequencing model for phosphoproteomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lauridsen, J., Ramasamy, P., Catzel, R., Canbay, V., Mabona, A., Eloff, K., Fullwood, P., Ferguson, J., Kirketerp-Moller, A., Goldschmidt, I. S., Claeys, T., van Puyenbroeck, S., Lopez Carranza, N., Schoof, E. M., Martens, L., Van Goey, J., Francavilla, C., Jenkins, T. P., Kalogeropoulos, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a phosphorylation specific version of our transformer-based InstaNovo model, fine-tuned on extensive phosphoproteomics datasets, which significantly surpasses existing methods in phosphorylated peptide detection.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Phosphorylation, a crucial post-translational modification (PTM), plays a central role in cellular signaling and disease mechanisms. Mass spectrometry-based phosphoproteomics is widely used for system-wide characterization of phosphorylation events. However, traditional methods struggle with accurate phosphorylated site localization, complex search spaces, and detecting sequences outside the reference database. Advances in de novo peptide sequencing offer opportunities to address these limitations, but have yet to become integrated and adapted for phosphoproteomics datasets. Here, we present InstaNovo-P, a phosphorylation specific version of our transformer-based InstaNovo model, fine-tuned on extensive phosphoproteomics datasets. InstaNovo-P significantly surpasses existing methods in phosphorylated peptide detection and phosphorylated site localization accuracy across multiple datasets, including complex experimental scenarios. Our model robustly identifies peptides with single and multiple phosphorylated sites, effectively localizing phosphorylation events on serine, threonine, and tyrosine residues. We experimentally validate our model predictions by studying FGFR2 signaling, further demonstrating that InstaNovo-P uncovers phosphorylated sites previously missed by traditional database searches. These predictions align with critical biological processes, confirming the model's capacity to yield valuable biological insights. InstaNovo-P adds value to phosphoproteomics experiments by effectively identifying biologically relevant phosphorylation events without prior information, providing a powerful analytical tool for the dissection of signaling pathways.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.14.654049v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Snekmer Learn/Apply: A kmer-based vector similarity approach to proteinclassification suitable for metagenomic datasets</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nitka, T. A., Jacobson, J., Chang, C. H., Krause, G. R., Wheeler, T. J., Egbert, R. G., Nelson, W. C., McDermott, J. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We use protein kmers as a fast filter to map functional annotations to new sequences. We first demonstrate that our method accurately applies TIGRFAMs annotations to protein fragments and to a low-sequence identity benchmark dataset.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Advances in whole genome sequencing have led to a rapid and ongoing increase in the amount of sequence data available, but 40-50% of known genes have no functional annotation and only 25-30% have specific functional annotations. Current functional annotation approaches typically rely on computationally expensive pairwise or multiple sequence alignments, preventing rapid development of models for novel protein functions and sometimes limiting methods to one ontology. Representation of sequence in short segments (kmers) has been used in many applications for nucleotide sequence, and more recently has been applied to protein sequence as well. We previously developed Snekmer, a tool which uses kmer patterns to develop alignment-free individual protein family models. Other approaches, such as MMSeqs2 and DIAMOND, use protein kmers as a fast filter to reduce search space for subsequent sequence alignment. Here, we describe a novel addition to the Snekmer tool which builds kmer libraries for protein families and uses those libraries to map functional annotations to new sequences. We first demonstrate that our method accurately applies TIGRFAMs annotations to protein fragments and to a low-sequence identity benchmark dataset, and further use it to annotate a set of drought stress associated soil and rhizosphere metagenome sequences with higher sensitivity towards several important protein function classes than that shown by HMMs. We have incorporated this workflow into Snekmer.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.16.654600v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An Updated Polygenic Index Repository: Expanded Phenotypes, New Cohorts, and Improved Causal Inference</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Alemu, R., Terskaya, A., Howell, M., Guan, J., Sands, H., Kleinman, A., 23andMe Research Team,, Bann, D., Morris, T., Ploubidis, G. B., Fitzsimons, E., Mullan Harris, K., Caspi, A., Corcoran, D. L., Moffitt, T. E., Poulton, R., Sugden, K., Williams, B. S., Steptoe, A., Ajnakina, O., Vainik, U., Esko, T., Estonian Biobank Research Team,, Campbell, A., Hayward, C., Iacono, W. G., McGue, M., Krueger, R. F., Docherty, A. R., Shabalin, A. A., Hertwig, R., Koellinger, P., Richter, D., Goebel, J., Ahlskog, R., Oskarsson, S., Magnusson, P. K. E., Harden, K. P., Tucker-Drob, E. M., Pahnke, C. K. L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Version 2 of the Polygenic Index Repository, which expands the number of phenotypes from 47 to 61, increases the number to 61 datasets, and adopts a more consistent and improved methodology for PGI construction.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Polygenic indexes (PGIs) - DNA-based phenotype predictors for individual phenotypes - have become essential tools across the biomedical and social sciences. We introduce Version 2 of the Polygenic Index Repository, which expands the number of phenotypes from 47 to 61, increases the number of participating datasets from 11 to 20, and adopts a more consistent and improved methodology for PGI construction. For 16 phenotypes, we leverage summary statistics from an updated GWAS meta-analysis with greater statistical power compared to the original release, thereby improving the PGI's predictive power. To improve power for family-based analyses, we provide imputed parental PGIs in all datasets with first-degree relatives and we provide a framework for interpreting results from analyses that control for parental PGIs. Together, the updates improve predictive accuracy, expand coverage to new cohorts and phenotypes, and introduce novel tools that reduce confounding bias and improve interpretability.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.14.653986v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Exploring the interplay of complex carbohydrate intake, the microbiome CAZymes pool and short-chain fatty acid production in the human gut: insights from different cohorts in the Argentine population</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Trotta, M., Agustini, C., Rohr, C., Ame, R. M., Giordano, L., Balderramo, D., Romagnoli, P., vazquez, m.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We examined the relationship between complex carbohydrate intake, CAZyme activity in the human gut microbiome, and SCFA production at the whole metagenomic level across three cohorts: a healthy reference-controlled cohort, an average cohort of individuals living in industrialized cities, and a cohort of inflammatory bowel disease</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Carbohydrate-active enzyme (CAZyme) activity in the gut microbiome has significant implications for health, including the release of nutrients otherwise inaccessible to the host and the enhancement of digestive efficiency. The primary end products of indigestible carbohydrate fermentation are short-chain fatty acids (SCFAs). We hypothesized that increased dietary fiber consumption could lead to greater SCFA production in the human gut. To investigate this, we examined the relationship between complex carbohydrate intake, CAZyme activity in the human gut microbiome, and SCFA production at the whole metagenomic level across three cohorts: a healthy reference-controlled cohort, an average cohort of individuals living in industrialized cities, and a cohort of patients with inflammatory bowel disease (IBD). Metagenomic sequencing and bioinformatic analyses were utilized to assess the diversity, abundance, and functionality of CAZymes, as well as the metabolic capacity for SCFA production. The average cohort exhibited higher alpha diversity of CAZyme families compared to the reference-controlled cohort, although subfamily composition was similar between both. A moderate negative correlation was identified between CAZyme abundance and SCFA production, indicating that a higher number of these enzymes does not directly translate to increased SCFA synthesis. In IBD patients, a decrease in the diversity and composition of CAZyme subfamilies was observed, suggesting a disruption in enzymatic functions associated with the disease. However, the overall functionality of CAZymes remained relatively stable across different health conditions, highlighting the resilience of the gut microbiome for these functions. These findings deepen our understanding of the gut microbiome role in health and disease, emphasizing that despite variations in microbial diversity, key enzymatic functions persist. The study underscores the complexity of the non-linear relationship between complex carbohydrate metabolism and SCFA production, laying the groundwork for future research on microbiome-targeted therapeutic/dietary profile interventions in both non-disease and chronic diseases conditions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.15.654276v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">eSPred: Explainable scRNA-seq Prediction via Customized Foundation Models and Pathway-Aware Fine-tuning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sun, L., Yang, Q., Zhang, J., Guo, W., Lin, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce eSPred, a customized foundation model designed for predictive analysis of single-cell RNA sequencing, which improves prediction accuracy and highlights pathways linked to disease mechanisms.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell RNA sequencing (scRNA-seq) has been widely used for studying cellular heterogeneity, but its use for subject-level prediction and clinical applications is still limited. We introduce eSPred, a customized foundation model designed for predictive analysis of scRNA-seq. It integrates cell-type information through a grouping strategy during pre-training and leverages pathway information to guide network flow during fine-tuning. Across multiple datasets, eSPred improves prediction accuracy and highlights pathways linked to disease mechanisms. These results suggest that eSPred can help bridge the gap between single-cell data and subject-level clinical insights, supporting more precise diagnosis and better-informed treatment decisions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.14.654052v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DNA methylation networks during pig fetal development: a joint fused ridge estimation approach</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wachala, K. M., de Vos, J., Madsen, O., Derks, M. F. L., Peeters, C. F. W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a new analytical framework for studying organism development using pig as a model species, and show that DNA methylation patterns underlying mammalian development are considerably more dynamic than previously recognized.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Although an organisms genetic information is predominantly identical among most of its cell types, the epigenome regulates the expression of the genome in a cell type- and context-dependent manner. In mammals, DNA methylation in regulatory regions, such as promoters, primarily regulates gene expression by inducing transcriptional inactivation. With genome-wide approaches came the realization that DNA methylation patterns underlying mammalian development are considerably more dynamic than previously recognized. This realization highlights the need for methodological approaches capable of capturing this phenomenon. In this study, we investigated the feasibility of modeling DNA methylation networks by jointly estimating regularized precision matrices from time- and tissue-specific omics data derived from the pig genome. For that, we utilized RNA-Seq and RRBS data that span seven pig tissues at three developmental stages: early organogenesis, late organogenesis, and newborn. Our analysis focused on 61, 48, and 74 genes -- differentially expressed across developmental stages and CpG-methylated in promoter regions, from endoderm-, mesoderm-, and ectoderm-derived tissues, respectively. Using a joint fused ridge approach, we were able to borrow information across tissues and time points, enabling more robust network inference. This analytical framework advances exploratory methods for studying organism development using pig as a model species. Our results highlight the importance of fetal-maternal immunity and the circulatory system in early development, and shed light on dynamic interactions across tissues, organ systems, and germ layers. We anticipate that this flexible framework can be extended to other omics data and species, facilitating future research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.15.654211v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Macroscopic Analyses of RNA-Seq Data to Reveal Chromatin Modifications in Aging and Disease</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mahajan, A., Ratti, F., Wang, B., El-Samad, H., Kaufman, J. H., Gopalakrishnan, V.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We apply an energy landscape model based on the Arrhenius equation to estimate the energetic barriers associated with chromatin state transitions, which can provide insight into chromatin dynamics on aging and disease.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Regulation of gene expression is fundamental for proper cellular function, and is constrained by the local chromatin environment of each gene, which varies spatially along the chromosome and is shaped by epigenetic modifications. Epigenetic modifications induce changes in the local chromatin structure, which can influence gene expression, by affecting the accessibility of DNA to transcription factors. Such changes are particularly relevant in aging and genetic disorders like Hutchinson-Gilford Progeria Syndrome (HGPS) and Werner Syndrome (WRN), where altered chromatin structure contributes to disease pathology. In this study, we analyze RNA-seq data using macroscopic metrics designed to be explicitly sensitive to chromatin modifications. The first metric, intra-chromosomal gene correlation length, measures spatial correlations in gene expressions along the chromosome. The second metric employs an energy landscape model based on the Arrhenius equation to estimate the energetic barriers associated with chromatin state transitions. We apply these metrics to various aging-related datasets, demonstrating their sensitivity to changes in the chromatin structure and the interpretability of the resulting outputs. The intra-chromosomal gene correlation length is particularly effective in quantifying changes in RNA-seq profiles due to increased chromatin accessibility during aging (and conversely, reduced accessibility due to treatment). This metric not only accurately distinguishes cell states, but also provides insight into the direction of aging. For instance, our observations on the effects of anti-sense oligonucleotide (ASO) treatment align with the existing literature, demonstrating that ASO partially restores chromatin structure in diseased cells. They additionally quantify the more pronounced effects in HGPS compared to WRN. The barrier energy landscape further extends this capability by offering a framework for understanding the progressive degradation of the regulatory mechanisms. Together, these metrics provide robust screening tools that enhance our ability to exploit common measurements such as RNA-seq to derive new phenotypes such as chromatin dynamics on aging and disease, offering an alternative perspective that complements traditional analytical techniques and enriches our understanding of cellular states.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.14.654062v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrative analysis across metagenomic taxonomic classifiers: A case study of the gut microbiome in aging and longevity</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Karagiannis, T. T., Chen, Y., Bald, S., Tai, A., Milman, S., Andersen, S. L., Perls, T. T., Segre, D., Sebastiani, P., Short, M. I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We compare inferences from two broadly used classifiers, MetaPhlAn4 and Kraken2, applied to stool metagenomic samples from participants in the Integrative Longevity Omics study to measure associations of taxonomic diversity and relative abundance with age, replicating analyses in an independent cohort.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Despite calls for the development of consensus methods, most analyses of shotgun metagenomics data for microbiome studies use a single taxonomic classifier. In this study, we compare inferences from two broadly used classifiers, MetaPhlAn4 (marker-gene-based) and Kraken2 (k-mer-based), applied to stool metagenomic samples from participants in the Integrative Longevity Omics study to measure associations of taxonomic diversity and relative abundance with age, replicating analyses in an independent cohort. We also introduce consensus and meta-analytic approaches to compare and integrate results from multiple classifiers. While many results are consistent across the two classifiers, we find classifier-specific inferences that would be lost when using one classifier alone. When using a correlated meta-analysis approach across classifiers, differential abundance analysis captures more age-associated taxa, including 17 taxa robustly age-associated across cohorts. This study emphasizes the value of employing multiple classifiers and recommends novel approaches that facilitate the integration of results from multiple methodologies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.14.654104v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Engineered Acinetobacter baylyi ADP1-ISx cells are sensitive DNA biosensors for antibiotic resistance genes and a fungal pathogen of bats</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chuong, J., Brown, K. W., Mishler, D. M., Gifford, I., Barrick, J. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We create Acinetobacter baylyi ADP1-ISx biosensors that detect specific target DNA sequences through natural transformation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Naturally competent bacteria can be engineered into platforms for detecting environmental DNA. This capability could be used to monitor the spread of pathogens, invasive species, and resistance genes, among other applications. Here, we create Acinetobacter baylyi ADP1-ISx biosensors that detect specific target DNA sequences through natural transformation. We tested strains with DNA sensors that consisted of either a mutated antibiotic resistance gene (TEM-1 bla or nptII) or a counterselectable gene flanked by sequences from the fungus Pseudogymnoascus destructans, which causes white-nose syndrome in bats. Upon uptake of homologous DNA, recombination restored antibiotic resistance gene function or removed the counterselectable gene, enabling selection of cells that sensed the target DNA. The antibiotic resistance gene and P. destructans biosensors could detect as few as 3,000 or 5,000,000 molecules of their DNA targets, respectively, and their sensitivity was not affected by excess off-target DNA. These results demonstrate how A. baylyi can be reprogrammed into a modular platform for monitoring environmental DNA.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.17.654686v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">fMRIPrep Lifespan: Extending A Robust Pipeline for Functional MRI Preprocessing to Developmental Neuroimaging</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Goncalves, M., Moser, J., Madison, T. J., McCollum, r., Lundquist, J. T., Fayzullobekova, B., Hadera, L., Pham, H. H. N., Moore, L. A., Houghton, A. M., Conan, G., Styner, M. A., Alexopoulos, D., Smyser, C. D., Stoyell, S. M., Koirala, S., Nelson, S. M., Weldon, K. B., Lee, E., Hermosillo, R. J. M., Vizioli, L., Yacoub, E., Patel, G. H., Sanchez, J., Wengler, K., Salo, T., Satterthwaite, T. D., Elison, J. T., Markiewicz, C. J., Poldrack, R. A., Feczko, E., Esteban, O., Fair, D. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce fMRIPrep Lifespan, a standardized preprocessing workflow from childhood to senescence to include neonatal, infant, and toddler structural and functional MRI data preprocessing.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The adoption of a standardized preprocessing workflow is vital for fostering community, sharing, and reproducibility. fMRIPrep has been a critical advancement towards this end, however, it is limited in its capacity to be applied to data across the lifespan, starting from infancy. Here, we introduce fMRIPrep Lifespan, an extension of fMRIPrep that extends the standardized processing from childhood to senescence to include neonatal, infant, and toddler structural and functional MRI data preprocessing. This effort involves a NiPreps integration of 1) a workflow akin to fMRIPrep optimized for MRI data in the first years of life (previously NiBabies) and 2) upstream enhancements to the entire NiPreps suite, including multi-echo data processing, modularization of workflow components, and convergence of processing with other popular workflows (ABCD-BIDS, Human Connectome Project Pipelines). Using data from the Baby Connectome Project (participants 1-43 months of age), we demonstrate that fMRIPrep Lifespan produces high-quality outputs across a wide age range. Moving forward, the scalable, modular infrastructure of fMRIPrep Lifespan will ensure adaptability to data from birth to old age while maintaining robust and reproducible frameworks for functional MRI research across the lifespan.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.14.654069v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

