<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Thursday, 10 July 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrated Nanopore and short-read RNA sequencing identifies dysregulation of METTL3- m6A modifications in endocrine therapy- sensitive and resistant breast cancer cells.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Belinda J Petri, Kellianne M Piell, Bailey L Avila-Valdes, Christian G Stanley, Laura J Winkler, Johnny Tyler Brown, Rye Ulett, Gabriela Sanchez, Julia H Chariker, Eric C Rouchka, Carolyn M Klinge</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We tested the hypothesis that inhibition of METTL3, the methyltransferase responsible for the mRNA modification N-6 methyladenosine (m6A), alters m6A modifications and differentially regulates the abundance of mRNA transcripts in acquired endocrine therapy (ET) resistance in breast cancer.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The role of epitranscriptomic changes in the development of acquired endocrine therapy (ET)- resistance in estrogen receptor α (ER) expressing breast cancer (BC) is unknown. We tested the hypothesis that inhibition of METTL3, the methyltransferase responsible for the mRNA modification N-6 methyladenosine (m6A), alters m6A modifications and differentially regulates the abundance of mRNA transcripts in ET-sensitive MCF-7 versus resistant LCC9 ER + human BC cells. Differential m6A modifications were identified using direct mRNA sequencing (DRS) performed on five replicates for each cell line ± 1 µM STM2457, a selective METTL3 inhibitor, using Nanopore MinION long read RNA-seq. Parallel short read Illumina RNA-seq quantified differential transcript abundance in the same samples. Selected results were validated by RT-qPCR, m6A-RIP-qPCR, reporter assays, and western blot analysis. Statistical analysis combined m6Anet, a machine-learning algorithm designed to call m6A modified bases, with a generalized linear model following a binomial distribution analysis to identify significant differential m6A modification ratios (DMR). Distinct METTL3 dependent m6A modification patterns in LCC9 and MCF-7 cells were observed in differentially expressed genes (DEG) associated with ET-resistance, including EEF1A2, ACTB, FLNA, PDIA6, AMIGO2, TPT1, XBP1, and CITED4. Select results were validated in additional ET-resistant BC cell lines. m6A-RIP-RT-qPCR validated specific m6A sites. We examined the proximity of m6A sites to estrogen receptor α (ER α)-mRNA binding sites reported in MCF-7 cells. ACTB, PDIA6, and XBP1 demonstrated a short-range proximity, with m6A sites located within 100 bp of ERα binding sites, suggesting a role for m6A in influencing ERα-mRNA binding. Our work provides a framework for integrating DRS and DEG omics data. Our results suggest a role for dysregulation of m6A modifications in pathways implicated in ET resistance in BC.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40632306/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An Efficient Graph Algorithm for Diploid Local Ancestry Inference</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jafarzadeh, N., Eizenga, J. M., Paten, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present diploid sequence graphs, graphs whose paths encode pairs of haplotypes, and an efficient algorithm for creating them from directed acyclic graphs.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In this paper, we present diploid sequence graphs, graphs whose paths encode pairs of haplotypes. We describe an efficient algorithm for creating a diploid graph from a directed acyclic (haploid) sequence graph, such that the diploid graph represents all the possible pairings of haplotypes present in the sequence graph and their similarity relationships. Starting with the sequence graph, our method uses a graph decomposition approach based on an extension of the SPQR-tree to systematically identify structural patterns that reduce redundancy while preserving genetic variation. We develop a polynomial-time algorithm that parsimoniously enumerates all disjoint paths with shared endpoints in two-terminal directed acyclic graphs. In the future, we envisage that diploid graphs may enable more accurate modeling of recombination, phasing, and variation-aware alignment in diploid genomes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.05.662656v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PromptBio: A Multi-Agent AI Platform for Bioinformatics Data Analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yang, X., Shashidhar, K., Zhang, M., Gu, W., Han, B., Guo, V., Zheng, J., Lin, X., Addoni, C., Zheng, Y., Chen, J., Li, K., Wang, J., Yu, L., Wu, L., Shi, S., Wang, W., Leng, Y., Ma, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a modular AI platform for scalable, reproducible, and user-adaptable bioinformatics analysis, powered by generative AI and natural language interaction, and a multi-agent system that enables stepwise, human-in-the-loop workflows.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> PromptBio is a modular AI platform for scalable, reproducible, and user-adaptable bioinformatics analysis, powered by generative AI and natural language interaction. It supports three complementary modes of analysis designed to meet diverse research needs. PromptGenie is a multi-agent system that enables stepwise, human-in-the-loop workflows using prevalidated domain-standard tools. Within PromptGenie, specialized agents, including DataAgent, OmicsAgent, AnalysisAgent, and QAgent, collaborate to manage tasks such as data ingestion, pipeline execution, statistical analysis, and interactive summarization. DiscoverFlow provides integrated, automated workflows for large-scale multi-omics analysis, offering end-to-end execution and streamlined orchestration. ToolsGenie complements these modes by dynamically generating executable bioinformatics code for custom, user-defined analyses, enabling flexibility beyond standardized workflows. PromptGenie and DiscoverFlow leverage a suite of domain-specific tools, including Omics Tools for standardized omics pipelines, Analysis Tools for downstream statistical interpretation, and MLGenie for machine learning and multi-omics modeling. We present the design, capabilities, and validation of these components, highlight their integration into automated and customizable workflows, and discuss extensibility, monitoring, and compliance. PromptBio aims to democratize high-throughput bioinformatics through a large language model-powered, natural language understanding, workflow generation and agent orchestration.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.05.663295v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Development of a Flex-Seq SNP panel for raspberry (Rubus idaeus L.) and validation through linkage map construction and identification of QTL for several traits of agronomic importance to raspberry breeding</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Davik, J., Zucchi, P., Buti, M., Milne, L., Graham, J., Sargent, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A custom single nucleotide polymorphism (SNP) panel using the Flex-Seq genotyping platform for red raspberry (Rubus idaeus L.) .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> High-throughput and reproducible genotyping platforms are critical for advancing genetic research and breeding in horticultural crops. Here, the development and validation of a custom single nucleotide polymorphism (SNP) panel using the Flex-Seq genotyping platform for red raspberry (Rubus idaeus L.) is described. SNPs were derived from existing linkage maps and RNA-seq data, resulting in a panel of 5,639 high-confidence, bi-allelic markers distributed across the seven chromosomes of the R. idaeus Malling Jewel reference genome. The panel was used to genotype 459 red raspberry accessions including 161 individuals from a bi-parental mapping population (Parisx486), enabling the construction of high-density linkage maps and the identification of quantitative trait loci (QTL) for fruit size, leaf colour, plant vigour, and thorn density. Genome-wide association studies (GWAS) identified a major QTL for thornlessness on chromosome 4, co-locating with a candidate HOX3 gene, and multiple QTLs associated with anthocyanin biosynthesis genes for leaf colour. The SNP panel demonstrated utility for linkage mapping and trait association analyses, offering a powerful resource for marker-assisted selection and genetic improvement in red raspberry.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.05.663326v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DGAT: A Dual-Graph Attention Network for Inferring Spatial Protein Landscapes from Transcriptomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, H., Cody, B. A., Osmanbeyoglu, H. U.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present DGAT (Dual-Graph Attention Network), a deep learning framework that imputes spatial protein expression from transcriptomics-only ST data by learning RNA-protein relationships from spatial CITE-seq datasets.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Spatial transcriptomics (ST) technologies provide genome-wide mRNA profiles in tissue context but lack direct protein-level measurements, which are critical for interpreting cellular function and microenvironmental organization. We present DGAT (Dual-Graph Attention Network), a deep learning framework that imputes spatial protein expression from transcriptomics-only ST data by learning RNA-protein relationships from spatial CITE-seq datasets. DGAT constructs heterogeneous graphs integrating transcriptomic, proteomic, and spatial information, encoded using graph attention networks. Task-specific decoders reconstruct mRNA and predict protein abundance from a shared latent representation. Benchmarking across public and in-house datasets, including tonsil, breast cancer, glioblastoma, and malignant mesothelioma, demonstrates that DGAT outperforms existing methods in protein imputation accuracy. Applied to ST datasets lacking protein measurements, DGAT reveals spatially distinct cell states, immune phenotypes, and tissue architectures not evident from transcriptomics alone. DGAT enables proteome-level insights from transcriptomics-only data, bridging a critical gap in spatial omics and enhancing functional interpretation in cancer, immunology, and precision medicine.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.05.662121v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">GARN3: A coarse-grained helix centered technique for RNA 3D structures prediction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Silva, J., Cohen, J., Cordeiro, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We improved the last version of the GARN (Game Algorithms for RNa 3D sampling) technique by adding more pseudoatoms to the 3D model, and improved the scoring calculation, aiming to improve the prediction of new molecules.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The study of the prediction of three-dimensional structures of RNA (ribonucleic acids) has been increasing over the last decades, especially with advances in artificial intelligence. However, there are still many gaps. Among the known techniques, GARN (Game Algorithms for RNa 3D sampling) has demonstrated good results in molecules with long structures containing hundreds of millions of nucleotides. Nevertheless, the GARN technique also left room for improvements when considering its final 3D structure for predicted molecules, which can be more refined by presenting more elements, also known as pseudoatoms. In this study, we improved the last version of the technique GARN, which is GARN2, enhancing the visualization of the 3D model by adding more pseudoatoms to the helices in our technique GARN3. In GARN3, we also added a machine learning prediction model to improve the scoring calculation, aiming to improve the prediction of new molecules. In our tests, we demonstrated good results when comparing GARN3 with other techniques in the literature. GARN3, in the same way as GARN2, presents better results for large molecules. In our tests, the simulations with GARN3 demonstrated good results, where GARN3 predicted the majority of molecules (approximately 80% of the test set) better than the older versions of GARN.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.05.663322v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Dissecting the effect of single- and co- infection of TB and COVID-19 pathogens on the sputum microbiome</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> VP, B., Shanmugavel, A. K., Shanmugam, S., Narayanan, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> COVID-19 affects the airway microbiome of TB patients and encourages the use of co-microbial/co-pathogen profiling to personalize TB treatment to match the pathogen profile.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Tuberculosis (TB) and COVID-19 are both respiratory diseases and understanding their interaction is important for effective co-infection management. Although some studies have investigated TB and COVID-19 co-infection in terms of immune responses, microbial dysbiosis in such cases remains unexplored. In this study, we understand the interface between TB and COVID-19 by systematically inspecting the microbial composition of sputum samples collected from four groups of individuals: TB only, COVID-19 only, and both TB and COVID-19 (TBCOVID) infected patients, and uninfected group (Controls). Besides metagenomic analysis of the microbiome of these sputum samples, we also performed whole genome sequencing analysis of a subset of TB-positive samples. Different bioinformatic analyses ensured data quality and revealed significant differences in the microbial composition between Control vs. disease groups. To understand the effect of COVID-19 on TB, we compared TBCOVID vs. TB samples and observed (i) higher read counts of TB-causing bacteria in the TBCOVID group, and (ii) differential abundance of several taxa such as Capnocytophaga gingivalis, Escherichia coli, Prevotella melaninogenica and Veillonella parvula. Functional profiling with PICRUSt2 revealed significantly elevated pathways in the TBCOVID group relative to TB group, some of which are related to metabolism of pulmonary surfactant lipids (e.g., Cytidine diphosphate diacylglycerol (CDP-DAG) biosynthesis pathway with fold change of 7.46). Further clustering of these significantly elevated pathways revealed a sub-cluster of individuals with adverse treatment outcomes. Two individuals in this sub-cluster showed prevalence of respiratory pathogens like Stenotrophomonas maltophilia - knowing such information can help personalize the antibiotic treatment to match the pathogen profile of the individual. Overall, our study reveals the effect of COVID-19 in the airway microbiome of TB patients, and encourages the use of co-microbial/co-pathogen profiling to personalize TB treatment.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.08.661948v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">GREmLN: A Cellular Regulatory Network-Aware Transcriptomics Foundation Model</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, M., Swamy, V., Cassius, R., Dupire, L., Kanatsoulis, C., Paull, E., AlQuraishi, M., Karaletsos, T., Califano, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a graph structure-based model that leverages graph signal processing to embed gene-regulatory network structure directly within its attention mechanism, producing biologically informed single cell specific gene embeddings.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The ever-increasing availability of large-scale single-cell profiles presents an opportunity to develop foundation models to capture cell properties and behavior. However, standard language models such as transformers are best suited for sequentially structured data with well defined absolute or relative positional relationships, while single cell RNA data have orderless gene features. Molecular-interaction graphs, such as gene regulatory networks (GRN) or protein-protein interaction (PPI) networks, offer graph structure-based models that effectively encode both non-local gene-gene dependencies, as well as potential causal relationships. We introduce GREmLN, a foundation model that leverages graph signal processing to embed gene-regulatory network structure directly within its attention mechanism, producing biologically informed single cell specific gene embeddings. Our model faithfully captures transcriptomics landscapes and achieves superior performance relative to state-of-the-art baselines on both cell type annotation and graph structure understanding tasks. It offers a unified and interpretable framework for learning high-capacity foundational representations that capture complex, long-range regulatory dependencies from high-dimensional single-cell transcriptomic data. Moreover, the incorporation of graph-structured inductive biases enables more parameter-efficient architectures and accelerates training convergence.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.03.663009v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Robust regulatory interplay of enhancers, facilitators, and promoters in a native chromatin context</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhou, Z., Li, A., Zhang, J., Yu, H., Ozer, A., Lis, J. T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel recombinase-mediated genome rewriting platform and applied our divergent transcription architectural model to understand how a long-range human enhancer confers a remarkable 10,000-fold activation to its target gene, NMU, at its native locus.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Enhancers are gene-distal cis-regulatory elements that drive cell type-specific gene expression. While significant progress has been made in identifying enhancers and characterizing their epigenomic features, much less effort has been devoted to elucidating mechanistic interactions among clusters of functionally linked regulatory elements within their endogenous chromatin contexts. Here, we developed a novel recombinase-mediated genome rewriting platform and applied our divergent transcription architectural model to understand how a long-range human enhancer confers a remarkable 10,000-fold activation to its target gene, NMU, at its native locus. Our systematic dissection reveals transcription factor synergy at this enhancer and highlights the interplay between a divergently transcribed core enhancer unit and emerging new types of cis-regulatory elements--notably, intrinsically inactive facilitators that augment and buffer core enhancer activity, and an adjacent retroviral long terminal repeat promoter that represses enhancer activity. We discuss the broader implications of our focused study on enhancer mechanisms and regulation genome-wide.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.07.663560v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Nissolia brasiliensis as a non-nodulating model legume</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Girou, C., Keller, J., Libourel, C., van Beveren, F., Bianconi, M., Dufau, I., Cravero, C., Callot, C., Rodde, N., Coriton, O., Huteau, V., Delaux, P.-M., Vernie, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We describe the first chromosome-level assembly for a non-RNS-forming legume, the tropical papilionoid Nissolia brasiliensis, and develop a stable transformation protocol that can be deployed in the future to re-evolve RNS in legumes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The nitrogen-fixing root nodule symbiosis (RNS) is specifically formed by four orders of angiosperms. The largest of these four orders include the legume family, the Fabaceae. Among legumes, historical model species have emerged, such as the RNS-forming Medicago truncatula and Lotus japonicus, or, more recently, Aeschynomene aevenia. By contrast, legume species that have lost RNS have been largely ignored. Here, we describe the first chromosome-level assembly for a non-RNS-forming legume, the tropical papilionoid Nissolia brasiliensis. We compared its genome to closely related legumes and identified genes associated with RNS. Finally, we developed a stable transformation protocol that can be deployed in the future to re-evolve RNS in legumes, a first step toward the goal of engineering RNS in non-legume crops.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.08.663655v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Enhancement of Arabidopsis growth by Enterobacter sp. SA187 under elevated CO2 is dependent on ethylene signalling activation and primary metabolism reprogramming</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ilyas, A., Mauve, C., Pateyron, S., Paysant-Le Roux, C., Bigeard, J., Hodges, M., de Zelicourt, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> SA187, a plant growth-promoting bacterium, in enhancing growth and development of Arabidopsis thaliana under elevated atmospheric CO2 conditions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> As atmospheric CO2 levels continue to increase, optimizing the CO2 fertilization effect which often falls short of its potential due to the physiological and metabolic limitations of plants becomes crucial. This study investigates the role of Enterobacter sp. SA187 (SA187), a plant growth-promoting bacterium, in enhancing growth and development of Arabidopsis thaliana under elevated atmospheric CO2 (eCO2) conditions. While SA187 inoculation did not have major effects under ambient CO2, it was found to significantly enhance root and shoot biomass, and to increase N- and reduce C-contents under eCO2. Moreover, transcriptomics and metabolomics suggested that SA187 modulated phytohormonal homeostasis, with activation of the salicylic acid, jasmonic acid and ethylene signalling pathways, and increased primary metabolism including the TCA cycle, N and carbohydrate metabolisms. Finally, the growth-promoting effects of SA187 were shown to be mediated through ethylene-dependent pathways, as evidenced with the ethylene-insensitive mutant ein2-1 which did not show similar benefits in plant fresh weight and altered gene expression. This beneficial plant-microbe interaction under eCO2 in a non-leguminous plant highlights a novel aspect of microbial influence on plant physiology in the context of climate change. These insights underscore the potential of utilizing SA187 to enhance plant performance and adaptability in future high CO2 environments, providing a sustainable approach to agricultural productivity as global CO2 levels increase.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.08.663752v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Bigger is not always better: Optimizing leaf area index with narrow leaf shape in soybean</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tamang, B., Bernard, G., Bernacchi, C., Diers, B., Ainsworth, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed 204 near-isogenic soybean lines differing in leaf morphology by introgressing narrow-leaf alleles from donor parents PI 612713A and PI 547745 into the elite, broad-leaf cultivar LD11-2170.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Modern soybean varieties have higher than optimal leaf area index (LAI), which could divert resources from reproductive growth. Altering leaf shape could be a simple strategy to reduce LAI. To test this, we developed 204 near-isogenic soybean lines differing in leaf morphology by introgressing narrow-leaf alleles from donor parents PI 612713A and PI 547745 into the elite, broad-leaf cultivar LD11-2170. We evaluated the lines across two locations and two row spacings (38-cm and 76-cm) to assess how reduced investment in leaf area influences canopy architecture, crop physiology, and yield. Narrow-leaf lines showed 13% lower peak LAI and 3% lower digital biomass compared to broad-leaf counterparts yet maintained yield parity (5,756 vs. 5.801 kg ha-1, p = 0.43) across environmental conditions. Photosynthetic capacity remained largely unchanged, with narrow-leaf lines showing modest increases in electron transport rate and leaf mass per area. Narrow-leaf lines achieved similar canopy closure timing despite lower LAI, suggesting architectural compensation mechanisms. The most striking difference appeared in seed packaging, with narrow-leaf lines producing 34% four-seeded pods compared to only 1.8% in broad-leaf lines. There was a nonlinear relationship between peak LAI and yield, with optimal LAI values of 9-11 varying by environment. These findings show that the single-gene GmJAG1-controlled narrow-leaf trait offers a tractable strategy for reducing LAI and maintaining high productivity. This could reduce the metabolic costs associated with excessive canopy development and support sustainable agriculture under increasing climate variability.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.07.663573v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A coupled fluorescence assay for high-throughput screening of polyurethane-degrading enzymes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> yang, l., DENG, X., LIU, R., LI, J., ZHAI, J., MA, X., LIN, A. K. L., SHAO, Z., LU, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a novel, high-throughput enzyme screening assay for PU degradation, which is capable of unambiguous structure-activity analysis and direct quantification of degradation products (adipic acid and ethylene glycol).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The global accumulation of plastic waste, particularly persistent polymers like polyurethane (PU), demands urgent solutions. Enzymatic depolymerization offers a viable strategy for PU waste valorization. However, progress has been hindered by the lack of reliable high-throughput screening (HTS) assays capable of precise and quantitative evaluation of enzymatic activity. Current enzyme screening assays face significant limitations due to poor substrate relevance and inadequate quantification methods. Here, we present a novel HTS assay featuring a chemically defined, synthetic poly (ethylene adipate)-based PU substrate that enables unambiguous structure-activity analysis and direct quantification of degradation products (adipic acid and ethylene glycol). Coupled with this substrate is a highly sensitive fluorescence-based detection cascade, in which released ethylene glycol is stoichiometrically converted to resorufin via a two-enzyme system (glycerol dehydrogenase and diaphorase). This assay overcomes key limitations of commercial substrates (e.g., Impranil DLN) by providing quantitative, real-time monitoring of PU hydrolysis with molecular precision. Validation with known PU-degrading enzymes demonstrated that the assay can sensitively distinguish differences in enzymatic activity with high reproducibility and quantitative accuracy. Our platform enables rapid, cost-effective screening of enzyme libraries and engineered variants, significantly advancing enzyme discovery and optimization efforts. By bridging the gap between laboratory research and industrial application, this HTS assay accelerates the development of sustainable PU recycling solutions, offering a critical tool against plastic pollution.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.09.663332v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Derivation of cardiomyocyte-propelled motile aggregates from stem cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ho, C., Glykofrydis, F., Godage, G., Poon, K., Kunnan, M., Swedlund, B., Murillo, S., Morsut, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that muscle-powered biobots can be built from stem cells in a self-organized building process, and show that a novel protocol for cardiogenic gastruloid induction yields contractile aggregates with increased motility.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Robotics draws inspiration from biology, particularly animal locomotion based on muscle-driven contractions. While traditional engineering assembles components sequentially, locomotive animals are built via self-organized developmental programs. Stem cells, under the right conditions, can mimic these processes in vitro, offering a pathway to develop muscle-propelled biobots in a self-organized building process. Here, we demonstrate that existent cardiogenic gastruloid protocols can produce motile aggregates from mouse embryonic stem cells, although with very limited efficiency. We then identify a novel protocol that yields contractile aggregates with higher frequency and larger contractile areas. In this novel protocol, mesendoderm induction using TGF-beta ligands is followed by cardiogenic induction with FGFs and VEGF. Synthetic organizers further control contraction localization. Aggregates developed via this protocol show enhanced motility, marking a step forward towards building motile cardiobots from self-organized biological material. This strategy opens new possibilities for designing autonomous biobots and studying the evolution of muscle-powered movement of multicellular organisms and cardiovascular development.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.09.663178v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Engineering a human-based translational activator for targeted protein expression restoration</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sinnott, R., Solanki, A., Govind, A. P., Green, W., Dickinson, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify a compact (601 amino acid) CIRTS translational activator (CIRTS-4GT3), that can drive targeted, sustained translation increases up to 100% from three endogenous transcripts relevant to epilepsy and neurodevelopmental disorders.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Therapeutic modalities to programmably increase protein production are in critical need to address diseases caused by deficient gene expression via haploinsufficiency. Restoring physiological protein levels by increasing translation of their cognate mRNA would be an advantageous approach to correct gene expression, but has not been evaluated in an in vivo disease model. Here, we investigated if a translational activator could improve phenotype in a Dravet syndrome mouse model, a severe developmental and epileptic encephalopathy caused by SCN1a haploinsufficiency, by increasing translation of the SCN1a mRNA. We identifiy and engineere human proteins capable of increasing mRNA translation using the CRISPR-Cas Inspired RNA-targeting System (CIRTS) platform to enable programmable, guide RNA (gRNA)-directed translational activation with entirely engineered human proteins. We identify a compact (601 amino acid) CIRTS translational activator (CIRTS-4GT3), that can drive targeted, sustained translation increases up to 100% from three endogenous transcripts relevant to epilepsy and neurodevelopmental disorders. AAV-delivery of CIRTS-4GT3 targeting SCN1a mRNA to a Dravet syndrome mouse model led to increased SCN1a translation and improved survivability and seizure threshold - key phenotypic indicators of Dravet syndrome. This work validates a new strategy to address SCN1a haploinsufficiency and emphasizes the preclinical potential translational activation has to address neurological haploinsufficiency.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.09.663984v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Bioelectrochemical Crossbar Architecture Screening Platform (BiCASP) for Extracellular Electron Transfer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Suresh, H., Bird, P., Saha, K., Devaraj, S. V., Asci, C., Truong, A., Wallman, C., Patel, R. S., Sharma, A., Carpenter, M., Bontapalle, S., Verduzco, R., Silberg, J., Sonkusale, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We develop a cross-bar architecture screening platform for screening engineered biological components in cells, which allows for samples to be arrayed and characterized in individually addressable microwells.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Electroactive microbes can be used as components in electrical devices to leverage their unique behavior for biotechnology, but they remain challenging to engineer because the bioelectrochemical systems (BES) used for characterization are low-throughput. To overcome this challenge, we describe the development of the Bioelectrochemical Crossbar Architecture Screening Platform (BiCASP), which allows for samples to be arrayed and characterized in individually addressable microwells. This device reliably reports on the current generated by electroactive bacteria on the minute time scale, decreasing the time for data acquisition by several orders of magnitude compared to conventional BES. Also, this device increased the throughput of screening engineered biological components in cells, quickly identifying mutants of the membrane protein wire MtrA in Shewanella oneidensis that retain the ability to support extracellular electron transfer (EET). BiCASP is expected to enable the design of new components for bioelectronics by supporting directed evolution of electroactive proteins.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.09.663982v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Resolving Emergent Oscillations in Gene Circuits with a Growth-Coupled Model</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Namboothiri, H. R., Pandey, A., Hu, C. Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A dual-scale model for predicting emergent behaviors in synthetic gene circuits that accurately reproduces oscillations in growth phase dynamics .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Synthetic gene circuits often behave unpredictably in batch cultures, where shifting physiological states are rarely accounted for in conventional models. Here, we report the emergence of oscillatory expression dynamics in degradation-tagged protein reporters, an unexpected phenomenon that contradicts predictions from standard single-scale models. We resolve this discrepancy by developing GEAGS (Gene Expression Across Growth Stages), a dual-scale modeling framework that explicitly couples intracellular gene expression to logistic population growth. Using a chemical reaction network (CRN) model with growth-phase-dependent rate-modifying functions, GEAGS accurately reproduces the observed oscillations and identifies amino acid recycling and growth phase transition as key drivers. We reduce the model to an effective form for practical use and demonstrate its adaptability by applying it to layered feedback circuits, resolving longstanding mismatches between model predictions and measured dynamics. These results establish GEAGS as a generalizable platform for predicting emergent behaviors in synthetic gene circuits and underscore the importance of multiscale modeling for robust circuit design in dynamic environments.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.09.663992v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

