<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Friday, 29 August 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Evolutionary simulations reveal role for genomic recombination in the evolution of gene regulatory network complexity and robustness</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chapel, M. E., de Boer, C. G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The gene regulatory networks (GRNs) of eukaryotes are dramatically more complex than the GRNs of prokaryotes, but we lack a complete picture of the selective pressures that have shaped this difference. Here, we use a biochemically informed model of gene regulation to simulate GRN evolution and explore the role that reproductive strategy plays in shaping regulatory complexity. We find that recombining and non-recombining populations converge to the same level of complexity, even in the absence of selection. However, recombination modifies the rate at which complexity emerges, accelerating convergence to the complexity plateau in changing environments while slowing the process in static environments. Our results suggest that, rather than being under direct selection, regulatory complexity may emerge as a byproduct of other evolutionary processes. These results highlight how reproductive strategy and environmental change interact to influence evolutionary trajectories.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.28.672878v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">End-to-end proteogenomics for discovery of cryptic and non-canonical cancer proteoforms using long-read transcriptomics and multi-dimensional proteomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kulej, K., Preska Steinberg, A., Zhang, J., Casalena, G., Havasov, E., Shah, S. P., McPherson, A., Kentsis, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Modern mass spectrometry now permits genome-scale quantitative measurements of biological proteomes. However, current approaches remain hindered by incomplete representation of biological variability of protein sequences in canonical reference proteomes, limiting studies to already annotated proteoforms. This, combined with reliance on tryptic peptides and limited sample complexity reduction techniques render detection of variant proteoforms missing from current reference annotations virtually impossible. Here, we report a method for end-to-end proteogenomics which integrates (i) sample-specific transcriptome assembly using long-read sequencing with (ii) multi-protease, multi-dimensional (MPMD) trapped ion mobility spectrometry (TIMS) parallel accumulation-serial fragmentation (PASEF) mass spectrometry. Termed ProteomeGenerator3, we benchmark its sensitivity and specificity for unbiased human cancer proteome discovery. Using long-read transcriptome sequencing of human Ewing sarcoma cancer cells, we detected over 40,000 full-length transcript isoforms, including aberrantly expressed neogenes, splice isoforms, and gene fusions. Combined with MPMD-TIMS-PASEF mass spectrometry, our proteogenomics workflow enabled the detection of 12,189 unique proteoforms with an average sequence coverage exceeding 50%, including 3,216 putative, non-canonical proteoforms, constituting one of the most complete human proteomes to date. Likewise, we report the proteogenomic performance metrics for current state-of-the-art implementations of DIA-NN, Spectronaut and diaTracer-MSFragger algorithms for data-independent acquisition (DIA) and PASEF mass spectral data, including optimal Double Rainbow-diaPASEF parameters for non-tryptic proteomes. In all, end-to-end integrative ProteomeGenerator3 provides a versatile platform for deep sample-specific proteogenomic discovery, suitable for the identification of disease biomarkers and therapeutic targets, and is available open-source from https://github.com/kentsislab/proteomegenerator3.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.23.671943v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Leveraging Large Language Models for Redundancy-Aware Pathway Analysis and Deep Biological Interpretation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ge, Y., Zhang, F., Liu, Y., Jiang, C., Gao, P., Tan, N. S., Zhang, S., Shen, Y., Zhou, Q., Zhou, X., Wang, C., Shen, X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Extracting coherent, biologically meaningful insights from vast, complex multi-omics data remains challenging. Currently, pathway enrichment analysis serves as a cornerstone for the functional interpretation of such data. However, conventional approaches often suffer from extensive functional redundancy caused by shared molecular components and overlapping pathway definitions across databases. This redundancy can obscure key biological signals and compromise the interpretability of pathway enrichment results. Here, we present MAPA (Functional Module Identification and Annotation for Pathway Analysis Results Using Large Language Models [LLM]), an open-source computational framework that resolves redundancy and enhances pathway analysis result interpretation. MAPA computes functional similarity between pathways using LLM-based text embeddings, enabling comparison across different databases. It constructs pathway similarity networks and identifies functional modules via community detection algorithms. Crucially, MAPA employs LLMs for automated functional annotation, integrating Retrieval-Augmented Generation (RAG) to generate comprehensive and real-time biological summaries and reduce hallucinations. Benchmarking demonstrated MAPA's superior performance: the biotext embedding similarity showed a large effect size (Cliff's {delta} = 0.96) compared with the Jaccard index ({delta} = 0.73), and module identification achieved high accuracy (Adjusted Rand Index [ARI] = 0.95) versus existing methods (ARI = 0.23-0.33). Human expert evaluation confirmed that MAPA's annotations match expert-quality interpretations. Finally, a multi-omics aging case study illustrates that MAPA uncovers coherent functional modules and generates insights extending beyond conventional pathway analyses. Collectively, MAPA represents a significant advance in redundancy-aware pathway analysis, transforming pathway enrichment results from fragmented lists into biologically coherent narratives. By leveraging the capabilities of LLMs, MAPA offers researchers a robust, scalable tool for deriving deep mechanistic insights from complex and vast multi-omics datasets, marking a new direction for AI-driven bioinformatics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.23.671949v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Compact Matrix Representation for Hydrocarbon and Its Applications on Monoterpene Carbocation Rearrangement Elementary Steps</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tse, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> This work proposes a compact matrix-based representation for hydrocarbon carbocations. In addition, its application is explored for the elementary steps of non-stereoisomeric monoterpene backbone.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.23.671925v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Transcriptome Universal Single-isoform COntrol: A Framework for Evaluating Transcriptome reconstruction Quality</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liu, T., Paniagua, A., Jetzinger, F., Ferrandez-Peral, L., Frankish, A., Conesa, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Long-read sequencing (LRS) platforms, such as Oxford Nanopore and Pacific Biosciences, enable comprehensive transcriptome analysis but face challenges such as sequencing errors, sample quality variability, and library preparation biases. Current benchmarking approaches address these issues insufficiently: BUSCO assesses transcriptome completeness using conserved single-copy orthologs but can misinterpret alternative splicing as gene duplications, while spike-ins (SIRVs, ERCCs) oversimplify real-sample complexity, neglecting RNA degradation and RNA extraction artifacts, thus inflating performance metrics. Simulation algorithms are limited to recapitulate this complexity. To overcome these limitations, we introduce the Transcriptome Universal Single-isoform Control (TUSCO), a curated internal reference set of genes lacking alternative isoforms. TUSCO evaluates precision by identifying transcripts deviating from reference annotations and assesses sensitivity by verifying detection completeness in human and mouse samples. Masking TUSCO transcripts, and optionally inserting decoy splice variants, creates a novel-isoform challenge that assesses recovery of the true, now-unannotated isoforms. Our validation demonstrates that TUSCO provides accurate and reliable benchmarking without external controls, significantly improving quality control standards for transcriptome reconstruction using LRS.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.23.671926v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Estimating changes in RNA Polymerase II elongation and intron splicing speeds from total RNA-seq data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Koubele, J., Beyer, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We present a computational method to jointly estimate changes in the elongation speed of RNA Polymerase II, and the speed of intron splicing, using total RNA-seq. An implementation of our method is available at https://github.com/jkoubele/pol-ii-speed.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.24.672013v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Chromatin state dynamics during the Plasmodium falciparum intraerythrocytic development cycle</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Brown, A. S., Llinas, M., Mahony, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The interdependence of chromatin states and transcription factor (TF) binding in eukaryotic genomes is critical for the proper regulation of gene expression. In this study, we explore the connection between TFs and chromatin states in the human malaria parasite, Plasmodium falciparum, throughout its 48-hour asexual intraerythrocytic developmental cycle (IDC). Most P. falciparum genes are expressed in a periodic manner during the IDC, accompanied by dynamic shifts in histone modifications and chromatin accessibility. Leveraging genome-wide profiles of chromatin accessibility, histone modifications, and Heterochromatin Protein 1 (HP1) occupancy, we characterize chromatin state dynamics during the IDC. Our results indicate that several chromatin states remain stable throughout the lifecycle, while others are dynamic and are linked to gene activation or repression. We further characterize chromatin state dynamics at the genome-wide DNA binding sites for a selection of Plasmodium TFs, allowing us to group TFs according to their chromatin preferences. By correlating changes in chromatin accessibility, histone modifications, and TF binding, we provide a global overview of the chromatin state dynamics that coordinate P. falciparum asexual blood stage development.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.22.671872v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Artificial Intelligence for Genomics and Bioinformatics: Is Africa Lagging Behind?</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tessema, S. K., Tanui, C., Onywera, H., Christoffels, A. G., Dereje, N., Fallah, M. P., Kebede, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Africa's diverse pathogen and human genetics present significant opportunities for genomic medicine research using artificial intelligence (AI). We surveyed 250 genomics researchers across 46 of the 55 African Union Member States to assess the state of AI use. The survey revealed that only 4% of respondents reported high familiarity with the application of AI for genomics and bioinformatics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.27.668885v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Computational Analysis of ELOVL6 Structure and Inhibition for Rational Drug Design</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> G Ibarluzea, M., Ramis, R., Gil-Bea, F. J., Gerenu, G., Lopez de Munain, A., Aizpurua, J. M., Miranda, J. I., Bergara, A., Leonardo, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> ELOVL6 is a key enzyme in long-chain fatty acid elongation, catalyzing the conversion of C16 fatty acids into C18 fatty acids. While its role in lipid metabolism is well established, recent studies have linked ELOVL6 to metabolic and neurodegenerative diseases, making it an attractive therapeutic target. However, the absence of a resolved crystal structure and limited mechanistic understanding of its inhibition pose significant challenges for drug discovery. In this study, we employ a multi-tiered computational approach, including structure prediction, molecular dynamics (MD) simulations, and free energy calculations, to investigate the structural basis of ELOVL6 function and inhibition. We identify the most thermodynamically favorable substrate binding pathway and characterize key conformational changes associated with ligand binding. By analyzing potential inhibitor binding pockets, we determine that known inhibitors preferentially target the active site, and we validate their binding affinities against experimental data. Additionally, by comparing ELOVL6 with homologous elongases, we pinpoint potentially key amino acid residues responsible for selectivity, providing insights that could guide structure-based drug design. Our findings establish a mechanistic framework for rational inhibitor development, offering a foundation for future efforts in optimizing ELOVL6-targeting therapeutics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.23.671927v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">TCP3-mediated regulation of cell expansion in Arabidopsis thaliana</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Koyama, T., Kunieda, T., Toyonaga, H., Nobuhara, M., Mitsuda, N., Soga, K., Ishida, J., Seki, M., Takahashi, K., Kinoshita, T., Bessho, A., Demura, T., Ohme-Takagi, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cell expansion is crucial for organ morphogenesis in multicellular organisms. Apoplast acidification triggers plant cell expansion. Plant hormones and transcription factors such as TEOSINTE BRANCHED, CYCLOIDEA, and PROLIFERATING CELL NUCLEAR ANTIGEN BINDING FACTORs (TCPs) control cell expansion. However, details regarding the regulatory mechanism of cell expansion for organ morphogenesis remain unclear. In this study, we used molecular, biochemical, cellular, genetic, atomic force microscopy, and tensile testing analyses and showed that miR319-targeted TCPs integrated cell expansion with organ morphogenesis in Arabidopsis thaliana. We found that TCPs directly induce the expression of genes encoding cell wall loosening proteins and SMALL AUXIN UP RNAs (SAURs), activators of plasma membrane-localized H+-ATPases. TCP-mediated activation of plasma membrane-localized H+-ATPases stimulates apoplast acidification, reduce cell stiffness, promote cell expansion, and thus exaggerate elongation of the hypocotyl. Ectopic expression of a SAUR gene in sextuple tcp mutant plants substantially recovered the hypocotyl morphology of the tcp mutant, providing genetic evidence of TCP-mediated SAUR regulation. Collectively, our data show that TCPs regulate apoplast acidification for cell expansion.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.24.672037v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Single-cell-level response to drought in Sorghum bicolor reveals novel targets for improving water use efficiency</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Stata, M., Greenblum, S., Yoshinaga, Y., Koriabine, M., Keymanesh, K., Karia, P., Zhao, C., O'Malley, R. C., Rhee, S. Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Increasing drought threatens global agriculture, especially in water-limited regions. Sorghum bicolor, a drought-tolerant C4 grass, is a promising bioenergy crop for cultivation on marginal lands, yet its molecular drought responses remain poorly understood. To uncover single-cell-level transcriptional responses to drought, we performed single-nucleus RNA sequencing on mature sorghum leaves under well-watered and drought conditions. We identified major cell types and analyzed differential gene expression across mesophyll, bundle sheath, epidermal, vascular, and stomatal cells. Surprisingly, drought effects on transcriptomes exceeded differences due to cell identity, revealing a shared response across cell types. We leveraged this convergence to identify candidate regulators of drought-responsive gene expression. These findings advance our understanding of sorghum drought adaptation and offer new targets for engineering enhanced water use efficiency in bioenergy crops.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.28.671794v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PAM18-3, a J-domain protein, maintains mitochondrial integrity and plant growth and development in Arabidopsis thaliana</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sharma, N., S, S., Sahi, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mitochondrial protein import is indispensable for organelle biogenesis and function, and is powered by the evolutionarily conserved presequence translocase-associated motor (PAM) complex. In Arabidopsis thaliana, three paralogs: PAM18-1, PAM18-2, and PAM18-3, encode J-domain proteins homologous to yeast PAM18, which stimulates the ATPase activity of mitochondrial HSP70 (mtHSP70) during protein translocation. Here, we identify PAM18-3 as the most highly and ubiquitously expressed paralog and demonstrate its critical role in mitochondrial function and plant development. Genetic disruption of PAM18-3 caused severe vegetative and reproductive defects, including reduced root length, smaller rosette size with fewer leaves, decreased plant height, shorter siliques, reduced seed set, and increased seed abortion. These phenotypes were fully rescued in complemented lines expressing PAM18-3. Ultrastructural analyses revealed profound mitochondrial abnormalities in mutants, whereas chloroplast architecture remained unaffected. Functional assays showed reduced mitochondrial membrane potential, and altered respiratory flux with a compensatory induction of the alternative oxidase (AOX) pathway. Transcript profiling revealed upregulation of AOX genes and multiple components of the mitochondrial TIM23 import apparatus and associated chaperones. Import assays demonstrated reduced mitochondrial accumulation of canonical TIM23 substrates, including IDH, ATP{beta}, and SHMT1, confirming a defect in matrix protein translocation. Consistently, pam18-3 mutants accumulated elevated reactive oxygen species (ROS) and exhibited strong induction of mitochondrial dysfunction stimulon (MDS) genes, including key transcription factors mediating retrograde signaling. Together, our findings establish PAM18-3 as a central component of the mitochondrial protein import machinery, supporting plant growth and development in Arabidopsis thaliana.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.25.671915v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Exploration of mRNA-sized RNA import into Saccharomyces cerevisiae mitochondria by a combined synthetic biology and adaptive laboratory evolution approach</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Koster, C., Kohabir, K., den Ridder, M., Luttik, M., de Hulster, E., Pabst, M., Daran-Lapujade, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Efficient gene integration using RNA-guided endonucleases has not yet been achieved in the mitochondrial genome. Import of nucleic acids into mitochondria, a controversial feature, is essential for implementation of Cas9-mediated genome engineering of mitochondria. Import of short RNAs naturally occurs in mitochondria, and several putative import mechanisms and determinants have been proposed. However to date, import of gene-length RNA, required for gene integration in the mitochondrial genome, has never been described. The goal of this study was to devise and test experimental strategies to detect and improve the import of mRNA-sized RNA in mitochondria, using Saccharomyces cerevisiae as model. A first fluorescence-based screening approach, relying on mitochondrial import of a fluorescent protein encoding mRNA was analyzed by fluorescence measurements, western blot and mRNA-FISH. Confounding results obtained with these different techniques made it difficult to unambiguously conclude on the occurrence of import of mRNA-sized RNAs into mitochondria. An adaptive laboratory evolution (ALE) approach, imposing a strong selection pressure for mRNA import to mitochondria, was then designed and tested to improve mitochondrial mRNA import. While the ALE approach did not improve mitochondrial mRNA import in the present study, it is a promising, unambiguous method for future studies testing different RNAs or mutants. The present study highlights remaining challenges in analytical techniques to identify RNA import to mitochondria, and introduces a novel application of ALE for studies on mitochondrial import of short and long RNA species.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.28.672929v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">APDeeM: A machine Learning strategy towards Effective Peptide Vaccine Candidates Identification against Different Types of Viruses</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hossain, M. U., Alom, M. R., Hasan, S. S., Sakib, M. N., Suchi, M. A., Sanjida, Z., Rahman, A. B. Z. N., Bhattacharjee, A., Chowdhury, Z. M., Ahammad, I., Rahman, M. A., Azad, S., Salimullah, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Viral infections pose significant global health challenges, underscoring the urgent need for improved medications. Nevertheless, traditional medicinal approaches depend significantly on labor-intensive laboratory tests, which impede efficient identification and prolong vaccine development, particularly when screening a huge number of samples. To address these obstacles, we present a comprehensive Antiviral Peptide (AVP) Detection Dataset, comprising 14 unique features to improve the characterization of antiviral and non-antiviral peptides. Subsequently, we introduce the Antiviral Peptide detection enhanced by Ensemble Machine Learning (APDeeM) system. This advanced computational framework considerably reduces the time required for AVP detection by utilizing ensemble learning methodologies. The APDeeM system incorporates Gradient Boosting, Random Forest, K-Nearest Neighbors (KNN), and AdaBoost algorithms to facilitate the swift selection of AVP candidates without requiring urgent laboratory testing. Our proposed ensemble methodology showed superior performance, with an accuracy of 85.99%, F1 score of 87.60%, recall of 88.91%, and precision of 86.32%, exceeding the efficacy of all tested antiviral peptide prediction models in this research. The APDeeM approach signifies a substantial improvement over conventional detection techniques, expediting the identification of prospective vaccine candidates and facilitating the advancement of more effective antiviral peptides. The most promising AVP candidates may urge laboratory validation, optimize resources, and accelerate vaccine development.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.25.671769v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Covalent Inhibition of the Human Papillomavirus Type 16 E6 Protein Restores p53 and Suppresses HPV-Driven Tumorigenesis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rietz, A., Kumari, L., Koirala, A., Pelletier, S., Lu, Z., Androphy, E. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> High-risk human papillomavirus (HPV) infections are the etiology of approximately 5% of all cancers worldwide, including cervical, anal, and oropharyngeal malignancies. HPV E6 is a multifunctional oncoprotein that drives tumorigenesis and is best known for bridging the ubiquitin ligase E6AP (UBE3A) and p53 into a complex that leads to proteasome mediated destruction of p53. We developed small molecule inhibitors that covalently bind to cysteine-51 (Cys-51) in HPV16 E6. In HPV16-positive cancer cells, these compounds increase p53 protein levels and activate p53-dependent transcriptional programs associated with apoptosis and senescence, resulting in reduced tumor cell viability. In vivo, E6 inhibition suppresses the growth of human HPV16 expressing cervical and oropharyngeal tumor cell lines in mice. The strategy of targeting a viral oncoprotein with a covalent inhibitor demonstrates a genotype-specific therapeutic strategy for HPV-associated cancers and premalignant infections, addressing a significant unmet need in current treatment options.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.24.671874v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">EIF3H Regulates ERK-Driven Oncogenic Signaling in Breast Cancer Metastasis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bandyopadhyaya, S., Patel, J., Karyala, P., Agrawal, H., Tripathi, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Breast cancer remains a leading cause of cancer-related mortality among women, with metastasis being the primary driver of poor prognosis. The ubiquitin proteasome system (UPS) is a central regulator of protein homeostasis, and its dysregulation is associated with multiple cancers. Within this system, deubiquitinating enzymes (DUBs), which remove ubiquitin moieties from target proteins and thereby modulate their stability and function, have emerged as attractive therapeutic targets. Eukaryotic initiation factor 3 subunit H (EIF3H), a JAMM family DUB, is overexpressed in multiple cancers and implicated in stabilizing oncogenic proteins. Using clinical transcriptomic datasets, we identified EIF3H as significantly upregulated in breast invasive carcinoma, with high expression correlating with poor patient outcomes. Functional assays demonstrated that EIF3H overexpression enhances proliferation, migration, and invasion of breast cancer cells, whereas its knockdown suppresses these traits. Mechanistically, EIF3H physically interacts with and deubiquitinates phosphorylated ERK (pERK), preventing its degradation and sustaining MAPK pathway activation. This represents the first report of pERK as a direct EIF3H substrate, revealing a novel mechanism linking EIF3H to metastatic progression. Moreover, EIF3H deficient cells display increased sensitivity to chemotherapeutic drugs, suggesting that pharmacological inhibition of EIF3H may simultaneously impair metastasis and improve therapeutic efficacy. Collectively, our findings identify EIF3H as a potential therapeutic target for combating metastatic breast cancer.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.24.672041v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ESPWA: a deep learning-enabled tool for precision-based use of endocrine therapy in resource-limited settings</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pulido-Arias, D., Henderson, R., Millien, C., Lormil, J., Jose, M. D., Flambert, G., Bontemps, J., Georges, E., Gunturi, A., Shah, P., Goncalves, T., Kalpathy-Cramer, J., Gerstner, E., Wander, S., Sirintrapun, S. J., Sgroi, D., Jeronimo, J., Castle, P. E., Landgraf, K., Brown, A., Fadelu, T., Shulman, L. N., Guttag, J., Milner, D., Brock, J., Bridge, C. P., Kim, A. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;">Background: Cancer morbidity disproportionately affects patients in low- and middle-income countries (LMICs), where timely and accurate tumor profiling is often nonexistent. Immunohistochemistry-based assessment of estrogen receptor (ER) status, a critical step to guide use of endocrine therapy (ET) in breast cancer, is often delayed or unavailable. As a result, ET is often prescribed empirically, leading to ineffective and toxic treatment for ER-negative patients. To address this unmet need, we developed ESPWA (Estrogen Receptor Status Prediction for Haitian patients using deep learning-enabled histopathology Whole Slide Imaging Analysis), a deep-learning (DL) model that predicts ER status directly from hematoxylin-and-eosin (H&E)-stained whole slide images (WSIs). Methods: We curated two cohorts of H&E WSIs with tissue-matched ER status: The Cancer Genome Atlas (TCGA, n = 1085) and Zanmi Lasante (ZL, n = 3448) from Haiti. We trained two models using weakly supervised attention-based multiple instance learning: a 'TCGA' model, trained on TCGA data, and ESPWA, trained on the ZL dataset. Model performance was evaluated using 10-fold cross validation. Results: Performance of the 'TCGA' model was sensitive to the domain shift between the TCGA and ZL datasets, with a performance of an area under receiver operating characteristic (AUROC) of 0.846 on the TCGA test sets and 0.671 on the ZL test sets. Compared to the 'TCGA' model, ESPWA demonstrated improved performance on the ZL cohort (AUROC=0.790; p=0.005). Subgroup analyses revealed clinically relevant populations in which ESPWA demonstrated improved performance relative to the overall cohort. Finally, ESPWA outperformed an academic breast pathologist (accuracy: 0.726 vs 0.639 respectively; p <0.001) in determining ER status from H&E WSIs. Conclusion: ESPWA ('Hope' in Haitian Creole) offers an accessible framework to identify individualized therapeutic insights from H&E WSIs in LMICs. We have initiated clinical trials, using ESPWA, in ZL and sub-Saharan African countries to inform precision-based use of ET for prospective patients.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.27.672012v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Decitabine reverses innate immune gene suppression in rare melanomas</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> MacBeth, M. L., Bagby, S. M., Borgers, J. S. W., Turner, J. A., Anderson, K. C., Whitty, P. A., Hartman, S., Yacob, B., Rebecca, V. W., Davila, E., Pitts, T. M., Medina, T. M., Patel, S. P., McCarter, M. D., Robinson, W. A., Tobin, R. P., Couts, K. L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Rare melanoma subtypes, including acral, mucosal, and uveal melanomas, exhibit limited responses to immune checkpoint inhibitors (ICIs), yet the molecular mechanisms of immune resistance remain poorly defined. Here, we performed transcriptomic profiling of patient-derived xenografts (PDXs) and publicly available tumor datasets to systematically compare intratumoral gene expression across cutaneous and rare melanoma subtypes. We identified a convergent downregulation of innate immune pathogen sensing (IIPS) and type I interferon signaling pathways in rare melanomas compared to cutaneous, with lower expression also observed in anti-PD-1 non-responder tumors. CIBERSORT deconvolution of immune populations revealed that lower IIPS gene-expressing tumors exhibited reduced CD8 T cell and memory CD4 T cell infiltration, and enrichment of M2 macrophages, consistent with a more immunosuppressive tumor microenvironment. In vitro screening of epigenetic and immunomodulatory compounds revealed that the DNA hypomethylating agent decitabine robustly induced IIPS and adaptive immune gene expression in rare melanoma cell lines. In vivo treatment of mucosal and uveal melanoma xenograft models with decitabine resulted in durable upregulation of IIPS and antigen presentation genes, and whole transcriptome analysis confirmed that IIPS gene re-expression was the dominant transcriptional consequence of decitabine treatment. These findings highlight silencing of IIPS genes as a recurrent immune evasion mechanism in rare melanomas and nominate decitabine as a potential immunomodulatory strategy for enhancing immune responsiveness.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.24.671992v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Microtubule Inhibition Triggers MYC-Mediated Immunogenic Cell Death in Breast Cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Aung, J., Liu, R., Peura, A., Hiltunen, A. O., Id, L. J., Savelius, M., Peltonen, M., Turpin, R. J., Salmelin, N., Anttila, J. M., Tervonen, T. A., Suleymanova, I., Nicorici, D., Mutka, M., Mattson, J., Kovanen, P. E., Niinikoski, L., Meretoja, T., Prajapati, B., Goga, A., Munne, P., Pouwels, J., Klefstrom, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Oncogenic MYC promotes cancer cell proliferation, metabolism, and death, while also driving immunosuppression in the tumour microenvironment, complicating immune-based therapies. To counter MYC-driven immune evasion while leveraging MYC-dependent synthetic lethality (MYC-SL), we identified microtubule-targeting agents, including eribulin, as potent inducers of immunogenic cell death in MYChigh triple-negative breast cancer (TNBC). A screen of 528 oncology compounds using damage-associated molecular pattern (DAMP) reporters revealed that microtubule inhibitors induced key DAMPs, including HMGB1 secretion, calreticulin exposure, and double-stranded DNA release, leading to gasdermin-E associated cell death in MYChigh TNBCs. Immune cell co-culture assays showed immune activation, and patient-derived explant cultures confirmed pro-inflammatory cytokine responses. In vivo, cell-free media from eribulin-treated MYChigh murine TNBCs enhanced tumour protection in vaccination models compared to MYC-knockdown controls, linking MYC-dependent DAMP release to immunogenicity. These findings highlight a dual-function therapeutic strategy: agents that selectively induce MYC-dependent immunogenic cell death can provide both targeted cytotoxicity and local immune stimulation, thereby addressing a key limitation of conventional chemotherapeutics, offering a new approach for MYC-driven cancers.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.26.671755v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Unravelling the in vivo traits of vasculogenic mimicry</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Oraiopoulou, M.-E., Lefebvre, T. L., Tzamali, E., Else, T. R., Cannell, I. G., Wright, L. C., Bunce, E. V., Brodie, C., Sweeney, P. W., Porcu, L., Couturier, D.-L., Sakkalis, V., Hannon, G. J., Bohndiek, S. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Vasculogenic mimicry (VM) describes the ability of cancer cells to acquire endothelial properties and form vessel-like channels that facilitate tumour blood supply. While the molecular drivers of VM have been well-explored in cell cultures and biopsies, an in vivo description remains elusive. Here, we used graph theory to define VM biomarkers and elucidate the spatiotemporal dynamics of VM using in vitro and in vivo breast cancer models with and without anti-angiogenic treatment. Optical microscopy was used to assess pseudo-vascular networks in vitro while photoacoustic imaging across scales was applied in vivo to identify and locate haemoglobin contrast-derived blood vessel morphology and functionality. VM was associated with greater oxygenation heterogeneity and poorer anti-angiogenic response, reflected as stable meshed networks in vitro and blood-containing circular structures in vivo. We demonstrate for the first time a multi-scale approach bridging the in vitro-in vivo translational gap to assess anti-vascular treatment resistance and the therapeutic potential in vasculogenic mimicry-rich tumours, exploring novel avenues in preclinical drug screening and systemic drug delivery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.28.671205v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The caninized anti-canine PD-1 monoclonal antibody in canine oral malignant melanoma: Efficacy and exploratory biomarker analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Igase, M., Hagimori, K., Inanaga, S., Mizoguchi, H., Itamoto, K., Sakurai, M., Motegi, T., Yamamoto, H., Kato, M., Shiga, T., Tsukui, T., Kobayashi, T., Mizuno, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Canine oral malignant melanoma (OMM) is a highly aggressive tumor with few treatment options. Because of its biological similarity to human mucosal melanoma, canine OMM represents a valuable spontaneous model for translational immunotherapy studies. Anti-PD-1 antibody therapy has shown promise in canine OMM; however, predictive biomarkers for treatment response and survival have not been identified. Methods: We conducted a multicenter, prospective, investigator-initiated clinical trial to evaluate the safety and efficacy of a caninized anti-canine PD-1 monoclonal antibody (ca-4F12-E6) in 150 dogs with advanced OMM. The dogs were administered 3 mg/kg ca-4F12-E6 intravenously every two weeks. Treatment response was assessed using cRECIST v1.0. Biomarker analyses included peripheral blood parameters, cytokine/chemokines, peripheral lymphocyte subsets, microsatellite instability (MSI), immunohistochemistry for immune cell and mismatch repair protein markers, and RNA sequencing of the tumor tissue. Associations with clinical outcomes were determined by logistic regression and Cox proportional hazard models. Results: The best overall response rate was 16.7%. Treatment-related adverse events occurred in 40.0% of the dogs, which were primarily grade 1-3. Increased baseline white blood cell, neutrophil count, and C-reactive protein (CRP) levels were significantly associated with poor response, shorter progression-free survival (PFS), and reduced overall survival (OS). MSI-high tumors were associated with significantly prolonged OS compared with MSI-low/microsatellite stable tumors. Transcriptome analysis revealed differentially expressed genes and enriched immune-related pathways in responders, though limited by sample size. Conclusion: ca-4F12-E6 exhibited durable antitumor activity and an acceptable safety profile in dogs with OMM. Baseline systemic inflammatory markers and MSI status may serve as predictive biomarkers for clinical outcomes. The results support the use of canine OMM as a comparative model for human immuno-oncology and biomarker discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.26.671889v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

