<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Saturday, 02 August 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genome assemblies for seven families of birds from the Global South</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> K L, V., Goyal, N., Warudkar, A., Arvind, C., Robin, V. V.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present high-quality, de novo genome assemblies for seven birds, representing seven families distributed in the Western Ghats of India, a biodiversity hotspot with high endemism.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Tropical regions are biodiversity-rich, yet remain underrepresented in the availability of genomic resources, as is evident in the Western Ghats of India, a biodiversity hotspot with high endemism. Here, we present high-quality, de novo genome assemblies for seven birds, representing seven families distributed in the Western Ghats: Black-naped Monarch (Monarchidae: Hypothymis azurea), Indian Yellow Tit (Paridae: Machlolophus aplonotus), Brown-cheeked Fulvetta (Leiothrichidae: Alcippe poioicephala), Malabar Trogon (Trogonidae: Harpactes fasciatus), Blue-bearded Bee-eater (Meropidae: Nyctyornis athertoni), Malabar Whistling-Thrush (Muscicapidae: Myophonus horsfieldii), Orange-headed Thrush (Turdidae: Geokichla citrina). Using a hybrid Oxford Nanopore long reads -Illumina short reads approach, we assembled genomes with sizes ranging from 1.03 to 1.13 Gbp. All assemblies demonstrated high contiguity and completeness (BUSCO scores >97%, UCEs >4799). Repeat masking identified ~10% of the genomes as interspersed repeats, and functional annotations yielded an average of 9,619 protein-coding genes per species. Comparative analysis showed our assemblies had significantly higher contiguity than the median of existing avian genomes on NCBI (Wilcoxson test, p = 0.00226). Our genome assemblies fill a key geographic and taxonomic gap in the genomic data and provide a foundational resource for evolutionary and ecological research in the Old-World tropics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.31.667860v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An aphid resistant wheat variety reduces the transmission of Barley Yellow Dwarf Virus (BYDV) by Rhopalosiphum padi (L.)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Qonaah, I. A., Simon, A. L., Warner, D., Bruce, T. J. A., Ray, R. V.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A aphid-resistant wheat variety with reduced BYDV transmission efficiency and reduced systemic infection.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> INTRODUCTION: Rhopalosiphum padi (L.) is a vector of Barley Yellow Dwarf Virus (BYDV) infecting major cereals including wheat. Recently, a winter wheat variety (G1) was identified as exhibiting significant aphid resistance through antixenosis and antibiosis. This study compares resistance to viruliferous aphids, and BYDV transmission, in G1 with RGT Wolverine and RGT Illustrious, a BYDV resistant and susceptible wheat varieties, respectively. We aimed to define how aphid resistance affects BYDV transmission, infection and spread. RESULTS: Seedling choice and olfactometer bioassays using wheat volatile organic compounds revealed that G1 emits an aphid repellent compound, identified as 2-tridecanone using GC-MS. Electrical penetration graph recordings showed restricted phloem access and salivation of viruliferous R. padi in G1, associated with lower BYDV transmission efficiency. Quantitative RT-PCR revealed a three-fold reduction in BYDV gene expression ratio on G1 transmission leaves compared to RGT Wolverine or RGT Illustrious. In contrast, reduced systemic infection in RGT Wolverine implied a BYDV resistance mechanism of limiting viral replication and/or movement. Rearing aphids on the aphid or BYDV resistant varieties modified their host selection behaviours suggesting vector conditioning with implications for viral transmission and spread on susceptible hosts. CONCLUSION: This study demonstrates that the aphid resistance in G1 reduced BYDV transmission. Contrastingly, RGT Wolverine appeared to limit systemic viral infection despite high transmission efficiency. Combining these two distinct resistance mechanisms by breeding offers valuable strategy against both the aphid and the virus. To further define aphid and BYDV defence responses in G1, transcriptomic and metabolomic studies will be required.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.30.667415v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">TISCalling: leveraging machine learning to identify translational initiation sites in plants and viruses.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ming-Ren Yen, Ya-Ru Li, Chia-Yi Cheng, Ting-Ying Wu, Ming-Jung Liu</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present TISCalling, a robust framework that combines machine learning (ML) models and statistical analysis to identify and rank novel translational initiation sites (TISs) across eukaryotes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The recognition of translational initiation sites (TISs) offers complementary insights into identifying genes encoding novel proteins or small peptides. Conventional computational methods primarily identify Ribo-seq-supported TISs and lack the capacity of systematic and global identification of TIS, especially for non-AUG sites in plants. Additionally, these methods are often unsuitable for evaluating the importance of mRNA sequence features for TIS determination. In this study, we present TISCalling, a robust framework that combines machine learning (ML) models and statistical analysis to identify and rank novel TISs across eukaryotes. TISCalling generalized and ranks important features common to multiple plant and mammalian species while identifying kingdom-specific features such as mRNA secondary structures and "G"-nucleotide contents. Furthermore, TISCalling achieved high predictive power for identifying novel viral TISs. Importantly, TISCalling provides prediction scores for putative TIS along plant transcripts, enabling prioritization of those of interest for further validation. We offer TISCalling as a command-line-based package [ https://github.com/yenmr/TISCalling ], capable of generating prediction models and identifying key sequence features. Additionally, we provide web tools [ https://predict.southerngenomics.org/TISCalling/ ] for visualizing pre-computed potential TISs, making it accessible to users without programming experience. The TISCalling framework offers a sequence-aware and interpretable approach for decoding genome sequences and exploring functional proteins in plants and viruses.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40748408/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Engineering Plant Holobionts for Climate-Resilient Agriculture.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nayanci Portal-Gonzalez, Wenbo Wang, Wenxing He, Ramon Santos-Bermudez</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce the concept of the programmable holobiont: an engineered plant-microbe partnership capable of dynamic feedback, interkingdom signaling, and ecological memory.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The plant holobiont-an integrated unit of the host and its microbiome-has co-evolved through ecological and genetic interactions. Microbiome engineering offers a promising route to enhance resilience in response to climate stress, soil degradation, and yield stagnation. This review presents an integrated framework combining microbial ecology, synthetic biology, and computational modeling to rationally design synthetic microbial communities (SynComs) for agriculture. We outline ecological principles-priority effects, keystone taxa, and functional redundancy-that shape microbiome assembly and guide SynCom design. Strategies like CRISPR interference, biosensor circuits, and quorum-sensing modules enable programmable microbial functions. We also highlight the predictive potential of in silico modeling-including genome-scale metabolic models, dynamic flux balance analysis, and machine learning-to simulate interactions, optimize SynCom composition, and enhance design accuracy. To bridge lab and field, we discuss native microbial chassis, encapsulation, and precision delivery as tools for scalable, ecosystem-integrated deployment. We introduce the concept of the programmable holobiont: an engineered plant-microbe partnership capable of dynamic feedback, interkingdom signaling, and ecological memory. This systems-level perspective reframes plants as designable ecosystems. By synthesizing cross-disciplinary advances, we offer a roadmap for climate-resilient agriculture, where engineered microbiomes improve sustainability, yield stability, and environmental adaptation.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40748243/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Approaching an Error-Free Diploid Human Genome</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chu, Y., Huang, Z., Shao, C., Guo, S., Yu, X., Wang, J., Tian, Y., Chen, J., Li, R., He, Y., Yu, J., Huang, J., Gao, Z., Kang, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We first developed Sufficient Alignment Support (SAS), a sensitive alignment-based metric that identifies both structural and base-level assembly erroneous windows by quantifying inadequate read support across sequencing platforms.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Achieving a truly error-free human diploid genome remains a central challenge in genomics. Here, we present T2T-YAO v2.0, a nearly perfect diploid genome assembly for a Han Chinese individual. We first developed Sufficient Alignment Support (SAS), a sensitive alignment-based metric that identifies both structural and base-level assembly erroneous windows by quantifying inadequate read support across sequencing platforms. Unlike conventional methods that detect erroneous k-mers (e.g., Merqury) or inconsistencies between assemblies and aligned reads from single platform, SAS integrates supportive information from various platform and avoids false positives caused by sequencing bias and artificial alignments. Employing a "Structural Error-first" polishing strategy, prioritizing structural error correction using ultra-long Oxford Nanopore reads, we iteratively refined the assembly with high-accuracy PacBio and Element sequencing data. This approach eliminates all detectable structural and non-homopolymer-related base-level errors outside ribosomal DNA clusters. The final T2T-YAO v2.0 assembly exhibits complete non-split ONT read coverage and free of erroneous 21-mers outside sequencing data, representing an error-free human genome by Merqury standards. Our results not only highlight the limitations of current genome quality assessment methods but also provide a roadmap for constructing high-fidelity diploid references essential for precision medicine and genome-based diagnostics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.01.667781v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Phyling: phylogenetic inference from annotated genomes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tsai, C.-H., Stajich, J. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Phyling is a fast, scalable, and user-friendly tool supporting phylogenomic reconstruction of species phylogenies directly from protein-encoded genomic data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Phyling is a fast, scalable, and user-friendly tool supporting phylogenomic reconstruction of species phylogenies directly from protein-encoded genomic data. It identifies orthologous genes by searching a sample's protein sequences against a Hidden Markov Models marker set, containing single-copy orthologs, retrieved from the BUSCO database. In the final step, users can choose between consensus and concatenation strategies to construct the species tree from the aligned orthologs. Phyling efficiently resolves large phylogenies by optimizing memory usage and data processing. Its checkpoint system enables users to incrementally add or remove samples without repeating the entire search process. For analyses involving closely related taxa, Phyling supports the use of nucleotide coding sequences, which may capture phylogenetic signals missed by protein sequences. The benchmark results show that Phyling substantially runs faster than OrthoFinder, a Reciprocal Best Hit based method, while achieving equal or better accuracy.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.30.666921v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SWIF-TE: identifying novel transposable element insertions from short read data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Menard, C., Catlin, N., Platts, A., Qiu, Y., Roback, E., Munasinghe, M., Springer, N. M., Josephs, E., Hirsch, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A fast and memory-efficient tool to identify novel transposable element insertion polymorphisms from short read sequences.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Transposable element (TE) insertion polymorphisms (TIPs) are TEs not in the same location between individuals. TIPs have contributed to genomic and phenomic variation but have been historically difficult to study due to their repetitive nature. Here, we describe a fast and memory-efficient tool to identify novel TIPs from short read sequences. SWIF-TE was able to identify 1,438 insertions at a precision rate of 27% using 0.10 Gb of memory and 0.82 hours of runtime from 15x resequencing data of a non-reference maize inbred. SWIF-TE is a powerful tool for studying TE variation in species with TE-rich genomes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.30.667279v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An Agentic AI Framework for Ingestion and Standardization of Single-Cell RNA-seq Data Analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nouri, N., Artzi, R., Savova, V.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed an agentic AI framework that enables dialogue-driven, document-to-analysis automation of single-cell data ingestion and analysis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The proliferation of publicly available single-cell RNA sequencing (scRNA-seq) data has created significant opportunities in biomedical research. However, the reuse of these resources is constrained by a series of preparatory steps, including metadata extraction from primary literature, retrieval of datasets from corresponding repositories, and the subsequent manual execution of standardized downstream analysis. These tasks often require manual scripting and rely on fragmented workflows, limiting accessibility and increasing turnaround time. To address these challenges, we designed a two-component system consisting of an artificial intelligence (AI) agent coordinating an automated analysis pipeline. CellAtria (Agentic Triage of Regulated single-cell data Ingestion and Analysis) is an agentic AI framework that enables dialogue-driven, document-to-analysis automation through a chatbot interface. Built on a graph-based, multi-actor architecture, CellAtria integrates a large language model (LLM) with tool-execution capabilities to orchestrate the full lifecycle of data reuse. To support downstream analysis, CellAtria incorporates CellExpress, a co-developed pipeline that applies state-of-the-art scRNA-seq processing steps to transform raw count matrices into analysis-ready single-cell profiles. Thus, CellAtria provides computational skill-agnostic and time-efficient access to standardized single-cell data ingestion and analysis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.31.667880v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Cosmos: A Position-Resolution Causal Model for Direct and Indirect Effects in Protein Functions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rao, J., Wang, M., Howard, M., Coyote-Maestas, W., Pimentel, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Cosmos, a Bayesian framework for residue-level causal inference in multi-phenotype deep mutational scanning (DMS) experiments, and show that it effectively distinguishes direct from indirect functional effects.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Multi-phenotype deep mutational scanning (DMS) experiments provide a powerful means to dissect how protein variants affect different layers of molecular function, such as abundance, surface expression, and ligand binding. When these phenotypes are connected through a molecular pathway, interpreting variant effects becomes challenging because downstream phenotypes often reflect both direct and indirect consequences of mutation. We introduce Cosmos, a Bayesian framework for residue-level causal inference in multi-phenotype DMS data. Cosmos addresses three key questions: (1) whether a causal relationship exists between two phenotypes; (2) the strength of that relationship; and (3) the expected downstream phenotype if the upstream phenotype were normalized, enabling counterfactual interpretation. The framework uses position-level aggregation and Bayesian model selection to infer interpretable causal structures, without requiring phenotype-specific biophysical assumptions. We apply Cosmos to three datasets - Kir2.1 (abundance and surface expression), PSD95-PDZ3 (abundance and CRIPT binding), and KRAS (abundance and RAF1-RBD binding) and show that it effectively distinguishes direct from indirect functional effects. Across these applications, Cosmos provides a generalizable and interpretable approach to disentangle causal relationships in high-throughput protein functional screens.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.01.667517v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Haplotype-resolved chromosome-level genome assemblies of four Diamesa species reveal the genetic basis of cold tolerance and high-altitude adaptations in arctic chironomids</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Martin, S. L. F., La Torre, R., Danneels, B., Tooming-Klunderud, A., Skage, M., Kollias, S., Torresen, O. K., Falahati Anbaran, M., Stur, E., Jakobsen, K. S., Martin, M. D., Ekrem, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the first haplotype-resolved, chromosome-level genomes for four species of Diamesa, a genus of cold-adapted midges inhabiting glacial and high-altitude freshwater ecosystems.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Arctic and alpine insects face extreme environmental stressors, yet the genomic basis of their adaptation remains poorly understood. Here, we present the first haplotype-resolved, chromosome-level genomes for four species of Diamesa (Diptera: Chironomidae), a genus of cold-adapted midges inhabiting glacial and high-altitude freshwater ecosystems. Using PacBio HiFi sequencing and Hi-C scaffolding, we assembled high-quality genomes with chromosome-level resolution and high k-mer completeness. Phylogenomic analyses support Diamesinae as sister to other Chironomidae except Podonominae, and genomic comparisons provide evidence for introgression between the evolutionary distinct D. hyperborea and D. tonsa. Comparative genomic analyses across 20 Diptera species revealed significant gene family contractions in Diamesa associated with oxygen transport and metabolism, suggesting adaptations to high-altitude, low-oxygen environments. Conversely, expansions were detected in histone-related and Toll-like receptor gene families, likely enhancing chromatin remodeling and immune regulation under cold stress. A single gene family encoding glucose dehydrogenase was significantly expanded across all cold-adapted species studied, implicating its role in cryoprotectant synthesis and oxidative stress mitigation. Notably, Diamesa species exhibit the largest gene family contraction at any node, with minimal overlap in expansions with other cold-adapted Diptera, indicating lineage-specific adaptation. Our findings support the hypothesis that genome size condensation and selective gene family changes underpin survival in cold environments. These genome assemblies represent a valuable resource for investigating adaptation, speciation, and conservation in cold-specialist insects. Future work integrating gene expression and population genomics will further illuminate the evolutionary resilience of Diamesa in a warming world.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.30.667619v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AEMB: a computationally efficient abundance estimation method for metagenomic binning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pan, S., Tolstoganov, I., Sahlin, K., Martin, M., Zhao, X.-M., Coelho, L. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose AEMB (Abundance Estimation for Metagenomic Binning), a novel mapping mode implemented in strobealign, a fast abundance estimation method that skips base-level alignment.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Metagenomic binning is a crucial step in metagenomic analysis, namely grouping together contigs that are predicted to originate from the same genome to enable the recovery of metagenome-assembled genomes (MAGs). It has been shown that using information from multiple samples yields better results than binning each sample independently. However, for N metagenomic samples, using full multi-against-multi binning requires N2 alignments, making it computationally challenging to apply in large-scale metagenomic studies. Here, we propose AEMB (Abundance Estimation for Metagenomic Binning), a novel mapping mode implemented in strobealign. AEMB is a computationally efficient abundance estimation method that uses a prefix-lookup vector as an indexing structure to reduce memory usage and randstrobes to estimate the abundance of contigs without performing base-level alignment. Compared to the hash table used in the previous version of strobealign, the indexing structure reduces peak memory usage by 25.2% with almost the same runtime. Furthermore, we implemented a fast abundance estimation method that skips base-level alignment. Altogether, AEMB reduces the runtime for abundance estimation by 88% to 96% compared to commonly used alignment methods such as Bowtie2 and BWA, while achieving similar binning results. AEMB is available as a mapping mode in strobealign https://github.com/ksahlin/strobealign and SemiBin2 (v2.1 and later) accepts its inputs for binning.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.30.667338v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">plasmoRUtils: A one-stop R Package for Plasmodium and other Apicomplexan parasite-related Bioinformatics analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Satyam, R., DE INFANTE, A. M. R., SUBUDHI, A. K., Salunke, R. P., Cabrero, D. G., Pain, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present plasmoRUtils, an R package designed to streamline database access and data harmonization for Apicomplexan research, by automating data retrieval and transformation within RStudio.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Bioinformatics analysis of non-model organisms remains challenging due to the limited availability of specialized tools, as most R packages are optimized for well-annotated model species. This problem is exacerbated by genomic and proteomic data being scattered across multiple databases, each employing different identifiers based on varying reference annotations. Comprehensive databases have been developed to disseminate knowledge related to Apicomplexan genomics and proteomics, such as VEupathDB. Several specialised databases, particularly for the malaria parasite Plasmodium, have been developed such as ApicoTFDB, malaria.tools, MPMP, MIIP, Phenoplasm, PlasmoBase, alongside broader resources like HitPredict and TED. However, these platforms often suffer from manual query interfaces, outdated identifiers, and inefficient data retrieval methods, complicating their use for large-scale bioinformatic analyses. To address these limitations, we present plasmoRUtils, an R package designed to streamline database access and data harmonization for Apicomplexan research. plasmoRUtils enables the retrieval of data tables using single-line R functions and standardized Ensembl gene IDs as input. Thanks to the APIs available for some databases such as VEuPathDB, the package also provides functions to build CLI-based queries for VEuPathDBs component databases. Additional support includes performing overrepresentation analyses and estimating parasite transcriptomic age/stage using single-cell or bulk RNA-Seq references. By automating data retrieval and transformation within RStudio, plasmoRUtils eliminates the need for manual database queries, facilitating end-to-end workflow development without leaving the R environment. Available on GitHub https://github.com/Rohit-Satyam/plasmoRUtils, with comprehensive documentation, plasmoRUtils represents a critical step toward efficient and reproducible bioinformatics for Apicomplexan research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.30.667718v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Viral metagenomics of synanthropic urban bats: a surveillance strategy for uncovering potentially zoonotic viruses</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Conselheiro, J. A., Barone, G. T., Reis-Menezes, A. A., da Rosa, A. R., de Oliveira, D. C., Moreira, F. R. R., Chaves, B. A., Sampaio, V. d. S., Rocha, F., Vigilato, M. A. N., Stabeli, R. G., Said, R. F. d. C., Wallau, G. L., Brandao, P. E., de Brito, A. F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a surveillance framework that leverages Brazil's rabies passive surveillance program to detect bat-borne viruses using metagenomic sequencing and demonstrate previously undetected viral diversity in Brazilian bats.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Zoonotic spillover events are increasing in frequency, highlighting the urgent need for integrated surveillance systems capable of detecting emerging viruses at the human-animal-environment interface. Bats are natural reservoirs for diverse zoonotic viruses, including coronaviruses, filoviruses, and paramyxoviruses. Here, we present a surveillance framework that leverages Brazil's rabies passive surveillance program to detect bat-borne viruses using metagenomic sequencing. We developed an algorithm to select representative samples from 2,422 bats collected across Sao Paulo state between October 2023 and February 2024. Paired lung and intestine samples from 150 specimens underwent RNA extraction and nanopore sequencing. A custom bioinformatics pipeline enabled the detection of 66 viral contigs from nine zoonotic-relevant families, including Arenaviridae, Coronaviridae, Filoviridae, and Paramyxoviridae. Notably, we report the first detection of a filovirus in bats in the Americas and characterize the presence of multiple potentially public health-relevant viruses in close contact with humans. These findings reveal previously undetected viral diversity in Brazilian bats and demonstrate how existing animal surveillance initiatives can be leveraged to monitor broader pathogen threats. Our approach provides a scalable, cost-effective model for operationalizing surveillance towards a One Health approach, with potential applications in epidemic and pandemic prevention, particularly in low- and middle-income countries.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.30.666427v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Balancing the supply and demand for taxonomy: an analysis of European taxonomic capacity and policy needs.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Groom, Q. J., De Nolf, M., Estupinan-Suarez, L. M., Meeus, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a meta-research analysis of taxonomic activity in Europe over the past decade, drawing on publication metadata from OpenAlex, Wikidata, and GBIF.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Taxonomy is a cornerstone of biological science and essential to biodiversity policy, yet it faces persistent structural challenges collectively known as the 'taxonomic impediment'. These include limited capacity, uneven geographic and taxonomic coverage, and a disconnect between the supply of expertise and its societal demand. In this study, we present a meta-research analysis of taxonomic activity in Europe over the past decade, drawing on publication metadata from OpenAlex, Wikidata, and GBIF. Using an open and reproducible workflow, we identify more than 31,000 authors affiliated with European institutions who have contributed to taxonomic publications, and we assess their taxonomic and institutional distribution. We explore how this supply of expertise compares with the kinds of demands that arise from European biodiversity policy, including legally binding instruments such as the Birds and Habitats Directives and the Marine Strategy Framework Directive, as well as strategic initiatives focused on invasive alien species, crop wild relatives, and species of conservation concern. Our results highlight clear imbalances in capacity across taxonomic groups and regions, with some politically and ecologically significant taxa receiving comparatively little attention. This work illustrates how openly available data can be used to evaluate taxonomic capacity and its alignment with policy needs, and it provides a framework for others to adopt in support of more strategic planning and investment in taxonomy.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.31.666655v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A photosynthetic checkpoint halts Synechocystis sp. PCC 6803 response to nitrogen deprivation during glucose-mediated metabolic imbalance</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ortega-Martinez, P., Giner-Lamia, J., Wey, L. T., Muro-Pastor, M. I., Florencio, F. J., Diaz-Troya, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a redox-sensitive checkpoint that integrates metabolic state with photosynthetic performance, offering new insights into stress adaptation in cyanobacteria.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cyanobacteria adapt to nitrogen starvation by undergoing chlorosis, a regulated bleaching process involving the degradation of phycobilisomes, the light harvesting antennae complexes, and accumulation of glycogen. While this response is well-characterized under photoautotrophic conditions, its modulation by external organic carbon sources such as glucose remains poorly characterized. Here, we investigated how glucose affects chlorosis onset and the underlying regulatory mechanisms, focusing on redox control and photosynthetic electron transport. Using an integrative approach combining physiological assays, targeted metabolomics, RNA-sequencing, chlorophyll fluorescence and absorbance spectroscopy, we studied responses in wild-type cells of the model cyanobacterium Synechocystis sp. PCC 6803. Glucose supplementation prevented bleaching, even when added after nitrogen deprivation symptoms had begun. This effect was associated with excess glycogen accumulation, disrupted carbon partitioning, and buildup of metabolic intermediates, indicating a metabolic overflow. Despite these physiological differences, transcriptomic responses to nitrogen deprivation were largely similar regardless of glucose supplementation, suggesting regulation at the post-transcriptional or metabolic level. Glucose also impaired photosynthetic electron transport by creating a redox bottleneck at the photosystem II (PSII) acceptor side, leading to decreased electron transport to photosystem I (PSI) and accumulation of oxidized P700. These findings suggest that reduction of the PSI acceptor side is required to trigger chlorosis. Our results demonstrate that glucose uncouples nitrogen sensing from the bleaching process by altering redox balance and photosynthetic electron flow. We propose the existence of a redox-sensitive checkpoint that integrates metabolic state with photosynthetic performance, offering new insights into stress adaptation in cyanobacteria.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.30.667207v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Sequence mismatch between gene-drive and target-site flanking regions significantly impairs homing efficiency in Culex quinquefasciatus</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Harvey-Samuel, T., Kaur, R., Leftwich, P. T., Feng, X., Gantz, V., Alphey, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel mechanism by which a homing-drives efficiency can be significantly impaired by natural sequence variation within a population into which it is spreading.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> CRISPR/Cas9-based homing gene-drives (homing-drives) hold enormous potential as control tools for mosquito disease-vectors. These genomically-encoded technologies spread themselves through target populations by creating double-stranded DNA breaks on homologous chromosomes, into which the homing-drives are copied (homed). Homing is dependent on sequence homology between the genomic regions flanking the transgene insertion and the break site. Homing efficiency (i.e. copying rate) substantially impacts the power of these systems: less efficient homing-drives spread slower, have fewer applications and are more resistance-prone. Understanding what influences homing-drive efficiency is therefore vital to the successful use of these technologies. Here we report a novel mechanism by which a homing-drives efficiency can be significantly impaired by natural sequence variation within a population into which it is spreading. Using a kmo-targeting split homing-drive in the West Nile virus mosquito Culex quinquefasciatus, we found that target-site heterology (sequence mismatch between the genomic regions flanking the target cut-site and the homing-drive transgene) of less than 10% reduced homing efficiency by up to 54%. While substantial research effort has been dedicated to increasing homing-drive efficiency through optimisation of within-construct components, our results highlight that the real-world efficacy of these systems may in part depend on variation beyond these controllable factors.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.01.668174v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Design-driven optimization of low-cost reagent formulations for reproducible and high-yielding cell-free gene expression</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Olsen, M. L., Copeland, C. E., Sundberg, C. A., Aw, R., Shaver, Z. M., Rao, G., Swartz, J. R., Karim, A. S., Jewett, M. C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a simple and reproducible reagent formulation for cell-free gene expression systems that can produce full-length aglycosylated monoclonal antibodies, including full- length aglycolytic antibodies, in a cost-effective manner.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Access to recombinant proteins is vital in basic science and biotechnology research. Cell-free gene expression systems provide one approach to address this need, but widespread utilization remains limited by the cost, complexity, and inconsistency of current platforms. To address these limitations, we carry out a multi-dimensional definitive screening design to reduce the number of reagent components and remove costly secondary energy substrates. From more than 1,200 reagent formulations, we discover a simple and reproducible system based on 12 components. The optimized reagent formulation can produce 2.4 {+/-} 0.3 g/L of protein product at the 15-{micro}L scale (~$55/g protein) and 3.7 {+/-} 0.2 g/L (~$36/g protein) at the 4-mL scale with oxygen supplementation. This provides an 84 to 99% reduction in cost over previous cell-free reagent formulations. We further show that the optimized reagent formulation can produce nucleoside triphosphates from nucleotides and ribose and that it is robust to failure across batches of cell lysates, users/locations, and in the synthesis of different proteins. Specifically, we demonstrate the production of fifteen therapeutically relevant products, including full-length aglycosylated monoclonal antibodies. We anticipate that our optimized reagent formulation will further democratize the use of cell-free systems for protein manufacturing and synthetic biology applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.01.668204v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">cNPDB: A comprehensive empirical crustacean neuropeptide database</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tran, V. N. H., Duong, T. U., Fields, L., Tourlouskis, K., Beaver, M., Li, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the first centralized resource dedicated to crustacean neuropeptidome research and offering a blueprint for cross-species translational studies of Neuropeptide signaling.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Neuropeptides, key signaling molecules essential for dynamic regulation of biological processes, have been studied via crustacean model systems for more than 40 years. We present cNPDB, the first centralized resource dedicated to this dynamic area of research, promisingly accelerating crustacean neuropeptidome research and offering a blueprint for cross-species translational studies of neuropeptide signaling. cNPDB is comprised of 1364 published neuropeptides from 29 crustacean species, each annotated with corresponding taxonomic information, computed physicochemical properties, spatial localization, and predicted three-dimensional structure. The intuitive web-interface supports keyword and property-based filtering, sequence alignment, property calculation, and links to available mass spectrometry imaging data, streamlining discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.29.667494v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Evaluating zero-shot prediction of protein design success by AlphaFold, ESMFold, and ProteinMPNN</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Garcia, M., Rocklin, G. J., Dixit, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that structure and sequence prediction models can discriminate experimental successes from failures in de novo protein design, but their utility is limited without task-specific training.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> De novo protein design has enabled the creation of proteins with diverse functionalities that are not found in nature. Despite recent advances, experimental success rates remain inconsistent and context-dependent, posing a bottleneck for broader applications of de novo design. To overcome this, structure and sequence prediction models have been applied to assess design quality prior to experimental testing to save time and resources. In this study, we aimed to determine the extent to which AlphaFold2, Protein MPNN, and ESMFold can discriminate between experimentally successful and unsuccessful designs. For this, we curated a benchmark dataset of 614 experimentally characterized de novo designed monomers from 11 different design studies between 2012 and 2021. All predictive models demonstrated moderate ability to discriminate experimental successes (expressed, soluble, monomeric, and fold into the intended design structure) from failures, with many failed designs having better confidence metrics than successful designs. Among all computational models evaluated, ESMFold average pLDDT yielded the best individual performance at distinguishing between successful and unsuccessful designs. A logistic regression model combining all confidence metrics provided only modest improvement over ESMFold pLDDT alone. Overall, these results show that these models can serve as an initial filtering strategy prior to experimental validation; however, their utility at accurately predicting experimental successful designs remains limited without task-specific training.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.29.667290v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Biological Function Assignment across Taxonomic Levels in Mass-Spectrometry-Based Metaproteomics via a Modified Expectation Maximization Algorithm.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gelio Alves, Aleksey Y Ogurtsov, Yi-Kuo Yu</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A major challenge in mass-spectrometry-based metaproteomics is accurately identifying and quantifying biological functions across the full taxonomic lineage of microorganisms.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> A major challenge in mass-spectrometry-based metaproteomics is accurately identifying and quantifying biological functions across the full taxonomic lineage of microorganisms. This issue stems from what we refer to as the "shared confidently identified peptide problem‚Ä≥. To address this issue, most metaproteomics tools rely on the lowest common ancestor (LCA) algorithm to assign biological functions, which often leads to incomplete biological function assignments across the full taxonomic lineage of identified microorganisms. To overcome this limitation, we implemented an expectation-maximization (EM) algorithm, along with a biological function database, within the MiCId workflow. Using synthetic datasets, our study demonstrates that the enhanced MiCId workflow achieves better control over false discoveries and improved accuracy in microorganism identification and biomass estimation compared to Unipept and MetaGOmics. Additionally, the updated MiCId offers improved accuracy and better control of false discoveries in biological function identification compared to Unipept, along with reliable computation of function abundances across the full taxonomic lineage of identified microorganisms. Reanalyzing human oral and gut microbiome datasets using the enhanced MiCId workflow, we show that the results are consistent with those reported in the original publications, which were analyzed using the Galaxy-P platform with MEGAN5 and the MetaPro-IQ approach with Unipept, respectively.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40679470/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Plasma Metabolomics and Machine Learning Reveals Metabolic Alterations and Diagnostic Biomarkers for Deep Venous Thrombosis in Hypertensive Patients after Traumatic Fracture.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mukhtar Aliyu, Kun Zhang, Shi-Hao Tang, Jia-Hao Wang, Hao Wu, Pengfei Wang, Hanzhong Xue, Tie-Lin Yang, Wei Huang, Yan Guo</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We aimed to explore the metabolic dysregulations and diagnostic biomarkers for post-traumatic deep venous thrombosis (pt-DVT) in hypertensive (HPT) patients after fracture.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We aimed to explore the metabolic dysregulations and diagnostic biomarkers for post-traumatic deep venous thrombosis (pt-DVT) in hypertensive (HPT) patients after fracture. An untargeted ultraperformance liquid chromatography-mass spectrometry-based metabolomics approach was employed to perform a comprehensive metabolic analysis of plasma samples of 80 patients with post-traumatic deep venous thrombosis and hypertension (pt-DVT&HPT) and 117 patients with hypertension only (HPT). Thirty-seven (37) differential metabolites were identified between pt-DVT&HPT and HPT patients. Purine metabolism, citric acid cycle, sphingolipid metabolism, histidine metabolism, aminoacyl-tRNA biosynthesis, valine, leucine, and isoleucine degradation were the most significantly altered metabolic pathways in pt-DVT in HPT patients. Metabolite-protein interaction network analysis unveils ten (10) proteins/genes that could serve as therapeutic targets. Multivariate methods with Unbiased Variable selection in the R package (MUVR) algorithm identified four metabolites as novel biomarkers for pt-DVT&HPT, and receiver operating characteristics (ROC) analysis showed that a predictive model based on the four biomarkers (l-carnitine, lactic acid, adenine, and l-acetylcarnitine) exhibited better predictive capability for pt-DVT than the D-Dimer. The integration of these diagnostic biomarkers and D-Dimer increased its diagnostic potential. The generalization ability of this biomarker panel was validated in an independent cohort. This study contributed to our understanding of metabolic alterations associated with pt-DVT and paved the way for early diagnosis of pt-DVT in HPT patients.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40663364/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Circulating Proteomic Panels for Noninvasive Diagnosis and Prognostication of Thromboangiitis Obliterans.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gang Fang, Yuan Fang, Lutong Yan, Bichen Ren, Jingyang Luan, Ziang Zuo, Lingwei Zou, Yuning Wang, Shiyang Gu, Tianyue Pan, Hao Liu, Xiaolang Jiang, Yige Lu, Lu Yu, Chenke Ding, Zheng Wei, Peng Liu, Weiguo Fu, Zhihui Dong</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new serum proteomic landscape of thromboangiitis obliterans in TAO and a combined prognostic panel consisting of serum proteins and clinical indicators .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Thromboangiitis obliterans (TAO) is often diagnosed late and characterized by high amputation rates. TAO-specific early diagnostic and disease-staging biomarkers are urgently needed. A staged mass spectrometry (MS)-based discovery-verification-validation workflow was utilized in this study. Ten healthy controls (HCs) and 20 TAOs were included for data-independent acquisition (DIA)-MS quantitative proteomic analysis for the discovery cohort. The DIA-MS analysis acquired 842 identified proteins and 470 quantifiable proteins. Twenty-three candidate biomarkers were further quantified using targeted proteomics based on parallel monitoring reaction (PRM) analysis in the verification stage. A 9-protein and a 7-protein serum biomarker panels were built by machine learning to accurately distinguish TAOs from HCs and active TAOs (A-TAOs) from inactive TAOs. A combined prognostic panel consisting of serum proteins and clinical indicators was established, allowing for risk stratification of A-TAOs. During the validation stage, an independent prospective validation cohort was recruited to validate the proteomic panels based on the enzyme-linked immunosorbent assay analysis, demonstrating the stability and robustness of the predictive models. This study presented the serum proteomic landscape of a TAO cohort and provided novel insights into further biological research. Meanwhile, serum protein signatures have a great potential to improve the early diagnosis and risk stratification in TAOs.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40590214/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">High-Throughput Workflow for Detergent-free Cell-Based Proteomic Characterization.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Saeed Seyedmohammad, Alejandro Rivas, Maxim Zhgamadze, Ali Haghani, Simion Kreimer, Ajay Bharadwaj, Niveda Sundararaman, Sameer Vasantgadkar, Kasturi Pal, Eugenio Daviso, Aleksandr Stotland, Chris Murray, Koen Raedschelders, Sudha Savant, Jennifer E Van Eyk</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We have developed an automated cell-based workflow for the quantification of proteins by liquid chromatography-mass spectrometry (LC-MS) that facilitates large-scale perturbation studies carried out in a 96-well plate format and enables the preparation of one full plate in approximately 4 h</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We have developed an automated cell-based workflow for the quantification of proteins by liquid chromatography-mass spectrometry (LC-MS) that facilitates large-scale perturbation studies carried out in a 96-well plate format and enables the preparation of one full plate in approximately 4 h, showcasing a high-throughput (HTP) concept. Cells were grown in a 96-well plate and lysed via ultrasonication. Proteins were subsequently solubilized, extracted, and processed into tryptic peptides for 2 h before being acquired by data-independent acquisition mass spectrometry (DIA-MS). This workflow leverages adaptive focused acoustics (AFA) technology for ultrasonication to aid cell lysis and protein solubilization on an automated liquid handling platform. As proof of principle, AC16 human cardiomyocyte-like cells were cultured in a 96-well plate under optimized conditions that were compatible with the downstream HTP pipeline. Over 30,000 peptides were identified, corresponding to the detection of 5100 unique proteins. 50% of measured proteins had an average coefficient of variation (CV) under 25% from approximately 30,000 cells. Our optimized detergent-free buffer consisting of ammonium bicarbonate yielded comparable findings. For the same number of cells, 5000 proteins were identified from 29,000 peptides, 40% of which demonstrated a CV under 25%.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40255039/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

