<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Thursday, 09 October 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DeepEmbCas9: Cas9 coevolution and sgRNA structural information for CRISPR-Cas9 cleavage activity prediction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mak, J. K., Minary, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce 4 interpretable DeepEmbCas9 models for the cleavage activity prediction of 40 type II- A and II-C Cas9 variants, leveraging protein and RNA language model embeddings to encode Cas9 and sgRNA, respectively.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The development of CRISPR-Cas9 cleavage activity prediction tools hinges on data produced from high-throughput guide-target lentiviral library screens for different Cas9 variants. However, the majority of such tools remain limited to predictions for one or few Cas9 variants, making it difficult to quantify the effects of Cas9 residues on cleavage activity. To bridge the gap, we introduce 4 interpretable DeepEmbCas9 models for the cleavage activity prediction of 40 type II- A and II-C Cas9 variants -- DeepEmbCas9, DeepEmbCas9-MVE, DeepEnsEmbCas9_naive, and DeepEnsEmbCas9 -- leveraging protein and RNA language model embeddings to encode Cas9 and sgRNA, respectively. Among the 4 neural network models, DeepEnsEmbCas9 naive performed the best in both in-distribution and out-of-distribution settings, where DeepEnsEmbCas9_naive outperformed individual Cas9 cleavage activity prediction tools on 18 out of 51 and 17 out of 48 benchmark test sets, respectively, and performed comparably otherwise. Concerning uncertainty quantification, DeepEnsEmbCas9 yields quantile-calibrated uncertainty estimates while keeping a minimal performance drop compared to DeepEnsEmbCas9 naive. SHAP importance analysis on DeepEmbCas9 reaffirms the importance of Cas9-target PAM binding as a first step for Cas9 cleavage, and reveals the L2 linker and PLL-WED-PI as important Cas9 domains modulating DeepEmbCas9's predicted activity change when introducing increased-fidelity and PAM-altering Cas9 mutations, respectively. Our findings demonstrate the usefulness of protein language model embeddings in uncertainty-aware Cas9 cleavage activity prediction. More generally, DeepEmbCas9 models serves as an initial step towards cleavage activity prediction modelling for the whole Cas9 protein family.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.08.681228v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">iFLinkC-EZ: A scalable and automatable method for the assembly of complex fusion proteins and multi-gene expression constructs based on the iFLinkC framework</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kemp, P., Blat Belmonte, M., Schmidt, M., Eisenhauer, K., Gebhardt, C., Kabuth, K., Koeppl, H., Stein, V.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> iFLinkC-EZ is a new assembly algorithm for the assembly of DNA constructs based on pairwise ligations of any two DNA fragments via a two-base overlap.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Standardized methods for the assembly of DNA constructs have become indispensable in synthetic biology, biotechnology and basic research. The majority of frameworks - most prominently based on Golden Gate - have been developed and optimized for the assembly of transcriptional units (TUs) and thereof composed multi-gene expression constructs. This is significantly enabled by the modular organization of functional elements underlying any given TU. In contrast, the assembly of protein coding sequences typically necessitates tailored approaches. Addressing this technological gap, iterative functional linker cloning (iFLinkC) was recently developed to provide a standardized framework for assembling protein coding sequences of arbitrary size and complexity from the ground up based on pairwise ligations of any two DNA fragments via a two-base overlap. Yet, a need for physically purifying DNA fragments imposes significant procedural complexity and operator skill. Overcoming these limitations, a new assembly algorithm, termed iFLinkC-EZ, is presented. Crucially, the DNA assembly products are now purified by genetic means which greatly simplifies the underlying assembly process and also renders it compatible with robotic automation. Further, the potency of iFLinkC-EZ is demonstrated in the assembly of mono- and poly-chromatic poly-fluorescent fusion proteins as well as poly-cistronic expression constructs while examining the effect of poly-PT and poly-GGS linkers as well as ribosome binding sites on the functional expression of poly-fluorescent fusion protein constructs. Further, the utility of poly-fluorescent proteins is demonstrated for the enhanced labeling of surface displayed recombinant binders in yeast display. Given its ease and efficiency, iFLinkC-EZ is anticipated to be applicable to the assembly of many different types of fusion proteins and thereof based multi-gene expression constructs including but not limited to synthetic protein switches, sensors and multi-enzyme complexes as well as thereof composed genetic circuits and metabolic pathways.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.08.680606v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">JAAG: a JSON input file Assembler for AlphaFold 3 with Glycan integration</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Huang, C., Moremen, K. W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed JAAG (JSON input file Assembler for AlphaFold 3 with Glycan integration), a web-based graphical interface that streamlines AF3 JSON file creation, reduces errors, and facilitates modeling of glycans and glycan-macromolecule interactions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Standalone AlphaFold 3 (AF3) models proteins, post-translational modifications, and ligands (including glycans) from a single JSON input file. Plausible glycan stereochemistry is more consistently achieved when monosaccharides are specified as Chemical Component Dictionary (CCD) entries and connected using bondedAtomPairs (BAP) syntax. Although this approach preserves glycan stereochemistry, assembling JSON input files manually is challenging due to the diversity of monosaccharides, linkages, branching, and structural complexity. To simplify this process, we developed JAAG (JSON input file Assembler for AlphaFold 3 with Glycan integration (https://biofgreat.org/JAAG and https://github.com/chinchc/JAAG)), a web-based graphical interface that streamlines AF3 JSON file creation, reduces errors, and facilitates modeling of glycans and glycan-macromolecule interactions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.08.681238v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Optimal Gene Panel Selection for Targeted Spatial Transcriptomics Experiments</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lu, H., Fang, L., Zeng, O., Ma, P., Zhong, W., Yuan, G.-C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a novel method, named ReconST, to automatically design optimal gene panels for spatial transcriptomics profiling, and identify the optimal subset of genes by using a gated autoencoder.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Spatial transcriptomics analysis is a powerful approach for dissecting the structure of tissue microenvironment and uncovering the mechanism of cell-cell communications. However, existing technologies are limited by either spatial resolution or gene coverage. Most single-cell resolution technologies target only a few hundreds of pre-selected genes, whose choice plays an important role in the overall analysis. It remains a challenge to optimally design a gene panel to maximize the utility of spatial transcriptomics profiling. To fill this gap, we introduce a novel method, named ReconST, to automatically design optimal gene panels for spatial transcriptomics profiling. ReconST leverages information from existing scRNA-seq data, and identifies the optimal subset of genes by using a gated autoencoder. By using a high-coverage mouse brain MERFISH dataset as the reference benchmark, we showed that ReconST outperforms existing methods in terms of both reconstruction accuracy and spatial pattern preservation. As such, ReconST provides a useful and generally-applicable tool for optimal gene panel design, which in turn can significantly enhance the utility of spatial transcriptomics profiling in a wide range of biomedical investigations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.08.681071v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Sequence-free landscape inference for directed evolution</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Towers, S., James, J., Steel, H., Kempf, I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose SLIDE, Sequence-free Landscape Inference for Directed Evolution, which consists of two parts: First, SLIDE provides an estimation for landscape ruggedness from a mutating population using only population-level phenotypic data and an estimation of mutation rate.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Directed evolution is a method for engineering biological systems or components, such as proteins, wherein desired traits are optimised through iterative rounds of mutagenesis and selection of fit variants. The process of protein directed evolution can be envisaged as navigation over high-dimensional landscapes with numerous local maxima. The performance of any strategy in navigating such a landscape is dependent on the ruggedness of that landscape. However, this information is generally unavailable at the outset of an experiment, and cannot currently be computed using analytical methods. Here we propose SLIDE, Sequence-free Landscape Inference for Directed Evolution, which consists of two parts. First, SLIDE provides an estimation for landscape ruggedness from a mutating population using only population-level phenotypic data and an estimation of mutation rate. Ruggedness information in itself is valuable in protein design, for instance in predicting evolutionary stability. Second, SLIDE offers a framework for using the estimated ruggedness metric to select high-performing parameters for directed evolution control. Using theoretical NK landscapes and four real-world protein fitness landscapes, we demonstrate improvement upon the performance of standard selection strategies, particularly on rugged landscapes, using a pipeline that could also be combined with emerging AI-based methods for driving direction evolution.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.08.681123v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">From Genomics Alterations to Expression Dynamics: A Hierarchical Multi-Omics Integration Framework with gINTomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Velle, A., Patane', F., Romualdi, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present gINTomics, a hierarchical computational framework that captures directional biological information flow across genomic, transcriptomic, and epigenomic layers.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Despite the abundance of multi-omic integration methods, few adopt a hierarchical approach, which is crucial for understanding how different genomic alterations influence downstream molecular processes. We present gINTomics, a hierarchical computational framework that captures directional biological information flow across genomic, transcriptomic, and epigenomic layers. Unlike existing meta-dimensional approaches, gINTomics follows a multi-staged paradigm reflecting biological hierarchy, modeling molecular modifications by linking genomic alterations to downstream transcriptional consequences. The framework offers flexible integration strategies for different omic combinations and tailored statistical modeling with optional Random Forest for variable selection. A key feature is the comprehensive interactive Shiny application that provides intuitive visualization through interactive plots and circos diagrams, enabling users to explore complex regulatory relationships across multiple analytical perspectives. Validation on TCGA ovarian cancer data confirmed the framework's effectiveness in identifying biologically relevant regulatory mechanisms and prognostic biomarkers. Available as a Bioconductor package, gINTomics provides researchers with a powerful and accessible tool for mechanistic multi-omic integration and biomarker discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.07.680960v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">From Metabolism to Malignancy: Profiling Diabetes-Related Genes in Hepatocellular Carcinoma</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sheraz, M., Shakoor, D., Hayat, K., Tariq, H., Khan, H., Malik, M. F., Haq, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A genetic link between Type 1 Diabetes and HCC prognosis, with HLA-DOB and HLADPB1 among the most significant genes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Hepatocellular carcinoma (HCC) and diabetes mellitus both affect the liver, a key metabolic organ. This study uses bioinformatics to explore genetic links between Type 1 Diabetes (T1D) and HCC prognosis. Eleven HCC gene expression datasets from GEO and one TCGA-LIHC dataset were analyzed. Gene Set Enrichment Analysis (GSEA) identified up- and downregulated genes after data normalization in R. Four datasets (GSE64041, GSE78737, GSE107170, TCGA) revealed increased expression of T1D-related genes. Ten genes, including HLA-DOB and HLA-DPB1, were consistently upregulated. Statistical analysis (Kruskal-Wallis and Mann-Whitney U tests) showed these two genes were significantly associated with tumor grade and T-stage, with p-values ranging from 0.008 to 0.019. Co-expression analysis with 96 literature-curated T1D genes identified 23 related genes. Survival analysis using Kaplan-Meier curves highlighted five genes (IL7R, CD69, CCR5, RUNX3, PRF1), with CD69 showing strong associations with T-stage and disease-free survival. PRF1, RUNX3, and CCR5 were also linked to survival outcomes. Seven diabetes-related GEO datasets were used for validation. GSEA showed T1D gene enrichment in two datasets (GSE228267, GSE232310), with HLA-DOB significantly expressed in GSE228267 (p = 0.004). These findings suggest that HLA-DOB, HLA-DPB1, and three other T1D-related genes may serve as potential biomarkers for understanding the genetic connection between T1D and HCC. Though computational, this study lays the groundwork for future experimental validation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.07.680510v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">NUCGEN3D: A synthetic framework for large-scale 3D nuclear segmentation with open-source training data and models</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Grandgirard, E., Dmitrasinovic, T., Barreau, C., Sengenes, C., Serrurier, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present NucGen3D, a customizable simulation framework that generates large-scale, annotated 3D microscopy datasets from limited 2D input, specifically the 2018 Data Science Bowl dataset.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Robust nuclear segmentation in 3D microscopy images is a critical yet unresolved challenge in quantitative cell biology, hindered by the scarcity and variability of annotated volumetric datasets. Because such data are difficult to obtain, most state-of-the-art approaches, including Cellpose, segment individual 2D slices and then heuristically reconstruct 3D volumes, thereby losing critical spatial context. Our analysis of human expert annotator performance confirms that ignoring 3D context introduces substantial variability in nuclear detection and annotation. While a few 3D models have been trained on small or toy datasets, no large-scale, openly available resource currently exists to enable robust training of high-capacity 3D segmentation networks. To address this, we present NucGen3D, a customizable simulation framework that generates large-scale, annotated 3D microscopy datasets from limited 2D input, specifically the 2018 Data Science Bowl dataset. NucGen3D produces realistic 3D volumes across diverse biological and imaging scenarios, including variations in nuclear morphology, spatial arrangement, acquisition artifacts, and imaging noise. Using this synthetic data, we trained two models from scratch: a 2D convolutional neural network under Cellpose-like conditions, and a fully 3D convolutional model that extends the 2D settings. We evaluated both on a challenging, independent real-world dataset with complex nuclear architectures. Both models, especially the 3D model, consistently outperformed state-of-the-art methods, including those trained on larger annotated datasets or based on more complex architectures. These results demonstrate that synthetic data can effectively substitute for real 3D annotations in training performing models at scale. To promote reproducibility and further research, we release both the NucGen3D framework and the fully trained 3D segmentation model as open source, making this the first end-to-end open resource for large-scale 3D nuclear segmentation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.08.681092v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Boosting the speed and accuracy of protein quantification algorithms in mass spectrometry-based proteomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pham, T. V., Tran, C. T., Henneman, A. A., Pham, L. H., Le, D. G., Can, A. H., Bui, P. H., Piersma, S. R., Jimenez, C. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a new data structure that supports parsing of input as data stream, improve state of the art quantitation methods in performance by orders of magnitudes, and provide a generic method to boost the precision and accuracy by fragment weighting.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Current methods for protein level quantification in mass spectrometry-based proteomics do not scale with the increasing number of samples because of limited system memory and algorithmic complexities. Here we propose a new data structure that supports parsing of input as data stream, improve state of the art quantitation methods in performance by orders of magnitudes, and provide a generic method to boost the precision and accuracy by fragment weighting.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.06.680769v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Deconvolving mutation and selection reveals stage-specific drivers of thyroid cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rajaei, M., Ju, A., Adeniran, A. J., Townsend, J. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We quantify cancer effect sizes accounting for substantial gene- and site-specific mutation rates, to quantify somatic selection across stages of thyroid cancer.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Objective: Mutational profiles of primary and metastatic thyroid cancer (THCA) have been examined by comparison of somatic mutation prevalences in the two stages, using P values to identify significant differences. However, prevalences and P values for mutations do not directly quantify the cancer effects of somatic variants. Methods: We calculated cancer effect sizes accounting for substantial gene- and site-specific mutation rates, to quantify somatic selection across stages. This approach provides a direct measure of cancer-driving impact of new mutations and reveals the genes that drive tumorigenesis and progression, and whether mutations have greater effects in primary or metastatic THCA. Results: Trinucleotide mutation profiles were similar between primary and metastatic THCA. Most canonical driver genes (BRAF, NRAS, TP53, ATM, EIF1AX, KMT2C, NF1, RBM10, ARID1A, PIK3CA, and NKX2-1) exhibited stronger selection during initial tumorigenesis (from organogenesis to primary THCA) than during progression (from primary to metastatic THCA). Notably, TERT mutations have been shown to be at higher prevalence in metastatic tumors, yet their strongest selection occurred earlier, during tumor initiation. In contrast, RET mutations exhibited the opposite trajectory, experiencing weaker selection during tumorigenesis but stronger selection during metastatic progression. Conclusion: Cancer effect size analysis revealed dynamic shifts in selective pressures across THCA evolution, distinguishing genes that drive initiation from those that promote metastatic progression. This evolutionary framework provides a quantitative basis for understanding THCA pathogenesis and provides guidance for stage-specific precision-medicine therapeutic strategies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.08.681166v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Co-encapsulated liposomal formulations of tubacin and erlotinib.HCl: a comprehensive stability strategy</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Schelker, C., Monteiro, C., Borchard, G., Nowak-Sliwinska, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A preliminary study on the formulation stability of liposomes co-encapsulating tubacin and erlotinib.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The co-encapsulation of two anticancer drugs within liposomes is a promising strategy for delivering a synergistic drug ratio to tumors. In this preliminary study, we systematically investigated the formulation stability of liposomes co-encapsulating tubacin and erlotinib.HCl, as well as the development of a freeze-drying (FD) method as a stabilization strategy. Notably, the effect of cryoprotectants on size and drug encapsulation conservation was studied both before and after freezing and thawing (FD). Tubacin was passively loaded into the lipid bilayer, while erlotinib.HCl was remotely loaded via an ammonium sulfate gradient. Lipid composition studies revealed that erlotinib.HCl encapsulation was the highest at pH 3. Additionally, DSPC:Chol ratios significantly impacted drug loading: high cholesterol content improved erlotinib.HCl retention but reduced tubacin encapsulation. Despite optimization, substantial drug losses occurred during storage at 4 deg C, leading to the development of freeze-drying as a stabilization strategy. Sucrose and trehalose were evaluated as cryoprotectants at different concentrations. Sucrose (10% w/v) demonstrated higher cryoprotection, maintaining liposome sizes within a range optimal for the enhanced permeability and retention (EPR) effect. However, the inclusion of internal sucrose did not further improve stability. Process optimization identified 150 mM ammonium sulfate as the optimal concentration for remote loading while preserving formulation stability. Comparative evaluation of two freeze-drying protocols revealed that the controlled stepwise system (Telstar) outperformed the alternative approach, demonstrating superior drug retention and particle size preservation. Asymmetric flow field-flow fractionation coupled with dynamic light scattering and multi-angle light scattering (AF4-DLS-MALS) analysis confirmed minimal particle aggregation post-lyophilization, with preserved monodispersity and spherical morphology. These findings highlight the critical importance of cryoprotectant selection, buffer optimization, and freeze-drying protocol design for developing stable, co-encapsulated liposomal formulations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.08.681158v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Histology and spatial transcriptomic integration revealed infiltration zone with specific cell composition as a prognostic hotspot in glioblastoma</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fidon, L., Lubrano, M., Hoffmann, C., Baena, E., Thiriez, C., Ayestas, L. C., Cornish, A. J., Madissoon, E., Ducret, V., Esposito, C., Durand, E. Y., Schmauch, B., Pronier, E., Romagnoni, A., Espin-Perez, A., MOSAIC consortium,, Eckstein, M., Maragkou, T., Shelan, M., Youssef, A., Watson, S., Sanson, M., Bielle, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose an interpretable machine learning (ML) methodology for the discovery of histological biomarkers associated with end-of-spectrum overall survival (OS) to provide novel biological insight with a translational potential.</p><p style="font-size: 16px; text-align: justify;">Background: Glioblastoma (GBM), the most aggressive primary brain tumor, has a median survival of approximately 15 months. Twenty percent of patients survive beyond three years, but known clinical factors like age, performance status, resection extent, and MGMT promoter methylation status do not fully explain the observed outcomes. Objective: Our objective was to identify novel histology derived biomarkers associated with end-of-spectrum overall survival (OS) to provide novel biological insight with a translational potential. Methods: We analyzed a total of 748 GBM patients from 3 different cohorts, uniquely enriched in long survivors (n=98 with overall survival (OS) > 5y including n=196 with OS[‚â•]3y), with clinical data and H&E slides obtained from the primary tumor at baseline. We propose an interpretable machine learning (ML) methodology for the discovery of histological biomarkers. Our method learned to segment each H&E slide into three distinct regions associated with long-term survival, short-term survival, and non-informative tissue. We characterized these regions by integrating unsupervised learning, nuclei segmentation, blood vessels detection, pathologist annotations, and multimodal data including spatial transcriptomics from n=31 patients of the GBM MOSAIC dataset to discover fully interpretable biomarkers. Results: Our OS prediction model using histology and clinical data as input achieved an area under the curve (AUC) of 0.85 for the classification of patients between OS<2 and OS[‚â•]3y in external cohort validation, outperforming significantly models trained on clinical data or on histology alone (AUC of 0.76; 0.73, respectively). Two novel biomarkers were predicting poor survival: the presence of regions of lowly infiltrated white matter enriched in malignant cells with a mesenchymal-like phenotype, and lower levels of angiogenesis associated with higher hypoxia response in the main tumor regions. We also found that a subtype of immunosuppressive tumor macrophages - defined by high PLIN2 expression and lipid accumulation- is consistently enriched in histological areas predictive of poor prognosis. Conclusion: Our interpretable ML methodology identified a novel prognostic impact of biological processes and cell types according to distinct tumor regions of GBM. These results pave the way for spatially-informed biomarkers to improve risk stratification and for personalized spatially-targeted therapeutic strategies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.08.681087v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Riluzole as a Dual-Targeted Radiosensitizer for Osteosarcoma: Targeting Tumor Cells and Angiogenic Vasculature to Enhance Single High Dose Radiotherapy Efficacy</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rao, P. P., Herbert, C., Azeem, S. M., Garray, E., Ho, G., Munira, R., Askarifirouzja, H., Haimovitz-Friedman, A., Mahajan, S. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that Riluzole, the FDA-approved drug for Amyotrophic Lateral Sclerosis, is a promising radiosensitizer for osteosarcoma, with the potential to improve SDRT efficacy by overcoming both tumor-intrinsic and microvasculature</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Osteosarcoma is a highly aggressive bone malignancy primarily affecting children and young adults. It presents significant treatment challenges due to its inherent resistance to conventional fractionated radiotherapy (CFRT). Single high dose radiation therapy (SDRT) has promise for the treatment of radioresistant sarcomas, especially those characterized with extensive vascularity. However, its clinical application is severely constrained by toxicity to adjacent critical tissues. Radiosensitizers can enhance tumor cell susceptibility to radiation-induced DNA damage, improving therapeutic efficacy and potentially reducing collateral toxicity. Monotherapies targeting tumor vasculature alone in solid tumors have shown limited success as radiosensitizers in clinical settings. This highlights the importance of compounds that can simultaneously target both tumor cells and its associated microvasculature to maximize the therapeutic outcome to SDRT. Riluzole, the FDA-approved drug for Amyotrophic Lateral Sclerosis, is currently under investigation as a therapeutic agent for osteosarcoma. Riluzole acts to inhibit glutamate release, reduce glutathione levels in cancer cells, and mitigate tumor angiogenesis, positioning it as a potent radiosensitizing agent for the treatment of osteosarcoma. We hypothesize that Riluzole enhances osteosarcoma radiosensitivity to SDRT by simultaneously targeting intrinsic tumor radioresistance and pro-angiogenic signaling. Our findings demonstrate that Riluzole radiosensitizes osteosarcoma cells in vitro by reducing clonogenic survival and enhancing apoptosis. Mechanistically, Riluzole potentiates irradiation-induced reactive oxygen species (ROS) production, induces G2/M phase cell cycle arrest, inhibits DNA repair, and thereby amplifies radiation-induced DNA damage. Additionally, Riluzole suppresses radiation-induced Vascular Endothelial growth factor A (VEGFA) expression indicating its ability to overcome endothelial cell mediated radioresistance. Collectively, these results establish Riluzole as a promising radiosensitizer for osteosarcoma, with the potential to improve SDRT efficacy by overcoming both tumor-intrinsic and microvasculature-mediated radioresistance.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.07.681036v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Associations of the intestinal microbiota with plasma bile acids and inflammation markers in Crohn's disease and ulcerative colitis.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Stefanie Prast-Nielsen, Anna L√∂f Granstr√∂m, Ali Kiasat, Gustav Ahlstr√∂m, Gabriella Edfeldt, Susanne Rautiainen, Fredrik Boulund, Fredrik O Andersson, Johan Lindberg, Ina Schuppe-Koistinen, Ulf O Gustafsson, Lars Engstrand</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study explores signatures for Crohn's disease (CD) and Ulcerative Colitis (UC) reflecting an interplay between the intestinal microbiota, systemic inflammation, and plasma bile acid homeostasis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Our study explores signatures for Crohn's disease (CD) and Ulcerative Colitis (UC) reflecting an interplay between the intestinal microbiota, systemic inflammation, and plasma bile acid homeostasis. For this, 1,257 individuals scheduled for colonoscopy were included and completed a comprehensive questionnaire. Individuals with IBD ('CD' n = 64 and 'UC' n = 55), were age- and gender-matched to controls without findings during colonoscopy. Shotgun metagenomic profiles of the fecal microbiota and plasma profiles of inflammatory proteins and bile acids were used to build disease classifiers. Omics integration identified associations across datasets. B. hydrogenotrophica was associated with CD and C. eutactus, C. sp. CAG167, B. cellulosilyticus, C. mitsuokai with controls. Ten inflammation markers were increased in CD, and eleven bile acids and derivatives were decreased in CD, while 7a-Hydroxy-3-oxo-4-cholestenoate (7-HOCA) and chenodeoxycholic acid (CDCA) were increased compared to controls.In UC, commensals such as F. prausnitzii and A. muciniphila were depleted. CCL11, IL-17A, and TNF were increased in UC and associated to gut microbial changes. Correlations between taxa and bile acids were all positive. For both CD and UC, taxonomic differences were primarily characterized by a reduction in commensal gut microbes which exhibited positive correlations with secondary bile acids and negative correlations with inflammation markers.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41062543/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Spatial proteomics for investigating solid tumor resistance mechanisms.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xin Ming M Zhou, Anjali J D'Amiano, Charles Lu, Vrinda Madan, Sara Khoshniyati, Jack Kollings, Noah E Sunshine, Sachin S Surwase, Joel C Sunshine</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Spatial proteomics technologies have been pivotal in profiling tumor immune microenvironments at single-cell resolution, advancing our understanding of cancer biology, identifying key cell populations in solid tumors, and predicting treatment responses.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Spatial proteomics technologies have been pivotal in profiling tumor immune microenvironments at single-cell resolution, advancing our understanding of cancer biology, identifying key cell populations in solid tumors, and predicting treatment responses. Although immune checkpoint and molecular inhibitors have revolutionized cancer care, resistance mechanisms remain a major therapeutic challenge that hinder productive responses in a notable fraction of cancer patients. In this review, we outline current spatial proteomics and computational analysis tools for studying the tumor immune microenvironment and discuss how spatial proteomics techniques have helped elucidate cancer resistance mechanisms across multiple tumor types. In this process, we highlight the importance of investigating immunosuppressive cell populations that can mediate cancer resistance, specifically with regard to their localization, protein signatures, and surrounding interactions. Finally, we provide a look ahead at future applications of artificial intelligence/machine learning and multi-omics approaches that will help further propel our understanding of cancer resistance mechanisms through spatial biology research.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41060425/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

