<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Monday, 06 October 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Mapping the evolution of computationally designed protein binders</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Alcantar, M. A., Paulk, A. M., Moradi, S., Bhar, D., Keller, G. L. J., Sanyal, T., Bai, H., Camdere, G., Han, S. J., Jain, M., Jew, B., Vatansever Inak, S., Langmead, C. J., Tinberg, C. E., Chen, I., Liu, C. C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We use OrthoRep, a system for continuous hypermutation in vivo, to drive the evolution of computationally designed mini protein binders ("minibinders") that target a mammalian receptor.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Computational protein design enables the generation of binders that target specific epitopes on proteins. However, current approaches often require substantial screening from which hits require further affinity maturation. Methods for experimentally improving designed proteins and exploring their sequence-affinity landscapes could therefore streamline the development of high-affinity binders and inform future design strategies. Here, we use OrthoRep, a system for continuous hypermutation in vivo, to drive the evolution of computationally designed mini protein binders ("minibinders") that target a mammalian receptor. Despite their small sizes (59-72 amino acids), we successfully affinity matured multiple minibinders through strong selection for improved binding and also sampled new regions of minibinder fitness landscapes through extensive neutral drift. One evolved minibinder variant was used to construct a combinatorially complete sequence-affinity map for its six affinity increasing mutations, which revealed nearly full additivity in their contributions to binding. Another minibinder was subjected to both deep mutational scanning and extensive evolution under weak selection, resulting in an evolutionarily diverged collection of binder sequences that revealed non-additive relationships among mutations. Our results highlight that the affinity of computationally designed binders can be rapidly increased through evolution and provide a scalable approach for the evolutionary exploration and subsequent mapping of sequence-affinity landscapes. We suggest that this work will complement protein binder design both as a reliable experimental optimization process and as a vehicle for generating new training data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.04.680454v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Gradient-aware modeling advances AI-driven prediction of genetic perturbation effects</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jerby, L., Zhu, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present GARM (Gradient Aligned Regression with Multi-decoder), a machine learning (ML) framework that leverages gradient-aware supervision to capture both absolute and relative perturbational effects.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Predicting the transcriptional effects of genetic perturbations across diverse contexts is a central challenge in functional genomics. While single-cell perturbational assays such as Perturb-seq have generated valuable datasets, exhaustively profiling all perturbations is infeasible, underscoring the need for predictive models. We present GARM (Gradient Aligned Regression with Multi-decoder), a machine learning (ML) framework that leverages gradient-aware supervision to capture both absolute and relative perturbational effects. Across multiple large-scale datasets, GARM consistently outperforms leading approaches, including GEARS, scGPT, and GenePert, in predicting responses to unseen perturbations within and across contexts. Complementing this, we show that widely used evaluation metrics substantially overestimate performance, allowing trivial models to appear predictive. To address this, we introduce perturbation-ranking criteria (PrtR) that better reflect model utility for experimental design. Finally, we provide insight into gene-specific predictability, revealing pathways and gene classes systematically easier or harder to predict, with implications for model development and biological interpretation. Together, these advances establish a unified methodological and conceptual framework that improves perturbation modeling, sets rigorous evaluation standards, and provides biological insight into gene-specific predictability in functional genomics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.03.680360v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Facial photographs as proxies for inflammatory aging</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Belic, M., Schneider, K., Furman, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed Healthy Selfie, a digital predictor of iAge, from simple 2D frontal face photographs, and showed that ordinary facial features can predict blood inflammation and can be used as a scalable, ultra low-cost tool for biological aging and advancing precision health.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Systemic chronic inflammation is a major determinant of aging and disease risk, yet current biomarkers such as the Inflammatory Age (iAge) clock and other proteomic assays remain costly, invasive, and poorly scalable. The skin, as both a visible marker and contributor to age-related inflammation (inflammaging), offers a potential non-invasive window into this health metric. Here, we developed Healthy Selfie, a digital predictor of iAge, from simple 2D frontal face photographs. We leveraged data from the Edifice Health Pre-Market trial, a clinical study of 750 participants aged 20 to 90 years, using a subset with available iAge measurements paired to facial images, demographic, clinical, and functional data. Facial embeddings from a pre-trained deep learning image model were combined with chronological age, sex and other easily obtainable metadata, and mapped to blood-derived iAge using regression within a leave-one-out cross-validation framework. Photo-predicted iAge values were significantly correlated with blood iAge values (r = 0.536) and were used to identify increased iAge acceleration (accuracy 66.39%, sensitivity 65.61%, specificity 67.24%). Validation on an external dataset containing ~100,000 images showed no significant demographic bias across ethnicities, including Asian, Black, Indian, Middle Eastern, and Latino populations. Beyond iAge, we also showed that facial features could predict blood levels of individual inflammaging proteins (CXCL9, CXCL1, CCL11, TNFSF10, and IFNG). Our findings suggest that ordinary facial photos can provide information on blood inflammation and can be used as a scalable, ultra low-cost tool for assessing biological aging and advancing precision health.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.02.680162v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Structure of the Gq-coupled adhesion receptor ADGRL4, a GPCR implicated in cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chen, Q., Gusach, A., Diamante, A., Patel, J. C., Edwards, P. C., Tate, C. G., Favara, D. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that ADGRL4 is a G protein-coupled receptor with weak coupling to G protein coupled pathways and show that it can be inhibitors of tumour pathogenesis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Adhesion G protein-coupled receptors (aGPCRs) are a 32-member family of Class B GPCRs that have diverse cellular roles including mechanosensation, cell-fate determination, neurodevelopment, immune function and tumour biology. ADGRL4 is upregulated in the tumour microenvironment and is implicated in tumour pathogenesis across a broad range of malignancies. Inhibiting ADGRL4 is a potential therapeutic treatment for currently intractable cancers such as glioblastoma. Previous work suggested that ADGRL4 does not signal through G protein coupled pathways. However, using a sensitive bioluminescent assay, we demonstrate here that ADGRL4 couples weakly to the heterotrimeric G protein Gq, whilst there was no robust coupling to other G proteins (Gs, G12, Go) or {beta}-arrestin 1 or 2. We determined the cryo-EM structure of ADGRL4 coupled to a heterotrimeric Gq complex to a resolution of 3.1 [A]. The overall fold of ADGRL4 is similar to that of other aGPCRs, but the coupling to Gq is distinct with fewer interactions between the receptor and G protein. The structure is consistent with ADGRL4 being activated by its tethered agonist and represents an important step towards the development of potential inhibitors for the treatment of multiple tumour types.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.03.680274v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Inducing TRIB2 Targeted Protein Degradation to Reverse Chemoresistance in Acute Myeloid Leukemia</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rigby, E., Narayanan, A., Kania, E., Harris, J. A., Williams, J., Zhang, B., Liu, L., Richmond, L., Zhou, F., Ding, K., Carmody, R., Eyers, P. A., Keeshan, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify Afatinib, a multi-ERBB covalent inhibitor, as a rapid inducer of TRIB2 degradation, triggering AML cell death via an ERBB-independent pathway.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The myeloid oncogene TRIB2 is a key driver of acute myeloid leukaemia (AML) pathogenesis, promoting chemoresistance and blocking differentiation through ubiquitin-mediated degradation of the C/EBP transcription factor. Despite its stable and sometimes elevated expression across AML subtypes, TRIB2 remains a clinically-untargeted vulnerability. Here, we present a comprehensive investigation into TRIB2 degradation mechanisms using multimodal approaches, including CRISPR knockout, mutational protein stability, small molecule TRIB2 engagement and evaluation of a novel targeted protein degrader (TRIB2-PROTAC). We identify Afatinib, a multi-ERBB covalent inhibitor, as a rapid inducer of TRIB2 degradation, triggering AML cell death via an ERBB-independent pathway. Importantly, TRIB2 degradation synergized with cytarabine, the frontline AML chemotherapy, amplifying therapeutic efficacy. Mapping of TRIB2 ubiquitination sites revealed Lys-63 as critical for its own proteolytic turnover, and a Lys to Arg degradation-resistant mutant (KallR) conferred enhanced chemoresistance and increased leukaemic engraftment in vivo. CRISPR-mediated TRIB2 knockout validated an essential role in AML cell survival. Consistently, the novel TRIB2-PROTAC (compound 5K) achieved robust TRIB2 degradation and AML cell killing at low micromolar concentrations. These findings establish TRIB2 as a compelling therapeutic target in AML and demonstrate that leveraging the ubiquitin-proteasome system to degrade TRIB2 offers a promising strategy to overcome chemoresistance. This work provides strong preclinical rationale for the development of TRIB2-targeting therapies in AML.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.03.680259v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

