<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Thursday, 24 April 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Longitudinal prediction of DNA methylation to forecast epigenetic outcomes.</h2>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new method for measuring epigenetic changes in DNA methylation in early life .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Epigenetic changes in early life play an important role in the development of health conditions in children. Longitudinally measuring and forecasting changes in DNA methylation can reveal patterns of ageing and disease progression, but biosamples may not always be available.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40267756/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The Signed Two-Space Proximity Model for Learning Representations in Protein-Protein Interaction Networks.</h2>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the Signed Two-Space Proximity Model (S2-SPM) for signed PPI networks, which explicitly incorporates both positive and negative interactions, reflecting the complex regulatory mechanisms within biological systems.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurately predicting complex protein-protein interactions (PPIs) is crucial for decoding biological processes, from cellular functioning to disease mechanisms. However, experimental methods for determining PPIs are computationally expensive. Thus, attention has been recently drawn to machine learning approaches. Furthermore, insufficient effort has been made toward analyzing signed PPI networks, which capture both activating (positive) and inhibitory (negative) interactions. To accurately represent biological relationships, we present the Signed Two-Space Proximity Model (S2-SPM) for signed PPI networks, which explicitly incorporates both types of interactions, reflecting the complex regulatory mechanisms within biological systems. This is achieved by leveraging two independent latent spaces to differentiate between positive and negative interactions while representing protein similarity through proximity in these spaces. Our approach also enables the identification of archetypes representing extreme protein profiles.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40266854/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Custom CRISPR-Cas9 PAM variants via scalable engineering and machine learning.</h2>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> The first generalist CRISPR-Cas enzymes were developed in the 1970s, and are used in the discovery of new proteins.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Engineering and characterizing proteins can be time-consuming and cumbersome, motivating the development of generalist CRISPR-Cas enzymes</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40262634/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The Dawn of High-Throughput and Genome-Scale Kinetic Modeling: Recent Advances and Future Directions.</h2>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A review of the state-of-the-art in kinetic modeling, highlighting the potential of these advances to drive progress in systems and synthetic biology, metabolic engineering, and medical research at an unprecedented scale and pace.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Researchers have invested much effort into developing kinetic models due to their ability to capture dynamic behaviors, transient states, and regulatory mechanisms of metabolism, providing a detailed and realistic representation of cellular processes. Historically, the requirements for detailed parametrization and significant computational resources created barriers to their development and adoption for high-throughput studies. However, recent advancements, including the integration of machine learning with mechanistic metabolic models, the development of novel kinetic parameter databases, and the use of tailor-made parametrization strategies, are reshaping the field of kinetic modeling. In this Review, we discuss these developments and offer future directions, highlighting the potential of these advances to drive progress in systems and synthetic biology, metabolic engineering, and medical research at an unprecedented scale and pace.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40262025/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrating machine learning models with multi-omics analysis to decipher the prognostic significance of mitotic catastrophe heterogeneity in bladder cancer.</h2>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Mitotic catastrophe is well-known as a major pathway of endogenous tumor death, but the prognostic significance of its heterogeneity regarding bladder cancer (BLCA) remains unclear.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mitotic catastrophe is well-known as a major pathway of endogenous tumor death, but the prognostic significance of its heterogeneity regarding bladder cancer (BLCA) remains unclear.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40259382/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrated multiomics analysis identifies potential biomarkers and therapeutic targets for autophagy associated AKI to CKD transition.</h2>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified 14 autophagy-related genes and their immune infiltration during the transition from acute kidney injury to chronic kidney disease (CKD).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> This study explored the relationship between acute kidney injury (AKI) and chronic kidney disease (CKD), focusing on autophagy-related genes and their immune infiltration during the transition from AKI to CKD. We performed weighted correlation network analysis (WGCNA) using two microarray datasets (GSE139061 and GSE66494) in the GEO database and identified autophagy signatures by Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG), and GSEA enrichment analysis. Machine learning algorithms such as LASSO, random forest, and XGBoost were used to construct the diagnostic model, and the diagnostic performance of GSE30718 (AKI) and GSE37171 (CKD) was used as validation cohorts to evaluate its diagnostic performance. The study identified 14 autophagy candidate genes, among which ATP6V1C1 and COPA were identified as key biomarkers that were able to effectively distinguish between AKI and CKD. Immune cell infiltration and GSEA analysis revealed immune dysregulation in AKI, and these genes were associated with inflammation and immune pathways. Single-cell analysis showed that ATP6V1C1 and COPA were specifically expressed in AKI and CKD, which may be related to renal fibrosis. In addition, drug prediction and molecular docking analysis proposed SZ(+)-(S)-202-791 and PDE4 inhibitor 16 as potential therapeutic agents. In summary, this study provides new insights into the relationship between AKI and CKD and lays a foundation for the development of new treatment strategies.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40258914/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrated bioinformatic analysis of immune infiltration and disulfidptosis related gene subgroups in type A aortic dissection.</h2>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study investigates the role of disulfidptosis-related genes in immune infiltration in type A aortic dissection and identifies potential therapeutic targets for TAAD.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Type A aortic dissection (TAAD) is a lethal cardiovascular disease characterized by the separation of the layers within the aortic wall. The underlying pathological mechanisms of TAAD requires further elucidation to develop effective prevention and pharmacological treatment strategies. Inflammation plays a crucial role in TAAD pathogenesis. Disulfidptosis, an emerging type of cell death, may shed light on disease mechanisms. This study investigates the role of disulfidptosis-related genes in immune infiltration in TAAD. TAAD gene expression datasets were obtained from the Gene Expression Omnibus (GEO) database. Immune cell infiltration analysis assessed immune cell dysregulation in TAAD. Differentially expressed genes (DEGs) between TAAD samples and controls were identified and intersected with known disulfidptosis-related gene sets to obtain relevant DEGs. Hub genes were identified using machine learning algorithms. A diagnostic model was constructed using Least Absolute Shrinkage and Selection Operator (LASSO) regression on 25 TAAD samples. Consensus clustering classified TAAD samples based on disulfidptosis-related gene expression. Functional enrichment analyses, including Gene Ontology (GO) terms and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway analyses, elucidated associated biological processes and pathways. A total of 13,316 DEGs were identified, among which 11 disulfidptosis-related genes were screened: INF2, CD2AP, PDLIM1, ACTN4, MYH10, MYH9, FLNA, FLNB, TLN1, MYL6, ACTB, CAPZB, DSTN, and IQGAP1. Most of these genes exhibited lower expression levels in TAAD samples, except CAPZB, and were correlated with immune cell infiltration. Cluster-specific DEGs were found in one cluster, involving several immune response processes. Co-clustering analysis based on disulfidptosis-related genes classified TAAD samples into two clusters, with higher gene expression levels observed in cluster C2 compared to cluster C1. Three key hub genes were identified, and potential therapeutic mechanisms for TAAD were explored. Immuno-infiltration results revealed significant differences in immune profiles, with higher immunological scores and more extensive immune infiltration in TAAD. Disulfidptosis occurs in TAAD and is associated with immune cell infiltration and metabolic activity, influencing immune cell function and responses. These findings suggest that disulfidptosis may promote TAAD progression through the induction of immune responses and metabolic activities. This research provides new insights into the pathogenesis and identifies potential therapeutic targets for TAAD.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40258895/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Deconvolution of cell types and states in spatial multiomics utilizing TACIT.</h2>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> TACIT, an unsupervised algorithm for cell annotation using predefined signatures that operates without training data, identifies under- and overrepresented immune cell types and states in regions of interest, and discovers new phenotypes in inflammatory gland diseases.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Identifying cell types and states remains a time-consuming, error-prone challenge for spatial biology. While deep learning increasingly plays a role, it is difficult to generalize due to variability at the level of cells, neighborhoods, and niches in health and disease. To address this, we develop TACIT, an unsupervised algorithm for cell annotation using predefined signatures that operates without training data. TACIT uses unbiased thresholding to distinguish positive cells from background, focusing on relevant markers to identify ambiguous cells in multiomic assays. Using five datasets (5,000,000 cells; 51 cell types) from three niches (brain, intestine, gland), TACIT outperforms existing unsupervised methods in accuracy and scalability. Integrating TACIT-identified cell types reveals new phenotypes in two inflammatory gland diseases. Finally, using combined spatial transcriptomics and proteomics, we discover under- and overrepresented immune cell types and states in regions of interest, suggesting multimodality is essential for translating spatial biology to clinical applications.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40258827/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Design of nanobody targeting SARS-CoV-2 spike glycoprotein using CDR-grafting assisted by molecular simulation and machine learning.</h2>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a complementarity-determining region (CDR) grafting approach for designing nanobodies (Nbs) that target specific epitopes, with the aid of computer simulation and machine learning.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The design of proteins capable effectively binding to specific protein targets is crucial for developing therapies, diagnostics, and vaccine candidates for viral infections. Here, we introduce a complementarity-determining region (CDR) grafting approach for designing nanobodies (Nbs) that target specific epitopes, with the aid of computer simulation and machine learning. As a proof-of-concept, we designed, evaluated, and characterized a high-affinity Nb against the spike protein of SARS-CoV-2, the causative agent of the COVID-19 pandemic. The designed Nb, referred to as Nb Ab.2, was synthesized and displayed high-affinity for both the purified receptor-binding domain protein and to the virus-like particle, demonstrating affinities of 9 nM and 60 nM, respectively, as measured with microscale thermophoresis. Circular dichroism showed the designed protein's structural integrity and its proper folding, whereas molecular dynamics simulations provided insights into the internal dynamics of Nb Ab.2. This study shows that our computational pipeline can be used to efficiently design high-affinity Nbs with diagnostic and prophylactic potential, which can be tailored to tackle different viral targets.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40257976/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Ibaqpy: A scalable Python package for baseline quantification in proteomics leveraging SDRF metadata.</h2>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present ibaqpy, a Python package designed to compute iBAQ values efficiently for experiments of any scale while incorporating experimental metadata into the quantification workflow.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Intensity-based absolute quantification (iBAQ) is essential in proteomics as it allows for the assessment of a protein's absolute abundance in various samples or conditions. However, the computation of these values for increasingly large-scale and high-throughput experiments, such as those using DIA, TMT, or LFQ workflows, poses significant challenges in scalability and reproducibility. Here, we present ibaqpy (https://github.com/bigbio/ibaqpy), a Python package designed to compute iBAQ values efficiently for experiments of any scale. Ibaqpy leverages the Sample and Data Relationship Format (SDRF) metadata standard to incorporate experimental metadata into the quantification workflow. This allows for automatic normalization and batch correction while accounting for key aspects of the experimental design, such as technical and biological replicates, fractionation strategies, and sample conditions. Designed for large-scale proteomics datasets, ibaqpy can also recompute iBAQ values for existing experiments when an SDRF is available. We showcased ibaqpy's capabilities by reanalyzing 17 public proteomics datasets from ProteomeXchange, covering HeLa cell lines with 4921 samples and 5766 MS runs, quantifying a total of 11,014 proteins. In our reanalysis, ibaqpy is a key component in automating reproducible quantification, reducing manual effort and making quantitative proteomics more accessible while supporting FAIR principles for data reuse. SIGNIFICANCE: Proteomics studies often rely on intensity-based absolute quantification (iBAQ) to assess protein abundance across various biological conditions. Despite its widespread use, computing iBAQ values at scale remains challenging due to the increasing complexity and volume of proteomics experiments. Existing tools frequently lack metadata integration, limiting their ability to handle experimental design intricacies such as replicates, fractions, and batch effects. Our work introduces ibaqpy, a scalable Python package that leverages the Sample and Data Relationship Format (SDRF) to compute iBAQ values efficiently while incorporating critical experimental metadata. By enabling automated normalization and batch correction, ibaqpy ensures reproducible and comparable quantification across large-scale datasets. We validated the utility of ibaqpy through the reanalysis of 17 public HeLa datasets, comprising over 200 million peptide features and quantifying 11,000 proteins across thousands of samples. This comprehensive reanalysis highlights the robustness and scalability of ibaqpy, making it an essential tool for researchers conducting large-scale proteomics experiments. Moreover, by promoting FAIR principles for data reuse and interoperability, ibaqpy offers a transformative approach to baseline protein quantification, supporting reproducible research and data integration within the proteomics community.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40268243/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Rapid assay development for low input targeted proteomics using a versatile linear ion trap.</h2>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show consistent quantification across three orders of magnitude in a matched-matrix background of low-level proteins such as transcription factors and cytokines in a 1 ng sample without requiring stable isotope-labeled standards.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Advances in proteomics and mass spectrometry enable the study of limited cell populations, where high-mass accuracy instruments are typically required. While triple quadrupoles offer fast and sensitive low-mass specificity measurements, these instruments are effectively restricted to targeted proteomics. Linear ion traps (LITs) offer a versatile, cost-effective alternative capable of both targeted and global proteomics. Here, we describe a workflow using a hybrid quadrupole-LIT instrument that rapidly develops targeted proteomics assays from global data-independent acquisition (DIA) measurements without high-mass accuracy. Using an automated software approach for scheduling parallel reaction monitoring assays (PRM), we show consistent quantification across three orders of magnitude in a matched-matrix background. We demonstrate measuring low-level proteins such as transcription factors and cytokines with quantitative linearity below two orders of magnitude in a 1 ng background proteome without requiring stable isotope-labeled standards. From a 1 ng sample, we found clear consistency between proteins in subsets of CD4</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40263265/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Tsbrowse: an interactive browser for Ancestral Recombination Graphs</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present tsbrowse, an open-source Python web-app for the interactive visualisation of the fundamental building-blocks of Ancestral Recombination Graphs, i.e., nodes, edges and mutations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation Ancestral Recombination Graphs (ARGs) represent the interwoven paths of genetic ancestry for a set of recombining sequences. The ability to capture the evolutionary history of samples makes ARGs valuable in a wide range of applications in population and statistical genetics. ARG-based approaches are increasingly becoming a part of genetic data analysis pipelines due to breakthroughs enabling ARG inference at biobank-scale. However, there is a lack of visualisation tools, which are crucial for validating inferences and generating hypotheses. Results We present tsbrowse, an open-source Python web-app for the interactive visualisation of the fundamental building-blocks of ARGs, i.e., nodes, edges and mutations. We demonstrate the application of tsbrowse to various data sources and scenarios, and highlight its key features of browsability along the genome, user interactivity, and scalability to very large sample sizes. Availability: Python package https://pypi.org/project/tsbrowse/ Development version: https://github.com/tskit.dev/tsbrowse Documentation: https://tskit.dev/tsbrowse/docs</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.23.649987v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Discovery of Novel Trypanothione Reductase Inhibitors through Pharmacophore Modeling, virtual Screening and Molecular Dynamics Simulations. New insights for Human African Trypanosomiasis</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified two promising compounds for the treatment of human African Trypanosomiasis, VS-1 and VS-2, based on superior docking scores and molecular dynamics simulations, which suggest that these compounds are potential inhibitors of T.b TR.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Human African Trypanosomiasis (HAT), caused by Trypanosome brucei, remains a critical health concern, with treatments such as Melarsoprol hampered by toxicity and resistance. To address this, we aimed to identify inhibitors targeting Trypanothione Reductase (T.b TR), a key enzyme in the Trypanosome survival. A pharmacophore model was derived from Melarsoprol and validated using Receiver Operating Characteristic (ROC) curves and Enrichment Factor (EF). Virtual screening of compound libraries identified two promising candidates, VS-1 and VS-2, based on superior docking scores. Subsequent molecular dynamics simulations confirmed the stability of the ligand-enzyme complexes, while binding free energy calculations revealed strong binding affinities for VS-1 and VS-2. Per-residue decomposition pointed critical interactions with active site residues, including MET 260, ASN 130, and HIS 128, contributing to the compounds stability and activity. These findings suggest that VS-1 and VS-2 are potential inhibitors of T.b TR, with better efficacy than Melarsoprol. However, further in vitro studies, including IC50 determination, cell cycle analysis, and morphological assays, are essential to confirm the therapeutic potential of these compounds for the treatment of HAT.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.18.649577v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Ecologically Significant Genetic Loci of P. allii: Contributions to Pathogenesis and Competition</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that onion-pathogenic Pantoea allii strains from onion and rainwater are genetically distinct lineages, with two key virulence genes from onion-associated strains and two from onion isolate without them.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Pantoea allii, one of four Pantoea species known to cause onion center rot, is infrequently isolated from onion compared to its closely related onion-pathogenic sister taxa. To better understand the genomic diversity and genetic determinants of pathogenicity in this species, we analyzed a collection of 38 P. allii strains isolated from two primary ecological niches, plants and water, across three continents using comparative genomics and phylogenetic approaches. Core-genome phylogeny, average nucleotide identity (ANI), and gene presence and absence analyses revealed three genetically distinct lineages. All strains harbored conserved biosynthetic gene clusters (BGCs) for quorum sensing, carotenoid production, siderophores, and thiopeptides. In contrast, two phosphonate BGCs, key determinants of onion pathogenicity, exhibited lineage-specific distributions. Onion-associated strains from Lineages 1 and 2 carried the Halophos BGC associated with onion tissue necrosis, and onion isolates encoded the alt gene cluster conferring thiosulfinate tolerance. Lineage 3 strains, isolated from both onion and rainwater, either lacked a phosphonate BGC loci or carried the HiVir phosphonate BGC. In addition, Lineage 3 strains lacked the alt cluster altogether. The localization of these virulence genes in the genome varied, with Halophos integrated in the chromosome, HiVir encoded on the conserved Large Pantoea Plasmid, and alt located on a small, variable plasmids (plasmid B). The Type IV and Type VI secretion systems showed variable genomic architectures, with plasmid-borne T4SSs and two chromosomal T6SS loci differing in conservation and gene content. Additionally, conserved Pantailocin phage islands were detected in most genomes. Overall, this study reveals that while core metabolic and competitive traits are conserved across P. allii, virulence-associated loci display lineage-specific partitioning, reflecting ecological differentiation and evolutionary plasticity within the species.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.23.650278v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A flaw in using pre-trained pLLMs in protein-protein interaction inference models</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify and confirm that existing pre-trained protein large language models (pLLMs) are a source of data leakage for the downstream protein-protein interaction (PPI) inference task.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> With the growing pervasiveness of pre-trained protein large language models (pLLMs), pLLM-based methods are increasingly being put forward for the protein-protein interaction (PPI) inference task. Here, we identify and confirm that existing pre-trained pLLMs are a source of data leakage for the downstream PPI task. We characterize the extent of the data leakage problem by training and comparing small and efficient pLLMs on a dataset that controls for data leakage (''strict'') with one that does not (''non-strict''). While data leakage from pre-trained pLLMs cause measurable inflation of testing scores, we find that this does not necessarily extend to other, non-paired biological tasks such as protein keyword annotation. Further, we find no connection between the context-lengths of pLLMs and the performance of pLLM-based PPI inference methods on proteins with sequence lengths that surpass it. Furthermore, we show that pLLM-based and non-pLLM-based models fail to generalize in tasks such as prediction of the human-SARS-CoV-2 PPIs or the effect of point mutations on binding-affinities. This study demonstrates the importance of extending existing protocols for the evaluation of pLLM-based models applied to paired biological datasets and identifies areas of weakness of current pLLM models. With the growing pervasiveness of pre-trained protein large language models (pLLMs), pLLM-based methods are increasingly being put forward for the protein-protein interaction (PPI) inference task. Here, we identify and confirm that existing pre-trained pLLMs are a source of data leakage for the downstream PPI task. We characterize the extent of the data leakage problem by training and comparing small and efficient pLLMs on a dataset that controls for data leakage (``strict'') with one that does not (``non-strict''). While data leakage from pre-trained pLLMs cause measurable inflation of testing scores, we find that this does not necessarily extend to other, non-paired biological tasks such as protein keyword annotation. Further, we find no connection between the context-lengths of pLLMs and the performance of pLLM-based PPI inference methods on proteins with sequence lengths that surpass it. Furthermore, we show that pLLM-based and non-pLLM-based models fail to generalize in tasks such as prediction of the human-SARS-CoV-2 PPIs or the effect of point mutations on binding-affinities. This study demonstrates the importance of extending existing protocols for the evaluation of pLLM-based models applied to paired biological datasets and identifies areas of weakness of current pLLM models.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.21.649858v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AOP-helpFinder 3.0: from text mining to network visualization of key event relationships, and knowledge integration from multi-sources.</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Adverse Outcome Pathways (AOP) framework advances alternative toxicology by prioritizing the mechanisms underlying toxic effects.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: The Adverse Outcome Pathways (AOP) framework advances alternative toxicology by prioritizing the mechanisms underlying toxic effects. It organizes existing knowledge in a structured way, tracing the progression from the initial perturbation of a molecular event caused by various stressors through key events (KEs) across different biological levels, ultimately leading to adverse outcomes that affect human health and ecosystems. However, the increasing volume of toxicological data presents a significant challenge for integrating all available knowledge effectively. Results: Artificial intelligence provides powerful methods to analyze and integrate large, heterogeneous data sources. Within this framework, the AOP-helpFinder text mining tool, accessible as a web server, was designed to identify stressor-event and event-event relationships by automatically screening scientific literature in the PubMed database, facilitating the development of AOPs. The proposed new version introduces enhanced functionality by incorporating additional data sources, automatically annotating events from the literature with toxicological database information in a systems biology context. Users can now visualize results as interactive networks directly on the web server. With these advancements, AOP-helpFinder 3.0 offers a robust solution for integrative and predictive toxicology, as demonstrated in a case study exploring toxicological mechanisms associated with radon exposure.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.22.648318v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">rnaends: an R package targeted to study the exact RNA ends at the nucleotide resolution</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the rnaends R package which is dedicated to RNA-end sequencing analysis, which offers functions for raw read pre-processing, RNA-ends mapping and quantification, RNA count matrix post-processing and further count matrix downstream analyses such as TSS identification, fast Fourier transform for</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> 5' and 3' RNA-end sequencing protocols have unlocked new opportunities to study aspects of RNA metabolism such as synthesis, maturation and degradation, by enabling the quantification of exact ends of RNA molecules in vivo. From RNA-Seq data that have been generated with one of the specialized protocols, it is possible to identify transcription start sites (TSS) and/or endoribonucleolytic cleavage sites, and even, in some cases, co-translational 5' to 3' degradation dynamics. Furthermore, post-transcriptional addition of ribonucleotides at the 3' end of RNA can be studied at the nucleotide resolution. While different RNA-end sequencing library protocols can vary, and have their own specificities, the generated RNA-Seq data are very similar and share common processing steps. Most importantly, the major aspect of RNA-end sequencing is that only the 5' or 3' end mapped location is of interest, contrary to conventional RNA sequencing that considers genomic ranges for gene expression analysis. This translates to a simple representation of the quantitative data as a count matrix of RNA-end location on the reference sequences. This representation seems under-exploited and is, to our knowledge, not available in a generic package focused on the analyses on the exact transcriptome ends. Here, we present the rnaends R package which is dedicated to RNA-end sequencing analysis. It offers functions for raw read pre-processing, RNA-ends mapping and quantification, RNA-ends count matrix post-processing, and further count matrix downstream analyses such as TSS identification, fast Fourier transform for signal periodic patterns analysis, or differential proportion of RNA-ends analysis. The use of rnaends is illustrated with applications in RNA metabolism studies through selected workflows on published RNA-end datasets: (i) TSS identification, (ii) ribosome translation speed and co-translational degradation, (iii) post-transcriptional modifications analysis and differential proportion analysis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.18.649472v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Development a computational framework for risk evaluation of emerging flu viruses based on genomic sequences and AI</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present FluRisk, an innovative computational framework that integrates viral genomic data with artificial intelligence (AI) to enable rapid and comprehensive risk evaluation of emerging influenza strains.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The recent resurgence of highly pathogenic avian influenza H5N1 viruses in North America and Europe has heightened global concerns regarding potential influenza pandemics. Despite significant progress in the surveillance and prevention of emerging influenza viruses, effective tools for rapid and accurate risk assessment remain limited. Here, we present FluRisk, an innovative computational framework that integrates viral genomic data with artificial intelligence (AI) to enable rapid and comprehensive risk evaluation of emerging influenza strains. FluRisk incorporates a curated database of over 1,000 experimentally validated molecular markers linked to key viral phenotypes, including human adaptation, mammalian virulence, human-to-human transmission, receptor binding specificity, and antiviral drug resistance. Leveraging these markers, we developed three state-of-the-art machine learning models to predict human adaptation, mammalian virulence, and human receptor-binding potential, all of which demonstrated superior performance compared to traditional approaches such as BLAST, prior models, and baseline classifiers. In addition, a reference-based method was implemented to provide preliminary estimates of transmissibility and resistance to six commonly used antiviral drugs. To facilitate broad accessibility and practical application, we developed a user-friendly web server that integrates both the molecular marker atlas and predictive tools for influenza virus phenotyping (available at: http://www.computationalbiology.cn/FluRisk/#/). This computational platform offers a valuable resource for the timely risk assessment of emerging influenza viruses and supports global influenza surveillance efforts.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.18.649608v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Chimeric Reference Panels for Genomic Imputation</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Retriever is a framework that bypasses the need for external reference panels and achieves genotype imputation with accuracy consistently exceeding 95%.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Despite transformative advances in genomic technologies, missing data remains a fundamental constraint that limits the full potential of genomic research across biological systems. Genotype imputation offers a remedy by inferring unobserved genotypes from observed data. However, conventional methods typically rely on external reference panels constructed from complete genome sequences of hundreds of individuals, a costly approach largely inaccessible for non-model organisms. Moreover, these methods generally overlook novel genomic positions not captured in existing panels. To overcome these limitations, we developed Retriever, a framework that bypasses the need for external reference panels. Retriever constructs a chimeric reference panel directly from the target samples using a sliding-window approach to identify and retrieve genomic partitions with complete data. By exploiting the complementary distribution of missing data across samples, Retriever assembles a panel that preserves local patterns of linkage disequilibrium and captures novel variants. Integrated with Beagle, Retriever achieves genotype imputation with accuracy consistently exceeding 95% across diverse datasets, including plants, animals, and fungi. By eliminating the need for costly external panels, Retriever offers an accessible, cost-effective solution that broadens the application of sophisticated genomic analyses across species.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.22.648973v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Single-Cell Epigenomics Uncovers Heterochromatin Instability and Transcription Factor Dysfunction during Mouse Brain Aging</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify vulnerable brain regions and cell types and pinpoint key molecular pathways altered in brain aging.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The mechanisms regulating transcriptional changes in brain aging remain poorly understood. Here, we use single-cell epigenomics to profile chromatin accessibility and gene expression across eight brain regions in the mouse brain at 2, 9, and 18 months of age. In addition to a significant decline in progenitor cell populations involved in neurogenesis and myelination, we observed widespread and concordant changes of transcription and chromatin accessibility during aging in glial and neuronal cell types. These alterations are accompanied by dysregulation of master transcription factors and a shift toward stress-responsive programs driven by AP-1, indicating a progressive loss of cell identity with aging. We also identify region- and cell-type-specific heterochromatin decay, characterized by increased accessibility at H3K9me3-marked domains, activation of transposable elements, and upregulation of long non- coding RNAs, particularly in glutamatergic neurons. Together, these results reveal age-related disruption of heterochromatin maintenance and transcriptional programs, identify vulnerable brain regions and cell types, and pinpoint key molecular pathways altered in brain aging.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.21.649585v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Azurify integrates cancer genomics with machine learning to classify the clinical significance of somatic variants</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Azurify - a computational tool that integrates machine learning, public resources recommended by professional societies, and clinically annotated data to classify the pathogenicity of variations in precision cancer medicine.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurate classification of somatic variations from high-throughput sequencing data has become integral to diagnostics and prognostics across various cancers. However, the classification of these variations remains highly manual, inherently variable, and largely inaccessible outside specialized laboratories. Here, we introduce Azurify - a computational tool that integrates machine learning, public resources recommended by professional societies, and clinically annotated data to classify the pathogenicity of variations in precision cancer medicine. Trained on over 15,000 clinically classified variants from 8,202 patients across 138 cancer phenotypes, Azurify achieves 99.1% classification accuracy for concordant pathogenic variants in data from two external clinical laboratories. Additionally, Azurify reliably performs precise molecular profiling in leukemia cases. Azurify's unified, scalable, and modular framework can be easily deployed within bioinformatics pipelines and retrained as new data emerges. In addition to supporting clinical workflows, Azurify offers a high-throughput screening solution for research, enabling genomic studies to identify meaningful variant-disease associations with greater efficiency and consistency.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.18.649588v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Novel AI-based approach for cell type deconvolution in Visium datasets when reference scRNA-seqs include multiple minor cell types</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We have applied tensor decomposition (TD)-based unsupervised feature extraction (FE) to integrate multiple Visium datasets, as a platform for spatial gene expression profiling (spatial transcriptomics).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We have applied tensor decomposition (TD)-based unsupervised feature extraction (FE) to integrate multiple Visium datasets, as a platform for spatial gene expression profiling (spatial transcriptomics). As a result, TD-based unsupervised FE successfully obtains singular value vectors consistent with spatial distribution, i.e., singular value vectors with similar values are assigned to neighboring spots. Further, TD-based unsupervised FE successfully infers the cell type fractions within individual Visium spots (i.e., successful deconvolution) by referencing single-cell RNA-seq experiments that include multiple minor cell types, for which other conventional methods - RCTD, SPOTlight, SpaCET, and cell2location - fail. Therefore, TD-based unsupervised FE can be used to perform deconvolution even when other conventional methods fail as it includes multiple minor cell types in the reference profiles. TD-based unsupervised FE is thus expected to be applied to a wide range of deconvolution applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.18.649484v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">GOALS: Gene Ontology Analysis with Layered Shells for Enhanced Functional Insight and Visualization</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Gene Ontology Analysis using Layered Shells (GOALS), a novel tool that discretizes GOs into optimal layers and unsupervised clustering of GO's latent projections.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Gene Ontologies (GOs) are standardized descriptions of gene functions in terms of biological processes, molecular functions, and cellular components, capturing their Parent-Child relationships in a structured framework and advancing cancer biological modeling to provide consistent and meaningful insights into functional genomics analysis. The conventional GO hierarchical structure is defined by human curation experts, with levels determined by the shortest path to the root term. However, grouping GOs poses challenges due to the uneven distribution of gene members within GO terms and inconsistencies in the level of detail across terms at the same GO level. In this work, we introduce Gene Ontology Analysis using Layered Shells (GOALS), a novel tool that discretizes GOAs into optimal layers. GOALS creates scalable GO layers while maintaining a balanced number of genes across GOs in each layer. Unlike existing tools, the GOALS framework organizes GO terms using a bottom-up approach based on their co-membership network, discretizing GOs to achieve an exponential fit with GO's gene member size. Meanwhile, GOALS reveals clusters or supersets reflecting biological relevance by unsupervised clustering of GO's latent projections. In a case study on mouse natural killer (NK) cell development, GOALS identified distinct GO functional clusters with multi-GO layers to reveal multiple levels of detail from specific to abstract contexts to maximize signal discovery and uncover those signals' associations with trajectory divergence. More importantly, GOALS enhances enrichment analysis by introducing additional GO stratification and latent GO map that enables more accurate classification of functional differences. GOALS offers a robust and innovative framework for exploring disordered GO clusters, mining GO activities, and analyzing potential GO-GO interplays. By addressing critical challenges in functional genomics, GOALS provides a powerful tool for advancing our understanding of cell heterogeneity and potentially uncovering actionable insights for therapeutic development.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.22.650095v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ShinyCell2: An extended library for simple and sharable visualisation of spatial, peak-based and multi-omic single-cell data</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present ShinyCell2, an enhanced R package for interactive visualisation of single-cell multi-omics and spatial transcriptomics data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell technologies now span multiple modalities, generating large, complex datasets that challenge analysis and sharing. We present ShinyCell2, an enhanced R package for interactive visualisation of single-cell multi-omics and spatial transcriptomics data. ShinyCell2 retains the simplicity and lightweight deployment of its predecessor while introducing advanced visualisations, cross-modality comparisons, and statistical tools tailored to spatial and multi-omic data. It enables intuitive, rapid exploration of high-dimensional data without requiring extensive computational expertise.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.22.650045v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">GAT-HiC: Efficient Reconstruction of 3D Chromosome Structure via Residual Graph Attention Neural Networks</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We come up with graph attention and residual network-based GAT-HiC to predict three-dimensional chromosome structure from Hi-C interactions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Hi-C is an experimental technique to measure the genome-wide topological dynamics and three-dimensional (3D) shape of chromosomes indirectly via counting the number of interactions between distinct sets of loci. One can estimate the 3D shape of a chromosome over these indirect interaction datasets. Here, we come up with graph attention and residual network-based GAT-HiC to predict three-dimensional chromosome structure from Hi-C interactions. GAT-HiC is distinct from the existing 3D chromosome shape prediction approaches in a way that it can generalize to data that is different than train data. So, we can train GAT-HiC on one type of Hi-C interaction matrix and infer on a completely dissimilar interaction matrix. GAT-HiC combines the unsupervised vertex embedding method Node2vec with an attention-based graph neural network when predicting each genomic locis three-dimensional coordinates from Hi-C interaction matrix. We test the performance of our method across multiple Hi-C interaction datasets, where a trained model can be generalized across distinct cell populations, distinct restriction enzymes, and distinct Hi-C resolutions over human and mouse. GAT-HiC can reconstruct accurately in all these scenarios. Our method outperforms the existing approaches in terms of the accuracy of three-dimensional chromosome shape inference over interaction datasets.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.18.649477v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

