<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Friday, 09 May 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Bonsai: Tree representations for distortion-free visualization and exploratory analysis of single-cell omics data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> de Groot, D. H., Leonardo, S. X. M., Pachkov, M., van Nimwegen, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present Bonsai: a novel method that reconstructs the most likely tree relating any set of high-dimensional objects while rigorously accounting for heterogeneous measurement noise.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell omics methods promise to revolutionize our understanding of gene regulatory processes during cell differentiation, but analysis of such data continues to pose a major challenge. Apart from technical challenges such as the sparsity and heterogeneous noise properties of these data, the crucial problem is that we know little about the potentially very complex high-dimensional structures that the data represent. Consequently, there is an urgent need for exploratory analysis methods that allow rigorous representation and visualization of the structure in the data. However, currently popular methods such as UMAP and t-SNE are unsatisfactory because they are ad hoc, stochastic, uninterpretable, and known to severely distort the structure in the data. Here we show that these challenges can be overcome by representing the data on tree structures and present Bonsai: a novel method that reconstructs the most likely tree relating any set of high-dimensional objects while rigorously accounting for heterogeneous measurement noise. We show that, in contrast to other visualization methods, distances along the Bonsai trees accurately represent true distances between the objects in high-dimensional space across many types of datasets. Moreover, Bonsai automatically regularizes measurement noise, outperforming even methods specifically designed for that purpose on tasks such as nearest-neighbor identification. By analyzing a blood cell dataset, we show that Bonsai trees not only capture known lineage relationships but also provide novel biological insights. For example, Bonsai uncovers that different subsets of NK cells derive from the myeloid and lymphoid lineage, and pinpoints genes that distinguish myeloid-NK from lymphoid-NK cells. Bonsai is free from tunable parameters and scales to datasets of hundreds of thousands of cells. The accompanying tool, Bonsai-scout, provides visualizations of the Bonsai trees and allows for interactive data exploration such as identifying subclades and their markers, visualizing features along the tree, changing the tree layout, and zooming in on substructures. Finally, application to a dataset of football statistics shows the generality of Bonsai in successfully capturing complex structures in high-dimensional data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.08.652944v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identity-by-descent captures Shared Environmental Factors at Biobank Scale</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Marsico, F., Buonaiuto, S., Amos-Abanyie, E., Chinthala, L., Mohammed, A., Genetics Center, R., Rooney, R., Williams, R. W., Davis, R. L., Finkel, T. H., Brown, C. W., Prins, P., Colonna, V.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that genetic relatedness can serve as an indicator of health conditions by capturing the combined influences of genetic inheritance and shared environmental factors, and we show the potential of our framework and its possible application to public health interventions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The apple does not fall far from the tree is an old idiom that encapsulates a key concept: being related extends beyond merely sharing genetic material. It often implies sharing a common environment, including culture, language, dietary habits, and geographical location. In this study, we show that the analysis of genetic relatedness can serve as an indicator of health conditions by capturing the combined influences of genetic inheritance and shared environmental factors. We mapped the genomic data and electronic health records from 13k individuals in the Integrative Genomics Biorepository cohort, to neighborhood-level geographic data and integrated census-based environmental metrics. We used an identity-by-descent (IBD) based hierarchical community detection algorithm to identify four main communities closely aligning with continental ancestry, and seventeen subcommunities. We found uneven exposure to elevated environmental stressors across subcommunities, and we were able to identify subcommuinities at high risk of respiratory and dermatological conditions, demonstrating the potential of our framework and its possible application to public health interventions. Interestingly, for conditions such as congenital disorders, the most important differences were detected between subcommunities within the same community, indicating the relevance of considering a fine-grained population structure. These findings show how genetic relatedness, even at distant levels, can reflect shared environments and social determinants of health, providing a framework for understanding health disparities.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.03.652048v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CRISMER: A transformer-based Interpretable Deep Learning Approach for Genome-wide CRISPR Cas-9 Off-Target Prediction and Optimization</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Emtiaj, A. H., Rafi, R. H., Nayeem, M. A., Rahman, M. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a deep neural network architecture that improves the accuracy and safety of CRISPR-Cas9 gene editing by focusing on critical PAM-proximal regions and mismatch patterns.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> CRISPR-Cas9 gene editing holds transformative promise for genetic therapies, but is hindered by off-target effects that undermine its precision and safety. To address this, we developed CRISMER, a hybrid deep-learning architecture that uses multi-branch convolutional neural networks to extract k-mer features and transformer blocks to capture long-range dependencies. This hybrid approach enhances the prediction and optimization of single-guide RNA (sgRNA) designs. CRISMER was trained on Change-seq and Site-seq datasets, using a sparse one-hot encoding scheme, and evaluated on independent datasets including Circle-seq, Guide-seq, Surro-seq, and TTISS. CRISMER outperformed existing tools, achieving an F1 score of 0.7092 and a PR-AUC of 0.8006 on the CRISPR-DIPOff dataset. It also excelled in measuring sgRNA specificity and optimizing designs for genes, such as PCSK9 and BCL11A, yielding sgRNAs with reduced off-target activity. For example, a G-to-C mutation at position 12 in the sgRNA for PCSK9 and at position 11 for BCL11A led to significant improvements in off-target profiles. Interpretability analysis via integrated gradients confirmed the model's focus on critical PAM-proximal regions and mismatch patterns. These results demonstrate that CRISMER significantly improves the accuracy and safety of CRISPR-Cas9, advancing its reliability for therapeutic applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.03.652008v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Spatiotemporal profiling reveals the impact of caloric restriction on mammalian brain aging</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, Z., Epstein, A., Schaefer, C., Abdulraouf, A., Jiang, W., Zhou, W., Cao, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We study the molecular and cellular effects of caloric restriction (CR) on the aging brain and propose a novel framework for understanding the complex effects of anti-aging interventions at the cellular and molecular levels.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Aging induces functional declines in the mammalian brain, increasing its vulnerability to cognitive impairments and neurodegenerative disorders. Among various interventions to slow the aging process, caloric restriction (CR) has consistently demonstrated the ability to extend lifespan and enhance brain function across different species. Yet the precise molecular and cellular mechanisms by which CR benefits the aging brain remain elusive, especially at region-specific and cell type-specific resolution. In this study, we performed spatiotemporal profiling of mouse brains to elucidate the detailed mechanisms driving the anti-aging effects of CR. Utilizing highly scalable single-nucleus genomics and spatial transcriptomics platforms, EasySci and IRISeq, we profiled over 500,000 cells from 36 mouse brains across three age groups and conducted spatial transcriptomic analysis on twelve brain sections from aged mice under CR and control conditions. This comprehensive approach allowed us to explore the impact of CR on over 300 cellular states and assess region-specific molecular alterations. Our findings reveal that CR effectively modulates key aging-associated changes, notably by delaying the expansion of inflammatory cell populations and preserving cells critical to the neurovascular system and myelination pathways. Moreover, CR significantly reduced the expression of aging-associated genes involved in oxidative stress, unfolded protein stress, and DNA damage stress across various cell types and regions. A notable reduction in senescence-associated genes and restoration of circadian rhythm genes were observed, particularly in ventricles and white matter. Furthermore, CR exhibited region-specific restoration in genes linked to cognitive function and myelin maintenance, underscoring its targeted effects on brain aging. In summary, the integration of single-nucleus and spatial genomics provides a novel framework for understanding the complex effects of anti-aging interventions at the cellular and molecular levels, offering potential therapeutic targets for aging and neurodegenerative diseases.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.04.652093v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrative analysis of RNA binding proteins identifies DDX55 as a novel regulator of 3'UTR isoform diversity</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gazzara, M. R., Cater, T., Mallory, M. J., Barash, Y., Lynch, K. W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify and validate a new regulator of alternative polyadenylation at the 3' end of 3' untranslated regions of mRNAs, and propose a number of previously unrecognized regulators of3'UTR processing and differential stability.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The 3' untranslated regions (3'UTRs) of mRNAs play a critical role in controlling gene expression and function because they contain binding sites for microRNAs and RNA binding proteins (RBPs) that alter mRNA stability, localization, and translation. Most mRNA 3' ends contain multiple polyadenylation sites (PAS) that can be utilized in condition-specific manners, a process known as alternative polyadenylation (APA), however the mechanisms driving the regulation of APA remain poorly characterized. By integrating a large set of over 500 RNA binding protein (RBP) depletion and binding experiments across two cell lines generated by the ENCODE consortium, we uncovered a number of RBPs in each cell type whose depletion leads to widespread alteration of 3'UTR patterns. These include not only known regulators of APA, but also many putative novel regulators of 3'UTR isoform expression. We focused analysis on the largely unstudied DEAD box RNA helicase, DDX55, and validate its novel role in 3'UTR isoform regulation using molecular assays and targeted 3' end sequencing experiments. Our findings identify DDX55 as a new regulator of APA, particularly at PAS that contain features of RNA secondary structure. Our data also suggest additional previously unrecognized regulators of 3'UTR processing and differential stability.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.06.652471v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">In silico analysis and Predictive linkage of Deubiquitinating Enzymes underlying Early Development</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Munavar-K, F., Lenka, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of the role of understudied DUBs in early embryonic development and its implications for drug discovery and development .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Deubiquitinating enzymes (DUBs) exert their functions by catalyzing the ubiquitinated proteins and maintaining ubiquitination dynamics during post-translational modification. They carry out wide-gamut of functions in various cellular contexts by being associated with regulatory entities concerning transcription, translation, cell signalling, etc. However, limited studies are available concerning their specific attributes during organismal development. In this study, we have employed the integrated bioinformatics and experimental approaches to investigate the involvement of DUBs in embryonic stem cell (ESC) maintenance and differentiation. The StemMapper database revealed the distinctive expression profiles of various DUBs in ESCs and their differentiated derivatives. Further, experimental validation by qRT-PCR with a selected group of understudied DUBs including USP46, USP47, USP4, USP40, CYLD, and BRCC3 confirmed their expression patterns during cardiac and neural differentiation from ESCs, with USP46, USP47, and USP40 showing an increasing trend in contrast to USP4 during differentiation. While the TRANS-DSI database revealed novel DUB-Substrate interactions (DSI), the Gene Ontology and pathway enrichment analysis helped link these DUBs to critical cellular processes including transcriptional regulation, cell cycle control, and DNA repair. Further, the UbiBrowser 2.0 facilitated identifying several understudied DUBs as potential modulators of evolutionarily conserved developmental signalling cascades, notably Wnt, Notch, Hippo, and Hedgehog pathways. Our in silico analysis and predictions do provide crucial insights into the complex regulatory roles of DUBs in early development and establish a foundation for further investigations to unveil the molecular mechanisms and identify potential therapeutics thereof.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.03.652026v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">NanoLoop: A deep learning framework leveraging Nanopore sequencing for chromatin loop prediction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Huang, W., Tang, L., Hill, M. C., Zhang, Y., Xie, J., Li, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce NanoLoop, the first algorithmic framework attempting to predict genome-wide chromatin interactions using Nanopore data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Chromatin loops play a crucial role in gene regulation and cellular function, providing key insights into understanding the three-dimensional structure of the genome and its impact on cellular homeostasis. Nanopore sequencing technology, with its advantages in simultaneously detecting sequences and methylation patterns, brings new opportunities for studying three-dimensional genome structures. We introduce NanoLoop, the first algorithmic framework attempting to predict genome-wide chromatin interactions using Nanopore data. In experiments across four human lymphoblastoid cell lines, NanoLoop demonstrated excellent predictive performance and cross-cell line generalization capabilities. We also discovered four distinct methylation patterns at loop anchors that influence histone modification levels and determine various loop types. NanoLoop further predicted previously uncharacterized long-range chromatin loops, highlighting DNA methylation's role in three-dimensional genome regulation and providing new insights into the complex regulatory relationships between epigenetic modifications and three-dimensional genome organization.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.03.651998v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Deep learning inference of miRNA expression from bulk and single-cell mRNA expression</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ripan, R. C., Athaya, T., Li, x., Hu, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce two deep learning models for predicting miRNA expression from single-cell mRNA data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding the activity of miRNA in individual cells presents a challenge due to the limitations of single-cell technologies in capturing miRNAs. To tackle this obstacle, we introduce two deep learning models: Cross-Modality (CM) and Single-Modality (SM). These models utilize encoder-decoder architectures to predict miRNA expression at the bulk and single-cell levels from mRNA data. We compared CM and SM with a state-of-the-art approach, miRSCAPE, using both bulk and single-cell datasets. We found that both CM and SM outperformed miRSCAPE in terms of accuracy. We also observed that integrating miRNA target information led to a significant enhancement in performance compared to using all genes. These models offer valuable tools for predicting miRNA expression from single-cell mRNA data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.03.652014v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">High-Throughput Multiomics Profiling of Model Systems Using the AVITI24 Platform</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lopez, T., Honigfort, D., Mah, A., Thompson, C., Dien, V., Dawood, M., Frankel, E., Fuller, D., Jamison, C., Kelley, R., Kwon, J., Liu, Y., Ren, P., Saha, S., Wang, H., Wong, J., Zhao, D., Abtahi, M., Altomare, A., Bajari, R., Bellizzi, A., Bracamonte, S., Bradfield, A., Brown-Greaves, C., Bui, C., Charov, K., Constandse, R., Cullion, K., Damron, W., Dangelo, M., Davis, L., Dawood, N., Dhonde, R., Elango, K., Espinosa, S., Garcia, F., Gavrila, V., Gomez, L., Hastings, D., Hoang, D., Juan, W., Kia, A., Kim, M., Krivit, M., Lama, R., Mandla, K., Martinez, A., Mass, M., Miller, A., Neysmith, J.,</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a multiomics platform comprising Teton, a detection assay system, and AVITI24, a dual-flowcell instrument that performs both cellular imaging and sequencing readout.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We present a multiomics platform comprising Teton, a detection assay system, and AVITI24, a dual-flowcell instrument that performs both cellular imaging and sequencing readout. Teton integrates a compartmentalized flowcell for cell culture with methods to measure morphology, RNA, and protein at subcellular resolution. The platform quantifies morphological features through cell painting of 6 cellular components, RNA expression of up to 350 transcripts via sequencing of oligonucleotides hybridized to mRNA, and protein expression of up to 200 targets using antibody-linked oligonucleotide sequencing. The flow cell accommodates >1 million cells in a 10 cm sqaured open-well format or can be subdivided into 12 or 48 wells to support experiments with multiple conditions or time points. We describe and validate the detection methods of the platform and showcase its capabilities by co-culturing three cancer cell lines and elucidating the cellular pathways triggered by various drug treatments as a function of time. Using multiple time points enables us to capture the dynamics of cellular processes including receptor activation and signaling cascades. The results demonstrate how different cancer cells evade TNFalpha-induced apoptosis by activating compensatory signaling programs that maintain survival despite pro-apoptotic cues. Our model system replicates previously published results and highlights the versatility of the platform in enabling rapid, high-throughput analysis of complex cellular responses in varied biological contexts.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.03.651997v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A computational framework for mapping isoform landscape and regulatory mechanisms from spatial transcriptomics data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Su, J., Qu, Y., Schertzer, M. D., Yang, H., Jiang, J., Lhakhang, T., Nelson, T. M., Park, S., Lai, Q., Fu, X., Choi, S.-W., Knowles, D. A., Rabadan, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> SpatiaL ISOform Statistical Modeling, a computational framework for detecting isoform-resolution patterns from spatial transcriptomics data, and a comprehensive spatial splicing analysis in the brain under normal and neoplastic conditions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Transcript diversity including splicing and alternative 3'end usage is crucial for cellular identity and adaptation, yet its spatial coordination remains poorly understood. Here, we present SPLISOSM (SpatiaL ISOform Statistical Modeling), a computational framework for detecting isoform-resolution patterns from spatial transcriptomics data. SPLISOSM leverages multivariate testing to account for spot- and isoform-level dependencies, demonstrating robust and theoretically grounded performance on sparse data. In the mouse brain, we identify over 1,000 spatially variable transcript diversity events, primarily in synaptic signaling pathways linked to neuropsychiatric disorders, and uncover both known and novel regulatory relationships with region-specific RNA binding proteins. We further show that these patterns are evolutionarily conserved between mouse and human prefrontal cortex. Analysis of human glioblastoma highlights pervasive transcript diversity in antigen presentation and adhesion genes associated with specific microenvironmental conditions. Together, we present a comprehensive spatial splicing analysis in the brain under normal and neoplastic conditions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.02.651907v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">BioMedTools: a language model-powered community for biomedical computational tools</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liu, S., Xing, H., Han, M., Zhang, D., Gong, L., Liu, D., Chen, J., Cai, P., Hu, Q.-N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce BioMedTools, a community of biomedical computational tools that mainly implements LM-based tool identification and a chat assistant.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> A large number of biomedical computational tools have spawned several tool registries. However, in the face of the rapid growth in the number of tools, existing tool registries, which are manually curated or community-driven, are difficult to keep up to date, resulting in inadequate tool repository data. In this paper, we show that language models (LMs) can aid in building a community of tools. We introduce BioMedTools (https://biomed.tools), a community of biomedical computational tools that mainly implements LM-based tool identification and a chat assistant. Compared with existing tool registries, BioMedTools achieves excellence in terms of the number of tools, frequency of data updates, and functionality. Meanwhile, the Model Context Protocol (MCP) servers hub in BioMedTools may promote the building of agents in the biomedical field. BioMedTools enables the efficient collection of tools and enhances their findability and accessibility.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.02.651919v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Next generation statistical framework for next generation spatial transcriptomics data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mangane, F., Bost, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present TranspaceR, a new R package that introduces computational and statistical methods tailored to the analysis of next-generation spatial transcriptomic datasets, paving the way for deeper insights into tissue organization.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The rapid advancement of spatial transcriptomic technologies, particularly in situ hybridization based methods, has enabled the profiling of gene expression at sub cellular resolution across large tissue sections. Commercial platforms such as Xenium and CosMx now routinely generate high-quality datasets of increasing size and complexity. However, existing analytical approaches, often adapted from single-cell genomics, fall short in addressing the specific challenges posed by spatial data, especially at scale. In this work, we present TranspaceR, a new R package that introduces computational and statistical methods tailored to the analysis of next-generation spatial transcriptomic datasets. Our framework includes novel quality control procedures, scalable gene selection strategies especially for spatially variable genes, and optimized normalization and dimensionality reduction techniques based on in-depth statistical characterization of spatial data. We also demonstrate how single-cell annotation tools can be leveraged for automated cell-type labeling within spatial contexts. Together, these tools enable the efficient and robust analysis of imaging-based spatial transcriptomics datasets comprising millions of cells, paving the way for deeper insights into tissue organization.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.02.651852v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Computational Screening of Filamin Mechanical Binding Proteins using AlphaFold2</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Johnson, J., Gonzalez-Morales, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We used AlphaFold2 modeling to identify mechanical binding proteins that specifically recognize the open conformation of filamins.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Filamins are dimeric actin binding protein that play a critical role in mechanical signaling. They contain a mechanosensory region (MSR) that naturally folds into a globular closed conformation. Under mechanical stress, the MSR unfolds into an open conformation, exposing binding sites for numerous proteins. Filamins are involved in diverse cellular functions, and their mechanical binding targets are highly context dependent. In this study, we employed AlphaFold2 modeling for screening proteins that specifically recognize the open conformation of filamins. We focused on the Drosophila melanogaster filamin, Cheerio, and conducted a biased screen to identify mechanical binding proteins. We selected the top 132 hits from the initial screening for further characterization. All identified binding proteins specifically recognize the open conformation of the MSR and not the closed conformation. Interestingly, the binding regions of these proteins lack obvious sequence similarity. While some false positives were identified, they could be effectively filtered out based on the secondary structure formed at the binding interface. This study provides a framework for identifying specifically filamin interactions in mechanosignaling.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.02.651884v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DIST: Distance-based Inference of Species Trees</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> de Jong, M. J., Janke, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Inferring species trees from concatenated loci using distance matrices containing multi-locus estimates of Dxy, Fst or coalescence units .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Inferring species trees from concatenated loci is often criticised for failing to account for gene tree discordance, particularly when using character-based methods. However, this criticism does not apply to distance-based concatenation trees, which can be shown to be statistically consistent even in anomaly zones. Building on this insight, we introduce DIST (Distance-based Inference of Species Trees), an intuitive and scalable method that infers species trees from population-level distance matrices containing multi-locus estimates of Dxy, Fst or coalescence units. DIST derives these values from between-individual sequence dissimilarity estimates, E(p), using basic equations from coalescence theory. Under certain conditions, DIST can also quantify gene tree discordance and distinguish whether it arises from gene flow or incomplete lineage sorting alone. While conceptually related to more sophisticated summary methods, DIST differs in that it does not seek the species tree which best explains a set of gene trees. Instead, it searches for the species tree which best explains an average gene tree, of which all branch lengths reflect mean coalescence time, E(t). Although this average gene tree is rarely observed empirically, it is approximated by an individual-level distance-based tree, traditionally referred to as a "tree of individuals". The DIST algorithm is implemented in the R package SambaR, which now accepts input in the form of pairwise E(p) estimates.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.02.651899v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Atrial remodelling and atrial fibrillation self-sustaining: the role of circulating circDGCR8.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gao Y, Dong Y, Jiang N, Zhang H, Liu Z, Wang Q, Fu Y, Li J, Li Z, Pan H, Zheng X, Zhan L, Yang X, Xu L, Chen M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #fedbf2; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, pan-genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified circular RNAs associated with atrial remodelling and to evaluate their ability to predict AF progression and post-ablation recurrence.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The prediction of atrial fibrillation (AF) progression and post-ablation recurrence is currently based on empirical estimates, leading to suboptimal predictive accuracy. This study investigates whether atrial remodelling, a key factor in the severity of atrial cardiomyopathy, could serve as a shared substrate influencing both AF progression and recurrence. We aimed to identify circular RNAs (circRNAs) associated with atrial remodelling and to evaluate their ability to predict AF progression and recurrence.We assessed the differential expression of plasma circRNAs between paroxysmal (PAF) and persistent AF (PsAF) patients using microarray analysis. Selected candidate circRNAs were validated through qPCR following rigorous statistical and bioinformatics analysis. circDGCR8 was consistently found to be up-regulated in PsAF compared with PAF patients. Additionally, circDGCR8 was significantly up-regulated in human atrial fibroblasts treated with angiotensin II (AngII). Gain- and loss-of-function studies suggested that circDGCR8 could promote atrial remodelling at cellular level by enhancing collagen production and fibroblast proliferation. Overexpression of circDGCR8 in human cardiac fibroblasts significantly altered the gene expression spectrum, impacting pathways including IL-17 signalling and TNF signalling. Moreover, circDGCR8 levels were positively correlated with atrial fibrosis, as indicated by increased percentages of low voltage zones. The predictive value of circDGCR8 was evaluated in two cohorts: (i) PAF patients monitored for 36 months with progression to PsAF as the endpoint, and (ii) AF patients who underwent radiofrequency ablation followed for 12 months to assess recurrence. In both cohorts, higher level of circDGCR8 was associated with increased risks of AF progression and post-ablation recurrence.Our results suggest that circDGCR8, associated with atrial remodelling, holds potential as a predictive biomarker for both AF progression and post-ablation recurrence.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40336343/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

