<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Saturday, 14 June 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">HiFIseek: gene-specific enrichment of high-impact mutations in associated genomic regions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Riudavets Puig, R., Brorson, I. S., O'Mahony, D., Kristensen, V. N., Mathelier, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We apply HiFIseek to detect genes exhibiting high FI CRVs in eight cancer cohorts and a set of breast cancer risk-associated single-nucleotide polymorphisms (SNPs). We observe a small degree of agreement across CRR-score combinations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Transcription is regulated through the sequence-specific binding of transcription factors (TFs) to cis-regulatory regions (CRRs). Although scattered along the genome, multiple CRRs are brought in close spatial proximity to the genes they regulate through the formation of DNA loops. The sequence component of transcriptional regulation suggests that DNA variants in CRRs could disrupt TF-DNA interactions and gene regulatory networks through a cascading effect. To date, only a few cases of recurrent cis-regulatory variants have been described. As an alternative to variant recurrence, some methods utilize genomic annotations that indicate the functional impact (FI) of a variant and the CRRs of each gene to detect the enrichment of high-impact cis-regulatory variants (CRVs). However, the agreement across the different associations and FI scoring methods remains unexplored. This work demonstrates that gene-CRR and FI scoring methods exhibit little consensus, highlighting the impact of the choice of a specific CRR-score combination in a given analysis. In addition, we demonstrate that cancer genes exhibit a higher frequency of FI variants compared to non-cancer genes. Based on this, we developed HiFIseek, a Snakemake pipeline exploring the enrichment of FI variants in associated genomic regions using all region-score combinations and two enrichment detection methods. We apply HiFIseek to detect genes exhibiting high FI CRVs in eight cancer cohorts and a set of breast cancer risk-associated single-nucleotide polymorphisms (SNPs). In the cancer cohorts, we observe a small degree of agreement across CRR-score combinations. Although HiFIseek detected known cancer-related genes with FI CRVs, most of them did not show significant changes in their expression. In the SNPs cohort, HiFIseek found small consensus across CRR-score combinations. However, one of the combinations returned 18 known cancer-related genes, including BRCA1, showing an enrichment of high-impact CRVs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.12.659305v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Spatially-explicit genomics of An. gambiae s.l uncovers fine-scale population structure and mechanisms of insecticide resistance</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nagi, S. C., Essandoh, J., Ato-Egyir, N., Lucas, E. R., Miles, A., Desewu, K., Williams, I., Egyir-Yawson, A., Sedda, L., Weetman, D., Donnelly, M. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We examine methods to estimate mosquito kinship, demonstrating that polymorphic chromosomal inversions significantly confound established analytical tools, while revealing how the mosquito's chromosome architecture generates high variance in genetic relatedness among kin.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Progress in malaria control in sub-Saharan Africa is stalling, partly due to the spread of insecticide resistance in Anopheles vectors. Monitoring the evolution of insecticide resistance alleles and their spatial heterogeneity is important for malaria control programmes, and genomic surveillance has emerged as a pivotal tool for this purpose. Earlier genomics research has typically employed convenience-based sampling, and research has yet to be performed to optimise sampling regimens for malaria vector genomics. In an earlier study, we developed a spatially explicit sampling framework that considers the underlying ecology to enable sampling mosquitoes with reduced bias. We applied this framework to sample and perform whole-genome sequencing on 485 individual specimens of An. gambiae s.l mosquitoes from Obuasi, central Ghana, an area with extensive artisanal-gold mining activities. In this region, An. gambiae s.l have been documented as highly resistant to multiple insecticides, including pyrethroids used for the treatment of bed-nets. High resolution of fine-scale population structure enabled the detection of isolation-by-distance in An. coluzzii in Obuasi, whilst finding that at this scale, geographic distance, rather than the underlying habitat, drives population structure. We examine methods to estimate mosquito kinship, demonstrating that polymorphic chromosomal inversions significantly confound established analytical tools, while revealing how the mosquito's chromosome architecture generates high variance in genetic relatedness among kin. Using genome-wide selection scans and clustering, we discover the novel mutations Gste2-F120L and Cyp9K1-N225I in detoxification enzymes that are driving selective sweeps, likely in response to insecticidal pressure. We find that the frequencies of some resistance variants are associated with artisanal-gold mines, and elucidate the continued evolution of the Voltage-gated sodium channel, the target of pyrethroid insecticides. Overall, we show that sampling vectors strategically may enhance our ability to perform effective genomic surveillance.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.13.655782v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Uncovering Functional Gene Regulatory Networks in Bulk and Single-Cell Data through Robust Transcription Factor Activity Estimation and Model-Guided Experimental Validation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Siahpirani, A. F., McCalla, S. G., Pyne, S., Dillingham, C. M., Sridharan, R., Roy, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a new approach, MERLIN+P+TFA, that uses prior knowledge-guided sparsity regularization to robustly and accurately estimate TFA and downstream inferred GRNs.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Reconstructing genome-scale gene regulatory networks (GRNs) remains a difficult problem in systems biology, and many experimental and computational methods have been developed to address this problem. Recent computational methods have aimed to more accurately model GRNs by estimating the hidden Transcription Factor Activity (TFA), from prior knowledge of TF target regulatory connections, encoded as an input directed graph, to relax the assumption that mRNA level of the regulator correlates with the protein activity of the regulator. However, the noise in the prior knowledge can adversely affect the estimated TFA levels and the quality of the downstream inferred GRNs. Here, we present a new approach, MERLIN+P+TFA, that uses prior knowledge-guided sparsity regularization to robustly and accurately estimate TFA and downstream GRNs. We apply our method to simulated and real expression data in yeast and mammalian systems and show improved quality of inferred GRNs for both bulk and single-cell datasets. Regularized TFA offers benefits to a variety of other GRN inference algorithms, including those that have traditionally be used with expression alone, in both bulk and scRNA-seq settings. We used the inferred GRN to prioritize key regulators for the mouse Embryonic Stem Cell (mESC) state and validate 58 regulators experimentally. We identify both known and novel regulators of the mESC state and further validate the targets of 4 known and novel regulators. Our validation experiments suggest that computationally inferred networks can capture functional targets of TFs with higher precision than estimated in current benchmarks, however, it is important to generate context-specific gold standards.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.09.658650v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AbDesign: Database of point mutants of antibodies with associated structures reveals poor generalization of binding predictions from machine learning models.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Janusz, B., Chomicz, D., Demharter, S., Arts, M., Kanter, J. d., Wilke, Y., Britze, H., Wrobel, S., Gawlowski, T., Dudzic, P., Ukkivi, K., Peil, L., Spreafico, R., Krawczyk, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a heterogeneous set of mutations to the CDR-H3 measured by ELISA, for which we introduce a diverse set of antigens with two antibodies each, and perform benchmarking of state-of-the-art affinity prediction algorithms to gauge their effectiveness.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Antibodies are naturally evolved molecular recognition scaffolds that can bind a variety of surfaces. Their designability is crucial to the development of biologics, with computational methods holding promise in accelerating the delivery of medicines to the clinic. Modeling antibody-antigen recognition is prohibitively difficult, with data paucity being one of the biggest hurdles. Current affinity datasets comprise a small number of experimental measurements, which are often not standardized between molecules. Here, we address these issues by creating a dataset of seven antigens with two antibodies each, for which we introduce a heterogeneous set of mutations to the CDR-H3 measured by ELISA. Each of the parental complexes has a known crystal structure. We perform benchmarking of state-of-the-art affinity prediction algorithms to gauge their effectiveness. Current computational methods exhibit significant limitations in accurately predicting the effects of single-point mutations. In contrast, the older empirical, physics-based method FoldX, performs well in identifying mutants that retain binding. These findings highlight the need for more resources like the one presented here - large, molecularly diverse, and experimentally consistent datasets.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.09.658639v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">cfDNAanalyzer: a comprehensive toolkit for analyzing cell-free DNA genomic sequencing data in liquid biopsy</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhou, J., Zhu, K., Huang, X., Yuan, J., Li, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present cfDNAanalyzer, a comprehensive, user-friendly toolkit that streamlines the feature extraction, selection, and construction of machine learning-based models for disease detection using cfDNA.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Liquid biopsy, through the analysis of plasma cell-free DNA (cfDNA), is transforming diagnostic medicine by enabling non-invasive insights into the genomic and epigenomic landscape of diseases, most notably cancer. As cfDNA reflects dynamic contributions from multiple tissues, it holds immense potential for early detection, disease monitoring, and personalized treatment. However, the rapid expansion of cfDNA applications has outpaced the availability of unified, versatile computational tools for the systematic analysis of cfDNA data. To address this unmet need, we present cfDNAanalyzer, a comprehensive, user-friendly toolkit that streamlines the feature extraction, selection, and construction of machine learning-based models for disease detection using cfDNA. Designed for flexibility and scalability, cfDNAanalyzer supports multimodal feature extraction, integration, and interpretable outputs, enabling both exploratory research and translational applications. By standardizing and accelerating cfDNA data analysis, cfDNAanalyzer provides a critical resource to advance precision diagnostics and deepen biological insight into disease processes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.09.658767v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">EndoGenius: Enabling comprehensive identification and quantitation of endogenous peptides</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fields, L., Dang, T. C., Gray, M. D., Protya, S. S., Li, l.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present EndoGenius, a software package for the search for endogenous peptides from mass spectrometry data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The investigation of endogenous peptides, specifically with respect to neuropeptides, from mass spectrometry data is rife with bioinformatics bottlenecks, stemming from the low in vivo abundance of these analytes, increased susceptibility to degradation, and an immense search space of possible peptides. To address this, we present EndoGenius in its expanded form, strategically designed to optimize the searching for these endogenous peptides complemented with a pipeline designed for tasks including quantitation, spectral library building, motif extraction, and usage with data-independent acquisition workflows. EndoGenius is released as an open-source software package under an MIT License. The EndoGenius package with a user interface can be installed from https://www.lilabs.org/resources. The source code for EndoGenius can be accessed at https://github.com/lingjunli-research/EndoGenius-v2.0.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.12.659347v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Concordance and dissonance: A genome-wide analysis of self-declared versus inferred ancestry in 10,250 participants from the HostSeq cohort</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Warren, R. L., Birol, I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A large-scale analysis comparing self-declared ancestry with genetically inferred ancestry in 10,250 participants from the pan-Canadian HostSeq cohort, revealing a strong relationship between discordance and ancestry heterogeneity.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurately characterizing human diversity is foundational to equitable genomics. In this study, we present a large-scale analysis comparing self-declared ancestry with genetically inferred ancestry in 10,250 participants from the pan-Canadian HostSeq cohort. Using the ntRoot algorithm on whole genome sequencing data, we inferred both global and local continental-level ancestry and assessed concordance with self-identified sociocultural categories. High agreement was observed among individuals self-identifying as White (concordance rate=98.8%), Black (97.2%), East Asian (96.1%), and South Asian (89.9%), while substantial discordance was found in those self-identifying as Hispanic (concordance rate=74.6%), Middle Eastern / Central Asian (67.9%) or Indigenous (40.7%). We quantified agreement using Cohens kappa ({kappa} = -0.01 unweighted; 0.35 weighted) and assessed admixture complexity with Shannon entropy, revealing a strong relationship between discordance and ancestry heterogeneity. Principal component analysis further revealed that tightly clustered genetic profiles often corresponded with lower admixture complexity, whereas broader, overlapping distributions were observed in groups with more heterogeneous ancestry and complex sociocultural histories. These findings underscore the complex interplay between sociocultural identity and genomic data, with discordance patterns reflecting the historical and cultural complexity of human populations. By quantifying this relationship systematically with ntRoot, our approach provides a framework for moving rigid categorical labels toward more nuanced genome-derived ancestry characterization that can improve both scientific rigor and representational equity in genomics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.10.658783v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">HAVEN: Hierarchical Attention for Viral protEin-based host iNference</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Antony, B., Haghani, M., Lauring, A. S., Karpatne, A., Murali, T. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We develop a machine learning model for predicting the host infected by a virus, given only the sequence of a protein encoded by the genome of that virus, and generalize to unseen hosts and viruses.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> It is crucial to accurately predict hosts of viruses to understand and anticipate human infectious diseases that originate from animals. There is a lack of versatile models that handle out-of-distribution factors such as unseen hosts and viruses. We develop a machine learning model for predicting the host infected by a virus, given only the sequence of a protein encoded by the genome of that virus. Our approach, HAVEN, is the first to apply to multiple hosts and to generalize to unseen hosts and viruses. HAVEN is a transformer-based architecture coupled with hierarchical self-attention that can accept sequences of highly diverse lengths. We integrate HAVEN with a prototype-based few-shot learning classifier to predict rare classes. We demonstrate the accuracy, robustness, and generalizability of HAVEN through a comprehensive series of experiments. In particular, we show that HAVEN can achieve a median AUPRC of 0.67 while predicting common hosts. Moreover, HAVEN retains this AUPRC value even for rare hosts (median prevalence as low as 0.09%). Our model performs on par with state-of-the-art foundation models, which are 65 to 5, 000 times larger in size, and outperforms them in identifying hosts of SARS-CoV-2 variants of concern.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.09.658367v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Phosphorylation-inducing chimera rewires oncogenic kinase to trigger apoptosis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Merz, M. M., Shoba, V. M., Pergu, R., Severance, Z. C., Munkanatta Godage, D. N. P., Deb, A., Kwok, H. S., Singh, P., Singh, S., Allen, J. B., Tian, W., Gosavi, P. M., Chaudhary, S. K., Anokhina, V., Weisberg, E. L., Payne, N. C., He, Y., Osadchey, R., Shekhar, M., Mazitschek, R., Rees, M. G., Roth, J. A., Cui, Q., Griffin, J. D., Liau, B. B., Choudhary, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that hyperactive enzymes with orthogonality of their resistance mechanisms to occupancy-driven drugs can be inhibited by a novel 'event-driven' mechanism that rewires the hyperactivity of an oncogenic kinase to phosphorylate its active site residue.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Hyperactive enzymes drive the pathology of several diseases, and classically, ''occupancy-driven'' drugs (e.g., active site or allosteric inhibitors) are used to target these enzymes. However, the stoichiometric nature of such inhibitors and the emergence of resistance highlight the need for new modalities. Here, we report a Phosphorylation-Inducing Chimeric Small molecule (PHICS) that rewires the hyperactivity of an oncogenic kinase, BCR-ABL, to phosphorylate its active site residue. Molecular dynamics simulations suggest this phosphorylation inhibits BCR-ABL by inducing electrostatic rearrangements of its active site. This ''event-driven'' mechanism selectively induces apoptosis of BCR-ABL-dependent cancer cells at substoichiometric concentrations (vs. stoichiometric concentrations of occupancy-driven drugs). Furthermore, PHICS is effective on other oncogenic ABL fusions or clinically observed resistance mutations, including to occupancy-driven drugs with the same binding site as PHICS, pointing to the orthogonality of their resistance mechanisms. These studies lay the foundation for electric-field and ''event-driven'' modalities to control hyperactive enzymes with orthogonal resistance mechanisms to occupancy-driven modalities.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.13.659082v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Humanized Caffeine-Inducible Systems for Controlling Cellular Functions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Scheller, L., Correia, B., Elia, M., Kang, C., Bonati, L., Gainza, P., Garodia, Y., Deshpande, P., Schmidt, J., Georgeon, S., Tang, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Caffeine-inducible heterodimers for inducible gene expression and a safer, more effective cell therapy .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Current cell therapies are limited by the lack of tools for controlling gene expression using humanized systems responsive to non-toxic stimuli. Starting from nanobodies that homodimerize in response to caffeine, we computationally designed inducible heterodimers and humanized the best-performing pairs. We used the resulting caffeine-inducible domains in engineered cytokine receptors for caffeine-inducible STAT3 signaling and in split transcription factors (caff-TFs) containing human-derived zinc-finger proteins. Heterodimerization of split transcription factors drastically enhanced their performance compared to homodimerization. We demonstrate that caff-TFs are compatible with lentiviral and retroviral delivery to Jurkat T-cells and enable inducible expression of therapeutic genes such as chimeric antigen receptors (CARs) in response to caffeine concentrations consistent with normal coffee consumption. By using the common, non-toxic molecule caffeine, and exclusively humanized protein components, these systems promise to be a safer alternative to existing systems and may be used in synthetic biology applications and for safer, more effective cell therapies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.13.659463v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Caffeine-regulated molecular switches for functional control of CAR T cells in vivo</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sylvander, E., Salzer, B., Emminger, D., Baik, H., D'Accardio, G., Schaefer, M., Mouratidis, K., Buri, M. C., Maresch, D., Urbanetz, A., Seigner, J., Marchiori, E., Munoz-Lopez, A., Engert, F., Engels, B., Mittelstaet, J., van der Veeken, J., Rosato, A., Zuber, J., Putz, E. M., Zajc, C. U., Lehner, M., Traxlmayr, M. W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Caffeine-responsive heterodimeric ON-switches that can be regulated with a non-toxic and readily available drug .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The limited controllability of CAR T cells in patients represents a key challenge of this highly potent immunotherapy. A molecular ON-switch, which can be regulated with a non-toxic and readily available small molecule drug, would represent a major advance towards controllable CAR T therapeutics. For that purpose, we engineered caffeine-responsive heterodimeric ON-switches (CaffSwitches) and demonstrate their high caffeine-dependency and virtually absent leakiness. When incorporating these CaffSwitches into CARs, the resulting CaffCARs were efficiently activated by caffeine concentrations achieved in human plasma after drinking one cup of coffee. Moreover, CaffCAR T cells also showed efficient tumor clearance in an in vivo mouse model, which was completely abolished in the absence of caffeine. This tight control was even observed with c-Jun overexpressing CaffCAR T cells, despite their massive expansion. Together, we anticipate that these novel CaffSwitches will be valuable tools for the development of safe and efficient next generation CAR T cells.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.13.659487v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Treating Myotonic Dystrophy with artificial RNA endonucleases to specifically degrade toxic RNA expansions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wei, T., Hu, H., Jiang, X., Li, Y., Han, W., Yang, Y., Xiao, X., Mao, M., Wang, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new class of programmable artificial enzymes (artificial RNA cleavers, or ARCs) that preferentially degrade the pathogenic repeats in the 3' UTR of DMPK gene .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Myotonic dystrophy type 1 (DM1), the most common autosomal dominant muscular disorder, is driven by expanded CUG repeats in the 3' UTR of DMPK gene, which sequester RNA-binding proteins and cause aberrant alternative splicing. Despite extensive study, no effective treatment exists, and current care remains limited to symptomatic management. Using a customized dual-color reporter carrying RNA repeats in different lengths, we engineered a new class of programmable artificial enzymes (artificial RNA cleavers, or ARCs) that preferentially degrade the pathogenic repeats both in cells and in a mouse model. We further examined the in vivo efficacy of ARCs by systematically delivering ARCs into the DM1 mouse model with muscle-specific adeno-associated viruses, and validated that ARCs can rescue the muscular pathology and movement ability. The treatment by ARCs showed sustained efficacy, low immunogenicity, and minimal off-target effects in animal model, which are key concerns in clinical translation of gene therapies. Collectively, these findings establish ARCs as a potent and precise RNA-targeting platform with strong clinical potential for the treatment of DM1 and other repeat expansion disorders.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.10.658795v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Pan-omics insights into abiotic stress responses: bridging functional genomics and precision crop breeding.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tayachew Admas, Shu Jiao, Rui Pan, Wenying Zhang</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #fedbf2; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, pan-genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A review of recent trends in pan-omics for studying abiotic stress responses and their applications in crop improvement.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Crop production has been regarded as the major goal of agricultural activities, but the rapidly growing population and climate change have become more complex in the agricultural systems. Abiotic stress greatly affects crop productivity globally; developing more resilient crop varieties has become imperative. However, we can understand how plants tolerate abiotic stress better by using new methods that combine different scientific approaches like pan-genomics, pan-transcriptomics, pan-proteomics, pan-metabolomics, and pan-phenomics. Investigations using a pan-omics approach are necessary to consider the variation resulting from complex interactions among genes, proteins, metabolites, and regulatory networks within a species. A comparative study of core, dispensable, and unique components across different accessions assists in identifying novel genes, proteins, and metabolites responsible for stress tolerance. Moreover, databases and online repositories now enable the storage, analysis, and retrieval of data generated by high-throughput technologies. The combination provides guidelines for researchers to harness the potential of pan-omics in promoting sustainable agricultural practices. Therefore, the review focuses on recent trends in pan-omics for studying abiotic stress responses and their applications in crop improvement. It also highlights the application of artificial intelligence (AI) in data integration and monitoring crop environments.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40512264/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

