<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Thursday, 15 May 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Prognostic value of circadian rhythm-associated genes in breast cancer.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ling Wang, Xiang Gao, Ximeng Zuo, Tangshun Wang, Xiaoguang Shi</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify key circadian rhythm-related genes in breast cancer and develop a prognostic model to predict clinical outcomes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Breast cancer (BC) remains the most prevalent malignancy among women. Clinical evidence indicates that genetic variations related to circadian rhythms, as well as the timing of therapeutic interventions, influence the response to radiation therapy and the toxicity of pharmacological treatments in women with BC. This study aimed to identify key circadian rhythm-related genes (CRGs) using bioinformatics and machine learning, and construct a prognostic model to predict clinical outcomes.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40369575/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Tutorial: guidelines for the use of machine learning methods to mine genomes and proteomes for antibiotic discovery.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fangping Wan, Marcelo D T Torres, Changge Guan, Cesar de la Fuente-Nunez</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a review of computational approaches to identify antimicrobial compounds, including machine learning and artificial intelligence (AI) tools, which have been used to date, with a special emphasis on peptides, and propose potential avenues for future exploration in this rapidly evolving field.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genomes and proteomes constitute a rich reservoir of molecular diversity. However, they have remained underexplored because of a lack of appropriate tools. In recent years, computational approaches have been developed to mine this unexplored biological information, or dark matter, accelerating the discovery of new antibiotic molecules. Such efforts have yielded a wide range of new molecules. These include peptides released via predicted proteolytic cleavage of larger proteins, termed 'encrypted peptides', which have been found to be widespread in nature. Molecules encoded by and translated from small open reading frames within genomic sequences have also been uncovered, further expanding the landscape of bioactive compounds. Here, we discuss computational approaches, including machine learning and artificial intelligence (AI) tools, which have been used to date to identify antimicrobial compounds, with a special emphasis on peptides. We also propose potential avenues for future exploration in this rapidly evolving field. Moreover, we provide an overview of the experimental methods commonly used to validate these computational predictions. We anticipate that efforts combining cutting-edge AI and experimental approaches for biological sequence mining will reveal new insights into host immunity and continue to accelerate discoveries in the fields of antibiotics and infectious diseases.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40369233/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Assembling unmapped reads reveals hidden variation in South Asian genomes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Das, A., Biddanda, A., McCoy, R. C., Schatz, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We discovered hidden genetic variation within 1 kbp of a known GWAS hit in millions of large contigs in the genomes of South Asian populations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Conventional genome mapping-based approaches systematically miss genetic variation, particularly in regions that substantially differ from the reference. To explore this hidden variation, we examined unmapped and poorly mapped reads from the genomes of 640 human individuals from South Asian (SAS) populations in the 1000 Genomes Project and the Simons Genome Diversity Project. We assembled tens of megabases of non-redundant sequence in tens of thousands of large contigs, much of which is present in both SAS and non-SAS populations. We demonstrated that much of this sequence is not discovered by traditional variant discovery approaches even when using complete genomes and pangenomes. Across 20,000 placed contigs, we found 8,215 intersections with 106 protein coding genes and >15,000 placements within 1 kbp of a known GWAS hit. We used long read data from a subset of samples to validate the majority of their assembled sequences, aligned RNA-seq data to identify hundreds of unplaced contigs with transcriptional potential, and queried existing nucleotide databases to evaluate the origins of the remaining unplaced sequences. Our results highlight the limitations of even the most complete reference genomes and provide a model for understanding the distribution of hidden variation in any human population.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.14.653340v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Hemolytik2: An Updated Database of Hemolytic Peptides and Proteins</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Singh, A., S A, K. R. S. A., Rathore, A. S., Raghava, G. P. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Hemolytik 2.0 is a comprehensive, manually curated peptide database that provides experimentally validated information on both hemolytic and non-hemolytic peptides.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Hemolytik 2.0 (http://webs.iiitd.edu.in/raghava/hemolytik2/) is a comprehensive, manually curated database that provides experimentally validated information on both hemolytic and non-hemolytic peptides. Data were meticulously extracted from peer-reviewed publications and established peptide repositories, including the Antimicrobial Peptide Database (APD), UniProt, the Collection of Anti-Microbial Peptides (CAMP-R4), and the Dragon Antimicrobial Peptide Database (DAMPD). This updated version of the original Hemolytik resource encompasses 13,215 entries, representing approximately 8,700 unique peptides. Each entry in Hemolytik 2.0 offers detailed annotations, including peptide name, amino acid sequence, biological source and origin, functional characterization, terminal modifications, stereochemistry, structural classification (linear or cyclic), and experimentally determined hemolytic activity. In addition, the database provides molecular representations of peptides in SMILES (Simplified Molecular Input Line Entry System) format, alongside predicted tertiary structures and annotated secondary structural states. Additionally, a RESTful API has been integrated into the Hemolytik 2.0 repository to enable programmatic access and automated retrieval of peptide data. Hemolytik 2.0 serves as a valuable resource for the scientific community, particularly for researchers involved in the design and development of therapeutic peptides, by facilitating the identification and optimization of peptide candidates with minimal hemolytic potential and enhanced safety profiles.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.12.653624v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">VaMiAnalyzer: An open source, python-based application for analysis of 3D in vitro vasculogenic mimicry assays</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Moore, S. P. G., Zhang, X., Jonathan, O. C., Zou, A., Lang, D., Zhang, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed an open source, python-based application, VaMiAnalyzer, allowing straightforward quantification of several VM structural features from microscopy images, allowing speedy, non-biased analysis of VM from microscope images.</p><p style="font-size: 16px; text-align: justify;">Background: Vasculogenic mimicry (VM) is the phenomenon whereby non-vascular tumor cells develop vascular-like structures. VM is linked to more aggressive tumor phenotypes including higher rates of metastasis and invasion and is potentially resistant to anti-angiogenic cancer therapies. VM is investigated in vitro using 3D VM assays with microscopy images capturing the resulting VM structures. The standard method to quantify endpoint data is to count various structural features manually, which is time-consuming and open to bias. At present, no software solutions have been developed to specifically address the analysis and quantification of VM structures. Results: To address this limitation, we developed an open source, python-based application, VaMiAnalyzer, allowing straightforward quantification of several VM structural features. The application follows a two-step approach that optionally corrects and enhances the raw input images and then analyzes and quantifies the VM features. Conclusions: VaMiAnalyzer is stand-alone software that allows automated measurement of VM structural features from phase-contrast microscopy images. It produces results that are strongly consistent with manual counts but in a significantly shorter time, allowing speedy, non-biased analysis of VM from microscopy images.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.13.653881v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Atlas-scale metabolic activities inferred from single-cell and spatial transcriptomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Armingol, E., Ashcroft, J., Mareckova, M., Prete, M., Lorenzi, V., Icoresi Mazzeo, C., Lee, J. T. H., Moullet, M., Bayraktar, O. A., Becker, C., Zondervan, K., Garcia-Alonso, L., Lewis, N. E., Vento-Tormo, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present scCellFie, a computational framework to infer metabolic activities from human and mouse transcriptomic data at single-cell and spatial resolution, which reveals endometriosis, endometrial carcinoma, and other disease-associated metabolic alterations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Metabolism supplies energy, building blocks, and signaling molecules vital for cell function and communication, but methods to directly measure it at single-cell and/or spatial resolutions remain technically challenging and inaccessible for most researchers. Single-cell and spatial transcriptomics offer high-throughput data alternatives with a rich ecosystem of computational tools. Here, we present scCellFie, a computational framework to infer metabolic activities from human and mouse transcriptomic data at single-cell and spatial resolution. Applied to ~30 million cell profiles, we generated a comprehensive metabolic atlas across human organs, identifying organ- and cell-type-specific activities. In the endometrium, scCellFie reveals metabolic programs contributing to healthy tissue remodeling during the menstrual cycle, with temporal patterns replicated in data from in vitro cultures. We also uncover disease-associated metabolic alterations in endometriosis and endometrial carcinoma, linked to proinflammatory macrophages, and metabolite-mediated epithelial cell communication, respectively. Ultimately, scCellFie provides a scalable toolbox for extracting interpretable metabolic functionalities from transcriptomic data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.09.653038v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SANTON: Sequencing Analysis Toolkits for Off-target Nomination</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Qiu, H., Wang, R., Wang, S., Mulbagal, A. M., Anderson, E., Ge, H. G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a Sequencing Analysis Toolkits for Off-target Nomination (SANTON) for the identification and quantification of potential off-target sites using datasets generated from synthetic oligonucleotide-based sequencing and whole genome-basedseq methods.</p><p style="font-size: 16px; text-align: justify;">Background Genome-wide off-target nomination and screening sequencing methods, such as synthetic oligonucleotide-based sequencing and whole genome-based sequencing, evaluate the precision and safety of gene-editing technologies. However, there remains a lack of comprehensive bioinformatics tools for analyzing the sequencing data generated by off-target nomination assays for various gene editors, including Cas9, Cas12a, and base editors. Results We introduce a Sequencing Analysis Toolkits for Off-target Nomination (SANTON) for the identification and quantification of potential off-target sites using datasets generated from synthetic oligonucleotide-based sequencing and whole genome-based sequencing (e.g., Digenome-seq) methods. By applying SANTON to a Cas9 treated oligo synthesization-based sequencing data, a comprehensive set of potential off-target sites are evaluated in which the top off-target sites displayed highly constituency with a published study using in vivo method (e.g, GUIDE-seq). Utilizing SANTON to previously published Digenome-seq datasets, we identified more potential off-target regions than previous studies, including ones showing significantly higher cleavage level than the on-target site. To the best of our knowledge, SANTON is the first public package capable of analyzing synthetic oligonucleotide-based sequencing data and the first to specific optimized for the analysis of whole genome-based sequencing analysis for Cas12a and base editors. We demonstrated the capability of SANTON to effectively analyze complex cleavage patterns by a diverse editing system. Conclusions SANTON provides a powerful computational solution to support genome editing research, facilitating more reliable and comprehensive off-target profiling.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.09.653082v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MicroFinder: Conserved gene-set mapping and assembly ordering for manual curation of bird microchromosomes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mathers, T. C., Paulini, M., Sotero-Caio, C. G., Wood, J. M. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify a set of conserved microchromosome genes and developed MicroFinder, a pipeline that uses this gene set to find small microchromesome fragments in draft genome assemblies to act as anchors for manual curation of microchromsomes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Obtaining chromosomally complete genome assemblies across the tree of life is a major goal of biodiversity genomics. However, some lineages remain recalcitrant to assembly despite recent advances in sequencing technologies and assembly tools. Birds present a substantial genome assembly challenge due to the presence of tiny, hard to assemble microchromosomes that are often highly fragmented or even missing in draft genome assemblies. As such, bird genomes require a large amount of expert manual curation effort via manipulation of genome-wide HI-C contact maps and many current chromosome-level bird genome assemblies do not resolve the known karyotype. Microchromosomes have distinct genetic and epigenetic features. They are GC-biased, gene-rich, highly methylated, and have distinct spatial organisation in the centre of the nucleus. Importantly, they are conserved across avian evolution. Here, using a reference set of expert curated bird genomes, we have identified a set of conserved microchromosome genes and developed MicroFinder, a pipeline that uses this gene set to find small microchromosome fragments in draft genome assemblies to act as anchors for manual curation of microchromosomes. We demonstrate how MicroFinder can be used to improve the speed and accuracy of bird genome curation. Furthermore, we highlight the usefulness of MicroFinder by carrying out MicroFinder-enabled re-curation of 12 previously released chromosome-scale bird genome assemblies, increasing the sequence content of microchromosome models.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.09.653066v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Universal Transformer-Based Tracker for Accurate Tracking of Particles and Cells in Microscopy</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, Y., Liu, D., Yang, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a Universal Transformer-Based Tracker that achieves the accurate tracking of subcellular structures and cells under microscopy.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurate tracking of subcellular structures and cells under microscopy supports heavily the studies of their dynamic processes. However, the complex motion and the similar appearances of objects pose significant challenges in accurately identifying the identical object across multiple detection results without introducing ambiguities in trajectory tracking. Here we propose a Universal Transformer-Based Tracker (UTT) that achieves the accurate tracking in various biologically relevant scenarios. The self-attention of Transformer extracts features and patterns from trajectories, and the cross-attention of Transformer captures matching relations between history tracks and future hypothesis tracklets (fragments of track). Our tracker uses the Transformer to handle the tracking of particles with diverse motion dynamics, and outperforms existing tracking algorithms in terms of accuracy. Furthermore,the tracker utilizes estimated particle positions to counteract missed detections, thereby improving tracking robustness under varying ratios of missing detections. We demonstrate the flexibility and versatility of our approach by adaptively integrating motion and appearance cues in cell tracking applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.09.653028v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MALER: a web server to build, evaluate, and apply machine learning models for numerical biomedical data analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhai, Z., Lin, Z., Li, Q., Pan, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We have developed Maler, a machine learning model framework for high-dimensional biomedical data analysis .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The explosive growth of numerical biomedical data poses a challenge in uncovering meaningful insights within from vast omics and clinical data. In recent years, machine learning has emerged as a powerful tool for processing and dissecting numerical biomedical data, making it a popular choice for addressing analytical challenges in biomedical research. Nevertheless, the intricate algorithms and complex optimization processes of machine learning model frameworks have deterred numerous non-machine learning experts. To address the need for comprehensive signatures exploration in biomedical field, we have developed Maler (http://www.inbirg.com/maler/home), a platform designed to facilitate the effortless and efficient application of machine learning models to high-dimensional biomedical data for bioscientists. Maler primarily offers four machine learning modes and a variety of machine learning algorithms to accommodate diverse analysis requirements. Simultaneously, it employs multiple feature selection methods to acquire relevant features and selects the optimal combinations of features based on machine learning models to achieve the best predictive outcomes. Maler is expected to provide a user-friendly and efficient platform for marker discovery, catering to the needs of biologists and clinical experts.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.09.653008v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Reducing Manual Labour in Forensic Microtrace Recognition with Deep Learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rijpkema, G., Kalisvaart, D., Korovin, S., Spengler, D., Pals, A., van der Weerd, J., Smith, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We use deep learning to automate trace recognition in images captured with automated microscopy.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Forensic microtrace investigation relies on time- and labor-intensive microscopic analyses. To aid forensic experts in their investigations, an image recognition model for microtrace localisation and classification is needed. In this work, we use deep learning to automate trace recognition in images captured with automated microscopy. We localise and classify fibres, hairs, skin, glass and sand in microscopy scans through pixel-wise classification of tape-lift samples. As deep learning requires extensive amounts of annotated training data, we additionally investigate various pretraining strategies to minimise the required annotation workload. We compare ImageNet pretraining, pretraining with self-supervised learning and a sequential application of these approaches. We find that pretrained models are able to reduce the required annotated data twofold compared to models trained from scratch while retaining the prediction accuracy. While our ImageNet-pretrained models outperform our self-supervised-pretrained models, we achieve the highest accuracy by combining the two approaches, resulting in a factor of 4 reduction of manual annotated microtraces or a 65% improvement in recognition and localisation accuracy (mean intersection over union increases from 0.34 to 0.56 due to pretraining) when training on only 2.2 dm2 of annotated tape lift scans. Our model is, therefore, the method of choice for the automatic analysis of large forensic microtrace scans.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.09.653056v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">VEFill: a model for accurate and generalizable deep mutational scanning score imputation across protein domains</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Polunina, P. V., Maier, W., Rubin, A. F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed VEFill, a gradient boosting model for imputing missing DMS scores across protein domains.</p><p style="font-size: 16px; text-align: justify;">Background: Deep Mutational Scanning (DMS) assays can systematically assess the effects of amino acid substitutions on protein function. While DMS datasets have been generated for many targets, they often suffer from incomplete variant coverage due to technical constraints, limiting their utility in variant interpretation and downstream analyses. Results: We developed VEFill, a gradient boosting model for imputing missing DMS scores across protein domains. VEFill is trained on the Human Domainome 1 dataset, a large, standardized set of DMS experiments using a uniform stability-based assay, and integrates a broad set of additional biologically informative features including ESM-1v sequence embeddings, evolutionary conservation (EVE scores), amino acid substitution matrices, and physicochemical descriptors. The model achieved robust predictive performance (R squared = 0.64, Pearson r = 0.80). It also demonstrated reliable generalization to unseen proteins in other stability-based datasets, while showing weaker performance on activity-based assays. Per-protein models further confirmed VEFill's effectiveness under limited-data conditions. A reduced two-feature version using only ESM-1v embeddings and mean DMS scores performed comparably to the full model, suggesting a computationally efficient alternative. However, true zero-shot prediction without positional context remains a challenge, particularly for functionally complex proteins. Conclusions: VEFill offers an interpretable, scalable framework for DMS score imputation, especially effective in stability-focused and sparse-data settings. It enables systematic mutation prioritization and may support the design of efficient experimental libraries for variant effect studies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.14.653991v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ProGenFixer: an ultra-fast and accurate tool for correcting prokaryotic genome sequences using a mapping-free algorithm</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Song, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present ProGenFixer, an ultra-fast, mapping-free tool specifically designed to efficiently identify and correct errors in prokaryotic genomes using next-generation sequencing (NGS).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Although prokaryotic genomes are simpler, making variation analysis more straightforward, an efficient and user-friendly tool for their rapid sequence correction is lacking. We present ProGenFixer, an ultra-fast, mapping-free tool specifically designed to efficiently identify and correct errors in prokaryotic genomes using next-generation sequencing (NGS) data. ProGenFixer compares k-mer profiles between the reference genome and reads to pinpoint discrepancies, employs a local assembly-based algorithm to estimate corrected sequences, and automatically implements corrections. Benchmarking demonstrates ProGenFixer's speed and accuracy advantages. It is over >5x faster than the swiftest existing tool tested while maintaining higher accuracy. Compared to slower, highly accurate tools, ProGenFixer shows comparable accuracy but is >17x faster and demonstrates enhanced performance on long indels (>10 bp). Implemented in C, ProGenFixer is a standalone, user-friendly tool offering a comprehensive solution for prokaryotic genome correction. The software is freely available at https://github.com/Scilence2022/ProGenFixer. Additionally, a companion web-based platform offering a ready-to-use interface can be accessed at https://progenfixer.biodesign.ac.cn.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.09.653025v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Hierarchical Approaches for Integrating Sparse, Multivariate Toxicological Effect Data in Whole Organism Molecular Dynamics Models</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Schunck, F., Busch, W., Focks, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified inadequate dynamics of the uptake and metabolization kinetics in the ODE system, which led to strong overfitting when coupled to the hierarchical approach when wide prior distributions for the experimental error were used.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: Integrating time-resolved toxicity data observed along multiple steps along the whole-organism exposure-to-effect pathway into mechanistic ODE models promises improved chemical risk assessment and predictive toxicology. However, effective incorporation of such multivariate observations is mainly hindered by the inherent sparsity and variability of observation matrices characterized by low overlap in observation types between different experiments, due to tissue consumption and cost of whole-organism experiments. Results: We analyzed the potential of bayesian hierarchical ODE models to estimate model parameters from sparse, non-overlapping, multivariate, time resolved datasets subject to experimental variability. The dataset in this study consisted of time series of chemical residue, gene-expression and survival obtained from 5 years of chemical exposure experiments with zebrafish embryos. Using the hierarchical approach, it was possible to increase the number of observations used in the model because experimental deviations in the exposure concentration could be accounted for. We identified inadequate dynamics of the uptake and metabolization kinetics in the ODE system, which led to strong overfitting when coupled to the hierarchical approach when wide prior distributions for the experimental error were used. In contrast, narrow distribution for the experimental error successfully avoided overfitting, despite model inadequacy. An additional simulation study of an adequate model confirmed that hierarchical models estimate parameters with low (10%) bias even under complete sparsity (i.e. no experimental overlap between measured variables); filling even small amounts (2.5%) of the missing information further reduced the parameter bias to (5%). Availability and Implementation: The ODE model used in this study was implemented in the pymob model-building framework (https://github.com/flo-schu/pymob/), which provides the infrastructure for specifying high-dimensional ODE models and provides tools for bayesian inference. The case-study is available under https://github.com/flo-schu/hierarchical_molecular_tktd. The snakemake workflow for fitting pymob models is available under https://github.com/flo-schu/pymob-workflow.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.09.652942v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Pathogenomic fingerprinting to identify associations between tumor morphology and epigenetic states.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shayan Monabbati, Germán Corredor, Tilak Pathak, Craig Peacock, Kailin Yang, Shlomo Koyfman, Peter Scacheri, James Lewis, Anant Madabhushi, Satish E Viswanath, Berkley Gryder</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that changes in chromatin accessibility to the chromatin state of a tumor can be linked to the pathological signatures of complex tumors.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Measuring the chromatin state of a tumor provides a powerful map of its epigenetic commitments; however, as these are generally bulk measurements, it has not yet been possible to connect changes in chromatin accessibility to the pathological signatures of complex tumors. In parallel, recent advances in computational pathology have enabled the identification of spatial features and immune cells within oral cavity tumors and their microenvironment.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40239399/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Advancing optical nanosensors with artificial intelligence: A powerful tool to identify disease-specific biomarkers in multi-omics profiling.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bakr Ahmed Taha, Zahraa Mustafa Abdulrahm, Ali J Addie, Adawiya J Haider, Ali Najem Alkawaz, Isam Ahmed M Yaqoob, Norhana Arsad</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-02-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> This review highlights the transformative potential of combining optical nanosensors with artificial intelligence (AI) to revolutionize disease diagnosis and treatment.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Multi-omics profiling integrates genomic, epigenomic, transcriptomic, and proteomic data, essential for understanding complex health and disease pathways. This review highlights the transformative potential of combining optical nanosensors with artificial intelligence (AI). It is possible to identify disease-specific biomarkers using real-time and sensitive molecular interactions. These technologies are precious for genetic, epigenetic, and proteomic changes critical to disease progression and treatment response. AI improves multi-omics profiling by analyzing large, diverse data sets and common patterns traditional methods overlook. Machine learning tools Biomarkers Discovery is revolutionizing, drug resistance is being understood, and medicine is being personalized as the combination of AI and nanosensors has advanced the detection of DNA methylation and proteomic signatures and improved our understanding of cancer, cardiovascular disease and vascular disease. Despite these advances, challenges still exist. Difficulties in integrating data sets, retaining sensors, and building scalable computing tools are the biggest obstacles. It also examines various solutions with advanced AI algorithms and innovations, including fabrication in nanosensor design. Moreover, it highlights the potential of nanosensor-assisted, AI-driven multi-omics profiling to revolutionize disease diagnosis and treatment. As technology advances, these tools pave the way for faster diagnosis, more accurate treatment and improved patient outcomes, offering new hope for personalized medicine.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/39919475/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

