<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Monday, 09 June 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrated muti-omics data and machine learning reveal CD151 as a key biomarker inducing chemoresistance in metabolic syndrome-related early-onset left-sided colorectal cancer.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yingdong Hou, Hubin Xia, Chenshan Xu, Yuhua Yu, Chenghao Ji, Wenli Ruan, Wencheng Kong, Yifeng Zhou, Xiaofeng Zhang</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified CD151 as a key biomarker in MetS-related EOLCC, offering valuable insights into prognosis, tumor biology, and personalized treatment strategies.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Emerging evidence has suggested a potential pathological association between early-onset left-sided colorectal cancer (EOLCC) and metabolic syndrome (MetS). However, the underlying genetic and molecular mechanisms remain insufficiently elucidated. This study aimed to identify and characterize key biomarkers associated with the progression and treatment response of MetS-related EOLCC. An in-hospital cohort was utilized to assess the clinical implications of primary tumor location in early-onset colorectal cancer (EOCRC). Differentially expressed genes (DEGs) and weighted gene coexpression network analysis (WGCNA) were employed to identify genes potentially associated with MetS-related EOLCC. Functional enrichment analyses were conducted to explore the underlying mechanisms. Candidate biomarkers were screened using random forest (RF) and support vector machine-recursive feature elimination (SVM-RFE) algorithms. Survival relevance, expression profiles, and diagnostic performance were analyzed to identify key biomarkers. Treatment responses were evaluated, and potential therapeutic compounds were identified through molecular docking. Single-cell RNA sequencing (scRNA-seq) data and in vitro experiments were used to validate gene expression and functional characteristics. The in-hospital cohort revealed a higher proportion of EOLCC among EOCRC patients. Using the edgeR package and WGCNA, we identified coexpressed genes common to both EOLCC and MetS, significantly enriched in pathways associated with stromal remodeling and metabolic regulation. Machine learning algorithms highlighted three candidate biomarkers. Among them, only CD151 was associated with prognosis and advanced disease stage. CD151 was strongly correlated with stromal remodeling and chemoresistance. Additionally, potential therapeutic compounds targeting MetS-related EOLCC were identified via molecular docking. scRNA-seq analysis confirmed the expression and functional patterns of CD151, particularly in tumor cells. The bioinformatics results were further validated through quantitative real-time PCR (qRT-PCR), western blotting, and immunohistochemical (IHC) staining. This study identified CD151 as a key biomarker in MetS-related EOLCC, offering valuable insights into prognosis, tumor biology, and personalized treatment strategies. CD151 may serve as a reference for future research and clinical applications targeting this disease subtype.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40484863/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Dynamic Retuning of Precursor Supply Before and After the Evolution of the Dual Lignin Pathway in Poales</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> El-Azaz, J., Moore, B., Maeda, H. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We study the evolution of aromatic amino acid (AAA) biosynthesis during the emergence of tyrosine-derived lignin pathway uniquely found in grasses and closely related Poales.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Plants produce diverse and abundant natural products, critical for plant adaptation and human society. While tremendous diversification of plant natural product pathways is well documented, how the upstream primary metabolism evolved in coordination with the lineage-specific pathways remain poorly understood. Here we studied the evolution of aromatic amino acid (AAA) biosynthesis during the emergence of tyrosine-derived lignin pathway uniquely found in grasses and closely related Poales. Phylogeny-guided in vitro and in planta functional analyses were conducted on two key regulatory enzymes, 3-deoxy-D-arabino-heptulosonate 7-phosphate synthase (DHS) and arogenate dehydrogenase (TyrA), catalyzing the first and last steps of AAA and tyrosine biosynthesis, respectively. Deregulated TyrAs were broadly present across Poales, with PACMAD grasses exhibiting stronger deregulation, suggesting that the active tyrosine biosynthesis likely emerged before, but were also selected upon, the evolution of tyrosine-lignin pathway. Conversely, DHS deregulation occurred more recently within BOP grasses after the emergence of tyrosine-lignin pathway, via multiple mutations that acted synergistically in the allosteric regulatory domain of the enzyme. The study highlights how the regulation of upstream primary metabolism is dynamically tuned both before and after the evolution of emerging natural product pathways. These findings underscore the importance of coordinating primary and secondary metabolism for efficient production of natural products in plants.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.05.658145v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Phytoene synthase modulates seed longevity via the action of β-carotene derived metabolites</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hazra, A., Rao, S., Zhou, X., Christel, M. J., Wrightstone, E., Yang, Y., Fish, T., Thannhauser, T., Liu, J., Mazourek, M., Li, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We found that phytoene synthase (PSY), the gene encoding a major rate-limiting enzyme in carotenoid biosynthesis, modulated seed storability under both natural and artificial aging conditions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Seed longevity, the ability of seeds to stay viable over time, is an important trait in agriculture. Despite extensive research, seed longevity remains one of the fundamental topics in plant biology. Here, we discovered the novel role of carotenoid metabolites in prolonging seed lifespan. We found that phytoene synthase (PSY), the gene encoding a major rate-limiting enzyme in carotenoid biosynthesis, modulated seed storability in Arabidopsis under both natural and artificial aging conditions. Seeds from the PSY overexpression (OE) lines exhibited significantly enhanced lifespan with low levels of reactive oxygen species (ROS), a major factor affecting longevity, whereas those from the psy mutants had decreased viability with high ROS levels. While lutein and beta-carotene were detected in seeds, only beta-carotene and its derived apocarotenoids, i.e. beta-cyclocitral and beta-ionone, were found to improve seed lifespan. Notably, the carotenoid cleavage dioxygenase 1 and 4 (ccd1 ccd4) double mutant and PSY ccd1 ccd4 seeds showed significantly reduced seed longevity, indicating that beta-carotene cleavage is necessary for or it is apocarotenoids playing the role in preserving seed lifespan. Comparative proteomic analysis identified TIP2;2, an aquaporin protein, which showed differential abundances in seeds of PSY OE and psy mutant vs wild type. The mutant tip2;2 had reduced seed longevity, and its promoter was transactivated by apocarotenoids. Collectively, this study uncovers a novel role of apocarotenoids in protecting seed longevity and highlights the importance of seed carotenoid production in strengthening agriculture.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.05.658166v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Drought stress modulates the molecular response of Arabidopsis plants to root-knot nematode infection</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Refaiy, A., Lilley, C. J., Atkinson, N. J., Urwin, P. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A unique suite of genes in response to joint joint stress, which significantly differed from the response to either individual stress.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Plants are exposed to multiple concurrent stresses under field conditions that can include a combination of biotic and abiotic factors. Drought stress and plant parasitic nematodes are among the most damaging stress factors limiting plant growth and production. Plants have developed complex and specific responses to combined biotic and abiotic stresses, which differ from their responses to individual stresses. Therefore, there is an imperative need to understand plant responses to multiple stresses as a central avenue for development of robust plants with the ability to sustain growth and crop production in times of global climate change. RNA-Seq analysis was performed on Arabidopsis plants to investigate the transcriptomic responses to infection with the root-knot nematode Meloidogyne incognita and drought stress either individually or concurrently. Arabidopsis plants activated a unique suite of genes in response to the joint stress, which significantly differed from the response to either individual stress. Among these differentially expressed genes, Azelaic Acid Induced 1 (AZI1), small auxin upregulated RNA 71 (SAUR71), and Disease Related Nonspecific Lipid Transfer Protein 1 (DRN1) may play important roles in plant responses to concurrent drought stress and root-knot nematode infection. The expression of AZI1, which is involved in priming defences via systemic acquired resistance, was uniquely upregulated in Arabidopsis leaves in response to the combined stress. SAUR71, which is a member of the largest family of primary auxin response genes, was induced solely in response to combined stress. In contrast, the expression of DRN1, a member of the non-specific lipid-transfer protein family, was strongly suppressed in Arabidopsis roots by combined drought stress and nematode infection.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.05.658137v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Features fusion or not: harnessing multiple pathological foundation models using Meta-Encoder for downstream tasks fine-tuning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gao, R., Yang, Z., Yuan, X., Wang, Y., Xia, Y., Zhang, Y., Zheng, B., Gong, Y., Yue, Y., Yu, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose the Meta-Encoder framework, which integrates features from multiple foundation models to generate a comprehensive representation, improving downstream fine-tuning task performance and alleviating concerns over model selection.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The emergence of diverse pathological foundation models has empowered computational pathology tasks, including tumor classification, biomarker prediction, and RNA expression prediction. However, variations in model architecture and data sources lead to inconsistent downstream performance and complicate centralized training. Specifically, the lack of data sharing makes retraining foundation models with pooled data infeasible. Alternatively, the release of model parameters enables combining multiple models during fine-tuning. Inspired by the meta-analysis method, we propose the Meta-Encoder framework, which integrates features from multiple foundation models to generate a comprehensive representation, improving downstream fine-tuning task performance. Comparative experiments demonstrate that Meta-Encoder is more effective than individual foundation models, with its strengths more pronounced in handling complex tasks. While single models may perform sufficiently well for simple tasks, Meta-Encoder can match or even surpass the best-performing single model, alleviating concerns over model selection. Moderately challenging tasks benefit from Meta-Encoder's concatenation or self-attention strategies, with the latter demonstrating superior performance in more challenging scenarios. For highly complex tasks, such as high-dimensional gene expression prediction, self-attention proves to be the most effective Meta-Encoder strategy, balancing feature integration and computational efficiency. For three patch-level spatial gene expression prediction tasks (HEST-Benchmark, CRC-inhouse, and Her2ST), the self-attention strategy improved the Pearson correlation by 38.58%, 26.06%, and 20.39%, respectively, compared to the average performance of three patch-level single models. Similarly, for the TCGA-BRCA, TCGA-NSCLC, and TCGA-CRC WSI-level bulk gene expression prediction tasks, the Pearson correlation increased by 14.36%, 9.27%, and 42.55%, respectively, compared to the average performance of two WSI-level single models. By leveraging multiple pathological foundation models using Meta-Encoder, it can further improve molecular characterization in pathology images to advance precision oncology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.05.657960v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">tidypopgen: Tidy Population Genetics in R</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Carter, E. J., Tysall, E. E., Hodgson, J., Manica, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce tidypopgen, a comprehensive R package for population genetic analysis of biallelic SNP data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> As genome-wide data has become increasingly available, software libraries for their analysis have proliferated. While new tools for downstream analyses are constantly emerging, existing workflows are hindered by inefficiencies. Switching between coding languages and object types in the early stages of pipelines wastes researchers' time, impedes reproducibility, and creates opportunity for error. To confront these obstacles, we introduce tidypopgen, a comprehensive R package for population genetic analysis of biallelic SNP data. Genotype data can be read, filtered, and analysed within a single environment, without the need for prior data cleaning or setup with other software. tidypopgen's gen_tibble object structure makes analysis efficient and intuitive, while standardised tidy grammar makes data manipulation clear. Functionality within tidypopgen supports cleaning and merging datasets, basic descriptive statistics, multivariate analysis, clustering algorithms, and F-statistics, as well as integrating with existing tools for population genetic analyses in R. We use the Human Genome Diversity Project SNP dataset (Li et al., 2008) to show that a basic population genetic workflow can be executed in under 25 lines of code in a single environment using one file set, without the need to write superfluous outputs or change directories. By supporting data assembly through to data analysis, tidypopgen significantly streamlines workflows without compromising speed or functionality.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.06.658325v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Cell-specific RNA isoform remodeling in the aging mouse brain</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rehman, A., Duffy, M., Gresova, K., Belair, C., Yang, N., Wang, L., Lee, C. T., Payea, M., Malla, S., Dar, S. A., Sen, P., Cookson, M. R., Maragkakis, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify non-linear, cell-type-specific isoform expression changes and isoform usage shifts, primarily driven by transcription start site selection, in the aging mouse cortex and hippocampus.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Aging is associated with increases in risk of multiple chronic diseases and at the cellular level, with disruptions in RNA homeostasis. Single-cell transcriptomic studies have revealed RNA abundance heterogeneity across brain cell types but the more complex changes in RNA processing such as alternative splicing, isoform usage, transcription start and polyadenylation site selection remain uncharacterized. Here, we combine single-cell analysis with long-read nanopore sequencing to capture full-length RNA isoforms and uncover temporal changes in RNA transcription, processing, and alternative splicing in the aging mouse cortex and hippocampus. By profiling transcriptomes from young adult to very old mice, we identify non-linear, cell-type-specific isoform expression changes and isoform usage shifts, primarily driven by transcription start site selection. These aging-associated isoform changes alter the coding potential and poly(A) site position of genes. Our data also reveal a high proportion of senescence in immune cells, far exceeding that of other cell types. We also identify isoform markers that, when applied to a machine learning model, distinguish senescent from normal immune cells. This study provides a full-length RNA isoform-based atlas of the aging mouse brain, offering insights into RNA metabolism remodeling across brain cell types throughout the lifespan.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.05.658133v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Novel Genus of Endogenous Pararetroviruses with Long Terminal Repeats in Grasses</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gao, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new EPRV with long terminal repeats in barley, Hvu-EPRV, and its homologs in 17 grasses, which may be ancient invaders from the grass family.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Despite being widespread in plants, endogenous pararetroviruses (EPRVs) are still poorly understood in barley and many other cereal crops. In this study, the barley reference genome was examined and from that a new EPRV was identified and named Hvu-EPRV. In contrast to all EPRVs identified thus far, Hvu- EPRV contains long terminal repeats (LTRs) which are similar to LTR retrotransposons. Homologous sequences of Hvu-EPRV were found in a wide range of plants, however, only those in 17 grasses belonging to the six tribes contain LTRs. The insertion times of nested LTR retrotransposons indicated that Hvu-EPRVs inserted into barley more than 2.37 million years ago, but the invasion and endogenization of Hvu-EPRV related elements in the grass family may be ancient, and horizontal transfers may have occurred between grasses. Phylogenetic analysis revealed that Hvu-EPRV and its homologs in grasses were grouped apart from all 13 reported genera of exogenous and endogenous pararetroviruses, thus the EPRVs in grasses represent a novel genus of the Caulimoviridae family named Moridahovirus. Genome-wide comparisons of Hvu-EPRVs were conducted between the reference genome and other 84 genomes of cultivated and wild barley, three independent integration events were observed and suggested that the integrations likely occurred after the divergence between barley and its wild progenitor. This is the first time to identify EPRVs with LTRs and to detect their recent integrations, and this research provides new insights into the evolution of plant EPRVs and their invasion history in the grass family.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.08.658526v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Enhancing CYP450-Ligand Binding Predictions: A Comparative Analysis of Ligand-Based and Hybrid Machine Learning Models</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tikhonova, A., Chan, E. C. Y., Fan, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a hybrid machine learning framework integrating ligand descriptors, protein descriptors and protein-ligand interaction descriptors that include molecular docking-derived parameters, rescoring function components from multiple algorithms and structural interaction fingerprints (SIFt).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Predicting cytochrome P450 (CYP450) ligand binding is critical in early-stage drug discovery, as CYP450-mediated metabolism profoundly influences drug efficacy, safety, and adverse reaction risks. However, experimental determination of CYP450-ligand interactions remains resource- and time-intensive, underscoring the need for robust computational alternatives. While ligand-based methods are commonly employed, they often fail to fully account for structural intricacies governing protein-ligand interactions. To address this gap, we developed a hybrid machine learning framework integrating ligand descriptors, protein descriptors, and protein-ligand interaction descriptors, that include molecular docking-derived parameters, rescoring function components from multiple algorithms and structural interaction fingerprints (SIFt). Evaluated on CYP1A2 and CYP17A1 isoforms, our model demonstrated superior predictive accuracy in cross validation compared to stand-alone molecular docking and ligand-based approaches. Furthermore, benchmarking against state-of-the-art tools (SwissADME and ADMETlab 3.0) revealed enhanced performance in binding prediction. This work establishes a versatile framework for advancing computational tools to prioritize CYP450 binding assessments during drug discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.05.658010v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Quantifying Protein-Protein Interaction with a Spatial Attention Kinetic Graph Neural Network</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> xu, y., Xia, W., Zhang, C., Liu, X., Ju, C., Dai, X., Xie, P., Wang, Y., Chen, G., Zhang, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present SAKE-PP, a physics-inspired, spatial-attention equivariant graph neural network that directly regresses interface RMSD (iRMSD) without any native references.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurate prioritization of near-native protein-protein interaction (PPI) models remains a major bottleneck in structural biology. Here we present SAKE-PP, a physics-inspired, spatial-attention equivariant graph neural network that directly regresses interface RMSD (iRMSD) without any native references. Trained on docking decoys generated through our novel hierarchical sampling strategy applied to the PDBBind dataset, SAKE-PP combines force-field-like attention with Laplacian-eigenvector orientation to couple local interaction forces with global topology. On the 2024PDB benchmark comprising 176 heterodimers, SAKE-PP demonstrates effective optimization and selection of AF3 decoys, achieving improvements of 13.75% based on iRMSD statistics and 12.5% based on DockQ scores. It consistently outperforms the AF3 ranking score across multiple metrics, including overlap, hit rate, and correlation. In zero-shot evaluation of 139 antibody-antigen complexes, SAKE-PP increases the score-iRMSD correlation by 0.4. By unifying geometric deep learning with physics-based realism, SAKE-PP provides a robust, plug-and-play scoring function that streamlines reliable PPI evaluation and accelerates downstream structure-guided drug-design workflows.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.04.657832v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Evidence for Transcriptomic Conservation Between the Main Cells of the Drosophila Prostate-Like Accessory Gland and Basal Cells of the Mammalian Prostate.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Church, S. J., Kriebel, A. R., Welch, J. D., Buttitta, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A cross-species comparative analysis of single-cell RNA sequencing data from adult tissues using data from the Drosophila Fly Cell Atlas and mammalian adult prostate single- cell datasets.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The Drosophila accessory gland performs functions analogous to the mammalian prostate in production of seminal fluid components that are essential for male fertility. The mammalian prostate and Drosophila accessory glands share a similar tissue organization and structure. Both organs contain secretory epithelial cells forming a gland lumen, surrounded by a stroma with extracellular matrix enveloped by innervated muscle for organ contraction and fluid release. However, the Drosophila accessory gland secretory epithelium is postmitotic, polyploid and binucleate, and lacks a known stem cell population. By contrast, the mammalian prostate epithelium is made up of diploid luminal secretory cells and diploid basal cells that are maintained by at least two stem cell populations. Despite the differences in the tissues, it has been argued these tissues may share a deep homology based on the expression of conserved genes during development. Here we performed a cross-species comparative analysis using single-cell RNA sequencing data from adult tissues using data from the Drosophila Fly Cell Atlas and mammalian adult prostate single-cell datasets. Our analysis provides additional evidence of transcriptomic similarity between the main epithelial cells of the Drosophila prostate-like accessory gland and the basal epithelial cells of the mammalian prostate. While we do not know whether these similarities reflect shared evolutionary homology, or independently derived features due to shared tissue functions, our results strengthen the arguments that the Drosophila accessory gland can be used to effectively model aspects of human prostate biology and disease.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.05.658085v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identification of chromatin-associated RNAs at human centromeres</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fryer, K. A., Limouse, C., Straight, A. F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify and map RNAs associated with centromeric chromatin in human centromeres and propose a new class of centromere bound RNAs that may regulate the organization and activity of the centromere.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Centromeres are a specialized chromatin domain that are required for the assembly of the mitotic kinetochore and the accurate segregation of chromosomes. Non-coding RNAs play essential roles in regulating genome organization including at the unique chromatin environment present at human centromeres. We performed Chromatin-Associated RNA sequencing (ChAR-seq) in three different human cell lines to identify and map RNAs associated with centromeric chromatin. Centromere enriched RNAs display distinct contact behaviors across repeat arrays and generally belong to three categories: centromere encoded, nucleolar localized, and highly abundant, broad-binding RNAs. Most centromere encoded RNAs remain locally associated with their transcription locus with the exception of a subset of human satellite RNAs. This work provides a comprehensive identification of centromere bound RNAs that may regulate the organization and activity of the centromere.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.05.658139v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Mammalian epigenomic conservation of promoters and enhancers in the heart associates with trait-associated variation and impacts cardiomyocyte phenotypes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Frost, S., Fernandez-Aroca, D. M., Parey, E., Rodriguez-Gonzalez, A., Pavon-Heredia, D., Villar, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comparative genomics approach to identify epigenomically-conserved promoters and enhancers in the heart, as well as elements with primate-specific or human-only epigenomic signals.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Interindividual genetic variation associated with cardiovascular traits and disease is commonly found in the non-coding genome, suggesting regulatory changes underlie many genetic association signals. Mammalian gene regulation is controlled by promoters and enhancers with rapidly divergent activities. However, the interplay between human non-coding genetic variation and regulatory evolution remains largely unexplored. Here, we investigated promoter and enhancer evolution in the mammalian heart using genome-wide epigenomic profiling, intersected these elements with genetic variants associated with cardiovascular traits, and experimentally tested candidate regions in human cardiomyocytes derived from pluripotent stem cells. First, we applied a comparative genomics approach to identify epigenomically-conserved promoters and enhancers in the heart, as well as elements with primate-specific or human-only epigenomic signals. Second, we evaluated the association of common genetic variation and signatures of cell-type specific gene regulation with mammalian epigenomic conservation. We report an enrichment of common genetic variants in epigenomically-conserved promoters and enhancers, which also associate with regulatory pleiotropy across cardiac cell types, and promoter-enhancer contacts in cardiomyocytes. Based on these findings, we selected candidate epigenomically-conserved elements across three cardiovascular genetics loci (KCNH2, CPEB4 and PRKCE), and investigated gene expression and cellular outcomes upon CRISPR/Cas9-mediated deletion of each region in human cardiomyocytes. These analyses inform how mammalian epigenomic conservation associates with specific gene expression contributions and downstream cellular responses, such as cardiomyocyte hypertrophy and sensitivity to hypoxia/reoxygenation. Moreover, the comparative analyses we present here contribute to ongoing efforts to prioritise and functionally characterise cardiovascular association signals from human population genetics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.05.658128v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Spatial transcriptomics of developing wheat seed reveals radial expression patterns in endosperm and subgenome biased expression of key genes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Millsteed, T., Kainer, D., Sullivan, R., Sun, X., Li, K. L., Mao, L., Macdonald, A., Henry, R. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Spatial transcriptomics of gene expression within the wheat seed .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Gene expression of developing seeds drives essential processes such as nutrient storage, stress tolerance and germination. However, the spatial organisation of gene expression within the complex structure of the seed remains largely unexplored. Here we report the use of the STOmics spatial transcriptomics platform to visualise spatial expression patterns in the wheat (Triticum aestivum) seed at the critical period of grain filling in mid seed development. We analysed >4,000,000 spatially resolved transcripts, achieving subcellular resolution of transcript localization across multiple tissue domains, and identified gene expression clusters linked to eight functional cellular groups. Notably, our analysis characterised four distinct clusters within the endosperm, which exhibited radial expression patterns from the inner to outer regions of the grain, and identified novel marker gene candidates for the clusters found. We further investigated known tissue-specific genes and identified subgenome biased expression between paralogs of puroindoline-B, metallothionein protein, and -amylase/subtilisin inhibitor. These findings provide new detail about gene expression across and within different functional cellular groups of the developing seed and demonstrate that spatial transcriptomics could further our understanding of subgenome differences in polyploid plants.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.08.658529v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Vertical rooting caused by enhanced functional allele of qSOR1 improves rice yield under drought stress</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kawai, T., Kitomi, Y., Kuroda, R., Nakata, R., Iba, M., Soma, F., Teramoto, S., Yamazaki, T., Sugimoto, K., Uga, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that deep root system architecture promoted by the qSOR1-v mutant allele (an enhanced functional allele of the quantitative trait locus for SOIL SURFACE ROOTING 1) significantly improves rice yield under drought when compared to the deep RSA achieved through the functional qSor1</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Drought considerably affects crop productivity, and its severity is being intensified by climate change. Therefore, enhancing drought resistance is a crucial priority in crop breeding for ensuring sustainable agriculture. The root system architecture (RSA) influences the efficiency of water acquisition from land; therefore, a deep RSA is advantageous for avoiding drought stress. Here, we demonstrated that deeper RSA promoted by the qSOR1-v mutant allele (an enhanced functional allele of the quantitative trait locus for SOIL SURFACE ROOTING 1) significantly improves rice yield under drought when compared to the deep RSA achieved through the functional qSOR1 allele that originated from natural variation. The qSOR1-v mutant exhibited stronger root gravitropism than the wild type. This was characterized by a more pronounced polarization of auxin on the lower side during root curvature, leading to a robust vertical rooting phenotype that was consistently expressed across different soil-water environments. Additionally, the qSOR1-v mutation site was well conserved among angiosperm orthologs, and the corresponding mutation in LZY3 of Arabidopsis (qSOR1 ortholog) resulted in a steeper root growth angle. The rice introgression line, which was substituted from the functional qSOR1 allele to qSOR1-v through marker-assisted selection, showed vertical rooting, resulting in increased grain yield in an upland field under drought stress. No yield penalty was observed for this line under well-watered upland conditions than the original variety. These findings highlight the potential of qSOR1-v and corresponding mutations in angiosperm orthologs to promote vertical rooting across plant species, which can help sustain crop yields in drought-prone areas.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.07.658467v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Salicylic acid accumulation correlates with low anthocyanin production in Arabidopsis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Drs, M., Iakovenko, O., Orozco, J. S. H., Trhlinova, P. B., Markovice, V., Zarsky, V., Pecenkova, T., Janda, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that high endogenous SA accumulation correlates with low anthocyanin production in Arabidopsis thaliana, and suggest that downstream signaling independent of NPR1 is responsible for lower anthcyanin accumulation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Anthocyanins, flavonoid pigments, are essential photoprotective agents and play a pivotal role in enhancing plant resilience to environmental stressors. It has been shown that anthocyanin production is inhibited when pattern-triggered immunity (PTI) is activated in Arabidopsis thaliana. An important component of PTI is the phytohormone salicylic acid (SA). Interestingly, exogenous treatment with SA has been shown to induce anthocyanin content in grape, apple, maize roots, rose callus, or Arabidopsis seedlings. In this study, we used several A. thaliana mutants with modulated SA content to decipher the role of endogenous SA in anthocyanin production in A. thaliana. We treated WT and mutants with anthocyanin-inducible conditions and measured anthocyanin content using spectroscopy. We showed that high endogenous SA accumulation correlates with low anthocyanin production. This was confirmed by the treatment of the A. thaliana seedlings with exogenous SA. Additionally, using microscopy in the 5gt mutant, which exhibits enhanced production of anthocyanin vesicular inclusions (AVIs) due to the inhibition of ligandin-dependent vacuolar import, we showed that high endogenous SA also correlates with lower AVI abundance. Comparative analysis of Arabidopsis WT and mutants used in this study indicates a possible inhibitory effect of SA accumulation on anthocyanin content under anthocyanin-inducible conditions (AICs). We suggest that under AICs, SA downstream signaling independent of NPR1 is responsible for lower anthocyanin accumulation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.08.658514v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identification of mitochondria and macrophage activation related hub genes in sarcopenia</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, Y., Gu, S., Li, Y., Zhou, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study to explore hub genes and pathways related to sarcopenia progression associated with mitochondria and macrophage activation.</p><p style="font-size: 16px; text-align: justify;">BACKGROUND: To explore hub genes and pathways related to sarcopenia progression associated with mitochondria and macrophage activation. METHODS: GEO datasets GSE8479, GSE1428, and GSE136344 were obtained based on GEO database. MRGs and MARGs were acquired. Thereafter, GO, KEGG, GSEA and PPI analysis were implemented. Later, pooled GEO datasets (Combined Datasets) were further obtained to construct the sarcopenia diagnostic model and sarcopenia subtypes. RESULTS: A total of 62 M&MARDEGs were finally identified. There were altogether 9 model genes incorporated into the multifactorial regression model. Immunoinfiltration analysis revealed low scores of M&MARDEGs in the samples with sarcopenia. For macrophage activation in sarcopenia samples, most immune cells were strongly correlated in LowScore group (r = -0.707, p < 0.05). Many immune cells in Mitochondria & Macrophage Activation (HighScore) group were strongly correlated with each other (r = -0.661, p < 0.05). CONCLUSION: Screening key genes for sarcopenia associated with mitochondrial and macrophage activation in this study assists in understanding the pathogenesis of sarcopenia and providing new targets for intervention.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.04.657824v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Prioritization of Potential Drugs Through Pathway-Based Drug Repurposing and Network Proximity Analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Otero Carrasco, B., Prieto Santamaria, L., Rodriguez Gonzalez, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a pathway-based drug repurposing approach that combines protein-protein interaction (PPI) networks and transcriptomic data to prioritize candidate drugs.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Drug repurposing is an effective strategy to identify novel therapeutic options by leveraging existing drugs with known mechanisms of action and safety profiles. This work introduces a pathway-based drug repurposing approach that combines protein-protein interaction (PPI) networks and transcriptomic data to prioritize candidate drugs. The underlying assumption is that if disease 1 and disease 2 are both associated with the same biological pathway, then drugs known to be effective for disease 2 may be potential treatments for disease 1. To explore these relationships, disease.pathway.drug triplets are constructed. Genes shared between disease 1 and the associated pathway are identified, and their proximity is calculated within the PPI network using a Z score based on random permutations. Candidate drugs for disease 2 are then analyzed using the Connectivity Map data, assessing whether their up- or down-regulated gene signatures are enriched for the genes contributing to the proximity between disease 1 and the pathway. This integration enables the ranking of candidate drugs based on their potential biological impact on disease 1. Finally, top-ranked drug disease associations are evaluated through manual curation of the scientific literature to assess existing evidence supporting the proposed repurposing hypothesis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.05.658004v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ImmunoFusion: A Unified Platform for Investigating RNA-seq-Derived Gene Fusions in Cancer and Immunotherapy</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> zhao, y., Wang, S., Li, S., Chen, M., Jin, S.-h., Gaipl, U. S., Ma, H., Zhou, J.-G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A unified cancer fusion database and analytical tool to explore the influence of gene fusions on cancer and immune responses.</p><p style="font-size: 16px; text-align: justify;">Background Gene fusions play a critical role in cancer development by persistently activating kinases or inactivating tumor suppressor genes, leading to altered signal transduction and gene expression regulation. However, their impact on treatment responses remains poorly understood. Although existing cancer databases catalog numerous fusion events or immune checkpoint blockade (ICB) studies, no unified platform integrates gene fusion data across cancer types while linking them to the tumor microenvironment (TME) and patient outcomes. Such integration is essential for elucidating how fusions shape immune responses and for developing improved biomarkers for personalized cancer therapies. Methods To address this gap, we constructed the Fusion Immune Atlas (ImmunoFusion), a platform integrating data from TCGA, TARGET, CPTAC, and 29 ICB cohorts (including ICB treatments and other treatment modalities) across diverse cancer types. We identified fusion events from raw RNA-seq data using Arriba and STAR-Fusion, and standardized fusion calls by adapting MetaFusion to the GRCh38 reference genome. Additionally, we curated clinical data and estimated TME signatures and cell fractions using the Immuno-Oncology Biological Research (IOBR) approach. ImmunoFusion was developed using the high-quality Rhino framework for Shiny applications, built on R. Results ImmunoFusion (https://shiny.zhoulab.ac.cn/ImmunoFusion) encompasses 21,014 clinical samples, serving as a comprehensive RNA-seq-derived gene fusion database and analytical tool. It offers functionalities to investigate fusion breakpoints, confidence scores, frequency patterns, and associations with clinical variables. The platform also enables TME evaluation and interactive exploration of fusions for analyzing tumor immunophenotypes in cancer and immunotherapy contexts. As an illustration, analysis of MTAP fusions in lung cancer cohorts revealed their association with a metabolically depleted, "cold" (immune-suppressive) tumor environment and poorer patient outcomes, pinpointing MTAP fusions as a novel biomarker for treatment selection. Conclusions ImmunoFusion represents a significant advancement, delivering a unified platform to explore the influence of gene fusions on cancer and immune responses. It offers particular value in understanding fusion-driven immunotherapy outcomes, paving the way for more effective therapeutic strategies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.05.658082v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">scMetaIntegrator: a meta-analysis approach to paired single-cell differential expression analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ratnasiri, K., Mach, S. N., Blish, C. A., Khatri, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel meta-analysis approach that accounts for biological replicates and cell variability in paired scRNA-seq data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Traditional differential gene expression methods are limited for analysis of single cell RNA-sequencing (scRNA-seq) studies that use paired repeated measures and matched cohort designs. Many existing approaches consider cells as independent samples, leading to high false positive rates while ignoring inherent sampling structures. Although pseudobulk methods address this, they ignore intra-sample expression variability and have higher false negatives rates. We propose a novel meta-analysis approach that accounts for biological replicates and cell variability in paired scRNA-seq data. Using both real and synthetic datasets, we show that our method, single-cell MetaIntegrator (https://github.com/Khatri-Lab/scMetaIntegrator), provides robust effect size estimates and reproducible p-values.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.04.657898v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Mechanistic insights into Down syndrome comorbidities via convergent RNA-seq and TWAS signals</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Subirana-Granes, M., Zhang, H., Gupta, P., Pividori, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A network-based, integrative framework that combines whole-blood transcriptomic data with gene-trait associations to uncover mechanistic insights into DS-associated conditions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Down syndrome (DS) is caused by trisomy of chromosome 21 and is associated with diverse clinical manifestations, yet the molecular pathways linking chromosome-21 dosage effects to DS comorbidities remain poorly defined. Here we address this gap by applying a network-based, integrative framework that combines whole-blood transcriptomic data with gene-trait associations to uncover mechanistic insights into DS-associated conditions. First, we performed matrix factorization using PLIER on Human Trisome Project (HTP) RNA-Seq profiles from 304 trisomy-21 (T21) and 95 euploid (D21) individuals, deriving 156 biologically interpretable gene modules. We then identified 92 modules whose activity differed significantly between T21 and D21 and annotated these with prior-knowledge and KEGG pathways. To connect modules to clinical traits, we integrated PrediXcan-derived TWAS results from the UK Biobank, revealing 25 T21-specific modules with significant gene-trait associations (FDR 0.65, mAPS > 0.65). Most notably module 37, an interferon-stimulated gene network, whose elevated expression robustly distinguished DS individuals with pulmonary hypertension (AUC = 0.76, mAPS = 0.73). Overall, our study demonstrates that integrating blood-derived gene modules with population-scale genetic data uncovers coherent molecular signatures underlying DS comorbidities, identifies candidate biomarkers and therapeutic targets (e.g., ISG15, IFITs, MX1), and highlights the power of combining transcriptomic and genetic evidence to elucidate complex disease mechanisms.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.05.658129v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

