<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Tuesday, 22 April 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrated multiomics analysis identifies potential biomarkers and therapeutic targets for autophagy associated AKI to CKD transition.</h2>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">This study explored the relationship between acute kidney injury (AKI) and chronic kidney disease (CKD), focusing on autophagy-related genes and their immune infiltration during the transition from AKI to CKD. We performed weighted correlation network analysis (WGCNA) using two microarray datasets (GSE139061 and GSE66494) in the GEO database and identified autophagy signatures by Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG), and GSEA enrichment analysis. Machine learning algorithms such as LASSO, random forest, and XGBoost were used to construct the diagnostic model, and the diagnostic performance of GSE30718 (AKI) and GSE37171 (CKD) was used as validation cohorts to evaluate its diagnostic performance. The study identified 14 autophagy candidate genes, among which ATP6V1C1 and COPA were identified as key biomarkers that were able to effectively distinguish between AKI and CKD. Immune cell infiltration and GSEA analysis revealed immune dysregulation in AKI, and these genes were associated with inflammation and immune pathways. Single-cell analysis showed that ATP6V1C1 and COPA were specifically expressed in AKI and CKD, which may be related to renal fibrosis. In addition, drug prediction and molecular docking analysis proposed SZ(+)-(S)-202-791 and PDE4 inhibitor 16 as potential therapeutic agents. In summary, this study provides new insights into the relationship between AKI and CKD and lays a foundation for the development of new treatment strategies.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40258914/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrated bioinformatic analysis of immune infiltration and disulfidptosis related gene subgroups in type A aortic dissection.</h2>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Type A aortic dissection (TAAD) is a lethal cardiovascular disease characterized by the separation of the layers within the aortic wall. The underlying pathological mechanisms of TAAD requires further elucidation to develop effective prevention and pharmacological treatment strategies. Inflammation plays a crucial role in TAAD pathogenesis. Disulfidptosis, an emerging type of cell death, may shed light on disease mechanisms. This study investigates the role of disulfidptosis-related genes in immune infiltration in TAAD. TAAD gene expression datasets were obtained from the Gene Expression Omnibus (GEO) database. Immune cell infiltration analysis assessed immune cell dysregulation in TAAD. Differentially expressed genes (DEGs) between TAAD samples and controls were identified and intersected with known disulfidptosis-related gene sets to obtain relevant DEGs. Hub genes were identified using machine learning algorithms. A diagnostic model was constructed using Least Absolute Shrinkage and Selection Operator (LASSO) regression on 25 TAAD samples. Consensus clustering classified TAAD samples based on disulfidptosis-related gene expression. Functional enrichment analyses, including Gene Ontology (GO) terms and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway analyses, elucidated associated biological processes and pathways. A total of 13,316 DEGs were identified, among which 11 disulfidptosis-related genes were screened: INF2, CD2AP, PDLIM1, ACTN4, MYH10, MYH9, FLNA, FLNB, TLN1, MYL6, ACTB, CAPZB, DSTN, and IQGAP1. Most of these genes exhibited lower expression levels in TAAD samples, except CAPZB, and were correlated with immune cell infiltration. Cluster-specific DEGs were found in one cluster, involving several immune response processes. Co-clustering analysis based on disulfidptosis-related genes classified TAAD samples into two clusters, with higher gene expression levels observed in cluster C2 compared to cluster C1. Three key hub genes were identified, and potential therapeutic mechanisms for TAAD were explored. Immuno-infiltration results revealed significant differences in immune profiles, with higher immunological scores and more extensive immune infiltration in TAAD. Disulfidptosis occurs in TAAD and is associated with immune cell infiltration and metabolic activity, influencing immune cell function and responses. These findings suggest that disulfidptosis may promote TAAD progression through the induction of immune responses and metabolic activities. This research provides new insights into the pathogenesis and identifies potential therapeutic targets for TAAD.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40258895/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Deconvolution of cell types and states in spatial multiomics utilizing TACIT.</h2>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Identifying cell types and states remains a time-consuming, error-prone challenge for spatial biology. While deep learning increasingly plays a role, it is difficult to generalize due to variability at the level of cells, neighborhoods, and niches in health and disease. To address this, we develop TACIT, an unsupervised algorithm for cell annotation using predefined signatures that operates without training data. TACIT uses unbiased thresholding to distinguish positive cells from background, focusing on relevant markers to identify ambiguous cells in multiomic assays. Using five datasets (5,000,000 cells; 51 cell types) from three niches (brain, intestine, gland), TACIT outperforms existing unsupervised methods in accuracy and scalability. Integrating TACIT-identified cell types reveals new phenotypes in two inflammatory gland diseases. Finally, using combined spatial transcriptomics and proteomics, we discover under- and overrepresented immune cell types and states in regions of interest, suggesting multimodality is essential for translating spatial biology to clinical applications.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40258827/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Assembly and reasoning over semantic mappings at scale for biomedical data integration</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Motivation: Hundreds of resources assign identifiers to biomedical concepts including genes, small molecules, biological processes, diseases, and cell types. Often, these resources overlap by assigning identifiers to the same or related concepts. This creates a data interoperability bottleneck, as integrating data sets and knowledge bases that use identifiers for the same concepts from different resources require such identifiers to be mapped to each other. However, available mappings are incomplete and fragmented across individual resources, motivating their large-scale integration. Results: We developed SeMRA, a software tool that integrates mappings from multiple sources into a graph data structure. Using graph algorithms, it infers missing mappings implied by available ones while keeping track of provenance and confidence. This allows connecting identifier spaces between which direct mapping was previously not possible. SeMRA is customizable and takes a declarative specification as input describing sources to integrate with additional configuration parameters. We make available an aggregated mappings resource produced by SeMRA consisting of 43.4 million mappings from 127 sources that jointly cover identifiers from 445 ontologies and databases. We also describe benchmarks on specific use cases such as integrating mappings between resources cataloging diseases or cell types. Availability: The code is available under the MIT license at https://github.com/biopragmatics/semra. The mappings database assembled by SeMRA is available at https://zenodo.org/records/15208251.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.16.649126v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">scTrimClust: A Fast Approach to Robust scRNA-seq Analysis Using Trimmed Cell Clusters</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Detection of marker genes and other data analyses in single cell RNA-sequencing (scRNA-seq) experiments very much rely on the result of unsupervised clustering of cells. However, in 2-dimensional representation of clustering results, several cells appear as outliers or in the border area of a cluster suggesting that these cells may not adequately represent a particular cell type. We propose a novel and fast approach, scTrimClust, for identifying cells that may be interpreted of extreme specimens of their cell type. Identification is based on concave hulls build around each 2-dimensional cell cluster and the distance of each cell to the border area of its population. We study in two data examples, how cells with non-representative expression profile can influence the results of the analysis. We found that some sets of marker genes are little influenced by extreme cells while other sets are strongly modified, and must therefore be treated carefully. scTrimClust is also useful to compare the influence of other parameters of an scRNA-seq analysis, e.g. normalization or the clustering approach, on the results. We also provide a software implementation of scTrimClust.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.16.649082v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SeroBA(v2.0) and SeroBAnk: a robust genome-based serotyping scheme and comprehensive atlas of capsular diversity in Streptococcus pneumoniae</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">The unprecedented number of Streptococcus pneumoniae (the pneumococcus) genomes sequenced in recent years has accelerated the discovery of novel serotypes and highlighted the genetic diversity both between and within each serotype. A novel serotype should demonstrate a distinct cps locus, capsular structure, and serological profile. In only the past four years, nine new serotypes have been identified. Accurate and timely serotyping of pneumococcal isolates is key to understanding its global distribution, evolution, and the response of the bacterial population to vaccination. However, current bioinformatics serotyping tools are infrequently updated, and struggle to accommodate the rapid discovery of new serotypes in a timely manner. To address these limitations, we built a comprehensive and curated library (SeroBAnk) encompassing all known pneumococcal serotypes; this resource is presented as an atlas on a dedicated publicly accessible webpage (https://www.pneumogen.net/gps/#/serobank). Building upon this resource, we developed SeroBA(v2.0), a tool with an easy-to-update database that can accurately identify 102 of 107 known pneumococcal serotypes (except for serotypes 24B, 24C, 24F, 7D and 6H) and 18 genetic subtypes within serotypes 6A, 6B, 11A, 19A, 19F and 33F. We validated SeroBA(v2.0) on 26,306 genomes from the Global Pneumococcal Sequencing project, reference isolates and simulated reads derived from the reference genetic sequences of capsular polysaccharide biosynthetic (cps) locus and showed that SeroBA(v2.0) can reliably detect the nine recently discovered serotypes. Additionally, we show that in silico serotypes inferred by SeroBA(v2.0) had high concordance with phenotypic serotypes determined by either Quellung or latex agglutination at the serotype level (88.9%; 15,945/17,933), and at the serogroup level (91.9%; 16,480/17,933). Finally, we propose a community-contribution based approach to ensure that SeroBA(v2.0) is maintained and updated as novel serotypes continue to be discovered. The global community can submit putative novel serotypes through our public repository on GitHub (https://github.com/GlobalPneumoSeq/seroba/issues). The submitted putative novel serotypes will be curated based on the genetic sequence of cps region, capsular structure and serological profile by people of relevant expertise in the field. SeroBA(v2.0) can be accessed at https://github.com/GlobalPneumoSeq/seroba.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.16.648953v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Ridge Redundancy Analysis for High-Dimensional Omics Data</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Redundancy Analysis (RDA) is a popular multivariate approach for modeling the relationships between two sets of variables. In omics studies, RDA provides a flexible framework for investigating associations between high-dimensional molecular data. However, omics datasets often exhibit strong multicollinearity and suffer from the "large p, small n" problem, where the number of predictor variables p (features) exceeds the number of samples n. Ridge RDA addresses multicollinearity by introducing a ridge penalty in addition to the rank restriction. However, its application to high-dimensional omics data remains challenging due to (i) difficulties in manipulating the large coefficient matrix of dimensions p x q, where q represents the number of response variables, and (ii) the need to select the rank and the regularization parameter jointly as the two of them influence the performance of the method. We propose an efficient computational framework for ridge RDA that overcomes these challenges by leveraging the Singular Value Decomposition (SVD) of the predictor matrix X. This approach eliminates the need for direct covariance matrix inversion, improving computational efficiency. Furthermore, we introduce a novel strategy that reduces the memory burden associated with the storage of the coefficient matrix from pq to (p + q)r, with r the chosen reduced rank dimensionality. Our method enables an efficient grid search to jointly select the ridge penalty {lambda} and r via cross-validation. The proposed framework is implemented in the R package rrda, providing a practical and scalable solution for the analysis of high-dimensional omics data. The source code for the R package rrda, along with the data used for implementation and analysis, is available on GitHub: https://github.com/Yoska393/rrda</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.16.649138v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Outcrossing Complicates Mutation Purging by Trapping Single Nucleotide Polymorphisms in Structural Variant Mutations</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Outcrossing influences the ability of a populations to purge deleterious mutations, but its impact on larger structural genomic changes like insertions, deletions, and inversions remains unclear. To test this, we developed a theoretical model of structural variant (SV) mutation dynamics, implemented it in population simulations, and tested our predictions by exposing three C. elegans strains to ethyl methanesulfonate and formaldehyde to induce diverse mutations. Contrary to expectations for single nucleotide polymorphisms (SNPs), our model and whole-genome sequencing revealed that outcrossing could retain SVs in populations even after recovery from mutagenesis. The strain with highest levels of outcrossing accumulated the largest number of SVs, including megabase-scale rearrangements, and showed the highest transposable element activity. Moreover, SNPs persisted when embedded within retained SVs. These findings contradict classical population genetic expectations, showing that outcrossing can hinder SV removal and complicate mutation purging by trapping SNPs within SVs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.16.649179v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ANARCII: A Generalised Language Model for Antigen Receptor Numbering</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Antigen receptor numbering allows the rapid delineation of the antigen-binding regions of antibody and T cell receptor (TCR) sequences, from sequence alone. It also allows the comparison of the vast diversity of antigen receptors in a consistent frame of reference. Numbering of antigen receptors is currently achieved by aligning sequences to a reference set. This approach may result in different numbering, depending on the reference set used or may fail to number query sequences derived from new species or rare sequence types. To address this problem, we have built a new numbering method (ANARCII) which requires no alignment step and is based on a Seq2Seq language model. Our results show that ANARCII can deal with the complexity that arises in experimentally collected sequencing data and generalise to sequences which are highly dissimilar to those in training. In test sets designed to contain challenging and ambiguous sequence patterns ANARCII numbering was identical to existing methods for over 99.99% of conserved residues and over 99.94% for complete CDR regions. The lightweight architecture allows numbering of over 90,000 sequences per minute on a single A100 GPU. Furthermore, the ANARCII package can be conditioned to fit rare sequence types and provide new training data for fine-tuning. We demonstrate that fine-tuned versions of ANARCII can correctly number other immunoglobulin domains such as TCRs and VNARs. Our model is freely available as a web tool (https://github.com/oxpig/ANARCII), as well as a package for high throughput numbering of next generation sequencing data (https://opig.stats.ox.ac.uk/webapps/sabdab-sabpred/sabpred/anarcii/).</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.16.648720v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Efficient trace reconstruction in DNA storage systems using Bidirectional Beam Search</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Motivation: As DNA data storage systems gain popularity, the need for an efficient trace reconstruction algorithm becomes increasingly important. These algorithms aim to reconstruct the original encoded sequence from its noisy sequenced copies (or "traces"), enabling a faster and more reliable decoding process. Previous works have often been adaptations of methods for multiple sequence alignment or read error correction, typically operating under strict assumptions such as fixed error rates. However, such methods demonstrate limited generalizability to real datasets with higher error rates and suffer from slow processing times when dealing with a large number of traces. Results: We propose a new probabilistic formulation of the trace reconstruction problem. Instead of optimizing alignment among traces, we model the traces as observations of a k-th order Markov chain and try to predict the sequence that is generated by the Markov chain with the highest probability. Such a formulation inspires a novel solution, i.e. Bidirectional Beam Search (BBS), whose reconstruction phase operates in linear time with respect to the length of the encoded sequences. Experiments on multiple in-house and public Nanopore datasets demonstrate that BBS achieves top-tier accuracy compared with the state-of-the-art methods while being ~20x faster, showing its potential to enhance the efficiency of DNA data storage systems. Availability and Implementation: The implementation of BBS is available at https://github.com/GZHoffie/bbs, and the dataset and scripts for reproducibility are available at https://github.com/GZHoffie/bbs-test.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.16.644694v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The oral cavity of chronically homeless adults is home for unusual extremophile environmental bacteria</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Chronically homeless adults (CHA) often face limited healthcare access, poor nutrition, frequent substance use, and close contact with stray animals. These factors can significantly alter their oral microenvironment. This study aimed to characterize the oral microbiota of this vulnerable and understudied population. We analyzed saliva samples from 60 chronically homeless men and 40 asymptomatic men with no history of homelessness, all living in Mexico City. DNA was extracted, and the V1-V3 regions of the 16S rRNA gene were sequenced. Taxonomic classification was performed using human and non-human databases. Diversity metrics were calculated using the vegan v2.6-4 package in R, and informative species were identified through machine learning and statistical approaches. Microbial correlation networks were inferred using NetCoMi in RStudio. Bacterial diversity was significantly higher in the CHA group. While both groups shared 369 species, only eight were exclusive to the control group, and 149 were unique to CHA. Several of these taxa had never been reported in humans and are typically found in extreme environments. For example, Megasphaera cerevisiae, adapted to high ethanol and low pH, was the most abundant species. Syntrophocurvum alkaliphilum, from a hypersaline lake in Siberia, and Sinanaerobacter chloroacetimidivorans, from anaerobic bioreactors, were also prevalent. Marked differences in microbial network structure were observed between groups. These findings highlight the adaptability of extremophile bacteria to human environments under chronic stress and poor hygiene. This study provides a first look into the oral microbiota of individuals experiencing chronic homelessness, emphasizing the need for more research into marginalized populations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.16.649188v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Cell-type-specific transposable element demethylation and TAD remodeling in the aging mouse brain</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Aging is a major risk factor for neurodegenerative diseases, yet underlying epigenetic mechanisms remain unclear. Here, we generated a comprehensive single-nucleus cell atlas of brain aging across multiple brain regions, comprising 132,551 single-cell methylomes and 72,666 joint chromatin conformation-methylome nuclei. Integration with companion transcriptomic and chromatin accessibility data yielded a cross-modality taxonomy of 36 major cell types. We observed that age-related methylation changes were more pronounced in non-neuronal cells. Transposable element methylation alone distinguished age groups, showing cell-type-specific genome-wide demethylation. Chromatin conformation analysis demonstrated age-related increases in TAD boundary strength with enhanced accessibility at CTCF binding sites. Spatial transcriptomics across 895,296 cells revealed regional heterogeneity during aging within identical cell types. Finally, we developed novel deep-learning models that accurately predict age-related gene expression changes using multi-modal epigenetic features, providing mechanistic insights into gene regulation. This dataset advances our understanding of brain aging and offers potential translational applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.21.648266v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Evolution of the cellular landscape in mammalian striatum</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">The dorsal striatum is important for highly specialized functions including movement, learning, and habit formation. However, it is not known if species-specialized behaviors are associated with cellular specializations in the striatum. Here, we compared single-nucleus RNA sequencing (snRNA-seq) data from human, chimpanzee, rhesus macaque, common marmoset, and pale spear-nosed bat caudate (CN) and putamen (Pu) separately as well as mouse caudoputamen (C-Pu), which represents divergence among species spanning approximately 94 million years of evolution. We observed a lower neuron-to-glia ratio in primate striata compared to non-primates, reflecting the allometric scaling of neuron density and relative glia density invariance in larger brains. Among neurons, eccentric spiny projection neurons (eSPNs) - an SPN of unknown function - showed significantly lower proportions in non-primate striata for both CN and Pu. Focusing on the heterogeneity within interneurons, we identified two bat striatal interneuron cell types that are nearly absent in other species: which express LMO3, and co-express FOXP2 and TSHZ2. Other striatal interneurons also showed significantly differential abundance between primates and non-primates. In summary, we provide a comprehensive snRNA-seq dataset of dorsal striatum, identify novel interneuron innovations in bat, and uncover fundamental cellular composition differences between primate and non-primate striata.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.20.649707v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Accelerated evolution of whole gene clusters by an engineered lytic phage system in E. coli</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Directed evolution excels at optimizing individual proteins, but simultaneous evolution of multiple genes remains a significant challenge. Here, we establish a robust phage-based system for accelerated evolution of large gene clusters of up to 39 kilobases. Lytic Selection and Evolution (LySE) selectively replicates and mutagenizes the target gene cluster, carried on a phagemid through alternating cycles of lysis and transduction in Escherichia coli. We engineered a hypermutagenic T7 DNA polymerase (T7 DNAP) that increases mutation rates of the replicating phagemid 7,000-fold during the lytic cycle. We further optimized the mutational spectrum by fusing the T7 DNAP to a dual adenine-cytosine deaminase to install all possible transition mutations at similar frequencies. LySE-mediated transduction enables selection for desired metabolic functions by coupling gene cluster expression to host fitness, while eliminating genomic off-target mutations by refreshing the host in each cycle. Using LySE, we evolved a 25-fold increase in tigecycline resistance in 5 cycles, and a 50.9% increase in end-point biomass of a bacterial strain that utilizes the PET monomer, ethylene glycol, as its sole carbon source.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.21.649713v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

