<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Wednesday, 30 April 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Transcriptomic analysis reveals immune signatures associated with specific cutaneous manifestations of lupus in systemic lupus erythematosus</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lee, E. Y., Patterson, S., Cutts, Z., Lanata, C. M., Dall'Era, M., Yazdany, J., Criswell, L. A., Haemel, A., Katz, P., Ye, C. J., Langelier, C., Sirota, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We map molecular pathways linked to ten distinct SLE-related rashes, highlighting subtype-specific mechanistic targets, and presenting opportunities to identify precision therapies for Sle-associated skin phenotypes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Systemic lupus erythematosus (SLE) presents with diverse and heterogenous cutaneous manifestations. However, the molecular and immunologic pathways driving specific cutaneous manifestations of SLE are poorly understood. Here, we leverage transcriptomics from a large well-phenotyped longitudinal cohort of SLE patients to map molecular pathways linked to ten distinct SLE-related rashes. Through whole blood and immune cell-sorted bulk RNA sequencing, we identified immune signatures specific to cutaneous subtypes of SLE. Subacute cutaneous lupus (SCLE) exhibited broad upregulation of type I interferon, TNF-, and IL6-JAK-STAT3, pathways suggesting potential unique therapeutic responses to JAK and type I interferon inhibition. While interferon signaling is prominent in SCLE, discoid lupus, and acute lupus, it is unexpectedly absent in patients with skin and mucosal ulcers. Pathway and cell-type enrichment analysis revealed unexpected roles for CD14+ monocytes in photosensitivity of SLE and NK cells in alopecia, mucosal ulceration, and livedo reticularis. These findings illuminate the immune heterogeneity of rashes in SLE, highlighting subtype-specific mechanistic targets, and presenting opportunities to identify precision therapies for SLE-associated skin phenotypes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.27.649460v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Diverse Genomic Landscape of Swine Influenza A Virus in England (2014 - 2021)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mollett, B. C., Byrne, A. M. P., Everett, H. E., Reid, S. M., Williamson, S., Anderson, T. K., James, J., Banyard, A. C., Brown, I. H., Lewis, N. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified 24 new H1N1 and H1n2 subtypes from the 1A and 1B human-seasonal lineages from the classical swine and human seasonal viruses.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Surveillance of influenza A viruses in pigs (SwIAV) is critical for identification of novel genetic groups that pose a risk to pig health and might have zoonotic potential. SwIAVs circulating in pigs in England between 2014 and 2021 were characterised using whole genome sequencing (WGS). Haemagglutinin (HA) and neuraminidase (NA) sequencing data from 82 of 368 influenza A positive samples (71 submissions) were determined, identifying H1N1 and H1N2 subtypes from the 1A classical swine and 1B human-seasonal lineages respectively. The 1B lineage viruses were predominant, accounting for 68.29% of sequenced viruses, with 1A lineage viruses comprising 31.71%, primarily from the 1A.3.3.2 clade (2009 H1N1 pandemic origin). This study characterised previously undefined diversity within the 1B lineage which led to the designation of new HA clades 1B.1.1.1, 1B.1.1.2 and 1B.1.1.3. Complete genome data were obtained from 64/82 viruses thereby updating the definition of genetic diversity thresholds and leading to the identification of 24 unique genotypes. All these 64 viruses contained PB2, PB1, PA, NP, MP, and NS gene segments of 2009 H1N1 pandemic origin. These data highlight the increasing divergence of SwIAV within pig populations England and emphasise the requirement for continued genomic surveillance to improve animal health and monitor zoonotic risk.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.28.650978v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comparative Virome Profiling and Discovery of Novel Viruses in Managed and Wild Bees using RNA-seq</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Li-Byarlay, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified multiple known bee pathogens including Black Queen Cell Virus and Deformed Wing Virus in A. mellifera, and discovered three novel insect viruses - two iflaviruses and one entomopoxvirus -associated with C. calcarata.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Insect pollinators face growing challenges from viral pathogens that may contribute to population declines, yet comparative data on virome composition in managed versus wild bees remain limited. Using RNA-Seq-based metatranscriptomic analysis, we compared the virome of Apis mellifera and Ceratina calcarata collected from agricultural and non-agricultural landscapes. We identified multiple known bee pathogens including Black Queen Cell Virus and Deformed Wing Virus in A. mellifera, and discovered three novel insect viruses - two iflaviruses and one entomopoxvirus -associated with C. calcarata. Notably, one novel iflavirus was identified in A. mellifera. Our findings reveal species- and landscape-specific viral prevalence, highlight viral diversity beyond canonical bee pathogens, and provide genomic resources for future functional studies. These data offer new insights into virus-host dynamics and underscore the utility of virome profiling in understanding pollinator health at the molecular level.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.27.650873v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Machine learning: Python tools for studying biomolecules and drug design.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fedor V Ryzhkov, Yuliya E Ryzhkova, Michail N Elinson</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A list of Python scripts, libraries, and ready-made programs for routine tasks in medical chemistry and bioinformatics.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The increasing adoption of computational methods and artificial intelligence in scientific research has led to a growing interest in versatile tools like Python. In the fields of medical chemistry, biochemistry, and bioinformatics, Python has emerged as a key language for tackling complex challenges. It is used to solve various tasks, such as drug discovery, high-throughput and virtual screening, protein and genome analysis, and predicting drug efficacy. This review presents a list of tools for these tasks, including scripts, libraries, and ready-made programs, and serves as a starting point for scientists wishing to apply automation or optimization to routine tasks in medical chemistry and bioinformatics.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40301135/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">UNISOM: Unified Somatic Calling and Machine Learning-based Classification Enhance the Discovery of CHIP.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shulan Tian, Garrett Jenkinson, Alejandro Ferrer, Huihuang Yan, Joel A Morales-Rosado, Kevin L Wang, Terra L Lasho, Benjamin B Yan, Saurabh Baheti, Janet E Olson, Linda B Baughn, Wei Ding, Susan L Slager, Mrinal S Patnaik, Konstantinos N Lazaridis, Eric W Klee</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present UNISOM, a streamlined workflow for enhancing CHIP detection from whole-genome and whole-exome sequencing data that is underpowered, especially for low VAFs.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Clonal hematopoiesis (CH) of indeterminate potential (CHIP), driven by somatic mutations in leukemia-associated genes, confers increased risk of hematologic malignancies, cardiovascular disease, and all-cause mortality. In blood of healthy individuals, small CH clones can expand over time to reach 2% variant allele frequency (VAF), the current threshold for CHIP. Nevertheless, reliable detection of low-VAF CHIP mutations is challenging, often relying on deep targeted sequencing. Here, we present UNISOM, a streamlined workflow for enhancing CHIP detection from whole-genome and whole-exome sequencing data that are underpowered, especially for low VAFs. UNISOM utilizes a meta-caller for variant detection, in couple with machine learning models which classify variants into CHIP, germline, and artifact. In whole-exome data, UNISOM recovered nearly 80% of the CHIP mutations identified via deep targeted sequencing in the same cohort. Applied to whole-genome sequencing data from Mayo Clinic Biobank, it recapitulated the patterns previously established in much larger cohorts, including the most frequently mutated CHIP genes, predominant mutation types and signatures, as well as strong associations of CHIP with age and smoking status. Notably, 30% of the identified CHIP mutations had < 5% VAFs, demonstrating its high sensitivity toward small mutant clones. This workflow is applicable to CHIP screening in population genomic studies. The UNISOM pipeline is freely available at https://github.com/shulanmayo/UNISOM and https://ngdc.cncb.ac.cn/biocode/tool/7816.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40300108/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Effectidor II: A pan-genomic AI-based algorithm for the prediction of type III secretion system effectors.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Naama Wagner, Ella Baumer, Iris Lyubman, Yair Shimony, Noam Bracha, Leonor Martins, Neha Potnis, Jeff H Chang, Doron Teper, Ralf Koebnik, Tal Pupko</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify type 3 effector gene repertoires from multiple genome sequences simultaneously, including the identification of T3Es from multiple sequences simultaneously.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Type III secretion systems are used by many Gram-negative bacteria to inject type 3 effectors (T3Es) directly into eukaryotic cells, promoting disease or provoking immune response. Because of these opposing evolutionary forces, T3E repertoires often vary within taxonomic groups. Identifying the full effector gene repertoire in genomes of related individuals is crucial for determining core and specialized effectors, understanding the disease dynamics, and developing appropriate management strategies against pathogens. It can also help uncover novel T3Es that have recently emerged in a population. Our previously published Effectidor web server successfully addressed the challenge of identifying T3Es in a single bacterial genome. Here, we enriched the web server with various novel capabilities, including the identification of T3Es from multiple genome sequences simultaneously.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40300094/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Natural language processing in drug discovery: bridging the gap between text and therapeutics with artificial intelligence.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Christine Ann Withers, Amina Mardiyyah Rufai, Aravind Venkatesan, Santosh Tirunagari, Sebastian Lobentanzer, Melissa Harrison, Barbara Zdrazil</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> The field of Natural Language Processing (NLP) within the life sciences has exploded in its capacity to aid the extraction and analysis of data from scientific texts in recent years through the advancement of Artificial Intelligence (AI).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The field of Natural Language Processing (NLP) within the life sciences has exploded in its capacity to aid the extraction and analysis of data from scientific texts in recent years through the advancement of Artificial Intelligence (AI). Drug discovery pipelines have been innovated and accelerated by the uptake of AI/Machine Learning (ML) techniques.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40298230/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genomic comparison of highly related pairs of E. coli and K. pneumoniae isolated from faeces and blood of the same neonatal patients hospitalized with fever in Dar es Salaam, Tanzania</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Goodman, R. N., Moyo, S. J., Memelis, I., Khanijau, A., Manyahi, J., Kibwana, U. O., Aboud, S., Blomberg, B., Langeland, N., Roberts, A. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that both E. coli and Klebsiella pneumoniae likely translocated from the GI tract to the blood in multiple cases of BSI.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Blood stream infections (BSIs) are a major cause of hospitalisation and death for children under the age of five in sub-Saharan Africa with members of the Gram-negative bacteria Enterobacterales such as Klebsiella pneumoniae and Escherichia coli among the most common causative agents. These bacteria usually colonise the human gastrointestinal (GI) tract which has been identified as a reservoir for invasive infections into extra-intestinal environments such as the urinary tract and bloodstream. In this study we used comparative genomics to compare hybrid genome assemblies of blood and faecal isolates taken from the same patients (all neonates under 19 days old) to determine if the BSI associated bacterial isolates originated in their GI tract. We show that both E. coli and K. pneumoniae likely translocated from the GI tract to the blood in multiple cases of BSI. We also highlight key virulence genes and acquired mutations that are indicative of pathogenic strains capable of BSI. These findings expand our understanding of BSI pathogenesis and could help guide targeted interventions to prevent future BSI infections in neonates.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.28.650962v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MYC and AP-1 oncogenes synergistically bind enhancers to rewire transcription</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kalyan Sundaram, R., Radhakrishnan, R., Lim, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We found that cancer-specific co-enrichment of the TRE motifs (AP-1 binding sites), alongside MYC's canonical EBOX motifs, is associated with transcriptional repression in cancers.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The transcription factor c-MYC (MYC) is deregulated in about 70% of human cancers. Through de novo motif discovery analysis on published MYC ChIP-seq datasets from cancer cell lines, we found cell-type specific co-enrichment of the TRE motifs (AP-1 binding sites), alongside MYC's canonical EBOX motif. TRE motifs serve as indirect MYC binding sites in synergy with AP-1, predominantly at enhancers rather than promoters. At elevated MYC levels, as seen in cancers, MYC preferentially occupies TRE sites over EBOX sites at enhancers. Integration of ChIP-seq and RNA-seq data revealed TRE enhancer binding sites are frequently associated with MYC-mediated transcriptional repression. Gene Ontology analysis showed that MYC utilizes TRE sites to transcriptionally rewire cells, modulating cancer hallmarks like proliferation, apoptosis, and cell adhesion. These molecular insights into how increased MYC levels alter its DNA binding preference and gene regulation could inform new therapeutic strategies targeting cancer-specific MYC functions and its co-regulators.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.28.650480v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Discovery of a tRNA-regulatory transcription factor that suppresses breast cancer metastasis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chen, S., Markett, D., Karimzadeh, M., Luo, Y., Khoroshkin, M. S., Boyraz, B., Lee, S., Carpenter, C., Nguyen, P., Garcia, K., Joshi, T., Martin, C., Hanisch, B., Molina, H., Tavazoie, S., Ramani, V., Navickas, A., Goodarzi, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified Zinc Finger ZZ-Type And EF-Hand Domain Containing 1 (ZZEF1) as the first sequence-specific transcription factor of a particular tRNA.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Transfer RNAs (tRNAs), once viewed as static adaptors in translation, are now recognized as dynamic regulators of gene expression. While recent studies have illuminated roles for tRNA stability, the upstream mechanisms governing tRNA transcription remain poorly understood. To address this gap, we generated the EXpression atlas of tRNA (EXTRNA), a high-resolution tRNA expression dataset spanning 24 cell lines across 9 human tissues. EXTRNA revealed both tissue-type-specific expression programs ("tRNAomes") and unexpected intra-tissue heterogeneity across breast cancer samples. Integrating EXTRNA with computational network analysis and data from other publicly available datasets, we identified Zinc Finger ZZ-Type And EF-Hand Domain Containing 1 (ZZEF1) as the first sequence-specific transcription factor of a particular tRNA. ZZEF1 promoted tRNA-LysUUU transcription by partnering with the ATP-dependent chromatin remodeler Chromodomain Helicase DNA Binding Protein 6 (CHD6), enhancing chromatin accessibility at tRNA-Lys-TTT-3 loci. ZZEF1 deficiency reduced tRNA-LysUUU abundance, decreased the translational efficiency of AAR codon-enriched mRNAs-including the tumor suppressor Serine/Threonine Kinase 3 (STK3)-and promoted metastatic progression in breast cancer in vivo. Together, our findings establish a previously unrecognized mechanism for RNA polymerase III-mediated tRNA transcription and define a regulatory circuit linking chromatin remodeling, codon-specific translation, and tumor suppression. More broadly, this work introduces a framework for dissecting the regulatory logic of the tRNAome and highlights tRNA expression control as a promising avenue for therapeutic intervention.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.26.650725v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Extrachromosomal DNA Gives Cancer a New Evolutionary Pathway</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, Y., Cope, O., Chen, J., Mehta, A., Fleifel, D., Ford, C., Benhamie, P., Haase, S., Gulec, S., Elston, T., Spanheimer, P. M., Tomblin, C. A., Rojas, A., Tate, T., Purvis, J., Wang, J. R., Dahl, J. M., Wolff, S. C., Cook, J., Brunk, E. C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that graded variation in ecDNA copy numbers across cells enhances the adaptability of the entire tumor cell population, and show that this population-based adaptability makes ecDNA-driven cancers exceptionally resilient.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Studying tumor evolution has traditionally focused on individual cells gaining fitness advantages to outcompete their neighbors. However, this perspective may miss a crucial aspect of tumor resilience, particularly in cancers driven by extrachromosomal DNA (ecDNA). Rather than evolving at the individual level, ecDNA-driven cancers may operate on a population scale, where graded variation in oncogene copy numbers across cells enhances the adaptability of the entire tumor cell population. Leveraging single cell multi-omics sequencing and machine learning, we demonstrate that cell-to-cell variation in ecDNA copy number scales directly with functional phenotypes, including gene expression, chromatin accessibility, protein levels, cell cycle distributions and growth. For the first time, we show that when this graded variation is disrupted, cells rapidly restore it, far faster than random ecDNA segregation would allow. Time-course experiments and computational simulations suggest that the inheritance of ecDNAs into daughter cells is governed by active, deterministic processes. Our findings position ecDNA as a key driver of population-level adaptability, challenging the view of ecDNA as a mere byproduct of genomic instability. Instead, ecDNA forms the foundation of selection processes that enhance the tumor's ability to evolve and survive in dynamic environments. This population-based adaptability makes ecDNA-driven cancers exceptionally resilient, offering new insights into their persistence and therapeutic resistance.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.26.650733v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Telomerase RNA gene duplications drive telomeric repeatdiversity and evolution in Andrena bees</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Klapproth, C., Israel, E., Ahmed, A., Remmel, A., Kaether, K.-K., Reinhardt, F., Chen, J. J. L., Prohaska, S. J., Steidle, J., Lemke, S., Stadler, P. F., Findeiss, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show evidence for the existence of multiple TR paralogs that diverged during the evolution of the genus Andrena and propose an evolutionary mechanism for diversification of telomeric repeat sequences in certain insect species through telomerase RNA gene duplication.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Most organisms in the animal kingdom require a non-coding telomerease RNA (TR) in conjunction with the telomerase reverse transcriptase (TERT) to add telomere tandem repeats to chromosome ends to genomic instability. Recent studies reported an extensive diversity in the sequence of telomeric repeats in some insect species. Our investigation of TR genes in the Andrena genus provides convincing evidence for the presence of multiple TR gene copies with different template sequences for synthesis of distinct telomeric repeat sequences in several species. In this study we describe the structure, genomic coordinates and abundance of these TR genes, and correlate our findings with the levels of tandem repeats found in DNAseq data. Based on an analysis of the synthenic context of these newly predicted TR genes, we show evidence for the existence of multiple TR paralogs that diverged during the evolution of the genus Andrena. To our knowledge this is the first time such a phenomenon is observed in animals, although recently reported for plants. Interestingly, the comprehensive annotation of all TERT genes found in yet unannotated Andrena species shows no corresponding evolutionary changes in related TERT proteins encoded by a single copy gene. Our study suggests an evolutionary mechanism for diversification of telomeric repeat sequences in certain insect species through telomerase RNA gene duplication.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.29.651168v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Demographic history explains mitochondrial genome capture and mitonuclear discordance among South American collared Tropidurus	 lizards</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Salles, M. M. A., Carvalho, A. L. G., Leache, A. D., Martinez, N., Bauer, F., Motte, M., Espinola, V., Rodrigues, M. T., Piantoni, C., Pie, M. R., Olivotto, A., Colli, G. R., Choueri, E. L., Werneck, F. P., Domingos, F. M. C. B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify a complex evolutionary history of mitonuclear discordance in Tropidurus spinulosus, a group of lizards in South America, and show that it is most consistent with neutral demographic processes, coupled with ancient mitochondrial introgression, as the main factors underlying the mismatch between nuclear</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mitonuclear discordance-evolutionary discrepancies between mitochondrial and nuclear DNA phylogenies-can arise from various factors, including introgression, incomplete lineage sorting, recent or ancient demographic fluctuations, sex-biased dispersal asymmetries, among others. Understanding this phenomenon is crucial for accurately reconstructing evolutionary histories, as failing to account for discordance can lead to misinterpretations of species boundaries, phylogenetic relationships, and historical biogeographic patterns. In this study, we investigate the evolutionary drivers of mitonuclear discordance in the Tropidurus spinulosus species group, which contains nine species of lizards inhabiting open tropical and subtropical environments in South America. Using a combination of population genetic and phylogenomic approaches, applied to mitochondrial and nuclear data, we identified different instances of gene flow that occurred in ancestral lineages of extant species. Our results point to a complex evolutionary history marked by prolonged isolation between species, demographic fluctuations, and potential episodes of secondary contact with genetic admixture. These conditions likely facilitated mitochondrial genome capture while diluting signals of nuclear introgression. Furthermore, we found no strong evidence supporting incomplete lineage sorting or natural selection as primary drivers of the observed mitonuclear discordance. Therefore, the unveiled patterns are most consistent with neutral demographic processes, coupled with ancient mitochondrial introgression, as the main factors underlying the mismatch between nuclear and mitochondrial phylogenies in this system. Future research could further explore the role of other demographic processes, such as asymmetric sex-biased dispersal, in shaping these complex evolutionary patterns.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.25.650633v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Anchors for Homology-Based Scaffolding</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kaether, K.-K., Lemke, S., Stadler, P. F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a protoypical pipeline based on the anchor-based synteny detection tool AncST and show that it produces scaffolds with accuracies comparable to competing tools at acceptable computational costs.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Homology-based scaffolding is the task of ordering contigs based on their co-linearity with the DNA sequence of other, closely re- lated reference genomes. We investigate here how a recent approach to identify synteny anchors can be adapted to this task. The advantage of using anchors is that "sufficiently unique" anchor candidates can be precomputed independently for each genome assembly, including contig- level drafts. This allows an efficient identification of co-linear runs of anchor matches between a contig and a reference genome and, conse- quently, also order their contigs in relation to the reference. Such data can be improved by aggregating scaffolding results for multiple reference genomes. We present a protoypical pipeline based on the anchor-based synteny detection tool AncST and show that it produces scaffolds with accuracies comparable to competing tools at acceptable computational costs. Assuming that anchor candidates have been precomputed for the genomes of interest, the remaining computational efforts are on par with the fastest methods.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.28.650980v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CertPrime: a new oligonucleotide design tool for gene synthesis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Luna-Cerralbo, D., Serrano, A., Blasco-Machin, I., Hamdan, F., Martinez-Olivan, J., Broset, E., Bruscolini, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed CertPrime, an innovative tool designed for scalable and efficient handling of long DNA sequences, which reduces the occurrence of non-specific bands and leads to efficient gene assembly.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The design of oligonucleotides with uniform hybridisation temperatures is essential for successful gene synthesis. However, current computational tools for oligonucleotide design face significant limitations, including difficulties in processing long DNA sequences, poor adaptability to specific experimental conditions, limited control over oligonucleotide length, and challenges in minimising spurious dimer formation. To address these issues, we developed CertPrime, an innovative tool designed for scalable and efficient handling of long DNA sequences. CertPrime enables precise customisation of experimental parameters, provides flexibility to limit the maximum oligonucleotide length, and generates designs with reduced deviations in melting temperatures across overlapping regions compared to existing tools. We experimentally compared CertPrime designs with benchmark design methods for a complex DNA sequence and found that CertPrime design led to more efficient gene assembly, significantly reducing the occurrence of non-specific bands. These results make CertPrime a powerful and versatile tool for oligonucleotide design in gene synthesis applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.29.650686v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Controlling the taxonomic composition of biological information storage in 16S ribosomal RNA</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gamas, K. R., Seamons, T., Dysart, M., Fang, L., Chappell, J., Stadler, L., Silberg, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show how a program called Ribodesigner can be used to identify cat-RNA designs that enable kingdom-wide barcoding, or selective barcoded of specific taxonomic groups within a kingdom.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Microbes can be programmed to record participation in gene transfer by coding biological-recording devices into mobile DNA. Upon DNA uptake, these devices transcribe a catalytic RNA (cat-RNA) that binds to conserved sequences within ribosomal RNA (rRNA) and perform a trans-splicing reaction that adds a barcode to the rRNA. Existing cat-RNA designs were manually generated to be broad-host range, providing no control over the organisms that were barcoded. To achieve control over the organisms barcoded by cat-RNA, we created a program called Ribodesigner that uses input sets of rRNA sequences to create designs with varying specificities. We show how this algorithm can be used to identify designs that enable kingdom-wide barcoding, or selective barcoding of specific taxonomic groups within a kingdom. We use Ribodesigner to create cat-RNA designs that target Pseudomonadales while avoiding Enterobacterales, and we compare the performance of one design to a cat-RNA that was previously found to be broad host range. When conjugated into a mixture of Escherichia coli and Pseudomonas putida, the new design presents increased selectivity compared to a broad host range cat-RNA. Ribodesigner is expected to aid in developing cat-RNA that store information within user-defined sets of microbes in environmental communities for gene transfer studies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.29.651329v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identification of the Best Filter-based Feature Selection Techniques for Microarray Datasets</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Islam, A., Bhadra, T., Mali, K., Giri, J., Aurangzeb, K., Budhathoki, R. K., Mallik, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> This article basically explores the impact of univariate & multivariate filter-based feature selection methodologies on enhancing the classification performance for the real-life classification problems.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> This article basically explores the impact of univariate & multivariate filter-based feature selection methodologies on enhancing the classification performance for the real-life classification problems. Our study considers two univariate filter-based feature selection techniques, namely, Chi-square and Fisher score, as well as two multivariate filter-based feature selection techniques, viz., Symmetrical Uncertainty and Minimum Redundancy-Maximum Relevance(mRMR). These methods are applied to feature selection from Five diverse collections of datasets, including datasets related to Mixed-lineage Leukaemia (MLL), Lung Cancer, Ovarian Cancer, Central Nervous System (CNS), and Colon Cancer. For each feature, fitness values are calculated using the four aforementioned feature selection methods. After that, a stratified 10-fold cross-validation procedure is conducted using Support Vector Machines (SVM) and Multilayer Perceptrons (MLP) to determine the classification accuracy for each feature. A set of five microarray datasets was used in this evaluation in order to assess the effectiveness of the filter methods. The results of this study represent the first comprehensive analysis and comparison of gene expression datasets filtered using a variety of ranking strategies. Among these approaches, entropy-based methods (e.g., mRMR) emerge as the most effective. The mRMR method demonstrates Outstanding performance outcomes of accuracy, F1-score, and Root Mean Square Error (RMSE). When comparing classifier performance, the F1-score, which combines precision and recall, is particularly useful, while the RMSE measures prediction accuracy. Chi-square, Fisher Score, and Symmetrical Uncertainty (SU) follow as the second, third, and fourth best approaches, respectively. Although the SVM classifier demonstrates superior performance, the difference in accuracy between SVM and the MLP classifier is marginal.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.25.650719v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Expression and immunological role of FUNDC2 in pan-cancer.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Qiu X, Wang S, Li C, Wang Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #fedbf2; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, pan-genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We investigate the possible role and mechanism of FUNDC2 in pan-cancer and the relationship between level of the protein and prognosis of the patients with pan- cancer.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> FUNDC2 is a novel mitochondrial protein and is highly involved in various cancers. However, expression pattern and possible role and mechanism of FUNDC2 in pan-cancer remain to be investigated. TIMER 2.0 was used to investigate the expression patterns and immune infiltration of FUNDC2. GEPIA was applied to study the relationship between level of FUNDC2 and prognosis of the patients with pan-cancer. STRING was employed to analyze the potential interacting proteins of FUNDC2. The phosphorylation sites were predicted by cBioPortal and PhosphoNet. Furthermore, variations of FUNDC2 in cancers were investigated by cBioPortal. Finally, AlphaFold was used to predict the structure of FUNDC2. The data show that there were significant differences in the expression levels of FUNDC2 between cancer tissues and controls. Specifically, the levels of FUNDC2 in 8 cancers were significantly lower than the respective controls. The survival time of the cancer patients with higher levels of FUNDC2 was longer than that of lower FUNDC2 in most different types of cancers. The pattern of FUNDC2 was significantly related to immune infiltration of B cells of cancer patients. STRING analysis revealed that FUNDC2 can interact with FUNDC1, et al. Fifteen phosphorylation sites were predicted by PhosphoNet and cBioPortal, of which the S167 also overlapped with the mutation sites of FUNDC2. These data collectively show that the mitochondrial protein FUNDC2 may serve as a possible prognostic biomarker across various cancers and the mechanism may include immune infiltration.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40294153/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Are you with me? Co-occurrence tests from community ecology can identify positive and negative epistasis between inversions in Mimulus guttatus.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Madrigal-Roca LJ, Kelly JK.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #fedbf2; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, pan-genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show how co-occurrence metrics used in community ecology can provide additional insight regarding coupling and repulsion of inversions at genotypic level.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Chromosomal inversions are structural genetic variants that can play a crucial role in adaptive evolution and speciation. Patterns of attraction and repulsion among unlinked inversions - whether they tend to be carried by the same or different individuals- can indicate how selection is acting on these polymorphisms. In this study, we compare analytical techniques using data from 64 inversions that segregate among 1373 F2 plants of the yellow monkeyflower Mimulus guttatus. Mendelian assortment provides a strong null hypothesis for [Formula: see text] contingency tests. Here, we show how co-occurrence metrics used in community ecology can provide additional insight regarding coupling and repulsion of inversions at genotypic level. The centered Jaccard/Tanimoto index and the affinity score describe the specific way that inversions interact to generate epistasis for plant survival. We further explore the use of network analysis to visualize inversion interactions and to identify essential third and fourth order interactions, which expand the traditional pairwise scope of the co-occurrence metrics. We suggest that a combination of different statistical approaches will provide the most complete characterization of the fitness effects, both for inversions and other polymorphisms essential to adaptation and speciation.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40294049/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Versatile Marker-Free Genome Engineering Platform to Overcome Homologous Recombination Bias in Microbes: A Case Study in Synechococcus elongatus UTEX 2973</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Li, S., Sun, T., Liu, D., Zhang, T., Chen, L., Zhang, W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a new platform for single crossover-free DNA delivery and a new CRISPR/Cpf1-based platform for the sucrose biosynthetic pathway .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The single crossover occurring via homologous recombination is a common phenomenon exited among most microbes like Escherichia, Clostridium, Streptococcus, Lactobacillus, and cyanobacteria, threatening the stability of engineered strains. Among them, fast-growing cyanobacterium Synechococcus elongatus UTEX 2973 (Syn2973) is an attractive photosynthetic chassis for CO2 bioconversion. To address the challenge of homologous recombination, we constructed two endogenous plasmid-based shuttle vectors, pSES and pSEL, enabling stable DNA delivery without reliance on homologous recombination. In parallel, two robust counter-selection systems were developed based on sepT2 and rpsL, which, in combination with positive selection markers, significantly improved the screening efficiency of double-crossover mutants. Building upon these tools, we established three marker-free platforms: (i) T4CROSS, which employs two plasmids and four rounds of single crossover; (ii) TRIPLEARM, which uses a single plasmid containing three homologous arms for three rounds of single crossover; and (iii) CRISPRARM, which integrates CRISPR/Cpf1-mediated genome editing with homologous recombination. All three methods successfully repaired mutations in the pilMNOQ pilus gene cluster in Syn2973, restoring natural competence with high efficiency and positive selection rates. As proof of concept, we employed the CRISPRARM platform for a three-step sequential engineering of the sucrose biosynthetic pathway. The final engineered strain produced 7.12 g*L-1 of sucrose within four days.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.27.650839v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Synthetic Cytosolic Splicing Enables Programmable mRNA-Encoded Receptors</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lampis, M., Benenson, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We design a family of synthetic mRNA splicing (SMS) receptors that sense diverse intracellular and extracellular inputs and produce an anti-inflammatory output in response.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> A typical cell therapy product comprises engineered cells that detect disease-related molecular cues in their surrounding or on a surface of a target cell, and activate a response that alleviates disease symptoms or eradicates diseased cells. mRNA as a therapeutic substrate has become prevalent in the last decade across multiple therapeutic areas, and it has also been evaluated as a building block of cell therapies. However, compared to DNA-based building blocks, it is much more challenging to use mRNA in a programmable manner to engineer complex multi-input/multi-output processes that can fully support the next generation of cell and gene therapies. Addressing this challenge requires the exploration of novel post-transcriptional control mechanisms that bridge mRNA regulation with extracellular surroundings. Here, we engineer a family of synthetic mRNA splicing (SMS) receptors by redesigning the Inositol-requiring enzyme 1 (IRE1) to regulate protein synthesis from a precursor mRNA. We design SMS-based receptors that sense diverse intracellular and extracellular inputs, highlighting the versatility and modularity of this platform. We apply this approach to design a 'cytokine-converter' receptor that detects inflammatory cytokines and produces an anti-inflammatory output in response. That receptor is successfully validated in cell lines and primary T cells upon mRNA delivery. These cells generate anti-inflammatory IL-10 upon stimulation by physiological levels of either TNF- or IL-1{beta} secreted by macrophage-like cells, highlighting their potential as a cell therapy for inflammatory diseases. With its modular and programmable architecture, the SMS platform is poised to become an important enabling tool for sophisticated programmable mRNA therapeutics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.27.650769v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Network-Based Kidney Allocation Simulation: Evaluating Organ Matching Strategies in Variable Hospital Networks</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ananthanarayanan, A., Hu, B., Sha, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a modular network-based kidney allocation simulation designed to function as a standardized benchmark to evaluate organ matching strategies in more realistic hospital networks.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Kidney allocation is a complex and high-stakes process shaped by various factors. Although existing simulations have improved our understanding of transplant policies, they often stem from the perspectives of medical urgency and biological compatibility. Hence, they often fail to account for the geographic and structural variability of hospital networks that heavily influence organ distribution. In this paper, we present a modular network-based kidney allocation simulation designed to function as a standardized benchmark to evaluate organ matching strategies in more realistic hospital networks. Our simulation models hospitals as nodes in a dynamic network that incorporates detailed donor-recipient compatibility criteria and provides an intuitive interface for testing custom matching policies. Then, through two case studies, we demonstrate how varying network structures and matching strategies affect patient outcomes, quantified through transplant success rates and the percentage of positive patient outcomes. Through these case studies, we derive results that closely resemble real-world observations and identify clear biological explanations for the observed trends, aligning with established empirical knowledge. Our results highlight the importance of geography-/network-aware benchmarks and allocation strategies and provide a standardized platform for future research on optimizing kidney distribution in healthcare networks.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.22.650043v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

