<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Sunday, 24 August 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Jaccard Index Network Analysis for pangenome analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Penil-Celis, A., Redondo-Salvo, S., Tagg, K. A., Webb, H. E. E., Garcillan-Barcia, M. P., de la Cruz, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Jaccard Index Network Analysis (JINA) is a comprehensive workflow designed to explore bacterial genome relationships through an integrated network-based approach.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Jaccard Index Network Analysis (JINA) is a comprehensive workflow designed to explore bacterial genome relationships through an integrated network-based approach. This workflow combines existing tools such as Jaccard Index (1), Gephi (2) and Pangraph (3). By integrating these methodologies into a unified framework, JINA enables efficient visualization and stratification of genomic data, facilitating the identification of meaningful patterns, groups, and associations within bacterial populations. The use of JINA ensures precision in capturing genomic variation including single nucleotide polymorphisms, insertions and deletions. While JINA does not implement Gephi, BLAST, and PanGraph directly in a single software, it guides their coordinated use to analyze and interpret genomic data effectively.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.20.669834v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Polymer-derived distance penalties improve chromatin interaction predictions from single-cell data across crop genomes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Schlegel, L., Gomez-Cano, F., Marand, A. P., Johannes, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a penalty function grounded in polymer physics, derived by fitting a multi-component power-law model to experimental Hi-C data from maize, rice, and soybean, to correct for long-range chromatin contacts.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Scalable proxies of 3D genome interactions, such as from single-cell co-accessibility or Deep Learning, systematically overestimate long-range chromatin contacts. To correct this bias, we introduce a penalty function grounded in polymer physics, derived by fitting a multi-component power-law model to experimental Hi-C data from maize, rice, and soybean. This correction substantially improves concordance with Hi-C, reduces false-positive rates of long-range interactions by up to 95%, and reveals distinct decay exponents corresponding to different scales of chromatin organization. We provide open-source code and derived parameters to facilitate broad application across plant species.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.20.671329v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PathPCNet: Pathway Principal Component-Based Interpretable Framework for Drug Sensitivity Prediction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Adhikari, B., Sobhan, M., Sutradhar, A., Narasimhan, G., Mondal, A. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A deep learning framework for drug response prediction and drug sensitivity discovery .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Precision medicine aims to identify significant biomarkers and effective drugs tailored to individual genomic profiles, thereby enabling personalized treatment strategies. Drug efficacy is often attributed to drug response, commonly measured as the concentration of a drug required to inhibit a biological activity. In contrast, drug sensitivity reflects how strongly a tumor responds to a drug, where a lower effective dose indicates higher sensitivity. Machine learning-based drug response prediction has the potential to accelerate biomarker discovery and facilitate the development of more effective therapeutics. In this study, we present PathPCNet, a novel interpretable deep learning framework that integrates multi-omics data (copy number variation, mutation, and RNA sequencing) fused with biological pathways, drug molecular structure, and Principal Component Analysis for drug response prediction. Our model achieves a Pearson correlation coefficient of 0.941 and an R-squared of 0.885, outperforming the existing pathway-based approaches. We employ SHAP-based model interpretation to quantify the contributions of omics and drug features, uncovering key pathways and gene-drug interactions associated with resistance mechanisms. These results demonstrate the utility of integrative deep learning models not only for accurate prediction but also for generating biologically meaningful insights, which can advance drug discovery and precision oncology. In addition, the framework also facilitates the identification of important pathways, genes, and atomic attributes of drugs related to drug sensitivity and different cancer types.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.20.668802v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">FluxRETAP: A REaction TArget Prioritization G e nome-Scale Modeling Technique for Selecting Genetic Targets.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jeffrey J Czajka, Joonhoon Kim, Yinjie J Tang, Kyle R Pomraning, Aindrila Mukhopadhyay, Hector Garcia Martin</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We use machine learning to identify genetic engineering targets that effectively guide the metabolic engineering process, but current technical limitations prevent the independent use of ML approaches without previous biological knowledge.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Metabolic engineering is rapidly evolving as a result of new advances in synthetic biology tools and automation platforms that enable high throughput strain construction, as well as the development of machine learning tools (ML) for biology. However, selecting genetic engineering targets that effectively guide the metabolic engineering process is still challenging. ML can provide predictive power for synthetic biology, but current technical limitations prevent the independent use of ML approaches without previous biological knowledge.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40848245/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Optimizing effect sizes and specificity trumps machine learning when building DNA methylation reference panels for cell-type deconvolution</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Guo, X., Teschendorff, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that a DNA methylation reference panel built from single-cell DNAm data is preferable to one based on optimizing effect size and cell type specificity, because, owing to the relatively small number of sorted samples used in building panels, the latter approach is preferable.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurate cell-type deconvolution is critical for correct interpretation of Epigenome-Wide Association Studies. Such cell-type deconvolution involves estimating underlying cell-type fractions in a sample, which is accomplished using a DNA methylation reference panel built from sorted or single-cell DNAm data. Two competing approaches have emerged to build such reference panels, one which uses machine-learning, and another based on optimizing effect size and cell type specificity. Here we demonstrate that the latter approach is preferable, because, owing to the relatively small number of sorted samples used in building panels, standard machine learning does not optimize effect size and cell-type specificity, causing the model to overfit and underperform when tested in independent data. Furthermore, adult blood panels built from cell-type specific hypomethylated markers improves inference when compared to panels built from hypermethylated ones. These insights provide important guidelines for optimizing the construction of future DNAm reference panels. To aid this task, we have added a function for building an optimized DNAm reference panel to our EpiDISH R package.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.20.671293v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Limitations of de novo sequencing in resolving sequence ambiguity</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> van Puyenbroeck, S., Beslic, D., Suomi, T., Holstein, T., Muth, T., Elo, L. L., Martens, L., Bouwmeester, R., Van Den Bossche, T., Claeys, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We benchmark eight state-of-the-art de novo peptide sequencing models on three publicly available proteomics datasets, comparing performance using established metrics and quantifying intermodel agreement, and perform an error analysis to identify common mispredictions and their causes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> De novo peptide sequencing enables peptide identification from fragmentation spectra without relying on sequence databases. However, incomplete spectra create ambiguity, making unambiguous identification challenging. Recent deep learning advances have produced numerous de novo models that predict sequences and refine peptide-spectrum matches under such conditions. Yet, their relative strengths, weaknesses, and ability to handle spectrum ambiguity remain unclear. Here, we benchmark eight state-of-the-art models on three publicly available proteomics datasets, comparing performance using established metrics and quantifying intermodel agreement. We assess post-processing approaches, including iterative refinement, rescoring, and reranking, for their ability to improve identification accuracy, and perform an error analysis to identify common mispredictions and their causes. Model performance varied, with considerable overlap of correct identifications. Post-processing yielded no or only modest improvements. Most sequencing errors were model-independent and driven by limited fragment ion coverage, a limitation also observed in database searches with large search spaces.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.19.671052v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MetaMiner: Streamlined GUI Tool for Retrieving, Normalizing and Exploring Metadata</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Patel, J. K., Elangovan, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce MetaMiner, a GUI tool designed to retrieve, normalize, and interactively visualize metadata associated with prokaryotic genomes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Over the past decade, publicly available genomic data has expanded exponentially. However, the usability of these genomes heavily depends on their associated metadata. Metadata provides context about the data that from where, when and how the data was obtained. When not consistent, it limits the re-usage of the available public data and prevents data's integration into broader genomic analyses. Here, we introduce MetaMiner, a GUI tool designed to retrieve, normalize, and interactively visualize metadata associated with prokaryotic genomes. MetaMiner streamlines the normalization of key metadata elements such as geographical locations, isolation sources, and sequencing technologies. To assess its usability, we used metadata of Acinetobacter baumannii genomes and compared it with NCBI's data processing tools and manually curated datasets. MetaMiner demonstrated superior performance in data transformation and showed high classification accuracy, with only minor discrepancies arising from recent geopolitical changes or ambiguous submissions. This cleaned and standardized metadata is integrated into an interactive dashboard, which allows users to explore data holistically through visualizations and custom filtering options. The dashboard has interactive plots related to submission date trend, geographical origins, sequencing methods, genome quality scores (e.g., ANI > 95, BUSCO > 95), coverage, etc. Overall, MetaMiner minimizes manual curation efforts of genomic metadata and enhances the accessibility, and re-usability of publicly available genomes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.20.666107v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Human-Mediated Dispersal and Breeding Reshape Global Genomic Patterns in Black Soldier Flies</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Muchina, P., Kinyua, J., Khamis, F., Tanga, C., Bett, R., Ssepuuya, G., Nakimbugwe, D., Sinding, M.-H. S., Gebreyesus, G., Sahana, G., Cai, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A genetic analysis of black soldier fly populations from wild and captive wild and captivity .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Human activities, either intentional or unintentional, have significantly influenced the global distribution and genetic composition of many species. Black soldier fly (Hermetia illucens; BSF) is a species that has rapidly gained commercial importance due to its bioconversion efficiency of upcycling organic waste into new products of higher value and quality through a circular economy approach. Despite its global distribution, the current Old-World demography of the wild and captive BSF populations remains poorly understood. This work combined whole-genome sequencing and population genomic analyses to determine the genetic diversity, population structure, and historical spread of global wild and captive BSF populations. Our results reveal that most global captive BSF lines were largely derived from a single primary captive lineage, likely from North America. In contrast to the genetically diverse and geographically structured wild populations, captive populations consistently exhibited reduced heterozygosity, elevated inbreeding, and extensive runs of homozygosity. These patterns reflect demographic processes such as founder effects and genetic drift, rather than intentional selection or domestication. This strongly highlights the lasting genomic impact of human-mediated dispersal and uncoordinated breeding practices. Thus, there is an urgent need for genetically informed management strategies to ensure long-term viability, adaptability, and productivity of BSF for sustainable organic waste bioconversion.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.19.670820v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Validation of the NovaSeq6000 Platform and automated library preparation for CE-IVD equivalence</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pasquinelli, E., Rollo, G., Tinella, F., Danelli, M., Minetto, S., Casamassima, G., Bader, M., Calonaci, L., Colavecchio, O. L., Tita, R., Mancini, R., Baldassarri, M., Lo Rizzo, C., Pinto, A. M., Ariani, F., Fallerini, C., Montagna, P., Mezzatesta, V., Bruttini, M., Renieri, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We validate the analytical equivalence of the NovaSeq6000 RUO configuration with automated library preparation for whole-exome sequencing and show that it is comparable to CE-IVD certified systems.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The implementation of next-generation sequencing (NGS) technologies in clinical diagnostics requires rigorous validation of sequencing platforms and analytical workflows. In this study, we validated the performance of the Illumina NovaSeq6000 Research Use Only (RUO) platform, combined with automated library preparation using the Hamilton Microlab STAR system, by comparison to the CE-IVD certified NovaSeq6000Dx platform, which currently relies on manual library preparation. A total of 96 clinical samples underwent whole-exome sequencing (WES) on both platforms. Variant detection performance was assessed for single nucleotide variants (SNVs) and copy number variants (CNVs). The RUO platform demonstrated 100% concordance with the CE-IVD system for clinically relevant SNVs, with full agreement across positive, negative, and overall percent agreement metrics. For CNVs larger than 150 kb, the positive percent agreement was 79%, rising to 91.7% for CNVs >900 kb. Coverage uniformity and autosomal callability were consistently high across platforms. These results confirm the analytical equivalence of the NovaSeq6000 RUO configuration with automated library preparation for clinical-grade WES. This validation framework supports the adoption of scalable, cost-effective workflows that can achieve diagnostic performance comparable to CE-IVD certified systems and may facilitate routine implementation of exome sequencing in clinical laboratories.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.19.671033v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comparative Temporal Transcriptomic Analysis of SOD1 Mutations in iPSC-Motor Neurons</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Huang, M., Zhang, K., Yu, F., Chen, Q., Ling, Y., Li, P., Qin, D., Su, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We investigate the molecular mechanisms driving the variable severity of amyotrophic lateral sclerosis (ALS) and investigate the role of translation and metabolic dysregulation in disease progression.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mutations in the SOD1 gene are among the most significant genetic contributors to amyotrophic lateral sclerosis (ALS), with different variants linked to varying disease severity. To investigate the molecular mechanisms driving this variability, we conducted RNA sequencing on spinal motor neurons (MNs) differentiated from isogenic human induced pluripotent stem cell (iPSC) lines engineered via CRISPR/Cas9. These lines carried two representative SOD1 heterogenous mutations, D91A and G94A, and were analyzed at Days 10 and 20 of neuronal maturation stage to capture the temporal changes of gene expression. We aim to explore how these mutations affect MN function, identify distinct molecular pathways that may explain the variable severity of ALS, and investigate the role of translation and metabolic dysregulation in disease progression.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.19.670632v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AutoLead: An LLM-Guided Bayesian Optimization Framework for Multi-Objective Lead Optimization</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, Y., Choong, J. j., Ozawa, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present AutoLead, a novel framework that integrates Large Language Models (LLMs) with multi-objective Bayesian optimization to tackle the drug discovery challenge.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The process of lead optimization in drug discovery is a complex, multi-objective challenge that remains a major bottleneck in the development of new therapeutics. Traditional approaches often struggle to efficiently explore the vast chemical space while simultaneously optimizing multiple, and sometimes conflicting, molecular properties. In this work, we present AutoLead, a novel framework that integrates Large Language Models (LLMs) with multi-objective Bayesian optimization to tackle this challenge. By leveraging the chemical reasoning capabilities of LLMs, AutoLead effectively guides the search for novel drug-like molecules that satisfy multiple objectives. We evaluate our approach on two molecule optimization tasks, achieving state-of-the-art results. Furthermore, we introduce a new benchmark dataset designed around a more realistic lead optimization scenario, where the task is to modify compounds that violate Lipinski's Rule of Five to simultaneously meet all criteria and improve their QED score. Through extensive experiments and a detailed case study, we demonstrate the potential of combining LLMs with black-box optimization techniques for more efficient and practical drug discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.19.671029v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Partial retention of ancient function increases genetic pleiotropy in grass evolution</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> de Neve, A. E., Kelly, O. A., Kelly, T., Leiboff, S., Bartlett, M. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that, rather than wholesale adoption of genetic networks, developmental genes can retain ancient functions and be recruited into other programs in the evolution of form, thereby increasing genetic pleiotropy, and that distinct environmental mechanisms regulate axillary meristem versus floral organ suppression.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Changes in form are driven by the differential, context-dependent regulation of pleiotropic genes. How genetic pleiotropy itself emerges, however, remains unclear. The maize genes GRASSY TILLERS1 (GT1) and RAMOSA3 (RA3) are required for axillary meristem suppression, a deeply conserved trait across angiosperms, and for floral organ suppression, a trait which evolved within the grass family. To determine how these pleiotropic functions are regulated, we first established a high-throughput method for quantitative phenotyping of grass flowers. Using this method, we show that distinct environmental mechanisms regulate axillary meristem versus floral organ suppression. In line with these differences, we find upstream regulation of GT1 and RA3 has diverged, consistent with their redeployment in flowers. Our results show that, rather than wholesale adoption of genetic networks, developmental genes can retain ancient functions and be recruited into other programs in the evolution of form, thereby increasing genetic pleiotropy.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.22.670905v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Biosynthesis of a major plant immunity hormone, salicylate, changed drastically during evolution of flowering plants</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Katagiri, F., Larson, E. A., Hong, K., Tsuda, K., Cohen, J. D., Hegeman, A. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We found that the main SA biosynthetic pathway in two basal angiosperm species was neither the ICS nor the PAL pathway, but a type III polyketide synthase.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Salicylate (SA) is a key plant immunity hormone, which induces immunity against biotrophic pathogens. Two SA biosynthetic pathways are known: the isochorismate synthase (ICS) and the phenylalanine ammonia-lyase (PAL) pathways. However, the primary pathways used are known for only a few plant species. For example, the ICS pathway evolved within the Brassicales order, and Arabidopsis, a member of this order, primarily uses it. It is challenging to determine the proportions of these pathway contributions using genetic perturbations because the SA biosynthetic pathways are typically regulated by feedback from SA. We developed a simple method to directly measure the relative contributions of these two pathways to SA synthesis using [13C6]glucose feeding to leaves and LC-MS/MS analysis of the leaf extracts. A survey of 16 diverse eudicot species and a monocot species, rice, showed that all these species, except for the Brassicales species with the ICS pathway, mainly use the PAL pathway. The Brassicales species mostly used the ICS pathway but did not use it exclusively: for example, the contribution of the ICS pathway in Arabidopsis was 84% on average. We found that the main pathway in two basal angiosperm species was neither the ICS nor the PAL pathway. Instead, the [13C]-labeling patterns of SA in these species were consistent with possible involvement of a type III polyketide synthase. Thus, during evolution of flowering plants, the main SA biosynthetic pathways changed drastically at least twice. Pathogen effectors targeting SA biosynthetic pathway components likely exerted selection pressure driving these drastic changes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.19.670929v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Arabidopsis Group I Pumilio RNA-binding factors are vital for embryo development and balancing between growth and stress resistance</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wu, W., Li, D., Lin, D., Xu, W., Chen, T., Chen, X., Guo, W., Long, Z., Xu, X., Tu, X., Huang, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new perspective on the trade-offs between growth and stress resilience in plants .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Pumilio (PUM) RNA binding proteins are crucial for regulating gene expression by binding to a conserved motif in the 3'-untranslated region (3'-UTR). Despite their importance, the role of PUM in plants is largely unexplored. Here, we investigated the functions of Arabidopsis group I PUMs (APUM1-APUM6), which are ubiquitously expressed and localized in the cytosol. While single apum mutants exhibit no visible phenotypes, CRISPR/Cas9-generated apum1 apum2 apum3 triple mutants (apum1/2/3) display reduced growth in both vegetative and reproductive organs, alongside hypersensitivity to various stresses. Remarkably, apum1/2/3/4 quadruple mutants are embryonically lethal, highlighting their essential role in embryo development. Transcriptomic profiling revealed that differentially expressed genes (DEGs) upregulated in apum1/2/3 are enriched in pathways related to photosynthesis, stress responses and anthocyanin biosynthesis, while downregulated DEGs are associated with biotic stress defense and hydrogen peroxide metabolism. This suggests that APUM1/2/3 act as molecular hubs balancing plant growth and stress adaptation. Biochemical assays showed that recombinant APUM homologous domains bind to the 5'-UGUGUAUA-3' core motif in the 3'-UTR of the transcription factor Production of Anthocyanin Pigment1 (PAP1), crucial for anthocyanin biosynthesis. Notably, single nucleotide substitutions, except for the third U, do not affect binding, while multiple mutations disrupt interaction. Consistently, apum1/2/3 mutants accumulate significantly more anthocyanin than wild-type plants. Furthermore, we predicted 7053 potential target genes for APUM1/2/3, with 1609 present among the upregulated DEGs in apum1/2/3. Taken together, our findings demonstrate that group I APUMs are vital posttranscriptional regulators, providing a new perspective on the trade-offs between growth and stress resilience in plants.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.21.671196v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Detergent-free reconstitution of transmembrane proteins in giant liposomes of complex curvature by the Synthetic Membrane Transfer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Meena, H., Zdanowicz, R., Bock-Bierbaum, T., Stephan, T., Ottsen Knudsen, S. D., Jakobs, S., Daumke, O., Pedersen, P. A., De Franceschi, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a new method for bottom-up reconstitution of transmembrane proteins in giant Unilamellar Vesicles using synthetic Membrane Transfer (SMT) and a new membrane geometry for the MICOS complex .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Transmembrane proteins perform many crucial functions in the cell, including transfer of matter and information across the membrane. Giant Unilamellar Vesicles (GUVs) are the ideal tool for studying these processes; yet functional transmembrane protein reconstitution in GUVs still represents a bottleneck in bottom-up synthetic biology. Here, we developed a novel approach that we call Synthetic Membrane Transfer (SMT), where transmembrane proteins reconstituted into styrene-maleic acid particles in a detergent-free environment can be transferred to the GUV membrane. The SMT approach is a one-step, facile and general method that works for structurally diverse proteins, and it does not require any specific lipid or buffer composition. Moreover, the SMT is fully compatible with the Synthetic Membrane Shaper (SMS) technology, which allows to deform GUVs in a controlled fashion, obtaining dumbbell-shaped GUVs exhibiting a catenoid-like geometry. The combination of SMT and SMS results in functional reconstitution of transmembrane proteins in catenoid membrane necks. Using this approach, we demonstrate that Mic10, a component of the MICOS complex, directly senses membrane curvature and localizes at the neck of dumbbells GUVs, a geometry that recapitulates the shape of mitochondria cristae junctions. This paves the way for bottom-up reconstitution of the MICOS complex on a physiological membrane geometry.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.23.671895v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">GCAD: a Computational Framework for Mammalian Genetic Program Computer-Aided Design</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dreyer, K. S., Nguyen, A. V., Bora, G. G., Redus, L. E., Edelstein, H. I., Garcia, J., Anastasia, E., Dray, K. E., Leonard, J. N., Mangan, N. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a genetic algorithm for mammalian genetic program computer-aided design, and demonstrate that the algorithm can successfully find optimal circuit designs.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genetic programs can direct living systems to perform diverse, pre-specified functions. As the library of parts available for building such programs continues to expand, computation-guided design is increasingly helpful and necessary. Yet key gaps exist for designing programs for use in mammalian cells in particular. Predictive models aid the challenging design process, but iterative simulation and experimentation are intractable for complex functions. Computer-aided design accelerates this process, but existing tools do not yet capture the behavior of mammalian-specific parts and population-level effects needed for mammalian synthetic biologists. To address these needs, we developed a framework for mammalian genetic program computer-aided design. Starting with a user-defined design specification to quantify circuit performance, the framework uses a genetic algorithm to search through possible designs. Circuit space is defined by a library of experimentally characterized parts and dynamical systems models for gene expression in a heterogeneous cell population. We developed this genetic algorithm using a directed graph-based formulation with biologically constrained rules to explore regulatory connections and parts. We evaluated the framework for design problems of varying complexity, including programs we describe as an amplifier, signal conditioner, and pulse generator, demonstrating that the algorithm can successfully find optimal circuit designs. Finally, we experimentally evaluated selected circuits, demonstrating the path from a predicted circuit design to experimental testing and highlighting the importance of characterization for enabling predictive design. Overall, this framework establishes general approaches that can be refined and expanded, accelerating the design and implementation of mammalian genetic programs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.23.671908v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Engineering non-exponential proliferation in Escherichia coli using functionalized protein aggregates</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Van Eyken, R., Jordens, S., Oome, D., Broux, K., Aertsen, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A bacterial chassis that adheres to linear proliferation for a finite number of generations, based on the asymmetric segregation of a gradually declining intracellular protein aggregate.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Uncurbed exponential proliferation might not always be required for genetically modified microorganisms, and might even cause unpredictable liabilities in their behavior and impact. We therefore constructed a bacterial chassis that adheres to linear proliferation for a finite number of generations, based on the asymmetric segregation of a gradually declining intracellular protein aggregate that is engineered to produce a conditionally essential metabolite such as cAMP.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.23.671898v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Novel Allosteric Inhibitor Targeting IMPDH2 at Y233 Overcomes Resistance to Tyrosine Kinase Inhibitors in Lymphoma</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pattabiraman, N., Lobello, C., Rushmore, D., Mologni, L., Wasik, M., Basappa, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify IMPDH2 as a direct substrate of the oncogenic kinases ALK and SRC, which phosphorylate tyrosine 233 within the enzymes allosteric Bateman domain, and describe a new IMPDh2 inhibitor, comp-10, with an IC50</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> IMPDH2 is a rate-limiting enzyme in the de novo biosynthesis of guanine nucleotides and is often overexpressed in hematologic malignancies and solid tumors. However, its regulatory mechanisms in cancer remain poorly understood. Here, we identify IMPDH2 as a direct substrate of the oncogenic kinases ALK and SRC, which phosphorylate tyrosine 233 within the enzymes allosteric Bateman domain. Using peptide-based mutagenesis and enzymatic assays, we demonstrate that Y233 phosphorylation is essential for IMPDH2 activity. We found that phosphoinositide-3-phosphate (PI3P), a signaling phospholipid, selectively binds to and inhibits IMPDH2, but not its isoform IMPDH1, revealing a novel lipid-mediated, isoform-specific regulatory mechanism. Next, we conducted structure-based virtual screening and identified a potent allosteric inhibitor of IMPDH2, comp-10, with an IC50 of 260 nM. Comp-10 significantly impairs cell proliferation in ALK-positive anaplastic large cell lymphoma (ALCL) cell lines, including those resistant to the ALK inhibitors crizotinib and lorlatinib, and outperformed the FDA-approved IMPDH inhibitor mycophenolic acid. These findings reveal the dual regulation of IMPDH2 through tyrosine phosphorylation and binding to PI3P, and describe the discovery of a new IMPDH2 inhibitor, suggesting a potential therapeutic strategy to overcome resistance to tyrosine kinase inhibitors.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.19.667179v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PHLDA2 promotes breast cancer metastasis by co-opting a developmental program for placental vascular remodeling</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Halas, P. V., Savage, H., Hachey, S. J., Lev, T., Prem Antony Samy, A. L., Chan, C., Jimenez, S., Domadia, R. M., Lin, I., Mallya, S., Insua-Rodriguez, J., Gonzalez, J., Adam, I., Alshetaiwi, H., Hernandez, G. A., Longworth, A., McMullen, T. P., Kessenbrock, K., Hughes, C. C., Ma, D., Lawson, D. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify PHLDA2 as a robust driver of breast cancer metastasis by increasing gene expression of vascular remodeling and promoting metastasis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Identifying drivers of metastasis is essential for developing new treatments for patients with advanced disease. Here, we identify PHLDA2 as a robust driver of breast cancer metastasis. Previous work established PHLDA2 as an imprinted gene expressed by trophoblasts which are critical for vascular remodeling during placental development. We find that hypomethylation of PHLDA2 in breast tumors correlates with increased gene expression, which is associated with metastasis and poor survival in breast cancer patients. RNA-sequencing showed that PHLDA2 overexpression results in upregulation of genes that control invasion, extracellular matrix assembly, and vascular remodeling, consistent with trophoblast functions in placental development. Using an in vitro vascularized microtumor (VMT) system, we find that PHLDA2 functions through SPARC, which promotes metastasis by inducing vascular permeability and enhancing tumor dissemination. These data suggest that increased expression of PHLDA2 through hypomethylation promotes metastasis by ectopic expression of a developmental program for vascular remodeling.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.19.670946v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Structure-guided engineering of CCL27 enhances natural ligand CAR T-cells against CCR10 for multiple myeloma</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chilakapati, N., Patino-Escobar, B., Chen, E. Y., Dalal, R., de Montagnac, J., Johnson, H., Kang, A. S., Akagi, N., Ramos, E., Phojanakong, P., Salangsang, F., Zeng, Y., Chari, A., Chung, A., Kumar, A. D., Martin, T. G., Shy, B. R., Steri, V., Karlon, W. J., Kortemme, T., Barwick, B. G., Wiita, A. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed CCL27 mutants with an additional N-terminal aromatic amino acid that dramatically improved the efficacy of BCMA CAR-Ts to near that of current anti-BCMA CARs, demonstrating the potential of engineering natural ligand CAR-T beyond their wild-type sequences.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Despite the success of BCMA CAR-Ts, many multiple myeloma patients relapse and require additional therapeutic options. Our group previously identified the chemokine receptor CCR10 as a potential alternate target to address this need. Here, we validated CCR10 expression on primary myeloma tumors and sought to develop CAR T-cells against CCR10, utilizing its natural ligand CCL27 as a CAR binding element. However, CARs based on the native CCL27 sequence were ineffective. We thus utilized computational modeling and structure-guided engineering to inform rational mutations along the CCL27-CCR10 interface, exploiting a hydrophobic pocket on CCR10. This effort identified CCL27 mutants with an additional N-terminal aromatic amino acid that dramatically improved the efficacy of CCL27-based CAR-Ts to near that of current anti-BCMA CAR-Ts. We validated key amino acid contacts at the CCL27-CCR10 interface, which contribute to increased CAR binding avidity, predicted to be influenced by increased Van der Waals interactions. Lastly, we found that the CCL27 mutants have no toxicity in the hematopoietic compartment. This work illustrates the potential of engineering natural ligand CAR-Ts beyond their wild-type sequences and underscores the translational potential of engineered CCL27 mutant CAR-Ts.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.19.670119v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

