<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Tuesday, 23 September 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Novel radiation-derived gene blueprint stratifying patients with breast cancer.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hao Zhang, HongHua Lin, Enyi Qiu, Wenqi Jin, Shi Dong</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> The role of radiation-related genes in breast cancer and the effectiveness of predictive, preventive, and personalized treatment strategies.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Breast cancer (BC) is a prevalent global malignancy with a high recurrence rate. The effectiveness of predictive, preventive, and personalized treatment strategies is limited by a lack of reliable prognostic biomarkers. Radiotherapy significantly reduces breast cancer recurrence risk and prolongs patients' lives. However, the role of radiation-related genes in breast cancer remains unclear.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40983794/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AI-empowered human microbiome research.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tian Zhou, Fangqing Zhao</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We systematically explore AI-driven methodologies in microbiome research, including clustering algorithms, dimensionality reduction techniques, convolutional and recurrent neural networks, and emerging large language models, and explore how these approaches enable the extraction of meaningful biological patterns from complex microbial data from a multiscale perspective.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent advances in high-throughput microbiome profiling have generated expansive data sets that offer unprecedented opportunities to investigate the role of microbes in human health. However, the complexity and high dimensionality of these data present significant analytical challenges that often exceed the capabilities of traditional computational methods. Artificial intelligence (AI), encompassing both classical machine learning and modern deep learning approaches, has emerged as a powerful solution to these challenges. In this review, we systematically explore AI-driven methodologies in microbiome research, including clustering algorithms, dimensionality reduction techniques, convolutional and recurrent neural networks, and emerging large language models. We assess how these approaches enable the extraction of meaningful biological patterns from complex microbial data from a multiscale perspective, facilitating insights into community dynamics, host-microbe interactions and functional genomics. Additionally, we explore the transformative impact of AI on translational applications across both academic research and real-world clinical settings, including disease diagnostics, therapeutic development and precision microbiome engineering. By critically evaluating the current capabilities and limitations of AI in this context, this review aims to chart a path forward for the integration of AI into microbiome research, ultimately accelerating innovations in personalised medicine and deepening our understanding of host-microbiome relationships.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40983504/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AI methods and biologically informed data curation enable accurate RNA m5C prediction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Saitto, E., Casiraghi, E., Paccanaro, A., Valentini, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We trained a high-confidence, methyltransferase-specific catalog of m5C sites from millions of unmodified cytosines, which led to a biologically informed analysis of transcriptome-wide methyltransfer enzyme-specific sites and improved the accuracy of m 5C identification.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> 5-methylcytosine (m5C) RNA modifications influence nearly every aspect of RNA metabolism, but their transcriptome-wide detection is limited by costly, error-prone assays. To bridge this experimental gap, a wave of AI tools now predicts putative m5C sites in silico. However, most existing approaches prioritize architectural complexity while neglecting data quality, so their reported gains mainly reflect the artifacts inherited from noisy datasets. We inverted this paradigm by constructing a high-confidence, methyltransferase-specific catalog of m5C sites, removing artifacts that confound existing resources. Using this curated corpus, we trained (for the first time in a multiclass setting) three different models (Bi-GRU, CNN, Transformer) to distinguish writer-specific m5C sites from unmethylated cytosines. All AI models converged to similar, nearly optimal, performance (AUPRC > 0.97), and a biologically informed analysis revealed that most errors clustered in unmethylated sites mimicking true positives. By augmenting the training set with these hard-to-predict negatives, mined from millions of unmodified cytosines, the models were forced to exploit more nuanced features such as RNA secondary structure and subtle sequence cues, which sharply reduced transcriptome-wide false positive predictions, and predicted methylated transcripts exhibited strong concordance with known methyltransferase biology. Explainable AI techniques also showed that our AI models effectively capture how sequence mutations disrupt m5C sites, underscoring their potential to prioritize disease-relevant variants. The main findings of our study underscore that AI models can be decisive levers for reliable m5C identification only if fed with curated data and validated through biologically informed computational analysis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.677824v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Inferring Personalized Cell-Cell Communication Networks in Colorectal Cancer with Individualized Causal Discovery</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Qiu, A., Lu, B., Cooper, G., Lu, X., Chen, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed an integrative computational framework combining the nested hierarchical Dirichlet process (nHDP) model for identifying hierarchically structured gene expression modules, with instance-specific Greedy Fast Causal Inference (iGFCI) for inferring individualized CCCNs (iCCCN</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding tumor heterogeneity at the resolution of individualized cell-cell communication networks (CCCNs) remains a major computational challenge in precision oncology. Existing inference methods largely rely on population-level correlation and thus fail to capture patient-specific signaling patterns across diverse cell types. To address this limitation, we developed an integrative computational framework combining the nested hierarchical Dirichlet process (nHDP) model for identifying hierarchically structured gene expression modules, with instance-specific Greedy Fast Causal Inference (iGFCI) for inferring individualized CCCNs (iCCCNs) in colorectal cancer (CRC). Applied to large-scale single-cell RNA-seq data from over 625,000 cells, our model successfully decomposed complex gene expression modules (GEMs), potentially representing the cell lineage and cellular signaling states, and uncovered iCCCNs across detailed cell subtypes. We further used TCGA bulk RNA-seq data with survival data to validate the clinical relevance of these individualized gene expression module causal interactions, demonstrating their potential as robust prognostic signatures in CRC. Finally, we used principled causal inference methods to search for ligand-receptor pairs that mediate cell-cell communication. This framework enables mechanistic insights into immune evasion, and our computational method represents a significant advance toward realizing personalized oncology, enabling precise patient stratification and identification of actionable biomarkers for improved therapeutic targeting in cancers.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.20.677543v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The UTRs of Leishmania donovani vary in length and are enriched in potential regulatory structures</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dumetz, F., Watson, K. J., Perry, M., Bromley, R. E., Shome, A. R., Dunning Hotopp, J. C., Hamza, I., Serre, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We generated high-quality genome and transcriptome datasets for Leishmania donovani strain 1S2D (Ld1S) by combining PacBio HiFi de novo assembly with Oxford Nanopore direct RNA sequencing of promastigotes and axenic amastigote.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Leishmania spp. regulate gene expression largely post-transcriptionally, yet untranslated regions (UTRs) remain poorly delineated. We generated high-quality genome and transcriptome datasets for Leishmania donovani strain 1S2D (Ld1S) by combining PacBio HiFi de novo assembly with Oxford Nanopore direct RNA sequencing of promastigotes and axenic amastigotes. The genome assembly consists of 65 scaffolds totaling ~33.3 Mb. Structural comparisons to LdBPK282A1 revealed numerous rearrangements, including some reshuffling genes among polycistronic transcription units and validated by polycistronic reads from RNA sequencing. Promastigote and amastigote RNA sequencing produced 469,010 and 46,729 monocistronic reads containing a spliced-leader and a polyA tail sequences, defining 8,479 transcripts and supporting 7,415 of the 7,969 annotated protein coding genes, as well as 604 putative long non-coding RNAs. We annotated UTRs for 4,921 genes and observed that putative RNA G-quadruplexes were markedly enriched in UTRs. We also noted that 31.9% and 11.5% were expressed into multiple isoforms in promastigotes and amastigotes, respectively. Collectively, these data provide a comprehensive annotation of L. donovani genes and their UTRs and reveal widespread and stage-specific UTR length polymorphisms, and, overall, points to an important role of 3 UTR in post-transcriptional regulation in L. donovani.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.677690v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Closed-loop Optogenetic Control in a Microplate Reader</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Namboothiri, H. R., Pochana, K., Jaiswal, B., Emami, A., Hu, C. Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A low-cost LED-embedded microplate that operates inside a commercial microplate reader, keeping cultures in the reader for the entire experiment while syn-chronizing illumination with measurement.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Living cells sense diverse molecules and synthesize complex compounds, but their responses are slow and context dependent, which makes precise control difficult. Electronic systems excel at fast computation, communication, and control, but their direct access to biochemistry is limited. These two classes can be integrated through optogenetics to form cell-silicon systems for sensing and biomanufacturing. However, prototyping these systems remains nontrivial: current tools either pair high-accuracy sensing with robotic sample transfer that requires extensive setup, or avoid transfers by using custom sensors with limited accuracy and dynamic range that compromise performance. Here we present LEMOS, a low-cost LED-embedded microplate that operates inside a commercial microplate reader, keeping cultures in the reader for the entire experiment while syn-chronizing illumination with measurement. Using LEMOS and a growth-aware multiscale model, we rapidly executed the design-build-test-learn cycle to diagnose and correct closed-loop over-shoot in gene-expression setpoint tracking. We demonstrate closed-loop gene-expression control in batch culture within a standard plate reader and show that growth state is a key design variable for controller choice and tuning. By combining accessible hardware with a predictive model, LEMOS lowers the barrier to model-informed DBTL, shortens iteration, and provides a path to portable control strategies that extend to other hosts and optogenetic systems.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.22.677797v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Knocking out two polyphenol oxidase genes significantly improves recombinant protein purification in Nicotiana benthamiana</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Diao, H.-P., Meng, H.-X., Xu, X.-J., Guo, Y.-F., Hwang, I., Song, S.-J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified two key polyphenol oxidase genes, PPOa and PPOb, from N. benthamiana, that significantly improved the purification of functional proteins like SARS-CoV-2 Spike trimer and influenza HA trimer.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Efficient purification remains one of the major bottlenecks in the development of plant-based systems for recombinant protein production. The complex metabolites, particularly polyphenols, which usually cause recombinant protein aggregation during purification. In this study, we identified two key polyphenol oxidase genes, PPOa and PPOb from N.benthamiana as responsible for these effects. Using CRISPR/Cas9, we generated two ppoa;ppob double knockout lines that significantly improved the purification of functional proteins like SARS-CoV-2 Spike trimer and influenza HA trimer. These lines showed reduced polyphenol-protein interactions, minimized aggregation, and higher purification yields. Our work establishes a clean, high-efficiency N. benthamiana chassis for scalable recombinant protein production.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.21.677675v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Precision in Predicting Protein-Nucleic Acid Complexes: Establishing a Benchmark Data Set and Comparative Metrics.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Huizi Cui, Yuxuan Wang, Yu Fu, Xiangyu Yu, Wannan Li, Feng Lin, Weiwei Han</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We systematically evaluated four deep learning algorithms (AlphaFold3, Chai-1, HelixFold 3, and Protenix) and two physically driven docking methods (HDOCK and HDOCK_NT) and found that physically driven methods significantly outperform current DL approaches in predicting protein</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein-nucleic acid interactions are fundamental to biological processes and biotechnology, yet their computational prediction lags behind protein structure or protein-protein interaction modeling. This study introduces ProNASet, a benchmark data set of 100 experimentally resolved protein-nucleic acid complex structures, alongside a multidimensional evaluation framework using root mean square deviation (RMSD), TM-score, and local distance difference test (LDDT) metrics. We systematically evaluated four deep learning (DL) algorithms (AlphaFold3, Chai-1, HelixFold3, and Protenix) and two physically driven docking methods (HDOCK and HDOCK_NT). Our analysis revealed that physically driven methods significantly outperform current DL approaches in predicting protein-nucleic acid complex structures. The template-less HDOCK_NT demonstrated the highest success rate at 74.5% (using thresholds RMSD 0.9, and LDDT >0.6), compared to 63.8% for template docking and only 34.0% for the best-performing DL method, AlphaFold3. These results underscore the substantial need for improvement in DL methods for this specific task. The ProNASet benchmark provides a standardized testing platform, highlights intrinsic shortcomings in current DL models for capturing protein-nucleic acid interaction features, and guides the development of next-generation computational tools crucial for advancing genome editing and synthetic biology.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40932245/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multiview Deep Learning Framework for Precise Prediction of Transcription Factor Binding Sites.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yiben Lin, Huiliang Luo, Liang Yan, Changmiao Wang, Yao Li, Ruiquan Ge</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel multiview deep learning model for transcription factor binding prediction that leverages multiple views of DNA sequences and a bidirectional reverse complement module.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Transcription factors (TFs) are essential proteins that regulate gene expression by specifically binding to transcription factor binding sites (TFBSs) within DNA sequences. Their ability to precisely control the transcription process is crucial for understanding gene regulatory networks, uncovering disease mechanisms, and designing synthetic biology tools. Accurate TFBS prediction, therefore, holds significant importance in advancing these areas of research. While machine learning methods, particularly deep learning approaches, have achieved notable progress in TFBS prediction in recent years, several challenges persist. These include modeling the intricate structural features of the DNA double helix, capturing long-range dependencies within sequences and integrating diverse biological data sources. To address these issues, we propose an innovative solution known as multiview deep learning for Transcription Factor Binding Prediction (MDNet-TFP), which leverages multiple views of DNA sequences‚îÄincluding different representational forms and diverse processing strategies‚îÄto enhance prediction capabilities. Specifically, our framework introduces a bidirectional reverse complement module (BiRC-Mamba) that effectively accounts for the bidirectional and reverse complement properties characteristic of DNA sequences. Furthermore, we developed a multiscale convolutional recurrent attention network (MCRAN) that extracts both structural and functional DNA features across multiple dimensions while integrating information from various biological data sets. These advancements allow our model to outperform existing methods across 165 ChIP-seq data sets, achieving an average ACC of 88.13% (¬±0.47), an ROC-AUC of 93.72% (¬±0.15), and a PR-AUC of 93.40% (¬±0.21). The model not only excels with this specific data set but also maintains its high performance across a wider array of 690 ChIP-seq data sets. To further validate the model's effectiveness, we employ motif visualization techniques. This approach reveals that the regions receiving high attention from our model align with known transcription factor binding motifs, offering valuable biological insights. Additionally, this correspondence substantiates the model's ability to generalize and interpret complex genomic data effectively. By addressing critical limitations in the field, MDNet-TFP offers a promising new avenue for advancing research in transcriptional regulation and biomedical applications.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40914877/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

