<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Thursday, 02 October 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Microbial stem cells support productivity in dedicated factory cells in an asymmetrically dividing E. coli system</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mushnikov, N. V., Guidry, J., Park, A., Bowman, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A major challenge for many bio-manufacturing operations is that cells are burdened by the high fitness cost of product synthesis, limiting their growth and productivity, and a potential solution is to decouple cell reproduction from product synthesis by dividing these conflicting tasks between two differentiated cell types.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> A major challenge for many bio-manufacturing operations is that cells are burdened by the high fitness cost of product synthesis, limiting their growth and productivity. A potential solution is to decouple cell reproduction from product synthesis by dividing these conflicting tasks between two differentiated cell types. This work describes the use of an asymmetrically inherited protein cue to differentiate an E. coli culture into reproductive stem cells and fully dedicated factory cells. Cell differentiation is based on the ability to accumulate two factors: a variant of phage-derived T7 RNA polymerase (T7RNAP) and GP2, a peptide that inhibits native host cell RNA polymerase. Activating these two factors in factory cells inhibited growth and focused them on T7RNAP-driven product synthesis. Preventing their accumulation in stem cells allowed this cell type to grow and divide asymmetrically, generating new factory cells in the process. These differentiating cell cultures generated over eight-fold higher target protein titers compared to factory cell-only controls. Because they include a mechanism for preventing leaky T7RNAP-driven gene expression in stem cells and pre-induction cultures, it was possible to generate strains with multiple plasmid-based copies of a cytotoxic target gene, whereas leaky expression made the same plasmid inviable in conventional protein expression strains. This versatile genetic system could be useful for generating higher product titers, particularly in cases where product synthesis causes cytotoxicity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.01.679865v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Overexpression of MusaVicilin Gene for Disease Resistance in Banana</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tripathi, J. N., Macharia, S., Muiruri, S., Ntui, V., Tripathi, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study on the overexpression of the MusaVicilin gene cloned from the wild progenitor Musa balbisiana to enhance resistance to Xanthomonas wilt in susceptible banana cultivars.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Banana Xanthomonas wilt (BXW) disease, caused by Xanthomonas vasicola pv. musacearum, is a major constraint to banana production in East and Central Africa. All cultivated banana varieties are susceptible, with the wild progenitor Musa balbisiana being the only known source of complete resistance. Limitations in classical breeding have prompted the exploration of molecular genetic tools, such as genetic modification, to develop resistant cultivars. Comparative transcriptomic analyses revealed a five-fold upregulation of MusaVicilin gene in M. balbisiana (BB genome) compared to the BXW- susceptible Pisang Awak when challenged with the pathogen, suggesting its role in defense. This study investigated whether constitutive overexpression of the MusaVicilin gene cloned from M. balbisiana could enhance resistance to BXW in the susceptible Sukali Ndiizi cultivar (AAB genome). Transgenic events were developed with the MusaVicilin gene under the control of the constitutive CaMV 35S promoter. These events exhibited enhanced disease resistance compared with non-transgenic control plants. The overexpression of MusaVicilin highlights its potential as a candidate gene for engineering resistance to BXW in susceptible cultivars. Moreover, MusaVicilin could serve as a valuable component in gene stacking strategies aimed at developing durable, disease-resistant banana varieties.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.01.679877v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A deep learning and co-conservation framework enable discovery of non-canonical Cas proteins</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> He, B., Qi, C., Feng, Y., Liu, D., Wu, R., Wang, Z., Wang, D., Yue, Z., Zhang, Y., Lan, H., Zheng, Y., Li, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present BioPrinCRISPR, a class-agnostic computational framework leveraging gene co-conservation, protein domain co-occurrence, and embedding similarity to identify and characterize CRISPR-Cas systems across prokaryotic genomes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> CRISPR-Cas systems are central to prokaryotic adaptive immunity, widely harnessed for biotechnology. Yet, their vast and uncharacterized diversity, especially non-canonical variants, impedes full exploitation. Here we present BioPrinCRISPR, a class-agnostic computational framework leveraging gene co-conservation, protein domain co-occurrence, and embedding similarity to identify and characterize CRISPR-Cas systems across prokaryotic genomes. Applying BioPrinCRISPR to over one million bacterial genomes, we uncovered extensive canonical and uncharacterized systems, revealing a rich landscape of atypical Cas proteins and novel domain architectures. Notably, we identified recurrent fusion proteins with unique enzymatic combinations, suggesting roles in regulatory control or nucleic acid remodeling. Experimental validation of two divergent Cas13an-like effectors demonstrated RNA knockdown capacity in human cells, confirming our framework's predictive power. These findings expand the functional repertoire of CRISPR-associated proteins and highlight unexplored modes of microbial immunity. BioPrinCRISPR thus stands as a powerful tool for comprehensively mapping CRISPR-Cas diversity, offering new insights into prokaryotic defense and facilitating discovery of novel candidates for next-generation genome engineering. An accompanying interactive web platform was also developed to facilitate data exploration.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.30.679098v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">FUSED: CROSS-DOMAIN INTEGRATION OF FOUNDATION MODELS FOR CANCER DRUG RESPONSE PREDICTION</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> RoÃàssner, T., Balke, J., Tang, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce FUSED (Fusion of Foundation Model Embeddings for Drug Response Prediction), a novel architecture for cross-domain foundation model integration, which significantly improves performance in both drug known and leave-one-drug-out scenarios.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> AI-driven methods for predicting drug responses hold promise for advancing personalized cancer therapy, but cancer heterogeneity and the high cost of data generation pose substantial challenges. Here we explore the transfer learning capability and introduce FUSED (Fusion of Foundation Model Embeddings for Drug Response Prediction), a novel architecture for cross-domain foundation model (FM) integration. By systematically benchmark FMs across two domains - molecular FM for drugs and single-cell FM for cell lines, we demonstrate that integrating single-cell FMs substantially reduces the number of input features required for cell line representation. Among FMs, Molformer significantly outperforms ChemBERTa, and scGPT surpasses scFoundation in predictive accuracy and training stability. Moreover, integrating single-cell FMs improves performance in both drugknown and leave-one-drug-out scenarios. These findings highlight the potential of cross-domain FM integration for more efficient and robust drug response prediction.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.30.679434v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comprehensive benchmarking with guidelines for analyzing transposable element-derived RNA expression</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> She, J., Wang, J., Yang, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present benchmarking of 16 representative tools with 120 simulated datasets and 60 real-world paired datasets (comprising both long- and short-read data), by evaluating the performance of teRNA identification or quantification across family-, unit-, exon-, and transcript-level.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Transposable element-derived RNAs (teRNAs) have been recognized with accelerating fundamental or pathogenic roles, especially in human. Despite the rapid development of computational methods, the best practice for accurate identification and quantification of teRNAs are currently lacking owing to the difficulties of evaluation. Here we present benchmarking of 16 representative tools with 120 simulated datasets and 60 real-world paired datasets (comprising both long- and short-read data), by evaluating the performance of teRNA identification or quantification across family-, unit-, exon-, and transcript-level. Our findings demonstrate not only the exon-level as a trade-off between accuracy and resolution for teRNA analysis, but also the level-dependent strengths and weaknesses of evaluated methods. To refine our benchmarking results, we present decision-tree-style guidelines and develop an integrated best-practice pipeline, serving as the basis for future functional researches. In addition, our evaluation framework also provides a gold standard for developing and benchmarking better computational tools in the field.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.30.679421v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A proteogenomic approach to discover novel lncRNA-derived peptides and their potential clinical utility in hepatocellular carcinoma</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bingwu, L., Joshi, K., Wang, D. O.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We discovered 105 novel long noncoding RNA (lncRNA) derived peptides including lncPeps differentially expressed between tumor and non-tumor tissues, and lnc Peps with significant correlation with prognosis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Peptides are increasingly recognized for their versatile functions in biological contexts but their clinical relevance and utility remain largely unexplored. Proteogenomic approaches can accelerate peptide discovery in clinical samples by integrating proteomic data with genomics and transcriptomics evidence. However, long noncoding RNA (lncRNA) derived peptides (lncPeps) remain largely unidentified, resulting in unmatchable MS/MS spectra. To solve this problem, we have used high-quality Ribo-seq translatomic datasets to generate an extensive database of human liver lncPeps, which we subsequently applied to proteomics data of tumor adjacent normal tissue pairs from hepatocellular carcinoma (HCC) patients. Using the new database, we discovered 105 novel lncPeps including lncPeps differentially expressed between tumor and non-tumor tissues, and lncPeps with significant correlation with prognosis. Remarkably, combining the expression of lncPeps with canonical proteins in a LASSO regression model improved predictive performance for recurrence, increasing the AUC by 0.005 to 0.085 across three recurrence time points. These findings suggest that lncPeps discovery contributes to our understanding of the molecular heterogeneity and progression of HCC, and broadens the range of potential biomarker candidates or treatment targets for the disease.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.30.679478v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">T cell-microbiome associations captured through T cell receptor convergence analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Vandoren, R., Ha, M. K., Van Deuren, V. M. L., De Roeck, N., Pu, T., Brand, E. C., Kuznetsova, M., Besbassi, H., Bartholomeus, E., Affaticati, F., De Boeck, I., Gehrmann, T., Lebeer, S., Oldenburg, B., van Wijk, F., Delputte, P., Verbandt, S., Tejpar, S., Laukens, K., Ogunjimi, B., Meysman, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present AIRRWAS, a computational framework that integrates TCR-microbiome interaction analysis with targeted in vitro validation to detect genus-level TCR convergence, enabling detection of shared immune signatures.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The gut microbiome modulates mucosal immunity, yet how specific bacterial taxa shape the diversity and specificity of T cell receptor (TCR) repertoires remains poorly understood. Existing approaches emphasize single-species effects or broad immune features, without pinpointing which microbes drive specific T cell clonotypes. We present AIRRWAS, a computational framework that integrates TCR-microbiome interaction analysis with targeted in vitro validation to detect genus-level TCR convergence. Applied to three independent cohorts, AIRRWAS identified reproducible associations between convergent TCR clusters and 21 bacterial genera spanning core commensals, probiotics and taxa with immunomodulatory roles. Predicted clonotypes were enriched within the TCR-microbiome interaction network and preferentially activated by genus-matched stimuli, eliciting different functional T cell responses. These findings demonstrate that distinct repertoires can share genus-specific TCR motifs, enabling detection of shared immune signatures. AIRRWAS can map these TCR-microbiome interactions, laying the groundwork for biomarker discovery immune monitoring and the development of microbiome-targeted therapies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.30.679432v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrating Multi-Structure Covalent Docking with Machine Learning Consensus Scoring Enhances Virtual Screening of Human Acetylcholinesterase Inhibitors</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rayakar, A. A., Jaladanki, C. K., Yap, X. H., Fan, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> In this study, we developed an in silico protocol that integrates multi-structure covalent docking and machine learning (ML) consensus scoring to improve the prediction of AChE inhibitors.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Acetylcholinesterase (AChE) inhibition is a key mechanism in the treatment of neurodegenerative diseases and in counteracting toxic exposures to pesticides and nerve agents. However, virtual screening of AChE remains challenging due to the enzyme's structural flexibility and the chemical diversity of its covalently binding inhibitors. In this study, we developed an in silico protocol that integrates multi-structure covalent docking and machine learning (ML) consensus scoring to improve the prediction of AChE inhibitors. We analyzed 65 ligand-bound (holo) human AChE crystal structures using hierarchical clustering to identify four representative conformations, along with one high-resolution apo structure, for multi-structure docking. A curated library of 412 organophosphate and carbamate inhibitors was then docked covalently and non-covalently into each receptor conformation. The resulting docking scores were evaluated against inhibitors' experimental logIC50 values using Spearman's rank correlation coefficient (). Covalent docking outperformed non-covalent docking ( values up to 0.54 vs 0.18), and our ML consensus model trained on the five structures' covalent docking scores achieved the highest predictive accuracy ( = 0.70), surpassing all single-structure and conventional consensus baselines. Chemical cluster analysis revealed structure-activity trends based on ligand flexibility, polarity, and aromaticity. SHapley Additive exPlanations analysis highlighted the ML consensus model's ability to flexibly distribute the influence each structure's scores played on its predictions. It identified and exploited relationships based on its training dataset that would be difficult to anticipate through a manual analysis of individual structures' docking performance metrics. This framework is broadly applicable to other covalently targeted proteins, offering a generalizable and interpretable strategy for data-driven covalent inhibitor discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.29.679420v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Mechanistic Insights into the inhibition of Plasmodium falciparum DNA gyrase A by withanolides derivatives through integrated computational analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dasari, J. B., Soren, B. C., Vastrad, S. J., Junied, S., Srikanth, D., Chimalamari, A., Jayappa, B. M. K. B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A structure based computational approach to characterize the binding mechanism and dynamic stability of three bioactive withanolide derivatives (D, E, and O) against pfDNA gyrase.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Malaria is a fatal disease affecting millions of people worldwide, primarily due to infection by Plasmodium falciparum. The emergence of multidrug-resistant parasite strains has necessitated the exploration of novel therapeutic targets, among which DNA gyrase represents a unique and underexploited enzyme in the parasites replication machinery. Plasmodium falciparum DNA gyrase A (pfDNA gyrase), an essential topoisomerase II that is not present in humans, has been identified as a promising target for antimalarial drug development. Present study deals with a structure based computational approach to characterize the binding mechanism and dynamic stability of three bioactive withanolide derivatives (D, E, and O) against pfDNA gyrase. Molecular docking revealed high binding affinities for withanolide D (-9.14kcal/mol), E (-9.73kcal/mol), and O (-9.00kcal/mol), with interactions mediated through key catalytic residues such as GLU648, LYS647, and TRY590 via hydrogen bonding and hydrophobic contacts. Stability of the ligand-protein complexes was further assessed through molecular dynamics simulations, where analyses of RMSD, RMSF, radius of gyration (Rg), and solvent accessible surface area (SASA) analysis confirmed the structural integrity and compactness of the complexes, notably withanolide O exhibited the most favorable dynamic profile, whereas withanolide E induced confirmational rigidity. MM/GBSA calculations are further supported by showing the lowest binding free energy for withanolide O and E ({Delta}Gbind =-20.89 and -20.22 kcal/mol). The ADME studies showed favorable pharmacokinetic and physiochemical properties of three ligands. Collectively, these findings highlight the potential of withanolide derivatives as promising inhibitors of pfDNA gyrase, thereby paving a way for the foundation of future antimalarial drug development.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.30.679425v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Disentangling covariate effects on single cell-resolved epigenomes with DeepDive</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Moeller, A., Madsen, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce DeepDive, a novel deep learning framework designed to systematically disentangle known and unknown sources of variation in single-nucleus ATAC-seq data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding the effects of individual biological factors from single cell-resolved epigenomic data is hindered by multicollinearity, particularly in human cohorts. We introduce DeepDive, a novel deep learning framework designed to systematically disentangle known and unknown sources of variation in single-nucleus ATAC-seq data. DeepDive accurately reconstructs chromatin accessibility, outperforms state-of-the-art methods with incomplete covariate information, and robustly recovers true biological signals from even highly entangled covariates, unlocking counter-factual, what-if, analyses. Applying DeepDive to pancreatic islet cells, we perform counter-factual analyses to prioritize covariates associated with a type 2 diabetes-linked beta cell subtype and nominate transcription regulators. DeepDive offers a powerful and unbiased tool for mechanistic discovery in complex human disease cohorts.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.30.679466v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">p53 activity licenses transcriptional regulation by YAP/TAZ-TEAD to shape expression landscapes during tumorigenesis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Martin, C. P., Bang, S., Sullivan, W. B., Gomes, J. R., Smith, M. D., Fan, C., Edwards, A. C., Der, C. J., Liu, Y., Morris, J. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Restoration of p53 activity in pancreatic cancer cells expands transcriptional regulation by reorganizing TEAD binding at newly accessible enhancer landscapes, resulting in distinct TEAD-driven gene regulatory networks when p53 is inactivated versus engaged.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The tumor suppressor p53 is the most frequently disabled gene in cancer, where its inactivation promotes the transition between transcriptionally distinct pre-malignant and malignant disease. How p53 action shapes these stage-dependent expression landscapes is poorly understood. Using a mouse model where p53 function is restored in advanced pancreatic cancer, we demonstrate that p53 dependent gene expression is determined by regulation of YAP/TAZ-TEAD transcriptional complexes at multiple levels of molecular control. Restoration of p53 activity in pancreatic cancer cells expands transcriptional regulation by reorganizing TEAD binding at newly accessible enhancer landscapes, resulting in distinct TEAD-driven gene regulatory networks when p53 is inactivated versus engaged. Furthermore, p53 potentiates expression of newly licensed TEAD targets by increasing TAZ levels in response to the actin remodeling associated with cell cycle arrest and senescence. Thus, p53 activity controls the qualitative and quantitative output of YAP/TAZ-TEAD, including secretory programs involved in tumor-immune communication and remodeling of the microenvironment. Our work nominates regulation of YAP/TAZ-TEAD via p53 as a mechanism that determines stage dependent transcriptional landscapes during stepwise tumorigenesis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.29.679387v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Adversarial erasing enhanced multiple instance learning (siMILe): Discriminative identification of oligomeric protein structures in single molecule localization microscopy</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hallgrimson, C. D., Li, Y. L., Cardoen, B., Lim, J., Wong, T. H., Khater, I. M., Nabi, I. R., Hamarneh, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present siMILe, a novel weakly-supervised machine learning method based on multiple instance learning (MIL), leveraging shape and network features of protein assemblies, to close the important gap in interpretable subcellular discovery.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-molecule localization microscopy (SMLM) achieves nanoscale imaging of complex protein structures in the cell. However, the ability to capture structural variability across cell conditions (e.g., cell lines, gene expression, or treatment) from 3D point cloud SMLM data remains limited. We present siMILe, a novel weakly-supervised machine learning method based on multiple instance learning (MIL), leveraging shape and network features of protein assemblies, to close this important gap in interpretable subcellular discovery. siMILe identifies condition-specific changes in protein structures, without requiring structure-level supervision, and improves structure classification by extending embedded instance selection (MILES) through adversarial erasing and a symmetric classifier. siMILe is validated on simulated SMLM data and by detecting caveolae from caveolin-1 (Cav1) labeled PC3 prostate cancer cells differentially expressing cavin-1. In PC3-CAVIN1 cells dually labeled for Cav1 and cavin-1, cavin-1 closely associates with siMILe-identified caveolae, to a lesser extent with higher-order non-caveolar Cav1 scaffolds, but not with base Cav1 oligomers that correspond to 8S complexes, supporting a role for progressive cavin-1 interaction in 8S complex oligomerization. These results highlight siMILe's potential to identify differential molecular structures in distinct cell conditions. siMILe extends the SuperResNET SMLM software platform with the ability to detect interpretable structural differences across conditions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.29.679377v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Robust metabolomics data normalization across scales and experimental designs</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Vynck, M., Vangeenderhuysen, P., De Paepe, E., Nawrot, T., Plekhova, V., Vanhaecke, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce rLOESS, rGAM and tGAM, three robust normalization methods that improve resistance to outliers by downweighting or accommodating them, and improve replicate concordance, and reduced batch effects.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Metabolomics studies employing liquid chromatography-mass spectrometry are affected by signal drift and batch effects, introducing technical variance that impedes biological knowledge discovery. Quality control (QC) sample-based normalization strategies are widely implemented but remain vulnerable to outliers, thereby reducing normalization performance. We introduce rLOESS, rGAM and tGAM, three robust normalization methods that improve resistance to outliers by downweighting or accommodating them. Leveraging additive models, the rGAM and tGAM methods allow flexible non-linear modeling, differential sample weighting, and data-driven QC representativeness evaluation. Implementations of these methods are gathered in the Metanorm R package, integrating robust normalization with visualization for performance verification, while supporting efficient parallel processing. Spanning in silico and experimental datasets, the robust methods, relative to existing strategies, consistently demonstrated a reduction in false positive and false negative differentially abundant metabolites, improved replicate concordance, and reduced batch effects. Metanorm is versatile, supporting normalization in metabolomics studies across scales and experimental setups.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.30.679445v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Large language models forecast patient health trajectories enabling digital twins.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nikita Makarov, Maria Bordukova, Papichaya Quengdaeng, Daniel Garger, Raul Rodriguez-Esteban, Fabian Schmich, Michael P Menden</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed the Digital Twin-Generative Pretrained Transformer (DT-GPT), extending LLM-based forecasting solutions to clinical trajectory prediction.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Generative artificial intelligence is revolutionizing digital twin development, enabling virtual patient representations that predict health trajectories, with large language models (LLMs) showcasing untapped clinical forecasting potential. We developed the Digital Twin-Generative Pretrained Transformer (DT-GPT), extending LLM-based forecasting solutions to clinical trajectory prediction. DT-GPT leverages electronic health records without requiring data imputation or normalization and overcomes real-world data challenges such as missingness, noise, and limited sample sizes. Benchmarking on non-small cell lung cancer, intensive care unit, and Alzheimer's disease datasets, DT-GPT outperformed state-of-the-art machine learning models, reducing the scaled mean absolute error by 3.4%, 1.3% and 1.8%, respectively. It maintained distributions and cross-correlations of clinical variables, and demonstrated explainability through a human-interpretable interface. Additionally, DT-GPT's ability to perform zero-shot forecasting highlights potential advantages of LLMs as clinical forecasting platforms, proposing a path towards digital twin applications in clinical trials, treatment selection, and adverse event mitigation.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41034564/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Bioinformatics and machine learning reveal novel prognostic biomarkers in head and neck squamous cell carcinoma.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Amir Ali Judaki, Mohammad Shirinpoor, Masoumeh Farahani, Tahmineh Aldaghi, Afsaneh Arefi-Oskouie, Elham Nazari</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A deep learning algorithm and bioinformatics analysis of HNSCC biomarkers for cancer diagnosis and prognosis .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Head and neck squamous cell carcinoma (HNSCC), the seventh most common cancer worldwide, has become more closely linked to poor lifestyle habits. Despite improvements in cancer treatment approaches, patients with stage I-II HNSCC have a 70-90% 5-year survival rate, and for patients with advanced stages III-IV, this rate falls to about 40%. This controversy is all about the heterogeneity of HNSCC. Finding diagnosis and prognosis biomarkers has the potential to make significant improvements in the life expectancy and overall health of these patients. The combination of bioinformatics and machine learning has facilitated the finding of the best markers for HNSCC. In this regard, RNA expression data were obtained to identify genes that were expressed differently (DEGs) and utilize a deep learning algorithm to identify genes that exhibited significant variability. In addition, correlations between clinical data and DEGs, the building of a Receiver Operating Characteristic (ROC) curve, and the prediction of tumor-infiltrating immune cells were analyzed. Deep learning analysis identified diagnostic and prognostic biomarkers strongly associated with carcinogenesis, such as KRT33B, KRTAP3-3, C14orf34, and ACADM. In addition, after analyzing the ROC curve, it was found that the combination of ACADM, KRT33B, and C14orf34 is the most practical combination of diagnostic markers. This combination achieved sensitivity, specificity, and Area Under the Curve (AUC) values of 0.92, 0.86, and 0.93, respectively.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41028529/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Pan-Cancer Analyses Refine the Single-Cell Portrait of Tumor-Infiltrating Dendritic Cells.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ma T, Chu X, Wang J, Li X, Zhang Y, Tong D, Xu W, Dang G, Qi L, Miao Y, Zhang Z, Cheng S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comprehensive atlas of tumor-infiltrating dendritic cells and a machine learning model to guide the development of antitumor DC vaccination.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Dendritic cells (DC) are pivotal orchestrators of antitumor immunity. DC-based antitumor treatments are being actively developed, but effective clinical responses have not yet been achieved. Further exploration of DC heterogeneity in the tumor microenvironment and across cancer types could provide insights for developing DC-based immunotherapies. In this study, we integrated single-cell RNA sequencing data of DCs from more than 2,500 samples across 33 cancer types and established a comprehensive blueprint of human DCs. Several rare subsets of DCs infiltrated the tumors, including AXL+SIGLEC6+ DCs and Langerhans cell-like DCs, and displayed functional potentials marked with distinct transcriptomic characteristics. Computational analyses demonstrated that the Langerhans cell-like subset could be an additional cellular origin of tumor-enriched LAMP3+ DCs and that distinct cellular origins are associated with the pleiotropic functional potentials of LAMP3+ DCs. Furthermore, this DC atlas enabled the development of a machine learning model to guide DC annotation for subsequent single-cell analysis and prioritization of a valuable target for enhancing antitumor DC vaccination. This integrative resource provides a panoramic view to unravel the complexity of tumor-infiltrating DCs and offers valuable insights for developing therapies targeting DCs.The comprehensive dissection of tumor-infiltrating dendritic cells redefined cell subsets with different regulations, tissue preferences, and functional potentials and provided an atlas as a rich resource with promising applications. This article is part of a special series: Driving Cancer Discoveries with Computational Research, Data Science, and Machine Learning/AI.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40742316/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AI-directed gene fusing prolongs the evolutionary half-life of synthetic gene circuits. </h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Menuhin-Gruman I, Arbel-Groissman M, Naki D, Bergman S, Tuller T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a gene fusion strategy that links a gene of interest (GOI) to an essential endogenous gene (EG), with a "leaky" stop codon in between.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Evolutionary instability is a persistent challenge in synthetic biology, often leading to the loss of heterologous gene expression over time. Here, we present STABLES, a gene fusion strategy that links a gene of interest (GOI) to an essential endogenous gene (EG), with a "leaky" stop codon in between. This ensures both selective pressure against deleterious mutations and the high expression of the GOI. By leveraging a machine learning framework, we predict optimal GOI-EG pairs on the basis of bioinformatic and biophysical features, identify linkers likely to minimize protein misfolding, and optimize DNA sequences for stability and expression. Experimental validation in Saccharomyces cerevisiae demonstrated substantial improvements in stability and productivity for fluorescent proteins and human proinsulin. The results highlight a scalable, adaptable, and organism-agnostic method to enhance the evolutionary stability of engineered strains, with broad implications for industrial biotechnology and synthetic biology.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41032600/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Challenges and Opportunities in Analyzing Cancer-Associated Microbiomes.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chia M, Pop M, Salzberg SL, Nagarajan N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We explore the key computational challenges and opportunities in analyzing cancer-associated microbiomes (in tumor/normal tissues and other body sites, e.g., gut, oral, and skin), focusing on sequencing-driven strategies and associated considerations for taxonomic and functional characterization.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The study of cancer-associated microbiomes has gained significant attention in recent years, spurred by advances in high-throughput sequencing and metagenomic analysis. Microbiome research holds promise for identifying noninvasive biomarkers and possibly new paradigms for cancer treatment. In this review, we explore the key computational challenges and opportunities in analyzing cancer-associated microbiomes (in tumor/normal tissues and other body sites, e.g., gut, oral, and skin), focusing on sequencing-driven strategies and associated considerations for taxonomic and functional characterization. The discussion covers the strengths and limitations of current analysis tools for identifying contamination, determining compositional bias, and resolving species and strains, as well as the statistical, metabolic, and network inferences that are essential to uncover host-microbiome interactions. Several key considerations are required to guide the choice of databases used for metagenomic analysis in such studies. Recent advances in spatial and single-cell technologies have provided insights into cancer-associated microbiomes, and Artificial Intelligence-driven protein function prediction might enable rapid advances in this field. Finally, we provide a perspective on how the field can evolve to manage the ever-growing size of datasets and generate robust and testable hypotheses. This article is part of a special series: Driving Cancer Discoveries with Computational Research, Data Science, and Machine Learning/AI .</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40794770/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

