<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Thursday, 17 July 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Uncovering the Genetic Blueprint of the UAE's National Tree: Genomic Evidence to Guide Prosopis cineraria (L.) Druce Conservation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gkanogiannis, A., Almansoori, S. R. O., Kabshawi, M., Shahid, M., Almansoori, S., Rahman, H., Lopez-Lavalle, A. B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the first comprehensive population genomics study of Ghaf, based on whole-genome re-sequencing of 204 individual trees collected across the UAE.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Prosopis cineraria (L.) Druce is a keystone tree species of the arid and semi-arid regions of the West and South Asia, with critical ecological, cultural, and conservation significance. In the United Arab Emirates (UAE) and other regions of the Arabian Peninsula this beneficial tree is called Ghaf. Despite its importance, genomic resources and population-level diversity data for the tree remain limited. Here, we present the first comprehensive population genomics study of Ghaf, based on whole-genome re-sequencing of 204 individual trees collected across the UAE. Following Single Nucleotide Polymorphism (SNP) discovery and stringent filtering, we analyzed 57,183 high-quality LD-pruned SNPs to assess population structure, diversity, and gene flow. Principal component analysis (PCA), sparse non-negative matrix factorization (sNMF), and discriminant analysis of principal components (DAPC) revealed four well-defined genetic clusters, broadly corresponding to geographic origins. Genetic diversity varied significantly among groups, with observed heterozygosity (Ho), inbreeding coefficients (F), and nucleotide diversity ({pi}) showing strong population-specific trends. Genome-wide Fixation index FST scans identified multiple highly differentiated genomic regions, enriched for genes involved in stress response, transport, and signaling. Functional enrichment using Gene Ontology (GO), Kyoto Encyclopedia of Genes and Genomes (KEGG), and Pfam annotations indicated overrepresentation of protein kinase activity, ATP binding, and hormone signaling pathways. TreeMix analysis revealed gene flow into one of the genetic clusters from both others, suggesting historical admixture and geographic connectivity. To support data interpretation and stakeholder engagement, we developed two web-based tools for interactive visualization of sample geolocation and genetic structure. This work provides foundational insights into the genetic landscape of P. cineraria (L.) Druce, supporting conservation planning and long-term monitoring in the region.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.15.663052v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PromoterAtlas: decoding regulatory sequences across Gammaproteobacteria using a transformer model</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Coppens, L., Ledesma-Amaro, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present PromoterAtlas, a 1.8M parameter transformer model trained on 9M regulatory sequences from 3,371 gammaproteobacterial species that demonstrates recognition of various regulatory elements across different species, including ribosomal binding sites, various types of bacterial promoters, transcription factor binding</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent advances in deep learning, particularly transformer architectures, have improved computational approaches for biological sequence analysis. Despite these advances, computational models for bacterial promoter prediction have remained limited by small datasets, species-specific training, and binary classification approaches rather than comprehensive annotation frameworks. We present PromoterAtlas, a 1.8M parameter transformer model trained on 9M regulatory sequences from 3,371 gammaproteobacterial species. The model demonstrates recognition of various regulatory elements across different species, including ribosomal binding sites, various types of bacterial promoters, transcription factor binding sites, and terminators. Using this model, we developed a whole-genome promoter annotation tool for Gammaproteobacteria, with various levels of validation that support the predictions of promoters associated with different sigma ({sigma}) factors. Furthermore, we show that the model embeddings encode cross-species evolutionary relationships, clustering promoters by {sigma} factor identity rather than species-specific sequence features. Finally, we show that model embeddings encode regulatory sequence information that enables effective prediction of transcription and translation levels. PromoterAtlas can contribute to our understanding of and ability to engineer bacterial regulatory sequences with potential applications in bacterial biology, synthetic biology, and comparative genomics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.11.664312v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Regionalization of gene expression and cell types in the silk gland of the pantry moth Plodia interpunctella</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Alqassar, J. D., Biot, M., Eccles, L. E., Stoppel, W. L., Martin, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We characterized the silk gland from the pantry moth Plodia interpunctella and identified sharp cellular boundaries between two secretory tissues, and establish the silk diversity of lepidopteran silks.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Lepidopteran silk glands include a posterior silk gland (PSG) that secretes core fibers and a middle silk gland (MSG) that secretes adhesive sericins. While well-studied in the silkworm (Bombyx mori), little is known about the gene expression profiles underlying the diversity of lepidopteran silks. Here we characterized the silk gland from the pantry moth Plodia interpunctella using a combination of quantitative and spatial assays. RNA-seq analyses of differential gene expression between the MSG and PSG depict the transcriptomic divergence between these two secretory tissues. In the PSG, high expression of fibroin genes FibL and FibH were detected, whereas MSG samples were dominated by transcripts encoding major sericins -- bioadhesive proteins that form the coating of the silk fiber. Hybridization Chain Reaction (HCR) mRNA profiling revealed sharp cellular boundaries within the silk gland: PSG cells exclusively expressed fibroins, while MSG comprised two compartments each expressing different combinations of sericins. Our findings corroborate the conserved organization of lepidopteran silk glands into specialized secretory subdivisions, and establish Plodia as a promising comparative model for studying silk diversity. This work provides a foundation for future research into the cellular and evolutionary basis of silk production across Lepidoptera.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.11.664249v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Isoform-specific single-cell perturb-seq reveals distinct functions of alternative promoters in drug response</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> King, H. E., Vieira, H. G. S., Sterne-Weiler, T., Weatheritt, R. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that CRISPR-dCas9-based screens exhibit widespread promoter specificity, with untargeted promoters often showing compensatory upregulation to maintain gene expression.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> CRISPR interference (CRISPRi) screens have emerged as powerful tools for dissecting gene function, yet their application to genes with multiple promoters, which comprise over 60% of human genes, remains poorly understood. Here, we demonstrate that CRISPR-dCas9-based screens exhibit widespread promoter specificity, with untargeted promoters often showing compensatory upregulation to maintain gene expression. Leveraging this selective targeting of individual promoters within the same gene, we developed isoform-specific single-cell Perturb-Seq to systematically analyse alternative promoter function. Our analysis revealed that alternative promoters in 48.3% of targeted genes drive distinct transcriptional programs. This suggests that promoter selection represents a fundamental mechanism for generating cellular diversity rather than mere transcriptional redundancy. In breast cancer models, this promoter-specific targeting revealed differential effects on drug sensitivity, where distinct estrogen receptor (ESR1) promoters showed opposing influences on tamoxifen response and patient survival. These findings demonstrate the necessity of promoter-level analysis in functional genomics and suggest new strategies for therapeutic intervention through promoter-specific targeting.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.14.664827v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Distribution Patterns of rRNA Copy Number Repeats in Prokaryotic Genomes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Williamson, M. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We compare the distribution of 16S ribosomal RNA gene copies across prokaryotic genomes using a metric-normalized range-that allows for comparisons between copy number and phylogenetic signal.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Unlike tandem repeats in eukaryotes, ribosomal RNA (rRNA) operons across prokaryotic genomes are widely distributed. Here, I examined the distribution of 16S ribosomal RNA gene copies using all entries from the Ribosomal RNA Operon Copy Number Database with a copy number of 2 or greater, using a metric-normalized range-that allows for comparisons between copy number. Normalized range varied across rRNA copy number, with the greatest distribution of rRNA gene copies in prokaryotic accessions with a copy number of 5. There was a significant phylogenetic signal for both bacteria and archaea (p<0.05). Archaea had higher normalized range than bacteria, even when comparing bacteria with comparable copy number (n=2-5) (p<0.05). Normalized range varied across bacterial phyla (p<0.05), with Cyanobacteria having the largest normalized range and Chlamydiae the smallest. Furthermore, normalized range was predicted by copy number, tRNA number, and pseudo-gene number, after correcting for phylum. When restricted to low bacterial copy numbers (n=2-5), normalized range was predicted by genome size, GC content, tRNA number, coding gene number, and pseudo-gene number. For archaea, normalized range was predicted by genome size, GC content, and tRNA number. Comparisons of bacteria and archaea suggest different mechanisms for rRNA copy distribution and maintenance across genomes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.15.664977v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Alpha satellite RNA marks the perinucleolar compartment and represses ribosomal RNA expression in naïve human embryonic stem cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mittal, K., Ataei, L., Ramalho-Santos, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We found a remarkable enrichment for RNA transcribed from alpha satellite repeat (ALR) in the chromatin fraction of naive and primed human embryonic stem cells, as well as fibroblasts, and show a role for ALR RNA in nuclear compartmentalization and tuning rRNA synthesis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> While most newly synthesized RNA is exported to the cytoplasm, a portion of non-coding RNA is retained in the nucleus and remains highly associated with chromatin. The strong binding of this RNA fraction to insoluble chromatin impairs its recovery in standard transcriptomic studies. Therefore, the landscape and potential functions of chromatin-associated RNAs are poorly understood. Recent studies indicate that chromatin-associated transcripts can have regulatory roles, particularly during mammalian development. Here we compare the dynamics of cytoplasmic versus chromatin-bound transcriptomes of naive and primed human embryonic stem cells (hESCs), as well as fibroblasts. We found a remarkable enrichment for RNA transcribed from alpha satellite repeat (ALR) in the chromatin fraction of naive hESCs, compared to primed hESCs. The co-localization and interaction of ALR RNA with Polypyrimidine Tract Binding Protein 1 (PTBP1) protein indicate that ALR RNA foci mark the perinucleolar compartment (PNC), a nuclear sub-compartment previously thought to be exclusive to cancer cells. Knockdown of ALR RNA leads to dispersion of PTBP1 foci, up-regulation of ribosomal RNA and global hypertranscription in naive hESCs. In contrast, loss of PTBP1 does not disturb ALR RNA localization, indicating that ALR is upstream in the hierarchy of organization of the PNC in hESCs. These results reveal a role for ALR RNA in nuclear compartmentalization and tuning rRNA synthesis in naive hESCs. Moreover, this study opens new avenues to dissect the function of ALR RNA and the PNC in cancer contexts.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.11.664446v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Helitrons are enriched in lichenized fungi with long generation lengths and small distribution sizes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Paulsen, J., Sharrett, S. T., Mumey, D., Larsen, E. M., Nguyen, N. K., Lendemer, J., Calabria, L. M., Hoffman, J. R., Magori, K., Allen, J. L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of mobile genetic elements (MGEs) in lichenized fungi, which form obligate mutualistic symbioses with algae or cyanobacteria, and the relationships between species traits (i.e., dominant reproductive mode, distribution size, and generation length) and the abundance and</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mobile genetic elements (MGEs) have the potential to drive genome evolution by introducing mutations and causing structural instability and chromosomal rearrangements, particularly under conditions like environmental or genetic stress. In this study, we generated 18 new long-read based reference genomes for lichenized fungi, which form obligate mutualistic symbioses with algae or cyanobacteria. We used the new genomes and 10 publically available genomes to investigate the relationships between species traits (i.e., dominant reproductive mode, distribution size, and generation length) and the abundance and spatial distribution of MGEs using a phylogenetic comparative framework. We found that species with smaller distribution sizes and longer generation lengths had a higher genomic DNA transposon load. Specifically, their genomes were enriched with Rolling Circle transposons, which contradicts previous research that has identified high proportions of retrotransposons in rare species. Disproportionate distributions of MGEs in rare and range-restricted species may disrupt genomic stability, decrease fitness, and be reflective of species experiencing a greater degree of stress. Conversely, greater MGE activity may be an important source of novel genetic diversity in isolated populations with limited gene flow. Further research is needed to understand the potential mechanisms driving MGE proliferation in rare species genomes, and if MGE content is predictive of increased extinction risk.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.11.664412v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Programmable translational inhibition by a molecular glue-oligonucleotide conjugate</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, S., Kovalski, J. R., Zapatero-Belinchon, F. J., Bennett, M., Kuzuoglu-Öztürk, D., Li, Q., Stevenson, E., Liu, J., Krogan, N. J., Ott, M., Swaney, D. L., Ruggero, D., Lou, K., Shokat, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that RocASO is a versatile platform for programmable translational inhibition with therapeutic potential.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Selective inhibition of mRNA translation is a promising strategy for modulating the activity of disease-associated genes, yet achieving both high potency and specificity remains challenging. Rocaglamide A (RocA), a molecular glue, inhibits translation by clamping eIF4A onto polypurine motifs found in many transcripts, thereby limiting RocA's specificity. Here, we developed RocASO, a chemical conjugate that links RocA to an antisense oligonucleotide (ASO) capable of base-pairing with defined mRNA sequences, thus directing RocA's clamping mechanism to chosen targets and enhancing overall potency and specificity. We show that RocASOs are compatible with various types of ASO modalities, including gapmers that induce the degradation of target RNAs. RocASOs were designed to effectively knock down endogenous genes (PTGES3, HSPA1B) and SARS-CoV-2 viral RNA, the latter conferring potent antiviral activity in cells. These findings establish RocASO as a versatile platform for programmable translational inhibition with therapeutic potential.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.15.664547v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genomic and epidemiologic characteristics of SARS-CoV-2 persistent infections in California, January 2021 - July 2023</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bell, J. M., Elder, J., Ryder, R., Smith, E. A., Scribner, M., Gilliam, S., Borthwick, D., Crumpler, M., Skarbinski, J., Morales, C., Wadford, D. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified 69 SARS-CoV-2 persistent infection cases in California and identified a 400-day B.1.429 infection that demonstrates substantial intra-host evolution, and a BA.5.11 persistent infection revealing apparent competition between two intra-Host viral subpopulations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Novel SARS-CoV-2 variants demonstrating considerable intra-host evolution emerged throughout the pandemic. The persistent infections thought to give rise to these variants, however, have been difficult to identify at scale. This study sought to detect and characterize persistent infection cases in California using routine epidemiologic and genomic surveillance data. We identified 69 persistent infection cases with collection dates between January 2021 and July 2023 ranging from 21 to 400 days in duration, with an average of 44 days. Significant differences were identified in age distribution, sex, hospitalizations, and deaths between persistent infection cases and all sequenced California SARS-CoV-2 cases. Underlying health conditions were identified for the majority of cases with available medical records. The mutations found in these cases were suggestive of positive selection in the Spike receptor binding domain and convergent evolution toward immune evasion and residues observed in previous persistent infections. We describe a 400-day B.1.429 infection that demonstrates substantial intra-host evolution, and a BA.5.11 persistent infection revealing apparent competition between two intra-host viral subpopulations. By establishing a framework for detecting persistent infections, this study lays the groundwork for other public health organizations to monitor and investigate highly divergent SARS-CoV-2 viruses.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.14.664650v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Occupancy estimation of wild species in a palm oil plantation using unstructured data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> van Strien, A. J., Meijaard, E., Nardiyono,, Suief, S., Zaini, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We used occupancy models to estimate what proportion of spatial units (both non-natural and forest) a given species occupied annually in a large oil palm plantation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In 2020-2024, plantation workers at a large oil palm plantation in West Kalimantan recorded sightings of wild species of several species groups. They did not use a standardized field method and produced no comprehensive reports, but recorded only one or a few species per visit. Such unstructured data are generally viewed as challenging for statistical analysis. Here, we used occupancy models to estimate what proportion of spatial units (both non-natural and forest) a given species occupied annually in the plantation. Models with different covariates were tested for species for which most data were available: 13 birds, 7 mammals, among which the iconic Orangutan Pongo pygmaeus, and 3 reptiles. For each species, the model which best fitted the data was selected. Two shortcomings in our data complicated the analyses. First, occupancy models require detection and non-detection records, but because no fixed species lists were used, there is no unique way to generate non-detections. We generated non-detection records in several ways and ran the models again to evaluate the consequences for occupancy estimates. Second, imbalances in sampling may occur because of a lack of sampling design to select study sites. Most concerning are sites were surveyed only once in 2020-2024. We ran the models without those sites to examine whether results were different. Although the shortcomings mentioned turned out not to distort our results, the occupancy estimates were imprecise for many study species because of low detection rates, and extra efforts are needed to improve that.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.11.664290v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Post-Domestication selection of MKK3 Shaped Seed Dormancy and End-Use Traits in Barley</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jorgensen, M. E., Vequaud, D., Wang, Y., Andersen, C. B., Bayer, M., Box, A., Braune, K., Cai, Y., Chen, F., Antonio Cuesta-Seijo, J., Dong, H., Fincher, G. B., Gojkovic, Z., Huang, Z., Jaegle, B., Kale, S. M., Krsticevic, F., Roux, P.-M. L., Lozier, A., Lu, Q., Mascher, M., Murozuka, E., Nakamura, S., Simmelsgaard, M. U., Pedas, P. R., Pin, P., Sato, K., Spannagl, M., Rasmussen, M. W., Russell, J., Schreiber, M., Thomsen, H. C., Tulloch, S., Thomsen, N. W., Voss, C., Skadhauge, B., Stein, N., Waugh, R., Willerslev, E., Dockter, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the Mitogen-Activated Protein Kinase (MAPK) pathway regulates dormancy primarily through a mosaic of locus haplotypes comprising copy-number variation and inherent kinase activity. We provide evidence supporting the historical selection of specific MKK3 haplotypes that shape dormancy levels</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Anthropogenic selection of grain traits such as dormancy has shaped the developmental trajectories of crop plants (1). In cereals, shortening dormancy provides rapid and even post-harvest germination, but increases the risk of weather-induced pre-harvest sprouting (PHS) with harvest losses estimated at beyond 1 billion USD a year (2, 3, 4). Our understanding of how, why, when and where diversification of cereal dormancy arose is fragmentary. Here, we show in the founder cereal crop barley (Hordeum vulgare) that the Mitogen-Activated Protein Kinase (MAPK) pathway regulates dormancy primarily through a mosaic of locus haplotypes comprising copy-number variation and inherent kinase activity of Mitogen-activated protein kinase kinase 3 (MKK3). We provide evidence supporting the historical selection of specific MKK3 haplotypes that shape dormancy levels according to changing climatic pressures. Understanding the regulatory landscape of MKK3 provides a genetic framework to balance short grain dormancy and PHS-avoidance during a period of rapid climate change.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.11.664137v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Engineering fluoroacetate dehalogenase by growth-based selections to degrade non-natural organofluorides</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jansen, S. C., van Beers, P., Mayer, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the first large-scale engineering campaign of fluoroacetate dehalogenases (FAcD) and introduce an operationally simple selection platform that paves the way toward adapting these enzymes for the sustainable degradation of contaminating organofluorides.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The widespread use of organofluorides in modern society has inadvertently led to the bioaccumulation of harmful pollutants, most prominently per- and polyfluorinated alkyl substances (PFAS). In principle, tailored biocatalysts able to cleave C-F bonds represent an attractive strategy to combat this (emerging) environmental crisis. However, Nature is largely impartial to C-F bonds, with fluoroacetate dehalogenases (FAcDs) standing out as a notable exception, catalyzing the hydrolysis of single C-F bonds in fluoroacetate at high turnover rates. To expand the substrate scope of FAcDs and harness its catalytic prowess for non-natural organofluorides, we designed and applied a robust growth-based selection strategy for large-scale FAcD engineering. Specifically, we demonstrate that FAcD-catalyzed C-F bond cleavage of (natural and) synthetic organofluorides generates metabolizable carbon sources for bacteria, enabling in vivo enrichment of active FAcD variants. By forcing populations expressing diverse FAcD-libraries to utilize various organofluorides as sole carbon source, we elicited a broad panel of FAcD variants that displayed improved activities and drastically altered substrate profiles. In these efforts, we also identified a previously overlooked inhibition pathway, which largely impedes the conversion of gem-difluoride compounds. Overall, our study presents the first large-scale engineering campaign of FAcDs and introduces an operationally simple selection platform that paves the way toward adapting these enzymes for the sustainable degradation of contaminating organofluorides.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.16.665192v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CF2H: a Cell-Free Two-Hybrid platform for rapid protein binder screening</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Capin, J., Mayonove, P., DeVisch, A., Becher, A., Ngo, G., Courbet, A. P., Ragotte, R. J., Cohen-Gonsaud, M., Espeut, J., Bonnet, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that cell-free two-hybrid method for detecting high-affinity protein-protein interactions (PPI) that does not require cloning, protein purification nor high-end laboratory equipment.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein binders that detect, activate, inhibit or otherwise modulate their targets are pivotal for biomedical applications. With the increasing accuracy and accessibility of de novo protein design, faster and cheaper experimental screening methods would democratize and accelerate the identification of high-affinity binders. Here we present Cell-Free Two-Hybrid (CF2H), a rapid and sensitive method for detecting high-affinity protein-protein interactions (PPI) that does not require cloning, protein purification nor high-end laboratory equipment. CF2H uses a dimerization-activated DNA binding domain (DBD) fused to prey and bait proteins to trigger transcription upon protein-protein interaction. We demonstrate that CF2H enables the detection of interactions between various types of target and binder proteins such as single-domain antibodies, DARPins and de novo designed binders. We benchmark CF2H as a screening platform by validating previously reported binders for Mdm2 and discover high-affinity binders targeting the checkpoint inhibitor PD-L1 in less than 24 hours. Finally, we show that CF2H can be used to characterize small-molecules modulators of PPI and detect protein biomarkers, opening the door for a new class of cell-free biosensors.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.16.665152v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Stimuli-driven cyclical content exchange in a composite synthetic cell</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cooney, A., Zhang, W., Di Michele, L., Elani, Y., Matsuura, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a protein-free approach for dynamic content modulation in liposome-based synthetic cells using an internal gelation strategy for reversible membrane permeabilisation via osmotic swelling and shrinking, facilitating externally gated material exchange without reconstituted protein pores or electroporation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Engineering sophisticated behaviours in synthetic cells lacking complex biomolecular machinery remains a central challenge in synthetic biology. Here, we introduce a protein-free approach for dynamic content modulation in liposome-based synthetic cells using an internal gelation strategy. By crosslinking a polymer hydrogel within the lumen of giant vesicles and tethering it to the inner membrane leaflet, we create a composite architecture that enables controlled and reversible membrane permeabilisation via osmotic swelling and shrinking, facilitating externally gated material exchange without reconstituted protein pores or electroporation. Simultaneously, the hydrogel matrix affords control over membrane fluidity and the diffusion of cytoplasmic clients. We deploy the transport-regulation platform to construct a synthetic-cell bioreactor whereby reversible membrane permeabilisation enables content supplementation and fuels a biocatalytic reaction. The composite gel-GUV chassis provides an adaptive, robust and expandable solution for engineering increasingly modular and functional synthetic cellular systems. These findings may echo how primordial cells harnessed environmental fluctuations for content exchange through chemically distinct pathways.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.16.665112v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Simple high-throughput encoding of deep mutational scanning libraries by oligo-based Golden Gate assembly</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hanning, K. R., Walker, E. J., Beijerling, K., Irvine, E. B., Steel, J. J., Kelton, W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel library assembly approach for multi-mutant libraries that can be used to improve affinity maturation against cyclic peptide targets.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Control over mutational library diversity is an essential consideration when engineering proteins, but is often fraught with trade-offs between diversity, specificity, and affordability. Contemporary library assembly approaches often incorporate oligonucleotide pool synthesis to achieve affordable, precise mutagenesis; however, these oligos are often reliant on complex designs to facilitate downstream PCR and/or restriction digests. Direct hybridisation of oligo pools is an overlooked strategy to simplify mutagenesis, especially when paired with a type IIS restriction cloning approach. We validate this approach by designing, hybridising, and deep sequencing single and dual substitution CDR region parts derived from nanobody GA10. Assembly of these parts into a full-length nanobody CDS facilitated the phage display of variant libraries for affinity maturation against its cyclic peptide target. Variants identified through enrichment analysis were expressed in isolation and yielded improved affinities by more than 100-fold. Recent advances in machine learning have successfully inferred improved variants outside of screened library space, but require controlled, multi-mutant libraries. The library assembly approach outlined in this research is well-suited for such approaches.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.16.665225v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comprehensive Error Profiling of NovaSeq6000, NovaSeqX, and Salus Pro Using Overlapping Paired-End Reads</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yao, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present ErrorProfiler, a reference-free, paired-end overlap-based workflow for quantifying and characterizing base-call errors.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurate base calling is essential for ensuring the reliability of next-generation sequencing (NGS) data. However, current approaches to error profiling often rely on reference genomes or variant-calling pipelines, which can introduce platform-specific biases or overlook systematic sequencing errors. Here, we present ErrorProfiler, a reference-free, paired-end overlap-based workflow for quantifying and characterizing base-call errors. We applied this approach to profile three high-throughput sequencing platforms - NovaSeq6000, NovaSeqX, and Salus Pro - using Escherichia coli datasets. By analyzing mismatches within fully overlapping 150 bp read regions, ErrorProfiler effectively distinguished sequencing-specific errors from PCR or library preparation artifacts. NovaSeqX exhibited the lowest median sequencing error rate (0.0089%), with minimal cycle-dependent drift. Salus PRO followed (0.0124%), outperforming NovaSeq6000 (0.0185%). Base quality score calibration analysis revealed systematic underestimation of base-call accuracy across all platforms. Illumina binned Q-score models (Q11/Q25/Q37; Q12/Q24/Q40) masked substantial intra-bin variability, while Salus Pro continuous Q-score system offered finer granularity but still overestimated error probabilities in key bins. A previously unreported "upstream effect" was observed, where the empirical accuracy of a base-call was significantly influenced by the Q-score of its preceding base. This effect was most pronounced in high-Q bins and consistent across all platforms. Substitution error spectra revealed distinct platform-specific and sequence-context-dependent biases, driven by optical systems and detection chemistry. Our findings provide a rigorous, platform-agnostic framework for sequencing error analysis and quality assessment. ErrorProfiler is open-source and compatible with diverse sequencing platforms, offering a valuable tool for improving calibration, variant calling, and downstream bioinformatics workflows.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.15.665004v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Robust inference of cancer progression pathways using Conjunctive Bayesian Networks</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hosseini, S.-R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A robust framework for the inference of cancer progression pathways from cross-sectional genomic data .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cancer is an evolutionary disorder driven by stepwise accumulation of selectively advantageous mutations forming mutational pathways, characterization of which is essential for diagnosis, prognosis and treatment of cancer. Conjunctive Bayesian networks (CBN) are probabilistic graphical models that have enabled the inference of these pathways of cancer progression from genomic data. Previously, we showed that the CBN model can estimate the predictability of cancer evolution by reflecting the underlying fitness landscapes. However, the reliability of the inferred pathway probability distributions has not yet been ascertained. Hence, here I introduce the robust-CBN framework (R-CBN) to ensure a reliable inference of cancer progression pathways from cross-sectional genomic data. Using synthetic, simulated and real data, I rigorously compared R-CBN with previous versions of the CBN model, including CT-CBN, H-CBN, and B-CBN. The results indicate a superior robustness of the R-CBN model in various settings, while it is still able to largely retain predictability. Hence, R-CBN is ready to be used as a reliable framework to distill mechanistic insights from genomic data, while it also has the potential to serve as a building block that inspires further methodological innovations in the field.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.15.663924v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Auto-MedCalc: Automated Biomarkers Discovery and Risk Score Generation with AI Agents</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ding, S., Bhattacharya, S., Butte, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose Auto-MedCalc, a data-driven pipeline to automatically identify biomarkers and generate risk scores for diagnosis for diagnosis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Identifying biomarkers and generating risk scores are usually essential tasks in many biomedicine and clinical scenarios. However, this is a highly hypothesis-driven and experience-dependent process requiring extensive experiments as well. How to discover biomarkers from multi-modality, multi-source data and generate risk scores with higher precision motivates us to design the framework Auto-MedCalc, a data-driven pipeline to automatically identify biomarkers and generate risk scores for diagnosis. Auto-MedCalc is a multi-agent AI system with large language models and computational tools. Three representative studies of transplantation from the public ImmPort data portal are used for the experiment. Auto-MedCalc achieved 0.93, 0.88, and 0.88 ROC-AUC on the rejection prediction after kidney, liver, and heart transplants, surpassing the best expert-designed medical calculators by 50%, 4.7%, and 39%, respectively. Auto-MedCalc also validated the previously found human biomarkers and discovered new biomarkers, {beta}-Glucuronidase and Alpha-1-microglobulin, in heart transplants, which had not attracted much attention before. The superior performance from experiments shows that Auto-MedCalc can be used for both clinical applications and scientific discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.10.664265v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comparison of state-of-the-art error-correction coding for sequence-based DNA data storage</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gimpel, A. L., Remschak, A., Stark, W. J., Heckel, R., Grass, R. N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A wide range of codecs with vastly different error-correction approaches have been proposed and implemented for DNA data storage to date, but no studies have systematically benchmarked codec implementations to establish the current state-of-the-art in DNA data data storage.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> A wide range of codecs with vastly different error-correction approaches have been proposed and implemented for DNA data storage to date. However, while many codecs claim to provide superior performance, no studies have systematically benchmarked codec implementations to establish the current state-of-the-art in DNA data storage. In this study, we use standardized error scenarios - both in silico and in vitro - to compare the performance of six representative codecs from the literature. We find synthetic benchmarks commonly used in literature to be unsuitable indicators of codec performance, as our data shows that common experimental benchmarks fail to differentiate codecs under standardized conditions. Instead, we implement a comprehensive benchmark covering the major experimental parameters to assess codec performance under realistic DNA data storage conditions, while establishing important baselines for future codec development. Verifying our results with fair and standardized experiments, we demonstrate data storage at 43 EB g-1 using synthesis by material deposition and 13 EB g-1 using the more error prone electrochemical synthesis, employing only existing codecs from the literature. Besides closing in on the physical limits of DNA data storage, this study thus showcases the maturity of error-correction coding and defines its current state-of-the-art.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.11.664297v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

