<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Thursday, 19 June 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Divergence in a eukaryotic transcription factor's co-TF dependence involves multiple intrinsically disordered regions.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lindsey F Snyder, Emily M O'Brien, Jia Zhao, Jinye Liang, Baylee J Bruce, Yuning Zhang, Wei Zhu, Thomas H Cassier, Nicholas J Schnicker, Xu Zhou, Raluca Gordân, Bin Z He</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of the genetic basis and species variation in TF interdependence in the yeast phosphate starvation response .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Combinatorial control by transcription factors (TFs) is central to eukaryotic gene regulation, yet its mechanism, evolution, and regulatory impact are not well understood. Here we use natural variation in the yeast phosphate starvation (PHO) response to examine the genetic basis and species variation in TF interdependence. In Saccharomyces cerevisiae, the main TF Pho4 relies on the co-TF Pho2 to regulate ~28 genes, whereas in the related pathogen Candida glabrata, Pho4 has reduced Pho2 dependence and regulates ~70 genes. We found C. glabrata Pho4 (CgPho4) binds the same motif with 3-4 fold higher affinity. Machine learning and yeast one-hybrid assay identify two intrinsically disordered regions (IDRs) in CgPho4 that boost its activation domain's activity. In ScPho4, an IDR next to the DNA binding domain both allows for enhanced activity with Pho2 and inhibits activity without Pho2. This study reveals how IDR divergence drives TF interdependence evolution by influencing activation potential and autoinhibition.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40533454/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DeepRice6mA: A convolutional neural network approach for 6mA site prediction in the rice Genome.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hussam Alsharif</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce DeepRice6mA, a sophisticated comprehensive predictive tool for identifying rice 6mA sites, using a deep learning approach that incorporates ensemble strategies from one-hot encoding and 3-kmer feature embedding.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> As one of the most critical post-replication modifications, N6-methylation (6mA) at adenine residue plays an important role in a variety of biological functions. Existing computational methods for identifying 6mA sites across large genomic regions tend to fall short in either accuracy or computational efficiency. To address this, we introduce DeepRice6mA, a sophisticated comprehensive predictive tool for identifying rice 6mA sites, using a deep learning approach that incorporates ensemble strategies from one-hot encoding and 3-kmer feature embedding. The proposed model, labeled DeepRice6mA, reaches state-of-the-art results compared to current approaches, with 10-fold cross-validation scores of 98% for accuracy, 98% for sensitivity, 98% for specificity, a Matthew's correlation coefficient (MCC) of 0.96, and an area under the receiver operating characteristic curve (AUC) of 0.99. We anticipate that DeepRice6mA will significantly enhance our understanding of DNA methylation and its implications for biological processes and disease states.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40531834/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Effective data visualization strategies in untargeted metabolomics.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mildau K, Ehlers H, Meisenburg M, Del Pup E, Koetsier RA, Torres Ortega LR, de Jonge NF, Singh KS, Ferreira D, Othibeng K, Tugizimana F, Huber F, van der Hooft JJJ.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce data visualization to the untargeted metabolomics field as a topic worthy of its own dedicated research, and provide a primer on cutting-edge visualization research into data visualization for both researchers as well as developers active in metabolomics.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Covering: 2014 to 2023 for metabolomics, 2002 to 2023 for information visualizationLC-MS/MS-based untargeted metabolomics is a rapidly developing research field spawning increasing numbers of computational metabolomics tools assisting researchers with their complex data processing, analysis, and interpretation tasks. In this article, we review the entire untargeted metabolomics workflow from the perspective of information visualization, visual analytics and visual data integration. Data visualization is a crucial step at every stage of the metabolomics workflow, where it provides core components of data inspection, evaluation, and sharing capabilities. However, due to the large number of available data analysis tools and corresponding visualization components, it is hard for both users and developers to get an overview of what is already available and which tools are suitable for their analysis. In addition, there is little cross-pollination between the fields of data visualization and metabolomics, leaving visual tools to be designed in a secondary and mostly ad hoc fashion. With this review, we aim to bridge the gap between the fields of untargeted metabolomics and data visualization. First, we introduce data visualization to the untargeted metabolomics field as a topic worthy of its own dedicated research, and provide a primer on cutting-edge visualization research into data visualization for both researchers as well as developers active in metabolomics. We extend this primer with a discussion of best practices for data visualization as they have emerged from data visualization studies. Second, we provide a practical roadmap to the visual tool landscape and its use within the untargeted metabolomics field. Here, for several computational analysis stages within the untargeted metabolomics workflow, we provide an overview of commonly used visual strategies with practical examples. In this context, we will also outline promising areas for further research and development. We end the review with a set of recommendations for developers and users on how to make the best use of visualizations for more effective and transparent communication of results.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/39620439/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Monitoring of vaccine targets and interventions using global genome data: vaccines.watch</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> David, S., Abudahab, K., Couto, N., Daningrat, W. O. D., Yeats, C., Molloy, A., Ashton, P. M., Alikhan, N.-F., Aanensen, D. M., NIHR Global Health Research Unit on Genomics and enabling data for Surveillance of AMR</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present an interactive platform, vaccines.watch (https://vaccines.watch), which aims to support decision-making around vaccine formulations and roll-out by enabling interrogation of vaccine target diversity from global genome data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The expansion of pathogen genome sequencing into routine disease surveillance programmes is set to bring rapidly-growing volumes of increasingly structured data on a global scale. This has the potential to deliver exciting opportunities for accelerating vaccine development and monitoring. Here we present an interactive platform, vaccines.watch (https://vaccines.watch), which aims to support decision-making around vaccine formulations and roll-out by enabling interrogation of vaccine target diversity from global genome data. We have initially focused on targets included in existing or prospective multivalent polysaccharide-based vaccines for Streptococcus pneumoniae, Klebsiella pneumoniae (and related species) and Acinetobacter baumannii. The platform currently displays data for >100k high-quality genomes with geotemporal sampling information (post-2010), with new genomes assembled, analysed and incorporated on an ongoing basis (every 4 hours) as public data are newly deposited. Crucially, users can view vaccine target information in the broader context of genotypic variants (e.g. sequence types) and antimicrobial resistance markers. The platform also enables users to review the composite serotypes of pneumococcal vaccine formulations among the available genomes. For example, using data in vaccines.watch from 3 June 2025, we observed that serotypes included in the PCV13 and PCV21 formulations accounted for 36.2% (11,907/32,918) and 87.4% (28,764/32,918) of global public genomes, respectively. The platform also enables continuous review of the global genomic landscape of the included pathogens, enabling identification of gaps (e.g. in geographic coverage) that should be targeted with increased genomic surveillance. Indeed we demonstrate that substantial geographic gaps remain in the coverage of available genomes, with over half of countries contributing no genomes for each of the three pathogens. However, while caution in interpretation is important, as global representativeness of genome data grows, vaccines.watch is positioned to support different stages of the vaccine pipeline, from selection of target antigens to post-rollout monitoring of population changes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.13.659488v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CellOntologyMapper: Consensus mapping of cell type annotation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zeng, Z., Wang, X., Du, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present CellOntologyMapper, an automated framework that standardizes cell type annotations by intelligently mapping user-defined names to established Cell Ontology and Cell Taxonomy identifiers.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell RNA sequencing has revolutionized cellular biology, with atlases now encompassing over 100 million cells. However, researchers employ vastly different naming conventions when annotating cell types, creating a fragmented landscape that severely impedes data integration and comparative analysis. Here, we present CellOntologyMapper, an automated framework that standardizes cell type annotations by intelligently mapping user-defined names to established Cell Ontology and Cell Taxonomy identifiers. Our approach leverages advanced natural language processing, including sentence transformers and large language models, to interpret diverse naming conventions and resolve them to standardized ontological terms. The system handles complex challenges including abbreviated cell names, synonym resolution, and context-dependent interpretation. We built a comprehensive query system based on 19,381 cell type entries from established Cell Ontology and Cell Taxonomy databases, which organize into 24 biologically coherent clusters. Systematic validation across datasets spanning different scales from 17-cell-type lung studies to 91-cell-type immune atlases and complex developmental systems consistently demonstrated robust performance and high accuracy. CellOntologyMapper successfully resolved annotation challenges across conventional tissue studies, rare cell populations, and developmentally intricate datasets rich in abbreviated nomenclature. By providing automated, scalable annotation harmonization, our framework enables researchers to leverage existing single-cell datasets while ensuring compatibility with future atlas efforts. When incorporated into publications, CellOntologyMapper enhances the credibility and reference value of cell type annotations, representing a crucial step toward an integrated single-cell genomics ecosystem.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.10.658951v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Prediction of soil probiotics based on foundation model representation enhancement and stacked aggregation classifier</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kang, Q., Sun, H., Wang, Y., Fang, X., Li, Y., Zhang, Y., Wei, T., Yin, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a method to predict soil probiotics using genomic foundation models and a stacked aggregation classifier .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Soil probiotics are indispensable in agro-ecosystem, which enhances crop yield through nutrient solubilization, pathogen suppression, and soil structure improvement. However, reliable prediction methods for soil probiotics remain absent. In this study, we utilize genomic foundation models to generate representations from samples' sequences, and then, enhance them by deeply integrating domain-specific engineered features. The enhanced representations enable training a powerful classifier for a target task, instead of common parameter fine-tuning. Inspired by the stacking ensemble learning framework, we also design a stacked aggregation classifier. It predicts the label of a sample with only leveraging partial sequence segments from this sample, effectively addressing the challenges in processing long sequences. The proposed method is applied on prediction of soil probiotics and obtains 97.50% accuracy and 0.9807 AUC value on balanced and imbalanced test sets, respectively. Potential functional genes are revealed from the predicted probiotics, providing biologically insights for more related studies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.13.659431v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multiresolution Clustering of Genomic Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Turfah, A., Wen, X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a unified and principled clustering algorithm for multi-resolution cluster analysis of complex genomic data and show that it can uncover complex, nested biological relationships and reconcile discrepancies among clustering results.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cluster analysis is a widely used unsupervised learning technique in genomic data analysis, with critical applications such as inferring genetic population structures and annotating cell types from single-cell RNA-seq data. However, most existing clustering methods focus on identifying a single optimal partition while overlooking intrinsic relationships among the inferred clusters. Moreover, clustering results produced by different algorithms often appear inconsistent, and there is a lack of principled approaches to extract shared, biologically meaningful patterns across diverse clustering outputs. In this work, we introduce a computational framework that enables systematic exploration of multi-resolution clustering structures in genomic data, starting from an initial configuration generated by any available clustering algorithm. The proposed algorithm provides a unified and principled approach for uncovering complex, nested biological relationships and reconciling discrepancies among clustering results. We demonstrate the utility of our framework through comprehensive simulations and applications to both genetic and single-cell transcriptomic datasets, highlighting its ability to recover interpretable and reproducible clustering structures. Furthermore, we show that our multi-resolution cluster analysis of complex genomic data yields valuable insights into patterns of human population migration and cell differentiation trajectories.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.13.659529v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Oops: A Language for Formalizing Fallible Biological Protocols</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shang, M., Klavins, E., Bernstein, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Oops, a programming language for describing biological protocols along with how they can fail.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Formalizing protocols used in wetlab biological research as programs improves reproducibility by making protocols replicable and standardized. However, existing protocol languages have limited capacity for codifying error sources and standardizing error handling. When protocols inevitably go wrong, debugging must still be done by technicians, a process which is challenging for unfamiliar protocols and a scalability constraint in automated facilities. We introduce Oops, a programming language for describing biological protocols along with how they can fail. Oops makes expression of how errors affect outcomes concise through probabilistic semantics, an extensible abstraction of the lab, and metaprogramming. Given an outcome, Oops enables analyses of what errors occurred by mapping lab observations to enforced program conditions and applying existing probabilistic inference algorithms. We formalize a collection of molecular cloning protocols and present case studies demonstrating how Oops can explain errors and assess diagnostic capabilities.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.17.660245v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An index-based system for early alerts of potential zoonotic disease outbreaks</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Angelo, J., Abdala, L., Augusto, D. A., Krempser, E., Chame, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a dynamic alert system based on a Multi-Attribute Zoonotic Alert Index (Z-Alert) to enhance disease outbreak detection and response in wildlife.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The Information System on Wildlife Health (SISS-Geo) is a free platform designed for the real-time collection of georeferenced data on wildlife health and environmental conditions via mobile devices. It serves as a collaborative tool, enabling health professionals, researchers, environmental managers, and the general public to report information about wildlife health occurrences directly within the system. Since its launch in 2014, SISS-Geo has been successfully applied in supporting decision-making during significant wildlife health events in Brazil. In this paper, we introduce a dynamic alert system based on a Multi-Attribute Zoonotic Alert Index (Z-Alert) to enhance disease outbreak detection and response in wildlife. The proposed approach improves the current alert system of the SISS-Geo platform by integrating clustering techniques with multi-objective optimization. The clustering stage allows to group SISS-Geo records in a way that reflects both spatial and temporal variations, capturing relevant patterns for epidemiological surveillance. Each identified cluster is assigned a numerical alert index, the Z-Alert index, reflecting its criticality level, based on optimally weighted attributes such as the percentage of dead animals, temporal interval, and geographical spread. The system offers customization alert thresholds, allowing health managers to adapt the alert system to their specific needs. To validate the model, we used historical yellow fever case data from Brazils Ministry of Health, achieving over 94% accuracy in identifying critical clusters. By enabling the prioritization of prevention and investigation actions, the system strengthens the responsiveness of health managers, optimizing resource allocation and promoting smart economic strategies for outbreak containment.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.17.660271v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Immune pathways and perinatal environmental exposures contribute to epigenetic gestational age prediction and acceleration</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Eapen, A. A., Loveless, I. M., Pan, M., Liang, X., Straughen, J., Cassidy-Bushrow, A., Sitarik, A. R., Simmerman, N., Thompson, E. E., Kottyan, L., Ober, C., Johnson, C. C., Zoratti, E., Levin, A. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified 2,435 CpG sites associated with chronological GA at single CnG sites and regions, including HLA class II (HLA-DRB1, HLA-DQB1,HLA -DRB6) and IL10 signaling, which are also enriched in</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> DNA methylation (DNAm), capturing biological gestational age (GA) and epigenetic gestational age acceleration (EGAA), can be modified by environmental exposures. The Asthma&Allergy array is a new DNAm array developed with content focused on asthma and allergy loci. The association between content on the Asthma&Allergy array and chronological GA and EGAA has not been evaluated alone or in the context of perinatal exposures. We performed an epigenome-wide association study (EWAS) based on chronological GA at single CpG sites and regions. We further constructed a multi-CpG site methylation model to predict chronological GA in cord blood from 391 newborn children from a Detroit-based birth cohort. Associations between perinatal environmental factors with GA, epigenetic gestational age (EGA), and EGAA were assessed. We identified 2,435 CpG sites associated with chronological GA. HLA class II (HLA-DRB1,HLA-DQB1,HLA-DRB6) were the most significantly associated with chronological GA. Our multi-CpG site model attained predictive accuracy (cross-validated Pearson correlation=0.75) comparable to other EGA methods. Using genes implicated in region-based analyses (n=395 regions), the pathways most significantly enriched with chronological GA-associated CpGs included T helper 1(Th1) and 2(Th2) activation, macrophage classical activation, and IL10 signaling, which were also enriched in at least one of the other published epigenetic clocks. In multi-exposure models, prenatal indoor pet exposure and unplanned C-section were associated with EGA deceleration, while infant first-born status was associated with EGAA. Our findings highlight enrichment for T cell modulated pathways and antigen presentation as biological processes enriched in chronological GA, as well as novel perinatal factors that may impact EGAA.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.13.659090v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Salicylic Acid Engages Central Metabolic Regulators SnRK1 and TOR to Govern Immunity by Differential Phosphorylation of NPR1</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chen, Y., Withers, J., Chen, T., Karapetyan, S., Draken, J., Xiang, Y., Droege-Laser, W., Dong, X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that nonEXPRESSER of PR GENES 1 (NPR1), the central plant immune regulator of salicylic acid (SA)-mediated defense responses, is controlled by a cascade of posttranslational modifications (PTMs) involving two master nutrient-sensing kinases.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Immunity is a delicate balance between combating infection and preserving the metabolic functions vital for host survival. However, the mechanisms by which immune responses are coordinated with cellular metabolism remain largely unknown. Here, we show that NONEXPRESSER OF PR GENES 1 (NPR1), the central plant immune regulator of salicylic acid (SA)-mediated defense responses, is controlled by a cascade of posttranslational modifications (PTMs) involving two master nutrient-sensing kinases. Under normal growth conditions, TARGET OF RAPAMYCIN (TOR) inhibits NPR1 through phosphorylation at Ser 55/59. During defense responses, elevated SA enhances SNF1-RELATED KINASE 1 (SnRK1) activity, which in turn inhibits TOR signaling and phosphorylates NPR1 at Ser-557. This phosphorylation event activates NPR1 and facilitates its subsequent PTMs. Together, our results reveal an integral role of SA (the active metabolite of aspirin) in controlling central metabolic regulators SnRK1 and TOR to coordinate immune responses and growth through antagonistic modifications of NPR1.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.17.660129v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

