<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Tuesday, 14 October 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Massively parallel assay of human splice variants reveals cis-regulatory drivers of disease-associated and cell type-specific splicing regulation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Koplik, S. E., Yu, A. M., Shelby, M. R., Fonseca, G. C., Roco, C. M., Zhang, Y., Bogard, N., Sabo, A. K., Rosenberg, A. B., Linder, J., Seelig, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We develop the largest cell type-resolved, base-resolution atlas of splicing variant effects to date, providing a resource to support variant reclassification in clinical genomics and aid in the selection of therapeutic targets.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Splice-disrupting variants (SDVs) underlie many human diseases, yet systematic functional maps of their effects across cell types remain limited. We developed Cell-type Oriented Massively Parallel reporter Assay of Splicing Signatures (COMPASS) to measure splicing outcomes for 87,546 single and double variants across more than 1,700 genes in five human cell lines of diverse tissue origin. COMPASS targets disease relevant gene sets, including ACMG actionable genes and SFARI autism-associated genes, enabling systematic dissection of splicing impacts in health and disease. Our measurements reveal numerous SDVs, including ClinVar variants currently classified as variants of uncertain significance. Using prime editing, we validate variant effects in the genome for ClinVar variants in BIN1, a gene implicated in Alzheimer's disease, cancer, and cardiac pathology. Benchmarking our MPRA against state-of-the-art predictive models shows that while these approaches capture broad variant effects, important gaps remain. Our data reveal putative RNA-binding protein motifs whose disruption drives splicing changes, providing mechanistic insight into variant impact. Finally, cross-cell line comparisons reveal a distinct subset of variants that drive cell type-specific splicing programs. This study delivers the largest cell type-resolved, base-resolution atlas of splicing variant effects to date, providing a resource to support variant reclassification in clinical genomics and aid in the selection of therapeutic targets.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.12.681955v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">High-Throughput De Novo Protein Design Yields Novel Immunomodulatory Agonists</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Abedi, M., Exposit, M., Coventry, B., Mathew, D., Jain, S., Krishnakumar, A., Goreshnik, I., Gray-Gaillard, S., Lunn-Halbert, M., Yu, T.-Y., Glögl, M., Mitchell, U., Keshri, R., Chun, J. H., Ruohola-Baker, H., Wherry, E. J., Baker, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a high-throughput de-novo design approach to create novel cytokines by generating and fusing pairs of computationally designed receptor-binding domains.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cytokines regulate cell behavior by bringing together specific receptor subunits to trigger downstream signaling. Designed molecules that bring together non-natural receptor pairs could have novel signaling responses and cell specificities. We present a high-throughput de-novo design approach to create novel cytokines by generating and fusing pairs of computationally designed binders. By combining 33 designed receptor-binding domains, we generated over a thousand potential de novo designed "Novokines", of which 75 activated pSTAT signaling in peripheral blood mononuclear cells. We characterized 18 of these, including new pairings of established common receptors, cross-family pairings such as TrkA-{gamma}common, and a series of pairings with interferon receptor-1 (IFNAR1), revealing that IFNAR1 can function as a versatile common receptor similar to {gamma}common or {beta}common. We identify novokines that drive monocyte proliferation, T cell survival and CD4+ T cell-specific proliferation. Our framework provides a blueprint for expanding the understanding of cytokine signaling and generating novel therapeutic proteins.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.12.681920v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Geometric Tuning of Cytokine Receptor Association Modulates Synthetic Agonist Signaling</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Exposit, M., Abedi, M., Krishnakumar, A., Jain, S., Yu, T.-Y., Hercus, T. R., Mathew, D., Gray-Gaillard, S., Chen, Z., Grubbe, W. S., Favor, A., Kan, W. L., Schlichthaerle, T., Chen, W., Parker, M. W., Mendoza, J. L., Lopez, A. F., Wherry, E. J., Baker, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that by varying receptor geometry, we can bias pSTAT pathway usage and tune functional outcomes and provide a platform for programmable immune modulation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cytokines signal by bringing receptor subunits together, but the role of receptor geometry in shaping signaling remains unclear because natural ligands enforce fixed assemblies. Here, we present a de novo protein design platform that rigidly scaffolds receptor-binding domains into defined spatial arrangements. Applying this across IL-7, type I and III interferons, IL-10, gp130, {beta} common, and synthetic receptor pairs, we show that by varying geometry, we can bias pSTAT pathway usage and tune functional outcomes. Geometric control allowed us to decouple pSTAT1 from pSTAT5 in IL-7, separate antigen presentation (MHC-I) from checkpoint induction (PD-L1) in type I interferons, and suppress pro-inflammatory IFN{gamma} secretion while retaining anti-inflammatory activity in IL-10. We further created minimal IL-6 and IL-3 agonists and strengthened synthetic receptor pairings inaccessible with present cytokines. These results establish receptor geometry as a central determinant of cytokine activity and provide a platform for programmable immune modulation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.12.681819v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CoxMDS: Multiple Data Splitting for High-dimensional Mediation Analysis with Survival Outcomes in Epigenome-wide Studies</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yao, M., Tian, P., Li, X., Bian, S., Wang, G., Gu, Y., Navas-Acien, A., Vardarajan, B. N., Belsky, D. W., Miller, G. W., Baccarelli, A. A., Liu, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce CoxMDS, a multiple data splitting method that uses Cox proportional hazards models to identify putative causal mediators for survival outcomes, including DNA methylation and the effect of smoking on lung cancer survival.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Causal mediation analysis investigates whether the effect of an exposure on an outcome operates through intermediate variables known as mediators. Although progress has been made in high-dimensional mediation analysis, current methods do not reliably control the false discovery rate (FDR) in finite samples, especially when mediators are moderately to highly correlated or follow non-Gaussian distributions. These challenges frequently arise in DNA methylation studies. We introduce CoxMDS, a multiple data splitting method that uses Cox proportional hazards models to identify putative causal mediators for survival outcomes. CoxMDS ensures finite-sample FDR control even in the presence of correlated or non-Gaussian mediators. Through simulations, CoxMDS is shown to maintain FDR control and achieve higher statistical power compared with existing approaches. In applications to DNA methylation data with survival outcomes, CoxMDS identified eight CpG sites in The Cancer Genome Atlas (TCGA) that are consistent with the hypothesis that DNA methylation may mediate the effect of smoking on lung cancer survival, and two CpG sites in the Alzheimer's Disease Neuroimaging Initiative (ADNI) that are consistent with the hypothesis that DNA methylation may mediate the effect of smoking on time to Alzheimer's disease conversion.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.11.681836v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Bioinformatic inference of the exercise-responsive control of p70 S6 kinase through RPS6KB1 expression</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> McColl, T. J., Zhang, R., Dugourd, A., Saez-Rodriguez, J., Clarke, D. C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We applied a novel bioinformatic network inference algorithm called CARNIVAL (CAusal Reasoning pipeline for Network identification using Integer VALue programming) to infer the molecular regulatory network downstream of canonical exercise sensors controlling RPS6KB1-specific transcription factors (TFs) after acute aerobic</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Previous research suggests that the absolute levels of p70 S6 kinase (p70S6K) are a key determinant of the rate of skeletal muscle protein synthesis (MPS). p70S6K levels are in part determined by the transcriptional control of the gene encoding p70S6K, RPS6KB1, but the molecular mechanisms governing its expression are poorly understood. The purpose of this study was to infer the molecular regulatory network governing RPS6KB1 expression. We applied a novel bioinformatic network inference algorithm called CARNIVAL (CAusal Reasoning pipeline for Network identification using Integer VALue programming) to infer the signaling network downstream of canonical exercise sensors controlling RPS6KB1-specific transcription factors (TFs) after acute aerobic (AE) or resistance exercise (RE). CARNIVAL integrates a prior knowledge network, TF and signaling pathway activities inferred from transcriptomic data, and perturbation targets to predict the network that best explains the data. The networks revealed intracellular sensors and hormone receptors controlling RPS6KB1-specific TFs. Both exercise types resulted in AMPK-mediated SNAI1 regulation, but HIF1A was distinctly controlled (AE: PHD1-3, FIH; RE: AMPK). AE controlled FOXA1 via insulin, TGF-{beta}, and myostatin signalling, while RE controlled CEBPA via MAP3Ks. Our study is the first to apply a comprehensive bioinformatic network inference algorithm to infer causal exercise-responsive signaling networks. The results of our analysis motivate experimentally testable hypotheses pertaining to the molecular control of RPS6KB1 transcription in human skeletal muscle in response to aerobic and resistance exercise.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.11.681841v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An open CAR-T single-cell atlas to enable in-depth characterization and rational engineering of CAR-T products</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Camara-Pena, S., Rodriguez-Marquez, P., Planell, N., Calleja-Cervantes, M. E., Jordana-Urriza, L., Cinnirella, G., Reich-Zeliger, S., Rodriguez-Otero, P., Tamariz, E., Ochoa, I., Yosef, N., Rodriguez-Madoz, J. R., Prosper, F., Hernaez, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We built a functional atlas of CAR-T cell function from over one million cells across 13 studies, integrating data from patients and healthy donors.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We built a CAR-T cell functional atlas from over one million cells across 13 studies, integrating data from patients and healthy donors. The atlas captures 11 phenotypes, links infusion product composition with clinical response, and reveals sex- and age-dependent effects, metabolic signatures, and rare ICANS-associated cells. This open-access resource provides a foundation to understand CAR-T cell function and guide the design of next-generation therapies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.11.681788v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Machine learning-based definition of cellular senescence reveals pro-senescence potential implication in lung adenocarcinoma</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ma, L., Li, H., Li, Y., Lin, Z.-A., Li, J.-Q., Zhang, Y.-Z., Zhang, P., Yao, Z., Li, J., Xiong, M., Cao, Y., Li, R., Yang, C., Tang, X., Chen, M., Wang, H., Zheng, W., Yang, J., Wang, X., Wei, G.-H., Wang, X., Chen, H.-Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed the Predictive Cellular Senescence Model (PreCSenM), a machine learning-based tool that assigns a CS score by integrating senescence-associated features across 888 samples across normal and cancerous transcriptomic datasets.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Despite growing evidence implicating cellular senescence in tumor progression, methodological challenges in objectively quantifying senescent cell burden across cancer types continue to limit mechanistic studies and clinical translation. To address this, we developed the Predictive Cellular Senescence Model (PreCSenM), a machine learning-based tool that assigns a CS score by integrating senescence-associated features across 888 samples. Benchmarking analyses confirmed that PreCSenM markedly outperforms existing methodologies in reliably predicting senescent states within both normal and cancerous transcriptomic datasets. In lung adenocarcinoma (LUAD), PreCSenM identified the CS score as a robust predictor of clinical outcomes. Multi-omics analyses further revealed that lower CS levels correlate with decreased genomic stability and enhanced immune responses, aligning with clinical observations. Notably, drug discovery using PreCSenM identified histone deacetylase inhibitors (HDACis) as potent inducers of CS in LUAD. Transcriptional and epigenetic profiling of HDACi-treated cells pinpointed FOSB as a core transcription factor driving CS, and FOSB knockdown blocked HDACi-induced senescence. Overall, our study establishes PreCSenM as a multidimensional senescence quantification tool, bridging computational prediction with clinical relevance and mechanistic validation to enable senescence-directed therapies in precision oncology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.11.681821v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">scRetinaDB: A Comprehensive Database of Single-Cell and Spatial Omics from Cross-Species Retinas</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tang, M., Jiang, F., Stoica, A.-F., Chen, J., Wang, J., Yao, K., Chen, Z., Xu, X., Wang, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed scRetinaDB, a comprehensive retinal omics data database that aggregates single-cell RNA sequencing (scRNA-seq), single- cell assay for transposase-accessible chromatin using sequencing ( scATAC-seq) and spatial RNA sequencing from retinas across species</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The retina is essential for encoding visual signals, and its dysregulation can lead to retinal diseases. Recent advances in single-cell and spatial sequencing technologies have yielded extensive omics data from retinal tissues across species and biological conditions. However, existing retinal omics data are dispersed across various repositories without uniform processing, which limits integrative analysis. To address this we developed scRetinaDB (https://casapp.dnayun.com/scretina/), a comprehensive resource that aggregates single-cell RNA sequencing (scRNA-seq), single-cell assay for transposase-accessible chromatin using sequencing (scATAC-seq), and spatial RNA sequencing (spRNA-seq) data from retinas across species and diverse biological conditions. The database comprises over 2.79 million retinal cells collected from 453 scRNA-seq datasets spanning 34 studies, 17 species and 27 biological conditions. For each species, these scRNA-seq datasets were integrated to construct a retinal cell atlas. In addition, scRetinaDB also contains 107 scATAC-seq and 18 spRNA-seq datasets from human and mouse retinas. The scRetinaDB website provides four major modules separately for browsing species-specific omics data, searching cross-species omics profiles, performing analyses of cell type annotation and cell similarity analysis, and downloading preprocessed multi-omics datasets. Overall, scRetinaDB is a valuable resource for retinal single-cell and spatial omics, advancing cross-species studies particularly in vision research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.11.681556v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multiplex design and discovery of proximity handles for programmable proteome editing</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Suiter, C. C., Ahn, G., Chiu, M., Fu, Y., Sadre, S., Simon, J. J., Lee, D. S., Fowler, D. M., Maly, D. J., Baker, D., Shendure, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We discovered hundreds of de novo designed effector handles that reproducibly drove either intracellular protein degradation (n = 277) or stabilization (n= 204) of a reporter protein.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Although we now have a rich toolset for genome editing, an equivalent framework for manipulating the proteome with a comparable flexibility and specificity remains elusive. A promising strategy for "proteome editing" is to use bifunctional molecules (e.g. PROteolysis-Targeting Chimeras or PROTACs) that bring a target protein into proximity with a degradation or stabilization effector, but their broader application is constrained by a limited repertoire of well-characterized target or effector "handles". We asked whether coupling de novo protein design to a multiplex screening framework could address this gap by accelerating the discovery of effector handles for intracellular protein degradation, stabilization, or relocalization. Using LABEL-seq, a sequencing-based assay that enables multiplex, quantitative measurement of protein abundance, we screened 9,720 de novo designed candidate effector handles for their ability to recruit a target protein to components of the ubiquitin-proteasome system (UPS) (FBXL12, TRAF2, UCHL1, USP38) or the autophagy pathway (GABARAP, GABARAPL2, MAP1LC3A). In a single experiment, we discovered hundreds of de novo designed effector handles that reproducibly drove either intracellular degradation (n = 277) or stabilization (n = 204) of a reporter protein. Validation of a subset of these hits in an orthogonal assay confirmed that sequencing-based measurements from the primary screen reliably reflected changes in intracellular abundance of the target protein. Successful effector handles were discovered for both the UPS (n = 194) and autophagy (n = 287) pathways, which provide complementary routes for programmable proteome editing. Autophagy-recruiting effector handles generalized to endogenous targets, as substituting the reporter-specific target handle with a high-affinity Mcl-1 binder reduced endogenous levels of this intracellular oncoprotein. Moreover, directing autophagy-recruiting effector handles to the outer mitochondrial membrane dramatically perturbed mitochondrial networks in a manner consistent with synthetic tethering and sequestration. Beyond generating a diverse repertoire of protein abundance or localization effector handles, our results establish a scalable, low-cost platform that links deep learning-guided protein design to functional cellular readouts, and chart a course toward a general framework for programmable proteome editing.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.13.681693v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">gAIRR-wgs: An Algorithm to Genotype T Cell Receptor Alleles Using Whole Genome Sequencing Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Huang, K.-T., Yang, Y.-H., Lin, M.-J., Lai, S.-K., Chou, T.-H., Lee, C.-Y., Hung, T.-K., Hsu, C.-L., Yang, Y.-C., Chen, C.-Y., Chen, P.-L., Hsu, J. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed gAIRR-wgs, for WGS-based TR allele typing, which can accurately identify germline TR from standard WGS data using feasible computational resources and reveals substantial immunogenetic diversity in population cohorts.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> T cell receptor (TR) genes, including variable (TR_V), diversity (TR_D), and joining (TR_J) segments, exhibit allelic diversity that is critical to adaptive immunity. Growing evidence has identified associations between TR genes and immune-related diseases. Germline variants may influence TR gene function and subsequent usage, highlighting the importance of accurate TR allele profiling. However, accurately identifying germline TR from standard WGS data remains challenging due to short read lengths, limited depth, and high sequence similarity. To address these challenges, we developed gAIRR-wgs, for WGS-based TR allele typing. By incorporating novel alleles from HPRC individuals, gAIRR-wgs exhibited excellent performance in allele calling, with F1 scores of 100.0% for TR_D, 99.8% for TR_J, and 98.3% for TR_V. Applying this pipeline to 1,492 individuals from the Taiwan Biobank (TWB), we identified 449 novel TR alleles, 277 of which overlapped with HPRC release 1 data of mixed ethnicity and are absent in the IMGT database. Further population comparison analysis revealed significant TR allele distribution differences across global populations, showing population-specific patterns and diversity variations between ethnic groups. We also discovered TWB-specific deletion polymorphisms affecting contiguous TRGV and TRBV genes, which are not recorded in the gnomAD database and undetected by standard structural variant callers, highlighting the need for tailored approaches to resolve complex immune gene regions. In conclusion, gAIRR-wgs enables accurate TR allele calling from standard WGS data using feasible computational resources and reveals substantial immunogenetic diversity in population cohorts.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.11.681280v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Pharmacokinetics and efficacy of tank-water administered BRAF-inhibitor dabrafenib in a zebrafish model of BRAF-mutant melanoma</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Villman, J., Dibus, N., Huhn, J., Haapalainen-Saaristo, A.-M., Evstafev, I., Dickens, A., Scherf-Clavel, O., Paatero, I., Kurppa, K. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Continuous tank water-administered BRAF-inhibitor dabrafenib in adult zebrafish provides a feasible, efficient, and well-tolerated dosing strategy to study treatment-related effects in melanoma biology .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Zebrafish models are widely used to study the biology of BRAF-mutant melanoma. However, long-term treatment of adult fish with small molecule BRAF inhibitors is challenging, limiting the usefulness of this model to study treatment-induced effects in melanoma biology. In addition, pharmacokinetic studies on small molecule inhibitors in zebrafish that could inform rational dosing strategies, are largely lacking. Here, we have assessed the pharmacokinetics, metabolism and efficacy of continuous tank water -administered BRAF-inhibitor dabrafenib in adult zebrafish. Our results demonstrate that dabrafenib is quickly absorbed from the tank water, reaching efficacious plasma levels within one hour following treatment, but also shows fast elimination kinetics with a half-life of 1.6 hours. We could detect most of the human metabolites of dabrafenib in zebrafish, suggesting that dabrafenib metabolism in zebrafish follows a similar process as in humans. Continuous tank water -administered dabrafenib led to therapeutically relevant steady-state plasma levels that inhibited the BRAF-driven signaling and growth in zebrafish melanoma cells in vitro, and resulted in robust in vivo efficacy in a genetic zebrafish model of BRAF-mutant melanoma, with no apparent toxicity. Together, our results demonstrate that continuous tank water -administered dabrafenib provides a feasible, efficient, and well-tolerated dosing strategy to study treatment-related effects in zebrafish models of BRAF-mutant melanoma. We expect that tank water-administration may also facilitate the dosing of other small molecule inhibitors, especially those with short in vivo half-life in zebrafish.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.12.681850v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Hypoxia Imaging as a Radiomics Signature in Tumors Using Novel 19F-Eu-Based Contrast Agents for Magnetic Resonance Imaging</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> White, S., Subasinghe, S. A. A. S., Romero, J., Samee, M. A. H., Yustein, J., Pautler, R., Allen, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that convection-enhanced delivery (CED) is a reliable and robust methodology to distribute MRI contrast agents throughout a tumor and show that integration of hypoxia-detecting imaging modalities and spatial biology enables real-time in vivo insights into biology and identification of biomarkers</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The ability to accurately identify and characterize regions of hypoxia has been an active area of study due to the biological ramifications of low oxygen tension in different pathological conditions, including inflammation, infections, wound healing, cardiovascular conditions, kidney and pulmonary diseases, hepatic and neurological toxicities, and cancer. Although hypoxia contributes a significant role in these conditions, the ability to accurately and instantaneously monitor the presence of hypoxia in vivo and correlate comprehensive analysis of hypoxia-dependent molecular signatures in response to treatments regimens is lacking. With the advent of hypoxia-responsive contrast agents for magnetic resonance imaging (MRI), including the recent development and characterization by our team of a novel probe that involves both 19F and EuII, the capability to integrate direct hypoxia imaging in real-time with new technologies that enable spatial transcriptomic profiling has become a possibility. To assess the capability of this agent for studying hypoxia, we used osteosarcoma as a model. In this preliminary study we demonstrate two major results: First, we show that convection-enhanced delivery (CED) is a reliable and robust methodology to distribute MRI contrast agents throughout a tumor. Second, we show that integration of direct hypoxia-detecting imaging modalities and spatial biology enables real-time in vivo insights into biology and identification of biomarkers directly applicable to disease development and response to therapy. The ability to identify and define hypoxia-mediated biology using direct rather than indirect MRI methods has extremely significant implications in the care of a wide-range of pathological conditions. Consequently, the framework outlined in these preliminary studies is applicable to other pathological conditions and provides the basis for direct in vivo hypoxia detection, monitoring, and biological analyses with translational applications to patient care and management.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.11.681845v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Obesity Impairs the Antitumor Activity of CAR-T Cells in Triple-Negative Breast Cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Malian, H. M., Marnata Pellegry, C., Oh, H. M., Glenny, E. M., Ho, A. N., Dotti, G., Hursting, S. D., Coleman, M. F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Obesity-related inflammatory cytokines promote B7-H3 expression in human and murine TNBC cells in vitro and B7/H3-related inflammation in vivo correlates with tumor aggressiveness in vivo.</p><p style="font-size: 16px; text-align: justify;">Background: We have reported that chimeric antigen receptor (CAR) T cells targeting B7-H3 (B7-H3.CAR) are effective in a preclinical model of triple negative breast cancer (TNBC), and have initiated a Phase I study to assess safety and efficacy. However, heterogeneous antigen expression and immunosuppressive tumor microenvironments (TME) remain roadblocks for effective CAR-T cell therapy. In particular, obesity represents a negative prognostic factor in TNBC partly due to chronic inflammation and impaired adaptive immune responses. Hence, we sought to determine if obesity can affect the antitumor activity of B7-H3.CAR-T cells. Methods: We used qPCR and western blotting to determine if cytokines associated with obesity affect B7-H3 expression in TNBC cell lines. Furthermore, we used shRNA to suppress B7-H3 expression in a syngeneic orthotopic E0771 tumor model and measured tumor growth in control and diet-induced obese (DIO) mice. Finally, we evaluated the antitumor effects of B7-H3.CAR-T cells in both control and DIO mice orthotopically engrafted with the E0771 tumor cell line. Immune profiling was conducted using flow cytometry. Results: Obesity-related inflammatory cytokines promote B7-H3 expression in human and murine TNBC cells in vitro and B7-H3 expression correlates with tumor aggressiveness in vivo. CAR-T cells obtained from control or DIO mice were equally cytotoxic in vitro but activated T cells and B7-H3.CAR-T cells obtained from DIO mice show transcriptomic changes (enriched Tox2, Prdm1, Batf) and impaired glycolytic capacity, respectively. Finally, we demonstrated that obesity impairs CAR-T cell antitumor effects and durability of response in vivo with a near complete loss of memory formation. Conclusions: Here we identified a correlation between B7-H3 expression, obesity, and rate of tumor growth in TNBC. Furthermore, we showed that obesity constrains both the ability of B7-H3.CAR-T cells to control tumor growth and to elicit durable immunological memory. Taken together, these data identify obesity as an underappreciated and potent modulator of CAR-T cell functionality.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.10.681694v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Single-cell lineage trajectory defines CDK inhibitor-sensitive cells-of-origin in esophageal squamous cell cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ko, K.-P., Zhang, J., Jun, S., Park, J.-I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A single-cell-derived therapeutic strategy that exposes vulnerabilities in tumor-initiating cells.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding the cells of origin is essential for overcoming therapy resistance in esophageal squamous cell carcinoma (ESCC). We utilized machine learning-based single-cell trajectory analysis on 4NQO-induced murine models and genetically engineered organoids to identify multiple distinct cell clusters that serve as cellular origins of ESCC. Gene regulatory network analysis of these populations indicated activation of stem/progenitor cell regulators, including PRRX2 and CEBP{beta}. Translating these findings, a transcriptome-based drug repurposing screen identified five chemical candidates, four of which are potent Cyclin-Dependent Kinase (CDK) inhibitors, aligning with the frequent loss-of-function mutations in TP53 and CDKN2A observed in ESCC. Notably, CDK inhibitors markedly inhibit ESCC cell proliferation. This research delineates the potential cellular origins of ESCC and their key regulons, thereby pioneering a single-cell-derived therapeutic strategy that exposes vulnerabilities in tumor-initiating cells.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.11.681802v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Oncogenic EME1 promotes tumor progression and immune modulation in human cancers with therapeutic targeting potential.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Muhammad Alaa Eldeen, Abdelrahman Mostafa, Farag Mamdouh, Walid K Abdulsahib, Dalal Sulaiman Alshaya, Eman Fayad, Hassan Otifi, Hesham M Hassan, Mohammed Alshehri, Aiysha Althobaiti, Ghadi Alsharif, Mohamed A Soltan</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a new analysis of EEME1, a critical DNA repair endonuclease implicated in genome instability and cancer progression, and its pan-cancer roles.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> EME1, a critical DNA repair endonuclease, has emerged as a potential oncogene implicated in genome instability and cancer progression. However, its pan-cancer roles, prognostic significance, immune interactions, and therapeutic targeting remain underexplored.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41082058/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

