<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Thursday, 14 August 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Paired plus-minus sequencing is an ultra-high throughput and accurate method for dual strand sequencing of DNA molecules</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cheng, A. P., Rusinek, I., Sossin, A., Widman, A. J., Meiri, E., Krieger, G., Hirschberg, O., Tov, D. S., Gilad, S., Jaimovich, A., Barad, O., Avaylon, S., Rajagopalan, S., Potenski, C., Prieto, T., Yuan, D. J., Furatero, R., Runnels, A., Costa, B. M., Shoag, J. E., Al Assaad, M., Sigouros, M., Manohar, J., King, A., Wilkes, D., Otilano, J., Malbari, M. S., Elemento, O., Mosquera, J. M., Altorki, N. K., Saxena, A., Callahan, M. K., Robine, N., Germer, S., Evrony, G., Faltas, B. M., Landau, D. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce paired plus-minus sequencing (ppmSeq) technology, in which both strands of a double-stranded DNA molecule contribute to a single sequencing read, allowing for a duplex yield that scales linearly with sequencing coverage across a wide range of inputs (1.8-98</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Distinguishing real biological variation in the form of single-nucleotide variants (SNVs) from errors is a major challenge for genome sequencing technologies. This is particularly true in settings where SNVs are at low frequency such as cancer detection through liquid biopsy, or human somatic mosaicism. State-of-the-art molecular denoising approaches for DNA sequencing rely on duplex sequencing, where both strands of a single DNA molecule are sequenced to discern true variants from errors arising from single stranded DNA damage. However, such duplex approaches typically require massive over-sequencing to overcome low capture rates of duplex molecules. To address these challenges, we introduce paired plus-minus sequencing (ppmSeq) technology, in which both DNA strands are partitioned and clonally amplified on sequencing beads through emulsion PCR. In this reaction, both strands of a double-stranded DNA molecule contribute to a single sequencing read, allowing for a duplex yield that scales linearly with sequencing coverage across a wide range of inputs (1.8-98 ng). We benchmarked ppmSeq against current duplex sequencing technologies, demonstrating superior duplex recovery with ppmSeq, with a rate of 44%{+/-}5.5% (compared to ~5-11% for leading duplex technologies). Using both genomic as well as cell-free DNA, we established error rates for ppmSeq, which had residual SNV detection error rates as low as 7.98x10-8 for gDNA (using an end-repair protocol with dideoxy nucleotides) and 3.5x10-7{+/-}7.5x10-8 for cell-free DNA. To test the capabilities of ppmSeq for error-corrected whole-genome sequencing (WGS) for clinical application, we assessed circulating tumor DNA (ctDNA) detection for disease monitoring in cancer patients. We demonstrated that ppmSeq enables powerful tumor-informed ctDNA detection at concentrations of 10-5 across most cancers, and up to 10-7 in cancers with high mutation burden. We then leveraged genome-wide trinucleotide mutation patterns characteristic of urothelial (APOBEC3-related and platinum exposure-related signatures) and lung (tobacco-exposure-related signatures) cancers to perform tumor-naive ctDNA detection, showing that ppmSeq can identify a disease-specific signal in plasma cell-free DNA without a matched tumor, and that this signal correlates with imaging-based disease metrics. Altogether, ppmSeq provides an error-corrected, cost-efficient and scalable approach for high-fidelity WGS that can be harnessed for challenging clinical applications and emerging frontiers in human somatic genetics where high accuracy is required for mutation identification.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.11.669689v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">HallmarkGraph: a cancer hallmark informed graph neural network for classifying hierarchical tumor subtypes.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Qingsong Zhang, Fei Liu, Xin Lai</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new method for hierarchical tumor subtype diagnosis and analysis .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurate tumor subtype diagnosis is crucial for precision oncology, yet current methodologies face significant challenges. These include balancing model accuracy with interpretability and the high costs of generating multi-omics data in clinical settings. Moreover, there is a lack of validated models capable of classifying hierarchical tumor subtypes across a comprehensive pan-cancer cohort.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40802546/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Analyzing Generative AI and Machine Learning in Auto-Assessing Schizophrenia's Negative Symptoms.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chih-Min Liu, Yi-Hsuan Chan, Ming-Yang Ho, Chen-Chung Liu, Ming-Hsuan Lu, Yi-An Liao, Ming-Hsien Hsieh, Yufeng Jane Tseng</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We assess negative symptoms of schizophrenia using generative AI and machine learning to assess expression and motivation and pleasure domains.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Traditional assessments of schizophrenia's negative symptoms rely on subjective and time-consuming psychiatric interviews. To provide more objective and efficient evaluations, this study examines the efficacy of an automated system utilizing generative AI (GenAI) and machine learning (ML) to assess negative symptoms of schizophrenia, including expression (EXP) and motivation and pleasure (MAP) domains.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40801756/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Machine Learning-Aided Meta-Analysis Reveals Changes in Penaeus vannamei Gut Bacterial Communities Upon Dietary Supplementation-Induced Immunostimulation.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> John Paul Matthew D Guzman, Samuel Mwakisha Mwamburi, Supatcha Lurkpranee, Keiichiro Koiwai, Hidehiro Kondo, Ikuo Hirono</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A meta-analysis of gut bacterial community studies of immunostimulants for Penaeus vannamei shrimp and identified key features in gut bacterial communities which provide a novel perspective on the interplay between gut bacterialcommunity and immunity.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Gut bacterial communities play a key role in shrimp health; thus, their modulation has been a target of dietary supplements which also function in enhancing disease and stress resistance of shrimp. However, this also raised the question of whether immunostimulants yield distinct changes in the gut bacterial composition or whether there are consistent features across all treatments. Here, we performed a machine learning-aided meta-analysis of 16S rRNA gut bacterial community studies of immunostimulants for Penaeus vannamei. Sequence reads from the selected studies were obtained and processed through bioinformatics tools. While beta diversity analysis suggests similarities between the normal, infected and stimulated shrimp, alpha diversity indices showed higher species richness in the gut bacterial communities of shrimp fed with immunostimulants. Specific beneficial taxa were enriched upon immunostimulation, while potentially pathogenic taxa decreased in abundance. Random forest modelling also identified key predictor taxa which may be used to classify gut bacterial communities based on immune status, type of immunostimulant and the specific immunostimulant. Despite some shared patterns in differential abundance-having decreased relative abundances of Photobacterium and other members of Gammaproteobacteria-the influence of immunostimulation on gut bacterial community composition was type- and treatment-specific, as evident in the distinct abundance profiles of the predictor taxa. Functional prediction analysis also showed distinct pathways enriched in immunostimulated shrimp, as influenced by the type of the immunostimulant. This study highlighted the specific impacts of dietary supplementation-induced immunostimulation on gut bacterial communities and identified key features in immunostimulated shrimp which provide a novel perspective on the interplay between gut bacterial community and immunity.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40801302/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Innovations in Yeast Synthetic Biology: Engineered Discovery Systems for Immunotherapy.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ethan W Slaton, Natalie Clay, Nathan Phan, Blaise R Kimmel</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> In this review, we will highlight the latest innovations regarding using engineered yeast to display proteins (e.g., nanobodies) and screen for potential antigens for immune receptors (E.g. GPCRs, TCRs).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Yeast-based platforms are emerging as innovative synthetic biology tools for the discovery of immunotherapeutic proteins. Through the integration of (i) high-throughput surface display technologies, (ii) automated evolution systems (such as OrthoRep), and (iii) computational design strategies, the field of synthetic biology can make a direct impact toward rapidly identifying and engineering novel protein-based therapeutics. In this review, we will highlight the latest innovations regarding using engineered yeast to display proteins (e.g., nanobodies) and screen for potential antigens for immune receptors (e.g., GPCRs, TCRs). We will also discuss emerging areas in which the field has recently progressed and how the innovative technologies from these efforts help bridge the gap between synthetic biology and immunology such as identifying therapeutic binding events for engineered proteins of interest with the potential to actuate downstream immune responses. These innovations illustrate how yeast enables new design, build, test, and learn (DBTL) workflows in immunoengineering and offers a scalable, programmable chassis for developing tools and technologies for the construction of next-generation biotherapeutics.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40802971/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Specialised super-enhancer networks in stem cells and neurons</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Harabula, I., Speakman, L., Musella, F., Fiorillo, L., Zea-Redondo, L., Kukalev, A., Beagrie, R. A., Morris, K. J., Fernandes, L., Irastorza-Azcarate, I., Fernandes, A. M., Carvalho, S., Szabo, D., Ferrai, C., Nicodemi, M., Welch, L., Pombo, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We compare the structural and functional properties of super-enhancers between embryonic stem cells and post-mitotic neurons by combining Genome Architecture Mapping, chromatin accessibility, histone modification, and transcriptome data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Super-enhancers (SEs) are clusters of enhancers with high transcriptional activity that play essential roles in defining cell identity through regulation of nearby genes. SEs preferentially form multiway chromatin interactions with other SEs and highly transcribed regions in embryonic stem cells. However, the properties of the interacting SEs and their specific contributions to complex regulatory interactions in differentiated cell types remain poorly understood. Here, we compare the structural and functional properties of SEs between embryonic stem cells (ESCs) and dopaminergic neurons (DNs) by combining Genome Architecture Mapping (GAM), chromatin accessibility, histone modification, and transcriptome data. Most SEs are cell-type specific and establish extensive pairwise and multiway chromatin interactions with other SEs and genes with cell-type specific expression. SE interactions span megabase genomic distances and frequently connect distant topologically associating domains. By applying network centrality analyses, we detected SEs with different hierarchical importance. Highest network centrality SEs contain binding motifs for cell-type specific transcription factors, and are candidate regulatory hubs. The functional heterogeneity of SEs is also highlighted by their organisation into modular sub-networks that differ in structure and spatial scale between ESCs and DNs, with more specific and strongly connected SE modules in post-mitotic neurons. Our results uncover both the high complexity and specificity of SE-based 3D regulatory networks and provide a resource for prioritizing SEs with potential roles in transcriptional regulation and disease.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.13.670083v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Weighted Off-target and Efficiency Scoring Reveal Genome Composition-Dependent Optimal CRISPR/Cas9 Guide Design</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Krishna Y K, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A computational method for optimizing CRISPR/Cas9 guide RNA that combines PAM diversity, local efficiency penalties, and weighted off-target scoring to find high-performing guides across a range of genome compositions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The efficiency and specificity of guide RNAs continue to be crucial obstacles for successful experimental design, despite the fact that CRISPR/Cas9 has transformed genome editing. In this work, we introduce a computational method for optimizing CRISPR/Cas9 guide RNA that combines PAM diversity, local efficiency penalties, and weighted off-target scoring to find high-performing guides across a range of genome compositions. To capture a variety of natural genomic complexity, we simulated five sample genomes: AT-rich, GC-rich, balanced GC content, and high-repeat variations. All twenty-nucleotide target sequences were scanned for each genome, and off-target potential was assessed by permitting up to two mismatches with weighted penalties for seed region sites. To accommodate for any secondary structure impacts, efficiency assessment included both local sliding window penalties and global GC content. Furthermore, we looked at several PAM sequences that were pertinent to various Cas9 variations in order to assess how they affected guide selection. The findings show that efficiency scores vary by genome composition, with the highest scoring guides consistently displaying zero anticipated off-target events. While balanced genomes showed intermediate tendencies, GC-rich genomes tended to choose slightly higher efficiency guides than AT-rich genomes. PAM type affects guide efficiency, according to analysis across several genomes, and the combination of efficiency and off-target score consistently indicates guides with good expected performance. Three-dimensional scatter plots of efficiency and off-target counts versus genomic position, violin plots of off-target distributions, and genome-wide heatmaps emphasizing the best guide positions were used to illustrate these findings. In addition to offering a generalizable computational method for choosing CRISPR/Cas9 guides that optimize specificity and efficiency, our study gives fresh insights into the interactions among genome composition, PAM selection, and guide design criteria. By taking into account weighted off-target penalties, genome complexity, and local efficiency effects, this in silico framework overcomes some of the main drawbacks of earlier simulations. It is also easily applicable to direct selection for experimental research on a variety of organisms. The results provide the groundwork for future advancements in genome editing techniques by establishing a predictive computational framework that can expedite CRISPR/Cas9 research and minimize trial and error in guide selection.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.13.670047v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Predicting protein variant properties with electrostatic representations</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> van der Flier, F. J., van Dijk, A. D. J., de Ridder, D., Redestig, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We explore electrostatic representations obtained from solving the Poisson-Boltzmann equation as novel features to fit VEP models to deep mutational scanning data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Does evolution capture the full functional potential of proteins, or is this potential restricted by selective pressures? If the former is true, providing variant effect prediction (VEP) models with evolutionary derived representations should be sufficient to guide the optimization of proteins. In the latter scenario however, VEP models require different sources of information. In this work, we explore whether physics-based representations of protein variants benefit the performance of VEP models. More specifically, we explore electrostatic representations obtained from solving the Poisson-Boltzmann equation as novel features to fit VEP models to deep mutational scanning (DMS) data. We contrast and combine these representations with ones derived from evolutionary models. To this end, we perform a range of experiments; benchmarking, ensembling with evolutionary models, accounting for assay conditions and extrapolating to new screening data. Though our model displays significant predictive capacity, we find no instance where it provides a better alternative over existing evolutionary models, suggesting that electrostatic representations derived by our methods do not capture extra information compared to evolutionary representations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.11.669649v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Chromosome-scale genome assembly of Malcolmia littorea using long-read sequencing and single-pollen genotyping technologies</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shirasawa, K., Yoshitake, K., Kondo, H., Kikuchi, S., Koiwai, K., Fujii, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a genome assembly of 10 chromosome-scale sequences for Malcolmia littorea, an understudied species with ecological and evolutionary significance, with the aim of understanding the molecular mechanisms underlying key traits in the species.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Malcolmia littorea, a member of the family Brassicaceae, is adapted to coastal and sandy environments and has become a model in studies of reproductive barriers. However, genomic resources for the species are limited. Here, with the aim of understanding the molecular mechanisms underlying key traits in M. littorea, including its survival under harsh conditions, we present a de novo genome assembly consisting of 10 chromosome-scale sequences. We employed a high-fidelity long-read sequencing technology for genome assembly. To anchor the sequences to chromosomes, we developed a single-pollen genotyping method to construct a genetic linkage map based on SNPs derived from transcriptomes of pollen grains, possessing recombinant haploid genomes. We built a genome assembly consisting of 10 chromosome-scale sequences (214 Mb in total) for M. littorea containing 30,861 predicted genes. A comparative genome analysis and gene prediction indicated that the genome of M. littorea is double the size of the Arabidopsis thaliana genome, consistent with a whole-genome duplication followed by gene subfunctionalization and/or neofunctionalization in M. littorea. This study provides a basis for research on M. littorea, an understudied species with ecological and evolutionary significance.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.12.669985v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Synteny and linkage decay in bacteriophage pangenomes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fendley, J. M., Molari, M., Neher, R. A., Shraiman, B. I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We investigate the evolution of actinobacteriophage genomes and show that homologous recombination is restricted by strong gene order conservation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Bacteriophages are rich in genetic diversity, due to frequent horizontal transfer and recombination. This makes traditional microbial phylogenetic analyses, often based on the assumption of vertical inheritance, not suitable for interpreting this diversity. Here, inspired by recent work on bacterial pangenomes, we investigate the evolution of a collection of 3425 actinobacteriophage genomes. We find that synteny is strongly conserved: core genes have a well-defined order, and most accessory genes are localized in a few locations along the core genome backbone. Within the core genome alignment, linkage disequilibrium decays rapidly with distance in some groups, while phylogenetic structure in other groups causes long-range linkage. Our quantitative characterizations extend across many groups of phages and indicate widespread homologous recombination restricted by strong gene order conservation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.12.669904v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Interpretable Machine Learning on Soybean Multi Omics Data Reveals Drought-Driven Shifts of Plant-Microbe Interactions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yoshioka, H., Debeljak, P., Prado, S., Iwata, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A model-agnostic, interpretable machine learning approach to the plant phenotype prediction framework identifies multi-omics biomarkers and interactions, providing insights into plant adaptation to drought stress through environment-dependent rhizosphere networks and symbiotic associations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Plant-microbe interactions in the rhizosphere are central to plant growth, nutrient acquisition, and stress resilience. Multi-omics approaches enable comprehensive profiling of different biological layers, yet integrating these data to understand the mechanisms underlying plant-microbe symbiosis, particularly under drought stress, remains a major challenge. We integrated genomic, metabolomic, and microbiome data from 198 soybean accessions grown under both control and drought conditions to identify environment-specific predictive features of plant phenotypes. We compared best linear unbiased prediction (BLUP), genome-wide association study (GWAS), and a nonlinear machine learning model (Random Forest; RF) to evaluate their ability to detect informative features. RF models provided flexible variable selection and outperformed linear models in capturing nonlinear dependencies. Model interpretation using SHapley Additive exPlanations (SHAP) revealed that the isoflavone derivative daidzin and the drought-tolerant Candidatus Nitrosocosmicus are major contributors to phenotypic variation, specifically under drought stress. SHAP-based interaction networks revealed cross-omics links, such as connections between daidzin, gamma-aminobutyric acid (GABA), and Paenibacillus. The application of an interpretable machine learning approach to the plant phenotype prediction framework identifies multi-omics biomarkers and interactions, providing insights into plant adaptation to drought stress through environment-dependent rhizosphere networks and symbiotic associations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.13.670005v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Foliar Application of Polymer-coated Manganese Dioxide Nanoparticles: Mechanisms of Uptake and Metabolic Responses in Manganese Deficient Barley</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pinna, A., Gronbak, A. K., Kristensen, E. V., Thiebaut, N., Szameitat, A., Martin-Bertelsen, B., Hassenkam, T., Grivel, J.-C., Mokso, R., Husted, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that colloidally stable, pH-responsive polyacrylic acid (PAA)-coated manganese dioxide (MnO2) nanoparticles can be designed to significantly restore key metabolic functionalities in barley (Hordeum vulgare) within a few days.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The application of nanotechnology in plant science is unlocking innovative approaches to enhance nutrient use efficiency in crops, particularly through foliar fertilization. This study demonstrates that colloidally stable, pH-responsive polyacrylic acid (PAA)-coated manganese dioxide (MnO2) nanoparticles (nPAA-MnO2) can be designed to significantly restore key metabolic functionalities in manganese (Mn)-deficient barley (Hordeum vulgare) within a few days. Using a combination of advanced bioimaging techniques - including confocal laser scanning microscopy (CLSM), laser ablation-inductively coupled plasma-mass spectrometry (LA-ICP-MS), and X-ray nano-computed tomography (nano-CT), we mapped the uptake and distribution pathways of nPAA-MnO2 compared to ionic Mn. While soluble Mn2+ ions primarily enter through hydrophilic cuticular pores, nPAA-MnO2 penetrates leaves via stomata, facilitated by the application of an organosilicone surfactant and glycerol to enhance wetting and hydraulic activation of stomatal pores. Within a few hours, nPAA-MnO2 accumulated in the sub-stomatal cavity and mesophyll apoplast, gradually releasing bioavailable Mn ions in the acidic apoplast environment. Moreover, labeling experiments with tracer ions revealed nPAA-MnO2 hot-spots around the vascular bundles and a limited but significant basipetal translocation of intact nanoparticles out of the foliar application zone, a pivotal step towards converting immobile nutrients such as Mn into mobile ones. Importantly, and unlike ionic Mn solutions, nPAA-MnO2 could be applied at high doses without causing leaf scorching and cytotoxicity, paving the way for more sustainable and efficient foliar fertilization practices. These novel aspects of nanoparticle uptake, translocation, and assimilation underscores the potential of nanotechnology to address nutrient mobility challenges in agriculture, representing an important contribution to the green transition of modern crop production.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.11.669670v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Esca Disease triggers local transcriptomic response and systemic DNA methylation changes in grapevine</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Berger, M. M. J., Garcia, V., Rubio, B., Bortolami, G., Gambetta, G., Delmas, C. E. L., Gallusci, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that metabolic alterations and extensive transcriptomic reprogramming are restricted to symptomatic leaves and partially associated with local changes in DNA methylation, suggesting a systemic response to the disease at the epigenetic level.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Woody plants such as grapevine are vulnerable to trunk diseases caused by pathogens that colonize the wood, disrupt vascular function, and induce recurrent leaf symptoms associated with major metabolic disturbances and canopy decline. Over time, these diseases can irreversibly alter plant physiology and phenotype, ultimately reducing vine longevity. One of the most predominant of such diseases is esca, which is a major cause of vineyard dieback, with rising incidence worldwide over the past decade. However, the molecular mechanisms underlying esca symptom development remain unclear. In this study, we leveraged the heterogeneous expression of esca-symptoms within individual grapevines to investigate molecular responses in both symptomatic and asymptomatic leaf tissues. By combining metabolite profiling, RNA-seq and whole genome bisulfite sequencing, we show that metabolic alterations and extensive transcriptomic reprogramming are restricted to symptomatic leaves and are partially associated with local changes in DNA methylation. Asymptomatic leaves display distinct DNA methylation changes, some of which are shared with symptomatic tissues, suggesting a systemic response to the disease at the epigenetic level. Notably, subset of these methylation marks are observable prior to symptom emergence, highlighting the potential of epigenetic biomarkers for the early detection of trunk diseases in perennial plants.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.11.669596v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Predicting flowering time using integrated morphophysiological and genomic data with machine learning models</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Babaei, M., Nemati, H., Arouiee, H., Torkamaneh, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel approach combining temporal phenotypic analysis, genomic data, and machine learning (ML) to identify key features associated with early, medium, and late flowering in cannabis landraces.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Indigenous Cannabis sativa populations have adapted to diverse environments, resulting in genetic and phenotypic diversity. Understanding the mechanisms underlying flowering time variation is crucial for optimizing cultivation and breeding. This study employed a novel approach combining temporal phenotypic analysis, genomic data, and machine learning (ML) to identify key features associated with early, medium, and late flowering in cannabis landraces. We collected weekly data on six morphophysiological traits--stem diameter, height, growth rate, node number, internode length, and SPAD chlorophyll index--from 25 cannabis landrace populations 13 weeks for female plants and 11 weeks for male plants. Additionally, 145 accessions were genotyped using high-density genotyping-by-sequencing, resulting in 233,624 high-quality single nucleotide polymorphisms (SNPs). A comprehensive ML framework integrating mutual information (MI), recursive feature elimination (RFE), random forest (RF), and support vector machine (SVM), was used to investigate 234,002 features, encompassing SNPs, morphophysiological traits, and environmental factors. This approach identified 53 key features--22 genetic variants and 31 morphophysiological traits--that effectively distinguish between early, medium, and late flowering types with an accuracy of 96.6%. The identified SNPs were distributed across multiple chromosomes, including chromosomes 08, 09, and X. Notably, key loci like AutoFlower3 (CsFT3) (on chromosome 08) and CircadianFloweringLocus1 (CsCFL1) (on chromosome 09) were identified, with several SNPs located within or near annotated genes. These findings contribute significantly to the understanding of cannabis chronobiology and support the development of "smart crop" strategies by providing valuable markers for early selection and targeted breeding programs aimed at optimizing flowering time under diverse conditions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.11.669697v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Dynamic Tuning of Genetic Circuit Gain via Controllable Plasmid Copy Number</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Namura, H., Hori, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a genetic circuit that enables dynamic regulation of circuit gain through the regulation of plasmid copy numbers.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Plasmid copy number directly influences gene expression strength in synthetic genetic circuits and serves as a tunable parameter for adjusting circuit gain and output levels. Building upon recent progress in copy number control, this report presents a genetic circuit that enables dynamic regulation of circuit gain through the regulation of plasmid copy numbers. To illustrate the underlying concept, we first developed a mathematical model describing the relationship between plasmid copy number and circuit gain and simulated the circuit dynamics to examine how gain responds to copy number changes. We then constructed the circuit and experimentally confirmed that plasmid copy number, i.e., circuit gain, can be externally controlled in E. coli. In particular, we demonstrate communication-based gain control using AHL-mediated quorum sensing from co-cultured cells. These results support the feasibility of plasmid copy number control as a practical and predictable strategy for tuning synthetic gene circuit behavior.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.13.670071v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Toehold-VISTA: A machine learning approach to decipher programmable RNA sensor-target interactions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Robson, J. M., Green, A. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a machine learning-guided framework for the rapid design of RNA sensors with improved function against SARS-CoV-2 RNA .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> RNA-based biosensors have emerged as essential tools in synthetic biology and diagnostics, enabling precise and programmable responses to diverse RNA inputs. However, the time to design, produce, and screen high-performance RNA sensors remains a critical challenge. The fundamental rules governing RNA-RNA interactions--specifically the structure-function relationships that determine sensor performance--remain poorly understood. Here, we present a method enabling versatile in-silico RNA-targeting analysis (VISTA), a machine learning-guided framework for the rapid design of RNA sensors. VISTA integrates biophysical modeling of both sensor and target RNAs with a partial least squares discriminant analysis (PLS-DA) machine learning framework. Using high-throughput experimental measurements with sequence-structure feature extraction to train predictive models, we capture the key determinants of RNA sensor performance. We find that by using toehold switches as a model RNA sensor, Toehold-VISTA successfully designs RNA sensors with improved function against SARS-CoV-2 RNA. These findings establish a broadly applicable, target-aware design strategy for accelerating RNA sensor engineering across biotechnology and diagnostic applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.12.669990v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Automated Assembly of Programmable RNA-Based Sensors</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Robson, J. M., Arevalos, N. R., Green, A. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present an automated plasmid assembly pipeline using liquid handling robotics to enable high-throughput construction of plasmids with arbitrary sequences.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Engineered programmable RNA sensors have been applied in low-cost diagnostics, endogenous RNA detection, and multi-input genetic circuits. However, designing, producing, and screening high-performance RNA sensors remains time consuming and labor intensive. Here, we present an automated plasmid assembly pipeline using liquid handling robotics to enable high-throughput construction of plasmids with arbitrary sequences. We compare automated and manual assembly methods using the NGS Hamilton Microlab STAR across two plasmid backbones to evaluate efficiency and reliability. As a proof of concept, we use this modular platform to construct 144 total plasmids encoding riboregulators targeting diverse viral targets along with their cognate trigger sequences. We further demonstrate that the assembled plasmids are functional in both bacterial and cell-free expression systems.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.12.669972v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genetic entanglement enables ultra-stable biocontainment in the mammalian gut</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Foo, G., Uruthirapathy, A., Zhang, C., Batko, I. Z., Heinrichs, D. E., Edgell, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A dual-layered failsafe biocontainment strategy that entangles genetic sequences to create pseudo-essentiality and buffer against mutations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Imbalances in the mammalian gut are associated with acute and chronic conditions, and using engineered probiotic strains to deliver synthetic constructs to treat them is a promising strategy. However, high rates of mutational escape and genetic instability in vivo limit the effectiveness of biocontainment circuits needed for safe and effective use. Here, we describe STALEMATE (Sequence enTAngLEd Multi lAyered geneTic buffEring), a dual-layered failsafe biocontainment strategy that entangles genetic sequences to create pseudo-essentiality and buffer against mutations. We entangled the colicin E9 immunity protein (Im9) with a thermoregulated meganuclease (TSM) by overlapping the reading frames. Mutations that disrupted this entanglement simultaneously inactivated both biocontainment layers, leading to cell death by the ColE9 nuclease and the elimination of escape mutants. By lengthening the entangled region, refining ColE9 expression, and optimizing the TSM sequence against IS911 insertion, we achieved escape rates below 10-10 as compared to rates of 10-5 with the non-entangled TSM. The STALEMATE system contained plasmids in E. coli Nissle 1917 for over a week in the mouse gastrointestinal tract with nearly undetectable escape rates upon excretion. STALEMATE offers a modular and simple biocontainment approach to buffer against mutational inactivation in the mammalian gut without a requirement for engineered bacteria or exogenous signaling ligands.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.13.670093v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ATP-Powered Signaling between Synthetic and Living Cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sethi, S., Sharma, C., Walther, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce ATP-dissipative delivery of DNA signals from synthetic cells to living cells by exploiting an ATP-driven reaction network that transiently ejects DNA Signal strands from the shielded synthetic cell interior to the extracellular medium of living cells.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> ATP is the energy currency of life and overabundant in the tumor microenvironment, where it has been suggested as a target for cancer therapy. We introduce ATP-dissipative delivery of DNA signals from synthetic cells to living cells by exploiting an ATP-driven reaction network that transiently ejects DNA Signal strands from the shielded synthetic cell interior to the extracellular medium of living cells. We customize the Signal for intracellular uptake or for extracellular instruction using a cytokine-ssDNA chimera that can trigger efficient intracellular downstream signaling programs. Our study discusses details of system design on a timer circuit and synthetic cell level, system integration challenges, and how ATP concentrations regulate the transient delivery. The strategy can be extended to deliver therapeutic oligonucleotides for applications in gene therapy and gene silencing. For cancer therapy, it can use naturally enhanced ATP levels to induce selective delivery of therapeutic oligonucleotides.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.13.670118v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Uncovering Cas9 PAM diversity through metagenomic mining and machine learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fang, T., Bogensperger, L., Feer, L., Allam, A., Bezshapkin, V., Balazs, Z., von Mering, C., Sunagawa, S., Krauthammer, M., Schwank, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present CRISPR-PAMdb, a comprehensive and publicly accessible database compiling Cas9 protein sequences from 3.8 million bacterial and archaeal genomes and PAM profiles from 7.4 million phage and plasmid sequences.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recognition of protospacer adjacent motifs (PAMs) is crucial for target site recognition by CRISPR-Cas systems. In genome editing applications, the requirement for specific PAM sequences at the target locus imposes substantial constraints, driving efforts to search for novel Cas9 orthologs with extended or alternative PAM compatibilities. Here, we present CRISPR-PAMdb, a comprehensive and publicly accessible database compiling Cas9 protein sequences from 3.8 million bacterial and archaeal genomes and PAM profiles from 7.4 million phage and plasmid sequences. Through spacer-protospacer alignment, we inferred consensus PAM preferences for 8,003 unique Cas9 clusters. To extend PAM discovery beyond traditional alignment-based approaches, we developed CICERO, a machine learning model predicting PAM preferences directly from Cas9 protein sequences. Built on the ESM2 protein language model and trained on the CRISPR-PAMdb database, CICERO achieved an average accuracy of 0.68 on test data and 0.75 on experimentally validated Cas9 orthologs. For Cas9 clusters where alignment-based predictions were infeasible, CICERO generated PAM profiles for an additional 50,308 Cas9 proteins, including 17,453 high-confidence predictions with accuracies above 0.86. CRISPR-PAMdb, alongside CICERO models, enables large-scale exploration of PAM diversity across Cas9 proteins, accelerating design of next-generation CRISPR-Cas9 tools for precise genome engineering.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.13.668647v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

