<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Sunday, 01 June 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Project automation in CCP4 Cloud: Enabling customization and high-throughput efficiency.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Krissinel E, Fando M, Kovalevskiy O, Keegan R, Wills J, Lebedev A, Uski V, Ballard C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present new developments in CCP4 Cloud that address these challenges by providing predefined and customizable automatic workflows, which can be seamlessly integrated with experimental facilities.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The rapid advancement of automatic structure solution methods, driven by the availability of high-quality predicted structures from AlphaFold and the growing adoption of multi-crystal and serial experiments, has created a pressing need for streamlining routine operations, automating structure solution projects, and efficiently handling large volumes of data. Modern software solutions must be both robust and user-friendly, supporting manual workflows while enabling high-throughput operations to keep pace with the high data collection rates of modern beamlines. Here, we present new developments in CCP4 Cloud that address these challenges by providing predefined and customizable automatic workflows, which can be seamlessly integrated with experimental facilities, offering a powerful solution for modern macromolecular crystallography. CCP4 Cloud is available as a public service at https://cloud.ccp4.ac.uk.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40411413/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">3-D substructure search by transitive closure in AlphaFold database.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liu H, Laiho A, Törönen P, Holm L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present ISS_ProtSci, a Python package designed for structural similarity searches within the AlphaFold Database v2 (AFDB2).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Identifying structural relationships between proteins is crucial for understanding their functions and evolutionary histories. We present ISS_ProtSci, a Python package designed for structural similarity searches within the AlphaFold Database v2 (AFDB2). ISS_ProtSci incorporates DaliLite to identify geometrically similar structures and uses a transitive closure algorithm to iteratively explore neighboring shells of proteins. The precomputed all-against-all comparisons generated by Foldseek, chosen for its speed, are validated by DaliLite for precision. Search results are annotated with metadata from UniProtKB and Pfam protein family classifications, using hmmsearch to identify protein domains. Outputs, including Dali pairwise alignment data, are provided in TSV format for easy filtering and analysis. Our method offers a significant improvement in recall over existing tools like Foldseek, especially in detecting more distantly related proteins. This is particularly valuable in structurally diverse protein families where traditional sequence-based or fast structural methods struggle. ISS_ProtSci delivers practical runtimes and flexibility, allowing users to input a PDB file, define the minimum size of the common core, and evaluate results using Pfam clans. In evaluating our method across 12 test cases based on Pfam clans, we achieved over 99% recall of relevant proteins, even in challenging cases where Foldseek's recall dropped below 50%. ISS_ProtSci not only identifies closely related proteins but also uncovers previously unrecognized structural relationships, contributing to more accurate protein family classifications. The software can be downloaded from http://ekhidna2.biocenter.helsinki.fi/ISS_ProtSci/.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40400345/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Cell-type-specific patterns and consequences of somatic mutation in development and aging brain</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kriz, A. J., Mao, S., Shao, D. D., Snellings, D. A., Andersen, R., Dong, G., Ma, C. C., Cline, H. E., Huang, A. Y., Lee, E. A., Walsh, C. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed Duplex-Multiome, incorporating duplex consensus sequencing to accurately identify somatic single-nucleotide variants (sSNV) from the same nucleus simultaneously analyzed for single-Nucleus ATAC-seq and RNA-seq .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Elucidating the role of somatic mutations in cancer, healthy tissues, and aging depends on methods that can accurately characterize somatic mosaicism across different cell types, as well as assay their impact on cellular function. Current technologies to study cell-type-specific somatic mutations within tissues are low-throughput. We developed Duplex-Multiome, incorporating duplex consensus sequencing to accurately identify somatic single-nucleotide variants (sSNV) from the same nucleus simultaneously analyzed for single-nucleus ATAC-seq (snATAC-seq) and RNA-seq (snRNA-seq). By introducing strand-tagging into the construction of snATAC-seq libraries, duplex sequencing reduces sequencing error by >10,000-fold while eliminating artifactual mutational signatures. When applied to 98%/2% mixed cell lines, Duplex-Multiome identified sSNVs present in 2% of cells with 92% precision and accurately captured known sSNV mutational spectra, while revealing unexpected subclonal lineages. Duplex-Multiome of > 51,400 nuclei from postmortem brain tissue captured sSNV burdens and spectra across all major brain cell types and subtypes, including those difficult to assay by single-cell whole-genome sequencing (scWGS). This revealed for the first time that diverse neuronal and glial cell types show distinct rates and patterns of age-related mutation, while also directly discovering developmental cell lineage relationships. Duplex-Multiome identified clonal sSNVs occurring at increased rates in glia of certain aged brains, as well as clonal sSNVs that correlated with changes in expression of nearby genes, in both neurotypical and autism spectrum disorder (ASD) individuals, directly demonstrating that somatic mutagenesis can contribute to gene expression phenotypes. Duplex-Multiome can be easily adopted into the 10X Multiome protocol and will bridge somatic mosaicism to a wide range of phenotypic readouts across cell types and tissues.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.30.656844v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">cytoFlagR: A comprehensive framework to objectively assess high-parameter cytometry data for batch effects</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Eswar, S., Koenig, Z. T., Tursi, A. R., Cobena-Reyes, J., Tilburgs, T., Andorf, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce cytoFlagR, a comprehensive tool to flag batch-related problems at the marker and cell cluster level based on robust statistical evaluations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: High-parameter cytometry is widely used in longitudinal studies, but technical variation across batches can confound biological signals. However, tools that objectively identify problematic batches and markers are limited. Results: We introduce cytoFlagR, a comprehensive tool to flag batch-related problems at the marker and cell cluster level based on robust statistical evaluations. Batch and marker variations are assessed based on median signal intensities of negative and positive cell populations and positive cell frequencies, along with Earth Mover's Distance (EMD) of signal intensity distributions. Additionally, cytoFlagR identifies cell type specific batch problems via unsupervised clustering and is suitable for mass and spectral cytometry datasets where it objectively detects distinct types of batch issues. We demonstrated cytoFlagR's utility for assessing datasets that include or lack reference controls. Thus, cytoFlagR improves quality control by objective identification of technical variations that may impact downstream analysis. Availability and Implementation: cytoFlagR is freely available as R scripts with documentation and an example at https://github.com/AndorfLab/cytoFlagR.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.27.656370v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Differences in GenBank and RefSeq annotations may affect genomics data interpretation for Pseudomonas putida KT2440</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> de Siqueira, G. M. V., Eng, T., Mukhopadhyay, A., Guazzaroni, M.-E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that 16% of Pseudomonas putida KT2440 ORFs show differences in their predicted genomic positions across GenBank and RefSeq, despite sharing equivalent locus tag codes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Annotations of genomic features are cornerstone data that support routine workflows in conventional omics analyses in Pseudomonas putida KT2440 and other organisms. The GenBank and the RefSeq versions of the annotated KT2440 genome are two popular resources widely cited in the literature; yet, they originate from distinct prediction pipelines and possess potentially different biological information that is often overlooked. In this study, we systematically compared the features present in these resources and found that approximately 16% of the total of KT2440 ORFs show differences in their predicted genomic positions across GenBank and RefSeq, despite sharing equivalent locus tag codes. Furthermore, we show that these discrepancies can affect the results of high-throughput analyses by processing a collection of RNAseq expression datasets utilizing both annotations. Our findings provide a comprehensive overview of the current state of available resources for genomics research in P. putida KT2440 and highlight a rarely addressed yet widespread potential pitfall in the literature on this organism, with possible implications for other prokaryotes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.27.656432v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Analysis of isobaric quantitative proteomic data using TMT-Integrator and FragPipe computational platform</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chang, H.-Y., Deng, Y., Li, R., Avtonomov, D., Wen, B., Haynes, S. E., Leprevost, F. d. V., Zhang, B., Yu, F., Nesvizhskii, A. I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present TMT-Integrator, a bioinformatics tool for processing quantitation results from TMT and iTRAQ experiments, offering integrative reports at the gene, protein, peptide, and post-translational modification site levels.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Isobaric mass tags, such as iTRAQ and TMT, are widely utilized for peptide and protein quantification in multiplex quantitative proteomics. We present TMT-Integrator, a bioinformatics tool for processing quantitation results from TMT and iTRAQ experiments, offering integrative reports at the gene, protein, peptide, and post-translational modification site levels. We demonstrate the versatility of TMT-Integrator using five publicly available TMT datasets: an E. coli dataset with 13 spike-in proteins, the clear cell renal cell carcinoma (ccRCC) whole proteome and phosphopeptide-enriched datasets from the Clinical Proteomic Tumor Analysis Consortium, and two human cell lysate datasets showcasing the latest advances with the Astral instrument and TMT 35-plex reagents. Integrated into the widely used FragPipe computational platform (https://fragpipe.nesvilab.org/), TMT-Integrator is a core component of TMT and iTRAQ data analysis workflows. We evaluate the FragPipe/TMT-Integrator analysis pipeline's performance against MaxQuant and Proteome Discoverer with multiple benchmarks, facilitated by the bioinformatics tool OmicsEV. Our results show that FragPipe/TMT-Integrator quantifies more proteins in the E. coli and ccRCC whole proteome datasets, identifies more phosphorylated sites in the ccRCC phosphoproteome dataset, and delivers overall more robust quantification performance compared to other tools.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.27.656447v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Full-length isoform constructor (FLIC) - a tool for isoform discovery based on long reads</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kasianova, A. M., Klepikova, A. V., Gusev, O. A., Gazizova, G. R., Logacheva, M. D., Penin, A. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a new pipeline, FLIC (Full-Length Isoform Constructor), based on the long-read transcriptome data and integrates several key features: 1) utilizing biological replicate concordance to filter out noise and artifacts; 2) employing peak calling to precisely identify transcription start and</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Advances in high-throughput sequencing have illuminated a complexity of transcriptome landscape in eukaryotes. An inherent part of this complexity is the presence of multiple isoforms generated by the alternative splicing and the use of alternative transcription start and polyadenylation sites. However, currently available tools have limited capacity to infer full-length isoforms. To address this problem, we developed a new pipeline, FLIC (Full-Length Isoform Constructor). FLIC is based on the long-read transcriptome data and integrates several key features: 1) utilizing biological replicate concordance to filter out noise and artifacts; 2) employing peak calling to precisely identify transcription start and polyadenylation sites; 3) enabling robust isoform reconstruction with minimal reliance on existing annotations. We evaluated FLIC using a dedicated set of real and simulated data of Arabidopsis thaliana cDNA sequencing. Results demonstrate that FLIC accurately reconstructs known and novel isoforms, outperforming existing tools, especially in the absence of reference annotations. A direct comparison with CAGE, currently regarded as golden standard for TSS identification shows that FLIC is equally accurate, while being much less time-consuming. Thus FLIC provides a valuable tool for comprehensive transcript characterization, particularly for non-model organisms or when dealing with incomplete or inaccurate annotations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.27.656444v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Improved quantitative accuracy in data-independent acquisition proteomics via retention time boundary imputation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Harris, L. J., Riffle, M., Noble, W. S., MacCoss, M. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present an alternative approach based on imputing peptide retention times (RTs) rather than quantitations to handle missing values in DIA proteomics .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The traditional approaches to handling missing values in DIA proteomics are to either remove high-missingness proteins or impute them with statistical procedures. Both have their disadvantages--removal can limit statistical power, while imputation can introduce spurious correlations or dilute signal. We present an alternative approach based on imputing peptide retention times (RTs) rather than quantitations. For each missing value, we impute the RT boundaries, then obtain a quantitation by integrating the chromatographic signal within the imputed boundaries. Our method yields more accurate quantitations than existing proteomics imputation methods. RT boundary imputation also identifies differentially abundant peptides from key Alzheimer genes that were not identified with library search alone. RT boundary imputation improves the ability to estimate radiation exposure in biological tissues. RT boundary imputation significantly increases the number of peptides with quantitations, leading to increases in statistical power. Finally, RT boundary imputation better quantifies low abundance peptides than library search alone. Our RT boundary imputation method, called Nettle, is available as a standalone tool.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.27.656394v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Confounding factors in assessing the enriched expression of somatic mutant allele in bulk tumor samples</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hagiwara, K., Thrasher, A., Terekhanova, N. V., Zhang, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that tumor/normal expression difference and low tumor purity are major confounding factors in ASE analysis of somatic mutations and demonstrate the utility of using RNA-Seq for driver mutation analysis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Allele-specific expression (ASE) of somatic mutations can be caused by cis-activation of the mutant allele or silencing of the wildtype allele and has been investigated by examining the enrichment of mutant allele in RNA relative to DNA. Here we show that this mutation-based approach can be confounded by gene expression differences in tumor and normal cells which co-exist in most bulk tumor samples. We modeled mutant allele expression by incorporating tumor/normal expression difference, mutant allele dosage, tumor purity, and nonsense-mediated decay (NMD) efficiency, projecting that the enrichment can occur without ASE. This confounding effect is exacerbated with low tumor purity and is dependent on mutant allele dosage for NMD-sensitive mutations. The model was validated empirically using somatic insertions/deletions (indels) from 9,101 the Cancer Genome Atlas (TCGA) samples, showing three-fold higher enrichment in driver genes vs. non-drivers. To explore if the enriched mutant allele expression can be leveraged to complement DNA-based somatic mutation detection, we performed de novo somatic indel calling using TCGA RNA-Seq, increasing the TCGA driver indel repertoire by ~14%, mostly in samples with lower tumor purity. Our study not only revealed confounding factors in ASE analysis but also demonstrated the utility of leveraging RNA-Seq for driver mutation analysis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.27.656366v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Novel Machine Learning Approach for Tumor Detection Based on Telomeric Signatures</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shah, P., Sethuraman, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a supervised machine learning model trained on telomeric read content, genomic variants, and phenotypic features to predict tumor status, using data from 33 cancer types.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cancer remains one of the most complex diseases faced by humanity, with over 200 distinct types, each characterized by unique molecular profiles that demand specialized therapeutic approaches (Tomczak et al., 2015). Prior studies have shown that both short and long telomere lengths are associated with elevated cancer risk, underscoring the intricate relationship between telomere length variation and tumorigenesis (The Telomeres Mendelian Randomization Collaboration et al., 2017). To investigate this relationship, we developed a supervised machine learning model trained on telomeric read content, genomic variants, and phenotypic features to predict tumor status. Using data from 33 cancer types within The Cancer Genome Atlas (TCGA) program, our model achieved an accuracy of 82.62% in predicting tumor status. The trained model is available for public use and further development through the project's GitHub repository: https://github.com/paribytes/TeloQuest. This work represents a novel, multidisciplinary approach to improving cancer diagnostics and risk assessment by integrating telomere biology with Biobank-scale genomic and phenotypic data. Furthermore, we highlight the potential of telomere length variation as a meaningful predictive biomarker in oncology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.28.656542v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Evidence-driven biases in alternative splicing inferred from NCBI Eukaryotic Genome Annotation Pipeline metadata</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> de la Fuente, R., Diaz-Villanueva, W., Arnau, V., Moya, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We quantify potential biases associated with the EGAP annotation pipeline, and find that specific annotation features strongly influence these estimates, particularly the percentage of coding sequences that are supported by experimental evidence.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The NCBI Eukaryotic Genome Annotation Pipeline (EGAP) predicts coding sequences by integrating transcriptomic and proteomic data with computational approaches, providing structural information that can be used to infer alternative transcript isoforms. However, accurate estimation of alternative splicing events depends on high-quality genome annotations, particularly in genome-wide analyses. The extent to which annotation pipelines influence these inferred splicing patterns has remained largely unexplored. In this study, we quantify potential biases associated with the EGAP annotation pipeline, and find that specific annotation features strongly influence these estimates, particularly the percentage of coding sequences that are supported by experimental evidence. Further, we implemented a polynomial regression model to normalize splicing levels, generating an adjusted metric that minimizes evidence-driven biases. This framework may serve as a basis for future investigations into splicing complexity and comparative genomics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.27.656353v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">EnhancerDetector: Enhancer Discovery from Human to Fly via Interpretable Deep Learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Solis, L. M., Sterling-Lentsch, G., Halfon, M. S., Girgis, H. Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce EnhancerDetector, a convolutional neural network-based framework for cross-species enhancer prediction that combines high accuracy with biological interpretability.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Enhancers are essential non-coding DNA elements that regulate gene expression, yet their accurate identification remains a major challenge. We introduce EnhancerDetector, a convolutional neural network-based framework for cross-species enhancer prediction that combines high accuracy with biological interpretability. Trained on human data, EnhancerDetector achieves strong performance across human, mouse, and fly datasets, consistently outperforming existing methods in precision and F1 score. It generalizes effectively to datasets generated using diverse experimental assays. An ensemble strategy further improves prediction reliability by reducing false positives (critical for genome-wide applications). EnhancerDetector supports fine-tuning on new species and retains strong performance even when adapted with as few as 20,000 enhancer sequences, making it ideal for newly sequenced genomes with limited experimental data. For interpretability and visualization, we apply class activation maps to identify sequence regions predictive of enhancer activity. Experimental validation in transgenic flies confirms the predictive power of EnhancerDetector: five of six tested candidates drove reporter expression, and four exhibited expression patterns supported by prior literature. These analyses highlight distinct sequence and contextual features that confer what we term enhancerness: enhancer sequences possess a characteristic, identifiable signature. Together, these findings position EnhancerDetector as a practical, accurate, and interpretable framework for enhancer discovery across diverse species.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.28.656532v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DNA framework array enables ultra-high throughput DNA synthesis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Li, C., Saimaiti, Y., Li, M., Wang, S., Yin, F., Fang, Z., Jia, S., Wang, F., Ye, D., Wang, L., Zuo, X., Fan, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a DNA framework-based bottom-up enzymatic synthesis strategy that enables single-molecule level control of DNA synthesis with high-throughput.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> High-throughput DNA synthesis has revolutionized synthetic biology, molecular diagnostics, genome engineering, and DNA data storage by enabling the scalable and precise construction of nucleotide-based information systems. However, existing high-throughput synthesis technologies rely on top-down fabrication strategies to maximize throughput, yet they are inherently constrained by physical limitations. These constraints make it exceedingly difficult to achieve single-molecule level synthesis control that is essential for advancing next-generation DNA synthesis. Here, we present a DNA framework-based bottom-up enzymatic synthesis strategy that enables single-molecule level control of DNA synthesis with high-throughput. Leveraging the nanoscale addressability of DNA frameworks, we achieved a synthesis site pitch of 10.9 nm, 183-fold smaller than that of previously reported electrode arrays. Theoretical projections suggest that this approach could scale synthesis throughput to 530 billion sequences per square centimeter, representing a four-order-of-magnitude improvement over existing synthesis arrays. This approach lays the foundation for gigabyte-scale DNA writing and offers a highly accessible pathway toward large-scale DNA synthesis for emerging applications such as molecular data storage.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.30.657018v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Unified Genomic and Chemical Representations Enable Bidirectional Bio-synthetic Gene Cluster and Natural Product Retrieval</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liu, G., Li, Y., Ong, G., Wong, F. T., Tay, D. W. P., Lim, Y. H., Foo, C. S., Koh, L. C. W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a multimodal framework that integrates genomic and chemical information by projecting embeddings derived from pretrained language models into a common representation space.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Natural product discovery is increasingly driven by the ability to analyze microbial genomes for biosynthetic gene clusters (BGCs) that encode secondary metabolites. While existing approaches have successfully linked BGCs to broad classes of chemical products, they typically operate in a single modality (genomic or chemical) limiting the scope of bidirectional prediction. In this work, we propose a multimodal framework that integrates genomic and chemical information by projecting embeddings derived from pretrained language models into a common representation space. We embed genomic sequences using a BGC foundation model and represent molecules through a chemical language model, then use a metric learning model to co-embed BGCs and their associated chemical structures. This co-embedding space allows us to quantify the similarity between BGCs and compounds using similarity measures, enabling both forward and inverse retrieval tasks. Beyond retrieval, we show that the shared space can guide strain selection for targeted compounds. By identifying BGCs closest to a query compound in the embedding space, we prioritize microbial strains that encode similar clusters, thereby streamlining genome mining and retrobiosynthetic design efforts. This approach represents a generalizable, scalable strategy to bridge biological and chemical modalities in natural product discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.31.656985v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DNA Flipping as Facile Mechanism for Transmembrane Signaling in Synthetic Cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Baranda Pellejero, L., Vonk-de Roy, M., Baykal, D., Lehmann, L., Walther, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A simple and robust DNA-based mechanism for transmembrane signaling in synthetic cells using cholesterol-modified single-stranded DNA (Chol-ssDNA).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Transmembrane signaling is essential for cellular communication, yet reconstituting such mechanisms in synthetic systems remains challenging. Here, we report a simple and robust DNA-based mechanism for transmembrane signaling in synthetic cells using cholesterol-modified single-stranded DNA (Chol-ssDNA). We discovered that anchored Chol-ssDNA spontaneously flips across the membrane of giant unilamellar lipid vesicles (GUVs) in a nucleation-driven, defect-mediated process. This flipping enables internal signal processing through hybridization with encapsulated complementary DNA and activation of downstream processes such as RNA transcription. The phenomenon shows a high transduction efficiency, is generic across DNA sequences and lipid compositions, and can be enhanced by glycerol, which modulates membrane dynamics. Mechanistic insights using fluorescence microscopy, nuclease degradation assays, and membrane permeability assays reveal that flipping is dominated by transient membrane pores. Leveraging this facile translocation process, we demonstrate selective transcriptional activation inside synthetic cells, underscoring the potential of CholssDNA flipping as a programmable tool for synthetic biology and bottom-up synthetic cell design.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.31.656672v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Carbon-Efficient Conversion through Hydrogen and Formate-Driven Metabolism in Escherichia coli</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bertrand, R. L., Panich, J., Cowan, A. E., Roberts, J. B., Artier, J., Toppari, E., Baidoo, E. E., Chen, Y., Petzold, C. J., Hudson, G. A., Shih, P. M., Singer, S. W., Keasling, J. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We engineered Escherichia coli to utilize hydrogen gas (H2) and formate (HCOO-) as alternative sources of energy and reducing equivalents, thereby decoupling energy generation from carbon metabolism.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Product yields for biomanufacturing processes are often constrained by the tight coupling of cellular energy generation and carbon metabolism in sugar-based fermentation systems. To overcome this limitation, we engineered Escherichia coli to utilize hydrogen gas (H2) and formate (HCOO-) as alternative sources of energy and reducing equivalents, thereby decoupling energy generation from carbon metabolism. This approach enabled precise suppression of decarboxylative oxidation during acetate growth, with 86.6 +/- 1.6% of electrons from hydrogen gas (via soluble hydrogenase from Cupriavidus necator H16) and 98.4 +/- 3.6% of electrons from formate (via formate dehydrogenase from Pseudomonas sp. 101) offsetting acetate oxidation. Hydrogen gas supplementation led to a titratable and stoichiometric reduction in CO2; evolution in acetate-fed cultures. Metabolomic analysis suggests that this metabolic decoupling redirects carbon flux through the glyoxylate shunt, partially bypassing two decarboxylative steps in the TCA cycle. We demonstrated the utility of this strategy by applying it to mevalonate biosynthesis, where formate supplementation during glucose fermentation increased titers by 57.6% in our best-performing strain. Flux balance analysis further estimated that 99.0 +/- 2.8% of electrons from formate were used to enhance mevalonate production. These findings highlight a broadly applicable strategy for enhancing biomanufacturing efficiency by leveraging external reducing power to optimize carbon and energy use.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.30.657096v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Natural language processing in drug discovery: bridging the gap between text and therapeutics with artificial intelligence.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Christine Ann Withers, Amina Mardiyyah Rufai, Aravind Venkatesan, Santosh Tirunagari, Sebastian Lobentanzer, Melissa Harrison, Barbara Zdrazil</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> The field of Natural Language Processing (NLP) within the life sciences has exploded in its capacity to aid the extraction and analysis of data from scientific texts in recent years through the advancement of Artificial Intelligence (AI).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The field of Natural Language Processing (NLP) within the life sciences has exploded in its capacity to aid the extraction and analysis of data from scientific texts in recent years through the advancement of Artificial Intelligence (AI). Drug discovery pipelines have been innovated and accelerated by the uptake of AI/Machine Learning (ML) techniques.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40298230/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Pato: prediction of probiotic bacteria using metabolic features.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rafaella Sinnott Dias, Daniela Peres Martinez, Fábio Pereira Leivas Leite, Luciana Farias da Costa de Avila, Frederico Schmitt Kremer</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel classification approach based on functional and metabolism-related elements, which offers improved accuracy and explainability compared to traditional k-mer-based methods, and a clearer understanding of the traits associated with probiotic activity.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Probiotics have gained recognition for their health-promoting benefits, particularly in the gastrointestinal and immunological systems. Among promising probiotic candidates, Lactobacillus strains, belonging to the lactic acid bacteria (LAB) group, play a significant role in human microbiota. To aid in the in silico identification of Lactobacillus strains with probiotic potential, this study presents a novel classification approach based on functional and metabolism-related elements, which offers improved accuracy and explainability compared to traditional k-mer-based methods. By considering the functional characteristics of genomic sequences, this approach contributes to a clearer understanding of the traits associated with probiotic activity, facilitating the selection of strains with optimal health-promoting attributes. The webserver is available at http://200.132.101.156:5001/ .</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40266486/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Deep learning and genomic best linear unbiased prediction integration: An approach to identify potential nonlinear genetic relationships between traits.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> F Shokor, P Croiseau, H Gangloff, R Saintilan, T Tribout, T Mary-Huard, B C D Cuyabano</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We proposed a novel hybrid model that combines both DL and GBLUP, which uses the output of the traditional G BLUP, and enhances its predicted genetic values (PGV) by accounting for nonlinear genetic relationships between traits using DL.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genomic prediction (GP) aims to predict the breeding values of multiple complex traits, usually assumed to be multivariate normally distributed by the largely used statistical methods, thus imposing linear genetic relationships between traits. Although these methods are valuable for GP, they do not account for potential nonlinear genetic relationships between traits in scenarios. For individual traits, this oversight may minimally affect prediction accuracy, but it can limit genetic progress when selection involves multiple traits. Deep learning (DL) offers a promising alternative for capturing nonlinear genetic relationships due to its ability to identify complex patterns without prior assumptions about the data structure. We proposed a novel hybrid model that that combines both DL and GBLUP (DLGBLUP), which uses the output of the traditional GBLUP, and enhances its predicted genetic values (PGV) by accounting for nonlinear genetic relationships between traits using DL. We simulated data with linear and nonlinear genetic relationships between traits in order to verify whether DLGBLUP was able to identify nonlinearity when present and avoid inducing it when absent. We found that DLGBLUP consistently provided more accurate PGV for traits simulated with strong nonlinear genetic relationships, accurately identifying these relationships. Over 7 generations of selection, a greater genetic progress was achieved with PGV that accounted for nonlinear relationships (DLGBLUP), compared with GBLUP. When applied to a real dataset from the French Holstein dairy cattle population, DLGBLUP detected nonlinear genetic relationships between pairs of traits, such as conception rate and protein content, and SCC and fat yield, although, no significant increase in prediction accuracy was observed. The integration of DL into GP enabled the modeling of nonlinear genetic relationships between traits, a possibility not previously discussed, given the linear nature of GBLUP. The detection of nonlinear genetic relationships between traits in the French Holstein population when using DLGBLUP indicates the presence of such relationships in real breeding data, suggesting that it may be relevant to further explore nonlinear relationships. This possibility of nonlinear genetic relationships between traits offers a different perspective into multitrait evaluations, with potential to further improve selection strategies in commercial livestock breeding programs. This is particularly relevant when integrating new traits into multitrait evaluations or incorporating new subpopulations, which may introduce different forms of nonlinearity. Finally, it is shown that DL can be used as a complement to the statistical methods deployed in routine genetic evaluations, rather than as an alternative, by enhancing their performance.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40252763/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identification and Validation of Circadian Rhythm-Related Genes Involved in Intervertebral Disc Degeneration and Analysis of Immune Cell Infiltration via Machine Learning.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yongbo Zhang, Liuyang Chen, Sheng Yang, Rui Dai, Hua Sun, Liang Zhang</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study on the influence of circadian rhythms and immunity on intervertebral disc degeneration .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Low back pain is a significant burden worldwide, and intervertebral disc degeneration (IVDD) is identified as the primary cause. Recent research has emphasized the significant role of circadian rhythms (CRs) and immunity in affecting intervertebral discs (IVD). However, the influence of circadian rhythms and immunity on the mechanism of IVDD remains unclear. This study aimed to identify and validate key rhythm-related genes in IVDD and analyze their correlation with immune cell infiltration.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40225045/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Uncovering hepatic transcriptomic and circulating proteomic signatures in MASH: A meta-analysis and machine learning-based biomarker discovery.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Elena Cristina Rusu, Helena Clavero-Mestres, Mario Sánchez-Álvarez, Marina Veciana-Molins, Laia Bertran, Pablo Monfort-Lanzas, Carmen Aguilar, Javier Camaron, Teresa Auguet</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-12
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A machine learning-based biomarker discovery model for metabolic-associated steatohepatitis .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Metabolic-associated steatohepatitis (MASH), the progressive form of metabolic-associated steatotic liver disease (MASLD), poses significant risks for liver fibrosis and cardiovascular complications. Despite extensive research, reliable biomarkers for MASH diagnosis and progression remain elusive. This study aimed to identify hepatic transcriptomic and circulating proteomic signatures specific to MASH, and to develop a machine learning-based biomarker discovery model.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40220593/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Data independent acquisition proteomics and machine learning reveals that proteins associated with immunity are potential molecular markers for early diagnosis of autism.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Erlin Hu, Xiaoni Kuang, Sun Zhaohui, Sifeng Wang, Tuoyu Gan, Wenjuan Zhou, Zhu Ming, Yuxia Cheng, Chunhua Ye, Kang Yan, Xiaohui Gong, Tuanmei Wang, Xiangwen Peng</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Scientists have identified a new molecular marker for autism, which could be used to help diagnose the condition in children.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Early diagnosis of autism is critical to its treatment, but so far, there is no clear molecular marker for early diagnosis in children.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40185380/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Efficient identification of de novo mutations in family trios: a consensus-based informatic approach.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mariya Shadrina, Özem Kalay, Sinem Demirkaya-Budak, Charles A LeDuc, Wendy K Chung, Deniz Turgut, Gungor Budak, Elif Arslan, Vladimir Semenyuk, Brandi Davis-Dusenbery, Christine E Seidman, H Joseph Yost, Amit Jain, Bruce D Gelb</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-03-28
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A consensus-based approach for identifying de novo variants in proband-parent trios using short-read genome sequencing data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurate identification of de novo variants (DNVs) remains challenging despite advances in sequencing technologies, often requiring ad hoc filters and manual inspection. Here, we explored a purely informatic, consensus-based approach for identifying DNVs in proband-parent trios using short-read genome sequencing data. We evaluated variant calls generated by three sequence analysis pipelines-GATK HaplotypeCaller, DeepTrio, and Velsera GRAF-and examined the assumption that a requirement of consensus can serve as an effective filter for high-quality DNVs. Comparison with a highly accurate DNV set, validated previously by manual inspection and Sanger sequencing, demonstrated that consensus filtering, followed by a force-calling procedure, effectively removed false-positive calls, achieving 98.0-99.4% precision. At the same time, sensitivity of the workflow based on the previously established DNVs reached 99.4%. Validation in the HG002-3-4 Genome-in-a-Bottle trio confirmed its robustness, with precision reaching 99.2% and sensitivity up to 96.6%. We believe that this consensus approach can be widely implemented as an automated bioinformatics workflow suitable for large-scale analyses without the need for manual intervention, especially when very high precision is valued over sensitivity.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40155050/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Imaging and spatially resolved mass spectrometry applications in nephrology.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Brittney L Gorman, Catelynn C Shafer, Nagarjunachary Ragi, Kumar Sharma, Elizabeth K Neumann, Christopher R Anderton</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-03-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Sp spatially resolved mass spectrometry and MS imaging methods can fill much of the omics gap in systems biology analysis of the kidney and provide functional outputs that cannot be obtained using genomics and transcriptomic methods.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The application of spatially resolved mass spectrometry (MS) and MS imaging approaches for studying biomolecular processes in the kidney is rapidly growing. These powerful methods, which enable label-free and multiplexed detection of many molecular classes across omics domains (including metabolites, drugs, proteins and protein post-translational modifications), are beginning to reveal new molecular insights related to kidney health and disease. The complexity of the kidney often necessitates multiple scales of analysis for interrogating biofluids, whole organs, functional tissue units, single cells and subcellular compartments. Various MS methods can generate omics data across these spatial domains and facilitate both basic science and pathological assessment of the kidney. Optimal processes related to sample preparation and handling for different MS applications are rapidly evolving. Emerging technology and methods, improvement of spatial resolution, broader molecular characterization, multimodal and multiomics approaches and the use of machine learning and artificial intelligence approaches promise to make these applications even more valuable in the field of nephology. Overall, spatially resolved MS and MS imaging methods have the potential to fill much of the omics gap in systems biology analysis of the kidney and provide functional outputs that cannot be obtained using genomics and transcriptomic methods.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40148534/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Can AI reveal the next generation of high-impact bone genomics targets?</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Casey S Greene, Christopher R Gignoux, Marc Subirana-Granés, Milton Pividori, Stephanie C Hicks, Cheryl L Ackert-Bicknell</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-03-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We review success stories including PCSK9-related drugs in cardiovascular disease and evidence supporting the use of human genetics to guide drug discovery, while highlighting advances in artificial intelligence and machine learning with the potential to improve target discovery in skeletal biology.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genetic studies have revealed hundreds of loci associated with bone-related phenotypes, including bone mineral density (BMD) and fracture risk. However, translating discovered loci into effective new therapies remains challenging. We review success stories including PCSK9-related drugs in cardiovascular disease and evidence supporting the use of human genetics to guide drug discovery, while highlighting advances in artificial intelligence and machine learning with the potential to improve target discovery in skeletal biology. These strategies are poised to improve how we integrate diverse data types, from genetic and electronic health records data to single-cell profiles and knowledge graphs. Such emerging computational methods can position bone genomics for a future of more precise, effective treatments, ultimately improving the outcomes for patients with common and rare skeletal disorders.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40225702/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Microscope Upcycling: Transforming legacy microscopes into automated cloud-integrated imaging systems.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Drew Ehrlich, Yohei Rosen, David F Parks, Kivilcim Doganyigit, Ryan Fenimore, Samira Vera-Choqqueccota, Sebastian Hernandez, Anna Toledo, David Haussler, Sri Kurniawan, Mircea Teodorescu</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-03-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We bring the features of modern computerized microscopes to decades-old legacy laboratory inverted microscopes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Computerized microscopes improve repeatability, throughput, antisepsis, data analysis and data sharing in the biological laboratory, but these machines are cost-prohibitive in most academic environments. This is a barrier into collecting the large and consistent datasets required for machine learning analyses of microscopy data. We demonstrate hardware modifications and software to bring the features of modern computerized microscopes to decades-old legacy laboratory inverted microscopes. We demonstrate automation of X-Y positioning, focus stacking, image acquisition and image storage.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40213781/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Leveraging FDA Labeling Documents and Large Language Model to Enhance Annotation, Profiling, and Classification of Drug Adverse Events with AskFDALabel.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Leihong Wu, Hong Fang, Yanyan Qu, Joshua Xu, Weida Tong</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-02-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Drug adverse events (AEs) represent a significant public health concern, but manual extraction of AEs is labor-intensive, requires specialized expertise, and is challenging to maintain, due to frequent updates of the labeling documents.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Drug adverse events (AEs) represent a significant public health concern. US Food and Drug Administration (FDA) drug labeling documents are an essential resource for studying drug safety such as assessing a drug's likelihood to cause certain organ toxicities; however, the manual extraction of AEs is labor-intensive, requires specialized expertise, and is challenging to maintain, due to frequent updates of the labeling documents.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/39979771/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Protein identification using Cryo-EM and artificial intelligence guides improved sample purification.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kenneth D Carr, Dane Evan D Zambrano, Connor Weidle, Alex Goodson, Helen E Eisenach, Harley Pyles, Alexis Courbet, Neil P King, Andrew J Borst</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-01-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify and remove a contaminant from protein purification by using cryo-EM, Cryo-electron microscopy, model-agnostic model-building, protein BLAST, and AlphaFold 3 to identify and identify a co-purifying protein.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein purification is essential in protein biochemistry, structural biology, and protein design, enabling the determination of protein structures, the study of biological mechanisms, and the characterization of both natural and de novo designed proteins. However, standard purification strategies often encounter challenges, such as unintended co-purification of contaminants alongside the target protein. This issue is particularly problematic for self-assembling protein nanomaterials, where unexpected geometries may reflect novel assembly states, cross-contamination, or native proteins originating from the expression host. Here, we used an automated structure-to-sequence pipeline to first identify an unknown co-purifying protein found in several purified designed protein samples. By integrating cryo-electron microscopy (Cryo-EM), ModelAngelo's sequence-agnostic model-building, and Protein BLAST, we identified the contaminant as dihydrolipoamide succinyltransferase (DLST). This identification was validated through comparisons with DLST structures in the Protein Data Bank, AlphaFold 3 predictions based on the DLST sequence from our E. coli expression vector, and traditional biochemical methods. The identification informed subsequent modifications to our purification protocol, which successfully excluded DLST from future preparations. To explore the potential broader utility of this approach, we benchmarked four computational methods for DLST identification across varying resolution ranges. This study demonstrates the successful application of a structure-to-sequence protein identification workflow, integrating Cryo-EM, ModelAngelo, Protein BLAST, and AlphaFold 3 predictions, to identify and ultimately help guide the removal of DLST from sample purification efforts. It highlights the potential of combining Cryo-EM with AI-driven tools for accurate protein identification and addressing purification challenges across diverse contexts in protein science.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/39958810/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

