<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Friday, 26 September 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">HKDE-LACM: a hybrid model for lactic acid bacteria classification via k-mer and DNABERT-2 embedding fusion with cyclic DE-BO optimization.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jie Zou, Weichi Liu, Jinhui Dai, Gaifang Dong</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new method for genome sequence classification of lactic acid bacteria .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Lactic acid bacteria (LAB) play vital roles in food production and clinical applications. Accurate classification of LAB strains facilitates their functional development and targeted utilization. Although machine learning and deep learning methods have been widely applied to genome sequence classification, challenges remain in capturing comprehensive feature representations and enhancing model generalizability.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40999328/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">TEtrimmer: a tool to automate the manual curation of transposable elements.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jiangzhao Qian, Hang Xue, Shujun Ou, Ludwig Mann, Jessica Storer, Lisa Fürtauer, Tony Heitkam, Mary C Wildermuth, Stefan Kusch, Ralph Panstruga</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present TEtrimmer, a tool that automates and replaces key steps of traditional manual curation of transposable elements (TEs) in genomes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Transposable elements (TEs) are repetitive DNA sequences that move within genomes and play important roles in gene regulation and genome evolution. Accurate TE annotation in genomes is crucial for downstream analyses but challenging due to their sequence diversity and frequent fragmentation, including the occurrence of nested copies. We here present TEtrimmer, a tool that automates and replaces key steps of traditional manual curation of TEs. TEtrimmer combines phylogenetic tree analysis with the machine learning method DBSCAN to cluster TE sequences accurately and applies a sliding-window strategy to remove poorly conserved regions of TE-derived multiple sequence alignments. TEtrimmer also provides detailed report plots and features a graphical user interface (GUI) application. Tested on the genomes of six organisms belonging to various kingdoms of eukaryotic life and three simulated genomes, TEtrimmer consistently improved the identification of intact TEs compared to the established tools EDTA and RepeatModeler2.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40998822/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Predicting rhizosphere-competence-related catabolic gene clusters in plant-associated bacteria with rhizoSMASH.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yuze Li, Mingxue Sun, Jos M Raaijmakers, Liesje Mommer, Fusuo Zhang, Chunxu Song, Marnix H Medema</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We develop a genome-synteny-based algorithm for genome-based annotation of rhizosphere-competence-related catabolic gene clusters (rCGCs) in bacteria with 58 curated detection rules.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Plants release a substantial fraction of their photosynthesized carbon into the rhizosphere as root exudates that drive microbiome assembly. Deciphering how plants modulate the composition and activities of rhizosphere microbiota through root exudates is challenging, as no dedicated computational methods exist to systematically identify microbial root exudate catabolic pathways. Here, we integrate published information on catabolic genes in bacteria that contribute to their rhizosphere competence and develop the rhizoSMASH algorithm for genome-synteny-based annotation of rhizosphere-competence-related catabolic gene clusters (rCGCs) in bacteria with 58 curated detection rules. Our analysis reveals heterogeneity in rCGC prevalence both across and within plant-associated bacterial taxa, indicating extensive niche specialization. Furthermore, we demonstrate the predictive value of the presence or absence of rCGCs for rhizosphere competence in machine learning with two case studies. rhizoSMASH provides an extensible framework for studying rhizosphere bacterial catabolism, facilitating microbiome-assisted breeding approaches for sustainable agriculture.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40998779/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Phylogenomic Approaches to Study Adaptive Evolution in Mammals: From Aging to Aquatic Lifestyles.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nathan L Clark, Amanda Kowalczyk, Emily E K Kopania, Maria Chikina</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> The natural world is full of valuable lessons about genetic adaptation as organisms respond to changing conditions around them.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The natural world is full of valuable lessons about genetic adaptation as organisms respond to changing conditions around them. Deciphering these changes is a major goal of evolutionary genetics. Advances have been made through phylogenomic approaches using the wealth of closely related genome sequences in mammals. These studies bring us lessons about the adaptive capacity allowed by the evolutionary process as well as the underlying genetic mechanisms controlling important traits. Diverse methods are now routinely used to identify the genetic basis of these adaptations. These reveal new functions of genes and regulatory regions that have responded to changes in lifestyle, such as aquatic life and flight, as well as major life history axes, such as lifespan. Phylogenomic studies have been equally revealing of specific traits that evolve in response to different selective pressures, such as hair formation and vocal learning. These approaches continue to develop to overcome challenges inherent in information-poor regulatory regions to find changes to gene regulatory networks as well. The development of these approaches is expected to accelerate as new tools, such as machine learning models, are incorporated and deployed on ever denser phylogenies containing new interesting traits.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40997294/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Applications of New Generation Sequencing (NGS) in Ocular Oncology.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mizuki Tagami, Shigeru Honda</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Next-generation sequencing (NGS) has revolutionized cancer genomics, offering unparalleled insights into the molecular landscape of various malignancies, including ocular cancers, and its impact on understanding genetic alterations, identifying biomarkers, and advancing personalized treatment approaches.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Next-generation sequencing (NGS) has revolutionized cancer genomics, offering unparalleled insights into the molecular landscape of various malignancies, including ocular cancers. This review explores the role of NGS in ocular oncology, highlighting its impact on understanding genetic alterations, identifying biomarkers, and advancing personalized treatment approaches. Key applications in uveal melanoma, retinoblastoma, and other ocular tumors, including ocular adnexa lymphoma and IgG4-related ophthalmic disease are discussed, along with the challenges and future directions in the field. As sequencing technologies continue to evolve, integrating NGS into clinical practice promises to enhance early diagnosis, risk assessment, and therapeutic strategies in ocular oncology.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40993891/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Discrete Subdomains Establish Epigenetic Diversity in Subtelomeric Heterochromatin</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mazumder, A., Cooper, J., Goksal, C., Khanduja, J. S., Joh, R. I., Groos, J. J., Brockhausen, R. Y., Kanoh, J., Motamedi, M., Finkelstein, I. J., Al-Sady, B., Braun, S. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that subtelomeric heterochromatin is organized into discrete subdomains that nucleate at telomere-proximal and cryptic internal sites.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Subtelomeres are imperfect repeats adjacent to telomeres that are repressed by heterochromatin. Although essential for genome integrity, their repetitive nature has thwarted dissection of local heterochromatin assembly and maintenance mechanisms. Here, we engineered Schizosaccharomyces pombe strains carrying fluorescent reporters at a single subtelomere. We find that subtelomeric heterochromatin is organized into discrete subdomains that nucleate at telomere-proximal and cryptic internal sites. Telomere-proximal regions depend on canonical shelterin or RNA interference nucleation pathways, while telomere-distal regions require nucleosome remodelers, histone chaperones, and boundary-associated factors. Using multi-generational live imaging and targeted perturbations, we show that subtelomeric subdomains display position-specific, clonally variable silencing across a spectrum of robust to fragile epigenetic states. This clonal variegation is also induced by naturally occurring subtelomeric structural variants. These findings demonstrate that subtelomeric heterochromatin maintenance is not uniform but rather governed by local chromatin context and architecture.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.25.678047v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Endogenous variability in transcription factor concentrations shapes their genome-wide occupancy</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mizeret, R., Tollenaere, A., Deluz, C., Suter, D. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that NANOG occupancy increases monotonously with its concentration, even though a common set of pluripotency regions made accessible by OCT4 and SOX2 are highly and uniformly bound at all NanOG concentrations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The control of transcription factor (TF) concentrations is crucial for the precise regulation of gene expression and cell fate decisions during development. Yet, TF concentrations can display substantial temporal fluctuations and intercellular variations. How TF levels quantitatively shape genome-wide occupancy patterns remains largely unexplored. Here, we systematically investigate how physiological fluctuations in the concentrations of the pluripotency TFs OCT4, SOX2, and NANOG impact their genomic binding profiles in mouse embryonic stem cells. We uncover distinct concentration-dependent binding behaviors for each TF. NANOG occupancy increased monotonously with its concentration, even though a common set of regions made accessible by OCT4 and SOX2 are highly and uniformly bound at all NANOG concentrations. In contrast, SOX2 occupancy does not increase with its concentrations, while OCT4 displayed an intermediate behavior. Our results challenge the notion that TF binding follows simple mass-action dynamics and reveal a core set of pluripotency regions that are bound by all three TFs at all TF concentrations, revealing how pluripotency can be maintained despite marked fluctuations in core pluripotency TFs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.24.678270v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A principled approach to the identification of biologically relevant transcription factor binding sites</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pillai, M., Amaral, L. A. N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We develop a two-state mechanistic model that captures the positional distribution of both biologically relevant and spurious binding events, allowing us to identify interaction distance thresholds that minimize spurious binding and increase reproducibility of analysis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Transcription factors (TF) regulate gene expression in cells by binding to specific DNA sequences. However, previous studies indicate that TF binding is a necessary but not sufficient condition for regulating gene expression. While recent advances in sequencing technologies such as Chromatin Immunoprecipitation sequencing (ChIP-seq) enable researchers to identify binding sequences at single nucleotide resolution, they often fail to distinguish between biologically relevant and non-relevant - i.e., spurious - binding events. Reliably distinguishing between these two types of binding is essential if one aims for reproducible findings. Here, we report on a systematic study of the sensitivity of "called" TF-target gene interactions to the choice of various data processing parameters. We find that acceptable distance from the transcription start site (TSS) is a critical parameter for reducing subjectivity. We, thus, develop a two-state mechanistic model that captures the positional distribution of both biologically relevant and spurious binding events, allowing us to identify interaction distance thresholds that minimize spurious binding. We validate our model with independent data by showing that biologically relevant binding events identified by our model are capable of recruiting RNA Polymerase II and, subsequently, regulating mRNA expression levels of the target gene. Finally, we investigate the impact of our approach on the detection of genes that are differentially bound and regulated by BCL3, a transcription coactivator, during molecular perturbation experiments. Within the genes expected to show differential expression, our model predicts that over 70% of reported interactions are spurious and we demonstrate the lack of change in mRNA expression in these genes following perturbation. We also identify genes that display a shift from spurious to biologically relevant binding that account for close to 17% of the genes displaying a sharp change in mRNA expression and strong association with known functions of BCL3 such as cell cycle regulation and DNA damage response. By providing a systematic TF-specific method to identify binding events that are directly involved in transcriptional regulation our model will increase reproducibility of analysis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.25.678593v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Ciprofloxacin resistance in Klebsiella pneumoniae: phenotype prediction from genotype and global distribution of resistance determinants</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tsang, K. K., Amke, I., Aanensen, D. M., Abdulahi, J., Aiken, A., Bachman, M. A., Baker, S., Barry, K., Biffignandi, G. B., Belitardo, E. M. M. d. A., Bloise, I., Boutiba, I., Brand, C., Brisse, S., Campino, S., Canton, R., Chiaverini, A., Cirillo, D. M., Clark, T. G., Coque, T. M., Corander, J., Corbella, M., Cornacchia, A., Cornick, J., Correia, A., Cuenod, A., Vo, T. T. D., D'Alterio, N., David, S., Di Marco, F., Donado-Godoy, P., Draper, J., Egli, A., Farzana, R., Feasey, N. A., Feil, E. J., Ferrando, M. L., Forde, B. M., Fostervold, A., Foster-Nyarko, E., Giani, T., Gorrie, C. L., Gütlin</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a rules-based classifier to predict ciprofloxacin resistance phenotypes from whole genome sequence data and demonstrate the value of aggregating genotype-phenotype data to explore resistance mechanisms and develop predictors.</p><p style="font-size: 16px; text-align: justify;">BACKGROUND Ciprofloxacin resistant Klebsiella pneumoniae is common or emerging in many geographies, and knowledge of local resistance rates is important for empirical therapy. Whilst there are known K. pneumoniae ciprofloxacin resistance determinants, there is a lack of systematic data on the effect of determinants, alone and in combination, and there are no publicly accessible tools for predicting resistance from whole genome sequence data. METHODS The KlebNET-GSP AMR Genotype-Phenotype Group aggregated a matched genotype-phenotype dataset of n=12,167 K. pneumoniae species complex (KpSC) isolates from 27 countries between 2001-2021. We developed a rules-based classifier to predict ciprofloxacin resistance by categorizing the number of quinolone resistance determining regions mutations in gyrA and parC, the number of plasmid-mediated quinolone resistance genes, and the presence/absence of aac(6')-Ib-cr (which can acetylate ciprofloxacin). Predictive performance was assessed using the discovery dataset, for which we re-phenotyped discrepant isolates; and validated using externally contributed datasets (n=7,030 KpSC isolates). RESULTS The rules-based classifier predicted R vs S/I with categorical agreement, sensitivity, and specificity >96%, and major/very major error rates <4%. Performance was similar across diverse KpSC sources (human, animal, other), species, and intra-species lineages. External validation of the classifier yielded overall 93.12% categorical agreement [95% confidence interval (CI), 92.50-93.74%], 8.65% major errors [95% CI, 7.34-9.97%], and 6.20% very major errors [95% CI, 5.51-6.90%]. We implemented the classifier in Kleborate, a command-line tool that is integrated into the Pathogenwatch web platform. Using this to assess the global distribution of ciprofloxacin resistance determinants in KpSC genomes available in Pathogenwatch (n=31,319, from 109 countries between years 2000-2023), we observed a significant positive association between national quinolone consumption rates and predicted ciprofloxacin resistance (R2=0.20, p=0.004). CONCLUSIONS Ciprofloxacin resistance phenotypes can be reasonably predicted from genotypes, which is sufficient for informing surveillance. However, unexplained resistance remains and accuracy is insufficient for clinical applications. We demonstrate the value of aggregating genotype-phenotype data to explore resistance mechanisms and develop predictors, but highlight complexities in combining phenotype data from different assays and standards.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.24.678318v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">OT-RMSD: A Generalization of the Root-Mean-Square Deviation for Aligning Unequal-Length Protein Structures</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hu, Y., Cao, Z., Liu, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce OT-RMSD, a novel, parameter-light method that generalizes the Root-Mean-Square Deviation metric for comparing unequal-length protein structures without a pre-defined alignment.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The Root-Mean-Square Deviation (RMSD) metric, coupled with the Kabsch algorithm for optimal superposition, represents a cornerstone of quantitative structural biology. However, its application is fundamentally limited to the comparison of structures with a pre-defined, one-to-one correspondence of equal length, precluding its direct use for the vast majority of biologically relevant comparisons involving proteins of different lengths or unknown equivalences. To overcome this limitation, we introduce OT-RMSD, a novel, parameter-light method that generalizes the RMSD for comparing unequal-length protein structures without a pre-defined alignment. Our approach recasts the correspondence problem as an Optimal Transport (OT) task, utilizing a fundamental, distance-based cost function to identify the most economical mapping between two protein structures. This transport plan is then used as a soft-weighting scheme within an iterative Sinkhorn-Kabsch framework to simultaneously solve for correspondence and optimal superposition. We test our method on a benchmark dataset of 40 challenging protein pairs and demonstrate its superiority over both the widely-used PyMOL 'align' command and a more complex, heuristic-driven variant of our own framework. On average, OT-RMSD identifies larger and more accurate alignment cores, achieving a significantly lowerRMSD. We conclude that OT-RMSD provides a mathematically principled, robust, and effective extension of the classic RMSD concept, offering a powerful tool for modern structural analysis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.23.678064v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Sensitive, direct detection of non-coding off-target base editor unwinding and editing in primary cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, T., Jessa, S., Marinov, G. K., Klemm, S., Kundaje, A., Greenleaf, W. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We develop and deploy a direct, in cellulo sequencing assay that simultaneously measures both Cas9-mediated unwinding and deaminase editing of genomic DNA (beCasKAS). Our strategy nominates >460-fold more potential off-target sites than other methods by enriching for Cas9</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Base editors create precise nucleotide changes in DNA, but their off-target activity remains challenging to quantify. Here, we develop and deploy a direct, in cellulo sequencing assay that simultaneously measures both Cas9-mediated unwinding and deaminase editing of genomic DNA (beCasKAS). Our strategy nominates >460-fold more potential off-target sites than other methods by enriching for Cas9-dependent R-loops immediately preceding editing. Using beCasKAS in primary human T-cells, we observe that mRNA-encoded ABE8e and PAMless ABE8e-SpRY base editors have distinct off-target profiles that can be mitigated by optimizing mRNA dose. Finally, we combine beCasKAS with base-resolution deep learning models to risk-stratify off-target edits by their likelihood of epigenetic dysregulation. Collectively, beCasKAS offers a sensitive and facile tool to optimize the balance between base editor on- and off-target activity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.25.678665v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AVITI sequencing of a four-generation CEPH/Utah pedigree confirms low mutation rates at homopolymer loci despite their low sequence complexity</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Happ, H. C., Sasani, T. A., Warner, D., Neklason, D. W., Quinlan, A. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that homopolymer loci exhibit lower mutation rates and lack strong paternal age effects compared to other STRs, likely owing to the combination of a lower propensity to form slippage-causing secondary structures and more efficient mismatch repair.</p><p style="font-size: 16px; text-align: justify;">Background: Short tandem repeats (STRs) and homopolymers are among the most mutable loci in the human genome. Despite their presumed mutability owing to replication slippage, homopolymer loci exhibit lower mutation rates and minimal paternal age effects compared to other STRs. This paradox questions if technical limitations, rather than biological mechanisms, explain these observations. Results: We used the Element Biosciences AVITI platform to sequence the genomes of a 48-member, four-generation CEPH/Utah pedigree. As the AVITI platform reduces error rates at repetitive sequences compared to Illumina, this design enabled accurate mutation discovery at 90% of assayed homopolymers and a 1.7-fold increase in discoverable mutations compared to Illumina. We identified a median of 35 de novo homopolymer mutations per trio and a mutation rate of 5.28 x 10-5 DNMs per locus per generation, confirming a lower rate than dinucleotides (1.94 x 10-4). Most DNMs were single base-pair expansions or contractions. Despite comprising <1% of homopolymer loci, G/C homopolymers showed 18-fold higher mutation rates than A/T homopolymers; in contrast, the high dinucleotide mutation rate is not driven by a particular motif class. Parent-of-origin analysis revealed 78% of homopolymer mutations are paternal in origin, but no significant paternal age effect was observed. Conclusions: This study confirms that homopolymers exhibit lower mutation rates and lack strong paternal age effects compared to other STRs, likely owing to the combination of a lower propensity to form slippage-causing secondary structures and more efficient mismatch repair. Our set of high-quality mutations suggest these phenomena are biological rather than technical in nature. Finally, we demonstrate that AVITI sequencing unlocks previously intractable regions of the genome and will be a powerful tool for continued investigation of repeat mutation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.25.678675v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Screen-VarCal: An Interpretable Probabilistic Framework for Recalibrating ACMG Rule-Based Variant Classification in Preventive Medicine</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mishra, D., Tiwari, A., Srivastava, S., Tripathi, M. B., Kapoor, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Screen-VarCal, an interpretable probabilistic framework that combines logistic regression with isotonic adjustment to align InterVar outputs with observed ClinVar P/LP distributions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: Whole-genome sequencing (WGS) is increasingly used for preventive genomics, yet rule-based ACMG engines such as InterVar were tuned for high pre-test probability diagnostics. In screening contexts, these heuristics can inflate pathogenic/likely pathogenic (P/LP) calls, prompting unnecessary follow-up. We sought an interpretable, data- driven recalibration tailored to proactive use. Results: Across 20 WGS cases, InterVar flagged 109 variants as P/LP; only 18 (16.5%) were concordant with ClinVar P/LP assertions. The remaining 83.5% were largely absent from ClinVar (n=68) or mapped to benign/likely benign (n=13). Nearly all flagged variants (96%) were heterozygous, predominantly in autosomal recessive genes (e.g., FAM20C, MTMR2), indicating a dominant carrier inflation effect; recurrent loci (e.g., ATXN3, FAM20C) further amplified yield. We introduce Screen-VarCal, an interpretable probabilistic framework that combines logistic regression with isotonic adjustment to align InterVar outputs with observed ClinVar P/LP distributions. Screen-VarCal reduced false positives by [~]60% while retaining all ClinVar-concordant findings and yields calibrated probabilities with coefficients that transparently link ACMG evidence categories (PVS/PS/PM/PP/BS/BP) and zygosity to risk.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.23.678112v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Exploring endothelial cell environments across organs in spatially resolved omics data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jain, Y., Jepson, J., Chen, R., Maier, E., Herr, B. W., Puig-Barbe, A., Quardokus, E. M., Qaurooni, D., Yapp, C., Ewing, S. L., Enninful, A., Farzad, N., Bueckle, A., Easter, Q. T., Matuck, B., Zhu, C., Monte, E. M., Purkerson, J. M., Jehrio, M., Misra, R. S., Fan, R., Ginty, F., Karunamurthy, A., Fan, J., Campbell-Thompson, M., Pryhuber, G. S., Byrd, K. M., Hickey, J. W., Börner, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> This paper presents cell-to-nearest-endothelial-cell distance distributions for various cell types using 399 spatially resolved omics datasets from 14 studies comprising 12 tissue types with a total of 47,349,496 cells.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Endothelial cells are ubiquitously present in the human body and line the luminal surface of blood and lymphatic vessels. The oxygen-dependence of cells impacts their proximity to blood vessels, and consequently, to endothelial cells depending on their functional properties and priorities. This paper presents cell-to-nearest-endothelial-cell distance distributions for various cell types using 399 spatially resolved omics datasets from 14 studies comprising 12 tissue types with a total of 47,349,496 cells. Additionally, we developed an open-source web-based interactive tool, Cell Distance Explorer, that allows researchers to interactively visualize cell graphs and linkages in 2D and 3D datasets. Finally, we present a hierarchical neighborhood analysis focused on the endothelial cell neighborhoods in small and large intestine datasets. This paper provides an open-access resource (datasets, tools, and analyses) to characterize and compare cell distances and cell neighborhoods in spatially resolved omics data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.23.678129v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrative Spatial Modelling of Cellular Plasticity using Graph Neural Networks and Geostatistics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Withnell, E., Celik, C., Secrier, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We formalize state predictability as a quantitative proxy for plasticity, where stable states are predictable and plastic states are not, and show that stromal remodeling can propagate plasticity across tissue regions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cellular plasticity - the ability of cells to change phenotype in response to intrinsic and environmental cues - is central to development, regeneration, and disease, but remains difficult to quantify due to its dynamic, context-dependent nature. Here we introduce a framework that unites AI and geostatistics - graph neural networks and spatial regression models - to both predict and explain cell state variation in spatial transcriptomics data. We formalize state predictability as a quantitative proxy for plasticity, where stable states are predictable and plastic states are not. Applied to epithelial-mesenchymal plasticity (EMP) in breast cancer, the framework shows that mesenchymal states are stabilized by recurrent copy number alterations and microenvironmental cues, whereas hybrid states remain unpredictable and plastic. It also uncovers a long-range influence of myofibroblasts on EMP, demonstrating that stromal remodeling can propagate plasticity across tissue regions. Our framework yields interpretable, scale-aware insights into intrinsic and extrinsic drivers of state transitions, providing a broadly applicable strategy for modelling dynamic cell states in spatial biology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.24.678189v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SlATG8f enhances tomato thermotolerance and fruit quality via autophagy and HS pathways</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cheng, q., Xu, w., wen, c., He, Z., Song, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that overexpression of SlATG8f upregulates the expression of autophagy-related and heat shock protein genes, promotes early fruit ripening, and improves fruit quality under high-temperature stress.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Plant responses to high-temperature stress involve dynamic and complex changes in physiology, biochemistry, and gene expression. Autophagy, a highly conserved intracellular degradation system, facilitates the removal and recycling of damaged cytoplasmic components such as proteins and organelles. It plays an essential role in plant growth, development, and adaptation to various stresses. Although the functions of many autophagy-related genes under high-temperature stress have been characterized, the role of SlATG8f, a key member of the SlATG8 family in tomato, remains poorly understood. In this study, we generated SlATG8f-Overexpressing tomato lines using the recombinant vector pBWA(V)HS. Using quantitative reverse transcription polymerase chain reaction (qRT-PCR), we analyzed physiological indices and expression levels of ATG8 family members and heat shock protein-related genes in fruits of wild-type (WT) and SlATG8f-overexpressing plants across four developmental stages under high-temperature stress. Our results indicate that overexpression of SlATG8f upregulates the expression of autophagy-related and heat shock protein genes, promotes early fruit ripening, and improves fruit quality under high-temperature stress. These findings reveal a regulatory role for SlATG8f in maintaining tomato fruit quality under heat stress and provide a theoretical foundation for tomato variety improvement.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.23.678159v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Image-based phenotypic sorting of synthetic cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> van den Brink, M., Stam, M., Claassens, N. J., Danelon, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that liposomes can be selected for fluorescence intensity, protein localization, membrane morphology, and dynamic behaviors, and their phenotype can be linked to genetic content.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding the relationships between genotype and phenotype is key to many areas of biological research and to the development of synthetic cells. We describe an image-based screening and sorting workflow that explores the phenotypes of gene-expressing vesicles within nonclonal populations and selects the desired variants. Using automated confocal microscopy and real-time, neural-network-assisted image analysis, we demonstrate that liposomes can be selected for fluorescence intensity, protein localization, membrane morphology, and dynamic behaviors, and their phenotype can be linked to genetic content. This approach could substantially accelerate the evolution of cellular functions in a minimal synthetic context.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.25.678515v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Development and characterisation of pNarsenic: a naringenin-inducible biosensor for arsenic in Escherichia coli</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Crabbe, V., Unal, E., De Graeve, S., Guerra Giraldez, D., Peeters, T., de Buyl, S., Peeters, E., Bervoets, I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present an inducible arsenic biosensor in which the transcription factor gene arsR is inducable by naringenin, a plant-derived secondary metabolite, and raise the potential practical limit to dose-response curve engineering by tuning transcription factor expression alone.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Whole-cell biosensors detecting the heavy metal arsenic have been widely studied for their potential in environmental monitoring. And while inducible biosensors have been shown to be an effective tool to tune the operational range, a thoroughly characterised inducible biosensor is currently lacking. Here, we present an Escherichia coli biosensor for arsenic in which the transcription factor gene arsR is inducible by naringenin, a plant-derived secondary metabolite. Increasing naringenin concentrations reduced the basal output while increasing both the dynamic range and sensing threshold of the biosensor dose-response curves, but the operational ranges appeared constrained by a fixed upper limit. Comparison with a previously published phenomenological model revealed good overall agreement between experimental data and model predictions, except for the behaviour of the maximum output and threshold. This work expands the biosensor toolbox with a profoundly characterised arsenic biosensor and raises a potential practical limit to dose-response curve engineering by tuning transcription factor expression alone.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.25.678503v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Pocket-based molecule generation with an SE(3)-equivariant language model leads to a potent and selective HPK1 inhibitor with in vivo efficacy</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xi, B., Wang, H., Sun, G., Zhang, B., Mao, R., Ge, Y., Wang, Y., Zhang, J., Pan, Y., Zhou, F., Wang, Y., Liu, Z., Jiang, D., Wang, H., Zhou, W., Huang, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce SE3-BiLingoMol, an SE(3)-equivariant Transformer for pocket-based 3D molecule generation, addressing two key limitations of existing language-model approaches.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Deep learning shows promise in structure-based drug discovery, yet challenges persist in generating pharmacologically plausible molecules with valid 3D conformation and decent binding mode in the pocket. We introduce SE3-BiLingoMol, an SE(3)-equivariant Transformer for pocket-based 3D molecule generation, addressing two key limitations of existing language-model approaches. First, it uses Geometric Algebra Transformers for SE(3)-equivariant handling of continuous 3D coordinates. Second, a bidirectional attention mechanism mitigates conformational errors accumulated during autoregressive sampling. These innovations enable SE3-BiLingoMol to generate 2D drug-like, 3D geometrically valid molecules with superior binding modes. Validated on DUD-E dataset containing over 100 targets, the model achieved state-of-the-art performance in de novo design and optimization. We applied SE3-BiLingoMol to design potent and selective inhibitors for HPK1, a promising immunotherapy target. Through an iterative human-AI workflow, integrating AI generation with experimental validations (X-ray crystallography, bioassays), we identified Cmpd. 6. This novel tetracyclic compound demonstrates potent HPK1 inhibition, excellent cellular activity, favorable pharmacokinetics, and robust anti-tumor in vivo efficacy as monotherapy and with PD-1 blockade. Our work establishes a sophisticated generative AI framework for 3D molecule design and demonstrates its application in developing cancer immunotherapy.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.23.678079v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Exploring Augmentation-Driven Invariances for Graph Self-supervised Learning in Spatial Omics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rabuzin, L., Tarnow, M., Boeva, V.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A suite of graph augmentations for spatial omics, including transformations tailored to biological variation, including unsupervised domain identification in healthy tissue and supervised phenotype prediction in cancer tissue.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Spatial omics technologies provide rich insights into biological processes by jointly capturing molecular profiles and the spatial organization of cells. The resulting high-dimensional data can be naturally represented as graphs, where Graph Neural Networks (GNNs) offer an effective framework to model interactions in the tissue. Self-supervised pretraining methods such as Bootstrapped Graph Latents (BGRL) and GRACE leverage graph augmentations to build invariances without costly labels. Yet, the design of augmentation strategies remains underexplored, particularly in the context of spatial omics. In this work, we systematically investigate how different graph augmentations affect embedding quality and downstream performance in spatial omics. We evaluate a suite of existing and novel augmentations, including transformations tailored to biological variation, across two representative tasks: unsupervised domain identification in healthy tissue and supervised phenotype prediction in cancer tissue. Our results show that carefully chosen augmentations substantially improve performance, whereas poorly aligned or overly complex augmentations may fail to help or even degrade performance. These findings highlight the central role of augmentation design in enforcing meaningful invariances for graph contrastive pretraining in spatial omics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.23.678050v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Machine learning and multi-omic analysis reveal contrasting recombination landscape of A and C subgenomes of winter oilseed rape</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Montero-Tena, J. A., Zanini, S. F., Yildiz, G., Kox, T., Abbadi, A., Snowdon, R. J., Golicz, A. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We discovered distinct recombination landscape patterns between subgenomes, with crossovers clustering near subtelomeres in the A subgenome and more evenly spread across the C subgenite.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Meiotic recombination is essential for generating genetic diversity, driving plant evolution, and enabling crop improvement, yet its uneven distribution across genomes constrains breeding efforts. Here, we investigated the multi-omic landmarks that shape the recombination landscape in Brassica napus by integrating epigenomic, genomic and transcriptomic data with recombination maps derived from large multi-parental rapeseed populations. Predictive machine-learning accurately predicted recombination rates and hotspot location using only feature information. Recombination was generally suppressed in centromeres and other repeat-rich, methylated regions and enriched in gene-dense, transcriptionally active domains. Proxies for chromatin configuration such as DNA methylation, transposable elements or genes consistently achieved the highest predictive power with the random forest algorithm. We discovered distinct recombination landscape patterns between subgenomes, with crossovers clustering near subtelomeres in the A subgenome and more evenly spread across the C subgenome. Models trained on A-subgenome data outperformed those based on the C subgenome, although combining both subgenomes improved overall accuracy.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.23.677995v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Generation and Validation of a Lower Limb Muscle Single-Cell RNA Sequencing Data Set Identifies Pathogenic Endothelial Metabolism in Peripheral Arterial Disease</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Noral, M., Alhusban, S., Han, Q., Pressley, A. H., O'Keefe, S. H., Leanhart, S., Southerland, K. W., McClung, J. M., Annex, B. H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified cell-specific alterations in gene and metabolism pathways in endothelial and muscle cells from leg muscle between patients with chronic limb-threatening ischemia (CLTI) and non-PAD controls.</p><p style="font-size: 16px; text-align: justify;">Background: Peripheral arterial disease (PAD) results from atherosclerotic occlusion(s) in leg arteries. Chronic limb-threatening ischemia (CLTI) is the most severe form of PAD. Patients with CLTI suffer from rest pain, ulcers, or gangrene. Clinical outcomes remain poor in patients with CLTI and many investigational approaches, such as promoting angiogenesis, have failed. Understanding cell-specific vs. bulk-RNA changes within muscle offers an opportunity to better understand this disease. Objective: To assess cell-specific alterations in gene and metabolism pathways in endothelial and muscle cells from leg muscle between patients with PAD vs. non-PAD controls. Methods: We analyzed publicly available bulk RNA-seq leg muscle data from non-PAD controls, patients with intermittent claudication (IC) and CLTI. An emphasis was on glycolysis and the hexosamine biosynthetic pathway (HBP). Two single-cell RNA-seq datasets were integrated, cell types identified, and metabolism pathway scores calculated across identified cell types. Validations included quantitative mRNA in flow-sorted ECs, and immunofluorescence from clinically-phenotyped human samples. Results: Bulk RNA-seq revealed downregulation of both glycolysis and HBP in CLTI vs. IC or control tissues. Single-cell analysis uncovered cell-type specific changes. In patients with CLTI, gene expression in ECs showed increased glycolysis and decreased HBP. In myonuclei gene expression of both pathways were reduced. Flow-sorted ECs confirmed higher glycolysis and reduced HBP in CLTI vs. controls. Immunofluorescence analysis of muscle revealed significantly lower HBP?ECs (9.5% vs. 31.3%, n=5/group, p=0.004) and increased glycolytic ECs (42.0% vs. 14.4%, n=7/group, p<0.001) in CLTI vs. control. The findings were independent of capillary density. In IC patients, supervised exercise vs. optimal medical care reduced ischemic muscle PFKFB3+ ECs (2.67% vs. 14.61%, n=11 vs. 9, p<0.001). Conclusions: We generated and validated a single-cell RNA-seq data set that confirms maladaptive metabolic reprogramming in CLTI ECs. Strategies designed to alter metabolism need to account for differences between bulk vs. single-cell analyses.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.23.678171v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DeepGene: An Efficient Foundation Model for Genomics based on Pan-genome Graph Transformer. </h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang X, Yang M, Yin X, Qian Y, Sun F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #fedbf2; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, pan-genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce DeepGene, a model leveraging Pan-genome and Minigraph representations to encompass the broad diversity of genetic language, and improve length extrapolation in various genetic analysis tasks.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Decoding the language of DNA sequences is a fundamental problem in genome research. Mainstream pre-trained models like DNABERT-2 and Nucleotide Transformer have demonstrated remarkable achievements across a spectrum of DNA analysis tasks. Yet, these models still face the pivotal challenge of (1) genetic language diversity, or the capability to capture genetic variations across individuals or populations in the foundation models; (2) model efficiency, specifically how to enhance performance at scalable costs for large-scale genetic foundational models; (3) length extrapolation, or the ability to accurately interpret sequences ranging from short to long within a unified model framework. In response, we introduce DeepGene, a model leveraging Pan-genome and Minigraph representations to encompass the broad diversity of genetic language. DeepGene employs the rotary position embedding to improve the length extrapolation in various genetic analysis tasks. On the 28 tasks in Genome Understanding Evaluation, DeepGene achieves the overall best score. DeepGene outperforms other cutting-edge models for its compact model size and superior efficiency in processing sequences of varying lengths. The datasets and source code of DeepGene are available at GitHub (https://github.com/wds-seu/DeepGene).</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40996989/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Colorectal cancer-associated PCBP1 mutations disrupt protein stability in a dominant negative manner</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Blinkiewicz, P. V., Hoenes, N. C., Afzal, M. X., Liu, Y., Hill, U. J., Bjarnadottir, B., Burgin, T. E., Malaney, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that cancer-associated mutations substituting leucine residues 100 and 102 with glutamine, proline, or arginine destabilize PCBP1, leading to increased protein turnover.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mutations in RNA-binding proteins are increasingly identified in cancers through tumor sequencing and are correlated with disease progression, therapy response, and overall patient outcomes, underscoring the need to study them. Here, we focus on the RNA-binding protein Poly-C binding protein 1 (PCBP1), which binds target RNAs through K-homology (KH) domains to regulate RNA fate. PCBP1 is a tumor suppressor gene and hotspot missense mutations at leucine residues 100 and 102 are observed in colorectal cancer (CRC). PCBP1 mutations have been recurrently reported in CRC genome-wide mutation studies and are associated with poor clinical outcomes; however, their effects on PCBP1 expression and function remain largely unexplored. We show that cancer-associated mutations substituting leucine 100 and 102 with glutamine, proline, or arginine destabilize PCBP1, leading to increased protein turnover. The L100/L102 residues occur at the interface of the RNA-binding KH1 and KH2 domains, and our molecular dynamics simulations show that mutations at these residues disrupt the secondary structure of PCBP1. Additionally, these mutants display increased cytoplasmic localization. Importantly, mutant PCBP1 physically interacts with wild type PCBP1 and suppresses its expression through a dominant-negative mechanism. Together, our data demonstrate that CRC-associated PCBP1 mutations destabilize the protein and act as dominant-negative variants, revealing a novel mechanism of tumor suppressor inactivation in colorectal cancer.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.23.678070v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Optimal Cannabinoid-Terpene Combination Ratios Suppress Mutagenicity of Gastric Reflux in Normal and Metaplastic Esophageal Cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Goldman, A., Gonzalez, G., Karpova, S., Buon, L., SHAMMAS, M. A., Mashimo, H., Frank, M., Frank, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A 1:5 ratio of cannabigerol (CBG) to Phytol (Phy) to Cannabinoid (Cannabigerols) combination can mitigate bile acid induced esophageal damage, preserve mitochondrial membrane potential, and decrease ROS levels.</p><p style="font-size: 16px; text-align: justify;">Background: Esophageal adenocarcinoma (EAC) frequently arises from chronic exposure to acid and bile reflux, with secondary bile acids, such as deoxycholic acid (DCA), contributing to its pathogenesis through mechanisms involving reactive oxygen species (ROS), oxidative DNA damage, and resistance to apoptosis. The human endocannabinoid system (ECS) regulates diverse anti-inflammatory, antioxidant, and analgesic pathways implicated in disease modulation. Despite its therapeutic promise, effective pharmacological activation of the ECS remains challenging. Objectives: This study aimed to evaluate whether specific cannabinoid-terpene combinations targeting the ECS could attenuate the mutagenic and cytotoxic effects of bile acid induced stress in esophageal cell models. Additionally, we assessed the clinical significance of ECS-related protein receptors in the progression of EAC. Design: In vitro experimental models combined with clinical samples analyses. Methods: We utilized in vitro models, including human esophageal epithelial cell lines exposed to DCA and a Barrett's esophagus gastroesophageal reflux (GER) model subjected to low pH and a bile acid cocktail. Patient derived samples were analyzed to investigate the clinical association of ECS pathway markers with EAC progression. Experimental models were treated with varying ratios of phyto-cannabinoids and terpenes. Endpoints included assessment of DNA damage, mitochondrial membrane potential, and ROS production to identify optimal compound combinations. Expression of ECS-related protein receptors was evaluated in clinical samples to elucidate their role in EAC development. Results: A 1:5 ratio of cannabigerol (CBG) to Phytol (Phy) was found to significantly reduce DCA induced DNA damage, preserve mitochondrial membrane potential, and decrease ROS levels. This combination also enhanced apoptosis in damaged cells and diminished mutagenicity. Analysis of patient samples revealed that the expression of the ECS-associated receptor protein CB1 correlated with EAC progression, suggesting a broader clinical role for ECS modulation in cancer prevention. Conclusion: Modulation of the ECS through carefully selected cannabinoid terpene ratios can mitigate bile acid induced esophageal damage and may reduce carcinogenic progression. These findings support further in vivo investigations and raise the possibility of expanding cannabinoid terpene therapeutics to other conditions involving similar pathogenic processes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.23.678062v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">JAK2V617F Myeloproliferative Neoplasms Support Parallel Evolution of Independent Leukemic Clones</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Parsons, T. M., Krishnan, A., Xavier Raj, I., Young, A. L., O'Leary, D. R., Arand, J., Cox, M., Oh, S. T., Challen, G. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that MPN can arise from parallel acute myeloid leukemia (pAML) clones and show that leukemic clones can emerge independently of JAK2-mutant cells and undergo positive selection in the pro-inflammatory MPN environment, leading to parallel disease evolution.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Myeloproliferative neoplasms (MPNs) are hematological diseases predominantly driven by the JAK2V617F mutation. Progression from chronic-phase MPN to secondary acute myeloid leukemia (sAML) is a severe complication that dramatically worsens disease prognosis. While progression to sAML is classically linked to MPN clones acquiring additional mutations, the absence of JAK2V617F in some cases of post-MPN sAML cases suggests alternative mechanisms of transformation. Utilizing patient samples and in vivo modeling, we establish that leukemic clones can emerge independently of JAK2-mutant cells and undergo positive selection in the pro-inflammatory MPN environment, leading to parallel disease evolution. Genetic and pharmacological inhibition of IL-12 and TNF mitigates this competitive advantage. Our data establish a new paradigm and show that disease progression in MPN can arise from parallel acute myeloid leukemia (pAML) clones.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.23.678057v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Rare Multipotent Peg-like Epithelial Cell is a Candidate Cell-of-Origin for High-Grade Serous Ovarian Cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ritting, M. L., Yang, W., Aalam, S. M. M., Zhao, H., Feng, L., Song, J., Dumbrava, M., Ismail, W. M., Mun, D.-G., Hu, C., Roy, O., Chaludiya, K., Shi, G. X., Crasta, D. N., Schaufelberger, K., Janus, J. R., Kalthur, G., Weroha, S. J., Kaufmann, S., Sadanandam, A., Knapp, D. J., Wang, C., Pandey, A., Gaspar-Maia, A., Couch, F. J., Sherman, M. E., Bakkum-Gamez, J. N., Kannan, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We have created a comprehensive living organoid biobank of human fallopian tube tissue, which is thought to be the origin of high-grade serous ovarian cancer, and a rare, multipotent epithelial subpopulation with hybrid epithelial mesenchymal features.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> To illuminate the origins of high-grade serous ovarian cancer (HGSOC), the most lethal and common form of ovarian cancer, we have created a comprehensive living organoid biobank of human fallopian tube tissue, which is thought to be the origin of this cancer. Through optimized culture protocols and integrated multiomic profiling including single-cell RNA sequencing, chromatin accessibility (ATAC) analysis, proteomics, and secretomics we assembled the largest molecular atlas of the fallopian tube epithelium to date. This resource revealed diverse epithelial lineages and regulatory networks, including a rare, multipotent epithelial subpopulation with hybrid epithelial mesenchymal features. Spatially localized to the basal epithelium and resembling mesonephric developmental precursors, these cells exhibit transcriptomic and proteomic similarities to the mesenchyme-like subtype of HGSOC, implicating them as potential cells-of-origin. Their molecular identity is preserved in organoid models, enabling future mechanistic and translational studies. This resource, which advances fundamental understanding of epithelial hierarchy and cancer susceptibility, provides a platform to inform early detection and prevention strategies for aggressive forms of ovarian cancer.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.23.677404v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The Growth Inhibitory study of Methanol Extract and Fractions of Seeds of Chrysobalanus icaco (Chrysobalanceae)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Apitikori-Owumi, J. E., Sonibare, M. A., Nwankwo, L. U., Nweke, S. C., Akpomadaye, E. C., Firdous, S. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study on the growth inhibitory effect of Chrysobalanus icaco seed and its chromatographic fractions on Sorghum bicolor .</p><p style="font-size: 16px; text-align: justify;">Background: Chrysobalanus icaco L. (Chrysobalanaceae) is a spice commonly used by the Ijaws, Itsekiris, and Urhobos in Delta State. In ethnomedicine, it helps treat conditions such as diarrhoea, diabetes, high cholesterol, infections, and inflammation. Objective: The study evaluated the growth inhibitory effect of C. icaco seed and its chromatographic fractions. Method: The powdered seed sample of C. icaco was extracted with 70% ethanol by cold maceration, and the extract was screened for the presence of secondary metabolites. The extract was subjected to a vacuum liquid chromatographic technique, and the extract and fractions were subjected to bench-top biological assay using the growth inhibitory radicle of Sorghum bicolor at a concentration (1-30 mg/mL) for duration of 96 h. One way analysis of variance (ANOVA) was used in data analysis and was represented as Mean Standard Error of Mean (SEM). Results: Secondary metabolites present in the extract were alkaloids, tannins, flavonoids, phenolic compounds, reducing sugar, terpenoids, proteins and amino acids, and phytosterols. The results revealed that the length of seed radicles progressively increased over time. The results showed that the length of seed radicles progressively increased over time. The extract reduced the length of the seed radicle by 14.51-58.39% and 66.44-84.74% at 20 and 30 mg/mL. The bulked vlc fractions BAF, BBF, and BDF significantly reduced the length of the radicle at 5 and 10 mg/mL, and bulked vlc BCF was effective at 10 mg/mL only. Conclusion: The study showed that the extract of C. icaco and its vlc fractions possess an anti-proliferative effect with medical relevance in the development of anticancer therapy. The growth inhibitory effects of the extract of C. icaco and its vlc fraction on Sorghum bicolor could be linked to the phytochemicals present, associated with potential anticancer activity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.23.676451v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Constitutive NF-kB Activation is Amplified by VSV in Aggressive PC3 Prostate Cancer Cells that Resist Viral Oncolysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Abdelmageed, A. A., Smerczynski, J., Kandwal, M., Douglas, L., Russell, T., Morris, M., Dewhurst, S., Ferran, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We found that NF{kappa}B activation andNF{kapp}B dependent antiviral signaling contributes to resistance to VSV infection in PC3 cells, but not in the VSV susceptible LNCaP PrCa cell line.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cancer cells often have defects in antiviral pathways, making them susceptible to oncolytic viruses like vesicular stomatitis virus (VSV). However, some cancer cells resist viral infection through the constitutive expression of interferon-stimulated genes. This study examined whether NF{kappa}B activation and NF{kappa}B dependent antiviral signaling contributes to resistance to VSV infection in the PC3 cell line, derived from an aggressive metastatic prostate cancer (PrCa) tumor. We found that NF{kappa}B localized to the nucleus in VSV-infected PC3 cells, but not in the VSV susceptible LNCaP PrCa cell line. Analysis of the upstream NF{kappa}B inhibitor I{kappa}Ba revealed higher levels of both total and phosphorylated I{kappa}Ba in PC3 cells compared to LNCaP cells, indicating constitutive activation of the NF{kappa}B pathway via an I{kappa}Ba dependent mechanism. Notably, VSV infection did not alter I{kappa}Ba phosphorylation in PC3 cells, suggesting that VSV may amplify NF{kappa}B signaling through an I{kappa}Ba independent pathway. Furthermore, PC3 cells displayed elevated levels of the NF-{kappa}B p65 protein subunit compared to LNCaP cells, with its phosphorylated form significantly increased upon VSV infection. These results from phosphorylation assays confirm that multiple steps in the NF{kappa}B pathway are differentially activated in PC3 and LNCaP cells. Additionally, the expression of several NF{kappa}B-dependent cytokine and proinflammatory genes, including IL12 and IL6, were upregulated following VSV infection in PC3 cells, as compared to LNCaP cells. Blocking the NF{kappa}B pathway using a pharmaceutical inhibitor resulted in increased PC3 cell death with VSV infection. Collectively, these findings suggest that enhanced NF{kappa}B signaling may underlie the resistance of PC3 cells to VSV oncolysis, potentially offering new insights into therapeutic strategies targeting NF{kappa}B in resistant prostate cancers.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.23.678061v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Reproducible autosomal gene expression changes with loss of typical X and Y complement across tumor types</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Plaisier, S., Phavong, R., Farmwald, M., Lee, M., Allen, T., Swamy, M., Rodriguez, I., Wells, M., Phaneuf, N., Massey, S. C., Del Rosario, J., Hart, J., Mangelsdorf, A., Van Der Jagt, M., DeCasien, A. R., Buetow, K. H., Wilson, M. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We assess sex chromosome complement in three widely-used human genomics datasets from normal (non-cancerous) tissues, primary tumors, and cancer cell lines and study the effects on genome-wide gene expression.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Although there are known sex differences in cancer incidence, severity, and treatment, the sex chromosomes are typically excluded from genomic analyses because of the unique technical challenges associated with assessing their copy number, sequence variation, and expression. Here we assess sex chromosome complement in three widely-used human genomics datasets from normal (non-cancerous) tissues, primary tumors, and cancer cell lines and study the effects on genome-wide gene expression. Expected sex chromosome complements based on reported patient sex were observed in non-cancerous tissues, but about half of tumors and cancer cell lines showed loss of typical sex chromosome gene expression across tissue types with three categories: loss of chromosome Y (LOY), loss of chromosome X (LOX) and reactivation of the inactive X chromosome (XaXa). Genes consistently differentially expressed in tumors with loss of chromosome X, loss of chromosome Y, or loss of X chromosome inactivation are associated with the hallmarks of cancer and include both sex-linked and autosomal genes from nearly all chromosomes, druggable genes, and genes with molecular functions relevant to cancer signaling, such as kinase activity. Strikingly, tumors that are X0, including tumors from female patients that have lost an X chromosome and tumors from male patients that have lost a Y chromosome, cluster together by gene expression profile. Patients with tumors that have LOX or LOY had poorer survival outcomes compared to those with tumors that had maintained their sex chromosome complement. Further, LOX and LOY eliminates nearly all of the differential gene expression between tumors from different patient sexes, affecting sex chromosomal and autosomal gene expression. Going forward, considering patient sex as well as the entire genome, including assessment of the sex chromosome complement, will provide additional insights into personalized tumor etiology, progression, treatment, and patient outcome.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.24.678401v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

