<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Friday, 25 July 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Machine learning identification of key genes in cardioembolic stroke and atherosclerosis: their association with pan-cancer and immune cells.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tianxiang Zhang, Chunhui Yuan, Mo Chen, Jinjiang Liu, Wei Shao, Ning Cheng</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> This study aims to identify the hub genes and pathways involved in developing both diseases.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cardioembolic stroke (CS) and atherosclerosis (AS) are closely related diseases. Ferroptosis, a novel form of programmed cell death, may play a key role in CS and AS. However, the pathophysiological mechanisms underlying their coexistence remain unclear. This study aims to identify the hub genes and pathways involved in developing both diseases.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40708018/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Limitations of Current Machine-Learning Models in Predicting Enzymatic Functions for Uncharacterized Proteins.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Valérie de Crécy-Lagard, Raquel Dias, Nick Sexson, Iddo Friedberg, Yifeng Yuan, Manal A Swairjo</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that current machine-learning methods not only mostly fail to make novel predictions but also make basic logic errors in their predictions that human annotators avoid.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Thirty to seventy percent of proteins in any given genome have no assigned function and have been labeled as the protein "unknome". This large knowledge shortfall is one of the final frontiers of biology. Machine-Learning (ML) approaches are enticing, with early successes demonstrating the ability to propagate functional knowledge from experimentally characterized proteins. An open question is the ability of machine-learning approaches to predict enzymatic functions unseen in the training sets. By integrating literature and a combination of bioinformatic approaches, we evaluated individually Enzyme Commission number predictions for over 450 Escherichia coli unknowns made using state-of-the-art machine-learning approaches. We found that current ML methods not only mostly fail to make novel predictions but also make basic logic errors in their predictions that human annotators avoid by leveraging the available knowledge base. This underscores the need to include assessments of prediction uncertainty in model output and to test for 'hallucinations' (logic failures) as a part of model evaluation. Explainable AI (XAI) analysis can be used to identify indicators of prediction errors, potentially identifying the most relevant data to include in the next generation of computational models.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40703034/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Cattle BodyMap of Transcriptome across 52 Tissues and 3 Developmental Stages Reveals New Genetic Insights into Beef Production Traits</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cai, W., Zhang, Y., Xu, L., Wang, Y., Hu, X., Li, Q., Zhu, L., Wang, Z., Gao, H., Xu, L., Li, J., Zhang, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comprehensive analysis of gene expression, alternative splicing (AS), and RNA editing across various organs and developmental stages in cattle has not been reported.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cattle are important livestock that provide essential meat and milk resources. However, a comprehensive analysis of gene expression, alternative splicing (AS), and RNA editing across various organs and developmental stages in cattle has not been reported. This study aims to create a comprehensive transcriptomic BodyMap across various tissues and developmental stages, integrating this information into genomic predictions of beef production traits. We created a comprehensive transcriptomic BodyMap using 400 samples collected from 52 organs of newborn, young, and adult cattle, and estimated their contributions to genetic variance and genomic predictions for 23 beef production traits in 1476 beef cattle. We cataloged the expression of 25,530 annotated genes, 28,533 novel long non-coding RNAs (lncRNAs), 215,754 AS events, and 3,093,058 A-to-I RNA-editing sites. Integrating transcriptome BodyMap with 23 beef production traits, we found lncRNAs influenced traits like rib-eye area and carcass length, while RNA editing associated with chunk roll weight and daily gain. We observed trait-relevant tissues between different stages, including the differential expressed genes of cerebellum, longissimus muscle, and testis between newborn and adult stages are more relevant to beef production traits. The tissue-specific genes and development-associated genes in several tissues could improve the reliability of genomic prediction in beef production traits. We developed Cattle BodyMap Transcriptome Database (https://cattlegenomics.online/cattle_bodymap) to retrieve, analyze, and visualize gene expression, lncRNA, splicing and RNA editing data. Our results demonstrated the potential of using transcriptome data as a valuable resource for genomic selection and breeding programs in beef cattle. Additionally, our transcriptome BodyMap serves as a valuable resource for biological interpretation, functional validation, and genomic improvement in livestock.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.21.665934v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A framework to infer de novo exonic variants when parental genotypes are missing enhances association studies of autism</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Moon, H., Sloofman, L., Avila, M. N., Klei, L., Devlin, B., Buxbaum, J., Roeder, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a new Random Draw (RD) model to use de novo scoring for gene discovery and show that it is more powerful than the current inheritance model.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation : Gene-damaging mutations are highly informative for studies seeking to discover genes underlying developmental disorders. Traditionally, these de novo variants are recognized by evaluating high-quality DNA sequence from affected offspring and parents. However, when parental sequence is unavailable, methods are required to infer de novo status and use this inference for association studies. Results : We use data from autism spectrum disorder to illustrate and evaluate methods. Separating de novo from rare inherited variants is challenging because the latter are far more common. Using a classifier for unbalanced data and variants of known inheritance class, we build an inheritance model and then a de novo score for variants when parental data are missing. Next, we propose a new Random Draw (RD) model to use this score for gene discovery. Built into an existing inferential framework, RD produces a more powerful gene-based association test and controls the false discovery rate. Availability and Implementation : The implementation code and publicly available data are provided at https://github.com/HaeunM/TADA-RD.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.24.666675v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Gene model for the ortholog of Pi3K21B in Drosophila eugracilis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lieser, B. C., Larsen, C. I. S., Bonthius, R. M., Thompson, J. S., Toering Peters, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Gene model for the ortholog of Phosphatidylinositol 3-kinase 21B (Pi3K21B) in the Drosophila eugracilis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Gene model for the ortholog of Phosphatidylinositol 3-kinase 21B (Pi3K21B) in the D. eugracilis May 2021 (Stanford ASM1815383v1/DeugRefSeq2) Genome Assembly (GenBank Accession: GCF_018153835.1) of Drosophila eugracilis. This ortholog was characterized as part of a developing dataset to study the evolution of the Insulin/insulin-like growth factor signaling pathway (IIS) across the genus Drosophila using the Genomics Education Partnership gene annotation protocol for Course-based Undergraduate Research Experiences.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.21.665947v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An image-based transcriptomics atlas reveals the regional and microbiota-dependent molecular, cellular, and spatial structure of the murine gut</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xu, R. J., Cadinu, P., Nicol, P. B., Herrmann, U. S., Lee, T., Geistlinger, L., Irizarry, R. A., Moffitt, J. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We used MERFISH to profile receptor expression across 2.1 million cells in multiple regions of the murine gut under specific-pathogen-free (SPF) and germ-free conditions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The gastrointestinal environment is home to a massive diversity of diet-, host-, and microbiota-derived small molecules, collectively sensed by a remarkable variety of cells. To explore the cellular and spatial organization of sensation, we used MERFISH to profile receptor expression across 2.1 million cells in multiple regions of the murine gut under specific-pathogen-free (SPF) and germ-free (GF) conditions. This atlas revealed expected and novel cell types-including a candidate murine homolog of human BEST4+; enterocytes-demonstrated cell-type regional specialization, discovered extensive location-dependent spatial fine-tuning in mucosal cell expression, and suggested cell-type specific mediators of the effects of microbiota-derived small molecules. In addition, this atlas revealed that, aside from immune cell abundance, many aspects of the murine gut are host-intrinsic and modified only modestly in the absence of a microbiota. Collectively, this atlas provides a valuable resource for understanding the cellular and spatial organization underlying small molecule sensation in the gut.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.21.665958v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Remodeling of the immune-metabolic landscape triggered by long-term high-altitude exposure</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, J., Dong, Y., Xue, R., Huang, Y., Yang, W., Zhang, C., Zhang, Y., Wang, F., Yang, R., Wang, J., Yu, M., Xu, Y., Guo, M., Tian, Y., Jian, R., Zhang, J., Ruan, Y., Hu, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of metabolic and immune dynamic regulation in long-term high-altitude exposure (LTHAE) adaption .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Growing evidence indicates that immunological and metabolic outcomes are key mediators of long-term high-altitude exposure (LTHAE) adaption, but the underlying mechanisms remain poorly understood. This study employs plasma metabolomics and peripheral blood single-cell transcriptomic sequencing to analyze the metabolic and immune dynamic regulation in 46 young male lowlanders following a 90-day adaptation period at high altitude. Single-cell analysis shows a pattern of "innate immune activation and adaptive immune suppression" under LTHAE, characterized by facilitated maturation of neutrophils, enhanced cytotoxicity of CD56dim NK cells, and increased immune responsiveness of cDC2 and pDC, while inhibited maturation of plasmablasts and suppressed immune responsiveness of CD8 TEM and CD4+ T cells. Plasma metabolic analysis reveals significant alterations, involving enhanced steroid hormone synthesis, unsaturated fatty acid and amino acid metabolism under LTHAE, which in turn are associated with immune remodeling. Moreover, transcriptomic-metabolic integration analysis indicates the molecular mechanisms of enhanced aerobic oxidation efficiency under LTHAE. Collectively, these findings provide integrated insights into immune-metabolic landscape remodeling and suggest potential mutual regulatory relationship between immune and metabolic state following LTHAE, offering a molecular foundation for high-altitude adaptation research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.21.665899v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ProteinReasoner: A Multi-Modal Protein Language Model with Chain-of-Thought Reasoning for Efficient Protein Design</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liu, C., Chao, L., Ji, S., Wang, H., Jiang, T., Gao, Z., Guo, Y., Yang, M., Zhang, X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A multi-modal protein language model that explicitly incorporates the "evolutionary profile" as the intermediate reasoning step between structure and sequence modalities within a chain-of-thought (CoT) framework.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein language models (PLMs) have advanced the understanding and engineering of proteins by learning rich representations from large-scale sequence data. However, sequence-only models are limited in their ability to capture structural and evolutionary constraints essential for protein tasks. Although recent multi-modal PLMs integrate sequence and structure, they often fail to explicitly model the stepwise reasoning processes fundamental to protein science, particularly the evolutionary constraints and decision-making logic critical for protein design and optimization. Here, we introduce ProteinReasoner, a multi-modal protein language model that explicitly incorporates the "evolutionary profile" as the intermediate reasoning step between structure and sequence modalities within a chain-of-thought (CoT) framework. We demonstrated that ProteinReasoner achieved improved zero-shot performance in structure prediction, inverse protein folding, and fitness prediction tasks, consistently outperforming larger baselines including ESM3 and DPLM-2. Furthermore, we developed a novel In-context learning (ICL) paradigm for protein optimization that leverages ProteinReasoner's reasoning capabilities to guide sequence generation based on prior experimental feedback. ProteinReasoner outperformed the conventional active learning paradigm in protein optimization tasks, achieving higher predictive accuracy and better generalization. ProteinReasoner offers a scalable, efficient, and generalizable framework for protein modeling and optimization, providing a practical path to accelerate protein engineering workflows and enhance mechanistic understanding of protein biology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.21.665832v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AlphaGEM Enables Precise Genome-Scale Metabolic Modelling by Integrating Protein Structure Alignment with deep-learning-based Dark Metabolism Mining</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Han, W., Xiao, L., Sun, H., Xiang, G., Jia, Q., Wang, H., Ji, B., Zhang, C., Kerkhoven, E. J., Nielsen, J., Lu, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that structural alignment or protein-language-model-based prediction (i.e., PLMSearch) could identify more homologous protein relationships than sequence-blast-based alignment, contributing to the accurate profiling of metabolism from target organisms.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Constructing high-quality genome-scale metabolic models (GEMs) for less-studied species remains challenging. To address this, we developed AlphaGEM, a versatile toolbox leveraging proteome-scale structural alignment and deep-learning-based predictions for efficient genomic mining to generate GEMs ready for applications. Our findings show that the structural alignment or protein-language-model-based prediction (i.e., PLMSearch), could identify more homologous protein relationships than sequence-blast-based alignment, contributing to the accurate profiling of metabolism from target organisms. Additionally, AlphaGEM encompasses an ensemble procedure empowered by multiple deep learning toolboxes to effectively mine the dark metabolic functions encoded by nonhomologous proteins, significantly expanding species-specific metabolic networks. We demonstrate AlphaGEM's accuracy by building GEMs for eukaryotes (e.g., S. pombe, C. albicans) and prokaryotes (e.g., K. pneumoniae, B. subtilis), achieving predictions comparable to manually curated models while outperforming existing tools. AlphaGEM also successfully reconstructs GEMs for M. musculus and C. griseus, showcasing its great potential for uncovering dark metabolism in complex mammals. Lastly, we demonstrate that AlphaGEM could facilitate the automatic GEMs reconstruction for 332 distinct yeast species with high prediction fidelity. In conclusion, AlphaGEM provides unprecedented opportunities for the precise, rapid construction of GEMs across diverse domains, which sets a solid foundation for universal functional analysis of non-model organisms having genome sequences available.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.21.665674v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">BioinAI: a general bioinformatic framework for multi-level transcriptomic data analysis using multiple semi-agents</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cui, M., Wang, S., Yang, F., Wang, Y., Kong, F., Kong, N., Li, M., Qiao, X., Xu, Z., Yan, Z., Yan, Y., Zhang, J., Chen, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce BioinAI, a general bioinformatic framework designed for multi-level transcriptomic data analysis, which provides a user-friendly online analysis platform powered by multiple semi-agents (BioinAI).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Clinical and biological insights from large-scale transcriptomic data are often limited by technical variability and analytical complexity. Here, we introduce BioinAI, a general bioinformatic framework designed for multi-level transcriptomic data analysis. As part of the BioinAI framework, two algorithms named DeepAdvancer and stNiche were developed to improve data integration and analytical efficiency. DeepAdvancer leverages a class-aware adversarial autoencoder to reconstruct gene-expression profiles from 49,738 samples across 131 skin conditions. These profiles revealed a conserved inflammatory axis and a transcriptomic continuum which link diverse diseases through shared immune responses and structural programs. stNiche leverages spatial graph networks and symmetry-aware matching to identify functional cellular niches and reveals microstructural alterations across development, homeostasis, and disease. For instance, it identified a fibroblast-immune niche surrounding hair follicles in healthy skin, which disappears in pathological states. In addition, BioinAI provides a user-friendly online analysis platform powered by multiple semi-agents (www.bioinai.com), significantly facilitating the extraction of biological insights to advance scientific research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.21.665890v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">multiSMD - a Python toolset for multidirectional Steered Molecular Dynamics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Walczewska-Szewc, K., Niklas, B., Nowak, W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present multiSMD, a Python-based tool that automates the setup and analysis of multi-directional SMD simulations in NAMD and GROMACS.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding the direction-dependence of molecular interactions is critical for elucidating biological processes such as protein-protein binding, ligand dissociation, and mechanotransduction. While steered molecular dynamics (SMD) simulations enable the study of force-induced transitions, conventional single-direction approaches may overlook anisotropic responses inherent to biomolecular systems. Here, we present multiSMD, a Python-based tool that automates the setup and analysis of multi-directional SMD simulations in NAMD and GROMACS. By systematically probing forces along multiple spatial vectors, multiSMD captures direction-dependent phenomena - such as varying energy barriers or structural resilience - that remain hidden in traditional SMD. We demonstrate the utility of our approach through three distinct applications: (i) anisotropic unbinding in a protein-protein interaction, (ii) ligand dissociation pathways dependent on pulling direction, and (iii) force-induced remodeling of intrinsically disordered regions. multiSMD streamlines the exploration of mechanical anisotropy in biomolecules, offering a computational framework to guide experiments (e.g., AFM or optical tweezers) and uncover mechanistic insights inaccessible to single-axis methods.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.21.665906v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DNA Methylation Ageing Atlas Across 17 Human Tissues</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jacques, M., Seale, K., Voisin, S., Lysenko, A., Grolaux, R., Jones-Freeman, B., Lamon, S., Levinger, I., Bauer, C., Sharples, A. P., Heikkinen, A., Sillanpaa, E., Ollikainen, M., Smith, C., Broatch, J. R., Zarekookandeh, N., Gillberg, L., Blom, I., Poganik, J. R., Moqri, M., Gladyshev, V., Taper, M., Malecki, C., Lal, S., Saurat, N., Horvath, S., Teschendorff, A., EYNON, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Aging involves widespread epigenetic remodeling across tissues, yet the nature and consistency of these changes remain unclear. We examine the molecular architecture of human aging and identify testable targets for intervention, biomarkers, and translational epigenetic therapies.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Aging involves widespread epigenetic remodeling across tissues, yet the nature and consistency of these changes remain unclear. We conducted a meta-analysis of more than 15,000 human methylomes spanning 17 tissues, identifying both conserved and tissue-specific aging signatures. We examined linear changes via differentially methylated positions, variability shifts via variably methylated positions, and Shannon-entropy to capture methylation disorder. Network analysis revealed fragile co-methylation modules largely resistant to beneficial perturbation. Key disruptors, including PCDHGA1, MEST, HDAC4, and HOX genes, exacerbated aging signals across tissues. Notably, a resilient module enriched for NAD salvage metabolism supports therapeutic targeting of NAD in aging. PCDHGA1 emerged as a conserved cross-tissue driver, suggesting protocadherin-mediated adhesion plays a broader role in maintaining structural and signaling stability in multiple organ systems. Our open-access atlas provides a foundational resource for dissecting the molecular architecture of human aging and identifying testable targets for intervention, biomarkers, and translational epigenetic therapies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.21.665830v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The Ralstonia solanacearum E3 ligase effector RipV1 targets subfamily IXb receptor-like cytoplasmic kinases that negatively regulate immunity in Nicotiana benthamiana.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Choi, J., Dharmasena, T., Lee, Y., Monaghan, J., Segonzac, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that RLCK-IXb-1 could be a pathogen effector in plant immune signaling and show that over-expression of R LCK-ixb can lead to cell death.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Plants detect microbe-associated molecular patterns from pathogens via plasma membrane-localized receptors which activate multiple signaling cascades that lead to pattern-triggered immunity (PTI). Receptor-like cytoplasmic kinases (RLCKs) are essential hubs of plant immune signaling, associating with receptors and intracellular proteins through phosphorylation events. As a consequence, RLCKs have emerged as common targets of pathogen effectors. To improve our knowledge on Solanaceae responses to the bacterial wilt pathogen Ralstonia solanacearum, we conducted a yeast two-hybrid screen between tomato RLCKs and effectors conserved in R. solanacearum Korean isolates. Several members of RLCK subfamily IXb, which contain a ubiquitin-ligase plant U-box domain in addition to the kinase domain, interacted with RipV1, an effector containing a novel E3 ligase domain (NEL). In vitro assays revealed that RLCK-IXb-1 displayed ubiquitin-ligase activity but no detectable kinase activity. RipV1 could trans-ubiquitinate RLCK-IXb-1 in vitro and promote its stability in planta. Using virus-induced gene silencing of RLCK-IXb homologs in Nicotiana benthamiana, we could further show that several RLCK-IXb proteins act as negative regulators of early PTI signaling. RipV1 was previously reported to contribute to R. solanacearum virulence in potato and to elicit cell death in an E3 ligase activity-dependent manner in N. benthamiana. Here we show that RipV1-induced cell death occurred in plants impaired for effector recognition but could be suppressed by over-expression of RLCK-IXb-1, suggesting that this response is related to the virulence function of RipV1. Altogether, our work identifies possible substrates of an NEL effector and underlines the complex roles of RLCKs in plant immune signaling.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.21.665888v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ARABIDOPSIS Bsister and SEEDSTICK MADS-box transcription factors modulate maternal nutrient flow for seed development in Arabidopsis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Banfi, C., Babolin, N., Astori, C., Mizzotti, C., Vega-Leon, R., Leo, G., Grossniklaus, U., Tucker, M. R., Araniti, F., Aiese Cigliano, R., Sanseverino, W., Ezquer, I., Muino, J. M., Kaufmann, K., Di Marzo, M., Colombo, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the MIKC MADS domain transcription factors ABS/TT16 and STK play critical roles in regulating maternal nutrient flow during ovule maturation and seed development, and provide new insights into the metabolic reprogramming required for successful seed formation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Successful seed development in angiosperms depends on the coordinated transport and allocation of sugars from maternal tissues to the developing embryo and endosperm. In Arabidopsis thaliana, ovules function as carbohydrate sink organs, accumulating starch in both gametophytic and sporophytic domains prior to fertilization. This stored starch is later mobilized to support early embryogenesis. Despite extensive knowledge of starch metabolism in photosynthetic tissues, the regulatory mechanisms governing sugar transport in reproductive organs remain poorly understood. Recent studies have identified fertilization-dependent changes in nutrient flow, including callose-mediated modulation of symplastic transport at the phloem unloading site. However, the molecular players orchestrating these transitions are largely unknown. Here, we show that the MIKC MADS domain transcription factors ABS/TT16 and STK play critical roles in regulating maternal nutrient flow during ovule maturation and seed development. We dissect their functional redundancy using omics and genetic approaches, underscoring the importance of different ovule tissues in coordinating sugar transport pathways for post-fertilization development. Our findings reveal a previously underappreciated layer of genetic control over nutrient allocation in reproductive tissues and provide new insights into the metabolic reprogramming required for successful seed formation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.21.665905v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

