<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Sunday, 22 June 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">New iPSC resource with long-read whole genome sequencing characterizations for enhanced in vitro modeling</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Scheinfeldt, L. B., Pompetti, A., Calendo, G., Pozner, T., Grandizio, C., Smith, G., Hodges, K., Gharani, N., Kusic, D., Mitchell, M. W., Turan, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a new iPSC resource of apparently healthy subject biospecimens available to the research community through the NIGMS Repository.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Here we present a new iPSC resource of apparently healthy subject biospecimens available to the research community through the National Institute of General Medical Sciences Human Genetic Cell Repository (NIGMS Repository). This resource includes five iPSCs and matched parental cell lines with accompanying publicly available, HiFi whole-genome sequencing data. Structural variant (SV) and single nucleotide variant (SNV) concordance between iPSC and parental lines was generally high; however, we found a notable reduction in concordance between the iPSC reprogrammed with retroviral reprogramming and its parental line consistent with previous work showing newer Sendai approaches to be more robust in preserving genomic integrity. This iPSC resource additionally includes pharmacogenomic and human leukocyte antigen (HLA) gene annotations as well as a set of user-friendly, web-based search tools to visualize and explore SVs and SNVs. This new resource is designed to offer a highly characterized set of in vitro models for research into cell-type specific functional characterization of genetic, genomic and pharmacogenomic variation. More generally, these renewable biospecimens and genomic data search tools are available to the scientific community to support high-quality and reproducible biomedical research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.17.660113v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Context-aware geometric deep learning for RNA sequence design</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bibekar, P., Krapp, L. F., Dal Peraro, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a parameter-free deep learning approach that generates RNA sequences conditioned on both their backbone scaffolds and the surrounding molecular context, and further validate them by predicting their secondary structures in silico.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> RNA design has emerged to play a crucial role in synthetic biology and therapeutics. Although tertiary structure-based RNA design methods have been developed recently, they still overlook the broader molecular context, such as interactions with proteins, ligands, DNA, or ions, limiting the accuracy and functionality of designed sequences. To address this challenge, we present RISoTTo (RIbonucleic acid Sequence design from TerTiary structure), a parameter-free geometric deep learning approach that generates RNA sequences conditioned on both their backbone scaffolds and the surrounding molecular context. We evaluate the designed sequences based on their native sequence recovery rate and further validate them by predicting their secondary structures in silico and comparing them to the corresponding native structures. RISoTTo performs well on both metrics, demonstrating its ability to generate accurate and structurally consistent RNA sequences. Additionally, we present an in silico design study of domain 1 of the NAD+ riboswitch, where RISoTTo generated sequences are predicted to exhibit enhanced binding affinity for both the U1A protein and the NAD+ ligand.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.21.660801v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Nextflow pipeline for molecular quantitative trait loci mapping in small sample size datasets with an application in Atlantic salmon</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nguyen, D. T., Sandve, S. R., Lien, S., Gronvold, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We extend the RASQUAL method to a comprehensive multi-omics dataset from Atlantic salmon, identifying numerous significant expression and chromatin accessibility QTLs across multiple tissues.</p><p style="font-size: 16px; text-align: justify;">Background: Molecular quantitative trait loci (molQTL) mapping, particularly for gene expression and chromatin accessibility, provides crucial insights into the regulatory and functional potential of genetic variation. While significant progress has been made in humans and model organisms, aquatic genomics remains underexplored due to the large sample sizes typically required for statistical power. Results: In this work, we enhance the scalability, reproducibility, and accessibility of the well-established RASQUAL method, which efficiently detects molQTLs in small datasets, by leveraging the Nextflow workflow framework. This adaptation, named nf-RASQUAL, supports fully automated QTL mapping and incorporates a robust, comprehensive multiple-testing correction process. We apply the pipeline to a comprehensive multi-omics dataset from Atlantic salmon, identifying numerous significant expression and chromatin accessibility QTLs across multiple tissues. Our analysis reveals that a large proportion of lead variants for these loci reside in non-coding regions, with caQTL lead SNPs more likely to disrupt transcription factor motifs. Additionally, the enriched colocalization of eQTL and caQTL lead SNPs in brain, liver, and gonad tissues suggests shared regulatory mechanisms. Conclusions: These findings highlight the scalability and utility of nf-RASQUAL for advancing genetic regulation research in aquaculture, facilitating molQTL studies in less-explored species, and improving our understanding of molecular phenotypes shaped by genetic diversity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.17.660077v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Atomic resolution ensembles of intrinsically disordered and multi-domain proteins with Alphafold</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Schnapka, V., Morozova, T., Sen, S., Bonomi, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a Bayesian framework that integrates AlphaFold2 predictions with physico-chemical molecular mechanics force fields to generate accurate atomic-resolution ensembles of IDPs.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Intrinsically disordered proteins are ubiquitous in biological systems and play essential roles in a wide range of biological processes and diseases. Despite recent advances in high-resolution structural biology techniques and breakthroughs in deep learning-based protein structure prediction, accurately determining structural ensembles of IDPs at atomic resolution remains a major challenge. Here we introduce bAIes, a Bayesian framework that integrates AlphaFold2 predictions with physico-chemical molecular mechanics force fields to generate accurate atomic-resolution ensembles of IDPs. We show that bAIes produces structural ensembles that match a wide range of high- and low-resolution experimental data across diverse systems, achieving accuracy comparable to atomistic molecular dynamics simulations but at a fraction of their computational cost. Furthermore, bAIes outperforms state-of-the-art IDP models based on coarse-grained potentials as well as deep-learning approaches. Our findings pave the way for integrating structural information from modern deep-learning approaches with molecular simulations, advancing ensemble-based understanding of disordered and multi-domain proteins.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.18.660298v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Flexible docking of cyclic peptides to proteins using CABS-dock</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zalewski, M., Badaczewska-Dawid, A. E., Kmiecik, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a protocol based on the well-established CABS-dock method, enhanced with cyclic restraints and Rosetta refinement.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cyclic peptides are promising therapeutics, but their flexible docking remains challenging. We present a protocol based on the well-established CABS-dock method, enhanced with cyclic restraints and Rosetta refinement. The approach was evaluated on 38 benchmark complexes previously used in other docking method studies. While selecting the truly best model remains difficult, near-native solutions are frequently sampled. CABS-dock offers global, unbiased docking without prior binding site knowledge, making it valuable for pose generation, structural ensemble modeling, and integration into AI-driven peptide-protein docking workflows.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.18.660300v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">FunCLAN: A novel and generalisable method for categorising protein complex conformations, measuring their similarities, and solving the chain equivalence problem</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Celis Garza, D. C., Ellaway, J. I. J., Appasamy, S. D., Krissinel, E., Velankar, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present FunCLAN, a novel method for finding, categorising, and measuring conformational relationships between protein complexes, as well as for finding corresponding chain pairs between them.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We present FunCLAN, a novel method for finding, categorising, and measuring conformational relationships between protein complexes, as well as for finding corresponding chain pairs between them. Its purpose is to provide a general tool, applicable to a broad range of complexes. As such, it only utilises sequence similarity and superposition transformations to compare rigid bodies (chains), and how they transform with relation to each other. This lets us clusterise transformations, which lets us identify sets of similar and dissimilar transformations across the dataset. We use this information to classify conformations according to how the transformations relate to each other at chain and complex level. We have also included two data, and scale agnostic methods that can make informed predictions as to the number of clusters (conformers) present in the dataset. We also present a way to superpose entire complexes using the alignment and chain correspondence data for visualisation purposes. The entire process can be generalised to domains and even single chains, but this requires bespoke/specialised preprocessing of the data as well as prior knowledge of the structures to be analysed. This is out of the scope of this paper, but we outline a method to do so in the supplementary information.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.16.659920v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Balancing Speed and Precision in Protein Folding: A Comparison of AlphaFold2, ESMFold, and OmegaFold</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hyskova, A., Marsalkova, E., Simecek, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We compared the performance of three widely used protein structure prediction tools - AlphaFold2, ESMFold, and OmegaFold - using a dataset of over 1,300 newly created records from the PDB database.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We compared the performance of three widely used protein structure prediction tools - AlphaFold2, ESMFold, and OmegaFold - using a dataset of over 1,300 newly created records from the PDB database. These structures, resolved between July 2022 and July 2024, ensure unbiased evaluation, as they were unavailable during the training of these tools. Using metrics such as root mean square deviation (RMSD), template modeling score (TM-score), and predicted local distance difference test (pLDDT), we found that AlphaFold2 consistently achieves the highest accuracy but depends on high-quality sequence alignments. In contrast, ESMFold and OmegaFold provide faster predictions and excel in challenging cases, such as rapidly evolving or designed proteins with limited sequence homology. Comparing ESMFold and OmegaFold, ESMFold achieves higher confidence scores (pLDDT) and structural similarity (TM-score). OmegaFold is competitive in specific contexts, such as de novo-designed proteins or sequences with limited evolutionary information. Additionally, we demonstrate that machine learning models trained on protein language model embeddings and pLDDT confidence scores can predict potential structure prediction failures, helping to identify challenging cases early in the pipeline.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.20.660709v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Enhancer Poising Enables Pathogenic Gene Activation by Noncoding Variants</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hollingsworth, E., Chen, Z., Chen, C. X., Jacinto, S. H., Liu, T. A., Kvon, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that spatial enhancer poising can lead to ectopic gene activation in anterior limb bud cells, and propose that this poising signature explains how over 20 rare variants within the ZRS cause Shh misexpression in the same anterior limbs bud cell population.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single nucleotide variants within enhancers, non-coding DNA elements that regulate transcription, often lead to aberrant gene activation and contribute to a wide range of genetic disorders. The mechanism by which ectopic gene activation occurs through these gain-of-function enhancer mutations remains poorly understood. Using the ZRS, a benchmark disease-associated enhancer of Sonic hedgehog (Shh), as a model, we demonstrate that poised (i.e., accessible but inactive) chromatin sensitizes Shh to aberrant activation in anterior limb bud, leading to polydactyly. In the anterior limb cells of wild-type mice, Shh is inactive, but the ZRS is accessible and marked by enhancer-associated histone modifications. We demonstrate that this poising signature explains how over 20 independent rare variants within the ZRS cause Shh misexpression in the same anterior limb bud cell population, resulting in similar limb malformations, despite affecting binding sites for different activators and repressors. Disabling pioneer transcription factor binding to the ZRS suppresses its poised state in anterior cells, prevents aberrant activation of the ZRS by rare variants, and fully rescues limb malformations in variant knock-in mice. A thorough examination of other disease-associated enhancers with pathogenic gain-of-function variants revealed that they are all poised in tissues with ectopic activity. We use this poising signature to predict and validate in vivo ectopic forebrain activity of previously uncharacterized autism-associated non-coding variants. Our findings suggest that spatial enhancer poising, likely a byproduct of development, creates a susceptibility to non-coding mutations and offers a potential mechanistic explanation for the burden of disease-associated non-coding variants.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.20.660819v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Overhang Optimizer for Golden Gate Assembly (OOGGA)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mukundan, S., Madhusudhan, M. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A dynamic programming approach for golden gate cloning using complementary four-base pair overhangs .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Golden gate cloning is a popular method for achieving ordered assembly of DNA fragments using complementary four-base pair overhangs. The success of this method relies on the ability of DNA ligases to ligate the different fragments. This ability is affected by the relative efficiency and accuracy with which the ligase recognizes and joins overhanging regions. In this study, we report a dynamic programming approach, called Overhang Optimizer for Golden Gate Assembly (OOGGA), that optimizes these overhangs for their accuracy and efficiency. OOGGA provides the theoretically optimal fragments for golden gate assembly, provided a DNA sequence, a length range of the fragments and/or the number of required fragments. As expected, OOGGA outperforms NEBridge SplitSet in both fidelity and efficiency metrics in a set of 3 example DNA sequences. OOGGA also provides support of degenerate nucleotide codes, options to exclude/include motifs for predictions and weights to bias the program for efficiency and fidelity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.16.659877v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The evolution of allostery in a protein family</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Marti-Aranda, A., Lehner, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the first comprehensive allosteric maps for homologous human proteins and suggest that allostery evolves via the gain-and-loss of peripheral extensions to a conserved allosterIC core.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Allosteric interactions in proteins are key to biological regulation and the efficacy of many drugs. The extent to which allostery is conserved in evolutionarily-related proteins is unknown, with implications for predicting, engineering and therapeutically targeting allostery. Here we directly address this question by constructing seven comprehensive allosteric maps for five homologous human proteins. The comparative maps reveal a modular allosteric architecture with conserved distant-dependent allosteric decay across the protein core connecting to protein-specific allosteric extensions to surfaces. These allosteric augmentations use both structurally-conserved residues and homolog-specific domain extensions. Our data provide the first comparative multidimensional protein energy landscapes and suggest that allostery evolves via the gain-and-loss of peripheral extensions to a conserved allosteric core.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.20.660748v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Advanced illumination-imaging reveals photosynthesis-triggered pH, ATP and NAD redox signatures across plant cell compartments</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zheng, K., ElsaÃàsser, M., Niemeier, J.-O., Barreto, P., Cislaghi, A. P., Hoang, M., Feitosa-Araujo, E., Wagner, S., Giese, J., Kotnik, F., Martinez, M. d. P., Buchert, F. E., Ugalde, J. M., Armbruster, U., Hippler, M., Meyer, A. J., Kunz, H.-H., Maurino, V. G., Finkemeier, I., Schallenberg-RuÃàdinger, M., SchwarzlaÃànder, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new standard for the live-monitoring of subcellular energy physiology by combining confocal imaging of genetically encoded fluorescent protein biosensors with advanced on-stage illumination technology to investigate pH, MgATP2- and NADH/NAD+ dynamics at dark-light transitions in</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Photosynthesis provides energy and organic substrates to most life. In plants, photosynthesis dominates chloroplast physiology but represents only a fraction of the tightly interconnected metabolic network that spans the entire cell. Here, we explore how photosynthetic activity affects energy physiology within and beyond the chloroplast. We developed a new standard for the live-monitoring of subcellular energy physiology by combining confocal imaging of genetically encoded fluorescent protein biosensors with advanced on-stage illumination technology to investigate pH, MgATP2- and NADH/NAD+ dynamics at dark-light transitions in Arabidopsis mesophyll cells. Our findings reveal a stromal alkalinization signature induced by photosynthetic proton pumping, extending to the cytosol and mitochondria as an 'alkalinization wave'. Photosynthesis leads to increased MgATP2- levels in both the stroma and cytosol. Additionally, we observed reduction of the NAD pool driven by photosynthesis-derived electron export. Arabidopsis lines defective in chloroplast NADP- and mitochondrial NAD-dependent malate dehydrogenases show more reduced cytosolic NAD redox status even in darkness, highlighting the involvement of chloroplasts and mitochondria in shaping cytosolic redox metabolism via malate metabolism. Our study sets a novel methodological standard for precision live-monitoring of photosynthetic cell physiology. Applying this technology reveals signatures of photosynthetic physiology within and beyond the chloroplast with unprecedented resolution. Those signatures link photosynthetic activity and the fundamental biochemical functions of phototrophic cells.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.16.659786v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A lectin receptor-like kinase controls self-pollen recognition in Phlox (Polemoniaceae)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Burgin, G., Lewis, N., Hopkins, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify a gene causing self-pollen recognition in Phlox drummondii and show that it is a G-type lectin receptor-like kinase, which is a member of the same gene family as SRK, the gene controlling self- pollen recognition.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Self-incompatibility (SI) describes a widespread collection of genetic mechanisms in flowering plants used to specifically recognize and reject self-pollen. These mechanisms are fundamental to plant sexual reproduction and offer valuable insight into the molecular basis of cell-cell communication and self-recognition more broadly. Here, we leverage an independent evolution of SI in the lineage containing Phlox (Polemoniaceae) to characterize a novel gene causing self-pollen recognition which we name Phlox drummondii Pistil Identity Receptor Kinase (PdPIRK). Recognition of self-pollen associates with a single genomic region containing the Phlox S-locus. We generate predictions regarding how S-loci must function and evolve to identify a single candidate gene within this S-associated region. This gene, PdPIRK, is highly and specifically expressed in the pistil and has exceptionally high polymorphism maintained by negative frequency dependent selection, two hallmarks of self-pollen recognition genes. Functional validation with gene silencing confirms that PdPIRK is necessary for self-incompatibility, and we further demonstrate allele specific activity, confirming its role in self-pollen recognition per se. PdPIRK encodes a G-type lectin receptor-like kinase, which is a member of the same gene family as SRK, the gene controlling self-pollen recognition in the distantly related Brassicaceae. Our findings suggest the presence of genetic constraints or paths of least resistance governing how S-loci evolve and add to our understanding of the diverse molecular mechanisms through which organisms achieve self-recognition.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.17.660178v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Metabolomics-informed coarse-grained model enables prediction of cell-free protein expression dynamics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Poole, W., Kapasiawala, M., Roychoudhury, A., Haines, M., Freemont, P., Murray, R. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the majority of E. coli's metabolism is active in cell lysate during cell-free protein synthesis, and that the build-up of multiple metabolic waste products is fundamentally responsible for stopping cell- free protein expression as opposed to depletion of fuel.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cell-free expression systems have gained considerable attention for their potential in biomanufacturing, biosensing, and circuit prototyping. All these endeavors make use of the innate metabolic activity of cell lysates. However, our knowledge of the underlying nature of cell-free metabolism remains lacking. In this work, we use non-targeted mass spectrometry to generate time-course measurements of small molecules in E. coli cell lysates (metabolomics) during cell-free protein synthesis (CFPS) to show that the majority of E. coli's metabolism is active in cell lysate. Furthermore, these data indicate that protein synthesis is a relatively small metabolic burden on cell lysates compared to the background metabolism, and that the build-up of multiple metabolic waste products is fundamentally responsible for stopping cell-free protein expression as opposed to depletion of fuel sources. We use these insights coupled with high-throughput CFPS experiments to develop a foundational coarse-grained mechanistic model of CFPS which we show can be easily recalibrated using Bayesian parameter inference techniques to provide accurate models across hundreds of novel experimental conditions. This work provides new experimental insights into the effects of cell-free metabolism on CFPS and establishes a novel framework for leveraging existing CFPS models in new experimental contexts, opening the avenue to future cell-free applications aimed at building complex systems, from multilayered biological circuits to synthetic biological cells.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.20.660830v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Reconstitution of human cytochrome P450 activity using a Leishmania cell-free protein expression system</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Johnston, W., Thomson, R. E. S., Alfaro-Palma, J., Alexandrov, K., Speight, R. E., Gillam, E. M. J., Behrendorff, J. B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We demonstrate cell-free protein synthesis and assay of human P450s 3A4 and 2D6 using Leishmania tarentolae translational extract.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cytochrome P450 enzymes (P450s) are ubiquitous in drug metabolism and natural product biosynthesis. Studying eukaryotic P450s has been limited by their dependence on membrane association and requirement for partner reductases. Here, we demonstrate cell-free synthesis and assay of human P450s 3A4 and 2D6 using Leishmania tarentolae translational extract. These P450s were co-expressed with various NADPH-cytochrome P450 reductases (CPRs), and activity was assayed directly using unpurified reactions. P450s 3A4 and 2D6 showed distinct preferences in reductase coupling: P450 3A4 activity was greatest when coupled to the human CPR, whereas P450 2D6 performed better when co-expressed with CPRs from Arabidopsis thaliana. Inhibition assays with chloramphenicol, terbinafine, and erythromycin yielded results consistent with known P450-drug interactions. We conclude that Leishmania-based cell-free protein synthesis resolves previous challenges in eukaryotic P450 expression, allowing for rapid and convenient functional studies of unmodified eukaryotic P450 systems, offering a practical tool for drug metabolism studies and biocatalyst discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.20.660707v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multi-Level Regulation in RNA-Protein Hybrid Incoherent Feedforward Loop Circuits for Tunable Pulse Dynamics in Escherichia coli</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hong, S., Shoaib, S. S., Foo, M., Tang, X., Kim, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-21
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We develop novel RNA-protein hybrid type-1 incoherent feed-forward loop circuits in Escherichia coli that integrate transcriptional and translational regulators to achieve multi-level control of gene expression.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Regulating gene expression with precision is essential for cellular engineering and biosensing applications, where rapid, programmable, and sensitive control is desired. Current approaches to regulatory circuit design often rely on control at a single regulatory level, primarily the transcriptional level, thereby limiting the capability of fine-tuning the regulatory dynamics in response to complex stimuli. To address this challenge, we developed four novel RNA-protein hybrid type-1 incoherent feed-forward loop (I1-FFL) circuits in Escherichia coli that integrate transcriptional and translational regulators to achieve multi-level control of gene expression. These hybrid circuits leverage the modularity and rapid dynamics of RNA-based activators alongside the versatile inhibition capabilities of the protein-based repressors, to endow tunable pulse dynamics through engineered delays that act as transient repressor decoys. By repurposing synthetic RNA regulators at multiple regulatory levels together with aptamer and RNA-binding proteins, we demonstrate previously unexplored circuits with tunable dynamics. Complementary simulation results highlighted the importance of the engineered delays in achieving tunable pulse dynamics in these circuits. Integrating modeling insights with experimental validation, we demonstrated the flexibility of designing the RNA-protein hybrid I1-FFL circuits, as well as the tunability of their dynamics, highlighting their suitability for applications in environmental monitoring, metabolic engineering, and other engineered biological systems, where precise temporal control and adaptable gene regulation are desired.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.20.660716v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

