<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Sunday, 17 August 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genome-wide analysis of social behaviour in context: a meta-regression approach across social domains, reporters and developmental stages</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> de Hoyos, L., Schlag, F., Jahagirdar, S., Corfield, E. C., Allegrini, A. G., Admiraal, D. A. G., de Zeeuw, E. L., Nolte, I. M., Llonga, N., Neumann, A., Lange, K., van den Bedem, S., Du Rietz, E., Motazedi, E., Eising, E., Ng, N. Y. T., Palviainen, T., Wang, C. A., Thiering, E., Scatolin, S., Choudhary, P., Vilor-Tejedor, N., Liao, Z., Alemany, S., Pedersen, C.-E. T., Fernandez-Jimenez, N., Campbell, M. L., Girchenko, P., Barthome, E., Kumar, A., Jeanne, A., Seelemeyer, H., Ahluwalia, T. S., Black, M. H., Buitelaar, J. K., Fisher, S. E., Heiskala, A., Hernandez-Lorca, M., Jaddoe, V. W., Järve</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-17
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the first genome-wide meta-regression study of social behaviour from infancy to early adulthood, leveraging 491,246 repeat measures of low prosocial behaviour and peer/social difficulties in European-ancestry cohorts.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Social behaviour is a heritable, context-dependent trait that changes across social settings and development, influencing wellbeing and mental health. We present the first genome-wide meta-regression study of social behaviour from infancy to early adulthood, leveraging 491,246 repeat measures of low prosocial behaviour and peer/social difficulties in European-ancestry cohorts (Neff=121,777, Nind=73,321). We modelled heterogeneity in genetic effects across social domains, informants, and ages (2-29 years), capturing social context through genomic influences. Six loci were identified, including variation within CADM2 (p=2.51x10-9). The SNP-based heritability was modest (2-7%), and the genetic architecture of social behaviour multidimensional. Polygenic scores demonstrated predictability and accuracy in independent European-ancestry cohorts and, partially, in African-ancestry cohorts (Nind=16,305). Genetic correlations with later-life and mental health outcomes showed context-dependent patterns. Modelling predicted onsets of associations with social behaviour revealed distinct profiles, as observed for autism, ADHD, depression and schizophrenia, highlighting novel opportunities to genetically proxy developmental trajectories.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.16.667148v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Epigenetic markers of middle-age: non-linear DNA methylation changes with aging in humans</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shimura, A., Yamanishi, K., Ishii, T., Seki, T., Nishiguchi, T., Aoyama, B., Santiago, T., Dwaraka, V. B., Smith, R., Shinozaki, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of DNA methylation patterns with peaks or troughs at specific ages in addition to simple linear changes.</p><p style="font-size: 16px; text-align: justify;">Background: Human DNA is known to exhibit an overall tendency toward demethylation with aging. However, assuming a simple linear relationship between DNA methylation and age does not align with the phenotype of human development and the aging process. This study aimed to investigate the existence of DNA methylation patterns with peaks or troughs at specific ages in addition to simple linear changes. Methods: A large-scale dataset of genome-wide DNA methylation data from 10,420 individuals was analyzed. Hierarchical multiple regression models were applied to detect patterns of the association between age and DNA methylation: linear increase, linear decrease, U-shaped curve, and inverse U-shaped curve. Results: Among the 864,627 CpG sites analyzed, 8.4% exhibited an increase in DNA methylation with age, 23.9% showed a decrease, and 5.5% were better explained by a quadratic model (P < 5.7815*10^-8). Within the non-linear subset, inverse U-shaped CpG sites peaking in methylation during middle age were predominant. Genes exhibiting quadratic association patterns between DNA methylation and age, and those linked to diseases with common onset during middle age, were also detected. Conclusions: Non-linear age-related DNA methylation patterns, with peaks or troughs occurring at specific ages, were detected. This suggests that humans do not simply age linearly, but that programmed mechanisms or cascade-like processes may exist to promote or suppress the expression of specific genes at certain ages, contributing onset of certain diseases at specific timings.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.14.670237v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A prognostic signature for lung adenocarcinoma in people who have never smoked</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhao, W., Zhang, T., Hua, X., Hoang, P. H., Miraftab, M., Saha, M., McElderry, J. P., Sang, J., Lee, O. W., Hartman, C., Khandekar, A., Sharma, S., Colon-Matos, F. J., Anyaso-Samuel, S., Wang, D., Jones, K., Hutchinson, A., Hicks, B., Rosenbaum, J., Zhong, X., Yang, Y., Pesatori, A., Consonni, D., Christiani, D. C., Leung, K. C., Wong, M. P., Manczuk, M., Lissowska, J., _wi_tkowska, B., Mukeria, A., Shangina, O., Zaridze, D., Holcatova, I., Mates, D., Milosavljevic, S., Ognjanovic, S., Savic, M., Kontic, M., Gaborieau, V., Brennan, P., Arrieta, O., Bosse, Y., Edell, E. S., Schabath, M. B., Hof</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified three subtypes of lung adenocarcinoma in people who have never smoked and identified a 60-gene signature that strongly predicts survival even within subgroups based on tumor stage or known genomic features.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Knowledge of tumor cell dynamics can inform prognosis and treatment yet is largely lacking for lung adenocarcinoma in people who have never smoked (NS-LUAD). With RNA-seq data from 684 NS-LUAD and validation in an independent dataset, we identified three subtypes with distinct phenotypic traits and cell compositions. Additional genomic and histological data further characterized the subtypes. Steady, marked by low proliferation, high alveolar cell fraction, moderate-to-well differentiation, and fewer driver genes alterations, is linked to prolonged survival and low immune evasion. Proliferative shows high proliferation markers, TP53 mutations, and gene fusions. Chaotic, with high epithelial-to-mesenchymal transition markers, has the worst prognosis even within stage I tumors. Lacking known molecular or histological characteristics, this aggressive subtype is solely identified by transcriptomic data. A 60-gene signature recapitulates the overall classification and strongly predicts survival even within subgroups based on tumor stage or known genomic features, emphasizing its potential for improving NS-LUAD prognostication in clinical settings.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.13.670169v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PLM-ICE: A Protein Language Model-based Approach for Prediction of Ice nucleating and Antifreeze Proteins</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ullah, F., Pratyush, P., KC, D., Techtmann, S. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce PLM-ICE, a computational system designed to predict ice-binding proteins with high precision and sensitivity.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Many microbial species have developed adaptations for coping with life in extreme cold and in particular in the cryosphere. Ice-binding proteins (IBPs) play a critical role in enabling organisms to survive in extreme cold environments. IBPs can be divided into two distinct functional classes - antifreeze proteins (AFPs) and ice-nucleation proteins (INPs). These classes have been identified based on their specific modes of interaction with ice. Here, we introduce PLM-ICE, a computational system designed to predict IBPs with high precision and sensitivity. Leveraging ESM-2 embeddings, which incorporate evolutionary and functional sequence signals more effectively than conventional embeddings, our model employs a frozen ESM-2 encoder coupled to a Multi-layer Perceptron (MLP) prediction head. The application of this architecture allows for accurate determination of AFPs and INPs, surpassing existing methods (e.g., VotePLMs-AFP) in metrics such as Matthews correlation coefficient (MCC), area under the precision-recall curve (AUPR), and area under the receiver operating characteristic curve (AUROC). Our findings indicate that PLM-ICE exhibits robust performance across broad datasets encompassing bacterial genomic sequences, highlighting its potential for wide-ranging implementation. Notably, the ability of ESM-2 to capture essential sequence patterns confers PLM-ICE with advantages in both basic research and industrial settings, where prompt and reliable identification of IBPs remains a priority. Further, the model's strong performance underscores the broader promise of protein language model-based pipelines for decoding complex biological networks and driving innovations in cryopreservation, food technology, and climate studies. Together, these data demonstrate that PLM-ICE provides novel insight into IBP classification and stands poised to advance biotechnology applications focused on freezing tolerance and specialized temperature adaptations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.13.669380v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Cross-ancestry information transfer framework improves protein abundance prediction and protein-trait association identification</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhai, W., Sun, L., Fang, W., Dong, Y., Cheng, C., Liu, Y., Zhou, Y., Ji, J., Wu, L., Pan, A., Gamazon, E. R., Pan, X.-F., Zhou, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a multi-Ancestry-based best-performing model for protein abundance prediction accuracy in proteome-wide association studies (PWAS) and demonstrate its utility and reproducibility.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genetics-informed proteome-wide association studies (PWAS) provide an effective way to map the complex molecular landscape of biological mechanisms for complex diseases. PWAS relies on an ancestry-matched reference panel to model protein expression using genetic variants as features and determine the protein's impact on phenotype. However, reference panels from underrepresented populations remain relatively limited. In this study, we developed an analytic framework that borrows information from potentially multiple ancestries to boost the protein abundance prediction accuracy in an underrepresented population. We illustrate the framework's utility and reproducibility through application to PWAS in East Asians: BioBank Japan (BBJ), Korean Genome and Epidemiology Study (KoGES), and Taiwan Biobank (TWB). An ensemble of information-sharing approaches was integrated to build the Multi-Ancestry-based Best-performing Model (MABM). MABM substantially improved the prediction performance with higher performance observed in both cross-validation and an external validation dataset (Tongji-Huaxi-Shuangliu Birth Cohort). Leveraging the BBJ, we identified three times as many significant PWAS associations with MABM as with the baseline Lasso model. Notably, 47.5% of the MABM specific associations were reproduced in independent East Asian datasets with concordant effect sizes. Furthermore, MABM enhanced gene/protein prioritization for downstream functional validation by (1) confirming a greater number of well-established gene/protein-trait associations and (2) identifying previously uncharacterized trait-associated genes. The benefits of MABM were further validated in additional ancestries and demonstrated in brain tissue-based PWAS, underscoring its broad applicability. Our findings close critical gaps in multi-omics research, develop a new reference resource of genetic models of protein abundance, and facilitate trait-relevant protein discovery in underrepresented populations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.13.670235v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Deep learning-based image quantification of epithelial cell shapes and its application to polycystic kidney disease</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jahn, J., Hofherr, A., Consoli, C., Fajen, B., Goll, R., Liedtke, G. T., Boehm, A., Selbach, F., Zeisler, P. C., Weichselberger, V., Classen, A.-K., Westermann, L., Busch, T., Koettgen, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a fully automated, deep learning-based image analysis pipeline to quantify epithelial cell shape and tight junction morphology from immunofluorescence images, with broad applications in phenotypic screening, disease modeling, and morphogenesis research.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cell shape is a fundamental determinant of tissue architecture and organ function. In epithelial tissues, cytoskeletal organization and tight junctions regulate cell geometry, shaping functional tissue units. Disruption of these mechanisms may cause diseases such as autosomal dominant polycystic kidney disease (ADPKD), in which cyst formation is characterized by abnormal regulation of epithelial cell shape. The mechanisms of cystogenesis remain incompletely understood, highlighting the need for robust, high-throughput methods to quantify the morphology of epithelial cells. Here, we present a fully automated, deep learning-based image analysis pipeline to quantify epithelial cell shape and tight junction morphology from immunofluorescence images. Our approach employs a U-Net convolutional neural network for accurate segmentation of fluorescence labeled tight junctions. We introduce novel algorithms to quantify overall cell shape and tight junction morphology, as well as to estimate cytoskeletal traction at shared cell borders. Our analysis pipeline objectively identifies subtle morphogenetic changes associated with disease-related mutations, applied to a genetically modified Madin-Darby Canine Kidney cell model of ADPKD. The method enables high-throughput, standardized analysis, reduces observer bias, and facilitates comparison across experiments. We further demonstrate the pipeline's generalizability by applying it to Drosophila egg chamber epithelia. Our results establish a robust and scalable framework for analyzing cell shape and mechanical interactions in epithelial tissues, with broad applications in phenotypic screening, disease modeling, and morphogenesis research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.13.670046v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Accurate and scalable multi-disease classification from adaptive immune repertoires</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jiravejchakul, N., Sengupta, A., Li, S., Upadhyaya, D., Llamas-Covarrubias, M. A., Hauer, F., Haruna, S., Standley, D. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We reanalyzed publicly available T cell receptor (TCR) repertoire data from 1,421 donors across 15 disease groups and a large control group, encompassing approximately 81 million TCR sequences.</p><p style="font-size: 16px; text-align: justify;">Background: Machine learning models trained on paratope-similarity networks have shown superior accuracy compared with clonotype-based models in binary disease classification. However, the computational demands of paratope networks hinder their use on large datasets and multi-disease classification. Methods: We reanalyzed publicly available T cell receptor (TCR) repertoire data from 1,421 donors across 15 disease groups and a large control group, encompassing approximately 81 million TCR sequences. To address computational bottlenecks, we replaced the paratope-similarity network approach (Paratope Cluster Occupancy or PCO) with a new pipeline Fast Approximate Clustering Techniques (FACTS) which comprises of four main steps: (1) high-dimensional vector encoding of sequences; (2) efficient clustering of vectors; (3) donor-level feature construction from cluster distributions; and (4) gradient-boosted decision tree classification for multi-class disease prediction. Findings: FACTS processed 10 sequences in under 120 CPU hours. Using only TCR data, and evaluated with 3-fold cross-validation, it achieved a mean ROC AUC of 0.99 across 16 disease classes. Compared with the recently reported Mal-ID model, FACTS achieved higher donor-level classification accuracy for BCR (0.840 vs. 0.740), TCR (0.862 vs. 0.751), and combined BCR+TCR datasets (0.898 vs. 0.853). FACTS also preserved biologically meaningful signals, as shown by unsupervised t-SNE projections revealing distinct disease-associated and age-associated clusters. Interpretation: Paratope-based encoding with FACTS-derived features provides a scalable and biologically grounded approach for adaptive immune receptor (AIR) repertoire classification. The resulting classifier achieves superior multi-disease diagnostic performance while maintaining interpretability, supporting its potential for clinical and population-scale health profiling. Funding: This study was supported by the Japan Society for the Promotion of Science (JSPS) KAKENHI [JA23H034980] and by the Japan Agency for Medical Research and Development (AMED) [JP21am0101001].</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.12.669991v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ChemGenXplore: An Interactive Tool for Exploring and Analysing Chemical Genomic Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ahmad, H., Doherty, H., Benedict, S., Haycocks, J. R., Zhou, G., Moynihan, P. R., Moradigaravand, D., Banzhaf, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed ChemGenXplore, a web-based Shiny application designed to streamline the visualisation and analysis of chemical genomic screens.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: Chemical genomics is a useful high-throughput approach to systematically link phenotypes to genotypes. However, the vast datasets generated remain challenging to explore due to the lack of integrated, interactive tools for visualisation and analysis. Existing workflows often require multiple independent software tools, limiting data accessibility and collaboration. Therefore, we created a user-friendly platform that enable efficient exploration and sharing of chemical genomics data. Results: We developed ChemGenXplore, a web-based Shiny application designed to streamline the visualisation and analysis of chemical genomic screens. It offers two primary functionalities: one for exploring pre-implemented datasets and another for analysing user-uploaded datasets. ChemGenXplore enables users to visualise phenotypic profiles, assess gene-gene and condition-condition correlations, perform GO and KEGG enrichment analysis, and generate customisable, interactive heatmaps. By consolidating these features into an interactive and accessible tool, ChemGenXplore facilitates data sharing, enhances reproducibility, and promotes collaboration within the research community. Availability and Implementation: ChemGenXplore is freely accessible as a web application at https://chemgenxplore.kaust.edu.sa/ and is also available for local deployment via GitHub at https://github.com/Hudaahmadd/ChemGenXplore.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.13.670066v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">nf-core/proteinfamilies: A scalable pipeline for the generation of protein families</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Karatzas, E., Beracochea, M., Baltoumas, F. A., Aplakidou, E., Richardson, L., Fellows Yates, J. A., Lundin, D., nf-core community,, Buluc, A., Kyrpides, N., Georgakopoulos-Soares, I., Pavlopoulos, G. A., Finn, R. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a scalable, open-source, parametrizable Nextflow pipeline (nf-core/proteinfamilies) that generates protein families or assigns new proteins to existing families.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The growth of metagenomics-derived amino acid sequence data has transformed our understanding of protein function, microbial diversity and evolutionary relationships. However, the vast majority of these proteins remain functionally uncharacterized. Grouping the millions of such uncharacterised sequences with the few experimentally characterised ones allows the transfer of annotations, while the inspection of conserved residues with multiple sequence alignments can provide clues to function, even in the absence of existing functional information. To address the challenges associated with this data surge and the need to group sequences, we present a scalable, open-source, parametrizable Nextflow pipeline (nf-core/proteinfamilies) that generates protein nascent families or assigns new proteins to existing families. The computational benchmarks demonstrated that resource usage can scale approximately linearly with input size, while the biological benchmarks showed that the generated protein families closely resemble manually curated families found in widely used databases.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.12.670010v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

