<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Wednesday, 08 October 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genome-wide screening reveals producer-cell modifications that improve virus-like particle production and delivery potency</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ly, D., Jang, H., Goel, A., Singh, A., Raguram, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A genome-wide screening approach to systematically investigate genetic perturbations in producer cells influence eVLP production, and we applied this approach to identify several genes that regulate eV LP cargo expression and loading into particles during the production process.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Engineered virus-like particles (eVLPs) are promising vehicles for transient delivery of gene editing agents. While extensive particle optimization has yielded efficient eVLPs, it remains underexplored whether engineering the cells used to produce eVLPs could further improve eVLP properties. We developed a genome-wide screening approach to systematically investigate how genetic perturbations in producer cells influence eVLP production, and we applied this approach to identify several genes that regulate eVLP cargo expression and loading into particles during the production process. Leveraging these insights, we engineered producer cells that support increased eVLP cargo packaging and a 2- to 9-fold increase in eVLP delivery potency across several cargo, particle, and target-cell types in cultured cells and in mice. Our findings suggest the potential of producer-cell engineering as a useful strategy for improving the utility of eVLPs and related delivery methods.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.07.681010v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Citizen science gamers enable automated flow cytometry gating through machine learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Montante, S., Yokosawa, D., Li, L., Butyaev, A., Malek, M., Movassaghi, R., Michalchuk, Q., Chieh-Ting, H. J., Shmil, D., Rahim, A., Cossarizza, A., Esteban, J. B., Erhart, K., Finnbogason, B., Kelion, G., Leifsson, H., Rivers, J., Ecker, D., Szantner, A., Waldispühl, J., Brinkman, R. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed flowMagic, a machine learning algorithm for automated flow cytometry gating that is trained on both expert-curated data (template data) and crowdsourced annotations from citizen science gaming.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Manual flow cytometry gating requires up to one hour per sample with 32% inter-expert variability, creating critical bottlenecks in immunological research reproducibility. To address this, we developed flowMagic, a machine learning algorithm for automated gating that is trained on both expert-curated data (template data) and crowdsourced annotations from citizen science gaming. Through EVE Online, 839,199 players analyzed 52,178 bivariate plots from 37 studies, generating 31,703 quality-controlled training plots. Evaluated against 92,203 expert-validated files spanning 79 immune populations (i.e., a biologically defined cell subset within each bivariate plot), flowMagic achieved 90% accuracy for abundant populations and 65% for rare populations, outperforming existing methods. The algorithm reproduced biological patterns including neutrophil dynamics in COVID-19 patients and immune development in newborns. This gaming-based approach demonstrates that crowd-sourced pattern recognition generates robust training data for complex biomedical applications, offering transformative potential for standardizing flow cytometry analysis and accelerating immunological discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.07.679685v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Overcoming resistance to anti-PD-L1 immunotherapy: mechanisms, combination strategies, and future directions.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kartik Mandal, Ganesh Kumar Barik, Manas Kumar Santra</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comprehensive review of the molecular and molecular mechanisms of PD-L1, including factors that regulate its expression, tumor-immune interactions, and the resistance mechanisms associated with anti-PD- L1 immunotherapy.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cancer cells express high levels of programmed cell death-ligand 1 (PD-L1) to evade immune surveillance. PD-L1 interacts with PD-1 on T cells to make them non-functional. Thus, PD-L1 and PD-1 are pivotal targets in cancer immunotherapy. While anti-PD-1/PD-L1 therapies have offered renewed hope for many patients, their modest efficacy remains a critical concern. This underscores the urgent need to unravel the intricate mechanisms that govern both therapeutic responses as well as resistance to immunotherapy. This review explores the multifaceted nature of PD-L1, including factors that regulate its expression, tumor-immune interactions, and the resistance mechanisms associated with anti-PD-L1 immunotherapy. Several promising strategies have been explored to overcome these challenges, such as combination therapies, modulation of the tumor microenvironment, neoantigen targeting, and dynamic biomarker monitoring. Outcomes of these approaches, integrating advanced technologies like high-resolution imaging, machine learning, multi-omics profiling, and liquid biopsy for soluble PD-L1 detection emerge as a powerful means to refine patient stratification. Together, these innovations pave the way toward more precise and personalized immunotherapy, maximizing clinical benefits for cancer patients. Additionally, the evolving landscape of clinical trials involving anti-PD-1/PD-L1 monoclonal antibodies has been explored, emphasizing the integration of immune checkpoint therapies with chemotherapy, radiotherapy, targeted therapy, CAR-T, and metabolic immunotherapy to overcome resistance in refractory cancers. By embarking on these challenges and leveraging novel therapeutic strategies, this review intends to advance the understanding of more effective, personalized cancer immunotherapies, ultimately improving outcomes for a broader range of patients.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41057853/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Serum multiomics prediction of prognosis and adverse reactions to concurrent chemoradiotherapy in patients with esophageal cancer.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wen-Zhi Wu, He-Cheng Huang, Guang-Hui Zhu, Lian-Di Liao, Xu-Li Chen, Zhi-Mao Li, Man-Yu Chu, Shuai-Xia Yu, Dian Wang, En-Min Li, Li-Yan Xu</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Concurrent chemoradiotherapy is an important treatment for patients with locally advanced esophageal squamous cell carcinoma (ESCC).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Concurrent chemoradiotherapy (CCRT) is an important treatment for patients with locally advanced esophageal squamous cell carcinoma (ESCC). There is still a lack of reliable means to predict efficacy, prognosis and hematologic toxicity.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41057535/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Protein Language Models are Accidental Taxonomists</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hallee, L. P., Peleg, T., Rafailidis, N., Gleghorn, J. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose the accidental taxonomist hypothesis, proposing that neural networks can exploit the phylogenetic distances across labels in protein datasets rather than genuine interaction features, allowing models to "cheat" by learning phylogeny instead of genuine PPI features.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein-protein interactions (PPIs) are fundamental to nearly all biological processes, yet their experimental characterization remains costly and time-consuming. While computational methods, particularly those using protein language models (pLMs), offer higher-throughput solutions, they often report unexpectedly high performance on multi-species datasets. Here, we introduce the accidental taxonomist hypothesis, proposing that neural networks can exploit the phylogenetic distances across labels in protein datasets rather than genuine interaction features. We show that in PPI datasets with random negative sampling, protein pairs for real PPIs are almost exclusively from the same species, while negatives almost always originate from different species. We then demonstrate that pLM embeddings can be used to accurately distinguish whether two proteins share a taxonomic origin, allowing models to "cheat" by learning phylogeny instead of genuine PPI features. By employing a strategic sampling strategy that restricts negative examples to protein pairs from the same species, we reveal a marked drop in model performance, confirming our hypothesis. Compellingly, these strategically trained models still outperform single-species models, suggesting that multi-species data can improve performance if carefully curated. These findings suggest that accidental taxonomist behavior is a particularly influential confounder for PPI, and it is also broadly applicable to any supervised-learning protein dataset.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.07.681002v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Nanoscale spatial-omics via contrastive embedding of single-molecule localisation data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shirgill, S., Jensen, L. G., Nieves, D. J., Wales, M. C., Kaminer, A., Sovoye, K., Peters, R., Heilemann, M., Henriques, R., Lee, S. F., Rubin-Delanchy, P., Owen, D. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a contrastive learning framework for nanoscale spatial-omics that embeds single-molecule localisation microscopy data into a latent space capturing protein architecture directly to enabling comparative analysis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Omics approaches have revolutionised biology, and cells can now be routinely characterised on the genomic, transcriptomic and proteomic levels. However, there is an additional level of complexity; the (nanoscale) spatial organisation of molecules in the cell - information now accessible through super-resolution microscopy. We present a contrastive learning framework for nanoscale spatial-omics that embeds single-molecule localisation microscopy data into a latent space capturing protein architecture directly to enabling comparative analysis. Using simulated and experimental data, we demonstrate its ability to enable new bioanalysis capabilities including assessing changes to cellular nanoscale architecture arising from pharmacological treatments cell type, fluorophore selection or data-processing workflows. The approach supports downstream tasks such as clustering proteins by nanoscale organisation, mapping dose-response trajectories and identifying batch effects in replicate datasets, establishing contrastive learning as a scalable foundation for nanoscale spatial-omics and providing a platform for comparative phenotyping, quality control, and hypothesis generation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.07.679170v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Temporal Expression Prediction by Integrating Genome Dynamics via Spatio-temporal GNNs</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kaya, B., Sefer, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose neural network-based spatio-temporal graph approaches STEPmr and STEPmi to predict changes in mRNA and miRNA expression over time, respectively.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Temporal gene expression is being analyzed via high-throughput profiling of molecular data over time. The expression values of genes are impacted by their previous expression values as well as the expression of interacting genes over time. Hi-C provides us with a broad genome-wide perspective on the interacting dynamics of genes. In this paper, we propose neural network-based spatio-temporal graph approaches STEPmr and STEPmi to predict changes in mRNA and miRNA expression over time, respectively. Both approaches can integrate a diverse set of Hi-C datasets and features obtained from Hi-C when predicting temporal expression patterns. Our methods can predict mRNA and miRNA expression with 77% and 93% correlation and ith mean squared errors of 0.21 and 0.01, explaining 59.1% and 88% of the variance, respectively. Important characteristics of the genes with the highest performances in both datasets are that they are structural signaling genes or transcriptional regulators involved in fundamental processes such as homeostasis, development, and RNA processing. Additionally, they are not limited to a specific cell type, but rather show constant expression throughout different tissues. In contrast, the lowest-performed genes generally behave in context-dependent expression patterns, where they include condition-specific biological functions instead of vital biological activities. These findings suggest a model of gene regulation and its predictability that is impacted by interacting gene dynamics. Our code and datasets are publicly available at https://github.com/seferlab/temporalhic.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.07.678022v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An improved epigenetic age estimation with TFMethyl Clock reveals DNA methylation changes during aging in transcription factor binding sites</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Patel, T., Schwarz, R., Riege, K., Varshavsky, M., Kestler, H. A., Kaplan, T., Hoffman, S., Bommel, A. v.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify age-associated CpGs that overlap known transcription factor binding sites and propose a new epigenetic clock based on TFMethyl Clock, which is able to predict biological aging by incorporating regulatory loci into the design of epigenetic predictors.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Methylation-based epigenetic clocks are among the most accurate tools for predicting chronological age. Although DNA methylation (DNAm) at genomic CpG sites is linked to various regulatory mechanisms, the biological interpretability of epigenetic clocks remains surprisingly limited. One primary mechanism by which DNAm is thought to influence gene regulation is by modulating transcription factor binding activity. In this study, we examine established epigenetic clocks to assess the regulatory potential of their predictive CpGs during the aging process. Our analysis reveals that generally most CpG sites used by epigenetic clocks do not overlap known transcription factor binding sites (TFBSs), indicating that changes in TFBS dynamics may not account for prediction accuracy of these models. On the other hand, by identifying age-associated CpGs that overlap TFBSs, we identified transcription factors that may be involved in the aging process. Specifically, the TFBSs of ZBED1, NFE2, CEBPB, FOXP1, EGR1, SP1, PAX5, and MAZ were particularly enriched for age-associated CpGs, while RBPJ, NFIC, RELA, IKZF1, STAT3, and USF2 were significantly protected against methylation changes. By focusing on TFBS-associated CpGs, combined with additional feature selection and engineering steps, we developed an alternative, TFMethyl Clock model, outperforming several existing approaches. Target genes of model-selected, age-predictive CpGs are enriched in the interleukin-1{beta} production and long-chain fatty acid metabolism pathways. In contrast, these CpGs themselves are enriched mainly at binding sites of NR2C2 TF. Furthermore, approximately three-fourths of the target genes downstream of age-predictive CpGs exhibit significant age-related changes, suggesting that our approach captures deeper insights into possible methylation-driven biological aging processes. Our findings demonstrate that incorporating regulatory loci into the design of epigenetic predictors may provide mechanistic insights into the aging process while maintaining or even improving the predictive power.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.07.680024v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Sharing Patterns Between Proteins and Exploring Possible Drug Interactions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tejera Nevado, P., Rodriguez Gonzalez, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study on the localization of small amino acid sequences that are commonly found among proteins involved in various types of cancer and their impact on drug specificity .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Due to the nature of amino acids that make up proteins, it is possible to detect small patterns within their sequence. Some of these sequences have been identified as motifs, characterized by defined signatures that are shared across different proteins. In this study, patterns are defined as short amino acid sequences that are commonly found among proteins involved in various types of cancer. The focus of this research is on the localization of such patterns in ALDH2, which is used as a reference protein because it is known to be inhibited by disulfiram. Disulfiram, originally used to treat alcohol use disorder (AUD), has been repurposed for the treatment of lung cancer. Analyzing shared patterns can help identify common protein structures and may assist in locating potential binding cavities and interaction sites. Moreover, it can improve our understanding of how closely related or structurally similar proteins might also be affected by certain drugs. This approach could offer a new perspective on drug specificity and targeting strategies in the context of disease.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.07.680866v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Pseudomonas putida JM37 as a novel bacterial chassis for ethylene glycol upcycling</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Molpeceres-Garcia, F. J., Garcia-Miro, A., Mateos, E., Prieto, A., Sanz, D., Jimenez, J. I., Barriuso, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Pseudomonas putida JM37 is a novel bacterial chassis for EG valorization and a promising platform for circular bioproduction.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Ethylene glycol (EG), one of the main monomers of polyethylene terephthalate (PET), is an attractive target for microbial upcycling. Despite this interest, there is a limited number of described organisms that can efficiently metabolise EG. Here, we report the metabolic and biotechnological potential of Pseudomonas putida JM37 as a novel bacterial chassis for EG valorization. We show that JM37 efficiently grows on EG as the sole carbon and energy source, outperforming other Pseudomonas strains. Genome sequencing and directed mutagenesis revealed that genetic redundancies in the glyoxylate assimilation pathways underlie its robust EG metabolism. Beyond biomass generation, we demonstrated the biotechnological potential of JM37. This strain was able to accumulate medium-chain polyhydroxyalkanoates (mcl-PHAs), dominated by C10 monomers, directly from EG. Moreover, JM37 successfully expressed heterologous biosynthetic pathways, including a violacein biosynthetic operon and a PET-hydrolase which has been secreted actively into the extracellular medium. Together, our results support the use of P. putida JM37 as a versatile synthetic biology chassis for sustainable EG upcycling and as a promising platform for circular bioproduction.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.07.680640v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Evaluation and Aggregation of Active Module Identification Algorithms</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liu, J., Xu, M., Xing, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> No single algorithm outperforms the others across all biological datasets, suggesting that a comprehensive analysis requires the aggregation of outputs from multiple algorithms.</p><p style="font-size: 16px; text-align: justify;">Background: High-throughput sequencing methods have generated vast amounts of genetic data for candidate gene studies. As a part of the analysis, candidate genes are often analyzed through Gene-Gene interaction (GGI) networks. These networks can become very large, necessitating efficient methods to reduce their complexity. Active Module Identification (AMI) is a common method to analyze GGI networks by identifying enriched subnetworks representing relevant biological processes. Multiple AMI algorithms have been developed for biological datasets, and a comprehensive assessment of these algorithms and a comparative analysis of their behaviors across a variety of use-cases are crucial for their appropriate applications. Results: In this study, we use the Empirical Pipeline (EMP) to evaluate four AMI algorithms -- PAPER, DOMINO, FDRnet, and HotNet2 -- on their ability to produce context-specific enrichment. When testing the algorithms on four biological datasets, our results reveal that no single algorithm outperforms the others across all datasets. Moreover, the output modules are often dissimilar, suggesting that different algorithms capture complementary biological signals. Our results suggest that a comprehensive analysis requires the aggregation of outputs from multiple algorithms. We propose two methods to this end: a spectral clustering approach for module aggregation, and an algorithm that combines modules with similar network structures called Greedy Conductance-based Merging (GCM). Conclusions: Overall, our results advance our understanding of AMI algorithms and how they should be applied. Tools and workflows developed in this study will facilitate researchers working with AMI algorithms to enhance their analyses. Our code is freely available at https://github.com/LiuJ0/AMI-Benchmark/.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.06.680790v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Boolean Logic Coupled with Overrepresentation Analysis Reveals Activation of the Platelet-derived Growth Factor Receptor Beta Pathway in a Model of Oliguric Acute Kidney Injury; Implications for Transition to Chronic Kidney Disease</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chavre, H. M., Bissoondial, T., Narayan, M., Narayan, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We used Boolean logic coupled with overrepresentation analysis to identify the pathway activation signature associated with oliguric AKI in a published study of rat kidney-ischemia reperfusion and identify an array of targets to potentially mitigate transition to CKD.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Acute Kidney Injury (AKI) can occur secondary to insults including sepsis, ischemia and contrast dye administration. A time-sensitive increase in serum creatinine (SCr) or reduction in urine output (UO) has been used to define AKI and stage its severity. Oliguria or significantly reduced UO in AKI or oliguric AKI can have a major impact on outcomes including a transition to chronic kidney disease (CKD). We used Boolean logic coupled with overrepresentation analysis to identify the pathway activation signature associated with oliguric AKI in a published study of rat kidney-ischemia reperfusion. In the reperfused kidney, bulk transcriptomic analysis revealed 1068 differentially expressed genes (DEGs). Those DEGs that correlated with UO and SCr were submitted to gene ontology biological process overexpression analysis. The pathway activation signature associated with oliguric AKI included positive regulation of profibrotic platelet-derived growth factor receptor beta signaling (fold-enrichment >44) driven by src, hip1 and hip1r. Together these findings not only suggest that oliguric AKI may be associated with activation of a pathway leading to fibrosis and CKD but also informs an array of targets to potentially mitigate transition to CKD.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.06.680804v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Reduced CSF1R expression in myeloid cells has limited impact on chronic lymphocytic leukemia progression</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rosen, N., Rodriguez-Real, G., Kohlhas, V., Truong, T.-T., vom Stein, A. F., Koch, M., Reinartz, S., Iqbal, S., Ilyas, S., Mathur, S., Reinart, N., Nguyen, P.-H., Hallek, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Reducing CSF1r expression on circulating monocytes and macrophages has only a limited effect on disease progression in chronic lymphocytic leukemia, and this effect fades with disease progression.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Targeting the colony-stimulating factor 1 receptor (CSF1R) to remove tumor-associated macrophages is being explored as cancer therapy. This strategy may be relevant for chronic lymphocytic leukemia (CLL), which strongly depends on support from myeloid cells. However, it is unclear how CSF1R expression affects the CLL microenvironment and leukemic progression. To examine this question, we created CLL mice with Csf1r haploinsufficiency to investigate changes in myeloid cells, Csf1r expression, and leukemia progression. Reducing Csf1r expression on circulating monocytes did not change the overall numbers of monocytes or macrophages in blood and lymphoid tissues. Mice with lower Csf1r levels had less leukemia during early disease, but this effect faded with disease progression, and their overall survival was similar to controls. Furthermore, reduced Csf1r expression on macrophages did not affect the survival or migration of patient-derived CLL cells. In summary, our results show that while the CSF1R pathway is important for maintaining the myeloid cells that support CLL, simply reducing CSF1R expression has only a limited effect on disease progression. Attempts to target the CSF1R for leukemic therapy might benefit from a stronger depletion of macrophages or the combination with other agents.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.07.680920v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Unveiling the temporal impact: Exploring dynamic changes in the paediatric solid tumour immune microenvironment through time</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Raufaste-cazavieille, V., Bianco, S., Hermann, L., George Clement, J., Texeraud, E., Langlois, S., Sontag, T., Tremblay-Dauphinais, P., Richard-St-Hilaire, A., Joly Beauparlant, C., Faury, D., Vairy, S., Jabado, N., Cellot, S., Lavallee, V.-P., Hoa Tran, T., Sinnett, D., Droit, A., Santiago, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the composition of the tumour immune microenvironment (TIME) influences tumour evolution and responsiveness to immunotherapy, and identify time-dependent immune alterations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The composition of the tumour immune microenvironment (TIME) influences tumour evolution and responsiveness to immunotherapy. While longitudinal changes in TIME have been well-characterized in adult cancers, its dynamics in childhood cancers remain poorly documented, limiting our ability to predict treatment responses and tailor immunotherapeutic strategies. This study aimed to evaluate the plasticity of TIME in paediatric solid tumours, investigate its longitudinal evolution, and identify time-dependent immune alterations. Transcriptomic data from longitudinal samples of 27 paediatric patients (<21 years old) with relapsed or refractory solid tumours were analysed, encompassing 70 timepoints: 16 diagnoses and 54 successive relapses. TIME plasticity was assessed using gene expression clustering and immune cell infiltration enumeration. Patient-adjusted longitudinal analyses were performed using generalised linear mixed models (glmmSeq), adjusted for age and sex. Temporal associations of immune changes were further explored using dynamic regression models. Thirteen patients exhibited significant changes in their TIME profile, indicating high TIME plasticity. Over time, the TIME shifted toward a tolerogenic and immunosuppressive state, characterised by decreased activity in immune pathways (e.g., T cell receptor signalling) and enrichment of tolerogenic (e.g., macrophage differentiation) and oncogenic pathways (e.g., IL6-JAK-STAT3). The core enrichment of upregulated pathways contained key immunosuppressive factors: immune checkpoints (CTLA-4), tumour-associated macrophage activators (CSF1/CSF1R), T-regulatory cell activators (TGFB1), and immunosuppressive genes (IL10RA). This study provides evidence that the TIME in paediatric solid tumours is plastic and remodels towards immune depletion and tolerogenicity. This evolution may underlie treatment resistance and disease progression, underscoring the need for TIME-informed therapeutic approaches in paediatric oncology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.07.680325v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Rapid expansion of podoplanin-positive fibroblasts following radiation limits the anti-tumour CD8+ T-cell response to radiotherapy.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wilkins, A. C., Makris, S., Green, M., Hooper, S., Mikolajczak, A., Giangreco, G., Allanki, S., Nye, E., Connick, E., Mitter, R., Cooper, S., Durie, E., Pedersen, M., Burley, A., O'Fee, K., O'Leary, B., Shewring, D., Ramsden, Z., Melcher, A., Harrington, K., Somaiah, N., Acton, S. E., Sahai, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify a mechanism whereby PDPN+ fibrosis limits immune-mediated radiation cell kill and demonstrate that disruption of P DPN signalling favours tumour control.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Radiotherapy is known to cause changes in the tumour stroma which can undermine treatment efficacy. Our understanding of this process has historically centred around effects driven by Transforming Growth Factor-beta (TGF-{beta}) and alpha-smooth muscle actin (-SMA)+ fibroblasts. Here, we identified a rapid expansion of podoplanin (PDPN)+ fibroblasts following radiotherapy in breast, head and neck and melanoma tumours. This fibrosis was not dependent on TGF-{beta}, but was downstream of a radiotherapy-induced adaptive immune response. CD8+ T-cells entering the tumour after radiation were sequestered at the interface between residual tumour cells and PDPN+ fibroblasts and failed to enter the tumour core. Genetic deletion of PDPN in fibroblasts impacted their cytoskeleton and ability to organise extracellular matrix. This was associated with increased CD8+ T-cell entry and spontaneous tumour regression. Overall, we identify a mechanism whereby PDPN+ fibrosis limits immune-mediated radiation cell kill and demonstrate that disruption of PDPN signalling favours tumour control.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.07.680949v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Bovine colostrum-derived extracellular vesicles impair cancer cell proliferation through transcriptional repression.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Huesa Carballo, C. B., Puga, B., Bravo, S., Rana, I., Hermida, B., Antelo, M. T., Lopez Lopez, R., Abal, M., Barbazan, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that colostrum-derived extracellular vesicles can selectively and reversibly suppress cancer cell proliferation by reprogramming transcriptional and nuclear architecture, offering a natural, biocompatible strategy for modulating tumor growth and sensitizing cancer cells.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Milk-derived extracellular vesicles (EVs) are a promising source of molecules with therapeutic potential. Bovine colostrum is particularly enriched in EVs, which carry a unique cargo of proteins involved in immune regulation, development, and cellular signaling. However, despite promising bioactive effects in various fields, little is known about their potential as anti-cancer agents. Here, we demonstrate that colostrum-derived EVs (Col-EVs) exert a potent and specific anti-proliferative effect on gastrointestinal cancer cell lines, independent of apoptosis induction. Using a multi-modal approach combining proteomics and functional assays, we show that Col-EVs induce a reversible proliferative arrest through transcriptional repression and chromatin and nuclear remodeling. Col-EV treatment leads to widespread dysregulation of RNA processing and transcriptional machinery, including the downregulation of splicing factors and chromatin regulators essential for cell cycle progression. These molecular changes are accompanied by chromatin compaction, nuclear reorganization, and cytoskeletal remodeling. Notably, these effects do not induce cell death and are reversible upon EV removal, suggesting a modulatory mechanism rather than cytotoxicity. Furthermore, Col-EVs enhance the efficacy of DNA-targeting chemotherapies such as 5-fluorouracil, indicating their potential as adjunctive agents in cancer treatment. Overall, our findings reveal that Col-EVs can selectively and reversibly suppress cancer cell proliferation by reprogramming transcriptional and nuclear architecture, offering a natural, biocompatible strategy for modulating tumor growth and sensitizing cancer cells to conventional therapies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.07.680845v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Targeting Kinases to Reshape the Immunopeptidome landscape in Chronic Myeloid Leukemia</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Venafra, V., Wahle, M., Massacci, G., Bica, V., Chiusolo, P., Mougiakakos, D., Boettcher, M., Fischer, T., Perfetto, L., Mann, M., Sacco, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that pharmacological inhibition of key kinases, including BCR-ABL, JNK and LCK, could be leveraged to reshape the immunopeptidome of CML cells, enhancing the exposure of specific tumor-associated antigens.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Antileukemia immunity is essential for disease control and maintaining treatment-free remission in chronic myeloid leukemia (CML). Immunotherapeutic approaches hold significant promise for enhancing immune-mediated disease control. However, successful immunotherapy relies on identifying CML-associated antigens and achieving robust antigen presentation. Protein kinases are central regulators of signaling, and protein turnover, influencing which peptides are processed and presented on HLA molecules. Indeed, previous works with CDK4/6 or MEK inhibitors showed that kinases influence immune recognition of cancer cells. However, the molecular mechanisms underlying the modulation of the immunopeptidome by kinases inhibitors treatment remained unclear. Here, we investigated whether pharmacological inhibition of key kinases, including BCR-ABL, JNK and LCK, could be leveraged to reshape the immunopeptidome of CML cells, enhancing the exposure of specific tumor-associated antigens. By employing high-sensitive mass spectrometry (MS)-based immunopeptidomics, we quantified more than 20,000 HLA-I peptides. Pharmacological suppression of LCK, JNK and BCR-ABL drastically remodels the antigen landscape, impacting the exposure of about 4,000 HLA-I ligands. By integrating immunopeptidomics, phospho-immunopeptidomics, and proteomic, our study revealed three complementary molecular mechanisms of kinase-dependent antigen control: direct phosphorylation of HLA-I peptides (i), modulation of source protein stability (ii), and control of transcription factors governing antigen expression (iii). Moreover, comparative HLA ligandome analysis of benign hematological specimens and CML primary samples identified a panel of about 90 frequently presented CML-exclusive peptides, whose presentation can be significantly increased by LCK and JNK inhibition. Our study establishes a framework to rationally combine kinase inhibitors with immunotherapies to enhance antigen visibility and improve antileukemic immunity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.10.07.680859v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A ternary-code DNA methylome atlas of mouse tissues.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sol Moe Lee, David C Goldberg, Cameron Cloud, Jared B Parker, Christopher Krapp, Christian E Loo, Elliot Kim, Ivan Zhao, Chengcheng Jin, Rishi Porecha, Marisa S Bartolomei, Rahul M Kohli, Wanding Zhou</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new method for profiling DNA cytosine modifications in the ternary code .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> DNA cytosine modifications, including 5-methylcytosine (5mC) and 5-hydroxymethylcytosine (5hmC), are key epigenetic regulators with distinct functions. Dissecting the ternary code (C, 5mC, 5hmC) across tissues and cell types remains a critical priority due to the limitations of traditional profiling methods based on bisulfite conversion.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41057935/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">OncoMark: a high-throughput neural multi-task learning framework for comprehensive cancer hallmark quantification.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shreyansh Priyadarshi, Camellia Mazumder, Bhavesh Neekhra, Sayan Biswas, Debojyoti Chowdhury, Debayan Gupta, Shubhasis Haldar</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a neural multi-task learning-based framework that estimates hallmark activity using gene expression data from tumor biopsies.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Quantifying the biological processes that drive cancer progression remains a key challenge in oncology. Although the hallmarks of cancer provide a foundational framework for understanding tumor behavior, existing diagnostic tools rarely measure these hallmarks directly. Here we present a neural multi-task learning-based framework that estimates hallmark activity using gene expression data from tumor biopsies. The model was trained on transcriptomic profiles from 941 tumors spanning 14 tissue types and tested on five independent datasets. It predicts the activity of ten cancer hallmarks simultaneously and with high accuracy. Additional validation on large-scale datasets including normal and cancer samples confirmed its sensitivity and specificity. Predicted hallmark activity was associated with clinical staging, suggesting biological relevance. A web-based tool was developed to facilitate integration into research and clinical workflows. This approach enables efficient analysis of transcriptomic data to inform understanding of tumor biology and support individualized treatment strategies.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41057477/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A histomorphological atlas of resected mesothelioma discovered by self-supervised learning from 3446 whole-slide images.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Farzaneh Seyedshahi, Kai Rakovic, Nicolas Poulain, Adalberto Claudio Quiros, Ian R Powley, Cathy Richards, Hussein Uraiby, Sonja Klebe, David A Moore, Apostolos Nakas, Claire R Wilson, Marco Sereno, Leah Officer-Jones, Catherine Ficken, Ana Teodosio, Fiona Ballantyne, Daniel Murphy, Ke Yuan, John Le Quesne</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We map the histomorphological landscape of mesothelioma using self-supervised AI (Artificial Intelligence) to map the molecular underpinnings of the disease and derive highly interpretable predictions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mesothelioma is a highly lethal and poorly biologically understood disease which presents diagnostic challenges due to its morphological complexity. This study uses self-supervised AI (Artificial Intelligence) to map the histomorphological landscape of the disease. The resulting atlas consists of recurrent patterns identified from 3446 Hematoxylin and Eosin (H&E) stained images scanned from resected tumour slides. These patterns generate highly interpretable predictions, achieving state-of-the-art performance with 0.65 concordance index (c-index) for outcomes and 88% AUC in subtyping. Their clinical relevance is endorsed by comprehensive human pathological assessment. Furthermore, we characterise the molecular underpinnings of these diverse, meaningful, predictive patterns. Our approach both improves diagnosis and deepens our understanding of mesothelioma biology, highlighting the power of this self-learning method in clinical applications and scientific discovery.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41057342/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Modular, open-sourced multiplexing for democratizing spatial omics.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang N, Fang Z, Kadakia P, Guo J, Vijay D, Thapa M, Dembowitz S, Grakoui A, Coskun AF.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-10-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Python-based robotic imaging and staining for modular spatial omics (PRISMS), an open-sourced, automated multiplexing pipeline compatible with several sample types and Nikon NIS Elements Basic Research software, as well as Python- based biodevices.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Spatial omics technologies have revolutionized the field of biology by enabling the visualization of biomolecules within their native tissue context. However, the high costs associated with proprietary instrumentation, specialized reagents, and complex workflows have limited the broad application of these techniques. In this study, we introduce Python-based robotic imaging and staining for modular spatial omics (PRISMS), an open-sourced, automated multiplexing pipeline compatible with several biospecimen targets and streamlined microscopy software tools. PRISMS utilizes a liquid handling robot with thermal control to enable the rapid and automated staining of RNA and protein samples. The modular sample holders and Python control facilitate high-throughput, single-molecule fluorescence imaging on widefield and confocal microscopes. We successfully demonstrated the versatility of PRISMS by imaging tissue slides and adherent cells. We demonstrate that PRISMS can be utilized to perform super-resolved imaging, such as super-resolution radial fluctuations (SRRF). PRISMS is a powerful tool that can be used to democratize spatial omics by providing researchers with an accessible, reproducible, and cost-effective solution for multiplex imaging. Specifically, PRISMS is an open-source, automated multiplexing pipeline for spatial omics, compatible with several sample types and Nikon NIS Elements Basic Research software, as well as Python-based biodevices. It performs high-throughput, single-molecule fluorescence imaging both on widefield and confocal microscopes, and can be used to perform super-resolved imaging, such as SRRF. Overall, PRISMS is a powerful tool that can be used to democratize spatial omics by providing researchers with an accessible, reproducible, and cost-effective solution for multiplex imaging. This open-source platform will enable researchers to push the boundaries of spatial biology and make groundbreaking discoveries.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40936347/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Single-Cell Raman Spectroscopy Combined with Metabolomics Reveals Distinct Metabolic Profiles in Medulloblastoma Subtypes.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yijia Peng, Minghui Li, Yujie Liu, Ren Zhang, Baohong Liu, Kun Zhang</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We demonstrate the utility of Raman spectroscopy as a label-free, noninvasive approach for molecular characterization and subtype discrimination of MB at the single-cell level and demonstrate subtype-dependent metabolic shifts in drug response.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Tumor heterogeneity poses a major challenge to the precision treatment of medulloblastoma (MB). Rapid and accurate subtyping tools are urgently needed for informed clinical decision-making. Herein, we demonstrate the utility of Raman spectroscopy (RS) as a label-free, noninvasive approach for molecular characterization and subtype discrimination of MB at the single-cell level. Five representative MB cell lines from group 3 and sonic hedgehog (SHH) subtypes, along with microglial controls, were profiled by RS and validated using mass spectrometry-based metabolomics. RS-derived single-cell metabolic fingerprints revealed subtype-specific variations in nucleic acids, amino acids, lipids, and proteins. Group 3 cells, exemplified by metastatic D283, exhibited elevated levels of unsaturated lipids compared with most SHH cells. Notably, Daoy cells from the SHH subtype displayed unsaturation levels comparable to D283, reflecting intragroup heterogeneity and membrane remodeling in MB. Machine learning classifiers achieved high diagnostic performance, with an average area under the curve of 0.994 and a Matthews correlation coefficient of 0.935. Furthermore, cisplatin-treated D283 cells showed increased unsaturated lipid content relative to Daoy cells, revealing subtype-dependent metabolic shifts in drug response. These findings underscore the metabolic diversity of MB subtypes and the role of lipid metabolism in tumor progression and therapy. Collectively, our results exemplify the power of RS combined with metabolomics for molecule-resolved, label-free MB subtyping and therapeutic stratification.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/41004607/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MassVision: An Open-Source End-to-End Platform for AI-Driven Mass Spectrometry Imaging Analysis.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Amoon Jamzad, Jade Warren, Ayesha Syeda, Martin Kaufmann, Natasha Iaboni, Christopher J B Nicol, John F Rudan, Kevin Y M Ren, David Hurlbut, Sonal Varma, Gabor Fichtinger, Parvin Mousavi</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce MassVision, a comprehensive software platform for MSI data analysis and demonstrate its effectiveness through different experimental use cases such as exploratory data analysis, ion identification, and tissue-type classification.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mass spectrometry imaging (MSI) combines spatial and spectral data to reveal detailed molecular compositions within biological samples. Despite their immense potential, MSI workflows are hindered by the complexity and high dimensionality of the data, making their analysis computationally intensive and often requiring expertise in coding. Existing tools frequently lack the integration needed for seamless, scalable, and end-to-end workflows, forcing researchers to rely on local solutions or multiple platforms, which hinders efficiency and accessibility. We introduce MassVision, a comprehensive software platform for MSI analysis. Built on the 3D Slicer ecosystem, MassVision integrates MSI-specific functionalities while addressing general user requirements for accessibility and usability. Its intuitive interface lowers barriers for researchers with varying levels of computational expertise, while its scalability supports high-throughput studies and multislide data sets. Key functionalities include visualization, segmentation, colocalization, data set curation, data set merging, spectral and spatial preprocessing, statistical analysis, AI model training, and AI deployment on full MSI data. We detail the workflow and functionalities of MassVision and demonstrate its effectiveness through different experimental use cases such as exploratory data analysis, ion identification, and tissue-type classification on in-house and publicly available data from different MSI modalities. These use cases underscore MassVision's ability to seamlessly integrate MSI data handling steps into a single platform and highlight its potential to reveal new insights and structures when examining biological samples. By combining cutting-edge functionality with user-centric design, MassVision addresses longstanding challenges in MSI data analysis and provides a robust tool for advancing the user's ability to achieve biologically meaningful insights from MSI data. MassVision is freely available via 3D Slicer (documentation: https://SlicerMassVision.readthedocs.io/). The in-house MSI data have been made publicly available in MetaboLights with the identifier MTBLS12868.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40966530/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

