<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Tuesday, 01 July 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Accelerating Medicines Partnership in Type 2 Diabetes and Common Metabolic Diseases: Collaborating to Maximize the Value of Genetic and Genomic Data.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Costanzo MC, Akolkar B, Claussnitzer M, Florez JC, Gloyn AL, Grant SFA, Kaestner KH, Manning AK, Mohlke KL, Parker SCJ, Titchenell PM, Udler MS, Jones MA, Kamphaus TN, Fischer RA, McCarthy MI, Miller MR, Boehnke M, AMP T2D Consortium, AMP CMD Consortium, Flannick J, Burtt NP.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We review the history of NIDDK support for the Accelerating Medicines Partnership Program for Type 2 Diabetes (AMP T2D) and AcceleratingMedicines Partnership program for Common Metabolic Diseases (AMP CMD).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In the last two decades, significant progress has been made toward understanding the genetic basis of type 2 diabetes. An important supporter of this research has been the National Institute of Diabetes and Digestive and Kidney Diseases (NIDDK), most recently through the Accelerating Medicines Partnership Program for Type 2 Diabetes (AMP T2D) and Accelerating Medicines Partnership Program for Common Metabolic Diseases (AMP CMD). These public-private partnerships of the National Institutes of Health, multiple biopharmaceutical and life sciences companies, and nonprofit organizations, facilitated and managed by the Foundation for the National Institutes of Health, were designed to improve understanding of therapeutically relevant biological pathways for type 2 diabetes. On the occasion of NIDDK's 75th anniversary, we review the history of NIDDK support for these partnerships, which saw the convergence of research directions prioritized by academic consortia, the pharmaceutical industry, and government funders. Although the NIDDK was not the sole originator or funder of these efforts, its support and leadership have been pivotal to the partnerships' success and have enabled their research to be broadly accessible through the AMP Common Metabolic Diseases Knowledge Portal (CMDKP) and the AMP Common Metabolic Diseases Genome Atlas (CMDGA). Findings from AMP CMD align with NIDDK's mission to conduct research and share results with the goal of improving health and quality of life.The Accelerating Medicines Partnership Program for Type 2 Diabetes (AMP T2D) and Accelerating Medicines Partnership Program for Common Metabolic Diseases (AMP CMD) were created to accelerate the translation of genetic and genomic data into knowledge about the biology of disease. Their goal was to gain a better understanding of the mechanisms underlying types 1 and 2 diabetes and prediabetes, obesity, cardiovascular disease, kidney disease, and nonalcoholic steatohepatitis. This work identified multiple genes and pathways underlying these diseases. The findings of AMP T2D and AMP CMD have implications for drug development and improved risk prediction, diagnosis, and treatment for common metabolic diseases.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40272257/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Mitochondrial proteome landscape unveils key insights into melanoma severity and treatment strategies.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kim Y, Doma V, Çakır U, Kuras M, Betancourt LH, Pla I, Sanchez A, Sugihara Y, Appelqvist R, Oskolas H, Lee B, Guedes J, Monnerat G, Carneiro GRA, Nogueira FCS, Domont GB, Malm J, Baldetorp B, Wieslander E, Németh IB, Szász AM, Hong R, Pawłowski K, Rezeli M, Kwon HJ, Timar J, Fenyö D, Kárpáti S, Marko-Varga G, Gil J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Inhibition of mitochondrial pathways using antibiotics (doxycycline, tigecycline, and azithromycin) and OXPHOS inhibitors (VLX600, IACS-010759, and BAY 87-2243) demonstrated dose-dependent antiprolifer</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Melanoma, the deadliest form of skin cancer, exhibits resistance to conventional therapies, particularly in advanced and metastatic stages. Mitochondrial pathways, including oxidative phosphorylation and mitochondrial translation, have emerged as critical drivers of melanoma progression and therapy resistance. This study investigates the mitochondrial proteome in melanoma to uncover novel therapeutic vulnerabilities.Quantitative proteomics was performed on 151 melanoma-related samples from a prospective cohort and postmortem tissues. Differential expression analysis identified mitochondrial proteins linked to disease aggression and treatment resistance. Functional enrichment analyses and in vitro validation using mitochondrial inhibitors were conducted to evaluate therapeutic potential.Mitochondrial translation and oxidative phosphorylation (OXPHOS) were significantly upregulated in aggressive melanomas, particularly in BRAF-mutant and metastatic tumors. Inhibition of mitochondrial pathways using antibiotics (doxycycline, tigecycline, and azithromycin) and OXPHOS inhibitors (VLX600, IACS-010759, and BAY 87-2243) demonstrated dose-dependent antiproliferative effects in melanoma cell lines, sparing noncancerous melanocytes. These treatments disrupted mitochondrial function, suppressed key metabolic pathways, and induced apoptosis, highlighting the clinical relevance of targeting these pathways.This study reveals mitochondrial pathways as critical drivers of melanoma progression and resistance, providing a rationale for targeting mitochondrial translation and OXPHOS in advanced melanoma. Combining mitochondrial inhibitors with existing therapies could overcome treatment resistance and improve patient outcomes.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40545870/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The Vertebrate Breed Ontology: Toward Effective Breed Data Standardization.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mullen KR, Tammen I, Matentzoglu NA, Mather M, Balhoff JP, Esdaile E, Leroy G, Park CA, Rando HM, Saklou NT, Webb TL, Vasilevsky NA, Mungall CJ, Haendel MA, Nicholas FW, Toro S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> The Vertebrate Breed Ontology (VBO) is an open, community-driven ontology representing over 19 500 livestock and companion animal breed concepts covering 49 species.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Limited universally-adopted data standards in veterinary medicine hinder data interoperability and therefore integration and comparison; this ultimately impedes the application of existing information-based tools to support advancement in diagnostics, treatments, and precision medicine.A single, coherent, logic-based standard for documenting breed names in health, production, and research-related records will improve data use capabilities in veterinary and comparative medicine.No live animals were used.The Vertebrate Breed Ontology (VBO) was created from breed names and related information compiled from the Food and Agriculture Organization of the United Nations, breed registries, communities, and experts, using manual and computational approaches. Each breed is represented by a VBO term that includes breed information and provenance as metadata. VBO terms are classified using description logic to allow computational applications and Artificial Intelligence-readiness.VBO is an open, community-driven ontology representing over 19 500 livestock and companion animal breed concepts covering 49 species. Breeds are classified based on community and expert conventions (e.g., cattle breed) and supported by relations to the breed's genus and species indicated by National Center for Biotechnology Information (NCBI) Taxonomy terms. Relationships between VBO terms (e.g., relating breeds to their foundation stock) provide additional context to support advanced data analytics. VBO term metadata includes synonyms, breed identifiers/codes, and attributed cross-references to other databases.The adoption of VBO as a standard for breed names in databases and veterinary electronic health records enhances veterinary data interoperability and computability, supporting precision medicine.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40413720/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The dual-coding gene SLC35A4 protects against oxidative stress.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ajala I, Tiaiba I, Grondin B, Lipuma D, Jmii S, Benlyamani H, Cappadocia L, Vanderperre B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #fedbf2; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, pan-genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the dual-coding nature of the SLC35A4 gene leads to the emergence of two alternative protein isoforms during oxidative stress in an upstream ORF-dependent manner, with a direct implication for stress resistance.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Alternative proteins (AltProts) represent a newly recognized class of biologically active proteins encoded from alternative open reading frames (AltORFs) within already annotated genes. This study focuses on the SLC35A4 gene, which encodes both the reference protein SLC35A4 and the alternative protein AltSLC35A4. Using a combination of microscopy and biochemical analyses, we confirmed the presence of AltSLC35A4 in the inner mitochondrial membrane, resolving previous conflicting reports. Previous studies employing ribosome profiling have revealed that during oxidative stress induced by sodium arsenite, the reference coding sequence of SLC35A4 exhibits the largest increase in translational efficiency among all cellular mRNAs. Our results confirmed this translational upregulation, with the emergence of SLC35A4 protein isoforms during oxidative stress in an upstream ORF-dependent manner. Notably, the expression of AltSLC35A4 remained unchanged during oxidative stress. Knock out of SLC35A4 or AltSLC35A4 enhanced sensitivity to oxidative stress in a rescuable manner, indicating a direct implication for these proteins in stress resistance. In conclusion, our research provides compelling evidence for the functional significance of the dual-coding nature of SLC35A4 for resistance to oxidative stress and highlights the importance of considering AltProts in the functional study of eukaryotic genes.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40545711/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CpGeneAge: multi-omics aging clocks associated with Nf-κB signaling pathway in aging</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Varga, B., Kerepesi, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed the first multi-omics aging clocks based on combined epigenetics and transcriptomics features.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Aging clocks have emerged as the primary tools for measuring biological aging and have been developed for a wide range of single-omic measurements. Epigenetic aging clocks showed high accuracy in age prediction, however, their biological interpretation is still a challenging task. Transcriptomics aging clocks provide better interpretability but worse age prediction accuracy. To exploit the benefits of both omics techniques, the main goal of this study was to develop the first multi-omics aging clocks based on combined epigenetics and transcriptomics features. For this purpose, we utilized a dataset where reduced representation bisulfite sequencing (RRBS) and RNA-seq measurements were measured at the same time for peripheral blood samples of 182 individuals. Then we trained machine learning models (ElasticNet) using the methylation and gene expression features at the same time. While the most accurate models tended to use exclusively methylation features, we were able to develop highly accurate multi-omics aging clocks too (called CpGenAge). Both the canonical and the non-canonical Nf-{kappa}B signaling pathways, with the genes EDA, EDA2R, EDARADD, and CD70, were overrepresented among the gene expression features of the multi-omics aging clocks. The EDARADD, which is a unique hallmark of aging, was represented among both the gene expression and methylation features. By developing single-omic clocks on the same multi-omics dataset, we found that epigenetic age acceleration and transcriptomics age acceleration do not correlate with each other, further supporting the benefits of our multi-omics approach. In summary, here, we demonstrate that multi-omics aging clocks are useful tools to investigate aging and biological age at the multi-omics level.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.26.661785v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Quantitative analysis of genetic interactions in human cells from genome-wide CRISPR-Cas9 screens</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Billmann, M., Costanzo, M., Rahman, M., Chan, K., Tong, A., Ward, H. N., Hassan, A. Z., Zhang, X., Brown, K. R., Rohde, T., Shaw, A. H., Ross, C., van Leeuwen, J., Aregger, M., Lawson, K., Mair, B., Mero, P., Usaj, M., Andrews, B. J., Boone, C., Moffat, J., Myers, C. L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed the quantitative Genetic Interaction (qGI) score, a method for precise GI measurement from genome-wide CRISPR-Cas9 screens in isogenic human cell lines.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genetic interaction (GI) networks in model organisms have revealed how combinations of genome variants can impact phenotypes and underscored the value of GI maps for functional genomics. To advance efforts toward a reference human GI network, we developed the quantitative Genetic Interaction (qGI) score, a method for precise GI measurement from genome-wide CRISPR-Cas9 screens in isogenic human cell lines. We found surprising systematic variation unrelated to genetic interactions in CRISPR screen data, including both genomically linked effects and functionally coherent covariation. Leveraging ~40 control screens and half a billion differential fitness effect measurements, we developed a CRISPR screen data processing and normalization pipeline to correct these artifacts and measure accurate, quantitative GIs. Finally, we comprehensively characterized GI reproducibility by recording 4 - 5 biological replicates for ~125,000 unique gene pairs. The qGI framework enables systematic identification of human GIs and provides broadly applicable strategies for analyzing context-specific CRISPR screen data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.30.662330v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Representation Learning Methods for Single-Cell Microscopy are Confounded by Background Cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gupta, A., Moses, A., Lu, A. X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel evaluation framework that directly tests the robustness of crop-based models to background information.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Deep learning models are widely used to extract feature representations from microscopy images. While these models are used for single-cell analyses, such as studying single-cell heterogeneity, they typically operate on image crops centered on individual cells with background information present, such as other cells, and it remains unclear to what extent the conclusions of single-cell analyses may be altered by this. In this paper, we introduce a novel evaluation framework that directly tests the robustness of crop-based models to background information. We create synthetic single-cell crops where the center cell's localization is fixed and the background is swapped--e.g., with backgrounds from other protein localizations. We measure how different backgrounds affect localization classification performance using model-extracted features. Applying this framework to three leading models for single-cell microscopy for analyzing yeast protein localization, we find that all lack robustness to background cells. Localization classification accuracy drops by up to 15.8% when background cells differ in localization from the center cell compared to when the localization is the same. We further show that this lack of robustness can affect downstream biological analyses, such as the task of estimating proportions of cells for proteins with single-cell heterogeneity in localization. Ultimately, our framework provides a concrete way to evaluate single-cell model robustness to background information and highlights the importance of learning background-invariant features for reliable single-cell analysis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.26.661577v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Increased rate of de novo single nucleotide mutation in house mice born through assisted reproduction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Blanco-Berdugo, L., Garretson, A., Dumont, B. L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that a standard ART regimen (ovarian hyperstimulation, gamete isolation, IVF, embryo culture, and embryo transfer) affects genome stability, and show that the rate of de novo single nucleotide variant discovery in ART-derived mice is a general genome-wide phenomenon.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Approximately 2.6% of live births in the United States are conceived using assisted reproductive technologies (ARTs). While some ART procedures, including in vitro fertilization (IVF) and intracytoplasmic sperm injection, are known to alter the epigenetic landscape of early embryonic development, their impact on DNA sequence stability is unclear. Here, we leverage the strengths of the laboratory mouse model system to investigate whether a standard ART regimen (ovarian hyperstimulation, gamete isolation, IVF, embryo culture, and embryo transfer) affects genome stability. Age-matched cohorts of ART-derived and naturally conceived C57BL/6J inbred mice were reared in a controlled setting and whole genome sequenced to ~50x coverage. Using a rigorous pipeline for de novo single nucleotide variant (dnSNV) discovery, we observe a ~30% increase in the dnSNV rate in ART- compared to naturally-conceived mice. Analysis of the dnSNV mutation spectrum identified signature contributions related to germline DNA repair activity, affirming expectations and evidencing the quality of our dnSNV calls. We observed no enrichment of dnSNVs in specific genomic contexts, suggesting that the observed rate increase in ART-derived mice is a general genome-wide phenomenon. Similarly, we show that the developmental timing of dnSNVs is similar in ART- and natural-born cohorts. Together, our findings show that ART is moderately mutagenic in house mice and motivate future work to define the precise procedure(s) associated with this increased mutational vulnerability. While we caution that our findings cannot be immediately translated to humans, they nonetheless emphasize a pressing need for investigations on the potential mutagenicity of ART in our species.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.27.662069v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Super-silencers are crucial for development and carcinogenesis in B cells.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Huang, D., M Petrykowska, H. M., Kumar, D., Kardava, L., Susan Moir, S., Elnitski, L. L., Ovcharenko, I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Super-silencers in B-cell lymphoma are highly expressed in stem cells, and their target genes are highly expression in stem cell stem cells. The strength of the repressive H3 lysine 27 trimethylation modification signal varies drastically at individual silencers.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Abstract The strength of the repressive histone H3 lysine 27 trimethylation modification signal varies drastically at individual silencers. Focusing on cases of an unusually strong repressive signal in regions that we refer to as super-silencers, we demonstrate that the regions that become B-cell super-silencers are originally associated with gene upregulation during development, and their target genes are highly expressed in stem cells, especially during early developmental stages. About 13% of B-cell super-silencers transmute to super-enhancers in B-cell lymphoma and 22% of these conversions recur across more than half of patients. Notably, genes associated with these conversions, like BCL6 and BACH2, are downregulated more swiftly than others when subjected to JQ1, a super-enhancer-disrupting bromodomain and extra-terminal domain inhibitor utilized in cancer chemotherapy. Furthermore, super-silencers are characterized by an over-representation of B-cell-cancer-associated mutations, both somatic and germline, and B-cell-cancer translocation breakpoints. This surpasses the prevalence found in other regulatory elements, such as CTCF binding sites, underlining the crucial role of super-silencers in forming and stabilizing regulatory topologies in standard B cells. For example, over 80% of cases involving the B-cell-lymphoma translocation t(3;14)(q27;q32) fuse super-silencers in the BCL6 locus with enhancer-rich domains. Finally, we demonstrate that the repressive mechanisms of super-silencers are partially governed by the CpG content in their sequences. While CpG-rich super-silencers often prevent promoters from interacting with enhancers, CpG-depleted super-silencers typically suppress the chromatin looping of nearby enhancers. In summary, our findings accentuate the critical role super-silencers play in the normal function of B-cells, suggesting that sequence mutations and activity modifications in these elements could be primary factors in B-cell carcinogenesis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.27.662063v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Simultaneous epigenomic profiling and regulatory activity measurement using e2MPRA</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, Z., Georgakopoulos Soares, I., Bourque, G., Ahituv, N., Inoue, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We demonstrate that e2MPRA can be used to dissect the epigenetic functions of TF motifs arranged in synthetic enhancers, as well as to analyze the effect of enhancer sequence variants on epigenetic modifications.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cis-regulatory elements (CREs) have a major effect on phenotypes including disease. They are identified in a genome-wide manner by analyzing the binding of transcription factors (TFs), various co-factors and histone modifications in DNA using assays such as ChIP-seq, Cut&Tag and ATAC-seq. However, these assays are descriptive and require high-throughput technologies, such as massively parallel reporter assays (MPRAs), to test the functional activity and variant effect on these sequences. Currently, technologies that can simultaneously analyze both the regulatory function of a specific sequence and the TFs, cofactors and epigenomic modifications that determine it do not exist. Here, we developed enrichment followed by epigenomic profiling MPRA (e2MPRA), a novel technology that utilizes lentivirus-based MPRA to enrich for the integration of specific CREs into the genome followed by Cut&Tag or ATAC-seq targeted specifically for these sequences. This method allows to simultaneously analyze in a high-throughput manner regulatory activity, protein binding and epigenetic modification of thousands of candidate CREs and their variants. We demonstrate that e2MPRA can be used to dissect the epigenetic functions of TF motifs arranged in synthetic enhancers, as well as to analyze the effect of enhancer sequence variants on epigenetic modifications. In summary, this technology will increase our understanding of the regulatory code, its effect on the epigenome and how its alteration can lead to a variety of phenotypes including human disease.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.26.661849v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Molecular characterization of unique multi-domain harbouring fungal rhodopsin for establishing their novel opto-synthetic biological usages</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kumari, A., Kumar, A., Sharma, K., Pati, S. R., Mohanty, S., KATERIYA, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We report the identification of novel modular rhodopsins in fungi, which highlights their potential usages towards the unexplored opto-biotechnological applications (e.g., biomanufacturing of terpenoids, cytoskeleton regulation, DNA metabolism, light-controlled acetyl</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Organisms employ light as an external stimulus for regulating cellular functions. The light-sensitive photoreceptors detect light at varying wavelengths, activating signaling cascades and triggering a range of physiological responses. Rhodopsin is a transmembrane heptahelical protein that functions as an ion channel, or a pump, and sensory receptor, respectively. It consists of a light-sensing chromophore, a retinal that upon absorbing light, initiates a series of signaling pathways of sensory perception, growth and survival. Modular rhodopsin (Different from Rhodopsin-Cyclase Module) has been reported in lower eukaryotes, its identification, characterisation and functional significance in the Fungal Kingdom largely unknown. Here, we report the identification of novel modular rhodopsins in fungi, which highlights their potential usages towards the unexplored opto-biotechnological applications (e.g., biomanufacturing of terpenoids, cytoskeleton regulation, DNA metabolism, light-controlled acetyltransferase, etc.) simply by illumination. Furthermore, identification of novel modular rhodopsins augments the expansion of the new optogenetic tools for a wide range of relevant applications. The structural and homology analysis of these identified domains sheds light on their evolutionary lineage and relatedness with the well-characterised bacteriorhodopsin, sensory and channelrhodopsin. The interactome analysis effector domain coupled with the microbial rhodopsin (Rh) reveals RPEL-mediated gene expression and metabolite regulation, which further modulates the retinol synthesis pathway. The role of the fungal Rh-RPEL effector domain in modulating the terpenoid and sphingolipid metabolism in response to light was successfully elucidated via protein-protein interaction and Biosynthesis Gene Cluster (BGC) analysis. This highlights the potential of these novel opto-synthetic biological usages that can induce the light-dependent production of commercially relevant fungal bioactive(s).</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.26.661701v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comparative regulomics provides novel insight into the evolution of wood formation across dicot and conifer trees</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rodriguez, E., Birkeland, S., Chapple, E. D., Fredriksson, S., Lorenzo, Z. C., Kalman, T. A., Kumar, V., Hill, J., Tuominen, H., Mellerowicz, E. J., Street, N. R., Hvidsten, T. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the first high-spatial-resolution evo-devo resource spanning the wood transcriptomes of six tree species - three dicots and three conifers - capturing 250 million years of evolutionary divergence.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding the regulatory program underlying wood formation is key to improving biomass production and carbon sequestration in trees. However, how wood formation evolved and how these programs have been rewired across lineages remains unclear. Here, we present the first high-spatial-resolution evo-devo resource spanning the wood transcriptomes of six tree species - three dicots and three conifers - capturing 250 million years of evolutionary divergence. Using orthology-aware co-expression network analysis, we identified genes with conserved and lineage-specific expression patterns. By integrating chromatin accessibility data and transcription factor motif analysis, we further inferred regulatory networks for xylem differentiation and secondary cell wall formation. We demonstrate how this dataset can be used to answer long standing questions in wood biology related to differences in acetylation of cell wall polymers and master regulators of xylem specification across dicot and conifer tree species. The data offer a foundational resource for the tree biology and evo-devo communities, and are publicly available at PlantGenIE.org.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.28.661522v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genomic Touchstone: Benchmarking Genomic Language Models in the Context of the Central Dogma</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, Y., Cai, Z., Zeng, Q., Gao, Y., Ouyang, J., Xu, Y., Yang, S., He, S., Nie, Y., Cai, Y., Zhou, F., Jin, C., Wang, X., Xie, Z., Zhu, D., Xie, T., Cheng, K.-T., Yang, C., Fu, X., Wang, J., Zhang, K., Yao, J., Rabadan, R., Chen, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present Genomic Touchstone, a comprehensive benchmark designed to evaluate gLMs across 36 diverse tasks and 88 datasets structured along the central dogma's modalities of DNA, RNA, and protein, encompassing 5.34 billion base pairs of genomic sequences.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The emergence of genomic language models (gLMs) has revolutionized the analysis of genomic sequences, enabling robust capture of biologically meaningful patterns from DNA sequences for an improved understanding of human genome-wide regulatory programs, variant pathogenicity and therapeutic discovery. Given that DNA serves as the foundational blueprint within the central dogma, the ultimate evaluation of a gLM is its ability to generalize across this entire biological cascade. However, existing evaluations lack this holistic and consistent framework, leaving researchers uncertain about which model best translates sequence understanding into downstream biological prediction. Here we present Genomic Touchstone, a comprehensive benchmark designed to evaluate gLMs across 36 diverse tasks and 88 datasets structured along the central dogma's modalities of DNA, RNA, and protein, encompassing 5.34 billion base pairs of genomic sequences. We evaluate 34 representative models encompassing transformers and convolutional neural networks, as well as emerging efficient architectures such as Hyena and Mamba. Our analysis yield four key insights. First, gLMs achieve comparable or superior performance on RNA and protein tasks compared to models pretrained on these molecules. Second, transformer-based models continue to lead in overall performance, yet efficient sequence models show promising task-specific capabilities and deserve further exploration. Third, the scaling behavior of gLMs remains incompletely understood. While longer input sequences and more diverse pretraining data consistently improve performance, increases in model size do not always translate into better results. Fourth, pretraining strategies, including the choice of training objectives and the composition of pretraining corpora, exert substantial influence on downstream generalization across different genomic contexts. Genomic Touchstone establishes a unified evaluation framework tailored to human genomics. By spanning multiple molecular modalities and biological tasks, it offers a valuable foundation for guiding future gLM design and understanding model generalizability in complex biological contexts.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.25.661622v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Recessive antimorph alleles reveal novel functions of the OPAQUE1 myosin XI in maize</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zebosi, B., Martinez, S. E., Wimalanathan, K., Ssengo, J., Brown, G., Best, N. B., Facette, M., Rasmussen, C. G., Vollbrecht, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that O1-2995 and o1-tan62 are novel alleles of opaque1, and play a role in plant growth via internode elongation, subsidiary cell division positioning, leaf patterning, inflorescence development, and overall plant architecture.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Ideal plant architecture optimizes canopy structure and increases grain yield in maize. However, its underlying genetic mechanisms remain poorly characterized. Two recessive, EMS-induced maize mutants were identified that have reduced stature, opaque kernels, and abnormal subsidiary cell division, and are allelic to opaque1 (o1). These two new missense alleles, o1-2995 and o1-tan62, unlike the loss-of-function alleles previously identified, generate O1 protein but paradoxically generate more severe morphological defects. These defects include reduced internode elongation and partial suppression of excessive tassel and ear branching in ramosa1 (ra1), ra2, and ra3 mutants. We show that o1-2995 and o1-tan62 are novel alleles of o1, and play a role in plant growth via internode elongation, subsidiary cell division positioning, leaf patterning, inflorescence development, and overall plant architecture.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.26.661838v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Zinc deficiency induces spatially distinct responses in roots and impacts ZIP12-dependent zinc homeostasis in Arabidopsis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Thiebaut, N., Persson, D. P., Sarthou, M., Stevenne, P., Bosman, B., Carnol, M., Fanara, S., Verbruggen, N., Hanikenne, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Zn deficiency alters root zonation and function in Arabidopsis, with ZIP12 emerging as a central player in maintaining Zn homeostasis and root meristem function.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> How zinc (Zn) deficiency shapes root development remains unclear, with conflicting reports on its effect on primary root growth in Arabidopsis thaliana (Arabidopsis). The impact of Zn shortage on the root apical meristem (RAM) in particular has not been systematically explored. Using an integrative approach combining cell biology, transcriptomics, and ionomics, we dissected how Zn deficiency alters root zonation and function. We showed that Zn deficiency triggers a striking reorganization of the root tip (RT): the RAM size is reduced, yet meristematic activity and local Zn levels are preserved. This is accompanied by promoted cell elongation and differentiation. Transcriptome profiling revealed a distinct Zn deficiency response in the RAM-enriched RT compared to mature root tissues, with ZIP12 emerging as the most strongly induced gene in the RT. Functional analysis of zip12 mutants uncovered major defects in root growth, RAM structure, expression of Zn-responsive genes, and metal partitioning. Our work unveiled a new layer of root developmental plasticity under Zn deficiency and identified ZIP12 as a central player in maintaining Zn homeostasis and root meristem function in Arabidopsis. These findings provide a framework to better understand how plants adapt root growth to fluctuating micronutrient availability.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.26.661794v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Rhizobial infection-specific accumulation ofphosphatidylinositol 4,5-bisphosphate inhibits the excessive infection of rhizobia in Lotus japonicus</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Akamatsu, A., Ishikawa, T., Tanaka, H., Kawano, Y., Hayashi, M., Takeda, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We performed LC-MS and RNA-seq analysis using rhizobia-infected Lotus japonicus roots and investigated the role of phosphatidylinositol (PI) and phosphate phosphates (PIPs) in the symbiosis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> {middle dot}During the symbiosis of legume with nitrogen-fixing bacteria, collectively called rhizobia, suppression of excessive rhizobial infection by host plants is important to maximize the benefits of symbiotic nitrogen fixation. However, it remains relatively poorly understood the molecular mechanism involved in the suppression. {middle dot}We performed LC-MS and RNA-seq analysis using rhizobia-infected Lotus japonicus roots and investigated the role of phosphatidylinositol (PI) and phosphatidylinositol phosphates (PIPs) in the symbiosis. Phosphatidylinositol transfer protein (PITP)-like proteins 4 (PLP4), phosphatidylinositol 3-phosphate 5-kinase 4 (PIP5K4) and PIP5K6 mutants, which are involved in the vesicular transport of lipids and phosphorylation of PIPs, were used to show the involvement of the signaling of PI and PIPs. Accumulation of phosphatidylinositol 4,5-bisphosphate [PI(4,5)P2] during rhizobial infection were examined by a fluorescent marker 1xTUBBY-C (TUBBY). {middle dot}We found that PI signaling-related genes were upregulated, and the amount of PIP2 increased in L. japonicus roots during rhizobial infection. In the PLP4, PIP5K4 and PIP5K6 mutants, rhizobial infection increased and PIP2 accumulation was failed. Furthermore, the observation of PI(4,5)P2 in rhizobia-infected roots revealed that the ectopic accumulation was closely related to suppression of rhizobial infection. {middle dot}Our findings indicate that the accumulation of PI(4,5)P2, which is mediated by PLP and PIP5Ks, suppresses excessive rhizobial infection in the root epidermis and cortex, leading to the optimal number of nodules.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.26.661640v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Impaired trap closure in the counting-deficient Venus flytrap mutant DYSCALCULIA is caused by cell wall biomechanics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Trebing, S., Freund, M., Iosip, A. L., Diblasi, C., Krennerich, V., Kirshner, J., Sato, M. P., Marques, A., Saitou, M., Becker, D., Albert, V. A., Tessmer, I., Fukushima, K., Hedrich, R., Kreuzer, I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the Dionaea muscipula DYSCALCULIA mutant is able to fire touch-induced APs, but unlike WT plants, it does not snap-close its traps after two consecutive APs.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Living in nutrient-poor environments, the carnivorous Venus flytrap Dionaea muscipula captures animal prey to compensate for this deficiency. Stimulation of trigger hairs located on the inner trap surface elicits an action potential (AP). While two consecutive APs result in fast trap closure in wildtype (WT) plants, sustained AP generation by the insect struggling to escape the trap leads to jasmonic acid (JA) biosynthesis, formation of the digestive stomach, and release of enzymes needed to decompose the victim. The Dionaea muscipula DYSCALCULIA (DYSC) mutant is able to fire touch-induced APs, but unlike WT plants, it does not snap-close its traps after two consecutive APs. Moreover, DYSC plants fail to properly initiate the JA pathway in response to mechanostimulation and even wounding, a well-known JA-dependent process conserved among plants. As demonstrated in previous studies, this DYSC mutant defect is associated with impaired decoding of mechanostimulation (i.e. touch) -induced Ca2+ signals. External JA application to the trap, however, restores slow trap closure and digestive gland function in DYSC, while rapid trap closure is JA-independent and cannot be rescued by exogenous JA application. Higher frequency mechanostimulation and thus more APs, however, revealed that DYSC is still able to close its traps, albeit much slower than WT plants. To reveal the molecular underpinnings of DYSCs delayed trap movement, we generated a chromosome scale Dionaea genome assembly and profiled gene expression. The refined transcriptomic analysis uncovered widespread misregulation of cell wall related genes in DYSC, implicating altered cell wall plasticity in the sluggish mutant. Cell indentation studies by atomic force microscopy revealed a strictly localized and strikingly enhanced stiffening of the cell wall for DYSC that may hinder rapid trap closure and snap buckling. Together, these genomic, transcriptomic, and biophysical data identify cell wall elasticity as a key constraint on voltage and Ca2+ dependent trap kinetics. This finding documents the interrelationship between mechanosensing and Ca2+ signaling in the ultrafast capture organ of the Venus flytrap.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.26.661685v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">BAR-CAT: Targeted Recovery of Synthetic Genes via Barcode-Directed CRISPR-dCas9 Enrichment</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Villegas, N. K., Tran, M. H., Keller, A., Plesa, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed BAR-CAT (Barcode-Assisted Retrieval CRISPR-Activated Targeting), an in-vitro enrichment method that couples unique PAM-adjacent 20-nt barcodes to each library member and uses multiplexed dCas9-</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Modern gene-synthesis platforms let us probe protein function and genome biology at unprecedented scale. Yet in large, diverse gene libraries the proportion of error-free constructs decreases with length due to the propagation of oligo synthesis errors. To rescue these rare, error-free molecules we developed BAR-CAT (Barcode-Assisted Retrieval CRISPR-Activated Targeting), an in-vitro enrichment method that couples unique PAM-adjacent 20-nt barcodes to each library member and uses multiplexed dCas9-sgRNA complexes to fish out the barcodes corresponding to perfect assemblies. After a single 15-min reaction and optimized wash regime (BAR-CAT v1.0), three low-abundance targets in a 300,000-member test library were enriched 600-fold, greatly reducing downstream requirements. When applied to 384x and 1,536x member DropSynth gene libraries, BAR-CAT retrieved up to 122-fold enrichment for 12 targets and revealed practical limits imposed by sgRNA competition and library complexity, which now guide ongoing protocol scaling. By eliminating laborious clone-by-clone validation and working directly on plasmid libraries, BAR-CAT provides a versatile platform for recovering perfect synthetic genes, subsetting large libraries, and ultimately lowering the cost of functional genomics at scale.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.27.658158v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Biosynthesis of lavandulol and lavandulyl acetate in Escherichia coli</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yang, D., Ma, X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We report the first de novo microbial biosynthesis of lavandulol and its ester derivative, lavandulyl acetate in Escherichia coli through modular pathway engineering.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Lavandulol and its ester derivative, lavandulyl acetate, are key fragrance constituents of lavender essential oil with widespread applications in cosmetics, perfumery, and food. However, traditional plant extraction suffers from low yield and unsustainable practices, and chemical synthesis relies on petrochemical feedstocks. Here, we report the first de novo microbial biosynthesis of lavandulol and lavandulyl acetate in Escherichia coli through modular pathway engineering. By screening 13 pyrophosphatases from E. coli, we identified RdgB as an efficient pyrophosphatase catalyzing the conversion of lavandulyl diphosphate into lavandulol, enabling its production at 24.9 mg/L. Building on this, we established a three-plasmid expression system and introduced a lavender-derived alcohol acyltransferase, LiAAT4, to achieve the biosynthesis of lavandulyl acetate at 42.4 mg/L. This study establishes the biosynthetic route to lavandulol in E. coli, demonstrating a viable and sustainable microbial platform for producing significant natural components of lavender oil. Our work provides the basis for future strain optimization and industrial-scale biomanufacturing of high-value monoterpene fragrance molecules.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.28.662091v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">GRASP: a modular toolkit for synthetic pentatricopeptide repeat RNA-binding proteins</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dennis, M. L., Low, S. Y., Viljoen, A., Pullakhandam, A., Colas des Francs-Small, C., Campbell-Clause, L., Bond, C. S., Small, I., Kwok van der Giezen, F. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-30
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the GRASP kit for assembly of synthetic PPR proteins, a library of 42 plasmids which can be combined to assemble synthetic P PR proteins with 9, 14 or 19 motifs to target any RNA sequence of the same length.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Pentatricopeptide repeat (PPR) proteins are eukaryotic RNA binding proteins with multiple roles in mitochondrial and chloroplast transcript processing. PPR proteins are naturally modular, and hold great potential for development into tools for RNA processing or controlling RNA folding or expression. However, construction of synthetic PPR proteins is challenging due to their highly repetitive sequences. Here, we present the GRASP kit for assembly of synthetic PPR proteins. Utilising the S-variant of PPR motifs, we designed a library of 42 plasmids which can be combined to assemble synthetic PPR proteins with 9, 14 or 19 motifs to target any RNA sequence of the same length. The GRASP kit enables rapid design and construction of PPR proteins of any desired specificity, and is compatible with the MoClo assembly standard. To demonstrate the capabilities of GRASP, we assembled a synthetic PPR RNA editing protein and variants with altered sequence specificity. We tested the functionality of 31 synthetic PPR protein variants against a set of 46 RNA targets, and used RNA sequencing to determine levels of RNA editing. The variations in editing provide a wealth of insights into PPR-RNA interactions. The GRASP kit provides a foundation for further development of synthetic PPR protein technologies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.29.661641v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrating AI/ML and multi-omics approaches to investigate the role of TNFRSF10A/TRAILR1 and its potential targets in pancreatic cancer.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sudhanshu Sharma, Rajesh Singh, Shiva Kant, Manoj K Mishra</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-26
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose TNFRSF10A as a therapeutically important PDAC vulnerability nurtured by spatially resolved expression patterns and dynamic molecular modeling. We propose a multi-layered system biology and drug discovery pipeline that integrates bulk genomics, single-cell spatial transcriptomics, proteomics,</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Pancreatic ductal adenocarcinoma (PDAC) remains one of the most lethal malignancies, with a five-year survival of under 10 % despite current therapies. Aggressive tumor biology, a desmoplastic stroma that limits drug delivery and immune cell infiltration, and profound resistance to apoptosis make it more complex to treat. Here, we describe a multi-layered system biology and drug discovery pipeline that integrates bulk genomics, single-cell spatial transcriptomics, proteomics, competing endogenous RNA (ceRNA) network analysis, and deep learning-driven quantitative structure-activity relationship (QSAR) modeling. By implementing this pipeline, we predicted that TNFRSF10A encodes for the TRAILR1 death receptor as a potential therapeutic target in PDAC. Mutational and expressional analysis also confirmed TNFRSF10A as a putative target in PDAC. Cancer cells within the PDAC microenvironment exhibit aberrantly elevated TNFRSF10A expression. Immune-excluded tumor niches and pro-survival signaling link this elevated expression. Using an advanced transformer-based deep learning approach, SELFormer, combined with QSAR analysis-based virtual screening, we identified previously unexplored FDA-approved drugs and natural compounds, i.e., Temsirolimus, Ergotamine, and capivasertib, with potential TRAILR1 modulatory effects. During molecular dynamics simulations, these repurposed candidates showed the highest binding affinities against TNFRSF10A for 300 ns. These showed favorable binding energies (MM-PBSA), minimal RMSD drift, PCA, and SASA. We propose TNFRSF10A as a therapeutically important PDAC vulnerability nurtured by spatially resolved expression patterns and dynamic molecular modeling. This study has used a novel integration of AI-implemented chemical modeling, high-throughput screening, and a multi-omics approach to unravel and pharmacologically target a cancer compartment-specific weakness in a notoriously drug-resistant cancer.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40424767/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A novel Harris Hawks Optimization-based clustering method for elucidating genetic associations in osteoarthritis and Diverse Cancer Types.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pınar Karadayı Ataş</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-23
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a new clustering approach based on Harris Hawks Optimization for the first time in the study of genetic interactions between OA and cancer types, including cancers of the bladder, kidney, breast, and prostate.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Considering the high incidence of osteoarthritis (OA), especially of the knee and hip, this study explores the possible genetic associations between OA and cancer types, including cancers of the bladder, kidney, breast, and prostate. The objective of our study is to decipher the complex genetic connections among these common disorders, emphasizing potential correlations and underlying biological processes. However, the genetic connections between these diseases remain largely unexplored. It fills a vacuum in the literature by using a new clustering approach based on Harris Hawks Optimization (HHO-C), which is a first for applying machine learning methods to this particular set of genetic data. To address this gap, we introduce HHO-C, a novel machine learning-based clustering approach, for the first time in this specific genetic dataset. The work accomplishes three noteworthy firsts: firstly, it is the first to apply machine learning to the study of the genetic interactions between OA and these cancers. Second, it creates a flexible genetic dataset that will be very helpful for further studies in this field. Finally, it presents the novel HHO-C approach, showcasing how well it manages intricate genetic data and providing fresh perspectives on genetic data analysis. It is anticipated that the results of this investigation will clarify the genetic relationships between OA and these malignancies, which could result in novel understandings of medical genetics and the creation of fresh approaches to diagnosis and treatment.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40412087/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Evaluating an information theoretic approach for selecting multimodal data fusion methods.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tengyue Zhang, Ruiwen Ding, Kha-Dinh Luong, William Hsu</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We evaluate the effect of parameter selection on multimodal data interactions and propose potential improvements to the partial information decomposition (PID) metrics.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Interest has grown in combining radiology, pathology, genomic, and clinical data to improve the accuracy of diagnostic and prognostic predictions toward precision health. However, most existing works choose their datasets and modeling approaches empirically and in an ad hoc manner. A prior study proposed four partial information decomposition (PID)-based metrics to provide a theoretical understanding of multimodal data interactions: redundancy, uniqueness of each modality, and synergy. However, these metrics have only been evaluated in a limited collection of biomedical data, and the existing work does not elucidate the effect of parameter selection when calculating the PID metrics. In this work, we evaluate PID metrics on a wider range of biomedical data, including clinical, radiology, pathology, and genomic data, and propose potential improvements to the PID metrics.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40354908/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrative Multi-Omics Analysis Reveals Molecular Subtypes of Ovarian Cancer and Constructs Prognostic Models.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Min Zhou, Jie Pi, Yuzi Zhao</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify molecular subtypes of ovarian cancer through multi-omics analysis and construct machine learning-based prognostic models for predicting the efficacy of immunotherapy.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Ovarian cancer (OV) remains the most lethal gynecological malignancy. The aim of this study was to identify molecular subtypes of OV through integrative multi-omics analysis and construct machine learning-based prognostic models for predicting the efficacy of immunotherapy. In here, the mutation, copy number variation, RNA sequencing expression profiles, and clinical information were obtained from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO) databases. Multi-omics data were stratified using the MOVICS package, identifying different molecular subtypes. Our analysis identified 2 molecular subtypes (CS1 and CS2) with significant survival differences. Transcriptional regulatory network analysis revealed differential activation of transcription factors such as FOXA1 and GATA3 in CS1, whereas AR and ESR2 were enriched in CS2. A robust prognostic signature comprising 5 key genes was developed through the integration of 10 machine learning algorithms, demonstrating high predictive power across data sets. Immune cell infiltration analysis revealed that anti-tumor immune cells were more abundant in low-risk groups, whereas pro-tumor immune cells predominated in high-risk groups. Furthermore, low-risk patients exhibited better immunotherapy responses and higher tumor mutational burden (TMB). In conclusion, our findings underscore the potential of multi-omics integration in unveiling novel OV subtypes and constructing predictive models that inform personalized treatment strategies. Future research should focus on validating these findings in larger cohorts to enhance OV management through targeted therapeutic approaches.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40200832/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Subtractive genomics approach: A guide to unveiling therapeutic targets across pathogens.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bhavya Mudgal, Devvret Verma, Divya Venogopal, Suraj V Atram, Debasis Mitra, Sugam Gupta</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comprehensive review of the subtractive genomics technique which was applied to various pathogens, often highlighting the effectiveness of the methodology in drug target discovery and novel therapeutics development.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Subtractive genomics is an adaptable bioinformatics technique that is used to identify potential therapeutic targets by differentiating essential genes in pathogens and non-pathogenic genes. Since, identification of therapeutic targets and understanding of their structure, function, and role in pathogenesis is important in development of drug design. Therefore, this review will provide a comprehensive look at the subtractive genomics technique which was applied to various pathogens, often highlighting the effectiveness of the methodology in drug target discovery and novel therapeutics development. Tools and software such as BLAST, Roary, and AutoDock Vina are widely utilized in this methodology for various aspects such as, genome comparison, essential gene identification, clustering, subcellular localization, pathway analysis, molecular docking etc. Diseases such as tuberculosis, botulism, staphylococcal infections, ventilator-associated pneumonia, secondary meningitis, gonorrhoea, septicaemia, etc., are among the infectious diseases targeted using subtractive genomics. Comparison of basic principles, tools, and advancements use these subtractive genomics studies, will provide insight into the adaptable nature of this technique and the diversity of pathogens, which have benefited with this methodology into providing successful results. The main focus is on the genome sequencing advancements, annotation and validation through in-silico techniques, to find effective drug targets while, decreasing the possibility of toxicity in the host. We have also discussed the possibility of taking a multi-omics approach and incorporating AI and machine learning to expand on the current data and finding effective therapeutics for helping globally on health challenges.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40204082/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Pancreatic Cancer Detection and Differentiation from Chronic Pancreatitis: Potential Biomarkers Identified through a High-Throughput Multiplex Proteomic Assay and Machine Learning-Based Analysis.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Young-Gon Kim, Sang-Mi Kim, Soo-Youn Lee</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify blood protein biomarkers for PC diagnosis and differential diagnosis of CP using high-throughput multiplex proteomic analysis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Pancreatic cancer (PC)-screening methods have limited accuracy despite their high clinical demand. Differential diagnosis of chronic pancreatitis (CP) poses another challenge for PC diagnosis. Therefore, we aimed to identify blood protein biomarkers for PC diagnosis and differential diagnosis of CP using high-throughput multiplex proteomic analysis.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40170581/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multi-Omics clustering by integrating clinical features from large language model.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xiucai Ye, Tianyi Shi, Dong Huang, Tetsuya Sakurai</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel framework that incorporates clinical features extracted from large language model (LLM) to enhance multi-omics clustering for cancer subtyping .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Multi-omics clustering has emerged as a powerful approach for understanding complex biological systems and enabling cancer subtyping by integrating diverse omics data. Existing methods primarily focus on the integration of different types of omics data, often overlooking the value of clinical context. In this study, we propose a novel framework that incorporates clinical features extracted from large language model (LLM) to enhance multi-omics clustering. Leveraging clinical data extracted from pathology reports using a BERT-based model, our framework converts unstructured medical text into structured clinical features. These features are integrated with omics data through an autoencoder, enriching the information content of each omics layer to improve feature extraction. The extracted features are then projected into a latent subspace using singular value decomposition (SVD), followed by spectral clustering to obtain the final clustering result. We evaluate the proposed framework on six cancer datasets on three omics levels, comparing it with several state-of-the-art methods. The experimental results demonstrate that the proposed framework outperforms existing methods in multi-omics clustering for cancer subtyping. Moreover, the results highlight the efficacy of integrating clinical features derived from LLM, significantly enhancing clustering performance. This work underscores the importance of clinical context in multi-omics analysis and showcases the transformative potential of LLM in advancing precision medicine.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40180255/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Development and validation of a machine learning prognostic model based on an epigenomic signature in patients with pancreatic ductal adenocarcinoma.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gian Maria Zaccaria, Nicola Altini, Valentina Mongelli, Francescomaria Marino, Vitoantonio Bevilacqua</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-03-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> In Pancreatic Ductal Adenocarcinoma (PDAC), current prognostic scores are unable to fully capture the biological heterogeneity of the disease.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In Pancreatic Ductal Adenocarcinoma (PDAC), current prognostic scores are unable to fully capture the biological heterogeneity of the disease. While some approaches investigating the role of multi-omics in PDAC are emerging, the analysis of methylation data is under exploited.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40157244/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Development of Machine Learning Models for Predicting Radiation Dermatitis in Breast Cancer Patients Using Clinical Risk Factors, Patient-Reported Outcomes, and Serum Cytokine Biomarkers.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Neil Lin, Farnoosh Abbas-Aghababazadeh, Jie Su, Alison J Wu, Cherie Lin, Wei Shi, Wei Xu, Benjamin Haibe-Kains, Fei-Fei Liu, Jennifer Y Y Kwan</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-03-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study on the utility of subjective and objective factors, such as patient-reported outcomes (PROs) and serum cytokines, for predicting RD in breast cancer patients.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Radiation dermatitis (RD) is a significant side effect of radiotherapy experienced by breast cancer patients. Severe symptoms include desquamation or ulceration of irradiated skin, which impacts quality of life and increases healthcare costs. Early identification of patients at risk for severe RD can facilitate preventive management and reduce severe symptoms. This study evaluated the utility of subjective and objective factors, such as patient-reported outcomes (PROs) and serum cytokines, for predicting RD in breast cancer patients. The performance of machine learning (ML) and logistic regression-based models were compared.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40155248/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

