<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Monday, 04 August 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Prediction of protein-protein interaction based on interaction-specific learning and hierarchical information.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tao Tang, Taiguang Shen, Jing Jiang, Weizhuo Li, Peng Wang, Sisi Yuan, Xiaofeng Cao, Yuansheng Liu</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-04
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A deep learning model for protein-protein interaction prediction .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Prediction of protein-protein interactions (PPIs) is fundamental for identifying drug targets and understanding cellular processes. The rapid growth of PPI studies necessitates the development of efficient and accurate tools for automated prediction of PPIs. In recent years, several robust deep learning models have been developed for PPI prediction and have found widespread application in proteomics research. Despite these advancements, current computational tools still face limitations in modeling both the pairwise interactions and the hierarchical relationships between proteins.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40754535/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Snappy: de novo identification of DNA methylation sites based on Oxford Nanopore reads</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Konanov, D. N., Krivonos, D. V., Babenko, V. V., Ilina, E. N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a new algorithm Snappy, which is actually rethinking of the original Snapper algorithm but does not use any enrichment heuristics and does not require control sample sequencing.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Summary Nowadays, search for methylation sites in bacteria is usually performed by direct detection of nucleotide motifs over-represented in modified contexts, using classical motif enrichment approaches oriented only on context sequences themselves. Herein, we present a new algorithm Snappy, which is actually rethinking of the original Snapper algorithm but does not use any enrichment heuristics and does not require control sample sequencing. Opposite to previous methods, Snappy uses raw basecalling data simultaneously with the motif enrichment process, thus significantly enhancing the enrichment sensitivity and accuracy compared with other enrichment algorithms. The versatility of the method was shown on both our and external data, representing different bacterial taxa with complex and simple methylome. Availability and implementation Source code and documentation is hosted on GitHub (https://github.com/DNKonanov/ont-snappy) and Zenodo (zenodo.org/records/16731817). For accessibility, Snappy is installable from PyPi using "pip install ont-snappy" command.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.03.668330v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Boost or Bust? The impact of supplementation on functional genetic diversity and selective processes in Tasmanian devils</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Schraven, A. L., Farquharson, K. A., Batley, K. C., Fox, S., Lee, A. V., Belov, K., Silver, L. W., Hogg, C. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We found that functional gene diversity increased post supplementation of Tasmanian devil populations, and found no evidence of swamping of local alleles or those putatively associated with DFTD.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Translocating individuals into existing populations of conspecifics can support threatened species by increasing population size, maintaining genetic diversity, and reducing the risk of inbreeding. However, for species whose adaptive potential is compromised due to ongoing threats, like disease, the outcome of such management interventions becomes more complex. The Tasmanian devil (Sarcophilus harrisii) is a prime example, where the emergence of Devil Facial Tumour Disease (DFTD) has led to significant population declines, raising concerns about their long-term survival. It is therefore critical to understand if the introduction of new functional genetic variants through supplementation actions enhances, or potentially hinders, their long-term persistence. We investigated changes in functional gene diversity at both the population- and individual-levels, pre and post supplementation, across multiple wild devil sites (four supplemented and four not supplemented). We found that functional diversity increased post supplementation. Though the magnitude of change was varied among sites, a similar site-specific pattern was also evident in genome-wide diversity. Importantly, we saw no evidence of swamping of local alleles or those putatively associated with DFTD regressions. This is likely due to the source population representing the broad wild genetic diversity and supplementations facilitating gene flow across the current fragmented landscape. Continued and long-term monitoring at multiple wild sites will be necessary to determine whether future generations retain this introduced genetic variation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.03.668351v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">GeneTerrain-GMM Unmasks a Coordinated Neuroinflammatory and Cell Death Network Perturbed by Dasatinib in a Human Neuronal Model of Alzheimer's Disease</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Song, K. M., Zhang, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the multi-target tyrosine kinase inhibitor dasatinib induces a profound, global reorganization of the gene expression landscape in Alzheimer's disease and demonstrate the ability of topological data analysis to uncover the higher-order logic of complex drug-gene interactions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Alzheimer's disease (AD) is a systems-level disorder driven by the failure of interconnected biological networks, demanding therapeutic strategies and analytical frameworks that operate beyond single-target paradigms. The multi-target tyrosine kinase inhibitor dasatinib is a promising therapeutic candidate due to its senolytic properties and ability to cross the blood-brain barrier. However, its precise systems-level impact on AD-relevant gene networks remains unresolved. Here, we employ the GeneTerrain-GMM (gtGMM) framework, a novel method combining topological data analysis with unsupervised clustering, to dissect the mechanism of dasatinib in a human SH-SY5Y neuronal model. By analyzing high-throughput gene expression data from a curated network of 100 AD-implicated genes, we demonstrate that dasatinib, unlike an inert control, induces a profound, global reorganization of the gene expression landscape. This response is characterized by a smooth, highly coordinated topography, which gtGMM decomposes into a dominant, statistically robust functional module. Strikingly, this single module integrates canonical AD pathways with distinct signatures of neuroinflammation, apoptosis, and ferroptosis, a convergence that provides a powerful mechanistic fingerprint of the drug's action. Our findings reveal that dasatinib's effect is not diffuse but is focused on a critical nexus linking protein misprocessing to inflammation and programmed cell death. This study provides a systems-level rationale for the therapeutic potential of dasatinib in AD and showcases the ability of topological data analysis to uncover the higher-order logic of complex drug-gene interactions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.03.667745v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Learning from B Cell Evolution: Adaptive Multi-Expert Diffusion for AntibodyDesign via Online Optimization</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Feng, H., Qiu, P., Poczos, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a biologically-motivated framework that leverages physics-based domain knowledge within an online meta-learning system to learn personalized optimization strategies for antibody design.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent advances in diffusion models have shown remarkable potential for antibody design, yet existing approaches apply uniform generation strategies that cannot adapt to each antigen's unique requirements. Inspired by B cell affinity maturation--where antibodies evolve through multi-objective optimization balancing affinity, stability, and self-avoidance--we propose the first biologically-motivated framework that leverages physics-based domain knowledge within an online meta-learning system. Our method employs multiple specialized experts (van der Waals, molecular recognition, energy balance, and interface geometry) whose parameters evolve during generation based on iterative feedback, mimicking natural antibody refinement cycles. Instead of fixed protocols, this adaptive guidance discovers personalized optimization strategies for each target. Our experiments demonstrate that this approach: (1) discovers optimal SE(3)-equivariant guidance strategies for different antigen classes without pre-training, preserving molecular symmetries throughout optimization; (2) significantly enhances hotspot coverage and interface quality through target-specific adaptation, achieving balanced multi-objective optimization characteristic of therapeutic antibodies; (3) establishes a paradigm for iterative refinement where each antibody-antigen system learns its unique optimization profile through online evaluation; (4) generalizes effectively across diverse design challenges, from small epitopes to large protein interfaces, enabling precision-focused campaigns for individual targets.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.02.668313v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MicrobioRel: A Set of Datasets for Microbiome Relation Extraction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> EL KHETTARI, O., Batteux, D., Quiniou, S., Chaffron, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a corpus of two datasets to study the relations between biological entities in the gut microbiome.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Biomedical knowledge curation relies on a variety of Natural Language Processing tasks, including biomedical entity recognition and document-level relation extraction. With the growing size and capabilities of Language Models, effectively deploying them in specific and specialised domains remains a persistent challenge, highlighting the need for high-quality, domain-adapted datasets. In this work, we present MicrobioRel, a corpus of two datasets to study the relations between biological entities in the gut microbiome. The first dataset, MicrobioRel-cur, is a document-level labelled corpus,corresponding to paragraphs from journal articles that were manually annotated with different types of relations between biomedical concepts in the gut microbiome domain. We describe its creation process, annotation guidelines, and keystatistics. On this dataset, we evaluated different architectures for relation extraction and identify PubMedBERT as the most effective model for this task. We also created a second dataset, MicrobioRel-pred, by generating relation predictionson other journal articles using the fine-tuned PubMedBERT model. We demonstrate its potential to extract meaningful interactions. The MicrobioRel is a crucial resource for advancing tasks like automatic knowledge extraction in specialised domains such as the gut microbiome, facilitating hypothesis generation and supporting scientific discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.03.666357v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A novel mobile genetic element with virus-like characteristics is widespread in the world's oceans</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kirchberger, P. C., Luque, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new group of small, circular DNA genetic elements, with striking similarities to the capsids of viruses from the realms Varidnaviria and Duplodnaviriania, found exclusively in marine datasets from the Arctic to the Antarctic.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> While the majority of viruses falls within a handful of well-defined realms, many novel viruses and virus-like genetic elements remain to be discovered. Given the abundance and diversity of small, circular DNA genomes in animal gut viromes, we hypothesized that viruses of similar size might have been overlooked in other environments. Re-analyzing existing ocean virome datasets for small viruses, we discover a novel group of circular mobile genetic elements of ~4.6kb in size, likely with double-stranded DNA, which we term Charybdis elements. Charybdis elements form a singular group based on cohesive gene content, encode around 10 hypothetical genes but apparently contain no virus, transposon or plasmid associated hallmark proteins. They are detected exclusively in marine datasets ranging from surface water to hydrothermal vents and sediments, from the Arctic to the Antarctic. Virome datasets separating virus-like particles by density gradients show that Charybdis elements accumulate in the fraction representing extracellular vesicles and small, tail-less viruses that are often associated with lipid membranes. One specific clade of Charybdis elements is shown to integrate into the genomes of the common marine group II archaeon candidate order Poseidoniales, the putative host of these elements. Analysis of thousands genomes shows a common genome architecture consisting of two modules: An accessory module containing a diverse range methyltransferases and other genes likely involved in host interactions, and a more conserved (putative) structural module. The structural module encodes proteins that are confidently predicted to form striking tri- or pentameric structures. These structures show stark resemblance and in some cases homology to protrusion or tail-elements on the capsids of viruses from the realms Varidnaviria and Duplodnaviria. Given these traits, Charybdis elements likely represent a hitherto undiscovered group of virus or virus-satellites, with at least one protein of a novel double-Greek-key fold serving as a potential capsid element. However, since their hosts remain unculturable, the true nature of these ubiquitous marine elements remains as of yet unresolvable.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.03.668216v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Fetal signatures in the 3D genome of iPSC-derived neurons: implications for disease modeling</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zagirova, D. R., Kononkova, A. D., Morozov, K. V., Molodova, M. N., Vaulin, N. S., Dudkovskaia, A. V., Dozorova, P. I., Efimova, O. I., Tvorogova, A. V., Ulianov, K. A., Khaitovich, P. E., Razin, S. V., Lagarkova, M. A., Ulianov, S. V., Khrameeva, E. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> iPSC-derived neurons largely retain chromatin state of undifferentiated cells and resemble fetal rather than mature neurons, but they effectively model interactions linking disease-associated SNPs to target genes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Induced pluripotent stem cells (iPSCs) have revolutionized neuroscience, providing an approach to generate patient-specific neurons for modeling of neurological diseases. However, it remains unclear how closely iPSC-derived neurons replicate the chromatin architecture of authentic brain neurons. Here, we collected and uniformly processed more than 300 human and mouse neuronal Hi-C datasets and examined chromatin features ranging from chromatin compartments and topologically associating domains (TADs) to chromatin loops, Polycomb-mediated contacts, and frequently interacting regions (FIREs). We found that iPSC-derived neurons largely retain chromatin state of undifferentiated cells and resemble fetal rather than mature neurons. iPSC-derived neurons exhibit unusually strong compartmentalization, an enrichment of developmental genes at TAD borders, and a marked reduction of long-range repressive Polycomb-mediated contacts that typically silence early fetal programs. Conversely, these cells robustly capture contacts involving GWAS SNPs relevant to neurological diseases. Thus, while iPSC-derived neurons may not fully replicate 3D genome architecture of adult neurons, they effectively model interactions linking disease-associated SNPs to target genes. Our study offers a valuable Hi-C resource for the community and provides a detailed comparison of chromatin architecture throughout neuronal maturation, underscoring its importance for validating neuronal models and providing a robust framework for future studies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.03.667702v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comparative Analysis of De Novo Assemblers and Quantification Software for RNA-sequencing Data in Non-Model Arthropods</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Brasseur, M. V., Leese, F., Mayer, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We evaluated the performance of different de nonvo transcriptome assembler (Trinity, rnaSPAdes, IDBA-tran) and transcript quantification software (RSEM, Salmon) on transcriptomic data of a non-model insect and freshwater crustacean species, as well</p><p style="font-size: 16px; text-align: justify;">Background: RNA-sequencing has greatly improved our understanding of the transcriptomic regulation of fundamental biological processes. Although the method has matured significantly within the last decade, bioinformatic processing of the resulting high-dimensional data sets is still challenging and the performance of algorithms can vary between data sets. As a consequence, for most non-model organisms, in particular arthropods, there is no or limited literature evidence which software is best suited to handle taxon-specific data characteristics. Therefore, we evaluated the performance of different de nonvo transcriptome assembler (Trinity, rnaSPAdes, IDBA-tran) and transcript quantification software (RSEM, Salmon) on transcriptomic data of a non-model insect and freshwater crustacean species, as well as the impact of different quality trimming strategies on the downstream bioinformatic processing results. Results: While the trimming strategy had no considerable effect on the quality of transcriptome assemblies, the choice of the assembler had a substantial impact. IDBA-tran was less sensitive than the two other assemblers and produced the most fragmented transcriptome assemblies. The low remapping rates of reads against IDBA-tran assemblies further suggest that the input read data was not effectively leveraged by this algorithm. In contrast, Trinity and rnaSPAdes both generated comprehensive and contiguous de novo transcriptome assemblies, although Trinity appeared to be slightly more sensitive. This increased sensitivity, however, was associated with a higher redundancy in Trinity-generated assemblies compared to assemblies produced with rnaSPAdes. When the quality of the transcriptome assembly was high, RSEM and Salmon were able to identify the origin of at least 90% of the read data in the reference. Despite their different underlying quantification approaches, the estimated transcript counts of both tools were highly correlated and their expression signal was consistent. Notably, the alignment-free quantification algorithm Salmon was substantially faster than the alignment-based approach of RSEM. Furthermore, it was also slightly more sensitive, increasing the average re-mapping rate to ~98%. Conclusion: Since the performance of bioinformatic algorithms, especially of de novo assemblers, varies for different RNA-sequencing data sets, establishing an appropriate analysis workflow remains an important task. Our results show that the better performing combinations of algorithms produce congruent count data sets with consistent expression signal, highlighting the robustness of RNA-sequencing data analysis software.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.01.668104v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ProteomeLM: A proteome-scale language model allowing fast prediction of protein-protein interactions and gene essentiality across taxa</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Malbranke, C., Zalaffi, G. P., Bitbol, A.-F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce ProteomeLM, a transformer-based language model that reasons on entire proteomes from species spanning the tree of life, and substantially outperforms amino-acid coevolution-based methods for protein-protein interactions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Language models starting from biological sequence data are advancing many inference problems, both at the scale of single proteins, and at the scale of genomic neighborhoods. In this paper, we introduce ProteomeLM, a transformer-based language model that reasons on entire proteomes from species spanning the tree of life. Leveraging protein language model embeddings, ProteomeLM is trained to reconstruct masked protein embeddings using the whole proteomic context. It thus learns contextualized protein representations reflecting proteome-scale functional constraints. We show that ProteomeLM spontaneously captures protein-protein interactions (PPI) in its attention coefficients. We demonstrate that it screens whole interactomes orders of magnitude faster than amino-acid coevolution-based methods, and substantially outperforms them. We further develop ProteomeLM-PPI, a supervised PPI prediction network that combines ProteomeLM embeddings and attention coefficients, and achieves state-of-the-art performance across species and benchmarks. Finally, we introduce ProteomeLM-Ess, a supervised predictor of gene essentiality that generalizes across diverse taxa. Our results highlight the power of proteome-scale language models for addressing function and interactions at the organism level.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.01.668221v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Drought and methyl jasmonate memory interact to determine plant functioning under current drought stress in a perennial grass</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bhatt, T., Rathore, N., Semerad, J., Cajthaml, T., Thakur, D., Munzbergova, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Drought memory and hormone induced memory are transgenerational and compositional, with synergistic roles in plant stress adaptation, with complex integration of environmental and hormonal legacies.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Climate-driven droughts necessitate deeper insights into plant resilience, particularly in clonal perennials. This study presents the first integrative analysis of how recurrent drought (D2), drought memory (D1), and methyl jasmonate-induced memory (M1) independently and interactively shape metabolomic diversity, composition, and morpho-physiological traits. Using Festuca rubra as a model species and using a full-factorial design, we found that current drought (D2) was the primary driver of metabolic reprogramming, reducing metabolite richness but enhancing uniqueness, suggesting a streamlined, stress-optimized strategy. D1 and M1 had significant, context-dependent impacts: D1 increased richness under well-watered conditions but reversed under drought, while M1 stabilized both metabolomic and trait variability. Importantly, D1 and M1 effects were non-additive, producing emergent biochemical states and morpho-physiology. A significant D1 x D2 x M1 interaction revealed complex integration of environmental and hormonal legacies. The metabolomic shifts closely aligned with traits such as specific leaf area and relative water content, indicating a coordinated adaptation at both metabolomic and morpho-physiological levels through memory-driven mechanisms. Our findings confirm that both drought and hormone induced memories are transgenerational and compositional, with synergistic roles in plant stress adaptation. This suggests that combining hormonal and environmental priming strategies may be a viable approach to enhancing plant resilience under increasing climate variability.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.03.668340v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Rewiring amino acid metabolism to boost nonnatural polyketide production in Yarrowia lipolytica</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, J., Hong, Y., Wu, Z., Fishman, A., Xu, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose two thermodynamically favorable malonyl-CoA pathways for the production of polyketides and a new-to-nature polyketide derived from the side activity of TAL pathway.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Malonyl-CoA is a central precursor involved in the synthesis of various bio-based chemicals, including polyketides, fatty acids and flavonoids. However, the production of these chemicals is often limited by the availability of malonyl-CoA. Based on retrosynthesis principles, we designed two thermodynamically favorable malonyl-CoA pathways using L-glutamate and L-aspartate as substrates. The novel pathways leverage oxidative deamination and decarboxylation reactions and efficiently channel metabolic flux toward malonyl-CoA, leading to increased triacetic acid lactone (TAL) production beyond the capability of the native acetyl-CoA carboxylase. We also discovered a new-to-nature polyketide (4-hydroxy-6-hydroxyethyl-2-pyrone) derived from the side activity of TAL pathway, reaching 5.4 g/L. This work highlights the utility of the novel malonyl-CoA pathways in enhancing polyketide production, and the possibility of using abundant amino acids or protein waste as feedstock to produce high-value nonnatural polyketides.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.03.668335v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

