<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Tuesday, 10 June 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Bridging Genomics and Clinical Medicine: RSVrecon Enhances RSV Surveillance with Automated Genotyping and Clinically-important Mutation Reporting</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Li L, Yi H, Brazelton JN, Webby R, Hayden RT, Wu G, Hijano DR.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PPR
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce RSVrecon, a bioinformatics pipeline for comprehensive RSV genome assembly and phylogenetic analysis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Respiratory Syncytial Virus (RSV) causes significant respiratory infections, particularly in young children and elderly adults. Genetic variations in the fusion (F) protein can reduce the efficacy of vaccination and monoclonal antibody treatments, emphasizing the need for genomic surveillance. Motivation: Current pipelines for RSV genome assembly focus on sequence reconstruction but often lack features for detecting genotypes, clinically relevant mutations, or presenting results in formats suitable for clinical researchers. We introduce RSVrecon, a bioinformatics pipeline for comprehensive RSV genome assembly and phylogenetic analysis. A key innovation of RSVrecon is its integrated detection of clinically critical features, including genotypes classification and F protein mutations calling, capabilities notably absent from most analytical pipelines. Its modular design, powered by Nextflow's modern framework, ensures a scalable and robust workflow, while user-friendly reports enable seamless translation of genomic data into actionable clinical insights. By bridging the gap between genomic sequencing and clinical application, RSVrecon outperforms existing pipelines in three key dimensions: functionality, result interpretation, and usability. This advancement enhances RSV research, supports informed clinical decision-making, and strengthens public health surveillance.</p>
                <p><a href="https://doi.org/10.1101/2025.06.03.657184" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Characterization of Somatic Mutations in Human tRNA Genes Reveals Tumor-Specific Mutational Loads, and the Generation of tRNA Variants that Alter the Genetic Code</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Murillo, M., Salvadores, M., Torres, A. G., Vaquer Pico, A., Tsapanou, L., Supek, F., Ribas de Pouplana, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that highly transcribed transfer RNA genes are hotspots for somatic mutagenesis, and show that they display mutational loads that are directly proportional to their transcription rates.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Transfer RNA genes (tDNAs) are essential genomic elements that safeguard translational fidelity. Using the T2T version of the human genome we have mapped the position of human tDNAs and analyzed their individual transcriptional activities. Then we have characterized, at single base resolution, the impact of somatic mutations in human tDNAs and its relationship to the transcriptional status of each gene. We confirm that tDNAs are hotspots for somatic mutagenesis, and show that they display mutational loads that are directly proportional to their transcription rates. Highly transcribed tDNAs in tumors or healthy tissues accumulate mutations at rates up to nine-fold higher than highly transcribed protein-coding genes. Mutational loads at tDNAs are tumor-specific, and increase with patient age. Mutations at structurally conserved tRNA positions appear to be under negative selection. Strikingly, anticodon nucleotides crucial for decoding frequently acquire somatic mutations, readily generating chimeric tRNAs species capable of systematically introducing amino acid substitutions across the proteome. Our results reveal a previously unrecognized source of somatic heterogeneity in human cancer and aging tissues that may directly impact upon translation efficiency and fidelity, and cause cell-specific proteostasis degeneration.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.09.658696v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A systematic delineation of 3'UTR regulatory elements and their contextual associations</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liang, N., Li, J.-Y., Ding, Y., Wang, Y., Champer, J., Ren, L.-C., Gao, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A deep learning model for understanding the 3' region of genes and its role in transcriptional and post-transcriptional regulation .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The 3' region of genes plays a crucial role in transcriptional and post-transcriptional regulation, yet the functional elements within it remain largely unknown. Here, we assessed the impact of over 1.6 million synthetic 3'UTR sequences on transcript abundance and subcellular localization in A549 cells, revealing a substantial number of relatively independent elements with distinct regulatory associations. By training a context-aware deep learning model, we found that these regulatory elements exhibit modest context dependency but possess unique preferences individually. Through relatively weak interactions and appropriate context arrangements, multiple elements can synergistically generate significant regulatory effects. Finally, we demonstrated the possibility for applying our model to identify disease-related 3'UTR variants.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.09.658412v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Boosting AlphaFold Protein Tertiary Structure Prediction through MSA Engineering and Extensive Model Sampling and Ranking in CASP16</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liu, J., Neupane, P., Cheng, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present MULTICOM4, an integrative protein structure prediction system that uses diverse MSA generation, large-scale model sampling, and an ensemble model quality assessment (QA) strategy of combining individual QA methods to improve model generation and ranking of AlphaFold2 and AlphaF</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> AlphaFold2 and AlphaFold3 have revolutionized protein structure prediction by enabling high-accuracy tertiary structure predictions for most single-chain proteins. However, obtaining high-quality predictions for some hard protein targets with shallow or noisy multiple sequence alignments (MSAs) and complicated multidomain architectures remains challenging. Here, we present MULTICOM4, an integrative protein structure prediction system that uses diverse MSA generation, large-scale model sampling, and an ensemble model quality assessment (QA) strategy of combining individual QA methods to improve model generation and ranking of AlphaFold2 and AlphaFold3. In the 16th Critical Assessment of Techniques for Protein Structure Prediction (CASP16), our predictors built on MULTICOM4 ranked among the top performers out of 120 predictors in tertiary structure prediction and outperformed a standard AlphaFold3 predictor. The average TM-score of our best performing predictor MULTCOM's top-1 prediction for 84 CASP16 domain is 0.902. It achieved high accuracy (TM-score > 0.9) for 73.8% of the 84 domains and correct fold predictions (TMscore > 0.5) for 97.6% domains in terms of top-1 prediction. In terms of best-of-top-5 prediction, it predicted correct folds for all the domains. The results show that MSA engineering through the use of different protein sequence databases, alignment tools, and domain segmentation as well as extensive model sampling are the key to generate accurate and correct structural models. Additionally, using multiple complementary QA methods and model clustering can improve the robustness and reliability of model ranking.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.06.658338v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AI-Driven Identification and Validation of RXFP1 as a CML Biomarker Using Gene Expression and Integrated GUI Tools</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Khan, M. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified the top-ranked gene in a random forest model trained to differentiate CML samples from normal samples, and found that RXFP1 is a robust and treatment-independent molecular biomarker for CML, laying a foundation for larger studies and therapeutic exploration.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Chronic myeloid leukemia (CML) is a hematologic malignancy primarily driven by the BCR-ABL1 fusion gene. While tyrosine kinase inhibitors (TKIs), such as imatinib, have transformed CML management, many patients still experience adverse side effects and therapy resistance. This study investigated the potential of RXFP1 (relaxin family peptide receptor 1) as a diagnostic biomarker for CML, leveraging bioinformatics, statistical analysis, and artificial intelligence (AI)-based modeling. Using the GSE97562 gene expression dataset, which consists of 40 human bone marrow samples, we identified RXFP1 as the top-ranked gene in a random forest model trained to differentiate CML samples from normal samples. The gene exhibited statistically significant overexpression in CML (p < 0.0001), with an area under the ROC curve (AUC) of 1.0, indicating perfect classification capability. Notably, RXFP1 retained its expression profile even after imatinib treatment, suggesting its independence from BCR-ABL signaling and making it a promising candidate for disease monitoring regardless of therapeutic status. To facilitate nonprogrammatic analysis and reproducibility, a GUI tool was developed using Python, which integrates statistical tests, visual plots (ROC, box plots, Mann-Whitney curve), and a text-to-speech reporting system. Manual validation further confirmed RXFP1's discriminatory power, with zero expression overlapping between CML and normal samples across all experimental subgroups. This research is also driven by a personal motivation: the author's wife is a CML patient undergoing TKI treatment and experiencing side effects. This study emerged from the author's desire to find a safer and more precise diagnostic pathway. These findings suggest that RXFP1 is a robust and treatment-independent molecular biomarker for CML, laying a foundation for larger studies and therapeutic exploration. This study also demonstrates the potential of integrated AI tools in accelerating biomarker discovery and empowering independent researchers to contribute to precision medicine.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.06.658261v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identification of reactive CpGs and RNA expression in early COVID-19 through cis-eQTM analysis reflecting disease severity and recovery</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ryu, H., An, K., Kwon, Y., Jeon, Y., Jeon, S., Choi, H., Kim, Y. J., Kim, S., Sul, O. J., Lee, S., Chun, A. Y., Shin, E.-S., Ra, S. W., Bhak, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified 16 differentially expressed genes and 30 nearby differentially methylated CpG sites in severe COVID-19 cases, which are associated with viral infiltration, immune activation, lung damage, and oxidative stress-related multi-organ failure.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Multi-omics analyses of severe COVID-19 cases are crucial in deciphering the complex interplay between genetic and epigenetic factors. Here, we present an analysis of Expression Quantitative Trait Methylation (eQTM) to investigate the complex interplay of methylation and gene expression pattern during the acute phase of severe COVID-19. We identified 16 differentially expressed genes and 30 nearby differentially methylated CpG sites. Six key genes SRXN1, FURIN, IL18RAP, FOXO3, GCNT4, and FKBP5 were either up-regulated or down-regulated near hypomethylated CpG sites. These genes are associated with viral infiltration, immune activation, lung damage, and oxidative stress-related multi-organ failure, which are the hallmarks of severe COVID-19. Interestingly, during the recovery phase, methylation and gene expression levels returned to baseline, underscoring the rapid and reversible nature of these molecular changes. These findings provide insight into the dynamics of epigenetic and transcriptomic shifts according to the infectious stage, supporting potential prognostic and therapeutic approaches for severe COVID-19.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.09.658549v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Chromosome-Level Assembly of the Pine Processionary Moth (Thaumetopoea pityocampa) genome</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gautier, M., Nouhaud, P., Lagnel, J., Saune, L., Legeai, F., Parinello, H., Gschloessl, B., Marande, W., Lopez-Roques, C., Dorkeld, F., Branco, M., Perrier, C., Kerdelhue, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a chromosome-level genome assembly and annotation of the pine processionary moth, Thaumetopoea pityocampa (Lepidoptera: Notodontidae), a key forest pest that is a public health concern.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We present a chromosome-level genome assembly and annotation of the pine processionary moth, Thaumetopoea pityocampa (Lepidoptera: Notodontidae), a key forest pest that is a public health concern. The nuclear genome spans 615.9 Mb, scaffolded into 50 chromosome scale and 115 smaller scaffolds, with high completeness (BUSCO score: 98.9%) that provides a decisive improvement over the previous assembly (537 Mb; 68,292 contigs; BUSCO 83.6%). Coverage differences in resequenced males and females allowed identification of the Z chromosome and several W-linked contigs. As expected from previous studies, we found that synteny was largely conserved with related Lepidoptera, although chromosomal fissions may explain the higher chromosome number of 49 autosomes compared to typical lepidopteran karyotypes. We also integrated into the assembly linkage map, allowing estimation of a genome-wide male recombination rate of 5.06 cM/Mb, varying from 11.6 cM/Mb to 1.98 cM/Mb from the smallest to the largest chromosomes. Repetitive elements represented 49.1% of this new assembly, dominated by LINEs (45.1% of classified repeats). Finally, gene prediction identified 12,898 gene models, among which 17 circadian rhythm genes were manually curated. Overall, this assembly represents the first chromosome-level genome for a member of the Thaumetopoeinae subfamily, significantly expanding the currently limited set of genomic resources available for Notodontidae. The fully annotated assembly is publicly accessible through the LepidoDB database (https://bipaa.genouest.org/is/lepidodb/) and will serve as a valuable resource for research on population genomics of this species.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.06.658235v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SMART: An Integrated Platform Revolutionizing Cryo-Electron Microscopy Workflow from Data Acquisition to Atomic Modeling</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liu, H., Wu, Y., Zhang, W., Xu, H., Guo, Z., Meng, F., Men, X., Guo, A. C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> This paper presents SMART, the first fully integrated cryo-EM platform unifying three critical AI-driven modules: DataSmart (automated data collection), CryoSmart (3D reconstruction), and ModelSmart (end-to-end atomic modeling).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent advancements in cryo-electron microscopy (cryo-EM) have revolutionized structural biology, yet persistent workflow fragmentation between data acquisition, reconstruction, and modeling limits its full potential. This paper presents SMART, the first fully integrated cryo-EM platform unifying three critical AI-driven modules: DataSmart (automated data collection), CryoSmart (3D reconstruction), and ModelSmart (end-to-end atomic modeling). The platform embeds more than 10 distinct AI algorithms across its workflow, achieving unprecedented automation levels. These advancements not only lower the expertise barrier for cryo-EM adoption but also establish a new paradigm for AI-empowered structural biology, with broad implications for high-throughput drug discovery. This protocol provides a comprehensive delineation of the operational procedures for the SMART platform. As an example, this protocol enabled the determination of the 2.1 angstrom resolution structure of TRPML1, a Ca2+-permeable, nonselective, six-transmembrane tetrameric cation channel found in late endosomes and lysosomes (LELs) of mammalian cells. This integrated workflow enables users, including those with minimal cryo-EM experience, to efficiently complete the entire pipeline from data collection to atomic model building.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.05.657588v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Uncovering Heterogeneous Effects via Localized Feature Selection</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liu, X., Gu, J., Chen, Z., Chu, B., Liu, L., Morrison, T., Butler, R. R., Edelson, J., Li, J., Longo, F. M., Tang, H., Ionita-laza, I., Sabatti, C., Candes, E., He, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel framework that utilizes localized test statistics to identify disease-associated features tailored to individual profiles, while accounting for heterogeneity across diverse populations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Identifying features that interact to trigger disease, while accounting for heterogeneity across diverse populations, is essential for the development of precision and targeted medicine. Despite the availability of vast and complex health-related datasets, most existing works focus on identifying disease-associated features at the population level or within a few subpopulations, often overlooking individual-level heterogeneity within these groups. To address this limitation, we propose a novel framework that utilizes localized test statistics to identify disease-associated features tailored to individual profiles. Our method leverages the recently developed knockoffs methodology to control the noise level of the selection set so that the results are replicable. Moreover, it allows for the discovery of hidden heterogeneous effects within the data, as demonstrated in an application to single-cell RNA sequencing data for Alzheimer's disease. By aggregating localized feature selection results, our framework also enables powerful population-level feature selection. Our framework provides a powerful tool for exploratory studies of precision medicine, offering the potential to generate novel hypotheses for confirmatory biological experiments.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.03.657761v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Forecasting oncogene amplification and tumour suppressor deletion</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hernando, B., Fernandez-Sanroman, A., Cadiz, A., Santamaria, P. G., Gomez-Sanchez, D., Escobar-Rey, M., Chaves-Urbano, B., Thompson, J. S., Torres, M., Ruiz de Garibay, G., Adradas, V., Alvarez, E., The Pan Prostate Cancer Group,, Tarabichi, M., Lesluyes, T., Coya, J. M., Zugazagoitia, J., Paz-Ares, L., Macintyre, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a forecasting framework that takes as input a DNA copy number profile and predicts whether the tumour will acquire an oncogene amplification or tumour suppressor deletion in the future, alongside gene-specific selection coefficients derived from a large cohort of 7,880 tumours.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Oncogene amplification and tumour suppressor deletion can drive tumour initiation, progression and treatment resistance. Detection at diagnosis often signals poor prognosis, but it can also enable opportunities for treatment with highly effective targeted therapies. Predicting the likelihood that a patient will acquire these driver alterations in the future using a genomic test represents an opportunity to realise the benefits of interventions earlier, potentially with preventative intent. Here, we present a forecasting framework that takes as input a DNA copy number profile and predicts whether the tumour will acquire an oncogene amplification or tumour suppressor deletion in the future. This framework leverages mutation rate estimates from the input tumour, alongside gene-specific selection coefficients derived from a large cohort of 7,880 tumours. We demonstrate feasibility using 7,042 single-time-point samples and longitudinally collected tumour pairs from 44 prostate and 100 lung cancers, identifying tumours that went on to acquire amplifications at a later time point with an average AUC of 0.87. We show potential clinical utility by forecasting poor prognosis in low-grade gliomas via CDK4/PDGFRA amplification or CDKN2A deletion, and osimertinib resistance in lung cancers via MET amplification. This study serves as a proof-of-concept for a new class of biomarker, wherein selective pressures and mutation-generating processes can be harnessed to anticipate future genomic alterations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.06.658212v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">De novo genome assembly of Ansell's mole-rat (Fukomys anselli)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bekavac, M., Coimbra, R., Busa, V. F., Behm, M., Wagner, R. E., Goncalves, A., Begall, S., Frye, M., Odom, D. T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We report the first genome assembly of a male Ansell's mole-rat and lay the foundation for dissecting the genetic and evolutionary basis of its extraordinary phenotypic traits and resolving African mole- rat phylogeny.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Ansell's mole-rat (Fukomys anselli) is an African rodent known for its subterranean lifestyle and unique phenotypic traits, including extreme longevity, magnetoreception, and a cooperative breeding social structure. Efforts to dissect the genetic architecture of these traits and to decipher their phylogenetic relationships within the broader African mole-rat family would greatly benefit from a reference-grade genome. Here, we report a first genome assembly of a male Ansell's mole-rat. By combining Oxford Nanopore Technologies (ONT) long-reads and Illumina short-reads with Hi-C data, we generated a chromosome level assembly with a total length of 2.27 Gb, 412 scaffolds and a scaffold N50 of 72.4 Mb. We identified 99.54% of expected genes and annotated 29,094 transcripts using RNA sequencing data. This high-quality de novo genome of Fukomys anselli lays the foundation for dissecting the genetic and evolutionary basis of its extraordinary traits and resolving African mole-rat phylogeny.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.03.657636v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The Vitis vinifera receptor VvLYK6 negatively regulates chitin-triggered immune responses and promotes fungal infections</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> VILLETTE, J., MARZARI, T., LANDRY, D., ROUDAIRE, T., ROMANET, R., KLINGUER, A., LEBORGNE-CASTEL, N., VICEDO, C., GASCIOLLI, V., POUZET, C., LEFEBVRE, B., HELOIR, M. C., POINSSOT, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified a novel member of the LysM receptor-like kinase family, VvLYK6, as a plant susceptibility gene during Botrytis cinerea infection and showed that it negatively regulates the chitin-triggered immune responses in V. vinifera.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Botrytis cinerea is recognized as one of the most harmful fungal diseases affecting grapevine (Vitis vinifera), directly impacting grape yield and wine quality. Identifying new genes involved in the interaction between V. vinifera and B. cinerea appears to be the most promising strategy for enhancing grapevine resistance to this pathogen in future breeding programs. During pathogen infection, plasma membrane-localized pattern recognition receptors (PRRs) are involved in the perception of conserved microbe-associated molecular patterns (MAMPs). Among PRRs, members of the LysM receptor-like kinase family are well known to mediate recognition of fungal MAMPs to induce the plant immune signaling pathway. Interestingly, a novel member of this receptor family, named VvLYK6, was identified in grapevine as the most upregulated during a Botrytis cinerea infection. Aiming to understand the role of VvLYK6 in plant immunity, we carried out an overexpression study in Arabidopsis thaliana and in grapevine cell suspension. The overexpression of VvLYK6 resulted in a reduction of chitin oligomer-induced MAPKs activation, expression of defense-related genes, a reduced callose deposition and an increased plant susceptibility to three fungal pathogens. At the opposite, the CRISPR-Cas9-mediated vvlyk6 knock-out lines generated in V. vinifera induce the phytoalexin-related stilbene pathway at the basal level. Based on our findings, we concluded that VvLYK6 negatively regulates the chitin-triggered immune responses in V. vinifera, suggesting its potential involvement as a susceptibility gene during fungal infections.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.06.658283v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A DNA part library for reliable engineering of the emerging model nematode symbiotic bacterium Xenorhabdus griffiniae HGB2511</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Larsson, E. M., Wang, O. Y., Murray, R. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We develop and characterize a DNA part library similar to the CIDAR MoClo extension library for E. coli to allow more efficient construction of genetic circuits in X. griffiniae.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Xenorhabdus griffiniae is a bacterium that lives inside the intestine of the entomopathogenic nematode Steinernema hermaphroditum and partners with the nematode to infect and kill insect larvae in soil. The construction of gene circuits, like reporters, in X. griffiniae would provide tools to study and better understand the symbiotic relationship it has with its host. However, because X. griffiniae is not a model organism, information about gene circuit construction in X. griffiniae is limited. We develop and characterize a DNA part library similar to the CIDAR MoClo extension library for E. coli to allow more efficient construction of genetic circuits in X. griffiniae. TurboRFP expressing strains with different constitutive Anderson promoters and different ribosome binding sites (RBS) were constructed to quantify promoter and RBS strengths in X. griffiniae. Furthermore, two fluorescent proteins sfGFP and sfYFP, as well as the bioluminescent luxCDABE operon were added to the part library and successfully expressed in X. griffiniae. We then used the characterized parts to build and characterize IPTG inducible constructs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.09.658710v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Evaluating the Influence of Disease-Gene Associations in the Significance of Disease Modules through the lens of Network Medicine</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gil, A., Prieto-Santamaria, L., Rodriguez-Gonzalez, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We investigate the relationship between disease module size, GDA score, and significance, and propose a number of potential cutoffs for robust module construction.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The rapid expansion of genomic and biomedical data has paved the way for constructing complex disease networks, offering new insights into disease mechanisms and therapeutic target identification. Central to these networks are disease modules, which are constructed from seed genes that are prioritized based on their relevance to a given disease. Gene prioritization aims to rank genes based on their strength of association with a disease. Resources like DisGeNET, integrated within DISNET knowledge base, assigns a Gene-Disease Association (GDA) score that reflects the confidence in the association between a gene and a disease. This study investigates how both disease module sizes and GDA scores influence the statistical significance of these modules. By characterizing disease modules, filtering the data based on their GDA score thresholds, and analysing the relationship between their size, GDA score, and significance, potential cutoffs for robust module construction are estimated. Our findings suggest that disease modules filtered to include GDA scores above 0.3 and module sizes greater than 10 tend to be significant. These insights provide guidance for optimal gene prioritization and module selection ultimately enhancing strategies for target identification and drug repurposing in network medicine.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.05.657982v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Harnessing Immunoinformatic for HIV-2: Multi-Epitope Vaccine Design Targeting the env Protein (CAA28914.1)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ali, M., Arafat, Y., Gull, S. A., Anwar, F., Zainab, S., Abbas, Q.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We design a multi-targeting HIV-2 vaccine using immunoinformatic approaches using Immunostimulatory peptides.</p><p style="font-size: 16px; text-align: justify;">Background: HIV-2 is a major threat to human health because of its rising global incidence. As a result, the development of effective immunostimulatory molecules for vaccine design against HIV-2 has garnered increasing interest. Objective: The main objective was to make a multi Targeting epitoptpic vaccine for HIV-2 envelope protein using immunoinformatic approaches. Methods: Sequence were retrieved from NCBI. B & T-cell epitopes were predicted with assessments of antigenicity, allergenicity, and cytokine-inducing potential. Physicochemical properties, docking studies, and codon optimization for expression was also performed using different software tools. Findings: The designed vaccine construct (477 amino acids, molecular weight of 54 kDa) showed a strong antigenicity with VaxiJen score of 0.7146 which was non-allergenic and non-toxic with favorable physicochemical parameters (Mw: ~53 kDa; pI: 9.89; stable and soluble). The construct demonstrated broad population coverage for MHC-I (West Africa: 69.51%, Pakistan: 78%, Global: 82%) and MHC-II (Global: 99%). Out of 16 peptides analyzed, 7 were predicted as IL-10 inducers 2 as a strong IL-4 inducer, and 2 were IFN-{gamma} inducing epitopes. Structural validation confirmed high-quality folding (96% residues in favored regions; ERRAT score: 84.35). Docking results revealed a strong binding affinity to TLR2 (PDB ID: 2AOZ; energy: -1481.1 kcal/mol). MD simulation confirmed structural stability (RMSD ~1.37 nm; Rg ~3.65 nm). Immune simulation predicted robust humoral and cellular responses with sustained antibody production and cytokine activation. Codon optimization is given to a Codon Adaptation Index (CAI) of 0.628 with a guanine (G) and cytosine (C) nucleotides content of 65.48%, suitable for E. coli expression. Conclusion: This vaccine construct is immunogenic, stable, and able to elicit both antibody-mediated immunity and cell-mediated immunity responses. Furthermore, In-vitro or In-Vivo Validations are recommended to verify its effectiveness.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.06.658249v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">{Psi}-Atlas: an integrated atlas for pseudouridine epitranscriptome</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, X., Luo, J., Lang, X., Liu, Q.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the {Psi}-Atlas, an extensive database meticulously curated for the comprehensive collection and annotation of RNA pseudouridine modifications.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Pseudouridine ({Psi}) is a C5 glycoside isomer of uridine, formed by breaking the N1 glycosyl bond and undergoing a 180{degrees} base rotation. This modification stands as one of the most widespread post-transcriptional alterations in RNA and is universally distributed among diverse RNA species. The pervasiveness of this modification enhances RNA structural integrity, bestows unique structural and functional attributes upon the RNA molecules it adorns, and facilitates additional hydrogen bonding. However, the absence of a convenient and integrated intuitive visualization database encompassing all currently reported species and RNA types is evident. Here, we present the {Psi}-Atlas, an extensive database meticulously curated for the comprehensive collection and annotation of RNA pseudouridine. This database encompasses 554,895 pseudouridine modification sites across various RNA categories, including mRNA, ncRNA, tRNA, rRNA, and others, in 55 distinct species reported in current literature. The sequencing methodologies employed comprise the next-generation sequencing (NGS) techniques such as {Psi}-Seq, Pseudo-Seq, CeU-Seq, PSI-Seq, RBS-Seq, HydraPsiSeq, BID-Seq, PRAISE-Seq, as well as third-generation sequencing methods like Direct RNA sequencing (DRS). {Psi}-Atlas as the most comprehensive and integrated resource for RNA pseudouridine modifications to date. And the {Psi}-Atlas database offers an intuitive interface for information display and a myriad of analytical tools, including PsiVar and PsiFinder. In essence, this platform serves as a robust search and visualization tool for the study of pseudouridylation in epitranscriptomics. {Psi}-Atlas is available at https://rnainformatics.org.cn/PsiAtlas.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.06.656536v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PLIERv2: bigger, better and faster</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Subirana-Granes, M., Nandi, S., Zhang, H., Chikina, M., Pividori, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce PLIERv2, an unsupervised matrix factorization method for biologically informed latent variable extraction that achieves 7x-41x speedups over PLIER, succeeds in modeling hundreds of thousands of samples that PLIER cannot handle, and maintains or improves biological specificity of latent variables as shown</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Gene expression analysis has long been fundamental for elucidating molecular pathways and gene-disease relationships, but traditional single-gene approaches cannot capture the coordinated regulatory networks underlying complex phenotypes; although unsupervised matrix factorization methods (e.g., PCA, NMF) reveal coexpression patterns, they lack the ability to incorporate prior biological knowledge and often struggle with interpretability and technical noise correction. Semi-supervised strategies such as PLIER have improved interpretability by integrating pathway annotations during latent variable extraction, yet the original PLIER implementation is prohibitively slow and memory-intensive, making it impractical for modern large-scale resources like ARCHS4 or recount3. Here, we introduce PLIERv2, which overcomes these constraints through a two-phase algorithmic design (an unsupervised "PLIERbase" initialization followed by a "PLIERfull" regression that incorporates priors via glmnet), rigorous internal cross-validation to tune regularization parameters for each latent variable, and efficient on-disk data handling using memory-mapped matrices from the bigstatsr package. Benchmarking on GTEx, recount2, and ARCHS4 demonstrates that PLIERv2 achieves 7x-41x speedups over PLIERv1, succeeds in modeling hundreds of thousands of samples that PLIERv1 cannot handle, and maintains or improves biological specificity of latent variables as shown by tissue-alignment and pathway enrichment analyses. By filling the gap in scalable, biologically informed latent variable extraction, PLIERv2 enables comprehensive analysis of modern transcriptomic compendia and paves the way for deeper insights into gene regulatory networks and downstream applications in translational genomics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.05.658122v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

