<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Wednesday, 14 May 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Assessing genomic reproducibility of read alignment tools</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Baykal, P. I., Simonov, M., Deshpande, D., Korukoglu, F. B., Moore, J., Chhugani, K., Liu, C., Sarwal, V., Rajkumar, N., Alser, M., Beerenwinkel, N., Mangul, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We evaluate the reproducibility of eight alignment tools (BWA-MEM22, Bowtie23, HISAT24, minimap25, NextGenMap6, SNAP7, SMALT7,8 and Subread9) under shuffling, reverse complementing, or combined shuff</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genomic research relies on accurate and reproducible computational analyses of DNA sequencing data to draw reliable biological conclusions. Read mapping, the process of aligning reads to a reference genome, is central to many applications, including variant detection and comparative genomics. While several tools have been developed for this task, genomic reproducibility1, defined as the consistency of results across replicates, remains underexplored. Here, we address this question by introducing a methodology based on synthetic replicates of sequencing data, generated by perturbing the original reads through shuffling, reverse complementing, or combined shuffling and reverse complementing. Our approach is able to simulate variability observed across sequencing runs due to differences in library preparation techniques. We evaluated the reproducibility of eight alignment tools (BWA-MEM22, Bowtie23, HISAT24, minimap25, NextGenMap6, SNAP7, SMALT7,8 and Subread9) under these perturbations using whole-genome sequencing (WGS) data. Synthetic replicates were aligned and compared to the original sample to quantify discrepancies. Mapping accuracy changes ranged from 0.0001% to 4.4% for primary reads, which are alignments not marked as secondary, supplementary, or duplicates, and up to 12.2% for high-quality primary reads. For primary reads, the percentage of reads commonly mapped in both the original and synthetic replicate ranged from 91.66% to 100% relative to the total number of mapped reads in the original dataset. High-quality filtering improved consistency, though some tools still failed to recover more than 70% of the original alignments. Within the set of common reads, the incidence of inconsistent mappings was as high as 13.53% for primary reads and 6.73% for high-quality primary reads. Bowtie 2 was fully reproducible under the shuffling replicate, Subread was fully reproducible under the reverse-complement replicate, whereas NextGenMap exhibited only minor inconsistencies. By contrast, SNAP and minimap2 displayed the most significant variability under reverse complementing. We further demonstrate that alignment inconsistencies propagate to the downstream task of calling structural genomic variants. Using Manta for structural-variant (SV) calling, we observed that Bowtie2, HISAT2, and minimap2 maintained perfect SV concordance between original and replicate alignments, whereas other tools exhibited lower concordance, with Subread scoring only 87% concordance. In conclusion, our comprehensive evaluation demonstrates that synthetic perturbations reveal critical differences in how alignment tools handle technical variability and how these differences propagate to downstream variant analyses, underscoring the necessity of incorporating reproducibility benchmarks into the selection and validation of read mappers to ensure robust and reliable genomic interpretations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.08.652934v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">RAMEN: Dissecting individual, additive and interactive gene-environment contributions to DNA methylome variability in cord blood</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Navarro-Delgado, E. I., Czamara, D., Edwards, K., Fu, M. P., Merrill, S. M., Konwar, C., MacIsaac, J. L., Lin, D. T., Mandhane, P. J., Simons, E., Subbarao, P., Moraes, T. J., Lahti, J., Miller, G. E., Binder, E. B., Räikkönen, K., Turvey, S., Korthauer, K., Kobor, M. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce RAMEN, a Findable, Accessible, Interoperable, and Reusable (FAIR) framework tailored for DNAme microarrays, using machine learning and statistical techniques, using gene-environment contributions to genome-wide Variably Methylated Regions (VMRs)</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> DNA methylation (DNAme) is the most commonly studied epigenetic mark in human populations. DNAme has gained attention in the Developmental Origins of Health and Disease field due to its gene expression regulation and potential long-term stability. Genetic variation and environmental exposures are amongst the main factors influencing inter-individual DNAme variability. However, the proportion and genomic distribution of their individual, additive and interactive effects on the DNA methylome remains unclear. Here, we introduce RAMEN, a Findable, Accessible, Interoperable, and Reusable (FAIR) framework tailored for DNAme microarrays. Using machine learning and statistical techniques, RAMEN models and dissects gene-environment contributions to genome-wide Variably Methylated Regions (VMRs), while controlling for spurious associations. To comprehensively test the power of RAMEN, we analyzed and characterized VMRs from cord blood samples from two independent cohorts (CHILD and PREDO; overall n=1,662). We identified genetics as a consistent key contributor to DNAme variability, usually in additive and interactive combinations with the environment, with genetic terms explaining the largest proportion of DNAme variance, compared to environmental and interaction terms. Operationalizing RAMEN as an R package to conduct scalable genome-exposome contribution analyses, our results highlighted the importance of genetic variation in sculpting DNAme patterns in early life.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.08.652964v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Automating AI Discovery for Biomedicine Through Knowledge Graphs And LLM Agents</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Aamer, N., Asim, M. N., Munir, S., Dengel, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel framework that employs a novel strategy to discover the most relevant pathway between any two biomedical entities in a knowledge graph, which is then leveraged by a multi-agent AI system to extract supporting facts from literature and design AI predictors for understanding the discovered pathway.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The biomedical domains accelerating progress in understanding, early detection, and treatment of diseases has led to an exponentially growing and overwhelming body of literature. Researchers typically rely on this literature to find information relevant to their discoveries, using it to validate their findings. And despite the emergence of LLM systems, retrieving precise, relevant facts from the literature remains a significant challenge. The primary objective of this paper is to expedite biomedical discoveries by the integration of a biomedical knowledge graph with the scientific literature embedded in a Large Language Model. This work introduces a framework that employs a novel strategy to discover the most relevant pathway between any two biomedical entities in a knowledge graph, which is then leveraged by a multi-agent AI system to extract supporting facts from literature and design AI predictors for understanding the discovered pathway. Experiments with various biomedical entity pairs demonstrate the frameworks ability to identify highly relevant pathways and design plausible and complex AI predictors across diverse therapeutic areas. In addition, we developed Intelliscope, a web-based dashboard that makes this framework available to researchers worldwide. This first-of-its-kind platform could significantly accelerate scientific discoveries, potentially leading to breakthroughs in disease understanding, drug repurposing, and therapeutic development.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.08.652829v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Magnetically controlled tension of cytoskeletal elements with magnetic nanoparticles affects the expression of signaling pathway genes associated with cytoskeletal elements</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pozdina, V., Karavashkova, O., Minin, A. S., Zubarev, I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified and described biological pathways involved in the regulation of mechanical deformations of cytoskeletal elements and their role in intracellular signaling.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Interactions with the extracellular environment and biological responses are based on biochemical pathways. Cytoskeletal reorganization is a dynamic process accompanied by filament polymerization and depolymerization, which allows the cell to effectively perceive and respond to external mechanical stimuli by changing biomechanical properties. The contribution of mechanical deformations of individual cytoskeletal elements to changes in intracellular signaling has not been covered in the literature to date. In our article, we investigated changes in gene expression after regulated cytoskeletal deformation using a constant magnetic field and magnetic nanoparticles associated with antibodies to cytoskeletal proteins (vimentin, beta-actin and acetylated tubulin). For the first time, we identified and described biological pathways involved in the regulation of mechanical deformations of cytoskeletal elements.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.08.652598v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">POPARI: Modeling multisample variation in spatial transcriptomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Alam, S., Zhou, T., Haber, E., Chidester, B., Liu, S., Chen, F., Ma, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a probabilistic graphical model for factor-based decomposition of multisample SRT that captures condition-specific changes in spatial organization and reveals sample-specific malignant-immune interactions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Integrating spatially-resolved transcriptomics (SRT) across biological samples is essential for understanding dynamic changes in tissue architecture and cell-cell interactions in situ. While tools exist for multisample single-cell RNA-seq, methods tailored to multisample SRT remain limited. Here, we introduce POPARI, a probabilistic graphical model for factor-based decomposition of multisample SRT that captures condition-specific changes in spatial organization. POPARI jointly learns spatial metagenes - linear gene expression programs - and their spatial affinities across samples. Its key innovations include a differential prior to regularize spatial accordance and spatial downsampling to enable multi-resolution, hierarchical analysis. Simulations show POPARI outperforms existing methods on multisample and multiresolution spatial metrics. Applications to real datasets uncover spatial metagene dynamics, spatial accordance, and cell identities. In mouse brain (STARmap PLUS), POPARI identifies spatial metagenes linked to AD; in thymus (Slide-TCR-seq), it captures increasing colocalization of V(D)J recombination and T cell proliferation; and in ovarian cancer (CosMx), it reveals sample-specific malignant-immune interactions. Overall, POPARI provides a general, interpretable framework for analyzing variation in multisample SRT.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.08.652741v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Toward Reliable Synthetic Omics: Statistical Distances for Generative Models Evaluation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Marchesi, R., Lazzaro, N., Leonardi, G., Rignanese, F., Bovo, S., Chierici, M., Jurman, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We validate generative networks for transcriptomics data generation by using statistical distances as robust evaluation metrics, which enable simultaneous evaluation of global and local data fidelity.</p><p style="font-size: 16px; text-align: justify;">Background. Synthetic data generation is emerging as an approach to overcome the limitations of real-world data scarcity in omics studies, especially in precision medicine and oncology. Omics datasets, with their high dimensionality and relatively small sample sizes, often lead to overfitting, especially in deep learning models. Generative models offer a promising way to generate realistic synthetic data preserving the original data distribution. However, there is still no objective consensus on how to evaluate their performance. In this study, we set out to validate generative networks for transcriptomics data generation by using statistical distances as robust evaluation metrics. Results. We observe that statistical distances enable simultaneous evaluation of global and local data fidelity of generated synthetic data. Because these distances satisfy the properties of true metrics, they also enable formal hypothesis testing to assess whether generative models have in fact converged or are merely approaching the reference distribution. Crucially, optimizing for these distances was found to implicitly select models maximizing other widely used metrics of generative performance, providing evidence of their broad applicability. Overall, our findings indicate that the adoption of these metrics can play a key role in guiding the development of generative models across a wide range of domains.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.08.652855v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">New gene annotations for three pea aphid genome assemblies allow comparative analyses of genes and gene family evolution</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Deem, K. D., Brisson, J. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We re-annotate the pea aphid reference genome, along with two long read long read assemblies, to facilitate future analyses of gene duplication and function in the model insect and a new comparative framework for discovery and analysis of gene function and duplication in this emerging model insect.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Reliable genome annotation is crucial for analyses of gene function, conservation, and evolution. Factors such as the sequencing technology used to create the assembly and the amount of duplicated sequence within the genome of interest can have a large impact on the quality of gene annotations. In particular, short read-based assemblies tend to mis-assemble duplicated genes as single loci, a problem that requires additional long read sequencing to resolve. Pea aphids exhibit a high level of gene duplication, resulting in mis-assembly and mis-annotation of genes in the short read reference genome. Here, we re-annotate the pea aphid reference genome, along with two long read pea aphid genomes, to facilitate future analyses of gene duplication and function in pea aphids. We use an integrated approach, consolidating both ab initio and RNAseq-based annotations into unified gene models. The new annotations contain genes that were missing, mis-annotated, or mis-assembled in the reference, and are generally consistent across assemblies, showing very good agreement between the long read assemblies. Our annotation method is sensitive enough to refine existing gene models, uncovering alternatively used promoters and isoforms, and aids in finding gene duplications. These data provide a useful supplement to the existing reference annotations and a new comparative framework for discovery and analysis of gene function and duplication in this important emerging model insect.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.08.652899v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The mouse pangenome reveals the structural complexity of the murine protein coding landscape</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Helmy, M., Li, J. U., Yan, X. F., Meade, R. K., Anderson, E., Chen, P., Czechanski, A. M., Di Domenico, T., Flint, J., Garrison, E., Guarracino, A., Gontijo, M. T., Haggerty, L., Heard, E., Howe, K., Meena, N., Martin, F. J., Miska, E., Rall, I., Ramakrishna, N. B., Sapetschnig, A., Sinha, S., Sun, D., Tricomi, F. F., Qu, R., Wood, J. M., Wu, T., Zhou, D. J., Reinholdt, L., Adams, D. J., Smith, C. M., Lilue, J., Keane, T. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the first mouse pangenome consisting of 17 high-quality inbred mouse strain genomes with complete annotation, including previously incomplete genomic regions, including the major histocompatibility complex (MHC), the defensin cluster, T-cell receptor, and Ly49 complexes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We present the first mouse pangenome consisting of 17 high-quality inbred mouse strain genomes with complete annotation. This collection includes 12 widely used classical laboratory strains and 5 wild-derived strains. We have fully resolved previously incomplete genomic regions, including the major histocompatibility complex (MHC), the defensin cluster, T-cell receptor, and Ly49 complexes. Hundreds of non-reference genes identified in previous publications not found in GRCm39, like Defa1, Raet1a, and Klra20 (Ly49T), were localised in the new reference genomes. We conducted the first genome-wide scan of variable number tandem repeats (VNTRs) within the coding regions of mice, identifying over 400 genes with VNTR polymorphisms up to more than 600 repeat copies and repeat units reaching 990 nucleotides. Our strain-specific annotations enhance RNA-Seq analyses, as demonstrated in PWK/PhJ, where we observed a 5.1% improvement in read mapping and expression level differences in 2.1% of coding genes compared to using GRCm39.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.13.653481v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Generative model of SARS-CoV-2 variants under functional and immune pressure unveils viral escape potential and antibody resilience</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Huot, M., Rosenbaum, P., Planchais, C., Mouquet, H., Monasson, R., Cocco, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a data-driven model that quantifies the impact of antibody pressure on RBD evolution and assesses antibody resilience beyond single mutations and known variants.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The evolutionary trajectory of SARS-CoV-2 variants is shaped by the selective pressures exerted by host immunity, in particular neutralizing antibodies targeting the receptor-binding domain (RBD). Here, we introduce a data-driven model that quantifies the impact of antibody pressure on RBD evolution and assesses antibody resilience beyond single mutations and known variants. We integrate deep mutational scanning of ACE2 and 31 antibodies with a generative model trained on pre-pandemic Coronaviridae sequences. We then use our generative model to design viable RBD variants under immune pressure from four monoclonal antibodies: SA55, S2E12, S309, and VIR-7229. Experimental validation of 22 variants, with up to 21 mutations from Wuhan wild-type, confirms 50% expression rate. Binding assays further reveal that S309 and VIR-7229 maintain binding across diverse mutational combinations, while SA55 is escaped by one variant and S2E12 exhibits lower resilience. In addition, our model captures correlated escape across antibody pairs (R = 0.65), guiding the selection of negatively correlated cocktails to reduce shared escape routes. By quantitatively linking viral adaptation to antibody resistance profiles, this framework provides a predictive foundation for optimizing therapeutic strategies and enhancing long-term pandemic preparedness.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.12.653592v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A generative language model decodes contextual constraints on codon choice for mRNA design</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Faizi, M., Sakharova, H., Lareau, L. F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present Trias, an encoder-decoder language model trained on the coding regions of millions of eukaryotic mRNA sequences that learns codon usage rules directly from sequence data, integrating local and global dependencies to generate species-specific codon sequences that align with biological constraints.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The genetic code allows multiple synonymous codons to encode the same amino acid, creating a vast sequence space for protein-coding regions. Codon choice can have dramatic implications on mRNA function and protein output, and the impact of these choices has become newly relevant with advances in mRNA technology for vaccines and therapeutics. Genomes have evolved to prefer some codons over others, but simple methods for codon optimization that select the most preferred codons fail to capture complex contextual patterns in natural sequences. To enable biologically informed design of synthetic coding sequences, we present Trias, an encoder-decoder language model trained on the coding regions of millions of eukaryotic mRNA sequences. Trias learns codon usage rules directly from sequence data, integrating local and global dependencies to generate species-specific codon sequences that align with biological constraints. With no explicit training on protein expression, Trias can generate sequences and compute scores that correlate strongly with experimental measurements of mRNA stability, ribosome load, and protein output. The model outperforms commercial codon optimization tools in generating sequences resembling high-expression codon sequence variants. By capturing natural constraints on codon usage within sequence context, Trias provides a powerful, data-driven framework for synthetic mRNA design and for understanding the molecular and evolutionary principles governing codon choice.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.13.653614v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">GnnDebugger: GNN based error correction in De Bruijn Graphs</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Simunovic, M., Sikic, M., Bankevich, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a method for correcting errors in De Bruijn Graphs using Graph Neural Networks and show that it can be used to improve the accuracy of genome assembly.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Modern sequencing technologies have enabled the reconstruction of complete mammalian genomes from telomere to telomere. However, scaling this achievement to thousands of species and population-level studies remains a challenge. Key bottlenecks include the low quality of the draft assemblies and the high coverage requirements. In particular, reconstructing complete and accurate sequences of both haplotypes in diploid genomes is especially difficult since the sequencing depth is not always sufficient to properly reconstruct diverged regions. Inspired by the success of neural networks in extracting patterns from the data on a massive scale, we introduce a method for correcting errors in De Bruijn Graphs using Graph Neural Networks. Our model provides a reliable classification of edges into correct and erroneous, especially for diploid genomes with coverage depth 35 and lower. We demonstrate that these predictions can guide the downstream read error correction algorithm and genome assembly, ultimately allowing for more accurate genome assembly.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.07.652713v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Explainable Multimodal Machine Learning Using Combined Environmental DNA and Biogeographic Features for Ecosystem Biomonitoring</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Koh, J. C. O.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A multimodal ML workflow using combined eDNA and biogeographic features for ecosystem biomonitoring.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Machine learning (ML) has been proposed as a taxonomy-independent approach using environmental DNA (eDNA) for ecosystem biomonitoring. Representations of eDNA amplicons either as clustered sequences termed operational taxonomic units (OTUs) or unique sequences termed amplicon sequence variants (ASVs) are used as inputs in current ML practices, with varied successes reported. Biogeographic data encompassing the physical, climate and ecological observations collected at biomonitoring sites provide a repository of potential informative features that can augment ML performance in combination with eDNA data. I introduce a multimodal ML workflow using combined eDNA and biogeographic features for ecosystem biomonitoring. Differentially abundant ASVs were merged with biogeographic data and used as input in an automated ML approach. Using Switzerland's freshwater macroinvertebrate eDNA dataset collected across 163 biomonitoring sites and impact prediction as an example, I show that the multimodal ML approach (83.3% accuracy) significantly outperformed ML using only ASVs (66.7% accuracy) or OTUs (64.6% accuracy). Shapley additive explanation of the best ML model revealed key biogeographic features and species/taxa impacting upon predictions. The proposed workflow can be readily adopted in existing bioinformatics/ML pipelines and will further advance the use of eDNA for biomonitoring.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.07.652781v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Locus-specific Convergent Evolution and Interchromosomal Rearrangements Contribute Type I Interferon Diversification in Amniotes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, L., Ma, F., Liu, J., Yu, Y., Ma, J., Zhang, L., Li, B., Li, C., Liu, P., Zhang, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified three evolutionarily conserved IFN-I loci (HACD4, MOB3B, and UBAP2) that maintain chromosomal colocalization across all major amniote lineages - mammals, squamates, and Archelosauria.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Type I interferons (IFN-Is) play essential roles in antiviral immune responses. The extensive diversification of IFN-Is into multiple subtypes and redundant gene copies has posed significant challenges for evolutionary reconstruction. To address this, we developed an integrated analytical pipeline combining the IFN-SCOPE classification model and GENE-GRADE algorithm. Through comprehensive synteny-guided phylogenetic analysis, we identified three evolutionarily conserved IFN-I loci (HACD4, MOB3B, and UBAP2) that maintain chromosomal colocalization across all major amniote lineages - mammals, squamates, and Archelosauria (turtles, crocodilians, and birds). While MOB3B-locus maintains a single IFN-{kappa} ortholog, the HACD4 (IFN-H) and UBAP2 (IFN-U) loci show lineage-specific expansion patterns: IFN-H proliferated in mammals/reptiles but remained single-copy in birds, whereas IFN-U expanded in birds but not in other lineages. Phylogenetic analysis reveals these independently evolved multicopy genes nevertheless cluster into two conserved subgroups (IFN-H2/H1 and IFN-U2/U1), suggesting convergent functional specialization. Within IFN-H and IFN-U clusters, the single-copy IFN-H2 and IFN-U2 genes - positioned at the ancestral ends of their respective genomic arrays - likely represent the progenitor sequences of each locus. Notably, among mammalian IFN-H2 genes, we identified the poorly characterized IFN-{upsilon} (rather than IFN-{beta}) as the ancestral form of mammalian IFN-H subtypes. Furthermore, we identified IFN-I genes at non-canonical loci resulting from interchromosomal duplication events in tortoises and diving ducks and provide clear evidence that interchromosomal duplications contributed to IFN-I gene diversity. These discoveries advance our understanding of the evolutionary mechanisms shaping intronless IFN-I genes in amniotes, and potentially beneficial for developing novel IFN-I based antiviral treatments through comparative immunological approaches.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.08.652797v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

