<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Sunday, 03 August 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">In-Silico Evaluation of Silk Fibroin-Conjugated Doxycycline for Sustained Drug Delivery</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bokara, K. K. K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We investigated the conjugation of Doxycycline with silk fibroin to achieve sustained drug release at the target site and found that Dxycycline has the strongest binding affinity compared to Tetracycline and Minocycline.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Tetracyclines such as Doxycycline and Minocycline are broad-spectrum antibiotics. Their high oral bioavailability aids systemic absorption but limits achieving sustained local drug levels at the infection site. Sustained drug release to enhance the therapeutic efficacy and minimise side effects remains a significant challenge. This study investigated the conjugation of Doxycycline with silk fibroin (SF), which would provide sustained drug release at the target site. We employed an in-silico approach using molecular docking and molecular dynamics (MD) simulations to investigate the interaction and stability of antibiotics with SF. We also assessed the structural stability of SF when conjugated with these antibiotics. Molecular level characterization using a blind docking approach and MD simulation revealed that Doxycycline exhibited the strongest binding affinity of -6.0 kcal/mol, compared with both Tetracycline (-5.9 kcal/mol) and Minocycline (-5.7 kcal/mol). The MMPBSA analysis also showed that Doxycycline exhibited the highest binding energy of -25.71 kcal/mol compared to Tetracycline (-18.27 kcal/mol) and Minocycline (-10.8 kcal/mol). The strong binding energy and stability of Doxycycline with SF was attributed mainly by van der Waals interactions and hydrogen bonds. To evaluate the stability of protein-antibiotic complexes, 100 ns MD simulations were performed. The RMSD, RMSF, radius of gyration, SASA, and DSSP analyses revealed that SF was stable without much deformation upon docking of all three antibiotics. In summary, our computational results suggest that Doxycycline and SF conjugates offer a promising strategy for sustained drug delivery, potentially improving therapeutic efficacy in the treatment of chronic inflammatory diseases.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.31.668034v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Early-life Exposure to Arsenic Primes the Offspring to Increased Asthma Risk: Transcriptome and Epigenome Analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sun, Y., Watfa, M., Sun, Q., Rajasundaram, D., Schlegel, B., Yeung-Luk, B., Pulczinski, J., Park, B., Barchowsky, A., Mitzner, W., Biswal, S., Tang, W. Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-03
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Maternal exposure to iAs in drinking water increases asthma risk via epigenetic reprogramming and may inform risk assessment and biomarker development.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Inorganic arsenic (iAs) in drinking water is a global health concern. This study tests whether maternal exposure to iAs in drinking water at the WHO provisional level (10 microgram/L) increases offspring asthma risk via epigenetic reprogramming. F1 mice prenatally exposed to iAs were analyzed at 5 months for blood transcriptome and methylome changes and challenged with house allergens before lung function testing. Prenatal iAs exposure led to increased airway hyperresponsiveness (AHR) and altered inflammation gene expression and DNA methylation changes. Notably, miR-101c was epigenetically reprogrammed early in development, with persistent downregulation in both target (fetal and adult lungs) and surrogate (amniotic fluid and blood) tissues. These changes correlated with increased allergic AHR and TGF{beta} pathway dysregulation. Findings suggest that maternal iAs exposure primes offspring for asthma risk through epigenetic alterations and may inform risk assessment and biomarker development in affected communities.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.01.668223v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Adaptive trade-offs between niche-driven defence system selection and horizontal gene transfer suggests clinical success in Acinetobacter spp.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Muthuraman, V., Roy, P., Dean, P., Lopes, B. S., Shehreen, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comparative genomic analysis of 132 genomes from 18 Acinetobacter species reveals pronounced differences in defence system repertoires between lineages, and the presence of mobile elements, antibiotics, and heavy metal resistance.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Introduction: Bacteria face relentless selective pressure from both antibiotics and bacteriophages, driving the evolution of diverse defence systems and shaping horizontal gene transfer (HGT) dynamics. Gap Statement: The relationship between bacterial defence system repertoires and horizontal gene transfer in clinically relevant Acinetobacter species remains poorly understood, limiting our ability to predict resistance evolution and design targeted phage therapies. Aim: To characterize defence arsenal across Acinetobacter species and investigate their associations with different HGT markers Methodology: We performed a comparative genomic analysis of 132 genomes from 18 Acinetobacter species, focusing on the interplay between defence architectures and HGT markers. Results: Our result reveals pronounced differences in defence system repertoires between lineages: while most Acinetobacter spp. harbour a multilayered arsenal, the clinically dominant A. baumannii international clone 2 (IC2) exhibits a depleted but specialised defence landscape, notably enriched in the phosphorothioation-based SspBCDE system and nearly devoid of restriction-modification systems. Strikingly, many defence systems display mutual exclusivity, and defence system is tightly linked to the presence of mobile elements, antibiotics, and heavy metal resistance. Plasmid-borne defence systems, especially BREX, are prevalent, highlighting the role of mobile elements in distributing both immunity and resistance traits. These patterns suggest that successful clinical lineages finely balance immune defences and genetic plasticity, facilitating rapid adaptation under antimicrobial selection and phage threats. Conclusion: These findings provide insights into evolutionary trade-offs underpinning multidrug resistance and have implications for designing targeted phage therapies against recalcitrant Acinetobacter infections.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.01.668115v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ET-Pfam: Ensemble transfer learning for protein family prediction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Escudero, S., Duarte, S., Vitale, R., Fenoy, E., Bugnon, L. A., Milone, D. H., Stegmayer, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel approach based on transfer learning and ensembles of multiple DL classifiers to predict functional families in the Pfam database.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: Due to the rapid growth of sequence generation, which has surpassed the expert curators ability to manually review and annotate them, the computational annotation of proteins remains a significant challenge in bioinformatics nowadays. The Pfam database contains a large collection of proteins that are nowadays annotated with domain families through multiple sequence alignments and profile Hidden Markov models (pHMMs). However, such computational annotation methods have some limitations such as problems for handling large datasets and the fact that multiple sequence alignments are computationally challenging to compute with high accuracy due to the increase in complexity as the number of sequences and lengths grow. Additionally, each HMM is independently obtained for each family missing the opportunity of learning patterns across families, that is from a complete view of all the dataset. As an alternative, some deep learning (DL) models have been recently proposed, nevertheless with simple representations of the inputs and moderate improvements in performance. Results: In this work we present ET-Pfam, a novel approach based on transfer learning and ensembles of multiple DL classifiers to predict functional families in the Pfam database. Several base DL models are first trained using learned representations from a protein large language model, with different hyperparameters to increase diversity. Then, the base models are integrated using classical ensemble strategies and novel voting approaches by learning weights for each model and for each Pfam family. Results demonstrate that the proposed ET-Pfam method can consistently diminish classification error rates compared to individual DL models and to pHMM models, boosting prediction performance. Among the strategies presented here, the learned weights by family voting achieved the best performance, with the lowest error rate (7.00%), significantly surpassing the best individual model error (12.91%) and the state-of-the-art pHMM error (29.28%) on the same Pfam dataset.Availability: Data and source code are available at https://github.com/sinc-lab/ET-Pfam.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.02.668111v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">spOT-NMF: Optimal Transport-Based Matrix Factorization for Accurate Deconvolution of Spatial Transcriptomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Abdelkareem, A. O., Manoharan, V. T., Gill, G. S., Verhey, T. B., Morrissy, A. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Spatial transcriptomics with Wasserstein distance-based non-negative matrix factorization method for unsupervised deconvolution .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Spatial transcriptomics technologies advance our understanding of complex biology by directly profiling cellular organization within tissues. However, accurate deconvolution of cell types and functional states remains challenging as most current computational methods either rely on high-quality matched single-cell reference profiles (often lacking for many tissues or disease states), struggle across spatial resolutions and under variable sequencing depths, and face scalability bottlenecks in large datasets. To address these challenges, we developed an optimal transport-based non-negative matrix factorization method (spOT-NMF) that leverages the Wasserstein distance to disentangle mixed gene expression signals in a reference-free manner. Benchmarking against well-established unsupervised deconvolution approaches demonstrates top performance of spOT-NMF in simulated and real spatial transcriptomics data spanning sub-cellular to multi-cellular resolutions, across multiple platforms, in two-species admixture scenarios such as xenografts, and in human cancer. We provide spOT-NMF as a freely available package for spatial data analysis, supporting GPU acceleration for large-scale analyses.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.02.668292v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Aggregation-Prone Region Mapping in Olfactomedin Domain of Myocilin through Classical and Enhanced Sampling Molecular Dynamics Simulations</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sardag, I., Duvenci, Z. S., Timucin, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the aggregation of the myocilin olfactomedin domain is not random but initiates at specific structural regions pertinent to the terminal blade A and E.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The aggregation of the myocilin olfactomedin (OLF) domain, generally driven by genetic mutations, is the leading cause of primary open-angle glaucoma (POAG). Developing therapeutic strategies requires a detailed understanding its initial unfolding events that expose aggregation-prone regions (APRs). However, it has been a challenge, as the slow conformational dynamics of OLF hinders classical molecular dynamics (MD) simulations from capturing aggregation-prone OLF intermediates. To overcome this, we employed a multi-pronged computational strategy, integrating over 15 s of simulation time across diverse conditions, including high-temperature, enhanced sampling, chemical denaturation, and simulations of the pathogenic I499F mutant. Our results reveal that OLF unfolding is not random but initiates at specific structural regions pertinent to the terminal blade A and E. Specifically, the blade interfaces between A-B and A-E showed unique regions rich in aromatic/hydrophobic residues as aggregation hotspots. Overall, our simulations proved effective to generate a detailed map of seven distinct APRs. The accuracy of these APRs is partially validated by the close localization of these predicted regions with both previously identified amyloid peptides and the sites of known disease-causing mutations. By scrutinizing the OLF structure and dynamics under different MD settings, our study provides potential molecular targets for developing new therapeutic interventions against POAG.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.02.668245v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Interpretable spatial multi-omics data integration and dimension reduction with SpaMV</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liu, Y., Ma, K., Xu, H., XU, K., Hu, Y., Lin, Z., Lin, J., Han, B., Li, S., Lin, Z., Zhou, X. M., Zhang, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the Spatial Multi-View (SpaMV) representation learning algorithm, which captures both the shared information across modalities and the distinct, modality-specific information, enabling a more comprehensive and interpretable representation of spatial multi-omics data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Spatial multi-omics technologies have revolutionized our understanding of biological systems by providing spatially resolved molecular profiles from multiple perspectives. Existing spatial multi-omics integration methods often assume that data from different modalities share a common underlying distribution, aiming to project them into a single unified latent space. This assumption, however, can obscure the unique insights offered by each modality, thereby limiting the full potential of multi-omics analyses. To address this limitation, we present the Spatial Multi-View (SpaMV) representation learning algorithm, which captures both the shared information across modalities and the distinct, modality-specific information, enabling a more comprehensive and interpretable representation of spatial multi-omics data. Through extensive evaluation on both simulated and real-world datasets, SpaMV demonstrates superior spatial domain clustering performance and provides users with more interpretable dimension reduction for downstream analysis. Moreover, SpaMV effectively annotates cell types within clusters of a mouse thymus dataset, highlighting its effectiveness in interpretable dimensionality reduction.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.02.668264v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">HPInet: Interpretable prediction of Host-Pathogen protein-protein Interactions using a transformer-based neural network</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hu, Y., Liu, L., Zhu, Y., Liu, E., chao, h., Li, S., Feng, C., Hu, Y., Chen, Y., Zhang, S., Chen, Y., Xie, L., Wang, Y., Chen, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a transformer-based deep learning model for predicting interactions between bacterial effector proteins and human host proteins.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Gram-negative bacteria utilize a series of secretion systems (T1SS-T10SS) to deliver secreted effector proteins (T1SE-T10SE) into host cells, leading to infections and diseases. Understanding the interactions between these effector proteins and host proteins is crucial for unraveling the pathogenic mechanisms of bacterial pathogens. Despite advancements in sequencing technologies that have significantly enhanced our knowledge of effector protein diversity and structure, the molecular mechanisms underlying their interactions with human host proteins remain unclear. Experimental approaches to validate these interactions are labor-intensive, time-consuming, and insufficient to comprehensively map the extensive effector-host protein interaction (PPI) network. Additionally, specialized computational tools for predicting these interactions remain scarce. To address this gap, we developed HPInet, a transformer-based deep learning model for predicting interactions between bacterial effector proteins and human host proteins. HPInet introduces convolutional layers and global response normalization (GRN) into the feed-forward network (FFN) of the transformer architecture, enhancing the extraction of local sequence features. Compared to existing state-of-the-art methods, HPInet significantly improves prediction performance, increasing accuracy from 0.502 to 0.891 and boosting sensitivity by 35.3%. Furthermore, by leveraging the model's attention mechanism, HPInet identifies critical residues at interaction sites, demonstrating its capability to capture local structural features of PPI sites solely from sequence information. To facilitate its application in research, HPInet has been implemented as a freely accessible web server available at https://bis.zju.edu.cn/hpinet. This platform provides a powerful and interpretable tool for studying effector-host protein interactions, offering valuable insights into bacterial pathogenesis and potential therapeutic targets.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.02.668281v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Federated Knowledge Retrieval Elevates Large Language Model Performance on Biomedical Benchmarks</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Joy, J., Su, A. I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Federated knowledge retrieval for large language models .</p><p style="font-size: 16px; text-align: justify;">Background Large language models (LLMs) have significantly advanced natural language processing in biomedical research, however, their reliance on implicit, statistical representations often results in factual inaccuracies or hallucinations, posing significant concerns in high stakes biomedical contexts. Results To overcome these limitations, we developed BTE RAG, a retrieval augmented generation framework that integrates the reasoning capabilities of advanced language models with explicit mechanistic evidence sourced from BioThings Explorer, an API federation of more than sixty authoritative biomedical knowledge sources. We systematically evaluated BTE RAG in comparison to traditional LLM only methods across three benchmark datasets that we created from DrugMechDB. These datasets specifically targeted gene centric mechanisms (798 questions), metabolite effects (201 questions), and drug biological process relationships (842 questions). On the gene centric task, BTE RAG increased accuracy from 51% to 75.8% for GPT4o mini and from 69.8% to 78.6% for GPT4o. In metabolite focused questions, the proportion of responses with cosine similarity scores of at least 0.90 rose by 82% for GPT4o mini and 77% for GPT4o. While overall accuracy was consistent in the drug biological process benchmark, the retrieval method enhanced response concordance, producing a greater than 10% increase in high agreement answers (from 129 to 144) using GPT4o. Conclusion Federated knowledge retrieval provides transparent improvements in accuracy for large language models, establishing BTE RAG as a valuable and practical tool for mechanistic exploration and translational biomedical research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.01.668022v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Measuring Dependencies between Biological Signals with Temporal Self-supervision, and its Limitations</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sariyanidi, E., Herrington, J. D., Yankowitz, L., Chaudhari, P., Satterthwaite, T. D., Zampella, C. J., Schultz, R. T., Shinohara, R. T., Tunc, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a self-supervised approach, concurrence, which is inspired by the observation that if two signals are dependent, then one should be able to distinguish between temporally aligned vs. misaligned segments extracted from them.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Measuring the statistical dependence between observed signals is a primary tool for scientific discovery. However, biological systems often exhibit complex non-linear interactions that currently cannot be captured without a priori knowledge regarding the nature of dependence. We introduce a self-supervised approach, concurrence, which is inspired by the observation that if two signals are dependent, then one should be able to distinguish between temporally aligned vs. misaligned segments extracted from them. Experiments with fMRI, physiological and behavioral signals show that, to our knowledge, concurrence is the first approach that can expose relationships across such a wide spectrum of signals and extract scientifically relevant differences without ad-hoc parameter tuning or reliance on a priori information, providing a potent tool for scientific discoveries across fields. However, dependencies caused by extraneous factors remain an open problem, thus researchers should validate that exposed relationships truly pertain to the question(s) of interest.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.02.668286v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Resolving cofactor imbalance in triacylglycerol biosynthesis in oilseeds through glycolytic shunts: a modeling study</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Schwender, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A minimal stoichiometric model for the carbohydrate-to-TAG conversion that satisfies complete cofactor balancing and cofactor imbalance.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> During seed development, carbohydrates are rapidly converted into triacylglycerols (TAGs), with glycolysis and the oxidative pentose phosphate pathway (OPPP) traditionally considered key sources of acetyl-CoA, ATP, NADH, and NADPH for fatty acid synthesis. However, how these classical pathways integrate into an overall stoichiometrically balanced conversion of sugars to TAGs remains unclear. Previous biochemical and isotope-tracing studies in oilseeds across species have revealed that glycolysis is partially bypassed by alternative routes involving ribulose-1,5-bisphosphate carboxylase/oxygenase (RubisCO) and NADPH-dependent malic enzyme (NADPH-ME). The role of these glycolytic shunts in the overall conversion of carbohydrates to TAG is not fully resolved. Here, we evaluate a minimal stoichiometric model for the carbohydrate-to-TAG conversion that satisfies complete cofactor balancing. Conversion solely through glycolysis and OPPP leads to NADH overproduction and cofactor imbalance. Balanced scenarios require inclusion of the malate shunt, the oxidative RubisCO shunt, or an NADPH-producing glyceraldehyde 3-phosphate dehydrogenase variant. All balanced routes also necessitate active mitochondrial oxidative phosphorylation to convert excess NADH to ATP. Applying a large-scale model of central metabolism in developing Brassica napus seeds, I further predict that glycolytic bypass usage increases in parallel with seed oil content, supporting their role in enabling high lipid biosynthetic fluxes under physiological constraints.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.31.667993v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Benchmarking RNA velocity methods across 17 independent studies</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Luo, Y., Ren, J., Yang, Q., Zhou, Y., You, Z., Li, Q.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-02
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We conducted a benchmark study for 14 existing RNA velocity methods in 17 independent datasets. We evaluated the performances of each method by measuring accuracy, stability, and usability.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> RNA velocity techniques offer great potential for unveiling trajectories of cell state transitions in different biological contexts. While diverse computational methods have been developed, there is no evidence-based guidelines for best-practice in RNA velocity inference. Here we conducted a benchmark study for 14 existing RNA velocity methods in 17 independent datasets. Many validations were done for the first time. We evaluated the performances of each method by measuring accuracy, stability, and usability. Our data showed no single method exhibited superior performance in all the assessments, and unexpected underperformance was observed in certain cases. Especially, the lack of uniformity in the inference results highlights the necessity to compare and control of multiple methods in a single analysis. Our study revealed current limitations and challenges in the RNA velocity methods and informed the best-practice for future studies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.02.668272v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

