<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Monday, 28 July 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Assessing the Performance of AF2 and AF3-Implementations on Antibody-Antigen Complexes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Unsal, S., Holland, B., Sardag, I., Timucin, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that AF2 remains a strong predictor against newer AlphaFold3-inspired methods in Ab-Ag complex prediction, with Chai-1 consistently ranks as the second-best performer across multiple success metrics, and develop AntiConf, a novel metric that achieves superior performance for all methods.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> This study evaluates the performance of AF2, Boltz-1, Boltz-1x, Boltz-2, Chai-1, Protenix and ESMFold, on a curated dataset of 200 novel antibody-antigen (Ab-Ag) complexes. Our findings reveal that AF2 remains a strong predictor against newer AlphaFold3 (AF3)-inspired methods in Ab-Ag complex prediction. Chai-1 consistently ranks as the second-best performer across multiple success metrics. We observed diverse effects of recycling iterations, with AF2, Chai-1 and Protenix benefiting from increased cycles, unlike Boltz variants. We analyzed various model confidence scores, noting high precision from pDockQ2 and high recall from pTM. By integrating these two scores, we developed AntiConf, a novel metric that achieves superior performance for all methods in terms of precision and unique target recall. This work addresses current limitations in AI-based Ab-Ag complex prediction, showcasing the potential of AntiConf for future predictions and assessments of these complexes, and providing a guideline for improving the accuracy of Ab-Ag complex prediction.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.25.666870v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Exploring the Potential of AlphaFold Distograms for Flexibility Assignment in Cryo-EM Maps</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Savas, B., Barlas, A. B., Karaca, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that distance probability distributions from AlphaFold models can reveal conformational flexibility, using a key metabolic complex as a test case.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> AlphaFold (AF) models are widely used in cryo-EM workflows, but their static structure predictions limit the interpretation of flexible or unresolved regions. To address this, we investigate whether AF-generated distance probability distributions (distograms) can reveal conformational flexibility, using the AIFM1/AK2 complex as a test case. Molecular dynamics simulations identified a hinge-driven motion in AK2 upon AIFM1 binding, which was accurately reflected in distograms from AF2.3 and AF3, though not in their predicted structures. Enhanced sampling and MSA perturbations did not improve flexibility detection in earlier AF2 versions. So, by analyzing a key metabolic complex, we demonstrate that distograms might offer a promising approach for structure-free identification of flexible protein regionsl for interpreting dynamic signals in cryo-EM maps.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.25.666757v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Protein Embeddings and Local Alignments</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Malec, J., Golding, G. B., Ilie, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a new scoring function and algorithm for local alignment of protein sequences, we offer a new comprehensive framework for evaluating local alignments, and a new Ankh-score-based program for protein embedding.</p><p style="font-size: 16px; text-align: justify;">Background: The advent of protein embeddings has revolutionized bioinformatics by providing contextual representations that capture functional and evolutionary patterns. They have become, alongside sequence alignments, the cornerstone of bioinformatics. While embeddings cannot replace alignments, they can greatly help improving their quality. Our goal is replacing the BLOSUM matrices, the decades-old standard scoring system for protein alignment, with an embedding-based scoring method. Results: We introduce a new scoring function and algorithm for local alignment of protein sequences, we offer a new comprehensive framework for evaluating local alignments. The score between two residues is given by the cosine similarity of their Ankh-embedding vectors and the algorithm uses dynamic programming with affine penalty. For the evaluation, we built multiple datasets, using both natural and inserted sequences, from the Conserved Domain Database, BALiBASE, and GPCRdb, designed a new algorithm for local alignment extraction, localization and quality evaluation, and employed five distance metrics to evalu- ate the similarity with the true alignment. We performed nearly one and a half million tests to compare the new algorithm with the best BLOSUM matrices, specialized GPCRtm matrices, and top programs, such as PEbA, ProtT5-score, DEDAL, vcMSA and pLM-BLAST. Regarding the protein embedding models, Ankh not only surpasses the best combination of ProtT5 and ESM2, but appears to better understand the "language" of proteins, as it behaves much better on natural sequences compared to artificial ones obtained by inserting domains in random protein sequences. Also, while ProtT5 and ESM2 combine to produce better results, Ankh does not combine well with other embeddings. Conclusions: The new Ankh-score-based program is vastly superior to the BLOSUM matrices and clearly superior to all existing methods. New light shed on the protein embeddings can guide future improvements. In order to facilitate the use of the new method and protocol, they are freely available as a web server at e-score.csd.uwo.ca and as source code at github.com/lucian-ilie/E-score.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.23.666343v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Deep Learning for genomic prediction accounting for heterosis in crossbreeding systems</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> SHOKOR, F., CROISEAU, P., GANGLOFF, H., FRITZ, S., MARTIN, A. A. A., MARY-HUARD, T., CUYABANO, B. C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the presence of heterosis reduces the accuracy of traditional statistical methods but improved the performance of deep learning models for crossbred prediction.</p><p style="font-size: 16px; text-align: justify;">Background: Crossbreeding is used in animal breeding to combine desirable traits from different breeds and to exploit hybrid vigor, and many approaches have been proposed to improve the prediction of genetic values of crossbred animals. This study aimed to assess whether deep learning methods can enhance the prediction of genetic values in crossbred populations by effectively capturing heterosis. We tested several prediction models that varied in how they incorporate genetic information, including purebred and crossbred data, breed composition, and breed-of-origin of alleles, on both simulated and real crossbred data. Finally, we also proposed an approach in which additive genetic effects and heterosis are predicted separately and then combined using effect-specific weights. Results: Our results show that the presence of heterosis reduced the accuracy of traditional statistical methods but improved the performance of deep learning models. When heterosis was present, the highest prediction accuracy for total genetic value was achieved by combining additive genetic values, predicted using statistical method, with the predicted heterosis values, using a weighted sum. The explicit prediction of heterosis using deep learning and a regression method yielded similar results. The integration of breed-of-origin of alleles achieved the highest prediction accuracy for the additive genetic effect, outperforming both methods based on breed composition, and those that train the GBLUP model by combining purebred and crossbred animals. When applied to real data, adding the predicted heterosis did not increase prediction accuracies of crossbred animals, a result that we attributed to the homogeneity of the real crossbred population and its limited sample size. Conclusions: The usefulness of deep learning for crossbred prediction depends on the degree of heterosis exhibited in the traits. Incorporating both additive genetic effects and heterosis considering the breed-of-origin of the alleles in the genomic information and effect-specific weighting leads to more accurate and robust predictions in our simulated data, particularly when crossbred data is heterogeneous. Even when prediction accuracy is not significantly improved, deep learning can still provide valuable insight into the degree of heterosis across traits, offering a deeper understanding of crossbred genetic architecture.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.24.666530v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Combined generalist and host-specific transcriptional strategies enable host generalism in the fungal pathogen Botrytis cinerea</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Singh, R., Muhich, A. J., Tom, C., McMillan, J., Srinivas, K., Faieta, L., Caseys, C., Kliebenstein, D. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We study the generalist necrotrophic fungus Botrytis cinerea and show that transcriptional plasticity is a key mechanism underlying generalism in plant-fungal interactions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> How generalist pathogens infect phylogenetically diverse hosts remains a central question in plant-pathogen biology. In particular, the extent to which broad host range is enabled by genetic variation versus transcriptional plasticity is unclear. To investigate how variation and plasticity contribute to generalism, we studied the generalist necrotrophic fungus Botrytis cinerea that infects more than 1,500 plant species. Using a cross-infection matrix of 72 B. cinerea isolates infected on 57 plant genotypes distributed across 15 eudicot species, we identified general and host-dependent fungal components of lesion formation. Transcriptome profiling at 48 hours post-inoculation revealed two distinct pathogen gene modules: (1) a set of general lesion-associated genes enriched in primary metabolism, showing similar expression across hosts but varied among isolates; and (2) a set of high-entropy, host specific-inducible genes, organized into distinct co-regulated modules that respond dynamically to specific host cues. Both gene sets were genomically dispersed, lacking structural clustering, and were under different levels of selective constraints. Our results demonstrate that B. cinerea employs a modular transcriptional strategy that integrates a core metabolic program along with a plastic, host-responsive regulatory network to achieve broad host colonization. This study presents the most comprehensive cross-species co-transcriptomic dataset to date for any fungal phytopathogen, highlighting transcriptional plasticity as a key mechanism underlying generalism in plant-fungal interactions. Moreover, the identification of conserved fungal gene targets across diverse hosts offers a foundation for developing broad-spectrum resistance strategies in multiple crops.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.24.666639v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Under which circumstances do genomic neural networks learn motifs and their interactions?</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Thompson, M., Lehner, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that a single convolutional layer is sufficient to discover motifs in a sequence-to-function model due to the way in which it shares the gradient locally amongst nucleotides, and that most models are to an extent interpretable, whether their architectures are inherently so or not.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The use of neural networks to model genomic data in sequence-to-function scenarios has soared over the last decade. There remains much debate about whether these models are interpretable, either inherently, or when using downstream interpretability or explainability techniques (xAI). Conclusions are further complicated by the steady publication of novel models, each with their own architectures, evaluations, and xAI experimental designs. Here, we posit that many of these complications arise due to a lack of explicit specification of a generative model, baseline comparators, and thorough evaluation. Consequently, we attempt to reconcile concerns of interpretability under a motif-based generative model by simulating at scale over 1000 motif-based genetic architectures and evaluating the ability of different model architectures to predict an outcome given a sequence as input. We first show that a single convolutional layer is sufficient to discover motifs in a sequence-to-function model due to the way in which it shares the gradient locally amongst nucleotides. We next build upon this by showing that across genetic and network architectures - including attention, LSTMs, and stacked convolutions - most models are capable of modeling motifs and their interactions, with certain models outperforming others across genetic contexts and sample sizes. Distinguishing between shallow-level interpretations of motifs and deeper, gradient-based interpretations of motifs, we show that these approaches discover separate but overlapping sets of motifs, depending on motif characteristics. Finally, we validate our findings on an experimental dataset, and conclude that while attention is accurate, there are genetic contexts in which other neural networks complement findings from attention-based models and produce higher correlations between predictive performance and interpretability. The work here suggests that when a generative model is correctly specified, most models are to an extent interpretable, whether their architectures are inherently so or not. Moreover, our work highlights opportunities for methods development in motif discovery and also implies that employing a mixture of model architectures may be best for biological discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.25.666754v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Pangenome of U.S. ex-PVP and Wild Sorghum Reveals Structural Variants and Selective Sweeps Shaping Adaptation and Trait Improvement</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Michael, T. P., Kitony, J. K., Murray, E. R., Colt, K., Lynch, R. C., Allsing, N., Hartwick, N. T., Duong, T., Saxton, J., Shakoor, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We assembled a sorghum pangenome from long-read sequences of 46 ex-Plant Variety Protection Act (ex-PVP) cultivars and a haplotype-resolved assembly of one wild accession.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Sorghum is a cereal crop grown for food, feed, and biofuel, yet the genomic diversity of elite commercial lines remains underexplored. Here, we assembled a sorghum pangenome from long-read sequences of 46 U.S. ex-Plant Variety Protection Act (ex-PVP) cultivars and a haplotype-resolved assembly of one wild accession. The pangenome revealed largely conserved genomic architecture across elite lines but identified presence-absence variations (PAVs) in 28% of gene families, including genes involved in stress responses and starch metabolism, such as STARCH BRANCHING ENZYME I (SBE1). Selective sweep analyses detected strong signals at key flowering-time loci (Ma1/SbPRR37, Ma2, Ma6) and circadian/light regulators (PHOT1, DET1, XAP5), with PHYA showing copy number variations (CNVs). Analysis of 24 wild accessions, beyond the core pangenome panel, detected CNVs in MULTIDRUG AND TOXIC COMPOUND EXTRUSION (MATE) transporters, potentially reflecting selection or breeding for enhanced herbicide detoxification. These results highlight breeding-shaped regions and position the pangenome as a resource for sorghum improvement.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.25.666886v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Novel Spatial-Structural-Zero-Aware Dissimilarity Measures for Subtype Discovery Using Single Cell Hi-C Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liu, Y., Jin, V. X., Lin, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce structural-zeros-aware Kendall's tau (szKendall), a novel dissimilarity metric that explicitly incorporates the spatial structure of 2D contact matrices and leverages the presence or absence of shared SZs across cells.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> High-throughput single-cell Hi-C (scHi-C) technologies have opened new avenues for investigating cell-to-cell variability in the three-dimensional organization of the genome within individual nuclei. Despite their potential, analyses of scHi-C data are hindered by data sparsity, which varies substantially across cells. To address this challenge, recent methods aim to denoise scHi-C data and differentiate between two types of zero entries: structural zeros (SZs), which reflect true absence of contacts due to biological structure, and dropouts (DOs), which arise from insufficient sequencing depth. However, current dissimilarity measures used in downstream analyses, such as Euclidean distance and Kendall's tau, treat all zeros as equivalent, thus do not distinguish between SZs and DOs nor recognize the special role of SZs in capturing cell-to-cell variability. Such oversight limits the ability to accurately capture biologically meaningful differences between cells. In this study, we introduce structural-zeros-aware Kendall's tau (szKendall), a novel dissimilarity metric that explicitly incorporates the spatial structure of 2D contact matrices and leverages the presence or absence of shared SZs across cells. Through comprehensive simulations and analysis of real scHi-C datasets, we demonstrate that szKendall more effectively captures key structural features and achieves superior performance in cell clustering tasks compared to existing approaches. Our results underscore the importance of SZ-aware dissimilarity measures in advancing single-cell 3D genomics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.23.666407v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Comprehensive Annotation of Conserved Protein Domains in Human Endogenous Retroviruses</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Esteve-Codina, A., Montserrat-Ayuso, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comprehensive resource annotating conserved retroviral domains across 120,000+ ORF regions derived from internal HERV regions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Human endogenous retroviruses (HERVs) occupy nearly 8% of the human genome, yet their protein-coding potential remains largely unexplored. Like their exogenous counterparts, HERVs derive from ancestral germinal infections and typically follow a canonical structure of LTR-gag-pol-env-LTR, where the gag, pol, and env genes encode structural, enzymatic, and envelope proteins, respectively. Here, we present a comprehensive resource annotating conserved retroviral domains across 120,000+ ORFs derived from internal HERV regions. Using a reproducible pipeline based on HMMER and InterProScan, we identified over 17,540 domain hits-primarily from pol genes such as reverse transcriptase, RNase H, and protease-and quantified their structural conservation. Hundreds of domains exceed 95% alignment coverage, revealing a surprising abundance of full-length, retrovirus-like domains in both young and ancient HERV families. While the HERVK subfamily retains the most complete polyprotein architecture-including 13 loci with nearly intact Gag, Pol, and Env domains-many full-length Pol domains were also found in other families such as HERVH, HERVW, and HERVE. Our high-resolution annotations recover conserved catalytic motifs in Pol domains and transmembrane features in Env, enabling fine-grained functional interpretation. All annotations (including BED, FASTA, domain sequences, InterProScan outputs, and transmembrane predictions) are provided as an open resource for functional genomics and HERV expression studies at Zenodo (DOI: 10.5281/zenodo.16318928). This dataset will support downstream analyses of HERV protein expression, immune modulation, and co-option, in diseases and normal physiological conditions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.25.666750v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Prenatal Environmental Determinants of Aromatase Brain-Promoter Methylation in Cord Blood: Chemical, Airborne, Pharmacological, and Nutritional Factors</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tanner, S., Drummond, K., Thomson, S., Vacy, K., Symeonides, C., Novakovic, B., Mansell, T., O'Hely, M., Saffery, R., Tang, M. L., Sly, P. D., Vuillermin, P., Boon, W. C., Jung, C.-H., Park, D., Ponsonby, A.-L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Aromatase, an enzyme encoded by the gene CYP19A1, plays central roles in neurodevelopmental outcomes, and prenatal exposure to BPA can increase levels of DNA methylation - a key epigenetic modification - at the brain-specific CYP 19A1 promoter.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Aromatase, an enzyme encoded by the gene CYP19A1, plays central roles in neurodevelopment. In the brain, its function is to convert androgens into neuroestrogens, ensuring balanced hormonal signalling. Both animal experiments and human studies have shown that, in males, disruption of aromatase, either genetically or epigenetically, can increase symptoms of autism. Prenatal exposure to bisphenol A (BPA), a common plastic chemical, can increase levels of DNA methylation - a key epigenetic modification - at the brain-specific CYP19A1 promoter, P1.f, reducing CYP19A1 expression. However, the extent to which other neurodevelopmentally relevant environmental exposures influence P1.f methylation remains unclear. Here, in the Barwon Infant Study (BIS) birth cohort (N = 906), we analysed the association between 25 prenatal exposures (from five classes previously linked to neurodevelopmental outcomes: manufactured chemicals, air pollution, and pharmacological, nutrition and sunlight-related factors) and methylation of the CYP19A1 P1.f promoter using Weighted Quantile Sum (WQS) regression. We found that the WQS mixture index, a weighted combination of the prenatal exposures, was positively associated with higher P1.f methylation (Adjusted Mean Difference (AMD) = 0.712 (95% CI 0.11, 1.315), p = 0.021), indicating reduced brain aromatase activity. Prenatal exposures with the strongest contribution to the mixture effect included bisphenols (including BPA), reduced sunlight, household mould, phthalates, low folate intake, and air pollution. These findings highlight epigenetic modification of the aromatase gene as a biologically plausible, convergent mechanism through which multiple environmental risk factors for autism may exert effects.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.24.666701v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multimodal Learning Reveals Plants' Hidden Sensory Integration Logic</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Vomo-Donfack, K. L., Ginot, G., Gonzalez Doblas, V., Morilla, I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new paradigm for understanding plant-microbe communication, showing how symbionts exploit latent hubs where sensory pathways converge.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Plants integrate complex environmental signals through interconnected molecular networks, but the fundamental rules governing this sensory integration remain unknown. Studying tomato roots interacting with fungal symbionts, we discover how microbial effectors systematically reprogram plant sensory systems by coordinating transcriptional, metabolic, and phenotypic responses. Our multimodal analysis not only confirmed prior experimental findings through purely computational means, but also revealed novel integration hubs where sensory pathways converge. This dual validation approach establishes iron homeostasis rewiring through citrate-mediated redox control. Next, targeted suppression of jasmonate defences. Thus, nuclear splicing isolation from metabolic noise. These findings establish a new paradigm for understanding plant-microbe communication, showing how symbionts exploit latent hubs where sensory pathways converge. The discovered integration logic provides both fundamental insights into plant perception and concrete targets for engineering stress-resilient crops.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.25.666865v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Role of methylation and siRNA on differential allelic expression in a hybrid of distantly related citrus species.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> DIOP, K., Gibert, A., Llauro, C., Froelicher, Y., Hufnagel, B., Picault, N., Pontvianne, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of DNA methylation and small RNAs in a Citrus hybrid resulting from the cross between C. reticulata and C. australasica .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> DNA methylation plays a central role in the regulation of gene expression. In plants, methylation occurs in the CG, CHG and CHH contexts, via distinct DNA methyltransferases including MET1, CMT3 and the RNA-directed DNA Methylation (RdDM) pathway via DRM2. In interspecific hybrids, these epigenetic mechanisms are confronted to a mixed small RNA population and two subgenomes harboring specific methylation patterns, therefore generating unique expression profiles. The aim of this work was to understand these regulations by analysing gene expression, DNA methylation and small RNAs in a Citrus hybrid resulting from the cross between C. reticulata (mandarin) and C. australasica (caviar lemon). Haplotype-resolved subgenomes assembly identified hundreds of alleles-specifically expressed genes. Asymmetric reprogramming of methylation was observed, in particular an increase in the CHH context in C. australasica haplotype. Surprisingly, CHH methylation, usually associated with gene silencing, was correlated here with increased expression, but also with abundant 24nt small RNA populations at their promoter regions. Similar analyses of the parental lines suggest the correlation between CHH methylation-enriched promoter and high expression level is not due to the hybridization, but seem to be generally true for all citrus. These observations suggest that, in citrus fruit, RdDM could activate transcription. This work also provides a full pipeline to analyse the expression profiles and DNA methylation in complex hybrids, which could be crucial for anticipating varieties resistant to diseases and the current threats affecting citriculture such as the Huanglongbing disease.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.25.666733v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A trade-off mechanism underpins the evolution of a young two-gene sex-determining system in plants</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mai, Y., Zheng, J., Sun, W., Qin, Z., Chen, C., Wei, W., Song, C., Liu, X., Su, X., Wu, F., Wu, Q., Yao, B., Hao, Y., Liu, Y., Zeng, Z., Xu, J., Charlesworth, D., Xia, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the XY sex determination system in the dioecious plant species Eurycorymbus cavaleriei in Sapindaceae involves two Y-linked mutations that act jointly within the developmental male-female trade-off system, together with the effect of YUN{Delta},</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Sexual systems in animals and plants are remarkably diverse, with dioecy having evolved independently in numerous lineages. In plants, dioecy often evolved more recently than in the best-studied animal systems, making plants especially important for understanding how separate sexes evolved independently from functionally hermaphrodite ancestors. Despite long-standing theories of developmental trade-offs in sex allocation, the underlying genetic mechanisms remain elusive. Here, we show that the XY sex determination system in the dioecious plant species Eurycorymbus cavaleriei in Sapindaceae involves two Y-linked mutations that act jointly within the developmental male-female trade-off: YUN{Delta}, a truncated allele that lowers the dosage of the D-class MADS-box gene YUN , and SUNMAO, a novel sRNA locus that silences the X-linked SUN allele. In females, SUN stabilizes the HD-ZIP transcription factor KUN, which is a known sex determinant in another dioecious plant, thereby promoting femaleness by increasing YUN expression; loss of SUN expression, together with the effect of YUN{Delta}, shifts development toward males. Two interlocking regulatory loops in this "SKY" module (SUN-KUN-YUN) fine-tunes YUN dosage. This dioecious system in E. cavaleriei likely evolved by sequential mutations in genes acting in the predicted male-female trade-off system, with their close linkage reflecting a translocation, and later recombination-suppressing inversions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.26.666942v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrated flexible DNA methylation-chromatin segmentation modelingenhances epigenomic state annotation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Aggarwal, N., Schmitz, J. E., Laufer, L., Rahmann, S., Walter, J., Salhab, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We develop EpiSegMixMeth, the first truly integrative segmentation model combining chromatin marks and DNA methylation, which substantially improves the annotation of broad heterochromatic regions -covering over 60% of the genome - that are frequently missed by chromatin-only models.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> DNA methylation and histone modifications together shape the cell-type-specific epigenomic landscape. To enable a more comprehensive genome-wide annotation, we developed EpiSegMixMeth (ESMM), the first truly integrative segmentation model combining chromatin marks and DNA methylation. ESMM extends hidden Markov models with flexible read count distributions and state duration modeling. Applied to 154 highquality human epigenomes from the IHEC EpiAtlas, ESMM substantially improves the annotation of broad heterochromatic regions -covering over 60% of the genome, that are frequently missed by chromatin-only models. Additionally, it precisely defines the boundaries of narrow regulatory elements and resolves local chromatin state transitions during cell differentiation. Notably, we demonstrate that DNA methylation can substitute for missing repressive histone marks in segmentation, ensuring robust annotation across diverse cell types. In memory B-cell development, ESMM reveals fine-scale chromatin state shifts that align with 3D genome architecture changes. Our results highlight the power of integrating DNA methylation into genome segmentation and provide a valuable resource for dissecting cell-type-specific epigenomic regulation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.25.666820v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Parabricks: GPU Accelerated Universal Pan-Instrument Genomics Analysis Software Suite</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Vats, P., Zhu, T., Onken, S., Dunstan, A., Zamirai, B., Puleri, D. F., Nair, A., Oliva, M., Gaihre, A., Sadhnani, P., Li, S., Arumugam, K., Chacon, A., Maric, M., Cohen, J., Sethia, A., Samadi, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Parabricks, a freely accessible, GPU-accelerated software suite supporting diverse workflows, including whole-genome, exome, transcriptome, and methylation analysis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Next-generation sequencing (NGS) has transformed genomics, enabling breakthroughs in biotechnology, healthcare, and pharmaceuticals. However, exponential data growth outpacing Moore's Law presents critical computational challenges for secondary analysis. We introduce Parabricks, a freely accessible, GPU-accelerated software suite supporting diverse workflows, including whole-genome, exome, transcriptome, and methylation analysis. Compatible across major short read and long read sequencing platforms, it reduces processing time from hours to minutes while maintaining concordance with established standards. This acceleration translates to significant cost savings and quick turnaround time. Deployable on-premises or through cloud platforms, Parabricks ensures scalable genomic processing. Integrated AI-driven workflows and deep learning models further enhance precision, offering a universal solution for high-throughput genomic research and clinical diagnostics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.23.666378v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">NeoPrecis: Enhancing Immunotherapy Response Prediction through Integration of Qualified Immunogenicity and Clonality-Aware Neoantigen Landscapes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lee, K.-H., Sears, T. J., Zanetti, M., Carter, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present NeoPrecis, a computational framework designed to refine neoantigen characterization and integrate tumor clonality to improve immunotherapy response prediction.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Despite the transformative impact of cancer immunotherapy, the need for improved patient stratification remains critical due to suboptimal response rates. While neoantigens are central to anti-tumor immunity, current metrics like tumor mutation burden are limited by their neglect of immunogenicity and tumor heterogeneity. We present NeoPrecis, a computational framework designed to refine neoantigen characterization across MHC-I and MHC-II pathways and integrate tumor clonality to improve immunotherapy response prediction. NeoPrecis features an interpretable T-cell recognition model that reveals the critical influence of MHC molecules on TCR recognition beyond mere antigen presentation. Benefit HLA alleles identified through model-driven contribution analysis exhibit significant predictive power for patient outcomes in immune checkpoint inhibitor treatment (melanoma: p-value = 0.04; NSCLC: p-value = 0.01). Applying NeoPrecis to immunotherapy-treated tumors, we show the clonality-aware neoantigen landscape improves response prediction in melanoma and heterogeneous NSCLC, achieving 11% and 20% improvement of AUROC compared to TMB respectively. Heterogeneous NSCLCs, more common among never smokers, retain more subclonal neoantigens due to lower immunoediting pressure, where NeoPrecis better captures the varying prevalence of neoantigens. We propose NeoPrecis as a more comprehensive evaluative framework for neoantigen assessment by incorporating both immunogenicity and tumor clonality, offering insights into the link between collective quality of neoantigen landscapes and immunotherapy response.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.23.666355v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">bacNeo: a computational toolkit for bacteria-derived neoantigen identification</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wang, Y., Li, Y., Gu, K. W., Cheng, X., Wen, B., Zhang, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a multi-omics based computational toolkit that contains three modules to classify bacterial taxonomy, identify host HLA alleles, and predict bacterial neoantigens.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Intratumour microbiome can restore anti-tumour immunity with bacterial peptides potentially serving as neoantigens, while robust in-silico tools for bacterial neoantigen identification remain lacking. We developed bacterial neoantigen (bacNeo), a multi-omics based computational toolkit that contains three modules to classify bacterial taxonomy, identify host HLA alleles, and predict bacterial neoantigens. bacNeo could enable personalised neoantigen discovery to advance therapeutic strategies across diverse cancer types.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.23.666275v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Efficient Identification of Phylogenetically Informative Alignment Sites via Sparse Learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Schrago, C. G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-27
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a novel, topology-agnostic framework for quantifying site-wise phylogenetic information using sparse learning via Lasso (Least Absolute Shrinkage and Selection Operator) regression.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Identifying phylogenetically informative sites in multiple sequence alignments is critical for accurate tree reconstruction and efficient data curation in phylogenomics. Existing approaches often rely on predefined topologies or heuristic criteria, limiting their generality and interpretability. Here, we introduce a novel, topology-agnostic framework for quantifying site-wise phylogenetic information using sparse learning via Lasso (Least Absolute Shrinkage and Selection Operator) regression. By modeling site log-likelihoods as predictors of tree likelihood across a large ensemble of random topologies, our approach isolates the minimal subset of sites that meaningfully contribute to phylogenetic signal. We validate the method using both simulated and empirical mammalian datasets, demonstrating that Lasso-selected sites yield topologies nearly identical to those inferred from full alignments. Furthermore, we show that a simple entropy-based proxy approximates Lasso results with high fidelity, enabling rapid site-level assessments. Compared to commonly used filtering tools, our method yields more concise alignments with equivalent or superior phylogenetic resolution. These findings establish sparse learning as a principled, scalable, and practical approach for evaluating and optimizing phylogenetic data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.24.666198v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

