<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Wednesday, 23 April 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Custom CRISPR-Cas9 PAM variants via scalable engineering and machine learning.</h2>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Engineering and characterizing proteins can be time-consuming and cumbersome, motivating the development of generalist CRISPR-Cas enzymes</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40262634/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The Dawn of High-Throughput and Genome-Scale Kinetic Modeling: Recent Advances and Future Directions.</h2>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Researchers have invested much effort into developing kinetic models due to their ability to capture dynamic behaviors, transient states, and regulatory mechanisms of metabolism, providing a detailed and realistic representation of cellular processes. Historically, the requirements for detailed parametrization and significant computational resources created barriers to their development and adoption for high-throughput studies. However, recent advancements, including the integration of machine learning with mechanistic metabolic models, the development of novel kinetic parameter databases, and the use of tailor-made parametrization strategies, are reshaping the field of kinetic modeling. In this Review, we discuss these developments and offer future directions, highlighting the potential of these advances to drive progress in systems and synthetic biology, metabolic engineering, and medical research at an unprecedented scale and pace.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40262025/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrating machine learning models with multi-omics analysis to decipher the prognostic significance of mitotic catastrophe heterogeneity in bladder cancer.</h2>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Mitotic catastrophe is well-known as a major pathway of endogenous tumor death, but the prognostic significance of its heterogeneity regarding bladder cancer (BLCA) remains unclear.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40259382/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Programmatic design and editing of cis-regulatory elements</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">The development of modern genome editing tools has enabled researchers to make such edits with high precision but has left unsolved the problem of designing these edits. As a solution, we propose Ledidi, a computational approach that rephrases the design of genomic edits as a continuous optimization problem where the goal is to produce the desired outcome as measured by one or more predictive models using as few edits from an initial sequence as possible. When applied across dozens of pre-trained machine learning models, we find that Ledidi can quickly design edits to precisely control transcription factor binding, chromatin accessibility, transcription, and enhancer activity across several species. By using several models simultaneously, Ledidi can programatically design edits that exhibit multiple desired characteristics, and we demonstrate this capability by designing cell type-specific enhancers and accessible regions with controllable patterns of transcription factor binding. Finally, we introduce the concept of an affinity catalog, in which multiple sets of edits are designed that induce a spectrum of outcomes, and demonstrate the practical benefits of this approach for design tasks and scientific understanding.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.22.650035v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Generative prediction of causal gene sets responsible for complex traits</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">The relationship between genotype and phenotype remains an outstanding question for organism-level traits because these traits are generally complex. The challenge arises from complex traits being determined by a combination of multiple genes (or loci), which leads to an explosion of possible genotype-phenotype mappings. The primary techniques to resolve these mappings are genome/transcriptome-wide association studies, which are limited by their lack of causal inference and statistical power. Here, we develop an approach that leverages transcriptional data endowed with causal information and a generative machine learning model to strengthen statistical power. Our implementation of the approach (dubbed TWAVE) includes a variational autoencoder trained on human transcriptional data, which is incorporated into an optimization framework. Given a trait phenotype, TWAVE generates expression profiles, which we dimensionally reduce by identifying independently varying generalized pathways (eigengenes). We then conduct constrained optimization to find causal gene sets that are the gene perturbations whose measured transcriptomic responses best explain trait phenotype differences. By considering several complex traits, we show that the approach identifies causal genes that cannot be detected by the primary existing techniques. Moreover, the approach identifies complex diseases caused by distinct sets of genes, meaning that the disease is polygenic and exhibits distinct subtypes driven by different genotype-phenotype mappings. We suggest that the approach will enable the design of tailored experiments to identify multi-genic targets to address complex diseases.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.17.649405v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genomics of freshwater adaptation in three-spine stickleback from eastern Canada</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">The threespine stickleback is an important species in the study of parallel evolution. However, the evidence for parallel evolution, and the genetic and phenotypic levels, has largely come from the west coast of North America and parts of southern Europe. Here, we use RAD-sequencing of pooled samples to study stickleback marine-freshwater differentiation in Atlantic Canada and test for evidence of parallel evolution. Our results show considerable heterogeneity in differentiation among populations, with some freshwater populations showing high degree of allele frequency divergence relative to marine ones. The strongest candidates were two SNPs near dopamine receptors (Drd24a and Drd2l). Gene ontology (GO) analysis of candidate genes shows dopamine neurotransmitter receptor activity as one of the most enriched gene functions, along with nervous system development. Our study suggests a more complex evolutionary history of stickleback populations in the Atlantic than previously thought, suggesting multiple colonization events or ongoing gene flow. The genomic regions showing repeated differentiation suggest adaptation to freshwater through changes in behaviour.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.18.649616v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Rare-variant aggregate association analysis using imputed data is a powerful approach.</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Imputation can cost-effectively generate genotypes for millions of variants. The power of performing rare variant aggregate association tests using imputed genotypes was evaluated. White Europeans from the UK Biobank with exome sequence and genotype array data were analyzed. Using the genotype type array data from the UK Biobank, imputation was performed using the HRC r1.1 (N=64,976 Haplotypes) and TOPMed r3 (N=267,194 haplotypes) reference panels. Simulations were used to compare the power of performing rare variant aggregate association analysis using sequence and imputed data. Additionally, the number of genes with >2 rare variants (missense, nonsense, splice site) was approximately the same for exome sequence and TOPMed imputed data, but HRC imputed data had ~10% fewer genes. For the imputed data, using a less stringent R2 threshold (i.e., >0.3 vs. >0.8) led to greater power to detect aggregate associations due to additional rare variants included in the test. Exome sequence data provided the highest power for rare variant aggregate association testing, with TOPMed imputed variants usually having less than a 20% reduction in power. HRC imputed variants provided substantially less power. We also performed rare variant aggregate association analyses using UK Biobank phenotype, exome sequence data, and imputed variants for PCSK9 and low-density lipoprotein (N=159,904 study subjects) and APOC3 and triglyceride levels (N=160,036 study subjects). For these analyses, even when ultra-rare variants (minor allele frequency<0.001) were analyzed, significant aggregate associations could be detected for the exome sequence data and TOPMed and HRC imputed variants. Although there is a decrease in power for rare variant aggregate association tests when analyzing imputed variants compared to sequence data due to missing variants and uncertainty in genotypes, analysis of imputed data is a viable approach to detect rare variant aggregate associations when sequence data is unavailable.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.17.649394v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">eDNA metabarcoding and whole genome sequencing detect European-American Eel hybrids in northeastern Canada</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">American and European Eels (Anguilla rostrata and Anguilla anguilla, respectively) are sister species of conservation concern, as they have declined in abundance in recent decades. Although there is evidence of gene flow between the two species, most previous hybrid detections have been found in Iceland. Here we expand upon existing reports of interspecific hybrids using a combination of an environmental DNA (eDNA) survey and whole genome sequencing of 347 eels from Newfoundland and Labrador. Metabarcoding of eDNA samples at multiple loci detected the presence of European Eel mitochondrial DNA (mtDNA) at eight locations primarily along the south coast of Newfoundland. Genome sequencing of eels collected in the region revealed three hybrid individuals with levels of European admixture consistent with one first generation hybrid and two North American backcrosses, all with European mtDNA. This work supports the presence of a previously unknown, seemingly localized region of hybrid occurrence in southern Newfoundland and expands our understanding of the relationship between these species across the North Atlantic.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.21.649106v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Detecting cell-level transcriptomic changes of Perturb-seq using Contrastive Fine-tuning of Single-Cell Foundation Models</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Genome-scale perturbation cell atlases are an exciting new resource to understand the transcriptomic and phenotypic impact of single-gene activation or knockdown. However, in terms of differentially expressed genes identified, the signal detected in these data atlases is low, leading to the exclusion of most data from downstream analyses. Recent advances in single-cell foundation models have shown promise in capturing complex biological insights. However, their application to perturbation analysis, especially in predicting perturbed single-cell transcriptomes, remains limited. In this paper, we focus on learning representations of single-cell transcriptomes that capture subtle, yet important, transcriptome-wide changes, and we propose a novel fine-tuning strategy using contrastive learning to leverage single-cell foundation models for this task. We pre-train a single-cell foundation model and fine-tune on a genome-scale perturbation dataset using a contrastive loss, which minimises the distance between cell embeddings from unperturbed cells while maximising the distance between perturbed and unperturbed cells. We validate and test the model on unseen perturbations, demonstrating its ability to identify global biologically meaningful transcriptional changes not captured by traditional differential expression methods. Our approach provides a novel framework for analysing single-cell perturbation data and offers a more effective means of identifying perturbations that drive systemic gene expression changes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.17.649395v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DESeq2-MultiBatch: Batch Correction for Multi-Factorial RNA-seq Experiments</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">RNA sequencing (RNA-seq) experiments frequently encounter batch effects that can significantly distort biological interpretations, particularly in complex, multifactorial studies where biological variables interact with experimental batch conditions. Existing batch correction tools primarily address technical variability and often neglect these critical interaction effects, resulting in incomplete adjustments. To address this gap, we introduce DESeq2-MultiBatch, a novel, lightweight batch correction method implemented entirely within the DESeq2 analytical framework. Unlike conventional approaches, DESeq2-MultiBatch directly leverages DESeq2's internal model estimates to correct raw gene count data, accurately adjusting for experimental batch effects, including interactions with biological variables. Here, we demonstrate that DESeq2-MultiBatch effectively mitigates batch-related variability while preserving genuine biological differences. Benchmarking against widely used methods highlights DESeq2-MultiBatch as a robust, practical solution for improving exploratory data visualization and downstream analyses in multifactorial RNA-seq studies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.20.649392v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multi-modal single-cell foundation models via dynamic token adaptation</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Recent advances in applying deep learning in genomics include DNA-language and single-cell foundation models. However, these models take only one data type as input. We introduce dynamic token adaptation and demonstrate how it combines these models to predict gene regulation at the single-cell level in different genetic contexts. Although the method is generalisable, we focus on an illustrative example by training an adapter from DNA-sequence embeddings to a single-cell foundation model's token embedding space. As a qualitative evaluation, we assess the impact of DNA sequence changes on the model's learned gene regulatory networks by mutating the transcriptional start site of the transcription factor GATA4 in silico, observing predicted expression changes in its target genes in fetal cardiomyocytes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.17.649387v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">scRegulate: Single-Cell Regulatory-Embedded Variational Inference of Transcription Factor Activity from Gene Expression</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Motivation: Accurately inferring transcription factor (TF) activity from single-cell RNA sequencing (scRNA-seq) data remains a fundamental challenge in computational biology. While existing methods rely on statistical models, motif enrichment, or prior-based inference, they often suffer from deterministic assumptions about regulatory relationships, reliance on static regulatory databases, or lack of interpretability. Moreover, few approaches can effectively integrate prior biological knowledge with data-driven inference to capture novel, dynamic, and context-specific regulatory interactions. Results: To address these limitations, we develop scRegulate, a generative deep learning framework that leverages variational inference to infer TF activities while incorporating gene regulatory network (GRN) priors. By integrating structured biological constraints with a probabilistic latent space model, scRegulate offers a scalable and biologically interpretable solution for prediction of regulatory interactions from scRNA-seq data. We comprehensively benchmark scRegulate using multiple public experimental and synthetic datasets generated from GRouNdGAN to demonstrate its ability to infer TF activities and GRNs that are consistent with the underlying ground-truth regulatory interactions. scRegulate outperforms existing TF inference methods, achieving AUROC values of 0.71-0.86 and AUPRC values of 0.80-0.95 on three synthetic datasets. Additionally, scRegulate accurately recapitulates experimentally validated TF knockdown effects on a Perturb-seq dataset, achieving a mean log2 fold change of -0.66 to -16.98 (p [&le;] 8.06x10-13) for key TFs such as ELK1, EGR1, and CREB1. Applied to the PBMC scRNA-seq data, scRegulate reconstructs cell-type-specific GRNs and identifies differentially active TFs that align with known immune regulatory pathways. Furthermore, we show that scRegulate's TF embeddings capture meaningful transcriptional heterogeneity, enabling accurate clustering of cell types. Collectively, our results establish scRegulate as a powerful, interpretable, and scalable framework for inferring TF activities and regulatory networks from single-cell transcriptomics. Availability: All are available on GitHub at github.com/YDaiLab/scRegulate.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.17.649372v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genetic mapping of resistance: A QTL and associated polymorphism conferring resistance to alpha-cypermethrin in Anopheles funestus</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">The heavy reliance on pyrethroid-based interventions has accelerated the spread of resistance malaria vectors including Anopheles funestus, jeopardising control efforts. The efficacy of insecticide-based interventions, especially insecticide-treated nets (ITNs), the cornerstone of malaria control and management, is threatened by the widespread resistance complicating malaria control. Alpha-cypermethrin, a type II pyrethroid, is increasingly utilised in various ITN formulations, including those combined with piperonyl butoxide (PBO) and chlorfenapyr-based Interceptor G2 (IG2) nets, to enhance effectiveness against resistant mosquito populations. Therefore, understanding the molecular basis of resistance is essential to monitor and track resistance trends for an effective malaria control program. In this study, we identified a 1.4 Mb QTL on the telomeric end of the left arm of chromosome 2, conferring resistance to alpha-cypermethrin (rap1 QTL). Different crossing schemes and sequencing approaches were explored to determine the most effective strategy. Individual-based QTL mapping performed on segregating individuals from an isofemale family identified a QTL at the F7 generation. Higher recombination density relative to the physical genome in the F7 isofemale family, with a recombination every 240 kb, facilitated the detection of a QTL compared to the F2 family (335 kb/cM). Additionally, we exploited bulk segregate analysis (BSA) between susceptible and resistant phenotypes from the F7 isofemale family and an F7 mixed cross-family to perform cost-effective and rapid QTL-mapping discovery. The strongest signal in both independent BSA analyses overlaps with the rap1 QTL, further supporting its role in alpha-cypermethrin resistance. The known resistant alleles of the cytochrome P450 CYP6P9a and 6.5-kb structural variant within the rap1 QTL strongly correlate with survival to alpha-cypermethrin. In this study, we validated that previously developed DNA-based assays, originally designed to monitor permethrin resistance, are effective for tracking resistance to alpha-cypermethrin as well. Additionally, we identified candidate variants that can serve as reliable markers for monitoring alpha-cypermethrin resistance.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.16.649184v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Improved CRISPR/Cas9 Off-target Prediction with DNABERT and Epigenetic Features</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">CRISPR/Cas9 genome editing is a powerful tool in genetic engineering and gene therapy; however, off-target effects pose significant challenges for clinical applications. Accurate prediction of these unintended edits is crucial for ensuring safety and efficacy. In this study, we propose a novel approach that integrates DNABERT, a pre-trained DNA language model, with epigenetic features to improve off-target effect prediction. We evaluated DNABERT-based models against five state-of-the-art baseline models (GRU-Emb, CRISPR-BERT, CRISPR-HW, CRISPR-DIPOFF, and CrisprBERT) using four key performance metrics (F1-score, MCC, ROC-AUC, and PR-AUC). Additionally, we conducted ablation studies to assess the impact of DNABERT's pre-training and epigenetic features, demonstrating that both significantly enhance predictive performance. Furthermore, we explored an ensemble modeling approach, which achieves superior prediction accuracy compared to individual models. Finally, we visualized DNABERT's attention weights to gain insights into its decision-making process, revealing biologically relevant patterns in off-target recognition. The source codes used in this study are available at github.com/kimatakai/CRISPR_DNABERT.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.16.649101v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Spatially aligned random partition models on spatially resolved transcriptomics data</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">We propose spatially aligned random partition (SARP) models for clustering multiple types of experimental units, incorporating dependence in a subvector of the cluster-specific parameters, e.g., a subvector of spatial information, as in the motivating application. The approach is developed for inference about co-localization of immune, stromal, and tumor cell sub-populations. The aim is to understand the recruitment of immune and stromal cell subtypes by tumor cells, formalized as spatial dependence of the corresponding homogeneous cell subpopulations. This is achieved by constructing Bayesian nonparametric random partition models for the different types of cells, with a hierarchically structured prior introducing the desired dependence. Specifically, we use Pitman-Yor priors and add dependence in the base measure for spatial features, while leaving the base measure corresponding to gene expression features a priori independent across different types of cells. Details of the model construction are designed to lead to a convenient MCMC algorithm for posterior inference. Simulation studies show favorable performance in identifying co-localization between types of cells. We apply the proposed approach with colorectal cancer (CRC) data and discover subtypes of immune and stromal cells that are spatially aligned with specific tumor regions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.16.649218v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Complex structural variant visualization with SVTopo</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Structural variants are genomic variants that impact at least 50 nucleotides and can play major roles in diversity and human health. Many structural variants are complex multi-breakpoint rearrangements that are difficult to comprehend with existing visualization tools. We present SVTopo, a tool to visualize complex structural variants with supporting evidence from high-accuracy long reads, in easily understood figures. We include examples of eleven categories of complex structural variants from seven human genomes. SVTopo shows breakpoint evidence in ways that aid reasoning about the impact of large, multi-breakpoint events such as inversions, translocations, and combinations of simpler structural variants.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.16.649185v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

