<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Wednesday, 23 July 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">From Dataset Curation to Unified Evaluation: Revisiting Structure Prediction Benchmarks with PXMeter</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ma, W., Liu, Z., Yang, J., Lu, C., Zhang, H., Xiao, W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce PXMeter, an open-source toolkit that support consistent and reproducible evaluation of diverse structure prediction models across a broad spectrum of biological complex structures.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent advances in deep learning have significantly improved the accuracy of structure prediction for biomolecular complexes; however, robust evaluation of these models remains a major challenge. We introduce PXMeter, an open-source toolkit that support consistent and reproducible evaluation of diverse predictive models across a broad spectrum of biological complex structures. PXMeter provides a unified and reproducible benchmarking framework, offering valuable insights to support the ongoing improvement of structure prediction methods. We also present a high-quality benchmark dataset curated from recently deposited structures in the Protein Data Bank (PDB). These entries are manually reviewed to exclude non-biological interactions, ensuring reliable evaluation. Using these resources, we conducted a comprehensive benchmark of several structure prediction models, namely Chai-1, Boltz-1, and Protenix. Our benchmarking results demonstrate the advancements achieved by deep learning models, while also identifying ongoing challenges - especially in modeling protein-protein and protein-RNA interactions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.17.664878v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ProtLoc-GRPO: Cell line-specific subcellular localization prediction using a graph-based model and reinforcement learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zeng, S., Zhang, W., Li, C., Jiang, Y., Wang, D., Shao, Q., Xu, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a reinforcement learning approach, ProtLoc-GRPO, to enhance subcellular localization prediction by optimizing the structure of the underlying PPI network.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Subcellular localization prediction is crucial for understanding protein functions and cellular processes. Subcellular localization is dependent on tissue and cell lines derived from different cell types. Predicting cell line-specific subcellular localization using the information of protein-protein interactions (PPIs) offers deeper insights into dynamic cellular organization and molecular mechanisms. However, many existing PPI networks contain systematic errors that limit prediction accuracy. In this study, we propose a reinforcement learning approach, ProtLoc-GRPO, to enhance subcellular localization prediction by optimizing the structure of the underlying PPI network. ProtLoc-GRPO learns to rank and retain the most informative PPI edges to maximize the macro-F1 score for cell line-specific subcellular localization. Our approach yields a 7% improvement in macro-F1 score over the baseline. We further evaluate its robustness across various edge pruning rates and benchmark it against conventional pruning strategies. Results show that our proposed method consistently outperforms existing approaches. To our knowledge, this work represents the first study to predict cell line-specific protein subcellular localization and the first application of the Group Relative Policy Optimization (GRPO) framework to a graph-based model for bioinformatics tasks.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.17.665451v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comparative Genomics of Transisthmian Damselfishes (Abudefduf saxatilis and A. troschelii)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tracy, C. B., Arias, C. F., Tysall, E., Hoeppner, M. P., McMillan, W. O., Puebla, O., Bernal, M. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present genome assemblies for the Sergeant Major Abudefduf saxatilis (Tropical Atlantic) and the Panamic Sergeant Major Troschelii (T Tropical Eastern Pacific) derived from PacBio long-read sequencing.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The uplift of the Central American Isthmus (CAI) represents a natural laboratory for the study of allopatric speciation in marine organisms. Several geminate species pairs of both vertebrates and invertebrates formed following the uplift of the CAI, including damselfishes of the Pomacentridae family. However, to date no studies have explored the genomic differences among geminate species in the Pacific and Atlantic Oceans. In this study we present genome assemblies for the transisthmian species pair consisting of the Sergeant Major Abudefduf saxatilis (Tropical Atlantic) and the Panamic Sergeant Major Abudefduf troschelii (Tropical Eastern Pacific) derived from PacBio long-read sequencing. The new genomes are near-chromosome level, and among the highest-quality genomes currently available for coral reef fishes. We show that large structural variants distinguish the two species, including a 9 megabase inversion in linkage group (putative chromosome) 6. Additionally, we show through an analysis of demographic history that alleles within the two genomes have different coalescence time distributions, which may be due to different effective population sizes, population structures, and/or selection regimes in the two oceans. Finally, we highlight gene families that were significantly expanded or contracted between A. troschelii and A. saxatilis. Some of these are related to the environmental conditions that differ between the two oceans, such as gamma crystallin M (crygm), which is linked to vision, and vitellogenin (vtg), which is associated with egg provisioning. These genomes set the stage for comparative analyses of genetic structure and selection on the marine organisms that originated with the formation of the CAI.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.17.665425v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Transfer learning across molecular graphs for predicting protein-ligand affinities and their changes upon mutations</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhou, Y., Myung, Y., de Sa, A. G. C., Ascher, D. B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present DDMuffin, a deep learning framework that integrates structural, sequence, and interaction graph features, employing transfer learning and stringent dataset partitioning to achieve reliable generalization.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Predicting protein-ligand binding affinity and mutation-induced affinity changes ({Delta}{Delta}G) remains challenging due to limited data and complex interaction mechanisms. Here we present DDMuffin, a deep learning framework that integrates structural, sequence, and interaction graph features, employing transfer learning and stringent dataset partitioning to achieve reliable generalization. DDMuffin demonstrates strong predictive accuracy on the rigorous LP-PDBBind benchmark (Pearson r up to 0.70 after excluding top 10% outliers; RMSE = 1.48 kcal mol-1). In evaluating mutation-induced affinity changes for clinically relevant kinase inhibitors, DDMuffin achieves competitive average performance (mean Spearman {rho} = 0.39), outperforming or matching several established methods. The approach provides valuable insights into mutation-driven drug resistance and interpretable ligand binding mechanisms, particularly advantageous for guiding inhibitor design and personalized therapeutic strategies. To facilitate broad application, we deployed DDMuffin as an accessible web server at https://biosig.lab.uq.edu.au/ddmuffin/, promoting systematic exploration of protein-ligand interactions in drug discovery research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.19.665665v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Segmental duplication-mediated rearrangements alter the landscape of mouse genomes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Francoeur, E. R., Audano, P. A., Ferraj, A., Balachandran, P., Beck, C. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify 223 rearrangements in mouse genomes and identify a potential role for transposable elements in promoting recombination and generating SDs within mice.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Segmental duplications (SDs) are dynamic regions of mammalian genomes that drive structural variation and phenotypic diversity. However, genome-wide characterization of the SD landscape and the rearrangements mediated by SD paralogs in mouse genomes has been limited by the near-exclusive reliance on short-read sequencing technologies. Here, we integrate long-read genome assemblies, optical mapping, and k-mer based copy number analysis across eight genetically diverse inbred mouse strains to identify and characterize SD-mediated rearrangements. We identify 223 rearrangements affecting over 14 Mb of sequence and reveal substantial variation in gene content. These rearrangements affect loci involved in immunity, sensory perception, and gene regulation, including variation in the amylase gene cluster (Amy2a) and KRAB-zinc finger genes. We observed that SD flanks in mouse genomes are significantly enriched for young LINE-1 transposable elements, suggesting a potential role for transposons in promoting recombination and generating SDs within mice. Our findings highlight the contribution of SDs to genome structure and intra-species variation, and provide a resource for identifying regions prone to rearrangement in a critical model organism for biomedical research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.18.665526v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">IMGT(R) Analysis of the Human IGH Locus: Unveiling Novel Polymorphisms and Copy Number Variations in Genome Assemblies from Diverse Ancestral Backgrounds</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Papadaki, A., Georga, M., Jabado-Michaloud, J., Folch, G., Zeitoun, G., Duroux, P., Giudicelli, V., Kossida, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We uncover novel IGH alleles, copy number variations (CNV), and polymorphisms, particularly within the variable (IGHV) region, and previously uncharacterized genetic variability in the constant (IGHC) region and distinct IMGT CNV forms.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Unraveling the genetic complexity of the human immunoglobulin heavy chain (IGH) locus provides valuable insights into the mechanisms underlying the efficacy and specificity of the adaptive immune response. Despite its crucial role, the IGH locus remains insufficiently characterized, with its allelic diversity and polymorphisms inadequately investigated. In this study, we present an analysis of the human IGH locus, incorporating 15 human genome assemblies from diverse ancestries, including African, European, Asian, Saudi, and mixed backgrounds. Through our examination of both maternal and paternal assemblies, we uncover novel IGH alleles, copy number variations (CNV), and polymorphisms, particularly within the variable (IGHV) region. Our findings reveal extensive and previously uncharacterized genetic variability in the constant (IGHC) region and distinct IMGT CNV forms across individuals. This research contributes to a significant enrichment of the IMGT(R) IGH reference directory, databases, tools and web resources and lays the groundwork for a comprehensive IMGT(R) haplotype database which can be progressively enriched to support future studies in population-specific immune profiles and adaptive immune related disease susceptibility, as comprehensive datasets become available. Such a resource promises to propel personalized immunogenomics forward, with exciting applications in cancer immunotherapy, COVID-19, and other immune-related diseases.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.18.665479v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">centriflaken: an automated data analysis pipeline for assembly and in silico analyses of foodborne pathogens from metagenomic samples</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Koganti, K., Kase, J., Gonzalez-Escalona, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present centriflaken, an automated precision metagenomics pipeline for detecting and characterizing Shiga toxin-producing Escherichia coli (STEC) from metagenomic data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Rapid and comprehensive analysis of metagenomic data from samples associated with foodborne outbreaks is of critical importance in food safety. Equally important is the need for automated analysis pipelines that allow the rapid and effective construction of metagenomic assembled genomes (MAGs) to enable bacterial source-tracking from metagenomic data. Here, we present centriflaken, an automated precision metagenomics pipeline for detecting and characterizing Shiga toxin-producing Escherichia coli (STEC) from metagenomic data. centriflaken streamlines the process of generating metagenome-assembled genomes (MAGs) and conducting in silico analyses of STECs, significantly reducing the time and manual effort required for comprehensive pathogen profiling. centriflaken was validated using Oxford Nanopore long-read sequencing data from agricultural water enrichments, successfully reproducing results from our previous study that involved multiple manual bioinformatics steps (Maguire et al., 2021). The tool's efficacy was further demonstrated through its application to ZymoBIOMICS microbial community standards and 21 additional irrigation water samples, completing STEC precision metagenomics analyses in less than 7 hours per sample. centriflaken's versatility allows for the analysis of user-defined taxa beyond STEC, including other foodborne pathogens like Listeria monocytogenes or Salmonella. The pipeline generates comprehensive summary plots and tables, accessible through a MultiQC HTML report. Designed for portability, centriflaken packages all software dependencies within containers and virtual environments. This open-source tool is available on GitHub under the MIT license (https://github.com/CFSAN-Biostatistics/centriflaken), offering a powerful resource for rapid, automated pathogen detection and characterization in food safety applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.18.665485v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multi-omics evaluation of cell lines as models for metastatic prostate cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liu, X., Wang, Y., Liu, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified VCaP as the most representative model for metastatic prostate adenocarcinoma, while highlighting the limited fidelity of the widely used PC3 cell line.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cancer cell lines are indispensable for prostate cancer research, yet their ability to model metastatic prostate cancer remains insufficiently characterized. In this study, we performed a systematic multi-omics evaluation integrating genomic and transcriptomic data from large public datasets. We identified VCaP as the most representative model for metastatic prostate adenocarcinoma, while highlighting the limited fidelity of the widely used PC3 cell line. Our analysis underscores the importance of selecting hypermutated cell lines, which display distinct tumor microenvironment characteristics. Moreover, overexpression of MET in VCaP induced partial stem-like traits but did not fully recapitulate the MSPC transcriptomic profile. Notably, stem-like prostate organoids demonstrated higher similarity to MSPC and may serve as more suitable MSPC models. Our study provides guidance for the selection of prostate cancer cell lines, assisting researchers in choosing more representative metastatic prostate cancer cell line models.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.17.665334v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Deep Learning Prediction of Intact N-/O-Glycopeptide Tandem Mass Spectra Enhances Glycoproteomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zong, Y., Wang, Y., Qiao, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce DeepGPO, a deep learning framework for the prediction of both N- and O-glycopeptides MS/MS spectra.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein glycosylation, a post-translational modification involving the attachment of glycans to proteins, plays critical roles in numerous physiological and pathological cellular functions. Characterization of protein glycosylation is among one of the most challenging problems in proteomics due to the high heterogeneity of glycosites and glycan structures. Recently, deep learning has been adopted to predict N-glycopeptide tandem mass (MS/MS) spectra and exhibited a promising effect in N-glycoproteomics. However, current deep learning frameworks struggle to accurately predict O-glycopeptide MS/MS spectra due to the complexity of O-glycopeptides and the limited availability of training data. In this study, we introduce DeepGPO, a deep learning framework for the prediction of both N- and O-glycopeptide MS/MS spectra. The DeepGPO incorporates a Transformer module alongside two graph neural network (GNN) modules specifically designed for handling branched glycans. To address the issue of data scarcity in O-glycoproteomics datasets, various training methods are adopted in DeepGPO, such as the introduction of training weights for different MS/MS spectra and the adoption of pre-training strategies. DeepGPO exhibits accurate prediction of N- and O-glycopeptides MS/MS spectra. With the predicted MS/MS, glycosylation sites can be localized even in the absence of site-determining ions, for instance using higher-energy collisional dissociation (HCD) for the localization of O-glycosylation. We also explored the possibility of differentiating O-glycosites and N-glycosites using the predicted MS/MS spectra. DeepGPO primarily addresses mono-glycosylated peptides and is capable of handling doubly glycosylated peptides, but it currently cannot process peptides with three or more glycan modifications due to limited training data and increased spectral complexity. We anticipate that DeepGPO will inspire future advancements in glycoproteomics research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.20.665808v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The genome of the early diverged amphioxus, Asymmetron lucayanum, illuminates the evolution of genome architecture and gene repertoires in cephalochordates</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ren, Y., Miao, Z., Lin, C.-Y., Yang, L., Li, H., Holland, L. Z., Cho, S.-J., Yu, J.-K., Yue, J.-X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We reconstruct the ancestral genomic architecture of the Cephalochordate lineage for tracing genome evolutionary process during Deuterostome and Chordate diversification, and show that purifying selection and constraints on co-transcriptional regulation may have contributed to the maintenance of conserved microsynt</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cephalochordates (amphioxus or lancelet) are considered as living proxies for ancestral chordates due to their key phylogenetic position and slow evolutionary rate. The genomes of living amphioxus thus can help to reveal the genetic basis shaping the evolutionary transition from invertebrates to vertebrates. To gain a comprehensive understanding of the genome architecture in amphioxus, here we generate a chromosome-anchored genome assembly for Asymmetron lucaynum, representing the earliest diverging cephalochordate genus. Our results show that Asymmetron has an enlarged genome compared to those of the other four cephalochordate genomes decoded so far (all in the genus Branchiostoma), causing by pervasive expansions of inter-genic transposable elements (TEs). Nevertheless, both macrosynteny and microsynteny remain highly conserved between Asymmetron and Branchiostoma, enabling reconstruction of the ancestral genomic architecture of the Cephalochordate lineage for tracing genome evolutionary process during Deuterostome and Chordate diversification. By coupling developmental transcriptomic analyses, we further show that purifying selection and constraints on co-transcriptional regulation may have contributed to the maintenance of the conserved microsynteny blocks among cephalochordate species. We also examine the evolutionary history of Hox cluster in cephalochordates and vertebrates, and identify species-specific inversions and TE invasions at this important locus in both Asymmetron and Branchiostoma. Finally, we survey key gene families involved in both innate and adaptive immunity (e.g., TLR, NLR, MHC, and RAG) and uncover their plausible ancestry and evolutionary dynamics in chordates. Taken together, our findings illuminate the genome and gene evolution of cephalochordates and provide valuable resources for understanding the early evolution of chordates and the origin of vertebrates.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.21.665930v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Phenotypic scoring of Canola Blackleg severity using machine learning image analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hu, Q., Anderson, S. N., Gardner, S., Ernst, T. W., Koscielny, C. B., Bahia, N. S., Johnson, C. G., Jarvis, A. C., Hynek, J., Coles, N., Falak, I., Charne, D. R., Ruidiaz, M. E., Linares, J. N., Mazis, A., Stanton, D. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a machine learning algorithm based on deep learning models that can score blackleg severity from cross-section images of infected plants.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Canola blackleg is a fungal disease that causes significant yield loss and plant death of infected canola (Brassica napus L., Brassica rapa L., Brassica juncea L.) fields worldwide. One of the most effective methods for controlling blackleg is through the cultivation of resistant varieties. Consequently, scoring blackleg disease severity of infected plants is a key metric for identifying and selecting resistant varieties. Traditionally, blackleg severity is scored by expert raters who evaluate disease in stem cross sections using established rating scales and reference images; however, human raters are expensive and inconsistent in their scoring. Here, we introduce a machine learning algorithm based on deep learning models that can score blackleg severity from cross-section images of infected plants. We find that expert ratings are largely inconsistent across raters and across years for the same rater, creating substantial noise in susceptibility ratings. Meanwhile, our trained machine learning model performs more consistently than the median rater while maintaining a similar heritability as expert raters for the blackleg susceptibility trait. This model can be used to standardize blackleg susceptibility scoring across locations and years to improve canola breeding outcomes across affected regions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.17.665372v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Spatially resolved analysis of longitudinal fruit growth in persimmon (Diospyros kaki) via three-dimensional phenotyping</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kusumi, A., Nishiyama, S., Yamane, H., Tao, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of fruit shape regulation and fruit morphology in persimmon .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding fruit development patterns is essential, as they are directly linked to fruit size and shape, which ultimately determine final yield and quality. While conventional models describe fruit development based on growth parameters for the whole fruit, it remains unclear how tissue growth at different fruit portions is coordinated to promote overall fruit development. Therefore, in this study, we investigated the relationship between differences in developmental rates among fruit portions and fruit morphology in persimmon (Diospyros kaki), a species known for its high morphological diversity. Starting two weeks after blooming (WAB), surface landmarks were routinely drawn along the calyx-apex line on fruits of four persimmon cultivars with diverse shapes, and the landmark displacement was quantified every two weeks using 3D phenotyping. Consistent with previous studies, all cultivars exhibited active development in the regions near the calyx, with the greatest difference in growth rates between the calyx and apex occurring at 6-8 WAB. However, the extent of these spatial differences varied among cultivars. In elongated fruits, a gradual growth gradient was observed from the calyx toward the apex, with relatively high growth rates even near the apex. In contrast, in flattened fruits, growth was highly concentrated near the calyx, while development in other regions remained minimal. The findings of this study provide a foundation for future research on fruit shape regulation and the elucidation of physiological disorders in persimmon.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.17.665444v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">T2T Gap-free Genome Assembly of Gp03, a Soybean Cultivar with High Genetic Transformation Efficiency</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Feng, w., Lian, L., Xun, H., Guo, D., Wang, X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We report the complete T2T genome assembly of Gp03, a soybean cultivar with high genetic transformation efficiency, developed in Northeast China.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Soybean is a major cultivated crop worldwide, serving as a crucial source of edible oil, plant protein, and animal feed. The continuous growth of the global population necessitates accelerated soybean breeding. High-quality reference genomes are foundational for soybean functional genomics and breeding research. Several telomere-to-telomere (T2T) reference genomes have been released, facilitating subsequent functional studies. However, soybean varieties show varying efficiencies in genetic transformation, with currently available reference genomes having low transformation efficiencies. Here, we report the complete T2T genome assembly of Gp03, a soybean cultivar with high genetic transformation efficiency, developed in Northeast China. Gp03 exhibited high transformation efficiency in comparative studies and favorable agronomic traits for dense planting. The assembly comprises 20 chromosomes with a total size of 1.01 GB, an N50 length of 51.5 Mb, and includes 40 telomeres and 20 putative centromere regions. Genome annotation identified 61,832 genes and 545.8 Mb of repetitive sequences, accounting for 53.8% of the genome. This work provides a valuable genetic resource for soybean breeding and is expected to accelerate molecular breeding efforts.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.21.665664v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DANCE 2.0: Transforming single-cell analysis from black box to transparent workflow</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ding, J., Xing, Z., Wang, Y., Liu, R., Liu, S., Huang, Z., Tang, W., Xie, Y., Zou, J., Qiu, X., Ma, J., Yu, G., Tang, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present DANCE 2.0, an automated and interpretable preprocessing platform featuring two key modules: the Method-Aware Preprocessing (MAP) module, which discovers optimal pipelines for task-specific methods via hierarchical search, and the Dataset- aware Pre Processing (DAP</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Preprocessing is a critical step in single-cell data analysis, yet current practices remain largely a black-box, trial-and-error process driven by user intuition, legacy defaults, and ad hoc heuristics. The optimal combination of steps, such as normalization, gene selection, and dimensionality reduction, varies across tasks, model architectures, and dataset characteristics, hindering reproducibility and method development. We present DANCE 2.0, an automated and interpretable preprocessing platform featuring two key modules: the Method-Aware Preprocessing (MAP) module, which discovers optimal pipelines for task-specific methods via hierarchical search, and the Dataset-Aware Preprocessing (DAP) module, which recommends pipelines for new datasets via similarity-based matching to a reference atlas. Together, MAP and DAP execute over 325,000 pipeline searches across six major tasks - clustering, cell type annotation, imputation, joint embedding, spatial domain identification, and cell type deconvolution - yielding robust and generalizable recommendations. MAP-recommended pipelines consistently outperform original method defaults, with substantial gains across all tasks. Beyond automation, DANCE 2.0 reveals interpretable preprocessing patterns across tasks, methods, and datasets, transforming preprocessing into a transparent, data-driven process. All resources are openly available at https://github.com/OmicsML/dance to support broad community adoption and future methodological advances.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.17.665427v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Protein structure prediction and design for high-throughput computing</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mathew, V. S., Kellogg, G. D., Lai, W. K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed containerized solutions for AlphaFold3, Chai-1, Boltz-2, and RFdiffusion, optimized across a variety of computational architectures (e.g., x86 and ARM).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent advances in structural biology and machines learning have resulted in a revolution in molecular biology. This revolution is driven by protein structure prediction and design tools such as Alphafold3, Chai-1, and Boltz-2 which are now able to accurately model protein structures as well as predict protein-complex formation with a variety of substrates at atomic resolution (i.e., DNA, RNA, small ligands, post-translational modifications). The impact of these protein-structure prediction algorithms has been matched by the emergence of in silico protein design platforms (RFdiffusion), which now promise to revolutionize synthetic biology and novel disease therapeutics. Despite their potential to transform molecular biology, the adoption of these algorithms is hindered in part, not only by their high computational requirements, but also by the difficulty in deploying these algorithms on available systems. To help address these barriers, we developed containerized solutions for AlphaFold3, Chai-1, Boltz-2, and RFdiffusion, optimized across a variety of computational architectures (e.g., x86 and ARM). Additionally, we present OmniFold, an optimized wrapper-platform with automatic QC report generation that enables AlphaFold3, Chai-1, and Boltz-2 to perform simultaneously while more efficiently utilizing GPU systems. Precompiled containers and their definition files are available as open source through Sylabs and GitHub. We hope that these containers and repos will help to facilitate reproducibility, accessibility, and accelerate scientific discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.18.665594v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SEAL: Spatially-resolved Embedding Analysis with Linked Imaging Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Warchol, S., Guo, G., Knittel, J., Freeman, D., Bhalla, U., Muhlich, J., Sorger, P. K., Pfister, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-22
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a novel hybrid-embedding visualization that preserves morphological and positional information while integrating critical high-dimensional feature data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Dimensionality reduction techniques help analysts make sense of complex, high-dimensional spatial datasets, such as multiplexed tissue imaging, satellite imagery, and astronomical observations, by projecting data attributes into a two-dimensional space. However, these techniques typically abstract away crucial spatial, positional, and morphological contexts, complicating interpretation and limiting insights. To address these limitations, we present SEAL, an interactive visual analytics system designed to bridge the gap between abstract 2D embeddings and their rich spatial imaging context. SEAL introduces a novel hybrid-embedding visualization that preserves morphological and positional information while integrating critical high-dimensional feature data. By adapting set visualization methods, SEAL allows analysts to identify, visualize, and compare selections, defined manually or algorithmically, in both the embedding and original spatial views, facilitating a deeper understanding of the spatial arrangement and morphological characteristics of entities of interest. To elucidate differences between selected sets of items, SEAL employs a scalable surrogate model to calculate feature importance scores, identifying the most influential features governing the position of objects within embeddings. These importance scores are visually summarized across selections, with mathematical set operations enabling detailed comparative analyses. We demonstrate SEAL's effectiveness and versatility through three case studies: colorectal cancer tissue analysis with a pharmacologist, melanoma investigation with a cell biologist, and exploration of sky survey data with an astronomer. These studies underscore the importance of integrating spatial context into embedding spaces when interpreting complex imaging datasets. Implemented as a standalone tool while also integrating seamlessly with computational notebooks, SEAL provides an interactive platform for spatially informed exploration of high-dimensional datasets, significantly enhancing interpretability and insight generation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.19.665696v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

