<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Wednesday, 21 May 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CMImpute: cross-species and tissue imputation of species-level DNA methylation samples across mammalian species.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Emily Maciejewski, Steve Horvath, Jason Ernst</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We develop CMImpute (Cross-species Methylation Imputation), a method based on a conditional variational autoencoder, to impute DNA methylation representing species-tissue combinations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The large-scale application of the mammalian methylation array has substantially expanded the availability of DNA methylation data in mammalian species. However, this data captures only a small portion of species-tissue combinations. To address this, we develop CMImpute (Cross-species Methylation Imputation), a method based on a conditional variational autoencoder, to impute DNA methylation representing species-tissue combinations. We demonstrate that CMImpute achieves strong sample-wise correlation between imputed and observed values. Using CMImpute and data from 348 species and 59 tissue types, we impute methylation data for 19,786 new species-tissue combinations. We expect CMImpute will be a useful resource for DNA methylation analyses.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40394556/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A rapid accurate approach to inferring pedigrees in endogamous populations.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cole M Williams, Brooke A Scelza, Sarah D Slack, Neus Font-Porterias, Dana R Al-Hindi, Rasika A Mathias, Harold Watson, Kathleen C Barnes, Ethan Lange, Randi K Johnson, Christopher R Gignoux, Sohini Ramachandran, Brenna M Henn</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a simple, rapid algorithm which initially uses only high-confidence first-degree relationships to seed a machine learning step based on summary statistics of identity-by-descent (IBD) sharing.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurate reconstruction of pedigrees from genetic data remains a challenging problem. Many relationship categories (e.g. half-sibships versus avuncular) can be difficult to distinguish without external information. Pedigree inference algorithms are often trained on European-descent families in urban locations. Thus, existing methods tend to perform poorly in endogamous populations for which there may be reticulations within the pedigrees and elevated haplotype sharing. We present a simple, rapid algorithm which initially uses only high-confidence first-degree relationships to seed a machine learning step based on summary statistics of identity-by-descent (IBD) sharing. One of these statistics, our ``haplotype score'', is novel and can be used to: (1) distinguish half-sibling pairs from avuncular or grandparent-grandchildren pairs; and (2) assign individuals to ancestor versus descendant generation. We test our approach in a sample of ∼700 individuals from northern Namibia, sampled from an endogamous population called the Himba. Due to a culture of concurrent relationships in the Himba, there is a high proportion of half-sibships. We accurately identify first through fourth-degree relationships and distinguish between various second-degree relationships: half-sibships, avuncular pairs, and grandparent-grandchildren. We further validate our approach in a second African-descent dataset, the Barbados Asthma Genetics Study (BAGS), and a European-descent founder population from Quebec. Accurate reconstruction of relatives facilitates estimation of allele frequencies, tracing allele trajectories, improved phasing, heritability and other population genomic questions.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40393068/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Interpreting biochemical text with language models:a machine learning framework for reaction extraction and cheminformatic validation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lim, D., Badrinarayanan, S., Sterling, K. C., Rajesh, G., Mistry, E., Yang, D., Lee, M., Hsu, K. B., Manjrekar, M., Areff, C., Xie, P., Kristanto, I. A., Chandran, A., Anderson, J. C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present an integrated pipeline that enhances LLM-based extraction of enzymatic reactions with machine learning and cheminformatics-informed validation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recent advancements in large language models (LLMs) offer new opportunities for automating the manual curation of biochemical reaction databases from scientific literature. In this study, we present an integrated pipeline that enhances LLM-based extraction of enzymatic reactions with machine learning and cheminformatics-informed validation. Using BRENDA-linked PubMed articles, we evaluate GPT-4's ability to extract reactions and infer missing chemical entities in textual descriptions of enzymatic reactions. Extracted reactions are converted to SMILES and InChI notations before being encoded into molecular fingerprint similarity scores and atom mapping metrics. These cheminformatics metrics are then used to train machine learning classifiers that validate GPT extractions. We employ a Positive-Unlabeled learning approach with synthetic invalid reactions to train various classifiers and assess model performances. The best classifier is then benchmarked on GPT extractions. Our findings show that GPT can accurately infer incomplete reactions and cheminformatics tools can serve as effective predictors of reaction validity. This work demonstrates a scalable framework for automated and reliable curation of enzymatic reaction databases, highlighting the potential of combining LLMs with cheminformatics and machine learning for reliable scientific knowledge extraction.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.15.654376v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Rapid and accurate protein structure database search using inverse folding model and contrastive learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lyu, Q., Wei, H., Chen, S., Peng, Z., Yang, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce mTM-align2, a novel two-step approach for rapid and accurate protein structure database search.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Protein structure database search has become increasingly challenging due to the growing number of experimental and computational structures. We introduce mTM-align2, a novel two-step approach for rapid and accurate protein structure database search. In the first step, protein structures are first transformed into embeddings using a pre-trained inverse folding model (ESM-IF) and 3D Zernike polynomials. The ESM-IF embeddings are further optimized through a contrastive learning network, which is trained on ~7 million structure pairs. Structures with similar embeddings are returned on the fly in this step. The second step employs a rapid structure alignment program to refine top candidates, ensuring high precision and producing high-quality alignments. Extensive benchmarks reveal that mTM-align2 performs competitively compared to other leading methods, completing monomeric structure search in seconds with over 90% precision for the top 10 hits. The t-SNE visualization of the mTM-align2 embeddings for thousands of structures demonstrates that our embeddings are structurally informed, capturing the global structural features. It uncovers insights such as structure misclassifications and ambiguous structural class boundaries. A web server for mTM-align2 is accessible at https://yanglab.qd.sdu.edu.cn/mTM-align/.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.15.654382v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Benchmarking Spectral Library and Database Search Approaches for Metaproteomics Using a Ground-Truth Microbiome Dataset</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rajczewski, A. T., Mehta, S., Wagner, R., Gabriel, W., Johnson, J., Do, K., Vintila, S., Wilhelm, M., Kleiner, M., Searle, B. C., Griffin, T., Jagtap, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Using Scribe with Prosit predicted spectral libraries for mass spectrometry-based metaproteomics data analysis and using FASTA databases for protein sequence search .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mass spectrometry-based metaproteomics, the identification and quantification of thousands of proteins expressed by complex microbial communities, has become pivotal for unraveling functional interactions within microbiomes. However, metaproteomics data analysis encounters many challenges, including the search of tandem mass spectra against a protein sequence database using proteomics database search algorithms. We used a ground-truth dataset to assess a spectral library searching method against established database searching approaches. Mass spectrometry data collected by data-dependent acquisition (DDA-MS) was analyzed using database searching approaches (MaxQuant and FragPipe), as well as using Scribe with Prosit predicted spectral libraries. We used FASTA databases that included protein sequences from microbial species present in the ground-truth dataset along with background protein sequences, to estimate error rates and assess the effects on detection, peptide-spectral match quality, and quantification. Using the Scribe search engine resulted in more proteins detected at a 1% false discovery rate (FDR) compared to MaxQuant or FragPipe, while FragPipe detected more peptides verified by PepQuery. Scribe was able to detect more low-abundance proteins in the microbiome dataset and was more accurate in quantifying the microbial community composition. This research provides insights and guidance for metaproteomics researchers aiming to optimize results in their analysis of DDA-MS data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.15.654320v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multimodal data analysis reveals asynchronous aging dynamics across female reproductive organs</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Soldatkina, O., Ventura-San Pedro, L., El Hommad, A., Ramirez, J. M., Ripoll-Cladellas, A., Sopena-Rios, M., Ordi, J., Pujol-Gualdo, N., Mele, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that female reproductive organs and tissues have asynchronous aging dynamics: while the ovary and vagina age gradually, the uterus undergoes an abrupt transcriptional and cellular transition around the age of menopause.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Female reproductive aging is a complex process with profound systemic health implications, yet the molecular and structural dynamics of aging across reproductive organs and tissues remain largely unexplored. Here, we integrate deep learning-based analysis of 1,112 histological images with RNA-seq data from 659 samples across seven reproductive organs in 304 female donors aged 20-70 years. We show that female reproductive organs and tissues have asynchronous aging dynamics: while the ovary and vagina age gradually, the uterus undergoes an abrupt transcriptional and cellular transition around the age of menopause. Tissue segmentation highlights that the myometrium in the uterine wall is the most age-affected tissue, marked by extracellular matrix remodeling and immune activation. Across reproductive organs, the epithelial tissue is also strongly affected by age, with the vaginal epithelium showing a unique sharp menopausal transition. Integration via multi-omics factor analysis links these tissue-specific histological transformations to specific molecular shifts, many with nonlinear expression trajectories and enriched in heritable reproductive traits such as pelvic organ prolapse and age at menarche. These findings position menopause as a key inflection point in female aging and provide insights with tissue-specific focus to support healthier menopausal transitions and reduce age-related disease risk.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.16.654406v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Medical Images as Biomarkers of Ageing - From Global and Local Patterns to Digital Twins</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mueller, T. T., Starck, S., Llalloshi, R., Kaissis, G., Ziller, A., Rueckert, D., Braren, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We analyse ageing in different local body systems (liver, lungs, spine, intestine, muscle, heart), the brain, and across the whole body using 70,000 subjects from the UK Biobank population study.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding the process of ageing has become a highly desirable goal in human health and disease research. A multitude of factors impact the way we age and understanding their connection can help with early detection of age-related diseases and give insights into what causes abnormal ageing. In this study, we analyse how medical images can be used as biomarkers for ageing via deep learning techniques. We evaluate ageing in different local body systems (liver, lungs, spine, intestine, muscle, heart), the brain, and across the whole body using 70,000 subjects from the UK Biobank population study. We analyse correlations between lifestyle factors --such as smoking-- or diseases and accelerated ageing. Furthermore, we show differences in survival between subjects with accelerated and decelerated ageing and study the impact of individual body regions on the whole body age by generating a Digital Twin that can aid personalised medicine. Our work gives insights into how medical imaging can be used to identify abnormal ageing both on a local and a global scale and investigates the interplay between different body regions. It can help understand ageing patterns and lay a foundation for future applications of age predictors for risk assessment.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.18.654705v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A high-quality draft genome assembly of the Neotropical butterfly, Batesia hypochlora (Nymphalidae: Biblidinae)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pham, T. N., Duplouy, A., See, J., Knowles, L. S., Marquina, E., Gallice, G. R., Molleman, F., Oostra, V.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We report a long-read high-coverage reference genome assembly of the Neotropical butterfly Batesia hypochlora (Nymphalidae: Biblidinae).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We report a long-read high-coverage reference genome assembly of the Neotropical butterfly, Batesia hypochlora (Nymphalidae: Biblidinae). This represents the first reference genome in the Biblidinae subfamily, a clade subject to ongoing studies on seasonal and climate adaptation in the Amazon. We assembled the genome from PacBio HiFi long reads (66X coverage), polished it with Illumina short reads (15X coverage), and annotated it using PacBio IsoSeq RNA data. We observed 15 chromosome-sized scaffolds varying in length from 13.2 Mbp to 37.6 Mbp (median 24.3 Mbp), combining a total genome size of 395.788 Mbp. This assembly is highly contiguous (contig N50 of 25.14 Mbp) and complete (BUSCO completeness score of 98.6% and 0.2% duplication rate). Repeat annotation revealed that the genome consists of about one-third transposable elements. Gene prediction using RNAseq evidence uncovered 19,395 genes, of which 17,400 were assigned to 2,883 orthogroups when including genomes of the fruitfly, silk moth, and three other Nymphalid butterfly species. The high sequencing depth also allowed us to assemble the genomes of the mitochondria and the common endosymbiotic bacterium Wolbachia. The mitochondrial genome was fully assembled (15,540 bp in size) with all expected genes annotated. The Wolbachia genome was fragmented, and we determined that it belongs to the B-supergroup. The high-quality assembly of B. hypochlora can represent the subfamily in further comparative analysis of evolution and provide a key resource for ongoing work to explore reproductive biology and adaptations to seasonality in Amazonian butterflies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.05.17.654659v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Decoding cancer prognosis with deep learning: the ASD-cancer framework for tumor microenvironment analysis.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ziyuan Huang, Yunzhan Li, Vanni Bucci, John P Haran</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Autoencoder-Based Subtypes Detector for Cancer (ASD-cancer) is a semi-supervised learning framework for multi-omics data analysis that can improve the cancer subtyping analysis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Deep learning is revolutionizing biomedical research by facilitating the integration of multi-omics data sets while bridging classical bioinformatics with existing knowledge. Building on this powerful potential, Zhang et al. proposed a semi-supervised learning framework called Autoencoder-Based Subtypes Detector for Cancer (ASD-cancer) to improve the multi-omics data analysis (H. Zhang, X. Xiong, M. Cheng, et al., 2024, mSystems 9:e01395-24, https://doi.org/10.1128/msystems.01395-24). By utilizing autoencoders pre-trained on The Cancer Genome Atlas data, the ASD-cancer framework outperforms the baseline model. This approach also makes the framework scalable, enabling it to process new data sets through transfer learning without retraining. This commentary explores the methodological innovations and scalability of ASD-cancer while suggesting future directions, such as the incorporation of additional data layers and the development of adaptive AI models through continuous learning. Notably, integrating large language models into ASD-cancer could enhance its interpretability, providing more profound insights into oncological research and increasing its influence in cancer subtyping and further analysis.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40237527/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

