<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Tuesday, 17 June 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The dawn of a new era: can machine learning and large language models reshape QSP modeling?</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ioannis P Androulakis, Lourdes Cucurull-Sanchez, Anna Kondic, Krina Mehta, Cesar Pichardo, Meghan Pryor, Marissa Renardy</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We explore the transformative role of AI and ML in reshaping QSP modeling workflows and propose a pathway toward a transformative era for QSP, leveraging advancements in AI andML to address these challenges and drive innovation in the field.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Quantitative Systems Pharmacology (QSP) has emerged as a cornerstone of modern drug development, providing a robust framework to integrate data from preclinical and clinical studies, enhance decision-making, and optimize therapeutic strategies. By modeling biological systems and drug interactions, QSP enables predictions of outcomes, optimization of dosing regimens, and personalized medicine applications. Recent advancements in artificial intelligence (AI) and machine learning (ML) hold the potential to significantly transform QSP by enabling enhanced data extraction, fostering the development of hybrid mechanistic ML models, and supporting the introduction of surrogate models and digital twins. This manuscript explores the transformative role of AI and ML in reshaping QSP modeling workflows. AI/ML tools now enable automated literature mining, the generation of dynamic models from data, and the creation of hybrid frameworks that blend mechanistic insights with data-driven approaches. Large Language Models (LLMs) further revolutionize the field by transitioning AI/ML from merely a tool to becoming an active partner in QSP modeling. By facilitating interdisciplinary collaboration, lowering barriers to entry, and democratizing QSP workflows, LLMs empower researchers without deep coding expertise to engage in complex modeling tasks. Additionally, the integration of Artificial General Intelligence (AGI) holds the potential to autonomously propose, refine, and validate models, further accelerating innovation across multiscale biological processes. Key challenges remain in integrating AI/ML into QSP workflows, particularly in ensuring rigorous validation pipelines, addressing ethical considerations, and establishing robust regulatory frameworks to address the reliability and reproducibility of AI-assisted models. Moreover, the complexity of multiscale biological integration, effective data management, and fostering interdisciplinary collaboration present ongoing hurdles. Despite these challenges, the potential of AI/ML to enhance hybrid model development, improve model interpretability, and democratize QSP modeling offers an exciting opportunity to revolutionize drug development and therapeutic innovation. This work highlights a pathway toward a transformative era for QSP, leveraging advancements in AI and ML to address these challenges and drive innovation in the field.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40524056/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Automated model refinement using perturbation-observation pairs.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kyu Hyong Park, Jordan C Rozum, Réka Albert</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a genetic algorithm-based workflow for Boolean model refinement that automates the laborious task of model validation and refinement.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In modeling signal transduction networks, it is common to manually integrate experimental evidence through a process that involves trial and error constrained by domain knowledge. We implement a genetic algorithm-based workflow (boolmore) to streamline Boolean model refinement. Boolmore adjusts the functions of the model to enhance agreement with a corpus of curated perturbation-observation pairs. It leverages existing mechanistic knowledge to automatically limit the search space to biologically plausible models. We demonstrate boolmore's effectiveness in a published plant signaling model that exemplifies the challenges of manual model construction and refinement. The refined models surpass the accuracy gain achieved over two years of manual revision and yield new, testable predictions. By automating the laborious task of model validation and refinement, this workflow is a step towards fast, fully automated, and reliable model construction.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40523884/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The gene regulatory effects of selective glucocorticoid receptor ligands</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Giroux, N. S., Johnson, G. D., Barrera, A. E., Reddy, T. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the GC ligands we tested do not selectively target specific subsets of the molecular GC response, and that the responses of each ligand alone can explain 93% of the variation in gene expression and gene regulatory element activity.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Synthetic glucocorticoids (GCs), which induce the transcription factor activity of the glucocorticoid receptor (GR), are frequently prescribed anti-inflammatory therapeutics that have been in use for over 70 years. Despite their broad immunosuppressive utility, sustained use of GCs is often intolerable due to the prevalence of adverse side effects. A longstanding goal has been to make synthetic GCs safer by developing selective GR ligands that have similar anti-inflammatory activity but without the burden of side effects. To evaluate the ability of synthetic GCs to target specific subsets of the GC response, we completed a genome-wide comparative analysis of changes in gene expression and gene regulatory element activity in response to ten ligands with various evidence of dissociated adverse side effects. We measured the gene expression response using mRNA-seq and the gene regulatory element response using genome-wide STARR-seq. Effects associated with each ligand were highly correlated with and linearly related to the response to dexamethasone, a strong, non-selective GR agonist used as a positive control for this study. Furthermore, 93% of the variation in regulatory element activity responses could be explained by the efficacy of each ligand alone. We also found limited evidence of differential enrichment of chromatin context-specific markers of regulatory activity with each ligand. Based on those findings, we developed a simulation framework to evaluate selectivity of GR ligands. We conclude that the ligands we tested elicit attenuated molecular responses according to their respective efficacies, and do not selectively target subsets of the molecular GC response.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.16.659962v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">LazyNet: Interpretable ODE Modeling of Sparse CRISPR Single-Cell Screens Reveals New Biological Insights</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yi, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce LazyNet, an explicitly Euler-integrated neural ODE whose paired log-linear-exp layer collapses multiplicative transcript interactions into a compact, mechanistically interpretable weight matrix.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell CRISPR activation/interference screens offer a direct route to causal gene-regulatory maps, yet existing deep-learning pipelines are GPU-intensive and yield hard-to-interpret latent factors. We introduce LazyNet, an explicitly Euler-integrated neural ODE whose paired log-linear-exp layer collapses multiplicative transcript interactions into a compact, mechanistically interpretable weight matrix. Training a three-replica ensemble on a 55k-cell, 30k-gene Perturb-seq dataset completes on a single CPU in <1 h, running 3 to 4 folder faster than transformer (scGPT) or state-space (RetNet) baselines while lowering global RMSE by 25 % and raising genome-wide Pearson r to 0.67. Averaged Jacobians, expanded in a 32*4 breadth-first search around seven ferroptosis seeds, recapitulated 15 of 27 benchmark regulators (56 % recall) within a 4 611 gene, 11 676 edge subgraph; 26.6 % of edges show ARCHS4 co-expression r [≥] 0.2 versus 5 % expected at random, and 523 overlap STRING interactions (hypergeometric p = 1.2e-5). Elasticity ranks uncover a previously unrecognized lysosomal-mitochondrial-immune module linking PSAP-mTOR, MFN2-TLR4 and ADCY10-SIRT3, generating experimentally testable hypotheses. By combining state-of-the-art predictive accuracy, laptop-level resource demands and one-to-one parameter interpretability, LazyNet democratizes causal network discovery from sparse two-snapshot screens, enabling small laboratories to move from large-scale perturbation data to mechanistic insight without GPUs or external pathway priors.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.11.658833v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Anthracyclines induce global changes in cardiomyocyte chromatin accessibility that overlap with cardiovascular disease loci</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Matthews, E. R., Abodunrin, R. O., Hurley, J. D., Gutierrez, J. A., Ward, M. C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We measure chromatin accessibility across iPSC-derived cardiomyocytes from four individuals treated with the ACs, Doxorubicin, Epirubicins, and Daunorubics, and show large-scale changes in Chromatin accessibility, including regions associated with</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Breast cancer drugs including anthracyclines (ACs) and Trastuzumab increase the risk for cardiovascular diseases (CVDs) such as atrial fibrillation (AF) and heart failure (HF) that ultimately affect the heart muscle. These CVDs are associated with hundreds of genetic variants in non-coding regions of the genome. However, how these drugs affect the regulatory potential of the non-coding genome of the heart and CVD risk loci is unknown. We therefore measured global chromatin accessibility across iPSC-derived cardiomyocytes from four individuals treated with the ACs, Doxorubicin, Epirubicin, and Daunorubicin, a related non-AC, Mitoxantrone, and the monoclonal antibody Trastuzumab, or a vehicle control for three and 24 hours. We identified 155,557 high-confidence regions of open chromatin across 48 samples where the major sources of variation are associated with drug type and time. Jointly modeling the data revealed three accessibility response signatures denoted as early-acute, early-sustained, and late that correspond to 67,329 regions that open or close in response to drug treatment. Sequences associated with drug-induced chromatin opening contain motifs for DNA damage-associated transcription factors including p53 and ZBTB14, and associate with increases in active histone acetylation and gene expression. 21 AF- and HF-associated SNPs directly overlap with regions associated with drug-induced opening. A shared intronic HF and AF SNP, rs3176326, that is also an eQTL for CDKN1A in heart tissue, associates with increased chromatin accessibility, histone acetylation, and CDKN1A expression in response to all ACs. Our results demonstrate large-scale changes in chromatin accessibility in cardiomyocytes treated with ACs, which correspond to several regions harboring CVD risk loci.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.11.658997v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The genetic control of rapid genome content divergence in Arabidopsis thaliana</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fiscus, C. J., Koenig, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify hypervariable regions that influence the rate of genome content evolution and purifying selection in plants.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genome evolution in eukaryotes is predominantly driven by the dynamics of repetitive sequences, which vary widely in both copy number and sequence composition. The rate of repeat evolution changes between species and within a species and is likely modulated by both genetics and environment. To uncover the factors shaping the rate of genome content evolution, we analyzed 1,142 resequenced Arabidopsis thaliana genomes using a novel K-mer based approach. With this dataset, we characterized genome content variation and identified hypervariable regions that contribute to major differences in repeat abundance. We then treated repeat abundance as a quantitative trait and performed genome-wide association studies to map the genetic basis of copy number variation across more than 400 repeat families. We jointly analyzed these results using a meta-GWAS approach, revealing both cis-acting variants and over 50 trans-acting loci that regulate repeat abundance genome-wide. Finally, we found that purifying selection acts against mutations that increase the rate of genome content divergence, favoring alleles that limit repeat expansion. Together, our results provide new insights into the genetic architecture and evolutionary forces shaping genome evolution in plants.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.11.659220v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">X-Atlas/Orion: Genome-wide Perturb-seq Datasets via a Scalable Fix-Cryopreserve Platform for Training Dose-Dependent Biological Foundation Models</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Huang, A. C., Hsieh, T.-H. S., Zhu, J., Michuda, J., Teng, A., Kim, S., Rumsey, E. M., Lam, S. K., Anigbogu, I., Wright, P., Ameen, M., You, K., Graves, C. J., Kim, H. J., Litterman, A. J., Sit, R. V., Blocker, A., Chu, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce "Fix-Cryopreserve-ScRNAseq" Perturb-seq, an industrialized platform for scalable Perturbed-seq data generation, and release X-Atlas: Orion edition (X-At Atlas/Orion), the largest publicly available Perturbing</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The rapid expansion of massively parallel sequencing technologies has enabled the development of foundation models to uncover novel biological findings. While these have the potential to significantly accelerate scientific discoveries by creating AI-driven virtual cell models, their progress has been greatly limited by the lack of large-scale high-quality perturbation data, which remains constrained due to scalability bottlenecks and assay variability. Here, we introduce "Fix-Cryopreserve-ScRNAseq" (FiCS) Perturb-seq, an industrialized platform for scalable Perturb-seq data generation. We demonstrate that FiCS Perturb-seq exhibits high sensitivity and low batch effects, effectively capturing perturbation-induced transcriptomic changes and recapitulating known biological pathways and protein complexes. In addition, we release X-Atlas: Orion edition (X-Atlas/Orion), the largest publicly available Perturb-seq atlas. This atlas, generated from two genome-wide FiCS Perturb-seq experiments targeting all human protein-coding genes, comprises eight million cells deeply sequenced to over 16,000 unique molecular identifiers (UMIs) per cell. Furthermore, we show that single guide RNA (sgRNA) abundance can serve as a proxy for gene knockdown (KD) efficacy. Leveraging the deep sequencing and substantial cell numbers per perturbation, we also show that stratification by sgRNA expression can reveal dose-dependent genetic effects. Taken together, we demonstrate that FiCS Perturb-seq is an efficient and scalable platform for high-throughput Perturb-seq screens. Through the release of X-Atlas/Orion, we highlight the potential of FiCS Perturb-seq to address current scalability and variability challenges in data generation, advance foundation model development that incorporates gene-dosage effects, and accelerate biological discoveries.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.11.659105v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Nanorate sequencing reveals the Arabidopsis somatic mutation landscape</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Meyer, C. A., Nelms, B., Schmitz, R. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Somatic mutations in Arabidopsis plants show higher mutation rates than germline mutations, and are enriched in transposable elements and depleted in genes, but this depletion is greater in somatic than in germline.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The rate and spectrum of somatic mutations can diverge from that of germline mutations. This is because somatic tissues experience different mutagenic processes than germline tissues. Here, we use nanorate sequencing (NanoSeq) to identify somatic mutations in Arabidopsis shoots with high sensitivity. We report a somatic mutation rate of 3.6x10^-8 mutations/bp, ~4-5x the germline mutation rate. Somatic mutations displayed elevated signatures consistent with oxidative damage, UV damage, and transcription-coupled nucleotide excision repair. Both somatic and germline mutations were enriched in transposable elements and depleted in genes, but this depletion was greater in germline mutations. Somatic mutation rate correlated with proximity to the centromere, DNA methylation, chromatin accessibility, and gene/TE content, properties which were also largely true of germline mutations. We note DNA methylation and chromatin accessibility have different predicted effects on mutation rate for genic and non-genic regions; DNA methylation associates with a greater increase in mutation rate when in non-genic regions, and accessible chromatin associates with a lower mutation rate in non-genic regions but a higher mutation rate in genic regions. Together, these results characterize key differences and similarities in the genomic distribution of somatic and germline mutations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.15.659769v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">S3Mirror: Making Genomic Data Transfers Fast, Reliable, and Observable with DBOS</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Vasquez-Grinnell, S., Poliakov, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> S3Mirror is an open source DBOS Python application that can run in a variety of environments, including DBOS Cloud Pro where it runs as much as 40x faster than AWS DataSync.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> To meet the needs of a large pharmaceutical organization, we set out to create S3Mirror - an application for transferring large genomic sequencing datasets between S3 buckets quickly, reliably, and observably. We used the DBOS-Transact durable execution framework to achieve these goals and benchmarked the performance and cost of the application. S3Mirror is an open source DBOS Python application that can run in a variety of environments, including DBOS Cloud Pro where it runs as much as 40x faster than AWS DataSync at a fraction of the cost. Moreover, S3Mirror is resilient to failures and allows for real-time file-wise observability of ongoing and past transfers.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.13.657723v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Protocol for GDT, Gene Dictionary Tool, to create and implement a gene dictionary across annotated genomes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dupin, B., Lima, M. S., Paschoal, A. R., Smith, D. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> GDT - Gene Dictionary Tool - is a protocol for the creation and implementation of a gene dictionary across any type of annotated genomes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> GDT - Gene Dictionary Tool - is a protocol for the creation and implementation of a gene dictionary across any type of annotated genomes. The final output, a gene dictionary (.gdt file), is meant to be not only comprehensive, but also versionable, Findable, Accessible, Interoperable, and Reusable (FAIR). The protocol is divided in a Python library and Jupyter notebooks that permit users to investigate their datasets incrementally. Here, we showcase GDTs functionalities by creating a gene dictionary of organelle genes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.15.659783v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ChemOrigins: A community curated database for origins of life chemistry</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cuevas, B., Adam, Z. R., Cove, D., Kacar, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> An open-access, community-curated knowledge graph that organizes experimentally supported prebiotic reactions and supports complex, query-driven exploration via a graph database architecture.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The origin of life is one of the most compelling questions in science. While experimental prebiotic chemistry has produced a wide range of reactions and plausible pathways, the resulting data remain fragmented across numerous publications and disciplinary journals. Here, we introduce ChemOrigins, an open-access, community-curated knowledge graph that organizes experimentally supported prebiotic reactions. By representing molecules, reactions, conditions, and literature sources as interconnected nodes, ChemOrigins enables modular grouping of reactions and supports complex, query-driven exploration via a graph database architecture. We demonstrate the utility of this framework through text-based searches, reaction network expansions, and the interactive visualization of user-annotated chemical modules. Unlike generative models, ChemOrigins prioritizes curated, evidence-based content and fosters community contributions through expert annotations and a user-friendly interface. As a structured resource, ChemOrigins is designed to complement existing chemical databases and serve as a foundation for computational, educational, and theoretical research in the origins-of-life field.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.10.658922v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">D-CoMEx: A Unified Approach to Identify Phenotype-specific miRNA Biomarkers Related to Diseases</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kakati, T., Bhattacharyya, D. K., Kalita, J. K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a unified approach, D-CoMEx, to mine miRNA biomarkers from dysregulated pathways associated with Parkinson's disease (PD) and Breast Cancer (BC).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Profiling of miRNAs is important to understand how they regulate biological pathways and contribute to diseases. With current technological advancements leading to capture of voluminous biological data, it is interesting and challenging to explore new methods that can discover disease-data associations and provide better insights into the underlying mechanisms of diseases. Such understanding may also help prevent severe pathogenesis of the disease and contribute to knowledge required for effective drug development. This paper proposes a unified approach, D-CoMEx, to mine miRNA biomarkers from dysregulated pathways associated with Parkinson's disease (PD) and Breast Cancer (BC). The approach integrates a co-expression network (CEN) module extraction technique with a differential analysis method to capture interactions between miRNAs and dysregulated pathways. Our method identifies more statistically and biologically significant dysregulated pathways compared to DCGL, MODA, and DiffCoEx. We present three evidence-backed recommendations supporting strong associations between miRNAs and dysregulated pathways, demonstrating that D-CoMEx extracts biologically relevant miRNA biomarkers linked to disease progression.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.10.658948v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">FvFT1-FvTFL1 epistasis drives flowering time adaptation in woodland strawberry</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lembinen, S., Koskela, E., Fan, G., Toivainen, T., Zhou, Q., Andres, J., Vaattovaara, A., Rastas, P. M. A., De Kort, H., Elomaa, P., Hytonen, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of flowering time variation in Fragaria vesca, a perennial herb from the Rosaceae family, and the role of FvFT1, a strawberry ortholog of FLOWERING LOCUS T, in the timing of floral transition and subsequent flowering.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Floral transition and flower bud development during autumn and winter are common in plants inhabiting temperate and cold climates. However, our understanding of the natural variation and underlying molecular mechanisms of such life-history strategy remain largely limited to Brassicaceae model species. To address this gap, we investigated flowering time variation across a broad geographical and bioclimatic range in Fragaria vesca (woodland strawberry), a perennial herb from the Rosaceae family. We found that the timing of floral transition and subsequent flowering in F. vesca populations is primarily shaped by population structure, reflecting postglacial colonization patterns from distinct refugia, and is further modulated by adaptations to winter temperatures. The association of increasing expression of FvFT1, the strawberry ortholog of FLOWERING LOCUS T, with delayed flowering in accessions from diverse climatic and genetic backgrounds suggested FvFT1 as a major candidate gene regulating flowering time variation. We further demonstrated how distinct florigen and antiflorigen roles of FvFT1 are dependent on functional alleles of F. vesca TERMINAL FLOWER1 (FvTFL1) and how the epistatic relationship between FvFT1 and FvTFL1 controls adaptive flowering time variation by adjusting sensitivity to floral inductive photoperiod and vernalization. Our findings reveal a major molecular basis of how timing of floral transition has adapted to temperature in a Rosaceae model inhabiting multiple climatic zones and highlight lineage-specific adaptive mechanisms underlying convergent life-history strategy.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.11.659068v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A canonical chloroplast unfolded protein response triggered by misfolded polypeptides</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cantoia, A., Bertero, F., Berrocal, R., Ceccarelli, E. A., Blanco, N. E., Rosano, G. L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that chloroplastic protein homeostasis is a specific stress response in chloroplasts and provide new tools for its characterization.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Photosynthetic efficiency and plant viability rely on chloroplast protein homeostasis. While unfolded protein responses (UPRs) in the endoplasmic reticulum and mitochondria have been extensively characterized, the chloroplast UPR (cpUPR) remains less defined, partly due to the off-target effects of traditional stress-inducing methods. In this study, we provide direct evidence for the existence of the cpUPR by expressing engineered, folding-defective variants of ferredoxin-NADP reductase (FNR) in plant chloroplasts. The expression of aggregation-prone proteins inside the organelle triggered a robust upregulation of chloroplast quality control components, including CLPB3, CLPC1/C2, and HSP90C, as revealed by immunodetection and quantitative proteomics. The proteomic response scaled with the severity of the folding defect, with the fully insoluble FNR {Delta}20 variant inducing broader changes than the partially soluble {Delta}3 variant. Network analysis showed that most differentially abundant proteins were chloroplastic and clustered into functional groups related to proteostasis and photosynthesis. Comparative analysis with lincomycin-treated plants highlighted the specificity and advantages of using misfolded proteins to study the cpUPR. Expression of the folding variants conferred tolerance to heat treatment, suggesting that the response can enhance plant fitness. Taken together, our work establishes the cpUPR as a specific stress response in chloroplasts and provides new tools for its characterization.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.11.659047v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Causes and consequences of experimental variation in Nicotiana benthamiana transient expression</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tang, S. N., Szarzanowicz, M., Lanctot, A., Sirirungruang, S., Kirkpatrick, L. D., Drako, K., Alamos, S., Cheng, A., Waldburger, L. M., Liu, S., Thompson, M., Shih, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comprehensive analysis of N. benthamiana agroinfiltration experiments, which reveals sources of variability that affect transient expression and provides practical guidelines for minimizing variability through strategic experimental design.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Infilitration of Agrobacterium tumefaciens into Nicotiana benthamiana has become a foundational technique in plant biology, enabling efficient delivery of transgenes in planta with technical ease, robust signal, and relatively high throughput. Despite the prevalence of transient expression in disciplines such as synthetic biology, little work has been done to describe and address the variability inherent in this system, a concern for experiments that rely on highly quantitative readouts. In a comprehensive analysis of N. benthamiana agroinfiltration experiments, we model sources of variability that affect transient expression. Our findings emphasize the need to validate normalization methods under the specific conditions of each study, as distinct normalization schemes do not always reduce variation either within or between experiments. Using a dataset of 1,813 plants collected over three years, we develop a comprehensive model of variation in N. benthamiana transient expression, using power analysis to determine the number of individual plants required for a given effect size. Drawing on our longitudinal data, these findings inform practical guidelines for minimizing variability through strategic experimental design and power analysis, providing a foundation for more robust and reproducible use of N. benthamiana in quantitative plant biology and synthetic biology applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.12.659391v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Single-Nucleus Transcriptomics Reveals How Tissue Context Shapes the Circadian Transcriptome of the Arabidopsis Leaf</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zulfiqar, A., Myers, Z. A., Menon, A., Greenham, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified over 7,400 genes with cluster-resolved circadian regulation, and used coexpression analysis to define five major temporal expression clades shared across all cell types.</p><p style="font-size: 16px; text-align: justify;">Background: Circadian regulation enables plants to coordinate cellular processes with daily environmental cycles, yet the dynamics and hierarchy of the clock across cells and tissue types remains poorly understood. Results: We developed a 24 hour single nucleus RNA-sequencing (snRNA-seq) circadian time course of the mature Arabidopsis thaliana leaf to characterize circadian regulation across tissues. We captured ~30,000 nuclei across seven circadian time points, recovering all major leaf cell types. We identified over 7,400 genes with cluster-resolved circadian regulation, and used coexpression analysis to define five major temporal expression clades shared across all cell types. We leveraged these assignments to identify genes with cell type-specific temporal shifts in expression. Single cell gene regulatory networks (scGRNs) were generated for each cluster through GENIE3, identifying many shared and unique transcription factor (TF) target interactions across clusters. A close examination of CCA1 targets identified Photosystem II repair components in a subset of mesophyll cells and a large number of auxin-associated genes in only a handful of clusters despite robust expression in other clusters, suggestive of cell type-specific CCA1 target regulation. Conclusions: Our results demonstrate the extent to which circadian transcriptional regulation is present in the mature Arabidopsis leaf, and highlight the immense complexity in cell type specific regulation. This dataset provides a high-resolution resource for dissecting circadian gene regulation at cellular resolution and advances our understanding of tissue-specific clock dynamics in plants.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.12.659411v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Establishing betaxanthin pigment biosynthesis in cyanobacteria</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hanamghar, S. S., Russo, D. A., Mellor, S. B., Zedler, J. A. Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We have developed a photoautotrophic pathway for the production of betaxanthin pigment production without the need for exogenous amino acid supplementation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Betalains are water-soluble pigments with two major classes: red-violet betacyanins and yellow-orange betaxanthins. These pigments are increasingly being sought after as natural replacements for synthetic pigments in the food industry. Traditionally, betalains are extracted from cultivated plants but, due to low native pigments, the process is inherently inefficient. Now, as consumer demand increases, scalable and sustainable production routes are needed. To address this challenge, we engineered a heterologous pathway for the production of betaxanthins into cyanobacteria. The pathway consists of the cytochrome P450 CYP76AD1 and the L-DOPA 4,5-dioxygenase DODA1 from Beta vulgaris (beet). Introduction of the two-enzyme betaxanthin pathway in Synechocystis sp. PCC 6803 did not result in detectable betaxanthins. Subsequent metabolic adjustments to the shikimate pathway, using a feedback resistant AroGfbr from E. coli, led to an overaccumulation of the aromatic amino acids phenylalanine, tryptophan, and tyrosine, and the production of 0.11 mg L-1 phenylalanine-betaxanthin. Optimization of the cultivation conditions (i.e., growth in nutrient-rich medium and CO-1-enriched air) increased titers almost 150 times and led to the production of phenylalanine-betaxanthin with a final titer of 15.3 mg L-1. Our work establishes a microbial system for photoautotrophic betaxanthin pigment production without the need for exogenous amino acid supplementation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.16.659934v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Pathway Sculptor for Compact and Versatile Combinatorial Genetic Perturbation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gu, B., Linton, J., Hendrickson, B. G., Li, H., Hadas, R., Manella, G., Gregrowicz, J., Elowitz, M. B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We develop a programmable Cas12a-based CRIPSRi system capable of strong, tunable, and simultaneous knockdown of six or more genes in a single cell without genomic DNA cleavage.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The ability to perturb multiple proteins simultaneously within the same cell is essential for understanding and re-engineering biological pathways. CRISPR-Cas12a mutants with inactivated DNAse but intact RNAse activity (dCas12a) retain the ability to process large CRISPR RNAs (crRNAs) arrays, enabling them to target multiple genomic loci in parallel. When coupled with transcriptional effector domains, these properties make Cas12a a promising platform for multi-locus transcriptional perturbation. However, current Cas12a-based CRISPRi systems exhibit limitations in processing of multi-crRNA arrays and transcriptional regulation. Here, we combine molecular and circuit-level engineering to develop a programmable Cas12a-based CRIPSRi system capable of strong, tunable, and simultaneous knockdown of six or more genes in a single cell without genomic DNA cleavage. We demonstrate the utility of this system by systematically perturbing a partially redundant set of Bone Morphogenetic Protein (BMP) receptors, enabling quantitative analysis of BMP signaling across diverse receptor configurations.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.15.659618v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A synthetic cell phage cycle</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Levrier, A., Soudier, P., Garenne, D., Izri, Z., Bowden, S. D., Lindner, A. B., Noireaux, V.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We establish an all-cell-free viral cycle where T7 phages infect synthetic cells, equipped with lipopolysaccharides on the outer leaflet of the lipid membrane while encapsulating a cell-free gene expression system.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Viral infection of living cells, exemplified by bacteriophage interaction with bacteria, is fundamental to biology and universal across living systems. Here, we establish an all-cell-free viral cycle where T7 phages infect synthetic cells, equipped with lipopolysaccharides on the outer leaflet of the lipid membrane while encapsulating a cell-free gene expression system. We track each cycle step to demonstrate T7 phage-specific adsorption onto the liposomes, genome ejection, replication, expression, and assembly of new infectious virions within the synthetic cells. We quantify key characteristics of the cycle, including the multiplicity of infection, replication efficiency, liposome size constraints, and phage rebinding dynamics. This work establishes a versatile, fully defined in vitro platform for reconstructing and investigating viral infections from individual molecular components.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.14.659709v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Synthetic autotrophic yeast enables high itaconic acid production from CO2 via integrated pathway and process design</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ata, O., Lutz, L., Baumschabl, M., Mattanovich, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a synthetic autotrophic Komagataella phaffii that is able to produce itaconic acid by the direct conversion of CO2, achieving final titers of approximately 12 g L-1 in bioreactor cultivations.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single carbon (C1) substrates are gaining importance as future feedstocks for the production of bio-based chemicals. Carbon dioxide, a major greenhouse gas, offers a promising alternative to the traditional feedstocks to shift towards C1-based, sustainable processes. Here, we present a synthetic autotrophic Komagataella phaffii (Pichia pastoris) that is able to produce itaconic acid by the direct conversion of CO2, achieving final titers of approximately 12 g L-1 in bioreactor cultivations. We show that a combined approach that integrates balancing the flux between the Calvin-Benson-Bassham (CBB) cycle and itaconic acid metabolism with process design was essential to enhance the production. Our study demonstrates the potential of K. phaffii as a microbial platform using CO2 as the direct carbon source, aligning with the future goals of establishing sustainable bioprocesses.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.16.659842v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CLUES2 Companion: Computational pipelines to estimate, visualize, and date selection on multi-locus sites</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lisi, A., Campbell, M. C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the CLUES2 Companion package, which contains user-friendly pipelines that seamlessly apply Relate to multiple sites within a target genomic region and then execute the CLues2 software to estimate selection coefficients for these sites.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Statistical methods that quantify the selection coefficient associated with alleles provide critical insights into the evolutionary processes underlying organismal adaptation. Among these approaches, CLUES2 was recently developed to estimate selection coefficients for alleles using a statistical framework that captures the maximum amount of information present in genomic data, making it a state-of-the-art method for identifying variation under selection. However, before executing this approach, users first need to apply Relate, a genealogy-based approach, to their data to generate the input files for CLUES2. Moreover, completing this pre-processing step inherently assumes that users have sufficient expertise to successfully run the Relate software. Here, we present the CLUES2 Companion package, which contains user-friendly pipelines that seamlessly apply Relate to multiple sites within a target genomic region and then execute the CLUES2 software to estimate selection coefficients for these sites. CLUES2 Companion also has the capability to present the output of CLUES2 analyses in tabular and graphical formats. In addition, as a new feature, we adapted Relate and CLUES2 to estimate the age of onset of a selective sweep of derived variation, expanding the functionality of our package. Results: To demonstrate the utility of our approach, we applied CLUES2 Companion to polymorphisms in the MCM6 gene on Chromosome 2 (including the known variants associated with lactase persistence) in the European Finnish, Middle Eastern Bedouin, and East African Maasai populations from the 1000 Genomes Project, the Human Genome Diversity Project (HGDP), and the haplotype map (HapMap) Project Phase 3, respectively. Our analyses uncovered significant selection coefficient estimates at the persistence-associated T-13910 allele (rs4988235; s = 0.09986, CI: 0.08678 - 0.11294) in the Finnish, the G-13915 allele (rs41380347; s = 0.09981, CI: 0.06515 - 0.13448) in the Bedouin, and the C-14010 allele (rs145946881; s = 0.09981, CI: 0.08799 - 0.11163) in the Maasai, indicative of a classic selective sweep. Furthermore, we inferred the age of onset of selection at these alleles to be 9,100 years ago (CI: 6,552 - 10,612 years ago) in the Finnish, 7,700 years ago (CI: 1,864 - 8,064 years ago) in the Bedouin, and 4,900 years ago (CI: 3,864 - 5,936 years ago) in the Maasai, respectively, which coincide well with other estimates based on genetic and archaeological data. To further validate our dating method, we simulated several datasets containing SNPs with known ages of onset of selection, s estimates, and genomic positions using a selective sweep framework implemented in msprime and then applied CLUES2 Companion to the simulated datasets. Using this approach, CLUES2 Companion produced similar estimates of selection onset as the ones specified in the simulations, corroborating the dependability of our method. Overall, CLUES2 Companion is a versatile package that enables users to efficiently explore, interpret, and report evidence of selection in genomic datasets, complementing the CLUES2 software.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.06.10.658975v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

