<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Friday, 01 August 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A robust machine learning model based on ribosomal-subunit-derived piRNAs for diagnostic potential of nonsmall cell lung cancer across multicentre, large-scale of sequencing data.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gao Z, Nasu M, Devendra G, Abdul-Ghani AA, Herrera AJ, Borgia JA, Seder CW, Kuehu DL, Feng Z, Chen Y, Gong T, Zhang Z, Chan O, Yang H, Yu J, Fu Y, Wu L, Deng Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a new machine learning model by integrating PIWI-interacting RNA (piRNA) to predict NSCLC and showed efficacy in identifying early-stage cancers with Area under the ROC Curve (AUC) values over .80.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Nonsmall cell lung cancer (NSCLC) is a lethal cancer and lacks robust biomarkers for noninvasive clinical diagnosis. Detecting NSCLC at the early stage can decrease the mortality rate and minimise harm caused by various treatments. We curated 2050 samples from public tissue and plasma datasets including both invasive and noninvasive types, then supplemented with in-house pooled plasma and exosome samples. Eleven independent transcriptome datasets were utilised to develop a new machine learning model by integrating PIWI-interacting RNA (piRNA) to predict NSCLC. Five piRNA signatures derived from ribosomal subunits identified to be tumour-specific exhibited robust diagnostic ability and were combined into a piRNA-Based Tumour Probability Index (pi-TPI) risk evaluation model. pi-TPI effectively distinguished NSCLC patients from healthy individuals and showed efficacy in identifying early-stage cancers with Area under the ROC Curve (AUC) values over .80. Plasma cohorts exhibited the diagnosis efficacy of pi-TPI with an AUC value of .85. Experimental exosomal data enhances the accuracy of diagnosing noncancerous, benign, and cancer cases. The pi-TPI marker in the noncancer/cancer subgroup exhibited superior predictive performance with an AUC value of .96. These findings underscore the significant clinical potential of the five piRNA signatures as a powerful diagnostic tool for NSCLC, particularly of noninvasive cancer diagnostics.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40714929/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Analyzing the Challenges and Opportunities Associated With Harnessing New Antibiotics From the Fungal Microbiome.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hossain MS, Amin MA, Islam S, Imam H, Das LC, Mahmud S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We explored historical developments and advanced genetic techniques to reveal the potential of fungi in antibiotic production and propose new approaches to overcome current obstacles and advance the development of antibiotic production technologies.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The rapid rise in antibiotic resistance is a critical global health issue, and few new classes of antibiotics have been discovered since 1990 compared to the antibiotic's golden era between 1950 and 1970. However, developing new antimicrobial compounds faces many challenges, improvements in cultivation methods, genetic engineering, and advanced technologies are opening new paths for discovering and producing effective antibiotics. This study focuses on the fungal microbiome as a promising source of new antibiotics. We explored historical developments and advanced genetic techniques to reveal the potential of fungi in antibiotic production. Although isolating and scaling up fungal antibiotic production presents challenges, innovative approaches like in situ separation during fermentation can effectively address these issues. Our research highlights the importance of understanding fungal communication and metabolite sharing to enhance antibiotic yields and the connection of cutting-edge technologies in accelerating the discovery and optimization of antibiotic-producing fungi. By focusing on these technical aspects and fostering teamwork across various fields, this study aims to overcome current obstacles, and advance the development of antibiotic production technologies.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40698515/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Discrimination of Klebsiella pneumoniae and Klebsiella quasipneumoniae by MALDI-TOF Mass Spectrometry Coupled With Machine Learning.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Nishikawa M, Tang W, Kostrzewa M, Rodgus J, Davies F, Liu Y, Jauneikaite E, Larrouy-Maumus G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel method to distinguish Klebsiella species using matrix-assisted laser desorption/ionization-time of flight mass spectrometry .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Klebsiella species, including Klebsiella pneumoniae and Klebsiella quasipneumoniae, present significant challenges in clinical microbiology due to their genetic similarity, which complicates accurate species identification using established methods, including matrix-assisted laser desorption/ionization-time of flight mass spectrometry (MALDI-TOF MS) on the protein/peptide level. Although the treatment choice for infections caused by these pathogens is often similar, precise species characterization enhances our epidemiological understanding. While whole-genome sequencing can accurately distinguish Klebsiella species accurately, those analyses are time-consuming, requiring specialized expertise, and are not currently used in routine clinical laboratories. Therefore, developing a timely and accurate pathogen characterization method is essential for effective treatment, management, and infection control measures. This study combined MALDI-TOF MS in negative ion mode with machine learning techniques to identify potential lipid biomarkers as a novel method to distinguish between K. pneumoniae and K. quasipneumoniae. Using this method, we identified discriminative features between the species, with peaks at m/z 2157, m/z 1931, m/z 1964, m/z 2042, and m/z 1407 highlighted as potential biomarkers for species identification. Our findings suggest that the lipid profiles of the species obtained from MALDI-TOF MS can serve as effective biomarkers for distinguishing Klebsiella species. Further research should focus on the structural identification of these biomarkers and expand the data set to include more isolates for each of the species. This approach holds promise for developing more cost-effective and rapid diagnostic tools in clinical microbiology, ultimately improving patient outcomes and infection control.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40660849/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CircGLIS3 Inhibits Intramuscular Adipogenesis and Alleviates Skeletal Muscle Fat Infiltration.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yu S, Wang J, Liu H, Guo J, Kuraz AB, Pan Y, Li A, Mei C, Cheng G, Zan L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #fedbf2; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, pan-genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We screened and characterized a novel circRNA, circGLIS3, which is highly expressed in bovine intramuscular IMF tissue and in the skeletal muscle of mice exhibiting fat infiltration.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Intramuscular fat (IMF) is a key determinant of meat quality. Excessive IMF deposition, commonly observed in human obesity and aging, negatively affects skeletal muscle function. Circular RNAs (circRNAs) play critical regulatory roles in muscle and fat development, as well as in the progression of related diseases. However, the specific functions and underlying mechanisms of circRNAs in IMF deposition have not been extensively studied.We screened adipogenic differentiation-related circRNAs using circRNA-seq combined with WGCNA. Functional analyses, including loss-of-function and gain-of-function experiments, were performed to determine the role of circGLIS3 in regulating adipogenesis in bovine intramuscular preadipocytes and 3T3-L1 cells. A high-fat diet-induced mouse model was established to investigate the in vivo effects of circGLIS3 on skeletal muscle fat infiltration in mice (C57BL/6; n = 6 males). Mechanistic studies involved transcriptomic analysis, dual-luciferase reporter assays, RNA immunoprecipitation, and rescue experiments.We screened and characterized a novel circRNA, circGLIS3, which is highly expressed in bovine IMF tissue and in the skeletal muscle of mice exhibiting fat infiltration. Functionally, circGLIS3 overexpression inhibited triglyceride synthesis (-16.19%, p < 0.01) and lipid droplet accumulation (-42.44%, p < 0.01) in intramuscular preadipocytes, concurrently downregulating the expression of key adipogenesis-related genes and proteins (p < 0.05). CircGLIS3 knockdown promoted triglyceride synthesis (+28.32%, p < 0.05) and lipid droplet accumulation (+47.40%, p < 0.01) in intramuscular preadipocytes, accompanied by a significant upregulation of adipogenesis-related genes and proteins (p < 0.05). Notably, circGLIS3 exhibits more than 80% sequence homology among bovine, mouse, and human species. CircGLIS3 overexpression inhibited triglyceride synthesis (-28.84%, p < 0.01) and lipid droplet accumulation (-46.49%, p < 0.01) in 3T3-L1 cells, concurrently downregulating the expression of adipogenesis-related genes and proteins (p < 0.05). Overexpression of circGLIS3 reduced skeletal muscle fat infiltration induced by high-fat diet in mice (-57.88%, p < 0.05). Mechanistically, circGLIS3 acts as a sponge for miR-21-3p, increasing the expression of its target gene, LEPR (+56.05%, p < 0.01), and promoting the phosphorylation level of its downstream signalling protein, AMPKŒ± (+68.68%, p < 0.01).CircGLIS3 inhibits bovine intramuscular adipogenesis by regulating the miR-21-3p/LEPR/AMPK axis, reducing fat infiltration in mouse skeletal muscle. These findings suggest that circGLIS3 is a promising target for improving meat quality in livestock, and as a potential therapeutic marker for alleviating skeletal muscle fat infiltration associated with obesity.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40735833/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Altered translation elongation contributes to key hallmarks of aging in the killifish brain.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Domenico Di Fraia, Antonio Marino, Jae Ho Lee, Erika Kelmer Sacramento, Mario Baumgart, Sara Bagnoli, Till Balla, Felix Schalk, Stephan Kamrad, Rui Guan, Cinzia Caterino, Chiara Giannuzzi, Pedro Tomaz da Silva, Amit Kumar Sahu, Hanna Gut, Giacomo Siano, Max Tiessen, Eva Terzibasi-Tozzini, Eugenio F Fornasiero, Julien Gagneur, Christoph Englert, Kiran R Patil, Clara Correia-Melo, Danny D Nedialkova, Judith Frydman, Alessandro Cellerino, Alessandro Ori</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Aging is a major risk factor for neurodegeneration and is characterized by diverse cellular and molecular hallmarks, including increased ribosome stalling and widespread depletion of proteins enriched in basic amino acids.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Aging is a major risk factor for neurodegeneration and is characterized by diverse cellular and molecular hallmarks. To understand the origin of these hallmarks, we studied the effects of aging on the transcriptome, translatome, and proteome in the brain of short-lived killifish. We identified a cascade of events in which aberrant translation pausing led to altered abundance of proteins independently of transcriptional regulation. In particular, aging caused increased ribosome stalling and widespread depletion of proteins enriched in basic amino acids. These findings uncover a potential vulnerable point in the aging brain's biology-the biogenesis of basic DNA and RNA binding proteins. This vulnerability may represent a unifying principle that connects various aging hallmarks, encompassing genome integrity, proteostasis, and the biosynthesis of macromolecules.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40743332/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Enhancing TFEA.ChIP with ENCODE Regulatory Maps for Generalizable Transcription Factor Enrichment</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Berrouayel, Y., del Peso, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A biologically grounded and biologically grounded method for inferring transcriptional regulators from gene expression data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Identifying transcription factors (TFs) responsible for gene expression changes remains a central challenge in functional genomics. TFEA.ChIP is a ChIP-seq-based TF enrichment analysis tool that addresses this by linking TF binding profiles to differentially expressed genes through experimentally supported cis-regulatory element (CRE)-gene associations. Unlike motif- or heuristic-based approaches, TFEA.ChIP adopts a biologically grounded strategy by intersecting TF binding data from ReMap2022 with regulatory maps from ENCODE's rE2G and CREdb. To overcome the high context-specificity of rE2G associations, we developed filtering strategies based on confidence scores and recurrence across biosamples. Benchmarking on 369 curated gene sets from the MSigDB C2 CGP collection showed that recurrence-based filtering significantly improved accuracy, outperforming the original GeneHancer-based implementation and leading tools including BARTv2.0, Lisa, ChEA3, and HOMER. A case study on hypoxia further validated the method, demonstrating accurate and pathway-specific enrichment of HIF-related TFs using both overrepresentation analysis and gene set enrichment analysis (GSEA). Additionally, the updated implementation of TFEA.ChIP in R/Bioconductor introduces several user-friendly features, including automated analysis workflows and expression-based filtering of candidate TFs. These additions streamline the integration of TFEA.ChIP into standard RNA-seq analysis pipelines, enabling more efficient and reproducible workflows. Together with its strong benchmarking performance and biologically grounded framework, the updated tool provides a robust and accessible solution for inferring transcriptional regulators from gene expression data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.28.667192v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SkeletAge: Transcriptomics-based Aging Clock Identifies 26 New Targets in Skeletal Muscle Aging</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ali, M., Li, F., Katari, M. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A method for building tissue-specific bulk RNA-seq-based age-estimators that can be used to identify the ageprint, the set of genes that drive baseline healthy aging in a tissue- specific, developmentally-linked fashion.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Identifying the set of genes that regulate baseline healthy aging - aging that is not confounded by illness - is critical to understating aging biology. Machine learning-based age-estimators (such as epigenetic clocks) offer a robust method for capturing biomarkers that strongly correlate with age. In principle, we can use these estimators to find novel targets for aging research, which can then be used for developing drugs that can extend the healthspan. However, methylation-based clocks do not provide direct mechanistic insight into aging, limiting their utility for drug discovery. Here, we describe a method for building tissue-specific bulk RNA-seq-based age-estimators that can be used to identify the ageprint. The ageprint is a set of genes that drive baseline healthy aging in a tissue-specific, developmentally-linked fashion. Using our age estimator, SkeletAge, we narrowed down the ageprint of human skeletal muscles to 128 genes, of which 26 genes have never been studied in the context of aging or aging-associated phenotypes. The ageprint of skeletal muscles can be linked to known phenotypes of skeletal muscle aging and development, which further supports our hypothesis that the ageprint genes drive (healthy) aging along the growth-development-aging axis, which is separate from (biological) aging that takes place due to illness or stochastic damage. Lastly, we show that using our method, we can find druggable targets for aging research and use the ageprint to accurately assess the effect of therapeutic interventions, which can further accelerate the discovery of longevity-enhancing drugs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.28.667277v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A comprehensive data repository of environmental toxicants exposed mouse epigenomes from the TaRGET II Consortium</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kuntala, P. K., Miao, B., Purushotam, D., Zhang, B., Li, D., Wang, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> The Toxicant Exposures and Responses by Genomic and Epigenomic Regulators of Transcription (TaRGET) is a multiphase program that aims to understand how environmental factors contribute to disease susceptibility using toxicant-exposed mouse models.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The Toxicant Exposures and Responses by Genomic and Epigenomic Regulators of Transcription (TaRGET) is a multiphase program that aims to understand how environmental factors contribute to disease susceptibility using toxicant-exposed mouse models. Here, we introduce the TaRGET II Data Portal (https://data.targetepigenomics.org/), a repository of exposomes from mice exposed to environmental toxicants, including arsenic (As), lead (Pb), bisphenol A (BPA), tributyltin (TBT), di(2-ethylhexyl) phthalate (DEHP), tetrachlorodibenzo-p-dioxin (TCDD), and air pollution (PM2.5). Sequencing assays capturing changes in chromatin accessibility, DNA methylation, gene expression, and post-translational histone modifications from multiple centers were quality-controlled and uniformly processed. The datasets cover multiple tissues collected at four time points: 3 weeks, 5 weeks, 20 weeks, and 40 weeks. The TaRGET II Data Portal offers an efficient way to browse, search, visualize, and download relevant datasets and associated metadata, serving as a key resource for studying the impact of environmental toxicant exposures on disease susceptibility for the broader scientific community.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.28.667245v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A guide to D-SPIN: constructing regulatory network models from single-cell RNA-seq perturbation data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jiang, J., Thomson, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> D-SPIN (Dimension-scalable Single-cell Perturbation Integration Network) is a computational framework for constructing regulatory network models from large-scale single-cell perturbation data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> D-SPIN (Dimension-scalable Single-cell Perturbation Integration Network) is a computational framework for constructing regulatory network models from large-scale single-cell perturbation data. D-SPIN models the transcriptional state of cells as a set of individual genes or gene programs that interact through a condition-dependent spin network model or Markov random field. A single unified network is learned across multiple conditions, where the perturbations modulate the activity of individual network nodes. The architecture allows information integration between different perturbation conditions and efficient inference algorithms that scale to thousands of genes, thousands of conditions, and millions of single cells. For interpretability, D-SPIN can also operate at the level of co-expressing gene programs, which are extracted via unsupervised orthogonal non-negative matrix factorization (oNMF). Here we present the theory and detailed procedures for applying D-SPIN to build program-level or gene-level network models from single-cell data. With a public immune dictionary dataset, we illustrate multiple applications of D-SPIN, including identifying gene programs and network modules, classifying perturbation responses of cytokines, and nominating key regulators mediating cytokine responses.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.28.667326v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">BioTransformer4.0 a comprehensive computational tool for small molecule metabolism prediction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tian, S., Feunang, Y. D., Oler, E., Wang, F., Greiner, R., Palm, E. H., Schymanski, E. L., Wishart, D. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> BioTransformer 4.0, the successor to BioTransformer 3.9, is a freely available in silico metabolism prediction tool that can predict metabolites for small molecules.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> BioTransformer 4.0, the successor to BioTransformer 3.0, is a freely available in silico metabolism prediction tool. It integrates both knowledge based and machine learning approaches to predict metabolites for small molecules using one of seven modules: abiotic, environmental, CYP450, phase II, enzyme commission-based, human gut microbial, and all human metabolism. It also provides a customizable sequence prediction module that allows users to simulate multi-step metabolic transformations by chaining among the first six different modules. BioTransformer4.0 can make predictions more efficiently and accurately than the previous version, as it includes more than 130 new reaction rules, and also an optional validation module to improve the efficiency by restricting the number of predicted metabolites, due to their similarity among real human metabolites. We evaluated its performance by running the six-step all-human metabolism prediction on the DrugBank dataset of 2,457 known biotransformations, and the PhytoHub dataset of 633 known biotransformations (R) achieving recall values of 87.2% (resp., 91.6%) for the DrugBank (resp.,PhytoHub) datasets.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.28.667289v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A draft genome assembly of the agricultural pest Leucoptera coffeella and analysis of its dsRNA processing machinery is a key step towards RNAi-based biopesticides in Lepidoptera</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Goldberg, J. K., Vidal, L. A., Queriroz, E. S. L., Nascimento, E. F. M. B., Viana, M. J. A., Clarindo, W. R., Maranhao, A. Q., Martins, N. F., Albuquerque, E. V. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We sequenced and assembled a draft L. coffeella genome using PacBio highly accurate long-reads (HiFi) to enable future research in this vein, to enable the development of novel techniques for controlling this insect pest.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The Coffee Leaf Miner (Lepidoptera: Lyonetiidae: Leucoptera coffeella) is a specialist herbivore and major global pest of coffee plants. Current pest control strategies primarily rely on chemical pesticides, leading to human and environmental constraints. Additionally, the emergence of insecticide-resistant populations underscores the urgent need for more specific and efficient pest management strategies. The development of novel techniques for controlling this insect pest requires rigorous interrogation of its physiology and interactions with host plants at a molecular/genetic level. To enable future research in this vein, we sequenced and assembled a draft L. coffeella genome using PacBio highly accurate long-reads (HiFi). Our assembly is comprised of 1615 contigs showing fragmentation, yet the majority of gene content is represented (BUSCO complete = 91.7%). We annotated 17467 protein-coding genes within our assembly, seven of which are core components of the small interfering RNA machinery. The expression of these genes was further confirmed via qPCR. This analysis - and the underlying genomic data - highlights potential targets for RNAi-based biopesticide development and will serve as the foundation for important future research aimed at protecting global coffee production from one of its most destructive pests.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.28.667214v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A New Detailed Mass Offset Search in MSFragger for Improved Interpretation of Complex PTMs</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rojas Ramirez, C., Yu, F., Polasky, D. A., Nesvizhskii, A. I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a new "detailed" mass offset (DMO) search for complex modification sets, allowing simultaneous search for hundreds of modifications with more straightforward downstream interpretation than open search.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Conventional database search methods for proteomics struggle when tasked to identify dozens or hundreds of modifications simultaneously. Open or error-tolerant searches can address this limitation, but at the cost of increased difficulty in downstream interpretation of the results and quantification. We and others have previously described "mass offset" or multi-notch searches that sit in between closed and open searches, allowing simultaneous search for hundreds of modifications with more straightforward downstream interpretation than open search. The original mass offset searches were closer to open search, lacking the ability to restrict modifications to specific amino acids. Here, we describe a new "detailed" mass offset (DMO) search implemented in the MSFragger search engine, which allows each mass offset to have its own site restrictions and fragmentation rules. The benefits of DMO search over existing mass offset searches are shown with three example searches of complex modification sets: nearly one hundred post-translational modifications, fast photochemical oxidation of proteins (FPOP)-derived modifications, and amino acid substitutions. The DMO search further improves the interpretability of results by reducing ambiguity in site localization, particularly when modifications have overlapping masses, and provides benefits that scale with the complexity of the search.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.28.667198v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Design of peptides with non-canonical amino acids using flow matching</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lee, J. S., Kim, P. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a flow-based generative model capable of incorporating any arbitrary non-canonical amino acid into a given protein.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The canonical vocabulary of twenty amino acids limits the chemical space available to proteins and peptides. Expanding this vocabulary to hundreds of non-canonical amino acids allows the engineering of proteins with novel function and activity, and is of great interest for the discovery of novel drugs such as macrocyclic peptides. Here we present NCFlow, a flow-based generative model capable of incorporating any arbitrary non-canonical amino acid into a given protein. To supplement sparse training data in the Protein Data Bank, NCFlow is pretrained on millions of small molecule structures and a large set of protein-ligand complexes before finetuning on native non-canonicals found within proteins in the Protein Data Bank. We show that NCFlow outperforms AlphaFold3-based methods in the structure prediction of unseen non-canonical amino acids. We present a peptide design pipeline akin to in silico deep mutational scanning, and propose a novel scoring strategy using a combination of deep learning-based and molecular dynamics-based alchemical binding free energy calculations to identify improved peptide variants. We apply the method on four protein-peptide complex test cases, and observe that incorporating non-canonicals can significantly improve binding affinity by up to-7.0 kcal/mol. Thus, NCFlow can be easily integrated into existing protein design platforms to further improve its properties outside of what is capable with standard amino acids.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.31.667780v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DeepRES: Deep learning enables reaction-based comprehensive enzyme screening</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hirota, K., Yamada, T.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed DeepRES, an AI-based framework for comprehensive enzyme screening, which is expected to facilitate high-throughput identification of orphan enzyme-encoding genes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Enzymes accelerate biochemical reactions in living organisms, thus playing an important role in metabolism. Although metabolic pathway databases are growing, many metabolic reactions, termed orphan enzymes, have not been annotated to gene sequences, which hinders functional annotation in genomic analysis. Moreover, protein databases contain many proteins of unknown function. Owing to this gap between known proteins and enzymatic reactions, various proteins of unknown function may be orphan enzymes; however, available tools cannot adequately predict these links. In this study, we developed DeepRES, an AI-based framework for comprehensive enzyme screening, to explore novel enzyme candidates from proteins of unknown function for reactions of interest. DeepRES implements enzyme screening via two steps: classification of enzymes and non-enzymes and prediction of catalytic capabilities for enzyme-reaction pairs. The two deep learning models comprising DeepRES showed comparable or superior performance to that of existing software. We performed screening of 1,255 orphan enzymes involved in the microbiome using DeepRES and successfully identified candidate proteins for 897 orphan enzymes. We then used those candidates as references for genomic analysis and explored novel biosynthetic gene clusters from microbial genomes to obtain promising candidate gene clusters, including those related to anthocyanin degradation. Comprehensive enzyme screening via DeepRES, which is the first computational tool designed to associate orphan enzymes with proteins of unknown function, is expected to facilitate high-throughput identification of orphan enzyme-encoding genes. Furthermore, DeepRES can be easily integrated into the current genomic analysis pipeline to extend the functional annotation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.28.667344v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SKiM-GPT: Combining Biomedical Literature-Based Discovery with Large Language Model Hypothesis Evaluation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Freeman, J., Millikin, R. J., Xu, L., Sharma, I., Moore, B. M., Lock, C., George, K. S., Bal, A., Stewart, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce SKiM-GPT, a retrieval-augmented generation (RAG) system that combines co-occurrence search and retrieval with frontier LLMs to evaluate user-defined hypotheses.</p><p style="font-size: 16px; text-align: justify;">Background Generating and testing hypotheses is a critical aspect of biomedical science. Typically, researchers generate hypotheses by carefully analyzing available information and making logical connections, which are then tested. The accelerating growth of biomedical literature makes it increasingly difficult to keep pace with connections between biological entities emerging across biomedical research. Recently developed automated means of generating hypotheses can generate many more hypotheses than can be easily tested. One such approach involves literature-based discovery (LBD) systems such as Serial KinderMiner (SKiM), which surfaces putative A-B-C links derived from term co-occurrence. However, LBD systems leave three critical gaps: (i) they find statistical associations, not biological relationships; (ii) they can produce false-positive leads; and (iii) they do not assess agreement with a hypothesis in question. As a result, LBD search results often require costly manual curation to be of practical utility to the researcher. Large language models (LLMs) have the potential to automate much of this curation step, but standalone LLMs are hampered by hallucinations, lack of transparency in information sources, and the inability to reference data not included in the training corpus. Results We introduce SKiM-GPT, a retrieval-augmented generation (RAG) system that combines SKiM's co-occurrence search and retrieval with frontier LLMs to evaluate user-defined hypotheses. For every chosen A-B-C SKiM hit, SKiM-GPT retrieves appropriate PubMed abstract texts, filters out irrelevant abstracts with a fine-tuned relevance model, and prompts an LLM to evaluate the user's hypothesis, given the relevant abstracts. Importantly, the SKiM-GPT system is transparent and human-verifiable: it displays the retrieved abstracts, the hypothesis score, and a justification for the score grounded in the texts and written in natural language. On a benchmark consisting of 14 disease-gene-drug hypotheses, SKiM-GPT achieves strong ordinal agreement with four expert biologists (Cohen's {kappa} = 0.84), demonstrating SKiM-GPT's ability to replicate expert judgment. Conclusions SKiM-GPT is open-source ( https://github.com/stewart-lab/skimgpt ) and available through a web interface ( https://skim.morgridge.org ), enabling both wet-lab and computational researchers to systematically and efficiently evaluate biomedical hypotheses at scale.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.28.664797v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Organ-Specific Long-Read Transcriptome Assembly of Stenochlaena palustris and Annotation of Anthocyanin Biosynthesis Genes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dwiyanti, M. S., Rahmawati, D., Gunawan Puteri, M. D. P. T., Panapitiya, D., Nion, Y. A., Wijaya, E., Santoso, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the first long-read transcriptomic resource for Stenochlaena palustris, a vegetable and medicinal fern native to Southeast Asia, and provide a high-quality organ-specific transcriptome assembly of young leaves.</p><p style="font-size: 16px; text-align: justify;">Background: Stenochlaena palustris is valued as a vegetable and medicinal fern native to Southeast Asia; however, it remains largely underrepresented in genomic studies. People in Kalimantan (Indonesia) collect young leaves and fronds from wild populations for use as vegetables or medicines to treat conditions, such as ulcers, stomachaches, fever, diarrhoea, and skin infections. The young leaves and fronds of S. palustris contain flavonoids, polyphenols, and anthocyanins. Here, we present a high-quality organ-specific transcriptome assembly of S. palustris based on long-read RNA sequencing of young leaves. Results: The de novo assembly yielded 47,759 transcripts, with an N50 of 1,524 bp and a BUSCO completeness of 66.6%, consistent with organ-specific transcriptomes. Functional annotation identified key structural and regulatory genes involved in anthocyanin biosynthesis, including genes for chalcone synthase (CHS) and dihydroflavonol 4-reductase (DFR). We further analysed the expression of the selected CHS and DFR genes via qRT-PCR of three phenotypically contrasting young leaf samples. Although no strong correlation was observed between gene expression levels and anthocyanin pigmentation, the results suggest that complex regulation involves post-transcriptional control or developmental timing. Conclusions: This study provides the first long-read transcriptomic resource for S. palustris and valuable data for future investigations of secondary metabolism and gene regulation in ferns. Our findings complement broader fern transcriptome studies by offering tissue-specific resolution and a focused view of pigment biosynthesis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.30.667584v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Visualization and Prediction of in vivo Phosphate Dynamics via Auto-Glowing Plant Sensors</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liu, M.-J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a low-cost and non-invasive biosensor to visualize phosphate dynamics in planta and predict nutrient deficiency for sustainable agriculture.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Monitoring endogenous nutrient levels is crucial for maximizing crop yields and optimizing fertilizer use. Here, focusing on phosphorus, an essential nutrient for plant growth, we developed a low-cost and non-invasive biosensor to visualize and predict early stress signaling in plants. By combining plant phosphate (Pi)-deficiency-induced promoter systems with fungal self-sustained bioluminescence systems genetically engineered into tobacco plants, we created sensor plants that emitted more light when experiencing Pi deficiency. This light emission correlated with the expressions of known phosphate-responsive genes and the total phosphorus content in plants, and decreased during Pi recovery conditions, demonstrating the responsiveness and robustness of the sensor plants in reflecting endogenous phosphorus deficiency. The sensor plants responded primarily to Pi deficiency rather than nitrogen or potassium deficiencies and were sensitive to different ranges of external Pi concentrations. Additionally, when grafted onto tomato and chili pepper plants, the sensor plants responded to external phosphorus deficiency, showing promise for monitoring stress signals in different crop species. Using deep-learning-based image analysis techniques, auto-luminescent signals of sensor plants could be detected and used to predict phosphorus deficiency. This study outlines a strategy of creating a self-luminous biosensor to visualize phosphate dynamics in planta and predict nutrient deficiency for sustainable agriculture.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.30.667770v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Leveraging High-dimensional Seed Colorspace in Phenomic Selection Models for Herbaceous Perennial Crops</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Keaggy, W., Harris, Z. N., Braley, J., Cassetta, E., Gutierrez, J., Piotter, E., Schuhl, H., Crain, J. L., DeHaan, L. R., Turner, M. K., Fahlgren, N., Hershberger, J. M., Schlautman, B., Van Tassel, D., Rubin, M. J., Miller, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We used image scans to predict germination traits from high-dimensional seed traits in perennial crop candidate species, and found that the most accurate predictions were made by combining HSV and morphology features.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Herbaceous perennial crops, which only recently entered the domestication pipeline, offer the potential of ecological benefits in agricultural systems but remain underdeveloped, partly due to slow breeding cycles. Early-stage phenomic selection, the process of using high-dimensional secondary traits to predict target traits, may accelerate improvement. We evaluated whether seed traits derived from image scans could be used to predict germination traits in three perennial crop candidate species: sainfoin (Onobrychis viciifolia), intermediate wheatgrass (Thinopyrum intermedium), and silflower (Silphium integrifolium). Over 20,000 seeds were scanned, and PlantCV was used to extract 692 hue, saturation, and value (HSV) color features along with five morphological traits. We demonstrated that seed color, morphology, and germination traits were all influenced by maternal family, but patterns of correlation between germination traits and color and morphology traits are complex. To enable prediction from high-dimensional seed traits, we constructed relationship matrices from combinations of HSV and morphology features for use in phenomic selection models to predict germination proportion and timing. Model performance varied by trait, species, and cohort, and in many cases, predictions were significantly better than chance. HSV features, when combined with morphology, often yielded the most accurate predictions, with performance reaching a maximum of r = 0.33. These results demonstrate the potential of low-cost, image-based phenomic data to inform early-stage selection and support the development of sustainable perennial crops.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.28.667321v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Large-Scale Multigenome-Wide Study Predicts the Existence of Transmembrane Phosphotransfer Proteins in Plant MSP Signaling Pathway</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lomin, S. N., Brenner, W. G., Savelieva, E. M., Arkhipov, D. V., Romanov, G. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new class of plant phosphotransfer proteins belonging to multistep phosphorelay (MSP) system was discovered by using large-scale bioinformatics methods.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> A new class of plant phosphotransfer proteins belonging to multistep phosphorelay (MSP) system was discovered by using large-scale bioinformatics methods. Unlike the canonical soluble nucleo-cytosolic forms, these proteins were predicted to have transmembrane (TM) domains and, apparently, should be localized on some kind of cell membranes. To date, 95 predicted TM-containing phosphotransmitter (TM-HPt) homologs were found in 61 plant species belonging to different clades, taxa and groups: bryophytes, gymnosperms, mono- and dicotyledons. The conserved HPt motif with phosphorylatable histidine was preserved in the most of TM-HPts under study that allowed us to consider these proteins potentially active in MSP signaling. For the found TM-HPts, a Bayesian analysis at the DNA level was performed and a relevant phylogenetic tree was constructed. According to evolutionary relationships, plant TM-HPts were divided into two main groups corresponding to Arabidopsis AHP1-3,5,6 and AHP4 orthologs. Transcriptomic analysis confirmed the expression of the most of the investigated TM-HPt-encoding genes. Their moderate-to-low overall transcription rate may be a consequence of inducible and/or tissue-specific expression. Using molecular modeling methods, a variety of potential spatial organizations of several such proteins are demonstrated. Possible roles of TM-HPts as modulators of MSP signaling pathway and corresponding putative mechanisms of their action are suggested.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.28.667123v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">InSillyClo, a user-friendly web application to assist large-scale Golden Gate Cloning and MoClo workflows</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Galez, H., Brancotte, B., Bonche, J., Fumey, J., Napolitano, S., Batt, G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present InSillyClo, an open-source web application to assist large-scale Golden Gate cloning and MoClo workflows.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Systems and synthetic biology developments often require the construction of many variants of a genetic circuit of interest, resulting in large-scale cloning campaigns. Golden Gate and Modular Cloning (MoClo), two powerful technologies enabling the scale-up of cloning workflows, play a central role for efficient circuit construction. These workflows include a number of dry-lab tasks, which are time-consuming and error-prone at scale. Currently, no software tool is available to handle these tasks in a dedicated, time-saving, and user-friendly manner. We present InSillyClo, an open-source web application to assist large-scale Golden Gate cloning and MoClo workflows. It supports an easy specification of genetic designs at any scale, followed by the automated generation of comprehensive workflow-related data. Moreover, InSillyClo leverages Modular Cloning with a versatile typing system of parts to generate user-defined workflows. InSillyClo is open source, accessible with or without user registration, and can also be used locally.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.31.667881v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">BRAVE: a highly accurate method for predicting HIV-1 antibody resistance using large language models for proteins</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> El Anbari, M., Bylund, T., O'Dell, S., Tourtellott, E., McKee, K., Schmidt, S. D., Mkhize, N. N., Moore, P. L., Doria-Rose, N., Zhou, T., Rawi, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-31
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a novel machine learning approach called BRAVE (Bnab Resistance Analysis Via Evolutionary scale modeling 2) designed to predict HIV-1 resistance against 33 known bNAbs.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: Broadly neutralizing antibodies (bNAbs) that target the envelope glycoprotein (Env) of human immunodeficiency virus-1 (HIV-1) have been utilized in clinical trials aimed at preventing and treating HIV-1 infections. However, the emergence of neutralization resistance to bNAbs occurs rapidly due to the high mutation rate of HIV-1. Previous studies have suggested the use of in silico methods to effectively predict the resistance of HIV-1 isolates to bNAbs. In this study, we present a novel machine learning approach called BRAVE (Bnab Resistance Analysis Via Evolutionary scale modeling 2) designed to predict HIV-1 resistance against 33 known bNAbs. This innovative tool employs a Random Forests classifier that uses a protein language model to reliably capture protein features. Results: BRAVE outperformed leading resistance prediction tools on various performance metrics, attaining the highest performance in established classification measures including accuracy, area under the curve, logarithmic loss, and F1-score. Importantly, rigorous statistical comparisons (p<0.001) show that BRAVE is significantly more accurate than state-of-the-art neutralization prediction tools. BRAVE will facilitate informed decisions of antibody usage and sequence-based monitoring of viral escape in clinical settings. Availability and implementation BRAVE software is available for download under GitHub (https://github.com/kiryst/BRAVE/tree/master).</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.28.667234v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A review of the journey of field crop phenotyping: From trait stamp collections and fancy robots to phenomics-informed crop performance predictions.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lukas Roth, Afef Marzougui, Achim Walter</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-13
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A review of phenotyping from high-throughput sensor data for trait extraction to advanced modeling approaches that integrate multi-temporal data, latent space representations, and learned crop models.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Crop phenotyping encompasses methodologies for measuring plant growth, architecture, and composition with high precision across scales, from organs to canopies. Field-based phenotyping is pivotal in bridging genomic data with crop performance, offering a promising pathway for predictive modeling in diverse environments. This review traces the evolution of phenotyping from high-throughput sensor data for trait extraction to advanced modeling approaches that integrate multi-temporal data, latent space representations, and learned crop models. This evolution is exemplified mostly by morphology- and growth-related examples from the core expertise of the authors. High-throughput trait extraction, facilitated by advanced imaging and sensor technologies, has enabled rapid and accurate characterization of complex traits essential for crop improvement. Carrier platforms, such as drones, rovers, and gantries, have played a critical role in capturing high-resolution data across large fields, enhancing the spatial and temporal resolution of phenotypic data. Publicly available datasets have further accelerated research by providing standardized, high-quality data for benchmarking and model development beyond the realm of crop growth as for example in crop photosynthesis. These advancements are transforming phenotyping into a predictive science capable of informing breeding and management decisions. As phenotyping methodologies continue to evolve, the integration of machine learning and data-driven approaches offers new opportunities for enhancing prediction accuracy and understanding genotype-environment interactions. While challenges such as data heterogeneity, scalability, and cost remain, we highlight key gaps and propose solutions, underscoring phenotyping's critical role in future agricultural innovation.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40532480/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrative machine learning reveals the biological function and prognostic significance of Œ±-ketoglutarate in gastric cancer.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fangyuan Liu, Xuemeng Sun, Yun Zeng, Xiangyun Meng, Rongrong Zhang, Liya Su, Gang Liu</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> The Œ±-Ketoglutarate Index (AKGI) is a potential prognostic biomarker for gastric cancer and can be used to identify patients likely to benefit from immunotherapy, enhance diagnostic precision and improve clinical outcomes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Gastric cancer (GC) has a poor response to treatment, an unfavorable prognosis and a lack of reliable biomarkers for predicting disease progression and therapeutic outcomes. Œ±-Ketoglutarate (Œ±-KG) is a critical metabolite involved in cellular energy metabolism and epigenetic regulation during tumor development, which has emerged as a potential prognostic biomarker for GC. The present study aimed to explore this potential using publicly available datasets from The Cancer Genome Atlas and Gene Expression Omnibus databases to analyze Œ±-KG-related genes and establish the Œ±-KG Index (AKGI). By assessing the predictive performance of the AKGI model, the results demonstrated its capability to predict survival outcomes in patients with GC. Notably, high AKGI scores were associated with worse prognoses. Building on these findings, the associations between AKGI and clinical variables, immune cell infiltration and tumor mutation characteristics were assessed, further identifying potential therapeutic drugs for patients with high AKGI scores. Additionally, by analyzing signaling pathways and biological functions correlated with AKGI, the regulatory mechanisms and biological roles of Œ±-KG in GC were elucidated. The findings of these analyses were further evaluated using cellular experiments, where Œ±-KG treatment was demonstrated to significantly inhibit GC cell proliferation, migration and invasion. In conclusion, the present study successfully constructed and validated the AKGI as a potential prognostic biomarker for GC. The findings indicate that AKGI can identify patients likely to benefit from immunotherapy, enhance diagnostic precision and improve clinical outcomes in GC management. Moreover, AKGI offers a valuable framework for advancing the understanding of the role and mechanisms of Œ±-KG in GC.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40606305/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genomics-assisted breeding for designing salinity-smart future crops.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ali Raza, Qamar U Zaman, Sergey Shabala, Mark Tester, Rana Munns, Zhangli Hu, Rajeev K Varshney</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-20
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #fedbf2; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, pan-genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We explore the advancement and prospects of genomics-assisted breeding (GAB) tools, including genome sequencing, QTL mapping, GWAS, GS, HBB, pan-genomics, single-cell/tissue genomics and phenotyping, epigenomics and transgenetics,</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Climate change induces many abiotic stresses, including soil salinity, significantly challenging global agriculture. Salinity stress tolerance (SST) is a complex trait, both physiologically and genetically, and is conferred at various levels of plant functional organization. As both the sustainability and profitability of agricultural production systems are critically dependent on SST, plant breeders are trying to design and develop salinity-smart crop plants capable of thriving under high salinity conditions. The accessibility of extreme-quality reference genomes for cultivated crops, naturally salinity-smart plants, and crop wild relatives has fast-tracked the discovery of key genes and quantitative trait loci (QTLs), marker development, genotyping assays and molecular breeding products with improved SST. Employing fast-forward breeding tools, namely genomic selection (GS), haplotype-based breeding (HBB), artificial intelligence (AI) and high-throughput phenotyping (HTP), has shown influence not only for fast-tracking genetic gains but also for reducing the time and cost of developing commercial cultivars with enhanced SST and yield stability. This review discusses the advancement and prospects of various genomics-assisted breeding (GAB) tools, including genome sequencing, QTL mapping, GWAS, GS, HBB, pan-genomics, single-cell/tissue genomics and phenotyping, epigenomics and transgenomics, to exploit the genetic landscape for improving SST. Additionally, we explore the integration of HTP and AI, which demonstrates how these innovative approaches can optimize breeding efficiency and guide large-scale breeding efforts for designing salinity-smart crops to ensure sustainable agriculture and global food security. The collective adoption of these tools suggests bridging the gap between research and field application to deliver stress-smart varieties designed for saline-affected regions worldwide.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40390692/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multi-positive contrastive learning-based cross-attention model for T cell receptor-antigen binding prediction.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yi Shuai, Pengcheng Shen, Xianrui Zhang</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new method for predicting TCR-antigen binding pairs that can be identified by the immune system and that are immunogenic.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> T cells play a vital role in the immune system by recognizing and eliminating infected or cancerous cells, thus driving adaptive immune responses. Their activation is triggered by the binding of T cell receptors (TCRs) to epitopes presented on Major Histocompatibility Complex (MHC) molecules. However, experimentally identifying antigens that could be recognizable by T cells and possess immunogenic properties is resource-intensive, with most candidates proving non-immunogenic, underscoring the need for computational tools to predict peptide-MHC (pMHC) and TCR binding. Despite extensive efforts, accurately predicting TCR-antigen binding pairs remains challenging due to the vast diversity of TCRs.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40378554/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">TopCysteineDB: A Cysteinome-wide Database Integrating Structural and Chemoproteomics Data for Cysteine Ligandability Prediction.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Michele Bonus, Julian Greb, Jaimeen D Majmudar, Markus Boehm, Magdalena Korczynska, Azadeh Nazemi, Alan M Mathiowetz, Holger Gohlke</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-05-08
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present TopCysteineDB, a comprehensive resource integrating structural information from the PDB with chemoproteomics data from activity-based protein profiling experiments, enabling systematic analysis and prediction of cysteine ligandability.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Development of targeted covalent inhibitors and covalent ligand-first approaches have emerged as a powerful strategy in drug design, with cysteines being attractive targets due to their nucleophilicity and relative scarcity. While structural biology and chemoproteomics approaches have generated extensive data on cysteine ligandability, these complementary data types remain largely disconnected. Here, we present TopCysteineDB, a comprehensive resource integrating structural information from the PDB with chemoproteomics data from activity-based protein profiling experiments. Analysis of the complete PDB yielded 264,234 unique cysteines, while the proteomics dataset encompasses 41,898 detectable cysteines across the human proteome. Using TopCovPDB, an automated classification pipeline complemented by manual curation, we identified 787 covalent cysteines and systematically categorized other functional roles, including metal-binding, cofactor-binding, and disulfide bonds. Mapping residue-wise structural information to sequence space enabled cross-referencing between structural and proteomics data, creating a unified view of cysteine ligandability. For TopCySPAL, a machine learning model was developed, integrating structural features and proteomics data, achieving strong predictive performance (AUROC: 0.964, AUPRC: 0.914) and robust generalization to novel cases. TopCysteineDB and TopCySPAL are freely accessible through a webinterface, TopCysteineDBApp (https://topcysteinedb.hhu.de/), designed to facilitate exploration of cysteine sites across the human proteome. The interface provides an interactive visualization featuring a color-coded mapping of chemoproteomics data onto cysteine site structures and the highlighting of identified peptide sequences. It offers customizable dataset downloads and ligandability predictions for user-provided structures. This resource advances targeted covalent inhibitor design by providing integrated access to previously dispersed data types and enabling systematic analysis and prediction of cysteine ligandability.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40348330/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Exploring deep learning in phage discovery and characterization.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Monyque Karoline de Paula Silva, Vit√≥ria Yumi Uetuki Nicoleti, Barbara da Paix√£o Perez Rodrigues, Ademir Sergio Ferreira Araujo, Joel Henrique Ellwanger, James Moraes de Almeida, Leandro Nascimento Lemos</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A review of the recent revolution in bacteriophage research using deep learning, including the adoption of neural network algorithms applied to metagenomic data to the use of pre-trained language models, such as BERT, which have improved the reconstruction of viral metagenome-assembled genomes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Bacteriophages, or bacterial viruses, play diverse ecological roles by shaping bacterial populations and also hold significant biotechnological and medical potential, including the treatment of infections caused by multidrug-resistant bacteria. The discovery of novel bacteriophages using large-scale metagenomic data has been accelerated by the accessibility of deep learning (Artificial Intelligence), the increased computing power of graphical processing units (GPUs), and new bioinformatics tools. This review addresses the recent revolution in bacteriophage research, ranging from the adoption of neural network algorithms applied to metagenomic data to the use of pre-trained language models, such as BERT, which have improved the reconstruction of viral metagenome-assembled genomes (vMAGs). This article also discusses the main aspects of bacteriophage biology using deep learning, highlighting the advances and limitations of this approach. Finally, prospects of deep-learning-based metagenomic algorithms and recommendations for future investigations are described.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40359589/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Machine learning: Python tools for studying biomolecules and drug design.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fedor V Ryzhkov, Yuliya E Ryzhkova, Michail N Elinson</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-29
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A list of Python scripts, libraries, and ready-made programs for routine tasks in medical chemistry and bioinformatics.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The increasing adoption of computational methods and artificial intelligence in scientific research has led to a growing interest in versatile tools like Python. In the fields of medical chemistry, biochemistry, and bioinformatics, Python has emerged as a key language for tackling complex challenges. It is used to solve various tasks, such as drug discovery, high-throughput and virtual screening, protein and genome analysis, and predicting drug efficacy. This review presents a list of tools for these tasks, including scripts, libraries, and ready-made programs, and serves as a starting point for scientists wishing to apply automation or optimization to routine tasks in medical chemistry and bioinformatics.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40301135/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A Multi-Omics-Based Prognostic Model for Elderly Breast Cancer by Machine Learning: Insights From Hypoxia and Immunity of Tumor Microenvironment.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yu Song, Changjun Wang, Yidong Zhou, Qiang Sun, Yan Lin</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-15
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel prognostic model for older adult breast cancer, based on multiomics and machine learning.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Older adult breast cancer (OABC) patients (‚â• 65 years) frequently experience poorer prognoses compared to younger adults, attributed to complex tumor biology and age-related factors. The present study employs a multiomics approach combined with machine learning to develop a novel prognostic model for OABC, with a focus on the hypoxic and immune characteristics of the tumor microenvironment.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40382303/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Artificial intelligence-driven integration of multi-omics and radiomics: A new hope for precision cancer diagnosis and prognosis.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jordi Camps, Andrea Jim√©nez-Franco, Raquel Garc√≠a-Pablo, Jorge Joven, Meritxell Arenas</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-04-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Multi-omics, radiomics, and artificial intelligence have improved detection, prognosis, and treatment monitoring, and their potential in personalized cancer treatment.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Despite advances in cancer diagnosis and treatment, the disease remains a major health challenge. Integrating multi-omics, radiomics, and artificial intelligence has improved detection, prognosis, and treatment monitoring. Molecular multi-omics provides insights into tumor biology, while radiomics extracts imaging features for outcome prediction. Liquid biopsy and circulating tumor DNA aid early detection and personalized therapy. Artificial intelligence-driven models integrate data to identify biomarkers and guide precision oncology. Despite challenges like cost and data integration, future advancements aim to enhance resolution, scalability, and non-invasive diagnostics. This mini-review explores these methodologies, their clinical impact, and their potential in personalized cancer treatment.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40216368/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

