<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Monday, 15 September 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PLiCat: Decoding protein-lipid interactions by large language model</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dong, F., Wu, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce PLiCat (Protein-Lipid interaction Categorization tool), a predictive framework designed to classify lipid categories interacting with proteins, offering new opportunities for exploring lipid-binding specificity and guiding rational protein design.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Motivation: Protein-lipid interactions are essential for many cellular processes, as proteins associate with diverse lipid molecules to exert distinct functions. However, existing approaches are limited in discriminating among lipid categories in these interactions. Recent advances in protein language models enable discovery of novel sequence insights. Results: We introduce PLiCat (Protein-Lipid interaction Categorization tool), a predictive framework designed to classify lipid categories interacting with proteins. By integrating ESMC and BERT models into a hybrid architecture, PLiCat achieves accurate and interpretable predictions to distinguish lipid-binding signatures across eight major lipid categories. Furthermore, we investigated the potential of PLiCat to identify lipid-binding sites and assess the impact of pathogenic mutations on lipid-binding events. Collectively, PLiCat provides a powerful framework for elucidating the lipid-binding codes, offering new opportunities for exploring lipid-binding specificity and guiding rational protein design. Availability and implementation: The PLiCat source code and processed datasets are available at https://github.com/Noora68/PLiCat.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.09.675043v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">GDEE: A Structure-Based Platform for Gene Discovery and Enzyme Engineering</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Souza, C. S., Correia, J. P. G., Rocha, I., Lousa, D., Soares, C. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present an automated platform for gene discovery and enzyme engineering, aimed at overcoming bottlenecks.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Enzymes are extensively utilized for catalyzing the production of value-added compounds in diverse sectors with significant economic impact. However, challenges such as poor protein expression, low catalytic activity, substrate/co-factor limitations, and toxicity of final products limit the efficiency of these biocatalysts. Protein engineering offers a solution by re-designing enzyme catalytic properties to enhance biosynthetic pathways. In this study, we present an automated platform for gene discovery and enzyme engineering, aimed at overcoming these bottlenecks. The GDEE platform finds and optimizes enzymes responsible for rate-limiting steps in biosynthetic pathways, with objectives ranging from improving catalytic efficiency to enabling novel transformations. The platform operates through four key steps: an initial sequence step that either sources natural enzyme sequences or generates mutant variants, followed by an atomistic protein structure prediction step, a docking step, and a binding energy evaluator, obtaining a set of variant enzyme sequences that are prioritized for experimental validation. Additionally, machine learning-based re-scoring of binding poses to improve binding affinities significantly enhances the accuracy of predicting the effect of mutations on binding, highlighting its potential in metabolic engineering applications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.09.675117v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Active learning enables discovery of transcriptional activators across fungal evolutionary space</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Waldburger, L., Nisonoff, H., Zintel, M., Kirkpatrick, L. D., Lam, A., Lanclos, N., Keasling, J. D., Staller, M. V., Shih, P. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a hybrid framework that leverages high-throughput molecular assays and active learning to quantify biological properties across evolutionary space, and provide the first functional annotation for 3,416 proteins from 670 non-model fungi.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Biological discovery and design are increasingly being guided by predictive models in place of costly experimentation. However, existing datasets are often biased by overrepresentation from model organisms, leading to failures in evolutionary studies of non-model species. We present a hybrid framework that leverages high-throughput molecular assays and active learning to quantify biological properties across evolutionary space. We focus on transcriptional activators, which contain activation domains (ADs) that promote gene expression. ADs are intrinsically disordered and poorly conserved, which limits their study using comparative genomics. Here, we developed ADhunter, a high-capacity regression model that outperforms state-of-the-art algorithms in identifying and quantifying the strength of transcriptional activators. Model uncertainty was used to guide evolutionary sampling across 7.8 million proteins from 2,400 fungal genomes. We functionally characterized 9,836 ADs from 1,071 fungal genomes, providing a 15.5-fold expansion in genome representation compared to existing datasets. Comprehensive sampling from non-model genomes improved model generalizability and provides the first functional annotation for 3,416 proteins from 670 non-model fungi. Model interpretability analysis aligns with the biophysical model of AD function and reveals novel, underrepresented protein codes, highlighting the importance of sampling from non-model organisms to build evolutionarily robust models for predicting biological properties.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.12.675635v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Identification of specific metabolic capacities associated with major extraintestinal pathogenic Escherichia coli lineages</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> ROYER, G., Chau, F., Vallenet, D., Denamur, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified metabolic specificities of pathogenic strains that could explain the emergence and success of major clones, including the ExPEC pandemic clones STc131 and STc14.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Bacterial niche colonization relies on multiple factors, among which the metabolic capacity to utilize specific substrates is pivotal, especially within complex microbiota where nutrient competition is intense. Escherichia coli, a gut commensal of humans and vertebrates, is also an opportunistic pathogen responsible for both intestinal and extraintestinal infections, including bloodstream infections (BSI). To identify metabolic specificities of pathogenic strains that could explain the emergence and success of major clones, we leveraged a well-characterized dataset of 1,498 genomes from both commensal and extraintestinal pathogenic epidemiologically relevant French isolates. Using a pangenomic framework, we conducted a large-scale analysis of metabolic pathways. Although metabolism was far more conserved than gene content, we found substantial metabolic diversity across the species, with over 50% of pathways being variable, mainly involving biosynthetic and degradation processes. Phylogeny had a major impact on metabolic profiles, while no specific pathway was associated with lifestyle (commensal vs. pathogenic) or BSI portal of entry. However, we identified clone-specific metabolic capacities. For example, pathways involved in 5'-deoxynucleoside recycling were enriched in the major extraintestinal pathogen (ExPEC) clone STc69. In phylogroup B2, several clone-specific pathways enabled degradation of plant-derived compounds. Notably, detailed analysis of the D-apiose degradation revealed a functional pathway strongly associated with the STc131 and STc14 pandemic clones. Overall, our study highlights multiple lineage-specific metabolic capacities that may contribute to the ecological success and dissemination of most prevalent ExPEC clones.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.14.675865v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">barbieQ: An R software package for analysing barcode count data from clonal tracking experiments</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fei, L., Maksimovic, J., Oshlack, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce barbieQ, a Bioconductor R package designed for analysing barcode count data across groups of samples.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Tracking the clonal composition of pools of cells as they differentiate or evolve can reveal important features of development and clonal evolution in fields such as stem cell research and cancer. A variety of new single-cell lineage tracing technologies now allow us to track progeny cells back to their original progenitor cell using unique DNA barcodes that are passed from each original cell to its offspring. The barcode count for each sample indicates the number of progeny cells derived from the original cell. However, as the different clonal tracking technologies often require tailored pipelines for barcode count calling, subsequent interpretation of barcode count data is often conducted through bespoke analyses, typically relying on visualisations and heuristics. A generalized workflow for preprocessing and robust statistical analysis of barcode count data across protocols enables studies to achieve the most biologically meaningful and reproducible results by rigorously accounting for sample-to-sample variability. To address this, we introduce barbieQ, a Bioconductor R package designed for analysing barcode count data across groups of samples. barbieQ is built upon the widely used SummarizedExperiment data structure, making it interoperable with low-level sequence processing tools. It provides data-driven quality control and filtering, implements statistical analysis for testing between samples, and supports visualisations throughout the analysis. Two statistical tests included in barbieQ are 1) Differential barcode proportion that identifies barcodes with significantly changing proportions between groups of samples, and 2) Differential barcode occurrence that identifies barcodes more frequently seen in the samples of one condition compared to another. barbieQ can handle complex experimental designs using regression models to test a factor of interest while accounting for other variables. Both tests maintain a type 1 error less than 5% under the null hypothesis using real-world and simulated data. Notably, in analysis of a real-world dataset of monkey blood stem cell differentiation, barbieQ effectively selected a larger set of informative barcodes and accurately identified novel NKCD56-CD16+ cell-specific clones whilst avoiding less-specific barcodes, in contrast to the original analysis based on heuristic cutoffs and visualisations commonly used in the field. Overall, barbieQ is a powerful Bioconductor package with greater power to detect biologically meaningful changes leading to robust and reproducible analysis.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.11.675529v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Computational screening and automatic filtering for the discovery of novel inhibitors of TMPRSS2, a type II transmembrane serine protease</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tongiorgi, L., Albani, S., Rigobello, L., Maccallini, C., Musiani, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A structure-based virtual screening method to find novel inhibitors of transmembrane Serine Protease 2 .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Transmembrane Serine Protease 2 (TMPRSS2) is a membrane protein of the type II serine protease family of enzymes implied in epithelial homeostasis. It is involved in several diseases, notably prostate cancer and SARS-CoV-2 infections. Over the years, only a few tested TMPRSS2 inhibitors showed consistent results. This prompted us to select it as target of structure-based virtual screening, to search for novel inhibitors among a library of 475,770 small molecules. Two sets of TMPRSS2 structures were selected, one taken from molecular dynamics simulations, the other from recently solved X-ray crystallographic structures. We designed a workflow to filter docking results in a reproducible way, allowing for a faster and more reliable selection. The program uses four metrics: the pose consistency of the ligand, docking score, number of interactions with key protein residues, and cluster analysis. This led to the selection and visual inspection of two sets of 500 compounds, which yielded 10 reasonable hit candidates.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.09.675039v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The genome assembly and detection of biosynthetic gene clusters among four novel microbial isolates from deep subsurface rock biofilms</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Agany, D. D. M., Zylla, J. L., Aryal, S., Gorbatenko, O., Bergmann, D., Gadhamshetty, V., Gnimpieba, E. Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We characterized the genomes of four novel isolates from biofilms collected in a previously abandoned gold mine, used as a model underground system.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Tunnels in deep underground mines form a unique interface between surface and subsurface habitats, where surface microbes can enter through ventilation systems. These extreme conditions foster microbial adaptations, including the potential to produce novel natural products of biotechnological and medicinal importance. In this study, we characterized the genomes of four novel isolates from biofilms collected in a previously abandoned gold mine, used as a model underground system. Samples were obtained from thin, whitish, glistening biofilms ("cave silver") on rock walls at 1,478 meters depth in SURF. From these samples, 10 GB of high-quality whole-genome sequences were generated, assembled, and annotated using multiple databases. The genomes were further analyzed for biosynthetic gene clusters (BGCs) encoding secondary metabolites using antiSMASH and NaPDoS. Draft assemblies revealed four genomes with diverse lengths and annotated features, including gene clusters associated with quorum sensing (QS). Several BGC-encoded compounds were identified, such as polyketides (PKS, PKS III, ketosynthase domain-KS), non-ribosomal peptides (NRPS, condensation domains), and terpenoids (terpenes I, polyenes type II). Enzyme analyses indicated both shared and unique activities among strains, highlighting potential cooperative and specialized functions within the biofilms. Additionally, CRISPR peptides were detected in three of the four strains. This work demonstrates how microbes colonizing extreme subsurface environments evolve specialized genomic features. Their biosynthetic capacity suggests the potential for producing natural products and secondary metabolites with applications in biotechnology and medicine.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.13.675994v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Automated Segmentation of Hepatic Vessels and Lobules in Whole-Slide Images Using U-Net Models</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bafna, M., KoÃànig, M., Saalfeld, S., Liska, V., Moulisova, V., Dahmen, U., Albadry, M. A. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a robust deep-learning pipeline using adaptive patch extraction and specialized U-Net architectures for segmenting vessels, bile ducts, and lobules in whole-slide histological images.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Automated analysis of hepatic vascular structures and lobules within whole-slide histological images is critical for ensuring accurate and timely morphometric evaluations and facilitating advancements in computational liver histology. Nonetheless, the intricate morphology of the tissue, variability in staining techniques, and the requirements for high-resolution imaging present substantial challenges to the precision of segmentation processes. We present a robust deep-learning pipeline using adaptive patch extraction and specialized U-Net architectures for segmenting vessels, bile ducts, and lobules in Glutamine Synthetase and Picro-Sirius-Red stained porcine liver sections. Our architecture incorporates a weight-boosted nnU-Net framework to effectively manage class imbalances and improve the detection of smaller vascular structures. Geometric data transformations enhanced the robustness and generalizability of the segmentation models. Evaluations conducted through five-fold cross-validation, as well as assessments utilizing independent test datasets, resulted in Dice similarity scores: 0.960 for lobules, 0.801 for central veins, 0.909 for hepatic arteries, 0.609 for portal veins, and 0.710 for bile ducts. The developed segmentation pipeline additionally supports comprehensive morphometric analyses of structural parameters, including the number and size (diameter, area) of vascular structures, bile ducts, and lobules. For e.g., the diameter of hepatic arteries ranges between 30-90 m. These findings underscore the practical relevance of adaptable segmentation frameworks in advancing computational histological analysis of liver tissue.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.08.674181v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrative proteomics and metabolomics map reveals key sectors of defense metabolism in ginger against Pythium myriotylum</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fernandez, F., Varghese, L., Ramaswami, V. M., Thomas, G., Kundu, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified TGA as the top defense-associated metabolite in ginger's rhizome rot and demonstrated strong anti-P. myriotylum activity of TGA, which inhibited mycelial growth and suppressed expression of key pathogenicity-related genes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Ginger (Zingiber officinale) cultivation is severely threatened by rhizome rot caused by Pythium myriotylum. Hitherto, detail molecular machineries and metabolic pathways of defense against P. myriotylum in edible ginger are elusive. To elucidate the defense mechanisms, we employed integrative quantitative proteomics (TMT labelling) and high resolution untargeted metabolomics to investigate temporal defense responses of ginger during infection. Proteomics analysis revealed enrichment of cellular detoxification and secondary metabolism pathways indicating co-activation of oxidative stress management and its mediated defense metabolite biosynthesis. Key phenylpropanoid enzymes (PAL, C4H, CAD) and LOX, a regulator of jasmonic acid (JA) biosynthesis, were significantly up-regulated, correlating JA signaling with secondary metabolic defenses. Metabolomics profiling confirmed accumulation of phenylpropanoids, flavonoids, and terpenoids, including known antimicrobial metabolites such as gingerol, shogaol, curcumin, and gingerdione, alongside expression of their key regulatory enzyme was also induced. Notably, machine learning identified the monoterpenoid TGA as the top defense-associated metabolite. Functional assays demonstrated strong anti-P. myriotylum activity of TGA, which inhibited mycelial growth and suppressed expression of key pathogenicity-related genes (NIP1, GHs, cellulase). These findings highlight coordinated activation of detoxification and secondary metabolic pathways in ginger defense and establish TGA as a promising bio-control agent against P. myriotylum.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.09.675089v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The Lotus japonicus alpha Expansin EXPA1 is recruited during intracellular and intercellular rhizobial colonization</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Montiel, J., Garcia-Soto, I., Monroy-Morales, E., Lace, B., Vestergaard, M., Sandal, N., Ott, T., Stougaard, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that alpha Expansin EXPA1 is critical for the successful establishment of Lotus-rhizobia symbiosis, regardless of the infection mechanism, and that the expression profile of EXPA 1 correlates with the symbiotic phenotype observed in homozygous mutants.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Most legumes establish a mutualistic association with rhizobia, a group of nitrogen-fixing bacteria. In Lotus japonicus, the symbiotic colonization occurs intracellularly, via root-hair infection threads by Mesorhizobium loti, or intercellularly, with Agrobacterium pusense IRBG74. In both mechanisms, cell wall remodeling is presumably an essential process. In plants, alpha-Expansins (EXPA) promote cell wall loosening by non-enzymatically triggering a pH-dependent relaxation. In this study, we show that LjEXPA1, is critical for the intracellular and intercellular symbiotic program in L. japonicus. Promoter activity and subcellular localization analyses revealed that EXPA1 is recruited at essential compartments and structures of epidermal and cortical cells in both mechanisms of rhizobial infection, such as the infection chambers, infection pockets and transcellular infection threads. Additionally, EXPA1-YFP abundantly accumulated in dividing cortical cells during nodule formation. The expression profile of EXPA1 correlates with the symbiotic phenotype observed in homozygous mutants disrupted in the EXPA1 gene (expA1-1 and expA1-2). Infection thread formation and intercellular colonization was drastically reduced in expA1-1 and expA1-2 mutants, respect to wild type plants. Similarly, nodule formation was significantly reduced in these mutants after M. loti or IRBG74 inoculation. Our results indicate that non-enzymatic cell wall remodeling by the alpha Expansin EXPA1 is crucial for the successful establishment of Lotus-rhizobia symbiosis, regardless of the infection mechanism.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.08.674960v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Radiotherapy drives lung metastasis in PDAC via a RhoGTPasesignaling shift towards MRCK-dependency</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> McLay, K. M., Tesson, M., Dutton, L., Sun, Y., Misquitta, J. M., Stevenson, K., Corbyn, R., Lilla, S., McGarry, L., Shaw, R., Miller, C. J., Carlin, L. M., Olson, M. F., Morton, J. P., Birch, J. L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We demonstrate for the first time that clinically relevant doses of image-guided targeted radiotherapy induce metastasis in a genetically engineered KPC mouse model of PDAC and identify a key role for myotonic dystrophy-related Cdc42-binding kinase in driving this response.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Pancreatic ductal adenocarcinoma (PDAC) is an aggressive cancer with an extremely poor prognosis, partly owing to its highly metastatic nature. Although radiotherapy is an effective and potentially curative treatment modality, research over the last few decades has indicated that it may induce a more metastatic phenotype in surviving cancer cells. We demonstrate for the first time that clinically relevant doses of image-guided targeted radiotherapy induce metastasis in a genetically engineered KPC mouse model of PDAC. Furthermore, this induction is largely driven by an organotropic switch in lung metastasis. Using an in vitro RNAi screening approach, we identified a key role for myotonic dystrophy-related Cdc42-binding kinase (MRCK) in driving this response. MRCK activity was spatially upregulated at the plasma membrane in response to radiotherapy in PDAC cell lines as well as at the invasive margins of PDAC tumors. This upregulation of activity was maintained in metastases, suggesting an important role for MRCK in not only triggering local invasion in response to radiotherapy but also promoting distant metastases. Importantly, inhibition of MRCK with a small-molecule inhibitor (BDP9066) specifically opposed radiation-driven MRCK upregulation and pro-metastatic response.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.09.673965v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Decoding interaction-induced proteome changes in co-cultures with hybrid quantification and SILAC-directed real-time search</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Carre, A., Ibanez-Molero, S., Peeper, D. S., Altelaar, M., Stecker, K. E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel and powerful approach combining SILAC, TMT and RTS to resolve cell-specific proteome dynamics in co-culture .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Intercellular communication between T cells and cancer cells plays a pivotal role in determining cancer cell survival or death. Yet, our understanding of this interaction remains incomplete. Methods to study heterotypic cell interactions are either limited to targeted studies relying on predefined set of proteins, or require cell separation, thus disrupting the native environment. Stable isotope labeling by amino acids in cell culture (SILAC) enables proteome distinction in heterologous co-cultures without the need for physical separation. But dynamic studies remain constrained by the need for numerous mass spectrometry (MS) runs, the challenges in detecting low-abundant proteins, particularly in immune cells and the limited data completeness due to the use of data-dependent MS1-based precursor quantification. To overcome these limitations, we evaluate the integration of SILAC with tandem mass tag (TMT) multiplexing and SILAC-directed real-time search (RTS). TMT labeling enables simultaneous analysis of multiple samples, while RTS-MS3 acquisition using SILAC-induced mass shifts as fixed modifications triggers MS3 scans for specific proteome populations within a mixed cell system, improving quantitative accuracy and proteome coverage for target protein populations. We benchmarked our acquisition methods using SILAC-labeled samples mixed at defined ratios and validated the approach in biologically relevant co-culture experiments. Additionally, we introduced a carrier channel to enhance detection of lower-abundant T cell proteins, while maintaining acceptable quantitative precision. Our results demonstrate that the combined SILAC-TMT-RTS strategy dramatically improves proteome depth, temporal resolution, and cell-type specificity for short-term co-culture interaction proteomics studies. In co-culture samples of T cells with non-small cell lung cancer cell lines that were either sensitive or resistant to T cell killing, our method revealed candidate mechanisms underlying their differential sensitivity. Our integrated approach combining SILAC, TMT and RTS to resolve cell-specific proteome dynamics in co-culture represents a novel and powerful advance.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.09.675040v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MRTX1133 is a potent non-covalent KRAS (G12C) inhibitor with tissue-specific activity</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sianoya, A. C., Zuo, Y., Pagba, C. V., Frost, J. A., Medina, P. M. B., Gorfe, A. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the KRAS (G12C) mutant KRAS also binds to G12C mutant KRas with high affinity and suppresses MAPK signaling in cancer cell lines harboring KRAS, but MRTX1133 robustly inhibits the proliferation of the pancreatic cancer cell line</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> KRAS is a high-value therapeutic target for the treatment of cancer. Two covalent inhibitors, sotorasib and adagrasib, which target a specific codon 12 mutation (G12C), have received accelerated approvals for clinical use. Studies of these inhibitors ushered in the development of new inhibitors such as MRTX1133, that had entered clinical trials as a KRAS (G12D)-selective, non-covalent inhibitor. However, the subsequent failure of sotorasib as monotherapy and the recent termination of an early-phase clinical trial for MRTX1133 indicates that developing clinically effective allele-specific KRAS inhibitors remains a challenge, and that there is a need for further evaluation of KRAS inhibition mechanisms. Here, we show that the reportedly KRAS (G12D)-selective MRTX1133 also binds to G12C mutant KRAS with high affinity and suppresses MAPK signaling in cancer cell lines harboring KRAS (G12C). Intriguingly, its effect on the proliferation of KRAS (G12C) cancer cells is context-dependent; MRTX1133 robustly inhibits the proliferation of the pancreatic cancer cell line MIA PaCa-2 as well as the tumor growth of MIA PaCa-2 mouse xenografts, but it has little effect in lung cancer cells. These findings, together with similar other recent reports, question if allele-specific KRAS inhibitors are truly selective and highlight the need for strategies that take into account tissue and context-specific processes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.09.674973v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Bidirectional Crosstalk Between Bladder Cancer Cells and Normal Fibroblasts Drives Phenotypic Reprogramming and Modulates Chemosensitivity</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gao, J., Ji, C. X., Hines, J., Stride, E., Bryan, R. T., Rohn, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that normal fibroblasts in bladder cancer are modulating the behaviour of cancer cells and show that fibroblast composition in the TME may influence individual variability in treatment response.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Bladder cancer is one of the most common malignancies worldwide and is marked by high recurrence rates, highlighting the urgent need to better understand the tumour microenvironment (TME). While the role of cancer-associated fibroblasts (CAF) in cancer progression and therapy resistance is well established, the contribution of normal fibroblasts (NF) remains underexplored. Here we dissect bidirectional interactions between bladder cancer cells and NF using indirect (conditioned media) and direct co-culture systems. Fibroblast-conditioned media (FCM) reduced proliferation in RT112 and T24 cells while accelerating cell migration. Immunofluorescence revealed a cadherin remodelling with increased N-cadherin and reduced E-cadherin after FCM exposure. On the stromal side, NF acquired CAF-like features within 48 h, with SMA and FAP increases evidenced by imaging in conditioned media and confirmed by flow cytometry in direct co-culture. In addition, direct co-culture reduced mitomycin C (MMC) efficacy in a fibroblast-ratio dependent manner in both cancer cell lines. It suggests that fibroblast composition in the TME may influence individual variability in treatment response. These findings highlight a significant role for NF in modulating bladder cancer behaviour and therapeutic resistance, supporting the need to consider fibroblast-cancer interactions in developing more effective treatment strategies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.09.675061v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Label-free monitoring of therapy response in 3D spheroids using lab-on-a-chip impedance spectroscopy</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Macke, G., Sheth, M., Sharma, M., Kongsomros, S., Lehn, M., Wise-Draper, T. M., Takiar, V., Esfandiari, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We demonstrate the application of Electrical impedance spectroscopy (EIS) to assess the bioeffects of radiotherapy on 3D head and neck cancer spheroids.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The high incidence and mortality of cancer continue to drive research and development of effective therapies. Lab-on-a-Chip (LOC) platforms have emerged as powerful alternatives to traditional biological evaluation methods, offering reduced complexity, lower costs, and improved throughput. In parallel, the integration of non-invasive and non-destructive sensing techniques have expanded opportunities for real-time and label-free analysis. Electrical impedance spectroscopy (EIS), which exploits the intrinsic dielectric properties of cells, has shown promise for the quantitative evaluation of 3D cellular structures. In this study, we demonstrate the application of LOC-based EIS to assess the bioeffects of radiotherapy on 3D head and neck cancer spheroids. Our results establish EIS as a viable tool for real-time monitoring of treatment-induced changes in 3D tumor models, supporting its potential in preclinical cancer research and therapeutic screening.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.08.674936v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Targeting HIF-2Œ± in Colorectal Cancer Reveals a Cholesterol Biosynthesis-Dependent Ferroptotic Vulnerability</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dalal, P. J., Singhal, R., Hu, B., Rossiter, N., Jain, C., Sajjakulnukit, P., Lyssiotis, C. A., Shah, Y. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-14
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identify a metabolic vulnerability of CRC to dual HIF-2 and cholesterol biosynthesis inhibition, supporting a clinically actionable strategy that leverages safe, FDA-approved statins to potentiate Hif-2-targeted therapy.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Colorectal carcinoma (CRC) remains a major cause of cancer-related mortality, with rising incidence in individuals under 55, highlighting the need for novel therapeutic strategies. Hypoxia-inducible factor 2 alpha (HIF-2) has been genetically validated as a critical driver of colorectal tumorigenesis, with intestinal epithelium-specific deletion in mice markedly reducing tumor formation. While selective HIF-2 inhibitors such as PT2385 are FDA-approved for renal cell carcinoma, pharmacologic HIF-2 inhibition has not been explored in CRC. Here, we demonstrate that HIF-2 inhibition alone fails to suppress CRC growth in vitro under normoxic or hypoxic conditions and in xenograft models in vivo. To identify vulnerabilities induced by HIF-2 blockade, we performed an unbiased CRISPR metabolic screen, revealing cholesterol biosynthesis as a critical dependency. Targeting this pathway with clinically approved statins (atorvastatin, pitavastatin, simvastatin) synergized with PT2385 to suppress CRC cell growth, reduce colony formation, and enhance cell death. Mechanistic studies show that combined HIF-2 and HMG-CoA reductase inhibition promotes ferroptosis, characterized by increased lipid peroxidation and depletion of antioxidant metabolites. These effects are fully reversed by the ferroptosis inhibitor liproxstatin-1. Genetic knockdown of HIF-2 or HMG-CoA reductase recapitulated the enhanced sensitivity to combination therapy. In vivo, co-administration of PT2385 and atorvastatin significantly reduced tumor growth and increased ferroptotic cell death in xenografts, confirming the mechanistic link. Collectively, these findings uncover a metabolic vulnerability of CRC to dual HIF-2 and cholesterol biosynthesis inhibition, supporting a clinically actionable strategy that leverages safe, FDA-approved statins to potentiate HIF-2-targeted therapy.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.08.674939v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A perspective on integrating digital pathology, proteomics, clinical data and AI analytics in cancer research.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> J√©ssica Guedes, Nicole Woldmar, A Marcell Szasz, Elisabet Wieslander, Krysztof Paw≈Çowski, Peter Horvatovich, Johan Malm, Leticia Szadai, Istv√°n Bal√°zs N√©meth, Gy√∂rgy Marko-Varga, Jeovanis Gil</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-16
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a new approach to precision cancer research by bridging the gap between traditional histopathological evaluation and modern molecular analytics, which leads to a multidimensional view of tumor biology that captures both morphological nuances and molecular heterogeneity.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Nearly 40 % of individuals will be diagnosed with cancer in their lifetime, translating to an estimated 20 million new cases annually. Despite remarkable therapeutic advances, only 15-20 % of patients achieve durable responses to immunotherapy, and the high cost of treatment (illustrated by immune checkpoint inhibitors like pembrolizumab and nivolumab, totaling roughly $191,000 per year) remains a formidable global challenge. The convergence of digital pathology, high-throughput molecular profiling, and advanced computational strategies has the potential to transform cancer research. By integrating high-resolution morphological data with proteomic, transcriptomic, and spatial molecular insights, we can elucidate the complex interplay between tumor cells and their microenvironment. In this perspective, we review how emerging techniques, from AI-driven image analysis to deep visual proteomics, can accelerate biomarker discovery, refine patient stratification, and ultimately improve clinical outcomes. We illustrate these principles with a case study in melanoma, where the integration of digital pathology and deep proteomic profiling uncovered a molecular signature predictive of recurrence in early-stage disease. As these technologies evolve, we foresee a future of precision oncology characterized by the seamless integration of morphological, clinical, and molecular data enabled by AI-driven analytics. SIGNIFICANCE: This perspective represents a pivotal step toward transforming cancer research by bridging the gap between traditional histopathological evaluation and modern molecular analytics. By integrating digital pathology with spatial proteomics and advanced AI-driven analytics, our approach provides a multidimensional view of tumor biology that captures both morphological nuances and molecular heterogeneity. This comprehensive framework not only enhances our understanding of the tumor microenvironment but also facilitates the discovery of robust biomarkers for disease recurrence and therapeutic response. Ultimately, our findings underscore the potential of precision oncology to tailor treatment strategies to individual patient profiles, thereby improving clinical outcomes and guiding the next generation of personalized cancer care.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40680990/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Transcriptomic and single-cell insights into mitochondrial genes NDUFA8, ECI2, and ACADM in acute myocardial infarction.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ying Hao, ChengHui Fan, Wei Wen, RuiLin Li, Yang Gao, Xia Hou, Linxiang Lu, YunLi Shen</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study investigates mitochondrial function-related genes (MFRGs) in acute myocardial infarction (AMI) through bioinformatics and rigorous experimental validation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mitochondrial function plays a crucial role in understanding the pathogenesis of acute myocardial infarction.This study investigates mitochondrial function-related genes (MFRGs) in acute myocardial infarction (AMI) through bioinformatics and rigorous experimental validation. Using three integrated GEO datasets (149 samples: 80 AMI, 69 control), we applied machine learning methods (Random Forest, Support Vector Machine, Least Absolute Shrinkage and Selection Operator), phenotype scoring, consensus clustering, and weighted gene co-expression network analysis in three subgrouping stages to refine the selection of MFRGs. The intersection of three subgroup analyses results identified 11 key Mitochondrial function-Related Differentially Expressed Genes (MRDEGs). Gene Ontology / Kyoto Encyclopedia of Genes and Genomes analysis (GOKEGG), Gene Set Enrichment Analysis (GSEA), Gene Set Variation Analysis (GSVA), and immune infiltration analyses revealed significant pathways and suggested alterations in the composition of immune cell subpopulations. Single-cell analysis revealed increased expression of MRDEGs (NDUFA8, ECI2, and ACADM) in cardiomyocytes, fibroblasts, and macrophages, along with dynamic expression trends along the pseudotime trajectory. Subgroup analysis of cardiomyocytes based on mitochondrial gene scoring was performed to explore functional enrichment characteristics. Furthermore, we robustly validated these genes' expression in the cardiomyocyte hypoxia/reoxygenation model and the mouse model of myocardial infarction using flow cytometry, immunohistochemistry, and Western blot. Finally, this study constructed the mRNA regulatory network of key MRDEGs and preliminarily explored the potential therapeutic value of valproic acid and acetaminophen through molecular docking.These findings reveal the role of mitochondrial dysfunction in the mechanisms of AMI and its associated cell subpopulations, along with the involved biological pathways, offering new insights for AMI research.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40619072/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Digital health framework for the predictive surveillance and diagnosis of atopic dermatitis.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Seungdae Oh, Haeil Byeon, Jonathan Wijaya</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-06-11
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A digital health framework combining urban big data analytics, machine learning modeling, and environmental bioinformatics for the predictive surveillance and diagnosis of the nationwide AD prevalence in Korea.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Atopic dermatitis (AD) is an inflammatory skin disease with immunological and environmental triggers that reduces the quality of life and increases the burden on health services. It is thus important to establish effective surveillance and diagnosis methods for the development of preventive and therapeutic interventions. In line with this, the present study established a digital health framework combining urban big data analytics, machine learning modeling, and environmental bioinformatics for the predictive surveillance and diagnosis of the nationwide AD prevalence in Korea. In this process, urban big data from environmental (e.g., immune response inducers), crowdsourced (web search keywords related to AD symptoms), and municipal microbiome sources (AD-associated bacteria detectable in wastewater) were combined and employed as input variables. Data preprocessing (i.e., feature selection, scaling, and normalization), model testing and selection, and hyperparameter tuning were then used to improve the prediction accuracy for AD prevalence. By applying explainable artificial intelligence methods, highly explanatory predictors, such as specific skin disease keywords associated with AD patients and environmental and inflammatory factors, were identified. Environmental genomics revealed that Streptococcus strains were dominant in human-derived wastewater, with operational taxonomic units that were strongly associated with inflammation-inducing bacteria originating from AD patients. Bioinformatic analysis subsequently revealed the pathogenotype and resistotype of these inflammation-related bacteria. Overall, our digital health framework holds great promise as an alternative to conventional complex and costly surveillance systems for the proactive guidance of the decision-making of health professionals regarding the surveillance, diagnosis, and therapeutic treatment of environmental diseases.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40532561/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

