<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Friday, 11 July 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comprehensive multi-omics and machine learning framework for glioma subtyping and precision therapeutics.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yi Ding, Zhaiyue Xu, Wenjing Hu, Peng Deng, Mian Ma, Jiandong Wu</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We derive three integrative molecular subtypes of glioma and a transferable prognostic signature for high-risk tumours, and propose a blueprint for biomarker-guided therapy.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Glioma is a highly heterogeneous and aggressive brain tumour that demands an integrated understanding of its molecular and immunological landscape. We collected multi-omics data from 575 TCGA diffuse-glioma patients (156 IDH-wild-type WHO-grade 4 glioblastomas and 419 IDH-mutant WHO-grade 2/3 diffuse gliomas) together with two validation cohorts (CGGA n = 970; GEO n = 110). Using the MOVICS framework, we derived three integrative molecular subtypes-CS1, CS2 and CS3. Ten machine-learning algorithms in MIME were benchmarked, and the Lasso + SuperPC combination yielded an eight-gene GloMICS (Glioma Multi-Omics Consensus Signature) prognostic score. The subtypes display discrete biology: CS1 (astrocyte-like) is characterized by glial lineage features, immune-regulatory signaling, and relatively favorable prognosis; CS2 (basal-like/mesenchymal) shows epithelial-mesenchymal transition, stromal activation, and high immune infiltration, including PD-L1 expression; CS3 (proneural-like/IDH-mut metabolic) exhibits metabolic reprogramming (OXPHOS, hypoxia) and an immunologically cold tumour microenvironment (TME). CS2 is associated with the worst overall survival, whereas CS1 confers the most favourable outcome. Dual checkpoint blockade or T-cell-rejuvenation strategies may benefit CS2 tumours, while metabolic inhibitors could prove effective in CS3. The eight-gene GloMICS score outperformed 95 published prognostic models (C-index 0.74-0.66 across TCGA, CGGA and GEO). TME deconvolution, immune checkpoint profiling and TIDE analysis indicate that high-risk GloMICS tumours harbour immunosuppressive fibroblast-rich niches and exhausted CD8⁺ T cells. Connectivity-map screening nominated dabrafenib, irinotecan and three additional CTRP/PRISM compounds as candidate agents for the high-risk group. Our study establishes robust glioma subtypes and a transferable prognostic signature, offering a blueprint for biomarker-guided therapy. Future work should include single-cell and immunohistochemical validation of subtype hallmarks and prospective trials stratified by GloMICS score.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40640380/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Automated Discovery of Patterns in T-Cell Receptor Physicochemical Signatures</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Shams, Z., Bishop, E., Mckee-Reid, L., Rumbelow, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We use the Discovery Engine, a novel system for automated knowledge discovery from data, to classify published tumour antigen-reactive T-cell receptors (TCRs) and non-reacted TCRs collected from cancer patients.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurately distinguishing antigen-reactive from non-reactive T-cell receptors (TCRs) is critical for advancing TCR-based immunotherapies and vaccines. Predicting antigen reactivity from physicochemical properties of the TCR sequence alone could enable rapid, low-cost identification of TCRs of interest, accelerating therapeutic discovery. In this paper, we use the Discovery Engine, a novel system for automated knowledge discovery from data, to classify published tumour antigen-reactive and non-reactive TCRs collected from cancer patients. Beyond classification, the Discovery Engine extracts interpretable combinatorial patterns (e.g., combinations of CDR3 length, net charge, and hydrophobicity) that predict whether a TCR is antigen-reactive. These patterns point to biologically meaningful features linked to tumour antigen recognition and could inform rational TCR design and prioritization. Notably, over half of the predictive patterns involve features from both the alpha and beta chains, highlighting the importance of considering both in assessing antigen specificity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.07.663455v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MolUNet++: Adaptive-grained Explicit Substructure and Interaction Aware Molecular Representation Learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> XU, F., Yang, Z., Su, W., Wang, L., Meng, D., Long, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose MolUNet++, a novel model that employs Molecular Edge Shrinkage Pooling (MESPool) for hierarchical substructure extraction, utilizes a Nested UNet framework for multi-granularity feature integration, and incorporates a substructure masking explainer for quantitative fragment analysis</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Molecular representation learning is a critical task in AI-driven drug development. While graph neural networks (GNNs) have demonstrated strong performance and gained widespread adoption in this field, efficiently extracting and explicitly analyzing functional groups remains a challenge. To address this issue, we propose MolUNet++, a novel model that employs Molecular Edge Shrinkage Pooling (MESPool) for hierarchical substructure extraction, utilizes a Nested UNet framework for multi-granularity feature integration, and incorporates a substructure masking explainer for quantitative fragment analysis. We evaluated MolUNet++ on tasks including molecular property prediction, drug-drug interaction (DDI) prediction, and drug-target interaction (DTI) prediction. Experimental results demonstrate that MolUNet++ not only outperforms traditional GNN models in predictive performance but also exhibits explicit, intuitive, and chemically logical interpretability. This capability provides valuable insights and tools for researchers in drug design and optimization.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.07.663425v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Benchmarking Large Language Models for Predictive Modeling in Biomedical Research With a Focus on Reproductive Health</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sarwal, R., Tarca, V., Kalavros, N., Bhatti, G., Bhattacharya, S., Butte, A. J., Romero, R. J., Stolovitzky, G., Oskotsky, T. T., Tarca, A. L., Sirota, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Generative AI, particularly large language models (LLMs), are able to generate functional R and Python code for predictive modeling tasks, leveraging standardized molecular datasets from several recent DREAM challenges.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Generative AI, particularly large language models (LLMs), is increasingly being used in computational biology to support code generation for data analysis. In this study, we evaluated the ability of LLMs to generate functional R and Python code for predictive modeling tasks, leveraging standardized molecular datasets from several recent DREAM (Dialogue for Reverse Engineering Assessments and Methods) Challenges focused on reproductive health. We assessed LLM performance across four predictive tasks derived from three DREAM challenges: gestational age regression from gene expression, gestational age regression from DNA methylation profiles, and classification of preterm birth and early preterm birth from microbiome data. LLMs were prompted with task descriptions, data locations, and target outcomes. LLM-generated code was then run to fit and apply prediction models and generate graphics, and they were ranked based on their success in completing the tasks and achieving strong test set performance. Among the eight LLMs tested, o3-mini-high, 4o, DeepseekR1 and Gemini 2.0 completed at least one task without error. Overall, R code generation was more successful (14/16 tasks) than Python (7/16), attributed to the utility of Bioconductor packages for querying Gene Expression Omnibus data. OpenAI's o3-mini-high outperformed others, completing 7/8 tasks. Test set performance of the top LLM matched or exceeded top-performing teams from the original DREAM challenges. These findings underscore the potential of LLMs to enhance exploratory analysis and democratize access to predictive modeling in omics by automating key components of analysis pipelines, and highlight the potential to increase research output when conducting analyses of standardized datasets from public repositories.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.07.663529v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Chromosome-level assembly of the octoploid cloudberry(Rubus chamaemorus)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Strand, M. A., To, T.-H., Torresen, O. K., Fullmer, S. E., Skage, M., Tooming-Klunderud, A., Sandve, S. R., Jakobsen, K. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We describe a chromosome-level genome assembly from an individual male plant of the cloudberry (Rubus chamaemorus) that shows polyploidization involving one divergent ({beta}) and three closely related () subgenomes.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We describe a chromosome-level genome assembly from an individual male plant of the cloudberry (Rubus chamaemorus). The haplotype-resolved assemblies contain one pseudo-haplotype spanning 1198 megabases and one pseudo-haplotype spanning 1161 megabases. Most of these two assemblies, 93.57% and 96.55% respectively, are each scaffolded into 28 pseudo-chromosomes. Both assemblies show high completeness, with the same BUSCO completeness score of 99.2%. Most BUSCO genes are duplicated in both pseudo-haplotypes, in line with the polyploid nature of the cloudberry genome. The assemblies contain 74,132 and 70,692 predicted protein-coding genes, respectively. Analysis of repetitive sequences classified ~60% of each haplotype as repeats. Comparative synteny with red raspberry (Rubus idaeus) reveals a 4:1 chromosome correspondence, supporting an octoploid origin. Ks distributions and k-mer clustering indicate a recent polyploidization involving one divergent ({beta}) and three closely related () subgenomes. Genome skimming data mapped to our assembly link the -subgenomes to a putative hexaploid lineage related to Rubus arcticus, and the {beta}-subgenome to Rubus pedatus, previously proposed as the maternal ancestor.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.07.663483v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genetic load proxies do not predict fitness better than inbreeding does in a wild population</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Robledo-Ruiz, D. A., Sunnucks, P., Quin, B., Magrath, M. M. J. L., Harley, D., Pavlova, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A key goal in conservation is managing endangered populations in order to maximise individual and population fitness, and genetic load proxies based on evolutionary conservation predict individual fitness better than inbreeding does.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> A key goal in conservation is managing endangered populations in order to maximise individual and population fitness. Genetic management that reduces inbreeding and increases genetic diversity has been consistently successful, but recent attention has been drawn to managing genetic load instead. We investigated whether genetic load proxies based on evolutionary conservation predict individual fitness better than inbreeding does. We re-sequenced the genomes of 43 wild helmeted honeyeaters with lifetime fitness data, and 28 individuals of other subspecies. None of the 28 genetic load proxies we tested predicted lifetime reproductive success better than three inbreeding metrics did, and different load proxies were inconsistent regarding the occurrence of purging or accumulation of deleterious alleles. Genetic load proxies need further validation as fitness indicators for conservation management.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.07.663445v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MuSHIN: A multi-way SMILES-based hypergraph inference network for metabolic model reconstruction</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhao, Y., Chen, Y., Yu, Y., Liu, X., Du, J., Wen, J., Liao, C., Sun, Q., Wang, R., Chen, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A novel deep hypergraph learning method that integrates network topology with biochemical domain knowledge to predict missing reactions in genome-scale metabolic models .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Genome-scale metabolic models (GEMs) are indispensable tools for probing cellular metabolism, enabling predictions of metabolic fluxes, guiding strain optimization, and advancing biomedical research. However, their predictive capacity is often compromised by incomplete reaction networks, stemming from gaps in biochemical knowledge, annotation inaccuracies, and insufficient experimental validations. Here we present MuSHIN (Multi-way SMILES-based Hypergraph Inference Network), a novel deep hypergraph learning method that integrates network topology with biochemical domain knowledge to predict missing reactions in GEMs. Evaluated on 926 high- and intermediate-quality GEMs with artificially removed reactions, MuSHIN significantly outperforms state-of-the-art methods, achieving up to a 17% improvement across multiple metrics and maintaining robust recovery even under severe network sparsity. Furthermore, MuSHIN substantially enhances phenotypic predictions in 24 draft GEMs associated with fermentation by resolving critical metabolic gaps, as validated against experimental measurements. Together, these findings highlight MuSHIN's potential to advance GEM reconstruction and accelerate discoveries in systems biology, metabolic engineering, and precision medicine.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.07.663526v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DREAM-GNN: Dual-route embedding-aware graph neural networks for drug repositioning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhao, Y., Chen, Y., Du, J., Sun, Q., Wang, R., Chen, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present DREAM-GNN (Dual-Route Embedding-Aware Model for Graph Neural Networks), a deep graph learning framework that incorporates biomedical domain knowledge with two complementary graphs capturing both topological structure and feature similarity to enable accurate and biologically meaningful prediction of drug-disease associations</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Drug repositioning presents a compelling strategy to accelerate therapeutic development by uncovering new indications for existing compounds. However, current computational methods are often limited in their ability to integrate heterogeneous biomedical data and model the intricate, multi-scale relationships underlying drug-disease associations, while large-scale experimental validation remains prohibitively resource-intensive. Here we present DREAM-GNN (Dual-Route Embedding-Aware Model for Graph Neural Networks), a multi-view deep graph learning framework that incorporates biomedical domain knowledge with two complementary graphs capturing both topological structure and feature similarity to enable accurate and biologically meaningful prediction of drug-disease associations. Extensive experiments on benchmark datasets demonstrate that DREAM-GNN significantly outperforms current state-of-the-art methods in recovering artificially removed repositioning candidates. These results establish DREAM-GNN as a robust and generalizable computational framework with broad potential to streamline drug discovery and advance precision medicine.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.07.663530v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Decoding Lymphangioleiomyomatosis (LAM) Niche Environment via Integrative Analysis of Single Cell Multiomics and Spatial Transcriptomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chen, K., Zhao, S., Guo, M., Reza, H., Wagner, A., Cakar, A. C., Jiang, C., Zhang, E., Green, J., Martin, E., Wikenheiser-Brokamp, K., Perl, A.-K. T., Sinner, D. I., Yu, J. J., Xu, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified two spatially and functionally distinct LAM subtypes: LAMCORE1 and LAMcORE2, with uterine origin, and a high-resolution blueprint of LAM pathogenesis.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Lymphangioleiomyomatosis (LAM) is a rare, progressive lung disease characterized by cystic destruction and metastatic growth of smooth muscle-like cells. Despite advances in understanding its genetic basis, the cellular heterogeneity, regulatory mechanisms, and microenvironmental interactions driving LAM progression remain poorly defined. In this study, we employed an integrative multi-omics approach combining single-cell RNA sequencing (scRNA-seq), single-nucleus ATAC sequencing (snATAC-seq), and high-resolution spatial transcriptomics (Visium, Visium HD, and Xenium) to decode the LAM niche in its native environment. We identified two spatially and functionally distinct LAM subtypes: LAMCORE1 and LAMCORE2. LAMCORE1 cells exhibited a uterine smooth muscle-like phenotype, expressing associated markers (ACTA2, MYH11) and were enriched in MTORC1 signaling and myogenic pathways, supporting a uterine origin. In contrast, LAMCORE2 cells displayed fibroblast-like features, with upregulated extracellular matrix (ECM) remodeling genes (COL1A1, MMP11) and epithelial-to-mesenchymal transition (EMT) pathways, suggesting a role in niche formation. Pseudotime and regulon analyses revealed dynamic transitions between these subtypes, driven by distinct transcriptional networks (e.g., HOX/PBX in LAMCORE1, TWIST/ZEB in LAMCORE2). The presence of the two distinct LAM subtypes was further validated by RNAscope and immunofluorescence microscopy. We identified LAM-associated fibroblasts (LAFs) as activated stromal cells expressing canonical markers (FAP, S100A4, VIM, IGFBP7, SPARC) and localized within LAM lesions. Subpopulations of LAFs, LAF-seed (proximal to LAMCORE1) and LAF-niche (surrounding LAM niches), exhibited unique functional profiles, including ECM deposition, TGF-{beta} signaling, and myofibroblast activation. Regulatory network analysis pinpointed EGR1 as a central hub governing LAF phenotype. Our comprehensive spatial profiling revealed niche structures dominated by LAMCORE1 cells and surrounded by lymphatic endothelial cells (LECs), LAFs, scattered LAMCORE2 cells, macrophages, and reprogrammed alveolar epithelial cells (AT2). ECM remodeling and aberrant organization of cable-like structures (-smooth muscle actin+) of the LAM niches were further validated by second harmonic generation microscopy. These findings provide a high-resolution blueprint of LAM pathogenesis, highlighting the interplay between uterine-derived LAMCORE cells, activated fibroblasts, and the remodeled lung microenvironment. They significantly enhance our understanding of the LAM niche microenvironment and offer insights into potential therapeutic targets and strategies for managing this complex disease.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.07.663390v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AmyloGram Reveals Amyloidogenic Potential in Stroke Thrombus Proteomes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kell, D. B., Doyle, K. M., Salcedo-Sora, E., Sekhar, A., Walker, M., Pretorius, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> AmyloGram reliably identifies known amyloid-forming proteins and reveals that stroke thrombi are enriched for proteins with high amyloidsogenic potential.</p><p style="font-size: 16px; text-align: justify;">Background: Amyloidogenic proteins play a central role in a range of pathological conditions, yet their presence in thrombi has only recently been recognized. Whether computational prediction tools can identify amyloid-forming potential in thrombus proteomes remains unclear. Methods: AmyloGram is a computational tool that estimates amyloid-forming potential based on n-gram sequence encoding and random forest classification. Using AmyloGram, we analyzed 204 proteins tagged by humans as amyloidogenic in UniProt. We then applied the same approach to proteins identified in thrombi retrieved using mechanical thrombectomy from patients with cardioembolic and atherothrombotic stroke. In addition we used AmyloGram to analyse the amyloidogenicity of 83,567 canonical human protein sequences. Results: Among the UniProt-annotated "amyloid" set, nearly all proteins received AmyloGram scores above 0.7, including 23 of the 24 human proteins. Even the lowest-scoring human protein, lysozyme (scoring 0.675), is known to form amyloid under certain conditions. In thrombi from both stroke subtypes, all detected proteins had AmyloGram scores above 0.7, suggesting a high likelihood of amyloid content. A majority of unannotated proteins also achieve AmyloGram scores exceeding 0.7. Conclusions: AmyloGram reliably identifies known amyloid-forming proteins and reveals that stroke thrombi are enriched for proteins with high amyloidogenic potential. These findings support the hypothesis that thrombus formation in stroke involves amyloid-related mechanisms and warrant further investigation using histological and functional validation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.07.663482v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Light-up nanostructures with allosteric DNA mimics of GFP</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, T., Qian, X., Zhang, J., Cui, H., Bai, T., Wang, W., Wang, Y., Gao, Z., Han, D., Zeng, W., Wei, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> In this study, we present a systematic approach for the rational design of synthetic allosteric DNA aptamers, allowing for the engineering of highly tunable fluorescent properties.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In this study, we present a systematic approach for the rational design of synthetic allosteric DNA aptamers. This methodology enables precise control over the allosteric ON-OFF transition in fluorescent DNA aptamers, allowing for the engineering of aptamers with highly tunable fluorescent properties. When combined with toehold-mediated strand displacement, we have developed a series of allosteric aptamers in which the target sequence functions as a specific allosteric modulator. Furthermore, these aptamers have been applied in synthetic DNA computing and in the construction of responsive nanostructures that light up upon activation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.10.664110v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Metalorian: De Novo Generation of Heavy Metal-Binding Peptides with Classifier-Guided Diffusion Sampling</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, Y., Srijay, D., Quinn, Z., Chatterjee, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Metalorian, a conditional diffusion model that generates de novo heavy metal-binding peptides, guided by MetaLATTE, a multi-label classifier fine-tuned to include underrepresented metals.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Heavy metal contamination is a severe and ongoing global problem that demands selective and efficient chelation strategies for environmental remediation. As a solution, we introduce Metalorian, a conditional diffusion model that generates de novo heavy metal-binding peptides, guided by MetaLATTE, a multi-label classifier fine-tuned to include underrepresented metal classes. Leveraging both continuous protein embeddings and discrete metal-binding constraints, Metalorian produces peptides with controllable lengths and user-defined binding specificity while preserving essential physicochemical properties. Crucially, our data augmentation strategy enables robust classification and generation even with sparse training data for certain metals. We demonstrate the effectiveness of our approach through peptides designed for copper (Cu), cadmium (Cd), and zinc (Zn), which retain key features such as net charge and hydrophobicity, while significantly reducing sequence length and molecular weight compared to known metal-binding proteins. Metalorian-generated peptides exhibit stable conformations and favorable binding energetics, as confirmed through molecular dynamics simulations. To validate binding in vitro, we developed a streamlined SUMO-fusion expression and cleavage system coupled with ELISA-based quantification, confirming robust Cu and Zn binding by multiple Metalorian-generated peptides. Overall, our work establishes a foundational platform for engineering heavy metal-binding peptides tailored to diverse and underrepresented targets, and highlights the potential of well-trained continuous latent spaces for diffusion-based de novo peptide design.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.10.664242v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A LARGE-SCALE EVOLUTIONARY AND STRUCTURAL ANALYSIS OF CLC CHANNELS AND TRANSPORTERS</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mishra, A., Diaz Vazquez, G., Robertson, J. L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-07-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that it is possible to scale up the structural information available using AlphaFold2 predictions and identify additional structural differences associated with each mechanistic class.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The CLC family of membrane transport proteins consists of chloride channels and anion/proton antiporters that share a similar structural scaffold. How the same fold accommodates two fundamentally distinct mechanisms is poorly understood, and while the current set of experimental structures provide some information, the changes appear limited. In this study, we show that it is possible to scale up the structural information available using AlphaFold2 predictions and identify additional structural differences associated with each mechanistic class. A phylogenetic analysis is carried out across all known CLC genes to expand the classification to include 569 channel and 1,051 transporter homologs that have been modeled. Using distance matrices, we validate AlphaFold2's ability to detect subtle structural differences among experimentally determined CLCs and use a random forest classifier to predict CLC channel vs. transporter sub-types to learn the structural changes that are the most important in the decision. The structural changes identified overlap with and contextualize the changes observed in experimental structures, expanding structural information across sequence space. The highest ranked change includes a contraction of distances between dimerization interface helices H, I, P & Q relative to the subunit core in the channel sub-types. This study lays out an approach for quantitative, large-scale structural analyses beyond experimental data and paves the way towards structural studies expanding on different conformational states and other protein families.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.07.07.663545v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

