<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Sunday, 10 August 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Examining the heritability of functional brain networks in adolescence</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Coffman, C., Koirala, S., Hermosillo, R., Lundquist, J., Grimsurd, G., Miranda Dominguez, O., Weldon, K. B., Anderson, M., Madison, T., Nelson, S., Elison, J. T., Wilson, S., Fair, D., Tervo-Clemmens, B., Basu, S., Feczko, E. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-10
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We found low SNP heritability for brain functional connectivity and topography and found additive genetic effects are minimally associated with functional brain features in adolescents highlighting the importance of considering both genetic and environmental factors.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Discovering mechanisms underlying mental illness requires disentangling genetic and environmental factors influencing mental health. Researchers have started investigating the brain's role as a potential intermediate biomarker linking genes and environment to mental health; however, understanding how much genetics shapes adolescent brain function remains elusive. Using data from the ABCD study (5,247 unrelated, 330 dizygotic, 248 monozygotic twin subjects), we estimated the heritability of functional connectivity and topography using both SNP and twin data. SNP-based heritability was calculated using genetic correlation and the recently developed AdjHE-RE estimator. We found low SNP heritability for brain functional connectivity (median = 2e-10% Gordon, 5.8e-7% probabilistic) and topography (max = 2%). Twin estimates using ACE models replicated prior findings from the literature (median = 8.9e-6% Gordon, 6% probabilistic, 27% topography). This suggest that additive genetic effects are minimally associated with functional brain features in adolescents highlighting the importance of considering both genetic and environmental factors when studying the development of functional brain networks and their relevance to mental health.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.08.669358v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multimodal analysis identifies pericyte-centered signaling programs altered by sex and brain region in Alzheimer's Disease</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> El-Ghazawi, K., Aberra, Y. T., Veeramasu, Y., Fares, W. A., Isakson, B. E., Sontheimer, H., Eyo, U. B., Peirce, S. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that pericyte-driven communication changes in NVU dysfunction in Alzheimer's disease are a mechanistic contributor to female-biased vulnerability in the disease and support the need for sex-aware and region-specific approaches in neurodegeneration research.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Pericytes are critical components of the neurovascular unit (NVU), regulating endothelial cell (EC) stability, blood-brain barrier (BBB) integrity, and neuroimmune signaling. However, their role in Alzheimer's Disease (AD), particularly in the context of sex differences and brain region specificity, remains poorly defined. Here, we use single-nucleus RNA sequencing (snRNA-seq) to characterize transcriptional and intercellular signaling changes in pericytes across the middle temporal gyrus (MTG) and dorsolateral prefrontal cortex (DLPFC) of the same AD and non-AD donors, stratified by sex. Using LIANA and Tensor-cell2cell, we identify latent communication programs altered in female AD donors, including a pericyte-EC signaling pattern that activates TGF-beta; via extracellular matrix ligands and is upregulated in the MTG but not the DLPFC. A second communication pattern, downregulated in female AD donors, reveals impaired estrogen pathway signaling through ligand-receptor interactions between pericytes and astrocytes. Supporting this, we observe reduced expression of pericyte-derived neuroligins and increased pericyte-astrocyte separation in a spatial transcriptomic subset. Additionally, we identify a microglia-to-pericyte signaling program conserved across brain regions, enriched for inflammatory pathways including hypoxia and p53, and elevated in both male and female AD donors with regional specificity. This result contrasts with the more sex- and region-specific pericyte signaling programs and suggests parallel mechanisms of NVU disruption between brain regions in AD. Our findings reveal brain region-specific and sex-specific pericyte signaling changes in AD and implicate vascular-, immune-, and synapse-associated pathways in NVU dysfunction. Altogether, the data suggest pericyte-driven communication as a mechanistic contributor to female-biased vulnerability in AD and support the need for sex-aware and region-specific approaches in neurodegeneration research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.06.668915v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">LLM-Assisted Functional Gene Annotation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Boyce, C., Pereira, C., Kim, G. D., Petti, A. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present Artificial Intelligence Gene Enrichment (AIGE), a careful automation of the current state of the art process used by bioinformatic experts to determine functions enriched in novel or experimentally derived gene lists.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Functional gene annotation is a highly manual and subjective process that requires analysis of large amounts of statistical and literature results. Here we present Artificial Intelligence Gene Enrichment (AIGE), a careful automation of the current state of the art process used by bioinformatic experts to determine functions enriched in novel or experimentally derived gene lists. In 1206 test cases, AIGE is able to accurately recover 87% of biological functions, pathways, and cell types, is robust to noise contamination, and accurately assesses its own self-confidence. AIGE reports provide accurate functional annotations and encourage further research in a broad set of contexts.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.06.668957v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Deriving longitudinal tumor phylogenies from single-cell sequencing data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Jakatdar, A., Sashittal, P., Raphael, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce Phyllochron, an algorithm that builds a longitudinal phylogeny from cancer cells sequenced from multiple timepoints and evaluate whether the constraints on clonal proportions imposed by longitudinal sampling are well supported by the data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Tumors evolve over time and in response to treatment, leading to changes in the proportions of clones within the tumor. Single-cell sequencing of tumor samples from multiple timepoints enables the reconstruction of clonal evolution and tracking of temporal changes in tumor composition. However, the high rates of missing data in single-cell sequencing complicate evolutionary analyses, and can lead to implausible conclusions, such as the reappearance of extinct clones. We introduce Phyllochron, an algorithm that builds a longitudinal phylogeny from cancer cells sequenced from multiple timepoints and evaluates whether the constraints on clonal proportions imposed by longitudinal sampling are well supported by the data. Phyllochron relies on a novel mathematical formulation of a longitudinal perfect phylogeny, an extension of the perfect phylogeny model that is widely used in cancer evolution. We show that Phyllochron outperforms existing phylogeny inference methods on simulated single-cell sequencing data. Applied to longitudinal single-cell DNA sequencing data from an acute myeloid leukemia (AML) patient, Phyllochron constructs a longitudinal phylogeny containing rare cancer clones that persist through multiple cycles of targeted drug treatment, a crucial finding missed by existing phylogeny inference methods. The Phyllochron statistical test supports the presence of these rare clones.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.06.668977v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Chamber-Specific Transcriptomic Insight into Cardiac Development using Guinea Pig and Human Heart Tissue</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Salameh, S., Guerrelli, D., Swift, L., Haski, A., Bruce, A., Desai, M., d'Udekem, Y., Posnack, N.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A molecular framework for understanding age and chamber specific cardiac development, supporting the guinea pig as a promising preclinical model for studying human heart maturation.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The heart undergoes significant molecular and functional adaptations throughout postnatal development. However, to date, our understanding of these dynamic changes in the human heart is limited. Moreover, advances in pediatric cardiac research can be hindered by a lack of preclinical models that accurately reflect human heart maturation. Guinea pigs may serve as a useful model for human cardiac research, as the guinea pig and human myocardium have similar ion channel expression and cardiovascular drug responsiveness. Despite these similarities, gene expression patterns during postnatal heart development have not been comprehensively investigated. In this study, we first characterized transcriptional changes in neonatal, juvenile, and adult guinea pig hearts and identified gene ontologies and pathways associated with cardiac maturation. Second, we compared the transcriptional profile of right atria and left ventricular tissue to highlight unique and shared chamber-specific patterns in guinea pigs over time. Finally, we conducted a cross-species comparison of the right atrial transcriptome between humans and guinea pigs to identify conserved maturation markers and gene expression patterns. Our findings provide a molecular framework for understanding age and chamber specific cardiac development, supporting the guinea pig as a promising preclinical model for studying human heart maturation. By identifying conserved gene programs and developmental markers across species, this study lays the groundwork for age-specific pharmacological strategies and computational models that can help to refine treatment decisions and outcomes for pediatric cardiology patients.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.06.666176v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Liver single-nucleus multiome profiling reveals cell-type mechanisms for cardiometabolic traits</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Alkhawaja, A. A., Currin, K. W., Perrin, H. J., Vadlamudi, S., Etheridge, A. S., Broadaway, K. A., Cannon, G. H., Anderson, C. W., Moxley, A. H., Iuga, A. C., Schuetz, E. G., Innocenti, F., Furey, T. S., Mohlke, K. L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We identified nearly 300,000 chromatin accessibility and gene expression regions in liver cell types, including 70,884 regions undetected in bulk tissue analyses, and mapped quantitative trait loci, which revealed previously obscured mechanisms.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The liver is a central regulator of cardiometabolic physiology, coordinating processes such as lipid and glucose metabolism, protein synthesis, and detoxification. Genome-wide association studies (GWAS) have identified hundreds of genetic variants associated with cardiometabolic traits, yet their molecular mechanisms in liver cell types remain unclear. Using multiome single-nucleus RNA and ATAC sequencing on liver samples from 39 individuals, we profiled gene expression and chromatin accessibility in 68,398 nuclei across six primary liver cell types. We identified 306,706 accessible chromatin regions, including 70,884 regions undetected in bulk tissue analyses and that predominantly represent less abundant cell types. To identify genetic effects on gene regulation in liver cell types, we mapped quantitative trait loci (QTLs) and detected 1,885 chromatin accessibility QTLs (caQTLs) and 67 expression QTLs (eQTLs). We integrated cell-type QTLs with GWAS signals and revealed cell types, genes, and chromatin regulatory elements involved in cardiometabolic traits, such as liver enzyme and cholesterol levels. Non-hepatocyte cell-type QTL analyses exposed previously obscured mechanisms, such as an eQTL for ADAMTS12 in liver sinusoidal endothelial cells potentially involved in liver fibrosis, demonstrating that single-nucleus approaches can capture regulatory events missed in bulk analyses. Furthermore, we predicted cell type of action for bulk liver caQTLs colocalized with GWAS signals, enhancing mechanistic insights for complex trait associations. Our findings provide a high-resolution map of the hepatic regulatory landscape and advance the understanding of cellular contexts and molecular mechanisms underlying cardiometabolic traits.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.06.668997v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Procrustean pseudo-landmark methods in Python to measure massive quantities of leaf shape data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hightower, A. T., Hall, S., Camacho, R. U., Papamichail, A., Adamski, E., Colligan, C., Deneen, A., Dunn, G., Haziza, J., Henley, C., Pawawongsak, A., Simms, L., Ward, S., Balant, M., Blackwood, C., Cannon, C., Case, A., Husbands, A., Josephs, E. H., Migicovsky, Z., Naegele, R., Patterson, E., Saavedra-Rojas, Y.-A., Chitwood, D. H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a simple and user-friendly leaf shape analysis in Jupyter Notebook and Python that uses pseudo-landmarks and Generalized Procrustes Analysis to measure and compare the shape of any leaf.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Premise: When examining leaf shapes that are different from one another, it can be difficult to compare both the overall leaf shape and points along the leaf margin in biologically and statistically meaningful ways. Method: To address this problem, we present a simple and user-friendly leaf shape analysis in Jupyter Notebook and Python that uses pseudo-landmarks and Generalized Procrustes Analysis to measure and compare the shape of any leaf. To demonstrate our analysis, we created a repository of real leaves gathered from eight experimental datasets. Results: Using our leaf repository, we explain how we can use pseudo-landmarks to compare all leaf shapes both within and between species using dimension reduction techniques like Principal Component Analysis and can predict leaf shapes using pseudo-landmarks through Linear Discriminant Analysis. Our leaf shape analysis also maps differences in shape as leaves grew around a rosette, showing the transition of shape across development (phyllotaxy). Finally, we showed how we can investigate the relationship between leaf shape variation and genetic diversity by combining shape with genetic data. Discussion: Through the use of Generalized Procrustes Analysis and pseudo-landmarks, our leaf shape analysis presents a powerful tool for examining the shape of any leaf across multiple biological, ecological, evolutionary, and developmental scales.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.08.669192v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Foliar mRNA spray induces protein synthesis in monocot crop and dicot model plant species</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Uslu, V. V., Dobrowitsch, M., Preussel-Danger, K., Furch, A. C. U., Noetzold, J., Richter, A. M., Schlemmer, T., Dong, Y., Krczal, G., Koch, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We apply unformulated, unmodified green fluorescent protein (GFP)-coding mRNA to monocot (Triticum aestivum, Hordeum vulgare, Zea mays) crops and dicot model plant Arabidopsis thaliana and observed GFP signals</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In planta gene expression via exogenous mRNAs has a wide range of potential biotechnological applications from genome editing to plant protection and to circumventing tissue culture. Yet, regarding mRNA delivery into intact plant cells, the purity, stability and translational efficiency of exogenous mRNAs appear as obstacles for effective exogenous mRNA application. Our study investigates the potential of exogenous mRNAs in mediating protein synthesis in plant cells. We applied unformulated, unmodified green fluorescent protein (GFP)-coding mRNA to monocot (Triticum aestivum, Hordeum vulgare, Zea mays) crops and dicot model plant Arabidopsis thaliana and observed GFP signals in all species, confirming the successful uptake and translation of mRNA into protein. This approach opens possibilities for broader, GM-free applications in plant science.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.07.668951v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A mechanistic framework for the recognition of chemically diverse brassinosteroids by BRI1-family receptor kinases</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Caregnato, A., Chen, H., Kvasnica, M., Hohmann, U., Oklestkova, J., Ferrer, K., Broger, L., Hothorn, L. A., Strnad, M., Hothorn, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-09
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present quantitative binding kinetics for four Arabidopsis thaliana BR receptors and fifteen BRs, which define the key chemical features required for high affinity receptor binding, ligand positioning, and co-receptor recognition.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Brassinosteroids (BRs) are chemically diverse plant steroid hormones produced via a branched biosynthetic pathway. The potent BR brassinolide is sensed by the membrane receptor kinase BRI1 and a SERK co-receptor, but the physiological functions of other abundant BRs remain to be characterized. Here, we present quantitative binding kinetics for four Arabidopsis thaliana BR receptors and fifteen BRs, which define the key chemical features required for high affinity receptor binding, ligand positioning, and co-receptor recognition. BRI1, BRL1, and BRL3 share overlapping ligand preferences, whereas BRL2 preferentially binds C28 BRs with moderate affinity. Structural analyses of BR-bound BRI1 and BRL3 ectodomains, combined with extensive in vitro and in vivo mutagenesis studies reveal a high structural plasticity of the hormone binding pocket. Functional assays using structure-based BR agonists and antagonists uncover that BR receptor - co-receptor signaling complexes can recognize chemically diverse BRs, introducing an additional, intriguing layer of BR signaling regulation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.08.669299v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

