<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Friday, 08 August 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrative analysis identifies FBXO5 as a critical mediator of CRPC progression and bone metastatic potential.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rajnikant Raut, Devesh Srivastava, Amit Kumar Chakraborty, Manish V Bais, Parul Mishra, Ashish Misra</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A new treatment for prostate cancer with E3 ubiquitin ligases .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Castration-resistant prostate cancer (CRPC) poses a major clinical challenge due to its aggressive nature, therapeutic resistance, and frequent progression to bone metastasis. Emerging evidence suggests that E3 ubiquitin ligases play a key role in cancer progression, yet their involvement in CRPC remains poorly understood.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40775408/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Urinary Complement proteome strongly linked to diabetic kidney disease progression.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zaipul I Md Dom, Salina Moon, Eiichiro Satake, Daigoro Hirohama, Nicholette D Palmer, Heather Lampert, Linda H Ficociello, Amin Abedini, Karen Fernandez, Xiujie Liang, Sara Pickett, Jonathan Levinsohn, Kristina O'Neil, Simon T Dillon, Michael Mauer, Andrzej T Galecki, Barry I Freedman, Katalin Susztak, Alessandro Doria, Andrzej S Krolewski, Monika A Niewczas</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that the top 5% urinary proteins representing multiple components of the Complement system are robustly associated with 10-year kidney failure risk, independent of clinical covariates, in diabetic kidney disease.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Diabetic kidney disease (DKD) progression is not well understood. Using high-throughput proteomics, biostatistical, pathway and machine learning tools, we examine the urinary Complement proteome in two prospective cohorts with type 1 or 2 diabetes and advanced DKD followed for 1,804 person-years. The top 5% urinary proteins representing multiple components of the Complement system (C2, C5a, CL-K1, C6, CFH and C7) are robustly associated with 10-year kidney failure risk, independent of clinical covariates. We confirm the top proteins in three early-to-moderate DKD cohorts (2,982 person-years). Associations are especially pronounced in advanced kidney disease stages, similar between the two diabetes types and far stronger for urinary than circulating proteins. We also observe increased Complement protein and single cell/spatial RNA expressions in diabetic kidney tissue. Here, our study shows Complement engagement in DKD progression and lays the groundwork for developing biomarker-guided treatments.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40775226/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Extrachromosomal DNA-Driven Oncogene Dosage Heterogeneity Promotes Rapid Adaptation to Therapy in MYCN-Amplified Cancers.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Giulia Montuori, Fengyu Tu, Di Qin, Rachel Schmargon, Elias Rodriguez-Fos, Konstantin Helmsauer, Hui Hui, Susmita Mandal, Karin Purshouse, Lara Fankhänel, Bartolomeo Bosco, Bastiaan Spanjaard, Hannah Seyboldt, Laura Grunewald, Matthias Jürgen Schmitt, Dennis Gürgen, Viktoria Buck, Mathias T Rosenfeldt, Frank P B Dubois, Simon Schallenberg, Annika Lehmann, Jessica Theißen, Sabine Taschner-Mandl, Arend Koch, Patrick Hundsdoerfer, Annette Künkele, Angelika Eggert, Matthias Fischer, Gaetano Gargiulo, Teresa G Krieger, Lukas Chavez, Fabian Coscia, Benjamin Werner, Weini Huang, Anton G Henssen, Jan R Dörr</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Extrachromosomal DNA (ecDNA) amplification enhances intercellular oncogene dosage variability and accelerates tumor evolution by violating foundational principles of genetic inheritance.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Extrachromosomal DNA (ecDNA) amplification enhances intercellular oncogene dosage variability and accelerates tumor evolution by violating foundational principles of genetic inheritance through its asymmetric mitotic segregation. Spotlighting high-risk neuroblastoma, we demonstrate how ecDNA amplification undermines the clinical efficacy of current therapies in cancers with extrachromosomal MYCN amplification. Integrating theoretical models of oncogene copy number-dependent fitness with single-cell ecDNA quantification and phenotype analyses, we reveal that ecDNA copy-number heterogeneity drives phenotypic diversity and determines treatment sensitivity through mechanisms unattainable by chromosomal oncogene amplification. We demonstrate that ecDNA copy number directly influences cell fate decisions in cancer cell lines, patient-derived xenografts, and primary neuroblastomas, illustrating how extrachromosomal oncogene dosage-driven phenotypic diversity offers a strong evolutionary advantage under therapeutic pressure. Furthermore, we identify senescent cells with reduced ecDNA copy numbers as a source of treatment resistance in neuroblastomas and outline a strategy for their targeted elimination to improve the treatment of MYCN-amplified cancers.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40773595/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Machine learning on multiple epigenetic features reveals H3K27Ac as a driver of gene expression prediction across patients with glioblastoma.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yusuke Suita, Hardy Bright, Yuan Pu, Merih Deniz Toruner, Jordan Idehen, Nikos Tapinos, Ritambhara Singh</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose that GSCs share a common epigenetic pattern of enhancer activation that defines their underlying transcriptomic expression pattern, and use this pattern to predict gene expression across patients.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Epigenetic mechanisms play a crucial role in driving transcript expression and shaping the phenotypic plasticity of glioblastoma stem cells (GSCs), contributing to tumor heterogeneity and therapeutic resistance. These mechanisms dynamically regulate the expression of key oncogenic and stemness-associated genes, enabling GSCs to adapt to environmental cues and evade targeted therapies. Importantly, epigenetic reprogramming allows GSCs to transition between cellular states, including therapy-resistant mesenchymal-like phenotypes, underscoring the need for epigenetic-targeting strategies to disrupt these adaptive processes. Understanding these epigenetic drivers of gene expression provides a foundation for novel therapeutic interventions aimed at eradicating GSCs and improving glioblastoma outcomes. Using machine learning (ML), we employ cross-patient prediction of transcript expression in GSCs by combining epigenetic features from various sources, including ATAC-seq, CTCF ChIP-seq, RNAPII ChIP-seq, H3K27Ac ChIP-seq, and RNA-seq. We investigate different ML and deep learning (DL) models for this task and ultimately build our final pipeline using XGBoost. The model trained on one patient generalizes to other 11 patients with high performance. Notably, H3K27Ac alone from a single patient is sufficient to predict gene expression in all 11 patients. Furthermore, the distribution of H3K27Ac peaks across the genomes of all patients is remarkably similar. These findings suggest that GSCs share a common distributional pattern of enhancer activity characterized by H3K27Ac, which can be utilized to predict gene expression in GSCs across patients. In summary, while GSCs are known for their transcriptomic and phenotypic heterogeneity, we propose that they share a common epigenetic pattern of enhancer activation that defines their underlying transcriptomic expression pattern. This pattern can predict gene expression across patient samples, providing valuable insights into the biology of GSCs.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40773517/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AdaptiveGS: an explainable genomic selection framework based on adaptive stacking ensemble machine learning.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhen Yang, Mei Song, Xianggeng Huang, Quanrui Rao, Shanghui Zhang, Zhongzheng Zhang, Chenyang Wang, Wenjia Li, Ran Qin, Chunhua Zhao, Yongzhen Wu, Han Sun, Guangchen Liu, Fa Cui</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed an adaptive and unified stacking genomic selection framework and designed a model interpretation strategy to identify the candidate significant SNPs of target traits.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We developed an adaptive and unified stacking genomic selection framework and designed a model interpretation strategy to identify the candidate significant SNPs of target traits. Genomic selection (GS) is an important technique in modern molecular breeding. As a powerful machine learning (ML) GS approach, stacking ensemble learning (SEL) combines multiple basic models (base learners, BLs) and effectively blends the strengths of different models to precisely depict the complex relationships between phenotypes and genotypes. However, in the key step of the SEL, there is currently a lack of an effective and unified framework for the selection of BLs. We developed adaptiveGS, an adaptive and explainable data-driven BLs selection strategy for the first time, to pre-screen the optimal BLs for stacking GS framework and improve the prediction accuracy. The adaptiveGS is performed based on the PR index, leveraging the Pearson correlation coefficient (PCC) and the normalized root mean square error (NRMSE), and the top 3 out of 7 (or self-setting) ML are tailored to be BLs via the PR index. We compared the adaptiveGS with 13 other GS algorithms based on a total of 21 traits (datasets) from 4 species. The results showed that adaptiveGS outperformed the 13 models on most of the 21 traits, with the average prediction accuracy (PCC) reaching 0.703, an average improvement of 14.4%, demonstrating superior predictive accuracy and robustness. Furthermore, the SHapley Additive exPlanations (SHAP) technique was utilized to interpret the adaptiveGS and identify significant SNPs for trait variations and potential interaction effects between SNPs. The adaptiveGS provides an operable and unified solution for stacking GS users to improve prediction accuracy in the breeding field. The adaptiveGS package is accessible at https://github.com/yangzhen0117/adaptiveGS .</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40772967/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Joint decomposition of Hi-C maps reveals salient features of genome architecture across tissues and development</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Reimonn, T., Yilmaz, V. O., Tran, H., Ng, G., Liu, D., Abdennur, N. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel approach to study long-range contact frequency profiles from Hi-C samples and show that it is possible to project interchromosomal contact frequencies onto a common set of components or basis vectors, enabling large-scale integrative analysis of genome compartmentalization.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The spatial organization of chromosomes in the nucleus is fundamental to cellular processes. Contact frequency maps from Hi-C and related chromosome conformation capture assays are increasingly available for a wide variety of biosamples and conditions, creating opportunities for comprehensive studies of genome compartmentalization and long-range interactions. However, the conventional dimensionality reduction approach to study long-range contact frequency profiles projects individual datasets into different and incomparable linear subspaces, making the resulting embeddings unsuitable for large-scale integrative analysis. To address this shortcoming and overcome the computational constraints involved in doing so, we introduce an analytic framework and Python toolkit that leverages incremental principal component analysis to project interchromosomal contact frequency profiles across arbitrarily many Hi-C datasets onto a common set of components or basis vectors. Our approach produces robust and directly comparable first and higher-order principal component (PC) scores that collectively capture biologically meaningful information beyond traditional A/B compartments. By applying our framework to a collection of 89 human Hi-C samples, we uncover distinct patterns of nuclear architecture reflecting cell state categories, associated with different heterochromatin state compositions. We also demonstrate that jointly-derived higher-order PCs improve the prediction of gene expression and regulatory element activity during differentiation. Together, our joint decomposition approach provides a powerful and scalable foundation for systematically investigating genome organization, providing critical insights into its role in development and disease.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.07.669116v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">The Hok bacterial toxin: diversity, toxicity, distribution and genomic localization</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Escalera-Maurer, A., Messineo, A., Renault, T. T., Nicollin, E., Castaneda-Sastre, E., Brunot, M., Berrehail, C., Le Rhun, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A comprehensive and curated catalog of annotated sequences, detailing genomic localization, toxic activity and distribution across diverse bacterial genomes, and the recent expansion of bacterial genome sequences.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The hok/Sok type I toxin-antitoxin system is encoded in chromosomes and plasmids of multiple Gram-negative bacteria. Current knowledge about hok/Sok regulation, toxicity and function originates mostly from the system present in the R1 plasmid, alongside a handful of homologs. The recent expansion of bacterial genome sequences calls for an updated analysis of hok/Sok diversity and distribution. Here, we used protein and DNA sequences as well as RNA structure to search for Hok homologs. Hok was detected in almost 80% of the available Enterobacteriaceae genomes and, more rarely, in three other bacterial families. Similarity clustering analysis of thousands of Hok homologs unveiled high Hok sequence variability and uncovered clusters of novel homologs. Experimental validation of representative sequences from the most abundant clusters revealed their toxicity, regardless of their localization. After scanning dedicated databases, we observed enrichment of Hok sequences in prophages and large conjugative plasmids. Finally, we identified horizontal gene transfer events across genera, predominantly mediated by plasmids. Our findings yield a comprehensive and curated catalog of annotated sequences, detailing genomic localization, toxic activity and distribution across diverse bacterial genomes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.07.668861v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ALLM-Ab: Active learning-driven antibody optimization using fine-tuned protein language models</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Furui, K., Ohue, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a novel active learning framework that leverages fine-tuned protein language models to accelerate antibody sequence optimization.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Antibody engineering requires a delicate balance between enhancing binding affinity and maintaining developability properties. In this study, we present ALLM-Ab (Active Learning with Language Models for Antibodies), a novel active learning framework that leverages fine-tuned protein language models to accelerate antibody sequence optimization. By employing parameter-efficient fine-tuning via low-rank adaptation, coupled with a learning-to-rank strategy, ALLM-Ab accurately assesses mutant fitness while efficiently generating candidate sequences through direct sampling from the model's probability distribution. Furthermore, by integrating a multi-objective optimization scheme incorporating antibody developability metrics, the framework ensures that optimized sequences retain therapeutic antibody-like properties alongside improved binding affinity. We validate ALLM-Ab in both offline experiments using deep mutational scanning (DMS) data from the BindingGYM dataset and online active learning trials targeting Flex ddG energy minimization across 15 antigens. Results demonstrate that ALLM-Ab not only expedites the discovery of high-affinity variants compared to baseline Gaussian process regression and genetic algorithm-based approaches, but also preserves critical antibody developability metrics. This work lays the foundation for more efficient and reliable antibody design strategies, with the potential to significantly reduce therapeutic development costs.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.05.668775v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">LICHEN: Light-chain Immunoglobulin sequence generation Conditioned on the Heavy chain and Experimental Needs</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Capel, H. L., Ellmen, I., Murray, C. J., Mignone, G., Black, M., Clarke, B., Breen, C., Tierney, S., Dougan, P., Buick, R. J., Greenshields-Watson, A., Deane, C. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present LICHEN, a heavy chain conditioned light sequence generation tool that enables collaborative light sequence design by leveraging computational capabilities alongside experimental expertise.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In developing therapeutic antibodies, the heavy chain is often prioritised due to its higher variability and its central role in antigen binding. An appropriate pairing of the light sequence is however important for antibody function. Here we present LICHEN, a heavy chain conditioned light sequence generation tool that enables collaborative light sequence design by leveraging computational capabilities alongside experimental expertise. LICHEN generates light sequences which are valid (antibody-like), diverse in sequence and structure, and conditioned on a specific heavy chain. LICHEN can also condition on germline and CDRs and automatically filter generated sequences for required properties. This allows LICHEN to be used across multiple antibody development use cases. We carry out experimental validation of the method conditioning only on the heavy sequence and on the heavy sequence and binding information. Our in vitro results show that sequences created by LICHEN have effective expression yields and can retain antigen-binding.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.06.668938v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Delivering trait-enhanced varieties to African smallholders through a pangenomic breeding network</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Maina, F., Faye, J. M., Kena, A. W., Diatta, C., Tankari, M. I., Ardaly, O. A., Diakite, O. S., Ibrahim, A. M., Moussa, O. A., Harou, A., Abdou, R., Sene, J. P., Bodian, S., Ndour, M., Sambakhe, D., Sine, B., Kadanga, P., Akata, E., Ouedraogo, N., Annor, B., Tetteh, I. T., Yahaya, M. A., Diallo, A., Kakeeto, R., Kisilu, R. K., Kondombo, C. P., Legesse, T., Mbulwe, L., Mutaliano, J., Mwenda, E. T., Naoura, G., Mafouasson, H. N. A., Opare-Obuobi, K., Abdulmalik, R. O., Runo, S., Abdelhalim, T. S., Yalaukani, L., Masanga, J., McLaughlin, C. M., Lasky, J. R., Harder, A. M., Healey, A. L., Lovell</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A global pangenomic breeding network for stress-resilience breeding of drought and pest resilient varieties for smallholder farmers in Niger .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Pangenomics has been promoted to accelerate breeding of orphan crops, but smallholder farmers in developing nations have seen little benefit so far. To address this gap, we built a global pangenomic breeding network, integrating African breeding programs, U.S. land grant universities, and international nonprofit research organizations. Here we demonstrate that pangenomics, when integrated with local crop improvement knowledge and global scientific partnerships, can facilitate breeding of drought and pest resilient varieties for smallholders. To breed trait-enhanced sorghum varieties with lgs1-1 resistance to witchweed (Striga hermonthica) for smallholders in Niger, one of the world's least developed nations, we used population genomics across local and global scales to develop lgs1-1 Striga resistance markers, and deployed them for rapid introgression of resistance into locally-preferred varieties. Genomic characterization, along with controlled experiments in laboratory, pot, field stations, and smallholder farms, confirmed lgs1-1 resistance was introgressed without loss of essential local-preference traits. New pangenomic resources, including global resequencing and graph pangenomes, further accelerated design of broadly-applicable markers. Unlocking the potential of pangenomics for stress-resilience breeding depended on stakeholder input, strong inference, South-led decision support software, and a dense collaborative network. The experience of the network provides a scalable roadmap for collaborative pangenomic breeding of trait-enhanced varieties for the world's lowest-resourced farmers.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.07.667917v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genomic footprints of domestication in almond (Prunus dulcis)</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lougmani, C., Mesnil, A., Fahmy, R., DECROOCQ, V., Bui, Q. T., Liu, S., Chen, X., Chague, A., Venon, A., Chagne, D., Eduardo, I., Alix, K., Cornille, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of genetic diversity in cultivated and wild almond cultivars reveals a complex, multi-regional domestication history for almonds, shaped by gene flow and recurrent selection.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The domestication of perennial crops in the Mediterranean Basin remains unclear, particularly regarding the genomic consequences of human-mediated demographic shifts and selection. We analyzed 8.1 million single nucleotide polymorphisms from 96 cultivated almond (Prunus dulcis) accessions from Europe, North America, Central Asia, and New Zealand, alongside four wild relatives. Population structure analyses revealed four geographically differentiated cultivated groups (Central Asian, North American, and two European) and three wild populations (P. spinosissima, P. orientalis, and P. fenzliana). Cultivated almonds retained high genetic diversity, consistent with weak domestication bottlenecks typical of outcrossing perennials. Elevated diversity and private allele counts in Central Asian cultivars, with limited gene flow, support this region as an independent cradle of domestication. In contrast, extensive wild-to-crop gene flow, especially involving P. orientalis, has shaped the genomes of European and North American almonds. Genome-wide scans for selective sweeps showed most candidate genes under selection were population-specific, though often associated with similar biological functions, including stress responses and agronomic traits. This suggests repeated targeting of comparable pathways during domestication, despite distinct selection histories. Several selected genes in cultivated populations overlapped with those in wild relatives, particularly P. orientalis. Combined with demographic inferences indicating wild populations persistence through past climate fluctuations, these findings suggest wild gene pools retain adaptive alleles, either ancestrally shared or introgressed, that contributed to cultivated diversity. Altogether, our results reveal a complex, multi-regional domestication history for almonds, shaped by gene flow and recurrent selection. This study emphasizes wild relatives as adaptive diversity sources and reveals genomic bases of perennial crop evolution.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.06.668913v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A fully-open structure-guided RNA foundation model for robust structural and functional inference</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhu, H., Li, R., Zhang, F., Tang, F., Ye, T., Li, X., Gu, Y., Xiong, P., Zhou, S. K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a structure-guided RNA foundation model that is pre-trained on millions of RNA sequence and secondary structures data by integrating base pairing interactions into masked language modeling through a novel pair matching operation, where the structure- guided mask and nucleotide-level mask are further balanced by a dynamic masking</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> RNA language models have achieved strong performances across diverse downstream tasks by leveraging large-scale sequence data. However, RNA function is fundamentally shaped by its hierarchical structure, making the integration of structural information into pre-training essential. Existing methods often depend on noisy structural annotations or introduce task-specific biases, limiting model generalizability. Here, we introduce structRFM, a structure-guided RNA foundation model that is pre-trained on millions of RNA sequence and secondary structures data by integrating base pairing interactions into masked language modeling through a novel pair matching operation, where the structure-guided mask and nucleotide-level mask are further balanced by a dynamic masking ratio. structRFM learns joint knowledge of sequential and structural data, producing versatile representations, including classification-level, sequence-level, and pairwise matrix features, that support a broad spectrum of downstream adaptations. structRFM ranks among the top models in zero-shot homology classification across fifteen biological language models, and sets new benchmarks for secondary structure prediction. structRFM further derives Zfold that enables robust and reliable tertiary structure prediction, with consistent improvements in estimating 3D structures and their accordingly extracted 2D structures, achieving a pronounced 19% performance gain compared with AlphaFold3 on RNA Puzzles dataset. In functional tasks such as internal ribosome entry site identification, structRFM achieves a whopping 49% performance gain in F1 score. These results demonstrate the effectiveness of structure-guided pre-training and highlight a promising direction for developing multi-modal RNA language models in computational biology. To support the broader scientific community, we have made the 21-million sequence-structure dataset and the pre-trained structRFM model fully open-source, facilitating the development of multimodal foundation models in biology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.06.668731v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Pangenome-guided sequence assembly via binary optimisation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cudby, J., Bonfield, J. K., Zhou, C., Durbin, R., Strelchuk, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a framework for pangenome-guided sequence assembly, which can resolve short-read data in complex regions without bias towards a single reference genome.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> De novo genome assembly is challenging in highly repetitive regions; however, reference-guided assemblers often suffer from bias. We propose a framework for pangenome-guided sequence assembly, which can resolve short-read data in complex regions without bias towards a single reference genome. Our method frames assembly as a graph traversal optimisation problem, which can be implemented on quantum computers. The pipeline first annotates pangenome graphs with estimated copy numbers for each node, then finds a path on the graph that best explains those copy numbers. On simulated data, our approach significantly reduces the number of contigs compared to de novo assemblers. While it introduces a small increase in inaccuracies, such as false joins, our optimisation-based methods are competitive with current exhaustive search techniques. They are also designed to scale more efficiently as the problem size grows and will run effectively on future quantum computers.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.06.668889v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Metabolic resistance of the tiger mosquito to pyrethroid insecticides in La Reunion Island likely results from local adaptation</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bacot, T., Bonneville, J.-M., Lacroix, V., Oberbach, P., Gaude, T., Nadalin, L., Laporte, F., Habchi-Hanriot, N., Dupuy, G., Czeher, C., Fontaine, M. C., Boyer, F., David, J.-P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study of pyrethroid resistance in the tiger mosquito in the South-West Indian Ocean .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The resistance of mosquitoes to insecticides is a valuable model system for studying the genetic bases of xenobiotic adaptation in insects. The spread of the Asian tiger mosquito Aedes albopictus combined to the massive use of pyrethroid insecticides to limit arbovirus transmission resulted in the rise of resistance in various continents. Here, we investigated the genetic mechanisms underlying the recent adaptation of this mosquito to deltamethrin in La Reunion island. Bioassays confirmed the presence of resistance alleles in field populations. The resistance phenotype was further enhanced in the laboratory following a few generations of controlled selection. Combining whole genome Pool-seq and RNA-seq revealed no evidence of target-site resistance mutations but the over-expression and variant selection of detoxification enzymes associated with pyrethroid metabolism including cytochrome P450s, transferases and ABC-transporters. Among over-expressed detoxification genes, only one was linked to a gene duplication while polymorphism data suggest most of them being trans-regulated. Genome-wide selection signatures revealed a 9 Mb inverted superlocus responding to insecticide selection whose phenotypical importance remains uncertain. Altogether, this study indicates that the multigenic metabolic resistance phenotype observed in this insular territory mainly results from local adaptation. From an applied perspective, this study provides a set of markers to track pyrethroid resistance in the tiger mosquito in the South-West Indian Ocean. As this region is subjected to recurrent arbovirus outbreaks, the additive resistance phenotype that may arise from the introduction of Kdr mutations from other territories also calls for improving resistance surveillance at the regional scale.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.06.668844v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comparative evaluation of genomic footprinting algorithms for predicting transcription factor binding sites in single-cell data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Everitt, A., Whalen, S., Pollard, K. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We provide practical guidelines for robust genomic footprinting in single-cell datasets and a roadmap for extracting deeper insights about how gene regulatory networks vary across cell types.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Transcription factors (TFs) have millions of potential binding sites across the human genome, but only a fraction are bound in a given context. Genomic footprinting aims to identify context-specific binding sites by detecting patterns in open chromatin data. While powerful, these approaches face technical challenges, especially in single-cell applications. We developed a benchmarking framework for cell-type specific footprinting and used it to evaluate the consistency, reproducibility, and equivalency of three leading methods across data quality scenarios and as a function of cell-type similarity. Peak-level read coverage emerged as the strongest predictor of stable footprints. Motivated by limited reproducibility across tools, we built an ensemble model that improved concordance with ChIP-seq. To encourage broader adoption and continued tool development, we provide practical guidelines for robust genomic footprinting in single-cell datasets and a roadmap for extracting deeper insights about how gene regulatory networks vary across cell types in complex tissues.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.07.669008v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">High Cross Pollination Frequency in Rice Landraces in Field Condition</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Deb, D., Bhattacharya, D., Nauri, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present the first contrary evidence of cross pollination frequency (CPF) in rice, and establish that CPF may reach 100% when the flower exposure overlap overlap exceeds 40 min.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Based on a plethora of experimental observations, cross pollination frequency (CPF) in rice (Oryza sativa L.) is described in all rice biology text books and official publications to be very low, rarely exceeding 2%. Our previous study reported in 2021 the first contrary evidence of CPF >80%. Here we present the results of our recent experiments involving 25 pairs of rice landraces, and establish that CPF tends to increase with the flower exposure duration (FED) overlap between the pollen parent and ovary parent, and that CPF may reach 100% when the flower exposure overlap exceeds 40 min. This study corroborates our previous finding that the success of cross pollination in rice crucially depends on FED, to which no attention was given in previous rice outcrossing experiments over a century.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.06.668876v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Cytoplasmic Abundant Heat-Soluble Proteins from Tardigrades Protect Synthetic Cells Under Stress</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xi, Y., Mao, J., Chen, S. J., Moghimianavval, H., Lee, Y. J., Huang, A. J., Zhou, D., Xu, A., Fu, K., Adera, S., Ferguson, A. L., Liu, A. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A protein-based strategy to impart synthetic cells with resilience to desiccation and other environmental stresses, expanding their utility in bioengineering, cold-chain-independent biomanufacturing, and adaptive biointerfaces.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cytoplasmic abundant heat-soluble (CAHS) proteins, a stress-responsive intrinsically disordered protein from tardigrades, have been discovered to form gel-like networks providing structural support during dehydration, thus enabling anhydrobiosis. However, the mechanism by which CAHS proteins protect the dehydrating cellular membrane remains enigmatic. Using giant unilamellar vesicles (GUVs) as a model membrane system, we show that encapsulated CAHS12 undergoes a reversible structural transformation that reinforces membrane integrity and preserves encapsulated components, mimicking natural anhydrobiosis. CAHS12-containing GUVs demonstrated stability for weeks and mechanical robustness under dehydration, elevated temperature, and osmotic stresses. Molecular simulations suggest that CAHS12 forms a filamentous network within the vesicle lumen that mitigates membrane collapse and preserves compartmental architecture. Notably, removal of the N-terminal domain of CAHS12 abrogates this protective function. Synthetic cells with cell-free transcription-translation capabilities survive desiccation and recover biochemical activities, akin to the tun state of the tardigrade. This discovery opens up a protein-based strategy to impart synthetic cells with resilience to desiccation and other environmental stresses, expanding their utility in bioengineering, cold-chain-independent biomanufacturing, and adaptive biointerfaces.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.07.669088v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Enabling reproducible and reusable genetic demultiplexing benchmarking with Nextflow and Apptainer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lynch, M. P., Gatto, L., Culhane, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Demux_bench for genetic demultiplexing methods in single-cell RNA sequencing, which meets the gold standard for reproducibility, reusability, scalability, and portability.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Reproducibility, reusability, and portability were identified as key requirements for analysis pipelines in genomics. A previous study identified that less than 4% of notebooks used for biomedical research were fully reproducible. Reusable and portable software are required to ensure compatibility with federated and cloud computing. The field requires computational workflows that meet high standards for reproducibility, reusability, and portability to allow benchmarking to be repeated and updated periodically. This is of particular importance in areas where new methods are frequently released, such as demultiplexing scRNA-seq, a critical early step in most scRNA-seq analysis pipelines. To address this, we developed demux_bench for benchmarking genetic demultiplexing methods in single-cell RNA sequencing, which meets the gold standard for reproducibility of computational workflows, and incorporates best practices for reusability and portability. We used workflow manager Nextflow to enable the simulation and testing of various benchmarking scenarios and methods in parallel from a single pipeline execution. Different experimental configurations can be simulated, including % doublets and class size imbalance. The pipeline includes genotype-free methods, Vireo and souporcell, and additional existing or novel methods can be added modularly. Demux_bench is configurable to reproduce specific analyses and generalisable to address new research questions. Software dependencies are handled through containers via Apptainer, allowing portability to different compute environments and avoiding the need for manual installation of software. Demux_bench is available on WorkflowHub (https://workflowhub.eu/workflows/1769), and can also be run on Galaxy and other platforms through RO-Crates. Demux_bench facilitates gold standard benchmarking for genetic demultiplexing through reproducibility, reusability, scalability, and portability.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.06.668897v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Scale Reliant Mixed Effects Models Enhance Microbiome Data Analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> McGovern, K. C., Silverman, J. D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce scale-reliant mixed-effects models (SR-MEM), which extend our earlier scale-RELiant inference framework by explicitly modeling uncertainty in the unmeasured scale via user-defined probability distributions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Linear models, including those used for differential abundance analyses, are frequently used in microbiome research to assess how experimental conditions (e.g., disease state or age) affect microbial abundance. Linear mixed-effects models (MEMs) extend linear models to accommodate complex designs, such as longitudinal sampling or hierarchical study structures. However, when applied to microbiome data, existing MEM approaches suffer from high false positive and false negative rates because sequence counts are compositional -- they reflect relative rather than absolute abundances. Current methods attempt to overcome this limitation through normalization, but these approaches imply on strong, often unrealistic assumptions about the unmeasured biological scale (e.g., total microbial load). Here we introduce scale-reliant mixed-effects models (SR-MEM), which extend our earlier scale-reliant inference framework by explicitly modeling uncertainty in the unmeasured scale via user-defined probability distributions. By treating scale as a latent variable rather than fixing it through normalization, SR-MEM yields robust inference for complex experimental designs. Across simulations and real datasets, SR-MEM is the only method that consistently controls the false discovery rate while achieving higher power than both normalization and bias-correction methods. SR-MEM can also incorporate external scale measurements (e.g., flow cytometry, qPCR) or leverage independent studies to further improve inference. An accessible implementation is provided in the ALDEx3 R package, enabling more rigorous and reproducible analysis of microbial communities.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.05.668734v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Benchmarking antigen-aware inverse folding methods for antibody design.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Janusz, B., Chomicz, D., Wrobel, S., Dudzic, P., Polasa, A., Martin, K., Darnell, S., Comeau, S. R., Krawczyk, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Inverse folding in antibody discovery is a language model that can be used to predict a sequence that would fit a given structure.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Computational antibody design has seen many recent advances pioneered via the use of language models and advanced structure prediction tools. Developing a de novo antibody against a specific antigen requires structural awareness that most language models lack. A prominent class of machine learning methods combining the best of language model and structural worlds is inverse folding. This approach aims to predict a sequence that would fit a given structure. Such methods are now increasingly used to predict alternate sequences given a structure of a binder. It is known that, just like language models, such methods have certain predictive power in identifying binders. Here we performed a set of tests to reveal where, if at all, such methods provide value in the realistic setting of antibody discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.05.668698v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Decoding the constraints acting on a coastal fish using landscape transcriptomics</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gairin, E., Miura, S., Chamot, Z., Sautereau, C. A., Zwahlen, J., Takamiyagi, H., Gibert, Y., Herrera, M., Laudet, V.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that fish from human-impacted sites exhibit elevated expression of genes involved in inflammation and immune responses, suggesting that urban reefs act as fast food environments, offering energy-rich diets while imposing chronic stress.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding how wild organisms respond to complex mixtures of environmental stressors remains a major challenge in ecology. To move beyond laboratory-based studies of the effect of individual stressors on model species, landscape transcriptomics - the study of how genome-wide expression patterns reveal environmental conditions - has been suggested. Here, we apply landscape transcriptomics to the damselfish Chrysiptera cyanea, a sentinel species from the coastal ecosystems of Okinawa, by sampling juveniles and adults across 18 sites from natural to urbanized habitats. We uncover a striking molecular signature of urbanization: fish from human-impacted sites exhibit elevated expression of genes involved in inflammation and immune responses. Remarkably, their transcriptomes resemble those of overfed individuals from laboratory feeding experiments - suggesting that urban reefs act as fast food environments, offering energy-rich diets while imposing chronic stress. This pattern is illustrated by the altered expression of key genes in metabolic and stress pathways (e.g., downregulation of ppargc1a and insrb, indicative of dietary intake surplus and insulin resistance; upregulation of interleukins, chemokines, toll-like receptors and caspase linked to the immune response and inflammation). Our results demonstrate that wild transcriptomes capture the invisible biological costs of environmental degradation, positioning fish as living biosensors for coastal ecosystem health.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.05.668635v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">TPClust: Temporal Profile-Guided Subtyping Using High-Dimensional Omics Data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hu, B., De Jager, P. L., Bennett, D. A., Vardarajan, B. N., Wang, Y., Lee, A. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-07
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a semi-parametric clustering method that integrates high-dimensional omics data with longitudinal phenotypes, including outcomes and covariates, for outcome-guided subtyping.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Clustering is widely used to identify subtypes in heterogeneous populations, yet most approaches rarely integrate longitudinal phenotypic trajectories with high-dimensional molecular profiles, limiting their ability to resolve biologically and clinically meaningful heterogeneity in progressive diseases. We developed TPClust, a supervised, semi-parametric clustering method that integrates high-dimensional omics data with longitudinal phenotypes, including outcomes and covariates, for outcome-guided subtyping. TPClust jointly models latent subtype membership and longitudinal outcome trajectories using multinomial logistic regression informed by molecular features selected via structured regularization, along with spline-based regression to capture subtype-specific, time-varying covariate effects. Simulations demonstrate valid inference for time-varying effects and robust feature selection. Applied to transcriptomic profiles and longitudinal cognitive data from 1,020 older adults in the Religious Orders Study and the Rush Memory and Aging Project, TPClust identified four aging subtypes, including intermediate subtypes not captured by unimodal approaches, with distinct cognitive trajectories, time-varying risk profiles, clinical and neuropathological features, and multimodal molecular signatures.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.05.668514v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

