<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Friday, 18 April 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Exploring the Genetic Link between Irritable Bowel Syndrome and Polycystic Ovary Syndrome: Bidirectional Mendelian Randomization and Machine Learning Approaches.</h2>
                <p style="font-size: 16px;"><em>Source: PubMed</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Research has shown a certain correlation between polycystic ovary syndrome (PCOS) and irritable bowel syndrome (IBS). The study aims to determine the directionality and underlying biological processes influencing the relationship between these two disorders.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40247811/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Methylation reprogramming associated with aggressive prostate cancer and ancestral disparities</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">African men are disproportionately impacted by aggressive prostate cancer (PCa). Key to this disparity both genetic and environmental factors, alluding to epigenetic modifications. However, African-inclusive prostate tumour DNA methylation studies are lacking. Assembling a multi-geo-ancestral prostate tissue cohort, including men with (57 African, 48 European, 23 Asian) or without (65 African) PCa, we interrogate for genome-wide differential methylation. Overall, methylation appears to be driven by ancestry over geography (152 southern Africa, 41 Australia). African tumours show substantial heterogeneity, with universal hypermethylation indicating epigenetic silencing, encompassing PCa suppressor genes and enhancer-targeted binding motifs. Conversely, African tumour-associated heterochromatic hypomethylation suggests permissive chromatin remodelling, with developmental pathway activation via enhancer targets. Taken together, we show methylation aberrations favour metastatic growth, genomic instability and disease aggressiveness in African tumours, which we hypothesise is driven by extensive plasticity of intergenic regulatory regions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.16.649098v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Domestication shaped the chromatin landscape of grain amaranth</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Plant domestication has had profound impacts on the morphology and genetic diversity of crops. Beyond sequence diversity, changes in chromatin structure can play an important role in plant adaptation. However, the interplay between the chromatin landscape and plant domestication remains unclear. We present a high-quality genome assembly and chromatin landscape map of the ancient pseudo-cereal, amaranth. Using ATAC-sequencing of multiple accessions of three grain amaranth species and two wild relatives, we show that the overall amount of accessible chromatin is highly conserved, about 2.5% of all chromatin switched states, with a higher fraction of the genome repeatedly opening during domestication processes. These differentially accessible chromatin regions, between the crops and their wild ancestor, were species-specific and significantly associated with selective sweeps - reflecting the repeated independent domestication of amaranth. Our findings reveal the dynamic interplay between domestication and the chromatin landscape, highlighting an additional layer of diversity in crops.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.17.649361v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SlHSFB3a developmentally regulates lateral root formation by modulating auxin signaling in tomato.</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Summary The function of HSFs, known otherwise as Master thermoregulators,in plant developmental remains largely uninvestigated. In this study, we strategically analyze SlHSFB3a, a class B sub member of HSF transcription factor family, uniquely expresses in age-dependent tomato roots and improves root architecture by synchronizing auxin homeostasis. This data demonstrates SlHSFB3a overexpressed transgenics display higher lateral root (LR) density and early LR emergence improving root architecture. Generation of CRISPR-Knockout mutants displayed contrasting phenotype, confirming SlHSFB3a vital role in root growth. In SlHSFB3a manipulated roots, concentration gradient auxin responses oscillated with increase in LR number.We highlight the signal transduction of SlHSFB3a mediated auxin activation that enhances tomato LRs. SlHSFB3a directly inhibits auxin repressors, increases auxin flow via ARF7/LOB20 pathway and positively modulates LR growth.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.15.648910v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Balancing nutrient remobilization and photosynthesis: the dual role of lupin cotyledons after germination</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Efficient nutrient mobilization from seed storage tissues is essential for seedling establishment, particularly in legumes such as Lupinus albus (white lupin), which thrive in nutrient-poor soils. This study investigates the role of cotyledons in nitrogen (N) and mineral remobilization after germination during their transition from storage organs to photosynthetically active tissues including the metabolic challenges posed by coexistence of these two functions in epigeal germination. We cultivated white lupin seedlings under nitrogen-deficient conditions, analyzing cotyledon composition and function over 28 days. Our results indicate that 60 % of cotyledon-stored proteins are degraded within the first eight days, with free amino acids transiently accumulating before being redistributed to support growth. The progressive depletion of cotyledon reserves was accompanied by structural and metabolic changes, including an increase in photosynthetic proteins. However, cotyledon photosynthetic capacity remained lower than that of true leaves, suggesting a transient role in energy metabolism. The loss of cotyledons before day 12 significantly impaired seedling development, emphasizing their critical contribution to nitrogen, phosphate, and micronutrient supply during early growth. Comparative proteomic analysis revealed dynamic shifts in nutrient transport, amino acid metabolism, and stress response pathways following cotyledon removal. These findings underscore the significance of cotyledon nutrient remobilization in legume adaptation to low-fertility soils and highlight potential targets for breeding strategies aimed at improving nutrient use efficiency. By optimizing cotyledon nutrient composition and function, future breeding efforts could enhance seedling vigor, reduce fertilizer dependency, and improve the nutritional value of lupin-based foods.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.15.648908v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Relationship Between Open Reading Frame 320, a Gene Causing Male Sterility in Tomatoes, and Cytoplasmic Male Sterility in Potatoes</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Cytoplasmic male sterility (CMS) is a phenotype wherein plants cannot develop normal male organs because of mitochondrial genes. Several cytoplasmic sterility factors are suggested to be involved in CMS phenotypes in potatoes (Solanum tuberosum); however, reports on the relevant mitochondrial genes remain scarce. Many potato cultivars carry mitochondrial genes that cause CMS, resulting in pollen sterility, thereby limiting their use as male parents in breeding. Therefore, identifying the causal genes is crucial for potato breeding. In this study, we focused on the T/{beta}-type cytoplasm, the most prevalent genome type of potato worldwide, to explore mitochondrial genes involved in CMS in potatoes. We identified a novel gene, open reading frame 320 (orf320), in potato genotypes with T/{beta}-type cytoplasm by comparing the mitochondrial genomes of potatoes. Functional analysis of tomatoes showed that overexpression of orf320 with a mitochondrial transit peptide induced male sterility phenotypes, including abnormal anther development and pollen abortion. Furthermore, an investigation of orf320 in 126 potato cultivars revealed that this gene is specific to the T/{beta}-type cytoplasm and is absent in investigated cultivars with other cytoplasmic types. These findings provide strong evidence that orf320 is a candidate CMS-causing gene in male sterility of T/{beta}-type cytoplasm, offering valuable insights for future potato breeding.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.17.649265v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">INDETERMINATE DOMAIN-DELLA protein interactions orchestrate gibberellin-mediated cell elongation in wheat and barley</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">DELLA proteins, members of the GRAS-domain family of transcriptional regulators, are critical for plant growth and development. They modulate transcription indirectly via interactions with hundreds of transcription factors. The phytohormone gibberellin (GA) triggers DELLA degradation, providing a mechanism by which plants can integrate developmental and environmental signals to regulate gene expression and optimize growth responses. In agriculture, DELLA mutations have been instrumental in improving crop performance. Most modern wheat (Triticum aestivum L.) varieties carry Rht-B1b or Rht-D1b alleles that encode DELLA proteins resistant to GA-mediated degradation, resulting in constitutive partial suppression of stem growth, a semi-dwarf stature and lodging resistance. However, these alleles also reduce nitrogen use efficiency and early vigour, limiting their utility in some environments. Understanding how DELLA proteins regulate growth and development is, therefore, critical for refining breeding strategies. In this study, we identified the orthologous C2H2 zinc-finger transcription factors INDETERMINATE DOMAIN 5 (IDD5) in wheat and SEMI-DWARF 3 (SDW3) in barley (Hordeum vulgare) as positive regulators of stem and leaf expansion. Both IDD5 and SDW3 physically interact with, and act downstream of, DELLA proteins as key components of GA-mediated growth responses. Altered expression levels of GA biosynthesis genes suggest that IDD5 helps maintain GA homeostasis in addition to growth regulation. Loss-of-function mutations in IDD5 and SDW3 confer a GA-insensitive semi-dwarf phenotype comparable to that of the Rht-D1b Green Revolution allele, highlighting their potential as novel dwarfing alleles for cereal improvement.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.17.649374v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Mobile immune signals potentiate salicylic acid-mediated plant immunity via WRKY38/62 transcription factors</h2>
                <p style="font-size: 16px;"><em>Source: bioRxiv</em></p>
                <h4 style="margin-bottom: 5px;">Background</h4><p style="font-size: 16px; text-align: justify;">Systemic acquired resistance (SAR) is a broad-spectrum plant immune response that provides protection against various pathogens. Activation of SAR requires mobile immune signals as well as the indispensable immune hormone salicylic acid (SA). Nonetheless, it remains unknown how mobile signals integrate with the SA signal to produce functional SAR responses. Here, we demonstrate that the mobile signals, azelaic acid (AzA) and N-hydroxy-pipecolic acid (NHP), respectively dampened and potentiated SA-induced transcriptional reprogramming of thousands of genes. Indeed, NHP enhanced stability of the SA receptor protein NPR1, and unlike AzA, it dramatically increased the effectiveness of SA-induced immunity against bacterial infection by 10-fold. Analysis of NHP-primed, SA-responsive gene promoters indicated that WRKY transcription factors play an important role in integrating these two immune signals. While responsiveness to SA remained largely unaffected by mutation of WRKY38 and WRKY62, it abolished NHP-mediated potentiation of SA-induced gene expression and immunity. Collectively, our findings reveal mobile signals potentiate SA-mediated plant immunity via WRKY38/62 transcription factors.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.04.17.649115v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

