<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Friday, 19 September 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">LieOTMap: A Differentiable Approach to Cryo-EM Fitting via Lie-Theoretic Optimal Transport</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hu, Y., Cao, Z., Liu, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-19
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present LieOTMap, a novel, fully differentiable framework for cryo-EM fitting, and demonstrate the effectiveness of combining Lie-theoretic representations with differentiable geometric loss functions for complex structural alignment tasks.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The integration of high-resolution atomic models with lower-resolution cryo-electron microscopy (cryo-EM) maps is a fundamental task in structural biology. However, this rigid-body fitting problem is challenged by complex scoring landscapes and the non-differentiable nature of standard structural similarity metrics like TM-score, precluding their direct use in modern gradient-based optimization pipelines. We present LieOTMap, a novel, fully differentiable framework for cryo-EM fitting. Our approach introduces two key innovations. First, we parameterize rigid-body transformations on the Lie algebra se(3), which provides a minimal, singularity-free representation of the motion. Second, we formulate a loss function based on the Sinkhorn divergence, a differentiable proxy for the Optimal Transport (Wasserstein) distance. This loss function serves as a robust, geometrically meaningful surrogate for non-differentiable scores, allowing us to leverage powerful gradient descent optimizers to navigate the search space. We demonstrate the effectiveness of LieOTMap by fitting the apo-state structure of E. coli GroEL (PDB: 1aon) into the ATP-bound state cryo-EM map (EMD-1046). Our method successfully navigates a large-scale conformational change, achieving a highly accurate final RMSD of 3.08 Angstrom with respect to the ground-truth structure (PDB: 1GRU). This result showcases the power of combining Lie-theoretic representations with differentiable geometric loss functions for complex structural alignment tasks.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.16.676533v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Genome-wide association study bridging genomics-phenomics gap in natural plant populations.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sarbani Roy, Hari Shankar Gadri, Vikas Sharma, Md Asif Chowdhary, Rohini Dwivedi, Pankaj Bhardwaj</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A critical literature synthesis and critical literature review of genome-wide association studies (GWAS) in natural populations .</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The planet hosts half a million plant species exhibiting a spectacular diversity of plant forms with genomes driving phenotypic variations. The genome information exists for less than 1% of species, limiting quantitative genomic studies in natural populations. This review explores how recent advances in cutting-edge genomic and phenomic techniques extended genome-wide association studies (GWAS) to wild, non-model species and other natural populations. We also discuss the incorporation of diverse bioinformatic tools into comprehensive in-silico pipelines and recommend implementing machine learning algorithms to address methodological challenges. The critical literature synthesis highlights several scopes of GWAS, bringing natural populations into the spotlight of genomic research. Thus, the study presents GWAS as a cornerstone for advancing quantitative genomics in natural populations. This shift holds great promise for understanding adaptation, trait evolution, and conservation genetics across diverse plant germplasm.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40965825/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comparative Landscape of Small RNAs in Tissue and Liquid Biopsies for Liver Transplant Outcomes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Anam, M., Watkins, C., Rucker, G., Marlow, K., Khalil, M., Dogan, M., Eason, J., Vanatta, J., Eymard, C., Kuscu, C., Su, Z., Kuscu, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that tRNA fragments (tRFs) are IRI-responsive biomolecules abundant in both donor liver tissue and perfusate, a non-invasive biofluid, and show their potential for early detection of EAD.</p><p style="font-size: 16px; text-align: justify;">Background: Ischemia-reperfusion injury (IRI) is an inevitable consequence of liver transplantation, arising during donor organ procurement and reoxygenation. Severe IRI is a leading contributor to early allograft dysfunction (EAD), a post-transplant complication associated with reduced graft survival. Current postoperative biomarkers provide limited time for intervention, highlighting a need to identify preoperative biomarkers of IRI. Meanwhile, tRNA fragments (tRFs) have emerged as novel biomarkers in various diseases, but remain unexplored in the context of liver transplant. Results: We performed small RNA sequencing on donor liver biopsies to investigate IRI-associated transcript changes. In parallel, donor liver perfusates were analyzed as a non-invasive surrogate for tissue profiling. Across samples, microRNAs (miRNAs) and tRFs were the most abundant small RNAs. Perfusate expression strongly correlated with paired biopsies, supporting its value as a non-invasive source. Comparison of post-reperfusion versus pre-implantation biopsies revealed that IRI reprogrammed tRF expression. Stratification by clinical outcome showed that patients who developed EAD exhibited specific tRNA fragments signature in both biopsy and perfusate. Receiver operating characteristic (ROC) analysis demonstrated that a tRF-based model achieved an AUC of 0.7748, outperforming the donor risk index alone (AUC = 0.6804), representing a 9.4% increase in discriminative capacity. Conclusions: These results are the first to establish tRFs as IRI-responsive biomolecules abundant in both donor liver tissue and perfusate, a non-invasive biofluid. In particular, various tRF species emerged as promising candidate biomarkers for early detection of EAD. These results lay the foundation to further investigate the diagnostic and prognostic utility of tRF species in clinical liver transplantation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.16.676609v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An increased number of heterozygous calls in the AxiomTM Equine Genotyping Array</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Neuditschko, M., Gmel, A. I., Pirani, A., McInnis, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We analysed the AxiomTM Equine 670K SNP array for genome-wide analysis of homozygosity patterns and found that re-genotyped horses with few ROH segments showed the lowest concordance compared to purebred horses.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single nucleotide polymorphism (SNP) arrays are commonly used in livestock genetics to investigate complex traits including genome-wide analysis and fine mapping, genomic prediction, genetic diversity and selection signature analyses. In the context of a European Equine diversity study, we analysed the AxiomTM Equine 670K SNP genotype data from 2,768 equids representing 20 horse breeds and one donkey breed. While assessing genome-wide runs of homozygosity (ROH) patterns, we observed an increased number of heterozygous calls in 167 purebred horses, which exhibited fewer ROH segments compared to F1 crosses, with 24 of them completely lacking any ROH segments. To further investigate this, we conducted a 4-fold genotype concordance analysis of replicate pairs on the same AxiomTM batch, between two different AxiomTM batches, between Illumina EquineSNP50 BeadChip and between Illumina paired-end HiSeq 2000 whole genome sequencing data. Additionally, we used SNPolisherTM classification on data from the AxiomTM Equine 670K SNP array to evaluate the genotype performance of the 670,806 genome-wide SNPs. When comparing the overlapping SNPs between the different genotyping platforms, replicated pairs within the same AxiomTM batch showed the highest average genotype concordance (98.81%), followed by Illumina 50K (97.88%) and whole genome sequencing (96.84%). In contrast, re-genotyped horses with few ROH segments (i.e., replicates on different batches) showed the lowest concordance (93.52%). A lower pass rate was observed in one batch, suggesting a processing performance issue that contributed to the reduced concordance between batches. According to SNPolisherTM classification a total of 120,838 genome-wide SNPs were not recommended for reproducibility. After calling genotypes of the two different batches together in accordance with AxiomTM Best Practice (e.g. removing failing samples before the final genotyping) and excluding non-recommended SNPs, genotype concordance improved in all comparisons: same AxiomTM batch (99.84%), Illumina 50K (98.33%), whole genome sequencing (97.81%), and different AxiomTM batches (98.59%). Based on these findings, we recommend excluding horses exhibiting an unusually high number of heterozygous calls, using only SNPs with validated genotype performance, and accounting for batch effects when analyzing AxiomTM Equine 670K SNP genotype data from different batches.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.16.676615v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">jsPCA: fast, scalable, and interpretable identification of spatial domains and variable genes across multi-slice and multi-sample spatial transcriptomics data</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Assali, I., Escande, P., Villoutreix, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce joint spatial PCA (jsPCA), a novel, fast, scalable and interpretable method for the automatic identification of spatial domains and variable genes in multi-slice and multi-sample spatial transcriptomics data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Spatially structured cell heterogeneity within tissues is essential for healthy organ function. This heterogeneity is reflected by differential gene expression activity at various spatial location. Spatial transcriptomics technologies record genome-wide measurements of gene expression at the scale of entire tissues with high spatial resolution. While they have revolutionized our quantitative understanding of tissue architecture, these technologies generate large and high dimensional datasets encompassing tens of thousands of genes recorded at tens of thousands of spatial locations, requiring efficient automated methods for their analysis. In this study we introduce joint spatial PCA (jsPCA), a novel, fast, scalable and interpretable method for the automatic identification of spatial domains and variable genes in multi-slice and multi-sample spatial transcriptomics data. jsPCA relies on a simple mathematical formulation of a spatial covariance defined as the product of the gene expression covariance with the spatial autocorrelation. The principal components of this spatial covariance yield a biologically meaningful low-dimensional representation. From this representation, we can derive spatial domains by simple clustering. In addition, spatially variable genes can be identified directly from the principal components coefficients. Moreover, this approach enables the joint representation of multiple slices and samples, a frequent experimental setting. This joint representation is obtained without spatial alignment by computing common principal components via joint diagonalization of the set of spatial covariance matrices obtained for each slice. By leveraging sparsity and non-convex optimization on manifold, jsPCA leads to computing time in the order of seconds to minutes, substantially outperforming state-of-the-art approaches. We benchmarked jsPCA on the Visium 10x dataset of human dorsolateral prefrontal cortex and the Stereo-seq MOSTA dataset of mouse embryonic development against 10 state-of-the-art methods. Our approach demonstrated excellent performances, comparable or better than state-of-the-art methods, such as SpatialPCA, BASS, GraphPCA or Stagate, while being much faster, interpretable, and scalable to very large datasets.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.16.676466v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CARD k-mers: Unmasking the pathogen hosts and genomic contexts of antimicrobial resistance genes in metagenomic sequences</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wlodarski, M. A., Lau, T. T. Y., Alcock, B. P., Raphenya, A. R., Ta, T. E., Maguire, F., Beiko, R. G., McArthur, A. G.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce CARD k-mers, the first tool built to jointly predict species-level taxonomy and genomic context (plasmid vs. chromosome) for antimicrobial resistance genes in short metagenomic reads.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Antimicrobial resistance (AMR) is a global health crisis requiring rapid surveillance across human, agricultural, and environmental systems. A major challenge during outbreaks is not only detecting antimicrobial resistance genes (ARGs), but also unmasking their pathogen hosts and genomic context, as ARGs alone do not fully capture AMR risk. Pathogen identification is often essential for guiding effective treatment. While culture-based methods remain the diagnostic gold standard, they are slow and sometimes impractical. Faster metagenomic (mNGS) tools typically detect either ARGs, taxonomy, or genomic context, but rarely all three, resulting in fragmented surveillance. Existing k-mer classifiers like Kraken2 and CLARK, designed for general taxonomy, often perform poorly on AMR-specific sequences. We introduce CARD k-mers, the first tool built to jointly predict species-level taxonomy and genomic context (plasmid vs. chromosome) for ARGs in short metagenomic reads. Integrated with the Comprehensive Antibiotic Resistance Database (CARD), CARD k-mers enables rapid, context-aware assignment of ARGs to their likely pathogen and genomic element origin. In benchmarking with 103,456 in-silico pathogen-specific AMR alleles, CARD k-mers outperformed Kraken2 and CLARK by 10.85% and 15.2%, respectively, and correctly classified the genomic context of 4,590 chromosome- and 176 plasmid-specific ARGs. The tool operates at speeds exceeding 675,000 metagenomic reads per minute. By delivering fast, accurate, and context-rich classification of ARGs, CARD k-mers significantly advances untargeted AMR surveillance and is accessible to users with basic command-line experience for use in both clinical and environmental pipelines. CARD k-mers is available at: https://github.com/arpcard/rgi.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.15.676352v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CluSeek: Bioinformatics Tool to Identify and Analyze Gene Clusters</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hrebicek, O., Kadlcik, S., Najmanova, L., Janata, J., Kamanova, J., Hanzlikova, L., Koberska, M., Kovarovic, V., Kamenik, Z.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present CluSeek, an open-source, Python-based platform for discovering, visualizing, and analyzing gene clusters across all GenBank data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Gene clusters are key structural and functional units that encode diverse phenotypes in genomes, from metabolism to pathogenesis. As genome sequencing expands, tools for systematic exploration of this growing data are increasingly needed. We present CluSeek, an open-source, Python-based platform for discovering, visualizing, and analyzing gene clusters across all GenBank data. Unlike existing tools, CluSeek does not rely on predefined cluster types or reference libraries but enables mining of any gene neighborhoods containing colocalized homologs of user-specified genes. It features an intuitive graphical interface suitable for non-bioinformaticians and is freely available at https://cluseek.com. We demonstrate the versatility of Cluseek in two distinct case studies: (i) mining of specialized metabolites, where CluSeek uncovered over 16 new classes containing the bioactivity-enhancing 4-alkyl-L-proline moiety, previously known in only three Golden Era antibiotic classes; and (ii) analysis of type III secretion systems present in Bordetella species, revealing previously unrecognized taxonomic distribution, and genetic variants, including gene multiplications and novel components with potential functional significance.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.16.676505v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MIND: Multimodal Integration with Neighbourhood-aware Distributions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Xing, H., Yau, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a multi-omics integration framework that learns patient-specific embeddings from incomplete multiomics data based on a multimodal Variational Autoencoder with a data-driven prior.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Multi-omics profiling has become a powerful tool for biomedical applications such as cancer patient stratification and clustering. However, the characterisation and integration of multi-omics data remain challenging because of missingness and inherent heterogeneity. Methods such as imputation and sample exclusion often rely on strong assumptions that could potentially lead to information loss or distortion. To address these limitations, we propose a multi-omics integration framework that learns patient-specific embeddings from incomplete multiomics data based on a multimodal Variational Autoencoder with a data-driven prior. Specifically, we inject neighbourhood structure of the observed dataset encoded as affinity matrices into the prior of embeddings through exponential tilting, and use this prior to penalise the configuration of the latent embeddings based on the discrepancy between the neighbourhood structures in the data spaces and in the latent space. Our proposed method handles high missing rate and unbalanced missingness pattern well, and is robust in the presence of data with a low signal-to-noise ratio. Compared with existing data integration methods, the proposed method achieves better performance on a range of supervised and unsupervised downstreaming tasks on both synthetic and real data.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.15.676314v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">AmyloDeep: pLM-based ensemble model for predicting amyloid propensity from the amino acid sequence</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Davtyan, A., Khachatryan, A., Petrosyan, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We used advanced machine learning and deep learning techniques to predict amyloid propensity from the amino acid sequence.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Amyloids are predominantly {beta}-sheet-rich, stable protein structures that can maintain their presence in the human body for multiple years. Amyloid protein aggregates contribute to the development of multiple neurodegenerative diseases, such as Alzheimer's, Parkinson's, and Huntington's, and are involved in different vital functions, such as memory formation and immune system function. Here, we used advanced machine learning and deep learning techniques to predict amyloid propensity from the amino acid sequence. First, we aggregated labeled amino acid sequence data from multiple sources, obtaining a roughly balanced dataset of 2366 sequences for binary classification. We leveraged that data to both fine-tune the ESM2 model and to train new models based on protein embeddings from ESM2 and UniRep. The predictions from these models were then unified into a single soft voting ensemble model, yielding highly robust and accurate results. We further made a tool where users can provide the amino acid sequence and get the amyloid formation probabilities of different segments of the input sequence. Users can access the light version of AmyloDeep through the web server at https://amylodeep.com/, and the full model is available as a Python package at https://pypi.org/project/amylodeep/.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.16.676495v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">DipGNNome: Diploid de novo genome assembly with geometric deep learning and beam-search</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Schmitz, M., Vrcek, L., Kawaguchi, K., Sikic, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present DipGNNome, the first deep learning based framework for diploid de novo genome assembly.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> De novo genome assembly remains a central challenge in computational biology, particularly for diploid genomes where maternal and paternal haplotypes must be accurately resolved. Existing assemblers achieve impressive results through carefully designed heuristics, yet modern deep learning methods remain largely unexplored in the diploid setting. We present DipGNNome, the first deep learning based framework for diploid de novo genome assembly. Our approach formulates assembly as an edge classification and traversal problem on haplotype-aware assembly graphs, training graph neural networks (GNNs) to guide contig construction. To enable this, we establish a novel pipeline for generating diploid graphs with ground-truth edge labels, providing the first systematic way to produce training data for machine learning models in this domain. This framework creates a foundation for applying and extending graph-based deep learning to diploid assembly. DipGNNome creates assemblies comparable to state-of-the-art and demonstrates the feasibility of deep learning for diploid assembly and introduces a paradigm that bridges algorithmic genomics with graph representation learning. Our code, dataset, and trained model are openly available at https://github.com/lbcb-sci/DipGNNome.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.16.676474v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Sequence-Independent RNA Sensing in Living Mammalian Cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kolber, N. S., Kaseniit, K. E., Lanz, T. V., Robinson, W. H., Gao, X. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Modular RNA sensors using adenosine deaminases acting on RNA ("modulADAR").</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Recently, several groups described sensors in living cells that take advantage of adenosine deaminases acting on RNA (ADARs) to link the presence of an RNA (a "target transcript") to the translation of a payload from a second, exogenously introduced mRNA. These sensors share the key mechanism of editing a stop codon opposite a specific sequence motif in the target transcript, where this motif requirement is dictated by ADAR's strong sequence preference. This constrains sensor design and precludes the sensing of short sequences that lack such motifs, often essential for key applications such as sensing viral RNAs and differentiating splice isoforms. Here we address this limitation with modular RNA sensors using adenosine deaminases acting on RNA ("modulADAR"). ModulADAR features two key elements that mirror the modularity of ADARs: regions that hybridize with the target transcript to recruit ADAR's dsRNA-binding domains, and a stem-loop for stop-codon editing by ADAR's catalytic domain. We optimize modulADAR and apply it to detect short subsequences that cannot be sensed by prior-generation sensors. We anticipate that modulADAR will empower broader basic science and therapeutic applications, especially those that will uniquely benefit from programmable RNA detection in living cells.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.18.677213v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Dynamic pharmacophores unveil binding mode ensembles for classical partial agonists at the M2 receptor</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wunsch, F., Kauk, M., Filor, J., Wolber, G., Holzgrabe, U., Hoffmann, C., Bermudez, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We apply dynamic pharmacophores to unveil binding mode ensembles for classical M2 partial agonists and show that orthosteric partial agonist can stabilize both active and inactive receptor states.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> For the prototypic M2 receptor it has been previously demonstrated that dualsteric partial agonists can stabilize both, active and inactive receptor states, but it remains unclear whether orthosteric partial agonists have a similar mechanism. Here, we apply dynamic pharmacophores to unveil binding mode ensembles for classical M2 partial agonists. We report correlations between the spatial distribution of lipophilic contacts and ligand efficacy and demonstrate the applicability of dynamic pharmacophores to analyze subtle binding mode changes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.17.671681v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">FoxP3 forms a head-to-head dimer in vivo and stabilizes its multimerization on adjacent microsatellites</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Leng, F., Clark, R., Zhang, W., Viennet, T., Arthanari, H., Wang, X., Hur, s.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We demonstrate that FoxP3 binds genomic DNA as both H-H dimers and multimers in Tregs, with H- H binding often seeding and stabilizing multimerization on adjacent TnG repeats, especially on shorter, suboptimal repeats.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> FoxP3, the master regulator of Tregs, employs two DNA-binding modes to recognize diverse DNA sequences. It multimerizes on long TnG repeats (n = 2-5) to bridge DNA segments and stabilize chromatin loops, and it forms head-to-head (H-H) dimers on inverted repeat forkhead motifs (IR-FKHM) without bridging DNA. Although genomic data confirm its multimeric role, in vivo evidence for H-H dimerization has been elusive. Here, unbiased pull-down sequencing uncovers a range of relaxed motifs that drive H-H dimerization, enabling systematic genome-wide analysis. We demonstrate that FoxP3 binds genomic DNA as both H-H dimers and multimers in Tregs, with H-H binding often seeding and stabilizing multimerization on adjacent TnG repeats, especially on shorter, suboptimal repeats. While multimerization is conserved across FoxP family members, H-H dimerization is unique to FoxP3 orthologs, conferred by its divergent accessory loop. This dual-mode strategy broadens FoxP3s sequence repertoire and enhances its architectural function in chromatin looping.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.16.676304v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ARID1A and ARID1B safeguard B cell identity, prevent myeloid transformation and expose therapeutic vulnerabilities in lymphoma</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lin, P. N., Park, J., Kang, Y.-A., Souroullas, G. P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A conditional deletion of the Arid1a/Arid1b gene in early B cell development leads to myeloid leukemia, and suggests context-dependent therapeutic vulnerabilities.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Chromatin remodeling by the SWI/SNF complex is essential for hematopoietic lineage commitment and differentiation. While core subunits ARID1A and ARID1B are frequently mutated in B cell malignancies, their lineage-specific functions remain unclear. Here, we used CD19-Cre-mediated conditional deletion initiated at the pro B cells stage to investigate the function of Arid1a/1b in vivo in mice. Loss of either gene partially blocked B cell differentiation in the bone marrow, reducing immature/recirculating B cell output, and impaired germinal center formation following antigen challenge. Combined deletion resulted in a more severe phenotype, with marked reduction in peripheral B cells, shortened survival, and development of an aggressive leukemia. Unexpectedly, the malignancy was of myeloid origin and arose from a small subset of CD19-expressing multipotent progenitors (MPPs). Arid1a/Arid1b-deficient MPPs exhibited abnormal expansion, reduced colony formation in methylcellulose, and transcriptional dysregulation of stemness and lineage-priming programs, including diminished CBFA2T3 (ETO2) and Fli1 signatures. In established B cell lymphoma cells in vitro, double ARID1A/ARID1B loss only modestly affected cell growth, whereas loss of ARID1A increased sensitivity to pharmacological EZH2 inhibition. Transcriptomic analysis revealed major alterations in cell adhesion/migration pathways, cytokine-receptor interactions and DNA repair mechanisms. Collectively, these findings reveal stage-specific and compensatory roles for ARID1A and ARID1B in B cell development, uncover a mechanism by which SWI/SNF loss in early progenitors can drive transformation towards myeloid leukemia, and suggests context-dependent therapeutic vulnerabilities.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.16.676393v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Gain-of-function mutant p53 regulates long-noncoding RNA LINC00643 to modulate HIF1α in glioblastoma</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, Y., Yuan, F., Gibert, M. K., Dube, C. J., Hudson, K., Grello, C., Reon, B., Saha, S., Sun, Y., Marcinkiewicz, P., Dutta, A., Chief, D., Holland, E., Abounader, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose a novel mechanism by which GOF-MUT-p53 drives GBM through repression of LINC00643 and dysregulation of the HIF1 pathway.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Gain-of-function mutations of p53 (GOF-MUT-p53) act as oncogenes by regulating gene transcription. We screened for the genome-wide transcriptional targets of GOF-MUT-p53 in glioblastoma (GBM) and found that a significant subset of them were long non-coding RNAs (lncRNA). Among these, LINC00643 was strongly repressed by GOF-MUT-p53 but not wild-type p53. LINC00643 was downregulated in GBM and low-grade glioma and correlated with patient survival. LINC00643 and its conserved third exon (Exon3) suppressed GBM cell proliferation, migration, invasion, stem cell self-renewal, and in vivo tumor growth Mechanistically, ChIRP-seq identified HIF1 as a key LINC00643 interactor. Under hypoxia, LINC00643 repressed HIF1 expression and its target genes by interacting with the HIF1 enhancer. Knockdown of GOF-MUT-p53 upregulated LINC00643 and reduces HIF1, revealing a regulatory axis. These findings show extensive regulation of lncRNAs by GOF-MUT-p53 and uncover a novel mechanism by which GOF-MUT-p53 drives GBM through repression of LINC00643 and dysregulation of the HIF1 pathway.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.16.676559v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">E2F3 amplification primes bladder cancer cells for premature mitosis.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wierenga, K. A., Liu, Q., Nieland, I. S., Rakers, P. R., van den Heuvel, A., Pateli, M., Wubbolts, R. W., Riemers, F. M., van Essen-Dorresteijn, S., van Liere, E. A., Westendorp, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We found that E2F3 amplification causes hyperactivation of the FOXM1-dependent mitotic gene expression program, including cyclin B1, which leads to premature mitosis in bladder cancer.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The E2F-RB pathway is essential to enforce the G1/S checkpoint. Hence, many tumors contain mutations that override this checkpoint, such as RB1 loss or CDKN2A loss, to activate E2F dependent transcription, which facilitates cell cycle entry. However, these mutations do not necessarily result in sustained E2F-dependent transcription throughout the cell cycle. This is different in tumors with E2F3 amplification, because the resulting E2F3 overexpression persists through S- and G2-phase to cause excessive and unscheduled E2F-dependent transcription. E2F3 is a bona fide oncogene, which is frequently amplified in bladder cancer. Here we investigated how E2F3 amplification impacts the cancer cell cycle. Using isogenic bladder cancer cell lines and patient data we found that E2F3 amplification causes hyperactivation of the FOXM1-dependent mitotic gene expression program, including cyclin B1. This primes cells to undergo unscheduled and catastrophic mitosis when the mitotic kinase CDK1 was hyperactivated by treating cells with the PKMYT1 inhibitor RP-6306, thus uncovering a promising synthetic lethal interaction. Bladder cancer cells with E2F3 amplification developed resistance to RP-6306 by reducing cyclin B1 expression, thereby mitigating premature mitotic entry. This resistance could be overcome by adding a low dose of WEE1 inhibitor to PKMYT1 inhibition. Together, these data strongly suggest that PKMYT1-dependent inactivation of CDK1 is essential to prevent premature mitosis in bladder cancer cells harboring E2F3 amplifications. This suggests an opportunity for precision medicine strategies in bladder cancer patients with E2F3 amplification and/or excessive cyclin B1 expression.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.16.676590v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Modulating Glutamine Metabolism Reprograms Pro-Inflammatory Differentiation in Macrophages</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Qi, S., Fan, J., Wu, D.-S., Ganguly, S., Shih, I.-M., Wang, T.-L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that prolonged glutamine inhibition elevates glutamine levels, which sustain pro-inflammatory gene transcription, resulting in sustained pro- inflammatory activation of M1 macrophages.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Previous in vivo studies demonstrated that JHU083/DON, a glutamine analog drug, potently reprograms M1/M2 macrophages. To determine whether these effects are direct or indirect, we utilized an in vitro murine bone marrow derived macrophage (BMDM) model, which recapitulates macrophage differentiation and polarization processes, to examine the impact of DON on the M1 macrophages. DON was applied during M1 differentiation or to fully polarized M1 macrophages, revealing that glutamine inhibition initially suppressed M1 activity but later enhanced it, resulting in sustained pro-inflammatory activation. Multi omics analyses (bulk RNA seq and LC MS), time course assays, and glutamine depletion experiments consistently suggested that prolonged glutamine inhibition elevates glutamine levels, which sustain pro-inflammatory gene transcription. In contrast, M2 and tumor-associated macrophages (TAM), which are immunosuppressive, were more susceptible to DON, leading to functional suppression. Collectively, our findings uncover stage specific mechanisms by which glutamine inhibition modulates M1 polarization, offering a mechanistic rationale for therapeutic strategies that sustain pro-inflammatory, anti tumor macrophage activity while concurrently suppressing immunosuppressive myeloid subsets in cancer.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.16.675216v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Targeting DNA Polymerase Epsilon Leads to Tumor Clearance and Activation of an NF-κB mediated inflammatory response in Triple Negative Breast Cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sher, E., Fujihara, K., Tao, A., Erenburg, D., Sviderskiy, V., Mir, H., Karakousi, T. R., Loomis, C., Deng, J., Wong, K.-K., Possemato, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that DNA replication stress from POLE inhibition in triple-negative breast cancer triggers robust inflammation and immune microenvironment remodeling in TNBC and can independently trigger tumor eradication.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Breast cancer remains the second leading cause of cancer-related mortality among women, with triple-negative breast cancer (TNBC) exhibiting a particularly poor five-year prognosis. Here, we demonstrate that among genetic and pharmacological perturbations targeting DNA replication, suppression of DNA polymerase epsilon (POLE) in TNBC, induces a potent, TNBC-specific gene expression signature enriched in inflammatory cytokines that are transcriptional targets of NF-KB. TNBC cells exhibit markedly higher levels of DNA damage and canonical NF-KB activation compared to luminal breast cancer cells. Notably, NF-KB activation in this context depends on the canonical component RELA, but not the non-canonical component RELB. Mechanistically, ATM, STING, and RIG-I each contribute to NF-KB activation following POLE suppression. In vivo, POLE suppression in a murine TNBC model leads to cancer cell-intrinsic elimination of tumor burden and increased immune cell infiltration. Together, these findings support a model in which replication stress from POLE inhibition triggers robust NF-KB-mediated inflammation and immune microenvironment remodeling in TNBC and can independently trigger tumor eradication. These results suggest a potential therapeutic avenue for targeting POLE in TNBC.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.18.677147v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Dual EZH1/2 inhibition enhances DNMT inhibitor efficacy in colon cancer through targeting H3K27me1</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chomiak, A. A., Wiseman, A. K., Hrit, J., Liu, Y., Stransky, S., Sidoli, S., Tiedemann, R. L., Rothbart, S. B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that EZH1/2 inhibition redistributes p300/CBP activity, reducing H3K27ac from oncogenic loci and redirecting it to bivalent regions that enable TSG re-expression.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Our recent work showed that low-dose DNMT inhibitor (DNMTi) treatment sensitizes colon cancer cells to EZH2 inhibitors (EZH2i), synergistically upregulating tumor suppressor genes (TSGs) and transposable elements through activation of the calcium-calcineurin-NFAT signaling pathway. A key observation was that EZH2i displayed variable sensitivities in combination therapy, which could not be explained solely by loss of lysine 27 trimethylation on histone H3 (H3K27me3), the most commonly studied EZH2 product. This led us to perform a comprehensive pharmacologic screen of Polycomb Repressive Complex 2 (PRC2) antagonists. Here, we show that compounds targeting both EZH2 and its interchangeable catalytic subunit, EZH1, achieved superior TSG re-expression when combined with DNMTi. Integrative proteomic and epigenomic analyses revealed that EZH1/2 inhibitors reduce all three H3K27 methylation states, whereas EZH2-selective inhibitors preserve EZH1-dependent H3K27me1 at deeply Polycomb-repressed genomic regions. Notably, H3K27me1 loss coincided with deposition of p300/CBP-dependent lysine 27 acetylation on histone H3 (H3K27ac), which proved essential for TSG re-expression. Paradoxically, blocking p300/CBP activity further enhanced the growth-inhibitory effects of combined DNMT and EZH1/2 inhibition. Mechanistically, we show that EZH1/2 inhibition redistributes p300/CBP activity, reducing H3K27ac from oncogenic loci and redirecting it to bivalent regions that enable TSG re-expression. Collectively, these findings reveal a coordinated role for EZH1-dependent H3K27me1 and DNA methylation in sustaining oncogenic transcriptional programs and provide strong rationale for advancing dual EZH1/2 inhibitors for combination epigenetic cancer therapy.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.16.676613v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Exploiting TGF-β-mediated Stromal Programming in Homologous Recombination-Deficient Pancreatic Cancer</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Roger, E., Mummey, H. M., Zimmer, E., Srinivasan, D., Härle, A., Moubri, L., Beutel, A. K., Singh, R., Ekizce, M., Melzer, M. K., Lee, Y., Silva, A., Härle, L., Engleitner, T., Arnold, F., Morawe, M., Naggay, B., Schneider, J., Gilberg, L., Mosler, J. P., Ludwig, C., Meng, C., Hirschenberger, M., Hunszinger, V., Kluck, K., Kirchner, M., Volckmar, A.-L., Wirth, M., Alhamdani, M. S. S., Hoheisel, J. D., Löhr, J.- M., Seufferlein, T., Abaei, A., Kemkemer, R., Rad, R., Budczies, J., Mulaw, M., Hermann, P. C., Hänle, M., Sparrer, K. M., Halbrook, C. J., Gaulton, K. J., Steinestel, K., Stenzinge</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that ATM-deficient tumors shape a cancer-promoting niche via TGF-{beta} signaling and identify dual targeting of intrinsic and extrinsic vulnerabilities as a promising precision oncology strategy.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The tumor microenvironment (TME) actively contributes to pancreatic ductal adenocarcinoma (PDAC) pathogenesis through dynamic bidirectional tumor-stroma interactions. Here, we demonstrate that homologous recombination-defective (HRD) tumor epithelium reprograms the TME in a genotype-specific manner to enhance cancer aggressiveness. Using genetically engineered mouse models, pancreatic stellate cell (PSC) and cancer-associated fibroblast (CAF) co-culture systems, single-nucleus multiomics, and human PDAC models, we show that tumoral loss of ATM serine/threonine kinase drives CAFs toward SMA+ myofibroblastic differentiation, independently of P53 status. These myCAFs, in turn, promote cancer aggressiveness and chemoresistance. Mechanistically, ATM deficiency increases reactive oxygen species and contractility signaling, enhancing TGF-{beta}1 secretion. Pharmacological TGF-{beta} inhibition reverses myCAF differentiation, sensitizes tumors to chemotherapy, and impairs tumor progression in both murine and human ATM-null models. Our findings reveal that ATM-deficient tumors shape a cancer-promoting niche via TGF-{beta} signaling and identify dual targeting of intrinsic and extrinsic vulnerabilities as a promising precision oncology strategy.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.16.676458v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Functional biomaterials and machine learning approaches for phenotyping heterogeneous tumor cells and extracellular vesicles.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rutwik Joshi, Raheel Ahmad, Karl Gardner, Hesaneh Ahmadi, Chau-Chyun Chen, Shannon L Stott, Wei Li</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A review of emerging biomaterial-based approaches that enable selective sorting, profiling, and detection of circulating tumor cells and tumor derived vesicles.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Heterogeneity in cancer is known to be a contributor to the formation of metastatic lesions, poor prognosis, and ultimately undermines therapeutic efficacy. This same tumor heterogeneity is reflected in circulating tumor cells (CTCs) and tumor derived extracellular vesicles (EVs), offering a less invasive snapshot into tumor status. The isolation and phenotypic characterization of CTCs and EVs has attracted significant interest in recent years as they offer great potential for deciphering the molecular basis of disease progression and the development of precision therapies. Next-generation biomaterials and advanced machine learning paradigms are transforming how we decipher phenotypic heterogeneity in these circulating biomarkers and help provide new insights into tumor biology and therapy resistance. In this review, we first briefly describe biomaterial-based platforms for the isolation of CTCs and EVs, followed by a detailed discussion of the pivotal role of phenotypic profiling and molecular identification. Finally, we provide a review of emerging biomaterial-based approaches that enable selective sorting, profiling, and detection of CTCs and EVs. In this process, we categorize the most widely utilized biomaterials into polymer-based materials, quantum dots and multifunctional magnetic nanospheres, fluorescent antibodies, Surface Enhanced Raman Spectroscopy (SERS) vectors, DNA-based multi-aptamer probes, cell-imprinted substrates, and silver nanoclusters. We then explore the application of machine learning algorithms in biomarker profiling of CTCs, extending to EVs. Furthermore, we provide a comprehensive analysis of relevant clinical studies and critically examine future challenges and research trajectories in this rapidly evolving field.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40963467/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

