<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Monday, 25 August 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Insights from controlled, comparative experiments highlight the limitations of using BSMV and FoMV for virus-enabled reverse genetics in rice</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Turra, G. M., Merotto, A., MacGregor, D. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-25
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Virus-enabled reverse genetics (VERG) is a powerful tool for transient gene expression modulation in plants, particularly where stable transformation is challenging. However, the efficacy of VERG varies across species. In this study, we tested two commonly used viral vectors, barley stripe mosaic virus (BSMV) and foxtail mosaic virus (FoMV), for their ability to induce virus-induced gene silencing (VIGS) and virus-mediated overexpression (VOX) in rice (Oryza sativa L.). While both vectors successfully altered gene expression in wheat (Triticum aestivum), they failed to do so in six rice cultivars from different subspecies, despite rigorous optimization of inoculation methods and environmental conditions. The BSMV vector carrying antisense phytoene desaturase (PDS) sequences did not induce the expected photobleaching phenotype, and FoMV-driven GFP expression was absent in rice. These findings contrast with previous reports of successful VERG in other monocots and suggest that intrinsic resistance mechanisms exist in rice which may inhibit or reduce viral vector efficacy. Our results highlight the species-specific limitations of VERG and underscore the need for alternative viral systems or novel vector designs for functional genomics research in rice. By sharing our unsuccessful attempts, we aim to prevent redundant efforts and encourage further exploration of VERG in Oryza species.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.21.671469v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">RnaXtract, a tool for extracting gene expression, variants, and cell-type composition from bulk RNA sequencing.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sophiane G Bouirdene, Simon Gotty, Mickaël Leclercq, Charles Joly-Beauparlant, Emeric Texeraud, Steve Bilodeau, Arnaud Droit</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> RNA sequencing (RNA-seq) is a widely used method in transcriptomics research, offering insights into gene expression, variant discovery, and, when deconvoluted, the cellular composition of complex tissues. However, existing RNA-seq pipelines frequently emphasize gene expression analysis and often lack cell deconvolution and variant calling. To address these limitations, we present RnaXtract, a comprehensive and user-friendly pipeline designed to maximize extraction of valuable information from bulk RNA-seq data. RnaXtract automates an entire workflow, encompassing quality control, gene expression quantification, variant calling, and the cell-type deconvolution. Built on the Snakemake framework, RnaXtract ensures robust reproducibility, efficient resource management, and flexibility to adapt to diverse research needs. The pipeline integrates state-of-the-art tools, from quality control to the new updates on variant calling and cell-type deconvolution tools such as EcoTyper and CIBERSORTx, enabling researchers to extract biological insights with precision. By providing an end-to-end solution for bulk RNA-seq, RnaXtract addresses critical gaps in existing workflows, empowering researchers to explore gene expression, genetic variation, and cellular heterogeneity within a single cohesive framework.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40850998/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Expanded Catalog of Gold Standard Ciliary Gene List: Integration of Novel Ciliary Genes into the Ciliome</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yenisert, F., Kaplan, O. I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Ciliopathies are rare genetic disorders caused by mutations in genes that encode proteins critical for cilium assembly and function, affecting various subcompartments, including the axoneme, ciliary membrane, transition zone, and basal body. These mutations lead to diverse developmental and degenerative conditions across multiple organ systems. The 2021 ciliary Gold Standard gene list (688 genes) provides a foundational resource, but many ciliary genes remain unidentified or unclassified. We developed CiliaHub (https://theciliahub.github.io/), a scalable and user-friendly platform that integrates automated literature mining with expert manual curation to identify human ciliary genes. The approach combines gene names, synonyms, and targeted keywords (e.g., cilia, flagella, basal body) to extract gene-specific mentions from published studies, followed by rigorous validation through independent expert review. CiliaHub identified 2011 experimentally supported ciliary genes, including 1323 not previously listed in curated databases. Notably, some of these genes are co-found in both cilia and other organelles, such as mitochondria and lysosomes. Cross-referencing with mouse knockout model data revealed strong associations with ciliopathy-relevant phenotypes, such as infertility in Cep70 and renal defects or hearing abnormalities in genes like TMEM145. This work also provides novel ciliopathy candidate genes for further investigation. CiliaHub provides a broad platform for fundamental and translational research, enabling the effective prioritization of candidate disease genes, including recently identified ciliopathy candidates.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.22.671678v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Improving RNA-seq protocols</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Pfoertner, F., Briem, E., Enard, W., Richter, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Bulk and single-cell RNA-seq are powerful tools for transcriptomic analysis, providing insights into many aspects of molecular and cellular phenotypes. Costs constrain the amount of biological insight obtainable within a given budget, and as sequencing prices decline, efficient library protocols have become a decisive factor. In this study, we introduce an approach to systematically optimize the number of usable reads that RNA-seq protocols generate. We applied this "funnel strategy" to prime-seq, an early-barcoding bulk RNA-seq protocol, by systematically testing critical protocol steps totaling 1080 samples in 49 libraries. This resulted in the optimized prime-seq2 protocol that increases the number of usable reads by 60 % and improves one of the most cost-efficient bulk RNA-seq protocols available. Our study also suggests that monitoring the filtering of usable reads can serve as a valuable quality control for many RNA-seq protocols and sheds light on the complexity of the conditions and interactions that shape RNA-seq library composition and their interpretation.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.20.671269v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Viruses associated with Nitrososphaerota archaea: diversity, evolution and horizontal gene transfers</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zheng, K., Liang, Y., Paez-Espino, D., McMinn, A., Wang, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Nitrososphaerota, an archaeal phylum mainly containing Thaumarchaea and Aigarchaea, is an ecological widespread archaeal lineage belonging to TACK super-phylum. However, viral diversity and evolution associated with this archaeal lineage, especially for extremophilic Thaumarchaea and Aigarchaea, remain uncharacterized. Here, we reveal the neglected diversity of the Thaumarchaea virome by identifying 152 species-level viral and mobile genetic element (MGE) sequences associated with extremophilic Thaumarchaea, which mainly colonize in hyperthermophilic environments, deep sea, and acidic soils. Importantly, we identified 263 previously unrecognized species-level viral/MGE sequences associated with Aigarchaea, including a novel rod-shaped viral family uniquely characterized by capsid heterodimers, thereby filling a long-standing gap in the viral diversity of the TACK super-phylum. Phylogenetic analysis revealed close evolutionary relationships between Aigarchaea-associated rod-shaped viruses and Methanophagales (ANME-1) rod-shaped viruses, suggesting that they may have evolved from a common rod-shaped viral ancestor. In contrast, Aigarchaea-associated head-tailed viruses formed three distinct monophyletic clades compared to other viruses, representing previously undiscovered evolutionary lineages within virosphere. Gene functional analysis identified 32 viral-encoded genes potentially involved in supplementing host metabolism, including, for the first time, genes that confer selective advantages to their extremophilic archaeal hosts, such as those involved in oxidative stress adaptation and antibiotic synthesis. Approximately 54% of 6,505 horizontal gene transfers (HGTs) occur bidirectionally between Nitrososphaerota and viruses/MGEs, highlight the significant role of viruses/MGEs in facilitating gene exchange within the archaeal communities. Collectively, this work sheds light on the previously unknown Aigarchaea-associated viromes, and unveils the underexplored diversity, evolution, functional repertoires, and HGTs of Nitrososphearota-associated viruses.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.23.671958v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Quantification method affects replicability of eQTL analysis, colocalization, and TWAS</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Cole, N., Wu, W. H., Head, S. T., Bresnahan, S. T., Bhattacharya, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> eQTL mapping and TWAS are widely used to contextualize GWAS, yet the impact of RNA-seq processing choices remains unexplored. We find that RNA-seq quantification method and transcriptomic reference substantially affect eQTL detection and gene expression prediction with significant downstream impact on colocalization and TWAS results. Our findings demonstrate that seemingly minor methodological decisions substantially affect these common analyses, highlighting the need for standardized practices to ensure reproducible genetic association studies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.20.671303v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MRDadaptis: Self-Adaptive Parameter Configuration Enhances Minimal Residual Disease Detection in Heterogeneous ctDNA Samples</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> wang, t., Wang, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Detection of structural variations (SV) through circulating tumor DNA (ctDNA) has become a key method for detecting minimal residual disease (MRD). However, the heterogeneity of ctDNA samples, characterized by variable limits of detection (LOD) and diverse structural variant types, significantly impacts detection stability and consistency, posing persistent challenges for conventional SV detection tools such as Delly and Manta. These widely-used methods require extensive manual parameter tuning, hindered by the combinatorial complexity of multiple parameters and heterogeneous sequencing data. To address this, we propose MRDadaptis, a novel SV detection tool that uniquely incorporates a self-adaptive parameter optimization mechanism. MRDadaptis distinguishes itself by integrating Bayesian optimization with meta-learning techniques to dynamically adjust detection parameters automatically, based on intrinsic features derived from the ctDNA sequencing data itself. This innovative approach not only reduces manual intervention but also effectively captures sample-specific characteristics, significantly improving detection stability and detection performance. Extensive validation experiments using both simulated and real-world ctDNA datasets demonstrates MRDadaptis distinct advantages, including markedly improved average F1-scores and superior stability (reduced variance, lower RMSE, increased kurtosis) These results highlight the significant advantages of MRDadaptis in addressing sample heterogeneity, underscoring its potential to improve the accuracy and reliability of MRD detecting through ctDNA analysis. https://github.com/aAT0047/MRDadaptis.git Keywords: structural variant detection, ctDNA, Self-Adaptive Parameter Configuration, Bayesian optimization, meta-learning, heterogeneous sequencing sample.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.20.671240v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">p63-GATA2 molecular switch mediates the tumor suppressor to oncogene transition of glucocorticoid receptor in the prostate</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Hiltunen, J., Aaltonen, N., Sohlberg, H., Kemppi, L., Paakinaho, V.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Although glucocorticoids are widely used to alleviate side effects of prostate cancer (PCa) treatment, the glucocorticoid receptor (GR) exhibits a dual role exerting tumor-suppressive effects by inhibiting early-stage PCa cell proliferation, while also promoting oncogenic progression by mediating antiandrogen resistance. The mechanisms underlying this functional dichotomy have remained elusive and poorly characterized. Using genome-wide analyses and CRISPR-based genome editing, we identified the tumor protein p63 as a key mediator of GR's tumor-suppressive chromatin activity. Loss of p63 reprograms GR activity toward an oncogenic state, marked by enhanced cell migration, invasion, and altered morphology. This shift is driven by increased GATA2 expression, which alters GR's chromatin binding and transcriptional output. Together, our findings uncover a p63-GATA2 molecular switch that governs the dual role of GR in PCa, establishing transcription factor crosstalk as a critical regulator of GR-driven oncogenic reprogramming and cellular plasticity.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.20.670078v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">CanID: a robust and accurate RNAseq Expression-based diagnostic classification scheme for pediatric malignancies</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Putnam, D. K., Gout, A. M., Rahbarinia, D., Jin, M., Finkelstein, D., Ma, X., Zhang, J., Wheeler, D. A., Furtado, L. V., Chen, X.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cancer subtype classification is critical for precision therapy and there is a growing trend of augmenting histopathology testing procedures with omics-based machine learning classifiers. However, analytical challenges remain for pediatric cancer on the scope and precision of the current classifiers as well as the evolving subtype standardization. To address these challenges, we built Cancer Identification or CanID, a stacked ensemble machine learning classification scheme, using the transcriptomic features derived from gene-level RNA sequencing count data as the sole input. CanID was developed primarily from 3203 pediatric cancer samples of 13 solid tumor subtypes and 38 hematologic malignancy subtypes with subtype labels curated without the use of RNA-seq data. The accuracies of independent testing in three independent or external data sets for Solid Tumor and Hematologic Malignancy are 99% and 92-93%, respectively. Notably, CanID was able to classify subtypes challenging for clinical histology evaluation and was robust to both biological and technical challenges, including differences in data collection protocols, class imbalance, potential mislabeled training samples and classes unobserved in training. The high accuracy, robustness, biological interpretability of this transcriptome-based classification scheme represents a valuable approach to advance tumor diagnosis and clinically meaningful stratification of tumor types. CanID can be accessed on GitHub at https://github.com/chenlab-sj/CanID.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.20.671349v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Benchmarking large language models for cell-free RNA diagnostic biomarker discovery</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> De Vlaminck, I., Gaudio, H., Bliss, A., Loy, C., Eweis-LaBolle, D., Gardella, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Large-language models (LLMs) can parse vast amounts of data and generate executable code, positioning them as promising tools for the development of biomarkers and classifiers from high-throughput omics data. Here, we benchmarked six LLMs, OpenAI's o3 and GPT-4o, Anthropic's Claude Opus 4 and Claude 3.7 Sonnet, and Google's Gemini 2.5 Pro and Gemini 2.0 Flash, for disease classification based on plasma cell-free RNA (cfRNA) profiles obtained by RNA sequencing. We analyzed data from cohorts of children with Kawasaki disease (KD) or multisystem inflammatory syndrome in children (MIS-C), adults with active tuberculosis (TB) or other non-TB respiratory conditions, and individuals with myalgic encephalomyelitis/chronic fatigue syndrome (ME/CFS) or sedentary lifestyle. We assessed two tasks: (i) gene-panel design, where each LLM mined public knowledge to nominate diagnostic genes for use in machine learning (ML), and (ii) end-to-end modeling, where LLMs built an ML workflow directly from raw RNA-seq counts. In the first task, the LLM-derived panels captured canonical immune pathways and outperformed randomly selected genes in all cohorts. They underperformed panels chosen by differential gene expression (DGE) analysis in the KD vs. MIS-C and ME/CFS cohorts but performed comparably or better for the TB cohort. In the second task, o3 produced classifiers for KD vs. MIS-C that performed just as well as conventional statistical methods without human intervention. Performance for TB and ME/CFS cohorts was slightly lower than the conventional approach. These findings delineate current capabilities and limitations of LLMs in diagnostics and open a path for their future use in biomarker discovery.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.20.671358v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Spatial transcriptomic profiling of the human aortic valve reveals cellular sex differences near sites of calcification</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Baddour, T., Ninh, V. K., Gorashi, R. M., Pena, R., King, K. R., Aguado, B. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Sex differences in aortic valve stenosis (AVS) progression have been documented clinically, but the underlying cellular mechanisms that drive sex-dependent calcification in aortic valve tissue remain poorly understood. Here, we harnessed single cell and spatial transcriptomics to investigate mechanisms that drive sex dependent spatial organization of valvular interstitial cell (VIC) and macrophage gene expression near calcification sites in human male and female aortic valve tissue. Histological analyses of aortic valve tissues stratified into healthy and diseased cohorts based on degree of calcification reveal increased valve calcification area in diseased male aortic valves relative to female, and increased valve thickening in diseased female aortic valves. Single cell sequencing analysis of heterogeneous valvular interstitial cell (VIC) populations reveals male-dependent gene expression of the Activator Protein 1 (AP-1) transcription factor complex. Spatial transcriptomics and RNA-FISH analyses of VIC populations near sites of calcification revealed male-dependent gene expression localization of Cartilage Oligomeric Matrix Protein (COMP), as opposed to diffuse COMP expression in female VICs. Cell-cell communication analyses were used to determine female-specific macrophage-VIC interactions. Secreted phosphoprotein 1 (also known as osteopontin) expressed from macrophages interacts with the cell surface receptor CD44 expressed by VICs to drive a pro-fibrotic phenotype in female aortic valves. Together, our results reveal sex differences in VIC and macrophage heterogeneity and functions near sites of calcification in aortic valve tissue. Our results highlight the importance of sex-based transcriptomics analyses to understand the cellular phenotypes responsible for causing sex differences in aortic valve fibrosis and calcification.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.20.671175v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">KICDB: A Causality-Oriented Multi-Omics Database for Kinase Inhibitor-Induced Cardiotoxicity</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wei, J., Liu, Y., Wu, M., Li, G., Zheng, X., Fu, H., ZHANG, J., Lin, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;">Background: Kinase inhibitors (KIs) are mainstays of targeted cancer therapy, but their clinical utility is frequently limited by cardiotoxicity. A systematic resource to explore the underlying causal mechanisms is urgently needed. Methods: We present the KICDB (Kinase Inhibitor Cardiotoxicity Database), a comprehensive and interactive web server. KICDB is built upon a framework integrating large-scale transcriptomics meta-analysis with causal inference. Results: This database centralizes the findings from a comprehensive meta-analysis of 26 kinase inhibitors (KIs) across 7 studies (n=5291) identified 8,907 significant gene expression changes in human cardiomyocytes. To establish causality, we performed a two-pronged Mendelian randomization (MR) analysis testing hundreds of downstream genes and a panel of 43 key kinase proteins against 46 cardiovascular outcomes. This large-scale analysis revealed 26 significant causal associations, implicating novel molecular mediators in KI-induced cardiotoxicity. Conclusions: KICDB serves as a valuable and accessible platform for the cardio-oncology community. By integrating transcriptomic signatures with causal inference data, the database empowers researchers to formulate mechanistic hypotheses, accelerate biomarker validation, and guide the design of future cardioprotective strategies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.20.671397v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Targeting Pseudomonas aeruginosa Ventilator-associated pneumonia with a non-antibiotic and biolfilm-disrupting Live Biotherapeutic: Preclinical Safety and Efficacy study</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Fernandez-Barat, L., Motos, A., Segura, C., Kasra Kiarostami, K., Cabrera, R., Soler, A., Broto, A., Mazzolini, R., Galli, F., Llonch, B., Macip, G., Vazquez, N., Calabretta, D., Arrieta, M., Lluch, M., Rigol, M., Serrano, L., Rodriguez-Arce, I., Torres, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Ventilator-associated pneumonia (VAP) is the second most common hospital-acquired infection and is associated with high morbidity and mortality. The insertion of the endotracheal tube (ETT) compromises the host bronchial defenses and facilitates the formation of microbial biofilms and VAP. Although antibiotics remain the cornerstone treatment, their efficacy is often compromised by poor biofilm penetration and the prevalence of multidrug-resistant bacteria. This reveals the need to develop novel non-antibiotic therapeutic strategies. We engineered a non-pathogenic strain of the pulmonary bacteria Mycoplasma pneumoniae CV2 to express anti-biofilm and bactericidal enzymes (VAP platform). The therapeutic effect was validated in a murine model of Pseudomonas aeruginosa infection and ex vivo in ETTs of patients with P. aeruginosa-induced VAP. We evaluated the safety of an improved version of CV2 (CV8, Mycochassis) in a miniature pig model, and its efficacy with the VAP therapeutic platform (CV8_VAP) in a piglet model of P. aeruginosa-induced VAP. CV8_VAP was administered via intralobe instillation or nebulization. We assessed the platform ability to degrade biofilms formed on ETTs in vivo and its impact on the airway microbiota. We evaluated bacterial load in the lungs, inflammatory responses, histopathological and the clinical outcomes to provide a comprehensive characterization of therapeutic efficacy and safety. VAP platform reduced biofilm thickness and P. aeruginosa load in ETTs and lungs, with the nebulized route showing enhanced efficacy. CV8_VAP contributed on microbiome diversity restoration. Inflammatory markers in the lung were markedly decreased, suggesting a local immunomodulatory effect. These findings support that CV8_VAP as a promising, non-antibiotic therapeutic strategy for treating VAP caused by P. aeruginosa.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.24.671965v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Advancing Codon Language Modeling with Synonymous Codon Constrained Masking</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Heuschkel, J., Kingsley, L., Pefaur, N., Nixon, A., Cramer, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Codon language models offer a promising framework for modeling protein-coding DNA sequences, yet current approaches often conflate codon usage with amino acid semantics, limiting their ability to capture DNA-level biology. We introduce SynCodonLM, a codon language model that enforces a biologically grounded constraint: masked codons are only predicted from synonymous options, guided by the known protein sequence. This design disentangles codon-level from protein-level semantics, enabling the model to learn nucleotide-specific patterns. The constraint is implemented by masking non-synonymous codons from the prediction space prior to softmax. Unlike existing models, which cluster codons by amino acid identity, SynCodonLM clusters by nucleotide properties, revealing structure aligned with DNA-level biology. Furthermore, SynCodonLM outperforms existing models on 6 of 7 benchmarks sensitive to DNA-level features, including mRNA and protein expression. Our approach advances domain-specific representation learning and opens avenues for sequence design in synthetic biology, as well as deeper insights into diverse bioprocesses.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.19.671089v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">FAVABEAN and FALAPhyl: Open-Source Pipelines for Scalable 16s rRNA Microbiome Data Processing and Visualization</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Faridoun, A., Carvalho, R., Smith, J., Gibb, A., Jain, L., Zhang, A., Sran, A., Redmond, J., Malik, M. Z., Gibson, M., Haider, A., Rekhi, U., Bhagirath, A., Rock, L. D., Altabtbaei, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Reproducible and scalable analysis of 16S rRNA amplicon sequencing data remains a persistent challenge in microbiome research due to the complexity of available tools, incompatibilities between platforms, and the need for extensive bioinformatics expertise. We developed two containerized workflows--FAVABEAN (Fast Amplicon Variant Annotation, Binning, Error-correction And ANalysis) and FALAPhyl (Forays into Automating Laborious Analyses of Phylogeny)--to address these challenges. FAVABEAN and FALAPhyl are Snakemake-based pipelines designed for flexible execution across local, cluster, and cloud environments. FAVABEAN automates preprocessing, ASV inference, and taxonomic assignment using DADA2 and FIGARO, including primer averaging when samples are sequenced with multiple primers. FALAPhyl supports downstream analysis including alpha/beta diversity, network analysis, and differential abundance testing, with integrated provenance tracking. We validated both pipelines using three case studies involving oral microbiome datasets. In Case Study 1, we compared oral microbiota across family members and niches, showing primer-dependent variability in ASV-based similarity and minimal reseeding from familial sources after prophylaxis. Case Study 2 analyzed dental aerosol samples, revealing no significant microbial differences between pre-, intra-, and post-procedure air. Case Study 3, a randomized trial of a nitrate mouthrinse, demonstrated no significant microbiome shifts, highlighting oral microbial stability. FALAPhyls integration of DAtest enabled empirical evaluation of multiple statistical tests, aiding robust differential abundance inference. FAVABEAN and FALAPhyl offer a reproducible, automated solution for 16S rRNA amplicon data analysis. Their modular design, containerization, and provenance tracking enhance accessibility and scientific rigor in microbiome research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.19.670115v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SRRM1 coordinates an alternative splicing program that promotes expression of oncogenic protein isoforms.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Othman, K., Viola, L., Fatima, H., Lapierre, J., Macleod, G. J., Simpson, C., Chu, C., Zhang, Y., Angers, S., Saulnier, O., McGlade, C. J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The alternative splicing of the adapter protein NUMB is dysregulated in multiple cancer types, regulating its functional divergence towards either tumor suppression or oncogenesis in an isoform dependant manner. Here we utilized a NUMB exon 9 (E9) splicing reporter in a genome-wide CRISPR screen to identify splicing regulators SRRM1 and SRSF11 that promote NUMB oncogenic splicing in colorectal, lung and breast cancer cell lines. Furthermore, SRRM1 and SRSF11 share common protein interactors, RNA targets and effects on an oncogenic splicing program which favors the expression of pro-tumorigenic isoforms. In addition to NUMB E9, SRRM1 regulates oncogenic splicing events in genes encoding signaling proteins, transcription factors and actin cytoskeleton regulators, many of which also undergo developmentally regulated splicing, including CD44, MKNK2, ECT2, DIAPH1, KAT5, TCF7L2, FOXM1 and TBX3. Loss of SRRM1 in colon cancer cells reduces cell proliferation and colony formation capabilities as well as expression of tumour promoters Cyclin D1, Notum, and PRDX2. Our data indicate that SRRM1 regulation of alternative splicing represents a node to target multiple properties of malignant cells, with broad effects on cellular signaling, proliferation, EMT, apoptosis resistance and stemness.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.20.671097v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">RAB5 NUCLEOTIDE BINDING PROMOTES β-OXIDATION TO FUEL HEPATOCELLULAR CARCINOMA CELL PROLIFERATION</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Otakhor, K. O., Zaidi, M. A. A., Deegan, B. O., Ponnusamy, M. P., Schott, M. B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-24
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Altered lipid metabolism and lipid droplet (LD) dynamics are hallmark features of hepatocellular carcinoma (HCC) subtypes, but the molecular mechanisms governing LD trafficking and catabolism in HCC cells remain unclear. The small GTPase Rab5, a key regulator of early endosomal dynamics, has been observed to localize to the surface of LDs, suggesting it may play a role in LD turnover. However, the regulation of Rab5-LD interactions and its functional consequences in HCC cell metabolism and proliferation have not been elucidated. In this study, we explored the role of Rab5 in governing LD homeostasis and its impact on HCC cell proliferation. We found that the GTP-bound (Q79L), active form of Rab5 exhibited increased association with LDs compared to the GDP-bound, inactive mutant (S34N). Nutrient starvation enhanced Rab5 GTP-loading and its recruitment to LDs, indicating that Rab5's GTPase cycle regulates its LD localization. Importantly, inhibition of Rab5 GTP-binding impaired LD catabolism, reduced mitochondrial oxidative phosphorylation, and significantly impaired HCC cell proliferation. Transcriptomic analyses further revealed that RAB5 is significantly overexpressed in HCC patient samples, and this overexpression correlated with poorer overall survival. These findings demonstrate that Rab5's GTPase cycle is a critical regulator of LD dynamics in HCC cells, governing LD turnover to sustain mitochondrial energy production and support cancer cell proliferation. Targeting the Rab5-mediated regulation of LD metabolism may represent a novel therapeutic strategy to disrupt the metabolic adaptations that fuel liver cancer progression.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.20.670915v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

