<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">← back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Wednesday, 06 August 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Molecular co-accessibility identifies coordinated regulation between distant cis-regulatory elements</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Boulanger, M., Chatsirisupachai, K., Snajder, R., Bonder, M. J., Lapouge, K., Remans, K., Krebs, A. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-06
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that chromatin accessibility and transcription factor binding co-occur across entire CRLs in the Drosophila genome, and suggest that coordinated enhancer activation is a widespread mechanism regulating gene expression.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In metazoans, gene expression is typically regulated by a cis-regulatory landscape (CRL) composed of a promoter and multiple enhancers. How these cis-regulatory elements (CREs) coordinate their function across large genomic distances remains unclear. For example, is the simultaneous activation of multiple enhancers required to promote transcription? Here, we combined single-molecule footprinting with long-read sequencing to quantify how often chromatin accessibility and transcription factor binding co-occur across entire CRLs in the Drosophila genome. Analysis of thousands of individual DNA molecules at each locus revealed that CREs form a specific network with shared single-molecule chromatin accessibility profiles. Co-accessibility is not limited to adjacent CREs, and is frequently observed between CREs brought into proximity by chromatin looping. Co-accessible CREs exhibit strong coordination in their cell type-specific accessibility, linking enhancer activity with transcription activation. Our data uncover dependencies between CREs genome-wide, and suggest that coordinated enhancer activation is a widespread mechanism regulating gene expression.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.05.668660v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PreMode predicts mode-of-action of missense variants by deep graph representation learning of protein sequence and structural context.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Guojie Zhong, Yige Zhao, Demi Zhuang, Wendy K Chung, Yufeng Shen</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We develop a new method, PreMode, to perform gene-specific mode-of-action predictions of missense variants using SE(3)-equivariant graph neural networks on protein sequences and structures.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Accurate prediction of the functional impact of missense variants is important for disease gene discovery, clinical genetic diagnostics, therapeutic strategies, and protein engineering. Previous efforts have focused on predicting a binary pathogenicity classification, but the functional impact of missense variants is multi-dimensional. Pathogenic missense variants in the same gene may act through different modes of action (i.e., gain/loss-of-function) by affecting different aspects of protein function. They may result in distinct clinical conditions that require different treatments. We develop a new method, PreMode, to perform gene-specific mode-of-action predictions. PreMode models effects of coding sequence variants using SE(3)-equivariant graph neural networks on protein sequences and structures. Using the largest-to-date set of missense variants with known modes of action, we show that PreMode reaches state-of-the-art performance in multiple types of mode-of-action predictions by efficient transfer-learning. Additionally, PreMode's prediction of G/LoF variants in a kinase is consistent with inactive-active conformation transition energy changes. Finally, we show that PreMode enables efficient study design of deep mutational scans and can be expanded to fitness optimization of non-human proteins with active learning.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40764308/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">A computational framework for inferring species dynamics and interactions with applications in microbiota ecology.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Yuanwei Xu, Georgios V Gkoutos</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present MBPert, a generic computational framework for inferring species interactions and predicting dynamics in time-evolving ecosystems from perturbation and time-series data.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> We present MBPert, a generic computational framework for inferring species interactions and predicting dynamics in time-evolving ecosystems from perturbation and time-series data. In this work, we contextualize the framework in microbial ecosystem modeling by coupling a modified generalized Lotka-Volterra formulation with machine learning optimization. Unlike traditional methods that rely on gradient matching, MBPert leverages numerical solutions of differential equations and iterative parameter estimation to robustly capture microbial dynamics. The framework is assessed within the context of two experimental scenarios: (i) paired before-and-after measurements under targeted perturbations, and (ii) longitudinal time-series data with time-dependent perturbations. Extensive simulation studies, benchmarking on standardized MTIST datasets, and application to Clostridium difficile infection in mice and repeated antibiotic perturbations of human gut micribiota, demonstrate that MBPert accurately recapitulates species interactions and predicts system dynamics. Our results highlight MBPert as a powerful and flexible tool for mechanistic insight into microbiota ecology, with broad potential applicability to other complex dynamical systems.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40764303/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Accurate detection and quantification of single-base m6A RNA modification using nanopore signals with multi-view deep learning</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhang, J., Qiao, J., Zhao, Z., Yin, C., Jin, J., Wang, D., Gao, W., Wei, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present MultiNano, a multi-view deep learning framework that converts variable-length raw signals into image-like feature representations, effectively resolving the length inconsistency problem.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> N6-methyladenosine (m6A) is a crucial epitranscriptomic mark. While Nanopore Direct RNA Sequencing (DRS) enables transcriptome-wide detection, most existing methods neglect or lack the capacity to effectively process the intrinsically variable-length raw signals generated by DRS reads. Here, we present MultiNano, a multi-view deep learning framework that converts variable-length raw signals into image-like feature representations, effectively resolving the length inconsistency problem. By integrating raw signal and basecalling features, MultiNano enables accurate and comprehensive transcriptome-wide detection of m6A modifications. Our model achieved state-of-the-art (SOTA) performance in various tasks, including site-level prediction, read-level prediction, cross-species transfer learning, and modification rate estimation. Furthermore, the false positive control strategy implemented in MultiNano significantly enhances the model's robustness and predictive accuracy, offering a powerful alternative to traditional thresholding-based filtering algorithms. Collectively, our approach provides novel insights for the absolute quantification and single-base resolution of RNA modifications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.04.668591v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Extensive endemic transmission of multidrug resistant Mycobacterium tuberculosis in Bhutan: A retrospective genomic-epidemiological study</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dorji, T., Tshering, K., Adhikari, L., Jamtsho, T., Bhujel, P., Lhaden, P., Wangchuk, S., Wirth, W., Horan, K., Denholm, J. T., Sherry, N. L., Sait, M., Stinear, T. P., Howden, B. P., Andersson, P.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A large, endemic cluster of multidrug-resistant tuberculosis cases in Bhutan, which is due to recent local transmission of cases resulting in a large, single endemic cluster.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Despite decreasing overall tuberculosis notifications, the proportion of multidrug-resistant tuberculosis (MDR-TB) cases are increasing in Bhutan. While most MDR-TB cases are diagnosed among patients in the bordering districts and the capital, current diagnostic tests are limited in their ability to differentiate between the recurrent introductions and local transmission. For the first time, we conducted a retrospective genomic-epidemiological study to provide insights into the population structure, genotypic resistance patterns, and explore recent transmission of drug-resistant TB in Bhutan. Whole genome sequencing was performed on randomly selected drug-resistant and drug-sensitive TB isolates from Bhutan, collected between 2018-2022 at Microbiological Diagnostic Unit Public Health Laboratory in Melbourne, Australia. We investigated drug resistance mutations, and genomic clustering of cases using different single nucleotide polymorphisms (SNPs) thresholds. Of the 203 sequences that passed the quality control, 126 (62.1%) were MDR-TB and 15 (7.4%) were isoniazid-resistant TB. There were four different circulating lineages, with most sequences belonging to lineage 2 (86.2%). Using a SNP-threshold of [≤]12 SNPs, 71% of sequences formed 12 genomic clusters. Surprisingly, the largest cluster included of 88% of all MDR-TB sequences and spanned the entire study period. These cases were highly clonal (mean pairwise SNP-distance of 10, range 0-25). Phylogenetic analysis with publicly available international sequence data showed that this MDR-TB cluster formed a distinct clade. The major burden of MDR-TB in Bhutan appears to be due to recent local transmission of cases resulting in a large, single endemic cluster. This information will be critical for TB control program in Bhutan to tackle the major burden of MDR-TB through enhanced contact tracing of this MDR-TB clade. Additionally, this genomic data will be valuable to regional neighbours to monitor for dissemination of the strain. This study highlights the significant value of investing in TB genomics in resource limited settings globally to gain actionable insights into transmission dynamics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.05.668595v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Deep Profiling of EV Long RNAs Reveals Biofluid-Specific Transcriptomes and Splicing Landscapes</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chakrabortty, S. K., Xing, S., Nguyen, S., George, A., Miller, D. L., Babcock, K., Manning, K., Cole, T. J., Mitsock, E., Ray, C. J., Gowrisankar, S., Skog, J. K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a targeted RNA capture method that enriches long non-coding RNA in EVs from plasma and demonstrate high-sensitivity, noninvasive biomarker profiling across tissues and diseases.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> RNA profiling of extracellular vesicles (EVs) from human biofluids has historically been limited to small RNA species, with long RNAs, such as mRNA exons and long non-coding RNAs, remaining largely underexplored. Moreover, the dominance of hematopoietic-derived EVs in complex fluids like plasma has posed significant challenges for detecting low-abundance, tissue-specific transcripts. Here, we establish foundational transcriptomic maps of long RNAs in EVs from plasma, urine, and cerebrospinal fluid (CSF) using ultra-deep whole transcriptome sequencing (WTS), revealing both fluid-specific and shared expression and splicing signatures. We then introduce a targeted RNA capture method that enriches for all protein-coding and long non-coding transcripts, dramatically enhancing sensitivity for gene and splice variant detection. Applying this approach to brain-specific transcripts, we achieve >85-fold enrichment of target gene expression and, on average, 3.1-fold increase in detected splice junctions per gene compared to untargeted WTS. As a proof of concept, we apply this brain-targeted RNA panel to EVs from plasma in a Parkinsons disease cohort of 40 plasma samples and compare its performance to exome sequencing as well as untargeted WTS. This work advances EV transcriptomics into the long RNA domain and establishes a framework for high-sensitivity, noninvasive biomarker profiling across tissues and diseases.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.05.668716v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Complete genomes reveal the full extent of Mycobacterium tuberculosis complex diversity across evolutionary scales</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Garcia-Marin, A. M., Torres-Puente, M., Martinez-Priego, L., De Marco, G., Moreno-Molina, M., Hunt, M., Iqbal, Z., Gil-Brusola, A., Valencia Region TB Working Group,, Lopez, M. G., Gonzalez-Candelas, F., Alonso-del-Real, J., Comas, I.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Long-read sequencing of Mycobacterium tuberculosis complex (MTBC) reveals deep insights into its evolution across timescales, including hyperconserved PE/PPE epitopes and a significant increase in within-host diversity.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Advances in short-read sequencing have enhanced our understanding of Mycobacterium tuberculosis complex (MTBC), but fail to capture its complete genomic diversity. We applied long-read sequencing to 216 isolates from the Valencia Region (Spain) and generated high-quality, complete genomes, revealing detailed insights into MTBC evolution across timescales. Complete genome comparisons increased the estimated evolutionary rate by 1.5-fold, resulting in a median of 312 (-1 to 792) additional SNPs per pairwise comparison. Multiple diversity hotspots were identified, mostly in the pe/ppe genes and driven by gene conversion. However, most PE/PPE epitopes were hyperconserved, with notable exceptions involving vaccine candidates. Incorporating previously undetected SNPs and indels improved resolution in transmission analyses. Furthermore, patient-specific reference mapping validates only 5-10% of within-host diversity detected by standard pipelines, indicating substantial overestimation in previous studies. These findings expand our view of MTBC diversity and have important implications for understanding host-pathogen interactions, epidemiology, and transmission dynamics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.05.667821v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Integrative genomic study of mutation dynamics and Evolutionary trends in SARS-CoV-2 omicron BA.3</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kambarami, M. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that saltatory events play a critical role in shaping the genetic landscape of SARS CoV 2 BA.3, with important implications for viral infectivity, immune escape, and the design of next generation vaccines.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> This study investigates the evolutionary dynamics of the SARS CoV 2 BA.3 lineage with an emphasis on saltation-driven adaptation. Using 81 high coverage BA.3 genome sequences and 1,011 complete patient status records obtained from GISAID, we conducted a comprehensive analysis that integrated temporal, mutational, phylogenetic, and selection pressure assessments. Temporal analysis of patient records revealed that BA.3 sequences were predominantly collected between November 2021 and March 2022, with highly mutated variants emerging in the last quarter of 2024. Pairwise alignment of the spike gene demonstrated near identical sequences among recent isolates from Gauteng Province and subtle yet significant differences in a KwaZulu Natal isolate when compared to the Wuhan Hu 1 reference strain. Domain specific mutation mapping showed that mutations were concentrated in key functional regions of the spike protein, particularly within the receptor binding domain and its binding motif. Phylogenetic reconstruction and pervasive selection analysis further revealed that recent BA.3 variants form a divergent clade characterized by extensive adaptive mutations. These findings indicate that saltatory events play a critical role in shaping the genetic landscape of BA.3, with important implications for viral infectivity, immune escape, and the design of next generation vaccines.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.03.668320v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Protein language models reveal evolutionary constraints on synonymous codon choice</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sakharova, H., Lareau, L. F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We show that information about codon constraints can be extracted from protein sequence alone, combining implicit information about position and protein structure to learn subtle but generalizable constraints on codon choice in yeast.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Evolution has shaped the genetic code, with subtle pressures leading to preferences for some synonymous codons over others. Codons are translated at different speeds by the ribosome, imposing constraints on codon choice related to the process of translation. The structure and function of a protein may impose pressure to translate the associated mRNA at a particular speed in order to enable proper protein production, but the molecular basis and scope of these evolutionary constraints have remained elusive. Here, we show that information about codon constraints can be extracted from protein sequence alone. We leverage a protein language model to predict codon choice from amino acid sequence, combining implicit information about position and protein structure to learn subtle but generalizable constraints on codon choice in yeast. In parallel, we conduct a genome-wide screen of thousands of synonymous codon substitutions in endogenous loci in yeast, reliably identifying a small set of several hundred synonymous variants that increase or decrease fitness while showing that most positions have no measurable effect on growth. Our results suggest that cotranslational localization and translational accuracy, more than cotranslational protein folding, are major drivers of selective pressure on codon choice in eukaryotes. By considering both the small but wide-reaching effects of codon choice that can be learned from evolution and the strong but highly specific effects determined via experiment, we expose unappreciated biological constraints on codon choice.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.05.668603v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Hi-Cformer enables multi-scale chromatin contact map modeling for single-cell Hi-C data analysis</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wu, X., Chen, X., Jiang, R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose Hi-Cformer, a transformer-based method that simultaneously models multi-scale chromatin contact maps and incorporates a specially designed attention mechanism to capture the dependencies between chromatin interactions across genomic regions and scales, enabling the integration of both global and fine-grained chromatin interaction features.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell Hi-C captures the three-dimensional organization of chromatin in individual cells and provides insights into fundamental genomic processes such as gene regulation and transcription. While analyses of bulk Hi-C data have revealed multi-scale chromatin structures like A/B compartments and topologically associating domains, single-cell Hi-C data remain challenging to analyze due to sparsity and uneven distribution of chromatin contacts across genomic distances. These characteristics lead to strong signals near the diagonal and complex multi-scale local patterns in single-cell contact maps. Here, we propose Hi-Cformer, a transformer-based method that simultaneously models multi-scale blocks of chromatin contact maps and incorporates a specially designed attention mechanism to capture the dependencies between chromatin interactions across genomic regions and scales, enabling the integration of both global and fine-grained chromatin interaction features. Building on this architecture, Hi-Cformer robustly derives low-dimensional representations of cells from single-cell Hi-C data, achieving clearer separation of cell types compared to existing methods. Hi-Cformer can also accurately impute chromatin interaction signals associated with cellular heterogeneity, including 3D genome features such as topologically associating domain-like boundaries and A/B compartments. Furthermore, by leveraging its learned embeddings, Hi-Cformer can be extended to cell type annotation, achieving high accuracy and robustness across both intra- and inter-dataset scenarios.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.04.668453v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SingleRust: A High-Performance Toolkit for Single-Cell Data Analysis at Scale</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Diks, I. F., Flotho, M., Keller, A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> Single-cell RNA sequencing studies increasingly generate datasets exceeding 10 million cells, surpassing the memory capacity of standard analytical tools on typical institutional infrastructure. We introduce SingleRust, a computational framework that addresses these constraints through systematic algorithmic optimizations and systems-level design.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Single-cell RNA sequencing studies increasingly generate datasets exceeding 10 million cells, surpassing the memory capacity of standard analytical tools on typical institutional infrastructure. Here we introduce SingleRust, a computational framework that addresses these constraints through systematic algorithmic optimizations and systems-level design. Key improvements include sparse masked principal component analysis that reduces memory footprint while preserving biological signal, lock-free parallel implementations for differential expression testing, and adaptive k-nearest neighbor algorithms that automatically select optimal data structures based on dataset size. These optimizations achieve 2.3-24.6-fold performance improvements and 1.3-2.7-fold memory reduction compared to Scanpy, enabling routine analysis of 20 million cells on our representative test system with 512 GB RAM. Comprehensive validation confirms numerical equivalence with established methods while maintaining biological interpretation fidelity. SingleRust maintains full compatibility with the AnnData ecosystem while providing researchers immediate access to population-scale analyses on existing infrastructure, addressing a critical bottleneck in single-cell genomics workflows.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.04.668429v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Python-based automation of INDIGO webserver using Selenium: A high throughput analysis of Sanger sequence data to detect allelic variations created by CRISPR/Cas9-mediated genome editing of crop plants</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Suresh, V., Girish, C., Tavva, V. S. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We developed a Python-based automation pipeline using Selenium with integrated highlighting of protospacer adjacent motif (PAM) regions in the resulting HTML reports.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> CRISPR-Cas9 has revolutionized plant genome editing by enabling precise introduction of insertion/deletion (indel) mutations, critical for functional genomics and crop improvement studies. Sanger sequencing, combined with bioinformatics tools like the INDIGO webserver from Gear Genomics, is essential for validating these mutations. However, manual analysis of large numbers of Sanger sequencing (.ab1) files is labor-intensive, particularly when analyzing multiple guide RNA (gRNA) target regions. We developed a Python-based automation pipeline using Selenium with integrated highlighting of protospacer adjacent motif (PAM) regions in the resulting HTML reports. This pipeline enhances scalability of Sanger sequence data analysis and improves result interpretability by automating PAM region identification and supporting multiple gRNA regions. This tool significantly accelerates CRISPR-Cas9-mediated mutation analysis, offering a high throughput, reproducible solution for genome editing research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.04.668409v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Metabolically robust autoluminescent reporters</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Morozov, V. V., Balakireva, A. V., Gao, F., Fleiss, A., Mishina, N. M., Perfilov, M. M., Mitiouchkina, T. Y., Gorbachev, D. A., Wood, K. V., Yampolsky, I. V., Sarkisyan, K. S., Mishin, A. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We present a novel approach for self-sustained autoluminescence in non-luminous organisms that dramatically reduces signal variation, enabling quantitative physiological imaging amid changing metabolic activity.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Self-sustained luminescence can be engineered in non-luminous organisms by integrating luciferin biosynthesis pathways with the host's metabolism. When conditionally expressed, such pathways enable non-invasive monitoring of virtually any transcriptional event, providing a wide dynamic range and high resolution. However, because light emission depends on substrate availability within the cell, autoluminescence reporters are inherently non-quantitative: signal intensity reflects both gene expression and cellular metabolism. Here, we present an approach that disentangles the contributions of gene expression and metabolism, rendering autoluminescence signals robust against metabolic perturbations. By co-expressing an engineered reference luciferase variant that uses the same substrate but emits light at a different wavelength, and monitoring the resulting colour ratio, we dramatically reduce signal variation, enabling quantitative physiological imaging amid changing metabolic activity. Our strategy is applicable across known autoluminescence pathways from bacteria and fungi.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.04.668150v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">ANTIBACTERIAL ACTIVITY OF CHINESE CABBAGE (Brassica rapa subsp. chinensis) AGAINST SELECTED FOODBORNE PATHOGENS</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Gabucan, V. J. M. G., Revilla, C. K. C., Quinoviva, S. R., Armada, D. M. S., Manluyang, A. L. G. S., Brieta, F. M. B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #97beca; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics, plant biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A study aimed to evaluate the antibacterial activity of ethanolic extract of Chinese cabbage (Brassica rapa subsp. chinensis) against selected foodborne pathogens and to identify the functional groups present in the extract using Fourier-Transform Infrared (FTIR) spectroscopy.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Foodborne pathogens pose significant global health risks, contributing to foodborne illnesses and antimicrobial resistance. The increasing demand for plant-derived alternatives to synthetic antimicrobials highlights the relevance of investigating locally available vegetables with potential antibacterial properties. This study aimed to evaluate the antibacterial activity of the ethanolic extract of Chinese cabbage (Brassica rapa subsp. chinensis) against selected foodborne pathogens and to identify the functional groups present in the extract using Fourier-Transform Infrared (FTIR) spectroscopy. Fresh Chinese cabbage leaves were dried, powdered, and extracted using 80% ethanol. The antibacterial activity was assessed through the Kirby-Bauer disk diffusion method against E. coli, S. enteritidis, and S. aureus, with ciprofloxacin and vancomycin as positive controls. FTIR analysis revealed the presence of alcohol, alkene, amine, and alkyl functional groups, indicating the presence of potentially bioactive phytochemicals. In the antibacterial assay, the extract produced mean zones of inhibition of 24.18{+/-}0.305 mm for S. enteritidis, 9.49{+/-}2.726 mm for S. aureus, and 7.78{+/-}2.019 mm for E. coli. Among the tested pathogens, S. enteritidis exhibited the highest sensitivity to the extract. However, all inhibition zones were significantly smaller and statistically different than those of the positive control antibiotics, indicating moderate antibacterial activity. The findings support the antibacterial potential of Brassica rapa subsp. chinensis, particularly against S. enteritidis. The presence of functional groups associated with antimicrobial properties suggests its suitability as a natural antibacterial agent. Further phytochemical analysis and formulation studies are recommended to enhance its efficacy and explore its potential applications in food safety and alternative medicine.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.04.668494v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Bioluminescent sentinel plants enable autonomous diagnostics of viral infections</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Calvache, C., Rodriguez-Rodriguez, M., Vazquez-Vilriales, V., Garcia-Perez, E., Fleiss, A., Ezzeddin-Ayoub, M., Pasin, F., Daros, J.-A., Sarkisyan, K. S., Orzaez, D., Vazquez-Vilar, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> A bioluminescent platform for autonomous viral sensing based on the fungal biol Luminescence pathway (FBP).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Plants engineered with synthetic genetic programs can transform how we monitor and manage the extension of crop pests and diseases. Here, we establish a bioluminescent platform in Nicotiana benthamiana for autonomous viral sensing based on the fungal bioluminescence pathway (FBP). We first demonstrate that recombinant viruses can deliver missing pathway components, enabling spatially resolved tracking of infection dynamics. Leveraging this starting point, we developed a dual-output sentinel circuit that uses a protease-responsive Bioluminescence Resonance Energy Transfer (BRET) module to report infection through a virus-triggered spectral shift in luminescence. In the absence of infection, plants emit a stable yellow glow indicating system integrity. Upon infection with potyviruses, cleavage of the BRET fusion by the virus-encoded NIa-Pro protease activates a distinct colour change detectable with low-cost imaging. This modular design is compatible with other pathogens carrying specific proteases and supports future multiplexing strategies. Our results highlight the potential of synthetic sentinel gene circuits as autonomous biosensors for precision crop protection.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.05.668616v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Simulating a for-loop in the human genome: Design and evaluation of recombinase genetic programs that count to three.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Chao, G., Appleton, E., Gutierrez, C. S., Evgeniou, L., Daifuku, T., Wannier, T. M., Mintzer, E., Church, G. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We develop a system for cellular memory-driven behaviors by simulating a "for-loop" that counts to three using recombinase STepwise gene Expression Programs (STEPs).</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Pluripotent cells specialize into numerous cell types by receiving external signals, making fate decisions, and executing differentiation functions - a paradigm similar to computer algorithms. While advances in biosensor design have enabled cells to respond to diverse stimuli, the ability to maintain a synthetic memory of the cell's experiences that then informs its behaviors remains elusive. Here, we developed a system for cellular memory-driven behaviors by simulating a "for-loop" that counts to three using recombinase STepwise gene Expression Programs (STEPs). STEPs were genomically integrated into human cells, genotyped through targeted nanopore sequencing, and evaluated for function through changes in fluorescent reporter expression. While all STEPs were capable of heritable memory and sequential gene expression, the STEP design using tyrosine recombinases for successive excisions (TRex) significantly outperformed the others tested. We then used live cell imaging to track TRex cells as they incremented from count zero to three and observed the successive emergence of four cell states from an initially homogeneous population. The STEPs framework provides biological memory and conditional expression capabilities that, coupled with input mechanisms such as biosensors, can start to approach a programming language for biology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.04.668540v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Bistability in Gene Regulation: Simulating Positive Feedback and Toggle Circuits Using Python and Hill Functions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Krishna Y K, Y.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce a simple and reproducible Python framework for modeling bistability in genetic feedback systems by means of ordinary differential equations (ODEs) driven by Hill functions.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cellular decision-making relies heavily on bistable gene regulatory networks, which allow systems to respond to internal or external stimuli by switching between several stable expression states. Processes like cell differentiation, epigenetic memory, and the creation of artificial biological switches all depend on these dynamics. In this work, we introduce a simple and reproducible Python framework for modeling bistability in genetic feedback systems by means of ordinary differential equations (ODEs) driven by Hill functions. We employ two fundamental motifs, both of which are recognized for their ability to generate bistable behavior: a two-gene mutual inhibition toggle switch and a single-gene positive feedback loop. We investigate the effects of different Hill coefficients, production rates, and initial expression levels on system dynamics by numerical integration using SciPy. Our simulations show phase-plane convergence to several attractors, map expression outcomes over a grid of beginning circumstances, and illustrate the onset of bistability above a key Hill threshold. The delicate reliance of final states on cooperativity and beginning values is further demonstrated by heatmaps and bifurcation-like graphs. For accessibility, all code is hosted at Google Colab and is written in open-source Python. This study promotes research and teaching in synthetic biology, systems biology, and computational modeling while providing a simple yet effective computational framework for investigating the fundamentals of gene circuit bistability.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.05.668746v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">In silico pharmacological analysis of Tinospora cordifolia compounds targeting African swine fever virus B175L</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ranathunga, L., Karunarathne, K., Poorni, S., Jayampathi, N., Dhananjaya, H., Hasintha, A., Hulugalla, W. M. M. P., Anupama, N. M. T., Nethmini, N., Iqbal, N., Jayawardana, B.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We propose Tinospora cordifolia-derived Benzaldehyde, 5-bromo-2-hydroxy-, (5-trifluoromethyl-2pyridyl)hydrazone as a promising B175L inhibitor, advancing exploration towards effective antiviral solutions</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> African swine fever virus (ASFV) is a highly lethal DNA virus that suppresses the hosts immune response by establishing infection. B175L, one of its key immune evasion proteins, directly inhibits STING-mediated type I interferon (IFN-I) signaling, preventing antiviral defense activation. Thus, targeting B175L could be a promising strategy for antiviral drug development as effective ASFV inhibitors remain unidentified. In this study, we investigated the potential of the Tinospora cordifolia plants bioactive compounds to disrupt B175Ls function, restoring immune signaling. Gas chromatography-mass spectrometry (GC-MS) analysis of T. cordifolia stems methanol extract identified 124 compounds, of which 52 met SwissADME and DataWarrior criteria for drug-likeness and safety. We generated a highly accurate 3D B175L model with strong confidence scores for the structure accuracy, utilizing AlphaFold3. Virtual screening was performed using PyRx 0.8, and the top 4 ligands with binding affinities exceeding -6 kcal/mol advanced to CB-Dock2 and UCSF Chimera. Among them, Benzaldehyde, 5-bromo-2-hydroxy-, (5-trifluoromethyl-2pyridyl)hydrazone exhibited the highest binding affinity (-8.2 kcal/mol), confirming its strong interactions at the active site of B175L. Molecular dynamics (MD) simulations demonstrated the compounds stability, with root mean square deviation and fluctuation (RMSD, RMSF) indicating minimal conformational changes (<2 [A]). The compound maintained stable hydrogen bonds and hydrophobic interactions, reinforcing structural robustness. Taken together, these results clearly emphasize T. cordifolia-derived Benzaldehyde, 5-bromo-2-hydroxy-, (5-trifluoromethyl-2-pyridyl)hydrazone as a promising B175L inhibitor, advancing exploration towards effective antiviral solutions for ASFV.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.03.668349v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">scMomer: A modality-aware pretraining framework for single-cell multi-omics modeling under missing modality conditions</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Liu, Y., Zou, Q., Su, R., Wei, L.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-05
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify; margin-bottom: 10px;"><span style="font-weight: bold; font-size: 16px;">tl;dr:</span> We introduce scMomer, a modality-aware pretraining framework designed for multi-modal representation learning under missing modality conditions and help capture cellular heterogeneity.</p><p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Foundation models offer new opportunities to capture cellular behavior from large-scale single-cell data. However, their development has been greatly constrained due to the limited availability of multi-omics profiles. Consequently, most models are designed for a single modality (e.g. scRNA-seq, or scATAC-seq, etc.), restricting their ability to capture the diversity of heterogeneous biological systems. Here, we introduce scMomer, a modality-aware pretraining framework designed for multi-modal representation learning under missing modality conditions. scMomer adopts a three-stage pretraining strategy that learns unimodal cell representations, models joint representations from multi-omics data, and distills multi-modal knowledge to enable multi-omics-like representations from unimodal input. Its modality-specific architecture and three-stage pretraining strategy enable effective learning under missing modality conditions and help capture cellular heterogeneity. Through extensive experiments, scMomer generates biologically meaningful embeddings and outperforms state-of-the-art unimodal approaches across diverse gene-level and cell-level downstream tasks, including cross-modality translation, gene function prediction, cell annotation, drug response prediction, and perturbation prediction. Overall, these results demonstrate that scMomer serves as a robust, generalizable, and scalable foundation for single-cell multi-modal analysis under missing modality conditions.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.04.668374v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You’re receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

