<html>
<head>
    <meta charset="UTF-8">
    <title>paper-trackr newsletter</title>
</head>
<body style="font-family: Georgia, serif; background-color: #f4f4f4; padding: 20px; font-size: 16px;">
    <div style="background-color: white; padding: 30px; max-width: 700px; margin: auto; box-shadow: 0 2px 8px rgba(0,0,0,0.1);">
        <div style="display: flex; justify-content: space-between; align-items: center; border-bottom: 2px solid #000000; margin-bottom: 10px;">
  <h1 style="font-size: 28px; margin: 0;">paper-trackr newsletter</h1>
  <a href="https://felipevzps.github.io/" style="font-size: 16px; text-decoration: none; color: #1a0dab;">‚Üê back to my website</a>
</div>
        <p style="color: gray; font-size: 16px;"><strong>Tuesday, 02 September 2025</strong></p>
        <p style="font-size: 16px;">Hello science reader,</p>
        <p style="font-size: 16px;">Welcome to your daily dose of research. Dive into the latest discoveries from the world of science!<div style="border-left: 4px solid #ccc; padding-left: 12px; margin-top: 20px; color: gray; font-size: 15px; font-style: italic;">
    This newsletter was built with 
    <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a> and is tailored to my interests.<br>
    Customize your own filters and receive personalized updates.
</div></p>
        <hr style="margin: 20px 0;">
        
            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Microbiome data integration via shared dictionary learning.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Bo Yuan, Shulei Wang</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #84d69a; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Data integration is a powerful tool for facilitating a comprehensive and generalizable understanding of microbial communities and their association with outcomes of interest. However, integrating data sets from different studies remains a challenging problem because of severe batch effects, unobserved confounding variables, and high heterogeneity across data sets. We propose a new data integration method called MetaDICT, which initially estimates the batch effects by weighting methods in causal inference literature and then refines the estimation via novel shared dictionary learning. Compared with existing methods, MetaDICT can better avoid the overcorrection of batch effects and preserve biological variation when there exist unobserved confounding variables, data sets are highly heterogeneous across studies, or the batch is completely confounded with some covariates. Furthermore, MetaDICT can generate comparable embedding at both taxa and sample levels that can be used to unravel the hidden structure of the integrated data and improve the integrative analysis. Applications to synthetic and real microbiome data sets demonstrate the robustness and effectiveness of MetaDICT in integrative analysis. Using MetaDICT, we characterize microbial interaction, identify generalizable microbial signatures, and enhance the accuracy of outcome prediction in two real integrative studies, including an integrative analysis of colorectal cancer metagenomics studies and a meta-analysis of immunotherapy microbiome studies.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40890119/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">MRDagent: Iterative and Adaptive Parameter Optimisation for stable ctDNA-Based MRD Detection in Heterogeneous Samples.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Tianci Wang, Xin Lai, Shenjie Wang, Yuqian Liu, Xiaoyan Zhu, Jiayin Wang</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #f4dbac; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, workflow, automated</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Minimal residual disease (MRD) as critical biomarker for cancer prognosis and management, plays a crucial role in improving patient outcomes. However, detecting minimal residual disease (MRD) via next-generation sequencing (NGS)-based circulating tumor DNA (ctDNA) variant calling remains unstable due to the extremely low variant allele frequency (VAF) and significant inter- and intra-sample heterogeneity. Although parameter optimization can theoretically enhance variants detection performance, achieving stable MRD detection remains challenging due to three key factors: (i) the necessity for individualized parameter tuning across numerous heterogeneous genomic intervals within each sample, (ii) the tightly interdependent parameter requirements across different stages of variant detection workflows, and (iii) the limitations of current automated parameter optimization methods.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40889285/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Collateral mutagenesis funnels multiple sources of DNA damage into a ubiquitous mutational signature</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Spisak, N., de Manuel, M., Przeworski, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Mutations reflect the net effects of myriad types of damage, replication errors, and repair mechanisms, and thus are expected to differ across cell types with distinct exposures to mutagens, division rates, and cellular programs. Yet when mutations in humans are decomposed into a set of "signatures" one single base substitution signature, SBS5, is present across cell types and tissues, and predominates in post-mitotic neurons as well as male and female germlines (Alexandrov et al. 2020, Lodato et al. 2018, Spisak et al. 2024). The etiology of SBS5 is unknown. By modeling the processes by which mutations arise, we infer that SBS5 is the footprint of errors in DNA synthesis triggered by distinct types of DNA damage. Supporting this hypothesis, we find that SBS5 rates increase with signatures of endogenous and exogenous DNA damage in cancerous and non-cancerous cells and co-vary with repair rates along the genome as expected from model predictions. These analyses indicate that SBS5 captures the output of a "funnel", through which multiple sources of damage result in a similar mutation spectrum. As we further show, SBS5 mutations arise not only from translesion synthesis but also from DNA repair, suggesting that the signature reflects the occasional, shared use of a polymerase.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.28.672844v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">An RNA polymerase III tissue and tumor atlas uncovers context-specific activities linked to 3D epigenome regulatory mechanisms</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lizarazo, S., Zhou, S., Cheng, R., K C, R., Shen, Y., Liu, Q., Van Bortle, K.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> RNA polymerase III (Pol III) produces a plethora of small noncoding RNA species involved in diverse cellular processes, from transcription regulation and splicing to RNA stability, translation, and proteostasis. Though Pol III activity is broadly coupled with cellular demands for protein synthesis and growth, a more precise understanding of gene-level dynamics and context-specific expression patterns remains missing, in part due to challenges related to sequencing and mapping Pol III-derived small ncRNAs. Here, we establish a predictive multi-tissue map of human Pol III activity across 19 tissues and 23 primary cancer subtypes by comprehensively profiling the chromatin accessibility of canonical Pol III-transcribed gene classes. Our framework relies on the unique relationship between gene accessibility and Pol III transcription, inferring activity through uniform binary classification of ATAC-seq enrichment at Pol III-transcribed genes. By characterizing multi-context gene uniformity, we provide a definition of the core Pol III transcriptome, broadly active across specialized tissues, and catalog genes with varied levels of context specificity. Our genomic Pol III atlas uncovers variable levels of activity across tissues, including sharp contraction of the Pol III transcriptome in heart and brain tissues and frequent expansion across diverse cancers. We show that both tissue- and tumor-specific genes are significantly enriched within lamina-associated domains (LADs), and that aberrant expression of nuclear lamin proteins is sufficient to induce Pol III-emergent patterns at tumor-specific genes. Together, these findings link Pol III dynamics to subnuclear compartmentalization and provide a resource for better understanding Pol III expansion and small RNA biogenesis in cancer.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.28.672650v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">SPAGHETTI leverages massive H\&E morphological models for phase contrast microscopy images with a generative deep learning approach</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Dong, Z. F., McIntosh, C., Schwartz, G. W.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Phase contrast microscopy (PCM) is a powerful cell imaging method, one of the few technologies to delineate and track cell structure in live cells without staining. Despite PCM's great potential and popularity in monitoring live-cell populations, there is a lack of algorithms to extract morphological information from these images due to the lack of large training datasets. To overcome this challenge and enable advanced, high throughput quantitative analysis of PCM images, we introduce SPAGHETTI: a lightweight image translator built on a modified cycle-consistent generative adversarial network. SPAGHETTI translates PCM images into those resembling hematoxylin and eosin (H&E) pathological images which, due to the pervasive and widespread use in clinical settings, are the basis for most large-scale deep learning models for quantitative analyses. We demonstrate that by first using SPAGHETTI to translate PCM images into H&E-like images, we could achieve significantly improved performance on cell segmentation through the use of tissue and H&E-specific cell segmentation models. We also show that by passing translated PCM images across several independent datasets into H&E feature extractor models, we improve the performance of cell-type annotation, experimental media classification, and cell viability prediction. Overall, SPAGHETTI enables many quantitative analyses of PCM that were previously impossible and acts as a valuable preprocessing step to help researchers gather novel information about cell states through the downstream quantitative analysis of morphological features.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.27.672636v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">shinyUMAP: an online tool for promoting understanding of single cell omics data visualization</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Misra, R., O'Leary, K., Chen, W., Zheng, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Visualization is widely used to help explore and interpret high dimensional single cell (sc) omics data, such as scRNA-seq expression data. In particular, uniform manifold approximation and projection (UMAP) has become nearly ubiquitous in scientific publications that apply single cell omics technologies. Some experts have expressed concerns that the global cell-cell relationship, especially the spatial distances among cell clusters in a dataset, may not be faithfully depicted in a 2-dimensional (2D) UMAP. To help users to better appreciate this issue with their own data, we created an online server for the community to upload their single cell data and interactively make UMAPs with different hyper-parameters to witness how the distribution of cell clusters changes. The server thus can help promote proper usages of UMAP, especially to avoid the common pitfalls in misinterpretation of inter-cluster relationships in single cell studies. Availability and Implementation: ShinyUMAP is freely available as an online Shiny server implemented in Python at https://scviewer.shinyapps.io/shinyUMAP/.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.27.672621v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Improved peptide search for identification of SUMO and sequence-based modifications, in MaxSBM</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Lennartsson, C., Kyriakidou, P., Nielsen, M. L., Olsen, J. V., Cox, J., Hendriks, I. A.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Post-translational modifications (PTMs), such as SUMOylation and ubiquitination, regulate key cellular processes by covalently attaching to lysine residues. While mass spectrometry allows site-specific identification of PTMs, most existing search engines are optimized for small, non-fragmenting modifications and struggle to detect large, fragmenting protein-based modifiers. We refer to these as Sequence-Based Modifiers (SBMs). To overcome this limitation, we developed an SBM-specific search strategy within MaxQuant that accounts for the fragmentation behavior of SBMs during peptide identification. Using publicly available datasets, we validated our approach for SUMO2/3. Our analysis identified distinct diagnostic features and characteristic mass shifts associated with SBM fragmentation, referred to in this study as d-ions (diagnostic ions) and p-ions. By leveraging these features, our method improved the identification of SUMOylated peptides from human cell lines by 13%, SUMOylation sites in mouse embryonic cells by 18%, and in mouse adipocytes by 25%. Our search method improved spectral annotation of SBMs by up to 9% increase in the median Andromeda score. Taken together, we highlight the potential of our SBM search to enhance the discovery of protein-based modifications.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.27.672604v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Anyone can be the best: Impact of diverse methodologies on the evaluation of structural variant callers</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Denti, L., Krannich, T., Vinar, T., Chikhi, R., Bonizzoni, P., Brejova, B., Hormozdiari, F.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Structural variants (SVs) are medium and large-scale genomic alterations that shape phenotypic diversity and disease risk. Numerous methods have been proposed for discovering SVs, however their benchmarking has been inconsistent across studies, often resulting in contradictory findings. One of the main sources of conflicting evaluation results is the lack of consistency in the SV callsets used as ground truth, ranging from curated callsets released by consortia to more recent approaches that construct callsets from high-quality telomere-to-telomere de novo haplotype assemblies. The discrepancies between benchmarks are further compounded by the choice of the reference genome (GRCh37, GRCh38, and T2T-CHM13), where using T2T-CHM13 reveals a different deletion/insertion profile, indicating reduced reference bias. We evaluated the performance of several state-of-the-art SV discovery methods from long-read whole-genome sequencing data and observed substantial variation in their performance and rankings, depending on the choice of ground truth, reference genome, and genomic regions used for evaluation. Counter-intuitively, the more complete reference genome T2T-CHM13 does not inherently solve the problem of SV benchmarking; instead it reveals the limitations of each detection method in complex genomic regions. The substantial variation in detection accuracy across different genomic regions calls for additional caution in downstream analyses and in drawing conclusions based on predicted SVs. These findings underscore the complexity of evaluating SV detection methods and highlight the need for careful consideration and, ideally, field-standard best practices when reporting performance metrics.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.28.672546v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Copy number variation in the genome of four South American camelid species</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rojas, C. T. M., Sanchez, D., Condori, E., Zapana, J., Ttito, A., Hachircana, L. F., Lopez, V., Rodriguez, H., Melo, M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Copy number variation (CNV), a type of structural genomic variant, has been widely studied in humans, plants, and livestock. CNVs may contribute to phenotypic variation and traits of economic importance. Evaluating CNVs in domestic populations and their wild ancestors can reveal genetic changes associated with phenotypic differences that have emerged during domestication. In this study, whole-genome data from sixteen alpacas, six llamas, one guanaco, and one vicuna were used to investigate CNVs. A total of 4,247 CNVs were identified through genome-wide analysis. Functional analysis of Gene Ontology (GO) groups indicated that CNVs in alpacas are mainly related to immune-related genes, olfactory receptor genes, and keratinization. Species-level differences were observed, with the vicuna showing the least variation and the alpaca the most. This study provides the first CNV map for camelids and contributes to expanding knowledge of CNV diversity across the four South American camelid species.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.28.672513v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Fusion-driven oncogenic programs shape the immune landscape in translocation renal cell carcinoma</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Konda, P., Weiss, C. N., Cui, Y., Matar, S., Wang, J., Nabil, Y. L., Deodhar, R., Li, J., Horst, J., Camp, S. Y., Sheshdeh, A. B., Hecht, J. L., Einstein, D. J., Rustagi, Y., Nag, A., Thorner, A. R., Zhang, C.-Z., Van Allen, E. M., Signoretti, S., Choueiri, T., Viswanathan, S. R.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Renal cell carcinomas comprise multiple molecularly distinct cancers but most are treated empirically with therapies designed for clear cell RCC (ccRCC), the most common subtype, due to incomplete understanding of subtype-specific biology. We analyzed single-cell transcriptomes and chromatin accessibility profiles from translocation renal cell carcinoma (tRCC), an aggressive RCC defined by oncogenic TFE3 gene fusions. Unexpectedly, despite arising from a proximal tubule cell of origin similar to ccRCC, tRCCs display markedly distinct oncogenic programs and an immunosuppressive tumor microenvironment. tRCCs exhibit six conserved tumor meta-programs, including epithelial-mesenchymal transition and proximal tubule identity programs whose balance is regulated by TFE3 fusion activity. The fusion-driven EMT program drives a suppressive TME marked by progenitor-exhausted CD8+ T cells, anti-inflammatory SPP1+ macrophages, and matrix-associated fibroblasts (mCAFs). Our findings highlight unique TFE3 fusion-driven biology in tRCC, explaining its reduced immunotherapy responsiveness relative to ccRCC, and suggesting strategies for targeting fusion-driven oncogenic programs and TME reprogramming.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.28.672849v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Context-Aware Synthetic Promoter Design Using Neural Networks Enables Rewiring of Eukaryotic Transcriptional Networks</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Kuhajda, L., Honzik, T., Svec, J., Georgiev, D.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #c59cf3; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, genomics</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Gene regulation through promoter engineering is a cornerstone of synthetic biology, enabling precise control over transcriptional networks. However, experimental approaches remain labor-intensive. While artificial neural networks (ANNs) have improved regulatory element prediction, tools for promoter-transcription factor binding site (TFBS) recombination are still lacking. We present an ANN framework for context-aware design of synthetic promoters in Saccharomyces cerevisiae. The model predicts optimal TFBS insertion sites and the extent of promoter rewriting needed for successful integration. Applying this, we screened 6,011 native yeast promoters for compatibility with the TetR TFBS, generating a ranked list of high-confidence promoter-TFBS pairs. Experimental validation showed that model-designed promoters achieved repression rates up to 98.4%, without prior experimental characterization or tuning. We further rewired the yeast transcriptional network by introducing glucose-dependent regulation of an essential gene via Mig1 TFBS insertion. These results establish a scalable, predictive method for engineering regulatory sequences and reprogramming transcriptional logic.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.27.672570v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Software System Design to Support Scale in Mammalian Cell Line Engineering.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> McClymont, D. W., McIlwraith, B., Green, A., Coulson, S., Scott, E.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Cell line engineering (CLE) is the process of gene editing living cell lines for various purposes such as protein production, fluorescent tagging, compound screening tools, reference standards or diagnostics. Manufacturing processes of the line were typically low throughput and we aimed to increase the capacity of the platform to meet market demand. Software is required to match any throughput and equally as important as any lab based manual or automated methods. To support this increased throughput, a Laboratory Information Management System (LIMS) was developed which contained a data model that represents the CLE process and tracks all projects at each stage. A connection was also made to our own internal ordering system to track projects and register engineering designs based on a markup language we developed called Genetic Engineering Markup Language (GEML). Additionally, we developed an ML based method for analysing clones using an automated imaging platform. The combined wet-lab and software approach allowed us to treble the capacity of the platform allowing us to reach the potential of engineering biology.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.09.01.673445v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Comprehensive analysis of metabolic reprogramming-related gene signaturesmetabolic reprogramming related gene signature for predicting ovarian cancer prognosis, immune landscape, and potential treatment options</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Han, R., Guo, E., Li, Z., Zhou, H., Liu, B., Wang, Y., Hou, X., Zheng, F., Xu, Y., Yu, J.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Ovarian cancer (OV) is the most lethal gynecologic malignancy. Metabolic reprogramming is a distinctive feature of cancer and is associated with tumorigenesis and progression. It could be a potential therapeutic target for cancer treatments and a biomarker for assessing cancer prognosis. In this study, we identified metabolic reprogramming-related differentially expressed genes (MRRDEGs) in OV through differential gene expression analysis and conducted a comprehensive characterization of these MRRDEGs. Based on the MRRDEGs, we constructed an effective prognostic risk model including five model genes for OV. The risk score was a valid independent prognostic factor that could more accurately predict the survival of OV patients. It could classify OV patients into distinct risk groups with significant differences in survival. We observed significant differences between risk groups in biological pathway activity, immune cell infiltration patterns, and immunotherapy responses. Specifically, the low-risk group demonstrated superior immunotherapy response compared to the high-risk group. These findings significantly advance our understanding of the relationship between metabolic reprogramming and OV pathogenesis, progression, prognosis, and immunotherapy response, laying a foundation for developing novel biomarkers and therapeutic targets in the future and providing an important reference for the formulation of precision medicine strategies.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.28.672868v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Statistical Methods for Binary Outcomes Adjusting for Outcome Dependent Sampling in Longitudinal Studies with Nonignorable Dropout</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sevick, C. J., WaWHinney, S., Anderson, P. L., Moore, C. M.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Longitudinal clinical trials and cohort studies often collect clinical data paired with stored biospecimens. An increasing focus of biomedical research is aimed at leveraging these existing specimens to address new research questions. When a hypothesis of interest proposes to utilize costly, limited or difficult to obtain samples, it may not be possible or desirable to assay all samples. In these situations informed sampling strategies (ISS) can be used to minimize costs and preserve biospecimens by providing a framework to select a subset of subjects that is more informative than a simple random sample. The samples from selected subjects can be assayed and the resulting data can be analyzed in concert with an analytical correction. Dropout is common in longitudinal studies but existing ISS methods do not address nonignorable dropout. Ignoring cases where poor outcomes may influence the propensity to dropout could bias study results. We propose an expansion of current ISS frameworks to account for nonignorable dropout. Mixture models, commonly used to adjust for dropout, are modified to accommodate analysis of data from ISS designs. Methods are available in the BUILD R package.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.30.673241v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">PlantCAD2: A Long-Context DNA Language Model for Cross-Species Functional Annotation in Angiosperms</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Zhai, J., Gokaslan, A., Hsu, S.-K., Chen, S.-P., Liu, Z.-Y., Marroquin, E., Czech, E., Cannon, B., Berthel, A., Romay, C., Pennell, M., Kuleshov, V., Buckler, E. S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #ded59b; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, synthetic biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Understanding how DNA sequence encodes biological function remains a fundamental challenge in biology. Flowering plants (angiosperms), the dominant terrestrial clade, exhibit maximal biochemical complexity, extraordinary species diversity (over 100,000 species), relatively recent origins (~160 million years), ~200-fold variation in genome size and relative compact coding regions compared with other eukaryotes. These features present both a unique challenge and opportunity for pre-training DNA language models to understand plant-specific evolutionary conservation, regulatory architectures and genomic functions. Here, we introduce PlantCAD2, a long-context, plant-specific DNA language model with single-nucleotide resolution, pre-trained on 65 angiosperm genomes, together with a series of cross-species public benchmarks for evaluation. Comprehensive zero-shot testing shows that PlantCAD2 (676 million parameters) efficiently captures evolutionary conservation, surpassing the 7-billion-parameter Evo2 model in 10 of 12 tasks. With parameter-efficient fine-tuning, PlantCAD2 also outperforms the 1-billion-parameter AgroNT across seven cross-species tasks. Moreover, its 8 kb context window substantially improves accessible chromatin prediction in large genomes such as maize (AUPRC increasing from 0.587 to 0.711), underscoring the importance of long-range context for modeling distal regulation. Together, these results establish PlantCAD2 as a powerful, efficient, and versatile foundation model for plant genomics, enabling accurate genome annotation across diverse species.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.27.672609v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Preparation and characterization of Fe3O4-spermine-PCL-chitosan-PEG-FA nanoparticles for DNA delivery into AGS cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mohajeri, S., Baezzat, N., Noruzpuor, M., Bourang, S., Yaghoubi, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Gene therapy, a novel treatment approach for diseases such as cancer and genetic disorders, involves the transfer of genes to host cells or tissues. In recent decades, methods such as viruses, bacteria, and mechanical techniques have been utilized for gene transfer. However, Owing to the significant side effects of traditional methods for cancer treatment, especially chemotherapy, nanoparticles have emerged as a preferred alternative, enhancing targeted gene delivery and reducing complications. This study utilized copolymers such as spermine-chitosan-polycaprolactone (PCL) (PCS) and iron oxide nanoparticles (Fe3O4) to protect DNA and facilitate oligonucleotide transfer. Additionally, spermine-PCL-chitosan-polyethylene glycol (PEG)-folic acid (SPCPF) micelles were synthesized to control DNA release and target tumors. SCPPF/Fe3O4/DNA micelles were then prepared, and their properties, including protection against plasma degradation, ability to release DNA, physicochemical properties, transfection efficiency, and cytotoxicity, were evaluated in vitro. The study also investigated the impact of two buffer pH values (6 and 7.4) on DNA release from micelles, revealing that a lower pH significantly increased the release rate. Electrophoretic analysis confirmed that the micelle coating effectively protected DNA from degradation in plasma. VSM (vibrating sample magnetometer) analysis was used to assess the magnetic properties of the SCPPF/Fe3O4/DNA micelles, revealing that encapsulation with PCS and PCPF yielded nanoparticles with desirable magnetic characteristics. However, the encapsulation value reduced the saturation magnetic properties of the samples from 13.4 to 39.3 emu/g. Furthermore, the micelles significantly improved the DNA transfer efficiency compared with that of the polyethylenimine (PEI)/DNA complex in serum, achieving 14.42% efficiency with the SCPPF/Fe3O4/DNA micelles compared with 10.6% for the PEI/DNA complex. In terms of cytotoxicity, the SCPPF/Fe3O4 micelles exhibited low toxicity to the AGS cell line.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.27.672642v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Multifunctional PolySpermine-based Nanocapsules for Targeted Gene Delivery to Gastric Cancer Cells</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Mohajeri, S., Raei, M., Bourang, S., Noruzpuor, M., Yaghoubi, H.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> In this study, multifunctional nanocapsules were developed and evaluated for targeted gene delivery to AGS gastric cancer cells. The design of the nanoparticles utilized hyperbranched polyspermine (HS) for efficient DNA condensation, polyethylene glycol (PEG) to increase nanoparticle stability and prolong circulation time via stealth properties, and dual-targeting ligands, i.e., folic acid and glucose, to improve selective binding and internalization by cancer cells. Folic acid targets folate receptors (FR[alpha]), while glucose binds glucose transporters (GLUTs), both of which are overexpressed in gastric cancer cells, thereby increasing uptake specificity. The synthesized ternary copolymers composed of polyspermine, PEG, folic acid, and glucose (PSPFG) were comprehensively characterized via multiple analytical techniques, including proton nuclear magnetic resonance (1H-NMR), Fourier transform infrared spectroscopy (FTIR), thermogravimetric analysis (TGA), and derivative thermogravimetric (DTG) analysis, to confirm their chemical structure and thermal stability. After complexation with DNA, the PSPFG/100 DNA nanocapsules were analyzed by scanning electron microscopy (SEM) and transmission electron microscopy (TEM), which revealed uniform spherical nanoparticles with a nanoscale size. Dynamic light scattering (DLS) measurements confirmed a narrow size distribution, with an average particle size of 265 [plusmn]18 nm. Biocompatibility assays using the 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide (MTT) assay demonstrated significantly reduced cytotoxicity compared with the commonly used polyethylenimine (PEI) vector. Agarose gel electrophoresis revealed strong DNA binding, effective charge neutralization, and resistance to enzymatic degradation. Importantly, fluorescence microscopy and flow cytometry analyses demonstrated high transfection efficiency in AGS cells, with the optimized PSPFG50/DNA formulation achieving a transfection rate of 53.37%. These results collectively indicate that PSPFG-based nanocarriers exhibit favorable biocompatibility and enhanced gene delivery performance, addressing major limitations of traditional polycationic vectors. These findings suggest promising potential for the clinical translation of these spermine-derived nanocapsules in gastric cancer gene therapy.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.27.672643v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Decrease in dose per fraction impairs the FLASH sparing effect in murine intestine model</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Sesink, A., Geyer, R. W., Devanand, P., Boehlen, T. T., Soutter, L., Moeckli, R., Bailat, C., Herrera, F., Grilj, V.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Purpose: FLASH radiotherapy (FLASH) can ease radiation-induced normal tissue toxicities; however, its benefit in clinically relevant fractionation protocols remains insufficiently explored. This study investigated the FLASH sparing effect under two fractionated regimens using a murine model of acute gastrointestinal toxicity. Methods and Materials: Tumor-free C57BL/6 mice received abdominal irradiation with either conventional radiotherapy (CONV) or FLASH using a 9 MeV electron beam. Three dose delivery protocols were assessed: single-fraction delivery, two equal fractions over two consecutive days, and ten equal daily fractions over two weeks. Dose escalation was performed within each protocol, while overall survival was used to monitor normal tissue sparing. The FLASH dose modifying factor (FDMF) was derived from normal tissue toxicity probability (NTCP) curves to quantify the relative protective effect of FLASH. Results: The single-fraction irradiation demonstrated a significant FLASH sparing effect, with an FDMF of 1.14. In contrast, this protective effect was diminished in the fractionated protocols, with the two-fraction regimen yielding an FDMF of 1.03, while the ten-fraction regimen showed no measurable sparing (FDMF = 1.00). Conclusions: In an acute responding model of radiation-induced abdominal toxicity, the FLASH sparing effect was substantially reduced with a two-fraction regimen and completely absent with a ten-fraction regimen. These findings suggest that the benefit of FLASH may be limited at lower doses per fraction and highlight the need for further studies in other clinically relevant models to better define the boundaries of its therapeutic applicability.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.27.672569v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Lineage-Specific Epigenomic Mapping of Proximal and Distal Fallopian Tube Epithelium Reveals Mesenchymal/Wnt-Driven Tumor Phenotype Diversification</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Wei, L., Wang, H., Li, W., Lian, J., Huang, L., Wang, Y., Li, T., Li, S., Hu, Z., Tu, Z., Sun, W., Chen, S., Zhang, S.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> High-grade serous ovarian cancer (HGSC) typically originates from the distal fallopian tube epithelium (disFTE), while the proximal epithelium (proFTE) remains less explored. Here, Using isogenic murine organoid models with common oncogenic drivers (Tp53-/-; Krasmut; Pten-/-), we show that disFTE and proFTE generate distinct tumor subtypes driven by region-specific transcriptional programs. Unlike disFTE, proFTE demonstrates continuous activation of epithelial-mesenchymal transition (EMT) across wild-type, mutant organoids, and derived tumors. Additionally, mutated proFTE organoids and their tumors exhibit persistent WNT signaling activation and grow independently of WNT signals. Chromatin profiling shows that mutant proFTE maintains a distinctive epigenetic profile with promoters associated with EMT and WNT pathways. Human proFTE with BRCAmut also shows epigenetic activation of EMT- and WNT-related genes. Tumors near the uterotubal junction (UTJ)-anatomically akin to proFTE-exhibit a pronounced EMT signature, increased Vimentin+E-cadherin+ and nuclear {beta}-catenin expression. These findings suggest proFTE-derived tumors may occupy a unique niche in gynecological malignancies, with regional identity shaping tumor phenotypes.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.27.672603v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Studying a human genetic model of lung squamous cell carcinoma with organotypic cultures and xenografts uncovers distinct advantages of each system and implicates NOTCH1 loss in tumour development</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ogden, J., Sellers, R., Oojageer, A., Sahoo, S., Dive, C., Lopez-Garcia, C.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: bioRxiv
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #caac84; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">bioinformatics, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Selecting appropriate experimental systems is crucial in cancer research, where factors such as model relevance, cost, and resource availability guide decisions. A detailed understanding of the strengths and limitations of each model helps ensure their optimal use. We recently developed a human lung squamous cell carcinoma (LUSC) model using genetically engineered human bronchial epithelial cells (hBECs). These were studied through organotypic air-liquid interface (ALI) cultures and standard in vitro assays, including proliferation, invasion, and anchorage-independent growth. However, we did not evaluate whether the same mutant hBECs behaved similarly in vivo, or if in vivo models offered distinct advantages. To address this, we conducted a comparative phenotypic analysis of mutant hBECs derived from the same donor in both ALI cultures and xenografts in immunocompromised mice. Both models followed a similar oncogenic trajectory, involving squamous differentiation and activation of Nrf2 and PI3K/Akt pathways, characteristic of the classical LUSC subtype. However, some transcriptomic differences related to an increase in microtubule formation and cell motility in xenografts emerged. Additionally, xenograft gene expression profiles more closely matched classical LUSC tumours than ALI cultures. Importantly, we observed spontaneous squamous differentiation in the absence of SOX2 overexpression and detected selection for NOTCH1 mutations in specific in vivo mutants. Truncation of NOTCH1 promoted squamous differentiation and suppressed mucociliary features in ALI cultures, underscoring its role as a potential LUSC driver. In summary, mutant hBECs in vitro and in vivo showed largely consistent phenotypes, validating both systems. However, in vivo models can enable the unbiased discovery of new genetic LUSC driver genes. This highlights the complementary value of integrating both model types in LUSC research.</p>
                <p><a href="https://www.biorxiv.org/content/10.1101/2025.08.28.672595v1?rss=1" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Uncovering novel functions of NUF2 in glioblastoma and MRI-based expression prediction.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rong-de Zhong, Yun-Sheng Liu, Qian Li, Zeng-Wei Kou, Fan-Fan Chen, Heng Wang, Na Zhang, Han Tang, Yuan Zhang, Guo-Dong Huang</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Glioblastoma multiforme (GBM) is a lethal brain tumor with limited therapies. NUF2, a kinetochore protein involved in cell cycle regulation, shows oncogenic potential in various cancers; however, its role in GBM pathogenesis remains unclear. In this study, we investigated NUF2's function and mechanisms in GBM and developed an MRI-based machine learning model to predict its expression non-invasively, and evaluated its potential as a therapeutic target and prognostic biomarker. Functional assays (proliferation, colony formation, migration, and invasion) and cell cycle analysis were conducted using NUF2-knockdown U87/U251 cells. Western blotting was performed to assess the expression levels of Œ≤-catenin and MMP-9. Bioinformatic analyses included pathway enrichment, immune infiltration, and single-cell subtype characterization. Using preoperative T1CE Magnetic Resonance Imaging sequences from 61 patients, we extracted 1037 radiomic features and developed a predictive model using Least Absolute Shrinkage and Selection Operator regression for feature selection and random forest algorithms for classification with rigorous cross-validation. NUF2 overexpression in GBM tissues and cells was correlated with poor survival (p < 0.01). Knockdown of NUF2 significantly suppressed malignant phenotypes (p < 0.05), induced G0/G1 arrest (p < 0.01), and increased sensitivity to TMZ treatment via the Œ≤-catenin/MMP9 pathway. The radiomic model achieved superior NUF2 prediction (AUC = 0.897) using six optimized features. Key features demonstrated associations with MGMT methylation and 1p/19q co-deletion, serving as independent prognostic markers. NUF2 drives GBM progression through Œ≤-catenin/MMP9 activation, establishing its dual role as a therapeutic target and a prognostic biomarker. The developed radiogenomic model enables precise non-invasive NUF2 evaluation, thereby advancing personalized GBM management. This study highlights the translational value of integrating molecular biology with artificial intelligence in neuro-oncology.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40890233/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">mbSparse: an autoencoder-based imputation method to address sparsity in microbiome data.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Changlu Qi, Yiting Cai, Guoyou He, Kai Qian, Mian Guo, Liang Cheng</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> The involvement of gut microbiota in host physiological activities is crucial, yet the high sparsity of microbiome data, marked by numerous zeros in count matrices, presents huge analytical challenges. To overcome this, we developed mbSparse, an imputation algorithm that leverages deep learning rather than traditional predefined count distributions. Utilizing a feature autoencoder for learning sample representations and a conditional variational autoencoder (CVAE) for data reconstruction, mbSparse effectively integrates these processes to enhance imputation. Our results demonstrate that mbSparse achieves exceptional accuracy, with mean squared error reductions of up to 4.1 compared to existing microbiome methods, even amid outlier samples and varying sequencing depths. In colorectal cancer analysis, mbSparse increases the detection of validated disease-associated taxa from 7 to 27, while predictive accuracy improves, as evidenced by area under the precision-recall area under the curve values rising from 0.85 to 0.93. Additionally, mbSparse addresses non-biological zeros by restoring over 88% of removed counts and achieving a Pearson correlation of 0.9354 at a 10% removal rate, preserving essential taxonomic relationships. Finally, our exploration of mbSparse variants reveals that the CVAE is critical for enhancing accuracy, providing valuable insights for further optimizing microbiome data imputation techniques.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40888610/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Assessing PD-L1 expression in non-small cell lung carcinoma: a prospective study of matched fine-needle aspirates, core biopsies, and resection specimens using alcohol and forming fixatives. </h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Haragan A, Kipling N, Shackcloth M, Gosney JR, Davies MP, Field JK.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> PD-L1 expression for the prediction of response to immune-checkpoint blockade remains the most broadly utilised clinically validated biomarker in a range of tumour types. In this study, we aimed to assess, in a prospectively collected matched cohort, the impact of sampling technique and both formalin and alcohol fixation on PD-L1 expression and heterogeneity in non-small cell lung carcinoma (NSCLC). Patients undergoing surgical resection for NSCLC were consented. Surgical specimens were received directly from theatre and sampled fresh to produce two sets of core biopsies, two fine-needle aspirates (FNAs) and two whole-block tissue sections from each specimen. A matched biopsy, FNA, and whole-block were placed into formalin or an alcohol-based fixative (Cytolyt‚Ñ¢) prior to PD-L1 immunohistochemistry assessment. A total of 114 specimens from 57 patients were included. All whole-block cases (100%), 92% of core biopsies, and 88% of FNAs were adequate for PD-L1 expression analysis. Fixation had no significant impact on adequacy, but cytology specimens fixed in alcohol showed a significant reduction in PD-L1 expression, with 25% of cases placed into different clinically relevant categories of PD-L1 expression. PD-L1 expression by immunochemistry is an exemplar of the challenges of utilising a heterogeneously expressed protein-based predictive biomarker. Regardless of sampling technique, a good quality biopsy or FNA is likely to give a statistically representative PD-L1 expression, although expression ranges close to clinically relevant cut-offs of 1% and 50% remain a source of potential discordance.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40853762/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Using extension-based mRNA display to design antibody-like proteinogenic peptides for human PD-L1. </h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Ong JN, Grindel BJ, Rankin SA, Naylon SH, Srinivasamani A, Trusz GJ, Liang X, Uddin MN, Fuller L, Curran M, Roche SP, Takahashi TT, Roberts RW, Millward SW.</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: MED
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-09-01
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Many peptide drugs rely on nonproteinogenic amino acids and chemical modifications for improved activity and proteolytic stability. However, these features also make drug production expensive and challenging to scale. Here, we engineered small, linear, proteinogenic peptides that bind human programmed death-ligand 1 (hPD-L1) with high affinity and stability using mRNA display affinity maturation. The resulting peptides, SPAM2 and SPAM3, have antibody-like affinities for hPD-L1 (dissociation constants between ~250 and 300 pM) and are selective for hPD-L1. Both SPAM2 and SPAM3 compete with hPD-L1 ligands known to interact with the programmed cell death protein 1 site and are stable in human serum. SPAM3 bound human glioma D423 cells with high affinity in flow cytometry experiments comparable to that of a clinical therapeutic antibody. These results support the use of affinity maturation selections to dramatically enhance the biophysical properties of linear, proteinogenic peptides for translational applications.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40852896/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        

            <div style="margin-bottom: 30px;">
                <h2 style="color: #000000; font-size: 22px;">Evaluating rapid evaporative ionization mass spectrometry profiles of human breast cancer cells using a 3D in vitro assay.</h2>
                <p style="font-size: 14px; text-align: justify; margin-top: -10px; margin-bottom: 10px;"> Rachel E Rubino, Martin Kaufmann, Amoon Jamzad, Natasha Iaboni, Kevin Yi Mi Ren, Jian Yu, Haidy Metwally, Manuela Kunz, John F Rudan, Parvin Mousavi, Gabor Fichtinger, Richard Oleschuk, Christopher J B Nicol</p>
         
                <table role="presentation" width="100%" cellspacing="0" cellpadding="0" style="font-size:16px; margin-bottom: 12px;">
                    <tr>
                        <td align="left" style="font-style: italic;">
                            Source: PubMed
                        </td>
                        <td align="right" style="font-style: italic;">
                            Published: 2025-08-18
                        </td>
                    </tr>
                </table>
            
                <p><span style="background-color: #e8fef1; color: #000000; padding: 4px 8px; border-radius: 12px; font-size: 14px; box-shadow: 1px 1px 3px rgba(0, 0, 0, 1);">machine learning, cancer biology</span></p>
                <p style="font-size: 16px; text-align: justify;"><span style="font-weight: bold; font-size: 16px;">Abstract:</span> Rapid evaporative ionization mass spectrometry (REIMS) enables near real-time sampling and classification of tissues on the basis of lipid and small molecule profiles. Because samples are ablated during REIMS, validation of mass spectra by assignment of class labels using gold-standard methods remains challenging. Further, determining the number of abnormal cells within a mixture of normal cells by REIMS remains an elusive but critical parameter when evaluating the utility of REIMS for use in clinical settings. To address this, we developed a three-dimensional assay based on fluorescently labelled human breast cell lines, MCF-10A (pseudo-normal) and MDA-MB-231 (malignant), embedded in matrigel cubes that enables calculation of cell numbers ablated by REIMS by measuring fluorescence before and after REIMS sampling. We observed a reasonably uniform distribution of cells throughout the matrigel matrix, which yielded REIMS spectra rich in complex glycerophospholipids resembling solid tissues and cell pellets. A non-linear relationship between signal intensity and the number of cells ablated was observed. We trained multivariate models using 10% increments of MDA-MB-231 cells mixed with MCF-10A, and blindly tested the models on separate cubes containing 1-100% MDA-MB-231s. Percent difference from target MDA-MB-231 cell composition was within 34% for cubes containing as little as 5% MDA-MB-231 cells. Overall, our study highlights how a simple 3D in vitro approach may help reveal the quantitative potential of mass spectral profiling methods such as REIMS to assist with validating REIMS spectra containing mixtures of different cell lines. This assay platform may help calibrate REIMS-based methods to recognize specific cell type mixtures encountered at surgical resection margins.</p>
                <p><a href="https://pubmed.ncbi.nlm.nih.gov/40820122/" style="color: #1a0dab; font-size: 16px;">Read full paper</a></p>
            </div>
            <hr style="border: none; border-top: 1px solid #ccc;">
        
        <p style="font-size: 14px; color: gray;">
            You‚Äôre receiving this email because you're using
            <a href="https://github.com/felipevzps/paper-trackr" style="color: #1a0dab; text-decoration: none;">paper-trackr</a>.
            Stay curious!
        </p>
    </div>
</body>
</html>

